<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/1/163" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/1/163.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/1/163.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/1/163.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/1/163/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/1/163.atom" />
<meta name="HW.pisa" content="ggg;4/1/163" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Pattern and Distribution of Deleterious Mutations in Maize" />
<meta name="DC.Identifier" content="10.1534/g3.113.008870" />
<meta name="DC.Date" content="2014-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Mezmouk and Ross-Ibarra. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Most nonsynonymous mutations are thought to be deleterious because of their effect on protein sequence and are expected to be removed or kept at low frequency by the action of natural selection. Nonetheless, the effect of positive selection on linked sites or drift in small or inbred populations may also impact the evolution of deleterious alleles. Despite their potential to affect complex trait phenotypes, deleterious alleles are difficult to study precisely because they are often at low frequency. Here, we made use of genome-wide genotyping data to characterize deleterious variants in a large panel of maize inbred lines. We show that, despite small effective population sizes and inbreeding, most putatively deleterious SNPs are indeed at low frequencies within individual genetic groups. We find that genes associated with a number of complex traits are enriched for deleterious variants. Together, these data are consistent with the dominance model of heterosis, in which complementation of numerous low-frequency, weak deleterious variants contribute to hybrid vigor." />
<meta name="DC.Contributor" content="Sofiane Mezmouk" />
<meta name="DC.Contributor" content="Jeffrey Ross-Ibarra" />
<meta name="article:published_time" content="2014-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Pattern and Distribution of Deleterious Mutations in Maize" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Most nonsynonymous mutations are thought to be deleterious because of their effect on protein sequence and are expected to be removed or kept at low frequency by the action of natural selection. Nonetheless, the effect of positive selection on linked sites or drift in small or inbred populations may also impact the evolution of deleterious alleles. Despite their potential to affect complex trait phenotypes, deleterious alleles are difficult to study precisely because they are often at low frequency. Here, we made use of genome-wide genotyping data to characterize deleterious variants in a large panel of maize inbred lines. We show that, despite small effective population sizes and inbreeding, most putatively deleterious SNPs are indeed at low frequencies within individual genetic groups. We find that genes associated with a number of complex traits are enriched for deleterious variants. Together, these data are consistent with the dominance model of heterosis, in which complementation of numerous low-frequency, weak deleterious variants contribute to hybrid vigor.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/01/01" />
<meta name="citation_mjid" content="ggg;4/1/163" />
<meta name="citation_id" content="4/1/163" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/1/163" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/1/163.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/1/163.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/1/163.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.008870" />
<meta name="citation_pmid" content="24281428" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="163" />
<meta name="citation_lastpage" content="171" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sofiane Mezmouk" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California–Davis, Davis, California 95616" />
<meta name="citation_author" content="Jeffrey Ross-Ibarra" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California–Davis, Davis, California 95616" />
<meta name="citation_author_institution" content="Center for Population Biology and Genome Center, University of California–Davis, Davis, California 95616" />
<meta name="citation_author_email" content="rossibarra@ucdavis.edu" />
<meta name="citation_reference" content="citation_title=Proceedings of the Forty-Ninth Annual Corn and Sorghum Research Conference;citation_title=The power and deceit of QTL experiments: lessons from comparative QTL studies;citation_author=W. D. Beavis;citation_pages=250-266;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc.;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing.;citation_pages=289-300;citation_volume=57;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=B. Boeckmann;citation_author=A. Bairoch;citation_author=R. Apweiler;citation_author=M. Blatter;citation_author=A. Estreicher;citation_title=The SWISS-PROT protein knowledgebase and its supplement TrEMBL in 2003;citation_pages=365-370;citation_volume=31;citation_year=2003;citation_issue=1;citation_pmid=12520024;citation_doi=10.1093/nar/gkg095" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=Y. Brandvain;citation_author=T. Slotte;citation_author=K. M. Hazzouri;citation_author=S. I. Wright;citation_author=G. Coop;citation_title=Genomic identification of founding haplotypes reveals the history of the selfing species capsella rubella.;citation_pages=e1003754;citation_volume=9;citation_year=2013;citation_pmid=24068948;citation_doi=10.1371/journal.pgen.1003754" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Cao;citation_author=K. Schneeberger;citation_author=S. Ossowski;citation_author=T. Günther;citation_author=S. Bender;citation_title=Whole-genome sequencing of multiple Arabidopsis thaliana populations.;citation_pages=956-963;citation_volume=43;citation_year=2011;citation_issue=10;citation_pmid=21874002;citation_doi=10.1038/ng.911" />
<meta name="citation_reference" content="citation_journal_title=Annual Review of Ecology and Systematics;citation_journal_abbrev=Annu. Rev. Ecol. Syst.;citation_author=D. Charlesworth;citation_author=B. Charlesworth;citation_title=Inbreeding Depression And Its Evolutionary Consequences;citation_pages=237-268;citation_volume=18;citation_year=1987;citation_issue=1;citation_doi=10.1146/annurev.es.18.110187.001321" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=D. Charlesworth;citation_author=J. H. Willis;citation_title=The genetics of inbreeding depression.;citation_pages=783-796;citation_volume=10;citation_year=2009;citation_issue=11;citation_pmid=19834483;citation_doi=10.1038/nrg2664" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=D. Charlesworth;citation_author=M. T. Morgan;citation_author=B. Charlesworth;citation_title=Mutation Accumulation in Finite Populations;citation_pages=321-325;citation_volume=84;citation_year=1993;citation_issue=5" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J.-M. Chia;citation_author=C. Song;citation_author=P. J. Bradbury;citation_author=D. Costich;citation_author=N. de Leon;citation_title=Maize HapMap2 identifies extant variation from a genome in flux.;citation_pages=803-807;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22660545;citation_doi=10.1038/ng.2313" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Chun;citation_author=J. C. Fay;citation_title=Evidence for hitchhiking of deleterious mutations within the human genome.;citation_pages=e1002240-e1002240;citation_volume=7;citation_year=2011;citation_issue=8;citation_pmid=21901107;citation_doi=10.1371/journal.pgen.1002240" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. C. Cohen;citation_author=R. S. Kiss;citation_author=A. Pertsemlidis;citation_author=Y. L. Marcel;citation_author=R. McPherson;citation_title=Multiple Rare Alleles Contribute to Low Plasma Levels of HDL Cholesterol;citation_pages=869-872;citation_volume=305;citation_year=2004;citation_issue=5685;citation_pmid=15297675;citation_doi=10.1126/science.1099870" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. P. Cook;citation_author=M. D. McMullen;citation_author=J. B. Holland;citation_author=F. Tian;citation_author=P. Bradbury;citation_title=Genetic architecture of maize kernel composition in the nested association mapping and inbred association panels.;citation_volume=158;citation_year=2012;citation_pmid=22135431;citation_doi=10.1104/pp.111.185033" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. P. Cummings;citation_author=M. T. Clegg;citation_title=Nucleotide sequence diversity at the alcohol dehydrogenase 1 locus in wild barley (Hordeum vulgare ssp. spontaneum): An evaluation of the background selection hypothesis;citation_pages=5637-5642;citation_volume=95;citation_year=1998;citation_issue=10;citation_pmid=9576936;citation_doi=10.1073/pnas.95.10.5637" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=S. P. Dickson;citation_author=K. Wang;citation_author=I. Krantz;citation_author=H. Hakonarson;citation_author=D. B. Goldstein;citation_title=Rare variants create synthetic genome-wide associations.;citation_pages=e1000294-e1000294;citation_volume=8;citation_year=2010;citation_issue=1;citation_pmid=20126254;citation_doi=10.1371/journal.pbio.1000294" />
<meta name="citation_reference" content="citation_title=Reports of the Connecticut Agricultural Experiment Station for Years, Volume Inbreeding in Corn;citation_author=E. East;citation_year=1907–1908" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. C. Fay;citation_author=G. J. Wyckoff;citation_author=C. I. Wu;citation_title=Positive and Negative Selection on the Human Genome;citation_pages=1227-1234;citation_volume=158;citation_year=2001;citation_issue=3;citation_pmid=11454770" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Felsenstein;citation_title=THE EVOLUTIONARY ADVANTAGE OF RECOMBINATION;citation_pages=737-756;citation_volume=78;citation_year=1974;citation_issue=2;citation_pmid=4448362" />
<meta name="citation_reference" content="citation_title=The Genetical Theory of Natural Selection;citation_author=R. A. Fisher;citation_year=1930" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=S. A. Flint-Garcia;citation_author=A.-C. Thuillet;citation_author=J. Yu;citation_author=G. Pressoir;citation_author=S. M. Romero;citation_title=Maize association population: a high-resolution platform for quantitative trait locus dissection.;citation_pages=1054-1064;citation_volume=44;citation_year=2005;citation_issue=6;citation_pmid=16359397;citation_doi=10.1111/j.1365-313X.2005.02591.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. A. Flint-Garcia;citation_author=E. S. Buckler;citation_author=P. Tiffin;citation_author=E. Ersoz;citation_author=N. M. Springer;citation_title=Heterosis is prevalent for multiple traits in diverse maize germplasm.;citation_pages=e7433-e7433;citation_volume=4;citation_year=2009;citation_issue=10;citation_pmid=19823591;citation_doi=10.1371/journal.pone.0007433" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Fournier-Level;citation_author=A. Korte;citation_author=M. Cooper;citation_author=M. Nordborg;citation_author=J. Schmitt;citation_title=A Map of Local Adaptation in Arabidopsis thaliana;citation_pages=86-89;citation_volume=334;citation_year=2011;citation_issue=6052;citation_pmid=21980109;citation_doi=10.1126/science.1209271" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=G. Gibson;citation_title=Rare and common variants: twenty arguments.;citation_pages=135-145;citation_volume=13;citation_year=2012;citation_pmid=22251874;citation_doi=10.1038/nrg3118" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Glémin;citation_author=J. Ronfort;citation_author=T. Bataillon;citation_title=Patterns of Inbreeding Depression and Architecture of the Load in Subdivided Populations;citation_pages=2193-2212;citation_volume=165;citation_year=2003;citation_issue=4;citation_pmid=14704197" />
<meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.;citation_author=D. B. Goldstein;citation_title=Common genetic variation and human traits.;citation_volume=360;citation_year=2009;citation_pmid=19369660;citation_doi=10.1056/NEJMp0806284" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. A. Gore;citation_author=J.-M. Chia;citation_author=R. J. Elshire;citation_author=Q. Sun;citation_author=E. S. Ersoz;citation_title=A First-Generation Haplotype Map of Maize;citation_pages=1115-1117;citation_volume=326;citation_year=2009;citation_issue=5956;citation_pmid=19965431;citation_doi=10.1126/science.1177837" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T. Gossmann;citation_author=B. Song;citation_author=A. Windsor;citation_author=T. Mitchell-Olds;citation_author=C. Dixon;citation_title=Genome wide analyses reveal little evidence for adaptive evolution in many plant species.;citation_volume=27;citation_year=2010;citation_pmid=20299543;citation_doi=10.1093/molbev/msq079" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. Günther;citation_author=K. J. Schmid;citation_title=Deleterious amino acid polymorphisms in Arabidopsis thaliana and rice.;citation_pages=157-168;citation_volume=121;citation_year=2010;citation_issue=1;citation_pmid=20198468;citation_doi=10.1007/s00122-010-1299-4" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=P. R. Haddrill;citation_author=D. L. Halligan;citation_author=D. Tomaras;citation_author=B. Charlesworth;citation_title=Reduced efficacy of selection in regions of the Drosophila genome that lack crossing over.;citation_pages=R18-R18;citation_volume=8;citation_year=2007;citation_issue=2;citation_pmid=17284312;citation_doi=10.1186/gb-2007-8-2-r18" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. Hill;citation_author=A. Robertson;citation_title=The effect of linkage on limits to artificial selection.;citation_pages=269-294;citation_volume=8;citation_year=1966;citation_issue=3;citation_pmid=5980116;citation_doi=10.1017/S0016672300010156" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. B. Hufford;citation_author=X. Xu;citation_author=J. van Heerwaarden;citation_author=T. Pyhäjärvi;citation_author=J.-M. Chia;citation_title=Comparative population genomics of maize domestication and improvement.;citation_pages=808-811;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22660546;citation_doi=10.1038/ng.2309" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. L. Hughes;citation_title=Evidence for abundant slightly deleterious polymorphisms in bacterial populations.;citation_volume=169;citation_year=2005;citation_pmid=15545641;citation_doi=10.1534/genetics.104.036939" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. B. Joseph;citation_author=D. W. Hall;citation_title=Spontaneous Mutations in Diploid Saccharomyces cerevisiae: More Beneficial Than Expected;citation_pages=1817-1825;citation_volume=168;citation_year=2004;citation_issue=4;citation_pmid=15611159;citation_doi=10.1534/genetics.104.033761" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. M. Kang;citation_author=N. A. Zaitlen;citation_author=C. M. Wade;citation_author=A. Kirby;citation_author=D. Heckerman;citation_title=Efficient Control of Population Structure in Model Organism Association Mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_issue=3;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_title=The Neutral Theory of Molecular Evolution.;citation_author=M. Kimura;citation_year=1983" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=R. Lande;citation_title=Risk of population extinction from fixation of new deleterious mutations.;citation_pages=1460-1469;citation_volume=48;citation_year=1994;citation_doi=10.2307/2410240" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. A. Larkin;citation_author=G. Blackshields;citation_author=N. P. Brown;citation_author=R. Chenna;citation_author=P. A. McGettigan;citation_title=Clustal w and clustal x version 2.0.;citation_volume=23;citation_year=2007;citation_pmid=17846036;citation_doi=10.1093/bioinformatics/btm404" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. J. Larsson;citation_author=A. E. Lipka;citation_author=E. S. Buckler;citation_title=Lessons from dwarf8 on the strengths and weaknesses of structured association mapping.;citation_pages=e1003246;citation_volume=9;citation_year=2013;citation_pmid=23437002;citation_doi=10.1371/journal.pgen.1003246" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. E. Lohmueller;citation_author=A. R. Indap;citation_author=S. Schmidt;citation_author=A. R. Boyko;citation_author=R. D. Hernandez;citation_title=Proportionally more deleterious genetic variation in European than in African populations.;citation_pages=994-997;citation_volume=451;citation_year=2008;citation_issue=7181;citation_pmid=18288194;citation_doi=10.1038/nature06611" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=J. Lu;citation_author=T. Tang;citation_author=H. Tang;citation_author=J. Huang;citation_author=S. Shi;citation_title=The accumulation of deleterious mutations in rice genomes: a hypothesis on the cost of domestication.;citation_pages=126-131;citation_volume=22;citation_year=2006;citation_issue=3;citation_pmid=16443304;citation_doi=10.1016/j.tig.2006.01.004" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=M. S. Lynch;citation_author=W. Gabriel;citation_title=Mutation load and the survival of small populations.;citation_pages=1725-1737;citation_volume=44;citation_year=1990;citation_doi=10.2307/2409502" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. D. McMullen;citation_author=S. Kresovich;citation_author=H. S. Villeda;citation_author=P. Bradbury;citation_author=H. Li;citation_title=Genetic Properties of the Maize Nested Association Mapping Population;citation_pages=737-740;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661427;citation_doi=10.1126/science.1174320" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Messmer;citation_author=A. Melchinger;citation_author=M. Lee;citation_author=W. Woodman;citation_author=K. Lamkey;citation_title=Genetic diversity among progenitors and elite lines from the Iowa Stiff Stalk Synthetic (Bsss) maize population: comparison of allozyme and RFLP data.;citation_pages=97-107;citation_volume=38;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. C. Ng;citation_author=S. Henikoff;citation_title=SIFT: predicting amino acid changes that affect protein function;citation_pages=3812-3814;citation_volume=31;citation_year=2003;citation_issue=13;citation_pmid=12824425;citation_doi=10.1093/nar/gkg509" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=P. C. Ng;citation_author=S. Henikoff;citation_title=Predicting the effects of amino acid substitutions on protein function.;citation_pages=61-80;citation_volume=7;citation_year=2006;citation_pmid=16824020;citation_doi=10.1146/annurev.genom.7.080505.115630" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. Ohta;citation_title=Slightly deleterious mutant substitutions in evolution.;citation_pages=96-98;citation_volume=246;citation_year=1973;citation_issue=5428;citation_pmid=4585855;citation_doi=10.1038/246096a0" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=T. Paape;citation_author=T. Bataillon;citation_author=P. Zhou;citation_author=J. Y. Kono T;citation_author=R. Briskine;citation_title=Selection, genome-wide fitness effects and evolutionary rates in the model legume Medicago truncatula.;citation_pages=3525-3538;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12329" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. Romay;citation_author=M. Millard;citation_author=J. Glaubitz;citation_author=J. Peiffer;citation_author=K. Swarts;citation_title=Comprehensive genotyping of the USA national maize inbred seed bank.;citation_pages=R55;citation_volume=14;citation_year=2013;citation_pmid=23759205;citation_doi=10.1186/gb-2013-14-6-r55" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. S. Schnable;citation_author=D. Ware;citation_author=R. S. Fulton;citation_author=J. C. Stein;citation_author=F. Wei;citation_title=The B73 Maize Genome: Complexity, Diversity, and Dynamics;citation_pages=1112-1115;citation_volume=326;citation_year=2009;citation_issue=5956;citation_pmid=19965430;citation_doi=10.1126/science.1178534" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=J. C. Schnable;citation_author=M. Freeling;citation_author=E. Lyons;citation_title=Genome-wide analysis of syntenic gene deletion in the grasses.;citation_volume=4;citation_year=2012;citation_pmid=22275519;citation_doi=10.1093/gbe/evs009" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=G. H. Shull;citation_title=The composition of a field of maize.;citation_pages=296-301;citation_volume=4;citation_year=1908" />
<meta name="citation_reference" content="citation_journal_title=Neurobiology of aging;citation_journal_abbrev=Neurobiol Aging;citation_author=R. Smigrodzki;citation_author=J. Parks;citation_author=W. D. Parker;citation_title=High frequency of mitochondrial complex I mutations in Parkinson's disease and aging.;citation_pages=1273-1281;citation_volume=25;citation_year=2004;citation_issue=10;citation_pmid=15465623;citation_doi=10.1016/j.neurobiolaging.2004.02.020" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. A. Stone;citation_author=A. Sidow;citation_title=Physicochemical constraint violation by missense substitutions mediates impairment of protein function and disease severity.;citation_volume=15;citation_year=2005;citation_pmid=15965030;citation_doi=10.1101/gr.3804205" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Protocols;citation_journal_abbrev=Cold Spring Harb Protoc;citation_author=J. Strable;citation_author=M. J. Scanlon;citation_title=Maize (Zea mays): A Model Organism for Basic and Applied Research in Plant Biology;citation_pages=pdb.emo132-pdb.emo132;citation_volume=2009;citation_year=2009;citation_issue=10;citation_pmid=20147033;citation_doi=10.1101/pdb.emo132" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb.);citation_author=A. Tellier;citation_author=I. Fischer;citation_author=C. Merino;citation_author=H. Xia;citation_author=L. Camus-Kulandaivelu;citation_title=Fitness effects of derived deleterious mutations in four closely related wild tomato species with spatial structure.;citation_pages=189-199;citation_volume=107;citation_year=2011;citation_pmid=21245893;citation_doi=10.1038/hdy.2010.175" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. Thornton;citation_title=libsequence: a C++ class library for evolutionary genetic analysis;citation_pages=2325-2327;citation_volume=19;citation_year=2003;citation_issue=17;citation_pmid=14630667;citation_doi=10.1093/bioinformatics/btg316" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. R. Thornton;citation_author=A. J. Foran;citation_author=A. D. Long;citation_title=Properties and modeling of GWAS when complex disease risk is due to non-complementing, deleterious mutations in genes of large effect.;citation_pages=e1003258;citation_volume=9;citation_year=2013;citation_pmid=23437004;citation_doi=10.1371/journal.pgen.1003258" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=C. Toomajian;citation_author=T. T. Hu;citation_author=M. J. Aranzana;citation_author=C. Lister;citation_author=C. Tang;citation_title=A nonparametric test reveals selection for rapid flowering in the Arabidopsis genome.;citation_pages=e137-e137;citation_volume=4;citation_year=2006;citation_issue=5;citation_pmid=16623598;citation_doi=10.1371/journal.pbio.0040137" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. Wang;citation_author=W. Hill;citation_author=D. Charlesworth;citation_author=B. Charlesworth;citation_title=Dynamics of inbreeding depression due to deleterious mutations in small populations: mutation parameters and inbreeding rate.;citation_pages=165-178;citation_volume=74;citation_year=1999;citation_issue=2;citation_pmid=10584559;citation_doi=10.1017/S0016672399003900" />
<meta name="citation_reference" content="citation_journal_title=Heridity;citation_author=M. Whitlock;citation_author=P. Ingvarsson;citation_author=T. Hatfield;citation_title=Local drift load and the heterosis of interconnected populations.;citation_pages=452-457;citation_volume=84;citation_year=2000;citation_doi=10.1046/j.1365-2540.2000.00693.x" />
<meta name="citation_reference" content="citation_journal_title=Ann. Zool. Fenn.;citation_author=M. C. Whitlock;citation_author=C. K. Grisworld;citation_author=A. D. Peters;citation_title=Compensating for meltdown: the critical effective size of a population with deleterious and compensatory mutations.;citation_pages=169-183;citation_volume=40;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yu;citation_author=G. Pressoir;citation_author=W. H. Briggs;citation_author=I. Vroh Bi;citation_author=M. Yamasaki;citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.;citation_pages=203-208;citation_volume=38;citation_year=2006;citation_issue=2;citation_pmid=16380716;citation_doi=10.1038/ng1702" />
<meta name="citation_reference" content="citation_journal_title=Cancer Research;citation_journal_abbrev=Cancer Res.;citation_author=Y. Zhu;citation_author=M. Spitz;citation_author=C. Amos;citation_author=J. Lin;citation_author=M. Schabath;citation_title=An Evolutionary Perspective on Single-Nucleotide Polymorphism Screening in Molecular Cancer Epidemiology;citation_pages=2251-2257;citation_volume=64;citation_year=2004;citation_issue=6;citation_pmid=15026370;citation_doi=10.1158/0008-5472.CAN-03-2800" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Pattern and Distribution of Deleterious Mutations in Maize" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1.cover-source.jpg" />
<meta name="twitter:description" content="Most nonsynonymous mutations are thought to be deleterious because of their effect on protein sequence and are expected to be removed or kept at low frequency by the action of natural selection. Nonetheless, the effect of positive selection on linked sites or drift in small or inbred populations may also impact the evolution of deleterious alleles. Despite their potential to affect complex trait phenotypes, deleterious alleles are difficult to study precisely because they are often at low frequency. Here, we made use of genome-wide genotyping data to characterize deleterious variants in a large panel of maize inbred lines. We show that, despite small effective population sizes and inbreeding, most putatively deleterious SNPs are indeed at low frequencies within individual genetic groups. We find that genes associated with a number of complex traits are enriched for deleterious variants. Together, these data are consistent with the dominance model of heterosis, in which complementation of numerous low-frequency, weak deleterious variants contribute to hybrid vigor." />
<meta name="og-title" property="og:title" content="The Pattern and Distribution of Deleterious Mutations in Maize" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/1/163" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Most nonsynonymous mutations are thought to be deleterious because of their effect on protein sequence and are expected to be removed or kept at low frequency by the action of natural selection. Nonetheless, the effect of positive selection on linked sites or drift in small or inbred populations may also impact the evolution of deleterious alleles. Despite their potential to affect complex trait phenotypes, deleterious alleles are difficult to study precisely because they are often at low frequency. Here, we made use of genome-wide genotyping data to characterize deleterious variants in a large panel of maize inbred lines. We show that, despite small effective population sizes and inbreeding, most putatively deleterious SNPs are indeed at low frequencies within individual genetic groups. We find that genes associated with a number of complex traits are enriched for deleterious variants. Together, these data are consistent with the dominance model of heterosis, in which complementation of numerous low-frequency, weak deleterious variants contribute to hybrid vigor." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1.cover-source.jpg" />
    <title>The Pattern and Distribution of Deleterious Mutations in Maize | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"lDT2Jfqav7hQFo_8QThii_1pXLhqsfioAnyccwVVLzw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459622","apath":"\/ggg\/4\/1\/163.atom","pisa":"ggg;4\/1\/163","ac":{"\/ggg\/4\/1\/163.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/1\/163.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/1\/163"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/1\/163"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/1\/163":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022EVku4Qd\\\\\\\/eHYzfysK5F7sPRwXUI5tzMeD8FNAok+UiSjiCEWSK0kNDSMR5yeND48idT1JAjv5FK8Wl2gAc0OlHmcaKUCNMezSnx928yf\\\\\\\/x9FbSPlmh03GbYwomLlYbJnzwLxhmJDJmrwrwLLySWgKod3M2zg5BPgu\\\\\\\/zSyyT3pSxEIcaDh0IGF5NxvtEr6nsd5f4HQGxgg4UGkz6hTNEgAG2WQlmMZ2dsqgJZxmoA6r8C89TnRoWL2p3Z343\\\\\\\/Ag9mGZjObKoC2tT381bA\\\\\\\/3xn53SF2AFKrMpcgyJGDa8ju84Sk\\\\\\\/R9IM+o0QGC2BpmLufyTShW34AUoif7UnkU4vfrnmjdZUz7HyXNSfHbFNR6qu1uH\\\\\\\/FulhAYYs584LahDuOlkmi7t35J8WJEUfO7IVLQe4PYgjuaX8gIIq3mwpkTxjTzc0\\\\\\\/z4DrWv5iUgW6qKtDNGVMjXqJp5LvyK9SE4EHd5c1vFRMbvdZcyvrGDGNNlDN3UKtzeuskUXOkmSHVmHP1+1cpfOPPBXFkmOJDgrqW7BNiawEqdoW6D8311PQ6bvQoRKXeqvbc3mM06u9mozAYKOO\\\\\\\/GsxPHhUKAGajHzUNlXsPMBzHY8J1P\\\\\\\/4I6YrpMZmrf38+b1LTV7UV62thdTIDbsXJfPRoIi+VT7aUEwtcXPucC3oaEm9E6vAAJGuxbpQ+wdrHVCY2g2Dt6Phcz+jJi63Xnd0pLlzzaopzf5aJQn4GMgFXxDtXyQNMcaUn3p2gzCz9z1S1LGz\\\\\\\/f4z9uktIP\\\\\\\/dQeDgmYjOS84RiXdU3QgC2MR9Z5qd5OB0j4V\\\\\\\/1ba7FwysgaX+gXqmql5JEWNeIs4obYsTILoTK45OC6h5rpfLa8SNGP43HT26FgW9A7\\\\\\\/vrbHcdhl5YrK8IXHEn0k8f1+ZK8U67xWYfV0v9yGCClAHwePoySjmAzf0U1SSQx079OQgvwyFgXHxjwg7ygbgYmpIlZy6\\\\\\\/RqTNegy98nTvD3tep\\\\\\\/tXyYlYECQhAmubkikm6tcW30ic+a50yD\\\\\\\/SYUpI95y9faxhNy0f4ThO7V51AvtxYldc2bCHhZ2CUEkVjq5ICNK\\\\\\\/IGWNhJ+kkwbX+Un36jKdGQoVQTr\\\\\\\/1pOAe7oOQnn0CxRgz3NnZ25d57HZPp8ip0mwEFNU6C+g4aFTrs1I7gFqlqvWR3BAN9rW\\\\\\\/+KL\\\\\\\/XKYLnhwTuix4qyZmZOcw\\\\\\\/eJNavdqKn5RuvBPcTKbbPXTQgwoxrViHNDYLUvJOs\\\\\\\/2LLpIiywgtSGr0iXiguB+5KrHJ5GcT\\\\\\\/FCw0nytnaNUg7Y\\\u0022,\\\u0022iv\\\u0022:\\\u0022MtR6upePZClg8+0NQFqbMw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c81bc67428b7618ab386a675c083ac456b7ff4bac782e5fa5dc415072a4b4aba\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225+EVuV3RU9QWK6REIuSw6Dn1jESWwINiy3tt4P9l8MOsmqYvk9dGXWnngUjiiZDPMS0\\\\\\\/FV3xXRYtydK5EbTGmsFk8aCNangZxfe\\\\\\\/PXXp+cMsiM6ZSsTuh+gJOzFibDbT+Ib73Btx+ghRe1tHqFJRYrDLH8vdNmq+g\\\\\\\/gexnSuwUVVBor4iN+pD5CkLxjOjdtl074LFaDYePeWZqGTS4ulRqDJlqGezzABDKjIG1yRuUS+3x56x59bjRf5ccqXxcmqNjg9rdwpvdB6Ici1B\\\\\\\/SuD8sDckPyHvdIJqCJXlvBuQGsW7w0J2fSn3SefTdiqYMGAHe72Ln05bzJWpKe6bMqZOioil\\\\\\\/n2dAJ1BKrDXrG\\\\\\\/PiBZeDo8J1hrhr1mkmE2TfyBQYv6uPSoOJw2iW2mw1aURcahWOovJVAidQUIFShtGiyR2FIHh3yypKCfquS+tx8yhk6l2JbsXf6cX1DYh1TJiqvouDEwQKquwNO0ni06c9EueCtU34C9t79Zbd4OBQinz50DpFUh\\\\\\\/ggFem7GpKZa8dY0849h1Bi+Ofn4VUusdyGan44sF4QSYcq+lIG0txs+lpCyDhN\\\\\\\/aq0y\\\\\\\/oDHVijNCed1Bn+jHs7yrTKWOEg70BXlolex0cQIcr3Nyw5vRbf8lnV4oM5\\\\\\\/CskuJEy2ex2LacnzDXlffJEKTYqP4zVxEyXOzfhFnnZzrtVhubFTv4vCoRWyc+2P0scTAKsg6JH99WeQ6q4c+h\\\\\\\/2iWzzJaWczN1BKH6x3pOyyub+sU+VrPjsp5XqTQ65eV\\\\\\\/I4enHqOs7aiAbUh4b3eC6\\\\\\\/ZFwizE1g3ZsR21GA1HoAw0NuZyJaHYKTx7UP3DKxeTHinaeAE8xzHrwDVedtsUkiQPEqZ6OsV1G5ZhtWn0OLOhgk6LQHYXFqGRd24By\\\\\\\/JIxIAOTIz22w4j1vFGmcR2pNFZkfn5N2mMLE+M2gnpARHHlO\\\\\\\/0FWdqu2LfXXcd6ijFGc8rE4fE85N4H3iUotNF\\\\\\\/qifFe2vVv3wDY\\\\\\\/AcqqD0BUVDUD7mRDzRF9mVUDOINFl+0Z0NQvoaaAVdY\\\\\\\/BDmtwdTZBmV7iC2y5tNzof6FJYQwa6pchlylvQzEILspk6Vy\\\\\\\/Uf37te1kxnij0PGtH23iuItFc+PITQgvkWpxVJVr+ovgEEIw3T+8s6oJMKaFXrgMcwZirw5tfaucK00i3UD+Pp5rpxltIKfsq0xfHWaT937yoFKnkK5K9hpEDqpT6OHiz8\\\\\\\/yms3UXTyy0YSDxNsu6armPMM7QccZPj46qi5ndiC8rlihohYOz\\\\\\\/h+DFPjXegE6NyNpWacl1RyG+XjHphu60hOFF8A1aapcUbYqb38MVA2hKEd563uBpvi9UymqZ0SuzUpkGjpmMFR\\\\\\\/HGrPmLddDtTa6st4ymhH5lfmrx+gK5bykvBg6YV1MfDVgRTPNMYpiRhJkh2Bt0Vn\\\\\\\/fTquNRQJ\\\\\\\/RHCmhUiSqH8Mvbf61JYpLLtQW3yo0re5uix50tw0xQUOegRXTU\\\\\\\/Afy4mmJVB8JHP9YJvB1WbN1mG0EsYYBYmzuQx8XCbuLv40Ku7WiCb9o16kVpoc26TiPjwZe4gQ2FdNvNF85A+i\\\\\\\/1Hml1XdkN8eLL9Vn\\\\\\\/jTPSw2jFAu+t7FNZE\\\\\\\/auxrCFSzTPyNW1bjtpdmzd3ZvFrDf\\\\\\\/ewfWjGbv+n9u2FnFuA5sXabxvorFrAlgrP8+ZPfRuoAP0bCeKiWREOlLlXYMGcPBfn82t74bTAtV\\\\\\\/X7V4WJmhuoTVaUi60thEZtAr\\\\\\\/M+P2hBLBSdcBmfFlcBjragQSV8N7+LPMEktJUDwjObdD1zXff3KBs+zIVebGTD2DM1Cllk3I6c5JYQn3D84NHYAhQ9OcoelAo0+kBpJkGXVJFLY2z801R2Acrcxm4cklHDg9\\\\\\\/nrIVxKgmGIae1+liIPbZJ7T\\\\\\\/6nJuDeOdDV3yACk\\\\\\\/\\\\\\\/cGZGNfoUr4UnRhsAxDyMHj2B\\\\\\\/z3d5PHSTX\\\\\\\/AFH\\\\\\\/M1wxSQekulyaiUDC430SU+kJ6zMz6lXUCwcLqSojTK9iCLVvKikeWFuavLlLg3A5dDbLSldGY29TYbJZVGQKHLYwbfLNCeJhzWxusoK8\\\\\\\/4QF2TPCjYFy33\\\\\\\/3K1146vSH30MrD\\\\\\\/hZ+MmhcrWm\\\\\\\/wzX99fmDw5akbiuMXQmxPJpHUJDqHKA9bYUE1mOEkpEq1pNpWLEQB0TVpsHvDlLmTSu8mfNA\\\\\\\/qPE8X80sBeHbvoGF8GqXCD\\\\\\\/WA2AEpxxfQYh+S+RaQaAaQVI9X74c6QLESafBz2Cd3\\\\\\\/Qza07ESzdhpdHKutL5Q2bSUUbBTXnESoQItLvK7McLMwJLMtch6XFvgw5EAKE+TUlJJ163lkyX+8r819\\\\\\\/0lsNdrXGjzYmZaHnSfQWA1b+X+e8xh0+jUIW3uRYLR4Fo0BrJtra1GHcv+i3a4EnRIeL12R0aPszP\\\\\\\/7kazMqPtXBSCQRO\\\\\\\/djgODoKhtpAz\\\\\\\/8rIT5HmorG\\\\\\\/hDvYRsrkR\\\\\\\/UVroHRzjwmrq1Od\\\\\\\/lskiTjTuTg2QDme7KbXnHVaYFn1+ctvG1gm+CsP7Th3g7O5iyB6kd66ZuXl2m5k0AxZq1aXHnrfPupKGGvEWxpeOQRGWt4l4NY\\\\\\\/qgm5GxT9NC49lYAo9P0XyJMC9gfqQlKp6r4CJJaDWGBiL5UbsS8+e0hZRRs2mdUysv9goz3v1Mfx27NNuYjPdGwaB\\\\\\\/OickNN\\\\\\\/rk3bemyIK\\\\\\\/ziYiMrGRHkvrSO\\\\\\\/WyPIPUQw2kahhH1tg9fhJanQKWi+LPIMKAgg2ZTJepLuQZpr2xlKt+R0LKy2ZdkvWmBb8mSom66hegrhIA6Mi3LHp1PX5h2CNtRLcWrQFft6B28SCkGrp9R1n0YVuVy1u04+BcYueUx2Oo\\\\\\\/+aXdyf\\\\\\\/rPUV8tasf744ax07GIpnP3Vroo2s\\\\\\\/mnK\\\\\\\/LfxXGrZeaAXU0oCvDwFyNzEWirjlDesaP63khOo+CJ7iB\\\\\\\/1V7\\\\\\\/FhscxHLDWoEQHCYJsIrwAks2LNQEou9MPdnzQjkiEQK\\\\\\\/aBR8dcb2doWBY9ChFgKw7bIuUv2CRmzasrAcGY9QVMOB9B\\\\\\\/7SuGAkX6Q8qmbRxMUsWJO+dKEW+QNuxYCdxEnFvBBzCuM4G7Cq5nvYeD1hlR\\\\\\\/t89koxEM8eMX56H5l8BpNIpRuGy8kh2DYvDwDnm7w9eYtph5Tk4jklJM5Lb0nb5RrTmosPDwQiEcsPwFMTNAtl+iq9DxBP8vfwT4eSBglFRQO74rPyQG6B2kLzTxSG9uwCc\\\\\\\/RuQSoq1\\\\\\\/pQbonH7+Q+yoQrgDdzY3nHn8dNnCsXV2hdZeYXG2KsAt9mIGt3Mtl4ub8TYaREKicZz6TTRWYgU6XOKXB\\\\\\\/JgIVFXcmIapsGbi0T5yh9wKMPJbpUCqeiDtPoxcVUTLy\\\\\\\/1o0QAAmUf0b8YbtQQAm7N1X3osn19hN6VC\\\\\\\/7YE5E7i2UCVG7ZIpMKb7fIroWkygtgokNFEjIO6deZ\\\\\\\/vwtgQHiaajapsogymYTKpEqY5HW1IOEmhiKhAfeJsWv9bAr18hWsovGDRTuRALRBDOd1NO46ayPE8kqGX\\\\\\\/Wna67LkJwqEmvPKqNk23MpWi4ITXvu9uLrtvv+X0GAwGlz+KbHkJ5HGIqhBf9O6DYpmMMIw\\\\\\\/8lJ8dW\\\\\\\/esKibJ3cwkVi6I9zC7xUHJCDGEyhf+Ww26BNHcLnOZM0wKakKe2aBQMJel91X5ZbfH2I1y2crN0rWYdq+ESo96H8fA8+Tuu1M\\\\\\\/63\\\\\\\/BSKgPz7IeGoIXkz6nravCqVigifCtiwzKGfTr+kKhDYAhwJJpL\\\\\\\/1I\\\\\\\/vJ3\\\\\\\/DAogsG\\\\\\\/9SR6q54LqklDRsC1\\\\\\\/OfxOYdklqyj0KfLl+NLDLr7PtFWyXL6wRJEBRKG16vaOBxq8jliajaDOxUrrJxlo6JvAvgI2xJTDBkGJ+PcS\\\\\\\/7KW\\\\\\\/NqJ4cfAE6it0e9\\\\\\\/oyKLvKCBvvwCqRKJyQPeVHPdtz7zqGWav5daqsVal1xnnszegJAeKZJ4\\\\\\\/WUaDpepD75OcFHURouyTfl1j4IlkScdw+YcR\\\\\\\/YQU6ex4CGO568NEjebamAFwwy1MdNXDa05\\\\\\\/+dt7KxAiqTxgr6UCj6Sk1aeL8o1U4befM8AuvxX+vrf1TqTVISkGaSgBwBctnEcmg\\\\\\\/KRWIBLpI5GSOgylhOf7hlxT8Veo8QxreuVZGht9uKkx8mBVJ1wEc0P+OlTTzjrMoymGj+FDnONGGvMjyTu2gyUysLHADpA\\\\\\\/90S59jmkZs5tEtY8pcvEvPetTGSLKW2vSrnfCbYidxoFS1Oiap\\\\\\\/bN1EFfaPkYTrCWZKTZUTILjhO8Sk\\\\\\\/5C\\\\\\\/xOcsjBpbIe9EUG3vZvVfiWUHrXPQABMCwbgPAikYj1FWqP\\\\\\\/\\\\\\\/nlnpWz1qVIo2iNpd43wERO3W+NMvr+pnvPKRKYeh3vfifnZ8ow9tTdcYDqrQ0dO+BuRh2MzR9bcUIeOTeu\\\\\\\/DxQlOmEGJ5aWM3nIZApJVnbXxSfY7H6l6RzigSmGbY7PvPCESKYDMkCDIaTV33O0ENNk0V1cjqaeAzhxJ3Z5D\\\\\\\/rEJxJiMmfZnKjh4HeBBiFnCUBWTZwsIKWS8FLw8UZr42WPxXn4VmEiEQzytyNgKL\\\\\\\/1RSWY4Zskr+BRrAQNNdQCNps1H61ic11B3m6P+sdVyw7yJzYwn+iT59XeA\\\\\\\/EcvETVn+QpOMzw6gcJiHKaLlSxm9RhedzGlhdGIg6GspwDuIGOOcH+SRBLL6LQX5v1HkIa75RRv1LjZ\\\\\\\/6QOl+Oz4SbJHDnsGZiKusQv595PVI\\\\\\\/jdL6kR0kAE5tE+IU3y\\\\\\\/ZQHRq6xd+ERxMOrQRV\\\\\\\/n7AL0K4MS\\\\\\\/CElaxthx7Dzl1OHzDMKvTHHdIFZEqjoblL4oxw1qjQqxiB5BiocidR8TsuuoxZhu5CabZAXO0B2Z2InLAlhEu9dGwlPBqmTPRhqYS8\\\\\\\/oyPb8EU2GbbQ5PETRZOUciFaepaWZY4qz4nxNRNmObZ+He2WlKp84dmrIMlwMdBXLa7Z0Z4m25\\\\\\\/PBMPgKo1+ep3BvekRm4EwYP4G62ZoWXqSEFxtFGsJ+Sr6coqPuXMPtFWD9YmK5aU\\\\\\\/7WiySRdv\\\\\\\/nyhgpDThCF95ulEIHXnLQSyXn9WW2\\\\\\\/FuWE2+QJtPTFYgdWndiO4oiFTyWPUDkMU8OLe4CSas7uXTr1ofHIxiGZrdTGkUV6bTrTUdnqarcK+0j7FWyQmn57kIHo61PxyRfwzYRY6eUfxWXf4kBeWenFkF3dLT0mVdAEuAQpkMvPmiyaIScAsZ6OQkZt1gsTx2MMjal1NCulXXGl2LE5lmLT\\\\\\\/+ITmw58w0o82O3HI5\\\\\\\/BsjZb5\\\\\\\/OgxmbBdCM0WZae0B8GVtHTAWti8SECzY3q88IZxVljvv+SdeDgzv9LFNd\\\\\\\/\\\\\\\/UbM1ybHezGRLIrrzLbRYcV725U2g3kq\\\\\\\/GQeBMmMs0nIkSRBOLNMjRHvfzB\\\\\\\/ayZ2xEQ+6e6Ons854ZLTltQpeZla5McPt+E+xcdLrB3o1+pZlRtZAURWHS7Nxceo92HfCqHxVSSEvw4wNRSVhY\\\\\\\/MbOOTnWrjlIEH\\\\\\\/8U72qEc7b6MbBeLHEXFsasSD3IKlvrKsufkHQ1ZAUSj4szFU5Kab5L0EpCQiMF1w3fNWL9X4\\\\\\\/zJaC1DkB86jzxpBHUO+d0jj4xxslj6Rjwt\\\\\\\/CJmUvn2pMuMY7SnX56MmFBU09XPcTnHXw74XBeTb+AIhfRwK6B6LYLAAUCS0S7XtqZQ0\\\\\\\/T4R2gSAduYIoBCTWg3z2lPPdxAa2nLmm598OX0mIuskZSvLwq08l3JPwuNxtET9vG3j6z\\\\\\\/ug3ij2+YK+Oz\\\\\\\/0ubhTf+3pSDCr9eIJKlUi5B2MiS1K276ulY44MoDWShmFkFE5f4KQmydKlHPsY\\\\\\\/x7xXFPno8nHhaLS48sGYagbi7Frar7ugmF\\\\\\\/XkXw5gbWzZ+Ao4yHpKMrBxbbctHCbShz1KG\\\\\\\/xJyjBdCSd77L\\\\\\\/Gz2y\\\\\\\/o76eQRT292pQvpXMUFaGekMKa1EeBMJK1Daq8z0Xs\\\\\\\/93VG1orNRpZfZjthsYLE+yXFRLy2ZGTfcOjoo8JeAuCt865zMuxrD1AWCxiJaOMgTPrsdYKYesgjz+tKI69Q8FHIqNtcC42KVAYjHsmEnz05qXmFhLhTdLYVQNsQCIBWVW6pC\\\\\\\/Cfb1vhitNS8MvAfzNofjzrwojkPwHq9UEWhTEhiMKLPgDzqoKavs7XCF2\\\\\\\/fvKRg\\\\\\\/Xs31MUEquJfshyZlA2Vi\\\\\\\/BnZVoRD38CFi5jAzMzZDwTtQx0qZWFrl+RVkH5VY3aOlMqsxNmi3leNt1im35Rpdf+tZmvvGb21US\\\\\\\/bWUldKmnleiUtvnqfdC9gegAz1Z7nhZvSqPPvgAwuQMRlgbJorDUt5j3qnJ8WcRiU6yDIlL9rMsZ28LJbMEqO\\\\\\\/wuqndvNwAzJgLW2\\\\\\\/i3VLKzVg7bBgJ8fHDAEFH+yOmGhlrPsvP31PNMB2dXtuJ9DvKumm\\\\\\\/e7CjNsy8\\\\\\\/gLXr4mU57PXSQGMTco4rDeQp9UqwORWB0f+4npOaY\\\\\\\/NjVmP3GjW2H0k+6HifkKWwXCayby9Wy+IZJY9KD7f35\\\\\\\/6Njr\\\\\\\/DXeyh4318Oa7FM1ICbKSTeJsZaOFPnwp7ZQb2r1Nfz+0nf7PyApsAYYDUtf3CscywxChw333vq0KbjQkEPBipfepPYVl9ZH5+IdZzTVFAXucKGok9NzBwpcI102Szre7CshCukUCsyvc3Gk2W+AEncldano5dWACKYi52y8uiaL6GQEEBg1TQLemZOlgog2P7a5Rn8DKFpqlqM5waNwCCNRs9DJ3P5r0dlf70d0mcnXEHHEkjwuj0QEYes3j33eOxN6qPQejfq48nkpJ+xZKdDt1OwSvv4wQ2E4y6UBSko+2n943nst8LETsq3Sv7CkYE3zTpCs7LKCPQN\\\\\\\/nCRKRzpAHlBLWVck19LZBOKD7q3o2Z836Z\\\\\\\/8xHQZ88joiLOKT0Qhe2fSXN7U1LoMU9oGa+LxPDqBOe6xIMnsoZzHNC1QlktdY+OWJJBNSysnmtjrdW3vWsIUX05Fdrta56iURBssyPbAknIBjpVQaxaXQGKFlvzF+SwbkN3brSy0jJk56WRETXa4BCqIhpqwx4rECWVrYFC43oXFoyc\\\\\\\/nmnw5\\\\\\\/ChYHs851vmX1iX7IUTd91PVEjjQ9\\\\\\\/47jt8GAVsBzGvyXqfeofqpCykJaDexzTe0x2Zs9plc54fV3HPE36OTUS1tUeSr2poDKnwaj2whDStYAiuLlql2bUuLjPorXbrWTLfJ9FBY0j0uxfvMom5SXTO5PwOlgkN+iffySVJtWz8UaC0Tkrnyb5JZW6pjG8RHwGlPuvfJBeLEBrxYA6PJkmE1SR5aANIRw+WBTCsPKxBeBCCiBOk8hukL3wa2+c+vwdxCSgeuAM\\\\\\\/Muo9VZiawVhnDTGhZbZ\\\\\\\/6isDYmOC6EjFE8Ck1WCcCuDp2wIhP05I9tUs7vMl\\\\\\\/JyQ7rCJzS258ppCpIe5EvbSR+MAfHtukSYD\\\\\\\/g8doKqpT2eUDpWwGHiEQmED6ktl2J786RFfXl\\\\\\\/DylO4aOdc69x+uc50hgFrnI8okiHRnAkTgZQcfUOiJ7IQjnELn3mx2lfKdyCi39pR+WiehzRn+C8X5fWW1CX6S16uC\\\\\\\/6Hf5Bq28ocienB+f0vnS4zGYjGVcoWDvFUT\\\\\\\/20ziE3lxCRHAZ7Ar3C5CIn0PrLrbBTUmNjJo0+OGojhVZWrkB8kvpglKHUgdvD\\\\\\\/o\\\\\\\/r607Q8Zrmq1X5MKMxeBsl2NmEC4pqkXRrGo4d0ZBjpewqxl93dFTNr4yrdyJiro+hnYiTvs0bYq4ERBrv0jx54gi2J7WEMaBkEyaiK1AVQ2ItoSoCsg0yvKYsUwaXeCbo0C5I4JlpUjprX433y6Y3M7tn2anFhaAQuPJmye5h6\\\\\\\/OUWbjnZsJ0sR5BxVuf2AAxguvx3\\\\\\\/llOpwRvod4HKLQ33Yrkx+7PdBlOn4JKqQ\\\\\\\/XGjTk9pp7G\\\\\\\/K\\\\\\\/mI+8GWeGGPDdV0C+0fJHqmppz69iubMPNO52Begu9oEKRmORAfOD5vQRnp1Ua4YnvZcJPEGkeGJCO5V+bMdTtIZVibz2dffLKOwekJXHFzcQPEIgVKgMzf6ZgZWo8NHaqc7iDGwPu4hYdBFkF3GTbBz2QTUCPLzdPjkoMmORH2wOtjBN6iIYNQz5sLd09pwQ+Ib5SkV8kjQwnB9HIJiovNLtFQAnQgUc9A\\\\\\\/opHGo8BaYjT\\\\\\\/1PV+Jmw\\\\\\\/8usKU68OU6Riod0QfjBuB7G8wBI6972o\\\\\\\/PpHUdeYJ13CvLRdR3sxhylVa5Ix+\\\\\\\/460LNvLeRFJzF7IZ3qYxIJVBycTtSRw8kXfvULOW47tbYZo+I5DAUlUk2IfPCLvCE8x0ChxwFTy4YkN3HRzdy+pkLsuloaJZIFJ5QlFG5QQOkHM7HpevBBgMn5FsBGbBA4nvFi0Nt0ucIxS08E8\\\\\\\/MvN6UpG3aymUxpiqCP3vgfP5ujV3HDeZpDNj9uhUXPw3KSPUD7gD\\\\\\\/xfUjUFexl17WwRjnxFtToIjk9R1cMBE6WiH+w6nuwLo8\\\\\\\/RP3tG+UwcM6Vb7\\\\\\\/tmTgQV\\\\\\\/pRLDa5u0yPSK9YHZT6MS1mjWQzPKHUG5XcLLBBIDJ5w9uqq136Q9DN9eKGdSMhc1KMWgJBhMskGOeGugdjci1wsAc0huWI40FsskCedvdlcsXC2ZJwy5bqkWmbJBbpnAjRX6eOQS9C\\\\\\\/RbH42CZ7VSnAuMlwUZByXeqZlBcjG\\\\\\\/8UH4mHIGQWqR\\\\\\\/3FTNE3gp3KUP\\\\\\\/ZhggDeBxaSqZ4\\\\\\\/NNhKMymYvjKlb8EYvORbdBE2sHps2ubyL8Vp3OixJIVya4wt+5E73Lh+HSrNhrrPdg0EiAVBrPba0aVbhxFjm4kjIPBo9WR\\\\\\\/44GpGuHIhplPnz6Fnq\\\\\\\/GHkNUddyQdNmAlEU7LiM+hjzJRKYEELDHjYn\\\\\\\/6NOHxzkQ0qvdUg6g0v34zqc1HIr9jjy84IT\\\\\\\/U0iz\\\\\\\/kYrnLghGUMk8m7OU2STOZlk5aBc0nGvswTxyfce2C8zfvqZgpAINlqy3juYrm5\\\\\\\/KxSmOzr\\\\\\\/O2wTzs03oK7ypZn3tQnikMAdfw8hyIMj3MmS3PqzC4q0nHz4qmYabKs+73JSMCX9YsSCpwGvLAt4hj5Zuy7EwsM\\\\\\\/OUeko8p\\\\\\\/+6bnV5XRDRMHDcfCbtsIWUa7FHRg9B7BzYZyDBFhs\\\\\\\/LdC7MZIww02Ad7Oc1TfBxVEaewTeun9wJcP9IF7NXg9FV7\\\\\\\/ZpQzKi\\\\\\\/0UrA56KLXZU+52\\\\\\\/mGu7\\\\\\\/vVryG+5vRBpLAB9GYp91zbDfjpSUEp1HiFyIDl4todrlZn+NujXQ9OnpImTbtPPFu+it4CCS5niQk4KQAYBkAVvxTGuacgI\\\\\\\/OyNju5\\\\\\\/U7zabxr1cVhDS9XlGQp7VdraLAvO8nVUFJwclEkt2amxaxcH7Fw07byREPcsmjqFeHMH6j6b5WVekGUdxCesx4VMe9YIJNU2XZh+pyPYBFOEYoOmopICdI68pRmhtVJ2LID3iYtmvFV5t1ResRXWRIXcgfZZ4ZA5p6DxpkbQHaT+nDlOaX5UKNE8j1DJzu0HFw3\\\\\\\/0l3rgA0hcCHj4T5bItdeAPibgsByBJDAehNP15vh3h0FVWNWgt+bwQA8W+I4TGKQ\\\\\\\/oU52a6OEEqtvJtcCOLdJF56TH5eo32fEo5Zc5ZL\\\\\\\/Mr1R9dBROJN5r+7\\\\\\\/4PDjvOP45t2ZyXxyLC1XBKIWnkraEy8brxo4VmbvvUFFRG479rtZCg+D6JQdGkkVSLD84Z+wCe8b3WuQ1wZnmcfTXwS54t2fPVw0J9Un9X27jeOz2t5kph57mChdTVNw\\\\\\\/y88w2\\\\\\\/sCERMqjkhAdOJYAdwAKeNfWlQ7sJhSKtZzE1+4K2r8wZe8KSIOU+yZ0UNCoR6gy3xjeyH18k3iBZ0IC\\\\\\\/ru3Ve+Gz1fZmVL4UcEwmoSqbl\\\\\\\/dO4dE5pUDkKn0Cp7FkdKT5hIa7Qt+zOFQdC7IKwYT7T3JIrLcx2RKOkfg3UwBSxQ\\\\\\\/U6a270GCkdjiluimKLwL7y\\\\\\\/6Tpf6t5VB6D6g5C\\\\\\\/sq6QEMjiYOa782Fv1DkM+tlwmSwYzEae1APH1ECbAPyMaDEkclrdrthyOFz2HFRx4Qi1gM0CRfHJxMp2cXAWGQX+69GN78qhyJyVt2BwvfLyCZH94EzK1SDcFvYyfRj99fgVcCQpptyu1Oq6H1DLkf1EjHUG+M2SJqejOF9NKwC25RSsRTmoiDsNpJgcGY5bakAJ2U66fZqQDMosEsb4ZM3Ndci5XYwzPCLbJSwTPDZsdcJki9\\\\\\\/BcRppaYqR7jqQYRMBp80c6ZJD3u5ihlVHn8TqZ\\\\\\\/ou4soODj6QDe6wqK25JXxr4LLxN1TSHxigitr3Bp+Jplm0PZ2t564dQNatbhfqJtp6wa0WZ0GLOC93\\\\\\\/y+DZJhLzU2MdmgXX15ji04zsxAfJSvKb\\\\\\\/6b\\\\\\\/8\\\\\\\/50dSYGWMmC+Gmv713KqSpGAWzvKbCoRrJUp3ORTTomaa3OW+af57apO3+\\\\\\\/tOPpRpww8kjP4oeQKPh5MGYOoFSGw8+AfTImG3Niq9nT7OCP1YA4P\\\\\\\/\\\\\\\/xPSDgbqiZHU2de+hvnolrIDuVtVqlDoupIjkP1SlNOqpm9JMse4ijXHOkh7eQuQIpQk604wj6zYErXs+lg8E8OObxPQsuLdMqaPWUGGUW0J5q4c8iIfDd4mcFkrmD+FXqjER9jfI+bl+TqzgTmeIsmR5mgFCHiuIox3UodhvdASZSODLMsxH059oUl4p8raa8Zq3vXIvWbya4AcNL4p2xrhnMKiE6POj4s3MbphI0ckDoYD4D22\\\\\\\/w4bR9m3GmID3b0gpaggbA5wJjYn+Z+J8CUCi2JbRD8\\\\\\\/DBQHbiplYXGghyjMSTLeAbCqTHEoIaIGCqnlfiGz1k9NLjk4QPUw1Iu\\\\\\\/LWN215KerdoHD31gt38AqVxHc\\\\\\\/k3UDjH2mZ6u5btAZ4qThzQFFb3IPnbgmKDYgGfBLzBmgRYPqI5SorbUxIIASAf507Wt01Cmp6wzXVyO\\\\\\\/UnSU+3qVhFYU16otXtoQqmTj74MAhrsdAckk54PoAata8yF+I9af6KY6jDSsvVAKJ1wbgbgd3b4N36t4i7rp98N8EeRHm4t\\\\\\\/CMifMClM9OrS59vRcYOwmT4UYREkrCVLn3gJYv8E9BTzMkhSnVX1qwNFhHnOvpg5swtIoZ8p5PZ\\\\\\\/L7lfCIDjrExMnhpT4my93RcPddF2seC0wGyyKP\\\\\\\/pPD5+hbMGcBMoQ6AoJLK4SqCXsoyOd5tcc2znctEmS+xfro8SA8zfqSRE8TWE1up7Vzh+cbPoieml41Iq61H00GXmx10jKUCE6iwODIMIhiHMLCrSbu+5Q66F483KBfQCL1GotQmxqCuOtwmDqBpzQHlRIjlLlQMkZDU6YA8hU4KQW6agtsz4gv+T16nBRl\\\\\\\/h4mkIBt24G4qm8\\\\\\\/Y6RYFz\\\\\\\/QN3gTenOlgOJHDr8LnsBcEwXDAsTrNiwLW+NqYo8lyUBa1X7li6CxnOQroYcKtwuoa3nOS0J9ER0cBE1mjG3V0OveLwU9OmTvuNUw63R0iebE3CDlLKmwiWURHi3HTM6JhPLWmsRliGW4W2T3ZxBIbwuUagdkjKt\\\\\\\/VH02sAu9U2bUuBHu\\\\\\\/9ItyDFQy+npL2crJIHgJxTnwN4pz2ji0k2Z2OG23PKxQ6Olk\\\\\\\/0YygeLxoRdD2gjWFoGDXAalB6zaR1F4M0+1KKBceCJUbtJihN896w4xd3cRq2s6O2F792Wpnwcte6L2VRIDq4VvypvO5vjXJnKAi4ZIPHW3Q0vjstg0G9PIHJu28B9P0V+tiuOV4cIU1BlSh9zBTP0r0LYs6ibgSQtWVOqTP2JqbL7pAM7eloUb5v2g13DB5ARSO1kdv7zYtZ4pRzp8I7e2mI0snFN45J1oaUCXUkZZZEltHFCLrvUB5R2uOroalhArJXTLNFlPjrXZy0nKzP0RKLiErnX6epYXjy+cnCMo4eK\\\\\\\/KuJjcBisBAH0is6o99mno4LuWuzzbJ1lJhUAEvjdffIk7Uc+QXsCygQaoLdIoA\\\\\\\/b0M8WgRWOnNGFyOOZWM0M1dbRD4KwGTIGRCvA\\\\\\\/75aMqRTu6AyXKV9Gz7ooIjfjXpSB6auKwGheJ\\\\\\\/rv+KvX1kj83gzBdF1nh8KYna6GhT2piyMxsRDZ62cNgjensHlaW7lk0zKm2sunmBSoELoxQpUf59TLd1aTKX+iqLzKfymgAYm7pVDccO5eVAHJlrdkqDdVKIBw1JKmzC9iBLhN26f3\\\\\\\/\\\\\\\/0NeabowsWWm2\\\\\\\/w8e1DUWwbq1DESAjlviNVdOLBbqRGa1BFDEwF1bmAl4q2qdBtJjFWy2zlF4qK4ZtrK6YuvM2U4xZi6MOvw1CZ0Zbz+4h2aLWSENkEQ0PduA+K4fRC0B8vuhCw6RSCbxnEj9X8gGLptFjHgugEe1wNZhF8n2M3vFRmKvz6FmU9PKHj2irDIcuzkQ72Opny5AQ4B6FRANcoKONVAN1QNlHJxGlE1hcGG5+jbHxQw8nZF9mv0OiR4Mns4+NSHRiAwSrZvGU8S4qxy2uSPrBO6QybJDT39CjEHqXiMwX18ImCi6G+ED4sjfXgzttrxb7iagFxjI+cw4PP7fEaJVys01+MbYZU16k7EeD6JrWggJSQhS3bfckRngOwW+inVcOXNWYZNTSDwyKcM88BOgA8T+ThITLRNoT0U3kFAfkzhzwlaKYoiIO\\\\\\\/AwLisa5lXFFX8V048HaYefFJo8Y6\\\\\\\/rQM+rbBDrE52LpOMtPeqxpG9qserSeX4P3d9nEX8+2cUkEJ8BiF5ay5+CPwtryz3RRO8LcC9z3Rayvx9\\\\\\\/3VqS9dSXvlvBqwgHk6nzOUJ0F2f+Mn1mue4dXbvf0B0PxCWIT+uGmsLVuX+\\\\\\\/X0eWVbK1KWU1g2uyaD0Eq7dDKCoSw+Q3G7kju+HQUAGm+rVp00reTedgz9RI66tTowmHvX1CV1aWJ6uBsOVo+v+GCVvAbLyZOJeN8Q\\\\\\\/0D9kSsUqBwOjnveXwkfwe+qMeTrNvx\\\\\\\/SyoqrzQWVytBeslAXadA1duN0BWrvxSqY15qnhSKbQlJLSygRPJkmeGkW9ayoD8R75I9KqkcWLEpSagxkBsGe6cnzmS+Ixp97HMKdM1BJO5q6RkQB5jHeNamX5My9nYxGMpf3rYLtspfozkWNJc8Art77gh2qV4IutyMzRhdhP8zJSgsZByUVA9Z0aVqSoL+uFkwAHMHhPjvC3HOVuD5kmuF\\\\\\\/cbr2Z0cMYu6vj9mA5bestXq38b1N2pchXYwU2Mcqeko63eZR8srlAPZsyWaFRU31oSFPuXeaH8FMMA6zzfvN1S3RC7qbbrul8OTCGG\\\\\\\/dNry6snrZs2YdAQgvEAOhk\\\\\\\/vZ8qij1tFec9Bhm7h72M8sn0GXJ9Yjot8Rm4\\\\\\\/mKoktZgj4XdVsQa5QmMFYfLKZKxSPhTBmuadumRrhLvssAd+IqbvlWf4HYKu2WupORNf\\\\\\\/B4+zZ3K5a8PYy2oihwmPhxnI1FWXudp1+602i0pxKZuKa\\\\\\\/rhsHUNgsI11ggO\\\\\\\/zcthADT+le4wWEQsl8cCEKeub935LCNGxs9uhcOmMqYhJS\\\\\\\/0JwU83tXmDImd7kVq02hvmmnwxITW6AcmxWMao2YJQ5eWhH1amiZwqYS+eqQ2IDjLl\\\\\\\/0m6RUsK7ceN69PlIUCPN0+1kkB35VT\\\\\\\/4lttIXNu3woueQKgVlxem3qGI8xAfIKMn0\\\\\\\/Qfvi0JqORj1Oh66M+ZNq8chh+Z\\\\\\\/iYamYkajUzw3hyYErUIlM9dibC\\\\\\\/H\\\\\\\/voYR4w\\\\\\\/P4I\\\\\\\/8wICmyPzNJ6Rp7eUD\\\\\\\/mLtfexqvRS8E2uPmlu\\\\\\\/SyzCk3Y8qxi74TndBQ0qzRA6F5Uxpqecpxa3RQx1FOI1a4ADlpY24V\\\\\\\/keR99onLF3vwpCtt8IxJpzd1H8R22qHbXl08A0bb3nTxQ\\\\\\\/kB7bCe941Z66iS9pFne+6\\\\\\\/6WrOKdU3CIy8KbRbEzFFaBuY0DsqZGPiMKytv1dWEBQDj3AQwbOIofStNq92hMYPU7jAzC77lERTWtja+Hty3WbkdzR2JR5tGlrXoIQH\\\\\\\/+\\\\\\\/sfqigQtscTt3vvqcPRophACUDID5GzrPqPBXxH6IsYKnpq1JZ6yr9xWE9hPwb48pM5Ug6VbS13Cd5zqf6auWe5wQ1et\\\\\\\/Wm78JdkGNha9hPk2fCi7w3vzaKdkAHYQhytv7F15Cu1OCLldFx7TIeWss9b8nT5pIrxAloqoqgcbkoyGOLS7YXx2IxWfuQP5eFPSYysFcV5ex7OS9aZh9H42oKOXUpCx+vSwyhwrym4aJHsYYn3M1opVn6KNHVCvbL7\\\\\\\/B+zEHHwnd6wtS7xtKjH1RV+ax3vkbWJZcddKY4fWNMQK+AHqRR38HY4Uc7+gQeEiQi6jubliVsisujR\\\\\\\/aNmEr9HIQXoZ16EVf9qOgcWoVIppEGtg6CAZwJDKH56AgW94WcQnOpOYNgct2SEzy6HpPO+jaY0Ba8XkHG\\\\\\\/0ZZ\\\\\\\/36QiYppYiv8TmfXNyefLJ4LZn9op6rseSELSTZmecMUrp505MsGqHqWhzn1a70BBhSQzn645aOFLFRhvS+jiDZ3oTZKt+ov+SVxriQMICq2yKEj9V1O1qfWcruw72Uv7lOL8LJwdLb90o6WlVI+hfIlsD6VagvXHLrVwZMPFEVgorx3Da5aUjbVc+8ED2phYx1LEb+BUxwONLevh5SaaANVEWrin\\\\\\\/zACel18g1qwYcnzl3IWuRPBzaI\\\\\\\/Vooeh3LZQtnnOmLYXLj0ka9CPP5Kb7QNM8vB3PulHUy3RWkZOIowlTtKXwRnmH+sVNlno25YvdWMhl2rykxwjuq3IhxRSC73ZnZ1TifTdvfX9GIHNy29nGDus6hosAlOUo\\\\\\\/6CFj3icozL6X+WjYL1iJZhWbhdOsjwG0MIldUsYsmJBmzByuGW+y9VNRHZhT1Yo1D7OZxJAG8H62rEiPw7xObK9BWL0bpJYzFf61spv4Vv6pTC8\\\\\\\/b06xtoY86HUwaUFHPi7RUVWppkkd5rCGLDh2+uw288mIomvZb1KnyyyB53mGVUUgcpDGpRFGm10jje7nC7HRpQOtRGz8LMqtXm4GHqBBbGqSDrQGmzx+BTSUoedUb\\\\\\\/tWpSc\\\\\\\/hspnnq\\\\\\\/UhW9XHx5U7E68ZZw+jWvXHyiT52RG+q\\\\\\\/UJOdRl9WtHS9RlKZhIBUEWOZYjcRpAz\\\\\\\/+au6a4xRJPpcZM4qrBLDu0U\\\\\\\/k60mEzliL2WP3JbovVwH8oO0U4QEPJfRY7VFdG0Rs2N+nP+QhBjyGtSVsdUb4WJ4\\\\\\\/x7HLjvjE54+v95Ssnt0DpBusegK\\\\\\\/CDOe9EYHfgjg6X7LW5M70Ro8kI+pEoRlOYm1gD3EVHa7v3fpkUj12ldfd2OjvkZr\\\\\\\/k1hjrmHMBOthk\\\\\\\/S3TRtjj2V\\\\\\\/CT37JDS34urMielFRsMsT5ItTE0KSPcnUDv0WDmJgsvMc9AcT4Q+XTpKkWvFGKxSMfsGM7OL+8+hNnjOcrBlk6\\\\\\\/K0qw69lMlDoO5d5RuQcC9iBD4NbwMqd+deE8DLMyqXxuyVBdzpp66aeL57DXz3+L6uB3ExZtWoL3Eq6ZfqxycFYTBJXsm7QXrSp00\\\\\\\/S8eudCogTmtgdlv8+Uc3H7D82RrGdDlufSQVV2vHqp0hdhfl7iFqqznEvzjjHQZDHJIuB0Wz5YD3keVaL7pt6\\\\\\\/zIgyHU8LjSF5PrrlhVjLjoqRik10XKqxlDEa2R8ZRh\\\\\\\/wTp15SFZ+noyPmW0M1\\\\\\\/N5CVmVlfM4YRB64u9BmAZNWIh+olAdIlAd9qmfG9FYMDSpB7hCiwCLA05bhIewpQ5pziwXuMQtznNPTHG\\\\\\\/yQ2UdNe3Y0jdj289uEeP4CqeQ9lqESZRxq+uMh4\\\\\\\/t58CGdeb9Lq94GiF5Yst+06iphtGqe8p8E3k9S1pnkb8W8O07Y201z2W9InkrSkRIkeLl+JZox2vFJhc1epc2SfV55s7cyKlYW1Orbsm6DBj1Gojo805mTQvDTUxJPObA54CRv0CMbdyCrgCSS4W\\\\\\\/XoXThFG0QKnNe+vg1lCQtDo6FTanfrt1seCFarz08XJk5lx+V8jllMG9+I9faex3CUKcXGOtd8Gr3hJru5Jma67pEpXTKVKTlbm0kY7CUmYqA45wAECCDUwlUoTfIJdYi6J9h13CnjJUKfBK1ktzgqTv66FZmPsjW5YEp1yDXXwW9sMgN6Vqa3uBrdx6+THmyoIpiS1IYe7JOcK9GYYTUuHT4Sl1fz6mp73PVV88Yx18WUQwQfKJ\\\\\\\/3sandpUKzd2NYRAd7Jv\\\\\\\/liKH2Ga4tDUiRepezCHqnHwp+SNVX0NvfNxoPXC5sg8u7TGWbvt5HvY8kIQukMyO1toI+WueIxKeub0dfb6osB7M8qYcd48sSy9uL3swYV2xBmmGJKYqB0bORAuc+Sno9rXmCDjt\\\\\\\/LHOKMRQ7WCyCrHnG6+eItcufFC6DkucRil3QemDDvIn1u3idhNhOp5QxJXr4jIsZCcFjEvfKbXHXonmfLHCG6\\\\\\\/Jqn5Ben\\\\\\\/8IS6CRjI1B3e8XSrdTSl4nJmmUn2ZwrCm95D2DaaqgpR2orV86c1NR1X8UWWXhM7rw+MgKQdgm6ed7nPonnZvllF9Y3FOACjMRKo45JoRkFfpG+uNPaL6+OAttsXXAZKIXjhGuFgd2JziTTL1mh5DsK\\\\\\\/dISPDU4JhHBFh9fdrWMQm3h7UpUsib4phV2v9B3wzGS4shXmb8b\\\\\\\/8F4A1RhgZ7WG4FLCVcZGSfcicpb0e2MuBaNx+Zgtx97YnaV0fNCzvesMGtYn0XlDboMO0nwr1izsQLC2X\\\\\\\/\\\\\\\/BLJon4gJC2ZwLPhqZjrt1kKZbPzoFQllfIfm1fYv1cMxobozRNldnSa5yaIocs40J8\\\\\\\/haKULq6ntYfHFJ1o8xqjeWAcQTLaDP3lqUBKqPpbOP84gNnjxFwmuzBuJG49bqmUYhaUmjJNdXINsEfiI17P1132h3xdRsBdZ\\\\\\\/pPFf3o23DaNFbWpMqPb\\\\\\\/z2TebUUvUsLnJ09UODVBVuCBPORqtnL3rKsjuP48bAG\\\\\\\/n+y0GgtvG8sT4Z1RuGooLh+aFyIbDEms\\\\\\\/Fq+2EV8WGY5wp8oB3kPOcamr75jtKZrkXkYuYrG4p\\\\\\\/3X0TlRn44XgwlAz+CfscRfhUP4CNqgfKnzE4KQXfBDpKZICwBKBUbrWQq3f+5fMfuabEuskYu5WHBt12PphCX324\\\\\\\/2GY\\\\\\\/mr8sG0w38YgUMfuWMp82RVnYbP2zrElob9AxdMy7jNg4pTOfW4L3uAvzjp8w2PS1Euq\\\\\\\/knWRHFg2v9EihqGIi4JU5kc7QOeL1pzCNGygno6piO+Ht\\\\\\\/S6Aqz5nf55uFQ6WompP3jfA2HuIVYzgd57YLUis3l3rcrSSKQXfUvaLinTg6CTIZJIxz1cNuk6eUPeN6V8gqMxFD0zJqn70msUlvaq0E+P7Bj9FZikxxIVI\\\\\\\/WBCFaVggbUMJRnXtjvSNMLkeGbJz9HDwXrMaxYBeKnMQerW6nmC6tw9mUnPA3h6DPfin4t4ca9Z11jLUTKqTDSZCrBJkunjBkCe6cdadX+z8iGWcCq\\\\\\\/YNfO4y4HBOwUKktptC82qsqKxJPqobNXdgzMVdhgvexJiqyni9mFxK6p7jt50xAy08EYkeakakIBiHALtfALc3Nt1Dy4cexI+ZCK9PFdVyd4GYKzkHZVOIiyP776UhigdslKGZJVOHdlwAQrdgto2ypELV7Ey7x\\\\\\\/DpkVB\\\\\\\/5vb3S+uCmbkwb0X7N74SEi1Eati5byG7cO0A2FcissqE0HkEcAsb3+xbdnXLtwGPs7ny5K73Xiy6h0rZ6VZvC31W0\\\\\\\/ZOJTxGtIJPfe2Ygv79ljIiBgcARnGN2tHfQXxPumYHoXLWYrU8\\\\\\\/gHGvJe+AiDozG9CSXp7iXmZl3Ec8QsMgQsPIFeps9aTrwOgGs6rSTFiWGfyoohDA6VdEcl\\\\\\\/FyDqkgOlt\\\\\\\/BpSfyrZDzkRIe8KHcswA6INm224qjqCgCCLKqOwjVc9ad9+XFsCxaiOAlLDnXGNCx8Z+XyEIcYl\\\\\\\/2cZ1kr9Oi4dQ7iNazPVCfmIugng+o3w3wWmA7gx7gMHnT8mAl847NjHRSzevTl8TQpFMFNdOXe28hAH6niMC5SjBGjz8hDc5Uv3tfVtz+jJroBdLB4aSHW\\\\\\\/npMMxHdtM6frksvAdNEWlaZPtE1xnAwcnr9T9zh4kMekkoQMDHnT1JzflaIcpOKfMmvqM2xq8kxFf2OqjnecRmUb1A2bNFm9BBJQTaJkW+5ml1i3sDBZYKaDdZd6UK\\\\\\\/lTXe5sojVuc5o6NP2r9xtCjWDfXgYDdpsxxnFFs6aoJNOo9ODXHEkMzmFK7ItDBVjO8fBWkyaTT66g0rnRaB1QwMAchJELnR+4uxI2XnTpf7S5TdpekDpdOMCjOP5uKEXQa9oloJKFXIPOTNTtNiSUMOv6l0nuTt9h9PXl8QSpddScD6g1OzXmBvt2Xl8aYyFCZgUG5snECFoLATlOTKBDtqxjA1FfiQPJcpE\\\\\\\/wlFj6pcCpbzfW+q3qbkXLIPdlvY1zC9xjG5IvSZtMCWIE2\\\\\\\/GtzNN2K7JdjtsAOk+iS4gPIRIhp44cpMFCr27qvK\\\\\\\/RM1RUVRoEfljA\\\\\\\/lH8AYqwy4HTPB1IevL0j8mCZHre1YOmTmjcbKSQwX2ZVyS53bQTeBQqFtS4bkyHkNT0iZwbDazQk1wXSKwwQOvcZaRXY5Vp\\\\\\\/lUHjgUrCIypa6GnOC9GIkdjSP4raBxoUnqOTllGQK66YoAwmOam\\\\\\\/t\\\\\\\/ySCMXLr+JAVDBuzSwCYY4LYkAzf21IKU6IupZvIl7TqplJckotwFU3+AtbtDMTdqXWAKw3xGhLMfqErjNjAmM6RApFmL0CnfWSz5hLrj5UWK+r2og97QMUWnSlMOZo56xOQNbCNqmtqpgnB1bLMTOy36cIwCsnKiWIKvdEQtY5ePcjPSpeUSIeoHky3egUSgswp4H+vVx7Uxsvw\\\\\\\/bwfje2j+XV2bSkXz+HLFAAJyZ29BnKV1Pf4cZPYyKq6GHUWURn3jHoZvk319f583SBUKxlKOOKVMpoRslUS3+A1hnixH2EMHoFDXclukjApXahIeh9YSmNJRzEy+1OcxuVcB0YLrnodlY4gZonVP+U0EW1mK\\\\\\\/gPy\\\\\\\/ewQstMMaD6eFiEzXoM+14NbPeipnrc518Bfs3XFxRerOprM\\\\\\\/kgKIL6DuuyIID+\\\\\\\/2KZG1i7xPE1\\\\\\\/tx5zx\\\\\\\/TgZROC21S4dF2inpLl+DRLUqubV+4Ruwco9ckFPF16Zzb9+zL0+e7SoLPes5jzW5xD7IYZRBxkV1m8Ax0xN6\\\\\\\/X062+TE11y\\\\\\\/9huXhV7IZP+qA\\\\\\\/9dkICpiA3FJnGQbqGJUbS9FV7cOB5dkg8N\\\\\\\/\\\\\\\/KACjLAeGvhDDQKPFZSdXBYq9UAfbW8PNTmLPZWWg1SPYy03E9Svm69nF\\\\\\\/Ni9bZAMe2ER4ix0ukQUPXYjQ3H8StBPqXTthIE6qvLzolmMi76T1i3qYMHJ+0cZsbH+PLMcPwYhjsI37k\\\\\\\/2x3Jny5Ab9RESRVu+ZLk9W2tcA8zuY6kRfM9OTLmScYh26RyQlSJ2etiwxAWS7wpedlQ6UzSj6EkJD4QMpmYaZEOzYekuL1DRD7YH8zJxBSf9izMSOBzGWwJzjIFVNe3ybQV4pHVdacSLZNm70E22\\\\\\\/aEshrJTVtfuaXJnuB65+951ZSu9DXwCiIuo806VJwo3HDUKD1QIMzQr3iB8T4dkE9ogYqfQNwSSzQY1uHNqB48exih6VwN0oUI4Ws41bPkIl07lZ0WKEYGfx6i2oTd7YgjIjVMy3b6L+RP3Nvcm78kq9HAUQhD\\\\\\\/++DBlbFhr1uyzUmUsC0o7ikJoGihoopC6FSMf6K3sf5+VkSM8LCY4phMTqfgY+7jYQ+88xm\\\\\\\/Jt0NklSPxnRroPCJ\\\\\\\/jqiKMYzn41RTYwmm8SErNXuE2Vg5PGg5iGPW2aLPVKMHi65zoW9r32azcnaYxxl1hI+q8UVqH0iV8HL0Pkg6ZGJnYCiosnqe+JrcB7kApchPyhypqJ4UmhfEaGMATLCpI\\\\\\\/S2ZCaJa1lY\\\\\\\/JfZkPOlsnwFV8VOIP3cSYH5boFoP8xTtx8RUa6gRknIfV95Wc6JfIPYfLzrTSs9EErbIr6J55oq3btKB1X3TylRCSddaHrSNlXR+lQE5QK0zvLqVlE\\\\\\\/n1dcyZzDMWbMAH6p8bl\\\\\\\/2VkgOVxYbnsG2OyyRH8oELZGpEDU7LqAuFcHIcI1RCcXpn+dbnvWMYnMW\\\\\\\/zssJW9YV+TONNEQNUq5qrdZ1t0JbpsIrFQibaL5roN4LSMVxqqtpuShqgB1VSWw6BrRRRntIIun0mxEWrF23gamjFCFHcE5TIcJjGOskdikXoDRVQtW7koafxu8Y8KAPM4bQF8qnOhniDA69DkMNv9lhTX9mHFCC9jiwTv4tGrDkvaon3zcoGoQFpEgLM9gVtKAKsBQML83XBtXEIZ1L3wh+ir3mxlb1XyBV3tWjY4xEbgTiPJpMnEkgksBcCWNAiQ4QFRWssoKu+KDC3BcreMOWXLCu+xEv7HSjLkcTvlVOkSfrcKZRBHXAqKuJVRnPNUyir\\\\\\\/zrqA7J2W\\\\\\\/j092Qxjnj4\\\\\\\/GujpeAVanCrmZBRTPwtKEiITlrc8pDx\\\\\\\/Ed7Wyps9slUmd02g11xxEWmzC\\\\\\\/iUVlYgMfyqqkaaxTyFadt4koFcwiuImHQiyAUPsRiM1itTqPFvJIui8YCZuHNItSiyDT4JeRjLU4KUOZX+KyPdJ7O8bHv0sYcjPAWXwdkFi0DkDcqfo5MZ171DH3dYB1ll8pLAIbv924SHISLQirBvYyzgMBlrDwkMVyJ3cgcab8RmJkQL6jtEJMkZYA4NakJCLYYDwGX3kp+c0XG6S75qrNCK\\\\\\\/oMzZTYq\\\\\\\/90pgjxN6zQS+FIJ0J3MNTD1SempJijZbjFoAD36\\\\\\\/6+c0Vd\\\\\\\/hUtbI1jckw9U4cQ23v1uHpREtLVchNZXzH6mJG4f4YjXr\\\\\\\/sXGZhtUmG9z78b9cw+D9NuO2HzPhxOsMV+Vu4s32srRoAHO6BQeE41s5VyR9LOfUnEGRwQTP1XwPlr\\\\\\\/n89AQFiJvliU7fENt\\\\\\\/HaS39aDkOATE97al2FPhRP0i16YSbKCRWeZfBwkxBt78rb3li9Hyd8dz2Wz8vim8hkth8FTz9E7LPetWD9mYp+J+bagr+KcYYbkX0tpl3WH9Vx\\\\\\\/NkeKHAWTtJCxl220e\\\\\\\/Fs3Ns7kBqUOuGzzRTKLthZ62ukPqV1eAM8fu708TW8hbQU+rrMBzW3k6nIFHbSd5TzvDNR7Qd9RGoxGLJMAwJ1d+1OUMEunQKmLKm3AXItgvXOBs5VBJzlgPC6hBK5e0sTEgL8EOkNPEtaZJ1zCZUwqWkwJQouM63uoelVKKk3KXWlJdZpc2iGZl22uArXUZaQyyBLLAhgNu7sLH0V6kS84G\\\\\\\/xfD2WITWyP+No86wbjenTw6LCO5o5hSoVxTyzlEn1r+cg2zy3T3V1XE5BkSZQ2QOUy4gKVqFlBJb7A5VD0vB+ZbGysZI+jvg5VWdRe0zMPB6yHo\\\\\\\/+0JzNBaZTCZzNYg9g+lWsTojftJMXJ2FTDxp7UnO0rYm1myIW9+bg1EaxLTfHk3zy5k9s7UctxZqnNoZ71dchL9NoUGaMtAJuhMmrtdC3kJCMioXAFMA5lvvuMEMXYSCbSfoI4DjCp6\\\\\\\/VkBdxU7LnTj14\\\\\\\/mP41RlK+WG5rdqvp89wJy0YQ72ONaOnbjqGt5iORIHmz0+PFm8fs\\\\\\\/OLxppBl6GPNXthlrAp\\\\\\\/RKeHSS5FJ88Oh+eCScJuLtUwv8YADnvb5XHf1+QZA8lBKAo6hz4BGMLvTmBi4wvkAKPV0c25oSqWwdxGcU9AX8dq6B40ejPNYNIpM7sGVNa\\\\\\\/vaW9UbAdGsCTrxlR0XM0uRrEijPwKL222kCWRHshDFs5GK4m1HG20KxxJavvvRSGQI\\\\\\\/QcHxfFlUMnaWvJZqIV16sLy1iVaq8pf2r+gMqZBafTKQtv6argNvmahN67SODqFXNTbT9fZzL3ngvgGBWVJPu7w9kMlhb0Fh4y51YuHF7ZtJPXu7Uen5w62RDuWceobw\\\\\\\/CZ\\\\\\\/zAYuS1Wv1Z2MNcMk8S2BRhoSjhgmm\\\\\\\/8GQlmESP1Bb3JJDHFPjU0duEd+oQlz7XqNoyEOIGZ9gdOWhqpq+eDI6HUzfjnqAy0F26zhKypRM5B4WTMHu446TfAJwm7Jh\\\\\\\/2yiIP0Owgp79agGNobCRF\\\\\\\/h2JPEp6B6hsEM\\\\\\\/OXzngomuPixbyCMbyLqfBsd9+VxquzgR0P9lLgLaIk5OFoU9LI5eXdI3JLhkpo7pU\\\\\\\/NyQJQiSgMf6OvPvF5shpgAm9W\\\\\\\/u9trKaO5KtrmaR4EY+Iba0bRBAA5GjbPRhbL7hoiaxjHPEmRZzntRy6yElixyh5mtwfArKP0xdO2FKx5FxJ4NlrwLYdzGQahjAdjiyxGThNvPTLpS1ZeEOQqwsW7ssPqmMW9Iq9jp6ze9ZU1eh\\\\\\\/L99iZgDvT0XCUJk9kFMZ5chrXBsH5q0SzQRXad8ZGHAcAj9GhneZ13E2kUsnz0i\\\\\\\/+XblyGMXNe7CI6lT52+DZl+9\\\\\\\/RrOjfoevTdpH1tnH+ltdXvxtNsbQXGAuQh4GUzkTVRbzSNQYG5GrlnduNP2FCLmqgNZnoLXyIEixbRFkB9M3KhKVqSbbowaU1BUNJdD6iU6JPgT2L+v6T0c6\\\\\\\/dRbtpterrpbu0VUni\\\\\\\/hrCFfjnqYqTa1vmrQ5xUnAU\\\\\\\/4avQDu+HhLICmCIH5E3WicaXe8B0oPnzEIrnkD+ooxbSfuk1NDZjVi718m4iznkiBNxAHYu9vCxH\\\\\\\/Rf\\\\\\\/0F9FUSCWFAREaSnev29iX94H6W4zE83tseyWudB2gHiN6mEr6E3gxeefBWWFztbpVCYd877KiSqsrfDql4G9mGgme7icLKyJSYScCRRaFaoLMUa+oCInVHtdBeVzWd9yPvLwY7B5\\\\\\\/ouAgchcAcpljdQEaiqv3JfeW3hF2yCFjG3yhqYuai+oBu8HZMgmAlcYwm28i+EtKKLduOw852vY+SgmOdvoPStyunNHtvz3FH066p7pJQy1fmqxHNa\\\\\\\/df3NMBC7EpkM\\\\\\\/KP5NgCfeF4GZwe8UIteA+mEpfbryEDwpolq69HkDVo4MceWQworQMBOeFUC0wmy7yFhIQeIAkUwgbtuZ7zoLu\\\\\\\/MV4+IVBsbZiURcRzKtqaIBNT6ASqtdlbROYXaRfErNPLdjApS8ny16\\\\\\\/U3Jah7OkiPiI9zpYDyIxywaAbNu6\\\\\\\/E5G6r7Ybqs+uEKAsdwr2mAUtO1xy1jBtDnCobTMfuDpkD77UqbRFkyUjACEihjRxsYIb14t2udrE4+p89IRieL4wOVZ4CnqA\\\\\\\/N+FIMEGnow1z8XvnbJnmHzaMp6zcJHe1sOiLrjh2jjW9vhpmjl5rHzs1jxy8Zh9h84mNZgaRl4FfZSLf+PohV0lYRSpcDwZmc4tQ8Uw\\\\\\\/bkBaMq0uw3JCt+ts2Ecs67Z2\\\\\\\/aQLst0imRtLXGseIHeV3TsGRhlkEDuWKuVNOOSCgS\\\\\\\/\\\\\\\/yC6AsYpQzgyuDK3hVJ+lvjZavZbDa\\\\\\\/b+UjX0gOSGZvN+VWRyFkiMNq37XeymQKvJclbsz9q7YRFXi+5KGIZ6BTbtdaOH2VyEVvcfiStjdAcJ3A9RbX+N1ZLmrco4nELOnHohSKNfV7MVXnPMwMpPUOVgvR5HOQvUtOeELu0kTe1d48kBf4KTcHdRu+z\\\\\\\/WfUovRB+EBY0\\\\\\\/7V+WDHtsMZLsVeO47CLx2Xop6P2ruWUAKKjCyvb7nIw8I18qv37ZTSA4HvkKgUn7rNDJABE1oOM55oyFJHdObpAvK2cRmm+Jx0lk5p7GuYlWgd82fOY+fG9QSa7Bbl2mnF1b2lrGCgCUG8JuVSnee7Zosd8weHDFswMsNphofokgd+xoXFVLXo6iyWTW6eKsiKEgdFmFLEvXwKSPTdr1oMg5l+lfydscnEhYxv8yyPQFY38gQ1oGaQ3bX2lozr5XAIyxFuIQOwl8sub2UpTklLatr35l6cYGrP+oPbDNO\\\\\\\/jW\\\\\\\/DmQIGwiiEhJK2JgE0IWh6hlWDLmK6aK7yZ1Nxln93y6RTcRUVpduzG2gZCeIf05PNYwhBIcJfgp9UD2UtCebVzphljSHzvJ\\\\\\\/6uiMRma9\\\\\\\/y34ut5NbjWdz\\\\\\\/St6FfVAZY1kQyM9duUnPYBWoOwC\\\\\\\/vNP8bkFa0llFoTPzOmdz+G87qF4+3hy38640yaU8gsUNar6IPQ0GfFaDCc0cWDwuPkgVMkd1WVCZoMOjR2s2pCks6CDtWAvn7Rx8Q5aOua1ApaNr1U3qy2Xz0zbfrtOI8DOS3w9\\\\\\\/qlvLTPfmjk6O+KXBa176nHb9RHNib+pW3vzZM7MYiAFXLnBmtAIlG7sW9pfufkLpEe56kajV2UTTe7BQvXftmtlLK0RaJJ7Hl\\\\\\\/7hj8wC83diBlcbRGVpabxBYcDDPRGiDeQk5JCcOqJT0i0Xr\\\\\\\/U2uIGHWjgVagE0mINgpvF0ykhjD8Y6A6LSCtpZfQdgMdFx9CKwor9UTk+DGufnBXByYB1KxatzBRuWJLwI+Z8a2vbwyHMktTbqUNzYTNX3stcI0h86pQiTzTAcVMQKFinxgonMu9UHheZ6AI94tjKD03MX9za1J8Eiy3Ws3Bz\\\\\\\/YY43mhoLTqepD+yGGFXz4qWKQjzN0Ec3JnV9DgGrnwa8ipOl8Chw0cGWyt806BNl8aXlJa7WFtzMxkoa1pIJ\\\\\\\/Qcc7ZkbuGBaQbIsMmCd7SZFEFo7C4MLpVEciw8v2tIhyNE1T3L7DX+I33UeYmfH2xm1ZwpkmWODUoUwId5vbZ6sPKFEuVM0ASUnkt0X6c\\\\\\\/WMSrcDes40QSf8ejmBq3YDOmnQ8Cyi5JM03X2ZtmWByAwfkz2ea\\\\\\\/FSFR6kcYIrcmpoidVqvt4bN8F0FQ+XHxI2O4dwrZtOewetit03BdgQdg+d8YzSnxcZPjZBDuLxkVwAx3wbsvNcdJjQ4MdYfMFo3kj92HkgBCjexkj69W\\\\\\\/fxuzjFCEm2uiAE3Tm\\\\\\\/wuylf68h4G3HqXIggtlWvWc5031Y2Glh3MiP3v97wNOnMLXEOFi4hwX+8hVnez4sm8JSlo2UrFyraqGdIbSpsukYn6Xb8vJnrj6hS65euaS6FFdIMc\\\\\\\/8JyG9wz9jqzfT30cXhXBnxUsanVlwvGX0wM\\\\\\\/d0WyaHwQT3BFdhV9ow5DPKzkdYp2JBfUCTUvLGKYQ621goIFx\\\\\\\/gzR8Z8mf3BQPtApK\\\\\\\/LB6rpqZry03RYE2aKpnZl4snTdDbvXUEkajSnwdFWWv6d+62PsFGwuN\\\\\\\/LdLdNucLhecoylLPNmQs9tBuNLorii5473uD51VsDDMvQ+ikr3xo3rdgMInJQVeyxRVJBJ8Ty\\\\\\\/qdGG5lFj9MY0vkiv7VPs+U5o98JPJaU1QJ0vFO1EhyYjg1qN0zKv0JhDfbnwsThR00+otpemwgiYJfHK4tJmKu\\\\\\\/Si6o+f20ac2PZHtU0DlFzWBQusp6zCsjoH2lvomDAvIEqYKnuGCFBDSC+we2IRjCP48QadCRrAEV3Oj7L2I94s3JAWk5n6jNPlsK3\\\\\\\/aJsMIKTevjgcxgsXT6gorvvUx5LIhq9P3QLN70CHMpGzodEty8k5+XdPYYPm95FWWSQhpmMKDsT3fwwg+P9Vx+hDkZR2Dv641B74uv07PGcBKBI9rjCeAj2pRYaweQtaiVxXBUd\\\\\\\/SOuG5vZjQcQRmJ+eb0q\\\\\\\/9O2ZjEX0olC+h84Vv3EmQomSoJCl8dtmKq2k++LOqRIlQdnJ6RwhNAlm62dFyOOnQoN0L2sNPxsNYfS9vP9tjvnAkdxCzLPnZijqfstvkRJVnmVyRyFFCmuFuMslq578Tc3lZLrAaSkCHY3KybZiIO+yFaAvTrwtEQwofm2h1G8YUVDXDybPjmx2Rj\\\\\\\/4i1vq0UDdbDYZSNIetGloZ6RJFOW8A2awNNC3Ye51SzGojenVAj94LYmzPetR3QbtyClKQ2AO1nxYhu653Uo9C+V9dCdehSu31k8degyLjjJq9VNLHJxkQdtJW+HBYM5qxzbiKo+eYWxlkkiAMwAWHvjmjA7jzCd1UypaprghsB\\\\\\\/K8Yu+jMz9P1vi+4je0fCVeFNLtjotp+u+2lw9LFrZvrivG2MAcwvS1xCOvW22rDPlKKd73g8CvGgbOFw+4fZ8YPKoMhVdWu3qfmj80JuZTpMWnJiaphSnI6XwZ1\\\\\\\/Gn7Rmx6pKQsktDpfdIYNKwU2FXF3GDzr0Tp\\\\\\\/p7LSdI8ywN6hyG7\\\\\\\/OnslQJ1PzweJ8AL4pRBBQirIYurCQFbgh+GalqkNw+7HKrgjRYp11wBHg\\\\\\\/6WUy77bEcHdFR4xH4ERnYBQHAetBDcqjigLLbrfGH\\\\\\\/llmbEBMyukS4iuHZEa3AMjlDfTZ5budPnSAS8FQkEq8FV4UNmSlMLvhBB4jfmuQaygIYDqARq7ZfK78w7HRC39ih0jj9Tv66Zf4gdgtnRx3+SwqC3v0ZgP\\\\\\\/fINo3nxEATktzhzY7NM5sRT3ClngHAHQ0iFrAf4\\\\\\\/3GAzzNj6zWeKWRDptShe26L\\\\\\\/fMsNDdkWTkdAyrxqi6bupQ5c3RX9vW3nlUSo7SRg9hWAKcC6BSThYGeE44S\\\\\\\/KDNv6Tqlv1AdgX5Go76RKIq+LBFiGYm0Yff4Q+qo3sS1QSOl05LyXVaPZ3VBwHYBS5UYaF6VqUNnIKttg13TeqHykeIz8b8sK3WPWdqmI7or2YuAClARzIiWPT6BZuJvEUX21ZS2wa+ixSCWO7nXkGwSCt+vaR8WTVJZbGGR+yHQxBkh\\\\\\\/azXxEP1pVxNhXth+JsT42ZZia4itbTG4gfk6w3X6zzyuKaWjBmGGwgndKFV3SULAEplbNtPLALTcuMtCgwagQ7gI9f4QHLprVgWDKy\\\\\\\/amkDPaJ0y7In4DUsUe1FE9B1GQ0oMkG+987SxvehjhAOzVLOkkGOk6cnnfl2LlkopGH28G1FxunfbnhnoGJCfYisTknQ00OePvuZJZ6pdojrk7O9+MBrD4jUUw9mz3oVRWmx1UoZ0Qx6mKDfMcZmN9Pz54ZjG6EDqXGT3GtuK5Ll1tJFBppwNZ9VDWgfv4MLLiQPr4u7eg4kIYY4s3mvNsquEzaaemcguUJvoMYw7PvCEqhEzI1kOozIOPW1NB+9TxVpcogQZr3e\\\\\\\/ldwFjdOj\\\\\\\/fuRyOkYGvfGvBF3\\\\\\\/r7kRFwq+Md05OlCgufR3s3NaQgzEl5CYTWqPq4zlfwXlmbT8SDt0aDL32E9x42VJb22Zq6z6EvHlvLcUYoL0\\\\\\\/fnUeBBsYq4dZ+t2qsIEjLpTZ+ITNAHojgEUdt4E1aTBieUP+3nMaK0k6Mm7r8nb79ti52\\\\\\\/IjKIs7W50k8CbL1bOlguZMgrRcryzvEE5z3KXZ1Qoq3hzqtQjGA4qaPFlmrI4WdczvGyL7PtAj+VJMxnwWV0GzOQruLCs33dJCC\\\\\\\/Fc2juoBT\\\\\\\/xY20zZ5oOy+QSxDUn3NOESQXtDsuQlOKoJJPQrF4+I521MaUZ\\\\\\\/tEsvBb2EVlaH9+xzQAgd2bn4CaVpvF2NXDuYln7WBeAxNk66E5EDnPmSaN45FLCR4DQvoiaVgXRvOrWYnecRd3TaUcaUZ+9TgtrcTghbJBP0ihLggW4SfhgUVy8VaiCsj0G\\\\\\\/D0Njbt\\\\\\\/QyznqcSR8eoj0opjfCHCHNXDlecMbjkZhvrR3aHiBrNTR\\\\\\\/bjkWuiW5yvo0c8JLgM\\\\\\\/hzg0nZaBRG481B3aZu1pb+Uq9gptX84I2wHDLXq9PCUn\\\\\\\/Lp\\\\\\\/2790W2JIhKgeZkw6MfntqnrXZvg77gkrAg9bS4D6ICQN5ac4XWcB+goK2ktkxHF\\\\\\\/rwdK4lHCwK2LmuOd1TMx9NZb7h1uCp169UdANLoAcYCJdA3b7\\\\\\\/ipnBHwxKRZxa\\\\\\\/Vc00w6LJViqnx+dcdVF3zHxa9JRhrb4bhZQZ2IbuNnDgmmc2cqTNFdlPMS8QcyqhiV9Q76KRSp5jm2ReV\\\\\\\/tU7fR3yH5oPsCRo5aaDIjhGCFV2bYEe\\\\\\\/5E7Xj6dNJA\\\\\\\/JSNfekqLsBbqOLWMZXJZlmBSDJdtJCRpuVc38vVByNm5aksco82BZ6giJ6QXDwN\\\\\\\/Q7coIlBSM5zoWHxwr3wC658jbahex\\\\\\\/tgYFYifBOMsUVvPEpQH5C8MUDPy65EG50WahFGi7k1BS24it15L\\\\\\\/ucikRvAOXQN3fHmkxe1YjFIzfJ0r6Aqa0pLNOINskklGJefZ0i3CRLBWY5NAfzj5niNpVIUeJLeSwYTpSaxiAIef9g8Kfk5ST7U8rvI\\\u0022,\\\u0022iv\\\u0022:\\\u00227hhfilFQqTG9WgolLpbI5Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227c84ddf5d0f8f2100d20d015e12899f49c42540056f8994b81dc14fd38b20a29\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mdXiAX\\\\\\\/SfDBYAF4Rp3BIF7fkalKOgfqPHDiBZF\\\\\\\/Z6azywQbtKT71DQaRA2Y\\\\\\\/c0VY1l27XL+zsQZGZ\\\\\\\/VGVbliovfjJuP7J2hBmOrNIW602gmor8xFnqyuv2H2RSk4\\\\\\\/375WVkXo4e8ZhctznCcl1lUY0Fi2el\\\\\\\/PBM8AGjyuer1RWIrQNIUmf+lZ1pinzay0eEvPi0NiMzKbD2LWZeWipsXuh38iMBGFaBUJLrrnH7jxuwwTOr4eptIFjkFWh6vnsG8z3YJinD36khSSxFU0eupr5NM+\\\\\\\/9pYvEbnYdtZeWZVZPCylSQ6S3JlMAWUB0qVLT1ln8RgkFdL3QaKRhrrJGNb7f\\\\\\\/oRb2erj0K4ApJCg+tXhdSLN\\\\\\\/J6oo7u+0qnJORDyn2rJWtKrioTwfTwS0YyPmMHkqKt5d5t8jhWsI93Yod0mXKC0qMj2z2P+g9\\\\\\\/ilwr5Y6gZ3PLqP13WT9ILyv6leI9Zk7JsaVxQGcXERPcY3HoprlBOpm2zTrndQSAvnyBZVIqw5pKWA1+LUnG9alyKx+N7\\\\\\\/VCy2lWFDZpRNo55fXNkgo2sFcCwV6xZHo8\\\\\\\/3A7JLKBZ6aQ1vMu0TDXZRwMmSvSEogAjQd59+jgGOYOnVFTU+olhzI3AbX7nov3vZ12gTWabcXaZLcbSRPnjmd4eblsNmpY1oVY+Mg3ChsURaRmRkV7jVMz0VMo4JxrWc+4Ict0DOTMVc\\\\\\\/jTnsnTWgzJG8l4j6weicQIYxwLps160Ls4wJ8rnaGiy8eFN9ddGjreiKrZmCspWmxoPq4ed3GxbLb3s12kd8EWZBzicgWqFK5jYGvEbpFYZWGiszRgNJPXBayM\\\\\\\/3gItnzUfbx0GGnnUGBgV4sP3\\\\\\\/36PwPzD0V0kv87vLIGH+YEVHVYPCBboHXB0UVigomr5Nj4AbaMGcJghoq\\\\\\\/e+fgNIPnhc5XX+MnGPe5IbCimuyDrFZfr5vvVd65RYaPEm5oDNfDfERMJQ5qkuelvWa75evkBg+3hnQwEZsjPvcfO++Bg7bVX4j3QbZbER65e33GlMAdnpP41F6zVXvrO6mmXWuDWbD25eb0S2YzxtT+VpMJ8Uo13XkCVytMAZoTJNf+4gYjB9IPj0\\\\\\\/F\\\\\\\/65rbgycIWFNhDvgLE4qQmsEjRP5GBbphgkTCI0s5jjaREKFLuDX9NylLJKPS3dtkgzEcgx8n6+Oaj+rnjw\\\\\\\/S0pNzwbF\\\\\\\/Gofw+zQUYo6PtgywzieYf\\\\\\\/fPNEX8keiNSooPLZ3S8BiXQyUTxEbG7tnt+vbf+WEP3+RaeWLQ9Zt4B4C5CrXj13dMIUDWfgJ1\\\\\\\/htKaLeWIjT9NrNvZ4qbAHNS2z1ImBtOubAgzy2x\\\\\\\/1LukVxJTnZTH9y1rHVzppl\\\\\\\/f8MKjxVZVEmKNvy+r4TAZZC5yF+NPVD0yqz5h\\\\\\\/LQVXUceKnkB1ecArrrFVs9mG0ExXykxpQaaByRGCZNWkfAC9yzbC+uwTeRY+2uFGaludJrz39vlQizy3RStThnO5XuaO8K\\\\\\\/kB+IJDzblvPooG8O8xUUobZGa9XohUAnHaEkK7EiUn4yN7YXx1zSh7rSlXe4nzWJoYyA+V88QcEN6AZq2fFmzvZ+GYsH6zDp8XxBEvA\\\\\\\/A64lOmqQopm1JKYJ2XMp6b10mJhQpRww9vAdgmysP+mDFvd2R+XGGHpRNFdH7iPZnLqKA6xlEyoxtDgjg\\\\\\\/Q8uHi\\\\\\\/uPBvK2QpyFjzIFOyDqZc2w0NTyhFs4A6nBLQlVal9\\\\\\\/wqvn5y2P84vbny4Vig90xC3f1PTk42XbO+J0BmFI9RhZMsCvKRzanbI8ByI10N9cunB\\\\\\\/D8YIwnSRCE0fR9YIbphRjYHPaLWgU+QgjVtNTtIZVaaCYOr7R9OFd0igofYZ9AkClOfy3rZ8CKR8sgsp\\\\\\\/sa+tJAJFFG5tjmUPwrzXFLyRCZR57TF\\\\\\\/4e+2nf+\\\\\\\/iqy5Z14aLNGXS1YrPqVyMLithEmI+m04\\\\\\\/xumjR5KqF\\\\\\\/HcJe6PloRn2QcDM+1j\\\\\\\/6M\\\\\\\/WgadQTI4u3MSyPZWf2+WR09QwI65extFANKD+BPZCD3Lj74OLIkirqfQOGLkxIKzagj\\\u0022,\\\u0022iv\\\u0022:\\\u0022AEUrgHIoNTVDtGVBtMEHtA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226436b75692da998e1dd8acbd0871c0932547e56c90da31428d7f565d87d1b548\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459622 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/1/163" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Lclu5zZWGCKdjPdN13zkxLkwZM7iweVh7vZRXB8VvNw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/1/163" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FTDw4J3e9rPMuqjas8GKZFUx8AzmVZET6xzJ727AUvY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/1/155" title="Generation of a Useful roX1 Allele by Targeted Gene Conversion" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/1/173" title="par-1, Atypical pkc, and PP2A/B55 sur-6 Are Implicated in the Regulation of Exocyst-Mediated Membrane Trafficking in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459622" id="node459622" data-pisa="ggg;4/1/163" data-pisa-master="ggg;g3.113.008870" data-apath="/ggg/4/1/163.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Pattern and Distribution of Deleterious Mutations in Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sofiane Mezmouk</span> and <span class="highwire-citation-author" data-delta="1">Jeffrey Ross-Ibarra</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">163-171;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008870 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459622" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sofiane Mezmouk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California–Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sofiane%2BMezmouk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mezmouk%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASofiane%2BMezmouk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jeffrey Ross-Ibarra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California–Davis, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Population Biology and Genome Center, University of California–Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BRoss-Ibarra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ross-Ibarra%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BRoss-Ibarra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rossibarra@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">rossibarra@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/1/163" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459622" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459622/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/1/163.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459622" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459622/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/1/163.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459622" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459622/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/1/163.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459622" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459622/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/1/163.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459622" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459622/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">Most nonsynonymous mutations are thought to be deleterious because of their effect on protein sequence and are expected to be removed or kept at low frequency by the action of natural selection. Nonetheless, the effect of positive selection on linked sites or drift in small or inbred populations may also impact the evolution of deleterious alleles. Despite their potential to affect complex trait phenotypes, deleterious alleles are difficult to study precisely because they are often at low frequency. Here, we made use of genome-wide genotyping data to characterize deleterious variants in a large panel of maize inbred lines. We show that, despite small effective population sizes and inbreeding, most putatively deleterious SNPs are indeed at low frequencies within individual genetic groups. We find that genes associated with a number of complex traits are enriched for deleterious variants. Together, these data are consistent with the dominance model of heterosis, in which complementation of numerous low-frequency, weak deleterious variants contribute to hybrid vigor.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/deleterious-mutations" class="hw-term hw-article-keyword hw-article-keyword-deleterious-mutations" rel="nofollow">Deleterious mutations</a></li><li class="kwd"><a href="/keyword/gbs-snps" class="hw-term hw-article-keyword hw-article-keyword-gbs-snps" rel="nofollow">GBS SNPs</a></li><li class="kwd"><a href="/keyword/heterosis" class="hw-term hw-article-keyword hw-article-keyword-heterosis" rel="nofollow">Heterosis</a></li><li class="kwd"><a href="/keyword/maize" class="hw-term hw-article-keyword hw-article-keyword-maize" rel="nofollow">Maize</a></li><li class="kwd"><a href="/keyword/quantitative-traits" class="hw-term hw-article-keyword hw-article-keyword-quantitative-traits" rel="nofollow">Quantitative traits</a></li></ul><p id="p-6">The effect of new mutations on organismal fitness is not well understood, but both theoretical considerations (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fisher 1930</a>) and empirical estimates (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Joseph and Hall 2004</a>) suggest that most new mutations are deleterious and that only a small minority are beneficial. Strongly deleterious mutations are expected to be kept at low frequencies by natural selection, whereas weakly deleterious alleles may be effectively neutral (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Ohta 1973</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kimura 1983</a>) and subject to the effects of genetic drift (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lynch and Gabriel 1990</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lande 1994</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Whitlock <em>et al.</em> 2003</a>). In addition to selection and drift, a number of other factors such as mating system and recombination rate also impact the evolution of deleterious alleles. Selfing species and inbreeding within populations will expose lethal mutations to selection faster than in an outcrossing population (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Wang <em>et al.</em> 1999</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Glémin <em>et al.</em> 2003</a>). Moreover, in genomic regions with low levels of recombination, selection against deleterious mutations will be less effective (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Charlesworth <em>et al.</em> 1993</a>), and the potential exists for deleterious mutations to rise to high frequency due to the effects of linked selection on beneficial mutations (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Felsenstein 1974</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hill and Robertson 1966</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chun and Fay 2011</a>).</p><p id="p-7">Deleterious alleles may play an important functional role in affecting the phenotype of traits of interest, and complementation between haplotypes carrying different deleterious alleles may explain much of the observation of hybrid vigor or heterosis (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Charlesworth and Willis 2009</a>). In studies of human disease, a significant correlation was observed between the deleterious predictions of single-nucleotide polymorphisms (SNPs) and their association with cancer (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Zhu <em>et al.</em> 2004</a>); predicted rare, deleterious SNPs also were shown to be involved in common diseases (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cohen <em>et al.</em> 2004</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Smigrodzki <em>et al.</em> 2004</a>). Furthermore, rare, deleterious SNPs have gained interest as the result of their potential role in explaining quantitative trait variation (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gibson 2012</a>), especially in populations that have experienced recent growth (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lohmueller 2013</a>).</p><p id="p-8">Evaluating the abundance and frequency of deleterious mutations is thus of considerable interest and has been investigated in a wide range of species. These analyses have varied in terms of the percentage of nonsynonymous sites estimated to be deleterious, from 3% in bacterial populations (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hughes 2005</a>) to 80% in the human genome (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fay <em>et al.</em> 2001</a>). They have also shown that recently bottlenecked populations may have a higher abundance of deleterious sites and that heterozygosity at deleterious SNPs is lower than at synonymous SNPs (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lohmueller <em>et al.</em> 2008</a>). In plants, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gossmann <em>et al.</em> (2010)</a> found that most new mutations in plants are strongly deleterious, with only 25% acting as effectively neutral. <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cao <em>et al.</em> (2011)</a> show that the abundance of deleterious variants correlates with effective population size in <em>Arabidopsis thaliana</em>, and a demographic bottleneck appears to have relaxed purifying selection in <em>Capsella rubella</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brandvain <em>et al.</em> 2013</a>). Other analyses of purifying selection in plants have implicated a role for environmental differences (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Tellier <em>et al.</em> 2011</a>) and identified differences among genes based on their level of expression (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Paape <em>et al.</em> 2013</a>). In natural populations of <em>Arabidopsis thaliana</em>, selection appears to act to maintain variants that are locally adaptive but deleterious elsewhere (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fournier-Level <em>et al.</em> 2011</a>), whereas positive selection on domestication genes may have increased the abundance of deleterious variants in domesticated genomes such as rice (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Günther and Schmid 2010</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lu <em>et al.</em> 2006</a>). Although these studies have provided insight into the evolutionary fate of deleterious mutations, we still understand relatively little about the role of deleterious variants in effecting phenotypic traits.</p><p id="p-9">Maize (<em>Zea mays</em>) is a economically important cereal worldwide, with the highest yield and one of largest cultivated areas (FAO statistics, <a href="http://faostat.fao.org">http://faostat.fao.org</a>); it is also an important model for basic and applied research (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Strable and Scanlon 2009</a>). Maize traditionally was cultivated in open pollinated populations (landraces) but, after the first documented observations of hybrid vigor in this species (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">East 1907−1908</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Shull 1908</a>), inbred lines were developed and structured into heterotic groups that maximize intergroup combining ability. The transition from heterozygous populations to strongly structured heterotic groups of inbred lines makes maize of interest for analyzing the distribution and frequency of deleterious mutations. Furthermore, high observed values of hybrid vigor or heterosis in maize hybrids makes it an excellent system for studying the effects of deleterious mutations and their contribution to heterosis. The dominance model of heterosis posits that inbred lines are homozygous for a number of recessive deleterious alleles and that crosses between inbreds carrying different complements of deleterious alleles will result in heterozygous progeny with higher fitness than either parent.</p><p id="p-10">The aim of the current study was to (1) carry out a genome-wide scan for deleterious mutations in a maize diversity panel, (2) analyze their distribution across the genome and within different genetic groups, and (3) test for enrichment of deleterious loci in the results of genome-wide association mapping. High-density SNPs and phenotypic data available for a large sample of inbred lines and hybrids were used to address these questions. Our results showed that maize inbred lines are segregating for a large number of predicted deleterious variants (20−40% of protein coding SNPs were predicted to have a deleterious allele), and that these alleles are generally at very low frequencies with few fixed differences observed among different genetic groups. Genome-wide association analysis of hybrid vigor finds little evidence for enrichment of individual deleterious SNPs, but significant enrichment for genes containing deleterious SNPs, suggesting a meaningful role for dominance and complementation in explaining observations of hybrid vigor.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and methods</h2><div id="sec-2" class="subsection"><h3>Plant material and phenotypic data</h3><p id="p-11">We used phenotypic data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1/FileS3.zip">File S3</a>) published in <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Flint-Garcia <em>et al.</em> (2005)</a> for 247 maize inbred lines (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1/008870SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1/FileS1.pdf">File S1</a> for a list of inbred lines). Each inbred line was crossed to the stiff-stalk inbred B73 (population A) and both the inbred lines and their B73-hybrids were evaluated in 2003, in adjacent blocks within three environments with a single replicate in each (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Flint-Garcia <em>et al.</em> 2009</a>). A subset of 102 inbreds were additionally crossed to both B73 (population B1) and Mo17 (population B2); both inbred lines and hybrids were evaluated in a single environment in 2006 (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Flint-Garcia <em>et al.</em> 2009</a>). <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1/TableS1.pdf">Table S1</a> lists the analyzed traits that are detailed in <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Flint-Garcia <em>et al.</em> (2009)</a>.</p><p id="p-12">The panel structure was previously analyzed (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Flint-Garcia <em>et al.</em> 2005</a>), and inbred lines were attributed to the following subpopulations: stiff-stalk (27 inbred lines), non-stiff stalk (90 inbred lines), tropicals (60 inbred lines), popcorns (eight inbred lines), sweet (six inbred lines), and mixed (56 inbred lines). For the main temperate inbred lines, these subpopulations correspond to the different heterotic groups.</p></div><div id="sec-3" class="subsection"><h3>Genotypic data</h3><p id="p-13">We made use of genotypic data from <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Larsson <em>et al.</em> (2013)</a> for the full set of 247 lines (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1/FileS4">File S4</a>). The latter were genotyped using the genotyping-by-sequencing approach (GBS; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Elshire <em>et al.</em> 2011</a>), resulting in a total of 437,650 partially imputed SNPs. Of these SNPs, 127,994 mapped to protein coding sequences representing 123,289 codons in 21,064 genes. The median (mean) percentage of missing data per SNP, including triallelic sites, was 1.06% (2.52%), whereas the percentage of heterozygous sites was 1.08% (2.52%). Only 4.5% of SNPs had more than 10% missing data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1/FigureS1.pdf">Figure S1</a>A), and 0.18% had more than 10% heterozygous genotypes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1/FigureS1.pdf">Figure S1</a>B).</p><p id="p-14">We estimated error rates by first comparing our genotyped inbred B73 to the B73 reference genome, then by comparing all our genotypes to those from 7225 overlapping SNPs on the maize SNP50 bead chip (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cook <em>et al.</em> 2012</a>). Compared with the reference genome, our B73 genotype differed (alternative homozygote allele) at 1.75% of SNPs, and across all lines our genotypes differed at a median (mean) rate of 1.83% (4.62%) from the maize SNP50 data (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Cook <em>et al.</em> 2012</a>).</p></div><div id="sec-4" class="subsection"><h3>Statistical analyses</h3><div id="sec-5" class="subsection"><h4>SNP annotation and analyses:</h4><p id="p-15">The first transcript of each gene in the B73 5b filtered gene set was used to annotate SNPs as synonymous and nonsynonymous with the software polydNdS from the analysis package of libsequence (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Thornton 2003</a>). The deleterious effects of amino acid changes were then predicted for proteins derived from the first transcript of each gene with both the SIFT (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Ng and Henikoff, 2003</a>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">2006</a>) and MAPP (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Stone and Sidow 2005</a>) software packages.</p><p id="p-16">SIFT uses homologous sequences identified by PSI-BLAST against protein databases to identify conserved amino acids. The software provides a scaled score of the putative deleterious effect of a particular amino acid at a position along a protein.</p><p id="p-17">MAPP predicts deleterious amino acid polymorphisms from a user-defined alignment of protein homologs. It uses the phylogenetic relatedness among sequences and the physicochemical properties of amino acids to quantify the potential deleterious effect of a given amino acid change. We created alignments for MAPP using three different methods. First, we made BLASTX comparisons of protein sequences from maize against the TrEMBL database (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Boeckmann <em>et al.</em> 2003</a>) retaining all proteins with an e-value ≤ 10<sup>−40</sup> and at least 60% identity with the query. Second, we used a reciprocal best BLAST criterion to compare protein sequences of maize against protein sequences from 31 plant genomes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1/FileS2.pdf">File S2</a>) from Phytozome version 8.0 (<a href="http://www.phytozome.net">http://www.phytozome.net</a>), retaining the best hit protein from each of the other genomes with an e-value ≤10<sup>−100</sup> and ≥70% coverage of the query length. Finally, we made use of a set of syntenic genes from the grasses <em>Zea mays</em>, <em>Sorghum bicolor</em>, <em>Oryza sativa</em>, and <em>Brachypodium distachyon</em> (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Schnable <em>et al.</em> 2012</a>). For each set of proteins, ClustalW2 (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Larkin <em>et al.</em> 2007</a>) was used to align the sequences and build a neighbor-joining tree. Custom R code (<a href="https://github.com/RILAB/siftmappR">https://github.com/RILAB/siftmappR</a>) was used to link amino acid positions to SNP positions and to link the amino acid polymorphisms to MAPP and SIFT predictions.</p><p id="p-18">The derived site frequency spectrum was calculated for all protein coding SNPs using <em>Tripsacum</em> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chia <em>et al.</em> 2012</a>) to determine ancestral state. The pattern of haplotype sharing (PHS) across the genome (PHS statistics; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Toomajian <em>et al.</em> 2006</a>) was analyzed within each of the tropical, stiff-stalk, non-stiff stalk, and mixed subpopulations as defined by <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Flint-Garcia <em>et al.</em> (2005)</a>. We will refer to these subpopulations as “genetic groups.”</p></div><div id="sec-6" class="subsection"><h4>Phenotypic data analyses:</h4><p id="p-19">Genetic values (the average phenotypic value of all individuals with the same genotype) of inbreds and hybrids in population B were taken from <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Flint-Garcia <em>et al.</em> (2009)</a>. Genetic values for population A were estimated from the raw phenotypic data using the model:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-1.gif"/></noscript></span></span>where <strong>y</strong> is the vector of phenotypic values, <em>μ</em> is the mean of <strong>y</strong>, <em>X</em> is an incidence matrix, <strong>g</strong> is the vector of fixed individual effects, and <em>ε</em> comprises the residuals assumed to be <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-2.gif"/></noscript></span></span>.</p><p id="p-20">Hybrid vigor for each individual was estimated by both best- and mid-parent heterosis (<em>BPH</em> and <em>MPH</em>, respectively):<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-3.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-4.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-5.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-6.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-7.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-8.gif"/></noscript></span></span> are the genetic values of the hybrid and its two parents <em>i</em> and <em>j</em>. <em>BPH<sub>min</sub></em> was used instead of <em>BPH<sub>max</sub></em> for days to anthesis, tassel branch count, tassel angle, and upper leaf angle.</p></div><div id="sec-7" class="subsection"><h4>Association mapping:</h4><p id="p-21">SNP association with the genetic values of the inbred lines was tested with the R package EMMA (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kang <em>et al.</em> 2008</a>), following a mixed linear model similar to <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Yu <em>et al.</em> (2006)</a>:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-9.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-10.gif"/></noscript></span></span> is the vector of estimated genetic values for inbred lines, <em>μ</em> is the mean of <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-11.gif"/></noscript></span></span>, <em>M</em> is the tested SNP, <em>ϑ</em> is the SNP effect, <em>S</em> indicates the structure covariates estimated by <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Flint-Garcia <em>et al.</em> (2005)</a> using the STRUCTURE software (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Pritchard <em>et al.</em> 2000</a>), <em>β</em> indicates the fixed structure effects, <em>Z</em> is an incidence matrix, <strong>u</strong> is a vector of polygenic background effects assumed to be <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-12.gif"/></noscript></span></span>, and <em>ε</em> comprises the model residuals assumed to be <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-13.gif"/></noscript></span></span>. The coancestry matrix <em>K</em> among inbred lines was approximated by an identity by state matrix calculated with the SNPs. Only SNPs with a minor allele frequency ≥0.05 were used for association mapping tests.</p><p id="p-22">In hybrids, we tested the effect of heterozygosity at a given locus on observed heterosis. Each SNP was assigned numerical values corresponding to 0 if the hybrid is homozygous or 1 if the hybrid is heterozygous. The association mapping tests were thus carried out between heterozygosity at a given locus and hybrid vigor:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-14.gif"/></noscript></span></span>where <em>PH</em> is the vector of heterosis values (either <em>MPH</em>, <em>BPH<sub>max</sub></em>, or <em>BPH<sub>min</sub></em>), <em>μ</em>′ is the mean of <em>PH</em>, <em>D</em> is the genetic distance between the tester (B73 or Mo17) and each inbred line, <em>β</em> is the fixed effect of that distance, <em>H</em> is the tested locus (1 if heterozygote and 0 if homozygote), <em>ϑ</em> the effect of the locus, and <em>ε</em>′ is the vector of residuals assumed to be <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/163/embed/mml-math-15.gif"/></noscript></span></span>. SNPs were deemed to be statistically significant at <em>P</em> ≤ 0.001. Analyses also were conducted in which we controlled for a false-discovery rate (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Benjamini and Hochberg 1995</a>) at 20%.</p></div></div></div><div class="section results-discussion" id="sec-8"><h2 class="">Results and Discussion</h2><div id="sec-9" class="subsection"><h3>Prediction of deleterious mutations</h3><p id="p-23">To investigate deleterious mutations in a diverse set of maize inbred lines, we first applied two complementary approaches to predict deleterious mutations across the maize genome. We applied the software packages SIFT (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Ng and Henikoff, 2003</a>, <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">2006</a>) and MAPP (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Stone and Sidow 2005</a>) to the 39,656 genes in version 5b of the maize filtered gene set (<a href="http://www.maizesequence.org">http://www.maizesequence.org</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Schnable <em>et al.</em> 2009</a>). SIFT predicted amino acid change consequences for nearly 12 million codons in 32,000 genes, whereas MAPP obtained predictions for a total of 11 million codons in 29,000 genes combined across the three ortholog datasets used (see the section <em>Materials and Methods</em>). More than 80% of predictions were congruent between the two approaches, similar to what has been seen in <em>Arabidopsis</em> and rice (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Günther and Schmid 2010</a>). SIFT and MAPP respectively identified ~80% and 60% of amino acid polymorphisms as “tolerated,” with the remainder predicted to be premature stop codons or “non-tolerated” amino acid changes; we will refer to these latter categories as predicted deleterious SNPs.</p><p id="p-24">We then took advantage of recently published GBS (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Elshire <em>et al.</em> 2011</a>) data to survey potentially deleterious mutations across a panel of 247 diverse maize inbred lines (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Larsson <em>et al.</em> 2013</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Romay <em>et al.</em> 2013</a>). The genotyping data include a total of 437,650 SNPs covering 123,289 codons. SIFT and MAPP predictions were obtained for 112,326 and 107,472 codons representing 19,145 and 18,255 genes, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1/FigureS2.pdf">Figure S2</a>). Nearly 50% of these codons showed no amino acid polymorphism in each dataset; although the vast majority of these monomorphic amino acids were attributable to synonymous polymorphisms in the GBS data, several hundred predicted deleterious amino acids were fixed across all maize lines analyzed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1/TableS2.pdf">Table S2</a>). Combining results from both SIFT and MAPP, our data consist of 25,352 predicted deleterious SNPs in 11,034 genes.</p></div><div id="sec-10" class="subsection"><h3>Characterization of deleterious SNPs in a diversity panel</h3><p id="p-25">Across all lines, the derived site frequency spectrum (SFS) of coding SNPs showed an excess of rare variants compared with neutral expectations, with 45% of predicted deleterious SNPs occurring at derived frequencies less than 5% in the SFS across all lines. Even so, nonsynonymous SNPs showed an excess of rare variants compared with synonymous SNPs (Mann-Whitney <em>U</em>-test <em>P</em> &lt; 10<sup>−15</sup>), and predicted deleterious SNPs showed a marked excess of rare variants compared with both synonymous and non-deleterious nonsynonymous variants (Mann-Whitney <em>U</em>-test <em>P</em> &lt; 10<sup>−15</sup> for both comparisons; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The SFS of nondeleterious nonsynonymous was not distinguishable from that of synonymous variants (Mann-Whitney <em>U</em>-test <em>P</em> = 0.07). These observations are consistent with the action of weak purifying selection (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cummings and Clegg 1998</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Fay <em>et al.</em> 2001</a>) and independent corroboration of the utility of MAPP and SIFT in predicting deleterious variants.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/163/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Derived site frequency spectrum of synonymous, nonsynonymous nondeleterious, and nonsynonymous deleterious SNPs. Tripsacum was used as outgroup for identifying the derived allele." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-521486900" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Derived site frequency spectrum of synonymous, nonsynonymous nondeleterious, and nonsynonymous deleterious SNPs. &lt;em&gt;Tripsacum&lt;/em&gt; was used as outgroup for identifying the derived allele.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/163/F1.medium.gif" width="440" height="242"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/1/163/F1.medium.gif" width="440" height="242"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/163/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/163/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463917" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-26" class="first-child">Derived site frequency spectrum of synonymous, nonsynonymous nondeleterious, and nonsynonymous deleterious SNPs. <em>Tripsacum</em> was used as outgroup for identifying the derived allele.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Although most predicted deleterious alleles were rare, 923 were found segregating at high frequency (≥0.80) across all lines. To test whether these alleles may have been driven to high frequency by selection at linked loci during domestication (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Lu <em>et al.</em> 2006</a>), we analyzed the pattern of haplotype sharing across the genome (PHS statistics; <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Toomajian <em>et al.</em> 2006</a>). Only 87 of these SNPs (9.4% of all tests) showed signs of positive selection in at least one of the genetic groups, and only 25 (2.7%) were found in candidate regions for selection during maize domestication (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hufford <em>et al.</em> 2012</a>), providing little evidence to support hitchhiking during domestication as a major influence on the distribution of deleterious alleles in the genome.</p><p id="p-28">The proportion of genic SNPs predicted to be deleterious appeared relatively uniform (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1/FigureS3.pdf">Figure S3</a>) across the genome, showing a very low correlation with recombination rate (Pearson’s <em>r</em> of 0.06; <em>P</em> = 0.005) from the IBM (Intermated B73xMo17) genetic map (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gerke <em>et al.</em> 2013</a>). Explicit comparison of 1778 nonsynonymous pericentromeric (± 5 cM around the functional centromere) SNPs did not show an elevated proportion of predicted deleterious SNPs in comparison to the whole genome (Fisher’s exact test <em>P</em> = 0.68), and no correlation was observed between gene density and the proportion of predicted deleterious mutations in 1 Megabase windows (Pearson’s <em>r</em> of −0.06; <em>P</em> = 0.01). The negative correlation between recombination and residual heterozygosity observed in recombinant inbred lines of the maize nested association mapping population has been attributed to the inefficiency of selection against deleterious alleles in low recombination regions of the genome (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">McMullen <em>et al.</em> 2009</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gore <em>et al.</em> 2009</a>). Our results do not provide support for this explanation, perhaps suggesting that recombination in these regions over longer periods of time is sufficient to avoid the accumulation of deleterious alleles. Consistent with this idea, although regions of the <em>Drosophila</em> genome completely lacking in recombination showed a severe reduction in the efficacy of selection, little difference was observed between regions with high and low rates of recombination (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Haddrill <em>et al.</em> 2007</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/163/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of genic SNPs predicted to be synonymous, nonsynonymous nondeleterious, and nonsynonymous deleterious in 1-Mb windows along chromosome 1. The vertical dashed black line indicates the centromere position, and blank columns indicate windows with missing data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-521486900" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Proportion of genic SNPs predicted to be synonymous, nonsynonymous nondeleterious, and nonsynonymous deleterious in 1-Mb windows along chromosome 1. The vertical dashed black line indicates the centromere position, and blank columns indicate windows with missing data.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/163/F2.medium.gif" width="440" height="191"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/1/163/F2.medium.gif" width="440" height="191"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/163/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/163/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463866" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Proportion of genic SNPs predicted to be synonymous, nonsynonymous nondeleterious, and nonsynonymous deleterious in 1-Mb windows along chromosome 1. The vertical dashed black line indicates the centromere position, and blank columns indicate windows with missing data.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Individual lines varied considerably in their content of predicted deleterious alleles, carrying between 4 and 16% of all predicted deleterious alleles. Lines from the stiff-stalk group carried on average fewer deleterious mutations (9%) than did lines from other groups (14–15%), even after weighting by the total SNPs in each group (data not shown). Although drift due to a historically low <em>N<sub>e</sub></em> (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Messmer <em>et al.</em> 1991</a>) could explain this observation, other groups with low <em>N<sub>e</sub></em> such as the popcorns do not show such a trend. Instead, we posit that both the SIFT and MAPP algorithms may be biased against identifying deleterious alleles found in the reference B73 genome. Because B73 is a stiff-stalk line and both programs use the reference allele in identifying deleterious alleles, nonsynonmyous SNPs at appreciable frequency in the stiff-stalk group may be more likely falsely identified as tolerated. Similar bias has recently been described in analyses of the human genome (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Simons <em>et al.</em> 2013</a>).</p><p id="p-31">Allele sharing at predicted deleterious SNPs generally followed genome-wide patterns of identity by state (IBS). Within the non-stiff stalk, tropical, popcorn, and sweet groups, correlations were generally high (Pearson’s <em>r</em> of 0.75−0.99) between numbers of shared predicted deleterious alleles (mean of 5−10%) and IBS. Correlations between inbreds from different genetic groups were much lower (<em>r</em> of 0.25−0.52), however, as has been previously seen in correlations between IBS and heterosis observed at SSR loci (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Flint-Garcia <em>et al.</em> 2009</a>). The “mixed” (within group <em>r</em> = 0.22 and <em>r</em> = −0.05 to 0.36 with other groups) and stiff-stalk (within-group <em>r</em> = 0.15 and <em>r</em> = −0.65 to 0.16 with other groups) groups appeared exceptions to this pattern, perhaps because of the aforementioned ascertainment bias or previously unrecognized population substructure within these groups (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1/FigureS4.pdf">Figure S4</a>).</p><p id="p-32">Across all genetic groups, levels of population differentiation were slightly lower for predicted deleterious (mean F<sub>ST</sub> = 0.07) than nondeleterious (mean F<sub>ST</sub> = 0.08) SNPs (Mann-Whitney <em>U</em>-test <em>P</em> &lt; 10<sup>−15</sup>; <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). After correcting for allele frequencies in both classes, however, these differences disappeared and the proportion of deleterious SNPs in the top 1% of F<sub>ST</sub> was not significantly different from the proportion observed for synonymous SNPs (Fisher’s exact test <em>P</em> = 0.94) or all SNPs in genic regions (Fisher’s exact test <em>P</em> = 0.51). After allele frequency correction, 287 genes had a predicted deleterious SNP in the top 1% of Fst among genetic groups, and 30 genes had two or more high-Fst predicted deleterious SNPs (see <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> for chromosome 1). Only 11 genes (4%) with high-Fst deleterious SNPs are found in regions thought to be selected during maize improvement (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Hufford <em>et al.</em> 2012</a>) and only 44 of the 287 genes (15%) show significant signs of positive selection with the PHS statistic. Neither result provides much evidence that selection on linked beneficial mutations strongly impacts frequencies of deleterious alleles.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/163/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="FST distribution for deleterious and nondeleterious SNPs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-521486900" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;F&lt;sub&gt;ST&lt;/sub&gt; distribution for deleterious and nondeleterious SNPs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/163/F3.medium.gif" width="440" height="347"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/1/163/F3.medium.gif" width="440" height="347"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/163/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/163/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463834" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">F<sub>ST</sub> distribution for deleterious and nondeleterious SNPs.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/163/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of FST along chromosome 1. Black dots represent SNPs in the top 1% of FST and those predicted to be deleterious are surrounded in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-521486900" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of F&lt;sub&gt;ST&lt;/sub&gt; along chromosome 1. Black dots represent SNPs in the top 1% of F&lt;sub&gt;ST&lt;/sub&gt; and those predicted to be deleterious are surrounded in red.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/163/F4.medium.gif" width="440" height="186"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/1/163/F4.medium.gif" width="440" height="186"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/163/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/163/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463844" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Distribution of F<sub>ST</sub> along chromosome 1. Black dots represent SNPs in the top 1% of F<sub>ST</sub> and those predicted to be deleterious are surrounded in red.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">Comparisons of the predicted deleterious SFS between stiff-stalk, non-stiff stalk, and tropical groups (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>) mirrored patterns of between-group F<sub>ST</sub>, revealing few fixed differences between groups and generally low frequencies within groups, as well as higher differentiation in comparisons involving the stiff-stalk group.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/163/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Joint site frequency spectrum of stiff-stalk, non-stiff stalk, and tropical genetic groups. Axes represent the frequency of the predicted deleterious alleles in a group and colors show the proportion of SNPs at a given frequency." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-521486900" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Joint site frequency spectrum of stiff-stalk, non-stiff stalk, and tropical genetic groups. Axes represent the frequency of the predicted deleterious alleles in a group and colors show the proportion of SNPs at a given frequency.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/163/F5.medium.gif" width="440" height="369"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/1/163/F5.medium.gif" width="440" height="369"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/163/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/163/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463871" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-36" class="first-child">Joint site frequency spectrum of stiff-stalk, non-stiff stalk, and tropical genetic groups. Axes represent the frequency of the predicted deleterious alleles in a group and colors show the proportion of SNPs at a given frequency.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Observed frequencies of deleterious SNPs in different populations (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>) may help explain patterns of hybrid vigor. Although F<sub>ST</sub> is generally low, inbreds from different genetic groups are nonetheless likely to share fewer deleterious variants than inbreds from the same group, and heterosis is higher among crosses between groups (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1/FigureS5.pdf">Figure S5</a>). Nonetheless, even crosses among inbreds from the same genetic group show evidence of heterosis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1/FigureS5.pdf">Figure S5</a>), likely due to the large number of deleterious SNPs segregating at low frequencies within individual populations.</p></div><div id="sec-11" class="subsection"><h3>Effect of deleterious mutations on traits of interest</h3><p id="p-38">To investigate the contribution of predicted deleterious alleles to observed levels of heterosis and inbreeding depression, we performed a genome-wide association analysis of 17 traits evaluated in two populations (see the section <em>Materials and Methods</em>). Analyses were carried out using the genetic values of inbred lines and both mid-parent and best-parent heterosis. Genome-wide association results using the genetic values of inbred lines identified between 219 (cob diameter) and 598 (cob length) significant SNPs with a high proportion of genic loci (up to 70%) but little evidence for significant enrichment of predicted deleterious SNPs (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1/TableS4.pdf">Table S4</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463842/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463842/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Total number of significant SNPs in genic regions (<em>n</em>) and fold enrichment (<em>f</em>) for deleterious SNPs in population A</span><div class="sb-div caption-clear"></div></div></div><p id="p-41">Results for association between SNP heterozygosity and heterosis showed highly variable numbers of significant loci (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1/TableS4.pdf">Table S4</a>), also with a high proportion of genic SNPs (up to 74%). Significant loci explained between 4 and 40% of the observed phenotypic variation in heterosis, although these values are likely inflated due to small sample size (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Beavis 1994</a>). The greatest numbers of associated SNPs were observed for plant height and yield-related traits, which also showed the greatest levels of observed heterosis. Furthermore, most traits exhibited some enrichment (5−45%) of predicted deleterious SNPs and the enrichment was statistically significant for whole plant yield and days to tasseling. These enrichment results hold even after an FDR control at 20% and similar enrichments were observed when comparing nonsynonymous deleterious with nonsynonymous nondeleterious SNPs (data not shown). Although crosses to both Mo17 and B73 showed evidence of enrichment in population B, only crosses to B73 were statistically significant, likely due to the lower number of deleterious SNPs identified in the stiff-stalk heterotic group.</p><p id="p-42">Because most deleterious SNPs are at frequencies too low for inclusion in association analyses (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), we expanded our test of enrichment to the gene level, asking whether genes with predicted deleterious SNPs were more likely than random to have SNPs significantly associated with traits of interest. At this level we see much stronger evidence of enrichment, even with an FDR control at 20%: a number of traits show statistically significant enrichment in population A, but virtually all traits in both populations show a positive enrichment for genes with predicted deleterious SNPs (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1/TableS5.pdf">Table S5</a>), a result that is highly unlikely by chance (sign test <em>P</em> = 3 × 10<sup>−5</sup> for population A and 0.01 for population B). Similar tests of low-frequency synonymous SNPs show no evidence of enrichment (<em>P</em> ≈ 1), and the low correlation between total SNPs in a gene and the number of significant associations (<em>r</em> ≤ 0.2) suggests that our observation is not an artifact of the number of SNPs analyzed per gene. Furthermore, the enrichment result holds for groups of genes with similar numbers of SNPs.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463854/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463854/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Total number of genes with significant SNPs (<em>n</em>) and fold enrichment for genes with predicted deleterious SNPs (<em>f</em>) in population A</span><div class="sb-div caption-clear"></div></div></div><p id="p-45">We posit that the observed excess of significant associations in genes with predicted deleterious variants may be due to so-called synthetic associations between rare deleterious alleles and a common allele at a linked locus at high enough frequency to be included in association mapping tests (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Goldstein 2009</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dickson <em>et al.</em> 2010</a>). Recent work suggests that this sort of association is only likely to hold for deleterious alleles with a relatively small effect on phenotype (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Thornton <em>et al.</em> 2013</a>), which is consistent with the expected weak-to-intermediate effects of deleterious alleles likely to be involved in heterosis (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Charlesworth and Charlesworth 1987</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Whitlock <em>et al.</em> 2000</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Glémin <em>et al.</em> 2003</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Charlesworth and Willis 2009</a>). Strongly deleterious alleles, although potentially playing a role in inbreeding depression (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Whitlock <em>et al.</em> 2000</a>), are less likely to be observed in our study as selection should effectively remove them from our panel of inbred lines.</p><p id="p-46">Although we have analyzed only a relatively small subset of the genome-wide diversity of maize (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Chia <em>et al.</em> 2012</a>), our data nonetheless present the first genome-wide scan of deleterious coding variants in maize. Our results provide evidence for the contribution of deleterious mutations to heterosis via complementation, consistent with the dominance hypothesis. The weak expected effects of these deleterious SNPs, combined with their low frequencies, make their detection difficult using conventional approaches. <em>A priori</em> prediction of the potential effect of rare polymorphisms, however, may improve predictions of inbred line breeding values and combining ability. Future analysis of full genome sequence data, allowing for the inclusion of all coding SNPs and noncoding variants, will provide an even richer catalog of variants that will expand our understanding of the role of rare deleterious variants in maize breeding.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-47">We thank S. Flint-Garcia and S. Takuno for help with data analysis; E. S. Buckler for early access to the genotyping data; C. Romay and J. Glaubitz for bioinformatics support; G. Coop, J. Gerke, P. Morrell, P. Ralph, and O. Smith for helpful comments on an earlier version of the manuscript; and two anonymous reviewers for their constructive comments. This project was supported by Agriculture and Food Research Initiative Competitive Grant 2009-01864 from the USDA National Institute of Food and Agriculture as well as a grant from DuPont Pioneer.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008870/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">SNP data are available at <a href="http://www.panzea.org/dynamic/derivative_data/genotypes/Maize282_GBS_genos_imputed_20120110.zip">http://www.panzea.org/dynamic/derivative_data/genotypes/Maize282_GBS_genos_imputed_20120110.zip</a>.</p></li><li class="fn" id="fn-3"><p id="p-3">Phenotypic data are available at <a href="http://www.plosone.org/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0007433.s001">http://www.plosone.org/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0007433.s001</a> and <a href="http://www.plosone.org/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0007433.s002">http://www.plosone.org/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0007433.s002</a></p></li><li class="fn" id="fn-8"><p id="p-48"><em>Communicating editor: K. Dawe</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-10-07"><span class="received-label">Received </span>October 7, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-11-19"><span class="accepted-label">Accepted </span>November 19, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Mezmouk and Ross-Ibarra</li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-4.1.163.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite> <span class="cit-pub-date">1994</span> <span class="cit-article-title">The power and deceit of QTL experiments: lessons from comparative QTL studies</span>, pp. <span class="cit-fpage">250</span>–<span class="cit-lpage">266</span>, in <span class="cit-source">Proceedings of the Forty-Ninth Annual Corn and Sorghum Research Conference</span>. <span class="cit-publ-name">American Seed Trade Association</span>, <span class="cit-publ-loc">Washington, DC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.3" data-doi="10.1093/nar/gkg095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boeckmann</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bairoch</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apweiler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blatter</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estreicher</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The swiss-prot protein knowledgebase and its supplement trembl in 2003.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">365</span>–<span class="cit-lpage">370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBoeckmann%26rft.auinit1%253DB.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D365%26rft.epage%253D370%26rft.atitle%253DThe%2BSWISS-PROT%2Bprotein%2Bknowledgebase%2Band%2Bits%2Bsupplement%2BTrEMBL%2Bin%2B2003%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg095%26rft_id%253Dinfo%253Apmid%252F12520024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjMxLzEvMzY1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8xLzE2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.4" data-doi="10.1371/journal.pgen.1003754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brandvain</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slotte</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hazzouri</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic identification of founding haplotypes reveals the history of the selfing species capsella rubella.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003754%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003754%26rft_id%253Dinfo%253Apmid%252F24068948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24068948&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.5" data-doi="10.1038/ng.911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Günther</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bender</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Whole-genome sequencing of multiple arabidopsis thaliana populations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">956</span>–<span class="cit-lpage">963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCao%26rft.auinit1%253DJ.%26rft.volume%253D43%26rft.issue%253D10%26rft.spage%253D956%26rft.epage%253D963%26rft.atitle%253DWhole-genome%2Bsequencing%2Bof%2Bmultiple%2BArabidopsis%2Bthaliana%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.911%26rft_id%253Dinfo%253Apmid%252F21874002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21874002&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.6" data-doi="10.1146/annurev.es.18.110187.001321"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Inbreeding depression and its evolutionary consequences.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Syst.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%2Band%2BSystematics%26rft.stitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft.aulast%253DCharlesworth%26rft.auinit1%253DD%26rft.volume%253D18%26rft.issue%253D1%26rft.spage%253D237%26rft.epage%253D268%26rft.atitle%253DInbreeding%2BDepression%2BAnd%2BIts%2BEvolutionary%2BConsequences%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.es.18.110187.001321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.es.18.110187.001321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1987K958800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.7" data-doi="10.1038/nrg2664"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetics of inbreeding depression.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">783</span>–<span class="cit-lpage">796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCharlesworth%26rft.auinit1%253DD.%26rft.volume%253D10%26rft.issue%253D11%26rft.spage%253D783%26rft.epage%253D796%26rft.atitle%253DThe%2Bgenetics%2Bof%2Binbreeding%2Bdepression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2664%26rft_id%253Dinfo%253Apmid%252F19834483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2664&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19834483&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270915400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Mutation accumulation in finite populations.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DCharlesworth%26rft.auinit1%253DD.%26rft.volume%253D84%26rft.issue%253D5%26rft.spage%253D321%26rft.epage%253D325%26rft.atitle%253DMutation%2BAccumulation%2Bin%2BFinite%2BPopulations%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6Ijg0LzUvMzIxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8xLzE2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.9" data-doi="10.1038/ng.2313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Leon</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Maize hapmap2 identifies extant variation from a genome in flux.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChia%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D803%26rft.epage%253D807%26rft.atitle%253DMaize%2BHapMap2%2Bidentifies%2Bextant%2Bvariation%2Bfrom%2Ba%2Bgenome%2Bin%2Bflux.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2313%26rft_id%253Dinfo%253Apmid%252F22660545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660545&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.10" data-doi="10.1371/journal.pgen.1002240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chun</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evidence for hitchhiking of deleterious mutations within the human genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DChun%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253De1002240%26rft.epage%253De1002240%26rft.atitle%253DEvidence%2Bfor%2Bhitchhiking%2Bof%2Bdeleterious%2Bmutations%2Bwithin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002240%26rft_id%253Dinfo%253Apmid%252F21901107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21901107&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.11" data-doi="10.1126/science.1099870"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiss</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertsemlidis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcel</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McPherson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Multiple rare alleles contribute to low plasma levels of HDL cholesterol.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">305</span>: <span class="cit-fpage">869</span>–<span class="cit-lpage">872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCohen%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D305%26rft.issue%253D5685%26rft.spage%253D869%26rft.epage%253D872%26rft.atitle%253DMultiple%2BRare%2BAlleles%2BContribute%2Bto%2BLow%2BPlasma%2BLevels%2Bof%2BHDL%2BCholesterol%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1099870%26rft_id%253Dinfo%253Apmid%252F15297675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDUvNTY4NS84NjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzEvMTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.12" data-doi="10.1104/pp.111.185033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic architecture of maize kernel composition in the nested association mapping and inbred association panels.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">824</span>–<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.185033%26rft_id%253Dinfo%253Apmid%252F22135431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTgvMi84MjQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzEvMTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.13" data-doi="10.1073/pnas.95.10.5637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cummings</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clegg</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Nucleotide sequence diversity at the alcohol dehydrogenase 1 locus in wild barley (hordeum vulgare ssp. spontaneum): an evaluation of the background selection hypothesis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">5637</span>–<span class="cit-lpage">5642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DCummings%26rft.auinit1%253DM.%2BP.%26rft.volume%253D95%26rft.issue%253D10%26rft.spage%253D5637%26rft.epage%253D5642%26rft.atitle%253DNucleotide%2Bsequence%2Bdiversity%2Bat%2Bthe%2Balcohol%2Bdehydrogenase%2B1%2Blocus%2Bin%2Bwild%2Bbarley%2B%2528Hordeum%2Bvulgare%2Bssp.%2Bspontaneum%2529%253A%2BAn%2Bevaluation%2Bof%2Bthe%2Bbackground%2Bselection%2Bhypothesis%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.10.5637%26rft_id%253Dinfo%253Apmid%252F9576936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTUvMTAvNTYzNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMS8xNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.14" data-doi="10.1371/journal.pbio.1000294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dickson</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krantz</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hakonarson</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rare variants create synthetic genome-wide associations.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1000294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDickson%26rft.auinit1%253DS.%2BP.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253De1000294%26rft.epage%253De1000294%26rft.atitle%253DRare%2Bvariants%2Bcreate%2Bsynthetic%2Bgenome-wide%2Bassociations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000294%26rft_id%253Dinfo%253Apmid%252F20126254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20126254&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-4.1.163.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">East</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1907–1908</span> <span class="cit-source">Reports of the Connecticut Agricultural Experiment Station for Years, Volume Inbreeding in Corn</span>. <span class="cit-publ-name">Connecticut Agricultural Experiment Station, New Haven, Connecticut Agricultural Experiment Station</span>, <span class="cit-publ-loc">New Haven, CT</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.16" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (gbs) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wyckoff</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Positive and negative selection on the human genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">1227</span>–<span class="cit-lpage">1234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFay%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D158%26rft.issue%253D3%26rft.spage%253D1227%26rft.epage%253D1234%26rft.atitle%253DPositive%2Band%2BNegative%2BSelection%2Bon%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Apmid%252F11454770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1OC8zLzEyMjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzEvMTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The evolutionary advantage of recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ.%26rft.volume%253D78%26rft.issue%253D2%26rft.spage%253D737%26rft.epage%253D756%26rft.atitle%253DTHE%2BEVOLUTIONARY%2BADVANTAGE%2BOF%2BRECOMBINATION%26rft_id%253Dinfo%253Apmid%252F4448362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzgvMi83MzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzEvMTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-4.1.163.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1930</span> <span class="cit-source">The Genetical Theory of Natural Selection</span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.20" data-doi="10.1111/j.1365-313X.2005.02591.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint-Garcia</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thuillet</span>  <span class="cit-name-given-names">A.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Maize association population: a high-resolution platform for quantitative trait locus dissection.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1054</span>–<span class="cit-lpage">1064</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DFlint-Garcia%26rft.auinit1%253DS.%2BA.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D1054%26rft.epage%253D1064%26rft.atitle%253DMaize%2Bassociation%2Bpopulation%253A%2Ba%2Bhigh-resolution%2Bplatform%2Bfor%2Bquantitative%2Btrait%2Blocus%2Bdissection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2005.02591.x%26rft_id%253Dinfo%253Apmid%252F16359397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2005.02591.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16359397&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233776900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.21" data-doi="10.1371/journal.pone.0007433"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint-Garcia</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiffin</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">N. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Heterosis is prevalent for multiple traits in diverse maize germplasm.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e7433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFlint-Garcia%26rft.auinit1%253DS.%2BA.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De7433%26rft.epage%253De7433%26rft.atitle%253DHeterosis%2Bis%2Bprevalent%2Bfor%2Bmultiple%2Btraits%2Bin%2Bdiverse%2Bmaize%2Bgermplasm.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0007433%26rft_id%253Dinfo%253Apmid%252F19823591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0007433&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19823591&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.22" data-doi="10.1126/science.1209271"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fournier-Level</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmitt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A map of local adaptation in arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">334</span>: <span class="cit-fpage">86</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFournier-Level%26rft.auinit1%253DA.%26rft.volume%253D334%26rft.issue%253D6052%26rft.spage%253D86%26rft.epage%253D89%26rft.atitle%253DA%2BMap%2Bof%2BLocal%2BAdaptation%2Bin%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1209271%26rft_id%253Dinfo%253Apmid%252F21980109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzQvNjA1Mi84NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMS8xNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-web" id="cit-4.1.163.23"><div class="cit-metadata unstructured">Gerke, J. P., J. W. Edwards, K. E. Guill, J. Ross-Ibarra, and M. D. McMullen, 2013 The genomic impacts of drift and selection for hybrid performance in maize. Available at: <a href="http://arxiv.org/abs/1307.7313">http://arxiv.org/abs/1307.7313</a>. Accessed December 4, 2012.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.24" data-doi="10.1038/nrg3118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Rare and common variants: twenty arguments.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D13%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3118%26rft_id%253Dinfo%253Apmid%252F22251874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22251874&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glémin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronfort</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bataillon</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Patterns of inbreeding depression and architecture of the load in subdivided populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">2193</span>–<span class="cit-lpage">2212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGlemin%26rft.auinit1%253DS.%26rft.volume%253D165%26rft.issue%253D4%26rft.spage%253D2193%26rft.epage%253D2212%26rft.atitle%253DPatterns%2Bof%2BInbreeding%2BDepression%2Band%2BArchitecture%2Bof%2Bthe%2BLoad%2Bin%2BSubdivided%2BPopulations%26rft_id%253Dinfo%253Apmid%252F14704197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS80LzIxOTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzEvMTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.26" data-doi="10.1056/NEJMp0806284"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Common genetic variation and human traits.</span> <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">360</span>: <span class="cit-fpage">1696</span>–<span class="cit-lpage">1698</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMp0806284%26rft_id%253Dinfo%253Apmid%252F19369660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMp0806284&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19369660&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265377800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.27" data-doi="10.1126/science.1177837"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A first-generation haplotype map of maize.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">1115</span>–<span class="cit-lpage">1117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGore%26rft.auinit1%253DM.%2BA.%26rft.volume%253D326%26rft.issue%253D5956%26rft.spage%253D1115%26rft.epage%253D1117%26rft.atitle%253DA%2BFirst-Generation%2BHaplotype%2BMap%2Bof%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1177837%26rft_id%253Dinfo%253Apmid%252F19965431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk1Ni8xMTE1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8xLzE2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.28" data-doi="10.1093/molbev/msq079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gossmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Windsor</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell-Olds</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome wide analyses reveal little evidence for adaptive evolution in many plant species.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1822</span>–<span class="cit-lpage">1832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq079%26rft_id%253Dinfo%253Apmid%252F20299543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI3LzgvMTgyMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMS8xNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.29" data-doi="10.1007/s00122-010-1299-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Günther</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Deleterious amino acid polymorphisms in arabidopsis thaliana and rice.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGunther%26rft.auinit1%253DT.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D157%26rft.epage%253D168%26rft.atitle%253DDeleterious%2Bamino%2Bacid%2Bpolymorphisms%2Bin%2BArabidopsis%2Bthaliana%2Band%2Brice.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1299-4%26rft_id%253Dinfo%253Apmid%252F20198468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1299-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20198468&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.30" data-doi="10.1186/gb-2007-8-2-r18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haddrill</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halligan</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomaras</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Reduced efficacy of selection in regions of the drosophila genome that lack crossing over.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">R18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHaddrill%26rft.auinit1%253DP.%2BR.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253DR18%26rft.epage%253DR18%26rft.atitle%253DReduced%2Befficacy%2Bof%2Bselection%2Bin%2Bregions%2Bof%2Bthe%2BDrosophila%2Bgenome%2Bthat%2Black%2Bcrossing%2Bover.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-2-r18%26rft_id%253Dinfo%253Apmid%252F17284312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-2-r18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17284312&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.31" data-doi="10.1017/S0016672300010156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">The effect of linkage on limits to artificial selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D269%26rft.epage%253D294%26rft.atitle%253DThe%2Beffect%2Bof%2Blinkage%2Bon%2Blimits%2Bto%2Bartificial%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300010156%26rft_id%253Dinfo%253Apmid%252F5980116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300010156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5980116&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19668750300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.32" data-doi="10.1038/ng.2309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hufford</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Heerwaarden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pyhäjärvi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparative population genomics of maize domestication and improvement.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">808</span>–<span class="cit-lpage">811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHufford%26rft.auinit1%253DM.%2BB.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D808%26rft.epage%253D811%26rft.atitle%253DComparative%2Bpopulation%2Bgenomics%2Bof%2Bmaize%2Bdomestication%2Band%2Bimprovement.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2309%26rft_id%253Dinfo%253Apmid%252F22660546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2309&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660546&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.33" data-doi="10.1534/genetics.104.036939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evidence for abundant slightly deleterious polymorphisms in bacterial populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">533</span>–<span class="cit-lpage">538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.036939%26rft_id%253Dinfo%253Apmid%252F15545641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY5LzIvNTMzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8xLzE2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.34" data-doi="10.1534/genetics.104.033761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Spontaneous mutations in diploid <em>Saccharomyces cerevisiae</em>: more beneficial than expected.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">1817</span>–<span class="cit-lpage">1825</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DJoseph%26rft.auinit1%253DS.%2BB.%26rft.volume%253D168%26rft.issue%253D4%26rft.spage%253D1817%26rft.epage%253D1825%26rft.atitle%253DSpontaneous%2BMutations%2Bin%2BDiploid%2BSaccharomyces%2Bcerevisiae%253A%2BMore%2BBeneficial%2BThan%2BExpected%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.033761%26rft_id%253Dinfo%253Apmid%252F15611159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OC80LzE4MTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzEvMTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.35" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient control of population structure in model organism association mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzEvMTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-book" id="cit-4.1.163.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-source">The Neutral Theory of Molecular Evolution.</span> <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.37" data-doi="10.2307/2410240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lande</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Risk of population extinction from fixation of new deleterious mutations.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1460</span>–<span class="cit-lpage">1469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D48%26rft.spage%253D1460%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2410240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2410240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994QQ65900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.38" data-doi="10.1093/bioinformatics/btm404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larkin</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackshields</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chenna</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGettigan</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Clustal w and clustal x version 2.0.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2947</span>–<span class="cit-lpage">2948</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm404%26rft_id%253Dinfo%253Apmid%252F17846036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMjEvMjk0NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMS8xNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.39" data-doi="10.1371/journal.pgen.1003246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larsson</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Lessons from dwarf8 on the strengths and weaknesses of structured association mapping.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003246%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003246%26rft_id%253Dinfo%253Apmid%252F23437002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003246&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23437002&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-other" id="cit-4.1.163.40"><div class="cit-metadata unstructured">Lohmueller, K. E. 2013 The impact of population demography and selection on genetic architecture of complex traits. Available at: <a href="http://arxiv.org/abs/1306.5261.htm">http://arxiv.org/abs/1306.5261</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.41" data-doi="10.1038/nature06611"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Indap</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Proportionally more deleterious genetic variation in European than in African populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">451</span>: <span class="cit-fpage">994</span>–<span class="cit-lpage">997</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLohmueller%26rft.auinit1%253DK.%2BE.%26rft.volume%253D451%26rft.issue%253D7181%26rft.spage%253D994%26rft.epage%253D997%26rft.atitle%253DProportionally%2Bmore%2Bdeleterious%2Bgenetic%2Bvariation%2Bin%2BEuropean%2Bthan%2Bin%2BAfrican%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06611%26rft_id%253Dinfo%253Apmid%252F18288194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06611&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18288194&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253313100049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.42" data-doi="10.1016/j.tig.2006.01.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The accumulation of deleterious mutations in rice genomes: a hypothesis on the cost of domestication.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">126</span>–<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DLu%26rft.auinit1%253DJ.%26rft.volume%253D22%26rft.issue%253D3%26rft.spage%253D126%26rft.epage%253D131%26rft.atitle%253DThe%2Baccumulation%2Bof%2Bdeleterious%2Bmutations%2Bin%2Brice%2Bgenomes%253A%2Ba%2Bhypothesis%2Bon%2Bthe%2Bcost%2Bof%2Bdomestication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2006.01.004%26rft_id%253Dinfo%253Apmid%252F16443304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2006.01.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16443304&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236534100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.43" data-doi="10.2307/2409502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabriel</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Mutation load and the survival of small populations.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1725</span>–<span class="cit-lpage">1737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D44%26rft.spage%253D1725%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2409502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2409502&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1990EP84500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.44" data-doi="10.1126/science.1174320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villeda</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic properties of the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMcMullen%26rft.auinit1%253DM.%2BD.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D737%26rft.epage%253D740%26rft.atitle%253DGenetic%2BProperties%2Bof%2Bthe%2BMaize%2BNested%2BAssociation%2BMapping%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174320%26rft_id%253Dinfo%253Apmid%252F19661427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzEvMTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Messmer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodman</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamkey</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Genetic diversity among progenitors and elite lines from the Iowa Stiff Stalk Synthetic (Bsss) maize population: comparison of allozyme and RFLP data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D38%26rft.spage%253D97%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.46" data-doi="10.1093/nar/gkg509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">SIFT: predicting amino acid changes that affect protein function.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3812</span>–<span class="cit-lpage">3814</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DNg%26rft.auinit1%253DP.%2BC.%26rft.volume%253D31%26rft.issue%253D13%26rft.spage%253D3812%26rft.epage%253D3814%26rft.atitle%253DSIFT%253A%2Bpredicting%2Bamino%2Bacid%2Bchanges%2Bthat%2Baffect%2Bprotein%2Bfunction%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg509%26rft_id%253Dinfo%253Apmid%252F12824425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8xMy8zODEyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8xLzE2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.47" data-doi="10.1146/annurev.genom.7.080505.115630"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Predicting the effects of amino acid substitutions on protein function.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">61</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DNg%26rft.auinit1%253DP.%2BC.%26rft.volume%253D7%26rft.spage%253D61%26rft.epage%253D80%26rft.atitle%253DPredicting%2Bthe%2Beffects%2Bof%2Bamino%2Bacid%2Bsubstitutions%2Bon%2Bprotein%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.7.080505.115630%26rft_id%253Dinfo%253Apmid%252F16824020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.7.080505.115630&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16824020&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242032200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.48" data-doi="10.1038/246096a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohta</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Slightly deleterious mutant substitutions in evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">246</span>: <span class="cit-fpage">96</span>–<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DOhta%26rft.auinit1%253DT.%26rft.volume%253D246%26rft.issue%253D5428%26rft.spage%253D96%26rft.epage%253D98%26rft.atitle%253DSlightly%2Bdeleterious%2Bmutant%2Bsubstitutions%2Bin%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F246096a0%26rft_id%253Dinfo%253Apmid%252F4585855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/246096a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4585855&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1973R229300031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.49" data-doi="10.1111/mec.12329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paape</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bataillon</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kono T</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briskine</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Selection, genome-wide fitness effects and evolutionary rates in the model legume Medicago truncatula.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3525</span>–<span class="cit-lpage">3538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3525%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000320942000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.50" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8xLzE2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.51" data-doi="10.1186/gb-2013-14-6-r55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comprehensive genotyping of the USA national maize inbred seed bank.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR55%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-6-r55%26rft_id%253Dinfo%253Apmid%252F23759205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-6-r55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23759205&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.52" data-doi="10.1126/science.1178534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ware</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The b73 maize genome: complexity, diversity, and dynamics.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">1112</span>–<span class="cit-lpage">1115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSchnable%26rft.auinit1%253DP.%2BS.%26rft.volume%253D326%26rft.issue%253D5956%26rft.spage%253D1112%26rft.epage%253D1115%26rft.atitle%253DThe%2BB73%2BMaize%2BGenome%253A%2BComplexity%252C%2BDiversity%252C%2Band%2BDynamics%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1178534%26rft_id%253Dinfo%253Apmid%252F19965430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk1Ni8xMTEyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8xLzE2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.53" data-doi="10.1093/gbe/evs009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeling</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyons</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide analysis of syntenic gene deletion in the grasses.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevs009%26rft_id%253Dinfo%253Apmid%252F22275519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjQvMy8yNjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzEvMTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shull</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1908</span> <span class="cit-article-title">The composition of a field of maize.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">296</span>–<span class="cit-lpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft.volume%253D4%26rft.spage%253D296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-web" id="cit-4.1.163.55"><div class="cit-metadata unstructured">Simons, Y. B., M. C. Turchin, and J. K. Pritchard, 2013 The deleterious mutation load is sensitive to recent population history. Available at: <a href="http://arxiv.org/abs/1305.2061">http://arxiv.org/abs/1305.2061</a>. Accessed December 4, 2013.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.56" data-doi="10.1016/j.neurobiolaging.2004.02.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smigrodzki</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parks</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">High frequency of mitochondrial complex i mutations in Parkinson’s disease and aging.</span> <abbr class="cit-jnl-abbrev">Neurobiol. Aging</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1273</span>–<span class="cit-lpage">1281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeurobiology%2Bof%2Baging%26rft.stitle%253DNeurobiol%2BAging%26rft.aulast%253DSmigrodzki%26rft.auinit1%253DR.%26rft.volume%253D25%26rft.issue%253D10%26rft.spage%253D1273%26rft.epage%253D1281%26rft.atitle%253DHigh%2Bfrequency%2Bof%2Bmitochondrial%2Bcomplex%2BI%2Bmutations%2Bin%2BParkinson%2527s%2Bdisease%2Band%2Baging.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neurobiolaging.2004.02.020%26rft_id%253Dinfo%253Apmid%252F15465623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neurobiolaging.2004.02.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15465623&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224659600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.57" data-doi="10.1101/gr.3804205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sidow</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Physicochemical constraint violation by missense substitutions mediates impairment of protein function and disease severity.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">978</span>–<span class="cit-lpage">986</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3804205%26rft_id%253Dinfo%253Apmid%252F15965030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE1LzcvOTc4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8xLzE2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.58" data-doi="10.1101/pdb.emo132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strable</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scanlon</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Maize (zea mays): a model organism for basic and applied research in plant biology.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Protoc.</abbr> <span class="cit-vol">2009</span>:<span class="cit-fpage">pdb.emo132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BProtocols%26rft.stitle%253DCold%2BSpring%2BHarb%2BProtoc%26rft.aulast%253DStrable%26rft.auinit1%253DJ.%26rft.volume%253D2009%26rft.issue%253D10%26rft.spage%253Dpdb.emo132%26rft.epage%253Dpdb.emo132%26rft.atitle%253DMaize%2B%2528Zea%2Bmays%2529%253A%2BA%2BModel%2BOrganism%2Bfor%2BBasic%2Band%2BApplied%2BResearch%2Bin%2BPlant%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fpdb.emo132%26rft_id%253Dinfo%253Apmid%252F20147033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicHJvdG9jb2xzIjtzOjU6InJlc2lkIjtzOjE4OiIyMDA5LzEwL3BkYi5lbW8xMzIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzEvMTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.59" data-doi="10.1038/hdy.2010.175"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tellier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merino</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camus-Kulandaivelu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fitness effects of derived deleterious mutations in four closely related wild tomato species with spatial structure.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb.)</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">189</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb.%2529%26rft.volume%253D107%26rft.spage%253D189%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.175%26rft_id%253Dinfo%253Apmid%252F21245893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.175&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21245893&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.60" data-doi="10.1093/bioinformatics/btg316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Libsequence: a c++ class library for evolutionary genetic analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2325</span>–<span class="cit-lpage">2327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DThornton%26rft.auinit1%253DK.%26rft.volume%253D19%26rft.issue%253D17%26rft.spage%253D2325%26rft.epage%253D2327%26rft.atitle%253Dlibsequence%253A%2Ba%2BC%252B%252B%2Bclass%2Blibrary%2Bfor%2Bevolutionary%2Bgenetic%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg316%26rft_id%253Dinfo%253Apmid%252F14630667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMTkvMTcvMjMyNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMS8xNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.61" data-doi="10.1371/journal.pgen.1003258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foran</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Properties and modeling of GWAS when complex disease risk is due to non-complementing, deleterious mutations in genes of large effect.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003258%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003258%26rft_id%253Dinfo%253Apmid%252F23437004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23437004&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.62" data-doi="10.1371/journal.pbio.0040137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toomajian</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aranzana</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A nonparametric test reveals selection for rapid flowering in the Arabidopsis genome.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DToomajian%26rft.auinit1%253DC.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253De137%26rft.epage%253De137%26rft.atitle%253DA%2Bnonparametric%2Btest%2Breveals%2Bselection%2Bfor%2Brapid%2Bflowering%2Bin%2Bthe%2BArabidopsis%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040137%26rft_id%253Dinfo%253Apmid%252F16623598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16623598&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.63" data-doi="10.1017/S0016672399003900"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Dynamics of inbreeding depression due to deleterious mutations in small populations: mutation parameters and inbreeding rate.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D74%26rft.issue%253D2%26rft.spage%253D165%26rft.epage%253D178%26rft.atitle%253DDynamics%2Bof%2Binbreeding%2Bdepression%2Bdue%2Bto%2Bdeleterious%2Bmutations%2Bin%2Bsmall%2Bpopulations%253A%2Bmutation%2Bparameters%2Band%2Binbreeding%2Brate.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672399003900%26rft_id%253Dinfo%253Apmid%252F10584559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672399003900&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10584559&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083662500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.64" data-doi="10.1046/j.1365-2540.2000.00693.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hatfield</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Local drift load and the heterosis of interconnected populations.</span> <abbr class="cit-jnl-abbrev">Heridity</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">452</span>–<span class="cit-lpage">457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeridity%26rft.volume%253D84%26rft.spage%253D452%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2540.2000.00693.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2540.2000.00693.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grisworld</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Compensating for meltdown: the critical effective size of a population with deleterious and compensatory mutations.</span> <abbr class="cit-jnl-abbrev">Ann. Zool. Fenn.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BZool.%2BFenn.%26rft.volume%253D40%26rft.spage%253D169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000183295600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.66" data-doi="10.1038/ng1702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vroh Bi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.163.67" data-doi="10.1158/0008-5472.CAN-03-2800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spitz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schabath</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">An evolutionary perspective on single-nucleotide polymorphism screening in molecular cancer epidemiology.</span> <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">2251</span>–<span class="cit-lpage">2257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DZhu%26rft.auinit1%253DY.%26rft.volume%253D64%26rft.issue%253D6%26rft.spage%253D2251%26rft.epage%253D2257%26rft.atitle%253DAn%2BEvolutionary%2BPerspective%2Bon%2BSingle-Nucleotide%2BPolymorphism%2BScreening%2Bin%2BMolecular%2BCancer%2BEpidemiology%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-03-2800%26rft_id%253Dinfo%253Apmid%252F15026370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjk6IjY0LzYvMjI1MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMS8xNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/1/163.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/1/155" title="Generation of a Useful roX1 Allele by Targeted Gene Conversion" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/1/173" title="par-1, Atypical pkc, and PP2A/B55 sur-6 Are Implicated in the Regulation of Exocyst-Mediated Membrane Trafficking in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/1"><span>Volume 4 Issue 1, January 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/1.cover-source.jpg?itok=oIsy65sY" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/1/163" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mpOzcdjmTwu4LmZvu2CtILNv8efVCXUFYNCwfY9g0YU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/1/163" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/1/163" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/1/163" class="active" data-icon-position="" data-hide-link-title="0">The Pattern and Distribution of Deleterious Mutations in Maize</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459622" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-qAwxsT8_BAP5yeECAi42vgtjca1YHv0ZVTJOpK3N61g" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/1/163.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/1/163" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-bMo0V2jq52NSSuy1OMk3C5Dmw9nfbEzu2xjfr16NRIY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/1/163" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/1/163.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459622" id="node459622--2" data-pisa="ggg;4/1/163" data-pisa-master="ggg;g3.113.008870" data-apath="/ggg/4/1/163.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Pattern and Distribution of Deleterious Mutations in Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sofiane Mezmouk</span> and <span class="highwire-citation-author" data-delta="1">Jeffrey Ross-Ibarra</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">163-171;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008870 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459622--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sofiane Mezmouk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California–Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sofiane%2BMezmouk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mezmouk%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASofiane%2BMezmouk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jeffrey Ross-Ibarra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California–Davis, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Population Biology and Genome Center, University of California–Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BRoss-Ibarra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ross-Ibarra%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BRoss-Ibarra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rossibarra@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">rossibarra@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/1/163" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/1/163&amp;title=The%20Pattern%20and%20Distribution%20of%20Deleterious%20Mutations%20in%20Maize" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/1/163&amp;title=The%20Pattern%20and%20Distribution%20of%20Deleterious%20Mutations%20in%20Maize" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/1/163&amp;title=The%20Pattern%20and%20Distribution%20of%20Deleterious%20Mutations%20in%20Maize" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/1/163&amp;text=The%20Pattern%20and%20Distribution%20of%20Deleterious%20Mutations%20in%20Maize" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/1/163&amp;title=The%20Pattern%20and%20Distribution%20of%20Deleterious%20Mutations%20in%20Maize" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/1/163&amp;t=The%20Pattern%20and%20Distribution%20of%20Deleterious%20Mutations%20in%20Maize" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/1/163&amp;title=The%20Pattern%20and%20Distribution%20of%20Deleterious%20Mutations%20in%20Maize" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/1/163&amp;title=The%20Pattern%20and%20Distribution%20of%20Deleterious%20Mutations%20in%20Maize" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459622/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459622" id="node459622--3" data-pisa="ggg;4/1/163" data-pisa-master="ggg;g3.113.008870" data-apath="/ggg/4/1/163.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Pattern and Distribution of Deleterious Mutations in Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sofiane Mezmouk</span> and <span class="highwire-citation-author" data-delta="1">Jeffrey Ross-Ibarra</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">163-171;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008870 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459622--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sofiane Mezmouk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California–Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sofiane%2BMezmouk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mezmouk%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASofiane%2BMezmouk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jeffrey Ross-Ibarra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California–Davis, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Population Biology and Genome Center, University of California–Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BRoss-Ibarra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ross-Ibarra%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BRoss-Ibarra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rossibarra@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">rossibarra@ucdavis.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=163&amp;rft.epage=171&amp;rft.atitle=The%20Pattern%20and%20Distribution%20of%20Deleterious%20Mutations%20in%20Maize&amp;rft.volume=4&amp;rft.issue=1&amp;rft.date=2014-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Mezmouk%2C+Sofiane&amp;rft.au=Ross-Ibarra%2C+Jeffrey"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459622/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459622/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459622/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459622/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459622/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459622/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459622/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459622/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459622/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459622/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459622/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459622/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/1/163" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/1/163.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/1/163.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/1/163.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/1/163.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
