<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/1/11" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/1/11.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/1/11.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/1/11.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/1/11/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/1/11.atom" />
<meta name="HW.pisa" content="ggg;4/1/11" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Design and Analysis of Bar-seq Experiments" />
<meta name="DC.Identifier" content="10.1534/g3.113.008565" />
<meta name="DC.Date" content="2014-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Robinson et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="High-throughput quantitative DNA sequencing enables the parallel phenotyping of pools of thousands of mutants. However, the appropriate analytical methods and experimental design that maximize the efficiency of these methods while maintaining statistical power are currently unknown. Here, we have used Bar-seq analysis of the Saccharomyces cerevisiae yeast deletion library to systematically test the effect of experimental design parameters and sequence read depth on experimental results. We present computational methods that efficiently and accurately estimate effect sizes and their statistical significance by adapting existing methods for RNA-seq analysis. Using simulated variation of experimental designs, we found that biological replicates are critical for statistical analysis of Bar-seq data, whereas technical replicates are of less value. By subsampling sequence reads, we found that when using four-fold biological replication, 6 million reads per condition achieved 96% power to detect a two-fold change (or more) at a 5% false discovery rate. Our guidelines for experimental design and computational analysis enables the study of the yeast deletion collection in up to 30 different conditions in a single sequencing lane. These findings are relevant to a variety of pooled genetic screening methods that use high-throughput quantitative DNA sequencing, including Tn-seq." />
<meta name="DC.Contributor" content="David G. Robinson" />
<meta name="DC.Contributor" content="Wei Chen" />
<meta name="DC.Contributor" content="John D. Storey" />
<meta name="DC.Contributor" content="David Gresham" />
<meta name="article:published_time" content="2014-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Design and Analysis of Bar-seq Experiments" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;High-throughput quantitative DNA sequencing enables the parallel phenotyping of pools of thousands of mutants. However, the appropriate analytical methods and experimental design that maximize the efficiency of these methods while maintaining statistical power are currently unknown. Here, we have used Bar-seq analysis of the &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; yeast deletion library to systematically test the effect of experimental design parameters and sequence read depth on experimental results. We present computational methods that efficiently and accurately estimate effect sizes and their statistical significance by adapting existing methods for RNA-seq analysis. Using simulated variation of experimental designs, we found that biological replicates are critical for statistical analysis of Bar-seq data, whereas technical replicates are of less value. By subsampling sequence reads, we found that when using four-fold biological replication, 6 million reads per condition achieved 96% power to detect a two-fold change (or more) at a 5% false discovery rate. Our guidelines for experimental design and computational analysis enables the study of the yeast deletion collection in up to 30 different conditions in a single sequencing lane. These findings are relevant to a variety of pooled genetic screening methods that use high-throughput quantitative DNA sequencing, including Tn-seq.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/01/01" />
<meta name="citation_mjid" content="ggg;4/1/11" />
<meta name="citation_id" content="4/1/11" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/1/11" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/1/11.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/1/11.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/1/11.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.008565" />
<meta name="citation_pmid" content="24192834" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="11" />
<meta name="citation_lastpage" content="18" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="David G. Robinson" />
<meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544" />
<meta name="citation_author" content="Wei Chen" />
<meta name="citation_author_institution" content="Berlin Institute for Medical Systems Biology, Max-Delbrück-Center for Molecular Medicine, 13125 Berlin, Germany" />
<meta name="citation_author" content="John D. Storey" />
<meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544" />
<meta name="citation_author_email" content="jstorey@princeton.edu" />
<meta name="citation_author_email" content="dgresham@nyu.edu" />
<meta name="citation_author" content="David Gresham" />
<meta name="citation_author_institution" content="Center for Genomics and Systems Biology, Department of Biology, New York University, New York, New York 10003" />
<meta name="citation_author_email" content="jstorey@princeton.edu" />
<meta name="citation_author_email" content="dgresham@nyu.edu" />
<meta name="citation_reference" content="citation_title=Methods in Yeast Genetics. A Cold Spring Harbor Laboratory Course Manual;citation_author=D. C. Amberg;citation_author=D. Burke;citation_author=J. N. Strathern;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Anders;citation_author=W. Huber;citation_title=Differential expression analysis for sequence count data.;citation_pages=R106-R106;citation_volume=11;citation_year=2010;citation_issue=10;citation_pmid=20979621;citation_doi=10.1186/gb-2010-11-10-r106" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=E. D. Brutinel;citation_author=J. A. Gralnick;citation_title=Anomalies of the anaerobic tricarboxylic acid cycle in Shewanella oneidensis revealed by Tn-seq.;citation_pages=273-283;citation_volume=86;citation_year=2012;citation_issue=2;citation_pmid=22925268;citation_doi=10.1111/j.1365-2958.2012.08196.x" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. E. Carette;citation_author=C. P. Guimaraes;citation_author=I. Wuethrich;citation_author=V. A. Blomen;citation_author=M. Varadarajan;citation_title=Global gene disruption in human cells to assign genes to phenotypes by deep sequencing.;citation_pages=542-546;citation_volume=29;citation_year=2011;citation_issue=6;citation_pmid=21623355;citation_doi=10.1038/nbt.1857" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics.;citation_author=L. Chen;citation_author=J. Storey;citation_title=Eigen-R2 for dissecting variation in high-dimensional studies.;citation_volume=24;citation_year=2008;citation_pmid=18718946;citation_doi=10.1093/bioinformatics/btn411" />
<meta name="citation_reference" content="citation_journal_title=MBio.;citation_author=L. A. Gallagher;citation_author=J. Shendure;citation_author=C. Manoil;citation_title=Genome-scale identification of resistance functions in Pseudomonas aeruginosa using Tn-seq.;citation_pages=e00315-10;citation_volume=2;citation_year=2011;citation_pmid=21253457;citation_doi=10.1128/mBio.00315-10" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. D. Gawronski;citation_author=S. M. S. Wong;citation_author=G. Giannoukos;citation_author=D. V. Ward;citation_author=B. J. Akerley;citation_title=Tracking insertion mutants within libraries by deep sequencing and a genome-wide screen for Haemophilus genes required in the lung.;citation_volume=106;citation_year=2009;citation_pmid=19805314;citation_doi=10.1073/pnas.0906627106" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Giaever;citation_author=A. M. Chu;citation_author=L. Ni;citation_author=C. Connelly;citation_author=L. Riles;citation_title=Functional profiling of the Saccharomyces cerevisiae genome.;citation_pages=387-391;citation_volume=418;citation_year=2002;citation_issue=6896;citation_pmid=12140549;citation_doi=10.1038/nature00935" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gresham;citation_author=V. M. Boer;citation_author=A. Caudy;citation_author=N. Ziv;citation_author=N. J. Brandt;citation_title=System-level analysis of genes and functions affecting survival during nutrient starvation in Saccharomyces cerevisiae.;citation_volume=187;citation_year=2011;citation_pmid=20944018;citation_doi=10.1534/genetics.110.120766" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=T. X. Han;citation_author=X.-Y. Xu;citation_author=M.-J. Zhang;citation_author=X. Peng;citation_author=L.-L. Du;citation_title=Global fitness profiling of fission yeast deletion strains by barcode sequencing.;citation_pages=R60-R60;citation_volume=11;citation_year=2010;citation_issue=6;citation_pmid=20537132;citation_doi=10.1186/gb-2010-11-6-r60" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. Kendziorski;citation_author=R. A. Irizarry;citation_author=K. S. Chen;citation_author=J. D. Haag;citation_author=M. N. Gould;citation_title=On the utility of pooling biological samples in microarray experiments.;citation_volume=102;citation_year=2005;citation_pmid=15755808;citation_doi=10.1073/pnas.0500607102" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=K. D. Hansen;citation_author=J. T. Leek;citation_title=Cloud-scale RNA-sequencing differential expression analysis with Myrna.;citation_pages=R83-R83;citation_volume=11;citation_year=2010;citation_issue=8;citation_pmid=20701754;citation_doi=10.1186/gb-2010-11-8-r83" />
<meta name="citation_reference" content="citation_journal_title=Sov. Phys. Dokl.;citation_author=V. Levenshtein;citation_title=Binary codes capable of correcting deletions, insertions and reversals.;citation_pages=707710;citation_volume=10;citation_year=1966" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=X. Peng;citation_author=C. L. Wood;citation_author=E. M. Blalock;citation_author=K. Chen;citation_author=P. W. Landfield;citation_title=Statistical implications of pooling RNA samples for microarray experiments.;citation_pages=26-26;citation_volume=4;citation_year=2003;citation_pmid=12823867;citation_doi=10.1186/1471-2105-4-26" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Robinson;citation_author=D. McCarthy;citation_title=edger: a bioconductor package for differential expression analysis of digital gene expression data.;citation_volume=26;citation_year=2010;citation_pmid=19910308;citation_doi=10.1093/bioinformatics/btp616" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=M. Robinson;citation_author=A. Oshlack;citation_title=A scaling normalization method for differential expression analysis of RNA-seq data.;citation_pages=R25-R25;citation_volume=11;citation_year=2010;citation_issue=3;citation_pmid=20196867;citation_doi=10.1186/gb-2010-11-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Science (New York, NY);citation_author=M. R. Schlabach;citation_author=J. Luo;citation_author=N. L. Solimini;citation_author=G. Hu;citation_author=Q. Xu;citation_title=Cancer proliferation gene discovery through functional genomics.;citation_pages=620-624;citation_volume=319;citation_year=2008;citation_doi=10.1126/science.1149200" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. M. Silva;citation_author=K. Marran;citation_author=J. S. Parker;citation_author=J. Silva;citation_author=M. Golding;citation_title=Profiling Essential Genes in Human Mammary Cells by Multiplex RNAi Screening;citation_pages=617-620;citation_volume=319;citation_year=2008;citation_issue=5863;citation_pmid=18239125;citation_doi=10.1126/science.1149185" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=D. Sims;citation_author=A. M. Mendes-Pereira;citation_author=J. Frankum;citation_author=D. Burgess;citation_author=M.-A. Cerone;citation_title=High-throughput RNA interference screening using pooled shRNA libraries and next generation sequencing.;citation_pages=R104-R104;citation_volume=12;citation_year=2011;citation_issue=10;citation_pmid=22018332;citation_doi=10.1186/gb-2011-12-10-r104" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. M. Smith;citation_author=L. E. Heisler;citation_author=J. Mellor;citation_author=F. Kaper;citation_author=M. J. Thompson;citation_title=Quantitative phenotyping via deep barcode sequencing.;citation_volume=19;citation_year=2009;citation_pmid=19622793;citation_doi=10.1101/gr.093955.109" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. M. Smith;citation_author=L. E. Heisler;citation_author=R. P. St. Onge;citation_author=E. Farias-Hesson;citation_author=I. M. Wallace;citation_title=Highly-multiplexed barcode sequencing: an efficient method for parallel analysis of pooled samples.;citation_volume=38;citation_year=2010;citation_pmid=20460461;citation_doi=10.1093/nar/gkq368" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. D. Storey;citation_author=R. Tibshirani;citation_title=Statistical significance for genomewide studies.;citation_volume=100;citation_year=2003;citation_pmid=12883005;citation_doi=10.1073/pnas.1530509100" />
<meta name="citation_reference" content="citation_title=Dyn Biochem Process Biotech Mol Biol;citation_author=D. J. Timson;citation_title=GALACTOSE METABOLISM IN SACCHAROMYCES CEREVISIAE.;citation_pages=63-73;citation_volume=1;citation_year=2007;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. H. Tong;citation_author=M. Evangelista;citation_author=A. B. Parsons;citation_author=H. Xu;citation_author=G. D. Bader;citation_title=Systematic Genetic Analysis with Ordered Arrays of Yeast Deletion Mutants;citation_pages=2364-2368;citation_volume=294;citation_year=2001;citation_issue=5550;citation_pmid=11743205;citation_doi=10.1126/science.1065810" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=C. Trapnell;citation_author=D. G. Hendrickson;citation_author=M. Sauvageau;citation_author=L. Goff;citation_author=J. L. Rinn;citation_title=Differential analysis of gene regulation at transcript resolution with RNA-seq.;citation_pages=46-53;citation_volume=31;citation_year=2013;citation_pmid=23222703;citation_doi=10.1038/nbt.2450" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=T. van Opijnen;citation_author=K. L. Bodi;citation_author=A. Camilli;citation_title=Tn-seq: high-throughput parallel sequencing for fitness and genetic interaction studies in microorganisms.;citation_pages=767-772;citation_volume=6;citation_year=2009;citation_issue=10;citation_pmid=19767758;citation_doi=10.1038/nmeth.1377" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=L. Wang;citation_author=Z. Feng;citation_author=X. Wang;citation_author=X. Wang;citation_title=DEGseq: an R package for identifying differentially expressed genes from RNA-seq data.;citation_volume=26;citation_year=2010;citation_pmid=19855105;citation_doi=10.1093/bioinformatics/btp612" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. A. Winzeler;citation_author=D. D. Shoemaker;citation_author=A. Astromoff;citation_author=H. Liang;citation_author=K. Anderson;citation_title=Functional Characterization of the S. cerevisiae Genome by Gene Deletion and Parallel Analysis;citation_pages=901-906;citation_volume=285;citation_year=1999;citation_issue=5429;citation_pmid=10436161;citation_doi=10.1126/science.285.5429.901" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Design and Analysis of Bar-seq Experiments" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1.cover-source.jpg" />
<meta name="twitter:description" content="High-throughput quantitative DNA sequencing enables the parallel phenotyping of pools of thousands of mutants. However, the appropriate analytical methods and experimental design that maximize the efficiency of these methods while maintaining statistical power are currently unknown. Here, we have used Bar-seq analysis of the Saccharomyces cerevisiae yeast deletion library to systematically test the effect of experimental design parameters and sequence read depth on experimental results. We present computational methods that efficiently and accurately estimate effect sizes and their statistical significance by adapting existing methods for RNA-seq analysis. Using simulated variation of experimental designs, we found that biological replicates are critical for statistical analysis of Bar-seq data, whereas technical replicates are of less value. By subsampling sequence reads, we found that when using four-fold biological replication, 6 million reads per condition achieved 96% power to detect a two-fold change (or more) at a 5% false discovery rate. Our guidelines for experimental design and computational analysis enables the study of the yeast deletion collection in up to 30 different conditions in a single sequencing lane. These findings are relevant to a variety of pooled genetic screening methods that use high-throughput quantitative DNA sequencing, including Tn-seq." />
<meta name="og-title" property="og:title" content="Design and Analysis of Bar-seq Experiments" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/1/11" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="High-throughput quantitative DNA sequencing enables the parallel phenotyping of pools of thousands of mutants. However, the appropriate analytical methods and experimental design that maximize the efficiency of these methods while maintaining statistical power are currently unknown. Here, we have used Bar-seq analysis of the Saccharomyces cerevisiae yeast deletion library to systematically test the effect of experimental design parameters and sequence read depth on experimental results. We present computational methods that efficiently and accurately estimate effect sizes and their statistical significance by adapting existing methods for RNA-seq analysis. Using simulated variation of experimental designs, we found that biological replicates are critical for statistical analysis of Bar-seq data, whereas technical replicates are of less value. By subsampling sequence reads, we found that when using four-fold biological replication, 6 million reads per condition achieved 96% power to detect a two-fold change (or more) at a 5% false discovery rate. Our guidelines for experimental design and computational analysis enables the study of the yeast deletion collection in up to 30 different conditions in a single sequencing lane. These findings are relevant to a variety of pooled genetic screening methods that use high-throughput quantitative DNA sequencing, including Tn-seq." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1.cover-source.jpg" />
    <title>Design and Analysis of Bar-seq Experiments | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"utyZUkWdzs0XeEG7VJCWvul0iVj-2WsEOh3H_00dsGk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459613","apath":"\/ggg\/4\/1\/11.atom","pisa":"ggg;4\/1\/11","ac":{"\/ggg\/4\/1\/11.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/1\/11.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/1\/11"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/1\/11"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/1\/11":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Tb4HUkkYZiaVi2ZlcgBXuZzuTouNd87Xsffphnq0VGEsezzfQ8GMLGVaLf67ay0nFqT16LV\\\\\\\/dxwHGfzgDHHSqI3YuKWd9UZaMoYNNxA0iqHG0PC3d76jLebi\\\\\\\/Eya5bZhUkWKq4ETURnNP65xthC5y1I13Ba3g5DJen1QtGAM7smBljek63xJ6IRl94EvcOVEa7CdwfMCQ0U0IJ7q9ror\\\\\\\/iJhJTdIEIj2t4ZjL5XgBwT6SopQuU1Uj3+EDyOJ06olWy83cdU33uD\\\\\\\/Om59x3R3Dy7RfGnLgCe8YS1HXKwKuQ21dGaPXCo30O5SQvYSzAWG288Vzn1UhvI5bk0TKeXBpH4zVxtY6gVgiwBvBUKWEFWoJoX1JiiYKy5P7vVdvnTvvRTtvhRmbbOZ5tIhr\\\\\\\/eOtl6\\\\\\\/4WKJ1npYqV4iW9IBRLSc3GVum5nAOW0zKNhpcreX7mMQ4UPCiLhRwizpxiETtjO2dc4Otbj1iFxOgtzGnwiFarhypMhqow\\\\\\\/qva2uaGOz4Nbtulpy24We+pV89PqDRm1yLk7k\\\\\\\/ktoPCyEc9kSrUl+2T3scbKiP4WqvdC+YAghS395h6UT0kB2y+VBq9XjVHM2+GfW0c9yZgsWZ\\\\\\\/GjUnWnWd21xvrdAxqYRngoJqYU8zpY0o1vzvnrCyZ0LvrGFXpmp4b4UgtVq3SHuF25eBSyZipT94CrdZK2rKzTQ1T9hV9HdO4C8yUaBur0VpInyfTyTrkSzHT0OInyY7oK5ayqTHl9CYj\\\\\\\/BgUypYHc5UdMNdO4qXppOHZo1v7s7C94VQ1zviMTD6fjuZoWXI6gb5YyJNlrNOliMHamHCOFKOIYQjD1OFwpUB7QLawoPb6e3rv9Lj7PNfq4v\\\\\\\/pjxT0WVLxwElN6+XujErPJ8\\\\\\\/ZP+kMw2X6CS0u7Pc0aOAFwVAadfLDki51jNw5sxv\\\\\\\/oFJtar7k9ZFEq1wpzPOw8X0M\\\\\\\/qBmHmfKWnuuR97pYFGC56uj5F+dmlsaHvIPKEYOXE7voZMEZbnPC9TRQHj9kFFZaH40JR2kgndOK7XQ4QMm\\\\\\\/8c25Z65vDvAtd6iue3qyc8LrSFEJKbsFdX+SnABBRSvl9fsnbX5xVqFHPJjbRSYaow7ciGiM2Mh2xKxi8urEsn6lCfh8VWNGGdBtCabZhsFDoP0RPyV+nONK7tqHTUpsB5EXcC9wvKua711fkprmogHpV3yFVX2MwPTRmLtpVBCHEoQlMUwnU2zkdtlMJ\\\\\\\/G+UkO4tAWdGNomlibo16N5jqpdqH+I8PFx550oIQOJ1v+1V7U4\\\u0022,\\\u0022iv\\\u0022:\\\u0022HSCr6ze8NmZU+s4Z\\\\\\\/gicGg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002261c6932cdf10351feb6dff6e6cf6d82d09d5f4d3fc76260fad29ed9d834f0e13\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Z6qRqhJ713kN7r51H0mB60z26hdvN2\\\\\\\/x9fQ1\\\\\\\/5aN5NWyOfKzjEp7ljgAzgsyuhd+du\\\\\\\/mTZ6zTgpjqVjalLGdR91et213rP4y2X2W2fm\\\\\\\/XF640JenRXGSQvyPOtCdmIkvREtDp110dEvo5KC0umqceDxPv8C\\\\\\\/hWCCzdidod+rPNxHYwSi3btWgBfUJX32SGCl0buNQjLZdJoHULmHkFvaYoiHfJK59gZODWeQ14OiyNIU9AiYZSdE3WD7ZkLQ\\\\\\\/0maBOESqBwwPSBIYSycARjBLdfk4CdslH9ljVaRPz3bbsN0sGnPxfwTtoFThoFqovn9\\\\\\\/Q5\\\\\\\/aiA38gWGnnbpsXxY3M6N5HuATlbhUQ\\\\\\\/wJJI06RjLYmtTpBAWKXVNGmVKyuybRTZDrN\\\\\\\/qCxAzrMQ7HbsuCnHx1uxUVqxjE\\\\\\\/Euv93w8kM7yZMDnMw6u3aFTMMG5jZPjtIznMI907QasV6ZT4ISiPOgl\\\\\\\/zcJLZsoqKXJuTqWPk4+9pZsTm8+\\\\\\\/LVAxM2P9wEHCvjh+wQUvRuT1S4z3yt3GNC\\\\\\\/Fn0Q+4qpNknidjY76cD5sgPQu2h+oFz7YNaf5SwK0VmHWr+5+qAf9gTERZh3XP3qzDFTD9x13t3IBCzJdf\\\\\\\/bg3BRxSCOD1LzxiVFwTPKtA99jIWBL7O5CI\\\\\\\/1+l+T\\\\\\\/Q\\\\\\\/TuL6WyEfmLv5l2ODer+yarO8A9brd8yTyy33UjdWcBfJGsonQvhNb1+d9b2NIPzpjm2s4yL+w\\\\\\\/5I0bWdIuGpLE968oK2WIqgdAUcu+LIpWDoA\\\\\\\/X+g921zHw3NqsCtGvpiOELyRfmfjGfFjR4JZhiTDXH0w43AVsDxps9j0tfQYLoHHjtS3SgpZr3mAb6bpCuwen7V3ianUhYYVLoNdRGXifE1V2eueHZLvF0FFVcShvg4I1Hbv+DtiijObpBzbMU9MARmtrQFnaQevOBXfmF1GjA8r13NLcfhL+FbpF40lewR+mJY65TGaz+4TdP4VNVXYuShSSsvYBYGTR820gPYiEWRD9\\\\\\\/v3RjXYICj6qUDULu58lVWv6TQlgySUp+BzQ\\\\\\\/BmAvjt1kysZbX6Wz0wS\\\\\\\/Emeb4TM4dkl+peQr4rEopZE9cubuJiWhdrqLRU9iS8oHNzibINrRPkt7M14VBezuCRyS1eJr7zcRpkcLBIHbWVsumypUivY23RvJejHcDrdW0e7mk\\\\\\\/hWJ5frzWrQvcTip9z3w6xvJoNbUkU2qhQfpa8Y\\\\\\\/+Nqx6feHv7mUGiejhmKrc3fkqXNcmyLyLeeyODaW\\\\\\\/tUR1kcb9nmINEX67oJ67qJb6kXzNUaFoLFlPVTAWUQZfIEv0nkQOpzbwos4ZpAIQZJNzVuMBoX48pys3TPfvpbFqiCwrqowHj3+TWyh58dKbhZqgtCqGrL5sO2IcXlUzr78aXgXUmjAnN4n0Gn+n7LRvhNUWk7921a\\\\\\\/jSVlO6PkDoKfRRfM18nuuO1T\\\\\\\/wi5eB35WoG+pY+qHa3BRcEq+\\\\\\\/15uwG6wv\\\\\\\/Q+VnkRxBB8MXiL9iLrgODoRK1WY3swK1S3+tITldECt+EaUqT2w9dtm6nM3C5Xkcoq5VXZ0hYIWko9IygJNGBdZGSmmGObs3dDqbTPQjDNZRInqW6hBOuxIAN8gDb7TZjXluE8XLO8+X3wX1ymhZ2XSuwBAToLWD7Tjc3EXEiSfQOgA7S+4dwMQbQrY35mD1GKyJfBM7fyr\\\\\\\/FJHc6peghfJx+Ozkhbpcu7w5T8xumiDIwQoxCubFQn6zUa3\\\\\\\/4wRzXrx5XjJnJSV\\\\\\\/nYgQxrBLk\\\\\\\/8vLJtO2cVSaXD6O3a+ZmOJxBBThDQOsby5ixdplBOCR9KIMrnzNqzpYjlIZfy3b7jsjOu0SwV2QgAqBJCseE8I6dcPI0p9J+uMdNfh3njCp+sf+lo8mOhrNjOZrjXOi83WXiMmdDLXQlUYzMc8iIgirdnvU18\\\\\\\/E+6d+2i7r0OsAHLpYCQ0Gd+u2JNDRKniZT2tYire0zJ4xYLs31nLZ6mhI2aViOijHnPzcOp77HCsNvf\\\\\\\/bi02tgDchgw7v+MoEdW91Cl+etfG\\\\\\\/5C+6q\\\\\\\/irGqZb+tiE1Po4okpy3RvUg3IRzBWuJOtNNE4v+M1Vi7EixavDsaR5z8JvDfiRPNxeUsLMnAF0kvtgUT9rFydO0+exLDOTMK3OkMIKl03\\\\\\\/FlcFVl01EeMu1qXziORo1YgbEKRW73NESVIKF1dgzNYilcuwbp\\\\\\\/wZHwJ09gEmIPHkP1DJpW75RwiCRtVmHDz5lWxfYpZTbVuCMk493rF40A\\\\\\\/EsB06Jd91MdwsO+TUUHcs2A\\\\\\\/rLhW+JWjyEwIT7sO9eEhPKxmfA24zCnRtuSkoTSdSuQqGDaV65xFdr8Dy5JLUdNhOBFDthnoma4LDuP8LCCdC9XEVilfhhR2DcMh+a\\\\\\\/P1n1MGYEnXOjnDtgaNs4akmsHMQHQvwgahx0LGP3pbRfrh\\\\\\\/F4K4+R\\\\\\\/\\\\\\\/Ol2pMOyK969JSxPJrR29D5jOqdtgzylMtg0HyLludcSC2inwSpBa\\\\\\\/Tsz94swiF4HbfS8QzQjX4xX\\\\\\\/4fBVIZYEtahbIueJuqEJOpQwbJVFU36aCQEbpmCMu6J5hr1YUMJvRAuSrMatLMRjR9YuB89BZSjz\\\\\\\/x+GLpGTx6VdX1VceadZp1XmQze9f8W14zDEoDUG\\\\\\\/og7pibVN8Tg0tCMjI5UVzUUtHiKKzuuLFwA0zdFAmTRVxbKeIkM7oXKKKsUWjEal1+xzEecFiidYGv4cqN2RqxUOtxmndmAgLIgq61vcWXeE87aWcFDp6c\\\\\\\/ecQKvORY367EpIHWdd\\\\\\\/zdXzXBR3EY5uPN0W0OmhTx0s5l9oLhsaGgqnJm2SrQ1HmfSapKAiV+RLlUfXeeute4OPRpjf3fiqQIUrZS2s8ww2TRoWgE4q1TtgRm+yzHj0qBWzKr7tRdKdOHDUFppD+nXLCnkV\\\\\\\/X\\\\\\\/zcAvFBaFM8fte4itR1DHnxAMuO5OR\\\\\\\/JyDH79e9ZXwbknlx0gkK0\\\\\\\/2svI+cb2dy20JlyWrefnmLIwJ+\\\\\\\/9JkVrrJYUrTcu3FFNHVisg+l1aUqldy6S\\\\\\\/29OqtTdDrFhtkeplVeIIHK6F8YWdjlOsGmH+vOsW9N75mjRYmMn8NPHnIlgk4hIOUmLcPUjFhk7J2PN0q2dbfz2+OMeO9E\\\\\\\/YC7d5Yl9Hmw15jDvx2XsiTpWwyPHxUAUXGHXjSyFoS2QoJSIaVEjv9rhRsUu7zLPWOa\\\\\\\/dYts8t2KKhp9+AES+2AMA3cw9OMxS\\\\\\\/4oLMzClu5iI+qC\\\\\\\/At+iS9IWoBet5\\\\\\\/04XvDoGotN820tBywcpYCH5rLpyewy+5L94IaPmizrj44VIOGr+TwxdKsgE+cXJeZ8I0F0z5nmBttdjwzDLu3JL\\\\\\\/FdNIJQY9ZNThv8Yax0vr3oHx9RIEA09A3yxiHmFPgyPtbVY3InW0Wwd0MsqsvS9CDCwadYS8tivJWYCq5q4wkzTxCsZXVCigc4e\\\\\\\/w3bdgOJyFpdjUbQEJwzgN+2ky6HDf0OC1zkyxoZPvYaG9ifQjs0wOnv6BPsg3KOTCOABR+\\\\\\\/vpec9aQjpJiH6SB6QgSwqT2yYk5VpUqda0JzIs1y38473T48Yb8jff\\\\\\\/gDyNkmWYrl4FL7Z\\\\\\\/T4U0zworwZdZgE5O+zxU4POYIhEgeQpnh867++t8tOxDFwuy2W89o6uklbGj0Dc9JRsksru0lo8vQaUxKKFudqStaRaBKxRXOcAZaH3ZZQpAeOjhNLZmuNXxDy9CnlBe0xu0LW6v0FDYWyPICC+f3w8xXTfjD97EXkRXVygA3AjaSyCQHrdZNWGNyZAMajg3kzs8ypqjCucZV68hIc4pVwT0xlJjs0QO6f69iVlJc4nOaf+VaUkKaiOkSCykZ5xZPkWVQ+Z6RuUK1TgZwPtotSTQFzQFJFiyQR24kvncAA6AoHDddSyZGlKPG4zqoXeJjq5UB20scRLFAr6vMdaueCX\\\\\\\/IqtplrCsZZ1TPoP\\\\\\\/gMWHKwC9Fj+0+Zwj\\\\\\\/hdGEEu75WxyUzNPLTg3CnES81Hui8kmCnuqMz\\\\\\\/Chf7UQuv8H8kO+3qgXx2VTBkreXjDAgcgcaXt99+t1WdbTvexEm+d40TEqEdc5WxDrssX+S1OR4iW7dvyViGjAnFdMRAWzQXcM7bi2zs8X7jofdokZ8\\\\\\\/H3KX6q2kMYzCS7X9KB2+cH3YEjQGAD9eyRbgz0zChhwwzRcEz6QFRGruUPbIYdJQbBpUr4wB6ulySR6GnjTASITBsS34hK\\\\\\\/\\\\\\\/C1c5WIGL7x5q30\\\\\\\/2sPjSPHlNVaHcspE\\\\\\\/rz9y6ktQpM3KbLvfh+Hui8j5Uf4O2N3fRsroP04syduW5b0Rj1v8bNgdx+11kYKCozbuf+5eaWb47\\\\\\\/ozCb8gGUkQuT52LG7kAU2nS0zr\\\\\\\/TCsDDmPP9dWqAuor0rxhJWaixFL+0HzcXCKOzh9+uhMCx7mvRa83QfAh3m+RrB670ComI\\\\\\\/R5Lem8cV9AQk3\\\\\\\/bv+V4AaPyGHYt2t7n3DBtENQ38Zb2oQb6g50rLo+tCE9jR3vx3F\\\\\\\/sa3BVj8EYnJ0Zx9WOxuweIH3V2PZVuVt\\\\\\\/3D6clBXFefIeV7rnZBA5Z8lkVOdjjnbt1lJdZDnLFmuOHiX8UUNu8BcKvjjrNmL9PZtQbMPiJnHUH4kJkF\\\\\\\/bgd0Scjyw1T0UOjd0ZB7+MXhicULDQbrfUrq+Y3m4d1Rk3RBM+YWtE3BUv6WnFn8BUFCDRQhMTOXijDpAuQVIQmHDrNeXjHhjPQJGp7Y5MQ5pYIpzrBxRbWDNiyBjgfvIyqJjARE3SLyCBINHwB\\\\\\\/J0BVxG9JDn7+2FqRMT1kW4Af+jLBylVXJhb67xfJOBqab1RdXG2Z52\\\\\\\/V82u61XvidMgWdGiMLEekiQsMu8T3Q6RKOSRwcGv2GNgaulYlpu6Aq+i2\\\\\\\/AVnkPGqx0MfZwPzJcKVsdADaFR7Qcgssf1BxohB\\\\\\\/TiJuVcIhxUjxMrLzezv44rbvy6U5KgTu7bmhk6r8asj+kw8yBJV8OmPagTsEfVNjUTfSSSNNyRf87YP+pCdH2ZJbpzHNVVpIu3f9uh2JGFP9ys4E+rzPpK2SlaX87bnHvDA+zIjyXQ3CbkxcqFVkIG2Cra5E4OC6dU+6HqDNP7t9XRhxaNRBmt8zZuqze85TuAcpifrWFtqqzVUVZzxiJQsqlg+AXy1MRBN\\\\\\\/kjgSgvqQrdohvQDnKKO+mGg8WV3l2IXAGXmGjTBc6Zhs8W8f\\\\\\\/V8\\\\\\\/6YBbu+rUzEMVizFSgFhCj2zj1fi+0NiWHcEYPQfMiPofZj2LX798JkcWrKSXKEcWP5wVdoUceA5zYk4Qu0nCFpi0ZyGFj3k2mV9VteSqvO9SQP7EQiLHXDIdXZZ9CNrSaiB1qrp1tz99ddcZLveaxkWsLidmnjqWMVRvZK5gWREoq0cedvdF0p3tx\\\\\\\/xdreki9tbBdcD6IzpEhRWdxWAXCjiPtOT02dwBgixl02rRqi8pwCHuWn3FFrFt1wffyNBmaPipnTTnoyNh7UvdDhjEADyIXEawn3VkNyE064M+z1d0y13EQ7sJG1u5RMdAIJQVOLvOMaxzi7smbmXRMEqWlE2QOVG0xCjAdh0W8JUmP8zJHtP9XKfDMIvSKv9qNb3veDx31+dp4X++GMS33bprmimZBFAf8sXe1oi5FZly5PkMQ6KS2q4oqB2+Quetv\\\\\\\/Twd4j2k\\\\\\\/lcSTI7EfFIKScyoVYO6cgV1E6bj7qhTxxwrOlKuRj4+xcZOKwik71pK7T3CVtE+0Orof+ghU+DKCVtVPuMV4qgIqL\\\\\\\/96diMzUPvxffUlv9Q44l2QTEerbzAkPVqUv4AAXwDXB9b6OzJ16BvVyCf7OCTJ4FGhMC7FY9RnmH6MCczjBJGfK9PHVujhT2mwQmk6dSV9phjQcmwpO18qJfOhXGZTYtae9Oh07PAi6BbMVXAsqesdla3P0p3XqPHGfYHNEnqgkOGd8Lvqp9y7rpGmuYIwVnSlTxuCr6x0KM+fcN03LM9qwtzxfEpIwmkfYhYl+\\\\\\\/u70WKq9bCN5DClaGxil3sItqr85KtwxP38Waj6nBe8VWLkY3CR6bVfJ1\\\\\\\/03N0QizQOnHl0onL3KLMTXZMrJb4E2R8OabBpdniRQIm5fYwVenAlTcreglFJ+UhbjWSEPShPe5\\\\\\\/MefNQHx1a3GLCLMgMsSOnneGzGYCvDJOOSFBfZn5tNJ\\\\\\\/Hfm2rHDBqyP1TZ340m7YJkmcm1ICKHgOfAuNPnL4iY\\\\\\\/Jf+f1lPvp8ZB0THd+eqA\\\\\\\/+CNzwn8vNZ8hMOeGVXXWQ6BoLdvGLzdRAwX7BLJQYF\\\\\\\/yWISzPfgMZOj1\\\\\\\/0DYuchOAkpj4YTbbnAc3eea3oK8etNX7Z39qMH3qY43UcDD6VEUDXFwchO8Vx9kA3FyHRHVBWHTMeHzR16vkCvSOvVxH7p\\\\\\\/vrQ45joWW3mVuOWBisfgtXXI7Ag6WDxYZkWfEM1CpoSC8KtrLwr0TkANq4ujEKpadljZzxcQz19gZ\\\\\\\/FsyKlVuK2G6NWvrxw9okWbMA3MtkmdmV98GvrGASkiyiY4pHIMojDJk75dYY6dG2ehRj2KgyRUyAsXD88MF+duKCwG\\\\\\\/EgZN1YIwjlyHooh0Sxwm4Wo6zd3OhmtwO8GHTuJlWTYeTw4qcO8K2CUL2MtDHuW3vYcXTy6mYTLu+AyFCTYTxhwokhkK4XVmz3EHUNHJZ48\\\\\\\/DpQiQDkTWmMD0EiN9xcZegsGassSN3BlgtfmygGJttY2m6ZdaXYA10w8Yh6d5DqG4TPj9vOb8ZI9n+Jjy\\\\\\\/kDHOjiTiwt3iYZzYuvCCBx+HcKKKa+bDCX4yRS4nBDT04ZfIALjgQ6woVCqEzGRQWOqf7eDiSH1eK+n9EVIVCmXOhonSuEF71B7OP\\\\\\\/YFfmA68DCQuVWh0CYXnsoF3yCILKEeUniWT7HDGgTelWcb7lElpJFBxk4xApqWBm1s36f1m96NyVEaEBn3eumg7cxOIC1zuDTphRCDSBfNPw1CyIO31t7dwaSrFdfLJ8URL4WuVQ9h5BqwYFgRiknkMKuxA1YpaUa+bDO+k9LDuCTxCsn0JkFXCsBKHP\\\\\\\/lTQYYmtt+ZwhVczTCg+Ewa2E2YP\\\\\\\/vgs4CsNJqhlJz\\\\\\\/xvVSTv+zfi4CA7NbF6kESmh\\\\\\\/q5ORxlYFEPA9zaqUSiSvP8oLrCLial7I6EZHe2q12mZEFfFiZULjUk2USSB\\\\\\\/ya0NBmpdz9gDUCGovGu0KxCM4JhUnv\\\\\\\/PNQsfmaTjzMKRmSrIajUcPmlI+muyvZQXjIqRcEZN3Ew75fRXEQ62fzTln2hUiKX2CTCpFqW5iPUJzm38QYJVr8YIu5Gvx+7iqoVt+E60prmf7RM1Fzc5eAnT8ZhyE1XuEp6A3aFeerLs8JshKFnKxDlOdwuCua6XPnhhFpO84dpMrNHKf8808hpVz2aw5A\\\\\\\/qPLkBqgbl1ow92X2D+p6Ff59CfZSCgma8Qew\\\\\\\/1\\\\\\\/+MJQwzGLRKZO38KASk\\\\\\\/cL4z3QwoTa1fhAb1Eh0TT\\\\\\\/5fKp8HYO09X2b4+7aZNcIDU4dd92QY8IapCyCkDMk8rKEzCuywSByOb412N+1IaN8W1ZciD70KqiATFLcbtF7q5Y5SC2ytaNAugl5dRJvZLqQcEa7WnGHrdS2yIZsk0Hxw5gYinmcuIWZjq3ANaosQy5OavGO2ZvB4fEN3oSDWy0sZK0C+WdtWn\\\\\\\/xu8inSkckEtk9Lt2YuKJcHjFaHlDv\\\\\\\/LwrGyG+Etrv0UJkYGMpVgoGe9WjEwwtsHkE7tjF0b9IhzLGQB\\\\\\\/PgvSPpAa5iCGpS8iEGLUuGc6EOHpjEpgvm2rYqDq+xSHBGCG83EpzR6bxkSqzCFAsDVE6zMzzXL0RfYtupwqq1PJ+yabunpPbwALtWJkmAd+YD1Ps2yN1E9pa2Zhwr4rl4MArL1Cvh0rYaXR+cvRYoPbhOt+HrHyGb3OW0rEhSBFEbnX7LZ9sK97vUkqskyUZfYyMvwaqyrbzM0oL5\\\\\\\/x5VR9V+KHRMI461GtNvXh1oJASp5ist5rdpezC3+NxMYYtXAybj81nvZfP1F82TJMeqlgRdEF6GdSNGM+gULaFvMk7Z6HPl0G395Tdv4AayQncBaDwzMyQ91B8M7QtfYEg6sg6DkCDJpVZqa\\\\\\\/XxlhgFT07bCSMp7gusAiUlqCVXOxyFG1C7BoG5pupjzTDRQxrZcMT2okE5q9rGziiieTEtgPnG+c\\\\\\\/z9kMQWIkSQhfjpFIwUn1IytnoeTYveaUalqiQPtlIvk0pPnHu9P9LQEPX85LZnh+S2PgM7FmV+7i093w8XFQwgtE1+lI8szktED3ByiKEhnDVNLuAMYGGwSruzxxEr8KDVRKmVRcWkpLXYCjFWhsDtuBlxzKlcN3v0LB7hAaXHjk8O6CLpnWIT12J1add8MjdlkGqD0YlQEz+rcdqBUlCBGxJsk5uucOWYlfAjvYmWcOG2ZrckNBEY+SgBxX5dxtVtd1NZbRKtZhetyK5OpQf7ajBKzq6ehIK2e8T9hXbbLFDZoqzYXrv3lZcs9\\\\\\\/9iFMBR+WxSA2Ka6tFjuwLGJyzY36bKRPEkd\\\\\\\/LfKdAH3lckuayuQ1RGyj3IYmT+FRX6Fs83mMJFyf6iK\\\\\\\/ZBILWlGdl25E8PuNgkTBEDZdzzCO1hTKcI98N3J2RJA04sDUhTDlQVqiEUyjnZzGwbxtlqLKy4slnLhiLHUNF1uh4v7Hui5Wok8CqkN5n+0m6cssdFW8M1WHTONA8N4E\\\\\\\/7HDgMEymxwEV+\\\\\\\/fs4AewazR\\\\\\\/l0246jDA+Gp3ZI+41G00rY5D4YZW2h3ecZ5Qkwjwkr4SJ77+rnPlsdKZBlt7MZzUShBl61XRvqK13Ukmyo\\\\\\\/18mndVe6tMtkAbIcik26aE1w1hfqWi+Go08gquTr6\\\\\\\/mRV9cBqQsGzVkAqU3d29BYG6xaN3HgI7tw0XYxxhpG0SVUVG+ooRzoFTiue5HouGEQ0B7vqAu1ossACtJc+HBtCUXfemUyLnBI8Ul1AalAUxp6YoyWoXlyAaTDoP0B+uz2TqE\\\\\\\/bj\\\\\\\/lziNGBJ\\\\\\\/WSzpHGDKfnQ1PSgck57PgUvldDAUrmlJtr86q2TZvfLrM3Vd8xXjrO1wFV39VXTFynfHmscBCdkAz0j6POfRA7S43a7gho2tcVzPrzJ+Lma+mOJWUXxPmX2VrMhiIRA7jkbOSuVcM\\\\\\\/bxvpn7b+VIGUHt7gWWNcz19dT9q5H8zPn0ewnuY4nqArpsomqNMPypDDGHtAIq17OWcAoDjsZboAqrDJ5Dqs9GVxzx5ZmgmDVGezysAR6V8hj3SOd\\\\\\\/rrLf7Z1sQBufxgsqp8XX0l3ZBUaD1IRfk1piYZQgysXaufXvz59esUtNOAWw0\\\\\\\/urToc8T6na4w6SUXJj8EJNT1fx+WYQ0lIYt5ff9nSlDDfYiQCH29cZ7sMGYHV+YhusDm8xQ3m9xN308mxkS6DNb5EXZdUet9xfeaL3xcEeb6AOC8szTOUSr2FsUGv5yaEh\\\\\\\/IPr9CD6UgDd1yVODcG1rJLRMpa6Ukg\\\\\\\/qRRoJo3cvk0K717TxrilRZHg1yzXqmKHryD63aHoAJLj94hERAspKXhc1l6fYEYdNE4Ra2yl5YC7YH2BZN+2\\\\\\\/al3kNzdIONBNOLFm+GplDp4oel2e0B5Jk0\\\\\\\/6VwNdRqJkj3RDGj4z16SZ8mMPKD29BXRYohvKVOY\\\\\\\/SsLxLAZm7yzGwj7V7nN9jtaQVe86YQhMaa9yvJGQTPvZeIMNRRsv290lVsdmEYSqsjHnG\\\\\\\/cBWj\\\\\\\/VeMitbrwoPrk2JPCw6k1cgMnIIGZxVCx1ALj1mJ7a\\\\\\\/NO1bpjS2zoMfeeHOcaJcNsJgJzCqh+MpY+ZbzH4Qh1lDKdxW9cBZFYa+asVB6nZmdbVR\\\\\\\/U64frfGUdzl1ZjJD\\\\\\\/57EWaErPhm+OJ72QWzOtHGxBzG0l9QTusDKwKFcHviDp9VE7nmPcrZfTdJfqwQDSJ7UOiX2YzK08\\\\\\\/AzxR2uEIlfOhwpTGbgsoH1f3xbTPIjyAKSQCKR4ecY9c75uhYMHG7Ni9hayPsxgO4MorUTIX9PoKp4s\\\\\\\/B\\\\\\\/p4JGZRi3Ud9NXSw7aYalGu9A9dl6JKd57TTXsKOr061NW0DKmkAPVxuA4aedsjRMRpKEV4P0HII8iVyRxSw2KINRffrR\\\\\\\/\\\\\\\/6fTdZg9wF30WFoHlitYiY7Usa7ATU7HBNs5fgiGv9OoA4mwfDQb+0NyhkDDwI2bAk9d4YF0FusyBK\\\\\\\/X2aAJPaKjq84ZdaZo7ScNxQ+seeMpVQFuTvJ5m5AMcE+88R4MWXMWksCRqXh0TbzeXQXK4aFymmf4\\\\\\\/6dhdv0ogfi6tFP6bKWjJQ5dSK1HSYcHKtpQFERZQXdnCKYt9dRjfit7Mh+lnCYgYNenc+GRYMNLzyYYT4K5W5o7SG0QzD6aEb9PR\\\\\\\/yI1g5hYGpts7o2Y4ZgQhn3sxyuVmYeoOcVxTDDsuYB6qjEMtc5deXu2AhBo8N3a8mprcekZNDEAtEhNf6vmlHUWM3AuE1aISReSoi0eSygG2SwLnDP9YjyKcbPEP2T\\\\\\\/wCITMF0RSJl2aRYiJ05nX\\\\\\\/tIn+StruPDiUgORLSGJM5GigiUzEEbIohGQuheo9NmcxuXNCA\\\\\\\/KY6qXSTBaPsXrFk\\\\\\\/unWxvUUAemzS\\\\\\\/qBXsIwByAZTv8SzunW8YU4LH11aRCvQ0W+G8zeNLtqo9oQodShuP56Y1X+BZbi2qfg97hpAyDkhloy1cMI5\\\\\\\/9A4MbhbRdQ9KTCjBR7PRGwxAbGNLfgQPes8nNbioKz1iruI\\\\\\\/YJKKARR8nEEkSTy9+xiS0fp99Pnf1FVccTdF3y07SNA7sZJvKVYD0mZX2dVcswQdpE\\\\\\\/8umGRCEbK2\\\\\\\/+UM95ahihFkrSsr9Mv1oL8caGx8PG52Xh+s1jJzTO9d9gPhikxw+HiN8NqITpAsgfoIhk+5Vg+UGIa1y1CxCXpGyKYj2a1iGhpfoLvQuqEXanMgCz1VZxS72ERpbXujy91redN4EEqfALGF3jqpLfyS4lQUCEgjs1OYaPyQZgoIcXzUZ6vABvzWUujdzgHy\\\\\\\/iw\\\\\\\/ZHRUKuq2J57HYR500w2TkSTqsS9h5ibmT\\\\\\\/1SYCyODGGYtqF8TXHwDhv7o+MrcQKu9XurKtAH\\\\\\\/hKkRmv2dctv+L7AYWEG2PJoFlDwDAQrIvUDF5jDkbS3rfhylUz+Tvu4OTHR15\\\\\\\/YsayVF69Gok\\\\\\\/Mp3FdVlSKmwLd\\\\\\\/Ay68dNcmopZgTK6E+2lU\\\\\\\/RvB\\\\\\\/Wk8KcqdmVH80LMdPwqmvac+RnCypU\\\\\\\/0TyzVTKWPgiUZch5VT2x8pU6kty9dDxy8rI7Mp3k5vRfxq7hp83U4f96N34E7Q5BxaXG2OiFzaq5G2uTTTOYVdXQwdmLSWfRNShDsNC3tFZ6MyD5ykwTFeDrWKJq1+Vnxy8U5rNKo419aYlKCqsxRibqmU2cN+lvDlzW\\\\\\\/NuUwMy7xWAjFBKJLR4QhyZK9diuEX405U+ejmENlOQLwrKBaYf0kMQltzQoZw281bbIRZOj3MyItXjfFc9dpkLCEjepj65Ur+k5MA+TEixqdUjExxeiGZJV9KGcHvIwwptQxkMj0ERWOsMq4KWuAwQ8dHQo9+vkWaELP0zaiabb2SbGgfRwWIo3aqB2j3pLiEvIRM1d8D9FxvYL9eGRjJQzDGK8OF5uWhJCptf1kimjv+BoPQzcovTex+kTGILp\\\\\\\/iVlEdEqpROaX2vADRyNrD8GHzx0xB\\\\\\\/LCCO11SHyeYvgTprB8MgYNrUg4GZveAqU1fM291A5\\\\\\\/ccpJEjj3Y\\\\\\\/Dobk+DaXkMPJHnNcN74HOWsXlImor+w6UKoCcAUxuJDA25WUCRGS2y4dKeT\\\\\\\/riwy0in1UmgHLJZNsOZ2oV0p1gxJj8Qm3slH9v7Ch+acO9MPvaewjvZKbcFe1tUnDAWDVd3aNZ7I4577sDYVeXUBPFWDSxswpEMW\\\\\\\/hRmyIs6tPl\\\\\\\/QC36sDbsoWbhj8RWjZMU0lvu\\\\\\\/APEK14VwaCd+XSVQevQi50cyxwm6k\\\\\\\/WWuzlFQuUgSbxhWObr+vel\\\\\\\/67uMEUSCcuGXc3wwNsN05vJ+XiUd2\\\\\\\/nbfn5pJm13G9VP0iGiesjO++wEMaPdtfruB0ZnMplM1zDy3tJNbumM9nYGKKpUhg59EyX6G+OikS7MqZ71DrBGVCJw1+FXFcSHFA8kO5bC05zngLg5YsQrwewBwIHM6VCCtHhqdzF3zFdKf0B02pFFmiPG8T70jAnrGriE7q+H72hvqD0\\\\\\\/ABCfalTthJRE6kbH8esm5b2dcOs3W9DZwv3IIU7ZzPdRapfGEZL5OF\\\\\\\/k4qTmQNuTBXi8A7n5qltvJ7oUdnvWGWR30+zhcLjR5vj6wD\\\\\\\/nyhZjJ9VBy+B\\\\\\\/yuifX0L72mp5WFHRXNT\\\\\\\/K8LKKQ+VDjUBasTkXFYlQs53V6kUk4EH2X5LkoU5aG2KyDeDvqyDNZ1XTG96T1Ppi+G3kXFQmxWg2vJ\\\\\\\/CA8MS7yyahn4CpX7Otpyqd8Gp2kD8o2R\\\\\\\/69givT8qcMAvp\\\\\\\/W9pMnhP8rlWHfehU4DISt1MV4i5uQ+VUqwbcYzk4PI9FfYd5nrOINr+T1UgJTj5Qb5sjbBZ7VwMnQk8BHLFj4tLmbzMW4F3uNgDbELTBrysoWUV77jLuFex3IM9VZUL8NbIPnYX8VqO5Bk9EuDrsifEJiIW66ByPjwQVupxa7tT5RA2F7oOIdkfbkmzBQZdFxSSF4YdQ7zIU7hXGD4YVR7LFrG5cAP3yIS8eYVi1IbrHFbtFeIy0rDAaRU+Sy12Nrlq9xxMBVpmq1Fcz3XucB0OhVJpW0tygGD4o+Zkrs53VvogKcaWhaiBvIa7KtmulIBhu+o55HHU8UrzfLMYBOI6onQWcvkslY3WZHxO7DRaVRMcjcl6BukRJZCAs+otBmxzP3GV3QM+DQikCmT6Vhbz9+3ukSkNidCsd8qRPe3gcMob4FXsfwTGvc+rJtXLebsdSUjtvTjSZVlbAVWFCA6y+wd4+SXKcw3YgG+hlpYziALwExmy5sT4Y8nS25dyNG3dNlJ9i\\\\\\\/8YtlUfP\\\\\\\/FYLOYheyd8o0IjG7zH\\\\\\\/Cm2SNK9rGoWY44J3aJuL9TQdtsSDca1f\\\\\\\/MPb9XtNL3cbVtSKN+uzGVJTFnaBsHYoUSBURLbbB90fDaJtM+GdeyGzRyMp8STdsmZW2uIgGmSSTiwo8lf9QCu6wyowwgPjEHk1jz9kg5N29+ZPZHw0Nh+5rQmpfySvHu0FMwE7yt\\\\\\\/PBmfGpunTKidNM6wjsQRAqBKGP3c+Sx5yPsejk\\\\\\\/\\\\\\\/rrOHBGHFN85vfllYCYsCC8zTndgfU35aFiVpPCpzFWeKotNC7OUQLJN8PuzLJOHZX9UQEezOpXKBmDQO2822vr0A540NiB1fSwbNRlHMOJ4m1fdYqglezUhmIo8c5XLlbbST3cgrXrYFB3rdOh3KK4knTJg33acmoN7ychwMepKyBOlan3BaGU7hP3+XAdoi19J2YtXl3KprA84o84i1fUDuCPYNSIoZjLfxLH2p0bJiIoLqg4g6EDaLrs+2wTd0fjvMSXKPu+N9EvSNLhNTl8dnehw6nBCXUba2Cmwo+erczV61PP6Xh+aqVfLlNImiLQO6+Io6uUPVLsdnphTnboaeR0ctzd0YSZ3eGVt6CUZKalfGeJniJSceOSxCpSKYNQoktuwm1XPzvXN2Yqqw6auZ4wnYA06rwSUaGYG4g4O2gZPAQ54ccWgpg3yobD7d3vSn\\\\\\\/hdpgUXOsqbses9YFxan8vNSQDToLAMlul6AFl8aVJjRQG+K9n6F6ArAizgkLHPMWbqq\\\\\\\/VfUU91OdBcRD6y2it0Xq5ZlMe+9QF+nZypJnZ5YhZ+s\\\\\\\/N8AgfFk0CDoUL6XKLWYC7FIljhQ3g3K0KKSFNn2sTmoc\\\\\\\/QVNtZCToz\\\\\\\/3KlymMdyWUr0s9X2GY6bz3vKxPvSAoY934wtJpB805+48hmW3ZSK7e6XUTE5Xg+GXYfLudAJTbuTQNJ+0EdskEMjnNeSVkWPKcmCL4YLdThkSmDXUSLFAUup+xPi8yHCDzXmMnGzLr7XGc7g7t6rGthO4AYFAB6kbJOLu5HXNcZb9yUFuQx1Viy7NZ2xM662ONLpWQkfVu6WbwJwSZpt1SmiPmDpOHabcfNaI0yVbNFGPTK8O7cdmc1vepWcqeZRsCXin9uawukPmF+a8vqsZ\\\\\\\/Xtrlasi5l+cgA6pskwzD1jGBiEFpbV9bULL9d6KYJvWdiOXceXVXVn9f6c7H+8xDGS0XPof4q06q2UPP6n5Cqa0dmehIDU4YMzpRyL1Z6xXMsu8IcXMNO\\\\\\\/4I5Vz8sCd07WTeEI+qZzlC4aDCFll1cbk6z2Ug6+FoYUA7z+8t7vhiWIdnfBWdyMdG45GgQi7xfIDj9dXvuvTWS\\\\\\\/3sIJxP6VV3r5gZBSpaGOGmO\\\\\\\/dgoj6Ahn1AfxmgkUXy8Hih6BuMfovUcURSQKKt6cmE4CFmbeVkQ4UnoFFcL7gFlWQMF31pP+R3CzpGIWfjbsin\\\\\\\/16S2S6g96MigigFqUvqS8tUXMejA8oxkGhDtqg3q0+co0NUCrn\\\\\\\/M3uklXuczq3i008zaxPYe1tZawuFzjn6tK67aGiUr2\\\\\\\/Ybnt4Ndpvr9sxyqc0APBJfhiWKcsRKB\\\\\\\/uV6hwGhDrAuVlAIc\\\\\\\/pJnZl\\\\\\\/xdLAxvdfFF3ppN5aqLrBwGJGvy9t3Uv\\\\\\\/zUT26w5GQb4q3w7q7EUhpLzvHRdCy645OA3xv5jfw+oc50SKFHSOGqhIVqQJ7jArLyFwvTDSmBzdx20ezzI3H84Jj69t2E4wxjuRMuo9zjyfR4\\\\\\\/2VgwG+ok1QOTt6EkMDs\\\\\\\/dqdbKtdLHX9kLbTZzhw0YA4Ys+mrYflB+ZF5F56m1vn8rlyFwO6guvl0664YpBIeOT9g4E\\\\\\\/Se8zGBUyG3SfvJGBWEJ9wVXru5vxWQRXhOxI4vrCQawyqBPbV8xz6ox5+4hmS\\\\\\\/EfEY618StLRWH17d+h2ZqCj1CMYv0V6Rwen4sl7ipPfKfPv+MjWZ1Jpey7Wzc61DAVxL1Fs9Gko0nN5GHEdGmEzs3L7dZiNunLbj+pD6t3YiKE8CMW5mrrMmQknLQ91ePdqE1KdKxuyQqQ6FYX+ztFKMzlA4pWUgp\\\\\\\/+EnTCa3RjA88IzFL4cx5tWkyo522SK3W0kGsYg6wnkvYmpuk\\\\\\\/FB32LqWgGv2Q0z0EDXWzJp0emtXmoF4NsEWWeBSrDh\\\\\\\/cLRWXwM3td+RCdpPsQG17+dw6qEQiMYdQoCUI52uxpJLSVuGX9\\\\\\\/jk8mlpim425g5BBWLGU++s4g9a9oXvMEZks3aA+W3cFjEu\\\\\\\/FmPnX2fGWvF8GPUR4hFZSLP799eYO3eA6tWXOfFowfkaORM\\\\\\\/w35Uu1MmNCLZLTOZ9ET7tz6CjCZnHi+4P5YjeuFpOAdLME+OKsdJXJnj9z2UzxS0S2WBRvLPriyMPWCrtORrmn3nwhjFEvxEk418WmFGCCxVmf2bZ0AfCQPGsWh4ri2VfbgX7k9Kri3IiMfkmK05KeUqcwBXpkI4vSBU3g74IlrKPNvfSxm+qoHK0LeAABxjJkPO40aBLhdlR5\\\\\\\/Cl1egCDw\\\\\\\/EjuE3Canaas+vq\\\\\\\/577YOK8QU8I1EQ1Ljk+ble76YJCElJE8oSGZGKzCOsv0sSzBjRxlqbw1N9UMTbult3twgO2QTqc3\\\\\\\/Zyw7A5MIMlGOzPkqrxQwJxM6L1fvl17w2U5KO9xz5hN6y\\\\\\\/8DxOEgT2dusnhg7hHHD4yMfvUMlg0WnAs0j2WbkbaCgiecHdd4bW8KpRo4u\\\\\\\/XwOcnzy6ZZTHmo4\\\\\\\/oXPYfnF1P2qhos1y0uESFGGSrKwKDR70buZ4ICflo+kTj0IpjMBbhavGVUTSajHkoDjbMAFlqur7ndeAkpquTVxoCxFg9kiZIlBkqXVvLxxV\\\\\\\/qDiFhBoWAugHETLeOQ6Ne+dIdemeEU7KTexe4dq9Xa8SObAqkChcBSN6c131+wvTE2US2AmjuV2BYPAfmkGbz\\\\\\\/59FPhj\\\\\\\/yixkYePQKO8obLlP7bjLUwqVwPpkVdouLFO\\\\\\\/hm2nc6Vx\\\\\\\/5vlpHccAB8R2ARSOHuCNNEjNq9IjDquEjEpVshGMzqfDNXf+Zim3PeAqEf\\\\\\\/YQmZLWZUFeuIr+NUnRU20UWR77f9jlqrUPTKzYLU1SHP2NYM\\\\\\\/eFoGXpkY5g4DZxv5Mm5ssYDqirrFiRRpndmjI+Y9+EjOCjKcyDQnsVGGK+f29L\\\\\\\/P\\\\\\\/umeJhwcQamPlgPHsM4Gr33NcCnlfXbh7JrhSsnrX36YU9ROPGLqg6C\\\\\\\/orHFh8xMMVQXZLLT44gFqbL25VMKQ8VyGuFi4HeyCEIts8MyY\\\\\\\/WrdUrJ9weXPm8N6BfARCjhiKuFFMTPw8\\\\\\\/CgZ7IlEecw4NK+y2RaLzwqhwa7hhbUwnXnVi2QnvkwSvAJjp93SPLeYNlRuSMTsZSt1d9xEI0v09sCtG\\\\\\\/PU6MAqd\\\\\\\/ah0aB83UaTRZn79Bo5y+djJQb20yMVf3IUYrCooQIGXBDEjy+1FzVSu9rEbKkMtLdgqlgJMZjz8gDAiWKKQDmT3Ny3Byed8hRf4i6wsi0L7GziuR1F\\\\\\\/i6dZttqUScBPoF3r\\\\\\\/xjUJ6TpHXX5NaiYIf+Xi8xBNWFkIZZCax7ET0GJ5kY\\\\\\\/Oaf87ZXEmCLyzKIUDrRukS9GBr1DB1zf6Z383hTVEsEhilRj2NNErhbFEBP\\\\\\\/vGZFXoOBiLVM8mo03iYAEAcRax+67BWjB5TDHUapBBBA7w4TL8pKK3+sLFZ4YrOhoAijcfToUswPWtKovBq+M7PiHlJBkYZSKt9LtRMl\\\\\\\/1ceN0ZOCQ9gcNyzSgFDL18abgcqrfJK\\\\\\\/wZo6a+gXox8qBoQDXNwQ8f6UKo15PpYZDq+7f\\\\\\\/OP0qD4atgPh3VwSicxbXZUbBfjykeeQn\\\\\\\/vuOIR+fZ28IcQ+AydNuJXMmkhxdhcBejrjaMc8tZNyyoAQz2RQsdRjmVpwTBUWvrvtFlj17T7\\\\\\\/dXk1asqaqlPS8Ws5ZDGRAyvAt347puQ\\\\\\\/uSSq3oF6LkuRDYDjDP7adacnKjYFdtsyUdgpLxkAKk7KBsBe1keIWNuUNvLI1e\\\\\\\/3cvfourKv+4O78k61aBO\\\\\\\/03R4vsJ1Qs57Qq6ctQZcOuruPRfffhKMazIPVFSXMHpPjwxbm9nEh1PtmRQPAKzMsjVBMolT8DK99Ev6BjJokk1jKiDCAsLz1W5Q1x9\\\\\\\/tCgN4k\\\\\\\/vFQ4xbMcpxkll29xn4oXClXH5NeYYrdI+XuDyPGrxFjUC7hP6NxfE01JnyWJblJh6k84nu\\\\\\\/yo7ppjVe\\\\\\\/fM14rw1vaodXPhb6z3uZvEdvCGp4VkrfNOHHAtMO+T9RdKVecQtvaw90+14IeVQagdtmNXEoybXtqexQMopfhkossQt5u0UGlSbWhLoH5hojLKkV1Ggm\\\\\\\/gFl15VarVCQbup5Kw4KleAXLFnT75uH\\\\\\\/I9DC3ySubIg53Ocxz90XgnjmdUtwjMY7WM5dh7ZpbRXfa0PwxuymyLxf6v1gF6NQQd9avcjjDBZrGGi1i1\\\\\\\/CxF1BK0Hr9ouqXLyeIr1oR34+jJWLpu\\\\\\\/wkRjBJMoTG7ZX6hxoGSQzE+EPopX0qlTIfg0n+KUL4fzhdzOzxsdFc4X3Lk7DVLRBz7KIcDzse\\\\\\\/M\\\\\\\/YgqnCGWqW8EQeFRORqwkBx+KvFW5SD\\\\\\\/lEf+BHonyX4RWcSLgItfFzWSqpHhqpBslv2TT27v4wKeGd0MTPDCwndiiuPjJgAUylOHWJksQ2NhKeWtDgu\\\\\\\/wZI962xGpghxJED6Bug3mSzjZ1kS9hxy0K6QdRiShlA1W18pmUvcd0odbztRWpRtANy5r6fbUNj5QGU4Sl\\\\\\\/ZJ5D6iZxuNi3OgrueRmWNE9G3mSYJz\\\\\\\/h+e6EzQa3ZaHIlCUXeIWthZBjnvov6AnP1zf6YKpagfACJkbYzmyC6hU04DpffHEYkId8F6ixOsCVF4WpoKq+8pDkz1CODK0HLQiVjud3hyqaE8+i+9bnccn0XqEgvZ9lgejetEYmc+8knzfYb97G403drOdRZk7VG6yVWrke92LRx7d1pwcFUBrcJJLya5y7pn7GsaVVDGpd\\\\\\\/0F7zghfRhhc1L7cFhRT16VErGKmijftWmtpKIlyH400SOAEVVXLLdzO9Uk5KjcuKbRC1DTyO79soK+30BHGTMUfSWkhvYITBHLwoZZkY45pQwVd5Z1fVXut\\\\\\\/6ioXFYAJ\\\\\\\/lVR9WxJ9U3rQzPRs81L3htWbY\\\\\\\/kVSJNixUi3ANNZOB72Um3M53DyCGZFZfsNam7n2MF3y8d0qN5uLUqhTwVNYskhnml51mjUnbFHRp6+Erl9apQRgb2QN3C+RrQSUdA6uE9UkTuxhPeBbjkjnW1tB1+Kx\\\\\\\/2O6VasHWG00DZh0EuJLYI3NJPotjkAO+yvPC6ZBGp2clXTcENQ7CjxfdhxSU7sCFULxznIWZTywOoZw\\\\\\\/Ur9\\\\\\\/tzHiprfobiIsUHAs6BkyGaVBMSJBC57bJxBA1hjdtgUH6vNaLNTS9xjx7sAcdQH\\\\\\\/BFgIWOec7ajIeqXaFod5Xk1vNoihHRZCHmg1+mHpXp6ebAmtZ92UeIN+u76BnvmNUhKPFv3AsPwwLu6EHa1mcu8ZjpnhhRVv28X3L3rwe0EtvPSbRR3KrOS08wWwudaLLaqjOR179qdDwFPbkuQO0Qo3lDz73dpH9qE08LH3kQ0vVeez79P05Ji40\\\\\\\/9fuNrY5sCdxxNrgwUDQGa5PuQ+OE59oW2jLx3CGncC\\\\\\\/x7oDkN84R3jwYpOEOjleQtU6dBueZ8ecc7ALf+71Iti7VMxlr+1O3iX+7Yz9+CzVqQwyY57GLy4ixOtkB1Qi6W8lQcZp44UzpecXE1UwEvrdN0ufXdii8ebki\\\\\\\/y5RQn93xgQPu2R8L1m2eeydYCeMV4EDOTcTrLPCl+\\\\\\\/\\\\\\\/\\\\\\\/FARH6IpfpOPqUzEYYS0cdQD4V7wa89lX5bwVOgJD+6hTUS7LDnyJKtqcoJ1fx8QDCYsJHOTawX29sQ1aJxf\\\\\\\/xMbUUeAWZna5e\\\\\\\/vF5\\\\\\\/C3tHqTnv0jVS+fowyGd7GOQDyX6tqVXxL+dJO4z13BT6nRf4H86M2C1KxaWc\\\\\\\/eiRY+JHmPG\\\\\\\/XGmg4311oYBSsvb+aIazR2Q6wMG+RbvrYMIRtC1kxOCsXAlAvHYehHlMVB39OwP2HG4IFR1+OlPBDWITp8WwaRVTBf4q1B06eWFg\\\\\\\/f9PGiR4VRu5yyl6bZ0QWKsnH6WyhoHzdko\\\\\\\/1X6sfA6n\\\\\\\/gbc\\\\\\\/GNy6BFSJ1nm9IasP2MdYpYmWg1VbMe3AY9qZW7kUq+Aj23eI3mzNRKcqlwTcpSqWmqQQeUOJdobvck0hxD94SkVuN6+2qK7hud6old48MnzK08sHIzgsrjMn3MBAyWtLEcwtxSWOmIou5eftEOQJelXg6h9tF0uSfFlvegDdR0SgVQZI7DMprlRBoRYFz7PBumFw4fP1bMN\\\\\\\/ztQi0PHMVecuQ8gkB36McOpoBuMQHmlq0ZdmRM93ANUEHlQVpbdaDhBr4vxoNkE9BNww22yF16GkSt3ruJ6w9iBdIUm47zvhLwxMD9DLCZepVcUttB8aAlmetTprSbHPvvB5nxtsrlfyV5xDpqJTlMk1T1Bjc43JWrWsKhTRpFEzQUYKJoUeLrot7V1PdWXzBjamuoWgsSUM2T8xL1RcJYrwv8vy35D5fobbm\\\\\\\/RVxIjpzNLwZOn5lCClaePL4Kf0d4dyQnC2yWf6PA4hDF+Ni4Uq1sE5eStSXatXzz3YeSnqrBoCAaTbNQ7+qRl5yMPYfaA5vitl4huCLECkHE\\\\\\\/DAR0HwI\\\\\\\/qDC9XPFkLx8GdofTCP1mqalX2NrkDpvWtgY+6RmVj7N4O185aN9XPkeTTZpLgVqlZHgu2cHOoiuTcr27yxdMWvTSjl64gJlp6dPhC03WPQ2+W5Mc2Hq9TuKJGGvEzAVolL+2aKpkpcXIrn70WBGOWxk1VYGx4OE\\\\\\\/fu0R33tTrSeMgNfS8vfma4tF8qpMlnn0xf4DsnzR13VNRMiYWsGd9RaX2i+McId46cTvAh7ZzKc80mafVAtTjz8MTZbhfTiKqXgS2O5rKXdcPDEmJ5lTYaDNh0q4WXlbqkM7kMOZ1+C5066PwJYa0FJ1H0ExWTDKZufx+1T95RfOyBKbaUFT8v3fnRoZP27wRhpjw72dGGMAHHtil7npE+ZJS8sc2nDlKZGja2bpxMQoES7HOiWMIEu4JToLaGFjPa3KW9EN5QaeyiT9pBeqNn0ItMRnEr1mrDmO0uCNR3Zpm\\\\\\\/+uxLojprPRG35HuHvXzv+RJovyl\\\\\\\/LYKwrr0GNJ4AKc2mAb3gILfAHibcUhWP5NCzdDhRlVK4nXjckhFuOhLV71eMsz9PpXEVSBRKK45bynYF7D+Wjy6v5G2RshEfrw+\\\\\\\/bc2jZecL1U8rjQCnehrkjf6A8GsQQAIGfsxUk+nau4VjaEFNGVaZxtqXeB15kjQ3i5V15z1P040s0FtSLe5A0JRtMJZSUQ05Oo4N8A9LkZNiI+Cix\\\\\\\/VVbs0H\\\\\\\/OBYEkC50+yWw8cqhHa8Bm4fVOU+3bi4A4kzmZ8GGNo7d9bLpZAyZsbxOizwtYP6tyP6uZGKFJAutwMGHu9gb06NxNkB67AWu9Xvoo7NORW\\\\\\\/DF+LidwX5XETMqiqyFTMxJJcU4s8xcrh2xRzevzplTEHdDZ2R4ko\\\\\\\/eA\\\\\\\/98FcL6MaS7JAYT5bldk8GYMSI9XLTpWJrOwFzsgpuIEZZ1uY1kFXJxM9WzXaiWRGsTsyTe+0C\\\\\\\/1Wwj7DaAn\\\\\\\/3B\\\\\\\/U14Tlw0F1Lyg1qGxRJHZ1wVtfVFj\\\\\\\/aAVdwHY0Gm\\\\\\\/WtoPmovICu4jE9NpyYQCYfcmAgza7wqj91RKHguAvl24GEl4dOlZ5NvTPnn2NFqcNgSCqkACqkqzLvsXC3hO9GPBkw2uevBcOaLniEJ6It\\\\\\\/\\\\\\\/sftNKC8\\\\\\\/8UbpsCC0eRn8RsVxdCQocL4jkCvQns9Jm3Tg+0nwDjGJJNrgBKHwZ5xEwo5fP9RNelqVuATE5a7hk2EcpCm3+PkQzQ+Mc9Ec12sEMUTkremiCXzKBzOk0dig+neOVZGzr+8rn8BlbKAOsQdBHXweeq0eWQ7ideEFx1snz8xY1\\\\\\\/3WAzX1B1RAa47biVKspnrN1n0UngERy\\\\\\\/F0xuqXGNLYesUifWEneLPTgxsh7Wwy+Tre9CWiRFjZSKD0oeFxT\\\\\\\/oqpyYkY4oIiZZWVBXr6JpOShBfqshNMQc+RHVFDG0vhYmjuAAayuO8ByPhMw\\\\\\\/vBhMBLFeCEb\\\\\\\/evLzMg7whU1SSJuezqjiNGI0j5zBNXL3hbslq4Sp\\\\\\\/tlRIPykk3taBSFMWxwcLg4N7LLUmO8CiJhHW9\\\\\\\/+i7EEZeXR0QYTdZw8RYbVPpRSlnx4cSYpmmJXT\\\\\\\/bNKNIh+hnF49DIVmoEY5tfsn1G5J31a3qqbmdDW8rKo4Gp6qOQZTl+BWqRZzENVVToBRbbtbPda4SswdgHy5076IyIjMsZq6dlES5FcFK8uREoXF52fHImf\\\\\\\/TEvd3\\\\\\\/X3V+ADRASi+YZco3u+9fST7A4ogqSe8cvC3huqHcfQghK5TISpnR1K8dhgueKTT+acV08AFo0V8gu54Iwg7s8xPr1HAlhJsUyq7906fUkviq4kMetMYlPBT2pboE\\\\\\\/3woteB7VEM+\\\\\\\/FvQfX0HTqexUqEmSHez3u5I9+7IqtJoqIvNJBNShDTiHNl58MEDkllgegNoFPXmT4V5hEST8pqbS7FKUdNYFxjjgtgL2C3BisBVF9aa4ya9iFvHtbDikUCesp6gkoS+ArToPY42o3\\\\\\\/ZGfaSOQTZV2mxLENY2ZmKkkrrNqEOPBzzS6nRuasqvHIHcsHuMJHZgeJy4sx\\\\\\\/kXKTPZMzgqDM\\\\\\\/FjFBW5JgMQSmQCOuWkl+xTQLeBVRJWNH0eFBcvwl\\\\\\\/g5OvGn8qZ4sJvPAeXB29Pthg5M0SB5fnwpXjHhraAu2f0rT0FO5T5aTo510Qolv0sb8NEA1xU8BOHCfzQEX2Id+pScvrCUhjtwGPf96eh304Dr+u5mBZOXbZ4meZ5n5SbOEjS6EAGDZ4uSFINasIgE\\\\\\\/dNDgwT52+QpkQu6ROhegsERur741SJo9sa8oE2dhfy2AaOrR+C0GjGCHcEsWSZXEWV8xE1JPXlUyT6oAZGp3Ppu1lpiDqZLINvP8ll1VDVFGlypBGy6SyeQkCpY7jLkL21w5kH6wkZCJ6ZTQoxtiTaVOZPLLhsSLIGDgP\\\\\\\/32FlodhrT8f95z20YNT3Nv\\\\\\\/nM3LW7isjiMb\\\\\\\/h5VWfOcAtStC5ZSSRUMRq+yc8\\\\\\\/dia2JbcgRJlVK6VC8e4GQ9DIOHmAJKw65gEQJeo0NcdskIYbfpS6goe2ciQwXyLKOSF6nE9EL00\\\\\\\/LLFDFeIZIndX6NGUkdUnNalcMr6jZF9I0dSXmaf0HTA+daWRVbfnRPSGv\\\\\\\/7YH5j7w9A6frtaxSJaiofrie9eK3v0PUZmCMttFre19qJgyqW21vHIGVbaBkHluDtkJSkr\\\\\\\/ipKOuJPgTczZ6ajtQ\\\\\\\/9BOPmZg2HPdpBnIjVMGYea3I6\\\\\\\/22DfZqdsn2ZmpT+Jhn0pZbo+k42zfrJfOwFcX8hnwP24gmgAC\\\\\\\/zeQoMeGFu7Jm4SHtYyuXC7rjSB+Nsf1i+\\\\\\\/ReqI2ZEODc6d948C5aEydfG9SAGnpKpRMBNjA4tkoOUNBmxqUEutknQtxozf1zUq4fKuXaaFbbKko8aAHF8Uf9GsFt4iiQbF3jxYguswEddCnMTZycjznKKAI0p+EVCB4F1qNOzoQTWSGUXp77xIrNL4+Z6909r\\\\\\\/3ilWTLraXF+1AmMU\\\\\\\/GTvrUhdbv+xiiCZLIXjfuayF51Qq6+nCmDBzrnE+zJkxEDvD8DQIiLD1nV0kEhlCDf+Kq4OAqX7ZzWKQk9EZzWKk7ZZzzuSbtBNKRlcBTcTUslqw4DFuy4nCxq2eFQoXuapEs7caIzAyJJJwXy2hdPo23p\\\\\\\/Ru0SCtI611dH7aViMlA6n4ta9pL6ycZ1uUE\\\\\\\/7Zu\\\\\\\/W0bVrEfhHBAPn6wN0NexRuIT+KTMuiLq4sf4VvKU1z7ffc1I0Zg4++iVtyBJqrCppwduYSqt81CdhayNYzXXJjpz18A8YdofZQpJVFuVfEalP+wwLxvTXb+5BSE0P2cRvW0WScGcAdm0xfdPIYHhu9UMtKxJtD77BzE54fycK8LqNlYIJBCPvqNNavvVHoGDd1lcnCyPfYec2Ln5WEpIbM0dzH\\\\\\\/aqsini\\\\\\\/uMNA4UhZlsD3roY5rFlsDkrAa8pqxV6Q1ridgwBb603LbZ0TdzZt2biTbic2cdD8qyB\\\\\\\/1zzC4yXRsNXuP3HyLrcLhLdFFW\\\\\\\/PtETBOqH7e0vhYQF3ifkabYR0nGHdIpRMBX1R709zkMMo9KdNa1C5nBBtMjKDnDMfEJGDIXCqzlJGN1MkRMFIDqHiOHH7gljm8Uisbxte1wg9yemKg1c8W1KNzCAhibXSK4Za9hX1YxqtkDr3QqVo8OQMx0RO0pNzJiwd7JqHUKR7Kzo8YEuORrxm9WMyCs8ByowT8paCr6E1O31i+GhmCD7gWx8DPa9JVwYh5S1dJzf2KUnRKA2KtE+qq09gOFJ86qXUAunzENNfwaDfFm4ls2VVz+EszUlv2m+gSxyPGzp7OY3SbvhQ5I66oFqvKQlW\\\\\\\/p6Zz59xMx0UUQVdvqs4\\\\\\\/5fA7uPmyxsEo23HCw\\\\\\\/vvttrwoCTxmxm1ujbpb0ju5GJfMP\\\\\\\/jSKCfJBAST7l910sr1LhjhjJFnnddQfJ8ZVA2ZIpOwP1bKm53Mw56oD\\\\\\\/ytSadz8HiX9HvECY5RYaoNCJBffFmFr763xy\\\\\\\/6Dn2g9C144njM\\\\\\\/dtRs\\\\\\\/oAmg4b\\\\\\\/fc49PTBg9xDYhqd7NPPWdbKQjS2PKGiDniyqKDkv5wjA88v3JN+N79pbi1ekCX72Yxf5Qx3xC5rm5iF2dfCIJYdu3Qx0zEiMTvclyW0jeEU\\\\\\\/xm2YCZ43SpA4xRqk+sowGxUyA6zkzukLzglDqZc5U6OBtd0SJqCVq5+8e06zruI+ZFr1ZNRilqkkw1Nl4Vs4f0owdWnFcZiZcUR\\\\\\\/kZAgFcgM\\\\\\\/Ux+\\\\\\\/Nq\\\\\\\/nWwTXw0ZeDH9EKdrMeafmFknVAcs1ZBPBujQhHdeHQNWOqqQVbamedfGBarf15p7Wg3AJ5GJYdnoTpMbeGtTkChYRWUOKnmN3tNTmKUDJ5Md6vTvEouUxgZLo9Y8gkTDbtwjSNw2TrCMhyPD1F+I9ZEkwWWMRTCjfTDETZlcsPbw2zet4GZ7shhDAG13nnJCilS3QBHoPvJ\\\\\\\/74zkek7yX3op4El8ZlsY1FHAPF7wjpyS8jjCcO3e7tHuEQwIl3REd00\\\\\\\/TjL4emaqHGS01VY\\\\\\\/3dKKvm6jdbttYWD4k0Wl0FQa6e8yW+9pFEfL5Amj5Zw63aqTR2Vy4sQqwJBves5Rm53C4azoR+qFM5eaOrJVBI6cSFT7hYmvxHHImxv2vV9Ha+R6cSG5LKlszPtZ\\\\\\\/gyVwUdJ4exmyOBljF7x6iFB17\\\\\\\/fwBWrhhKArfpjz8ZFZzGDu5A5YNyrCLZsCk3vqMBer3UE1bAt3qtDK3PU3TXQkq9iA7\\\\\\\/Ud\\\\\\\/PTLkZntKNYWy3BMfP7KLFFGorFNJ1D0ZifNZ2eyh0RVOTG59jL2bYty5NomLS0qV3yZRPB\\\\\\\/PVVmFdJt1z1ww70aB6aT01zt6\\\\\\\/73yoBllJUXFkktnujppm6ST1TPZMdgjPNIjMztaR+YKlplNjuUhF5glN3foprg0rcDJYNQpjFGzdNMs7F\\\\\\\/GOlRAe5QWyJLTT7nFfw2iNIRDSkvn1pwX7+xVGfu6W++n4MyTgI2HQF7rW6CLDQyzmYFmYhb2Zj2baCmEr7uq6t\\\\\\\/9bzdHsiz5UX7rYwx\\\\\\\/6j5GL3unlClBzHItrjiI8ZN+6\\\\\\\/dZkwqE\\\\\\\/EL5w\\\\\\\/iWLmr+QFBIbxEb5mmxqTJ8ZNGvP8nUjVKP4lzbBOMsGCv1AejdJ4U88M1pb6VCyjAGySSSfflG+VfxyFjtddft5zKeRjM\\\\\\\/09OKN+IEYwwgLclZoz7\\\\\\\/C10uORXZokCNhzFnV0QDUBHwaWUbh6o0ObH8\\\\\\\/KoxV79x8mr+Ql6m6H9muhnN0txb7Ub4sbJVEcZO88\\\\\\\/Mhw3i8k3xYA\\\\\\\/SfXbx\\\\\\\/kRnRWpDAtBRDF1xakYzWv6MXL1oOO6dXyoO89k4JP71zINqTMXNkmZC9ETUbYWhZHoLVtTm08BzYMUC\\\\\\\/ETJu3dHfL81en8Z1wLiUvf\\\\\\\/EM+lJMNlgEc3jS5Sf3RC8GigoiXxLsN0SmM3cNq8r+ZjxY\\\\\\\/U82wz8jsNNXyZPoGseylVnzI51YjuKrEWFcEg+eUAimtRlhBBycLzsCqixr1sdq3NqrJUMWeOtnCVryK5BGlntqu1qLo+1naTrSjPbJgUMuUD+4K8yUwuaKPsCHDzaWtCVlubpwuFbvqi6BrwuavmgVSFPb+iyFrO0z2WIdsyc6FrxsYMOBIS6rSc4mFh7dTa1YXYTW9ZwP8xS\\\\\\\/BvNeYJX9+qyKJqop\\\\\\\/PRsAbnYvHiVW\\\\\\\/ZjkN0e4VL1EBdk8YmZZLWRJQ+wVF88msxleP1DgWngZRqGVg2A0BNVurzi2gTD1BxIK5oOf\\\\\\\/Qz59A9wnPo+ZN+4JJCMxWdBR6NfdoO4X2mlbq\\\\\\\/Wq8AgMuiTVsg30N1dKrIgPdmqPlIDczquQDXtosBibBYyr52IjpT79OOImwpi87udUku9Dy2BhUvpL5GmiksU5yMk+1NAXLfNG9BEWmVCxsxwgZZNtaZ0GCIgSHWACJd\\\\\\\/xVWK1U3DoEKN4Nfogg+Z9xDOS+nhiwFt5SE\\\\\\\/zuCT2IaSKDM0exK\\\\\\\/AHwLDQUkPAClZwWmyyfUd9f6kdL9yCoSE4GSMUjS19Hm25TdNojwJ1rZKS0d8K4FBxkqd1zRTMkrgOu8e0asy8PBShJ5mTSsV2xxtJWvVIP2hdqw56Y2HJojTCQmkf3Na0wHRdj5SYG+0DiDRKE7J0MZaWwLRqplrv5s2LTtlcDQfHUUF3NZZ3KBeJEO3iJFL9UU6cLkpZqLmSBApo7pql6nHcjicWMx2EnCDk69DhboE8d2ou8KtY6OMJ9hjNNfvQ8WCr4fLDBGSPC+x4zXXdYK\\\\\\\/z0p3W2yxl7Gnz7UDC1oviJSP2n3yzP8HE5vBxDXhlZ\\\\\\\/hGrC\\\\\\\/DsAORXLtbqheg7cSP34TDWxI4Lbyvhj\\\\\\\/OLrpDbTjADbLt+o3BXtLRdBaGhU7+BhDnaSHDbFgSkeTG1n6bHw2N6unGtdpHPIzQn\\\\\\\/syp9UpfnPnYmBfjsghX0YwsQ3fzQ2Vdq9TNXjpHNYqRxufjnB9gteJwAXqVoygBIHgYSzVTfk48M2eQZZoQgsKNQibYmpgdISa+vUxBCLOtYmDnRMRyzlOvHpcSujsOngqCJM61NK63QLn0OefBDdkry3ReOhkoj+yLwEL\\\\\\\/6gpVkkvkoYMWNMdL9HXMGuRNNJsXh+6cUlDZ9OBKQgUiKJsdhgUR+nOTahhuwYwiBO5BeDC+kQH2jLG8\\\\\\\/oN9crj0myuacHsQYa6kAbZEstDhSgov9tc1pa28+QEwb+JNfBBKKfR4ty\\\\\\\/nsmWmrVQz0CSQI7drBQPC1Rcji4aiOT6DzonmKwG\\\\\\\/X1es335u7CK8IktsmuOZqLvxMsMqynkVSEc2gwdxxbBCPZreMRvFx5O6bU10QlfEYFHHwLvLNlZT3VS9NKX5Y2PI9NAJKbOz4CzVhzMzV+\\\\\\\/xWk+H5YZW1K9f9NRjLJPU2ejZfbNTQRcuwC2I8AoM77ZDIFyQLY859jF7gMwEbMEB7Z8V0u90plU7+IewEOQsbPPbLUUylCRQdHhJh6t8PdtCp3GKX+P4su+u\\\\\\\/LpGrwrSCph9ZWFg7WdMDpkYfsyJLWw5K7FmSbvTSOx4di0N6NuP+VfaCc2K+B7K2WpMoNV5lj45o54G4KrWqV9MZDeqkwp5gEMvoUx4xJqDd5gDTmMkoNfMoxJZFlZhTsgizR5AhSeHOpLFPR2Jmwb8pvqyGfZnHav+8agbEVksaV1nWH+98hs9KPZv4x1cmN6NtsMC4H2Vrm5MtUzq7o\\\\\\\/6TrmhZQPGqnI2Q8d\\\\\\\/oC4y3560WzFwrRC\\\\\\\/XZKXJhzQgzTUR5KgV6Osyc+IZtXxHQmSReqMb61XtRmWTW4Rf8h\\\\\\\/Jqsz3FMdUXdJB7x6SNAqeg5BRvucDLB3tKFoi5bRNZo24krQmpr4OiW1nvu4PdZwFqRNCe79ZJLVb4AEgWuYcj0kXP9zjKg1uZjYMyygkybepvFBSDKzCNcifvVL+D3f3XHBbJy+hrFZk1eyPcqXUIP4ZxYtV1\\\\\\\/eVHNk9svBK5p3r79S5uyuv\\\\\\\/k+m1EWcEmotUq1mK9v+74TbXFIYBL7vWDy34K2J9ICKIQL8PoJPbISf6hIeJqUtGyOr1\\\\\\\/eQQ+tuNWgwxNnj34ZgumNGXoS+H0mf6AW01NwsquG08gVGSpdLAfnTkmEXR9CIhp73aaYu9Z2U9+JCRQ4MriWowhp12HTul+Zb5FMZMmLccPt9KEoWf0Tu6oIHXzxpUCWnsT29vcSRv+SHG3AjROHeKczXPHdxhEHr0potEmngM3sZO4TmZsnejQThe5gSRvFA92EeQuHYba6LBb7bqgIg5ntHZ2i9AeTKmZwsW2+\\\\\\\/UUA\\\\\\\/SquvUZ6der8JQt0JkawKpoxePUA2UXw1cZPs\\\\\\\/UOCbpc+rly4nCX1DVIQH1uFVn60E0iPDCI1BL7lzjx0E3srdB0TqCmN2aClPE9VcDEztQA57DOXTSq3ReN+J\\\\\\\/C1omVSCnROlghvW4ElPxDItR0WCI+E\\\\\\\/D0iutGz7SSGW0LdRG1XlQ4zWMXS1M+zPYzlnFvNv8dyBY9VxAOcVV9eI1zm2jUsknQ1QJfSZUtjo0pjKrwmm6iCtlQV75FvAFLKgLyTeSz+4rYfaE0pwFefp5pceYXzIMN9py5I8idbdlzrLS8rJUedIr8Btmo01lW4P7uHYroRB3V\\\\\\\/v8WMB9xH+8nSPYramWF0x9Px2\\\\\\\/OVyH6hbaw5wyau5F+O6bpBETcr5qb3r\\\\\\\/czTPpY\\\\\\\/1C5nqW+OPSpUfzm7L0anMz2wjfsc2QHPdrtaYFD86jF50tpoqfx8nHw3OFKqt6r+Njw4IA+grilzDYSlkLxTYQZBM\\\\\\\/WjOREWsmPs0BIFjttgWpU5awn6KKq3jLvgacflAeOGBP600QPQi2zIWUBLGujYDMPc58V36LIdPK897wYwGO\\\\\\\/TIdXZ\\\\\\\/lmcdJBUjVK6AATA2XgQPYAHVA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022dhTwiriy+123WpbTa7gRYg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228cbd123d37d12e15a4e0f156c7be992e0297a054da4090af79dde20feafe279d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KBOmWVRgkwETKzvDMxla\\\\\\\/TWpcHGOOSfOUIogg5xngM\\\\\\\/8aqMHE9ouHtz4Q95nCWS9jLnF265CWQMnJgU1B4Npxpv4jiqENe2+z\\\\\\\/glJUrqSOi26yxA9UPUnpML6t0A5yzihBKPtxAp4QOqj4ZZElWRY0vo\\\\\\\/aPuYC05kHCQDlUkfD\\\\\\\/oZpKUYj+n8JHMGmAp4xCPYWDDiHkZL+tC98SNSjIyRFq8cyd3zXmJW\\\\\\\/xYC3IwOFcigZNDpFeFwsB6EY00Lly\\\\\\\/dNSbIAcux1WtVpB1h8S05UzIToy3tZmyf5We0\\\\\\\/cSRNG3IkPKy4SJER323lEuztddbfZQ+pEfbWT3bf21tJNtBmmXkq3TsT7orwr12WPXQ+zcI1IQaQWvEolbGw1z99ND8QXDO8fTkn5PA26DycxunI2iiQodn2lw6yngmyoKsTXQgGC\\\\\\\/rtwV5ZsRrJcSxruapfF1CV\\\\\\\/d0NW1y4pUjFgX8csiH9oxxyVbaQxjqmjQjIY4VJ\\\\\\\/hEMaxu60zxz0GXObZ1r\\\\\\\/pQRIm1A9qSHRXYsPmG8a7NewfFEUJJY8k6HxfXNPHW4kRQTGUoftp\\\\\\\/n5DWCBFVs6ttmaoH7mfV8U8kbxYUUGmNbLR9P\\\\\\\/WXmw9PyyKnTG+fP0xUFINHzIkPrzE1DZBTqblw6vtgAqul4Ft3s8K5d5iS5RHmMJhFGWtEJkQIf8A951Fs+bbeRk253Y78y\\\\\\\/ZLaI1fChRYhxO8b5nPOLNs8cThQFUQzrmI317sCWZy6PzzERYZwU6i5OEP4TGAhfNIz28fSnCxlcZkkRr3NdtNQMD4U+H45b4wSL7SEtpViXouPPd2+noPHfKcaK4y9reT\\\\\\\/nEOG0SPglNgSGWdclJQzhvEO2RglwGUBMZm+oj0K+BW2c9IBe1HJeibSEoGbACLL6itOy0rTB0ENhag+6tBEVlJoHJniwE2aSdrxgudoqpoCH9eWd1JWyQMx8UPFf1kr8a8FNdUHKVJEIYeV250+uoH2s+IlBab5EGasQxVNpg2ygKtjV1bn7JNNfrvk8Hg5FErZ7dV6x7+KwNgwPVv65sLI+syAYvzIUstket9GarNslyWpvqyKeQ1VkATQD11fqkkolbJvGnHeC1o2QKR4EX2cdXwlAk6QmxJXO2dkhn7xpITBaOSNP55XlHuhUBaTPyH62Rz5njzK1ELR+EKdU8+t+7SHA9QQjqxsxIHYpCwMlRkGDzlg3mrjysvuJQxgnzFEI9yhgAWuO8O5NeRYBHImSRvRy19e31AFSQcenHi1RYpvtsqJdh\\\\\\\/o8yh\\\\\\\/wi8WMB+ZlJmIw3yS1FI71bGyfNFbWpqXcWTh1S7xDzrBi+0BZjPqiJGzTXN7ulU34gK1337Xj7VPp52OVadGodUKdZfdkw3brvgYaf3RWm3AWQC40UqpgVBpT7EcULESF++KwOSd2L9TTDwQZ6lLb9jxZXaOc6EcT88nnJ\\\\\\\/U1h4nULa0\\\\\\\/LfoWb8I8VzZ1OnljpzfrNZhvQbH2GeQc+YTJd1FUZaIfy1otvIPfygXC\\\\\\\/y\\\\\\\/WLMUsSyyCdbPbz6JQPSpiPGR4wXWkkgjPxIB3oBwJo59iswPxZdc7LcKnkGKeifSmwjOE+TEDk4wp1b4\\\\\\\/a318mFplSYsWgRMTuXRCRfeCPyVuhPNzUdwuP7geXNAhS5GFjEUPGkJLKjDOHycWE594b23YMHZLNOOLykf4sXDFSv\\\\\\\/8iM6TwnCTKAPuKOycQOeiQkOeCXSfWYORLDq0Ke5uGu+Ua7DPzyPuI3tLDIWZF8mY9uHmbYWE6x8YXuvNm2\\\\\\\/jHb\\\\\\\/IqFLWTRGeqra0P\\\\\\\/22eVp+4RgiFVpG7B1IIrBkR\\\\\\\/WkQg+UdM5kCZ\\\\\\\/STdWASi09FP84IfJlV8wDvOgocbFUfb4H4z8Zk3+lt\\\\\\\/g+UxJitJ1NrPGee7pq5Ou15k8KY67hbSrn40we3PsP8uGYzf2pmoeiFwyW\\\\\\\/z5soI5JN\\\\\\\/M2uMkV4WKYNXAz0vcT4BCc8l2eMt4wrMQASRep2sOEOI06RiU5yIdsIrSrC+5+AJUK0GQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Z1NKHKx1WJ+kSqX4UAB3RA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ace10588103ffc9be9c71ef054836ebba312c75e32bb1e5f4dc72ba3fc255629\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459613 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/1/11" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-d1VK5dYWvuAdpopWmjlWihmUXthu97VvOP1BQu6Zetg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/1/11" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-y29wbQkm2mOb3fFHB-CEAou5icbLMff7VCLyw4zzLPk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/1/1" title="Genome-Wide Gene Expression Effects of Sex Chromosome Imprinting in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/1/19" title="Origin and Length Distribution of Unidirectional Prokaryotic Overlapping Genes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459613" id="node459613" data-pisa="ggg;4/1/11" data-pisa-master="ggg;g3.113.008565" data-apath="/ggg/4/1/11.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Design and Analysis of Bar-seq Experiments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">David G. Robinson</span>, <span class="highwire-citation-author" data-delta="1">Wei Chen</span>, <span class="highwire-citation-author" data-delta="2">John D. Storey</span> and <span class="highwire-citation-author" data-delta="3">David Gresham</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">11-18;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008565 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459613" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David G. Robinson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BG.%2BRobinson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Robinson%20DG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BG.%2BRobinson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wei Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Berlin Institute for Medical Systems Biology, Max-Delbrück-Center for Molecular Medicine, 13125 Berlin, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWei%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John D. Storey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BD.%2BStorey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Storey%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BD.%2BStorey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jstorey@princeton.edu" class="" data-icon-position="" data-hide-link-title="0">jstorey@princeton.edu</a>
<a href="mailto:dgresham@nyu.edu" class="" data-icon-position="" data-hide-link-title="0">dgresham@nyu.edu</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David Gresham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Genomics and Systems Biology, Department of Biology, New York University, New York, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BGresham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gresham%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BGresham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jstorey@princeton.edu" class="" data-icon-position="" data-hide-link-title="0">jstorey@princeton.edu</a>
<a href="mailto:dgresham@nyu.edu" class="" data-icon-position="" data-hide-link-title="0">dgresham@nyu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/1/11" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459613" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459613/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/1/11.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459613" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459613/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/1/11.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459613" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459613/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/1/11.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459613" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459613/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/1/11.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459613" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459613/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">High-throughput quantitative DNA sequencing enables the parallel phenotyping of pools of thousands of mutants. However, the appropriate analytical methods and experimental design that maximize the efficiency of these methods while maintaining statistical power are currently unknown. Here, we have used Bar-seq analysis of the <em>Saccharomyces cerevisiae</em> yeast deletion library to systematically test the effect of experimental design parameters and sequence read depth on experimental results. We present computational methods that efficiently and accurately estimate effect sizes and their statistical significance by adapting existing methods for RNA-seq analysis. Using simulated variation of experimental designs, we found that biological replicates are critical for statistical analysis of Bar-seq data, whereas technical replicates are of less value. By subsampling sequence reads, we found that when using four-fold biological replication, 6 million reads per condition achieved 96% power to detect a two-fold change (or more) at a 5% false discovery rate. Our guidelines for experimental design and computational analysis enables the study of the yeast deletion collection in up to 30 different conditions in a single sequencing lane. These findings are relevant to a variety of pooled genetic screening methods that use high-throughput quantitative DNA sequencing, including Tn-seq.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li><li class="kwd"><a href="/keyword/bar-seq" class="hw-term hw-article-keyword hw-article-keyword-bar-seq" rel="nofollow">Bar-seq</a></li><li class="kwd"><a href="/keyword/galactose" class="hw-term hw-article-keyword hw-article-keyword-galactose" rel="nofollow">galactose</a></li><li class="kwd"><a href="/keyword/functional-genomics" class="hw-term hw-article-keyword hw-article-keyword-functional-genomics" rel="nofollow">functional genomics</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/sacchromyces-cerevisiae" class="hw-term hw-article-keyword hw-article-keyword-sacchromyces-cerevisiae" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Sacchromyces cerevisiae</em></a></li></ul><p id="p-4">Uncovering the connection between genotype and phenotype remains one of the central challenges of modern genetics. At the same time, the rate at which new genomes are sequenced currently outpaces our capacity to functionally annotate those genomes. Addressing these challenges requires efficient means of quantifying phenotypes associated with defined genetic perturbations. Methods for uniquely identifying and quantifying phenotypic effects of mutant alleles in complex mixtures enable the parallel analysis of hundreds to thousands of genotypes. Pooled mutant analysis entails the use of either libraries of defined mutants tagged with unique DNA sequences (molecular barcodes) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Winzeler <em>et al.</em> 1999</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Giaever <em>et al.</em> 2002</a>) or complex libraries of tens of thousands of unique mutants generated by random insertional mutagenesis. Analogously, comprehensive libraries of short hairpin RNAs (shRNAs) enable parallel analysis of perturbations of mammalian genes in cell culture (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Schlabach <em>et al.</em> 2008</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Silva <em>et al.</em> 2008</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Sims <em>et al.</em> 2011</a>).</p><p id="p-5">Recently, methods for estimating mutant abundances in complex mixtures have been introduced that capitalize on advances in high-throughput quantitative DNA sequencing. Barcode analysis by sequencing (Bar-seq) was first developed to analyze libraries of thousands of <em>Saccharomyces cerevisiae</em> gene deletion mutants (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Smith <em>et al.</em> 2009</a>) and has subsequently been used to analyze a library of deletion mutants in <em>Schizzosaccharomyces pombe</em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Han <em>et al.</em> 2010</a>). The use of Bar-seq enables efficient, accurate, and comprehensive genetic screens for addressing a variety of questions, such as defining the genetic requirements for initiation and maintenance of cell quiescence in response to distinct starvation signals (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gresham <em>et al.</em> 2011</a>). In organisms for which barcoded mutant libraries are not available, high-throughput DNA sequencing of pools of transposon insertion mutants (Tn-seq) enables multiplexed mutant analysis. Tn-seq was initially applied in studies of <em>Streptococcus pneumonia</em> (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">van Opijnen <em>et al.</em> 2009</a>) and <em>Haemophilus influenzae</em> (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Gawronski <em>et al.</em> 2009</a>) and has subsequently been adapted for use in diverse organisms (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Brutinel and Gralnick 2012</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Gallagher <em>et al.</em> 2011</a>). Similarly, PhiTSeq facilitates simultaneous analysis of thousands of transposon-mutagenized haploid human cells (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Carette <em>et al.</em> 2011</a>). The widespread adoption of pooled mutant screens using high-throughput quantitative DNA sequencing attests to the power of these methods for efficient genetic analysis.</p><p id="p-6">In contrast to the rapid technological advances in pooled mutant analysis, there has not yet been a statistical treatment of the experimental design and analysis of data generated by high-throughput DNA sequence analysis of these complex libraries. Thus, major methodological and analytical questions remain unanswered. What is the appropriate statistical framework for analyzing DNA sequence count data? What are the sources of variation? What is the appropriate study design for maximizing the power and accuracy to detect differences in mutant abundances? What sequence read depth maximizes the precision of these methods while minimizing the cost and resources required?</p><p id="p-7">We undertook a study that aimed to address these questions with the goal of providing guidance for the design and analysis of pooled mutant screens using high-throughput DNA sequencing. Using experimental analysis of the <em>S. cerevisiae</em> gene deletion collection in two different conditions, we studied the contribution of treatment and biological and technical variation to Bar-seq data (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). We demonstrated that the negative binomial models used to analyze RNA-seq data are also directly applicable to Bar-seq data. Using computational subsampling of our experimental data, we studied the effect of different experimental designs on the results from Bar-seq analysis. We found that biological replicates substantially improved statistical power, whereas technical replicates provided only moderate additional statistical power. We also found that increasing sequencing depth beyond 6 million reads per condition provided limited improvement in the experimental results, regardless of experimental design.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/11/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Experimental design and results. (A) Our experimental design entailed two treatments (twenty-four hours of growth in glucose/YPD or galactose/YPGal), four biological replicates, and two technical replicates, along with four samples at time point 0 [not shown in (A)]. (B) Heat map of the Spearman correlation matrix of mutant counts by sample. Samples cluster according to time point and also by treatment (YPD vs. YPGal) and biological replicate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1099236970" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Experimental design and results. (A) Our experimental design entailed two treatments (twenty-four hours of growth in glucose/YPD or galactose/YPGal), four biological replicates, and two technical replicates, along with four samples at time point 0 [not shown in (A)]. (B) Heat map of the Spearman correlation matrix of mutant counts by sample. Samples cluster according to time point and also by treatment (YPD &lt;em&gt;vs.&lt;/em&gt; YPGal) and biological replicate.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/11/F1.medium.gif" width="440" height="216"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/1/11/F1.medium.gif" width="440" height="216"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/11/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/11/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463759" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Experimental design and results. (A) Our experimental design entailed two treatments (twenty-four hours of growth in glucose/YPD or galactose/YPGal), four biological replicates, and two technical replicates, along with four samples at time point 0 [not shown in (A)]. (B) Heat map of the Spearman correlation matrix of mutant counts by sample. Samples cluster according to time point and also by treatment (YPD <em>vs.</em> YPGal) and biological replicate.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">Our results provide information directly relevant to designing future high-throughput quantitative DNA sequencing experiments of pooled mutants. For example, using an experimental design of four-fold biological replication and no technical replication, we showed that detection of mutants in the 4295 mutant yeast deletion collection with two-fold (or more) change between conditions can be achieved with 96% power at a 5% false discovery rate (FDR) using as few as 6 million reads per condition. This corresponds to a requirement of 1397 sequence reads per mutant per condition or 349 reads per biological replicate library. Using our experimental and analytical methods for Bar-seq analysis, it is possible to analyze the yeast deletion collection in up to 30 different conditions using a single 200-million read lane without sacrificing statistical power. Our findings should be informative for other methods of pooled mutant analysis such as Tn-seq.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains, media, and sampling procedures</h3><p id="p-10">We used a haploid prototrophic gene deletion collection constructed using the synthetic genetic array method (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Tong <em>et al.</em> 2001</a>). The library contains the identical gene deletion alleles as the standard yeast knockout collection (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Winzeler <em>et al.</em> 1999</a>), excluding gene deletions that result in auxotrophies. Gene deletion alleles are marked with the kanMX4 cassette conferring G418 resistance, which is flanked by a unique 5′ molecular barcode (the UPTAG) and a unique 3′ molecular barcode (the DNTAG). Each MAT<strong>a</strong> mutant contains a functional copy of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">LYS2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">MET15</a></em> genes and the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000789">can1</a></em>Δ::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001868">STE2</a><sub>pr</sub>-SpHIS5</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005212">lyp1</a></em>Δ<em>0</em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>Δ<em>1</em> alleles. We used standard YPD and YPGal media containing either 2% glucose or 2% galactose, respectively (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Amberg <em>et al.</em> 2005</a>).</p><p id="p-11">After growth of individual mutants on YPD agar plates, all mutants were pooled to a final density of 1.5 × 10<sup>9</sup> cells/ml. Each agar plate contained single colonies of individual genotypes and replicated colonies of the control <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">HO</a></em>Δ<em>0</em> strain. To define the replicated time zero (t0) samples, we obtained two independent samples of 0.5 ml (<em>i.e.</em>, 7.5 × 10<sup>8</sup> cells) from the pooled library. We inoculated 5 μl from the pooled library (<em>i.e.</em>, 7.5 × 10<sup>6</sup> cells) into four-fold replicated cultures of either 5 ml YPD or YPGal. Cells were grown for 24 hr (t24) to a final density of 3.3 × 10<sup>8</sup> cells/ml in both conditions. We removed 2 ml (<em>i.e.</em>, 6.6 × 10<sup>8</sup> cells) samples from each of the four YPD cultures and four YPGal cultures and purified genomic DNA using Qiagen Genomic-Tip 100 columns.</p></div><div id="sec-3" class="subsection"><h3>Library preparation and sequencing</h3><p id="p-12">We designed a two-step PCR protocol for efficient multiplexing of Bar-seq libraries. In the first PCR step, UPTAGs from a single sample were amplified with the primers <em>Illumina UPTAG Index</em> (5′-ACG CTC TTC CGA TCT <span class="underline">NNNNN</span> GTC CAC GAG GTC TCT-3′) and <em>Illumina UPkanMX</em> (CAA GCA GAA GAC GGC ATA CGA GAT GTC GAC CTG CAG CGT ACG-3′), and DNTAGs from the same sample were amplified with the primers <em>Illumina DNTAG Index</em> (5′-ACG CTC TTC CGA TCT <span class="underline">NNNNN</span> GTG TCG GTC TCG TAG-3′) and <em>IlluminaDNkanMX</em> (5′-CAA GCA GAA GAC GGC ATA CGA GAT ACG AGC TCG AAT TCA TCG-3′) in separate PCR reactions. Illumina UPTAG and Illumina DNTAG primers contain a 5-bp sequence (denoted as <span class="underline">NNNNN</span> in the primer sequence) that uniquely identifies the sample. We designed 120 unique sample indices that differed by at least two nucleotides. A complete list of primer sequences is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1/008565SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1/TableS1.xlsx">Table S1</a>. We normalized genomic DNA concentrations to 10 ng/μl and using 100 ng template amplified barcodes using the following PCR program: 2 min at 98° followed by 20 cycles of 10 sec at 98°, 10 sec at 50°, 10 sec at 72°, and a final extension step of 2 min at 72°. PCR products were confirmed on 2% agarose gels and purified using QIAquick PCR purification columns.</p><p id="p-13">We quantified purified PCR products using a Qubit fluorimeter and combined 60 ng from each of the 20 different UPTAG libraries and, in a separate tube, 60 ng from each of the 20 different DNTAG libraries. The multiplexed UPTAG libraries were then amplified using the primers <em>P5</em> (5′-A ATG ATA CGG CGA CCA CCG AGA TCT ACA CTC TTT CCC TAC ACG ACG CTC TTC CGA TCT-3′) and <em>Illumina UPkanMX</em>, and the combined DNTAG libraries were amplified using the <em>P5</em> and <em>IlluminaDNkanMX</em> primers using the identical PCR program as the first step with 20 ng template. The 140-bp UPTAG and DNTAG libraries were purified using QIAquick PCR purification columns, quantified using a Qubit fluorometer, combined in equimolar amounts, and adjusted to a final concentration of 10 nM (<em>i.e.</em>, 0.924 ng/μl). In total, the sequencing library contained 20 UPTAG and 20 DNTAG libraries from 20 different samples (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1/TableS2.xlsx">Table S2</a>). The library was sequenced on a single lane of an Illumina HiSeq 2000 using standard methods, including the use of the standard Illumina sequencing primer (5′-ACA CTC TTT CCC TAC ACG ACG CTC TTC CGA TCT-3′). The qseq files for each of the 20 samples are available from the NCBI Short Read Archive with the accession number SRA101498.</p></div><div id="sec-4" class="subsection"><h3>Read matching and statistical analysis</h3><p id="p-14">Sequence reads were matched to the yeast deletion collection barcodes reannotated by <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Smith <em>et al.</em> (2009)</a>. Inexact matching was performed by identifying barcode sequences that were within a Levenshtein distance of 2 from each read (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Levenshtein 1966</a>). Reads matching equally to multiple barcodes were discarded. Sample indices were similarly matched using a maximum Levenshtein distance of 1. The final matrix of counts matching the UPTAG and DNTAG of each of the 20 samples is provided as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1/TableS3.xlsx">Table S3</a>. A set of 359 outliers was identified that had fewer than 100 total reads across all 20 samples (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1/FigureS1.pdf">Figure S1</a>). These low-count matches were likely due to sequencing error and were removed. In addition, our pooled yeast gene deletion library included a highly abundant strain (the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">HO</a> gene deletion mutant, which was present on each of the 96-well plates containing individual mutants before pooling). The <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">HO</a> deletion mutant represented 19% of all reads and was removed before computational analyses, leaving a total of 139.8 million reads mapped to 4295 mutants.</p><p id="p-15">Eigen <em>R</em><sup>2</sup> was used to determine the percent of variance explained by the different factors in our experimental design for the t<sub>24</sub> samples (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen and Storey 2008</a>). Barcode counts were normalized using the TMM method (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Robinson and Oshlack 2010</a>) after adding 1 to each value and then were log-transformed, to avoid including differences in per library read depth as a source of variation. The bottom 10% of mutants was filtered out because lower counts have a disproportionate effect on the technical variation. Eigen <em>R</em><sup>2</sup> was used to compute the percent of variance explained by the treatment factor <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/11/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/11/embed/mml-math-1.gif"/></noscript></span></span> and the biological replicate factor <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/11/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/11/embed/mml-math-2.gif"/></noscript></span></span>. Because the treatment factor is contained within the biological factor, we report the biological percent of variation as <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/11/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/11/embed/mml-math-3.gif"/></noscript></span></span>, and the technical variation as <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/11/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/11/embed/mml-math-4.gif"/></noscript></span></span>.</p><p id="p-16">For differential abundance analysis, we first summed UPTAGs and DNTAGs for technical replicates within each biological replicate. The edgeR package (version 3.2.4) was used to perform dispersion estimation and to perform an exact negative binomial test to calculate a p-value and log-fold change for each mutant using the exactTest function using the default parameters (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Robinson and Mccarthy 2010</a>). The qvalue package was used to compute q-values (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Storey and Tibshirani 2003</a>).</p><p id="p-17">Gene set enrichment analysis was performed using the Biological Process ontology from SGD. Gene sets that had fewer than four genes among the detected deletions were discarded in advance. We used the Wilcoxon rank-sum test to compare the distribution of the estimated log-fold changes within each gene set to those outside of the set (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Gresham <em>et al.</em> 2011</a>). We used the qvalue package to set a 5% FDR threshold, above which gene sets were declared significantly enriched.</p></div><div id="sec-5" class="subsection"><h3>Read subsampling</h3><p id="p-18">Separate subsamplings were performed for each combination of replicates in each design. This requires one combination for the full 2 treatments × 4 biological replicates × 3 technical replicates design, two combinations for the 2 × 4 × 1 design, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/11/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/11/embed/mml-math-5.gif"/></noscript></span></span> combinations for the 2 × 3 × 2 design, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/11/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/11/embed/mml-math-6.gif"/></noscript></span></span> combinations for the 2 × 3 × 2 design, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/11/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/11/embed/mml-math-7.gif"/></noscript></span></span> combinations for the 2 ×2 × 2 design, and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/11/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/11/embed/mml-math-8.gif"/></noscript></span></span> combinations for the 2 × 2 × 1 design. For each combination, we performed subsampling over a sequence of 400 evenly spaced fractions of reads corresponding to 0.25%, 0.50%, …, 99.75%, and 100%.</p><p id="p-19">For each fraction <em>p</em>, a subsampled count matrix <em>S</em> was generated based on the full experiment matrix as <em>S<sub>i</sub></em><sub>,</sub><em><sub>j</sub></em> ∼Binom(<em>X<sub>i</sub></em><sub>,</sub><em><sub>j</sub></em>, <em>p</em>). This is equivalent to choosing a random sample of the sequenced reads and then mapping them. The same analysis steps used for the full data set were used to analyze each subsample and the same metrics were applied to assess the results as used for the full experiment.</p><p id="p-20">As the results for each experimental design depend on which of the replicates was chosen for subsampling the results were smoothed for each experimental design using a natural cubic spline with 20 degrees of freedom for estimates of the power, accuracy and FDR. For estimates of the informativeness of each experimental design, we used 15 degrees of freedom because the number of significant gene sets identified in each subsample showed greater variance than the other three metrics.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>Experimental results</h3><p id="p-21">We aimed to dissect the sources of variation in pooled mutant screens and to determine the appropriate analytical framework and experimental design that maximizes the value of the assay while minimizing cost and resources. All pooled genetic screens using mixtures of mutants require generation of a library of mutants, experimental treatment of the pooled mutants, and identification and quantification of DNA sequences that uniquely identify each mutant using high-throughput DNA sequencing. We designed an experiment to compare growth of haploid yeast nonessential gene deletion mutants in two different carbon sources, glucose (YPD) and galactose (YPGal), using Bar-seq analysis of the molecular barcodes that uniquely identify each mutant. To address the goals of our study, we prepared four biological replicates grown for 24 hr in each condition and two technical replicates (<em>i.e.</em>, independent sequencing library preparation of the same DNA sample) of each biological replicate (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1/TableS2.xlsx">Table S2</a>). We also obtained two independent samples from the unselected library (time point 0) from which we prepared technical replicates.</p><p id="p-22">To generate libraries for sequencing with an Illumina HiSeq, we designed a simple two-stage PCR protocol (see <em>Materials and Methods</em>). Each gene deletion is marked by two different molecular barcodes: one 5′ (the UPTAG) and one 3′ (the DNTAG) of the drug resistance cassette. To multiplex sequencing of different Bar-seq libraries, we developed a PCR-based method for library preparation that incorporates a unique sequence index for each library (see <em>Materials and Methods</em>). We sequenced 40 libraries (20 UPTAG and 20 DNTAG) from 20 samples in a single lane of an Illumina HiSeq 2000. We obtained 185.2 million reads that passed quality filters and matched them to the molecular barcodes by identifying sequences within a Levenshtein edit distance of 2, which resulted in mapping 93.3% of reads. Using a Levenshtein distance cutoff of 0 (<em>i.e.</em>, an exact match) or 1 results in successful mapping of 62.6% and 84.6% of the reads, respectively.</p><p id="p-23">For the majority of mutants, the number of reads per barcode across all experiments follows an approximately log-normal distribution and ranges between 1000 and 100,000 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1/FigureS1.pdf">Figure S1</a>). Low-count outliers that likely resulted from sequencing errors were removed (<em>Materials and Methods</em>). We found that UPTAGs and DNTAGs for each mutant had similar counts in the majority of samples, with 2574 mutants within a two-fold difference of each other (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1/FigureS2.pdf">Figure S2</a>). However, many mutants had highly divergent counts: 1264 had more than a 10-fold difference and 1052 had more than a 100-fold difference. These discrepancies were likely attributable to one of the barcodes being lost because of sequencing error in either the barcode or the PCR priming site.</p><p id="p-24">Correlation analysis of barcode counts showed that the lowest correlations were between mutant abundance in the unselected library (t<sub>0</sub>) and mutant abundance after 24 hr of growth in either glucose-containing or galactose-containing media, indicating that differences in cell growth rates results in substantial changes in the relative abundance of mutants (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). Growth in YPD yields higher correlation with the t<sub>0</sub> sample than growth in YPGal, indicating that growth in galactose led to a greater shift in the relative abundance of mutants than did growth in glucose. To identify differential effects of mutants during growth in glucose and galactose, we restricted our analysis to the t<sub>24</sub> samples. We used eigen <em>R</em><sup>2</sup> (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Chen and Storey 2008</a>) to partition the variance among these samples and found that 63.5% of the variance was explained by the treatment, 20.3% was explained by biological variation, and 16.1% was explained by technical variation (<em>Materials and Methods</em> ). The apportionment of variance was consistent across a wide range of percentile thresholds and using a variety of normalization methods (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1/FigureS3.pdf">Figure S3</a>).</p></div><div id="sec-8" class="subsection"><h3>Computational analysis of differential mutant abundance</h3><p id="p-25">The goal of pooled mutant screens is to identify mutants that exhibit differences in abundance as a result of a defined treatment. The appropriate statistical methods depend on the nature of the data, which in the case of quantitative DNA sequencing of molecular barcodes are discrete count data. As we observed in the work of <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Gresham <em>et al.</em> (2011)</a>, the data are best described by an overdispersed Poisson distribution (<em>i.e.</em>, the variance of biological replicates is greater than the mean) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1/FigureS4.pdf">Figure S4</a>). The problem of comparing count data between samples with different read depths while assuming overdispersed Poisson variation is related to that presented by differential expression analysis of RNA-seq data, for which a negative binomial test is used. In addition to the fact that Bar-seq data present some characteristics problematic for <em>t</em> tests (<em>i.e.</em>, lack of normality and a strong mean–variance relationship), there is important motivation for utilizing models specifically designed for count data. For example, consider two mutants in two different conditions in which the data of one are simply 1000× the other in read depth (<em>e.g.</em>, counts 8 and 9 <em>vs.</em> 13 and 14 for mutant <em>A</em> and 8000 and 9000 <em>vs.</em> 13,000 and 14,000 for mutant <em>B</em>). Whereas a <em>t</em> test results in the same p-value for both mutants, a negative binomial model directly takes into account the difference in read depth, resulting in drastically different p-values. Because the difference between the mutants with the lowest total number of reads to the highest number of reads is ∼2600-fold in our experiment (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1/FigureS1.pdf">Figure S1</a>), this is a valid issue. Therefore, we used a negative binomial model to test for mutants that were differentially abundant as a result of the treatment.</p><p id="p-26">We utilized the edgeR software package (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Robinson and McCarthy 2010</a>), which has an efficient implementation of the negative binomial test that accounts for differing read depth and uses shrinkage to help estimate dispersion parameters. We observed that dispersion estimates underwent considerable shrinkage even when four biological replicates were used (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1/FigureS4.pdf">Figure S4</a>). We found RNA-seq analysis methods that also fit a negative binomial model, such as that implemented in DESeq (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anders and Huber 2010</a>), produced qualitatively comparable results (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1/FigureS5.pdf">Figure S5</a>). Alternative methods, including DEGSeq (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Wang <em>et al.</em> 2010</a>) and Myrna (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Langmead <em>et al.</em> 2010</a>), make overdispersion assumptions less consistent with our data, whereas other methods, including Cuffdiff, use an implementation specific to RNA-seq (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Trapnell <em>et al.</em> 2013</a>).</p><p id="p-27">Previous studies have used measurements of the UPTAG and DNTAG for each deletion mutant in different ways, including selection of the barcode for each mutant with the highest count (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Smith <em>et al.</em> 2010</a>) and independent analysis of each barcode (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Gresham <em>et al.</em> 2011</a>). Because the UPTAG and DNTAG were measurements of the same mutant, summing the counts within each sample provided a means of combining the information from both barcodes while remaining robust to cases in which one barcode was lost. Furthermore, with count data, summing across technical replicates provided a superior method for minimizing technical variation compared with calculating an average value. Therefore, we summed UPTAGs and DNTAGs for each mutant over technical replicates, such that each condition had four biological replicates, and applied tests using a negative binomial model to identify mutants that were significantly different in abundance in YPGal compared with YPD after 24 hr of growth. The 16 samples comprising this dataset included a total of 112 million reads.</p><p id="p-28">Analysis of our dataset identified 2036 mutants that were differentially abundant between the two conditions at 5% FDR. The effect sizes of individual gene deletions were widely distributed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1/TableS4.xlsx">Table S4</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Notably, the gene deletion mutants for 8 of the 11 genes required for galactose metabolism (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Timson 2007</a>) were significantly decreased in abundance in YPGal and mutants deleted for two genes known to repress galactose metabolism were significantly increased in abundance in YPGal (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). Gene set enrichment analysis using a Wilcoxon rank-sum test found 192 enriched gene sets at FDR of 5%, and the top sets were related to respiration and mitochondrial processes, consistent with the increased importance of respirative metabolism when yeast cells grow in galactose (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1/TableS5.xlsx">Table S5</a>). Mutants identified as significantly differing in abundance between YPGal and YPD were identified across a range of sequence read depths, although smaller effect sizes tended to be called statistically significant as read depth increased (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). The ability to detect significant changes in mutant abundance was not greatly affected when total read counts were more than 1000, and two-fold differences were still detected as statistically significant with total read depths as low as 100. These observations suggest that we oversampled in our study and that similar results would be obtained with approximately an order of magnitude fewer reads.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/11/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bar-seq quantifies mutant effects across a range of sequence read depths. (A) Volcano plot showing the relationship between the p-value (log-scale) and log-fold change. Genes known to be involved in activation or repression of the galactose utilization pathway are highlighted. The p-value of the rightmost red point is computationally indistinguishable from 0. (B) Plot of reads per mutant in the entire experiment compared with the estimated fold change after treatment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1099236970" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bar-seq quantifies mutant effects across a range of sequence read depths. (A) Volcano plot showing the relationship between the p-value (log-scale) and log-fold change. Genes known to be involved in activation or repression of the galactose utilization pathway are highlighted. The p-value of the rightmost red point is computationally indistinguishable from 0. (B) Plot of reads per mutant in the entire experiment compared with the estimated fold change after treatment.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/11/F2.medium.gif" width="432" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/1/11/F2.medium.gif" width="432" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/11/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/11/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463818" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Bar-seq quantifies mutant effects across a range of sequence read depths. (A) Volcano plot showing the relationship between the p-value (log-scale) and log-fold change. Genes known to be involved in activation or repression of the galactose utilization pathway are highlighted. The p-value of the rightmost red point is computationally indistinguishable from 0. (B) Plot of reads per mutant in the entire experiment compared with the estimated fold change after treatment.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Effect of experimental design on statistical results</h3><p id="p-30">We aimed to identify the experimental design features that have the greatest effect on the results of a Bar-seq experiment. In practice, the experimental considerations that are most easily controlled are the extent of biological and technical replication and the depth to which each library is sequenced. We computationally simulated variation in each of these experiment design parameters using random subsampling of sequence reads from our complete experiment (<em>Materials and Methods</em>). For the purpose of assessing results from these subsamples, we compared them to results obtained from analysis of the complete dataset, which we defined as the gold standard. The negative binomial model we fit requires at least two biological replicates. Therefore, to study the effect of biological replication, we simulated the use of experimental designs using three or two biological replicates while retaining two technical replicates for each sample. To study the effect of technical replicates, we simulated the use of experimental designs using one technical replicate for each of the biological replicates. For each simulated experimental design, we sampled a subset of the reads to simulate varying read depths. We considered four metrics that assess the quality of each simulated experimental dataset: statistical power; accuracy; informativeness; and FDR.</p><p id="p-31">We assessed the power of each experimental design for different sequence read depths by determining the number of mutants identified as differentially abundant at FDR of 5%. In all cases, the statistical power of each experimental design increased with read depth; however, it rapidly saturated (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). Considering our full experimental design, it took just 1.7 million reads per condition to detect half of the significant mutants that were detected using the complete dataset and 75% were detected with 4.3 million reads per condition. Mutants that are most differentially abundant could be detected at very low read depths: the 13 most significant mutants identified using the complete dataset were all identified as significant even at the lowest depth tested, 140,000 mapped reads (<em>i.e.</em>, a 400-fold lower sequence read depth than the total), and were ranked among the 15 most significant mutants in all but the lowest read depth. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1/TableS6.xlsx">Table S6</a> shows the effect size, significance, and rank of the seven most significant galactose-related genes at each level of subsampling, demonstrating that they remained highly significant even at very low read depths.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/11/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulation analysis of variation in experimental design. The effect of read depth on (A) the number of mutants found significant at FDR = 5%, (B) the mean squared error between the estimate of the log-fold change and the value for the full experiment, (C) the number of significant GO terms identified using a Wilcoxon rank-sum test at FDR = 5%, and (D) the percentage of significant genes that were not found to be significant in the full experiment. Curves are shown for the full experiment, 2 treatments &#xD7; 4 biological replicates &#xD7; 2 technical replicates, as well as for subsampled 2 &#xD7; 3 &#xD7; 2, 2 &#xD7; 2 &#xD7; 2 experimental designs (solid lines). Subsamplings were also performed to simulate each experimental design using a single technical replicate (dashed lines). Each curve was smoothed using a natural cubic spline." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1099236970" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simulation analysis of variation in experimental design. The effect of read depth on (A) the number of mutants found significant at FDR = 5%, (B) the mean squared error between the estimate of the log-fold change and the value for the full experiment, (C) the number of significant GO terms identified using a Wilcoxon rank-sum test at FDR = 5%, and (D) the percentage of significant genes that were not found to be significant in the full experiment. Curves are shown for the full experiment, 2 treatments &#xD7; 4 biological replicates &#xD7; 2 technical replicates, as well as for subsampled 2 &#xD7; 3 &#xD7; 2, 2 &#xD7; 2 &#xD7; 2 experimental designs (solid lines). Subsamplings were also performed to simulate each experimental design using a single technical replicate (dashed lines). Each curve was smoothed using a natural cubic spline.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/11/F3.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/1/11/F3.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/11/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/11/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463799" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Simulation analysis of variation in experimental design. The effect of read depth on (A) the number of mutants found significant at FDR = 5%, (B) the mean squared error between the estimate of the log-fold change and the value for the full experiment, (C) the number of significant GO terms identified using a Wilcoxon rank-sum test at FDR = 5%, and (D) the percentage of significant genes that were not found to be significant in the full experiment. Curves are shown for the full experiment, 2 treatments × 4 biological replicates × 2 technical replicates, as well as for subsampled 2 × 3 × 2, 2 × 2 × 2 experimental designs (solid lines). Subsamplings were also performed to simulate each experimental design using a single technical replicate (dashed lines). Each curve was smoothed using a natural cubic spline.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Reducing the number of biological replicates results in reduced statistical power for a given read depth. Using three biological replicates rather than four decreases the statistical power by approximately 16%, and using only two biological replicates decreases it by 38%. In practice, this effect is far more relevant than the read depth: 10 million mapped reads using two biological replicates achieves approximately the same power as 2 million total reads across four biological replicates, and the difference cannot be compensated by increasing sequence read depth. Technical replicates only marginally increase the power of the experimental design. This improvement is because pooling multiple replicates decreases the noise added by the library preparation and therefore decreases the within-treatment variation, analogous to previously studied strategies of pooling multiple replicates on a single microarray (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Peng <em>et al.</em> 2003</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Kendziorski <em>et al.</em> 2005</a>).</p><p id="p-34">Although the maximum power possible with each experimental design differs, it is interesting to note that the point at which statistical power begins to asymptote is very similar across experimental design, at approximately 6 million reads per condition (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). This suggests that at this point, experimental noise attributable to the sequencing machine itself no longer decreases and additional variation is attributable to noise introduced by biological variability and library preparation. Statistical power varies within each subset of the designs depending on which replicates were selected (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1/FigureS6.pdf">Figure S6</a>), indicating that different replicates added different amounts of variance to the experiment, which cannot be predicted <em>a priori</em>.</p><p id="p-35">The utility of an experimental design can also be assessed in terms of the accuracy with which effect sizes are estimated, as quantified by the mean square error, the informativeness of the analysis, as quantified by the number of significant gene sets identified by gene set enrichment analysis, or the FDR, as quantified by the proportion of genes found significant that are not significant in the full experiment. Assessments of the quality of each experimental design considering accuracy (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>), informativeness (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>), and FDR (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3D</a>) show that the greatest improvements are found with addition of biological replicates and that improvements beyond 6 million reads per condition are minimal, regardless of experimental design. Although there is some variation in the point at which each metric ultimately saturates, the points at which each metric begins to asymptote are highly concordant. Thus, beyond a surprisingly low threshold of 6 million reads per condition, additional sequencing depth provides little additional value.</p><p id="p-36">Although pooled mutant screens enable simultaneous sensitive measurement of the effect of each mutant, they are frequently used as a means of identifying those mutants of greatest effect. We analyzed the statistical power of an experimental design using four biological replicates and no technical replication for different effect sizes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). As few as 2.5 million sequence reads per condition (625,000 reads per sample) are sufficient to detect 90% of mutants that change more than two-fold in the full experiment. Increasing the read depth to 6 million reads per condition detects 96% of mutants that change more than two-fold, 91% of all mutants that change more than 1.5-fold and 72% of all mutants that are significant in the full experiment.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/11/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Statistical power varies with effect size and sequence read depth. The effect of read depth on the proportion of genes identified as significant (FDR = 5%) at different fold -change thresholds using 4 biological replicates &#xD7; 1 technical replicate for each condition. The fold change for each mutant determined from the full 4 biological replicate &#xD7; 2 technical replicate experiment is defined as the gold standard. The solid curve shows the proportion of genes found significant relative to the total experiment, whereas the dotted and dashed curves show the proportion of mutants that had at least a 1.5-fold or a two-fold change, respectively. The horizontal dashed line indicates the 90% power level." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1099236970" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Statistical power varies with effect size and sequence read depth. The effect of read depth on the proportion of genes identified as significant (FDR = 5%) at different fold -change thresholds using 4 biological replicates &#xD7; 1 technical replicate for each condition. The fold change for each mutant determined from the full 4 biological replicate &#xD7; 2 technical replicate experiment is defined as the gold standard. The solid curve shows the proportion of genes found significant relative to the total experiment, whereas the dotted and dashed curves show the proportion of mutants that had at least a 1.5-fold or a two-fold change, respectively. The horizontal dashed line indicates the 90% power level.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/11/F4.medium.gif" width="422" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/1/11/F4.medium.gif" width="422" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/11/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/11/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463867" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-37" class="first-child">Statistical power varies with effect size and sequence read depth. The effect of read depth on the proportion of genes identified as significant (FDR = 5%) at different fold -change thresholds using 4 biological replicates × 1 technical replicate for each condition. The fold change for each mutant determined from the full 4 biological replicate × 2 technical replicate experiment is defined as the gold standard. The solid curve shows the proportion of genes found significant relative to the total experiment, whereas the dotted and dashed curves show the proportion of mutants that had at least a 1.5-fold or a two-fold change, respectively. The horizontal dashed line indicates the 90% power level.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-10"><h2 class="">Discussion</h2><p id="p-38">High-throughput quantitative DNA sequencing has resulted in rapid advances in a range of problems from the analysis of genome variation to the three-dimensional organization of genomes. The coupling of high-throughput quantitative sequencing with large-scale mutagenesis (either systematic or random) enables the pooled analysis of mutant phenotypes with broad applications, including the study of gene function, drug targets, and genetic interactions. Here, we have studied one realization of pooled mutant analysis, Bar-seq, with the goal of determining experimental designs and analytical methods that provide excellent levels of sensitivity, specificity, and efficiency.</p><p id="p-39">We have shown that statistical models used for RNA-seq analysis are directly applicable to the analysis of Bar-seq data. Tools for RNA-seq analysis, such as those used here, are therefore readily adapted to Bar-seq analysis, providing estimates of effect sizes and statistical significance for each mutant. For Bar-seq analysis, UPTAGs and DNTAGs represent additive measurements of the same genotype and therefore should be summed for each sample. Similarly, technical replicates should be combined by addition of barcode counts.</p><p id="p-40">Biological replication is essential for rigorous assessment of statistical significance. At least two biological replicates should always be performed to use the within-treatment variation for determining statistical significance. Some software packages have the option of guessing the dispersion in advance, but this is not recommended because an incorrect estimate would make subsequent tests for statistical significance either too conservative or too generous. Moreover, we have found that different experiments can contribute different amounts of variation. Therefore, we recommend performing at least four biological replicates to maximize statistical power and accuracy of effect size. The use of technical replicates of biological replicates results in marginal improvements and is likely unnecessary.</p><p id="p-41">Importantly, we found that Bar-seq does not require a high read depth to accurately detect differential abundance of mutants and that additional reads add little to the results. In our study using nearly 60 million mapped reads per condition to analyze 4295 mutants, we demonstrated that the quality of our dataset was maintained with approximately 10-fold fewer reads. Our experimental method for Bar-seq includes 120 uniquely indexed adaptors (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1/TableS2.xlsx">Table S2</a>), meaning that on a 200-million read sequencing lane, one can analyze four biological replicates of 30 different conditions, resulting in approximately 6 million reads per condition. Based on our analysis, that read depth would be expected to identify 96% of genes with a two-fold change, 91% of mutants with a 1.5-fold change, and 72% of all mutants that would be detected with 10-times greater read depth and two technical replicates (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). These findings can be extended to other methods for pooled genetic screens by noting that it corresponds to ∼1400 reads per genomic target per condition. Increasing sequence read depth beyond this value provides only an incremental increase. Thus, our analysis provides guidelines about the tradeoff between per-condition read depth and statistical power that can be used for the design of future experiments.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-42">We thank members of the Gresham and Storey laboratories for helpful discussions. We thank Amy Caudy and David Hess for construction of the prototrophic deletion collection and Zachary Kurtz for design of sample index primers. This work was supported by the National Science Foundation (MCB-1244219 to D.G.), the National Institutes of Health (1R01GM107466 to D.G.; R01HG002913 and R21HG006769 to J.D.S.), and a DuPont Young Professor Award (to D.G.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008565/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-43"><em>Communicating editor: L. M. Steinmetz</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-09-16"><span class="received-label">Received </span>September 16, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-10-20"><span class="accepted-label">Accepted </span>October 20, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Robinson <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-4.1.11.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amberg</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strathern</span>  <span class="cit-name-given-names">J. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">Methods in Yeast Genetics. A Cold Spring Harbor Laboratory Course Manual</span>, <span class="cit-publ-name">Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.2" data-doi="10.1186/gb-2010-11-10-r106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Differential expression analysis for sequence count data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DAnders%26rft.auinit1%253DS.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253DR106%26rft.epage%253DR106%26rft.atitle%253DDifferential%2Bexpression%2Banalysis%2Bfor%2Bsequence%2Bcount%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-10-r106%26rft_id%253Dinfo%253Apmid%252F20979621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-10-r106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20979621&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.3" data-doi="10.1111/j.1365-2958.2012.08196.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brutinel</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gralnick</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Anomalies of the anaerobic tricarboxylic acid cycle in Shewanella oneidensis revealed by Tn-seq.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DBrutinel%26rft.auinit1%253DE.%2BD.%26rft.volume%253D86%26rft.issue%253D2%26rft.spage%253D273%26rft.epage%253D283%26rft.atitle%253DAnomalies%2Bof%2Bthe%2Banaerobic%2Btricarboxylic%2Bacid%2Bcycle%2Bin%2BShewanella%2Boneidensis%2Brevealed%2Bby%2BTn-seq.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2012.08196.x%26rft_id%253Dinfo%253Apmid%252F22925268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2012.08196.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22925268&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.4" data-doi="10.1038/nbt.1857"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carette</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guimaraes</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wuethrich</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blomen</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varadarajan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Global gene disruption in human cells to assign genes to phenotypes by deep sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">542</span>–<span class="cit-lpage">546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DCarette%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D29%26rft.issue%253D6%26rft.spage%253D542%26rft.epage%253D546%26rft.atitle%253DGlobal%2Bgene%2Bdisruption%2Bin%2Bhuman%2Bcells%2Bto%2Bassign%2Bgenes%2Bto%2Bphenotypes%2Bby%2Bdeep%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1857%26rft_id%253Dinfo%253Apmid%252F21623355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1857&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21623355&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291342100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.5" data-doi="10.1093/bioinformatics/btn411"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Eigen-R2 for dissecting variation in high-dimensional studies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2260</span>–<span class="cit-lpage">2262</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn411%26rft_id%253Dinfo%253Apmid%252F18718946%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjQvMTkvMjI2MCI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzQvMS8xMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.6" data-doi="10.1128/mBio.00315-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallagher</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shendure</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manoil</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-scale identification of resistance functions in Pseudomonas aeruginosa using Tn-seq.</span> <abbr class="cit-jnl-abbrev">MBio.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e00315</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio.%26rft.volume%253D2%26rft.spage%253De00315%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00315-10%26rft_id%253Dinfo%253Apmid%252F21253457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.00315-10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21253457&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.7" data-doi="10.1073/pnas.0906627106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gawronski</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">S. M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giannoukos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akerley</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Tracking insertion mutants within libraries by deep sequencing and a genome-wide screen for Haemophilus genes required in the lung.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">16422</span>–<span class="cit-lpage">16427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0906627106%26rft_id%253Dinfo%253Apmid%252F19805314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzM4LzE2NDIyIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.8" data-doi="10.1038/nature00935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riles</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional profiling of the Saccharomyces cerevisiae genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">418</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGiaever%26rft.auinit1%253DG.%26rft.volume%253D418%26rft.issue%253D6896%26rft.spage%253D387%26rft.epage%253D391%26rft.atitle%253DFunctional%2Bprofiling%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00935%26rft_id%253Dinfo%253Apmid%252F12140549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12140549&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177009700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.9" data-doi="10.1534/genetics.110.120766"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caudy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ziv</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brandt</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">System-level analysis of genes and functions affecting survival during nutrient starvation in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.120766%26rft_id%253Dinfo%253Apmid%252F20944018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg3LzEvMjk5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.10" data-doi="10.1186/gb-2010-11-6-r60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">T. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">L.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Global fitness profiling of fission yeast deletion strains by barcode sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHan%26rft.auinit1%253DT.%2BX.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253DR60%26rft.epage%253DR60%26rft.atitle%253DGlobal%2Bfitness%2Bprofiling%2Bof%2Bfission%2Byeast%2Bdeletion%2Bstrains%2Bby%2Bbarcode%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-6-r60%26rft_id%253Dinfo%253Apmid%252F20537132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-6-r60&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20537132&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.11" data-doi="10.1073/pnas.0500607102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kendziorski</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irizarry</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haag</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gould</span>  <span class="cit-name-given-names">M. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">On the utility of pooling biological samples in microarray experiments.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">4252</span>–<span class="cit-lpage">4257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0500607102%26rft_id%253Dinfo%253Apmid%252F15755808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzEyLzQyNTIiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy80LzEvMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.12" data-doi="10.1186/gb-2010-11-8-r83"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leek</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Cloud-scale RNA-sequencing differential expression analysis with Myrna.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253DR83%26rft.epage%253DR83%26rft.atitle%253DCloud-scale%2BRNA-sequencing%2Bdifferential%2Bexpression%2Banalysis%2Bwith%2BMyrna.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-8-r83%26rft_id%253Dinfo%253Apmid%252F20701754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-8-r83&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20701754&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levenshtein</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">Binary codes capable of correcting deletions, insertions and reversals.</span> <abbr class="cit-jnl-abbrev">Sov. Phys. Dokl.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">707710</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSov.%2BPhys.%2BDokl.%26rft.volume%253D10%26rft.spage%253D707710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.14" data-doi="10.1186/1471-2105-4-26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blalock</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landfield</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Statistical implications of pooling RNA samples for microarray experiments.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DPeng%26rft.auinit1%253DX.%26rft.volume%253D4%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DStatistical%2Bimplications%2Bof%2Bpooling%2BRNA%2Bsamples%2Bfor%2Bmicroarray%2Bexperiments.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-4-26%26rft_id%253Dinfo%253Apmid%252F12823867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-4-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12823867&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.15" data-doi="10.1093/bioinformatics/btp616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">edger: a bioconductor package for differential expression analysis of digital gene expression data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp616%26rft_id%253Dinfo%253Apmid%252F19910308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi8xLzEzOSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzQvMS8xMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.16" data-doi="10.1186/gb-2010-11-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oshlack</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A scaling normalization method for differential expression analysis of RNA-seq data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DRobinson%26rft.auinit1%253DM.%2BD.%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DA%2Bscaling%2Bnormalization%2Bmethod%2Bfor%2Bdifferential%2Bexpression%2Banalysis%2Bof%2BRNA-seq%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-3-r25%26rft_id%253Dinfo%253Apmid%252F20196867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20196867&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.17" data-doi="10.1126/science.1149200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schlabach</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solimini</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Cancer proliferation gene discovery through functional genomics.</span> <abbr class="cit-jnl-abbrev">Science (New York, NY)</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">620</span>–<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2B%2528New%2BYork%252C%2BNY%2529%26rft.volume%253D319%26rft.spage%253D620%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1149200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1126/science.1149200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.18" data-doi="10.1126/science.1149185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marran</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golding</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Profiling essential genes in human mammary cells by multiplex RNAi screening.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSilva%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D319%26rft.issue%253D5863%26rft.spage%253D617%26rft.epage%253D620%26rft.atitle%253DProfiling%2BEssential%2BGenes%2Bin%2BHuman%2BMammary%2BCells%2Bby%2BMultiplex%2BRNAi%2BScreening%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1149185%26rft_id%253Dinfo%253Apmid%252F18239125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTkvNTg2My82MTciO3M6NDoiYXRvbSI7czoxNjoiL2dnZy80LzEvMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.19" data-doi="10.1186/gb-2011-12-10-r104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sims</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendes-Pereira</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frankum</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgess</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cerone</span>  <span class="cit-name-given-names">M.-A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High-throughput RNA interference screening using pooled shRNA libraries and next generation sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DSims%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253DR104%26rft.epage%253DR104%26rft.atitle%253DHigh-throughput%2BRNA%2Binterference%2Bscreening%2Busing%2Bpooled%2BshRNA%2Blibraries%2Band%2Bnext%2Bgeneration%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-10-r104%26rft_id%253Dinfo%253Apmid%252F22018332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-10-r104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22018332&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.20" data-doi="10.1101/gr.093955.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heisler</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mellor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaper</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Quantitative phenotyping via deep barcode sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1836</span>–<span class="cit-lpage">1842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.093955.109%26rft_id%253Dinfo%253Apmid%252F19622793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMC8xODM2IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.21" data-doi="10.1093/nar/gkq368"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heisler</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St. Onge</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farias-Hesson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Highly-multiplexed barcode sequencing: an efficient method for parallel analysis of pooled samples.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">e142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq368%26rft_id%253Dinfo%253Apmid%252F20460461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOC8xMy9lMTQyIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.22" data-doi="10.1073/pnas.1530509100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Statistical significance for genomewide studies.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">9440</span>–<span class="cit-lpage">9445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1530509100%26rft_id%253Dinfo%253Apmid%252F12883005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzE2Lzk0NDAiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy80LzEvMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-4.1.11.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Timson</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Galactose metabolism in <em>Saccharomyces cerevisiae</em>.</span> <span class="cit-source">Dyn Biochem Process Biotech Mol Biol</span> <span class="cit-vol">1</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">63</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDyn%2BBiochem%2BProcess%2BBiotech%2BMol%2BBiol%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D63%26rft.atitle%253DGALACTOSE%2BMETABOLISM%2BIN%2BSACCHAROMYCES%2BCEREVISIAE.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.24" data-doi="10.1126/science.1065810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evangelista</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Systematic genetic analysis with ordered arrays of yeast deletion mutants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">294</span>: <span class="cit-fpage">2364</span>–<span class="cit-lpage">2368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTong%26rft.auinit1%253DA.%2BH.%2BY.%26rft.volume%253D294%26rft.issue%253D5550%26rft.spage%253D2364%26rft.epage%253D2368%26rft.atitle%253DSystematic%2BGenetic%2BAnalysis%2Bwith%2BOrdered%2BArrays%2Bof%2BYeast%2BDeletion%2BMutants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1065810%26rft_id%253Dinfo%253Apmid%252F11743205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTQvNTU1MC8yMzY0IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.25" data-doi="10.1038/nbt.2450"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hendrickson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauvageau</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goff</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Differential analysis of gene regulation at transcript resolution with RNA-seq.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">46</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D46%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2450%26rft_id%253Dinfo%253Apmid%252F23222703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2450&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23222703&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.26" data-doi="10.1038/nmeth.1377"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Opijnen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bodi</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camilli</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Tn-seq: high-throughput parallel sequencing for fitness and genetic interaction studies in microorganisms.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">772</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253Dvan%2BOpijnen%26rft.auinit1%253DT.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253D767%26rft.epage%253D772%26rft.atitle%253DTn-seq%253A%2Bhigh-throughput%2Bparallel%2Bsequencing%2Bfor%2Bfitness%2Band%2Bgenetic%2Binteraction%2Bstudies%2Bin%2Bmicroorganisms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1377%26rft_id%253Dinfo%253Apmid%252F19767758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1377&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19767758&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270355200023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.27" data-doi="10.1093/bioinformatics/btp612"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">DEGseq: an R package for identifying differentially expressed genes from RNA-seq data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp612%26rft_id%253Dinfo%253Apmid%252F19855105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi8xLzEzNiI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzQvMS8xMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.11.28" data-doi="10.1126/science.285.5429.901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Astromoff</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Functional characterization of the S. cerevisiae genome by gene deletion and parallel analysis.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">901</span>–<span class="cit-lpage">906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWinzeler%26rft.auinit1%253DE.%2BA.%26rft.volume%253D285%26rft.issue%253D5429%26rft.spage%253D901%26rft.epage%253D906%26rft.atitle%253DFunctional%2BCharacterization%2Bof%2Bthe%2BS.%2Bcerevisiae%2BGenome%2Bby%2BGene%2BDeletion%2Band%2BParallel%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.285.5429.901%26rft_id%253Dinfo%253Apmid%252F10436161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvNTQyOS85MDEiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy80LzEvMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/1/11.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/1/1" title="Genome-Wide Gene Expression Effects of Sex Chromosome Imprinting in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/1/19" title="Origin and Length Distribution of Unidirectional Prokaryotic Overlapping Genes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/1"><span>Volume 4 Issue 1, January 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/1.cover-source.jpg?itok=oIsy65sY" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/1/11" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DRjeG2WBlm6sFDZKf1I6dOplhOsjHKZMr7CxfbdzfJo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/1/11" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/1/11" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/1/11" class="active" data-icon-position="" data-hide-link-title="0">Design and Analysis of Bar-seq Experiments</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459613" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Qjne4mP6NjuuxCI3WqfkdyebYvIWyDvpVXZ_UaYTxtw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/1/11.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/1/11" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-M_fAXTzKvl6mzwKJitIePf_s0tklgIg0aYFVWRjS3iE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/1/11" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/1/11.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459613" id="node459613--2" data-pisa="ggg;4/1/11" data-pisa-master="ggg;g3.113.008565" data-apath="/ggg/4/1/11.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Design and Analysis of Bar-seq Experiments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">David G. Robinson</span>, <span class="highwire-citation-author" data-delta="1">Wei Chen</span>, <span class="highwire-citation-author" data-delta="2">John D. Storey</span> and <span class="highwire-citation-author" data-delta="3">David Gresham</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">11-18;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008565 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459613--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David G. Robinson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BG.%2BRobinson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Robinson%20DG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BG.%2BRobinson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wei Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Berlin Institute for Medical Systems Biology, Max-Delbrück-Center for Molecular Medicine, 13125 Berlin, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWei%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John D. Storey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BD.%2BStorey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Storey%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BD.%2BStorey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jstorey@princeton.edu" class="" data-icon-position="" data-hide-link-title="0">jstorey@princeton.edu</a>
<a href="mailto:dgresham@nyu.edu" class="" data-icon-position="" data-hide-link-title="0">dgresham@nyu.edu</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David Gresham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Genomics and Systems Biology, Department of Biology, New York University, New York, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BGresham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gresham%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BGresham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jstorey@princeton.edu" class="" data-icon-position="" data-hide-link-title="0">jstorey@princeton.edu</a>
<a href="mailto:dgresham@nyu.edu" class="" data-icon-position="" data-hide-link-title="0">dgresham@nyu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/1/11" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/1/11&amp;title=Design%20and%20Analysis%20of%20Bar-seq%20Experiments" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/1/11&amp;title=Design%20and%20Analysis%20of%20Bar-seq%20Experiments" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/1/11&amp;title=Design%20and%20Analysis%20of%20Bar-seq%20Experiments" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/1/11&amp;text=Design%20and%20Analysis%20of%20Bar-seq%20Experiments" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/1/11&amp;title=Design%20and%20Analysis%20of%20Bar-seq%20Experiments" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/1/11&amp;t=Design%20and%20Analysis%20of%20Bar-seq%20Experiments" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/1/11&amp;title=Design%20and%20Analysis%20of%20Bar-seq%20Experiments" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/1/11&amp;title=Design%20and%20Analysis%20of%20Bar-seq%20Experiments" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459613/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459613" id="node459613--3" data-pisa="ggg;4/1/11" data-pisa-master="ggg;g3.113.008565" data-apath="/ggg/4/1/11.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Design and Analysis of Bar-seq Experiments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">David G. Robinson</span>, <span class="highwire-citation-author" data-delta="1">Wei Chen</span>, <span class="highwire-citation-author" data-delta="2">John D. Storey</span> and <span class="highwire-citation-author" data-delta="3">David Gresham</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">11-18;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008565 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459613--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David G. Robinson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BG.%2BRobinson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Robinson%20DG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BG.%2BRobinson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wei Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Berlin Institute for Medical Systems Biology, Max-Delbrück-Center for Molecular Medicine, 13125 Berlin, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWei%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John D. Storey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BD.%2BStorey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Storey%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BD.%2BStorey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jstorey@princeton.edu" class="" data-icon-position="" data-hide-link-title="0">jstorey@princeton.edu</a>
<a href="mailto:dgresham@nyu.edu" class="" data-icon-position="" data-hide-link-title="0">dgresham@nyu.edu</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David Gresham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Genomics and Systems Biology, Department of Biology, New York University, New York, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BGresham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gresham%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BGresham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jstorey@princeton.edu" class="" data-icon-position="" data-hide-link-title="0">jstorey@princeton.edu</a>
<a href="mailto:dgresham@nyu.edu" class="" data-icon-position="" data-hide-link-title="0">dgresham@nyu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=11&amp;rft.epage=18&amp;rft.atitle=Design%20and%20Analysis%20of%20Bar-seq%20Experiments&amp;rft.volume=4&amp;rft.issue=1&amp;rft.date=2014-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Robinson%2C+David+G.&amp;rft.au=Chen%2C+Wei&amp;rft.au=Storey%2C+John+D.&amp;rft.au=Gresham%2C+David"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459613/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459613/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459613/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459613/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459613/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459613/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459613/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459613/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459613/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459613/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459613/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459613/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/1/11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/1/11.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/1/11.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/1/11.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/1/11.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
