<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/1/89" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/1/89.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/1/89.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/1/89.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/1/89.atom" />
<meta name="HW.pisa" content="ggg;4/1/89" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Use of SNP Markers for Linkage Mapping in Diploid and Tetraploid Peanuts" />
<meta name="DC.Identifier" content="10.1534/g3.113.007617" />
<meta name="DC.Date" content="2014-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Bertioli et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Single nucleotide polymorphic markers (SNPs) are attractive for use in genetic mapping and marker-assisted breeding because they can be scored in parallel assays at favorable costs. However, scoring SNP markers in polyploid plants like the peanut is problematic because of interfering signal generated from the DNA bases that are homeologous to those being assayed. The present study used a previously constructed 1536 GoldenGate SNP assay developed using SNPs identified between two A. duranensis accessions. In this study, the performance of this assay was tested on two RIL mapping populations, one diploid ( A. duranensis × A. stenosperma ) and one tetraploid [ A. hypogaea cv. Runner IAC 886 × synthetic tetraploid ( A. ipaënsis × A. duranensis )4×]. The scoring was performed using the software GenomeStudio version 2011.1. For the diploid, polymorphic markers provided excellent genotyping scores with default software parameters. In the tetraploid, as expected, most of the polymorphic markers provided signal intensity plots that were distorted compared to diploid patterns and that were incorrectly scored using default parameters. However, these scorings were easily corrected using the GenomeStudio software. The degree of distortion was highly variable. Of the polymorphic markers, approximately 10% showed no distortion at all behaving as expected for single-dose markers, and another 30% showed low distortion and could be considered high-quality. The genotyped markers were incorporated into diploid and tetraploid genetic maps of Arachis and, in the latter case, were located almost entirely on A genome linkage groups." />
<meta name="DC.Contributor" content="David J. Bertioli" />
<meta name="DC.Contributor" content="Peggy Ozias-Akins" />
<meta name="DC.Contributor" content="Ye Chu" />
<meta name="DC.Contributor" content="Karinne M. Dantas" />
<meta name="DC.Contributor" content="Silvio P. Santos" />
<meta name="DC.Contributor" content="Ediene Gouvea" />
<meta name="DC.Contributor" content="Patricia M. Guimarães" />
<meta name="DC.Contributor" content="Soraya C. M. Leal-Bertioli" />
<meta name="DC.Contributor" content="Steven J. Knapp" />
<meta name="DC.Contributor" content="Marcio C. Moretzsohn" />
<meta name="article:published_time" content="2014-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Use of SNP Markers for Linkage Mapping in Diploid and Tetraploid Peanuts" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Single nucleotide polymorphic markers (SNPs) are attractive for use in genetic mapping and marker-assisted breeding because they can be scored in parallel assays at favorable costs. However, scoring SNP markers in polyploid plants like the peanut is problematic because of interfering signal generated from the DNA bases that are homeologous to those being assayed. The present study used a previously constructed 1536 GoldenGate SNP assay developed using SNPs identified between two &lt;i&gt;A. duranensis&lt;/i&gt; accessions. In this study, the performance of this assay was tested on two RIL mapping populations, one diploid (&lt;i&gt;A. duranensis&lt;/i&gt; × &lt;i&gt;A. stenosperma&lt;/i&gt;) and one tetraploid [&lt;i&gt;A. hypogaea&lt;/i&gt; cv. Runner IAC 886 × synthetic tetraploid (&lt;i&gt;A. ipaënsis&lt;/i&gt; × &lt;i&gt;A. duranensis&lt;/i&gt;)&lt;sup&gt;4×&lt;/sup&gt;]. The scoring was performed using the software GenomeStudio version 2011.1. For the diploid, polymorphic markers provided excellent genotyping scores with default software parameters. In the tetraploid, as expected, most of the polymorphic markers provided signal intensity plots that were distorted compared to diploid patterns and that were incorrectly scored using default parameters. However, these scorings were easily corrected using the GenomeStudio software. The degree of distortion was highly variable. Of the polymorphic markers, approximately 10% showed no distortion at all behaving as expected for single-dose markers, and another 30% showed low distortion and could be considered high-quality. The genotyped markers were incorporated into diploid and tetraploid genetic maps of &lt;i&gt;Arachis&lt;/i&gt; and, in the latter case, were located almost entirely on A genome linkage groups.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/01/01" />
<meta name="citation_mjid" content="ggg;4/1/89" />
<meta name="citation_id" content="4/1/89" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/1/89" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/1/89.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/1/89.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/1/89.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.007617" />
<meta name="citation_pmid" content="24212082" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="89" />
<meta name="citation_lastpage" content="96" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="David J. Bertioli" />
<meta name="citation_author_institution" content="Departamento de Genética/IB, Universidade de Brasília, Brasília, DF, 70910-900, Brazil" />
<meta name="citation_author_email" content="davidbertioli@unb.br" />
<meta name="citation_author" content="Peggy Ozias-Akins" />
<meta name="citation_author_institution" content="Department of Horticulture, University of Georgia, Tifton, Georgia 31973" />
<meta name="citation_author" content="Ye Chu" />
<meta name="citation_author_institution" content="Department of Horticulture, University of Georgia, Tifton, Georgia 31973" />
<meta name="citation_author" content="Karinne M. Dantas" />
<meta name="citation_author_institution" content="Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil" />
<meta name="citation_author" content="Silvio P. Santos" />
<meta name="citation_author_institution" content="Universidade Católica de Brasília, Brasília, DF, 70790-160, Brazil" />
<meta name="citation_author" content="Ediene Gouvea" />
<meta name="citation_author_institution" content="Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil" />
<meta name="citation_author" content="Patricia M. Guimarães" />
<meta name="citation_author_institution" content="Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil" />
<meta name="citation_author" content="Soraya C. M. Leal-Bertioli" />
<meta name="citation_author_institution" content="Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil" />
<meta name="citation_author" content="Steven J. Knapp" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Genetics, and Genomics, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="Marcio C. Moretzsohn" />
<meta name="citation_author_institution" content="Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=E. Akhunov;citation_author=C. Nicolet;citation_author=J. Dvorak;citation_title=Single nucleotide polymorphism genotyping in polyploid wheat with the Illumina GoldenGate assay.;citation_pages=507-517;citation_volume=119;citation_year=2009;citation_issue=3;citation_pmid=19449174;citation_doi=10.1007/s00122-009-1059-5" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=D. Bertioli;citation_author=M. Moretzsohn;citation_author=L. H. Madsen;citation_author=N. Sandal;citation_author=S. Leal-Bertioli;citation_title=An analysis of synteny of Arachis with Lotus and Medicago sheds new light on the structure, stability and evolution of legume genomes.;citation_pages=45-45;citation_volume=10;citation_year=2009;citation_pmid=19166586;citation_doi=10.1186/1471-2164-10-45" />
<meta name="citation_reference" content="citation_journal_title=Plant Genet. Resour. Characterization Util.;citation_author=D. J. Bertioli;citation_author=G. Seijo;citation_author=F. O. Freitas;citation_author=J. F. M. Valls;citation_author=S. C. M. Leal-Bertioli;citation_title=An overview of peanut and its wild relatives.;citation_pages=134-149;citation_volume=9;citation_year=2011;citation_doi=10.1017/S1479262110000444" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=D. J. Bertioli;citation_author=B. Vidigal;citation_author=S. Nielen;citation_author=M. B. Ratnaparkhe;citation_author=T.-H. Lee;citation_title=The repetitive component of the A genome of peanut (Arachis hypogaea) and its role in remodelling intergenic sequence space since its evolutionary divergence from the B genome.;citation_volume=112;citation_year=2013;citation_pmid=23828319;citation_doi=10.1093/aob/mct128" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. D. Burow;citation_author=C. E. Simpson;citation_author=J. L. Starr;citation_author=A. H. Paterson;citation_title=Transmission Genetics of Chromatin From a Synthetic Amphidiploid to Cultivated Peanut (Arachis hypogaea L.): Broadening the Gene Pool of a Monophyletic Polyploid Species;citation_pages=823-837;citation_volume=159;citation_year=2001;citation_issue=2;citation_pmid=11606556" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=A. Butcher;citation_author=R. Williams;citation_author=D. Whitaker;citation_author=S. Ling;citation_author=P. Speed;citation_title=Improving linkage analysis in outcrossed forest trees - an example from Acacia mangium.;citation_pages=1185-1191;citation_volume=104;citation_year=2002;citation_issue=6-7;citation_pmid=12582629;citation_doi=10.1007/s00122-001-0820-1" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=K. K. Cheema;citation_author=N. S. Bains;citation_author=G. S. Mangat;citation_author=A. Das;citation_author=Y. Vikal;citation_title=Development of high yielding IR64 x Oryza rufipogon (Griff.) introgression lines and identification of introgressed alien chromosome segments using SSR markers.;citation_pages=401-409;citation_volume=160;citation_year=2008;citation_doi=10.1007/s10681-007-9560-x" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=H.-K. Choi;citation_author=M. A. Luckow;citation_author=J. Doyle;citation_author=D. R. Cook;citation_title=Development of nuclear gene-derived molecular markers linked to legume genetic maps.;citation_pages=56-70;citation_volume=276;citation_year=2006;citation_issue=1;citation_pmid=16642337;citation_doi=10.1007/s00438-006-0118-8" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=A. P. Fávero;citation_author=C. E. Simpson;citation_author=J. F. M. Valls;citation_author=N. A. Vello;citation_title=Study of the Evolution of Cultivated Peanut through Crossability Studies among Arachis ipaensis, A. duranensis, and A. hypogaea;citation_pages=1546-1552;citation_volume=46;citation_year=2006;citation_issue=4;citation_doi=10.2135/cropsci2005.09-0331" />
<meta name="citation_reference" content="citation_journal_title=Bonplandia;citation_author=A. Fernández;citation_author=A. Krapovickas;citation_title=Cromosomas y evolución en Arachis (Leguminosae).;citation_pages=187-200;citation_volume=8;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=D. Foncéka;citation_author=T. Hodo-Abalo;citation_author=R. Rivallan;citation_author=I. Faye;citation_author=M. N. Sall;citation_title=Genetic mapping of wild introgressions into cultivated peanut: a way toward enlarging the genetic basis of a recent allotetraploid.;citation_pages=103-103;citation_volume=9;citation_year=2009;citation_pmid=19650911;citation_doi=10.1186/1471-2229-9-103" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. Foncéka;citation_author=H.-A. Tossim;citation_author=R. Rivallan;citation_author=H. Vignes;citation_author=E. Lacut;citation_title=Construction of Chromosome Segment Substitution Lines in Peanut (Arachis hypogaea L.) Using a Wild Synthetic and QTL Mapping for Plant Morphology.;citation_pages=e48642-e48642;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23185268;citation_doi=10.1371/journal.pone.0048642" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. Gautami;citation_author=D. Foncéka;citation_author=M. K. Pandey;citation_author=M. C. Moretzsohn;citation_author=V. Sujay;citation_title=An international reference consensus genetic map with 897 marker loci based on 11 mapping populations for tetraploid groundnut (Arachis hypogaea L.).;citation_pages=e41213-e41213;citation_volume=7;citation_year=2012;citation_issue=7;citation_pmid=22815973;citation_doi=10.1371/journal.pone.0041213" />
<meta name="citation_reference" content="citation_journal_title=Pisum Genet.;citation_author=M. Gawłowska;citation_author=W. Święcicki;citation_author=B. Wolko;citation_title=Comparison of genetic maps for two pea populations (Pisum sativum L.).;citation_pages=19-23;citation_volume=37;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Grattapaglia;citation_author=R. Sederoff;citation_title=Genetic linkage maps of Eucalyptus grandis and Eucalyptus urophylla using a pseudo-testcross: mapping strategy and RAPD markers.;citation_pages=1121-1137;citation_volume=137;citation_year=1994;citation_issue=4;citation_pmid=7982566" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=Y. Guo;citation_author=S. Khanal;citation_author=S. Tang;citation_author=J. E. Bowers;citation_author=A. F. Heesacker;citation_title=Comparative mapping in intraspecific populations uncovers a high degree of macrosynteny between A - and B - genome diploid species of peanut.;citation_pages=608-608;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23140574;citation_doi=10.1186/1471-2164-13-608" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=T. M. Halward;citation_author=H. T. Stalker;citation_author=E. A. Larue;citation_author=G. Kochert;citation_title=Genetic variation detectable with molecular markers among unadapted germplasm resources of cultivated peanut and related wild species.;citation_pages=1013-1020;citation_volume=34;citation_year=1991;citation_doi=10.1139/g91-156" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=G. He;citation_author=C. S. Prakash;citation_title=Identification of polymorphic DNA markers in cultivated peanut (Arachis hypogaea L.).;citation_pages=143-149;citation_volume=97;citation_year=1997;citation_doi=10.1023/A:1002949813052" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. Hopkins;citation_author=A. Casa;citation_author=T. Wang;citation_author=S. Mitchell;citation_author=R. Dean;citation_title=Discovery and Characterization of Polymorphic Simple Sequence Repeats (SSRs) in Peanut;citation_pages=1243-1247;citation_volume=39;citation_year=1999;citation_issue=4;citation_doi=10.2135/cropsci1999.0011183X003900040047x" />
<meta name="citation_reference" content="citation_journal_title=Plant Syst. Evol.;citation_author=K. W. Hilu;citation_author=H. T. Stalker;citation_title=Genetic relationships between peanut and wild species of Arachis sect. Arachis (Fabaceae): Evidence from RAPDs.;citation_pages=167-178;citation_volume=198;citation_year=1995;citation_doi=10.1007/BF00984735" />
<meta name="citation_reference" content="citation_journal_title=Cytologia (Tokyo);citation_author=L. Husted;citation_title=Cytological Studies on the Peanut, Arachis. II. Chromosome number, morphology and behavior, and their application to the problem of the cultivated forms.;citation_pages=396-423;citation_volume=7;citation_year=1936;citation_doi=10.1508/cytologia.7.396" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=G. Kochert;citation_author=T. Halward;citation_author=W. D. Branch;citation_author=C. E. Simpson;citation_title=RFLP variability in peanut (Arachis hypogaea L.) cultivars and wild species.;citation_pages=565-570;citation_volume=81;citation_year=1991;citation_pmid=24221368" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=G. Kochert;citation_author=H. T. Stalker;citation_author=M. Gimenes;citation_author=L. Galgaro;citation_author=C. R. Lopes;citation_title=RFLP and cytogenetic evidence on the origin and evolution of allotetraploid domesticated peanut, Arachis hypogaea (Leguminosae).;citation_pages=1282-1291;citation_volume=83;citation_year=1996;citation_doi=10.2307/2446112" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=T. G. Krishna;citation_author=R. Mitra;citation_title=The probable genome donors to Arachis hypogaea L. based on arachin seed storage protein.;citation_pages=47-52;citation_volume=37;citation_year=1988;citation_doi=10.1007/BF00037222" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=E. S. Lander;citation_author=P. Green;citation_author=J. Abrahamson;citation_author=A. Barlow;citation_author=M. J. Daly;citation_title=MAPMAKER: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations.;citation_pages=174-181;citation_volume=1;citation_year=1987;citation_issue=2;citation_pmid=3692487;citation_doi=10.1016/0888-7543(87)90010-3" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=S. C. M. Leal-Bertioli;citation_author=A. C. F. V. José;citation_author=D. M. T. Alves-Freitas;citation_author=M. C. Moretzsohn;citation_author=P. M. Guimarães;citation_title=Identification of candidate genome regions controlling disease resistance in Arachis.;citation_pages=112-112;citation_volume=9;citation_year=2009;citation_pmid=19698131;citation_doi=10.1186/1471-2229-9-112" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Lu;citation_author=B. Pickersgill;citation_title=Isozyme variation and species relationships in peanut and its wild relatives (Arachis L. - Leguminosae).;citation_pages=550-560;citation_volume=85;citation_year=1993;citation_pmid=24195929" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Res Notes;citation_author=S. E. Macedo;citation_author=M. C. Moretzsohn;citation_author=S. C. M. Leal-Bertioli;citation_author=D. M. T. Alves;citation_author=E. G. Gouvea;citation_title=Development and characterization of highly polymorphic long TC repeat microsatellite markers for genetic analysis of peanut.;citation_pages=86-86;citation_volume=5;citation_year=2012;citation_pmid=22305491;citation_doi=10.1186/1756-0500-5-86" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. C. Moretzsohn;citation_author=L. Leoi;citation_author=K. Proite;citation_author=P. M. Guimarães;citation_author=S. C. M. Leal- Bertioli;citation_title=A microsatellite-based, gene-rich linkage map for the AA genome of Arachis (Fabaceae).;citation_pages=1060-1071;citation_volume=111;citation_year=2005;citation_issue=6;citation_pmid=16088397;citation_doi=10.1007/s00122-005-0028-x" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=M. C. Moretzsohn;citation_author=A. V. Gobbi Barbosa;citation_author=D. M. Alves-Freitas;citation_author=C. Teixeira;citation_author=S. C. M. Leal-Bertioli;citation_title=A linkage map for the B-genome of Arachis (Fabaceae) and its synteny to the A-genome.;citation_pages=40-40;citation_volume=9;citation_year=2009;citation_pmid=19351409;citation_doi=10.1186/1471-2229-9-40" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=M. C. Moretzsohn;citation_author=E. G. Gouvea;citation_author=P. W. Inglis;citation_author=S. C. M. Leal-Bertioli;citation_author=J. F. M. Valls;citation_title=A study of the relationships of cultivated peanut (Arachis hypogaea) and its most closely related wild species using intron sequences and microsatellite markers.;citation_volume=111;citation_year=2013;citation_pmid=23131301;citation_doi=10.1093/aob/mcs237" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=E. D. Nagy;citation_author=Y. Guo;citation_author=S. Tang;citation_author=J. E. Bowers;citation_author=R. A. Okashah;citation_title=A high-density genetic map of Arachis duranensis, a diploid ancestor of cultivated peanut.;citation_pages=469-469;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22967170;citation_doi=10.1186/1471-2164-13-469" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=S. Nielen;citation_author=B. S. Vidigal;citation_author=S. C. M. Leal-Bertioli;citation_author=M. Ratnaparkhe;citation_author=A. H. Paterson;citation_title=Matita, a new retroelement from peanut: characterization and evolutionary context in the light of the Arachis A-B genome divergence.;citation_pages=21-38;citation_volume=287;citation_year=2012;citation_issue=1;citation_pmid=22120641;citation_doi=10.1007/s00438-011-0656-6" />
<meta name="citation_reference" content="citation_author=A. Oliphant;citation_author=DL. Barker;citation_author=JR. Stuelpnagel;citation_author=MS. Chee;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=G. Robledo;citation_author=G. Seijo;citation_title=Species relationships among the wild B genome of Arachis species (section Arachis) based on FISH mapping of rDNA loci and heterochromatin detection: a new proposal for genome arrangement.;citation_pages=1033-1046;citation_volume=121;citation_year=2010;citation_issue=6;citation_pmid=20552326;citation_doi=10.1007/s00122-010-1369-7" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Botany;citation_journal_abbrev=Am. J. Bot.;citation_author=G. Seijo;citation_author=G. I. Lavia;citation_author=A. Fernandez;citation_author=A. Krapovickas;citation_author=D. A. Ducasse;citation_title=Genomic relationships between the cultivated peanut (Arachis hypogaea, Leguminosae) and its close relatives revealed by double GISH;citation_pages=1963-1971;citation_volume=94;citation_year=2007;citation_issue=12;citation_pmid=21636391;citation_doi=10.3732/ajb.94.12.1963" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. S. Senchina;citation_author=I. Alvarez;citation_author=R. C. Cronn;citation_author=B. Liu;citation_author=J. Rong;citation_title=Rate variation among nuclear genes and the age of polyploidy in Gossypium.;citation_volume=20;citation_year=2003;citation_pmid=12679546;citation_doi=10.1093/molbev/msg065" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. M. Sewell;citation_author=B. K. Sherman;citation_author=D. B. Neale;citation_title=A Consensus Map for Loblolly Pine (Pinus taeda L.). I. Construction and Integration of Individual Linkage Maps From Two Outbred Three-Generation Pedigrees;citation_pages=321-330;citation_volume=151;citation_year=1999;citation_issue=1;citation_pmid=9872970" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=K. Shirasawa;citation_author=D. J. Bertioli;citation_author=M. C. Moretzsohn;citation_author=S. C. M. Leal-Bertioli;citation_author=R. K. Varshney;citation_title=An integrated consensus map of cultivated peanut and wild relatives reveals structures of the A and B genomes of Arachis and divergences with other legume genomes.;citation_volume=20;citation_year=2013;citation_pmid=23315685;citation_doi=10.1093/dnares/dss042" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Schmutz;citation_author=S. B. Cannon;citation_author=J. Schlueter;citation_author=J. Ma;citation_author=T. Mitros;citation_title=Genome sequence of the palaeopolyploid soybean.;citation_pages=178-183;citation_volume=463;citation_year=2010;citation_issue=7278;citation_pmid=20075913;citation_doi=10.1038/nature08670" />
<meta name="citation_reference" content="citation_title=Grain Legumes: Evolution and Genetic Resources;citation_title=The groundnut, Arachis hypogaea L;citation_author=J. Smartt;citation_pages=30-84;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. P. Tallury;citation_author=K. W. Hilu;citation_author=S. R. Milla;citation_author=S. A. Friend;citation_author=M. Alsaghir;citation_title=Genomic affinities in Arachis section Arachis (Fabaceae): molecular and cytogenetic evidence.;citation_pages=1229-1237;citation_volume=111;citation_year=2005;citation_issue=7;citation_pmid=16187123;citation_doi=10.1007/s00122-005-0017-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. Tani;citation_author=T. Takahashi;citation_author=H. Iwata;citation_author=Y. Mukai;citation_author=T. Ujino-Ihara;citation_title=A Consensus Linkage Map for Sugi (Cryptomeria japonica) From Two Pedigrees, Based on Microsatellites and Expressed Sequence Tags;citation_pages=1551-1568;citation_volume=165;citation_year=2003;citation_issue=3;citation_pmid=14668402" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. E. Van’t Hof;citation_author=F. Marec;citation_author=I. J. Saccheri;citation_author=P. M. Brakefield;citation_author=B. J. Zwaan;citation_title=Cytogenetic characterization and AFLP-based genetic linkage mapping for the butterfly Bicyclus anynana, covering all 28 karyotyped chromosomes.;citation_pages=e3882-e3882;citation_volume=3;citation_year=2008;citation_issue=12;citation_pmid=19060955;citation_doi=10.1371/journal.pone.0003882" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. E. Voorrips;citation_author=G. Gort;citation_author=B. Vosman;citation_title=Genotype calling in tetraploid species from bi-allelic marker data using mixture models.;citation_pages=172-172;citation_volume=12;citation_year=2011;citation_pmid=21595880;citation_doi=10.1186/1471-2105-12-172" />
<meta name="citation_reference" content="citation_journal_title=J. Plant Breeding Crop Sci.;citation_author=B. Wang;citation_author=P. W. Chee;citation_title=Application of advanced backcross quantitative trait locus (QTL) analysis in crop improvement.;citation_pages=221-232;citation_volume=2;citation_year=2010" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Use of SNP Markers for Linkage Mapping in Diploid and Tetraploid Peanuts" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1.cover-source.jpg" />
<meta name="twitter:description" content="Single nucleotide polymorphic markers (SNPs) are attractive for use in genetic mapping and marker-assisted breeding because they can be scored in parallel assays at favorable costs. However, scoring SNP markers in polyploid plants like the peanut is problematic because of interfering signal generated from the DNA bases that are homeologous to those being assayed. The present study used a previously constructed 1536 GoldenGate SNP assay developed using SNPs identified between two A. duranensis accessions. In this study, the performance of this assay was tested on two RIL mapping populations, one diploid ( A. duranensis × A. stenosperma ) and one tetraploid [ A. hypogaea cv. Runner IAC 886 × synthetic tetraploid ( A. ipaënsis × A. duranensis )4×]. The scoring was performed using the software GenomeStudio version 2011.1. For the diploid, polymorphic markers provided excellent genotyping scores with default software parameters. In the tetraploid, as expected, most of the polymorphic markers provided signal intensity plots that were distorted compared to diploid patterns and that were incorrectly scored using default parameters. However, these scorings were easily corrected using the GenomeStudio software. The degree of distortion was highly variable. Of the polymorphic markers, approximately 10% showed no distortion at all behaving as expected for single-dose markers, and another 30% showed low distortion and could be considered high-quality. The genotyped markers were incorporated into diploid and tetraploid genetic maps of Arachis and, in the latter case, were located almost entirely on A genome linkage groups." />
<meta name="og-title" property="og:title" content="The Use of SNP Markers for Linkage Mapping in Diploid and Tetraploid Peanuts" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/1/89" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Single nucleotide polymorphic markers (SNPs) are attractive for use in genetic mapping and marker-assisted breeding because they can be scored in parallel assays at favorable costs. However, scoring SNP markers in polyploid plants like the peanut is problematic because of interfering signal generated from the DNA bases that are homeologous to those being assayed. The present study used a previously constructed 1536 GoldenGate SNP assay developed using SNPs identified between two A. duranensis accessions. In this study, the performance of this assay was tested on two RIL mapping populations, one diploid ( A. duranensis × A. stenosperma ) and one tetraploid [ A. hypogaea cv. Runner IAC 886 × synthetic tetraploid ( A. ipaënsis × A. duranensis )4×]. The scoring was performed using the software GenomeStudio version 2011.1. For the diploid, polymorphic markers provided excellent genotyping scores with default software parameters. In the tetraploid, as expected, most of the polymorphic markers provided signal intensity plots that were distorted compared to diploid patterns and that were incorrectly scored using default parameters. However, these scorings were easily corrected using the GenomeStudio software. The degree of distortion was highly variable. Of the polymorphic markers, approximately 10% showed no distortion at all behaving as expected for single-dose markers, and another 30% showed low distortion and could be considered high-quality. The genotyped markers were incorporated into diploid and tetraploid genetic maps of Arachis and, in the latter case, were located almost entirely on A genome linkage groups." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1.cover-source.jpg" />
    <title>The Use of SNP Markers for Linkage Mapping in Diploid and Tetraploid Peanuts | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Q09WnuqxiVBUZPOJwXljExSmiv9s9f_nofQuow3tPp8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459619","apath":"\/ggg\/4\/1\/89.atom","pisa":"ggg;4\/1\/89","ac":{"\/ggg\/4\/1\/89.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/1\/89.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/1\/89"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/1\/89"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/1\/89":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022U84Q+JbuNd4HY2hJdyU538tLmMX82Z41OWV0B09p32h03IdENQ7yO8WO6iUzV0OvWxRgOFeZwyiCh\\\\\\\/gZ4jVBx6SPwir1qXS6DwpU0a6sTeUqu\\\\\\\/bhCi0o6ufRGcTVEpQcCpJAIV9UzrETp15SebAMPkcoeyhErQwNRWQQ2GN4sIEcLscfmK2haWpzda5wGZbmPvAt\\\\\\\/9oWSve9JoX5Lu9n5tc7X7AYEB47SbFaDU01wRhN8winJnsY2c0+Zmq9PLIRbYH4TdKSsCaFL0fpAA+Om6pdvqKmzkM5mUppBpA2x2XY2bx4as2ieGL05mKQnw1nSnMyiSJjjVlzuaxrQ5oOw3DPaidemkMOWYsD30cgJ1AopEsBBotvdRjoP162\\\\\\\/dASGaUEB7hEtHXlcz7J\\\\\\\/XetwO5jWVr10DtxBemE9WiS5e1vWZJXKK7ShcjrXKUYBXvS0lZse+pnI3tP6hYZHyp0f4mAc6xTV0nHo66WD7viZTtAl7oglCUvf5icIJOsoFzeWEHwQj4kW+OSSbBb9lJVcNLH7jfNEf4hyxM36f6N2O22gt07y4XkMnVz2lTrq8p3O78wjcRnns6fq1o6IH21ugVG9QjMGJBGsbv++RwsG82LCQf0\\\\\\\/3LbsbSAHeAtSNXl0xH0gyV0\\\\\\\/TV3iDuxAYNymEfYALCPX5QrXqnmYMMg6cvLDEK0Ne2059M5u1FkHbz\\\\\\\/i\\\\\\\/x7ZrpsZVKF9su3gTfzOHA7Y2Fv7XenJzhghBXTitY2Y9Bfmffsydkjdwsys74llPLjL8BpRZFsyyVEt1dRaKUsg8abGnjUqfiMOgMfSFDt7QxdaA6uBdCoDA1coisDJ3KiQ7U+XceCtINO\\\\\\\/MKE4P0B3bz0rrbMQy2N3eLP0ooPF6gnuZrpzffezOOI0Xkkt8opVcpHuA9ZiiL2ZUroEqMQ5kbmW1yNyeJac97NHUcOyK+dq9gLZcwOIg9wtsVQD8cTBnqDCAVr68A9Me4sqHtUpMQTvzc37D+Vdk9\\\\\\\/zCivWPJht6uoImLX2Im0KatUahiU\\\\\\\/DH3SuA59SSsZjYndiGGBncaNQr+4hIywVjH0xSJS3wvp\\\\\\\/1DhCRM8IAw49VGmPjwmiN7wmPPFgI1nsGgG1V\\\\\\\/+IiLYbN+XBz9UQDg8ZxF4ZAsYZp\\\\\\\/ghI6lgJLb4tT4puJATOZFXt1xfUQhJYSTYCcYSrjZ7FnITKMzoaGsKN18wrKdBLXCq68NZYlbQcB9ECneo0c5DAwJfH5xoy1FBgvIjVD1f22URgBVHt2I3XNRbujz1O+aM\\\\\\\/m2DG8ti+b\\\u0022,\\\u0022iv\\\u0022:\\\u0022XQBrD85PPoxzQGm0Sp6oSA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002293d809e83f202ac8bf82955ad41cbb7ce11140534e39b0145d9d4094eff61522\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022fdbLJn+sYvZUVR\\\\\\\/9FkNQk0guehONRrNWAkHepwa2w0NjyO4DvLG+9Evq3kr41j5Pw\\\\\\\/eSAdFAdEo88BNdMib9Mlh8u7RhTc8Rdo6e5DV7MPqtgOT6bQOIeiDdUo\\\\\\\/BL7t\\\\\\\/g6TwhS4hXrNyGoTpv9+R3Hc1tF4t\\\\\\\/4OE5\\\\\\\/4w9Pty5OZkD\\\\\\\/66NVghy6ebfDE7rum2jufjgWJjfXRfpfUARM3rBjy\\\\\\\/zA0k8D8YVWEEEpr8\\\\\\\/0ByinNuBHLFUaJ2McYf0epCEv2eY26CyFGRfMRPdahqxG+YBuunopPyQMR\\\\\\\/+ucMpgfKoubwmHKOavbM9ZweNWgxFIr\\\\\\\/51qa613sv6cxQfbnMJqcTIN7YLWEQDjQpnQCWJLaDF91Wgogow1YNDZlNiUBT6Q27BJU3rlXZuJYuDgIArP73Br8idQt5DuLYLtXAwY+LN8PbavDZOFZ0KaqpH28cVSFc+qlqXubukX4gYcFT1Q9T2IcaVs1+W7EY5noOpWJ9hlUl\\\\\\\/LFRi12+jlAaoU8T0i2+eOpxQoOqRKFISGrTXJpgM5DvDSd4jakTUJe7zC0EEyBqVz2xragF\\\\\\\/5uEzcLmRaKsKFcegnuP1mq\\\\\\\/H8VELFktvDSx7rOEATCrklt2GBY7tiDqeCBRuCIO5s19H+mNrP5gAnUxZqlIGhp+hhFb1NaQIe6GuPINVgeMpSD9ibanObCQ7Tgj9yTBmjZB\\\\\\\/H\\\\\\\/4MdSAsF8\\\\\\\/9ZLMYRrJMbaI6p3VsgVQ3Xw\\\\\\\/zqnZNHfzFPy9bAddJizB8oaLc+hat70LyX9aCpkOrsbLnmdl6T0ZPOI6dX5PPqJAbBM51BehUz4cN+uFb4d3GXIVhfLtLqkQUshzMOUZJgp\\\\\\\/TN3EIqaFSjRdhJ2JOOKndgQZBS0FvpUGnPJVM35wxxjs7CwhIhqmjq7iGs\\\\\\\/AHaybNC2ITQDRa82uSo9viXYOiGlvA2e0vqsv\\\\\\\/FkOQQQBj6PssGZJkid8hsGolCYSJ2SknusKR168qVErevhhk46yZEAB8\\\\\\\/EQL3eMUEzymXcbgEvJAy+yK9MkR5kQfRc8o7ypHIrhJYrJsWuX54i\\\\\\\/pnd1cmd5J+ByjhCu6SuQ2uFn65fuPjWxsXGGQW7KSBmMLSfqMh+86t0sXt9LhHSOwwMacx1H8nJrdg\\\\\\\/FXP58nD8f3jkbFeronnylcmRPeAHHafE4w+BNhic2bCPtbARNsulcS0PxZqI4ENlRAg69dxiZn\\\\\\\/cZu01dNPF6ckoj1s0QniNf2A0As3JwQ9wIWCX3AhMxdgdnduGuJwXooVbBgPqunqfFYLQchJ8esKz0L75ANu\\\\\\\/dJcV3Kbhn8C9fugORoQdSvkzIgEWPWvPCta5jDmZWgp2ZhwRmCPREC81XQaa8fDClY5QASbNOaQ1StHsJ\\\\\\\/VFTtCHFtfi4hthore9AbsQBjI\\\\\\\/DBERv2tI\\\\\\\/Q\\\\\\\/AfJ40xfJyBboWaXV+dC2LxFWcYNv291GjgULihwOD4foeLGddJv2pJdm0cDCRxzYp6BP5AjcyLIxUXeuRwlEdtcxd2RtX52DfNDsHzhgXMEZCDvBdfyvI1l6QpnqmrjcNtRYZ2HDNaJR9SA5Go5plUg6Ya1kZ6F3oGdWDgCWmgFLCSOIGDd2a5BhoxPzeTqvQZcdnZZUiAAHZY74g7X6DDSCPHczVcrq954FLAOezMxhxK2QzWaaY8eOmwj2I06L257xnmA+VOOcKKBYZix25QofcU3P\\\\\\\/TBwl25DYELjusJg1q213T4HoylAUAgELlEB9pib6n1Sct4q9AoSEalqc8i8TaztcLvZ+ZmmOhFSOzI4SvNbMIyIDvxc+dzDhZv92106xbmZvtpBLTih5GlqhGZs+eCw99hC6+QT\\\\\\\/oDBerEf+B0R5P9wzM0NiRK1OYb9b\\\\\\\/Y+gW0xCJVHITjtGCRkj51+tWKG5j7P3lBASYdaJFNMsd2KwX5S9LJGBaKKcfdPPu9VTdbfKJyoxIBA65dRatuTDP05459h0sGTK\\\\\\\/JqnlwQu49gOqvorOYNqwxif\\\\\\\/KNS7TBH2dLfgwvL5SxuV0H8eMZjah22536WDKaEg5cmiIFOU94MY5c2QnO\\\\\\\/8qL83792tzZrGcyIy4llL7pshGYbfhGfPMVYT\\\\\\\/h7DNxC8BK91h\\\\\\\/pKU4NurNMmoRSny+ZJYMYn2feKgQUrhE9eAwt3lKhI86aO0OZW\\\\\\\/pOeI8vYt6sfmjgUruhmIyY0YEQDc4\\\\\\\/lm\\\\\\\/FBccYxZdqxWvThsBtJKlPlKxeMsjfJnOIsbqz5Gi3H6xpPhjWNKVNWJ01OGWbeMwt8jOZIgCqw2YcVtQi4V2OhnTe1aWd+jOEBJKSvNRCG0\\\\\\\/CLOc66ShxZe7sdK0lO9U4AM7SEkIaZbliKGLrCFB\\\\\\\/KdLIQUSRen6+1fHoN5MlNwu50NcKeoTOk6lKtiLlqmETJuQbHG8uo1NDBkYLOWBs7AFHlqyW\\\\\\\/WC3EbkIIhBCyPkIOTvUSxmnQG6QkuPW3HjAJ\\\\\\\/7mdMbwGoQF6sN4fBIpPK39o1kyPHCLq2MFqVGpiuoZDu\\\\\\\/I9ci7zbGukkDV8XBtDvnJRENSc5AZeqjJlYUVVPDzhkUR5I8Y43xeFLSodLMsXXpRG2keOtxrzY\\\\\\\/3eq1wThJigcrcqIWu9ynfNd2j6PETPAZEjA\\\\\\\/1ByqwJ0apkkufe3eqtVo0HkfaMSUXMYm5JNMa1BlyFBeALI4Ac9NYW1pMfI81yyvzMfZWQvD6fyFtDbiDfxDIDrxp5j\\\\\\\/L+NOFjHn4NkOOqRlOv9Hj4plYLbM\\\\\\\/PdVTg\\\\\\\/6Cp1QmoqOjq73+3Rz89f89nMjqdFwAa6Bolm6cb7tOQOIuET7HWjrhKKJeuPwMtALRJZMN\\\\\\\/\\\\\\\/fVcv1WKhmejm530viMIfXOM\\\\\\\/5I9TOEbhQSI1LJ8FVy6\\\\\\\/dGqOK03XHbykRzEJ3QLOzIiykqA8tep8TGo6BZBtGQ\\\\\\\/HdPi5CCNQeVyac5TaWWcTqdeuvGLNP+gEyhv\\\\\\\/AaF4T1g6Atff9nHL65ZRt2kIPueLZ47jhkrQm2ozlSOkmKoSsyKqkI+6MpPWFDdokaLKy4oCMUxNgUYZ0DjRODyb9RbMGOI+20hTM3QN5xE\\\\\\\/gKUDDaHL5x6gQW0tU6deQA1oNt9E7NvS1qKzjG1OG4604kTINqAP+HlPpf+N49SSZFexXEoEOo0bi0GDDsBBfa4oiaZgXhFpig14ZigWf47APCfOLwSFoWcmtVzdXRPDggdYIl2NXZF4y1OrSaE2S1jpnUuEaEgW1j+tfUqimsrMTP8krYV+lfDK5ojQKLzYy7IiA0e9\\\\\\\/VUB+kXAjSuFT\\\\\\\/MenraEnAdckm1mqgRiCaSwDtcV6J7YfesLs6HEapW7EtM5ZhRz0lgNVdTWC9O2pq0Erc\\\\\\\/8maj9RE4bHtDVpisPbXTE1+OhT6R5ETM1\\\\\\\/Re1vpwgWdd5apzckPMYwy79isGhPY+jjYQM3UsCmNnujr2pzMy5jSwGBUJLyxQetNS85eVbh1UoosmGI+BFbZRbYncUNDgnq8Ji+5ik+L+Z0YXTJsp+ds5cptQS3N94hjZsclNXZP5UsKfrZnBD54+RG2RDdHCMnWKz4aK6HjuUZT7QTsIhyXF\\\\\\\/EwFIqIBx9z5UMzXz6STr3J1nvn2P3cQjafG+Q40sZMYXA7CfNZx3+eC9n2IN7LpN5oIFBENwKAdOyirCEF+jZaC9oOEXxtldwTVjZi\\\\\\\/0TYoSwcbytJ6t28JpbrUEYlcdhfLijj72WNZWq9XQXpPWA4hDpd+q38XzY+ij3nnIups4g3D8+srAtLzZwNe+ekBXAWB6gQsQLX9rsXNmkDo4HtV3qCDeLJfjNysF9KMQDk1RCMAUp8EWnebs4w7OTGmdSKWFTdBtPKVtbKpp\\\\\\\/\\\\\\\/GBW9loQXc7zSLhanvZ2lvXw+Ny\\\\\\\/ulJo39JpZfnPguJuqkXuPBN9RIVhcQP+YnKa2uxEHuLKMNLneuddZdUDOEs5rMJ7WiWrCjVDxFKQz5f0vMWJfdnoPVnlYpZcfho8VOPPYcYsYYo7a7CkAmOvmxjhLKsx4OY6k5eOSwFUg55dtHY1yEPJ8jsFTD8ho2\\\\\\\/BT+rSLQcUpEEyqFDNFVyXwI6+z3hUVVfag0YFkg2keeAvhDI76EyGCXZK2U2kbimuqk1AO9nN\\\\\\\/tcRrRjSMP2lUoRg2WIQluq4w2L2e8JroNp41xg\\\\\\\/eAHfhEw6kjko3w50s0YlaszX7d1yggMvf+AiK2ICVcecy3xM65fDo6yTs3SQAK\\\\\\\/i1OOursyUy+I7GdRowrGd04ZuFDhLllTRji5suT4Vz6TIVEbF5R8r6G6BG3lJD\\\\\\\/zcwvnu93ew1Fwrw+d+iMpgu3eaCHllm4etwmQUS87s+4EcRI+ohQ884uUofRVa\\\\\\\/UW\\\\\\\/qubigUH2C3JyfYNaBYGdRZ3um2PJEmrbPeqe0NTo7ejj+e0m1s\\\\\\\/MxNIiOn9g8TrnzToDCQZsH0AHbSfAjCHUwaPbbWrpgQ4SgxEdnq6D7jozxDYJkS+brcHwDxBDbe0sp3y9HU5G0q8UgyvfTrgXmlyTvJSbZQNiQLpQVHjgdHv6Ag0OE39uNO4P6mOT+DiWA+cj88cch9RI0ZRxstb4M6OH4fCscFOF+GH2xyHz+GRWzElq7bEK3OYVEvLLTn4qtahHQZ7ubEz043D62rAXZG0w5C8VP7GHhS1We3rwotYWdGH+2Xw9IDgj6meMWjVx+STT8qj7N2bIPLiLZ2+QpuRroml0bkdwBZnAUV\\\\\\\/6n4anppMoNqt5\\\\\\\/I8SCJ03LAE4hc+flR84xq922iP1QBxon2XTA4IlTWfuWbl17Bj9EuAUlJwrCHT0vvb4dbVlVeLspcxh0P2Wgt7pJR8kbhQ2KFLWgKyH6xJcCgExw2Ql4PtrVYPte1bvphNPcJDT\\\\\\\/0Kd2ZMpc2fceLxIhguLYLEtg2iea1YJ8L4kl+kdUTRxlxhsfYt2Twp9OeqYiB4g70hCt38K+bm+Pv7CyhEcgnhkuNZiWL3POzekYQ40eWxqrsuTjqxLJKAVJbtfNPSLMgnevMtgN6D40MKo2RPjKgJ1acFUsxidz7c2zMBPHAFM+5KjmUAMoQZwPnTn9wUGxck\\\\\\\/BIvV+fDgmqIsl904x6d9ZC868uz0mDN+u1JnLAcBOi+horKYWQZA5z7FCQwnINiRBca9ROT\\\\\\\/Q9NA0OJari8wS2Zm\\\\\\\/\\\\\\\/hoS4P+HzE8bB+NLLhszZ7e4rlHMa5LXaP88dWh+P6syClcR7clClyzPg25cpJm8dBVqRipgr4gtCQhx1gAlbTI5cjneCJ0+ERH4YnLlw+glJmXIyTT8Fd0dyiO+1KOt+uCBwTrS7OmgNwOh9j\\\\\\\/i2rxa3UKmSmx2rTnDwDvPPj2SdzpHrfudq9Jx1wG9XZX3HWy37rgU2FUkZfYYjOaY0uptB6pHqH14d4f0we7SktQAZFy8GsSClKo6fOYnJz71pLZ9pNVfR81lGUin3\\\\\\\/zUzbh6kJUgfOkd7309doZr2ezoqbUek\\\\\\\/5KgwkbiWQMEChjcFzgnxhSFlpiICbVKZcMx5LmRGK0ExOksQ0iobdozihJtSCVAskIE7QLtS4KOSz3b578jLgElkdAlfZ6z1akXoZaIh+1DdYw0l5YMpr4h4jQBp\\\\\\\/YNpvqSuNfi1uXwVCb+V+Ab\\\\\\\/mglCYi4M1qpdo+jAmqVtgO+N6SmKqE8D699aA+wFlnAooIdSAhZG7G1b8mAOmUOJdn90J7f3Ol8FJCqC+XayruQoIgiBN1AYzWVKBLPZD17UNYja7SM6kMPKllg0U44BbXPQCBg\\\\\\\/LGTLFvbM7Efl7L9e+zkDg4nRQ6W0eiTLAWmNow0J9oK\\\\\\\/GruQ0dovfCRVhcILtAz9obb5cXwlu+ZUlP4iIYsyOrtrJWbJYhXu5kuDXWPADfWGzvu0QAiRcvDU9289qchteHb8H8T5\\\\\\\/ZmM8oljsu7V3jy3kcC20T8\\\\\\\/zos2LZhiBcYzuF4WYmwGqjoIZHVKlNYnRXVZWxoRyKNNJZu6VTCMINPETo+57POIAr5VmzKRNZl4nyht8Z1eiUO+XXEi2PhrQXc4ZZZaSCwnrnzG\\\\\\\/KxH+MY+QSffctwnjCk+IamYv7TcpP0DXH93139zg\\\\\\\/letz6CatLxqDLtk0fLHTmrkH1XrFYvlL103LSw7MzXFiRQ8kfhK1h6B1aHFq+ZFsnQPFl5kyTeSE4D8a6kE4aFd2kD+VqLw+33TQ0nYavfSeqB0cOa74FdOV3esyeTk2rIhfAYncR8KkyNytm5ITcBggv\\\\\\\/ASsb1aD9\\\\\\\/iDqa2PHu+E7BCfDI1EUk90DzscKP3nrG3DvMco7zpL1iK\\\\\\\/nLrnV\\\\\\\/id1cZ7iPd9xXtw00rvrlBTSdjV57ZdkOJhbsoPCNgn7sfLnjK\\\\\\\/53NQDtf5genhGbLyJXbkL4ILjbAevOXmZNRK+bp2gy7g9mFnVePzsdE3a4ix3m0S8UK74W5FIX1Z77Rshqb3xYaFwteaq2U8kc8HvrVR6s3K51rqY+8rLIdqIK\\\\\\\/zolqifVEae6ncy9xMKInRZnYfZhlwcZkA+UEsQUj1x6wsYJW6i9Xg+\\\\\\\/k+NkKYqpUBcbflchBicoaZZ\\\\\\\/t4HVKXFv9+oEy\\\\\\\/eiKKJKgeqrAXxfUVBqEr8FTwR8\\\\\\\/\\\\\\\/Wduqux9d6OSf+vDRtV5jDjDgM2GegXfO1dfNfFXJzJ6OE8riSEzkV5dDqUxklMg0odCuy7YxtfJ2Xu\\\\\\\/1FGblCrbh660\\\\\\\/Py6+z+fe7k65U3XES197GWW7LKXlSgDyR5Ey395xuVWoOKqbss3+LL8k46DEkJFQhFnWYmV58dDlXobOflMwFQyPhAl3DjTKD07jXBxmi5mL5Z+bAl+F633V7dTHPL7G8+4HNNnqrfGjQO3Ehpq3E4jpGgcNKtY46h73G5AaFZR2u5mTmHUzz+h3BvfzgsaK4r83rLFuaZbh15RXKk7drwqs2JFEgh6JZ6hNnnPcLjpGd53vK8EX0XMRHv52IhRVEzxJL4XhyacKdGAcHYlawwdwYq+6svf1Sm4pwYmAF3fbrmaUaoUrilEj79ppb7mSd80cr56eCmBAaruyIqitreTU+3BfNok59\\\\\\\/FXkMYL\\\\\\\/D4QK7FAOFpwmjGzUC+AX6fuFaH1MSijS5AaBXohT3Df5sGAXxfIWXfdwo7E74yyB09huIF5HclYbEXummSvpjepyH\\\\\\\/RowmtX6+DwLFmPTs4HlC9YpWu8Dc9CTmuKN98Q+anS0s9MGU\\\\\\\/oiAU4s3AhF+UQZZ0HaYJI95c44t96sA05ujWjUMIDS\\\\\\\/Js8n5Kyr+aocPmmHLtQ8NQPD1DrPDVRch53lqOtN3ZuMAo\\\\\\\/qkeIe+Le9s+VHfMNBlN3bW0PzG0cTdf3z7T8SYYke3ZGFXiYjw0R63IyZF8fQ8pANMkQZ3qNRCjVCdh+h2Qmvkrpyd3KolkiX9EC0D\\\\\\\/\\\\\\\/o3qnZMyShaPguW\\\\\\\/Ke5Rr2uh\\\\\\\/CgKP5ZP4pfXc4vq9CKYOupyeU1IRVytfMC66U1cVgx+0Y3NWvfu0JpeG0nOJd+k8r63nG5SOCzVidgRb79\\\\\\\/HkUTfyV9Bh7LllxdGFVR0i8v61z0DNkTvEJHeNnWM3idskTyR8oONFEX6vd+XNmo3qhFHT7kBCC9fxycRG8\\\\\\\/gpKrYVXVr3DUE4w8DRGwrnxydsCVTXjee4ZlZ4mk\\\\\\\/Fu27m7oEobVntqtfUnu6BdAL9KdpESV4XJj7PVe1gbT\\\\\\\/jpsQlBGaJQHL5XNHoXcwYs4RjhIvICLOdEILVChFrLTggLmmYdyAhbS2vqXk2BMwA8NMHfH8O2lvYO7pO6Uj6KLGWZASpNC3uYvDS2kE3OncRI7wmJ0gF99uFDTkdPWV9n2YmwF1XZNZ7gevOmCyWuZ5227J2ag+pgeoENkopXF4aTKFUiQ2uTy4fMPzoNWwvypDSYR\\\\\\\/Q8W1lpLF5XWRa6LGHF0p4CuqwqXcGrYdraB9NTtOZjk1lojwj5dNQ0kXxdpW2fL8LUBT+X8OHxVpUPGYhngmr8G1pf4VswbqqcSzZZmNNLSIvMdQtQiE\\\\\\\/62QzwHy+NYQO+3rJJ03S5ZKgz2CYbsDTFtadzCrEUXECLw46Hjicc80jOZG3cucVT8MdkCF\\\\\\\/nFOx1b\\\\\\\/\\\\\\\/podjg6bKpPpL8txvpbiZTDg9We+8SxkRXw2lHDPWu4fpHNNGoEqTLRPsa0t3yxF7gA5lC7T7lpH7jvEhgw3SzyQa38YklSDFZQnnCj8JdYF+6S8N1gLdz7MdDoB+eevWEqP\\\\\\\/3hha9SP+db3NdBxxMJD5TWQOEuGkshIsTT1kZblx5NdjFVDVR+5AaWxqCWgdmRfLO40hMvWUz2kGIJokLtlREWfwWpO\\\\\\\/X3opVxIZPkRKvvGiGXM\\\\\\\/EAgW4QjKawqxD7xwa\\\\\\\/Abr9Lq5lJqnZwUDYeWXUgG42rWEZfdwaQLaM6SMLpK5\\\\\\\/vS5c1qQBuXHXQSZ7yKZB\\\\\\\/2\\\\\\\/ov1m70nKezKIrmz25XDrQ1amJr49Bm55BmykE6ahIg6YVsjnYFPQsFmX\\\\\\\/GfEI0HSAudFjQmdSsuUMn5I1WEUSoLII\\\\\\\/aSgJEYPpB9Oqpaytk8LdQSAPVd11tXODjxuyU\\\\\\\/ZQXd2mGinkU0Eo4J\\\\\\\/VGX210M2N\\\\\\\/J1U2x4wh4A74ScZEGPNdMYbvf40WYpHR83gyP98CZApgfKvDa9YnNs+DBBHzxPfQWT6LX7eqPeReDb+W84O7fkOPg6qpRvYHeFvbTvljmB57SOu246IeGsYYexRhRZbBU7kjDcGJZprrBOatLMS4BZw1gn6GB\\\\\\\/Wr+yA29xseaEEjE1d\\\\\\\/QgIF6cuMxMxk604rsge\\\\\\\/H+92rv7fNjxxzZL1Soh5OG9dCKTpkqogeyp44eDWxOUdpEsSaFhy24GDdpG+ZsdpVuQYEGqhfUDiGDVDZX3K3PTVNkWWQ+lU6e+TYdQG9+Hx8Hnk9U7JEtS4tNKXedRcwQgr9DwMBYD7lX3pdtp8B+wkeYcbElqAeDBceAmbQqhm7faltf8iKevXQe\\\\\\\/8EXX\\\\\\\/3ScaJltDQdvRF8s9gp54LhAUYf5phZmUztWyFW9rW1DvMdNWUAY5+koxFJFzRxL0Cz1T+Kb7BGxOd7xTjcJpZ1pJLshxBdBEwkRnYLDaxR5rj3CN\\\\\\\/uY9YV\\\\\\\/AXJkuPkrr+lseoqPjQ0\\\\\\\/R6yzIEqD8wEKBdmX1phyOLxza0+SfrogEbBEIUUDhyvWzP6p0w8Au1ZsTC4qmLfEUJRuXsuDAyI8LoNsXf9sykk5HqoYz\\\\\\\/wLwZhkbtHnEuGacrNtDMbz8P9g+5gOJFVVEjkn+r\\\\\\\/3M8uzrNgqjwa5ITRdS2bdk63FCFpmoHZ8fQXoJl50SpV5Jd4ijMQIcvm8f2KpX8pc\\\\\\\/IsFJ5HmpHPKEUK8myhY9GNFYoKuh2cvc85EpMyq3Yz4GKtH3EYoT58IpxdTX3j2U7a86jNSvzYFeb+fBtvJNAvNN\\\\\\\/7MwzIW8xFR3LFszf8zx1Dz8tHIFpQegkgjN4XK78P3d1yiEjzLP5p8dc1E+O1QVpe8HZyq3N4MHZgwJAYoWYMCyZNl\\\\\\\/BXpGv0O6B2tWOPFUznZGrXFWKrbkZFXgw7omvIFspYtEf79tye0REjT0RfpK0dAYEtb2FdDL5MtUuBzPoDt0T0fGgaDn0D1QwrCEhzTyDnfhXZj1LLj\\\\\\\/X7SU9BdYp0Dbrws3DyUrEqqfoAWc9x2Werd5hk6nMd5p9D6vJu5rl3yCJncvrENgEkFEh6xOpE3sodD\\\\\\\/eatThGN4Ut7Cp8TQFeB2\\\\\\\/Exh1ZES+xcuNF1aeKaCbFH+e9q3IVsfeLpRAiQikk\\\\\\\/aAZxTVNOn5cyUziYTtla0wU\\\\\\\/vfUbY8z36ZSwjLy4w2Y0e2rVdC6zo\\\\\\\/Ggtci0R6c9MqKGgy4KFDJ\\\\\\\/fErEf40sl5xpcT6Gv7m99vwVRwxvuh735CcApJXQxwg9xKZ8nRKMxlNa\\\\\\\/kAeNK8\\\\\\\/YYNPLTkw0yc+9\\\\\\\/JmeJbCkUYaRNSMuTDvzajP1KhuUCFmS0HITsxuxeAy3NDUI1dZQ2PEpZCM6kZNyr9y9zWSEqIkKXvpr3DhZ+xae+1lLrvmWJkdCWTClZEDhkM2qp06m67+Mdkift00625SSsce0Sxzphb58HT7sh2u8feP+\\\\\\\/B9B0JsPwEDRfW+bVOJDeKqeq8jLkV+OYeuHtsFar5I\\\\\\\/LUemiMGEYss+G59qmyXqpwSmDt\\\\\\\/CFn5xmJfMYpUyTuPAPsttWN\\\\\\\/xDfW4G9quoleALesn4nwOixL+xOWt67Rjt0L4xk+gnQVB23vKio+i9jMfCedCBlnVhz5JSvntO6R+JnDDm3TT+H6ZqdoJSbInzFjqZduN\\\\\\\/kpF3gkli5r2ZdNkkhM0jiyBgTLVa9SCYAlRzFEuXQDH\\\\\\\/IS10WQpKl4\\\\\\\/R5vSlP6AApZTRTdXQjZE2WebmnvZJigq4VgbWShjO3VbsF9X9c3L5ZAziYgkhYDjJVQ9ASolFLciACSjG1PrOsXgF+8q3ri3BazxAT+pYxq68rMxxAu3GntOLLlVRO0u0\\\\\\\/DxRbGYUlzQoFT33Et5iBrDUcAn2K1gMDshYW0kF1RSvaaRUmVOrtfRyGMgejL9YDI4aaJO8kcv5lEPCbBRuPZcqHembt3I7N4wVS7x2zSzKnKBBUDhbvhg71mUuWqEpR6i2G1SANsuD5jvYO13ZS4EIzZ7a3N5C23r7VsiVI3RY9LPzcgqWOF0xNBioOSslObdLT1Hu3LHMdZ\\\\\\\/0dkranSq3vLCpv\\\\\\\/9zF\\\\\\\/qgiI0qwWX8rwz6bUIN+27ZTBxDW1AMT3Qd9ytQNHlotDlnhP1gCDBlv+1TvP\\\\\\\/kSOQ0e2jWwUBu\\\\\\\/dJiOxEFtjGB5Un2PtFpGtKqrFLo5eL29XnOGu\\\\\\\/oXN8QvyBrAix2iMJX4LVZDQULtIpPqHlQ9B0P0SsU8ZZ5V01dqmQMTUplokrJOAusG2ObIILS6LaDLxarAsZ8QbNkxR9S7vZQBTnzucQpsBV38OBfKkWQfqk2XV4upL59OQnjCFePpHK3BHhZcLJcYUt7BoR2PNEhC5GpNj5fNwkF8VzRLdMEFUrPA9rBJzJMp5WWaO7Lx+fLmW7YpFZsYOUwnakRvV1m4UbwHJorevbv7Sdga\\\\\\\/LZvxAxtBp77cWX3AXfz3kX214a6PfkkARovnLsFTm94JJH38wjLwI6UgXjAjf\\\\\\\/PVDZfIc\\\\\\\/dwTG0uIFNRoKnqgFYw7JymSymlNXJRq\\\\\\\/WdKh452Zcx2EFHpqMmsA7TWsMMwg8SHbwVWZXkoPX7GPrBalTWsiq\\\\\\\/LJyvAsbLDblMctxvfCT0HIldKqSE8bSPxFayblmQvkV5aMrbKgEezWsYzNafDlNPkFdAjc5uNfZFuDf71Twh16gOqtRd16GBSi8GYTE7MhYpr2GkHC5LeKZnfwAaNxa4F7U7sT7cuGcIADt+OqDCHdQbypeZNtQPdiXVBmcAYnLB8Ws+ZHe2fwO0kH0SEpKGmRg+QrMr5q0Ep7AQXBAYstTQu4zCzxeNbO6bgWAxG0N67q3L0xcueHoqLWPVjZoS9AgLpcOBC+h7Dm6uzk1JB2wYI9FFQY8DMlf6RJbyi0F8JR5CAZbobNY05etw7osyL9imHP6IrNz4zCNx\\\\\\\/grqH5s2hvgi24S6iPDjzzFz62KF0us8lQpFMEkigzbl5\\\\\\\/cxm1NiQXGw3D1GRoO5BZJbSe50i8ND2K+Z7e5PorYVTWV0QaG7fwJXVhRt4fR1gtucCilRQMBz+J9ebOPRhHaS4FWnwmfI94ZuGkF6p\\\\\\\/ytDRQ3gYphaZp9HBIsjq75qbydU5fBwSFTh90HexVzoNmDs6ccD5C8AVbrhk\\\\\\\/caFO1KgLl1TEK\\\\\\\/Sqw+hazAi0BVAskQhzPGFwPgRrIMOLpQqegoaF4isEsF1TfIYIQn\\\\\\\/d2V\\\\\\\/f10tisY\\\\\\\/mCeRGavadeAyS7LCgMs+yzWX4U1PhwitmWH2QxDdkW3qE8MCTLSYJHt03GLnqcP2N8bSHEdJy3T43ghPpXZ6YBhy\\\\\\\/6aaw6XKVcPoqSpTddFdfB2Hfpqdz7i7WogK\\\\\\\/MfVXONYWx8kYb2HW3yN2HNtGTrRVts\\\\\\\/6pZ1+tvEN2Nu7YXBByvTjB7L75cvqYbrUmYLRR45Ig5ZDRBObnsSivfsnc4ZHvXK3UWpd3nZZNPwcEbpva9T+AdI3mZEBVtvTCPvJ38yhXrVor3YyURK98D\\\\\\\/18mkSx70c4izJihJZmHfUA8JthEluBOMmKjb1jG9XCvMopjFZLEKRPiNB1pDertZPAwrfZ25q0mqR9g+I5T8sULQOR6F\\\\\\\/8tTo9zSf8oURBI7h27T7WSEM72wag982e\\\\\\\/6x8yq7G5fD5a7W4USw5g9koGWgeoebRsHgbs11+x\\\\\\\/w6AcoVUnjou1ewieaLJZsGU4HqM00nJk36C2j9D5+AmFRfzE+rQ3oafHvXNoCFlnfj9q38OJKz53meZLW\\\\\\\/vhM5Nhk7GlXrcbE0hS8TG2Ht9vsfB1avluzV9AQ2eFB+biZqMg7ltY6xoDloX645QdmvVm6YExKADGBJdOpCoVbGFn3Ka12xM7B6KzKp3orcttd4YOAeM5xUKaRXRhB3wMVV58QdpH+QMQf3c05tgFBdUpaj+4WSeW\\\\\\\/9dvoW\\\\\\\/wFnIeeiQ8g+wm+CYoj+XwRTtulugFMvHQ9MQJ05jw8ui2+gwRDMZWt0yHitaIKww5wbcBxZSRFIPcfJLbYJmN5ehXXEP+kl\\\\\\\/yNPANj7EBx7XVFMYpWS8KmvpD4UHq89EI5c\\\\\\\/TmEM\\\\\\\/cmQw0nvu+dohglwhQYOKhCzQUeolD0Zn4iYowt9\\\\\\\/1cE50+vYPB2rfQUkSxRuFzd3JpiX\\\\\\\/UmtCiK4Tjxby2qBmLAjZevS7mK8X6hl+8V8zHC07aahw3N6Vc5MEc0o6q0xpO5sLLd8NhK5Ag\\\\\\\/WlQw1RlPEsiRntQbBXvc3jmDI3gruSzbY37PIjXGxRTTrrWiLbqYkWCA43oh4bJQoTPBkYlWfaFG+t0wMUSB85j41EK2wMSf43O2MBmVDG5+kHu9RUT+POmz\\\\\\\/3T3QShYnCLnHFpplS\\\\\\\/fUdD+Ib1viFdNSKJLq8yYHH6xp5\\\\\\\/CH8uaYA83KE\\\\\\\/5BnpsKJaJFJcq4siapP8j6f7HQ2\\\\\\\/0aU9WG88s5kmnfFVprgdtBkHNApUUnwMnXGYa8J4uuo0ibDwQsMT6YXtTIRxFbIzckHjBItEIPZbQizyN947iEB5qpLob5H0PM6b94G9+SWisuELCCYLIOlEMuflEjMT4rSa\\\\\\\/5j6ki0XVjio\\\\\\\/PIg5YLd40EFGz+s8yRzCEE6gL7ZwRRuQCwkJi+MRK6SMc7g1n2dqRAmB7nfBi0Z5xcUmIga7tO41ttH6EFg9M9CGZIZpwwq1aHRu6wLOQM\\\\\\\/DPIacas\\\\\\\/XrZn12YAUDsx4WQGw\\\\\\\/hlAa2a68eyr8jKCuJcE2rQUfk9fKv2vcbfVWCCdP4mnCV5vSvlq\\\\\\\/w9f8a3598QSJyZTFNbLbEuyjW3bAJzjWd2IG2n4hm\\\\\\\/bNZr5uhhFH\\\\\\\/ifLni1Ki9F+1IK+mQ8Y4Gd943Z\\\\\\\/6nyBTLTyMTH3LbZfpuC5u08jh69S\\\\\\\/MxfkZBZEIulhatfWTKjP7+atAJp7wgv6CI4U5Y5L2G\\\\\\\/5jx0HdingjqDW7XrRjb4WeDvbVdOu0wQgyw9nfoykUkwrAaIDWW8\\\\\\\/Ms3xzBLRwS1tCwSRkYjnn9iq0LeSpkr5xXSnXyL9h0FkCA6G72c+7m1SyasGFcEbPPLZ0Jlr0aS7F6NdRTFjtuaXm47drx+dmMv3JaSOW+Vi2BHzycmhS9la7YKkYOy0BMezuCxpMN0bqcPHSZ1nVhg1pdGQcA669LmXMBNCi+wZBJyki3zZQ2mR+lBLmHqKn84pblkmfq78Wy338xDDrGtgoIvQAXqRAQpLw8C3ijF99iS3fGaxQDBr0ogOpOW6x2TUkyTnocXenySRuzq+9kTuNZAVt41rA8Gqfd01pQ9ChlaSsUlHzETwPgCjR2e6Bn5CcBNHNwvoUgbtqJs\\\\\\\/CNXUOseTaf1XBAqwlqkkeuSbo6lj\\\\\\\/IKh1ggirSI4DHD5LOhffYOTZhM95LIXsyhfkcNieRLrNjllwtRHxrDmGoTNRonLYTrr+7vBcMdc+6HwPCApZucpMD9La7gz1heHqavVQ9PbhzDMbWzch1yA9jrm3vjRtNJ8G4zWgaLzabPgYJ54SDG2P02gI1YeSl18M5\\\\\\\/ZqEiJ6fbTpT7+ezspxwLhXHUJMceDR6wzRayN37BEg+W28JReKsunTrqmSqOPmtGassIZ6CMUJAOGAO7vEC5RwTo34yA+d292fALIF2blRLpVpImORmBrnSZaQFIyDfrTPoCU44yjNGAwhTYZT9\\\\\\\/t6Q4nCUkYzvjW0T0XIG68on9sKsMnrqDVgJUBO9vWojBrVxrZhGE3cwue9llCdN6St9pBQLpHwRiGnHCZR6KEXEvpPnfXWu4PaUqDR8IU1\\\\\\\/CZg5dkFIFD4vpC24ngJcjENDGX6VEgH5i+NFmvhR8+49nnN\\\\\\\/CK+veWyWgGS9SwSBDYLIdoulJJLHGZDHoe7YiCKA6K8vcDJhA+EWBhdxVk5K\\\\\\\/GqaySKopLYB5qvxbpSv0+Vk3ZYLFfeYyB8x3dlWW7XfzXJI3eb0pMbeZoGbSiADc+posACj+lVrSqJUB15IscAScda1QCVpbMlMPlMB6UcDXoonNk2Zc+JrtqlAiOJl+FghioGuJMYm16EYPTpFVyqfaCZNIASgZJHpppKEy4+sHjI3Fl673uzhNLJ7DIOKle2nOvwWo4IFqecPD4owbUJdMWnyb8kuy82d9+GeqqdkFaW9WN9ytzFIPvp16bkkCx+xLrKS\\\\\\\/VDwlAx2wkEE0H61rUPVXMbYVFd99TtQ3fh10dWmivtlxZsBktPrgDr9PxmBPXMi32udpb4Z\\\\\\\/GXceae8qq5A9ndRw3VlPdknszdi45jzoYRzyi8niwjM\\\\\\\/ksdC8\\\\\\\/E+YAkOj79LgD4kVx6xaGIH4S4rmXXnBh54cNwgZ+eXXfMRwnHvzhrreTj1n\\\\\\\/YBv5ko8t1mgEqSqveYVG65JA11SsR6A18IYRniDUBIzLXnNV8B+UAPp8LSXiMr733NpTU+DhHcnXGCW5Y0nEnUTdrhtnvJccOFFmSnMejg3zhaCcVzkRzwwbSv4YjfhE2xlaYMYpk8pVkW7UXwH+saIsFWpzMl67Awl4ZUl+0zESVwdjiRd4Ouy3MufFLCoUDX4oePk9jOlA1YCRA1UPuIs9Bzrje2IzLtWmRrhwbtYgydA9NleTRJI6LnN4CkYCR8bqw+\\\\\\\/JXJ2GXs85SNQlmvdvGE2cc6qt1r1mddryNgolj2MhbiIaEFeFwCeNtSBFxwtazcW4TqenPTpCyhRCoCaZS8orDOe0okSsQMCsyMF3J91Kcf+dOc981\\\\\\\/CD0YExy02U2kkgGpIFlIeFxCGB1j3Su2I+urImTNDA\\\\\\\/KsrpuZxmJ\\\\\\\/XY8W3d2\\\\\\\/fdrCD5UXJ3nv48jGPvzQf7ITNWlOXmyy61H9PvdR3wVfXlrRMn7L5qAPkgM8WthYlTJ9UqYKvaWjdKOWWhTIMwE6r6uO01WDIUnVcyLOXBH8+QPIyIpE0XzanVKJZxr60AHB8tHXC6RyTxOyzuqpYKsI\\\\\\\/SOmJrBxj8vtibhiAvJpTsdrss4\\\\\\\/SGq6biVyyVYi0HkCP8HefPInpm5VIxUPOOiwBMsEToSyd3xxahCT0SiTxYDtUld3elVGPK\\\\\\\/be3bX9CppR0D7WdcquNpBCrZwjY2BGATrUDuGw0nRPEQXaaqLMqFHQNIcY17F6BThNqdUwdKni0rqx7Hu18U4YH9ZBmO+PRzcDeTchks3dQHPnyt\\\\\\\/GJKaEM2ScMmDHS+w9HFUzU0gsDAGdWrn+zaisQE9OT+B3NJtnmX4HV4Y6OJKnYcRQWgmr9JWfulrWhT9cuWQ9xGJA9eCJ7YdYnQsgg0udDGe1qhVEsPE9dqYZYXSDMVLgNmYiz7FsHU5mTnttUftEI13wVzpbOXTR70Py20VpFeV9QzwUIWLmn0aKMIV71iWB4BpmT0p\\\\\\\/n0SaXeTiCKj7ie65An9wZBnMdD9pXdPNTEOdoQPb7MKVde3V4cNu8xjA7BQZOzYQdb46CO+lHTneUMYkCD7ZkKj\\\\\\\/SpzZCsgEDjCNYDi4\\\\\\\/ZocuHg62L6fN2VEAKNmD9isyIDFzQvaAMqUA4XSwvzlDOHy2p+JSRgtm6q+ovuy0bExxXQF8J+ZqH9XJu4DxQQkN1d2E\\\\\\\/My+K7xPmPHAn5WA5Zp9XcIlHge6x1ZLFBluaAXtradhCqReytYmliIZDq8q6x1aT4UqA+i\\\\\\\/6hDZFVGRH\\\\\\\/ky5HDe4lkZyr6+A68Sjdhm6eMv1zJJxx8u5P3jUHfirKhqpvHcb\\\\\\\/sNX1mcIw+mr8fVEkzMeWlGs4cetGbQPvgVO8CSLbknJhifigb6es8oAraev8gJkTToIkuCHjIRooD11FPeMOkC5SjgBAlG+YfNgm4afI1rQq72bFxPHitvpsLQNHWK1cvwtpjcK9dW\\\\\\\/61d7RnIpcL6J2cSdjEUWfOS0S7P5UClR3IXtZJckdY8boCYqYktgiTCyYSMvSm70MQ71nmUV2epFxQ9D7u5zXv08eUe7mk6gDe3Mp\\\\\\\/cM4+kv45BCGuT36fFGGimxKu+wayTYDLCYnBx\\\\\\\/cZbB9Cqu1kOpUI3B0RJ2OmZ9KEHpzbkz1h4G4pAGSqowyQoeXvZg2BELleR5hx9hG4V4TJkGouqFbbKloZFPT\\\\\\\/332DZmgcXDFIsq4SC4HEvha2W3Nka5l4KZabhANJpxiFTe5Vorrwy5qf32fbrwl3ev8x8vHJBAzxZSwaIHE5fPDHvnnSR9fX2SVKCiVlHKGH45yjlVhVkRPwCMGpxwpa9PBdh5CtcmA2wZz90M8Ct05xrwrViVjH+Qh\\\\\\\/VfXtj4+Iv7bmCXXfxQNh7J9jU+pDmHWZrIH4OZ6uOYBYFAAiZzpbXzbEajhRv06MYteSc6fGQN9hkq3wE\\\\\\\/ZyryXpxfuBS1WUzBp6KjeRpTx3BpU4+sRYOSChqbOxHy+0HNP23ZAS7kwxvApv\\\\\\\/W9\\\\\\\/i0Tc2BOl8QgNWHXeBxiKp9e3NIViMajAnV9yOytYrszZP8sbRT1Dy5hEwy5y4kUK8oXM\\\\\\\/Bt7\\\\\\\/LfMwiOWRR3L2s95hF1XYkO1MEb7O3EKmUFLJA160LElO8MYMldDbdNWCkGbrvFrgpG3EmTGMDLiMjhcbo2JARurjq+TFGJrTCQFHcvn2IVcsJRfzw9wwJnSHEajmfBMVS2K1htluWYXxlBeawyp7IBNZMqgofFbrnZEGyURPiIarkLdMDCzLdiXsYjptdcCq3bXaQcU7QOqtQDP1CdYcocS\\\\\\\/Whwbu19r\\\\\\\/2VIUSrwuILtBKiAFf9+s3kEWHqvFUQ7k\\\\\\\/x3ow3zYhLjinjm7taTq\\\\\\\/0TSI4JclWf73KdfEHIrMJUle54043zLcoYl1G71UYEMXfO6WCpMIT4GIculoiZezMvwq6r0Uy3QedOMtq3qMLfHbns1MStcrMms6YpcFk4xnTgIaQ7lL7588bTKIs4CMIt5N+GD4g4yy1zPL8g6Vzbim+BAark7TugUQCL8LkPJZIDD4vkB\\\\\\\/2fnrTBAj8Di9lsS0r6YUOg3bcvmzVmCnpZlJQACCRe55qDr\\\\\\\/M7KC\\\\\\\/aCihPcEYH4j93WvOtvehi2aYK1v2GwbF1Rjz+dWRK4FycDq+e9JkWkduuhO5MvViL92VXmHu0Rdu7IZPpXx4EhKJPFxFgc0EPXjMMoEU\\\\\\\/zGVj7yCixwEmxiPDn4zqmiKHJlfhX2m4DEhXRuepjJWj0ObUujwrXobpIIipBT8Zh6GuS\\\\\\\/WC4dk3kFNU4791IbVhInrWiP8JvYURsIV+ZVcK8FvaDdMnEZUhfF5B1erHcnHi9u5d26YW1ywzeAEYxdPqgtYq8dx6ygxR9wk6DvqCMirNNWLN+sqaiu9KRXTe52C\\\\\\\/PcBoztVDsGhq10xNKSBqzbN2mYpp9NOGCZMSxgxYoRp4L+hoTsUieCNTvPH8+ArGYgkrOybHNVUDHwja9JDJQ03jz1gbuElqfjnVA50CXCN8PJnogYcwrDIbZRI6qjb88kbg5ucxeGViBsGLdoXbcv5fEIbkrr7+fmm9NUBABYWZsWQfIMfs2hOnJE4joAB8IzzNsYsF3Fe88UtWRJBSHYmHJsiVfvYItIYvyQkMJ5fbmxdX1++dIjAdC7LFm1LTUtZcg7dQP\\\\\\\/pEn\\\\\\\/ywxfOqLt5qMOsK0eiwaNJKAzEiksMnNG0qsEEKa1adUQ8zoX6L0KyW5UkGHa6lQkDcjYvtUBMMKJ3ILNN\\\\\\\/6wJx7iJBro9PFfEISIzJr0V8zqz2fZ1IenXRySSaqIyQ7j48izcpW4CGt4L00UJL2vNCzLaow1yxjOh6A6AvxP3gnVNnRjSk\\\\\\\/gb03J6zVa3u4WKhhfd97E0P2iUF2al4tXPTgqkXGFUoHzguTfrD+yfK9fPJ6BPTuzuO\\\\\\\/CcdAEmyoNoaRPH9Pn1+URb1eedoyZ+MT6Aj\\\\\\\/IRiV0vB9HB\\\\\\\/DLrRQ1ZYJoTyGQYeSuJoAjk7leaz1z1JJE\\\\\\\/5uWV6MhzMfQYcDNu\\\\\\\/ioi9HbQNnZKqFsjV\\\\\\\/MyJjZ1umPGiWQGPzGCZSzcC\\\\\\\/p8d+oZ3iv0y4nE7+cSAi7jxi6KWCGkKeVKCk9fTKW6HZn3x\\\\\\\/3XgESrhL7DPgUptziM4w8clnYrdKX8eGLL71QBY\\\\\\\/3cqPcWvCTFhTBjrG3uxCHLUvGTr1j7Nu2YN8ZKn\\\\\\\/a5zqsdHPud+L\\\\\\\/amPvEGpNDPCuMG6rgp+z5guPUoYOyEHi5c28r7wUZSVbULUk97oEIzrhTVS4z2BD81d5PdCVMHZF1VKj68KbB+de4WDmXcUqb1ZaBiwWMAlWLrt8erG+D5WIC5dL3yacnYjGV\\\\\\\/L1kDREftRvMllV63s9PXltxckj3vOynbxsgTtfzwHjHBC+BMP2pPtZLR9U3Ssdk8i0+lfbjLLehr7\\\\\\\/8Xwsw5cbjofVMvDO06l6RQv0YK03YFjrCfvDKpATyBo9gLZngd+IT5F3BL8v7kTnXY8bCkVYyo63WPPZ2FhXScHCvZc+KAO3U5bfpBiWPTiB\\\\\\\/\\\\\\\/Mi9pBVNEnOriJem+pNxOIf1sOIKEkuzGzpRCaCzNyGoKgAk12307Ys7yqG5FBZZbSKLOc+4VYMiwqd4\\\\\\\/38buyraWpwmpGUsZMp6kuBZrUOVT8B7tRxCtLB5avyot3F+v9MmN3koBY8CUaFSeJeTBAI5gL+60NrEIrafrQFltlapuk\\\\\\\/3d4+WsSVIby\\\\\\\/smzZx5n6GgXKZwOJsKmC567rDyqvafttOTl1GS8AK6FO7eGzuz65fFBf6qy78ud8eltaQd\\\\\\\/L2WU0woG+6GFwX3s4BboX1d5hba0rQtWA7EqxdXAPMbP2e2a6TtJdaXrXE6lU46DeyGVs7YK2T\\\\\\\/g53Zot9kBQsx2PA\\\\\\\/9Wt84qD9h2hTr1Q9TDCuUpu6XVjQHA0Sp20PvRdnjv28yNOe+FM7ooBI1z+EZIpnq6i8bhnVraaaB5IHwP8M00Ml9S8Yi3rwIMTeHr6xjsfFr9AJmgz4EqwOH\\\\\\\/CUtnZ8orikBXqFVtuLPQgpOYlnTL8kfuhZFUFzpNQLPnx8eqNNm\\\\\\\/MhCenX2d\\\\\\\/GI03G4SDceCghpGgwJ7G3SH\\\\\\\/aD80ceMBZ23Mhs\\\\\\\/V1Ifo3rPSK1Yq3AWougYMI7gDkDvuLk6Mx+ArLQ8aLFPefWOhG18E6bJkYgTeNOsMhJ4rJHvhXtK\\\\\\\/jzIkItbiESxha\\\\\\\/uJwq4h38vqIG8rgg8botXl7XZh8cHJSzl+DIf6YP1LH5L4LzjnRGAlZbYLvlsLwylKyOIyGR5B\\\\\\\/CdoOkgdhbdwBxnRyIy5TjAFHFKpRQONVUJe+mc1NGqgCtrlnU7YtDD9z4A9orBn1vA0DFrv9Vza44ycNHERzyES9Z3f6AiP5HbPyDc+raCclafLDhEfMiqWCckz\\\\\\\/seFutBWNh28SnpLerm2ypA2Lelx5tqzSyUZa5gOD0LjAozvny5hsQuEgt0MZmhNWm5psdCdNYOIV0QqjHKVTZkOVZvxHrzEJe7B+a3BmuLELjW3\\\\\\\/0f8ITvjmfl3NkgN4s9O+C9JWDPMBxkWah2jo6v\\\\\\\/9RdFhBoWFnjQFTGpfvAd1dve7hdJKLwDDBaZJ\\\\\\\/ppv4bTcxw\\\\\\\/mXIivH\\\\\\\/YrOcNPiC84Iro9FWju8aWwVYf7BoCN+Oa10h+FXa5rJqVtf06ViEXa+kHYfmmOcnREJs3jHkEAPNsXgS4msE2QI\\\\\\\/GVxFIcQ8xLCI2Hi27pgirRuNQuU6mIa4+5EXww7P3KuBWnsUhVcavlTyR6y5thCmZAnoXp\\\\\\\/bJYAUBmk2uHaUtqf7KwBpxN2efw5sLr2b3dgzzYR5s2ReaE+Lj11rrGpoC0rbj+ks\\\\\\\/lwo2AsSfL\\\\\\\/jo9x5SaWBUM6k8UPgMzVWJyJBoRnendu64Q0Q5B3m\\\\\\\/wmNnJB79wBonHFL3ZA0+Ii5bqFIvwH1RivpbyzNkX11bxd4\\\\\\\/PdI4AAQd+0pqo9aLzWpDKcjLceompo97sRBY1Ej+Aq6P0DppkwIp71W47TpmHDEQQXQ50JdHergL8F7Mykvnu31tV3cFbXcVq9IFFnbqHjkflpULeIAl951oZ2f0sIywFNxFXRYu36mGvS2tZ4tKj377qDioLJvEt8UDvqQ5tvLhyqqsbbCJhD69yvv5HJWCwJAX0k27IKIw8M+iFpAaPp7aZaUUS738jGLAU\\\\\\\/bJOQUID\\\\\\\/JEp4ilYhU2DxcTVPMV0ySHr6usT8MFAljLvxTdzh+E1eqswH53SIRMmr9MhsN8BIMpUVjIi\\\\\\\/We\\\\\\\/Xrqa9XUx4gFi2bnJkeLCt9hhXqVaIuRVuimT+VbNTca42cMSf6M7Q3D1AvrauOoppXcUYYkd3P+u2BJjj9HVXmlJp3AtKsPCnV4FIemjgZY613e\\\\\\\/NnZOg0rSVtCKNyDfOD3DmedRbjZEnsay3+GkE2xtA0hPKYtLy4cQ\\\\\\\/xhDrdFXFFjhrL7nLlNQYBIU9EwX3q7pd6HfxuHPhK1PneQOHH5uG8wQYPIz5p6jGt21am541PWNEE\\\\\\\/b09LZyzi\\\\\\\/l3yoSSmuO2djF+HhD16LWlpOk\\\\\\\/LF9ipCWUhggPwRzpaxipfMQ70V9s0cH9iMFgN87B42BjNo\\\\\\\/k6F7O\\\\\\\/u3lLOxM0AFCPfksZVOs2bkVa3c3B4f6LoDhqbSr4FsoJtrClju3QThou817+O6cWz6DhAoVD1EDk\\\\\\\/uxf4NS2I3qxmny4UALrbLt8ZPxZ8ceN+pTJ4yywrxwlb6T11qJphGOo8LqEoc4Cko2xHuWJT4FlOFC3TUOO3BfZDbrcBRrcaosUel4Rcbx8xRd7EhLOCD8cnNZKkKr0nYfN\\\\\\\/h6oPH+X2mta1g38UooPgvN0+67ZkUOzp1yTAIC3Mh1BQO7R\\\\\\\/R6zK9xhEKViWDhB1XFZ2ECPBXumzCv+IeanesVov1IiF4DzUbOxm0lCP9c57hEWxB1KV48aGqpQOJVNAMXksEU3En9q95oUcKl1qPl2KxUu\\\\\\\/QaChdDKRE7M5tU6ri4Caq2huBKcudJnEs4ub0NN7UHAmf+IM5rYfS5Yp08hQoYks7hsB\\\\\\\/CRjmVnrPgc7FEeSumWKxJEwtk5prEUI4akZOEiVOi9v7ak8TBGrMQsNdFBQKyruiJgptw4vrj+\\\\\\\/qqPIBulaS+h+egPBKnTBm9om0v+Hcy9HBnqtB\\\\\\\/oufdcuR4OR1xzdqxHKGS2\\\\\\\/rwRnWzsNF7GUXB+SJ6FWapE++uAyvOTqXt+Jcso6kx8aoJkRza1BrszaRx2a1ivCWtf\\\\\\\/zdjZXh6EHKE+BYebaWUCW04vIIxJKUooGX9KpdBahR3UJ5HX2TIw38T2L+popDFGof6Tsjs99q8qlB7NKZjm2ZgKr0pyHJN4GOR2wzS\\\\\\\/UcUwqGbQ\\\\\\\/p85ZQQvdmOWhvX5HtVXXMNnpIj7B0zhMC67QduE8g4ICe5VyalTrW9Ra17Z9JU+q1RnM7CbkgRzJe3d0pEeIcobbXZWyiGRpXT9nGOC83hBRuNlWjSVT58fwN4eqPkH7eUdQEAuzkci8nEMN+nNGwR96zEEcrmsjC2NXxKI9yNyMZNZgNMmnRxkCxGgklxh9m\\\\\\\/PGQdBdUMJWSMoMZ85QlBobSckXMPt0X7cKFPoA3lO5rIaJPIlSTb3aRGfLBdf8W\\\\\\\/Bfrk2etRRrVKwlGLnN+C0R\\\\\\\/klyWrsJncW3\\\\\\\/+w5pPMtOd1OdB8Wph5uwJr51W1m48L6SVSRqVRZ99GW\\\\\\\/5\\\\\\\/+Bh+DezgfkP4HL4vaR\\\\\\\/X1PWdgRIQ5SEuDNEglCGkWtHGNUrh534qCu4Uc56xV9+CNe1VHEtTSxk2Bra6UICfWdlVI8w9h5JsEoaCmpEtdNso6zIl7sd1o4MxwYqsZpuqHbe75SdTHFV8WHziQ3O8V\\\\\\\/AWJYqwbePKePMNtgtnulvSnRFRCk0YDKBjtL92UFRsdLvZb4qNCzWLoanI9gYaurHDS494Rm4VVLm26NLKmif\\\\\\\/9Je6gSlD4+o9KrOPiyaGg1vos8XfZgkef\\\\\\\/YWxNQH5ID\\\\\\\/0Ma6JZ9A18aMy3MrzrfNIUTF2uItHgWsc7Zf3gShkYPOY4NPpOy2tpEKwQVy4Dc8fRIJ2lPk8zDipJKLQkO0HabskmpJ+k8Ox1GZHZI0JgmYt4\\\\\\\/aZMHp0XOU3MNXYMTrBDThGzEsh\\\\\\\/oBgnTqDa4agiFqmEN7aE4zsLl+k7fGGD9k3Yi8dmEKslXuFvqJCUikpHSXw9PXY0X29ipzamb6K6ygmmyH3E3ynK8ekiWSKlNagC6gtSot2zeab97ewL4USSZbivdiEsSx66YYZ4kZHxq0ieeL3iRlScYi1+bCp6s3s8Knts2UtZcJMpSWvisWrtPmkABD7EmTwy28ck1bntZdf8tXby5eCf7VwHXbCTjXEDrb5uzlPy+rSyf9s5mkXtJLT5gZNhfpWe2Rj1G1zvEs7PrPvRkSB\\\\\\\/Ufg5UUEOGl3xU0Uh3odxDrE0LY\\\\\\\/+PYpMY8pQMnup57\\\\\\\/EXWLFoxHymXrBXFlFUOGltrqvUiDv04BuiTtIa2c5fKSOTfFAsG8rBS1q1wCNFPqJ1J86J\\\\\\\/khTjmdJDlBZwfEyJMn\\\\\\\/NHS58voK4rE277dEx1OSec\\\\\\\/lUQKnNPa3WpfMzP2OQh5\\\\\\\/l7hoxqWCNXDsci4Kji+jbj9CCWUfvk8m\\\\\\\/eXFcyhQM\\\\\\\/\\\\\\\/F0TAUmBXW19BdAQW9\\\\\\\/kggrlQP3U\\\\\\\/nOlrluLYe6Ez2xAjw1fZWwugv9mzWM+qrx0WXcWu7Dl37WwTqCkTnX4xYuSh6HR8avWALJWhGGX3YOTkdGZjvsc9G1\\\\\\\/MitPYpFJvx4c0L55c\\\\\\\/U+DzrU20ZuU8PdVfHNMPqeglWYwIQgBhBLE+esOxEn5ELvCprBEdy\\\\\\\/lMHU2axqX0ptCk2WNSQSX0GzP7DkBXNSqfFMogf58NxqcFX34XzhF1R\\\\\\\/78B9XG+Zj6R1BQjeB4HJzbZ\\\\\\\/NNksk6N3dOB5rLzPJe9GUqCQaMDgdiXE2rHhNIsI\\\\\\\/hE4VyCD\\\\\\\/XVHnC+WJ1R6poSqK8Ds+m9KOkdTCdfcLoytGAVYT2d1bpyQlynUy9XuJAvnzdgyeughpMRJB4D1m939gPUTahhAfGyKGnEuF+aceJC2d9rqAlL3ED4tb93\\\\\\\/wNG6OvV848Wq7egkvE73PgWKKSEoIvt+jdHVCfVO\\\\\\\/9EqSjrJ998xntW9jPJRJ+GvWfVW0mdEzp1W+clVKeb7UWEa992kyDSJkamljgmnBQYF76IXkxnQxqc56SMyZXBgjL3ti9tgxWDLpz3AZIdIdulAePtnUF+XxqfhViZaySWxRcorWlcaCkk1FGLC7TgjFjm\\\\\\\/wcJQY7JK2Hs68pHnIvA+do0sV17lGETuSWUXxrbNot7Fk1WLu9qqtvknfUUKMELj2S+dwzJbp4d\\\\\\\/pcC46wUcZMM\\\\\\\/G9qLRTJlX8r1e9LeL+o39MG3jGDCPndjCZro0trZEDOWSSj++8rPTIb8lg1tUfM2ixtI2t2JWcWqqrMWafzzW00tIJUFQsscb+cpQDTdMBMnYT8CXn7BezGRH7CLLJkLFx10CKYLh0sjFqrWMkV8IP6K105UVh0o76cYnRRz3lk6LqHvPNJqMgXfr8\\\\\\\/4PAiNcJ7X5J6f4IxJpUIfaBMV9y4zpjwJW6Rtqsgqh7azBHGKIrw8BLON8ppe+u5ZL+3N7VRRl\\\\\\\/fkwljd477C6idYaQpyBZrTrWfp0opFm98jR3a2zat076pEjPy5zv04lQckkj\\\\\\\/eteOSIoPr+04q5Zyvo1RzXKJKnPP+8WYDgaZRpXXuMHBdRnLnagIxclofMd8X4tyfCB8teIkSKp6xR2RSOSoFwtmGGSx\\\\\\\/YwvtIUa\\\\\\\/nvWlMSI0yWAT\\\\\\\/8VtyUJoZHsLSHF7bDGx9Dj4\\\\\\\/FhCIw+4zffuBfFUDfi1qL9w\\\\\\\/2ssm4yLnmMExhIvHxXuVQHHOgRJtuEXoS8H8WinKFEeNj0WxSTjruwchAbJVW81TXx0Oc5nTnbmgSD8EC+skG74qGEzLsAl85BT9kLHObxErdJJU\\\\\\\/PKA7aLXuiy+9CCXGySv1wumC0heJN+L9unUaixZ8fjaNa2Zuhr03Q40A0U+DNPLrswBBh7EfSllSRKQilKgz5W7bFq60inXTEvt6EYy9vjkgxEehGGR\\\\\\\/qXaV+98AuvtyZGMdDb204AwvsKIcQwCo8cW8l7wDh62PKdTe53h44eCSD0J0k0d50ADUQJd9DsCHoU63VTHxicZ3sDoqiJJ2uRuVsFfS4I+J3Bv8N5ZhVLdAdQ9h+Ne\\\\\\\/U6x5iZdk+XcOrwyu21\\\\\\\/bjW5WmaZ7D74mGnmam+EN6ZlKaN94X\\\\\\\/kdrupSq7TPT7b4UB68Lzr73yuIf2o5GQNeCYJlQrkNzzlDQ1rFWvzGuDbWStsZmY7KZbP4je05nGgFyQDcH2A2ZJc3pN0NazXScgH\\\\\\\/ZnZfKcFMDcWWTx3GaOj\\\\\\\/zXmJ9Se5wagziJEUtgJ0s3TA3XEncbkwbXJ\\\\\\\/+rg0nkU\\\\\\\/nPwbvPNKr6HMzW6EbvStV6EyvU4ACr8YrSxfvrLLcDhZGWWXmAeBWI45lliSJo1nrdo\\\\\\\/H9SEU4wR2YOZRmJ7XnUArPnWYOFtd\\\\\\\/LgkYfxcr+yg7k0rLKwXrnBA1X31M47g+zFJsPBoDgd84XP+ExpXd5JX79p4sofFB2hA+DTaL9NjVTZpMHdIO+3LyDUMEm5qWVAe8XjFVduKCr0sNeFAWn3l0++Izpv5pz3bCVA2OuAdzzInUWRXJ8gym\\\\\\\/tFKZB90En07ET2eZ0vXdNtW5ee4RU+pbzmaLlYFALk8f53vcRHyGmDJs3taxKQ58cpbv3ioothUWifELUHpvBC7\\\\\\\/pEWc3OA\\\\\\\/UJPFJZIuKJWNKbFOkwGa\\\\\\\/9EwvaVXQkK2BMeZHPEx6rZSaW5mkKQCRltzCvs1D70jC2uVVz9ta03CFMji7B1+\\\\\\\/y9le6+li3C62Tg\\\\\\\/hwPrvzvxKvftp5eG+8gh19ftaNOlyLqCgnzqI7iAlIojT+6ZLsNjGSeKmCRThzMJDkoHz49aKn+EBd7J4gHO7887fmJsJtdkEkwFxp1ajMMXN5PklFHbBh7l4pJe7K4LqevsmXmnqnS+V5NMR4DmXkydsr8\\\\\\\/MqzEllW2x4\\\\\\\/FHOiwrUT8G\\\\\\\/G8S6lo035rMLq43R\\\\\\\/E6CB7aLsK2aXWLFvzo5fnifVzIJD0z5zY1zHj3N+B9GnSvWJeOh+aNejr4\\\\\\\/+Ih2Oyypwvb58j3xVWMyz5asEh0vQhK7EP4b6SnGN5hRoU8BZGqrTnVlhxoo3riQ7VICCHBOItzFXYRBt1y7rF+u6GVqqUMkorLPTunOPahaQHHsdDHYCtiBC9A4Jy+ej2e2FWOz8ZlOvT\\\\\\\/\\\\\\\/VRgmx9CEuup795r1s7ftLo0yUyYZxH8sV11xQYgmPvzckVF\\\\\\\/VZ+8F\\\\\\\/5nOiLDmD9MelKXPep88A3bPNfsaf+\\\\\\\/1lIHsG+ZDT9k\\\\\\\/4TbQwCcdb\\\\\\\/53sVwtKrpm2rn0ADSNwHt5+Il5gxbowsy+WajulF4B+rlujaI1WYac\\\\\\\/\\\\\\\/sOjR7LMEGuNK2qlr4ZIn85i7FGEKg\\\\\\\/yjRggrmfx1YcblxMGCtK4+\\\\\\\/OKA4R8G4KfY43a0iz\\\\\\\/arn+SuMFPROYLnFsRdKAKYUi+qnB+aXScLzOISfeTPNKS6pJ1+nKbL6GeJVUmgxnlbP7N6ZH1y2xZ0di8z4Euzp9zkGkM4fbYqN5sOPZKiKsrryhqc9id91Q5JW2wOPoN0Q0PC5OUsNngFFeMTQ\\\\\\\/6o9pQW90vBQHa3p8EO5xqqH9rFt2cJos\\\\\\\/aNkFaJ26KomEscnFbUtE2no1\\\\\\\/e5T1p\\\\\\\/Z0tnxa7IGWUzHNZyxuvU9lhf++LEcEpGGg4COEIlkTVDBhaHTKLC3HG7bSVkYWCo+S0zBAh5hKSqAcmwEB04gHV4KhqAAZxV6sPaWSuLa0Si76PGjGs4WuJtxM0GGgAncDFU+5d6D8iDro4TxYZVGRTtZ\\\\\\\/Bl\\\\\\\/mFnJZG5s0zxn9+VNQxPqHdN2FCG08kfJdz\\\\\\\/\\\\\\\/Djv8qRo8f6pek17ixq12Eebl+N3XAo6IKKNst+\\\\\\\/8TQMdq9I0wyar0QlwiSIHdcX9\\\\\\\/cJJQAiUONBMjYng8kNZyNnOwW1wOSaGq0THcssDPGpeLAG5ZoL\\\\\\\/15OPgAeBvCifYlmeWL0N4oiAT\\\\\\\/2GWmaPPWI7hYB2O3CqvrgRJo2eSTNJ7z4nFKYY7KahSkuNQfKs1c3Gb9h7vOA1v5mZHPBZ7+Sh5eX43Ahey6KzgTvB\\\\\\\/Mb2oJV03pFyrcAw\\\\\\\/HEG5Boy2GcKEm\\\\\\\/eBYsgfhYFZl9I+sfRcABzYq3vjycEYSfN8TuTSRok2tfBolDOsFUbbY1gERir6BOc6tAftRzOJW4D\\\\\\\/lfrKIxCogiBT9mXnuhyNPKPNpeNDRjQGIAdQb55cTW3dvvjxtVuAA1\\\\\\\/FmqZfb2faiKJLp3eNZyBJS036yr0j\\\\\\\/COi6UXBIDV3U1EoiA4sX\\\\\\\/\\\\\\\/q8Y3nm1XVIrgw3KYiNsA0qbEsr\\\\\\\/toOUc8njXcTyJQzN33EWnWLpfl0YmAYzDZIY8zhy2zUSHqDQtCFDw6VcHSjmRfCyPbDKM3+L92D3z3ZaXMMsPPbOPQ4IvPIWhE8vtx0Uw7gpeI2CJRLQleWwbGgBqSG1MHgWDYKED1Z8ISNs2qit2MkqF3aaMCtuZ3FJNUaYOHxFYwUBjJN237FVDOloAGp7DlGNqCGUmvaPxlZEULmz6WSK06cuSAvVEUttdtoX7OyD2L6ulOSSdR3sf6j5hju54MKVhMV9Wz6YnCZRICWM23vqWQuqD2JkrON5a5UZNwoyb\\\\\\\/Osctcg9Gsq\\\\\\\/cpxP8d7y+rHvGumiwBd4\\\\\\\/2XaLbNlGLPQRKV34122ynaiWfU7rFQMt403XHKrFPMJbrqeQU8\\\\\\\/9uQfpDjfWlzyBB89rukZwZin7X3M2A7bStZna5AzQRmxPs2b5qg6bJ3SCOvphRfGrtpdrWVADLbpuyfxdHQruzwbUwRqdacC5CWHoHjXimABXkPJvibDLW742CLaJTpSsH3St8vbNB5NgqPx9WPyFsyGLddA2TqK+uS46VSdSmdubQ9qw9zSB5p\\\\\\\/qwHelw4vAGC\\\\\\\/4bU2VyNYYHpd4mpLiFi8FV\\\\\\\/kr9a01bKTNELCdS8f\\\\\\\/LbKtaa8rNoULAg92tk05NJpfTstNMuz+fr2iQf8mtLctUBkv8AkHtv04lqLGZFFp1TXRUoNPxSdLZBVCT98Be4TaWSUY9eLBCOpcTmq0QE8bwTupz1midZMlANwvOLix\\\\\\\/ooeZegxIe7FCCELpDQutZynpnxRGpgX67f1CJ5Hu9Z8xwwf\\\\\\\/dFtK48dATZlmaB3C+aNMOgUSAPDksi0R69scr32pH0V5FBREk8pzKZtCMTAa\\\\\\\/DCqjZjNrMmzbNASM1IZFeudiqaqznjQiFOkfN\\\\\\\/mu1eP2HyhgYbeOyg3f3QvJEWrqLs5dE++XEJ6hdnmXXDIG3SjZkqvTzSK2RMfGUqSkSAT9XzisqrbUJ5+t1IOhfH4XW2Bg6xmvFhIC7PfNJKlegtKFVFiexuOusSQPcdgCTnzNkhej8PTB9vzcURlALeeKBHdgL6UtlyBDN8P3dEOExRM+oafHm5LQR4QT3Q9lsmyFiwDh58cg5T31PPoyZRsAbkMSiWLi\\\\\\\/oHjicnibf6B\\\\\\\/yeQuYA2WatJlAwWvAW1QnwiA5pf3lN4ZzSfMoYqNrVrAvbRlai1Mn8S6hiOvC6CadyvwDYl+JUh3dlE\\\\\\\/bzsiL1e59f5\\\\\\\/f71hj35Cn+5DkFE7GAzPOw+hAkODO6XUrEeyQOZxIKAM6b1rex7W7AKiLsj1JTs90ELGx3gOqhdKWCpegeXo5HgBAsF4nVWTiVwM8hg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022F728SydgzCgAfMGWHfMCUg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f2e79b0f71a6f90c22fea063f1b80b3bb25b50b073d4d6ae85773f0188467455\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229+65Y6RrtEs3W1UxbUp0yO23lvImk3gq+rwN0CdfLjN8DqjizuO4umU6I9rbLj4fecaflLoxpa3w+Cmz3lSJh3gmnVYByVUtgdffm101vacaSq720z8hwH2OyiUbPmujvPd3Obi45r2lrVwNzcS3BJPg\\\\\\\/YsowB3XvAk0OCALcGG5X0YdExCPr5IB17Wr44zOIVrZoUVvyCiD4nsfPYaOU9wQlIugU53Jq7WYQIz3v9LCokGScmw+EpDmkOKYhdEoTp\\\\\\\/rvO8gL4yoUvkcxEhFF5IAFb5M2k3VOfiGRbVzTmEYnvrGu14K7xEVA1riSQ1ntqWcIwr99iliuEPWxjVobqHYszHkRSVZDQ\\\\\\\/GVHY7lCfArUjSVt7WMWZ7SGm85D+bV+evPN\\\\\\\/mLAP3To63aHuujXfpKdso5a9QlM1ZlxoBHDFAWXR0bRA07MM3yi0xnNUZm9dATL4HhzJDeWn3NpZcweGYeu4M3R4XoMa\\\\\\\/nvMBMpzPjaRy0LsHNiKzQtAlmA+4E3c2XRCB+Kb52lu9olCZ\\\\\\\/Ay6PKitsYmqiNKF6+xZNS77z\\\\\\\/fIvvtqONTIZNhuPuNyurO6cjejcgpMGZ4znlFEigMF8BzfYSKSxMjR6TO0gGdvv9QjM31H8uJ6MHuNQ\\\\\\\/cgfTSvBm+1Uk3NmFNX2hBlDgZzQTeZDaguG6vvxiX9GQZ5E+V7VElvZKfgYIe8gfdKbSw4u\\\\\\\/XDSFr+PWUzZcvmm+UboZt6BgYId6oOBTvte48rksPB9yyAavuGqY4ZgZbu7dbHpxW5BzOYotfbCGwLpIiY+pLg7HW7nEJ\\\\\\\/naq6yICSzPjH3kkd7MVpOEuda8aJ+KsIE5o0Pnh0fXYz\\\\\\\/2wbbZHdf\\\\\\\/Qh800Sh586uQcizfbU1U+QDOjznZI4sQc796tsjjnTwlYRJMe5jyYqHL4l5Q2uyb2IjzBlyPuZeBUpJzpn9hyUZg5bbmPBrL2gDN+BM40yxIxS+mTuW9dpiveyy27mijS5ALPXX0XdoahabIm71aQ3YdoYwYjOcMpqHHtYMfzBm5U4\\\\\\\/8NB1\\\\\\\/gi2JZxX8mxfZW929mJ7OrXC\\\\\\\/6FV3TuxWf\\\\\\\/FlTkqEKRhYYtaj5BsKT3PFZGMklcAE5ru\\\\\\\/pPcSW+r15bv+UmHnRmRmKD4WEhLXX0JFe9fsdk4xrIIPZC6VQnnMLhUehp6K4Gs0vOvfbEw9\\\\\\\/E\\\\\\\/g4HQj5+8cVUsN+iASFvR8ZLg5zsGhRajVPTyvMzVr39YB\\\\\\\/3vksJuyYSL1\\\\\\\/\\\\\\\/MWUEL+t\\\\\\\/lYH5OCSL9VmXXQgErE\\\\\\\/r0ztcoEqpfgS\\\\\\\/bAZ+gXzAWNR4EDiGhzr6Qp+QJLpV16CfMfxkdp4a2MyYDb13WU9zLkI5BF7o6iXh+Gm2WPN0AA83bDHn9\\\\\\\/lt4KHMTdt6vZHPY4gWj22h5tkMr35zxJD9Z\\\\\\\/5e9WLMYlAHmpKw6AVJ\\\\\\\/V4Yhktnr\\\\\\\/KylDJyknlM\\\\\\\/G4Ankw79kCXnXe2yjFQaVODD0yrorDMk9Am1UolI4YychHU7sLD3bOxjKH5xzsDjW6qypW6ik02aYCX3EwCLjcIogOLHPEeovHKIJn4R6gWec3fFWqVFuylBdxTe9tk+ZCdXwOSNnMb3HUYDEYctIrd9mkExVTQv5HqUKGqKRTi6AelTfzH14JwrkFPwLUJVEL0WeOzTow\\\\\\\/p2aaBT+bSn+F481qQYRGXmfrQS2rP9oiYIPkhQiOTljUb83c9YDEnk44yi5Pwl86jPMqa1e95Rnk8RyuZ6i\\\\\\\/6+re05Obe8598W9r0b3Qi9n+jlYtgjpZYkbApQMY+6pz069P3I9FXKqpqgNqoCOti1H8SFoqzTA34Gb6y8Pl41WpmzF\\\\\\\/cu9Fy+k9\\\\\\\/ipxaxvnosg3WdTY5B5ya7e5Leh4y78sXlqFRJykIP+l4EKV3x1bgDNlJJCIONmc1BPBIJBL6j3X8HaSJ\\\\\\\/FXDcbxQY+7kEYjmpEC4a91zuKFZv22ESVoGYvTNikWzK4Ap3rLyqBAZViGNTTqywGex7gaE3RCYkTdOEXwKA807tBDzsp05thMCA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022B\\\\\\\/RXjKDJmyULD6NEKTRgZg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ca368d85d23c6ace2e563c43260edfa79cac8d19fde47305a9abcc63ee2382b6\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459619 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/1/89" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Od1tiQH_ZYV_QbwNuR_tPvech7W-KddLJ8emB8bq3qs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/1/89" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rV0soicYzBSvIOTxnpO99TpQd42L2OJ0S8p04fVD7_Q" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/1/79" title="Genetic Variation and Covariation in Male Attractiveness and Female Mating Preferences in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/1/97" title="An Alternative to the Breeder’s and Lande’s Equations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459619" id="node459619" data-pisa="ggg;4/1/89" data-pisa-master="ggg;g3.113.007617" data-apath="/ggg/4/1/89.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Use of SNP Markers for Linkage Mapping in Diploid and Tetraploid Peanuts</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">David J. Bertioli</span>, <span class="highwire-citation-author" data-delta="1">Peggy Ozias-Akins</span>, <span class="highwire-citation-author" data-delta="2">Ye Chu</span>, <span class="highwire-citation-author" data-delta="3">Karinne M. Dantas</span>, <span class="highwire-citation-author" data-delta="4">Silvio P. Santos</span>, <span class="highwire-citation-author" data-delta="5">Ediene Gouvea</span>, <span class="highwire-citation-author" data-delta="6">Patricia M. Guimarães</span>, <span class="highwire-citation-author" data-delta="7">Soraya C. M. Leal-Bertioli</span>, <span class="highwire-citation-author" data-delta="8">Steven J. Knapp</span> and <span class="highwire-citation-author" data-delta="9">Marcio C. Moretzsohn</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">89-96;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007617 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459619" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David J. Bertioli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departamento de Genética/IB, Universidade de Brasília, Brasília, DF, 70910-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BBertioli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bertioli%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BJ.%2BBertioli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:davidbertioli@unb.br" class="" data-icon-position="" data-hide-link-title="0">davidbertioli@unb.br</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peggy Ozias-Akins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticulture, University of Georgia, Tifton, Georgia 31973</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peggy%2BOzias-Akins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ozias-Akins%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeggy%2BOzias-Akins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ye Chu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticulture, University of Georgia, Tifton, Georgia 31973</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ye%2BChu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYe%2BChu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Karinne M. Dantas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karinne%2BM.%2BDantas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dantas%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarinne%2BM.%2BDantas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Silvio P. Santos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Universidade Católica de Brasília, Brasília, DF, 70790-160, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Silvio%2BP.%2BSantos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santos%20SP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASilvio%2BP.%2BSantos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ediene Gouvea </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ediene%2BGouvea%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gouvea%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdiene%2BGouvea%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Patricia M. Guimarães </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BM.%2BGuimar%C3%A3es%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guimar%C3%A3es%20PM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatricia%2BM.%2BGuimar%25C3%25A3es%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Soraya C. M. Leal-Bertioli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Soraya%2BC.%2BM.%2BLeal-Bertioli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leal-Bertioli%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASoraya%2BC.%2BM.%2BLeal-Bertioli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Steven J. Knapp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Plant Breeding, Genetics, and Genomics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BJ.%2BKnapp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Knapp%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteven%2BJ.%2BKnapp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Marcio C. Moretzsohn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcio%2BC.%2BMoretzsohn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moretzsohn%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcio%2BC.%2BMoretzsohn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/1/89" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459619" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459619/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/1/89.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459619" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459619/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/1/89.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459619" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459619/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/1/89.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459619" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459619/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/1/89.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459619" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459619/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Single nucleotide polymorphic markers (SNPs) are attractive for use in genetic mapping and marker-assisted breeding because they can be scored in parallel assays at favorable costs. However, scoring SNP markers in polyploid plants like the peanut is problematic because of interfering signal generated from the DNA bases that are homeologous to those being assayed. The present study used a previously constructed 1536 GoldenGate SNP assay developed using SNPs identified between two <em>A. duranensis</em> accessions. In this study, the performance of this assay was tested on two RIL mapping populations, one diploid (<em>A. duranensis</em> × <em>A. stenosperma</em>) and one tetraploid [<em>A. hypogaea</em> cv. Runner IAC 886 × synthetic tetraploid (<em>A. ipaënsis</em> × <em>A. duranensis</em>)<sup>4×</sup>]. The scoring was performed using the software GenomeStudio version 2011.1. For the diploid, polymorphic markers provided excellent genotyping scores with default software parameters. In the tetraploid, as expected, most of the polymorphic markers provided signal intensity plots that were distorted compared to diploid patterns and that were incorrectly scored using default parameters. However, these scorings were easily corrected using the GenomeStudio software. The degree of distortion was highly variable. Of the polymorphic markers, approximately 10% showed no distortion at all behaving as expected for single-dose markers, and another 30% showed low distortion and could be considered high-quality. The genotyped markers were incorporated into diploid and tetraploid genetic maps of <em>Arachis</em> and, in the latter case, were located almost entirely on A genome linkage groups.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/arachis" class="hw-term hw-article-keyword hw-article-keyword-arachis" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Arachis</em></a></li><li class="kwd"><a href="/keyword/breeding" class="hw-term hw-article-keyword hw-article-keyword-breeding" rel="nofollow">breeding</a></li><li class="kwd"><a href="/keyword/genotyping" class="hw-term hw-article-keyword hw-article-keyword-genotyping" rel="nofollow">genotyping</a></li><li class="kwd"><a href="/keyword/markers" class="hw-term hw-article-keyword hw-article-keyword-markers" rel="nofollow">markers</a></li><li class="kwd"><a href="/keyword/wild" class="hw-term hw-article-keyword hw-article-keyword-wild" rel="nofollow">wild</a></li></ul><p id="p-4">The peanut (<em>Arachis hypogaea</em> L.) is a grain legume and oil crop of very significant importance in tropical and subtropical regions of the world. It has a narrow genetic base because of its recent allotetraploid origin (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Halward <em>et al.</em> 1991</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kochert <em>et al.</em> 1991</a>). This limits improvement for some agronomically important traits and has hindered the development of genetic maps and the application of molecular breeding.</p><p id="p-5">Cytogenetic, phylogeographical, and molecular evidence indicate that the most probable A and B genome donors to <em>A. hypogaea</em> are <em>A. duranensis</em> and <em>A. ipaënsis</em>, respectively (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kochert <em>et al.</em> 1996</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Seijo <em>et al.</em> 2007</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Robledo and Seijo 2010</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Moretzsohn <em>et al.</em> 2013</a>). It is estimated that the A and B genomes diverged approximately 3–3.5 million years ago (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nielen <em>et al.</em> 2012</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Moretzsohn <em>et al.</em> 2013</a>), much more recently than the subgenomes of cotton or soybean, which diverged approximately 6.7 and 13 million years ago, respectively (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Senchina <em>et al.</em> 2003</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Schmutz <em>et al.</em> 2010</a>).</p><p id="p-6">From genetic maps, it can be seen that the molecular marker order in the A and B genomes is conserved, with only a few major rearrangements between them (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Burow <em>et al.</em> 2001</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Moretzsohn <em>et al.</em> 2009</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Guo <em>et al.</em> 2012</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Shirasawa <em>et al.</em> 2013</a>). This gives further support to the evidence of recent divergence of the two genome components. However, it is also evident that the A and B genomes have diverged in a very significant way: cultivated peanut behaves genetically as a diploid and almost all pairing of chromosomes during meiosis is bivalent (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Smartt 1990</a>).</p><p id="p-7">The development of molecular markers for peanut has followed the technical trends of the times. The first studies were based on isozymes and proteins (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Krishna and Mitra 1988</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lu and Pickersgill 1993</a>), followed by restriction fragment length polymorphism (RFLP) (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Kochert <em>et al.</em> 1991</a>, <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">1996</a>), random amplified polymorphic DNA (RAPD) (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Halward <em>et al.</em> 1991</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hilu and Stalker 1995</a>), amplified fragment length polymorphism (AFLP) (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">He and Prakash 1997</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Tallury <em>et al.</em> 2005</a>), and, more recently, microsatellite markers (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hopkins <em>et al.</em> 1999</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Macedo <em>et al.</em> 2012</a>) and MITE-based markers (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Shirasawa <em>et al.</em> 2013</a>). Generally, these markers have shown a trend toward becoming more informative, cost-effective, and easier to use. However, all these marker types show low polymorphism in cultivated peanut germplasm. This has stimulated the interest in wild species as sources of genetic polymorphism for mapping and as sources of new phenotypic traits such as disease resistance in breeding programs (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bertioli <em>et al.</em> 2011</a>).</p><p id="p-8">Currently, microsatellites, being co-dominant and easy to score in the tetraploid genome, are the markers of choice for molecular breeding and genetic studies. For foreground selection, in which flanking markers are used to track QTL in breeding programs, microsatellite markers are an excellent option (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cheema <em>et al.</em> 2008</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wang and Chee 2010</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Foncéka <em>et al.</em> 2012</a>). However, for background selection, in which a larger number of markers are needed to provide an overview of the different parental contributions to a progeny’s genome, the cost and time needed for microsatellite assays are high or even prohibitive.</p><p id="p-9">Single nucleotide polymorphism (SNP) markers have proved very powerful in the genetic analysis of other species and useful in breeding programs. Perhaps most importantly, SNPs can be used in parallel assays, and the cost per data point is favorable compared to other marker types (when used in large-scale assays). However, SNP markers have proved very difficult to detect and apply in the cultivated peanut. This is because true SNP polymorphisms (A <em>vs.</em> A genome or B <em>vs.</em> B) are very rare and difficult to detect against the background of false A <em>vs.</em> B polymorphisms. If we consider that even very diverse peanut cultivars have diverged only a few thousand years ago, whereas the A and B genomes diverged a few million years ago, then we can expect true SNP rates to be in the region of 1000-times less frequent than false A <em>vs.</em> B rates. In addition to this difficulty of discovering SNPs are the difficulties of SNP scoring in a tetraploid genome.</p><p id="p-10">Here, we report the use of SNP markers in diploid and tetraploid recombinant inbred mapping populations of <em>Arachis</em>. We anticipate that these results provide a very significant step toward the use of highly parallel SNP assays for background selections in breeding progeny derived from cultivated peanut × wild crosses.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mapping populations</h3><p id="p-11">Recombinant inbred mapping populations were produced from unique F<sub>1</sub> plants cloned by cuttings to produce enough F<sub>2</sub> seeds and single-seed descent to the F<sub>5</sub>/F<sub>6</sub> generations. The F<sub>1</sub> plants were derived from the following: for the diploid population, a cross between <em>A. duranensis</em> K7988 and <em>A. stenosperma</em> V10309 (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Moretzsohn <em>et al.</em> 2005</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Leal-Bertioli <em>et al.</em> 2009</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bertioli <em>et al.</em> 2009</a>); and for the tetraploid population, from a cross between <em>A. hypogaea</em> cv. Runner IAC 886 and a colchicine-induced tetraploid with amphidiploid genetic behavior, made using the most probable ancestral species of cultivated peanut (<em>A. ipaënsis</em> K30076 and <em>A. duranensis</em> V14167)<sup>4×</sup> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Fávero <em>et al.</em> 2006</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fonceka <em>et al.</em> 2009</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Fonceka <em>et al.</em> 2012</a>).</p></div><div id="sec-3" class="subsection"><h3>Extraction of DNA and genotyping</h3><p id="p-12">Total genomic DNA was extracted from young leaves using the protocol of <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Grattapaglia and Sederoff (1994)</a>, modified by the inclusion of an additional precipitation step with 1.2 M NaCl. DNA concentration was estimated by agarose gel electrophoresis comparing the fluorescence intensities of the ethidium bromide–stained samples to those of lambda DNA standards. Samples were further quantified with picogreen and diluted to 10–50 ng/ul.</p><p id="p-13">SNP genotyping was performed using the 1536 Illumina GoldenGate array described by <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Nagy <em>et al.</em> (2012)</a>. This SNP array was constructed using SNP polymorphisms identified between two <em>A. duranensis</em> accessions, PI 475887 and Grif 15036. Most polymorphisms were between ESTs (1236), with the remaining being between intron sequences of predicted single copy genes (300) (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Choi <em>et al.</em> 2006</a>). As expected from the study of <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Nagy <em>et al.</em> (2012)</a>, not all SNP assays were functional. Of the total 1536 assays, 1054 produced marker genotypes that were incorporated into the diploid, intrapecific, <em>A. duranensis</em> A-genome map.</p><p id="p-14">The calling of genotypes was performed using the software GenomeStudio version 2011.1. The software assigns GenCall scores to each data point. The GenCall score is a value between zero and one and is primarily designed to filter out failed genotypes, DNAs, and/or loci (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Oliphant <em>et al.</em> 2002</a>). Scores less than 0.2 usually indicate failed genotypes and scores more than 0.7 usually report high-quality genotypes. For this work, all GenCall scores less than 0.25 were marked as “uncalled.”</p><p id="p-15">Also, the software assigns another score, the GenTrain score, to each SNP assay as a whole (compared to GenCall scores that relate to each individual data point). This score depends on the grouping of the clusters in the signal intensity plots and their relative distance from each other. This score is designed to mimic overall quality evaluations for each assay made by a human expert and, again, ranges from zero to one.</p></div><div id="sec-4" class="subsection"><h3>Construction of linkage maps</h3><p id="p-16">Linkage maps for the two populations used in this study have already been generated (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Shirasawa <em>et al.</em> 2013</a>). The SNP markers genotyped in this study were used together with selected markers from these previous maps to further saturate the maps and to observe the behavior of SNP markers in the contrasting diploid and tetraploid genetic states. In total, 387 and 800 polymorphic markers were used for map construction in the diploid and tetraploid populations, respectively. These consisted of the following: for the diploid A genome population, 329 SNPs and 58 selected syntenic microsatellite markers; and for the tetraploid population, 394 SNPs and 406 chosen microsatellite or MITE markers. The 58 microsatellite markers were included on the diploid A genome map to enable the linkage groups to be assigned according to previously published <em>Arachis</em> maps (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Moretzsohn <em>et al.</em> 2005</a>, <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">2009</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Leal-Bertioli <em>et al.</em> 2009</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gautami <em>et al.</em> 2012</a>; <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Shirasawa <em>et al.</em> 2013</a>). Additionally, 100 selected syntenic microsatellites were included on the A genome component of the tetraploid map to enable A–B homeolog identification. The remaining 306 microsatellites were mapped on the B genome component of the tetraploid map. Because the SNPs were developed for an A genome species (<em>A. duranensis</em>), only a few SNP markers were mapped on the B genome. Therefore, the inclusion of a higher number of microsatellite markers in the B genome component map was necessary for construction of the 10 linkage groups. Mapping populations consisted of 90 F<sub>5</sub> and 89 F<sub>6</sub> individuals for the diploid and tetraploid populations, respectively.</p><p id="p-17">A chi-squared test was performed to test for 1:1 segregations on all scored markers. Only marker loci that did not show segregation distortion (<em>P</em> &lt; 0.05) were used for the initial map construction to eliminate spurious linkages. The linkage analysis was performed using Mapmaker Macintosh version 2.0 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lander <em>et al.</em> 1987</a>). A minimum LOD score of 4.0 and maximum recombination fraction (θ) of 0.35 were set as thresholds for linkage group (LG) determination with the “group” command. The most likely marker order within each LG was estimated by the matrix correlation method using the “first order” command. Marker orders were confirmed by permuting all adjacent triple orders (“ripple” command). After establishment of the group orders, the LOD score was set to 3.0 to include additional markers in the groups. The exact position of the new markers within each group was determined by using the “try” command, which compares the maximum likelihood of each marker order after placing the markers, one by one, into every interval of the established order. In a next step, distorted markers were included using the “group” command. The new marker orders were again confirmed with the “first order” and/or “ripple” commands. Recombination fractions were converted into map distances in centimorgans using the Kosambi mapping function and the “error detection” available in Mapmaker/EXP 3.0 (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Lander <em>et al.</em> 1987</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lincoln <em>et al.</em> 1992</a>).</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>Production of mapping populations</h3><p id="p-18">When generations were advanced, the vigor of the lines generally had a tendency to decline. In the case of the tetraploid population, this tendency was very slight and, generally, the vigor and seed set of the lines were very high. In the case of the diploid population, the loss of vigor and poor seed set were much more common. Lines of both populations contrasted in many phenotypic traits, such as flower color, height of central stem, number and length of side branches, seed size, pod morphology, resistance to foliar diseases and nematodes, and many other characteristics.</p></div><div id="sec-7" class="subsection"><h3>Genotyping</h3><p id="p-19">From the 1536 SNP assays on the Illumina array, some detected polymorphism but others did not. Initial screening of parental DNAs indicated polymorphism for approximately 500 of the SNP assays in the diploid A genome and tetraploid mapping populations. The parentals of a B genome mapping population derived from a cross between <em>A. ipaënsis</em> and <em>A. magna</em> (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Moretzsohn <em>et al.</em> 2009</a>) were also screened, but this indicated that only approximately 100 assays may be informative.</p><p id="p-20">In the full A genome diploid population, 329 assays were informative. Generally, separation of the genotypic states was very clear with the points lying along the “x” and “y” axes of the signal intensity plots (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Genotyping calling of signal intensity plots were inspected manually and only 30 needed to be manually changed. GenTrain scores ranged from 0.01 to 0.90, with a median value of 0.70 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/89/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Examples of plots of signal intensities from SNP assay on the diploid population (A) and the tetraploid population after manual correction (B). Points in red and blue are homozygotes; points in purple are heterozygotes. Note how, in the tetraploid, the clustering is distorted and the separation between the genotype groups is reduced (B)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1558178324" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Examples of plots of signal intensities from SNP assay on the diploid population (A) and the tetraploid population after manual correction (B). Points in red and blue are homozygotes; points in purple are heterozygotes. Note how, in the tetraploid, the clustering is distorted and the separation between the genotype groups is reduced (B).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/89/F1.medium.gif" width="297" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/1/89/F1.medium.gif" width="297" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/89/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/89/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463810" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Examples of plots of signal intensities from SNP assay on the diploid population (A) and the tetraploid population after manual correction (B). Points in red and blue are homozygotes; points in purple are heterozygotes. Note how, in the tetraploid, the clustering is distorted and the separation between the genotype groups is reduced (B).</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/89/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequency distributions in numbers of SNPs in quality score (Gentrain scores) classes for genotyping assays in the diploid (green) and tetraploid (yellow) mapping populations. The numbers of assays in each class are written above the bars. The scores for the diploid population form a single high-scoring peak, the scores for the tetraploid population form a double peak. Note that a significant number of the assays in the tetraploid population are high-quality." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1558178324" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Frequency distributions in numbers of SNPs in quality score (Gentrain scores) classes for genotyping assays in the diploid (green) and tetraploid (yellow) mapping populations. The numbers of assays in each class are written above the bars. The scores for the diploid population form a single high-scoring peak, the scores for the tetraploid population form a double peak. Note that a significant number of the assays in the tetraploid population are high-quality.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/89/F2.medium.gif" width="440" height="356"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/1/89/F2.medium.gif" width="440" height="356"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/89/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/89/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463873" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">Frequency distributions in numbers of SNPs in quality score (Gentrain scores) classes for genotyping assays in the diploid (green) and tetraploid (yellow) mapping populations. The numbers of assays in each class are written above the bars. The scores for the diploid population form a single high-scoring peak, the scores for the tetraploid population form a double peak. Note that a significant number of the assays in the tetraploid population are high-quality.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">In the tetraploid population, 394 assays were informative. Signal intensity plots were generally distorted compared to the diploid, with the points representing one allele in the homozygous state lying along one axis and points representing the other homozygous state being shifted at an angle (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Almost all the default parameter genotyping calls needed to be manually modified (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>); only 43 were left unchanged. GenTrain scores ranged from 0.15 to 0.88, with a median value of 0.42 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Full genotyping information for both mapping populations is available (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007617/-/DC1">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007617/-/DC1/FileS1.xlsx">File S1</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/89/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="An example of a plot of signal intensities from SNP assay on the tetraploid population using default parameters (A) and after manual correction (B). Points in red and blue are called as homozygotes; points in purple are heterozygotes. Note how genotype calling using default parameters falsely assigns one of the homozygous genotypes as heterozygous." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1558178324" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;An example of a plot of signal intensities from SNP assay on the tetraploid population using default parameters (A) and after manual correction (B). Points in red and blue are called as homozygotes; points in purple are heterozygotes. Note how genotype calling using default parameters falsely assigns one of the homozygous genotypes as heterozygous.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/89/F3.medium.gif" width="301" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/1/89/F3.medium.gif" width="301" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/89/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/89/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463840" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">An example of a plot of signal intensities from SNP assay on the tetraploid population using default parameters (A) and after manual correction (B). Points in red and blue are called as homozygotes; points in purple are heterozygotes. Note how genotype calling using default parameters falsely assigns one of the homozygous genotypes as heterozygous.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">The frequency distributions of Gentrain scores in the diploid and tetraploid populations are distinct (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). The diploid shows a single peak at 0.7–0.8. The tetraploid shows two peaks, one at 0.2–0.3 and the other at 0.5–0.6.</p></div><div id="sec-8" class="subsection"><h3>Linkage maps</h3><div id="sec-9" class="subsection"><h4>Diploid A genome map:</h4><p id="p-26">Using a minimum LOD score of 3.0 and a maximum recombination fraction of 0.35, 384 markers mapped into 10 linkage groups. These markers included 326 SNPs and 58 microsatellites. The map covered a total distance of 705.1 cM (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4, A and B</a>). Groups ranged from 49.5 cM to 120.1 cM, with an average distance of 1.84 cM between adjacent markers. Linkage groups were numbered according to the LG numbers of the A genome map (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Moretzsohn <em>et al.</em> 2005</a>; <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Leal-Bertioli <em>et al.</em> 2009</a>) by the identification of syntenic markers.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/89/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Linkage maps of the diploid A and tetraploid AB Arachis genomes (linkage groups 1&#x2013;5). The representation of the diploid A (linkage groups assigned AN) was generated using recombinant inbred lines derived from an A. duranensis &#xD7; A. stenosperma cross. The representation of the tetraploid AB genome (linkage groups assigned TAN and TBN) was derived from a cross between cultivated peanut and an artificially induced tetraploid derived from two diploid wild species, A. ipa&#xEB;nsis and A. duranensis. Distances are centimorgans. Lines between linkage groups indicate marker correspondences and show genome syntenies. Single nucleotide polymorphism markers (SNPs) are represented in red; other markers, the incorporation of which were chosen to assign linkage groups and show synteny between A and B genomes, are shown in black. Note that almost all SNP markers are incorporated into the A genome of the tetraploid. (B) Linkage maps of the diploid A and tetraploid AB Arachis genomes (linkage groups 6&#x2013;10). The representation of the diploid A (linkage groups assigned AN) was generated using recombinant inbred lines derived from an A. duranensis &#xD7; A. stenosperma cross. The representation of the tetraploid AB genome (linkage groups assigned TAN and TBN) was derived from a cross between cultivated peanut and an artificially induced tetraploid derived from two diploid wild species, A. ipa&#xEB;nsis and A. duranensis. Distances are in centimorgans. Lines between linkage groups indicate marker correspondences and show genome syntenies. SNPs are represented in red; other markers, the incorporation of which was chosen to assign linkage groups and show synteny between A and B genomes, are shown in black. Note that almost all SNP markers are incorporated into the A genome of the tetraploid." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1558178324" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Linkage maps of the diploid A and tetraploid AB &lt;em&gt;Arachis&lt;/em&gt; genomes (linkage groups 1&#x2013;5). The representation of the diploid A (linkage groups assigned AN) was generated using recombinant inbred lines derived from an &lt;em&gt;A. duranensis&lt;/em&gt; &#xD7; &lt;em&gt;A. stenosperma&lt;/em&gt; cross. The representation of the tetraploid AB genome (linkage groups assigned TAN and TBN) was derived from a cross between cultivated peanut and an artificially induced tetraploid derived from two diploid wild species, &lt;em&gt;A. ipa&#xEB;nsis&lt;/em&gt; and &lt;em&gt;A. duranensis&lt;/em&gt;. Distances are centimorgans. Lines between linkage groups indicate marker correspondences and show genome syntenies. Single nucleotide polymorphism markers (SNPs) are represented in red; other markers, the incorporation of which were chosen to assign linkage groups and show synteny between A and B genomes, are shown in black. Note that almost all SNP markers are incorporated into the A genome of the tetraploid. (B) Linkage maps of the diploid A and tetraploid AB &lt;em&gt;Arachis&lt;/em&gt; genomes (linkage groups 6&#x2013;10). The representation of the diploid A (linkage groups assigned AN) was generated using recombinant inbred lines derived from an &lt;em&gt;A. duranensis&lt;/em&gt; &#xD7; &lt;em&gt;A. stenosperma&lt;/em&gt; cross. The representation of the tetraploid AB genome (linkage groups assigned TAN and TBN) was derived from a cross between cultivated peanut and an artificially induced tetraploid derived from two diploid wild species, &lt;em&gt;A. ipa&#xEB;nsis&lt;/em&gt; and &lt;em&gt;A. duranensis&lt;/em&gt;. Distances are in centimorgans. Lines between linkage groups indicate marker correspondences and show genome syntenies. SNPs are represented in red; other markers, the incorporation of which was chosen to assign linkage groups and show synteny between A and B genomes, are shown in black. Note that almost all SNP markers are incorporated into the A genome of the tetraploid.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/89/F4.medium.gif" width="440" height="412"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/1/89/F4.medium.gif" width="440" height="412"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/89/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/89/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463804" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-27" class="first-child">(A) Linkage maps of the diploid A and tetraploid AB <em>Arachis</em> genomes (linkage groups 1–5). The representation of the diploid A (linkage groups assigned AN) was generated using recombinant inbred lines derived from an <em>A. duranensis</em> × <em>A. stenosperma</em> cross. The representation of the tetraploid AB genome (linkage groups assigned TAN and TBN) was derived from a cross between cultivated peanut and an artificially induced tetraploid derived from two diploid wild species, <em>A. ipaënsis</em> and <em>A. duranensis</em>. Distances are centimorgans. Lines between linkage groups indicate marker correspondences and show genome syntenies. Single nucleotide polymorphism markers (SNPs) are represented in red; other markers, the incorporation of which were chosen to assign linkage groups and show synteny between A and B genomes, are shown in black. Note that almost all SNP markers are incorporated into the A genome of the tetraploid. (B) Linkage maps of the diploid A and tetraploid AB <em>Arachis</em> genomes (linkage groups 6–10). The representation of the diploid A (linkage groups assigned AN) was generated using recombinant inbred lines derived from an <em>A. duranensis</em> × <em>A. stenosperma</em> cross. The representation of the tetraploid AB genome (linkage groups assigned TAN and TBN) was derived from a cross between cultivated peanut and an artificially induced tetraploid derived from two diploid wild species, <em>A. ipaënsis</em> and <em>A. duranensis</em>. Distances are in centimorgans. Lines between linkage groups indicate marker correspondences and show genome syntenies. SNPs are represented in red; other markers, the incorporation of which was chosen to assign linkage groups and show synteny between A and B genomes, are shown in black. Note that almost all SNP markers are incorporated into the A genome of the tetraploid.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h4>Map for the tetraploid genome:</h4><p id="p-28">Using a minimum LOD score of 3.0 and a maximum recombination fraction of 0.35, 772 markers mapped into 20 linkage groups. Of these, 460 markers mapped into 10 linkage groups corresponding to the A genome component, and 312 markers mapped into the 10 B genome linkage groups. The 460 A genome markers included 360 SNPs and 100 microsatellites, whereas the B markers included 6 SNPs and 306 microsatellites. The map covered a total distance of 1487.3 cM, with 941.9 cM and 545.4 cM for the A and B genome components, respectively (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, A and B</a>). Groups ranged from 62.2 cM to 127.1 cM, with an average distance of 2.05 cM, on the A genome and from 28.4 cM to 97.8 cM, with an average of 1.75 cM between adjacent markers, on the B genome component. Linkage groups were numbered according to syntenic markers as compared to the A diploid map.</p></div><div id="sec-11" class="subsection"><h4>Length of linkage maps:</h4><p id="p-29">The total genetic distances of our maps that were constructed using Mapmaker and the Kosambi mapping function were longer than those produced by <a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Shirasawa <em>et al.</em> (2013)</a>, which were constructed using JoinMap with the Haldane mapping function. Generally, the Haldane mapping function provides longer distances than the Kosambi function, so the use of different mapping functions could not account for the differences. Initially, we had calculated the genetic distances in Mapmaker without genotyping error correction. The discrepancies between the total distances in these initial versions of our maps and the maps of Shirasawa were very considerable: 2445.4 <em>vs.</em> 1442 cM for the tetraploid and 987.6 <em>vs.</em> 544 cM for the diploid. When Mapmaker was run with error correction, the discrepancies were reduced and became small for the tetraploid map (1487.3 <em>vs.</em> 1442 cM) but remained substantial for the diploid A map (705.1 <em>vs.</em> 544 cM).</p><p id="p-30">We investigated whether the remaining differences were attributable to different numbers of recombination events implicit in the different maps and/or were attributable to the influence of the different algorithms used in the mapping programs. First, using Excel sheets, we calculated estimations of the numbers of recombination events implicit in the two maps. Exact comparisons were difficult because of the large number of dominant markers in the Shirasawa map, and because of different error rates for the different types of markers used. However, the numbers of recombination events for the different maps seem similar. Then, we investigated the influence of the different mapping algorithms. We recalculated linkage group distances using Mapmaker and both Kosambi and Haldane functions, but using the marker orders obtained by <a id="xref-ref-40-6" class="xref-bibr" href="#ref-40">Shirasawa <em>et al.</em> (2013)</a>. The newly calculated distances were longer than the ones obtained using JoinMap. To discard the possibility that the differences were attributable to the presence of both co-dominant and dominant markers in the Shirasawa map, we reanalyzed their data using only the co-dominant markers and, again, differences were obtained. We also reordered the markers with Mapmaker. This usually provided a result somewhat different from the JoinMap orders. The calculated differences decreased with these reordered markers, but they were still longer than distances calculated using JoinMap (results not shown).</p></div><div id="sec-12" class="subsection"><h4>Synteny analysis:</h4><p id="p-31">A total of 155 common markers were mapped in both the A and the tetraploid genome maps, while 59 were common to the A and B genome components of the tetraploid map (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4, A and B</a>). The 10 A genome linkage groups showed direct correspondences in both maps and the great majority of the loci were mapped in the same order. Markers were also generally collinear in both genome components of the tetraploid map, but linkage group B7 showed common markers to both A7 and A8 linkage groups. Direct correspondences were observed for all the other linkage groups. The linkage maps and genotyping data are available in an Excel sheet (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007617/-/DC1/FileS1.xlsx">File S1</a>).</p></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-32">SNP markers are attractive for use in genetic mapping and marker-assisted breeding because they can be scored in parallel assays at favorable costs. Genotyping assays that detect SNPs are highly suitable to diploid plants. The assays, which are designed to detect a single locus (a DNA base) within the haploid genome, can efficiently distinguish the different combinations of bases present in the diploid genome (<em>e.g.</em>, GG, GT, TT). The application of SNPs in tetraploid plants is more difficult because the assays often detect four DNA bases (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akhunov <em>et al.</em> 2009</a>).</p><p id="p-33">Peanut is a crop with an allotetraploid genome of the type AABB (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Husted 1936</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Smartt <em>et al.</em> 1990</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fernández and Krapovickas 1994</a>). It is estimated that the two genomes diverged approximately 3–3.5 million years ago (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Nielen <em>et al.</em> 2012</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Moretzsohn <em>et al.</em> 2013</a>). Examination of A–B homeologous genome regions at the DNA sequence level reveals conserved segments of high sequence identity (approximately 95%), punctuated by <em>indel</em> regions without significant similarity, often consisting of repetitive DNA (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bertioli <em>et al.</em> 2009</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">2013</a>). Sequence identity within genes is even higher (approximately 97%). This study confirmed that, as indicated in previous studies (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Burow <em>et al.</em> 2001</a>; <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Moretzsohn <em>et al.</em> 2009</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Foncéka <em>et al.</em> 2009</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Guo <em>et al.</em> 2012</a>; <a id="xref-ref-40-7" class="xref-bibr" href="#ref-40">Shirasawa <em>et al.</em> 2013</a>), A and B genomes are highly syntenic and most linkage groups present single correspondences and collinear marker order. For the markers that are in common with those of <a id="xref-ref-40-8" class="xref-bibr" href="#ref-40">Shirasawa <em>et al.</em> (2013)</a>, the orders generated here are very similar; however, overall, colinearity between the A and B genomes is somewhat higher. This most likely indicates a slight improvement in accuracy of marker order because of the iterative use of the “ripple” command in the Mapmaker software. The maps generated here are longer than those obtained by <a id="xref-ref-40-9" class="xref-bibr" href="#ref-40">Shirasawa <em>et al.</em> (2013)</a>, and this difference is small for the tetraploid map (1487.3 <em>vs.</em> 1442 cM) but is substantial for the diploid A map (705.1 <em>vs.</em> 544 cM). These differences cannot be explained by an increase in the number of predicted recombination events but are apparently attributable to the different software used, with Mapmaker used in this study and JoinMap used by <a id="xref-ref-40-10" class="xref-bibr" href="#ref-40">Shirasawa <em>et al.</em> (2013)</a>. Several studies have shown that maps constructed with JoinMap are shorter than those constructed with Mapmaker (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Sewell <em>et al.</em> 1999</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Butcher <em>et al.</em> 2002</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Tani <em>et al.</em> 2003</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gawłowska <em>et al.</em> 2005</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Van’t Hof <em>et al.</em> 2008</a>). Although the source of these differences is not completely clear, different algorithms for the treatment of genotyping errors seem to be the most likely cause. The large-scale exception in colinearity between the A and B maps observed here was also observed previously (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Foncéka <em>et al.</em> 2009</a>; <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Moretzsohn <em>et al.</em> 2009</a>; <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Guo <em>et al.</em> 2012</a>; <a id="xref-ref-40-11" class="xref-bibr" href="#ref-40">Shirasawa <em>et al.</em> 2013</a>). It indicates an apparent translocation event between LGs B7 and B8. Because this rearrangement was initially observed in a comparison between linkage maps developed for diploid species with A and B genomes (<a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">Moretzsohn <em>et al.</em> 2009</a>), this chromosomal rearrangement must have happened before the tetraploidization of the cultivated peanut.</p><p id="p-34">Here, we used a previously developed 1536 Illumina assay (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Nagy <em>et al.</em> 2012</a>) in diploid and tetraploid RIL mapping populations. The diploid population is derived from a cross of two wild A genome species, <em>A. duranensis</em> and <em>A. stenosperma</em>. The tetraploid population is derived from the cultivated peanut crossed with an artificially induced tetraploid [<em>A. ipaënsis</em> × <em>A. duranensis</em>]<sup>4×</sup>. These two populations provide an excellent genetic environment for study because the diploid is genetically simple, and the structure of the tetraploid population allows the A and B homeologs to be assigned. Although the SNPs were discovered using <em>A. duranensis</em> accessions not used for making these mapping populations (PI 475887 and Grif 15036), many of the markers did transfer to both populations used in this study. Transfer of markers was significantly higher for the tetraploid population (394) than the diploid population (329). This shows that the <em>A. duranensis vs. A. duranensis</em> comparison used for SNP discovery is more similar to the <em>A. duranensis</em> V14167 <em>vs. A. hypogaea</em> A genome comparison in the tetraploid population than to <em>A. duranensis</em> K7988 <em>vs. A. stenosperma</em> V10309 comparison in the diploid population used here.</p><p id="p-35">Signal intensity plots from our A genome diploid mapping population had high average genotyping quality (GenTrain) scores, and the signal intensity plots were generally easy to interpret, with good separation between genotype calls (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). The frequency distribution in GenTrain scores in the diploid shows a single high-scoring peak (at 0.7–0.79) (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). In the tetraploid, as expected, most of the polymorphic markers provided signal intensity plots that were distorted compared to diploid patterns and that were incorrectly scored using default parameters (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). However, these scorings were easily manually corrected using the GenomeStudio software (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). The degree of distortion in the tetraploid plots was variable, and the markers showed two distinct peaks in the frequency distribution of quality scores (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>).</p><p id="p-36">To explain this distribution of GenTrain scores in the tetraploid genotyping, we can envisage a scenario when the common ancestral species of the A and B genomes both harbored a monomorphic base at one locus in their genomes, <em>e.g.</em>, G. During evolution, a mutation may occur at this locus. This may happen in either the A or the B genome lineages, but it is very unlikely to happen in both. For this scenario, let us consider a G-to-T substitution in the A genome. Considering this polymorphism is present in our tetraploid population, this locus on the A genome will then have two possible allelic states, G or T. However, the homeologous position on the B genomes will harbor the monomorphic ancestral state with only one possible allelic state, G. In the mapping population, the genotypic combinations possible will be GG:GG, TG:GG, and TT:GG. In RIL populations, as in the case here, the heterozygous genotype is rare, making the genotype assays easier to score. Most of the signal intensity plots we observed were distorted in a way that was consistent with this type of scenario. The excess of one base (in this scenario, G) shifts one of the genotype clusters and reduces the separation between the groupings (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). However, the degree of distortion was variable and formed two peaks in a frequency distribution of Gentrain scores (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>). Genotyping assays with quality scores near the lower-scoring peak in frequency distribution (0.2–0.29) possibly detect bases on the A and B genomes with equal efficiency. Genotyping assays with quality scores near the higher-scoring peak in frequency distribution (0.5–0.59) possibly detect bases on one subgenome (here, almost always the A genome) with significantly greater efficiency than the other (presumably because of other A or B genome-specific substitutions or indels at or near the locus being assayed). In either case, the manual correction of tetraploid genotyping was simple; the software allows the calls to be adjusted for each marker on the whole population in a few steps (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). As for mapping populations with a greater proportion of heterozygous individuals (such as F<sub>2</sub> populations), we anticipate that approximately 169 (40%) of the polymorphic assays had sufficiently low distortion that they would provide high-quality scores of all possible genotypes. SNP genotyping in the autotetraploid potato, in which five genotypes from nulliplex to quadruplex are possible, recently has been implemented with new algorithms (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Voorrips <em>et al.</em> 2011</a>) and features in GenomeStudio (Tech Note 970-2011-001). However, only three genotypes would be expected in allotetraploid peanut RIL populations with some residual heterozygosity because chromosome pairing is almost exclusively bivalent and only one of the two subgenomes would contain alternate alleles.</p><p id="p-37">Genotyped markers in both the diploid and tetraploid populations had very high success in mapping into the linkage maps (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). Generally, the markers mapped with small genetic distances to their flanking markers. This shows that genotyping errors were low, because errors artificially increase distances between markers. In the diploid population, markers were distributed on all linkage groups. In the tetraploid, markers were almost exclusively placed on the A genome linkage groups and scattered throughout the 10 linkage groups (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). If only these SNP markers had been used for linkage map construction, then an A genome diploid map would have been constructed within a tetraploid context! This indicates that the scenario described in the previous paragraph is a good model for the origin of most polymorphisms. SNPs are highly diagnostic of either the A or the B genome clades. It also provides very graphic evidence that further supports the highly diploid genetic behavior of allotetraploid peanut.</p><p id="p-38">The subgenome specificity of SNPs points to another possible utility, aiding the assembly of the cultivated peanut genome, which is currently being sequenced (<a href="http://www.peanutbioscience.com/peanutgenomeinitiative.html">http://www.peanutbioscience.com/peanutgenomeinitiative.html</a>). One of the difficulties in this project arises from the similarity of the A and B genomes of the cultivated peanut. This similarity causes problems during assembly in which reads from the A and B genomes are falsely assembled into the same contig, causing errors and breaks in the genome assembly. One possible solution is the use of sequencing technologies that produce longer sequence reads. Another, perhaps complementary, solution would be to use knowledge of A and B genome-specific SNPs to segregate reads into A types or B types before assembly. The extreme genome specificity of the SNPs assayed here supports the feasibility of this approach.</p><p id="p-39">Cultivated peanut has a very narrow genetic base and this has hampered the progress of genetic studies using cultivated × cultivated crosses. Our results here show that genotyping and map construction using SNPs in cultivated peanut are likely to be feasible. However, the discovery of a sufficient number of SNPs present in cultivated germplasm will be very challenging. The SNP assay used here would be of very limited use for cultivated peanut: only 88 SNPs were polymorphic across a panel of 70 cultivated <em>A. hypogaea</em> inbreds (P. Ozias-Akins, unpublished results). Furthermore, of these 88, only 42 showed a minor allele frequency more than 0.04. The narrow genetic base of cultivated peanut also has limited the gains obtained for some agronomic traits of interest in breeding programs that use only cultivated germplasm. This has stimulated interest in wild species that are a rich source of new alleles and new traits. However, wild species are not agronomically adapted and most wild alleles need to be eliminated by backcrossing to obtain a peanut cultivar. In this context, foreground and background genotyping schemes are particularly attractive in progeny derived from cultivated × wild crosses. Selected SNPs from the assay used here can be effectively used to monitor wild introgressions into cultivated peanut on the A genome. To generate an assay for monitoring wild introgressions on the B genome, SNPs identified in a panel of wild B genome species could be used to construct an informative assay. Together with the A genome SNPs identified here, this would allow the whole genome monitoring of wild allele introgression into cultivated peanut on a scale, and at a cost, that was previously impossible.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">The authors thank Dr. John Bowers and Dr. James Holland for helpful advice. This research was supported by funding from the Peanut Foundation and the USDA National Institute of Food and the Agriculture National Research Initiative Competitive Grants Program (#2006-35604-17242 awarded to S.J.K. and P.O.A.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007617/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007617/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-41"><em>Communicating editor: J. B. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-07-23"><span class="received-label">Received </span>July 23, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-11-01"><span class="accepted-label">Accepted </span>November 1, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Bertioli <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.1" data-doi="10.1007/s00122-009-1059-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akhunov</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Single nucleotide polymorphism genotyping in polyploid wheat with the Illumina GoldenGate assay.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">507</span>–<span class="cit-lpage">517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAkhunov%26rft.auinit1%253DE.%26rft.volume%253D119%26rft.issue%253D3%26rft.spage%253D507%26rft.epage%253D517%26rft.atitle%253DSingle%2Bnucleotide%2Bpolymorphism%2Bgenotyping%2Bin%2Bpolyploid%2Bwheat%2Bwith%2Bthe%2BIllumina%2BGoldenGate%2Bassay.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1059-5%26rft_id%253Dinfo%253Apmid%252F19449174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1059-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19449174&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268329400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.2" data-doi="10.1186/1471-2164-10-45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bertioli</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandal</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leal-Bertioli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">An analysis of synteny of <em>Arachis</em> with <em>Lotus</em> and <em>Medicago</em> sheds new light on the structure, stability and evolution of legume genomes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBertioli%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D10%26rft.spage%253D45%26rft.epage%253D45%26rft.atitle%253DAn%2Banalysis%2Bof%2Bsynteny%2Bof%2BArachis%2Bwith%2BLotus%2Band%2BMedicago%2Bsheds%2Bnew%2Blight%2Bon%2Bthe%2Bstructure%252C%2Bstability%2Band%2Bevolution%2Bof%2Blegume%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-45%26rft_id%253Dinfo%253Apmid%252F19166586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-45&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19166586&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.3" data-doi="10.1017/S1479262110000444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bertioli</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seijo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freitas</span>  <span class="cit-name-given-names">F. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valls</span>  <span class="cit-name-given-names">J. F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leal-Bertioli</span>  <span class="cit-name-given-names">S. C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">An overview of peanut and its wild relatives.</span> <abbr class="cit-jnl-abbrev">Plant Genet. Resour. Characterization Util.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">134</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenet.%2BResour.%2BCharacterization%2BUtil.%26rft.volume%253D9%26rft.spage%253D134%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1479262110000444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1479262110000444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.4" data-doi="10.1093/aob/mct128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bertioli</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidigal</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ratnaparkhe</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T.-H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The repetitive component of the A genome of peanut (<em>Arachis hypogaea</em>) and its role in remodelling intergenic sequence space since its evolutionary divergence from the B genome.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmct128%26rft_id%253Dinfo%253Apmid%252F23828319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjk6IjExMi8zLzU0NSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzQvMS84OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starr</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Transmission genetics of chromatin from a synthetic amphidiploid to cultivated peanut (<em>Arachis hypogaea</em> L.): Broadening the gene pool of a monophyletic polyploid species.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">823</span>–<span class="cit-lpage">837</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBurow%26rft.auinit1%253DM.%2BD.%26rft.volume%253D159%26rft.issue%253D2%26rft.spage%253D823%26rft.epage%253D837%26rft.atitle%253DTransmission%2BGenetics%2Bof%2BChromatin%2BFrom%2Ba%2BSynthetic%2BAmphidiploid%2Bto%2BCultivated%2BPeanut%2B%2528Arachis%2Bhypogaea%2BL.%2529%253A%2BBroadening%2Bthe%2BGene%2BPool%2Bof%2Ba%2BMonophyletic%2BPolyploid%2BSpecies%26rft_id%253Dinfo%253Apmid%252F11606556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzIvODIzIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.6" data-doi="10.1007/s00122-001-0820-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butcher</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitaker</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ling</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Improving linkage analysis in outcrossed forest trees—an example from <em>Acacia mangium</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">1185</span>–<span class="cit-lpage">1191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DButcher%26rft.auinit1%253DA.%26rft.volume%253D104%26rft.issue%253D6-7%26rft.spage%253D1185%26rft.epage%253D1191%26rft.atitle%253DImproving%2Blinkage%2Banalysis%2Bin%2Boutcrossed%2Bforest%2Btrees%2B-%2Ban%2Bexample%2Bfrom%2BAcacia%2Bmangium.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-001-0820-1%26rft_id%253Dinfo%253Apmid%252F12582629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-001-0820-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12582629&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176083200036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.7" data-doi="10.1007/s10681-007-9560-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheema</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bains</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mangat</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Das</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vikal</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Development of high yielding IR64 x <em>Oryza rufipogon</em> (Griff.) introgression lines and identification of introgressed alien chromosome segments using SSR markers.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D160%26rft.spage%253D401%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-007-9560-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-007-9560-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.8" data-doi="10.1007/s00438-006-0118-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">H.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luckow</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Development of nuclear gene-derived molecular markers linked to legume genetic maps.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">276</span>: <span class="cit-fpage">56</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DChoi%26rft.auinit1%253DH.%2BK.%26rft.volume%253D276%26rft.issue%253D1%26rft.spage%253D56%26rft.epage%253D70%26rft.atitle%253DDevelopment%2Bof%2Bnuclear%2Bgene-derived%2Bmolecular%2Bmarkers%2Blinked%2Bto%2Blegume%2Bgenetic%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-006-0118-8%26rft_id%253Dinfo%253Apmid%252F16642337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-006-0118-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16642337&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238521500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.9" data-doi="10.2135/cropsci2005.09-0331"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fávero</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valls</span>  <span class="cit-name-given-names">J. F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vello</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Study of the evolution of cultivated peanut through crossability studies among <em>Arachis ipaënsis</em>, <em>A. duranensis</em>, and <em>A. hypogaea</em>.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1546</span>–<span class="cit-lpage">1552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DFavero%26rft.auinit1%253DA.%2BP.%26rft.volume%253D46%26rft.issue%253D4%26rft.spage%253D1546%26rft.epage%253D1552%26rft.atitle%253DStudy%2Bof%2Bthe%2BEvolution%2Bof%2BCultivated%2BPeanut%2Bthrough%2BCrossability%2BStudies%2Bamong%2BArachis%2Bipaensis%252C%2BA.%2Bduranensis%252C%2Band%2BA.%2Bhypogaea%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.09-0331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.09-0331&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000239054100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernández</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krapovickas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Cromosomas y evolución en <em>Arachis</em> (Leguminosae).</span> <abbr class="cit-jnl-abbrev">Bonplandia</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBonplandia%26rft.volume%253D8%26rft.spage%253D187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.11" data-doi="10.1186/1471-2229-9-103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foncéka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hodo-Abalo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rivallan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faye</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sall</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic mapping of wild introgressions into cultivated peanut: a way toward enlarging the genetic basis of a recent allotetraploid.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DFonceka%26rft.auinit1%253DD.%26rft.volume%253D9%26rft.spage%253D103%26rft.epage%253D103%26rft.atitle%253DGenetic%2Bmapping%2Bof%2Bwild%2Bintrogressions%2Binto%2Bcultivated%2Bpeanut%253A%2Ba%2Bway%2Btoward%2Benlarging%2Bthe%2Bgenetic%2Bbasis%2Bof%2Ba%2Brecent%2Ballotetraploid.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-103%26rft_id%253Dinfo%253Apmid%252F19650911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19650911&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.12" data-doi="10.1371/journal.pone.0048642"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foncéka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tossim</span>  <span class="cit-name-given-names">H.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rivallan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vignes</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacut</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Construction of chromosome segment substitution lines in peanut (<em>Arachis hypogaea</em> L.) using a wild synthetic and QTL mapping for plant morphology.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e48642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFonceka%26rft.auinit1%253DD.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De48642%26rft.epage%253De48642%26rft.atitle%253DConstruction%2Bof%2BChromosome%2BSegment%2BSubstitution%2BLines%2Bin%2BPeanut%2B%2528Arachis%2Bhypogaea%2BL.%2529%2BUsing%2Ba%2BWild%2BSynthetic%2Band%2BQTL%2BMapping%2Bfor%2BPlant%2BMorphology.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0048642%26rft_id%253Dinfo%253Apmid%252F23185268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0048642&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23185268&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.13" data-doi="10.1371/journal.pone.0041213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gautami</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foncéka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sujay</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An international reference consensus genetic map with 897 marker loci based on 11 mapping populations for tetraploid groundnut (<em>Arachis hypogaea</em> L.)</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e41213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGautami%26rft.auinit1%253DB.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De41213%26rft.epage%253De41213%26rft.atitle%253DAn%2Binternational%2Breference%2Bconsensus%2Bgenetic%2Bmap%2Bwith%2B897%2Bmarker%2Bloci%2Bbased%2Bon%2B11%2Bmapping%2Bpopulations%2Bfor%2Btetraploid%2Bgroundnut%2B%2528Arachis%2Bhypogaea%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0041213%26rft_id%253Dinfo%253Apmid%252F22815973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0041213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22815973&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gawłowska</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Święcicki</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolko</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Comparison of genetic maps for two pea populations (<em>Pisum sativum</em> L.).</span> <abbr class="cit-jnl-abbrev">Pisum Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPisum%2BGenet.%26rft.volume%253D37%26rft.spage%253D19%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sederoff</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genetic linkage maps of <em>Eucalyptus grandis</em> and <em>Eucalyptus urophylla</em> using a pseudo-testcross: mapping strategy and RAPD markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1121</span>–<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGrattapaglia%26rft.auinit1%253DD.%26rft.volume%253D137%26rft.issue%253D4%26rft.spage%253D1121%26rft.epage%253D1137%26rft.atitle%253DGenetic%2Blinkage%2Bmaps%2Bof%2BEucalyptus%2Bgrandis%2Band%2BEucalyptus%2Burophylla%2Busing%2Ba%2Bpseudo-testcross%253A%2Bmapping%2Bstrategy%2Band%2BRAPD%2Bmarkers.%26rft_id%253Dinfo%253Apmid%252F7982566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNy80LzExMjEiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy80LzEvODkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.16" data-doi="10.1186/1471-2164-13-608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khanal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heesacker</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparative mapping in intraspecific populations uncovers a high degree of macrosynteny between A - and B - genome diploid species of peanut.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DGuo%26rft.auinit1%253DY.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D608%26rft.epage%253D608%26rft.atitle%253DComparative%2Bmapping%2Bin%2Bintraspecific%2Bpopulations%2Buncovers%2Ba%2Bhigh%2Bdegree%2Bof%2Bmacrosynteny%2Bbetween%2BA%2B-%2Band%2BB%2B-%2Bgenome%2Bdiploid%2Bspecies%2Bof%2Bpeanut.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-608%26rft_id%253Dinfo%253Apmid%252F23140574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23140574&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.17" data-doi="10.1139/g91-156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halward</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stalker</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larue</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kochert</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Genetic variation detectable with molecular markers among unadapted germplasm resources of cultivated peanut and related wild species.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1013</span>–<span class="cit-lpage">1020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft.volume%253D34%26rft.spage%253D1013%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg91-156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g91-156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.18" data-doi="10.1023/A:1002949813052"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Identification of polymorphic DNA markers in cultivated peanut (<em>Arachis hypogaea</em> L.).</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D97%26rft.spage%253D143%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1002949813052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1002949813052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XZ63600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.19" data-doi="10.2135/cropsci1999.0011183X003900040047x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hopkins</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Discovery and characterization of polymorphic simple sequence repeats (SSRs) in peanut.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1243</span>–<span class="cit-lpage">1247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHopkins%26rft.auinit1%253DM.%2BS.%26rft.volume%253D39%26rft.issue%253D4%26rft.spage%253D1243%26rft.epage%253D1247%26rft.atitle%253DDiscovery%2Band%2BCharacterization%2Bof%2BPolymorphic%2BSimple%2BSequence%2BRepeats%2B%2528SSRs%2529%2Bin%2BPeanut%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1999.0011183X003900040047x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1999.0011183X003900040047x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000081515700047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.20" data-doi="10.1007/BF00984735"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hilu</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stalker</span>  <span class="cit-name-given-names">H. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Genetic relationships between peanut and wild species of Arachis sect. Arachis (Fabaceae): Evidence from RAPDs.</span> <abbr class="cit-jnl-abbrev">Plant Syst. Evol.</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSyst.%2BEvol.%26rft.volume%253D198%26rft.spage%253D167%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00984735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00984735&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.21" data-doi="10.1508/cytologia.7.396"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Husted</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1936</span> <span class="cit-article-title">Cytological Studies on the Peanut, <em>Arachis</em>. II. Chromosome number, morphology and behavior, and their application to the problem of the cultivated forms.</span> <abbr class="cit-jnl-abbrev">Cytologia (Tokyo)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">396</span>–<span class="cit-lpage">423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytologia%2B%2528Tokyo%2529%26rft.volume%253D7%26rft.spage%253D396%26rft_id%253Dinfo%253Adoi%252F10.1508%252Fcytologia.7.396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1508/cytologia.7.396&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kochert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halward</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Branch</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">RFLP variability in peanut (Arachis hypogaea L.) cultivars and wild species.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D81%26rft.spage%253D565%26rft_id%253Dinfo%253Apmid%252F24221368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24221368&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FP48900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.23" data-doi="10.2307/2446112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kochert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stalker</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gimenes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galgaro</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">RFLP and cytogenetic evidence on the origin and evolution of allotetraploid domesticated peanut, <em>Arachis hypogaea</em> (Leguminosae).</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">1282</span>–<span class="cit-lpage">1291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft.volume%253D83%26rft.spage%253D1282%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2446112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2446112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996VN75500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.24" data-doi="10.1007/BF00037222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krishna</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitra</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">The probable genome donors to <em>Arachis hypogaea</em> L. based on arachin seed storage protein.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D37%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00037222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00037222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1988M172400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.25" data-doi="10.1016/0888-7543(87)90010-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abrahamson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barlow</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">MAPMAKER: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations.</span> <abbr class="cit-jnl-abbrev">Genomics.</abbr> <span class="cit-vol">1</span>:<span class="cit-fpage">174</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253D174%26rft.epage%253D181%26rft.atitle%253DMAPMAKER%253A%2Ban%2Binteractive%2Bcomputer%2Bpackage%2Bfor%2Bconstructing%2Bprimary%2Bgenetic%2Blinkage%2Bmaps%2Bof%2Bexperimental%2Band%2Bnatural%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0888-7543%252887%252990010-3%26rft_id%253Dinfo%253Apmid%252F3692487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0888-7543(87)90010-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3692487&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.26" data-doi="10.1186/1471-2229-9-112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leal-Bertioli</span>  <span class="cit-name-given-names">S. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">José</span>  <span class="cit-name-given-names">A. C. F. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alves-Freitas</span>  <span class="cit-name-given-names">D. M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guimarães</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identification of candidate genome regions controlling disease resistance in <em>Arachis</em>.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DLeal-Bertioli%26rft.auinit1%253DS.%2BC.%26rft.volume%253D9%26rft.spage%253D112%26rft.epage%253D112%26rft.atitle%253DIdentification%2Bof%2Bcandidate%2Bgenome%2Bregions%2Bcontrolling%2Bdisease%2Bresistance%2Bin%2BArachis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-112%26rft_id%253Dinfo%253Apmid%252F19698131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19698131&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-other" id="cit-4.1.89.27"><div class="cit-metadata unstructured">Lincoln, S., M. Daly, and E. Lander, 1992 Constructing genetic maps with MAPMAKER/EXP 3.0. Whitehead Institute Technical Report. 3rd edition. Whitehead Institute, Cambridge, MA.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pickersgill</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Isozyme variation and species relationships in peanut and its wild relatives (<em>Arachis</em> L. - Leguminosae).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">550</span>–<span class="cit-lpage">560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D85%26rft.spage%253D550%26rft_id%253Dinfo%253Apmid%252F24195929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24195929&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KH04900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.29" data-doi="10.1186/1756-0500-5-86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Macedo</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leal-Bertioli</span>  <span class="cit-name-given-names">S. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alves</span>  <span class="cit-name-given-names">D. M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gouvea</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development and characterization of highly polymorphic long TC repeat microsatellite markers for genetic analysis of peanut.</span> <abbr class="cit-jnl-abbrev">BMC Res. Notes</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BRes%2BNotes%26rft.aulast%253DMacedo%26rft.auinit1%253DS.%2BE.%26rft.volume%253D5%26rft.spage%253D86%26rft.epage%253D86%26rft.atitle%253DDevelopment%2Band%2Bcharacterization%2Bof%2Bhighly%2Bpolymorphic%2Blong%2BTC%2Brepeat%2Bmicrosatellite%2Bmarkers%2Bfor%2Bgenetic%2Banalysis%2Bof%2Bpeanut.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-0500-5-86%26rft_id%253Dinfo%253Apmid%252F22305491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-0500-5-86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22305491&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.30" data-doi="10.1007/s00122-005-0028-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leoi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proite</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guimarães</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leal- Bertioli</span>  <span class="cit-name-given-names">S. C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Microsatellite based, gene-rich linkage map for the AA genome of <em>Arachis</em> (Fabaceae).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">1060</span>–<span class="cit-lpage">1071</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMoretzsohn%26rft.auinit1%253DM.%2BC.%26rft.volume%253D111%26rft.issue%253D6%26rft.spage%253D1060%26rft.epage%253D1071%26rft.atitle%253DA%2Bmicrosatellite-based%252C%2Bgene-rich%2Blinkage%2Bmap%2Bfor%2Bthe%2BAA%2Bgenome%2Bof%2BArachis%2B%2528Fabaceae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0028-x%26rft_id%253Dinfo%253Apmid%252F16088397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0028-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16088397&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232500000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.31" data-doi="10.1186/1471-2229-9-40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gobbi Barbosa</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alves-Freitas</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teixeira</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leal-Bertioli</span>  <span class="cit-name-given-names">S. C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A linkage map for the B-genome of <em>Arachis</em> (Fabaceae) and its synteny to the A-genome.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DMoretzsohn%26rft.auinit1%253DM.%2BC.%26rft.volume%253D9%26rft.spage%253D40%26rft.epage%253D40%26rft.atitle%253DA%2Blinkage%2Bmap%2Bfor%2Bthe%2BB-genome%2Bof%2BArachis%2B%2528Fabaceae%2529%2Band%2Bits%2Bsynteny%2Bto%2Bthe%2BA-genome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-40%26rft_id%253Dinfo%253Apmid%252F19351409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-40&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19351409&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.32" data-doi="10.1093/aob/mcs237"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gouvea</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inglis</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leal-Bertioli</span>  <span class="cit-name-given-names">S. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valls</span>  <span class="cit-name-given-names">J. F. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A study of the relationships of cultivated peanut (<em>Arachis hypogaea</em>) and its most closely related wild species using intron sequences and microsatellite markers.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcs237%26rft_id%253Dinfo%253Apmid%252F23131301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjk6IjExMS8xLzExMyI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzQvMS84OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.33" data-doi="10.1186/1471-2164-13-469"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagy</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okashah</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high-density genetic map of Arachis duranensis, a diploid ancestor of cultivated peanut.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DNagy%26rft.auinit1%253DE.%2BD.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D469%26rft.epage%253D469%26rft.atitle%253DA%2Bhigh-density%2Bgenetic%2Bmap%2Bof%2BArachis%2Bduranensis%252C%2Ba%2Bdiploid%2Bancestor%2Bof%2Bcultivated%2Bpeanut.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-469%26rft_id%253Dinfo%253Apmid%252F22967170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-469&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22967170&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.34" data-doi="10.1007/s00438-011-0656-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidigal</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leal-Bertioli</span>  <span class="cit-name-given-names">S. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ratnaparkhe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Matita, a new retroelement from peanut: characterization and evolutionary context in the light of the <em>Arachis</em> A–B genome divergence.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DNielen%26rft.auinit1%253DS.%26rft.volume%253D287%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D38%26rft.atitle%253DMatita%252C%2Ba%2Bnew%2Bretroelement%2Bfrom%2Bpeanut%253A%2Bcharacterization%2Band%2Bevolutionary%2Bcontext%2Bin%2Bthe%2Blight%2Bof%2Bthe%2BArachis%2BA-B%2Bgenome%2Bdivergence.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-011-0656-6%26rft_id%253Dinfo%253Apmid%252F22120641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-011-0656-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22120641&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliphant</span>  <span class="cit-name-given-names">A</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">DL</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stuelpnagel</span>  <span class="cit-name-given-names">JR</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chee</span>  <span class="cit-name-given-names">MS</span></span></li></ol><cite>, BeadArray Technology. <span class="cit-pub-date">2002</span> Enabling an accurate, cost-effective approach to high-throughput genotyping. Biotechniques Suppl: 56–58, 60–61.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.36" data-doi="10.1007/s00122-010-1369-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robledo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seijo</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Species relationships among the wild B genome of <em>Arachis</em> species (section <em>Arachis</em>) based on FISH mapping of rDNA loci and heterochromatin detection: a new proposal for genome arrangement.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">1033</span>–<span class="cit-lpage">1046</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DRobledo%26rft.auinit1%253DG.%26rft.volume%253D121%26rft.issue%253D6%26rft.spage%253D1033%26rft.epage%253D1046%26rft.atitle%253DSpecies%2Brelationships%2Bamong%2Bthe%2Bwild%2BB%2Bgenome%2Bof%2BArachis%2Bspecies%2B%2528section%2BArachis%2529%2Bbased%2Bon%2BFISH%2Bmapping%2Bof%2BrDNA%2Bloci%2Band%2Bheterochromatin%2Bdetection%253A%2Ba%2Bnew%2Bproposal%2Bfor%2Bgenome%2Barrangement.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1369-7%26rft_id%253Dinfo%253Apmid%252F20552326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1369-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20552326&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.37" data-doi="10.3732/ajb.94.12.1963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seijo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavia</span>  <span class="cit-name-given-names">G. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krapovickas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ducasse</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic relationships between the cultivated peanut (<em>Arachis hypogaea</em>, Leguminosae) and its close relatives revealed by double GISH.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">1963</span>–<span class="cit-lpage">1971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.stitle%253DAm.%2BJ.%2BBot.%26rft.aulast%253DSeijo%26rft.auinit1%253DG.%26rft.volume%253D94%26rft.issue%253D12%26rft.spage%253D1963%26rft.epage%253D1971%26rft.atitle%253DGenomic%2Brelationships%2Bbetween%2Bthe%2Bcultivated%2Bpeanut%2B%2528Arachis%2Bhypogaea%252C%2BLeguminosae%2529%2Band%2Bits%2Bclose%2Brelatives%2Brevealed%2Bby%2Bdouble%2BGISH%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.94.12.1963%26rft_id%253Dinfo%253Apmid%252F21636391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjEwOiI5NC8xMi8xOTYzIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.38" data-doi="10.1093/molbev/msg065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Senchina</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cronn</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Rate variation among nuclear genes and the age of polyploidy in Gossypium.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">633</span>–<span class="cit-lpage">643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg065%26rft_id%253Dinfo%253Apmid%252F12679546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIwLzQvNjMzIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sewell</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">A consensus map for loblolly pine (<em>Pinus taeda</em> L.). I. Construction and integration of individual linkage maps from two outbred three-generation pedigrees.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSewell%26rft.auinit1%253DM.%2BM.%26rft.volume%253D151%26rft.issue%253D1%26rft.spage%253D321%26rft.epage%253D330%26rft.atitle%253DA%2BConsensus%2BMap%2Bfor%2BLoblolly%2BPine%2B%2528Pinus%2Btaeda%2BL.%2529.%2BI.%2BConstruction%2Band%2BIntegration%2Bof%2BIndividual%2BLinkage%2BMaps%2BFrom%2BTwo%2BOutbred%2BThree-Generation%2BPedigrees%26rft_id%253Dinfo%253Apmid%252F9872970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUxLzEvMzIxIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.40" data-doi="10.1093/dnares/dss042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shirasawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertioli</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leal-Bertioli</span>  <span class="cit-name-given-names">S. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An integrated consensus map of cultivated peanut and wild relatives reveals structures of the A and B genomes of <em>Arachis</em> and divergences with other legume genomes.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdss042%26rft_id%253Dinfo%253Apmid%252F23315685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjIwLzIvMTczIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.41" data-doi="10.1038/nature08670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmutz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlueter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitros</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome sequence of the palaeopolyploid soybean.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">463</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchmutz%26rft.auinit1%253DJ.%26rft.volume%253D463%26rft.issue%253D7278%26rft.spage%253D178%26rft.epage%253D183%26rft.atitle%253DGenome%2Bsequence%2Bof%2Bthe%2Bpalaeopolyploid%2Bsoybean.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08670%26rft_id%253Dinfo%253Apmid%252F20075913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20075913&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273582700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-book" id="cit-4.1.89.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smartt</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">The groundnut, <em>Arachis hypogaea</em> L</span>, pp. <span class="cit-fpage">30</span>–<span class="cit-lpage">84</span> in <span class="cit-source">Grain Legumes: Evolution and Genetic Resources</span>, edited by<span class="cit-ed"><span class="cit-name-surname">Smartt</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.43" data-doi="10.1007/s00122-005-0017-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tallury</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hilu</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milla</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friend</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alsaghir</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genomic affinities in <em>Arachis</em> section <em>Arachis</em> (Fabaceae): molecular and cytogenetic evidence.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">1229</span>–<span class="cit-lpage">1237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DTallury%26rft.auinit1%253DS.%2BP.%26rft.volume%253D111%26rft.issue%253D7%26rft.spage%253D1229%26rft.epage%253D1237%26rft.atitle%253DGenomic%2Baffinities%2Bin%2BArachis%2Bsection%2BArachis%2B%2528Fabaceae%2529%253A%2Bmolecular%2Band%2Bcytogenetic%2Bevidence.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0017-0%26rft_id%253Dinfo%253Apmid%252F16187123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0017-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16187123&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tani</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mukai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ujino-Ihara</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A consensus linkage map for sugi (<em>Cryptomeria japonica</em>) from two pedigrees, based on microsatellites and expressed sequence tags.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">1551</span>–<span class="cit-lpage">1568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTani%26rft.auinit1%253DN.%26rft.volume%253D165%26rft.issue%253D3%26rft.spage%253D1551%26rft.epage%253D1568%26rft.atitle%253DA%2BConsensus%2BLinkage%2BMap%2Bfor%2BSugi%2B%2528Cryptomeria%2Bjaponica%2529%2BFrom%2BTwo%2BPedigrees%252C%2BBased%2Bon%2BMicrosatellites%2Band%2BExpressed%2BSequence%2BTags%26rft_id%253Dinfo%253Apmid%252F14668402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS8zLzE1NTEiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy80LzEvODkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.45" data-doi="10.1371/journal.pone.0003882"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van’t Hof</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marec</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saccheri</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brakefield</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zwaan</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Cytogenetic characterization and AFLP-based genetic linkage mapping for the butterfly bicyclus anynana, covering all 28 karyotyped chromosomes.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3882</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DVan%2527t%2BHof%26rft.auinit1%253DA.%2BE.%26rft.volume%253D3%26rft.issue%253D12%26rft.spage%253De3882%26rft.epage%253De3882%26rft.atitle%253DCytogenetic%2Bcharacterization%2Band%2BAFLP-based%2Bgenetic%2Blinkage%2Bmapping%2Bfor%2Bthe%2Bbutterfly%2BBicyclus%2Banynana%252C%2Bcovering%2Ball%2B28%2Bkaryotyped%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003882%26rft_id%253Dinfo%253Apmid%252F19060955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003882&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19060955&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.46" data-doi="10.1186/1471-2105-12-172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gort</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vosman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype calling in tetraploid species from bi-allelic marker data using mixture models.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D12%26rft.spage%253D172%26rft.epage%253D172%26rft.atitle%253DGenotype%2Bcalling%2Bin%2Btetraploid%2Bspecies%2Bfrom%2Bbi-allelic%2Bmarker%2Bdata%2Busing%2Bmixture%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-172%26rft_id%253Dinfo%253Apmid%252F21595880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21595880&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F89.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.89.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chee</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Application of advanced backcross quantitative trait locus (QTL) analysis in crop improvement.</span> <abbr class="cit-jnl-abbrev">J. Plant Breeding Crop Sci.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPlant%2BBreeding%2BCrop%2BSci.%26rft.volume%253D2%26rft.spage%253D221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/1/89.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/1/79" title="Genetic Variation and Covariation in Male Attractiveness and Female Mating Preferences in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/1/97" title="An Alternative to the Breeder’s and Lande’s Equations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/1"><span>Volume 4 Issue 1, January 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/1.cover-source.jpg?itok=oIsy65sY" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/1/89" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3O_E74ZVyaOLwwZ8VyoOeariqpppZSnxmRDn1CQsbwA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/1/89" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/1/89" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/1/89" class="active" data-icon-position="" data-hide-link-title="0">The Use of SNP Markers for Linkage Mapping in Diploid and Tetraploid Peanuts</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459619" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-nPdHDSbvkXtpXm8GzJw1U_dUoV_jyYyVT7nE1N35BGw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/1/89.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/1/89" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-77O7ySUkHHz6cQEyTeep7JgasXPkqLEF6f88_kUFQys" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/1/89" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/1/89.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459619" id="node459619--2" data-pisa="ggg;4/1/89" data-pisa-master="ggg;g3.113.007617" data-apath="/ggg/4/1/89.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Use of SNP Markers for Linkage Mapping in Diploid and Tetraploid Peanuts</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">David J. Bertioli</span>, <span class="highwire-citation-author" data-delta="1">Peggy Ozias-Akins</span>, <span class="highwire-citation-author" data-delta="2">Ye Chu</span>, <span class="highwire-citation-author" data-delta="3">Karinne M. Dantas</span>, <span class="highwire-citation-author" data-delta="4">Silvio P. Santos</span>, <span class="highwire-citation-author" data-delta="5">Ediene Gouvea</span>, <span class="highwire-citation-author" data-delta="6">Patricia M. Guimarães</span>, <span class="highwire-citation-author" data-delta="7">Soraya C. M. Leal-Bertioli</span>, <span class="highwire-citation-author" data-delta="8">Steven J. Knapp</span> and <span class="highwire-citation-author" data-delta="9">Marcio C. Moretzsohn</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">89-96;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007617 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459619--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David J. Bertioli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departamento de Genética/IB, Universidade de Brasília, Brasília, DF, 70910-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BBertioli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bertioli%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BJ.%2BBertioli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:davidbertioli@unb.br" class="" data-icon-position="" data-hide-link-title="0">davidbertioli@unb.br</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peggy Ozias-Akins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticulture, University of Georgia, Tifton, Georgia 31973</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peggy%2BOzias-Akins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ozias-Akins%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeggy%2BOzias-Akins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ye Chu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticulture, University of Georgia, Tifton, Georgia 31973</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ye%2BChu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYe%2BChu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Karinne M. Dantas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karinne%2BM.%2BDantas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dantas%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarinne%2BM.%2BDantas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Silvio P. Santos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Universidade Católica de Brasília, Brasília, DF, 70790-160, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Silvio%2BP.%2BSantos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santos%20SP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASilvio%2BP.%2BSantos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ediene Gouvea </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ediene%2BGouvea%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gouvea%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdiene%2BGouvea%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Patricia M. Guimarães </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BM.%2BGuimar%C3%A3es%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guimar%C3%A3es%20PM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatricia%2BM.%2BGuimar%25C3%25A3es%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Soraya C. M. Leal-Bertioli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Soraya%2BC.%2BM.%2BLeal-Bertioli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leal-Bertioli%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASoraya%2BC.%2BM.%2BLeal-Bertioli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Steven J. Knapp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Plant Breeding, Genetics, and Genomics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BJ.%2BKnapp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Knapp%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteven%2BJ.%2BKnapp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Marcio C. Moretzsohn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcio%2BC.%2BMoretzsohn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moretzsohn%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcio%2BC.%2BMoretzsohn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/1/89" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/1/89&amp;title=The%20Use%20of%20SNP%20Markers%20for%20Linkage%20Mapping%20in%20Diploid%20and%20Tetraploid%20Peanuts" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/1/89&amp;title=The%20Use%20of%20SNP%20Markers%20for%20Linkage%20Mapping%20in%20Diploid%20and%20Tetraploid%20Peanuts" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/1/89&amp;title=The%20Use%20of%20SNP%20Markers%20for%20Linkage%20Mapping%20in%20Diploid%20and%20Tetraploid%20Peanuts" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/1/89&amp;text=The%20Use%20of%20SNP%20Markers%20for%20Linkage%20Mapping%20in%20Diploid%20and%20Tetraploid%20Peanuts" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/1/89&amp;title=The%20Use%20of%20SNP%20Markers%20for%20Linkage%20Mapping%20in%20Diploid%20and%20Tetraploid%20Peanuts" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/1/89&amp;t=The%20Use%20of%20SNP%20Markers%20for%20Linkage%20Mapping%20in%20Diploid%20and%20Tetraploid%20Peanuts" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/1/89&amp;title=The%20Use%20of%20SNP%20Markers%20for%20Linkage%20Mapping%20in%20Diploid%20and%20Tetraploid%20Peanuts" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/1/89&amp;title=The%20Use%20of%20SNP%20Markers%20for%20Linkage%20Mapping%20in%20Diploid%20and%20Tetraploid%20Peanuts" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459619/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459619" id="node459619--3" data-pisa="ggg;4/1/89" data-pisa-master="ggg;g3.113.007617" data-apath="/ggg/4/1/89.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Use of SNP Markers for Linkage Mapping in Diploid and Tetraploid Peanuts</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">David J. Bertioli</span>, <span class="highwire-citation-author" data-delta="1">Peggy Ozias-Akins</span>, <span class="highwire-citation-author" data-delta="2">Ye Chu</span>, <span class="highwire-citation-author" data-delta="3">Karinne M. Dantas</span>, <span class="highwire-citation-author" data-delta="4">Silvio P. Santos</span>, <span class="highwire-citation-author" data-delta="5">Ediene Gouvea</span>, <span class="highwire-citation-author" data-delta="6">Patricia M. Guimarães</span>, <span class="highwire-citation-author" data-delta="7">Soraya C. M. Leal-Bertioli</span>, <span class="highwire-citation-author" data-delta="8">Steven J. Knapp</span> and <span class="highwire-citation-author" data-delta="9">Marcio C. Moretzsohn</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">89-96;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007617 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459619--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David J. Bertioli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departamento de Genética/IB, Universidade de Brasília, Brasília, DF, 70910-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BBertioli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bertioli%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BJ.%2BBertioli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:davidbertioli@unb.br" class="" data-icon-position="" data-hide-link-title="0">davidbertioli@unb.br</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peggy Ozias-Akins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticulture, University of Georgia, Tifton, Georgia 31973</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peggy%2BOzias-Akins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ozias-Akins%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeggy%2BOzias-Akins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ye Chu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticulture, University of Georgia, Tifton, Georgia 31973</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ye%2BChu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYe%2BChu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Karinne M. Dantas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karinne%2BM.%2BDantas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dantas%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarinne%2BM.%2BDantas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Silvio P. Santos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Universidade Católica de Brasília, Brasília, DF, 70790-160, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Silvio%2BP.%2BSantos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santos%20SP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASilvio%2BP.%2BSantos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ediene Gouvea </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ediene%2BGouvea%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gouvea%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdiene%2BGouvea%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Patricia M. Guimarães </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BM.%2BGuimar%C3%A3es%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guimar%C3%A3es%20PM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatricia%2BM.%2BGuimar%25C3%25A3es%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Soraya C. M. Leal-Bertioli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Soraya%2BC.%2BM.%2BLeal-Bertioli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leal-Bertioli%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASoraya%2BC.%2BM.%2BLeal-Bertioli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Steven J. Knapp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Plant Breeding, Genetics, and Genomics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BJ.%2BKnapp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Knapp%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteven%2BJ.%2BKnapp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Marcio C. Moretzsohn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Embrapa Recursos Genéticos e Biotecnologia, Brasília, DF, 70770-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcio%2BC.%2BMoretzsohn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moretzsohn%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcio%2BC.%2BMoretzsohn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=89&amp;rft.epage=96&amp;rft.atitle=The%20Use%20of%20SNP%20Markers%20for%20Linkage%20Mapping%20in%20Diploid%20and%20Tetraploid%20Peanuts&amp;rft.volume=4&amp;rft.issue=1&amp;rft.date=2014-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Bertioli%2C+David+J.&amp;rft.au=Ozias-Akins%2C+Peggy&amp;rft.au=Chu%2C+Ye&amp;rft.au=Dantas%2C+Karinne+M.&amp;rft.au=Santos%2C+Silvio+P.&amp;rft.au=Gouvea%2C+Ediene&amp;rft.au=Guimar%C3%A3es%2C+Patricia+M.&amp;rft.au=Leal-Bertioli%2C+Soraya+C.+M.&amp;rft.au=Knapp%2C+Steven+J.&amp;rft.au=Moretzsohn%2C+Marcio+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459619/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459619/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459619/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459619/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459619/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459619/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459619/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459619/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459619/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459619/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459619/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459619/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/1/89" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/1/89.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/1/89.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/1/89.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/1/89.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
