<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/1/29" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/1/29.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/1/29.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/1/29.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/1/29/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/1/29.atom" />
<meta name="HW.pisa" content="ggg;4/1/29" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A High-Density Gene Map of Loblolly Pine (Pinus taeda L.) Based on Exome Sequence Capture Genotyping" />
<meta name="DC.Identifier" content="10.1534/g3.113.008714" />
<meta name="DC.Date" content="2014-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Neves et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Loblolly pine ( Pinus taeda L.) is an economically and ecologically important conifer for which a suite of genomic resources is being generated. Despite recent attempts to sequence the large genome of conifers, their assembly and the positioning of genes remains largely incomplete. The interspecific synteny in pines suggests that a gene-based map would be useful to support genome assemblies and analysis of conifers. To establish a reference gene-based genetic map, we performed exome sequencing of 14729 genes on a mapping population of 72 haploid samples, generating a resource of 7434 sequence variants segregating for 3787 genes. Most markers are single-nucleotide polymorphisms, although short insertions/deletions and multiple nucleotide polymorphisms also were used. Marker segregation in the population was used to generate a high-density, gene-based genetic map. A total of 2841 genes were mapped to pine’s 12 linkage groups with an average of one marker every 0.58 cM. Capture data were used to detect gene presence/absence variations and position 65 genes on the map. We compared the marker order of genes previously mapped in loblolly pine and found high agreement. We estimated that 4123 genes had enough sequencing depth for reliable detection of markers, suggesting a high marker conversation rate of 92% (3787/4123). This is possible because a significant portion of the gene is captured and sequenced, increasing the chances of identifying a polymorphic site for characterization and mapping. This sub-centiMorgan genetic map provides a valuable resource for gene positioning on chromosomes and guide for the assembly of a reference pine genome." />
<meta name="DC.Contributor" content="Leandro Gomide Neves" />
<meta name="DC.Contributor" content="John M. Davis" />
<meta name="DC.Contributor" content="William B. Barbazuk" />
<meta name="DC.Contributor" content="Matias Kirst" />
<meta name="article:published_time" content="2014-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A High-Density Gene Map of Loblolly Pine (Pinus taeda L.) Based on Exome Sequence Capture Genotyping" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Loblolly pine (&lt;i&gt;Pinus taeda&lt;/i&gt; L.) is an economically and ecologically important conifer for which a suite of genomic resources is being generated. Despite recent attempts to sequence the large genome of conifers, their assembly and the positioning of genes remains largely incomplete. The interspecific synteny in pines suggests that a gene-based map would be useful to support genome assemblies and analysis of conifers. To establish a reference gene-based genetic map, we performed exome sequencing of 14729 genes on a mapping population of 72 haploid samples, generating a resource of 7434 sequence variants segregating for 3787 genes. Most markers are single-nucleotide polymorphisms, although short insertions/deletions and multiple nucleotide polymorphisms also were used. Marker segregation in the population was used to generate a high-density, gene-based genetic map. A total of 2841 genes were mapped to pine’s 12 linkage groups with an average of one marker every 0.58 cM. Capture data were used to detect gene presence/absence variations and position 65 genes on the map. We compared the marker order of genes previously mapped in loblolly pine and found high agreement. We estimated that 4123 genes had enough sequencing depth for reliable detection of markers, suggesting a high marker conversation rate of 92% (3787/4123). This is possible because a significant portion of the gene is captured and sequenced, increasing the chances of identifying a polymorphic site for characterization and mapping. This sub-centiMorgan genetic map provides a valuable resource for gene positioning on chromosomes and guide for the assembly of a reference pine genome.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/01/01" />
<meta name="citation_mjid" content="ggg;4/1/29" />
<meta name="citation_id" content="4/1/29" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/1/29" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/1/29.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/1/29.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/1/29.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.008714" />
<meta name="citation_pmid" content="24192835" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="29" />
<meta name="citation_lastpage" content="37" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Leandro Gomide Neves" />
<meta name="citation_author_institution" content="Graduate Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, Florida 32611" />
<meta name="citation_author" content="John M. Davis" />
<meta name="citation_author_institution" content="Graduate Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, Florida 32611" />
<meta name="citation_author_institution" content="School of Forest Resources and Conservation, University of Florida, Gainesville, Florida 32611" />
<meta name="citation_author_institution" content="University of Florida Genetics Institute, University of Florida, Gainesville, Florida 32611" />
<meta name="citation_author" content="William B. Barbazuk" />
<meta name="citation_author_institution" content="Graduate Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, Florida 32611" />
<meta name="citation_author_institution" content="University of Florida Genetics Institute, University of Florida, Gainesville, Florida 32611" />
<meta name="citation_author_institution" content="Department of Biology, University of Florida, Gainesville, Florida 32611" />
<meta name="citation_author_email" content="bbarbazuk@ufl.edu" />
<meta name="citation_author_email" content="mkirst@ufl.edu" />
<meta name="citation_author" content="Matias Kirst" />
<meta name="citation_author_institution" content="Graduate Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, Florida 32611" />
<meta name="citation_author_institution" content="School of Forest Resources and Conservation, University of Florida, Gainesville, Florida 32611" />
<meta name="citation_author_institution" content="Department of Biology, University of Florida, Gainesville, Florida 32611" />
<meta name="citation_author_email" content="bbarbazuk@ufl.edu" />
<meta name="citation_author_email" content="mkirst@ufl.edu" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=W. T. Adams;citation_author=R. J. Joly;citation_title=Genetics of allozyme variants in loblolly pine;citation_pages=33-40;citation_volume=71;citation_year=1980;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Barnett;citation_author=E. Garrison;citation_author=A. Quinlan;citation_author=M. Stromberg;citation_author=G. Marth;citation_title=BamTools: a C++ API and toolkit for analyzing and managing BAM files.;citation_volume=27;citation_year=2011;citation_pmid=21493652;citation_doi=10.1093/bioinformatics/btr174" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=A. Belo;citation_author=M. K. Beatty;citation_author=D. Hondred;citation_author=K. A. Fengler;citation_author=B. Li;citation_title=Allelic genome structural variations in maize detected by array comparative genome hybridization.;citation_pages=355-367;citation_volume=120;citation_year=2010;citation_issue=2;citation_pmid=19756477;citation_doi=10.1007/s00122-009-1128-9" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_title=Molecular Markers and Selection for Complex Traits in Plants: Learning from the Last 20 Years;citation_pages=1649-1664;citation_volume=48;citation_year=2008;citation_issue=5;citation_doi=10.2135/cropsci2008.03.0131" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. R. Brown;citation_author=G. P. Gill;citation_author=R. J. Kuntz;citation_author=C. H. Langley;citation_author=D. B. Neale;citation_title=Nucleotide diversity and linkage disequilibrium in loblolly pine.;citation_volume=101;citation_year=2004;citation_pmid=15477602;citation_doi=10.1073/pnas.0404231101" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=P. C. Bundock;citation_author=R. E. Casu;citation_author=R. J. Henry;citation_title=Enrichment of genomic DNA for polymorphism detection in a non-model highly polyploid crop plant.;citation_pages=657-667;citation_volume=10;citation_year=2012;citation_issue=6;citation_pmid=22624722;citation_doi=10.1111/j.1467-7652.2012.00707.x" />
<meta name="citation_reference" content="citation_journal_title=J. Bot.;citation_author=J. G. Burleigh;citation_author=W. B. Barbazuk;citation_author=J. M. Davis;citation_author=A. M. Morse;citation_author=P. S. Soltis;citation_title=Exploring diversification and genome size evolution in extant gymnosperms through phylogenetic synthesis.;citation_pages=292857;citation_volume=2012;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Chakravarti;citation_author=L. K. Lasher;citation_author=J. E. Reefer;citation_title=A maximum likelihood method for estimating genome length using genetic linkage data.;citation_pages=175-182;citation_volume=128;citation_year=1991;citation_issue=1;citation_pmid=2060775" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. W. Davey;citation_author=P. A. Hohenlohe;citation_author=P. D. Etter;citation_author=J. Q. Boone;citation_author=J. M. Catchen;citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing.;citation_pages=499-510;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21681211;citation_doi=10.1038/nrg3012" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=D. R. Drost;citation_author=E. Novaes;citation_author=C. Boaventura-Novaes;citation_author=C. I. Benedict;citation_author=R. S. Brown;citation_title=A microarray-based genotyping and genetic mapping approach for highly heterozygous outcrossing species enables localization of a large fraction of the unassembled Populus trichocarpa genome sequence.;citation_pages=1054-1067;citation_volume=12;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=C. S. Echt;citation_author=S. Saha;citation_author=K. V. Krutovsky;citation_author=K. Wimalanathan;citation_author=J. E. Erpelding;citation_title=An annotated genetic map of loblolly pine based on microsatellite and cDNA markers.;citation_pages=17-17;citation_volume=12;citation_year=2011;citation_pmid=21269494;citation_doi=10.1186/1471-2156-12-17" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=A. J. Eckert;citation_author=B. Pande;citation_author=E. S. Ersoz;citation_author=M. H. Wright;citation_author=V. K. Rashbrook;citation_title=High-throughput genotyping and mapping of single nucleotide polymorphisms in loblolly pine (Pinus taeda L.).;citation_pages=225-234;citation_volume=5;citation_year=2009;citation_doi=10.1007/s11295-008-0183-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. J. Eckert;citation_author=J. van Heerwaarden;citation_author=J. L. Wegrzyn;citation_author=C. D. Nelson;citation_author=J. Ross-Ibarra;citation_title=Patterns of population structure and environmental associations to aridity across the range of loblolly pine (Pinus taeda L., Pinaceae).;citation_volume=185;citation_year=2010;citation_pmid=20439779;citation_doi=10.1534/genetics.110.115543" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. G. Elsik;citation_author=C. G. Williams;citation_title=Low-copy microsatellite recovery from a conifer genome.;citation_pages=1189-1195;citation_volume=103;citation_year=2001;citation_doi=10.1007/s001220100725" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=A. Gnirke;citation_author=A. Melnikov;citation_author=J. Maguire;citation_author=P. Rogov;citation_author=E. M. LeProust;citation_title=Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing.;citation_pages=182-189;citation_volume=27;citation_year=2009;citation_issue=2;citation_pmid=19182786;citation_doi=10.1038/nbt.1523" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. C. Gonzalez-Martinez;citation_author=N. C. Wheeler;citation_author=E. Ersoz;citation_author=C. D. Nelson;citation_author=D. B. Neale;citation_title=Association genetics in Pinus taeda L. I. Wood property traits.;citation_volume=175;citation_year=2007;citation_pmid=17110498;citation_doi=10.1534/genetics.106.061127" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=D. Grattapaglia;citation_author=O. B. Silva-Junior;citation_author=M. Kirst;citation_author=B. M. de Lima;citation_author=D. A. Faria;citation_title=High-throughput SNP genotyping in the highly heterozygous genome of Eucalyptus: assay success, polymorphism and transferability across species.;citation_pages=65-65;citation_volume=11;citation_year=2011;citation_pmid=21492434;citation_doi=10.1186/1471-2229-11-65" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. A. Hackett;citation_author=L. B. Broadfoot;citation_title=Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.;citation_pages=33-38;citation_volume=90;citation_year=2003;citation_issue=1;citation_pmid=12522423;citation_doi=10.1038/sj.hdy.6800173" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=G. C. Kayihan;citation_author=D. A. Huber;citation_author=A. M. Morse;citation_author=T. L. White;citation_author=J. M. Davis;citation_title=Genetic dissection of fusiform rust and pitch canker disease traits in loblolly pine.;citation_pages=948-958;citation_volume=110;citation_year=2005;citation_issue=5;citation_pmid=15700146;citation_doi=10.1007/s00122-004-1915-2" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. Kovach;citation_author=J. L. Wegrzyn;citation_author=G. Parra;citation_author=C. Holt;citation_author=G. E. Bruening;citation_title=The Pinus taeda genome is characterized by diverse and highly diverged repetitive sequences.;citation_pages=420-420;citation_volume=11;citation_year=2010;citation_pmid=20609256;citation_doi=10.1186/1471-2164-11-420" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. V. Krutovsky;citation_author=M. Troggio;citation_author=G. R. Brown;citation_author=K. D. Jermstad;citation_author=D. B. Neale;citation_title=Comparative Mapping in the Pinaceae;citation_pages=447-461;citation_volume=168;citation_year=2004;citation_issue=1;citation_pmid=15454556;citation_doi=10.1534/genetics.104.028381" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=K. Lange;citation_author=M. Boehnke;citation_title=How many polymorphic genes will it take to span the human genome?;citation_pages=842-845;citation_volume=34;citation_year=1982;citation_issue=6;citation_pmid=6960692" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=J. Mackay;citation_author=J. F. Dean;citation_author=C. Plomion;citation_author=D. G. Peterson;citation_author=F. M. Canovas;citation_title=Towards decoding the conifer giga-genome.;citation_pages=555-569;citation_volume=80;citation_year=2012;citation_issue=6;citation_pmid=22960864;citation_doi=10.1007/s11103-012-9961-7" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. G. Maron;citation_author=C. T. Guimaraes;citation_author=M. Kirst;citation_author=P. S. Albert;citation_author=J. A. Birchler;citation_title=Aluminum tolerance in maize is associated with higher MATE1 gene copy number.;citation_volume=110;citation_year=2013;citation_pmid=23479633;citation_doi=10.1073/pnas.1220766110" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_title=Combination of multipoint maximum likelihood (MML) and regression mapping algorithms to construct a high-density genetic linkage map for loblolly pine (Pinus taeda L.).;citation_author=P. Martinez-Garcia;citation_author=K. Stevens;citation_author=J. Wegrzyn;citation_author=J. Liechty;citation_author=M. Crepeau;citation_pages=1-7;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=L. G. Neves;citation_author=E. M. Mamani;citation_author=A. C. Alfenas;citation_author=M. Kirst;citation_author=D. Grattapaglia;citation_title=A high-density transcript linkage map with 1,845 expressed genes positioned by microarray-based Single Feature Polymorphisms (SFP) in Eucalyptus.;citation_pages=189-189;citation_volume=12;citation_year=2011;citation_pmid=21492453;citation_doi=10.1186/1471-2164-12-189" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=L. G. Neves;citation_author=J. M. Davis;citation_author=W. B. Barbazuk;citation_author=M. Kirst;citation_title=Whole-exome targeted sequencing of the uncharacterized pine genome.;citation_pages=146-156;citation_volume=75;citation_year=2013;citation_pmid=23551702;citation_doi=10.1111/tpj.12193" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. L. Remington;citation_author=R. W. Whetten;citation_author=B. H. Liu;citation_author=D. M. O’Malley;citation_title=Construction of an AFLP genetic map with nearly complete genome coverage in Pinus taeda.;citation_pages=1279-1292;citation_volume=98;citation_year=1999;citation_issue=8;citation_pmid=12238515;citation_doi=10.1007/s001220051194" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=M. F. Resende;citation_author=P. Munoz;citation_author=J. J. Acosta;citation_author=G. F. Peter;citation_author=J. M. Davis;citation_title=Accelerating the domestication of trees using genomic selection: accuracy of prediction models across ages and environments.;citation_pages=617-624;citation_volume=193;citation_year=2012;citation_issue=3;citation_pmid=21973055;citation_doi=10.1111/j.1469-8137.2011.03895.x" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=C. Saintenac;citation_author=D. Jiang;citation_author=E. D. Akhunov;citation_title=Targeted analysis of nucleotide and copy number variation by exon capture in allotetraploid wheat genome.;citation_pages=R88-R88;citation_volume=12;citation_year=2011;citation_issue=9;citation_pmid=21917144;citation_doi=10.1186/gb-2011-12-9-r88" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Sato;citation_author=S. Tabata;citation_author=H. Hirakawa;citation_author=E. Asamizu;citation_author=K. Shirasawa;citation_title=The tomato genome sequence provides insights into fleshy fruit evolution.;citation_pages=635-641;citation_volume=485;citation_year=2012;citation_issue=7400;citation_pmid=22660326;citation_doi=10.1038/nature11119" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=J. F. Stewart;citation_author=C. G. Tauer;citation_author=C. D. Nelson;citation_title=Bidirectional introgression between loblolly pine (Pinus taeda L.) and shortleaf pine (P. echinata Mill.) has increased since the 1950s.;citation_pages=725-735;citation_volume=8;citation_year=2012;citation_doi=10.1007/s11295-011-0459-2" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. A. Swanson-Wagner;citation_author=S. R. Eichten;citation_author=S. Kumari;citation_author=P. Tiffin;citation_author=J. C. Stein;citation_title=Pervasive gene content variation and copy number variation in maize and its undomesticated progenitor.;citation_volume=20;citation_year=2010;citation_pmid=21036921;citation_doi=10.1101/gr.109165.110" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. M. Teo;citation_author=Y. Pawitan;citation_author=C. S. Ku;citation_author=K. S. Chia;citation_author=A. Salim;citation_title=Statistical challenges associated with detecting copy number variations with next-generation sequencing.;citation_volume=28;citation_year=2012;citation_pmid=22942022;citation_doi=10.1093/bioinformatics/bts535" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. A. Tuskan;citation_author=S. Difazio;citation_author=S. Jansson;citation_author=J. Bohlmann;citation_author=I. Grigoriev;citation_title=The genome of black cottonwood, Populus trichocarpa (Torr. &amp;amp; Gray).;citation_pages=1596-1604;citation_volume=313;citation_year=2006;citation_issue=5793;citation_pmid=16973872;citation_doi=10.1126/science.1128691" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=R. Valdes-Mas;citation_author=S. Bea;citation_author=D. A. Puente;citation_author=C. Lopez-Otin;citation_author=X. S. Puente;citation_title=Estimation of copy number alterations from exome sequencing data.;citation_pages=e51422;citation_volume=7;citation_year=2012;citation_pmid=23284693;citation_doi=10.1371/journal.pone.0051422" />
<meta name="citation_reference" content="citation_title=JoinMap 3.0, Software for the Calculation of Genetic Linkage Maps;citation_author=J. W. Van Ooijen;citation_author=R. E. Voorrips;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. E. Voorrips;citation_title=MapChart: Software for the Graphical Presentation of Linkage Maps and QTLs;citation_pages=77-78;citation_volume=93;citation_year=2002;citation_issue=1;citation_pmid=12011185;citation_doi=10.1093/jhered/93.1.77" />
<meta name="citation_reference" content="citation_title=The southern forest resource assessment summary report;citation_author=D. N. Wear;citation_author=J. G. Greis;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=L. Zhou;citation_author=J. A. Holliday;citation_title=Targeted enrichment of the black cottonwood (Populus trichocarpa) gene space using sequence capture.;citation_pages=703;citation_volume=13;citation_year=2012;citation_pmid=23241106;citation_doi=10.1186/1471-2164-13-703" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A High-Density Gene Map of Loblolly Pine (Pinus taeda L.) Based on Exome Sequence Capture Genotyping" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1.cover-source.jpg" />
<meta name="twitter:description" content="Loblolly pine ( Pinus taeda L.) is an economically and ecologically important conifer for which a suite of genomic resources is being generated. Despite recent attempts to sequence the large genome of conifers, their assembly and the positioning of genes remains largely incomplete. The interspecific synteny in pines suggests that a gene-based map would be useful to support genome assemblies and analysis of conifers. To establish a reference gene-based genetic map, we performed exome sequencing of 14729 genes on a mapping population of 72 haploid samples, generating a resource of 7434 sequence variants segregating for 3787 genes. Most markers are single-nucleotide polymorphisms, although short insertions/deletions and multiple nucleotide polymorphisms also were used. Marker segregation in the population was used to generate a high-density, gene-based genetic map. A total of 2841 genes were mapped to pine’s 12 linkage groups with an average of one marker every 0.58 cM. Capture data were used to detect gene presence/absence variations and position 65 genes on the map. We compared the marker order of genes previously mapped in loblolly pine and found high agreement. We estimated that 4123 genes had enough sequencing depth for reliable detection of markers, suggesting a high marker conversation rate of 92% (3787/4123). This is possible because a significant portion of the gene is captured and sequenced, increasing the chances of identifying a polymorphic site for characterization and mapping. This sub-centiMorgan genetic map provides a valuable resource for gene positioning on chromosomes and guide for the assembly of a reference pine genome." />
<meta name="og-title" property="og:title" content="A High-Density Gene Map of Loblolly Pine (Pinus taeda L.) Based on Exome Sequence Capture Genotyping" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/1/29" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Loblolly pine ( Pinus taeda L.) is an economically and ecologically important conifer for which a suite of genomic resources is being generated. Despite recent attempts to sequence the large genome of conifers, their assembly and the positioning of genes remains largely incomplete. The interspecific synteny in pines suggests that a gene-based map would be useful to support genome assemblies and analysis of conifers. To establish a reference gene-based genetic map, we performed exome sequencing of 14729 genes on a mapping population of 72 haploid samples, generating a resource of 7434 sequence variants segregating for 3787 genes. Most markers are single-nucleotide polymorphisms, although short insertions/deletions and multiple nucleotide polymorphisms also were used. Marker segregation in the population was used to generate a high-density, gene-based genetic map. A total of 2841 genes were mapped to pine’s 12 linkage groups with an average of one marker every 0.58 cM. Capture data were used to detect gene presence/absence variations and position 65 genes on the map. We compared the marker order of genes previously mapped in loblolly pine and found high agreement. We estimated that 4123 genes had enough sequencing depth for reliable detection of markers, suggesting a high marker conversation rate of 92% (3787/4123). This is possible because a significant portion of the gene is captured and sequenced, increasing the chances of identifying a polymorphic site for characterization and mapping. This sub-centiMorgan genetic map provides a valuable resource for gene positioning on chromosomes and guide for the assembly of a reference pine genome." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1.cover-source.jpg" />
    <title>A High-Density Gene Map of Loblolly Pine (Pinus taeda L.) Based on Exome Sequence Capture Genotyping | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"6ZOTvkkapetTGv8DPksOPzxHwXL_7hE-iaazQOj-HWI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459608","apath":"\/ggg\/4\/1\/29.atom","pisa":"ggg;4\/1\/29","ac":{"\/ggg\/4\/1\/29.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/1\/29.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/1\/29"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/1\/29"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/1\/29":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SIx2kkpgwHkN5eV5uMuSfFDw1Hd8KPJ9hhD8WKsrXdw4iMst1HqzAiBq22lrgSXPi\\\\\\\/EX0iq+UJlxoUUcDIrY7jpmd1EeeJ4FmTMXNh1j5hp8Yrwm6rPZ+IjcECKo\\\\\\\/LaGqXHOdBpwPhZ0gvdnpfU3jKHYXlRy6t9JKuS7OQn7auw+5WM5P3R7Qm2NVeb5EDkw7ET+Gqors1euNfS1QLCmn\\\\\\\/807XYXuM7UdNZ0ikcFK1z21vJ0FnivtkUTglTsy7mBk3E01Dry0Xo1OBJQ2vna26oGnEw7ZvwekDl4zQoBFB+ECAOb5L13gyEfNZKyED2XTzcLg43XVnGiU9ycoj2m9jDRbRS0tXb6WZaGyoxNVpCqIAxqcquKU9cBtSuGvh5EDk8piVavPKIs+bqD0169DYhckSRrB\\\\\\\/MSGfAKKU6aIDR+FeTcb3YSVgRGLEvOHcqkcrzYZr0BsiqqB2apXO5xyO7sFMJ\\\\\\\/pp4f1uJe4\\\\\\\/rgQfaCpsut1TffxQwpUH0iJPiHYIMnvBECEJMqualB3M4+thzX1S4QAVUJ7\\\\\\\/Uut8WT5YrS4l69D8yKRkwI4cvbrDT2EJZJMdtNiQFYu9HTRkfCNastuKSyE8RynCSLMOI6K1rbTkj3zaTpds2Fl\\\\\\\/upx4u3dQTsXPJApIyWhlDAPcqezW5m17XNwoeoEVVCixYD9HbQaM\\\\\\\/Hwsj4p0ZZ1lmpr3jTJUH\\\\\\\/v5Ffq\\\\\\\/PolcZRCpAd2oSisGPY66q9aMMCuZXYVVF4oMhK\\\\\\\/wBnSRGi+ByQ6PBjTE91EPRDGLWw0WDz5SCcywt6mBxFWqNBYik5Zqs4f2BAurImbpI05qsv1fAFEHU5v03Fn679exCIpRo4D7WPz0Dh6OfmCFpU6R+Ya0mY1L1c9n6EJMtwJeeGXc39LHnZFFIASdsp39IHQwddKuzC9Z+SsIgoKWCrnGQ6FIdhFHqgsH7xCR5UQgsGS58\\\\\\\/VCtoOtSP0SywoFDHOGXG0K84Pc\\\\\\\/BhyaLGgpRb8qU66h3HUTK0uVPFfpQfeopEOnEQu5iqhcIwPSJkDhwuuv0Y2BL0ULoDuVulcW+lm+mccXUlEoSnCQGZasZTQXJLNMOAz\\\\\\\/odn+3Tgw4SeoR6EZm9mRhUToh271t6HI9OfMdyA8vMVvIAnJ\\\\\\\/o0U2Vlou+kksZuTz4pmXBI3Suz6geaE8EJUv8AhI1Hw+2yVNS4o6uNFjwR1\\\\\\\/G1HD\\\\\\\/IDNne0aWfpoBJ9zr77kCbdO3frcUpfvjM9b0u5qWRGuQzAu4HrLH7f1MbjHPZ9+ly00oIovN\\\\\\\/P3u4nG\\\u0022,\\\u0022iv\\\u0022:\\\u0022s\\\\\\\/W0L0pfSUCK08CKVRhf5w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229877085b98d51bd67ede236f00e5bd070db8d65055d3dca0ef7506a9e96b9990\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PbTza7AWW32qu8erEcnv5mVeKwG21TdKeALVGTV1dSXZCdJHevDus3AQOTQ6OyNs9RdPYEVPwQeVY1hg7NwmhaR0LvMKkU+Dl1yD7tvLL8imoQ3eqCckmAi5HcFK1fqORaeqE+D6QDDsn\\\\\\\/XoCTbhW5RZyeONzKYcakjruQ0bLxrESVh6g0BvaC6CugEwCnSjh8LyGvWm7wiyNlIGBSjMe7ppm28Z0WC0TuEsbZoAD3McM4k6mYzLgsfZMM2wEWx9+DMlPo46f5qpMUX5mxfqkEn4vIMtRSpHVHoKlnxAsvf12UEArV40ZjapruAtyPayp0DznU203L8B7g+\\\\\\\/tUUc+UgzwweRkQhRmbv5lhsssbU3TuxhONTAljtmaXbk7aYi1TgfzUBLPdhfOpCI1TaSJDz\\\\\\\/6sB2aSmLcUvRaq69lY18wL0k62B9ea7IxSPz9Reb99\\\\\\\/XErzY\\\\\\\/r+LU5GB68DYYI57eyC\\\\\\\/tLg1WvaVKzW1qqIBxIl3+1jzShQhhZgCXFupAuk0c9PfgBMPzzx9xT0abIteOdTEXkCxEhZXmbEfO\\\\\\\/Xo0c+wehD4iMicffDARf6yid3EJKwcTwrFzbZLiF7719WnokYa7Ol494gb9h5sIwbo8TqSaCF6vNJJLD3UCz1UjbcpTvlcJU163d9h6QJk1S\\\\\\\/6tvSkVfER6xmWhQznOWN+lLPrct8S2nToW4jkLTPcYRaqTIyQTOnYeo7aPrgcNGE1JWWdaQLMLxGlAYpN7AtHYfwoyKNnjzR3eheBJm9kWRA+4q1zFprpmf4avpCp+up\\\\\\\/YuHe5cBPRkgkaiKmj4EfAqa3QUaKFHf6o8BWycyg9qHlSci2qOcMoXMGQOrqASOV20o9itEOBEaYgW78kthUPyxjlSzNEnizjts85dvMLzdKjDkmC7snLdcYpymqlb2GQqAbx4esc1u7i2AlouL+XAJe7KaXArrHzNXs14rz29jWoXD\\\\\\\/PSHD65wrOZaGMK77+retEYYPTm2V1XYF6FaTXjEIMCndyGNxkMbId4OCw754T9YyCPFxexe8X\\\\\\\/EyS\\\\\\\/ns7b4xsDtEqTeVJGbz9C\\\\\\\/qK+lYaOXKyxHsenW9LaoUII8imJ5Dprz7x1lonlcVkkOyWFpfbQEeHvU5nd9P4BhHbsuuDB0Uh7jPcNlnMOESzv43ioRjv9waNR\\\\\\\/jDFUNfeFAD4DAULuEz0htDxxIMi+dYAwiDPm02xJwVeDTaVR4vouHDJG7tE+yVwagXUwY16\\\\\\\/+tDa6kUyhqS8irDNCA1re1AWZDwOt2M5Ne0w1IhGawG63ZU47pp1xXj2vbau5eOtIUBsLv6\\\\\\\/AbMadzg+rxaKrYU1IhtNtwN+F5S239llFTNlMRq0Qy2oLxs8aXJtEu0e+6ftOlaLZJZTxRy32WW4q\\\\\\\/qga\\\\\\\/FxTw9dQ3lpH\\\\\\\/LLaMkShvYUqBUBc660PaxtAQqPgyGlCZYKQdjWHR\\\\\\\/vwsgpgixtRNLFo9aNJCw\\\\\\\/J7CTIGox6H8LX3sIjUnkZr7QmXrsvgiKVn3a30vCkxX32kU567dSa8gdFDDZ7BhRnKdQEWvmrXx8xuFxHZ+I1h8mGgrIufmV59VEUEOtyi+UopMR8Q4dkp3lOQeBYFnraHNqMm+KW0NQfdgQZ1FK8NmyaB5sTkXO66wV35KSaiLa2PJP9+cGJdnIV25XfL\\\\\\\/QEkvvnNCX9laG0vBwadPW++X\\\\\\\/h6OyeKrbtdk1LyV2Q8LI9ty02\\\\\\\/hwgEt2N6howBXthf5\\\\\\\/+gKoKRvb4DNUZZoZD1yuWCj4HzamKMPv7h5fWy67wIsWWARn5BhXUDnjzQ6Yb7FZtFWYlRq8Eyr\\\\\\\/eopsQ9GWGLTqd6hfPAZJbMgCIunL4YgyxIShNzmfZMQrzcE5gttxIjrWnsLPrX6OvggAogKReprxHN5rJAowxDWpnS9Lg9QIOsfwvIpLQVPu7IUu+A7OwBlZXltd9gJNL8j5ZFUmMjG8uef23hH6MCHWuvcjzG4HEzMYSpz7eIjznaa8D4LQSt1rk61PSfpU0kM4meMy9JiMN9Z6DZi5gIfLY46+JpNk1bRfMzR86CCpPPgV+HdbExP+oUXTtgQk5MMFLNZn\\\\\\\/TbrtlBYeFkir8BnegrYqHEfuwGYP+DqdEhaZrTiLEH4f4U9TFBQSMKvfcD2fZaWc6uIppNBMAti6ciVu\\\\\\\/XVb\\\\\\\/QcnNpgMW9YlmXuTC0CPN4DUiOJa2UMGrvfOM2xwcxujqBz2XSqYYBDiRi\\\\\\\/0tTJG6CjNh\\\\\\\/ZPZA+pZKIZm\\\\\\\/tZng4bOjYGyNT+4Fy4PS3VbqVMftGVjX7y4xy7okGHiuTbDitcy3gHTLsf1gfJv6+\\\\\\\/HazFdlh7KDuuHHGtXKbZfSiJFdBlZ\\\\\\\/7vmKzYPz6j9v4BSVbVGS7NdNH\\\\\\\/0ngfKt63C3icZVE\\\\\\\/rQMkcQYjrLhQn1aQe2VvyLCHC8es64ADDMm\\\\\\\/s0NcX8GEYT4GTfVS8bRYMESVkcutD2cEQe2GjUYhhaTKgTWp1g\\\\\\\/O8wzErDj6g\\\\\\\/NYxz7aGat0UPbc6ZENct9WhRAg1ydL+D3SHcO2O6FFxEF\\\\\\\/iCXqg\\\\\\\/LVWr40DivKFQ\\\\\\\/kG8f1giwQFlMPBZI9WEkxv5kF8y+LbiN4mfvB5GZ8lp7EXE5ZxGhUGXAYQQjAr+fcuIpGXm2eJA3UC3RO\\\\\\\/J9oOQ\\\\\\\/u8RachKxyyTv\\\\\\\/I\\\\\\\/GNEsnjOOsq7oNULiD+C9K6k6ZP9DEEjh9UXiRFaiiuBKVhWRdJPqiDSQOE4klsBr9aglfo9eBWY\\\\\\\/WngzAbYqL4vVGQFTFAux9BVFmrLjV7yg4jQfJugUQlCr2Z93okPAFqTLIR2vz3On+0UXIOngDrGr9dMg3+htQJhzPakg\\\\\\\/xvMEk3aF+kxUhw+vHGTsJCSrelkHRQJ65FCVl34HCAXToWuIiooWxt8RGKl4rf36nt\\\\\\\/VMTtL8o\\\\\\\/buMggr8McantarBGvtl+nkNxJgM5MapauQnwWs14y14bugs5Vyav+pWnNhxGgXMa8Tn4Qv5EdyLsYqA1QY5se4n4jadGyugll9hsUagvWv8Mbrdh41Hsg7leM5IrcaVqh6yBpc9aTfL+6EkToj43O\\\\\\\/fpEvPNkEW2zdOGe2DSpcKt2lQIUO\\\\\\\/LC9wNAaPFWwWLw30y6Gy+lVm0lcl3+rouoMVwVrZsvB6MAB5Ps2J6fO08Bx6vs991HNi+qgVU+EgaTPHPHRs6TnG944zw0aQwVEEhBsK3YmeFrxUG1YyQVpR7TF55qpCyZAKEWTEqRPmxDymdKwAIrIcowwT+PaMiQw2fEH+gcmrxX5grlCxx9eT0AJ53yrcjsKwiJp6z\\\\\\\/SD7okL1FnYE0iSzxB2R6zxD5T1nIyq3ECueEX56Xxd\\\\\\\/5ZJ5n9sEslzxeACkeOjZYErZD9aLEK5x2eAO1FayeeoLEHCtqd1Bn3OGA\\\\\\\/8Bj8c26kNF3w5QcMnsXYmfaJsIGMYvDA5tgJ\\\\\\\/3VxSCht7e0m9XK4MYlc5GYAr1I85Ecp\\\\\\\/oqA3B9VyuVJc1\\\\\\\/9ThgCtewgAT5ER1oCDok75ASZl9WiCVnGhQFBZniOMYmzcFx8t1fN9yaRCJgxvEINBrr7TvLgG8wIJ10DBw\\\\\\\/VSw2TvGxp\\\\\\\/6CT1ZC+q0gTLTojZOSvdtESNQRfSiOPBq3Vl8gUG3Qh4DOuDIB5Xe0lgyMJ7Am2qVCgPVhrtGabbyYPaQeSKpzgMJurFLb3eczImPga5ZtA2gvBoyHwyr+NILNhKMPilO0RPMgmBTZiccq3uIUKyCpk1k7IyHD55URD5fMZLGy5ZdXqtoOnSyHbXtTozimVE9n2FDTJvWn\\\\\\\/AqZkGnMXzJGkEMeyRVeVHkGloHyvRgoyjAGZr4507vNZiGpw+qDzRZv\\\\\\\/gh1Fd2nf2r35iTjsMvTYA6G8HwUKRQ62UmTvqrlIY4ziK2Yz8a6Bhm62baiGtx1C+0AzcJFABJ4eBSZrKY+mL3+5J57KIA8O33UdzfzMaUArrO2o4UQBhAm7lMln9H+72dzjTmmTlj5VSi5RRZ5EIIRxhem4ZEEizVTkIwua11+Z5nW\\\\\\\/lOmYUv2uvL9y7pmNk1z3wrENFKpZf5J8BywXuGXOwTHGOvoSYasT5k8qA9xFmc+r7kQ8lfFcEWQ8T+R7S3x9nvJBnErkDWQhx8HGV9yt5DScokv4mhte6UzIcJrhN1twRP8ZZQuDkSpdd5WQXJuqThOkU5bs3g7yelCQhyXJFAaTi23ZXaXCMXUj9fUFJuC7LD3V4R5zpw6TTG0l8JEaTUoifjzTUHKNWDfBl57OQ9Z63xmREt0KV+GIbRk\\\\\\\/v49apvAQGi3\\\\\\\/vcm6Eul69KGN+ZPlugVpFVlS\\\\\\\/CmUMx9goGjJ8Qvm1giDYJbN\\\\\\\/z1+5xqGlssLG0RHQAwu\\\\\\\/G3yDQo4pkD+SKN1P1l6E\\\\\\\/ZaS1tGeAbebtvX\\\\\\\/BsVPS0FW8D8OJg2gM0+pnGM\\\\\\\/1loNqU0Ygu+nWkHmpoTQAVfTFjeeda296V5wLu+6VZ14qpJ9ekcosSBIWKiXqWu5jJ5mzNh91zvCrZCbpkcoMF1qy+vrz3cd\\\\\\\/9ssFl30Pe9P0WNCt1La0jl1aDlrcgMK5b507E0aUJBuzm\\\\\\\/Xl9rDScycVg0u2cOL\\\\\\\/IYfsSHFN6RQgSUXs4ecToV1+VsoCFoja3G1tW65WUNW6LmV+MBcNo9kc2hJ3Za+oa581k4o2+iIglrZ7X5lnoe5IbfY4LW7iBDlQuudcE5ZzjGD6YGuR9k0DPLg9LrNtVHZab3HNj\\\\\\\/gI9AmEqBoG3wc\\\\\\\/gzXMEfmPkPEhkLZeiAFjdHfo1IbkBa\\\\\\\/JK4jb8WxJTXfp8QNxxvWaZM24w9rX+OMuKYTn4d6m7N1Y5iEE1kOjw3FIp5MoSMCwwhKUAerVOnY6ySBT7axQ5dB9gmkAfdiZ6NKyy\\\\\\\/E56x4jGpYLN1SmzVVaCNQ0VgkyB9tbwSV5wymcKBQB3pvwmr5Om8mmBZxPLZtKWNp3aDrU1NbNPEsUnBtU\\\\\\\/87rU7fmE0Mn81RM3YbvyPQwAmv7PzDSMz6SvBozMwxX+XmhyXISpKaN2E8\\\\\\\/xsYxZd5x28skRBC2TOeoA6wtvF+Gw5pIRvdJ02vcv7yK107ZEMH0xplZcfSnq95cN7LrQxCStGOzfH7Gf8QS2QUFZKDW1HxwV\\\\\\\/\\\\\\\/cxmxr5lIm+bY9YmV\\\\\\\/ELSTVd6B+J435L4KJc1p2c7Sd9q51fg5m+EvePPJOCv178tEMOdU\\\\\\\/qEvyeW6c9u7UH4kdeaS2Sa5E5TRvYJzRqoFTJvrg9JwVAdFNkwjLWDlyuK6eZW7gv96GAd6UPee+aagw6teKq20f+pQzEQMAJeeX9NKlxexzTvbLvqLB8AfjW7po\\\\\\\/zSqrZswRXYgwmkFD3znKzaHqEU8X0+1k9bVXTKvdN3gHWboq6dhBPtEEt9mmxOhslVfykqnLKZN0X4o7HPzGc4htUn\\\\\\\/r6b2bkp9ZecB4KffsNopCle6l0Kk9sEk2bpbqgdlJMooo1vOj2TYQ7qaeZ65KdEgG\\\\\\\/rT7mqmijCZ1yH9iz\\\\\\\/Z0Xmty0yP+nI15ZaffArLe6sCoDlFRaAV+hYuEJ12A\\\\\\\/ydJ7aJkfiFzUOHsExSQVHfg8IAKmeaOcP4dkC9mLQluw1rY8\\\\\\\/icNHIajV\\\\\\\/9Z7o3lskE7utzueft6ECUNU7BCQjqbI+sroHFAY1S9ar7msQJmU0yfNxEXe6UHrENRWFrv0AKFCvpNrr2LMzTmZTjP\\\\\\\/NxodB7m26cWsrpmSeDbdn20gWRWtvFB8K1kH\\\\\\\/BFZW+H7rnHno28mV3g1\\\\\\\/Qtwvb62n23NLf92L3wfNb38VlHXEtjyzxlNcU5uT2LEVBpwalR1nzAcBhoW7XR1E60cMg9jEAQ0xTPiJvNxTHusw2subJdIPkH\\\\\\\/etXZaexMUtyWCdXjPllsc4Ms7yia+jG+STdYaUUHia6rAJXxLqsQqQ2gVYlfvnqmR\\\\\\\/H5\\\\\\\/GIePGFzr29HfeWq1MyjO1u6SGJU4T5baby+CXgrNCni3oyPldhYs+7rxcOMFP+TIDQwDy5nE0c04gGyOJvlFmXdfG8pJcgam0xYB61anGxPp+4KogfC2iSSkr5pYedSk677qv2WNJ6l8WBdXlUjKnf\\\\\\\/oT0NgKnFm2HT+Y6dsb7HWzS0mQ94Lg6h76db\\\\\\\/WqNOlbnFmvDsfO83r4a14NHYsHGv07\\\\\\\/tbGznZSTm+odLyAv0Qa7W8d5caxhNCvvrdC\\\\\\\/ehwjYk+nhp3WV4dEqU2BsO0hAHXfmVTJydjKH3DHJFHYb0YGsiF1M5tugECCTwS2ERJ8YuAHbHGkcDZgwJ2OaImhAkQmVQ5iMklp9oX+M1vrQ\\\\\\\/d7dvCbqlN+D1WRwvr4I2zN91GA0vPl58MW04RDykqweI3wLLHnOvqtpP1GYxu7VauHCRNwIRd\\\\\\\/O5BHGPLnOpEMkaQhmDxtb0LSX0lvR7ux0+I7v+DTXsSp25U2Y5LDnGLZwaxCvvofR6RcIv3OjUgXwgchgEGXBlo5m2XFa6wUuWlvim9EwbYApjqYwb2ByyVeva5WBXOfd4fnQjWynV+IOJyr9d8ZCxFDPvXIEyl3qMrq0jyCSj2VBiT8veWr33Tjx1XEg55P4IMx790wdZ55H7tmbMV13M4rcGEPYPTNL3R4Sh0mdwMzCR97IQ45hw4gR\\\\\\\/oUmCSLk7iZyaNrHtcaVNILZod9ASHw6QBATAILdhtyJ95d35zp4VjLtPkD6j6CzITppyw6hUei2ID3fmvrzF92QRP5CPhLrKh1FKRx6GYZ3pAB600JJOcUdgKVg2cgkdnDtOrEFp3vm4RnUURdF1voYlEN2pmDfVsF9oeYZCF6bTb+c7ax0+tDkMDFczJzy\\\\\\\/OZD3mkJLnZWHlBi6P5cV49ptt0YIBQ5hiyoUmemk+K\\\\\\\/0lQuKh7thmiddXLUPuaIIeZsNL8LtJ7aT9rbAP06LdomBJjr5g5ZUKkM9iAQMkegm+GWP\\\\\\\/+kmpWV86jW0JMCkW\\\\\\\/YPuGouvpBEZWbFTqR\\\\\\\/sL3syMz8eXxS4eBwqTXxMDf31p2EjhiP6TBHdnJdwa95HcoAmIxFDhX6Z6tLYPfnWmIj0\\\\\\\/GBmfq323ylqSjHm\\\\\\\/rbaRxKH6AQKHVwwKTDxwQZCxDSrXDJ7XJpnKMQossqvWL6p1SyQx6CNfwJVwRfilARZUAC+QLqtLgj6LkoI1YtQYM832STbrjgebHzYMOwpHr5p6Tm4WQUJh6zIt3L8CQ5OEEWlt1FNCTEoA81mFiuMvo5S6pYNu4vlrJmi5c7TIDLI+aHW8jypctv8QIxbDx4PX+ZX\\\\\\\/s7Zj7G8QmsjDuuhi4EeFrItjTNap8WDtMmUpL\\\\\\\/wOWE7H+mHPzAEJcDuG5aNUStXYMRufmp3HtycyUZ35wHYyvslAYaW79dF6krPmOBeM5TRs7vrOckGy7iNGTSchGVknHQ64x0gCCRNv+QMlb1Y3tay209HIWwjN8PGtVfqLro\\\\\\\/1QnvdOkwIUsrUbiC6xF+zhuch0U2zhgQ855HYosODvUdAiayPNwC1wSe6sdIHiG5hsdq2oqXAuUH+OpobN23a7G6kyCy2\\\\\\\/TEIp01s1gro+xi0qHu2SrvEBJbpKbiMs7OZ6ATuHbwHcEPX46rC+F7mWXniWcPn8t6QfLnRNfLLUQosSAl+2UoU3u6rZ7yo95ttoZQ5IfAUt8wRTKGFBNTOj849mVK9LIQj44epmdzXpL0b9+D3x+DGU2A\\\\\\\/plKAqtF8JsbJQ4CYFW5PlX7bDq7phxe51\\\\\\\/+NNvggBMyYW0drIeVmtg9ISABnLJ4rNz1o63baiFj1nJo92cWvBJ0+AlxAnGSoXG0w9rVA9ArzouDZwmsRD0OQUOSp88tf7JjKJiYi7wT+Ry+Y7DBcZLNXZdGPjyC9v9+fgxSe9B8OrGQyygTke6HmJcfwCjWKmtwbzVTvc1zkVbc0bAq1dErpt0EzIQzcOgw8\\\\\\\/3CPy1LJBi7XHWPUclp2Zx6vfe+MnBUTQM2Qe4GUiFmQS\\\\\\\/g3pSOj6\\\\\\\/estS1IhsLOCQdz9B4s1UecPIJQZlVDfrfndFAMs\\\\\\\/ZfrNXrcf3iuNr3aObGjKapsBCdblXMHIZDHtv9h+cPYWgwXODUAwXFjsaSNNTy7P7VDFdLbd3KqnDBZvksugt+vc3Hwh+Ki8Z4CY59k5c6AgptnwXi\\\\\\\/TINzVmJbCzDtyWmHUx+6+TMeOqOkMJPmgzmmdB\\\\\\\/eyosp\\\\\\\/NHy93Il\\\\\\\/SVc4AOkhd88qOqbTSY1VO3q1u\\\\\\\/ygXon\\\\\\\/jsFcPFRUMtmSyd31vhdgyLIZnftbN+E+pojsKjPVrxuyHMcJVlD83f1ia7uF5bRwSWdgPsbMCg6h+YpK7pDGPiuatYmYoHWEnGsGHc84KriFJjSoSxO1OjBTsIm7Qi+p4aFoLp8y\\\\\\\/Da5cmsM4YFiyfQwcnNaknQRi3jakftm8BJOWDI+zDWfqqla9JU\\\\\\\/3eyTnM17PsunXqYA\\\\\\\/VdGUUUfqiZtiUurX4TCDhvrEXU0rab1TEpTLLO32Mhsb80VDaB2xb4TKfuyYpmNx\\\\\\\/spP+RpaE5kfunHbCQo5XfJLz9fYY2dniDqMz4QZAhj6OoLdK4lDUgRYK8DcJ74f6nGz\\\\\\\/nam6ohDgGW3nFavpuFOKF3Gm7pmGroSybOL\\\\\\\/COS4EMa8PPb\\\\\\\/IShI0YYTBuPAuSj6fSC43KuY9nDqNNd5511E\\\\\\\/nPVrnEKAOKzEekXTsQ+oYqnMkuFeJ\\\\\\\/0OFT\\\\\\\/rvrBlekcp3mR0rbuSEydUmjhUzRovjHdaZFZvVx4BmQoWaVtofVTaFZV5qmGMNPgjzxgk5o2ngl5c392ZG8LdgHlMBFYurIga7tn2bHIV1ExOx21mqpcLHKjUNf5WBH1JvCB41Jsee6B5rAeGpyLIkpP0dX7NgPZO1+OUNQUfak59WyxRQnxpdvmFtsEUlo0mwwMnXYupFD\\\\\\\/v0kG+EDXsjxyiGH3qz8gqWSJVRqed+\\\\\\\/Brnawg\\\\\\\/KEMp3xARIzimZCZG3PujXF3\\\\\\\/lnmrbKh79o6SKOCx4+a1AfhRDt4KA22HeEnWz18mUfH4Sj3SBgYxiuHXgoOJNXPNvpe3us1ES8YzeN5FBbxJBLW9lZmNvZcKfbHEf1rP05lnpwxqg9i1aXsuFdVIY96SQhelHQtMPQ5IZUWSmTFWj6XyvmtaK3ebiSCFcS22jOWf7C+nXmew0VGSMeaPhBhc69KhyFnE2zZzZKw70pTUp+dnqeHa\\\\\\\/IP3UgIkRO4F7BVlIhEISJjPJ5Wyhwf9rW14yG5xf4VLYdnH5RoCs3IOjbgoBcz3YM5JnP\\\\\\\/8HSuJieal+D2ah\\\\\\\/hraZvY7ARUw4Z89Z8NZVgmXxOI0aomqwCFo+glw+g0+BU+j8wr4n0b9\\\\\\\/603+NmzrYrp7MkQbydQO7HYhgBKHBMIycCnn27uIYtvgP3lS9lH8b+aImf2XvbyNnGAUkdb3FoeKpEbTTQgelRSh\\\\\\\/agBiPen8PbcIkWYw9drdfzI21zlXEsE+\\\\\\\/kgW1YCn1f7gExKnbxU0vrYGXatV9ulIjPeQ1AiRRKhicfSD4PmDpAaMVcGdVl1WrfPcf2SdzYUV6iJsyolcIbyys\\\\\\\/IyhfT9FFledyPcze9pnYnuGbijKQBKLbRYV6bH5Zwh8mP+IpU77ExuxkGF56CyLQQ8RJeUO0l7oKqRfYVAYEIZhmRhC2bhkg0OrwDd0Gj8lFB5nYsbhkqbfr3QAnsgN1AI67ut1P+xc1zN+XKfl+6cLy1FhWOYc4+XHQT\\\\\\\/WHbJ9BzsWptakXMKG5OmCaA9djDcEoQW9hRrb059g\\\\\\\/BcLWQYyVXLsp29JOa35PMnyjFjbFfqU0WI3oTiV+bcHrVvc7YrXgWzmgZLHw+bhKVl8AqjGt1\\\\\\\/5jUnyZ5\\\\\\\/Mzl32S3S296bSuQ3sksGYBWlgllFLOt17DUyfRkEdBoqp4ojujOa6Wow2WMbAynax3TsMsAoNukNbDIJQzOkmlLhBbCKtOUDC3otSTsUHM80hfIQ7l2HhiBVbWWRLQsVPahPcfYzXdCVdrv0BEQ0H2d25t7gvzoacHfCn+VoTfX6yQ8LTxXoaV\\\\\\\/Bmu6MwSacvsXJDopMeQhQIEFETl5Q+2TTZB3xvapisZiFPjReFr3gmIXdAu1bEoBCXFw0e8v4Ltj+26l2xrn5eJh9FrculpawD36O52ekFs5KMQJNiEp6G0AFG4eTrAfj\\\\\\\/i9m9eQH9ZRDt85QbglYw7CfHSa9BC9VfrLVbrGIffzSSbRI7GDq2e0d2eF52vlY9WArCgKnZQcSug9dLZvmoGdINjrVE1ALHNshpGa7\\\\\\\/gOjBpi8LmpG7B0YlkQ09hKZZ8TIVS3aZv18rQ5soouJBL7lMKYYI3d14g\\\\\\\/wTEiGSsQ5bsuHFn3PK\\\\\\\/QP3LtnM48U7k0ToQO6Cb5eEq+zS6mtveuJRLRDTkL8GHYyD6gVCTr8NUFHYfzUa5jgcZk1gotna88TvFDsrezL\\\\\\\/tR3zuIfYbOGS+eEKOdovgrMDO8kfWgmY58plW6K9JHTyVcwkt8ffwwZX\\\\\\\/uUDmNKQRrAEtllvUBU3TomGzKxsWLYPlcsNI692YGg\\\\\\\/HGsayDAydyVZ3Vqtx2seC5Z\\\\\\\/8pfYUkpuHmW681SMrXKVtXhh0eJ3cCyUgp+w61\\\\\\\/garg4C1hUOCeGzQV9b7xWEVCEf8\\\\\\\/nEDTVvoPCiTQFTWrL2DtFXSfVo4a9jA5LzyOKlh0wFTF+kzvS6Rn9h7DTEZ2G1SynqTsGD65KEpk1u6WBsiAXqQi5GO\\\\\\\/jeuUabueQD69cXZdf2nqePX5oWAUvhdTV9C7kAXPLWW5WG\\\\\\\/NRWnJYi6DftFT\\\\\\\/CnJdwmeQFd4DSBaOGOpSy0BZFbRg91RkllhaZvZC8s3zpyazMIBYcW6mcy\\\\\\\/SDMbsr6vM631QMtGFXumRYmcwsEBcgkWq7EfVSgXveV8NcThpRSunvlW6jc\\\\\\\/dF2Yq32sE7C64rVrRTzzZ7XBNanC\\\\\\\/FKA+5wKzBovCSiFLXsKswsU6Hg7vnHPDkiRMIH5L5\\\\\\\/lJgNfOjI4rWZSVrb63qKO6cM7JyHbTBVARk+Gos18c2\\\\\\\/Ct1Nx18M2Tdu5bd4nFNC6EzPyPz2\\\\\\\/6U0xn0QQpzYn8FuzIGn6HiWEKZZHk5n5qKBgKJubpb\\\\\\\/ugxeKbTm3qFKzgVSLjykiVFXUuhdPC60FX3ym+KBj1Iuhi6JJG6kqGlNVohPD7UlLwf\\\\\\\/kfJmWV4\\\\\\\/5upXcpJGd0TGuQvDlcaxd5cS97ZyUSrhRjYTar0gvbljlRWVhXGR9hjVKxpCtVPli5ZszWeSlh+8Iclx3TewV3qYdlVIRlSmBE8Otv52OgUlRrUTEtUviZp6vgJvTIkQ+mIkoig6COcwJ\\\\\\\/BAr\\\\\\\/DB1DRJxVOBQik4OaOH6o2AhPPB0DjpRehsMq3h4jWVc1f1Ioa2bl3YLFy5aWUDPWRrtBiGM\\\\\\\/kV98dWhNxBm0xPeEzOqqZrMBRG8CIfsGy+DjQKTP5BdKOG8fP9uyUxipqqQldhq67pAkkwdDZQBxcrBtS+JX04Da6KjEKR0BMzGwm\\\\\\\/4wc1aHEf0lNiApq7lTabH2YfVz82vE6vNGMKiK4v+wfH2VNxFfEJ5uYdaclyWgnadrZeCEauYR1sw1GhCpuouWktylMbzncwQPxSBIc\\\\\\\/5YqzlDkSKASNWR6U2+npj5FFWnRveqag9DDOxTMHpIS8MId86T2yOqd2x7RgyuAKZWxhd20+PNI4sTzxwWjmmExE5FyKUaqL2gyJ7xd+XwIYdepHdge0PGhPL+H8AtCyueALxz5gvpXMNEoujvvXS1yDNyfJ1NNS2jZr4PLJxDE+OuPIE0Y32aEAgX6XHpQBsxThyJjX1BgcFaP3UQchw7jWE8sYIL15bWNlqPDiCO4TldmdPJc1WAfL93aYSXZCcXHdeYubsHeg2EZ78tkp7fBX62Idb6HCw6gDs7fn4BGT3iarJZJCLV0aje9WVCvvAKUt22AXL1wGrBvtULuYciTW2T3QUwwymw8BUtPx4WCzdcQ2TrQo1ryT4K6KAXzfDKAXa6c\\\\\\\/Ispj538EzGQHkOHZ8M8S+zw8wAi8Aim3Gg9y86plAeT9Ze19eglFS3WoAdOJS0bsqzeMl0pUZAwlzLYUBoF1ao6pfy1AkoS8+c9PpIApIuHTsjkqgFe3iQzcRmqCwKhW40QRmtxTrrfwupjsCqy\\\\\\\/4BQT296EjjQCAB+EO2OEJ1osWEAkggNDC8pEcuv5O63MXx2Jn5NMK6gQu\\\\\\\/DyHmyGTPHI9pOH2zBmapZIMY3wPrjx0UO335cX\\\\\\\/dO5xwmAT0YUJq4+OG6XJfyZqWjB65Irk9Yv8+tKJYMZes+kWvblMjLL1ATMiMpkSWemzjV5QMOjtZIWJrMUHbS+q1k16bVPRe+NpEBu\\\\\\\/y5HtsSsiuaOGLWSt6haRsi575XKjJcKMQFUmKFNQ3XwwNWfLQntPCU3OR6y4q7Wt92nN8kqKLidrba0kkzCC4t7aYD14H\\\\\\\/VCj274ZB5C9TG0uZ4B2eV+tzLdezqhMkkwmWZoSU7aEL50MGIQi0PyfgGH5EeoBmT5H+yO0C7r36dABpNGR9Jq+Ej3Yqs1zIALySH8mDx1sWovkrM4IAIYNBx67sjOMkPcowHyfWttsIZ\\\\\\\/jm9o6D+KCIIuyo80q0GPTtVrZxus6wE4s\\\\\\\/V6cB71saFzmO\\\\\\\/p9Mkl98xO1uN6toRABc38yIZ5DGbaOoHQrOTCQS7gQcki7D82javx+FpYFk6R0xF75o8BJN7s6igAL8Pw8n6C0ASsVZlhlSISvb2TD3pdZL1jwr9qddVVKuC7qaUq3kypTmYYohSEu\\\\\\\/dS3U\\\\\\\/P0lr0TjOAv1kzrZ1f2V8es3nIYT4QmGLdBLckKTthDgd4HcyEGelNzNycs8eSqzmma4doBUPkoJhf3VzrLlO6SPvXyl+xLgkLsK4VQh+rULC6b6U4IRal+hG6UiOHhHI\\\\\\\/W9RvDjpCqbVQmEG0YoHnBWLFCANtfPvWNSMcdUneBhaL0WOHYhElI77fOTnTQfHpyP\\\\\\\/YXLZLd\\\\\\\/FKNHt+4FBvEjF0zkL8i7nvYFu4ml5EHuGX3fkT9r5cdezWsyD0xL+ZJDoCzJdwuZrc7eT1OWzdICzfQ5kKLp9NF2jy4Qon3kvkXgk4D8SNYYXpt5EpFRNbKNroZqm9IaASydyal8Bb7CmQVLyUXu0LO3XhSwq8e\\\\\\\/xhMzo3qL047WZPHL5p3gdAsb1DjCcOKlBgTVdxREq7V1ad3RlGNDuJs04PqJN5KwHBrKphQIa6l0w36X7OBIqloE3NSRnvHywifm2mR8992xWkML1NGVpEW9hnCsKPqi8pEWAz+glYzkGq16pe\\\\\\\/ztSWAW4NPe1ZDG4tvCv6wpsqNAa\\\\\\\/M\\\\\\\/rqx+bZXJqbjgA8Cl18O+KfmKlQ+0Zn0\\\\\\\/8w19Imu87k1VcU1AnsnK2RxBZ7pk6HH13OUbOHLit2vGPM3FUo2fKPGY9kqklfC0UUrhv2opQXE2GiDsn59090I4XwFngbv\\\\\\\/uzkOn9muMThfwzYjmB5XZJh53Rk0dgzAvpmtVWWR5t6KAh8RM1JNeKblcp6u2RPIMK7ZVmeuxevLeIfHQJNH9M3hkea8EFAQSoSMJVg1+tWSFZuaTrUfdTpQi9wGWSEWF4j6D1y1ZDSKrUBQNsJLf7ka57Yr5njYWusf0OfFi8Cw+eeDFIZ4nSDLAu8yUGDz5O6GVTCoATrafnonq\\\\\\\/t5yCJvfs\\\\\\\/NqWMXcQaRNtI99ZL5OgmyMe5ABTqf7XK0\\\\\\\/BkNgsmjn2F38DphPzxk4Z9Z0FbvPGn1hCh3MTaES2Gx9F0hqb7u+1yif6HCI2kfcQkQ5\\\\\\\/k2ZkpS9atB9aoIZBgFNslqQIiKNjR+Mbt+cwH7vwOF9+aCaU35oEIWhOFLSRctdqq9ajw0r26hJjRnze9Jsy97H89jiNGEZfss37GFiF9+wMGdXRtEGLQgHV1DoztOqt7Eue\\\\\\\/ZoHlUA8nSAK6FcK+DwQe8X4xGd\\\\\\\/x7+KNKGCF79hPbAC5+LmkwrAFQDsjWyYdbEMDWaFncEdhEAqjcB+6MdgdNiY7DDH4PZIMarNgjwvHra4H11G89pFFZvJDUusAm\\\\\\\/vihX1SfNnSb1mKV8kFWGelR\\\\\\\/Oo6df62F5Z6QuuG4+1\\\\\\\/cOwaIas80TLa+Wm2ndTIgDZBdLV2AuXxC4U4OpzzIyclKBxsCQh0l26IQKXwa7L+U4IsvVIcCGaOeEe1djfleZtCLtAio33fe\\\\\\\/agCUCyqNKvr0DgC\\\\\\\/5VmbYxlsMrn8odUh4vmH\\\\\\\/B3zC53Mckoo+wQrjOqf\\\\\\\/VGrbaTiuWxMavV30\\\\\\\/NeETF4IpSuUJlxOrCCf4AZ5m37KQ8Mdyp7qwTG9FPrbOpCpdWuqpEfoHaeEQG6Wmi4lowlCs4LbLU2Vv8aZSLu\\\\\\\/3TpZa32fqyyTKlZ8OSatum5\\\\\\\/Yf3w\\\\\\\/Zo1WEonefD6P9rSCUakt1j7s5rmcwdnPDiYIMSBMCa\\\\\\\/30kaZZfxv0fg9wwRPJJmfMyJT1YdQDuFuKjcoA9bS9P4N9oIHCJR\\\\\\\/z7+SqeYcV\\\\\\\/FUChrUec8KSOvZDbHLgR+O56kslJcUlZjt\\\\\\\/Z\\\\\\\/ZZDyusDPLi+eiWfSmtE6XkWBDNC8GbGirHqNlmkzTee8tV03Vjx1JBqySBY6fOGL5CyrfAhJSwYCwz\\\\\\\/lzJFKvqL6XYbKnhtmH2gPhEZnju\\\\\\\/Mbr+pfEH4+TiUPGXxzvcq\\\\\\\/D4xjfVyp4FuYJ+y6mphqJLIoCrUVc7rl2512WUb7fXMsm8KTKy2zSnplYMmPPPJE2VgXv+5D8q9pLpv8buTO6+YechitRiFoBGkNURio+JDp74zpfXnxdHoXbiDoJ7mlN22q4MecG3AONlxXnNwz8kKEuefeXm8arORSY0pyLtP7Uyoew3lL2xqpPBSUz+iwlbQNR2XA\\\\\\\/wbZcTJJhBSXDsU8Dz\\\\\\\/1wbFbiIZS0xTadJEqqWemOzMuX+7zbje2jPLFYGSerSZmk1ymKGy9L7aVbqEpQBdXqLUyhsn9Q86h1JZpkByxiz0nLenLa+ma\\\\\\\/SpR+WNJu2bMuHInMtGvZDaaIy1DF31gV9nHW2DCS5t2qNqgXQbN\\\\\\\/Ilhlm8KLCvsxXZp1nc9dgqp0ALr0yXfeD3lUT9Pw5W4b9jR96nFiXdpwG3ywLjhkBJShfhkiHYwyl2ehfTTIRC4IesKZROhWg\\\\\\\/xClrQV35b0tCPbYw95RympZN8Qj8UmpxRrKHAJybGAEF9VImM2mQPq+tiDalwDXoa+u7rW71zTnjkmVBsbgPfjXdHnsSexCI21xLcS1uYnOvoyWwuuXbffhQkjwkJoYDgIoh528KLshbQ29mKdr+PHFLY5N5KfnDQCaVOLeIS5jKKbL\\\\\\\/o74bSzsn5bIrk4HYaUn8lzBkt9c3RvNorriyL5cJ8yue4TiUP+rScnv8lpA4aeABHfsuUcS6E6xOcy4hqVwGZq2BFgbUUzIScobeN1AayM05mboXJeLfqfp98E\\\\\\\/9RGB4pq9SsiYnuD6SeF9dIYRre3oioxYwdfn0E2ZPSa+4wkXDbA1Y7zVtupBnj3UUD+03aRHXxIlawjDOdETHPcNCTAgydSkVEY4U5SYY4JgKa\\\\\\\/whkWCyP4kUY6Jgd2aQpxX7yWjke13H1N7nbKQ+L9pVYCJH5ixEz7d5xumFiQoYLbt8v7Fq8o8ZF+IqSTMwKordmdyY+noiTZUfB8KIhW5D7KAlEeTbTKZfQr5Nqftq0fuDumebRDukBJemZUzmW0zUKJZWm6jNFHj2DkJnHaEDwLu4iXob37jQh0T4ly1dOsLRLJak96b\\\\\\\/gM7sMQ5ppiimOKtp6aVpJW\\\\\\\/h3S1qHYjv9F0mDPQxrdDSJ9G\\\\\\\/RV6cX08qL9Jjlbl2QuNp8G8OayQEC0tShEnMy\\\\\\\/PAlgOMh8IvyKQBy6g4dO7CVz6RbF\\\\\\\/xU1U7uxC+MxuvEvKF6zLZli06m6ETKvH4ef3bOh904y3jCVI4w5hHR2JMbIAYFVWvIvP7laj4\\\\\\\/kx2GnpfS\\\\\\\/Rn83oLCtk+GuAhAKG1umzYQviZPOZLkMpR0S1kmIXjFSQkl3rUnWIHlyxiUeSilzBUxg+JhKdBIgIjL\\\\\\\/KUYQlFfaA+uIF+c6L0qC5yw0ocwdBRkEHVTZ1XwrsGsVm\\\\\\\/\\\\\\\/Rtf2C8FIvkrKWPjRIMARSGpu1ZBd9SqKQGJRrOYRdOYqspYqFr0YXcqLeIFgVs2tbqekz9Aenvm9f6MxFbnLemGUGBmwdnsJqH4eDOCGRkDXhYbbyB\\\\\\\/VHMCJ2pQTo\\\\\\\/Y6VS26zevjcW\\\\\\\/7tOciF26ypWFtjWXNvOR0TQqP8zOXQGJn6vVIXqTUo\\\\\\\/5E\\\\\\\/WX3K9tgxNZUbmzJ4UathUxhE6cUW26hkibZy87uzSHCRrBL63fwZd0A1nLs3xzIyB4+NGsH+jsdXDvksDup5eCY\\\\\\\/tYjDkYfONuqHlR0LLKQkf\\\\\\\/qgCi6tPt0zL2Pd4YP7d2KMfMgPj8gIWtBGzfXdxLcXr\\\\\\\/8rMFRIKpv5I1W3fHUUYGfHIKT5VAGYOIX7H2tlnqr1kkgZswTLM96phrEaHWb7SAjcg2\\\\\\\/6rtGbcnlSqg9JIqCzk0JvZy5gGkxDFuw2wavWdbodBgTMx0iCp7SRbRa9Tsypr+H5Ux342BGRXam6YJy5nfKRX6+HUJ05rpXi07Daq3gHIq5Oji1uLxXJUbyVek86FeqCOJ6xHg+2FCxwA3Qu6L5FyEEhgnhhJ06luFH875hqrLvnfxQP0NZh623pGcHAuEOsQ+t2y+VaSRmT5tG2+KEidxJHfbfntjOrec1wVwnc9TmGXE\\\\\\\/MGOvDLgEtDYc\\\\\\\/Z161N8CXFYQ6GHqRb0SW+FMaCDWHEa6PGtT9b7qESDm7mffuv+naU0+Rci\\\\\\\/tEn\\\\\\\/qv8Ec5w5nhm1CrtG23Hlk0tJuJrnfSZpdPx1L6s1a\\\\\\\/BDM6N\\\\\\\/9UTldBQ8sGFeF7Sa+z3BNIVxzbIdYgDKPmjMaJtM7ltq53K9c1OLihXAgLX0sJC44R0unlqQxW6JA\\\\\\\/CkmrWP8yzxFh9+IwOZVkc7tgb5mbrxPIHmnYoWLXZgpCFMxUQy\\\\\\\/Pq2IuSVpBdNW+d++sBmRYQgVChyJ7NIqlws\\\\\\\/hhr4xQcH6S3r5YbM5+P5mgRhZk9Kxs+JGhSSjGBTIj5LTE06tEqPmjYWj8Cvq23fkIFyWoYQkrKvetl\\\\\\\/ahjjLSnngAbB7Hq6OzKXX\\\\\\\/yG\\\\\\\/SoDEol6ywITTpx+oHgvDcxshDcU7JDke2kNnoV97WBfWEfxQ4CZCxXuMNfLCTii5bE8KFzQAlQok\\\\\\\/ltqWDbUvPgRLehkzr3UfAbKcB3EUJrQgaZr1FWeunyKCJOGRYQ+UJ4Oiv24oxrIaVrUr1KWXQ9cPaiCd5VMRKZcGw6wsP2DivQIw2zDKz01LQruEVrrJ3CPAm9tuGgVCw2rAmhzin+g\\\\\\\/AiMYxrkIeBDh9L4MzMbsYPqpIiuh051HfkR+gjSNEdKI2RQR0RD+iGfBJ73wlpkm+D6\\\\\\\/d6rTqgXbfFCnt+90be5bbF6D5otvK2ZKWxUyXwAC8SQEToatKD\\\\\\\/EXBes+QmubWEFRJR2Vhg0Q4ogQn+wqZTkScQWJf0cPVBZ\\\\\\\/vn7lTTVshTtPro5CzJ8DH8a87xL3r29MYDGeC4OhIAvO\\\\\\\/K0DXj05khdW3rrwaMz\\\\\\\/S\\\\\\\/dmeGV7EO0YRDyHkaU5\\\\\\\/\\\\\\\/p127B9wXXpZClbemtf9+mndvXddVapd2XgdSMyFULsDr7SoSJfAi58TD+Jmtj2uDiLljR0srOXV+gvF8ASt8jq\\\\\\\/AU+NZb4CowOzMevDUraheSBLkXsRiUhqNS+RQHVAY8gcPhWmU6+e0AbgF3riZFXHmpakD6njBlCFpi2uu8OiAnDDyYNYDYjSlvlo1YfiPeRWb3QN1RgfPwvpsIwtIY2xQYjWOPSV66wTF8Kh+rKjTt\\\\\\\/HxgU58V7X07W9j5NqphGiImIPgA\\\\\\\/D4AcjLW10a8o9LxzqbTsxUsSN5AcliyLCKvRs6qO5OlrqU1+kXjyQoaYpvA2+Qb\\\\\\\/t2Ywj36vwrjYYOdcn\\\\\\\/CcIhmbmmgsBd\\\\\\\/Mw9zTHRK8x1Mm3sIQ3AwDCPCZbeJlVFu8fP7BCHgaI\\\\\\\/wsC9yI4k\\\\\\\/wIg0NK9U6Gz1NoieHMEf5wAIEOHJ7m0rZ2agCwyhoSXAOVurk3s1NwbIU1BECaQD8HhCwsAuX9Hjga+Hjv+eVZ\\\\\\\/cIRDTQRtRmvxo4EplgaOA5IR1FoYBSiQG2vyAFkkSR6mXPGYu5OsPDAc321ZfuI2hqRUDQL98PERUPEGMYQq+yVzlJ8ikb9Q\\\\\\\/HKN\\\\\\\/hs05s30abnuUE0kU8THgcIuY38cIjR8Qcw4WxGD8iPKIg+vH1wjprxcThUeE0pK\\\\\\\/OksNUJS2r0XGrJLrk+2Xfvsfh0MjEHrRNWiTgkjcouiL0ACty3l2rpJvGta17Mpll8KX+T4t7vrh5dCFTD\\\\\\\/r5PRAQui9zOi3tOu83YA\\\\\\\/Zy9pcQjKoVJ21u1s\\\\\\\/02euSaS8HZsBrS\\\\\\\/XefRkZGVzL38N06NeJSeRbzmTLBpb3p4uKTfhCwKhjip+fUlnlAUVi+wBqwPzJL4OE\\\\\\\/9y6OUy8e0c+DmDCgFyaInt\\\\\\\/fEPpEEqCK62rSzkjMo+irKVJzDSNeTx9EPGOjUbtz17szIbLArw1jeAFBbRxMbud20qf7xptFmd6OMSfpu61dxduw0FSjPTxuqxuxAKnt8s9MVoAoiwPwUtPe12V+pTDfhOd4ujSKJSgvTjMOAp+vY8inaKusvwqHST+yTVhMEgBhhkXxYHPLNVFWi\\\\\\\/PSNVkfdwra2n52jj6Ai9Q3skQa084hsXO8LSpcAMTWMAOfufAHasvtZDG2qTvYYucQNkyyQZT\\\\\\\/OHXbLw8G\\\\\\\/V3b8MhypqADR7yRNtSSAVb\\\\\\\/+847b7Sp+3ezxDokQtp7ELf38VVfiE9c1X0DCg06FZe8kAGvFfKT6G3aNIPqXf6voVkOAT3q7hb9bWN0S9iUDA+KVGQ8Y2vtn032xac7\\\\\\\/BsNrWqylwYvRGnFd9ujMCppIViS1rT7JPDLpn67LAZJhgEMbAdRkHCrBXjVHZyw4yPqUsOx+sedZfjkYH2o3Z40V01LcbK5\\\\\\\/vj6kddYtg2vjforPD2dOqU\\\\\\\/wqA2hkqBluo63Q2Bhu3no6cDjVL1\\\\\\\/tKcAF17oIbwOVRbYpUj4UclR5MwJirdoP8WZjoDoGnuCSNeB+cOvX7\\\\\\\/F7wBleX1cdNukeCXhOd1JdqZm0JRgnM+K9FlALC4L+qBeNgwUtg5C91698yIaq9LueLI\\\\\\\/3bbZHSS7AbXvgbOu4hk\\\\\\\/WdTYUhgANmkRNDEdA1Fjbsh2c+012wj4Ci\\\\\\\/8W6dL06L3lhO3jU3xqX3cdOEpIKFMOTejkZdb9bJ7vnH\\\\\\\/ApiBOlgq1O3oVq4cZ3knGg4\\\\\\\/I9B+zPMmlAlSO79S8RIDJ2aJB6fkOBE5pH4TSHxar2Dy3zPWlZg\\\\\\\/q3Or9ep8FEwCdaTPGc17YzmHj9ftunxiMdK1L\\\\\\\/jnCPpzlMHmbo49qQGyHhQ5Jz2mJ4obJOq\\\\\\\/BUVi6ovJ4eHvEie5YSoKwKtt3TcOpn4oqNAuNa9v4pKsjrshSXdI0UdkOiL7d6vF1wYVMMjMT7dpkpooRu7HZjLdQpP6ynW+nN3xfIKqpQt+efwYMZ7SUNYdlzfUR8KTiL5HPTXJsCMCqu9mofqrbKj30BXXbPXu6N\\\\\\\/Z7\\\\\\\/5N+cJ\\\\\\\/CIisd1rOKAt\\\\\\\/VEmwUjvzXyVHj6zSDcYnN0Xru0G9WjSAL92QeoXiSVXSbL6RBRYBUuVz4ZsPdRz1iOpuGGTHKbo2GOztY0BY8SSV+IFF\\\\\\\/na8jKjK6aUebgrf9WWxVHKKXleWO\\\\\\\/eyI7qEQotkkxG6Ywubw2vb8c2BLYdkgo9ohCnesua8NAlDxfUA+txuiAaY\\\\\\\/pEORkQdB+pgpB1q+U6OzU9MOvw4zu1s9aF0Ts06jM7U23xQX8uCxfVbFQkFFtqUlxXySJyHBgsyc6gczxL7UbwBnOWQf3SgVLeljyJPGhCg7TzGeo4eFh6KU2haF9ss3Y1TU4\\\\\\\/STPqnAtaEQDoJp5z34h1maGOtgBUtQ2XXUPxrsGUc3tS7SbIpiUeTkV0ZG8xbtsYThEJbzTw5Q0efhtLmc\\\\\\\/Og5LQ8uXBQeE0g\\\\\\\/XNtuLr9FWp5fZtJo93bS42wwnC5+5ZK79\\\\\\\/NJGS6sBiqDCC8WErh7wlS69qof701yFeK8EiFAlAaSQtbZYIGaLam0teDSzFpKvRk\\\\\\\/u\\\\\\\/O8+UjYo4spzU5adne8p4nIiOUr+y6xyEy6XABIsC6nzsay8kx2tRe0JGYgmGWldhUWtKmR3+DKSxdrnhn40nnh4GvAb4cS1hFTwx+24AxeyWI6gOR5yo5IBJLIVgnL3OO8aWjJ85r4tnwf9J2csL\\\\\\\/oRnbpfbszsn2eaBxc+m322e9CRs44GmZ0M+HG0z2aduG7\\\\\\\/VNJWamzeR72jgtytDBOwEIkbsk0DNbYhO\\\\\\\/h02Y4CnVvyCR6Vyeppq2IPJudK9XmiqY3FBvZ4mu\\\\\\\/uDCzQMJMQczkhHGJuC9bN2BYSV5dJ0HcO2PVojgF+WWXlVUkMKJKbHxp3HY6MS7SpO1wqYLhwtpz0Yn1P09E+kGAIY6W0U9VcZ7G9fPzu+fsVEr4eaqNbboy1sFhm4mdq1HIXAwAYLFwgo7TrTXVx0voean\\\\\\\/sWEo\\\\\\\/k74PRDEhzAl8PtHQLbKkh\\\\\\\/lUpa2kiM9I7PToNNmjQ9YIJDodCup\\\\\\\/5U8B8ITwvayxlwj\\\\\\\/AHFcLpoRRVihm818f1DRpwknTv17evVzDC8g7Ko1WbjbCw\\\\\\\/4WJJ1HPNrSmohN0lcsZzu31osOvkxlX6YiGfKqaOwQv04o6iLhTXQ2xmJTMXyKYJaYmhJOqwT4DAnggmH9RYtjX3NqdhYUw2WU6qoSHQMsvbJD21GAgD8OdtXUMYyYQjep++CaUvvDM0i\\\\\\\/XgvHiHRt4NZhO7yI7GkRooGtYMbI+k3xXfHqXe165wfY6d4HSh1f\\\\\\\/O3P7Bi+9CeWc9eNHpvOZ2uI5ijZE5GpIclYz6cHzr5BZCqkSFYyipHsIo7LartL2pIoKI5B17qz2JRUaDw80PLq3F3KYeJEsoQJKz6DiazZCiBVVt4wuxj\\\\\\\/QfJC2aWa7BjxXzObxRoKtbvlvy2O7wE\\\\\\\/xAfFIgklDH60rNt2LRw0xwtsAh2C3NJnbe4Fxr7Q7p2E9l2NUNvnWwfATHdXZD66S3MY7aJmnqLVOPbTa4VH9D+4T0hS\\\\\\\/DsYcpZV5EAeRfXNuWTAO8a\\\\\\\/3e6AZP+LDp6bE6vZwRDzNMHsY95FBxWezH4yoE2hXXVneiFRZZ1yi4tLUS4\\\\\\\/VOKIEDMYkdMx5vgr\\\\\\\/e6X3qpTSJQzr\\\\\\\/u0y2h\\\\\\\/lX02XbdqPqa10Zz8GBRAf2R0AzXKzCq+TaRAZtwFcFGPBdzHU8nI6qf6f8y7z7PcvHF3vlTf5iTGKDudcY0h59ynAVtkLV4Y4Qy6r2GyHQu3uJLPy4FrbcdmAlDc2+LilAxN+ujOhNdGhaaETc+xYuqkYMk9MlRO5tDwUjjwfNDfIyuA\\\\\\\/qitsC2n7QcWGrWKyNLPkrk53Y4jvcGkk8Fcc0lYzNquPDZfVJ2BDEZTqVokWck+QOOP5w9A71nBHa54e898CkXDvCWTrfxey1ZyDBKKldru2AT0GFrJDeCzp7GfJyVYmhf+U19WKktLVzaljuxALM9ZeMUr75kAfCs6HGFiMfHRD2wyc0ta+90MiCfS6uKehRoiW+a1ic5zpMsYWZzyEOp7Hd8fHlZoiWO\\\\\\\/kFNTH+H3E+nRs3CcxB5uqhbBctCoXOaB9Y\\\\\\\/qcsdmNwGZSj1vBSFcjCUY8KzZ\\\\\\\/LGeJmSB7R19oudsUdupPRqcO2UVwM138RbymkeMs7M6l5s9oEAmghoLwmoEZfajMMJ7yedipHT8uSv0+g9nlAdIBjLiHxrjT8gT7an5POes0LlqjZxua+M5q+Sx2TJV\\\\\\\/7lQZzZXSUZe0hBv4f8ca3koJXJ9JLzOsMkOZnIHeN0GGx9A9JckZQRa3cJhd3kf7yUib\\\\\\\/7xBE+FYwrOjWbgvoy+vTbYzk\\\\\\\/ml7ID6B3uYBDcYFqftt9kWrIPW4uP5Um5Y2cSLz1Z813oT7QMiiNpES0ZUq6qZC1S0ISy3HdOs1W80YH6lS2WMl8\\\\\\\/EZKrknEXtAd9QinwhyftD26H9evQd6NYzUCS5NkF56NJdqdkIAXdB288GitoRiC0gzMZAAFmWn4\\\\\\\/rZtA+8M8gHvC\\\\\\\/WB+MOJYRjHMcP3bfxJDSmbX0v3eDT3x3UT9QtVYCKb4fgEqFwjnWiYZ0o4ach+Mm71gQbg8phaXOedSOup4xeenMI62S5QU9okyoKcnfCSMbIom\\\\\\\/A6riux\\\\\\\/qFRsYfi3F+9BJ+3m2Xk\\\\\\\/ZqiJy6ujJOWBRcI+nxGNWR3DNXp\\\\\\\/+SZZ6MkEFzmfdUNxsRURBZXE1U8fOvRBmAMTmXiZtsDqq7h5yNXDvyfZgEtFbrnaB5u0Ussv7yljP20cV9FP4Z+Tx9DKoJD1eB5iaq0aDEBMsoNVKm9mzq2507WFHlJoHhb+VqipeLR4xKeFYVzifIH652Yq\\\\\\\/7KiHNLnsZZwsPbofVJvk3lk6N65de6fkr1z2iUZxMcNvmn\\\\\\\/iHaiaTnctE\\\\\\\/xme0LfEW\\\\\\\/0ST7+wmcyEABPFxLNqsEbzE4BouRZYTBryaKJEVr+ybZeHYPZ8Gb1xqrCAaKC+CXfKsrx18rk\\\\\\\/HZb3BQL37lQMvF+22Ittucz+R2ZNuX8K0BlBAq90qgc286nektBrHHnfkabG6YSZcWQdNdIVkxGWb1lcKZlMErVFx8vTQijOWmpbKJcyS8Tk6eK\\\\\\\/bFLHQW0jx++uyd6dUK4rsG6\\\\\\\/Kfxb+GX71HS5Q\\\\\\\/CfDBHGNr3beV2SnTeF8EOxIQqmdujjbGwAOrb0RUOJa5TJaEbxIv7DbkoD6vnZaxYG6Tb7rBJXo\\\\\\\/ex01T2h8nLGo3CSh3Iy7E4htHgG062Ar2OBrd4nsJ8J7i1ZsuhF1C4Cblf9x59PjiC1ng\\\\\\\/CD8vX0wee6iBfEkxX6a\\\\\\\/UIsnlMCrdDeHXLA\\\\\\\/k3Ip133vbhZ9CpE5OHzXPegyRbXABgbEOFTmr\\\\\\\/0ssNIJSpBQvs9mv8PuenZkm7Hg+3fKw\\\\\\\/JLkVHzkRHmyIkj826QO1JhncD6D7FZCBeuqOwTdDm0yUxi\\\\\\\/2\\\\\\\/z901sxIAc48UszDV1c1P3gnEtxckoAvAYRVyaVfNHez93cas5GcnbSsdwnu8gJxIXArl0grI22Md05DE3WzR2PMEHaMZzdC2i+ixPJTurM1aWH3uoOrj7aqtDJfpIWF1IXYxVuAe8W4GzIBxlNf28cv+p0N3+WIxPSBRvSmrzB1E19jV2PHajUevS+VyHTFxznux9L9DAvUetrUw9CBz7siwTkXXM3U5l+CxTJvShe8DR5FT\\\\\\\/pLD14V4qgb6pQKFSCPXbOETO2ZHUJAKTVKYknJ2oAW8fg+rWBSH54dPOqf1rLMoXfva1GFA0nahqRTYs9F3nBE976eNDofW57oddw1V9iBBcpVhQ94NaBZ9gOATghAqNhvnXNhEHYjD4S54uXIYV8FS\\\\\\\/V9W4Sn5rrHGBejLT0k0aefmIsFBy2NfQ5Gbsrhn1QbfXEtKT1JjVowr2kjmUmoSnaqgufTVVxjqa+wSUKgVfkPSN+i6G51rPaAV2zkVkY07RtnRFQZ3aL\\\\\\\/zALQ5Z5Ad07hObtsVO\\\\\\\/XjfqM78bh8J8vM9lPO5H1Q5i7nGtAOaqrfaDEzm\\\\\\\/uVmJdXfycj8\\\\\\\/IFWHRiJJ5bOSJIzKHBD2\\\\\\\/4vOjvISCVky8cCGOxgqoueYEp4vlR71h16D3h9U+mAsTwJD6DPzi4INdl+cejoTylUOlRTmUPEcEuj\\\\\\\/rNuk+\\\\\\\/\\\\\\\/kmtVikIcJHZELbBODg9ub3MzXXqtJB1hk2y+piEPGJXb2jn+TNcQqoJwPFjzgjwF2sGsGsvJvEkPlRBzgCi1Ud9Tgi9ZB+hQjZjRVDEy9KErAjMuNxWCt310csJdEDhfKbE82qpm0NTEQoD6cEjs+3s+lnG\\\\\\\/ntvsj8co9ES1sT2nsQtgK77KQgEMV3cLTs8fAjtcGEplaCEfzw3O4Ze7L9ZX9o+TOo4dVvohADxjF8xv9zeR6fMimSU5PuZB1+LjOxMXSdA+G\\\\\\\/pVMhPAduNghgJDC1n9IhjMUxYDndIjetl9kZPvEZ\\\\\\\/ieWdVU37b\\\\\\\/XmGWEh3nwCupkLa3HJK6EajPfniTjtNgRtP8Luxd90M\\\\\\\/1ehdChp80x+oIFmU\\\\\\\/BcaDXZbGfCYVKAgwzBRzVIqP9JmxHHHhgLSVjgnWu8dX3IM4usYV581iAbg89eS2W2SQAs8cgpFyitVFgDPowElnYmEp+o99iVqJVFnAGX+ZdoCQ+ojIraIm+XnGgM3lFFO7MoKPn1LPzOfgFE4OK9uGvswLAiOzCDf8bDiD1gYY6euP4p17d17l6Qitx5fuqC+SzKFE7kh5olZjFpWhhrn6nXx4eU\\\\\\\/aJp9JcDPLZM0GiMUP5voYX5USmcClzLJFDgBqKVMEynRO53mBeaULdvOhf+JoCJGy15xP7bJviO0K4WVXm1vRm6rI5pmdKNyDPYmEEXkp3eZERCEsJeXc7KFFzfErDs7pMJD3WHBm4PL1HAd\\\\\\\/pj0CXMUgJRq6fndD\\\\\\\/uImawaL9dnPGw6dOUVPPTm\\\\\\\/zqlevE8wqXNz1BYAqWHlud3VkLiX6n9EbTZbCc12SF91j+LyJwihF+Zo2wOMGJb3DrH5oQjRU6igMPOVzR8zqoN4ZWgYAoLmel85Wu4ndioRB0JA5TR2XSiEk0fkZrlwp8eUtv11kJRVaJeh89pmNt3nH3vYJYg7OwujluUQt+JrIvP+TeBZy13Xi9jSxNKyoKKpGVoHO+L+JYZYgXdcDKUia3iFo5tF4VkeL+6EKh2+3uP3BIUe45CDlQkOUsPY5vPmBtkcluwrpUgWgf7zphqAXMlLLdIb47zPrms5G3Qo\\\\\\\/r69KIwRvpdLrVPVbdN9jEFtUFG99Ljd79lR0Er50uxi8CreJDZpMuUn\\\\\\\/or5yuNFn2Db6qs54mxsrEAKpFO45ElLNbJbLyuRLxP2U3fDB8Jr9kW8rYp5Xp4TNlZPAmofFOAxKS5uGazSdB2AE2P058VbMmZUvbmH468ugyQwntSJ\\\\\\\/b2kTze2koabGyy1fpvMq+4yNk9ozlryWB0uJXpG\\\\\\\/F2DxX4F4eTZqzVGYlY9wmlYY2ZxyysctMiJ5BVUGlkluv4sNm67PRDiyC4zaYEWDPgXTTIHOb6s7OTekE4LjLERmMfqe18E82nfKxzT9FjohxqU0CvKJjROc1syszgMTS2pp7hhzL6EG3IbUh+boj8ajSAFu2EuXwPIRk+dljwE95Q7Co7Zj7643Qg0RTIRzuLreNZKAC\\\\\\\/h4lOXUbObV4xdlCigkYK8l8cQip6Rx9p6h60Nju1\\\\\\\/vRcvHXfW1+gmQTpSSd3KBkCiLCumvm\\\\\\\/8fsfybNO72YnJ7wT\\\\\\\/fK3B3QGT4xJGHlrQiwBdGCauo0CzeAq92YViyqN0lBz7+Opv9zG10qD7P9pPdZ7lxJXWbzztl2HkM\\\\\\\/\\\\\\\/sUOj0IRWt+\\\\\\\/1LJ1YV4IRZLW5Kgi8is8zKSDSaRhFPxhXOnjxzUPxy6hWJYAUmBnBmIyiJbZmAqt6BU3GBK\\\\\\\/01Ky\\\\\\\/bdDAMmKrM3zeEB\\\\\\\/fPkKuIRa7w+ji0uhKZlIxwbXCkl1IHOsoKr\\\\\\\/2XfMaZOTCkf3r49JNSD+I\\\\\\\/K7g7RIbu3r66Na83xAbQEOBWM\\\\\\\/9lKfhoDzrqpDhw9YVTaIS\\\\\\\/Mq1fro2gg5lYgPlbNb8fnSKCybq9yA+n2K+xGZjQH+BwkGONlacvGy8Rp5D8RhRVAbJLlx9Pv5UDH2qnCmmMObtJRMz7sCfnxX2l0RDVzmD8xSvqvP+jIWCHj3+g\\\\\\\/vxMQbyxgXDFVoBgt9oSKRz1GLCes8kQcvFmYsfFhqe9bPFBgM8NtrbtpGe6aoqkVMXfk138y6JhdqGzWTUDVgQHenvVYrxzJ3hb7sGoArxiWIQ9LQarvcFdXlKVeoM8piiqTKB5dRKhcIHUtEcZtlFjIJLoZJEe5P4oCQGVytG3YpR5kX7ll3SMpv98LOqKuA1b0i\\\\\\\/zfYQhkoeBDVrpCN3pgEgOThpcFYJn86bMtvDjf9DoTL4DLg8S2uCjYw\\\\\\\/Q9bWlBay+qVxvLv+Z7DeuUILYWZsUIqynT9euM5uoz8kj50COpMJ5+d9IXr7N4AVbMgG4aOaojI7Ekp05clP6OeBuVJZYPKphj0B0aO\\\\\\\/hoGYQRT\\\\\\\/onx\\\\\\\/dnnbmoaahHa+kJ4SneLYsVAlPbOt6OnqKXERmqeulNfuJdIDrowdbDiSZwwcpD4+YC0OwU5KyHOgK8PYP8cGC+xuJDWDkw0w7CZohby358Hy3ue\\\\\\\/A2nknURWjQmy9pJ5dJ9UyjPP1xb5fypW2UukcxkR1hDqfZjyNKuU9UOtoHRSBEKzZIAED56ElPGfWLBVmUgGOSiI\\\\\\\/nY\\\\\\\/VFy\\\\\\\/KBjjJvK5nnplalAJ9OTxnIvt8O04FR6su94kdbNr193Xs5CEGK5ALO6bv7MMKMf3IxXQfz\\\\\\\/c3k\\\\\\\/P7DJhyKtgl+PJbpK\\\\\\\/Mem+4Qg79wqUYMYOMY6Jun8lhawWoPKhIc0x\\\\\\\/fRDwY2BkQ1yul0zDiTlJcAmJTlDLOFlzr+b83PkTvXvPAbgiB05V77JxMLVfwRuMuCkSiS5VxHi2ITtmRhbn3NtQSbEejxFmFD3mXAMjBtmCH9SkZpz3WN4tfu3wIuXh49\\\\\\\/JGIx9kN7+R2h5BOAO\\\\\\\/dG9q2tFcqbN2BXsoFrkvZoTTuhCDVACj\\\\\\\/BpcElXL3fVyfgWBtUgfWqoUd94LRDktTDczEBMdmMMytkC11KueWHOgWrOvem0rzRlzxfSSi\\\\\\\/74AtvqpTcIyrLCazKiOfikVHXxCJxDa6BW+5OGu5XQrS1rbFtI6DGUFf3xIbv7sTn1blSgheoFvoXUNF4gnCpxJWhUNGiOaV+wQg9p+VvANlvX6rvsn2hx3ko6nWLoer4Va+FDbENB6\\\\\\\/NMV3YkMSwtUQxhrEFVKbAv\\\\\\\/pmhqvHI\\\\\\\/vTbLUSkatdOJYEOHE1ZJifQ1FKcZJLY9ArltOgZA0+lckRG85mQTh8zCoRY2xZ5tBN9VfOjxkrGWQOgRoqJW8sCGFQeYQHojqJM2Wx\\\\\\\/I6BPUQ6zR2WxarsO0fk0DqYZdlsuU3mG8MsbzW2AfIFn0\\\\\\\/8rNzjIjrSYFqvh8F8m4Y7yL9gxEzRAP6XCfxirLNW79FV1jy+fRyQHmJl0bZNjKGqOG4lxntGdjJu9Iv3QnRXERI9GqD+Sd\\\\\\\/pOqFXmtBnG0AkgozsZmS+ux47AdxsvlEL4FtOnfh40ZS0AnphbtuTbhAaBrJcd3hMhAmigDmS988\\\\\\\/TBSZkNJntPsRcRYwbJPzK8cbuuoe1KNVCUUGH1ZZworBBNnjp0bONGsOgy6h8Jlv8C42ajUSZwCbXmp0gHZofhMzH0vz3d4mF+kyHkyyjP8k\\\\\\\/L5m9LpNLqKmirm2I1\\\\\\\/ZKxdvy70Q1r24ErJKzQlr+Zq1HnQsUXus9uDyvxRh67XMVGj0sq9YTxJwYyXzZqqTm0eacQ+MrbM4hmsF\\\\\\\/1epzx\\\\\\\/qEBHDK14LAbCvJ8EqPZJ\\\\\\\/HaF68IcHVhxhTOKOSR3kztEcbIKSJQWeLw7AOa7XdaqgpKaaUbpClS5HQ0\\\\\\\/EigwSePwTcsSLA2SUkoUVA209cRNwiDIqAexECsGc7Kg9wVmx7zghKjv6i4Dal\\\\\\\/ooRXDzquCddyPCgAmBzlpdzfV7u2t5buZ0YaSpRpTEyC7Kdn6t+CO28Zv5b+N08ykyGKWOsRXqqYzP8EglrE6yvadtJjQaCicTiwPlvt+C7\\\\\\\/hUSxWsAC8ZCEvm34e9ccU+MUmz1NntekZhUIo\\\\\\\/W8cdfsMSD4CvqtzAGSt9LLgF9gFWteyBehdf4\\\\\\\/\\\\\\\/FAkb7TLYA20AaDTwLhSIvwg3vSMGXLHuiR\\\\\\\/DoQnW8sU3tj1zlDY3Ug7Z02AIy\\\\\\\/gXM2TQdp\\\\\\\/V9WJI+xpf4iDCLZgJEQccc0ZRC0L0HCzewDx49DEvq8rc2XddOxTf2WVOb9nxlv7ENHXmYrMhzTJ3d6TSxziwGiGKF7oCbrrVMQc3Ae5VL9e2FqhVUjuDxGucxLiwLlbNM0u+WxRQiqdtW7vcgNwigNX2Yys+LkI6GYfKK6jg6I1peWg1iNQGAuphGCzsLa5UP8iaGHlFs49TkGcg9NqZjwfBwSbGW9rSVR+QT8qP+Rpv6vnM\\\\\\\/egNRVCdy56Zpf2kLTavYl7rOGgqfkW+I4KjDT3Ikvcqk9Fiu4tkO4gxljQy0VvbN9avs+yNa0PgmrMX2JUU5u5AiOb1liCJQClvHiQkn7Sqzr2AL3A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022AyiHThfxheYTk5jfjW9gUA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ecc5153e3ad70957e94e7e07042340db8722b37729ede87cf3ede79d935c0e86\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227hVTZl3APfyRTqm5GBbhtZV3gByh9PhAAlGXuoUvFXmkWFPqsxeLp\\\\\\\/76ByaBRmprN0dGCQavxPp6o3oEgC69B2\\\\\\\/UBmyaL3yWR0SJ\\\\\\\/uupSvxVzR7ga3uqiSB3Ha6sq87\\\\\\\/ajH3XqtThlLLQ08Hf8kTJcoN3V\\\\\\\/74Xz0dOh200XvPo1PMT6PBqOpSMAZxerwVu8pzUh2ggTfLdc8upbPYQjczlHOS7lEf+Plzd5dQp6BsNTW255zXyb22KrxZSgkugVIAhynul3CDm1KATnP6KKmJ3P8neFlaGtVkTIzckt6PRUZkenlLi66FoBLp3FDetlly8AHHnEyNQjT61PaAQhNnBnx4l2sXMNrrel2IckxmQZomBx242cs\\\\\\\/lmx60Bk4YcNfp\\\\\\\/Jn9BcFy+01JLYqIF7CmIKr4blbo7ogsp\\\\\\\/Q3SbdP\\\\\\\/ONgXV7fbSImdF+78Q6hg1c4frpXplE8y+FOPaCQdW5KjpXLl1O5dAAE6wCCfVQ5PJkaurB1s0CuqB0rdef5hrg8GQHTRK\\\\\\\/Dt9SZO\\\\\\\/gOZcbVrjbcxojjLT8B35ohN8WfFUPy3X7FQKqpzunS6pA25VBLe5I1I0TVMT6RYz0sXJPz5uu2hOSjVrkCB3atN3mRB+jBOAZEVmRtLiDk6qg6yB6XUJ1I6pVFFTCDwqFj+JS1mV4eGUFvGFrWVGWtSivlH72YXRdQA6dwjjjf6NRLUEZ2IFe131QEBbSP3ETAERG8RztckI\\\\\\\/9xSslPDJfQfonce0bZozL4Fb9ngKpDJuEkJt48dRulOs0v+HnxWacLl0PoCI7kdNmtyzYBAxsbCPEzZenXSwzoY7QS7zONO+YMxh7L3hcr3YTdB\\\\\\\/vR\\\\\\\/QsLkAgzWYhLsbTG3DB\\\\\\\/ffwKUoIpR7hGbBI+HrIJs7BShO8cXdjBKD6EBRAbkOmWZg3P93xHf31i8xJwsYYbl+z8tEpE8MOcL2XbHwgslUeBL8NaImALUOFQqqw9fEH6tFW6nZyOeBuehrXTiyKMEe2nqc2p6HsswsiEiaiJjeB5pNeLHHQsnq5gk8WwX+RN1o1QjD4GIxsSbVpvNiAO1DsauXJ4fmootkPwWESuPT9kzN+QKrXwOZwTJ0cVG7LhMqM8aqYUPabI8KwINTcjwLVBl+xhotKKvWVX2ValPqlFFjGqKulZ651YKJUuVkcjGoRCehRYQHguXusjZATE1tZCFakrREvMfxo9eNPlzSHQdE+wNGeG8UMSa0rQ6FngbKZV\\\\\\\/8vEsNgoaAlvR8kS+jjHNWZefBGtm86JKeMjo1wnGEbGPuQ8zp7IRPvt6TPjs2tPn9yrtRHr\\\\\\\/0lg8xrYVJp2OKHBGrqcjHfbTx9mg9zBTzpp0wDChkUigrxjvVdXrIpaRVjpbdu0RaRqU6Rfo+GeL+A5qwGsbiEFK327I\\\\\\\/H0f07YrVKNhdn0bisPKvthjUHUCluNHT6LdykKBfs4SNUInR2t1D6xFkwmWJOhr+Mhe1LgdNkiimzRmVKZnhQBfC+TpJhkprAH0fV5+JeocnUVTAoh1cM1c\\\\\\\/h61d\\\\\\\/TNYLvNWPtDfFPQ7KcH+jc4ngrbJSN3En3v532H47YscqARGsqxYUiGcmtvAOzZy4tLX\\\\\\\/wIr7tyYMD7appzi0vYe2BDH4Xree7miGw61y1ANX9xJIxWq4eFhaGBleZfHu4MI3uGUEgyV57G0CTfliiJusK+YqM0UqonyHDQGsC\\\\\\\/ydFh7OHM6qMg11yaRiaeHW8uNJk3p+jlp7kfws5hDV8E99ds8qej9E3seO4h5Ook9n0HO2V0+ISq\\\\\\\/XPzwNrHuMnL7oB5T75nwv1lZG38Q2mk7IBTCGCRHtSMiHZkrcNTRiqHNfh6kfz4uaoDKw3qp03Lj4Q6TcRn\\\\\\\/NLTZ\\\\\\\/sTfGY2TOoSjRZmZgJFAUgnz5e3R7+d2xq\\\\\\\/Io\\\\\\\/c52G\\\\\\\/eggpkyUABFBi+F0OVoSeV5TX0kNCC4xyriTehcXMY8xDtiIm5GqeDdgSMkHsOnwbae4OcsnjEbnNRKVT3B3IadE4kla6G+b49JmQTQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022NUOkrYV\\\\\\\/nmbcIzmgL7lUFg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022224045a3b3e5ff523de6092e4cb83e229c70059bac98cdf2cb91d058d5ad45a3\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459608 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/1/29" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JfYcd8nKI0AMzAK4d0LrMiU-vZIzq4xle3cyQSCIcEY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/1/29" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IP8sueOu1gqPHJ9Cr8uV6lCevrMk9mXjrUzsFwGO6Ik" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/1/19" title="Origin and Length Distribution of Unidirectional Prokaryotic Overlapping Genes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/1/39" title="Chemical Suppression of Defects in Mitotic Spindle Assembly, Redox Control, and Sterol Biosynthesis by Hydroxyurea" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459608" id="node459608" data-pisa="ggg;4/1/29" data-pisa-master="ggg;g3.113.008714" data-apath="/ggg/4/1/29.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Density Gene Map of Loblolly Pine (<em>Pinus taeda</em> L.) Based on Exome Sequence Capture Genotyping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Leandro Gomide Neves</span>, <span class="highwire-citation-author" data-delta="1">John M. Davis</span>, <span class="highwire-citation-author" data-delta="2">William B. Barbazuk</span> and <span class="highwire-citation-author" data-delta="3">Matias Kirst</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">29-37;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008714 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459608" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Leandro Gomide Neves </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Graduate Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, Florida 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leandro%2BGomide%2BNeves%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neves%20LG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeandro%2BGomide%2BNeves%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John M. Davis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Graduate Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Forest Resources and Conservation, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>University of Florida Genetics Institute, University of Florida, Gainesville, Florida 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BDavis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davis%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BM.%2BDavis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">William B. Barbazuk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Graduate Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>University of Florida Genetics Institute, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, University of Florida, Gainesville, Florida 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BB.%2BBarbazuk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barbazuk%20WB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BB.%2BBarbazuk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bbarbazuk@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">bbarbazuk@ufl.edu</a>
<a href="mailto:mkirst@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">mkirst@ufl.edu</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matias Kirst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Graduate Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Forest Resources and Conservation, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, University of Florida, Gainesville, Florida 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matias%2BKirst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kirst%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatias%2BKirst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bbarbazuk@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">bbarbazuk@ufl.edu</a>
<a href="mailto:mkirst@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">mkirst@ufl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/1/29" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459608" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459608/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/1/29.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459608" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459608/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/1/29.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459608" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459608/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/1/29.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459608" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459608/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/1/29.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459608" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459608/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Loblolly pine (<em>Pinus taeda</em> L.) is an economically and ecologically important conifer for which a suite of genomic resources is being generated. Despite recent attempts to sequence the large genome of conifers, their assembly and the positioning of genes remains largely incomplete. The interspecific synteny in pines suggests that a gene-based map would be useful to support genome assemblies and analysis of conifers. To establish a reference gene-based genetic map, we performed exome sequencing of 14729 genes on a mapping population of 72 haploid samples, generating a resource of 7434 sequence variants segregating for 3787 genes. Most markers are single-nucleotide polymorphisms, although short insertions/deletions and multiple nucleotide polymorphisms also were used. Marker segregation in the population was used to generate a high-density, gene-based genetic map. A total of 2841 genes were mapped to pine’s 12 linkage groups with an average of one marker every 0.58 cM. Capture data were used to detect gene presence/absence variations and position 65 genes on the map. We compared the marker order of genes previously mapped in loblolly pine and found high agreement. We estimated that 4123 genes had enough sequencing depth for reliable detection of markers, suggesting a high marker conversation rate of 92% (3787/4123). This is possible because a significant portion of the gene is captured and sequenced, increasing the chances of identifying a polymorphic site for characterization and mapping. This sub-centiMorgan genetic map provides a valuable resource for gene positioning on chromosomes and guide for the assembly of a reference pine genome.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/loblolly-pine" class="hw-term hw-article-keyword hw-article-keyword-loblolly-pine" rel="nofollow">loblolly pine</a></li><li class="kwd"><a href="/keyword/exome-sequencing" class="hw-term hw-article-keyword hw-article-keyword-exome-sequencing" rel="nofollow">exome sequencing</a></li><li class="kwd"><a href="/keyword/high-throughput-genotyping" class="hw-term hw-article-keyword hw-article-keyword-high-throughput-genotyping" rel="nofollow">high-throughput genotyping</a></li><li class="kwd"><a href="/keyword/high-density-genetic-map" class="hw-term hw-article-keyword hw-article-keyword-high-density-genetic-map" rel="nofollow">high-density genetic map</a></li><li class="kwd"><a href="/keyword/copy-number-variation" class="hw-term hw-article-keyword hw-article-keyword-copy-number-variation" rel="nofollow">copy number variation</a></li></ul><p id="p-4">Loblolly pine (<em>Pinus taeda</em> L.) covers 11.7 million hectares of natural and planted forests in North America and provides 58% of timber in the United States and 16% in the world’s (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wear and Greis 2002</a>). Loblolly pine is also an important species for comparative studies between gymnosperms and angiosperms, and genomic resources are becoming increasingly available to enable these studies (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Mackay <em>et al.</em> 2012</a>). For instance, single-nucleotide polymorphisms (SNPs) and microsatellites have been identified and applied to generate genetic maps (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Elsik and Williams 2001</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Eckert <em>et al.</em> 2009</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Echt <em>et al.</em> 2011</a>), identify population genetic parameters and associations to phenotype (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gonzalez-Martinez <em>et al.</em> 2007</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Eckert <em>et al.</em> 2010</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Stewart <em>et al.</em> 2012</a>), and develop genomic selection prediction models (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Resende <em>et al.</em> 2012</a>). However, the number of available genetic markers remains small, particularly considering the large size of the loblolly pine genome.</p><p id="p-5">Advances in high-throughput DNA sequencing and other genomic tools are making it possible to genotype large numbers of individuals by sequencing reduced representations of the genome (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Davey <em>et al.</em> 2011</a>). For example, targeted resequencing after <em>in solution</em> sequence capture (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gnirke <em>et al.</em> 2009</a>) has proven useful in variant detection. Using this method, probes complementary to the target regions of the genome are designed and hybridized to genomic DNA for sequence capture and subsequent sequencing. Sequence capture is being optimized for an increasing number of plant species (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Saintenac <em>et al.</em> 2011</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bundock <em>et al.</em> 2012</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Zhou and Holliday 2012</a>) including conifers (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Neves <em>et al.</em> 2013</a>). To evaluate the potential of targeted resequencing for genotyping in loblolly pine, we used sequence capture to detect polymorphisms based on the segregation of alleles in a recombining population.</p><p id="p-6">The analysis of a recombining population allows for the construction of genetic maps in which markers are grouped and ordered on the basis of observed recombination frequencies. When the markers are located within genic regions, the genetic map provides the relative position of genes in chromosomes (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Drost <em>et al.</em> 2009</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Neves <em>et al.</em> 2011</a>), which is particularly relevant for loblolly pine or other species without a reference sequence. High-density, gene-rich genetic maps are useful to examine genome synteny with other species (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Krutovsky <em>et al.</em> 2004</a>), for detection of quantitative trait loci and identification of candidate regulatory genes (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernardo 2008</a>), and to support genome assembly (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Tuskan <em>et al.</em> 2006</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Sato <em>et al.</em> 2012</a>). Currently, the best-resolved loblolly pine genetic map was reported by <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Eckert <em>et al.</em> (2010)</a>, which positioned 1635 genes, genotyped from a starting SNP panel of 7535 markers. More recently, <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Echt <em>et al.</em> (2011)</a> reported the identification of large numbers of microsatellites and their use to create a moderately dense map with 429 markers. A subset of 311 of these microsatellites was derived from cDNA and, therefore, reflects the position of the corresponding genes.</p><p id="p-7">The goal of this work was to genotype a loblolly pine mapping population by the use of sequence capture to analyze the efficiency of this method for polymorphism detection and to create a high-density, gene-rich genetic map. The mapping population consisted of 72 haploid megagametophytes extracted from seeds of a single tree. In pine seeds, the haploid DNA from individual megagametophyte represents the product of recombination of the maternal genomic DNA. Sequence capture was performed in each haploid sample and 7842 markers were detected that segregate in the population, representing 4195 genes. The best marker for each gene was selected and 2843 genes were ordered in a genetic map, with an average of one marker every 0.58 cM. Sequencing depth data were used to detect 408 genes segregating for presence/absence variation of which 65 were ultimately mapped in the genetic map. The comparison of our genetic map with the previous map of <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Eckert <em>et al.</em> (2010)</a> via 397 common genes showed high levels of agreement at the grouping and ordering level between the two maps.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mapping population, DNA extraction, and sequence capture</h3><p id="p-8">Seventy-two seeds were collected from the reference loblolly pine genotype 17 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kayihan <em>et al.</em> 2005</a>). Each seed was manually dissected to extract the haploid, maternally inherited megagametophyte tissue. Megagametophytes were lyophilized and DNA extracted using the DNeasy Plant Mini Kit (QIAGEN) according to the manufacturer’s protocol. DNA sample quality and integrity was analyzed by agarose gel electrophoresis, and DNA samples were quantified using the Quant-it PicoGreen dsDNA Assay Kit (Invitrogen). Libraries for each haploid sample were created as described previously (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Neves <em>et al.</em> 2013</a>). In summary, DNA was sheared, end-repaired, adenylated, and ligated to Illumina compatible adapters that contain 5′ barcode sequences. Fragments with ligated adapters were size-selected (250−500 bp) by agarose gel electrophoresis and enriched by polymerase chain reaction using universal primers. Libraries from eight barcoded haploid samples were combined in equimolar amounts and used as input for the sequence capture hybridization reactions following the Agilent SureSelect protocol. The probe set used for sequence capture contained 54773 probes representing 14729 <em>P. taeda</em> unigenes and are available in a previous study (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Neves <em>et al.</em> 2013</a>). Target enriched libraries were sequenced using a variety of Illumina machines and outputs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008714/-/DC1/008714SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008714/-/DC1/TableS1.pdf">Table S1</a>).</p></div><div id="sec-3" class="subsection"><h3>Sequencing data filtering and alignment</h3><p id="p-9">Raw reads for each sequencing lane were separated on the basis of their individual barcode using the FASTX-Toolkit barcode splitter (<a href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</a>), resulting in one file for each sample. The 3′ ends of the reads were trimmed to remove low-quality bases with the FASTX-Toolkit fastq quality trimmer with parameters -t 20 -l 50 (remove 3′ end bases if phred quality score &lt; 20 and discard read if resulting read is shorter than 50 bp) for reads sequenced for 100 bp or longer, and -t 20 -l 30 for reads sequenced for 40 bp. For samples sequenced in paired-end mode, the order of the reads in each mate file was resynchronized with the use of custom scripts, and reads without an appropriate mate were treated as single-end. The filtered reads were aligned to the unigene reference used for probe design with Mosaik Aligner (<a href="http://bioinformatics.bc.edu/marthlab/">http://bioinformatics.bc.edu/marthlab/</a>) with parameters as follows: maximum percentage of mismatches 0.05 (-mmp 0.05), unique alignment mode turned on (-m unique), most accurate hashing strategy (-a all), and hash size 15 (-hs 15). Multiple alignments for the same sample were combined by the use of BamTools (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barnett <em>et al.</em> 2011</a>) into a single alignment.</p></div><div id="sec-4" class="subsection"><h3>Detection of SNP and PAV</h3><p id="p-10">SNPs, short insertions and deletions (indels), and multinucleotide polymorphisms (MNPs) were detected in a combined population analysis that use all individuals of the mapping population simultaneously with Freebayes v0.8.7 (<a href="http://bioinformatics.bc.edu/marthlab/">http://bioinformatics.bc.edu/marthlab/</a>) with parameters as follows: SNP probability 0.75 (–pvar 0.75), pairwise nucleotide diversity 0.01 (–theta 0.01), haploid mode (–ploidy 1), and accepting indels (–indels); MNPs (–mnps), minimum alternative allele frequency of 0.8 (–min-alternate-fraction 0.8) and minimum alternative allele count of 1 (–min-alternate-count 1). For a polymorphism to be considered for mapping, reference and alternative alleles had to be observed cumulatively in 50 or more individuals of the population, and the alleles had to segregate 1:1 in the population (χ<sup>2</sup>-test <em>P</em>-value 0.05). If more than two alleles were observed at a polymorphic position, a problem that might arise as the result of contamination with diploid tissue (<em>e.g.</em>, embryo), paralog capture, and sequencing error, the individuals with the two most frequent alleles in the population were considered in further analysis and the remaining were categorized as missing data. To detect presence/absence variants (PAV), a sequencing depth matrix was calculated with genes as rows, haploid samples as columns, and every observation being the average number of times each position of the gene was sequenced. The observations were coded as 1, when at least one read aligned to the gene region, or 0 when no read aligning to the gene region was detected. A test for 1:1 segregation for the presence of the allele was performed (χ<sup>2</sup>-test <em>P</em>-value 0.05) and PAV markers passing these criteria were used for mapping.</p></div><div id="sec-5" class="subsection"><h3>Genetic map construction</h3><p id="p-11">The SNP, short indel, MNP, and PAV markers that exhibited segregation in the population were combined, and in case of gene redundancy, the marker with the least amount of missing data was used as a representative of the gene. The genetic map was constructed with JoinMap 3.0 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Van Ooijen and Voorrips 2001</a>) with parameters as follows: population type haploid, grouping logarithm of odds (LOD) score ≥6, recombination fraction ≤0.4, ripple value = 1, using the Kosambi mapping function. JoinMap 3.0 builds a genetic map adding markers in three successive steps. The first and second steps remove markers that reduce the map goodness-of-fit or creates negative distances. The third step then adds all markers, ignoring the thresholds of goodness-of-fit and negative distances. The complete map was generated by the use of all three rounds to order all linked markers to the map. For a conservative map, we stopped the mapping process after round two to provide a better estimation of gene order, as this removes markers that contribute to unstable ordering. The group number (1−12) and the relative orientation of the markers within the group were defined following the genetic map published by <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Eckert <em>et al.</em> (2010)</a> to facilitate comparisons between studies. Genome length and coverage were estimated following <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Echt <em>et al.</em> (2011)</a>. The genome length was estimated using method 4 of <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chakravarti <em>et al.</em> (1991)</a>, where the observed Kosambi genetic distance of each linkage group (LG) was adjusted by multiplying it by (<em>m</em>+1)/(<em>m</em>-1), where <em>m</em> is the number of markers mapped to the LG. Genome coverage (<em>c</em>), defined as the proportion of the genome contained within a distance <em>d</em> (cM) from a marker, was estimated by <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/29/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1/29/embed/mml-math-1.gif"/></noscript></span></span>, where, <em>L</em> is the total estimated genome length (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lange and Boehnke 1982</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Remington <em>et al.</em> 1999</a>; <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Echt <em>et al.</em> 2011</a>).</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>Genotyping of a mapping population with sequence capture</h3><p id="p-12">We used a multiplexed sequence capture method (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Neves <em>et al.</em> 2013</a>) to characterize genetic variants in 14729 genes. Seventy-two seeds were collected from a single female tree, and the maternally inherited, haploid megagametophyte tissue was used for library preparation. The use of DNA from the seed megagametophyte provides an effective way to assess recombination frequency between genetic markers, without the need for crosses (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adams and Joly 1980</a>). The 72 haploid samples were captured in nine pools of eight megagametophytes each, and sequenced with a median depth of 2.8× across 54773 probes (average of 3.7 probes/genes). Of the 14729 genes targeted, 6283 (43%) were not captured and/or sequenced in any sample. This occurred either because these probes failed to hybridize to genomic DNA or because the captured fragments were not sampled in the sequencing process due to the shallow sequencing depth used. Probes were designed primarily on the basis of a transcriptome assembly, and in a previous analysis of probe performance (<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Neves <em>et al.</em> 2013</a>) we discovered that a significant number of them were complementary to the boundary of two exons, justifying the high failure rate and illustrating a limitation of designing probes for species with uncharacterized genomes. For the remainder 8446 genes (57%), sequence capture was successful for at least one probe, for at least one individual. Of these, 4123 (28%) were sequenced in 50 or more haploid samples of the population. If sequencing data complementary to a given probe was available for fewer than 50 samples, the locus was excluded from further analysis. Therefore, from the 14729 genes initially considered for capture, 28% (4123) were actually captured in enough individuals, and sufficient sequencing depth was generated to allow identification of segregating sequence variants.</p><p id="p-13">The analysis of the sequence data identified 7434 markers segregating 1:1 in at least 50 haploid samples of the population. These markers represented 3787 genes (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The majority of these markers were SNPs (6857), whereas short deletions, short insertions, and MNPs accounted for 210, 208, and 159 markers, respectively. The average sequencing depth for each gene was used to identify possible PAVs (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Swanson-Wagner <em>et al.</em> 2010</a>). The average sequencing depth at the gene level was used to compensate for variations that exist at the probe level and 408 putative PAVs segregating 1:1 in the mapping population were considered for mapping (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Altogether, 7842 segregating markers were identified for 4195 genes, with an average of 2 markers per gene and a range of 1 to 13 (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). A list of the markers and their allele representation within the population is described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008714/-/DC1/TableS2.csv">Table S2</a>. Because the presence of multiple markers per gene provided redundant data for mapping, the marker with the least missing data were selected for the gene, resulting in a final set of 4195 markers for mapping.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463849/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463849/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Description of markers segregating and mapped in the population</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>A <em>P. taeda</em> genetic map with 2841 gene markers</h3><p id="p-15">The segregation of 4195 markers in 72 haploid samples was used to construct a high-density, gene-based genetic map (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008714/-/DC1/TableS3.csv">Table S3</a>). The final map comprises 2841 genes mapped to the expected 12 LGs of the species. The number of genes mapped in each LG varied from 193 genes for LG 1, to 291 genes in LG 8, with an average of 237 genes per LG. Of the 2841 markers mapped, 2622 (92%) were SNPs, 63 (2%) were short deletions, 53 (2%) were short deletions, 38 (1%) were multiple nucleotide polymorphisms, and 65 (2%) were gene PAVs (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). The final map spans 1637.4 cM, which is comparable with previous estimations of map length for the species (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Remington <em>et al.</em> 1999</a>; <a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Eckert <em>et al.</em> 2010</a>; <a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Echt <em>et al.</em> 2011</a>). LG 11 spanned the longest genetic distance with 190 cM, whereas LG 5 spanned the shortest, with 105.1 cM. On average, LGs spanned 132.9 cM (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The average interval between markers was 0.58 cM, across 12 LGs, with 95% of the intervals smaller than 1.5 cM, and a maximum interval of 10.9 cM at the end of LG 2. Following previous methods (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Remington <em>et al.</em> 1999</a>; <a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Echt <em>et al.</em> 2011</a>) we estimated the genome length at 1651.5 cM, and the map coverage as 99%. For this marker density, the estimated map coverage (c) predicts that any locus on the genome has 99% probability of being within 3 cM of a marker and 82% probability of being within 1 cM of a marker. A map with a more conservative marker order, obtained by stopping the mapping process at the second mapping step of JoinMap, was generated and is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008714/-/DC1/TableS4.csv">Table S4</a>. When considering this map, there are still 1371 genes mapped with an average of one marker every 1.3 cM, providing an estimated map coverage (c) that predicts that any locus on the genome has 98% probability of being within 5 cM of a marker.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/29/F1/graphic-1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="High-density genetic map of loblolly pine. The final map is composed of 2841 genes distributed among the 12 LGs with an average distribution of a gene every 0.6 cM and a total length of 1637.4 cM. For easy comparison, the LG name and orientation of the map follow that published by Eckert et al. (2009), and common genes between the two maps are shown in green. Genes for which presence/absence variation was identified and used to map the gene is shown in green and have suffix PAV (e.g., 2_6131_PAV, for unigene 2_6131). The remaining markers were coded as &#x201C;unigene_marker position&#x201D; (e.g., 0_2563_337, for unigene 0_2563 and a polymorphic marker at position 337 of the unigene). The genetic map graphical visualization was prepared using MapChart 2.2 (Voorrips 2002)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-951798219" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;High-density genetic map of loblolly pine. The final map is composed of 2841 genes distributed among the 12 LGs with an average distribution of a gene every 0.6 cM and a total length of 1637.4 cM. For easy comparison, the LG name and orientation of the map follow that published by Eckert &lt;em&gt;et al.&lt;/em&gt; (2009), and common genes between the two maps are shown in green. Genes for which presence/absence variation was identified and used to map the gene is shown in green and have suffix PAV (&lt;em&gt;e.g.&lt;/em&gt;, 2_6131_PAV, for unigene 2_6131). The remaining markers were coded as &#x201C;unigene_marker position&#x201D; (&lt;em&gt;e.g.&lt;/em&gt;, 0_2563_337, for unigene 0_2563 and a polymorphic marker at position 337 of the unigene). The genetic map graphical visualization was prepared using MapChart 2.2 (Voorrips 2002).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/29/F1/graphic-1.medium.gif" width="322" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/4/1/29/F1/graphic-1.medium.gif" width="322" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/29/F1/graphic-1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/29/F1/graphic-1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463897" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/29/F1/graphic-2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="High-density genetic map of loblolly pine. The final map is composed of 2841 genes distributed among the 12 LGs with an average distribution of a gene every 0.6 cM and a total length of 1637.4 cM. For easy comparison, the LG name and orientation of the map follow that published by Eckert et al. (2009), and common genes between the two maps are shown in green. Genes for which presence/absence variation was identified and used to map the gene is shown in green and have suffix PAV (e.g., 2_6131_PAV, for unigene 2_6131). The remaining markers were coded as &#x201C;unigene_marker position&#x201D; (e.g., 0_2563_337, for unigene 0_2563 and a polymorphic marker at position 337 of the unigene). The genetic map graphical visualization was prepared using MapChart 2.2 (Voorrips 2002)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-951798219" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;High-density genetic map of loblolly pine. The final map is composed of 2841 genes distributed among the 12 LGs with an average distribution of a gene every 0.6 cM and a total length of 1637.4 cM. For easy comparison, the LG name and orientation of the map follow that published by Eckert &lt;em&gt;et al.&lt;/em&gt; (2009), and common genes between the two maps are shown in green. Genes for which presence/absence variation was identified and used to map the gene is shown in green and have suffix PAV (&lt;em&gt;e.g.&lt;/em&gt;, 2_6131_PAV, for unigene 2_6131). The remaining markers were coded as &#x201C;unigene_marker position&#x201D; (&lt;em&gt;e.g.&lt;/em&gt;, 0_2563_337, for unigene 0_2563 and a polymorphic marker at position 337 of the unigene). The genetic map graphical visualization was prepared using MapChart 2.2 (Voorrips 2002).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/29/F1/graphic-2.medium.gif" width="289" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/4/1/29/F1/graphic-2.medium.gif" width="289" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/29/F1/graphic-2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/29/F1/graphic-2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463897" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-16" class="first-child">High-density genetic map of loblolly pine. The final map is composed of 2841 genes distributed among the 12 LGs with an average distribution of a gene every 0.6 cM and a total length of 1637.4 cM. For easy comparison, the LG name and orientation of the map follow that published by <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Eckert <em>et al.</em> (2009)</a>, and common genes between the two maps are shown in green. Genes for which presence/absence variation was identified and used to map the gene is shown in green and have suffix PAV (<em>e.g.</em>, 2_6131_PAV, for unigene 2_6131). The remaining markers were coded as “unigene_marker position” (<em>e.g.</em>, 0_2563_337, for unigene 0_2563 and a polymorphic marker at position 337 of the unigene). The genetic map graphical visualization was prepared using MapChart 2.2 (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Voorrips 2002</a>).</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463758/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463758/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Description of genetic map</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Map validation</h3><p id="p-18">The high-density map developed provides an overview of the grouping and orientation of 2841 genes in each loblolly pine chromosome. However, because the number of markers far exceeds the number of meiotic recombinations sampled to generate the map, it can be expected that their order might present incongruences with the physical position on the genome, particularly between closely mapped markers. To analyze the consistency of our map, we compared it with the genetic map built by <a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">Eckert <em>et al.</em> (2010)</a>, hereafter referred to as “<em>Eckert map</em>” because a large number of genes were shared between the two. Ideally, this comparison should be conducted relative to the genome sequence of loblolly pine, but the current assembly (version 0.8) is unordered and still largely fragmented. Thus, the comparison relative to another gene-based genetic map remains the only viable option to validate our results. The <em>Eckert map</em> was generated with SNPs detected by the use of an Illumina Infinium genotyping assay with markers selected from a unigene dataset that overlapped with the one used in our sequence capture probe design. Although the SNPs analyzed within the genes were not the same, a total of 397 genes had markers in common between the two maps and could be used for a comparative analysis. The analysis occurred in two steps: (1) First, we analyzed whether the common genes mapped to the same LG. (2) Next, we evaluated if the order of the genes within the LG were similar between the two maps.</p><p id="p-19">In the first analysis, of the 397 genes positioned in both maps, 392 (99%) mapped to the same LG as that observed in the <em>Eckert map</em>. Thus, there is a nearly perfect agreement between the two maps at the grouping level (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). For the comparison of the genetic position of genes within LGs, the data are illustrated by plotting the normalized position of the common genes on our map (X) relative to the normalized position on the <em>Eckert map</em> (Y) (for instance, LG 2 and LG 4 in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). In the comparative analysis, the position of each gene was normalized to control for the different length of the two maps. For example, a gene mapped to position 61.56 cM on LG 2, which has a length of 123.12 on our map (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), would have a normalized position of 50%. The same approach was used for genes in the <em>Eckert map</em>. Assuming gene synteny between the populations used for mapping, the expected results would be a linear relationship between the markers. For LG 1, 3, 4, 8, 9, 10, and 11, all the common genes were collinear across the entire extension of the group (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008714/-/DC1/FigureS1.pdf">Figure S1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008714/-/DC1/FigureS2.pdf">Figure S2</a>). For LG 2, 6, 7, and 12, most of the genes were collinear, but a block defined by several genes shows incongruences between the two maps (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008714/-/DC1/FigureS1.pdf">Figure S1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008714/-/DC1/FigureS2.pdf">Figure S2</a>). Interestingly, in all cases, these blocks were previously mapped to the beginning or end of the LG in the <em>Eckert map</em> and were mapped toward the middle of the LG in our map. In only one LG, LG5, there is low agreement between the two maps (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008714/-/DC1/FigureS1.pdf">Figure S1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008714/-/DC1/FigureS2.pdf">Figure S2</a>). Overall, the results show a high agreement between the genes mapped in the two maps, both at the grouping level and at the ordering level.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463907/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463907/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Marker validation at the grouping level: comparison of the maps from <a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">Eckert <em>et al.</em> (2010)</a> and the present map</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/29/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the normalized relative order of shared genes used in our study (X-axis) and that of Eckert et al. (2009) (Y-axis). Because of space constraints, only plots for LGs 2 (left) and 4 (right) are shown. Assuming genes syntheny between the two populations, a straight line would illustrate perfect agreement at the gene ordering level between the two maps. Graphs for all 12 LGs are provided in Figure S1 and Figure S2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-951798219" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of the normalized relative order of shared genes used in our study (X-axis) and that of Eckert &lt;em&gt;et al.&lt;/em&gt; (2009) (Y-axis). Because of space constraints, only plots for LGs 2 (left) and 4 (right) are shown. Assuming genes syntheny between the two populations, a straight line would illustrate perfect agreement at the gene ordering level between the two maps. Graphs for all 12 LGs are provided in Figure S1 and Figure S2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/29/F2.medium.gif" width="440" height="238"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/1/29/F2.medium.gif" width="440" height="238"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/29/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/29/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463756" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Comparison of the normalized relative order of shared genes used in our study (X-axis) and that of <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Eckert <em>et al.</em> (2009)</a> (Y-axis). Because of space constraints, only plots for LGs 2 (left) and 4 (right) are shown. Assuming genes syntheny between the two populations, a straight line would illustrate perfect agreement at the gene ordering level between the two maps. Graphs for all 12 LGs are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008714/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008714/-/DC1/FigureS2.pdf">Figure S2</a>.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-10"><h2 class="">Discussion</h2><p id="p-22">The two main objectives of this study were to evaluate the potential of targeted resequencing on the basis of sequence capture for polymorphism detection in loblolly pine and to expand our knowledge about the position of genes in the pine genome. Loblolly pine is one of the most economically and ecologically important species within the conifers, and its genome is currently being sequenced (<a href="http://pinegenome.org/pinerefseq">http://pinegenome.org/pinerefseq</a>). The large and complex pine genome, characterized by large retrotransposons expansions and high nucleotide diversity (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brown <em>et al.</em> 2004</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Burleigh <em>et al.</em> 2012</a>), offers challenges for polymorphism genotyping when traditional platforms, such as SNP chips, are used, resulting in low rates of SNP conversion and high rates of missing data (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Eckert <em>et al.</em> 2009</a>). From previous work (<a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Neves <em>et al.</em> 2013</a>) we expected that probes used for sequence capture could be capturing nonunique regions in the pine genome (<em>e.g.</em>, paralogs and pseudogenes), hampering the detection of site-specific polymorphisms. The use of a mapping population provides a method to verify detected polymorphisms because alternative alleles are expected to segregate in equal proportion (1:1) in a sample of megagametophytes. Even when adopting a shallow sequencing depth, we detected a total of 7434 sequence variant markers for 3787 genes (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>) segregating in the population. From the 14729 genes initially considered for capture, we estimated that 4123 (28%) had sequence properties that allowed us to detect sequence variation, namely having at least 50 individuals with one or more reads aligned to the gene in each individual. Thus, given appropriate sequence capture efficiency and sequencing depth, 92% (3787/4123) of the genes contained at least one segregating marker that was used for mapping in the population. This is a high SNP conversion rate considering that no previous information was available for the population. A greater number of genes likely could have been mapped if we had sequenced the population at higher depth or increased the number of megagametophytes sequenced to sample more meiotic events.</p><p id="p-23">The megagametophyte used as source of DNA sequence capture is a haploid, maternally inherited tissue that is commonly used in conifer mapping studies (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Remington <em>et al.</em> 1999</a>). Thus, a single haplotype was anticipated for any given locus. The pipeline for detection of sequence polymorphism required that the most common base detected in a given megagametophyte be present in at least 80% of the reads that overlap the variant locus. In other words, for a particular position with sequencing depth of 10×, eight or more reads need to have the same base to be considered. This step was implemented to exclude possible errors added during library construction and sequencing, and to avoid cases where unspecific paralogs and pseudogenes were also being captured, which is likely considering their abundance in the pine genome (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kovach <em>et al.</em> 2010</a>). Although we did not formally quantify instances where the latter happens, we observed it while optimizing the bioinformatics pipeline. For example, only reads aligned to the unigene reference with less than 5% mismatch were considered for polymorphism detection and mapping. At this threshold, we detected the 7434 sequence polymorphisms reported. When up to 10% mismatch to the unigene reference were permitted for use of a sequencing read, there was a significant reduction in the number of polymorphisms detected that segregated in the expected ratio in the population. Accepting 10% of mismatches lead to the detection of only 4563 polymorphisms, represented within 2210 genes, that segregate in a 1:1 ratio. This finding suggests that <em>bona fide</em> polymorphisms were confounded by alleles from paralogs and pseudogenes mapped with less stringent alignment criteria.</p><p id="p-24">The large size of the pine genome of pine is attributed to retrotransposon activity, rather than whole-genome duplications (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Mackay <em>et al.</em> 2012</a>). On the basis of the dynamic nature of retrotransposons, we hypothesized that a subset of the pine genes might be present in multiple copies in the genome, as observed in maize and another genomes characterized by retrotransposon genome expansions (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Belo <em>et al.</em> 2010</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Swanson-Wagner <em>et al.</em> 2010</a>). To test this hypothesis, we aimed to detect PAVs. In the specific case of a maternally derived haploid segregating population, these are hemizygous genes that segregate in the population with 50% of the haploid samples containing the gene in their genome, and 50% with the gene absent in their genome. We focused on PAVs, because these events are easier to detect than copy number variations (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Teo <em>et al.</em> 2012</a>), particularly considering that samples were sequenced at low depth. In our analysis, a gene was considered absent when no reads aligned throughout the entire gene region, and present when at least one read was aligned to the gene region. Using this simple analysis criteria allowed us to detect 408 putative PAV. However, only 16% (65) of the PAVs could be mapped, suggesting that many of PAVs were false positives or contained genotyping errors that excluded them from mapping. These false positive or genotyping errors were expected as a consequence of the low sequencing depth, and we anticipated that the genetic mapping process would be able to filter those out, providing a more accurate subset of PAVs in <em>P. taeda</em>.</p><p id="p-25">The final genetic map was obtained by analyzing the segregation of 4195 markers, each representing a unique gene of the <em>P. taeda</em> genome, in 72 haploid samples. Ultimately, 2841 (68%) genes were mapped in the expected 12 LGs. To the best of our knowledge, this is the most dense genetic map for conifers or trees in general, although other highly saturated maps are becoming available (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Martinez-Garcia <em>et al.</em> 2013</a>). Because this map was constructed using gene-based marker, it provides links with other published maps for loblolly pine, allowing map cross-checking and eventual consolidation. The genes mapped were not selected based on any specific criteria. Thus, the map provides an unbiased representation of the loblolly pine gene set that can be used for studies that benefit from a preliminary ordering of the genes until a reference genome sequence of reasonable quality becomes available. This high-density map should also be valuable when anchoring and orienting scaffolds in the assembly of the reference genome (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Tuskan <em>et al.</em> 2006</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Sato <em>et al.</em> 2012</a>).</p><p id="p-26">The comparison to a previous map of <em>P. taeda</em> (<a id="xref-ref-13-8" class="xref-bibr" href="#ref-13">Eckert <em>et al.</em> 2010</a>) that shares 397 common genes with our new map revealed a high-level of agreement at the grouping and ordering level (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). These results provide strong evidence that markers were correctly identified and genotyped in the mapping population, despite the complexity of the genome of pine. For some LGs (2, 6, 7, 8, and 12; <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008714/-/DC1/FigureS1.pdf">Figure S1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008714/-/DC1/FigureS2.pdf">Figure S2</a>), blocks of adjacent markers that were previously assigned to the edge of the LG in the map of Eckert <em>et al.</em> were mapped toward the center of the LG in our map. Such discrepancies between blocks of genetic markers have been observed in other studies in <em>Pinaceae</em> (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Krutovsky <em>et al.</em> 2004</a>; <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Eckert <em>et al.</em> 2009</a>; <a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Echt <em>et al.</em> 2011</a>) and were usually attributed to genotyping errors. Although we cannot test for genotyping errors, we did not observe higher rates of missing data in the markers that are out of order compared to the collinear ones (data not shown). Also, from simulated experiments, it is expected that genotyping errors would inflate the length of the genetic map (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hackett and Broadfoot 2003</a>), but this was not observed in our map. Therefore, we are unable to conclusively identify the causes for noncollinear blocks of markers. Attributing such discrepancies to real genome rearrangements will require further experiments based on larger numbers of individuals or whole-genome resequencing.</p><p id="p-27">Genotyping samples of a large population using sequence capture presents some advantages over conventional genotyping methods. For example, sequence capture does not require one to previously characterize the polymorphisms to be detected. We show that this is of greater importance for a segregating population because even if previously characterized polymorphisms are available, it is not known which of those sites will segregate in a single biparental segregating population and have the potential to be mapped. In this study, the probes used for capture were designed to span the entire unigene length, and we were able to detect an average of two polymorphic sites segregating in the population for every gene. Thus, the chances of detecting a marker to map the gene increased and we were able to map 68% of the genes containing a segregating marker (2841/4195), even adopting an overall shallow sequencing depth of 2.8×. For several genes we detected additional polymorphic sites within 60 bp of the position used for mapping, without impairing the efficiency of the capture. This is an usual observation in pine and other highly polymorphic species, and a common limitation for developing traditional genotyping methods, such as SNP chips (<a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Eckert <em>et al.</em> 2009</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Grattapaglia <em>et al.</em> 2011</a>). Finally, another advantage illustrated is the potential to detect other types of genetic variants in addition to SNPs, such as the presence/absence markers that we were able to detect and map for 65 genes. Although a small percentage of the mapped markers were genotyped with PAV, sequencing at higher depth will likely increase this number and enable more accurate detection of gene copy number variation, as achieved by other studies in plants (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Saintenac <em>et al.</em> 2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Valdes-Mas <em>et al.</em> 2012</a>). With an ultimate goal to use genetic markers for phenotypic characterization, recent studies are showing that copy number variations might explain a portion of phenotypic variation not sampled by SNPs, justifying its characterization (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Maron <em>et al.</em> 2013</a>).</p><p id="p-28">We report the application of exome capture on loblolly pine for polymorphism detection and gene mapping by analyzing a mapping population of 72 haploid samples. An important advantage of this approach for polymorphism detection is that it allows for the enrichment and sequencing of several segments of the gene being targeted, which increases the chances of detecting a marker segregating in the population. As a result, we were able to detect at least one segregating marker for 4195 genes, despite the shallow sequencing depth employed. The markers were used to generate a gene-rich genetic map that ultimately positions 2841 genes with an average intergenic distance of 0.6 cM. Also, because exome capture is a genotyping by sequencing method, we show that other types of segregating markers can be identified in the population, such as gene presence/absence variation. This is particularly important, because it highlights the possibility to study gene dosage using exome capture in plants, which traditionally has required additional experimentation. To our knowledge, this is the most saturated map in conifers and trees in general. Because this is a gene-based map, we expect that it will help in the construction of the pine reference sequence that is currently being generated while facilitating other basic and applied research efforts that use a high-density, gene-rich genetic maps.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-29">The work was supported by the US Department of Agriculture National Institute of Food and Agriculture Plant Breeding and Education Program (award no. 2010-85117-20569 to MK).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008714/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008714/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-30"><em>Communicating editor: A. H. Paterson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-09-30"><span class="received-label">Received </span>September 30, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-10-22"><span class="accepted-label">Accepted </span>October 22, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Neves <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">W. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joly</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Genetic of allozyme variants in loblolly pine.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DADAMS%26rft.auinit1%253DW.%2BT.%26rft.volume%253D71%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D40%26rft.atitle%253DGenetics%2Bof%2Ballozyme%2Bvariants%2Bin%2Bloblolly%2Bpine%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo2OiJqaGVyZWQiO3M6NToicmVzaWQiO3M6NzoiNzEvMS8zMyI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzQvMS8yOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.2" data-doi="10.1093/bioinformatics/btr174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barnett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrison</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stromberg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marth</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">BamTools: a C++ API and toolkit for analyzing and managing BAM files.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1691</span>–<span class="cit-lpage">1692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr174%26rft_id%253Dinfo%253Apmid%252F21493652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTIvMTY5MSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzQvMS8yOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.3" data-doi="10.1007/s00122-009-1128-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beatty</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hondred</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fengler</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Allelic genome structural variations in maize detected by array comparative genome hybridization.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBelo%26rft.auinit1%253DA.%26rft.volume%253D120%26rft.issue%253D2%26rft.spage%253D355%26rft.epage%253D367%26rft.atitle%253DAllelic%2Bgenome%2Bstructural%2Bvariations%2Bin%2Bmaize%2Bdetected%2Bby%2Barray%2Bcomparative%2Bgenome%2Bhybridization.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1128-9%26rft_id%253Dinfo%253Apmid%252F19756477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1128-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19756477&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272803700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.4" data-doi="10.2135/cropsci2008.03.0131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Molecular markers and selection for complex traits in plants: Learning from the last 20 years.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1649</span>–<span class="cit-lpage">1664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D48%26rft.issue%253D5%26rft.spage%253D1649%26rft.epage%253D1664%26rft.atitle%253DMolecular%2BMarkers%2Band%2BSelection%2Bfor%2BComplex%2BTraits%2Bin%2BPlants%253A%2BLearning%2Bfrom%2Bthe%2BLast%2B20%2BYears%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.03.0131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.03.0131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000259792100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.5" data-doi="10.1073/pnas.0404231101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuntz</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Nucleotide diversity and linkage disequilibrium in loblolly pine.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">15255</span>–<span class="cit-lpage">15260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0404231101%26rft_id%253Dinfo%253Apmid%252F15477602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzQyLzE1MjU1IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.6" data-doi="10.1111/j.1467-7652.2012.00707.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bundock</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casu</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Enrichment of genomic DNA for polymorphism detection in a non-model highly polyploid crop plant.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DBundock%26rft.auinit1%253DP.%2BC.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D657%26rft.epage%253D667%26rft.atitle%253DEnrichment%2Bof%2Bgenomic%2BDNA%2Bfor%2Bpolymorphism%2Bdetection%2Bin%2Ba%2Bnon-model%2Bhighly%2Bpolyploid%2Bcrop%2Bplant.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2012.00707.x%26rft_id%253Dinfo%253Apmid%252F22624722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2012.00707.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22624722&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306131400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burleigh</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbazuk</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morse</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Exploring diversification and genome size evolution in extant gymnosperms through phylogenetic synthesis.</span> <abbr class="cit-jnl-abbrev">J. Bot.</abbr> <span class="cit-vol">2012</span>: <span class="cit-fpage">292857</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBot.%26rft.volume%253D2012%26rft.spage%253D292857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chakravarti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lasher</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reefer</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">A maximum likelihood method for estimating genome length using genetic linkage data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChakravarti%26rft.auinit1%253DA.%26rft.volume%253D128%26rft.issue%253D1%26rft.spage%253D175%26rft.epage%253D182%26rft.atitle%253DA%2Bmaximum%2Blikelihood%2Bmethod%2Bfor%2Bestimating%2Bgenome%2Blength%2Busing%2Bgenetic%2Blinkage%2Bdata.%26rft_id%253Dinfo%253Apmid%252F2060775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI4LzEvMTc1IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.9" data-doi="10.1038/nrg3012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavey%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D510%26rft.atitle%253DGenome-wide%2Bgenetic%2Bmarker%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3012%26rft_id%253Dinfo%253Apmid%252F21681211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681211&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drost</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novaes</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boaventura-Novaes</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benedict</span>  <span class="cit-name-given-names">C. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A microarray-based genotyping and genetic mapping approach for highly heterozygous outcrossing species enables localization of a large fraction of the unassembled <em>Populus trichocarpa</em> genome sequence.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1054</span>–<span class="cit-lpage">1067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D12%26rft.spage%253D1054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.11" data-doi="10.1186/1471-2156-12-17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Echt</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krutovsky</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimalanathan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erpelding</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">An annotated genetic map of loblolly pine based on microsatellite and cDNA markers.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DEcht%26rft.auinit1%253DC.%2BS.%26rft.volume%253D12%26rft.spage%253D17%26rft.epage%253D17%26rft.atitle%253DAn%2Bannotated%2Bgenetic%2Bmap%2Bof%2Bloblolly%2Bpine%2Bbased%2Bon%2Bmicrosatellite%2Band%2BcDNA%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-12-17%26rft_id%253Dinfo%253Apmid%252F21269494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-12-17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21269494&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.12" data-doi="10.1007/s11295-008-0183-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pande</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rashbrook</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">High-throughput genotyping and mapping of single nucleotide polymorphisms in loblolly pine (<em>Pinus taeda</em> L.).</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D5%26rft.spage%253D225%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-008-0183-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-008-0183-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000261265100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.13" data-doi="10.1534/genetics.110.115543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Heerwaarden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Patterns of population structure and environmental associations to aridity across the range of loblolly pine (<em>Pinus taeda</em> L., Pinaceae).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">969</span>–<span class="cit-lpage">982</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.115543%26rft_id%253Dinfo%253Apmid%252F20439779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzMvOTY5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.14" data-doi="10.1007/s001220100725"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elsik</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">C. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Low-copy microsatellite recovery from a conifer genome.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">1189</span>–<span class="cit-lpage">1195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D103%26rft.spage%253D1189%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220100725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220100725&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.15" data-doi="10.1038/nbt.1523"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gnirke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melnikov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogov</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LeProust</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">182</span>–<span class="cit-lpage">189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGnirke%26rft.auinit1%253DA.%26rft.volume%253D27%26rft.issue%253D2%26rft.spage%253D182%26rft.epage%253D189%26rft.atitle%253DSolution%2Bhybrid%2Bselection%2Bwith%2Bultra-long%2Boligonucleotides%2Bfor%2Bmassively%2Bparallel%2Btargeted%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1523%26rft_id%253Dinfo%253Apmid%252F19182786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1523&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19182786&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263209000025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.16" data-doi="10.1534/genetics.106.061127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Martinez</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Association genetics in <em>Pinus taeda</em> L. I. Wood property traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.061127%26rft_id%253Dinfo%253Apmid%252F17110498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc1LzEvMzk5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.17" data-doi="10.1186/1471-2229-11-65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva-Junior</span>  <span class="cit-name-given-names">O. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirst</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Lima</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faria</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High-throughput SNP genotyping in the highly heterozygous genome of Eucalyptus: assay success, polymorphism and transferability across species.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DGrattapaglia%26rft.auinit1%253DD.%26rft.volume%253D11%26rft.spage%253D65%26rft.epage%253D65%26rft.atitle%253DHigh-throughput%2BSNP%2Bgenotyping%2Bin%2Bthe%2Bhighly%2Bheterozygous%2Bgenome%2Bof%2BEucalyptus%253A%2Bassay%2Bsuccess%252C%2Bpolymorphism%2Band%2Btransferability%2Bacross%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-11-65%26rft_id%253Dinfo%253Apmid%252F21492434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-11-65&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21492434&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.18" data-doi="10.1038/sj.hdy.6800173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broadfoot</span>  <span class="cit-name-given-names">L. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D38%26rft.atitle%253DEffects%2Bof%2Bgenotyping%2Berrors%252C%2Bmissing%2Bvalues%2Band%2Bsegregation%2Bdistortion%2Bin%2Bmolecular%2Bmarker%2Bdata%2Bon%2Bthe%2Bconstruction%2Bof%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800173%26rft_id%253Dinfo%253Apmid%252F12522423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12522423&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181165800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.19" data-doi="10.1007/s00122-004-1915-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kayihan</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morse</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetic dissection of fusiform rust and pitch canker disease traits in loblolly pine.</span> <abbr class="cit-jnl-abbrev">Theor. Appl Genet.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">948</span>–<span class="cit-lpage">958</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DKayihan%26rft.auinit1%253DG.%2BC.%26rft.volume%253D110%26rft.issue%253D5%26rft.spage%253D948%26rft.epage%253D958%26rft.atitle%253DGenetic%2Bdissection%2Bof%2Bfusiform%2Brust%2Band%2Bpitch%2Bcanker%2Bdisease%2Btraits%2Bin%2Bloblolly%2Bpine.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1915-2%26rft_id%253Dinfo%253Apmid%252F15700146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1915-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15700146&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.20" data-doi="10.1186/1471-2164-11-420"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kovach</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruening</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The <em>Pinus taeda</em> genome is characterized by diverse and highly diverged repetitive sequences.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DKovach%26rft.auinit1%253DA.%26rft.volume%253D11%26rft.spage%253D420%26rft.epage%253D420%26rft.atitle%253DThe%2BPinus%2Btaeda%2Bgenome%2Bis%2Bcharacterized%2Bby%2Bdiverse%2Band%2Bhighly%2Bdiverged%2Brepetitive%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-420%26rft_id%253Dinfo%253Apmid%252F20609256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-420&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20609256&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.21" data-doi="10.1534/genetics.104.028381"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krutovsky</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troggio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jermstad</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Comparative mapping in the Pinaceae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKrutovsky%26rft.auinit1%253DK.%2BV.%26rft.volume%253D168%26rft.issue%253D1%26rft.spage%253D447%26rft.epage%253D461%26rft.atitle%253DComparative%2BMapping%2Bin%2Bthe%2BPinaceae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.028381%26rft_id%253Dinfo%253Apmid%252F15454556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY4LzEvNDQ3IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boehnke</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">How many polymorphic genes will it take to span the human genome?</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">842</span>–<span class="cit-lpage">845</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DLange%26rft.auinit1%253DK.%26rft.volume%253D34%26rft.issue%253D6%26rft.spage%253D842%26rft.epage%253D845%26rft.atitle%253DHow%2Bmany%2Bpolymorphic%2Bgenes%2Bwill%2Bit%2Btake%2Bto%2Bspan%2Bthe%2Bhuman%2Bgenome%253F%26rft_id%253Dinfo%253Apmid%252F6960692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=6960692&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.23" data-doi="10.1007/s11103-012-9961-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plomion</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Canovas</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Towards decoding the conifer giga-genome.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">555</span>–<span class="cit-lpage">569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DMackay%26rft.auinit1%253DJ.%26rft.volume%253D80%26rft.issue%253D6%26rft.spage%253D555%26rft.epage%253D569%26rft.atitle%253DTowards%2Bdecoding%2Bthe%2Bconifer%2Bgiga-genome.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-012-9961-7%26rft_id%253Dinfo%253Apmid%252F22960864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-012-9961-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22960864&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312134200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.24" data-doi="10.1073/pnas.1220766110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maron</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guimaraes</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirst</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birchler</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Aluminum tolerance in maize is associated with higher MATE1 gene copy number.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">5241</span>–<span class="cit-lpage">5246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1220766110%26rft_id%253Dinfo%253Apmid%252F23479633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzEzLzUyNDEiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy80LzEvMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martinez-Garcia</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liechty</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crepeau</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Combination of multipoint maximum likelihood (MML) and regression mapping algorithms to construct a high-density genetic linkage map for loblolly pine (<em>Pinus taeda</em> L.).</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.26" data-doi="10.1186/1471-2164-12-189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neves</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mamani</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alfenas</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirst</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A high-density transcript linkage map with 1,845 expressed genes positioned by microarray-based Single Feature Polymorphisms (SFP) in Eucalyptus.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DNeves%26rft.auinit1%253DL.%2BG.%26rft.volume%253D12%26rft.spage%253D189%26rft.epage%253D189%26rft.atitle%253DA%2Bhigh-density%2Btranscript%2Blinkage%2Bmap%2Bwith%2B1%252C845%2Bexpressed%2Bgenes%2Bpositioned%2Bby%2Bmicroarray-based%2BSingle%2BFeature%2BPolymorphisms%2B%2528SFP%2529%2Bin%2BEucalyptus.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-189%26rft_id%253Dinfo%253Apmid%252F21492453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21492453&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.27" data-doi="10.1111/tpj.12193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neves</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbazuk</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirst</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-exome targeted sequencing of the uncharacterized pine genome.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D75%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12193%26rft_id%253Dinfo%253Apmid%252F23551702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23551702&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000320708300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.28" data-doi="10.1007/s001220051194"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Remington</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whetten</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Malley</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Construction of an AFLP genetic map with nearly complete genome coverage in <em>Pinus taeda</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">1279</span>–<span class="cit-lpage">1292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DRemington%26rft.auinit1%253DD.%2BL.%26rft.volume%253D98%26rft.issue%253D8%26rft.spage%253D1279%26rft.epage%253D1292%26rft.atitle%253DConstruction%2Bof%2Ban%2BAFLP%2Bgenetic%2Bmap%2Bwith%2Bnearly%2Bcomplete%2Bgenome%2Bcoverage%2Bin%2BPinus%2Btaeda.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051194%26rft_id%253Dinfo%253Apmid%252F12238515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051194&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12238515&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081124700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.29" data-doi="10.1111/j.1469-8137.2011.03895.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munoz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acosta</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peter</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accelerating the domestication of trees using genomic selection: accuracy of prediction models across ages and environments.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DResende%26rft.auinit1%253DM.%2BF.%26rft.volume%253D193%26rft.issue%253D3%26rft.spage%253D617%26rft.epage%253D624%26rft.atitle%253DAccelerating%2Bthe%2Bdomestication%2Bof%2Btrees%2Busing%2Bgenomic%2Bselection%253A%2Baccuracy%2Bof%2Bprediction%2Bmodels%2Bacross%2Bages%2Band%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.03895.x%26rft_id%253Dinfo%253Apmid%252F21973055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.03895.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21973055&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298984900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.30" data-doi="10.1186/gb-2011-12-9-r88"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saintenac</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akhunov</span>  <span class="cit-name-given-names">E. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Targeted analysis of nucleotide and copy number variation by exon capture in allotetraploid wheat genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DSaintenac%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.issue%253D9%26rft.spage%253DR88%26rft.epage%253DR88%26rft.atitle%253DTargeted%2Banalysis%2Bof%2Bnucleotide%2Band%2Bcopy%2Bnumber%2Bvariation%2Bby%2Bexon%2Bcapture%2Bin%2Ballotetraploid%2Bwheat%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-9-r88%26rft_id%253Dinfo%253Apmid%252F21917144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-9-r88&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21917144&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.31" data-doi="10.1038/nature11119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabata</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirakawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asamizu</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirasawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The tomato genome sequence provides insights into fleshy fruit evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">485</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSato%26rft.auinit1%253DS.%26rft.volume%253D485%26rft.issue%253D7400%26rft.spage%253D635%26rft.epage%253D641%26rft.atitle%253DThe%2Btomato%2Bgenome%2Bsequence%2Bprovides%2Binsights%2Binto%2Bfleshy%2Bfruit%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11119%26rft_id%253Dinfo%253Apmid%252F22660326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660326&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304608000047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.32" data-doi="10.1007/s11295-011-0459-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tauer</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Bidirectional introgression between loblolly pine (<em>Pinus taeda</em> L.) and shortleaf pine (<em>P. echinata</em> Mill.) has increased since the 1950s.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">725</span>–<span class="cit-lpage">735</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D8%26rft.spage%253D725%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-011-0459-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-011-0459-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.33" data-doi="10.1101/gr.109165.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swanson-Wagner</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichten</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumari</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiffin</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Pervasive gene content variation and copy number variation in maize and its undomesticated progenitor.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1689</span>–<span class="cit-lpage">1699</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.109165.110%26rft_id%253Dinfo%253Apmid%252F21036921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMi8xNjg5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.34" data-doi="10.1093/bioinformatics/bts535"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teo</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pawitan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ku</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salim</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Statistical challenges associated with detecting copy number variations with next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2711</span>–<span class="cit-lpage">2718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts535%26rft_id%253Dinfo%253Apmid%252F22942022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMjEvMjcxMSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzQvMS8yOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.35" data-doi="10.1126/science.1128691"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuskan</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Difazio</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bohlmann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grigoriev</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The genome of black cottonwood, <em>Populus trichocarpa</em> (Torr. &amp; Gray).</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">313</span>: <span class="cit-fpage">1596</span>–<span class="cit-lpage">1604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTuskan%26rft.auinit1%253DG.%2BA.%26rft.volume%253D313%26rft.issue%253D5793%26rft.spage%253D1596%26rft.epage%253D1604%26rft.atitle%253DThe%2Bgenome%2Bof%2Bblack%2Bcottonwood%252C%2BPopulus%2Btrichocarpa%2B%2528Torr.%2B%2526amp%253B%2BGray%2529.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1128691%26rft_id%253Dinfo%253Apmid%252F16973872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTMvNTc5My8xNTk2IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.36" data-doi="10.1371/journal.pone.0051422"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdes-Mas</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bea</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puente</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez-Otin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puente</span>  <span class="cit-name-given-names">X. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimation of copy number alterations from exome sequencing data.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e51422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D7%26rft.spage%253De51422%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0051422%26rft_id%253Dinfo%253Apmid%252F23284693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0051422&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284693&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-4.1.29.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-source">JoinMap 3.0, Software for the Calculation of Genetic Linkage Maps</span>. <span class="cit-ed cit-collab">Plant Research International</span>, <span class="cit-publ-loc">Wageningen, The Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.38" data-doi="10.1093/jhered/93.1.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MapChart: software for the graphical presentation of linkage maps and QTLs.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D78%26rft.atitle%253DMapChart%253A%2BSoftware%2Bfor%2Bthe%2BGraphical%2BPresentation%2Bof%2BLinkage%2BMaps%2Band%2BQTLs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F93.1.77%26rft_id%253Dinfo%253Apmid%252F12011185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjc6IjkzLzEvNzciO3M6NDoiYXRvbSI7czoxNjoiL2dnZy80LzEvMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-4.1.29.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wear</span>  <span class="cit-name-given-names">D. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greis</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-source">The southern forest resource assessment summary report</span>. <span class="cit-publ-name">USDA Forest Service, Southern Research Station</span>. <span class="cit-publ-loc">Asheville, NC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.29.40" data-doi="10.1186/1471-2164-13-703"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holliday</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Targeted enrichment of the black cottonwood (<em>Populus trichocarpa</em>) gene space using sequence capture.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D13%26rft.spage%253D703%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-703%26rft_id%253Dinfo%253Apmid%252F23241106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-703&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23241106&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/1/29.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/1/19" title="Origin and Length Distribution of Unidirectional Prokaryotic Overlapping Genes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/1/39" title="Chemical Suppression of Defects in Mitotic Spindle Assembly, Redox Control, and Sterol Biosynthesis by Hydroxyurea" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/1"><span>Volume 4 Issue 1, January 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/1.cover-source.jpg?itok=oIsy65sY" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/1/29" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YVhjhmLYIwPOY-4PculK5lcrLh9zCVVvjAzEcNobc7s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/1/29" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/1/29" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/1/29" class="active" data-icon-position="" data-hide-link-title="0">A High-Density Gene Map of Loblolly Pine (Pinus taeda L.) Based on Exome Sequence Capture Genotyping</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459608" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-_oX21Lm_rPkix2Pk5xow3ynw-n56g59GB-bGGG_D2jc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/1/29.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/1/29" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-qtq21KFQWAJQpGFbp5c3vb7NK7igohZKXdzD9Ke502E" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/1/29" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/1/29.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459608" id="node459608--2" data-pisa="ggg;4/1/29" data-pisa-master="ggg;g3.113.008714" data-apath="/ggg/4/1/29.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Density Gene Map of Loblolly Pine (<em>Pinus taeda</em> L.) Based on Exome Sequence Capture Genotyping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Leandro Gomide Neves</span>, <span class="highwire-citation-author" data-delta="1">John M. Davis</span>, <span class="highwire-citation-author" data-delta="2">William B. Barbazuk</span> and <span class="highwire-citation-author" data-delta="3">Matias Kirst</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">29-37;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008714 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459608--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Leandro Gomide Neves </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Graduate Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, Florida 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leandro%2BGomide%2BNeves%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neves%20LG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeandro%2BGomide%2BNeves%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John M. Davis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Graduate Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Forest Resources and Conservation, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>University of Florida Genetics Institute, University of Florida, Gainesville, Florida 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BDavis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davis%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BM.%2BDavis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">William B. Barbazuk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Graduate Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>University of Florida Genetics Institute, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, University of Florida, Gainesville, Florida 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BB.%2BBarbazuk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barbazuk%20WB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BB.%2BBarbazuk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bbarbazuk@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">bbarbazuk@ufl.edu</a>
<a href="mailto:mkirst@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">mkirst@ufl.edu</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matias Kirst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Graduate Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Forest Resources and Conservation, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, University of Florida, Gainesville, Florida 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matias%2BKirst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kirst%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatias%2BKirst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bbarbazuk@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">bbarbazuk@ufl.edu</a>
<a href="mailto:mkirst@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">mkirst@ufl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/1/29" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/1/29&amp;title=A%20High-Density%20Gene%20Map%20of%20Loblolly%20Pine%20%28Pinus%20taeda%20L.%29%20Based%20on%20Exome%20Sequence%20Capture%20Genotyping" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/1/29&amp;title=A%20High-Density%20Gene%20Map%20of%20Loblolly%20Pine%20%28Pinus%20taeda%20L.%29%20Based%20on%20Exome%20Sequence%20Capture%20Genotyping" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/1/29&amp;title=A%20High-Density%20Gene%20Map%20of%20Loblolly%20Pine%20%28Pinus%20taeda%20L.%29%20Based%20on%20Exome%20Sequence%20Capture%20Genotyping" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/1/29&amp;text=A%20High-Density%20Gene%20Map%20of%20Loblolly%20Pine%20%28Pinus%20taeda%20L.%29%20Based%20on%20Exome%20Sequence%20Capture%20Genotyping" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/1/29&amp;title=A%20High-Density%20Gene%20Map%20of%20Loblolly%20Pine%20%28Pinus%20taeda%20L.%29%20Based%20on%20Exome%20Sequence%20Capture%20Genotyping" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/1/29&amp;t=A%20High-Density%20Gene%20Map%20of%20Loblolly%20Pine%20%28Pinus%20taeda%20L.%29%20Based%20on%20Exome%20Sequence%20Capture%20Genotyping" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/1/29&amp;title=A%20High-Density%20Gene%20Map%20of%20Loblolly%20Pine%20%28Pinus%20taeda%20L.%29%20Based%20on%20Exome%20Sequence%20Capture%20Genotyping" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/1/29&amp;title=A%20High-Density%20Gene%20Map%20of%20Loblolly%20Pine%20%28Pinus%20taeda%20L.%29%20Based%20on%20Exome%20Sequence%20Capture%20Genotyping" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459608/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459608" id="node459608--3" data-pisa="ggg;4/1/29" data-pisa-master="ggg;g3.113.008714" data-apath="/ggg/4/1/29.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Density Gene Map of Loblolly Pine (<em>Pinus taeda</em> L.) Based on Exome Sequence Capture Genotyping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Leandro Gomide Neves</span>, <span class="highwire-citation-author" data-delta="1">John M. Davis</span>, <span class="highwire-citation-author" data-delta="2">William B. Barbazuk</span> and <span class="highwire-citation-author" data-delta="3">Matias Kirst</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">29-37;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008714 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459608--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Leandro Gomide Neves </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Graduate Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, Florida 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leandro%2BGomide%2BNeves%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neves%20LG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeandro%2BGomide%2BNeves%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John M. Davis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Graduate Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Forest Resources and Conservation, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>University of Florida Genetics Institute, University of Florida, Gainesville, Florida 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BDavis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davis%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BM.%2BDavis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">William B. Barbazuk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Graduate Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>University of Florida Genetics Institute, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, University of Florida, Gainesville, Florida 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BB.%2BBarbazuk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barbazuk%20WB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BB.%2BBarbazuk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bbarbazuk@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">bbarbazuk@ufl.edu</a>
<a href="mailto:mkirst@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">mkirst@ufl.edu</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matias Kirst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Graduate Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Forest Resources and Conservation, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, University of Florida, Gainesville, Florida 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matias%2BKirst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kirst%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatias%2BKirst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bbarbazuk@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">bbarbazuk@ufl.edu</a>
<a href="mailto:mkirst@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">mkirst@ufl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=29&amp;rft.epage=37&amp;rft.atitle=A%20High-Density%20Gene%20Map%20of%20Loblolly%20Pine%20%28Pinus%20taeda%20L.%29%20Based%20on%20Exome%20Sequence%20Capture%20Genotyping&amp;rft.volume=4&amp;rft.issue=1&amp;rft.date=2014-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Neves%2C+Leandro+Gomide&amp;rft.au=Davis%2C+John+M.&amp;rft.au=Barbazuk%2C+William+B.&amp;rft.au=Kirst%2C+Matias"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459608/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459608/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459608/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459608/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459608/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459608/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459608/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459608/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459608/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459608/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459608/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459608/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/1/29" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgements</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/1/29.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/1/29.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/1/29.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/1/29.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
