<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/1/19" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/1/19.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/1/19.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/1/19.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/1/19.atom" />
<meta name="HW.pisa" content="ggg;4/1/19" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Origin and Length Distribution of Unidirectional Prokaryotic Overlapping Genes" />
<meta name="DC.Identifier" content="10.1534/g3.113.005652" />
<meta name="DC.Date" content="2014-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Fonseca et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Prokaryotic unidirectional overlapping genes can be originated by disrupting and replacing of the start or stop codon of one protein-coding gene with another start or stop codon within the adjacent gene. However, the probability of disruption and replacement of a start or stop codon may differ significantly depending on the number and redundancy of the start and stop codons sets. Here, we performed a simulation study of the formation of unidirectional overlapping genes using a simple model of nucleotide change and contrasted it with empirical data. Our results suggest that overlaps originated by an elongation of the 3′-end of the upstream gene are significantly more frequent than those originated by an elongation of the 5′-end of the downstream gene. According to this, we propose a model for the creation of unidirectional overlaps that is based on the disruption probabilities of start codon and stop codon sets and on the different probabilities of phase 1 and phase 2 overlaps. Additionally, our results suggest that phase 2 overlaps are formed at higher rates than phase 1 overlaps, given the same evolutionary time. Finally, we propose that there is no need to invoke selection to explain the prevalence of long phase 1 unidirectional overlaps. Rather, the overrepresentation of long phase 1 relative to long phase 2 overlaps might occur because it is highly probable that phase 2 overlaps are retained as short overlaps by chance. Such a pattern is stronger if selection against very long overlaps is included in the model. Our model as a whole is able to explain to a large extent the empirical length distribution of unidirectional overlaps in prokaryotic genomes." />
<meta name="DC.Contributor" content="Miguel M. Fonseca" />
<meta name="DC.Contributor" content="D. James Harris" />
<meta name="DC.Contributor" content="David Posada" />
<meta name="article:published_time" content="2014-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Origin and Length Distribution of Unidirectional Prokaryotic Overlapping Genes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Prokaryotic unidirectional overlapping genes can be originated by disrupting and replacing of the start or stop codon of one protein-coding gene with another start or stop codon within the adjacent gene. However, the probability of disruption and replacement of a start or stop codon may differ significantly depending on the number and redundancy of the start and stop codons sets. Here, we performed a simulation study of the formation of unidirectional overlapping genes using a simple model of nucleotide change and contrasted it with empirical data. Our results suggest that overlaps originated by an elongation of the 3′-end of the upstream gene are significantly more frequent than those originated by an elongation of the 5′-end of the downstream gene. According to this, we propose a model for the creation of unidirectional overlaps that is based on the disruption probabilities of start codon and stop codon sets and on the different probabilities of phase 1 and phase 2 overlaps. Additionally, our results suggest that phase 2 overlaps are formed at higher rates than phase 1 overlaps, given the same evolutionary time. Finally, we propose that there is no need to invoke selection to explain the prevalence of long phase 1 unidirectional overlaps. Rather, the overrepresentation of long phase 1 relative to long phase 2 overlaps might occur because it is highly probable that phase 2 overlaps are retained as short overlaps by chance. Such a pattern is stronger if selection against very long overlaps is included in the model. Our model as a whole is able to explain to a large extent the empirical length distribution of unidirectional overlaps in prokaryotic genomes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/01/01" />
<meta name="citation_mjid" content="ggg;4/1/19" />
<meta name="citation_id" content="4/1/19" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/1/19" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/1/19.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/1/19.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/1/19.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.005652" />
<meta name="citation_pmid" content="24192837" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="19" />
<meta name="citation_lastpage" content="27" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Miguel M. Fonseca" />
<meta name="citation_author_institution" content="Department of Biochemistry, Genetics and Immunology, University of Vigo, 36310 Vigo, Spain" />
<meta name="citation_author_institution" content="CIBIO, Research Center in Biodiversity and Genetic Resources, InBIO Laboratório Associado, 4485-661 Vairão, University of Porto, Portugal" />
<meta name="citation_author_email" content="mig.m.fonseca@gmail.com" />
<meta name="citation_author" content="D. James Harris" />
<meta name="citation_author_institution" content="CIBIO, Research Center in Biodiversity and Genetic Resources, InBIO Laboratório Associado, 4485-661 Vairão, University of Porto, Portugal" />
<meta name="citation_author" content="David Posada" />
<meta name="citation_author_institution" content="Department of Biochemistry, Genetics and Immunology, University of Vigo, 36310 Vigo, Spain" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. G. Barrell;citation_author=G. M. Air;citation_author=C. A. Hutchison;citation_title=Overlapping genes in bacteriophage phiX174.;citation_pages=34-41;citation_volume=264;citation_year=1976;citation_issue=5581;citation_pmid=1004533;citation_doi=10.1038/264034a0" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. Q. Chen;citation_author=Y. Wu;citation_author=H. W. Yang;citation_author=J. Bergelson;citation_author=M. Kreitman;citation_title=Variation in the ratio of nucleotide substitution and indel rates across genomes in mammals and bacteria.;citation_volume=26;citation_year=2009;citation_pmid=19329651;citation_doi=10.1093/molbev/msp063" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=P. J. A. Cock;citation_author=D. E. Whitworth;citation_title=Evolution of gene overlaps: relative reading frame bias in prokaryotic two-component system genes.;citation_pages=457-462;citation_volume=64;citation_year=2007;citation_issue=4;citation_pmid=17479344;citation_doi=10.1007/s00239-006-0180-1" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. J. A. Cock;citation_author=D. E. Whitworth;citation_title=Evolution of relative reading frame bias in unidirectional prokaryotic gene overlaps.;citation_volume=27;citation_year=2010;citation_pmid=20008458;citation_doi=10.1093/molbev/msp302" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=P. R. Cooper;citation_author=N. J. Smilinich;citation_author=C. D. Day;citation_author=N. J. Nowak;citation_author=L. H. Reid;citation_title=Divergently transcribed overlapping genes expressed in liver and kidney and located in the 11p15.5 imprinted domain.;citation_pages=38-51;citation_volume=49;citation_year=1998;citation_issue=1;citation_pmid=9570947;citation_doi=10.1006/geno.1998.5221" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. Fukuda;citation_author=T. Washio;citation_author=M. Tomita;citation_title=Comparative study of overlapping genes in the genomes of Mycoplasma genitalium and Mycoplasma pneumoniae.;citation_volume=27;citation_year=1999;citation_pmid=10101192;citation_doi=10.1093/nar/27.8.1847" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=Y. Fukuda;citation_author=Y. Nakayama;citation_author=M. Tomita;citation_title=On dynamics of overlapping genes in bacterial genomes.;citation_pages=181-187;citation_volume=323;citation_year=2003;citation_pmid=14659892;citation_doi=10.1016/j.gene.2003.09.021" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=Z. I. Johnson;citation_author=S. W. Chisholm;citation_title=Properties of overlapping genes are conserved across microbial genomes;citation_pages=2268-2272;citation_volume=14;citation_year=2004;citation_issue=11;citation_pmid=15520290;citation_doi=10.1101/gr.2433104" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=P. K. Keese;citation_author=A. Gibbs;citation_title=Origins of genes: &quot;big bang&quot; or continuous creation?;citation_pages=9489-9493;citation_volume=89;citation_year=1992;citation_issue=20;citation_pmid=1329098;citation_doi=10.1073/pnas.89.20.9489" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=C. Kingsford;citation_author=A. L. Delcher;citation_author=S. L. Salzberg;citation_title=A unified model explaining the offsets of overlapping and near-overlapping prokaryotic genes.;citation_volume=24;citation_year=2007;citation_pmid=17642473;citation_doi=10.1093/molbev/msm145" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=D. C. Krakauer;citation_title=Stability and evolution of overlapping genes.;citation_pages=731-739;citation_volume=54;citation_year=2000;citation_issue=3;citation_pmid=10937248;citation_doi=10.1111/j.0014-3820.2000.tb00075.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Biol Direct;citation_author=F. Lillo;citation_author=D. Krakauer;citation_title=A statistical analysis of the three-fold evolution of genomic compression through frame overlaps in prokaryotes.;citation_pages=22-22;citation_volume=2;citation_year=2007;citation_pmid=17877818;citation_doi=10.1186/1745-6150-2-22" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=I. Makalowska;citation_author=C. F. Lin;citation_author=K. Hernandez;citation_title=Birth and death of gene overlaps in vertebrates.;citation_pages=193-193;citation_volume=7;citation_year=2007;citation_pmid=17939861;citation_doi=10.1186/1471-2148-7-193" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. Miyata;citation_author=T. Yasunaga;citation_title=Evolution of overlapping genes.;citation_pages=532-535;citation_volume=272;citation_year=1978;citation_issue=5653;citation_pmid=692657;citation_doi=10.1038/272532a0" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. Montoya;citation_author=G. L. Gaines;citation_author=G. Attardi;citation_title=The pattern of transcription of the human mitochondrial rRNA genes reveals two overlapping transcription units.;citation_pages=151-159;citation_volume=34;citation_year=1983;citation_issue=1;citation_pmid=6883508;citation_doi=10.1016/0092-8674(83)90145-9" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=S. Normark;citation_author=S. Bergstrom;citation_author=T. Edlund;citation_author=T. Grundstrom;citation_author=B. Jaurin;citation_title=Overlapping genes.;citation_pages=499-525;citation_volume=17;citation_year=1983;citation_pmid=6198955;citation_doi=10.1146/annurev.ge.17.120183.002435" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. Pallejá;citation_author=E. D. Harrington;citation_author=P. Bork;citation_title=Large gene overlaps in prokaryotic genomes: result of functional constraints or mispredictions?;citation_pages=335-335;citation_volume=9;citation_year=2008;citation_pmid=18627618;citation_doi=10.1186/1471-2164-9-335" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=I. B. Rogozin;citation_author=A. N. Spiridonov;citation_author=A. V. Sorokin;citation_author=Y. I. Wolf;citation_author=I. K. Jordan;citation_title=Purifying and directional selection in overlapping prokaryotic genes.;citation_pages=228-232;citation_volume=18;citation_year=2002;citation_issue=5;citation_pmid=12047938;citation_doi=10.1016/S0168-9525(02)02649-5" />
<meta name="citation_reference" content="citation_journal_abbrev=Biol Direct;citation_author=N. Sabath;citation_author=D. Graur;citation_author=G. Landan;citation_title=Same-strand overlapping genes in bacteria: compositional determinants of phase bias.;citation_pages=36-36;citation_volume=3;citation_year=2008;citation_pmid=18717987;citation_doi=10.1186/1745-6150-3-36" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=N. Sabath;citation_author=D. Graur;citation_title=Detection of functional overlapping genes: simulation and case studies.;citation_pages=308-316;citation_volume=71;citation_year=2010;citation_issue=4;citation_pmid=20820768;citation_doi=10.1007/s00239-010-9386-3" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Sabath;citation_author=A. Wagner;citation_author=D. Karlin;citation_title=Evolution of viral proteins originated de novo by overprinting.;citation_volume=29;citation_year=2012;citation_pmid=22821011;citation_doi=10.1093/molbev/mss179" />
<meta name="citation_reference" content="citation_journal_title=International Journal of Systematic and Evolutionary Microbiology;citation_journal_abbrev=Int. J. Syst. Evol. Microbiol.;citation_author=K. R. Sakharkar;citation_author=M. K. Sakharkar;citation_author=C. Verma;citation_author=V. T. Chow;citation_title=Comparative study of overlapping genes in bacteria, with special reference to Rickettsia prowazekii and Rickettsia conorii;citation_pages=1205-1209;citation_volume=55;citation_year=2005;citation_issue=3;citation_pmid=15879256;citation_doi=10.1099/ijs.0.63446-0" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=L. B. Scharff;citation_author=L. Childs;citation_author=D. Walther;citation_author=R. Bock;citation_title=Local absence of secondary structure permits translation of mRNAs that lack ribosome-binding sites.;citation_pages=e1002155-e1002155;citation_volume=7;citation_year=2011;citation_issue=6;citation_pmid=21731509;citation_doi=10.1371/journal.pgen.1002155" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=P. Skorski;citation_author=P. Leroy;citation_author=O. Fayet;citation_author=M. Dreyfus;citation_author=S. Hermann-Le Denmat;citation_title=The Highly Efficient Translation Initiation Region from the Escherichia coli rpsA Gene Lacks a Shine-Dalgarno Element;citation_pages=6277-6285;citation_volume=188;citation_year=2006;citation_issue=17;citation_pmid=16923895;citation_doi=10.1128/JB.00591-06" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. A. Smith;citation_author=J. S. Parkinson;citation_title=Overlapping genes at the cheA locus of Escherichia coli.;citation_pages=5370-5374;citation_volume=77;citation_year=1980;citation_issue=9;citation_pmid=6449010;citation_doi=10.1073/pnas.77.9.5370" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. A. Spencer;citation_author=R. D. Gietz;citation_author=R. B. Hodgetts;citation_title=Overlapping transcription units in the dopa decarboxylase region of Drosophila.;citation_pages=279-281;citation_volume=322;citation_year=1986;citation_issue=6076;citation_pmid=2874495;citation_doi=10.1038/322279a0" />
<meta name="citation_reference" content="citation_journal_title=Advances in genetics;citation_journal_abbrev=Adv Genet;citation_author=E. G. Wagner;citation_author=S. Altuvia;citation_author=P. Romby;citation_title=Antisense RNAs in bacteria and their genetic elements.;citation_pages=361-398;citation_volume=46;citation_year=2002;citation_pmid=11931231;citation_doi=10.1016/S0065-2660(02)46013-0" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=Y. Zheng;citation_author=J. D. Szustakowski;citation_author=L. Fortnow;citation_author=R. J. Roberts;citation_author=S. Kasif;citation_title=Computational Identification of Operons in Microbial Genomes;citation_pages=1221-1230;citation_volume=12;citation_year=2002;citation_issue=8;citation_pmid=12176930;citation_doi=10.1101/gr.200601" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Origin and Length Distribution of Unidirectional Prokaryotic Overlapping Genes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1.cover-source.jpg" />
<meta name="twitter:description" content="Prokaryotic unidirectional overlapping genes can be originated by disrupting and replacing of the start or stop codon of one protein-coding gene with another start or stop codon within the adjacent gene. However, the probability of disruption and replacement of a start or stop codon may differ significantly depending on the number and redundancy of the start and stop codons sets. Here, we performed a simulation study of the formation of unidirectional overlapping genes using a simple model of nucleotide change and contrasted it with empirical data. Our results suggest that overlaps originated by an elongation of the 3′-end of the upstream gene are significantly more frequent than those originated by an elongation of the 5′-end of the downstream gene. According to this, we propose a model for the creation of unidirectional overlaps that is based on the disruption probabilities of start codon and stop codon sets and on the different probabilities of phase 1 and phase 2 overlaps. Additionally, our results suggest that phase 2 overlaps are formed at higher rates than phase 1 overlaps, given the same evolutionary time. Finally, we propose that there is no need to invoke selection to explain the prevalence of long phase 1 unidirectional overlaps. Rather, the overrepresentation of long phase 1 relative to long phase 2 overlaps might occur because it is highly probable that phase 2 overlaps are retained as short overlaps by chance. Such a pattern is stronger if selection against very long overlaps is included in the model. Our model as a whole is able to explain to a large extent the empirical length distribution of unidirectional overlaps in prokaryotic genomes." />
<meta name="og-title" property="og:title" content="Origin and Length Distribution of Unidirectional Prokaryotic Overlapping Genes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/1/19" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Prokaryotic unidirectional overlapping genes can be originated by disrupting and replacing of the start or stop codon of one protein-coding gene with another start or stop codon within the adjacent gene. However, the probability of disruption and replacement of a start or stop codon may differ significantly depending on the number and redundancy of the start and stop codons sets. Here, we performed a simulation study of the formation of unidirectional overlapping genes using a simple model of nucleotide change and contrasted it with empirical data. Our results suggest that overlaps originated by an elongation of the 3′-end of the upstream gene are significantly more frequent than those originated by an elongation of the 5′-end of the downstream gene. According to this, we propose a model for the creation of unidirectional overlaps that is based on the disruption probabilities of start codon and stop codon sets and on the different probabilities of phase 1 and phase 2 overlaps. Additionally, our results suggest that phase 2 overlaps are formed at higher rates than phase 1 overlaps, given the same evolutionary time. Finally, we propose that there is no need to invoke selection to explain the prevalence of long phase 1 unidirectional overlaps. Rather, the overrepresentation of long phase 1 relative to long phase 2 overlaps might occur because it is highly probable that phase 2 overlaps are retained as short overlaps by chance. Such a pattern is stronger if selection against very long overlaps is included in the model. Our model as a whole is able to explain to a large extent the empirical length distribution of unidirectional overlaps in prokaryotic genomes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/1.cover-source.jpg" />
    <title>Origin and Length Distribution of Unidirectional Prokaryotic Overlapping Genes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"tqSDH2rLAHmhN8xUfI1gx105kv8U8AaSho7_2a414nk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459606","apath":"\/ggg\/4\/1\/19.atom","pisa":"ggg;4\/1\/19","ac":{"\/ggg\/4\/1\/19.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/1\/19.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/1\/19"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/1\/19"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/1\/19.short":true,"\/content\/4\/1\/19":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XPb995nkOSvRUi9XgziP1WmCh8NOKG9IxlVeZD9gIo2msHUcA8\\\\\\\/ICR7aZLPUMX1l4DO1FWBJfOtD\\\\\\\/3DsRzTzwmo+EnvoR6LjysXbMQWYLm29u9JlXiWEev9QI7NhB8F0gq1heTzDS3PsVK27frkkw+vfqztDt\\\\\\\/qkk3pQbFIhO4EWk7xiT7sjcAgVunqMhSTd\\\\\\\/fzEP9vGuvtbI4k\\\\\\\/TKekxE+c94ap8HF0xvAqatZOZT02CXsLKCkoGwGlOp1SCKaSY5I9SOjWHi9VA5dS6uJ6m4IiIlZRO8Ia5v5Op5ivrM69eW1KwWhNDKrLhLSXm+2AlX6IV8AWZ\\\\\\\/EKAS6UZGhNpslY+WwQRUytiT0hmtF0sxgxHaV93R3qoyBYYID1wrNZulOx21G12osGWwn2s5zuRvvJy\\\\\\\/CIrjFdQLrmqg5stLTMz3cB4PrF0P3IPuVae5lcVZL\\\\\\\/orn+OhqdXZEayCJtOwZQOUnBcFFH16WFIRqwKcYrAdn+z9EK1sWnw+t4o4sBIK1dJUVm+wDiMZ1COXFTSqLFf70haF4VaIW\\\\\\\/mcWNgPMemHuzCa5Sdea4chkl502SMhmipkhUNwvR5tyGc4hULuQKaF5HoLva8fNEgZmTDDh966YpETlxfvcEct7oiJk\\\\\\\/2\\\\\\\/R6J0bP5pbw3MfyIHuvco8L0z8HwyAkZ5tOnEG2pEAWYnqIOSMlzULWG\\\\\\\/WySOqh65mg4CasIQm0eO9ddtRrNEBi0aIrZ\\\\\\\/oMOWVl3Rx95XRyb9iCVPiSViGSUcYG2HxytURa2jXanEpyDgycWaCPvx9yjpDsUrdtJNm5X5IgFWMpfM4HqRlW7u50+WnsbAd43zkZntVaoAM3XD4\\\\\\\/F8YxGxAh4\\\\\\\/r0nAeem1KMddZ3g4Ir0bOXgWbxikvWDUCUMZRmPK\\\\\\\/GpIB2Qs2nKphDQh58JjWlkiKPkLQa\\\\\\\/qAHEun\\\\\\\/owx\\\\\\\/x2OKDlfihpb+seXvZlExXymuTAjCemH5Q\\\\\\\/pGqdOFhSV8s5l30n7c9cxqi7dgnLYD7z329lVO1BmMyTCEMGjdfkK9jIoYEEp+Od8dY5QAvA6tiIGtNgm5vfv7b\\\\\\\/+W+Lpn5ZjM2fc8VcbK1SjqYCdfL7RRA\\\\\\\/G\\\\\\\/dxhweacqHLP8u6p6zuBqr5zaEZ96J26E4zGbk1YawdQ1mOOjF92tfHz\\\\\\\/mumcg1ctVcy1vMfOIzM9FYKwjRYfn9Gp0kCu7ONrXsZA+6pDnxvOSlVImgsFNHVdhZ8C+w8S084yQniw+hc5She8vLKfQEz5iGssllkaWjXNPS4RiY74FuC6WCrLE7nD\\\u0022,\\\u0022iv\\\u0022:\\\u0022FQfiVvHtNX+EhWxFq9SgTQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002262096611b80dea74427109140f2887bc0a028a25dc2c2b732cfbd1d9776e7968\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022L8DZF9S24zUVETbAgGo46DMsdPhWCKS7bxEYrjV2m9X2YEIMliS+4DmuAVl1Q8NA3YqS8CyRhRdqe5kV3E4an\\\\\\\/hGjXy\\\\\\\/tGdlvEvvuXxrwD0\\\\\\\/KXDbCXe+Yyd8M+i3rT640ckNF9LdVFEQ3rCYVk3gz4VrzWF2wxPKMww4uEh\\\\\\\/oOVBQOK924Vso3NtDs4TftRy\\\\\\\/CT5sGUDTRP2HyLw3DViQT1HrPLbz+dA8hKb6lG\\\\\\\/ipBbxTs63WlvujXBABeJ1Uin4XNoHi0L6QtECDstlYqaOJ+AhsuaQmLBGvtjDPuL4xCRHrwLBhtOoSOlqYdmlNyXQ2RHXxQqzVySTnaS0eAws7B0LIqbzZfrSIqqTwchWVqg\\\\\\\/o6pXiFUEa9uiJZqZd+2TXBy+Gt\\\\\\\/wzpk538F7jDYKVhGt3DQ59FMxonuE+MguqEcYyVOYxO28TDP+cVAiBCuLEWbXeOjI0wQp\\\\\\\/sNoDDEMk699AYTcIX1gbqWJuKUIoxGj4gxk0y5ZNsHOL3x59QHQrwXR4\\\\\\\/JsColhAJ2yoMQpOm\\\\\\\/zJqotuPIKmzMu11XoldabIzeD824CYEpmU0hLFdV81gNADymBaVRxgoSbFN8tONHxZt1LKkWUsTW\\\\\\\/uVHKrKJeMNATYIbEPk3XgJ0MSOb63s7PKMcc7dyUcn7BE1r3SXXMl\\\\\\\/nq8Jrc2PEKayhlfwFxacjtUI3QYm8czIM+Vcr9QxTTvPYmbP9XOKkz79ULcxT\\\\\\\/h40ukGH9dMJ+h3WxUkX1uHOliTmYBh0uyo12ndVzuDj7VMU3BefwrradmcAPn3OQbplhiFWDTSeg+W8z8+X9ReFS00yKMOz8fKrpu53INZ1q7j3j3RrtKjS7GZtV27B6jBjb54zHt\\\\\\\/DtLOaupJ3Ff2xsRtt8gic5Eus4rGurfIRyjYFXwhpYVnGANYS1m6rKFtFM27Rg9fFkH2\\\\\\\/lHJCnOx6ITGES8emlXysSmAZskHm2FR\\\\\\\/hipgFw7PP+8OVwoinJzTF7ma4b3tSGI9AX7G1jDMx3PbqQSsKgyAbIZ8pFEk3iZ6gfd6tN3LB0W4Dcezrv1VRKN5I6fN0pVSXyCdyM7JDP5OMfKC8uTiIuL1SYy6NmhmPCvP8hAlluhr\\\\\\\/9IJiEK7mTTphea90AcmXiCBTbmD6Z16\\\\\\\/tRJzfm1W9ZZLe0IDWKWF+WvBTAJ5PBrYfLbCotxa+3mDysh6VcJKqgDqB6OwTHhgtVIkz\\\\\\\/0sQ\\\\\\\/h3jvyCMbPPhxw6oWR92834C3CftVooAJwNEW45nNjS\\\\\\\/nbGkL\\\\\\\/6Bggvd3+HridVwezghwINuE0lfCC9AFCUyWPRLIhR\\\\\\\/7MkReUweHAOgw+0z3ETtcdngjSDM7dqKlvY\\\\\\\/y6957xYCtHTgcHo5XFCgipvcjBo6t35V10jMEVUL8svUAg6F3nSpDLhJtEvqmY7ie\\\\\\\/Po5Z9bq8J8nyPoiAlgys2yoUJPhyPEtaFqVRA3hUeDVNFqK2DRj2OFRLAgqJb\\\\\\\/LKsAMv3Wp4Qa8WbtpN4PZW0UTf9kl90LRCcXemHx8nHRnd+CGExMIpXqBquKKD2jcseqk5Gpjpu\\\\\\\/+mkJlpRolkzVbTab6+X93zxeajevdnLEl18sqCrVqKYDCmIc1PzuqVp5uGi8XLSAqVi38EFhbNhh2PHHy9V0BQ4ew2CeNrqoELYtwAH8K+c1wpspn5KCpqf+GpjAejePYxTYdrzNLIT\\\\\\\/A7GEN5bPLHTko57oTbzWxUU4E4qoR77QrpEUBbCV10xiXEuawAlWRtu28eOjRWxgJMf8XIuY+77S+U86DLOK3r9EkViho2n0nXTc+MlcXkDAMuzEz0JIElmo2jP4+swzmwUTa5x36sqMGnXa18LppuEI3ojDbpdb7opZ+Vl7SZArvxh4+lW0BUMbV9uElInngx3GJGdz9SSjbwhr\\\\\\\/MAL1xYQ\\\\\\\/+DAbrvSMXqMFQNmdlyHwrxIQJdWWXQmhaaylALVGc+fPrCw74kpRgaTt\\\\\\\/CCAzWe5NwK7hYKPE8+4G6M6jEsYuj\\\\\\\/5b0o8agbsvHle\\\\\\\/93ZK+78qXA8rg2F3oK8hqp7LwVKWcaGTui3WMlpC\\\\\\\/AxEB2kWAXmD3rNqqU3PU2eEy3v+ZEEF6GEKrqHjKx+zK6YS028IGBPh\\\\\\\/V+lA5pb5KWuUmZJq6\\\\\\\/uWUWlT\\\\\\\/wb\\\\\\\/rpX\\\\\\\/GUlzqd6xgDc1Qo7y4RZguEhKhZNFYTDviY7EBp74joSXlhFnPsSPb3F8LyT2d1IpzgTg\\\\\\\/bnoSPk+EFq2cJQQFMuBzmlNVRFDM0wpoy0H9+ZUnGshPvmUQrWi4oyiCAvPinZO3XGp5AiexNqCWMXlj\\\\\\\/XXi4tW4QZ15aufVYSuO0Kz+aNFyM6X8PPyf0e\\\\\\\/qT50dmUtdTUXLXbfDqzHfywxQqhmrgncHOGltoWw\\\\\\\/vmtTGf5u1MJXXprgetejPlDwhSbwMAPigqx7nu509OYy9UJixNPQ3eC839FrSOYLsNXguTvpQOooAEyP1cujD667+hy3N6JQNgP1OeEr9jsAnD3CPgPrBEjcwaUiqNkLp1I4qXfWX\\\\\\\/deHCIa6gAtyrRrRrF4kE8xcsMPniqgeYX3uYsKSCrOScordGSTXkNbqFEPT\\\\\\\/P2B7aMGhYtTgE5omtiY6vxdPM\\\\\\\/07q+abHI1K0AWuavGYuEDWK78+7vV3wUNbWlyPlxr11EbHDeMh\\\\\\\/h3kFs32OeZTI3r4mVQ32mRsef0S9bhNpILZG+4AnFqn4b\\\\\\\/dQ5vU4AlY5KNJVlvB+AKH7RJ4M\\\\\\\/XJJ+Gfpa7Yw5A6PAma7mvnK9ltsgd6N3sURwYQxTgjebNDb5CgUIjmOm1e6zkFgYFDAknpWFhWSsY8YPT8PApU9TgdSUZXLVlhJYyuGmFhd15UWWjW\\\\\\\/TQcc1\\\\\\\/lEEXhEKAz2Rk72U8mK5XphPpP0oYuZfNfeOeiu9adnJH6oDJ5Bc\\\\\\\/BNIZ\\\\\\\/UALFmRzmI3jkrXoPCH72Z1xOu5\\\\\\\/hynccEOY67tybm3XMYQqpU5OFZetjgK6DQqpIoWaCf+65mQlhqQLG\\\\\\\/goANSlNXMr1dUQJyfU08l47kdk+Qv2CFn8D2nXOoKy0VfmefzFn0tYPR98nDwf7kYcp+zvnifQlkCGfKWe2rmTOXaULbAxIqGs2r9H0fQIrEPC9MhesqJ\\\\\\\/ulz4aHaShbepLuGhPDWQOR\\\\\\\/OUAAFEsCRsTdGmDKQwU6OgBsnQNsAl+fKbN1Wj6cBUH\\\\\\\/E1Kp8z50GzL4X1vyjVenGxLqAzXrL1v\\\\\\\/CtaOB8oea7nS6sg5GKUbrSuhyViFSlUOpWxzlW8CPq+uur8BK3A7zow8JtDQT9D7PsWEwc1kl1CExCZ5E946ABndFl+7smOnFbG0Al1HuDBf9Md1XPq5gZKhx3Wh5VNeDhGqqyyeJ3k\\\\\\\/96ej6e5wAQsGg+eOb6sKLPdYiW\\\\\\\/pO7h7Z0o5nb2MpNfpmEnkXUKkGG+lQAEm3KO7qrc3gryUbaXhTc60u71u0BTP22wZ\\\\\\\/96RzMgFkc5v2GON1gaNm7FzaC+dzq6VJX\\\\\\\/wV56LD0knUaBOAAyArhkYEKwB98Tjy83n0hbP1lpBlYs+SdH33YmqvJWinDk99L1Tv3FOmyJX1OhGHnZRxWxw9BOEz9Q+FFkXnOZCnl0qXVruUoUmvjjW8TOPTYcpi\\\\\\\/1b0v4bpDZwvQLiFFLBPZTXzKkUo8pB9QhdNRPJKlw9jQ0BUxIGYgEiTXd\\\\\\\/LxGrRuoOYyRRjv9IToNpwezSwMNYLfJ9LqXOq4ateq5yJRY8qIhcRPdUTWpRICi3ttK8J2n0lpde9q0qGSwB8dsJmJFr8MDbodG+d10+7ay4l0QjDuGvO3cCYOIrRKRrXji6jaBMbFwfbMALAr+2aDLz789CgASOpQLJsSzwEE36DIuANIPZ0rpIXbvTzo+ohSWMfWNlvT5HR3UMmVtCvFCCMjPHSXiumzrIqanqAffDK+tq8tVIfhe4lCLdiEI15vHRbfsfcfnKBd3oU7kDBwkqPwkp+gOKj2oMpfvjiAo4JVHzTk0BT5\\\\\\\/NwzusKxit+1wo2WCkE\\\\\\\/HN3tgb3Pclyp6iAe3wPdDvLon6UfmMCG3wQ0gekNPf1CrFlntLA22HO+0E\\\\\\\/eegpGQXqg\\\\\\\/gDipezit03z7X6SG72i6dsdYDWHSvFF35Gp8\\\\\\\/k\\\\\\\/0naibpEcMPWETDLEXR1JTXi3nlOGLMlZTA8hPJTsFHbvq06rQLX4+HZWh+ZPBx5tUU3NJjUPNnKEl05XyIunlxxagagqYYbOJ15XIrFlRNM4Te7RHBhZU+YLqlxEzFQBDAwi9goB1z24iR7peleOYc4RoxSmX3zUqy+UMQXP6n2kVJJR2iJZDMy7gRuXFSzD2klyzJY6sL1VQSl60hvXCKUKkZa8bOzxNNMTtaGu4Nc\\\\\\\/N4FZcKN96\\\\\\\/k7+seTNOmN74Jl2TkQ7Lk7K+wkcZ8iRty1Z\\\\\\\/fHUaVtpsvr3oeJlrf\\\\\\\/gobHP+j7zQJWqGD9RpBELpxhHbqi+nYVkdfbzCsNSLmAi6AhPYoAJGOilVjjyWAJMQJ8VMbFrgM5wWmy6qNSxhcfLNUfxMHV6MFvc6\\\\\\\/FaQ\\\\\\\/qPhg1DJl+m3BB\\\\\\\/Rn0Gu0QGWWPECiy7CS9wXCd3ZYXqm7VQ2RyKKCklWrW1dJQ1U4B9JYVAl5GvMLLVWfjDYOPujIMAYvIQyRaYGLxbsF4mP4rKHgz06FDdsMxbojLxkhUmKCOr096FwIiPnL9N9HHJGKAUJXEg7v\\\\\\\/XDh\\\\\\\/cxYgiQ27cj6pwYnpQrCUrfeE3oPl1rlCJFXjHQuA9nfuWlgm409ay+uoQLhurS04yAtZlwg\\\\\\\/lfyd2ZeKyzjztERu72OHS9xDoHZQ4S\\\\\\\/QbK3R6Ut31J\\\\\\\/DE2qAbqDd7lI4Ib35RL+AInm7AzncZKDfmR1I6m961729AfzoyIEOiwWszZpZiu7XhcA8fdJUt8+mV9u7YjSe1AeXqLMWqWhLgvjiCTlIZfx3YwtSvRc7WkLSfxmb0r\\\\\\\/uAq2yFt0h9FbzT7y7VpZ79yC4gEhoTtxVb8BKA3rHyvMZyUbbHs4P0AtvOnEvmneTeml3lNYkasTbZBrh5YwB8Gt\\\\\\\/WXfx3NzOBASswecUSzQB72AuBqROcj6RfSlr2r\\\\\\\/CtCX9+4oJxdZcV6gaLSNUio7Vos7qOqc6iy8b098csVeuQb5fvjL6T5x3U57o3snUpXsWgLgmkh2B5SaqYUT3rvIIgwUJj2WIbgsBKIXUiGegNUM5bjIQfv8Ar8i46PItmSwyYE0nU2Q0POFWQpLwks2HBeTBYtNiK\\\\\\\/QJQlhRJMS4Zj19dzCUoie40Ii7AIfWff0BD6m5yxR3Skn4+nYMS+ak9Fugr5IZB38cuUOw7MG+C0Ni4mLTK4q4qVXNqyM5\\\\\\\/pHc5AKp2LkhLo1ndZAoiKWQ7nUK4DTVexSPh+7oxiiQsIEzxErpzudRhH4mMax23DfcdVHyYIi4uNpu9Tx6S7OwGSDrjo23Bb8QU6+DRhOdteYX81ldioqCq30T9PWXfDZk1tjHYlqQvYX4ZYjCGIWgO+fOeas7+1gS2EsZIQKZ8fMhxCSnmLmLB+EPg+QMe8wh36K1Iv01Mqs58C2y9rH3WK09ZsMuOe7Ksp4dFAexCC7mQMqbqGxjB+PdpZJMx3BU\\\\\\\/kp8OaKRcUGXcFVbYo4M7rlAIrNBsOtTPjb0SGMqLrmWC94qSMqOMF7LGNjkXyyANbF0bfz4Jbk+kwN569VsVadumpSzGmDK4FjfZi3BsfuByh2XZdhso0crMui9v9dQccI8g4zHKgMocJp3Vxjclh0ZdkiiAUj4Lj2tuEmijnMcMs86LehLC6l8HXMgGQlPWC2Z\\\\\\\/h78d9722p1bECUajLjWpsY7\\\\\\\/Nl3mYAl3PqE6z6p6W1PNrnzbwOS8ZYkIkxvNMjmGYzzRGCHZ820OfzcYxXpHlz75gupEzngiqEZdFMpeFkHvAjqzfk0i6J51sC1w83F4jmK5UNChP\\\\\\\/qoEMrD9fOKkv1GtG8hU+yuJFQJ9JfF1ueu2XZkzzZW4X1sye\\\\\\\/7DSFJP\\\\\\\/TQIi36beYqstzS+vgQNTlHTiU3nwYM3fqtat5dlyh1tX1IUQ+i5phxV\\\\\\\/FW56DeEsqSnLuoVqE6AJBzRtXQaxVhUamzZ1bJmCicWKom3rpeVH8w\\\\\\\/3skriL\\\\\\\/tbtZ5NKf5ozgFetUAeUdU+mVNvam2HmaJ7a5m4JDoluEgY23t1FJe\\\\\\\/Nni02uP3R2oxsSluKLVG\\\\\\\/h4W6qDgrfcJCrVI99twlRxdwkjhjoJbRNuVALe\\\\\\\/iIDSKcB44XtYJfAEB330xQOjTA3BKcb96H0yzlzINE1laZbxjGhKoXJJKsr+yvkzuQIOrlxGS4\\\\\\\/\\\\\\\/tPfMJhbnl643FACX\\\\\\\/qkBGbnTOgCj7BNMifbo+0KjsBdAUv9+0X737X\\\\\\\/Y\\\\\\\/3HLC6+Gxzc9YRb0G5PmUucU0H2po38UsmU+zGl1Zh5j3XQINauCfDk7fO64ZqkTyfAaFY\\\\\\\/q1mww3ZAXrfsPwM5LbAwOWRLVwLZ9xbMZXQd0ykkijZOBplMzDIuvDAlS8qYNzBt1HWX9yuo+TVgUcQ5utFML3Pp\\\\\\\/+wVD+VqtC1k1LhRwoYTY+l8\\\\\\\/B3OqK7cTqX75Or4MkQA7WG8fX+w8URgNWB6hnbGAQ517WvymAV83Fb1riijFoWHdvR+aNmTJYcFKIYNh4KU8SFYMwpVjEyzPDJNN1IVmuEJV1VhUM30uxw+WHlGERz5SWdtMDWmhnGc1a5eMEqYLREF9K\\\\\\\/IYRsCPq0httfK+PmgZnfI\\\\\\\/DtthqE85PuQ+sgic1opl\\\\\\\/SUQunmh701Ev\\\\\\\/h2M1gizsJBHOr30LXqVOQygEFVkmvAPznvonz+deGyIG6yFbg7pifX1keLBwV1c+Df2QLEN5zLVRzPA8xL0N0RHmC73ZxkMttq1decizVumTppvgWcD1RAcWErbo2Q95\\\\\\\/aDjINU61uCCskPch1PVOL0qYOG\\\\\\\/3KjEnlBo\\\\\\\/IIt23VOWVbq\\\\\\\/Y4q8QRK+yRO5nPQbFGi8+GI44HYeO+H8IYbK1vlleDBXqPC8izCo0jVT9huvMBDRygLMPoiLFAdoT5GBb7u0bePAtWZmyTpcLDmyubhUt+TBZjeDsh9XZ+lThO1NRUUeT8WOOCqRc5VxTHgOgW7hdVSoUj3AQsW9FaPzRQWWPgQ83SeO96bbqJ86g7WpxWPl0lLMhXDZXZjut0XFA8bXchYyaVO99qtlyvljt3Tvt9RdOSeMVr3gbJwM86Wrd8E4SnnblmKNW4S+NIloLe2mVvabOfyW+NmlS8\\\\\\\/aosb3OcKRCcBahpxB+I3bG6CZWJtwDTxSh2j\\\\\\\/xF0ZyGWKuadvMVTz+yi\\\\\\\/3GpIr2hYbSyab7FtDj+jt\\\\\\\/QcpTQ9o7NpGiDtX+NTjzVB9bBx2pd8k19Faq8La+TVvibxw7RxMxZjvTcLmgqLn\\\\\\\/bDSW1WF9siAy4l9m5NYpGLG0q7I4z82LHICGcHwLy7eEL9CPLDc6HgaggdEMZAhgSXYEVEVFFfIJBCW0FLGUB7JnAxjaNz7IHO2ZDG69oJsckE2h7Z7nJPo28ae0kIiSW8HHy65cTAwGcZB0+fQBDVfKPIUk0v\\\\\\\/BfAhn9PHxBviHEU7O+NTq+1OUOfi9XyKGxQDx3mY+5DsxUu9aUd\\\\\\\/JeE3os9tPTCCbVX14IByqVnzMIwn2sSa7b5ZMMeyx3EVB91o7DqtIVwdzEx\\\\\\\/l6ChqNEmE5O+iy6VE+APJnUu9o+BI+0jZrf5g87wJlZJFWARx8MdLfbEdMEgBuFeGSr5f80qBi0DV2qesIAj4GBCwYvIoLzQlORsJ7Y3k1h4bCroyfkwu+r3506zPGznFgSNQ\\\\\\\/X\\\\\\\/9tNx8JKPWtggtaMzqGEHAjtgF7CN6fJzBnXgSpzeq4hTm+xpVJ9okFP7p92\\\\\\\/ps+Z7D3FX1OBzIycaz63RaWeaxg3Qk8JjdFrOyRleRNeYW1bUJG4S+jRL9ZGP0NZxQ3i+mSeTXWTowfRddExRpPrdtyUzTcv9t2pp+gVTNxZf\\\\\\\/VAae3tM4c957RyQj21rzr6P6rK1MGCeLCttVzDFhf888FwJfGP+HsoxOLx4A6EXX3IK\\\\\\\/hgbKIWqiRKKantfj1jg1K1dX\\\\\\\/3brtvrbE6tKVNrP1TOa4sOoal9xUQvEeqkNY9v0QaB7gicqgZ0zC5+g10CRj5DXf0+N+fk09RDjTc4aJ+Iyd24wAkRLJQPCBNRFwGu7tiHvm+DekPAXVyPWK4FkRiR6ZT2XPSyHTW\\\\\\\/DunSYyxc1VKrhuEP20aQsiSRG80PJ5fEvxY+xMC3MsrR2gHgOlj08gqkg6kPmSsFTNFhRhuSrCaazUOv6IDLtrLTo8YKLNB3i0Tbf8q\\\\\\\/+eYepKdSd3nDUShNVByN60vL4usFDumRdiRpbgUrnPwzBgCvh0FTdzfN\\\\\\\/uJVy2p1CtX9yKMYZE3pfVcFiOx1xPP3e4MMbPyUG8zn+Fz6rl1pn57x0UZ0xRfuegvTaAb0NGgbSvFRgNZIPBKb4fxa6h2D66qRlL2\\\\\\\/CRTNUGv7wNucfTEAwDmaYMKSYeVh7Al2I6rLMA1ksFlBZ1WxRVbUP1Y+OGVzGu8G\\\\\\\/i4defhHFxohchT1dELV3lS++vH8t5vgsZ0yTx0Bs1mmCCju2T9dH7cQjbrY1Ub1EGjK+8QW4vKaeRONej1GmP9Xat8f8LWMRzpCCMX8A5fD+eKqKzz9HPkQaeMHTht6K8mXMUI8kolfpbz4URBb115snT6gCGAkMbyoSJykwwCwX\\\\\\\/ypML1Ha5v7CLDcLQypMlQuPkUOjsgKTImk8UC+bKv\\\\\\\/a7\\\\\\\/PHVnQC3h\\\\\\\/7VMacsfVO6lWUqdybGGcJlChfPB6EQFox+uJucCP7cMIyhfSE3965DUExAbGige70+Q3ycb0cC2oJw4PHAI\\\\\\\/\\\\\\\/TTx1s5XbyUWMHkk6AZ8in3Zk8RWpBDoB3NkBzB0uw5dIEqCAiocbnhw4ZqxJBUyRM8d1iWC5wzde9wpWnrrrE3Yu9tyxK5hU1k\\\\\\\/lNZ4aI4VuThKKiwJrbNYN1p0aqfxufVS1c9ejquEvBuo9dKx3V380otvCbg4q+xnXWTPKBHGkrNEpY7Ije9cOa0V4MnEwRNYuahIN5AiQ+D6GGIHZ4P3feiN8Ho8csI06HDKdtKxesdXYXsLDiuNEYxBxy5rCLE6G6ebqYxtLMhpmuqpTgzwCtRqfV5DsWlbX4d0AMhT5R1ZdLoTjawwuYAEWMemy\\\\\\\/Of6Qw2GT5qoZTmW1TAOO74PG7My2OhtaQzx7WTgV8\\\\\\\/hv3FEaJL3QW9WTbjopBHHZ2RV3pHdegh2IVY0QEDjFytPshT8awG1NNNqNQBR78Zi7NDv8gPHpiMbTL6PXmMUMpKJdUpHw9bYl+Of4VJ1PN7QnGtLf6+fbqoNOQ3E6ESOyN7mjZq4S9PzxK5bkLNoqjhk2e31pjL0kAEMNYvfCdbw83WmIFzJvi+h1RHdFbbHeTN0ZkF2aNbi117sqje\\\\\\\/u1keFJ4gVUGO\\\\\\\/GuFmQDGXZiXfuHNZEh3z1aai51DIIXYpTJcLphQ0kYdJKzBO\\\\\\\/+9UF7w6xUHk02cH4pWqOXKfVWHubT99vxS5bp9mteoqmD7PA+sn5Z4SpXCKByH3UnFaFMPixuJ56uMckSuUEwPhUfuIzIWHIlLiFINp3BWQlpqodAsGo9HsIiXpsVhla\\\\\\\/MsB504fpGe\\\\\\\/FVGxFozJYl3rMDcEvTboa7x5LS7\\\\\\\/Hy\\\\\\\/XLZGpmzRbZmMgCv+bRAdNXwenV9uXKNzquZEhlwdi06RFVrxQH0Dt+YWqUyQqg2BQ4Ex+hv8rZiqQchMtHL0k8bmBGUKDB4h01RE5qTqLK97gfOqn5aX7nHo\\\\\\\/JWDXshxIEsxNhTbHn\\\\\\\/Z9jftT\\\\\\\/5MuoAQ57aODJOQ5Tx4khQy2uzZ5rNeYKxZwtZqpOkGYO8jh+Bp\\\\\\\/iEXDvs0iKr0Vv5\\\\\\\/Zh1wzgzfcZYiaz7AZCrWSTvaBol3D2DoZwL2cY9J\\\\\\\/xO6BlMV738yUrKt4hg+d7\\\\\\\/qieuHzz7NpbCcdiNHWRihjJo2830j4nUU2NHB+jYpbohIyG\\\\\\\/CCknz3oJf3ohxLiPWycXRZa0ecbl+\\\\\\\/2q218EglNJJhUCCwcKVKVCzpiY+aCA\\\\\\\/9LTa83jeOu0PFYUw+V94\\\\\\\/XzB9mZmBKqs58xxBlXUDu30NK6h7rrcPLN0VEqnVfAOdwNK73kCKCmyjL4\\\\\\\/kdxa+RWk6gfc14gzM9VTLgYOUg0h\\\\\\\/3PUWuYwdxWxverfI9AjHqfFl9d0IQPQvplqa+cVhNxaoZwIm5h6UMKCnPc4bQh4iwajLCksF6eT6Q00Lz3goNnhqlzxZ0DFBoDJQQ0EgsnWUofBC5icqa1Mu3clw1LRb\\\\\\\/SdEm6QHEcZWiZ16N6e1Xkc2XYueZDrOc10rPr3dexqR\\\\\\\/CM3Vq01yoVqO+BR0lRxxTtc0NRNwxzg8kwGhz7uWIQWt1fttZZcISi3B+nZ9Ac4Nm3Mm5VLmrA7UOb1a6ld3\\\\\\\/sxLHAOkwMRQxXwtp8TeTEgdjU1s2SDzyvnxW\\\\\\\/OML06TEuJkQDfghLdxSVfP\\\\\\\/EC4XRLjxcudDRcCRnAi8Wxs3oZKOiBi1QoWWnAejhdkiUatW1xH5Cucp73PQajh7ZoDMMwuTdSYEv9MNFJ7BOQk8qdTOK4+5bhzxjJJRxzE5CankL4waamTA85PuOiUlDeINSxzGwlw8TQII6F8DywtZHB2b+J84ccgNXrzasJY\\\\\\\/avUEvKZiHqFWeVvcB9qg4oPjb5wiexvxNGXbIGFc0msaQPL1tjFrRjsDWxS+oFxFvnxWFMdjpzIX7ARu4YVBNS8QHJ3lup1XmM3VkvyJWM8ER0sp8hjc42OxXUMTa78taB9ff9bIePLWh3FRdLjTBGzxgz4fHzpuEGrmkD9cj+RawnKb365ncMCuaOxCkyologhiDoOxuAm+p+aGaJz7LXERNhUNb5lxw\\\\\\\/yQ7GP1a4cbvFJDh\\\\\\\/FvVj5cZ6HwU1GUYvEWKbVR+CCKMK0l8uymgt91iicnvrea2iEGV3c6XgdJoW1Mb3zHEnOnA0jTlEm2pbufjS6cSTtvxWa+TcQSeKclMSnRuiqISXNesA0QL1ItKZ\\\\\\\/prQk6bF+KMFCUc3s1kmoNdzolyOryTe2cblCdTm3f2jmImHZ\\\\\\\/kqdoHJeVd6r3l5yIhF9A4X9aR9r\\\\\\\/nRjLtV0QuivORmZSRfXOm4Pwoc8LKIGNgnk5ci5Xp+5c6E3KUZ8jH4wRZE9LQDdD\\\\\\\/hb4U2Oaz7UwO2VVjOBqSb3QLHHNl0NfcXLUYqLHjnAkCjIw6f9ekcpm8FEh4crOJ4RRKd7EWVKMLBKNS5NSPm\\\\\\\/vOAWU46jYh7IhAfxWhzj0IfGZLrF1O1bvwjibOWNxOT5whIj0IQWzgMekGcR5+QL1Y4S0NbXgBrrymWmYUBQ9OqzIRPA1ARMxCdKD7yxmrbKUHJTE9hSrujyJ9wy1PKn4nwarfdSOTX0kMJlFUf4xhjX9svf5HCmSzSxwtU7TKdgI26Q6GIM+a1OjIfcb3Xz2uMw43i+B6Mg5CFz+g8cnnTrkx7WnFyYGkRsySLjOCahsqLluTsrWZy0lGbHwEmujRFVS7hONYmdx8Gp7P6VJ2MKYD+3\\\\\\\/NgDPR3HkBNmOVdvUR6Fp7ydweoAIW35k2w240Qvo+tgHYm6v2nSgPHZTOhM1G8C8gNKvWMfPRYWnPbO1djrmWDrwMnnNLfZeirMjwjwnaKf\\\\\\\/pUU00XDHR\\\\\\\/cd6JvzobyAxGVEUepJIk2y6JF5nHW0j0D0PerHkIDIPX3IWm0yr+GaWcuzTmwfqSwSfgJ+wvPRaTaroxFqgMRvnSQdwQ2B+TjKG85vN7hBsb550A2XXWIGKR7svNpORJjGhfdCU5wpunal+ZcZ1IIsI5bX75oCr4U0RR7GQXH7gvc+vE7M7iICUttz4JQDu6e5qu99tJqH5UDSv0KlzIgm7i3Qy+h6uBqBNKG73zaFiEY5KsUoW+acGey+E19XkCTVrX++\\\\\\\/doESNHzpHSgEs6tM4TFM8Ep80vc4\\\\\\\/TTlXtgkqAd6bjaJP4UmH63WDZFE1FzLwa3uS443G0tx0WZWR+vAhR\\\\\\\/6puf3PP0OOpAkAetckO45IEaxMZB9H4s6ciWVWN6zuPyOruVSWFcsKDpUQfBTeJyR0AFWOS0LnIu8g7cdMAUfziC4yEwqmxt8brd1wnDEzkSIxPLjxMt7+tb9vGQh2rVQ3Eb2\\\\\\\/4\\\\\\\/lavgXbN4e4zJzYr1RcixVcBbz7VKCQ8HzHzAghAGOpwGGn+g2+FdvDCfKeN6mcqJvpFo+eMd0TWW1ZqF7vIbShw+sotOW+ddGco5N8nUnC8F+1BEX0Tq3caxiUHsUPeLXHgSxq+\\\\\\\/hLzJRbR47RNL6zCBUOyxO5dhtxGCngoor9dNHfwYZ90FocugAHu9FCgu8xSIP\\\\\\\/pl\\\\\\\/IntL4iJrM+qV93PfpZTrTRHnw2A6WjOb9DvzgConKz8MHc6Jsja87JDpEXBR5sWtwgBE1gcEVo9kCqOFx2qx9gftZQ2G4AF8Z2TvRiYVDTIfEXz+yM1L0AA0kc9czn1K+0tJp9pNHii5AtbbTIhoiLLvauHzEUaqsAte7DRkzRjA3LY6pSQ5\\\\\\\/uN50\\\\\\\/DsLNE3tlqFmJdsWfeOuIY6OCl8t5BEezh3mNYNPyJwCopsS0YrVRDV3zhGybhK6sDWfNnx3l\\\\\\\/ym4qYGipzFAvHoFZXkgFZxLd5FcIQ9fDiQjGTfRGA5OUAjcnmFlGazf27QYHSiGD6+UL0fS0Y4EfCYQfc0fkcWvYK8EtPVMtXR9FYyRbbXvvenKuU\\\\\\\/owIf\\\\\\\/hQ\\\\\\\/Z7B2gRsflX71WY7E53qF0A3D4lEns5hoUehA8MR4XSCnF1vgdTD1nBn\\\\\\\/ZFrgEA6T3uUZfZaYzIkEjkrS+7DeTp1lZsdBAZN4BEfcqR8VSoxZ1xmxpMPTG1fpnLpRuQrynwRJFlFzAfTGpBkVfvWRylSf2goLyilhjPaXm91NFI3E9v1mdRuuXv6tXSWrFCIN+3NVCZ3GCBCGbfOhO5YNmrscYmiXKJsR42\\\\\\\/agSms3gH8p1Fz6DoJ9LLPSvGdhWeLbEU57yrn9Jhyscb16tEYi+9f3CjWVtAjkynTFmMHbqAfTGYmlAHMmvP9cRj2f\\\\\\\/8z2vlaCik+icCbCQERKswUqNrVg5lTsL3t+gTeAqlCv+1S4NBmqD6zz9rj3y9Wk8JQsTTT\\\\\\\/hv4yfzulXIKwZOMYB8XC2cwHrcFXN2LhJGUsxRHMzUA7+qLYtUADvCzXkFIwAvC2AliwzHvBB6W7\\\\\\\/pgHDY0aFEVFQWaSLkMH3Z3dUuBHvZ6oIXhD2cgd\\\\\\\/kfX89jDtWlkOJgVR3ZyVE4yi1aT9lZlejenmI17TqLGHQT23HZ6Q2M7hjhfQBgNwfx67ZHiqWiuTkk7JXa0TDB+2DoXuWRCdvLrURAhPXBHBElZP\\\\\\\/YJrXM7Gufw6Ti9NDEkqlvzj931jbcmLoLU49ZE9ipbjIWbH4+wF2WarTMrGLYJtT4fSjucNcK9ZoeLzfiW8SUf4JjP9WoJG5c0ZkrOFyz\\\\\\\/W1AoCM4i9GE7Q+lzWA5Qc2B+eF7lCXimED2tO048eVkWD\\\\\\\/7Fx00\\\\\\\/rrpZghT0XdWLjkFFq2Um6awoi28dPF079AUfnAvP8TbEb5z8ynsv2+9cpvHHnr9hlvIKqk15CyqsFPcOdiL1Z7z\\\\\\\/A+bXXsMQZ9n6pnJ7P\\\\\\\/e\\\\\\\/f+ZWOrAIIVyHjvZ\\\\\\\/MixnKCkKJgYzfC97ciLX+3dku6RaojV3SaU\\\\\\\/S31G1WcoUQuJNdi2qfTIVCa\\\\\\\/ajfG67oLwzHuDZMxm2AHwV\\\\\\\/IlelxdzsCGWJ3mR+HNR45lXK1il3ZBYSc5Adp0OVTRZHc0rE59heF7YV8rZsc\\\\\\\/UrMxBxfcu415nG0U8d0ZSEK3f600r\\\\\\\/MOwFaCuN\\\\\\\/PNt6Is7Smpf8yGoT7CaCnq4vHZUCz3kzC1iMxiAZm\\\\\\\/XMxNJ28IzpbLLHjDwvs8ci4pKs0kMYzEviwcrcgtbvSZsoVbNQ\\\\\\\/Zd7jCJKLMzVnx6yv3iy+RYAqi3gbOv3wCqKuu4p6oUzqMnodGLOm+eoVjzg5\\\\\\\/oSs7x9exbwg8QzH02Rf2RXyvPbp1AGcnZvEYySfY+cFqu\\\\\\\/H\\\\\\\/rEsdzVZwRyiYf0kTnBrB4Gp0GZjoeTF2AWKVF9sRKy15hFRKQwDcoMszCMLvsKCVL8RepdwUNMWRI+2R7ktbWDgEGcTCfAiiVc2wGm5RJ7+FyTTjMUlSnVn9ZqzwKCGhyhV4zdsUwYVaU6+1OPbvzzr6lX10TNOKiP3+v7rFp9xya1D33Mhzx5G0Jpi01VPyQflgCSvBgEq1GsMCy+zHr7adqi9ATPdW\\\\\\\/RGmvKhYytf4oxDpFNu+F4ht9ZrJW1EczyuoaKjrhElriHWwDw2mUmBzG8qF15sFH38crIi19KrjFb9cKjqPna4U+jv2Kp+LXQGWpc9SW97qdNfHkJNk5ipdRvb5IPDh1jWzjJXoULX2BzbG\\\\\\\/ep\\\\\\\/I0aH6EooBNqQyC9OAfJriyxm\\\\\\\/vFhPDpU86cP3a3Mx8eTEJvDXEJeCEIW7zdsTxY8scF\\\\\\\/dN1nYYqH29rz84xGDKejac8vNpHM7IsGtH5+2lIS\\\\\\\/\\\\\\\/37R1QIF82gS\\\\\\\/TTAoPA91Eq3kYrXXo1GtTu29Htl0643Ov0+KjB5NuFxPeQssyfm9aOJEJuFcJCiswk+e+c\\\\\\\/FlX2n5TavvGdwcQXHe13flC+PP8qDU1O+uXy+\\\\\\\/dc36Gac\\\\\\\/nz9AzuukZ6BRfmDG4i6JblR4kBXdw\\\\\\\/eFpbOCkn+IexgwNH7pA553S1GAS25dQPK0QtpStBgPwr8jnhatahtHNl9ogsT9o+a8ILeXPKbo5fPmKHaI2Pcgku6dID\\\\\\\/WUWHZ82DOO5Am+87ANveAHaR8y4oiC7NWULzFzM\\\\\\\/f9pWxdAX+b8yLSbNfgfMtEGIWgObOD1254rh9\\\\\\\/2YjhZSPMgNsPHiebPJxQundXePLrWXUd9EqRU7e+5eKEPsBI81V7rje3uGMiZvlBwjvoVHPuvOmp7eqtXwe7x2bPxiEditeqHyZuulAX0v3Wzb7j2bDO3DEqyPM+5\\\\\\\/AEvEEZrJr8CTEvuhgENdNJChBS7UWYjkv7x1aaf2WO4dCr22p8SGW3FFr4+MW+cnLwcfLv6gVn1eqhVdZX9ss4Sk7nL6P8dIHSM3BRIH0rvxcIaOatJaTtLEFpCWln3ZnEFwkdrIYmpOhNsl+vIm8E7Pj2oJ4RBhHh9qG2CTHvOalOPLNOSCYzGAGk68iazbxn2+uvhBRm178OpsaxgQgA6mdTeLQh32xdpbvIlZ7NLbSh4JYddxhGb7afGaF68sQzdm20gjwfH+iEcDMDjHMZfNVCQgORNl9oyvj16Qofi4BitxxNhryY19Hunp3Qa5fj2iPRKaVuugx2Tbnjsr6i03CTixpvy0d\\\\\\\/oo4bjcyz+6w3dTYFJrdwNpP\\\\\\\/4JzkT9A5yRzHyW\\\\\\\/Wj4HNTm7JX3hVz03pZtdrShMpSBVpUCEOB+gxz5gy9qnniPjnhVSy7ieOlKh0GTgm38c2lzlrwUS+VburAzST0PoNCfhQQMB\\\\\\\/7nOReg5zlCXYp\\\\\\\/+x93WPmEkk62xv6FtqQt0XJqmg0RXt23h68Toadx3zpENuuQggzrmC57SUqmyvCY2dZ9lfzsvUuvmiyeS1Cgc6H8gqiJyNuUBAMRHCfYb8y17RrMoY3\\\\\\\/TtNi5bA5Uu32uA6D3y969F2OOvKxeThu\\\\\\\/zszeH75gXceylFQ54sDJeIw1h6InAsHa3gI5RT54j1qmoKOis6inHKGoybLBl5uZWctbGwltWgrlLaWV9OgWzu1gkU5hmPmg7\\\\\\\/kigv3ep5Fxrm5Ht\\\\\\\/EWiLbJmKqyntZ4JAi46\\\\\\\/eDgXm3jUdZnl9YAuhssrsyxBDjod1PDUAkvqpreOo6pRjL5cyy1V09Ju5mTZVqx\\\\\\\/h35Dz9aVvGuSRZoqQL3bts2tmSs0Daa+WWlhD7hX4062g1UXZEi70\\\\\\\/GSJTDfBTtcnXqqGuVAiYH07H+\\\\\\\/Wj97w4o9lSOOszQLKsp0nuEc9DMItfXdyk7nQP95L77YDFKS9bg6nArpoK0m2cr0LKhaqvYzSXOHYlweqJlA3g1hgHvzB0BFxJz81Yi9Quol+2Hbh9NPKdz7zXDSnGu2dLhhfJkooiyblFNq8MzShCVv5U52INh08UBQKv1wXawPEMKvRveJ+o1GWxEDvB0CYmVWfoNomzIKlcu1NJpggSzT2ijvF16oAXp4Ro3IUmPvQkqvdzqWi+40LGYATcxcm9jnAvCLIM8yssUU8anfYk8SLa9WP3v78tLh6gjSp6YiU32iDoHtYZ\\\\\\\/MXA1P45K10f8EtDAWJAQNlVWkZtMZl1ng8pphDeWTwO07Ku\\\\\\\/DxBRHxWfF8g5a1kZfKoXYqsreUdSbZQ8usiIpHfr6PHgI2kNCYpaudS1vEvO4ZlgqKRfvdE2MEjiOVQKIMxUaaFGyWrk\\\\\\\/RBV1ihMKJ5U3QXQdHA7q6IsPxorp3oylJg\\\\\\\/eLae50LLb0dxO3n+D+Sy1wSzEloQswZQG9t6UuUJBhXfu4m\\\\\\\/wHVXx6J\\\\\\\/g6Yhe3mUOd16djwuYv9FEYD9yv+FrNbCJ4ZCugCTbHjwk2lEqQ0apKSr4iEpLOwCil60aLpLwWof0ozdFjOEcYbyaifEpc\\\\\\\/qeXPTyWYI5Fz3J5UC5PdfBWgDqAQyinLA+Z5kspCk6sbp+iDrOghP7C7DXXxh7hbm+da9d\\\\\\\/gTpYea+UCLecA1YkpHOJvQK8vrGs9BFMqfsnnet686mk6Bqn+I0tsAdVfTW86mvV2vI900ghGtoOlQp1jIJz1MAqWb1iydoiDkUzf+\\\\\\\/Uvl9EnijjJGJWYg2ADtvlx8EeMiHWObJtGmkGODYyjdbve7hWsAddtk3Z\\\\\\\/EDbSp7gF0tYQwswftD7e+wQR3GSf7ddDDTsxAmdYYqwN4RKY\\\\\\\/NjRuSMtDI7dZcoEMVqUaUlb\\\\\\\/\\\\\\\/8xJej0OQU\\\\\\\/L+vxIb7EjDX2sdR1IKVKhy5lz91Rl32NDPy7a74GM1kBxOw5SQ6gX5Hycfhv1pV+bavIUl4VBvUZFFK7vJHSZ\\\\\\\/nwY0aztImwKWOa8QL6\\\\\\\/1bOUT1Ekt9JW+TgGQm81\\\\\\\/qDjG2fbGgPfvwUWrftU1s8jQ50I68XCauTL7UyN6RJHDgXpbQFRwyTg4WbNy6hN8IhGhKfAhlKIRZ3AwamTAE6SHxZbgy3sLMwJfxGXUmakI81m8dMvRVfFvN5qxP4IR9yqVGMFo8Tp55ZV5hK5P2VF+ekf5WfeBCbKwTWJcZjGiqHpEM7cKFIUMhVHMHCET\\\\\\\/bADCVRJt6nM3kJFUZ68g761M1jP0wmo02YQX3d9b8RvbL90avnXT\\\\\\\/\\\\\\\/UVgD2FfJ2szJ0BqNHRrPxRiz5aaJKiQ2xaOmTLDGmhjlJfhjR3gufkoSRsjZHBtNSMzFs2mNIYhffT1hcli06UAnhZAXIGNs8wZq0oMHZOj3uG\\\\\\\/v+hVlxUEJB1GjX\\\\\\\/vGreReHiACK+6wraSTSIdlH75IFpYPQ\\\\\\\/\\\\\\\/GMfLxHzvtRke5EfWz0\\\\\\\/A958NTtrCD04iOTByVP0LOfD3xs8mMsZo2JF6smjCogOLwRYHTjcZP\\\\\\\/NpIHh\\\\\\\/43RxJZHmTWyiVW7BpdwnwJpgvQjkojrrec5lGNtaA+wjRlyA2QLsoZyRUkOHvpLYPBOufqZESUDrK8D0cHE3AhX1pjPiAoy3ihG4U+6gotXarweYjfuCeQGQNppsrmtfjweo\\\\\\\/C\\\\\\\/fQHxyqrSj6yeiKcCRk\\\\\\\/+ORe4qFIt40cz+nrMH\\\\\\\/PFQNek1vffz6UKKFH4WdGsK8kqd37Pr+ywzK95FHqKJ\\\\\\\/JpXIXnW99mxN8Vfwd60q4N+Y2vvYysBuLuiks7u9bCVe9J\\\\\\\/UuSRSLaEhAox6R1O69JRnlYcPYCyk++RrJ21yoDXwilySUUwWsv\\\\\\\/yZW8aU0yD2+oB64TLBiDipVoHf7rUW1MHMblTale4VUL8YhDzmjU+PdqFjfpr6ctHp\\\\\\\/CP5RkoqfOu8Ne6dGr8\\\\\\\/qXiGIBHyOC\\\\\\\/locMm0CllxLHIDVfR1vfSoM2iERNdMv7xhIlLc1\\\\\\\/9lpjo\\\\\\\/qDfV4MAyh2AA0hVeF+aKL8JJyxl+uVPZZQLXA71dUJ0jFHVyYn8Lvk1POMb+zsdl8TlFeqRxrgBr3RvRE4gs5y74yG1uhksAhWbfsoguk6KpGsQuefhb0BuroQztWbNpMxdwviJiGBmI1IOvx23912nxvWaiz05Om3m4EcOpvzkhSaf77vivHVDcRGe7rwUF7fCVnyfW11zqwRfq947xKm2DPWz8mqsi+CSsTAJPlvwOazNGiUK3IUqMKzZ+3mBmodBrXZNcEKUvRG4tcK+zKTDIZ9j6IqJqmywfxTNqCNlGF62hzsGAlWobu4IGcyETi3XS\\\\\\\/gAJIGIZKpVkpk9\\\\\\\/\\\\\\\/0K+211Xg8dc\\\\\\\/DPO3pWQNcQUWWUKiaNRzMzOl8DA\\\\\\\/NBMmwtVjcmAIK\\\\\\\/ARQkvC8cYCO06CgtFGvGz+ZUY3vrfeyWFf\\\\\\\/9tTqD0RSSglywJWEu5sKDvzp\\\\\\\/dwPUrdQ58L2YSSr6srxvRD6HeJChXqdlUSjrSb+jMJra4Rzx379mdIQeX5mipdQvCCielMhdTnREXH0nev7PM4LfqwMfRPrUTKtGW3t98YszVFAmyqo6eqGAIw0o7PVD3FDScRsgmqvURWsfrYPqJsHasIhyDwgt02IcImC4HhaZZG659uqeeOn3o6s\\\\\\\/tX39Od6L56\\\\\\\/uRwtgCJ2JQImhKa4EMihJqICtMze7vks5knL0xhAV0ezrzkeCyb3vW+fTzOEkYAfCiRnNET6\\\\\\\/yNNs2f5dvT+Ix\\\\\\\/87UL1fFHUS5tpxu2dZj8LHDNZbHPoq3\\\\\\\/Mx91QCjhqmwgyxypINQFRu4jmIPC28NCnjJpQ6wj6WCqi2BGF1Td77GAy1RkkcfRAVSE7NeKt6ef6AVCyDZQgAtrfup4nY9ytlkX8ixgXh2xjzSEBS4J\\\\\\\/vaEHZg7jfEnvJYl\\\\\\\/IXix4Boixjl2zi6Va7N9o+1CM5orBRqBOuyJuaU2NmGyrSRDiGOEzeuF2g4I8SpYvqSUaMWuNnLIYvydCymEzLCD7n5\\\\\\\/5TX2gy79vVx6efBJCousguSabALAE6OEYZE0au3Acd0wMH1XHUVxNl1zoglb7gxT2JYQvLI4KE4GcOOdMHkWfxzDlJNr1Y8V1aoFNyDIoD52iVbPo2cL54NdtICsLa464uznJYyHUGNhR+79S2nHsoHnsowAVpQjbbYyXR5gtQwbGVLS2ZzDmvuvdveGID\\\\\\\/vU2IEGRJ0sPAVC5LMPfLLD1Za1vDJxElmLb7Q4hDRLIjzBxoQqbjSvGnkPm\\\\\\\/ZNfJWK3+lFju25UKqbGuS6umUr9sDPiBPdYlpBwm4b4qweZxpkZ6FEL\\\\\\\/FpnKXwyEg6vXML868GJmtwJwd+3LNAkzl\\\\\\\/ehiPgIJYn9Zj70KOsgf8QXB82KnjjYsikBvuVM2dz7DEtBSw\\\\\\\/dRV46HKO2BUfyLA0JaGCKEPUy\\\\\\\/iYWP9gMIUNhJls4TBVkFoY+VggLw2Oe63bWoPbjg+y22LgLh8WH7fn+8CijBQQOD7punrDsK6BKtiaViKREunioq3lthPNN4gqNIV+dZD4NQdnXhnaaQ2eJnRzyIl2pWdknNdg35gMF1VjDojueKK83Arh2b5cvF6v46z3lwyfiphSB76zSPQFLVhhNFLFpHJw4q89ac5Q8dChJccSTiEl+lppJoX2CvkB9Jg+xlIq2lVdMvL\\\\\\\/Pt6soZ8EzWRMl3AQZk+HLnFQYOIgdE0COLgUOz9MFK1Ow0qMkTzc1QUBMQJjO5m2eph5TrRVPEFY+XGPtk8fNC8kiIovPc\\\\\\\/ZkYTLVwAbkhziamQ5aoadtJ2Jd2k41JsOklmBQIfhx8jvqz7460iDubWJak0cMmm7BRIA6rYX182iOlokB5oGKp5UwR+N1qTZNS4J3HpiospBZTTiYe3I\\\\\\\/cFJktE15BO15rdclHTjbxOyqxgtyPyfrDnjo93nq\\\\\\\/sO400gQ\\\\\\\/hssRu9AbUD9qdXdlOsWmOCHCY\\\\\\\/xaKgqgKC9ndyIcUwbruXz+YAxBOxSdc0RMtD8VLia3V2Yu1bHIJHJAQ\\\\\\\/etPullV1oxHniwkxhUV7ym\\\\\\\/hu4R9k0uPVJ5Ae0CBmAhljwo1QPa+O3LeeW78k6dTqgFQLRq3vurVbS36M9Drbx\\\\\\\/5lLhWBmd4fTEUPu9IBkwsm+FskalnXtYJVQFm\\\\\\\/6r+2pxjnWK0PVVIHr6CCqF7cRPwFUw2saPhLrgTM+o4aHAzZKnClR5Fx7ekyUD5E1HEY7JaN0W6kJOcSny2MTKcSK2iXr1cTLiBAz9yb2OpQVmK6rBltjQPVdIqoqOUyileCHSrz7HSA86EFDdOTNQz0Bji1QVCIbIq8gB1L39+4tqr0JfATN9qMbom4aWPanXlr1yx7X2+PKUef\\\\\\\/BDH1NlExHRH6KBAeMf8fcNlIt5zjsucAGnM84MsQee7Zw3we3NX9AeHH2DgA5TWNGDlbnstpGhpVOAF76\\\\\\\/cRC+Nv82YjIVrzmRe7mB9L9HKKGgtRIUTRcui31QPcT+yQalV8cHbBUvbe9wystLHTnvdwVEde5zywqzyiO40DBYy6LUIPJVMKicYS98jBPYkjKOywvEm8mTYEmLOyJ9uNKRdMkkS6NaXYBCfFwNplpnGlHUeRNN\\\\\\\/k13J5w\\\\\\\/\\\\\\\/s4w9I9sE\\\\\\\/wciSswEFlbx2okwJR\\\\\\\/ayX2AXc1vqUIeKmSYW8zYLPzVpK4vO7+KyJRquxYPkQQm\\\\\\\/PBVIEMTmZhFOzMz\\\\\\\/e6Z3F79FFSyD0O0LD6V3ypD8fzVhyCBunq+e21k1MFDWfefQhLAJMXZJNxOH5brl\\\\\\\/IAxXqpYg60D8oBO7x+CYxKJyIXvb3KNmhG\\\\\\\/svTgiEYtDAOsXlxLflBTDq\\\\\\\/RL9baA9zT6o4xXRhJjyRzsp1BsVdbuJjD1ENzltfRZfsNekfNV4Pgb69Q1okd4qyuSPgUXFoblxtZ\\\\\\\/PqOQySLbrWaLRtmfdKP0XNTnWcEppTmv3szPJu5tVzr5lcps6IkI9aoXrwsFonvOl+gaZkVwSxEDeqECGmhob1UBkFwFBBR7JTBN03mi08sizpW1XmKZ8rQDdO\\\\\\\/E6joH3v\\\\\\\/2Pan7H2uzn2q0vwifn1QPiqdruCrxNQ5cKz\\\\\\\/5rj3YY30s357Xfoq0zhsTcVaBXRAGa8tD0FTfyJ5JGx9v2JwUp2lfQt1g2nc5sPxgAMu1gnoZw2qWR988QZ7u\\\\\\\/p+AtmjkkqUTkkykr4EVS7756CmKWaPsa8GDlfJ8sJEztZ9QPK2HSWfFNN9vYSPAd\\\\\\\/U3hGOtaD280aIIN\\\\\\\/gqDY2ymdiXTFdGMhGg+CK4iIBDPdBNHE1XIcZtMli4pYqyL9MPfWbGfIzwg6lHdMa2Cuyo1baaePkPMUVwBpnF4o57AEQCKXLz+c6dowMfERwWiuc6rAxWX06XFX9MPsmZpx8WsD9E0rGvLpeGUZLaxD0H5SeocijJo28sUvjCmezpCSllRjna0Ptqo2lV6dq6H5hIelqvvpOWTkiGwCPUgdgfyF4CrlXYmM9mAzpkgxL7rVrzRaLe85o4rVQ3TRImQuQ38fTCOOyxihV7JKphftdwfAs1wBVksQayQpc6XVK0J4fXetDOccx8ovJBUjNcbd++NGJMd8wvLTfScjtpM5iFayrjMa9+Kd2Lu55\\\\\\\/OHESb7rhgfVkZQmjMJMGVH70eZseGZBIAXYHoAWEWXTLrGQf6hobS5lLDBaKqSbGxxUV4tiy4\\\\\\\/ldxJHd\\\\\\\/YMg6rlBncUNZJZNgEINaEVMJBtiL1JYV07nmMIeGbbBKSRSKKhcovrOwzgs9M27844tHGFAcKsBRZiZ9W1RfGhfhWBAC+sv7rIQT9lklauOUHNSzuXYAZIc7QRP\\\\\\\/HRFLofwau5PsR0cjacXUaW3XbX73PnrDJJEYzNoRzC5CCp76beST\\\\\\\/9167xxS7dyeK1gnEX8NxYBwVlrY5LCu0dHiWdmmYI0lbLWWEH37\\\\\\\/+DFkqq7CP4QBGLK+cyD2Hbqh3bkAXqMJ0eX6rX\\\\\\\/kcr4v0zL5JVy6FOUcncnn0ALK1WGjxorKNEV\\\\\\\/Uat+Iy+1w5+Dvn0i2jidTZ2JUUgzzUffijaDBFE91tF8DCvvSWdYC\\\\\\\/0ZbhHfAtArFHSNv1q1EdjVg7TL2UA0BzH6C4C66euESeNCOe5uGsD1fA\\\\\\\/dEF55tmFB5zVljoXXaq6ZujZmEXo1tflyqIcngUQMbbMfxpzN6XFT62PSc55pwvUb2hdsTZATsJwWp8zUhQLo1c\\\\\\\/8M0yuT9itdeTfnHs3pyleVkmE2CJAj9c4UXhzSgG7KnR+kqiUS8r+zpCklacMpio5RiA1JyQcezgpoBTO5gNF8hWVLdtk4qSGilW6TsU1KnbY\\\\\\\/Sp19qnFjn8PSjt2ECv7\\\\\\\/q3oyT2UDuXKd6YJ0Aoe1So7sbrNfjFRJpX7E5dVYNjVCgyVYRmdiwAf6Y6W5tF\\\\\\\/0FqDFYejEGESrX4gXDwmG908b99Xifxqn4RVlhwYNqPWp1wAfV8wuPy8cLCiLfn\\\\\\\/EeISil196CJYZ6c8uF2zmdOy8ERA8GaTfizcPLsHvKLXRnIh70Sb7D95LUeBcpFK2ld\\\\\\\/z\\\\\\\/QaOkkP415XlUIiEZymCrmEczP45yxHytu55nhIhetpEuOjNe9AlWw\\\\\\\/JMjfwZ6ayn3T+mlWG\\\\\\\/WEFtpkRjSLxBo3LzdJGEo82Ijq1UJwTWDjVaDorBKxMJkPDAxVXLMu2v3VujFac7YmNg56m+okqMy\\\\\\\/RcsIBXbsm0EX19UpWuMpcwU3kB4Oop9Gnfh4rpLJ2g+qHjtK+gJ9chGVMqXYe767oRwmTBssfpDJR6XpOmvp8u9nL8RrdPP2pKF1VAOIw3Mxj2ll0VSN7AsRxDbxHTrLOUf54euelN4aJG6uTYHnFYwXAqRgwq5jloe83IHR8Buw5dCvAoRUOGHDuA3SGw5pOa9wrvPhfTNKzyCNDKI6FGU8Fj9RSkg2cvOoGPpIq0Wa2IqrMXDKp4QaYnp7jt\\\\\\\/1YWvDecQP\\\\\\\/ZRpVeqtCkSSJl9SGoaIaWpRfXjmhaPTkQ\\\\\\\/uwBaP6iVqFCE3uJGS4GwA1zt7a6iHfsbrG07i3Oghgh6EJBS1lL\\\\\\\/hhDUBqIkFQ64vWeH7FEqjJ4PQ8kHTUumulmkiCSk61aXoVeOz4dvEzLHtju1Rw6PnKrqyxPF1RK4NdNTonHtCLyCD8+zkFPAbvz1unD6NjmJarQ9aJ2oiCd9kcDCeGPJZSoj9+ONANp3R4PhompO2W2cVb6RCCWALaxwp+c1jvv3qhqHe3uKbiGS5gsuEIReduhQCBNgWjtpv72Z\\\\\\\/ttiCKKPXBpktKpmCsan7gaIOfP9CFdXGtk+p09RmB5fbjQa\\\\\\\/82Xpq7qUrotUYbF3QfVhgpk0EsHDKaMDJFVWFVP\\\\\\\/FJFG9GKT1TgoD+Epu2E8+shlJMDaSK59aBgr03o12yqpo4Yz5izwTmECAa7W5Miugtd1ZUFfypkrBgoOEl5DDWUQqWLBVHKa4NnBCb7TN9Bq+vRbpOfune4gMPp9pcLKxxBPJT9R64TsJFmWcfhayNzpk0FIUThT9GhgrcpUqtffPGNPcSkwcSAa\\\\\\\/CLcSM0y6Aj9IuWajmx3jd4lYFx7QWU0+L0z53AjiLLefOoKwRwBCwyKXl1Z7QSoBnhkCcm3+rONR\\\\\\\/RxpPXcxa9PHkh9WCWrxdJ7iTZg3DWyi2wi3yjaYyB84YXvDl8fMO5hIXQhh67PxmhoP5Sig2yXPPg+cMSfxHKnKo+NT3x\\\\\\\/iCGhPEzZop4JoViUEHcspa1ztiaDQnEidZTUU774wHKDzcv9ugJUOtv7n+Fu3jGSYnxrIPZqB74KBlv6j5w1YHtwwV6jDaFsHHwRgz0Gohg+rnvRN6H38jDThZRebNxbNsUTMrol\\\\\\\/mR0Jp1e1UUu2E0HEG\\\\\\\/yFN8rCTi64LtwQ6Wc6cRmgR8MTCiRPtR3lUpuZBgcnW7bpKjrvKLebMg0hAqHSgeJECFGG8Nm7CrWN4kL9GHj3P1q4Se3T3BOLTW2CWHWIoeTGSQYAxovNTh0sQeYv8b16TB4+tz2imCZTylbBjR7DQc3lmyVJ4G7tjFjlZ3QcxVgAibqPJJkYWeCmgEx0wOfppItlB++CtPuXwLOecjilWhUORBfz6CFkj\\\\\\\/\\\\\\\/NQtibdqRT9HmiBRMnHUHaeK1kG8P7eiT4k2wK+9XsFjV0IMt5eTxLcR\\\\\\\/iu9yYZz8P1FzQCFhQtGLFfFpXZotx4I1p3VL5NexqQpFGUQjURHcovL5yveKOvTu8OTkefAWnwmbgzpBRY1bFsDIVpcL3am2p73Lm48BI44KVZnxc\\\\\\\/5rtLGbSpysB8HUhskPESLyFLcS3\\\\\\\/kw1Z+MK0lZsOW6b5sbUej4ldifj9IWF2nsXGHMofP2QJLxhMvjOz83sogfuXW5cBPDXZ6\\\\\\\/D9hxfX8Z5LEzaQ+10pv5jrkJDBLFq6BSmMbsErQX79uTTP7f74El6dXkcYFU7Lv\\\\\\\/tkUj\\\\\\\/UQpAu7xjoA8Wc6NeG+eT6f5tHqaPYdkePv1C\\\\\\\/TFhenCDsxMTRuf585dy8sfNotUiez1X9\\\\\\\/wd1VQhEG9sdaIk+Av1NWE3vK9yjER76D4nJJq7+dHmbOuUVz6UMe8Nb1TMp4wuQD+akLU7dqhFQzBsFEGIIOwm4IBYbOIrP8OVvUOsuo3qaAicNg9gVVGgue4U4HZMWXxlLHnlVZb67re+6OnbOMKYyEEDJB4YQDwniYa5UNHAoCntcQDdmiEZWI5I1ez2VNERL4W2RQjg7J6VRDcVYNjk7Gj3Oad3D\\\\\\\/OpbHey0Q1BS+hmLy\\\\\\\/S2tuUuZlLQwaARqJj7mKWM72TfS8beBn8F4O77kD8uTLQCHhwWBSP1SBSIL6IJt\\\\\\\/AWWOGx202xnibg6737RzkRgBuQOZunzQJrc7kLz8jS6IPjPylX91q0oAzrVNQ+AcK9kcrBbDrwsO7tyBuL8dB1srkfcyrBzQv1mjzWNXVkWOI9gtxiSSyJGuFxuIBABcp1RXYB\\\\\\\/axjaf77SDwQjV2XcXUCFpO7hhMFk67Wr\\\\\\\/8jqaMtt8gO8UoETDnKkWi41aHVI7SZjoxCAr4xRiSrqrlYvG5BOmRAjAlKbqSe\\\\\\\/VmPPeQW8g2ptNSRkEKs7ISwbDsHrL21gHXJDeHQiTv9KWHRZ8VnLKN168SNGXJtG3sQbNpcUaZvygJ+oUxMHCVIXT6LLdQeDXAvqDLErsoDlnHMEk3orChqWziU+TVQntvXkDERHO7hyPsC9ioKzbG394MBbCeryetrvoOXWO31qGlPkqp9gKHlRoeDnLE9W5M75dRfF9+2IZR7ih4ZbUIh6mctDXshXLiBXV17+ijnAytqsVSwPLfO9dSxgGR3ysI6y1WidIP+lHPZEjpUdpxjdk5Jh7nvngehWDsIQq1BCdk8aEBN\\\\\\\/NwP\\\\\\\/vYK1l8qViOj4\\\\\\\/p2S3DBaOCx\\\\\\\/qJXZMbh5avNntyzOlPbE9W0fzF9Lll5kqPI+UVmFHqetT\\\\\\\/+mtMv9sR472+xWaW6NbCw9ucPuYyppA+GgTFb2XiSP2E1H70rIXixtQs4N5DWxIB7fJUWiHbox+KrCHt7rhVq4L4kCEyndKwRMdMS0TBn0eMUrPj7i6Kog6fhkORIy6T9cG533jcWU9CQDHptlxlgErC9N\\\\\\\/mpxiuGVX\\\\\\\/xkhQBEBJ6+2\\\\\\\/mCqZtj0d8toH93G4mtqV6C44\\\\\\\/mAU+TL9ZznTZ+C04n5TAAfwgRIO97MuqtjDsqgxs7etQ6QhSTmc\\\\\\\/GnuYeFKp5YJobQorjhWUUvYrDKCYQrQLrRAqvDI0rS245GfWgGQZqkqY1vf8zXb78V4xQXVPU2KT9lFl6PZjlEi6nlcBObnMvqfe3rjdTB4ZqQFp+YCzpM+J4IzPj8I0JEV12HhN6m4gaw6vC2gWyK7hNeq4ZYOk0P0qBgjoTRKqHDD\\\\\\\/\\\\\\\/PQ2x4n7cWSkhpGPjPszMQlH\\\\\\\/hEsaED+c30Ocjfkd1q9XbCD0fTR6imKnn2bKkZJFiOJPT9NnCtLrdeT6bQgGDZ9KvCV2aL3hijyQjVpwMgxZEbDocaHQ5CupWJ13oiEjD6tK1fhjChDMAtQDtkaANKRe07cdzdH8Kpve34cPyrBNykCnZBTaBndLwEAEWioKe47ImS23UdYK\\\\\\\/nk7vt+q8HxUffmmmTyDwu+wMWY19ZaAt+NIJQewElF7e2mkULJ3R5CKXjzRCxRBgVx8RnJ1fWzjhnKqJenK5LB0ESI3FyLsttCJS79vKJ5PEZQDzrXSyXRU6W5P4DvgSVn6YX4fzDT3h10hg1yHwjYNAfLZ8n9n4hjSqgUk9OtVBN+SgJKnkhmNynsvaEB4xIP3WLVyIpQo2fTb5ujV3ncvxoD\\\\\\\/tyj5qK+Vag0qT7jfH0qlyFerfRAavdPZiRaV0BLZrcisdVXINKCom6HGVCUT2HGFTotSOR4oLRS1BrdBOG8ELZCy\\\\\\\/XxuSUHTeRQOb8rwwjtn44bfttiJDx0XtJpJKp+mJV2aJgY6+D2tWzMRC5P2wuUTEOCEV\\\\\\\/sd14ASi3qe+HbuQGc73jxo\\\\\\\/N2bMaNHQcNj9qloZ6ySO5lTC1NZbLopeY\\\\\\\/xQZiDNt+ffaJLoIUiT5\\\\\\\/L9wr\\\\\\\/Bl2rswBqK5MLNBBAv+yxQfubQkeWZPcNpELfgK+wncdeyTACMWBDjI9hThjHtq3OQjgCS+5f7Zoqq8o03tkyovUlTUtOlleAMFLcQxX1XUNUmFNm2h125TFqU24hOTqJ4fZbLpQls6P5ZtYvaciZc1uKe+OJ3tlunGIdPVdfUlWRsTokLZVGEq0iavrqZ4YEACSdRezfL1W5KH2ZvclQTK+9Gtd0DQyCCwsENyybHKizKRW7V9DIdsw0tIA2vf\\\\\\\/x6kUWXh4IQW83IoCSIey1WL9uzMVW+w\\\\\\\/utFaE\\\\\\\/Fvd6ryVhrAGkq2P\\\\\\\/6GYUGa6GEYkwWxAzoYGSboZEmhQQcG2z7ItVc+er7kz7CSXwOcJWENqevBGatclMv7f70vyS9i1WwJS8IjTL4ORy1GcXWjsvXOrdZK5\\\\\\\/LBCRPoXA2tXlKmVIXbjyK5piLUZ3kpjYM\\\\\\\/SPkpVZEU+5cC88+Vk3FHhHBoFu6lNZy9Vj3NfYL3GP2haHDiUj+nvg0kcMuoIPzIQmvWESWZ6QTMdjsDOQ830tKLbIcdr5R9wiHkeAn99U50ZsCH9CHPjaLYNnIB7mp6cUd5n4BFR5MudcN5YqsgPGv2WrSoYTaxkLHZY2AjwqW3uOTqEAlPexSRYEZsi+xYBfYp7zM+2+fdmQ3ZRpb\\\\\\\/TXrInHJ0lp1RDmHJej1abnonJCFnv0urg2F6Lf1Rixd2TGSpJI7GS5FG6kPvpXtWZduTxnPw0pk3CZuS2wLpmI\\\\\\\/WIdRnNEAKcigW\\\\\\\/3YBCKX\\\\\\\/VlptjmaWs6SBbuQxP\\\\\\\/1x\\\\\\\/3TkqEJz1a72OR9h15c1wqWyQ676gIC8PHQBQY7yenCdXKJcA8GDE4X1aAwPxH9+SuhP5ootQhJbmcZ2aTBF5RsYKy3p4JQajqDYlnHjLoEM9sjuMpsh2vSJHA4CKrjezSrjlMAy6CZcBi7DUs\\\\\\\/VVS7cPQNruOB9miv4E4qaYfIcdTdEsmiKDQL2HSz2aSPqWNmmdul2XGEMekoRVi8y7+QigfuECSpboQ6n\\\\\\\/9NxNbG\\\\\\\/HwdUTSjB7H6GqwRtpHGjCKGKwzDLJv1oJnFgom+7uWPXC9wZxULdmno20ZZHC5oO91QcrBXbnbxFYrHb4MRLuzExuJN2MLMpsXX6PcQK6l7N0NxHz7kWQ\\\\\\\/NkZxVcNBQrZpH7dp1UnLjGDQb\\\\\\\/VSDbKYvuaziI19qOihM0RqdFSPjeofIgYIWh6r9\\\\\\\/auHmIQYJepAnpo+nfT4xy0Jv9M9CY7c9aH0WD4cLIcAuW6lrXkRVl5KMWux1F\\\\\\\/hAk2DmbUS1C6veBWCbkdaN2PYwgwXYeJ8l9d0UTo+JfW7KgAB7+r4jixbSZcMV2lUKHuFrZI64QXsfw7gEdRuPjwRopawxXSzmbUUcjFMrrw6P8jN7vb9\\\\\\\/HvZKqNflL5mnQKBYndMRb0nMAyqph5Dlv1AQis3FhYrj1wWOAtoJewgieYizqWOVp99UPNJn0zBbHevQ1BQTS5N9HCEhf47GLhFAD3Ghfl9fi+SVq8iqhwhMDZae9+njJ7932JuIYGKsI4R2U\\\\\\\/E44LMwCD8rg\\\\\\\/xRI7E0cRIvPiU7GHo7hIsUqRcABD2JI9R0r1jBE4MgwurvbJtJPEuK2KqPFuUav7JE+hd0z0zk+i5a\\\\\\\/PuTxqWy7mxM99ARJ3Oaj1touNCdFZw13r0AsG0CCWKoUFEwLeX5AteNUBP4Z\\\u0022,\\\u0022iv\\\u0022:\\\u0022R9HSPQUcQO+0w9kKktifog==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002276dcbfb29ade5548aed896ec7c2ea88bb38a608165d80388985cde90425f6293\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kQ\\\\\\\/YXe\\\\\\\/DAtwjDhSn5KQvX+IUEUPabu4tOCV2A49OZtWTB9RZkQ\\\\\\\/nE5yERK9HPCzar32Yrj0Kq\\\\\\\/B\\\\\\\/JQH\\\\\\\/FYrWqw2LmairXi3epAIhPiYhT50oBHrzkD3psMqdrcHZi0uKpYovrZs9Z9+DBOTrqOdwvV9u+8Yv1PX3tpmJ18YA8+SqqX0T1jOerCX\\\\\\\/rQQoNuH6N4mSykHXvZI29FMXVutb7huNNpBoOPgINJTPs5kgAtE0dMopF7EumxzatZpcLZmFTKYLxW46csWcl09J9hL5e6ppWXoPbnQSIjqkHMLDZDqj\\\\\\\/T0e9h0sGqRkisCPmZ0Q2O1GatOIdxKLPQ6ooBzvi\\\\\\\/i6OFPNvZYXuwxeDaU50+gOV4ZDhfJTIQFEEISr5O7Llfl7Qx12TGEq2ra2Ej98xwjv3OiIwbvS8KH9KlP12htHzttdFhzeired4GmExo\\\\\\\/fxERzX3n\\\\\\\/HecgO+mH78jsI4sAlWrc7+\\\\\\\/fuvr4TkJDy42c6BDCJ33+VVkahnhBhCrDEB5wt6D1zYOgodYjY3IdFtQcffOwmVi1NaY+NRKMU6ZWhBybKVrERm02FwXjJPm6ic435R5nE8RAgy95QRR5HZdSu\\\\\\\/7YnWcglStSg7PYf9AsqVyYLn6muRKIhLAmmkNF+wYmOPlMxZEBrpj8HlTUyOISfjqrsyuaFZPDOW89eWq1KxUvwiBj\\\\\\\/R6xIz4uZWWaIP8ca0QfL3OMSiO6EPQkIzf56+td53QIvNYTwbyKwIb5oX7xNLQ1vS9DNww6B0R29NDoV9NzHT\\\\\\\/XhkybiezX89fakl+3DkbCCviGyhYj7gaT2clerSk+\\\\\\\/hiPC6Fewcn10fpEtSxutIwKpDNOyzEp2vruwXoC5UN41QNH2\\\\\\\/txUFGq5kRtaEg7W5ojicMyaQL5BAZ+uTQSF4Y0943bq8zJ9mVXkICMCAKVMijI4sfcjWEayoXXIO94YciPFbNSSn67zO0XopyIbkvo9HjUBT4IL4LY9LPt7vW6+bPFRmJjTk0A7mMXsktXVDzAxUm2maXstqNoqSKwcKKptqbZxTO\\\\\\\/1q4FzERC339boz8H0Q1yl3bGRq6a+ibl4LNfHtj76OWANiwKhfliilgAGP5prfbf\\\\\\\/B\\\\\\\/lCq1aRZxjlmSmNz5yfFu4iV+QOkFEbRAzASTl6I7nzW\\\\\\\/rr1doQz1nQLVgq1D9ZvEpjAGEnIKiFAT0kNToFAITtoHwc\\\\\\\/arYvZNHhJnWAspYol3K4frYjL4UVEGlOSIHUhjZ8YTX5+1hz9qXoVn6HoeN19a+iC7NjIYAnPtHavVi1+fFFUh4X5b6JiPNFTiOwwlQ23hn9vueq40T5k\\\\\\\/POAIOKP\\\\\\\/MbbK5tJqgB5+ecjq0bbbEEs9V4Qjl5VJW6zb8ZHF2uD5el3GuAVXlD1FXhhEfFMVx0NqxL\\\\\\\/88u68h2xtvQTtKE\\\\\\\/5eEf9xrLmhfoHrBhsBJfteJE4qCd0VTLLGh6aBpuQv3+Uh3bl3rxQoN9vEGSLmlF7FOXqzrLgPqL\\\\\\\/Dlgp5i5m80XoPTflO2\\\\\\\/ic6dsE7DGb63f9Io++xDEWIJW\\\\\\\/2RTN\\\\\\\/yQV6P3bUhgWIEkIWlYNVkFCJrBA+yMED0IhcX8xudZ+ZQ8sRQC6vHUREHgMa2CfsW8i4rP\\\\\\\/VXc6p8AoYWPIOXu2Ccl2hdXYhmoYJZr5GUiIq8ZB9XVaJGHx+VIIG9NG1PvEcf\\\\\\\/KT1MXLZJ2iIKt63TJh+BOsHiHUR5tRgsTlDe14mP9RgGIgH3Ddr5KefR1Et6HS887W6mHyZUXu4fuHHWtV5BYEj558bqVGMlwBrilDSStkhszfQG0yl5w5IvlObKHBFp6p4ID8ndnrtFCy4UTwYlthPJgauDIzC8XIAwD3F+zTA1YUF+u0db9SQ\\\\\\\/ghutjeZdHHZw9APLTbtHfT4F8Aj1WWVktvg9XfjiAwayeWnUIE3m4divouxvWF1fJM0J3CpoCegocCiOutU4YAJvxn\\\\\\\/bGNSnPDowGVQCUjl5VgBXWFZ80H26hLDZSLNIT3FQpZWupmYdXM0CrWFgpVOeh\\\\\\\/QO0jT2085tKEc=\\\u0022,\\\u0022iv\\\u0022:\\\u0022I0nreGHqz7GvpEK2Fjh9cQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002272bf838d97c1231696fbaa87cbb6d6a66a2976e788867be46732bb190e0d046d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459606 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/1/19" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aklGPQIs-FvbktMXrTCuJX2iEItTkHUhZwtvHCc4L5Q" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/1/19" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-X1xWy60cy8cyMOEuUDKOJQLfqFMs4pBgOLUnARj-oFM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/1/11" title="Design and Analysis of Bar-seq Experiments" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/1/29" title="A High-Density Gene Map of Loblolly Pine (Pinus taeda L.) Based on Exome Sequence Capture Genotyping" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459606" id="node459606" data-pisa="ggg;4/1/19" data-pisa-master="ggg;g3.113.005652" data-apath="/ggg/4/1/19.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Origin and Length Distribution of Unidirectional Prokaryotic Overlapping Genes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Miguel M. Fonseca</span>, <span class="highwire-citation-author" data-delta="1">D. James Harris</span> and <span class="highwire-citation-author" data-delta="2">David Posada</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">19-27;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005652 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459606" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Miguel M. Fonseca </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Genetics and Immunology, University of Vigo, 36310 Vigo, Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>CIBIO, Research Center in Biodiversity and Genetic Resources, InBIO Laboratório Associado, 4485-661 Vairão, University of Porto, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miguel%2BM.%2BFonseca%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fonseca%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiguel%2BM.%2BFonseca%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mig.m.fonseca@gmail.com" class="" data-icon-position="" data-hide-link-title="0">mig.m.fonseca@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">D. James Harris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>CIBIO, Research Center in Biodiversity and Genetic Resources, InBIO Laboratório Associado, 4485-661 Vairão, University of Porto, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D.%2BJames%2BHarris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harris%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD.%2BJames%2BHarris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David Posada </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Genetics and Immunology, University of Vigo, 36310 Vigo, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BPosada%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Posada%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BPosada%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/1/19" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459606" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459606/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/1/19.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459606" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459606/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/1/19.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459606" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459606/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/1/19.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459606" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459606/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/1/19.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459606" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459606/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Prokaryotic unidirectional overlapping genes can be originated by disrupting and replacing of the start or stop codon of one protein-coding gene with another start or stop codon within the adjacent gene. However, the probability of disruption and replacement of a start or stop codon may differ significantly depending on the number and redundancy of the start and stop codons sets. Here, we performed a simulation study of the formation of unidirectional overlapping genes using a simple model of nucleotide change and contrasted it with empirical data. Our results suggest that overlaps originated by an elongation of the 3′-end of the upstream gene are significantly more frequent than those originated by an elongation of the 5′-end of the downstream gene. According to this, we propose a model for the creation of unidirectional overlaps that is based on the disruption probabilities of start codon and stop codon sets and on the different probabilities of phase 1 and phase 2 overlaps. Additionally, our results suggest that phase 2 overlaps are formed at higher rates than phase 1 overlaps, given the same evolutionary time. Finally, we propose that there is no need to invoke selection to explain the prevalence of long phase 1 unidirectional overlaps. Rather, the overrepresentation of long phase 1 relative to long phase 2 overlaps might occur because it is highly probable that phase 2 overlaps are retained as short overlaps by chance. Such a pattern is stronger if selection against very long overlaps is included in the model. Our model as a whole is able to explain to a large extent the empirical length distribution of unidirectional overlaps in prokaryotic genomes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/overlapping-genes" class="hw-term hw-article-keyword hw-article-keyword-overlapping-genes" rel="nofollow">overlapping genes</a></li><li class="kwd"><a href="/keyword/prokaryotic-genomes" class="hw-term hw-article-keyword hw-article-keyword-prokaryotic-genomes" rel="nofollow">prokaryotic genomes</a></li><li class="kwd"><a href="/keyword/unidirectional-genes" class="hw-term hw-article-keyword hw-article-keyword-unidirectional-genes" rel="nofollow">unidirectional genes</a></li><li class="kwd"><a href="/keyword/overlap-length-distribution" class="hw-term hw-article-keyword hw-article-keyword-overlap-length-distribution" rel="nofollow">overlap length distribution</a></li></ul><p id="p-4">Overlapping genes—genes that share nucleotides—were originally discovered in viruses (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Barrell <em>et al.</em> 1976</a>) but are also known to occur in prokaryotic, organelle, and eukaryotic genomes (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Smith and Parkinson 1980</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Montoya <em>et al.</em> 1983</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Spencer <em>et al.</em> 1986</a>). Overlapping genes regulate the expression of the genes involved through translational coupling in operons (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Normark <em>et al.</em> 1983</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cooper <em>et al.</em> 1998</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Zheng <em>et al.</em> 2002</a>) and transcriptional regulation (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Wagner <em>et al.</em> 2002</a>), play an important role in genome compaction (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Miyata and Yasunaga 1978</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Sakharkar <em>et al.</em> 2005</a>), and may be involved in the creation of new genes (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Keese and Gibbs 1992</a>). The evolution of overlapping coding regions is particularly constrained because they simultaneously code for two different genes and any mutation affects both of them (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Krakauer 2000</a>).</p><p id="p-5">Overlapping genes can share a fraction of their sequence—<em>partial</em> or <em>terminal</em> overlap—or one of the genes can be completely nested within the other—<em>complete</em> or <em>internal</em> overlap. The overlaps are termed unidirectional or same-strand if the genes involved are transcribed in the same direction, whereas if the two are located on complementary strands the overlap is referred to as opposite-strand. In partial unidirectional overlaps, the 3′-end of the upstream gene (henceforth called gene 1) shares nucleotides with the 5′-end of the downstream gene (henceforth called gene 2) (head-to-tail, →→). In partial opposite-strand overlaps, both genes share a common termini and are classified as convergent (involving 3′-ends, tail-to-tail, →←) or divergent (involving 5′-ends, head-to-head, ←→).</p><p id="p-6">In prokaryotic genomes, most overlapping genes are unidirectional, which probably reflects the common orientation of neighboring genes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Fukuda <em>et al.</em> 2003</a>). This type of overlap can be generated if the loss of the start/stop codon in a gene (by deletion, point mutation, or frameshift) results in the elongation of that gene to the next in-frame start/stop codon located in the coding region of an adjacent gene. Regarding the overlapping genes originated by point mutations, no consensus exists about which gene is usually extended to form the overlap. A mutation can occur in the stop codon, extending the 3′-end of gene 1 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Fukuda <em>et al.</em> 1999</a>, <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">2003</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Rogozin <em>et al.</em> 2002</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Sakharkar <em>et al.</em> 2005</a>), or in the start codon, extending the 5′-end of gene 2 (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cock and Whitworth 2007</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">2010</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Sabath <em>et al.</em> 2008</a>).</p><p id="p-7">Importantly, unidirectional gene overlaps may occur in different translational phases, such as phase 0, phase 1, or phase 2, if the first codon position of gene 1 is the first, second, or third codon position of gene 2, respectively (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Phase 0 overlaps can be considered just as alternative start sites of the same gene and are not considered here. In phase 1 and phase 2 overlaps, gene 1 and gene 2 are frameshifted one or two nucleotides, respectively. Moreover, overlaps can be classified as short if the stop codon of gene 1 and the start codon of the gene 2 share less than six nucleotides; otherwise, they are classified as long. Short overlaps are mainly found in phase 2, which is expected because in phase 1 the number of possible combinations given the start/stop codons sets is smaller because of the nature of the genetic code (three possible combinations for short phase 1 overlaps and eight combinations for short phase 2 overlaps). Furthermore, short phase 1 overlaps require ATT to be the start codon of gene 2, which is extremely rare, in combination with any of the stop codons (AT<strong><span class="underline">T</span></strong>AA, AT<strong><span class="underline">T</span></strong>AG, or AT<strong><span class="underline">T</span></strong>GA), whereas short phase 2 overlaps can choose different combinations of start and stop codons. For example, if the stop codon of gene 1 ends in A (TG<strong><span class="underline">A</span></strong> or TA<strong><span class="underline">A</span></strong>) or G (TA<strong><span class="underline">G</span></strong>), then in phase 1 this site can become the first codon position of the start codons <strong><span class="underline">A</span></strong>TG/<strong><span class="underline">A</span></strong>TT or <strong><span class="underline">G</span></strong>TG, respectively. Alternatively, in phase 2 short overlaps, the stop codon TGA of gene 1 can share its first two positions (<strong><span class="underline">TG</span></strong>A) with the last two positions of the common start codons (A<strong><span class="underline">TG</span></strong>, G<strong><span class="underline">TG</span></strong> or T<strong><span class="underline">TG</span></strong>) of gene 2 and create a four-nucleotide-long overlap—(A|G|T)TGA.</p><p id="p-8">In long overlaps, phase 1 is the predominant phase (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Fukuda <em>et al.</em> 2003</a>) and, in recent years, different hypotheses have been put forward to explain this observation. <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Kingsford <em>et al.</em> (2007)</a> showed that random 3′-end extensions of genes to the next occurring downstream stop codon could explain major patterns of the distribution of overlap lengths distribution in oppositely oriented genes (convergent overlaps). A similar effect was observed among unidirectional gene pairs (published in supporting information of <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Kingsford <em>et al.</em> 2007</a>). <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Cock and Whitworth (2007)</a> suggested that most overlaps originate through a 5′-end extension of gene 2 and that selection would favor long overlaps in phase 1 over phase 2 because phase 1 maximizes the mutational redundancy of the original gene (gene 1) and promotes sequence mutability of the newly extended region of gene 2. However, <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Sabath <em>et al.</em> (2008)</a> proposed that compositional (the structure of the genetic code and amino-acid composition) rather than selective factors would be responsible for the predominance of long phase 1 overlaps. They observed that the empirical frequency of the stop codons did not differ significantly between phases, whereas start codons were more frequent in phase 1 overlaps, and suggested that the higher frequency of potential start codons in phase 1 would favor the formation of overlaps by 5′-end elongation of gene 2, increasing long phase 1 overlaps. Recently, <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Cock and Whitworth (2010)</a> also suggested that most overlaps were created by the elongation of the 5′-end of gene 2, based on the distribution similarity between real overlaps and simulated overlaps of real nonoverlapping neighboring genes.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/19/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic representation of the three possible phases of unidirectional overlapping genes. Numbers in boxes indicate codon position of each gene. Phase 0 overlaps can be considered just as alternative start sites of the same gene and were not considered in this study." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2130579706" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic representation of the three possible phases of unidirectional overlapping genes. Numbers in boxes indicate codon position of each gene. Phase 0 overlaps can be considered just as alternative start sites of the same gene and were not considered in this study.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/19/F1.medium.gif" width="440" height="225"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/1/19/F1.medium.gif" width="440" height="225"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/19/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/19/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463908" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">Schematic representation of the three possible phases of unidirectional overlapping genes. Numbers in boxes indicate codon position of each gene. Phase 0 overlaps can be considered just as alternative start sites of the same gene and were not considered in this study.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">Contradicting these results, some empirical studies suggest that most overlaps are formed by the elongation of the 3′-end of gene 1 (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Fukuda <em>et al.</em> 1999</a>, <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">2003</a>; <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Sakharkar <em>et al.</em> 2005</a>). Importantly, all studies assumed that the disruption of start and stop codons is equally probable, although this seems unlikely given that start and stop codons differ in their robustness to mutations just because of the genetic code. In summary, a conclusive explanation for the distribution of unidirectional gene overlaps is still lacking. Here, we help solve this debate by answering three specific questions: (1) how will the robustness of start codon and stop codon sets influence the formation of overlapping genes?; (2) why most unidirectional overlapping genes occur in phase 2?; and (3) why is there a phase 1 predominance over phase 2 in long overlaps?</p><p id="p-11">Using a simple model of random nucleotide change, we show that overlapping genes formed after the elongation of the 3′-end of gene 1 are twice as frequent as those originated by the elongation of the 5′-end of gene 2. We suggest that this is a consequence of the redundancy asymmetry between start codon and stop codon sets. Additionally, our results suggest that phase 2 overlaps are formed at higher rates than phase 1 overlaps, given the same evolutionary time. This could occur because shorter overlaps are more probable than longer overlaps (for both phases). However, only phase 2 overlaps can be short as a consequence of the genetic code itself. Thus, phase 2 overlaps have two more triplets that could work as alternative start/stop codons, making a phase 2 overlap more probable than a phase 1 overlap. Finally, we argue that there is no phase 1 selective preference in long overlaps; the bias favoring long phase 1 is, to a large extent, a simple consequence of the retention of most phase 2 overlaps as short overlaps and of the concentration of virtually all phase 1 overlaps as long overlaps.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Simulation of point mutations in two neighboring nonoverlapping genes</h3><p id="p-12">To test whether the formation of an overlapping region caused by elongation of gene 1 or gene 2 was equally frequent, we performed a computer simulation using a simple neutral model for the accumulation of nucleotide changes. We considered overlapping regions formed by point mutations and not by insertion/deletions based on the following observations to keep the simulations and the interpretations of the results simple: previous theoretical/<em>in silico</em> studies focused also on point mutations and nucleotide substitutions are much more common than indels (<em>i.e.</em>, the average ratio of nucleotide substitutions to indels for bacteria is 19.6) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Chen <em>et al.</em> 2009</a>).</p><p id="p-13">We adopted the bacterial/archaeal genetic code (NCBI genetic code 11). We generated each initial sequence according to the different GC contents: 30%, 50%, and 70%. The empirical GC content among the bacterial genomes studied here varies between 13.5% and 74.9%. We take into account the effect of different codon usages by constraining our simulations to three distinct %GC scenarios. In a first set of simulations (scenario 1), we performed 100,000 replicates for each scenario. Each replicate consisted of a random 186-nt-long (plus one or two nucleotides, depending on the phase) sequence containing two neighboring and nonoverlapping protein-coding sequences (the 3′-end of gene 1, an intergenic region, and the 5′-end of gene 2) oriented in the same direction (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Both gene ends started with the same length, 21 codons, and the intergenic distance was set to 60-nt-long (plus one or two nucleotides, depending on the phase). We defined these lengths to constrain the maximum overlapping region to 59-nt-long (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Pallejà <em>et al.</em> 2008</a>). Then, we chose a start codon of gene 2 either according to the empirical start codon usage in prokaryotic genomes (80% ATG, 17% GTG, and 3% TTG) or assuming equal usage (one-third each). The alternative start codon ATT was not included in the simulations because it is rarely observed in prokaryotic genomes (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Cock and Whitworth 2010</a>). The stop codon of gene 1 was chosen at random (one-third TAA, one-third TAG, and one-third TGA). At this initial stage, we did not accept stop codons within the coding sequence, except the predefined one at the 3′-end of gene 1. In a second step, we generated random nucleotide changes, one at a time, with uniform probability along the sequence until an overlapping region between genes 1 and 2 was formed. However, we only accepted these nucleotide changes if gene 1 still had a single stop codon in the sequence and if gene 2 still had a start codon and no in-frame stop codons (premature stop codons). Here, we assumed that the function of the gene was not altered by its elongation or contraction. The disruption of stop codons must necessarily extend the 3′-end of a gene until the next occurring downstream stop codon is found. However, the elongation of the 5′-end of a gene after start codon disruption might depend on the availability of other genomic features [<em>i.e.</em>, ribosome-binding sites (RBS)]. To keep the simulations simple, we did not include RBS. Previous studies suggest that a large number of prokaryotic genes initiate translation without possessing RBS (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Skorski <em>et al.</em> 2006</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Scharff <em>et al.</em> 2011</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/19/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scheme of the overlapping genes formation simulations (scenario 1). (A) In the initial stage ,we generated a sequence that contained the last 21 codons of gene 1 (including the stop codon) and the first 21 codons of gene 2 (including the start codon). These two partial gene sequences have the same direction of transcription and are separated from each other by 60 plus phase nucleotides. (B) Both genes have the same limits of maximum contraction (down to one codon) or elongation (up to the end of the sequence). Light gray arrows represent the limits of gene 1 size variation, and dark gray arrows and boxes represent gene 2. (C) Next, we made random changes (small open arrows) at the nucleotide level across the sequence until an overlapping region was created (*). No gene size variation occurred if the changes occurred in the intergenic region or if they caused a synonymous/nonsynonymous mutation within the coding region (C1). However, if the changes caused the disruption of the existing start/stop codons or the creation of upstream stop codon for gene 1, then genes may alter their length (C2, C3) and ultimately create an overlap (C4, long overlap; C5, short overlap). Gene size variation was allowed to occur even without the formation of an overlapping region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2130579706" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Scheme of the overlapping genes formation simulations (scenario 1). (A) In the initial stage ,we generated a sequence that contained the last 21 codons of gene 1 (including the stop codon) and the first 21 codons of gene 2 (including the start codon). These two partial gene sequences have the same direction of transcription and are separated from each other by 60 plus phase nucleotides. (B) Both genes have the same limits of maximum contraction (down to one codon) or elongation (up to the end of the sequence). Light gray arrows represent the limits of gene 1 size variation, and dark gray arrows and boxes represent gene 2. (C) Next, we made random changes (small open arrows) at the nucleotide level across the sequence until an overlapping region was created (*). No gene size variation occurred if the changes occurred in the intergenic region or if they caused a synonymous/nonsynonymous mutation within the coding region (C1). However, if the changes caused the disruption of the existing start/stop codons or the creation of upstream stop codon for gene 1, then genes may alter their length (C2, C3) and ultimately create an overlap (C4, long overlap; C5, short overlap). Gene size variation was allowed to occur even without the formation of an overlapping region.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/19/F2.medium.gif" width="440" height="338"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/1/19/F2.medium.gif" width="440" height="338"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/19/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/19/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463826" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-14" class="first-child">Scheme of the overlapping genes formation simulations (scenario 1). (A) In the initial stage ,we generated a sequence that contained the last 21 codons of gene 1 (including the stop codon) and the first 21 codons of gene 2 (including the start codon). These two partial gene sequences have the same direction of transcription and are separated from each other by 60 plus phase nucleotides. (B) Both genes have the same limits of maximum contraction (down to one codon) or elongation (up to the end of the sequence). Light gray arrows represent the limits of gene 1 size variation, and dark gray arrows and boxes represent gene 2. (C) Next, we made random changes (small open arrows) at the nucleotide level across the sequence until an overlapping region was created (*). No gene size variation occurred if the changes occurred in the intergenic region or if they caused a synonymous/nonsynonymous mutation within the coding region (C1). However, if the changes caused the disruption of the existing start/stop codons or the creation of upstream stop codon for gene 1, then genes may alter their length (C2, C3) and ultimately create an overlap (C4, long overlap; C5, short overlap). Gene size variation was allowed to occur even without the formation of an overlapping region.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">The first in-frame stop codon for gene 1 defined the limit of its 3′-end. Gene 1 extended only with the disruption of the functional stop codon and if another stop codon was present downstream in the same frame (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, C3 and C4). The contraction of gene 1 occurred when a nonstop codon within this gene experienced a change that resulted in a premature stop codon (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, C2). Gene 2 changed its 5′-end position only when its start codon was disrupted. Then, we defined two possible scenarios: gene 2 contracted or elongated, depending on whether the next in-frame start codon occurred downstream (gene contraction) (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, C2) or upstream (gene elongation) (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>, C3 and C5) of the original (and disrupted) start codon, respectively. However, we also included a third scenario in which potential start codons occurred downstream and upstream of the disrupted start codon at the same time. Which start codon should be used? To account for this, we performed the simulations alternatively under the following three scenarios regarding the disruption of the start codon of gene 2: “elongation first,” with preference for gene 2 elongation, <em>i.e.</em>, we first searched for a potential start codon in the upstream region of the disrupted start codon; “contraction first,” with preference for gene 2 contraction, <em>i.e.</em>, we first searched for a potential start codon in the downstream region of the disrupted start codon; and “both,” with gene 2 elongation/contraction being equally probable, <em>i.e.</em>, we randomly searched for a potential start codon with no preference for the downstream or upstream region of the disrupted start codon. The simulation ended when genes 1 and 2 overlapped. At the end of each replicate, we recorded the gene that “caused” the overlap and the length of the resultant overlap, so we ended up with overlap length distributions for the different scenarios and phases. Finally, for each run we also recorded the total number of mutations tried as a surrogate for evolutionary time. Because each replicate consisted of only one set of two adjacent genes, we could easily specify whether the overlap would finally occur in phase 1 or in phase 2, simply by setting the intergenic distance between gene 1 and gene 2 to 61 and 62 nucleotides for phase 1 and phase 2 simulations, respectively. However, in prokaryotic genomes, both phases are not equally frequent, with phase 2 overlaps being approximately 2.5-times more frequent than phase 1 overlaps (<a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Fukuda <em>et al.</em> 2003</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Sabath <em>et al.</em> 2008</a>; <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Cock and Whitworth 2010</a>). To make meaningful quantitative comparisons between simulated and real phase 1/phase 2 frequencies, we weighted (<em>a posteriori</em>) the representativeness of each simulated phase in the overall overlap length distribution. In the simulations the sequences evolved until an overlap was formed, but this time was longer for phase 1 than for phase 2 overlaps. However, in real data phase 1 overlaps necessarily resulted from the same evolutionary time than phase 2 overlaps. Therefore, we weighted the phase 1/phase 2 frequencies obtained in the simulation with the number of mutations, which is a surrogate for the average time needed for phase 1/phase 2 overlaps to occur.</p><p id="p-16">We also performed two more simulations (scenarios 2 and 3) to test the effect of selection against very long overlaps (&gt;60 bp). In these scenarios, gene size and intergenic distance were retrieved from an empirical distribution of prokaryotic genomes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS2.pdf">Figure S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS3.pdf">Figure S3</a>) and no <em>a posteriori</em> weighting scheme was applied. For scenario 2 the overlap length had no restriction (apart from the full length of the overlapped gene), whereas in scenario 3 only overlaps shorter than 60 bp were accepted. In both simulation sets, gene sizes and intergenic distances were limited (50–1000 codons for gene size; 0–99 plus phase for intergenic distance) to make computations feasible.</p></div><div id="sec-3" class="subsection"><h3>Length distribution of real unidirectional prokaryotic overlapping genes</h3><p id="p-17">To compare the results of our simulations with real data, we retrieved nucleotide sequences from unidirectional overlapping gene pairs (OGPs) from 2151 prokaryotic genomes available in NCBI (until 20 September 2012; plasmid and phage sequences were excluded). To avoid counting near-identical OGPs, we grouped them according to sequence similarity using pairwise <em>blastx</em> searches. We clustered different OGPs if their amino acid sequence identity was ≥70% and if the open reading frame (ORF) coverage was ≥80%. We grouped the initial 1,076,734 OGPs into 84,258 clusters containing at least two OGPs. The remaining 247,532 OGPs were considered unique. From each cluster, we selected one OGP for each unique overlap length. Overlaps larger than 59 bp were not analyzed because these could contain a large number of misannotations (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Pallejà <em>et al.</em> 2008</a>). The final nonredundant dataset of unique OGPs consisted of 400,363 OGPs, from which we constructed the distribution of overlap length. We also calculated the overlap length distributions for different taxa in different domain and phyla and used them to test whether the overall frequency of short and long phase 1 and phase 2 overlaps was significantly different across groups. Finally, for each genome and for each phase, we computed the proportion of real overlapping genes relative to the potential ones, <em>i.e.</em>, unidirectional but nonoverlapping genes, to test if the degree of success of overlap formation differed between phases.</p></div></div><div class="section results" id="sec-4"><h2 class="">Results</h2><div id="sec-5" class="subsection"><h3>Simulation of point mutations in two neighboring nonoverlapping genes</h3><p id="p-18">In simulation scenario 1, overlapping regions originated by the elongation of the 3′-end of gene 1 were 1.9-times to 2.8-times more frequent than those originated by the 5′ elongation of gene 2 (<em>P</em> ≤ 0.001) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS5.pdf">Figure S5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS6.pdf">Figure S6</a>). Phase 1 simulations required approximately 1.5-times more mutations than phase 2 simulations to form overlaps (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). All simulated sets showed the same overall overlap length distribution (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5, A and B</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS7.pdf">Figure S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS8.pdf">Figure S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS9.pdf">Figure S9</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS10.pdf">Figure S10</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS11.pdf">Figure S11</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS12.pdf">Figure S12</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS13.pdf">Figure S13</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS14.pdf">Figure S14</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS15.pdf">Figure S15</a>), with the majority of phase 2 overlaps being short. The most frequent overlap length was 4 nt, followed by 1-nt overlaps. All phase 1 overlaps were long overlaps, and long overlaps were more frequent in phase 1 than in phase 2. When scenarios 2 and 3 were compared, we observed that scenario 3, which includes selection against very long overlaps, resulted in an overlap length distribution most similar to the empirical distribution (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Variation in GC content equally affected scenarios 2 and 3: low %GC resulted in shorter overlaps on average (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS7.pdf">Figure S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS8.pdf">Figure S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS9.pdf">Figure S9</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS10.pdf">Figure S10</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS11.pdf">Figure S11</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS12.pdf">Figure S12</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS13.pdf">Figure S13</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS14.pdf">Figure S14</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS15.pdf">Figure S15</a>). In addition, short overlaps were more frequent in genomes with low GC content (scenario 2, no limit for overlap length) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS10.pdf">Figure S10</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS11.pdf">Figure S11</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS12.pdf">Figure S12</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/19/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of overlaps caused by the elongation of gene 2 (downstream gene). The proportion of overlapping genes formed after the disruption of the start codon and elongation of gene 2 was significantly lower than the proportion formed by the elongation of gene 1 (P &#x2264; 0.001) for all simulations (scenario 1). Data shown correspond to the 50% GC content scenario, but the results with 30% and 70% GC content were similar (Supporting Information). The simulations were performed separately with three criteria: preference for gene 2 contraction (criterion &#x201C;Con&#x201D;); gene 2 elongation/contraction equally probable (criterion &#x201C;Both&#x201D;); and preference for gene 2 elongation (criterion &#x201C;Elong&#x201D;). Prop = F, start codons were chosen at random. Prop = T, start codons were chosen according to empirical codon usage in prokaryotic genomes (80% ATG, 17% GTG, and 3% TTG). Each simulation was replicated 106 times." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2130579706" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Proportion of overlaps caused by the elongation of gene 2 (downstream gene). The proportion of overlapping genes formed after the disruption of the start codon and elongation of gene 2 was significantly lower than the proportion formed by the elongation of gene 1 (&lt;em&gt;P&lt;/em&gt; &#x2264; 0.001) for all simulations (scenario 1). Data shown correspond to the 50% GC content scenario, but the results with 30% and 70% GC content were similar (Supporting Information). The simulations were performed separately with three criteria: preference for gene 2 contraction (criterion &#x201C;Con&#x201D;); gene 2 elongation/contraction equally probable (criterion &#x201C;Both&#x201D;); and preference for gene 2 elongation (criterion &#x201C;Elong&#x201D;). Prop = F, start codons were chosen at random. Prop = T, start codons were chosen according to empirical codon usage in prokaryotic genomes (80% ATG, 17% GTG, and 3% TTG). Each simulation was replicated 10&lt;sup&gt;6&lt;/sup&gt; times.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/19/F3.medium.gif" width="440" height="434"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/1/19/F3.medium.gif" width="440" height="434"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/19/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/19/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463926" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-19" class="first-child">Proportion of overlaps caused by the elongation of gene 2 (downstream gene). The proportion of overlapping genes formed after the disruption of the start codon and elongation of gene 2 was significantly lower than the proportion formed by the elongation of gene 1 (<em>P</em> ≤ 0.001) for all simulations (scenario 1). Data shown correspond to the 50% GC content scenario, but the results with 30% and 70% GC content were similar (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/008797SI.pdf">Supporting Information</a>). The simulations were performed separately with three criteria: preference for gene 2 contraction (criterion “Con”); gene 2 elongation/contraction equally probable (criterion “Both”); and preference for gene 2 elongation (criterion “Elong”). Prop = F, start codons were chosen at random. Prop = T, start codons were chosen according to empirical codon usage in prokaryotic genomes (80% ATG, 17% GTG, and 3% TTG). Each simulation was replicated 10<sup>6</sup> times.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463904/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463904/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Average number of mutations per replicate of phase 1 <em>vs.</em> phase 2 scenarios (simulation set 1)</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/19/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the unidirectional overlapping gene length distributions between simulated (scenario 1) and empirical data. (A) Hypothetical prokaryotic overlap lengths of unidirectional adjacent genes, calculated from simulated dataset (simulated parameters: GC content = 50%; criterion = Both; proportions of start codons = TRUE). Frequencies of overlaps in both phases were weighted according to the mutation rate between phase 1 and phase 2 simulations (rate = 1.59; see Table 1). Frequency proportions used were as follows: phase 1 = 38.6%; and phase 2 = 61.4%. (B) Distribution of empirical prokaryotic overlap lengths of unidirectional adjacent genes calculated from a nonredundant set of 400,363 overlapping gene pairs (phase 1 = 31.6%; phase 2 = 68.4%)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2130579706" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of the unidirectional overlapping gene length distributions between simulated (scenario 1) and empirical data. (A) Hypothetical prokaryotic overlap lengths of unidirectional adjacent genes, calculated from simulated dataset (simulated parameters: GC content = 50%; criterion = Both; proportions of start codons = TRUE). Frequencies of overlaps in both phases were weighted according to the mutation rate between phase 1 and phase 2 simulations (rate = 1.59; see Table 1). Frequency proportions used were as follows: phase 1 = 38.6%; and phase 2 = 61.4%. (B) Distribution of empirical prokaryotic overlap lengths of unidirectional adjacent genes calculated from a nonredundant set of 400,363 overlapping gene pairs (phase 1 = 31.6%; phase 2 = 68.4%).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/19/F4.medium.gif" width="440" height="200"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/1/19/F4.medium.gif" width="440" height="200"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/19/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/19/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463824" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-22" class="first-child">Comparison of the unidirectional overlapping gene length distributions between simulated (scenario 1) and empirical data. (A) Hypothetical prokaryotic overlap lengths of unidirectional adjacent genes, calculated from simulated dataset (simulated parameters: GC content = 50%; criterion = Both; proportions of start codons = TRUE). Frequencies of overlaps in both phases were weighted according to the mutation rate between phase 1 and phase 2 simulations (rate = 1.59; see <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Frequency proportions used were as follows: phase 1 = 38.6%; and phase 2 = 61.4%. (B) Distribution of empirical prokaryotic overlap lengths of unidirectional adjacent genes calculated from a nonredundant set of 400,363 overlapping gene pairs (phase 1 = 31.6%; phase 2 = 68.4%).</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/19/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the unidirectional overlapping gene length distributions between simulated scenarios (scenario 2 vs. scenario 3) testing the effect of selection against very long overlaps (overlaps &gt;60 bp). These simulations differ from the ones presented in Figure 4A. Gene sizes and intergenic distances were retrieved from the empirical distribution of prokaryotic genomes and no a posteriori weighting scheme was applied to the representativeness of phase 1 or phase 2. For practical reasons, gene sizes and intergenic distances were limited (50&#x2013;1000 codons for gene size; 0&#x2013;99 plus phase for intergenic distance). (A) Simulation scenario 2. Hypothetical prokaryotic overlap lengths of unidirectional adjacent genes calculated from simulated dataset (simulated parameters: GC content = 50%; criterion = Both; proportions of start codons = TRUE). No selection against overlap length &gt;60 bp was included. Bar plot is limited to show only overlap length 60 bp was included." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2130579706" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of the unidirectional overlapping gene length distributions between simulated scenarios (scenario 2 &lt;em&gt;vs.&lt;/em&gt; scenario 3) testing the effect of selection against very long overlaps (overlaps &gt;60 bp). These simulations differ from the ones presented in Figure 4A. Gene sizes and intergenic distances were retrieved from the empirical distribution of prokaryotic genomes and no &lt;em&gt;a posteriori&lt;/em&gt; weighting scheme was applied to the representativeness of phase 1 or phase 2. For practical reasons, gene sizes and intergenic distances were limited (50&#x2013;1000 codons for gene size; 0&#x2013;99 plus phase for intergenic distance). (A) Simulation scenario 2. Hypothetical prokaryotic overlap lengths of unidirectional adjacent genes calculated from simulated dataset (simulated parameters: GC content = 50%; criterion = Both; proportions of start codons = TRUE). No selection against overlap length &gt;60 bp was included. Bar plot is limited to show only overlap length &lt;60 bp. (B) Simulation scenario 3. Hypothetical prokaryotic overlap lengths of unidirectional adjacent genes calculated from simulated dataset (simulated parameters: GC content = 50%; criterion = Both; proportions of start codons = TRUE). Selection against overlap length &gt;60 bp was included.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/19/F5.medium.gif" width="440" height="212"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/1/19/F5.medium.gif" width="440" height="212"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/19/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/19/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463848" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-23" class="first-child">Comparison of the unidirectional overlapping gene length distributions between simulated scenarios (scenario 2 <em>vs.</em> scenario 3) testing the effect of selection against very long overlaps (overlaps &gt;60 bp). These simulations differ from the ones presented in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>. Gene sizes and intergenic distances were retrieved from the empirical distribution of prokaryotic genomes and no <em>a posteriori</em> weighting scheme was applied to the representativeness of phase 1 or phase 2. For practical reasons, gene sizes and intergenic distances were limited (50–1000 codons for gene size; 0–99 plus phase for intergenic distance). (A) Simulation scenario 2. Hypothetical prokaryotic overlap lengths of unidirectional adjacent genes calculated from simulated dataset (simulated parameters: GC content = 50%; criterion = Both; proportions of start codons = TRUE). No selection against overlap length &gt;60 bp was included. Bar plot is limited to show only overlap length &lt;60 bp. (B) Simulation scenario 3. Hypothetical prokaryotic overlap lengths of unidirectional adjacent genes calculated from simulated dataset (simulated parameters: GC content = 50%; criterion = Both; proportions of start codons = TRUE). Selection against overlap length &gt;60 bp was included.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Length distribution of real unidirectional prokaryotic overlapping genes</h3><p id="p-24">The empirical nonredundant set of prokaryotic overlap length distribution was very similar to the simulated distribution described. Thus, phase 2 represented more than 68% of all overlaps (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). Most phase 2 overlaps were short (87% of all phase 2 overlaps and 59% of all overlaps), whereas phase 1 overlaps were essentially long overlaps. Long overlaps were significantly more frequent in phase 1 than in phase 2 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/TableS1.pdf">Table S1</a>). The most frequent overlap length was 4-nt-long (short phase 2 overlap; representing approximately 45% of all overlaps), and the second most frequent overlap length was 1-nt-long (also short phase 2 overlap; representing approximately 15% of all overlaps). A similar trend was found within all taxonomic groups analyzed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/TableS1.pdf">Table S1</a>). The ratio between observed and potential overlaps was significantly higher in phase 2 than in phase 1 overlaps (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>) (Kolmogorov-Smirnov test, one-tailed test, <em>P</em> ≤ 0.001).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/1/19/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Density plot of the ratio between real and potential prokaryotic unidirectional overlapping genes. Data were computed for each genome and for each overlapping phase. The relative degree of success of phase 2 overlaps is significantly higher than the degree of success of phase 1 overlaps (Kolmogorov-Smirnov test, one-tailed test, P &#x2264; 0.001)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2130579706" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Density plot of the ratio between real and potential prokaryotic unidirectional overlapping genes. Data were computed for each genome and for each overlapping phase. The relative degree of success of phase 2 overlaps is significantly higher than the degree of success of phase 1 overlaps (Kolmogorov-Smirnov test, one-tailed test, &lt;em&gt;P&lt;/em&gt; &#x2264; 0.001).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/1/19/F6.medium.gif" width="440" height="407"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/1/19/F6.medium.gif" width="440" height="407"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/1/19/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/1/19/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463855" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-25" class="first-child">Density plot of the ratio between real and potential prokaryotic unidirectional overlapping genes. Data were computed for each genome and for each overlapping phase. The relative degree of success of phase 2 overlaps is significantly higher than the degree of success of phase 1 overlaps (Kolmogorov-Smirnov test, one-tailed test, <em>P</em> ≤ 0.001).</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The empirical data show a negative correlation between GC content and frequency of long phase 1 and, to a lesser extent, phase 2 overlaps (long phase 1, <em>r<sup>2</sup></em> = 0.72, <em>P</em> &lt; 0.001; long phase 2, <em>r<sup>2</sup></em> = 0.06, <em>P</em> &lt; 0.001) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS19.pdf">Figure S19</a>) and a positive correlation between the frequency of short phase 2 overlaps and GC content (<em>r<sup>2</sup></em> = 0.69, <em>P</em> &lt; 0.001) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS19.pdf">Figure S19</a>).</p><p id="p-27">The empirical GC content was negatively correlated with the frequency of long phase 1 (<em>r<sup>2</sup></em> = 0.72, <em>P</em> &lt; 0.001) and long phase 2 (<em>r<sup>2</sup></em> = 0.06, <em>P</em> &lt; 0.001) overlaps, but was positively correlated with the frequency of short phase 2 overlaps (<em>r<sup>2</sup></em> = 0.69, <em>P</em> &lt; 0.001) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/FigureS19.pdf">Figure S19</a>).</p></div></div><div class="section discussion" id="sec-7"><h2 class="">Discussion</h2><div id="sec-8" class="subsection"><h3><em>In silico</em> patterns of overlapping genes formation</h3><p id="p-28">We have shown that unidirectional gene overlaps caused by the elongation of the upstream gene (gene 1) are more frequent than those caused by elongation of the downstream gene (gene 2) when stochastic nucleotide changes are simulated in a sequence containing two adjacent and nonoverlapping protein-coding genes. We believe that this pattern of formation of overlapping genes is attributable, in part, to the different robustness of the start codon and stop codon sets. Some nucleotide changes in start/stop codons are functionally redundant, <em>i.e.</em>, changes that convert a start codon into another start codon, or a stop codon into another stop codon. Under the same mutational pressures, the prokaryotic stop codon set is less redundant/robust than the start codon set, because only 4 out of 27 nucleotide changes will be functionally redundant. For the start codon set, 6 out of 27 possible nucleotide changes are functionally redundant. Thus, under a stochastic model of nucleotide change, the disruption probability for the start codon set is 0.78, whereas for the stop codon set it is 0.85. If stop codons are more frequently disrupted, then the 3′-end of protein-coding genes will also vary more frequently, and its elongation could, eventually, originate more gene overlaps than the elongation of the 5′-end.</p><p id="p-29">Interestingly, several empirical studies have also pointed out that overlapping regions originated by the 3′-end elongation of gene 1 are more frequent (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Fukuda <em>et al.</em> 1999</a>, <a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">2003</a>; <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Sakharkar <em>et al.</em> 2005</a>). However, redundancy asymmetry was not invoked to explain it. Instead, it was suggested that 3′-end elongation originates more overlapping genes because selective constraints are stronger at the 5′-end than at the 3′-end of protein-coding genes. The logic is that because the 5′-end together with its upstream region incorporates essential structures such as promoters, then it should be more constrained in size variation and consequently would originate less gene overlaps (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Rogozin <em>et al.</em> 2002</a>; <a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Fukuda <em>et al.</em> 2003</a>; <a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Sakharkar <em>et al.</em> 2005</a>). The two hypotheses, the redundancy asymmetry of start/stop codons and the different selective constraints between the 5′-end and 3′-end, are not mutually exclusive; therefore, both should be considered in future studies.</p><p id="p-30">On the contrary, <em>in silico</em> and theoretical studies have proposed that most unidirectional overlaps are formed with the elongation of the 5′-end of gene 2 (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Cock and Whitworth 2007</a>, <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">2010</a>; <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Sabath <em>et al.</em> 2008</a>). However, these studies focused on features that could influence the patterns of overlapping genes formation, but, importantly, given that the disruption of a start/stop codon already occurred. Our study suggests that taking into account the redundancy asymmetry between start/stop codon sets is key because both codon sets are not equiprobably disrupted, as we have shown. Additionally, we allowed elongation of gene 1 or gene 2 to freely occur in each simulation, which we think is more realistic than considering both elongations to be (almost) mutually exclusive (<a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Cock and Whitworth 2010</a>).</p><p id="p-31">Other factors or features could also affect the patterns of the formation of overlapping genes, such as variable mutation rates, Shine-Dalgarno translation initiation sites, and different selective constraints between 5′-end and 3′-end elongations or between 5′-UTRs and 3′-UTRs. However, the inclusion of these additional factors would complicate the interpretation of the observed patterns. Our strategy here was to simulate simple scenarios to understand the main features of the overlapping process; in fact, our results were in agreement with those of previous empirical studies (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Rogozin <em>et al.</em> 2002</a>; <a id="xref-ref-7-8" class="xref-bibr" href="#ref-7">Fukuda <em>et al.</em> 2003</a>; <a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">Sakharkar <em>et al.</em> 2005</a>).</p></div><div id="sec-9" class="subsection"><h3>Overlap length distribution in unidirectional overlapping genes</h3><p id="p-32">The resulting simulated and empirical overlap length distributions share four relevant features: phase 2 overlaps are more frequent than those in phase 1; the most frequent type of overlaps is short phase 2; the frequency of long overlaps decreases with the length of the overlap; and in long overlaps, those in phase 1 are more frequent.</p><p id="p-33">Previous studies have proposed that the evolution of overlapping genes may be related to the evolutionary time-scale, suggesting that they occur at a constant rate across species (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Fukuda <em>et al.</em> 1999</a>, <a id="xref-ref-7-9" class="xref-bibr" href="#ref-7">2003</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Johnson and Chisholm 2004</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1/008797SI.pdf">Supporting Information</a>). However, these studies did not distinguish between the evolutionary rates of phase 1 and phase 2 overlaps, which in turn could affect their proportion within a given genome. In set 1 simulations, the average number of mutations in phase 2 overlaps will be proportional to the evolutionary time, suggesting that in prokaryotic genomes phase 2 overlaps should form at a higher rate than phase 1 overlaps. We should also consider that phase 2 overlaps can be short, but all phase 1 overlaps have to be long if the (rare) start codon ATT is not used, as in our study. Thus, given the same potential elongation length, phase 2 overlaps have more length categories in which potential overlaps can occur (one-nucleotide and four-nucleotide overlaps). Ultimately, this may explain the higher formation rate for phase 2 overlaps. The fact that the proportion between real and potential overlaps is significantly higher in phase 2 supports this idea.</p><p id="p-34">Although translational coupling provides a biological justification for the higher frequency of short overlaps (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Lillo and Krakauer 2007</a>; <a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">Cock and Whitworth 2010</a>), our simulations suggest that these overlaps could be partially obtained without including any selective advantage (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Lillo and Krakauer 2007</a>). In fact, just looking at the genetic code we can realize that a short overlap is highly likely to occur after the disruption of a start or stop codon. For example, to form a 4-nt overlap with the elongation of the 3′-end of a gene, we only need to have an adenine after the start codon (<span class="underline">ATG</span><strong>A</strong>, <span class="underline">GTG</span><strong>A,</strong> or <span class="underline">TTG</span><strong>A</strong>) of gene 2, forming a TGA stop codon. Likewise, after the elongation of the 5′-end of gene 2, we only need to have an adenine just before the stop codon <strong>A</strong><span class="underline">TGA</span> to form an ATG start codon and therefore a 4-nt overlap. In previous simulation studies, the higher frequency of short phase 2 overlaps was also recovered without providing any selective advantage to short phase 2 overlaps (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Kingsford <em>et al.</em> 2007</a>; <a id="xref-ref-4-8" class="xref-bibr" href="#ref-4">Cock and Whitworth 2010</a>). <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Kingsford <em>et al.</em> (2007)</a> suggested that the overlap length distribution in unidirectional overlaps was significantly determined by the expected location of the stop codons for gene 1 within gene 2. On the contrary, <a id="xref-ref-4-9" class="xref-bibr" href="#ref-4">Cock and Whitworth (2010)</a> indicated that the overlap length distribution was instead significantly determined by the expected location of the start codons for gene 2 within gene 1. Importantly, we highlight the fact that in these studies only one scenario—elongation of gene 1 or elongation of gene 2—was invoked to explain the overlap length distribution. Here, we integrated both scenarios and recovered an overlap length distribution in agreement with that of the empirical data. Our results suggest that both the expected location of potential stop codons for gene 1 within gene 2 and the expected location of potential start codons for gene 2 within gene 1, together with the redundancy asymmetry between start and stop codons sets, can explain the higher frequency of short phase 2 overlaps. However, translational coupling, as a biological justification for the higher frequency of short overlaps (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Kingsford <em>et al.</em> 2007</a>; <a id="xref-ref-4-10" class="xref-bibr" href="#ref-4">Cock and Whitworth 2010</a>), could also be relevant in shaping the distribution of overlap length given the differences we found between simulated and empirical distributions.</p><p id="p-35">As in prokaryotic genomes, in our simulations the frequency of long overlaps decreased with the length of the overlap. However a lower decay rate with increasing overlap length resulted in “less shorter” overlaps and “more longer” overlaps for the simulation set with no selection against overlap length larger than 60 bp. These results suggest that selection against very long overlaps plays an important role in shaping real overlap length distribution, as previously suggested (<a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Kingsford <em>et al.</em> 2007</a>; <a id="xref-ref-4-11" class="xref-bibr" href="#ref-4">Cock and Whitworth 2010</a>). Nevertheless, our results also indicate that major patterns in the overlapping length distribution (short phase 2 as the most frequent; long phase 1 overlaps more frequent than long phase 2) and a decay rate can be partially explained by the probability of finding frameshifted start and stop codons across a protein-coding gene and by taking into account start/stop codon redundancy.</p><p id="p-36">Finally, both the simulated and the empirical overlap length distributions had a bias favoring long phase 1 over long phase 2 overlaps. If both phases have the same selective constraints (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Krakauer 2000</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Lillo and Krakauer 2007</a>), then why are long overlaps mainly in phase 1? Existing hypotheses involve selective factors (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Cock and Whitworth 2007</a>), compositional factors (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Sabath <em>et al.</em> 2008</a>), the genetic code itself, and intergenetic distances coupled with selection against longer overlaps (<a id="xref-ref-10-7" class="xref-bibr" href="#ref-10">Kingsford <em>et al.</em> 2007</a>; <a id="xref-ref-4-12" class="xref-bibr" href="#ref-4">Cock and Whitworth 2010</a>). All these studies compared intrinsic features between long phase 1 and phase 2 overlaps that could explain the higher frequency of long phase 1 overlaps. We hypothesize that short overlaps will have priority over long overlaps because the overlap limit will be defined by the first in-frame start/stop codon within the neighboring gene. In other words, an overlap will only be long if it cannot be short. In our simulations, most phase 2 overlaps were short, whereas all phase 1 overlaps were long. Very likely, phase 2 overlaps were “retained” as short and, for this reason, long phase 2 overlaps were underrepresented. It should be highlighted that short phase 2 retention is stronger if overlap length is limited up to 60 bp, suggesting again that selection against longer overlaps also plays a role in the overlapping genes length distribution.</p></div><div id="sec-10" class="subsection"><h3>Genomic GC content and overlap length distribution</h3><p id="p-37">The formation of overlapping genes depends on the existence of frameshifted start/stop codons, and the frequency of frameshifted start/stop codons decreases with increasing genomic GC content (<a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Sabath <em>et al.</em> 2008</a>). Lower frequencies of frameshifted start/stop codons favor the formation of longer overlapping genes. Thus, under neutrality, genomes with low GC content, which have lower frequencies of frameshifted start/stop codons, should have longer overlapping genes on average, and the proportions of long overlaps should increase over short overlaps. We did observe such a pattern in the simulations (scenario 2). However, the empirical data show a negative correlation between GC content and frequency of long phase 1 and, to a lesser extent, of phase 2 overlaps, which contradicts neutral expectations. Purifying selection against long overlaps could explain this negative correlation between long overlap frequency and GC content (<a id="xref-ref-10-8" class="xref-bibr" href="#ref-10">Kingsford <em>et al.</em> 2007</a>; <a id="xref-ref-19-7" class="xref-bibr" href="#ref-19">Sabath <em>et al.</em> 2008</a>; <a id="xref-ref-4-13" class="xref-bibr" href="#ref-4">Cock and Whitworth 2010</a>). Longer overlapping gene regions are more probable in genomes with higher GC content; therefore, stronger purifying selection is expected against these overlaps, leading to lower frequencies of long overlaps.</p><p id="p-38">Likewise, in the absence of selection, short overlaps should be more frequent in low GC content genomes. However, empirical data show a positive correlation between the frequency of short phase 2 overlaps and GC content. Positive selection for translational coupling of pairs of genes may explain this observation: in genomes with high GC content, in which selection acts against longer overlaps, short overlaps are formed to maintain the translational coupling of the gene pairs involved.</p></div></div><div class="section conclusions" id="sec-11"><h2 class="">Conclusions</h2><p id="p-39">In general, our integrated approach is concordant with previous studies in showing that the length distribution of unidirectional overlaps can be, in part, influenced by the genetic code (<a id="xref-ref-4-14" class="xref-bibr" href="#ref-4">Cock and Whitworth 2010</a>) and by the expected location of potential start (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Lillo and Krakauer 2007</a>; <a id="xref-ref-19-8" class="xref-bibr" href="#ref-19">Sabath <em>et al.</em> 2008</a>; <a id="xref-ref-4-15" class="xref-bibr" href="#ref-4">Cock and Whitworth 2010</a>) and stop codons (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Lillo and Krakauer 2007</a>; <a id="xref-ref-10-9" class="xref-bibr" href="#ref-10">Kingsford <em>et al.</em> 2007</a>). Additionally, we advocate that it is important to take into account the asymmetric redundancy of start/stop codon sets and also to consider the two possible scenarios, elongation of gene 1 or gene 2. Finally, we suggest that phase 2 overlaps are more frequent and occur at higher evolutionary rates than phase 1 overlaps because only phase 2 overlaps can be short, which in turn is the most probable category of overlap.</p><p id="p-40">To better understand the formation of overlapping genes, one could extend these simulations incorporating selective pressures in the novel overlapping regions (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Sabath and Graur 2010</a>) or include more complex structures, such as Shine-Dalgarno translation initiation sites. Further simulated and empirical studies are needed to decipher the contribution of overlapping gene death or overlap length contraction in the distribution of prokaryotic or viral overlapping genes (for eukaryotic genomes, see <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Makalowska <em>et al.</em> 2007</a>). We hope our results are useful to research related to the origin and evolution of internal overlaps formed by overprinting (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Sabath <em>et al.</em> 2012</a>), especially if redundancy asymmetries between start and stop are taken into account.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">We thank Josep Comeron for valuable contributions as associate editor and the anonymous reviewers. The European Social Fund and the Operational Human of Potential Program (POPH) supported this work. M.M.F. acknowledges support from the Portuguese Institution “Fundação para a Ciência e a Tecnologia” grant SFRH/BPD/70654/2010. D.P. was financially supported by the European Research Council (ERC-2007-Stg 203161-PHYGENOM).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005652/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-42"><em>Communicating editor: J. M. Comeron</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-01-14"><span class="received-label">Received </span>January 14, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-10-22"><span class="accepted-label">Accepted </span>October 22, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Fonseca <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.1" data-doi="10.1038/264034a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Air</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutchison</span>  <span class="cit-name-given-names">C. A.</span>  <span class="cit-name-suffix">3rd</span>.</span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Overlapping genes in bacteriophage phiX174.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">264</span>: <span class="cit-fpage">34</span>–<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBarrell%26rft.auinit1%253DB.%2BG.%26rft.volume%253D264%26rft.issue%253D5581%26rft.spage%253D34%26rft.epage%253D41%26rft.atitle%253DOverlapping%2Bgenes%2Bin%2Bbacteriophage%2BphiX174.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F264034a0%26rft_id%253Dinfo%253Apmid%252F1004533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/264034a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1004533&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1976CJ56500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.2" data-doi="10.1093/molbev/msp063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergelson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Variation in the ratio of nucleotide substitution and indel rates across genomes in mammals and bacteria.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1523</span>–<span class="cit-lpage">1531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp063%26rft_id%253Dinfo%253Apmid%252F19329651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI2LzcvMTUyMyI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzQvMS8xOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.3" data-doi="10.1007/s00239-006-0180-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cock</span>  <span class="cit-name-given-names">P. J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitworth</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evolution of gene overlaps: relative reading frame bias in prokaryotic two-component system genes.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DCock%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D64%26rft.issue%253D4%26rft.spage%253D457%26rft.epage%253D462%26rft.atitle%253DEvolution%2Bof%2Bgene%2Boverlaps%253A%2Brelative%2Breading%2Bframe%2Bbias%2Bin%2Bprokaryotic%2Btwo-component%2Bsystem%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-006-0180-1%26rft_id%253Dinfo%253Apmid%252F17479344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-006-0180-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17479344&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246275600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.4" data-doi="10.1093/molbev/msp302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cock</span>  <span class="cit-name-given-names">P. J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitworth</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evolution of relative reading frame bias in unidirectional prokaryotic gene overlaps.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">753</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp302%26rft_id%253Dinfo%253Apmid%252F20008458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI3LzQvNzUzIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.5" data-doi="10.1006/geno.1998.5221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smilinich</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Day</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nowak</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Divergently transcribed overlapping genes expressed in liver and kidney and located in the 11p15.5 imprinted domain.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DCooper%26rft.auinit1%253DP.%2BR.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D38%26rft.epage%253D51%26rft.atitle%253DDivergently%2Btranscribed%2Boverlapping%2Bgenes%2Bexpressed%2Bin%2Bliver%2Band%2Bkidney%2Band%2Blocated%2Bin%2Bthe%2B11p15.5%2Bimprinted%2Bdomain.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fgeno.1998.5221%26rft_id%253Dinfo%253Apmid%252F9570947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/geno.1998.5221&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9570947&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073166900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.6" data-doi="10.1093/nar/27.8.1847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fukuda</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Washio</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomita</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Comparative study of overlapping genes in the genomes of <em>Mycoplasma genitalium</em> and <em>Mycoplasma pneumoniae</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1847</span>–<span class="cit-lpage">1853</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.8.1847%26rft_id%253Dinfo%253Apmid%252F10101192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjI3LzgvMTg0NyI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzQvMS8xOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.7" data-doi="10.1016/j.gene.2003.09.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fukuda</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakayama</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomita</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">On dynamics of overlapping genes in bacterial genomes.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DFukuda%26rft.auinit1%253DY.%26rft.volume%253D323%26rft.spage%253D181%26rft.epage%253D187%26rft.atitle%253DOn%2Bdynamics%2Bof%2Boverlapping%2Bgenes%2Bin%2Bbacterial%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2003.09.021%26rft_id%253Dinfo%253Apmid%252F14659892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2003.09.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14659892&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187458100019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.8" data-doi="10.1101/gr.2433104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">Z. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chisholm</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Properties of overlapping genes are conserved across microbial genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2268</span>–<span class="cit-lpage">2272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DJohnson%26rft.auinit1%253DZ.%2BI.%26rft.volume%253D14%26rft.issue%253D11%26rft.spage%253D2268%26rft.epage%253D2272%26rft.atitle%253DProperties%2Bof%2Boverlapping%2Bgenes%2Bare%2Bconserved%2Bacross%2Bmicrobial%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2433104%26rft_id%253Dinfo%253Apmid%252F15520290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8xMS8yMjY4IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.9" data-doi="10.1073/pnas.89.20.9489"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keese</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbs</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Origins of genes: “big bang” or continuous creation?</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">9489</span>–<span class="cit-lpage">9493</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKeese%26rft.auinit1%253DP.%2BK.%26rft.volume%253D89%26rft.issue%253D20%26rft.spage%253D9489%26rft.epage%253D9493%26rft.atitle%253DOrigins%2Bof%2Bgenes%253A%2B%2522big%2Bbang%2522%2Bor%2Bcontinuous%2Bcreation%253F%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.89.20.9489%26rft_id%253Dinfo%253Apmid%252F1329098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODkvMjAvOTQ4OSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzQvMS8xOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.10" data-doi="10.1093/molbev/msm145"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kingsford</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A unified model explaining the offsets of overlapping and near-overlapping prokaryotic genes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2091</span>–<span class="cit-lpage">2098</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm145%26rft_id%253Dinfo%253Apmid%252F17642473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI0LzkvMjA5MSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzQvMS8xOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.11" data-doi="10.1111/j.0014-3820.2000.tb00075.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krakauer</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Stability and evolution of overlapping genes.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">731</span>–<span class="cit-lpage">739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DKrakauer%26rft.auinit1%253DD.%2BC.%26rft.volume%253D54%26rft.issue%253D3%26rft.spage%253D731%26rft.epage%253D739%26rft.atitle%253DStability%2Band%2Bevolution%2Bof%2Boverlapping%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0014-3820.2000.tb00075.x%26rft_id%253Dinfo%253Apmid%252F10937248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0014-3820.2000.tb00075.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10937248&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088090600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.12" data-doi="10.1186/1745-6150-2-22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lillo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krakauer</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A statistical analysis of the three-fold evolution of genomic compression through frame overlaps in prokaryotes.</span> <abbr class="cit-jnl-abbrev">Biol. Direct</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBiol%2BDirect%26rft.aulast%253DLillo%26rft.auinit1%253DF.%26rft.volume%253D2%26rft.spage%253D22%26rft.epage%253D22%26rft.atitle%253DA%2Bstatistical%2Banalysis%2Bof%2Bthe%2Bthree-fold%2Bevolution%2Bof%2Bgenomic%2Bcompression%2Bthrough%2Bframe%2Boverlaps%2Bin%2Bprokaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1745-6150-2-22%26rft_id%253Dinfo%253Apmid%252F17877818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1745-6150-2-22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17877818&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.13" data-doi="10.1186/1471-2148-7-193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Makalowska</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Birth and death of gene overlaps in vertebrates.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DMakalowska%26rft.auinit1%253DI.%26rft.volume%253D7%26rft.spage%253D193%26rft.epage%253D193%26rft.atitle%253DBirth%2Band%2Bdeath%2Bof%2Bgene%2Boverlaps%2Bin%2Bvertebrates.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-7-193%26rft_id%253Dinfo%253Apmid%252F17939861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-7-193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17939861&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.14" data-doi="10.1038/272532a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miyata</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yasunaga</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Evolution of overlapping genes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">272</span>: <span class="cit-fpage">532</span>–<span class="cit-lpage">535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMiyata%26rft.auinit1%253DT.%26rft.volume%253D272%26rft.issue%253D5653%26rft.spage%253D532%26rft.epage%253D535%26rft.atitle%253DEvolution%2Bof%2Boverlapping%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F272532a0%26rft_id%253Dinfo%253Apmid%252F692657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/272532a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=692657&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1978ET37900042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.15" data-doi="10.1016/0092-8674(83)90145-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montoya</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaines</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attardi</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">The pattern of transcription of the human mitochondrial rRNA genes reveals two overlapping transcription units.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMontoya%26rft.auinit1%253DJ.%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D151%26rft.epage%253D159%26rft.atitle%253DThe%2Bpattern%2Bof%2Btranscription%2Bof%2Bthe%2Bhuman%2Bmitochondrial%2BrRNA%2Bgenes%2Breveals%2Btwo%2Boverlapping%2Btranscription%2Bunits.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252883%252990145-9%26rft_id%253Dinfo%253Apmid%252F6883508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(83)90145-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6883508&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983RE12500020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.16" data-doi="10.1146/annurev.ge.17.120183.002435"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Normark</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergstrom</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edlund</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grundstrom</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaurin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Overlapping genes.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DNormark%26rft.auinit1%253DS.%26rft.volume%253D17%26rft.spage%253D499%26rft.epage%253D525%26rft.atitle%253DOverlapping%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ge.17.120183.002435%26rft_id%253Dinfo%253Apmid%252F6198955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ge.17.120183.002435&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6198955&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983RW26500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.17" data-doi="10.1186/1471-2164-9-335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pallejá</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrington</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bork</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Large gene overlaps in prokaryotic genomes: result of functional constraints or mispredictions?</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">335</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DPalleja%26rft.auinit1%253DA.%26rft.volume%253D9%26rft.spage%253D335%26rft.epage%253D335%26rft.atitle%253DLarge%2Bgene%2Boverlaps%2Bin%2Bprokaryotic%2Bgenomes%253A%2Bresult%2Bof%2Bfunctional%2Bconstraints%2Bor%2Bmispredictions%253F%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-335%26rft_id%253Dinfo%253Apmid%252F18627618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18627618&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.18" data-doi="10.1016/S0168-9525(02)02649-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogozin</span>  <span class="cit-name-given-names">I. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spiridonov</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorokin</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">Y. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">I. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Purifying and directional selection in overlapping prokaryotic genes.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">228</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DRogozin%26rft.auinit1%253DI.%2BB.%26rft.volume%253D18%26rft.issue%253D5%26rft.spage%253D228%26rft.epage%253D232%26rft.atitle%253DPurifying%2Band%2Bdirectional%2Bselection%2Bin%2Boverlapping%2Bprokaryotic%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252802%252902649-5%26rft_id%253Dinfo%253Apmid%252F12047938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(02)02649-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12047938&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175235200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.19" data-doi="10.1186/1745-6150-3-36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabath</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graur</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landan</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Unidirectional overlapping genes in bacteria: compositional determinants of phase bias.</span> <abbr class="cit-jnl-abbrev">Biol. Direct</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBiol%2BDirect%26rft.aulast%253DSabath%26rft.auinit1%253DN.%26rft.volume%253D3%26rft.spage%253D36%26rft.epage%253D36%26rft.atitle%253DSame-strand%2Boverlapping%2Bgenes%2Bin%2Bbacteria%253A%2Bcompositional%2Bdeterminants%2Bof%2Bphase%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1745-6150-3-36%26rft_id%253Dinfo%253Apmid%252F18717987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1745-6150-3-36&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18717987&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.20" data-doi="10.1007/s00239-010-9386-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabath</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graur</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Detection of functional overlapping genes: simulation and case studies.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">308</span>–<span class="cit-lpage">316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DSabath%26rft.auinit1%253DN.%26rft.volume%253D71%26rft.issue%253D4%26rft.spage%253D308%26rft.epage%253D316%26rft.atitle%253DDetection%2Bof%2Bfunctional%2Boverlapping%2Bgenes%253A%2Bsimulation%2Band%2Bcase%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-010-9386-3%26rft_id%253Dinfo%253Apmid%252F20820768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-010-9386-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20820768&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282475700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.21" data-doi="10.1093/molbev/mss179"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabath</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karlin</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolution of viral proteins originated <em>de novo</em> by overprinting.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">3767</span>–<span class="cit-lpage">3780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss179%26rft_id%253Dinfo%253Apmid%252F22821011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8xMi8zNzY3IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.22" data-doi="10.1099/ijs.0.63446-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakharkar</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakharkar</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verma</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chow</span>  <span class="cit-name-given-names">V. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Comparative study of overlapping genes in bacteria, with special reference to <em>Rickettsia prowazekii</em> and <em>Rickettsia conorii</em>.</span> <abbr class="cit-jnl-abbrev">Int. J. Syst. Evol. Microbiol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1205</span>–<span class="cit-lpage">1209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BSystematic%2Band%2BEvolutionary%2BMicrobiology%26rft.stitle%253DInt.%2BJ.%2BSyst.%2BEvol.%2BMicrobiol.%26rft.aulast%253DSakharkar%26rft.auinit1%253DK.%2BR.%26rft.volume%253D55%26rft.issue%253D3%26rft.spage%253D1205%26rft.epage%253D1209%26rft.atitle%253DComparative%2Bstudy%2Bof%2Boverlapping%2Bgenes%2Bin%2Bbacteria%252C%2Bwith%2Bspecial%2Breference%2Bto%2BRickettsia%2Bprowazekii%2Band%2BRickettsia%2Bconorii%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fijs.0.63446-0%26rft_id%253Dinfo%253Apmid%252F15879256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/ijs.0.63446-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15879256&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229362700034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.23" data-doi="10.1371/journal.pgen.1002155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scharff</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Childs</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walther</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bock</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Local absence of secondary structure permits translation of mRNAs that lack ribosome-binding sites.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DScharff%26rft.auinit1%253DL.%2BB.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De1002155%26rft.epage%253De1002155%26rft.atitle%253DLocal%2Babsence%2Bof%2Bsecondary%2Bstructure%2Bpermits%2Btranslation%2Bof%2BmRNAs%2Bthat%2Black%2Bribosome-binding%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002155%26rft_id%253Dinfo%253Apmid%252F21731509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21731509&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.24" data-doi="10.1128/JB.00591-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Skorski</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leroy</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fayet</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreyfus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hermann-Le Denmat</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The highly efficient translation initiation region from the <em>Escherichia coli</em> rpsA gene lacks a Shine-Dalgarno element.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">6277</span>–<span class="cit-lpage">6285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DSkorski%26rft.auinit1%253DP.%26rft.volume%253D188%26rft.issue%253D17%26rft.spage%253D6277%26rft.epage%253D6285%26rft.atitle%253DThe%2BHighly%2BEfficient%2BTranslation%2BInitiation%2BRegion%2Bfrom%2Bthe%2BEscherichia%2Bcoli%2BrpsA%2BGene%2BLacks%2Ba%2BShine-Dalgarno%2BElement%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00591-06%26rft_id%253Dinfo%253Apmid%252F16923895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4OC8xNy82Mjc3IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNC8xLzE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.25" data-doi="10.1073/pnas.77.9.5370"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkinson</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Overlapping genes at the cheA locus of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">5370</span>–<span class="cit-lpage">5374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSmith%26rft.auinit1%253DR.%2BA.%26rft.volume%253D77%26rft.issue%253D9%26rft.spage%253D5370%26rft.epage%253D5374%26rft.atitle%253DOverlapping%2Bgenes%2Bat%2Bthe%2BcheA%2Blocus%2Bof%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.77.9.5370%26rft_id%253Dinfo%253Apmid%252F6449010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3Ny85LzUzNzAiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy80LzEvMTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.26" data-doi="10.1038/322279a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spencer</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gietz</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hodgetts</span>  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Overlapping transcription units in the dopa decarboxylase region of <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">322</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSpencer%26rft.auinit1%253DC.%2BA.%26rft.volume%253D322%26rft.issue%253D6076%26rft.spage%253D279%26rft.epage%253D281%26rft.atitle%253DOverlapping%2Btranscription%2Bunits%2Bin%2Bthe%2Bdopa%2Bdecarboxylase%2Bregion%2Bof%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F322279a0%26rft_id%253Dinfo%253Apmid%252F2874495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/322279a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2874495&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986D250300059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.27" data-doi="10.1016/S0065-2660(02)46013-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altuvia</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romby</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Antisense RNAs in bacteria and their genetic elements.</span> <abbr class="cit-jnl-abbrev">Adv. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">361</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bgenetics%26rft.stitle%253DAdv%2BGenet%26rft.aulast%253DWagner%26rft.auinit1%253DE.%2BG.%26rft.volume%253D46%26rft.spage%253D361%26rft.epage%253D398%26rft.atitle%253DAntisense%2BRNAs%2Bin%2Bbacteria%2Band%2Btheir%2Bgenetic%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-2660%252802%252946013-0%26rft_id%253Dinfo%253Apmid%252F11931231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-2660(02)46013-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11931231&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185944200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.1.19.28" data-doi="10.1101/gr.200601"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szustakowski</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fortnow</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasif</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Computational identification of operons in microbial genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1221</span>–<span class="cit-lpage">1230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DZheng%26rft.auinit1%253DY.%26rft.volume%253D12%26rft.issue%253D8%26rft.spage%253D1221%26rft.epage%253D1230%26rft.atitle%253DComputational%2BIdentification%2Bof%2BOperons%2Bin%2BMicrobial%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.200601%26rft_id%253Dinfo%253Apmid%252F12176930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEyLzgvMTIyMSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzQvMS8xOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/1/19.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/1/11" title="Design and Analysis of Bar-seq Experiments" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/1/29" title="A High-Density Gene Map of Loblolly Pine (Pinus taeda L.) Based on Exome Sequence Capture Genotyping" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/1"><span>Volume 4 Issue 1, January 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/1.cover-source.jpg?itok=oIsy65sY" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/1/19.short" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AHHEPFWg1tSwKKinlmBfHdkFUVeCMx8QHRLPb9W2ADo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/1/19" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/1/19" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/1/19" class="active" data-icon-position="" data-hide-link-title="0">Origin and Length Distribution of Unidirectional Prokaryotic Overlapping Genes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459606" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-K7slCTvFAtY1b8rHQI6sdLgoKSkwaqLXispwoOVefnE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/1/19.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/1/19" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-YirAIWeBHQDbL3UzN-x3yCLlzvjI8Y2qPie2Wle-2s4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/1/19" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/1/19.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459606" id="node459606--2" data-pisa="ggg;4/1/19" data-pisa-master="ggg;g3.113.005652" data-apath="/ggg/4/1/19.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Origin and Length Distribution of Unidirectional Prokaryotic Overlapping Genes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Miguel M. Fonseca</span>, <span class="highwire-citation-author" data-delta="1">D. James Harris</span> and <span class="highwire-citation-author" data-delta="2">David Posada</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">19-27;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005652 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459606--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Miguel M. Fonseca </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Genetics and Immunology, University of Vigo, 36310 Vigo, Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>CIBIO, Research Center in Biodiversity and Genetic Resources, InBIO Laboratório Associado, 4485-661 Vairão, University of Porto, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miguel%2BM.%2BFonseca%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fonseca%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiguel%2BM.%2BFonseca%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mig.m.fonseca@gmail.com" class="" data-icon-position="" data-hide-link-title="0">mig.m.fonseca@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">D. James Harris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>CIBIO, Research Center in Biodiversity and Genetic Resources, InBIO Laboratório Associado, 4485-661 Vairão, University of Porto, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D.%2BJames%2BHarris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harris%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD.%2BJames%2BHarris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David Posada </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Genetics and Immunology, University of Vigo, 36310 Vigo, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BPosada%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Posada%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BPosada%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/1/19" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/1/19&amp;title=Origin%20and%20Length%20Distribution%20of%20Unidirectional%20Prokaryotic%20Overlapping%20Genes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/1/19&amp;title=Origin%20and%20Length%20Distribution%20of%20Unidirectional%20Prokaryotic%20Overlapping%20Genes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/1/19&amp;title=Origin%20and%20Length%20Distribution%20of%20Unidirectional%20Prokaryotic%20Overlapping%20Genes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/1/19&amp;text=Origin%20and%20Length%20Distribution%20of%20Unidirectional%20Prokaryotic%20Overlapping%20Genes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/1/19&amp;title=Origin%20and%20Length%20Distribution%20of%20Unidirectional%20Prokaryotic%20Overlapping%20Genes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/1/19&amp;t=Origin%20and%20Length%20Distribution%20of%20Unidirectional%20Prokaryotic%20Overlapping%20Genes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/1/19&amp;title=Origin%20and%20Length%20Distribution%20of%20Unidirectional%20Prokaryotic%20Overlapping%20Genes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/1/19&amp;title=Origin%20and%20Length%20Distribution%20of%20Unidirectional%20Prokaryotic%20Overlapping%20Genes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459606/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459606" id="node459606--3" data-pisa="ggg;4/1/19" data-pisa-master="ggg;g3.113.005652" data-apath="/ggg/4/1/19.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Origin and Length Distribution of Unidirectional Prokaryotic Overlapping Genes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Miguel M. Fonseca</span>, <span class="highwire-citation-author" data-delta="1">D. James Harris</span> and <span class="highwire-citation-author" data-delta="2">David Posada</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">19-27;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005652 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459606--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Miguel M. Fonseca </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Genetics and Immunology, University of Vigo, 36310 Vigo, Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>CIBIO, Research Center in Biodiversity and Genetic Resources, InBIO Laboratório Associado, 4485-661 Vairão, University of Porto, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miguel%2BM.%2BFonseca%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fonseca%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiguel%2BM.%2BFonseca%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mig.m.fonseca@gmail.com" class="" data-icon-position="" data-hide-link-title="0">mig.m.fonseca@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">D. James Harris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>CIBIO, Research Center in Biodiversity and Genetic Resources, InBIO Laboratório Associado, 4485-661 Vairão, University of Porto, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D.%2BJames%2BHarris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harris%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD.%2BJames%2BHarris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David Posada </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Genetics and Immunology, University of Vigo, 36310 Vigo, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BPosada%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Posada%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BPosada%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=19&amp;rft.epage=27&amp;rft.atitle=Origin%20and%20Length%20Distribution%20of%20Unidirectional%20Prokaryotic%20Overlapping%20Genes&amp;rft.volume=4&amp;rft.issue=1&amp;rft.date=2014-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Fonseca%2C+Miguel+M.&amp;rft.au=Harris%2C+D.+James&amp;rft.au=Posada%2C+David"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459606/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459606/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459606/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459606/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459606/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459606/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459606/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459606/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459606/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459606/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459606/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459606/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/1/19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/1/19.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/1/19.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/1/19.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/1/19.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
