<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/10/3243" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/10/3243.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/10/3243.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/10/3243.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/10/3243.atom" />
<meta name="HW.pisa" content="ggg;7/10/3243" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="High-Quality de Novo Genome Assembly of the Dekkera bruxellensis Yeast Using Nanopore MinION Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.117.300128" />
<meta name="DC.Date" content="2017-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Fournier et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genetic variation in natural populations represents the raw material for phenotypic diversity. Species-wide characterization of genetic variants is crucial to have a deeper insight into the genotype-phenotype relationship. With the advent of new sequencing strategies and more recently the release of long-read sequencing platforms, it is now possible to explore the genetic diversity of any nonmodel organisms, representing a fundamental resource for biological research. In the frame of population genomic surveys, a first step is to obtain the complete sequence and high-quality assembly of a reference genome. Here, we sequenced and assembled a reference genome of the nonconventional Dekkera bruxellensis yeast. While this species is a major cause of wine spoilage, it paradoxically contributes to the specific flavor profile of some Belgium beers. In addition, an extreme karyotype variability is observed across natural isolates, highlighting that D. bruxellensis genome is very dynamic. The whole genome of the D. bruxellensis UMY321 isolate was sequenced using a combination of Nanopore long-read and Illumina short-read sequencing data. We generated the most complete and contiguous de novo assembly of D. bruxellensis to date and obtained a first glimpse into the genomic variability within this species by comparing the sequences of several isolates. This genome sequence is therefore of high value for population genomic surveys and represents a reference to study genome dynamic in this yeast species." />
<meta name="DC.Contributor" content="Téo Fournier" />
<meta name="DC.Contributor" content="Jean-Sébastien Gounot" />
<meta name="DC.Contributor" content="Kelle Freel" />
<meta name="DC.Contributor" content="Corinne Cruaud" />
<meta name="DC.Contributor" content="Arnaud Lemainque" />
<meta name="DC.Contributor" content="Jean-Marc Aury" />
<meta name="DC.Contributor" content="Patrick Wincker" />
<meta name="DC.Contributor" content="Joseph Schacherer" />
<meta name="DC.Contributor" content="Anne Friedrich" />
<meta name="article:published_time" content="2017-10-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="High-Quality de Novo Genome Assembly of the Dekkera bruxellensis Yeast Using Nanopore MinION Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genetic variation in natural populations represents the raw material for phenotypic diversity. Species-wide characterization of genetic variants is crucial to have a deeper insight into the genotype-phenotype relationship. With the advent of new sequencing strategies and more recently the release of long-read sequencing platforms, it is now possible to explore the genetic diversity of any nonmodel organisms, representing a fundamental resource for biological research. In the frame of population genomic surveys, a first step is to obtain the complete sequence and high-quality assembly of a reference genome. Here, we sequenced and assembled a reference genome of the nonconventional &lt;i&gt;Dekkera bruxellensis&lt;/i&gt; yeast. While this species is a major cause of wine spoilage, it paradoxically contributes to the specific flavor profile of some Belgium beers. In addition, an extreme karyotype variability is observed across natural isolates, highlighting that &lt;i&gt;D. bruxellensis&lt;/i&gt; genome is very dynamic. The whole genome of the &lt;i&gt;D. bruxellensis&lt;/i&gt; UMY321 isolate was sequenced using a combination of Nanopore long-read and Illumina short-read sequencing data. We generated the most complete and contiguous &lt;i&gt;de novo&lt;/i&gt; assembly of &lt;i&gt;D. bruxellensis&lt;/i&gt; to date and obtained a first glimpse into the genomic variability within this species by comparing the sequences of several isolates. This genome sequence is therefore of high value for population genomic surveys and represents a reference to study genome dynamic in this yeast species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/10/01" />
<meta name="citation_mjid" content="ggg;7/10/3243" />
<meta name="citation_id" content="7/10/3243" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/10/3243" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/10/3243.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/10/3243.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/10/3243.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300128" />
<meta name="citation_pmid" content="28983066" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="3243" />
<meta name="citation_lastpage" content="3250" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Téo Fournier" />
<meta name="citation_author_institution" content="Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France" />
<meta name="citation_author" content="Jean-Sébastien Gounot" />
<meta name="citation_author_institution" content="Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France" />
<meta name="citation_author" content="Kelle Freel" />
<meta name="citation_author_institution" content="Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France" />
<meta name="citation_author" content="Corinne Cruaud" />
<meta name="citation_author_institution" content="Commissariat à l’Energie Atomique, Institut de Biologie François-Jacob, Genoscope, BP5706, 91057 Evry, France" />
<meta name="citation_author" content="Arnaud Lemainque" />
<meta name="citation_author_institution" content="Commissariat à l’Energie Atomique, Institut de Biologie François-Jacob, Genoscope, BP5706, 91057 Evry, France" />
<meta name="citation_author" content="Jean-Marc Aury" />
<meta name="citation_author_institution" content="Commissariat à l’Energie Atomique, Institut de Biologie François-Jacob, Genoscope, BP5706, 91057 Evry, France" />
<meta name="citation_author" content="Patrick Wincker" />
<meta name="citation_author_institution" content="Commissariat à l’Energie Atomique, Institut de Biologie François-Jacob, Genoscope, BP5706, 91057 Evry, France" />
<meta name="citation_author" content="Joseph Schacherer" />
<meta name="citation_author_institution" content="Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France" />
<meta name="citation_author_email" content="anne.friedrich@unistra.fr" />
<meta name="citation_author_email" content="schacherer@unistra.fr" />
<meta name="citation_author" content="Anne Friedrich" />
<meta name="citation_author_institution" content="Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France" />
<meta name="citation_author_email" content="anne.friedrich@unistra.fr" />
<meta name="citation_author_email" content="schacherer@unistra.fr" />
<meta name="citation_reference" content="citation_journal_title=Food Microbiol.;citation_author=W. Albertin;citation_author=A. Panfili;citation_author=C. Miot-Sertier;citation_author=A. Goulielmakis;citation_author=A. Delcamp;citation_title=Development of microsatellite markers for the rapid and reliable genotyping of Brettanomyces bruxellensis at strain level.;citation_pages=188-195;citation_volume=42;citation_year=2014;citation_pmid=24929736;citation_doi=10.1016/j.fm.2014.03.012" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=P. Almeida;citation_author=C. Gonçalves;citation_author=S. Teixeira;citation_author=D. Libkind;citation_author=M. Bontrager;citation_title=A Gondwanan imprint on global diversity and domestication of wine and cider yeast Saccharomyces uvarum.;citation_pages=4044;citation_volume=5;citation_year=2014;citation_pmid=24887054" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=P. Almeida;citation_author=R. Barbosa;citation_author=P. Zalar;citation_author=Y. Imanishi;citation_author=K. Shimizu;citation_title=A population genomics insight into the Mediterranean origins of wine yeast domestication.;citation_pages=5412-5427;citation_volume=24;citation_year=2015;citation_doi=10.1111/mec.13341" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Bergström;citation_author=J. T. Simpson;citation_author=F. Salinas;citation_author=B. Barré;citation_author=L. Parts;citation_title=A high-definition view of functional genetic variation from natural yeast genomes.;citation_volume=31;citation_year=2014;citation_pmid=24425782;citation_doi=10.1093/molbev/msu037" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=K. Berlin;citation_author=S. Koren;citation_author=C.-S. Chin;citation_author=J. P. Drake;citation_author=J. M. Landolin;citation_title=Assembling large genomes with single-molecule sequencing and locality-sensitive hashing.;citation_pages=623-630;citation_volume=33;citation_year=2015;citation_pmid=26006009;citation_doi=10.1038/nbt.3238" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=M. Boetzer;citation_author=W. Pirovano;citation_title=SSPACE-LongRead: scaffolding bacterial draft genomes using long read sequence information.;citation_pages=211;citation_volume=15;citation_year=2014;citation_pmid=24950923;citation_doi=10.1186/1471-2105-15-211" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. R. Borneman;citation_author=R. Zeppel;citation_author=P. J. Chambers;citation_author=C. D. Curtin;citation_title=Insights into the Dekkera bruxellensis genomic landscape: comparative genomics reveals variations in ploidy and nutrient utilisation potential amongst wine isolates.;citation_pages=e1004161;citation_volume=10;citation_year=2014;citation_pmid=24550744;citation_doi=10.1371/journal.pgen.1004161" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. Brion;citation_author=D. Pflieger;citation_author=A. Friedrich;citation_author=J. Schacherer;citation_title=Evolution of intraspecific transcriptomic landscapes in yeasts.;citation_volume=43;citation_year=2015;citation_pmid=25897111;citation_doi=10.1093/nar/gkv363" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=C. Brion;citation_author=D. Pflieger;citation_author=S. Souali-Crespo;citation_author=A. Friedrich;citation_author=J. Schacherer;citation_title=Differences in environmental stress response among yeasts is consistent with species-specific lifestyles.;citation_volume=27;citation_year=2016;citation_pmid=27009200;citation_doi=10.1091/mbc.E15-12-0816" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=S. Crauwels;citation_author=B. Zhu;citation_author=J. Steensels;citation_author=P. Busschaert;citation_author=G. De Samblanx;citation_title=Assessing genetic diversity among Brettanomyces yeasts by DNA fingerprinting and whole-genome sequencing.;citation_volume=80;citation_year=2014;citation_pmid=24814796;citation_doi=10.1128/AEM.00601-14" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=C. D. Curtin;citation_author=I. S. Pretorius;citation_title=Genomic insights into the evolution of industrial yeast species Brettanomyces bruxellensis.;citation_volume=14;citation_year=2014;citation_pmid=25142832;citation_doi=10.1111/1567-1364.12198" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=C. D. Curtin;citation_author=A. R. Borneman;citation_author=P. J. Chambers;citation_author=I. S. Pretorius;citation_title=De-novo assembly and analysis of the heterozygous triploid genome of the wine spoilage yeast Dekkera bruxellensis AWRI1499.;citation_pages=e33840-e33840;citation_volume=7;citation_year=2012;citation_issue=3;citation_pmid=22470482;citation_doi=10.1371/journal.pone.0033840" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. A. Fawcett;citation_author=T. Iida;citation_author=S. Takuno;citation_author=R. P. Sugino;citation_author=T. Kado;citation_title=Population genomics of the fission yeast Schizosaccharomyces pombe.;citation_pages=e104241;citation_volume=9;citation_year=2014;citation_pmid=25111393;citation_doi=10.1371/journal.pone.0104241" />
<meta name="citation_reference" content="citation_journal_title=Elife;citation_author=C. B. Ford;citation_author=J. M. Funt;citation_author=D. Abbey;citation_author=L. Issi;citation_author=C. Guiducci;citation_title=The evolution of drug resistance in clinical isolates of Candida albicans.;citation_pages=e00662;citation_volume=4;citation_year=2015;citation_pmid=25646566;citation_doi=10.7554/eLife.00662" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Friedrich;citation_author=P. Jung;citation_author=C. Reisser;citation_author=G. Fischer;citation_author=J. Schacherer;citation_title=Population genomics reveals chromosome-scale heterogeneous evolution in a protoploid yeast.;citation_volume=32;citation_year=2015;citation_pmid=25349286;citation_doi=10.1093/molbev/msu295" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=B. Gallone;citation_author=J. Steensels;citation_author=T. Prahl;citation_author=L. Soriaga;citation_author=V. Saels;citation_title=Domestication and divergence of Saccharomyces cerevisiae beer yeasts.;citation_pages=1397-1410.e16;citation_volume=166;citation_year=2016;citation_pmid=27610566;citation_doi=10.1016/j.cell.2016.08.020" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=M. Gonçalves;citation_author=A. Pontes;citation_author=P. Almeida;citation_author=R. Barbosa;citation_author=M. Serra;citation_title=Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts.;citation_pages=2750-2761;citation_volume=26;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=L. Hellborg;citation_author=J. Piškur;citation_title=Complex nature of the genome in a wine spoilage yeast, Dekkera bruxellensis.;citation_volume=8;citation_year=2009;citation_pmid=19717738;citation_doi=10.1128/EC.00115-09" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. P. Hirakawa;citation_author=D. A. Martinez;citation_author=S. Sakthikumar;citation_author=M. Z. Anderson;citation_author=A. Berlin;citation_title=Genetic and phenotypic intra-species variation in Candida albicans.;citation_volume=25;citation_year=2015;citation_pmid=25504520;citation_doi=10.1101/gr.174623.114" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=B. Istace;citation_author=A. Friedrich;citation_author=L. d’Agata;citation_author=S. Faye;citation_author=E. Payen;citation_title=De novo assembly and population genomic survey of natural yeast isolates with the Oxford Nanopore MinION sequencer.;citation_pages=1-13;citation_volume=6;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. Jain;citation_author=H. E. Olsen;citation_author=B. Paten;citation_author=M. Akeson;citation_title=The Oxford Nanopore MinION: delivery of nanopore sequencing to the genomics community.;citation_pages=239;citation_volume=17;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=D. C. Jeffares;citation_author=C. Rallis;citation_author=A. Rieux;citation_author=D. Speed;citation_author=M. Převorovský;citation_title=The genomic and phenotypic diversity of Schizosaccharomyces pombe.;citation_pages=235-241;citation_volume=47;citation_year=2015;citation_pmid=25665008;citation_doi=10.1038/ng.3215" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Kurtz;citation_author=A. Phillippy;citation_author=A. L. Delcher;citation_author=M. Smoot;citation_author=M. Shumway;citation_title=Versatile and open software for comparing large genomes.;citation_pages=R12-R12;citation_volume=5;citation_year=2004;citation_issue=2;citation_pmid=14759262;citation_doi=10.1186/gb-2004-5-2-r12" />
<meta name="citation_reference" content="citation_journal_title=Nat. Microbiol.;citation_author=J.-B. Leducq;citation_author=L. Nielly-Thibault;citation_author=G. Charron;citation_author=C. Eberlein;citation_author=J. P. Verta;citation_title=Speciation driven by hybridization and chromosomal plasticity in a wild yeast.;citation_pages=15003;citation_volume=1;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_title=Minimap and miniasm: fast mapping and de novo assembly for noisy long sequences.;citation_volume=32;citation_year=2016;citation_pmid=27153593;citation_doi=10.1093/bioinformatics/btw152" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Lin;citation_author=J. Yuan;citation_author=M. Kolmogorov;citation_author=M. W. Shen;citation_author=M. Chaisson;citation_title=Assembly of long error-prone reads using de Bruijn graphs.;citation_volume=113;citation_year=2016;citation_pmid=27956617;citation_doi=10.1073/pnas.1604560113" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. M. Magwene;citation_author=Ö. Kayıkçı;citation_author=J. A. Granek;citation_author=J. M. Reininga;citation_author=Z. Scholl;citation_title=Outcrossing, mitotic recombination, and life-history trade-offs shape genome evolution in Saccharomyces cerevisiae.;citation_volume=108;citation_year=2011;citation_pmid=21245305;citation_doi=10.1073/pnas.1012544108" />
<meta name="citation_reference" content="citation_journal_title=Front Microbiol.;citation_author=I. Masneuf-Pomarede;citation_author=M. Bely;citation_author=P. Marullo;citation_author=W. Albertin;citation_title=The genetics of non-conventional wine yeasts: current knowledge and future challenges.;citation_pages=1563;citation_volume=6;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=R.-A. Olsen;citation_author=I. Bunikis;citation_author=I. Tiukova;citation_author=K. Holmberg;citation_author=B. Lötstedt;citation_title=De novo assembly of Dekkera bruxellensis: a multi technology approach using short and long-read sequencing and optical mapping.;citation_pages=56;citation_volume=4;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Parra;citation_author=K. Bradnam;citation_author=I. Korf;citation_title=CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes.;citation_volume=23;citation_year=2007;citation_pmid=17332020;citation_doi=10.1093/bioinformatics/btm071" />
<meta name="citation_reference" content="citation_journal_title=Yeast;citation_author=J. Peter;citation_author=J. Schacherer;citation_title=Population genomics of yeasts: towards a comprehensive view across a broad evolutionary scale.;citation_pages=73-81;citation_volume=33;citation_year=2016;citation_pmid=26592376;citation_doi=10.1002/yea.3142" />
<meta name="citation_reference" content="citation_journal_title=International journal of food microbiology;citation_journal_abbrev=Int J Food Microbiol;citation_author=J. Piškur;citation_author=Z. Ling;citation_author=M. Marcet-Houben;citation_author=O. P. Ishchuk;citation_author=A. Aerts;citation_title=The genome of wine yeast Dekkera bruxellensis provides a tool to explore its food-related properties.;citation_pages=202-209;citation_volume=157;citation_year=2012;citation_issue=2;citation_pmid=22663979;citation_doi=10.1016/j.ijfoodmicro.2012.05.008" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Schacherer;citation_author=J. A. Shapiro;citation_author=D. M. Ruderfer;citation_author=L. Kruglyak;citation_title=Comprehensive polymorphism survey elucidates population structure of Saccharomyces cerevisiae.;citation_pages=342-345;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212320;citation_doi=10.1038/nature07670" />
<meta name="citation_reference" content="citation_journal_title=Yeast;citation_author=A. J. Schifferdecker;citation_author=S. Dashko;citation_author=O. P. Ishchuk;citation_author=J. Piškur;citation_title=The wine and beer yeast Dekkera bruxellensis.;citation_pages=323-332;citation_volume=31;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. A. Skelly;citation_author=G. E. Merrihew;citation_author=M. Riffle;citation_author=C. F. Connelly;citation_author=E. O. Kerr;citation_title=Integrative phenomics reveals insight into the structure of phenotypic diversity in budding yeast.;citation_volume=23;citation_year=2013;citation_pmid=23720455;citation_doi=10.1101/gr.155762.113" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=C. S. Smukowski Heil;citation_author=C. G. DeSevo;citation_author=D. A. Pai;citation_author=C. M. Tucker;citation_author=M. L. Hoang;citation_title=Loss of heterozygosity drives adaptation in hybrid yeast.;citation_pages=1596-1612;citation_volume=34;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. K. Strope;citation_author=D. A. Skelly;citation_author=S. G. Kozmin;citation_author=G. Mahadevan;citation_author=E. A. Stone;citation_title=The 100-genomes strains, an S. cerevisiae resource that illuminates its natural phenotypic and genotypic variation and emergence as an opportunistic pathogen.;citation_volume=25;citation_year=2015;citation_pmid=25840857;citation_doi=10.1101/gr.185538.114" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. J. Walker;citation_author=T. Abeel;citation_author=T. Shea;citation_author=M. Priest;citation_author=A. Abouelliel;citation_title=Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement.;citation_pages=e112963;citation_volume=9;citation_year=2014;citation_pmid=25409509;citation_doi=10.1371/journal.pone.0112963" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=Y. O. Zhu;citation_author=G. Sherlock;citation_author=D. A. Petrov;citation_title=Whole genome analysis of 132 clinical Saccharomyces cerevisiae strains reveals extensive ploidy variation.;citation_pages=2421-2434;citation_volume=6;citation_year=2016" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="High-Quality de Novo Genome Assembly of the Dekkera bruxellensis Yeast Using Nanopore MinION Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
<meta name="twitter:description" content="Genetic variation in natural populations represents the raw material for phenotypic diversity. Species-wide characterization of genetic variants is crucial to have a deeper insight into the genotype-phenotype relationship. With the advent of new sequencing strategies and more recently the release of long-read sequencing platforms, it is now possible to explore the genetic diversity of any nonmodel organisms, representing a fundamental resource for biological research. In the frame of population genomic surveys, a first step is to obtain the complete sequence and high-quality assembly of a reference genome. Here, we sequenced and assembled a reference genome of the nonconventional Dekkera bruxellensis yeast. While this species is a major cause of wine spoilage, it paradoxically contributes to the specific flavor profile of some Belgium beers. In addition, an extreme karyotype variability is observed across natural isolates, highlighting that D. bruxellensis genome is very dynamic. The whole genome of the D. bruxellensis UMY321 isolate was sequenced using a combination of Nanopore long-read and Illumina short-read sequencing data. We generated the most complete and contiguous de novo assembly of D. bruxellensis to date and obtained a first glimpse into the genomic variability within this species by comparing the sequences of several isolates. This genome sequence is therefore of high value for population genomic surveys and represents a reference to study genome dynamic in this yeast species." />
<meta name="og-title" property="og:title" content="High-Quality de Novo Genome Assembly of the Dekkera bruxellensis Yeast Using Nanopore MinION Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/10/3243" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genetic variation in natural populations represents the raw material for phenotypic diversity. Species-wide characterization of genetic variants is crucial to have a deeper insight into the genotype-phenotype relationship. With the advent of new sequencing strategies and more recently the release of long-read sequencing platforms, it is now possible to explore the genetic diversity of any nonmodel organisms, representing a fundamental resource for biological research. In the frame of population genomic surveys, a first step is to obtain the complete sequence and high-quality assembly of a reference genome. Here, we sequenced and assembled a reference genome of the nonconventional Dekkera bruxellensis yeast. While this species is a major cause of wine spoilage, it paradoxically contributes to the specific flavor profile of some Belgium beers. In addition, an extreme karyotype variability is observed across natural isolates, highlighting that D. bruxellensis genome is very dynamic. The whole genome of the D. bruxellensis UMY321 isolate was sequenced using a combination of Nanopore long-read and Illumina short-read sequencing data. We generated the most complete and contiguous de novo assembly of D. bruxellensis to date and obtained a first glimpse into the genomic variability within this species by comparing the sequences of several isolates. This genome sequence is therefore of high value for population genomic surveys and represents a reference to study genome dynamic in this yeast species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
    <title>High-Quality de Novo Genome Assembly of the Dekkera bruxellensis Yeast Using Nanopore MinION Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"r-NhuYLMxhv4BxFpuSTSCIBN8YtnMwDXupoMqithWKs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"486771","apath":"\/ggg\/7\/10\/3243.atom","pisa":"ggg;7\/10\/3243","ac":{"\/ggg\/7\/10\/3243.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/10\/3243.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/10\/3243"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/10\/3243"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/10\/3243":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zfXh5bK226Lhb\\\\\\\/y+IrseQ5FliH54xTSeeXTZqu8fWmDuxGciRypRsH5JQnl9SYCll9g8QByAZyjG0FsIU33Y+BBZX2vwSvPTOZyRc\\\\\\\/Xfa6Wyr8cxmMUUhjIEnSA6JtCq72S9oVCofBuR6\\\\\\\/FBdHjq\\\\\\\/WvgxNrp8p7GaXSDxuB6GzzhlViLTBB5vJCuMNMCt87hrOnP4BT7aI93WhVFgAXf1wcZeqZ4Ke9ISYsQvfrm4d6w7jTRVeRcY0QJwuslAKKPDGFuU++WnUBqjytJbKDoVOWje61\\\\\\\/simb5ulN\\\\\\\/5sKsGKuHMivjZLDuHz6+k7shPd9ZL876twSTUK7aYpUF8M1ZMpG7lzoh8+qs6FUbMkV1jWlaNG7+PGpH6zjdkncmaJSyK9j3bEDVcZQh9c22kubk54jnG45We8nvmpp5IMiRyFebtQrn6ErucVO0ZKmlK26wzFx+o3BA3u01WvOcRtb7FoIUg5z9vBZQThtrwf4ULL\\\\\\\/s\\\\\\\/wZMB8vGV+pXLha8nyafEnj5EjrmYXye1vZpJbj0Evs9XAaykDOcoNKFhdZx7PBuAalubqIvTiOUe0LPteZgMIj43bJALebN0CQm\\\\\\\/9z2TJCWRhbIKCBBSkXLhaZVDz4W6fkLKOnKMEZ+tF8fQbArkGW+UYEwt8GLbHufOBjUWPhj+5bZ3yNWBbWQxkGo+jVV6h6m6gq1IeoTvlkm2pUEchJofzSiA+Q4x+vwk7qOepPkk5VBh1pwUefpkzi1rnGX9szS80GZ2BkCdorEcAtb4cYUdBsK7SnyKX6tX+ystBgzkt710ypOMenWIOgMAtYs1YN266cfvEtH74WtBM0vbiKrU4rcEy425CZ5Xof\\\\\\\/Gmn8nrG4jhz8Ehu545ifAyOzHFjwOcWm3AmT8sZVWl0WrgGdKh8\\\\\\\/+D7geGnbmQIVydOtMGT2KvG0tae0nEJQjjo9YUL1mMyWtJFlOl4OHPJ3xwWuFbnNqqK+6d0e\\\\\\\/wcusErb02ev1DfT6eY2tyJIbkgf\\\\\\\/IgrmT4AKKJ7S8tb87AdVsz9RiPM3W0jdsHpOop6PZyklFrkVSkX2oqAYGQDYkgJXA\\\\\\\/QwX\\\\\\\/jcTXCyaZFGYc6NLcQdiG\\\\\\\/n6B8hXaUJ169ItO6I9AHuaYAgUxiik5\\\\\\\/hSUApmUwpqV57lm+SwIHbr3EpGFNUUxWxQOQamC1aKAcCqcaFpDGq5sc\\\\\\\/hmrGCyxbYkSvitTgh58HNygU69Z8W8vxnUMpXJCpwOxTZwKUJXebffpWYxif8utCANXe9w\\\\\\\/ccy6irpn6PNNRBv+zoDlDRn\\\u0022,\\\u0022iv\\\u0022:\\\u0022aEUG8wIsXXFgJz6n9PuJ8g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cec21a3ac3659e3f9af1acbedd23952f8ae9d0a3362188ff905134bb21bbaffb\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222YXnOgbnfX0Hfw9eP1I4aFQowQjmr4kTZpfUwHy+9QazprLdEkgrnspioZApOn7eXhc8kd0tRNe8nnho0TQB4XxDVmG7en1sqr4jpuf\\\\\\\/j647pZNaN1hxJZ0MkuXbL4XHv4B5Ra83YFc8OIWC6YrqBYPNDuW8JWzxZo3ZNpxi5ey9U3r644TXdhfq59Q10pPefa\\\\\\\/kpXJD8D2iXyzjOH0go0E3TQNgReiQJq3X5VFprWm6G93r3DJqdupbjIHTK0kj5NhaWgIo+hXZ+YDRFBYPNhXNQMp8hEdcpii40zQAHXukXZGf1vammI0LmO25GvyPX51CcL7j830q7l7UNLLFcUTTsgx3ezBJjKjVepG4FRghvwJmOggd8KsDUIdR0sT1eaOYpoKcExMfdmxGqIdJJFv5F\\\\\\\/bYihIZSjBCnCMQggdblct5fpg1xRGcjPYDFUUpnfGFt8QptFnKvSGRJyxG1e+EXEiIiwHo8\\\\\\\/sn6saGfsTV1AHOXpavJeELvTWy6tHAsvl4CgVK7eQ0JPzvonuI89TMC\\\\\\\/yZyJW\\\\\\\/aGOC1bDwddkXVcsBOXlGx8UCz6QZWdlyR0mN3J4ESFz63FdSq3gzKQgM2YvbKyUJmHBO0gFWegs2w+hMD3VjpbHA4Qa4MTVQEGSouN8vBf8TOf+QpC7RShPh1J2bb1dhJ7AVAesHCEFdgcMU8VDNW3AaDrYNYoS41mpenW2NPSE6WYX7\\\\\\\/b8KpWfs6HEqMf1jYJ8S96pFiXwYT6pXcaPMHwvp3T5HHGqh2fp\\\\\\\/dVrixeIjemzOdIPJip\\\\\\\/yHjQFFSnx2sNMcuGQA1uQXETdad1Pr4+69dyKOkmpx8iAAjZDe0LchgUfYQAUw8ORp1yV2I6\\\\\\\/ef7GfpNIawpD9Ce+MmFu31DzZS\\\\\\\/fS4I+u6ob\\\\\\\/NufeTQwOVnJe+Mk0AHXjVMJfJv+Kzyo1QWs+waA4ZHNos9435k7oKy7dOeUyQYGF0DocS18ac7YV6I6pTRTnTP9yVbETYoKkDHRCCvTtLvaj3qlqlgdO5uD5YOw5l43dZc6kmX0O8ltEoXyzf539kCtp2hvk\\\\\\\/MPeK8YfsalpoQ2vDLeoU2Omnw4eXgJ2yrhDrQT1fAwl6Lo5AjF6AJpGanDuQbPXmnmduch2Tlf+ZG0XLatjMyZWjl2Pq\\\\\\\/6ux6NYCOsQp8T0GChJsW1+AAMGCGiKwR1PMlevnNjNGJJ5rw7IUpTcKgx1dZoOlGI7aPcHBQqeUtAqVCRIJw\\\\\\\/IROUUFGrKFSOdRlsJkLdo5e6z2XCNb5kRcv3ZI9jD1Ira7zo+Bq++BV54SiIOffFA5NT\\\\\\\/hOBXNA4h0Wmknk0k2mxX6WncoIQkaeieP49z+cc5SMd8F7z49GX424UR\\\\\\\/zUliXnkiXPFgFGQbuBUj7MMHzTA37XO2TJRbChfv7l3ksZrnntz8ccwZUfg3jS77FgsAFWb\\\\\\\/6sAKxMrlGThIEvlnxfV0kKIftk\\\\\\\/uQ4eaZNQBo0crviA97AKc\\\\\\\/py3sJ1T+ROLFAmpuTS0zsbd+RZt+ivaSrIfl1r14AKlhjUhUp9xBv3GgCYwhJh0k+XOcZ2XCHHwU6ZJrQcSn2Xnh0QH93TsCcUdoAwwGh+ZAXUqt1lBwi6oNMnOsC6u1lrBWIl9NN0NUw17EXT5RqbUTWtpfSLdSf5Uus5KnxNKCJPvWAf8G2kVoy\\\\\\\/BI6G7Mm8S8iPgQ7L3d5SWF8cmpkH2kFl0bbRruDVG31a3wE8QYViHWwto4U6Bw1oLLq0D8Q9h\\\\\\\/tALlrCHi2hvUiZthb6JdNYqLtn6hqxLE6lGFyJd5YozNumQB6E4PxzJR0XM\\\\\\\/16zOVZnbotEaUByp2mEDkGV9mDliNCr99LgOAqxtHye\\\\\\\/VAmUDlGutEOQ6FCEW9V5a20bjnxmqIevmO4snAoBpqZR3TMXvel90TSvXKnYrVf4PBy7PHFx+O1xnF5JyUPvaMVzo4zqKFjMCOeW5e7dSyEwJM7y2499pXy+loTYWP3phMQUhxz9X56RQIKbXMkcB7NlG3WxCSwUzjQ8Y\\\\\\\/ZJQ034gW+d+\\\\\\\/bgrtEDj9CxTMUOH2upleMwxS1cUs3gx9c9HvgsYNQpKwJWnswWnojDd7GVLdAstLAke9HLmanGUwJTASSe8J\\\\\\\/mMe53mhuL8fu+lSDmYoKPWhqBxuojMqcFrm26kOAibE40xHycAH4YOg+CVHLBsJF9oC6oIaFHOm+oZWj00+qQ0u5K9mNjLOlevS3XhByWFgvZd5X7z\\\\\\\/Vqs4pxejV0MdPhGcwl74RO44GVpJyhGzyLRnTnDb3E\\\\\\\/xdT+j2Kq9WI99H2vDwEFmloMIfkK4jwr9p6K3pttozHo0kQ0TKQsnZha8ZeNLiJqNBX6eLw4\\\\\\\/NtIBEnMKGxtyYNTgjfC7ffQiqDokwq27VzOLp+YSCUx6y9QvFuS9Vp37TjZ26bpCH8CPzF5RThM3Km6Wzf8y6q9RbsvhoFm63YetnFB9OPcdNKHsgbxmLTNaf5XB+qzT0RZJniNzWycBMD1LhA1QjigXjf\\\\\\\/6F6Cqsp3KI4iQFrQkFobJse1x46J7gIzeACoB3xFmRwqHovo3IAxPvFe8903hNAq70V8qhQ7yOfBymIk+icVcGAhhZFoYNhp8OXfytCXpgE\\\\\\\/j0RPmEkF5l+rUdeRt6K3gY2ckl1wc7rOqPbK4ZCm4XxBNEiNVSbwPESpBEFIP8oO5PA36T52vM7fYxIe1HK2k4Ya5DDBHOn2343V6A2lZp\\\\\\\/YOaMbszi87Jyb\\\\\\\/ImpWG7erl0MlMxgEgex8UjYRfyTX\\\\\\\/YX6r9Dphh2fNFv7iXUE2ZRjzgFdIlhQM3sMUrInhKitvOO+dpR5Li+2s6YXCr86WkPScqLxZCExrdQFi70\\\\\\\/2f5jjClij6qhKXrUpCWOxr5s7ONR02R1t9CycG2dhFDznKfiic2Ps2XQFdTxQ6jTJ3n+YpHYFePs1p7CcJjZtBoNZIc1Cd\\\\\\\/IFHzlBtJa61aDLRVmsQTulx9svJHWWonZ+T4dl1VokE4LmjUjhT1\\\\\\\/E4Kxe5fsFGIXtNnnv9R+YxTjpG8p1yPhvlhxM59ynIQ1sy0n53+5s+VSym+2Non1c3JyuLpVOqwp2PQL7tvIiKHwZfYZCLHSp\\\\\\\/DIBjyzi6w8hOwAwVM0r42Q2r5qHzzcf9MVKuItYYeio9xjPsBT6zNjGkvnrg2rPh5ORa4GMDicZKHKY\\\\\\\/Hx+9ZNUarbGn2yGMoLEvzQwW+z9zFBTOf7yInlHgc7SuowJ2ZBWMCbdNLlW1kQibZyeJyAtdef3hqEHDV\\\\\\\/kdIZIs1SKKjizxLJFF\\\\\\\/e+rg97kMteESJFHGg0vxhGknbMQ0HUOOHjU\\\\\\\/9NL+SINtrl1bJPVJPvy7dZeicq7WGY6hGiyUJzGb0bgfdcAV1bcp2YbZ\\\\\\\/5O+fviUgJigpYP\\\\\\\/DEQAFFks30Zqme6wxQgztUqUwpNIIy\\\\\\\/EEFCBjuxE0lz0fmtGFzRUkVud4\\\\\\\/CMHmxvP9UUFeq84QqSR13e0A0D4vyQkratoYyC4\\\\\\\/1kYUmEP9BVeRQ14j+2\\\\\\\/cmaBghLVjoQcPHc32ZU7Ug2KlVc87rwYbR5pWEIXXKGmEiIHUU4tT5P5fkVCFsKLWkPamHa+GW1x5rdX7SCsMrZy5dsIrlpkhFyuVzndnHpa6ivqBI+s2dCH7c+7wdioqtE\\\\\\\/auNPkFE1Z7+fMx34pDz9bjLF12btpgyZfKArBODbD6lEoX+xcGZqb9V6NZ+nbaM9zLqheNd2hUUphqfZuamroI1tfB21zyoxzrV\\\\\\\/gi6oOsqSMnGy4qJZw+f0dB140r82OG3QPrV\\\\\\\/MKuo0G5Na+T65UFoIHlA089zWmcUmeYmpdTE3giLQGYuCjQ\\\\\\\/ykHqggAugo1DirSvlWLgpvDJ3hqjIVN5yXKBQfvr5xJ4KtJsib0yr89tSeP1sekK+ltyBL+U51iZULYbefDaWbIJu8pLvc2CcoBtJ9v1QZkP6L7Z86WZzaNsyzdyFdZoKr9rrOcA2y5GUQGKEgMc592Dk3rP3JqSQd+ZU7WysWGxsZ5QUIwOPiIsHmRGASzg8mNeGZsZMieyy4DKdRCFY\\\\\\\/M0j+qRfuLY23f0IJwrToY+6azO3TBOFZBh3M1ab6TsPywBqeN5Oue5McYY0QXcT\\\\\\\/KdvcTKzj9U3EHgHjC8+Y4iaX6q24W4bJqf0RnqZjCRO\\\\\\\/J4bfeQhXIGiO6tmdGAc1ZpDL7X9J6GTTsCX\\\\\\\/ApXqTmdLw8a+4wO7YCkYvjWOzyWMw0+zWRO6Cqg7dxhzy4ZoY5k\\\\\\\/y2kf4K\\\\\\\/LURFoN55SBxjxbWdm2NnlWtt2+69kNvr28WIi7iwZK9rzV+nqD0rDY2gF3MEFYYw8WnVv7mqrjprlFgpCHX\\\\\\\/2n6+5Gn3ZVB+Jx3DS93hqlv6K1VdGrH2Nvm4hdo3COl2aIA8yl6CKLjRvsvuzlYCN1Uu6pdAnQ8onqvKd8BQEKR1uOpgas1W\\\\\\\/H40Q4Ocwwhm7WowUUsob00st+pqW2tlAd\\\\\\\/SDqdHk\\\\\\\/95C\\\\\\\/BzUUThkcWgd\\\\\\\/cIOA\\\\\\\/Nk4RXYUSci91AyisDOF2mwY\\\\\\\/1oVKu82n5CWaKPztubGmaz\\\\\\\/8kXbwJvt3GRAnHQuNy47W9WteSGnMKu0cxQJ6Avu+6B+Twrxhnv9FM805jfBxRtYWu6i3GFFkY0IYAFmhuRzIRV7FlFmHCSf+fc6nWrlmCGS45iTqDCw1G44sZKUpfMp70KmKdDVCwBnhhRUy+PBrOeyCx04ptQuwVE1GawOSqmFj9G19lX4\\\\\\\/eEhi3g\\\\\\\/neEr9aACmUOmxth+D9wK8UPDf7YU8kH+j2jSWG\\\\\\\/pbEcqanGnGahmkAyLHiB4XdaWR0Ip2+jb3i36fA1mv\\\\\\\/8OZ3yJkxFsSmRFmy+vE5+6zDqPI9EOjtSAYbUGzWJNJR9XRu6xqVG4ojEsQ0w\\\\\\\/VJ5iMzoNtXMROY+3igZy7RRlndAxno7QyGmGyP3zOWtoQe0k2Sk2flmCYcWnx2MRvgus2YsB4b0ICHvUuPYQFcegFHVbbamqvIUbhkCdWod3PxiDRZD0DkYnyDnr9OOYy92BBayIZq9AKAledCKYTVqOLjZgGTs14CgjktEIgpelk\\\\\\\/cxn1zgfEqguSs5rB36EI55h2ArA6T+GqZ\\\\\\\/hvj2+V3FMChh7jBtZU5fffH5jc1r7J92XXu30AhTuMKu\\\\\\\/znvfDC\\\\\\\/m0hz8pO9KnY4XuwHGEt31\\\\\\\/Y5Gi3TfIlS0f0n89gcQ3vWiLUSWljgH3aQ0gdWOJgvGSpP7S37Wwt0YOSYwNhZ5InNWkmVg3Ltud1qbz6Z2tZZyGWwmwrmL3TsYssriWYaY497rTgEXec\\\\\\\/inauSvQuq8mFs7RE51zMx9v4zWaLzrqV4b5K\\\\\\\/lKGwpGZKYqchswqqlB9yGmEcDpkpxxqsRPmzyst\\\\\\\/HESuzGBqStjpChdmVbI3OE5GYIB9fLJhZp43\\\\\\\/C94lxqZNRtCmWLjXXye+bnK6rURJjtue4gTpvfS\\\\\\\/PU\\\\\\\/XRjk7ZSJWFJNRNtezpszwbyy2fv5qlr7yFTFreVphb73S2vQ7jvmJ\\\\\\\/rT+jvynHqbTYNl9e6wGcC\\\\\\\/ZmYhARsLsFrYx4rYdXxrPvDvDoGZMGapeq7rQqIudWu63pig4z1U9KOpm4tkfWw2xYYorDBsBdsijBfX74qCDaxwvuWttDZ3mVsP8SJKGzoqZXu+YRnJxoMP8Dns+oW7VMJHzYygxHAOS2rOnWxoVuBH\\\\\\\/SF4Pi7Ji\\\\\\\/Sp+fQxU1csPwZcD0KO5TNbPpJqo+XVs9FzHqpBOrUH\\\\\\\/L91By\\\\\\\/9uJ1CKqAYkaqxZdONBH0cKZzE2tHfbpY3OZUC29UHdmWAo+6tGoIjyF88YhOHML3Jp8eh9f4+DPKQphC3qdTp0eFWH1vxSI+t3zQ6Z5yfIjbZGmmlJ4kedoT6BkJVl6ZRPOP0VWAJ8aJzgedUSM7eQjkE6gBY5iz8rROv3C\\\\\\\/CaYYqKWdhJ3L3eoXmEq6Ctav\\\\\\\/EkZpDWi68nCHBGbPauymq7x\\\\\\\/50aICbA9HY\\\\\\\/HykcsfQMBFWuzQG\\\\\\\/zNCJS+taoaHqCy+V7Fc7AXrE36slYdaYWtTdUKCpZdbDzF0bRwvD4JLmyngKNSjgKAcp\\\\\\\/4pKjDOlA1JLEPg1UCnJnNJYRKKPmKps6iqov2HB7TV8NI+Ujf4J0ajV1Th9Sy8c+gLRYTtbKxdZbCEssEWGBnYn2bYVZLQ7q78PXNyvgxwewsvfEIJxmfeuqA7R4ptZdS8HjLKwJ46cfGoVo4YlkpTanrqvsDurHWSEQXcFDVdRytKqECeVEB8olS98eIsyvY+hKzMvQb6sj1qhfYu13uAczOGa1aVRUL9kjJKpbusw5mQS9\\\\\\\/B1zRMaqIzo3DyYMSVdBjnqqjwrr20iHLj1DtCra5WuluFE788iWrWZ7EIrogNfUA2P4LCrYKsfcbRsBAxPi3wKH9fMShr+hq6C26GCziWJGup1UBGpuCcYrnmNwl9iXxn1UljjAnYH403+bHmSdiK08PfKGs7Af8F1XU4WGVpA474yI2BDCuBEy189ebSENZZ5itA6QxH+kuNPcC5xMp3hUtN8EDDKd0NwTqm0ZMo934hOP\\\\\\\/2NicwVQ9Baf2Ljp2Vc4zRQIUjkomQfFAaSMN21b9XKl7ToTAVVJvH3eUyUC51uZ\\\\\\\/HKQJHIMnauFabZkklYZWtAfo\\\\\\\/tKEd733GPkxLYN0qr0nxXMI66dAkwsLBtCMsBKOb9MsTb+nDGGlTrl7xcH6Sq0WHmsAwxKPJI6Tloiyxk1Bluwkkfu2dvuoenSchMS\\\\\\\/aMigkI3sAFkhq70cKNSys5qYZS3oaJ+CONjjc76E+5E7C\\\\\\\/6UrVpKNbDoWPI\\\\\\\/sHAdFak2M8KGczfNDxcJlCcPPr4wST6502GKyowav2oXBN44pYF6EX\\\\\\\/lNPx1eXYfSrMli3X95dEVYODLKjCoen3+NSjF8h6oLNUtSm2it9vdKPQtgbsin0qeH8ld1tWQpNMSamsWHuI6uL0EC5IREBAJ1uoBAznXb\\\\\\\/fjJHVOzcj2t9o7T15mZQXp8qGp7aZSU8lL1XxkIy19SVR1JvfUM1xx68EG0UOwpXBCGRg4fnSb7S9x8zbb+XikHup+ndIHh9mMVn6W5ECyLh7tRJQNB3aVK2cRGcFVkUWVFLc0MAcP7j193\\\\\\\/zLdvxbuByk0zZVKFvNo11kX3sYMUWrp\\\\\\\/lrdUSyxEtrRvpeQdcnhd4l0a4AGrgkCGrwLnnSf6BALZQyt72xloxcXax+\\\\\\\/hPnbxPK1o23EOmMqaexliC24JqnXAS6Gng5t+aPC+N\\\\\\\/SOwkrn5YLEOjJnQhOOVcWOctPBcFvSo2RcRPn8tto2k\\\\\\\/DFDYk7vucNpPJJcrt9ainpd0mpPab9vWApGQ1bhhQ98Sdq2SV53dDOhCrQstL0QzeZTSnNYXCIJRkpkDm5T0p99\\\\\\\/BzSdV92kCFC2dvOJCpHIa\\\\\\\/SNYEbi8rlMfIYSlg65AxGOXbK1WGi0Y380MAos1V4IkK0H3FCZmOtR6OSvF2syCfygbJhTFFSVdGD54fIJGplxo5sKeVjAHXvAv+dAPozKEHTucS+aqlRGHiEUqYfJ1EbB7YSiDU6gyHPQWUhXYDkksSchoRxP6pfwLW0Y3PQupuNAE6mjc1icChVjj2ZzgiF3rHL5F6LHPRrwcLxgjMFnm0p2Iq8Z5enaK9Z\\\\\\\/lBhUZBkbfQ+zeztJXogNNXXZinFdNs688Keb4Iveljc9Cmf1oBDeNU9nQZJHcz7KTkWxM2D3DbF0xE+PN3gCKXjxEgBLxKlHexdwRG4yXKL+k6miDN084hnWp+48+qVjBhz6oPZkxiqJ+SB3HDATDydDBEdR1dp9dkUoiFAEX1nuB7YzYEv3HmF9XrTwEtX4QWS2rRDFK+d15WWmDDT1UGC0w5fJpyvXpa+SnYoeeEmG4EUI2kUDb7eR7MR6Zd1eep+ds\\\\\\\/Ku0hiIXs3C9Fxrq6PAyPjkKLjkLkEiN\\\\\\\/aWQPukY6zTn9ijih68wbdMNx\\\\\\\/WqTMTCRm\\\\\\\/koqdwiPVElLBKr\\\\\\\/r8jm5bb6vqe+0rC46jysvNt\\\\\\\/zOjNQHI5Rle1ikTImAljAZFETDKvpfKxaRmXOVZzvTgIsX2H8aDg0466WXrtfIBCcoJxLhEuysCn+yVmg3UdM\\\\\\\/sgeqcDNK1PUI8YGD1avah7\\\\\\\/upSxfH6Zbm7+oErVJzxDZ7Jj3YSDVVZhpIaycxnSNAqFb06KgTCs+MBL+IwvqomQzJBuUi63n8zhCuCLI7DSJh7frVRQ\\\\\\\/d\\\\\\\/yd6N+kV5G7DmSfiHUbbPmnSkw2rDnaoaZW0CSSjnfqR1eqgFGee4R+qLx5xBPqbROpma1Q1vvGQGaktI0tuT5YfXR2Xkdh5JWIYfkA2xh+WWj4P5rwQMZMOM6lp0qd1WnLhtYyxe2dubYqF6Da263E3qF0JkJXg9ySemwIRPo6kdSHYrjCTlkt1jsAv5Mgqcb0WwQJB71XZf3ZPNoqHnnWJpgzAJCaOOtTwuG+tssqegIy6rHCENzHbajGYsOgMB7dZlfYw9s6XSwnUVY1zyDvG8tYXyUI2rMxyvvw5+ctYJdlYbDC0jGs3mlksilU\\\\\\\/hUtn0gpJYBh3+OYe2A7E67xUtNKcEyEj0U4sE\\\\\\\/Nit\\\\\\\/pbg2fr443+5B3a\\\\\\\/BK6+ZXVEC1zeZFAwwsFQwLeTWMoRiU+9XMZDeutPbXeO4O8HMksiN8\\\\\\\/m9DGBEbvxZ6ioOUZN3yDmTIfTkTPtpkNXjW1tHY\\\\\\\/1JW4SwsSwDHXlmznqRODc0B0SP6D5x1czMvwD+zOxhCY64iHrGPxpO7ga+ouOPof1Vs5YgqU4Q6s73qILpSDjMQ+UcEE9K9AG0j9+yvp44KnpITYuFrT2Vd7dBYkP2DUWD4+\\\\\\\/PdniI8vNm9KjIlT0v9is9Uv6mMXoq1pTGI5tBt\\\\\\\/lG3Xv2MDiueHxNGVGtFnMJUTwYhuDs\\\\\\\/TQ6\\\\\\\/lurApoA9qfGikO5DU9g6Ld2Vv4oPd\\\\\\\/4df5toPlQyB8g8Yxh3ifoNE08rDR2dhofuuD36HPrLQPPIdMuQggtZleQpifl5hMHAbg6EHbeqFRCgg5Tti5f8gr45lmrtbm2Lt2B+kbB0kDUpc9VXuRXaPd3W7l5iwlsBLhYYKiWo6XNApoJKJuy+4oc\\\\\\\/qNpGfh6H+acDjJHL33DUf7U4E0cPd8fWzjlBlfTvsLmmi9z5cobK1bQ\\\\\\\/QLRYbR0mOyGFGV++fJSo2jDN9d8UH5iJqhRTUeTyp\\\\\\\/y1zNxrJhQJqgaMcT3b5HpHUoQYDu9f5iq3D5sHNhDJoBvN+HimLC6iAvjfoCQ8MQ+2lzcTyWrjbZzzHKbpuUr1U+Anc4az9\\\\\\\/4R9j\\\\\\\/0KK8RtW+N6KepFJzgnuKu0Ai0pswUNsQvQfgxWEJTfkLpmJTvKkpNudZ2NoSZhd8A04YKvyOy\\\\\\\/ke+tnfWUCu7ZqIQ5Rmthu+ECXTPa4o2EqPNK0KP6cG+yaoRbUeGrOxVwZzrFZND7pcrtv9qeG88JGoLxNn+0dJStS44RT7EMLNHVR8K2x1QmKw2\\\\\\\/PutUWTXmBhGqjQiu\\\\\\\/H23LabLj6zL5I9cVBRMI5mk5Fh7L0AEeHuK7\\\\\\\/Ti4SMzbA1iZnFqwJeIH+d8hj0LLRU+bGyOPB8m6PgSYec811o3HbdnwBiYkG5RqKOyHA1EWVObc4QBI0TIdTwZFfjvfshSsLz1snckniPjfBUNUNGFGTJc9IY\\\\\\\/X9gSGNTYVvP22t8TQmYMuVffDKSdwsOCE7cUeJwweBZHP0xytNeholoOV+W3R35wx2\\\\\\\/oP5ZAD4VjfkcthhvxhH5PeItfV1JwdtH831rLwzvrP\\\\\\\/vbdpXZeom\\\\\\\/nxHviYWIsKBVLYs8EqKZUNWCoRaiVxqcZrBHW8ENu+QHI3AX4uw2Ee0D6lFT1iM9OJX1detN62NUFXN\\\\\\\/gtCV7o9NKhwsjTfx5Glud8Nf6Vnn\\\\\\\/bp4o02z7iVApDozVT3Xj7AqAI5UK3cdyBOzuVJN\\\\\\\/q1pIrf1Zm14V5ms90plhCwn0ri6QrrXbJwgAxQgKCCO6FPcEjFCXbCHbtwVsVD312M5BJBOldbNknuuoJs8gfhz\\\\\\\/Ih99VeQixnxF0Sb3oQDw5q0wgZj4W1R37\\\\\\\/TrVNKebg5zYm5SKntcrhugJa2tOrF+VygkFAhndQhO4OoWjTln+Bmgm7HnEesySNKScRSokjCh+6ACVj5lNbL5Eeeruf9xEH10bx4FeNoLvF\\\\\\\/35ra\\\\\\\/iH7dsjVJbai5R2ZttlSePELMneH3f8x\\\\\\\/w1JFNa+0To9MX\\\\\\\/\\\\\\\/tnbO6VT4rISQU+KqRYJlGHZSlrAP3gwwQBKOMUdBJk2ZNjx8obiyVlVpiuC+W7L5V7BpnREBRPntsZ2g\\\\\\\/xpe0Cj8uFrB6A1lPvHlVgNCaKKWigd+8P7qVg+fII0x8Vwahiq7zLGCPSBxjHqtHDDLnjC5L8xZN0YgWAUiqEvgB7zyWkiYHSGUMCCdHWyLAmc7r2dF4+E+Cp1roxkReY58wr7AdCvvR1Zv\\\\\\\/hgvC9UN95NwIqxlp2MOVzGo5x9v6JVWkmJWdiOhxEMGEJInK2MZe9EE3CQgU+F7gVEk6lzV+B3UaYHFKRn57TzrUQQcirAtxk9npQglOdCIlDGMeRfW+5P4f8+a5qwflLWBruoRt4FqLVE0k\\\\\\\/6ADfOk2xXKQmhhoimh2ItUt\\\\\\\/NwCa6WoAv9wYsMpIg4QD1xijm8SQBsZYhFn+9DekxHDmIls9vkX8l5MKuxTNrVANbofyoycd97Y80xVPQK0Z9aeXMeCdmN+0bs\\\\\\\/mFmLu1MRbwds1JUvt+AY2TAl7xYq637s+wgdzufqTR20Xg9M6KG+66JKXp6e84OIt2fh9CGFPjQtIctW0xg\\\\\\\/nr1pWuRI7f7aXKXAFA5ZSxIySafyfPbAecKd+ZOYJZgZkyFgU5GvZSVEBw2qJv7nKusK90aGIY+NtWsnYyQ6cS2VxH6KOnuf4bopnDt8ixn1pmciXz+e1MOlCIPeR2pDFpJvEDkbqcZAbpcKFbyJQjnU8cEeanvnfdW1tkjJXlSHPsFLm1LnpvnpH5kt1gpQjpYxhZovRY3dIDceCJGMhuNACI3BSmERskvH4LIUctXxrMSn05jsa1hM2b2gPf2UJRJ398VP+eKWdj71OPdZmUaGG3E0kd0KKpyw8DwejMV02ubJ+A8ltgu3POHwcWP9pGu4w8xrfHtb0G8k5VvmDo8So4sR\\\\\\\/OzMAAPdXHI9\\\\\\\/wBNwGAGt5b3e1lEt2DOZC26PLdivg4voQOGOhpwOe+Lm1lRnAkxQroLyyKIR+EcEw1b0oDp+qGl\\\\\\\/uHmmDcG2wtoP+dzcme6po9u5HDIgK\\\\\\\/mSVwJteapFvpLfRyGp92hivr8xmwVO93\\\\\\\/06yvYj0pg3Xm17iOTjhHM33Ah4FPgfDFqhtZWfmzGwXYVp4KgGPZXGAjLLTw\\\\\\\/HDeKws+WmRChwXGmIe5ZxiIT3qAdCno18cx9wKHfZ89TOeAXgv1rfZ\\\\\\\/0\\\\\\\/Ag6PKAUphJsU0UDVfpexGnj1ZYfdJ9gsqka6zKe8jm2fLvRuWh0Ry1Z+8GzTSJBA0M7kYSxsYixTV4MBfkHV0C700ksodjBwSFZ2lJixpuWrPrFSPaMLjGhuviEl6Ftv5YXAFJpTgTys8aats2Lg3jdbQa8GSp+KpyBjsS9WkYkN9wWtBhK\\\\\\\/Zya5D\\\\\\\/LLtM92HRtpbY6\\\\\\\/pnFNEvi7FmjciGia3MdRHKyD4j2pgtooeuVIJbMDRJfQaDifez+Hhm2OUga60OPCJxp1f\\\\\\\/qoVCPXUs0dZDBSGZsIoDgGG53sR2c1PwAYLbDqu09D3+AjS8zCrq8AnLCan6cxPMFLQEMFcd7EkUgPW10CV11oRd+FQOJCnXn93zW13V5TWIF2kxOxwzmFTqzv3+7Bo3dbz7P0ae78s1a+msoSSEIE2pKOfm30IlQdugagoHDLiURiwlkErFypGY6BRKkT0ezVurpuDlCFF2fyHN7Ke53E+HwWHEtHKUWbr8n6P8y3gl9JFj3No69VYBMXaoC2coe+UbZ+SIZWI1\\\\\\\/nCYjiEeppjI6r9LNktOjFBp\\\\\\\/zKCqo7NJL4HWaJ6RxCqah3uOLOFR7tyP5RdY4Cns7KjRL\\\\\\\/ia0AivGA3N4lPMKWCyjqkpH20fXZjS0CXqks\\\\\\\/AEwRltPJuSIRK22UUQ6\\\\\\\/55iiWmA2B69hTkgZ9V5TRXD14Bj\\\\\\\/8gMtHy1jB4AUhxs2y7A8w58uzFn3NoQ\\\\\\\/YfO3SgBqDWRSSeQc5YV5UBYemw3+pGdBdUUA20v+XPaAmQrd83mQ2rikvJGZ2ey28RkT9sIpyCqXBgEosIao\\\\\\\/ZljBK3\\\\\\\/3Ub44y3IJS+mdUVnJZKlxh4dWStMVqU\\\\\\\/Uu4fSgQOZ8TPB7vcS9HVxa\\\\\\\/v3a3TgbRYjNVB2wyl0l7DHfIjgCaf9y1kxHxKx1paXkm7cYQKjoZ6OXPDfLgZppo5vS\\\\\\\/Nf5qcMm2BrUZ8+et\\\\\\\/xh\\\\\\\/b0WrCGNPbnqfpFOHwmad5yU2XoQ5SEw1frlc22N7FhXb6dZrGQUyI1rVcZ9Bk9gkP0Dfu9V5yJAM4G7m\\\\\\\/VxW1cqB1CLnICljxxZUxKZX\\\\\\\/2NCODlrr4rEqMfI6n7Qaa4bQFBqaYZAMAYXHSKpTkHgCE2v4TQCXVNfXwiQ1HJNd+2eRJwHaKGal7j\\\\\\\/chqta5s31flo7UI9dFa4ZyoHkeKmobsmQr+H6XLDXu1oFLOkAfNU7KWvfCLGfF0sERPk33qTo592R4iVTriqTulutWAHvp78BzMVCqaXCPseqBJNTx7sdt7hi79onlS92zp\\\\\\\/MMrITMljCYOqtpJN5pEW90wa4RI0QFHgcc7wUkZSX9oYYs5NM5w8FbaIA\\\\\\\/Gjyti6qx5+ejSjFy+AmEvNE2CGVEUHfmcdum1dfmuNkfnbM3VRR8IlXgbMHtHtznv9P9ZelL5m6HngwqkEhkYJenmxxI+gwaUekABe1+n8FQRUBZzyu9rPZlw9AeUhXXcspvfQ8eXo3ZrE9P5PRDLLPLCOqFuuT\\\\\\\/gmxwngr8mtiA36eJDSV2F34gbSSTN6Nxoff3u6BTyI1BxyTvChVPTAhy+mV9quseU\\\\\\\/ocRsuYAGCm53dNelc\\\\\\\/JD9ZjR4d5O7alHIGkUL1eLD+7Ep85POGPRJi6P3cwVm596Xn1pLo3j9wjd\\\\\\\/uzR\\\\\\\/2i5GlEijojiINlG0KVc+Gq+sPdztnvZmKmeeYCRXpSuNZcvaGy4F0cLceJsqB+PahPF4eXg6SJ+mjp5T7KW0h1c\\\\\\\/tb8B08hQIOD8KLeaglYHArhu7FYcmKQB2TKJ+1oS8FGkrNZf6mNF+RTz0bVRYBuRqqLVQClCNBsUUvvtWlT7kew\\\\\\\/7ZlF\\\\\\\/b0cxO\\\\\\\/32DvrzcnCdv+oij\\\\\\\/+RKD7bG+7aYYWSAO8tkfB2mTgRBjdbDFKswGyIV7HxX2P8F7ziyhYTJOW6BDnKOrhcGeyLqM2YOrexxsOERAngkHoNTKUBMOhuXJMEVBZ9KMvgJ2LxB6ij0T4VDAM3Qjag6ngHMe2w1efMI57jpz0ASGCIACnWSDLETQmshrGDMiR42S6SiUQPBfv2BE\\\\\\\/YfewbGBjlovhzU1pZYWvr6QLzaAP1hjjTwpSwl1wG3Ft1wR\\\\\\\/6QHgKjkc1O9V8LRYmfrdHXu6Uhsm2MJaOAMQPxd6BN0FT0j2T6YV5dYrKXk5BxysxURezg5zrlcYsHUp10l13q0gxd1N6aMu90OqUHhYahS5E6IMYuukZIqch9a6h9CgM4T0iGHBtmpSajT\\\\\\\/aOowdyabebn0RsU\\\\\\\/ywzUIpqr9lkvyUE\\\\\\\/qjAjnC0oJbYIvN2MO3usQ0P6IdYz0tLmOqh20zqd6xQ73TksJykLSmXwtbax+fGue7SJJ4UsEn4UO7ba6usO3pKjidoydgtLJt1Mjx0fTlkLaPfrCDLGB0YKyoKe1+AATO7jom0\\\\\\\/cawXi0HaHmyKiO+dr4KRqx2XNxtjBBLB6c2HdR42hlrAY2oPqt\\\\\\\/BJ9Vw9KE0XvudNrNffmvvEmnUbBbFg1cpYoY4VuDVbHX\\\\\\\/0xf8bXRvTfPoEz6nMcOrRlOmArIm4KVXXbvDdvDM8T+hCmSfcJDWXRZD3nDh8I46WynReF7Qc0QPlOJXpipl7y2J+2mkeZj0gAqgQMhKlo837vz8t139k6+nMljAqdOUP2loL8tUSwztLo8LZXCwr4QuTUZpiiQt6\\\\\\\/KL741EV\\\\\\\/byZB\\\\\\\/zBdyLVb3WsX906SG6GPiHtir+dAqxQkZKzfNioCxyJuwlJk92tLLhiSaRFiBsAq+gQhMaokxvMJ0oirAjY4NzDxacYNlazA52i4FyhdE0jAwYUXSfX3npGfM05XyQ+go5KThbH38nSidfxOwDMYkgsSmnPZSsGOAIb\\\\\\\/pxWGnhj6BmDoYu1EzUmrfJfKbZ\\\\\\\/zERb72nEOPhnbRUAPsGJjzvk8eKbb0uz+WcirLLB6QJvFvBJDRvmMBehgLyhAX+4WjuPasKW5aA0ij9xgpTlixRZAm68MyMpiPdkLg2S\\\\\\\/mfxBj7GX8sDibRdEU9Ot6rX8bCvJCR0HqiX3JbW2Ezclx7s0BRYQYoG8bspgyl2ThS5Y84rMKk3oeHbT+Wh4rfhRQ9NklkHGABqvkznFbVqEDX8bD\\\\\\\/S1Aggw0mg218DqILSYexPzO2BrkohvTkR1599EcZ8F1AVMJIEid+InrHkFH1wiZARBgF94ZAc0bBEiV1uYqxr\\\\\\\/SdA60J45icMh9gLpf7Q7oGfljsy\\\\\\\/gUgKQuvd7KXQ2k\\\\\\\/q2B5PQc1HrlKmt11fUZRrgm3HQsBd2hEHZnYnwrlNH7QfhV0zWbdgo1Ix+YuF4fhoElsKzDNVhDdSBE0icTxcUOj+YZAtwhNS4kZot\\\\\\\/7f5YB5Egie53dgh+FGOOwO8aD4p21QollTNanj38xKyVds51HcKtAgkjGNYvp+y1+xoqUV9kuub2xajBmDlcQItoHLogd25P3Sct4Uc2D7c\\\\\\\/FS9qbr8rjQNACtgb9WlGn\\\\\\\/D2m\\\\\\\/tL1uYNEY6MpmM16QCF9nPw7IjmoX82StH+D0OYybg5HLh8pH9w7bVbYnqcyd2mhw22hjvQD\\\\\\\/x3mZGSHVwf3cnnuJDcxCoEBqdYXCqrvLHIuiihhKqlX9WV24IKQv80EPZEECEI0d4eVWrSPRm12vGvCFHUQPFOvwtf2Oo6V7ZDtmNBqnqE9\\\\\\\/WzUggPXj5KIJiFnGnbAfAp9a9FN1AdrtZJ3lPP\\\\\\\/z3Lj0LXMyYpTOeHfErFPapfr5JZ9m4dUCut7UqNmonp8b7Jn6kFSGDonzaK4LR8ew46UpF2JOWyB7CjdSG19ehodOHOYljs1wZHWXBXGb1i6R3WsW\\\\\\\/ehY9vScJwMtpVN9\\\\\\\/plrbyd69K3tBCRmTM67ZApckKWPBqvS6Yw85EbdCmiGk5G32XZRhUhlFksR7kqmDYULTfv0quj4fvw4dYT\\\\\\\/gKcugIzZD\\\\\\\/p4fRjNleDlpVMh9jpPFVjXDrKVCVBGP5no4bX8TWKB00ORizLtm5FKvQn2FSvZ4G\\\\\\\/LnFcy7GW4EsxsYzvCGeRK\\\\\\\/+v6tIl1zeKfHktEzigFeU6yKrLb4UBkzSooiqfKvPJ4nv5Xqza\\\\\\\/uXynIGdIAxa7ubjI+qaQMuBLE6yGGmwKikkyLsTktG0ePTxdEgUWtGP82uOwympJwNPy7ZJ09i4+GygtNvVzr7OktFhxQ45jMTpWkPPPX9HIlmh5GkC+LaAiaH9spfBBowc38gYsltcWPUljikZRk2fFIiN6EHFbsJqFmzIRQqTX2v6eLJ15\\\\\\\/Ta3etm6iQbdKWRLAcSs5\\\\\\\/X3z0fI\\\\\\\/KBbC+uyAF3sUBupc7mXU8QksOCJAnD7wkLCMXFSdOaDEBitW1WaDnw2tSngpLBYdpqw6lTfNSc9A6tLUr6ajF+8KF2LMiTTE6\\\\\\\/6LEb14EO3f1ifJj0vPJuDRg7JuwhKMEs7OncOTU7gAd0WFoPjEba41qI8Zp7azKlPmNOJTWtBxnA3WodZF7ZrWZFCMfsWz0ZCZsfIQrBk7UiJnegFx3kXwXxS3BmCzUdd2xG4XsR28qyafXzWxv0t+2MavfhjofcLyjMr\\\\\\\/e32TrfgRTXVOxGVsgccK9H1rlEyZriCKp1oCzq3xQSUFmAzHJXOsCaYNwGd8jOIohdiLaRWwCQFDa4IThMweVn8syhG1abYKBSu1uBZ021GxC6AjJe444zRO5v76yEa9zY4FAlIUZq\\\\\\\/doLH9\\\\\\\/jpdZpw9+UyKOFqP\\\\\\\/7tG3xnBy3P9w7ogo2qLAThOzskv0oJnufumdbyevb5apzfvK0OlJVO9IBjQtNIORVTd8wnsQtW5T6zcnsckvRF1UvTTgbD5UGYhk7Pcv9d8TQyxbwkUOMWBovn4uDScYqLDxhQDICrS2U2VSbYS8W7ONtHdv6rVV1XoSRxfWIjXUG1wDp7Zu4YiIWetm6UBfJ\\\\\\\/0ftk5o0YLU2a+fCPaz0gAXI2FNoS6hMpDaS5ettRsP4CnSnV9w\\\\\\\/yL78N1MvMWsp\\\\\\\/teOcb28HnXQ+rFKmeva6xOFGQ8ZE1zSDB7iP1ldMmYRksP3uZVCTV1AcrW4YODBFIksjGXZ0Rao1kzOh0JPTY13ZrZSDWqKtxyXSmy9g5WpouCS5srj9OT3u22kMBqE+66VBB180sNvO4Md9ATOypTjM\\\\\\\/uUOga7Hp7m9aSTExgxQGX2NbVGIsbVfSrKYDg5sYovrcN3AZBJp49leqwpff+k963usAWvmkjq6CZgi19BpBKH5R7q3nN5lcFlJZUVqj0zGIsxdCD0+FCNMVXBDKHDb3hG+hmpeDbwwrfUTlArpF7iA27n1tYarrXP997wthQRRK2BpXlqaGe\\\\\\\/73RRHKOBPuH86OZdxMzOH8FSkb5kXWE6nCK9hR6l3Ppqn5vI0Np3EU8\\\\\\\/AczQFHZSTEYg\\\\\\\/IdLGoDi0tT2Wy7QPgLsdt3yhCGMyG+uJ4yv8IJKcJJ0WT0d0wxKczcFAS8mTrfSbznXRfyd42EHdZaVhgtr5klyvu4VZT1IZ5JgLb1tOmn0fIH1g4G1BLYdh99KdVIKrgu8NJN50SgT4cwjj3zgPk\\\\\\\/35iga6ldKPSfsEHnLjZz+zg5kNhmgJnGgCMQFMRIG5NWypQCmvcTzxGxi9Fhe1aK2Pr\\\\\\\/+\\\\\\\/3F0+W9qb46gyZQdVTemj+KYxcSMmg1cd2obz9es1LpP7o82QrNujeSnOaJN7\\\\\\\/1R1MfqAf6yOCEAeEbsxz17OrPIkpyAzQlBMaW9OYxZ8j6jjO+\\\\\\\/pXA5Z2\\\\\\\/wiXYoKVDKUSnSHLrwz8NUaPHIse0gVCdLEJRtH8UFJkPNGy5mpwmFeVpcM6rBP6BnTLD+rLPoCnN6qL1VL4\\\\\\\/YW0u2cew6fZL3wu\\\\\\\/jKfMEcUpJAa8tsbo\\\\\\\/OXeS5gQITQEzpEEBSCaB8y0Y6+1uHP4Spu8FD9xW5cRw3o6ZiVtyep2ak7Y8LqrmAMooHCgD9dngX6b5u+FdLwFT\\\\\\\/HiIwT6laCGw1aUcSN+N7BLtsaOZ8oF\\\\\\\/CE4RHk8YjJ+bTRMVQMjFXczkRknt1uBM7goZGYtETKeVaEzFHmMsai75Q9WQOsgy+aryYmB4jBc8vZk9BhV1ca0Z7AM0f7iT309pbLvvyBaF4a3sjmH7eH8glNG1al07Ex0WttKNjeAazM\\\\\\\/M70KieeNUouWVcpr3vQzxUDyHfwUgautOMqhHmdrj8qABr3tQsKqUtY+DMxryX+eQR1ZbkEGANUiqmw+Qs7ecpsso6Uu5M5HClrodbPdmXUYZLuyQq1KoYSDE28TyV0lY3tsXeG+Q8eUCZQBRfY2AFvu0QP2l3oiGHpXMve7vrJnf2YkxN82X2tTj9B5f3yd1glWrBrzugwVlgaWMRHW6tWQb1X99pxNBiZj1f65\\\\\\\/YphtPO+uCKv\\\\\\\/qVHwmEDckXGAgCEpoq3quAGzVqCjT5IuqEpCeiVcjhERYvG\\\\\\\/dETypkofGA2zzSIO20NK5PFnvt9\\\\\\\/dluzAkyU4rd2Y7KjAWMOOPXq6cpJL\\\\\\\/2n3HWymX0Kb0FOuHfbp3uNyYlN85RTQAICeWce6c4OBanCR7i84P0FzCmbtcmV9Zw9GhdTumDHKar7IGspUS0Dffc\\\\\\\/6pyzZQWlIkjK24ZwuKM13uUYB\\\\\\\/sBDub94j5oZnkTLjfLTid1UMzLY5hoFiHf0RiZ199\\\\\\\/F1ZENX7cHVqpxOBnHIEFi2QTGCRuGH9SFdXQH1LdMl+MJUWcpotIxJRkybrxK5YCeX9Uxvw1a+N6mRPmRCYgsdvmHAMgWrNxmnKQgenbBL0n+xXgsYGdYIhaps+UKlVymfUJKjzlRizc8jFNpCBVdsufYXzqcLyjKhNu2PcFXl\\\\\\\/d1325LG\\\\\\\/wTw6yv0jJ+eCmDu5aNCcNzEASGA3r1Ee1KTCK1NeV363PXfR7+ML9E6mRQ2m32oA4YeiZvOI+QgX2M9CmWdsJ4w8RjHHy0pn4tZgukmhFlViLF5u+0Xsp7gVu5vIyK4OSFGIRWxwZnI0DnVgkhdTX5Znte1\\\\\\\/znbURZQ3jerYJzA4HOIIASbtPqgIoXVfRsuXk0JxAaHZWEdtXudqLWrTnzx5TRj8nI\\\\\\\/nIXBDJI3G0oUAPn7ATQI55k9phSW4w11Z7xqJU8XOGGPHulpsZul91AxfX7lmSgHjSIgw\\\\\\\/o9Vov07AHovlpGvgRW1mv5KDF2AwSch9ki2nQQCEV5IxilHIcMqdcttNjguhe8F6cO674GKvmgGCby41j9qeVNOZx9N9zyyH\\\\\\\/JGl3C47gK1x75NDj3zhGTw0dL5vA\\\\\\\/YfbsMrM9wH\\\\\\\/liA5c+rLUStUkUOsj30Gwb5RIYlvqFos7lWTl4rH1lLZzQkbqukrg2tjTGEBHSuGsHYaUuNpJ08f2lt1jEd68iQCGtWMLX8zDvVz3tSIU9uBhHfKgH3t1ANrk\\\\\\\/2ebpyROfSHVzG5n89Rs\\\\\\\/R4eReKrfQL1YyUKSEw0DyC1W41K7pf7pkMcelU\\\\\\\/CBZZPBXTvMR15f7bhFb1Qtl70EHXTfpVscfb3DfzMqKXLYMN9LOdo9qSudclnVMj\\\\\\\/R4SC7Cm8VeKa9yCcKzX3FCcoskhC2\\\\\\\/69yG4C2um8BSYDrKO7JvzL4bwpMRMoPqKusemZj6pDR6oIzFT2I2nv7MTKUxQjq54VQeBYDXK\\\\\\\/kTCuXnfIzi4glYVUGN\\\\\\\/6IG5Y65HR6LmUM57Ms1QV3VC7VH1iuxM4C\\\\\\\/jQzbUiEDt+41k0c9VU\\\\\\\/SsCrJmPYPmNS15ojthMudzCMUKU7xYVoB4gpK\\\\\\\/4ZO+hPZYrgc2ANDrWKj4GscuscwmM68+\\\\\\\/\\\\\\\/\\\\\\\/w6K3YDKyWGaslc7\\\\\\\/ePC99aPvampdvkyTUyVr+8uYVY2DbCpd2j7zBq7RKMT7jIM8eqd9GHWRNLV6hxyby7qikaZmH6gE3b1TAs4JxRifK2NcEyke9SqTX\\\\\\\/eFs5nJWohtv2dtAjTh97QSafhqoEqLRZtskDH0aAuu0D1H3YKYRA950jCtfd24Mz\\\\\\\/gx3PZ8RWtYXhygPsTP\\\\\\\/BwBqwdAiL3zAeiUFX0BEXUoFKJ8xm0uN9F+DMx\\\\\\\/0DXCJedZNnZCWKvHi6XkvOdK7yrKOiuPOhYy9MpEGkL29JI0op2cmA2v3H2mEtqr2RSDQoAot\\\\\\\/3\\\\\\\/duEg+TWGeoMRb2vm29iLMGQ9ye39qkVJ5zH9FsMHWCKGPkfDFsCfp7TOGPAjLH3AHZU3NlFCQK7ssBnI97FYw9DfMQTINWfSNjLCPawviH2WUXUL0L440N1k2bGvi8fJOkJelazhN36zpzLAKVNqky7\\\\\\\/sUPX3W3pArruzXcy1n\\\\\\\/OONbB3Z1S43XvcAIm+uJ2g3GshZb62d0sp1M5fTn6i2aalwl9UEdOsIxJyfqORjd4\\\\\\\/LNGXWkJ\\\\\\\/a8bjz9ESkDWa2k5BEKK+Km6vq5LZrVi1ZNv3fY+IP0aPzIhtVBi\\\\\\\/DKwkKZVCeaeoTXHn9hYMp3NLQCx3fkkx5ZGnjR0mVZQ5OOLqgHrPMNZEb1KNHBB9u0L4UEKHa2rZbbxbGUDzQfuQjJqByVl3x\\\\\\\/77rXqGm7P+BJfWiF9ID8MAJ+sW\\\\\\\/CDSQSudZeMfGuhl3DYU0L3YkPG39vgyAueSAaJgHGV2CYoAh16P1iIGul+0xNAuPxBbjie60Uj91jQj6o22+hsMTYAi02RhCaKc2FLMALWJAZko16w3\\\\\\\/Uh0lYobgRzPBxlgekBxUdY1dht43rcf4kux48+2FHep4UnN++ICrBa1DBNJ8Ny6X5BE++iBlpmfnssguz2\\\\\\\/S3kUTaFoB7G0KfS5qvUoG0K3gI7wfVUKV5y46A\\\\\\\/LZtBdNuqNvE\\\\\\\/ycttYl4hrmXl3H4UT9Pxt9c3c+2r6y9AgknYKx6oZO8V4Mh6bMFIV7rVzqJ5xDxJsFx7Y2x+6bkI+OgwTQLw677LG7deHh1pvMHabQKxzBmt83AsydbT5VGzU5rkBU978ptePLtfqX0T9fs7mMwQVCTtJv6WuSEWsqsvc5sj0umr1VJDB\\\\\\\/Nwtvnmgjc5DpGzytuavzah+R1xtFPPUZrypKISsP\\\\\\\/ahjk87Vpwa4jBAjOrW+I4mCdJgYdzAtzbIPNJii0+zKTush1kj+Wl8f2B+\\\\\\\/iSTbkKaCveEize4oICmN2fQjpRA9muc6ogAUp0cmwMX2qTlp\\\\\\\/WPWLPuis10wxp\\\\\\\/RDrspGtDZuqLSVrC9VKzz1tiLQyTd1JdfyOtdzLvQV9MJg5YO1RtJX0uc7HTBYUwsZXdp3PILweYTCcZJ8mivP0Sx8G4kRiZJyrbOHDR79\\\\\\\/sbeS+8kV\\\\\\\/uqpWG0FzDhL8SttMnm6WKPRUecoPzctMo9MaMcl2UdHVaJhV\\\\\\\/9SSGCYZp3nnEbo2EGmbdSqOQIZx6GaR+kyeaG2dp1jdInb\\\\\\\/ooMAwTSfjpf5n+KYGJixIwoM9AlD9RTXB7ty+sm8c6hbVX6ILBju+v1VjqoE\\\\\\\/97qahRNzZ3kfdWw\\\\\\\/68fDQdQQS4T3mX85LCmUf09rT4rGcB32HE\\\\\\\/4nlUtrRoPQo0j9eOJoevOUEBJa3tIP+9ytWlSwF41Mez0WrzSys6JywjcssoOGhqBhMYt3Aq8Yu+aWsQF5T9KjxusMA8CBzzzuppX4LnyqMI1M211zkMzRUn6oE8eWFMJpSQKKTW17jbc8Yu1sSweg\\\\\\\/hKkorRYmhN5qsCNV0KZUQeW3ZhtVQe+KIENueGfFSM94DBF8St0xNdVneri4YzQNjprQAtMrwrFfl\\\\\\\/avf2VdNhyz3+W2EmpQpDDPpZFl\\\\\\\/GpM6cFu8F1rCSDv1UzTp7ls+7a2YK2rFgOsUzV1LmElaU0TEpFA4kWWaGl82ody\\\\\\\/LeVsCijRkBJcGcD15QgZaz85h1ndV21yoZwGuzBVd2zToDeO7F8gOZ9J1u\\\\\\\/TuhPHeK4uijT5JKBu+\\\\\\\/F4AWiNqYhNapEmjfrAmqt5+sQlGWaV+Th5Z\\\\\\\/AHwheU1PBXHvx4QBXbDkEPA82\\\\\\\/xdwpxqsn1usnUA4d2TLk\\\\\\\/nudVyVvRO9XJvF0msBvv+5nqCLNLRTQOTRfxi9dRkty3xDpsG\\\\\\\/Oio+YMMIoKwtl+hir\\\\\\\/5+5lQlqoaGy7ezlV1HtdJfWnWtSK6NAkCs12ATcs2VXxjHwUecS9dfpzTBYvJ4ytbipwGK68BU2luMhT\\\\\\\/+zBTwsMCeCCvowad\\\\\\\/B963xzohEYkHfnil5Jfe+9Ln9299axMvHagVnbUUVmmZwcc0ghfA1MqCDmscvLh3LbE6h0YROx6mkzGRH8zKFkdweCJT8\\\\\\\/SvZjF3j10NCgv2sTWKQyeUqGmBy3+0usxHQ2qtEKzaKBRFUoKU4BN\\\\\\\/pEr9hbmQ4RLgSOehR+s+w5gbnvyPzKcRROzplfVWbAfxoXBy0AIKRHAIRKrM9eiFDsXoGUvHc08726MFWAhjix4kCjuBPaNtjqDwjlK1zfsXkZZXQ0IV11S\\\\\\\/9elb29NszRSxrpqLZZvc2nrpxuQ8ZyissSSM\\\\\\\/D91v2\\\\\\\/ySrGJ9L4wkdwCtsYCFqz+JtMRf4KGFkn9tJWxZKA5xdS\\\\\\\/xW+csBgWsRcGDlmE8a1oRbuozxmRp\\\\\\\/\\\\\\\/32nMv6JKegM69uYHqw3kUharKtF5UjKKYxgkRPnqG8YgZK6lmp2V0gWFibzGRt++qyvxsL4y6iX3i1zJJnINFPoLVl8taNnBVBYhpeMcWFcOdbdJqkUjdrUyb8XGTgKEbWwFsG2QiUx14aj\\\\\\\/6OrmU6M1Bwtv3K8uIi7Ngy2Csvrfo+UUFYBUfFXL3p\\\\\\\/KD6IAMAxvIpfJvvlfi0\\\\\\\/88AEcbEJ5T7e\\\\\\\/NTSxkgoyLPZl2SMqrIgZGmEJIx34giMUfLzeAGQWe2fw35o6YSgDpleWhIb1rn2rrNmNnEmuQLkFMlEAsqXcbDSO36M2DIQ0bAiG6PC48+HqgRp7j8yGoN8BmwW8zNHN6SCuK9Lw2dOEwbitY4AQbVvPQogpjyV5kH16\\\\\\\/a6D+D0FuB992RbHwuHIWZzppptdCeMn4MxXWpSFkg\\\\\\\/8gFM9E5Ob5qgJJ3L7+EkNkD6IQSbfyN7vI3M9lTldHE5ntfrkmNw1WJ47u4jRWz1o9OnaH9rPUEg1+jdN60FVl\\\\\\\/kF64WkfxqLoPGd1HWWiMktWRLBEXlftectAZaOvtBw0aPMoCjRvs37Qk2yKJ18FckAAqViqpNaZrrW57LFZ2fcaMYAINkKpxGPMPphGipzdRZ9jjJ30gbzvNKPs0BB3922qTjR+lYNgYg6EUqyTHBo15TQRjQ0QefiOkUGfQT2hleQsTBdzhBhahBwAszXd8iu2im2Py87pJlparodnU4Ta\\\\\\\/lythxPsgNvMc2y2CqxVlLPMs0MoD3IbVb7k8ODf8nl1gJwaMfhF7qNeW7M3cpe5Hw7XOKw4j+\\\\\\\/HymZP6Tj3JmKxYcWhSf2X7fN2V1CuczCFU\\\\\\\/o5VPJfx91z0R9QIhanTqI3VLFXEU0pfoD1kZ\\\\\\\/RUsI45ZVt7ebceXol2TWx312RRM4ee+xtGvA\\\\\\\/w4NoZJ9Edx\\\\\\\/p7spXOBunRRt3TT5xuBdmomXo36Vc0z\\\\\\\/w+DhdP0muHmUuyA9pH3mtqze9zB986e8+isxdf0IVbTvDSTML7qnNFUuimmBW\\\\\\\/OEKJFhI1sCp1zkfYAU0haG2ioXfb2hj8ueVO289bBmCOAXDSfImxHeUQejqYFTD38k49xRkKUVDXAcG+c4drlrQcHll2yKVQ5VRFoFy9YpB4dNm8XdGF5\\\\\\\/f4+cVc2VZRVzZWS7qWSsKogf1alxstvS+hOx0RZMjD1q1ZDkGRZwRjg9HEHMWZqeg90Je4XYd7dzSCAohs6D68DRgTeZGQZKuolfLQ+fvjA9wvZ1fIDNC2gewJdFPY7wOM6XAs8IJLcRVeaNnfVHPvR6vNBH2fhT6Rnt\\\\\\\/R61penByTvVDgrHDIwIYB8zuNpU04ZJKIBu+NTxeCVbtaN2asw12NcZE8ywnYhzZCxmUmWXwLORw9tTcxtGrfR8eI4gy7At0nrkcBNkc1jMYeFzm5aDapqOcddgJmTJCNx8rEGi3KhKM7rlJ8WbZNtSkouHfRxCkbU2I9PmT2vRFFqI\\\\\\\/TfvkoMZFt8w2gZHNIlXV73B4DaMa4t8dXMtFcsOLx+2PgVRFqMr6Qyh5QD8xcONND\\\\\\\/kgpRNABpRnreQooFYKMmIAg+UTIu4tamWSPOgXWdlpJto+DH0pN7sWYltM6Rie8MnJGOG+4TwFU1xVc0Zlzkhuz8zSrTKigomKTXGYIeCqzzxzHmjxi+4Tm+yFsmPUEAJ14mFkpQc\\\\\\\/LInr8NX\\\\\\\/V40ImtDOg3hVqpnEXWzoSi\\\\\\\/S24jtCkkAy2cuymQmYwyzOjcftCX\\\\\\\/Wq+4gm03\\\\\\\/EL8fJVx8sn+DzypRZXBX\\\\\\\/xL7QP\\\\\\\/JKEwGHVqD3xxhCzNpDX5Ty7\\\\\\\/go91pMjixKpLsBcSf0+PQlz4vG1IPiRKMujSp1ayWKRPOqC9fuzhMoIVJIkIFzOhiybK\\\\\\\/dW6fQntR1d8AuLbmfK9uOniCRSAynGKEllbiq7N2e7ek3wxzp1dnANMFvorMrIRnhlr\\\\\\\/e6ZoPL4swUQ0mqEDDinfw7obJm\\\\\\\/zU3HRiD1pBiJE2FFkMjDrbqBJiiMTSxClMJSAA7JbDVto1zKvvpXz99I1JK2ICppj3r+XRmdFmZ\\\\\\\/3F5KRIjLoJ9agC2H5JAfiP9MG2XXAtImnZ7fqHhZMbQYJ5XhFy6cvApXquVn4PUorB742zdIWANbKSVqkm2c+eHjOP\\\\\\\/XY4WXnMlb1L8v+UqLEMD\\\\\\\/tyKwXpgrvOkuIK01nbxjHeaB2orowKYDoFC+IuKRMUhnDxqX9rEbw1XMiDY1hZlJsVDHvAEjdaxeHbEOK2utZmGuQu+wNWmLpaoNsg2tCH76b0Zy8k5gC90hYOjXuXdEGfO\\\\\\\/ltfGbe1UtaKsHSAOesHWabQaZZH+o64xa7H9hTKgQ1\\\\\\\/2OkFZj924NhfOZgKZSWvq38IxpQPg+fgYPBgwsvTln\\\\\\\/06buGbNfz835ovFIoK7nM89Q8vTn+5PvP+1TZQ44dnaL4agARVS5lGQyj+A0zQHN46GmxzPClCsacBZVS8snyb+IFELxp7eue1rm8Ni+Laq1pkVvbS0cBkKlg2OGhP2relgYdYWgpqfMPXgvd4BbOwX7LKhMcOEWcioeB6fvnsS5mjELcwWIRWX24J48\\\\\\\/miRE1b\\\\\\\/+fSd87YgmtS16T0eNCygaVNQWl+n8e3ke\\\\\\\/DFvb6slIzbPm1+Br2lxU5ewjAY8rdvZ9nWkA\\\\\\\/DLfdI2r3I+4eMLBFf8NOvUOM\\\\\\\/EwzFq9E405SV+KUmd4s5lEQ69Ifj9lATvW8vrtlsQBGXzxCMBOkAq5UOCnE4IHOLcTK7k8SKkECgCSKc5RlHeFOyotJbT3HoEpeLXssVDjrx9n76acUqzokP7Y14X08VTNwwiV\\\\\\\/7QfsT0cBT9fpt1yGn\\\\\\\/PDDJAvwO00Uy2+Qx8Qd0AtWgWSj650UcyGhHgG91HWV78\\\\\\\/16mpO97ElZdv+nMEtGcZwHdn41iPui6ZWtoKJh3Dvgt6xi6AYaypUSMKUzck2vB7ZiZZZ4vjTC700EHvIRLRBQnoBJ+4PrmSCvw1b7T61wwlfUi62ygm1pZ3ah\\\\\\\/+MqBFONtJ9gTAKixK3UmdCSWcycAm9DjKVj3F4+uI4x7NZSQTFQ3\\\\\\\/EKxYe3XKC35CruUmvcxWpUf2OlE4xP47LuIUAGsWMO7fQvN\\\\\\\/aGII+\\\\\\\/x+mJOvSj0btMz742afuTguQk4QKBlu\\\\\\\/D\\\\\\\/u93uIz6v1yE8x5HZ\\\\\\\/QoZU8j+JymzYo3KLL3BWB2MJiARrrHG+03RUPLxgnis3IkvMzEzqFLvgD\\\\\\\/jg1wkycSCxf5oZDqXcwprxiH+zr4GkXrSO2VvyI\\\\\\\/IxUZOBBrwefrybchpezXCh8B\\\\\\\/ze5x3ulI9kUwI1YBBWsajDM41bE371bmU9TGg+3YmiqCI46s3lluArzptsVltNns\\\\\\\/G2xFF691LTU0fZwNE36gv5I6+ZXQtmwLXotbO7T3VFi4pXIH2FvxyW4++BYDc\\\\\\\/cBW+omdYcLjwZW1mhLd\\\\\\\/UEkY29Ok4HISdNaxd5eIPJE6Na9w1gjPUF+E6RGhTyBcR2AvNvFuAgs3eNrSXDvkjLqvUMF9haUCOKoDFyf2gcauOvPfI7DLXC\\\\\\\/EnKc703JZbLO1hLvJIgSsUwgeJVMvPykYBmhjuo8wzGEQ3yh0sFPl4sVXSbbrj3BIIf4N8+6btJm8Jlggh3iWhj94kRQ3SEMzzgYW1AyEtPV7\\\\\\\/2PCLGlMkZF+XNDfMKz9yR2euKpS5\\\\\\\/w22NnxWuyihYA3gEUHVpJscWTDWOU3jDcRZUlDXSCKK2zyvVbSkUvo3ymThwGn81wfxYfX3gw8tqmRGXlfEwPzxX8GPydknWE9QAZ3XqXMD+3lMWEWPdhJYnuUUvbr+VCRoeYm2wOb9b\\\\\\\/h\\\\\\\/9Tbo1nEKEokCr7WZDu9rrxRz8uks150GZfDsSvAN2XdeDIFmu9g6G38piIx6tjcqixYrehha6ghqzgfu1rEboDxVtRzBaMXsFujBQdpPstZMxxOktz6pCWTsfyov9jr9XIY1uyPxH3l1RITZ1jnjvBInoUqoe3600ugpEvobheMNf6HzcpUenWVnbZ66TW6QtRErMGSccZPW3BcoGFrsb36aLPxAzS10a89N8vnopj9k6LtfsTlU1pECaKfE8c7TKU\\\\\\\/96LWbTTqx2qYxijliC4eWYFlY0YRmBa8KZnjKd3Q7wbDHk4gNWP8JZ0yAbgZoUvOSB0mQbvdaNAq4+KrPe8klYvx8Og8L\\\\\\\/L4yXxbSVlc0hR6c\\\\\\\/lYpnNr41BJgnj1Wo2h7QLLrlzrgI7ZLDFdY12ARB2vFiia2QEktudqjrGX2Ng9eV71PtzE5apD2oWrcVJsZUjHxLhq3B8SI5O5EA9yl30drhhYYTTQ7tzJfhBWGy3uyPwmjR8JnYDadon56rPpX+PkGfbbClP1DTJ6L7r+nxuyJdXVXyhRfrfCZYYRc6fpRxvHo02IRb0iwbAV02Iqx\\\\\\\/wERecMqxDxUcl4I3b0kKMQcWu41bOPkbgqfR\\\\\\\/\\\\\\\/Pao5hiD\\\\\\\/pEXIenT19176sNcO8ByK5Y7fqjewgKpMWhvR1ohy6EEv5N6Pb9jfqQPdssvqv7WncJTGjmsOf5l7sNMFtw803H0TUdVh+Th7KQmFn1z3KL\\\\\\\/RjTrUSHvnScfe3AANlhKOqFA5LXhDKVV\\\\\\\/V\\\\\\\/AG\\\\\\\/UJjSvl\\\\\\\/Nrg6dO2FNtCHrrh1AbRNAwWDYbrG1f\\\\\\\/NF3o+GnS83t8+8tIdzIq4OVoCi9+f8VF2ZKnjWOg7eSHDJ5GIxrzrYjT31Qy\\\\\\\/vpm4pWtqEzKpcSsmwl5pyTyO0ViiqTJJDP1jcWfjn8MdeBCVoqUvsf+nRB0JOpmsz6ihBBx0m4QKTzHbR6JhTrdTDTAX0Zp6zt+IOSaviMzgw4gkcAKwWUlLSxJSvhPy+zvRiDCamDOnOsUdu7WZr9ozYGAg7FfrADUdJillDF8PeENNJMX5JPV\\\\\\\/8O9LWGKan6DKNM\\\\\\\/tFX4eH53z052OTeuQQPIYUAKcMf6kQ59oLstZiwW5bgdPX1Fnk67FoDsKux21b8HDRQW8q22SF55HK1Sx5vPYDncGZp6UOoHvnizsPoYKH61jYxxgqd2NJOCrK9MEEPH0s9YMnwSUqPwBkuC\\\\\\\/trYB2BKEOWDoR2kEVV493cFrc30Wh3uhB5Lvg9lJRc\\\\\\\/68k9ZdkZ5PHpJanGvyt8Me0WRNH1zVnelLYbF\\\\\\\/2U5oBK4FhX28IjO5m49OxGQcG3AdamFnxHoq4GtjGQodqvt6b\\\\\\\/JwTHPQQzMDUk9amf\\\\\\\/V1FqnsmWf4JILy\\\\\\\/HFYg94ti6WNyO1MPf6qp2bbjqeqOwSr+zQ\\\\\\\/vxiowpHPMu9yCXm4PYmTvBf6Ac3azK+SUW0Kp\\\\\\\/MoQcsbybIZzcJblIEskjftehRk1hbYvj9CczVLanmdU347wSKswsVOIn5UYUnibhwhIhbFUI00rpmiNb1ppTm5CznT6aNFb7tMA8wVi5gpEC5KINNCw3s7jC7XFdZJQpYsh8fvVGg8wo4Fs1MCI5BxdzHoS59LKjGvmghRdyJsU+XY4NqCMQtb6GCDhDZ4K1hYKwa\\\\\\\/bwdMEDh6aeD3iOXplkTsenr7VB6h6mUdqVpT8j6wUWLBA5Ipxqiwlu6gdrhbtwfpNgk6K\\\\\\\/bzMG1hXDATizOCXZA66F7aQPG07Ibb8zBIva+ng9dBXou4U5MPs0vr0FJv6ESPOEv+EYdrNwMBvhrqJuDaWPszIEpsHo3hk1hDZ3WfZ5LpReb9MMnHyKEiYsqtcSvD+ctg0Fa2VdDDUIkWWStv2s7iGxdrKiZOLYfhU97UwIsCWmUDbnQoiKtB7lSw55AH6K1GzfUdtTL3HTovkvBCBactyXDHTxC0xvaGjDqLF9NsRpGydyG5S8S8EsGoQaVopOfS2eO9pKGH2K2zWfzZZmGQowGjeAhee3qvB04QmP4xj0nR7DuaCk7T7c7GtnSpkr6ijVD8qZrc5jnHD47fEw41Au4cJm+Axnr9pQsg5zOnWvPVc9fMU5zhIZL6ZU35k0uEUu5Vq+26J0YRzY6942j2+xNLZKevRe7R7qkGxHVjSzwzBYvpGnneZ4bXbC7AYAn3VBYgv6mz2Tzi8yBx\\\\\\\/XunUU2F+AvD2ZtUbqFfDP8ltcHIJkd9UIywFe2RYWK\\\\\\\/UZvg1Qg8RloWos6ll0a4aF+WljPvUv5d31xZUcXXz\\\\\\\/gigKfUxX0k26lAZpztU9XSSgoVpI2CGlnSPM0Qgu3TEAMKU6vytgQbalPLw1Yu6tJMIqmE8Yn\\\\\\\/qbesfcdeYT\\\\\\\/fvVaKZ0wtcEl3HkBEEtpVwKcOhmozpzakFihP7ZBndiNwiti5ujgLNtrjerH6sCVFCEXchkXLAg1LA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022dQMRP4+3CooodqotpgwmMw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022968e2c56ff200554b7c69e50ea94fcf991b3dcf0693e3a295ed0ff7a93db8e82\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QVW9wu0ubI0EmRHY7+nHGwpNMsJO7Ai\\\\\\\/lVTNASypuA78u\\\\\\\/PEYLWxUSQTShfh6LOVwaiHuQcJUBd3jwSvMQPPtfQZ91gvdAsKtaiQXfi+WVD4guKiqw56VVv5NQ\\\\\\\/Eg9Pq41JyGgmEdAwpYhSVWQMS7P1GkLHUfDdzJYUeMpOtYxH6OhhIwdrqYYZjJoXJ8mPL6C1oQTBnz95hHWKwPkNJjMriiV3O2ygSqHo0uynHe1bfMMb1lZrwIJ3CFZqkgxk9gBMhMKIBp2NyI0a7yM12LQwvPQH9pWF56sQCvbco7EsZelWxAmAY8Qb3fl4wGX9mDv2G7zPxbz0cUSU1gzon842gTeBbr8gjhZSYqI5ohcbreTWtNqrc6GlfstH0\\\\\\\/5aqpWyBJqbgZLq5elUzcL3VTZCw3lDe954u8++uMNPy+OfVY9HDy63lPBV1q\\\\\\\/WkOU8bc9ZPY3Gi3IRI7ZoMnvxZUqg7rtbF8fHDKlMdpeYpzLx7feCqkeM4cAKgjeL\\\\\\\/a2RahLjwEKtkcD0fTgR+Y6DJBuBFePNEwmrHGa1K2w7l5ENqc\\\\\\\/bC5xKFFSayE0jm9SrlXpZTxgLNvxF6iBYWN9xEnL8R6jOITxGUEzCoFkio+cQbkDXnTb1F5zQ65ZHeBrZRjhqGFMaiZCKsRo+XNikpQY220l0VhookGqcyvuOHzV5gA1ucEof\\\\\\\/wQHdR4cVKnbAC1E3qcJWuHNHsdMMq\\\\\\\/jkH7e0k2jWGY4WJITR3siuSJlpHKKOesk5KxZvILfJk6JZK0EZyhOuDCtwXA4\\\\\\\/B5sL2d50IA8+AoEtwmpwMGVygj0OYH9S5BBl\\\\\\\/cMFfL1Rww9uBlR04IMyeE5Ggsq7qoJUZE0m\\\\\\\/J2ph+oDcnup7DZ9eaAYKFCMHcUoBEgGZRFQgS3wodWwyKL6kbR3GJi6+\\\\\\\/KhQWkFjdHGZecFIVmB8aGEODeJlwEiIDByyx0P0b1VJI6p4BihVtvbupAFr2j\\\\\\\/bZEAuOItt7muLhhKKnKuduj\\\\\\\/fmXpq17QhjgBqSxhLdZSPxJg9k9WePVlI8caCu8qARV0lM9TZ5VU62gyK91KN+YtPF4KtqEhB96asF96JSS1eqNv5jh3ztrHyGY0zCYgifIhFYVfpk7efabcMJXprlYDVylsVY1govU7jrDFWJOIm9Lb\\\\\\\/Y3CX4ngzXISwhjcgPgP1fDk2VpMVvOs25jIWyygxT8GJBjdb\\\\\\\/p9CdVaTpYgcwtr6Ku84mBWz0dW\\\\\\\/fp7vrxyGUgjXcJlB1AsadH59ekuty\\\\\\\/EyNnLWBMefe\\\\\\\/2Z7Owoh\\\\\\\/k3khCgyp3khDxbfLf2QBItQR2Tukg94aD5xxnrOrmQn5LXqjcRllrwLrriWHp2sp6P1rikJaNkElzGSrZD\\\\\\\/4VvLUxV52J1nigwbqbe5kudUarlO29HYg7aZ0mkSDEIJKnI29yFbU9QSAqis1xhFThwatkPYjKP3wK+XEUMj\\\\\\\/5qtqTV89wZHPkYhUGNqeYkYK7M81xt9ybgXHxyMaqMMr4\\\\\\\/dwUpUSh5t5JU1YMXNQ5IeYGXyjRIN9h2ii9GPslGDolwLb\\\\\\\/gN66TuJoN9AFgqPJFq4P8k64S58dDGtXevqbMmIvw3xELcUy0FBwL2ru2Ljxi2I2225yGVTzTuy0eBHHhQYJRUIK71px3VILwJAOoGUYFsz\\\\\\\/O9KwyzPxYTUubAkHcLaiVznuDJNpl4A6\\\\\\\/jsUvMJkgbFcWRnCnZw7L5WuvbRpoMjbaAsGJRfUmhpeqcekqi484ovzbJDga0lzfg54JTFP58hB6Gocf9YUfJpX4QTjPWr\\\\\\\/c0hWiWM7okRDwTpwPJf+gYyLpQL4pk0V5oT2y3RGWGPVxA8bom95plNY60O7qwTgwLAoz36Zus7OCzpOBJbVpPdg57xLoL6BSavxlOLKiRws6SvKzEMESYVgm4UlUjtHCSJsgCBu9NAxcqyHCvC8qB+4TpqnwtTsxdgx\\\\\\\/SqMrtFJXzmSiXTC8CIQFqnOaTWUSUvniXPSUVwhghVqVw1jTPDRtPP9CSBVAQNjkw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022pCQBkpBynUlWm1DhKhYHaA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222173c4ad7366b710fb2bc8150bcd37ea647cd297838b53d8c95c08144af2d888\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-486771 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3243" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-p_p-0-yaxlPf3NhP2W-3Ord2QMEzFV1p8ZgAVCZVF60" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3243" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KGYjyQBWItrePBulBOUjWopKt60oWPOBPJvTb41AKIs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/10/3237" title="Meeting Report on Experimental Approaches to Evolution and Ecology Using Yeast and Other Model Systems" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/10/3251" title="Exploring Potential Germline-Associated Roles of the TRIM-NHL Protein NHL-2 Through RNAi Screening" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486771" id="node486771" data-pisa="ggg;7/10/3243" data-pisa-master="ggg;g3.117.300128" data-apath="/ggg/7/10/3243.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Quality <em>de Novo</em> Genome Assembly of the <em>Dekkera bruxellensis</em> Yeast Using Nanopore MinION Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Téo Fournier</span>, <span class="highwire-citation-author" data-delta="1">Jean-Sébastien Gounot</span>, <span class="highwire-citation-author" data-delta="2">Kelle Freel</span>, <span class="highwire-citation-author" data-delta="3">Corinne Cruaud</span>, <span class="highwire-citation-author" data-delta="4">Arnaud Lemainque</span>, <span class="highwire-citation-author" data-delta="5">Jean-Marc Aury</span>, <span class="highwire-citation-author" data-delta="6">Patrick Wincker</span>, <span class="highwire-citation-author" data-delta="7">Joseph Schacherer</span> and <span class="highwire-citation-author" data-delta="8">Anne Friedrich</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3243-3250;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300128 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486771" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Téo Fournier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=T%C3%A9o%2BFournier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fournier%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AT%25C3%25A9o%2BFournier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jean-Sébastien Gounot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-S%C3%A9bastien%2BGounot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gounot%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-S%25C3%25A9bastien%2BGounot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kelle Freel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelle%2BFreel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Freel%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKelle%2BFreel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Corinne Cruaud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Commissariat à l’Energie Atomique, Institut de Biologie François-Jacob, Genoscope, BP5706, 91057 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Corinne%2BCruaud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cruaud%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACorinne%2BCruaud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Arnaud Lemainque </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Commissariat à l’Energie Atomique, Institut de Biologie François-Jacob, Genoscope, BP5706, 91057 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arnaud%2BLemainque%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lemainque%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArnaud%2BLemainque%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jean-Marc Aury </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Commissariat à l’Energie Atomique, Institut de Biologie François-Jacob, Genoscope, BP5706, 91057 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Marc%2BAury%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aury%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Marc%2BAury%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Patrick Wincker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Commissariat à l’Energie Atomique, Institut de Biologie François-Jacob, Genoscope, BP5706, 91057 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BWincker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wincker%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BWincker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Joseph Schacherer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BSchacherer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schacherer%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BSchacherer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anne.friedrich@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">anne.friedrich@unistra.fr</a>
<a href="mailto:schacherer@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">schacherer@unistra.fr</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Anne Friedrich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BFriedrich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Friedrich%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnne%2BFriedrich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anne.friedrich@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">anne.friedrich@unistra.fr</a>
<a href="mailto:schacherer@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">schacherer@unistra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/10/3243" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:486771" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:486771/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3243.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:486771" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:486771/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3243.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:486771" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:486771/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3243.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:486771" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:486771/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/10/3243.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:486771" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:486771/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genetic variation in natural populations represents the raw material for phenotypic diversity. Species-wide characterization of genetic variants is crucial to have a deeper insight into the genotype-phenotype relationship. With the advent of new sequencing strategies and more recently the release of long-read sequencing platforms, it is now possible to explore the genetic diversity of any nonmodel organisms, representing a fundamental resource for biological research. In the frame of population genomic surveys, a first step is to obtain the complete sequence and high-quality assembly of a reference genome. Here, we sequenced and assembled a reference genome of the nonconventional <em>Dekkera bruxellensis</em> yeast. While this species is a major cause of wine spoilage, it paradoxically contributes to the specific flavor profile of some Belgium beers. In addition, an extreme karyotype variability is observed across natural isolates, highlighting that <em>D. bruxellensis</em> genome is very dynamic. The whole genome of the <em>D. bruxellensis</em> UMY321 isolate was sequenced using a combination of Nanopore long-read and Illumina short-read sequencing data. We generated the most complete and contiguous <em>de novo</em> assembly of <em>D. bruxellensis</em> to date and obtained a first glimpse into the genomic variability within this species by comparing the sequences of several isolates. This genome sequence is therefore of high value for population genomic surveys and represents a reference to study genome dynamic in this yeast species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genetic-diversity" class="hw-term hw-article-keyword hw-article-keyword-genetic-diversity" rel="nofollow">genetic diversity</a></li><li class="kwd"><a href="/keyword/reference-genome" class="hw-term hw-article-keyword hw-article-keyword-reference-genome" rel="nofollow">reference genome</a></li><li class="kwd"><a href="/keyword/population-genomics" class="hw-term hw-article-keyword hw-article-keyword-population-genomics" rel="nofollow">population genomics</a></li><li class="kwd"><a href="/keyword/minion-sequencing" class="hw-term hw-article-keyword hw-article-keyword-minion-sequencing" rel="nofollow">MinION sequencing</a></li><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/dekkera-bruxellensis" class="hw-term hw-article-keyword hw-article-keyword-dekkera-bruxellensis" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Dekkera bruxellensis</em></a></li><li class="kwd"><a href="/keyword/genome-report" class="hw-term hw-article-keyword hw-article-keyword-genome-report" rel="nofollow">genome report</a></li></ul><p id="p-4">Knowledge in biology has been greatly improved by exploring a large diversity of species as well as evolutionary contexts. No single species is representative of the evolution of either an entire phylum or a whole genus. Exploration of the genetic diversity of nonmodel species is essential to have a better insight into the variation of the population history, recombination, selection, mutation, and the genotype-phenotype relationship. In this context, the Saccharomycotina subphylum (budding yeasts), which includes the baker’s yeast <em>Saccharomyces cerevisiae</em>, represents an ideal group of nonmodel organisms for population genomic studies (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Peter and Schacherer 2016</a>).</p><p id="p-5">Recent years have seen a burst of population genomic surveys focusing on various nonconventional yeasts associated with different objectives. This has a bearing on several aspects of evolutionary biology. Analysis of resequencing data of a large sample of isolates from the same species has been focused on yeast model organisms such as <em>S. cerevisiae</em> (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Schacherer <em>et al.</em> 2009</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Skelly <em>et al.</em> 2013</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bergström <em>et al.</em> 2014</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Almeida <em>et al.</em> 2015</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Strope <em>et al.</em> 2015</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gallone <em>et al.</em> 2016</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gonçalves <em>et al.</em> 2016</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Zhu <em>et al.</em> 2016</a>) and the fission yeast <em>Schizosaccharomyces pombe</em> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fawcett <em>et al.</em> 2014</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jeffares <em>et al.</em> 2015</a>), as well as on the nonmodel yeast species <em>Saccharomyces paradoxus</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Leducq <em>et al.</em> 2016</a>), <em>Saccharomyces uvarum</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Almeida <em>et al.</em> 2014</a>), <em>Candida albicans</em> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ford <em>et al.</em> 2015</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hirakawa <em>et al.</em> 2015</a>), and <em>Lachancea kluyveri</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brion <em>et al.</em> 2015</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">2016</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Friedrich <em>et al.</em> 2015</a>). Altogether, these data and analysis enhanced our knowledge about the evolutionary history of species (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Almeida <em>et al.</em> 2014</a>), the forces involved in genome evolution (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Friedrich <em>et al.</em> 2015</a>), and the genetic basis of the phenotypic diversity (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Ford <em>et al.</em> 2015</a>).</p><p id="p-6">Among the Saccharomycotina, <em>Dekkera bruxellensis</em> is a yeast species associated with human fermentation processes that is well known as a major cause of wine spoilage, and also as an essential contributor to Belgium lambic and gueuze beer fermentation (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schifferdecker <em>et al.</em> 2014</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Masneuf-Pomarede <em>et al.</em> 2015</a>). In addition to its industrial properties, this species is of interest at the evolutionary level. Natural isolates show different ploidy levels (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Borneman <em>et al.</em> 2014</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Curtin and Pretorius 2014</a>) and extensive chromosomal rearrangements, which were observed through electrophoretic karyotypes (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hellborg and Piškur 2009</a>). These observations indicate a rapid evolution at the intraspecific level. Recent findings suggest that the ploidy level could be linked to the substrate of origin of the strain and related to adaptive processes linked to specific environments (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albertin <em>et al.</em> 2014</a>). Consequently, a genome-wide polymorphism survey based on a representative set of <em>D. bruxellensis</em> individuals would be of interest. The exploration of single nucleotide polymorphisms (SNPs), small indels, as well as structural variants such as large indels, and inversions and translocations at the species level would help provide insight into the forces that shape genomic architecture and evolution. However, to conduct a population genomic survey, the availability of a high-quality reference sequence for the species at a completeness level to cover the majority of the genomic variation and a contiguity level to efficiently detect structural variants, is a prerequisite.</p><p id="p-7">To date, population genomic studies have mostly been performed on species for which chromosomal-scale genome assemblies were available; however, this necessary high-quality assembly was unfortunately not yet available for the <em>D. bruxellensis</em> species. Here, we present the <em>de novo</em> sequence and high-quality genome assembly of the UMY321 <em>D. bruxellensis</em> isolate with a combination of long Oxford Nanopore and short Illumina reads. By aligning the short-read sequencing data from a total of eight sequenced natural isolates on the generated assembly, as well as other previously available assemblies (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Curtin <em>et al.</em> 2012</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Piškur <em>et al.</em> 2012</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Borneman <em>et al.</em> 2014</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Crauwels <em>et al.</em> 2014</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Olsen <em>et al.</em> 2015</a>), we tested the capacity of our assembly to be used as a reference assembly for future population genomic studies of this nonmodel species. The results showed that we generated the most complete and contiguous <em>de novo</em> assembly of <em>D. bruxellensis</em> necessary to explore the intraspecific genetic diversity of this unique and economically relevant species.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains and DNA preparation</h3><p id="p-8">We selected three <em>D. bruxellensis</em> diploid isolates from various ecological and geographical origins (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The UMY321 isolate was chosen for the generation of a high-quality assembly and was therefore subjected to Oxford Nanopore and Illumina sequencing. The two other isolates, UMY315 and 133, were only subjected to Illumina sequencing for comparative analysis purposes.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486959/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486959/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Description of the <em>D. bruxellensis</em> isolates used in this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-9">Yeast cell cultures were grown overnight at 30° in 20 ml of YPD medium to early stationary phase before cells were harvested by centrifugation. Total genomic DNA was than extracted using the QIAGEN Genomic-tip 100/G according to the manufacturer’s instructions.</p></div><div id="sec-3" class="subsection"><h3>Flow cytometry</h3><p id="p-10">Samples were prepared for DNA content analysis using flow cytometry. Cells were grown in YPD medium at 30° to reach exponential phase. They were then pelleted and washed with 1 ml water. In order to fix the cell, the pellet was resuspended in 1 ml of 70% ethanol. After centrifugation, supernatant was discarded and cells were resuspended in 1 ml sodium citrate buffer (trisodium citrate 50 mM; pH 7.5). Cells were pelleted once more and resuspended in 1 ml sodium citrate buffer supplemented with 10 µl of RNase A (100 mg/ml) and incubated at 37° for 2 hr. Samples were then sonicated (Sonics Vibra-Cell VC750) for 20 sec with a 20% amplitude. After sonication, 1 ml sodium citrate buffer supplemented with 10 µl propidium iodide (1.6 mg/ml) and left in the dark at 4° for 12 hr. Once the cells were stained with propidium iodide, cell DNA content was assessed by measuring fluorescence intensity using flow cytometry (CyFlow Space; Partec).</p></div><div id="sec-4" class="subsection"><h3>MinION library preparation and sequencing</h3><p id="p-11">We sheared 2 μg genomic DNA to ∼8000 bp with g-TUBE. After clean-up using 1× AMPure XP beads, Nanopore’s 8-kb two-dimensional (2D) sequencing libraries were prepared according to the SQK-MAP005-MinION gDNA Sequencing Kit protocol.</p><p id="p-12">The sequencing mix was prepared with 8 μl of the DNA library, water, Fuel Mix, and Running buffer, according to the SQK-MAP005 protocol. The sequencing mix was added to the R7.3 flowcell for a 48 hr run. The flowcell was reloaded one time at 24 hr with an addition of 8 μl of the DNA library.</p></div><div id="sec-5" class="subsection"><h3>Illumina sequencing</h3><p id="p-13">Genomic Illumina sequencing libraries were prepared with a mean insert size of 280 bp and were subjected to paired-end sequencing (2 × 100 bp) on Illumina HiSeq2000 sequencers.</p></div><div id="sec-6" class="subsection"><h3>De novo genome assembly</h3><p id="p-14">Various sets of the longest MinION 2D reads, which refer to various theoretical genome coverage (10×, 15×, 20×, and all the 2D reads, <em>i.e.</em> ∼25×) taking 15 Mb as genome size estimate (Supplemental Material, Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300128/-/DC1/FileS1.xls">File S1</a>), were subjected to four assemblers: ABruijn (v0.3b) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lin <em>et al.</em> 2016</a>), Canu (v1.1) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Berlin <em>et al.</em> 2015</a>), miniasm (v0.2-r137-dirty) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li 2016</a>), and SMARTdenovo (<a href="https://github.com/ruanjue/smartdenovo">https://github.com/ruanjue/smartdenovo</a>). ABruijn and miniasm were run with default parameters, while “genomeSize=13 m, minReadLength=2500, mhapSensitivity=high, corMhapSensitivity=high, and corOutCoverage=500” was set for Canu and “-c 1 -k 14 -J 2500 -e zmo” for SMARTdenovo. After the assembly step, we polished each set of contigs with Pilon (v1.18) (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Walker <em>et al.</em> 2014</a>), using ∼100× of Illumina 2 × 100 bp paired-end reads. SSPACE-LongRead (v1.1) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Boetzer and Pirovano 2014</a>) was finally used to scaffold the selected assembly using long-reads information.</p></div><div id="sec-7" class="subsection"><h3>Assembly completeness evaluation</h3><p id="p-15">The completeness of our assembly was evaluated firstly, through the proportion of unmapped short reads (see <em>Short-read mapping</em>) determined with Samtools (v0.1.19) (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2009</a>) using the option “view -f 4 -c”; and secondly, through the proportion of ultraconserved core eukaryotic genes recovered by CEGMA (v2.5) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Parra <em>et al.</em> 2007</a>), with default parameters.</p></div><div id="sec-8" class="subsection"><h3>Whole genome comparison</h3><p id="p-16">Whole genome comparisons were performed with MUMmer (v3.0) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kurtz <em>et al.</em> 2004</a>). nucmer was used to align the sequences (with –maxmatch option). The alignments coordinates were extracted to determine the proportion of non-N residues of each assembly that were covered. The delta files were filtered for alignments &lt;5 kb and plots were generated with mummerplot.</p></div><div id="sec-9" class="subsection"><h3>Short-read mapping</h3><p id="p-17">Reads were mapped with BWA (v0.7.4) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li and Durbin 2009</a>) and unmapped reads were estimated with Samtools (v0.1.19) (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2009</a>). GATK (v3.3) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">McKenna <em>et al.</em> 2010</a>) was used for local realignment of the reads around indels, SNPs calling, and to add allele balance information in the vcf file.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-18">All sequencing data generated in this study, as well as the UMY321 reference assembly (in FASTA format), have been deposited in the European Nucleotide Archive under the accession number PRJEB21262.</p></div></div><div class="section results-discussion" id="sec-11"><h2 class="">Results and Discussion</h2><p id="p-19">Three <em>D. bruxellensis</em> isolates (UMY321, UMY315, and 133) were sequenced in this study (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). These strains were determined to be diploid based on flow cytometry analysis and were all isolated from wine or grape must in Italy or South Africa. The genome of the UMY321 isolate was sequenced using a combination of Nanopore long-read and Illumina short-read sequencing data to obtain a high-quality assembly. By contrast, the UM315 and 133 isolates were only sequenced using a short-read strategy. In addition, these genomes were compared to previously genome sequences of six other <em>D. bruxellensis</em> isolates (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Curtin <em>et al.</em> 2012</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Piškur <em>et al.</em> 2012</a>; <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Borneman <em>et al.</em> 2014</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Crauwels <em>et al.</em> 2014</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Olsen <em>et al.</em> 2015</a>).</p><div id="sec-12" class="subsection"><h3>De novo genome assembly construction and comparison</h3><p id="p-20">For the UMY321 isolate, a total of three MinION Mk1 runs were performed with the R7.3 chemistry using 2D library types with 8 kb mean fragmentation size. A total of 115,559 reads representing a cumulative size of 1.15 Gb were generated, among which 41,686 2D reads showed an average quality greater than nine (2D pass reads). We focused on these 2D pass reads representing a total of 376.8 Mb, with the longest read being 70,058 bp (mean = 9033 bp and median = 8676 bp) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300128/-/DC1/FigureS1.eps">Figure S1</a>). Four subsets of our 2D pass reads (10×, 15×, and 20× of the longest 2D pass reads, and all of them, <em>i.e.</em> ∼25×) (Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300128/-/DC1/FileS1.xls">File S1</a>) were submitted to four assemblers: ABruijn (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Lin <em>et al.</em> 2016</a>), Canu (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Berlin <em>et al.</em> 2015</a>), miniasm (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Li 2016</a>), and SMARTdenovo (<a href="https://github.com/ruanjue/smartdenovo">https://github.com/ruanjue/smartdenovo</a>). As the MinION sequencing technology is known to be associated with high error rates (∼10% for 2D pass reads) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jain <em>et al.</em> 2016</a>), we polished the assemblies with Pilon (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Walker <em>et al.</em> 2014</a>) using 100× of Illumina paired-end reads. The lengths of the constructed assemblies were all in the same order of magnitude and ranged from 11.7 to 13.7 Mb (Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300128/-/DC1/FileS1.xls">File S1</a>).</p><p id="p-21">Using these various datasets and assemblies, the objective was to define the best assembler and the minimal coverage needed. Hence, we computed the standard contiguity metrics for all assemblies to evaluate their quality, which is related to both the assembler and the dataset (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300128/-/DC1/FileS1.xls">File S1</a>). First, we observed that, considering the results by assembler, the number of scaffolds obtained with the 10× dataset is much higher compared to the other datasets, which suggests that a 10× coverage of MinION reads is too low to obtain a good quality assembly. By assembler, the results obtained for the higher coverages are comparable. Using Canu, the number of scaffolds is much higher and N90 as well as N50 are much lower, producing the less connected assemblies (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300128/-/DC1/FileS1.xls">File S1</a>). The contiguity metrics associated with the assemblies constructed with SMARTdenovo, ABruijn, and miniasm were closely related, and it seemed difficult to select a single best assembly on the sole basis of these measurements, especially since good contiguity metrics are not necessarily associated with assembly completeness.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3243/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Metrics related to the constructed assemblies, per assembler and dataset." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1499191924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Metrics related to the constructed assemblies, per assembler and dataset.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3243/F1.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/10/3243/F1.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3243/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3243/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486939" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-22" class="first-child">Metrics related to the constructed assemblies, per assembler and dataset.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Considering these results, we decided to map the Illumina paired-end reads back on the generated assemblies with BWA (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Li and Durbin 2009</a>). Among all the assemblies, the proportion of unmapped reads ranged from 1.12 to 11% (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300128/-/DC1/FileS1.xls">File S1</a>). Surprisingly, the assemblies constructed with miniasm were less complete, as &gt;5% of the reads did not map back, compared to &lt;1.5% for the ABruijn and SMARTdenovo assemblies.</p><p id="p-24">By comparing standard metrics and the proportion of unmapped reads, the most accurate assembly was obtained with the 20× 2D reads dataset combined with the SMARTdenovo assembler. This assembly is composed of nine scaffolds, <em>i.e.</em>, very close to the estimated number of chromosomes, which appears to vary between four and nine among different strains of this species (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hellborg and Piškur 2009</a>), for a complete assembly size of 12.97 Mb. This was then submitted to SSPACE-longreads, which reduced the number of scaffolds to eight after grouping the two smallest ones, based on our long-read information, and a further Pilon run. The final assembly contains eight scaffolds and shows a cumulative size of 12,965,163 bp (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). We also evaluated the completeness of our assembly at the gene content level by running CEGMA (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Parra <em>et al.</em> 2007</a>): 245 out of the 248 most extremely conserved genes in eukaryotes were detected in our assembly, through 242 complete and three partial alignments. Altogether, these results reveal a high level of completeness of our assembly.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486970/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486970/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Metrics associated with the <em>D. bruxellensis</em> publicly available assemblies</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Comparison with available assemblies of D. bruxellensis</h3><p id="p-25">To date, several assemblies of the <em>D. bruxellensis</em> species have already been released (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Curtin <em>et al.</em> 2012</a>; <a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Piškur <em>et al.</em> 2012</a>; <a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Borneman <em>et al.</em> 2014</a>; <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Crauwels <em>et al.</em> 2014</a>; <a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Olsen <em>et al.</em> 2015</a>). These assemblies are related to isolates from different ecological and geographical origins (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). They were mostly constructed by combining several sequencing methods, such as 454, PacBio, and Illumina, as well as optical mapping in the most recently published assembly (<a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Olsen <em>et al.</em> 2015</a>).</p><p id="p-26">The assemblies have very variable metrics associated with each of them (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). In terms of contiguity, our assembly and the assembly generated for the CBS11270 isolate are close, and reach a chromosome-scale resolution. However, the CBS11270 assembly is much larger than the others (17.3 Mb <em>vs.</em> 12.7–13.4 Mb), although it does also contain ∼2.5 Mb of undetermined (N) residues.</p><p id="p-27">By comparing the assembly metrics, we determined that our assembly is closer to that for CBS11270, which was generated by combining PacBio and Illumina sequencing methods as well as optical mapping, and much better than the other three available for comparison, which were much more fragmented and comprised at least 84 scaffolds.</p><p id="p-28">A MUMmer comparison of our UMY321 assembly to that of CBS11270 indicates that 91 and 99.6% of the assemblies aligned, respectively, with one another and revealed that the scaffolds are mostly collinear (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). However, some large repetitive regions can be observed in the CBS11270 assembly, <em>e.g.</em>, on chromosome 1, between chromosomes 1 and 6, and between chromosomes 4 and 5 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300128/-/DC1/FigureS2.eps">Figure S2</a>) that are absent in our assembly, and could explain the size differences between the assemblies (17.3 Mb <em>vs.</em> 12.97 Mb). Moreover, some synteny breaks can be observed, at the level of scaffolds, specifically between three and four. All the inconsistencies between the assemblies could be related either to structural rearrangements between the isolates or to assembly errors, and would require further investigations to reach a conclusion as to their most likely source.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3243/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the CBS11270 and UMY321 assemblies. The alignments and the plot were generated with the MUMmer software suite. Red lines: sequences aligning in the same direction. Blue lines: sequences aligning in the opposite direction." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1499191924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of the CBS11270 and UMY321 assemblies. The alignments and the plot were generated with the MUMmer software suite. Red lines: sequences aligning in the same direction. Blue lines: sequences aligning in the opposite direction.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3243/F2.medium.gif" width="440" height="324"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/10/3243/F2.medium.gif" width="440" height="324"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3243/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3243/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486795" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Comparison of the CBS11270 and UMY321 assemblies. The alignments and the plot were generated with the MUMmer software suite. Red lines: sequences aligning in the same direction. Blue lines: sequences aligning in the opposite direction.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Suitability of our assembly for population genomics studies</h3><p id="p-30">As previously mentioned, to function as a valuable resource for conducting population genomics studies, a reference genome should combine high contiguity (for the detection of structural variants) and completeness (for the efficient detection of SNPs and small indels). At the contiguity level, our assembly is close from a chromosomal-scale resolution, which suggests that it would be highly suitable for gross structural rearrangement detection (translocations, inversions, and long insertions/deletions).</p><p id="p-31">To test our assembly for the detection of polymorphism along the genome, we further investigated the mapping of the Illumina reads. As previously mentioned, 98.89% of the UMY321 Illumina reads mapped on our assembly. The read coverage was homogeneous along the scaffolds (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>), which suggests that the strain is devoid of aneuploidy and segmental duplication, and confirms the lack of large repetitive regions within our assembly.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3243/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mapping of the Illumina reads vs. the UMY321 reference assembly. (A) Illumina reads coverage along the reference genome. (B) Frequency of the reference allele at heterozygous sites along the genome. (Each color corresponds to a scaffold)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1499191924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mapping of the Illumina reads &lt;em&gt;vs.&lt;/em&gt; the UMY321 reference assembly. (A) Illumina reads coverage along the reference genome. (B) Frequency of the reference allele at heterozygous sites along the genome. (Each color corresponds to a scaffold).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3243/F3.medium.gif" width="440" height="235"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/10/3243/F3.medium.gif" width="440" height="235"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3243/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3243/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486882" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Mapping of the Illumina reads <em>vs.</em> the UMY321 reference assembly. (A) Illumina reads coverage along the reference genome. (B) Frequency of the reference allele at heterozygous sites along the genome. (Each color corresponds to a scaffold).</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">A total of 83,006 SNPs were detected with GATK (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">McKenna <em>et al.</em> 2010</a>), among which 374 were homozygous and 82,632 were heterozygous (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300128/-/DC1/FileS1.xls">File S1</a>). The 374 homozygous SNPs could be considered as false positives. Although not completely negligible, this number is very low and could be related to the high error rate of the MinION technology, which is not completely compensated by using Illumina short reads (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Istace <em>et al.</em> 2017</a>).</p><p id="p-34">The UMY321 isolate that we sequenced is diploid, and the detection of these 82,632 heterozygous SNPs revealed that the two genomic copies are not identical and have a high heterozygosity level. These heterozygous positions are mostly evenly distributed all along the genome, with several regions showing loss of heterozygosity (LOH) on scaffolds 1, 2, 3, and 6 (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>).</p><p id="p-35">Altogether, these results confirmed that our assembly performs well when mapping the reads that were used for its construction. However, to determine if an assembly is relevant in the context of population genomic studies, we also analyzed its performance when mapping reads from other isolates. To survey polymorphisms within a species, resequencing projects rely mainly on Illumina sequencing technology, therefore we mapped the short reads related to this species that were publically available as well as from two isolates we sequenced in the context of this project (Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300128/-/DC1/FileS1.xls">File S1</a>) against our assembly and reported the proportion of unmapped reads. We also aligned these reads against the publicly available assemblies to perform a comparative analysis (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). As expected, the UMY321 Illumina paired-end reads mapped better on our assembly with only 1.11% of unmapped reads. More surprisingly, short reads generated in the context of the other projects also mapped better on our assembly compared to their related assemblies, and more generally compared to all other assemblies (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). It is also worth noting that all the reads, including those related to the CBS11270 isolate, mapped less efficiently to the CBS11270 assembly compared to all other assemblies, which suggests that although this assembly is highly contiguous and much larger than the others available, it is less complete.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486804/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486804/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Proportion of <em>D. bruxellensis</em> unmapped Illumina reads on the available assemblies</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3243/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illumina unmapped reads per assembly. Boxplot of the percentage of unmapped Illumina reads, according to the assembly used for the mapping." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1499191924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Illumina unmapped reads per assembly. Boxplot of the percentage of unmapped Illumina reads, according to the assembly used for the mapping.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3243/F4.medium.gif" width="440" height="347"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/10/3243/F4.medium.gif" width="440" height="347"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3243/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3243/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486858" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">Illumina unmapped reads per assembly. Boxplot of the percentage of unmapped Illumina reads, according to the assembly used for the mapping.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Insight into the intraspecific genetic variability</h3><p id="p-37">Finally, we took advantage of the availability of Illumina reads related to different isolates in order to obtain a first glimpse into the genomic variability within this species, using our UMY321 assembly as a reference. The read coverage along the reference sequence was mostly homogeneous for all isolates, and only few deviations were observed, limited to small genomic regions, which are characteristic of segmental duplications, in the ST05.12/22 isolate (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>). This suggests that the structural variants within this species are mostly balanced. It can also be noticed that the coverage plot obtained from the CBS11270 Illumina reads did not show twofold deviations on scaffolds 1, 3, or 4 (data not shown), as expected from the comparison of the CBS11270 and UMY321 assemblies (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300128/-/DC1/FigureS2.eps">Figure S2</a>), suggesting that the repetitive regions highlighted in the CBS11270 assembly are most probably related to assembly errors.</p><p id="p-38">Among the eight studied isolates, one is triploid (AWRI1608) and all the others are diploid (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). A total of 1,268,172 SNPs were detected across these eight isolates, among which 82% are heterozygous (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300128/-/DC1/FileS1.xls">File S1</a>). These SNPs are distributed over 500,707 polymorphic positions, with a majority present as singletons (68.8% of the polymorphic sites). However, a significant proportion of this variability is related to the triploid strain AWRI1608. Indeed, when this strain was not included in the analysis, 829,313 SNPs were detected over 188,717 polymorphic positions with only 50,702 singletons (27%). This is in agreement with the proposition that AWRI1608 consists of a slightly heterozygous diploid set of chromosomes with an additional full set of more distantly related chromosomes (<a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Borneman <em>et al.</em> 2014</a>). The phylogenetic relationships between this small sample of isolates based on the whole set of polymorphic positions also reflect the high divergence of this triploid isolate (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300128/-/DC1/FigureS3.eps">Figure S3</a>A). Ploidy levels across the genomes were also confirmed by taking advantage of allele frequency at heterozygous positions, which was ∼0.5 for diploid isolates and 0.33/0.66 for the AWRI1608 genome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300128/-/DC1/FigureS3.eps">Figure S3</a>B). These heterozygous positions are evenly distributed along the genome; however, LOH regions were detected in all the diploid isolates (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>).</p></div><div id="sec-16" class="subsection"><h3>Conclusions</h3><p id="p-39"><em>D. bruxellensis</em> is a yeast species of great importance in fermented beverage industries, largely thought of as a contaminant organism (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Schifferdecker <em>et al.</em> 2014</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Masneuf-Pomarede <em>et al.</em> 2015</a>). This species is also an interesting model to study genome evolution and dynamics as it is characterized by a large genomic plasticity. For these reasons, we sought to generate a high-quality genome assembly and ultimately obtain a suitable reference genome for population genomics. Our analyses show that the <em>D. bruxellensis</em> assembly that we generated with a combination of moderate coverage (20×) MinION long-reads in addition to a higher coverage (100×) of Illumina reads utilized for sequence polishing purposes, is highly valuable for population genomic studies and outperforms previously available sequences. Preliminary comparison among a small set of nine isolates already highlights the presence of large regions of LOH, which appears to be key factor in the genome evolution and adaptation of a large number of yeast species (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Magwene <em>et al.</em> 2011</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Ford <em>et al.</em> 2015</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Smukowski Heil <em>et al.</em> 2017</a>). To obtain a species-wide view of the genetic variability of <em>D. bruxellensis</em>, many more isolates should be surveyed using both short-read as well as long-read sequencing techniques, which will allow for the exploration of the structural variant landscape.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">We are grateful to Warren Albertin and Isabelle Masneuf-Pomarede for fruitful discussions and invaluable advice. We thank Jure Piskur and Anna Schifferdecker for generously providing the 133, UMY315 and UMY321 <em>Dekkera bruxellensis</em> isolates. We also thank the BioImage platform (IBMP-CNRS, Strasbourg, France) for their support. This work was supported by France Génomique (grant ANR-10-INBS-09-08), and the Agence Nationale de la Recherche (grant ANR-16-CE12-0019 to J.S.). T.F. and J.-S.G. are supported by a grant from the French “Ministère de l’Enseignement Supérieur et de la Recherche.” J.S. is a Fellow of the University of Strasbourg Institute for Advanced Study (USIAS) and a member of the Institut Universitaire de France.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-41">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300128/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300128/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-42"><em>Communicating editor: A. Dudley</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-06-16"><span class="received-label">Received </span>June 16, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-08-07"><span class="accepted-label">Accepted </span>August 7, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Fournier <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.1" data-doi="10.1016/j.fm.2014.03.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albertin</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panfili</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miot-Sertier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goulielmakis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcamp</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development of microsatellite markers for the rapid and reliable genotyping of <em>Brettanomyces bruxellensis</em> at strain level.</span> <abbr class="cit-jnl-abbrev">Food Microbiol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">188</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFood%2BMicrobiol.%26rft.volume%253D42%26rft.spage%253D188%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fm.2014.03.012%26rft_id%253Dinfo%253Apmid%252F24929736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fm.2014.03.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24929736&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000338597100028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonçalves</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teixeira</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Libkind</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bontrager</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A Gondwanan imprint on global diversity and domestication of wine and cider yeast <em>Saccharomyces uvarum</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">4044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D4044%26rft_id%253Dinfo%253Apmid%252F24887054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24887054&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.3" data-doi="10.1111/mec.13341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbosa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zalar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imanishi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A population genomics insight into the Mediterranean origins of wine yeast domestication.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">5412</span>–<span class="cit-lpage">5427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D5412%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.4" data-doi="10.1093/molbev/msu037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergström</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barré</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A high-definition view of functional genetic variation from natural yeast genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">872</span>–<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu037%26rft_id%253Dinfo%253Apmid%252F24425782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24425782&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000333283200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.5" data-doi="10.1038/nbt.3238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berlin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">C.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landolin</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assembling large genomes with single-molecule sequencing and locality-sensitive hashing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D623%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3238%26rft_id%253Dinfo%253Apmid%252F26006009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26006009&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.6" data-doi="10.1186/1471-2105-15-211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boetzer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pirovano</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">SSPACE-LongRead: scaffolding bacterial draft genomes using long read sequence information.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D211%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-211%26rft_id%253Dinfo%253Apmid%252F24950923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24950923&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.7" data-doi="10.1371/journal.pgen.1004161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borneman</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeppel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chambers</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curtin</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Insights into the <em>Dekkera bruxellensis</em> genomic landscape: comparative genomics reveals variations in ploidy and nutrient utilisation potential amongst wine isolates.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004161%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004161%26rft_id%253Dinfo%253Apmid%252F24550744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004161&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24550744&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.8" data-doi="10.1093/nar/gkv363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brion</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pflieger</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedrich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evolution of intraspecific transcriptomic landscapes in yeasts.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">4558</span>–<span class="cit-lpage">4568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv363%26rft_id%253Dinfo%253Apmid%252F25897111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25897111&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.9" data-doi="10.1091/mbc.E15-12-0816"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brion</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pflieger</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souali-Crespo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedrich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Differences in environmental stress response among yeasts is consistent with species-specific lifestyles.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1694</span>–<span class="cit-lpage">1705</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E15-12-0816%26rft_id%253Dinfo%253Apmid%252F27009200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xMC8xNjk0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zMjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.10" data-doi="10.1128/AEM.00601-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crauwels</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steensels</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Busschaert</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Samblanx</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Assessing genetic diversity among <em>Brettanomyces</em> yeasts by DNA fingerprinting and whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">4398</span>–<span class="cit-lpage">4413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.00601-14%26rft_id%253Dinfo%253Apmid%252F24814796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI4MC8xNC80Mzk4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zMjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.11" data-doi="10.1111/1567-1364.12198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Curtin</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pretorius</span>  <span class="cit-name-given-names">I. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic insights into the evolution of industrial yeast species <em>Brettanomyces bruxellensis</em>.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">997</span>–<span class="cit-lpage">1005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1567-1364.12198%26rft_id%253Dinfo%253Apmid%252F25142832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1567-1364.12198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25142832&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.12" data-doi="10.1371/journal.pone.0033840"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Curtin</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borneman</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chambers</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pretorius</span>  <span class="cit-name-given-names">I. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">De-novo assembly and analysis of the heterozygous triploid genome of the wine spoilage yeast <em>Dekkera bruxellensis</em> AWRI1499.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e33840</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DCurtin%26rft.auinit1%253DC.%2BD.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De33840%26rft.epage%253De33840%26rft.atitle%253DDe-novo%2Bassembly%2Band%2Banalysis%2Bof%2Bthe%2Bheterozygous%2Btriploid%2Bgenome%2Bof%2Bthe%2Bwine%2Bspoilage%2Byeast%2BDekkera%2Bbruxellensis%2BAWRI1499.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0033840%26rft_id%253Dinfo%253Apmid%252F22470482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0033840&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22470482&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.13" data-doi="10.1371/journal.pone.0104241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fawcett</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iida</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takuno</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugino</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kado</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Population genomics of the fission yeast <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e104241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De104241%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0104241%26rft_id%253Dinfo%253Apmid%252F25111393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0104241&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25111393&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.14" data-doi="10.7554/eLife.00662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ford</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Funt</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbey</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Issi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guiducci</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The evolution of drug resistance in clinical isolates of <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e00662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D4%26rft.spage%253De00662%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.00662%26rft_id%253Dinfo%253Apmid%252F25646566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.00662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25646566&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.15" data-doi="10.1093/molbev/msu295"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedrich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reisser</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Population genomics reveals chromosome-scale heterogeneous evolution in a protoploid yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">184</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu295%26rft_id%253Dinfo%253Apmid%252F25349286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu295&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25349286&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.16" data-doi="10.1016/j.cell.2016.08.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallone</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steensels</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prahl</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soriaga</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saels</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Domestication and divergence of <em>Saccharomyces cerevisiae</em> beer yeasts.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">1397</span>–<span class="cit-lpage">1410.e16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D166%26rft.spage%253D1397%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.08.020%26rft_id%253Dinfo%253Apmid%252F27610566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.08.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27610566&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonçalves</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pontes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbosa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serra</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2750</span>–<span class="cit-lpage">2761</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D26%26rft.spage%253D2750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.18" data-doi="10.1128/EC.00115-09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hellborg</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piškur</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Complex nature of the genome in a wine spoilage yeast, <em>Dekkera bruxellensis</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1739</span>–<span class="cit-lpage">1749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00115-09%26rft_id%253Dinfo%253Apmid%252F19717738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo5OiI4LzExLzE3MzkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEwLzMyNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.19" data-doi="10.1101/gr.174623.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirakawa</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakthikumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">M. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berlin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic and phenotypic intra-species variation in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.174623.114%26rft_id%253Dinfo%253Apmid%252F25504520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzMvNDEzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zMjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Istace</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedrich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">d’Agata</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faye</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">De novo assembly and population genomic survey of natural yeast isolates with the Oxford Nanopore MinION sequencer.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paten</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akeson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Oxford Nanopore MinION: delivery of nanopore sequencing to the genomics community.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.22" data-doi="10.1038/ng.3215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeffares</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rallis</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieux</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Převorovský</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genomic and phenotypic diversity of <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D235%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3215%26rft_id%253Dinfo%253Apmid%252F25665008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25665008&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.23" data-doi="10.1186/gb-2004-5-2-r12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillippy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smoot</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shumway</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Versatile and open software for comparing large genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">R12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKurtz%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253DR12%26rft.epage%253DR12%26rft.atitle%253DVersatile%2Band%2Bopen%2Bsoftware%2Bfor%2Bcomparing%2Blarge%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2004-5-2-r12%26rft_id%253Dinfo%253Apmid%252F14759262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2004-5-2-r12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14759262&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leducq</span>  <span class="cit-name-given-names">J.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielly-Thibault</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charron</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberlein</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verta</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Speciation driven by hybridization and chromosomal plasticity in a wild yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Microbiol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">15003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMicrobiol.%26rft.volume%253D1%26rft.spage%253D15003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.25" data-doi="10.1093/bioinformatics/btw152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Minimap and miniasm: fast mapping and de novo assembly for noisy long sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2103</span>–<span class="cit-lpage">2110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw152%26rft_id%253Dinfo%253Apmid%252F27153593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27153593&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.26" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.27" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.28" data-doi="10.1073/pnas.1604560113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolmogorov</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaisson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Assembly of long error-prone reads using de Bruijn graphs.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">E8396</span>–<span class="cit-lpage">E8405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1604560113%26rft_id%253Dinfo%253Apmid%252F27956617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzUyL0U4Mzk2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zMjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.29" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomics of domestic and wild yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.30" data-doi="10.1073/pnas.1012544108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magwene</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kayıkçı</span>  <span class="cit-name-given-names">Ö.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granek</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reininga</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scholl</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Outcrossing, mitotic recombination, and life-history trade-offs shape genome evolution in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1987</span>–<span class="cit-lpage">1992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1012544108%26rft_id%253Dinfo%253Apmid%252F21245305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzUvMTk4NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTAvMzI0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masneuf-Pomarede</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bely</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marullo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albertin</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genetics of non-conventional wine yeasts: current knowledge and future challenges.</span> <abbr class="cit-jnl-abbrev">Front Microbiol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront%2BMicrobiol.%26rft.volume%253D6%26rft.spage%253D1563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.32" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTAvMzI0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">R.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bunikis</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiukova</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmberg</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lötstedt</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">De novo assembly of <em>Dekkera bruxellensis</em>: a multi technology approach using short and long-read sequencing and optical mapping.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D4%26rft.spage%253D56%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.34" data-doi="10.1093/bioinformatics/btm071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradnam</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm071%26rft_id%253Dinfo%253Apmid%252F17332020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm071&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17332020&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246773300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.35" data-doi="10.1002/yea.3142"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Population genomics of yeasts: towards a comprehensive view across a broad evolutionary scale.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%26rft.volume%253D33%26rft.spage%253D73%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.3142%26rft_id%253Dinfo%253Apmid%252F26592376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.3142&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26592376&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.36" data-doi="10.1016/j.ijfoodmicro.2012.05.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piškur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ling</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcet-Houben</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishchuk</span>  <span class="cit-name-given-names">O. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aerts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome of wine yeast <em>Dekkera bruxellensis</em> provides a tool to explore its food-related properties.</span> <abbr class="cit-jnl-abbrev">Int. J. Food Microbiol.</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">202</span>–<span class="cit-lpage">209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bfood%2Bmicrobiology%26rft.stitle%253DInt%2BJ%2BFood%2BMicrobiol%26rft.aulast%253DPiskur%26rft.auinit1%253DJ.%26rft.volume%253D157%26rft.issue%253D2%26rft.spage%253D202%26rft.epage%253D209%26rft.atitle%253DThe%2Bgenome%2Bof%2Bwine%2Byeast%2BDekkera%2Bbruxellensis%2Bprovides%2Ba%2Btool%2Bto%2Bexplore%2Bits%2Bfood-related%2Bproperties.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ijfoodmicro.2012.05.008%26rft_id%253Dinfo%253Apmid%252F22663979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ijfoodmicro.2012.05.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22663979&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.37" data-doi="10.1038/nature07670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruderfer</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comprehensive polymorphism survey elucidates population structure of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchacherer%26rft.auinit1%253DJ.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D342%26rft.epage%253D345%26rft.atitle%253DComprehensive%2Bpolymorphism%2Bsurvey%2Belucidates%2Bpopulation%2Bstructure%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07670%26rft_id%253Dinfo%253Apmid%252F19212320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212320&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schifferdecker</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dashko</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishchuk</span>  <span class="cit-name-given-names">O. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piškur</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The wine and beer yeast <em>Dekkera bruxellensis</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%26rft.volume%253D31%26rft.spage%253D323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.39" data-doi="10.1101/gr.155762.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Skelly</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merrihew</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riffle</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">E. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Integrative phenomics reveals insight into the structure of phenotypic diversity in budding yeast.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1496</span>–<span class="cit-lpage">1504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.155762.113%26rft_id%253Dinfo%253Apmid%252F23720455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIzLzkvMTQ5NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTAvMzI0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smukowski Heil</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeSevo</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pai</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoang</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Loss of heterozygosity drives adaptation in hybrid yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1596</span>–<span class="cit-lpage">1612</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D34%26rft.spage%253D1596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.41" data-doi="10.1101/gr.185538.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strope</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skelly</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozmin</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahadevan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The 100-genomes strains, an <em>S. cerevisiae</em> resource that illuminates its natural phenotypic and genotypic variation and emergence as an opportunistic pathogen.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">762</span>–<span class="cit-lpage">774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.185538.114%26rft_id%253Dinfo%253Apmid%252F25840857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzUvNzYyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zMjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.42" data-doi="10.1371/journal.pone.0112963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abeel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shea</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Priest</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abouelliel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e112963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112963%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0112963%26rft_id%253Dinfo%253Apmid%252F25409509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0112963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25409509&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3243.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Whole genome analysis of 132 clinical <em>Saccharomyces cerevisiae</em> strains reveals extensive ploidy variation.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2421</span>–<span class="cit-lpage">2434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D6%26rft.spage%253D2421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/10/3243.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/10/3237" title="Meeting Report on Experimental Approaches to Evolution and Ecology Using Yeast and Other Model Systems" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/10/3251" title="Exploring Potential Germline-Associated Roles of the TRIM-NHL Protein NHL-2 Through RNAi Screening" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/10"><span>Volume 7 Issue 10, October 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/10.cover-source.jpg?itok=lurMO2QE" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3243" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ydukeLeIJNcPDzvsQk_elTTDeFSr1rZ6LqhLjz8lhsU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/10/3243" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/10/3243" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/10/3243" class="active" data-icon-position="" data-hide-link-title="0">High-Quality de Novo Genome Assembly of the Dekkera bruxellensis Yeast Using Nanopore MinION Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/486771" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Io_r0fV1OU9PvzE48a3TcJvM31eOdJYuk2Yv3VxFDD4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/10/3243.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/10/3243" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Kes65mmvTOLdX3RIue8MxN3TxZEnV1KjYjAqmEO28Ds" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/10/3243" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/10/3243.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486771" id="node486771--2" data-pisa="ggg;7/10/3243" data-pisa-master="ggg;g3.117.300128" data-apath="/ggg/7/10/3243.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Quality <em>de Novo</em> Genome Assembly of the <em>Dekkera bruxellensis</em> Yeast Using Nanopore MinION Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Téo Fournier</span>, <span class="highwire-citation-author" data-delta="1">Jean-Sébastien Gounot</span>, <span class="highwire-citation-author" data-delta="2">Kelle Freel</span>, <span class="highwire-citation-author" data-delta="3">Corinne Cruaud</span>, <span class="highwire-citation-author" data-delta="4">Arnaud Lemainque</span>, <span class="highwire-citation-author" data-delta="5">Jean-Marc Aury</span>, <span class="highwire-citation-author" data-delta="6">Patrick Wincker</span>, <span class="highwire-citation-author" data-delta="7">Joseph Schacherer</span> and <span class="highwire-citation-author" data-delta="8">Anne Friedrich</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3243-3250;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300128 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486771--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Téo Fournier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=T%C3%A9o%2BFournier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fournier%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AT%25C3%25A9o%2BFournier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jean-Sébastien Gounot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-S%C3%A9bastien%2BGounot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gounot%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-S%25C3%25A9bastien%2BGounot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kelle Freel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelle%2BFreel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Freel%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKelle%2BFreel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Corinne Cruaud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Commissariat à l’Energie Atomique, Institut de Biologie François-Jacob, Genoscope, BP5706, 91057 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Corinne%2BCruaud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cruaud%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACorinne%2BCruaud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Arnaud Lemainque </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Commissariat à l’Energie Atomique, Institut de Biologie François-Jacob, Genoscope, BP5706, 91057 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arnaud%2BLemainque%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lemainque%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArnaud%2BLemainque%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jean-Marc Aury </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Commissariat à l’Energie Atomique, Institut de Biologie François-Jacob, Genoscope, BP5706, 91057 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Marc%2BAury%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aury%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Marc%2BAury%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Patrick Wincker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Commissariat à l’Energie Atomique, Institut de Biologie François-Jacob, Genoscope, BP5706, 91057 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BWincker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wincker%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BWincker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Joseph Schacherer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BSchacherer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schacherer%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BSchacherer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anne.friedrich@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">anne.friedrich@unistra.fr</a>
<a href="mailto:schacherer@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">schacherer@unistra.fr</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Anne Friedrich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BFriedrich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Friedrich%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnne%2BFriedrich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anne.friedrich@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">anne.friedrich@unistra.fr</a>
<a href="mailto:schacherer@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">schacherer@unistra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/10/3243" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/10/3243&amp;title=High-Quality%20de%20Novo%20Genome%20Assembly%20of%20the%20Dekkera%20bruxellensis%20Yeast%20Using%20Nanopore%20MinION%20Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/10/3243&amp;title=High-Quality%20de%20Novo%20Genome%20Assembly%20of%20the%20Dekkera%20bruxellensis%20Yeast%20Using%20Nanopore%20MinION%20Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/10/3243&amp;title=High-Quality%20de%20Novo%20Genome%20Assembly%20of%20the%20Dekkera%20bruxellensis%20Yeast%20Using%20Nanopore%20MinION%20Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/10/3243&amp;text=High-Quality%20de%20Novo%20Genome%20Assembly%20of%20the%20Dekkera%20bruxellensis%20Yeast%20Using%20Nanopore%20MinION%20Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/10/3243&amp;title=High-Quality%20de%20Novo%20Genome%20Assembly%20of%20the%20Dekkera%20bruxellensis%20Yeast%20Using%20Nanopore%20MinION%20Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/10/3243&amp;t=High-Quality%20de%20Novo%20Genome%20Assembly%20of%20the%20Dekkera%20bruxellensis%20Yeast%20Using%20Nanopore%20MinION%20Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/10/3243&amp;title=High-Quality%20de%20Novo%20Genome%20Assembly%20of%20the%20Dekkera%20bruxellensis%20Yeast%20Using%20Nanopore%20MinION%20Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/10/3243&amp;title=High-Quality%20de%20Novo%20Genome%20Assembly%20of%20the%20Dekkera%20bruxellensis%20Yeast%20Using%20Nanopore%20MinION%20Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/486771/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486771" id="node486771--3" data-pisa="ggg;7/10/3243" data-pisa-master="ggg;g3.117.300128" data-apath="/ggg/7/10/3243.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Quality <em>de Novo</em> Genome Assembly of the <em>Dekkera bruxellensis</em> Yeast Using Nanopore MinION Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Téo Fournier</span>, <span class="highwire-citation-author" data-delta="1">Jean-Sébastien Gounot</span>, <span class="highwire-citation-author" data-delta="2">Kelle Freel</span>, <span class="highwire-citation-author" data-delta="3">Corinne Cruaud</span>, <span class="highwire-citation-author" data-delta="4">Arnaud Lemainque</span>, <span class="highwire-citation-author" data-delta="5">Jean-Marc Aury</span>, <span class="highwire-citation-author" data-delta="6">Patrick Wincker</span>, <span class="highwire-citation-author" data-delta="7">Joseph Schacherer</span> and <span class="highwire-citation-author" data-delta="8">Anne Friedrich</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3243-3250;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300128 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486771--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Téo Fournier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=T%C3%A9o%2BFournier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fournier%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AT%25C3%25A9o%2BFournier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jean-Sébastien Gounot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-S%C3%A9bastien%2BGounot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gounot%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-S%25C3%25A9bastien%2BGounot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kelle Freel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelle%2BFreel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Freel%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKelle%2BFreel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Corinne Cruaud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Commissariat à l’Energie Atomique, Institut de Biologie François-Jacob, Genoscope, BP5706, 91057 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Corinne%2BCruaud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cruaud%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACorinne%2BCruaud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Arnaud Lemainque </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Commissariat à l’Energie Atomique, Institut de Biologie François-Jacob, Genoscope, BP5706, 91057 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arnaud%2BLemainque%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lemainque%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArnaud%2BLemainque%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jean-Marc Aury </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Commissariat à l’Energie Atomique, Institut de Biologie François-Jacob, Genoscope, BP5706, 91057 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Marc%2BAury%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aury%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Marc%2BAury%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Patrick Wincker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Commissariat à l’Energie Atomique, Institut de Biologie François-Jacob, Genoscope, BP5706, 91057 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BWincker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wincker%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BWincker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Joseph Schacherer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BSchacherer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schacherer%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BSchacherer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anne.friedrich@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">anne.friedrich@unistra.fr</a>
<a href="mailto:schacherer@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">schacherer@unistra.fr</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Anne Friedrich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Moléculaire, Génomique, Microbiologie, Unité Mixte de Recherche 7156, University of Strasbourg, Centre National de la Recherche Scientifique, F-67000, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BFriedrich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Friedrich%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnne%2BFriedrich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anne.friedrich@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">anne.friedrich@unistra.fr</a>
<a href="mailto:schacherer@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">schacherer@unistra.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3243&amp;rft.epage=3250&amp;rft.atitle=High-Quality%20de%20Novo%20Genome%20Assembly%20of%20the%20Dekkera%20bruxellensis%20Yeast%20Using%20Nanopore%20MinION%20Sequencing&amp;rft.volume=7&amp;rft.issue=10&amp;rft.date=2017-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Fournier%2C+T%C3%A9o&amp;rft.au=Gounot%2C+Jean-S%C3%A9bastien&amp;rft.au=Freel%2C+Kelle&amp;rft.au=Cruaud%2C+Corinne&amp;rft.au=Lemainque%2C+Arnaud&amp;rft.au=Aury%2C+Jean-Marc&amp;rft.au=Wincker%2C+Patrick&amp;rft.au=Schacherer%2C+Joseph&amp;rft.au=Friedrich%2C+Anne"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/486771/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/486771/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/486771/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/486771/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/486771/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/486771/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/486771/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/486771/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/486771/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/486771/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/486771/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/486771/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496037" id="node496037" data-pisa="ggg;g3.120.401071v1" data-pisa-master="ggg;g3.120.401071" data-apath="/ggg/early/2020/03/11/g3.120.401071.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/11/g3.120.401071" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the Meadow Brown Butterfly, <em>Maniola jurtina</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/10/3243" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/10/3243.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/10/3243.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/10/3243.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/10/3243.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
