<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/10/3349" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/10/3349.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/10/3349.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/10/3349.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/10/3349/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/10/3349.atom" />
<meta name="HW.pisa" content="ggg;7/10/3349" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="KinFin: Software for Taxon-Aware Analysis of Clustered Protein Sequences" />
<meta name="DC.Identifier" content="10.1534/g3.117.300233" />
<meta name="DC.Date" content="2017-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Laetsch and Blaxter. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The field of comparative genomics is concerned with the study of similarities and differences between the information encoded in the genomes of organisms. A common approach is to define gene families by clustering protein sequences based on sequence similarity, and analyze protein cluster presence and absence in different species groups as a guide to biology. Due to the high dimensionality of these data, downstream analysis of protein clusters inferred from large numbers of species, or species with many genes, is nontrivial, and few solutions exist for transparent, reproducible, and customizable analyses. We present KinFin, a streamlined software solution capable of integrating data from common file formats and delivering aggregative annotation of protein clusters. KinFin delivers analyses based on systematic taxonomy of the species analyzed, or on user-defined, groupings of taxa, for example, sets based on attributes such as life history traits, organismal phenotypes, or competing phylogenetic hypotheses. Results are reported through graphical and detailed text output files. We illustrate the utility of the KinFin pipeline by addressing questions regarding the biology of filarial nematodes, which include parasites of veterinary and medical importance. We resolve the phylogenetic relationships between the species and explore functional annotation of proteins in clusters in key lineages and between custom taxon sets, identifying gene families of interest. KinFin can easily be integrated into existing comparative genomic workflows, and promotes transparent and reproducible analysis of clustered protein data." />
<meta name="DC.Contributor" content="Dominik R. Laetsch" />
<meta name="DC.Contributor" content="Mark L. Blaxter" />
<meta name="article:published_time" content="2017-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="KinFin: Software for Taxon-Aware Analysis of Clustered Protein Sequences" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The field of comparative genomics is concerned with the study of similarities and differences between the information encoded in the genomes of organisms. A common approach is to define gene families by clustering protein sequences based on sequence similarity, and analyze protein cluster presence and absence in different species groups as a guide to biology. Due to the high dimensionality of these data, downstream analysis of protein clusters inferred from large numbers of species, or species with many genes, is nontrivial, and few solutions exist for transparent, reproducible, and customizable analyses. We present KinFin, a streamlined software solution capable of integrating data from common file formats and delivering aggregative annotation of protein clusters. KinFin delivers analyses based on systematic taxonomy of the species analyzed, or on user-defined, groupings of taxa, for example, sets based on attributes such as life history traits, organismal phenotypes, or competing phylogenetic hypotheses. Results are reported through graphical and detailed text output files. We illustrate the utility of the KinFin pipeline by addressing questions regarding the biology of filarial nematodes, which include parasites of veterinary and medical importance. We resolve the phylogenetic relationships between the species and explore functional annotation of proteins in clusters in key lineages and between custom taxon sets, identifying gene families of interest. KinFin can easily be integrated into existing comparative genomic workflows, and promotes transparent and reproducible analysis of clustered protein data.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/10/01" />
<meta name="citation_mjid" content="ggg;7/10/3349" />
<meta name="citation_id" content="7/10/3349" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/10/3349" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/10/3349.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/10/3349.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/10/3349.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300233" />
<meta name="citation_pmid" content="28866640" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3349" />
<meta name="citation_lastpage" content="3357" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Dominik R. Laetsch" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh, EH9 3JT, United Kingdom" />
<meta name="citation_author_institution" content="The James Hutton Institute, DD2 5DA Dundee, United Kingdom" />
<meta name="citation_author_email" content="dominik.laetsch@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7887-0186" />
<meta name="citation_author" content="Mark L. Blaxter" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh, EH9 3JT, United Kingdom" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2861-949X" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Immunology;citation_journal_abbrev=Nat Rev Immunol;citation_author=W. S. Alexander;citation_title=Suppressors of cytokine signalling (SOCS) in the immune system.;citation_pages=410-416;citation_volume=2;citation_year=2002;citation_issue=6;citation_pmid=12093007;citation_doi=10.1038/nri818" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=H. Antonicka;citation_author=A. Mattman;citation_author=C. G. Carlson;citation_author=D. M. Glerum;citation_author=K. C. Hoffbuhr;citation_title=Mutations in COX15 produce a defect in the mitochondrial heme biosynthetic pathway, causing early-onset fatal hypertrophic cardiomyopathy.;citation_pages=101-114;citation_volume=72;citation_year=2003;citation_issue=1;citation_pmid=12474143;citation_doi=10.1086/345489" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Camacho;citation_author=G. Coulouris;citation_author=V. Avagyan;citation_author=N. Ma;citation_author=J. Papadopoulos;citation_title=BLAST+: architecture and applications.;citation_pages=421-421;citation_volume=10;citation_year=2009;citation_pmid=20003500;citation_doi=10.1186/1471-2105-10-421" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Capella-Gutiérrez;citation_author=J. M. Silla-Martínez;citation_author=T. Gabaldón;citation_title=Trimal: a tool for automated alignment trimming in large-scale phylogenetic analyses.;citation_volume=25;citation_year=2009;citation_pmid=19505945;citation_doi=10.1093/bioinformatics/btp348" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=N. M. Chaudhari;citation_author=V. K. Gupta;citation_author=C. Dutta;citation_title=BPGA—an ultra-fast pan-genome analysis pipeline.;citation_pages=24373;citation_volume=6;citation_year=2016;citation_pmid=27071527;citation_doi=10.1038/srep24373" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. Chen;citation_author=T. K. Samuel;citation_author=J. Sinclair;citation_author=H. A. Dailey;citation_author=I. Hamza;citation_title=An intercellular heme-trafficking protein delivers maternal heme to the embryo during development in C. elegans.;citation_pages=720-731;citation_volume=145;citation_year=2011;citation_issue=5;citation_pmid=21620137;citation_doi=10.1016/j.cell.2011.04.025" />
<meta name="citation_reference" content="citation_journal_title=International journal for parasitology;citation_journal_abbrev=Int J Parasitol;citation_author=B. Elsworth;citation_author=J. Wasmuth;citation_author=M. Blaxter;citation_title=NEMBASE4: the nematode transcriptome resource.;citation_pages=881-894;citation_volume=41;citation_year=2011;citation_issue=8;citation_pmid=21550347;citation_doi=10.1016/j.ijpara.2011.03.009" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. M. Emms;citation_author=S. Kelly;citation_title=Orthofinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy.;citation_pages=157;citation_volume=16;citation_year=2015;citation_pmid=26243257;citation_doi=10.1186/s13059-015-0721-2" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. J. Enright;citation_author=V. Kunin;citation_author=C. A. Ouzounis;citation_title=Protein families and TRIBES in genome sequence space;citation_pages=4632-4638;citation_volume=31;citation_year=2003;citation_issue=15;citation_pmid=12888524;citation_doi=10.1093/nar/gkg495" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=J. S. Farris;citation_title=Phylogenetic analysis under Dollo’s law.;citation_volume=26;citation_year=1977;citation_doi=10.1093/sysbio/26.1.77" />
<meta name="citation_reference" content="citation_journal_title=FASEB J.;citation_author=C. Godel;citation_author=S. Kumar;citation_author=G. Koutsovoulos;citation_author=P. Ludin;citation_author=D. Nilsson;citation_title=The genome of the heartworm, Dirofilaria immitis, reveals drug and vaccine targets.;citation_volume=26;citation_year=2012;citation_pmid=22889830;citation_doi=10.1096/fj.12-205096" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=P. González-Cabo;citation_author=A. Bolinches-Amorós;citation_author=J. Cabello;citation_author=S. Ros;citation_author=S. Moreno;citation_title=Disruption of the ATP-binding cassette B7 (ABTM-1/ABCB7) induces oxidative stress and premature cell death in Caenorhabditis elegans.;citation_volume=286;citation_year=2011;citation_pmid=21464130;citation_doi=10.1074/jbc.M110.211201" />
<meta name="citation_reference" content="citation_journal_title=Infection and Immunity;citation_journal_abbrev=Infect. Immun.;citation_author=W. F. Gregory;citation_author=A. K. Atmadja;citation_author=J. E. Allen;citation_author=R. M. Maizels;citation_title=The Abundant Larval Transcript-1 and -2 Genes of Brugia malayi Encode Stage-Specific Candidate Vaccine Antigens for Filariasis;citation_pages=4174-4179;citation_volume=68;citation_year=2000;citation_issue=7;citation_pmid=10858234;citation_doi=10.1128/IAI.68.7.4174-4179.2000" />
<meta name="citation_reference" content="citation_journal_title=Database;citation_journal_abbrev=Database;citation_author=J. Herrero;citation_author=M. Muffato;citation_author=K. Beal;citation_author=S. Fitzgerald;citation_author=L. Gordon;citation_title=Ensembl comparative genomics resources;citation_pages=bav096-bav096;citation_volume=2016;citation_year=2016;citation_issue=0;citation_pmid=26896847;citation_doi=10.1093/database/bav096" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. L. Howe;citation_author=B. J. Bolt;citation_author=S. Cain;citation_author=J. Chan;citation_author=W. J. Chen;citation_title=Wormbase 2016: expanding to enable helminth genomic research.;citation_volume=44;citation_year=2016;citation_pmid=26578572;citation_doi=10.1093/nar/gkv1217" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biochem. Parasitol.;citation_author=K. L. Howe;citation_author=B. J. Bolt;citation_author=M. Shafie;citation_author=P. Kersey;citation_author=M. Berriman;citation_title=Wormbase parasite—a comprehensive resource for helminth genomics.;citation_pages=2-10;citation_volume=215;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Z. Hu;citation_author=C. Sun;citation_author=K.-c. Lu;citation_author=X. Chu;citation_author=Y. Zhao;citation_title=EUPAN enables pan-genome studies of a large number of eukaryotic genomes.;citation_pages=2408-2409;citation_volume=33;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Huerta-Cepas;citation_author=S. Capella-Gutiérrez;citation_author=L. P. Pryszcz;citation_author=M. Marcet-Houben;citation_author=T. Gabaldón;citation_title=Phylomedb v4: zooming into the plurality of evolutionary histories of a genome.;citation_volume=42;citation_year=2014;citation_pmid=24275491;citation_doi=10.1093/nar/gkt1177" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Jones;citation_author=D. Binns;citation_author=H.-Y. Chang;citation_author=M. Fraser;citation_author=W. Li;citation_title=Interproscan 5: genome-scale protein function classification.;citation_volume=30;citation_year=2014;citation_pmid=24451626;citation_doi=10.1093/bioinformatics/btu031" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Katoh;citation_author=D. M. Standley;citation_title=Mafft multiple sequence alignment software version 7: improvements in performance and usability.;citation_volume=30;citation_year=2013;citation_pmid=23329690;citation_doi=10.1093/molbev/mst010" />
<meta name="citation_reference" content="citation_journal_title=Evol. Bioinform. Online;citation_author=K. M. Kocot;citation_author=M. R. Citarella;citation_author=L. L. Moroz;citation_author=K. M. Halanych;citation_title=Phylotreepruner: a phylogenetic tree-based approach for selection of orthologous sequences for phylogenomics.;citation_pages=429-435;citation_volume=9;citation_year=2013;citation_pmid=24250218" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=P. Kück;citation_author=K. Meusemann;citation_title=Fasconcat: convenient handling of data matrices.;citation_pages=1115-1118;citation_volume=56;citation_year=2010;citation_pmid=20416383;citation_doi=10.1016/j.ympev.2010.04.024" />
<meta name="citation_reference" content="citation_journal_title=PeerJ;citation_author=E. Lefoulon;citation_author=O. Bain;citation_author=B. L. Makepeace;citation_author=C. d’Haese;citation_author=S. Uni;citation_title=Breakdown of coevolution between symbiotic bacteria Wolbachia and their filarial hosts.;citation_pages=e1840;citation_volume=4;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=L. Li;citation_author=C. J. Stoeckert;citation_author=D. S. Roos;citation_title=OrthoMCL: Identification of Ortholog Groups for Eukaryotic Genomes;citation_pages=2178-2189;citation_volume=13;citation_year=2003;citation_issue=9;citation_pmid=12952885;citation_doi=10.1101/gr.1224503" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. N. Luck;citation_author=C. C. Evans;citation_author=M. D. Riggs;citation_author=J. M. Foster;citation_author=A. R. Moorhead;citation_title=Concurrent transcriptional profiling of Dirofilaria immitis and its Wolbachia endosymbiont throughout the nematode life cycle reveals coordinated gene expression.;citation_pages=1041;citation_volume=15;citation_year=2014;citation_pmid=25433394;citation_doi=10.1186/1471-2164-15-1041" />
<meta name="citation_reference" content="citation_journal_title=FASEB J.;citation_author=A. N. Luck;citation_author=X. Yuan;citation_author=D. Voronin;citation_author=B. E. Slatko;citation_author=I. Hamza;citation_title=Heme acquisition in the parasitic filarial nematode Brugia malayi.;citation_volume=30;citation_year=2016;citation_pmid=27363426;citation_doi=10.1096/fj.201600603R" />
<meta name="citation_reference" content="citation_journal_title=Parasitology;citation_author=S. A. Nadler;citation_author=R. A. Carreno;citation_author=H. Mejía-Madrid;citation_author=J. Ullberg;citation_author=C. Pagan;citation_title=Molecular phylogeny of clade III nematodes reveals multiple origins of tissue parasitism.;citation_pages=1421-1442;citation_volume=134;citation_year=2007;citation_pmid=17506928;citation_doi=10.1017/S0031182007002880" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=E. Nagayasu;citation_author=S. A. Ishikawa;citation_author=S. Taketani;citation_author=G. Chakraborty;citation_author=A. Yoshida;citation_title=Identification of a bacteria-like ferrochelatase in Strongyloides venezuelensis, an animal parasitic nematode.;citation_pages=e58458;citation_volume=8;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J.-K. Park;citation_author=T. Sultana;citation_author=S.-H. Lee;citation_author=S. Kang;citation_author=H. K. Kim;citation_title=Monophyly of clade III nematodes is not supported by phylogenetic analysis of complete mitochondrial genome sequences.;citation_pages=392-392;citation_volume=12;citation_year=2011;citation_pmid=21813000" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=T. N. Petersen;citation_author=S. Brunak;citation_author=G. von Heijne;citation_author=H. Nielsen;citation_title=SignalP 4.0: discriminating signal peptides from transmembrane regions.;citation_pages=785-786;citation_volume=8;citation_year=2011;citation_issue=10;citation_pmid=21959131;citation_doi=10.1038/nmeth.1701" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Rajagopal;citation_author=A. U. Rao;citation_author=J. Amigo;citation_author=M. Tian;citation_author=S. K. Upadhyay;citation_title=Haem homeostasis is regulated by the conserved and concerted functions of HRG-1 proteins.;citation_pages=1127-1131;citation_volume=453;citation_year=2008;citation_issue=7198;citation_pmid=18418376;citation_doi=10.1038/nature06934" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. U. Rao;citation_author=L. K. Carta;citation_author=E. Lesuisse;citation_author=I. Hamza;citation_title=Lack of heme synthesis in a free-living eukaryote.;citation_volume=102;citation_year=2005;citation_pmid=15767563;citation_doi=10.1073/pnas.0500877102" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Severance;citation_author=A. Rajagopal;citation_author=A. U. Rao;citation_author=G. C. Cerqueira;citation_author=M. Mitreva;citation_title=Genome-wide analysis reveals novel genes essential for heme homeostasis in Caenorhabditis elegans.;citation_pages=e1001044-e1001044;citation_volume=6;citation_year=2010;citation_issue=7;citation_pmid=20686661;citation_doi=10.1371/journal.pgen.1001044" />
<meta name="citation_reference" content="citation_journal_title=Biometals;citation_author=J. Sinclair;citation_author=I. Hamza;citation_title=Lessons from bloodless worms: heme homeostasis in C. elegans.;citation_pages=481-489;citation_volume=28;citation_year=2015;citation_pmid=25724951;citation_doi=10.1007/s10534-015-9841-0" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=G. Song;citation_author=B. J. A. Dickins;citation_author=J. Demeter;citation_author=S. Engel;citation_author=J. Gallagher;citation_title=Agape (automated genome analysis pipeline) for pan-genome analysis of Saccharomyces cerevisiae.;citation_pages=e0120671;citation_volume=10;citation_year=2015;citation_pmid=25781462;citation_doi=10.1371/journal.pone.0120671" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=Raxml version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.;citation_volume=30;citation_year=2014;citation_pmid=24451623;citation_doi=10.1093/bioinformatics/btu033" />
<meta name="citation_reference" content="citation_journal_title=Proteins;citation_journal_abbrev=Proteins;citation_author=R. Unger;citation_author=S. Uliel;citation_author=S. Havlin;citation_title=Scaling law in sizes of protein sequence families: from super-families to orphan genes.;citation_pages=569-576;citation_volume=51;citation_year=2003;citation_issue=4;citation_pmid=12784216;citation_doi=10.1002/prot.10347" />
<meta name="citation_reference" content="citation_journal_title=Methods Mol. Biol.;citation_author=P. Vinuesa;citation_author=B. Contreras-Moreira;citation_title=Robust identification of orthologues and paralogues for microbial pan-genomics using get_homologues: a case study of pinca/c plasmids.;citation_pages=203-232;citation_volume=1231;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=A. D. Winter;citation_author=G. McCormack;citation_author=J. Myllyharju;citation_author=A. P. Page;citation_title=Prolyl 4-hydroxlase activity is essential for development and cuticle formation in the human infective parasitic nematode Brugia malayi.;citation_volume=288;citation_year=2013;citation_pmid=23223450;citation_doi=10.1074/jbc.M112.397604" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. Wu;citation_author=J. Novelli;citation_author=D. Jiang;citation_author=H. A. Dailey;citation_author=F. Landmann;citation_title=Interdomain lateral gene transfer of an essential ferrochelatase gene in human parasitic nematodes.;citation_volume=110;citation_year=2013;citation_pmid=23610429;citation_doi=10.1073/pnas.1304049110" />
<meta name="citation_reference" content="citation_journal_title=Genomics Proteomics Bioinformatics;citation_author=J. Xiao;citation_author=Z. Zhang;citation_author=J. Wu;citation_author=J. Yu;citation_title=A brief review of software tools for pangenomics.;citation_pages=73-76;citation_volume=13;citation_year=2015;citation_pmid=25721608;citation_doi=10.1016/j.gpb.2015.01.007" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=Y. Yoshida;citation_author=G. Koutsovoulos;citation_author=D. R. Laetsch;citation_author=L. Stevens;citation_author=S. Kumar;citation_title=Comparative genomics of the tardigrades Hypsibius dujardini and Ramazzottius varieornatus.;citation_pages=1-40;citation_volume=15;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. M. Zdobnov;citation_author=F. Tegenfeldt;citation_author=D. Kuznetsov;citation_author=R. M. Waterhouse;citation_author=F. A. Simão;citation_title=Orthodb v9.1: cataloging evolutionary and functional annotations for animal, fungal, plant, archaeal, bacterial and viral orthologs.;citation_volume=45;citation_year=2017;citation_pmid=27899580;citation_doi=10.1093/nar/gkw1119" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="KinFin: Software for Taxon-Aware Analysis of Clustered Protein Sequences" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
<meta name="twitter:description" content="The field of comparative genomics is concerned with the study of similarities and differences between the information encoded in the genomes of organisms. A common approach is to define gene families by clustering protein sequences based on sequence similarity, and analyze protein cluster presence and absence in different species groups as a guide to biology. Due to the high dimensionality of these data, downstream analysis of protein clusters inferred from large numbers of species, or species with many genes, is nontrivial, and few solutions exist for transparent, reproducible, and customizable analyses. We present KinFin, a streamlined software solution capable of integrating data from common file formats and delivering aggregative annotation of protein clusters. KinFin delivers analyses based on systematic taxonomy of the species analyzed, or on user-defined, groupings of taxa, for example, sets based on attributes such as life history traits, organismal phenotypes, or competing phylogenetic hypotheses. Results are reported through graphical and detailed text output files. We illustrate the utility of the KinFin pipeline by addressing questions regarding the biology of filarial nematodes, which include parasites of veterinary and medical importance. We resolve the phylogenetic relationships between the species and explore functional annotation of proteins in clusters in key lineages and between custom taxon sets, identifying gene families of interest. KinFin can easily be integrated into existing comparative genomic workflows, and promotes transparent and reproducible analysis of clustered protein data." />
<meta name="og-title" property="og:title" content="KinFin: Software for Taxon-Aware Analysis of Clustered Protein Sequences" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/10/3349" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The field of comparative genomics is concerned with the study of similarities and differences between the information encoded in the genomes of organisms. A common approach is to define gene families by clustering protein sequences based on sequence similarity, and analyze protein cluster presence and absence in different species groups as a guide to biology. Due to the high dimensionality of these data, downstream analysis of protein clusters inferred from large numbers of species, or species with many genes, is nontrivial, and few solutions exist for transparent, reproducible, and customizable analyses. We present KinFin, a streamlined software solution capable of integrating data from common file formats and delivering aggregative annotation of protein clusters. KinFin delivers analyses based on systematic taxonomy of the species analyzed, or on user-defined, groupings of taxa, for example, sets based on attributes such as life history traits, organismal phenotypes, or competing phylogenetic hypotheses. Results are reported through graphical and detailed text output files. We illustrate the utility of the KinFin pipeline by addressing questions regarding the biology of filarial nematodes, which include parasites of veterinary and medical importance. We resolve the phylogenetic relationships between the species and explore functional annotation of proteins in clusters in key lineages and between custom taxon sets, identifying gene families of interest. KinFin can easily be integrated into existing comparative genomic workflows, and promotes transparent and reproducible analysis of clustered protein data." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
    <title>KinFin: Software for Taxon-Aware Analysis of Clustered Protein Sequences | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"G07009al61M5UD2hl70g3JLkSVRvKMmWu84nxUcOhnI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"486779","apath":"\/ggg\/7\/10\/3349.atom","pisa":"ggg;7\/10\/3349","ac":{"\/ggg\/7\/10\/3349.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/10\/3349.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/10\/3349"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/10\/3349"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/10\/3349":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222AJTSlbDnFTfbtFVZDzXAwyED\\\\\\\/Xacta2Wo3D2TcFsgYfcuftxSHIatjv7OujKUhDardIAeH3mDfvzXiYTJ+BpKuiv6iXvm2LuzQcyz5yng72Z29SETBaLaq5BESFxWnpgyEkAqlWGOquBy4xvE8wwo6PjBNW2WHfVwopB2e1pzReSRJVg+m5NFoPRRE6QDqK790WoOTRfTEzIA7qZR92iNFZ\\\\\\\/s3FhQzCTuHlDB+aY0xleZrNjWkXP1x+mG2Vss0aipZg1ntJhFrk3mLBBkXVYaqDzbEu0pVN9fFCFASBKQLXjBnx7XGnyrE0AwKgz8ZKSeVCpaAhIyHYXWEduYi1ClJeArLoZjd2uySddOyLihbgWp1tbWBH0J+Bl+Q6sntIrePIj2Atn3FaP7EqTPPpcaCBPh4HnzKFAbSL2fGbJg8WKYkd\\\\\\\/wa44JESokw0objWEW4COnjJQy7763Z8lJdT8KkbMLcWBXk2qHbNVfUPnaSOqlTuVBaULR2YtSVnzv3lkbqbzqmKy1lTXaQ1zt7Y+fW\\\\\\\/UfU38qMx2jD+SeUM2joGchCzCJQQUpJDgYfmuzMTPbhrCK5nBNwDXJ1NW6Ot09PsDeIcZvAjZeq8n2invx8oFPVrUiEz42BJXDkUOTshJeXR6WOAR2u8xZxzIrit9N\\\\\\\/BPlLUKhMYPqZPyR1DrdOm7V5SJMBBNS72THchRwya6YPOlSiKX2YaVFJ8EKjiOzZBXvOyoniKH6bdubBDyVpdphr1KuHvjUIXCHB\\\\\\\/RxX9lX8PyTZwOsiHrFd7\\\\\\\/tykxoIFv6fXNBjCIBxPD93J\\\\\\\/Qb4NAkqx7\\\\\\\/2arD3n2ysH8GIw+UHi7B3Sw8+7dlVRe1Df\\\\\\\/b89CiYHmSc9riv\\\\\\\/4NZ4hOCDA0JUjjf1nuwOseoCfkGhWAso2EjosCM+EZITUOlUGUhuH4Ed9+obOqrA4+JOrZarz2wv4ZCh+X\\\\\\\/zidetwwmzhRyxzGZ2G1\\\\\\\/IG0lP558or4VTF7Htb3ya1fkhvqRRoI4QQYd0NImFxLkhBcgLbRu4yAGYr\\\\\\\/ssn9YbceGdZ8UHedF9pb50shHmgn9tDCFXQOrh7daVP\\\\\\\/ZnNzh36vrzd3LtMYh4PM+NRwzZCUU3GYfyq7eoKP09bb3eSbEfZtzpFEmpYCZuHaJ+ge5jdazr6prLQYt\\\\\\\/Vd\\\\\\\/p7MheWXKKP\\\\\\\/olrNJ9wOgbVCrpgEQ+f5XJZEQGWHnUtEh1ri8JduqdiHhaLk6i4kAs4WHi8pgn9KUebpLnaMXU8kOUjj2bLIPCr7mI7Gykuqr7aUkwV9DePLJ\\\u0022,\\\u0022iv\\\u0022:\\\u0022j30mwCYMwirM47tMNkHcYQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a943d712f8dec3cb58a1e70c0e02091894229237ceafb8dde6ed0017072c5d25\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DOeWdcezUPwxy4365seViX85sEvmQbQ\\\\\\\/Q3Ac+EalT9AAOKPEtrPeoQXlHVmAUDMFLcMLW1bmCr8Y67gYwkHq28Q8H8ppf7PH0tY6llmqpq+YJ16XQXUmxL2DlGlmeDdMp5QlXEZGvb1oZNugN389CZqLHCtSn5xios3EqkHeLCPI+GG4SP1HDP8z5CDbc2WDuf1orT\\\\\\\/Erg3Un4x\\\\\\\/MvSIhSrzBScaVqCNfFRvCNT1n\\\\\\\/dXvndWc2Qdd7YhMfpsxw+op8IL2bh5QxMwbHzD17SRfEYjnp2lgLgpNoreWz35\\\\\\\/+KxoDY4PsCxDpRNMeX4v+Uo8ppnbGn7K2K7YyKTsJaGakyl7Rv5hHb4pMC2CJqFiITpc7+pxoQGrLXkzSIp0bVj7l9b9z7R+q2DxA0kdC5NrZuhfUgEXV0LnQOtcwNYDK5OF4VEtxNlgcoO+GXDjf1DSG1vI32DnPWw36QBa1mY3whhhy2joFW5CcR2eqzc+Za2i6cv3e\\\\\\\/X\\\\\\\/Ol5MsRFt9iFnQW++0VVSgxyVm4YvLH3cVwvSS2LqmvRKWVTyMf73xXvTaAsRROOG2oymi5PIykTQiFkUtNvJPJT3xgyp9+OKa1XbKIkjNiOC7Ynv4B+2bVRPNWdBp8HXe8R35UUwDbhC5NHdZH+ZOs9BnVbLoVE0lsJ05k7A+uxCwinsYz488XkTYWr7czsmauIU0jc3XCEGG8iRlLxK7g3GhKrQs+Ln3iUcMafWNXz1kLWpiOY\\\\\\\/Hu3toQxUVcdWFnA957fkUoFwkniCC4gtcafH\\\\\\\/wyja0lDlNrgjqiICh4ke8PI3FFTtJ5x7LDD3YBHIgpOKdJJZCKF+30smpVyxr1fuPYo\\\\\\\/jNgQXQ5\\\\\\\/uN6Qiy8OVx5sPmmgcs0GzMt6DNH6+a8O6EatJ4frUsSWyqSdvIZHhvbbPV6QqA8eOOXvbJurP6DTt79hT3oeKaUX4lORaDaZlsCstJyr2SzB70Ztn2tCw710sebuYmjOZ58giqya3sQUN8fr4MW3qK+poRmOtrmSuGnwiwNYzLHWHRDn222sOmbxUpAMf52s+YywGaMwv3U+6sFEbCpt80CIVDLQz3o\\\\\\\/gaJ5\\\\\\\/9p\\\\\\\/sBmJVuZ5dEaoahFUXLQ3IhB+VFJAApn8yqAXbIqEifDMmKchtSb3PCBrI\\\\\\\/J5GsW06G9sSo9z7SjcKliNKPGhGcR\\\\\\\/pHuxyQiUFckqW8\\\\\\\/M2PH7lmN+UCT8bCbAKbIHueVzZHJQbM65vktK76yAZbSfqAVvxAi7dwrk1GnvxCNv+58iZTB4nYxRu4t28MqSAJ65qw12GmmEEFySXN4j\\\\\\\/DzzP63UFnPt6uslDUaZi1r6BdmC6R+uOA8vtvB+2vfXz15kLfxjxsyEfetnlOkICqZYvLqXN53M3PJs7jOiSVUSKKwWBcMese570v3Vrl2qsojIkiWeZ3H57OTGQj14I31KWFFZKtopcb2sJmB0ci9bPT8Ip0w28snA5N77+fSFdWSzw0QhQ34a8Yl+RzTi\\\\\\\/ZK3sPCKKQeofQyBO0yBoJtYuk4s11BaNh7EbWnMaBD8Mon7fiwlMO5EH4R70PD7s9W\\\\\\\/h\\\\\\\/TC1kn\\\\\\\/nUVIANNGLv0G\\\\\\\/GHIFCGSGBgpevI+R3ai1ecP4ZuKmuMGXOlfxIzEf215ljNWkLLi1AAcQoQWku4t6M+2PCu\\\\\\\/DYEIjDA7ZjtlAn6ojGzSbVHbp9qeOFAVitQifse4ctK5SnepSIEGOJJ0Iy0whirHXEUm9qV5O\\\\\\\/xoETEqZhc7z3O4UpPgGS7qHRBnG0A0m6VSqp1EVqNyajIz1bdhPwQW0kMKxmsxm7KJeVoK1n7Obb\\\\\\\/EdStddhmaEorMHCQpFE3nwWba\\\\\\\/fM3SHCBvxKekcVkvcz8UcsV7pni7ZU6X4bO6MmSPHkBf6IaI6535CQUKdcveVpQpFPoDH1rZq446xO4gMy7pZK+EUGy4zoh20t+54PsvfmKKxlEhcyJP2LLpclC4HUb4TlSPGY9hOAVvdslLgp3tFs0GvZp4KOpNUoDpbcPGuze+UDnEzDN9bG0NiaM1ueQT04wY4Yl7f2+7zgyzneDM1gnsoUJkNFj7n2QcOTZBQROZOmnGExLMD0JgVWRaLNwMSTJ8W0vVw6V8VxPajwXBzqpP5+DigaaZdsYH+ob4KUBw7RiBRo5GCgTTvnJxrNONTqRPpydfDZdJ2YtScjWYXm1KuWIGJwI1zi76xrULgJthJm7VBBqBLthembfP\\\\\\\/UY7eolcTV+tvgyBqyRfqDzi5\\\\\\\/\\\\\\\/dm0Dbutq0HpAbo258UN4YnbNJjFzj\\\\\\\/c+k85M3DSlbTEskmNE\\\\\\\/lvfwgq6C74Mz0deWU5NIiKX8kBs47c5sk6Pv5+STO7F7RgvpMBDtJTiGxhqRQJ0die\\\\\\\/sL87F68JZWxn6MnuJKk4CmcNYMc6A+wO5IHZAPxdAt6C6h97HqAgTSTGCDe+LlL8b5+VlpyBD\\\\\\\/g5Gju3OHyu3m77O622vz2t2QG+Qkdnyow3JozPea0h+jevUIGMXU5iZFDCKhwkumzSgYSQbCoz72DgYJXk3U34YF1BMXR9fP9OuhJwpfXgMNK+EtkDJ9JdIojhBMMlaRndBr3U4PTN4tR7DPF8coOGV9mON0\\\\\\\/ofY8qpEbFfj+v1uzANYugJKiEOKGozOXBK83qa3Vtd6OG0q0ASrxqu3FkKlVRU1UOAPgyTPMNfb02\\\\\\\/d4ymMrLRQGuNoBFX1jfElGfy+36VjctFQINqPh3Dg0Y4WEwOiM1mxgrBKMsYAYajXp0jnlrPmD3latEmh1Is47yvSRmiGAEtR2H4eol7+8+KVRWOeznQV9IxP5lBZQyVMl8A5i4OPIe3USg90LjFsCczp5o6Qr2LFyjrLQ0YAOIIfXmhImWYPeMyiacWVilbcEfgytLijBsO1mdzyrwYcI5WPg9m2XAAL6jCw+NW0HxCSq1v9+LC8DheF3rYkzSMis\\\\\\\/RunGJHXX9QkPePvFXG7vdy7kUPyt7WGgPatKv+C3sobm8msqKLzfRsAHpiVLBM\\\\\\\/L7+VZAs3L2F68xCJUzpObCO53WO0b5Wxukmnzdi+YZr6ASp62AlT28MB3CqaaH8+wXETumlNdN8ft+c+oqoL3wu7vzlV\\\\\\\/WSxSGkZO\\\\\\\/71LSeh7EXq+JrF6XzyJNhOJhP+Xgi4BPWSXpXYC4VHKxFF4V07oI+BDQ9kyeZAWzfGN2J6X3kA9RmciIWcPvdVkcOfAF+daNHa8cW1wlrgHbrw8IjMNri7vABLULPDH8x\\\\\\\/Z+fyK8nJVOJpiYW9Hg2OM2vOqrjRpqJIdUWTr0ua7UO\\\\\\\/osrjZ\\\\\\\/DLqBOPPWr\\\\\\\/kuRxzWtvBXzZXsPpHQ+3yili5H\\\\\\\/PffeZakb3HFbx9sZbl7ACDNwFR5Ky2F6ihp74bBBMHjnHmz7agI6bO67BWi\\\\\\\/i1xhsb29EfNFojuzJovddIL+BRfCr33l7Ap\\\\\\\/0K0OQEzBl+CGsBCTP3TusSF+SxVhKBKeraaCsDG7FJjYuuwAbX5XoaznLLMz1rXf5QkA3mnNGRzBIraksk0JlbsthgHmsji4D9m5E\\\\\\\/liGduEyWB4uakEY0vPwbMT9jBn1alvJUoT5zwvzXFUIgQsZiN1gQpU0u8Yb1zMuAcz1GfWRK+ZN5FXekdsLSNlgqah67SydDf5wRCoMnSQwfgmhzfkWqlffUo0ULrFVknt7rD9+nvXfraH1NRf3htnHrdgT2uXnirUb2DQA3oOnRvksYXYDDR5+VXWIuqvxLyOo9r1JqFQ8QG8HmepAAmoU8jyHCTcAfGFklVvtAojP+ANMazARXSsqS9LAGya4v+Gw0QS1L98FaUSTbcVTbwk9BYICNIuRisGCGsxuBCa1zTKgxMU5YBh7ppiE53rH5XS\\\\\\\/3ESGKOe1oIj9ZH6DEeTC9N6xj667givXGW4l4e8SIOBCTnr3fm0gn04tHuKqbSUAPBApEouyZSgYEdATjYEEeDJxJlJUEyNz7Iumm3zxQOpH8WwNbIIktXG9KzfJip0WCNUQ8342zqdsynKqFS4CNQDEyFMqs2ROTd7+taQkIj4RvzGQB8edKCVgxgP73ajJW\\\\\\\/jTaEmUHPFHD+3rLDNJOZgt9SeKf\\\\\\\/W95wzxTD4BkeQxNvCWfbIBmATKZ1VckIvsVTuJo28O0SIEDGen\\\\\\\/AfFiQhGD7QPJMkWDUKC+pXD82uNDhZs\\\\\\\/yOJ2h7B1lbbVz1ymMvVUUZy\\\\\\\/bDxszwQOYBVNaf4rOOPBeQ7TgF8\\\\\\\/0RfXSXs7ct+Crl+GeyDf31X0Q4+PcUdxX7ghFAblZwAYkOmlTH3WokTLHiMQ87H2p6wErCNOo\\\\\\\/O2CNAbzJp2xPBkTF\\\\\\\/shDtJe3xqJD856tjjByp0IP\\\\\\\/ZHT+CiYBdW1e+SWzp6F+1Sr1ZVjdUE7Vidgnyaz0St3O3\\\\\\\/stFAvqOKqSdp86qlvfBSa+LHFcMc7fsL0mdiDS6tGo0mDY\\\\\\\/NVVt0B\\\\\\\/RLyCIR1G+BBh\\\\\\\/NWwc6BellcjBi1DpaVPPUP9BzoUMST1ecPhlL0DwnvxRL63wi65D0oAxtwWwg7XWg23sN0kcPgaaszeW7y+sHAhOoHNl+vgyY0PeQgUNPvOTxBc2JKVart5oTDnKHp5iOr+TR1NjPtxEV7rTAyAsHsjZRJVmOFKXjTtv6NLFm5HApgJay+NpJ+ZaUKKTtkzOMjtoRfYEapASBGHoffCkbvDu1AprOlPMyWCXGs8ETrJO+H+mLu70C1zbCKsDoMk4NpBf8k81UDhhWGn8l7Qk5Wl+Oaehmv9Qgke0RBI\\\\\\\/s+8\\\\\\\/FmDQ7usJo5T+M8mG5ZUsNkn0HQv\\\\\\\/QwgmX9zd8JCvjM8T5y7YcdtjDWjJN2NEXKHP3\\\\\\\/PEM+Ote\\\\\\\/diECy8TIq2n0a\\\\\\\/A6Eeg2M3PXfayUHWxQXKSYsbLyOpc\\\\\\\/vfv5ZqPOj+IEUOsBCOSzxWgYn\\\\\\\/+zDSwF+4LXcO1RRQi3hq4QRjNFTSuh5waYa30p1EM\\\\\\\/tyI\\\\\\\/glp6ohhkvZRyLxHymFTHqfGt+okM7gBGPQeqd4wJF35eGpLLK1K5\\\\\\\/EykX093dGYwjoJlhrrZP2WK2QbzezzPeaA\\\\\\\/aTSIWsAFGKZmGICWbuhTgZTEYwfl1DfD7yLgxwPGcqQzXM9DwvBh+Hru6UO6cJk5Mn3d4A2GpU7RK4xQeeVeQCb2tq\\\\\\\/QMxSrlif05E+9lqOK1FfEDvgYYfRiGPtQIcH2IDk0E3ux51QOzUNkh5+IuLTKY\\\\\\\/PI8Ox0DTSw3P1ogQ7hfRCQ6Wg6LuaOM+IyVK1NWF2hTX7huUwGLIXbzq2IzlJxxciQha5dSYduWrQVSXdIfoKIGiEVHv38FHw3C3xuj6fJUavcji\\\\\\\/2AtF7Aw3SDa0TBVwiwULEl9A1vIrhqXyLkNBDmKt1tw+nQISx8i8kvE3QbQ+Go7rnQWSVVI05P8IAy9XlJTZj4IWDTUNEyGoebNBIkS7QRotAJ0El1A4v1xSsUlun0S0Gd+uTsWwXva66u+93h\\\\\\\/X\\\\\\\/wYu6W9uqVkmW0mX7pTPS0JkwGZmNcPF8S62Av89MuJ+r73ymCvK6YOrCjYj\\\\\\\/CM2R\\\\\\\/+\\\\\\\/IrTkklL2m\\\\\\\/TkY1telzV1AW7iJ7Zc+EKTxU0eZz5+92M92\\\\\\\/pv7b0y+uTsBZdhHDF1ZuFnLVA7qO9FJZU9JgcdooB3bhneS86ABLv9Kzkmgn8mBbOX5MmnNbeGGoWeykgHArcdEWCH9w8PVfHXfJiPmaTMVgVVFcjeMSdoB11yJkgnBaKywF3\\\\\\\/Sj2IQFYBVkO\\\\\\\/cd0\\\\\\\/59P+yNyJnxdBsSvcWC0erIYkQDchtjPoPCauUmUtIjtsxloQ3t7Vp28v6yWtBHHWNWcaRWl1eri3MoeZC1hD\\\\\\\/1\\\\\\\/CF7X5sBuY7jTzQhB\\\\\\\/Nkryf7Z8T42HXt4vTZfRkqErYOz288ozkZ93eBIvft+RVtWeigZoGxP4FKm2a3PDWIsExnTTPKz\\\\\\\/LWDg3v+3uwAQMoqWYvHaKcq7Enakh78l2zckY7sgfLUFqnSCMO8nhQ8egUoTefuNy3BrhwJSg1nPxbLeGL\\\\\\\/kXaYzMGHgnkNBDHeW45lPegMtYmpNYyNkK2vggfVHsQpwqx5hb4rfZpWBIaMV\\\\\\\/NkFtgenjLgshTd+mwGJCIMxogcN7uP+3FxROfiYkbguV+VzaXgfkdbbOCqEApSMIpWWQDP2MfnEp2kdbUur1le1cpBoizvstcLJz+IAQtD+zoJg8X4xuyDWTGPpHTaKDsISopwHblpQ0gGTGGb0bqcc5626aYNPc4vyLVBbI3xtxXFXYu\\\\\\\/8lRR8d8uPB\\\\\\\/2hO9LP4YPttv6R9uwCBpgacBelKP1IWsL9i+oxvYMqskW3cxZEj2JL4HMziaJbYYNalDUtbXP8x1fH4uBt6IOadDHhybX\\\\\\\/\\\\\\\/taEGpuCzAnwVNhQHkplwdtCELs1y943N+13sfGJDCXEh0w0KcKLHhD2QZyBiHc0WonQZcUGyFcdRtzRb8zOmOWgGNFvobXF1Vbgoh0egOZsL85PvkXakKxK29Ckv35TsQBfvsYj0Nr\\\\\\\/SJv\\\\\\\/3AnkGgzVjhi4WkquyIH\\\\\\\/KHauBdOW+1Fry2qC6bTvs19xj8kEfjngKGCMWDG8BiII+e3Ah+E6o4yduHmi7AXwGHNM3YPCU3f8wN\\\\\\\/rdckUZS+0qqlRXQT+pmj+hj29dwfac8lWg+voh6Icq88\\\\\\\/APKfFYBLAZBE0uaiZwkVckEPQZGwONfN72mvbiCtKiJY8kI+iiDyMcWV0JLkML6GaPpkmg82f2X0s1A+HLEeMe07wh1LKz2kYVuKBOoe3CakRfHGMyUI5tIp7nJ2j69Kk5U50EtCAm1rZ12nKyfT7rbfkHMR0CNE5Y3x6K92+4pe6gxT8C32F7lDe2kvzFQHm+9PfGF4v4WF3eUzWLAjD4J+8BJGkqIg\\\\\\\/AS76Si2yYpeJ0Pk1gMAnCiTkCsOpUb\\\\\\\/WkbZzuLCEfEQT8WE1Pg1O1ZG5gOY90epxPXWZjUM5iMsjOZzn4QpPaIrwKBDlJZtWbM0ry0UY5IE\\\\\\\/qyA76bYBBXN9c80O2xM6fbwyzTjei2a1i1YONgTQo8rjUhdziHbBiRKM7\\\\\\\/Rr9thwY5kW6Rx0k0YgzuSO\\\\\\\/CmRSkH8NW4K7yZBrvQC3PK0cpEJ2tuAx38y3nfVgrAIWYqVJ9IjKY+4ZeKb1G5UXIAtGc646\\\\\\\/mPvRlGrFVUi859MoLgA6WKpI2YnyW\\\\\\\/U8nTxUf5So5eKMrSbuV1RZNKKGo2mtNQxH\\\\\\\/7un3aPgFpkNGbO1u24zlVqdhIrt3rhQdRplgR6YRli03reZWcyrmeVWmUs5ERuu1dJww2MStwxHUQlvP+0nLEh2DFxxAKXJ7psa\\\\\\\/ta06dZ5w2I7ayxUFNlOIHH+PZ+uuhAVu3J7GAFfz8mRpWJJ6Izi6t4aNLD3Xz2UIbsRJgme1UDMaJ1ZzAHQNQDSHv974uPxeERzjbIpOaMmlQxSOVhwZ3BnFyewuWe37+F5BXumwsncTOqKgXY1fbPx2KLpyiIvROQmBsO3nZQzdzvNt\\\\\\\/vRrqwjDFbobSfYIQLZbBHzcGzT2NJoJGtCtDKkzT0bFxoinf4ZHcmE7Zu6aOKz1zkSoYfUwaUpTW6oDDKvJsoKITTZM6QnTeVA4Q+TXO\\\\\\\/xeLB\\\\\\\/\\\\\\\/45FBO4G9bsvSDrAA6CmRTUqFyF1CzzuE\\\\\\\/bhuBX78T60cKoJdkvYqgjkxcPVrahDWnHKmxXYONuh6z6VgwipgwPN5eBG27uTp6QIhpUeJnxU2eibHlGR4Zz1qLo+LdpeDwNckoCwaufol7OKqF+EiBYOQHBVL79weFWiond3OUmdo3u9J8sJ+yZu1dWW+J3\\\\\\\/dPWoEgilBX0WESnzRgsoOZmkdv0S+zvbJLoKI8iOEDeIPzxKEs4sGJjABQnc5IPaC+EpoetAeEncMIk40ZXqOIQ7T6zNZ+8Ye2o1apeSSTng1zMC4ji3IQ8otpFvv98+C4oIiWTivfOcyt5It6aIqF2NgMT0XatXu26xuJXRkgxSCO3sBvQ3Sj7M2+cD+WrApa+eGh+r3RMlsZDvOIkvpiDDX3FVkHDCJNMT1+AjNQBSeWyCGjPRCM387MqzDAFGJeg9UrJY8aUhho6hqoRNfMy++dbgIdYEcBltCBTNkkm+BJjPi4BqoaX\\\\\\\/eQRQF7z1Clp5+WC6t\\\\\\\/mWwC5K\\\\\\\/aj2IDGUaSnzGx2Hx4RL5amksFtpgWJ4VBbHBwIwDXvm5kkDS8uJD5q0qZbmtmg0pVaMP4ahLtv2F2MKV87TWKdLM537Brm8IOWyocQkCJATx5tUSY6dOBLERowjDLg\\\\\\\/oKTQ0KoMH4lHzJG1UKTOWvucV394XQtuOQ8aPPAXenS+KV5aISsyxG4YFs9ckzAyATcZo3zi4tWANOW2fH\\\\\\\/LiJH0xvDBvjwYsO\\\\\\\/8I1NKS9CVJ1KGmQN8h7ZtLuc8qiqIMhNUN7wJ60z0ENXTgvgOEBYYZhm2SI7eYouwogAphVUoaRyl2c1Lc4koVK+Xnel9\\\\\\\/hDbWj6pVs2Jq\\\\\\\/Pna9sbRlx\\\\\\\/0sMgBZCZ3ZLf5Y+EeA6v+mHXXJladmrbp9vaS9j15aEpu6qqFhN+yQ1GW2qTjsE03zLlbEkNZIrC8iuvat75Zvu7XoO7P5GBke9dOVNqj73EZj181KlA8G6\\\\\\\/BY6hqEj0JjYW6Y1M216Ra0WSvGEhlBR0kR2G7yGUwJPmrfNI\\\\\\\/wPlawV3XVq2qVfaulH4cWUboGCy6O1\\\\\\\/h7460xkUzFkEV8qRwPpalXTHII\\\\\\\/qx5Hn9paKpGSWfQZ0zxDuVWgRcbtY\\\\\\\/H7IbFBb6tO2Agv2Au5\\\\\\\/2QYUiEDlCCrjXwNbrBS0BgBsoIBg0wFinWRq\\\\\\\/KuX7xe1LGVxdvnQcKVR5J8b+8xUAH9jz55MHhygXREZcXLrBzN8isMZoNg52\\\\\\\/MFJlmIasrN4el2mYTfwQV77NgFP8dSWkpwk4PFdXu7JPIAZztwpK0yIvFzLeP+Q1hhMOJJMojf89vnXRRYkEv7gcllTr32D0H3JqOd4fL\\\\\\\/1eCRbHjiTy5\\\\\\\/fynyszhQbheb7QyZ0Fhh7rVmA8lYts8PZhA8t548rXbzW8WY1AJIJOgCCxobkEG+pdQUUHWw6N\\\\\\\/dSzRQkOZWcHU+\\\\\\\/KGQjkTauCKBeC9ldplErTS8VBCQB1l\\\\\\\/RV+Drma4S\\\\\\\/ZPSKIUDOXapLr4Whf+4BcoQqEinZyl3pxg3KHH8ISI+4rcy\\\\\\\/TUq\\\\\\\/FGA9\\\\\\\/ul0htD8XpK\\\\\\\/Z69C9pS4wuyp1wH2FpkcpXPvLqv\\\\\\\/QCr7TGu6WkTKR0mZCacNjjw+EQofASSbDW3o0ZfAQezmvkOdn1rvJoQ2xhLO5Pj70njF70qUu77PGnWMSniDnTwqtxh2LFlnDoOEVSiFjtNmKvtOndeY9efV+866RWwcb\\\\\\\/8XItLwWu4F6hxND+BaP6pp9cOdxxjH064DBCW7MKu2x5F5r9p5IapIxAmZoeuHCOqqUuilDHE1KMp\\\\\\\/5CEFXojLyJDnYXcKEm3NlpKVe7jNz\\\\\\\/hZwwqs\\\\\\\/n\\\\\\\/87zl2FEsixwcVsC9p0rgMx5q7buQeWQARhdQrD2\\\\\\\/1ajBRwjgPrjX7HNBqEReWzTF068RJhqvAA5CFTUEAfY68t7ayozhyvWW7P2fs8kKkilJZT3\\\\\\\/9d8AHQkbogSOSmosTS4KqwIcvq73Q3\\\\\\\/GWXAnsQBmmvTa+Iqu84szFk2sxojXgk\\\\\\\/\\\\\\\/7Z0Yyzt0O9gNPvNWiEu4qYhjKCRn+BaRkXB4czfJu7wKuu3YM9DuVAC63ooY2lewtBGI66YfXgW7Hp\\\\\\\/4dIhus\\\\\\\/3DQ8jKOlZPRTnaLihXvE+VU6m6vIZQ56gJ\\\\\\\/o4EKW4iU84pWX0+bNkvMRiM\\\\\\\/JYfqKmwQ+iw3NhINetxtGFou9+h2Lh7uQMbNAAXyM3V0TlNZ2oLjGr1NFICiaN4RjHDZg\\\\\\\/6ibki6+SWEkV4MVSFLD8ToIjGbFJTJhSkYGIiHyVua9av9jGG8quUUxhl3pnuVXHPPLNZu\\\\\\\/EZJWyknrOtl0uHgob\\\\\\\/tx2OaoAuqpFiNRmGBIxCIjfaK7XsT70eBTk8OAbUaef9QvzwjxGF6kLebAfOESxgdlEtWHf4zmuGeCm5y1zLaX3cqUygB7mGpOF2k3eGbWXCTBDNmY08RbdbV73PVuL8sgDdLQ7y7gF8pZmzFKuqYhadGjHYN79L2Wi47NtXrWd\\\\\\\/FSLX2WfWFpadbZG9Na+cFKuHypyFVluxY1C\\\\\\\/AUHYjVu+7u\\\\\\\/BFOd\\\\\\\/8TVki8ByH8pxhGGt8supV3Ur1B83PTgAjKN1Tbs9yNBXhVQKcUB+ISSvhQ5B9tJzQdJMS\\\\\\\/FVqsOsh5xqGmRA9QDmKD8GlOSA02smWn87xqeXJimJe\\\\\\\/ATi\\\\\\\/tNsxKn8gbXufviY+JTMGdNtP4L8On6fOE4svpfNA4z33a9peDZA2H8kIPNfyxELQgfMOWplZ8RaRYu9ybgSs\\\\\\\/rQzktn\\\\\\\/D0kcSwd1rmOnzOj+IHZ8q0IUjBw2Q\\\\\\\/\\\\\\\/L8\\\\\\\/sjs1b+0nN+hGcLRDqOlEjS3M5EUUcSgoM+FmpcQ9uZHNKFInkLRjKcv\\\\\\\/GIaZLoy7ApE+4HK3qhM7GGLXaOmy6wlBTQy\\\\\\\/Dd38B8ROXE5JEc0+bXSCnCMeYYLin7zui9EsiSRc4Yi1aTjLjVw2qvAw\\\\\\\/IrCdn8yljuvVnSWHUbZIswtApNedQIlKI7MYnjXN\\\\\\\/J4eCFkISWYhNFCZLlKHglNfeEmGM3cF7eFC7Fm3fgaw\\\\\\\/meF188bLNAot6Yubz2BaBzhXr3rYIkG\\\\\\\/FL+2m7tzWW3LlQ7Kn\\\\\\\/wcINvD0p4VJLSal\\\\\\\/ls6RF9awpZb7a3PgL468HhvrtDp7uL8U1sNP6UmO9bOUahR0W7\\\\\\\/Oub+QAKdgk+2hoLJE71W+c6ED6eGOGsJIliHEV4IUdcys+MxS13Wuunc7UMQQXB\\\\\\\/lR+\\\\\\\/X\\\\\\\/OmFDZGJHGnsoffp4vhZVl+6G8oyigOvDDTQ+k8RS0oEerA2XVLc6wRTnwwXVvldpk8AClM++NHJTgzdvsELQqB6BKa\\\\\\\/n5\\\\\\\/3MZ8KeDejckQZGKWfx24+olJkKKAZwVxYIBkAE2K8HX2Vatn+C06oQgH5HAAA4e+pwzr1OcM4Yw9axhmvXUkozH+Mmmw8xZ1iQXqh+JptgMxDBfM77FkLVOMZzMq55cMAHTRCpVduB7cW\\\\\\\/JdyjZpn69j6qUaOYk7PNgRRfYjj692ZrXwJCZRnYdnq76f75MEBJZo+vcqdk3o\\\\\\\/R3wGoX\\\\\\\/mQgHXJzBa77FmoxiSvhlLpBXHQ\\\\\\\/hWXsstz5kZIwgUVdpdqtO2AWvykQgmbnsXYIBtIHqoGRvuYNVjBl7dc3FxbYCz\\\\\\\/N7LYyTWnpEX2AhEOOS+5uzr+rL1ChI+tFpmYZIsWv\\\\\\\/gSTE3I0acCdz3Rp5xQYooffzD\\\\\\\/4jIGgaydC5yekLM9nQmySWGebCqYW76itriLsxkz0j3+2L+QvTlUdnfEFXj9QM0\\\\\\\/y2w2Z77DI+4FJHnkCOfPeN1hA4OCGZKwJ1Ulcp7I\\\\\\\/a7bZt4d8Sjt0FI+7zJUWMd1YsLDvCjPUf0B3zlI5lnZAIq5+MeyesTpYT8r+R31GA+\\\\\\\/b8Z\\\\\\\/2XEl0ewElZwvDpCJDWPDR8gNDN+6DMejAfYLwMu210Q6htvMIugZTgCcZZxpaiTjhBq4mQY2QThKWzD06\\\\\\\/plns2n2YMPr5dV7o\\\\\\\/zoErMQqG7LUL1xeFqo125xvWwsrd4KeD5uDBmOhdrEmdleawm+hq+nne6NF7BUu\\\\\\\/ngK7thtJWBTk3C\\\\\\\/Kb8tyZAzGNQIxBDRbEcN4Tkp7g66ERuMndKSAYUCswoO8oEQWXYH7CvU7SP4HYayXneNdLLy8aJ3NahH5g0WgFdTvUvnu8n4FKTUl9JDtxk8sBHPsii5bvq0nJ0ktMVfk2v64BjSaU4n3g\\\\\\\/fi9BzQBSg\\\\\\\/vDCn1loVWsD92DoUR1f8T\\\\\\\/cagQyhlEDuBpn1XvBBqGixEyglK1ktT6AWfErNUJwZ3uYswwTHDRS8sObhVB2oPw6R8fFJKIH6jxzjGEsB5sQs2qlmCQagzkYrwK4Zc6duDWq1I+n4jirfvP88pS7biF7RNuzVVKK9caYC0wZPWbcz2+MdxPdVar1aobAHWLg4Uf5qgIU5r1YxzBNzdIyssTDr+EUX\\\\\\\/lqsCAn6npcVN\\\\\\\/2WFXhsEA1FK6qMdO1wkTchdbqnjiBOZZuzlXwVhu\\\\\\\/VpkoWtnAjYO0AfilgGQWqWFPwqe0fb7ev4xvjV76mBtohNLUPjHRczN5wSfzddTM+0hFv8\\\\\\\/7Y\\\\\\\/cL8xOfjpnxiMusplF5bpA5LbEUfV5awNoNyWmNfm8EKOj3MzoxmbU3yPQJUCC6vF\\\\\\\/GRGkw2daNpn4Aq3azu8ZGDSXnut0i6saEjRt62aLNIm8waq1sqDHqw42U+ny6b1K9TYVGaHP+sa3wStM3CvcL95kMM\\\\\\\/njaXWCjEkqyC4bgJIlV+Me+z1roQuC7QV7ytPF6JMGXOd4rsanRuwOe3wBf8TuB32xz+OZ1z+7bWGOt1xC0g18xof5gg1M+ismgt6cM7l1ywiXroWqrdMoLS4x7yMHkFpfIxGHuNtXQ7aa5T6ZvqXDikog6KFE0n8pcFr9tfMP0qhyOSllSeKM+vcSx5UHCecKaivPB3VpojuQT4ql4oSdappT3qauff5tX+UGk4i7G5kQzRw\\\\\\\/finLmsNOn8Nz2Ott11TObuzKZBH60PLwVlrzUIAbPITdJk6doAYw\\\\\\\/E2Cier9Rjmhy7GsajipdfD4tpwUB0TswIHTAHzen3Tb69vDRyf8v84Npxp7tBTOlSw36YFyCyCjnaZm9SXRByw52aBG2H8x5nv0ToczrrHyPe5iO4uv9ZLXMJnRTRhTWDa4uBYCxx\\\\\\\/J1cmXzhs+LET9\\\\\\\/cDQZ3r\\\\\\\/K6jFOQ\\\\\\\/FbFydAV+Qk0JHEgV+E3KWRi2GE9qiqd94214\\\\\\\/Fz9AHoY+ioo48CYb+Aafy1vnk3O7bGjBVDjQH1\\\\\\\/+a\\\\\\\/6uC1Zqnvhb612ikytnUDS9p\\\\\\\/Tqoxg4mTy1DTD\\\\\\\/BlVxQ6RLXNy7lhWCk1xuqztyFhqGCuoAWUobGQEGpAEd1QPgabEVLoeTWiHhKfwpqL\\\\\\\/imyF7KnKuBMx\\\\\\\/65XzEmcwrRXfvyNKz7qL6xxpQlun5RAEbgPxK4ypQ5MheiKMSHXGJ0qu5NPlDIDm74DkBiMVmV5DSlSiyeG1axP7KDiEbAF1ASd9g3WQ3J\\\\\\\/\\\\\\\/qJHBioOCz\\\\\\\/djfsw7u8KAQ9U822wjpfmGi5RPKyBWELWDj5CZoHNYxKM9wcIT+YnF9BxjjVSNPCUMS2F\\\\\\\/Bj+1Np7PT\\\\\\\/FE3T1nToWGuO75S\\\\\\\/FjfMZTlVTDbWEaw\\\\\\\/G9\\\\\\\/vsQfJbPYIgN+n+jI8yCED97b3D+6SL+1Z6t0zjL6WMV\\\\\\\/6oaN7BYwOJ\\\\\\\/OiTOkVqVNB1QPbPB0ZKGNPgbjcZG0oQcgQa6EphZ21ZvzIMbL967lzkoX\\\\\\\/GtRX78ZyQmfb6lvA3rtM5zl0Go080xsIJuqw6rtf8yxkHE4pVKMcGFq7RMnB68A94NqQQMy40RTYBwBCDoUUTjDz\\\\\\\/aQlUzxUwXWWHkSoxptRZSGbdB1V6\\\\\\\/ifTD3YuoVII5JAFCLENUT6jgjQDlUi4Fb3F+zOm2uQ63ARdxrvgL1Vs36vLNXP\\\\\\\/ORSQcOTp\\\\\\\/gtz3p3UGen9a09HnBKT8sOB9nL4g+pnHv6ElKmM5d5LZHZOgkGl4iULlzl2QSf+4p3jR3eGrUN3jhE8HvJSrjCVg9IaFtzdwsCFpVeN+JAGZ+\\\\\\\/LjKlw3jXoR\\\\\\\/fRfG81\\\\\\\/d6aX6aMdoYF3UNXat+QfERhYOSt2B\\\\\\\/mNqC09ZnImusu4GFJjtMNfL3BuvR1CsvOM9CLu\\\\\\\/HbnYSeBTleXDvEcRIngtkew6a4yMIGp\\\\\\\/32z8H4buVYO9IES2IhDTXIKnVKG7x0M2xCcXZdydkG2owwOddjV6coliy5eQveGGN9OdR9M7wFPEJtV0XhVA2ejJy3GnspXhj6hfuPlg47qJyHNaeWdZH4Q7pjn7ER3I77XGChK41aQXSuyAVzYTIK0F8T\\\\\\\/IcpcJ62aPhBjfVCc5GEUqhXaC5hw6iinRGxJDvazktEXNXm+s\\\\\\\/TXsTspTc2a+jDVlK95olwi7nyGdEldiKgnQuLUG5aaKh7iZ+YYAz658K4wkU28PDlbBctZ7z1QVOyK6qGufs+A5fQ\\\\\\\/5xy3g4tCvb7A6IpSU6OqSs\\\\\\\/Of1WJ\\\\\\\/XsUMrSzxIiTS5CTmJVFIX4sMyQFzf3T+zErxHuceHbOCl3OHg09IrRLzQkQ23Ruf45C8gUPq2NE4ZHvfmDkNOzDVro8mKLlBAeXSHjqcD5Q5NJbr+6gb4lzd8u\\\\\\\/uFgtOrHClHdJCgV7U7QV4zEo6\\\\\\\/3Tw40iEUDdk9g5SRfAKi86IfG8xpftqDOpy6c2vJG6f+UvaS1x0nqPBRzc3tmEOeIo8ROKsTHJkG\\\\\\\/o7vszalNtYFmr33GYyadnHYw4r4nj53d1I25n7y3Ukys4da9RFbfZL9NyIUGoxFc8xCkmE5aOJEB0+wlthIX5D9eBDZKjWuCVe6M5Nnwt8cgKamb6n+imaxqpthZjl\\\\\\\/7QOAIDcK0Lg+S+8ksXtEV2ObUh+wYZRim5v\\\\\\\/TbpORIHsWC6V0tFyf20zVYEqR26APJ57y5Hkw0FNgVSRPAxCZF17Gz1pqEhupFSPQsM7bDHg+R6\\\\\\\/A\\\\\\\/ixmKzhLpggoLdJWV+EpHryG8Pq8Rr4JFkna1EthBIp6DfKXnZztcbuymDVtP\\\\\\\/X1YU8iyz6GRdTKHNXvDqzN8aIS5P7tsTGe5bXSR2cHoQOs+aqnN\\\\\\\/gH136vrzd2IxHaXUht2iG7vkxlezQLYO6R75UgIEdoPjiSQSU0kEibDNYMG6RVhhl2+9Eopzb6SfjJcQJy+IeZK7X6DzKoAjPevMPfGencbGQsZoii1Ftg\\\\\\\/8iuzIux\\\\\\\/CY2siUy776vlO9C0vJs8xoB+6i04yk5YOEDHyUfV\\\\\\\/XHvVsaXdWZ69e2I\\\\\\\/DkSN0ytUE7toaypVYFsbnpfqeQSpMT97ik93PgI3qL7kXbKrsez15TBncEztwJPnuYl8BHxl680Kp6m7PdIxPYoLMHUwC2bIQlx8ERa4jWYP6oh2f8lv4VdQBWHoGr7\\\\\\\/53IW3HyBCUvGbxLv3ex23QB9R+zRXJjr2YD75AAuIoWw3MN3wVv82frar5u\\\\\\\/rymFN0kfb\\\\\\\/frLZKCCUifgyghzrW2oEnnvB6pXJGvmkmCjSOppy6sGCg5FPR\\\\\\\/SG50KB8fPzUgaauVY5f4svfWglJdrs+2ePhZjtLyygTjGI+FVFNy0AGfBDpbXz+fgQgAtIcrEigVtrpN06\\\\\\\/ztRcfbq1ZN4mUFVepMr6IgexS+BMRHRXwXtyJutkRm7GN4t473\\\\\\\/yd14EmCz6ICpZa0A7tPkZNDybhaT9je54E9i4xhTtX6ykwCIMmfIsdVMuh3TMIPk3nia4mxpdBc+4n1sJW\\\\\\\/GLtBKV46JshOWPXPfJ8WTkVeu4ftHeHG2iBs9ea9w3wIe\\\\\\\/aetixsdY60ws14MJ2TC+Ppq+O33H\\\\\\\/9HkqG1+9XOXsqS3fin94KesMQE0ymmEWe3y4tL5DqDQ4HGEwCXW0jtR+0CBss6SvObKRTY0CgvrPxEKYr1CjYpHkdNS5BmitynPLiGoVQuhgOKOl+2a5Qd+3Wg\\\\\\\/uH6yHbGRTof7cLEPhm+MhU4BJBNu+gVa7DI\\\\\\\/cmxoKZIX+jvdhATdr\\\\\\\/rVje1QO93o+++p1oRmO8jJ8iTPCGqIuMuNDmgAkv8y5JYq6CnG2Gmme8KfdfmVbcfb6WTNV5xjZozOmyAMu15ppbhr0epZsqm7UbhcPw1dvAo\\\\\\\/N9yr\\\\\\\/yPQmkpXcHx1vzsSLcbNgZ3cHCuBSDbHCutJpyiuKI9mrISZZi8TbE6Y8cUoe4I1FJhqxd20Fq82omMh8GJyCo8a8tG780oE0r87izrMJyS1nbrW2Vg4cW44ZOh\\\\\\\/2uyLShCOUuNwQntvKqgGHODcn9fpd0V2qITEx0NL\\\\\\\/TriGgYginj3sMjCco+GFjOLpTOr1kK2GyJI4nS8XACakFprgFsv+ZzLITMg+BmStXM7I3hfiNFAKPO3TZKt4ea31KT8Bcu\\\\\\\/2Ilt3TM4dZvA+rr8RqnM05Hb7JdhFK7Z2fKjNFj4YdhwApS0CZDLV3jrvlPIJaOZMlDQm7zHNbNrQjWQIJzl5XOLUFONA\\\\\\\/58\\\\\\\/QGL2csMovi7VB0utStZClSgGx0vlgfd6HtQUB5RgQ4t58cZ\\\\\\\/mlhEb0byNOeXqbyxnB\\\\\\\/J9aBJgh20zAF5vT1TvQtw3JHJTVD+JnqIGTW7u92jWTGJ3\\\\\\\/wg1KqROgp5YcVFl4wXFgSr61vZ6Zqfl8aLrsnLKysBFWcIT2H0khB65x9CFD0J8SPQABpPVgU3PQZQSJ2ABKErFNW3Rj5\\\\\\\/dhRnW3A6FpgrhHJS0mkM0gzJVWu+vIdYH4UCirn6CCXIU05SGnWAH0MWplMR5q77FdE4GUSQaj5s\\\\\\\/BdvskPiaovQlUREcAhqlnfjgbLWIVmNF7rSFttwbAhWjhQ5xqB9igAsnn1rR\\\\\\\/6pOWo641MdbkntQD4jjbcoQOMypU6iD+zeTy2KA0ZsXjRyKoVDK\\\\\\\/wHdbUmPCgEB6AM2CfpOF1\\\\\\\/73JlyNKKcJNch5Bf9i6nEAkOKsCRrBXxJywSLIL7nFCZ7rpFy2pzfizvMws6T2acKOwNFMUR1wanyzPReAxyAmq5t6Y72+wVOgYgcbNV0rFXe5kzbGznevC9kf5ulaRtMms0LLA75hMaTDKnxEIgo5zrVr1vv7\\\\\\\/qaInmvNDFXJB8gRt5fHNwQIxLf3TVJfXNTMhTcsqizFlqlrUsrnM+gR5VkrM5kDo1Y8vl3zO0IPbntS+2GvtccWitLWvpj1MtJFxQJuAzfVTvInFS+qy8H0Y3Kos1Qg6zu8TLsVTw4Uf+C+92dkkn89nq06rQmreFVePF0LzINIFJFg6si5I4p\\\\\\\/pGwqX65kvsGSzjxEN3bHW4NBtt24lo3JVK0UkfROrO7jxBGXtNoDQu11RsBbiXZ+eYvqxg8yFBSIKXyJ1hU2ESz4sa\\\\\\\/rgIs3sIw0b+0gFkmWlNON17AfFJlC45DWdhlFipfRA1TIaWJf\\\\\\\/HFwG9wO5IrE9rUY\\\\\\\/IoVMpU8hJWzLy0Lnudl7P7YuCu+TJ8gCh+HFcm0nMB1aUa26Ny\\\\\\\/xWflv1u5FJLD5dMfSNophdqklHLt\\\\\\\/rpzeZbP\\\\\\\/SgHBBUZ+beYibvUKE8ghl6GyGv7ZSFXv0IlaG8NcbZgZHBFEC\\\\\\\/58HCgYDpHsNxKJAyJ439xPKB7lq7tFRaCnN9atX\\\\\\\/JfgpabueRtjCj3uIL8hef5CjAwoP+7mz+KtRII4h1smFZ8cCd1GzANlhj9OeXeOaOZggiZaIPE7EVopmGRUkCZY2446m52xUI9JN3ih+DfiJSbjsGcH\\\\\\\/6lmoHAYRXLoehUW618VD\\\\\\\/Q6HdM8oOjVY5xU\\\\\\\/\\\\\\\/Olfjdk4heGFX\\\\\\\/TVDVhaBNyjo8q5sHsQlBnZUkQIuhX86\\\\\\\/ej5DapCX8FL9bxpnoClXs9YckTdmNNYqZckHPlfKLdGI0QxDDoV0uuVTdsvqXPJT3GUz\\\\\\\/FvCV8ecWaCybNMFe6J+E72MVR2NJ+mewFsk0oJJjZSXc1a4WNFjjWxv5vKyHGkRKLnmGUC+cdCFte+DoIq3w0oYhjBB9cOIY8lSaHO04vqJ4ZeXfKHe\\\\\\\/ZXKZNwAnGzmjn19XU4cdXdLFBomlUbDqmxacHzK97AOzY2LLlVA1kgBaWtjzWAR3j9AvxpMY7Mfsc8Bf0GX4J\\\\\\\/de5jovHPoRtR0TdSgWa7RmwbZIdaxQq+oHaFwQCNOQWIGw7qnp3XC6\\\\\\\/y244+fS2\\\\\\\/IHL3NirY+AuWqdtVzRvceHKE7dzCFDrLkCvTtwtyhVeyB35hWhUq28kFJuP7CsKF0oibHA1Nty4PDpPiGdSqmPOLq5d9Zg\\\\\\\/hzxLxRTrIkQUECAemW2BGJAt0Cf\\\\\\\/DFjZpEnhThInWOBVHM8Wx8wxIN+7h7JAydxlMMhcg6DEAmZDoecUguolqctoct7qQVMCToSl+7J9MQ99vuAsbUMA6pl5B6oT9S7MtDJXfTPUHzI98w1IcnXgtmOiLTRASYsUtC473nSoW9CFisnRa8Sv5IEjLSVIUsihJwCkIfuomUAOcYXyqu7QREmeDezwK1cJ0JW1iciDrU8\\\\\\\/2CYQNFdW5pMBqwE0\\\\\\\/MpSrCouKRn1YQVIcs6aaJlU8Zm6cO6vSzNJmDzvU9yW7Pugugsk676ENGZyERWIRa9iWnHJihluMMAvvlPj2MA\\\\\\\/jRbktuoCymvFNGzxZBpZuPQorvq4kT7TDNqNnFIHij4ftvaZpC02IGCc43Thmid\\\\\\\/NZyGtwShA4eo63y6MpRuUaifFLskTBkpNSlR4aeqgo\\\\\\\/BJczWk1XC5zjZuGwL09s9raOUJvvyELTzFgsK1D57eriyeoDjyd91IdvmTlDQsrMM6CtqzwMfH+Ka\\\\\\\/1OAbFbRkW2rKQcpx\\\\\\\/UYnXb5wA4LDQAg3+LGON3PXbS\\\\\\\/TFhtm3HDfH3ECIEn4dW3MQMCz5DTyPR+551\\\\\\\/QyMIsxb7QRjlFFMWZ9LdBQNVq7ij+v96vBtYga9lGTugw04rFYNhc54Si7wNUZCYyPIh4i53eDiXC3CWNeVY3gwGxvDwcwuXNfECli52jrXbKhjA6nM1RMwyVTdM8aqy3wDaP65icxPO3QwSuJixnp2ORcprjDwgZvkpH1zlxe7PW9\\\\\\\/jzc3c\\\\\\\/J6D9nhUGUyvTGDO5wJZrk5ggDy+c8be9tTtDaYChICYVKXPnCRnR55f\\\\\\\/WfqEhzOBG9qqjheYBXnWNTneoQjsDFcmLfdU5jlMR4lW8TpCWX+cRu21cgzk7pEvPr6MKZ+C9B5HgAP57ehcYGSvZ1+0eOmEw2bSsVUQBcoLBA2cgG3ogEeCf4Tnz8vFPky6nQqFhE\\\\\\\/SDV+4z7V\\\\\\\/qfEqgsm\\\\\\\/RKwdT5DOLkrQQNotpwLbKUL+SH4JZhb7QOuCo6j75tmkd5xGToCD\\\\\\\/3M+2adcKxRuNloTKs9HbwUNxQ6WN8TfWFVJLkH\\\\\\\/6n0FrG5bCKiwhgl\\\\\\\/DrM45mcsxUOM5dBTPQ+sx881bkPbFzEqKDNE9GoCPjrLFhL8l+PyFlRtbk+VqET86dC50++yceFpcN0SOQPA2WBEXBKtGapbaEei2rkUVp\\\\\\\/Ohmr9s1fZiXHejP3NLT8yr2yk6B0Kf+1cueKMyGFbW1006vThz02vFgoD64khdGMmWMz4Ae6yJB95wRzLgDnd8Hh4ei++2daDzNh4X9tIg7R5rxauEEWOr36aHFqw\\\\\\\/00f22Q22J1VBwleG+gnA6osSrIcdAcW\\\\\\\/td9uBxRtQ7bjBS4alYlp8rHqZkS0nSdQu7fwfJZDjJ5bvVPf3KbP0HgjL0EN\\\\\\\/\\\\\\\/t1NixxOnWOoCNfHefJROk2qcGnG4GhafoKJREOUqFiOE8hb0r1KU+mtPDiGIKkyw8qBGPzbZ0DtnEx7tno0OdoEO7mQPSevc\\\\\\\/dKc7qemiW8VQBmwUxYqaj4NPK9b8HH8oTauxB+NZeyX94X3CWRtbvuFQPIShpLrUwtGgHPXg6aHQJJkkoV+rVPjkTYrHG0cIHSPCPQ8qJ2KDuLw2sY73VLYmLULHSUCvdO+bEJ8fpwuVbAgTAB4GqGtm5KPTS0IF1tjhmW8yadyYfT8d8sqKfm9D1VIRYAAe4IfIOVXPMWOvMPNVDqvvn79\\\\\\\/JRYPZzunZKcQWVudwLAxiYkRTXhNCJ4aplzs3havZYOphr8Nijv68iT2PI5IGyUPqSSmtFlRvG7ezq8nIu40FtYBEtvZNapn7PUix6Kn82Q9eoaj9I6GGNiBasqHJcYwLTishFvi9Xicy+SQkNwIL+nRdPHynBD9cirmvcP8VRnbLKJONmIH5uMErosIqHZUgI082GjHJvtzVYi0XZanevhgzMqIPVZ\\\\\\\/73fWFzPOetBaJhkyRbhDa05A6R30t\\\\\\\/CgwGeNeIiS8aoWfH82EmkUFCPMCEyr3cxD01PBBJ8kJYwIfOg5ExtTmn2ary0ScWnCPt6F6a3as0NLwNIPZkCGe9QjeZHdj6Y6i7Z8BsCG0sjKQu4wVdH3RIxVJ+IbyftIQQciC8Ao\\\\\\\/MO7w04JEVO6E\\\\\\\/bNOmbzBz0ZZsGFCbvRDPtrPhuTjZB03gnSsKQEDZXTzY7bXmWLv53mpF3I5YTcv6TslnWcF2gCAdjOVXIx4GClZtHLUT5nH0HW+yo0kVD3Q0OpT2bPnQrHeN326LWJ7nIkj2MuXB4StDeO4vZHi8KuJo1JP6d8cDuUqAkhHOC7p4jSYbfhkQksbxUnbhveHN\\\\\\\/Aoat5gRSzGRfXwS6cU+kPLRBvdavAD3iVVVQgy1OmdzueaUFAaQG8vUzEuo+EB6WA4z49wryIy76Kj23P5dj9uyFSOo5uk23ayPK8HsZ0Acgbac9dFmth6umYwmh7yNuNkziyYsRjDpP72g9m9pII\\\\\\\/nb4v64vqyCtdG+OVoEi1DS1V3Yqc7fFiro7bnM7pVcjk5ZbCucad2uT\\\\\\\/oe8n+Nn0TOgpHbYpCvpadDpd23yXHb6IgyTCr2q4liXlwhF3ByCtP9bzqoD5dmJ6s+qrwXVrGUTlIDFDnCXBH6+etIzTBVRpd2KaLHIupC486BBwC2BacZ7NvdSSn\\\\\\\/kItTJU18adUmnya1IPRIEAT8TXbG5e7Rg+gG1nqA6xJ\\\\\\\/P\\\\\\\/7UoQB3k8f9zG83\\\\\\\/vDvEryT8SVhxn+mqEsgaXlXZRcvczcMGOUdpPdPJSOtQP2Ayx++pTbhk00b2JIY6OPU4DRb9RBHPbUiY0R7oSEwzJ+OBPQ7crw8doFr+o7lKJh63rkXk5xsXXDkqRadtcnd4QpRTwevpIDMXn8kjPDOBnLYOaDt71\\\\\\\/6YKCiMzAJyTzp+MOVgw606By1CUJ5AK+AN7PQyjvwpJrbIqo2yLAnzLCFnZAS4DMs3q7PxdQS0noa8+14nS+Ywcy6Th5eUsuLrlv8OmjCmAfnCy14dm50tv83FGPUN7tn627doSrWX3Y4i6bTqDPjOkq9B9pU+y1LhtSjgDbgD81Vz0fitCB\\\\\\\/eUga3SVNFxR2ndSYiuZhN2604xTmVFq5k1oDfnE67bOTjCz58beorlRr8vd3BjPoL3O+Vx7zKu272A+kDmSlGdL9SAx20DndeGuTPQIhjZbWZSfHI1WiZAlQf94di9SjQyqOApMkvTAnCyai4ef5sJ8H2z05VKMEdj\\\\\\\/Tx9l6n9OcspShzwDVBl4b86Wp6nVMUQJVV3+2SSh0mirSofgm3w+Vj\\\\\\\/8IzJHAkeJ0ptKmD7JekIwTUvBEAj8Z8aM7WQemjOhXJjqi55QemFsQBhl7t68TjaRFRJaogQjp8WYp81gHBpCNFAGnlqfU3c1OcuakBFBiDsL8CujPKrFPiDh5vQUQslY3SYaSm\\\\\\\/figINBFaluMukwLs24nCXWQu2bewajauwZxSG4xgqTJyvAUgedMz0ab26QM\\\\\\\/s7S5MglwfwkEn3KZjgktegO06rFw61ec\\\\\\\/6QYLN41vo\\\\\\\/abawpNedVYzKv7NAmCGo8urikCFaMIZFrWmJxocVaH0HgH+9dJuSQiYD44DuHcjw335MStEctZlPDdlw39K6xxG9JXEmxHT14RInUdREYEWp\\\\\\\/wCoKZoi2sThyxFbfuzuYLOVFA6+VzhBab5lUDpsFTGZeNdy\\\\\\\/ZQVHQ7g3ZLNTh7BKk1Gd83+eF5DcWaeuku50XopqB8cI+jDAX7trR1NpvZRrwOrzzVNDiIzG\\\\\\\/9\\\\\\\/U390EAS1D5M5ReRlHykth3+nATmNA0HBY2E0em7KVy9Ll8TvmidIAnckY7\\\\\\\/DMbDAVlVfjPHoxj+5YVDO9Qbzn\\\\\\\/YBsXIEe+JDfNwxpEZsUKSXc0AKtmtxUmtSjwdklcDUWBntXooEog+stFe2a6QawICHtLIvq2w7Z4eXeaysN8AVPQJinWJovHltnFW8dfp67W3DXEGTtZP5A1EGQAdKHW20SQtQjs1vDt1qyTXERlDep5F8ai+ZDNq4xcQo9wg8RBspvu5GvM4xko3YW\\\\\\\/Ixu+36LcpCfza2rRBFbqcBReYQXXZqg\\\\\\\/uSlYAfVG+uD9KbfXChfFkWjLIaxASbXvr0C+vithKCl2QhMFJqDVIlKXNYNo0lWvQyEW9PdmyoCTjKJ36MNiOw5XgRmktd4v4w\\\\\\\/ogaLuTWRUs9LujkvLAFzkQTyBomVjJ8LSPM9E+UXWa3IAklNMVHPa\\\\\\\/\\\\\\\/f5FEsSb\\\\\\\/O4j2M73n3ULRGdFr83o0qqhqDxzJa891rfGsJAOmXfIpS8ItSIAcV7RKUShR7GeaTMjpM2TTiXnKlyun3Dlgk5STfprYtPIq\\\\\\\/KddNwd0Fsrx8zCv\\\\\\\/vIXyIdLZxqWwHjBSWFORG07FlRPtnjjthB1K2i13iiI+BGlGO64ycKIeZseMrzRUzZd9iGovbcnstlkOBbPtQPY57Gru4j5ewDkUria+gSVYkhPAF0hlQIDCTvPrekKiv0Zpa12VWGR7uIQcRrEuZCOekpXMEce7Dg6M\\\\\\\/yneHY1nS+Z3VhCFocA0w0aFuyPgNR9rX0UR4PPaQbAOy0\\\\\\\/lxjoszlhzOQdd6Zs\\\\\\\/X4JxvqvuefeXcjFO0FRq9uSpFLazUj2Q5GETVSlWDZAVAro4PLi5j\\\\\\\/6zy+1yc2gALpHfVvZmWAZIMYc9NRPDIQPppX\\\\\\\/2WyzXvyFe8LMy5rcGmWQCTHYYAw03hzWQyldIMQa4F2\\\\\\\/pIyy+6yE0zUm8sJogBg37XAEw\\\\\\\/4NKecSWojPNG9bwSpgE+YSmoGW3WEDTdU1X\\\\\\\/6laQnSr84YThTtIy\\\\\\\/CNuYPWC\\\\\\\/va1H+7sCGzHs3quRQxGiGIgW6+NpvVIT\\\\\\\/4MGBt2t29NUn7G9dkb1e53SYqPuymFlJagcsylwEa6EGbuOq7DaTuDAw7BNN692t80Y0F+mOzk+5oPbLkonBZc5tG0k7FZp++3zz9PyA3AffLjubdLDB+ipnpnoYqUiHd9RG1YcRzpCiYp4qDfl9GEXozTAv0oDZWVb\\\\\\\/yq57qPDkG\\\\\\\/j9aUgXrTwal+T2AmXGGUvplAhd8z71G5+70425\\\\\\\/J+FqZleCc8llk4MnO7zRZ56kohGTi9TKXCBQ+oNn\\\\\\\/uu3CbUY70bajo0OiVl0x1060xMy\\\\\\\/iGkVM4c0bkrWDtif4QzJp9ddANg8dcblNWEzqLwIU3mEleIVunPUlM26vlFcpH\\\\\\\/Xdg0R6oUzUAswYf+VzAYfrKAxKc\\\\\\\/zmC75RDUhqrnpHo2C0Wi4CF5A5CroR5wnNFAvqULuVR5sfllMHnzrJZjwHUzlrzwWc8Vgk5ulmhAFNNPAqKUZjcpW0n41sgA55E5K6i1Xdpw9SXfaABm0t2NaIINNjPG+Y5Scjfz7cn0B5UvxwH+oJjrasc0GJ13eYzZCza+6MaCjc5HHz\\\\\\\/XGZ4m0is\\\\\\\/yFuT4j+CkjRNs0OXxmWRNmT7npcttTlTq7c4Z7u4s6UZd9VDJoualk1pEre1ZVbGXZfYI\\\\\\\/m7VrZ9r9uRRaHI4RsBbAmvhiFk0OJ0tyv5ApkSOVp2AquvZSrer9niFFyN1Oc9jzgSoIj5\\\\\\\/wIaL42m5azK\\\\\\\/HuwcOPgRbBv77KwrtUOCrEigLEyXeIVjPwuWsD4hmuJdqzyeAqk9aeXaLg2\\\\\\\/P9037EXGzi5dDBZ\\\\\\\/JreucTnEsEZS+WUc+ML+L7L9XvzsSxWX2hbuIsvZRIQw8lzEnIrqEfh9SnyV39+Yhxm\\\\\\\/yJFr5maR2vVjluYyUEmdMNJ2yqkKXm7qdbHiot5MzTwSbt7MQu1hXXAakqogkU9DcORGyHVUHf4VixaeKy4tzb7Gwxdqpj8y+5NWCSrjoV8til8ArxBiNo1Kdmwr9TmEHTMNSR2vJgrGFWddIQlj7Ly\\\\\\\/Pngy2xZs\\\\\\\/3QRj8b1lXio3DRBdAP10Uja9MYnEeKX6EMxazc9wUFNjIrwz07pCqEVZh18ScJHo1kQ3G5iw6nziMqRL90xNZwx2tcFeWZaF9qDn7cmC3HVsSNvbsU2K5NCn5HySOPHiFYw0yvkHgGOC\\\\\\\/SCxFUDg9+m3ct7aQSVZdp5xrgHIuWWmBBmHdwMPw2z0NIoHTZeuDEIELLrhbeFgYpPvGG1LltpMdZFVxFHw4FiHZBIi1uhIxDPjR+WqY4uv9sIH78p6c1n64CUKV1eRCjwQzzLOmKSoCdmWyS1bpGXP6E7sCb517Fpi3Iu3se\\\\\\\/PxsJVJRbITq5AFgh2AH3k+VsjgEp1X4IRufR9G5dv0EtyPPhcS\\\\\\\/D8XWNQ+IVv+6DfgLWtHBPo9ckTnX5nTX0TEAtvPCQOsPa4A104k4NF1JTJK48yvlwF6fAE8xNNioc5rl\\\\\\\/55il7bm2FWZFou8l2b0WXEU2YG6slecAgLqMk+6hZrqm0KOF6JM2l2\\\\\\\/0j5q6uf1mLGiiODVwXPMuKeljXsaedjBVtxnllRopKXqXxaC3ev9e7+l+gOS7YvcLY8iZHnUoeFLxm\\\\\\\/W6pf3iJ2zGJ++hEsiHpR7lSZxtP5QDusBKeAqZqVXHRcJ35W3URcxmaS2J6WV5kZJ652quKdTI8h6hWoitnoigwjF1MjxkmG5n7nr1PjrmcWZM2fnzgbrH\\\\\\\/caLUQ8cHCvlL3FtpwAiu9IF5WTS5xWPZ\\\\\\\/eMtlYphogZ\\\\\\\/dKTjBlQ8r16Di0FhVvAkrlADCTAxBi3Zpx5Rs\\\\\\\/nxa8wQJer50qNv0s+bePbpLmK\\\\\\\/UdgKVEj330CAihsnl18LJ+hM6XPmZj5+bdP6J4PgBH6WvUTpLLKcuYA+7lTLtZnIU2ejQ5b7gPIBVsHokdaZBBqJrz95WoCEZvajUpBO0u6HqGzXOL+QYObQsfbnC0FKSkJxSDtW1uhXzLUnPwIp\\\\\\\/Ppd1+BdXnEOCoZTHoPH8rZAiKJNWjGZGea8FdF47d6SGf0SfJceQmButTuXis\\\\\\\/Ryh1KAEv92OSHkh+jawIQDPyUFJePz0wQld9oelS0fEi2VQPm2DPhB9b6gujAX3aAQHQGXHTS8vy3l5NngRZkSjU+NCj4CI+fmemd56NxjR5fPwM\\\\\\\/ul1u2uG7j8YaUCFwIJgmiQcv\\\\\\\/+D3RogVflS08L2P6r2r\\\\\\\/C5lEuSlNZSA4BWeTFfD3VoFvbG5GLMJMNYsTZIzP3xqp81zrUYiMB7wAoGlKJrAJcLNl\\\\\\\/k5a6pebaWn9NSD9JItQISNgp782xRfgf7hvPAdpvirl71vg\\\\\\\/sB8E0F45wMXdQtqWgKObpOeipV+MPb0a\\\\\\\/d\\\\\\\/zhriwH4UtJl75N07OtmlRlkL\\\\\\\/uX3Ye0xNzy7O4fwh+5HhqoFlzLX2A9SZ3eFASBMX6ubJzAeoF7j0\\\\\\\/mbmDT85IzGMBHzhIe2ARTrwBxU2KWRt3bDMPpNqyYVz83w01qsDax9vkN\\\\\\\/rBO8id2f6ltjrl1c3cLBv8FzZtcmPYhRPogLvW0vWfPYQ2AU6qpYeYGMQ\\\\\\\/SxrGqd4D8VIXbAfNGeEotSr\\\\\\\/7\\\\\\\/6p+9IPdWNz6xUKSEli4Vycwk9Uwb0rKaudPq5WiOAb98E0ksyf+nJxgC0RJB3twRQXA8+zsyg0pS7b+e6plhLvdJFYoTvFBtIsp8bRklcAi9LH8r3iU0qsQcqUAB3OYpYtw60XaeeHCmkzRhskJ+GUQpuCkonbcwpId1\\\\\\\/JfwRUVoJMdkXE8DySXHNqG1KiNndDxQMjMQ9VElvSZ1d79WaLK+OBJVJc5xHY2tA0ZEJChQ3UlxSKKL38L6PS57eMBF7o\\\\\\\/K22z5U++0v3gixMdZqm5u+cJRZPpR4TO8qeIEfD\\\\\\\/DVo5HJ1kS6kYE3aM3VMN+S72qe2+CAc8ilZYsovUgs9aFV7QpqsBknV3Mw6T5LSIosLaFAAeHB6h\\\\\\\/m1KY12++RUYRZWezdz4Bshy0ZANoi4QmNdIvA7VljmjbZeBQ4hmu6I7VYEYKCk+NEpTauRxCYGW8Pts7GLSMMq4ncA8py2sT4i01QcOlnNQ0eZXE8z1858ceBWkA8OK4OKCoCmK0b1Fxcmd90yT5nHJcl1ijCxuMFBR4gDGlEabnZEM1HSiznR2TTrPy6FN9TjEv6a+fxmu8o1rBvkak6DxTloULlcVqdnDrOrfGX9T7\\\\\\\/fM2jz5CZfPC8mmuyJoqqwh4PRWHItoTnmFVtIe3IFLjLrAHPv\\\\\\\/Tq6Z62b6nJ7odMT1a5Hkp+EXSkT5VrbgtykBc9C9IlOkTTOo87nmYjeE5IiL9wHMaDm0r9GwNHZNC4bNlr33C9j55247olKzvKRrFxyR9a7eZ875iS2ZcH3hpW5JoEsM+xHeIU7d6EaiFWiN6qmcvZYOen3K4lugllTfxpPVqAJP3I0oFgUgaZunR75HWYHzet1v+8zrZ8nBTMPvQZLLWDmr\\\\\\\/58LFHmhw6tVp9hAXvc3djuSHz97YGZOmUfdk2gf8N1lEynK7Tp3g3XL\\\\\\\/0ZGC7xY9uNJ9Pm3L+JLN7GSKi479+Bf48R7ewUMlH0c3Cl3v7MYydyqjue5LtQ8vuhxzfttGR0olxb1VErpCK1Qa+HfMtbg7KKRBlG+F5oS\\\\\\\/HfGJnxlYkoAB4M6j1vov\\\\\\\/kHxuDjmXbPmMtwIfRyWktojBgOwboJglN+OklQ5ARUqGNjgz6RULYkyTL5umzZnD3pXW4rbx3sBb6fUKnnZxKxrqrT1f3mZYkolum3iUBWzHhHBQU2ObuPqJoemO7D8Ef52q0Y\\\\\\\/T4u50fN9YNXbpqCy3B3hAE509fm7c4iA81Ajm\\\\\\\/AMkYYwD6qCjh240r8R+kAbyrqAQAzRcITLgB+CcDcstc9Iz257lMfh4pwxMmLDs44ODERa07jPufv9F02e5apxij9bzUJEnX5rQEULpu8rm54MWwXJK9twHEVgxZKDrRjncO\\\\\\\/YqazBncWkN\\\\\\\/D8Dkn54c\\\\\\\/V5CalUE3Bu7P9PCrvx5jcWy4RqXEJ7iqXHKFmXB8KC0r2aHNx2p5+ms+F2eRyynswRZ1rMLX9fadbLZJ\\\\\\\/MQmLhNowFrkljF1W561Lkac7CUNaX9gqOTfG69tPvAQos+EQUDJtkQ\\\\\\\/dgvWoOEyNTJOUz90kpB5Hwr4yJzeNR\\\\\\\/HRkFQPxzBbfKrA8GAEu\\\\\\\/nyWxUbEBN\\\\\\\/qrFCqmNctiLcJ4ZUyCAr9Q4HkzIFUit3ehoLXcvh8ig3LgMQjRTSpQdsgI9\\\\\\\/LY5hb4gCfN3erQ9zEKFeySec4Lp8xKxCmDZqB\\\\\\\/4B4eScMo28OuwH\\\\\\\/2FcRjd4YIHlWVi2XrBhI8K\\\\\\\/aD1AliV0Oh+tzEBcKpmMl3qlE9pEAe4SXkUN6OBYjMyovucXBAxSRvFnT5lth0\\\\\\\/x6BrRWEG1AZswMuKABYKuuLRYEJgfjY6ok4ZSEd9YfJb7ipWw56kccfcQEhMPbQlT46nbm7hhz8a5jCmO71t+Sh0izxoxr9yyEP7sXbGQXdWYdpOMSnkXohQL+SsGZ5wxpRF5Afx2lpfbJBISVQlY6SnNYb1ercuy72u1barN\\\\\\\/LHYcc8TxeoB8\\\\\\\/rHCXFJOT8t3L9NF2cJPV0mOpYGG+FjnA6\\\\\\\/\\\\\\\/sgOyKcI\\\\\\\/d6CQr5hNoItizlPPm5ZciUGXXbJcT+UfEg+kGIRERnNzOOioBciz2A3FEIoVDT3Il\\\\\\\/HhRarrDeu9YCog6r2GNTwNgjsknn1c4xlQiYEo7F8RGlORhJw\\\\\\\/M0jc+IkkrXqF99oRUxtdXEIDx0TTpxG\\\\\\\/nMsFNYGBo++oMEFQhV\\\\\\\/hLizlC3w83qDjlVks\\\\\\\/soojCtwldQ5mo1YytxNGN778yXBXVAugZfyi8i8SO1IUKql6RXJmdheN1mtr6NsToY5IYrqS8WT3w5X2egYeGdOVcoOp6OyJMAtGn2JdIquAPbP9TdnCl8P613x2CH0E2yC\\\\\\\/WmANegoTPq3hYIvr3pWbuV57Hd8MSg\\\\\\\/X3PfoqK9kLBMNL8k4JFA2KED\\\\\\\/VFBSO6VfuXd4EUYh6sG3x88OOeRHWUe4ezU6anXIWkzP5VblFdCB2K1EvXiURZEMNqraNbkfDsSzYKsm1UYO5gc5S2gpaoYNO2IAIkH+evIK1VwQWLmNIcqmLLxiwjp1k4UD383doixcXYZFuMJLNV+WicnIogCFJOynYpPqTxY0XDCnVXPk\\\\\\\/f9914UJ\\\\\\\/sTU3S3x4B04fEQTeQYe7xApnzctEb7oXtfKjvO8ywKsI0Hcf8bUh8rsotK4RC88ljuyiUZMO37++J7AVZZYlZYaXawXI10kzM17L6Taq8BRyxG\\\\\\\/Q2ndm2Ho+J5MvY72qkSWmA55VQLSW0GEJso4EvcQadQirmaW1LrIWpE\\\\\\\/vHrYZkf+BhXvDfNVP21aejOl7xnsgiCTAkJXic0lhh9aS7bjyEZp1LmZ7bpgSgUcKemycU8HgTx1X7Q9PGN9hN6t1UPkX27SOfO+4uXtujxwDocnzFvS1CQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022VYe7NS\\\\\\\/Tbx2E8LwGfVz7Uw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002240580d3697a32f9d92a54b10b5948e762ef558cde37571c54d62899c863defd1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221PfUvkoju6reiSOITIqsLPr7t47zxmakuksV8TJhUliD+NvK3m5l0PrVJ56oHTy0rWS9gBbv8MvkI0YEN+mNwlpg\\\\\\\/o+VygKnAX7Qrmx3NPbLoIW63fVoOA\\\\\\\/6oPnctxgc86Fc9wylD7SNDZ0kb15W8DdVjB5dOJfCOunPFxJhvF89c8UaKEOCYh1uSPasXD3HKbmfgwhxTWGDl2BdwM6KJMbTR3e6uvSxUtJJ09Lyez5oeZMoe1qIN9no1MMew+HjiZOq1K3UUcBc6nScwJ4z8RU+vHGMJuaovfnReBCIg680fJgei4n+hRDJNg9uePzj17qus\\\\\\\/XoVNrLu4Qu6FQSbMMgGcJvCdzZVxrICYYzJalwHxIOYYsKLTj\\\\\\\/ryQPp8QCQ2Sqxw\\\\\\\/wOxiiuXTGRFyL78ERdm1cT2QgqsKioAsMv00Fi7LjUy7kUjbci6ImDDv74uuJponSnc\\\\\\\/teSXYdjvWQnzDmyRn5vOjDQlxjkqkJXRLeJXFmzFHw8XdkCDtj5zjDyDyDs9Pm5tQauUcB2tzi3wPL5uhNlhg7cK0r5X+TVW3FHqQ7j5SE6DOklKrkkUn\\\\\\\/5zO4uHgAw6zhhiPKAgJxTWl0vD7jwAF73QivLMvaw+TCkcJDefG\\\\\\\/glhKHoZt89AO3IAiGWLX0fK9Mn4zLH0yaz8W\\\\\\\/Dm0Jo+auXviPnP\\\\\\\/UYwqgyhwMQZa42nluznbNSUiCdx0HjYXfVtiV0kDTu61k7sJPhV+WhnQTlwcGsTI82PeNhdPW7XzezqCPImNSh\\\\\\\/hv\\\\\\\/qAvgVCb79wAw0Lm7QTPZD4DP7am4r\\\\\\\/6QbjbjJJF0veobipDKx7EQC5fUmUT7Y7odBdI0r9Osfmml9kfJKqSlMxHi8cTYJoMwXiA\\\\\\\/AG1nS8AT9CRBwb\\\\\\\/dy8lw0HaEVFDzT8nFmw82KtTNg4mvNbCZBP+WxLsIO47k1eMwryLR6URyzMxlbRAmMr\\\\\\\/WJmLLwNbUWAKgSV0NldsHWTT7myaV9ThLEUIJBMUza8z4npLIae4yr5RbNGdSENN8g0oNwWUSLf0NtAp+YsO1XryasJFqFxv0kIp1Y9cl7Ib92PABORTPWoAqLAewXvJJIpZWPuS6Rlvku2SBbDtHfOEZB3EMa9LVTtolcFe1LPuW9nyFupGGJoq6HkoAKHm76Eo47UtagFpMe2BHyyoIG3plzjSuqGEz6csI8ZbD7pOmBoFVR3VAzYB3hiNBmvJbkJSY8MXO4nrCj+5U9oDX8TT5rliKN4C6I+eHNof7DgHFP0ebzCvU\\\\\\\/BpSD2MZLCMTLFbupsz8ca0tjbffqqtesXzDf41hN8ZmCUr6a7n73w1pSdYZ6ReFc3xQHTD1tIJr7B6zQIhRnpsQC2a8T6ssbEQ62iYwR0Sbh9bDj5YRYQgUo+3gTq7W1FVswN4BHlBcFs2Lrmud6PDz\\\\\\\/ezHnebSj2GxNmchSLKmNIJT6hh7dmVPAPQgY8V0O6y4tHc7gUUeLVdeljsftuvSmosFAtzWhj9NiEQqZ135FTp7yCLQeMWRhE+65UhLlDu5qM4S31n6qhlQyY8dehP8tXK078M0cCPx8QvkwIJTgmk2Ji6Il87eVs+M7yvJa+A3lJijN6tlqq1Z5vX55n\\\\\\\/hDLXfzm4t5PmiFtzceGgElEQwM5B1\\\\\\\/DLHkJNiWifgUffENrYqJoCh2fJaF+7zg8qfB5395xgq9Qa3ewDYbY+0k+aJoVcFYYL2UTRnu4W5NmhjnwfJWsrpxUOcbaC9WI\\\\\\\/thR8LBiWp+dKGRJmaFNqD+9Mvux50r501N0rgX2\\\\\\\/z\\\\\\\/ds8VpOmoNr0fpmm\\\\\\\/c4FjRNkqUYOzOCpc2EqoEC89HADUKjyjjF41Qfsl3qquXibCRYm+WcOUG+MHBpwBLWgmJnHwy9PV\\\\\\\/ES9xnY\\\\\\\/tqqfYnL8614VwTlEgX5moRxg8uctVI+5KI5jk7\\\\\\\/uIAG7mk\\\\\\\/El5BrYStAl2lJESXuSFzpr9DTn6DrorhYjWlyusWfHSwxWhN797pcrGU5Y89hFqDM2VpZkOA9q+MKzfc9jA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022wOYjlLfnXO\\\\\\\/FVAfghdVwDw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002207c9e27e70a1887fc1ef91514721c97b157a1fec07d1dca9db34de237c05ecaa\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-486779 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3349" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-leiOLkiSnDVYdPQQ7diWe07Tw9Y4fscDTo-a0-zAn_E" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3349" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-W5bdBjERx189JDiS_MofdXKhg3g6kkXsHSwI0al7i0s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/10/3337" title="Taxonomically Restricted Genes with Essential Functions Frequently Play Roles in Chromosome Segregation in Caenorhabditis elegans and Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/10/3359" title="CONE: Community Oriented Network Estimation Is a Versatile Framework for Inferring Population Structure in Large-Scale Sequencing Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486779" id="node486779" data-pisa="ggg;7/10/3349" data-pisa-master="ggg;g3.117.300233" data-apath="/ggg/7/10/3349.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">KinFin: Software for Taxon-Aware Analysis of Clustered Protein Sequences</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7887-0186" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dominik R. Laetsch</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2861-949X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark L. Blaxter</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3349-3357;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300233 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486779" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dominik R. Laetsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolutionary Biology, University of Edinburgh, EH9 3JT, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>†</span>The James Hutton Institute, DD2 5DA Dundee, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominik%2BR.%2BLaetsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laetsch%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADominik%2BR.%2BLaetsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7887-0186" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dominik R. Laetsch</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dominik.laetsch@gmail.com" class="" data-icon-position="" data-hide-link-title="0">dominik.laetsch@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mark L. Blaxter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolutionary Biology, University of Edinburgh, EH9 3JT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BL.%2BBlaxter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blaxter%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BL.%2BBlaxter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2861-949X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark L. Blaxter</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/10/3349" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:486779" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:486779/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3349.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:486779" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:486779/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3349.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:486779" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:486779/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3349.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:486779" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:486779/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/10/3349.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:486779" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:486779/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The field of comparative genomics is concerned with the study of similarities and differences between the information encoded in the genomes of organisms. A common approach is to define gene families by clustering protein sequences based on sequence similarity, and analyze protein cluster presence and absence in different species groups as a guide to biology. Due to the high dimensionality of these data, downstream analysis of protein clusters inferred from large numbers of species, or species with many genes, is nontrivial, and few solutions exist for transparent, reproducible, and customizable analyses. We present KinFin, a streamlined software solution capable of integrating data from common file formats and delivering aggregative annotation of protein clusters. KinFin delivers analyses based on systematic taxonomy of the species analyzed, or on user-defined, groupings of taxa, for example, sets based on attributes such as life history traits, organismal phenotypes, or competing phylogenetic hypotheses. Results are reported through graphical and detailed text output files. We illustrate the utility of the KinFin pipeline by addressing questions regarding the biology of filarial nematodes, which include parasites of veterinary and medical importance. We resolve the phylogenetic relationships between the species and explore functional annotation of proteins in clusters in key lineages and between custom taxon sets, identifying gene families of interest. KinFin can easily be integrated into existing comparative genomic workflows, and promotes transparent and reproducible analysis of clustered protein data.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/bioinformatics" class="hw-term hw-article-keyword hw-article-keyword-bioinformatics" rel="nofollow">bioinformatics</a></li><li class="kwd"><a href="/keyword/protein-orthology" class="hw-term hw-article-keyword hw-article-keyword-protein-orthology" rel="nofollow">protein orthology</a></li><li class="kwd"><a href="/keyword/protein-family-evolution" class="hw-term hw-article-keyword hw-article-keyword-protein-family-evolution" rel="nofollow">protein family evolution</a></li><li class="kwd"><a href="/keyword/comparative-genomics" class="hw-term hw-article-keyword hw-article-keyword-comparative-genomics" rel="nofollow">comparative genomics</a></li><li class="kwd"><a href="/keyword/filarial-nematodes" class="hw-term hw-article-keyword hw-article-keyword-filarial-nematodes" rel="nofollow">filarial nematodes</a></li></ul><p id="p-3">Inference of gene homology across taxa is an integral part of comparative genomic analysis. Candidate orthologs, paralogs, and xenologs between taxa are commonly identified through clustering of protein sequence data using tools such as OrthoFinder (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Emms and Kelly 2015</a>), OrthoMCL (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li <em>et al.</em> 2003</a>), and others. Exploitation of ortholog definitions across species, the study of gene family evolution, genome evolution, species phylogenetics, and as loci for population genetics and ecological genetics, is demanding. Many research projects aim to identify orthologs of interest that have a specific distribution across species, for example, identifying gene families that are synapomorphic for, or that have been specifically lost from, a particular clade. Exploring the effects of assuming different underlying phylogenies on the analysis of the origins of orthologs may assist in discriminating between competing hypotheses. Grouping species by nonphylogenetic classifiers (such as habitat, mating system, or life history) may also identify protein families uniquely present/absent or exhibiting differential copy-number. Several high-quality solutions to orthology analysis have been proposed. OrthoDB is a high-quality curated orthology resource (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zdobnov <em>et al.</em> 2017</a>). The current release (2015) includes 3600 bacterial and 590 eukaryotic taxa, and is accessed through a responsive web interface or direct download and interrogation. OrthoDB includes rich functional annotation of sequences. While the main database includes only published genomes, and is centrally managed (<em>i.e.</em>, users cannot submit datasets for analysis), the OrthoDb software toolkit is available for local installation and deployment. PhylomeDB is a database of defined orthology groups, built with manual curation (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Huerta-Cepas <em>et al.</em> 2014</a>), but was last updated in 2014, and is, again, managed centrally and focused on published genomes. In the ENSEMBL databases, the Compara toolkit is used to parse gene homology sets, and infers orthology and paralogy based on a given species tree (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Herrero <em>et al.</em> 2016</a>). Updating of Compara analyses is not trivial, and requires the ENSEMBL web toolkit for display and interrogation. For ongoing research programs, few tools exist for orthology analysis. For bacterial data, several tools for pan-genome analysis have been developed (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Vinuesa and Contreras-Moreira 2015</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chaudhari <em>et al.</em> 2016</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Xiao <em>et al.</em> 2015</a>) but solutions that cope well with the data richness of eukaryotic species are often tailored to defined taxonomic groups (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Song <em>et al.</em> 2015</a>) or expect closely related taxa. EUPAN is a pipeline for pan-genome analysis of closely related eukaryotic genomes developed within the scope of the 3000 Rice Genomes Project (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hu <em>et al.</em> 2017</a>). The approach parts from mapping of raw reads to reference genomes followed by coordinated assembly and lift-over of gene annotations for inferring presence/absence of gene models. In the absence of toolkits that allow local implementation of clustering analyses, custom taxon grouping and dynamic analysis, we have developed KinFin. KinFin takes a protein clustering output by tools such as OrthoFinder or OrthoMCL, alongside functional annotation data, and user-defined species taxonomies, to derive rich aggregative annotation of orthology groups. KinFin reads from standard file formats produced from commonly used genome sequencing and annotation projects. KinFin can easily be integrated in comparative genomics projects for the identification of protein clusters of interest in user-defined, taxon-aware contexts.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><p id="p-4">KinFin is a standalone Python 2.7 application. A detailed description of the functionality of KinFin can be found at <a href="https://kinfin.readme.io/">https://kinfin.readme.io/</a>. Required input for KinFin is an orthology clustering (format defined by OrthoMCL/OrthoFinder), a file linking protein sequences to taxa (SequenceIDs defined by OrthoFinder), and a user-defined config file. The config file guides analyses by grouping taxa into user-defined sets under arbitrary attributes. These attributes could include, for instance, standard taxonomy (as embodied in the NCBI Taxonomy “TaxID” identifiers), alternate systematic arrangements of the taxa involved, lifestyle, geographical source, or any other aspect of phenotype or other metadata. KinFin dynamically constructs sets based on the config file and computes metrics, statistics, and visualizations, which allows identification of clusters that are diagnostic for, or expanded/contracted in, each taxon set. Optional input files include proteome FASTA files (to extract length statistics for clusters, taxa and taxon sets), functional annotations of proteins in InterProScan (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jones <em>et al.</em> 2014</a>) format, and a phylogenetic tree topology in Newick format.</p><div id="sec-2" class="subsection"><h3>Visualization of ortholog clustering</h3><p id="p-5">In KinFin, global analysis of the clustering of protein sequences can be performed from the perspective of the clusters themselves (their properties and patterns) or of the constituent proteomes. The distribution of cluster size (<em>i.e.</em>, the number of proteins contained in a cluster) is an important feature of analyses, and KinFin simplifies the comparison of alternative clusterings (for example ones using different MCL inflation parameters, or with overlapping but distinct taxon inclusion) by generating frequency histograms of cluster size, which can be interrogated for deviations from the expected power law-like distribution. To aid understanding of the distribution, the user can generate a more detailed frequency histogram which considers the number of taxa contributing to each cluster (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The behavior of individual proteomes can be explored by creating a network representation of the clustering. KinFin can produce a graph file with nodes representing proteomes and edges connecting nodes weighted by the number of times two proteomes co-occur in clusters. Optionally, universal clusters with members from all the proteomes can be excluded. The graph can be interrogated using graph analysis and visualization tools such as Gephi (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bastian <em>et al.</em> 2009</a>) (see Supplemental Material, Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300233/-/DC1/FileS1.pdf">File S1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3349/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of cluster sizes in clustering. The distribution of counts of proteins per cluster is colored based on the number of proteomes present. Total values of counts of each cluster size are indicated with gray dots. A fitted power-law curve (gray) is drawn for reference." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1327459055" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of cluster sizes in clustering. The distribution of counts of proteins per cluster is colored based on the number of proteomes present. Total values of counts of each cluster size are indicated with gray dots. A fitted power-law curve (gray) is drawn for reference.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3349/F1.medium.gif" width="440" height="213"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/10/3349/F1.medium.gif" width="440" height="213"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3349/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3349/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486933" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-6" class="first-child">Distribution of cluster sizes in clustering. The distribution of counts of proteins per cluster is colored based on the number of proteomes present. Total values of counts of each cluster size are indicated with gray dots. A fitted power-law curve (gray) is drawn for reference.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Analyses based on arbitrary sets of input proteomes</h3><p id="p-7">Through the config file, the user can instruct KinFin to analyze the clustering under arbitrary sets. For taxonomy-based analyses, KinFin derives analyses at different taxonomic ranks (by default “phylum,” “order,” and “genus”; can be modified by the user) by parsing the NCBI TaxIDs given for each proteome. Any other classification of the input proteomes can be given, and nested taxonomies specified by use of multiple, ranked attribute types. This allows, for example, the testing of congruence of clustering data with competing phylogenetic hypotheses regarding relationships of the taxa from which the input proteomes were derived.</p></div><div id="sec-4" class="subsection"><h3>Classification of clusters</h3><p id="p-8">KinFin builds a series of matrices associating clusters and proteomes, and clusters and arbitrary proteome sets. Each cluster is classified as “absent” or “present” for each proteome or taxon set, and is assigned a cluster type (“singleton,” “specific” - where there are ≥2 members and all come from one taxon set - or “shared”) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3349/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Count of proteins by type of cluster. &#x201C;Shared&#x201D;: clusters containing proteins from multiple taxa, &#x201C;Specific&#x201D;: clusters containing two or more proteins from a single proteome, &#x201C;Singleton&#x201D;: clusters containing a single protein." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1327459055" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Count of proteins by type of cluster. &#x201C;Shared&#x201D;: clusters containing proteins from multiple taxa, &#x201C;Specific&#x201D;: clusters containing two or more proteins from a single proteome, &#x201C;Singleton&#x201D;: clusters containing a single protein.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3349/F2.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/10/3349/F2.medium.gif" width="440" height="292"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3349/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3349/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486918" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-9" class="first-child">Count of proteins by type of cluster. “Shared”: clusters containing proteins from multiple taxa, “Specific”: clusters containing two or more proteins from a single proteome, “Singleton”: clusters containing a single protein.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Single-copy ortholog definition</h3><p id="p-10">Clusters composed of a single protein from each proteome (<em>i.e.</em>, putative single-copy orthologs) are useful for downstream phylogenetic analyses (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). However, due to the intrinsic difficulties of genome assembly and annotation, the number of single-copy orthologs decreases the more proteomes are included in the clustering. To compensate for this, KinFin can identify “fuzzy” single-copy ortholog clusters using the parameters target_count (target number of copies per proteome, default “1”), target_fraction (proportion of proteomes at “target number”), and lower/upper bounds for proteomes outside of target_fraction (min and max).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3349/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Phylogenetic tree based on 781 single-copy orthologs. Nonparametric bootstrap support for all branches is 100. Internal nodes are labeled. (B) Table summarizing &#x201C;complete presence&#x201D; synapomorphic clusters (100% taxon-coverage) and &#x201C;partial absence&#x201D; synapomorphic clusters (75% &#x2264; taxon-coverage &lt; 100%), and the percentage for which a representative functional annotation could be inferred. &#x201C;N/A&#x201D; is used for cases in which nodes are ancestors of more than four taxa, or when percentage of functional annotation could not be calculated due to lack of clusters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1327459055" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Phylogenetic tree based on 781 single-copy orthologs. Nonparametric bootstrap support for all branches is 100. Internal nodes are labeled. (B) Table summarizing &#x201C;complete presence&#x201D; synapomorphic clusters (100% taxon-coverage) and &#x201C;partial absence&#x201D; synapomorphic clusters (75% &#x2264; taxon-coverage &lt; 100%), and the percentage for which a representative functional annotation could be inferred. &#x201C;N/A&#x201D; is used for cases in which nodes are ancestors of more than four taxa, or when percentage of functional annotation could not be calculated due to lack of clusters.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3349/F3.medium.gif" width="440" height="246"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/10/3349/F3.medium.gif" width="440" height="246"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3349/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3349/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486915" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-11" class="first-child">(A) Phylogenetic tree based on 781 single-copy orthologs. Nonparametric bootstrap support for all branches is 100. Internal nodes are labeled. (B) Table summarizing “complete presence” synapomorphic clusters (100% taxon-coverage) and “partial absence” synapomorphic clusters (75% ≤ taxon-coverage &lt; 100%), and the percentage for which a representative functional annotation could be inferred. “N/A” is used for cases in which nodes are ancestors of more than four taxa, or when percentage of functional annotation could not be calculated due to lack of clusters.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Rarefaction curves</h3><p id="p-12">The concept of the pangenome is frequently used in microbial genomics to describe all the genes, both core and accessory, that are found in the varied genomes of a species. The size of the pangenome can be visualized using rarefaction curves, and KinFin deploys this framework to visualize the size of the pan-proteome of the different arbitrary sets defined by the user. Curves are calculated by repeated, random sampling of the proteomes in each arbitrary set, and cumulative summation of novel nonsingleton clusters (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3349/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Rarefaction curve of proteomes within sets defined by major clades within the onchocercid nematodes. n11, node 11 (D. immitis and Onchocerca species); n15, node 15 (W. bancrofti, Brugia species and L. loa); n16, node 16 (L. sigmodontis, A. viteae, and E. elaphi)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1327459055" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Rarefaction curve of proteomes within sets defined by major clades within the onchocercid nematodes. n11, node 11 (&lt;em&gt;D. immitis&lt;/em&gt; and &lt;em&gt;Onchocerca&lt;/em&gt; species); n15, node 15 (&lt;em&gt;W. bancrofti&lt;/em&gt;, &lt;em&gt;Brugia&lt;/em&gt; species and &lt;em&gt;L. loa&lt;/em&gt;); n16, node 16 (&lt;em&gt;L. sigmodontis&lt;/em&gt;, &lt;em&gt;A. viteae&lt;/em&gt;, and &lt;em&gt;E. elaphi&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3349/F4.medium.gif" width="440" height="221"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/10/3349/F4.medium.gif" width="440" height="221"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3349/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3349/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486830" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-13" class="first-child">Rarefaction curve of proteomes within sets defined by major clades within the onchocercid nematodes. n11, node 11 (<em>D. immitis</em> and <em>Onchocerca</em> species); n15, node 15 (<em>W. bancrofti</em>, <em>Brugia</em> species and <em>L. loa</em>); n16, node 16 (<em>L. sigmodontis</em>, <em>A. viteae</em>, and <em>E. elaphi</em>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Pairwise protein count representation tests</h3><p id="p-14">For user-defined attributes involving two (or more) taxon sets, pairwise representation tests of protein counts are computed for clusters containing proteins from each taxon set using either two-sided Mann-Whitney <em>U</em>-tests (default), Welch’s <em>t</em>-tests, or Student’s <em>t</em>-tests. From this, clusters “enriched” or “depleted” in count in one set compared with another can be identified. It should be noted that these statistical tests test for nonhomogeneity of the distributions of protein counts between the sets, and, due to limited “sample size,” might not achieve significance even when counts differ substantially between sets. In addition to text outputs, volcano plots (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3349/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3349/embed/mml-math-1.gif"/></noscript></span></span>-fold change in means <em>vs.</em> test <em>P</em>-value) are drawn (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). As a visual aid, horizontal lines are drawn at <em>P</em>-values 0.05 and 0.01 and vertical lines at <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3349/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3349/embed/mml-math-2.gif"/></noscript></span></span> = 1 and 95%-percentile of <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3349/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3349/embed/mml-math-3.gif"/></noscript></span></span>-fold changes in mean.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3349/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Volcano plot of the protein count representation tests for clusters shared between taxa at node n11 (D. immitis and Onchocerca species) and node n15 (W. bancrofti, Brugia species, and L. loa)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1327459055" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Volcano plot of the protein count representation tests for clusters shared between taxa at node n11 (&lt;em&gt;D. immitis&lt;/em&gt; and &lt;em&gt;Onchocerca&lt;/em&gt; species) and node n15 (&lt;em&gt;W. bancrofti&lt;/em&gt;, &lt;em&gt;Brugia&lt;/em&gt; species, and &lt;em&gt;L. loa&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3349/F5.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/10/3349/F5.medium.gif" width="440" height="292"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3349/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3349/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486914" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-15" class="first-child">Volcano plot of the protein count representation tests for clusters shared between taxa at node n11 (<em>D. immitis</em> and <em>Onchocerca</em> species) and node n15 (<em>W. bancrofti</em>, <em>Brugia</em> species, and <em>L. loa</em>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Analyses based on functional annotation and protein length data</h3><p id="p-16">KinFin integrates functional annotation and protein length data into analyses. If the necessary input files are provided, KinFin generates output files tabulating mean and SD of sequence lengths, domain, and Gene Ontology (GO) term entropy within clusters, and the fraction of proteins per cluster which are putatively secreted, based on “SignalP_Euk” (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Petersen <em>et al.</em> 2011</a>) annotation. Additionally, for each cluster, all matching domains and inferred GO terms are listed with description and information regarding their frequency within both proteins and proteomes in the cluster.</p><p id="p-17">While inference of functional annotation of a protein is relatively straightforward, no clear standards exist for inferring representative functional annotation of clusters of proteins. We provide a script with parameters domain_taxon_cov (minimum fraction of taxa in cluster that have at least one protein annotated with that domain) and domain_protein_cov (minimum fraction of proteins in cluster annotated with that domain), to grant users fine control over cluster functional annotation.</p></div><div id="sec-9" class="subsection"><h3>Analyses based on phylogeny</h3><p id="p-18">Analysis of clusters in a phylogenetic context permits the identification and quantification of clusters that are unique innovations of certain monophyletic groups (<em>i.e.</em>, synapomorphies). Based on a user-defined tree topology, KinFin identifies synapomorphic clusters at nodes using Dollo parsimony. The Dollo parsimony method (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Farris 1977</a>) assumes that while multiple independent losses of a gene in different lineages are common, multiple independent gains of the same gene are improbable. The implementation of Dollo parsimony in KinFin requires that only the proteomes under a given node are members of the cluster, and that at least one taxon from each child node is a member. Because Dollo parsimony does not penalize multiple losses, KinFin classifies synapomorphies into “complete presence” and “partial absence” subgroups. The outputs include lists of synapomorphies and apomorphies (“singleton” and “nonsingleton” proteome-specific clusters), and detailed description of synapomorphic clusters at each node. Prominent or consistent functional annotation can be mapped onto synapomorphic clusters, filtered by node_taxon_cov (minimum presence of proteomes as fraction of total proteomes under the node), domain_taxon_cov (minimum proportion of proteomes represented by proteins with a specific domain annotation), and domain_protein_cov (minimum proportion of all the proteins annotated with a specific domain).</p></div><div id="sec-10" class="subsection"><h3>Analyses of clusters containing genes of interest</h3><p id="p-19">Output of protein clustering analysis often serves as substrate for the identification of homologs of genes of interest from a model species in the target species. KinFin is distributed with a script that takes as input a list of protein IDs or of gene IDs (to obtain all isoforms or only the isoforms included in the clustering), and writes tables indicating the counts of proteins in each cluster and their representative functional annotations.</p></div><div id="sec-11" class="subsection"><h3>Output</h3><p id="p-20">KinFin generates output folders for each relevant column in the config file, and writes overall metrics for all taxon sets, detailed metrics for each cluster and results of pairwise representation test, draws the rarefaction curve and volcano plots, and lists clusters classified as “true” and “fuzzy” single-copy orthologs. Resulting text files can easily be interrogated using common UNIX command line tools or spreadsheet software.</p></div><div id="sec-12" class="subsection"><h3>Operation</h3><p id="p-21">KinFin is freely available under GNU General Public License v3.0 at <a href="https://github.com/DRL/kinfin">https://github.com/DRL/kinfin</a>. System requirements for KinFin include a UNIX-based operating system, Python 2.7, and pip. An installation script is provided, which installs Python dependencies and downloads mapping files for Pfam, Interpro and GO IDs from the European Bioinformatics Institute (EBI) website. Instructions for installation and execution of KinFin can be found on the GitHub repository and detailed documentation is available at <a href="https://kinfin.readme.io">https://kinfin.readme.io</a>.</p></div><div id="sec-13" class="subsection"><h3>KinFin in action: filarial nematodes parasitic in humans and other vertebrates</h3><p id="p-22">We illustrate some of the main functionalities of KinFin by addressing questions regarding the biology of filarial nematodes. Filarial nematodes (Onchocercidae) include many species of medical and veterinary interest, and the phylogenetic relationships among them remain under debate (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Park <em>et al.</em> 2011</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Nadler <em>et al.</em> 2007</a>), with the current NCBI reference taxonomy likely to be incorrect. We analyzed the proteomes of 16 species: 11 filarial nematodes, three related spirurid nematodes and two <em>Caenorhabditis</em> species. <em>Caenorhabditis</em> were included because of the quality of available structural and functional annotations. For three species, two independent assemblies and proteome predictions were included.</p><p id="p-23">We will use KinFin to generate a robust multi-locus alignment and phylogeny, and then incorporate this tree into KinFin analyses of synapomorphies and other features of groups of filaria. What gene family and functional differences characterize the different groups? Finally, we will use sets of <em>Caenorhabditis elegans</em> genes implicated in pathways of interest to highlight orthology and paralogy in the filarial nematodes.</p></div><div id="sec-14" class="subsection"><h3>KinFin analysis protocol</h3><p id="p-24">Here, we summarize the protocol used to generate the analyses reported. Detailed protocols including program versions and command lines are given in Supplemental Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300233/-/DC1/FileS1.pdf">File S1</a>.</p><p id="p-25">The proteomes detailed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> were downloaded from WormBase parasite (WBPS8) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Howe <em>et al.</em> 2016</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">2017</a>) and <a href="http://ngenomes.org">http://ngenomes.org</a>. For currently unpublished genomes, manuscripts giving details of assembly and annotation strategies are in preparation. The files were filtered by keeping only the longest isoforms and excluding sequences &lt;30 residues or containing nonterminal stops. Proteins were functionally annotated using InterProScan (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Jones <em>et al.</em> 2014</a>), and clustered via OrthoFinder (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Emms and Kelly 2015</a>) at default MCL inflation value of 1.5 using BLASTp (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Camacho <em>et al.</em> 2009</a>) similarity data.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486837/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486837/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Proteomes used in OrthoFinder clustering and subsequent KinFin analyses</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">An initial KinFin analysis identified 781 one-to-one single-copy orthologs. Sequences for these 781 clusters were extracted and aligned using mafft (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Katoh and Standley 2013</a>). Alignments were trimmed using trimal (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Capella-Gutiérrez <em>et al.</em> 2009</a>), concatenated using FASconCAT (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kück and Meusemann 2010</a>), and analyzed using RAxML (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Stamatakis 2014</a>) under the PROTGAMMAGTR model of sequence evolution. Nonparametric bootstrap analysis was carried out for 100 replicates.</p><p id="p-29">KinFin was then rerun, providing additional classification in the config file and functional annotation data (Supplemental Dataset 2). The topology of the tree inferred through phylogenetic analysis was provided in Newick format, and the two <em>Caenorhabditis</em> species were specified as outgroups. The Mann-Whitney <em>U</em>-test was selected for pairwise protein count representation tests. Representative functional annotation was inferred for all clusters, and analyzed specifically for subsets of taxa. Genes involved in heme biosynthesis and homeostasis were identified based on representatives from <em>C. elegans</em>, and absence of missing genes was confirmed using TBLASTn (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Camacho <em>et al.</em> 2009</a>) against the respective genomes. Presence of paralogs was confirmed by manual inspection of gene models on WormBase ParaSite.</p></div><div id="sec-15" class="subsection"><h3>Data availability</h3><p id="p-30">Supplemental Material from this study is deposited at <a href="https://zenodo.org">https://zenodo.org</a> under 10.5281/zenodo.844364.</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-31">Supplemental_data_1.tar.gz contains input files for basic KinFin analysis.</p></li><li id="list-item-2"><p id="p-32">Supplemental_data_2.tar.gz contains input files for advanced KinFin analysis.</p></li><li id="list-item-3"><p id="p-33">Supplemental_data_3.tar.gz contains results of KinFin analyses.</p></li></ul></div></div><div class="section results" id="sec-16"><h2 class="">Results</h2><p id="p-34">The 19 proteomes (derived from 16 species) included 248,750 protein sequences (with a total amino-acid length of 95,162,557 residues) after filtering. OrthoFinder places these into 42,691 clusters, of which 57.97% were singletons (containing 9.95% of protein sequences). The clusters displayed a power-law-like frequency distribution, but with a marked deviation from this expectation at a cluster size of 19 (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). This pattern, although less pronounced, has been observed before for protein databases such as COG (Clusters of Orthologous Groups of proteins) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Unger <em>et al.</em> 2003</a>) and TRIBES (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Enright <em>et al.</em> 2003</a>), and has been seen in other datasets analyzed with KinFin (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Yoshida <em>et al.</em> 2017</a>). These clusters contain a large number of strict (“true”) single-copy orthologs, and many “fuzzy” single-copy orthologs.</p><p id="p-35">KinFin can assist in deciding which of several alternative proteome predictions is more likely to be correct, and identify major differences between alternative predictions. Analysis of between-proteome linkage using Gephi (see Supplemental Results in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300233/-/DC1/FileS1.pdf">File S1</a>) showed that while some proteome pairs (for example, the two from <em>Onchocerca ochengi</em>) were found to be neighbors, the two <em>Wuchereria bancrofti</em> proteomes were not placed together, suggesting divergence in prediction. Examination of the distribution of clusters within species can highlight outlier datasets. Both <em>C. elegans</em> and <em>Caenorhabditis briggsae</em> have higher total protein counts than any of the filarial species, and display the highest proportion of singletons (15.7 and 24.4%) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). For those species for which two assemblies were analyzed, variation in proportion of singletons is most severe for <em>Loa loa</em> (15.1% <em>vs.</em> 4.6%).</p><p id="p-36">The 781 single-copy orthology clusters yielded a robustly supported phylogeny (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). Rooting the tree with <em>Caenorhabditis</em>, the three nonfilarial taxa are recovered in expected positions, with <em>Setaria labiatopapillosa</em> most closely related to the onchocercids, followed by <em>Thelazia callipaeda</em> and <em>Dracunculus medinensis</em>. The relationships between the onchocercid taxa is not congruent with the reference NCBI taxonomy, but, with a previous analysis using a smaller number of loci (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lefoulon <em>et al.</em> 2016</a>). <em>Dirofilaria immitis</em> is recovered as sister to <em>Onchocerca</em> species (the clade defined by node n11 in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>), and <em>W. bancrofti</em>, <em>Brugia</em> species and <em>L. loa</em> (node n15) form a clade distinct from <em>Litomosomoides sigmodontis</em>, <em>Acanthocheilonema viteae</em> and <em>Elaeophora elaphi</em> (node n16). We identified 3887 “fuzzy” single-copy orthologs. These are useful for analyses of proteomes from more distantly related species, where stochastic absence and duplication can severely limit the number of single-copy loci recovered for phylogenetic analyses. “Fuzzy” orthologs can be used in combination with tools such as PhyloTreePruner (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kocot <em>et al.</em> 2013</a>), which filters out-paralogs and selects appropriate in-paralogs.</p><p id="p-37">We explored the proteomic diversity represented by the three clades within Onchocercidae (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>, at nodes n11, n15, and n16). We defined taxon sets under each of the nodes, and used KinFin to generate rarefaction curves for each set (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Curves for node n11 (<em>D. immitis</em> and <em>Onchocerca</em> species) and node n15 (<em>W. bancrofti</em>, <em>Brugia</em> species, and <em>L. loa</em>) show comparable slopes, and the number of nonsingleton clusters recovered in both is very similar (11,084 clusters for n11 and 10,989 for n15). Fewer unique protein clusters (9393) were recovered when sampling node n16 (<em>L. sigmodontis</em>, <em>A. viteae</em>, and <em>E. elaphi</em>). The fact that none of the curves reaches a plateau suggests that their protein space has not been sampled exhaustively.</p><p id="p-38">Of the proteins used in the analysis, 157,873 (63.47%) were annotated with InterPro (IPR) domains. By inferring representative IPR annotations, we functionally annotated 12,026 (28.17%) of the clusters. Using the phylogeny inferred from single-copy orthologs (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a>) we inferred synapomorphies at each node and investigated their representative functional annotations (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3B</a>). While many clusters are inferred to be synapomorphic for deeper nodes, Onchocercidae and the three groups within Onchocercidae have few synapomorphic gene family births (10 at node n10, seven at node n11, six at node n15 and none at node n16). Of those, only two clusters at node n11 received representative functional annotation: a Chromadorea ALT cluster (cluster “OG0007060”) and a SOCS box domain cluster (“OG0009843”). The Chromadorea ALT domain is found across Nematoda and in several clusters. <em>Brugia malayi</em> ALT-1, the first described Chromadorea ALT protein (contained in cluster “OG0000082”), has been proposed as a candidate vaccine target for human lymphatic filariasis (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gregory <em>et al.</em> 2000</a>). The synapomorphic Chromadorea ALT cluster is specific to <em>Onchocerca</em> spp. and <em>D. immitis</em>, and might harbor the same potential for onchocerciasis. SOCS box domains were first identified in proteins involved in suppression of cytokine signaling, and are key regulators of both innate and adaptive immunity (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alexander 2002</a>). Proteins in “OG0009843” do not contain any of the other domains usually associated with SOCS, such as SH2 (a combination found in “OG0000874” and “OG0007539”) or Ankyrin repeat-containing domains (a combination found in “OG0001559” and “OG0015826”). However, they may still play an immunomodulatory role during infection, as has been suggested for SOCS box proteins in <em>L. sigmodontis</em> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Godel <em>et al.</em> 2012</a>). An additional SOCS box cluster is synapomorphic for <em>Onchocerca</em> species.</p><p id="p-39">Definition of taxon sets based on host species (“human” <em>vs.</em> “other” <em>vs.</em> “outgroup”) recovered 628 clusters specific to filarial nematodes, but none had proteins of more than four out of seven taxa. Of the eight clusters containing proteins of four taxa, six contained only <em>B. malayi</em>, <em>L. loa</em>, and <em>W. bancrofti</em>. Hence, we found no evidence of systematic convergent adaptation to human hosts in the analyzed proteomes of filarial nematodes. KinFin permits rapid assessment of differences in copy number between species and taxon sets using protein count representation tests. Analysis of clusters shared between taxa either side of the basal split in Onchocercinae (node n11: <em>Dirofilaria</em> plus <em>Onchocerca</em>, and node n15: other filaria) (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>) identified 10 clusters with extreme differences (see Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300233/-/DC1/FileS1.pdf">File S1</a>). Among these was cluster “OG0000051,” which includes prolyl 4-hydroxylase orthologs, including Bm-<a href="http://www.wormbase.org/db/get?name=WBGene00224104;class=Gene">PHY-1</a> and Bm-<a href="http://www.wormbase.org/db/get?name=WBGene00231085;class=Gene">PHY-2</a>, which are essential for development and cuticle formation, and have been suggested as potential targets for parasite control (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Winter <em>et al.</em> 2013</a>). While all n15 taxa have exactly two paralogs (WBANC2 had only Wb-PHY-1, but Wb-PHY-2 was located through a TBLASTN search and was present in WBANC1), counts in n11-taxa ranged from five (OFLEX) to 14 (OVOLV). We identified three additional, singleton prolyl 4-hydroxylase clusters, all from n15 taxa. The number of paralogous prolyl 4-hydroxylases in <em>D. immitis</em> and <em>Onchocerca</em> species could have negative implications in control measures against this locus.</p><p id="p-40">To demonstrate the utility of KinFin for directed analyses of genes and gene families, we revisited the biology of heme synthesis and transport in the Onchocercidae. This pathway is a target of active investigation for drug development. While most organisms can synthesize heme, a complete heme biosynthetic pathway is lacking in all nematodes studied to date (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Rao <em>et al.</em> 2005</a>), and proteins of only two of 12 catabolic steps (<a href="http://www.wormbase.org/db/get?name=WBGene00012895;class=Gene">COX-10</a> and <a href="http://www.wormbase.org/db/get?name=WBGene00011526;class=Gene">COX-15</a>) have been described in <em>C. elegans</em>. In <em>C. elegans</em>, multiple heme responsive genes (HRGs) have been characterized (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Rajagopal <em>et al.</em> 2008</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> 2011</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Sinclair and Hamza 2015</a>) and orthologs have been identified in <em>B. malayi</em> (Bm-<a href="http://www.wormbase.org/db/get?name=WBGene00225443;class=Gene">HRG-1</a> and Bm-<a href="http://www.wormbase.org/db/get?name=WBGene00222644;class=Gene">HRG-2</a>) and <em>D. immitis</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Luck <em>et al.</em> 2016</a>). In <em>C. elegans</em>, HRG are involved in heme trafficking within the epidermis (<a href="http://www.wormbase.org/db/get?name=WBGene00010473;class=Gene">HRG-2</a>), to oocytes (<a href="http://www.wormbase.org/db/get?name=WBGene00010256;class=Gene">HRG-3</a>) and within the intestine (<a href="http://www.wormbase.org/db/get?name=WBGene00019830;class=Gene">HRG-1</a>/<a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00009493#0-9g-3">4</a>/<a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00009494#0-9g-3">5</a>/<a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00009495#0-9g-3">6</a>). Other ABC transporters in <em>C. elegans</em> have been implicated in heme homeostasis (<a href="http://www.wormbase.org/db/get?name=WBGene00003411;class=Gene">MRP-5</a>, <a href="http://www.wormbase.org/db/get?name=WBGene00009042;class=Gene">F22B5.4</a>, and <a href="http://www.wormbase.org/db/get?name=WBGene00022281;class=Gene">ABTM-1</a>) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Antonicka <em>et al.</em> 2003</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Severance <em>et al.</em> 2010</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">González-Cabo <em>et al.</em> 2011</a>). An ortholog of <a href="http://www.wormbase.org/db/get?name=WBGene00003411;class=Gene">MRP-5</a> has been described in <em>B. malayi</em> (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Luck <em>et al.</em> 2016</a>). Several animal parasitic nematodes (including <em>B. malayi</em>, <em>D. immitis</em>, and <em>Onchocerca volvulus</em>) have been shown to harbor a functional ferrochelatase (FeCH) acquired through horizontal gene transfer from an alphaproteobacterium (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Elsworth <em>et al.</em> 2011</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Nagayasu <em>et al.</em> 2013</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wu <em>et al.</em> 2013</a>). Other nematodes have distinct ferrochelatase-like (FeCL) homologs that lack the active site. We cataloged homologs of these proteins in our clustering analysis (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3349/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heme biosynthesis and transport genes. The columns showing counts of proteins per taxon are ordered based on phylogeny (cartoon above). Counts are colored orange if they result from fragmented predictions, and green if the gene was identified only by directed TBLASTN search." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1327459055" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Heme biosynthesis and transport genes. The columns showing counts of proteins per taxon are ordered based on phylogeny (cartoon above). Counts are colored orange if they result from fragmented predictions, and green if the gene was identified only by directed TBLASTN search.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3349/F6.medium.gif" width="440" height="123"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/10/3349/F6.medium.gif" width="440" height="123"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3349/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3349/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486900" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-41" class="first-child">Heme biosynthesis and transport genes. The columns showing counts of proteins per taxon are ordered based on phylogeny (cartoon above). Counts are colored orange if they result from fragmented predictions, and green if the gene was identified only by directed TBLASTN search.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">FeCL proteins were identified in all species. <em>Brugia pahangi</em> has two FeCL proteins (BPAG_0000227101 and BPAG_0000340601-mRNA-1) while all other taxa have one, but both are located at scaffold borders and may be the result of an assembly artifact. The horizontally acquired FeCH is absent from the <em>Caenorhabditis</em> proteomes (and genomes) but present in all the other taxa analyzed. Paralogues in one of the <em>O. ochengi</em> proteomes (nOo.2.0.1.t09359-RA and nOo.2.0.1.t09412-RA) are suggestive of misprediction. <a href="http://www.wormbase.org/db/get?name=WBGene00012895;class=Gene">COX-10</a> and <a href="http://www.wormbase.org/db/get?name=WBGene00011526;class=Gene">COX-15</a> are present in most taxa in the analysis; <a href="http://www.wormbase.org/db/get?name=WBGene00011526;class=Gene">COX-15</a> paralogs in <em>B. pahangi</em> (BPAG_0000577701-mRNA-1 and BPAG_0000682101-mRNA-1), and <em>O. flexuosa</em> (OFLC_0000640801-mRNA-1 and OFLC_0000765101-mRNA-1) are a result of fragmented assemblies. <a href="http://www.wormbase.org/db/get?name=WBGene00012895;class=Gene">COX-10</a> is present in <em>W. bancrofti</em> WBANC1 (on scaffold WBA_contig0009713), but the gene was not predicted. <a href="http://www.wormbase.org/db/get?name=WBGene00012895;class=Gene">COX-10</a> was not found in <em>E. elaphi</em>, which suggests that either the corresponding genomic region was not assembled, or that the gene has been lost.</p><p id="p-43">Presence/absence of proteins involved in heme homeostasis show a more complex pattern. Ce-<a href="http://www.wormbase.org/db/get?name=WBGene00001815;class=Gene">HMT-1</a>, an ATP-dependent phytochelatin transporter, is restricted to <em>Caenorhabditis</em> spp. and <em>D. medinensis</em>. The other ABC-transporter-like proteins (<a href="http://www.wormbase.org/db/get?name=WBGene00022281;class=Gene">ABTM-1</a>, <a href="http://www.wormbase.org/db/get?name=WBGene00003411;class=Gene">MRP-5</a>, and <a href="http://www.wormbase.org/db/get?name=WBGene00009042;class=Gene">F22B5.4</a>) are present across all taxa. For <a href="http://www.wormbase.org/db/get?name=WBGene00009042;class=Gene">F22B5.4</a>, genuine paralogs are found in both <em>Caenorhabditis</em> species and <em>O. volvulus</em>. Ce-<a href="http://www.wormbase.org/db/get?name=WBGene00003411;class=Gene">MRP-5</a> and Bm-<a href="http://www.wormbase.org/db/get?name=WBGene00223634;class=Gene">MRP-5</a> are located within the same cluster, and apparent paralogs in <em>O. flexuosa</em> (OFLC_0000058301-mRNA-1 and OFLC_0001303001-mRNA-1) and <em>W. bancrofti</em> WBANC1 (WBA_0001142501-mRNA-1 and WBA_0000255201-mRNA-1) derive from predictions located at the ends of scaffolds. While no orthologs of Ce-<a href="http://www.wormbase.org/db/get?name=WBGene00010473;class=Gene">HRG-2</a>/<a href="http://www.wormbase.org/db/get?name=WBGene00010256;class=Gene">3</a>/<a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00009493#0-9g-3">4</a>/<a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00009494#0-9g-3">5</a>/<a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00009495#0-9g-3">6</a> were identified, the cluster containing Ce-<a href="http://www.wormbase.org/db/get?name=WBGene00019830;class=Gene">HRG-1</a> included representatives from most species. Missing orthologs of <a href="http://www.wormbase.org/db/get?name=WBGene00019830;class=Gene">HRG-1</a> were identified using TBLASTN searches in <em>S. labiatopapillosa</em> (nSl.1.1.scaf00038), <em>O. ochengi</em> OOCHE2 (nOo.2.0.Scaf03259), <em>W. bancrofti</em> WBANC1 (WBA_contig0001821), and <em>A. viteae</em> (nAv.1.0.scaf00129). The two <a href="http://www.wormbase.org/db/get?name=WBGene00019830;class=Gene">HRG-1</a> paralogs in <em>D. immitis</em> were identified previously (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Luck <em>et al.</em> 2014</a>, <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">2016</a>). Interestingly, Bm-<a href="http://www.wormbase.org/db/get?name=WBGene00222644;class=Gene">HRG-2</a> is not orthologous to Ce-<a href="http://www.wormbase.org/db/get?name=WBGene00010473;class=Gene">HRG-2</a> but rather to Ce-<a href="http://www.wormbase.org/db/get?name=WBGene00016116;class=Gene">C25H3.7</a>, an ortholog of human FAXC (failed axon connection).</p><p id="p-44">In summary, all non-<em>Caenorhabditis</em> nematodes analyzed have a functional FeCH orthologous to the one acquired through HGT in <em>B. malayi</em>. Proteins responsible for the only two steps in heme biosynthesis described in <em>C. elegans</em> are also found in all taxa, apart from <a href="http://www.wormbase.org/db/get?name=WBGene00012895;class=Gene">COX-10</a> in <em>E. elaphi</em>. The heme transporters <a href="http://www.wormbase.org/db/get?name=WBGene00010473;class=Gene">HRG-2</a>/<a href="http://www.wormbase.org/db/get?name=WBGene00010256;class=Gene">3</a>/<a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00009493#0-9g-3">4</a>/<a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00009494#0-9g-3">5</a>/<a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00009495#0-9g-3">6</a> are (in this analysis) restricted to <em>Caenorhabditis</em>, but all the spirurid nematodes analyzed have retained <a href="http://www.wormbase.org/db/get?name=WBGene00019830;class=Gene">HRG-1</a>, a FAXC-like cluster orthologous to Bm-<a href="http://www.wormbase.org/db/get?name=WBGene00222644;class=Gene">HRG-2</a>, and <a href="http://www.wormbase.org/db/get?name=WBGene00003411;class=Gene">MRP-5</a>, and these may mediate heme transport from the intestine.</p><div id="sec-17" class="subsection"><h3>Conclusion</h3><p id="p-45">As ever more genomes are sequenced, and our understanding of the diversity of protein space increases, it concomitantly becomes more difficult to see the patterns in complex orthology data. To ease this bottleneck, we have presented KinFin—a tool that takes the output of standard orthology inference toolkits, and provides a user-friendly but rich analytical toolkit to review and interrogate orthology clustering. By permitting user definition of custom taxon sets, KinFin can be used to highlight changes in presence or membership of ortholog groups associated with either taxonomy or phenotypes of interest. Its reliance on standard input file formats and explicit parameters makes integration in comparative genomics projects easy, and thus promotes transparent and reproducible analysis of clustered protein data. KinFin is under regular maintenance and we welcome user feedback. Further development will involve a HTML user interface to access and interact with the generated output files.</p><p id="p-46">We presented some of the main capabilities of KinFin through the analysis of proteomes of filarial nematodes and outgroup species. By extracting single-copy orthologs, we resolved the phylogenetic relationships between filarial nematodes. We explored synapomorphic clusters and their functional annotations across the phylogeny and identified putative gene families of interest. Through definition of (phylogenetic) taxon sets, we explored and visualized proteomic diversity across key clades, and analyzed differences in protein counts between the sets. To illustrate targeted analysis of homologs of genes of interest, we analyzed clusters containing proteins involved in heme metabolism and homeostasis using characterized orthologs from model organisms.</p><p id="p-47">One of the advantages of KinFin is the speed of computation. Analyses reported here took 40 sec (basic KinFin analysis) and 4 min (advanced KinFin analysis) on a MacBookPro (15-inch 2016; 2.9 GHz I7, 16 GB RAM). While addition of taxa to the clustering and taxon sets to the config file will increase computation time, we think run time is within acceptable bounds for this type of analysis. KinFin’s speed promotes hypothesis exploration, such as comparing alternative phylogenetic topologies—an example of this has been published recently (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Yoshida <em>et al.</em> 2017</a>)—or contrasting taxon sets.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">We thank members of the Blaxter “Nematode and neglected genomics” laboratory in Edinburgh for support, criticism, and suggestions. We thank colleagues who supplied materials for nematode genome sequencing, and the Berriman Lab of the Wellcome Trust Sanger Institute and the staff of WormBase ParaSite for access to genomes. We thank Duncan Berger for testing the software. D.R.L. is supported by a James Hutton Institute/Edinburgh University School of Biological Sciences fellowship.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-49">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300233/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300233/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-50"><em>Communicating editor: K. Gunsalus</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-07-03"><span class="received-label">Received </span>July 3, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-08-11"><span class="accepted-label">Accepted </span>August 11, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Laetsch and Blaxter</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.1" data-doi="10.1038/nri818"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Suppressors of cytokine signalling (socs) in the immune system.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Immunol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">410</span>–<span class="cit-lpage">416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BImmunology%26rft.stitle%253DNat%2BRev%2BImmunol%26rft.aulast%253DAlexander%26rft.auinit1%253DW.%2BS.%26rft.volume%253D2%26rft.issue%253D6%26rft.spage%253D410%26rft.epage%253D416%26rft.atitle%253DSuppressors%2Bof%2Bcytokine%2Bsignalling%2B%2528SOCS%2529%2Bin%2Bthe%2Bimmune%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnri818%26rft_id%253Dinfo%253Apmid%252F12093007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nri818&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12093007&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180438600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.2" data-doi="10.1086/345489"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Antonicka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mattman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glerum</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffbuhr</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Mutations in cox15 produce a defect in the mitochondrial heme biosynthetic pathway, causing early-onset fatal hypertrophic cardiomyopathy.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DAntonicka%26rft.auinit1%253DH.%26rft.volume%253D72%26rft.issue%253D1%26rft.spage%253D101%26rft.epage%253D114%26rft.atitle%253DMutations%2Bin%2BCOX15%2Bproduce%2Ba%2Bdefect%2Bin%2Bthe%2Bmitochondrial%2Bheme%2Bbiosynthetic%2Bpathway%252C%2Bcausing%2Bearly-onset%2Bfatal%2Bhypertrophic%2Bcardiomyopathy.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F345489%26rft_id%253Dinfo%253Apmid%252F12474143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/345489&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12474143&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180186800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.10.3349.3"><div class="cit-metadata unstructured">Bastian, M., S. Heymann, and M. Jacomy, 2009 Gephi: an open source software for exploring and manipulating networks, pp. 361–362 in <em>International AAAI Conference on Weblogs and Social Media</em>. AAAI, Palo Alto, CA.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.4" data-doi="10.1186/1471-2105-10-421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camacho</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulouris</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avagyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papadopoulos</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Blast+: architecture and applications.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCamacho%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.spage%253D421%26rft.epage%253D421%26rft.atitle%253DBLAST%252B%253A%2Barchitecture%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-421%26rft_id%253Dinfo%253Apmid%252F20003500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20003500&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.5" data-doi="10.1093/bioinformatics/btp348"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Capella-Gutiérrez</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silla-Martínez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabaldón</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Trimal: a tool for automated alignment trimming in large-scale phylogenetic analyses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1972</span>–<span class="cit-lpage">1973</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp348%26rft_id%253Dinfo%253Apmid%252F19505945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp348&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505945&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268107100022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.6" data-doi="10.1038/srep24373"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaudhari</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dutta</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">BPGA—an ultra-fast pan-genome analysis pipeline.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">24373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D24373%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep24373%26rft_id%253Dinfo%253Apmid%252F27071527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep24373&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27071527&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.7" data-doi="10.1016/j.cell.2011.04.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samuel</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinclair</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dailey</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamza</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">An intercellular heme-trafficking protein delivers maternal heme to the embryo during development in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">720</span>–<span class="cit-lpage">731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DChen%26rft.auinit1%253DC.%26rft.volume%253D145%26rft.issue%253D5%26rft.spage%253D720%26rft.epage%253D731%26rft.atitle%253DAn%2Bintercellular%2Bheme-trafficking%2Bprotein%2Bdelivers%2Bmaternal%2Bheme%2Bto%2Bthe%2Bembryo%2Bduring%2Bdevelopment%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.04.025%26rft_id%253Dinfo%253Apmid%252F21620137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.04.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21620137&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291018600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.8" data-doi="10.1016/j.ijpara.2011.03.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elsworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wasmuth</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blaxter</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Nembase4: the nematode transcriptome resource.</span> <abbr class="cit-jnl-abbrev">Int. J. Parasitol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">881</span>–<span class="cit-lpage">894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bfor%2Bparasitology%26rft.stitle%253DInt%2BJ%2BParasitol%26rft.aulast%253DElsworth%26rft.auinit1%253DB.%26rft.volume%253D41%26rft.issue%253D8%26rft.spage%253D881%26rft.epage%253D894%26rft.atitle%253DNEMBASE4%253A%2Bthe%2Bnematode%2Btranscriptome%2Bresource.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ijpara.2011.03.009%26rft_id%253Dinfo%253Apmid%252F21550347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ijpara.2011.03.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21550347&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.9" data-doi="10.1186/s13059-015-0721-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emms</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Orthofinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D157%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0721-2%26rft_id%253Dinfo%253Apmid%252F26243257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0721-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26243257&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.10" data-doi="10.1093/nar/gkg495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Enright</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ouzounis</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Protein families and tribes in genome sequence space.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">4632</span>–<span class="cit-lpage">4638</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEnright%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D31%26rft.issue%253D15%26rft.spage%253D4632%26rft.epage%253D4638%26rft.atitle%253DProtein%2Bfamilies%2Band%2BTRIBES%2Bin%2Bgenome%2Bsequence%2Bspace%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg495%26rft_id%253Dinfo%253Apmid%252F12888524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12888524&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184532900048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.11" data-doi="10.1093/sysbio/26.1.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Farris</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Phylogenetic analysis under Dollo’s law.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252F26.1.77%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/26.1.77&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.12" data-doi="10.1096/fj.12-205096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Godel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koutsovoulos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ludin</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nilsson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome of the heartworm, <em>Dirofilaria immitis</em>, reveals drug and vaccine targets.</span> <abbr class="cit-jnl-abbrev">FASEB J.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4650</span>–<span class="cit-lpage">4661</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFASEB%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1096%252Ffj.12-205096%26rft_id%253Dinfo%253Apmid%252F22889830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmFzZWJqIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMS80NjUwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zMzQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.13" data-doi="10.1074/jbc.M110.211201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Cabo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolinches-Amorós</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabello</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ros</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Disruption of the ATP-binding cassette B7 (ABTM-1/ABCB7) induces oxidative stress and premature cell death in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">21304</span>–<span class="cit-lpage">21314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M110.211201%26rft_id%253Dinfo%253Apmid%252F21464130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODYvMjQvMjEzMDQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEwLzMzNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.14" data-doi="10.1128/IAI.68.7.4174-4179.2000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atmadja</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maizels</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The abundant larval transcript-1 and -2 genes of <em>Brugia malayi</em> encode stage-specific candidate vaccine antigens for filariasis.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">4174</span>–<span class="cit-lpage">4179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%2Band%2BImmunity%26rft.stitle%253DInfect.%2BImmun.%26rft.aulast%253DGregory%26rft.auinit1%253DW.%2BF.%26rft.volume%253D68%26rft.issue%253D7%26rft.spage%253D4174%26rft.epage%253D4179%26rft.atitle%253DThe%2BAbundant%2BLarval%2BTranscript-1%2Band%2B-2%2BGenes%2Bof%2BBrugia%2Bmalayi%2BEncode%2BStage-Specific%2BCandidate%2BVaccine%2BAntigens%2Bfor%2BFilariasis%26rft_id%253Dinfo%253Adoi%252F10.1128%252FIAI.68.7.4174-4179.2000%26rft_id%253Dinfo%253Apmid%252F10858234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjk6IjY4LzcvNDE3NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTAvMzM0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.15" data-doi="10.1093/database/bav096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herrero</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muffato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beal</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitzgerald</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Ensembl comparative genomics resources.</span> <abbr class="cit-jnl-abbrev">Database (Oxford)</abbr> <span class="cit-vol">2016</span>: <span class="cit-fpage">bav096</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDatabase%26rft.stitle%253DDatabase%26rft.volume%253D2016%26rft.issue%253D0%26rft.spage%253Dbav096%26rft.epage%253Dbav096%26rft.atitle%253DEnsembl%2Bcomparative%2Bgenomics%2Bresources%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdatabase%252Fbav096%26rft_id%253Dinfo%253Apmid%252F26896847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/database/bav096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26896847&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.16" data-doi="10.1093/nar/gkv1217"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolt</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cain</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Wormbase 2016: expanding to enable helminth genomic research.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D774</span>–<span class="cit-lpage">D780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1217%26rft_id%253Dinfo%253Apmid%252F26578572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1217&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26578572&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolt</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shafie</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kersey</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berriman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Wormbase parasite—a comprehensive resource for helminth genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Biochem. Parasitol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">2</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiochem.%2BParasitol.%26rft.volume%253D215%26rft.spage%253D2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">K.-c.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">EUPAN enables pan-genome studies of a large number of eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">2408</span>–<span class="cit-lpage">2409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D2408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.19" data-doi="10.1093/nar/gkt1177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Cepas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Capella-Gutiérrez</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pryszcz</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcet-Houben</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabaldón</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Phylomedb v4: zooming into the plurality of evolutionary histories of a genome.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D897</span>–<span class="cit-lpage">D902</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1177%26rft_id%253Dinfo%253Apmid%252F24275491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1177&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24275491&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800131&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.20" data-doi="10.1093/bioinformatics/btu031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binns</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">H.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Interproscan 5: genome-scale protein function classification.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1236</span>–<span class="cit-lpage">1240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu031%26rft_id%253Dinfo%253Apmid%252F24451626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451626&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.21" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Standley</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mafft multiple sequence alignment software version 7: improvements in performance and usability.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">772</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23329690&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317002300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kocot</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Citarella</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moroz</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halanych</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Phylotreepruner: a phylogenetic tree-based approach for selection of orthologous sequences for phylogenomics.</span> <abbr class="cit-jnl-abbrev">Evol. Bioinform. Online</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BBioinform.%2BOnline%26rft.volume%253D9%26rft.spage%253D429%26rft_id%253Dinfo%253Apmid%252F24250218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24250218&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.23" data-doi="10.1016/j.ympev.2010.04.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kück</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meusemann</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fasconcat: convenient handling of data matrices.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">1115</span>–<span class="cit-lpage">1118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D56%26rft.spage%253D1115%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2010.04.024%26rft_id%253Dinfo%253Apmid%252F20416383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2010.04.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20416383&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lefoulon</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bain</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makepeace</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">d’Haese</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Breakdown of coevolution between symbiotic bacteria <em>Wolbachia</em> and their filarial hosts.</span> <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1840</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D4%26rft.spage%253De1840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.25" data-doi="10.1101/gr.1224503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoeckert</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roos</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Orthomcl: identification of ortholog groups for eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2178</span>–<span class="cit-lpage">2189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLi%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D2178%26rft.epage%253D2189%26rft.atitle%253DOrthoMCL%253A%2BIdentification%2Bof%2BOrtholog%2BGroups%2Bfor%2BEukaryotic%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1224503%26rft_id%253Dinfo%253Apmid%252F12952885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEzLzkvMjE3OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTAvMzM0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.26" data-doi="10.1186/1471-2164-15-1041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luck</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riggs</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moorhead</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Concurrent transcriptional profiling of <em>Dirofilaria immitis</em> and its <em>Wolbachia</em> endosymbiont throughout the nematode life cycle reveals coordinated gene expression.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1041%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-1041%26rft_id%253Dinfo%253Apmid%252F25433394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-1041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25433394&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.27" data-doi="10.1096/fj.201600603R"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luck</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voronin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slatko</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamza</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Heme acquisition in the parasitic filarial nematode <em>Brugia malayi</em>.</span> <abbr class="cit-jnl-abbrev">FASEB J.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">3501</span>–<span class="cit-lpage">3514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFASEB%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1096%252Ffj.201600603R%26rft_id%253Dinfo%253Apmid%252F27363426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmFzZWJqIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8xMC8zNTAxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zMzQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.28" data-doi="10.1017/S0031182007002880"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nadler</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carreno</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mejía-Madrid</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ullberg</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pagan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Molecular phylogeny of clade III nematodes reveals multiple origins of tissue parasitism.</span> <abbr class="cit-jnl-abbrev">Parasitology</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">1421</span>–<span class="cit-lpage">1442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DParasitology%26rft.volume%253D134%26rft.spage%253D1421%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0031182007002880%26rft_id%253Dinfo%253Apmid%252F17506928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0031182007002880&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17506928&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagayasu</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishikawa</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taketani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification of a bacteria-like ferrochelatase in <em>Strongyloides venezuelensis</em>, an animal parasitic nematode.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e58458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De58458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">J.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sultana</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Monophyly of clade iii nematodes is not supported by phylogenetic analysis of complete mitochondrial genome sequences.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DPark%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D12%26rft.spage%253D392%26rft.epage%253D392%26rft.atitle%253DMonophyly%2Bof%2Bclade%2BIII%2Bnematodes%2Bis%2Bnot%2Bsupported%2Bby%2Bphylogenetic%2Banalysis%2Bof%2Bcomplete%2Bmitochondrial%2Bgenome%2Bsequences.%26rft_id%253Dinfo%253Apmid%252F21813000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21813000&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.31" data-doi="10.1038/nmeth.1701"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Heijne</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Signalp 4.0: discriminating signal peptides from transmembrane regions.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">785</span>–<span class="cit-lpage">786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DPetersen%26rft.auinit1%253DT.%2BN.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D785%26rft.epage%253D786%26rft.atitle%253DSignalP%2B4.0%253A%2Bdiscriminating%2Bsignal%2Bpeptides%2Bfrom%2Btransmembrane%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1701%26rft_id%253Dinfo%253Apmid%252F21959131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1701&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21959131&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295358000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.32" data-doi="10.1038/nature06934"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rajagopal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">A. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amigo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Upadhyay</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Haem homeostasis is regulated by the conserved and concerted functions of hrg-1 proteins.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">453</span>: <span class="cit-fpage">1127</span>–<span class="cit-lpage">1131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRajagopal%26rft.auinit1%253DA.%26rft.volume%253D453%26rft.issue%253D7198%26rft.spage%253D1127%26rft.epage%253D1131%26rft.atitle%253DHaem%2Bhomeostasis%2Bis%2Bregulated%2Bby%2Bthe%2Bconserved%2Band%2Bconcerted%2Bfunctions%2Bof%2BHRG-1%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06934%26rft_id%253Dinfo%253Apmid%252F18418376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06934&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18418376&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256839900062&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.33" data-doi="10.1073/pnas.0500877102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">A. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carta</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lesuisse</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamza</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Lack of heme synthesis in a free-living eukaryote.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">4270</span>–<span class="cit-lpage">4275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0500877102%26rft_id%253Dinfo%253Apmid%252F15767563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzEyLzQyNzAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEwLzMzNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.34" data-doi="10.1371/journal.pgen.1001044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Severance</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajagopal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">A. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cerqueira</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitreva</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide analysis reveals novel genes essential for heme homeostasis in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSeverance%26rft.auinit1%253DS.%26rft.volume%253D6%26rft.issue%253D7%26rft.spage%253De1001044%26rft.epage%253De1001044%26rft.atitle%253DGenome-wide%2Banalysis%2Breveals%2Bnovel%2Bgenes%2Bessential%2Bfor%2Bheme%2Bhomeostasis%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001044%26rft_id%253Dinfo%253Apmid%252F20686661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20686661&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.35" data-doi="10.1007/s10534-015-9841-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sinclair</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamza</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Lessons from bloodless worms: heme homeostasis in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Biometals</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">489</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometals%26rft.volume%253D28%26rft.spage%253D481%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10534-015-9841-0%26rft_id%253Dinfo%253Apmid%252F25724951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10534-015-9841-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25724951&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.36" data-doi="10.1371/journal.pone.0120671"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dickins</span>  <span class="cit-name-given-names">B. J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demeter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallagher</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Agape (automated genome analysis pipeline) for pan-genome analysis of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0120671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0120671%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0120671%26rft_id%253Dinfo%253Apmid%252F25781462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0120671&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25781462&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.37" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Raxml version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451623&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.38" data-doi="10.1002/prot.10347"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Unger</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uliel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Havlin</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Scaling law in sizes of protein sequence families: from super-families to orphan genes.</span> <abbr class="cit-jnl-abbrev">Proteins</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%26rft.stitle%253DProteins%26rft.aulast%253DUnger%26rft.auinit1%253DR.%26rft.volume%253D51%26rft.issue%253D4%26rft.spage%253D569%26rft.epage%253D576%26rft.atitle%253DScaling%2Blaw%2Bin%2Bsizes%2Bof%2Bprotein%2Bsequence%2Bfamilies%253A%2Bfrom%2Bsuper-families%2Bto%2Borphan%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fprot.10347%26rft_id%253Dinfo%253Apmid%252F12784216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/prot.10347&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12784216&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183386600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vinuesa</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Contreras-Moreira</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Robust identification of orthologues and paralogues for microbial pan-genomics using get_homologues: a case study of pinca/c plasmids.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">1231</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%26rft.volume%253D1231%26rft.spage%253D203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.40" data-doi="10.1074/jbc.M112.397604"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winter</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCormack</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myllyharju</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Prolyl 4-hydroxlase activity is essential for development and cuticle formation in the human infective parasitic nematode <em>Brugia malayi</em>.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">288</span>: <span class="cit-fpage">1750</span>–<span class="cit-lpage">1761</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M112.397604%26rft_id%253Dinfo%253Apmid%252F23223450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODgvMy8xNzUwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zMzQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.41" data-doi="10.1073/pnas.1304049110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novelli</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dailey</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landmann</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Interdomain lateral gene transfer of an essential ferrochelatase gene in human parasitic nematodes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">7748</span>–<span class="cit-lpage">7753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1304049110%26rft_id%253Dinfo%253Apmid%252F23610429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzE5Lzc3NDgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEwLzMzNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.42" data-doi="10.1016/j.gpb.2015.01.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A brief review of software tools for pangenomics.</span> <abbr class="cit-jnl-abbrev">Genomics Proteomics Bioinformatics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%2BProteomics%2BBioinformatics%26rft.volume%253D13%26rft.spage%253D73%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gpb.2015.01.007%26rft_id%253Dinfo%253Apmid%252F25721608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gpb.2015.01.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25721608&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koutsovoulos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laetsch</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Comparative genomics of the tardigrades <em>Hypsibius dujardini</em> and <em>Ramazzottius varieornatus</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D15%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3349.44" data-doi="10.1093/nar/gkw1119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tegenfeldt</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuznetsov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Orthodb v9.1: cataloging evolutionary and functional annotations for animal, fungal, plant, archaeal, bacterial and viral orthologs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">D744</span>–<span class="cit-lpage">D749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1119%26rft_id%253Dinfo%253Apmid%252F27899580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27899580&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/10/3349.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/10/3337" title="Taxonomically Restricted Genes with Essential Functions Frequently Play Roles in Chromosome Segregation in Caenorhabditis elegans and Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/10/3359" title="CONE: Community Oriented Network Estimation Is a Versatile Framework for Inferring Population Structure in Large-Scale Sequencing Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/10"><span>Volume 7 Issue 10, October 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/10.cover-source.jpg?itok=lurMO2QE" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3349" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PupTEfcipTwBiJngpPp3lnp7CoQA0jlAPAwWKhXH5UM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/10/3349" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/10/3349" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/10/3349" class="active" data-icon-position="" data-hide-link-title="0">KinFin: Software for Taxon-Aware Analysis of Clustered Protein Sequences</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/486779" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ACrdDU2bGkxhYRpwjCFjOVgWssACoT3X8XSvHU9Fjcs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/10/3349.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/10/3349" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-9lMN9ENMgVxqwI_VCCBlyimp2C0Akh63xhhZ4DktCcM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/10/3349" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/10/3349.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486779" id="node486779--2" data-pisa="ggg;7/10/3349" data-pisa-master="ggg;g3.117.300233" data-apath="/ggg/7/10/3349.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">KinFin: Software for Taxon-Aware Analysis of Clustered Protein Sequences</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7887-0186" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dominik R. Laetsch</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2861-949X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark L. Blaxter</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3349-3357;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300233 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486779--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dominik R. Laetsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolutionary Biology, University of Edinburgh, EH9 3JT, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>†</span>The James Hutton Institute, DD2 5DA Dundee, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominik%2BR.%2BLaetsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laetsch%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADominik%2BR.%2BLaetsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7887-0186" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dominik R. Laetsch</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dominik.laetsch@gmail.com" class="" data-icon-position="" data-hide-link-title="0">dominik.laetsch@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mark L. Blaxter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolutionary Biology, University of Edinburgh, EH9 3JT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BL.%2BBlaxter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blaxter%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BL.%2BBlaxter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2861-949X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark L. Blaxter</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/10/3349" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/10/3349&amp;title=KinFin%3A%20Software%20for%20Taxon-Aware%20Analysis%20of%20Clustered%20Protein%20Sequences" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/10/3349&amp;title=KinFin%3A%20Software%20for%20Taxon-Aware%20Analysis%20of%20Clustered%20Protein%20Sequences" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/10/3349&amp;title=KinFin%3A%20Software%20for%20Taxon-Aware%20Analysis%20of%20Clustered%20Protein%20Sequences" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/10/3349&amp;text=KinFin%3A%20Software%20for%20Taxon-Aware%20Analysis%20of%20Clustered%20Protein%20Sequences" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/10/3349&amp;title=KinFin%3A%20Software%20for%20Taxon-Aware%20Analysis%20of%20Clustered%20Protein%20Sequences" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/10/3349&amp;t=KinFin%3A%20Software%20for%20Taxon-Aware%20Analysis%20of%20Clustered%20Protein%20Sequences" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/10/3349&amp;title=KinFin%3A%20Software%20for%20Taxon-Aware%20Analysis%20of%20Clustered%20Protein%20Sequences" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/10/3349&amp;title=KinFin%3A%20Software%20for%20Taxon-Aware%20Analysis%20of%20Clustered%20Protein%20Sequences" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/486779/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486779" id="node486779--3" data-pisa="ggg;7/10/3349" data-pisa-master="ggg;g3.117.300233" data-apath="/ggg/7/10/3349.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">KinFin: Software for Taxon-Aware Analysis of Clustered Protein Sequences</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7887-0186" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dominik R. Laetsch</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2861-949X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark L. Blaxter</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3349-3357;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300233 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486779--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dominik R. Laetsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolutionary Biology, University of Edinburgh, EH9 3JT, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>†</span>The James Hutton Institute, DD2 5DA Dundee, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominik%2BR.%2BLaetsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laetsch%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADominik%2BR.%2BLaetsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7887-0186" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dominik R. Laetsch</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dominik.laetsch@gmail.com" class="" data-icon-position="" data-hide-link-title="0">dominik.laetsch@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mark L. Blaxter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolutionary Biology, University of Edinburgh, EH9 3JT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BL.%2BBlaxter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blaxter%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BL.%2BBlaxter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2861-949X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark L. Blaxter</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3349&amp;rft.epage=3357&amp;rft.atitle=KinFin%3A%20Software%20for%20Taxon-Aware%20Analysis%20of%20Clustered%20Protein%20Sequences&amp;rft.volume=7&amp;rft.issue=10&amp;rft.date=2017-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Laetsch%2C+Dominik+R.&amp;rft.au=Blaxter%2C+Mark+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/486779/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/486779/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/486779/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/486779/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/486779/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/486779/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/486779/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/486779/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/486779/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/486779/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/486779/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/486779/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/10/3349" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/10/3349.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/10/3349.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/10/3349.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/10/3349.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
