<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/10/3359" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/10/3359.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/10/3359.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/10/3359.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/10/3359/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/10/3359.atom" />
<meta name="HW.pisa" content="ggg;7/10/3359" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="CONE: Community Oriented Network Estimation Is a Versatile Framework for Inferring Population Structure in Large-Scale Sequencing Data" />
<meta name="DC.Identifier" content="10.1534/g3.117.300131" />
<meta name="DC.Date" content="2017-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Kuismin et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Estimation of genetic population structure based on molecular markers is a common task in population genetics and ecology. We apply a generalized linear model with LASSO regularization to infer relationships between individuals and populations from molecular marker data. Specifically, we apply a neighborhood selection algorithm to infer population genetic structure and gene flow between populations. The resulting relationships are used to construct an individual-level population graph. Different network substructures known as communities are then dissociated from each other using a community detection algorithm. Inference of population structure using networks combines the good properties of: (i) network theory (broad collection of tools, including aesthetically pleasing visualization), (ii) principal component analysis (dimension reduction together with simple visual inspection), and (iii) model-based methods ( e.g. , ancestry coefficient estimates). We have named our process CONE (for community oriented network estimation). CONE has fewer restrictions than conventional assignment methods in that properties such as the number of subpopulations need not be fixed before the analysis and the sample may include close relatives or involve uneven sampling. Applying CONE on simulated data sets resulted in more accurate estimates of the true number of subpopulations than model-based methods, and provided comparable ancestry coefficient estimates. Inference of empirical data sets of teosinte single nucleotide polymorphism, bacterial disease outbreak, and the human genome diversity panel illustrate that population structures estimated with CONE are consistent with the earlier findings" />
<meta name="DC.Contributor" content="Markku O. Kuismin" />
<meta name="DC.Contributor" content="Jon Ahlinder" />
<meta name="DC.Contributor" content="Mikko J. Sillanpӓӓ" />
<meta name="article:published_time" content="2017-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="CONE: Community Oriented Network Estimation Is a Versatile Framework for Inferring Population Structure in Large-Scale Sequencing Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Estimation of genetic population structure based on molecular markers is a common task in population genetics and ecology. We apply a generalized linear model with LASSO regularization to infer relationships between individuals and populations from molecular marker data. Specifically, we apply a neighborhood selection algorithm to infer population genetic structure and gene flow between populations. The resulting relationships are used to construct an individual-level population graph. Different network substructures known as &lt;i&gt;communities&lt;/i&gt; are then dissociated from each other using a community detection algorithm. Inference of population structure using networks combines the good properties of: (i) network theory (broad collection of tools, including aesthetically pleasing visualization), (ii) principal component analysis (dimension reduction together with simple visual inspection), and (iii) model-based methods (&lt;i&gt;e.g.&lt;/i&gt;, ancestry coefficient estimates). We have named our process CONE (for community oriented network estimation). CONE has fewer restrictions than conventional assignment methods in that properties such as the number of subpopulations need not be fixed before the analysis and the sample may include close relatives or involve uneven sampling. Applying CONE on simulated data sets resulted in more accurate estimates of the true number of subpopulations than model-based methods, and provided comparable ancestry coefficient estimates. Inference of empirical data sets of teosinte single nucleotide polymorphism, bacterial disease outbreak, and the human genome diversity panel illustrate that population structures estimated with CONE are consistent with the earlier findings&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/10/01" />
<meta name="citation_mjid" content="ggg;7/10/3359" />
<meta name="citation_id" content="7/10/3359" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/10/3359" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/10/3359.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/10/3359.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/10/3359.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300131" />
<meta name="citation_pmid" content="28830924" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="19" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3359" />
<meta name="citation_lastpage" content="3377" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Markku O. Kuismin" />
<meta name="citation_author_institution" content="Department of Mathematical Sciences, University of Oulu, FI-90014, Finland" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9074-7420" />
<meta name="citation_author" content="Jon Ahlinder" />
<meta name="citation_author_institution" content="Swedish Defense Research Agency, SE-906 21 Umeå, Sweden" />
<meta name="citation_author" content="Mikko J. Sillanpӓӓ" />
<meta name="citation_author_institution" content="Department of Mathematical Sciences, University of Oulu, FI-90014, Finland" />
<meta name="citation_author_institution" content="Biocenter Oulu, FI-90014, Finland" />
<meta name="citation_author_email" content="mikko.sillanpaa@oulu.fi" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2808-2768" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. R. Abecasis;citation_author=A. Auton;citation_author=L. D. Brooks;citation_author=M. A. DePristo;citation_author=R. M. Durbin;citation_title=An integrated map of genetic variation from 1,092 human genomes.;citation_pages=56-65;citation_volume=491;citation_year=2012;citation_issue=7422;citation_pmid=23128226;citation_doi=10.1038/nature11632" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. H. J. Alexander;citation_author=J. November;citation_author=K. Lange;citation_title=Fast model-based estimation of ancestry in unrelated individuals.;citation_volume=19;citation_year=2009;citation_pmid=19648217;citation_doi=10.1101/gr.094052.109" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=F. Balloux;citation_title=EASYPOP (Version 1.7): A Computer Program for Population Genetics Simulations;citation_pages=301-302;citation_volume=92;citation_year=2001;citation_issue=3;citation_pmid=11447253;citation_doi=10.1093/jhered/92.3.301" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Notes;citation_author=C. Chen;citation_author=E. Durand;citation_author=F. Forbes;citation_author=O. François;citation_title=Bayesian clustering algorithms ascertaining spatial population structure: a new computer program and a comparison study.;citation_pages=747-756;citation_volume=7;citation_year=2007;citation_doi=10.1111/j.1471-8286.2007.01769.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Corander;citation_author=P. Waldmann;citation_author=M. J. Sillanpää;citation_title=Bayesian Analysis of Genetic Differentiation Between Populations;citation_pages=367-374;citation_volume=163;citation_year=2003;citation_issue=1;citation_pmid=12586722" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=S. A. Cushman;citation_author=E. L. Landguth;citation_title=Spurious correlations and inference in landscape genetics.;citation_pages=3592-3602;citation_volume=19;citation_year=2010;citation_issue=17;citation_pmid=20618896;citation_doi=10.1111/j.1365-294X.2010.04656.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. E. Darling;citation_author=B. Mau;citation_author=N. T. Perna;citation_title=progressiveMauve: multiple genome alignment with gene gain, loss and rearrangement.;citation_pages=e11147-e11147;citation_volume=5;citation_year=2010;citation_issue=6;citation_pmid=20593022;citation_doi=10.1371/journal.pone.0011147" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=R. J. Dyer;citation_title=Population graphs and landscape genetics.;citation_pages=327-342;citation_volume=46;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=R. J. Dyer;citation_author=J. D. Nason;citation_title=Population Graphs: the graph theoretic shape of genetic structure.;citation_pages=1713-1727;citation_volume=13;citation_year=2004;citation_issue=7;citation_pmid=15189198;citation_doi=10.1111/j.1365-294X.2004.02177.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=R. J. Dyer;citation_author=J. D. Nason;citation_author=R. C. Garrick;citation_title=Landscape modelling of gene flow: improved power using conditional genetic distance derived from the topology of population networks.;citation_pages=3746-3759;citation_volume=19;citation_year=2010;citation_issue=17;citation_pmid=20723052;citation_doi=10.1111/j.1365-294X.2010.04748.x" />
<meta name="citation_reference" content="citation_title=Introduction to Graphical Modelling;citation_author=D. Edwards;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=S. Epskamp;citation_author=A. O. J. Cramer;citation_author=L. J. Waldorp;citation_author=V. D. Schmittmann;citation_author=D. Borsboom;citation_title=qgraph: network visualizations of relationships in psychometric data.;citation_pages=1-18;citation_volume=48;citation_year=2012;citation_doi=10.18637/jss.v048.i04" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=G. Evanno;citation_author=S. Regnaut;citation_author=J. Goudet;citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study.;citation_pages=2611-2620;citation_volume=14;citation_year=2005;citation_issue=8;citation_pmid=15969739;citation_doi=10.1111/j.1365-294X.2005.02553.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Falush;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=Inference of Population Structure Using Multilocus Genotype Data: Linked Loci and Correlated Allele Frequencies;citation_pages=1567-1587;citation_volume=164;citation_year=2003;citation_issue=4;citation_pmid=12930761" />
<meta name="citation_reference" content="citation_journal_title=Phys. Rep.;citation_author=S. Fortunato;citation_title=Community detection in graphs.;citation_pages=75-174;citation_volume=486;citation_year=2010;citation_doi=10.1016/j.physrep.2009.11.002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Frichot;citation_author=F. Mathieu;citation_author=T. Trouillon;citation_author=G. Bouchard;citation_author=O. François;citation_title=Fast and efficient estimation of individual ancestry coefficients.;citation_volume=196;citation_year=2014;citation_pmid=24496008;citation_doi=10.1534/genetics.113.160572" />
<meta name="citation_reference" content="citation_journal_title=Biostatistics;citation_author=J. Friedman;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_title=Sparse inverse covariance estimation with the graphical lasso.;citation_volume=9;citation_year=2008;citation_pmid=18079126;citation_doi=10.1093/biostatistics/kxm045" />
<meta name="citation_reference" content="citation_journal_abbrev=J Stat Softw;citation_author=J. Friedman;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_title=Regularization Paths for Generalized Linear Models via Coordinate Descent.;citation_pages=1-22;citation_volume=33;citation_year=2010;citation_issue=1;citation_pmid=20808728;citation_doi=10.1111/j.1467-9868.2005.00503.x" />
<meta name="citation_reference" content="citation_journal_title=Softw. Pract. Exper.;citation_author=T. M. Fruchterman;citation_author=E. M. Reingold;citation_title=Graph drawing with force-directed placement.;citation_pages=1129-1164;citation_volume=21;citation_year=1991;citation_doi=10.1002/spe.4380211102" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Girvan;citation_author=M. E. J. Newman;citation_title=Community structure in social and biological networks;citation_pages=7821-7826;citation_volume=99;citation_year=2002;citation_issue=12;citation_pmid=12060727;citation_doi=10.1073/pnas.122653799" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. Greenbaum;citation_author=A. R. Templeton;citation_author=S. Bar-David;citation_title=Inference and analysis of population structure using genetic data and network theory.;citation_volume=202;citation_year=2016;citation_pmid=26888080;citation_doi=10.1534/genetics.115.182626" />
<meta name="citation_reference" content="citation_journal_title=Wiley Interdiscip. Rev. Comput. Stat.;citation_author=S. Harenberg;citation_author=G. Bello;citation_author=L. Gjeltema;citation_author=S. Ranshous;citation_author=J. Harlalka;citation_title=Community detection in large-scale networks: a survey and empirical evaluation.;citation_pages=426-439;citation_volume=6;citation_year=2014" />
<meta name="citation_reference" content="citation_title=The Elements of Statistical Learning: Data Mining, Inference, and Prediction;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_author=J. Friedman;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=J. Mach. Learn. Res.;citation_author=D. Heckerman;citation_author=D. M. Chickering;citation_author=C. Meek;citation_author=R. Rounthwaite;citation_author=C. Kadie;citation_title=Dependency networks for inference, collaboration filtering, and data visualization.;citation_pages=49-75;citation_volume=1;citation_year=2000;citation_doi=10.1162/153244301753344614" />
<meta name="citation_reference" content="citation_title=Advances in Neural Information Processing Systems;citation_title=BIG &amp; QUIC: sparse inverse covariance estimation for a million variables;citation_author=C.-J. Hsieh;citation_author=M. A. Sustik;citation_author=I. S. Dhillon;citation_author=P. K. Ravikumar;citation_author=R. Poldrack;citation_pages=3165-3173;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Jakobsson;citation_author=N. A. Rosenberg;citation_title=CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure.;citation_volume=23;citation_year=2007;citation_pmid=17485429;citation_doi=10.1093/bioinformatics/btm233" />
<meta name="citation_reference" content="citation_author=A. Jalali;citation_author=P. Ravikumar;citation_author=V. Vasuki;citation_author=S. Sanghavi;citation_pages=378-387;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=T. Jombart;citation_author=S. Devillard;citation_author=F. Balloux;citation_title=Discriminant analysis of principal components: a new method for the analysis of genetically structured populations.;citation_pages=94-94;citation_volume=11;citation_year=2010;citation_pmid=20950446;citation_doi=10.1186/1471-2156-11-94" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=T. Jombart;citation_author=R. M. Eggo;citation_author=P. J. Dodd;citation_author=F. Balloux;citation_title=Reconstructing disease outbreaks from genetic data: a graph approach.;citation_pages=383-390;citation_volume=106;citation_year=2011;citation_issue=2;citation_pmid=20551981;citation_doi=10.1038/hdy.2010.78" />
<meta name="citation_reference" content="citation_journal_title=Physical review. E, Statistical, nonlinear, and soft matter physics;citation_journal_abbrev=Phys Rev E Stat Nonlin Soft Matter Phys;citation_author=A. Lancichinetti;citation_author=S. Fortunato;citation_title=Community detection algorithms: a comparative analysis.;citation_pages=056117-056117;citation_volume=80;citation_year=2009;citation_issue=5 Pt 2;citation_pmid=20365053;citation_doi=10.1103/PhysRevE.80.056117" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Z. Li;citation_author=D. M. Absher;citation_author=H. Tang;citation_author=A. M. Southwick;citation_author=A. M. Casto;citation_title=Worldwide Human Relationships Inferred from Genome-Wide Patterns of Variation;citation_pages=1100-1104;citation_volume=319;citation_year=2008;citation_issue=5866;citation_pmid=18292342;citation_doi=10.1126/science.1153717" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=Y. Li;citation_author=S. A. Jackson;citation_title=Gene network reconstruction by integration of prior biological knowledge.;citation_pages=1075-1079;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_title=Advances in Neural Information Processing Systems;citation_title=Stability approach to regularization selection (StARS) for high dimensional graphical models;citation_author=H. Liu;citation_author=K. Roeder;citation_author=L. Wasserman;citation_pages=1432-1440;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=N. Meinshausen;citation_author=P. Bühlmann;citation_title=High dimensional graphs and variable selection with the LASSO.;citation_pages=1436-1462;citation_volume=34;citation_year=2006;citation_doi=10.1214/009053606000000281" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Series B (Statistical Methodology);citation_author=N. Meinshausen;citation_author=P. Bühlmann;citation_title=Stability selection (with discussion).;citation_pages=417-473;citation_volume=72;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=J. Graph Algorithms Appl.;citation_author=P. Pons;citation_author=M. Latapy;citation_title=Computing communities in large networks using random walks.;citation_pages=191-218;citation_volume=10;citation_year=2006;citation_doi=10.7155/jgaa.00124" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=D. Posada;citation_author=K. A. Crandall;citation_title=The effect of recombination on the accuracy of phylogeny estimation.;citation_pages=396-402;citation_volume=54;citation_year=2002;citation_issue=3;citation_pmid=11847565;citation_doi=10.1007/s00239-001-0034-9" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. L. Price;citation_author=N. J. Patterson;citation_author=R. M. Plenge;citation_author=M. E. Weinblatt;citation_author=N. A. Shadick;citation_title=Principal components analysis corrects for stratification in genome-wide association studies.;citation_pages=904-909;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16862161;citation_doi=10.1038/ng1847" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=S. R. Proulx;citation_author=D. E. L. Promislow;citation_author=P. C. Phillips;citation_title=Network thinking in ecology and evolution.;citation_pages=345-353;citation_volume=20;citation_year=2005;citation_issue=6;citation_pmid=16701391;citation_doi=10.1016/j.tree.2005.04.004" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=S. J. Puechmaille;citation_title=The program STRUCTURE does not reliably recover the correct population structure when sampling is uneven: subsampling and new estimators alleviate the problem.;citation_pages=608-627;citation_volume=16;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=T. Pyhäjärvi;citation_author=M. B. Hufford;citation_author=S. Mezmouk;citation_author=J. Ross-Ibarra;citation_title=Complex patterns of local adaptation in teosinte.;citation_volume=5;citation_year=2013;citation_pmid=23902747;citation_doi=10.1093/gbe/evt109" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Raj;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=fastSTRUCTURE: variational inference of population structure in large SNP data sets.;citation_volume=197;citation_year=2014;citation_pmid=24700103;citation_doi=10.1534/genetics.114.164350" />
<meta name="citation_reference" content="citation_title=Advances in Neural Information Processing Systems;citation_title=Model selection in Gaussian graphical models: high-dimensional consistency of l1-regularized MLE;citation_author=G. Raskutti;citation_author=B. Yu;citation_author=M. J. Wainwright;citation_author=P. K. Ravikumar;citation_pages=1329-1336;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=N. A. Rosenberg;citation_author=J. K. Pritchard;citation_author=J. L. Weber;citation_author=H. M. Cann;citation_author=K. K. Kidd;citation_title=Genetic Structure of Human Populations;citation_pages=2381-2385;citation_volume=298;citation_year=2002;citation_issue=5602;citation_pmid=12493913;citation_doi=10.1126/science.1078311" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. J. Salipante;citation_author=D. J. Roach;citation_author=J. O. Kitzman;citation_author=M. W. Snyder;citation_author=B. Stackhouse;citation_title=Large-scale genomic sequencing of extraintestinal pathogenic Escherichia coli strains.;citation_volume=25;citation_year=2015;citation_pmid=25373147;citation_doi=10.1101/gr.180190.114" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Tamura;citation_author=G. Stecher;citation_author=D. Peterson;citation_author=A. Filipski;citation_author=S. Kumar;citation_title=MEGA6: molecular evolutionary genetics analysis version 6.0.;citation_volume=30;citation_year=2013;citation_pmid=24132122;citation_doi=10.1093/molbev/mst197" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=H. Tang;citation_author=J. Peng;citation_author=P. Wang;citation_author=N. J. Risch;citation_title=Estimation of individual admixture: analytical and study design considerations.;citation_pages=289-301;citation_volume=28;citation_year=2005;citation_issue=4;citation_pmid=15712363;citation_doi=10.1002/gepi.20064" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=R. Tibshirani;citation_title=Regression shrinkage and selection via the LASSO.;citation_pages=267-288;citation_volume=58;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_abbrev=Ann Stat;citation_author=L. Wasserman;citation_author=K. Roeder;citation_title=HIGH DIMENSIONAL VARIABLE SELECTION.;citation_pages=2178-2201;citation_volume=37;citation_year=2009;citation_issue=5A;citation_pmid=19784398;citation_doi=10.1214/08-AOS646" />
<meta name="citation_reference" content="citation_journal_title=Investig. Genet.;citation_author=A. Wollstein;citation_author=O. Lao;citation_title=Detecting individual ancestry in the human genome.;citation_pages=1-12;citation_volume=6;citation_year=2015;citation_pmid=25750707;citation_doi=10.1186/s13323-014-0018-3" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. M. Yu;citation_author=G. Pressoir;citation_author=W. H. Briggs;citation_author=I. V. Bi;citation_author=M. Yamasaki;citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.;citation_pages=203-208;citation_volume=38;citation_year=2006;citation_issue=2;citation_pmid=16380716;citation_doi=10.1038/ng1702" />
<meta name="citation_reference" content="citation_journal_title=Stat. Appl. Genet. Mol. Biol.;citation_author=B. Zhang;citation_author=S. Horvath;citation_title=A general framework for weighted gene coexpression network analysis.;citation_pages=17;citation_volume=4;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=J. Mach. Learn. Res.;citation_author=T. Zhao;citation_author=H. Liu;citation_author=K. Roeder;citation_author=J. Lafferty;citation_author=L. Wasserman;citation_title=The huge package for high-dimensional undirected graph estimation in R.;citation_pages=1059-1062;citation_volume=13;citation_year=2012;citation_pmid=26834510" />
<meta name="citation_reference" content="citation_title=Practical Data Science with R;citation_author=N. Zumel;citation_author=J. Mount;citation_year=2014" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="CONE: Community Oriented Network Estimation Is a Versatile Framework for Inferring Population Structure in Large-Scale Sequencing Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
<meta name="twitter:description" content="Estimation of genetic population structure based on molecular markers is a common task in population genetics and ecology. We apply a generalized linear model with LASSO regularization to infer relationships between individuals and populations from molecular marker data. Specifically, we apply a neighborhood selection algorithm to infer population genetic structure and gene flow between populations. The resulting relationships are used to construct an individual-level population graph. Different network substructures known as communities are then dissociated from each other using a community detection algorithm. Inference of population structure using networks combines the good properties of: (i) network theory (broad collection of tools, including aesthetically pleasing visualization), (ii) principal component analysis (dimension reduction together with simple visual inspection), and (iii) model-based methods ( e.g. , ancestry coefficient estimates). We have named our process CONE (for community oriented network estimation). CONE has fewer restrictions than conventional assignment methods in that properties such as the number of subpopulations need not be fixed before the analysis and the sample may include close relatives or involve uneven sampling. Applying CONE on simulated data sets resulted in more accurate estimates of the true number of subpopulations than model-based methods, and provided comparable ancestry coefficient estimates. Inference of empirical data sets of teosinte single nucleotide polymorphism, bacterial disease outbreak, and the human genome diversity panel illustrate that population structures estimated with CONE are consistent with the earlier findings" />
<meta name="og-title" property="og:title" content="CONE: Community Oriented Network Estimation Is a Versatile Framework for Inferring Population Structure in Large-Scale Sequencing Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/10/3359" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Estimation of genetic population structure based on molecular markers is a common task in population genetics and ecology. We apply a generalized linear model with LASSO regularization to infer relationships between individuals and populations from molecular marker data. Specifically, we apply a neighborhood selection algorithm to infer population genetic structure and gene flow between populations. The resulting relationships are used to construct an individual-level population graph. Different network substructures known as communities are then dissociated from each other using a community detection algorithm. Inference of population structure using networks combines the good properties of: (i) network theory (broad collection of tools, including aesthetically pleasing visualization), (ii) principal component analysis (dimension reduction together with simple visual inspection), and (iii) model-based methods ( e.g. , ancestry coefficient estimates). We have named our process CONE (for community oriented network estimation). CONE has fewer restrictions than conventional assignment methods in that properties such as the number of subpopulations need not be fixed before the analysis and the sample may include close relatives or involve uneven sampling. Applying CONE on simulated data sets resulted in more accurate estimates of the true number of subpopulations than model-based methods, and provided comparable ancestry coefficient estimates. Inference of empirical data sets of teosinte single nucleotide polymorphism, bacterial disease outbreak, and the human genome diversity panel illustrate that population structures estimated with CONE are consistent with the earlier findings" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
    <title>CONE: Community Oriented Network Estimation Is a Versatile Framework for Inferring Population Structure in Large-Scale Sequencing Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"_hHpnrR0ltGDsaswkT4k3q0jrZsVjUzzjcczYEcKZ3U","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"486778","apath":"\/ggg\/7\/10\/3359.atom","pisa":"ggg;7\/10\/3359","ac":{"\/ggg\/7\/10\/3359.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/10\/3359.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/10\/3359"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/10\/3359"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/10\/3359":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ivhfpdl0zyl2yVLc5RlM2NdWNV2OGLlmzXKt51MjNh\\\\\\\/jNEzDWQgCcVRMFFDO151bDBawFxAecUOAd27YyemK45JSwOO7NxdGkmtASMclaKRBFKIjnExVC4tBwlz612peBpkAVGmZEq6AQnjAdr5+P4gyIdTSVawYy+jRcUllj4ftScaaVpfkkDkH+EEkG+jqZnL7EP+RgtoschJNxr4DN3wvE71J1fo2PbQ302a8RarK\\\\\\\/vHaunDKZTaiRPMT9vVinZqbX8z9je+T6XVF0tJZ5fJKELPSna4ZQPw2zWcoFiLGgEMa83iVxwQH8TrhHks+CZzXrD1TC+BCE+LXguoHx4tZgqLjKtLvCp35PZ1EKh5fym109bxNJey0N0vEJGLObnPmOngfGa8tUdD3du\\\\\\\/j\\\\\\\/c3VmQckvwk74RQvNjIDX15tdTGPGx3XTAGlCfYa5+829OIkqfJft+ozqiZ0mIhflOKnmVUPJQsa10gv3TXD95XLCUtpKhPvT67LipOxLHNTZ3YbSG5RffziJM2WI\\\\\\\/+lZcyYBwC9\\\\\\\/7AwOVqY3f2nfFZw5yToKVgNUKpxMf51BkTAT1SQAi9FnidfXa2AJU8plg22b3pLinDdw0EqbfAdTc\\\\\\\/bzLUUkH8v0TOU7fwE\\\\\\\/PReD9KqRnehBAKedFLw5TAoBKY9fXMobJHzSexDKWYBu2x3N+5+dTfdtKu3eRGS+b4DXAOY1VQZqyg7hWKG\\\\\\\/IFsSV\\\\\\\/fpjqNzCXafPSjCZKaRI9BMO1ex0JRYKCXpXjkSqHFwGqlWCMv75fS\\\\\\\/l\\\\\\\/JMuMt7kSjozoq\\\\\\\/RUSgWLuD\\\\\\\/5wFKsUD8v9mB5Mru3rEEsn0IGVS18C0cUAtNT7m7Hek2014G5jJw5VX7zSpyDltjIzYxTLP0EYWRk6ctIlyuVAtTpUnh4Jv2iwBBvIMU\\\\\\\/jZvxlVifUW0JbuYqz4zlwENRjVed+hVysH88lGpLKdz2CNrcTmvk4NQ02m+IyBhlpRsRzha4XCWnzHNmcSQQEnS0vIMOWAv0fNs29KuUPLq88q28nfIxi0+lh5FO4IYOIzy4eBTu81X72DmA20zmy+qwXRhvaG1EfrMSI7QdrYysdWWjdGXbq4VpwbQ7hsqdGRfmcMniVMmWfII5ujmf5IzocY6XH8SmdtZ9PSfZ5yXc4QQVsRKLvrppY2H9EFWIvxhlj6v0ftRinh7IyYqyg4ugZaFdCeZkgsmzo5uqkJPi6zVQA8BKZaZBOWWl71mNk44Qpqi1gr4cwdLXJ6ULV5ePt6RaMIj6pG1ApYuPLMf60XRMv\\\u0022,\\\u0022iv\\\u0022:\\\u0022bt82bCjMPGoh8OjFOQfMsQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022914c7ce3bd9749bffb8085dd5a6ef19c25dae1751fbb40d8a257e165f5548a1d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022S6zemRhgVQtBiUnZ7W++gZhdPd7nFq1aduWEkREeqbln\\\\\\\/B1Y03oiiG1ITYux+Nf4PG6VRDtPxU2640D6o9GbhIPNtPt7iAbo\\\\\\\/CBqDgHzkoisT6b44FKNtQi95mnjmdjJKUI\\\\\\\/SGi3GrOwyovuAWtAyiKx\\\\\\\/ZBNMvRc\\\\\\\/Bh09CmMy5Da4LGigs63ZYev6yiSeaeZfn4aC35v3W9jlUC2utvqCCw69m9kIGkOg+i9ooive993S4iiSRMxEU1KTJDQXKolvWoMg3iRaq4CPKO1RhL3SwecWUShQk9R9RoZSFGm3nJCRolfpWuAvUouuGzAvFaPdq6R3USvGYoeAd\\\\\\\/rrQRnvYh9y\\\\\\\/nKTXmngGHlsacLtToAHMkuTJFIxI1Yb8b2z6aXOg4\\\\\\\/Ag9fcsZBOQC2H6PPFzqauIZ0hIffLvh+Xt463QrFmgijRPB0RKZQHE7bvY14tVUOPmmBsv8ivKyrQc+tdwc8MOdFsYDhhx3BpLATzt6lSIlaUYWoVgh8IEmcA8r6CcKXpX7zHtsFJK\\\\\\\/KjcL8LJBD8Zok2Tzmmy0b5sbbGMA0sNxmcbroGYeTc0hTYgAvC9R9SD4KSpD7puLZRXLl2KQqeJ4HEER\\\\\\\/90jOWTAkfIjnbolhpQSj3VOlPxZUrcO1Y\\\\\\\/L2HjuDR0cJsQjapveF9SOIzSjnKKxB0KzlzZUFcKyPYt1abkY9HTPERysT852GXOhydCsMOHVGKvzpPS\\\\\\\/NsWx93lUvR5\\\\\\\/tGewpg3r4tLka\\\\\\\/xSZsz\\\\\\\/ip3e7WgHXbvhg9ZUS+gQHcLVpkWWfPnreLow3P77J8713ePiq7Dgnh\\\\\\\/oWraCb29t6orT2aWUc1IRFjfWn8h6qx8smzGRxxycplpvsJMjsURakkqlDDrl97Py8s5HF1XY\\\\\\\/Z3+i2Q8RvdJbUAvDv52urICYyHlhvT2Dd0cf01nHjUUjm4H2lYB9hJo5UhEOA6shzP7yED3hKoZJ9uJlfp0hZHL6eQB10QUaH81fX4Bh8nhZR0qRQkkQPFDls5tu9xBeMu8Uac4PuhrC6YS72HW5JKBtmpnuUm4pTUy6j8M8GwtBdkzRRvJiyfPem8tt2LYwVaHSUGqTcXaPw2X8dRQ4qcYy4vKJZVnNPiy6yly5Tvtn960a27jfNG3S\\\\\\\/6ID8Yh\\\\\\\/8du8IUUe\\\\\\\/GgX+9B6qXzjreacEkOpm0DB0AAQg3tgkucBVCrmNOYD7kf1FqckO\\\\\\\/QJ3NhvP0RqSZzOhJY9z9Tup3GKLl7j7Wop5AwRdkuSv+VH+S9PppCVJzH9u1HdywwncYxgrgX34ZABFZNBP\\\\\\\/I5vqsFzekRC+briNZW3EGkAI4ymzhjONsOQtGB6eppL7gVzZmZAPre\\\\\\\/X8LzJUjj7kJHSdwBl4LF\\\\\\\/swy19kmyjRqzklK6vxOfKaGceP20doTSpBpWLIybczmHaYf4l5MrGH3XMfh7Ft4tMpB5Gf4RZd2eaSCADbnb+s3g8BcDyVh9lkUeyB+g0Rdwj4eaE9NJHuHIEHy1xxrTim\\\\\\\/zYxnd7zIhG9+iCxrJPQZC61GUleV8Lui6BHM2Qzi2lTTzav7IMCqjVxpcPVlOfiq3n3A6LSwnSIx23SJww5QoDi4G8e8GuJluxYbhkG9O2Je5ICAOxptkAmfEWFYEv3BqdbkLYOL2QjQcmDDrhpPSDo43uczl2XpCdtK6eRhZjLWpoiiCopydKudQcvuyjy8Q4KibfuuCGIBw9IFIk7zGLW6S3QXcJhtG5aQ29M+iWNB1\\\\\\\/YFOCgkGzu6MHjYHPITPHLP\\\\\\\/KZZURPpx76iUdLcMR8RZz43yJ4VSJcUX9XAeKxG5dcmC0LZKsbdvoCZBz\\\\\\\/OrahvaTdhRWXp6\\\\\\\/EYwNHfEtplh+ZGrJZbuJRsxPk9Mn+yAu+pel5fRMMYVqGQIbsVCSXtdnAzTNJpDFGHHVvbQK3youEFRWnLas4rkHFhM0ZHRl6FqXqcLIUXIZR2aZc8kKQuVyfHBiUrEMdBcWhOx8LofYFgetHMJNjSViDkyu7AF1uM09kZt24frp1gmVLeLXQurUIyAzGymqYUPy6R+ftQX3\\\\\\\/Of0IvEGdJDGrV5SRp\\\\\\\/8EmORNVlztlr+IbTlIjXscsGucJLTg0mB9mrmxEbt\\\\\\\/fAxzghOtO4zwrAn5kNnHME6ZL72QLQKxianTb0eRhLLnlR35wzUrUDxmFTw6DZLcXGclR4A3aLJAaj8Rb+2pPk9RtDMW8C2FS4MSZh5dTx+pzR+tmAQr6+IkJVyJsQi4VUjewiHVXpOUmFFy17ZPS8iPZtY6uA+h8VJFvDHjIoa3NB\\\\\\\/DVycY\\\\\\\/SETYvkAVyuR41Wf1H\\\\\\\/uRCtcKk3dsyMR9xKRaUHW+VnIjpCTWpU0DKBgBbRVoK+tPkXVHF9mlopm7KjJJiRy\\\\\\\/QYxQkCVjEu9uSoys\\\\\\\/TRv6ZnlkoAq0K3fvhIHMKbXa+Obd149yE+MrDrlcepRvBirDso9KuDNu8ciinKSXqI+RiZrUgVaedoteTc+qLgZflQ\\\\\\\/PfUqXKBYwQHEh5wA0bYGdL6MaP8uJS0gP\\\\\\\/v5AZzLWNVGAKaq2rsWECaMVZ+QgZToCZ76McLGL\\\\\\\/YmFnz3KAKYtQfkcogWJKtWu8srf7BltCcLZlRMVvC1dY8CptO31hQyVQkhqvgbQeIgUW6YHHfyD0OdWhpahVEloyZLvlrsmjhN8ZeK1SIEd3kc4pTvc0t8lEtoUeTsZNgTIcXshtrRAwC4nmOKALF83Q2F6iD+3zLrkG2iRa+xJw5L4X6zmNEu6nEpgAwLY21gcFhozJs6qRR6Oyg7ARBLWgEhq\\\\\\\/D0yIXsBfP0vqFCm8tiNTaDvHO8dXbyULuRP6eQO1gtRiGE8P+V3PcyiMh\\\\\\\/kPXFClUDHUmTW2JEP9R7YF7HxF0zx78GnzoSYNOd0cGBPoKLk7N9mYMO9Vlrk1I6sPza+GXELSEyu3neiHgrZzpHTIY6gf4iKdYeL7QlcqgLx932Lw1vbW34U+B+FUdI4jirsPqvMw2wg4WzoJMwgC1g+vu9knyE30m1Sz5g7jldrwFtR1+J0kXEldlx\\\\\\\/H65My6DCTllXnVPQmLw7kXDK0yHihjJ6GLtz4092+8g8rX775osY7TtSh3lIzJBTXBqLDBgUcOwZQ6id7WdCs0wCBasorEjHnzcDLaPz5CtR1vVR+hwXK7yiSpmIAG3G3DNUtcswRFOW8lSFd0HvQE2ZKx\\\\\\\/TZuL\\\\\\\/tgQsr8nH3oC6BF\\\\\\\/RuQOaGxKvnJiib8HqCptKMkNGf\\\\\\\/nLSEo6e6LTLubeMSgd2Rb70toHMPIKZ4EduI46W5m\\\\\\\/uxMwcieMyk5y47P61rfBqEFCIR4Cmul+1e1ZfHUEdn7epPaz0+OSD8yRRst+4XuYCMsbVbFNA7yKpu9LaYfoSnpCfHvMBOjv3onSrBH0HuzaD2Sc90jJUSSlVP6xw06aMISeJmwLAEw7ZEvevY\\\\\\\/aMx60+SXBmX+5WalVLP9RMm6KQ4l7kvqmlDqb1V76eiE24oiiPDjl\\\\\\\/Xup66I16u73znHWS15jFXnXyHfQriR+7O6eLg8SK6d4JCSO\\\\\\\/SFrTSNwf5GaqVK1dp1achUq9Im3QnvrsvFIajBZMLN9pGPsnmlGubFzVwEeC3LUfDPtcSnzvGJe2jBPk\\\\\\\/djElpkdc1SMSQ584OtWYvHzvZ0KjMs4\\\\\\\/JL0BWyPAsCBVU8Dr9DanEtErd1VbWSFLqLVFtROCjmFfr4+S4QKcOSpAZVmBQmMdYC3RH5n7lhPLVCyMXdXgHMwAzfW9KYn1ftUyFu1\\\\\\\/6CoT53zN56St6vNe4f5fRPkVVuoltZoUpzHJD64gt7cqfA8AzlcPFgakRq90+GPqO+8Anls5QLJkNvsOGAL4FhqpfWhj0UgbAM1pStEjzQG8THl4+qi7QbGERAEbOazTKVXQE9qy3mgbMIGojQSF8VxGI6BDeywUMCEcRAVEQO0UY89AMNyBfoQy\\\\\\\/W0cOURAZpiDd2nluee0diqxyDyVxeKnzNClZqO6Z+iZzFr7TXlswm3oG1yBVkKoIB+OqJejtxdeM6BudVBcq\\\\\\\/SFVnkvAfyVHEldgZQqgR7BVckK2fLh9kCQool+Bxo7PVx9Iity++99YFG+EHX6A9OTopOPrf37BRXRimikwkokEGi45AILQA11m1iPR0nI\\\\\\\/OhGfamz\\\\\\\/N1ZasEwk6YE\\\\\\\/z9IziT\\\\\\\/KqBK4mqWAW3bBA9S\\\\\\\/1mTl4r324Hne+LYoAiVvnaW0CfNLbXnk7Me3b2lWQjehVWNFcje8DwnstONfIakpLFTOfysCcS\\\\\\\/QTbs0cDFF5ZFDSMiLpBjcQH14UHgd1ZTBEHS6zsUIL+gaymvHb4xfZtvjhKjK6bD+eWAyeew\\\\\\\/v4rKPVqu07\\\\\\\/\\\\\\\/Q4CsGy831HseclHx7yMUTBhBNI06dcMiu8GtdM5l3Imyu2pVsI2AMfo61MDkoDAaRHcsf35FkjUbmvT33UJNQHy3G+AU8ojwtIYI3CjeASNppihBGSoc9tyCoZMfNVtpW5kuWQIxcEb029bPzyBY\\\\\\\/ZyaYzQJbObLNUYHicbzErOZoIHlYZoPQQnI4Q6qTyrjegzdKCREr2Oj\\\\\\\/Zi8OioTTsefIyN5wHcEPoFaxhz95M1YaXem2s8V9rkLOT0PhKuSTOqAdraw8EOV31bUrHL\\\\\\\/M\\\\\\\/SoR+LhPRnS29Jqssbr4kmu62HVIY3V4fyBV6Gi5FNwxE0ozbw9qLSMaSedc5iOQqe8GT5boekbp0xms1G6INmhkQv44muwClfUEqLoFRXSPDRCSE7aCuQf0UhmhB\\\\\\\/mXFJjCmVSHcuWY46qL4EIy\\\\\\\/J9sydxB6U8aQPIiPPtgVlXHV0Hwci2gzwB5z\\\\\\\/Xr2f6xVXUKq+RebTBHpEp90s1iyZcADT+OvDtrkjZwPFQT5v2iJ8oHAX5aE5jYm6slkEwzUz0WvIXPJ6nQdL38Lv7m8dNKVfaUShKwkSFaRbXxcPv+jEy4dtjgbzNw8RKZ50+8+tE91XTOd77\\\\\\\/BZwe56Yt79gxiF8kGT2aVrfJpPNckYF4wiozrpiEFYsfOLvMKJFwV7WIX2WFxotFBvFtfVz7q07zZQiLqEGK+uuLqYSKpoSLSj1Nat67f60kZQf0mxg3yU9peDEoHPIa+6BNen1NZEk18rddoQKO+DK5A0cSAc68cwKvccnAW9bULIecl0uYgGNGU3b2IoQhDl++McZ4K8mzuU9TSXLq8TYLOjgEhL30AzxkJcIwzWWABFZnel9QXgRukADAOMmg1mS7sKR0gaYUG6lHOu7x8A\\\\\\\/KB+AJOzqDt1AKT43wB7RmYIQqaozYEPB0RwOpaaPu0Tqh6svWnEwGXWsMpZYEOEbyhMIJLA\\\\\\\/8jMC0fvW2gPUJ7JAbylS\\\\\\\/3jF2QlxQxQFbDmskgkMGQAJPK6LbZfjsQwdWdwc65Hq2WXnNCk5bagccp2JwFjFk8InXcXLa\\\\\\\/RAt9qhFg7F7DcVvOx3BjDaa7wcA4wfQlfUd3puXdWm1MFsaZdyvIqIY1UC6d0vDbalgkcLre5lLJxgYEiK3RIf6Ym80n5I8dWrPjxJCgi2ephisxVvqBbAVJFH83sXryElOGrK1uvPLoP+gDKu0fHMq10OJNicDD8RU\\\\\\\/licpgoyEwJYhwvcTUzFOmvVHJ4n5tCeuMGKFakRW+1UaCfrL3wAz27FmbYhMD91qyqmKUPxGnVBLZcG54c8cowYOcZZmScbA6hdw12PP+DXPVR72Z5pFZyHc4egcCAnHnTc1nT7HYvpNhVphQhnaMmzo9Sl5kCkOqPkUwd6IIwJlDVn00qAfZ2qd+AuAKszcyhxnwFmjR6qQc1Jo\\\\\\\/PmTU4dlZp0lVIPhaTCeDizOAz\\\\\\\/zYNIevnA5GCK8Lm8fzhZXifvw7god0Wo6+vkuBiafxnJet0As6TnJ58zogspposbgHQx6wcRZb+GoMgCWlqpNM\\\\\\\/bMiHmJpWHb9yvtBkKAUIm6J5Fxh12U64p5q+ksm1W9jebF+avD1M\\\\\\\/8tRgYV6f\\\\\\\/CdSnoIedcK+MGZRtToJNYRsQE0f6fRgkQmFtan5VHOKvZt7VotiAMlO03Jr6rkGRPStjIekhqwGukp7SsugEg+\\\\\\\/0d7XB9MEtPBS2m2qjOvbb5DQhhMNyz\\\\\\\/WheYPiKH+M9T0\\\\\\\/nWWaxiGA65b5ZZXr201D4nF1XttSf\\\\\\\/5a0O6p2DXYdzKLfeqHJx8zaqhUA4lW5H8KbN8isrtnM5BvVnI3xg+NQBJMKwdnrST7y6MToTNboDkFaU8GGXwftxQ8HRZ0\\\\\\\/R9SqrJgE\\\\\\\/V5FspbXo3ilfpy3U6uWpBiUDBS1qRo+QP\\\\\\\/a22FET+bM5WpnE09jCgQvTphU9NibYSw9sqQdcJVwKGMkVC7uGuTkU+u6QOeH137KxibuZFP2D7mAr51X1dVg0Pb5BQSZI3t06qOyWqdnVzpcXksUgKPMYY2vkNK6ltcVWAkfEjorfHCcbDGx+w6ZKEXI0Foj8tgBNvpYJc4IbuIEJdVqoCl+oaJsbKRUBtG8KmYMezLJwv0\\\\\\\/BvsT6dmj2dT6W9Lxrk3y1QsVuHYQTxDYrtLIY1bJYw6zS8G6tG8lHcYUO5iWT7eQgQE74mfgEAfji96MOBFpfOgRyhWzHOlQIrg3KKc5EloPJz+afdyDxvLGdzqvuX7H79Y6wM3kGlqdrfp5\\\\\\\/\\\\\\\/8gB65xlQUO9xUQJ2gejWehbjM5jE89+SQIOw+QX3TU+1aNOvpof7HAukh8KX26lCMW8r8MApvX0afbUDWd539757+o+D42G6TbTVYGhGOUF0oyoWLiNUZODu+4WIkIwOQkUyKWD86uZY9zE+HYKzuhuYY3i3qrnR1q8sAShCXO6uSljxkr5tXs4247dqzpr0bFPLdqA1GkIvxeYim0KXrDFraODt\\\\\\\/VxPpoAzsfMk25Ro9oqqkVK+JULbt6tvbt5gdTWW44ym89HMAcU2oZajo3NV1Ow7wuODdKCYLQnZZtPGeHtonaA5kmBRKRfIinFTcvhmfJQMxQWZdYmU4E\\\\\\\/8aN1E3SfmiETy9LEGbaOWEp9ks7vwXvlu6N5oRNv65UnsWBU2KHfHlCoeEfYKsR7qEnmuErKjTvZMPz3MW5gYe3jAysmnQobU6LV0RGuQC1etc7hlvoJj92D9qU06HzODsEnd9uWUaWqhot9Gsf6kL9fOMReb2MJZ3ZldgnP5nY3R2fToMkZPS9G9Df41+L2\\\\\\\/FFuRKjNcQAdKqztLMlWRl9TjAodOhDMRmcS\\\\\\\/SPG4HBfqAwl45k2V+1bP12lablwb+WB\\\\\\\/ByfTilMUBiAfznRXd1oCQCw4cBJ1O6M+wn2PmCRz+rQBt+hTMbqBTyNocPSP7D8txt3\\\\\\\/AHKZO9Gysdx0\\\\\\\/LJm4TAgcXS7IoKYYjaQrnq7pOWqpH5VD0Gth+QadmIuopUVgA94ZZLYm03Lt\\\\\\\/43dtAVE34KIIMaAx8M7DLh3veV5odmiNxOZFyC3RIaMAXvWbfVKbeXNrFOQduxqCdlyZaXoKodpzlUkvUUySeT3QKHWYLtzSw2xd8bvSK4m\\\\\\\/Elff6xo0438k3WIE0QO9NwA811fgDvG4IA6YgGH93VNbq0GGA\\\\\\\/+88m01\\\\\\\/s6\\\\\\\/ZqgXNdydWsUI1eaV2PyTJFhyHDI67f+YoSZ9LHjTi+t7XzqOK1PMxBQb+o3D4PNRo+SZQuHoFPwHislk3VYnXk0gCgJr5w0\\\\\\\/yQIBwa7g+t74\\\\\\\/mFvw3wmuC\\\\\\\/SbD2uKyK93vkt4nD6FC0g8wNheT7e3AJ6EbYBsS\\\\\\\/ENvEGKmbN1LGf9+40RjVi8XNbTprKMDa6OeQrqrIt4CS3qsTr3k+tE5H0FN+1xchbNw7zFqUO0Nzl5ipIoYDViou6SwnmpQP+mxkYmKnXNSC0PI11zRWKjAPB3V22SalIXyGMql9YV7QkFjeS+z\\\\\\\/0wLs3GGK9i2DpqkqR5uaoS53ZZuqXXUg4MLFNf7mkLtIq1fJ9tbs5Sfr6st+oDfRibz+VmAUclVLQxguaJ7XH4Ig4l8yIjj7EUuel8i5rE1BkvOdx4al8dXviK44EiD1R4Ye+uauG0BisYR2WTqBU2MTMN5qKzAi3P8yZ8YPNy6VieUqU+JpF1X6Ky2KWdyYkxGbi\\\\\\\/cgU8eezqAfOJwrXOQzxBD2cZrEAON8Olv28VXg\\\\\\\/YDCVBmhkkOHFczE9yBxcE0Jy+HoyUuCZoJZg2j4JM9wbmViGo8dP\\\\\\\/61DKIBHd7CGxEB2nY3kBCcpCQ+C+C3pXN9acjRvuB78bM0WmJS\\\\\\\/9sEvmPOLMXL7UmTeATi9+2zVQPTzb2JG5NZXHMPqmZ8FAexYZPhEqbafIwnHHZY3zRgciRuQI7b6uQnTy6EQaepF4e11GXC4hN9e+3q8Bb2z6hFiGikpLUUYw7qVGXA3ktZAl4eXxqwEOhkjyigr7VwnJoxD7y+OzhGX1PvcS6SxzQA7EmisuJQ6lYAaBp6EHcO0YYfkiRRrvZgPVhjJJHmjMjqHmpdRrrhHDabZikON3UPjLTpBC\\\\\\\/s1zKHBp8BMF9AnCTgB9BIrIoIVX3ON\\\\\\\/uE359uUO2x7NuKPRxQ1uU94udvbcrSsG0sOFY1xm8Zac4oi1uczuQuLT5HPuF0HVVMeIi4AzyQ9GHPyKuv1TAwPZ3ZWDRG1kEBtSrPOVy83ovvUBbAqjhuk1dm8GNuCEeh1U6koJHtf5bKk3E5ztTCOaeYvhq6P23QccvvxwCXCf\\\\\\\/Vf9sjBT7Df0gcGN0kduq8UtTyDlnFUR4tJZS\\\\\\\/f6RBTCIdCiv9mWrVSgvvjjwfhCqsDpkdRWmTmOUF5vWvv1I\\\\\\\/J9HjpI1Xc6e6CgiyExw4fyvkuqfqIbQFJChg7DVl7HQQDarymm8sY4Z6N2rkowzhd\\\\\\\/JzDEHShrEqJ1sq0DPJBfrMruxKXRmPcCPtdc4EQjlHS0\\\\\\\/njYCFdVaY5Ms+ZZSp6FrK4tqB3CYzOqsOF25a74KSiVY7jiOXLoct1qu6EnFMARlfWmKIqSfAGOxdxxsY\\\\\\\/6grpUA+Sv4s+x4e01D5I5Ou\\\\\\\/IsehwEZYD\\\\\\\/Wa4sjgsGBLuDmR37dpUtpEbYlzzZp7dXSLrKxwhpsnaOALdNC5fccuspWjWc+vncF66\\\\\\\/zS5nz8ljDjVg6u\\\\\\\/djsxDuWDeARgytW0zsjuO3JzNcBlifu1VGmRR8vPpiub3lO7s2HLav2sZgaU6Aar5lcOTpkQeA8Qo\\\\\\\/GXcuCDc+h9s3FgsTcinZyMpaGrZYkUHpFXga13mTpY\\\\\\\/A4GcdJagFawZvTkViHTH0Cd\\\\\\\/rR0xfXj\\\\\\\/f5hYADmgJKy9WBJnb7CigoQ1KTj5j+fArr20a5\\\\\\\/lQESmIq+QHZP0byOC7XJihZWSTOD5i6NpXqK8WGkhshHbHGC8OKnKMBUBD3Gpl7CS8mM3t7LzXtO6pUjfiVBz7YTb+wgO4YnJy4eW\\\\\\\/t2HukRuWfsK\\\\\\\/RHt9q\\\\\\\/nG3NmqBHmw+pQWAkEsSo6cdflWIRVrqs6lHeRwT2JFxbKuJsg+GFam47+CyBOYf59oOpIx57wkO4eb\\\\\\\/KQpwq7qMoKR3dBtNVq+qLjXP9qagq+nqrMTVpNcm\\\\\\\/1CgGjD1g14Cw5+UhTZjvJkTDS6GF1fyfBplU7V7BdSY0u7scoBVDJazNRmbEvNlpD4lGTBvThCUOIOcu4vPHE6zFmQtRkaPaCgKLmJHEcvl4u4J0wiiKpC1L6Y9WRb8Cvfr6sJHVpVbg1ZYJg3mrQD59+JGVeNRqKJFBwvbRMmdMlFZL2pO+Zb9DP6hjHyt0P7xrNRCtCDQ9W7CB2Ur945ZzGY25adhO5WuukAKfsWPA+vKMa9bJfb7RMfQkAHoee+4SltqLyzPqAenzF3hrjmmZYgtewiGi9rJzMgbP2qnrT+v7GqcZ4eGY9m\\\\\\\/7CIet3RCKQh8C8qvXTeEqVnclwBbISYQN\\\\\\\/2VGhx0Zbi1l744A66HqHyzqsuZ9iesMVZ6yGqAf4k3LROZZQAbnLCG7QyZthJobpGLdoGB+\\\\\\\/WHnYf9fv9N8tDNbyN8Mb02kVtR0irnkG+MZM71WqpoHcHXJoVW7QfJ\\\\\\\/OJJ3qiLNx7PAhfPw8zGniYDO0QDPhuJmowXLltR9W3EYIrYyy4sWJux8XFFKXfuslIB7Zj8uMFXfMiaHj\\\\\\\/vfk+mqreaEQfT48WEKE96cM34Z5yDViJTxEZ7nbHOZ5Vb4fbP6Ji7bXF3buaR4bYd\\\\\\\/oqoDQKWqtrfqBZVLIYDfjJQjvz3oAeUZjtf7+vzYWDBrCtqDn8fjYTlruJjFvW3lKbZplAcid8Tz6o5ssh0w+eg7y\\\\\\\/oz6CD5takyPYMdhO+0TwypEekXFzmN57TRV9sQm45uvOUcYZy95+4zajM64VfN1jxkCc60XRMspFIIg\\\\\\\/lIXv8q42BARWw4RL\\\\\\\/3xZ9ZiVcpZWgHF0esrpILCC4jdqVmKCoY5BiTUlxzc4e39XKm\\\\\\\/OKOKVEpOq4nX3PA1oJ7MOMdKTkocKJqUhIchwEVwuFm1FlQUpWkS8e9ZYP3weurLMFkEHAscyFH6w\\\\\\\/gFZyw\\\\\\\/RX9XFLhBJ7A87x5OOAy6ZDMx5UpHOytYX3R3CliF\\\\\\\/xy5B4FZMicc2DGdn\\\\\\\/UePzNOvmvFS3SNzUNlMZkJBR4woLHI8jVaoZzB0kFNK23lkVChCfacrgIfvGgy1aiKW+jlhpONYAY5A+v15DSotfHnCFwDZiNhB2U+q7PY+jHq5+E50Js5aQnWL95kHv8C4Cija0i\\\\\\\/C\\\\\\\/pj7Fm87NzYyuvvqm0vwh7MF+AvpRlmMYiHjzSFAdEJmu8t\\\\\\\/NqD78y\\\\\\\/NrucVbbYMJybckTQu7CG7Xkrp9\\\\\\\/RSzKc0Lxx3BGv68e\\\\\\\/XHAIRctlBcwbfgH3KRqTiBC32XusfFfSTh1Xmf6NLP2frIf+vz7d79B3kAKRyleAvpkRAsizKR1a3J1itmps2vZyu8wggOSnf505LnDGZSP0\\\\\\\/SSvmJBeKwycwaTGgQlR4P0gw4PpeAlghjpgPZ8+nwrQYPDE04HKygafIwR1Lxss+GHbDVGFNch4GKWQur9NhIDQj4eXo\\\\\\\/U8S+Bhd14XZmQESJMfZEBLqWXKrjgJT59GIXClOQ+jERr1NsPbD23RQnjV8UQd\\\\\\\/+WQdQdZB2JcsJ+4nwNtHganmcdb3G\\\\\\\/\\\\\\\/q6vl5SQJ4mCvZyk8XznwMN6qRsjvkVfQ5oQfi8\\\\\\\/pjeMpBZhrQ8a614KJJFPAlHHAlnVFx5kuA8G\\\\\\\/ehRDklkxGQiWVfdGoTzmBMtxqkMW4rIr35wZQzZJvjnnhTWsTe8MejtgS3VsyfGHlR8bLUxTxRbnMwINV+94x\\\\\\\/IqDtfNL2CXrlVet5vOc6xSAYA3WQdC4TejC8Er9+g\\\\\\\/vnnF+cXxS\\\\\\\/COaOybd7fq0eh1ZIxX7kNZGLPQa13BhjiLuQI5sg4\\\\\\\/yeY5fO\\\\\\\/g+OxM9a5e2GoAraHqE+oBDniOvpFB2frTMNzNDDCheUp9+2KdxKKSKsaL+M9FN+ju\\\\\\\/ctRqDMEUqZmiMeM+XSAFPOAgOxiiQRXNhzqjRU04wBNXrxvKwEZc3xGDIlcJNND\\\\\\\/fJFxqjGppPJGKc+PoPmbv2dQVDikNZRwBO1imjvAWm2wDMaHwoNcvMhitLBvedTJ9ZPAkqhg4LJpk5opHp12LC+G5RhBjD6w\\\\\\\/sR9\\\\\\\/DCW1hh85FA8zjVCrSfN19iE2Vs0WlvkIMBlSNCId4soB95rwXGARfyR2U3uCDatfE3Cfjqx2Wjl06CtggJ9+X6zqWrNxY+vGW2CcHqkwF8UTmQT7EN1R2+PEo72bCQ4JZnkqyR6A4lw\\\\\\\/4Mbsi4NfyqjhfQf10wr0fUHjLCABIkjCadpfeDlF0GRbrBtTDkAA+lcUJUWBdaJRSuXN56yY9tS8eEzJoyKOXNoqvMsfgB1c5Tq5biwVxzqh2PnjRRwiCm+KrKwcVQXGggxFFGsvtcRuPH5nEnQeONL2GqZHs1KL6bIyblaPb2hHwl01DLPjAtClzl\\\\\\\/wvzjpZqqikpllO6Ww7IcrJb8acPFSM9EkL6HFMrkw\\\\\\\/lnThlSnxiu5qtB\\\\\\\/52seY57KY2c2LBMhA1mixyKExIrf8BocQNem9ZCLSAM3fVHyonbPQ2K6+LtM0Uu+JFwkfUJqNP\\\\\\\/3SEssrVYuD48w0r1xMcdou75HdvvmHdUAWKF+UfM1FCrPWlq34o2IoJI2cpyBkqZyY4tQIddUYpVbaIZ\\\\\\\/LPOJ+SmFFY45\\\\\\\/o5DJ83odiSoWtTVULBaQvMQyMLVwCKwMxDxTloIWBKsXEhnuEohRS243OIo8ERu8RqwA3FUKR8+N6R7PDjb47yD8dL1z6\\\\\\\/3CCXY4OSFIMK9+aZ35b9IC0kxE8xZ\\\\\\\/tC+yKXSCnUXFCjqbPEWGDWXYQy+jhaYvec8MPCasc\\\\\\\/gODyZ6ysdCqMWu2QBlmAb1ofGfMCjJuT6jwKxQuXvAKbYBfyc41NN+3aDqdMaUO8tdSG\\\\\\\/evjdZHeGGiB6UbIsPJEahAoPAfPkqPgOiNHoY5PB182ucZqq7f5tgMfMkUhf5BUVaVmdQ6clP20cd6qfb+0LhMDTwNF4z0XTAd+JAmcWtfbVDJBjip7Y5uaAlhzlFw0fkavcdPgq+dQATtvX+Tnu+nvldFhqW\\\\\\\/oe6NZ0WkLxRztnCzT9o7TfMsLKEmmaCXdBrYb14kkEMsdILFBr4M6BWTCZyq9aE\\\\\\\/AsGsPv07PqOij2jguS7rZUFKmOK+qp+o3uwMGhFo1HAgt2gszCC\\\\\\\/HFrqRuqGw5pMTIeyKsBKOwNG0SaPTcq+sypaTuR1WWQVbtWLqBfM2DLJimJt+lvZIlSCBFlQQJ6xLjKwEkfWWS04fzZil+pKiGfCstemkcFEYD9Pj4HIvy1TrkVYS55b4m3UPM7TcB1SqhPhzwc7gl9ZYnPUfFdIM+i27aFJiBlH9d\\\\\\\/iPqi6T8jWc2gX9Sptjh6hebm1he0TPKhDnEK4PfnWI+DXyRerKHdGSU3jant7qCMN1Ww4pHh3zRUrh\\\\\\\/o0JLrRwOMLo35nrxNfPbdS4t+n5oLFMKen+SDGeduBZXpkrdfXkhN0vTF0lUZGoQHpV2jdjVpVbnd0XNbXRINaIvApyefoT0klK1y5hz5wEPoRtceTaNAgRTHSBXCf51ywIJvnwWfqrqTKwdai2cz1HvpyhyTMJEPUvVYKQ5cJDa9rIIkXPKxUxDM4Vqd3eYAVOJxn4qRfEUdLPXy0Fklx1TN\\\\\\\/qZcfQXM\\\\\\\/DJPV7IXCJPblWv6AwKbOuda3kVKuhiz0sz08dZf7yBRdtBOsfPAFJqy+J5I+1DEPhyT+YV+YBgcFyiFd3l2qPFRGTg0K4CdJ5WMyqGEzoW7R++\\\\\\\/T7HfSDBXztd7tsvdEp2LCjOkdzkXf+da5TKFfeEeefKERUj0gk6LKzrEwSgp9Y4GlTtLMFbXgx8qwTGhx5WEBBz3fdC8hzPb2deX0KoClsgCFTsun8ps9r6eStLmMawTSxkPANGLmm47IBgBdw2AM3kRiEiEk+WnNwYp37ZER+fwWJOakxYoLoqyOU47Xg7Z60rkwdPXeDMbNaKh3Q4Fz3JbAFITaj0nw1koZOqznXfj4N++Dxz01QWGAE3kCI7OpqrBjdKT+l0b97rWD0rByMEIBgsWC0DW6yYFB+gAbxrfqlImfxmDgSTtww08JIVmuHKtJLT+XQPDzRBrvmfmbcyS2bNjcgDabezFwas7NeCQe\\\\\\\/sYU9x+6kft9kojW3NoSe4uHC0Ne8mLWCcLbg037S6DWU2tpNvHT\\\\\\\/zAerVtuQSArGGTc4EjzAibnoxYOobjjOQCAXcI0va5A0PLMlyd\\\\\\\/eNSg3+hcARSxrFqfejw07Mgo5iaf9+H1hZ4NTbTzuXHpTcT84kxlwdyZgKpoGCR+VnrIOXna6y8NoVHkPWpYbVnwOwTu2XJCqth1cVXJ2gDUUzHChskkGrS9oN7gvfZWWrBfVC5i2BetfK53v9CVBj1eB8OzZv+TklIa401F96piIMGSlq+r0GSsHwVEUCBJaQChHyEZG7XYqrbxnC0WCdDPNd93yMejS5xHsHeiKDUKqcfH0gyMNz7KZnbJZasK3DuFWYrwHD1MlJ0mALJLEaZpFcgr7YiqU3i1LhETg5\\\\\\\/4RIqwajClV3Kc9tifaxfAMUjySu8\\\\\\\/ocsBMI2f4vwhN+Fa77Y0KbWCY9C92vtiikkS3uB\\\\\\\/OVYxv4SnUxJ1782K\\\\\\\/sc5FFR\\\\\\\/19lK\\\\\\\/xp14ZcRhwMdGzLqw0nc3oCfTY27fwPoCoa9sE6fOvuzsZhrw1ff5AgonmRk34Vd5IujO28jkU8zoRM5KMzIZkcHUobjFBjSEx3Lq13mPQjKObYemFXtbSZI8V0V5nzqONmpt6fMD4pKZ6Eq0XGMhsxTZgUccVKEaNzRi0iz7P\\\\\\\/qXWZaT2m3b0xosSCaJm25x3Z5id2u\\\\\\\/j88Wmf\\\\\\\/z5uSm9IGCw5wbUgqYOD5\\\\\\\/ubGFppbzF1MioN6xszd8eTweWk5AyNxGFHw2GwZhR0mIzrquwvr4nnLYAewFCoPkv+tJLjpkLWYAOyrmqEnSm7sAiCZFJI0AHpZ2uQcDJYWXJ5ycNC\\\\\\\/j7p11s3wAIlThb\\\\\\\/p11Ot1ZRIyRSavCN3P1ligOB+tLQH+3EY48XQRKpU4KmgGRnW0L9V\\\\\\\/ROYnojWUbQVfh0mXbQfQwKyyhRQ+WEMhvXGTGNZh4UcDyXoHBLdl3vS9znfZUUeseq2PlMO8J3S86AtKaV8\\\\\\\/2b\\\\\\\/e\\\\\\\/S1BnZeMSfc0lsO\\\\\\\/GOCNpVpEvJZiDxfTzKCHQPITAvg8KOhk7BUjE2M122BzTwz8SBVB5+PptJMox7WfZBuEgJmLVkGgkCPzb0KhIQAfEUW+SsEDA7PBoAdudhrP\\\\\\\/zF+3mSlf5t2Cv3GuWs\\\\\\\/D8RWHujwSLG2lDFH4ZBpr7c+ur5n6WaDo4TusZrWrx9Fd04N2ywlcoISPBf4ClfdAwumv43PKZ9MDE6w+3qzWWat3N906eI7P8uCiQ0y\\\\\\\/YMNS5e1veO131+EEVioa0Ny5vfMOJJvpqd48u23KaiVJkOzHCD9rZ9plif1eoaP\\\\\\\/3H7TL5YYoX4Vu3JNoUqZd2ahyY1Rpxwg0bsHD5po7Bl3COwOr\\\\\\\/4VUQuJTYuHteAu85dgRD4w6WkVP8FCpw1xS1CVzMqhsmkJOus1RKjyFi8FhY6j\\\\\\\/KK7nDxE9uc\\\\\\\/tq8o\\\\\\\/jFYpDVIHP\\\\\\\/hCKTPZv8Uh3QP3ADv1RrdwH9hifKthdA7m1DXBvSTfmGDrEmOG+\\\\\\\/tGPXK93yvC9p9YMr2PSWuc1eob2iA+aagEJoN9PwdIqm1tpuBje79xyVIJcJR4mJEZhrSLsWslP4RGLSCxcs\\\\\\\/AN9fgTS62cRWqplGL2QLoNc0TuefGm3XOWRW91c+UW0+MrW7I84z2xVR47gfZr3rj9qQQfOWG5eILou\\\\\\\/7XqP\\\\\\\/VabyIh9Iwcq9N\\\\\\\/L51euO8MUpQUIFmjDbbBaJz0MD\\\\\\\/fhG6gnzB8xIOgYAna8hRzkTm3oGy\\\\\\\/wk0wMR32gWlOQxtayTElgvecFQJDs1D1PQMnYcxAwj7BCrwO4DEkIxwf4y4cNlPRvL+iAQxiyAFWFSf6u0diY4goH5L+xOXV9GyGuSB2kXX79hUuxgQU5it45+zZ72PGQVBzfOXzbx72pKBO\\\\\\\/gv\\\\\\\/mkE1YTf8rciMzR\\\\\\\/ipnfxSgfRkuU8V\\\\\\\/cPp+rFtXXlamTmCUDrlD6Irt6O+NYOPYfbElbjWSZOun9m+5KNqtWtndX\\\\\\\/WUGnaO9DtFawdOkaWCCO3PYaGh5dbh\\\\\\\/2M7pjEZIF7d8Yr+i5RzkobeBOKw4NV9e3hzHDGira\\\\\\\/BVYm91Lres9mV\\\\\\\/DBvSCqDOwd3DZkCzMX83\\\\\\\/mCleouNKH\\\\\\\/T9v6XNthtHiwaGJHRlLEH0RrktOXJZdtnOooeMKz6GE38L7p7zIo\\\\\\\/VinZ9F2crHDck7jHeu2jG1QI2lsNglBHvLIKi7hfUbdjPjWnvXIBTosHi\\\\\\\/i3xdePzV2dc1LQwY0Yx26gkXm8NcaTKvCEOZudJU4bmjahzg8TFlpOBg93NDis5ME\\\\\\\/gb0qUhXlpDMwRMcy3rQDXyxpuGwAZDdt1c7qhU\\\\\\\/xn4qC7JXH4PlkVkUZAtXS0t2Htu9lrCzswOuQAqNfZJIiHF9VSw1vCASLsqdVQcDGXInij75YH+PXspJU3N\\\\\\\/HIdwzF6jpNeHtgor38yuyGcDujbE5Az+HNEFMu+D2K8yU2pJSS58BKfhbrtghWhUPDk8iKG7PAsk7HquBMbdyvtAlz8WPW9TsV+3ET4faaeTUnJ\\\\\\\/UIpYet8Zx+1XrhTK8P8g6JwXvwWfllfSTSveM7TPxcWQ7lsU7MJBBS0FP7IozWzUX83vaDCOYjI23k3b92CIFczQEuDMm7HUaef3UR1Ell6TxZIPGZF28y1Vscped40RyQfdWoSrlPuER1WGGWnhSkqqmZfuoufaIbnhgDdhFc2Q\\\\\\\/GabOv5QqTAcZte5M126SvfC0gFFc0nJHlWcYgRLzM5CgJS3TgwvLCUMd1BAvTkV3qU+ZCM6UsRY3QWHYzqHWxDp5xCl\\\\\\\/ze\\\\\\\/MBAUw0kLMkd+\\\\\\\/IjRiNbLzczkJsT\\\\\\\/PB5KrDxjad2l8EqOu8CMZSBh585sek527Uwf7OE4Q7ee0hrB4q1ia8iY3ub20CJkfAMcvBJHO2McYIfSGiLkgAtIOSRLuG8coVYW9fykcFd4bgt9nz9V8InygcP3sdtzZ+mb4EJjBKEFrAVUgeX4uynr3MSgDGMteSEPcxs0GrFCL8UIR5T1v3KZOgW5lgM2BGMtOr+BIHaZIK5zcnn\\\\\\\/z5G2eehMsvB9L6RT4tntvzdpnpzDsCArya5wnz6jQEBgP8sFZq5GS7apROQlUKBSAa+QJAHOBDwInY9LmQ3wSrHn3unzg3\\\\\\\/GCJWPE7gy8KjTpgauq9OSNqE3DP2cYvAqNgVEyaFxQiWigGBXxwSN\\\\\\\/DisgcEPctKQzdmnsFYFTCKEJdocbdd0Gl\\\\\\\/g0bojE5s3BokNenXygeHm9Yayr5QHvcz1TAq\\\\\\\/Vc2MYRGvnpZ+MchpAVn6NH5qlBWIBnZQ\\\\\\\/GepMda2k8iE9VHuw5JABGT87C4aYwalHuevfF772n1BbWKyOg73AA+2An8PRJ37wF3XeYUhuaWn+pGYbVxLkr\\\\\\\/tFPLxSazqOIupBn0ObhqhbxnzAdH5qAHH8NXWc64W2WRaBIvbH0E5Qg+rPuM88WAt7VEDB5EiDpxo+62PJnbLbWX0AP7tFbzLbbJa7BJk2Ho+LZDaOl5atGdje+HUxt0DWh04dryaZwjGsaykczhh\\\\\\\/b2TTR8sbmm56LowsDcy\\\\\\\/WFySyj+gTsgjpPDiwc+E+P1Hw2PsqtbSGFaYEHqQr9GS8vARpzqDrFj2nBebxdLTHY9B1Ly1Z79RGMUfH1X5cpbezPfR72Y5hZ55KYnTUqykHxBXSE8w5rPSwHn2V7L5ZULRdvAjtzeAe19gihTRL4sMViNdmuKhTZd+VH4tvcS1VsuhmmF0UXLqLMNnev2yIsHy+jgwp10xzA\\\\\\\/LwxPLWsM\\\\\\\/x7eaR3hoQIIlaq4QPXRbhCgV3ymgg\\\\\\\/iX9MmAvrRrl55W5q7vIWN\\\\\\\/UGCVWVi8PXxQkyvfzpQqdw07p6wMdfwuvYxkNfDXODYH45Uoie8NIRpp7wtyRv3lgICiAheYmwIveZbAWMoB4sBvx8AbpzW1j6MlLmGSH5v1aeuYGzWml1MVYA81BhSi0R4KaVfwBxlAJlTIv3w+flHz1qGeZza4mVKg4RMTVbHzr20AI5tnRt1uSAvUI8UGz+z\\\\\\\/P+T+2APol9xlAnmRdfJBkswUMV+T6MnFZBOd1rBobrMudnuT1uGOrF3J5dvLgZe8tmxaKmg3OzcYSKhGW0O+vKWscYBdSWmj8Ty\\\\\\\/7eNWRGKOpWnP3mnpSWxEiT\\\\\\\/P4g6VFrGvQHtwP6Ehpy77agTqa4CzG5Jxf2Ei8kRT31vitntFAZlrdrQB7EzPyrgaJnK6Q2tn3zHkPciEkN1ij6xMwquu8jubmIwz++35IZi8Spj5iYgp2dBhUpypYJv5aBfM62gb5ipmYRLKuiG+ODtTG1ytGgdak8ZWEnZ5XR1rqDMEid5EN4HzM+It0D6Pex+CkULvez6N3CIeIlT5+7ZnK29RD5UfMY+hMsUeczE+RLiZCj72OV15S\\\\\\\/tJGTr9v5Hyclz37FsIAhGXlOdDE4b1ZYQ68z4WsJA7bXf+CUc1UTnJbxrQt5gwwZ6GYTGhdH+hmBpPVQ4JBEWgYZFNhmKtj4H8uZq1o0Hz8g+OdwzK1Rd1ab8\\\\\\\/rmtxNpo4WpQ74BGgyxfVF5+sqpW+oIWb66LFjL1wQY8urukxwWSNBoJqdRQ\\\\\\\/dVvqqgobtN820SKveIvqAhWCH+oAhMTDe6HCkhbyoctMrzLnh6MB\\\\\\\/vybJTGeEpF0QMGqsTuGyisAvWfSSNZzsRex5L2SRq8xDlwEK2UxPXrnsspMxOD76pBE3zCgBCmY9MzI4djlkPcWf91pa2qLbSVuMeOqFbXeVp2T7skhX9b\\\\\\\/nUK7wHkwgJX3chkdOMq+1lUGH2pTfmhcoK009ewZ+Yilp+u63BSNEDp0eeUoCzUohKhc+zockonZRTkuhCKOXTQ25wTyn3mvaJSeDNDjMy70YNwWcQEGFyBB94Lg+qE+3pcvWf\\\\\\\/znJ1MX+sYnGDWVO1MYSbkR4FksRYATJ5idSbM9n1VJ7ygqpVMkGFfwLEWjMvOB+HlgZhydYm7ZD+grz+E7IEeE6eqkhqMh+lhS\\\\\\\/eUpkIC3DwVYQpmMQ\\\\\\\/f2s3dNtoA0fwYTKp+Glf+QRUpQYK3c+B+5irDkqr9KvjCDHb5JxsJdPjnScs8Xpttbte\\\\\\\/pm2kNf9GOQ6C+rMRPHl4OGSXx3NykUwEnk0LcD9pDcqDMEGp+Yme1XsMZr0qJZsQhgrOznnBNLnfy\\\\\\\/oUVE7D7nnbFVI57VxPQulzvwrvDWdNZ7I2iG1FBL\\\\\\\/jhzsjRsUVUgMsWFFV7HjE607dGY4fGxejpPCmXCzq2HQCnDm6ueufjkbwbDWTen79weVjCidE6S+IEsSg\\\\\\\/CKSXNjjdK04ilXrCrayZgXuUEp4z39+atbr7LOtXCOTYx0wgcQMhpILaV7DA+nzYZqRIO0K\\\\\\\/D0+rKcZuaHvwCNzEIMbVa4sbCFEcEW\\\\\\\/TyLYpGT+n8ufUr6wy5SaljjPA+Qc5HMCxeWuM7KKRre8k2Qj6kVZaBTlRRMdOWtcpse8ywkNEl8jGUERIB+btlYtmUYVzv9DHPtJaxXX+m8CHLmHkBpsKXqrEaziOe6o3oFuSE4SEA52XZSKq51QQJ2t6vDH2A\\\\\\\/smWAqrGcl3Xp7RB+3SStrYQYjcgQOZYWV8urX0CVpaKx15OewyrKYDa0bOEjKVQsdJq3wSoBFvFlG6cKf+vs6671nEGrM+Hd2J7WrNkmNfS\\\\\\\/hweEVSpdovJNqIfx2EPC6utzMGQgH9ksGvEjldNyA3Ytb6eF60XbE+zib9mhB+g9ZNlGxeeyw0OkNiHZYWaHW7lEezhrMJjJimpGpahrnOFugbonq2l9xxD10AW999auVQli5hlq6uVTm\\\\\\\/IUYOEzGMCSFOy24Jt19ErNJ4kn438ryREhIXqOudY5cmE6HoCUT2XJsUsFGg0mqnaJafwpTgaXSEkPT9UwPRyZnlQpOrJ8zikvWq48rEpnxfoNlHjDMriqcfhw6kLDwDTbeTNsbsNMVwSEDdfgq+JBTtVRLh9NdMjZkhhtFJaKs+sYreVcL13\\\\\\\/TcrdTQ6S8jw5DLjyyStXImSNbYCGen3RNuaimZzCjgbIZt2\\\\\\\/TGEe0P4QSRRmVyGtW+N3eyEW1hsZhpg3nnMRa7+Bt0APTjuq+NpNXfsotwjJzXrtZVeMrk+xwvbKcBw6DiTYUPIBn5TCZcNzShFJ0T+iaqMHy7ZprGX3Wc8btaaNgCVlmS8FGUEhTpdmPcz63D\\\\\\\/pgYfPpmEKsAFuATMXNAmrbaccGv8pvHUhXugu85Io8rMtksK1mwOPFvU9nxqfXHPNLn+tolLILP9nCbssMjciitZkAbqK6Dmd8DX4cYmNSLywMSeiU2MVVMPQrJpSu2nmqA6nuR5IHA8vitHALa3YDLn2jmjz2NA2ZTOo3J5JFz2yoQm3+w8HEdvZXtVVtiq48Ly+g+X+EG0J0IQpUcjZ24Z5Z9Lwz8oFVleSzEK6meVubL3ldpqmvhnNCE2m1u53kSuvmbnEB2QIPH41fp8JanIyh8cbmBrXCy3BC5KPTykEDJ2IcNFTuLlFQ3wmZlpm+DxStN9tN7nSzTYjVPwXiOAPrDdKiJjFjk2KHC6H71E2myj3i19drtQArlizCfItvtkFPkoUwKXxm7CxfRNmjEqfS6qCXlBmvbhrqVI8dbjKjUp74Mjr33CJbhZ1Ml+Hwp1vuqqQ4h3rYFFTOOWtfznPpqQmDBU2hk7IajZBYWeM06OKe1PIXZSZ0dgh8tu8MDMTSY7URndk6JDMio08\\\\\\\/iGpQPsokIhdL8xvBolSgvK2V8I0Kn0SuNJn50hlrKrZDy3N7CrB3Kt6nJMterAEcpZVfNxFXmstyvM+aAD0YQEBwJVZtQnShp5B5JBPPD08PXSTDVuZChMF7vthT7RZiSWfDEO\\\\\\\/xtRQnKMZhDq7EA1UnIJmi8TS8ZtMWUtNk20o5wy9ltVL+VQf6lNWzsL7PkU3rhO9OtIBLjvIPPnIg\\\\\\\/BK66J9Avu1\\\\\\\/TsLnZOXBVUwmN5UmAK2z37meKn7\\\\\\\/9g1jzO6lvhZKI97BktSWHp7Cf8r8wQ12kzG7QFD\\\\\\\/nUQTT7jP4bkO1PxEGFshNnPlkGUS7EQe7R3lA8ma7o6CJYHbxWQ+\\\\\\\/5WxNOdM7w\\\\\\\/v+vqk6SN\\\\\\\/zy6z0VpO5B0Pp3omN22apYWYEI6Kpph5+EzbEsvtre\\\\\\\/wz3wFgLf3+9ssm3HIsMH+1Ha2ZLU+csXrBKMEylEFCK8\\\\\\\/bUVqrugsbeP6sanfQpKKYjS\\\\\\\/0BelmA8bo8oqFFHiCSWCt95ORQy6K8V+1m8HKCWUvIpBbXoEFIRftpQ\\\\\\\/f25+AYJTNSgsLucailsYTxYOaqYrJ4lljhUe4gxR8hAS2S61HJEoMIgyo+I4Mu5je4INpWr+DSuCDo3BI9kfnOJ3tBIV8OIYho800CH\\\\\\\/SEmVk5axCDjhVCj+ZOoH5jZ6SLTt0ad8\\\\\\\/BeeqhZdPY\\\\\\\/T0MUv4Fqdt\\\\\\\/1oS4mIBdP3NkGRWgE0mvUdEs6F4axlM\\\\\\\/QGGRTjwgtKf1FjUA0gUswQLJMjPNVSpgySDabqQh9C31dnALP\\\\\\\/wUbxLatBWDOSEciM\\\\\\\/SoC+t\\\\\\\/YL0a\\\\\\\/u4P2sJska7LgrUDHTqA8VSNhsZ8SM5vClrJQUKUHVrzrFaCNPEMsXK9uHIVuH9DqyrwioLkO05PjsSvBkgq1IzYZmrXAshsfpVVk+1nz82m7ZiEKhSSyZVX2nQ3o7y+DvW1s9HwlRrZlsT0JK0x0e3H\\\\\\\/OpevBhUDjMHuKv6kaUPs1KJONhQUahlvgft4XMPSlpRpPD3YCT2wvIz\\\\\\\/YbWRAh4uNMeCGv4jMevsX9vwBuGBdirXWTrKqIzcTO+3Ag8HLgJ0oRMN\\\\\\\/Wg0Vv0cvnLUqQAFeaaDq2F+a89+VnxLOnW+bPeP2LTF6bW\\\\\\\/m\\\\\\\/llFY03651kglM\\\\\\\/KeyEEktq\\\\\\\/OSzjY5XSa7SV7vMRRrteodkszPeNj35TZ+kSDXmWhT9eRoHHYD9Vsvl2KUKYHGFByuLiEJOteiBeMpBTbQeEbHL8nrdHbwa+mkmIFHb\\\\\\\/fbeaNfhOjy6xdEV7o69veWH2CpJy\\\\\\\/YNSTR0UUnGSl4GmrQpjP4qIgOJOE7dBC6gXsL12MLD+I4EUxz16VkEWWrn\\\\\\\/yHqOMhRfOQ0ZPtZ8bMv3+ubLloZ978ckm+bU\\\\\\\/5UlyYVudYJcQ1oh3Jajcwl2zEaJ\\\\\\\/VuOJPH\\\\\\\/AXDaSNSTvPlajqgXG\\\\\\\/jdCgQ+sI1cGsieu0RKVPmbO2xh+8no\\\\\\\/iFRKIC6UoEQ8RL6uR\\\\\\\/C8pFD76n0QFx5HTXH0ptATdweZCwyc6wZ0CWm7b0\\\\\\\/6do+SOxTErJfVuC7Q+rfmY7Pak5F+KDRZzcHVM0hUCyFuLmXcj20WgsRzSMTmkfJ0Un8\\\\\\\/4A4Fh7tEkkrVUOryHE941t1omsPa0gw6iDHablQL35vyGxtIFPDH502UtO9TB0nculj3cUqJ3UjIrcEj5f2LX0Spk\\\\\\\/lmmOObFxBT7u5u6MujmvCQFQkh5e69feT\\\\\\\/V+6h8WlWQUYKC6GnKzn8spnLw1UMwuIFEMJyaOFlG+vQz9CT1kWOhPIIGvou9nrVsOl6L6+kfspFU5LKK3myjALwc8JMrOH1o48ijKdbI2zxMBBj3ht9PKw0azBQY\\\\\\\/xxIBnrLshDLdw88iPx67se5m\\\\\\\/J1YraCv0T0cQzAFD7CQYG6RMmvkJsAoVs5GUYwvWxNaF3d36TzEkYJZr5s\\\\\\\/HFCj5pixYrCReiTvxcabGSwbezdiwKWiVaM\\\\\\\/pcXAj5a7YU33xIGaN505cKT48okQ4pN2B9aePNi4YzZZrPdqvB4UoVYbxSOu7+RoPYHCe74Ja7dI+Ttf1DO0jHbdllVJISqdoB\\\\\\\/UNB9KFSFGfCxwiD3sPIMdJQ9Ky0nJ7JsMkCqTi8IbOfMt4K4brz1NykJNrxcOWiBh47rOUDbzcMLDVQ\\\\\\\/HTZ\\\\\\\/NCzwmtZzmBbJBsdJHsmq\\\\\\\/LZDTWbiX9WDKIUt9X0jUIqUwGLCFIoMiCuQJ8i1dlJPbu7zVqNALyoFkVC7msWFubAlg9uO\\\\\\\/owzf07hkn3XS3UXh\\\\\\\/FQypIlKLg1gh14wZgkJha91IMK4\\\\\\\/ZI9u3VwBEj5l1rAdDdLVWmpnNPfytJ2g4VEoyU5IVgvkOOb8\\\\\\\/uIflvx7U9YpXuUWS1M+5Rmit2DlVVdI2sLq14w36ZfvN4SnvebPRCzp+UdIU88JNArGXeO0ZrGXk1rVJqLDaTM5UuuLFp1j5uz2leN2K+7QvneHPl+xdFYMpug3lVz9AwXsWrVrx9zdKg2lkgQ5kEoOffT\\\\\\\/D9T2CgJ7F\\\\\\\/21ZALd0FE7v0Lyx2OWcY4O3fzai+rALE\\\\\\\/bpwau\\\\\\\/B1bTnx0V2xOsxBrjLO3VVZuMhXzRWpyJEUkOJradtwIUMYK2ol9q3qKLe8Z9Qr1QzYEiFXAfMDs8+2aUh89mkGoJmLe1oFSSnqXp4MiFodXzqYh4NYp3vv2QacWYS0EbBxxQJfAZe9XRk1RWzsk8iziJvDm+3OD1LRhd7l5J\\\\\\\/RHhRMvdpyd7zNdTrvY6YEU8j2ViXDzkH+IIrvaBUIguInQ9QHdNTcnK6AsX+Kaf8cDp3EYuvuiHoPTn5Ew429tAP3Z6Tld2lEfqksSAg5RUWWkNVBQDVT39L6dFJhhLzz\\\\\\\/5N6oLwkrUEqJ84oKOL3gIxKHo\\\\\\\/aUG6fQkESOYAdjm+Ma7UelGTzWluXg8g2N8wYTQeFeiIYT6USKG\\\\\\\/gMmiEZN7HKI7ONWW7PiqLzSztLSKQ3wTJJn5FxO8mcpuEScFIU5dodeeQ1Pb7GlYRw6SYFzI89mfuPNU51CXlbjPff9t+xJs\\\\\\\/mnILV9pq5Yu7hYn5NNVFyoFNkrG9TRtK+Tz8fQUJhWnMS80tdE9lArAhM7g3eKGDyItIBvoinQhGC1eIkmp9h0ksabwXcciEoldVg1cxwBI6H6ab1gznaHTS6kdFtYbmW6zq1KPRIKegZaowA+RoYXkuxo4uKyYZjOiH9+FwKZkp3\\\\\\\/N9Bb4Q8yxF6ETkjag6lQ1DxqBDz0MzU1O9clRfgnvCrRZh81KRpvuiuSrq8haLkxvuZ1oEPnPouWsmVvvfv+9WOzRJ79sA2SGJSBlgRVzj9X40wAHteCrYTdPnMYL4jEgmw1Y6YagMJq9LeYAh404XyBfQsYqC3NXN0ejYRneCz49ntB2Dbj9TFn3OLn3ofbHejuzwTRjoaRRfVYuUoP\\\\\\\/CXBeP0Ruv6t5WB54EgTAm82if4g8K8aNt9iYbtL\\\\\\\/7unt8mGiikYmeQUge3tZj3d7IKkPtebKxNyyQGSXhV0RI9AbYbZ8ODaFkCWwKS1+VRyPMwV2B6Mw7vRBLy1TafDYqamWEhn0Hy7YyaIG2wZ8wUNjMdBM4xW46BsBk07\\\\\\\/w6G2xG5GuSpyR5JSO1UX+HoXc+2Ak8YT45SnjzDXfo9zek224v0\\\\\\\/4gGw5yyC50QsMG1rYxRlvZIeLGryj3Nc5rGQ6i27vwEad213sc1qaCYuiT0YIY4CTC\\\\\\\/QL4txJPMJyDOY8JYhCI3OJ9dtjQycc8kkAOCb2GuDM65YYxNRNsEAFoLQgD3N9MfINUO4qThJbb8lMkkw8bvf0R38GJ1M+PKgS54YFsVanA+BTUozHZUXo+t3aePRACZbu5C0RWUYQmF+CKrP3lpx\\\\\\\/FW+kIglSJd2E48IrEJ8ZJH8OiT85AdRAJqbhPlt80Bz1IzBanaSi\\\\\\\/rXFU6U\\\\\\\/0jn0pCNhmhcBQTtO4ywntotwzYA\\\\\\\/q6dqcGcDstGD+yUi6szdfRL1OdNmnLWHGliUOnkvjL7fa7n+KhgA5qNMU9UxTkMIqONG6CzTfbqhNEQOE+dozJXR1Q2vdJKUtna+I5C0htV08rGQK9tmMMkPL4l7eAFwfgq3fdy382NECYcFvheUNnVfhIf03FAT5xGEbUlgT9eyST77p6sjrt9vHsm4cWm6cn8jA6hVbheserGdSAL4U3MA7tujTmBXNp8KzPrR8HR0nK5OUe7B1lLll+uMfi+NUOjEJk0WEojHeaQdNoq9sIogDbIMAZRopQCb0TXOLSxuMJ0atDBK1SuGdxCXDXfsL6L4TeGciA+oQTFhlpVYrDCXCTnsW70PZ05+V4r+J3aVK8kkYz4fDlz4U9yOeZIJ6RR+IH5\\\\\\\/HcgVmNzap6D7JkuUL\\\\\\\/nwqXwN1pRURs+94m84Lex9cqLsoc3tBobbCbNpgH\\\\\\\/i21qpJ5pgosZpgDEaxr3TEFTY5caocdzzDml8Jq9AyoLaqeHRpfrqC896aUrN+CEfsYcfDMprJpOVNeotkEbiZ3fuSOUzI2Yx0D5jk8FGXxE1MQwkIdGbF3SlfbgmSnwT3W11NgLjHLVBxQuGGDN0wCmDFjz7XqQhkqhIP4o5wczMIT9+5uOVxnFxyfSEBMa\\\\\\\/4FQyvnI3BZgm+s0cQfDmaLrwU2EnOQmoX8m3F22DVdWAPbUoaKQhE+2AVutFO20lmxh+6s3ZsCuoIzzuwUnECo9vLRV2jPpjG3h4BMtq50dUxsFPpEDseOGzX1CbqtkQuWf6d1Ihzg5Qt9CfNnnlJLMwS3qiYGZnwimxQlb\\\\\\\/3XAw\\\\\\\/TYVxj5xdTx8OgeAVGu8innhWTWQxhXzIcUkmLKhvXe9qNsVac9nO1DsFys\\\\\\\/0tBykeVoau4359Z6dPaK\\\\\\\/2\\\\\\\/pGvk29ByJPXbE9R47Jd3BojcRt11NfnR3R\\\\\\\/p\\\\\\\/RWk7a5ONbmht\\\\\\\/Q3kRdxkbsKXdbELcDJCph1hYIQ+8PMCRT2jTPBs5zW2iHysSMKkBQ8zhQaPlLEoyz3oqVrQOspDgjzR3hCdCn\\\\\\\/jMBnisSd29zdLkEpCkmmmXuSBSnU8gcm8siwbCx4pvr898b9fD8ESn4GU\\\\\\\/9i448n9nsgT0tDOwC7D7C6Aj7B9\\\\\\\/CuDMDn7Y5833GVAAPugKqkr47VoZhSjEo1upp2hwrbO696Pz2lc4r+owrbkLFClmKL6c2HglgD6h+oPnBe3X10AklkA9cADdLYQGhAsDnfN9ejvDL33kPrORyP+DGok\\\\\\\/QCRbXaBNGtuQ2cYT0ks5Y935lLwYWoIyGFoyRXmmcEc7gv5+VKPC6Et0oLJp33maTDnoLpz2xWtLXUGxL1BoaRamsxEHdcixJKMx0uNqVNph4yuMjunkam+juZKTxqpRu04Cc0JOhxkOzH9hQ4wDTm5CceKgg6wlJR0v47ViyC1v\\\\\\\/l9skJwlF+fHX5Q2Ud2e6QIO7fi0NDc4hpFUV4Snz1G\\\\\\\/uetMxuI2yC554H6CTXg7K37g4Z5Hvg6vls6SnCahE0d4dYtsOfCEam2Z0gKHSwkVXmFxOnNozJqVON3fn7FbHZQ+iEQGr0tJk1+d\\\\\\\/kS8D1QIGAEdYH\\\\\\\/Wny27V1s5wesJtGrtNhffP+HoN6FwZvjKPUbFp0Lv5iNABWDJUc5GZqgcSP\\\\\\\/M9UrSEL+EU9Ly0nQc2em0+g0ivn2xsj2KQxG2h\\\\\\\/PAHuASPUn6Zt292bST68x2WcbBzGIenGMJrKBas9+7fl4VIeqzCabgfRWJS5lsx8PCyPE8dF1m2IIn14k5mJ8kSk6Qr2pt9c40cKnAVKlqBaga\\\\\\\/Pd\\\\\\\/0objtWm16c8LwOZc6xiisSpnQRgYBdcFluPRx987gVw3XubuhZE45CmJtVvbJ2XArk16MWvh+JEcGlkvhORJIS2PI5rwbEvDCOdwrjoF1BJqSiudK\\\\\\\/MpqubQJP+wSv1129LRG3FOMRP780dkYeFYd5GUEunFX6l8VjRoUN0zDgY\\\\\\\/92mm45mXdmWN9lMprkRWdutPtrm5TV8N2M0xLeUi\\\\\\\/K9IMIP620UfSHTtoH2hUouDeXkny+sTkXoCH8Jfb+\\\\\\\/\\\\\\\/24TcDdf5\\\\\\\/eBszh\\\\\\\/sbiy4bGX2llcUYM9lwE7EMVl\\\\\\\/f9PC8ruERkjOa14hQzMpJXBLipeLJY4rGbeDPMX+jQ+ncWiGXt\\\\\\\/cnEJZ0OgC7CcXa6HGSJxPYZfnQmqUG\\\\\\\/n6hcuTlXFIYzX4AVeTbkmXscys+a2x8JaxZ3NTqcP+7hhrUfijq8EsZmv7lm0JjX3GzpWDi+9jau1T4WqaHqFvHxx1Ad52DVtsVQOj71BydOk9BOmwulKK2+cZWCSBNTtcr9k0GgKWNBuH1Zm6wmHeFSs\\\\\\\/83Y7RomMnppAsapEE\\\\\\\/TzsYAtyd2Z8sS+ro+TvyBoaIjzYA1zJYtkbv4uSBVJuTcM8yg+lmKhuAruVD2Nc+JqWsRSfLpWY4TnEM1eqbAVv23m7ZUWExgFhOEmVKqwdvjEhde8KuLAdgAlKxAx1xJcYXLqje0CwuWkys6Z\\\\\\\/JR+mt6tSCMDA2m6OThAcJDLsgYQlAt799af4MFX3Y\\\\\\\/ZYl92di86MDfrPTKMnacNGZc8kOrtdfddvLzlBgOiRpU5OsBFcOrbvYF51oms6HteJimzrXPQz5zmCrGmbaGeH47W\\\\\\\/R1lc8MFqLKkAn\\\\\\\/gtXdmKzdZFmOxw+On6W6WSJbx8JCqARBpgBx0zSXb7TEkGpjcZOgpnagjg\\\\\\\/wp70wHVJjliAUrzl4zBZ2H4UGzBqLnV5KaeuaHQY0yGt+zFPfEJ9Ad6T16rVQGtjGku01i\\\\\\\/tenuK1Ia5Ij1dADINAn1EfPq49k7lSy5peT51+WOr4wix+cF9XPhglELSymyOL\\\\\\\/ChPply850l6xBkuxlPokDvCnYdWP5n0VK775k6Q1Du6D9WS66qG3x8mOrSmz\\\\\\\/4LmBeriXRyF3dfZ\\\\\\\/LgQUD3qkry9FFZCWgMR0WYehd+dYnhiZ389kBmuB+3K+I4rss1EzOr5xIwpeuvcFmH4RVtk1TXsoxxuTTY59VXxaCxZBGiY0Vhn6wSUlCzl1sPNwkJpjAgFbWOnMlJJPpvJWENMgORytNMXdA+wEcTIqKaM02UdMaigUyEw2Bp\\\\\\\/EaK+1XQZru+EJ26gdR7ekjKIjoTdOA+CHynLVX5sYPxFOSfFmL+qURCEpLD3KIKNGDQuQWk60TJfv6q6weuvvfd5LaK+3Wa8GsMoiDdflpKf3ieNjf2bn3gFnAS3+QlxAF7ynZ\\\\\\\/uQOrwlDYLLkmly86wpn8VzM4O3uMzRjSmiMNjHyjZcBOF0t2\\\\\\\/4khSy8hhEOT8sgU56DrW3dDhRcCtXdUnl+WrsEsmto8BZ8FhQmQg6PsL\\\\\\\/egtWxyDx10PH7zjrstV+lexau8Bvrtqyjp9eqmEm8ZEaOlEkv5XuQ\\\\\\\/ys10iJmFWUeaO3l2fmM9vcLv9T1QILT\\\\\\\/ORb2f0ICk7oo3OLzJ40jLJrsEcYL0L3MNHYvIN1J\\\\\\\/skF5cerL\\\\\\\/SVbmvv0i3l0fBQHzkgQNIWvDNNV\\\\\\\/iTWM7j4Y9beT0V8amEQvlMroy475v4FxbdBEeYLJ9lUoR50UqEGHVGrXxvHKKSvmsaq2wplBKhcoFwX14MuIaQVf33TEC5JfXi48uabzgTQjgBIXYPRJgS58rzYGYjx145+OATsWqzOFAJH0jVUswVZbFC8TajKzrg6JGTN26kMVCJjMIxLG6qdoZuhrQV5A30V3BEGl614vmHfp5BfqKIkqbgT\\\\\\\/ywcfwwBL3vpZBQ7bML6j\\\\\\\/ghSLiUtIwW6Qj0bsT8jYjU\\\\\\\/u8fGrxk6uxhxsOo1Dx8fZ\\\\\\\/r66kQP5WLFHEUQu2wkCEDcPcvMHDErat65APPmEknUeVruoabT7jPUF4\\\\\\\/Rdh6Ji\\\\\\\/zWvG7TxWFnaB79o8uNZ28Bw\\\\\\\/Mv21pg\\\\\\\/ED9m2\\\\\\\/S86MrmUODPvs883KRQT4JXhOPyKYl\\\u0022,\\\u0022iv\\\u0022:\\\u0022qqlxYqIk6RUnu3NzqdhbKg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d1b15150c0f57aac5a3198d0d006fb4f5a049e86255cef9c9972f8ae14b15964\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022fm+9jFblc2luXGQMACIiENoEQY+ZMww4RIPhJj9Jd1Osht1LIdSmF\\\\\\\/QNeIPblfxYYoQjFSRWRWcm0csPDdxsqQQlnBMGkGMLD0DEZi8xptp+0SmVPZ8nsOu\\\\\\\/Em\\\\\\\/RDgCQxhtbQVYYse18A786+pcdDcVLcH2+qvA0dOSprzRIzxSb9k6TKO\\\\\\\/bHAUJ\\\\\\\/++u0A4opKgL5GvW1EQb0INdbj8YM6yAcM8XsTz5QCNAxecqPOvQhX6Wk29zGyf1q+0Qfs3+u+VPa\\\\\\\/t\\\\\\\/4Sw82TuGTIOxRmU916OOApFSETNFObFbyzgbJJ\\\\\\\/oODVbucfbtRxghET9JsHgnpeLZBeRPxCwiWbParLB36ybbKDmzSITw1wKOkivzZqE8Fz3MTmvLrKQZvwDuGD6w3B7HuUXWnyJVO8OlCe7JKvwVrHaheKstA+yBXLwtcNRn2lBr0OhGlWVTBfV1NvNkkkKek3wG\\\\\\\/ErKO4nJ7MdrDl+WyzKaWQz9lh4xXxkBwjMOWxaRC6R7aS3C0yy2OjoNQ+OE0h0MaR66kFKHwKGzvxXazQweEMT5QU31XGkO+vDbP8hTLoOwLdE4zvVTy48UK6Nrx8iOsjf4\\\\\\\/esJbglzGjxzkPCIsXZ0K\\\\\\\/9GCnZJ0IMguJz76Ou5W2E5Yz\\\\\\\/nw2zXeY4zi2+keenBOVEQcKHpedvSopxJ2NqKHa6UN2cfo6q+mPt3U7g\\\\\\\/xGF34ZqwRwb9\\\\\\\/nYI3Xv3V\\\\\\\/6wMrdv+xpSUdAzZEzA2GUPHVsj0dikg52rMosIk00A8iNs6hJgwTNgPf\\\\\\\/ef3IQmjiEd6iumKKms2is6QKEx7IU2WARWyoRc8CDdM9YB3jMO0pCCfvpcDTIJs2u6l70CtEhWiyw2cUSdwiRS9LHa1P6GXHXHQR417i9s\\\\\\\/1sP4lom8am8eJAu3NufC+7f+hWRsGe2O\\\\\\\/x2CoWI+qRrMTyZu9VneJqrWVFk4WsM1cyX5GogoAhkKpDq1fKlpfFP\\\\\\\/TmLZ72uPEAjcp8WiJgoj7E4jAYRvFH1bymrhzhKZxl14LCOTXSC2bd4RAnrpwR\\\\\\\/j+EWV8x2ddxqM4ZKIsynuuL25j35G9koMOmcJD12xgK7tEv23PBhLihf+NmeUrauloVs9EDZxAc9lb4aJK70vZQOpgsVanEoExJ0a0pACgjXKUNZxb5Hv6ZM73XQwfFhsP\\\\\\\/uBE38+Nq0ZypLU8AFbbHRdby7Wxz6iiuBl0EGG5xlYL8PAtM4tgDjk7BghpF17FmCFPIUfQcODYdjjYv7sawVAVnJSMg2GHb7+K0rNmCsqsGOY0tPJCyahN9nbm\\\\\\\/RZkkexMrqmx36KuMXl00BtU0gtTcPCYTYr60yROUGRNlYh53EzG30PI7sLPWtDzGfTcmgtGPc8kvH76ITePAa+eMFixMa0nYxdpFasb7We9\\\\\\\/kC8LDOfTlOF9GlV+f04vVbX72dnT7vm3XTZY5wFAMIgULZ\\\\\\\/lpVFpKBqq9kKWcO33eBAFONR+iisadDhHdAv3ohLTdJMOQWnRstNn0HzeFyrrCkAAwsPWK1FN+Cy8c+dKgd0ZIKHnpIC9+MEmxJjhO6IiAG78kuSPJeLKs87\\\\\\\/muZjk9AWVFbPc5ZtJzokpKZtpAHmer+hTpUXzqNG8vmX0U28\\\\\\\/rVZqL5VxvfOeD2mkfe3fj1FlS5uPsbBQNEDm5tObF9lmln4XqGzFd\\\\\\\/Qf09Srsnza8rt16nx1\\\\\\\/bBtwmkOuy8+ZH1i2GntEvTeEjTmSV3X8eWCGlrw134Xv2JI\\\\\\\/\\\\\\\/WQl7ZE96eyJZ7FeGkdIX\\\\\\\/BxOlXU20V4dQ2TTNF2ZbRCepdwzFgJMgLF0cwnVUb7dFlXEVJUxRjMjhRDJPrXkCV5+FvbqeNoT5BG\\\\\\\/FBGxz3GQSI3mn+H1xiAn3F2KOOXsB7yxY1I5aWllHMjtWZRt01FxsV+840QJtmIuNFJEcud+k54Uihp8aNyEUIsFrMf9TmTVMifJrdyL6T\\\\\\\/j6GSKXPAgATOb6VAOvUU22emC9DD8M6ctNj\\\\\\\/mPV3r5NQOJLPYbd32tHM14CnL\\\\\\\/4HmgnsK4ZXcZOzrN0Ulwjw=\\\u0022,\\\u0022iv\\\u0022:\\\u0022s96qlBYuT+GAR+knTj2V7w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002246c37c6e421fe1bfec1950a4ce787b2f4cb02c0b94ddeb5a69166b7e98115322\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-486778 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3359" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9gzbM5oElvzDcwQPY0QfkCXjUdzVmlivnqOyllL0LcY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3359" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0TJCShy0QDkEJst6YgG2nhrEieDGV8-uVr6pHk62Qys" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/10/3349" title="KinFin: Software for Taxon-Aware Analysis of Clustered Protein Sequences" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/10/3379" title="Genome-Scale Genetic Interactions and Cell Imaging Confirm Cytokinesis as Deleterious to Transient Topoisomerase II Deficiency in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486778" id="node486778" data-pisa="ggg;7/10/3359" data-pisa-master="ggg;g3.117.300131" data-apath="/ggg/7/10/3359.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">CONE: Community Oriented Network Estimation Is a Versatile Framework for Inferring Population Structure in Large-Scale Sequencing Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9074-7420" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Markku O. Kuismin</span>, <span class="highwire-citation-author" data-delta="1">Jon Ahlinder</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-2808-2768" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mikko J. Sillanpӓӓ</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3359-3377;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300131 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486778" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Markku O. Kuismin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Mathematical Sciences, University of Oulu, FI-90014, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Markku%2BO.%2BKuismin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuismin%20MO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarkku%2BO.%2BKuismin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9074-7420" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Markku O. Kuismin</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jon Ahlinder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Swedish Defense Research Agency, SE-906 21 Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jon%2BAhlinder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ahlinder%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJon%2BAhlinder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mikko J. Sillanpӓӓ </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Mathematical Sciences, University of Oulu, FI-90014, Finland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biocenter Oulu, FI-90014, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mikko%2BJ.%2BSillanp%D3%93%D3%93%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sillanp%D3%93%D3%93%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMikko%2BJ.%2BSillanp%25D3%2593%25D3%2593%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2808-2768" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mikko J. Sillanpӓӓ</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mikko.sillanpaa@oulu.fi" class="" data-icon-position="" data-hide-link-title="0">mikko.sillanpaa@oulu.fi</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/10/3359" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:486778" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:486778/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3359.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:486778" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:486778/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3359.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:486778" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:486778/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3359.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:486778" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:486778/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/10/3359.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:486778" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:486778/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Estimation of genetic population structure based on molecular markers is a common task in population genetics and ecology. We apply a generalized linear model with LASSO regularization to infer relationships between individuals and populations from molecular marker data. Specifically, we apply a neighborhood selection algorithm to infer population genetic structure and gene flow between populations. The resulting relationships are used to construct an individual-level population graph. Different network substructures known as <em>communities</em> are then dissociated from each other using a community detection algorithm. Inference of population structure using networks combines the good properties of: (i) network theory (broad collection of tools, including aesthetically pleasing visualization), (ii) principal component analysis (dimension reduction together with simple visual inspection), and (iii) model-based methods (<em>e.g.</em>, ancestry coefficient estimates). We have named our process CONE (for community oriented network estimation). CONE has fewer restrictions than conventional assignment methods in that properties such as the number of subpopulations need not be fixed before the analysis and the sample may include close relatives or involve uneven sampling. Applying CONE on simulated data sets resulted in more accurate estimates of the true number of subpopulations than model-based methods, and provided comparable ancestry coefficient estimates. Inference of empirical data sets of teosinte single nucleotide polymorphism, bacterial disease outbreak, and the human genome diversity panel illustrate that population structures estimated with CONE are consistent with the earlier findings</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/community-detection" class="hw-term hw-article-keyword hw-article-keyword-community-detection" rel="nofollow">community detection</a></li><li class="kwd"><a href="/keyword/graphical-models" class="hw-term hw-article-keyword hw-article-keyword-graphical-models" rel="nofollow">graphical models</a></li><li class="kwd"><a href="/keyword/neighborhood-selection" class="hw-term hw-article-keyword hw-article-keyword-neighborhood-selection" rel="nofollow">neighborhood selection</a></li><li class="kwd"><a href="/keyword/population-genetic-structure" class="hw-term hw-article-keyword hw-article-keyword-population-genetic-structure" rel="nofollow">population genetic structure</a></li><li class="kwd"><a href="/keyword/population-graph" class="hw-term hw-article-keyword hw-article-keyword-population-graph" rel="nofollow">population graph</a></li></ul><p id="p-3">Network thinking has opened up new avenues for biological research and has recently gained popularity in fields such as ecology, epidemiology, and population genetics (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Proulx <em>et al.</em> 2005</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dyer <em>et al.</em> 2010</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jombart <em>et al.</em> 2011</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dyer 2014</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Greenbaum <em>et al.</em> 2016</a>). For example, it has improved our ability to understand and disentangle the demographic and evolutionary processes that have shaped the genetic architecture of natural populations and maintained their genetic variation, which has important implications for conservation genetics, medicine, and epidemiological investigations. Here, we show that the concepts of network theory can help to infer interconnections within and between subpopulations, and to better visualize the associated demographic and evolutionary processes.</p><p id="p-4">Recent breakthroughs in high-throughput sequencing technologies have paved the way toward large-scale sequencing of nonmodel organisms. In human genetics, efforts are underway to screen the worldwide genetic variation by sequencing thousands of individuals to generate millions of sequence variants (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abecasis <em>et al.</em> 2012</a>). Today, the scope for drawing new conclusions from studies in population genetics is no longer limited by the data-generating process but by the statistical methods available to analyze the vast amounts of data that are generated. In particular, there is a need for new statistical models for handling so-called “large <em>p</em>, small <em>n</em>” cases, where <em>p</em> is the number of markers (parameters) and <em>n</em> is the number of individuals sequenced in the population of interest. One promising technique for this purpose is graph modeling, which has several qualities that are advantageous in the analysis of high-dimensional data (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Meinshausen and Bühlmann 2006</a>).</p><p id="p-5">Traditionally, distance-based and model-based techniques have been used to explore latent genetic structure at the population level and infer genetic ancestry. Distance-based methods generally rely on multivariate techniques and are also known as model-free or algorithmic methods (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wollstein and Lao 2015</a>). Perhaps the most popular model-free technique is principal component analysis (PCA) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Price <em>et al.</em> 2006</a>), which is implemented in software packages such as EIGENSOFT, adegenet, and DAPC (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Price <em>et al.</em> 2006</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Jombart <em>et al.</em> 2010</a>). An appealing property of these methods is that they require no assumptions about the underlying population genetic model, and the required computational time is manageable even when analyzing high-dimensional data sets. However, they have some notable drawbacks. First, one must choose a distance metric from a set of many candidates, which determines how the weighting of common and rare alleles in the populations of interest will be set. Second, the lack of a probabilistic framework necessitates the partitioning of individuals via <em>ad hoc</em> visualization methods, although alternative solutions do exist (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Jombart <em>et al.</em> 2010</a>).</p><p id="p-6">Model-based population assignment methods have been used extensively as alternatives to distance-based methods (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Pritchard <em>et al.</em> 2000</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Tang <em>et al.</em> 2005</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alexander <em>et al.</em> 2009</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Frichot <em>et al.</em> 2014</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Raj <em>et al.</em> 2014</a>). The objective of population assignment methods is to use genetic information (molecular markers measured along individual DNA sequences) to assign individuals to subpopulations and provide estimates for the ancestry coefficients. The multinomial-Dirichlet generative model is widely used in popular implementations of this approach such as STRUCTURE (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Pritchard <em>et al.</em> 2000</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Falush <em>et al.</em> 2003</a>), TESS (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chen <em>et al.</em> 2007</a>), and BAPS (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Corander <em>et al.</em> 2003</a>). Determining the number of subpopulations is challenging when using these approaches, as is handling large-scale data sets containing thousands of markers. An attempt to overcome these difficulties by using maximum <em>a posteriori</em> estimation has recently been published (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Raj <em>et al.</em> 2014</a>). Here we approach the same problem from a different angle, by estimating the network topology, which reflects the relatedness structure between individuals: We refer to the output (<em>i.e.</em>, the construction of a network from individuals) as an individual-level population graph, in contrast to previously reported subpopulation-level works (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dyer and Nason 2004</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Dyer <em>et al.</em> 2010</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Dyer 2014</a>).</p><p id="p-7">An individual-level population graph model in which connections between population members are evaluated based on genetic similarities was recently presented by <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Greenbaum <em>et al.</em> (2016)</a>. This approach, like the earlier work of <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Dyer and Nason (2004)</a> and <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Dyer <em>et al.</em> (2010)</a>, used Gaussian models, which might give rise to spurious relationships (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cushman and Landguth 2010</a>). An alternative method using likelihood theory of generalized linear models (GLMs) that can better account for the discrete nature of genotype data and exploit all of the samples in the data set would thus be desirable. Moreover, <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Greenbaum <em>et al.</em> (2016)</a> truncated genetic relationships at a preselected threshold to make their graph sparser and more realistic. Here, regularization techniques such as the LASSO could be appealing alternatives to this approach if paired with a stability selection method (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Meinshausen and Bühlmann 2010</a>) to introduce sparseness and find an optimal regularization of the graph in a data-dependent way.</p><p id="p-8">Thus, a novel graph-construction method is proposed that has some important advantages over graph methods which have previously been applied to population genetic data:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-9">The population structure problem is divided into smaller components via a neighborhood selection method (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Meinshausen and Bühlmann 2006</a>), which enables the analysis of high-dimensional genetic data.</p></li><li id="list-item-2"><p id="p-10">The genetic relationship between individuals is exploited to directly determine their connectedness by means of a GLM framework in which the discreteness of multi-locus genotype data are acknowledged.</p></li><li id="list-item-3"><p id="p-11">A stability-based subsampling and network modularity-based methods are used to choose the regularization parameter in a data-dependent manner.</p></li><li id="list-item-4"><p id="p-12">The number of latent populations does not need to be fixed in the analysis, and by using a suitable community detection algorithm we can better recognize <em>communities</em> (<em>i.e.</em>, clusters) in the inferred network.</p></li><li id="list-item-5"><p id="p-13">Related individuals can be included in the analysis as the method has a natural way of illustrating the relatedness structure.</p></li></ol><p id="p-14">We have named this multiphase procedure CONE, or “community oriented network estimation.” The CONE process can be divided into five steps: (i) identification of an appropriate model (<em>e.g.</em>, LASSO) for the neighborhood selection scheme; (ii) selection of an appropriate tuning parameter for LASSO with, <em>e.g.</em>, stability-based subsampling; (iii) analysis of the full data set using the selected neighborhood selection procedure and the tuning parameter determined in the preceding step; (iv) division of individuals into different communities using a suitable community detection algorithm; and (v) estimation of ancestry coefficients.</p><p id="p-15">We examine an undirected graph linked with the exponential family of distributions of random variables. We restrict our focus to genotype data for molecular markers associated with the multinomial distribution, and draw a parallel between the estimated graphical model and the underlying network structure. It is important to note here that our parameterization of multinomial distribution differs clearly from that of assignment methods: we use individual-level genotype probabilities instead of population-level frequency parameters. We also note that while there is a conceptual difference between a <em>graph</em> and a <em>network</em>, it is hard to make a practical distinction between the two concepts based on their usage in the statistical and genetically oriented literature (see <em>e.g.</em>, <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Dyer and Nason 2004</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Meinshausen and Bühlmann 2006</a>, <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">2010</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Friedman <em>et al.</em> 2008</a>; <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Dyer 2014</a>; <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Greenbaum <em>et al.</em> 2016</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Li and Jackson 2015</a>). We therefore use the terms network and graph interchangeably.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Simulated data</h3><p id="p-16">We simulated independent microsatellite data sets (two-digit coding) using the software EASYPOP (version 2.0.1) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Balloux 2001</a>). We examined three different scenarios with unequal sample sizes: (i) an island model (scenario 1), (ii) a hierarchical island model with two subpopulations equally grouped into two archipelagos (scenario 2), and (iii) a hierarchical stepping-stone model with two subpopulations separated by a central contact zone (scenario 3).</p><p id="p-17">For each model, 50 data sets were simulated with 10 populations, and (i) a different number of individuals per each population (160 individuals in the first and second population, 80 in the third and fourth, 40 in the fifth and sixth, 20 in the seventh and eighth, and 10 individuals in the ninth and tenth population), and (ii) an even number of individuals (62) for each population. Here, we followed one of the sampling scenarios of <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Puechmaille (2016)</a>. All parameters in EASYPOP were set according to the works of <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Evanno <em>et al.</em> (2005)</a> and <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Puechmaille (2016)</a>: individuals were diploid, with two sexes with an equal number of males and females in each population, and a random mating system was used. Migration rates within sets were set to 0.01 and 0.001 between sets, equal for both sexes. The mutation model was set as a mixed single-step mutation model with an equal probability (0.3) to mutate to any allelic state at a rate of 0.001. Data were generated for a set of 2000 individuals with free recombination between loci, each with two possible allelic states.</p><p id="p-18">The alleles were merged into numerical values 0 (allele pair 0101), 1 (allele pair 0102 or 0201), and 2 (allele pair 0202). Simulations were run for 10,000 generations to ensure equilibrium between drift, migration, and mutation. EASYPOP syntax files, used in the simulations, and the R scripts used for the simulation analysis are provided in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300131/-/DC1/FileS1.zip">File S1</a>.</p></div><div id="sec-3" class="subsection"><h3>Simulation of admixed populations</h3><p id="p-19">To simulate SNP data from a binomial model, we chose the HapMap phase 3 CEU, CHB, and YRI samples as our ancestral populations. We chose SNPs spaced at least 200 kb apart and the minor allele frequencies calculated over the three populations were used as the true values of the allele-frequency matrix. Then we chose an arbitrary subsample of loci so that the final allele-frequency matrix contained 2500 columns. Finally, we simulated genotypes following the binomial model presented for example in <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Alexander <em>et al.</em> (2009)</a> using six different models for the true values of the <em>Q</em> matrix of admixture coefficients. Each simulated data set included 600 unrelated individuals genotyped at 2500 loci. We generated the <em>Q</em> matrix independently from the Dirichlet distributions, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-1.gif"/></noscript></span></span> with different levels of admixture controlled with parameters <em>β</em>, <em>γ</em>, and <em>δ</em>.</p></div><div id="sec-4" class="subsection"><h3>Teosinte data</h3><p id="p-20">We initially evaluated the performance of CONE by applying it to a moderately sized SNP genotype data set gathered from 21 different populations of teosinte, the wild ancestor of maize, from various locations in Mexico (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Pyhäjärvi <em>et al.</em> 2013</a>). The data contains samples from two different subspecies, <em>Zea mays</em> subsp. <em>parviglumis</em> (11 populations) and <em>Z. mays</em> subsp. <em>mexicana</em> (10 populations).</p><p id="p-21">Here, we used the same set of markers as <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Pyhäjärvi <em>et al.</em> (2013)</a>. We recoded the data with numerical values of 0, 1, or 2, corresponding to the homozygous genotype <em>AA</em>, the heterozygous genotype <em>AB</em>, and the other homozygous genotype <em>BB</em>, respectively. There were no monomorphic markers and SNPs with &gt;10% missing data had previously been removed from the data set. Nevertheless, 16,891 markers had missing sample genotypes: We imputed these missing genotypes (once) with the most common genotype of the corresponding marker (the locus mode). There were also two individuals that did not belong to the wild maize populations and were identified as maize hybrids within the <em>mexicana</em> samples. These hybrids were not identified in the public data set so we did not remove them from our data; we expected that the corresponding individuals would have insignificant effects on our population structure analysis. Our final data set thus contained 250 samples (columns <em>N</em>) and 36,719 markers (rows <em>L</em>) with no missing genotypes. We also performed some fine-scale analysis over the <em>parviglumis</em> and <em>mexicana</em> subspecies. For these data subsets, we again used the same 36,719 SNPs as <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Pyhäjärvi <em>et al.</em> (2013)</a>, but due to the smaller sample sizes, we removed the monomorphic markers. Imputation was performed as described above. The final data set for <em>parviglumis</em> populations contained 130 samples and 35,727 SNPs, while the <em>mexicana</em> data set contained 120 samples and 35,500 SNPs. The teosinte SNP data set together with some supplemental files are available at <a href="http://datadryad.org/resource/doi:10.5061/dryad.8m648">http://datadryad.org/resource/doi:10.5061/dryad.8m648</a>.</p></div><div id="sec-5" class="subsection"><h3>Escherichia coli data</h3><p id="p-22"><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Salipante <em>et al.</em> (2015)</a> analyzed 312 blood- and urine-derived isolates of extraintestinal pathogenic <em>E. coli</em> (ExPEC), a common agent of sepsis and urinary tract infections. The isolates were obtained during the course of routine clinical care at a single institution. Whole genome analysis was performed to infer phylogenetic relationships and detect partitions in the population. A subset of 120 genomes was selected for analysis here. Draft genomes were aligned against the reference <em>E. coli</em> K12MG1665 genome (GenBank ID: 556503834) using ProgressiveMauve (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Darling <em>et al.</em> 2010</a>) with default settings. A phylogenetic tree was obtained using the neighbor-joining algorithm in MEGA6 (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Tamura <em>et al.</em> 2013</a>).</p><p id="p-23">The data set contained 121 strains, including the reference K12MG1665 and 348,859 SNP markers. Before conducting our analysis, we removed SNPs with &gt;5% missing alleles. We then imputed the remaining missing genotypes (once) with the most common variant at the corresponding site. The final data set contained 333,549 SNPs with no monomorphic markers.</p><p id="p-24">We recoded the data with numerical values of 1, 2, 3, and 4 according to the allele (<em>i.e.</em>, nucleotide) present at each locus. Because the data set pertains to a haploid organism, we note that in contrast to Equation 1 there are now four vectors to consider <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-2.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-3.gif"/></noscript></span></span> corresponding to each individual locus instead of allele pairs. Using numerical values 1, 2, 3, and 4 does not violate our neighborhood selection framework because CONE can be generalized to an arbitrary number of genotypes (classes). CONE can thus be implemented exactly as described later in the article: an estimated edge exists between nodes (strains) <em>i</em> and <em>j</em>, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-4.gif"/></noscript></span></span> if and only if <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-5.gif"/></noscript></span></span> is nonzero for some <em>g</em>.</p></div><div id="sec-6" class="subsection"><h3>HGDP data</h3><p id="p-25">We used a SNP data set, named the Stanford HGDP SNP genotyping data set (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Rosenberg <em>et al.</em> 2002</a>). The data set contains 1043 individuals from 51 different populations and 660,918 SNP markers. We recoded the data with numerical values of 0, 1, or 2 according to the number of reference alleles C or G in the genotype. As was also done by <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Raj <em>et al.</em> (2014)</a>, we removed SNPs that were monomorphic or had &gt;5% of the genotypes missing. We then imputed the missing genotypes (once) with the most common genotype of the corresponding marker, as was done with the teosinte data. The final data set included 659,421 markers and 1043 individuals. In contrast to the approach adopted by <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li <em>et al.</em> (2008)</a> and <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Raj <em>et al.</em> (2014)</a>, we did not modify the data prior to analysis by performing a Hardy–Weinberg equilibrium test or removing related individuals; this did not adversely affect the CONE procedure and the results obtained were comparable to those reported previously (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Li <em>et al.</em> 2008</a>; <a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Raj <em>et al.</em> 2014</a>). The data are freely available at <a href="http://www.hagsc.org/hgdp/files.html">http://www.hagsc.org/hgdp/files.html</a> and are also deposited at Fondation Jean Dausset-CEPH (<a href="http://www.cephb.fr/en/hgdp_panel.php#basedonnees">http://www.cephb.fr/en/hgdp_panel.php#basedonnees</a>).</p></div><div id="sec-7" class="subsection"><h3>Graphical model</h3><p id="p-26">Let <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-6.gif"/></noscript></span></span> be a random vector of <em>N</em> individuals. The undirected graph can be used to visualize the relationships between these individuals. We describe such a graph as a population graph because it can be used to illustrate the individual-level population structure in a similar way to software such as STRUCTURE, without requiring any prior information about the population structure. Let <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-7.gif"/></noscript></span></span> be a graphical model, where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-8.gif"/></noscript></span></span> is a set of nodes and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-9.gif"/></noscript></span></span> is a set of edges <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-10.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-11.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-12.gif"/></noscript></span></span> in <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-13.gif"/></noscript></span></span> Each individual corresponds to one node in the set <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-14.gif"/></noscript></span></span> and a pair <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-15.gif"/></noscript></span></span> is contained in the edge set <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-16.gif"/></noscript></span></span> if and only if individual <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-17.gif"/></noscript></span></span> is conditionally dependent on individual <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-18.gif"/></noscript></span></span> given all remaining individuals. More informally, there is a probabilistic relationship between individuals <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-19.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-20.gif"/></noscript></span></span> when the effect of all remaining variables is eliminated; if this condition is satisfied, we say that there is an edge between <em>i</em> and <em>j</em> or vice versa.</p><p id="p-27">With large data sets containing thousands of individuals and enormous numbers of SNP markers, solving the graph-selection problem quickly becomes unfeasible. However, the problem can be made simpler by adopting a regression approach whereby one estimates the neighborhood of the node <em>i</em> by performing a regression of the variable <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-21.gif"/></noscript></span></span> on the rest of the variables. The main motivation for using regression in network analysis is to divide the graph-selection problem into much smaller so-called “neighborhood selection problems” (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Meinshausen and Bühlmann 2006</a>). Analyzing each node one by one makes it possible to analyze large data sets by estimating <em>N</em> nodes, each of size <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-22.gif"/></noscript></span></span> Dividing the graph construction into smaller subproblems also makes it possible to use parallel computing to increase time efficiency. Graph estimates obtained using the strategy outlined above can be described as dependency networks (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Heckerman <em>et al.</em> 2000</a>).</p><p id="p-28">In this discussion we ignore some of the more sophisticated issues relating to neighborhood selection; we refer the reader to publications such as that of <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Meinshausen and Bühlmann (2006)</a> for information on graphical models associated with the Gaussian distribution. We approach the neighborhood selection as a regularized linear regression problem (<em>i.e.</em>, a problem of penalized regression) and use an <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-23.gif"/></noscript></span></span> symmetric adjacency matrix to depict the relationships between individuals (nodes). The elements of the adjacency matrix are either zero or one. Each zero entry of the matrix indicates absence of an edge between the nodes represented on the corresponding column and row. For convenience, we will refer to this kind of neighborhood selection procedure based on <em>L</em><sub>1</sub>-penalized (a so-called LASSO penalty) regression as Meinshausen and Bühlmann-style (MB-style) neighborhood selection.</p><p id="p-29">In the case of undirected graphs, the relationships between nodes are visualized by drawing undirected lines (edges) between related nodes. With Gaussian data, conditional independence is related to partial correlations, and is consistent with the examination of zero entries in the Gaussian inverse covariance matrix (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Edwards 2000</a>). Here we are more interested in neighborhood selection, which is a subproblem of covariance selection for Gaussian data.</p></div><div id="sec-8" class="subsection"><h3>The neighborhood selection scheme</h3><p id="p-30">For diploid organisms, we assume that each locus in the population has <em>P</em> possible genotypes with nonzero frequencies. In general, we can assume that <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-24.gif"/></noscript></span></span> and the categories have no specific order. For every sample (individual) there are now <em>P</em> different random variables corresponding to the different genotypes, and the molecular markers for each sample are divided into these categories. For one locus of a diploid organism, there would be three categories corresponding to three different genotypes <em>AA</em>, <em>AB</em>, and <em>BB</em>, meaning that <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-25.gif"/></noscript></span></span> For the sake of simplicity, we will describe our method for individuals representing diploid organisms. Thus, each individual determines multiple different random variables and their joint probability distribution through a multinomial distribution <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-26.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-27.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-28.gif"/></noscript></span></span> correspond to the genotype probabilities of the <em>L</em> measured markers of individual <em>i</em>, <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-29.gif"/></noscript></span></span> Again, <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-30.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-31.gif"/></noscript></span></span> have nothing to do with population frequencies. However, it is natural to expect that populations have shared genotype frequencies and, thus, individuals from the same population group have similar genotype probabilities. We also assume that the random variables (genotypes) of two related population groups affect the parameters (genotype probabilities) of both groups; this corresponds to gene flow, with no specific direction, between population groups and is expressed by the presence of undirected edges linking different individuals in the graphical model.</p><p id="p-31">In the original work of <a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">Meinshausen and Bühlmann (2006)</a>, only the properties arising from Gaussian data were examined. We apply MB-style neighborhood selection to SNP (genotype) data measured from either haploid or diploid organisms; different loci are seen as independent, replicated observations from the same model (linkage equilibrium). The LASSO method is applied in the usual way for Gaussian distributed variables, and coefficient estimates are obtained by minimization of the regularized mean squared error (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Tibshirani 1996</a>). With GLMs, the negative log-likelihood is minimized instead.</p><p id="p-32">We consider a multinomial model for the categorical responses (see, <em>e.g.</em>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hastie <em>et al.</em> 2009</a>, pp. 120–122). The data consist of a <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-32.gif"/></noscript></span></span> matrix <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-33.gif"/></noscript></span></span> and each vector <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-34.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-35.gif"/></noscript></span></span> can contain at most three genotypes per locus. We focus on one individual <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-36.gif"/></noscript></span></span> and denote the remaining marker data by <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-37.gif"/></noscript></span></span> The procedures described in the remainder of this section are then applied to each individual, one by one. Stated formally, our objective is to answer the question: “How does the distribution of genotypes of individuals <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-38.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-39.gif"/></noscript></span></span> explain the genotype of the individual <em>X<sub>i</sub></em>?”</p><p id="p-33">The basis of our analysis is the multi-logit model (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Friedman <em>et al.</em> 2010</a>)<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-40.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>in which <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-41.gif"/></noscript></span></span> is a vector of dimension <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-42.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-43.gif"/></noscript></span></span> For simplicity, let <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-44.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-45.gif"/></noscript></span></span> corresponds to the <em>g</em>th genotype for the <em>i</em>th individual at the locus <em>l</em>, <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-46.gif"/></noscript></span></span></p><p id="p-34">The negative log-likelihood for the individual <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-47.gif"/></noscript></span></span> is<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-48.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>We minimize the LASSO-style penalized negative log-likelihood<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-49.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-50.gif"/></noscript></span></span> is the <em>L</em><sub>1</sub>-norm of a vector <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-51.gif"/></noscript></span></span> and <em>λ</em> is the so-called tuning parameter, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-52.gif"/></noscript></span></span> It is well known (see for example <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Tibshirani 1996</a>) that in such cases, <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-53.gif"/></noscript></span></span> regularization will yield vectors <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-54.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-55.gif"/></noscript></span></span> nonzero elements depending on the value of <em>λ</em>.</p><p id="p-35">When dealing with multinomial regression, we obtain three sparse vectors whose elements differ from each other as estimates of <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-56.gif"/></noscript></span></span> for each genotype category <em>g</em>. If performing a standard LASSO regression for Gaussian data, we would construct a graph, as described by <a id="xref-ref-34-7" class="xref-bibr" href="#ref-34">Meinshausen and Bühlmann (2006)</a>, by noting that there is an undirected edge between nodes <em>i</em> and <em>j</em>, <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-57.gif"/></noscript></span></span> if and only if <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-58.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-59.gif"/></noscript></span></span> are conditionally dependent given all other individuals. Thus, in the LASSO setting for Gaussian data, there is no edge between <em>i</em> and <em>j</em> if <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-60.gif"/></noscript></span></span></p><p id="p-36">With our multinomial setting we must use a different guideline to determine the dependency between individuals. We propose an intuitive rule to determine whether or not there is an indication of dependency between different individuals: set an estimated edge between <em>i</em> and <em>j</em> if and only if <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-61.gif"/></noscript></span></span> is nonzero for some <em>g</em>. This corresponds to a value of one in the <em>j</em>th row of the <em>i</em>th column of the adjacency matrix.</p><p id="p-37">Because the probability <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-62.gif"/></noscript></span></span> also produces a dependency structure between individuals <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-63.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-64.gif"/></noscript></span></span> we use the so-called “and” rule to ensure that the adjacency matrix is symmetric: there is an edge between the nodes <em>i</em> and <em>j</em> if and only if “<em>X<sub>i</sub></em> depends on <em>X<sub>j</sub></em>” <em>and</em> “<em>X<sub>j</sub></em> depends on <em>X<sub>i</sub></em>” given all other individuals. If the rule holds, the adjacency matrix has one as an entry on the <em>i</em>th row and <em>j</em>th column, and vice versa. Otherwise, the entry is zero. Alternatively, one could use a so-called “or” rule that states that there is an edge between the nodes <em>i</em> and <em>j</em> if and only if “<em>X<sub>i</sub></em> depends on <em>X<sub>j</sub></em>” <em>or</em> “<em>X<sub>j</sub></em> depends on <em>X<sub>i</sub></em>” given all other individuals.</p><p id="p-38">The multinomial model cannot be used for neighborhood selection if one of the (three) possible genotypes has a very low or zero frequency over all loci of the samples. However, the probability of this happening with large marker data sets is practically zero.</p><p id="p-39">The advantage of the penalized multinomial model is that none of the individuals are forced to be part of some population, unlike in analyses performed with model-based methods. The neighborhood selection method can be used to identify these individuals and they are represented by free-floating nodes with no edges connecting them to other nodes. No predetermination of the number of subpopulations is needed because we assume that the method will naturally divide the individuals into different populations. By itself, constructing a network estimate is not sufficient to produce easily interpreted results; we still need to separate each cluster as a separate connected component, <em>i.e.</em>, a cluster of nodes with few or no edges linking them to other clusters. Dividing nodes (individuals) into clusters is discussed in the <em>Detecting communities</em> subsection.</p><p id="p-40">One can use either frequentist or Bayesian methods in the analysis. We used the R package glmnet (version 2.0-2) to fit the regularized GLMs. The glmnet package exploits the cyclic coordinate descent algorithm, which has proven to be a fast method for solving LASSO-type problems such as Equation 3, and can work on very large data sets. For more details about the glmnet package, see <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Friedman <em>et al.</em> (2010)</a>. Because R stores all objects in memory, analyzing large data sets becomes practically impossible even though the glmnet package can handle very large vectors and make good use of sparse data. We therefore used R packages such as bigmemory (version 4.5.10) and ff (version 2.2–13) to handle multi-gigabyte data sets and variables in R. Packages bigmemory and ff can store objects to the hard drive while preserving R’s strong capacity for efficient statistical analysis.</p></div><div id="sec-9" class="subsection"><h3>Choosing the tuning parameter</h3><p id="p-41">The neighborhood selection scheme is comparatively easy to use and implement as nice statistical properties are obtained by adjusting a single penalty parameter <em>λ</em> (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Raskutti <em>et al.</em> 2009</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Liu <em>et al.</em> 2010</a>). Various common model selection methods could be used to identify the optimal value of the tuning parameter, including k-fold cross-validation, Akaike’s information criterion, or the Bayesian information criterion (see, <em>e.g.</em>, <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Hastie <em>et al.</em> 2009</a>, pp. 230–245). In practice it is well known that cross-validation and information criteria are often impractical in graph construction because their application usually produces rather dense graphs with many false-positive edges (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wasserman and Roeder 2009</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Liu <em>et al.</em> 2010</a>).</p><p id="p-42">In this article we represent two methods that can be used to choose a proper value for the tuning parameter, one using a stability approach and the other, modularity approach, which is similar to the so-called “elbow” method, commonly applied in hierarchical clustering (see <em>e.g.</em>, <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Zhang and Horvath 2005</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Zumel and Mount 2014</a>, pp. 187–190).</p><div id="sec-10" class="subsection"><h4>Finding the optimal tuning parameter—The stability approach:</h4><p id="p-43"><a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Liu <em>et al.</em> (2010)</a> used a stability approach to regularization selection (StARS) to choose the most stable graph. The StARS is a general way of obtaining an undirected graph-selection criterion when the graph estimation is based on the penalized likelihood minimization problem as in Equation 3. StARS guarantees that the graph estimate will be consistent, provided that some moderate conditions are met (see theorem 4.2 in <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Liu <em>et al.</em> 2010</a>). Our regularization selection procedure is based strictly on the methods described in the work of <a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Liu <em>et al.</em> (2010)</a>.</p><p id="p-44">Suppose we want to choose the “best” value for the tuning parameter <em>λ</em> in Equation 3 over a sequence of different positive values. With StARS, we use subsampling to measure the uncertainty regarding the presence of edges between the nodes in the graph estimate <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-65.gif"/></noscript></span></span> for each fixed value of <em>λ</em>. Then we calculate the total instability of the estimated graph based on the subsamples. Finally, we choose an estimate <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-66.gif"/></noscript></span></span> for <em>λ</em> that gives the optimal stability among the sparse graphs based on a special instability measure <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-67.gif"/></noscript></span></span> associated with the corresponding graph estimate. Assuming the sequence of the instability measure is monotonic, we can choose <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-68.gif"/></noscript></span></span> with a threshold parameter <em>β</em>. We use <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-69.gif"/></noscript></span></span> as a default value for our StARS procedure, as proposed in <a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Liu <em>et al.</em> (2010)</a>. The selection of <em>λ</em> using the quantity <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-70.gif"/></noscript></span></span> is illustrated for the case of a data set in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The StARS instability measure plotted against the tuning parameter. The red dashed line indicates the value smaller than or equal to the threshold value () on the vertical axis, and the corresponding value of the tuning parameter on the horizontal axis. The simulation model considered here is the island model described in subsection Simulated data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1482786432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;The StARS instability measure plotted against the tuning parameter. The red dashed line indicates the value smaller than or equal to the threshold value (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-71&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) on the vertical axis, and the corresponding value of the tuning parameter on the horizontal axis. The simulation model considered here is the island model described in subsection &lt;em&gt;Simulated data&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3359/F1.medium.gif" width="440" height="432"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/10/3359/F1.medium.gif" width="440" height="432"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486889" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-45" class="first-child">The StARS instability measure plotted against the tuning parameter. The red dashed line indicates the value smaller than or equal to the threshold value (<span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/F1/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/F1/embed/mml-math-71.gif"/></noscript></span></span>) on the vertical axis, and the corresponding value of the tuning parameter on the horizontal axis. The simulation model considered here is the island model described in subsection <em>Simulated data</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">Although the StARS analysis is computationally intensive, we favor it because it is readily interpreted and efficiently reduces the number of false-positive edges. It also provides an additional graph estimate based on the subsamples of size <em>b</em>, which can be used to analyze the connectivity of the estimated graph as described above to determine how the subpopulations are differentiated from each other. We present some guidelines for reducing the computation time needed for graph estimation. In principle, the tuning parameter <em>λ</em> should be varied from zero to a moderately large value. However, the use of small values will slow down the glmnet package despite its ability to time efficiently produce the whole solution path. In practice, the graph will be overly dense if relatively small values of <em>λ</em> are used, so it is reasonable to impose a lower bound above zero on this parameter. Additionally, the number of subsamples should be quite large in principle: because the processing time is directly proportional to the number of subsamples, using many subsamples would greatly increase the time needed to execute the StARS procedure. However, our analysis results suggested that even relatively small numbers of subsamples yielded reasonable results. We use <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-72.gif"/></noscript></span></span> subsamples for our StARS selection procedure, which is also the default value for selecting the most stable tuning parameter for the graphical lasso (glasso) algorithm (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Friedman <em>et al.</em> 2008</a>) in StARS analyses performed with the R package huge (version 1.2.7) (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Zhao <em>et al.</em> 2012</a>). Taking that into account, under our experimental conditions, the cyclic coordinate descent algorithm used in the glmnet package may fail to converge for small values of the tuning parameter <em>λ</em>. This usually occurred when working with small data sets using the StARS procedure. This problem could be circumvented by tailoring the grid for the problematic candidate <em>λ</em> values or generating a new subsample. Problems of this kind were not encountered when working with real-world data sets here. Before we can describe how the elbow method can be used to choose a proper value for the tuning parameter or how CONE can be used for estimation of ancestry coefficients, we have to first explain some concepts in network <em>communities</em>.</p></div><div id="sec-11" class="subsection"><h4>Detecting communities:</h4><p id="p-47">The word “subpopulation” is used to refer to the known grouping of individuals and the words “cluster” or “community” to refer to the grouping of individuals inferred in the data set with the method described below. We do not assume that different subpopulations usually form groups that will be totally separate from all other subpopulations; rather, we assume that only very genetically distant individuals will not be directly joined together. However, it is natural to presume that the connections between more distantly related groups will be much weaker than those between closely related groups; namely, individuals from the same subpopulations will form different <em>communities</em> in the network. The definition of a <em>community</em> or <em>cluster</em> in network theory is quite loose; a community is a group of nodes having a relatively large number of interconnecting edges but looser connections to other groups or nodes in the network (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fortunato 2010</a>).</p><p id="p-48">Although the neighborhood selection procedure generates a network estimate that includes estimates of the strengths of the edges between different nodes, we must use some external algorithm to distinguish communities from one another. In a similar way to <a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Greenbaum <em>et al.</em> (2016)</a>, we use a community detection algorithm for this purpose. Specifically, we used the R packages qgraph (version 1.3.2) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Epskamp <em>et al.</em> 2012</a>) and igraph (version 1.0.1) to draw undirected graphs based on the adjacency matrices, and we used the Fruchterman–Reingold algorithm (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fruchterman and Reingold 1991</a>) and the Walktrap algorithm (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Pons and Latapy 2006</a>) as implemented in these packages for community partitioning.</p><p id="p-49">The basic principle of the Fruchterman–Reingold algorithm is that each node repulses the others, but nodes (individuals) that are connected by edges are drawn closer together: This gives rise to a force-directed layout that is easier to interpret than the original graph, with a minimal number of crossed edges and more or less separate communities. The principle behind the Walktrap algorithm is that nodes belonging to the same community have more edges connecting each node to another. The <em>random walks</em> on a graph tend to get stuck into these “dense” parts (communities) of the graph. Thus one can compute a distance measurement of the structural similarity between nodes and between communities, a <em>modularity</em> measure. Finally, a hierarchical clustering algorithm is used to detect different communities in the graph, and the partition that maximizes the modularity measure is chosen as the layout with different nodes grouped to more or less distinct communities.</p><p id="p-50">Here our work differs from that of <a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Greenbaum <em>et al.</em> (2016)</a>, who used the Girvan–Newman algorithm (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Girvan and Newman 2002</a>) in their main analysis (however, their supplemental materials include a discussion on the use of alternative community detection algorithms for network construction, including the Walktrap algorithm). Importantly, in the case of the additional StARS graph, the Fruchterman–Reingold algorithm can distinguish between nodes to separate communities in a way that facilitates interpretation. While alternative community detection algorithms are available, we have found Walktrap and Fruchterman–Reingold algorithms to give reasonable and consistent results.</p><p id="p-51"><a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Puechmaille (2016)</a> noticed that uneven subsampling affects STRUCTURE’s ability to infer the number of subpopulations from a data set. Specifically, small populations tend to separate into different subpopulations in STRUCTURE analyses. We did not encounter this problem with CONE in our real data analyses, although its community detection algorithms usually give the best results when applied to evenly distributed and sufficiently large communities. Moreover, the comparatively simple community layout produced by CONE analyses makes it possible to identify possible estimation errors by visual inspection.</p><p id="p-52">For more detailed descriptions of community analysis and reviews of the various community detection algorithms, see for example <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Fortunato (2010)</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lancichinetti and Fortunato (2009)</a>, and <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Harenberg <em>et al.</em> (2014)</a>.</p></div><div id="sec-12" class="subsection"><h4><em>Finding the optimal tuning parameter</em>—The network modularity approach:</h4><p id="p-53">In population structure analysis it is a common task to (pre)determine the number of populations in the data set when using model-based methods to derive the ancestry proportions from the data (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Pritchard <em>et al.</em> 2000</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Alexander <em>et al.</em> 2009</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Frichot <em>et al.</em> 2014</a>; <a id="xref-ref-43-6" class="xref-bibr" href="#ref-43">Raj <em>et al.</em> 2014</a>). Although somewhat of an artificial concept, one has to choose the number of populations (<em>K</em>) for the analysis, even if in practice there will be no “true” value of <em>K</em>, as samples from real populations rarely conform to the assumptions of the model.</p><p id="p-54">With CONE, one does not need to predetermine the number of populations. Nevertheless, it is useful to have knowledge about the number of clusters to help understanding of the (visual) appearance in the data. In particular, inferring the number of populations is useful if CONE is used for estimation of ancestry coefficients. In such cases, one can learn <em>K</em> directly from the data using methods common to hierarchical clustering (see, <em>e.g.</em>, <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Zhang and Horvath 2005</a>). For this purpose we use the Walktrap algorithm for community detection, considering that the hierarchical clustering algorithm is a part of the Walktrap algorithm. Similar to hierarchical clustering methods, we derive network estimates with various values of the tuning parameter <em>λ</em>, determine <em>K</em> to be equal to the number of communities found with the Walktrap algorithm, and network modularity equal to that of the given grouping. We note that the modularity value determined by the Walktrap varies in between −1 and 1. A positive modularity value indicates a better separation between communities of the graph. After inferring the number of clusters and network modularity with different values of the tuning parameter, we examine if there is an elbow in the network modularity <em>vs.</em> the number of inferred cluster plots. We observed that the number of inferred communities seem to stabilize similarly to the hierarchical cluster analysis.</p><p id="p-55">We have illustrated the procedure described above in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. The optimal value for the tuning parameter is a trade-off between the network modularity and sparseness. In <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, there is a clear elbow around the 17th and 27th value of <em>λ</em>. Here, the elbow is quite unambiguous but this may not always be the case; in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a> the subsampling of the data causes small instability in the procedure. One could also use a slightly smaller value for <em>λ</em> but “too small” values should be avoided because they will lead to low network modularity and network estimates with several insignificant edges (connections) between nodes (individuals).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The network modularity corresponding to different numbers of inferred communities () with different values of the tuning parameter. (A) Networks computed with subsampling. (B) Unweighted networks inferred with MB approximation. (C) Scaled (A). (D) Scaled (B). The red horizontal dashed line indicates the true number of subpopulations (). Each number in the plot area corresponds to an index number i () for different ascending values for the tuning parameter. The simulation model considered here is the island model described in subsection Simulated data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1482786432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;The network modularity corresponding to different numbers of inferred communities (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-73&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) with different values of the tuning parameter. (A) Networks computed with subsampling. (B) Unweighted networks inferred with MB approximation. (C) Scaled (A). (D) Scaled (B). The red horizontal dashed line indicates the true number of subpopulations (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-74&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). Each number in the plot area corresponds to an index number &lt;em&gt;i&lt;/em&gt; (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-75&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) for different ascending values for the tuning parameter. The simulation model considered here is the island model described in subsection &lt;em&gt;Simulated data&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3359/F2.medium.gif" width="440" height="437"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/10/3359/F2.medium.gif" width="440" height="437"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486853" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-56" class="first-child">The network modularity corresponding to different numbers of inferred communities (<span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/F2/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/F2/embed/mml-math-73.gif"/></noscript></span></span>) with different values of the tuning parameter. (A) Networks computed with subsampling. (B) Unweighted networks inferred with MB approximation. (C) Scaled (A). (D) Scaled (B). The red horizontal dashed line indicates the true number of subpopulations (<span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/F2/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/F2/embed/mml-math-74.gif"/></noscript></span></span>). Each number in the plot area corresponds to an index number <em>i</em> (<span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/F2/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/F2/embed/mml-math-75.gif"/></noscript></span></span>) for different ascending values for the tuning parameter. The simulation model considered here is the island model described in subsection <em>Simulated data</em>.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-13" class="subsection"><h3>Computing ancestry coefficients</h3><p id="p-57">Estimation of ancestry coefficients is an essential part of model-based methods. Considering how model-based methods derive the relationship between ancestry proportions and an individual’s alleles, degree of ancestry can be directly estimated from model parameters. In this section, we use the term “ancestry coefficients” to refer to the estimated ancestry in general. Denote so-called ancestry coefficient matrix as <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-76.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-77.gif"/></noscript></span></span> represents the fraction of individual <em>i</em>’s genome originating from the ancestral population <em>k</em>. The matrix <em>Q</em> has dimension <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-78.gif"/></noscript></span></span> An obvious drawback of visual inspection methods such as PCA and the network analysis is that they do not directly produce an estimate for the ancestry coefficient matrix <em>Q</em>, although there is a way to estimate <em>Q</em> with PCA using the DAPC program (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Jombart <em>et al.</em> 2010</a>).</p><p id="p-58">Considering that elements in <em>Q</em> represent ancestral proportions of individuals in <em>K</em> possible subpopulations, there seems to be a natural connection between ancestral proportions and network estimates; the edges in the network computed with the neighborhood selection algorithm represent the genetic relationship between different individuals in the network. It is consistent to assume that the proportion of edges of a node (representing an individual) between other nodes belonging to a community represents the fraction of genome from the corresponding ancestral population. Thus one can gain an estimate of <em>Q</em> with CONE using the method described below.</p><p id="p-59">Assume that the nodes <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-79.gif"/></noscript></span></span> in the graphical model <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-80.gif"/></noscript></span></span> are partitioned in <em>K</em> distinct communities, <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-81.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-82.gif"/></noscript></span></span> is the symmetric <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-83.gif"/></noscript></span></span> adjacency matrix which corresponds to the estimated network. Element <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-84.gif"/></noscript></span></span> can be either all zeros and ones, corresponding to an unweighted network, or can vary between zero and one, representing the weighted network. Diagonal elements of <em>A</em>, <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-85.gif"/></noscript></span></span> are set to zero. The degree of a node <em>i</em> denoted by <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-86.gif"/></noscript></span></span> can be computed as the column sum of the <em>i</em>th column of <em>A</em>, <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-87.gif"/></noscript></span></span> Clearly <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-88.gif"/></noscript></span></span> for all <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-89.gif"/></noscript></span></span> We determine the estimate of the ancestry proportion of individual <em>i</em> in the ancestral population <em>k</em> as the proportion of the number of edges belonging to the community <em>K</em> and the overall node degree, <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-90.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-91.gif"/></noscript></span></span> is the number of edges of node <em>i</em> which are connected to other nodes belonging to the community <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-92.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-93.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-94.gif"/></noscript></span></span> is an indicator variable that takes the value one if the node <em>j</em> belongs to a community <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-95.gif"/></noscript></span></span> and zero otherwise, and <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-96.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-97.gif"/></noscript></span></span></p><p id="p-60">If <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-98.gif"/></noscript></span></span> takes the value zero, the corresponding ancestry proportion estimate <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-99.gif"/></noscript></span></span> is set to one, meaning that this particular individual is not genetically related to any other individual in the sample and comes from some spurious cluster that has not been removed by the neighborhood selection method. Similarly to the model-based assignment methods, such as STRUCTURE, our approach suffers from a label-switching problem, <em>i.e.</em>, it is not invariant to the permutations in the terms of different communities and can alter with different genotype matrices with the same underlying ancestral populations. Solving the label-switching problem is an open research question but some solutions have been presented (see, <em>e.g.</em>, CLUMPP software; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jakobsson and Rosenberg 2007</a>).</p></div><div id="sec-14" class="subsection"><h3>Genomic relationship matrix</h3><p id="p-61">The genomic relationship matrix, as described by <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">VanRaden (2008)</a>, <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-100.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-101.gif"/></noscript></span></span> is obtained as follows: <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-102.gif"/></noscript></span></span> where <em>Z</em> is the matrix with mean allele effects set to zero. This in turn is obtained by subtracting <em>P</em> from <em>M</em>, where <em>P</em> is a <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-103.gif"/></noscript></span></span> column matrix with column elements <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-104.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-105.gif"/></noscript></span></span> is the frequency of the reference allele at locus <em>l</em>, <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-106.gif"/></noscript></span></span> Matrix <em>Z</em> is commensurate with our data matrix <em>X</em> after recoding to −1, 0, and 1 into matrix <em>M</em>. We define a pair of strongly genetically related individuals <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-107.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-108.gif"/></noscript></span></span> as any pair for which the corresponding element of <em>K</em> is &gt;0.5, that is <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-109.gif"/></noscript></span></span></p></div><div id="sec-15" class="subsection"><h3>Data availability</h3><p id="p-62">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article. EASYPOP software is available from <a href="http://www.unil.ch/dee/en/home/menuinst/softwares--dataset/softwares/easypop.html">http://www.unil.ch/dee/en/home/menuinst/softwares--dataset/softwares/easypop.html</a>. </p><p id="p-63">All SNP genotypes and related information for <em>Z. mays</em> subsp. <em>parviglumis</em>, <em>Z</em>. <em>mays</em> subsp. <em>mexicana</em>, and <em>Tripsacum dactyloides</em> files are available from the DRYAD database (<a href="http://dx.doi.org/10.5061/dryad.8m648">http://dx.doi.org/10.5061/dryad.8m648</a>).</p><p id="p-64"><em>E. coli</em> draft genomes are publically available at NCBI GenBank (<a href="http://www.ncbi.nlm.nih.gov/genbank">http://www.ncbi.nlm.nih.gov/genbank</a>) under accession numbers JSFQ00000000–JSST00000000. The reference strain K12 substrain MG1665 has the GenBank ID 556503834.</p><p id="p-65">All Stanford HGDP SNP genotyping data files are deposited at Fondation Jean Dausset-CEPH (<a href="http://www.cephb.fr/en/hgdp_panel.php#basedonnees">http://www.cephb.fr/en/hgdp_panel.php#basedonnees</a>).</p></div></div><div class="section results" id="sec-16"><h2 class="">Results</h2><div id="sec-17" class="subsection"><h3>Estimating the number of clusters in simulated data</h3><p id="p-66">To illustrate how CONE is able to estimate the true number of (sub)populations, we simulated SNP data under three different models of population structure: the island model (scenario 1), hierarchical island model (scenario 2), and hierarchical stepping-stone model (scenario 3). Overall there were 10 subpopulations in each migration model. In particular, we demonstrate how uneven sample sizes do not have a serious effect on CONE’s ability to detect the correct number of clusters (inferred subpopulations).</p><p id="p-67">We determined the number of inferred clusters <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-110.gif"/></noscript></span></span> with CONE and sNMF (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Frichot <em>et al.</em> 2014</a>), where the latter represents model-based methods. sNMF was run using the R package LEA (version 1.6.0). We used the Walktrap community detection algorithm implemented in the R’s igraph package to infer the number of different communities in our graph estimate. For sNMF we chose the optimal number of clusters which minimized the cross-entropy criterion (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Frichot <em>et al.</em> 2014</a>). We ran sNMF with the number of populations changing from 1 to 15. We calculated five runs for each <em>K</em> value in each scenario to gain a better estimate for the cross-entropy criterion. The nonnegative regularization parameter <em>α</em> was set to 100 with sNMF. The choices we have made are used only for illustrative purposes and both CONE and sNMF can be used in a more adaptable manner. The performance of the STRUCTURE program has been studied extensively (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Puechmaille 2016</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Evanno <em>et al.</em> 2005</a>) with multiple different sample sizes and sampling strategies; the program STRUCTURE using <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-111.gif"/></noscript></span></span> statistics (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Evanno <em>et al.</em> 2005</a>) either detects the uppermost hierarchical level of population structure (even sample sizes) or fails to detect both the upper or lower hierarchical levels of population structure (uneven sample sizes).</p><p id="p-68">With CONE, we used the network modularity approach and StARS to choose the optimal value for the tuning parameter to estimate <em>K</em>. For illustrative purposes only and to avoid unnecessarily long-lasting simulation loops, we used just one of the replicated data sets as a validation set with the modularity approach and chose the value of <em>λ</em> according to the analysis of the validation data set. Then, we continued the simulation analysis using the value of <em>λ</em> determined from the validation set for each of the 50 replicates. We used the root-mean-square error <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-112.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-113.gif"/></noscript></span></span> corresponds to an inferred number of clusters out of 50 estimates, as the error measure. Results are reported in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486878/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486878/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of the simulation results when detecting the true number of subpopulations <strong><em>K</em></strong> (uneven sample sizes)</span><div class="sb-div caption-clear"></div></div></div><p id="p-71">CONE, using the Walktrap community detection algorithm, outperformed the cross-entropy criterion used with sNMF in all scenarios when the tuning parameter is chosen based on the network modularity or StARS. When the tuning parameter is chosen via StARS using the default value 0.05 for <em>β</em>, the number of subpopulations is somewhat underestimated in all scenarios. Overall, CONE seems to detect the lower hierarchical level of structure in all scenarios. In an additional analysis, we examined how the additional weighted graph computed as a part of the StARS procedure was able to distinguish different communities from each other; these estimates are summarized in the last row in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, named as CONE (subsample). In both island models, communities detected from the weighted graph seem to produce downward-biased estimates of the true number of subpopulations. On the other hand, in the hierarchical stepping-stone model, the weighted graph overestimated the true number of subpopulations. The results obtained with sNMF, using the cross-entropy criterion, suffered from uneven sample sizes but, in contrast to the earlier findings (<a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">Puechmaille 2016</a>), it did not underestimate the true number of subpopulations. That is, sNMF was able to detect the lower level hierarchical structure and never detected the upper hierarchical level of structure in the hierarchical models simulated. In the island model, sNMF gave even more accurate estimates of the true number of subpopulations compared to the corresponding estimates inferred from the weighted network. To comprehensively illustrate that the bias in estimated number of subpopulations is due to uneven sampling, we repeated the simulation analysis using even samples. As expected, both CONE and, in particular, sNMF were able to produce more accurate estimates of the true number of subpopulations when the sample sizes were even (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486842/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486842/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of the simulation results when detecting the true number of subpopulations <strong><em>K</em></strong> (even sample sizes)</span><div class="sb-div caption-clear"></div></div></div><p id="p-74">The essential part of our method is to inspect visually how different samples are connected at an individual level. Although CONE did not detect the upper hierarchical level clusters, in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a> one can see how populations in the hierarchical island model from different archipelagos have more edges between them and are thus drawn more closely to each other. The community division graphically illustrated in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a> is neatly in line with the PC1–PC2 plot presented in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The population structure in hierarchical island model. (A) Detecting community structure through the unweighted graph estimate with no edge weights and the Walktrap algorithm. The tuning parameter was chosen via the modularity approach. (B) The PC1&#x2013;PC2 plot of the same data set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1482786432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The population structure in hierarchical island model. (A) Detecting community structure through the unweighted graph estimate with no edge weights and the Walktrap algorithm. The tuning parameter was chosen via the modularity approach. (B) The PC1&#x2013;PC2 plot of the same data set.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3359/F3.medium.gif" width="440" height="330"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/10/3359/F3.medium.gif" width="440" height="330"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486819" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-75" class="first-child">The population structure in hierarchical island model. (A) Detecting community structure through the unweighted graph estimate with no edge weights and the Walktrap algorithm. The tuning parameter was chosen via the modularity approach. (B) The PC1–PC2 plot of the same data set.</p><div class="sb-div caption-clear"></div></div></div><p id="p-76">In most realistic settings there may be no well-defined notion of the “number of populations,” and in these settings the results are usually explored with different values of <em>K</em>. For example, both <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Evanno <em>et al.</em> (2005)</a> and <a id="xref-ref-41-6" class="xref-bibr" href="#ref-41">Puechmaille (2016)</a> have shown that the STRUCTURE program usually detects the uppermost hierarchical structure in a sample. This hierarchy is not revealed with a distinct value of <em>K</em>. Therefore, inferring networks constructed with CONE by visual inspection is the easiest way to gain insight over the possible hierarchical structure of the data.</p></div><div id="sec-18" class="subsection"><h3>Estimating the ancestry coefficients</h3><p id="p-77">To illustrate how CONE is able to estimate admixture coefficients compared to model-based clustering, we conducted a simulation study following the guidelines of <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Alexander <em>et al.</em> (2009)</a> and <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Frichot <em>et al.</em> (2014)</a>. It should be noted that a fair comparison between the proposed method and model-based methods is not straightforward. Model-based methods describe the population structure by probabilistic assignment and our method characterizes the population structure with network communities. Nevertheless, paralleling our method with a state-of-the-art methodology serves as a good benchmark for our method’s potential. We compared our method to the model-based method sNMF.</p><p id="p-78">With CONE, we used one data set as a validation set to choose a proper tuning parameter value with the modularity approach. Nevertheless, using the same value for the tuning parameter in a simulation run does not absolutely rule out the possibility that the dimensions of the true <em>Q</em> and the one estimated with the CONE framework differ. If CONE detected more (or less) than three populations, we modified the cut height of the dendrogram leading to three clusters. We used the RMSE to measure the accuracy of the estimated <em>Q</em> matrix, <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-114.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-115.gif"/></noscript></span></span> is the Frobenius norm of an <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-116.gif"/></noscript></span></span> matrix <em>A</em>. With sNMF, <em>K</em> was set to the true number of populations (three). We ran sNMF just once per each simulated data set. With sNMF, the value of the nonnegative regularization parameter <em>α</em> was set to either 0 or 100 according to the value that produced the smallest median value of the estimated RMSEs. The simulation results are presented in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accuracy of CONE and sNMF. Box plots of RMSEs between the estimated Q matrices and known ancestry proportions over 50 simulation runs with both sNMF and CONE." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1482786432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Accuracy of CONE and sNMF. Box plots of RMSEs between the estimated &lt;em&gt;Q&lt;/em&gt; matrices and known ancestry proportions over 50 simulation runs with both sNMF and CONE.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3359/F4.medium.gif" width="440" height="225"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/10/3359/F4.medium.gif" width="440" height="225"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486866" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-79" class="first-child">Accuracy of CONE and sNMF. Box plots of RMSEs between the estimated <em>Q</em> matrices and known ancestry proportions over 50 simulation runs with both sNMF and CONE.</p><div class="sb-div caption-clear"></div></div></div><p id="p-80">Using sNMF, accurate ancestry estimates with completely admixed and unrelated individuals were obtained for all simulated population models. The error measures estimated with sNMF are in line with previous studies (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Alexander <em>et al.</em> 2009</a>; <a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">Frichot <em>et al.</em> 2014</a>; <a id="xref-ref-43-7" class="xref-bibr" href="#ref-43">Raj <em>et al.</em> 2014</a>). A larger dispersion of the CONE results can be explained by the fact that we did not optimize the tuning parameter in each simulated data set separately. To give a better illustration of CONE’s potential, we performed an additional analysis by determining the ancestry estimates with several values of the tuning parameter <em>λ</em>. Then, we choose the estimated <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-117.gif"/></noscript></span></span> which minimized the RMSE of the ancestry estimate. The results are presented in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. In each case, error estimates of CONE were diminished in every parameterization of the Dirichlet distribution. CONE even seemed to be able to give more accurate ancestry proportion estimates than sNMF in two examined parameterizations of the Dirichlet distribution. We have graphically represented typical results of the ancestry coefficients estimated with both sNMF and CONE in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>, <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>, and <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accuracy of CONE (solution path) and sNMF. Box plots of RMSEs between the estimated Q matrices and known ancestry proportions over 50 simulation runs with both sNMF and CONE." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1482786432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Accuracy of CONE (solution path) and sNMF. Box plots of RMSEs between the estimated &lt;em&gt;Q&lt;/em&gt; matrices and known ancestry proportions over 50 simulation runs with both sNMF and CONE.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3359/F5.medium.gif" width="440" height="225"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/10/3359/F5.medium.gif" width="440" height="225"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486857" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-81" class="first-child">Accuracy of CONE (solution path) and sNMF. Box plots of RMSEs between the estimated <em>Q</em> matrices and known ancestry proportions over 50 simulation runs with both sNMF and CONE.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graphical representation of ancestry coefficients. Simulated ancestry coefficients and the corresponding estimates computed with sNMF and CONE from an arbitrary simulated data set. The entries of the Q matrix considered here were generated from the  distribution." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1482786432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Graphical representation of ancestry coefficients. Simulated ancestry coefficients and the corresponding estimates computed with sNMF and CONE from an arbitrary simulated data set. The entries of the &lt;em&gt;Q&lt;/em&gt; matrix considered here were generated from the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-118&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; distribution.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3359/F6.medium.gif" width="440" height="282"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/10/3359/F6.medium.gif" width="440" height="282"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486808" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-82" class="first-child">Graphical representation of ancestry coefficients. Simulated ancestry coefficients and the corresponding estimates computed with sNMF and CONE from an arbitrary simulated data set. The entries of the <em>Q</em> matrix considered here were generated from the <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/F6/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/F6/embed/mml-math-118.gif"/></noscript></span></span> distribution.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graphical representation of ancestry coefficients. Simulated ancestry coefficients and the corresponding estimates computed with sNMF and CONE from an arbitrary simulated data set. The entries of the Q matrix considered here were generated from the  distribution." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1482786432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Graphical representation of ancestry coefficients. Simulated ancestry coefficients and the corresponding estimates computed with sNMF and CONE from an arbitrary simulated data set. The entries of the &lt;em&gt;Q&lt;/em&gt; matrix considered here were generated from the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-119&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; distribution.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3359/F7.medium.gif" width="440" height="282"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/10/3359/F7.medium.gif" width="440" height="282"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486823" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-83" class="first-child">Graphical representation of ancestry coefficients. Simulated ancestry coefficients and the corresponding estimates computed with sNMF and CONE from an arbitrary simulated data set. The entries of the <em>Q</em> matrix considered here were generated from the <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/F7/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/F7/embed/mml-math-119.gif"/></noscript></span></span> distribution.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graphical representation of ancestry coefficients. Simulated ancestry coefficients and the corresponding estimates computed with sNMF and CONE from an arbitrary simulated data set. The entries of the Q matrix considered here were generated from the  distribution." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1482786432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Graphical representation of ancestry coefficients. Simulated ancestry coefficients and the corresponding estimates computed with sNMF and CONE from an arbitrary simulated data set. The entries of the &lt;em&gt;Q&lt;/em&gt; matrix considered here were generated from the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-120&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; distribution.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3359/F8.medium.gif" width="440" height="282"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/7/10/3359/F8.medium.gif" width="440" height="282"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486926" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-84" class="first-child">Graphical representation of ancestry coefficients. Simulated ancestry coefficients and the corresponding estimates computed with sNMF and CONE from an arbitrary simulated data set. The entries of the <em>Q</em> matrix considered here were generated from the <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/F8/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/F8/embed/mml-math-120.gif"/></noscript></span></span> distribution.</p><div class="sb-div caption-clear"></div></div></div><p id="p-85">We find it remarkable that CONE was able to estimate ancestry coefficients even close to those obtained by model-based methods, such as sNMF. With CONE, we did not make any assumptions over the relationship of the <em>Q</em> matrix and the binomial model used in our simulation analysis. The relatively small error estimates suggests that a sparse individual-level network represents different levels of admixture in the data.</p></div><div id="sec-19" class="subsection"><h3>Application to teosinte data</h3><p id="p-86">To illustrate how CONE is able to describe population structure in a practical application, we applied it to moderately sized SNP genotype data sets previously described in <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Pyhäjärvi <em>et al.</em> (2013)</a>.</p><p id="p-87">We started our analysis of these data sets by running the StARS procedure with 20 subsamples each containing almost 2000 SNP markers, performing the logistic multinomial LASSO regression on each subsample in turn. 40 different values of the tuning parameter <em>λ</em> ranging from 0.05 to 0.5 were tested. Using our in-house R code, it took ∼30 min to perform the StARS procedure for both of the subspecies data sets on a standard desktop computer, and ∼90 min for the complete data set. After completing the StARS analysis, we used all of the SNP markers in the neighborhood selection procedure. This graph construction process took just a few minutes, and was finalized using the Fruchterman–Reingold algorithm as implemented in the qgraph package to delineate different communities. The final graphs, which we refer to as individual-level <em>dependency networks</em>, are presented in <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The individual-level dependency structure between different samples in the teosinte data. (A) Graph constructed by using our customized neighborhood selection scheme and the nodes are colored according to the subspecies they represent. (B) Weighted graph in which the additional StARS weights are used as edge weights; the nodes are colored based on the 21 populations identified in the teosinte data. (C) Network of the parviglumis subspecies. (D) Network of the mexicana subspecies. The coloring and population labels were added after graph estimation in all cases." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1482786432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The individual-level dependency structure between different samples in the teosinte data. (A) Graph constructed by using our customized neighborhood selection scheme and the nodes are colored according to the subspecies they represent. (B) Weighted graph in which the additional StARS weights are used as edge weights; the nodes are colored based on the 21 populations identified in the teosinte data. (C) Network of the &lt;em&gt;parviglumis&lt;/em&gt; subspecies. (D) Network of the &lt;em&gt;mexicana&lt;/em&gt; subspecies. The coloring and population labels were added after graph estimation in all cases.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3359/F9.medium.gif" width="440" height="393"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/7/10/3359/F9.medium.gif" width="440" height="393"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486894" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-88" class="first-child">The individual-level dependency structure between different samples in the teosinte data. (A) Graph constructed by using our customized neighborhood selection scheme and the nodes are colored according to the subspecies they represent. (B) Weighted graph in which the additional StARS weights are used as edge weights; the nodes are colored based on the 21 populations identified in the teosinte data. (C) Network of the <em>parviglumis</em> subspecies. (D) Network of the <em>mexicana</em> subspecies. The coloring and population labels were added after graph estimation in all cases.</p><div class="sb-div caption-clear"></div></div></div><p id="p-89"><a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9, A and B</a>, clearly shows that the samples are divided into two major subpopulations according to the two subspecies and one minor subpopulation (the El Rodeo population). The network shown in <a id="xref-fig-9-3" class="xref-fig" href="#F9">Figure 9B</a> clearly indicates that, while the samples are divided into several populations, these populations are arranged into two larger groups corresponding to the two subspecies, and gene flow occurs (or has occurred) mostly within these groups. Overall, the Walktrap algorithm divided the samples into 12 different communities. On the other hand, the network modularity revealed that the optimal number of different communities would be ∼12–20 in the data set (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300131/-/DC1/FigureS1.pdf">Figure S1</a>). In the STRUCTURE analysis from <a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Pyhäjärvi <em>et al.</em> (2013)</a>, where the number of populations was set to four, the El Rodeo subpopulation from <em>parviglumis</em> stands out as a separate population (their figure 4B). The divergence of the El Rodeo subpopulation is probably due some environmental factor; the edge lengths in our dependency network illustrate only the strength of the genetic relationship between individuals, not the physical distance. For example, the Nabogame population in both <a id="xref-fig-9-4" class="xref-fig" href="#F9">Figure 9, B and D</a>, is connected to other populations from the same subspecies, even though the corresponding samples are widely separated from other <em>mexicana</em> populations (figure 1 of <a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Pyhäjärvi <em>et al.</em> 2013</a>). The graphs in <a id="xref-fig-9-5" class="xref-fig" href="#F9">Figure 9, A and B</a>, indicate that there is some gene flow between the subspecies and that most of it occurs between the Ahuacatitlan and Puruandiro populations: this is consistent with the STRUCTURE analysis presented by <a id="xref-ref-42-7" class="xref-bibr" href="#ref-42">Pyhäjärvi <em>et al.</em> (2013)</a> (their figure 4B). As in their analysis, the Ahuacatitlan population remains strongly connected to other <em>parviglumis</em> populations, but there are only a few weak edges connecting it to <em>mexicana</em> populations. Our graph interpretation therefore does not support the hypothesis that Ahuacatitlan is a sister group to <em>mexicana</em>. In <a id="xref-fig-9-6" class="xref-fig" href="#F9">Figure 9, B and C</a>, it can be seen that the two Ejutla populations merge into a single population. This result is also consistent with the PCA of <a id="xref-ref-42-8" class="xref-bibr" href="#ref-42">Pyhäjärvi <em>et al.</em> (2013)</a> and, on the whole, our depencency network shown in <a id="xref-fig-9-7" class="xref-fig" href="#F9">Figure 9B</a> has the same characteristics as the PC1–PC2 plot of <a id="xref-ref-42-9" class="xref-bibr" href="#ref-42">Pyhäjärvi <em>et al.</em> (2013)</a> (their figure 4A). As mentioned above, the <em>mexicana</em> populations include two hybrid samples. We assume that hybrid samples are the two nodes from Puruandiro and Nabogame which connect these two populations in <a id="xref-fig-9-8" class="xref-fig" href="#F9">Figure 9D</a>. Estimated ancestry coefficients with <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-121.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-122.gif"/></noscript></span></span> are graphically represented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300131/-/DC1/FigureS2.pdf">Figure S2</a>.</p></div><div id="sec-20" class="subsection"><h3>Application to E. coli data</h3><p id="p-90">As a second practical example we evaluated the performance of our method by applying it to the SNP genotype data set previously described in <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Salipante <em>et al.</em> (2015)</a>.</p><p id="p-91">We began the CONE procedure with the <em>E. coli</em> data set by running the StARS procedure with 20 subsamples containing 5775 SNP markers each, performing the logistic multinomial LASSO regression to each subsample individually. We used 40 different values for the tuning parameter <em>λ</em> ranging from 0.05 to 0.5, and it took ∼90 min to complete the StARS procedure using our in-house R code. After the StARS analysis, we used all the SNP markers in the neighborhood selection procedure. Running the neighborhood construction process took ∼208 min to complete. As in the teosinte graph construction, we used the Fruchterman–Reingold algorithm, as implemented in the qgraph package, to distinguish different communities from each other for visual representation. The final graph estimates are presented in <a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10</a>.</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The dependency structure inferred between individuals from the E. coli data with the CONE framework. Each strain is represented by a colored node and labeled based on the division of the samples into 12 different communities (C1, C2, &#x2026;, C12) detected with CONE. (A) Additional graph estimated using the StARS procedure. (B) The graph resulting from CONE framework. (C) The weighted graph obtained by combining the two aforementioned graphs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1482786432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The dependency structure inferred between individuals from the &lt;em&gt;E. coli&lt;/em&gt; data with the CONE framework. Each strain is represented by a colored node and labeled based on the division of the samples into 12 different communities (C1, C2, &#x2026;, C12) detected with CONE. (A) Additional graph estimated using the StARS procedure. (B) The graph resulting from CONE framework. (C) The weighted graph obtained by combining the two aforementioned graphs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3359/F10.medium.gif" width="424" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/7/10/3359/F10.medium.gif" width="424" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486957" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-92" class="first-child">The dependency structure inferred between individuals from the <em>E. coli</em> data with the CONE framework. Each strain is represented by a colored node and labeled based on the division of the samples into 12 different communities (C1, C2, …, C12) detected with CONE. (A) Additional graph estimated using the StARS procedure. (B) The graph resulting from CONE framework. (C) The weighted graph obtained by combining the two aforementioned graphs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-93">In <a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure 10</a>, we colored the nodes according to their community membership. Community detection was done using the Walktrap algorithm. Based on the graph estimates, we concluded that there are 10–12 different communities represented in the complete data set. We did not include the strain IDs in the networks shown in <a id="xref-fig-10-3" class="xref-fig" href="#F10">Figure 10</a>, but a network including the strain IDs is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300131/-/DC1/FigureS3.pdf">Figure S3</a> for readers seeking more detailed information on the nodes. After making the division, we used the same coloring scheme to color the phylogenetic tree that we derived from the data; this phylogenetic tree is presented in <a id="xref-fig-11-1" class="xref-fig" href="#F11">Figure 11</a>, and a more detailed version can be seen in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300131/-/DC1/FigureS4.pdf">Figure S4</a>.</p><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The phylogenetic tree generated from the E. coli data. Strains are colored according to the community division presented in Figure 10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1482786432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The phylogenetic tree generated from the &lt;em&gt;E. coli&lt;/em&gt; data. Strains are colored according to the community division presented in Figure 10.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 11" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3359/F11.medium.gif" width="440" height="383"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 11" src="https://www.g3journal.org/content/ggg/7/10/3359/F11.medium.gif" width="440" height="383"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 11" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486800" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 11</span> <p id="p-94" class="first-child">The phylogenetic tree generated from the <em>E. coli</em> data. Strains are colored according to the community division presented in <a id="xref-fig-10-4" class="xref-fig" href="#F10">Figure 10</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-95">As clearly shown by <a id="xref-fig-10-5" class="xref-fig" href="#F10">Figure 10</a> (see also <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300131/-/DC1/FigureS3.pdf">Figure S3</a> for the strain names), the inferred community partition resembles the clades in the phylogenetic tree (<a id="xref-fig-11-2" class="xref-fig" href="#F11">Figure 11</a>). The resemblance is especially apparent when comparing the phylogenetic tree to communities C2, C3, C7, and C8. Moreover, the nodes linking communities C2 and C4 in <a id="xref-fig-10-6" class="xref-fig" href="#F10">Figure 10, A–C</a>, are reflected in the branch connecting strains from the corresponding communities in <a id="xref-fig-11-3" class="xref-fig" href="#F11">Figure 11</a>. However, there are some minor differences between the tree’s branch structure and the node arrangement in <a id="xref-fig-10-7" class="xref-fig" href="#F10">Figure 10, B and C</a>, which are based on analyses of the complete data set. For example, community C8 appears to be isolated, with no edges linking it to C3. However, in the phylogenetic tree, there are some branches connecting strains from these communities, while their genetic relationship is weak. In the StARS graph, there are some wavering edges between communities C3 and C8 (<a id="xref-fig-10-8" class="xref-fig" href="#F10">Figure 10A</a>). Additionally, the small communities identified by the CONE analysis (C7, C9, C10, C11, and C12) are separated from the other communities; this division is somewhat consistent with <a id="xref-fig-11-4" class="xref-fig" href="#F11">Figure 11</a> because strains from communities C7 and C11 are only distantly related to other strains. In addition, we computed the ancestry coefficients with CONE (<a id="xref-fig-12-1" class="xref-fig" href="#F12">Figure 12</a>).</p><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graphical representation of the estimated ancestry coefficients obtained from the E. coli data set. (A) The estimated ancestry coefficients using the weighted network (Figure 11A). With this network estimate, the number of detected clusters was 10 ( = 10). (B) The estimated ancestry coefficients using the unweighted network (Figure 11B). (C) The estimated coefficients using the weighted network (Figure 11C). With network estimates (B and C), the number of detected clusters was 12 ( = 12). Different clusters are colored according to the same coloring presented in Figure 10 and Figure 11." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1482786432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Graphical representation of the estimated ancestry coefficients obtained from the &lt;em&gt;E. coli&lt;/em&gt; data set. (A) The estimated ancestry coefficients using the weighted network (Figure 11A). With this network estimate, the number of detected clusters was 10 (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-123&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; = 10). (B) The estimated ancestry coefficients using the unweighted network (Figure 11B). (C) The estimated coefficients using the weighted network (Figure 11C). With network estimates (B and C), the number of detected clusters was 12 (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-124&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; = 12). Different clusters are colored according to the same coloring presented in Figure 10 and Figure 11.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 12" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3359/F12.medium.gif" width="440" height="316"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 12" src="https://www.g3journal.org/content/ggg/7/10/3359/F12.medium.gif" width="440" height="316"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 12" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486813" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 12</span> <p id="p-96" class="first-child">Graphical representation of the estimated ancestry coefficients obtained from the <em>E. coli</em> data set. (A) The estimated ancestry coefficients using the weighted network (<a id="xref-fig-11-5" class="xref-fig" href="#F11">Figure 11A</a>). With this network estimate, the number of detected clusters was 10 (<span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/F12/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/F12/embed/mml-math-123.gif"/></noscript></span></span> = 10). (B) The estimated ancestry coefficients using the unweighted network (<a id="xref-fig-11-6" class="xref-fig" href="#F11">Figure 11B</a>). (C) The estimated coefficients using the weighted network (<a id="xref-fig-11-7" class="xref-fig" href="#F11">Figure 11C</a>). With network estimates (B and C), the number of detected clusters was 12 (<span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/F12/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/F12/embed/mml-math-124.gif"/></noscript></span></span> = 12). Different clusters are colored according to the same coloring presented in <a id="xref-fig-10-9" class="xref-fig" href="#F10">Figure 10</a> and <a id="xref-fig-11-8" class="xref-fig" href="#F11">Figure 11</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h3>Application to HGDP data</h3><p id="p-97">To emphasize the potential of the CONE framework in the analysis of very large SNP data sets, we applied it to the so-called Stanford HGDP SNP genotyping data set (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Rosenberg <em>et al.</em> 2002</a>), which has previously been analyzed by authors including <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Li <em>et al.</em> (2008)</a> and <a id="xref-ref-43-8" class="xref-bibr" href="#ref-43">Raj <em>et al.</em> (2014)</a> using FRAPPE, ADMIXTURE, fastSTRUCTURE, and PCA. Overall there are 1043 individuals from 51 different populations and 660,918 SNP markers.</p><p id="p-98">As with the teosinte data, the first step in the CONE procedure was to perform the StARS procedure with 20 subsamples, each containing 8120 SNP markers, applying the logistic multinomial LASSO regression to each subsample separately. We used 40 different values for the tuning parameter <em>λ</em>, ranging from 0.05 to 0.5. It took ∼2 d to run StARS analysis in R. We ultimately set the value of the tuning parameter <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-125.gif"/></noscript></span></span> to 0.0812. Using this value we obtained our first graph estimate, which is presented in <a id="xref-fig-13-1" class="xref-fig" href="#F13">Figure 13A</a>.</p><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The dependency structure between individuals in the HGDP data set inferred with CONE framework. (A) The additional graph estimated from the StARS procedure. (B) The customized neighborhood selection graph. (C) The weighted graph obtained by combining the two. (D) Graph colored according to the different geographic areas from which the individuals in the data set originate. Each individual is represented by a colored node according to the division of the data into seven different populations as described by Raj et al. (2014). The associated coloring and population labels were added after the graph was estimated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1482786432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The dependency structure between individuals in the HGDP data set inferred with CONE framework. (A) The additional graph estimated from the StARS procedure. (B) The customized neighborhood selection graph. (C) The weighted graph obtained by combining the two. (D) Graph colored according to the different geographic areas from which the individuals in the data set originate. Each individual is represented by a colored node according to the division of the data into seven different populations as described by Raj &lt;em&gt;et al.&lt;/em&gt; (2014). The associated coloring and population labels were added after the graph was estimated.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 13" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3359/F13.medium.gif" width="440" height="397"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 13" src="https://www.g3journal.org/content/ggg/7/10/3359/F13.medium.gif" width="440" height="397"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 13" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486950" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 13</span> <p id="p-99" class="first-child">The dependency structure between individuals in the HGDP data set inferred with CONE framework. (A) The additional graph estimated from the StARS procedure. (B) The customized neighborhood selection graph. (C) The weighted graph obtained by combining the two. (D) Graph colored according to the different geographic areas from which the individuals in the data set originate. Each individual is represented by a colored node according to the division of the data into seven different populations as described by <a id="xref-ref-43-9" class="xref-bibr" href="#ref-43">Raj <em>et al.</em> (2014)</a>. The associated coloring and population labels were added after the graph was estimated.</p><div class="sb-div caption-clear"></div></div></div><p id="p-100">After the StARS run, we analyzed the full data set (all 659,421 SNPs) using the optimized <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-126.gif"/></noscript></span></span> value. Having obtained the corresponding adjacency matrix, we used the Fruchterman–Reingold algorithm to detect different communities among the nodes, yielding the graph estimate presented in <a id="xref-fig-13-2" class="xref-fig" href="#F13">Figure 13B</a>. As was done with the teosinte data, we used this graph along with the additional StARS graph and calculated the entrywise product of the corresponding adjacency matrices to produce an emphasized graph estimate: These graph estimates are presented in <a id="xref-fig-13-3" class="xref-fig" href="#F13">Figure 13, B and C</a>. To better illustrate the details of the population group division, we colored the graph in <a id="xref-fig-13-4" class="xref-fig" href="#F13">Figure 13D</a> according to the geographic areas associated with the samples. Again, we will refer to the graphs presented in <a id="xref-fig-13-5" class="xref-fig" href="#F13">Figure 13, A–D</a>, as dependency networks. It took almost 4 d to perform our customized neighborhood selection procedure on the whole data set using glmnet with R running on a desktop computer with 8 GB of RAM and a 3.1 GHz Intel Core i5-2400 processor. In our opinion, this demonstrates the CONE framework’s ability to analyze large data sets in reasonable time frames even when using only moderately powerful hardware and readily available software.</p><p id="p-101">The estimated networks have similarities with the ancestry proportions estimated by fastSTRUCTURE and ADMIXTURE, as reported by <a id="xref-ref-43-10" class="xref-bibr" href="#ref-43">Raj <em>et al.</em> (2014)</a>, and with the PCA analysis presented by <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Li <em>et al.</em> (2008)</a>. The graphs in <a id="xref-fig-13-6" class="xref-fig" href="#F13">Figure 13</a> show that the Middle Eastern populations are closely related to the European populations. There are also clear indications of gene flow between the populations living in Eurasia and North Africa, which is consistent with the PC1–PC2 plot (their figure S3B) presented in the supplemental material of <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Li <em>et al.</em> (2008)</a>. Both the Oceanian and American populations are estimated to belong to separate groups despite the presence of minor gene flow between Europe and America, as indicated by both the ADMIXTURE and fastSTRUCTURE analyses. The fastSTRUCTURE and ADMIXTURE estimates differ from each other with respect to the way they divide the African and Native American populations. The dependency networks also indicate that both the American and African populations are divided into multiple subgroups. The ADMIXTURE ancestry proportions imply the existence of two distinct American population groups with some gene flow between the Northern and Southern populations; this is consistent with our graph estimates, in which there are some edges linking Southern and Central American individuals. Moreover, the fastSTRUCTURE ancestry proportions indicate that the African populations are split into Northern and Central African groups, with some gene flow between them. Both the ADMIXTURE analysis and the dependency networks imply the existence of a weak gene flow between the Central Asian and African populations. The PCA analysis of <a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">Li <em>et al.</em> (2008)</a> suggests that the Oceanian populations are divided into two subpopulations, which is again consistent with the dependency networks. Neither the ADMIXTURE nor the fastSTRUCTURE analyses revealed a similar division.</p><p id="p-102">To determine how the inclusion of close relatives affected our graph estimation, we computed the genomic relationship matrix (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">VanRaden 2008</a>). In <a id="xref-fig-14-1" class="xref-fig" href="#F14">Figure 14</a>, nodes corresponding to closely related individuals are shown in white. As can be seen, close relatives are present in every population group but their presence did not seem to adversely affect the neighborhood selection process.</p><div id="F14" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F14.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The dependency structure between individuals in the HGDP data set as inferred with CONE. Closely related individuals (white nodes) were identified by considering the genomic relationship matrix." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1482786432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The dependency structure between individuals in the HGDP data set as inferred with CONE. Closely related individuals (white nodes) were identified by considering the genomic relationship matrix.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 14" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3359/F14.medium.gif" width="440" height="413"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 14" src="https://www.g3journal.org/content/ggg/7/10/3359/F14.medium.gif" width="440" height="413"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 14" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486879" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 14</span> <p id="p-103" class="first-child">The dependency structure between individuals in the HGDP data set as inferred with CONE. Closely related individuals (white nodes) were identified by considering the genomic relationship matrix.</p><div class="sb-div caption-clear"></div></div></div><p id="p-104">To further compare our results to those of <a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">Li <em>et al.</em> (2008)</a>, we also performed a fine-scale population graph analysis of the European and Middle Eastern populations. <a id="xref-fig-15-1" class="xref-fig" href="#F15">Figure 15</a> shows the results obtained by analyzing the European and the Middle Eastern populations with CONE, which can be compared to the PCA analysis of <a id="xref-ref-31-8" class="xref-bibr" href="#ref-31">Li <em>et al.</em> (2008)</a>. The graphs in <a id="xref-fig-15-2" class="xref-fig" href="#F15">Figure 15, C and D</a>, are so-called emphasized graphs, which were constructed in a similar way to those shown in <a id="xref-fig-13-7" class="xref-fig" href="#F13">Figure 13, C and D</a>, and <a id="xref-fig-14-2" class="xref-fig" href="#F14">Figure 14</a>. Both graphs are quite similar to the PC1–PC2 plots of <a id="xref-ref-31-9" class="xref-bibr" href="#ref-31">Li <em>et al.</em> (2008)</a> (figure 2, A and B, in their manuscript). As noted in Li <em>et al.</em>’s discussion of the PCA results, the European populations can be separated into four relatively separate subpopulations (Adygei, Basque, Russian, and Sardinian), and a number of seemingly independent individuals are present. In <a id="xref-fig-15-3" class="xref-fig" href="#F15">Figure 15B</a>, the Bedouins seem to divide evenly into three continuous groups, with some separated individuals indicating weak gene flow between other populations. The Mozabite are distinguished as a separate population, which is consistent with the structure of the North African region in the emphasized graph shown in <a id="xref-fig-15-4" class="xref-fig" href="#F15">Figure 15D</a>.</p><div id="F15" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F15.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The dependency structure inferred with CONE of a subsample of the HGDP data. (A) Unweighted graph of the European populations. (B) Unweighted graph of the Middle East populations. (C) Weighted graph of the European populations. (D) Weighted graph of the Middle East populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1482786432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The dependency structure inferred with CONE of a subsample of the HGDP data. (A) Unweighted graph of the European populations. (B) Unweighted graph of the Middle East populations. (C) Weighted graph of the European populations. (D) Weighted graph of the Middle East populations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 15" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3359/F15.medium.gif" width="440" height="397"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 15" src="https://www.g3journal.org/content/ggg/7/10/3359/F15.medium.gif" width="440" height="397"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 15" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3359/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486941" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 15</span> <p id="p-105" class="first-child">The dependency structure inferred with CONE of a subsample of the HGDP data. (A) Unweighted graph of the European populations. (B) Unweighted graph of the Middle East populations. (C) Weighted graph of the European populations. (D) Weighted graph of the Middle East populations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-106">There are some elements of the dependency networks that are not intuitively clear and whose interpretation would require a detailed analysis of the data. The first oddity is that one Middle Eastern individual was appointed to the central African population with no direct connection to any other individual among the Middle Eastern populations. Another difficult-to-interpret subpopulation is the small population group consisting of 12 samples from various backgrounds that is apparent in <a id="xref-fig-13-8" class="xref-fig" href="#F13">Figure 13</a> and <a id="xref-fig-14-3" class="xref-fig" href="#F14">Figure 14</a>. The elements of the genomic relationships between these individuals are also quite large (<span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-127.gif"/></noscript></span></span>), so these samples were at least partly excluded in previous analyses of the data set (<a id="xref-ref-31-10" class="xref-bibr" href="#ref-31">Li <em>et al.</em> 2008</a>; <a id="xref-ref-43-11" class="xref-bibr" href="#ref-43">Raj <em>et al.</em> 2014</a>). We have listed these individuals in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and indicated them using black nodes in <a id="xref-fig-14-4" class="xref-fig" href="#F14">Figure 14</a>. We suggest that the formation of this extra group is not due to problems with the CONE procedure and instead indicates that these individuals have interesting qualities that have not previously been identified. Only one individual (from the Pathan population, ID no. 00251 in the HGDP data set) was estimated to be totally independent from all of the other populations.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486881/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486881/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Twelve outlier individuals that differ remarkably from other individuals from the corresponding populations</span><div class="sb-div caption-clear"></div></div></div><p id="p-107">As also found by <a id="xref-ref-43-12" class="xref-bibr" href="#ref-43">Raj <em>et al.</em> (2014)</a>, both fastSTRUCTURE and ADMIXTURE analyses suggest that more than seven population groups are represented in the data. The dependency networks also indicate that there may be more than seven population groups. Estimated ancestry coefficients with different values of <em>K</em> (<span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-128.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-129.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-130.gif"/></noscript></span></span>) are graphically represented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300131/-/DC1/FigureS5.pdf">Figure S5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300131/-/DC1/FigureS6.pdf">Figure S6</a>.</p></div></div><div class="section discussion" id="sec-22"><h2 class="">Discussion</h2><p id="p-108">By applying the CONE method to certain simulated scenarios of population history as well as real-world data sets previously analyzed using PCA (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Price <em>et al.</em> 2006</a>), STRUCTURE (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Pritchard <em>et al.</em> 2000</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Falush <em>et al.</em> 2003</a>), ADMIXTURE (<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Alexander <em>et al.</em> 2009</a>), fastSTRUCTURE (<a id="xref-ref-43-13" class="xref-bibr" href="#ref-43">Raj <em>et al.</em> 2014</a>), and FRAPPE (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Tang <em>et al.</em> 2005</a>), we have demonstrated that graph inference using neighborhood selection can be used for individual-level population structure analysis based on SNP data.</p><p id="p-109">When applied to the <em>E. coli</em> data set, our network estimation method gave results that were consistent with the phylogenetic tree. In general, the inferred community partition closely resembled the distribution of clades within the tree. The potential for recombination among and within lineages in the <em>E. coli</em> population could give rise to many difficulties in estimating its relatedness structure, which is a well-known limitation in phylogenetic tree reconstruction (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Posada and Crandall 2002</a>). Population graphs constructed using our method could help to resolve such problems because the observation of multiple connections between strains coupled with the graph’s topology can reveal deviations from the clonal frame (corresponding to the absence of recombination).</p><p id="p-110">It has been shown that MB-style approximation is also consistent in the so-called high-dimensional setting as the number of continuous Gaussian random variables increases (<a id="xref-ref-34-8" class="xref-bibr" href="#ref-34">Meinshausen and Bühlmann 2006</a>). When using our graphical approach, the high-dimensional scenario would correspond to an uncommon case in which there are more individuals than measured “genotypes” in the data set. We did not examine the analytic properties of such settings, but they could potentially cause some practical problems in the StARS procedure, because the subsampling in StARS can give rise to very uneven allele frequencies; leading to failures of convergence when attempting to obtain maximum likelihood estimates. It may also be the case that the number of falsely estimated neighbors would make meaningful interpretation of the network impossible. We also examined the potential for graphical structure analysis based on multiallelic microsatellite data but found the CONE procedure to be unusable with such data sets that include large and variable numbers of classes.</p><p id="p-111">Our method is able to provide a <em>Q</em> matrix (<em>i.e.</em>, population membership probabilities for the individuals in the data), which could subsequently be used in an association analysis model to correct for population stratification (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Yu <em>et al.</em> 2006</a>). However, it is completely unknown to us how well it may work in practice and therefore we emphasize that one should proceed with caution. Further examination of the relationship between the ancestry coefficients and network estimation should be derived for more exact results, but further dwelling to the subject is left for future studies.</p><p id="p-112"><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jalali <em>et al.</em> (2011)</a> have shown that for a multinomial model, one can use a group LASSO penalty whereby the regression coefficients corresponding to the same explanatory variable (individual) for all three classes are forced to all become zero or nonzero together. Using the group LASSO penalty would also yield a consistent neighborhood selection procedure for the edge set <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-131.gif"/></noscript></span></span> under strict conditions. However, <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Jalali <em>et al.</em> (2011)</a> used cross-validation to choose the tuning parameter <em>λ</em>. In our experience, cross-validation tends to produce overly dense graphs that are hard to interpret. The glmnet package also includes the option to apply group LASSO penalties to the coefficients. We tested this penalization scheme in an analysis of the European and Middle Eastern populations from the HGDP data, using the StARS procedure to choose an optimal value for the regularization parameter <em>λ</em>. In the case of the European populations, the results were practically identical to those obtained using our simpler conditions for neighborhood selection (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300131/-/DC1/FigureS7.pdf">Figure S7</a>). For the Middle Eastern populations, the biggest difference was that the group LASSO penalty seemed to estimate individuals belonging to the Druze and Palestinian populations to be more closely related than was the case without penalization. The major drawback of the group LASSO approach is that it increases the time required to run the neighborhood selection procedure. For example, the StARS analysis for the Middle Eastern populations took ∼2.5 hr with our simple edge selection rule whereas the group LASSO method took &gt;9 hr to complete. When dealing with large SNP panels, because the choice between the group LASSO penalty and our rule-based approach seems to have minimal influence on the individual-level population graphs, the faster computation time favors our rule-based approach over the group penalty. Moreover, it is important to note that the dependency networks seem to be much more sensitive to the way of choosing the tuning parameter (StARS in our case) than to the use of a group penalty.</p><p id="p-113">We are interested in developing our method further and reducing its computational time. Novel statistical methods such as BIGQUIC (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hsieh <em>et al.</em> 2013</a>) have been proposed to handle large data sets rapidly. Similar to our approach, BIGQUIC can analyze data with <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3359/embed/mml-math-132.gif"/></noscript></span></span> variables with the R program. Parallel computing speeds BIGQUIC up even more. Implementing our method in user friendly software is left for future studies.</p><p id="p-114">The methods described here have been implemented using R and our own procedures. A collection of codes can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300131/-/DC1/FileS1.zip">File S1</a>.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-115">We are grateful to Tanja Pyhäjärvi for useful discussions concerning the teosinte data set, which helped us to improve our final work. We are also grateful to Jouni Aspi and Markus Harju for helpful discussions. This work was funded by the University of Oulu’s Technology and Natural Sciences Doctoral Programme (TNS-DP) and by the Swedish Ministry of Foreign Affairs, project A4952.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-116">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300131/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300131/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-117"><em>Communicating editor: D-J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-03-28"><span class="received-label">Received </span>March 28, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-08-11"><span class="accepted-label">Accepted </span>August 11, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Kuismin <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.1" data-doi="10.1038/nature11632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An integrated map of genetic variation from 1,092 human genomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">56</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAbecasis%26rft.auinit1%253DG.%2BR.%26rft.volume%253D491%26rft.issue%253D7422%26rft.spage%253D56%26rft.epage%253D65%26rft.atitle%253DAn%2Bintegrated%2Bmap%2Bof%2Bgenetic%2Bvariation%2Bfrom%2B1%252C092%2Bhuman%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11632%26rft_id%253Dinfo%253Apmid%252F23128226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23128226&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310434500030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.2" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">November</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1655</span>–<span class="cit-lpage">1664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTAvMzM1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.3" data-doi="10.1093/jhered/92.3.301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balloux</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">EASYPOP (version 1.7): a computer program for population genetics simulations.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DBalloux%26rft.auinit1%253DF.%26rft.volume%253D92%26rft.issue%253D3%26rft.spage%253D301%26rft.epage%253D302%26rft.atitle%253DEASYPOP%2B%2528Version%2B1.7%2529%253A%2BA%2BComputer%2BProgram%2Bfor%2BPopulation%2BGenetics%2BSimulations%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F92.3.301%26rft_id%253Dinfo%253Apmid%252F11447253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/92.3.301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11447253&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169680200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.4" data-doi="10.1111/j.1471-8286.2007.01769.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durand</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forbes</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">François</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Bayesian clustering algorithms ascertaining spatial population structure: a new computer program and a comparison study.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">747</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BNotes%26rft.volume%253D7%26rft.spage%253D747%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2007.01769.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2007.01769.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000249193000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corander</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waldmann</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Bayesian analysis of genetic differentiation between populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCorander%26rft.auinit1%253DJ.%26rft.volume%253D163%26rft.issue%253D1%26rft.spage%253D367%26rft.epage%253D374%26rft.atitle%253DBayesian%2BAnalysis%2Bof%2BGenetic%2BDifferentiation%2BBetween%2BPopulations%26rft_id%253Dinfo%253Apmid%252F12586722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzEvMzY3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zMzU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.6" data-doi="10.1111/j.1365-294X.2010.04656.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cushman</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landguth</span>  <span class="cit-name-given-names">E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Spurious correlations and inference in landscape genetics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3592</span>–<span class="cit-lpage">3602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DCushman%26rft.auinit1%253DS.%2BA.%26rft.volume%253D19%26rft.issue%253D17%26rft.spage%253D3592%26rft.epage%253D3602%26rft.atitle%253DSpurious%2Bcorrelations%2Band%2Binference%2Bin%2Blandscape%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04656.x%26rft_id%253Dinfo%253Apmid%252F20618896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04656.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20618896&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281285200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.7" data-doi="10.1371/journal.pone.0011147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darling</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mau</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perna</span>  <span class="cit-name-given-names">N. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">progressiveMauve: multiple genome alignment with gene gain, loss and rearrangement.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e11147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDarling%26rft.auinit1%253DA.%2BE.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De11147%26rft.epage%253De11147%26rft.atitle%253DprogressiveMauve%253A%2Bmultiple%2Bgenome%2Balignment%2Bwith%2Bgene%2Bgain%252C%2Bloss%2Band%2Brearrangement.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0011147%26rft_id%253Dinfo%253Apmid%252F20593022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0011147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20593022&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dyer</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Population graphs and landscape genetics.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D46%26rft.spage%253D327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.9" data-doi="10.1111/j.1365-294X.2004.02177.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dyer</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nason</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Population graphs: the graph theoretical shape of genetic structure.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1713</span>–<span class="cit-lpage">1727</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DDyer%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D13%26rft.issue%253D7%26rft.spage%253D1713%26rft.epage%253D1727%26rft.atitle%253DPopulation%2BGraphs%253A%2Bthe%2Bgraph%2Btheoretic%2Bshape%2Bof%2Bgenetic%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2004.02177.x%26rft_id%253Dinfo%253Apmid%252F15189198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2004.02177.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15189198&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221933000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.10" data-doi="10.1111/j.1365-294X.2010.04748.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dyer</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nason</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Landscape modelling of gene flow: improved power using conditional distance derived from the topology of population networks.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3746</span>–<span class="cit-lpage">3759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DDyer%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D19%26rft.issue%253D17%26rft.spage%253D3746%26rft.epage%253D3759%26rft.atitle%253DLandscape%2Bmodelling%2Bof%2Bgene%2Bflow%253A%2Bimproved%2Bpower%2Busing%2Bconditional%2Bgenetic%2Bdistance%2Bderived%2Bfrom%2Bthe%2Btopology%2Bof%2Bpopulation%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04748.x%26rft_id%253Dinfo%253Apmid%252F20723052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04748.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20723052&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281285200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-7.10.3359.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source">Introduction to Graphical Modelling</span>, <span class="cit-edition">Ed. 2</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.12" data-doi="10.18637/jss.v048.i04"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Epskamp</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cramer</span>  <span class="cit-name-given-names">A. O. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waldorp</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmittmann</span>  <span class="cit-name-given-names">V. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borsboom</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">qgraph: network visualizations of relationships in psychometric data.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D48%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v048.i04%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v048.i04&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.13" data-doi="10.1111/j.1365-294X.2005.02553.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evanno</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regnaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2611</span>–<span class="cit-lpage">2620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DEvanno%26rft.auinit1%253DG.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D2611%26rft.epage%253D2620%26rft.atitle%253DDetecting%2Bthe%2Bnumber%2Bof%2Bclusters%2Bof%2Bindividuals%2Busing%2Bthe%2Bsoftware%2BSTRUCTURE%253A%2Ba%2Bsimulation%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02553.x%26rft_id%253Dinfo%253Apmid%252F15969739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02553.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15969739&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229961500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1567</span>–<span class="cit-lpage">1587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFalush%26rft.auinit1%253DD.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1567%26rft.epage%253D1587%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%253A%2BLinked%2BLoci%2Band%2BCorrelated%2BAllele%2BFrequencies%26rft_id%253Dinfo%253Apmid%252F12930761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzE1NjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEwLzMzNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.15" data-doi="10.1016/j.physrep.2009.11.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fortunato</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Community detection in graphs.</span> <abbr class="cit-jnl-abbrev">Phys. Rep.</abbr> <span class="cit-vol">486</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhys.%2BRep.%26rft.volume%253D486%26rft.spage%253D75%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.physrep.2009.11.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.physrep.2009.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.16" data-doi="10.1534/genetics.113.160572"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frichot</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mathieu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trouillon</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouchard</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">François</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Fast and efficient estimation of individual ancestry coefficients.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">973</span>–<span class="cit-lpage">983</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.160572%26rft_id%253Dinfo%253Apmid%252F24496008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzQvOTczIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zMzU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.17" data-doi="10.1093/biostatistics/kxm045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sparse inverse covariance estimation with the graphical lasso.</span> <abbr class="cit-jnl-abbrev">Biostatistics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">432</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxm045%26rft_id%253Dinfo%253Apmid%252F18079126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biostatistics/kxm045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18079126&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256977000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.18" data-doi="10.1111/j.1467-9868.2005.00503.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Regularization paths for generalized linear models via coordinate descent.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DFriedman%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D22%26rft.atitle%253DRegularization%2BPaths%2Bfor%2BGeneralized%2BLinear%2BModels%2Bvia%2BCoordinate%2BDescent.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00503.x%26rft_id%253Dinfo%253Apmid%252F20808728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00503.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20808728&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275203200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.19" data-doi="10.1002/spe.4380211102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fruchterman</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reingold</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Graph drawing with force-directed placement.</span> <abbr class="cit-jnl-abbrev">Softw. Pract. Exper.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1129</span>–<span class="cit-lpage">1164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoftw.%2BPract.%2BExper.%26rft.volume%253D21%26rft.spage%253D1129%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fspe.4380211102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/spe.4380211102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.20" data-doi="10.1073/pnas.122653799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Girvan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">M. E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Community structure in social and biological networks.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">7821</span>–<span class="cit-lpage">7826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DGirvan%26rft.auinit1%253DM.%26rft.volume%253D99%26rft.issue%253D12%26rft.spage%253D7821%26rft.epage%253D7826%26rft.atitle%253DCommunity%2Bstructure%2Bin%2Bsocial%2Band%2Bbiological%2Bnetworks%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.122653799%26rft_id%253Dinfo%253Apmid%252F12060727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTkvMTIvNzgyMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTAvMzM1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.21" data-doi="10.1534/genetics.115.182626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greenbaum</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Templeton</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bar-David</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Inference and analysis of population structure using genetic data and network theory.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">1299</span>–<span class="cit-lpage">1312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.182626%26rft_id%253Dinfo%253Apmid%252F26888080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMi80LzEyOTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEwLzMzNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harenberg</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bello</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjeltema</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranshous</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harlalka</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Community detection in large-scale networks: a survey and empirical evaluation.</span> <abbr class="cit-jnl-abbrev">Wiley Interdiscip. Rev. Comput. Stat.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">426</span>–<span class="cit-lpage">439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWiley%2BInterdiscip.%2BRev.%2BComput.%2BStat.%26rft.volume%253D6%26rft.spage%253D426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-7.10.3359.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source"><em>The Elements of Statistical Learning: Data Mining</em>, <em>Inference</em>, <em>and Prediction</em></span>, <span class="cit-edition">Ed. 2</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.24" data-doi="10.1162/153244301753344614"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chickering</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meek</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rounthwaite</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadie</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Dependency networks for inference, collaboration filtering, and data visualization.</span> <abbr class="cit-jnl-abbrev">J. Mach. Learn. Res.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes.%26rft.volume%253D1%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1162%252F153244301753344614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1162/153244301753344614&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-7.10.3359.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hsieh</span>  <span class="cit-name-given-names">C.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sustik</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">I. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ravikumar</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poldrack</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">BIG &amp; QUIC: sparse inverse covariance estimation for a million variables</span>, pp. <span class="cit-fpage">3165</span>–<span class="cit-lpage">3173</span> in <span class="cit-source">Advances in Neural Information Processing Systems</span>, Vol. 26, edited by <span class="cit-ed"><span class="cit-name-surname">Burges</span>  <span class="cit-name-given-names">C. J. C.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Bottou</span>  <span class="cit-name-given-names">L.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Welling</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Ghahramani</span>  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Weinberger</span>  <span class="cit-name-given-names">K. Q.</span></span>. <span class="cit-publ-name">Curran Associates, Inc.</span>, <span class="cit-publ-loc">Red Hook, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.26" data-doi="10.1093/bioinformatics/btm233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1801</span>–<span class="cit-lpage">1806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm233%26rft_id%253Dinfo%253Apmid%252F17485429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17485429&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249248300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.10.3359.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jalali</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ravikumar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vasuki</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanghavi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> On learning discrete graphical models using group-sparse regularization, pp. <span class="cit-fpage">378</span>–<span class="cit-lpage">387</span> in Proceedings of the 14th International Conference on Artificial Intelligence and Statistics (AISTATS). Fort Lauderdale, FL.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.28" data-doi="10.1186/1471-2156-11-94"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jombart</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devillard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balloux</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Discriminant analysis of principal components: a new method for the analysis of genetically structured populations.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DJombart%26rft.auinit1%253DT.%26rft.volume%253D11%26rft.spage%253D94%26rft.epage%253D94%26rft.atitle%253DDiscriminant%2Banalysis%2Bof%2Bprincipal%2Bcomponents%253A%2Ba%2Bnew%2Bmethod%2Bfor%2Bthe%2Banalysis%2Bof%2Bgenetically%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-11-94%26rft_id%253Dinfo%253Apmid%252F20950446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-11-94&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20950446&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.29" data-doi="10.1038/hdy.2010.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jombart</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eggo</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dodd</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balloux</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Reconstructing disease outbreaks from genetic data: a graph approach.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DJombart%26rft.auinit1%253DT.%26rft.volume%253D106%26rft.issue%253D2%26rft.spage%253D383%26rft.epage%253D390%26rft.atitle%253DReconstructing%2Bdisease%2Boutbreaks%2Bfrom%2Bgenetic%2Bdata%253A%2Ba%2Bgraph%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.78%26rft_id%253Dinfo%253Apmid%252F20551981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.78&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20551981&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286375300020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.30" data-doi="10.1103/PhysRevE.80.056117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lancichinetti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fortunato</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Community detection algorithms: a comparative analysis.</span> <abbr class="cit-jnl-abbrev">Phys. Rev. E Stat. Nonlin. Soft Matter Phys.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">056117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysical%2Breview.%2BE%252C%2BStatistical%252C%2Bnonlinear%252C%2Band%2Bsoft%2Bmatter%2Bphysics%26rft.stitle%253DPhys%2BRev%2BE%2BStat%2BNonlin%2BSoft%2BMatter%2BPhys%26rft.aulast%253DLancichinetti%26rft.auinit1%253DA.%26rft.volume%253D80%26rft.issue%253D5%2BPt%2B2%26rft.spage%253D056117%26rft.epage%253D056117%26rft.atitle%253DCommunity%2Bdetection%2Balgorithms%253A%2Ba%2Bcomparative%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1103%252FPhysRevE.80.056117%26rft_id%253Dinfo%253Apmid%252F20365053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1103/PhysRevE.80.056117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20365053&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.31" data-doi="10.1126/science.1153717"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Absher</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Southwick</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casto</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Worldwide human relationships inferred from genome-wide patterns of variation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">1100</span>–<span class="cit-lpage">1104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLi%26rft.auinit1%253DJ.%2BZ.%26rft.volume%253D319%26rft.issue%253D5866%26rft.spage%253D1100%26rft.epage%253D1104%26rft.atitle%253DWorldwide%2BHuman%2BRelationships%2BInferred%2Bfrom%2BGenome-Wide%2BPatterns%2Bof%2BVariation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1153717%26rft_id%253Dinfo%253Apmid%252F18292342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTkvNTg2Ni8xMTAwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zMzU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Gene network reconstruction by integration of prior biological knowledge.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1075</span>–<span class="cit-lpage">1079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D5%26rft.spage%253D1075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-7.10.3359.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Stability approach to regularization selection (StARS) for high dimensional graphical models</span>, pp. <span class="cit-fpage">1432</span>–<span class="cit-lpage">1440</span> in <span class="cit-source">Advances in Neural Information Processing Systems</span>, Vol. 23, edited by <span class="cit-ed"><span class="cit-name-surname">Lafferty</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Shawe-Taylor</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Zemel</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Culotta</span>  <span class="cit-name-given-names">A.</span></span>. <span class="cit-publ-name">Curran Associates, Inc.</span>, <span class="cit-publ-loc">Red Hook, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.34" data-doi="10.1214/009053606000000281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meinshausen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bühlmann</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">High dimensional graphs and variable selection with the LASSO.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1436</span>–<span class="cit-lpage">1462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D34%26rft.spage%253D1436%26rft_id%253Dinfo%253Adoi%252F10.1214%252F009053606000000281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/009053606000000281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000239471300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meinshausen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bühlmann</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Stability selection (with discussion).</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Series B (Statistical Methodology)</abbr>. <span class="cit-vol">72</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSeries%2BB%2B%2528Statistical%2BMethodology%2529%26rft.volume%253D72%26rft.spage%253D417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.36" data-doi="10.7155/jgaa.00124"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pons</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Latapy</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Computing communities in large networks using random walks.</span> <abbr class="cit-jnl-abbrev">J. Graph Algorithms Appl.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">191</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGraph%2BAlgorithms%2BAppl.%26rft.volume%253D10%26rft.spage%253D191%26rft_id%253Dinfo%253Adoi%252F10.7155%252Fjgaa.00124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7155/jgaa.00124&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.37" data-doi="10.1007/s00239-001-0034-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crandall</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The effect of recombination on the accuracy of phylogeny estimation.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">396</span>–<span class="cit-lpage">402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DPosada%26rft.auinit1%253DD.%26rft.volume%253D54%26rft.issue%253D3%26rft.spage%253D396%26rft.epage%253D402%26rft.atitle%253DThe%2Beffect%2Bof%2Brecombination%2Bon%2Bthe%2Baccuracy%2Bof%2Bphylogeny%2Bestimation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-001-0034-9%26rft_id%253Dinfo%253Apmid%252F11847565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-001-0034-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11847565&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173928400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.38" data-doi="10.1038/ng1847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plenge</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinblatt</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shadick</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Principal components analysis corrects for stratification in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">904</span>–<span class="cit-lpage">909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTo%26rft.auinit1%253DM.%2BD.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D904%26rft.epage%253D909%26rft.atitle%253DPrincipal%2Bcomponents%2Banalysis%2Bcorrects%2Bfor%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1847%26rft_id%253Dinfo%253Apmid%252F16862161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16862161&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.39" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zMzU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.40" data-doi="10.1016/j.tree.2005.04.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Proulx</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Promislow</span>  <span class="cit-name-given-names">D. E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Network thinking in ecology and evolution.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">345</span>–<span class="cit-lpage">353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DProulx%26rft.auinit1%253DS.%2BR.%26rft.volume%253D20%26rft.issue%253D6%26rft.spage%253D345%26rft.epage%253D353%26rft.atitle%253DNetwork%2Bthinking%2Bin%2Becology%2Band%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2005.04.004%26rft_id%253Dinfo%253Apmid%252F16701391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2005.04.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16701391&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229887800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Puechmaille</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The program STRUCTURE does not reliably recover the correct population structure when sampling is uneven: subsampling and new estimators alleviate the problem.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">608</span>–<span class="cit-lpage">627</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.42" data-doi="10.1093/gbe/evt109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pyhäjärvi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hufford</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mezmouk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Complex patterns of local adaptation in teosinte.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1594</span>–<span class="cit-lpage">1609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevt109%26rft_id%253Dinfo%253Apmid%252F23902747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evt109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23902747&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.43" data-doi="10.1534/genetics.114.164350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raj</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">fastSTRUCTURE: variational inference of population structure in large SNP data sets.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164350%26rft_id%253Dinfo%253Apmid%252F24700103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzIvNTczIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zMzU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-7.10.3359.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raskutti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wainwright</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ravikumar</span>  <span class="cit-name-given-names">P. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Model selection in Gaussian graphical models: high-dimensional consistency of l<sub>1</sub>-regularized MLE</span>, pp. <span class="cit-fpage">1329</span>–<span class="cit-lpage">1336</span> in <span class="cit-source">Advances in Neural Information Processing Systems</span>, Vol. 21, edited by <span class="cit-ed"><span class="cit-name-surname">Koller</span>  <span class="cit-name-given-names">D.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Schuurmans</span>  <span class="cit-name-given-names">D.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Bengio</span>  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Bottou</span>  <span class="cit-name-given-names">L.</span></span>. <span class="cit-publ-name">Curran Associates, Inc.</span>, <span class="cit-publ-loc">Red Hook, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.45" data-doi="10.1126/science.1078311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cann</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic structure of human populations.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">298</span>: <span class="cit-fpage">2381</span>–<span class="cit-lpage">2385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRosenberg%26rft.auinit1%253DN.%2BA.%26rft.volume%253D298%26rft.issue%253D5602%26rft.spage%253D2381%26rft.epage%253D2385%26rft.atitle%253DGenetic%2BStructure%2Bof%2BHuman%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1078311%26rft_id%253Dinfo%253Apmid%252F12493913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTgvNTYwMi8yMzgxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zMzU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.46" data-doi="10.1101/gr.180190.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salipante</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roach</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitzman</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stackhouse</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Large-scale genomic sequencing of extraintestinal pathogenic <em>Escherichia coli</em> strains.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.180190.114%26rft_id%253Dinfo%253Apmid%252F25373147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzEvMTE5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zMzU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.47" data-doi="10.1093/molbev/mst197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filipski</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">MEGA6: molecular evolutionary genetics analysis version 6.0.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2725</span>–<span class="cit-lpage">2729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst197%26rft_id%253Dinfo%253Apmid%252F24132122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24132122&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327793000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.48" data-doi="10.1002/gepi.20064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Risch</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Estimation of individual admixture: analytical and study design considerations.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DTang%26rft.auinit1%253DH.%26rft.volume%253D28%26rft.issue%253D4%26rft.spage%253D289%26rft.epage%253D301%26rft.atitle%253DEstimation%2Bof%2Bindividual%2Badmixture%253A%2Banalytical%2Band%2Bstudy%2Bdesign%2Bconsiderations.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20064%26rft_id%253Dinfo%253Apmid%252F15712363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15712363&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228573700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Regression shrinkage and selection via the LASSO.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D58%26rft.spage%253D267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.50" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.51" data-doi="10.1214/08-AOS646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">High-dimensional variable selection.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">2178</span>–<span class="cit-lpage">2201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAnn%2BStat%26rft.aulast%253DWasserman%26rft.auinit1%253DL.%26rft.volume%253D37%26rft.issue%253D5A%26rft.spage%253D2178%26rft.epage%253D2201%26rft.atitle%253DHIGH%2BDIMENSIONAL%2BVARIABLE%2BSELECTION.%26rft_id%253Dinfo%253Adoi%252F10.1214%252F08-AOS646%26rft_id%253Dinfo%253Apmid%252F19784398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/08-AOS646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19784398&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.52" data-doi="10.1186/s13323-014-0018-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wollstein</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lao</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Detecting individual ancestry in the human genome.</span> <abbr class="cit-jnl-abbrev">Investig. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInvestig.%2BGenet.%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13323-014-0018-3%26rft_id%253Dinfo%253Apmid%252F25750707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13323-014-0018-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25750707&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.53" data-doi="10.1038/ng1702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bi</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A general framework for weighted gene coexpression network analysis.</span> <abbr class="cit-jnl-abbrev">Stat. Appl. Genet. Mol. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BAppl.%2BGenet.%2BMol.%2BBiol.%26rft.volume%253D4%26rft.spage%253D17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3359.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lafferty</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The huge package for high-dimensional undirected graph estimation in R.</span> <abbr class="cit-jnl-abbrev">J. Mach. Learn. Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1059</span>–<span class="cit-lpage">1062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes.%26rft.volume%253D13%26rft.spage%253D1059%26rft_id%253Dinfo%253Apmid%252F26834510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26834510&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-book" id="cit-7.10.3359.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zumel</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mount</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">Practical Data Science with R</span>, <span class="cit-edition">Ed. 1</span>. <span class="cit-publ-name">Manning Publications Co.</span>, <span class="cit-publ-loc">Greenwich, CT</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/10/3359.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/10/3349" title="KinFin: Software for Taxon-Aware Analysis of Clustered Protein Sequences" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/10/3379" title="Genome-Scale Genetic Interactions and Cell Imaging Confirm Cytokinesis as Deleterious to Transient Topoisomerase II Deficiency in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/10"><span>Volume 7 Issue 10, October 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/10.cover-source.jpg?itok=lurMO2QE" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3359" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-p-ho9o7Ok2LSTAvmQ6OgKrfq_0LzNBvbdPGHoVF-sQc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/10/3359" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/10/3359" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/10/3359" class="active" data-icon-position="" data-hide-link-title="0">CONE: Community Oriented Network Estimation Is a Versatile Framework for Inferring Population Structure in Large-Scale Sequencing Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/486778" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-PXr_teSk2uyLbtSzkLgZsnBmjRcru-uwie7VlkiBJRc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/10/3359.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/10/3359" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-c0vKAW5pPGGPTS24eVooet82JmKpuL1mibcPSdZSUDE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/10/3359" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/10/3359.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486778" id="node486778--2" data-pisa="ggg;7/10/3359" data-pisa-master="ggg;g3.117.300131" data-apath="/ggg/7/10/3359.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">CONE: Community Oriented Network Estimation Is a Versatile Framework for Inferring Population Structure in Large-Scale Sequencing Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9074-7420" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Markku O. Kuismin</span>, <span class="highwire-citation-author" data-delta="1">Jon Ahlinder</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-2808-2768" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mikko J. Sillanpӓӓ</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3359-3377;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300131 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486778--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Markku O. Kuismin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Mathematical Sciences, University of Oulu, FI-90014, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Markku%2BO.%2BKuismin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuismin%20MO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarkku%2BO.%2BKuismin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9074-7420" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Markku O. Kuismin</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jon Ahlinder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Swedish Defense Research Agency, SE-906 21 Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jon%2BAhlinder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ahlinder%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJon%2BAhlinder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mikko J. Sillanpӓӓ </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Mathematical Sciences, University of Oulu, FI-90014, Finland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biocenter Oulu, FI-90014, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mikko%2BJ.%2BSillanp%D3%93%D3%93%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sillanp%D3%93%D3%93%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMikko%2BJ.%2BSillanp%25D3%2593%25D3%2593%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2808-2768" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mikko J. Sillanpӓӓ</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mikko.sillanpaa@oulu.fi" class="" data-icon-position="" data-hide-link-title="0">mikko.sillanpaa@oulu.fi</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/10/3359" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/10/3359&amp;title=CONE%3A%20Community%20Oriented%20Network%20Estimation%20Is%20a%20Versatile%20Framework%20for%20Inferring%20Population%20Structure%20in%20Large-Scale%20Sequencing%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/10/3359&amp;title=CONE%3A%20Community%20Oriented%20Network%20Estimation%20Is%20a%20Versatile%20Framework%20for%20Inferring%20Population%20Structure%20in%20Large-Scale%20Sequencing%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/10/3359&amp;title=CONE%3A%20Community%20Oriented%20Network%20Estimation%20Is%20a%20Versatile%20Framework%20for%20Inferring%20Population%20Structure%20in%20Large-Scale%20Sequencing%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/10/3359&amp;text=CONE%3A%20Community%20Oriented%20Network%20Estimation%20Is%20a%20Versatile%20Framework%20for%20Inferring%20Population%20Structure%20in%20Large-Scale%20Sequencing%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/10/3359&amp;title=CONE%3A%20Community%20Oriented%20Network%20Estimation%20Is%20a%20Versatile%20Framework%20for%20Inferring%20Population%20Structure%20in%20Large-Scale%20Sequencing%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/10/3359&amp;t=CONE%3A%20Community%20Oriented%20Network%20Estimation%20Is%20a%20Versatile%20Framework%20for%20Inferring%20Population%20Structure%20in%20Large-Scale%20Sequencing%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/10/3359&amp;title=CONE%3A%20Community%20Oriented%20Network%20Estimation%20Is%20a%20Versatile%20Framework%20for%20Inferring%20Population%20Structure%20in%20Large-Scale%20Sequencing%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/10/3359&amp;title=CONE%3A%20Community%20Oriented%20Network%20Estimation%20Is%20a%20Versatile%20Framework%20for%20Inferring%20Population%20Structure%20in%20Large-Scale%20Sequencing%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/486778/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486778" id="node486778--3" data-pisa="ggg;7/10/3359" data-pisa-master="ggg;g3.117.300131" data-apath="/ggg/7/10/3359.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">CONE: Community Oriented Network Estimation Is a Versatile Framework for Inferring Population Structure in Large-Scale Sequencing Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9074-7420" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Markku O. Kuismin</span>, <span class="highwire-citation-author" data-delta="1">Jon Ahlinder</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-2808-2768" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mikko J. Sillanpӓӓ</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3359-3377;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300131 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486778--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Markku O. Kuismin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Mathematical Sciences, University of Oulu, FI-90014, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Markku%2BO.%2BKuismin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuismin%20MO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarkku%2BO.%2BKuismin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9074-7420" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Markku O. Kuismin</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jon Ahlinder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Swedish Defense Research Agency, SE-906 21 Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jon%2BAhlinder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ahlinder%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJon%2BAhlinder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mikko J. Sillanpӓӓ </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Mathematical Sciences, University of Oulu, FI-90014, Finland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biocenter Oulu, FI-90014, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mikko%2BJ.%2BSillanp%D3%93%D3%93%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sillanp%D3%93%D3%93%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMikko%2BJ.%2BSillanp%25D3%2593%25D3%2593%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2808-2768" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mikko J. Sillanpӓӓ</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mikko.sillanpaa@oulu.fi" class="" data-icon-position="" data-hide-link-title="0">mikko.sillanpaa@oulu.fi</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3359&amp;rft.epage=3377&amp;rft.atitle=CONE%3A%20Community%20Oriented%20Network%20Estimation%20Is%20a%20Versatile%20Framework%20for%20Inferring%20Population%20Structure%20in%20Large-Scale%20Sequencing%20Data&amp;rft.volume=7&amp;rft.issue=10&amp;rft.date=2017-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kuismin%2C+Markku+O.&amp;rft.au=Ahlinder%2C+Jon&amp;rft.au=Sillanp%D3%93%D3%93%2C+Mikko+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/486778/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/486778/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/486778/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/486778/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/486778/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/486778/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/486778/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/486778/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/486778/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/486778/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/486778/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/486778/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/10/3359" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/10/3359.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/10/3359.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/10/3359.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/10/3359.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
