<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/10/3427" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/10/3427.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/10/3427.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/10/3427.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/10/3427.atom" />
<meta name="HW.pisa" content="ggg;7/10/3427" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="High-Resolution Maps of Mouse Reference Populations" />
<meta name="DC.Identifier" content="10.1534/g3.117.300188" />
<meta name="DC.Date" content="2017-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Simecek et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genetic reference panels are widely used to map complex, quantitative traits in model organisms. We have generated new high-resolution genetic maps of 259 mouse inbred strains from recombinant inbred strain panels (C57BL/6J × DBA/2J, ILS/IbgTejJ × ISS/IbgTejJ, and C57BL/6J × A/J) and chromosome substitution strain panels (C57BL/6J-Chr#&lt;A/J&gt;, C57BL/6J-Chr#&lt;PWD/Ph&gt;, and C57BL/6J-Chr#&lt;MSM/Ms&gt;). We genotyped all samples using the Affymetrix Mouse Diversity Array with an average intermarker spacing of 4.3 kb. The new genetic maps provide increased precision in the localization of recombination breakpoints compared to the previous maps. Although the strains were presumed to be fully inbred, we found residual heterozygosity in 40% of individual mice from five of the six panels. We also identified de novo deletions and duplications, in homozygous or heterozygous state, ranging in size from 21 kb to 8.4 Mb. Almost two-thirds (46 out of 76) of these deletions overlap exons of protein coding genes and may have phenotypic consequences. Twenty-nine putative gene conversions were identified in the chromosome substitution strains. We find that gene conversions are more likely to occur in regions where the homologous chromosomes are more similar. The raw genotyping data and genetic maps of these strain panels are available at &lt;http://churchill-lab.jax.org/website/MDA&gt;." />
<meta name="DC.Contributor" content="Petr Simecek" />
<meta name="DC.Contributor" content="Jiri Forejt" />
<meta name="DC.Contributor" content="Robert W. Williams" />
<meta name="DC.Contributor" content="Toshihiko Shiroishi" />
<meta name="DC.Contributor" content="Toyoyuki Takada" />
<meta name="DC.Contributor" content="Lu Lu" />
<meta name="DC.Contributor" content="Thomas E. Johnson" />
<meta name="DC.Contributor" content="Beth Bennett" />
<meta name="DC.Contributor" content="Christian F. Deschepper" />
<meta name="DC.Contributor" content="Marie-Pier Scott-Boyer" />
<meta name="DC.Contributor" content="Fernando Pardo-Manuel de Villena" />
<meta name="DC.Contributor" content="Gary A. Churchill" />
<meta name="article:published_time" content="2017-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="High-Resolution Maps of Mouse Reference Populations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genetic reference panels are widely used to map complex, quantitative traits in model organisms. We have generated new high-resolution genetic maps of 259 mouse inbred strains from recombinant inbred strain panels (C57BL/6J × DBA/2J, ILS/IbgTejJ × ISS/IbgTejJ, and C57BL/6J × A/J) and chromosome substitution strain panels (C57BL/6J-Chr#&amp;lt;A/J&amp;gt;, C57BL/6J-Chr#&amp;lt;PWD/Ph&amp;gt;, and C57BL/6J-Chr#&amp;lt;MSM/Ms&amp;gt;). We genotyped all samples using the Affymetrix Mouse Diversity Array with an average intermarker spacing of 4.3&amp;nbsp;kb. The new genetic maps provide increased precision in the localization of recombination breakpoints compared to the previous maps. Although the strains were presumed to be fully inbred, we found residual heterozygosity in 40% of individual mice from five of the six panels. We also identified &lt;i&gt;de novo&lt;/i&gt; deletions and duplications, in homozygous or heterozygous state, ranging in size from 21&amp;nbsp;kb to 8.4&amp;nbsp;Mb. Almost two-thirds (46 out of 76) of these deletions overlap exons of protein coding genes and may have phenotypic consequences. Twenty-nine putative gene conversions were identified in the chromosome substitution strains. We find that gene conversions are more likely to occur in regions where the homologous chromosomes are more similar. The raw genotyping data and genetic maps of these strain panels are available at http://churchill-lab.jax.org/website/MDA.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/10/01" />
<meta name="citation_mjid" content="ggg;7/10/3427" />
<meta name="citation_id" content="7/10/3427" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/10/3427" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/10/3427.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/10/3427.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/10/3427.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300188" />
<meta name="citation_pmid" content="28839117" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3427" />
<meta name="citation_lastpage" content="3434" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Petr Simecek" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author_institution" content="Division Biotechnology and Biomedicine Centre of the Academy of Sciences, Institute of Molecular Genetics of the Academy of Sciences of the Czech Republic, Vestec, Prague 4, 142 20, Czech Republic" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2922-7183" />
<meta name="citation_author" content="Jiri Forejt" />
<meta name="citation_author_institution" content="Division Biotechnology and Biomedicine Centre of the Academy of Sciences, Institute of Molecular Genetics of the Academy of Sciences of the Czech Republic, Vestec, Prague 4, 142 20, Czech Republic" />
<meta name="citation_author" content="Robert W. Williams" />
<meta name="citation_author_institution" content="Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee 38163" />
<meta name="citation_author" content="Toshihiko Shiroishi" />
<meta name="citation_author_institution" content="National Institute of Genetics, Mishima, 411-8540, Japan" />
<meta name="citation_author" content="Toyoyuki Takada" />
<meta name="citation_author_institution" content="National Institute of Genetics, Mishima, 411-8540, Japan" />
<meta name="citation_author" content="Lu Lu" />
<meta name="citation_author_institution" content="Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee 38163" />
<meta name="citation_author" content="Thomas E. Johnson" />
<meta name="citation_author_institution" content="University of Colorado at Boulder, Colorado 80309" />
<meta name="citation_author" content="Beth Bennett" />
<meta name="citation_author_institution" content="University of Colorado at Boulder, Colorado 80309" />
<meta name="citation_author" content="Christian F. Deschepper" />
<meta name="citation_author_institution" content="Institut de Recherches Cliniques, Montreal, Quebec, H2W 1R7, Canada" />
<meta name="citation_author" content="Marie-Pier Scott-Boyer" />
<meta name="citation_author_institution" content="Institut de Recherches Cliniques, Montreal, Quebec, H2W 1R7, Canada" />
<meta name="citation_author" content="Fernando Pardo-Manuel de Villena" />
<meta name="citation_author_institution" content="Institute for Behavioral Genetics, Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264" />
<meta name="citation_author_email" content="fernando@med.unc.edu" />
<meta name="citation_author_email" content="gary.churchill@jax.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5738-5795" />
<meta name="citation_author" content="Gary A. Churchill" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author_email" content="fernando@med.unc.edu" />
<meta name="citation_author_email" content="gary.churchill@jax.org" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. A. Beck;citation_author=S. Lloyd;citation_author=M. Hafezparast;citation_author=M. Lennon-Pierce;citation_author=J. T. Eppig;citation_title=Genealogies of mouse inbred strains.;citation_pages=23-25;citation_volume=24;citation_year=2000;citation_issue=1;citation_pmid=10615122;citation_doi=10.1038/71641" />
<meta name="citation_reference" content="citation_journal_title=Behavior genetics;citation_journal_abbrev=Behav Genet;citation_author=J. Belknap;citation_title=Effect of within-strain sample size on QTL detection and mapping using recombinant inbred mouse strains.;citation_pages=29-38;citation_volume=28;citation_year=1998;citation_issue=1;citation_pmid=9573644;citation_doi=10.1023/A:1021404714631" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. W. Broman;citation_title=The genomes of recombinant inbred lines.;citation_volume=169;citation_year=2005;citation_pmid=15545647;citation_doi=10.1534/genetics.104.035212" />
<meta name="citation_reference" content="citation_title=A Guide to QTL Mapping with R/qtl;citation_author=K. W. Broman;citation_author=S. Sen;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. A. Buchner;citation_author=J. H. Nadeau;citation_title=Contrasting genetic architectures in different mouse reference populations used for studying complex traits.;citation_volume=25;citation_year=2015;citation_pmid=25953951;citation_doi=10.1101/gr.187450.114" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=E. J. Chesler;citation_author=D. M. Gatti;citation_author=A. P. Morgan;citation_author=M. Strobel;citation_author=L. Trepanier;citation_title=Diversity outbred mice at 21: maintaining allelic variation in the face of selection.;citation_pages=3893-3902;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. T. Chinwalla;citation_author=L. L. Cook;citation_author=K. D. Delehaunty;citation_author=G. A. Fewell;citation_author=L. A. Fulton;citation_title=Initial sequencing and comparative analysis of the mouse genome.;citation_pages=520-562;citation_volume=420;citation_year=2002;citation_issue=6915;citation_pmid=12466850;citation_doi=10.1038/nature01262" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=G. A. Churchill;citation_author=D. M. Gatti;citation_author=S. C. Munger;citation_author=K. L. Svenson;citation_title=The diversity outbred mouse population.;citation_pages=713-718;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22892839;citation_doi=10.1007/s00335-012-9414-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_title=The genome architecture of the Collaborative Cross mouse genetic reference population.;citation_volume=190;citation_year=2012;citation_pmid=22345608;citation_doi=10.1534/genetics.111.132639" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. P. Didion;citation_author=H. Yang;citation_author=K. Sheppard;citation_author=C. P. Fu;citation_author=L. McMillan;citation_title=Discovery of novel variants in genotyping arrays improves genotype retention and reduces ascertainment bias.;citation_pages=34-34;citation_volume=13;citation_year=2012;citation_pmid=22260749;citation_doi=10.1186/1471-2164-13-34" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Gregorova;citation_author=P. Divina;citation_author=R. Storchova;citation_author=Z. Trachtulec;citation_author=V. Fotopulosova;citation_title=Mouse consomic strains: exploiting genetic divergence between Mus m. musculus and Mus m. domesticus subspecies.;citation_volume=18;citation_year=2008;citation_pmid=18256238;citation_doi=10.1101/gr.7160508" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Acoust. Speech Signal Process.;citation_author=B.-H. Juang;citation_author=L. R. Rabiner;citation_title=The segmental K-means algorithm for estimating parameters of hidden Markov models.;citation_pages=1639-1641;citation_volume=38;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. M. Keane;citation_author=L. Goodstadt;citation_author=P. Danecek;citation_author=M. A. White;citation_author=K. Wong;citation_title=Mouse genomic variation and its effect on phenotypes and gene regulation.;citation_pages=289-294;citation_volume=477;citation_year=2011;citation_issue=7364;citation_pmid=21921910;citation_doi=10.1038/nature10413" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=V. Kumar;citation_author=K. Kim;citation_author=C. Joseph;citation_author=S. Kourrich;citation_author=S.-H. Yoo;citation_title=C57BL/6N Mutation in Cytoplasmic FMRP interacting protein 2 Regulates Cocaine Response;citation_pages=1508-1512;citation_volume=342;citation_year=2013;citation_issue=6165;citation_pmid=24357318;citation_doi=10.1126/science.1245503" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. Liu;citation_author=H. Song;citation_author=D. Liu;citation_author=T. Zuo;citation_author=F. Lu;citation_title=Extensive recombination due to heteroduplexes generates large amounts of artificial gene fragments during PCR.;citation_pages=e106658;citation_volume=9;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. P. Morgan;citation_author=J. P. Didion;citation_author=A. G. Doran;citation_author=J. M. Holt;citation_author=L. McMillan;citation_title=Whole genome sequence of two wild-derived Mus musculus domesticus inbred strains, LEWES/EiJ and ZALENDE/EiJ, with different diploid numbers.;citation_pages=4211-4216;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. H. Nadeau;citation_author=J. B. Singer;citation_author=A. Matin;citation_author=E. S. Lander;citation_title=Analysing complex genetic traits with chromosome substitution strains.;citation_pages=221-225;citation_volume=24;citation_year=2000;citation_issue=3;citation_pmid=10700173;citation_doi=10.1038/73427" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=J. H. Nadeau;citation_author=J. Forejt;citation_author=T. Takada;citation_author=T. Shiroishi;citation_title=Chromosome substitution strains: gene discovery, functional analysis, and systems studies.;citation_pages=693-705;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22961226;citation_doi=10.1007/s00335-012-9426-y" />
<meta name="citation_reference" content="citation_journal_title=Journal of Leukocyte Biology;citation_journal_abbrev=J. Leukoc. Biol.;citation_author=M. N. Nesbitt;citation_author=E. Skamene;citation_title=Recombinant inbred mouse strains derived from A/J and C57BL/6J: a tool for the study of genetic mechanisms in host resistance to infection and malignancy;citation_pages=357-364;citation_volume=36;citation_year=1984;citation_issue=3;citation_pmid=6592283" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=J. L. Peirce;citation_author=L. Lu;citation_author=J. Gu;citation_author=L. M. Silver;citation_author=R. W. Williams;citation_title=A new set of BXD recombinant inbred lines from advanced intercross populations in mice.;citation_pages=7-7;citation_volume=5;citation_year=2004;citation_pmid=15117419;citation_doi=10.1186/1471-2156-5-7" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=S. Shifman;citation_author=J. T. Bell;citation_author=R. R. Copley;citation_author=M. S. Taylor;citation_author=R. W. Williams;citation_title=A high-resolution single nucleotide polymorphism genetic map of the mouse genome.;citation_pages=e395-e395;citation_volume=4;citation_year=2006;citation_issue=12;citation_pmid=17105354;citation_doi=10.1371/journal.pbio.0040395" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. R. Shorter;citation_author=F. Odet;citation_author=D. L. Aylor;citation_author=W. Pan;citation_author=C.-Y. Kao;citation_title=Male infertility is responsible for nearly half of the extinction observed in the mouse Collaborative Cross.;citation_volume=206;citation_year=2017;citation_pmid=28592496;citation_doi=10.1534/genetics.116.199596" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=P. Simecek;citation_author=G. A. Churchill;citation_author=H. Yang;citation_author=L. B. Rowe;citation_author=L. Herberg;citation_title=Genetic analysis of substrain divergence in non-obese diabetic (NOD) mice.;citation_pages=771-775;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Srivastava;citation_author=A. P. Morgan;citation_author=M. L. Najarian;citation_author=V. K. Sarsani;citation_author=J. S. Sigmon;citation_title=Genomes of the mouse Collaborative Cross.;citation_volume=206;citation_year=2017;citation_pmid=28592495;citation_doi=10.1534/genetics.116.198838" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=R. A. Taft;citation_author=M. Davisson;citation_author=M. V. Wiles;citation_title=Know thy mouse.;citation_pages=649-653;citation_volume=22;citation_year=2006;citation_issue=12;citation_pmid=17007958;citation_doi=10.1016/j.tig.2006.09.010" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=T. Takada;citation_author=A. Mita;citation_author=A. Maeno;citation_author=T. Sakai;citation_author=H. Shitara;citation_title=Mouse inter-subspecific consomic strains for genetic dissection of quantitative complex traits.;citation_volume=18;citation_year=2008;citation_pmid=18256237;citation_doi=10.1101/gr.7175308" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=A. Taudt;citation_author=M. Colomé-Tatché;citation_author=F. Johannes;citation_title=Genetic sources of population epigenomic variation.;citation_pages=319-332;citation_volume=17;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. A. Taylor;citation_author=D. W. Bailey;citation_author=M. Cherry;citation_author=R. Riblet;citation_author=M. Weigert;citation_title=Genes for immunoglobulin heavy chain and serum prealbumin protein are linked in mouse.;citation_pages=644-646;citation_volume=256;citation_year=1975;citation_issue=5519;citation_pmid=807855;citation_doi=10.1038/256644a0" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=B. A. Taylor;citation_author=C. Wnek;citation_author=B. S. Kotlus;citation_author=N. Roemer;citation_author=T. MacTaggart;citation_title=Genotyping new BXD recombinant inbred mouse strains and comparison of BXD and consensus maps.;citation_pages=335-348;citation_volume=10;citation_year=1999;citation_issue=4;citation_pmid=10087289;citation_doi=10.1007/s003359900998" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=C. E. Welsh;citation_author=L. McMillan;citation_title=Accelerating the inbreeding of multi-parental recombinant inbred lines generated by sibling matings.;citation_pages=191-198;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=R. W. Williams;citation_author=J. Gu;citation_author=S. Qi;citation_author=L. Lu;citation_title=The genetic structure of recombinant inbred mice: high-resolution consensus maps for complex trait analysis.;citation_pages=RESEARCH0046-RESEARCH0046;citation_volume=2;citation_year=2001;citation_issue=11;citation_pmid=11737945;citation_doi=10.1186/gb-2001-2-11-research0046" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=R. W. Williams;citation_author=B. Bennett;citation_author=L. Lu;citation_author=J. Gu;citation_author=J. C. DeFries;citation_title=Genetic structure of the LXS panel of recombinant inbred mouse strains: a powerful resource for complex trait analysis.;citation_pages=637-647;citation_volume=15;citation_year=2004;citation_issue=8;citation_pmid=15457343;citation_doi=10.1007/s00335-004-2380-6" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=H. Yang;citation_author=Y. Ding;citation_author=L. N. Hutchins;citation_author=J. Szatkiewicz;citation_author=T. A. Bell;citation_title=A customized and versatile high-density genotyping array for the mouse.;citation_pages=663-666;citation_volume=6;citation_year=2009;citation_issue=9;citation_pmid=19668205;citation_doi=10.1038/nmeth.1359" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Yang;citation_author=J. R. Wang;citation_author=J. P. Didion;citation_author=R. J. Buus;citation_author=T. A. Bell;citation_title=Subspecific origin and haplotype diversity in the laboratory mouse.;citation_pages=648-655;citation_volume=43;citation_year=2011;citation_issue=7;citation_pmid=21623374;citation_doi=10.1038/ng.847" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="High-Resolution Maps of Mouse Reference Populations" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
<meta name="twitter:description" content="Genetic reference panels are widely used to map complex, quantitative traits in model organisms. We have generated new high-resolution genetic maps of 259 mouse inbred strains from recombinant inbred strain panels (C57BL/6J × DBA/2J, ILS/IbgTejJ × ISS/IbgTejJ, and C57BL/6J × A/J) and chromosome substitution strain panels (C57BL/6J-Chr#&lt;A/J&gt;, C57BL/6J-Chr#&lt;PWD/Ph&gt;, and C57BL/6J-Chr#&lt;MSM/Ms&gt;). We genotyped all samples using the Affymetrix Mouse Diversity Array with an average intermarker spacing of 4.3 kb. The new genetic maps provide increased precision in the localization of recombination breakpoints compared to the previous maps. Although the strains were presumed to be fully inbred, we found residual heterozygosity in 40% of individual mice from five of the six panels. We also identified de novo deletions and duplications, in homozygous or heterozygous state, ranging in size from 21 kb to 8.4 Mb. Almost two-thirds (46 out of 76) of these deletions overlap exons of protein coding genes and may have phenotypic consequences. Twenty-nine putative gene conversions were identified in the chromosome substitution strains. We find that gene conversions are more likely to occur in regions where the homologous chromosomes are more similar. The raw genotyping data and genetic maps of these strain panels are available at &lt;http://churchill-lab.jax.org/website/MDA&gt;." />
<meta name="og-title" property="og:title" content="High-Resolution Maps of Mouse Reference Populations" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/10/3427" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genetic reference panels are widely used to map complex, quantitative traits in model organisms. We have generated new high-resolution genetic maps of 259 mouse inbred strains from recombinant inbred strain panels (C57BL/6J × DBA/2J, ILS/IbgTejJ × ISS/IbgTejJ, and C57BL/6J × A/J) and chromosome substitution strain panels (C57BL/6J-Chr#&lt;A/J&gt;, C57BL/6J-Chr#&lt;PWD/Ph&gt;, and C57BL/6J-Chr#&lt;MSM/Ms&gt;). We genotyped all samples using the Affymetrix Mouse Diversity Array with an average intermarker spacing of 4.3 kb. The new genetic maps provide increased precision in the localization of recombination breakpoints compared to the previous maps. Although the strains were presumed to be fully inbred, we found residual heterozygosity in 40% of individual mice from five of the six panels. We also identified de novo deletions and duplications, in homozygous or heterozygous state, ranging in size from 21 kb to 8.4 Mb. Almost two-thirds (46 out of 76) of these deletions overlap exons of protein coding genes and may have phenotypic consequences. Twenty-nine putative gene conversions were identified in the chromosome substitution strains. We find that gene conversions are more likely to occur in regions where the homologous chromosomes are more similar. The raw genotyping data and genetic maps of these strain panels are available at &lt;http://churchill-lab.jax.org/website/MDA&gt;." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
    <title>High-Resolution Maps of Mouse Reference Populations | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__xRFCM2RdmYuTElvkVNViIiYmtgne5CdsO2p72JRsBog__5nfCKIBebvFfARteH5Scn-kp3rXvzgexaRVofFNQV8g__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"KkHSLcQp2OYPPOrsnEihzhyhNtpG3EV021QdzpHCcNM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"486770","apath":"\/ggg\/7\/10\/3427.atom","pisa":"ggg;7\/10\/3427","ac":{"\/ggg\/7\/10\/3427.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/10\/3427.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/10\/3427"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/10\/3427"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/10\/3427":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224VRRmMWUgH7lU15QFzw1KO\\\\\\\/kxZyUknW0QEl1UD9xsLYufXDAgz1zkJptW9oOeeI\\\\\\\/LYOq72AWTuUlUTyKSQEiGWdwtmew5UDvBzxBEZy\\\\\\\/srHeQTBDX+wdpcddcgBqeii7aLjEeXkT72SAFKxPUd9hkQSDxk4FToKiPLPeHELkbEt50X67WEko8BHrFEvBPcEBX0GzIyW\\\\\\\/fLnvzwJAV5Mfa+GOpUFg4bIl1eZi6bUqSH5qkQXI1+rfMaZ0LOBLgocEdhY955rwH8vLBz\\\\\\\/TkIhCsH3W+LoyVntlLUtqiH6R1Y+99xp4qkRBru9Uc0ZEOU8Dm3NRNoSu\\\\\\\/ojM62YgyAqO2D9JyBAzFVo1mWObv7hk1M\\\\\\\/gKSJI13fkK9kRRMn7LKOr7IByR1zxgBwQuiL1z3PqJghMqjO9BXN+nXJo0vHenEoEW+NhWM2mbbRv7kFrcyrgMv+kfz3WOVkA2inNlMQ6y59LhyhFZNUoRLpm2c2uSB+IWp1YgYItZ3AaT9O\\\\\\\/XgRCbfBEbsJkiosdZuKzJEXWBBxkoZsVRpLHE5l+PnamiZZ75l2sSmi7xi3ivw9j2dPSaY8IE+3zYromCLsxHPtYVzEirPgOC+5vLm8FudPLXNYDQRf9aH5aasDxocNiyCz5HO6j10qevwinEs8jJ7o+jzVl2MXmIb37QFAtGhYhbAD6DkkxaZSG\\\\\\\/YQVABBbE+S\\\\\\\/ODc0XCGLNfl0fuMVUuRS5T2iW4LI34Q7zR2Ezq0BIRnLcjPdLcW3lDvHfJWhM1MvnC38fvqtzUYGCeQYyEvLpSYEwhNVUB+UT3y6u4b5TwwvdFgvQSSxM2YHEHtwPVO8VKqlkuLIi9islpyUyiKBlwNYQAYqYvpzd+3UnYSWdkuC0D12VedgitbXkRRnJ+gpdt2HxoBHfcdXJ7\\\\\\\/mgUDDtO5CLp5V90X8CNRSYl6uzi1PdM6PKm580npmI8haCDSo4Ylrr7zA4RdWdD\\\\\\\/yctqyiW9fZQwOqN34KYG267x9RAUbY69ArywS9QUk3qzEJGg7W9EuFuHDfPo6qoyxIxjcS2ZLtvCFik\\\\\\\/09yzYD7bfYWvIPlvZR9+COIxBhquwLS8OGIVHCQEPbGoj9wXstZ3F\\\\\\\/DR9cgQqyWU3UdJTX\\\\\\\/bvTn8RuCuZZ2PNoU+GQKJN6Hb16LrarsONPFR0Sz5DFBZBB\\\\\\\/8BidQGsW4agDcl1xNvqJeZjxoVDLs7XroVkqU5IOZYXkHMHu\\\\\\\/z+BP8vvIvesy5KGcju3B\\\\\\\/OUvXPz6jZ9lVmxk6uKJC7FW3J3I2vX9NwoTO\\\u0022,\\\u0022iv\\\u0022:\\\u0022z42w5E3vLlrkL9Mr3hMYSw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225d2e45155c1193af65e9e2b36fe97e83d84e73fd7c9a9d0de4f3a6687c7345d5\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GPGdYpgGW9kGu4WZoN73XrV04kMrBm3uAVD\\\\\\\/i\\\\\\\/xsGgkxiJ2sj4VLeTl9aMTZWm5kDtE2nQSk5p3reyp+3JJl0Lstffo5eO4GbR\\\\\\\/PZTPIU6POpKmrYf0HKX2UaGJjlNkF2+Ckn6\\\\\\\/hewIxZOavHLtNvLETFaDktNBCO4Hb65DLiecIUtqxegudR0IA+62MYOhY6ozkYejMZRe2V9V18COP44b40O4R7Ypkl3RTs2\\\\\\\/polDrRxGtRFjv+N9dRELghtflX5kLZqeXYGbiEEO00gv40wPuAJ11QGfuKAvAsN\\\\\\\/Cw4K0wB7IiT2XPexW6K3M+f1n1epPaBgQEiNxBezEnwJSJJy1aWVC6Dui5VeSHnnluMO345I7yU32XUdzoVfT7SNUwcw2kd4w4+B6KA\\\\\\\/B12M+UxNTPgZENHuJKdrkZCogZW4epQUu2shIgQLkfVwnl1zXDQuOT+7k2i2QvU5EEM6ecfp0g+H2C5bIh0JkLxoc60HstPjuTcoB8OUwAhAobMWYLmMASrBwOsG2W91Ci93CSi6ja0B6Z8mZHWlayyfzHB7vzlElpw+nU9cQ6jW4\\\\\\\/ik04XzwdIzVY5X\\\\\\\/3zPoXL9xYJ\\\\\\\/PPZ2BFiAjpOb5ue5XMJS8qbObi1WGDhDv2\\\\\\\/vy18Ev1C51YRvWYMyDDTm9pHYCBMj96JbmE5MyHKK86+nYfEqpCNGLPcWOKsXTSEtWBUacw4t62kpepANqW5RmQIcIDHPbEERO1imrJ9SebmVYYHjcey0kn0809wW\\\\\\\/GnhkoHNy4jPX9aOtvisl46kNQ0JvDz+xC68wKalbGteGEmwtMAVUCS5JUVQaxsTa6lV2oPkYqGqd7gvw\\\\\\\/NeaVTRr0C3TODG2I2G2URkFvcIiUeWD6oQIJATIgswTxeVZZz3IC7J3TyZJ+SZ9yedAYaJHL62oKslWoxrkufdA4OM1tl32rf4Ep5RGd8nTn5oFTlIxRRouXMUdp6zMILKOqWH7rBOBfdAyWoggPcg3sQ0fExfTWzxVmA+66qAGNi7KLYYNq1QB2l7SlLTxs52rqx3U+hAvCPg8n3eQgAB5KqIaLgMklH\\\\\\\/ywCWWQYKUrIWp\\\\\\\/2HSKA6EtL7n8GEk3W3p9LMu4lhqI9z2pTioqix8xRJBRFo6vAnSFUeUd7OjawbgwA81wulk+Nm4iaF1iqnnBBICvp0dTsIA0C+118e5Yv5lI4pbEM\\\\\\\/pBzvkvr2e2tfDhxCjsEZL+EIFS1RYnYKwLKb\\\\\\\/+d5gZv2puwgdGcUAL9Wctdzx2JdzuUb5J0i7Rd\\\\\\\/08WtKdd4JlYi3ZcjDeXUmSwTqleq+haYHct+dKJeWL7aBJps7R53Bc+2VDHn7hJzfpdyraI5kYUcOktmgoEJodfNeOQto0Qn4PlmfcdZ9cyS15KcRpV7cCADgdeq2yZHC2o5sW2GZGChkyp8BoefyI1yupNAFhxGbYUn5Nr+pVkqDKwIwZw6EpC0\\\\\\\/SeIF+qzEsgnACo9UFYb2TWd8tiuyuflNEqrFfR3RPsegTNThFA62UjZ\\\\\\\/5Sz+xxVcJkON\\\\\\\/HzHMXR7hd7RMSQqk12czN+prWInrBAgwcXCUt4MTb9sJQJOkBVS0VPKiZQIZmOL6Nvfjn957yfqK4qIlwerddv6rg2pnFkoeAMG351k22mk74R0s6uy2fXZOghXEEpd1C\\\\\\\/GtvlrzQWWRsGF1DhwOquhkHzixIjcvta8qg2OlLHHj8Rl6i\\\\\\\/VGbPN1QRjx\\\\\\\/OVU2NXRZQCURL+URL3BNDDd9K0WtPua0tvQnyVyAjLkXt6\\\\\\\/kO6e0hI6hkYEjLMb1bKVT3CrWoqWAFIX6puIbMfVa5hvcxrZi+T8GvpskZsXGt2Uw\\\\\\\/yG6cdmVQijmIWtcH2xQj5eBGXAjIGwnobLbJFGRTzlQpBaIojCR6pQqVDUporKmuKBQowp7HoAymEbNSbmdzneK5K+mrV9AZRRwmT6s9RBgPrZiak4HAUwANZTF9VETLwBZaclNSxZ3GUl2bI80EZXwkSoHH52O6uiFuD+h+e+wlSn5IFQ4PN2fsZ8GSgjiTU554fkZ3IwGPOkIwt\\\\\\\/sVgqRtpzeAUTgCdZSSbidcAz82y8lRkLtAFkeQJnHLzsFaFYVSR4gqgm\\\\\\\/Vdm7mUWzujEPAWs4ssWYwfqMcXUucQ12da95cyccEqJwBrSu9m9vRq2PNt0z183tBn5XTh2wbjiZ+DU+2evUEtS20JSXz8QSwakdW43eeNRKG\\\\\\\/NWVX+TV4Xh2IgcGZ9Dw5zdlZ1JsbSWOKgu7Fml1VhIBPt1HmbQh43lcILNgPdQED1\\\\\\\/Dqbf0\\\\\\\/RVWXGwc\\\\\\\/CVxmVQJgsxpoKxDS\\\\\\\/95IIAcfsFiAklA81ejl1K5dDMmNQ\\\\\\\/XHmZjprMfP9ppsPXTHKx\\\\\\\/YBFzD38nvijpLMQKQtxtWGR1vKL38ilTKyIEpBtLgnZah1CFlJUBvvYmKgrAYS+L9+5ZQ41+XpbN8S13ydp0F8QOD+ljJw6ZlwMpdJd9ZZ2jqFCZScCNr3nHOrHvQrSHUQJ4dmh2vH8JXfkJ9ebO0w1\\\\\\\/NwwYMoILqvHfK7hucjjKRzT1J3+zt\\\\\\\/G668bLd1rtDmrQQ\\\\\\\/r6K1hXoiZ+5rWQ0D1k8x8I\\\\\\\/UnK1ii+iXk1ze6wwaVrkfrGgeFtGCcpIgJyAd1Sh7fcxuAU4U8mcm99NDTYSRbgT5rah9hvy5U+LTR0kV46heYc2bq7G5kRpI6AjM3AhDY4ZBxeBdrzE5aEGuIGjproRQ0iIZB3AnOsMxzKu7ZoOq8f3riaoqvURwGNhqZqoHOXLeq74+bgKCcXtwWInt8YxwJXEiYZyFRk\\\\\\\/UE7A1sNwVBsyli6FiYOiq1HwiWNzUhwkZOwAadgtNGFlrBJWrLD7RKaoWLNbVhNpc68M4v3moBmlBK6pyW3pLjOHmPPH2hUqDui1Lql\\\\\\\/yzurZe2NQfmpemYpFo\\\\\\\/2+SjCMbMPmTUSaFQEq2zNeBSqzW4Fusci+70Mj28wGT1tkD2It5iCADXdGxkwflQOs+Npl7bmcieJZXXQoawkTzj9m4DJeXAtc5BlVJ\\\\\\\/A4kt\\\\\\\/F7jinTLhBNRK9\\\\\\\/CChWihuegrAZ6VzSx+kordYoXU2ouQRclL813bPpS2Wepj1v7Y+Bv8eVGDEdWoKMu27Hg\\\\\\\/axPmxIZ5k5oQynZTkrM0ZCZR3IMoAz+PwiuYoSc6ZV\\\\\\\/GNYzxSjQp64\\\\\\\/RwPK+dhUy6az9R+ptxqlI4TtJKjzRio8RTPOcab44MdT\\\\\\\/8NsA9FeVWLMToqS7RhYuAQ6hXwGliMkvMLoAYusxzPqESXVWDEtf1HebsY9KYsbM7T5pc+NMQHl7VXNpi90broQAmJZFu+RphX0aVB9tcTkm7ozxwz583AgN53PWLMGuu1gg30Dr82Y+LAZnOia7ZbqsR0k2MyPbMjtozfQtKGBJwp0ANSzt4PV6Zqgzek7U7E2uOXLKK+huG+MVQXmWoYHgW4xvCigPUnFbaJZkYcBsJsSJ\\\\\\\/Dyb3fCpp8MK+glVoE9uG4s74+meHO4+XOs8M7fvZOnIHIqBCnGuo2jnYFzUW\\\\\\\/csV+2ChExpKPjS3YnvM81KFHJ+WIGvrLA1mCWMx2ZyVPIoEyAkZ17IYgvVu8Y8cIQD8PEM10X7TyFs3v374uRGOXaDCTNYwzW51wgEcr5rrD1AZaxELzMnEhaN7XM4uebYUZgLWaHFltVCO9oXhTDgwURMbuLxYdV2iFTZT9HQMKr56fTCVLR01OMslnPIDDiRWr8M31HUTSlXBn29kQ\\\\\\\/CtdZHcBardRl2ZFONbcPmZlvnFWQ8dyc63J+t3GwxaJ2SAOAlWC+EBNNMzVRGdxMFLhU8vVJEVH8zUOweqVFzduUzmVYb+ioRxapzgcAN1MEwB4OccrZty4gIB39QDArJLIeetOFO4qVlnOFtqp+GMJSYV27ADUGkYPeLeDSNeQbjkH2wfoiPT7HYv5l1T8A\\\\\\\/QlX5s6bnPquSmTphq0pcI8VWKuVlpuD44WIBtHk3IOpscKgvp2uZDWBdzgXAzC2Mcmgvm2Ez83mK002DdW9NOMaumnKNVmxEADwVWT56v59mS5qGPVYZzKrcVX25gdeSNHe6y7wBcR5DewgOYwhdq6SurCHFYV+A\\\\\\\/IjCCP+7VcOA0DJLCvM5LzIaDWAXwthlTUsq\\\\\\\/Ciyh7BHOdmw5raYZnzZvE5+N5VTrAuYjqAc4cOQwewPoTLNn9Yw5LiyKg9vcCjuojPFvRY74xxwQ5l1xOiBBszMzfbxoR52K4B8tRoIUZhJvBNcb13XBzSJ+ERiYGVAVGov\\\\\\\/tUquhhV91fHxVTh29lUsYzefiMuDwckojnoO7L\\\\\\\/lQ48r2C143sw2iZfR+aBvLcz5Mz40GMfp\\\\\\\/RwjNOf5v9SNPc0z4TRXnfpJvFgs7qd89JWQrEqI\\\\\\\/U\\\\\\\/C22VzUiSfjze5D5rkvQUffJKtM1EmKdQcNlErM7yeoNVmx08VY6wDDuFSh7xGEE4frhcXhMD2yt3DNGlvGFxGBPejH4vYJSold35glqehUUxQOzKAujFSRXfgeGFhum7m\\\\\\\/4Mn02FUVH3Y5ChCzH1o\\\\\\\/Ax+EZvFgtPoCzlsi\\\\\\\/K\\\\\\\/HmGagTTb0KgDZh8FWNZNHrZB\\\\\\\/X78usclHIE29kTJXqOh8wcsftoRAOY8IraNIsx6\\\\\\\/tchtgjeGZXwsY9YZrZ1WvdqAWuJ94MuyRYwGxl8bplLpxMu6MPOGm2CIb1ItdtWaGplI18OiPghpn0DFPKQWycH0sANYf6y5OOeb2LQMzcESbux5gp5zYrODJeDbOdu\\\\\\\/pKTHwBJbPdTPqWlq2hGnjluHjijqOH6Ey9zkKXO6nvuaUBYOT+k2Z\\\\\\\/Z2FmEi8SbBaBDvNoIrILn3TMxFbyOtNtSjbR4gDtxJ3225GWwhcm8e\\\\\\\/CxEb99yqsSCygLH+pk+gZpiMzBt1lXvNk0vw9kPbAZOje7gnI7T31\\\\\\\/0xZb3LQeynhfRbPCBuzZJZmvfAWSd2CPt9vlhDRw43wVPXt5urB3iKlb\\\\\\\/N1ZQEcXKNRCL7XmATYmoUKHwaVRmagekixEy1zJOOmQqTWb1SXpK+OpHlwsDM2y4s6t6HOs3QLNI7xO4PG+ozks9Uh0IYQksTVvXasvJd5\\\\\\\/6K1smPTHfq73XU+LzqfogeXHzhv6ILhuiLRYdHScW3ABa3OAygz8N9ZPaFQJlC13EmVptHRHm50PtzItkGojiTDv+J17m1sPyT7mF9jDhRs+9jxOLo4p6YUb8VbuRxA4D8jVdsrvenVoluKLHyaqrbs1C9g8USTjSCAqJN14uEi\\\\\\\/oJ9uo58G8VJTZgW4LQkluWqD8ELuZrJ7CqjIWxtHqgYcjmbHmuD\\\\\\\/kh3tCIVrohA1K8RzvmoPRP+VviElC\\\\\\\/ffJq7gH6Y9xk7FrBzjNi7K0J5jmMGWY35frsZgsfu3X8PlWWlsTjZ4hNg+889dPu31nG9Csf6dhXp9VZ289ptxBWWmwRjO67e8sxR5\\\\\\\/cW1jRmyHqn3f6khgcbkYT4d7NSEJggJBq48GpstmOtYgj33gF68a+tQsacxQAfZHq\\\\\\\/x0IExJTVSOOZW7Xk2WnBHwIdLwCXBJWheoifF4xhSfiHdQldeFXka6ZKGJTK4do1fD\\\\\\\/N2PoTm4+AXzIbemM7HjBnjKT6JX+MzgG92AhNY7Rgis\\\\\\\/LvhRN5OqVXkwfhrNMBRu\\\\\\\/gAtOa5GFzwz5wmuRehKSj6t5OsCxQKnYVQrL86NlUnrlCDRv7mtRxPVaLAmpZCCWG+N3WhB3\\\\\\\/3DO1T+W9lebTWIlljG2ABFu+7qWXnS5i5oTHcAkHl2vReCgoCp0U+yVe61Ev\\\\\\\/qSt3Hb1kFXLMMpNgS3sqpxOY0jOFDmFcIRyYE8jJxcVzWmgWM8AUw4ulkA8innvy4jlE7NkJ4f5TmmLlC2PcGOxfZQoQiR\\\\\\\/VfFdOaYQjc5tN++cUbWPSIvZiNWkweqxtsrVjWec6f9agV9Alqi8m0rvy6U5fsUKdsUJrUMce5+nIbqrCTLr6uGbjKmClYtJfKqSSONZ\\\\\\\/CD66prFUWMEJR+Vdfx3mx2Dty+JvRw0vJvxWsZphv4iwqKljcFNLrzU47uuArDC+UNAKtleZpkXhKQqmA7G5ywRTWMZA9pqEXn1HCRk4id7jmAfCcu0OBY7LRrgnOFYeURmtdxZ4nOL+xHKIJz9keMfTO8Og75NuSbVFC7gHVIm11DV8Z7ojpSeN3KIbZS6V4kBikJXfjZbutYv\\\\\\\/rf8nIWiP\\\\\\\/4W35njaXS3dKtdWuh5fbrrhC9eqJpCptNY+PkUw9Ypqo9f5ByOhbbjV6lh3KjNRWLb2nWc+34SAHWaQhKpUX8HqGriga5lLU13D9phxwmVFAoOGGn7GRta98L+YGJJCZaXwL+uwzWcHCJHrbMt1GkBscKGiUr9EnskkZg0w8mEH6exz4iQCEOjPxU032fE01aOhtz9hmgz17JUx+i6nf+FfNFN9h0N\\\\\\\/Gc4Tg0SkYUbxaJIht24HoI7qzej0J8OEr942ZSdW0Zpw+uFS9mlhTd35Sv1UAeXGDXgdLXMpSh7Zz9SF9BuKq6ZVNOVNVoxmUOhtPd2uyOAktGpQOJSSzQG8PZbYbUFlA\\\\\\\/Q5xEYV+apABCg+fLVSXjdFwgXYk8mLBjbiQcSLj61cMNpOLyOUsUN8Te3k598uaImhhZIMHDplorm+yDdn1F\\\\\\\/FMvKhGGlRxkhC\\\\\\\/NBQd\\\\\\\/K3u3bWt9xw+TsqvaLKsx9l0GH8A1XFCOgLmxiZoaLiTEFXFDaYbJVVZkPCla+GeHq1NXCtQEaGb0E1l+J2JhxsH1YJYG+RjkC8ZJYJDWSXUAiAyM4YnqO\\\\\\\/yoS3zPEWYhTJBRXdgbgSOp+HfuYqjVXFQxYfMiFnjoi8kfdkzlpViqtmRYlNCR+OCRZLsfOESM9t6S1\\\\\\\/buMxWDQWiep6mvGkJZSZKHGtVuvY6nZOyXVdYVFc18YVU6trIxllT\\\\\\\/mnhHRP+fCCVUKTEgsZvtJ6uau7MA9HFQWWpGwNCKaH6u7a49pip8\\\\\\\/znJq1mbB7jhjx0HLyhRejrQ5hbQvtcP29vuyRtB29pVIysmKTafRN0ENEN4yenj3My\\\\\\\/\\\\\\\/RsC1GnaMNqDz5ESKlXv7BsSQpU\\\\\\\/Ph7J5Pi47ExlD7D0usZ52SR9pH\\\\\\\/LBKYE2Z+4oRWmBdhjdFpGV8c3M9bQbB1XQC7Q\\\\\\\/dvyHnPZFOyNLfWX9xWdNSZWNGXmdYYOcgqQu8qCuiygwV4c66d66U0lMqvMwpnhYKg6FDrO3R9b6zC11AMcjq0jJxxKjcLKI9Qun4czu1RmRPCH+Bttr5hM1KFU7z+48jlzU5mWI3oGJ9yzcgsBLeAijK7tq1ywLP+X89Z4SK4C4FeF4JqPeytM6ZX7PbafQe2clpHYMFlZRzMHmB0rVJEQ0AHasyKHgTtykS0+j75OhlaLVZZMlNJCqw3fc1UyzKIg6TVfbiV\\\\\\\/WdSGZf1WxLUtfAktHqb0uptsYplt\\\\\\\/IRC97P9C1gC9OS2OivX9u16SoNwXjWtqOtmMpFm1uWczSih8ZMemuECeCoIyxXqmWuoppKLfAd\\\\\\\/iENT7IOF0Lh+b34KPYbwuz5e7kRa66EJ5y5Pmmj8rnjAm8mAOcQi9+\\\\\\\/KHkzXE4A8WnEKmwXqCbIp+a3l1qRwy6R5qcXrgvgS5rhC0sYRr3E3V5yKScW3l4fovBVy5p\\\\\\\/8ZFv8CeOVsHGWNmKpvTcv1CO\\\\\\\/JIznQoMehMUKVIgaQWHyYdODW3Acj5TAHHF1\\\\\\\/XX6xFp703z\\\\\\\/d64bXLLri4KjFmCE6r\\\\\\\/RWNEbigyViPtqYjbyayEKIhMbVVv5NBg3HXvJm8NjwacSLmPHGIysVAuZJyLcFbLZi10TLRnJTGAyF\\\\\\\/hEZh+UvyGIhRnzP8n6R6lKLcVfR8xxn\\\\\\\/4PtfAufYQtgyw60UMGcBGbXD81A48hXqdkl3q5lPDISOSQhngyx4FsZbDUgwhuaWa9pUn34pz0cjLDWR1TYgkpOuILSfunXCptXoo1PuIrbr9ylx\\\\\\\/zT06ngCTaoglsxQ\\\\\\\/rBPQ4ijAoKIHu8uvDjMlhHSrPd0QHhDSDCLIJhyccxnny0D5aHKTDyXTjeTj1K8fb7PZLWuccAhUkcY77TpWjQNUMiy4rFUA7m5fLzsghRcaGOjogAKvJaDdfvBzFTQDEjCLSwvSmV2eXUn8sbntdz8Z0HkoFxiTH0LW2z2Ob1ln8ri9uY5e4YhiRTVo99aRg7N5U1fQvuhx1EUS1C8Y0yCGeouLCxwAgsQaIz5+LknSg2ki2gBUcxqnEe7qqmDxU4jkQxXGfvDue+xuxoZCaefqEC15JdM1iOHft+wn0YglA5RQ877Wb1hgYIPLEaNBCu5ITqjzaymePlM5Rrq49uEgqCHz4kkzSJ\\\\\\\/qRLFQcKTodIjfka0ECLssU\\\\\\\/\\\\\\\/GbElX6srlcGUZLwsPsPemc9gd\\\\\\\/AwfUPfwvagQqchbFM4yS+3BKVpuXg+EHdlaTm2490yyyWQMkr3GppzeDE4dR4uJXfBblk93AFjNiveC3SW77QnXbg+YubH7Ck8KjaM4D7ow0yffo4qIr\\\\\\\/66NPYLsS2KbautTBT7Q7DE3OZ0Cu9AclGEPNhWz1j48wR8URZ5YDqjjGrlvgcj1Unv\\\\\\\/czAG46WtKOG7UeF0RAnaNGx0gtxIFijzHSbhTkQRf3iA0YHuxZy+Aya9LkeG0N1AS5s1ixaFrK8PYAXwOe9wIPhl1TZ4p3mRFoPhzHF2vQwWaaZ8GV1T2cLyylNW5SMRL585u1\\\\\\\/\\\\\\\/9q0ua9dV47jOaLega\\\\\\\/5TG7u59p2Nsy63qkC2qmVoFEHsoETSpSf0dHbAubwif7j03MoCsJIU6unHVZJOP6r4lAiDbak\\\\\\\/1fOOHNnpa3TZK6yxbZhpCpKEC4quinjOJVd1Vqdhb\\\\\\\/aumCqUxFHimNysFlesmr24QbswZE9kymLHDRTorFecYAyfmd4poEXNM79eHrQWIN04qQ7pRgeW2xYcomVvYyYiOA\\\\\\\/GmedPvgTwxHNiaPjE7+E9ng4fzJPWsMyopGaxjngVM6\\\\\\\/xxsT4zyF0rydVRh7d75jFCb2urwvSYfCi2BqiQd4ukO2w23CGK6oIKLz9sOy274UjFX4kl82qk\\\\\\\/uRoN2lsT+DyPKNd48iS45MhpJ\\\\\\\/IhNRJKxn\\\\\\\/Pt4GCq2Lf\\\\\\\/OpwU9W27MeNSCSMMKRdmBzkKjpTSeUNxkTqZ6eruEzmJMHsa18SJSUptkTDNnKpxiC7CtvoF4\\\\\\\/6uhTroL\\\\\\\/HOGjoM5fs5vQDWMbGpqSF+\\\\\\\/yVHKOhgaq7qhozJUgfoMK0a8QBCmK6fgGPKFmKwMlG47cB+Sl3ZlOBK6lg5\\\\\\\/JinKZOOY1Gus6GmRV2iSRknT994HsWWsY8uGQi1ZDZ15k3BkWTp7xSZfsCob3LpArfGXXtI\\\\\\\/\\\\\\\/egb6W8DDING6m6tkDzEfc639E7ydgfVH6jIeENtjKiF\\\\\\\/XdnxEEwpG4Ak3TGs0dvVmgZsjoaguw4mW8vJeiFT3F8ruyi5HRxt0TOEncbjCJc\\\\\\\/QtWkF52btb6URlTNk3w53n9tME6OoGZo+0ztcLxXlCsYDknA82t5EglSJbpQMhFZALP5\\\\\\\/WBuLSjXdUMizLfosY2\\\\\\\/Q8aqJHMdwJTjYOQUwlbs46DbFz3B1mYFgU4\\\\\\\/r77JZyX3qvP24CIiVOkDszZw+dkhG\\\\\\\/Ppiz2ZZUpZWktSb3ooREzlJH244ItpCN52epfWwvlgYreaStpGs6U25vXM0WOpDEJsJ3NDRoRDPBx5d8p+MK6SL6wruhIfsuir2lfZkUSLGcMjwUT58hKane0mVMVQP\\\\\\\/k\\\\\\\/3goYoDgYK7ASKo5VVoo5z8yopJJ5zEQClUPdMJLPmbKSrfuoViP0eoREcIkPKG2Rsum3RWvq3VMI41Ojqma4e+GINTckHFGfGzIgwR845RdSi7hOAvfpfA0agiTa9jAY5em+Zduuqb7gMa88eNrzBBZPnu4UQA6QTCeq0cze0o6kjxe8P\\\\\\\/9iHEb6IUv8R8BiAsLugETFAfL3rPMHUXOiDJUerSUWnCrvBNcP4RNurtcfN7QcthdDYWQi2hCBtKEJu2Y+IwlHWgoFzm7KzPPsrxANzvIqmdTaVac7X94\\\\\\\/92xFsYJNAHGaGgbCKClgyaLEnH71Q7w1ACmea2z6ClmMYJ9\\\\\\\/zJrrdt6r2TKDhNJeXdhZobsFX\\\\\\\/aY5HOyATY5LlwHXMz5icZf7H9xG1AwJDKXHL85+bdMeGmOW1APUuXQgB+tHveddO064gtSdQtuBNoF3Owfjk6yhlwrOvtZaK\\\\\\\/q1pFOwcOcBCH+gqzYl55+QyDnGlOISwNrQfX2ss9u0KoSb8pK9KhhKpe+iF1OHPezG4xX4ngrMJGasUK0JLUmULn7XQ3xb4tCrIokI3qYuMvGjheObm9C\\\\\\\/ncw1miGgA3MGCqV7L9GeNs9qwgEhdXFigoN8K23NzNrHx4S5TxBx\\\\\\\/QuYS\\\\\\\/4DknxrjdtBOL3YmUFd8mYAnOtTeW\\\\\\\/TewIXDBaeFfjg5ILnvO6lItZPo35sx+it+m7P1NkqWe7gvJpk3PH2WHiWOI4pX1iWlm1HMYOvHbdUUwhbc8Cx0YJuK6yfTg415eOX4nxBEy4g23HWIT8gk7z9P9MJSelUvnM3zPGnNUisQYtnAVXEvZ6GbG7rH5x5IkatMeK10gbJLptEV6mW8JxW8IAJy4peNlxpJq84tUWN2VgFttt5snOMlIgxvXTnEeqswbV2XOB1LYL2Ls\\\\\\\/Nj\\\\\\\/1+KL+l6URU5FC2tZ+U1mOz\\\\\\\/JtSf5jvl0lMbj32glXTN43qcvg18MC9bOKHVAgkIjTm1uD7fe20WtEWwQJn\\\\\\\/aTlm+pzX6uYriXcBT\\\\\\\/louVNEjTIvvrkCaPKP0oPb+cI0\\\\\\\/1UTwar7SQCEw+0Oucn2NMWGaySnzHx\\\\\\\/ZyBTf16TG3KV4j1sBzubM5MFWzsbwlv1Nz363kc5fmboIsrrGjQUz4+CzI11njgZmtTXzuLkJ3cWLWZmiKGpJgAXIYaw6Ia0EBODR6Yxe\\\\\\\/G\\\\\\\/hzvaeGMajSeGbVOgMt3Hg+\\\\\\\/xAJG50TTsa6hWvdaA8IPMd\\\\\\\/nZCvc2Nx4c0tQhDo1CHbcXwKUg2LAp0z4siH9hLNDGpkCD3Bd5p1yxobjCQ9Hhpi2sUFukZdm3zroauDb14tZ+VHVGng9qCPVAKjyu7ykq9r0JkswtqGzBm1GtO4DlXTx2GNNywXfQ1re4Sm1PX30g+Wnm2JcpEZ8ztjc9xyOoh0lPfQAnvt53zoD9Z\\\\\\\/uGLHbBwHxWmHYdXvyQmuomNAng1y0jRG3zVVLq+NHA1g1yXemj\\\\\\\/XkK6MkJkbTLxu4xEj1MdHD6qiNrefmHdZrPU\\\\\\\/MSTiPk75Ih\\\\\\\/y\\\\\\\/32fcLjBuGavf2So9D\\\\\\\/jObnGFk5fqhBjtp3fyqm2SDpef3umkpxSNC\\\\\\\/5bfJz7dUeB3snkMSlAOwFUlx\\\\\\\/s4tsRMpo5PZWKghuZY2aeJM+Df1iC8Bm+Vim66H+Wg\\\\\\\/XCmb1bu2ZTrl4hHrGAd5br1jMaChWWA6TO4u1D5AM6Mzohd8EtJPRtk0mqNstxBZAmVcf8nR4hsdC9ZFJtvB5ZcmT5vCQ\\\\\\\/e64eI0Bxt2vzLiJdS53PT4elRvl4MpdE5DNOO8pg8qbYsmJKy\\\\\\\/6ADspEvAOs7vhGIX0N0PuO7M8GB5HC3PAA6A5kdnKSG1FQ21B9STUV1Z84AVm+MXOJoCEQRpDSO6UjqwjNi3vAB9lnUk0iOuFlq2aLP3gQymF7BlwuVedhQkZpdyMpTMZqh6KNPNhyhfs35r4GJ5LJR4j1QsbwoIZDxjwUIqb+qPVB1dAVmdjkko0eJJ1uckqKvIxcp8bZDc1ZAe39dYSlm1ifzgtEsP8cUTggTARMrAm0wwLr14o7SpV0a6wEA1ywEIDF655bg\\\\\\\/ULYNpDE\\\\\\\/q3ESsEyDqK+NdZCj2\\\\\\\/HSRB7\\\\\\\/mvrl34JgCWP+ljQRr8Lmgfka1nr1hEI\\\\\\\/8Il\\\\\\\/lWrjsV8kZgIqPsX3bKmHEYRCYgWoBbdDNaEQNdWn5SpfALyUNmAl7cb\\\\\\\/tekdcQrLCTAyZYMqWMDGZlk9Jd\\\\\\\/StWhCmT6\\\\\\\/rbcyNykqFwuXoFTiAfbm9ER6REXCmoppcJIDzZcP+KC4ZJxvWy7cHocmR0gPHY8QiUy42ozmYlgYB\\\\\\\/m4D\\\\\\\/1wdfRDI+f+HYN4\\\\\\\/BZQGxgSh0CLMoCR0p\\\\\\\/3cxiFg6cU9EzfZdx+FS3offnClTKpDxXsF0wq6ptUPU3FqTXOU9JKCxE6wto+DL3qfeDlm+JAInS6WkW7FX286\\\\\\\/SmTrxHltt4tb\\\\\\\/jQHdbpQECr4rRTjlqjIyNO0VmibMkeKo146GM4n9\\\\\\\/AnFjDkbqWqEvHbFDl0I4JRj2H5BDKcuj5VPsOkTW\\\\\\\/txjpjdEltUD9KLAPxmoXLeCe8slh9H\\\\\\\/Y26Pf+epHG\\\\\\\/L4ZPee5hd0C4WrIueZWaH6Opm98090Ttoj3PYGD63hLv2wViIKGBWo0rvvrF63K2S2FEDAtICUtlRxJvKeGikoYdhbt8KS5l+gXhPoF3FFqka9c1Ma\\\\\\\/tXCQzVOlP\\\\\\\/6n2xOxYPY+x6J\\\\\\\/hWe7D8NURH7\\\\\\\/UjNYvnDq3yahPkF2lRE30M5BDYWmjI2TG2LGsfOvkdof1b6JOjLMNQiMFqzeNJ9saQ+8wTlQuysqmkfqcKlznNI3V3rLvUj4DATpHIXh5Syqd8C1xFxC4yOVckiKNYt6Cx3uJaGtxvE+Q0HwPcNaT3zWGsrqljnrHZHkVF+AV1MnL4JyWyzNDhom+jWx+cDRDRApXIG3GQdIMEPGkvbOblY5Wv1heC2GMPoMtBTzin5a4GSl8hSIQjVXJ+8VMMUzuAxaVdrNM4ICRpGIjbUxFnmwJLbivn2OTc5yqC28tW8wi0lTYzgIvLkfGsdUj1qs1iPAZtl+YGmgOva7F5v5XNABUhJdb4pSERKmPDE5r3pj4zuFvTCovBCM4yILEjsT9TahF\\\\\\\/rBdBlzzUbq42gzZ8gSX8PuwNGA66fRSJQDM2HwhAvgOIR34sTYyvlh82c6VJgCwHOtf0O1LqE2rUmJR4MRXyNhPyhLIzbqkdEkHdlrtKYYAMNLtuOryo88kDXBSBg3O23slxjsycMuQABoQUeOl8JUj7fqEhr7tBBz\\\\\\\/HJtkKl0aNsqdtU926E1s2\\\\\\\/BtFpa8JnYmaS\\\\\\\/ZhQX\\\\\\\/2ZsQwQl4TQ2OU0zpiMDE0pMgZj1Lqn28zM+efoKUNwOFE3bFqhhEQXKETnoD58Lw6SaG8BQWmvb0vZ2oAkf9obsRGSyTZQPchw9VrXXt8hsjwE1MyTc7AHnxLWR3sWk8Kka5InpEckDqRUhlGkdfQgeGdFBNmhVguya2WvDr\\\\\\\/A2v\\\\\\\/htZT7t3AWXzhk\\\\\\\/JwO2rc92mMNsm5t+j5fzi3DkBbLtv2\\\\\\\/Sxe4icKPzzS8y6y6t4XVbYUybxP5\\\\\\\/szE9tc5HMxzqPcfyO5GIpj97i1YIdDcaQ\\\\\\\/n8PxLaa3RqwCWeZmgNnzQYATtJQSEO537GxoNN6U5Jp2tJWtrWYzX2FJ4b3EkrtSe6jsT7CDfCGoy8zPL2GvAANna5kmzxr67YaqYTPEGs\\\\\\\/\\\\\\\/WVGfBD6HmyBzZxRb4IX0hITBlDQEqk1jkKRGCU4smdLtRUO8LmzBU\\\\\\\/IkjIbZv5YXYrKzaIZEpRjxXVQD05A5k7Qyj7bIC8m3Ocnxe9g2XPmBzJ+edueiKprMzL0gifmv3kBkpJT9EG9oN5FFM\\\\\\\/I4w9r0eCRApTMJpr9svs1EqLodb\\\\\\\/gtJfVufQ8qZW7s+65b9vIr9qxRIkZURLWRtvebVxW1BRccNj0BQfICpqwdmmUe2koFXFS83pwI\\\\\\\/mmdd38rclEbzp+M0htCRFMahc5rOsueLMzKBME\\\\\\\/NIRWXHRa+TrBxA2DMG5paRtmKx8RunKCzbBMU\\\\\\\/lXGvBp3EkWhIQ49tlzv7l42uuasfGrrgPmA1\\\\\\\/YHfIyFxklFuNGwCQdu+Mvk3rhU\\\\\\\/xD5tMZEz6wqsYX7i1aDyxFrT5+7ayF9cs0wMFThmKadhg70p89epfODfqWft+7XWVia74InJz2jwoYeFDe\\\\\\\/zT791yQti2cH1GwXsILa\\\\\\\/IRoUgV+e3CrVLfg+oMZntdWKwhDriyhIm4WBjAhVFxNSCjPnpNoNIjV6AKwSBJVcfW5BuX5WQ5kwcD9nKp0h0j0DXBVon2IoUTsfzDAWrA787ZUKlwex9nOhG8T3xZiQRaTGpaJOZpOIRhNjBnGX6hkXlPOI1\\\\\\\/GzbqTnKr\\\\\\\/hRbcr1HMAQf82YWQ+vy6CUYR7xfFXfZhm\\\\\\\/fWS0jLuKPGMY\\\\\\\/E5tmhPqsR8mvTOQtvm3+XcBjObXspVhULPYiYu6ZynP70IORTu7DDA4oY9CUmQHaQdoQhFcG7eVbvSTulAGdSM3ibsKG7GbG5zdtYGUUQtF+KCWlOHOP+cViN1L\\\\\\\/pAGYggFhh9+BQ\\\\\\\/jf2FN\\\\\\\/wZHj1qA+BoSVZM6s\\\\\\\/XfT+vMRKDZIl0CxjL9aIdwq0LZOrc9dyQRDT+0owg4kTUdvfu2H\\\\\\\/gRTgr1uJpLN5xTlcvW971py9wck5iqgjUdD++eyeS8DXc66ODy7Sd5XF92MmzXznIjgXB33VuEEqtlN31RlS4uSzMcrRPb8\\\\\\\/NPREDnPitg68bd0z5Bb6oern6mLVtH6Egbp3peGvCQ+FtpY4KeQy+FnfDLv1mQWJqCEQNlhlzLwjgCzJPaoye82tyR4a5yrMNQL++LdbwiG9KwT1SYKZmhbNUTkkxM2iAoDuVzlStRxdmbVJ7HBTJdHY+keSU99FT2oVphyzUX5+T4RC\\\\\\\/Xp1E2F21VB1CzXr0bCsa4BCP4perQmlnuAc5zgyJoEWGmxQp369gGA4LAxkdZteZw14uDsHdd08kYoXbmlxSYVPFmivZ7DuEr8P15ODmWBmsLJ0apc43VLp8H4BaV\\\\\\\/GtIbbyRHyQTQgkhSr6Y2igN5HOmbxtbbqBumsq91bCWn1H9ONzySRGefJyZ865WU4QBOLjWo7sTp5wDP4h+9yL9w64n4GGHEMxpSj4x38gHufO9pNEn+m9vNFgR12nrpHNtq7ktDb+8m4T0uIalpEbTp5H7bAOSFLGo5RtGnZVKt7X1J4seLdsA8Gm5MQ177xJDfYMXi9WfosXR70ZSsuoETRL6wT7qdoQdfC1biUBt8UsrPw4EEqAFtK9Mj0BUCpvNt1Ff+mwI0hNYSUIqrPWo7AZHUfRRDDLh6ktEpOVnRLnqZ9ODKvgl0wGxqqmb1rjBgAG01lq\\\\\\\/w1Hio2YKAPTEREAzbD16o\\\\\\\/l7EwPrNcWv++GwCYZ1+YdunLqsRWBl7wosu8J\\\\\\\/YEha1kUSWdPA0gSI\\\\\\\/2t3UgfTOG3NSF5v+3kiFqFmuNu6wo5lP9CmHEE8OzSYK9+cOzlc4AZZqcaqRtYkdNRcQyPiPYcVEEhUFkzq77CXbdkluER+8wE3hTfsR7IQgB3MoXlQXcueck+2kvniG+K\\\\\\\/QAoPFHgB1R7bEwT\\\\\\\/OtTumuFdXlVi08mRco4p00SDFr1O7LG9O+cB3WNLV78TVKWNns09y2B4yhXjnsMUac0spL\\\\\\\/njVRHf2jJ8NhGBnOy0FYo8XundsJ5U8eebK1b5HQgFVw41H+Qh4AvfVUD5NX4RMxyJO2HA2QLpLs5FludEEENhrExnY8aCdh0pD28lUzVPz2gM2+LJ7BECIbdlFEsDJdeSebcqlLXOy4pya5XZSvGfHWEHqFMmT5vlaAFCSFwJRrGdX694sC9+4bSyuqJH43j4rm7c085bsCPpn3fAruAXzEraU8pcQXwYpvhhEgN680EM8YDIs42jp5hG07d8KxqEoI5BenlBttl48Oa8YYDXhGGp7y8OingNH5HIpSyd\\\\\\\/LBIgMSQqzNlcxmJpzaNr4D9yErEZFPUVXFoa8HjJhgux6oRAroJmluKAWZ\\\\\\\/HUzs1iEsG7b0yaI7sFZN7J9TtlWrpNDh95Sszm8ynSgsoqFq0Rfo6X7SADEBMq2F+qMBt\\\\\\\/xAySinbVH8d2Xgvvxb2M+0D9C\\\\\\\/KulC9Gsr+qnSjx9bX2UrFcQtXLWZUPJIq+N+\\\\\\\/4pbv8ob++LK0Lrdgsrl+NUhn3ww1Vb+pG+xgIuRCAmkzyPg1J9TM5micQ9rbNEPwycAi6Y3BVGkpEgsyQ23u9GfuCLT3rj4U+BPywdoWl0uelidFKOhcNXwijnB5QVD+SmNCZBkFbokF4+OEaGRoTOYpFfMXiyHfsx9TZbYlbZ+RkaPxW64Ufe9oebEx7XQwg2l2FJv0C0ghfUUg1XcgeL6a9fNQfScmL6a5fM3MwUkwQUflHJ2M\\\\\\\/kSo1X5tFwZGV9flZBln5B6mmVyg4gygI\\\\\\\/tr1cgbFajC4Jm+4b3kauCWz\\\\\\\/Al7NFLpBohhGEC2v\\\\\\\/aEHOmVdQ+BsUpzlVkSJ7NuL0XlDn\\\\\\\/gYpr0A2ZG41gf232tnksB3UI964uKipbPPxBs9quPbZ1N56Y2Z+Zu3wdrogy4niEYtLTSpTuOWHzRJGDo\\\\\\\/iLlLtQHwSf\\\\\\\/IxFV4r\\\\\\\/irFnMqhO0HMpp0p7A2o9pnBJWmtCPOzx5rj+E2LaTkPlcT4ls\\\\\\\/n4D61ggJqjjR9heHN3mNg9y0g9iMyMK8S4wxGsaHqxeQFH9VPybQnY1GY5cn2lWjKexCsiGXzHXrnSqMAW42CcY7rkpezgEOpfCs2ZcVDjhJIMt6KEAyUfp\\\\\\\/xNEzPv5qSitS2Q3sGRNA5GjIwQQKq0gfvSJ3i0IyOHKSpGNfcfg9K4MwXeUFbckxkN7KPbKTQ7qWN84r1yX5abraBMtEVIfIR5HNqXZhWrnXACNdio03lTUhondhLBq\\\\\\\/nNOPjSa8HRdEQ8I0Xi3zO3c8BJH9LP3lijl+ekMiDtfWA9K\\\\\\\/TsPPFOsOJfolkGPwsRb4ZTDniw0UHT8cH5x3m1rRtHI1YoX7ROAIdSsXAreaW3omxsDKXnvNSV2+kCdRQBC2NfrgPS9+CwI+G4dXRCz6kKrUULY\\\\\\\/BXdtOcymBClDiB8FMF79OWD9XqpD9hW60UtUZXVz8Y4htupFjst+KpGAUOO4ZIygA2BUiyzVqpmd9Exaom7T3WIStKyNPnzG4bZ7eMknFRGKdbgfR4ygM9fEo9EGsL1c+gWmwP9su8A8la9xncBsO6Y7GmPmSdvRRRyZNEfZdfW893G0IMkMfxC3n4B51CSzHlUgaEIyiSgLQNT2zwTee37GFItdlFz3HtdUrCL57whBftCDlmLUeZazTpnporTuC7lBP4M+Z3sPmXcAPE8QtP7EIVhriLsMJn8qp468F9TgHL+xvEqrfO7Ep\\\\\\\/P2Op6QG2aRPRRbZhab\\\\\\\/G7ew3nAizMsFzbFs3vLo44RIeR3QZ3EijqtA841kO3P2lcfO5WjhsrM55IdC22bJKiYki5kmmT03Qyhadtzffl6tp7TJT\\\\\\\/r\\\\\\\/2AI6Xp3JzeVLGvz632AVwdju8y3DVvyN49bDr1IMWQlu0qMm9sGCjZn9sJ+Gs4iUGvGbQGuq4Kk8KmgFxWNmzij5TZA72KgHml\\\\\\\/rH7m9Thwzu+SrX4rs\\\\\\\/5jix7H9mffOw6GEnQG6WrRs3qsFNMkg\\\\\\\/PctiajL0JksZmYod4DaB\\\\\\\/wZski3mVgr8dTt4fpFwOljbp7U98vqTnqKSsMq2g5Czc8tUD5nerEJUlglTQZQb0JIM7PftYRqJ3WgOJW6eXjbuSAW8KsLqmguLP9qMXH8aX9snR3MJxmoNglxrr1Orvej28ATitJoL9NWnD3K+TuJJHDVgFgeXa6FL6ElRwyWsh1rfQpe6ynRyFQrq0dXfAYRIGaUnDf0WD198PT9GsJ\\\\\\\/LxpQupXRGEfVgA2KvW0Pu4XuTivaejeyRg\\\\\\\/vG973Sb3IpXNp\\\\\\\/w\\\\\\\/5mioarYZl6S9XeWpo5kzBpsFuSRTG9YCq8O87TFhgFFJqTNYRrO3CUJ3t0O2xWraFEVex+3HzA\\\\\\\/pG7AiwuQQZcMFqx54JLaaY22yKD3uRXCnxvNTsPO8HPkQroW4V2xtJu\\\\\\\/KSCtYZUN4t3YCCsa4kesO2Q+lyNNJv\\\\\\\/mQj\\\\\\\/R7gt3V+t79XCB2QKXrxAQttXRTjLBSl3mWkPldEHnHCkgNJ4gx0N+1n5KjSunS8T0kY2DCwl0e6iaBBewOcZVGLoDsPRCVOqZ8RdUjunhbYIAZhuFcXFKz4y28djMXUIqA45ekbI0EagGRKJLVlmCuFgV86v2OpbFzvXUX8bbA8EC1JhCOqABrlltov3v5tT2sYczrIL08RN2rAGSRnxrrf4aFqt6hT6Ph9n2iUFb2a5plcpF+OQHqTK7d4DOC9sXaibnbzdWnsebLDpVjcfNQClpf\\\\\\\/2OFXy0AwzLVjwmrIhTAgEGyKhlk3sGNdZXLzec8MvFfZCWBOI0SiuGhauFiu970lfvyMaF3pZTlpVwakxTB9z8FROJYpIW53RbiaE9wkoWNLj9cIUGXCbuIsOvWE6PB0WDp0MjFQ95QwQ90dhc7xQltAwBbVghGQc4B2aIu81JmMKPTLU2HFLKoOUvXjRDoKb8vxVbyhD38zZ7fVWyZwjUhX+RemzSNSt0gjoCA0rNOisvmJT0UXXEufM06nrZklZDd64bPthZNnVVQJPRlaGZm\\\\\\\/G4OtT41VjPQdlpS+IoEpiuG2VFUSsgLwb\\\\\\\/LDLqrBUU8m3ATRW3JFLC\\\\\\\/Ce+dV9Wn1O5Kbu8iBiLDiMXeF0MtVTQ5JRUKq3z8IGriYl\\\\\\\/LxWAhptejH6e05O6nQjUM16lr0IK7g457LaN2s4CUcnDC5OvlLWz\\\\\\\/QaPF8VsooPgYGPNVAM0QxCl5PsOJ1GIyXn2LSaLySQgV4m3\\\\\\\/2bmHIrEuyi+WoXFd\\\\\\\/BJPZI+5cfP7CVd8fCq2X5qNWW13wDJA3v9ycmabSBSXVk7D5wb3y9oAuBU+D9u0JNapL+b8xpO3iFbVO+PYNH6Hi\\\\\\\/JiTFmw\\\\\\\/dOOpBWKR+pZmrEF6W1whhQU+aB2AUTXa91l8LJIiSeIG133PoLMcEkTwNW0TKZB6+9HgN1JMmMmUsDYJxWL9v8LO7jETTa4iHlGhELEaZ8\\\\\\\/54c9XshkNsxhDc+oxTHdYF+1+uIHohZT4B78qOTErku3Oun0eZY7SsuyVHihn6cqSlKJbRbFVazWrPj33szQkFeDmBJ40PFZTk5Gp7VkPP8UQYWnZQhzWkfQ6UrYcBGxN7x2IpVuj\\\\\\\/frGrlpXGcC\\\\\\\/wEq4uAUbZkL40JPRbboP1zq+Q3rWrDErl3s6H25AHcqFcfdYItXwd2VGiBEzNI2NqAMZoiVjdnU7iiQUspbfKNoy1ixZQP4Tw7eplY81OhXJNWiyQQcs\\\\\\\/ngRrVcVZHV4CHOg8tLts4ebJH2vzITF19JaSHUZy07Fn12A+RYTddENVQrxyRyc8UV1Y9jzEHeNNLLfeX15we0bFsjyr509OsM82NPElHXCJOoLvxAA\\\\\\\/6HrI88RD+L4xmJG05M0D3HLvX4SaOjkDiaGDCcA4IzrjLP1BkpdA5920Rbew0ze\\\\\\\/LxcmEmwfxnFNXJRYL0+jdLjz8D4DuQG0rtMtoXpZJo8OExtlWGJSdTwXetu66uZ6ie6C0cHxZrKoRcXmMRaLHR66EtGIEg2ljWzhuEjD6dCFDA8wBQoNGfmFFdjztB829q240wI9cE6TYEq8IJED3pl3qrcBWv+CiIrWvxcl7pWcrQyGtMY0ku+M0Csrxf3pJaOdb0ZkLXagV55DE36OkKgY5niXUH5qOkJFEkT1ZzRBj9n0olqXffmQegnfLa8966keiZx4tfSh8Lv3+k1T2VBEMFjb6MH\\\\\\\/2VygAZBoAOFY23LlN5kHuS6Kjth\\\\\\\/oSeijOAg628\\\\\\\/SoYbOw2TGebddFdH5\\\\\\\/sVSexBEerVQkw8USzT+1O67R18iLo1fEjAhRkABPeCQ4rcCooDBqDYNgtfV0dCreMIOW1rDuvW3lcfab70LvEk9SjOtZ0XuZ4yBTlRF+UYPxiKr+61neuszLLqUNg0yrYGRqxtw8xfa1my3vJKs3BK3sBsOimlKnbeAFKui+\\\\\\\/UJe7arbCC2JkBcK+4bsEv6Vt4fCKd++ua46Ly5AsqMsVqTOy3N9RbynmoGNM61qv\\\\\\\/zrNf17O40wK4gIs2vHFrRw1sDTMs54yk4IXFYfC251qXu5puXGt\\\\\\\/HaXfWaxnH6x9j\\\\\\\/IOfH73gYluS3oYCgtq7+KNkkIVSP6Lhsq06JWjyw\\\\\\\/E31Zfu\\\\\\\/WDNIxk3HFMMN\\\\\\\/wEr8FFKyaFRspv6X4PE2Pmz5Z5d3eYyH9CLusgTXy0YfypgedEQlv5xUazeoCSi0Q9+WZk\\\\\\\/RQ8ONa8\\\\\\\/GAOPWyHzzGjCgn6I+UY3RrvKuZGejliytGujGDyFI9K8+mrk7Kc\\\\\\\/yUs5fJNODcLmwoBVc2Mhny4e9ZWJ7l9xcCUWx4AL4vCUxbIGdFw8cq2ywcrnqZE+eY5QGYE0nU\\\\\\\/3Bqpt\\\\\\\/Cg8wDo+0sXx8ofJtx+pGVRIgZRH5mxkLorvxmS8pVsGRdVdvv8QeZfTRpeRl3GPXgOSP+ibszKhBTQ7hHlsf\\\\\\\/PLnoY\\\\\\\/jxsnKHAMdxRXjnLf2sVWsWxBLfqvP596i1UBEjS5XufyF6\\\\\\\/mglCW+30NfW1YuMmX3OIAUNrW4Y4O99cHBVXpyix7VcNsACsL0vM5hpurUh8TjMBk9gpOgBMJliN1o3\\\\\\\/3o99VtGwjm9gk2GqR\\\\\\\/2nENuuASdUjOFdGvIgpOnPCbwThldS4GS4xyhQP\\\\\\\/8bvbRZWH57Ru+vfzm8GrMgdwYp\\\\\\\/AzuZ9K8k4Jx0gJNQTLqVHizSIEKwUL3sSJnyV1RlupaBlriQW3hZA38avyfzfbJNWnj7mzet1oc6kji2sHyHOp5v23JZntlT\\\\\\\/IEAVpbHzt34ow8WkTR2BrHTv4MHfyGg\\\\\\\/qxuRdRALuepryDO2hlPV9XNlPbstBTJVcupWZCwAf\\\\\\\/cxLCk8\\\\\\\/5OPmQ\\\\\\\/ZydFTII\\\\\\\/IzTqur3eSt2u9TANLwCf8RpziWjfsI1d+8D9UZ5v4Y0rZc5YV\\\\\\\/ZudhXfkVH3\\\\\\\/y\\\\\\\/V18y\\\\\\\/D7OYE+g8qRbqstIWCtU3Uy9G4LkUKhFCjNZ8\\\\\\\/9WSvUJoX5UmVO7XybWZl5M1zOH952YwuSLsswPfWr4hs9ONMpfMQavd6USCpU44V7tdgAB4t91k79XWNUdFtuLedkDr6QzUqd+4k6HpLj\\\\\\\/ytYHDFa\\\\\\\/Y3+LBmARnpY408nY6vjxZvUx00igj8cM9X15UXAIO07aA\\\\\\\/iZ6kqyaIj98MVt7Db6Pod+UYf2Ea9S404WDA6sduavCj6JCdvKOoqxkjnPv3eDkwmOrUHwPpPRmktaMhoNzqnxCSvpy9orRP0wAkQyHcU2h3fFQizfykENcqHhILnWZoaWaApo+kN7jYjeCP39z4sXV7Lukf5ggMZCRG6yFKu8xbTFPvLZ+19gqPI8ME4YeJxrnTt\\\\\\\/TKaW2074tXplB\\\\\\\/71P0tIsTQBD2GRU1lXvyS3nnwuO+\\\\\\\/N82hyu4d8BtPApJ2kwmfkhFsUcwycAZZXqBwgMF9qohibgnvmjCj8QwzAH+aN5f3srpm\\\\\\\/6og3Kn0uvPZwJTzBRj\\\\\\\/P0W7Veltn77jzgJ0FHFEdZIJDWl7d+HkrugMICNLRydR+Uh27q7lAdgc6s7+IiYzARaYLBvrcbCH5RnVqBlsmdkyiRzYtB+m8+aBnqjvTlErx\\\\\\\/jt6KL8mAyENQ88s2HMicNR2Y73HrAAh3DbC2HlsxgyC7C2ThMyDE6fu2qUO2aEcloj1Ss13EBGjhCFyGLeVkLKTtaoQigHnTe1naA08OCv5OSzuUrtRJcW+U6jGkxXqQfWVMl6v5YQUlAD3XJ87dSMukQBrq4O9ViOLnsKeowXjjvAgdceaOXejCW\\\\\\\/MPY3snfWc59rFuu5L0K\\\\\\\/v8VVS3XMHlGP\\\\\\\/I668QJpy847nbXlpM+8VAgrf+j2nyfP5Vs2uiiD4So4NqAijQ5hOZL+LJLWMFgYRK8AglMHDtSq9SsILSD7sodQ0\\\\\\\/65z48PgoMlr5+UzMi6H6LkxhU\\\\\\\/g+5m2VIPtE3tH3+EB5R8RH\\\\\\\/RoVUegTFdiZO\\\\\\\/3xjVWftlhBgUVk24v+E5EQOy0J4QV6ZEj2FvQA21lxskOtCwPRLur\\\\\\\/JrBN1HQcWomYX3gtlLJhJFdrz8cBr6dkfCtSnVMQ7p2cNiBPofliZyAHZCdWELii5yhgSpnQ34iTo2R+xNf58RnTpT6eGtts7JQkj0KDYUQYDCGLhoeQbM8StMSqPq\\\\\\\/UHZHaH4HAnERyTyW2ByhukpI9LYC6eK\\\\\\\/YlS3AIVmvgYO7Y6Sv\\\\\\\/knii0G9mNwayKlaijhS3YRHfhofWSpzE4rVmym3kMgLTcEZcnUaKyKXh43CuFkBZfcMal7\\\\\\\/EVBEPO8Mw7RRbkYpp3Cw4eWOtJjOXRB6C0wctFFEn1Fdg6SZ7fWInHsQMwR7ZH180c8ldvAGgHFrgtUp4QnAJoHlVxWHj9cep4y4X0SsSL1iFUKhJqZJmZ7kBbM8XHWl8H4gnVF8nlVueHZ4Y2pDljcwsNSAU84pgmWlulhTcqPTEWkeeqniDUdkstYj8T8FwwxIUDJf+9nyiIY292CuihIyZ1+Zc3QEsQ41BMxSZRD2wuQyWYH+z8hzYSfpLKeAHWuFjLTucwV5SBY1TYlCmWCz1NJ5DhAmCUFMdM5VCo8V7OfooNkICrH2ObcXXuSrzYJ9rVkTLdH25wVRQWfFi8PQbpUiH3VM\\\\\\\/j\\\\\\\/3XEdecRe8W2MxREp\\\\\\\/MAGWpJwLogqA5fe9eToWek9ImgvdR0cmRmOlCh8J+8AjjzFsYBXQudsuRhcQLb2tyVslTlxujA19cFAEf5vDaR1fU6t80AD0v31QuuIhyqPs8mJSSIQhLmCo6tfRA\\\\\\\/u2xXaRHfjdviYCJgE2MuqD8lDxPOMpvfj7YMl5HuMnqQa6O6YI1wLc8DbCjX+UbskkI2Cvtg3BEIiCIZKxCDzeHCgSWWtSjC1+Dd\\\\\\\/nB3\\\\\\\/5X7FULCpCdPu5AYQtlRYC79N0\\\\\\\/uTA\\\\\\\/yaHuT8JSVNpohVADmDuxpxCah9yAzZnQb0kAQuS57K8\\\\\\\/g0ey3EXEfPWqFP4ofL4TcVE05MlMqLKTWUpa31npGrBtqBCaIqcCWi8Iikqo9BjORYnDeUV\\\\\\\/+BEahIQoqas8i1VQP24LFjpzbH80TfKKv+2uvXvkVC\\\\\\\/V2nMLcvEVRlXFrc4iIvs30\\\\\\\/RxbsoOIcNGaU28CgIdiCcgvoQwqUxsnaame6IvdIrwSlXRqMGi+SJTBynyt+8pc6B6184prlZNo4xddxbQ4H0MALAnTwz7GS+42ZvKq4DXtkcJyGJ096BzO\\\\\\\/E6uH7qxvsom92eoNfgT0eQW4\\\\\\\/q5+l9BHiCYpNPT5lWAgvO7ISUsp4G0thvM2ld7nVrpQaytlHePW25zizZiw4sunxdHDxu+UsdJElp2jy+BRaHmUJayxrz3X9bD+IWGfwMMncmCrruUlGGhCPvbLvWrYYwU+kT+kvZaoWT8q91ief5DH39fHF3L9Fhmma\\\\\\\/DsFtHaWoLqkjRcau26coRsApAaGNT+6tSbBCZWh+1rrmbmyNMJAhMVAv19vswwvlAR80\\\\\\\/cEGKBKpzAWNu5rRHYA8u9CJ\\\\\\\/zs4GVI0KpfZ7Qfr+ieTOI5nNonxHVyutYJndLUNX08ynsuOa4znHVzPJiVyWRShDgDqAJbyTQMpxImOk9cAIStMW0VDYJU4ff\\\\\\\/s1x5wpAPxh4BT\\\\\\\/6pZ2yqZRUeIyUzw60SXeN3NsXLHnSO41fWeMPw2hzX5ANm0M9vHAXMdVoaVy+EpLTiYg+C70NQfjG\\\\\\\/p6vJK4BwM66rLGKzP5p9XRYjRiL0thV3KyMuXKo+32wsdS\\\\\\\/AUwT4BcAM26+LHrwM8PlSd5SIXKKvyTrMEEuSA\\\\\\\/2\\\\\\\/SZeIYLrRd+U\\\\\\\/9ocBGqRR60uICI1RiLKYHMzYAbTF\\\\\\\/l0ecHAib5LVW6g5wGo+1L\\\\\\\/QAT0OF97w\\\\\\\/6M8X5iZLyADT1dWAQxKZInS+SVbs+3RYHDjBzZ5Rg963bvgnqZsD\\\\\\\/FqXjgBMN\\\\\\\/QuCpU7Zpq3wnStjatiAUc8QBDd5c1ua3EBqyFrB7oSwrqK3Il3T5GA4IhW2y43OAcmO2CQtx1dRlcXg9CszF7f0AKc0SrGTU7Fr\\\\\\\/afu6yaZ\\\\\\\/CP+7LIk82GQ\\\\\\\/pYJ0ypO0XwoHSS4wTLQ6bfk\\\\\\\/icZRxnwAqkfbAHoBRAt7JSaCZhfoDnFHokxDqouNbwgE8URRZwxw5SAQy3aBI0HLxPEVCT3jMDRtmOSme6SKrvGghslIgzCEugBbxfOyg2MOmOiTQGp8zURAjIdUjKzETNxULFgH4f0UePLXgJluewWa2bPsrEgGDzggKM9RWg\\\\\\\/3YOQvvfTtBW54ac9BjepFgPkqmWCkOoZhz1buOCSPqzhSLwtkHjf6v6O2bbPh\\\\\\\/hsR9QlGPOqt+Ji7QFPRr\\\\\\\/kMTt8N4St9tNlIPpGqySG\\\\\\\/NvS42pUHJtazSURcVpo8FKQOyakWyCMgtIX6\\\\\\\/CHhDN8IL55\\\\\\\/7X9AndtkPaPlKD4ma1zZ9LjuMEiWgc0OnpxqYfWmB\\\\\\\/9Y8VTo+\\\\\\\/pedswA1UUCcnNqTnuS7CRLJlIf3lshO03p8pNisg\\\\\\\/hiUpZLRAV+P\\\\\\\/DH+Kbn7UdA8V45etsxzVTXr2b+4J4Byy6fDfLwafz\\\\\\\/WShmHt9nrWKeBG479SC5jwUoAO+tJWpETWqnOwsvOmAiFDRgGRFDmESkqiZerArfDKlcFT3KEWN887PVAEO7TkNZ8wYbjBOM2ytVYixq4pNlGTwyOoKnhfSfp0rJ9UxWHpW1S3FQl4Uco8iN+ZRacdLEAKNyvk58gm4Z2ZHwq4nz7vsFZaxqSspy\\\\\\\/a88Q8niMInBAbDJoEN5QoOWhSfkFhI2p0zNXy8W6zS1S5Tm7\\\\\\\/Sa3QStDhxLtljTh6iA+GRZOtfT0oMg8jFPYenxwWhaH2GpOUu4BFdCZ8Po\\\\\\\/5o+sagW\\\\\\\/CQWBRJi4MwXjEXrjM4D+y8tfjmNKfdmH7KAGqPocJ8QdQcjCQ8udaE+1dSt2HDcAKeUx0BwJLCrZF1Ooz5LGk4+jQkhrPwBi9p23KNt3Ao9nX5MlVIE4\\\\\\\/B8+x0tBdtdEYQyZ7Hr540fIN34FFH7x+8RjB0GCjkebijiiXB6bUziytRhiqKbRoI8ntdCpMngP9tuCds3+zjDe5FZC6Sk39TEsGMZaesdsvUVKWqDgcRCVsMzA38rNDPpP4wXRqm2IJMBygqAWoVzB8fT0OB36IIndlAaUnOLu7vzt624wKF95SroAM3eAMagkbnBcVFkSrkiI+vyu4Mhm0qrc3ZjW3JWkpqLX+iNYInyjzD5+GlXtLoIcbFPlZbfiUic76kjhH18OC1aXDbaOQ5vNiEO43mEnA9CgianLUZPYbYd4D2eHfMLugPVDUg8VSuCPcmyN\\\\\\\/O1+Wp3F6nzPJGC8Y\\\\\\\/1vreOQbobAUj5A4FJgsFeuMUQk\\\\\\\/SkGr+LZhBVtxznZw4YEkLilSfoganNyPfj1+qN+Lv939m0+v2OeyhMwJPNJshWudlaVU8KqzYwOotySB5eg2WMPdye406wYfJyeVNgJwdbsHcu6hCkemZXoeAWiPhvluSV3cy9i1oN63RtQI2U1F1gotZfe0Sf7xZHkQuwzvdMMtLhhd06L56iCR+dKLGCdCiXlEHb5wqErBE3EnsCOlRUE2YTzr5sKA+41DSdENNssG8VchinrAd4bAUg73xRS3VkwkxQeXWC5vgbrzNyKn95qsLmxNw\\\\\\\/c\\\\\\\/5HKwf8Pt890iHZ7cTAE21Bt4BOiAU9BTiki6XmssLv7L74zYdm2741geFwpdpV\\\\\\\/B\\\\\\\/yosqVb2PeFsWoXhEMRQp2rc\\\\\\\/mmXwiRy2IiTITkH4b\\\\\\\/gcCbRAUPnXlm41rbBG9NXeb3aB\\\\\\\/CfHyNpVawewHfGV9RGvHOItshtvFhhlqPJzj9ktcMtBsVjYh1RXlkZ4Cohahz5Tvvmil2ZfDTQPui+WcHe7xnrKIKmBkV3xeZAvOiP\\\\\\\/Lz2YDIdUdVlw7CNa65yLBJJ9UG2YtRujKQH34T9b4awn9D3XGZPWm6+\\\\\\\/CUrd+jemHJIYWhSTVfkYpnCzWBkaUCiA8jJcj4Cx3N\\\\\\\/DWxJYfr2HFSU89EJXmuK0mlm0NyB7YEdbBxNtJK8fR44oe8g8kAfDPuy6jy0Us6umhIaJib2GXXI8tvDuXi981VL40MtAIt41dgVQ33WESPOqhHScFcNAJw1eyXKTQURavAnDJXAmLIcZcCkGyNqTfhIBjfNmZ5gXaNhqsahC+K2oDhmHR7RnmylQiMuneP7Es3GOFD7qYU0WNvAX7oNiXlWQEmUepQ1HgrT6uo0GktxgFwIFPhrf6qbuRwqjJRnq7+Pjmzsgg2Kb6fCJheXHBRTAJJAbZdABcZY9pPhALbTiU4mRIrSkIMQL9MRE0gmMELtzpdgDPMhGn1bf8jMnlN75YiYFcvsZpFIwVCKXAGxgGxG3MPU1WA6x42ldDA70Zn5H67VO6\\\\\\\/0Ppn1OGGD0H8TVXjUgzKw4epEC6c7cwjQ0AqCCB2RvcXweU9oEguXcGOhQqNt\\\\\\\/tpn2San3VbHM2JfCgNE0M6l9te5JGZNPV89ATQU\\\\\\\/H16fC3a5LTJr4OPPbrpZ6tNzh92auwGv5wmlaXcsn+lvCmXnHA1aLTN9n61Msen5L3KyAsa\\\\\\\/ydxvYCCEP2HLnt1DBNmIQ+5MkCdZQmdpBpTg\\\\\\\/xvDnpvNPKv6PIHX4ww77GlKCF61MHFrFCYSH3\\\\\\\/wEyO7XNJZjRmXd40nnAvD\\\\\\\/84TrP9gLDXNKsjCbL6m1\\\\\\\/7Qv0nm56Z3oz9\\\\\\\/EoUR0cl3jgyIDgvsb+D7G\\\\\\\/q2fClOBbe5L38KTPFPI2keABz2tjPTQ\\\\\\\/frfD8zKVLEjDUFOAAu+zo\\\\\\\/17nQPZQzbW06zhr75WnH6cm40pRK3j5UZpIsIpwDcd0XBMZA98fO\\\\\\\/YBsw8pLqdLKC5LtVplAVFOk5Ixp1mTusHu7pa4PuV6bfxMztqxc4+DgkDoAHEoxrHfQhfmOIVwq\\\\\\\/96tFOgNH7BPxYH\\\\\\\/Ipow9QtlmKPsc3blt9AEvr662XYmkIIScsI9QnEjJAfzJLBIx2qDcjG6HD4TPfHrWOdu10uVkjW6SuaT0Ka42di6LfMXMjVEDzvSbgbOsTIAMDBF4d8TrjzD3BaYQJN1LqX1cx3tAanLxfPc4yZEvJGc0ucE8khghjk8wlmXZsJqQuYBWdamfDRLVZ8LVPGJhg7dXPohSxioAovLRjX5gEjPL1GCz4vuQEQNYlFP8gzfcoZf3fDVq+b3\\\\\\\/K2hMpEh00VA\\\\\\\/Cmu0VdVETVIVDzMN7Zy3ow7\\\\\\\/K2pn8mDe3ChnmxzQZTGpp+QAaNaM0P4UUQWWd+Off9LECA17oUqEhsi4obW+AL1XRmIEpBDVrQ\\\\\\\/5dg7ep5g4lMT53KIHuxJhkQb7wum34eH3yvwYxJnOWIL5ZWE5WdDN7XAiJPJNx7ox+l55pMFcSrHi7MegtTv0XpTgeSH\\\\\\\/d2XhyAuDfGi4zBIDpn1Rme3mDZksyHt0kaZmQFVn1UhhYHPsKjndc2RzuB3HXXXtDlujSJMx5KDpgrkrxeZ3\\\\\\\/y8R+\\\\\\\/xlIN4HgbW3Bk2WSQ6EMqUgJaVIzJ57Ap\\\\\\\/bAjLr6BmXKS+eq88TCR2QaEUMZFeK71mmkvoqX5WQCfrtzwV6L52Xj6XyO2sW5Axumj2VeBlaryOejDfoa9iYivCoRIb1EOtnEugL2lpwobKW6C7pghgtv8cMDtxPEvW6YTrgG7FUcqE0q+MXHgsUOkNWQTtZTz5RGhBR+RjepGtKZkY8FKKnpOLCr0MS6PhuKZMUj6FXgHB2gp0cRfnwdGV6FRiFPRN97jZ9sRp28LS9BLAWd8PVpJqunciP1c5bQxqQvReTg8KRLUE44jivUTucFq7gr0QJ+P29epQ39OKLndWXhH4JkNGwM77rQmeTy8rRd8tivOc3KhxITcqf06pFJXUQhf8IJmjAPV7Y6va6BR\\\\\\\/hpF9pY67+AywAOuvAkQo2+iraBKQkW\\\\\\\/WWCtmuP05pwjvcU4Wxj+q\\\\\\\/6CW9o5LfVm3P2ia\\\\\\\/hgOV4P7QXbehUt3HiBHZiiGTX1QsRjTRHuh4i5oQRuEUgPWBQphhQubXOjqPn8mfMofNHXMeR7DOmoVGTaupU0UQCOxqaUq28YySC+jnJEAE\\\\\\\/Q6r8g5Vivm4vh\\\\\\\/mjroV+BeZkl76gowc3BIERDL8rrlOSMpMx\\\\\\\/xsKh06RRICjzdQsWBTpNgdzpX17lWU3axQYeP\\\\\\\/SnJRlXUelaus1+CsUHBFKE6PnWikAKnVj6MaDKB4GB6TywBqGaszbBYR40STDMThYA8YeEFxO6jj52O4XRw83dZkR6hT4vAY0WQLzTQsRDVPMPRAcfmpqSKYb7+llOZmaJ5meLmQvBa7AT6Vl+CzZD5LOdA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022j3KPNYo7BZyJfkFTg7zfUw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221f7cc4218c0ef1093280da5e36e0a52e89fb4490359c6d0215d1bc38450e2ac3\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yJY+J\\\\\\\/aLwlj9tqeducMouJu5ENxOGCNPX8cYuTCVyA7WP3w5lrXoQ7qDr4LfkHoy2ggjklOtmggaDMxweHidQjEyBoCS5LhRQeVyKZpo6S5qjsuJINrApfyr67k65evINZT3Dm7I9XR6N8r\\\\\\\/nnJo2O2T+iWCGCVqt4nnd1rl32gKm6F01gv\\\\\\\/WZClrapo8Mhtrd8KHOt9lxEDv+PNbJb5APevK7oNssBCGqt7IIvZaDAAHKHbKx55EY1PHd2CM4DzOKvYl4ukBUj1h3cHiMs3eeVK\\\\\\\/MkIi7R\\\\\\\/yqaYtsB0enyc67i5CYEhAMelov1zHC8De3hmEOPhVzRQD4OAMoBo8IYJy\\\\\\\/CS7s7UVd7aM6llqFEemkDfS3WaXeHMNz0WhEFr5sjNjiNoSwjT+IiBi0XUwuHcX3Li6\\\\\\\/QbmdYaiHJvabxMzxT7\\\\\\\/AjvCkjpUx9yw7RjHwtP9bWEt\\\\\\\/7DzQTe3nKfKgC5\\\\\\\/abmhTdbFkA3nQ5te1WsAhuWBuKyFULGOXU\\\\\\\/CFHTIjNUXnbPXrarO3SANuolJXohRsKjocna2S0GYUiE6bsk1sOv8Y34\\\\\\\/OhI9GsPC6G0xI8GLuGRr4Cu6Y\\\\\\\/PUTeFZ+csLg89wUUMrDzKg8RjKt+BGdjV3D58ihcgrz9zSbmBkTTP7fR0zuro9Qim918K5PdKHfY+bfPML2gU890WbTKS3dNebp32ixlr6fMfcBG6H472zpcuFzU6BSd23oSvIv4VsuMTxgJ9Xe967EV\\\\\\\/+apVOqEjAuU3O+uQ4PXjIJ6ITMBL12pd32r87IZtgRaO8JY\\\\\\\/OzRXPswOHGMsvvbYtpJX0w6kUNL4GTC\\\\\\\/XkZIPw8BikTv1hQPKYTq70Ev1fhQ\\\\\\\/nO0rr6\\\\\\\/pTzY8oCaf1mvlbdOp5mdgQkODC0BXpHXXuFjYGHIVbrnvdx6hd2A0zBfPzIAI6ZZAR12XSh73fkgDbe1tSTvdlCAob+uTB2k7gKN7vlFpWw9ferGKyJyPJLI9NOSMpNTNqINtsTm1JvAuLX3CuoFktof3a6KNXhOr38dKJaxFZXFPZBvdj0qn8wd2zkGQAwsGtPAyuwyPL1JraUcoZvOyZkRSdyLBY3fyeT3KeZdsLQ\\\\\\\/HyOec9xFqee8DXisedcfRUckcApRhX83pD9YdjtLKDZH1LSzYxe7sz1bzYokpHMpbgbZBItetCZLQOjCY8HoI0N8KVcGFiZqpZM83LlLrLJsATO1TtWQPEVaBezzISYoZJPS1S9XGyOBz55dF12Nnu7+19hp56rAtdBRWKe\\\\\\\/4x9tb8ecP2YJ7eb4pAthPUdj0FScigy+Q8+UVup3Pf\\\\\\\/dJQpwS8Tmu0lJNaiFjo\\\\\\\/KVqf+Jjp6e6EAEYT8qQZN3X7gVFaEA2aAeWj4zU5D3WnX6HKCrJJeCHoQ9rLEmAHrg5iVE\\\\\\\/AgkmKHJHPScMLcmn5S8pG68MKNB3TWCKOz3CWMzJJoQ3FvO6uMyVO8mLWnFJU3BqKlIthVXLxlIFnRudw53tOf8BGQncmAKv4DOWntQX3wQxFxLDImiqhih+wweaofvaPjsGVEJtfG0XtWksevp7Dpo3O30CKK8wfuh0B4aW25P5X\\\\\\\/3L6wiIlw1oEj3iVy4g3ew+KHEEF++CgdhXpuDSVx5rJdqEeI95avUA1jNkmLjAZznBPc1dUAmUwyKajnsQdfg+sGM9iOnfN2XWAJRBwuH+I7yLaeVL9CNU3s6twFWrj6TEDVfBnECCw5q7yUPYDWV0wzl\\\\\\\/YAfWlbVJe2tt0E18rNJUu6LKVyRB64w8qKdH9IllNAHY+DT3nAcrL\\\\\\\/PvqZS9yEHiBVDl3o8hLszl6AGNltxbqAFRK6S3eCEiAlSrc\\\\\\\/Akd0NiGbzj6AUe4nxzyAxcCpQnV0m8of+DAk66D3z5BsDqve+0OXPGb5hM1zuSUEAfOeq\\\\\\\/D8YFTg6f7EInWMoumjzZlLFcDHddu0e\\\\\\\/Ud5nMKG+PDAFGWHRFPMumsIQw9JQt47SZUVQXKIwDMz8Q74zdV2cMbgqTt+6cbldJr58SFTw2J2w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022LS2sutIV2Ogx9NcCZOrGkQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225fe673fcd769c321e87025d36af37b3c8b595f575a415036a8f4b35b7472c030\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-486770 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3427" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-o8zPNvl4qOittTPQTbprJj5dfla4hV3R1Ij9HF7xSy0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3427" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-d0un4oWrzkIPDCrKmgXdqELk7ITcN4NG4NjXmvs-2Mc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/10/3415" title="Whole Genome Sequencing-Based Mapping and Candidate Identification of Mutations from Fixed Zebrafish Tissue" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/10/3435" title="A Comparative Analysis of Genetic Ancestry and Admixture in the Colombian Populations of Chocó and Medellín" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486770" id="node486770" data-pisa="ggg;7/10/3427" data-pisa-master="ggg;g3.117.300188" data-apath="/ggg/7/10/3427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Resolution Maps of Mouse Reference Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2922-7183" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Petr Simecek</span>, <span class="highwire-citation-author" data-delta="1">Jiri Forejt</span>, <span class="highwire-citation-author" data-delta="2">Robert W. Williams</span>, <span class="highwire-citation-author" data-delta="3">Toshihiko Shiroishi</span>, <span class="highwire-citation-author" data-delta="4">Toyoyuki Takada</span>, <span class="highwire-citation-author" data-delta="5">Lu Lu</span>, <span class="highwire-citation-author" data-delta="6">Thomas E. Johnson</span>, <span class="highwire-citation-author" data-delta="7">Beth Bennett</span>, <span class="highwire-citation-author" data-delta="8">Christian F. Deschepper</span>, <span class="highwire-citation-author" data-delta="9">Marie-Pier Scott-Boyer</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fernando Pardo-Manuel de Villena</span> and <span class="highwire-citation-author" data-delta="11">Gary A. Churchill</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3427-3434;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300188 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486770" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Petr Simecek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division Biotechnology and Biomedicine Centre of the Academy of Sciences, Institute of Molecular Genetics of the Academy of Sciences of the Czech Republic, Vestec, Prague 4, 142 20, Czech Republic</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Petr%2BSimecek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simecek%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APetr%2BSimecek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2922-7183" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Petr Simecek</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jiri Forejt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Division Biotechnology and Biomedicine Centre of the Academy of Sciences, Institute of Molecular Genetics of the Academy of Sciences of the Czech Republic, Vestec, Prague 4, 142 20, Czech Republic</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiri%2BForejt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Forejt%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiri%2BForejt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Robert W. Williams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee 38163</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BW.%2BWilliams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Williams%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BW.%2BWilliams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Toshihiko Shiroishi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>National Institute of Genetics, Mishima, 411-8540, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Toshihiko%2BShiroishi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shiroishi%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AToshihiko%2BShiroishi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Toyoyuki Takada </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>National Institute of Genetics, Mishima, 411-8540, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Toyoyuki%2BTakada%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takada%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AToyoyuki%2BTakada%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Lu Lu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee 38163</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lu%2BLu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALu%2BLu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Thomas E. Johnson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>University of Colorado at Boulder, Colorado 80309</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BE.%2BJohnson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnson%20TE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BE.%2BJohnson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Beth Bennett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>University of Colorado at Boulder, Colorado 80309</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beth%2BBennett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bennett%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeth%2BBennett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Christian F. Deschepper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Institut de Recherches Cliniques, Montreal, Quebec, H2W 1R7, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BF.%2BDeschepper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deschepper%20CF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristian%2BF.%2BDeschepper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Marie-Pier Scott-Boyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Institut de Recherches Cliniques, Montreal, Quebec, H2W 1R7, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-Pier%2BScott-Boyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scott-Boyer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-Pier%2BScott-Boyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institute for Behavioral Genetics, Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo-Manuel%20de%20Villena%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fernando Pardo-Manuel de Villena</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fernando@med.unc.edu" class="" data-icon-position="" data-hide-link-title="0">fernando@med.unc.edu</a>
<a href="mailto:gary.churchill@jax.org" class="" data-icon-position="" data-hide-link-title="0">gary.churchill@jax.org</a></span></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fernando@med.unc.edu" class="" data-icon-position="" data-hide-link-title="0">fernando@med.unc.edu</a>
<a href="mailto:gary.churchill@jax.org" class="" data-icon-position="" data-hide-link-title="0">gary.churchill@jax.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/10/3427" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:486770" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:486770/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3427.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:486770" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:486770/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3427.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:486770" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:486770/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3427.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:486770" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:486770/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/10/3427.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:486770" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:486770/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genetic reference panels are widely used to map complex, quantitative traits in model organisms. We have generated new high-resolution genetic maps of 259 mouse inbred strains from recombinant inbred strain panels (C57BL/6J × DBA/2J, ILS/IbgTejJ × ISS/IbgTejJ, and C57BL/6J × A/J) and chromosome substitution strain panels (C57BL/6J-Chr#&lt;A/J&gt;, C57BL/6J-Chr#&lt;PWD/Ph&gt;, and C57BL/6J-Chr#&lt;MSM/Ms&gt;). We genotyped all samples using the Affymetrix Mouse Diversity Array with an average intermarker spacing of 4.3 kb. The new genetic maps provide increased precision in the localization of recombination breakpoints compared to the previous maps. Although the strains were presumed to be fully inbred, we found residual heterozygosity in 40% of individual mice from five of the six panels. We also identified <em>de novo</em> deletions and duplications, in homozygous or heterozygous state, ranging in size from 21 kb to 8.4 Mb. Almost two-thirds (46 out of 76) of these deletions overlap exons of protein coding genes and may have phenotypic consequences. Twenty-nine putative gene conversions were identified in the chromosome substitution strains. We find that gene conversions are more likely to occur in regions where the homologous chromosomes are more similar. The raw genotyping data and genetic maps of these strain panels are available at <a href="http://churchill-lab.jax.org/website/MDA">http://churchill-lab.jax.org/website/MDA</a>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/chromosome-substitution-strains" class="hw-term hw-article-keyword hw-article-keyword-chromosome-substitution-strains" rel="nofollow">chromosome substitution strains</a></li><li class="kwd"><a href="/keyword/recombinant-inbred-strains" class="hw-term hw-article-keyword hw-article-keyword-recombinant-inbred-strains" rel="nofollow">recombinant inbred strains</a></li><li class="kwd"><a href="/keyword/mouse-diversity-genotyping-array" class="hw-term hw-article-keyword hw-article-keyword-mouse-diversity-genotyping-array" rel="nofollow">mouse diversity genotyping array</a></li><li class="kwd"><a href="/keyword/gene-conversions" class="hw-term hw-article-keyword hw-article-keyword-gene-conversions" rel="nofollow">gene conversions</a></li></ul><p id="p-3">The laboratory mouse is the most widely used mammalian model organism for biomedical research. Among the key advantages of mice are a well-annotated reference genome (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chinwalla <em>et al.</em> 2002</a>), over 100 strain-specific genome sequences (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Keane <em>et al.</em> 2011</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Morgan <em>et al.</em> 2016</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Srivastava <em>et al.</em> 2017</a>), and many genetic reference populations, including multi-parent strain panels (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Collaborative Cross Consortium 2012</a>) and outbred stocks (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Churchill <em>et al.</em> 2012</a>), and strains carrying null alleles at most protein coding genes. There are hundreds of readily available inbred strains that capture a wealth of genetic variants and display unique phenotypic characters (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Beck <em>et al.</em> 2000</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Yang <em>et al.</em> 2011</a>).</p><p id="p-4">Genetic reference populations of mice include collections of strains that reassort a fixed set of genetic variants such as chromosome substitution strain (CSS) and recombinant inbred strain (RIS) panels. CSS, also known as consomic strains, combine genomes of two founder inbred strains by substituting one chromosome pair from the <em>donor strain</em> into the genetic background of the <em>host strain</em> (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Nadeau <em>et al.</em> 2012</a>). The mouse genome is composed of 19 pairs of autosomal chromosomes, X and Y sex chromosomes, and a mitochondrial genome, thus a minimum of 22 strains could constitute a complete CSS panel. In some cases, it has proven difficult to introgress a specific entire donor strain chromosome into the host background, and the complete CSS panel may include partial chromosome substitutions and consists of &gt;22 strains. RIS also combine genomes of two founder strains; they are derived from one or more generations of outcrossing followed by sibling mating to produce new inbred strains whose genomes are mosaics of the founder genomes (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Williams <em>et al.</em> 2001</a>). Both RIS and CSS panels have been successfully applied to the mapping of complex traits (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Buchner and Nadeau 2015</a>). RIS can be used in validation experiments by selecting a limited set (5–10) of strains that differ in haplotype in a region of interest to test for effects on phenotypes without the need of screening the whole set. They can be also used in follow up QTL studies involving the mouse strains present in a RIS panel (or strains that share the same haplotypes at the QTL).</p><p id="p-5">We have carried out high-density genotyping of three RIS panels C57BL/6J × DBA/2J (BXD), ILS/IbgTejJ × ISS/IbgTejJ (LXS), C57BL/6J × A/J (AXB/BXA), and three CSS panels C57BL/6J-Chr#&lt;A/J&gt; (B6.A), C57BL/6J-Chr#&lt;PWD/Ph&gt; (B6.PWD), and C57BL/6J-Chr#&lt;MSM/Ms&gt; (B6.MSM) using the Affymetrix Mouse Diversity Array (MDA). The MDA includes ∼623,000 probe sets that assay single nucleotide polymorphisms (SNPs) plus an additional 916,000 invariant genomic probes targeted to genetic deletions or duplications (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Yang <em>et al.</em> 2009</a>). These data add value to the strain panels by more precisely localizing the recombination breakpoints between founder strains. In addition, they reveal some unexpected features in the genomes of individual strains.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Animals</h3><p id="p-6">We generated high-density genotype data for six mouse strain panels (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>): three panels of RIS and three panels of CSS. Mice for genotyping from five panels were available at the Jackson Laboratory (Bar Harbor, ME) or from BXD colony at University of Tennessee Health Science Center (UTHSC); DNA samples from the sixth panel, B6.MSM CSS, were provided by T. Shiroishi (National Institute of Genetics, Japan). Unless stated otherwise, we genotyped one mouse per strain. Most strains are represented by a single male animal (255 males) but, for four strains (BXD14, BXD54, BXD59, and BXD76), we genotyped an individual female. Samples were mainly from cases bred in 2008.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486888/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486888/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Overview of the six panels: a type, founder strains, a number of strains, and a number of informative SNPs</span><div class="sb-div caption-clear"></div></div></div><p id="p-7">The AXB/BXA RIS panel (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Nesbitt and Skamene 1984</a>) was derived from intercrosses of the C57BL/6J (B or B6) and A/J (A) strains. Note that, hereafter, the dam is denoted first and the sire last. Thus the difference between AXB and BXA strains is the direction of the intercross mating that generated (A×B)F1s or (B×A)F1s, respectively. We genotyped 25 strains: AXB strains 1, 2, 4–6, 8, 10, 12, 13, 15, 18, 23, and 24; and BXA strains 1, 2, 4, 11–14, 16, 17, and 24–26.</p><p id="p-8">The LXS RIS panel (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Williams <em>et al.</em> 2004</a>) was generated at the Institute for Behavioral Genetics, Boulder, CO, from founder strains, Inbred Long-Sleep (L or ILS) and Inbred Short-Sleep (S or ISS). These founder strains were in turn derived as selection lines from a cross population with eight founder strains (A, AKR, BALB/c, C3H/Crgl/2, C57BL/Crgl, DBA/2, IS/Bi, and RIII). We genotyped 64 strains: LXS 3, 5, 7–9, 13, 14, 16, 19, 22–26, 28, 32, 34–36, 39, 41–43, 46, 48–52, 56, 60, 62, 64, 66, 70, 72, 73, 75, 76, 78, 80, 84, 86, 87, 89, 90, 92–94, 96–103, 107, 110, 112, 114, 115, 122, and 123.</p><p id="p-9">The BXD RIS panel was derived from founder strains C57BL/6J (B or B6) and DBA/2J (D or D2) inbred mice in three epochs: epoch I, strains 1–32 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Taylor <em>et al.</em> 1975</a>); epoch II, 33–42 (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Taylor <em>et al.</em> 1999</a>), and the epoch III advanced RIS 43–102 (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Peirce <em>et al.</em> 2004</a>). The latter were outcrossed for multiple generations before inbreeding. We genotyped 91 strains: BXD 1, 2, 5, 6, 8, 9, 11–16, 18–25, 27–36, 38–40, 42–45, 47–56, 59–71, and 73–102 (note that the designation of several BXD strains have been modified as a result of the genotyping results described in the present study, and BXD103 is now known as BXD73b).</p><p id="p-10">The B6.A CSS panel (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Nadeau <em>et al.</em> 2000</a>) consists of 22 strains derived from C57BL/6J (host) and A/J (donor) by J. Nadeau at Case Western Reserve University. The panel includes 19 autosomes, X and Y chromosomes, and the mitochondrial genome.</p><p id="p-11">The B6.PWD CSS panel (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gregorova <em>et al.</em> 2008</a>) consists of 28 strains derived from C57BL/6J (host) and PWD/Ph (donor) by J. Forejt at the Institute of Molecular Genetics AS CR in Prague, Czech Republic, covering all chromosomes and the mitochondrial genome. To improve reproductive fitness, chromosomes 10, 11, and X were split between three strains each carrying either the proximal (p), middle (m), or distal (d) portion of the respective chromosome.</p><p id="p-12">The B6.MSM CSS panel (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Takada <em>et al.</em> 2008</a>) consists of 29 strains derived from C57BL/6J (host) and MSM/Ms (donor) by T. Shiroishi at National Institute of Genetics in Mishima, Japan covering all chromosomes. Chromosomes 2, 6, 7, 12, 13, and X were split between two strains, each carrying either the centromeric (C) or telomeric (T) portion of the respective chromosome.</p></div><div id="sec-3" class="subsection"><h3>Genotyping</h3><p id="p-13">DNA samples were prepared at the University of North Carolina according to the standard Affymetrix protocol, and were hybridized on the Affymetrix MDA at the Jackson Laboratory as described previously (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Yang <em>et al.</em> 2009</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Didion <em>et al.</em> 2012)</a>. The MDA probes (NCBI37/mm9) were mapped to genomic positions in GRCM38/mm10 assembly. CEL files and updated mapping information are available at <a href="ftp://ftp.jax.org/petrs/MDA/raw_data/">ftp://ftp.jax.org/petrs/MDA/raw_data/</a>. We used the R software package MouseDivGeno (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Didion <em>et al.</em> 2012</a>) to extract intensities from CEL files, but; for the purposes of this study, we developed a genotyping method that is based on the direct comparison of SNP probeset intensities between the sample and the founder strains of the corresponding panel. We selected the informative SNPs with intensity differences between founder strains for each panel (101,397 SNPs for AXB/BXA, 79,808 for LXS, and 103,340 for BXD). Both selection of informative SNPs and SNP calls were probeset intensity based. For each strain and each SNP, the call can be either A (if the signal is close to the first founder), B (if the signal is close to the second founder), or N to represent “notA/notB.” We note that the N category includes both no-call and heterozygous genotypes, and simply indicates that the intensity signal of the sample is far from both founder strains.</p></div><div id="sec-4" class="subsection"><h3>Founder haplotype blocks</h3><p id="p-14">In order to define the haplotype blocks of founder genotypes with allowance for errors in individual SNP level genotype calls, we applied the Viterbi algorithm to smooth the genotyping. We used software implemented in the Hidden Markov Model (HMM) R package (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Himmelmann 2010</a>). We call the Viterbi algorithm iteratively: at each iteration we reestimated the HMM transition probabilities based on the Viterbi reconstruction of haplotype blocks. The iterations are repeated until we reach the convergence (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Juang and Rabiner 1990</a>).</p><p id="p-15">Genetic maps computed from RIS panels consist of intervals assigned to one of the founders, and gaps that delimit the interval within which the inferred recombination event(s) have occurred. We refer to the latter as “recombination intervals.”</p><p id="p-16">For RIS panels, we compared our maps to those available at <a href="http://www.genenetwork.org">http://www.genenetwork.org</a>. GeneNetwork.org provides two genotype files for the BXDs: a “classic” set (pre-2017) of genotypes that have been used in most mapping studies since 2005 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Shifman <em>et al.</em> 2006</a>), and new consensus genotypes (2017) that include updated data for BXD43 through BXD220 that were collected November 2015 and processed using the GigaMUGA array (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Morgan <em>et al.</em> 2016</a>). In the current study, we have compared MDA genotypes to the classic genotypes used through the end of 2016.</p></div><div id="sec-5" class="subsection"><h3>Strain contamination</h3><p id="p-17">An RIS or CSS is considered to be contaminated if it carries a segment of genome that did not originate from one of the two founder strains. We developed an HMM to search for contamination. In contrast to our previous HMM analysis, here we select SNPs that were not informative (both founders have the same signal). In a contaminated region, the signal of a given strain is expected to contain a higher proportion of SNPs that differ from both founder strains. To reduce the false positive rate, only intervals covering three or more noninformative SNPs were reported.</p></div><div id="sec-6" class="subsection"><h3>Copy number variants</h3><p id="p-18">To determine if any of the RIS or CSS strains carried copy number variations (CNVs) that differed from the copy number in the founder strains, we applied the <em>simpleCNV</em> function of the MouseDivGeno package (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Didion <em>et al.</em> 2012</a>). We accepted only those candidate CNV detections that had length &gt;20 kb and covered ≤10 invariant genomic probes with <em>t</em>-statistic &gt;5 (<em>P</em> &lt; 1E−6).</p></div><div id="sec-7" class="subsection"><h3>Gene conversions</h3><p id="p-19">Gene conversions are short tracts (&lt;1 kb) of nonreciprocal transfer of genetic information between two homologs that occur during meiosis. In the case of RIS, it is difficult to distinguish gene conversion events from short haplotype blocks that are due to closely spaced recombination events that occurred in different meiosis. Therefore, we restricted our attention to the CSS panels. We searched for single or small groups of adjacent SNPs that derive from the host genotype, but occur on the donor chromosomes. We examined individual SNP intensities to identify those that are clearly derived from the host strain and are present in a region of donor strain haplotype.</p></div><div id="sec-8" class="subsection"><h3>Sister strains</h3><p id="p-20">In a typical RIS panel, the lineages that give rise to each RIS are independent, and thus there should be no sharing of recombination events between strains. BXD strains from epoch III are an exception because they may share recombinations that arose in the outbreeding generations (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Peirce <em>et al.</em> 2004</a>). Therefore, we excluded these strains from this analysis. We detected excess sharing of recombination junctions (Z-score &gt;5.0), as an indicator that two strains are more similar than expected by chance.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-21">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article. The raw genotyping data and genetic maps of these strain panels are available at <a href="http://churchill-lab.jax.org/website/MDA">http://churchill-lab.jax.org/website/MDA</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Global genotyping error</h3><p id="p-22">Global genotyping error—defined as a percentage of informative SNPs discordant with the haplotype assignment—is typically &lt;1%, but is higher for haplotype blocks of <em>M. m. musculus</em> (PWD) and <em>M. m. molossinus</em> (MSM) origin than for <em>M. m. domesticus</em> blocks (B6, A, and D2) (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300188/-/DC1/FigureS1.pdf">Figure S1</a>; the legends to supplementary figures are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300188/-/DC1/FileS1.docx">File S1</a>). This is likely to be caused by polymorphisms in, or near, the oligonucleotide probe sequence or its flanking restriction sites (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Didion <em>et al.</em> 2012</a>). There are a few outlying strains with a higher error rate than other strains from the same panel [AXB1, BXD15, BXD25, BXD85, BXD65a (formerly known as BXD92), BXD93, B6.A#Chr7, and B6.A#Chr10] likely due to low DNA quality or to processing of arrays.</p></div><div id="sec-12" class="subsection"><h3>Residual heterozygosity</h3><p id="p-23">Residual heterozygosity is present in some strains from each panel, except for the AXB/BXA strains, which appear to be fully inbred (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The detected heterozygous regions are an underestimate of percentage of segregating variation that is present in each strain because only a single animal per strain was genotyped. The presence of heterozygous strains in large RIS panels is not surprising. We estimated that, in the absence of selection, an RIS strain needs, on average, 24 generations of sib-mating to reach a heterozygosity rate &lt;1%, and 36 generations to reach complete fixation. However, there is a significant variation in the number of generations required to achieve these landmarks (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Broman 2005</a>). For a panel of 22 strains (the size of a full CSS panel), 53 generations are required, on average, to achieve complete fixation for all its strains in the absence of selection.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486976/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486976/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Residual heterozygosity and CNV (deletion/extra copy) in the six panels</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>De novo deletions and duplications</h3><p id="p-24">We detected 64 <em>de novo</em> deletions and 14 <em>de novo</em> duplications, with lengths ranging from 21 kb to 8.4 Mb, affecting 111 Ensembl genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300188/-/DC1/TableS1.xlsx">Table S1</a>). <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> summarizes the frequency of strains with heterozygosity, deletions, and duplications. We observe that a longer time of inbreeding is associated with lower heterozygosity but more structural changes. This is seen most clearly by comparing different epochs of the BXD panel.</p><div id="sec-14" class="subsection"><h4>High-density genotyping identifies unexpected haplotype blocks in CSS panels:</h4><p id="p-25">We observe 27 haplotype blocks from the host strain in the proximal or distal regions of the donor chromosome across the three CSS panels (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). These events are undesirable but not unexpected due to the distribution of markers used for CSS development (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Nadeau <em>et al.</em> 2000</a>). We also observe strains in which a host haplotype block occurs in the middle of an introgressed donor chromosome or a donor haplotype block occurs in a host chromosome. We observed seven such events distributed across all three CSS panels (bold face highlights in <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486948/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486948/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Unexpected haplotype blocks in all CSS panels</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h4>High-density genotyping improves map accuracy in RIS panels:</h4><p id="p-26">To validate our haplotype assignment, and to estimate the level of improvement, we compared our maps to the versions available at <a href="http://www.genenetwork.org">www.genenetwork.org</a> (LXS and BXD) or provided by Institut de recherches cliniques de Montréal (AXB/BXA). There was a high concordance (99.8% LXS, 98.1% BXD, and 99.5% ABX/BXA) between new and old maps for intervals that were assigned to one of the founder in both maps. The new maps decreased the level of uncertainty, measured as the sum of length of recombination intervals by 66% in the AXB/BXA panel, 41% in the BXD panel, and 5% in the LXS panel. This improvement mirrors the increase in the number of informative markers: from 792 to 101,397 (AXB/BXA), from 3796 to 103,341 (BXD), and from 2649 to 79,808 (LXS), respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300188/-/DC1/FigureS2.pdf">Figure S2</a>).</p><div id="sec-16" class="subsection"><h5>Strain contamination in the AXB/BXA panel:</h5><p id="p-27">An unexpected observation in AXB/BXA RIS panel, was the presence of six intervals that are not derived from either A or B6 inbred strains. Three chromosomes (chr14, chr15, and chr16) of the AXB1 strain, two chromosomes (chr5 and chr16) of AXB2, and one chromosome (chr13) of BXA1 are affected by contamination. Based on comparison to genotypes from a large panel of inbred strains (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Yang <em>et al.</em> 2011</a>), we conclude that the contamination derived from a strain that is closely related to DBA/2J.</p></div><div id="sec-17" class="subsection"><h5>Recombination rate:</h5><p id="p-28">The distribution of the number of recombination events is similar across all panels (see <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300188/-/DC1/TableS2.xls">Table S2</a>) with the exception of the advanced RIS BXD (epoch III), which has more recombination events per chromosome due to additional generations of outbreeding. The number of recombination events per strain ranges from 32 (BXD32) to 84 (BXA17) among the classical RIS, and from 60 (BXD53) to 127 (BXD47) among the advanced BXD panel. These numbers of recombination events fall within the 95% prediction interval from simulations (using Python code from <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Welsh and McMillan 2012</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3427/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Number of founder haplotype blocks in RIS panels. The number of founder blocks for each strain is indicated as a point, with jitter for clarity. The boxplot indicates median and quartiles of each panel. Results for the BXD panel are broken down by three breeding epochs (I, II, and III); the increased number of recombination event in epoch&#xA0;III reflects additional generation of outbreeding used in the derivation of these strains." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-86914000" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Number of founder haplotype blocks in RIS panels. The number of founder blocks for each strain is indicated as a point, with jitter for clarity. The boxplot indicates median and quartiles of each panel. Results for the BXD panel are broken down by three breeding epochs (I, II, and III); the increased number of recombination event in epoch&#xA0;III reflects additional generation of outbreeding used in the derivation of these strains.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3427/F1.medium.gif" width="440" height="312"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/10/3427/F1.medium.gif" width="440" height="312"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3427/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3427/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486873" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-29" class="first-child">Number of founder haplotype blocks in RIS panels. The number of founder blocks for each strain is indicated as a point, with jitter for clarity. The boxplot indicates median and quartiles of each panel. Results for the BXD panel are broken down by three breeding epochs (I, II, and III); the increased number of recombination event in epoch III reflects additional generation of outbreeding used in the derivation of these strains.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Most recombination events in the RIS panels are unique, but some recombination intervals overlap and could result from independent recurrent events or from shared ancestry between RIS during the inbreeding process. The most frequently shared recombination event occurs in 8 out of 25 samples of the AXB RIS panel (Chr10: 66,730,215–67,348,211). Moreover, in seven out of eight cases (<em>P</em> = 0.07), the polarity of the event is in the same direction: from B6 segment (proximal: 66,730,214 bp) to A/J segment (distal: 67,348,212 bp). Additional shared recombination intervals are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300188/-/DC1/TableS3.xlsx">Table S3</a>, and the recombination frequency is visualized in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300188/-/DC1/FigureS3.pdf">Figure S3</a>. Higher recombination rates observed in the distal region of chromosomes are expected (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Liu <em>et al.</em> 2014</a>).</p></div><div id="sec-18" class="subsection"><h5>Sister strains:</h5><p id="p-31">Sister strains are strains related by descent from incompletely inbred ancestors during the breeding process. They can be identified because they share a large number of recombination intervals with the same proximal to distal polarity of founder haplotypes. Not surprisingly, most of the sister strains are detected for the advanced BXD panel (six pairs + six larger groups, totally comprising 40 strains). However, two pairs of strains are present in the AXB and LXS panels: AXB6–AXB12 and LXS94–LXS107. These strains share more recombination intervals with the same founder strain polarity than expected by chance (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3427/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sister strains in RIS panels. Side-by-side comparison of sister strains AXB6 vs. AXB12 (red, B6; blue, A) and LXS94 vs. LXS107 (red, L; blue, S) illustrates the extent of shared haplotype blocks." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-86914000" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sister strains in RIS panels. Side-by-side comparison of sister strains AXB6 &lt;em&gt;vs.&lt;/em&gt; AXB12 (red, B6; blue, A) and LXS94 &lt;em&gt;vs.&lt;/em&gt; LXS107 (red, L; blue, S) illustrates the extent of shared haplotype blocks.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3427/F2.medium.gif" width="440" height="213"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/10/3427/F2.medium.gif" width="440" height="213"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3427/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3427/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486947" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Sister strains in RIS panels. Side-by-side comparison of sister strains AXB6 <em>vs.</em> AXB12 (red, B6; blue, A) and LXS94 <em>vs.</em> LXS107 (red, L; blue, S) illustrates the extent of shared haplotype blocks.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h5>The MDA array detects short gene conversions in CSS panels:</h5><p id="p-33">We searched for putative gene conversions in the introgressed donor chromosomes of CSS panels. We identified small regions typically spanning just one informative SNP that have genotypes consistent with the host strain instead of the donor strain (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). In total, we identified 28 putative gene conversions: 17 in the B6.A CSS panel, seven in the B6.PWD CSS panel, and four in the B6.MSM CSS panel (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3427/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gene conversion in a CSS strain. Strain B6.PWD13 has an unexpected founder genotype at marker JAX00357227 marker (Chr 13: 47,505,217 bp). Average and contrast signal intensities are plotted for all B6.PWD strains. Numbers indicate the CSS strains by substituted chromosome with B6. PWD13 is highlighted by the red circle. Also indicated on the plot are founder strains B6, and PWD and their F1 hybrids. The B.PWD13 data should be similar to PWD but is actually close to B6, indicating a putative gene conversion. Gray letters indicate genotype calls for 1902 additional samples in the MDA database (A, B6 allele homozygous; B, PWD allele homozygous; H, heterozygous; V, vino; N, no call)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-86914000" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Gene conversion in a CSS strain. Strain B6.PWD13 has an unexpected founder genotype at marker JAX00357227 marker (Chr 13: 47,505,217 bp). Average and contrast signal intensities are plotted for all B6.PWD strains. Numbers indicate the CSS strains by substituted chromosome with B6. PWD13 is highlighted by the red circle. Also indicated on the plot are founder strains B6, and PWD and their F1 hybrids. The B.PWD13 data should be similar to PWD but is actually close to B6, indicating a putative gene conversion. Gray letters indicate genotype calls for 1902 additional samples in the MDA database (A, B6 allele homozygous; B, PWD allele homozygous; H, heterozygous; V, vino; N, no call).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3427/F3.medium.gif" width="440" height="419"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/10/3427/F3.medium.gif" width="440" height="419"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3427/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3427/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486884" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-34" class="first-child">Gene conversion in a CSS strain. Strain B6.PWD13 has an unexpected founder genotype at marker JAX00357227 marker (Chr 13: 47,505,217 bp). Average and contrast signal intensities are plotted for all B6.PWD strains. Numbers indicate the CSS strains by substituted chromosome with B6. PWD13 is highlighted by the red circle. Also indicated on the plot are founder strains B6, and PWD and their F1 hybrids. The B.PWD13 data should be similar to PWD but is actually close to B6, indicating a putative gene conversion. Gray letters indicate genotype calls for 1902 additional samples in the MDA database (A, B6 allele homozygous; B, PWD allele homozygous; H, heterozygous; V, vino; N, no call).</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486802/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486802/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Short gene conversions in CSS panels</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h5>Online access to genetics maps and MDA genotypes:</h5><p id="p-35">For easy access, we provide a compilation of Mouse Diversity Array data, annotation and supporting software at <a href="http://churchill-lab.jax.org/website/MDA">http://churchill-lab.jax.org/website/MDA</a>. Resources to support our analysis of RIS and CSS strains include an online viewer, where maps can be viewed and downloaded either as a list of intervals or as CSV files ready to be imported to the R/qtl package (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Broman and Sen 2009</a>). Source code for the viewer is also available on Github, <a href="https://github.com/simecek/RIS-map-viewer">https://github.com/simecek/RIS-map-viewer</a>. Researchers interested in comparing those reference populations to genotypes of other mouse strains processed on MDA arrays can use the MDA viewer. The entire database, consisting of 1902 MDA arrays, is available for download as SQLite database or as individual CEL files <a href="ftp://ftp.jax.org/petrs/MDA/">ftp://ftp.jax.org/petrs/MDA/</a>.</p></div></div></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-36">We have characterized 180 RIS and 79 CSS strains from six popular and valuable resources, and provided online access to these data. These panels were developed at different times and genotyped with lower density sets of markers. High-density genotyping with the number of informative SNPs, ranging between 79,000 and 257,000, provide maps with higher resolution. In this study, we achieved a median spacing between informative markers 5.7 kb (AXB), 5.4 kb (BXD), 5.6 kb (LXS), 4.6 kb (B6.PWD), and 5.2 kb (B6.MSM), respectively. This enabled us to identify unusual features such as regions of residual heterozygosity, contamination by a nonfounder strain and <em>de novo</em> structural variants. These genotyping arrays are part of 1902 samples processed on MDA platform that can be accessed from <a href="http://churchill-lab.jax.org/website/MDA">http://churchill-lab.jax.org/website/MDA</a>.</p><p id="p-37">Genetic reference panels are valuable, in part, because of the ability of generate animals with identical genomes in the number and timespan dictated by the researcher. Replication increases the accuracy of phenotype measurements (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Belknap 1998</a>), and allows for integration of data over space, time, and environment. While it is convenient to think of all mice from an inbred strain as identical, we provide evidence that this view is not always warranted. Residual heterozygosity may be due to stochasticity in the inbreeding process, or it may reflect biological constraints that prevent full inbreeding of a strain. Genetic drift operates in each of these populations, and low-density genotyping in selected regions of the genome, leaves room for undesired or unexpected surprises. In a typical CSS strain the average proportion of the donor genome present in other chromosomes is expected to be 0.2% (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Nadeau <em>et al.</em> 2000</a>). Over our three CSS panels, the average length of unexpected genotype was 1.5 Mb. The length of intervals ranges (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>) from &lt;1 Mb (one gene) to 20 Mb (138 genes).</p><p id="p-38">For gene conversions, whole genome sequencing of CSS panels (and RIS) will likely reveal more examples and provide better estimates of converted regions and their length. However, our results suggest that gene conversions are more probable in regions where founders’ genomes are very similar. We observe significantly more conversions on the B6.A panel than in the other two CSS panels (17 <em>vs.</em> 7 and 4, Fisher exact test, <em>P</em> = 0.046), despite the fact that the number of informative markers is lower and therefore our ability to detect gene conversions reduced. Based on this result, we hypothesize that gene conversions occur preferably in regions of low sequence diversity between homologous chromosomes. If that is true, then they will have fewer genetic consequences due to lower chance to cause distinguishing polymorphism. Roughly, we estimate that 0.005% of the genome is affected by gene conversion (avg. # gene conversions/# informative SNPs = 28/3/200,000). The real number of gene conversions is likely to be higher because we were only able to identify gene conversions that overlap informative SNP probes in the array.</p><p id="p-39">We found no evidence of allele frequency imbalance (one allele present with significantly higher frequency than the other allele in a RIS panel) that has been observed in other species (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Taudt <em>et al.</em> 2016</a>). Nor did we detect any epistatic selection between founder strains or alleles with different subspecies origin (permutation test, data not shown). This is in sharp contrast with mouse multiparent populations such as the Collaborative Cross and Diversity Outbred (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chesler <em>et al.</em> 2016</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Shorter <em>et al.</em> 2017</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Srivastava <em>et al.</em> 2017</a>) in which both distorted allele frequencies and epistatic selection are common. Due to limited number of strains in mouse RI panels, we may have missed small distortions.</p><p id="p-40">We observed an inverse relationship between residual heterozygosity and drift (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). For a given panel, even 20 generations of inbreeding is not enough to fix all heterozygous regions. On the other hand, populations kept for many generations will accumulate SNPs, small indels, and structural variants in their genomes (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Simecek <em>et al.</em> 2015</a>; <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Srivastava <em>et al.</em> 2017</a>). Strategies to reduce drift in breeding colonies have been developed, including the embryo cryopreservation program at The Jackson Laboratory (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Taft <em>et al.</em> 2006</a>). However, genetic drift can be also harnessed by geneticists to simplify and accelerate the identification of causal variants responsible for phenotypic differences between substrains (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Srivastava <em>et al.</em> 2017</a>). These so-called reduced complexity crosses are excellent examples of the potential benefits of genetic drift (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kumar <em>et al.</em> 2013</a>).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">The work was supported by in part by National Institutes of Health (NIH) grants P50GM076468 (G.A.C.), U19AI100625 (F.P.-M.d.V.), and U01 AA016662 (R.W.W.), a grant-in-aid for scientific research from Japan Society for the Promotion Science grant No. 17018033 (T.S. and T.T.), Canadian Institutes of Health Research MOP-93583 (C.F.D. and M.-P.S.-B.), and by the Czech Science Foundation grant 16-01969S, LM2015040, and LQ1604 projects from the Ministry of Education, Youth, and Sports of the Czech Republic (J.F. and P.S.). Genotyping of the BXD RIS was supported by the University of Tennessee Center for Integrative and Translational Science.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-42">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300188/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300188/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-43"><em>Communicating editor: D. Threadgill</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-06-25"><span class="received-label">Received </span>June 25, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-08-21"><span class="accepted-label">Accepted </span>August 21, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Simecek <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.1" data-doi="10.1038/71641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beck</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hafezparast</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lennon-Pierce</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eppig</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genealogies of mouse inbred strains.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBeck%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D25%26rft.atitle%253DGenealogies%2Bof%2Bmouse%2Binbred%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F71641%26rft_id%253Dinfo%253Apmid%252F10615122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/71641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10615122&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084609200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.2" data-doi="10.1023/A:1021404714631"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belknap</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Effect of within-strain sample size on QTL detection and mapping using recombinant inbred mouse strains.</span> <abbr class="cit-jnl-abbrev">Behav. Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehavior%2Bgenetics%26rft.stitle%253DBehav%2BGenet%26rft.aulast%253DBelknap%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D29%26rft.epage%253D38%26rft.atitle%253DEffect%2Bof%2Bwithin-strain%2Bsample%2Bsize%2Bon%2BQTL%2Bdetection%2Band%2Bmapping%2Busing%2Brecombinant%2Binbred%2Bmouse%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1021404714631%26rft_id%253Dinfo%253Apmid%252F9573644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1021404714631&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9573644&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072994900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.3" data-doi="10.1534/genetics.104.035212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The genomes of recombinant inbred lines.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1133</span>–<span class="cit-lpage">1146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.035212%26rft_id%253Dinfo%253Apmid%252F15545647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS8yLzExMzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEwLzM0MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-7.10.3427.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source"><em>A Guide to QTL Mapping with R/qtl</em></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.5" data-doi="10.1101/gr.187450.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buchner</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nadeau</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Contrasting genetic architectures in different mouse reference populations used for studying complex traits.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">775</span>–<span class="cit-lpage">791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.187450.114%26rft_id%253Dinfo%253Apmid%252F25953951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzYvNzc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strobel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trepanier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Diversity outbred mice at 21: maintaining allelic variation in the face of selection.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3893</span>–<span class="cit-lpage">3902</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D6%26rft.spage%253D3893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.7" data-doi="10.1038/nature01262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chinwalla</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delehaunty</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fewell</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Initial sequencing and comparative analysis of the mouse genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">420</span>: <span class="cit-fpage">520</span>–<span class="cit-lpage">562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWaterston%26rft.auinit1%253DR.%2BH.%26rft.volume%253D420%26rft.issue%253D6915%26rft.spage%253D520%26rft.epage%253D562%26rft.atitle%253DInitial%2Bsequencing%2Band%2Bcomparative%2Banalysis%2Bof%2Bthe%2Bmouse%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01262%26rft_id%253Dinfo%253Apmid%252F12466850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12466850&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179611600053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.8" data-doi="10.1007/s00335-012-9414-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munger</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The diversity outbred mouse population.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D713%26rft.epage%253D718%26rft.atitle%253DThe%2Bdiversity%2Boutbred%2Bmouse%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9414-2%26rft_id%253Dinfo%253Apmid%252F22892839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9414-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22892839&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.9" data-doi="10.1534/genetics.111.132639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Collaborative Cross Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome architecture of the Collaborative Cross mouse genetic reference population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132639%26rft_id%253Dinfo%253Apmid%252F22345608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.10" data-doi="10.1186/1471-2164-13-34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheppard</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Discovery of novel variants in genotyping arrays improves genotype retention and reduces ascertainment bias.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDidion%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D13%26rft.spage%253D34%26rft.epage%253D34%26rft.atitle%253DDiscovery%2Bof%2Bnovel%2Bvariants%2Bin%2Bgenotyping%2Barrays%2Bimproves%2Bgenotype%2Bretention%2Band%2Breduces%2Bascertainment%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-34%26rft_id%253Dinfo%253Apmid%252F22260749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-34&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22260749&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.11" data-doi="10.1101/gr.7160508"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gregorova</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Divina</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storchova</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trachtulec</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fotopulosova</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mouse consomic strains: exploiting genetic divergence between <em>Mus m. musculus</em> and <em>Mus m. domesticus</em> subspecies.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">509</span>–<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.7160508%26rft_id%253Dinfo%253Apmid%252F18256238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzMvNTA5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-web" id="cit-7.10.3427.12"><div class="cit-metadata unstructured">Himmelmann, L., 2010 HMM: HMM - Hidden Markov Models. R package version 1.0. Available at: <a href="https://CRAN.R-project.org/package=HMM">https://CRAN.R-project.org/package=HMM</a>. Accessed: April 8, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Juang</span>  <span class="cit-name-given-names">B.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabiner</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">The segmental K-means algorithm for estimating parameters of hidden Markov models.</span> <abbr class="cit-jnl-abbrev">IEEE Trans. Acoust. Speech Signal Process.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1639</span>–<span class="cit-lpage">1641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BAcoust.%2BSpeech%2BSignal%2BProcess.%26rft.volume%253D38%26rft.spage%253D1639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.14" data-doi="10.1038/nature10413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mouse genomic variation and its effect on phenotypes and gene regulation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKeane%26rft.auinit1%253DT.%2BM.%26rft.volume%253D477%26rft.issue%253D7364%26rft.spage%253D289%26rft.epage%253D294%26rft.atitle%253DMouse%2Bgenomic%2Bvariation%2Band%2Bits%2Beffect%2Bon%2Bphenotypes%2Band%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10413%26rft_id%253Dinfo%253Apmid%252F21921910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921910&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294852400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.15" data-doi="10.1126/science.1245503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kourrich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoo</span>  <span class="cit-name-given-names">S.-H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">C57BL/6N mutation in cytoplasmic FMRP interacting protein 2 regulates cocaine response.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">342</span>: <span class="cit-fpage">1508</span>–<span class="cit-lpage">1512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKumar%26rft.auinit1%253DV.%26rft.volume%253D342%26rft.issue%253D6165%26rft.spage%253D1508%26rft.epage%253D1512%26rft.atitle%253DC57BL%252F6N%2BMutation%2Bin%2BCytoplasmic%2BFMRP%2Binteracting%2Bprotein%2B2%2BRegulates%2BCocaine%2BResponse%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1245503%26rft_id%253Dinfo%253Apmid%252F24357318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDIvNjE2NS8xNTA4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Extensive recombination due to heteroduplexes generates large amounts of artificial gene fragments during PCR.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e106658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De106658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doran</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Whole genome sequence of two wild-derived Mus musculus domesticus inbred strains, LEWES/EiJ and ZALENDE/EiJ, with different diploid numbers.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">4211</span>–<span class="cit-lpage">4216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D4211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.18" data-doi="10.1038/73427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nadeau</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Analysing complex genetic traits with chromosome substitution strains.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DNadeau%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D24%26rft.issue%253D3%26rft.spage%253D221%26rft.epage%253D225%26rft.atitle%253DAnalysing%2Bcomplex%2Bgenetic%2Btraits%2Bwith%2Bchromosome%2Bsubstitution%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F73427%26rft_id%253Dinfo%253Apmid%252F10700173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/73427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10700173&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085590600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.19" data-doi="10.1007/s00335-012-9426-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nadeau</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forejt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takada</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiroishi</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chromosome substitution strains: gene discovery, functional analysis, and systems studies.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">705</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DNadeau%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D693%26rft.epage%253D705%26rft.atitle%253DChromosome%2Bsubstitution%2Bstrains%253A%2Bgene%2Bdiscovery%252C%2Bfunctional%2Banalysis%252C%2Band%2Bsystems%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9426-y%26rft_id%253Dinfo%253Apmid%252F22961226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9426-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22961226&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nesbitt</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skamene</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Recombinant inbred mouse strains derived from A/J and C57BL/6J: a tool for the study of genetic mechanisms in host resistance to infection and malignancy.</span> <abbr class="cit-jnl-abbrev">J. Leukoc. Biol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BLeukocyte%2BBiology%26rft.stitle%253DJ.%2BLeukoc.%2BBiol.%26rft.aulast%253DNesbitt%26rft.auinit1%253DM.%2BN.%26rft.volume%253D36%26rft.issue%253D3%26rft.spage%253D357%26rft.epage%253D364%26rft.atitle%253DRecombinant%2Binbred%2Bmouse%2Bstrains%2Bderived%2Bfrom%2BA%252FJ%2Band%2BC57BL%252F6J%253A%2Ba%2Btool%2Bfor%2Bthe%2Bstudy%2Bof%2Bgenetic%2Bmechanisms%2Bin%2Bhost%2Bresistance%2Bto%2Binfection%2Band%2Bmalignancy%26rft_id%253Dinfo%253Apmid%252F6592283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiamxldWIiO3M6NToicmVzaWQiO3M6ODoiMzYvMy8zNTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEwLzM0MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.21" data-doi="10.1186/1471-2156-5-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peirce</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silver</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A new set of BXD recombinant inbred lines from advanced intercross populations in mice.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DPeirce%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D5%26rft.spage%253D7%26rft.epage%253D7%26rft.atitle%253DA%2Bnew%2Bset%2Bof%2BBXD%2Brecombinant%2Binbred%2Blines%2Bfrom%2Badvanced%2Bintercross%2Bpopulations%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-5-7%26rft_id%253Dinfo%253Apmid%252F15117419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-5-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15117419&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.22" data-doi="10.1371/journal.pbio.0040395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shifman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Copley</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A high-resolution single nucleotide polymorphism genetic map of the mouse genome.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DShifman%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De395%26rft.epage%253De395%26rft.atitle%253DA%2Bhigh-resolution%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bgenetic%2Bmap%2Bof%2Bthe%2Bmouse%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040395%26rft_id%253Dinfo%253Apmid%252F17105354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17105354&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.23" data-doi="10.1534/genetics.116.199596"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shorter</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Odet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C.-Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Male infertility is responsible for nearly half of the extinction observed in the mouse Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.199596%26rft_id%253Dinfo%253Apmid%252F28592496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNTU3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rowe</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herberg</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic analysis of substrain divergence in non-obese diabetic (NOD) mice.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">771</span>–<span class="cit-lpage">775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D5%26rft.spage%253D771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.25" data-doi="10.1534/genetics.116.198838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Najarian</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarsani</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sigmon</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomes of the mouse Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.198838%26rft_id%253Dinfo%253Apmid%252F28592495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNTM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.26" data-doi="10.1016/j.tig.2006.09.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taft</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davisson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiles</span>  <span class="cit-name-given-names">M. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Know thy mouse.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">649</span>–<span class="cit-lpage">653</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DTaft%26rft.auinit1%253DR.%2BA.%26rft.volume%253D22%26rft.issue%253D12%26rft.spage%253D649%26rft.epage%253D653%26rft.atitle%253DKnow%2Bthy%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2006.09.010%26rft_id%253Dinfo%253Apmid%252F17007958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2006.09.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17007958&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242639900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.27" data-doi="10.1101/gr.7175308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takada</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mita</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maeno</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shitara</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mouse inter-subspecific consomic strains for genetic dissection of quantitative complex traits.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">500</span>–<span class="cit-lpage">508</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.7175308%26rft_id%253Dinfo%253Apmid%252F18256237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzMvNTAwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taudt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colomé-Tatché</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johannes</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic sources of population epigenomic variation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D17%26rft.spage%253D319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.29" data-doi="10.1038/256644a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherry</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riblet</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigert</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Genes for immunoglobulin heavy chain and serum prealbumin protein are linked in mouse.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">256</span>: <span class="cit-fpage">644</span>–<span class="cit-lpage">646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DTaylor%26rft.auinit1%253DB.%2BA.%26rft.volume%253D256%26rft.issue%253D5519%26rft.spage%253D644%26rft.epage%253D646%26rft.atitle%253DGenes%2Bfor%2Bimmunoglobulin%2Bheavy%2Bchain%2Band%2Bserum%2Bprealbumin%2Bprotein%2Bare%2Blinked%2Bin%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F256644a0%26rft_id%253Dinfo%253Apmid%252F807855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/256644a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=807855&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.30" data-doi="10.1007/s003359900998"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wnek</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kotlus</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roemer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacTaggart</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genotyping new BXD recombinant inbred mouse strains and comparison of BXD and consensus maps.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">335</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DTaylor%26rft.auinit1%253DB.%2BA.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D335%26rft.epage%253D348%26rft.atitle%253DGenotyping%2Bnew%2BBXD%2Brecombinant%2Binbred%2Bmouse%2Bstrains%2Band%2Bcomparison%2Bof%2BBXD%2Band%2Bconsensus%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs003359900998%26rft_id%253Dinfo%253Apmid%252F10087289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s003359900998&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10087289&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079321300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accelerating the inbreeding of multi-parental recombinant inbred lines generated by sibling matings.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">191</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.32" data-doi="10.1186/gb-2001-2-11-research0046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The genetic structure of recombinant inbred mice: high-resolution consensus maps for complex trait analysis.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">RESEARCH0046</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DWilliams%26rft.auinit1%253DR.%2BW.%26rft.volume%253D2%26rft.issue%253D11%26rft.spage%253DRESEARCH0046%26rft.epage%253DRESEARCH0046%26rft.atitle%253DThe%2Bgenetic%2Bstructure%2Bof%2Brecombinant%2Binbred%2Bmice%253A%2Bhigh-resolution%2Bconsensus%2Bmaps%2Bfor%2Bcomplex%2Btrait%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2001-2-11-research0046%26rft_id%253Dinfo%253Apmid%252F11737945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2001-2-11-research0046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11737945&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.33" data-doi="10.1007/s00335-004-2380-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeFries</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic structure of the LXS panel of recombinant inbred mouse strains: a powerful resource for complex trait analysis.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">637</span>–<span class="cit-lpage">647</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DWilliams%26rft.auinit1%253DR.%2BW.%26rft.volume%253D15%26rft.issue%253D8%26rft.spage%253D637%26rft.epage%253D647%26rft.atitle%253DGenetic%2Bstructure%2Bof%2Bthe%2BLXS%2Bpanel%2Bof%2Brecombinant%2Binbred%2Bmouse%2Bstrains%253A%2Ba%2Bpowerful%2Bresource%2Bfor%2Bcomplex%2Btrait%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-004-2380-6%26rft_id%253Dinfo%253Apmid%252F15457343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-004-2380-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15457343&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223063600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.34" data-doi="10.1038/nmeth.1359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutchins</span>  <span class="cit-name-given-names">L. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szatkiewicz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A customized and versatile high-density genotyping array for the mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">663</span>–<span class="cit-lpage">666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253D663%26rft.epage%253D666%26rft.atitle%253DA%2Bcustomized%2Band%2Bversatile%2Bhigh-density%2Bgenotyping%2Barray%2Bfor%2Bthe%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1359%26rft_id%253Dinfo%253Apmid%252F19668205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19668205&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269382900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3427.35" data-doi="10.1038/ng.847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Subspecific origin and haplotype diversity in the laboratory mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">648</span>–<span class="cit-lpage">655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D43%26rft.issue%253D7%26rft.spage%253D648%26rft.epage%253D655%26rft.atitle%253DSubspecific%2Borigin%2Band%2Bhaplotype%2Bdiversity%2Bin%2Bthe%2Blaboratory%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.847%26rft_id%253Dinfo%253Apmid%252F21623374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21623374&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/10/3427.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/10/3415" title="Whole Genome Sequencing-Based Mapping and Candidate Identification of Mutations from Fixed Zebrafish Tissue" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/10/3435" title="A Comparative Analysis of Genetic Ancestry and Admixture in the Colombian Populations of Chocó and Medellín" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/10"><span>Volume 7 Issue 10, October 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/10.cover-source.jpg?itok=lurMO2QE" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3427" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DPAtzZiY3k2A4Z6lf2NR4ogEM81nNBoYNuOWPE5Pmjs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/10/3427" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/10/3427" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/10/3427" class="active" data-icon-position="" data-hide-link-title="0">High-Resolution Maps of Mouse Reference Populations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/486770" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-yxEFns0n9cO5yB-A8wBVkWys1eWUobWx7j_lc5bI4-M" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/10/3427.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/10/3427" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-49VPs7lLF6IQ5xOgkKdVgnArQfuK_zdpUZRPC3El7Cs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/10/3427" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/10/3427.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486770" id="node486770--2" data-pisa="ggg;7/10/3427" data-pisa-master="ggg;g3.117.300188" data-apath="/ggg/7/10/3427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Resolution Maps of Mouse Reference Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2922-7183" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Petr Simecek</span>, <span class="highwire-citation-author" data-delta="1">Jiri Forejt</span>, <span class="highwire-citation-author" data-delta="2">Robert W. Williams</span>, <span class="highwire-citation-author" data-delta="3">Toshihiko Shiroishi</span>, <span class="highwire-citation-author" data-delta="4">Toyoyuki Takada</span>, <span class="highwire-citation-author" data-delta="5">Lu Lu</span>, <span class="highwire-citation-author" data-delta="6">Thomas E. Johnson</span>, <span class="highwire-citation-author" data-delta="7">Beth Bennett</span>, <span class="highwire-citation-author" data-delta="8">Christian F. Deschepper</span>, <span class="highwire-citation-author" data-delta="9">Marie-Pier Scott-Boyer</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fernando Pardo-Manuel de Villena</span> and <span class="highwire-citation-author" data-delta="11">Gary A. Churchill</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3427-3434;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300188 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486770--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Petr Simecek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division Biotechnology and Biomedicine Centre of the Academy of Sciences, Institute of Molecular Genetics of the Academy of Sciences of the Czech Republic, Vestec, Prague 4, 142 20, Czech Republic</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Petr%2BSimecek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simecek%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APetr%2BSimecek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2922-7183" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Petr Simecek</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jiri Forejt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Division Biotechnology and Biomedicine Centre of the Academy of Sciences, Institute of Molecular Genetics of the Academy of Sciences of the Czech Republic, Vestec, Prague 4, 142 20, Czech Republic</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiri%2BForejt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Forejt%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiri%2BForejt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Robert W. Williams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee 38163</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BW.%2BWilliams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Williams%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BW.%2BWilliams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Toshihiko Shiroishi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>National Institute of Genetics, Mishima, 411-8540, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Toshihiko%2BShiroishi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shiroishi%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AToshihiko%2BShiroishi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Toyoyuki Takada </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>National Institute of Genetics, Mishima, 411-8540, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Toyoyuki%2BTakada%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takada%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AToyoyuki%2BTakada%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Lu Lu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee 38163</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lu%2BLu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALu%2BLu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Thomas E. Johnson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>University of Colorado at Boulder, Colorado 80309</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BE.%2BJohnson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnson%20TE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BE.%2BJohnson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Beth Bennett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>University of Colorado at Boulder, Colorado 80309</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beth%2BBennett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bennett%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeth%2BBennett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Christian F. Deschepper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Institut de Recherches Cliniques, Montreal, Quebec, H2W 1R7, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BF.%2BDeschepper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deschepper%20CF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristian%2BF.%2BDeschepper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Marie-Pier Scott-Boyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Institut de Recherches Cliniques, Montreal, Quebec, H2W 1R7, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-Pier%2BScott-Boyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scott-Boyer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-Pier%2BScott-Boyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institute for Behavioral Genetics, Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo-Manuel%20de%20Villena%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fernando Pardo-Manuel de Villena</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fernando@med.unc.edu" class="" data-icon-position="" data-hide-link-title="0">fernando@med.unc.edu</a>
<a href="mailto:gary.churchill@jax.org" class="" data-icon-position="" data-hide-link-title="0">gary.churchill@jax.org</a></span></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fernando@med.unc.edu" class="" data-icon-position="" data-hide-link-title="0">fernando@med.unc.edu</a>
<a href="mailto:gary.churchill@jax.org" class="" data-icon-position="" data-hide-link-title="0">gary.churchill@jax.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/10/3427" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/10/3427&amp;title=High-Resolution%20Maps%20of%20Mouse%20Reference%20Populations" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/10/3427&amp;title=High-Resolution%20Maps%20of%20Mouse%20Reference%20Populations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/10/3427&amp;title=High-Resolution%20Maps%20of%20Mouse%20Reference%20Populations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/10/3427&amp;text=High-Resolution%20Maps%20of%20Mouse%20Reference%20Populations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/10/3427&amp;title=High-Resolution%20Maps%20of%20Mouse%20Reference%20Populations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/10/3427&amp;t=High-Resolution%20Maps%20of%20Mouse%20Reference%20Populations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/10/3427&amp;title=High-Resolution%20Maps%20of%20Mouse%20Reference%20Populations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/10/3427&amp;title=High-Resolution%20Maps%20of%20Mouse%20Reference%20Populations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/486770/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486770" id="node486770--3" data-pisa="ggg;7/10/3427" data-pisa-master="ggg;g3.117.300188" data-apath="/ggg/7/10/3427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Resolution Maps of Mouse Reference Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2922-7183" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Petr Simecek</span>, <span class="highwire-citation-author" data-delta="1">Jiri Forejt</span>, <span class="highwire-citation-author" data-delta="2">Robert W. Williams</span>, <span class="highwire-citation-author" data-delta="3">Toshihiko Shiroishi</span>, <span class="highwire-citation-author" data-delta="4">Toyoyuki Takada</span>, <span class="highwire-citation-author" data-delta="5">Lu Lu</span>, <span class="highwire-citation-author" data-delta="6">Thomas E. Johnson</span>, <span class="highwire-citation-author" data-delta="7">Beth Bennett</span>, <span class="highwire-citation-author" data-delta="8">Christian F. Deschepper</span>, <span class="highwire-citation-author" data-delta="9">Marie-Pier Scott-Boyer</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fernando Pardo-Manuel de Villena</span> and <span class="highwire-citation-author" data-delta="11">Gary A. Churchill</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3427-3434;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300188 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486770--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Petr Simecek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division Biotechnology and Biomedicine Centre of the Academy of Sciences, Institute of Molecular Genetics of the Academy of Sciences of the Czech Republic, Vestec, Prague 4, 142 20, Czech Republic</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Petr%2BSimecek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simecek%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APetr%2BSimecek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2922-7183" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Petr Simecek</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jiri Forejt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Division Biotechnology and Biomedicine Centre of the Academy of Sciences, Institute of Molecular Genetics of the Academy of Sciences of the Czech Republic, Vestec, Prague 4, 142 20, Czech Republic</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiri%2BForejt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Forejt%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiri%2BForejt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Robert W. Williams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee 38163</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BW.%2BWilliams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Williams%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BW.%2BWilliams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Toshihiko Shiroishi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>National Institute of Genetics, Mishima, 411-8540, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Toshihiko%2BShiroishi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shiroishi%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AToshihiko%2BShiroishi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Toyoyuki Takada </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>National Institute of Genetics, Mishima, 411-8540, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Toyoyuki%2BTakada%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takada%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AToyoyuki%2BTakada%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Lu Lu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee 38163</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lu%2BLu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALu%2BLu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Thomas E. Johnson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>University of Colorado at Boulder, Colorado 80309</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BE.%2BJohnson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnson%20TE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BE.%2BJohnson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Beth Bennett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>University of Colorado at Boulder, Colorado 80309</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beth%2BBennett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bennett%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeth%2BBennett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Christian F. Deschepper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Institut de Recherches Cliniques, Montreal, Quebec, H2W 1R7, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BF.%2BDeschepper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deschepper%20CF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristian%2BF.%2BDeschepper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Marie-Pier Scott-Boyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Institut de Recherches Cliniques, Montreal, Quebec, H2W 1R7, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-Pier%2BScott-Boyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scott-Boyer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-Pier%2BScott-Boyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institute for Behavioral Genetics, Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo-Manuel%20de%20Villena%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fernando Pardo-Manuel de Villena</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fernando@med.unc.edu" class="" data-icon-position="" data-hide-link-title="0">fernando@med.unc.edu</a>
<a href="mailto:gary.churchill@jax.org" class="" data-icon-position="" data-hide-link-title="0">gary.churchill@jax.org</a></span></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fernando@med.unc.edu" class="" data-icon-position="" data-hide-link-title="0">fernando@med.unc.edu</a>
<a href="mailto:gary.churchill@jax.org" class="" data-icon-position="" data-hide-link-title="0">gary.churchill@jax.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3427&amp;rft.epage=3434&amp;rft.atitle=High-Resolution%20Maps%20of%20Mouse%20Reference%20Populations&amp;rft.volume=7&amp;rft.issue=10&amp;rft.date=2017-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Simecek%2C+Petr&amp;rft.au=Forejt%2C+Jiri&amp;rft.au=Williams%2C+Robert+W.&amp;rft.au=Shiroishi%2C+Toshihiko&amp;rft.au=Takada%2C+Toyoyuki&amp;rft.au=Lu%2C+Lu&amp;rft.au=Johnson%2C+Thomas+E.&amp;rft.au=Bennett%2C+Beth&amp;rft.au=Deschepper%2C+Christian+F.&amp;rft.au=Scott-Boyer%2C+Marie-Pier&amp;rft.au=Pardo-Manuel+de+Villena%2C+Fernando&amp;rft.au=Churchill%2C+Gary+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/486770/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/486770/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/486770/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/486770/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/486770/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/486770/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/486770/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/486770/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/486770/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/486770/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/486770/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/486770/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/10/3427" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/10/3427.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/10/3427.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/10/3427.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/10/3427.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
