<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/10/3509" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/10/3509.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/10/3509.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/10/3509.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/10/3509.atom" />
<meta name="HW.pisa" content="ggg;7/10/3509" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Bimolecular Fluorescence Complementation Tool for Identification of Protein-Protein Interactions in Candida albicans" />
<meta name="DC.Identifier" content="10.1534/g3.117.300149" />
<meta name="DC.Date" content="2017-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Subotic et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Investigation of protein-protein interactions (PPI) in Candida albicans is essential for understanding the regulation of the signal transduction network that triggers its pathogenic lifestyle. Unique features of C. albicans , such as its alternative codon usage and incomplete meiosis, have enforced the optimization of standard genetic methods as well as development of novel approaches. Since the existing methods for detection of PPI are limited for direct visualization of the interacting complex in vivo , we have established a bimolecular fluorescence complementation (BiFC) assay in C. albicans , a powerful technique for studying PPI. We have developed an optimized set of plasmids that allows for N- and C-terminal tagging of proteins with split yeast-enhanced monomeric Venus fragments, so that all eight combinations of fusion orientations can be analyzed. With the use of our BiFC assay we demonstrate three interaction complexes in vivo , which were also confirmed by two-hybrid analysis. Our Candida -optimized BiFC assay represents a useful molecular tool for PPI studies and shows great promise in expanding our knowledge of molecular mechanisms of protein functions." />
<meta name="DC.Contributor" content="Ana Subotić" />
<meta name="DC.Contributor" content="Erwin Swinnen" />
<meta name="DC.Contributor" content="Liesbeth Demuyser" />
<meta name="DC.Contributor" content="Herlinde De Keersmaecker" />
<meta name="DC.Contributor" content="Hideaki Mizuno" />
<meta name="DC.Contributor" content="Hélène Tournu" />
<meta name="DC.Contributor" content="Patrick Van Dijck" />
<meta name="article:published_time" content="2017-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Bimolecular Fluorescence Complementation Tool for Identification of Protein-Protein Interactions in Candida albicans" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Investigation of protein-protein interactions (PPI) in &lt;i&gt;Candida albicans&lt;/i&gt; is essential for understanding the regulation of the signal transduction network that triggers its pathogenic lifestyle. Unique features of &lt;i&gt;C. albicans&lt;/i&gt;, such as its alternative codon usage and incomplete meiosis, have enforced the optimization of standard genetic methods as well as development of novel approaches. Since the existing methods for detection of PPI are limited for direct visualization of the interacting complex &lt;i&gt;in vivo&lt;/i&gt;, we have established a bimolecular fluorescence complementation (BiFC) assay in &lt;i&gt;C. albicans&lt;/i&gt;, a powerful technique for studying PPI. We have developed an optimized set of plasmids that allows for N- and C-terminal tagging of proteins with split yeast-enhanced monomeric Venus fragments, so that all eight combinations of fusion orientations can be analyzed. With the use of our BiFC assay we demonstrate three interaction complexes &lt;i&gt;in vivo&lt;/i&gt;, which were also confirmed by two-hybrid analysis. Our &lt;i&gt;Candida&lt;/i&gt;-optimized BiFC assay represents a useful molecular tool for PPI studies and shows great promise in expanding our knowledge of molecular mechanisms of protein functions.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/10/01" />
<meta name="citation_mjid" content="ggg;7/10/3509" />
<meta name="citation_id" content="7/10/3509" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/10/3509" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/10/3509.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/10/3509.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/10/3509.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300149" />
<meta name="citation_pmid" content="28860184" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3509" />
<meta name="citation_lastpage" content="3520" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ana Subotić" />
<meta name="citation_author_institution" content="VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium" />
<meta name="citation_author_institution" content="Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium" />
<meta name="citation_author" content="Erwin Swinnen" />
<meta name="citation_author_institution" content="VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium" />
<meta name="citation_author_institution" content="Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium" />
<meta name="citation_author_institution" content="Laboratory of Functional Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5215-0922" />
<meta name="citation_author" content="Liesbeth Demuyser" />
<meta name="citation_author_institution" content="VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium" />
<meta name="citation_author_institution" content="Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium" />
<meta name="citation_author" content="Herlinde De Keersmaecker" />
<meta name="citation_author_institution" content="Laboratory of Biomolecular Network Dynamics, Biochemistry, Molecular and Structural Biology Section, Department of Chemistry, KU Leuven, 3001 Leuven, Belgium" />
<meta name="citation_author" content="Hideaki Mizuno" />
<meta name="citation_author_institution" content="Laboratory of Biomolecular Network Dynamics, Biochemistry, Molecular and Structural Biology Section, Department of Chemistry, KU Leuven, 3001 Leuven, Belgium" />
<meta name="citation_author" content="Hélène Tournu" />
<meta name="citation_author_institution" content="VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium" />
<meta name="citation_author_institution" content="Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium" />
<meta name="citation_author" content="Patrick Van Dijck" />
<meta name="citation_author_institution" content="VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium" />
<meta name="citation_author_institution" content="Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium" />
<meta name="citation_author_email" content="patrick.vandijck@kuleuven.vib.be" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1542-897X" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=C. Bai;citation_author=X. L. Xu;citation_author=H. S. Wang;citation_author=Y. M. Wang;citation_author=F. Y. Chan;citation_title=Characterization of a hyperactive Cyr1 mutant reveals new regulatory mechanisms for cellular cAMP levels in Candida albicans.;citation_pages=879-893;citation_volume=82;citation_year=2011;citation_issue=4;citation_pmid=21992526;citation_doi=10.1111/j.1365-2958.2011.07859.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=D. P. Bockmuhl;citation_author=S. Krishnamurthy;citation_author=M. Gerads;citation_author=A. Sonneborn;citation_author=J. F. Ernst;citation_title=Distinct and redundant roles of the two protein kinase A isoforms Tpk1p and Tpk2p in morphogenesis and growth of Candida albicans.;citation_pages=1243-1257;citation_volume=42;citation_year=2001;citation_issue=5;citation_pmid=11886556;citation_doi=10.1046/j.1365-2958.2001.02688.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. H. Boysen;citation_author=S. Fanning;citation_author=J. Newberg;citation_author=R. F. Murphy;citation_author=A. P. Mitchell;citation_title=Detection of protein-protein interactions through vesicle targeting.;citation_volume=182;citation_year=2009;citation_pmid=19307607;citation_doi=10.1534/genetics.109.101162" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=R. S. Care;citation_author=J. Trevethick;citation_author=K. M. Binley;citation_author=P. E. Sudbery;citation_title=The MET3 promoter: a new tool for Candida albicans molecular genetics.;citation_pages=792-798;citation_volume=34;citation_year=1999;citation_issue=4;citation_pmid=10564518;citation_doi=10.1046/j.1365-2958.1999.01641.x" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=A. Cassola;citation_author=M. Parrot;citation_author=S. Silberstein;citation_author=B. B. Magee;citation_author=S. Passeron;citation_title=Candida albicans Lacking the Gene Encoding the Regulatory Subunit of Protein Kinase A Displays a Defect in Hyphal Formation and an Altered Localization of the Catalytic Subunit;citation_pages=190-199;citation_volume=3;citation_year=2004;citation_issue=1;citation_pmid=14871949;citation_doi=10.1128/EC.3.1.190-199.2004" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=M. Cloutier;citation_author=R. Castilla;citation_author=N. Bolduc;citation_author=A. Zelada;citation_author=P. Martineau;citation_title=The two isoforms of the cAMP-dependent protein kinase catalytic subunit are involved in the control of dimorphism in the human fungal pathogen Candida albicans.;citation_pages=133-141;citation_volume=38;citation_year=2003;citation_issue=1;citation_pmid=12553943;citation_doi=10.1016/S1087-1845(02)00520-0" />
<meta name="citation_reference" content="citation_journal_title=Trends in cell biology;citation_journal_abbrev=Trends Cell Biol;citation_author=M. Colledge;citation_author=J. D. Scott;citation_title=AKAPs: from structure to function.;citation_pages=216-221;citation_volume=9;citation_year=1999;citation_issue=6;citation_pmid=10354567;citation_doi=10.1016/S0962-8924(99)01558-5" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=C. Corvey;citation_author=P. Koetter;citation_author=T. Beckhaus;citation_author=J. Hack;citation_author=S. Hofmann;citation_title=Carbon source-dependent assembly of the Snf1p kinase complex in Candida albicans.;citation_volume=280;citation_year=2005;citation_pmid=15890650;citation_doi=10.1074/jbc.M503719200" />
<meta name="citation_reference" content="citation_journal_title=Biochem. J.;citation_author=T. Daniel;citation_author=D. Carling;citation_title=Expression and regulation of the AMP-activated protein kinase-SNF1 (sucrose non-fermenting 1) kinase complexes in yeast and mammalian cells: studies using chimaeric catalytic subunits.;citation_volume=365;citation_year=2002;citation_pmid=11971761;citation_doi=10.1042/bj20020124" />
<meta name="citation_reference" content="citation_journal_title=Microscopy research and technique;citation_journal_abbrev=Microsc Res Tech;citation_author=G. Diaz;citation_author=L. Polonelli;citation_author=S. Conti;citation_author=I. Messana;citation_author=T. Cabras;citation_title=Mitochondrial alterations and autofluorescent conversion of Candida albicans induced by histatins.;citation_pages=219-228;citation_volume=66;citation_year=2005;citation_issue=5;citation_pmid=15940680;citation_doi=10.1002/jemt.20161" />
<meta name="citation_reference" content="citation_journal_title=Virulence;citation_author=K. K. Fuller;citation_author=J. C. Rhodes;citation_title=Protein kinase A and fungal virulence: a sinister side to a conserved nutrient sensing pathway.;citation_pages=109-121;citation_volume=3;citation_year=2012;citation_pmid=22460637;citation_doi=10.4161/viru.19396" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=M. Gerami-Nejad;citation_author=K. Dulmage;citation_author=J. Berman;citation_title=Additional cassettes for epitope and fluorescent fusion proteins in Candida albicans.;citation_pages=399-406;citation_volume=26;citation_year=2009;citation_issue=7;citation_pmid=19504625;citation_doi=10.1002/yea.1674" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=R. Giacometti;citation_author=G. Souto;citation_author=S. Silberstein;citation_author=L. Giasson;citation_author=M. L. Cantore;citation_title=Expression levels and subcellular localization of Bcy1p in Candida albicans mutant strains devoid of one BCY1 allele results in a defective morphogenetic behavior.;citation_pages=64-72;citation_volume=1763;citation_year=2006;citation_issue=1;citation_pmid=16473139;citation_doi=10.1016/j.bbamcr.2005.11.016" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=R. Giacometti;citation_author=F. Kronberg;citation_author=R. M. Biondi;citation_author=S. Passeron;citation_title=Catalytic isoforms Tpk1 and Tpk2 of Candida albicans PKA have non-redundant roles in stress response and glycogen storage.;citation_pages=273-285;citation_volume=26;citation_year=2009;citation_issue=5;citation_pmid=19391100;citation_doi=10.1002/yea.1665" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=R. D. Gietz;citation_author=R. H. Schiestl;citation_author=A. R. Willems;citation_author=R. A. Woods;citation_title=Studies on the transformation of intact yeast cells by the LiAc/SS-DNA/PEG procedure.;citation_pages=355-360;citation_volume=11;citation_year=1995;citation_issue=4;citation_pmid=7785336;citation_doi=10.1002/yea.320110408" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=A. M. Gillum;citation_author=E. Y. Tsay;citation_author=D. R. Kirsch;citation_title=Isolation of the Candida albicans gene for orotidine-5'-phosphate decarboxylase by complementation of S. cerevisiae ura3 and E. coli pyrF mutations.;citation_pages=179-182;citation_volume=198;citation_year=1984;citation_issue=1;citation_pmid=6394964;citation_doi=10.1007/BF00328721" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=G. Griffioen;citation_author=J. M. Thevelein;citation_title=Molecular mechanisms controlling the localisation of protein kinase A.;citation_pages=199-207;citation_volume=41;citation_year=2002;citation_issue=4;citation_pmid=12172960;citation_doi=10.1007/s00294-002-0308-9" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=C. D. Hu;citation_author=Y. Chinenov;citation_author=T. K. Kerppola;citation_title=Visualization of interactions among bZIP and Rel family proteins in living cells using bimolecular fluorescence complementation.;citation_pages=789-798;citation_volume=9;citation_year=2002;citation_issue=4;citation_pmid=11983170;citation_doi=10.1016/S1097-2765(02)00496-3" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=R. Jiang;citation_author=M. Carlson;citation_title=The Snf1 protein kinase and its activating subunit, Snf4, interact with distinct domains of the Sip1/Sip2/Gal83 component in the kinase complex;citation_pages=2099-2106;citation_volume=17;citation_year=1997;citation_issue=4;citation_pmid=9121458;citation_doi=10.1128/MCB.17.4.2099" />
<meta name="citation_reference" content="citation_journal_abbrev=Chem Soc Rev;citation_author=T. K. Kerppola;citation_title=Visualization of molecular interactions using bimolecular fluorescence complementation analysis: characteristics of protein fragment complementation.;citation_pages=2876-2886;citation_volume=38;citation_year=2009;citation_issue=10;citation_pmid=19771334;citation_doi=10.1039/b909638h" />
<meta name="citation_reference" content="citation_journal_title=Biotechniques;citation_author=Y. Kodama;citation_author=C. D. Hu;citation_title=Bimolecular fluorescence complementation (BiFC): a 5-year update and future perspectives.;citation_pages=285-298;citation_volume=53;citation_year=2012;citation_pmid=23148879" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=L. Kraakman;citation_author=K. Lemaire;citation_author=P. Ma;citation_author=A. W. Teunissen;citation_author=M. C. Donaton;citation_title=A Saccharomyces cerevisiae G-protein coupled receptor, Gpr1, is specifically required for glucose activation of the cAMP pathway during the transition to growth on glucose.;citation_pages=1002-1012;citation_volume=32;citation_year=1999;citation_issue=5;citation_pmid=10361302;citation_doi=10.1046/j.1365-2958.1999.01413.x" />
<meta name="citation_reference" content="citation_journal_title=Methods Mol. Biol.;citation_author=L. Y. Lee;citation_author=S. B. Gelvin;citation_title=Bimolecular fluorescence complementation for imaging protein interactions in plant hosts of microbial pathogens.;citation_pages=185-208;citation_volume=1197;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=M. M. Maidan;citation_author=L. De Rop;citation_author=J. Serneels;citation_author=S. Exler;citation_author=S. Rupp;citation_title=The G protein-coupled receptor Gpr1 and the Galpha protein Gpa2 act through the cAMP-protein kinase A pathway to induce morphogenesis in Candida albicans.;citation_volume=16;citation_year=2005;citation_pmid=15673611;citation_doi=10.1091/mbc.E04-09-0780" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Biol.;citation_author=K. E. Miller;citation_author=Y. Kim;citation_author=W.-K. Huh;citation_author=H.-O. Park;citation_title=Bimolecular fluorescence complementation (BiFC) analysis: advances and recent applications for genome-wide interaction studies.;citation_pages=2039-2055;citation_volume=427;citation_year=2015;citation_pmid=25772494;citation_doi=10.1016/j.jmb.2015.03.005" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=T. Miwa;citation_author=Y. Takagi;citation_author=M. Shinozaki;citation_author=C. W. Yun;citation_author=W. A. Schell;citation_title=Gpr1, a Putative G-Protein-Coupled Receptor, Regulates Morphogenesis and Hypha Formation in the Pathogenic Fungus Candida albicans;citation_pages=919-931;citation_volume=3;citation_year=2004;citation_issue=4;citation_pmid=15302825;citation_doi=10.1128/EC.3.4.919-931.2004" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=S. M. Noble;citation_author=A. D. Johnson;citation_title=Strains and Strategies for Large-Scale Gene Deletion Studies of the Diploid Human Fungal Pathogen Candida albicans;citation_pages=298-309;citation_volume=4;citation_year=2005;citation_issue=2;citation_pmid=15701792;citation_doi=10.1128/EC.4.2.298-309.2005" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=S. M. Noble;citation_author=A. D. Johnson;citation_title=Genetics of Candida albicans, a diploid human fungal pathogen.;citation_pages=193-211;citation_volume=41;citation_year=2007;citation_pmid=17614788;citation_doi=10.1146/annurev.genet.41.042007.170146" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=S. Palzer;citation_author=Y. Bantel;citation_author=F. Kazenwadel;citation_author=M. Berg;citation_author=S. Rupp;citation_title=An expanded genetic code in Candida albicans to study protein-protein interactions in vivo.;citation_volume=12;citation_year=2013;citation_pmid=23543672;citation_doi=10.1128/EC.00075-13" />
<meta name="citation_reference" content="citation_journal_title=Crit. Rev. Microbiol.;citation_author=D. Poulain;citation_title=Candida albicans, plasticity and pathogenesis.;citation_pages=208-217;citation_volume=41;citation_year=2015;citation_pmid=23962107;citation_doi=10.3109/1040841X.2013.813904" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. A. Santos;citation_author=M. F. Tuite;citation_title=The CUG codon is decoded in vivo as serine and not leucine in Candida albicans;citation_pages=1481-1486;citation_volume=23;citation_year=1995;citation_issue=9;citation_pmid=7784200;citation_doi=10.1093/nar/23.9.1481" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. Schaekel;citation_author=P. R. Desai;citation_author=J. F. Ernst;citation_title=Morphogenesis-regulated localization of protein kinase A to genomic sites in Candida albicans.;citation_pages=842;citation_volume=14;citation_year=2013;citation_pmid=24289325;citation_doi=10.1186/1471-2164-14-842" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. H. Shah;citation_author=R. Nostramo;citation_author=B. Zhang;citation_author=S. N. Varia;citation_author=B. M. Klett;citation_title=Protein kinases are associated with multiple, distinct cytoplasmic granules in quiescent yeast cells.;citation_volume=198;citation_year=2014;citation_pmid=25342717;citation_doi=10.1534/genetics.114.172031" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=N. C. Shaner;citation_author=P. A. Steinbach;citation_author=R. Y. Tsien;citation_title=A guide to choosing fluorescent proteins.;citation_pages=905-909;citation_volume=2;citation_year=2005;citation_issue=12;citation_pmid=16299475;citation_doi=10.1038/nmeth819" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=M. A. Sheff;citation_author=K. S. Thorn;citation_title=Optimized cassettes for fluorescent protein tagging in Saccharomyces cerevisiae.;citation_pages=661-670;citation_volume=21;citation_year=2004;citation_issue=8;citation_pmid=15197731;citation_doi=10.1002/yea.1130" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=V. R. Stoldt;citation_author=A. Sonneborn;citation_author=C. E. Leuker;citation_author=J. F. Ernst;citation_title=Efg1p, an essential regulator of morphogenesis of the human pathogen Candida albicans, is a member of a conserved class of bHLH proteins regulating morphogenetic processes in fungi;citation_pages=1982-1991;citation_volume=16;citation_year=1997;citation_issue=8;citation_pmid=9155024;citation_doi=10.1093/emboj/16.8.1982" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=B. Stynen;citation_author=P. Van Dijck;citation_author=H. Tournu;citation_title=A CUG codon adapted two-hybrid system for the pathogenic fungus Candida albicans.;citation_volume=38;citation_year=2010;citation_pmid=20719741;citation_doi=10.1093/nar/gkq725" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=P. E. Sudbery;citation_title=Growth of Candida albicans hyphae.;citation_pages=737-748;citation_volume=9;citation_year=2011;citation_issue=10;citation_pmid=21844880;citation_doi=10.1038/nrmicro2636" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. K. Sung;citation_author=G. Lim;citation_author=D. G. Yi;citation_author=Y. J. Chang;citation_author=E. B. Yang;citation_title=Genome-wide bimolecular fluorescence complementation analysis of SUMO interactome in yeast.;citation_volume=23;citation_year=2013;citation_pmid=23403034;citation_doi=10.1101/gr.148346.112" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=M. Walter;citation_author=C. Chaban;citation_author=K. Schutze;citation_author=O. Batistic;citation_author=K. Weckermann;citation_title=Visualization of protein interactions in living plant cells using bimolecular fluorescence complementation.;citation_pages=428-438;citation_volume=40;citation_year=2004;citation_issue=3;citation_pmid=15469500;citation_doi=10.1111/j.1365-313X.2004.02219.x" />
<meta name="citation_reference" content="citation_journal_title=Chemistry &amp; biology;citation_journal_abbrev=Chem Biol;citation_author=Q. Wang;citation_author=A. R. Parrish;citation_author=L. Wang;citation_title=Expanding the genetic code for biological studies.;citation_pages=323-336;citation_volume=16;citation_year=2009;citation_issue=3;citation_pmid=19318213;citation_doi=10.1016/j.chembiol.2009.03.001" />
<meta name="citation_reference" content="citation_journal_title=FEBS Lett.;citation_author=T. Ye;citation_author=L. Bendrioua;citation_author=D. Carmena;citation_author=R. Garcia-Salcedo;citation_author=P. Dahl;citation_title=The mammalian AMP-activated protein kinase complex mediates glucose regulation of gene expression in the yeast Saccharomyces cerevisiae.;citation_pages=2070-2077;citation_volume=588;citation_year=2014;citation_pmid=24815694;citation_doi=10.1016/j.febslet.2014.04.039" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Bimolecular Fluorescence Complementation Tool for Identification of Protein-Protein Interactions in Candida albicans" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
<meta name="twitter:description" content="Investigation of protein-protein interactions (PPI) in Candida albicans is essential for understanding the regulation of the signal transduction network that triggers its pathogenic lifestyle. Unique features of C. albicans , such as its alternative codon usage and incomplete meiosis, have enforced the optimization of standard genetic methods as well as development of novel approaches. Since the existing methods for detection of PPI are limited for direct visualization of the interacting complex in vivo , we have established a bimolecular fluorescence complementation (BiFC) assay in C. albicans , a powerful technique for studying PPI. We have developed an optimized set of plasmids that allows for N- and C-terminal tagging of proteins with split yeast-enhanced monomeric Venus fragments, so that all eight combinations of fusion orientations can be analyzed. With the use of our BiFC assay we demonstrate three interaction complexes in vivo , which were also confirmed by two-hybrid analysis. Our Candida -optimized BiFC assay represents a useful molecular tool for PPI studies and shows great promise in expanding our knowledge of molecular mechanisms of protein functions." />
<meta name="og-title" property="og:title" content="A Bimolecular Fluorescence Complementation Tool for Identification of Protein-Protein Interactions in Candida albicans" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/10/3509" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Investigation of protein-protein interactions (PPI) in Candida albicans is essential for understanding the regulation of the signal transduction network that triggers its pathogenic lifestyle. Unique features of C. albicans , such as its alternative codon usage and incomplete meiosis, have enforced the optimization of standard genetic methods as well as development of novel approaches. Since the existing methods for detection of PPI are limited for direct visualization of the interacting complex in vivo , we have established a bimolecular fluorescence complementation (BiFC) assay in C. albicans , a powerful technique for studying PPI. We have developed an optimized set of plasmids that allows for N- and C-terminal tagging of proteins with split yeast-enhanced monomeric Venus fragments, so that all eight combinations of fusion orientations can be analyzed. With the use of our BiFC assay we demonstrate three interaction complexes in vivo , which were also confirmed by two-hybrid analysis. Our Candida -optimized BiFC assay represents a useful molecular tool for PPI studies and shows great promise in expanding our knowledge of molecular mechanisms of protein functions." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
    <title>A Bimolecular Fluorescence Complementation Tool for Identification of Protein-Protein Interactions in Candida albicans | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"q3S57UgehSI8uN9YXhmlecDgaWVAPDMII2TGdou3lGI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"486780","apath":"\/ggg\/7\/10\/3509.atom","pisa":"ggg;7\/10\/3509","ac":{"\/ggg\/7\/10\/3509.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/10\/3509.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/10\/3509"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/10\/3509"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/10\/3509":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hNbkH7huGRctgJif4QSEmsnYnpfqReSSSCuYtW6ewsRHfO9qFLMWz+9p4zVvTCOIc95OcfgnRAGnqomP3ODq6jFAhstKpYvfT5lEKKSKDDwrbyha7bbkxUR5IdsPlPJlw1px5x6pgaPJl9Bmv2wSUQqWK+Ux+3oRepY8Nx046kCWkUtzsTWxZ++01mnSN9l6Zh2wxSOIetLRirfGFW74pPudwLZgdCAg1sMWTuvS9Ozgn49V7hlosHi94CVHEdwID\\\\\\\/6IS8AgyqBywex5bumtjPbO4wcQ\\\\\\\/ucORgBqCvCU1b87CiJM7O1gI\\\\\\\/0ezd1RYrfcW8ivOr3XNhkEZ7Qi6xTG2RMJPJkpl5uqbJO0gPJ8Ix4UQ+ehZwL4wofL4vHpEbqkbmjIZAoD8F9U+6S629z9uHFeDmOBrQ6ZiMghsgkYjHiJHjX1IemMZy8utvLKxMVphIl7pXWu6vEOsc1o9JBaEwF\\\\\\\/9wO\\\\\\\/1Mm0dSzob2E9iYzrj1vsJLnsR6Ir5TIkZc9H1OoRlxB1eswhT8mRgsrMb9IlO03teulpMeIZjLK6IWNuemu0G\\\\\\\/1VRw\\\\\\\/Ycb+p0irPBRpXAF\\\\\\\/0QZP6tbZGRaxh6jVba3ivSmEIlTmFulP\\\\\\\/X6WpfbezXVOTs\\\\\\\/8RtsVr78B\\\\\\\/X1A9DK+DsKpUp\\\\\\\/5fy0Zim6c294\\\\\\\/yBcaRo8sPTpvUCGK09iGetdT0nxsTREUS10OWwSTuP8SaDwPV3bfzAZ6AvMgSw1svWYHtyYxLrbAUxcOElUdQOAdHi+KMjH+4KgLGI00MIjORpTPIRvEz1eOlbvFRAh+P8nWBWcLg21xc5X8ZrGEuFnYpyveFf1Rzn8xvhIfqQTgvxHX4VK7pgXG7+KkRKxGx+1+FyceVBbG3MhE8M7NSywLVxARXFYCqH3rtznmvEhpKSL6DOnFB+guT6DtYkn7WKDYikyvuyzbzL9EIsQw8i9bB\\\\\\\/0Ym3I53RvbFnzHEUa0yriVQC1eVW25LN95mUPNz9e7xY\\\\\\\/L3izXboUWUNuDu9rxrfVh\\\\\\\/XTp9Bojr+rLn9tsx9x0lLkKw5b5dvX6upYXwL4l6+qxWbAdpR4poCv5X4FOvoRKpDhZW5T5gxQdo5kQNHhVglZVc3WG1ZAFHqPE0T1EyqVEfnBhtEK0JbM+cvl8eiNbSCjPdV6CX+Icu4cgIABGt567nQXP3ZUJj83Hhep5bHBFuAAPNh5OT3ODSXoxdLOFOuZfNvPgpQpUdHOdNayeGcngKjXb\\\\\\\/XiBZSRr7BEwUzaKQD\\\\\\\/OmhpQkRQGLC0fR0yL0GedO1fDi\\\u0022,\\\u0022iv\\\u0022:\\\u0022eusmA5aTqCE\\\\\\\/+4WxuyfaoQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224c0d64401dd7387d0403892144e06d5c036969909919614064b57449a658eb83\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pd9mx3RqoBinkYPMAB00viI0KlOs2L3XQpxF3xg9AJK\\\\\\\/BWzxT5i4CCdnk5KdChEfPBNVD9EBMczM5Qc9GWd+uWTdRXQ3WgUrCZWMdW4vaVsYCpB26pV5JNLdNj2sWIe3pJ6sU3ElmMdy4xwl27vdd0njwa5SS5+ssupct\\\\\\\/kTHqE2wtB5tegYjGqXfebgSKub2oC9mE5tEN38ZnWxoK4HHyHB+j71R\\\\\\\/DeMNMB0Zndg84RSnW3vHvcJz9Y7kV\\\\\\\/+2j22Lhs4SbyBTZm2p+EFSJ1UvuOh+zlzZtHZt+LfC\\\\\\\/eguUsPFA8WlpNfCrmvMmWf0rbt9CZ+O4ESbnae1dBfhQ46AVBkUVr3DKbreL8fuPT3PaBrGRzKztn\\\\\\\/SjCwrf5sE6t43uwsOjnkpj5i2EgRwU6it4O772gh24NOiWUdbyrU0jRynkoyF7+\\\\\\\/OVnnFSIoiX4tyEWcM7tQwi0pMupJTddVRWthLRhstXETQ\\\\\\\/zm+\\\\\\\/DpqLhjYJSRt6girSLNiVI2MB02sBznVvdymy\\\\\\\/CysJzWQkijViKN+lTvCvC7zJ0xRh0pDGtvAjOrtEWyy5ZjwiRDNed\\\\\\\/cDpVvVCWwWFBe692y1R1ZRrfOxJrbvYLILpqgOrtrA+Uc\\\\\\\/axg3bERx1fbBHHqQj+6UxUATdJ8mQuo0083A6tWAIg0OeH\\\\\\\/IXo6KdfDx8BsNYklxX80Nj7fpghg8JuQzqgGUS7jVFrw9jjB0gmWOBy61T2U2NtfkDCyCFR8FWe3FliOpm8S99aPHdAZFS\\\\\\\/oZACxNz\\\\\\\/QB5y+LIpyKuzEnOx\\\\\\\/K543wfq0D1OESNag4ZnmdYcoqyiXkxfcj7ZC5sURNtTbT7XjVl76ygmql5DEYSZF+ZWDNajF05aREOV1VwPULX0z+3kw\\\\\\\/Q\\\\\\\/2+QNjs\\\\\\\/k2G9ytsuW+9rqVZIvsLfcz27seGcPgIqi2Md3erBqut9dQkKmAgRzErbcDr7rBAg+gys4+z3vPeAPW49YfIaV1qdHPXq2gfzXrvtKHGrnMxnSCU7+WAObL633MLDQVOuZLjQY0KBFaeOBOEEFmmoSotYlWvHlrEIUB\\\\\\\/RntA4LQ5nhLiClFf2eeLJVsP4ti3POlOzc0H6bezGsgTba3tcnVmU6htuy+rvLzGH3+sYIIyXmQtrHSEWCuAH3OAU7oiS37ZytUS5\\\\\\\/EL6UxeoxQk6oOvU3tmdwmrUe7FNZwTPwGjkvn3\\\\\\\/6bJllkeFKtFSBZZ6LZaRr0+296qyC\\\\\\\/YHTFBd6G5IRjYhhHM4BphhJAI\\\\\\\/EDu7qcX8R3\\\\\\\/5jcLmmF1ZU3XmAdGrUmGRe7cYvXjVobvUU06lR\\\\\\\/A19qWa1R5JnU9BUn3A1Dsqzx9tsF6NsMdQZv3pQsz283yV3udhMsASiARFQFPTFIYH\\\\\\\/7Pb2Agvc4oDWDqTOuKo525gdtAt2YqhAtTiZECK1ok1SlPPv1ydefXWgxkBeNEO0tZ7XG7pxp9Gh24LLJRaU7HnOdSl5\\\\\\\/z2Ps3sjh0eiR0K+kzDTM67pnOHnBwL1eDowKzEND+b4msy\\\\\\\/j\\\\\\\/FPI2+YWQ04i11p2kRQHd58+W0kotnnd154G76CD++xqaciZBxjvnZ\\\\\\\/+fpgXlV6ovTYXWm8kaSL1yiL4VPuOrk01ejKdTny6623IsQqstdCTmzMbtdvPkSBQOFt5o1OcR1s3GM64c66WeoYdzhPPwaUoTk48tO+6frL0if3XK2Si7Xio\\\\\\\/2N3ClqcKEvGvSSv9wOVVIJkCaRqxatFU21jki62B0JyZgrZN0BC9audY3\\\\\\\/HN+GydEkBo8Syvf4Dssrfjhl0FAvzZaOh3MSk+ldLAP6CImUR6XjWaSH30rtCGn3rHeOv9rlQzPoib41MfUTs1cPNlP39b4qZL\\\\\\\/nOAJ0EWH53vq\\\\\\\/lLVuRxBIVntWJVt2XQuXGK1cWog6lOTGLIiCAN7vAtDf\\\\\\\/d+ypwf8AaoydM0J7gNLLC59S7+4OLncxPxof+P44BLT7V2YEih2NqVPY\\\\\\\/29+iF9ja8q7J9f\\\\\\\/327KjawKIiuIKhS2szbZbVNG0QJ6Lyov\\\\\\\/ObnTXi7\\\\\\\/1h4Bfa2RO05v1doeFaTEv\\\\\\\/yMW1QetaBQEPsgQXhaCG0FkUriBufbFnMhwEhFNzaUavGC\\\\\\\/UcFhA3rZnOk5mazysAaZtXYD2JXw1Zw9PtdfGEN\\\\\\\/HV8z0D83BF4sIPGSPv1rV1mEbEDeDWefLcUhbhKwlzW3A9wXGBsKsPCee47+72+EfUAenMe7bt7Zs+yOBXkaSSVetGCsSqC5jc3HigxnkA+eFPjcCA+Qyp6LAXapzQEgbBGh+dMNvPdX1QQOnkWR01HcmJRe2Ec\\\\\\\/pO98jGYIh\\\\\\\/oSFUw3zC0lQN\\\\\\\/tfAk5tIr27\\\\\\\/OZgAI+d8W0rh23hcF8mp9E6V9JTcd6zElTv0lPV+DUSUy0Uvs+Bv3FL5OBvdqAgawnU735YJz+\\\\\\\/FG5EstTr9DyprG1sS7S9auqm7RCEW2axCkYzZU24\\\\\\\/oDXDnEx1BZ5Ay\\\\\\\/MqqjT6fChx1SIR+anot7E1qaFTb\\\\\\\/IXhZugL5H+LkStSS16Fuyp98up01YNXK2pdCpPqPmGCRY8gLDDvJEDJcQ1hK3Coq0aYrAheDwDjNqfIptZjQUQPs7ts9rDGPpRfPUxGmFnV3yIY1ypNLlK0DxXZQeurTMY3HRuzh037O64gM8EehCpLiunFHA66Gowu\\\\\\\/erG5SlsamlBOdG9fefl7wyr1vkazANxMHmTCtSNvDAYw0PoLGzjrer8UBPzxRwBs4sgK90qvnMOpgLZko2jbZCQElbzQSu+UQYZRbtGBic3yyk7rDKhfXzA2ddji+qPqJ3ZXpxdu5xQkjZQpTTQ8WTRkLRGZh9dPTKtMrx0CE65vChlBKdB9k3du\\\\\\\/bkgagY9gBD7DazMOFbIqZl\\\\\\\/q+Ox86AIgpWCPno214zq\\\\\\\/aRPME02gZSOfOtgFknMxNhagbogmOHvUcNlnwwaCrs6A2Bd88k6opnb8fFNk\\\\\\\/tXKNz1RNm0Vwbp0FIHuDW7NUT6D5PvvrZq0j1CJOqiGxypRVO5vZ\\\\\\\/Mh9ozRguWQPSYJTFCXcUPc4FA+5pQxeHGNYZgSClAZyIUILjmpVgP953NAtZx4WouKrcGUeFRq0YsyKzDFk+NuNlrp6RQcRvm4az1iGIwb0b8nOAw8qu+VKgY+OA+6KWhVg+K1uLNAwDR2DBta1knKXLVQIzqOeBkLKn8iipfx9lpq\\\\\\\/9EJiskeV40LC9\\\\\\\/9gcTjd9U2YiDAeQ4pjJMOX2jiLNyWr1lRLgq7\\\\\\\/1D9gIX\\\\\\\/7EaUxcbHyiWbqk4Ue0OW\\\\\\\/+62c6mdL0rhHDXOns23U0CWYFXBNdpeE6Ip6FLeEgIhXBPmJ9oGYo\\\\\\\/IBIPe5oNfApeel4quaXjTBc1A65Xx8COaYEy1PjycbyZqLooCizpUQTZRCI\\\\\\\/8TTuPHaJUN3R+GZX+7lFKQJTfiADPf9Go1YtIAgGKBTI63lkiHyDz8NvLohCrlrg09w9nP6GJKhaYic2OabmHkWQ82ma0L5hHIQUqgE9FNHGcu9VpREeyi2XWct4KM2NFhabgrScM38u2xO77Sfk7BL9VvLiUGHmT6qSlodrvz\\\\\\\/kUGoEWafqGI+mVUCuzgPhxSmyFU\\\\\\\/I8Kjng96uHXHYmfmlYrPimYsrXTalDlUbpv1jBScq5MXYAA3PkXh07XMSILK9S1znWFJO3MMBeBOAdhnhfoVmAoyzM+I5si8cuajuyfWmKLCehjuxJXwamkOBTZezqSeyN4i0EDyrRG7ut6tTDpGOd+oNxnTcp6KNl0E5CVzh4aha4H9Qh3Z+kpHhY\\\\\\\/ddY+gAVEaKhaj+2Mw+wrrkzh4ZTsRmceJNK7KxVl3az3T5u2IwBx+HtcQq5oq1iUKL1n2oNqcOlXBxzRb3dVU3MHgVxL0+UxH\\\\\\\/rS6FfvWK2JtOVsS6jRFOqLmNJ2RTm7ZUP5BWs\\\\\\\/Lc8wnPGAKJ9IZL4HZSyqwUoVpIw4cISOWi3HniLGDwFZoCpHn\\\\\\\/4iswJK\\\\\\\/Vj6Iz619cq2x4ibGQC0ipMOauryMyUqRaiDW6e3A7Tq5yw5iSWSLE1dohijlr4RgPfuw3Q34wTE\\\\\\\/SQLNtNMf3ORZKh\\\\\\\/TcGhHceXkKWuVstjD0eE91b8fTkgJkj4IjHMCW6\\\\\\\/cQhBm2sArG1B+TY716RSfhgX6rG6m7jbtqwAlDrwgfpHpk13ph\\\\\\\/jaMLN6gjtwBWTxZ+yZInhwCMN7+xrYngsypcyNp8XfRLITNqZ4WjmQNeMDefoadn\\\\\\\/ih62KEX6wX6Io+6tCr8BiqOIJNQmvgmHWX8Oaso20DNbKkVqoYBcVBb3G6nttNkDwOTSER9NUKndoX4Y2GF\\\\\\\/rAz4OVEZNUSD8VS2TSSwmcUN96Jo0vD4iToSFsjX5ms7+QoiA74rPbqKriXGWwh3qn7ST\\\\\\\/l\\\\\\\/gElWxfC\\\\\\\/bKZMJDbzha8\\\\\\\/wy\\\\\\\/xLP0ln2dsicEqdSeLTXIdB8O\\\\\\\/TppwgCKJit6N6DhZYcAjskDTvackV\\\\\\\/iY8UIhPpkRk\\\\\\\/58toQ8Kbj+lv\\\\\\\/oVnyW0OJIcQu9t9y6y2SNabZ\\\\\\\/AC2I7tLHMji1dRYbuEAqsQMiGdrYhbaQMLOftTxugIsdlWgdiMqDUVtnz8+dbaFLZTM3U7H0fZBNQHJf5dEqEsDcz9YRwd4AwzWXLSAmsoDa0TeX+nHTz4NocAZEdxy5MqwrgtsMaEHvnzbEbiUXZLod0cqMXCscGvQvh\\\\\\\/8lK+XDoT1SZkW9mHuWGD3zDrKyjFQo0\\\\\\\/we0yUkbUyz6cXZvUQPeIOdzQyqo8RkkieJvOArepcS0H1livUlHbwHeljwYREhi9q4f2xydJswjCfri90fggpd3+lVxTtc6UQ6aJCAxEqEY0st7n980W6cpUOjQOs7n0oq8Gc8plS1qmeBSNoPaZy0Vdt2XzfKby9I8zg6uAxr5uFD78nx4Qkzx+WNjk4mxMIeKcFDfqp1SNpggBq6rrk0mJ1FBG\\\\\\\/UQpUWzRBn0A16au7vtmh7zCzzbUbzHlge61QGhWiVb71Xg6vyePHvOCpdffeZM2KtBNQTAl+1V5\\\\\\\/wWHYJP7A4kHWZhT+K\\\\\\\/BF+LXZJAgfaSEUvkiXjRzoM50djN0nTp0rj6bpkraC2yZ9biUIcbTJbDGyyo+L7\\\\\\\/fCOFWKEVYJtgT2KuNfxdZooV\\\\\\\/2kCrxpAcAUGj1g6WZ6v7X8hftSfKpppIRzCy+MyBsPLu6EGpQ02srvYe7EvgG2Btd2qhUGumuVRFlGWRdRrTE\\\\\\\/M6iKW2F64nvYywAptNO5cgBSvgmfD28gN489VN4h4qXrE6F9KibQSidpMkroplsSNSdi90e40WwZ5j0GpxhPDbx26ojkMkX0vrUWYH2QRtkq7MxxS2scjWMbZUm1fLyQnIWQBjjV\\\\\\\/038aRslX5VKXeNbVOp5KiMZaptplDvlihEjHc7rBt5MdKQa1aEvYpWEhK6WsfAIivzgGxuOKgclduOb7JWERZURtdcaHIUYV\\\\\\\/UPf1N43tcfD1YludXU52kfakr5T+32ysiOGofsdyXi3ZMAGGRTPOdAcgFkmhtEQoT4dVMzS0wUOAcH15uxFjT5vzXizS\\\\\\\/JIDXg6F3NjEPut2tXEhvVW5mYRJs27eUZiaWo4IDp3LSAvtjq49dJOF2usEoZEf9zPAvQxBS77I51QKyifXfBfzGLz8Md6oKts38s9H1bSudwesPi70Ag0Mm1UCW2+1OZYiaIT91O2qNS\\\\\\\/74JjQ8FMCm7D1snVdiLSJkgQaZx0TVK6bPlDe7T+pfNmAPi+e+BBc1iJYW\\\\\\\/fk2luH4RtP4nxUKMy0F0pN+8r6xwTjErtAh0ebAkGh1ITsENTutYLTudNdydpZgiAF\\\\\\\/7Wbo7VvO6\\\\\\\/neesctl7E5yvzzSrLnmEUwkCq3Gcm5Fkk\\\\\\\/3SwASjRipO+Gy0R6oVs36MJb8KinZp23gUybZI93VbT04l9csMS4X1j17lc7DgTQx8vKzCR\\\\\\\/C4\\\\\\\/k4ro4LDqkeOC4hSy9d8BDnPvTaieJuVLufn8tc\\\\\\\/a+RSzVwe3MJr4CTAp7L0lMSBMTpDVJa9tsnOOScbi\\\\\\\/77No1OuYB9tXCpCwvo7Ri+kH9Ii7MdfzVqkfmArzPUrKah8UDX0olKq0tyj\\\\\\\/H+2mBnRH5\\\\\\\/nxZy4pQDRIKVq81tu0Fj0O3jjFD\\\\\\\/q8Zjio7XNZsznuI\\\\\\\/6kuCuCebCDx33MBtfk15+l88ZD4gBot+7Mvmv+JDMBJNabmo+lqky4+fVSKbhQBJEEixld6FF4Zw9GCDKJ2FxPgTjwgGiHXZ5U+JiDgMmAShzWw+jII4\\\\\\\/dF9wQ0pRiktyHFOBZBmJ50YYCl98\\\\\\\/+cczkNjviMvx5\\\\\\\/BSWbRYfknZMaF3MJnQe3g\\\\\\\/OIX56e8FalDHe2Tg0sS4Es5SypFLh+AEyFoPevgqs1z4y8VpL0bzzkdx5wranX1Ym5brN\\\\\\\/fYEDZXuNfr8NRQ8MEMVL62H0+fVyRZSsJaCivX+xD69TKqkCanlb\\\\\\\/\\\\\\\/x1ShowkL7aP5GrOfVV1Qnf3Cs\\\\\\\/KqSvDVDoicXlEcpsD4LLFBmf2cElCcSQDbRg3vTQO2cmDiINOY1k9sSbs7m+OqHHmJ5FZK5jA5hz0p80snCK9a8Ea5TJrRiJgUwNGkD\\\\\\\/G446rNqVOHR62URNiQDx4ADKU0O+0LjONnaSmxrgA\\\\\\\/KkUUNH4Sk1j+wgZzYZnM5b36I5f5LL1q4H6f8fVXr+rA6KCHgUKOAdaXouQqm+LgWrmtuQyPzRjgVRrXki1E7cFAvz0+3GMf9baAKgeimi7TR8e84oEEaIu9wWxgT0G6F0O5Uz5SRrGkmokI+2ghslZsuwG2mbSqqIf\\\\\\\/vvqFWs0h5K7aGVxb6YmhzdCFkvVnYxMj+yT7ih8KvQrX\\\\\\\/JiaqPS5oj4MD93UbcME9BFcCYj2LWxL+cDNH43qb0Bg+jPBHY5HdJ1pWvpVDPn+l0RqBHc8saHcpTsfg2nixNLj3yHXSwvlRXnBZsHKc622Xu25YZ768GdRKDUStwBGFpXTNJQv\\\\\\\/BLSl6wPZ0SVKzL5BoKvMPJLmAQS8o9jfIs91eIHL0F4r3Yg5Y2+h2JAt6IgV0ND84U6aTsxL3S+zCVeF1tIlBps7uNS34F9n2l7ak4r7svwc6LrWnT1QoXDYxUM1vhTxNdEq8sj5Wm10ctO03mEeRaLwDsovUyvFFXMVbZ4pKkrMWOShaoGCEBKZReGVsbALp2TYAurHG3+SL99bIVqx6BEa1kO8+GAi43z0W3d\\\\\\\/pAbO\\\\\\\/D6vNgliH7DboGEpq7QDxtr7VE2We0AqQdF2HG9LDgR2hbOciwI49TJFybShAn\\\\\\\/iUBNH4xE7K1bC55gRkxcQD9ID60dUoSixlaL5OhxYHZf\\\\\\\/qNAknvyXIzJlc1VmL3abISo39YCC3lMXcDY7rHmZwzPyrD7O6z43CeNcVJ8zR\\\\\\\/fX3RG2hZcf4igIKFQArmuZLugGopbQg2hgbkojsC7hd1YYSG6KlSWuDxSXVl6k5iDOoNui4Cn+W2zifV7\\\\\\\/MgEHNC9A\\\\\\\/7qUOKAhqMRfXkmqjCl5RE2cCKCsh8VTr2TNxgJDSYEFxith9NaaNxcs2gTxx94gbD6I5sf2FteG4HuDU7x+gsK1Q2yyWpQbeNdAr5ZjWpEO49OAbDP3D+f1+4wX9UMfzL8u6cXNdkxjOkIbenXVREzpO+bGJZnBaarlVRxpY39Z\\\\\\\/5u6Oc8FcsZ4WyWf\\\\\\\/asQqn1rofqfdPQ2YVUsF3xaBQkbpEMVhSsidgYcOiMefeK2QKL+nvHQh3xmQFQWPt\\\\\\\/OmD8riEXG1ww9cn5Co7DmFHaAj9A6WMP56F8CRXLaiNAYoWf5efn\\\\\\\/68R4hDDmfel0Q\\\\\\\/5WGCoyZ5ODbJKXhEmrK+QZff5tm5KL8AIihxsIq\\\\\\\/M7ON0IJ5VsYSRplKdWzGg\\\\\\\/FnixImwzYfNKAC4DkANxi3Rvl6ORadDdGGdadLvO1rKmKJxnVHdCo5cFMXhqC7bz380wJdRQirvk487a94+iuYdTcOrL637xad1Gv2KHGsq\\\\\\\/vQ2JVG5WlvaUtO+MHqfERYzpS17p08Ioxrpvdde4s5op7HP83iJpSvoTwRn4Mp5oW6TMpPi0acl2YVRvpjyOqG5SxRE3Ml3gGTfW8AvQCyq0+BILhI0mbTxD8ob22AQMP448PWxNs4dx0z++kRLYGRJKB65MgFSQiWia9WQ9NLifCGk0g2MWro7ViCHrv3glRxtuT6oplFVUoHanTtOqkA+0YFQgmjQE+LxPHdYRIYoR4yU7\\\\\\\/MRWyR1UBJo2nt1ODXGJis8ifztt8K\\\\\\\/SMF5bZ27NODFBL70\\\\\\\/TGs+OMziIPliDFz+il+pc1UvD9iQ+8JWM+srOtwxbwCyoykyVLNTR9hF1ljMYnkkmRiuVajtfZUJPfFjcaT+7ErlgmoSLsHLBc4Hpr\\\\\\\/tfzVWRFJzqRRXmG4w\\\\\\\/JA8xSHSaiyFcqnP6xwUymaoOb02wakuQsuP7Pb9BC1C+J7MsGwoqI07vJoKXd9Y4O5reoolNaR8qkYqnDWyGbDeAQEcAf2He6iwh1DCTnory2hrbx8Hs4SbIiugyV2ANmLPsMKIEhI1INwDZEdYh4nb4E6Hz9Y6hPFR9qK2q7eGvZtbCVZhJ9413foG2ucqjScV86M9WdYT1b\\\\\\\/8TweL\\\\\\\/TXwvZtRT+TPQQfCdF6PZakBLAFFA62dwZGavqgxK\\\\\\\/jpss7zGAiBd086g8fFTfUgcp2wEtFwpEFZKcGQLtekZNDcmwH3JFqazmxvpp+\\\\\\\/w0ipFQe0JdfIyf\\\\\\\/V4\\\\\\\/fnbDdRnxUoNOYTe2RPcfwK44skrRJxoyVqUj\\\\\\\/i4Ec5gyjDrtYSXI9yrWa9fUKLA\\\\\\\/BOAYpO+rKxvlb5hMQO1sp88fAnKryfzNLwRRWAvS7EzDoKmh6t5w9WxWppF54ntBpoePJD+Xkhk\\\\\\\/1PNAw7y0LXdjFJ3pKHqTBwejxXgfyje06C+Q8VPKlGyiWjm0qrQY\\\\\\\/XMyIUtw\\\\\\\/\\\\\\\/UpTc+0aKCjATlXmHqR6JWMJTzpOenLqMMaaiMuHYzqLA9m03d2+50wJoMXY46Tnqj0n5zgPz7y6bhs\\\\\\\/yk+n404YbSqOlPnSqBM7eN\\\\\\\/v59vjZj+5qlNrBT5gdk6SySTp+AIj1fKShrNuRZEbbVUL3\\\\\\\/Eb67U1233eNMOr8GZ8L2w6XYju2pcwDTNhHXe63qMaOs6lbmh0Y8+Z71TA5Kv3xxjkUBEpHtrST\\\\\\\/Tcz\\\\\\\/zmNp9OsWqsFpmxMUqnGpMDIDjbtG8KeHe3WxN3lDdmuFS3lkLcvfppz7JM9TtBtH39D8bQnBZHGASwDpsgMRUKXhWxrjDlRRxj+p0xJvoIS8KEj+iEALDx3dHE0Z0KakCWwDkUjU5QpdlSHWY29E\\\\\\\/gBKpBwoxX+kDUWb\\\\\\\/u0Vh69Rj2vv8ji\\\\\\\/BbUqU3Fy2UguB7j3nemE4\\\\\\\/ici4DD+8cEkdN9CnWjXU0+8rdvCDm4SBVb+IZSV9mZkidL7DkI5A8RhR02zWGpmzjz5OTJrvaQhtjXuKZLxIXGL1oFK7IVCtyvnxO7Fvjn87S+Z5E85PpqFh9YDUT2xgjuejFQlDxupXTHuDLAmuDlsKhSUNXNbtBTuO7l4kwaO5TxQHzb\\\\\\\/QUEd2yL6TIaavgqIoQ4mY3LZCEWr3qVJ9iFVUINN7TRh3Z0OiZz\\\\\\\/4ClKqtQMIuzFA7Rfn7+Ce\\\\\\\/H0QlX6pfK28woo5BDTD8l72nCEspNt8jMvD6hH5UM9DNz2lEEREtH7EYSHOkqWkz3d6H5eIpSG2ZFDlokJIm88vZyPRJqyGWP+zMfsnqL0FnJkjPOKndeqn+A7f10w0HVLYWEdtYbvLzqO+9DnOhJQrN5VAXzdo0k7CyODNYjxsm0UZAWr6tzQykDdZRf56yi34zLGYZCBw166rkPPom\\\\\\\/MtQVYIQSt6uKZVOegdwMHm5of3UC2NjJBnTG8sQ49NnD7uk2DHn0tur2yP9qU6iE3ZkN1mRihHYMYZUDxlZ\\\\\\\/cKxQzzsRdh977rtGaqIbKWMLOXQR6QhmYYApyG7oU4L65F\\\\\\\/MjvhFjTroVY5yrytJiWJVB\\\\\\\/P0lPC6w+9ULHmwIgErHueaKKnU25p1WcioMzxS2JnRTb+h9i\\\\\\\/Z93CVUzCiNvbq2jZFVYYQQ8gSBhfY0awmQB+UPwZAizrAWzryu\\\\\\\/7+va7S6BxXQakB\\\\\\\/tx4eQeajT7DeiltUvaNpRtuR2fwECHsQhix1gD01u31C\\\\\\\/mBpzQV1DrSbFUznYxssk0\\\\\\\/BiEOSXmZXjIKDot6eR\\\\\\\/uhIM+fdb7UStzuKN39M6cchbU6ssfQSrnZvDAFB44TrPq7e2pseXJbKWqPegMf301\\\\\\\/lTqGIsnU1qxjHYesaeEaMxFUr4XdPqGQAyAgA+BnEPSNniVWTuea8dZxCAxeEMJilBu\\\\\\\/RikWroFeqjzBpmcQUeeVWb9hBaG5cu9Q49IAbQx2cNWrL7WPjPoITX2jURbW9Fp0pk5tmzugdINvKOtxfdkxjN88fT96cuuBBpYvoiLP8OUo0UCxKzYYxF8Hs0REbC80mX0WPMyIqc\\\\\\\/eec5zo4r2+9ZssINEVCFkdMgkLEMvoTNeQmSqmLboMc2bRnMU9MGFeT8iqnNQW2XdAnxCSJLILYE4UO2vrLEZ+sYcymI+RriZKqGc2wMNjCCPZS+v62AW78+dfrg5kkJfdXcG6WlSaj1PBmCsnV00Hz1OE3RHXkb+FIwYl506CwV321SdewSp3n5JlX+6zl6LDVgBR0ROD7u4\\\\\\\/l7RjvoZwhO3AeruZNPZXeb3bZGF4lbnEMCiTteAgTpcAEXZG\\\\\\\/La0j4BDnw0SxB49mmNDHHJeD4MGROwuzaMR4DWIalSd\\\\\\\/G2PlWvczPmvBensgw5V+\\\\\\\/yMUktwaDXC+t6sl8ltxilp1G+46qXT\\\\\\\/TQ1lCTDxriKXbQrvAGKOhQeW1oBmgcgTrZ1teFRbfwUBa+pvU4M9tbaE\\\\\\\/T0eZpHSRw4HT1BQXFHL5PHO2Lt5Rfi2+lrfuYybex3EzVn9JriQaDNSjcO7MGKmZiWtYfBqb00PU8za6OE9HN+TJJiN3sXQFRHvkwFjD0wpD8381\\\\\\\/yg1V\\\\\\\/gNkieWaOxvySUKUKiSPKDy1KDsZELdxbW7jFbhs7Joh1+3vUeMNNjDjHA8tA\\\\\\\/3RE4fW0KFBGW7GcQ7TI+dePCHMJXf7uF0zqJGkc7xs9xiSXDHzJf\\\\\\\/M+yq\\\\\\\/OKo1slVJQ1dsB0EfmEa1toINZZ7\\\\\\\/1zHtqOlF2tzw\\\\\\\/D7UDFtm3Qk8YdrDQpMNhADHGzeMbrimH\\\\\\\/wiOHLT8ypNvKynLas5aWsW0m+wcEcV0iKJoKQjvnJje4TowC8yWM1Vt4F5Hgf9HGM1NJyCd0mI63a+D7TOpYFVFccBos7UzNH44YaHNDXfMyrsgb6STavGGjZNvq5hY5S4AI9n6T5g7vstHYEiih4WzXVU7mggxLetqkEGtURpjT+3WF2Wm7oP9q7v47Tc6T3\\\\\\\/vSbM63JkIffV\\\\\\\/sIiAcj6+8vDi0pEAyS8wK3pXHF+AsWPNFCM7R\\\\\\\/uUC+5Hf5ehTZ6B3noCaHsU4mrYUVcLVyFIKVVxyptW3vdUnqYpLtBjMhsgTbqcWgdCHc+HutkYAroieVEyDoEHH9xdBPYcUpXSP4fHuDsSExLAM0cRDlRadNSTM9CCBCH4br9OTOIUT0qHTKfhud10hiyqnaBijHLF9LreggdYNxxSgSxZiGPhYtgCYRm3DBSms1eHLPUaFTmXhUl\\\\\\\/lAFM3m5PHQfZzv5+BEor9d8swaIDp2UzUBv7otwHPb6V+KH\\\\\\\/YUAd1gu2\\\\\\\/Pc3CiPgOelpKXATy4Ro7yuNY+eITJ+BoniwybjsReysaZRbiAbR4s\\\\\\\/SmMTJ+ACZFAWVc0bdPyOyz+UiDszEapLXDyMS00X2zDN5pEfaTFxa2RGXbegk2upD1MmjVPMRUYNrZY7hf0Y191SXXQorjtmgiz7hkdOpK5UJki3TMbvHP9WojxCNmbjNO\\\\\\\/PUKKk7OSNJbUaUptwK0AZJ5F7GARQ+wrCRUkakJBGR3lVGlVKRwxQlvsYOjMM1cBQoiOX5OvfXNbSaVVxh5RNZ32wGdTJMjjlf9B8Ep3HROXWzIUwU9vjLChn02MWFi2GjCBz7OIaO3\\\\\\\/paavDghAOTI08EGS95RvU\\\\\\\/LgXuyx4FVJwAfgPhrqpinvzzZDstA9sXr23rwvtHOwQdmug5uytYOjKQdCgGx6Ggk0OT3KlQX\\\\\\\/1UiUXeNZ1BJhbuosPFc97ta6oH8G0YmIY218wOdSv3QMDEwyc6UOuDx9SI4gR2vt4kI1bk58oFRuHUGgYI1ZdJqzYrGzaP2Wv4ilkMRBBXHjwEz0BjOWog\\\\\\\/FatE8W04cLxHQcHkVjWSlXfHJE8c04WAptuftbBeyEBRomLWfc5KfTKaUW2vCPKsMibEWILgdQIsGxbghwH+2GCbO0rREBGsFsSr2ukXNgJRvYEmhJe+o+nYJvR\\\\\\\/qhvhzWHdD3GgEhsvWdG25WjHz+CaE2g0mEAJIUlP+b\\\\\\\/E9JuA6\\\\\\\/oHRpCSJMLzXtkY\\\\\\\/KpqHg8eSUJi+2acTZ6B+fjMAcqMvyO9CpOF2olj1SWZlWtBc8jb\\\\\\\/CQt3xLF\\\\\\\/RrXbEe413EPo5yvaBHLskHrexBwkdo+IAB5PuQlz9kQqEL4CCBqh8NKogN3qyuLAhYhhb4Mxp\\\\\\\/ki5GVSVOw3OuiHtLLrn0EKoQafy5NE4EHWYbzTBNgCF5whNGOgTIUf8QkIoX5+Y06b\\\\\\\/N7L5odI1YmCknnflwZqTtts0F+gaonywdMnB70hmGH3JtUP7nFZugBMrUakRjABbtb3XCCV6Ji5UJKRdbEslzNpak1A+LMR6UmnQCjHHSkHbHCtdLHU\\\\\\\/uSR4O71fGFYOVxKmHnrV46\\\\\\\/rJNAibF6cvcam8Qy9zSpk7yyS0uARwsYQp\\\\\\\/w6volaRqQ1wemG3ohUWtksVHtyxcNjQHYDvD3khW3W3IyejZSRx28LXyEyzKJaPE\\\\\\\/DnVH4eiq+IBC0fHbpomqdJJRsBNbBzrSUz3cIHJRB6M4Gxc4xTti6raQdEH+4pJ\\\\\\\/7gzoWsCKsUwKK3PqZNlzuO1h3v8nUBqQdFzZ5p3a9jn+TCIZstLSORlrr1a9zb9SBr+SyfFHB\\\\\\\/4wy6qTB1\\\\\\\/QXxhwlJ+C8NlgiFODNY4S5lApnzSjCbBE6DnDijmK3GWt8Id0eg+BI78gPQAv4SO0ODX4FuzibbO9VQbwjgdE2pgGyvsu2m+5ukH+zY0QuG8oGVxzF6CqmySguzAUHxSKlCDY6RbmBAQCh7RVfXLoLUVxo4Z8IcomFrT2ELToiJOAnDfz0DoZC6zD\\\\\\\/dDHmO\\\\\\\/nFyiBRgTaU1wCAQX\\\\\\\/+A5R7YJ+cid3RVk\\\\\\\/du+U+d7cqI96VAVohUnKKM4Lb8PS85jL9VoPWo6MsDTabjGT\\\\\\\/ak8sucXb+FmLz7ur\\\\\\\/sWh6SI2DuGLhJlmuVFQvSs7SXn0cb1myN6jt9M6hHgL+bMRGzKbvlU3osqMCPr2cW\\\\\\\/NxxrjP\\\\\\\/KqNOrWgUg4AK5KOYlutYTeDM24ck9BlP47ECI\\\\\\\/BSaNy5QvRpKhpfuwK2OFpuG4uA3zLqWuLqGXgiF+WKbPGJpo\\\\\\\/h8Bedjvjc2potgfhhTbSyOm8Mxk7VjgClCvuFHkg7saMCGIFoJbvM+M8RFsaM0ZzoNqyRjQJQjSbt2LdqJrZYEY0dp4WvjqPX47KfWaq52b5tqQQIEiJSybQvCGwAKqvAjM51rATl0qMHj+LhAJuLTyNitokt0IwjoNN0uTG2xvZQfK7It5daf+DxJC6cNnoLPxpIoQlhT7d9ph2T12ukLqg7Z2Ve58c4IGVFyewEETdyj8ZMnrGMMD\\\\\\\/v4u3PmIskaSRmusk\\\\\\\/qVD1I5zYZwcMiZwdoFndG7AsVDwmuLOUhnCrMzzIuSihuz56t0UMLh+13XNg2CrQycYCl9qZeu0G4slJlSeH6WPYzLUNDzDXpGjt7j9BIT5ea1yK27fKT9GQh9Mjdvs4epZrXl1EYOc6VXc3OR4QwOKS9UYr0mRamUdoSmmv27fwa0rb41kxP6x27iaXBYxDLj2XJOTIUWS9xir4Yg37sJTKzs\\\\\\\/e+eV\\\\\\\/X9jzM6GA6P43cx5pwGlBY3hR1U9LE93ap3J3wZNnFOZHui9mjQKukfkUNynjPANQeGopUC\\\\\\\/SQbr0PNNZLaffHX82it+rBuN5X92ZZcfsVK2JhU8s8HlbkoNAZ1SUq4ov49FlceqQ75wjBdi5JIrW4vDl7vSckkg758BEhXOugKdKwTqCmy\\\\\\\/wU9LgRaIPeHfJ884MLkpNXrEN0RBmbrlQlSU4PBo3m\\\\\\\/d6SCMJzWsJtPV2AXDcMkMyzbmGZMjYiJoAnviAk\\\\\\\/wHNxd+DpCNOaNTKox7l4BmJFQkYp7bUjlgvOfWo\\\\\\\/4rn+cbxU4avcMW26ZHOs+wn16V7IrlgZHBVMZ3sENprp1+ocn2hg2Ce2+8sxg1FlYtn+Y\\\\\\\/WvUw4IjlurnQB+G\\\\\\\/Tn4iK4YXzWiN3jhXWpbgYKvSiy7sEVB4nJztJmIeDZVZiT0xYjTNPT1\\\\\\\/QY1FhCo5xrK40QrXexRQnqqtdIJNt7XYN2833EcCHI29DL1+YHUjKB4H91m0vDoCgePRj0cE0FnYt91dSGivjp4P0YCW1Nzqgzf9PTev6ww1DGjwlZkWT7O36aC+De+LyNv6Fqi1Ljay0SYmfNSRYo83kanFxPyuNDQ4Be91\\\\\\\/H\\\\\\\/jIpfAhU4T5Pg0d0RQNuskmzvvQcT\\\\\\\/S2\\\\\\\/dcKyfqFY0VFbBdghO6xreysxQ1aA2E8sO655R04QI9Cv2gG4G8Q39ISj3I9KoB0JJ+YIPw\\\\\\\/Nw+H3F9Cp8zrxSvK4Z4Y\\\\\\\/OyWlg7PpY9tq7ksvYKo1CNCMM\\\\\\\/iQD948Hhfo4srP+XY4JisigUPZri95Oap618fif8a\\\\\\\/gSZCyJlPhbRIR+bAqxiDPcWoenyEODvQror9KoTJjo1ieEtRpag3SXp0WJ5qu1s1+SLFykkKNnjXezBktGVYnkcgJeB51QXnND\\\\\\\/CeDwEgVh2rwfxkmAAt298dKQOoziv54n01GPsrk03wyimK2pdgLn10FeIIDU1cY\\\\\\\/P8U6ss\\\\\\\/Y0QXIhNIM7YUKNd0++MA1uql8mvWRSQRjAK2kV7QklsYAc35afhWltXhFkXlarSizflvtTW+wI2NwxRRVTaBhjHM2kVRE0jfgEMBpYAC2pG7wibbda+\\\\\\\/lzEAF7CGcymVNIy1Ja7E8Si55h87jITb+M0e2VUcJsGTTRarTlKdkwxTyxGacYp7JqHYwQYc8hasRilu9auLjaf2ufLFEd8LEQmSzdmqXvjorxumzceb3giV2qwVP0lS5B16ml\\\\\\\/FK8pg+Gcy6Wjtc6vWwkloYg0IubO2zRJHtJvQqCC2ZuWxqldsgDcVgiJLDDgZ9E2G7Qz7tM2nua1Iy5KVJSj1ARpCb+iBM+Ga\\\\\\\/YsPOKv3V6\\\\\\\/hrnXuSdtCMGlQFiJl\\\\\\\/98qw9opCwi6iui+DB2v2y8leqg7BPGve9c2L25xVtsZBSFAr3TiGJC+QjG74DlQLiec7l8VeY4V2ivdcq6ZTfpnLjve04BJCJW5tmMwKHZdlnH6+Ip\\\\\\\/BTZMBfOz\\\\\\\/nzCweYT4T9iPmEKDliqobDE6u2wQXxd7FKddUBaHU7xl2dXEHAYNoSKI4FEm\\\\\\\/pzAmTttdj7XwktRnOUr\\\\\\\/6Y9qj4U3VykEIoeXECs7TIQNxXyrYFh5XAEWtG3xX1swng3F0bQM4m6XUUGkuC36HG3b4FNHFUds\\\\\\\/96VjLML7YG1jIPfq7LJINiu3CZJ4FNugyeBvbjAwrfDjrKnFu0y6OUdLow0g6kYLNSFK+4OSXZiA+6cfBhMVX65l0DEv6FUhELaIsGuhJLGr+arcoVyHq6Uepc01ZXIgNz0py4DuRayi1mr7zIef7siEJ+JQ2k5TKYt2Jv6NH059EtHO30OpyZcN8pFQPOBRmRABdEWYOHVNuAoyAPt0XrkXXASJgFA6ake1bAiwtireGaW003viLno3ZbJFDXfHyT\\\\\\\/oivzvSoR0NNqY9D1E3leLhkU65J3r9Gxin4ahxIUWdmhoE9ClZnmsWW37HVQPRqTjtWUeLsoSPZvRjYa4MqrQWnIBZ0WDvwMiNmc3FZMc5JitC4QsuflCXJH6x7QkoKLVOez\\\\\\\/S5RzVzfGi4aQ0Y4ydLbsmL\\\\\\\/I\\\\\\\/vHBEyWTcCjeFoWOyxaJEqwAowZESipT3bV2dEJY13wNixQK8Q5i4YghYrnjE1\\\\\\\/+AMe2E8abW77kHLaAXC6AELadGjhW0l59Nh2ZZuQY+SMY3\\\\\\\/LtabLFAqOqEUG5oS13Wp\\\\\\\/LmdJssq2SRs1ihi\\\\\\\/jSVzuFuTfEpQGeniheWIESUNf2ShI\\\\\\\/AWVJWtxmZmFhLLbqj9kdFxnYwsjx97oC1Qoov7iPvrizYycXTdkQTUFSGlTpG4UNMcbsXLY3RYIU1fbOfLoCAos+9a9ieZE9NB7KGwcj+yRfrvw4QKpaEIPDratVTLMEnZf5LByIDTcOBkv9+wHLTE7XjtgwLGX\\\\\\\/B4iG56dhZp7\\\\\\\/YmSajlaI9Y7\\\\\\\/PUt6L0w5PhvLw6pdLx9OSkcvkDf8K1t7tU0io9X\\\\\\\/TSKSmdQQqGiJ5pYYcVZCcGOsE2nwCD9sA8Q4WyrnMUVNwmto+uhYr\\\\\\\/WQFV49Rkmd2Jg6Mn3D4mXTbBrUkHfZVcWYVuefJ7zCuwlFp6XfoX5flYNjVvlmgV4kO7xAdjFEt7rkFmeKiSiL5CE\\\\\\\/djERSoC23ROGLl1L1pW0B4p8lQB\\\\\\\/VJ8yCAdeFyhvfUNnSedzRZIEYlGZMCqxINDIXFD4DskLl0msLJLLp137vLIGbkDpJwO+veBkx2fNO+5w\\\\\\\/h5hOXK8zU6hyL7jQcuZ+Yn\\\\\\\/5c09PYSdU4pWcuj+ae06VKdjHBuT9wIrp7+QIU0QX0jZWY8kLeRmmXyfu9noBv6QEiRGAyjaj8\\\\\\\/STKeykORf\\\\\\\/8tMK3uT72124g2AsniIgWdiaJvSGy9W6j2mP3e12U1QqyEGuS3KKjVGSYocoQm9KV5uOXM\\\\\\\/k\\\\\\\/8DscS4eqDaWtA3y4ll6\\\\\\\/8Th7FEJMj2X4q9h+gYh22oi4d62NREpo\\\\\\\/5imoJLebCNEuoN\\\\\\\/JHXIXgPq0Ce7vEf9Tkl+UgdTMb9lMZhho7ulzwo3ePl7Zo65sEQtsljLVdZeLrNsTi2AKO3LI7iAZ0GTPONCmwslMjYbE3VjmvqPrDanOrHNK9L0B5OLZ6caCHla6AoQAITE62T+x43pn241KtFpElw0bYLWUCKQZ2XxrIih6cq3YFCNrRPSbbcuW9m1KjxZM3xAalajVzPpB20JhQs5d2enzNmbu1ODjkDzs8VRiKRc2z8oPQ3aWf8SKoxPDOl0rTdO72mggefhva+VQx0qeHIECc1dA8JHJ+c7oRH5S0VgivpAKJNDl+Ih+8Bgf5O+MSQtE6WFUcuBcHlxfpfubchK3PHXz\\\\\\\/VRm9BKr\\\\\\\/fnGEMCqn0zuYmjf4a8XppGXgj4Wiwx\\\\\\\/Zg\\\\\\\/ZNyR0rr0m5QF4UYpwcYtRLrorAQ7RB8ScV0o52jyDwi1sA66\\\\\\\/8VPiGPLd1cb+F0kmxVNbb3E4Fj7eKvAJjjYEHRyqDFoLNynTGbog5kRs7u3fxDxiiUB50ixQZw5CJnhPuS34YYCEEJlMGkY2ziJUWTjDdWew2+zPxH9la3fWHbV4AuFhg\\\\\\\/MMYUCTXhlHFykxnj+2Dybp1C7aoiB7DUaxbxM2M34wcI7FFsn8AdsfQ3l\\\\\\\/wFJW+KU9JfXr5fN62AvRK\\\\\\\/Yn99p+NhsaRJeKtQbSD4M18nmiZrBZ\\\\\\\/FH9zW46naAjV7XApZD8iQeBRg5uIaQ+KWXEePkn8b5LkIMkGxiBo2\\\\\\\/Iq0+fa7ze5U51\\\\\\\/7wCsH2Kn6EOmB4HohRPx9xXfrY3RFW9plZRgT\\\\\\\/KG\\\\\\\/5wtI1oGD9aD+1aQmlawSmZnVqSiETlm9gpqzZ6lNynNoZJHEYH0D+VXT6cMFJn+oE8XX1Zhc8mxaKknjXIL1DoJLhZlGDiU5hmM9QFK4TY3gjKU9kiPJ2OaEVCXO7sDK7GCgXXJi6OtVUfOSoHroM6kvuDuIQ+nI91sEsb8U4gPrSGvQx5iJC\\\\\\\/uxERAXxf9Eic2vjRZcR5lB4Q6b4U5vOC6ENZlwd+UTZwjM8F9LfsV9n8iGJPENYT7ncRigZGO0H0iwM2\\\\\\\/diBWWuuzaJ3EfF1ybNOme5XsBATx9tUEg16wYDjh69HV7w3g4jv+jL5FwB5XK6afm\\\\\\\/jFzt30AEwJV7gjDWCrx77hQcbR0gMsxuHUvgFMYMgSvhuFpu33gyIAAUKU0xBtf8xMRV3WnHGpD6pgW49RIcn58\\\\\\\/hZwCpce8ZuiMoSj\\\\\\\/ivQINFjVceDgxhfdauxxUpyhgxvyb8z3OIGJBJzph1gXqm+63EixZOVL4cSAlqjQYP1xGtiu+Cl99Fo79rk4NlW2X7uvxOYLO8fLL96Kd9k2oe57PZb4w2vePkuPUKlaERhXvtHTVFMQwP24pR8aVvtFjz5o95RUXkKzfKNG15jLeTdQZexxbu5oxxFN2IOOaH8GmR1SdI6407cyG3dczx6+5fRpl2r0LdBLVdyGc+1yRrIo7ZupBKzNdWk1yPfj2PjrhnJqCN5og\\\\\\\/uRgf1T7qTNU2l\\\\\\\/r7ogGH+osqWHtQYXk6ZnLUtEM\\\\\\\/2PiStK7fnLOyg\\\\\\\/NIHFb4QumfCU0celSIR8weMez+b5kmH3xgjnLpYkOJwS96kzZnDjtPDUaLiIGy6VGyk8NMxEKUiuhplWOaVA6zAYwv1Cb0DHNPHDgNIz+20NH5l0Ez+SAwY6gwwfw\\\\\\\/LEYE1eFLxRqwRREfyjp3SyyvWpO2p365QXauKPYquBFcjlAaWjV\\\\\\\/gNqhc9AXx5NJBovvsw5k5ZHqzgrJfDhBxC5zAEmY1I5v2PZVIJDSQ7RmkAvtfS1O8eqJLju8n9OLsBMXkcVwXH9NHU6VclM+FFD42jyIfNOTkGzwlJLiWlkCfQqdgLh+U9GXUAWl9vrSBonBAhQIqu\\\\\\\/75SdFwzV5la2wMGvTSoj4rLf+M6wS8N24EZzEO0uUK0qKJKxwS9+Oy71pQrjf9LpoGPU0eJzTTOyrgLt0iOYtxhVE8LyR8R43m4E\\\\\\\/+0EdLuf\\\\\\\/arqTahLhm5MLBPqw0jekaz+Is8KLDX1jnRsfSa8JwFkK1jLNz5WI7DjvObQRKQAf8fr9kQDzj6fhxslzebBDZGU76LdKM7kN5gyOVtwfv0bGyvEgC8c4TFRhNdVf7VSmSDITixghMt2SY2cAva8sFfyIF\\\\\\\/\\\\\\\/wjZjkZGk7dqGGhE3ujxGbSV53LWhmt86sZWY0K6gol1IWW4RcSnvKlPkQeD3mKc+kDPczJgDxpyzjc+tnYafQL6iV2C5xgi0WSSGKLIAfhvGfbc7at67bVs++iEzQPBlVXv6CI6S0ZDpxj3WOYEY\\\\\\\/cwG+2Yc7xC2WRf+SoADwxRBoHjcmeIf4K+EFAG0XUkppGFD+bgnq2Gt5dFcj3G+ABHcCKTBTdqAjBRAnoIajWHZMCe6wviN4nKBOPYjU9kO+xSLahhHwhq+biwBAc0t8jWyTbURkSKKCp3Mwhh0I5oGyOZAMBzcq2FpxjweefxsQ2dyT24PUS+QK\\\\\\\/55yobxiaKVWIZrJoZgySzjZCGTEkKGJ1HXguH2D+TfioQhXFsIswHP8UKt6M1qDtzSA6S8eLWqSDjXeSREM4Np7k89tHnxtFAS5\\\\\\\/C5k5n0rGZeLTq3HRandydze+Jz0szXERdSRnUoDjinxJkvkCLtVobbDx\\\\\\\/ZNIL9\\\\\\\/OriCXF1envNCk0Zalguq7TPHS\\\\\\\/172rSN9YSuuAxUMavJUhjWvkdp8Z+MQCzPj7azYs9LuIgIGL5wF\\\\\\\/nqdHOW9T\\\\\\\/r+zOfLn67kaDH7TJNppb0XwfcSPoiCjDzw7dA7PwvHYVie2HpVhByXg84uja7eXRXN3AVczTIM2NGYqrvuxC3wcetEhqNr2kSaqA8ScdL5ndTmZcHpqUvqIz\\\\\\\/Yffwz+D3i\\\\\\\/157RQRiKbevcpgoNNujn9Z0+97sfJ7SraOP9zAyu2aKdLAQOUZJy4HkWZg+Un86JtnkV0be0h5lO4cV+kn\\\\\\\/NV+V4u4FJzbTz2ykDVrnjGjdKgzxQGHuS2jFQ+T7uDbk7H4d2j7R\\\\\\\/DJaGa4WU\\\\\\\/lYl+1PmwNiZskgUL5hFiNXdAIICaedvU\\\\\\\/r0xbGteH5acZUbSXDzVwgxUGiShEp4YaNe\\\\\\\/SWz9cGtZPKplkUT4WwxqKmWG3LCLfr1gujoy\\\\\\\/NuFkvSe9qFB4zpKmC00Ll7bk6BhirE9o4SRidOG3aF5Op3Xx1nWcGySs2RWCAXuxoGV7tV7zosyRhP\\\\\\\/z7DFHOFKi+u\\\\\\\/ApvtvvcuYViaNOcjoU8jOmLG7gCfuulHmhdplvaacV0Nidkae4f2I1wm4sEdIrKoHK\\\\\\\/Lgtz0MwU\\\\\\\/XLU3Nh7PR+7ICc\\\\\\\/FX+9s+AfH\\\\\\\/RC53wIMkT\\\\\\\/XmpnweWtcExMFBDr2XWBSW+N7XpwbfhvmqRecZx+kN3I6ZdjDR0kVCw4nMkO3fZbTpdJY\\\\\\\/wQ78TNJq9sbkQPRxBvhDWNBIklLQ+auOayXPEgYBUaVq7Wk0srYW1jiwDlXSDHSXfETOlhaKP5CqLAq08eW0dAlrBH+s6RKs8ggjRxVR3FObZMckvoaOxa3fiq4KUcqpsmg+XagxPCN22uTT0GmerLN2PjWTABwNWqJpQdH2yKngE0OEZr1nGhg94MlWmnHXrts1MlgfxFjSmxF+BgUyVAcVJ7J2dJwU5Hufn95FwgBSX1fwnkEorzL\\\\\\\/eLB8qvlxdktuO6POzXrPGpHnqMUhsTmtGvDGbE8WPPdQ8CRbnwOtzOEBnlyocqWOryAtpk4NMmAf9OvAFKosbE\\\\\\\/X9DAetLHW1yRFUbITRI3e4w5x4OLcQy51fAKDoBp2JHrj9GwM25aiq4M7hh2nEl5I8HdN8D59QQhLkcRbPL7TynuNU2aFArVOb84MapdgJC0n6XOhS6KRFkwM1VHIGFEtPQ\\\\\\\/\\\\\\\/iyozCZQjShVxEEHvWaklsPu46Qebn6\\\\\\\/Ct1FOmShpG+NtobTSAvMNYImClFuBXrBva\\\\\\\/oyYHXmG9ZGpgiktOQ7C6HWS+Af9sF0Y5cAbwOS6VqYLCyzexFTupOnrgVT44MWLAjQtJvYskQCnkMlZ3B7+XATH0PuUO6pD8heL0y5EfKvi7cbpWdZSDSv7IaU9ldwp8fNj4ZKYPULRdMO6JyNL8Gk8oz52RGqede0G4Mi8iGwl7Zk1HoH9iAoZRTBbu1769mViRcmpNMow3MHeuOOkYTnpNey0YX73EZZwsw4Drcd6C63dIMSPkQbFRyNdChDA\\\\\\\/EtmP7MJB\\\\\\\/rhtz9+L9MeQHRcwFjdf4YkukKc9qESgy8Gte38L8KTQPO4pV0JM2axAAGEO2+h\\\\\\\/acQJMmUCZyY9Qf10GGQvPjYZ7DAbDr\\\\\\\/Le93Q3qQ0H3N92r12WY03Iel5O5+KjAWWEoZfnzPLDvX5alifQ4uFxjCffLfGgfv+rGY4AnhBcrmhTwghymrY61BhK2UfOil4nUpgFu6VYdBgDXqpCslPXPMXAtxQVw+BjO0cRlWEMFYIgbTvzdTVowjxR+CADAjEBZSvHlos4npam8fzRhwvefuDhbJwSQ1dmGmHQ+x1KKjf+KaiqFdG9bPONt8Vybl1dRtUaw+WzrU5Cn8r9w3tCuKiok1C\\\\\\\/8+xIsSMcfv2D3wr4HbVq7WVNaEAGX7UrUB4ndygAoql7Uh3kwQC95cFCaAg1jmAo47qUn9ZIDisRJBQtoQOy1wUknRP4UYa+AQSTvOgSYqd+VJN\\\\\\\/DpSSFiY3VXrs+6RTp7g5Equ3EPi5P75kkOiUX0wI7oKKPY\\\\\\\/tZyx3aMYv0zm8EFMaYpETWXMDXokqa2fm9bIoWLGfaMIw5tBPGfYZ15MAJBhy2CGXyEh\\\\\\\/sof1O4LtCw98S\\\\\\\/2w6ZceW29+TPDOwP8VCuZp6h55ugtnm4YpUN878fTFk+wnazDHQDzQOzgVrPRSJlqtcC8zz3TM4zNOH0XC8t04Whqm\\\\\\\/JPCrdj69nUJc99txjCb8ydIOFGoNGKZpOHb34e8vAd16J5Kx5S7IhqFtrgFwzK1YiPW3hiiQo4dRwB5hkC0+0lGMyu1Dv5K6QkSkB8yFR5i13zUn+YjtgQUQsofAh1rsKCw5TG9BiJGdVethMryuWcuT2oU\\\\\\\/E57fq6M3ebzVt4YgvOvjyC\\\\\\\/HLM1ILeptB5aNkrOUjyVya3l4NzXyQoZjr66zZdoZ010qa\\\\\\\/ixofa0BQKCh2u7ZSU5CD+xTholPhhSKXueq5K119K\\\\\\\/N3HmI4\\\\\\\/HD0ECqyRJ4vnFK4CtFraz7PIg8+6Xm0QhgM1Uj6IjFieKVjpQe3h79I8Vi6q6tGyaSs8mGgaQwTshG5r2SkMitliu29MzyWW2r6L7AA1YKc+RSFg46i\\\\\\\/pAsMJTC2DWZdStETE1e8hPhi2nKWwT9QiBm133\\\\\\\/GjP6Fu5EyM6ESw+Ffmg7iyUOhcOPDYSUfsqPqAD0vg\\\\\\\/Do\\\\\\\/Lia6nUmkdCQn0yhM2otsNyCe0NP2JCfLD\\\\\\\/4wpYjEYr82DOfN48vIRM9XWwlZpu7dNJ4tPd+0dSPTzjGsifon9Idwpgn7DiE7YdIRZytBBpa0loN5B8QvmJ6CVCze64MjqmG4m9keH9H59wXbS9HANRbaSPKqZUenXvlARch8LHXM3+FY3h\\\\\\\/+z60gz\\\\\\\/j1bXCDa2Wkx3Sxe3k28cB\\\\\\\/TWtQx8NMY7Qc9ixWeM9CMRFB\\\\\\\/YAg1kYVkZdMml99rjY3SN0KGvT+JEccupPhbvxh3NdhZdBcS8A5Ly4o5E108cgeu\\\\\\\/Ol4pkKw3P2q6mJ58+FLnFwgYdMuNn9lUmcaecApQA27IfTGWEGZIsAztdf8paIlqkpVmjiK\\\\\\\/X1f1MoHlwTeRriecS+CEoovd0vhA9prS8Rne4ivrlzWR6hHYg4A8b4a7KnxB8jX46Co+P0IviwMvKRwAK7bMui5fJ9p4nr7IqVWwoTepcM+0FiozMaffJ13yFoTR\\\\\\\/2oMtLWekWORg59P+d5ePhX4JmlRmrKAxnWC1TDYJD5ColX9JMXmABcTRooLCqyZ\\\\\\\/v1vc6PGagdrE2hmVhseM3QQIJDOt0N1OuagXhhefLlzbsexpLWsQ1px\\\\\\\/TxBrkZDQOOM8Df2kY+7K83HgSptZ4yvy22yoyS5G40jtrDcApELUUhoQz+D2WE1a9l6TggEVyTmfLKx6iGyFU6pUl3N6IukvIsQ\\\\\\\/EHHDsFTupSWdi4gwuq78a3l1O9QfCHAwddAxj6LqP3jZIBREYWS8KsVALikYZ2oJaWeeefCPw0mhCiaK32kCCQUfwy2j1ycbhcjI2nN3ORw06aunpkShnr8EylCYm2\\\\\\\/+a\\\\\\\/kTeJI3DwukrFJlbrwwej\\\\\\\/VMLdRFvEMUh\\\\\\\/bqjQG5jJ+uFfphelGOgYzcCDf7BqJjiVt1FsziuSXMoDVhFG0FBAJUZSFGIP40JJfgEB83hiwjMiZr\\\\\\\/YNoi+gFwVF+ca\\\\\\\/dPPQn3karIjbztwXdkfurpFqzHWSRCiG5dyJeFsZhSa4TJCYUXtAGquVO6xyDNzVg4Dnb6iAbzSsUGCRB+VcpwPvSz1V\\\\\\\/jD9RhACnk0f5ETe0aGHhqiwvw7FVRXgovGxviAQ+J3ZTm4qx0Awi\\\\\\\/Ukv97Vsr7K+bXakOlWHJCAyXFYMojIHKAZAVsDH86OK3nRoHdN8mIxzZ\\\\\\\/05CJzJCSJtTX\\\\\\\/D865Dw6yJkogWyuGh9E1qNqdToagA+jCROYNsrLBaAuNglx13EoiFPBzFuJuEAr1v14mw57XR1kO3fJS\\\\\\\/uRVaudbVo3+PbswbvRyNU7LqW\\\\\\\/Px4QKUvIPAWS2zX8HrelSycyMhWMr8k3FfktL\\\\\\\/UvvpE1P9r81Y2UslvxZaviJ7HwcvEySNx0tLRaG+hm1GFwOoaVGdQy0CMnjTbFyqbvcqe8cTWoSeEAZHs3dBHTB\\\\\\\/TUgLBMtPbxY3AZfphlEbgs9y5WjAcHDYcmP6PhW9B75H7xzkWjjTRMThIj6SoB0Tugd8\\\\\\\/nSYRl4avm2gXz+LR2xRYUQ0iFFwaqsK+vHeqqwf+tkd2fpBbtgnIheaW5A7af7C4jlYTDiKPtjWD4l9AjEV3qTx9HghIKtaPgabrPhgO8SyfPXQCFMd8rt4zrAtbKuBihakiuatyPzp8h5K2kwackllEnyWmDxpIA7YrAv+pcn0t7awHvJjowvSVThWnTlj7KY6rlHvr70ONpF6e4AKMj7CpgJkKRBwaHiov7dUrCEsQHP7msUtdIaF24Fy9adh93YLSNLXQz2d1M7RP6+7hH8bq7zTtWdlfx7+q6v9tX0gRBpeqAnNa1PHLrdZCRRHkpF5eP6f2JgdF+hZ40QA7G2uDKQ0c\\\\\\\/u4SbT0BzylQj84ahMBVwYRA8bcQSrjhRaf88M0O4X5+6uU1\\\\\\\/3Za+dXF4esqzc4gGsOxZwQbpghci4q\\\\\\\/6H4BspA00XFIDOSkJ1\\\\\\\/C4S+dA8K87hg2cPo50Wwbsx5\\\\\\\/YKgVzEu2G9EMXGp7LmqsK4453RXcji5jKpLUm38w03ZE1VooCJMTjtIo+9aM6CjK7RaOxr\\\\\\\/yMIYEbpxQq8wzuWFx12Bbj9R4vnmszoqfVKgVMrY5l4LXLbOSxduHmVpJ69blKWUA\\\\\\\/Flm0jzJDwxKD\\\\\\\/8JcYXJqumzJwqnKajQp3dVanoF3IkTLeBGCSZ4GobD6+lKgscMySe5CvrqNLGM2Ap51wJu\\\\\\\/PGjehpUTbzG4BOLTgTV6JCZg6X2NJ\\\\\\\/eiV\\\\\\\/c1hgZ0uK+QaGKGmwW+rNIX1WnYq9VgvZaOuVX9zEexK4K+0w9bwkot6gQqlzgBk8PF2F1RyegzFUaOfTS2bxmMgbctkWL+vMyGLcJkAkpcba\\\\\\\/MbaLlZHCOHbY18LJCtK\\\\\\\/g3EnPzDlmhHmh9rJRinJV+7\\\\\\\/z4C7vmByDEbc4tA\\\\\\\/OpUghV9Lyan5W9\\\\\\\/pVkm7BeKxqKIegUE0jopxGnNKeKUnDsHxt5mJfCkfSyg79kclUsgvpBuLnpkuLRuQ6jDLSgCeGPXH6KI6n9f5ru\\\\\\\/TIdstvQqcqK74GhmeCwB1joCdrDSLNdRixtFXLUwrEmbkqw9fTLi9ix37cr5W4O3FGsFIcRFBRiVZa5Te\\\\\\\/m9Ql6Gd3AHFLbb9HHeiaZUCW7Sum7lteC\\\\\\\/hTmLxehzlhhcdX1NYGUSAtGHZjU5732Eqvq2uQF+a1itfDyAEQ6y7rp9lvsd0uUsOKZ3IicuE1Hggq\\\\\\\/GeFjIWmHomRuh34KfI+d6pmDhtilNAwIqRrUvWBXvRnMH+TF\\\\\\\/58F3p4T0O+q21k6Y4KTVYOY+A7\\\\\\\/B1jcs8Z7JjtvFUXl\\\\\\\/ZvA8MkjMpizJEp8S8AGpmwuDH73Fg41ShozalYIFV7LfYxW636t+8xNfzdMffd2Uwi6x9yZSF7KjAKoK\\\\\\\/xsI\\\\\\\/xeP3HiIh+N7XzL6dPvIT3gjOTbtJ9VdlytJ0jxKPFKpzLeXdGKUcn4NWz\\\\\\\/Km4tuFPPMm5MypLh9i4aEbwS4SKqygdA5f+pqMcC1VK68RYNU3baV+pZ9xfz6Ynm14o45bb4bPEJAkTJ364nDwwmpjmsLlVgNOyEDu4PAmK5oJVIjZrF0JUcH7djKQCt\\\\\\\/1tpej32aCfWX6XMJvwpd4aEuI3LeUSd51NKFEIC4Csl9rZlTmO4jEamoDqR1+h4\\\\\\\/f0CiPbdJKFw178O1lK9K0zUA0HLUEWb2w6j6B1gJbd9+BKjyfzVDy8kE9R9JTa+v\\\\\\\/lFtOdEUKBB3Wt4GSiPIVc4osMsosAkD3HA\\\\\\\/0Is3zun\\\\\\\/1uhTFGDkak3f1+lqbFdg\\\\\\\/QiExOWkYpZdP1lqQZMKgptoNH+kw\\\\\\\/xcMGc5gmv4imYRcD9dbn8gHon8KDuAgVR2TKEfd5IiKYvhIfJcUPdpjC9eOf6WzXZiZdvKPWIMfAYzU6ZqI8oxPuoRkJ1R+gGqksdSQKRht4UiDaUJ+u3PWgYGrU+Z9OBkEx\\\\\\\/9iCxzodosFOvuHt\\\\\\\/yismU8darp4rLi8MOo+XxNP8BEDs9EpCMhThBtiMh0QTI3dXR95M00GUW0NEdWK7CgU+13dti918\\\\\\\/uNYQrTn1mQka+ZR7NThKL4Zja4ofIPkeSolokLo8z\\\\\\\/rdKV0100aS+my7FJnT68uWl0vaezPPcmr40y1h+bl0jke7FyosnCpqgELg3qSvAl5KZsu2wATOMynATfz1lVAlyusEDXDkRbepVzwVhIu9cNrKD9nE\\\\\\\/3SwRhHHjRSY0WMRNFkUbeigbCG8QqvgUc2HuesnZREu3Q7quwKi8iA7Ft11rfd+FxPcgHz8ODgamoZtGNb7FIYQkAeEppjQTyd86YPSvKh5NvipYvBsG4OLEw\\\\\\\/EP4AaXs6DPjLAg4QxW3OYHCWcHDPo5uvYZe5gCSXciF8WEiotB6MM6KQkGPdsysYGxypRZFrDQaCsdhZdng+\\\\\\\/OxXWzelhk\\\\\\\/VIYXPm1TBLI4ZNNawKfAyItDoLo4aJAJziZvriD7o9Y84LW8qC2iGe7F4kVJkKe4fq4PoT5b41aWTFExCJNH96p7D0eVFpG2uOsF54wWLZMDyOlFDR3zsZZsbJZrBDGCyZXWpV78nWVzW1M4dIM11P4MhTvVpnIwKgosWDDyiIpoq1VCNjYr60sprtaBv7VeIDmFQCWXtRAiDw\\\\\\\/JA3xMwQLVfjHwZUU9117Dy6iEkaCEBUHhhgS5fOvH7YRq+u2F3w3Lios9pgF1GDYjgWfYZkVS3kfFamcG4GKAJqt9ahNq\\\\\\\/HT3EzglaXLjzTij+9+8SfWUMG0kw57\\\\\\\/3TPZ2Rm0Sb4BOuIFu5EW8UKqOioasVhRD3ghOw98g5ujLefgllPWuhQ16jusIi4amnsIefq6QVRJHbj6\\\\\\\/hZxKlGhe1bA7phk0+0J27ZLuM+gF6mr8Lsr2kUF41S1uZtW+IP0ljkYlzvY6dTLJwwwRkIF6REviW0Bq1Bby3Zq1WOxt7eEGDhbiIcaMYVk6sKtahSbCrIa83MPrJ\\\\\\\/ouHCkBavXEYgUY8zChybONxb6v6mHm8T0Jtuynd\\\\\\\/xi6Q9P3WzWTXrjVsxzqpGK5xJ4TTUMosc2+hNeVmN2vMuExZiJCyaMW0xm+G+O8oq09ga3StHwSOH+p8akILMgJeCHZeHafnYF+QaA+Cfuey\\\\\\\/IKDMjh1ZL7mB8S3gB0AnQWJ3HqQiXX6NzFJ9r8P+\\\\\\\/RCwBSiYzAWXO1sobiUxmgGRpkp+qIG52Y+\\\\\\\/yRtgH7DwB0rS3nt00kl8wDn9EY3jbzyvADBahAFNSzDWKWNY9KZOLD1aqaL\\\\\\\/Al7rrse2j4cE90sE5qN7JOfZdyXcYGYAlIBn0iXn6epQ\\\\\\\/nNcSfEW0uY9hULR6AiCgPM+m6TOksSD2VO6AQ6XZyIQbcJ0kUdCTXn94dnFoq93f8Iq7+ndF7RSTf8lBEJ0lNASV6GNSxypJqxS1lsPK\\\\\\\/e7TK1TFLnXtAFoe+CVFMDtzEl4tN2oSwKs86rGerpRltZpyDa2ZDf7bhED0DwmGer9kRKQ8YDjnwkBEKSMEuiaaZwudnEIAI13hYIi\\\\\\\/1L5eiXGuIlORlTpxpzx1rIg5cEElJHyI9II0cjzD2Lie01Cr0BHcg\\\\\\\/ba4acHiuQ4cMhqlmErRgJubUVDmBXssfN5G2vTSFw4f1ekZwYQ7Xs1YwMyXcjD5oio1afmlUzhgJzRHosDpluB8MwoIgUhuK9M1f4p+KL2rE80ITY+Rk6zmSKWjnUoYHLbc3Qh5T7Kyzx4aCJhh+Jtq49SHXjLY\\\\\\\/5Wm+ntWq8N\\\\\\\/0AQOA+e69KtgDzJxpu8+V2JnmSNLVEMAZ+W5WRCJ71NeLHIcFinvYhcScyLNj8lfAn1SHdkcbtZNhLcQ\\\\\\\/fExIpdvI7lnmzN5Q2D6eoZUWPb9QQ8R\\\\\\\/g4JtXGXDdYCpDyZa4ESEzTxv96kmnAfbown7aX+NEwr6HxXAhGs7UPUBns8vlz8bWPyqzV84MbvazS5o9teGWCRDcNuP5n0EtpXm1egc64dw9QY3XoFaT7tcVY+cf1\\\\\\\/BDAlybcslboo9ZKCs7VfWXI0oufHZero4SLsoEZtjqVTWcoEcawDarErW\\\\\\\/bOm\\\\\\\/+L4Az7Pvj\\\\\\\/lWXWmRQFXJR5PPxp5ADYfAJp9HOE5XBywTx5S9ncDf7nIMRul+bTL0ZZ6WUnXIv7rAGZ1xHVHh5dI5Sv99Wo80nSdUNv0TeqeUbiLEPeKQBJFWfC7PMEa+zQtaHs2SYM7FiVYRj2Egh7y\\\\\\\/ZNURL7cGM2SUvz\\\\\\\/N2DvFvlkFZCJIlQSkPUPOXQG26GQxfRQUsy31OA1mPojN8xtekH+7l58\\\\\\\/IBZLExkoBcZUxmkl9lWoIBACiPmyKcsf7InNxMqKRwhLcMfQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022OFqXNF+AI3qoPa4ZVETPng==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e4a2a6088b07ead6571129e309287e93d767b81109a71f245cb569d4c1ec44ef\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jDrkj06LsufU4ojVZ49NILvVFGk8G5wVwMtKnwwpFwdcfnrf+gP9ZS5XQsXGKd9LE84qvCsFaYQ6IUAFjcZp5wklkUxlg4AJM+9oV7QxKlohGJQKE0cBEP2itqECM4Z4GMWf7Rs8Bll\\\\\\\/M4hyGmzJ0Sz2UG7Tmffljv531jnfCAGJutQYTNaOKLZwyadcxhgAwsD\\\\\\\/GbiPOcQuU\\\\\\\/+XabhOaWmfsnZAl56xHlLaAotVkXKrcNgvcGZnDq3r4ilQUf49PvUtKwJ9yfidWcelvFj+Kmp2XJVar8xxpR6+FZ2Npw8akN2NI9SLQ+OyOr03y7\\\\\\\/\\\\\\\/xZT8IAEvYl87wy0XK+NRxlZcaAtnwLMVfAomBIZITqbUG3hIN3TjVjQ5QwF+9WJ1kC70JHA6ILkYV2h7dIZggyi3mktW4kbH3BMW37IoJ+VjxDPLi31eyxWtFygDAaJD\\\\\\\/T3q9lnRdljX5C\\\\\\\/erLvdkLZS15zIbDMc11SMVB4ZtejcC6XuDxljlSrkyU+HNOcl+OOhjBv9MbFN3OrG8ZYiorz3YkNKv1XkHjIq3Fhtru14BSzFK36Cvf8bAgr5J0JGWSgbsPyAAdNqT0qNWFnaCCzhpRFf7PPq7YdSVVdOb\\\\\\\/+Xe6lzLc6H5S5pyoDrQdhML67uqdS+rtjosDCQgwX7nJ6vmgX8xhP1EXZ2Z03kobWCfnM09VXpRvq3tKw1cLAa58stKfoiOjdcO59KW8\\\\\\\/rzkhUijkVD\\\\\\\/e42dnJHdsyAibl1XfJ02IA\\\\\\\/tXIHbXseJb3U89M8Vzi25V8y5Y3hYPT2TgUo1zFnGJYVTV71UiwKGOhZs5VYgmo57SXOXc652QHACFGH6T5RaEzgFTlJXiV1tWq27C542S0YT9d82FBjgtrwh3MX+4HyTKIN30f9TZEHc1z+VMgQmO4D6JWJuSfTiiQZohq0grvS\\\\\\\/OCCg0Tp3FzpwElUaDXMKNUm3k1\\\\\\\/EFPH7ILOL1zPZ86ZiO\\\\\\\/Dv4gK\\\\\\\/jHr5Kfac7\\\\\\\/rhJbmlLdNwqkNVhkjlpkVDGmLqwYyDzlbFccssW2bECF+0T1gE6q+qElcNm92XQ8zFQJJonljDE\\\\\\\/fOS69D1kxP9LrKjXtC0iJu7rSVmHB5RHu5Jop\\\\\\\/2TTEL6zmuSJLjP3Bcj7fcLuG\\\\\\\/YERN9zJXbF0ADbd+4YiRLeLcJw9Dh2ENHLOvZU5anNFX8LdFD26qhkI1nF4y1w++ie8VLARcY2VcQoLcZhdVGfm10yKt+9M2WnRkFRKjJ1G6NkCm2mAxCM9QroCZMe8WOB3VjrGCS6pnANbI2dTGpz7WgLt3uSIcrzV9Usq5pylARKl5AfZvSWMhM7JZolZru9NX+ZTuz0NxDRzrE3+NFm0FPjaCkSafbHq7VpxEEmIrhnqDlXY4R4NLHqTAF4mf3LKa8g1iA3HKpm+gscQ8SA92KbiiaM5KQgPhFeqs9uR+PqgVPoKGCbi0TP6bN28j++4cYg+aTelBCGJwDhFBHGjE47t1vOUDupmN7DCGNcJqRnmZnfE2r4g5p5RYpSEL\\\\\\\/5k0jmo2jRxP8LntR2HsEcL847\\\\\\\/Nr8itjjXGlacWKErn02bK3QsuCZ9dgXjXC1qE4E9nrxPe\\\\\\\/J6CaG2eh1kpdsf5wI6uY+T5x59mwEbHePxA7HEKgGGz4vahPcdv2WaPGCr0mGMM\\\\\\\/Wv1SDY0rXmhD+kRde1Qua0+CDA+EYksOTKLfBqjhtAPKjgnND1dQpA9tpYOKTkfMgsAQ0t4lwwPut\\\\\\\/lYKq2lfxS31AxfR7PRe+VxRXnUcwmbnzYgbbMiu1BDva6IZ36O7T+Au9PLkVQBsZ4AgwEVwHK5tQaHBWtR9eTnAWmzp8sXFqlVdQ23qzKeZzeAXFHGSLuryzbjz1ppmYZwK7OrVxzU3mNG+5013UDoe2cfat+rBRpXvGDEPS7fuGpubgjEhmQBrDi\\\\\\\/\\\\\\\/m76nqVngIWm3Ta63AsvICRhHuX8M3ZKOkwJgcIhlsWDkxYfN9rm4T9zT3hiQ5OR3LgUDqB+O846AgO\\\\\\\/rZRa59ZHeA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022RtX+Y81G7c0Qqv30Pv5ykQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e963e24f7a586f8529e419475ca0fdc17f1242fed2cca6cf04b31540a5c8bd6f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-486780 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3509" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7HzdVXhONN-85x4vj29QFJUmfEqtCgTtoUa24wqfqtU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3509" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-i10IjBZq4xan3_K-95Esq4-PPRT23VKwc52OD4IgjZU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/10/3491" title="Genome-Wide Association Mapping of Stem Rust Resistance in Hordeum vulgare subsp. spontaneum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/10/3521" title="Overexpression of miRNA-9 Generates Muscle Hypercontraction Through Translational Repression of Troponin-T in Drosophila melanogaster Indirect Flight Muscles" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486780" id="node486780" data-pisa="ggg;7/10/3509" data-pisa-master="ggg;g3.117.300149" data-apath="/ggg/7/10/3509.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Bimolecular Fluorescence Complementation Tool for Identification of Protein-Protein Interactions in <em>Candida albicans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ana Subotić</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5215-0922" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Erwin Swinnen</span>, <span class="highwire-citation-author" data-delta="2">Liesbeth Demuyser</span>, <span class="highwire-citation-author" data-delta="3">Herlinde De Keersmaecker</span>, <span class="highwire-citation-author" data-delta="4">Hideaki Mizuno</span>, <span class="highwire-citation-author" data-delta="5">Hélène Tournu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-1542-897X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick Van Dijck</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3509-3520;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300149 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486780" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ana Subotić </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BSuboti%C4%87%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Suboti%C4%87%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAna%2BSuboti%25C4%2587%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Erwin Swinnen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Functional Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erwin%2BSwinnen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Swinnen%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErwin%2BSwinnen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5215-0922" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Erwin Swinnen</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Liesbeth Demuyser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liesbeth%2BDemuyser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Demuyser%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALiesbeth%2BDemuyser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Herlinde De Keersmaecker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Laboratory of Biomolecular Network Dynamics, Biochemistry, Molecular and Structural Biology Section, Department of Chemistry, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Herlinde%2BDe%2BKeersmaecker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De%20Keersmaecker%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHerlinde%2BDe%2BKeersmaecker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hideaki Mizuno </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Laboratory of Biomolecular Network Dynamics, Biochemistry, Molecular and Structural Biology Section, Department of Chemistry, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hideaki%2BMizuno%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mizuno%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHideaki%2BMizuno%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Hélène Tournu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H%C3%A9l%C3%A8ne%2BTournu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tournu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH%25C3%25A9l%25C3%25A8ne%2BTournu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Patrick Van Dijck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BVan%2BDijck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Dijck%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BVan%2BDijck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1542-897X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick Van Dijck</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:patrick.vandijck@kuleuven.vib.be" class="" data-icon-position="" data-hide-link-title="0">patrick.vandijck@kuleuven.vib.be</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/10/3509" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:486780" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:486780/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3509.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:486780" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:486780/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3509.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:486780" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:486780/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3509.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:486780" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:486780/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/10/3509.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:486780" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:486780/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Investigation of protein-protein interactions (PPI) in <em>Candida albicans</em> is essential for understanding the regulation of the signal transduction network that triggers its pathogenic lifestyle. Unique features of <em>C. albicans</em>, such as its alternative codon usage and incomplete meiosis, have enforced the optimization of standard genetic methods as well as development of novel approaches. Since the existing methods for detection of PPI are limited for direct visualization of the interacting complex <em>in vivo</em>, we have established a bimolecular fluorescence complementation (BiFC) assay in <em>C. albicans</em>, a powerful technique for studying PPI. We have developed an optimized set of plasmids that allows for N- and C-terminal tagging of proteins with split yeast-enhanced monomeric Venus fragments, so that all eight combinations of fusion orientations can be analyzed. With the use of our BiFC assay we demonstrate three interaction complexes <em>in vivo</em>, which were also confirmed by two-hybrid analysis. Our <em>Candida</em>-optimized BiFC assay represents a useful molecular tool for PPI studies and shows great promise in expanding our knowledge of molecular mechanisms of protein functions.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/bifc" class="hw-term hw-article-keyword hw-article-keyword-bifc" rel="nofollow">BiFC</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/candida-albicans" class="hw-term hw-article-keyword hw-article-keyword-candida-albicans" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Candida albicans</em></a></li><li class="kwd"><a href="/keyword/protein-protein-interactions" class="hw-term hw-article-keyword hw-article-keyword-protein-protein-interactions" rel="nofollow">protein-protein interactions</a></li></ul><p id="p-4"><em>Candida albicans</em> is the most prevalent opportunistic fungal pathogen in the human population. As a commensal organism, it is part of the normal human microbiota but can also lead to infections, ranging from superficial to systemic and often lethal (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Poulain 2015</a>). Unlike the model yeast <em>Saccharomyces cerevisiae</em>, <em>C. albicans</em> has an alternative genetic code and incomplete sexual cycle, which hinder the classical genetic approaches (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Santos and Tuite 1995</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Noble and Johnson 2007</a>). Hence, investigation of pathogenesis-related underlying processes, which are mediated by protein-protein interactions (PPI), is restricted to a limited number of genetic tools available in <em>C. albicans</em>. Two current methods suitable for detecting binary interactions are the <em>Candida</em> two-hybrid system (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Stynen <em>et al.</em> 2010</a>) and the vesicle capture interaction assay (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Boysen <em>et al.</em> 2009</a>). However, both genetic methods require specific localization of the proteins of interest in the cell. Another alternative approach described for detection of PPI in <em>C. albicans</em> is the “expanded genetic code” method (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Palzer <em>et al.</em> 2013</a>). In this method, a synthetic photo-cross-linker amino acid is genetically incorporated in a protein of interest to covalently capture its binding partner. It allows structural and functional analyses of PPI; however, the success of the assay depends on the incorporation efficiency of the unnatural amino acid (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wang <em>et al.</em> 2009</a>). Since these methods either fail or exhibit limitations in visualizing the native localization of the interacting proteins, as well as testing interactions of membrane proteins <em>in vivo</em>, we sought to optimize the use of bimolecular fluorescence complementation (BiFC) in <em>C. albicans</em>. BiFC is a method based on the reassembly of a fluorescent protein as a consequence of its two complementary N- and C-terminal fragments being brought together by a pair of interacting proteins (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kerppola 2009</a>). Consequently, the interaction complex can be directly visualized at its native subcellular location while allowing convenient quantitative analysis of the interaction by the fluorescence signal intensity. Over the last decade, the BiFC assay has been widely used for proteomic interaction studies in mammalian cells, plants, and the model yeast <em>S. cerevisiae</em> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hu <em>et al.</em> 2002</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Walter <em>et al.</em> 2004</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Sung <em>et al.</em> 2013</a>). It is of particular interest in host-pathogen interactions in plants (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lee and Gelvin 2014</a>), yet remains to be validated in fungal pathogens.</p><p id="p-5">Here we describe the development of a set of <em>C. albicans</em>-optimized BiFC plasmids for N- or C-terminal tagging of proteins with yeast-enhanced monomeric Venus (yEmVenus) fragments. To confirm the reliability of the assay, we have tested known interacting proteins, such as Snf4-Kis1, involved in carbohydrate metabolism, and components of the cAMP-dependent protein kinase A (PKA) pathway of <em>C. albicans</em>. We show the interaction of the upstream PKA receptor module Gpr1-Gpa2, and interaction of the PKA subunits Bcy1-Tpk1 and Bcy1-Tpk2. Together with complementary tools, our <em>Candida</em>-optimized BiFC assay shows immense potential in providing new insights into molecular mechanisms of PPI and signaling in <em>C. albicans</em>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains</h3><p id="p-6"><em>C. albicans</em> strains used in this study are detailed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. All the BiFC strains were generated in the SN152 strain (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Noble and Johnson 2005</a>). All the two-hybrid assays were performed using the previously described SC2H3 strain (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Stynen <em>et al.</em> 2010</a>). All strains were regularly replicated from −80° on to yeast extract peptone dextrose (YPD) plates (1% yeast extract, 2% peptone, 2% glucose 1, 5 % agar). Precultures were grown overnight in 3 ml YPD and washed in Milli-Q water prior to use. For two-hybrid assays, cells were plated on synthetic complete (SC) medium (0.17% Bacto-yeast nitrogen base, 0.5% ammonium sulfate, 2% glucose) lacking appropriate amino acids (US Biologicals). The pH for solid SC media was adjusted to 6.5 and to 5.5 for liquid media. Agar plates were made with 1.5% of agar. Strains were cultured at 30°, unless stated otherwise.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486848/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486848/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 C.</span> <span class="caption-title"><em>albicans</em> strains</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Construction of plasmids</h3><div id="sec-4" class="subsection"><h4>PCR cloning and templates:</h4><p id="p-7">All PCR-based cloning reactions were done using the Q5 polymerase from NEB (M0491S). For cloning of yEmVenus constructs, a full-length yEmVenus template was used, created by mutating the <em>C. albicans</em> codon-optimized yEVenus sequence [from plasmid pKT103, (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Sheff and Thorn 2004</a>)] at amino acid position 206, changing the alanine codon to a lysine codon. Cloning of the mCherry fluorophore sequence was done using the <em>C. albicans</em> codon-optimized template pMG2254, described in (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gerami-Nejad <em>et al.</em> 2009</a>). To clone <em>C. albicans</em> genes of interest, genomic DNA was prepared from the SC5314 strain (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Gillum <em>et al.</em> 1984</a>), and used as the template in our PCR reactions. All generated plasmids are listed in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1/TableS1.xlsx">Table S1</a>; all primers used are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1/TableS2.xlsx">Table S2</a>.</p></div><div id="sec-5" class="subsection"><h4>Construction of the BiFC plasmids:</h4><p id="p-8">The BiFC plasmids were constructed from the <em>C. albicans</em> two-hybrid vectors pC2HB and pC2HP (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Stynen <em>et al.</em> 2010</a>). In the first step, the two-hybrid specific inserts of pC2HB (NLS-LexA-HA) and pC2HP (NLS-VP16-FLAG) were cut out with <em>Bsp</em>EI and <em>Mlu</em>I, and new multiple cloning sites (MCS) were inserted by ligation with synthetic dsDNA fragments. To create the 2015-N and 2016-N vectors for N-terminal tagging of a gene of interest, the insert was formed by hybridizing the primers B-7412 and B-7413. In these vectors, the fluorophore part could be cloned into the <em>Bsp</em>EI and <em>Nhe</em>I sites, and separated by a linker sequence from the MCS (<em>Avr</em>II-<em>Xma</em>I-<em>Asc</em>I-<em>Mlu</em>I) for the gene of interest (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). To create the 2015-C and 2016-C vectors for C-terminal tagging of a gene of interest, the synthetic dsDNA insert was formed using the primers B-7414 and B-7415. In these vectors, a slightly different MCS for the gene of interest (<em>Bsp</em>EI-<em>Avr</em>II-<em>Xma</em>I-<em>Asc</em>I) was separated by the same linker from the fluorophore part, which could then be cloned into the <em>Nhe</em>I and <em>Mlu</em>I sites (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Backbone vectors with the new MCS were designated as 2015-N, 2015-C, 2016-N, and 2016-C (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3509/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structure of the MCS in backbone vectors for the BiFC plasmids. (A) MCS for the generation of N-terminal fusions of the fluorophore to the genes of interest. The fluorophore part is cloned in the BspEI and NheI sites on the left. Genes of interest are cloned using the AvrII-XmaI-AscI-MluI cloning site on the right, separating them from the fluorophore part by the indicated linker sequence. (B) MCS for the generation of C-terminal fusions of the fluorophore to the genes of interest. The fluorophore part is cloned in the NheI and MluI sites on the right. Genes of interest are cloned using the BspEI-AvrII-XmaI-AscI cloning site on the left, separating them from the fluorophore part by the indicated linker sequence. (C) Schematic representation of all possible BiFC fusion orientations with either the N-terminal (NV) or C-terminal (CV) part of yEmVenus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1979031841" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Structure of the MCS in backbone vectors for the BiFC plasmids. (A) MCS for the generation of N-terminal fusions of the fluorophore to the genes of interest. The fluorophore part is cloned in the &lt;em&gt;Bsp&lt;/em&gt;EI and &lt;em&gt;Nhe&lt;/em&gt;I sites on the left. Genes of interest are cloned using the &lt;em&gt;Avr&lt;/em&gt;II-&lt;em&gt;Xma&lt;/em&gt;I-&lt;em&gt;Asc&lt;/em&gt;I-&lt;em&gt;Mlu&lt;/em&gt;I cloning site on the right, separating them from the fluorophore part by the indicated linker sequence. (B) MCS for the generation of C-terminal fusions of the fluorophore to the genes of interest. The fluorophore part is cloned in the &lt;em&gt;Nhe&lt;/em&gt;I and &lt;em&gt;Mlu&lt;/em&gt;I sites on the right. Genes of interest are cloned using the &lt;em&gt;Bsp&lt;/em&gt;EI-&lt;em&gt;Avr&lt;/em&gt;II-&lt;em&gt;Xma&lt;/em&gt;I-&lt;em&gt;Asc&lt;/em&gt;I cloning site on the left, separating them from the fluorophore part by the indicated linker sequence. (C) Schematic representation of all possible BiFC fusion orientations with either the N-terminal (NV) or C-terminal (CV) part of yEmVenus.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3509/F1.medium.gif" width="413" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/10/3509/F1.medium.gif" width="413" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3509/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3509/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486935" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">Structure of the MCS in backbone vectors for the BiFC plasmids. (A) MCS for the generation of N-terminal fusions of the fluorophore to the genes of interest. The fluorophore part is cloned in the <em>Bsp</em>EI and <em>Nhe</em>I sites on the left. Genes of interest are cloned using the <em>Avr</em>II-<em>Xma</em>I-<em>Asc</em>I-<em>Mlu</em>I cloning site on the right, separating them from the fluorophore part by the indicated linker sequence. (B) MCS for the generation of C-terminal fusions of the fluorophore to the genes of interest. The fluorophore part is cloned in the <em>Nhe</em>I and <em>Mlu</em>I sites on the right. Genes of interest are cloned using the <em>Bsp</em>EI-<em>Avr</em>II-<em>Xma</em>I-<em>Asc</em>I cloning site on the left, separating them from the fluorophore part by the indicated linker sequence. (C) Schematic representation of all possible BiFC fusion orientations with either the N-terminal (NV) or C-terminal (CV) part of yEmVenus.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486960/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486960/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Backbone plasmids</span><div class="sb-div caption-clear"></div></div></div><p id="p-10">In the next step, the fluorophore parts were cloned into the new backbone vectors to generate the final BiFC plasmids. The large part of the yEmVenus (encoding amino acids 1–172) was cloned into both <em>CmLEU2</em> vector backbones, 2015-N and 2015-C. To generate the BiFC1 plasmid, yEmVenus (aa 1–172) was amplified with primers B-7567 and B-7568 and cloned into the 2015-N vector at the <em>Bsp</em>EI and <em>Nhe</em>I sites (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). The BiFC2 plasmid was created by cloning yEmVenus (aa 1–172), amplified with primers B-7569 and B-7570, into the 2015-C vector at the <em>Nhe</em>I and <em>Mlu</em>I sites (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>).</p><p id="p-11">The smaller complementing part of the yEmVenus fluorophore (encoding amino acids 173–238) was cloned into the <em>CdARG4</em> vector backbones, 2016-N and 2016-C. To generate the BiFC3 plasmid, yEmVenus (aa 173–238) was amplified with primers B-7571 and B-7572 and cloned into the 2016-N vector at the <em>Bsp</em>EI and <em>Nhe</em>I sites (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>). The BiFC4 plasmid was generated by cloning yEmVenus (aa 173–238), amplified with primers B-7573 and B-7574, into the 2016-C vector at the <em>Nhe</em>I and <em>Mlu</em>I sites (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>).</p><p id="p-12">The completed four BiFC plasmids (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1/FigureS1.pdf">Figure S1</a>) allowed generation of all the possible BiFC fusion orientations (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1C</a>). All constructs were expressed from the <em>C. albicans MET3</em> promoter, and the <em>C. albicans ACT1</em> terminator was used in all vectors. As such, expression was repressed by the presence of methionine and/or cysteine (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Care <em>et al.</em> 1999</a>). Expression was induced by growing the cells in medium lacking both methionine and cysteine.</p></div><div id="sec-6" class="subsection"><h4>Construction of plasmids for (co)localization studies:</h4><p id="p-13">To study the localization of proteins, full-length yEmVenus was cloned into the 2015-N and 2016-N vectors using primers B-7567 + B-7572 for N-terminal fusions, and into the 2015-C and 2016-C vectors using primers B-7569 + B-7574 for C-terminal fusions. In addition, similar vectors were made with full-length mCherry, using primers B-7579 and B-7584 to generate vectors for N-terminal fusions, and primers B-7581 and B-7586 for C-terminal fusions. Although not all backbone vectors were used in this study, the complete set of plasmids for N- and C-terminal tagging of genes for localization studies are available on request.</p></div><div id="sec-7" class="subsection"><h4>Cloning of genes of interest into the BiFC and two-hybrid plasmids:</h4><p id="p-14">In general, the genes of interest (or fragments thereof) were PCR-amplified and cloned into the BiFC plasmids, using In-Fusion cloning (Clontech). For each gene, two sets of primers were used, to discriminate between cloning into the BiFC1 and BiFC3 vectors, generating N-terminal fusions (here, genes were cloned with their stop codons), and cloning into the BiFC2 and BiFC4 vectors, generating C-terminal fusions (here, the genes were cloned without their stop codons).</p><p id="p-15">All BiFC plasmids were confirmed by sequencing. The complete list of plasmids and primers used in this study is given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1/TableS1.xlsx">Table S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1/TableS2.xlsx">Table S2</a>, respectively.</p></div></div><div id="sec-8" class="subsection"><h3>Candida two-hybrid analysis</h3><p id="p-16">For the two-hybrid analysis, genes of interest were cloned by the classical restriction-ligation method into the prey and bait vectors, pC2HP and pC2HB, containing the <em>C. dubliniensis ARG4</em> or <em>C. maltosa LEU2</em> marker for selection on growth media lacking arginine or leucine, respectively (<a id="xref-ref-37-7" class="xref-bibr" href="#ref-37">Stynen <em>et al.</em> 2010</a>). Constructs expressing the full-length <em>SNF4</em> and <em>KIS1</em> genes were retrieved from <a id="xref-ref-37-8" class="xref-bibr" href="#ref-37">Stynen <em>et al.</em> 2010</a>, and the pC2HP vector expressing the truncated <em>KIS1</em> (lacking its C-terminal amino acids 327–412) was made in this study. All two-hybrid plasmids were confirmed by sequencing. Both prey and bait vectors were transformed into the SC2H3 reporter strain, which is auxotrophic for arginine, leucine, and histidine. As with the BiFC constructs, expression of the two-hybrid constructs is controlled by a <em>MET3</em> repressible promoter, and thus induced by growing the strains in media lacking both methionine and cysteine (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Care <em>et al.</em> 1999</a>). The SC2H3 strain offers two reporter systems for PPI readout, a colorimetric reaction based on the expression of a <em>lacZ</em> gene, and growth on medium lacking histidine, due to the expression of the auxotrophic <em>HIS1</em> marker upon interaction of the proteins (<a id="xref-ref-37-9" class="xref-bibr" href="#ref-37">Stynen <em>et al.</em> 2010</a>). In this paper, interaction was scored by analyzing growth on media lacking methionine, cysteine, and histidine only.</p></div><div id="sec-9" class="subsection"><h3>Construction of strains</h3><p id="p-17">All <em>C. albicans</em> transformations were performed using a lithium acetate method (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gietz <em>et al.</em> 1995</a>). Prior to transformation, all plasmids were linearized with the <em>Not</em>I restriction enzyme for integration. BiFC1, BiFC2, and bait pC2HB plasmids integrated between loci <em>XOG1</em> and <em>HOL1</em> on chromosome 1, while the BiFC3, BiFC4, and prey pC2HP plasmids integrated between <em>RXT3</em> and <em>ORF19</em>.<em>3569</em> on chromosome 2 (<a id="xref-ref-37-10" class="xref-bibr" href="#ref-37">Stynen <em>et al.</em> 2010</a>). Integration of plasmids was verified by PCR diagnosis.</p></div><div id="sec-10" class="subsection"><h3>Microplate reader measurements</h3><p id="p-18">The H1 Synergy microplate reader (BioTek) was used for intermittent 24 hr measurements of optical density (OD) at 600 nm and fluorescence with excitation 485/20 and emission 528/20 nm filters. For these measurements, all cultures were grown overnight in YPD, washed, diluted to OD ≈ 0.1, and transferred to low-fluorescence SC –methionine –cysteine medium (2% glucose). A strain containing the backbone vectors 2015-N and 2016-N (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) was used to measure cellular autofluorescence under our experimental conditions. For all the tested strains, fluorescence intensity was normalized for cell density, by calculating the ratio of relative fluorescence units over the OD<sub>600nm</sub>. Then, these values for the test strains were divided by the corresponding values for autofluorescence, giving a final relative fluorescence intensity due to the BiFC interaction in our test strains.</p></div><div id="sec-11" class="subsection"><h3>Confocal microscopy and quantitative analysis of images</h3><p id="p-19">For fluorescence imaging, cells were grown overnight in YPD, washed, and transferred to low-fluorescence SC –methionine –cysteine medium supplemented with low-fluorescence YNB (Formedium) or standard SC –methionine –cysteine medium (2% glucose) until the midexponential phase when they were visualized. Imaging was performed on an Olympus FluoView FV1000 confocal laser scanning microscope equipped with UPLSAPO 60× O NA 1.35 objective lens. yEmVenus fluorescence was excited with the 515 nm laser line emitted from an argon laser, and emission was detected through a bandpass filter BA535-565. Nuclear staining was performed with NucBlue Live ReadyProbes (Thermo Fisher Scientific) according to the manufacturer’s protocol. NucBlue stain was excited with the 405 nm laser diode and detected through a bandpass filter BA430-470. mCherry fluorescence was excited with the 559 nm laser diode, and emission was detected through a bandpass filter BA575-675. Quantification of the signal was done with a home-developed MATLAB routine (version 7.11.0.584; MathWorks, Natick, MA), which is available upon request. Cells were discriminated from the background by an adaptive local threshold combined with morphological operations to fill and close the areas. Afterward, the selected areas were visually checked. The mean background intensity per pixel was calculated based on a selected area in the image without any cell present. Thereafter, the mean intensity per pixel, corrected by the mean background intensity, was calculated for each cell separately. Analysis and comparison of fluorescence intensity was done with raw images acquired with identical acquisition settings of the microscope (laser power, detector sensitivity, and pixel dwell time). For optimal display, copies of all original images and of their controls were equally adjusted for brightness, contrast, and color balance using Adobe Photoshop CS6.</p></div><div id="sec-12" class="subsection"><h3>Western blot analysis</h3><p id="p-20">Strains were grown overnight in YPD, washed, and transferred to SC –methionine –cysteine medium (2% glucose) until the exponential phase. Cells were then harvested and washed with lysis buffer [200 mM sorbitol, 20 mM HEPES–KOH (pH 6.8), 1 mM EDTA, 50 mM potassium acetate, and protease inhibitors (Roche)]. Next, the cells were broken with glass beads using a FastPrep machine. The concentration of proteins was determined with the Pierce protein assay (Thermo Fisher Scientific), and 14 µg/well was loaded on an Invitrogen NuPAGE Novex bis-Tris gradient gel (4–12%). For detection of the BiFC constructs, a polyclonal anti-GFP antibody (GeneTex, GTX113617) was used, which recognizes both the N- and C-terminal fragments of the yEmVenus fluorescent protein. Detection of histone H3 (loading control) was done using an anti-histone H3 antibody (Abcam, ab1791). The blots were visualized using a FujiFilm LAS-4000 mini system. Quantification was carried out with ImageJ 1.51d software (National Institutes of Health). The intensity of each band was corrected for the background and normalized to the intensity of its respective loading control band.</p></div><div id="sec-13" class="subsection"><h3>Staining of vacuoles</h3><p id="p-21">The cells were incubated with FM4-64 (<em>N</em>-(3-triethylammoniumpropyl)-4-(6-(4-(diethylamino) phenyl) hexatrienyl) pyridinium dibromide, Invitrogen) at a final concentration of 0.04 mg/ml for 30 min at 30°. After being washed, the cultures were imaged. The dye was excited with the 559 nm laser diode and emission was detected through a bandpass filter BA575-675.</p></div><div id="sec-14" class="subsection"><h3>Data availability</h3><p id="p-22">Complete lists of the plasmids and primers used in this study are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1/TableS1.xlsx">Table S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1/TableS2.xlsx">Table S2</a>, respectively. The four BiFC plasmids (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1/FigureS1.pdf">Figure S1</a>) are available from the Belgian Coordinated Collections of Microorganisms (BCCM/LMBP plasmid collection; <a href="http://bccm.belspo.be/about/lmbp.php">http://bccm.belspo.be/about/lmbp.php</a>) with the following accession numbers: LMBP 10464 (BiFC1), LMBP 10465 (BiFC2), LMBP 10466 (BiFC3), LMBP 10467 (BiFC4). Their sequences are available as supplementary GenBank standard files (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1/FileS1.txt">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1/FileS2.txt">File S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1/FileS3.txt">File S3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1/FileS4.txt">File S4</a>), while the sequences of all the other plasmids used in this study are available on request. Clustal Omega is a multiple sequence alignment program available online at <a href="http://www.ebi.ac.uk/Tools/msa/clustalo/">http://www. ebi.ac.uk/Tools/msa/clustalo/</a>. ImageJ software from the National Institutes of Health is available online at <a href="https://imagej.nih.gov/ij/">https://imagej.nih.gov/ij/</a>. The MATLAB code is available upon request.</p></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><div id="sec-16" class="subsection"><h3>Development of Candida-optimized BiFC plasmids</h3><p id="p-23"><em>Candida</em>-optimized BiFC plasmids were designed for tagging genes of interest, either N- or C-terminally, with split yEmVenus fragments (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Monomeric yEVenus was created by introducing the A206K mutation in the <em>C. albicans</em> codon-optimized yEVenus sequence (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Sheff and Thorn 2004</a>). Generally, monomeric fluorescent proteins display less aggregation and are nontoxic when expressed in cells (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Shaner <em>et al.</em> 2005</a>). In our BiFC system, we split the yEmVenus in the loop between the eighth and ninth β-strands of the fluorophore, as was done by others (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kodama and Hu 2012</a>). As such, our two yEmVenus fragments were a large fragment, from amino acid 1–172, and the complementing smaller fragment, encompassing amino acids 173–238. The BiFC plasmids contained the <em>MET3</em> repressible promoter (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Care <em>et al.</em> 1999</a>) for expression of gene fusions and either the <em>C. dubliniensis ARG4</em> or <em>C. maltosa LEU2</em> marker for selection on growth media lacking arginine or leucine, respectively. As appropriate start and stop codons were incorporated into the MCS sequences (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1, A and B</a>), the fluorophore segments were expressed even in the absence of the target gene. As such these empty BiFC vectors could be used as negative controls (self-assembly of the fluorophore) for BiFC analyses.</p></div><div id="sec-17" class="subsection"><h3>Proof-of-principle experiment—interaction of Snf4-Kis1 proteins</h3><p id="p-24">To validate the BiFC assay for use in <em>C. albicans</em>, we first tagged <em>SNF4</em> and <em>KIS1</em>, which encode known interacting proteins previously shown using the <em>Candida</em> two-hybrid method (<a id="xref-ref-37-11" class="xref-bibr" href="#ref-37">Stynen <em>et al.</em> 2010</a>). Snf4 and Kis1 belong to the Snf1 kinase complex, which is involved in the regulation of carbohydrate metabolism and growth adaptation on nonfermentable sugars in <em>C. albicans</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Corvey <em>et al.</em> 2005</a>). To check for interaction, <em>SNF4</em> and <em>KIS1</em> were cloned into the BiFC plasmids and integrated in the SN152 strain. Strains expressing all eight tagging combinations were tested with the microplate reader system. The BiFC orientation expressing Snf4-CV (C-terminal tagging with the shorter C-terminal segment of the fluorophore) and Kis1-NV (C-terminal tagging with the longer N-terminal segment of the fluorophore) showed the highest fluorescence complementation compared to the background, as well as the highest reproducibility in fluorescence levels (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Also, a significantly higher reassembly-derived fluorescence signal, relative to the autofluorescence under the same conditions, was observed in the exponential phase than in the stationary growth phase (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1/FigureS2.ai">Figure S2</a>). Therefore, for this BiFC orientation, appropriate negative controls were generated and the fluorescence signals were reanalyzed compared to these controls during the exponential growth phase (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Since both negative controls showed nearly no nonspecific complementation, we proceeded to microscopical analysis. When visualized, the two proteins strongly interacted in glucose-containing media with the fluorescence signal occurring throughout the cytoplasm (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3509/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Microplate reader analysis of all eight Snf4-Kis1 BiFC combinations. Quantitative analysis of Snf4-Kis1 BiFC signal in the exponential phase (black bars) and stationary phase (white bars). Relative fluorescence intensity represents the ratio of BiFC strains&#x2019; fluorescence over fluorescence of background strain containing backbone vectors. Data represent mean values &#xB1; SEM (n = 5). The experiment was performed twice with similar results; a representative experiment is shown. *Significant difference compared with the background strain determined in the same growth phase (two-way ANOVA, Sidak&#x2019;s multiple comparisons test, * P &#x2264; 0.05, ** P &#x2264; 0.01, *** P &#x2264; 0.001). (B) Microplate reader analysis of the optimal Snf4-Kis1 BiFC combination by including the negative controls. Snf4-C-yEmVenus and Kis1-N-yEmVenus both tagged C-terminally were selected as the optimal combination. Kis1&#x394;ASC shows significantly less fluorescence complementation with Snf4 compared to the full-length Kis1. Fluorescence was analyzed by the microplate system, in the exponential phase of growth. Relative fluorescence intensity represents the ratio of BiFC strains&#x2019; fluorescence over fluorescence of background strain containing backbone vectors. Data represent mean values &#xB1; SEM (n = 5). The experiment was performed twice with similar results; a representative experiment is shown. *Significant difference compared to control strains (one-way ANOVA, Dunnett&#x2019;s multiple comparisons test, ** P &#x2264; 0.01, **** P &#x2264; 0.0001). (C) Alignment of the C. albicans Kis1 and homologous Sip2 protein in S. cerevisiae. The identified ScSip2 ASC domain (Jiang and Carlson 1997) is highlighted in blue. Alignment was performed using the Clustal Omega online tool." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1979031841" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Microplate reader analysis of all eight Snf4-Kis1 BiFC combinations. Quantitative analysis of Snf4-Kis1 BiFC signal in the exponential phase (black bars) and stationary phase (white bars). Relative fluorescence intensity represents the ratio of BiFC strains&#x2019; fluorescence over fluorescence of background strain containing backbone vectors. Data represent mean values &#xB1; SEM (&lt;em&gt;n&lt;/em&gt; = 5). The experiment was performed twice with similar results; a representative experiment is shown. *Significant difference compared with the background strain determined in the same growth phase (two-way ANOVA, Sidak&#x2019;s multiple comparisons test, * &lt;em&gt;P&lt;/em&gt; &#x2264; 0.05, ** &lt;em&gt;P&lt;/em&gt; &#x2264; 0.01, *** &lt;em&gt;P&lt;/em&gt; &#x2264; 0.001). (B) Microplate reader analysis of the optimal Snf4-Kis1 BiFC combination by including the negative controls. Snf4-C-yEmVenus and Kis1-N-yEmVenus both tagged C-terminally were selected as the optimal combination. Kis1&#x394;ASC shows significantly less fluorescence complementation with Snf4 compared to the full-length Kis1. Fluorescence was analyzed by the microplate system, in the exponential phase of growth. Relative fluorescence intensity represents the ratio of BiFC strains&#x2019; fluorescence over fluorescence of background strain containing backbone vectors. Data represent mean values &#xB1; SEM (&lt;em&gt;n&lt;/em&gt; = 5). The experiment was performed twice with similar results; a representative experiment is shown. *Significant difference compared to control strains (one-way ANOVA, Dunnett&#x2019;s multiple comparisons test, ** &lt;em&gt;P&lt;/em&gt; &#x2264; 0.01, **** &lt;em&gt;P&lt;/em&gt; &#x2264; 0.0001). (C) Alignment of the &lt;em&gt;C. albicans&lt;/em&gt; Kis1 and homologous Sip2 protein in &lt;em&gt;S. cerevisiae&lt;/em&gt;. The identified ScSip2 ASC domain (Jiang and Carlson 1997) is highlighted in blue. Alignment was performed using the Clustal Omega online tool.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3509/F2.medium.gif" width="301" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/10/3509/F2.medium.gif" width="301" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3509/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3509/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486835" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">(A) Microplate reader analysis of all eight Snf4-Kis1 BiFC combinations. Quantitative analysis of Snf4-Kis1 BiFC signal in the exponential phase (black bars) and stationary phase (white bars). Relative fluorescence intensity represents the ratio of BiFC strains’ fluorescence over fluorescence of background strain containing backbone vectors. Data represent mean values ± SEM (<em>n</em> = 5). The experiment was performed twice with similar results; a representative experiment is shown. *Significant difference compared with the background strain determined in the same growth phase (two-way ANOVA, Sidak’s multiple comparisons test, * <em>P</em> ≤ 0.05, ** <em>P</em> ≤ 0.01, *** <em>P</em> ≤ 0.001). (B) Microplate reader analysis of the optimal Snf4-Kis1 BiFC combination by including the negative controls. Snf4-C-yEmVenus and Kis1-N-yEmVenus both tagged C-terminally were selected as the optimal combination. Kis1ΔASC shows significantly less fluorescence complementation with Snf4 compared to the full-length Kis1. Fluorescence was analyzed by the microplate system, in the exponential phase of growth. Relative fluorescence intensity represents the ratio of BiFC strains’ fluorescence over fluorescence of background strain containing backbone vectors. Data represent mean values ± SEM (<em>n</em> = 5). The experiment was performed twice with similar results; a representative experiment is shown. *Significant difference compared to control strains (one-way ANOVA, Dunnett’s multiple comparisons test, ** <em>P</em> ≤ 0.01, **** <em>P</em> ≤ 0.0001). (C) Alignment of the <em>C. albicans</em> Kis1 and homologous Sip2 protein in <em>S. cerevisiae</em>. The identified ScSip2 ASC domain (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jiang and Carlson 1997</a>) is highlighted in blue. Alignment was performed using the Clustal Omega online tool.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3509/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Representative BiFC images showing in vivo interaction of Snf4-CV + Kis1-NV and control strains Snf4-CV + Kis1&#x394;ASC-NV, CV + Kis1-NV, and Snf4-CV + NV. Bar is 10 &#xB5;m. (B) Quantification of fluorescence intensity per pixel of strains Snf4-CV + Kis1-NV and Snf4-CV + Kis1&#x394;ASC-NV. Controls could not be quantified as no signal was detected. Analysis of fluorescence intensity was done with original images using the same acquisition settings of the microscope. Data represent mean values &#xB1; SEM of (n = 3; &#xB1;100 cells per replicate). *Significant difference (unpaired t-test, * P &#x2264; 0.05). (C) Two-hybrid interaction of C. albicans Kis1 with Snf4 depends on the ASC domain of Kis1. Growth of all strains on SC -leucine - arginine indicates the presence of both yeast two-hybrid constructs, whereas growth on SC -methionine -cysteine -histidine demonstrates interaction between the proteins. Plates were incubated for 3 d at 30&#xB0; before pictures were taken." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1979031841" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Representative BiFC images showing &lt;em&gt;in vivo&lt;/em&gt; interaction of Snf4-CV + Kis1-NV and control strains Snf4-CV + Kis1&#x394;ASC-NV, CV + Kis1-NV, and Snf4-CV + NV. Bar is 10 &#xB5;m. (B) Quantification of fluorescence intensity per pixel of strains Snf4-CV + Kis1-NV and Snf4-CV + Kis1&#x394;ASC-NV. Controls could not be quantified as no signal was detected. Analysis of fluorescence intensity was done with original images using the same acquisition settings of the microscope. Data represent mean values &#xB1; SEM of (&lt;em&gt;n&lt;/em&gt; = 3; &#xB1;100 cells per replicate). *Significant difference (unpaired &lt;em&gt;t&lt;/em&gt;-test, * &lt;em&gt;P&lt;/em&gt; &#x2264; 0.05). (C) Two-hybrid interaction of &lt;em&gt;C. albicans&lt;/em&gt; Kis1 with Snf4 depends on the ASC domain of Kis1. Growth of all strains on SC -leucine - arginine indicates the presence of both yeast two-hybrid constructs, whereas growth on SC -methionine -cysteine -histidine demonstrates interaction between the proteins. Plates were incubated for 3 d at 30&#xB0; before pictures were taken.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3509/F3.medium.gif" width="440" height="432"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/10/3509/F3.medium.gif" width="440" height="432"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3509/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3509/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486820" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">(A) Representative BiFC images showing <em>in vivo</em> interaction of Snf4-CV + Kis1-NV and control strains Snf4-CV + Kis1ΔASC-NV, CV + Kis1-NV, and Snf4-CV + NV. Bar is 10 µm. (B) Quantification of fluorescence intensity per pixel of strains Snf4-CV + Kis1-NV and Snf4-CV + Kis1ΔASC-NV. Controls could not be quantified as no signal was detected. Analysis of fluorescence intensity was done with original images using the same acquisition settings of the microscope. Data represent mean values ± SEM of (<em>n</em> = 3; ±100 cells per replicate). *Significant difference (unpaired <em>t</em>-test, * <em>P</em> ≤ 0.05). (C) Two-hybrid interaction of <em>C. albicans</em> Kis1 with Snf4 depends on the ASC domain of Kis1. Growth of all strains on SC -leucine - arginine indicates the presence of both yeast two-hybrid constructs, whereas growth on SC -methionine -cysteine -histidine demonstrates interaction between the proteins. Plates were incubated for 3 d at 30° before pictures were taken.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">It was previously shown in <em>S. cerevisiae</em> that <em>Sc</em>Snf4 interacts with <em>Sc</em>Sip1/Sip2/Gal83 through an ASC (association with the Snf1p kinase complex) domain (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Jiang and Carlson 1997</a>). A homologous domain, aa 327–412, was also identified in the <em>C. albicans KIS1</em> gene (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>) (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Corvey <em>et al.</em> 2005</a>). When integrated in SN152, the truncated Kis1 showed significantly less fluorescence complementation with Snf4 compared to the full-length Kis1, in both the microplate system and in quantitative analysis of microscopy images (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a> and <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). This loss of interaction was also confirmed using the <em>Candida</em> two-hybrid assay (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). In addition, we assessed the protein expression levels of the full-length Kis1-NV and the Kis1ASCΔ-NV protein fusions in the Snf4-Kis1 BiFC strains. For the western blot analysis, we used an anti-GFP antibody, which recognizes both N- and C-terminal yEmVenus fragments. The results showed that the amount of the Kis1ASCΔ protein was decreased compared to the full-length Kis1-NV, indicating that the deletion of the ASC domain affects its stability (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1/FigureS3.ai">Figure S3</a>, A and B). However, the extent to which the instability of Kis1 is a cause or consequence of its loss of interaction with the complex remains to be investigated. Overall, these results indicate that it can be important to assess the stability of fusion proteins to verify that they are present in comparable amounts and to exclude the possibility of false negative BiFC results.</p></div><div id="sec-18" class="subsection"><h3>Application of BiFC on the interaction of membrane-bound Gpr1 and Gpa2</h3><p id="p-28">G-protein coupled receptors (GPCRs) are among the best-studied membrane sensors in eukaryotes, and the <em>Sc</em>Gpr1-Gpa2 GPCR system in <em>S. cerevisiae</em> represents the first-described glucose-sensing GPCR in eukaryotes (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kraakman <em>et al.</em> 1999</a>). Similarly, in <em>C. albicans</em>, Gpr1 together with Gpa2 acts upstream of the cAMP-PKA pathway and is involved in morphogenesis on different solid hyphae-inducing media (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Maidan <em>et al.</em> 2005</a>). One of the main advantages of the BiFC assay over the two-hybrid system is that it enables detection and visualization of PPI involving membrane proteins. Therefore, we have employed our BiFC system for visualizing the interaction between Gpr1 and Gpa2. The physical interaction between the C-terminal region of <em>C. albicans</em> Gpr1 (aa 660–823) and Gpa2 had already been shown using the <em>S. cerevisiae</em> two-hybrid assay (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Miwa <em>et al.</em> 2004</a>). Using the BiFC system, we tested the interaction of the full-length Gpr1 and the C-tail of Gpr1 (aa 660–823) with Gpa2 in <em>C. albicans</em>. Gpr1 was only tagged C-terminally, as its N-terminus is extracellularly located. As such, only four possible tagging combinations were tested, together with their respective controls. Compared with the Snf4-Kis1 interaction, the signal of Gpr1-Gpa2 interactions was much lower, reaching the detection limit of the microplate reader. Therefore, the Gpr1-Gpa2 BiFC combinations could not be scored using the microplate reader system, and selection was performed by direct imaging of the different combinations under the microscope.</p><p id="p-29">Visualization of these strains showed a membrane-localized interaction of full-length Gpr1 with Gpa2, whereas the C-tail of Gpr1 interacted with Gpa2 in the cytosol (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). Intriguingly, for the full-length Gpr1 interaction, an additional fluorescent signal was observed, which localized to the vacuole, indicating degradation of the interacting complex. As we observed a similar pattern of membrane localization, along with a vacuolar signal, when solely expressing the Gpr1-yEmVenus from the same vector (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1/FigureS4.ai">Figure S4</a>), it could be that the observed vacuolar targeting was due to the regular endocytic recycling process of Gpr1. Furthermore, this vacuolar signal was missing from strains showing the interaction of Gpa2 with the C-tail of Gpr1 only.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3509/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Representative BiFC images showing in vivo interaction of Gpr1-CV + NV-Gpa2, the C-tail of Gpr1 and Gpa2, and control strains expressing Gpr1-CV + NV, Gpr1-C-tail-CV + NV, and CV + NV-Gpa2. Bar is 10 &#xB5;m. (B) Quantification of fluorescence intensity per pixel of Snf4-Kis1, Gpr1-Gpa2, and Gpr1-C-tail-Gpa2 strains. Analysis of fluorescence intensity was done with original images using the same acquisition settings of the microscope. Data represent mean values &#xB1; SEM (n = 3; &#xB1;60 cells per replicate). *Significant difference (one-way ANOVA, Tukey&#x2019;s multiple comparison test, *** P &#x2264; 0.001)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1979031841" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Representative BiFC images showing &lt;em&gt;in vivo&lt;/em&gt; interaction of Gpr1-CV + NV-Gpa2, the C-tail of Gpr1 and Gpa2, and control strains expressing Gpr1-CV + NV, Gpr1-C-tail-CV + NV, and CV + NV-Gpa2. Bar is 10 &#xB5;m. (B) Quantification of fluorescence intensity per pixel of Snf4-Kis1, Gpr1-Gpa2, and Gpr1-C-tail-Gpa2 strains. Analysis of fluorescence intensity was done with original images using the same acquisition settings of the microscope. Data represent mean values &#xB1; SEM (&lt;em&gt;n&lt;/em&gt; = 3; &#xB1;60 cells per replicate). *Significant difference (one-way ANOVA, Tukey&#x2019;s multiple comparison test, *** &lt;em&gt;P&lt;/em&gt; &#x2264; 0.001).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3509/F4.medium.gif" width="415" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/10/3509/F4.medium.gif" width="415" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3509/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3509/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486937" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">(A) Representative BiFC images showing <em>in vivo</em> interaction of Gpr1-CV + NV-Gpa2, the C-tail of Gpr1 and Gpa2, and control strains expressing Gpr1-CV + NV, Gpr1-C-tail-CV + NV, and CV + NV-Gpa2. Bar is 10 µm. (B) Quantification of fluorescence intensity per pixel of Snf4-Kis1, Gpr1-Gpa2, and Gpr1-C-tail-Gpa2 strains. Analysis of fluorescence intensity was done with original images using the same acquisition settings of the microscope. Data represent mean values ± SEM (<em>n</em> = 3; ±60 cells per replicate). *Significant difference (one-way ANOVA, Tukey’s multiple comparison test, *** <em>P</em> ≤ 0.001).</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">In agreement with the observed lower signal in the microplate reader system, quantification of microscopy images of Gpr1-CV + NV-Gpa2 and Gpr1-C-tail-CV + NV-Gpa2 strains confirmed a significantly lower signal compared to the Snf4-Kis1 BiFC signal (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). Interestingly, the interaction of Gpa2 with the C-tail of Gpr1 appeared stronger than when full-length Gpr1 was used. This may be explained by the observed vacuolar targeting of specifically full-length Gpr1 upon overexpression. In addition to the low signals in the fluorescent assays, in a parallel two-hybrid analysis of all interactions, the Gpr1-C-tail-Gpa2 interaction strain showed less growth on selective medium compared to other interaction strains (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1/FigureS5.ai">Figure S5</a>). These different interaction assays may point to a weak interaction between Gpr1 and Gpa2, compared to the Snf4-Kis1 tested in the same conditions, although it is also possible that our testing conditions were not optimal for maximal Gpr1-Gpa2 interaction. It is, thus, not clear yet to what extent this Gpr1-Gpa2 interaction is inherently weaker. Overall, these results demonstrate that the BiFC assay can detect binary complexes involving membrane-bound proteins <em>in vivo</em>.</p></div><div id="sec-19" class="subsection"><h3>BiFC interaction of Bcy1-Tpk1 and Bcy1-Tpk2 subunits of PKA pathway</h3><p id="p-32">Visualization of the Gpr1-Gpa2 PKA receptor module led us to further characterize the interaction of the PKA subunits using BiFC in <em>C. albicans</em>. Besides being involved in growth and stress response, PKA is the key regulator of yeast-to-hyphae transition in <em>C. albicans</em> (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fuller and Rhodes 2012</a>). In <em>C. albicans</em>, PKA consists of two catalytic subunits, Tpk1 and Tpk2, which interact and are regulated by Bcy1, the regulatory subunit. Environmental cues specific to the host niche, such as serum, CO<sub>2</sub>, and changes in pH and body temperature (37°), alter the intracellular level of cAMP, which binds to the Bcy1 subunits, in turn releasing the active Tpk1 and Tpk2. Subsequently, active Tpk1 and Tpk2 promote hyphae development through the activation of a downstream transcription factor, Efg1 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Stoldt <em>et al.</em> 1997</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sudbery 2011</a>).</p><p id="p-33">Remarkably, little is known about PKA activity and its regulation <em>in vivo</em> in <em>C. albicans</em>. Previous attempts to assess the localization of PKA subunits in <em>C. albicans</em> were only possible in stationary-phase cells owing to low expression of endogenously tagged subunits in exponentially growing cells (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cassola <em>et al.</em> 2004</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Giacometti <em>et al.</em> 2006</a>).</p><p id="p-34">Here, we applied our BiFC system to test the interaction of both catalytic subunits, Tpk1 and Tpk2, with Bcy1. Initial selection of combinations was done by the microplate reader system. However, further analysis with the microplate reader detected high nonspecific fluorescence of both Tpk1- and Tpk2-tagged negative controls. The same nonspecific signal was detected when we expressed the tagged Tpk1 or Tpk2 without the other complementary fluorophore part (data not shown), indicating that the overexpression of only catalytic subunits leads to a production of fluorescence in the media, independent of BiFC complex formation. Previously reported data showed that a hyperactive adenylate cyclase mutant leads to increased production and secretion of riboflavin (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bai <em>et al.</em> 2011</a>), which is inherently fluorescent under the same conditions we used for detection of total fluorescence in the microplate reader system. Other unpublished results from our laboratory confirmed that overexpression of the genes encoding the catalytic Tpk subunits induces riboflavin production, which is secreted in the media (data not shown). Therefore, we have based the selection of optimal combinations on microscopy data, which proved to be stringent and most relevant in this case. Based on the microscopic analysis of the interaction strains together with their negative controls, both C-terminally tagged Bcy1 and Tpk subunits were selected as the optimal combinations. Imaging of interaction strains showed that Tpk1 and Tpk2 interact with Bcy1 in the cytoplasm and form interaction complexes observed as fluorescent foci in the exponential phase (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Nuclear staining of these strains excluded the presence of these foci in the nucleus (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). To further examine these results, we also performed colocalization experiments with Bcy1 and Tpk subunits, tagged with full-length yEmVenus and mCherry, respectively. We could observe that both Bcy1 and Tpk2 were predominantly present in the cytoplasm and did not localize to the nucleus, whereas Tpk1 was present in both the cytoplasm and the nucleus (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1/FigureS6.ai">Figure S6</a>). This indicates that, under our experimental conditions, there is a fraction of nuclear Tpk1, free from Bcy1-mediated inhibition, which might be involved in promoting filamentous growth. In agreement with such a Tpk1-specific role in promoting filamentation is the observation of increased Tpk1 genomic binding sites during hyphal induction using chromatin immunoprecipitation on chip in <em>C. albicans</em> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Schaekel <em>et al.</em> 2013</a>). Also, it was shown that Tpk2 is present in the cell in higher amounts than Tpk1 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cloutier <em>et al.</em> 2003</a>), so the specific upregulation of Tpk1 in the nucleus might have a physiological reason. Overall, these observations indicate that the interaction of PKA subunits takes place in the cytoplasm, as shown by the BiFC signal; however, the subunits did not show inclusions that would correspond to formation of foci. Both colocalization and BiFC experiments were done by overexpressing both the Bcy1 and Tpk subunits using the same vector backbone and promoter (<em>MET3</em>). The only difference was the tagging of both subunits, either fused with a full-length yEmVenus/mCherry, or with the BiFC yEmVenus split fragments. Therefore, we can assume that the amount of overexpression is comparable in both systems. As such, the reason we observed the foci only in the BiFC strain was not an effect of stress caused by overproduction, which also occurred in the colocalization experiment. Furthermore, we observed a similar pattern of filamentation for both the colocalization and BiFC strains, as well as the control strains tested in our conditions (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1/FigureS6.ai">Figure S6</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3509/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Representative BiFC images showing in vivo interaction of Bcy1-NV + Tpk1-CV, Bcy1-NV + Tpk2-CV with nuclear staining, controls strains expressing Bcy1-NV + CV, NV + Tpk1-CV, NV + Tpk2-CV, and the background strain. Images are taken in the exponential phase of growth in SC-Met-Cys 2% glucose at 37&#xB0;. Bar is 10 &#xB5;m." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1979031841" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Representative BiFC images showing &lt;em&gt;in vivo&lt;/em&gt; interaction of Bcy1-NV + Tpk1-CV, Bcy1-NV + Tpk2-CV with nuclear staining, controls strains expressing Bcy1-NV + CV, NV + Tpk1-CV, NV + Tpk2-CV, and the background strain. Images are taken in the exponential phase of growth in SC-Met-Cys 2% glucose at 37&#xB0;. Bar is 10 &#xB5;m.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3509/F5.medium.gif" width="343" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/10/3509/F5.medium.gif" width="343" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3509/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3509/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486811" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child">Representative BiFC images showing <em>in vivo</em> interaction of Bcy1-NV + Tpk1-CV, Bcy1-NV + Tpk2-CV with nuclear staining, controls strains expressing Bcy1-NV + CV, NV + Tpk1-CV, NV + Tpk2-CV, and the background strain. Images are taken in the exponential phase of growth in SC-Met-Cys 2% glucose at 37°. Bar is 10 µm.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">One possible explanation of the foci formation is that the interaction between Bcy1 and Tpk occurs transiently at the specific location, and this can be seen owing to the irreversible reconstitution of the BiFC complex, which stabilizes the interaction. This stabilized complex then possibly accumulates at the observed location, forming large foci. Formation of such large foci is probably not due to aggregation of the fluorophores, since we used a monomeric variant of yEVenus that is less prone to dimerize, but possibly an artificial enrichment of the interaction complex. Furthermore, no foci were observed when analyzing the Snf4-Kis1 interaction, implying that this is not a general phenomenon for cytoplasmic interaction in our BiFC system. However, further investigation of the functional and biological significance of these PKA interaction sites is necessary to fully describe their function in <em>C. albicans</em>.</p></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-37">In this study, we have generated a set of <em>Candida</em>-optimized BiFC plasmids for interaction analysis of proteins of interest. Our set of BiFC plasmids allows for N- and C-terminal tagging of proteins with split yEmVenus fragments in a way that all eight combinatorial possibilities can be analyzed.</p><p id="p-38">Optimally, the BiFC assay is performed with endogenously tagged proteins rather than overexpression. However, we have opted to use a BiFC plasmid-based system, which allows for controlled expression via the <em>MET3</em> promoter, and thereby enables detection independent of native expression, which may be too low for visualization. Another advantage of this system is that it enables tagging of genes in all eight combinations. Testing all combinations allows for selection of the optimal combination of fusion proteins and avoids possible topological constraints of certain tag orientations (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Miller <em>et al.</em> 2015</a>). If desired, our BiFC plasmids for C-terminal tagging are suitable as templates for endogenous tagging as well.</p><p id="p-39">Using our BiFC assay, we could visualize the cytosolic interaction of the Snf4-Kis1 pair. For this specific pair of proteins, easy quantitative analysis, using a microplate reader setup, proved to be useful for selecting the optimal interacting combination. We also tested the importance of the ASC domain of Kis1 for the interaction with Snf4. Interestingly, we found that truncation of this domain compromised the stability of Kis1. Previous research on the mammalian AMP-activated protein kinase complex has indicated that expression of all subunits was necessary for optimal stability of the entire complex, in both mammalian and yeast cells (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Daniel and Carling 2002</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Ye <em>et al.</em> 2014</a>). This indicates that when subunits are not part of the complex, <em>e.g.</em>, because of loss of their ability to interact with other subunits, their stability decreases. Hence, further research is needed to show whether the ASC domain of Kis1 is necessary for the protein’s inherent stability, or whether the instability of the truncated Kis1ΔASC is a result of the reduced interaction with its partner, Snf4.</p><p id="p-40">Second, we have confirmed and visualized the interaction of a membrane-bound receptor, Gpr1 and Gpa2. Full-length Gpr1 interacted with Gpa2 in the membrane, whereas the C-tail, devoid of the membrane-anchoring part, formed a complex with Gpa2 in the cytosol. Markedly, the BiFC signal of these interactions was much lower compared to Snf4-Kis1. As these differences could be due to our specific testing conditions, analyzing different environmental conditions using our BiFC system may help identify environmental cues that which influence Gpr1-Gpa2 coupling, and as such provide more information on the regulation of the PKA pathway by this GPCR. Still, these observations demonstrate that our BiFC assay is suitable for detection of both strong and weak interactions. However, a quantitative analysis using our microplate reader for selection of the optimal interacting combination was not suitable for weak BiFC signals. Therefore, depending on the detection limits of the microplate reader used, quantitative analysis of weak interactions may depend on fluorescence microscopic imaging.</p><p id="p-41">Finally, besides demonstrating interaction of the upstream membrane receptor system of the cAMP-PKA pathway, Gpr1-Gpa2, our BiFC assay proved to be useful in visualizing <em>in vivo</em>, for the first time, the interaction of PKA subunits Bcy1-Tpk1 and Bcy1-Tpk2 in <em>C. albicans</em>. We could observe that, in the exponential growth phase, both Tpk1 and Tpk2 interacted with Bcy1 in the cytoplasm, forming specific localized interaction complexes. Colocalization of subunits showed that Bcy1 and Tpk2 were situated in the cytoplasm, while only Tpk1 had a nucleocytoplasmic localization, indicating that Tpk1 specifically might have a role in transcriptional regulation of filamentation in our experimental conditions. Distinct roles of the Tpk1 and Tpk2 catalytic subunits have been described not only for morphogenesis in liquid <em>vs.</em> solid media (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bockmuhl <em>et al.</em> 2001</a>), but also in stress response and carbohydrate metabolism in <em>C. albicans</em> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Giacometti <em>et al.</em> 2009</a>). However, in the colocalization experiment, the subunits did not show inclusions corresponding to the BiFC interaction foci. Similar to the observed BiFC foci, previous studies identified lipofuscin-like autofluorescence bodies in <em>C. albicans</em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Diaz <em>et al.</em> 2005</a>). Still, the occurrence of these bodies usually precedes cell death and is accompanied by evident morphological changes, which were not observed with the BiFC strains. The BiFC strains, together with the controls and the colocalization strains, had similar morphology and filamentous growth, which confirms that the formation of foci was not due to stress induced by overproduction of Tpk subunits. On the other hand, formation of similar distinct cytoplasmic foci was already observed for various protein kinases in <em>S. cerevisiae</em> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Shah <em>et al.</em> 2014</a>). It is suggested that they serve as cellular compartments. Even though we did not observe the enrichment with the full-length C-terminal yEmVenus fusions, it is possible that a similar PKA signaling regulation also exists in <em>C. albicans</em>. Possibly this mechanism is specific for the PKA holoenzyme, and can therefore be visualized only by using the BiFC system, which stabilizes the Bcy1-Tpk holoenzyme.</p><p id="p-42">This specific localization of the PKA interaction complex could also indicate subcellular compartmentalization of PKA, which has been shown to be important for proper cAMP-mediated signaling (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Griffioen and Thevelein 2002</a>). In many multicellular organisms, A-kinase anchor proteins are known for fine-tuning PKA activity by targeting inactive PKA holoenzymes to specific intracellular locations (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Colledge and Scott 1999</a>). However, proteins with similar function have not been identified so far in yeast. Hence, for more detailed interpretation of our observations, further investigation needs to be done, to identify the functional relevance of these foci in <em>C. albicans</em> and confirm the presence of possible adaptor proteins.</p><p id="p-43">Although the BiFC assay proved to be useful for identification of PPI in <em>C. albicans</em>, the limitations of the tool need to be considered. To verify positive interactions, it is critical to have appropriate controls. This can be done by expressing one of the interacting proteins tagged with one fluorophore part and the other with the complementary part, or ideally abolishing the interaction domain, if it is known. In the latter case, it is important to determine the stability of the BiFC protein fusion lacking the interaction domain, as we demonstrated in the case of the Snf4-Kis1 interaction pair. An additional limitation of the BiFC complex formation is its irreversibility <em>in vitro</em>, which can limit assessment of dynamic interactions. However, this irreversible nature can aid in visualizing weak or transient interactions (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Miller <em>et al.</em> 2015</a>). One of the advantages of the BiFC assay is that it is suitable for genome-wide PPI analysis, as recently shown in a study of the SUMO interactome in <em>S. cerevisiae</em> (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Sung <em>et al.</em> 2013</a>). Similarly, the adaptation of BiFC plasmids for Gateway cloning could enable the use of the BiFC assay for genome-wide PPI studies using the recently established <em>Candida</em> ORFeome library (Carol Munro, University of Aberdeen, and Christophe d’Enfert, Institut Pasteur). Mating-competent NV- and/or CV- tagged collections of <em>C. albicans</em> strains could be crossed for high-throughput screening. In this case, only one set of plasmids for tagging would be feasible to use. Finally, we underscore the fact that there is no one ideal method for PPI, and validation of interactions by other methods is highly desirable, as shown in this study using the <em>Candida</em> two-hybrid method.</p><p id="p-44">In conclusion, we have successfully established an additional tool for PPI studies in <em>C. albicans</em>, by creating an optimized set of vectors allowing the use of BiFC in this organism. We validated our BiFC assay by demonstrating three interaction complexes <em>in vivo</em>, which were confirmed by <em>Candida</em> two-hybrid results. As a major advantage, our BiFC system allowed the visualization of the subcellular localization of the interaction complex, creating immense potential for elucidating novel (spatial) properties of molecular mechanisms operating in this important pathogen.</p></div><div class="section" id="sec-21"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">We thank Nico Vangoethem for generous help with preparation of the figures and Cindy Colombo for great technical assistance. We also thank Suzanne Noble for kindly providing the SN152 strain. This work was supported by the KU Leuven Research Fund (DBOF/12/019 to A.S., GOA/13/006 to P.V.D., and CREA/11/012 to H.T.); the Fund for Scientific Research Flanders (G.0D48.13 to P.V.D., 11P9816N to L.D., and 12C8915N to E.S.); and the Interuniversity Attraction Poles Program initiated by the Belgian Science Policy Office (IAP P7/28 to P.V.D.). The authors declare no conflicts of interest.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-46">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300149/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-47"><em>Communicating editor: J. Berman</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-05-25"><span class="received-label">Received </span>May 25, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-08-25"><span class="accepted-label">Accepted </span>August 25, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Subotic <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.1" data-doi="10.1111/j.1365-2958.2011.07859.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">F. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Characterization of a hyperactive Cyr1 mutant reveals new regulatory mechanisms for cellular cAMP levels in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">879</span>–<span class="cit-lpage">893</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DBai%26rft.auinit1%253DC.%26rft.volume%253D82%26rft.issue%253D4%26rft.spage%253D879%26rft.epage%253D893%26rft.atitle%253DCharacterization%2Bof%2Ba%2Bhyperactive%2BCyr1%2Bmutant%2Breveals%2Bnew%2Bregulatory%2Bmechanisms%2Bfor%2Bcellular%2BcAMP%2Blevels%2Bin%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2011.07859.x%26rft_id%253Dinfo%253Apmid%252F21992526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2011.07859.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21992526&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.2" data-doi="10.1046/j.1365-2958.2001.02688.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bockmuhl</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krishnamurthy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerads</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonneborn</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Distinct and redundant roles of the two protein kinase A isoforms Tpk1p and Tpk2p in morphogenesis and growth of <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">1243</span>–<span class="cit-lpage">1257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DBockm%25C3%25BChl%26rft.auinit1%253DD.%2BP.%26rft.volume%253D42%26rft.issue%253D5%26rft.spage%253D1243%26rft.epage%253D1257%26rft.atitle%253DDistinct%2Band%2Bredundant%2Broles%2Bof%2Bthe%2Btwo%2Bprotein%2Bkinase%2BA%2Bisoforms%2BTpk1p%2Band%2BTpk2p%2Bin%2Bmorphogenesis%2Band%2Bgrowth%2Bof%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2001.02688.x%26rft_id%253Dinfo%253Apmid%252F11886556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2001.02688.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11886556&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.3" data-doi="10.1534/genetics.109.101162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boysen</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fanning</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newberg</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Detection of protein-protein interactions through vesicle targeting.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101162%26rft_id%253Dinfo%253Apmid%252F19307607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTgyLzEvMzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEwLzM1MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.4" data-doi="10.1046/j.1365-2958.1999.01641.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Care</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trevethick</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binley</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sudbery</span>  <span class="cit-name-given-names">P. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The MET3 promoter: a new tool for <em>Candida albicans</em> molecular genetics.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">792</span>–<span class="cit-lpage">798</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DCare%26rft.auinit1%253DR.%2BS.%26rft.volume%253D34%26rft.issue%253D4%26rft.spage%253D792%26rft.epage%253D798%26rft.atitle%253DThe%2BMET3%2Bpromoter%253A%2Ba%2Bnew%2Btool%2Bfor%2BCandida%2Balbicans%2Bmolecular%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.1999.01641.x%26rft_id%253Dinfo%253Apmid%252F10564518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.1999.01641.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10564518&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084193300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.5" data-doi="10.1128/EC.3.1.190-199.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cassola</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parrot</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silberstein</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magee</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Passeron</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title"><em>Candida albicans</em> lacking the gene encoding the regulatory subunit of protein kinase A displays a defect in hyphal formation and an altered localization of the catalytic subunit.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">190</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DCassola%26rft.auinit1%253DA.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D190%26rft.epage%253D199%26rft.atitle%253DCandida%2Balbicans%2BLacking%2Bthe%2BGene%2BEncoding%2Bthe%2BRegulatory%2BSubunit%2Bof%2BProtein%2BKinase%2BA%2BDisplays%2Ba%2BDefect%2Bin%2BHyphal%2BFormation%2Band%2Ban%2BAltered%2BLocalization%2Bof%2Bthe%2BCatalytic%2BSubunit%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.3.1.190-199.2004%26rft_id%253Dinfo%253Apmid%252F14871949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiIzLzEvMTkwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zNTA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.6" data-doi="10.1016/S1087-1845(02)00520-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cloutier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castilla</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolduc</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zelada</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martineau</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The two isoforms of the cAMP-dependent protein kinase catalytic subunit are involved in the control of dimorphism in the human fungal pathogen <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DCloutier%26rft.auinit1%253DM.%26rft.volume%253D38%26rft.issue%253D1%26rft.spage%253D133%26rft.epage%253D141%26rft.atitle%253DThe%2Btwo%2Bisoforms%2Bof%2Bthe%2BcAMP-dependent%2Bprotein%2Bkinase%2Bcatalytic%2Bsubunit%2Bare%2Binvolved%2Bin%2Bthe%2Bcontrol%2Bof%2Bdimorphism%2Bin%2Bthe%2Bhuman%2Bfungal%2Bpathogen%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1087-1845%252802%252900520-0%26rft_id%253Dinfo%253Apmid%252F12553943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1087-1845(02)00520-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12553943&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.7" data-doi="10.1016/S0962-8924(99)01558-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colledge</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">AKAPs: from structure to function.</span> <abbr class="cit-jnl-abbrev">Trends Cell Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">216</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bcell%2Bbiology%26rft.stitle%253DTrends%2BCell%2BBiol%26rft.aulast%253DColledge%26rft.auinit1%253DM.%26rft.volume%253D9%26rft.issue%253D6%26rft.spage%253D216%26rft.epage%253D221%26rft.atitle%253DAKAPs%253A%2Bfrom%2Bstructure%2Bto%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0962-8924%252899%252901558-5%26rft_id%253Dinfo%253Apmid%252F10354567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0962-8924(99)01558-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10354567&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081758700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.8" data-doi="10.1074/jbc.M503719200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corvey</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koetter</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beckhaus</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hack</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hofmann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Carbon source-dependent assembly of the Snf1p kinase complex in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">280</span>: <span class="cit-fpage">25323</span>–<span class="cit-lpage">25330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M503719200%26rft_id%253Dinfo%253Apmid%252F15890650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODAvMjcvMjUzMjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEwLzM1MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.9" data-doi="10.1042/bj20020124"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daniel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carling</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Expression and regulation of the AMP-activated protein kinase-SNF1 (sucrose non-fermenting 1) kinase complexes in yeast and mammalian cells: studies using chimaeric catalytic subunits.</span> <abbr class="cit-jnl-abbrev">Biochem. J.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">638</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1042%252Fbj20020124%26rft_id%253Dinfo%253Apmid%252F11971761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6OToiMzY1LzMvNjI5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zNTA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.10" data-doi="10.1002/jemt.20161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diaz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polonelli</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conti</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messana</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabras</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mitochondrial alterations and autofluorescent conversion of <em>Candida albicans</em> induced by histatins.</span> <abbr class="cit-jnl-abbrev">Microsc. Res. Tech.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicroscopy%2Bresearch%2Band%2Btechnique%26rft.stitle%253DMicrosc%2BRes%2BTech%26rft.aulast%253DDiaz%26rft.auinit1%253DG.%26rft.volume%253D66%26rft.issue%253D5%26rft.spage%253D219%26rft.epage%253D228%26rft.atitle%253DMitochondrial%2Balterations%2Band%2Bautofluorescent%2Bconversion%2Bof%2BCandida%2Balbicans%2Binduced%2Bby%2Bhistatins.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjemt.20161%26rft_id%253Dinfo%253Apmid%252F15940680%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jemt.20161&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15940680&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.11" data-doi="10.4161/viru.19396"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fuller</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rhodes</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Protein kinase A and fungal virulence: a sinister side to a conserved nutrient sensing pathway.</span> <abbr class="cit-jnl-abbrev">Virulence</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirulence%26rft.volume%253D3%26rft.spage%253D109%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fviru.19396%26rft_id%253Dinfo%253Apmid%252F22460637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/viru.19396&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22460637&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305607700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.12" data-doi="10.1002/yea.1674"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerami-Nejad</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dulmage</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Additional cassettes for epitope and fluorescent fusion proteins in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DGerami-Nejad%26rft.auinit1%253DM.%26rft.volume%253D26%26rft.issue%253D7%26rft.spage%253D399%26rft.epage%253D406%26rft.atitle%253DAdditional%2Bcassettes%2Bfor%2Bepitope%2Band%2Bfluorescent%2Bfusion%2Bproteins%2Bin%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1674%26rft_id%253Dinfo%253Apmid%252F19504625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1674&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19504625&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268035200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.13" data-doi="10.1016/j.bbamcr.2005.11.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giacometti</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souto</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silberstein</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giasson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantore</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Expression levels and subcellular localization of Bcy1p in <em>Candida albicans</em> mutant strains devoid of one BCY1 allele results in a defective morphogenetic behavior.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1763</span>: <span class="cit-fpage">64</span>–<span class="cit-lpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DGiacometti%26rft.auinit1%253DR.%26rft.volume%253D1763%26rft.issue%253D1%26rft.spage%253D64%26rft.epage%253D72%26rft.atitle%253DExpression%2Blevels%2Band%2Bsubcellular%2Blocalization%2Bof%2BBcy1p%2Bin%2BCandida%2Balbicans%2Bmutant%2Bstrains%2Bdevoid%2Bof%2Bone%2BBCY1%2Ballele%2Bresults%2Bin%2Ba%2Bdefective%2Bmorphogenetic%2Bbehavior.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbamcr.2005.11.016%26rft_id%253Dinfo%253Apmid%252F16473139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbamcr.2005.11.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16473139&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.14" data-doi="10.1002/yea.1665"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giacometti</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kronberg</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biondi</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Passeron</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Catalytic isoforms Tpk1 and Tpk2 of <em>Candida albicans</em> PKA have non-redundant roles in stress response and glycogen storage.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DGiacometti%26rft.auinit1%253DR.%26rft.volume%253D26%26rft.issue%253D5%26rft.spage%253D273%26rft.epage%253D285%26rft.atitle%253DCatalytic%2Bisoforms%2BTpk1%2Band%2BTpk2%2Bof%2BCandida%2Balbicans%2BPKA%2Bhave%2Bnon-redundant%2Broles%2Bin%2Bstress%2Bresponse%2Band%2Bglycogen%2Bstorage.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1665%26rft_id%253Dinfo%253Apmid%252F19391100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1665&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19391100&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266270600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.15" data-doi="10.1002/yea.320110408"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gietz</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiestl</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willems</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woods</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Studies on the transformation of intact yeast cells by the LiAc/SS-DNA/PEG procedure.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DGietz%26rft.auinit1%253DR.%2BD.%26rft.volume%253D11%26rft.issue%253D4%26rft.spage%253D355%26rft.epage%253D360%26rft.atitle%253DStudies%2Bon%2Bthe%2Btransformation%2Bof%2Bintact%2Byeast%2Bcells%2Bby%2Bthe%2BLiAc%252FSS-DNA%252FPEG%2Bprocedure.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320110408%26rft_id%253Dinfo%253Apmid%252F7785336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320110408&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7785336&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QQ03300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-2" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.16" data-doi="10.1007/BF00328721"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gillum</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsay</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirsch</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Isolation of the <em>Candida albicans</em> gene for orotidine-5′-phosphate decarboxylase by complementation of <em>S. cerevisiae</em> ura3 and <em>E. coli</em> pyrF mutations.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DGillum%26rft.auinit1%253DA.%2BM.%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D179%26rft.epage%253D182%26rft.atitle%253DIsolation%2Bof%2Bthe%2BCandida%2Balbicans%2Bgene%2Bfor%2Borotidine-5%2527-phosphate%2Bdecarboxylase%2Bby%2Bcomplementation%2Bof%2BS.%2Bcerevisiae%2Bura3%2Band%2BE.%2Bcoli%2BpyrF%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00328721%26rft_id%253Dinfo%253Apmid%252F6394964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00328721&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6394964&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984AEM2900030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.17" data-doi="10.1007/s00294-002-0308-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffioen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thevelein</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Molecular mechanisms controlling the localisation of protein kinase A.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DGriffioen%26rft.auinit1%253DG.%26rft.volume%253D41%26rft.issue%253D4%26rft.spage%253D199%26rft.epage%253D207%26rft.atitle%253DMolecular%2Bmechanisms%2Bcontrolling%2Bthe%2Blocalisation%2Bof%2Bprotein%2Bkinase%2BA.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00294-002-0308-9%26rft_id%253Dinfo%253Apmid%252F12172960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00294-002-0308-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12172960&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177738100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.18" data-doi="10.1016/S1097-2765(02)00496-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chinenov</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerppola</span>  <span class="cit-name-given-names">T. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Visualization of interactions among bZIP and Rel family proteins in living cells using bimolecular fluorescence complementation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">798</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DHu%26rft.auinit1%253DC.%2BD.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D789%26rft.epage%253D798%26rft.atitle%253DVisualization%2Bof%2Binteractions%2Bamong%2BbZIP%2Band%2BRel%2Bfamily%2Bproteins%2Bin%2Bliving%2Bcells%2Busing%2Bbimolecular%2Bfluorescence%2Bcomplementation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252802%252900496-3%26rft_id%253Dinfo%253Apmid%252F11983170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(02)00496-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11983170&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175244400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.19" data-doi="10.1128/MCB.17.4.2099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The Snf1 protein kinase and its activating subunit, Snf4, interact with distinct domains of the Sip1/Sip2/Gal83 component in the kinase complex.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2099</span>–<span class="cit-lpage">2106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DJiang%26rft.auinit1%253DR.%26rft.volume%253D17%26rft.issue%253D4%26rft.spage%253D2099%26rft.epage%253D2106%26rft.atitle%253DThe%2BSnf1%2Bprotein%2Bkinase%2Band%2Bits%2Bactivating%2Bsubunit%252C%2BSnf4%252C%2Binteract%2Bwith%2Bdistinct%2Bdomains%2Bof%2Bthe%2BSip1%252FSip2%252FGal83%2Bcomponent%2Bin%2Bthe%2Bkinase%2Bcomplex%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.17.4.2099%26rft_id%253Dinfo%253Apmid%252F9121458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE3LzQvMjA5OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTAvMzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.20" data-doi="10.1039/b909638h"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kerppola</span>  <span class="cit-name-given-names">T. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Visualization of molecular interactions using bimolecular fluorescence complementation analysis: characteristics of protein fragment complementation.</span> <abbr class="cit-jnl-abbrev">Chem. Soc. Rev.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">2876</span>–<span class="cit-lpage">2886</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DChem%2BSoc%2BRev%26rft.aulast%253DKerppola%26rft.auinit1%253DT.%2BK.%26rft.volume%253D38%26rft.issue%253D10%26rft.spage%253D2876%26rft.epage%253D2886%26rft.atitle%253DVisualization%2Bof%2Bmolecular%2Binteractions%2Busing%2Bbimolecular%2Bfluorescence%2Bcomplementation%2Banalysis%253A%2Bcharacteristics%2Bof%2Bprotein%2Bfragment%2Bcomplementation.%26rft_id%253Dinfo%253Adoi%252F10.1039%252Fb909638h%26rft_id%253Dinfo%253Apmid%252F19771334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1039/b909638h&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19771334&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270032900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kodama</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Bimolecular fluorescence complementation (BiFC): a 5-year update and future perspectives.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechniques%26rft.volume%253D53%26rft.spage%253D285%26rft_id%253Dinfo%253Apmid%252F23148879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23148879&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311523600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.22" data-doi="10.1046/j.1365-2958.1999.01413.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kraakman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemaire</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teunissen</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donaton</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">A <em>Saccharomyces cerevisiae</em> G-protein coupled receptor, Gpr1, is specifically required for glucose activation of the cAMP pathway during the transition to growth on glucose.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1002</span>–<span class="cit-lpage">1012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DKraakman%26rft.auinit1%253DL.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1002%26rft.epage%253D1012%26rft.atitle%253DA%2BSaccharomyces%2Bcerevisiae%2BG-protein%2Bcoupled%2Breceptor%252C%2BGpr1%252C%2Bis%2Bspecifically%2Brequired%2Bfor%2Bglucose%2Bactivation%2Bof%2Bthe%2BcAMP%2Bpathway%2Bduring%2Bthe%2Btransition%2Bto%2Bgrowth%2Bon%2Bglucose.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.1999.01413.x%26rft_id%253Dinfo%253Apmid%252F10361302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.1999.01413.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10361302&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081027300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">L. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gelvin</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Bimolecular fluorescence complementation for imaging protein interactions in plant hosts of microbial pathogens.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">1197</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%26rft.volume%253D1197%26rft.spage%253D185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.24" data-doi="10.1091/mbc.E04-09-0780"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maidan</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Rop</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serneels</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Exler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rupp</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The G protein-coupled receptor Gpr1 and the Galpha protein Gpa2 act through the cAMP-protein kinase A pathway to induce morphogenesis in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1971</span>–<span class="cit-lpage">1986</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E04-09-0780%26rft_id%253Dinfo%253Apmid%252F15673611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE2LzQvMTk3MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTAvMzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.25" data-doi="10.1016/j.jmb.2015.03.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">W.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">H.-O.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Bimolecular fluorescence complementation (BiFC) analysis: advances and recent applications for genome-wide interaction studies.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">427</span>: <span class="cit-fpage">2039</span>–<span class="cit-lpage">2055</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BBiol.%26rft.volume%253D427%26rft.spage%253D2039%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2015.03.005%26rft_id%253Dinfo%253Apmid%252F25772494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2015.03.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25772494&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.26" data-doi="10.1128/EC.3.4.919-931.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miwa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takagi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinozaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yun</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schell</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Gpr1, a putative G-protein-coupled receptor, regulates morphogenesis and hypha formation in the pathogenic fungus <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">919</span>–<span class="cit-lpage">931</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DMiwa%26rft.auinit1%253DT.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253D919%26rft.epage%253D931%26rft.atitle%253DGpr1%252C%2Ba%2BPutative%2BG-Protein-Coupled%2BReceptor%252C%2BRegulates%2BMorphogenesis%2Band%2BHypha%2BFormation%2Bin%2Bthe%2BPathogenic%2BFungus%2BCandida%2Balbicans%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.3.4.919-931.2004%26rft_id%253Dinfo%253Apmid%252F15302825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiIzLzQvOTE5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zNTA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.27" data-doi="10.1128/EC.4.2.298-309.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Strains and strategies for large-scale gene deletion studies of the diploid human fungal pathogen <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">298</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DNoble%26rft.auinit1%253DS.%2BM.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D298%26rft.epage%253D309%26rft.atitle%253DStrains%2Band%2BStrategies%2Bfor%2BLarge-Scale%2BGene%2BDeletion%2BStudies%2Bof%2Bthe%2BDiploid%2BHuman%2BFungal%2BPathogen%2BCandida%2Balbicans%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.4.2.298-309.2005%26rft_id%253Dinfo%253Apmid%252F15701792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI0LzIvMjk4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zNTA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.28" data-doi="10.1146/annurev.genet.41.042007.170146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetics of <em>Candida albicans</em>, a diploid human fungal pathogen.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DNoble%26rft.auinit1%253DS.%2BM.%26rft.volume%253D41%26rft.spage%253D193%26rft.epage%253D211%26rft.atitle%253DGenetics%2Bof%2BCandida%2Balbicans%252C%2Ba%2Bdiploid%2Bhuman%2Bfungal%2Bpathogen.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.41.042007.170146%26rft_id%253Dinfo%253Apmid%252F17614788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.41.042007.170146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17614788&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252359500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.29" data-doi="10.1128/EC.00075-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palzer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bantel</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kazenwadel</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rupp</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An expanded genetic code in <em>Candida albicans</em> to study protein-protein interactions in vivo.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">816</span>–<span class="cit-lpage">827</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00075-13%26rft_id%253Dinfo%253Apmid%252F23543672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiIxMi82LzgxNiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTAvMzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.30" data-doi="10.3109/1040841X.2013.813904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poulain</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title"><em>Candida albicans</em>, plasticity and pathogenesis.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Microbiol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">208</span>–<span class="cit-lpage">217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrit.%2BRev.%2BMicrobiol.%26rft.volume%253D41%26rft.spage%253D208%26rft_id%253Dinfo%253Adoi%252F10.3109%252F1040841X.2013.813904%26rft_id%253Dinfo%253Apmid%252F23962107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3109/1040841X.2013.813904&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23962107&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.31" data-doi="10.1093/nar/23.9.1481"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuite</span>  <span class="cit-name-given-names">M. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">The CUG codon is decoded in vivo as serine and not leucine in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1481</span>–<span class="cit-lpage">1486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSantos%26rft.auinit1%253DM.%2BA.S.%26rft.volume%253D23%26rft.issue%253D9%26rft.spage%253D1481%26rft.epage%253D1486%26rft.atitle%253DThe%2BCUG%2Bcodon%2Bis%2Bdecoded%2Bin%2Bvivo%2Bas%2Bserine%2Band%2Bnot%2Bleucine%2Bin%2BCandida%2Balbicans%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F23.9.1481%26rft_id%253Dinfo%253Apmid%252F7784200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/23.9.1481&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7784200&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QZ47700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.32" data-doi="10.1186/1471-2164-14-842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaekel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Morphogenesis-regulated localization of protein kinase A to genomic sites in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D842%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-842%26rft_id%253Dinfo%253Apmid%252F24289325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24289325&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.33" data-doi="10.1534/genetics.114.172031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nostramo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varia</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klett</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Protein kinases are associated with multiple, distinct cytoplasmic granules in quiescent yeast cells.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1495</span>–<span class="cit-lpage">1512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.172031%26rft_id%253Dinfo%253Apmid%252F25342717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE0OTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEwLzM1MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.34" data-doi="10.1038/nmeth819"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shaner</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinbach</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsien</span>  <span class="cit-name-given-names">R. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A guide to choosing fluorescent proteins.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">905</span>–<span class="cit-lpage">909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DShaner%26rft.auinit1%253DN.%2BC.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253D905%26rft.epage%253D909%26rft.atitle%253DA%2Bguide%2Bto%2Bchoosing%2Bfluorescent%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth819%26rft_id%253Dinfo%253Apmid%252F16299475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth819&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16299475&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233767700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.35" data-doi="10.1002/yea.1130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sheff</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorn</span>  <span class="cit-name-given-names">K. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Optimized cassettes for fluorescent protein tagging in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">661</span>–<span class="cit-lpage">670</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DSheff%26rft.auinit1%253DM.%2BA.%26rft.volume%253D21%26rft.issue%253D8%26rft.spage%253D661%26rft.epage%253D670%26rft.atitle%253DOptimized%2Bcassettes%2Bfor%2Bfluorescent%2Bprotein%2Btagging%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1130%26rft_id%253Dinfo%253Apmid%252F15197731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15197731&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222156700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.36" data-doi="10.1093/emboj/16.8.1982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoldt</span>  <span class="cit-name-given-names">V. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonneborn</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leuker</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Efg1p, an essential regulator of morphogenesis of the human pathogen <em>Candida albicans</em>, is a member of a conserved class of bHLH proteins regulating morphogenetic processes in fungi.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1982</span>–<span class="cit-lpage">1991</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DStoldt%26rft.auinit1%253DV.%2BR.%26rft.volume%253D16%26rft.issue%253D8%26rft.spage%253D1982%26rft.epage%253D1991%26rft.atitle%253DEfg1p%252C%2Ban%2Bessential%2Bregulator%2Bof%2Bmorphogenesis%2Bof%2Bthe%2Bhuman%2Bpathogen%2BCandida%2Balbicans%252C%2Bis%2Ba%2Bmember%2Bof%2Ba%2Bconserved%2Bclass%2Bof%2BbHLH%2Bproteins%2Bregulating%2Bmorphogenetic%2Bprocesses%2Bin%2Bfungi%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F16.8.1982%26rft_id%253Dinfo%253Apmid%252F9155024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIxNi84LzE5ODIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEwLzM1MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.37" data-doi="10.1093/nar/gkq725"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stynen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Dijck</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tournu</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A CUG codon adapted two-hybrid system for the pathogenic fungus <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">e184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq725%26rft_id%253Dinfo%253Apmid%252F20719741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq725&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20719741&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.38" data-doi="10.1038/nrmicro2636"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sudbery</span>  <span class="cit-name-given-names">P. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Growth of <em>Candida albicans</em> hyphae.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">748</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DSudbery%26rft.auinit1%253DP.%2BE.%26rft.volume%253D9%26rft.issue%253D10%26rft.spage%253D737%26rft.epage%253D748%26rft.atitle%253DGrowth%2Bof%2BCandida%2Balbicans%2Bhyphae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2636%26rft_id%253Dinfo%253Apmid%252F21844880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2636&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21844880&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.39" data-doi="10.1101/gr.148346.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">Y. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide bimolecular fluorescence complementation analysis of SUMO interactome in yeast.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">736</span>–<span class="cit-lpage">746</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.148346.112%26rft_id%253Dinfo%253Apmid%252F23403034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIzLzQvNzM2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zNTA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.40" data-doi="10.1111/j.1365-313X.2004.02219.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaban</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schutze</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batistic</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weckermann</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Visualization of protein interactions in living plant cells using bimolecular fluorescence complementation.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">428</span>–<span class="cit-lpage">438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DWalter%26rft.auinit1%253DM.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D428%26rft.epage%253D438%26rft.atitle%253DVisualization%2Bof%2Bprotein%2Binteractions%2Bin%2Bliving%2Bplant%2Bcells%2Busing%2Bbimolecular%2Bfluorescence%2Bcomplementation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2004.02219.x%26rft_id%253Dinfo%253Apmid%252F15469500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2004.02219.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15469500&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224397700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.41" data-doi="10.1016/j.chembiol.2009.03.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parrish</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Expanding the genetic code for biological studies.</span> <abbr class="cit-jnl-abbrev">Chem. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChemistry%2B%2526%2Bbiology%26rft.stitle%253DChem%2BBiol%26rft.aulast%253DWang%26rft.auinit1%253DQ.%26rft.volume%253D16%26rft.issue%253D3%26rft.spage%253D323%26rft.epage%253D336%26rft.atitle%253DExpanding%2Bthe%2Bgenetic%2Bcode%2Bfor%2Bbiological%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chembiol.2009.03.001%26rft_id%253Dinfo%253Apmid%252F19318213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chembiol.2009.03.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19318213&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264859200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3509.42" data-doi="10.1016/j.febslet.2014.04.039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bendrioua</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmena</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Salcedo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dahl</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The mammalian AMP-activated protein kinase complex mediates glucose regulation of gene expression in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">588</span>: <span class="cit-fpage">2070</span>–<span class="cit-lpage">2077</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLett.%26rft.volume%253D588%26rft.spage%253D2070%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2014.04.039%26rft_id%253Dinfo%253Apmid%252F24815694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2014.04.039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24815694&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/10/3509.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/10/3491" title="Genome-Wide Association Mapping of Stem Rust Resistance in Hordeum vulgare subsp. spontaneum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/10/3521" title="Overexpression of miRNA-9 Generates Muscle Hypercontraction Through Translational Repression of Troponin-T in Drosophila melanogaster Indirect Flight Muscles" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/10"><span>Volume 7 Issue 10, October 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/10.cover-source.jpg?itok=lurMO2QE" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3509" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TomIKQ_f_DNmfi0WCAijw4VMiAYh-9uG94c5IiDgSZM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/10/3509" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/10/3509" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/10/3509" class="active" data-icon-position="" data-hide-link-title="0">A Bimolecular Fluorescence Complementation Tool for Identification of Protein-Protein Interactions in Candida albicans</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/486780" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Bk3Cfkg6DM4OVYsbmV3YyygjrTq-GWV4DRCK81gGUGI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/10/3509.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/10/3509" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-xZAXtmRuSNzgT1_cIV2nnV1_7DFmqdEZPJO9xBJcxIA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/10/3509" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/10/3509.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486780" id="node486780--2" data-pisa="ggg;7/10/3509" data-pisa-master="ggg;g3.117.300149" data-apath="/ggg/7/10/3509.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Bimolecular Fluorescence Complementation Tool for Identification of Protein-Protein Interactions in <em>Candida albicans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ana Subotić</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5215-0922" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Erwin Swinnen</span>, <span class="highwire-citation-author" data-delta="2">Liesbeth Demuyser</span>, <span class="highwire-citation-author" data-delta="3">Herlinde De Keersmaecker</span>, <span class="highwire-citation-author" data-delta="4">Hideaki Mizuno</span>, <span class="highwire-citation-author" data-delta="5">Hélène Tournu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-1542-897X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick Van Dijck</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3509-3520;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300149 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486780--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ana Subotić </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BSuboti%C4%87%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Suboti%C4%87%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAna%2BSuboti%25C4%2587%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Erwin Swinnen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Functional Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erwin%2BSwinnen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Swinnen%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErwin%2BSwinnen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5215-0922" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Erwin Swinnen</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Liesbeth Demuyser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liesbeth%2BDemuyser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Demuyser%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALiesbeth%2BDemuyser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Herlinde De Keersmaecker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Laboratory of Biomolecular Network Dynamics, Biochemistry, Molecular and Structural Biology Section, Department of Chemistry, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Herlinde%2BDe%2BKeersmaecker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De%20Keersmaecker%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHerlinde%2BDe%2BKeersmaecker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hideaki Mizuno </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Laboratory of Biomolecular Network Dynamics, Biochemistry, Molecular and Structural Biology Section, Department of Chemistry, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hideaki%2BMizuno%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mizuno%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHideaki%2BMizuno%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Hélène Tournu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H%C3%A9l%C3%A8ne%2BTournu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tournu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH%25C3%25A9l%25C3%25A8ne%2BTournu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Patrick Van Dijck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BVan%2BDijck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Dijck%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BVan%2BDijck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1542-897X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick Van Dijck</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:patrick.vandijck@kuleuven.vib.be" class="" data-icon-position="" data-hide-link-title="0">patrick.vandijck@kuleuven.vib.be</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/10/3509" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/10/3509&amp;title=A%20Bimolecular%20Fluorescence%20Complementation%20Tool%20for%20Identification%20of%20Protein-Protein%20Interactions%20in%20Candida%20albicans" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/10/3509&amp;title=A%20Bimolecular%20Fluorescence%20Complementation%20Tool%20for%20Identification%20of%20Protein-Protein%20Interactions%20in%20Candida%20albicans" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/10/3509&amp;title=A%20Bimolecular%20Fluorescence%20Complementation%20Tool%20for%20Identification%20of%20Protein-Protein%20Interactions%20in%20Candida%20albicans" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/10/3509&amp;text=A%20Bimolecular%20Fluorescence%20Complementation%20Tool%20for%20Identification%20of%20Protein-Protein%20Interactions%20in%20Candida%20albicans" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/10/3509&amp;title=A%20Bimolecular%20Fluorescence%20Complementation%20Tool%20for%20Identification%20of%20Protein-Protein%20Interactions%20in%20Candida%20albicans" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/10/3509&amp;t=A%20Bimolecular%20Fluorescence%20Complementation%20Tool%20for%20Identification%20of%20Protein-Protein%20Interactions%20in%20Candida%20albicans" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/10/3509&amp;title=A%20Bimolecular%20Fluorescence%20Complementation%20Tool%20for%20Identification%20of%20Protein-Protein%20Interactions%20in%20Candida%20albicans" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/10/3509&amp;title=A%20Bimolecular%20Fluorescence%20Complementation%20Tool%20for%20Identification%20of%20Protein-Protein%20Interactions%20in%20Candida%20albicans" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/486780/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486780" id="node486780--3" data-pisa="ggg;7/10/3509" data-pisa-master="ggg;g3.117.300149" data-apath="/ggg/7/10/3509.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Bimolecular Fluorescence Complementation Tool for Identification of Protein-Protein Interactions in <em>Candida albicans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ana Subotić</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5215-0922" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Erwin Swinnen</span>, <span class="highwire-citation-author" data-delta="2">Liesbeth Demuyser</span>, <span class="highwire-citation-author" data-delta="3">Herlinde De Keersmaecker</span>, <span class="highwire-citation-author" data-delta="4">Hideaki Mizuno</span>, <span class="highwire-citation-author" data-delta="5">Hélène Tournu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-1542-897X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick Van Dijck</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3509-3520;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300149 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486780--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ana Subotić </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BSuboti%C4%87%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Suboti%C4%87%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAna%2BSuboti%25C4%2587%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Erwin Swinnen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Functional Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erwin%2BSwinnen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Swinnen%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErwin%2BSwinnen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5215-0922" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Erwin Swinnen</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Liesbeth Demuyser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liesbeth%2BDemuyser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Demuyser%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALiesbeth%2BDemuyser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Herlinde De Keersmaecker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Laboratory of Biomolecular Network Dynamics, Biochemistry, Molecular and Structural Biology Section, Department of Chemistry, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Herlinde%2BDe%2BKeersmaecker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De%20Keersmaecker%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHerlinde%2BDe%2BKeersmaecker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hideaki Mizuno </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Laboratory of Biomolecular Network Dynamics, Biochemistry, Molecular and Structural Biology Section, Department of Chemistry, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hideaki%2BMizuno%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mizuno%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHideaki%2BMizuno%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Hélène Tournu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H%C3%A9l%C3%A8ne%2BTournu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tournu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH%25C3%25A9l%25C3%25A8ne%2BTournu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Patrick Van Dijck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>VIB-KU Leuven Center for Microbiology, KU Leuven, 3001 Leuven, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Molecular Cell Biology, Institute of Botany and Microbiology, KU Leuven, 3001 Leuven, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BVan%2BDijck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Dijck%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BVan%2BDijck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1542-897X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick Van Dijck</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:patrick.vandijck@kuleuven.vib.be" class="" data-icon-position="" data-hide-link-title="0">patrick.vandijck@kuleuven.vib.be</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3509&amp;rft.epage=3520&amp;rft.atitle=A%20Bimolecular%20Fluorescence%20Complementation%20Tool%20for%20Identification%20of%20Protein-Protein%20Interactions%20in%20Candida%20albicans&amp;rft.volume=7&amp;rft.issue=10&amp;rft.date=2017-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Suboti%C4%87%2C+Ana&amp;rft.au=Swinnen%2C+Erwin&amp;rft.au=Demuyser%2C+Liesbeth&amp;rft.au=De+Keersmaecker%2C+Herlinde&amp;rft.au=Mizuno%2C+Hideaki&amp;rft.au=Tournu%2C+H%C3%A9l%C3%A8ne&amp;rft.au=Van+Dijck%2C+Patrick"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/486780/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/486780/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/486780/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/486780/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/486780/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/486780/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/486780/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/486780/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/486780/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/486780/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/486780/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/486780/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/10/3509" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/10/3509.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/10/3509.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/10/3509.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/10/3509.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
