<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/10/3543" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/10/3543.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/10/3543.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/10/3543.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/10/3543/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/10/3543.atom" />
<meta name="HW.pisa" content="ggg;7/10/3543" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Relatedness Strengthens Genetic Connectedness Across Management Units" />
<meta name="DC.Identifier" content="10.1534/g3.117.300151" />
<meta name="DC.Date" content="2017-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Yu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genetic connectedness refers to a measure of genetic relatedness across management units ( e.g. , herds and flocks). With the presence of high genetic connectedness in management units, best linear unbiased prediction (BLUP) is known to provide reliable comparisons between estimated genetic values. Genetic connectedness has been studied for pedigree-based BLUP; however, relatively little attention has been paid to using genomic information to measure connectedness. In this study, we assessed genome-based connectedness across management units by applying prediction error variance of difference (PEVD), coefficient of determination (CD), and prediction error correlation r to a combination of computer simulation and real data (mice and cattle). We found that genomic information (![Formula][1]&lt;/img&gt;) increased the estimate of connectedness among individuals from different management units compared to that based on pedigree (![Formula][2]&lt;/img&gt;). A disconnected design benefited the most. In both datasets, PEVD and CD statistics inferred increased connectedness across units when using ![Formula][3]&lt;/img&gt;- rather than ![Formula][4]&lt;/img&gt;-based relatedness, suggesting stronger connectedness. With r once using allele frequencies equal to one-half or scaling ![Formula][5]&lt;/img&gt; to values between 0 and 2, which is intrinsic to ![Formula][6]&lt;/img&gt; connectedness also increased with genomic information. However, PEVD occasionally increased, and r decreased when obtained using the alternative form of ![Formula][7]&lt;/img&gt; instead suggesting less connectedness. Such inconsistencies were not found with CD. We contend that genomic relatedness strengthens measures of genetic connectedness across units and has the potential to aid genomic evaluation of livestock species.

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif
 [3]: /embed/mml-math-3.gif
 [4]: /embed/mml-math-4.gif
 [5]: /embed/mml-math-5.gif
 [6]: /embed/mml-math-6.gif
 [7]: /embed/mml-math-7.gif" />
<meta name="DC.Contributor" content="Haipeng Yu" />
<meta name="DC.Contributor" content="Matthew L. Spangler" />
<meta name="DC.Contributor" content="Ronald M. Lewis" />
<meta name="DC.Contributor" content="Gota Morota" />
<meta name="article:published_time" content="2017-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genomic Relatedness Strengthens Genetic Connectedness Across Management Units" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genetic connectedness refers to a measure of genetic relatedness across management units (&lt;i&gt;e.g.&lt;/i&gt;, herds and flocks). With the presence of high genetic connectedness in management units, best linear unbiased prediction (BLUP) is known to provide reliable comparisons between estimated genetic values. Genetic connectedness has been studied for pedigree-based BLUP; however, relatively little attention has been paid to using genomic information to measure connectedness. In this study, we assessed genome-based connectedness across management units by applying prediction error variance of difference (PEVD), coefficient of determination (CD), and prediction error correlation r to a combination of computer simulation and real data (mice and cattle). We found that genomic information (&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me1&quot; overflow=&quot;scroll&quot;&gt;&lt;mi mathvariant=&quot;bold&quot;&gt;G&lt;/mi&gt;&lt;/math&gt;) increased the estimate of connectedness among individuals from different management units compared to that based on pedigree (&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me2&quot; overflow=&quot;scroll&quot;&gt;&lt;mi mathvariant=&quot;bold&quot;&gt;A&lt;/mi&gt;&lt;/math&gt;). A disconnected design benefited the most. In both datasets, PEVD and CD statistics inferred increased connectedness across units when using &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me3&quot; overflow=&quot;scroll&quot;&gt;&lt;mi mathvariant=&quot;bold&quot;&gt;G&lt;/mi&gt;&lt;/math&gt;- rather than &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me4&quot; overflow=&quot;scroll&quot;&gt;&lt;mi mathvariant=&quot;bold&quot;&gt;A&lt;/mi&gt;&lt;/math&gt;-based relatedness, suggesting stronger connectedness. With r once using allele frequencies equal to one-half or scaling &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me5&quot; overflow=&quot;scroll&quot;&gt;&lt;mi mathvariant=&quot;bold&quot;&gt;G&lt;/mi&gt;&lt;/math&gt; to values between 0 and 2, which is intrinsic to &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me6&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;mi mathvariant=&quot;bold&quot;&gt;A&lt;/mi&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt; connectedness also increased with genomic information. However, PEVD occasionally increased, and r decreased when obtained using the alternative form of &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me7&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;mi mathvariant=&quot;bold&quot;&gt;G&lt;/mi&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt; instead suggesting less connectedness. Such inconsistencies were not found with CD. We contend that genomic relatedness strengthens measures of genetic connectedness across units and has the potential to aid genomic evaluation of livestock species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/10/01" />
<meta name="citation_mjid" content="ggg;7/10/3543" />
<meta name="citation_id" content="7/10/3543" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/10/3543" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/10/3543.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/10/3543.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/10/3543.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300151" />
<meta name="citation_pmid" content="28860185" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3543" />
<meta name="citation_lastpage" content="3556" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Haipeng Yu" />
<meta name="citation_author_institution" content="Department of Animal Science, University of Nebraska-Lincoln, Nebraska 68583" />
<meta name="citation_author" content="Matthew L. Spangler" />
<meta name="citation_author_institution" content="Department of Animal Science, University of Nebraska-Lincoln, Nebraska 68583" />
<meta name="citation_author" content="Ronald M. Lewis" />
<meta name="citation_author_institution" content="Department of Animal Science, University of Nebraska-Lincoln, Nebraska 68583" />
<meta name="citation_author" content="Gota Morota" />
<meta name="citation_author_institution" content="Department of Animal Science, University of Nebraska-Lincoln, Nebraska 68583" />
<meta name="citation_author_email" content="morota@unl.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3567-6911" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=O. F. Christensen;citation_author=M. S. Lund;citation_title=Genomic prediction when some animals are not genotyped.;citation_pages=2-2;citation_volume=42;citation_year=2010;citation_pmid=20105297;citation_doi=10.1186/1297-9686-42-2" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=L. S. Eikje;citation_author=R. M. Lewis;citation_title=Strong connectedness within Norwegian Cheviot and Fur sheep ram circles allows reliable estimation of breeding values.;citation_pages=3322-3330;citation_volume=93;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=R. Fernando;citation_author=M. Grossman;citation_title=Marker assisted selection using best linear unbiased prediction.;citation_pages=467-477;citation_volume=21;citation_year=1989;citation_doi=10.1051/gse:19890407" />
<meta name="citation_reference" content="citation_journal_title=Trans. R. Soc. Edinb.;citation_author=R. A. Fisher;citation_title=The correlation between relatives on the supposition of Mendelian inheritance.;citation_pages=399-433;citation_volume=52;citation_year=1918;citation_doi=10.1017/S0080456800012163" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. N. Fouilloux;citation_author=V. Clément;citation_author=D. Laloë;citation_title=Measuring connectedness among herds in mixed linear models: from theory to practice in large-sized genetic evaluations.;citation_pages=145-159;citation_volume=40;citation_year=2008;citation_issue=2;citation_pmid=18298932" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=J. L. Foulley;citation_author=E. Hanocq;citation_author=D. Boichard;citation_title=A criterion for measuring the degree of connectedness in linear models of genetic evaluation.;citation_pages=315-330;citation_volume=24;citation_year=1992;citation_doi=10.1051/gse:19920403" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=M. Goddard;citation_title=Genomic selection: prediction of accuracy and maximisation of long term response.;citation_pages=245-257;citation_volume=136;citation_year=2009;citation_pmid=18704696;citation_doi=10.1007/s10709-008-9308-0" />
<meta name="citation_reference" content="citation_title=Applications of Linear Models in Animal Breeding;citation_author=C. R. Henderson;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=B. S. Weir;citation_title=Variation in actual relationship as a consequence of Mendelian sampling and linkage.;citation_pages=47-64;citation_volume=93;citation_year=2011;citation_issue=1;citation_pmid=21226974;citation_doi=10.1017/S0016672310000480" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Isidro;citation_author=J.-L. Jannink;citation_author=D. Akdemir;citation_author=J. Poland;citation_author=N. Heslot;citation_title=Training set optimization under population structure in genomic selection.;citation_pages=145;citation_volume=128;citation_year=2015;citation_pmid=25367380;citation_doi=10.1007/s00122-014-2418-4" />
<meta name="citation_reference" content="citation_title=Finding Groups in Data: An Introduction to Cluster Analysis;citation_author=L. Kaufman;citation_author=P. J. Rousseeuw;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=B. W. Kennedy;citation_author=D. Trus;citation_title=Considerations on genetic connectedness between management units under an animal model;citation_pages=2341-2352;citation_volume=71;citation_year=1993;citation_issue=9;citation_pmid=8407646" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=L. A. Kuehn;citation_author=R. M. Lewis;citation_author=D. R. Notter;citation_title=Managing the risk of comparing estimated breeding values across flocks or herds through connectedness: a review and application.;citation_pages=225-247;citation_volume=39;citation_year=2007;citation_issue=3;citation_pmid=17433239" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=L. A. Kuehn;citation_author=D. R. Notter;citation_author=G. J. Nieuwhof;citation_author=R. M. Lewis;citation_title=Changes in connectedness over time in alternative sheep sire referencing schemes.;citation_volume=86;citation_year=2008;citation_pmid=18073292;citation_doi=10.2527/jas.2007-0256" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=L. A. Kuehn;citation_author=R. M. Lewis;citation_author=D. R. Notter;citation_title=Connectedness in Targhee and Suffolk flocks participating in the United States national sheep improvement program.;citation_volume=87;citation_year=2009;citation_pmid=18849390;citation_doi=10.2527/jas.2008-1092" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. Laloë;citation_title=Precision and information in linear models of genetic evaluation.;citation_pages=557;citation_volume=25;citation_year=1993;citation_doi=10.1051/gse:19930604" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. Laloë;citation_author=F. Phocas;citation_author=F. Ménissier;citation_title=Considerations on measures of precision and connectedness in mixed linear models of genetic evaluation.;citation_pages=359;citation_volume=28;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Legarra;citation_author=C. Robert-Granié;citation_author=E. Manfredi;citation_author=J.-M. Elsen;citation_title=Performance of genomic selection in mice.;citation_volume=180;citation_year=2008;citation_pmid=18757934;citation_doi=10.1534/genetics.108.088575" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=A. Legarra;citation_author=I. Aguilar;citation_author=I. Misztal;citation_title=A relationship matrix including full pedigree and genomic information;citation_pages=4656-4663;citation_volume=92;citation_year=2009;citation_issue=9;citation_pmid=19700729;citation_doi=10.3168/jds.2009-2061" />
<meta name="citation_reference" content="citation_title=Proceedings of the British Society of Animal Science;citation_title=Assessing connectedness in across-flock genetic evaluations;citation_author=R. M. Lewis;citation_author=R. E. Crump;citation_author=G. Simm;citation_author=R. Thompson;citation_pages=121-122;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Maenhout;citation_author=B. De Baets;citation_author=G. Haesaert;citation_title=Graph-based data selection for the construction of genomic prediction models.;citation_volume=185;citation_year=2010;citation_pmid=20479144;citation_doi=10.1534/genetics.110.116426" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=M. Momen;citation_author=A. A. Mehrgardi;citation_author=A. Sheikhy;citation_author=A. Esmailizadeh;citation_author=M. A. Fozi;citation_title=A predictive assessment of genetic correlations between traits in chickens using markers.;citation_pages=16;citation_volume=49;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. E. Powell;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Reconciling the analysis of IBD and IBS in complex trait studies.;citation_pages=800-805;citation_volume=11;citation_year=2010;citation_issue=11;citation_pmid=20877324;citation_doi=10.1038/nrg2865" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. Pszczola;citation_author=T. Strabel;citation_author=J. A. M. van Arendonk;citation_author=M. P. L. Calus;citation_title=The impact of genotyping different groups of animals on accuracy when moving from traditional to genomic selection.;citation_pages=5412-5421;citation_volume=95;citation_year=2012;citation_issue=9;citation_pmid=22916948" />
<meta name="citation_reference" content="citation_journal_title=J. Math. Model. Algorithms;citation_author=A. P. Reynolds;citation_author=G. Richards;citation_author=B. de la Iglesia;citation_author=V. J. Rayward-Smith;citation_title=Clustering rules: a comparison of partitioning and hierarchical clustering algorithms.;citation_pages=475;citation_volume=5;citation_year=2006;citation_doi=10.1007/s10852-005-9022-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Rincent;citation_author=D. Laloë;citation_author=S. Nicolas;citation_author=T. Altmann;citation_author=D. Brunel;citation_title=Maximizing the reliability of genomic selection by optimizing the calibration set of reference individuals: comparison of methods in two diverse groups of maize inbreds (Zea mays L.).;citation_volume=192;citation_year=2012;citation_pmid=22865733;citation_doi=10.1534/genetics.112.141473" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=L. C. Solberg;citation_author=W. Valdar;citation_author=D. Gauguier;citation_author=G. Nunez;citation_author=A. Taylor;citation_title=A protocol for high-throughput phenotyping, suitable for quantitative trait analysis in mice.;citation_pages=129-146;citation_volume=17;citation_year=2006;citation_issue=2;citation_pmid=16465593;citation_doi=10.1007/s00335-005-0112-1" />
<meta name="citation_reference" content="citation_journal_title=Conserv. Genet.;citation_author=M. Toro;citation_author=C. Barragán;citation_author=C. Óvilo;citation_author=J. Rodrigañez;citation_author=C. Rodriguez;citation_title=Estimation of coancestry in Iberian pigs using molecular markers.;citation_pages=309-320;citation_volume=3;citation_year=2002;citation_doi=10.1023/A:1019921131171" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=M. A. Toro;citation_author=L. A. García-Cortés;citation_author=A. Legarra;citation_title=A note on the rationale for estimating genealogical coancestry from molecular markers.;citation_pages=27;citation_volume=43;citation_year=2011;citation_doi=10.1186/1297-9686-43-27" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=W. Valdar;citation_author=L. C. Solberg;citation_author=D. Gauguier;citation_author=S. Burnett;citation_author=P. Klenerman;citation_title=Genome-wide genetic association of complex traits in heterogeneous stock mice.;citation_pages=879-887;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16832355;citation_doi=10.1038/ng1840" />
<meta name="citation_reference" content="citation_journal_title=Interbull Bull.;citation_author=P. M. VanRaden;citation_title=Genomic measures of relationship and inbreeding.;citation_pages=3336;citation_volume=37;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=Z. G. Vitezica;citation_author=I. Aguilar;citation_author=I. Misztal;citation_author=A. Legarra;citation_title=Bias in genomic predictions for populations under selection.;citation_pages=357-366;citation_volume=93;citation_year=2011;citation_issue=5;citation_pmid=21767459;citation_doi=10.1017/S001667231100022X" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Wright;citation_title=SYSTEMS OF MATING. I. THE BIOMETRIC RELATIONS BETWEEN PARENT AND OFFSPRING;citation_pages=111-123;citation_volume=6;citation_year=1921;citation_issue=2;citation_pmid=17245958" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=S. Wright;citation_title=Coefficients of inbreeding and relationship.;citation_pages=330-338;citation_volume=56;citation_year=1922;citation_doi=10.1086/279872" />
<meta name="citation_reference" content="citation_title=Methods Mol. Biol.;citation_author=J. Yang;citation_author=S. H. Lee;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Genome-wide complex trait analysis (GCTA): methods, data analyses, and interpretations.;citation_pages=215-236;citation_volume=1019;citation_year=2013;citation_pmid=23756893;citation_doi=10.1007/978-1-62703-447-0_9" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Relatedness Strengthens Genetic Connectedness Across Management Units" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
<meta name="twitter:description" content="Genetic connectedness refers to a measure of genetic relatedness across management units ( e.g. , herds and flocks). With the presence of high genetic connectedness in management units, best linear unbiased prediction (BLUP) is known to provide reliable comparisons between estimated genetic values. Genetic connectedness has been studied for pedigree-based BLUP; however, relatively little attention has been paid to using genomic information to measure connectedness. In this study, we assessed genome-based connectedness across management units by applying prediction error variance of difference (PEVD), coefficient of determination (CD), and prediction error correlation r to a combination of computer simulation and real data (mice and cattle). We found that genomic information (![Formula][1]&lt;/img&gt;) increased the estimate of connectedness among individuals from different management units compared to that based on pedigree (![Formula][2]&lt;/img&gt;). A disconnected design benefited the most. In both datasets, PEVD and CD statistics inferred increased connectedness across units when using ![Formula][3]&lt;/img&gt;- rather than ![Formula][4]&lt;/img&gt;-based relatedness, suggesting stronger connectedness. With r once using allele frequencies equal to one-half or scaling ![Formula][5]&lt;/img&gt; to values between 0 and 2, which is intrinsic to ![Formula][6]&lt;/img&gt; connectedness also increased with genomic information. However, PEVD occasionally increased, and r decreased when obtained using the alternative form of ![Formula][7]&lt;/img&gt; instead suggesting less connectedness. Such inconsistencies were not found with CD. We contend that genomic relatedness strengthens measures of genetic connectedness across units and has the potential to aid genomic evaluation of livestock species.

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif
 [3]: /embed/mml-math-3.gif
 [4]: /embed/mml-math-4.gif
 [5]: /embed/mml-math-5.gif
 [6]: /embed/mml-math-6.gif
 [7]: /embed/mml-math-7.gif" />
<meta name="og-title" property="og:title" content="Genomic Relatedness Strengthens Genetic Connectedness Across Management Units" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/10/3543" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genetic connectedness refers to a measure of genetic relatedness across management units ( e.g. , herds and flocks). With the presence of high genetic connectedness in management units, best linear unbiased prediction (BLUP) is known to provide reliable comparisons between estimated genetic values. Genetic connectedness has been studied for pedigree-based BLUP; however, relatively little attention has been paid to using genomic information to measure connectedness. In this study, we assessed genome-based connectedness across management units by applying prediction error variance of difference (PEVD), coefficient of determination (CD), and prediction error correlation r to a combination of computer simulation and real data (mice and cattle). We found that genomic information (![Formula][1]&lt;/img&gt;) increased the estimate of connectedness among individuals from different management units compared to that based on pedigree (![Formula][2]&lt;/img&gt;). A disconnected design benefited the most. In both datasets, PEVD and CD statistics inferred increased connectedness across units when using ![Formula][3]&lt;/img&gt;- rather than ![Formula][4]&lt;/img&gt;-based relatedness, suggesting stronger connectedness. With r once using allele frequencies equal to one-half or scaling ![Formula][5]&lt;/img&gt; to values between 0 and 2, which is intrinsic to ![Formula][6]&lt;/img&gt; connectedness also increased with genomic information. However, PEVD occasionally increased, and r decreased when obtained using the alternative form of ![Formula][7]&lt;/img&gt; instead suggesting less connectedness. Such inconsistencies were not found with CD. We contend that genomic relatedness strengthens measures of genetic connectedness across units and has the potential to aid genomic evaluation of livestock species.

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif
 [3]: /embed/mml-math-3.gif
 [4]: /embed/mml-math-4.gif
 [5]: /embed/mml-math-5.gif
 [6]: /embed/mml-math-6.gif
 [7]: /embed/mml-math-7.gif" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
    <title>Genomic Relatedness Strengthens Genetic Connectedness Across Management Units | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"1mA0m3Y9j8Ss8rip5QOrrwbess86ZTjfXoZJgvcywwM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"486783","apath":"\/ggg\/7\/10\/3543.atom","pisa":"ggg;7\/10\/3543","ac":{"\/ggg\/7\/10\/3543.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/10\/3543.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/10\/3543"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/10\/3543"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/10\/3543":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022j0CMELbJP3GZr\\\\\\\/AjOqliu5Tbqxxyu+yfduLqzr57aJ08zeKjeJ2nNqSHp13xdvHKmFTgKoFA02O9E1+HcfIByAVTFQzZ5B4TlpZXsdMUd822O6JGVdY3asM0jZvznhIy2YwLLhMw\\\\\\\/lLD8fKPk5vWLyTrGKh6FfwwHGs8Ny7dlo6CzpKBftGRGu6o\\\\\\\/NK3sjXKZluUX3SBO8jfgTz+jLm+B2G4RWoKexnU\\\\\\\/i64M0CUWNFzPwlpfyjH4dcg\\\\\\\/qZj5Z4l3LG+a3hvyU4HGpPgt7t18Ft3HZtdvKeUJ0RjNU6bcR98nLp7mufTe6x7rdl1mPlUwDdlR0EdW+q8Hnsd7p32Hed9SBHQjR3Oj+qnVCUXZY\\\\\\\/VmBU3+U7O\\\\\\\/Fxg7t7oBGuJyxRsEeId2VhwmOCzErkKuFyOS3JVxUiLxDFxBIosXsoLoT4yhxQH\\\\\\\/vNsrUDPiV9Nb\\\\\\\/Rm53n3RnPXCoDQ4MFl8DsHOPsNTEOtscOJUQX5ctYXX9FoOlRIem3Zizbda77FJ0t2+rZhuUtiO5WCvbTOCYmZT4ikYjBzv42NvMRlcx5EDwR8JVadiebQSc\\\\\\\/guw+BIfghKchUeR\\\\\\\/rvUNa9720UvhcXqbHWrz5W+tYQnxQFR+\\\\\\\/OrzECPdab6YGr2OHnD+hMeiHzWddfyRl8lMwRUtaFfNVUPtk69qkgBQ8i7+ROkz4fbMk3op6C21cEY4qFbyaL3jPwvuCOOrmPuYDG4xmNyHnbJ3z7Q7Z9\\\\\\\/l+XxZ7ZqQbeEy0yCpTyYwr0vPpZG3qghzJKGp6WVO8pgpqbDmBLvhDOX9vws1Z6mV0\\\\\\\/x40Go\\\\\\\/44l915wNitg5k0S+\\\\\\\/qhFnXpXggYP6GZI\\\\\\\/NQ06kuCY4wX5YBxlUUUf9m4sJS0Qw6VpoBd90aiAMJrtg7dJBxp5kbk1OXZDUIUI42LhGGlPlTLDbbmnCgg6zNKKz+OftcszFM42e\\\\\\\/s6BHV4zyBAsmZOQLxwrF8P1ugtk9G+b5jp5xRzQG5uZlagAkkSeIPUzklT8RbXHwJtN\\\\\\\/YTGXMuTCIRIuIElCx9+eP0Ma2b+pPqF4GwOOtQjLu53A4\\\\\\\/eiDc8VD0c8izZq8Uf6oYc6Xgg8dP1aGdohdRhirwCyrrfUQpNaycUoR6\\\\\\\/NRHa5l7+xvThBGP6c4KFxFFNuo\\\\\\\/1BXffJPz8YBzBcaCM0e1ULWzv6JPg8vJ3n+SUah2x7aOYpwUVDW+HmljaLky5tz69O\\\\\\\/XumW\\\\\\\/Iy5mFoCYFDUrd1ABMF0TIZY0O+wi051YIt3H7\\\\\\\/cj1oaevNGiWf7Ai+s0WlZ27dw4\\\u0022,\\\u0022iv\\\u0022:\\\u0022tJY714RriNDyi\\\\\\\/7q+GNazg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fff35d824a732410ac3f3f54d51b6ad33e91fdaf16463352bb800404c97527fe\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gjO0a2G9SB7MHtL438bx02m7hxFKAA2fr7YTlk1YrVG5ew\\\\\\\/yDc9ZG8aVb3NO9ivqFjzu5nU42Vtg5pMCBM1rlMEDaXS1fzk9XxCA2fDOFxZGCIIxiYz04YdosvTQS8LqC9ukwzhrwHLEAOhY88i2+cZdcMdHx\\\\\\\/sv0X8vLTGeNKsQ3l8DDj9QfcKfqLoVAJPtlPjN\\\\\\\/BPiZaRj6CL7Yyc3LS8VUjcXnlIeVp7jJJF6exX28RWduQ4iamGH2DXk6d42jqR\\\\\\\/IGc0g67mf3CxszDhRYhDxGpwu+KmxKzI9xQFQYBI74aa\\\\\\\/Zi38LISiNEC1aP8yIkvy1LAQ2o62BNYbA9XFoSFhQtiPH9RqtZHBkf2kho8MY6eVc9jHd2F9E\\\\\\\/XszipUGhYp2kugCnk8PIsnfVJe\\\\\\\/k6eSdW1\\\\\\\/VAB1J9\\\\\\\/SKaAm+4w+ewiEYKd2CEw40tgR1I7lM9\\\\\\\/XxdyEz\\\\\\\/o51\\\\\\\/mzm4WsNYdiRXEeSLllNeirNyoM7tDWvpxt7\\\\\\\/7GNfhdPvFe1vpA+aLh+lUX1WQQSlRInQEWLmrMMUVOp7enYu5VlveTooWgBgbQWrgiR4fQIMt\\\\\\\/o05ua8CLaoSQm0h9NzwDjKAoctUU7NFLfHLUEE\\\\\\\/HhQaTR6nZnD7+7dBKGj+oe3BXrLJlyhelD1FBvRiLbHqnqYpWjE5A0IuA7wfWgvwUO6yuzlY4QJubPuYapzX\\\\\\\/w5492\\\\\\\/O\\\\\\\/NN05x\\\\\\\/TK1bGqCZUUae\\\\\\\/y4pJ0I+HIjLH+lPeEVQtJb0XsWI\\\\\\\/OAN3riVBCz4OMdRnhk8pxG4jdiozVhesRGf7z6AlmDDdtppA3boxRqS1ygJoI\\\\\\\/IvSmPL2Q8K0hOcKqN95lSSt8ka+EIwzEbjw4i8gVIeHtWSHMOakcHL6dp2tMNwSi3XGDPUqlNjIUzczgVSyi2K8HkOgZsUqwWc7\\\\\\\/u0PqNXQBsHYSFjJXd5ezA4Ppl2L+iknRan7WIdtTkpEpOwg6KpZf7uUSXZY5yRJzM\\\\\\\/ACzPht34XOeE47bOq2\\\\\\\/4o3vrsG8\\\\\\\/hZBITYIh89czaWmNBZWCuJW\\\\\\\/7NtnV8vpS8\\\\\\\/Fo3NzxDuq+J0LVSh7dxxObjpFOtqFz8urb7+i8hFO6zl4eAtpHq9jKelrsXRfcWwV1\\\\\\\/OEv6mMc3M3U1kwi46SZavudsLl3dkLjsmCSw8lVxAzJ1toWbmo+qZLiMzVD5YaL0XBAq6HWwAmCW4uC+cPlOQaLRSIblKvmNQL22xVN\\\\\\\/U1W8LUGdZxEoCAtSWBwXsNw16FY0TWoGRNWiCcNqeEESU50xZMqomarW8nCiiQeQEOXTkg9qP\\\\\\\/8+Ebq\\\\\\\/Biayd8dHOGOGDHmwSrkcuszN5wQ99w4EDSs8uByn0feCax38uNrS9x5kYLWwyPi15ybWXW4V0qlEP+3U6z2AWd3ZYhyF8YiXiSRoF8+6MjO\\\\\\\/TkxvYXL3HsxtTrrOHmzBWzHYUh3uxRAb9FK8iQyAieBP5PlsUX5b7JbAby9ldDbCkuzOCBpydpfTAWmzQgYpGmLPUTmRuoAkJ6Dq+Z1JGDMk2VfozkHI+ACtUmZ1yAFLHqu7ylbqfVbASOaxDigje3tO3II0WyyQ2w4O+1abSNxLtOHNGagQxGopTNyqEJJwY5Tm0\\\\\\\/JVDvmGMsnz6o1Gk0qImMV7L09LEVw95KRXDr9FzR2Sxx1073+M2Lw7u+NVQOroKE1Hnp43wZHekD20jjv8MUFkT4djp9s\\\\\\\/oXVTlr0DPlaFiKo4K6XzIio9Fof2\\\\\\\/t7w+qeL1BuoRJHHKvgzuBhWgG9vFapuuAITYXTvX1+lHmiJGWOtibtl3TtDJBLQhW4SzKwQDyT6DbzLE04tzbi6A2hceeFH+BO0bFIwScsa7qSryiWtqmfYPDaKjGYIaC0lPsAFOYUNerp0xPfvRP+o9NqsCfeU5syeczo\\\\\\\/4JZx6+I3oGAZcYZ65YXvmSxiSTmayrE2aa3TzlVqbNF+EcknUjvHQO8qrBecCCNoPegv8rYPPlW\\\\\\\/fQMLLkKXVPTezLnI4eX+kPUm2\\\\\\\/3z99ffj2uLRKVx3ugeAyUAxcNfnUJeAs6ip\\\\\\\/cCKF09WzLLbIUC2uz7Q81M5caPpsNjblMQzelABY6HomPhLKw3WGkh7\\\\\\\/6wgt0jy21t6U\\\\\\\/WL1Ja\\\\\\\/rFzt1h\\\\\\\/un7\\\\\\\/g3sbIDfRUVsU2B\\\\\\\/I7L7QHxzIKECMRXnvsQuBNcVTnbSnJ7FmORTAN1L9tkOLwuI2Xfi31Qo333fbEv3XmjMCztZrIDGDoEMo3gCfN2CzdyjwWRg\\\\\\\/hGm0P8HJJIfsDxUXlkzn56lMvc+PeSICL2g0Kxf4KuLItT8T1ZIKMZwSQ9IbAdfd11yTJov8kEeybocPItmdbsuKN7UABJ8TyLxanCoHz49hzUaG9FbpGf0304h4iN4Qa3kxlHaqD6A9G6GJtsQky1xvD30wkhoaI8OakASlm2aYnRkE28QnpiUSn4zF82CMQBhJ3OYy3weNTFfiGvY4d+KRj0p7B8yYC0zNDFg4hX0dAXGhxri6B3YgSnOnXa70MVnMwbTfunkiBtSetyTba6VWCMtO1y0vbKef2MH7HKpmZ3UeVwRYzTU+vyHzA1asjv0bTb3kDh2PdLP0NGyt7vYcnyQzj29w29fogQik1Ec\\\\\\\/56PAvg+zrGCc\\\\\\\/FfbVb\\\\\\\/kRxKLJi43y5K5kQMWUW7sjGeuUm9MdV3UTI1coVKY0q0SMp2pJSqVTn3m8CXslra\\\\\\\/G+SqZoirN\\\\\\\/pvAIKHqDw5qkSTQ9rbtU8QINpVCfH17HJdft3cSvIl9ho3rQz9XBS+nG1w3ghWQ+tRkRt\\\\\\\/LiHG5CPRg4kXDYW5vd0jsDtw6lLAuUMUoGBspSSCrPQr4zSNzkWzWbz4gKkiJgW3qPSy2763wONzEdtEcizd0eNuIskawBYNAr+lUtdMkzsvophIPlmkfG7hjD7T6+zaEg4\\\\\\\/xeA1PIp3QBxoyIzeDCyEWGh1RLAG0lTOVwYNU3itKiiLWS3yCiTErHTGgKibbTi8hdWTzbAjQVH59ZwnOu2HYDOevegeUKHV9VUmuDFAqriqiCUaIQJzzKcuRJ9LGwvRl+3vc9fEgkFr++sgEzhwqY4GPpiW4DQbwZu82NkKnhg9GtT3iTNg5DaD1EJwvIhtUQSxZ1giKfhYJeYUmavCQVNnHPir1\\\\\\\/i25sbmNawBto0lKI6b7t4MPYYtPRKXdy1FjDps5rzhdO5oZJov0b8YyeW7H3cTcBIbU+Pet6JxPW02mg15lRJvgQPrPPVLGGm9VR070kF+cFFX7jCPrVpaGoMxNa\\\\\\\/0ZzkOl1xEEgetVOIdM+fQYpulNLuGbBhepsDqYzuSvM99uuSjFaadH8geDfbniuN2vZdhEdkeOGA8pfikAYeLCLn2leY8pdKPsARk890epY5YwlqlyQBx4JEA34t645XN12Ensmo+szpERouFNZ0f3id2tUKnq9pDnUL9QRTjw7QpxDnpUFQv\\\\\\\/lBb4a\\\\\\\/F1fb4kPMXalMDgpFPdDLe4CC0L5of\\\\\\\/DMOeqvwvKfwWM4e2zBuj+2BbdFrCLTU5B2xtJXqh801lnwUTEKSDNmidlcRYfJHKx\\\\\\\/odvZ2Hr2lAD+2TG\\\\\\\/BSi30nWsTkkpXUf4Ep3trftT5vprPOL6D\\\\\\\/\\\\\\\/NnyTQtK9WSBfWNeoZbaoW\\\\\\\/sjg2OYvYZlohbSJWICAmMkwL7ekbNLjPTjBItzkUArPwVpfx2mnFXd5UDWbX9xWfn1c42JPhwUPUjIAD6fX4Ed1cP\\\\\\\/DnOsRp7IgQ4KfkmKeU7STnNOYXdGcahs0O70ONPend19wrG6gKS3OECMwVF8oagjpeAhENDYSCjBx7rGd\\\\\\\/YGkU3h5Z2GMI+y7Ahacql2Ky6oJ3L04Ec0ORoVUBdJ1OaTBvvzcl56ZZ77ljz0ZrqWYvO38dT2cwZ9wiNoowO6teYAWhzuMyGEqGitYRdd28uLJsi+DmRJpSDwiIOt3y8dPV7ag5e8bqyuFYx76ZaZRTL0woCT\\\\\\\/s0zLtoC+\\\\\\\/I3O5v6Kk5\\\\\\\/Sg53pLgodZcTOid0GgBPkvxHmckEJi0\\\\\\\/4O1Xa9LurivSjZSg+aKFJ7ZCxoVR9\\\\\\\/qs1fwSshZIPXvFWczmHzeuJfMmyo1RvaeKR2TGP9vbJNjE+u5l12Tgg86oTiuxG9zTZoldcO1f7E5ypyZHA6iR2HHietReR4QJ6fPrMei6pfXZvt+LarPSWG6skaeQIGbhkjVK\\\\\\\/SY\\\\\\\/q8vb3yQGspSe8cfyhWvQlIq4+zGB1IlNuqWkLX0VtCKzNlEHg\\\\\\\/1IPm2Fh3zj3g7\\\\\\\/PMFbAPZBzoddlNiNMApPOue0LwODwEo\\\\\\\/6UlgSvn2tw42QYj2c0zmkYY0SWUUozdM6ntAGW7CScY4laQdKjHZDTds+T+EB+Tiodoy2HLsVrkb1Wa9nxwjSLT7mM\\\\\\\/H4Z8Gicdj+OAfgDNiWFw6jADYEebjcr+xVnR\\\\\\\/UHoOgoGzBqYIm7EBEwUhzUGbGZCLC9Z+CuhePo8HRxvdMQOmaQuQoxXdHwjhkAgmabj+3v3LGvj3cRJqTmregc1SfdFWRdxfL9DAUmNyJ7FR\\\\\\\/6bXQSfGzta\\\\\\\/Yl\\\\\\\/BR2Iuh3qtVeGwAnrvRIKeGQ1xohN500l7TVlaP1b1jIHM+zfU\\\\\\\/uxq0gTm2lliyBt0acH8XnoZqzP91Yt1aHjHwiTIp+5lpO5J3LjOqyi5DBLEN8g+rVhikjlMhmar3Kp+5rIFK4uc1\\\\\\\/0ULhNY9tc7lWWWqbQLXYRmjZsrsdemuUo8cwBeQMu0yhJpNEpxcc0BAhsi39DZWzfZOPa7fXDiX1hCVn+yCMdyGMblQqfPF33QEi2UnZuU9djzAAsYaM1OrEQcP9aPOZtT51HvWr0zOS+bWkbCEXVeuX0VmqMXXaEyJlsUlotnH7nKVuUsn7qcjXLGVk7Gt3ppINZkNuj2uFVKLaiUYCHwf7yOMLiXLw1kQmwgDOHMOIGWT1QvNaUHE1JPtZYf+Jkp1e9SwXUfJxLC5HuWHfTDtyau9XdwHZWFXD1SS6J00NymDsPMwmg0\\\\\\\/b3uhNC8aUz3CwY7uQDvWN9Q6wDHHFyHTo+y6SKEP0t6iQZCacHnBPFsDHkkZzBNPtodC1HhWbgfFd\\\\\\\/SIFHpnqxs044EI2PlJCjroraDamBmERTup5FnkCY74OPHwtJmoWjbXJxFCxx2SbkCmGkVzC1STM0wMeCK0zIGlRq7q135PW8BWC6gvqnPO28yAwy2PSeBWTGNbwGxD25Kk3gyD5fGULFqV8oMaQj3J1lHtkta6yRSrYKbarZKgYqjdMmaG70pcChaJZmB++drkgdtUdUXcZs1aNAu4gaUwXM6KA8cC7P77S2+LXx5gBkHHfmZ02XhDM9e9nMMXjg0PXr\\\\\\\/uIl7He+bXZ84uNulWCI7E9IJPkkau0cLAHbjvWq+1KmGo+REE1NHQPMQ7jLi3jB1h4uC0Rwn6XjwhQajvevm8tzaff\\\\\\\/oEALSjFAg0luHEE+Dne3X9C5JPCdNROhTZ4LAqzkA8XsZ98g9jZFIEKIRYqEEX6CaWuycXFymAlCA9ZPNxZEmZ46uwQ2nohH3L\\\\\\\/8CeEontDuApX9zN5h2hBCqP\\\\\\\/xLGQsJiQwyt8CvN\\\\\\\/4T2iYeIO\\\\\\\/dj+0O\\\\\\\/+7TWu4jnBfRasB9MXgSteBhdXhv2YBFdTUtzVr\\\\\\\/KOzby2whq4CtN2idF8KvZWEojIB5kAo9k4bJLzfkzH6g9SgUtoCEXbzidT6nH6UaSloONzGuVYstYbhhw4PEvqFEIvmzHvGBRvKR8OZhWd34dvZFHsrqcWYbco69kuWfKDO7btZL46dcNpa3DM7OjOXf0XEuVHve3FBHLbdrin9nLvxAYDo2Aso5y3Ap9xBy7q\\\\\\\/ZIIF5QQuo38AbpWRtBy1cYD3TWEZemGBz1VxAn2yKHRFjQpZlsCaZfIAp1r1ub4vusrANDCPEHH2Z7wnQB76q+YRxkF5A2rK9TtWJUx3VYo1KgM2lRLY8ZwTSeWqno0WHer8MeriBYHbUMMMAyUbKzSOt0t4UhXXFyG5v56PR6mLoZQCyU1LOkcxIE4YIx68CKPzWRz7cNQ7BpLwclsrRVFr2d\\\\\\\/fI03KG+c3mbMdf0lydmtYtMInCFbjEOMHHgrHCElIcXKFGonkixhTj4mZGhJKf4otZc\\\\\\\/LQRfb8GUMEaHl\\\\\\\/tMzEzzNARRgfKrFihPDZV75592oIp5B2HHoCH4RhTwVdTqVdZZgj6VrDrwpyLkU3+Ht3RdE\\\\\\\/t6E+ER4HFPXGv\\\\\\\/uCNWL2svXZ1JzeYA+1B9qbXuBkkyQaVVB4uMJM3EA3napO0TqsgpokHiDJyyHIMCp9S2m+x+gRL6wGlH\\\\\\\/6GGyxxPbkP5OKuKWftSNmOFyDzl8avOV8moI9tdQjJ0gJ7i1L1DAPigJnJVUVQXteOkCeppzIqooc70s6GUv\\\\\\\/zbo9IDM2Dhq3xmOqoe6IiCl6yAKp7FKEVQ+uzaSVgxCqM+p7\\\\\\\/c0cR+LTxa4\\\\\\\/4qwGoBA+rJKTDLckU1WbkVE83bIN9OHkT0mN89PxzUur8GdhoAdL4\\\\\\\/iogK9yJTPPiApt617fiQdfFHMohIxX9tHDWRxEFi90UX\\\\\\\/1n4AEteJdXF+D\\\\\\\/L7Slz5OAiH4jR1q8LckUXRt7nivIIWAfyi\\\\\\\/Cl8KQ2Vt9cY2if3LtTz09syfleKWJ1LeboPh+iL3kgn6WnSpXgKabrEqzyReh8nH2yaDZcJrM0KNLcMzST7fMSmCRZXeXBbjECRP87qPijtfuOE\\\\\\\/CeO0webX33llQtaDL9rGEztBwFqLBmePNAgrDXnSaCfJg0n90tQRLKxBmTTL4ZzA3EbgqoXRbZuK1+GlI+6MuS+sBKghNnd+RNalII13wHINNM8mpZ9yv8EmvAdfrTnIZ2D+rCRihXsHpUzAOh9MFmftpn2DcgxkaAXQ\\\\\\\/w6T40Ag8JVmv+aaIbMCkKvIXaLbtqrps835FlXe+\\\\\\\/aPWcqMLdUwIbZNLEuOAwI3mepksSGPd9HuT4o4zqzepUGPwsqnf5VP+MvzFANqf5iLQRFW5IQzT\\\\\\\/Kj5EA5O\\\\\\\/ELxgsJ2I7lXAcQ7nafdISlkioDRcyKIReOFeBVjxjzoa+vmC9HMXjxftUAtoeIojeW5Gss9yhGVFfv8GbjpgoV38qKqRauPqngRKYtZhhPK5EP\\\\\\\/fsy4x+n+COyR0P5IyGV+yIW0qJ6FLtEaqH7jVuA3gno5NRs+ImbWa3LcBwAUcYwUSiDIBEoAjt6faQaoqFPt3hBQ3STjXrvKlYhU\\\\\\\/kbAQAioot6FfB2ECjg13l00omVNT\\\\\\\/QmLmuSKbRZhstHM2hr2fHFWGM0XI3MCFyXkX7dNIWuWmG5WKNg7U\\\\\\\/y2NKQPEqRZCEbK13xY5A12aiNDHfDFywlwT0ymmCv9f0PkXnP9CuSh111NQwkiiR+xqoUbTKDcqQtJsWcBxsN4AQxevJ+oZgsps7a0YVbxOng89pE5amzf4R711I1CmOJ2qaONsb7KUMeSIwkSpaW+bgdrCVC+TQImcJreSVb4PpWLRnbZSvHPKpb9ac7tYavP5xxbUuTJQoXFNaz6QdAEuWSmHlwTcj1d6ka2p3Bl70FjNIwflNsEKtCB4BwoUZd1zrUkX+YW9kv0PoSctKOPh5wBmPBVAj3oUZHIHnzi5A\\\\\\\/DSDZZ8uk6lt6ZuX7WwCrRj9+yIiO4y+humhx8cMb\\\\\\\/TgsGjg8jTSN6v1e4oa3S\\\\\\\/pxuZYYKjsZ4ka027vDfga36k801v0k3tbfaT7vwyWkq9ATAcFx6Ek8mgHMP06yuevOJYlT7qNCz98L87aeB7SnEiuqoTksfTfew5X9cb8Sdwpoe1jWQw5GocBmVDGAOfujclUykM1JwhVK3MuKxO9a4wYf48d7mR8pFEOgKC9mywwYuDu12r6UJcN0TcC1tdkN\\\\\\\/sd682dPC2Y3CEplUOJbuW3DL40yJa4w68wZALdOeYYcrtZZnBuCABZDPdsEkMco4VB3xC4dt8lk1pnfTmz\\\\\\\/F6pIkAZmXK1KEPHC1atXvlksEjyKU0EYHr9dXCM\\\\\\\/bAFsAlcdlGWXN5vSJg9E0IARdzbQ+HUsHnMcdSvJrsr0gPleSqSTE20IHlmIRri6yyncZzGTP69JnwCUdpNYSoPiAapj5k8H\\\\\\\/qi1nLPBu8HHHbky1Iga1v6\\\\\\\/G1JbaVHR0X6PMo21khqgWuihV9\\\\\\\/qhgE7i9TYjMGDnQAHh50dQ5gljnfsZvNT+VO975DGVjCv4TuIYnc2S+jWyOJkOCsexgX7UjKi6uqdZ2iCgYxrHXUd9PshrWKi\\\\\\\/ImOIEC\\\\\\\/iFwQJbbrsAm\\\\\\\/+peefVYHjgStOfXw0WqnMqEcOL8yaKSYl2Ii7u9UkXel5xwEVGekJldP2YjX9aj\\\\\\\/3HkeiUdNl4\\\\\\\/rPGq+QajUwXdjcJ98tALczRZ6Sdwxfps7XckrB+As1XcKO9Ozy4igsQoD779RJIORzaJ8TmCU\\\\\\\/hRiqhs6jG7yjcy3NGqO67P08FbpJA4OvSVcIX2ilPc51\\\\\\\/s35\\\\\\\/E8C1gL3QWSY6DHq2\\\\\\\/g0v7OjBWUcvpUs0TaQS47nqrfOYdlQEUmIFReXaaF\\\\\\\/n3fpxwlTK\\\\\\\/9br5g\\\\\\\/1LH5h8ebAGb9vPnjWpujYXcb4ZMmfuiJUZdlUkSeqLOlfWUX+6N5omZVvp0mNsMrxPB+tKVvLpZR5cF5XgJoxLjwOGOYRtNpWwrGDS3xQ4rg7HdipWrRfAn2y1lqkji6FjK113wBYGPVLY3IZUmqG0LetZ657CUybACyr1wb2c0fY1bZiuNYDTvAZ4k1m8Q6xds4n4hrFdO5FZzVcQ\\\\\\\/WOiDT0P7fj7aTsw\\\\\\\/FQBZDwfTAOa78NDxlyo3CBEu+Ee62bCdzEVsJkLRZ3ByIhlm0cplWKxowzlWlEiKbFzk345iVxKlKcQj8i5UdNtw1C9MzEU5gy6LzOMwjImSUKiVgwMJLpJi\\\\\\\/l6ALiGP4BE1kVvLmMye36MGRHfafFcg5hmBAEuKRtzzSXla5WH\\\\\\\/Xl9UNVUjnPn59t6aEmS3mZlHNtUDzDY8+SNXoddtFb7Jkja8bnggvouFhFVr6UNVUMk7RnDzQPQBNRoT2JMwx7QH+NVhDSCKsH6L1K3XKvAAdhJjRPbvVFkQsuFKGv6VvePVJ6LOVS4ImKxT\\\\\\\/bWnhthW+cCNFPKBj1tK9xR4p9\\\\\\\/jFGwzPb5ysDgJmb0z1ouPPZxztGNOh5KCqaUJ+H96S1Aq087tYBWRF6RxY\\\\\\\/KM1J+Gu3sfv5eVhisnu9SJz3W2c1ZFd8MdLAXppmBuErSmk10TaftoD5zSdGBXWtXgv5G5BmlkygktDMYTFqVPe5KCrZVvxOEM8w6POT3rF09uHNF1pGNTgEjSz8GI98kplc2TjAKyP5coxpI+lxF2sAZMtRXEpsFredqweCymGcKo\\\\\\\/dVcG8b+HhfPq67AhbBVHJsFzwExHEBs4n2GvKVp\\\\\\\/\\\\\\\/uA1ngWsK1YLM7OWFDyAp26XjMdpdgV57Q+WikQn03\\\\\\\/6SSTWhP9juz\\\\\\\/lHT0GxCmd9o4whJ2gZhJie1JjytsXhE288IX0qN7N42fIPxTRa+Ryx0xWcF75HLKzDxe7Tqy9sz7evsSwe2m59KhKmncSLk\\\\\\\/mSek73rBlOJFCRSiSOnUi1PIPtTPQND7I83pobGOlausnDzQqjqzGmN9+8GPxjBlCtK82l+1mvPQ0kdC9SPfN1QauHp\\\\\\\/gIaF8V8+G8Bayv4or3fPauWFj+UEQrQpnLN1sdrMW\\\\\\\/7LufijxgMINxIQ8pbHGTqsq2wvRGSlCbd4eQBWvzg3EbK3KnwdIxkWxO7CTx0f9O2bvBcZPOCIreJ6Xfeve3knl3pLoIomKMdMRPt9OrrzLH7v7wY2KUXiRIszqEDAH10a4yfHvB9WMS7gPZBCJNR1IQ7u9SHLTIoIxiRLKqFo4CvuWvz8r3cP35HY451D\\\\\\\/SVXmu2aA4RRpEWmMDYkd3Ti\\\\\\\/A1ucG614jQDVAAahqDklM21GUrowzzkYaDh1RzG9WOH5Oa8MfN5C51Eqblk9daF3YbRgtoH9dhIpZrXqHKFu9UYfgNypj3XLCZcqMJ\\\\\\\/SWNneqKiFECPZSrNGJ85R5iRj1XpZS4AIH74eqWHUgP041H5o55txifQMJVt6MBDvVq\\\\\\\/awO16pjOOa3JHN8Qgm1z975lfTH8J66llzluoWok3VuSochpGO+br5i90XfGvsYT2dwR\\\\\\\/gNXvWjnkBR2U+cRBcnS7eYvdXq7WJ1u1es5PoaHUNcO3E3qk4eBNrhABT9YG3lCdW9EVsokVGPsO5BE7TsmXRnsaJkxmA9Lqa2v0Hc1+p\\\\\\\/AldNRIcrRpmSFgPy9xOKpFp3NvPjxtxwf8K32S1gxwe1Nm\\\\\\\/ccYvk3jOylsE8Q0HUX+Jxn9aPVlzWdH2QOCWDQn9QlODMdsgoslOTb4ii72Kp6gOUJhIb7pXaaWsQjWEvfb2\\\\\\\/5M0eo4oo189RH9jyGI2Lp6uYSA2rPVroMxOTN95dIqjSHR0u89eB9+yIToW5QQTHRAA+AAmQtskrE31zmU4BmP3y+yFBUebyDYzJmLqqkhtqbKIMh0thFPjCYNG6Y8U1+89Q2g7tO2fR07QqShzGMMRRQ3UiCKZaWyQXMRMEaY\\\\\\\/D+H3UkgA6BIl+4KCnV3PPNrIxlPxyF1MIcETL7yeTSBP+W0LSk4SAMhnUNKiG4ak4PyqnHsw2B7V7gL7pKW8Qw+NHbBcZC9tAkTvh37nTGunez+QkpSEt2j4BHqSWQz5ahKN3qgZn3Y4q8fJDnFQ1ro\\\\\\\/fanJHD1p\\\\\\\/f9kXSysq\\\\\\\/4eh5r64muS49zvVQqjq6IbYhtfwRJIOqIrMgxK8NhcGaIcFHbOIjN9rg3\\\\\\\/4hIb6DGsC1MLdemqvSrUT14cSIGWKnUpfyc7Dpoluw+ar3gqBNtZ4B8Xdy3WfFkX3djeTTaCE5kbNsPZusN4nAzBRErm3d3Vqn8OMV0qMZdbOukx6h+BmAntGgGiG7hr9qKujN8IvEjKsKcYddmcWyjvsKFz3dopy3PH9RS2MoaoQ7dwa\\\\\\\/tZpvRQdGbO7IMIIKb3GyC2OGWaUVn42TD1Aswh7e8jwRMi4mUebhJnhwha3AnJ5a1SFCwEOT6VkFiCGAiw6s04F0tGD3BTcTqanp+dQSMIu+MQ\\\\\\\/qbZ5jzOoRFntb\\\\\\\/WrUK8iUqrcLjHZ5qxgEldpYLpPH7eXXHu3mushWXmpzBRyYH6puFxYLRWWStb6SBipok3Z7KfhFqklBGkjxQ0Fazmb8EVyd5l8qbjGBaQwYbEeRvMOD2kETmK87WDXWJ\\\\\\\/Z6SvYgla7Msf1IT4fZ9Ajcgd9XCfmgaciOwRf31Gku8RE6jiTknTO4U7wkCawe6khh5iIrBYjaEgMFuq4DB0IhuBxG33Bjytxmb+e81s+oykQyiRRhXWuWyhonPIGDZxLCeVc8ec0U4toqHtmIC965R7S57Pc26QHNfatewpsgC7Hu8I2HZUQ5Z90LAUMr3NeAfW+lIPJK4WSpwYab7ox2zVPjAPHBo6mJxhyliI9CCdCxSKhJgWAfTNzSB8pSlRDLgV3A4jyGrAKehmlgrXpFhl81qT2LA4py4nBIrEH\\\\\\\/1FGCIAiJfHoDlhfzcvy15WvrbiOMcX4ZOJIh1hcYdBZtRXYNjUjL+ptDvNLkbwwsIa8oj96a8eumYDXpcqtZlWUiE0TCILx2+W9yG4jKHUg5ODaUQPi3h0JgoBFPazzrUeJsW7Nol4WwtLSwRpI+KTYVTRoJ6vjd+4uVrMib2IKfEwKobg+cxbRAiSwyFgtXNu+6M5lAa0YgfRVseyK3hUypj92Pg45\\\\\\\/wjDlRVjOB49Ga3mva9j2mGxoUIYr1Rt3EEMgG9X3jHjwE\\\\\\\/StauHOZEFnJPp6pWG7e5F9OcjwguCqtrxtZxQSObBR7n2IYJa\\\\\\\/y1n+UQku3wMVTq9CGrlbt4FXCstcrAp\\\\\\\/X4XmTOUOok3vVDd4Fe7O10PXqDhdwoAsRRG5lusBSNmyuIoe3K74exVsMl\\\\\\\/\\\\\\\/nnYSAjUwx+8FJq5jM0EnBlkAk9bfc2AnPP2z2NZ0NReTiFKxNcntodF6SnLt\\\\\\\/De8LcmxBkVphquTK7Zl06AsiKXlvwledBJe5rI+XJXTdPZDsIXgrYz1ZWnQdYsqWpta4JmClRFcwZf13bbNc4oacWOj7hKSBvA3a8h5kYkZyPAdO2IVKikPllR3HLYPQ5m\\\\\\\/sBeD3D8EP0\\\\\\\/Hs9CtEDr3qRcfj98lSAmv6dgzq+VzJ+mEikom9vMElRl59IDiDm+T1axujRqEpw3CkuQcGcl8wKEmLMXKlihBIPFqnbQ2cxxZ74+k2+mpbSPlOYfZ8Ps13eU56G+Hz95AEYKzhGYu1VTA5BQqoUjews0DA60UG7rWQJ8UckeFisoTSC6XslT3a4cd9t5iN2g+xhuEIW4cgLLN5ooEZ4AefrTa46x73lnSUnsr0bqtqC5aaZOejuMoBOF5Mwd4lR9CAuXW+afke2ItNJGEj48YSYqJo5V2DX1CfqsLuSnEnTDP1GEUUS5Se654CVacWG8QPpCqI2ksLfx\\\\\\\/WdKKDQajrt5EK9qIcAjUE+vMEfGDgIINz+OITBi6Kv+HZs4\\\\\\\/aUxe3\\\\\\\/cV2MWw0VAeeZVym0xujb5Ywq6RRvK8GgI8ePib9\\\\\\\/QEtKRECNV2q0lbf5uXIOuLwWnUnraqxzhBnton1ZdvxFPzdDE7q6WeXhybVViuI5CVMboVXTZyyvltOpmiA1RUPcf7IZ2p\\\\\\\/rBrNzYjnfAwhCErRJldam9Z5VbGbhkKxUTktdpiyldSgJ+wuRqBnuUAozCUVsQLbOlMVxYffVbckbwHIMq+SeJytsjOyC+rKQhr0fQEbktr+sDtyhlVDb9OmThuWHAnXSmM77Qy0lc1T8Y5gFYR5Aw24Cy9U0HxMgPvUjNGV6vyu1InHud2AkUYpVKjG2+KNxRbx58xesaevlIHYZrb2Pxhh6zyw7Lb5A1Z12urYG6WpLi6KihR6bzPMWCTwmIGkip9azgOkTLowc7YAN4k2X6pYGxYmhsw4kDmynvkCRlmSFfCtmk4o6652gGJxUfdZiO34TEY+alAmTKd063CQq\\\\\\\/hXoZq97kGd0uGmdtbZqjDiAt5h4pZvY\\\\\\\/BYCB\\\\\\\/LUA+O\\\\\\\/j4l0mATLuzM2\\\\\\\/hR9TuzDG1vPBb51BUw5qgtkIEZzPTD+nGmTiky1Y1WANCo1ts63HX1z+hr8WxG8wG1JXiAybMwTnAnd0rJ0KUGotbTwuoWByfd3irc6N8Op3a4BvtUGWD+8kRLFjEc+z\\\\\\\/4H8AOyGZ57Dfu\\\\\\\/h19rdsIaI+27SLrvTf\\\\\\\/docVw6ZbfTyMY4DIXS0i\\\\\\\/gnNCt64erTepFmQujRZq9ep5fL2q9x3ObNaq+VtgtIAK\\\\\\\/d3UUreO2W3+Gjs9vpquqGTxhJzXj8zitLT6cdoZ7OPFx5E3gsqSOoQ0M6scs+rf4ITBqPCmTTZfEoXewPznGdYTUBGuI6z7t6qqctDfbvKdpQLsPla0dYx\\\\\\\/ZBflO0\\\\\\\/eTcy3E4Zikiq1EMFkC3ZzLTBhGf6s+JNyYI8lkN2XBD4uzvypS4uPqF0fBUlcW4pzQt+QjpwZStPr2ubP6z0NL7V9ePnbSRW9Yo8Doh2+es2pVbbkt+99oVl69cEHzznPr0UN4RYveEJL9b\\\\\\\/Wt7HZSWspW8ldYDu61Jdepp2JN7xUV4gATE7ZJ\\\\\\\/Ol+O1ucp6+Zs8S3jzhLJimWzraQZAF2opNnzNAyjLuVZfB5zzBioAu1aVDvNOJwdPNm6GsJF5UuKlRIq+XMUYvqBItAeKuxb74XAgCMEw2WkREnQIC0FWmTDydxXTjeBJ+pUqD1ELsRCathq+m+IYUjaZ0m6XFuOTtDXvjQNlUDpzXvvkCu9105OB3SpYREl2iPaPsZ8N0OZU2o\\\\\\\/X9xP9jutoGcalfkbKVQ8CdP6\\\\\\\/ZeA\\\\\\\/n5XQHRnas819ON1nnv4v9p3OZsDqfFZ6A1NVYhWVJbW\\\\\\\/LVuVfNrPFnCt\\\\\\\/\\\\\\\/fNDFnknWesnqjmUqh\\\\\\\/b4WyNdy8QxbZQnQoIwFKEQp7fvnW8GXgwvyFoTCiX6yyzYOUT4AoSi4l+5ZxH8cPvkFdXhX\\\\\\\/sH6LXTvJfkzXIfEMJOophjfnVZQawg8Qxs1jikQkDYG1uXQZjVzQdOT36\\\\\\\/ufJzQaPInJf7YIq3v6JPL8LMVcnmMTBpuyHCC3nLBd7yl07TeI5r\\\\\\\/HbPzLLjVSer+QC1iVeRtGqOMrFvf8jIhQHg5NWeg9iG9U5n8BVj5IRD1IPMZAWqUrVaHbFt6ZGbf0\\\\\\\/sbx9AzRRiv0ptv+sq+HTj+Rc0iqXWnlqEeg+x+bXz59PQCHEUi1yYKN7YKn9GYxV7GgN7aYVLOvaHFsNIvhbZJUKiG++m0U42yfwkoccADhDn2MtUQYVkRkEHSBcUkwPATeiQc4Mv9hOMEtpWLEEO44k79QtRP6vwIoBy4OAG6S\\\\\\\/YuzOtvfj3\\\\\\\/50VOBPpRTSDvfLzYMedRZLjKYfzmUzXTFwv0bTdFM7zzLluOD\\\\\\\/+I+WVO4FnQtauYpLVvF5f3Q7dCZCW1WdOA+HiAfGqotevnD3K4SSTvSpvadPOenPEQggf0XqA2u9qfnSXM06VFrT7LIZrQQ9Fd+mOomaoi9C20mLSHFUJMj12yRvonjVG6Ca3aTEcfNCwiOOhY1kTtdQdXme5nu06QCGUFSsUIYjOUlhiwYIB5qaAl02axNzgnjcVuhDeDn3hg6u+G7x38mh3NeNTJ0PtJzgB49MsUgRsyZK5uYHErCgvvL8kdbDLc6vBHXcGlElOMXmn\\\\\\\/SRijN1ADE7dMqNMkaWwKpYxZ1o46cn7hU0vhsiamU9VR3a2lFnULIZ9WbC4Bqotb8uT2qaXyfRDWLETxtdYa7tLcSk02LnQX5kGK88k7r\\\\\\\/BK6PlEFg0bwUS8SDj3cZm8nm1cNXhNbF+wGtYsip8MsGN9Ul8L5SfNprq+X0lOE9lmkFl+KGW53VNEyEI9uJahVHz5yxTBwngs6E3jHpDeJbIfWO82lZv\\\\\\\/9Ek4PA3h8aichh\\\\\\\/bpYvcwLh01GVUH5eKXlcWZXokhyABib6HSjsJy0cDS56LhfF1plhoASkOYRhGqmc+qzmysW0nARgZ+GnX1Wtgfdv8p+Tdnmoy8pEkOw59AP0N4ZxQPMi8lNWVop6mYiF3tIG6gMiMm5P8xPB5qsnzogvBkQGQGRyf8PHKW+V5W8t6ZLDrCVvjVWBQxZpxADdCDCnABT0pks1ANV5o6PX+7sdC1jAzgN+iPhYHT6CXndOORubx4IPAi\\\\\\\/3r3DlgjtxLakiOXL2DtpveSgjQzHWfJJLRJp6uOV0hka4vIxZOLd5Jd8SZ+Ot+loaEK\\\\\\\/pKsFH66UDT59CUJQtFqIt7lJ\\\\\\\/OrDRijNnKn28J00nnEOY0Rfx8qtF+ytjj0hOuhUqF0tOlYcJOBXndLXA2vdHs9WU90lfbrLZYAlw9oABCXVxteJ4Kg9EAwENfhnGNkwyj98\\\\\\\/udD78Sa91rAcejJcAeGLDcXEfUOdySoR65xG7vcBlbHwmPYNqvKGjugL7P4f\\\\\\\/2k5MXa\\\\\\\/B88ufdIBdrMn0g8hg9nbUfTrkJMTcT+jbe+rN2QZ+TzU0wfvnPuGzixGPzryH76qtZFgpJ4Go2+5qeWpuE0zI1Y05N2ocbfIXBeLVgIod2bf2sbxjBOvVg6i7MpNrC1Zr44PBP8ooRuSY\\\\\\\/tzk0cjWxl7SzAqDNjfLGEHhCwkvtMFNMfgbD00rz80Ui6S3d9HPCafarx0oUEdO2TJzDDboosNo6rB78o2TQ7uqbEaEYFxoELnvmI2UM5Dx8Kl7ZoN3mByhr0r1gcfU4uT3BzEOrnu69Tpf+ojMmtFUUAX+onUo06geyC+TA9w7qSPCiY8lbQQrrQUy55\\\\\\\/DEV2bgGcdzskQxMkjYu5AXampwSYKvhEOyIhYt2XdL47PiDBfmTuGNfhOZL28+zKHYNXQWu8+hpdhE4RGQ9k3LjEkk6ihxWDYynGFITkFTHRESfNka90SbNr\\\\\\\/62korm4gsrEkNHiXzJZatmG9Zt\\\\\\\/0jbnn55qsgFy2ZbYteK2AgqeJOuokpnEDukTPNJYXBPcl1lEOlZM4hbeMUKk3Z7ccKNncVU52hW4cfCSDGMRetUw31hjTwagIlF+GQBQm3RJG\\\\\\\/66mqC6w3+UsL4wGEQh9K5N4MEPGMf8\\\\\\\/nl4OIfNDtjZxMZlpxBChgRWBRsjN\\\\\\\/oP3Ynz0DmDxH1\\\\\\\/w5HZPsECQ3+czMsd6onNBegbs2KmuvsGnBmo\\\\\\\/B3rxMKchxAxOllgOYyYtzDpd5fzbJJn8tWifsa8HToHg58JWHzdItZd7Gk2TtA1nvVa+OCMJfhK1Wh2269CCbuyl0na4yPO03kOMzJPFADFV75Z2gqKELAPNbVwTpVjgURwvkpbBNFY3bIKCCxnwY7jY01HnEaZrSlYZcFbiJ0+U5XB1Fk8G3NQzyXJ9osPqkb3jGrqAYBBOgRAwTu7f6bsozohnGe44APpWrLtjTmiqHJKAWzERy4pZVTtOkLWUdbUZ+LdD9B7vj1nISD8dfFSMtuayIEmsetvvPQ2bauM9hJIBP\\\\\\\/9p\\\\\\\/63Ke1GAYZ6AvZuV+cZS87KAhUk\\\\\\\/1Ya1O6LPj6YG0NCuDBRIvl9wsiJ0uDpNOatR4uhotelOe9vb2WEsBf\\\\\\\/8esUImg+7v87Mtq3HyCXqy2S5u8KynU9xNtbo41ilnStt31h1F4S1NtoHqf10zu9Osc8HIwkOw9X4pF7ez4LeW19tmHotl375agGxQOk4K8J0XwUps2w1ibqkzinGimLT3JYqDUZ0MnEoe1KW5dd+zQZVnW5zLWIPq+ufuJvo4Znut1\\\\\\\/SiZq9c\\\\\\\/xkIBRPUXmqEDrUv8diB3bkRPJqlHcP59lu3N8Tucdld+iLGm1PPHWaESTN8uZty\\\\\\\/\\\\\\\/iAmFHSdsq4T95Ek7W\\\\\\\/dtyMY+8yLpKgTOGoHaSX4YZEZUtibFQYX21SMvJwzzVqafomKlB6+leEwfiZDKoDHqWPOnRlMII05AK\\\\\\\/yIXxac\\\\\\\/0QKIX1hTCBCWETlUdYTu601c+ODt3UjTsE86At+tr7F7zPZSL\\\\\\\/q9SsR+H1Aqilo7Y1dHiYEaxg0YL3PWz2ILvKHUjGoBfel7UqTf5F0irwkHvQrZ\\\\\\\/Xi00EE+IJbVhhjRme+oFIJer9XC0c+T1jovWLwiYrPoG02rTApbTUorr1VabMCly7x8qw2DxoDUeDe8PCZ9PPKbMAc\\\\\\\/EIn3\\\\\\\/uSo67UJK4Qxi5HNvnGD4iB5IrjBNh0MPsLCcIggzKI+VhwEmc6klXFTbN\\\\\\\/M1JqkHCxOk4tV7JFkNBF6XkLKrpifX6UJkrylKA0If9695+vUVawHMRzEMknvLpeoYnuVJqqgzF1H+yb8qWN4fqWVzjSaqHqagPgkjSb2d+qAQpgDUrMqLY5+8kTYrvdV35iwfZbwKTSvXoJ6+i4GWgGGR2GySL5TKccUp1xPulJwqu4WymS\\\\\\\/8UnhPIpa7\\\\\\\/mfNpnbWjz3OvDn8TNwadjTrnhIX57jY+n7pHFeWPpAO8XdTjJx2wR0VMjE0SKo0IRj\\\\\\\/o9BFmlD0tEisi0jXbnDVAlgHgnqi5e0qNizzkWX4rsRo0uP3MooNm45RXADdt5ztBNKLh7DYFAoWDZl2oUI56xSUmCEdxfP1XaOfaFGp4kJpUQJo4jvkLUBrN1CBguj5ucCM1ABrCRW5+lwuZclPqoyVMZtJ07iqqw3AICHq0srw2iIFTNiwrPg9zLeYZd0UqRfVtoB9jJ4PJsTtbcPmJ7u1yDMTbi7EIiiSwMZCKEJvDvgEpr\\\\\\\/icX+EfqWXwkVn\\\\\\\/P0B0IzZMF4dVDaubvhagPio1HmQanNXjscR8uiOTUl3Jk3dMSeVyiLlzeToYNtyreL4PGye0Ish2QiO4+lbMGQOSIUDSrpWjL0DQAb+FNlnt9IhaEzjA+ecRnqY4EmiVBBNolb71RYZb+bRckuKbAjyV53qHuI84\\\\\\\/ugV0sCKH1obERc+tNefRg30T00dqGP6bwh0ZFqWh4XJENUm29Ru169Mo+tG4dRMgi7G3k+k4erQ+0EA6GmAxGs8c1rhYoKUv45i7iGReVqJOONHRcmOWko7p+F4Rx5h8ULyb7u2+vkeeF+B56d7ll+ZWcX0iosZEYEJ4SnzHrqtCGeq4\\\\\\\/0vP95qZmiReLijfWqXs0XBNtyUbKTezQZfJ5D1BrlV063Ex0quVpyb27+gA6dOU3NHU89ED+bITfi8zk5WGk1vMG0PkuqD0PMheMBG+4sXPX\\\\\\\/kcyCcRfGRcRQPXs8ua0w\\\\\\\/K4f5JEKmoPweP1\\\\\\\/0bmgpqKSTpX6S4uxgvwICKx2vAAshzt\\\\\\\/T\\\\\\\/79Tv8SvMkstbJbW8UbSiO8x2YmghY7\\\\\\\/dZYzjjzVYwCTx+BIaiFxb3ihMciv6x0+GGhbZjvFzu2DNAg3+Ad8GcDU+xQ3MaZR8uCKB1tZI62FqiM02HkMCcLw\\\\\\\/lCaCTkeHxTT1q\\\\\\\/TtgYJ48ScF5GC95rQdhlvsyYD5yO1cMP53r3PmPTxt8Vs\\\\\\\/zYx8a1f7cwixJPFENygmqM9INAVG7W3IifW8klt2SRig3LWs5lxRc0MpYRzcDsg1U0yl6wdh0T8ey6YZYfDS9y67GYv11eGNDhDjNArfdPPvQXv4QVaOViJVvpGasYOK578PevGvr1U86bdnTidiE1QXRyZ\\\\\\\/ZEG0qEumEXrnaL8yhzoIgrqU3QH+lROJJOew6ehPwc1+5HxsPBWkK9pU8r0WnawqdTk19kEKb9ILCV4LwE1Jrwzz8Xuc1XfP3RdcsXGXTU8ehB3leSXm9XlxHkkau6xSkTFRnaPwNGxTsiEo7GMCuvpl2gcTVznf5NRGKiF4JgVWTEbI5rDS1RRswtgktswcQJUzUvkoEtmYhvxl3PvR5CkMTv9Iws2o68AVGa73R7l8QoMBhNNKWtGPod\\\\\\\/2FNbZfUiU2qgEqwsBmlMAHHV0NN9VxlZOdNh3MAZB68DjUMf\\\\\\\/9SVfiZIraoW\\\\\\\/4klahehmKAmK+cZ5XyF1l+VJ61SSJvpuoxLOLHVRAe3FxDAtUsblNH3XcAUmxZSXFca1fpTaZeKaZlxphJbG7NQ\\\\\\\/k49rIyCl2ei2cE3Hs6PfskMbDh9Xa6zlGZWswbB2rAe07EONeZimieOTMUAWeuxo15IYBzbF71BpLUNEhLwDqNq5OJyTE6u\\\\\\\/wzIZO12scgN53NE0Gpvo0mAj6i3qqTraZs7tDUq7v6m59jM0TT7Urh7ZQG6fx+fwjAnf980mztd1k36\\\\\\\/pBuvIoHAZ5G6+tzDk6MQf6nLo8zvLA3Xr2y+K4g05wIUOpCU4atvhQbdTHXbAzeJkGWabhK3nnemw8+Iw8mKAmP4fJ66Wd67G9bgUBcDR4b\\\\\\\/BtvP1rfg3BrS21X\\\\\\\/3DeoACq6iwIuvB+cYSvzWQxvaT6u+rY1j16f5enAlXI5nHllC68EAymHUTvP+rzhFNsSY0NFWdyneXDsePWFg4T+4h4iG7dUyUFR0sHplpxT5fj9NSSIGn1lDpF+H4FcWod7qiIOj48S457fk\\\\\\\/BR1Av1E8fvzmWgLi2Z6+pVzTkuQU77bnDYmjBF3+LWPTbxQCnxwyNOtdCdD9YZvlYz1pZB+w3sevl9KGaGEMQFk92nJxgCHhETT2Pkq6Sjh9svKqnhc8h6kh+tKmWNSJuE8ceBPAdz5XDm1S5G8aXJdTrQrlwgrHGvJ0xahzTJB7vukjcSaRI\\\\\\\/t5iFq6bFIWUXTbh2+1byrEfSny3bYF5w0BODhEEZwyOVOeODtpTRVWtVQ9L2ztO57ryHG3Ve0uJ95OKT8gdULIl\\\\\\\/WQvzlyQnehMDqF3U1HdvyPiRVuP3RfVPN7nx8UOQmcw6Wo7psHuyHFvHyUN0piTSDxk89Su2xAJg14OS0aAF4h3NYc2TGabDFlnlJqIsHiiNGuYWWuqj7rfSA2r2rGFlKuigNPZMGdeHnJEhQhdbWfsSE0T0W+bXXTRTpUXoeiNsg1gMW+NnPyeQzkIdQe3bjL\\\\\\\/zpJnJvXKCI12tR5sy7O\\\\\\\/o2PO2efg2S1XKqvNnXpDU2JePN1mkFhH+PuB1BI5FxjE7AFquiYDaIJu8SYrOHm6BM6cZuTP0so4qiB60yA4hvQHEi00JBUgwlHCsoZ9aRvhTNmmPABd9j1VBKRax\\\\\\\/Xg55cjiGhHXzG65kL6aS7WY7azQ0PTGhCeDAXAFaBS+UPnZ1rbM+hfF3CBECaK8Arb4VW6NdcA0EkmgSKGsT55IpWIxnzuX\\\\\\\/CszVOloLlc6ujoM9zawtDXC5LSj5zsr0GgjEGRpoA\\\\\\\/UymbVvgzsX43cIrFkJN\\\\\\\/nEfW4pSqhAXBGIV\\\\\\\/36jQs8y\\\\\\\/svWKSlTiBUqJtZHJdtxZ2zJWuVLr65hL8Ejak5dofgFWNYSewTyNnnKt0aDgn8S4dUT\\\\\\\/pD2\\\\\\\/8a1T1cqI3GrRqBH38qUAH0+ZD1eqLdwZdgg+oCD0gIa8w29AuAstFyVpU6eoKv4srqE7Au2U0mi6FMV0SB5WtgiVRh3Xdex+zbq3SWVaFzvRiDfCL0Bo85UlPJMFM8qmYnmqam6OaCXxW8dUldZoUt4PTo9XV3\\\\\\\/UC4M51mgfgg1rZLNTgkiyO8i+\\\\\\\/UjfNRfaLkb1dFrrZgtZvqGOD9zymmbNWmli4tumdDKW8r7COfeNm8kBP7iOYowXH7Kjj+\\\\\\\/aHmGxSoio9y+mzrWyuELJugRLZLXZpz4nwQBeWK4rF1FDoSNWwhgjstmclqOWfo+KAMIBLUWfuNCsRh6KkB1vS1tnkvWfaJir9krNfgS6UhihENYnj3Jqk1vHOFIoDYbJwzX9\\\\\\\/s4a1JQdUWP\\\\\\\/hMuz8pDu2sJ09m5t74sP4CC6gt20BuoSExLNTP2Yv+c5SnqgUcR9K5BazhoFu6NVvcUUXheu91SZUkwUw8fIVXwo78OZPgMhwA\\\\\\\/27NSGo2xfDPvPE8Ad03l1lLujnoCWVBXUjJ1t+71Z8UFajcT1U2JS4c4ILfhwHVkP625L+dJmVeRifgcCkHDvXqIhrfoND+pcH+evmnvYv5aOcinV6G5q3ROQiL15dls4S6GqvhPFL+Hus7XFnCNUm4alu\\\\\\\/I\\\\\\\/EqCxtpiUy3V+0dvB4UPyUPRGmr\\\\\\\/LY2ER0o4dHcgpFMFrMC5u+1HWFrc2PS5oBxShpZEvjG3PClbEXNLPAs8QDG7RDYU1L0Cuof629D0UWW+zm\\\\\\\/IjggVsJFYqfQ2a\\\\\\\/qAgRUz\\\\\\\/bxGVnug0ckzNPJJSq7t0sgkyJXAYX99Ig+bL2JO5m67pi6sHZOwjhie8UT2rdsygPzpwCuXrG5ma0gJ7FSxsKGmksSwViVjoWLh8D0nX8fA2g13\\\\\\\/tTan+6fTmrGibONVyCBGtaLi+tdXzGnSHkFg+I4Uzjsu4\\\\\\\/agxoBNQlh8eah2HJELVRORCDMitEjwBj6\\\\\\\/KKbv6dHiuw8iv9v4P2Pym3MQG1\\\\\\\/ocsfgEbOswA7CvQtmYsyPDDoHQpXbwv\\\\\\\/XGBYrE5SUrOF28vkvJ2VBDhNWCiV1dr4cpCuLftVaoASbwZ5c+b5rl\\\\\\\/ijTQT52bZym75EWWqJHuI20WxevFcqu3Z\\\\\\\/zAEhj7HTBuGY9xsUV3vHskS46ptJlTfCDGVVTJ5S9EceRhMbXECLqoy4HM0AvGZg+IrNT8sUgQo\\\\\\\/pB9PJ2A88j5aLuYzFYNuW\\\\\\\/GbNgoffrt3pQPdHUITMbrPBrRkrKM4AojYMO\\\\\\\/i5mYXnhms2O4YAqBBB1RvzhEVkuGgIrSVP\\\\\\\/lx2Z4zAQ1m9IZX0d10oh75A0ku5H19kkl3hfPEf3wx5QbcPEJ+j9bE92okpXiAExhssdfEQxGA2WcPTxIWpTVkrMq3nEFyepDUJdhi09QBET2bg0R\\\\\\\/93dEqA9FVc0rnxweL5eRPHtFagfjhVYSa3fp1QMiyfq1rGBvVRqkCSV\\\\\\\/t6BRFF2nc0cCBayQOO3NaNkTZM4N2nPv0HV8nGGQI1ghAjHpmdkwJDzqh5OP+AOsa0mz2\\\\\\\/M6X6+\\\\\\\/JMxbKyRj78teFZVuZ91GBc8\\\\\\\/29Gs5Wk4xFoMfQT2hZKMZQ1Fbl\\\\\\\/MZxqDXu7XsqcZO4N2rM1++6SVQ05pz1ytTtGREVAnLwsvC5zKWmu2GwvoKwR9LXeBXLaCMKzEcMkZCN+zLw8pDk0N+ES0Vl+SgrNwCyXTFcu1CZifdo2yArICAQGN5092Ph194ls9WFx9nLD+F5EzZYTlOciUtllIslmNiAzE7jt5e+O8TXy2DPWU0S4UzfRIESrP+YJ38jS+E9YAkM6NWrDiKfoNXm40r8toMYMhgrF7t9HlAO54ZmhNLREVWopxp3WfO5M9NiGxcoLzLT9IxM\\\\\\\/5KTf68WmGedxFK0AfoCA\\\\\\\/vXYHZYXcMM+b35lMCk+CwOsaaiT\\\\\\\/Ch8QNjK\\\\\\\/qjRCy485e5\\\\\\\/JVKL2vVRMClSKNBslY\\\\\\\/PTE95lrGkT+A+XeKCePPPNmhBdUimNS7QwopKhSLE3MJVbembEQ2jyctPKzt\\\\\\\/NDVpOF\\\\\\\/ZGaZ1XlEmy7kTH5PuqZKjfHl9H5Vz0AgxxDH+6x0VaLyOi1Gtvj9M5H8XxmoZ13\\\\\\\/qcDjcZ1KblDvY3Du9cZJ2v5AQAp5+q+xx5Z\\\\\\\/JrVHWr45ZUTvi+Q\\\\\\\/33n+LmACUn2O8328xeRQg3cpsYbE7tQpmgAToo7iiAeK20yG1lVMdKhT\\\\\\\/jAGXdQNF9vM\\\\\\\/sG\\\\\\\/+Sb1WBC\\\\\\\/Cm40jmZZE2fx86SxStr5lc7AOW5OanBeFhYIQn\\\\\\\/MoR5yLzcbOkOQIISGAZE0YSHTmmVbvH3sFmca5dHbbVh87BriYZfMK+2zFAs7fu2Tsas6puOlf\\\\\\\/+cArs16+NOPtXaNH\\\\\\\/IqbGQ12esLgrabiLJRgr7P2kuMl9KUyG2N3pUKr5cht1oyg8t9Tp7A2KFSgAM60IeuCDsVlgNNoKzYHGYzyzSoeOpycQs50Fzs1dez6VNuxPg4Jg0UD8wezwNFGSu13jZXaKIrlKCfxOwCfQKNL1fpIn0eul83f3ad7xIINMMvEIADGxpOp5nTP7VoRuIOIR0O+zKQOk85NwczYNo0AQQGrwVHQtCu1VlU+sjfBzJ4xu5MdKrXap5zOqUC9NTuu8aadcn4lSlruReRy4QHcMSnr1ffrdwnmyoBK78uzT0B9oTJo5SLfEFM4L4MYIqGRBqAWq0U0Z31KRXne8MkQ5Pf68izOqeRE118U\\\\\\\/pqwI53jQPndvFpJwVF34iM6sTlz9e5YYEOGP7x2IniuujErOT9sAezvdZvhHnZQM3aAstIa3QpkbF290aaKWa3IcIgT28d+3PFW1RGg\\\\\\\/CkvOc6ab\\\\\\\/sZAUzvBEotVoVbcgMgy+ic6QjlNPoqy61IOKfTjmO+wXhZHxnKROf9rzPzGkqnjIJNZwkisbkE7O1Xe7Av0kjiI69g\\\\\\\/1P0pFqdARMUVHD\\\\\\\/uejndBd\\\\\\\/VW+pEbvQE8nsFQuANEr7NJzkcAoGJgJEEM0DpTZEONt9B\\\\\\\/lBaKkmtQ4tuwGJhWe+RMwfBDFLxeycg585oy+mlY\\\\\\\/c008\\\\\\\/cdw5TFoOtD9RcA7be6OxgL8qnnwReN1fW2xT+8k\\\\\\\/oaPB9Q3rpu7I8pHcw8aD1kySKB8ZkAwU0+xeYEaB+dpLLlrPNR2LOemXfaPeUqpcyBTWWAS6jkSkCAZlwV5ot9d9wu6+atqBDIONcBfQvB7tSrRrCyD3seFH1BB53\\\\\\\/ytr+JTZfSuNLWfvpyG7EPCz8FD31YCC0Wjrk\\\\\\\/\\\\\\\/VdwM1cxpk3igAYanBtJ1Alim4nm4mHOhE9ns3fG43Wj3RAIbpLG2bdby6cVaihRu3SgjXcbIRGxWNqtWjhnjLZ3KbLUKQ76LwZrEQRk8OjCRh\\\\\\\/m9icqYJBr1kLCW7nzXQkb7WkZqv3q73ooa9oT0MEorJFVQjjq4v2HphNorXbjyjRh+wO54jgIROMVZv4Tx756Mz6+31+17MUP5Mm\\\\\\\/Uq6Kdlyh3rvO20eiPFCH3zc92gJFjUCMeEZaQJYHlGxdUgIIfVSSE8wvHNqmxusDpe7SOsnKk7cRDh7vJZkhZNuUMfTn71wKvjxPY8O9\\\\\\\/iTBFTpTGZa5g6Uv2pV342JTPe7esSa+Vetx2WbIv02PEnRGzfrflfsxtG8tMTt0XI6I7QkWBEF+fikpXDnU\\\\\\\/R4F0xS1tZJZiyPekm3+CkbhJpRlV7p4KZCiO\\\\\\\/8pr7fKW\\\\\\\/8GBWUF4V+4BXjOoc7c9A0huP4dDMa51AAnVJsQO7snR7Nurj8iwkj7G8uO\\\\\\\/UuQW6AGpE96WefxNaylVcRz9YSrqIzNlLOHiMOPeHUtiPch3wtUuhYmPCvnPh\\\\\\\/0SmiGkgTqOOt\\\\\\\/xQKb\\\\\\\/Fi5U1fnxnwnECGDdq4nJ\\\\\\\/hgmP0GgxqrV2bARRPAm9ZCewbuVovz6OG26uNAR9DUl0iVEThwUDRCuP7qJYYYTpC8QI6PVFpY6CcG8py\\\\\\\/ghItYOjB7k+YKhf1w0c8fxB2UMFhStq0\\\\\\\/9TZF5ZSP1cmz4MDRpMJXlahkT662X6blEUHEVnU\\\\\\\/UrdQor8n3FIR8pMbTRP+dQi7yOco4T\\\\\\\/ktL7JqhtYi\\\\\\\/ScXgDHipnj41xj\\\\\\\/D5T1dcYQY9XL+bdYBc7Ab0yVhi9bKoiScXgfweT7bRnKyfWN5tBQBtWQ\\\\\\\/wQqQ+7SxgqaOdi3DGzqwx90FG0CzByozcsVI3UVkx9Dp1XXdEdBsi1u6eXhRshvZHtmgYg3HkIrYYgPO25HW3ms\\\\\\\/kaETJPgtF+6lJexOFp7K0X7gOwO1WJZgn1QcoRiT64XgItLUANo1W\\\\\\\/xRjPK51NKXfs2hwRSU\\\\\\\/nznm4eiLEk1StZbV7rZIcpUTaYKgF5cWsvq\\\\\\\/obiHftznBqCiloBZfSSGZdiKE6IdluFmiSiexgM\\\\\\\/8hnUdqCpyqhJCb9jj0WcPllwnCOaGLWpwNcy9q89pfZMa+IeKN6FqnYoIdAlBxX0omArzEdGOl\\\\\\\/2f2DRiZzuFaaO5rHVao9+kCpZP9S88FLx7ayu7EF9h5VYjtB0a65Vrwn\\\\\\\/bc8b8cXewO2BTPlUwcgR5VV85ZwGv6Kwn6bq\\\\\\\/4FsidQt1ed+WaZ6LngivdtjnN1X9mjDTKip4Ik3FMK0uXqRXh4Yu2xGGYs7m0LCGyti7wl7gbaVYPI+7ERRzkog5\\\\\\\/eZ8juJ2EkXkoqBDQakO9MjBQjI4g0Pc1GHEt0xBRtL4yddIymntPVgi5vYmiQ\\\\\\\/S6S4gWRXhxQ5JqBDp+pSzbePGsUt9FiW6u540mCwhQ6RU2ZmOgrt1tbVrQbYWZbXXeSJHx82m\\\\\\\/oPwuyN43h6cdWtaAdWLOTvUcAFZpUjsK2bGKKzqn6eDuhn0vMbeMAhAkDJOFyXjuv2Yc60EsMbUdr1IqwORoXSyLV2cD1iiUkQA2fwTU9dJzJV5E\\\\\\\/8SFVy\\\\\\\/yxPlT7nKeLsMbLk25Z2pxdSX8Ysm\\\\\\\/sm4d+tWCLUbPtseCIJ\\\\\\\/L5AgvTFQQbXNOwAGEAsp03ceDG7Ent4JwXLxd6YVY7CYbPGRPJuwthqOwn1M5B22DPcy+RhGsRa2LwJVwWOJ2ojVLMM8EIaAwPosKD1YQ8qc0ntr+krQxRFwDE4rIqsB9y33fvjvPsChgTuHM2D18WC1zz7hvVXjoLWo+y4E2xOF1aE0BtIU\\\\\\\/gbFCXGadtID81f5CpKARkbWfpZ4o0vvDaFWOjiaeWoVaLLXz+109GVsgyHKcugkcbMFCalpsp+wqPUw0hbAlg4Zh1OajjbQHJIOq+hmwzhm5fIdYo+3P9dmLieqfZ9cYjvmaS3GuI1vWbA\\\\\\\/KT4lZm\\\\\\\/D+p2ySj9TiMQiVZAYiMXkLz4C+9eM9dGhdmf5m5DmTWAbU+OmBCEWzMtLzSzyYn+lnBTcorciAApUZDnrvil\\\\\\\/g+xkD2eI2D5Qws3NK\\\\\\\/dQSKsovmVLyE\\\\\\\/f7i\\\\\\\/IXcUBl\\\\\\\/mT4hHdSLUkw3Wcz651fQNBPO9vjPQZCQUDqsZeYdtKbXV\\\\\\\/BbQbwQId79qU\\\\\\\/3gxh97daqFtLzeeozGRIpmK+c2Pq55+6xbo8AJzXKFVj1tNo02Qjldujtl12nvdiDrlcVIFWLQzGcI5otphK96VX+7eo0bJPh9jINesls6Fje2gc\\\\\\\/xDCRfUrc7mw2G2MDalkjwDzYQPbDnBodedmOMbQefV1HVTOuFyaK8bURfd0bEmQALdBNOmWXcD11AdOkHKP+palmqgO6HR5ogN2h0ZX8Gxsd\\\\\\\/r+t\\\\\\\/8iU8LW7N3Td5T+BuZvHSQiQqo0X7GwUq860HXt\\\\\\\/9V9517XWFwNn8qR2gFtZUX9jq8B\\\\\\\/j8exTaO9PlL7bRuYy\\\\\\\/BLldkyLmCgz8sfZ1KpUvkg5VorBSRd\\\\\\\/z9gf0lIgGUvceBaNTfm6NMOhSDUgCeRSe\\\\\\\/RpQE519aBxE0QSyo3fBeWZCMZ1Zf7bMKEhYVssKYYGaVr3AtXq96CBxGNlwNiOCWmUOpjdqDuAOmjQI9GgpmzcoBHr0VClxz7\\\\\\\/KhxHnYYF5UC0HgUrrtPg8SwvtyJWBKaHdime43Ag2m6aQF7k3wXFzDJEtYjsWXtX6UfZgl8nFn9rFFC5NPwWxYxMyEoiiia88vREclFMa7zrf0ogKSczxaSSrGqXGRoI603B74\\\\\\\/hwo0BlImF3QWz7bWgQ7TbPJEPXnuawjufZTEbKiA1NVe86gJeZWmpkSjZGd2380xeVNAf92hKqWJ4CAmysJFaGILo2P6FYdmPZVDv59Fbx1jrMLrYFNnccTxJ6V9U8o+BKJPsuV3Y613QTBfnj5FYmeCV0rALS+2glBL4V9B4DLR6aDGC+zzUVAI5SsYcgy\\\\\\\/pSxI0CyMF1EWZmkwCpK03wGNRCuqzVBwP19mTLh9vY2vvGWxSaD9TlPr8UG5xh3YvBDvMcT+1PUpmk4+JgGrrsimm2Nu8GLfSeezhOtI1fWUSarWEbPEPA+LMmdtXvUKZhsp1p0qX1f0jp0sNum10oSk+i8naJ0zGY7q7uARymbiAnyFxgPZj+hWdyNJeY\\\\\\\/TrUBWz+QDMxfL4PCvlDTNZLA44\\\\\\\/Rdw\\\\\\\/dkn9PpdX4uifX8NuQnpFw+Gs+0Vfrn26cJPRS8bka2QDEXjfGgIti\\\\\\\/i7+DzgTFthGlc6LDHnnwnAxnrqbIgYHSzfs524RVh\\\\\\\/yq4Tot3Ybh6FyvY8nmX6bbHFmjoTD84m4jVrX\\\\\\\/l9PjFnyJ\\\\\\\/b+pD2bCkRRgUcEO0ALuUE7riFmhavzQvzgD7ib3WIDpmZLxiv\\\\\\\/CcPLUbtdv7AQHwQDJYQeKKITj1HLtQrjRfoi9QnQVQUfYMme23rTMRBXa3L\\\\\\\/VT9jk4HlVLhZojkDM9Zz7LsI+v\\\\\\\/\\\\\\\/OLrQMaDdQ07snx0k6km3TOxINHI6F77VDnWYB3W0tRIVRiHYEuXltFOUV\\\\\\\/NLHpC9jPMCMLAGmlwFg4b5B\\\\\\\/5YXyciyaMu1txu0+aBPlyGk5M7q\\\\\\\/Ws+vOfGwNP+XdbqYwze6ujfAmg3ZPRYzm4i\\\\\\\/uNe43wXnkrz9yK4QLY3DWd79Zx+0Q9mSmbGpK3CCaWlmzattDwHVcYvIPINZjtyCzK5lSxndfc6aJBL+pjQcayfKeyhh4aeef7kPKdeAOB++l4u1XqJdffF1cVYPpZ4t7tktQCNKL7gAgAL9YzYrO\\\\\\\/PDGJOSwgYe6bzOMa8IFc1s9vFRUbdUR1iNeMVgbiw1hSzoL+h9SnC7ckrc1gkS2pE3aUW7icItmIUc8E7G3kHRHdKm8JCwxeblvlXAOZQ5JdLoiDL2\\\\\\\/REtdGCe5j4ErW31Mbn3sGjjOMUkwvUCTFHLtAGatE2T2v\\\\\\\/3l2c4Z5R\\\\\\\/mN6Mme3EHTe6y2bAGmbLQeMc6HRDhrQsDQmor66+4dDByEbFjlEfAvMZG+N6IlTehboBOoI5bsKwOIeouBWOubcHb\\\\\\\/j\\\\\\\/mvYpT3KXK8ql5UVywv+XyMN5EByc1TSCuSZcJyt+9lJo4o5cI2SDzbLZWv6Rf5kPXot9usL86Egc\\\\\\\/v0xBwlJNw5Mx15HEIEmDSDmSU7sgOc+mPyf3+tPmTd884sdGRdeN\\\\\\\/uvxLz9h0gkdHOnePMmVBLTRn7owHC2RuMzJ7hZq6nHllrbIkgOsnPkVnTBeSjVnVrmqGYbIVC6+KJtlY6VLNQgXami2lbAnO39a5U4gLikllM3DjTmqI8BLyuJjnh\\\\\\\/y9U\\\\\\\/IaIPAantLPSs5E04hT9IBqHe7W5NRHZ8N\\\\\\\/6Oph9YTxUQyJrBNO8Dew5yf7y4wi4gtnIduCeDDvdNH\\\\\\\/\\\\\\\/NasNHOLQ6EpxnUaeX488FG\\\\\\\/ku2z3F3oEJS\\\\\\\/YiDKFufdicCIinhLLpzb5+QrHt5nVzKUnmAkotHcLDU2Oz9vuO4VThPGutOijZgqfkKUSCcA04H6jGteb\\\\\\\/BOemUj53r60k5xTcNzro9kNSxs8+gWR94Dj5L8A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022yjQrYKpvt5sJCEinh7sntQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a7e3c9898f313cad1d764bafec9293a2241a4f7dd0c4626852b240fd482afbd3\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022div2VgjtErHT+a0qpLwGmNkXBXvGmrc00BTYMhgDS5uSr2t4n1IOFrjZMj\\\\\\\/PMWuPl3YV5uT7PufMlc6bYzIfO86CQGUgEeZfZetxHLeaS4LEUeIBIdaDjwh\\\\\\\/xyammjxuUvmMr4Rn3dTyosVxvjZw6olzyuZHt67Km5tk2b3r879fwukceHvWZ0C1eq4VujBoIhts48gqecHRJDPUCiU2zc3DoBwmCaanKoCFWv4E0zT5lgNcvMbGevIJt6Dr7pMVIbRL\\\\\\\/5AS3h8i4nwPwPUwb1oqdNJbXAiF5G4\\\\\\\/AuxJn8fOJjhNmZrcBO3e4JIu\\\\\\\/m6sFukxIoxUJP61rplWg9Ov6DPHEQoYPu4EMw8gD1eu2Ev\\\\\\\/5dZxTLJQVWR2mV+zujMkkv9jjX7huBgVUJHBDvbpMG6ys1EiA+yTvPCgz8NcFRxYCNASPSPfEYcYYixp2RCP2ahoXPnEboBNElVS35bliNguBU7k4+uOtwNpYo2Oj2ojLISnrW+nY\\\\\\\/83BgYXXlagcHBLpcy3e+WSZxbBM4FtuQ2eNOL4JGPxgyTe1XvFGl6vNveO4mMuCnkBE8lkRgQct5w4HFr+3rZrZF8VbgPu9bLuGXQ6u53Mfa8DEb8r8a+50WQZuygGzqfCPxDxVFdIrgr03NNc1uys6TbWRhqdi1BpPl7Y0sWzhp7hEmxjosG\\\\\\\/LTYmqgmLOecf7vqhumdN2VW6\\\\\\\/UAbXOWPnpnglXCTXXGsVe44ahbSfNA0Eq1xWbuKsqGTExP+1Ar5XYlA9LNcOcyvLCla2CXOqV8tDTao56FFMnoqJQAYvcwXgjflhZ1mY04Dn0EwMBt\\\\\\\/t+kbJcYirhD2XMnGmrKFmcO4FUBJmunw\\\\\\\/QgSEeQfv9oS4TI8JPoQj6nuYAMuwfSd0HseJ1HucqvO7HMy3PhgA\\\\\\\/Oj8jfKe920c\\\\\\\/75jUh4qMkaoIeQGYgLTFya5skwQqXosoJK0WzUYSKpjt5OksqsvLS8gHe6HwNN\\\\\\\/cbbH39sHBJZChqhzwRv+H\\\\\\\/ICaj\\\\\\\/WeS5cksNL4RFZ6x32Gcn38MtpmVpxoEzbOnE98\\\\\\\/IRQlE9yHZHfmeHcEL1sh1j4vhJHgxIRci4nlKxFe53osvwaJ49nhL4tXkDm6v7sdVqH\\\\\\\/NFUAD\\\\\\\/HE5y74pKikfvLyqi\\\\\\\/FOmoV4oa7RtXEaS7L1\\\\\\\/4pYLrqbA8na\\\\\\\/gf4dIPd434u+1KmwMtSUEeGc+bzzvnatXuUaFNmDWcYSaKS79wM+pNX+M1A1bj5vXgtuJ8APGJujPZrG5NRdd6OMZh7PLAOHG9lzLf5NWJlX1y\\\\\\\/LrWfAmKGphsnIdeHY3+VGeTTMbnZ4MW1VJHxIz38AjvM62FKVI+ZtHL2dwRSf10fS9SQV87Ukes6YQL9lt2LmsO4KEe1S9JNxTxYqjWP2YuqhObLjAvivG0NmoNN0HUzE+T7OGMUqik7xI7BLqpdl+Th6HmrxLmN7Kk2Ix4GyNjv9pcArdzRpozTyizTCVXIwZbP7F2PguYKjoQhjv2ayhOzF0p7mvlLWIQiqf7hHnIOiKLHhs+ZKEhs7GxkpBVIs3EiYkRiqLwxR+4Bl4A2pUAdthLRteDd7fum1Hp\\\\\\\/VMmiYFtHH\\\\\\\/nFl1fPxTjkheL2yBTgxezvlvGklT4SLUZtDktvhu7f9zjj+G4Xdvdz1J5wiAsibvAIdkH8ShfNhLY7HKjiOwaoeIPqUQo7iK6FK1RQaFIJUE8f5G1s5NOSS+OEp3yrzx4Lh7jzQN6+LCmelXG+jK5enl6BcaAsWpoBsyJcHufDh1UhkIzEWeZ0WmNtv530ie7ZgAcC4Jol3D19A1po6IzyiilECfW\\\\\\\/hpIoNGgqFLFSwkhb2rfGNMecLxRyaD7NBweIzXPEUakVpZa9Qn49WimrzO4OEKz\\\\\\\/Hcv0m7ydhKrMLCz4SKuhpKbPwjQCU5ay3u3j3IMhEnG5DEZZCWtlk4RF5+GtrSKrmlJEu7\\\\\\\/NbvAhOSliKt0foWIfALdHXXTaGu8QpRqq6JHb5USUKoAR1yFZYcYnHdrdWQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00222QTev5KBWua5wslw7luj4Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002232088d90e2b3f9710906bfab6655da9a47bf052a9fdc5824cfb1f74548993c66\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-486783 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3543" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Eco9_lMt-IaDneGTQfFNj_wO8lNIf7vvZlNq1_sGimA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3543" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-I79qAKgsgxNMc4EmMdXjcr67Piupd59H7BmaB4v_wk4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/10/3533" title="Single-Step qPCR and dPCR Detection of Diverse CRISPR-Cas9 Gene Editing Events in Vivo" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/10/3557" title="Corrigendum" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486783" id="node486783" data-pisa="ggg;7/10/3543" data-pisa-master="ggg;g3.117.300151" data-apath="/ggg/7/10/3543.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Relatedness Strengthens Genetic Connectedness Across Management Units</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Haipeng Yu</span>, <span class="highwire-citation-author" data-delta="1">Matthew L. Spangler</span>, <span class="highwire-citation-author" data-delta="2">Ronald M. Lewis</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3567-6911" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gota Morota</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3543-3556;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300151 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486783" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Haipeng Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Animal Science, University of Nebraska-Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haipeng%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaipeng%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Matthew L. Spangler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Animal Science, University of Nebraska-Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BL.%2BSpangler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spangler%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BL.%2BSpangler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ronald M. Lewis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Animal Science, University of Nebraska-Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ronald%2BM.%2BLewis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lewis%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARonald%2BM.%2BLewis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Gota Morota </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Animal Science, University of Nebraska-Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gota%2BMorota%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morota%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGota%2BMorota%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3567-6911" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gota Morota</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:morota@unl.edu" class="" data-icon-position="" data-hide-link-title="0">morota@unl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/10/3543" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:486783" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:486783/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3543.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:486783" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:486783/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3543.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:486783" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:486783/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3543.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:486783" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:486783/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/10/3543.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:486783" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:486783/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genetic connectedness refers to a measure of genetic relatedness across management units (<em>e.g.</em>, herds and flocks). With the presence of high genetic connectedness in management units, best linear unbiased prediction (BLUP) is known to provide reliable comparisons between estimated genetic values. Genetic connectedness has been studied for pedigree-based BLUP; however, relatively little attention has been paid to using genomic information to measure connectedness. In this study, we assessed genome-based connectedness across management units by applying prediction error variance of difference (PEVD), coefficient of determination (CD), and prediction error correlation r to a combination of computer simulation and real data (mice and cattle). We found that genomic information (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-1.gif"/></noscript></span></span>) increased the estimate of connectedness among individuals from different management units compared to that based on pedigree (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-2.gif"/></noscript></span></span>). A disconnected design benefited the most. In both datasets, PEVD and CD statistics inferred increased connectedness across units when using <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-3.gif"/></noscript></span></span>- rather than <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-4.gif"/></noscript></span></span>-based relatedness, suggesting stronger connectedness. With r once using allele frequencies equal to one-half or scaling <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-5.gif"/></noscript></span></span> to values between 0 and 2, which is intrinsic to <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-6.gif"/></noscript></span></span> connectedness also increased with genomic information. However, PEVD occasionally increased, and r decreased when obtained using the alternative form of <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-7.gif"/></noscript></span></span> instead suggesting less connectedness. Such inconsistencies were not found with CD. We contend that genomic relatedness strengthens measures of genetic connectedness across units and has the potential to aid genomic evaluation of livestock species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/coefficient-determination" class="hw-term hw-article-keyword hw-article-keyword-coefficient-of-determination" rel="nofollow">coefficient of determination</a></li><li class="kwd"><a href="/keyword/genomic-connectedness" class="hw-term hw-article-keyword hw-article-keyword-genomic-connectedness" rel="nofollow">genomic connectedness</a></li><li class="kwd"><a href="/keyword/prediction-error-correlation" class="hw-term hw-article-keyword hw-article-keyword-prediction-error-correlation" rel="nofollow">prediction error correlation</a></li><li class="kwd"><a href="/keyword/predication-error-variance-difference" class="hw-term hw-article-keyword hw-article-keyword-predication-error-variance-of-difference" rel="nofollow">predication error variance of difference</a></li><li class="kwd"><a href="/keyword/relatedness" class="hw-term hw-article-keyword hw-article-keyword-relatedness" rel="nofollow">relatedness</a></li></ul><p id="p-3">The problem of connectedness or disconnectedness is particularly important in genetic evaluation of managed populations such as domesticated livestock. When selecting among animals from different management units (<em>e.g.</em>, herds and flocks), caution is needed; choosing one animal over others across management units may be associated with greater uncertainty than selection within management units. Such uncertainty is reduced if individuals from different management units are genetically linked or connected. In such a case, BLUP offers meaningful comparison of the breeding values across management units for genetic evaluation (<em>e.g.</em>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Kuehn <em>et al.</em> 2007</a>).</p><p id="p-4">Structures of breeding programs have a direct influence on levels of connectedness. Wide use of artificial insemination (AI) programs generally increases genetic connectedness across management units. For example, dairy cattle populations are considered highly connected due to dissemination of genetic material from a small number of highly selected sires. The situation may be different for species with less use of AI and more use of natural service mating such as for beef cattle or sheep populations. Under these scenarios, the magnitude of connectedness across management units is reduced and genetic links are largely confined within management units.</p><p id="p-5">Pedigree-based genetic connectedness has been evaluated and applied in practice (<em>e.g.</em>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kuehn <em>et al.</em> 2009</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Eikje and Lewis 2015</a>). However, there is a relative paucity of use of genomic information such as single nucleotide polymorphisms (SNPs) to ascertain connectedness. In what scenarios genomics can strengthen connectedness, and how much gain can be expected relative to the use of pedigree information alone, still remains unknown. Connectedness statistics have been used to optimize selective genotyping and phenotyping in simulated livestock (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Pszczola <em>et al.</em> 2012</a>) and plant populations (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Maenhout <em>et al.</em> 2010</a>), and in real maize (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Rincent <em>et al.</em> 2012</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Isidro <em>et al.</em> 2015</a>) and rice data (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Isidro <em>et al.</em> 2015</a>). These studies concluded that the greater the connectedness between the reference and validation populations, the greater the predictive performance. However, (1) connectedness among different management units and (2) differences in connectedness measures between pedigree and genomic relatedness were not explored in those studies. For better understanding of genome-based connectedness, it is critical to examine how the presence of management units comes into play. For instance, genomic relatedness provides relationships between distant individuals that appear disconnected according to the available pedigree information. In addition, it captures Mendelian sampling that is not present in pedigree relationships (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Hill and Weir 2011</a>). Thus, genomic information is expected to strengthen measures of connectedness, which in turn refines comparisons of genetic values across different management units. The objective of this study was to assess measures of genetic connectedness across management units with use of genomic information. We leveraged the combination of real data and computer simulation to compare gains in measures of connectedness when moving from pedigree to genomic relationships. First, we studied a heterogenous mice dataset stratified by cage. Then, we investigated approaches to measure connectedness using real cattle data coupled with simulated management units to have greater control over the degree of confounding between fixed management groups and genetic relationships.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mice data</h3><p id="p-6">We analyzed a heterogeneous stock mouse population established for quantitative trait mapping (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Solberg <em>et al.</em> 2006</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Valdar <em>et al.</em> 2006</a>). It was originally derived from eight inbred strains (DBA/2J, C3H/HeJ, AKR/J, A/J, BALB/cJ, CBA/J, C57BL/6J, and LP/J), followed by 50 generations of pseudorandom mating. This process introduced recombinants that allow high-resolution mapping (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Solberg <em>et al.</em> 2006</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Valdar <em>et al.</em> 2006</a>). This population was used for one of the first empirical applications of genomic selection in animals (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Legarra <em>et al.</em> 2008</a>) and later used for an array of quantitative genetic studies. The data consisted of 1884 individuals from 169 full-sib families with ∼11 siblings per family. Each individual was genotyped with 10,946 SNPs, yet none of the full-sib parents were genotyped. We removed SNPs with a minor allele frequency (MAF) &lt; 0.05, resulting in 10,339 markers for analysis. The mice were reared in 523 cages or management units that created shared environments. The majority of full-sibs were housed in the same cages and distributed to three cages on average, <em>i.e.</em>, a full-sib family was typically reared together in three cages. Pedigree relationships within and across full-sib individuals were 0.5 and 0, respectively. This resulted in an extreme case of genetic disconnectedness across management units. Thus, the extent of connectedness was determined by the presence or absence of full-sibs in different management units.</p></div><div id="sec-3" class="subsection"><h3>Cattle data</h3><p id="p-7">Pedigree information of dairy cattle was available on 1929 cattle collected over six generations starting from a base generation 0 to generation 5 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wimmer <em>et al.</em> 2015</a>). Among those, 500 individuals, mostly coming from generations 2 and 3 (<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-8.gif"/></noscript></span></span>90%), had both phenotypes and genotypes. Historic pedigree information in addition to the 500 individuals are a source of connectedness as the pedigree-based relationship matrix was constructed from the entire pedigree. The 500 individuals were genotyped for 7250 SNP markers. The average missing rate of genotypes across the entire SNP was 0.0002. We imputed missing genotypes by sampling alleles from a Bernoulli distribution with the marginal allele frequency used as a parameter. We retained 6714 SNP after removing markers with MAF &lt; 0.05. We simulated management units in two steps: (1) individuals were clustered and (2) clusters were assigned to management units. The k-medoid clustering was performed to cluster individuals into distinctive groups. In particular, we used partitioning around medoids, which is considered a robust version of K-means (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Kaufman and Rousseeuw 1990</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Reynolds <em>et al.</em> 2006</a>). We formed sets of clusters so that individuals in the same groups were more similar to each other than to those in other groups. We selected the number of clusters by optimum average silhouette width algorithm implemented in the cluster and fpc R packages. This algorithm minimizes dissimilarity measures among individuals within the same cluster using the Euclidean metric and finds the optimal number of clusters that returns the lowest average dissimilarity computed from each cluster. The clustering was based on the <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-9.gif"/></noscript></span></span> matrix, which was converted to a dissimilarity matrix by calculating the distance from the highest similarity to each similarity value in such a way that the relationship with the largest value becomes zero. We simulated the four following scenarios.</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-8">Scenario 1: Completely disconnected, all clusters allocated to their own management units.</p></li><li id="list-item-2"><p id="p-9">Scenario 2: Disconnected, one-half of clusters allocated to management unit 1 and remaining half assigned to management unit 2.</p></li><li id="list-item-3"><p id="p-10">Scenario 3: Partially connected, approximately one-third of clusters allocated to management unit 1, another one-third to management unit 2, and the remaining one-third of clusters assigned to both managements to act as a link to connect the two management units indirectly.</p></li><li id="list-item-4"><p id="p-11">Scenario 4: Connected, all clusters equally allocated to the two management units.</p></li></ul><p id="p-12">Subsequently, appropriate incidence matrices were constructed and we computed connectedness statistics across management units employing pedigree and genomic relationships.</p></div><div id="sec-4" class="subsection"><h3>Prediction error variance (PEV)</h3><p id="p-13">Genetic connectedness statistics are typically defined as a function of the inverse of the coefficient matrix. For instance, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Kennedy and Trus (1993)</a> proposed a genetic connectedness measure as the average PEVD in predicted genetic values between all pairs of individuals in different management units. The PEV can be obtained from Henderson’s mixed model equations (MME) (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Henderson 1984</a>). We constructed MME according to a standard linear mixed model <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-10.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-11.gif"/></noscript></span></span> is a vector of phenotypes, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-12.gif"/></noscript></span></span> is an incidence matrix of management units, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-13.gif"/></noscript></span></span> is a vector of effects of management units, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-14.gif"/></noscript></span></span> is an incidence matrix relating individuals to phenotypic records, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-15.gif"/></noscript></span></span> is a vector of random additive genetic effects, and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-16.gif"/></noscript></span></span> is a vector of residuals. The phenotypic vector <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-17.gif"/></noscript></span></span> was standardized to have mean of 0 and variance of 1 so that results could be compared across different scenarios. The variance–covariance structure for this model is<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-18.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-19.gif"/></noscript></span></span> is the genetic variance, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-20.gif"/></noscript></span></span> is the residual variance, and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-21.gif"/></noscript></span></span> is a positive (semi)definite relationship matrix defined later.</p><p id="p-14">The inverse of the MME coefficient matrix is represented as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-22.gif"/></noscript></span></span>where λ is the ratio of variance components <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-23.gif"/></noscript></span></span> The PEV of genetic value for the <em>i</em>th individual (<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-24.gif"/></noscript></span></span>) is given by<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-25.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-26.gif"/></noscript></span></span> is the <em>i</em>th diagonal element of <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-27.gif"/></noscript></span></span> coefficient matrix. Note that PEV can be interpreted as the proportion of additive genetic variance not accounted for by the prediction. Equivalently, the matrix of PEV can be computed as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-28.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-29.gif"/></noscript></span></span> is the absorption (projection) matrix for fixed effects where <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-30.gif"/></noscript></span></span> which is orthogonal to the vector space defined by <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-31.gif"/></noscript></span></span> (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-32.gif"/></noscript></span></span>). This avoids calculating the inverse of the entire coefficient matrix, which is useful when the number of columns of <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-33.gif"/></noscript></span></span> is large or analysis involves repeated computation of PEV.</p></div><div id="sec-5" class="subsection"><h3>Genetic connectedness</h3><p id="p-15">We computed three genetic connectedness statistics: the PEVD between genetic values (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Kennedy and Trus 1993</a>), the CD of the difference between predicted genetic values (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Laloë 1993</a>), and the r between genetic values of individuals from different management units (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lewis <em>et al.</em> 1999</a>). The first two statistics were originally used to evaluate the accuracy of individual estimated breeding values and later extended to assess inherent risk in comparing individuals across management units. First, genetic connectedness between two individuals, <em>i</em> and <em>j</em>, was measured as PEVD (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Kennedy and Trus 1993</a>)<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-34.gif"/></noscript></span></span>where PEC<span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-35.gif"/></noscript></span></span> is the prediction error covariance (PEC) or covariance between errors of genetic values, which is the off-diagonal element of the PEV matrix. If PEVD is small, individuals are said to be connected. The idea behind using PEVD as a measure of connectedness is that the accurately estimated genetic values of individuals have smaller PEV and that the pairs of genetically related individuals in the different management units have a positive PEC. Throughout this study, we used a scaled PEVD following <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Kuehn <em>et al.</em> (2008)</a> by scaling PEVD by the additive genetic variance to express connectedness without units of measurement.</p><p id="p-16">Similarly, CD is closely related to PEVD and is defined by scaling the inverse of the coefficient matrix by corresponding coefficients from the relationship matrix. We can view CD as the squared correlation or reliability between the predicted and the true difference in the breeding values (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Laloë <em>et al.</em> 1996</a>). This is given by<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-36.gif"/></noscript></span></span>for pairwise comparison. In contrast to PEVD, CD accounts for the reduction of connectedness due to relationship variability between individuals under comparison. This statistic is bounded between 0 and 1, with larger values indicating increased connectedness.</p><p id="p-17">The r is obtained by transforming a PEV matrix into a predictive error correlation matrix. For individuals <em>i</em> and <em>j</em>, this statistic is given by<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-37.gif"/></noscript></span></span>The rationale behind r is that there is no connectedness when PEC is zero (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Lewis <em>et al.</em> 1999</a>). Similar to CD, r is also bounded between 0 and 1. The larger the r, the greater the connectedness.</p></div><div id="sec-6" class="subsection"><h3>Connectedness summary</h3><p id="p-18">We can generalize connectedness between any pair of management units <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-38.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-39.gif"/></noscript></span></span> by setting up a corresponding contrast vector <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-40.gif"/></noscript></span></span> that sums to zero (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-41.gif"/></noscript></span></span>) (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Laloë 1993</a>). The PEVD of contrast <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-42.gif"/></noscript></span></span> in genetic values is given by<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-43.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-44.gif"/></noscript></span></span> is a column vector including <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-45.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-46.gif"/></noscript></span></span> and 0, for the elements corresponding to <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-47.gif"/></noscript></span></span>th unit, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-48.gif"/></noscript></span></span>th unit, and the remaining units, respectively, where <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-49.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-50.gif"/></noscript></span></span> were the numbers of individuals belonging to <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-51.gif"/></noscript></span></span>th and <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-52.gif"/></noscript></span></span>th units, respectively. In a contrast vector notation, pairwise CD between management units <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-53.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-54.gif"/></noscript></span></span> is given by<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-55.gif"/></noscript></span></span>For the r statistic, a similar summary statistic can be derived as<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-56.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-57.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-58.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-59.gif"/></noscript></span></span> were the sums of the elements of <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-60.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-61.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-62.gif"/></noscript></span></span> respectively (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Kuehn <em>et al.</em> 2008</a>). However, in the <em>Appendix</em> we show that when this summary statistic is applied across units it provides a reasonable summary for a pedigree relationship matrix, but it is not suitable for a genomic relationship matrix when the total number of management units is two. Thus, we reported connectedness by averaging the r statistic for all pairs of individuals across management units.</p></div><div id="sec-7" class="subsection"><h3>Relationship matrix</h3><p id="p-19">Connectedness is realized through a genetic relationship matrix under the BLUP framework. Three genetic connectedness statistics defined above require information about covariance structures among individuals or genetic values that evaluate relatedness. We considered five <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-63.gif"/></noscript></span></span> relationship kernel matrices (<span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-64.gif"/></noscript></span></span>) in this study, where <em>n</em> is the number of individuals. The numerator relationship matrix, <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-65.gif"/></noscript></span></span> is based on relatedness due to expected additive genetic inheritance. This can be computed directly from pedigree information, and reflects the probability that alleles are inherited from a common ancestor and thereby are identical by descent (IBD). The off-diagonal elements are twice the kinship coefficients and are equivalent to the numerators of Wright’s correlation coefficients (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Wright 1921</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">1922</a>). The majority of genetic connectedness literature is based on the pedigree relationship matrix, <em>i.e.</em>, average relationships assuming conceptually, an infinite number of loci. On the other hand, the genomic relationship matrix, <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-66.gif"/></noscript></span></span> captures genomic similarity among individuals. The matrix <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-67.gif"/></noscript></span></span> is a function of the matrix of allelic counts <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-68.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-69.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-70.gif"/></noscript></span></span> denote the indices of individuals and of markers, respectively. Each element of the allele content matrix <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-71.gif"/></noscript></span></span> is the number of copies of the reference allele. Under Hardy–Weinberg equilibrium, <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-72.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-73.gif"/></noscript></span></span> so that <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-74.gif"/></noscript></span></span> is a standardized incidence matrix of allelic counts, where <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-75.gif"/></noscript></span></span> is the allele frequency at the <em>j</em>th marker. The <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-76.gif"/></noscript></span></span> matrix is constructed from a cross-product of scaled marker genotype matrix <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-77.gif"/></noscript></span></span> divided by some constant, <em>i.e.</em>, the number of markers under assumption of unity marker variance<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-78.gif"/></noscript></span></span>The standardization of <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-79.gif"/></noscript></span></span> and the constant in the denominator make the <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-80.gif"/></noscript></span></span> matrix analogous to the <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-81.gif"/></noscript></span></span> matrix (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">VanRaden 2008</a>). This genomic relationship matrix estimates the proportion of the genomes of two individuals that is identical by state (IBS).</p><p id="p-20">One concern that arises when comparing the <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-82.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-83.gif"/></noscript></span></span> matrices is that these two matrices are not on the same scale. The <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-84.gif"/></noscript></span></span> matrix represents the estimate of a covariance (correlation) structure among individuals marked by SNPs with the potential having some negative off-diagonal entries. Such negative values indicate that some individuals are molecularly less related than average pairs of individuals in the sense of IBS if the population were in Hardy–Weinberg equilibrium (<em>e.g.</em>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Toro <em>et al.</em> 2002</a>). This mostly happens when the current population is defined as a base population, namely, computing the <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-85.gif"/></noscript></span></span> matrix by using the estimates of observed allele frequencies from the current population (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Powell <em>et al.</em> 2010</a>). While the negative coefficients arising from IBS can be interpreted as negative correlations of alleles (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Toro <em>et al.</em> 2002</a>), this is in contrast to the <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-86.gif"/></noscript></span></span> matrix, which is defined as an IBD. In the <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-87.gif"/></noscript></span></span> matrix, a founder population is assumed to be the unselected base population. This may impact some of the connectedness statistics used in this study. For this reason, we also considered two other genomic relationship matrices: a <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-88.gif"/></noscript></span></span> matrix and a scaled <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-89.gif"/></noscript></span></span> matrix, <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-90.gif"/></noscript></span></span> so that the genomic relationship matrix is on nearly the same scale as the <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-91.gif"/></noscript></span></span> matrix. The <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-92.gif"/></noscript></span></span> matrix was created by scaling the <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-93.gif"/></noscript></span></span> by <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-94.gif"/></noscript></span></span> instead of <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-95.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-96.gif"/></noscript></span></span> is the estimate of allele frequency in the base population. Because allele frequencies in the base population are unknown, we set all <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-97.gif"/></noscript></span></span> equal to 0.5 under the assumption of no selection (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">VanRaden 2007</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Toro <em>et al.</em> 2011</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Vitezica <em>et al.</em> 2011</a>). The <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-98.gif"/></noscript></span></span> matrix constructed in this way does not create any negative coefficients for either the mice or cattle datasets. The correlations between <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-99.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-100.gif"/></noscript></span></span> (defined as correlation between elements of upper triangular matrix including diagonals) were 0.81 and 0.98 for mice and cattle, respectively.</p><p id="p-21">Alternatively, a min–max scaler, one of the common scaling methods, was employed to scale the <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-101.gif"/></noscript></span></span> matrix. The min–max scaler transforms inputs into the given range of minimum and maximum values. The scaled genomic relationship between <em>i</em>th and <em>j</em>th individual was given by<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-102.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-103.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-104.gif"/></noscript></span></span> are the minimum and maximum elements of <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-105.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-106.gif"/></noscript></span></span> is the <em>i</em>th, <em>j</em>th element of <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-107.gif"/></noscript></span></span> The <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-108.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-109.gif"/></noscript></span></span> define the range of minimum and maximum values of elements of <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-110.gif"/></noscript></span></span> These values were set to 0 and 2, respectively, according to the minimum and maximum values of numerators of Wright’s correlation coefficients. This scaling sets negative off-diagonal entries in the <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-111.gif"/></noscript></span></span> matrix to 0 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Momen <em>et al.</em> 2017</a>). Note that the correlation between <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-112.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-113.gif"/></noscript></span></span> is equal to one because a correlation is invariant to changes in scale.</p><p id="p-22">Lastly, the covariance between ungenotyped and genotyped individuals was jointly modeled through a hybrid matrix where <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-114.gif"/></noscript></span></span> The <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-115.gif"/></noscript></span></span> matrix can be viewed as a matrix that combines pedigree and genomic relationships. By considering the distribution of genetic values of ungenotyped individuals conditioned on genetic values of genotyped individuals, it can be shown (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Legarra <em>et al.</em> 2009</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Christensen and Lund 2010</a>) that<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-116.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-117.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-118.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-119.gif"/></noscript></span></span> are numerator relationship matrices among ungenotyped, ungenotyped and genotyped, and genotyped individuals, respectively. <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-120.gif"/></noscript></span></span> is the genomic relationship matrix for genotyped individuals. In addition to <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-121.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-122.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-123.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-124.gif"/></noscript></span></span> the <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-125.gif"/></noscript></span></span> matrix was used for the cattle dataset that spans several generations. We treated individuals at generations three, four, and five as genotyped individuals, and earlier generations as ungenotyped individuals. This reflects a practical situation in typical breeding programs, where the majority of genotyped individuals are concentrated in more recent generations. This partitioning resulted in 65% ungenotyped and 35% genotyped individuals, simulating a realistic scenario where there are more ungenotyped than genotyped individuals (<em>e.g.</em>, <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Legarra <em>et al.</em> 2009</a>).</p></div><div id="sec-8" class="subsection"><h3>Principal component analysis (PCA) of measures of connectedness</h3><p id="p-23">PCA of PEVD, CD, and r pairwise individual-based matrices computed under the four different simulated scenarios in the cattle dataset was used to cluster individuals. The prcomp function in R was used to produce principal component (PC) scores and the PC plots were generated with the ggbiplot package based on the first two PC.</p></div><div id="sec-9" class="subsection"><h3>Heritability</h3><p id="p-24">For simulation, we used two heritability values (<span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-126.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-127.gif"/></noscript></span></span>) by varying the ratio of variance components <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-128.gif"/></noscript></span></span> assuming an animal model, where <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-129.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-130.gif"/></noscript></span></span> are residual and genetic variances, respectively.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-25">The mouse dataset is available at <a href="http://wp.cs.ucl.ac.uk/outbredmice/heterogeneous-stock-mice/">http://wp.cs.ucl.ac.uk/outbredmice/heterogeneous-stock-mice/</a> and the cattle dataset is downloadable from the synbreedData R package at <a href="https://cran.r-project.org/web/packages/synbreedData/index.html">https://cran.r-project.org/web/packages/synbreedData/index.html</a>.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Mice data</h3><div id="sec-13" class="subsection"><h4>Absence of full-sibs:</h4><p id="p-26">The average (SD) of pedigree relationships among individuals in the same management units was 0.491 (0.058) because of the aforementioned full-sib family assignments. The genomic counterpart (<span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-131.gif"/></noscript></span></span>) gave a similar estimate of 0.494 with a slightly increased SD of 0.087 due to Mendelian sampling variation (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Hill and Weir 2011</a>). The average across-management unit pedigree-based genetic connectedness was 1.299 when measured by PEVD and <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-132.gif"/></noscript></span></span> (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Measures of connectedness increased using genomic data (<span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-133.gif"/></noscript></span></span>) by reducing PEVD to 0.456. With <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-134.gif"/></noscript></span></span> while the overall genetic connectedness decreased, genomic information (<span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-135.gif"/></noscript></span></span>) lowered PEVD compared to that of pedigree. Use of the <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-136.gif"/></noscript></span></span> reduced PEVD more than that of the <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-137.gif"/></noscript></span></span> hence increased the measures of connectedness. Using the scaled genomic relationship matrix increased connectedness statistics compared to those of the pedigree-based, but they were less than those with <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-138.gif"/></noscript></span></span> Similarly, use of the <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-139.gif"/></noscript></span></span> matrix compared to the <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-140.gif"/></noscript></span></span> matrix strengthened measured connectedness in CD for both <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-141.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-142.gif"/></noscript></span></span> The <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-143.gif"/></noscript></span></span> matrix also increased measures of connectedness compared to those of the <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-144.gif"/></noscript></span></span> and the <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-145.gif"/></noscript></span></span> matrix resulted in the greatest measures of connectedness among the four relatedness matrices. Both PEVD and CD statistics confirmed that genome-wide markers increased the degree of connectedness estimated between individuals across management units. However, the connectedness measures assessed by r were less when the <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-146.gif"/></noscript></span></span> was compared with the <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-147.gif"/></noscript></span></span> On the other hand, the <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-148.gif"/></noscript></span></span> and the scaled genomic relationship matrix <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-149.gif"/></noscript></span></span> estimated greater connectedness measures than those of the <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-150.gif"/></noscript></span></span></p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486980/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486980/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Average genetic connectedness measures across management units in the mice data</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h4>Presence of full-sibs:</h4><p id="p-28">The increased estimates of disconnectedness were less when at least one full-sib was present in different management units for PEVD. For instance, comparisons between absence or presence of full-sibs across management units were 1.299 <em>vs.</em> 0.354 and 0.456 <em>vs.</em> 0.127 for pedigree-based <em>vs.</em> genome-based (<span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-169.gif"/></noscript></span></span>) PEVD, respectively. The presence of full-sibs in different management units decreased PEVD. However, corresponding statistics for CD were lower with the existence of full-sibs. This is explained by the fact that CD penalizes the estimates of connectedness when genetic variability is small. The CD statistic attempts to decrease the average PEV of the contrast while maintaining the variability of relatedness. <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Laloë (1993)</a> stated that increased estimate of connectedness should not be achieved by simply using genetically similar individuals and that CD is the most relevant connectedness statistic in terms of genetic progress of agricultural species. This was confirmed in the mice data illustrating that the presence of full-sibs decreased the estimates of CD. Regardless of the absence or presence of full-sibs across units, genomic information elucidated additional relationships, thus increasing connectedness estimates relative to pedigree. This trend was also true for the <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-170.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-171.gif"/></noscript></span></span> matrices. With r, when transitioning from <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-172.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-173.gif"/></noscript></span></span> the values of the statistic reduced; however, the <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-174.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-175.gif"/></noscript></span></span> yielded greater values of connectedness than those of pedigree in the existence of full-sibs. In all cases, using one of the <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-176.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-177.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-178.gif"/></noscript></span></span> matrices increased the estimates of connectedness statistics as compared to using the <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-179.gif"/></noscript></span></span> As shown in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, we found a similar overall pattern when <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-180.gif"/></noscript></span></span> was set to 0.2, although connectedness remained less than the alternative higher heritability. Replacing pedigree with genome-wide markers increased the degree of connectedness captured among individuals in disconnected management units.</p></div><div id="sec-15" class="subsection"><h4>Illustrative examples:</h4><p id="p-29">To illustrate how <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-181.gif"/></noscript></span></span> matrix impacted our measures of connectedness, we chose five management units including full-sib and nonfull-sib individuals. In this example, management units “19F,” “29A,” and “36F” share at least one pair of full-sib individuals, whereas management units “12A” and “13C” do not share any full-sib individuals across management units. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows PEVD-derived connectedness across management units when <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-182.gif"/></noscript></span></span> Comparison across-management units with full-sibs in common had smaller PEVD, hence greater connectedness. The molecular information captures more of the genetic connectedness relative to pedigree across-management units. We further investigated how the <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-183.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-184.gif"/></noscript></span></span> increased connectedness measures across management units relative to the <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-185.gif"/></noscript></span></span> using PEVD and r. To do so, we examined the specific components in the PEV matrix derived from several management units including full-sib and nonfull-sib individuals. As shown in detail in Supplemental Material, Text S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300151/-/DC1/FileS1.pdf">File S1</a>, we found that the rates of PEV (diagonals) and PEC (off-diagonals) reductions from <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-186.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-187.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-188.gif"/></noscript></span></span> explain the changes of connectedness measures.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3543/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction error variance of the difference (PEVD) across five management units in the mice dataset. Management units &#x201C;19F,&#x201D; &#x201C;29A,&#x201D; and &#x201C;36F&#x201D; share at least one pair of full-sibs individuals with each other, whereas &#x201C;12A&#x201D; and &#x201C;13C&#x201D; do not share any individuals across management units. The left and right are pedigree-based () and genomic-based () connectedness, respectively. Darker color represents less genetic connectedness." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-249364327" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction error variance of the difference (PEVD) across five management units in the mice dataset. Management units &#x201C;19F,&#x201D; &#x201C;29A,&#x201D; and &#x201C;36F&#x201D; share at least one pair of full-sibs individuals with each other, whereas &#x201C;12A&#x201D; and &#x201C;13C&#x201D; do not share any individuals across management units. The left and right are pedigree-based (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-189&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) and genomic-based (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-190&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) connectedness, respectively. Darker color represents less genetic connectedness.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3543/F1.medium.gif" width="440" height="193"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/10/3543/F1.medium.gif" width="440" height="193"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3543/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3543/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486968" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-30" class="first-child">Prediction error variance of the difference (PEVD) across five management units in the mice dataset. Management units “19F,” “29A,” and “36F” share at least one pair of full-sibs individuals with each other, whereas “12A” and “13C” do not share any individuals across management units. The left and right are pedigree-based (<span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/F1/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/F1/embed/mml-math-189.gif"/></noscript></span></span>) and genomic-based (<span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/F1/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/F1/embed/mml-math-190.gif"/></noscript></span></span>) connectedness, respectively. Darker color represents less genetic connectedness.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-16" class="subsection"><h3>Cattle</h3><div id="sec-17" class="subsection"><h4>Clustering:</h4><p>The partitioning around medoids clustering method yielded eight clusters. <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> contains descriptive statistics for those clusters. The number of individuals per cluster varied from 36 to 127. The average of within-cluster pedigree-based relationships was ∼0.05, except for cluster 6 in which distant relatives were grouped together. Between clusters, all pedigree-based relationships were close to zero. Each cluster was assigned to management units in four simulated scenarios, as summarized in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>.<div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486834/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486834/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Descriptive statistics of the eight clusters in the cattle data</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3543/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Four simulation scenarios considered in the cattle dataset. Scenario 1: completely disconnected; eight clusters assigned to separate MU. Scenario 2: disconnected; clusters 1, 2, 3, 4, and 5 assigned to MU 1 and clusters 6, 7, and 8 assigned to MU 2. Scenario 3: partially connected; clusters 1, 2, and 3 assigned to MU 1, clusters 7 and 8 assigned to MU 2, and the remaining clusters 4, 5, and 6 assigned to both MUs 1 and 2, which act as links among clusters or individuals that partially connect the two MUs. Scenario 4: connected; all clusters (1&#x2013;8) were equally assigned to MUs 1 and 2. MU, management unit." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-249364327" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Four simulation scenarios considered in the cattle dataset. Scenario 1: completely disconnected; eight clusters assigned to separate MU. Scenario 2: disconnected; clusters 1, 2, 3, 4, and 5 assigned to MU 1 and clusters 6, 7, and 8 assigned to MU 2. Scenario 3: partially connected; clusters 1, 2, and 3 assigned to MU 1, clusters 7 and 8 assigned to MU 2, and the remaining clusters 4, 5, and 6 assigned to both MUs 1 and 2, which act as links among clusters or individuals that partially connect the two MUs. Scenario 4: connected; all clusters (1&#x2013;8) were equally assigned to MUs 1 and 2. MU, management unit.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3543/F2.medium.gif" width="440" height="315"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/10/3543/F2.medium.gif" width="440" height="315"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3543/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3543/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486963" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Four simulation scenarios considered in the cattle dataset. Scenario 1: completely disconnected; eight clusters assigned to separate MU. Scenario 2: disconnected; clusters 1, 2, 3, 4, and 5 assigned to MU 1 and clusters 6, 7, and 8 assigned to MU 2. Scenario 3: partially connected; clusters 1, 2, and 3 assigned to MU 1, clusters 7 and 8 assigned to MU 2, and the remaining clusters 4, 5, and 6 assigned to both MUs 1 and 2, which act as links among clusters or individuals that partially connect the two MUs. Scenario 4: connected; all clusters (1–8) were equally assigned to MUs 1 and 2. MU, management unit.</p><div class="sb-div caption-clear"></div></div></div></p><ul class="list-simple " id="list-2"><li id="list-item-5"><p id="p-33">Scenario 1: Each cluster was assigned to its own management unit.</p></li><li id="list-item-6"><p id="p-34">Scenario 2: Clusters 1, 2, 3, 4, and 5 were assigned to management unit 1 and clusters 6, 7, and 8 were assigned to management unit 2.</p></li><li id="list-item-7"><p id="p-35">Scenario 3: Clusters 1, 2, and 3 were assigned to management unit 1; clusters 7 and 8 were assigned to management unit 2; and individuals in clusters 4, 5, and 6 were assigned to both management units 1 and 2 to act as link among clusters or individuals that partially connect the two management units.</p></li><li id="list-item-8"><p id="p-36">Scenario 4: Individuals in clusters 1 to 8 were equally assigned to management units 1 and 2.</p></li></ul><p id="p-37">The number of individuals in management units 1 and 2 were approximately equal in scenarios 2, 3, and 4. We computed PEVD, CD, and r for each of the four scenarios and compared genetic connectedness when using the <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-191.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-192.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-193.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-194.gif"/></noscript></span></span> kernel matrices.</p></div><div id="sec-18" class="subsection"><h4>PEVD:</h4><p id="p-38">Across-management unit PEVDs for each of the four scenarios are presented in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. Connectedness estimates increased across management units when transitioning from scenario 1 to scenario 4 for both heritability levels. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows the relative increase of genetic connectedness as measured with PEVD, as a percentage, across management units in comparison to scenario 1. Genetic connectedness across management units in scenario 1 was compared to across-management unit connectedness obtained from scenarios 2, 3, and 4. We observed increased genetic connectedness as more individuals from the same clusters were shared between management units, resulting in the highest connectedness estimates in scenario 4. Transitioning from scenario 1 to scenario 4 increased connectedness for <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-195.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-196.gif"/></noscript></span></span> for both heritability levels. The proportional increases in genetic connectedness in pedigree-based relationships were larger than those of genomic-based relationships because <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-197.gif"/></noscript></span></span> matrix substantially increased measured connectedness between disconnected management units in scenario 1, reducing the gains in the following scenarios 2, 3, and 4. Also, as heritability increased, larger values of connectedness were observed. In general, <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-198.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-199.gif"/></noscript></span></span> increased the estimates of connectedness compared to those of the <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-200.gif"/></noscript></span></span> regardless of heritability levels. This is in agreement with the mice dataset. However, with <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-201.gif"/></noscript></span></span> values of PEVD were unexpected; although scaled <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-202.gif"/></noscript></span></span> produced estimates of connectedness that were higher than those with <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-203.gif"/></noscript></span></span> when <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-204.gif"/></noscript></span></span> was set to 0.8, the same pattern was not observed for <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-205.gif"/></noscript></span></span></p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486822/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486822/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Average genetic connectedness statistics across management units in the cattle data</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3543/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of relative increase in prediction error variance of the difference (PEVD) across management units in comparison to base scenario 1 (S1). Two heritability values 0.8 and 0.2 were simulated. S1 (completely disconnected), scenario 2 (S2, disconnected), scenario 3 (S3, partially connected), and scenario 4 (S4, connected) represent four management unit scenarios. Left:  matrix. Right:  matrix." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-249364327" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Percentage of relative increase in prediction error variance of the difference (PEVD) across management units in comparison to base scenario 1 (S1). Two heritability values 0.8 and 0.2 were simulated. S1 (completely disconnected), scenario 2 (S2, disconnected), scenario 3 (S3, partially connected), and scenario 4 (S4, connected) represent four management unit scenarios. Left: &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-263&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; matrix. Right: &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-264&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; matrix.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3543/F3.medium.gif" width="440" height="326"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/10/3543/F3.medium.gif" width="440" height="326"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3543/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3543/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486877" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-40" class="first-child">Percentage of relative increase in prediction error variance of the difference (PEVD) across management units in comparison to base scenario 1 (S1). Two heritability values 0.8 and 0.2 were simulated. S1 (completely disconnected), scenario 2 (S2, disconnected), scenario 3 (S3, partially connected), and scenario 4 (S4, connected) represent four management unit scenarios. Left: <span class="inline-formula" id="inline-formula-250"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/F3/embed/mml-math-263.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/F3/embed/mml-math-263.gif"/></noscript></span></span> matrix. Right: <span class="inline-formula" id="inline-formula-251"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/F3/embed/mml-math-264.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/F3/embed/mml-math-264.gif"/></noscript></span></span> matrix.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h4>CD:</h4><p id="p-41">Across CDs for each of the four scenarios are presented in <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>. Similar to PEVD, the extent of connectedness across management units increased when moving from scenario 1 to scenario 2 and 3 regardless of the heritability levels. Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300151/-/DC1/FileS1.pdf">File S1</a> shows the percentage increase in CD across management units when scenario 1 was treated as a base comparison. As with PEVD, CD statistics revealed an increase in the degree of connectedness as more individuals from the same clusters were assigned to different management units. However, the increase of CD was not observed when transitioning from scenario 1 to scenario 4. Again, this is because CD accounts for the reduction of connectedness due to reduced relatedness variability between individuals under comparison in scenario 4. This pattern was observed for both pedigree and genomic-based connectedness. Overall, <span class="inline-formula" id="inline-formula-252"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-265.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-265.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-253"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-266.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-266.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-254"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-267.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-267.gif"/></noscript></span></span> all produced CD greater than those with <span class="inline-formula" id="inline-formula-255"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-268.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-268.gif"/></noscript></span></span> regardless of heritability level, yielding consistent measures of connectedness.</p></div><div id="sec-20" class="subsection"><h4>Prediction error correlation:</h4><p id="p-42">Prediction error correlations across management units for each of four scenarios are presented in <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>. The results align with those of the mice dataset in that <span class="inline-formula" id="inline-formula-256"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-269.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-269.gif"/></noscript></span></span>-based r statistics behave erratically in all scenarios, making them difficult to interpret. However, the anticipated increases in r were observed with the transition from the <span class="inline-formula" id="inline-formula-257"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-270.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-270.gif"/></noscript></span></span> to the <span class="inline-formula" id="inline-formula-258"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-271.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-271.gif"/></noscript></span></span> or the scaled <span class="inline-formula" id="inline-formula-259"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-272.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-272.gif"/></noscript></span></span> matrix. Here, <span class="inline-formula" id="inline-formula-260"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-273.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-273.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-261"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-274.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-274.gif"/></noscript></span></span>-based measures consistently yielded greater connectedness values than those of pedigree counterparts. Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300151/-/DC1/FileS1.pdf">File S1</a> shows the percentage increases in r across management units when scenario 1 was treated as a base comparison. Here, the <span class="inline-formula" id="inline-formula-262"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-275.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-275.gif"/></noscript></span></span> instead of the <span class="inline-formula" id="inline-formula-263"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-276.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-276.gif"/></noscript></span></span> matrix was used. The results align with those of PEVD and CD, where the extent of pedigree-based and genomic-based r statistics increase the most when more individuals from the same clusters were assigned to different management units. The magnitude of the increase was larger when heritability was greater. However, the increase of connectedness moving from scenario 1 to 2 was not observed in pedigree-based measures. While both scenarios are not connected designs because pedigree-based relationships across the eight clusters were close to zero, it is interesting to note that with pedigree-based r scenario 2 was more disconnected than scenario 1. From <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Kennedy and Trus (1993)</a>, this is because stronger within unit connectedness can reduce between unit connectedness.</p></div><div id="sec-21" class="subsection"><h4>Ungenotyped and genotyped individuals:</h4><p id="p-43">We considered a scenario where only individuals in younger generations were genotyped in the cattle dataset. For this purpose, we used the <span class="inline-formula" id="inline-formula-264"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-277.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-277.gif"/></noscript></span></span> matrix, which blends ungenotyped and genotyped individuals. As shown in Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300151/-/DC1/FileS1.pdf">File S1</a>, results using the H matrix lie somewhere between the results obtained when using the <span class="inline-formula" id="inline-formula-265"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-278.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-278.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-266"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-279.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-279.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-267"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-280.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-280.gif"/></noscript></span></span> matrices. This is expected because the <span class="inline-formula" id="inline-formula-268"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-281.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-281.gif"/></noscript></span></span> matrix was created from a combination of <span class="inline-formula" id="inline-formula-269"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-282.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-282.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-270"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-283.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-283.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-271"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-284.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-284.gif"/></noscript></span></span> Although an increase in measures of connectedness was observed compared to using the pedigree alone, this increase was smaller than when all individuals were genotyped. This finding suggests the possibility of strengthening the degree of connectedness even when only a subset of individuals was genotyped. An exception was observed when <span class="inline-formula" id="inline-formula-272"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-285.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-285.gif"/></noscript></span></span> was constructed from <span class="inline-formula" id="inline-formula-273"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-286.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-286.gif"/></noscript></span></span> for PEVD; in this case, the measures of connectedness were less than that from <span class="inline-formula" id="inline-formula-274"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-287.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-287.gif"/></noscript></span></span></p></div><div id="sec-22" class="subsection"><h4>PCA of connectedness:</h4><p id="p-44">PC plots for CD derived from <span class="inline-formula" id="inline-formula-275"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-288.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-288.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-276"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-289.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-289.gif"/></noscript></span></span> matrices for scenarios 1 and 4 are presented in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, respectively. These correspond to the two extreme scenarios considered in the cattle dataset. In scenario 1 with <span class="inline-formula" id="inline-formula-277"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-290.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-290.gif"/></noscript></span></span> eight clusters assigned to distinctive management units were separated from each other as expected using pedigree-based relationships (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Genomic information brought these eight clusters closer to each other, thus shortening the distance between individuals from different management units. While eight clusters were less distinguishable from one another due to lower heritability, the same pattern was observed when <span class="inline-formula" id="inline-formula-278"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-291.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-291.gif"/></noscript></span></span> was 0.2. These findings align with the fact that use of genomic information increases measures of connectedness compared to pedigree. In both cases, cluster 6, which consisted of unrelated individuals, was clustered far away from the other clusters in the pedigree-based analysis. PCA yielded two clear clusters in scenario 4 when <span class="inline-formula" id="inline-formula-279"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-292.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-292.gif"/></noscript></span></span> which correspond to the two management units considered (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Replacing <span class="inline-formula" id="inline-formula-280"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-293.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-293.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-281"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-294.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-294.gif"/></noscript></span></span> resulted in a tighter concentration of a single cluster. A similar tendency was observed when <span class="inline-formula" id="inline-formula-282"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-295.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-295.gif"/></noscript></span></span> supporting the findings that the extent of measures of connectedness between individuals from different management units is enhanced with genomic information. The remaining PC plots for PEVD (<span class="inline-formula" id="inline-formula-283"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-296.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-296.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-284"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-297.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-297.gif"/></noscript></span></span>) and r (<span class="inline-formula" id="inline-formula-285"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-298.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-298.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-286"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-299.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-299.gif"/></noscript></span></span>) are in Figure S3, Figure S4, Figure S5, and Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300151/-/DC1/FileS1.pdf">File S1</a>, which present a patterns similar to those in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3543/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principal component (PC) plots for scenario 1 (SC1) with coefficient of determination (CD) statistics. The first and second rows are according to heritability of 0.8 and of 0.2. The first and second columns are derived from pedigree-based () and genome-based () CD, respectively. The PC plots were grouped by clusters and colored in different colors. Individuals within the same cluster were grouped by the circles. var., variance." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-249364327" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Principal component (PC) plots for scenario 1 (SC1) with coefficient of determination (CD) statistics. The first and second rows are according to heritability of 0.8 and of 0.2. The first and second columns are derived from pedigree-based (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-300&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) and genome-based (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-301&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) CD, respectively. The PC plots were grouped by clusters and colored in different colors. Individuals within the same cluster were grouped by the circles. var., variance.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3543/F4.medium.gif" width="440" height="328"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/10/3543/F4.medium.gif" width="440" height="328"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3543/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3543/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486867" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-45" class="first-child">Principal component (PC) plots for scenario 1 (SC1) with coefficient of determination (CD) statistics. The first and second rows are according to heritability of 0.8 and of 0.2. The first and second columns are derived from pedigree-based (<span class="inline-formula" id="inline-formula-287"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/F4/embed/mml-math-300.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/F4/embed/mml-math-300.gif"/></noscript></span></span>) and genome-based (<span class="inline-formula" id="inline-formula-288"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/F4/embed/mml-math-301.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/F4/embed/mml-math-301.gif"/></noscript></span></span>) CD, respectively. The PC plots were grouped by clusters and colored in different colors. Individuals within the same cluster were grouped by the circles. var., variance.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3543/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principal component (PC) plots for scenario 4 (SC4) with coefficient of determination (CD) statistics. The first and second rows are according to heritability of 0.8 and of 0.2. The first and second columns are derived from pedigree-based () and genome-based () CD, respectively. The PC plots were grouped by clusters and colored in different colors. Individuals within the same cluster were grouped by the circles." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-249364327" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Principal component (PC) plots for scenario 4 (SC4) with coefficient of determination (CD) statistics. The first and second rows are according to heritability of 0.8 and of 0.2. The first and second columns are derived from pedigree-based (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-302&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) and genome-based (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-303&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) CD, respectively. The PC plots were grouped by clusters and colored in different colors. Individuals within the same cluster were grouped by the circles.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3543/F5.medium.gif" width="440" height="337"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/10/3543/F5.medium.gif" width="440" height="337"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3543/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3543/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486954" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-46" class="first-child">Principal component (PC) plots for scenario 4 (SC4) with coefficient of determination (CD) statistics. The first and second rows are according to heritability of 0.8 and of 0.2. The first and second columns are derived from pedigree-based (<span class="inline-formula" id="inline-formula-289"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/F5/embed/mml-math-302.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/F5/embed/mml-math-302.gif"/></noscript></span></span>) and genome-based (<span class="inline-formula" id="inline-formula-290"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/F5/embed/mml-math-303.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/F5/embed/mml-math-303.gif"/></noscript></span></span>) CD, respectively. The PC plots were grouped by clusters and colored in different colors. Individuals within the same cluster were grouped by the circles.</p><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="section discussion" id="sec-23"><h2 class="">Discussion</h2><p id="p-47">With sufficient connectedness across management units, BLUP of genetic values can be fairly compared. Without such connectedness, making selection decisions based on breeding values of individuals from different management units might be associated with an increased risk of uncertainty in genetic evaluation due to imperfect separation of the genetic signal from noise. In addition to PEVD, CD, and r, other connectedness measures have been applied to pedigree data (<em>e.g.</em>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Foulley <em>et al.</em> 1992</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Fouilloux <em>et al.</em> 2008</a>), which have their own characteristics. Advancement of molecular biotechnology now enables us to assess connectedness at the genomic level. Although genomic data are clearly important in genetic evaluations due to increased accuracy of estimates of genetic merit for nonparent individuals, little consideration has been given to the effect of genomic information on connectedness measures. In this study, we employed three measures of connectedness to examine the extent to which genomic information increases the estimates of connectedness.</p><div id="sec-24" class="subsection"><h3>Relatedness in quantitative genetics</h3><p id="p-48">The majority of connectedness among management units was driven by the degree of genetic links or relatedness. The theory behind relatedness is largely entrenched in quantitative genetics dating back to the work of <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Fisher (1918)</a> and <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Wright (1921)</a>. Quantitative genetics offers a useful framework to study traits and diseases that are controlled by a considerable number of small effect genes. For traits with polygenic genetic architectures, inheritance does not exhibit a clear genotype–phenotype pattern. However, genetic resemblance between relatives (<em>e.g.</em>, the genetic correlations between parent and offspring or between pairs of different types of siblings) can be exploited to estimate quantitative genetic parameters. For this reason, genetic resemblance between relatives has been at the heart of quantitative genetics. Consequently, the vast majority of the theoretical developments and applications of the last century were built around family data. The availability of dense panels of common SNPs has made it possible to trace Mendelian sampling and hence capture more detailed relatedness compared to pedigree information. It enables the quantification of genomic kinships among related individuals that are not otherwise apparent because of incomplete pedigrees or the general assumption that animals in a baseline or founder population are unrelated. Thus, it has opened up new opportunities for quantitative genetic analysis using data from distant relatives. The rationale is that individuals are genomically related to some extent and that molecular similarity introduces covariance even if individuals are not related in the sense of known pedigree. These factors possibly contribute to the reduction of PEV or increase of PEC, and hence lead to increased capturing of genetic connectedness in PEVD, CD, and r such that genetic merit estimates can be better compared across management units.</p></div><div id="sec-25" class="subsection"><h3>The impact of genomic information on connectedness</h3><p id="p-49">We found from the mice data that genomic information increased favorable changes in measures of connectedness among individuals from different management units and reduced the risk of potential uncertainty in EBV-based comparisons when selecting individuals across management units. In addition, the rate of improvement in measures of connectedness in PEVD and r was greater when there was at least one full-sib in different management units. This is in concordance with <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Legarra <em>et al.</em> (2008)</a>, who used the same dataset and reported that the use of genome-wide selection increased predictive performance up to 0.22 across families and up to 0.03 within families compared to pedigree-based regression counterparts. On the other hand, CD accounted for the reduction of variability of relatedness between individuals under comparison resulting in decreased estimates of connectedness. Analysis of cattle data supported the results from mice and revealed that the benefit of using genomic information is greater for a disconnected design rather than a connected design. PCA was performed to visualize improvement in connectedness when moving from pedigree to genomic-based relationships. The PC plots supported the evidence that genomic information can improve detection of connectedness between individuals from different management units. This is particularly so when more individuals from the same clusters are assigned to different management units.</p></div><div id="sec-26" class="subsection"><h3>Choice of kernel matrices</h3><p id="p-50">Unlike PEVD and CD, comparisons between the <span class="inline-formula" id="inline-formula-291"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-304.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-304.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-292"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-305.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-305.gif"/></noscript></span></span> kernel matrices evaluated by the r statistic behaved irregularly. By examining the specific components of the PEV matrix for <span class="inline-formula" id="inline-formula-293"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-306.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-306.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-294"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-307.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-307.gif"/></noscript></span></span> in the mice dataset, we found that genomic information reduces off-diagonal elements more than diagonals. This illustrates a fundamental difference between r and either PEVD or CD because this statistic is based on the ratio, rather than the magnitude, of individual elements. It may be argued that the inconsistent connectedness results from r occur because the <span class="inline-formula" id="inline-formula-295"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-308.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-308.gif"/></noscript></span></span> matrix is not on the same scale as the <span class="inline-formula" id="inline-formula-296"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-309.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-309.gif"/></noscript></span></span> matrix, suggesting that r statistics are not invariant regarding how genomic relatedness is defined. Given pedigree information, the numerator relationship matrix is defined as IBD. On the other hand, given a marker matrix, there are a number of ways to construct a genomic relationship matrix, as discussed by <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Toro <em>et al.</em> (2011)</a>. The <span class="inline-formula" id="inline-formula-297"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-310.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-310.gif"/></noscript></span></span> matrix we used captures the proportion of the genome that is IBS by accounting for the covariance structure among individuals by molecular markers (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Toro <em>et al.</em> 2002</a>). This kernel matrix is an estimator of IBD relationships (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Powell <em>et al.</em> 2010</a>). Caution should be exercised when interpreting connectedness measures derived using genomic data, as the underlying assumption is that relationships are built based on alleles being IBS and not necessarily being IBD. Therefore, we attempted to make <span class="inline-formula" id="inline-formula-298"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-311.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-311.gif"/></noscript></span></span> more compatible to <span class="inline-formula" id="inline-formula-299"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-312.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-312.gif"/></noscript></span></span> by using <span class="inline-formula" id="inline-formula-300"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-313.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-313.gif"/></noscript></span></span> derived from allele frequencies equal to 0.5 and by using the min–max scaler transformation to produce the scaled genomic relationship matrix <span class="inline-formula" id="inline-formula-301"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-314.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-314.gif"/></noscript></span></span> For instance, compared to using <span class="inline-formula" id="inline-formula-302"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-315.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-315.gif"/></noscript></span></span> entries of PEV matrix from using <span class="inline-formula" id="inline-formula-303"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-316.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-316.gif"/></noscript></span></span> were more similar to those <span class="inline-formula" id="inline-formula-304"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-317.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-317.gif"/></noscript></span></span> especially when there was connectedness, and in turn r statistics yielded greater connectedness values. Although connecting marker-based genomic relatedness to classical theory is still an open question in quantitative genetics, care needs to be taken when comparing genetic connectedness with genomic connectedness, especially when the ratio-based statistic is used. Moreover, many additional factors may influence the elements of IBS matrix such as the choice of MAF, the density of SNP, imperfect linkage disequilibrium (LD) between markers and QTL, and errors associated with estimating genomic relationships from a finite set of markers (<em>e.g.</em>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Goddard 2009</a>).</p></div><div id="sec-27" class="subsection"><h3>Choice of connectedness statistics</h3><p id="p-51">There was an issue with PEVD coupled with the <span class="inline-formula" id="inline-formula-305"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-318.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-318.gif"/></noscript></span></span> matrix in the cattle dataset when <span class="inline-formula" id="inline-formula-306"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-319.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-319.gif"/></noscript></span></span> = 0.2, as the estimates of connectedness were less than those using <span class="inline-formula" id="inline-formula-307"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-320.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-320.gif"/></noscript></span></span> (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). Note that this was not the case when <span class="inline-formula" id="inline-formula-308"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-321.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-321.gif"/></noscript></span></span> = 0.8. The <span class="inline-formula" id="inline-formula-309"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-322.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-322.gif"/></noscript></span></span> matrix blended from the <span class="inline-formula" id="inline-formula-310"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-323.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-323.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-311"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-324.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-324.gif"/></noscript></span></span> kernel matrices yielded the estimates of connectedness that lie somewhere between the results obtained when using <span class="inline-formula" id="inline-formula-312"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-325.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-325.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-313"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-326.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-326.gif"/></noscript></span></span> alone. However, this pattern was not observed when <span class="inline-formula" id="inline-formula-314"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-327.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-327.gif"/></noscript></span></span> was used in conjunction with <span class="inline-formula" id="inline-formula-315"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-328.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-328.gif"/></noscript></span></span> to compute PEVD (Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300151/-/DC1/FileS1.pdf">File S1</a>). Apparently, scaling has a negative influence on blending for PEVD, which warrants further research. One potential reason with <span class="inline-formula" id="inline-formula-316"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-329.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-329.gif"/></noscript></span></span> for the discrepancy is that the proportional increase of PEC relative to PEV is larger when transitioning from the <span class="inline-formula" id="inline-formula-317"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-330.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-330.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-318"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-331.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-331.gif"/></noscript></span></span> This issue of proportional change is similar to that observed earlier with the r statistic coupled with <span class="inline-formula" id="inline-formula-319"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-332.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-332.gif"/></noscript></span></span> These results illustrate that connectedness statistics are not invariant with respect to how the genomic relationship matrix is created and that each of them captures different aspects of genomic connectedness. The CD was the only statistic that yielded consistent estimates of increased connectedness throughout this study. Its consistency was observed regardless of choice of kernel matrices, heritability levels, datasets used, and simulated scenarios for management units.</p></div><div id="sec-28" class="subsection"><h3>Inferring variance components from data</h3><p id="p-52">One concern with the current study is fixing heritability levels for all scenarios based on the assumption that both pedigree and genomic relationship matrices explain the equal amount of heritability. In practice, this assumption might not hold true when SNPs do not capture the entire QTL signals. To address this concern, additional analysis was carried out such that variance components were estimated from the data rather than assuming these were known. We analyzed two publicly available phenotypes in the synbreedData R package (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Wimmer <em>et al.</em> 2015</a>) for the cattle data used in this study. The heritabilities of these traits are 0.66 and 0.41. Connectedness analysis in <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a> was repeated based on variance components estimated by a restricted maximum likelihood. The measures of CD derived from the <strong>A</strong> and <strong>G</strong> matrices are shown in Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300151/-/DC1/FileS1.pdf">File S1</a>. We found that genomic relatedness increased connectedness measures more so than those of pedigree when variance components were directly estimated from the data. This result was consistent with what we found in the cattle data analysis reported in <a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>.</p></div><div id="sec-29" class="subsection"><h3>Future direction</h3><p id="p-53">One important direction for future study is to investigate whether increased connectedness observed by genomic relatedness also leads to increased predictive accuracy of genetic values across management units assessed by cross-validation. In this case, across-management units can be considered as training and testing sets. In addition, while the current norm of genomic prediction is to use an IBS relationship matrix that aims to capture relationships at unknown QTL through LD between markers and QTL, we argue that improving the quality of breeding value comparisons and improving the accuracy of genomic prediction can be viewed as relevant but two different items. In this regard, a genome-wide IBD relationship matrix (<em>e.g.</em>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Fernando and Grossman 1989</a>), where marker inheritance is traced through a known pedigree, may be worthwhile to revisit for the purpose of ascertaining connectedness in a future study.</p><p id="p-54">Also, for the r statistic, we summarized connectedness by averaging the r statistic of pairs of individuals across units rather than by averaging the relevant components of PEC and PEV followed by taking their ratio; our justification for that choice is provided in the <em>Appendix</em>. When the latter summary statistic was used for r, the differences were negligible in the mice data and the pattern was the same for scenario 1 of the cattle data.</p><p id="p-55">In conclusion, this study confirms that use of genomic relatedness improved genetic connectedness across management units compared to the use of pedigree relationships. To our knowledge, this marks the first thorough investigation of genomic connectedness. We contend that our work is a critical first step toward better understanding genetic connectedness that may have a positive impact on genomic evaluation of agricultural species.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-57">The authors thank Dale Van Vleck and Larry Kuehn for their valuable comments on the manuscript. This work was supported in part by the University of Nebraska startup funds to G.M. </p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-30"><h3>Prediction error correlation statistic across units</h3><p id="p-56">When a population is divided into two management units, and relatedness between those two units is based on the <span class="inline-formula" id="inline-formula-320"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-333.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-333.gif"/></noscript></span></span> matrix, the flock connectedness or unit connectedness correlation r of <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Kuehn <em>et al.</em> (2008)</a> always yields an estimate of −1. Here, we wish to illustrate that result. Flock connectedness is derived by averaging the relevant components of PEC and PEV followed by taking their ratio. Suppose that there are two units and the numbers of individuals in units <span class="inline-formula" id="inline-formula-321"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-334.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-334.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-322"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-335.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-335.gif"/></noscript></span></span> are <span class="inline-formula" id="inline-formula-323"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-336.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-336.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-324"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-337.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-337.gif"/></noscript></span></span> respectively. The total number of individuals is <span class="inline-formula" id="inline-formula-325"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-338.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-338.gif"/></noscript></span></span> The assumptions of the <span class="inline-formula" id="inline-formula-326"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-339.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-339.gif"/></noscript></span></span> matrix is that the genotyped individuals represent the base population where the expected value of self-relatedness is one, assuming no inbreeding, and that the mean relatedness of any one individual to the rest of the individuals is zero. Consequently, the expectation of diagonal elements of the <span class="inline-formula" id="inline-formula-327"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-340.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-340.gif"/></noscript></span></span> matrix is equal to the number of individuals assuming no inbreeding and the expectation of off-diagonal elements is <span class="inline-formula" id="inline-formula-328"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-341.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-341.gif"/></noscript></span></span> (<em>e.g.</em>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Yang <em>et al.</em> 2013</a>). Then, the expectation of numerator in the r statistic is proportional to<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-342.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-342.gif"/></noscript></span></span>and the expectation of denominator is the square root of the product between<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-343.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-343.gif"/></noscript></span></span>and<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-344.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-344.gif"/></noscript></span></span>so that<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-345.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-345.gif"/></noscript></span></span>Note that the first three terms in the numerator are equal to zero<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-346.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-346.gif"/></noscript></span></span>because <span class="inline-formula" id="inline-formula-329"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-347.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-347.gif"/></noscript></span></span> Therefore, the r statistic between units <span class="inline-formula" id="inline-formula-330"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-348.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-348.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-331"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-349.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-349.gif"/></noscript></span></span> is given by<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-350.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-350.gif"/></noscript></span></span>When <span class="inline-formula" id="inline-formula-332"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-351.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-351.gif"/></noscript></span></span> this result holds regardless of relatedness level, connectedness level, and how individuals are partitioned into the two management units <span class="inline-formula" id="inline-formula-333"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-352.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-352.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-334"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-353.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-353.gif"/></noscript></span></span> The partitioning of animals into two distinct units is particularly relevant in the context of genomic prediction, where animals may be divided into training and testing sets. In this scenario, computing the connectedness between the two sets along the lines of <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Rincent <em>et al.</em> (2012)</a> and <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Isidro <em>et al.</em> (2015)</a> is potentially informative relative to expectations of the performance of resulting genomic predictors. The <span class="inline-formula" id="inline-formula-335"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-354.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-354.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-336"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-355.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-355.gif"/></noscript></span></span> matrix changes the expectation of off-diagonal elements to positive values and shifts the statistic by a constant, as explained in the <em>Materials and Methods</em> section, yielding connectedness between units <span class="inline-formula" id="inline-formula-337"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-356.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-356.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-338"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-357.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10/3543/embed/mml-math-357.gif"/></noscript></span></span> of close to one. Because scenarios 2–4 in the cattle dataset simulated two management units, the average of the r statistic of pairs of individuals in different management units was used to summarize connectedness in this study. Note that this is shown as lamb connectedness or individual connectedness in <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Kuehn <em>et al.</em> (2008)</a>. The two types of connectedness differ mainly by whether we take the average followed by the ratio (unit connectedness) or take the ratio first followed by the average (individual connectedness).</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-58">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300151/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300151/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-59"><em>Communicating editor: G. de los Campos</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-05-06"><span class="received-label">Received </span>May 6, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-08-30"><span class="accepted-label">Accepted </span>August 30, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Yu <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.1" data-doi="10.1186/1297-9686-42-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic prediction when some animals are not genotyped.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DChristensen%26rft.auinit1%253DO.%2BF.%26rft.volume%253D42%26rft.spage%253D2%26rft.epage%253D2%26rft.atitle%253DGenomic%2Bprediction%2Bwhen%2Bsome%2Banimals%2Bare%2Bnot%2Bgenotyped.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-2%26rft_id%253Dinfo%253Apmid%252F20105297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20105297&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eikje</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Strong connectedness within Norwegian Cheviot and Fur sheep ram circles allows reliable estimation of breeding values.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">3322</span>–<span class="cit-lpage">3330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft.volume%253D93%26rft.spage%253D3322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.3" data-doi="10.1051/gse:19890407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grossman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Marker assisted selection using best linear unbiased prediction.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">467</span>–<span class="cit-lpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D21%26rft.spage%253D467%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19890407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19890407&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1989CK31500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.4" data-doi="10.1017/S0080456800012163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1918</span> <span class="cit-article-title">The correlation between relatives on the supposition of Mendelian inheritance.</span> <abbr class="cit-jnl-abbrev">Trans. R. Soc. Edinb.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BR.%2BSoc.%2BEdinb.%26rft.volume%253D52%26rft.spage%253D399%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0080456800012163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0080456800012163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fouilloux</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clément</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laloë</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Measuring connectedness among herds in mixed linear models: from theory to practice in large-sized genetic evaluations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DFouilloux%26rft.auinit1%253DM.%2BN.%26rft.volume%253D40%26rft.issue%253D2%26rft.spage%253D145%26rft.epage%253D159%26rft.atitle%253DMeasuring%2Bconnectedness%2Bamong%2Bherds%2Bin%2Bmixed%2Blinear%2Bmodels%253A%2Bfrom%2Btheory%2Bto%2Bpractice%2Bin%2Blarge-sized%2Bgenetic%2Bevaluations.%26rft_id%253Dinfo%253Apmid%252F18298932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18298932&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.6" data-doi="10.1051/gse:19920403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foulley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanocq</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boichard</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">A criterion for measuring the degree of connectedness in linear models of genetic evaluation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D24%26rft.spage%253D315%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19920403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19920403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.7" data-doi="10.1007/s10709-008-9308-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection: prediction of accuracy and maximisation of long term response.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D136%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9308-0%26rft_id%253Dinfo%253Apmid%252F18704696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-008-9308-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18704696&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265818900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-7.10.3543.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-source">Applications of Linear Models in Animal Breeding</span>, <span class="cit-edition">Ed. 3</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L. R.</span></span>. <span class="cit-publ-name">University of Guelph</span>, <span class="cit-publ-loc">Guelph, ON</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.9" data-doi="10.1017/S0016672310000480"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Variation in actual relationship as a consequence of Mendelian sampling and linkage.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D64%26rft.atitle%253DVariation%2Bin%2Bactual%2Brelationship%2Bas%2Ba%2Bconsequence%2Bof%2BMendelian%2Bsampling%2Band%2Blinkage.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000480%26rft_id%253Dinfo%253Apmid%252F21226974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000480&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21226974&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.10" data-doi="10.1007/s00122-014-2418-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Isidro</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Training set optimization under population structure in genomic selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D145%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2418-4%26rft_id%253Dinfo%253Apmid%252F25367380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2418-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25367380&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-7.10.3543.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaufman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rousseeuw</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-source">Finding Groups in Data: An Introduction to Cluster Analysis</span>. <span class="cit-publ-name">John Wiley and Sons</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kennedy</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trus</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Considerations on genetic connectedness between management units under an animal model.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">2341</span>–<span class="cit-lpage">2352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DKennedy%26rft.auinit1%253DB.%2BW.%26rft.volume%253D71%26rft.issue%253D9%26rft.spage%253D2341%26rft.epage%253D2352%26rft.atitle%253DConsiderations%2Bon%2Bgenetic%2Bconnectedness%2Bbetween%2Bmanagement%2Bunits%2Bunder%2Ban%2Banimal%2Bmodel%26rft_id%253Dinfo%253Apmid%252F8407646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8407646&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LV19300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuehn</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Notter</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Managing the risk of comparing estimated breeding values across flocks or herds through connectedness: a review and application.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DKuehn%26rft.auinit1%253DL.%2BA.%26rft.volume%253D39%26rft.issue%253D3%26rft.spage%253D225%26rft.epage%253D247%26rft.atitle%253DManaging%2Bthe%2Brisk%2Bof%2Bcomparing%2Bestimated%2Bbreeding%2Bvalues%2Bacross%2Bflocks%2Bor%2Bherds%2Bthrough%2Bconnectedness%253A%2Ba%2Breview%2Band%2Bapplication.%26rft_id%253Dinfo%253Apmid%252F17433239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17433239&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.14" data-doi="10.2527/jas.2007-0256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuehn</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Notter</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nieuwhof</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Changes in connectedness over time in alternative sheep sire referencing schemes.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">536</span>–<span class="cit-lpage">544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2007-0256%26rft_id%253Dinfo%253Apmid%252F18073292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2007-0256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18073292&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.15" data-doi="10.2527/jas.2008-1092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuehn</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Notter</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Connectedness in Targhee and Suffolk flocks participating in the United States national sheep improvement program.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">507</span>–<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2008-1092%26rft_id%253Dinfo%253Apmid%252F18849390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2008-1092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18849390&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.16" data-doi="10.1051/gse:19930604"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laloë</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Precision and information in linear models of genetic evaluation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D25%26rft.spage%253D557%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19930604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19930604&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laloë</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phocas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ménissier</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Considerations on measures of precision and connectedness in mixed linear models of genetic evaluation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D28%26rft.spage%253D359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.18" data-doi="10.1534/genetics.108.088575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert-Granié</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfredi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elsen</span>  <span class="cit-name-given-names">J.-M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Performance of genomic selection in mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">611</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.088575%26rft_id%253Dinfo%253Apmid%252F18757934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNjExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zNTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.19" data-doi="10.3168/jds.2009-2061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A relationship matrix including full pedigree and genomic information.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">4656</span>–<span class="cit-lpage">4663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DLegarra%26rft.auinit1%253DA.%26rft.volume%253D92%26rft.issue%253D9%26rft.spage%253D4656%26rft.epage%253D4663%26rft.atitle%253DA%2Brelationship%2Bmatrix%2Bincluding%2Bfull%2Bpedigree%2Band%2Bgenomic%2Binformation%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2061%26rft_id%253Dinfo%253Apmid%252F19700729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19700729&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269156600059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-7.10.3543.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crump</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simm</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Assessing connectedness in across-flock genetic evaluations</span>, pp. <span class="cit-fpage">121</span>–<span class="cit-lpage">122</span> in <span class="cit-source">Proceedings of the British Society of Animal Science</span>. <span class="cit-publ-loc">The British Society of Animal Science, Scarborough, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.21" data-doi="10.1534/genetics.110.116426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maenhout</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Baets</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haesaert</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Graph-based data selection for the construction of genomic prediction models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1463</span>–<span class="cit-lpage">1475</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116426%26rft_id%253Dinfo%253Apmid%252F20479144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS80LzE0NjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEwLzM1NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Momen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mehrgardi</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheikhy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esmailizadeh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fozi</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A predictive assessment of genetic correlations between traits in chickens using markers.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.23" data-doi="10.1038/nrg2865"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Reconciling the analysis of IBD and IBS in complex trait studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">800</span>–<span class="cit-lpage">805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPowell%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D800%26rft.epage%253D805%26rft.atitle%253DReconciling%2Bthe%2Banalysis%2Bof%2BIBD%2Band%2BIBS%2Bin%2Bcomplex%2Btrait%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2865%26rft_id%253Dinfo%253Apmid%252F20877324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2865&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20877324&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pszczola</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strabel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Arendonk</span>  <span class="cit-name-given-names">J. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The impact of genotyping different groups of animals on accuracy when moving from traditional to genomic selection.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">5412</span>–<span class="cit-lpage">5421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DPszczola%26rft.auinit1%253DM.%26rft.volume%253D95%26rft.issue%253D9%26rft.spage%253D5412%26rft.epage%253D5421%26rft.atitle%253DThe%2Bimpact%2Bof%2Bgenotyping%2Bdifferent%2Bgroups%2Bof%2Banimals%2Bon%2Baccuracy%2Bwhen%2Bmoving%2Bfrom%2Btraditional%2Bto%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Apmid%252F22916948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22916948&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.25" data-doi="10.1007/s10852-005-9022-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de la Iglesia</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rayward-Smith</span>  <span class="cit-name-given-names">V. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Clustering rules: a comparison of partitioning and hierarchical clustering algorithms.</span> <abbr class="cit-jnl-abbrev">J. Math. Model. Algorithms</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">475</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMath.%2BModel.%2BAlgorithms%26rft.volume%253D5%26rft.spage%253D475%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10852-005-9022-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10852-005-9022-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.26" data-doi="10.1534/genetics.112.141473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rincent</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laloë</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Maximizing the reliability of genomic selection by optimizing the calibration set of reference individuals: comparison of methods in two diverse groups of maize inbreds (Zea mays L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">715</span>–<span class="cit-lpage">728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141473%26rft_id%253Dinfo%253Apmid%252F22865733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNzE1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zNTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.27" data-doi="10.1007/s00335-005-0112-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauguier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunez</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A protocol for high-throughput phenotyping, suitable for quantitative trait analysis in mice.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DSolberg%26rft.auinit1%253DL.%2BC.%26rft.volume%253D17%26rft.issue%253D2%26rft.spage%253D129%26rft.epage%253D146%26rft.atitle%253DA%2Bprotocol%2Bfor%2Bhigh-throughput%2Bphenotyping%252C%2Bsuitable%2Bfor%2Bquantitative%2Btrait%2Banalysis%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-005-0112-1%26rft_id%253Dinfo%253Apmid%252F16465593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-005-0112-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16465593&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235219200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.28" data-doi="10.1023/A:1019921131171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barragán</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Óvilo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodrigañez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Estimation of coancestry in Iberian pigs using molecular markers.</span> <abbr class="cit-jnl-abbrev">Conserv. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BGenet.%26rft.volume%253D3%26rft.spage%253D309%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1019921131171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1019921131171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.29" data-doi="10.1186/1297-9686-43-27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Cortés</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A note on the rationale for estimating genealogical coancestry from molecular markers.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D43%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-27%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-27&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.30" data-doi="10.1038/ng1840"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauguier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burnett</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klenerman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide genetic association of complex traits in heterogeneous stock mice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">879</span>–<span class="cit-lpage">887</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DValdar%26rft.auinit1%253DW.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D879%26rft.epage%253D887%26rft.atitle%253DGenome-wide%2Bgenetic%2Bassociation%2Bof%2Bcomplex%2Btraits%2Bin%2Bheterogeneous%2Bstock%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1840%26rft_id%253Dinfo%253Apmid%252F16832355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1840&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16832355&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic measures of relationship and inbreeding.</span> <abbr class="cit-jnl-abbrev">Interbull Bull.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">3336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInterbull%2BBull.%26rft.volume%253D37%26rft.spage%253D3336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.32" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.33" data-doi="10.1017/S001667231100022X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitezica</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bias in genomic predictions for populations under selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DVitezica%26rft.auinit1%253DZ.%2BG.%26rft.volume%253D93%26rft.issue%253D5%26rft.spage%253D357%26rft.epage%253D366%26rft.atitle%253DBias%2Bin%2Bgenomic%2Bpredictions%2Bfor%2Bpopulations%2Bunder%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS001667231100022X%26rft_id%253Dinfo%253Apmid%252F21767459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S001667231100022X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21767459&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-web" id="cit-7.10.3543.34"><div class="cit-metadata unstructured">Wimmer, V., T. Albrecht, H.-J. Auinger, with contributions by C.-C. S. Malena Erbe, U. Ober, and C. Reimer, 2015 synbreedData: Data for the Synbreed Package. R package version 1.5. Available at: <a href="https://cran.r-project.org/web/packages/synbreedData/index.html">https://cran.r-project.org/web/packages/synbreedData/index.html</a> Accessed: May 6, 2017.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1921</span> <span class="cit-article-title">Systems of mating. I. The biometric relations between offspring and parent.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWright%26rft.auinit1%253DS.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D111%26rft.epage%253D123%26rft.atitle%253DSYSTEMS%2BOF%2BMATING.%2BI.%2BTHE%2BBIOMETRIC%2BRELATIONS%2BBETWEEN%2BPARENT%2BAND%2BOFFSPRING%26rft_id%253Dinfo%253Apmid%252F17245958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiI2LzIvMTExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMC8zNTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.10.3543.36" data-doi="10.1086/279872"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1922</span> <span class="cit-article-title">Coefficients of inbreeding and relationship.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">330</span>–<span class="cit-lpage">338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D56%26rft.spage%253D330%26rft_id%253Dinfo%253Adoi%252F10.1086%252F279872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/279872&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200856800029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-7.10.3543.37" data-doi="10.1007/978-1-62703-447-0_9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide complex trait analysis (GCTA): methods, data analyses, and interpretations.</span> <span class="cit-source">Methods Mol. Biol.</span> <span class="cit-vol">1019</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%26rft.volume%253D1019%26rft.spage%253D215%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-62703-447-0_9%26rft_id%253Dinfo%253Apmid%252F23756893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-62703-447-0_9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23756893&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/10/3543.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/10/3533" title="Single-Step qPCR and dPCR Detection of Diverse CRISPR-Cas9 Gene Editing Events in Vivo" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/10/3557" title="Corrigendum" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/10"><span>Volume 7 Issue 10, October 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/10.cover-source.jpg?itok=lurMO2QE" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3543" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-gZVecWl7hIhjqkRsgtXU_cosKYxcFXE-mmNZOcYWKek" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/10/3543" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/10/3543" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/10/3543" class="active" data-icon-position="" data-hide-link-title="0">Genomic Relatedness Strengthens Genetic Connectedness Across Management Units</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/486783" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-K9Wj80NnanTSq9NS8ClBSU6EuBM3B07aEXoxnF1_U1g" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/10/3543.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/10/3543" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-EXD2Ucg2I4Dodp4ZI8fYJB14pozyMtBaEfOGECKxFI4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/10/3543" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/10/3543.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486783" id="node486783--2" data-pisa="ggg;7/10/3543" data-pisa-master="ggg;g3.117.300151" data-apath="/ggg/7/10/3543.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Relatedness Strengthens Genetic Connectedness Across Management Units</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Haipeng Yu</span>, <span class="highwire-citation-author" data-delta="1">Matthew L. Spangler</span>, <span class="highwire-citation-author" data-delta="2">Ronald M. Lewis</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3567-6911" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gota Morota</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3543-3556;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300151 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486783--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Haipeng Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Animal Science, University of Nebraska-Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haipeng%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaipeng%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Matthew L. Spangler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Animal Science, University of Nebraska-Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BL.%2BSpangler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spangler%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BL.%2BSpangler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ronald M. Lewis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Animal Science, University of Nebraska-Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ronald%2BM.%2BLewis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lewis%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARonald%2BM.%2BLewis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Gota Morota </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Animal Science, University of Nebraska-Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gota%2BMorota%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morota%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGota%2BMorota%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3567-6911" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gota Morota</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:morota@unl.edu" class="" data-icon-position="" data-hide-link-title="0">morota@unl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/10/3543" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/10/3543&amp;title=Genomic%20Relatedness%20Strengthens%20Genetic%20Connectedness%20Across%20Management%20Units" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/10/3543&amp;title=Genomic%20Relatedness%20Strengthens%20Genetic%20Connectedness%20Across%20Management%20Units" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/10/3543&amp;title=Genomic%20Relatedness%20Strengthens%20Genetic%20Connectedness%20Across%20Management%20Units" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/10/3543&amp;text=Genomic%20Relatedness%20Strengthens%20Genetic%20Connectedness%20Across%20Management%20Units" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/10/3543&amp;title=Genomic%20Relatedness%20Strengthens%20Genetic%20Connectedness%20Across%20Management%20Units" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/10/3543&amp;t=Genomic%20Relatedness%20Strengthens%20Genetic%20Connectedness%20Across%20Management%20Units" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/10/3543&amp;title=Genomic%20Relatedness%20Strengthens%20Genetic%20Connectedness%20Across%20Management%20Units" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/10/3543&amp;title=Genomic%20Relatedness%20Strengthens%20Genetic%20Connectedness%20Across%20Management%20Units" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/486783/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486783" id="node486783--3" data-pisa="ggg;7/10/3543" data-pisa-master="ggg;g3.117.300151" data-apath="/ggg/7/10/3543.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Relatedness Strengthens Genetic Connectedness Across Management Units</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Haipeng Yu</span>, <span class="highwire-citation-author" data-delta="1">Matthew L. Spangler</span>, <span class="highwire-citation-author" data-delta="2">Ronald M. Lewis</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3567-6911" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gota Morota</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3543-3556;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300151 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486783--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Haipeng Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Animal Science, University of Nebraska-Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haipeng%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaipeng%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Matthew L. Spangler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Animal Science, University of Nebraska-Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BL.%2BSpangler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spangler%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BL.%2BSpangler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ronald M. Lewis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Animal Science, University of Nebraska-Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ronald%2BM.%2BLewis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lewis%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARonald%2BM.%2BLewis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Gota Morota </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Animal Science, University of Nebraska-Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gota%2BMorota%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morota%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGota%2BMorota%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3567-6911" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gota Morota</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:morota@unl.edu" class="" data-icon-position="" data-hide-link-title="0">morota@unl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3543&amp;rft.epage=3556&amp;rft.atitle=Genomic%20Relatedness%20Strengthens%20Genetic%20Connectedness%20Across%20Management%20Units&amp;rft.volume=7&amp;rft.issue=10&amp;rft.date=2017-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Yu%2C+Haipeng&amp;rft.au=Spangler%2C+Matthew+L.&amp;rft.au=Lewis%2C+Ronald+M.&amp;rft.au=Morota%2C+Gota"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/486783/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/486783/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/486783/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/486783/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/486783/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/486783/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/486783/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/486783/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/486783/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/486783/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/486783/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/486783/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/10/3543" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/10/3543.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/10/3543.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/10/3543.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/10/3543.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
