<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/10/3533" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/10/3533.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/10/3533.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/10/3533.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/10/3533.atom" />
<meta name="HW.pisa" content="ggg;7/10/3533" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Single-Step qPCR and dPCR Detection of Diverse CRISPR-Cas9 Gene Editing Events in Vivo" />
<meta name="DC.Identifier" content="10.1534/g3.117.300123" />
<meta name="DC.Date" content="2017-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2017 Falabella et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-CRISPR-associated protein 9 (Cas9)-based technology is currently the most flexible means to create targeted mutations by recombination or indel mutations by nonhomologous end joining. During mouse transgenesis, recombinant and indel alleles are often pursued simultaneously. Multiple alleles can be formed in each animal to create significant genetic complexity that complicates the CRISPR-Cas9 approach and analysis. Currently, there are no rapid methods to measure the extent of on-site editing with broad mutation sensitivity. In this study, we demonstrate the allelic diversity arising from targeted CRISPR editing in founder mice. Using this DNA sample collection, we validated specific quantitative and digital PCR methods (qPCR and dPCR, respectively) for measuring the frequency of on-target editing in founder mice. We found that locked nucleic acid (LNA) probes combined with an internal reference probe (Drop-Off Assay) provide accurate measurements of editing rates. The Drop-Off LNA Assay also detected on-target CRISPR-Cas9 gene editing in blastocysts with a sensitivity comparable to PCR-clone sequencing. Lastly, we demonstrate that the allele-specific LNA probes used in qPCR competitor assays can accurately detect recombinant mutations in founder mice. In summary, we show that LNA-based qPCR and dPCR assays provide a rapid method for quantifying the extent of on-target genome editing in vivo , testing RNA guides, and detecting recombinant mutations." />
<meta name="DC.Contributor" content="Micol Falabella" />
<meta name="DC.Contributor" content="Linqing Sun" />
<meta name="DC.Contributor" content="Justin Barr" />
<meta name="DC.Contributor" content="Andressa Z. Pena" />
<meta name="DC.Contributor" content="Erin E. Kershaw" />
<meta name="DC.Contributor" content="Sebastien Gingras" />
<meta name="DC.Contributor" content="Elena A. Goncharova" />
<meta name="DC.Contributor" content="Brett A. Kaufman" />
<meta name="article:published_time" content="2017-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Single-Step qPCR and dPCR Detection of Diverse CRISPR-Cas9 Gene Editing Events in Vivo" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-CRISPR-associated protein 9 (Cas9)-based technology is currently the most flexible means to create targeted mutations by recombination or indel mutations by nonhomologous end joining. During mouse transgenesis, recombinant and indel alleles are often pursued simultaneously. Multiple alleles can be formed in each animal to create significant genetic complexity that complicates the CRISPR-Cas9 approach and analysis. Currently, there are no rapid methods to measure the extent of on-site editing with broad mutation sensitivity. In this study, we demonstrate the allelic diversity arising from targeted CRISPR editing in founder mice. Using this DNA sample collection, we validated specific quantitative and digital PCR methods (qPCR and dPCR, respectively) for measuring the frequency of on-target editing in founder mice. We found that locked nucleic acid (LNA) probes combined with an internal reference probe (Drop-Off Assay) provide accurate measurements of editing rates. The Drop-Off LNA Assay also detected on-target CRISPR-Cas9 gene editing in blastocysts with a sensitivity comparable to PCR-clone sequencing. Lastly, we demonstrate that the allele-specific LNA probes used in qPCR competitor assays can accurately detect recombinant mutations in founder mice. In summary, we show that LNA-based qPCR and dPCR assays provide a rapid method for quantifying the extent of on-target genome editing &lt;i&gt;in vivo&lt;/i&gt;, testing RNA guides, and detecting recombinant mutations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/10/01" />
<meta name="citation_mjid" content="ggg;7/10/3533" />
<meta name="citation_id" content="7/10/3533" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/10/3533" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/10/3533.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/10/3533.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/10/3533.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300123" />
<meta name="citation_pmid" content="28860183" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3533" />
<meta name="citation_lastpage" content="3542" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Micol Falabella" />
<meta name="citation_author_institution" content="University of Pittsburgh School of Medicine, Department of Medicine, Division of Cardiology, Center for Metabolism and Mitochondrial Medicine and Vascular Medicine Institute, Pennsylvania 15261" />
<meta name="citation_author" content="Linqing Sun" />
<meta name="citation_author_institution" content="University of Pittsburgh School of Medicine, Department of Medicine, Division of Cardiology, Center for Metabolism and Mitochondrial Medicine and Vascular Medicine Institute, Pennsylvania 15261" />
<meta name="citation_author" content="Justin Barr" />
<meta name="citation_author_institution" content="Integrated DNA Technologies, Coralville, Iowa 52241" />
<meta name="citation_author" content="Andressa Z. Pena" />
<meta name="citation_author_institution" content="University of Pittsburgh School of Medicine, Department of Medicine, Vascular Medicine Institute, Pennsylvania 15261" />
<meta name="citation_author" content="Erin E. Kershaw" />
<meta name="citation_author_institution" content="University of Pittsburgh School of Medicine, Department of Medicine, Division of Endocrinology, Pennsylvania 15261" />
<meta name="citation_author" content="Sebastien Gingras" />
<meta name="citation_author_institution" content="University of Pittsburgh School of Medicine, Department of Immunology, Pennsylvania 15261" />
<meta name="citation_author" content="Elena A. Goncharova" />
<meta name="citation_author_institution" content="University of Pittsburgh School of Medicine, Department of Medicine, Division of Pulmonary, Allergy and Critical Care Medicine, Vascular Medicine Institute, and Department of Bioengineering, Pennsylvania 15261" />
<meta name="citation_author" content="Brett A. Kaufman" />
<meta name="citation_author_institution" content="University of Pittsburgh School of Medicine, Department of Medicine, Division of Cardiology, Center for Metabolism and Mitochondrial Medicine and Vascular Medicine Institute, Pennsylvania 15261" />
<meta name="citation_author_email" content="bkauf@pitt.edu" />
<meta name="citation_reference" content="citation_journal_title=Methods;citation_author=A. Bassett;citation_author=J.-L. Liu;citation_title=CRISPR/Cas9 mediated genome engineering in Drosophila.;citation_pages=128-136;citation_volume=69;citation_year=2014;citation_pmid=24576617;citation_doi=10.1016/j.ymeth.2014.02.019" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=S. Chen;citation_author=N. E. Sanjana;citation_author=K. Zheng;citation_author=O. Shalem;citation_author=K. Lee;citation_title=Genome-wide CRISPR screen in a mouse model of tumor growth and metastasis.;citation_pages=1246-1260;citation_volume=160;citation_year=2015;citation_pmid=25748654;citation_doi=10.1016/j.cell.2015.02.038" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=F. Corpet;citation_title=Multiple sequence alignment with hierarchical clustering;citation_pages=10881-10890;citation_volume=16;citation_year=1988;citation_issue=22;citation_pmid=2849754;citation_doi=10.1093/nar/16.22.10881" />
<meta name="citation_reference" content="citation_journal_title=Nat. Med.;citation_author=D. B. T. Cox;citation_author=R. J. Platt;citation_author=F. Zhang;citation_title=Therapeutic genome editing: prospects and challenges.;citation_pages=121-131;citation_volume=21;citation_year=2015;citation_pmid=25654603;citation_doi=10.1038/nm.3793" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=J. Dehairs;citation_author=A. Talebi;citation_author=Y. Cherifi;citation_author=J. V. Swinnen;citation_title=CRISP-ID: decoding CRISPR mediated indels by Sanger sequencing.;citation_pages=28973;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.;citation_author=M. Heidenreich;citation_author=F. Zhang;citation_title=Applications of CRISPR-Cas systems in neuroscience.;citation_pages=36-44;citation_volume=17;citation_year=2016;citation_pmid=26656253" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=T. Horii;citation_author=Y. Arai;citation_author=M. Yamazaki;citation_author=S. Morita;citation_author=M. Kimura;citation_title=Validation of microinjection methods for generating knockout mice by CRISPR/Cas-mediated genome engineering.;citation_pages=4513;citation_volume=4;citation_year=2014;citation_pmid=24675426;citation_doi=10.1038/srep04513" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. Jinek;citation_author=K. Chylinski;citation_author=I. Fonfara;citation_author=M. Hauer;citation_author=J. A. Doudna;citation_title=A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity.;citation_volume=337;citation_year=2012;citation_pmid=22745249;citation_doi=10.1126/science.1225829" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=R. Kc;citation_author=A. Srivastava;citation_author=J. M. Wilkowski;citation_author=C. E. Richter;citation_author=J. A. Shavit;citation_title=Detection of nucleotide-specific CRISPR/Cas9 modified alleles using multiplex ligation detection.;citation_pages=1-7;citation_volume=6;citation_year=2016;citation_doi=10.1038/s41598-016-0001-8" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. E. Kolesar;citation_author=C. Y. Wang;citation_author=Y. V. Taguchi;citation_author=S.-H. Chou;citation_author=B. A. Kaufman;citation_title=Two-dimensional intact mitochondrial DNA agarose electrophoresis reveals the structural complexity of the mammalian mitochondrial genome.;citation_volume=41;citation_year=2013;citation_pmid=23275548;citation_doi=10.1093/nar/gks1324" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=K. J. Livak;citation_author=T. D. Schmittgen;citation_title=Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method.;citation_pages=402-408;citation_volume=25;citation_year=2001;citation_issue=4;citation_pmid=11846609;citation_doi=10.1006/meth.2001.1262" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=P. Mali;citation_author=K. M. Esvelt;citation_author=G. M. Church;citation_title=Cas9 as a versatile tool for engineering biology.;citation_pages=957-963;citation_volume=10;citation_year=2013;citation_pmid=24076990;citation_doi=10.1038/nmeth.2649" />
<meta name="citation_reference" content="citation_journal_title=Expert review of molecular diagnostics;citation_journal_abbrev=Expert Rev Mol Diagn;citation_author=P. Mouritzen;citation_author=A. T. Nielsen;citation_author=H. M. Pfundheller;citation_author=Y. Choleva;citation_author=L. Kongsbak;citation_title=Single nucleotide polymorphism genotyping using locked nucleic acid (LNA).;citation_pages=27-38;citation_volume=3;citation_year=2003;citation_issue=1;citation_pmid=12528362;citation_doi=10.1586/14737159.3.1.27" />
<meta name="citation_reference" content="citation_title=PLoS One;citation_author=D. Oliver;citation_author=S. Yuan;citation_author=H. McSwiggin;citation_author=W. Yan;citation_title=Pervasive genotypic mosaicism in founder mice derived from genome editing through pronuclear injection.;citation_pages=e0129457;citation_volume=10;citation_year=2015;citation_pmid=26053263;citation_doi=10.1371/journal.pone.0129457" />
<meta name="citation_reference" content="citation_journal_title=Immunity;citation_author=S. Pelletier;citation_author=S. Gingras;citation_author=D. R. Green;citation_title=Mouse genome engineering via CRISPR-Cas9 for study of immune function.;citation_pages=18-27;citation_volume=42;citation_year=2015;citation_pmid=25607456;citation_doi=10.1016/j.immuni.2015.01.004" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=F. A. Ran;citation_author=P. D. Hsu;citation_author=C. Y. Lin;citation_author=J. S. Gootenberg;citation_author=S. Konermann;citation_title=Double nicking by RNA-guided CRISPR cas9 for enhanced genome editing specificity.;citation_pages=1380-1389;citation_volume=154;citation_year=2013;citation_pmid=23992846;citation_doi=10.1016/j.cell.2013.08.021" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=E. M. Ribot;citation_author=F. D. Quinn;citation_author=X. Bai;citation_author=J. J. Murtagh;citation_title=Comparative PCR: an improved method to detect gene amplification.;citation_pages=22-4, 26;citation_volume=24;citation_year=1998;citation_issue=1;citation_pmid=9454944" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=S. Ruiz;citation_author=C. Mayor-Ruiz;citation_author=V. Lafarga;citation_author=M. Murga;citation_author=M. Vega-Sendino;citation_title=A genome-wide CRISPR screen identifies CDC25A as a determinant of sensitivity to ATR inhibitors.;citation_pages=307-313;citation_volume=62;citation_year=2016;citation_pmid=27067599;citation_doi=10.1016/j.molcel.2016.03.006" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=S. H. Sternberg;citation_author=J. A. Doudna;citation_title=Expanding the biologistâs toolkit with CRISPR-Cas9.;citation_pages=568-574;citation_volume=58;citation_year=2015;citation_pmid=26000842;citation_doi=10.1016/j.molcel.2015.02.032" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=H. Wang;citation_author=H. Yang;citation_author=C. S. Shivalila;citation_author=M. M. Dawlaty;citation_author=A. W. Cheng;citation_title=One-step generation of mice carrying mutations in multiple genes by CRISPR/Cas-mediated genome engineering.;citation_pages=910-918;citation_volume=153;citation_year=2013;citation_pmid=23643243;citation_doi=10.1016/j.cell.2013.04.025" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Z. Yang;citation_author=C. Steentoft;citation_author=C. Hauge;citation_author=L. Hansen;citation_author=A. L. Thomsen;citation_title=Fast and sensitive detection of indels induced by precise gene targeting.;citation_volume=43;citation_year=2015;citation_pmid=25753669;citation_doi=10.1093/nar/gkv126" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=Y. You;citation_author=B. G. Moreira;citation_author=M. A. Behlke;citation_author=R. Owczarzy;citation_title=Design of LNA probes that improve mismatch discrimination.;citation_pages=e60-e60;citation_volume=34;citation_year=2006;citation_issue=8;citation_pmid=16670427;citation_doi=10.1093/nar/gkl175" />
<meta name="citation_reference" content="citation_title=PLoS One;citation_author=C. Yu;citation_author=Y. Zhang;citation_author=S. Yao;citation_author=Y. Wei;citation_title=A PCR based protocol for detecting indel mutations induced by TALENs and CRISPR/Cas9 in zebrafish;citation_pages=e98282;citation_volume=9;citation_year=2014;citation_pmid=24901507;citation_doi=10.1371/journal.pone.0098282" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=L. Zentilin;citation_author=M. Giacca;citation_title=Competitive PCR for precise nucleic acid quantification.;citation_pages=2092-2104;citation_volume=2;citation_year=2007;citation_issue=9;citation_pmid=17853864;citation_doi=10.1038/nprot.2007.299" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=X. Zhang;citation_author=P. Liang;citation_author=C. Ding;citation_author=Z. Zhang;citation_author=J. Zhou;citation_title=Efficient production of gene-modified mice using Staphylococcus aureus Cas9.;citation_pages=32565;citation_volume=6;citation_year=2016" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Single-Step qPCR and dPCR Detection of Diverse CRISPR-Cas9 Gene Editing Events in Vivo" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
<meta name="twitter:description" content="Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-CRISPR-associated protein 9 (Cas9)-based technology is currently the most flexible means to create targeted mutations by recombination or indel mutations by nonhomologous end joining. During mouse transgenesis, recombinant and indel alleles are often pursued simultaneously. Multiple alleles can be formed in each animal to create significant genetic complexity that complicates the CRISPR-Cas9 approach and analysis. Currently, there are no rapid methods to measure the extent of on-site editing with broad mutation sensitivity. In this study, we demonstrate the allelic diversity arising from targeted CRISPR editing in founder mice. Using this DNA sample collection, we validated specific quantitative and digital PCR methods (qPCR and dPCR, respectively) for measuring the frequency of on-target editing in founder mice. We found that locked nucleic acid (LNA) probes combined with an internal reference probe (Drop-Off Assay) provide accurate measurements of editing rates. The Drop-Off LNA Assay also detected on-target CRISPR-Cas9 gene editing in blastocysts with a sensitivity comparable to PCR-clone sequencing. Lastly, we demonstrate that the allele-specific LNA probes used in qPCR competitor assays can accurately detect recombinant mutations in founder mice. In summary, we show that LNA-based qPCR and dPCR assays provide a rapid method for quantifying the extent of on-target genome editing in vivo , testing RNA guides, and detecting recombinant mutations." />
<meta name="og-title" property="og:title" content="Single-Step qPCR and dPCR Detection of Diverse CRISPR-Cas9 Gene Editing Events in Vivo" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/10/3533" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-CRISPR-associated protein 9 (Cas9)-based technology is currently the most flexible means to create targeted mutations by recombination or indel mutations by nonhomologous end joining. During mouse transgenesis, recombinant and indel alleles are often pursued simultaneously. Multiple alleles can be formed in each animal to create significant genetic complexity that complicates the CRISPR-Cas9 approach and analysis. Currently, there are no rapid methods to measure the extent of on-site editing with broad mutation sensitivity. In this study, we demonstrate the allelic diversity arising from targeted CRISPR editing in founder mice. Using this DNA sample collection, we validated specific quantitative and digital PCR methods (qPCR and dPCR, respectively) for measuring the frequency of on-target editing in founder mice. We found that locked nucleic acid (LNA) probes combined with an internal reference probe (Drop-Off Assay) provide accurate measurements of editing rates. The Drop-Off LNA Assay also detected on-target CRISPR-Cas9 gene editing in blastocysts with a sensitivity comparable to PCR-clone sequencing. Lastly, we demonstrate that the allele-specific LNA probes used in qPCR competitor assays can accurately detect recombinant mutations in founder mice. In summary, we show that LNA-based qPCR and dPCR assays provide a rapid method for quantifying the extent of on-target genome editing in vivo , testing RNA guides, and detecting recombinant mutations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/10.cover-source.jpg" />
    <title>Single-Step qPCR and dPCR Detection of Diverse CRISPR-Cas9 Gene Editing Events in Vivo | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Co2HElbCLWebigueZol932hJhl4L7LKyrOYEPuR40WI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"486761","apath":"\/ggg\/7\/10\/3533.atom","pisa":"ggg;7\/10\/3533","ac":{"\/ggg\/7\/10\/3533.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/10\/3533.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/10\/3533"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/10\/3533"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/10\/3533":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ef8LmKBbXF8l6ng0cTypYtbe6dSg9kM9QUgoixc7JtASD+lJDFB8nn4bBn05p\\\\\\\/NQE2V+IIOC7EWM8XIjvi3HxLZP+aQ\\\\\\\/1k+1U9y+POdEN6oNiNEe4ejpS6ea34v4w0KBg6CtP93gYU9PLkdwtvTNV5r1GAnsN9UT3MXPG9lTIIqmkvFsafy3SWoWYQe7VMVjDTHZb0t1+IANdVlpwauMHcj9mmXIiakRMuuPl389QmZBtFAy52m9OlvOV\\\\\\\/U1dSP2WHroYQAwETSSpixZAVbA2CWYsesTr5FdJtiF4vWYVu\\\\\\\/\\\\\\\/gqZ6KQgOIe2tOem1NdXc\\\\\\\/snHJDyw\\\\\\\/ptUhBqLESE5\\\\\\\/qE9GmXc35p7v2zaVG8BOSXHNrxry6RPHl42a6paiyFsOtpYjjarUbcZjED9PbO1o3J4RfjpQnlsYX4K6v+wPKi4x7IxH1mKpnQPtdrmB0vv04LxjIffpia\\\\\\\/SL+574udmjlZoLiVE2hudtXUqDg8NuzzPDCD1MXgOR7ubxXPmKbkae9CAaApP6BK3xFB0o971P4jZfo0q0sR71GL5dDj\\\\\\\/gw8\\\\\\\/cCZWkuhbWyCF\\\\\\\/nqEKie2u1VNx3Rd25ZBPjEEbjDmEqOHrizLoihH4iGuUx\\\\\\\/\\\\\\\/b194ovmbUPpLF1IvXbpS2R5RULI6W2U4Tt56EbkA9aA4ntD5K7PerGevZdhTxk+V0SWClgbjGWxxTmy9A77uOfPcryR8X1uDoEEMoEwqOAV5PsWf+481V2Hif3\\\\\\\/Tm+JjHipUX2IZe3kVMKxzsQlFfKlwRG4u1nHMyA42YSW1FnkudDk\\\\\\\/8vYG\\\\\\\/XeQ4i9E75JTyF\\\\\\\/d\\\\\\\/U9EjgiB5hSRwSYlyfeG0xGBwcyCCBuixs7kpRt4PAgmIqawft3zrU9x61LKEzXIt9xPSr7nJryf+mGgBzRT9+9ulrx6SCFWvGfPwhONJpU8xfUwF5B8KvrTCp13jILNN7j0rz65ibKpyWzDzlb4NAt+IiV2Pz1PyKZsPWNNBJ8dCrkrU9UdxQMclppyTlN08WHP74dCaKrJSxokNoamcvI7L77zwr1PnYIRdt8vVSSbiUOiLMJg4Vo83Tt7M+AcbfopOgzt1P7PxeyjHlIhNC3TZ\\\\\\\/7fHOoDbfhu3r3uk\\\\\\\/wDLqHNLE1wgv43i4LyPChEiCieTzXEhterUCiDPrx0JmPt1rQ1vZ\\\\\\\/4HOGiFsP7PbQp3KrUoIooq5TuGqbzKGqwnmmYaQDeN78kRcgrAt7nz+t8uKoo6AwcbKVaGhqzYQf\\\\\\\/8jweGbQusDg+J312EBfEIukBhUi+0VJZl2hRaBm\\\u0022,\\\u0022iv\\\u0022:\\\u0022fX31soao+fvnLv71t+O7xw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022004cf45b4083ea62d280b81659e95e524cfa21d89ce6ac563b2f2b7efe8dc13c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZKUw5aJGV4x2\\\\\\\/gCEOvoQYGVYq9Ukj65GCDaeJmvWQb3JHV\\\\\\\/vwOU267\\\\\\\/8tP0DUo7hdRvuvONu6Jb1e5Qy64pEPdgi8JbC\\\\\\\/uBWdC5q7VATCF4O2479Yu2CpVmHO7583Ks4x\\\\\\\/PsattxcY14uyWkSvGX7sdTLIotED5ycvcO9TMeRYj80XPG7zdfM2kaspzyoMRD5cz\\\\\\\/0o5yU\\\\\\\/\\\\\\\/ucmU8\\\\\\\/zplEqdKjZjWT70ZC9by8Mjx200SB3xkNGF3fybRO3H+Xv4SxPtoANmJrOx95n311z5Is+eiByNWz8Y1tKE15UTVUwH+Uhz0GaHD3t381h39aB4V8yUop9Vmkk\\\\\\\/tAdBrug7Y4eO2VxtlfpqQtLIJ2PimCXaH8jPiCDjt9SYLbxRAQtClz07wEAWsXTJW1yDwsGwwczJcRYiTc8pgmLrwg1tmBxCDKyp0ygSJAh2PD4IkEN1ozvO6YOhrBBEyvgrT95dJSq8GCKdtUN+3Ym55BmI4\\\\\\\/MnoZCAOT95yE9k\\\\\\\/LHb7gCzQkZOXmn\\\\\\\/ywWyv4H3HxX+kjgGyohficgFX2ABgvsK9nKZiX+PC4iG8lJgVa\\\\\\\/waeJv1gZpw6+SSLYoiO9bxkBc8BE4a3i2PKb5e91jwbsRlk\\\\\\\/MrJnFtSE3yC2YS4B4Fiwsj+4y9ArkDxcP5bUx1josaRKENbeUoZc8bRkniZFsqQKa60OwzmaftjtoTN3o6v5JUMEi9FwkPKgt0m4r76ychkjIwyW5dK6+TZPOB\\\\\\\/OgsA\\\\\\\/i2fpNlPOOidQD+NEf0KZmeJtAO5l8fDCORRWxXWfzjfCGAZLmj62CTd2ZHrflhD6KwP+TCuIRF3bSJ5gIlaNwZNfu0E+yDm+0L8UNDDNgYN+Mf8\\\\\\\/L3AhZXlw6PlA96QsjIp2it3N5SI4iJMVAjKg\\\\\\\/qyiFtCCfhsBlfTKQnLg0nDntGx1\\\\\\\/hvCDcojcpDlqOIEa0kxuwGyqApZHKj8uJZ4aSkAuemys9Bcg3WpuVYB2BlzfWr\\\\\\\/vTn5+oMOZAlcwyhuCs\\\\\\\/m8P5e0pqLZdHQiLiatp70YB6\\\\\\\/iweeftSGafWhnuz5lGcHomcrPYvRxM81U\\\\\\\/tQEXpUD5bZ43GVkPVJUUZdzIgqXiFMOfaZZe6G35k1pesCsvOWRUveHEInB1v3DMxIg7hzZRDbK9YAa9vSqW1pTtvoasXXV7oqPbE+pIaUNFmd\\\\\\\/6UbDVgS8DoNsy7opDNi6vbOvq+C6UMczRoWW5iuDSNJIUglZzYi9v7b9Hp\\\\\\\/0fOEGzIU8tHUKA4xaPofSM1PccLYrQCrYQuodW+Nc74yMf1KGa\\\\\\\/4Ou6Oeonr1GQ95qHsOQO88yCVy1Uo2ePr6VxcMfRZIlCR\\\\\\\/M2ChhM4pH58rHvrC36swqSfIn0VsPTWQBkSkEsTKhLKWyrI4nYbCEslbbP2rXgh\\\\\\\/mk70sq8sIC13308RuNbq7nQn4KFAptEAnhJptJMzS566P3OjPYe61M48JmcRJSAoO2rUCH2MwrzWyvlhXtnBFrZHJ\\\\\\\/mGDGyq3Uipf8t86aZkILPIa9NGg1Q8TvjGm1gkhPcAnZzBUiM2KplY8f6ol7Z3MdtSJT8ys45aWm8gDsWOz2kQWi\\\\\\\/tdYfu3oxW9IhIYwW08CVJf8D6iG54xuO\\\\\\\/mAhXpzexv8RT\\\\\\\/NmTbPAJtd+s3Miacknmgwd8oYVWFbopPE8IhvK2KI6bgUAsuHxO+X8LBValx0PRqHLc6lo1WDQwsh16pj\\\\\\\/cTIp68SuG\\\\\\\/j74vlMnF\\\\\\\/18tX5Vd8MZaVtKnGTTE1Ch6jVefVW7s8F6dfrw3O5ZsZG5w9IpiYqAxfHsJvJMcXvu1+EgEXyY2cRmzUe8IM1EHdYnEsd1dGsTH3g\\\\\\\/1ogKLHLFuiLM7ipJUTwUOzclwg1UEXzzohFO7qzovw11gMA3LApT143ydzdrX8Bv2ZDkEpSGEg5ktR7oFHyXxSzrA2pBO1CRPIpdbP8NJSD5XL8182sSjpblT1NMKChvumhywG2pXdekRvgJDuZ\\\\\\\/QOJfuX\\\\\\\/UanhCs3eJl7Do1R6csDeJ1hikDWc9UwfTynpJdI4+RNwbOlhz8TfwdAkQ8KehR6aH7Oetfz0TvQERteuPiWJ3Xhawdf86yQQAG9lGYMvoq64NhDGqJB\\\\\\\/0WrZLzNy+p8zk6kIC04ZTgMIGZcUP5fDd\\\\\\\/OwgcUxksH91aGNYdm3FuTdMazgkDx8bocHRjqxu9XH5BVZ\\\\\\\/mTMcn8r+\\\\\\\/AD5o3Y\\\\\\\/bw83Rjvbao3jqA0iUb92GGhWwAmrZCbGGy+d7GBBeCT+HH\\\\\\\/HtrNUV1VSEnnn\\\\\\\/0\\\\\\\/POehJn4FjeNVmTb4269WlU9ZmFMRfIjFArgglvQmb\\\\\\\/CMewAAVzCMbaXisifrNp2nSj\\\\\\\/J+3ZmXKrQi36k\\\\\\\/iaYJeQQ4pf5GrVJ8BevvPdb+WpgMUwkr4m6DTeh4DmJ3tGT0kegNFg4orZywIqrzze0OPQNLCH68FONfMRL4KCdtM+VcPxpes7doO0o\\\\\\\/\\\\\\\/8T3AnbDVKwwRZvPIPIUoF9Izr7P4yPXJ9LbKJghrwraHH4aCX+tEZT0DbSmcaQiwqAsKzw2DrzzLRaT3q2aXr3pccxdzjQiMtPjburcGYoz\\\\\\\/4H2PYTsdE8us+F2dZVNu5FSUBiTOGxMn74xVpUJ02y74sI5DmzTb4WMqSBrtBSjFc1LAYC6njbL8BMdGuSkDPLlZ6Lphp+svkdFRW6zbKQY64gLI38T6pVF2aSdgRRJcr8cB2Y0UsV+0MqIe+ZDh5PPHpJltFS0daIzWuqtjXpU3YcOJACMtSaroj11uXKGQRHzXu5Xnhj95rBmjlQqurFOlVsQWhsFmtm6zyTCG4DiRQstxfwYBpg6qyfs4TyDpBabdchA8Nh3d8B3\\\\\\\/PSGhJT6fqM\\\\\\\/AH8S\\\\\\\/IXW5w1aRXVfNZTWjC8RNAOK1QdvYmKw\\\\\\\/0v1bzsS4t3kWmAv8lYZ0z2WTv0nVjLvYoLBQA9TEXewyVjF9rdZY6MkimvDEQguuH+DI++fi2HaXl5O0P1wHffn8jJlKSGY4\\\\\\\/u7peUaOlGWrSTBhINc7rIB1TIG42RRKEfqSR2wGYTTh9t\\\\\\\/pS9TCFq\\\\\\\/Y5sU8KT0XwrHaX6fxRRlc\\\\\\\/a1OW2t0t\\\\\\\/FTvwaLifoSKiJXkc2jU7+awWAaMxcfXTFOTZlip3LcPDtpitZhOXz5OfXZmOIxudEfHh5vRbtaTZKHIrswDwF6xysJKwQZ\\\\\\\/0NN0ndGT0DQeH25hYbhxDAAZG2fqe+rtgQqXVD0xqBaiEWqdEfRhF3dkyecvv09tnu9VWbBOZWhLecAfTiJ08Cm2oLS1LS\\\\\\\/ynIy4XjVQsU0E42WWSREq4JxFsTZn2t7kTVo4DuSNQHl9+hGpjBllOd+8NU9XChLHyEfv4mauY6WCyY+UQUU7nTqQhlj5xx54VEfw5jzfmOWlBW1xVwjIVjDr8m5xdAvmPG9lxiv+228lLQVwPVqjswwYuUCk8wLP4L53tZYul2bBCj4MSDZi1+3\\\\\\\/VXHREdZv6e0puewJRl2omqb4t2JaMgUDWkD3s+6k9\\\\\\\/Yp54AFDdsu1rKGD4PKBDjkFZ4MdKVrYkZ87629LgqC3WlIoJC00Wr08hZQ4AA4DxTPZcM9IRledfE4Jhwgm6bckLl5uTXQ7RdqqVILtISazzcQ4izYMcHQl32HTjGATwWnBwkYJScheY9F6tTG8IKb0IJ1L0eB07Dlw90RABhs4CiW7vsd6fuRCA0fGKHLd84JxLq7q6Ro0l\\\\\\\/i6KkTdXahLu\\\\\\\/zfWXexyug2gLhik\\\\\\\/uCwF8NJTmVeERUt8QFBChDVy8KxDhe6+0PVh\\\\\\\/+K3MCHa5QCY9Kbm2mXTk3gtfjyPv8PFZePGZb0USSXW6uW5CtXny3zRHq0NNDOy9tqH32VaKIDYNsssOr95qtYs7KKz93ZmvwNfOAbhQ2CB3TLLj0880WHvkMrHBJvpaWW+0JhgTW+aPMlB2Le9OcPDH+xVQ+cTcdfM8NWMsTmkpXMuE7ijhKHP4+5ayebVEVMv45QyEDxpBqVX8m98TTL21PBqJRsIS4B3EE1OaZt\\\\\\\/kA3DrT2ba\\\\\\\/32aNzSO4Wcvkey\\\\\\\/gQUlF2jmqSizH8fx2N7sG03ldrYYkUgKb8BMGd3kfh9oxzaijD\\\\\\\/7gY3PO9A99GLuxTmAUs13745UPsbfIOeuK6N4ejvuBkYnMkaQL2EBLONTqxhmnUbPITTG1+4hDME5v1BIqFl0C9VmHX\\\\\\\/v7froTtxFyu+jfRrLlDNjOuenjUtRIzBxoO7c01vI8gwvf0xFjAcQDgr3Xk36RSlu7x5+pbpEslMDV2TWMp6iNIgpeNbMf+K6vibRDPt1gt0U0nPrjgNz8x++dZjT7pd1PuEZzGK7SsKrV+np1I4MiZZpcYn7GN+saqZkGEkuaKp4rQ+BwJs3J7xHxwxLx+YWSEUILf7zEcLXM5eNIRdbU9snbKlGtoH3xoOI6p6vvbwzTdoGCY1QR4uf1e6hRT9+xDCWc7elv8qii7lUWh8v4CGAydhVIcd8KnzLd+Q1sn0tcg651IB7gmuU7Z\\\\\\\/\\\\\\\/7veqRuWaneFTOQ8\\\\\\\/bE27exXi0DyDk4TaHRuLYdPGhP0PbLl6eRMqGz9XgJjgJMIibyqV4aoxcMxm3Dr07yjthsU7MqF7C+O819WzxRaH9DQiqAB1Lh0Ao0xMWKZluJbzdclm0wyHF21f3PLrtJsVyVLNat4spbUoAyXdxbRZyDiGMAOIr1ctOxGjJIVp3K4p8y4FysNPKF0\\\\\\\/DKlOhYXW3N90nTluNUgl70Bc9h3OXT8nXy3G\\\\\\\/bV0Dv1t0vUdCSHuGcP1pTfb1CR3HZOVHOtB+G65xVYKjJPwrLP\\\\\\\/cBPkml+eAMNUvhPEx46iLKyzO1iRX7MxiW7+Pz+MWdSByaaNktwEnNbYEYUT172sWvcUsB2j\\\\\\\/IEjUR76xPhW8hMLGapmNCBoHCuOCFtBrX\\\\\\\/HP3gh5d0nke5ULXQt7un167PXM2wX5AL7tbzfoUByCVPAynk973cAJk9sKdDvkA1wa+89wc0SVWj5\\\\\\\/jeNf0zfb6TTDbK8z0YErvf4jxIyoJOlv5DMJ0DUZFDJNM1KuA0uhmmDiRTpyMOSFvpSqRXMBPlVlW8knJ6OGhvXfW94mrs3KMNu0yq1PRwYzvqofcycAM5KKa0kmmoSCO3UDobB+HUoqB7KnUHYVrdRz9kjhJblmMuunveCpPihWsdNxWczMK3VQpkWYVg8D32cLO\\\\\\\/8kqhnPAHXkye+x8mY5k91xM1GYLNgIpIr+ybRAtU6RpNrknDYMmZNhIkGmeQudJ6QWOHkZcwcAmKGcNbghYMkfgHBS4u5DZ6fRCa1SkuQGqUgkv1z7WuyHmBDZ8cBUJPLtTXGQO51sUAFa\\\\\\\/OfZ\\\\\\\/H1mU1P6PiXkMmleSrLUOP7JTjb6+QkdGFEPzQCFk8r50wjYJgl\\\\\\\/\\\\\\\/BPav6WVa7PSjSfnhYOcEDAtjX+6aUdgAL3Tq0ijrQT1xDvgr4XViVrdt1f9z2W2B4P0Zaw1eglovz2T7bLZJXTMa9sqk5XYY0Yfb\\\\\\\/svNdHJ6CD57wZ8+Mra3ugBR8EAutusJYfi\\\\\\\/LpLOaBSrQC7DfkzkkaUmFB1WS3LS1EQS5ZI2bVjyuj4G+Ql40D4AdJZ1kyH85HR7mC3LapGav2PAEgEVjpQ+QxNwp3d5TvyIzIka5yWlaGGn95tzMiM2kHvmVxkAKhgJbVagEjARfnerPCqFSEA7BzvJXqhhjfOfOlM0HFLh9nbFUj0jV6DH6eRUo6ycUZqoRZuNSyVUUckDPMILWTTgvg16b2+6zuRN1gF69Hnb3\\\\\\\/A3AvlFZ6RCsvAwRyY3cIsuqLxgjI6bdUqrsV5YKgd89BIRfUrEzxRHryKEg1HFRBqK1CyQG3F7FOuke2pG8tD5yqkcwfMtt0fC2M2\\\\\\\/4StP\\\\\\\/5Hsy70Ah8CL\\\\\\\/sZmoctt1aM5I9+j1dciIhsSF593qdoGTWpGazE+udL7LTFWvIafYOeH8+sODTcsFibNb4fY9f5gdLEQltdOhIaXjHKz0r+zN1o8OfWhVTETxcnugwxfqKqlD8PLxoh3+eORUlizx7L61juV2Go9VRpS+7o7GY3usIAtEOkkSVOX9u9gDAHI5gO+E\\\\\\\/XctpDgG+mTmGMoHXXD7wkVbhW4yAplq2sAYVBST06mvrjiX+0KqZAffXVhllLO1SSYdmSJm54Z\\\\\\\/z\\\\\\\/zKb6I7l\\\\\\\/ZLtGCNfbLy9KrsnGEdLDipQiS4g+uZCIlOhvhPRoOa9fr\\\\\\\/1MOL0oW0\\\\\\\/ABJb09MGqWoKSKT4IprWNAC5TD6Yfl3Gkd6Ltg4q1oztKoYzcFBuL8QurqL8OvhROjqbkIVDHp3nuichdiaybjpz1toYf2QN1BgRdvM0Xcqm4wfzFtx\\\\\\\/UGyEjteGY3A0PHfZemq1NK4a1FMwqJUsOBmZqVa943jl6g7ENDKJser4mZiI1BpumC4UY8kAL3PV1r71l7Z0seu5ubvdx\\\\\\\/CMT\\\\\\\/bDSMcARNcWEZLVAoba5KnfwZtvY5q74HCgejCYBzd9aT1vQ5eyAc3RAeIH\\\\\\\/wzFakKwY6QXqZ5pIA2TgaLiNXMZMF2075fmbz5czIKAFU9pd+6utMwKAL061JxFhaZRvAiN\\\\\\\/kEwhcQ7eftDTA3Nwpwh8N1VnHitcz5c13GFrLp46HBfbLwpWiO14OISDr\\\\\\\/Zqsvhkm9EE3wvvkqMl9l2nJ3eiy1U6pc9ELk6IzK1vhe59lcrXm+t4\\\\\\\/gR5LzQE3LR2Q\\\\\\\/4FA\\\\\\\/uiHP5MsAJOuC5Ce0wTiyGAXRFeVaEqo4LFGfC5iV0ftLiKh+MInTwUtc+PbOIQX76lwLSL8WnAp1zYGn2QUJrtijmzyFaAaGboj8B+63+ykzrIoJ9dEjpTBhwJP+Wbe5y14ui6gHnfRamu\\\\\\\/7KKolH\\\\\\\/yHOEoI6bW0LmEAjhuCov8ed\\\\\\\/S+i6cCh5IhJ92GstdBB4r6Lp8c+1THQHbCXXmK7mo0pZKWGKRbPGQsxLDCfgUS55gfybqNag\\\\\\\/MuppMjBdSsbjIydd8Qj0qAgFsXF2Yc0Wha7FTFdXFPYce94MZ2ysp7vmElGbJdyi2oNqoOcrkaXDuxUKWYwSI94NEyoB8T8sXy6wi\\\\\\\/0IF+vBS7Csln35NiGKa376bRET9jF0MsWOcig+FskFDXlegC6B0\\\\\\\/2D0JeKnjvXGqBKItmnyq3OARCp8R4tSnQ3SG60rYit1PS9cUV4ku3+r3VSrmB4nsl5VlQMGAplqljuO9IBx\\\\\\\/rC8G\\\\\\\/q8Hso9dV9gMaT7Ni+qWk8Z1Szc+0lFh4rGl06vXbJZMCO\\\\\\\/+J8RRGr0hjRgwvAA7TuPg0cIywhWTWaUknC8tuNUXJ3Uh0bODjdJHkPhHyWmS\\\\\\\/muzN4eea\\\\\\\/Nb6Q6ugcgvO5JheXO6hppycTNhdNTH7PxTgy+N2x9ZKPItEc9L0Ljgy\\\\\\\/tNcqTTbdUmtJptzajOBw10JRFhpv6uCwwtohFOB9iyoiZUzAXn7EcH9sarPPmZN7G9IPaowuaNiD871NlI2HF3B7fONBGMpzj3PpXwyT+aw+RNvClO0Ae5wlPKMyvHL8Zfx6ViH10U3BTt0CHH\\\\\\\/fCoKhqT4OBNz2A7PiTLWqPj\\\\\\\/80tR5bCWHnObxtJ\\\\\\\/bxP1fi2gIn5pMc\\\\\\\/ZUOI6xAqXBnPaiQeab6nuwn3zZ1+INlAMgImkdDpa1AUk\\\\\\\/VLh+vAZNz\\\\\\\/IuKuFdWoeK0IU9bTzPIQKrRQiDXrlM83hs0JzD1clMhSid7H0TGhuHa2ZwCfcwHEG27o3eOAhw4Nin+epyWTOCnTSb7T25e3shOHpxrgVk7Ichg\\\\\\\/Sj6SGt+a\\\\\\\/EakNtbCLajxFXgaCLOcSWmksg7eTHXuqqnIozwjO7Qn8QJUrbRLD2zCCkQqiuQ\\\\\\\/i1oFXsbdtKsT17Ko8RVZElixS0Fl17WBmcvGrmxKYP7Q4UKCxt5Sc9oPB31+WCsA2wkQHCJwP7M2m7l8mShr5oEsRPvKa5PsqhrXe270m8+91C28aGKa0PACqm+9\\\\\\\/9t57wEyAmjQOfy\\\\\\\/IfvDIq0s\\\\\\\/GASIXCCyRDWSNaYXdrslzF7TpQpTH7KJ\\\\\\\/YV2IHJwwQKytTtukLbRkhzUPFBlb7KShijteGxQ21zTE9SEB+JKBguTG5pa1F21\\\\\\\/re7jnyIZ1o8zYYTPod9ZTYuNRzo509vEQNtgbeKfbbZqVbg\\\\\\\/gm6SJCD3oTgpTebCHhku6IX0i5ZaC4UwzC84SBJXZ1dSNoXnHU4gHbso9AO+aEaUq463qdrQV5I45j1Dp0mQV+yn4kM0ZUcBPva3XIOGPXJbnUNRxYAV8QtfVHPZKnB8u2nFkPuzRrKz6RBHHAtCaK9BGamnVJHB\\\\\\\/\\\\\\\/nh20YrwVn1cNk\\\\\\\/ubo\\\\\\\/4qAmxARW0lrTRJhmnjPks79WWUz2d74ojdD+KYUBbDuxQIm9TzHIjVvWO5vw4TUPhXYLY3FFP4tg5yHAckuOq+Ps1JLFvIAluScacsh8t2d0xR+dcypkIvNQXlRW7x51SQ\\\\\\\/38\\\\\\\/wEkYDvXFwQq9UMtNaopFhrfOSFK3vZh5h5Y91VrAahM0hnXEvPTi\\\\\\\/9liw1D9kv4xNaUA9QkT0cd4HWnZj\\\\\\\/lvDMYx5AOWWovf5ItTfw+A+qAoZz4hJ7pOA2I4\\\\\\\/NglVnJv+4nAs7bgkYBSpzY\\\\\\\/CW2L18DDkxGigedqda9uDoBBSYGpw0B5XnlJAT8HzdCj2oawGD+WRl+8fzYDnd30Pw9aRQ4mBkSMlJacpq37Qv2w52QGMVdthTckX66DtcPtp71j8SnbjYtqP\\\\\\\/x4BOfXtBAnkcBS3W8AhZvhgVxouI9V0NrP0Fz6E9UaiCBdlgCKfU6eJGr10s\\\\\\\/uZuMAV\\\\\\\/9KpLrk4pq16gbXQ+jEa5C7oebgW\\\\\\\/9rAm6gRENC3CpszISBUhzEkl09yt3HX96nXcFgsFYyCNLoRLfauXgwZpOaSJsCO0ECnx687m7R5aE5YU4RsaMWxqzqZJrPRth7h\\\\\\\/pvsrqw13Z77GOSeZ4g85IoOvm4rZywJCgLHgPm2xAjWl3i6AVZwWohrM6GgUfEyOvCx+gMCpFgjP6rfJrlS7NMtFTaKGtIJLft0EK9l2xOVWuQwDbgvrRHbmMgNzA84VSUuvmE9pUJfvYJieLYDLXkwIhJ75ZA5W3VYldrFwTC0eXj0RABfV9cYUfDtuL8q\\\\\\\/UkR7uGqyjGomM8gVVhlidbnYJfwZDjOqpDI7i\\\\\\\/WjaI7116IRmqT6i2ZRR5PqyG1xUuCxYpEQvCbeqeeMsjKRZqdA4Cnl+CcASOCkItdMFsTGFsSPMKfVbXqx06i00nt33oppj+OlX6gZ7+M8zbBgph787r0tohDISa0I9vUaJSoq2+qGupirPTi30LNeX393iKRajMHgk3q89TPRP4N4ELSaRf+yXRkjqEwEtt+cBpdJ67MzjZfoAJ6y2XHo8HJ++r\\\\\\\/QFoF\\\\\\\/BhsPDyZg6tl3x1uky9PpLyyBibJGCb6vpmh9pE1iA1IItW6+13ZSagBqLgmKUUanMhWZnZHaziTITfM3NkzQkR9SfzFHNvqSx\\\\\\\/YClgvVbo+9cOfJytl4WMWGjCxIJy9ogMTTFteKFJNGk+\\\\\\\/zLvg382WjJU8NQPRLpFSo\\\\\\\/a44pg2a7p77+KW3VgM6TMNvMO8E3y9x0iqwSX1NgJFNGRVdOePHSsHZ6V7Wklh+Kj+I907qJjAIXoTtnSETxS1df1LbasexuWynU6s2zIqLwC3NX6kYEsx\\\\\\\/R9gnypAyVNhBSGfZrI+m0bF0MS1N5VGYgGGaxiGuzp9a4q1A+xMDZzuZFXr2DHgwzl1SiELpWwms2PrpLu0vaGTMxBE+KQeuDuvaa6eYsHuT3VqXWiGfGE0CLCQs9SaVGpKTZIwBSAyI6hCQlVsNgmnuc92jiL2iqfbmJ3fZAVkKnZDV\\\\\\\/oL4vM0iKuryjkjlvo4ngwbRKWyDRSUVFv8s861gRgoiJZlqhjCUJtnGXjaNmjBbPc35XBQ2MEqrfkZydcxkx+ueLuUlkizJzVaNR02PxThcv4Ld\\\\\\\/BWRieokARq+rhSNMW+sq2lAAp1xeVBJMhByGUkFaA9nH0WwCNGeD5I6r2Nw5Qif3pBzdfGYmtwIuHmVY00uailZLM3XUW8ao9uqqvMFzKa24P5\\\\\\\/jOc+sspbosk1RTULqIKKKcCmFNpHRHd+RtXt+1mWYkMWkthZlIsVbSwQpgmBSKU5Ctoh4Ei29QRAlyum4Sa03zv1myor9unk5CskAkX6KCtiiLlvglv\\\\\\\/SUXhevhoHq0aMgXSuzns5vNuZm6Iqonw\\\\\\\/SWGX4\\\\\\\/vByL6btNwIDEXF7W20f6ogdY7gZNITsBvcQuuV2FP0BHjl+Ao3nAZG8zovtqlIaeFUnicvTEAVpHrxazdPP6bPm\\\\\\\/xrnsi\\\\\\\/+pBXZzLP2kGWfeLln2gjKonXsxZHnrfs9wDeubAlNV6Ec5UXKrOFRebSx+NFsQWPc01TvOQdrKlkPkcs\\\\\\\/yNzRUl9yy1udG7g5E4LDTbS9BIxEDRTEC6fVJzxK4Ipt9jaav9kHkWTo+15rhCFyykejF+rls8lB2xsIySbsb0\\\\\\\/IZcm\\\\\\\/Jp7SgE\\\\\\\/WZMWAaYKIX0H\\\\\\\/PNY\\\\\\\/xlrDYXRZtu6cbDxDoGlKqqQDh8SIyfGz29c72Dzn2iuyT+zdzyuCXXnNrXvxn9tQh56ctnDLrphnlk0jE5h5HZUGW++dt9zov4Ym9jyoDHfdpGQOuZEtYjO2MXx0Ack8FdRUD0fvd8vL7yGszEJJVZfr0Z3OfqNvxa4F\\\\\\\/3uAddjgyhy9Uhru74afgViCU\\\\\\\/08VRJ0luxZncNEbT8DYnH7rX+4TX5QRrHXXOpH\\\\\\\/dB\\\\\\\/cPLDP5KPqy4Xki3KxEC2l0K6uTvH2vLvfhuolRF9H86mZ0NMkwndo3hjt74yR03yvrHq+ec1tCD2Tf8OQ2o3FYsK4zJVfWJzy12Rdx2FycMpJHKFGT8mJMxxIAAt5V6Vqs3QJDGje85DlXhXLDKznYyGXpz68I329mUwJsNY6C9a4ZEFjQlIQNswZS4WsI75CiEjvrBC4O\\\\\\\/2WDMB09T\\\\\\\/+tR0+nc8yJl+w70UTXhdLBIoOYxDHO4p40SyCX5oIgNjRF3Ap2hWS3dAUA2AijAEzLntWtKxfQFIrMOtupyPL+irQIqKA0GwtX\\\\\\\/CyWV\\\\\\\/CKsWCV+CThK0x9NeN84CX3SlWtIDlo1S2m4X4j4We24ZsGcoo5FLzLcZIRAf4uctK4TSlcusTCwuL\\\\\\\/H8+uzxGfs+va3S2Mc+dsKwhJdH5xShaCDPYMT7YSP6qXgfr1OlUDeWZuY0Ii\\\\\\\/MlWxtDxITZNYcizdhr4IMjkvFnS9xYc9gzrC\\\\\\\/YPmMDJyqCufUzDQXp6HcxheB6MakVH3SvvjFn1MrrzOY6smGq\\\\\\\/ELPcu8vy1jNHjww5fVxgaBBq58sI3YT8vRMi0pUJYAa5Q7TXdKXH3SNTRDj58xa+TPvjUpXkQzqar1pCQLK+6nvqyLAnLaPUTcIUT\\\\\\\/ZyoiyAiOndL7Pzx1RuzylXwl3hkq4tEfJzFw+q91y2PcXBJSYCpT9yQ4wvkPaTl0TDNk9lIeQpVOxdROJPnFmli\\\\\\\/f6sqgR1D6soK5U4Bk3a\\\\\\\/55VczK9JZBotHbFwKw9Zxti5jt8WeSRGS4ggJ3BAX7JzWEEu2EKDe74EYbX8MUKsqbX9gsZ80JCEnF5LD4y4U29\\\\\\\/rl8nE2R4JCifW+cwDxNYlWv2HMv6ROopYqxW0J5nNRGBeSTVg64Rtcmm1Tb6hM83zxr48vCJq2Oe7o70d0n+z96TPjGMS3BmWjAKNtrGxtH50sIHBN7K0x9+pEvZR7VMUBZD79lCgTblq48RRE9aT+OXfzuK8i6UzPMCCcrxEysLYFzGV0Eb34Gxz3T+7\\\\\\\/RYWI339qErXFS\\\\\\\/N7IQfNKcRfDW+E9SntW3VTI9XBuZGxt\\\\\\\/NJhIencLbeWxu2rz2uSf09zAkg7qk0sVh8dfPXVGSmv9SbuXu7zkJ5u8wgJjcvTR\\\\\\\/COC6WHqTZeSfpjIOQNDtBCiibEE+NeY\\\\\\\/sqgCc9K5JAEEt3U2CZrhVb1NUnv3jx6EbX6id+ESVFrwyyl6Gcr0BxQRx0fQ9Pma7SMcK7ITNqIyT\\\\\\\/rpib\\\\\\\/qNK4Nghz7SyXQkGehUk\\\\\\\/BUCsBQmlRgF9PikB9qBtA8rTB7osR7UlVpscAbEoMN4e3hGdkjlNt4loNPXmy3V2YjWd6dU8F02XULPfhhifYrh7hMY6vwyZGOkZb8M9U7y5Pp\\\\\\\/A8PvvuvSQTC9KAd439+2pehpGyENkb9qm71I7NB+V49+OW\\\\\\\/iJnVgmaqsd5JY6Rv5Rfg6d6YYDLUtb29hNcCu1r47VXRfZ3F5fCt4mSpY9Tw+d7TdrigFrX2\\\\\\\/\\\\\\\/LePQE2ayJ0TLlivT6jJLxIXPQKxwEKM6fXlLtN1BHWT\\\\\\\/Ttqn4EFrokIm\\\\\\\/xAaJdi0CFtV+hkENR5f94NQ3NyXyyOUfUb8m45nP5B986h2yYADcqd9SFXpF2UbG2XA2KBe36GPLEApIcXww3ZA70i\\\\\\\/GHH+myU4aw\\\\\\\/XXEZpfzP2mp+\\\\\\\/0broJDzUkhwHD\\\\\\\/wJ\\\\\\\/LIGAoqY5\\\\\\\/Bb8U\\\\\\\/N74PWQEK49kBgbbl0n9LRQACjGQhZ1qqlDDHl48qnSxMvZGUBfOMMiNjc7DRUMk75vY05X6cjkPfL\\\\\\\/VfI7cPImql1+aHTNAU\\\\\\\/\\\\\\\/cgPD3WHS8vd02W3PRFmO7BuP1SbeN5t\\\\\\\/LFAQI8LPDXf9DpuxQ9fj\\\\\\\/djP\\\\\\\/eo3\\\\\\\/4OXvHct\\\\\\\/yWOE0ZB6krd6aXNzdswZ58yIrqldA0lL7DbbXsxMittEgfgXJfawqBwzextD4khQibjJH3dmGfQ+we25+emsj5Ou\\\\\\\/46pKttJhaXfciT\\\\\\\/Zyn9iZ+i2+73Udmr1Tx5RXGx9bm3Tx90jibkK4d4d1ALdvxNuFUR50KnNwx8PNPVHmwHanOlSY0hQq8ruXSCYg0JOAV\\\\\\\/3AYI7WxkUXEmkkKWiEfvprTPW4bKEDnsviTJwIRPEH5Rs8o7xaAPuMgZI7WBYY9uioC9bf9dMsy1kk0cxTGWvG1xC5O95bCXC1M9FY6DpsZ+t8xc5buQiNfLrFN09X4UYzy7xL22K2Lm88HPISDxwp1omXmdO\\\\\\\/gDVDRTbvITi3RbnjqKlX97Ab5BgW3zOV8URmgAGagu21PMutoEkzi9+lOsr2i1yPCdcbM2w3h7OWIerJlz8hiib\\\\\\\/IB+7j+T751zM3L3z9kmTCZwgHp\\\\\\\/bWVu8Eln4tQL4XHTJrSpphw7xuvVjoiabq0Gh6P5QkVM5InQqXEL9qWOns2vLsW8qZMoysQcso3yR1Qxtz16Lh\\\\\\\/Dj51uIASM3B5C7ufRC4a2lw2kc1ntZLhYqSN+fQdY4QtkxdI+3Txe74SFhkWgXtELcj3tE1zffD5MHXFblV3S5JIMzjW9a\\\\\\\/U1J2uL9vNOaqkOy5B8qU\\\\\\\/Z80wln4Wa9Y1XuDpZF3tA\\\\\\\/2tlQfcGWLWjJh9XJtAYf0OoVIS0RuNKLhTk5GhP\\\\\\\/j3+CImOnF0YwR9XePCzR2H7Kv\\\\\\\/sofBa6opoz62Bn83j\\\\\\\/ZrOiav3lnAgQzQsmRMSBKRajSGEbIS87e1SigNBtp2P44Rzkbiy4gCeqO3MaPpunX6xCJWvlz8PKzfcFpirXNT9pN4sx1XfXGRlUW5mowI9YOZWmjcAlYrDMwVJm9frRZBo5t6d1twY1qzdmxeqLqnIXQOLdqJC9LQkQJB4FBbJaDZBB1pLPOI7UM6FDyUb1tDAlkNfhVcArzYmbe4mNNkQaTTyx9UPkKFRB4QbJt51pYgDd4lTCwWu26h6\\\\\\\/OqTjpuX4WbhaGAEJMulmusyrot62A8oFVSzgLedswFPabAX7frNP+Cq58R9DY6CLUbzuhG0V7cRRYjrzc8lKSt1oqHrh3CxcsPHuB\\\\\\\/oET+sRzICbXEfcMU91Fc6AQcu8Vw\\\\\\\/NdNuYw1FsRpvVYTDMinYSeHyiXmGbkjwsq\\\\\\\/TO1jPqvy9h3nxkNO+m1DDRhLB35gL\\\\\\\/QSqubDLYA8dr+xUjk1B4f6SqHLlRrXzG+oj9IKQDa07HgpzwNAEWGi61oMxE6VidM2Dg63bN3JBVtHuhl\\\\\\\/JRMFBHnZw7e5ztmDzdDWKS9\\\\\\\/YnIB0kA+ZVT2xvQvfF\\\\\\\/PKnByV\\\\\\\/Nr8UE69UHRo01nIFlsBYUtdRNUCdOGPvZq+Zfnu8rTqjSXi2H4uddpC5WXxG2d36AyULdL\\\\\\\/DLfAxGbmY2El\\\\\\\/7eSGbwLDju35Q6DfLA4aaomBV3guPrDP4WIuR6K5PO\\\\\\\/NQ+N3bFDR\\\\\\\/I3CFVdVf1ypCIzNwtOFXWcF0RzEF8EKHnWD0XT2GN\\\\\\\/aWnRoejQ3mrsD+EVjYXGKz+NaAN4j8iC\\\\\\\/OJfbfaDk8xkgzZ+JmaENzbi\\\\\\\/bCagXdYmgDmLKSpebU4CWQSJQ4PsO7pGiWQf4NVfT6jzzb8LiB6oKJvhR3dRsVCI\\\\\\\/\\\\\\\/gZQOyV4ks+3b833ACYf1ilVJqcqEbB+ICgsihne10Xn4qMGLh2oVCkczmTBtFNsFCt8FiY0ijYIM5GUvzIGxIA+27DcpNqfyXHt8ZjR7HH9IxPMxG3u23W9\\\\\\\/if1vzbA5zbipoG9KJm+vbbWjXWHB7ybgmj6zDy0IJMHr\\\\\\\/JFsj+WEolcuwXdvb8yvlI8iKtC1vel0jviNrVt8vNeT6UH2yBlmoX\\\\\\\/6Q5018\\\\\\\/WjhVXiABs3fZj9jh9m8o3QsxYl0MYY3rvv2npfZFbAEiRPPU8+6cHcOCVCJ413bubdbuXIdjmZTRbmXbrbhP2K+zT0Fgfn7fquGR8NZ3anNEyJpFyu88yPLaNms2hl7OjPEiSR9BbkwZsj1ivO+JCxeMa8As+\\\\\\\/KTh7UIP2pH4NmK6dIzTAGVHWOjKN1OTSqFZH41DP8B\\\\\\\/e\\\\\\\/Ss85oxYZMCRyHpkR7GgiSNj62A14lP927XeIYHSnwnhECvQ\\\\\\\/7oNTeXEiX2DK8hXyh7pcIFl2IsPUuAIkaSj17M+Y\\\\\\\/45sHlkFGLWDq5Yj72HgA7RRBSNBi8hmRxQx4klaqYQygTzdq5HNTuE9UfGg9Z8mzbZpJ1HMNudlxtyPVRaxGTyxNb+TRrMDumDl3otfWKBsLs5neVvV54JNv5pxjjD5AynE7NjfOvWmXHsTOEIcWvSky\\\\\\\/e+RG5A0zZDPgDbib+1JNLZXdBX5kBrOpqY3LSwMQmnbonzd\\\\\\\/WBkdKUOi+4krt7svfFjdsQtW3\\\\\\\/rbWMiOS0FWvqCFiW+mz1RFqhOMEWilAK60tWpcsswh\\\\\\\/B6QRXZmPejsmwDjfwVyxc8\\\\\\\/LaiiBBcu548V6gA2fJyvQavLlm21x4g3d8MzovW1uspcWn73KI1sdsmPqUyvIyDHoSNHjoHqZaS7m0ZQ6QsGgJVxFhbjxXeNuCAhJJDEdyYTCZrfZ3Mqik4SI0W9FQKtP6crRLCOwiIApIUmIeFcl8eV6mQ1hQEJu\\\\\\\/zrXbgtF1X3S9pOYEfLTIUHmeOSVmnxyRJDuE2LH0nrvNJm\\\\\\\/g+QB509gwkXhd63oAlrdn091+sga+gjbx0rW2oowgCMoIPQpgq0xHFMFuj6M0ejB8GvX8UdMm8u1cN4m3WmoJZH6UL8BwNzBF49TIwAcpwvjU1E0bDnLFCleNEuETbPbzKhbXsGp5LKBF6bUa16i7P+4+T4I\\\\\\\/trxQr6+92IE5ME1KQJoPt\\\\\\\/wePAKSKpB2bd23a3tWM9ErdMZ7ui\\\\\\\/fZ3ygXlERMie45IUznPEUPR28Kfu3bC2Z7G99boc4jTEIlXKLPGtTxEffzxXPJail8jeA41GDqgoULVR7xk\\\\\\\/vzPdDDEN5aNU0bzndXv2MKSCTk3ItuIeQo9RLxAHelqKr+20HaKEA3wx85YC6dw70l4j748UfnShIZMT+if+Ttt9PZPRYdWm6oWK9rYx4pxvHkPhxJOxFpGcjqZ2AOnEREzesJbdWKwRJnPaySG9qkw8IP9NBTy4xQN0WdlV+58XPftqmAfuasuUa0ZB25CRrq8jkaUxDByvswfFEWgAGo9PPUtGhkG+vBx7Q284kffTSWDnJy9vy9kMzcDl4UnkHJzW4kHWmxCUhos70bJzlMQ4610qx+PnMbcHJAAbx09C1Q2nSQvejyZPk8lV2V1cLPf9O7rRvDFW3qXjIVyXbFtfIxulE6zJgjx0FMKnZLGrocOB\\\\\\\/Xwfbi+1wtXObleKaJbmB+WVZQSQaOINnBXMGK0E+c+4xD+CWvdxCUkj+fIzwY3aJfjAsabYtqDADIjiBBa13lJNPkxMHiCHVOOKhfArlfsKXXDfrVY4iqR+J+IQ28bjsSdGNU4dCsKBGiNTIj968Z151FdfB98\\\\\\\/r1xsR9gw5q97FbOu5RcrEmwuuFb73nyIiCSbIbjNhNCXWCVlWzj+z6nkJsoOKf1TLW6WYwupmvwMEmr9ukMLhm3PhcImurrTvDmMBP2CcRxUfcoKy92TdIGYjO8GOQrqt5RdLLaGN3kCXmJTUGo37tvfga\\\\\\\/tVY+0bXOzALOAtOlwNrpwOMEraN4\\\\\\\/nF0eLQf\\\\\\\/ZYsEmTYERjfsJ8E7Dsq1N3cgztmidKpy80IxKkVtrvStJkZMxNYz6zWaV6n4ObmT89Yu++Cd4\\\\\\\/CV23EQG4aGMOlDUK1IPvP2FR4qcA5x6qCP8ON5eApY91Rfvgq9YK3j85jGKbL6CjPAZj1otMqe9JChfCg0CTOR54DhyLKJlFap8l\\\\\\\/B3TTkSN3kJ2hvwFIbLmh8++dPz1Ft7BmYqhMWR0pJ5rgdPGVuTRquyj1wohQRTex940NaW1kFotyVCMK3770rarw+letFYIwpyk2qsbZQQrY28v18EMZV7A6JUcfT5l3b+oN96\\\\\\\/vmzzHPPPGrhqx9XOO5PKuRDGZyLosdS\\\\\\\/zI2UhB5tkW1MnmWDWB5M7MMVMJP2OVEtdsV4oVnunAJLA2ISVs3GSmcHRDnYHd7fJIYC6TRQRQ4tYZy9Xg62vqpBR6j4ooglUFdbk6T0BCvJ9zNz1FRDkPU7jcfr1jglEOjRoQF5Dv13Qb1NuN9E8PH8VrM93pHqsBn+dRN+vYgIdiQoY\\\\\\\/FUoegZ82YZO3iYUtKEwdL\\\\\\\/MTfpcvE0RpbRJ1K9Yoq53sXCt\\\\\\\/As1Y8JWutwBjKEikh6I9s2sZnVUvHkQKjkgV+ZDXbss1XKIiwo67IiTXjrhfuGJzjQMjZxUv3LV4UL7fvg59H3u\\\\\\\/T\\\\\\\/a0Vy3d2mjivvsRCfvQPTyut13Lnj84BeRHsoNEQYGnSxyXcAz0ti3W8EsE++\\\\\\\/oMOFJSWFSxif5Zr5U\\\\\\\/6XARKArCl10U4vawDixBDGZvtL8Yldeizt2NbuThKJHphExGwIwO+ZQjAdXUTwYscAHX5ZXIAESGhEJVFCv1WtG2wGoz3ESqtz6t3xtZ04LfJ5234pxqKtId9njN1NemG4nEjWD0axeAnYn1gT4F+NlisZgQWUhUtU368HPTV6kmd5mC+LoGsheJJs7URfOkIUpCHX23GjTA9wcmUCaDbL10dSeiUMMVoCodPt9JfTi6ZoRq08+NM+HOvi9ctqQLnIJiB18MWffHjPUtEwoZQNGlcMcDKhNBXvZiQZzKWisSUiAiwUdo90ExpQVEzsxulMPPMiCFUkTzJy2wwvw7py1RMOJPotwveLcPQ3+iXC8\\\\\\\/FYns4pf9kgrxYmesEpFr\\\\\\\/YaDF4NyB64+yRDDWgcYEUMzql6cpNv\\\\\\\/VCRvsRUYtV7vcuYa2lMXIljTcTB7HBBeA1Gxo7V1emtyToBIrCr\\\\\\\/lVLb2oS2bmzge2KHTWzla0vY72+4Vc8N6WuoEg5fq4f6zem4v3Hf0sDwvD72MRXzSAmKYKtsX7dzECmiH2GJBduECN0T4NM3sJm\\\\\\\/Hw0Gys\\\\\\\/A5gGRHgKb46tV8L9+wFMTK3Car8yl19d23CrjCvPZbY\\\\\\\/zHBpCimYKwm6vtMaNOvkX2nVTCAVrs3b\\\\\\\/tbgqr2s5OxASvlwpra5HNmX6k8G1bg1cb6+k7VRdosSWs9hqGThcqI4wBPe+zTBi3Bz8DIzKIyeInB9zmd9KnNnd\\\\\\\/pZj\\\\\\\/R3Ky5VEhEb4p2qvbKyX+HMO5UNPW\\\\\\\/6uxXERXcEguvKaHuBjTT0Ya59j141xalv6dAaMUzR61EuXP8rDzDAmMydNdzlRNW0mjd3gZx1YOJOdIi\\\\\\\/apoZKectrwhO3YdWzU21eQRRvbRYVr2am12VFUBPkBBkzUXKNQmFte8SZW2h5LsdBnycs4l3HfNH2zcS61Bkfncb3FlBvT4rxf4TZ7gbbObLA84kmoqganZBoUwQh\\\\\\\/A72em8wbS7wXlIout1B5wd\\\\\\\/gLN+OY6p04\\\\\\\/hOCNH4bomEdgBGvccnhuO5VYCUbQGvqotM6On06kp89i7r\\\\\\\/+5p3qpXr\\\\\\\/+efpdvFMPX1vZwgntasWsQUUaDTs4BorWzKGDE7m1XzPSItkCDJdVP06oqNuLDJFrkeB0WROtUAOItR5+XoD5f4G4Ye7kqot3yBhFoxDE0rSc3Fjox3PO035xwaWiuI+yNJYpZyqXaOVYL2TRzEXdh7zlyFZqwuK0Nl2v3hGM1FPNxZokd6EXcDz8RHoK4ESJA4Uj7GYx0Wp78vjCIJa3l0iFBuR3+NLQrP56zHNDAQSgyRZ9xLVXOiUsVP5SUm69q+kpa0bR\\\\\\\/2y9GssQw2K8CvvDG5hS3pMxbXikb1zPhzPrBvUhTGPyWMBbH9vPxgrKl9Evp9a2drpeWvYMK5POhQz\\\\\\\/EsMFCNclx61F6xXBYXdFl1w12s3OQCdM9dFnsF8WeewjnK06dnGUN0LgLU+mf15eu6W46Z8FUTRZza0Bdby6lxCUx7N5sZZ1yc0UyxQD58YAV4y\\\\\\\/3GZa7hhP10e8ZsEPdPVuXLGx0G4eKc+SEM8skjuf\\\\\\\/bqUQ0opAYfQEFL2ngE7vd0xK7epYxLyAfWvRC2wdyblNck0JzhoL8oefQLn\\\\\\\/0NvFvkjydp2v\\\\\\\/7c2uaJ\\\\\\\/erc2aGoPGQYumF1V3RHYlJrRgAa9TSJIB723TlD5v869N2d85cWyxWbrj8P4AJ\\\\\\\/FhK7lJIUGRlNM76DxHAAYoXruinVZelbPK4NfErJLlYzD\\\\\\\/HMvNuXE4bLQxYzT43kIzFb5aYXE\\\\\\\/k0PGn+I9jddLGbEUfghK4Sb2jC6EEK\\\\\\\/S5kUibpzpn\\\\\\\/wa5iQ0Z8LAcqucsGWNAKHEE0LewQvc0AbI2NkifEFiisrSrrl8DXkL+kdejC0pEeFdrfd6Dc0JDXcySKWjvcq4gERCqZbWhm64J6etvxWYy4NYz3etHxXSGYqaMp5NwsxRhZApwz8aW7DL6iLusD3Cpo1Vfzm3GLyzWPox74r1IB0azaUcnM1ysTYGAZ9oeh6FA\\\\\\\/MEVKFMQS33nAmcrQIrkNpz\\\\\\\/24i2PrEPL18Jbas9Q3CPnMcgSw7OKAnLFEd6sVbY91u4PoMMxoT7XrTUPor7R9EJHZY5\\\\\\\/iNARO1aJ1Br1rqjzZ3xjDtQojsf5qbaT+5aFtXjNvdwY5guOmFCWowv+03kIJQ\\\\\\\/w4LsCnstW5b\\\\\\\/MPbZYdJJJk5vDtwEQby3n2tDXvBBQtQWLd26NbcN1cpbsdJMNotOGimB1TH2DhdoylU4Jbnl6kx6k5P+0S8JgdhsqZkaBptmLOk2TgrtS\\\\\\\/d7MrbUxdfe8MBXxQJ7ZCSMD73OZ260uGJs7Gjwe0rFhh3bp7j+76iRj42BURCDrYJnwunyyer\\\\\\\/EEs0pX0\\\\\\\/\\\\\\\/KIdn6fk0DhJ76hCBUpHDL3tQWigKdNEnLlapRme91wGDZwzgrqFKhyw4oMG9hUL9+63r\\\\\\\/4HZvn74B\\\\\\\/JMXlXOQv\\\\\\\/xanu2YM9DCCL0SRssCZsAYOogw3ZJSonw8SbLVxKfawwFbrRRnaUvE1B0hCSb4Ss9nharemvvsRUATWbpP7pJY4II3bDh3DB3KGCx8uou2hsXAXcUB\\\\\\\/zhm9sanVEj+XDdDtK3v2L7+lacvbN5blz1rHKJbmGwMjhi8InBLruDv4vk1+Qwq3eAOVntDOSq9zn75XceF8gu+ovXPQs\\\\\\\/y4eArABoyhlXF4tkppZJ\\\\\\\/\\\\\\\/8mO9lk0d0tFsAA9vRPNOUGUoiJkhXGZ0cQGxRdlK+LWdrI21ZaeyTweVhh0Sv9ji1XVurnv\\\\\\\/iIGCkgv7bwPIgnZ4I7FkYqg3YwVPg\\\\\\\/InBMgqLWWdfVseqk1wS1uDm+lGlEVOx9t7W5qiZ7FoyNbNaebrcEq5fM1YH63XhS9BewvDZajpei5O9DkkXoZWCL9oZmKWpMne8IilcpeiXQVp92yQP8WjT3Di56dcGau7ZDW0hxr6IJD+QzlYMPWMCmpNK02JXJUJGYourPY6JLYeLDQueqRx6MVlC97PS94sU3yr\\\\\\\/0ATXox6YfqJHLhBFr+TkqTPOA5jElJDczFavsNiHlh3P267UD7a850m4AUdoodngBslmF3epQzlaCS\\\\\\\/YX00CUk7crUEhaBTYfCQASlfT0OViaqSEI0xhv9X7sptuyPmIR0CnbDrWv50GHSIKWSfjSAqu1ve+gnc9s1wGiLof72rlkQH84RLeLtNedRl6HS4LlwteGHHrfsQ1nbA6YYfYQpaDOW7LXMYvYi00yszEjOtxiOyW5JExL+En28GEHhXZE9QrVUcPvkFGE0ZAjqwID5uusGElrOvYzn133+0Bnmuh4zAZ\\\\\\\/tzQXVq0wIumuNz1uQGUzC0cIwfCjbNPQLF20ucERbVp8cYPy2jO8kj1ssxlfRg1HK4QcKzcDxgNO4bqdLd+8rp8PDgz25FplhYX+828Tj5AWKSMW3Y7JweObmQ\\\\\\\/3g3iCGYNcWUKQh0asvIsqmTsiy4NUmkI3NzMKIvsrD\\\\\\\/LudVsAb7LqokJ6XAwqyiDpFyKCVld+B65M+oTOriyuD60bqNoblLE7v+G1rbmivdznFYfhAzP8QZ0m4cdxpA2IXPlkBsZY8mahHRtFHOAfE8L9lUOPcogV63ACy8yYncA1yE40n0RPUzYAycBj3mcLyvQO2i6f9R6oNM3nFSdapuMBmrhr8w65nRMgehBzP+iimtEBk6wnO+wOUtMr9oOR1RPmLrdTGX\\\\\\\/gT4ftL9MN4qGNQ+6GTq1oIwB78v0GCqSr1GUQTic0A7kkFUmQhWJUi9ActBSqZ6CAX4MNc\\\\\\\/XmQPhVnyeDD9QsRU0OE172vNUrcCWMPnMnZ8FlowO9smDKwNERZo0ov2agL91hvrLfooSehO6U8iZ5Vlmyxi2nBI6sTlPvNxz0HY7j4Ogo1gZLznE9dYPuTpmWESVd7AHWfAQ\\\\\\\/cJC\\\\\\\/rnntbrx3fOwDZDR8TSe+pm+XYcW2\\\\\\\/GhOWrjrGdK9Pd9VBCxhhL+5Dp+IbeF5cKvqeHbWAY+kyTX\\\\\\\/qtyS80\\\\\\\/YSaSjwR3CEMbx+WbcIff\\\\\\\/kbHMM0UmN40jnNzLqZooUUCjPGk8yOLtJYUEuhsmd7SIbp1XZUvr0S9R+q0t+JDeNVLUT3HDoBwKfAhaZa4dISKMEvC7PkLddP0RRQAfumTiK\\\\\\\/Ev3Mah1xh3rEjIa+hTnA+7PN5YVr+Bpby6oUSNo\\\\\\\/Dwne9nFiXx7cWl8kt7U9zj47RPjJ9D6Qp5rBcgQilBtqxWpUukFzl1DjXbDvMdUkObnjECI\\\\\\\/Xw5T8LJ89uZlOdcZEiH7aP\\\\\\\/AjllQ0XRPDUEK1Bjw+CzBlht37IHg2S3zParjTgrBbYRuozbvmOpi7JJWhicfRqp4gNays\\\\\\\/STETKacvyroHxwWdEO3Oi8CKq3AyQxMjw7QqTj5RXLGvSA0LEWPMutONNPFwQY19QBNyEH8pgVQCQL2eON7bkNBtoy2dGk6uOTAa3MEG6B6vPZkvgNUbCdre71tMvEtBG1KXQJbRGs+tCR12W\\\\\\\/mEoK7mOgRGOxyt7EAB9gmi3RJWdNh4tNosQBhh5+sitdoaat+jdhK9Iyg0YF3VMzGVNXC7+HRMVU5e8Idz28ERvhqCMFe6VUbbW5kkc55VW6Q4xDdHfw0wvdy3nBKLh+2KWLlw64u7CMl7eBBarZHVIldBcOrKPjkRxobswhTKVDla9C2TmgXLFbFF7UuPTkV\\\\\\\/CQmtnFp0x+RSexG0wx9ElsU6tOq79+eAGPszAL5G+IgH4Ne8Yr8IYxsUfBGX6+vMo8+XOvBDp7AOZ9SHx5Y6507vMtqVbURlJ5f+H9IbLy+I7SZKKQ+aXHTojHH8eEkP7u4mPy5HZ8Eugj0oen44pb5OIOuKiz8pEpsqtO1FFzfqCrU4KY89v5awcbcqX1JRZ9u6AS7KG2ttIi2U87QRJw0lo9DSTcInP+\\\\\\\/zE+\\\\\\\/P8R\\\\\\\/YusLjg\\\\\\\/9RbTLkfovk7yZy7aDvxpIQVDjBvui\\\\\\\/gxPUuThwtMf73h+i4ZkLCcUjlfV3\\\\\\\/TfNAIVhoqtHQhVUUQVxYgJC3YCi54JdNOOPYwRAM638Ak2xsNNeG1UYcUBytxu3F3vaqvAhApumlj3G\\\\\\\/EqmHdeDiDagfY0Vo2CMHhwuQpx8V34\\\\\\\/5+gcIhqq07nOCpfWmLRq57RbI9p9nhjgn8HiJ\\\\\\\/bvY5My7Hl9J0PbvT0pYpCNt57qz\\\\\\\/uDzE+N5yS5duvgxMR66xCTwwOK\\\\\\\/\\\\\\\/qyMnkI100SWCnY05BAHV4jJzXmI3MnVpVEZS3WwcwKOtVN6JwKv7HZ8zbghkoldjTh5U1v5nn4+c0jcTx\\\\\\\/VMcOZ6Jv8+ZXAXGWp4lmjHmci6HCHiWt9vvgWpGe14ujNDUSk3Sm64tpVpVn7+vHphobWLQLxtTvbA3vyzubi7JieStssZzlyXOAJ9peLdQ\\\\\\\/ZZzFgicaA2V0Ve31c88jvgc+tmPufYGIxUzXJ588wuA7ukwu8OsEdZ6H\\\\\\\/raJgYpHQ2wRCwA4EhPPr\\\\\\\/\\\\\\\/FK7iY+Oey7HTflelr7q\\\\\\\/bHDNxxWmtkHgy3cjqWcBEigT+hC2cuBOcckQ9RkXIwi4igUv84i5MF48dFqiMfTFeJjvantf3RLSbVnrnbIb1FfjOM+gRnFiGvq9sWM564y72tyhhuzOulSoWxMUttTg2TyURmUUQT6XmUnFxYwlDo3YMeERRinSq45SCQDvsn3MN2u+rWfXE2BHJ6dUKe4hSBE6xL0ZOrKl4l7YGFA5nj41GjMVGnBFo\\\\\\\/3fa\\\\\\\/9+OLzpUehNhxfdRYrmBgZ\\\\\\\/+JzOlxL9YhgyCNXkjJuRnHnilzII2dSoRUp83mUcYfVy1P4ATEq5zzYV4g1E7uAwmlkRvhjeixRTmnxJ2ayUCjJSoNiPVnyb2cJPTYw\\\\\\\/wr\\\\\\\/3BqrGZcLYE8PoO82nbgbCEvI3zY9zbF5XT11fSL2xh1KHvw6hTgKSfycOvXxnh9479IzgxVo18DxjRiHacph79CWxMQzpJqs9c8L81auY00ZFtrlvH8RAuOg5sze44+g9KJVXeyNyUyAaLynpTlv69Ah7jQVRaHThvYv7kbsncDgj5mNF7EIx6\\\\\\\/7itH470bEUkFd6EwbFmAqvDNMUVqiTLp9ZVxs3Xne6DAIn0qWxFQAinGPL2q585EnAIYd+vjJRIkck3InmIOIqZLPiSK4CN140WWaKZIq8ACole6UZTyJV5xJNZ\\\\\\\/HuOvZlbb7beduy\\\\\\\/w0dz3Yn1uwIk6MRzsXasO0uQoE10T0nsOtb4GQg65cyCwGaqi2rTGcbmlGDiTO30Shw8Xnq9QhPEkLkhDOrm188cZxRs+pRYXd\\\\\\\/+qZ+D9MXQZ0W8d0+ITtkrkFWM2MmYgAKH8Bee1\\\\\\\/Pj+4nI7pD8uNk7t9zbxzDM1pGu+A9xeuNLouXiFJBq3HWfOBovmR2FE+1gLk7GRE7ruvlfiakGxu3frFYdIurj4CKEWTvr1nlyOD5luDLkom3iunjATFEJ8W1AVdyto\\\\\\\/RYNeEEsF12c84Xbb2h2QBKgCiBZ+pKFb58VueWaEQh1X2GN0vz6n8ydD4OgnHzlCwfdMhD0Bv5\\\\\\\/L5oHkfMaXvBbuYxyVLrBwMSl0f3OvDAPZe\\\\\\\/bOO1LvEVyQwwve68jHXIiSldFjiiEV+lCeqEFtkGe9Yv35QjM+MDqqIwDDYlNDR20YkzeT3ZnSD8AXJlsxkAU++Umr7rWCvJNzqMK7DBnH51GyciTofgaPKg2ufHzubPS\\\\\\\/nQzvy0oUJJk+vRou8CBus13nFOcfVr4wBkx1OuH8O8LWWH76tFWCscik2XtoD3EWBOXu6foXsj6I+Jh7tcANknYnQfO8ilqVX+Jo3\\\\\\\/nH65wV4xV4P1CGWTu0+2Kjr6E5NGkY7KfOW\\\\\\\/vrDypcnS9ayDY0DEclgIyi6LnFAAjn+fFVY0gtGCBvPToDf62b8dIJgbnMIAVSLZp4ag4LbkW3DzsdI+YfxzE1qZbF\\\\\\\/fSuiy8yW6zeyYtYZd7jEGqZEVB2kZxMEzRJWNybNPCDG5oMgX5Mui+MEIju8wQoHsp2RXVOjNzxzs7iL5KXGlJbwZ5iM3rPhT\\\\\\\/2uwcsKiWDJpEgrh\\\\\\\/sToiO9vRGwldsWgQl\\\\\\\/ruSfw0+QMpoac5muqh2c7fy8znCqp0F5zGFmdV7ZVUvPxXMW9h\\\\\\\/tWsjRlRyRP5uQNW5ne3KMg1OnAy0Gy6OjVkQUE\\\\\\\/ZcRRx89rIotemavwZfaw3ZIG1eQZ1cid4zZx7jdG2UKO8HZu3+zpjkkH6dMKEtkWrKwxpf4jKJIfqUlmNKZagw7U14B0uetixg9eXRcBsSTOQOt\\\\\\\/vuuzllMWRPbTi1ELhvN2f7T4NfhczxzkNAY5dkAgBGt1hLUjgcZpXRMWMIXU3v\\\\\\\/gEli10IL5n8SfIavvq4pPH9Wmrp2YnWIreXDnfOGfrAgYojw403Z62j3C2gaWZ\\\\\\\/usqjEUc\\\\\\\/sRJgqo7qfxiczg5N3tRnvKOxvpZBgS1X0IEv+\\\\\\\/T6vSQT7ZW+fw0gkLkdgeHb+iivvfF121VWjjJ+LXTisqG4qvkc3cxcIobOI7cAmZqbqRikYwMGCrM3ZuBaunN5JA3fOwLv0KZ1OvNyk24IMkRRGm2Nu5fZKcrrD0LHF9QdXFG5CKMDKGhTjvmFuN4z0bactj5yp7E1UCbVIh4SeeQ4Q+1cdy1k+juEI\\\\\\\/z6Ko5zyXOKsQhoHH2OQFmV\\\\\\\/a6oAfvhSG9Ulu1+bHnowXV2nbvamUhiToHjAiD6zo1oWRlKeQhRa2gzPuB5C6W572YLqIFNDStAQXWdyjv3dNfp1t1jEdbTGZdsYRyuRABqwRwiT+yUy2qbvOQHFgXHIO3jZBD2wolBGcrf\\\\\\\/9zX1uOHTLfWOWz1lGweF+oq+IZ5tDwetfnwvyTI5\\\\\\\/qCKOGr0omkFIh2Y1phCpsHpsftxVrIj1liGwTswXOg2BkfBXkA2spA\\\\\\\/dFIs+UuejM6FUuREsSerbNa+sZiM6lgshtAUeGuTTYGx3MXrd8GAQs5MRTMWjbdoEuMv+sIfojZBYdrLtFkv0bPGAAhjCJn4LWUy1oPjsJBg3I2OpGhl7s5F9kQDHc52wGDxRh4wtjdt2RefkpGFDJ78AY4GLwah2pYNJLM8zxM26nLUW1Z1Ri\\\\\\\/ZyG0URLDL7PK1WbQyBUgIeVLvVMHvBxwMa0ms\\\\\\\/+0xEBO20uM4+vdQw019JfixZe+wj+iLAVAbjHCjbC8yQw70jcwsfj0sqbeOS52YY+6aL4eXgYf4CGR+rIDatHYPkL\\\\\\\/cl4N7gh\\\\\\\/gdJnQsF0LiRaL8oslCZIOKP2cZYq0REQtPaaf+Y5s2+A7ZRHXwVbMPaSq6MmS6+6XunTnkXnZ0Ew5\\\\\\\/LwRem6FM8f+PEQu8jyzSCZXZEp5P+gkr5t4o4QHXQ\\\\\\\/Sb\\\\\\\/TNlEAZT7xiA61O0r0pS4LGBs2IrqTVSAkja5\\\\\\\/z9JU7zJuQY3yKNsmu62dw5XyN7lIbhQknfvvsuA2gDF3giFIBAC\\\\\\\/BukCt8cvgFo8SETYyNDZUncXH+FkpSLarsjr\\\\\\\/40hofehnA\\\\\\\/6yB9ECvK2GpUIJ7pRyUCNQSPZJCjdaub2sdV9nOLzGt+Ers3RVAMhU4xCJMe0LU\\\\\\\/zA0\\\\\\\/Ko957aRPszfvB6arx8qis8pXoGiZ+yhCkbq3qzWNtRlsIHNey\\\\\\\/AIPlCRihGP0Lw9OLUqoIIbbGgTDVSV9dxwnl2\\\\\\\/yjjtaSYXJ8p2TwVYB+c3cb8yUxb\\\\\\\/10fcMQEFSlEubuKWUZlHimyy1XFioKONo36FE+58D3kIKez1euGf9ZObjB+VmX\\\\\\\/8tq1Cgunju8AM8p1xzQOcfhKXc2iw\\\\\\\/Ct1Dd4DWxkXFkyd408\\\\\\\/uzZauYKc80i+jQWZloKeJ+P6xGeJkOdo64KvhYQuL98wW506j8lj\\\\\\\/JSvmDFqKGJCHo\\\\\\\/lPQXSY6JGgTqoLpngLG78L7e06ZKA4gRPrS3XBAB4v9H5bl3gL62Li4OPaQAHSbKXETgSTrq1sJOHnd1HnhU4T6qUyRGv5jAzRYuFTHu2HZ3IQz43iTpIIG8s1OcVN6IL0pC8YXEKbp3i+0m+7\\\\\\\/pMNkBClcyKrxhYh3gGaRs+aG0L\\\\\\\/BLRrYhOk\\\\\\\/y5qQLTWnt+ZYIBVJRXn5gZfKGQwX5fvBVzpnHdXMVGLehMWCt\\\\\\\/CY45uX5zk90DVcquVfTb\\\\\\\/q8Ja13YHBkzm0qfcqXHEI+3pNLR0T3AgsqiaQ0yia3rujY4zddX4n8yd6PsCTPmfBYCWnOc+DZdjptrq6e\\\\\\\/jCrLPjgaSDhnndVPofiQl+NWG0WUW2jvn1lEYBkNUDwTKgrNUmk+g6aOmrXApMsMFgvrHDKsJSkEhYlRYaSsNIFysJPH4IdvK15M89YFlgZpmtRFKbcKTgNgX3qhZS\\\\\\\/0JX1quu35hFn4bAfEabtxEVVNJM+lfJl\\\\\\\/d9yZ0\\\\\\\/fcKY54R82CdcnbnATydkSMbpvFfZ0cksJ2zAmVFL7BFfUVpyNzROurI919lmwsWPO6VPVCpC3ajcg0kWEP3U+8QpLeEG9pT2HATlMYWWoCikNmBU23ayYJXRg6lhYpTfgF5+vg+3ySr7BjNBNf9FuvRlDDzo2mpDXajjkxEw\\\\\\\/GhUlZGdxM+iNU3ZEIslC4a\\\\\\\/0TT5G0hW+U+E30RTo5xB8uvuGI2LG7iluDBusWrPDWznRd+a\\\\\\\/Gu9M\\\\\\\/y4Qj3bKvvM\\\\\\\/u5IxnwMhmXpoduNKEhEeWF\\\\\\\/tcBPy1aHAQm0koc2kLZ+s6WrQDSN\\\\\\\/k9YFvt8P7bI02R8X24Q9lkM6\\\\\\\/BCRXZtMaLLXuRFOa2CFqEUXNjVd7g0ZOxV9MLucHv+fooovE2yGrlWQ8J8oZHDDVok6UThxKluPu7nNgsvCkhKzaXcsv22arbLPhbFdNvjvDGw\\\\\\\/EXS0MpFjPSvePBdH3kSzGW2VOUX0CKv1izLzPIwHGn+OX9np0FlQklZ2hWVchvYVA9GuWg2wDra2R+vce0rg5XyivZaSPWwaKXhbo0BwSzbfw8tkR\\\\\\\/hgizM9Ax5S\\\\\\\/Jw8nCaYcnp\\\\\\\/NlNh0LObZGxmYWQIQUyudRyCiq2WqYr07od3dpqD4HbL+xHD14FquOAhh9Pb+X2lQDnSzkvdjLN4XmIVYKTwPLSHtMNje+AnEKAtbyhaWyeaPDVzzKUg2jW19EVPcJTr4bnAKU5stEHfwfKw06G+Qzi0HKd33+1R233I2J3x2543PxvmFvCWfinlUshQ7yAYNUcqZQmhnz36SOqHKhf0P0uwIA1O3nuoB6Ol6OI\\\\\\\/qX\\\\\\\/qnPlT0ywIuDrXrUMBFSV63Xm1WgN6dSpIpPElacd5DYEqCpqSaDhBQfDNeh5kFwFwJDIpRSaPxqbk4WtHnOH5SOn3+pz2Ec1YGLiCvPz5NzrciKVBce419lDMlM2AD9oJh3vfkXKokuIP3dDrGuoJBkmmRDB4PEG5\\\\\\\/VD8x39zcR4NQ8hJpRHWmDAwnfUhVLlO0D3SQIfecYzvzUm\\\\\\\/0d2cjSTlUb1WNWK5i42WHUqm72f1ZiW23HUk\\\\\\\/jFVGCM77uTILWnGjfhXJ1m1vHdLbW3mDD4EahPQxbjDs9\\\\\\\/\\\\\\\/BnO9j+hv3V4iHdcUL0hWfjN963MIYuYL1EfUOVAUGniG82dDh9QrjcEFGGqjUcD\\\\\\\/fwnn6rsk7rDBRkNxIzIv4OsErzHYXmzaHDSvnhymQBZFDWl0o2YTmgZUl6IiMUwMtrsDmr7jwfT2eE6YHwVoyI3xTin60FVPyJOrDFJ7c3OvgZM\\\\\\\/uPQR15QXrB90U6cChapuV5Ah8BXQ87NGHUgQoKn3ATFZp6XvklpP5DUSkwGT4XDQnT9rGA\\\\\\\/dLqPxXfaqCmGbffe29qHsZENdD5S0Is22aa4QvjOZYrzUxW3W0YcINwC48PtxJydN+nens8BsePMqd3V\\\\\\\/UTyh\\\\\\\/mvIXpMYRTLmDhGrl7k3fRgoPaZdsDILJoDHo9OjpVo6SeQNYapWm1Jbj4FbUgsvP6Mi2xmobS74noJSOMjqEFfEse8QVbsZeOKNAlEGajrmIjMLLtfhjbtkK9eVcGzKh08XDhPwE5DHOck91009gRn0T5BCoY7Tz18XV86F0Y+rdpni4aV+KwJJHypbd8Tyvg2tCiBdd7KrrdTwaqkZ48kjPVDmMe9+g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022vNNxq37qtI3M2RBHINJZ3Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002223f5863eb4c48c48fc9af1b09be2d63b825229fc5bc2fe531a82e39dd4268b2f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ey+Kei5aO2EFH\\\\\\\/YluHUB6BwviD4CpVRIC2+uWEvEnSnxcgzMlvxeIraG8n2x0t68LyNo7vUFMlCVEEk+FhKkc1gMcF4m8Tdo\\\\\\\/9yihMA0IhESbzB38d8Vq+w0KMdjEiq2ObeCTrnYUozQXKhVat27UXxoPSulq6jPNUaolQsxUBeoplrpPPcvlOtltlvt9FdJFjcIUrqDhywDk27jZzZsNHp\\\\\\\/N\\\\\\\/u6rdw90fLfPdzouD8GSYt2XuuPvEHH7u6DlWZc1suHDuCmr8HqKSsp+tm56OBE42S7ddeaq2YENUQZTHx9P6ebRi8kG8OvENqnUcZ910mlol\\\\\\\/Q7XWWXZPNeze\\\\\\\/C\\\\\\\/XUg6sElS7wGsFfPrVCuW2aWZ4M6UHhYOqbGo+pZ2i+pkhbQdJCeImOnyhJATVSKo\\\\\\\/o6\\\\\\\/2VSarj+yTTWqU1EjqmM1SdjO+5bd9Kz6Lkh4ucl1XoT0IRvery45d5Y3mHVVzIgqgHmbHbtdN6sa7KtZ8\\\\\\\/OLo0f7Sea5rU6U931+AiIXXeyVqK8czD9zzkS09rB6x5Sq+SBcMKRyPCSnwRWMYhQoKczgFw\\\\\\\/mxYhwR60JXIB0mbt+JP7sl0KtPe\\\\\\\/QYKZH1bsSg2D8IVXBBvK9B9kGoKGRmAw77vKPyNXI2l\\\\\\\/GZBWAhfHkzG7wweTDzIoHU4bf1Ai7\\\\\\\/5GqfKpUqk0doAzQL7\\\\\\\/YSwiob7nt4biuco8CLGSS9GjyfpeYC\\\\\\\/NGmDmHcwwJvpH93Y1yahVZDwQLlz+7x\\\\\\\/bTfXn8sR2roX+A14\\\\\\\/rfxJD\\\\\\\/zGa\\\\\\\/FVfLixH9NqIrAorLssXsm00zaZoovLTTjAdyje06TM3fFNwldXU2HGUWiYir\\\\\\\/BVQt++uzOlCMLrTVIPK6JTU17YSh6SHijVxaDzkBrrShZYgjnH0vRTkujX0UI0ywCp03B29JQkKX5vc3DCCf9XZrDYXyiXzWQv2r5JPw2Xt\\\\\\\/2pw84ur9ia08\\\\\\\/5GpbcVa+M8Lo4uAAVkgbfcRwGS\\\\\\\/EW\\\\\\\/YzB+LqN8610bSBPvFBX1\\\\\\\/Q+QO789jvhhXMTfQUODaZ7Gklhy8UUxm+4jRhUbUWpb1OQrERLcP9BEcRmkPH1TWRh\\\\\\\/6UKG1Yt4EZLafIie2fMC2BsIUjcMTQbDisen13bhUyDfo3xX1tfz21jtdw2LxzDv2IFq3hKReT3Q9cAuFZHEAp5Gymty0TLcTV1N1tnZLFOWsFTK7W7+9LPCPPRPi\\\\\\\/nN\\\\\\\/48lJI5l\\\\\\\/yg0oOJXcIdqEGaTgWiWNNdXInN6T51F5PQytlQGQ\\\\\\\/3l3oAehQoim4ouNlgBl\\\\\\\/1EYOQQDB05QujOK5N6GF5PmuJWr1CjZQlq5\\\\\\\/xzCZf1XlFZIXlZsmDBGGY3Rv\\\\\\\/IT+zmzVwiLFrNiXEEajPXDS4Wmk4FuFsjCpaLF8l+bv1js8VB1pVgEFqV4pBqFrbie8tfMxdVKRWWQ7k+ADgBfyIvMi6NxTrg9w9fTNs+m5e\\\\\\\/3KlqnOL2xBbZleM6qb4wj2TWdModjQflznQM7Lh0mloWGGs3n3qaBSq4J9gESQsaSxKve6CsvszJaqSBKZN7ZtCM76MOIbIm5P1I4Oulg5ftXzhmhOdx0QXrlE6OLlXIidcacFrXRWX73EeJF0\\\\\\\/B7kUCDY+UyH6AX6dQcWOhHVyb6MwDPcUpJk7SLywCQb4bYlrx2\\\\\\\/SeFAPYeLfX9MXp808y8jX5VkktNhKwbDxml1vh6FHCDZww2\\\\\\\/vRIHHXKOK1wfwgWv+upNVAgQjR4voYU\\\\\\\/YdRadVad6MTs4DA5WrIn3fQWUFSrFFZrpBf8crINd6Ibyix0FcwNGciXMooBZcBcg2BqdCbpnE88qSRxti7MZOn0cuZv21f\\\\\\\/jwGd1wsTiMWJzjuYqAuMo5IcgJQaruCin8hAwiF2OZaV8JS+UZLUqoXwlkDZzsFi4TgIxvvYbRlUauEPEKQ9kx346WuG6GeguvQdz72glWVzNlETYUUxtkUJ\\\\\\\/PTnsESgsUsWaHyWA1lnqWRNB4+FreNqI3IKL3fRir7dIXr18IOXL5X2iecv8QnYEtH4Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/uQd\\\\\\\/LEcfWfSgesLoIVxqA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222009f8e73871070c99d049b3c43f359d764454dda0d9d1a6f6895fd3d43380ef\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-486761 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3533" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BpjI-hZ8xTz0XVszKFqfyjFj1BfwALLYUGDmV776je8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3533" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Dh-5yaZlNwCjct_Z73MZIX_uBtCj_Q7kufjuI9Ab4Vg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/10/3521" title="Overexpression of miRNA-9 Generates Muscle Hypercontraction Through Translational Repression of Troponin-T in Drosophila melanogaster Indirect Flight Muscles" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/10/3543" title="Genomic Relatedness Strengthens Genetic Connectedness Across Management Units" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486761" id="node486761" data-pisa="ggg;7/10/3533" data-pisa-master="ggg;g3.117.300123" data-apath="/ggg/7/10/3533.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Single-Step qPCR and dPCR Detection of Diverse CRISPR-Cas9 Gene Editing Events <em>in Vivo</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Micol Falabella</span>, <span class="highwire-citation-author" data-delta="1">Linqing Sun</span>, <span class="highwire-citation-author" data-delta="2">Justin Barr</span>, <span class="highwire-citation-author" data-delta="3">Andressa Z. Pena</span>, <span class="highwire-citation-author" data-delta="4">Erin E. Kershaw</span>, <span class="highwire-citation-author" data-delta="5">Sebastien Gingras</span>, <span class="highwire-citation-author" data-delta="6">Elena A. Goncharova</span> and <span class="highwire-citation-author" data-delta="7">Brett A. Kaufman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3533-3542;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300123 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486761" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Micol Falabella </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Pittsburgh School of Medicine, Department of Medicine, Division of Cardiology, Center for Metabolism and Mitochondrial Medicine and Vascular Medicine Institute, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Micol%2BFalabella%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Falabella%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMicol%2BFalabella%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Linqing Sun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Pittsburgh School of Medicine, Department of Medicine, Division of Cardiology, Center for Metabolism and Mitochondrial Medicine and Vascular Medicine Institute, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linqing%2BSun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sun%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinqing%2BSun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Justin Barr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Integrated DNA Technologies, Coralville, Iowa 52241</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BBarr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barr%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BBarr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Andressa Z. Pena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>University of Pittsburgh School of Medicine, Department of Medicine, Vascular Medicine Institute, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andressa%2BZ.%2BPena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pena%20AZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndressa%2BZ.%2BPena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Erin E. Kershaw </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>University of Pittsburgh School of Medicine, Department of Medicine, Division of Endocrinology, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erin%2BE.%2BKershaw%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kershaw%20EE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErin%2BE.%2BKershaw%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sebastien Gingras </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>University of Pittsburgh School of Medicine, Department of Immunology, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastien%2BGingras%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gingras%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastien%2BGingras%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Elena A. Goncharova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â â </span>University of Pittsburgh School of Medicine, Department of Medicine, Division of Pulmonary, Allergy and Critical Care Medicine, Vascular Medicine Institute, and Department of Bioengineering, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elena%2BA.%2BGoncharova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goncharova%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElena%2BA.%2BGoncharova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Brett A. Kaufman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Pittsburgh School of Medicine, Department of Medicine, Division of Cardiology, Center for Metabolism and Mitochondrial Medicine and Vascular Medicine Institute, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brett%2BA.%2BKaufman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaufman%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrett%2BA.%2BKaufman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bkauf@pitt.edu" class="" data-icon-position="" data-hide-link-title="0">bkauf@pitt.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/10/3533" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:486761" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:486761/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3533.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:486761" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:486761/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3533.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:486761" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:486761/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/10/3533.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:486761" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:486761/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/10/3533.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:486761" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:486761/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-CRISPR-associated protein 9 (Cas9)-based technology is currently the most flexible means to create targeted mutations by recombination or indel mutations by nonhomologous end joining. During mouse transgenesis, recombinant and indel alleles are often pursued simultaneously. Multiple alleles can be formed in each animal to create significant genetic complexity that complicates the CRISPR-Cas9 approach and analysis. Currently, there are no rapid methods to measure the extent of on-site editing with broad mutation sensitivity. In this study, we demonstrate the allelic diversity arising from targeted CRISPR editing in founder mice. Using this DNA sample collection, we validated specific quantitative and digital PCR methods (qPCR and dPCR, respectively) for measuring the frequency of on-target editing in founder mice. We found that locked nucleic acid (LNA) probes combined with an internal reference probe (Drop-Off Assay) provide accurate measurements of editing rates. The Drop-Off LNA Assay also detected on-target CRISPR-Cas9 gene editing in blastocysts with a sensitivity comparable to PCR-clone sequencing. Lastly, we demonstrate that the allele-specific LNA probes used in qPCR competitor assays can accurately detect recombinant mutations in founder mice. In summary, we show that LNA-based qPCR and dPCR assays provide a rapid method for quantifying the extent of on-target genome editing <em>in vivo</em>, testing RNA guides, and detecting recombinant mutations.</p></div><p id="p-3">CRISPR-Cas9 is a revolutionary genome editing tool derived from the bacterial adaptive immune system (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Jinek <em>et al.</em> 2012</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Mali <em>et al.</em> 2013</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Sternberg and Doudna 2015</a>). In the past few years, CRISPR-Cas9 technology has been used in numerous biomedical applications, including identification of the molecular basis of genetic disorders (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Heidenreich and Zhang 2016</a>), the causes of drug resistance (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Chen <em>et al.</em> 2015</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ruiz <em>et al.</em> 2016</a>), and in the development of new therapeutic strategies (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Wang <em>et al.</em> 2013</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cox <em>et al.</em> 2015</a>). The CRISPR-Cas9 system relies on the ability of the Cas9 endonuclease, directed by a single guide RNA (sgRNA) to the complementary DNA target site, to induce a site-specific double-strand break (DSB). The DNA damage generated by Cas9 can be then repaired by two different mechanisms: homology-directed repair (HDR) or nonhomologous end-joining (NHEJ) (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Heidenreich and Zhang 2016</a>). For genome editing, the HDR mechanism enables site-directed mutagenesis near the DSB site through recombination with a donor template, whereas the error-prone NHEJ repair mechanism leads to the formation of indels at the site of cleavage. Due to its ease of targeting, CRISPR-Cas9 enables the rapid generation of mutations, leading to an explosion of novel mouse models, which has consequently created a need for rapid, easy detection of <em>in vivo</em> editing in founder animals.</p><p id="p-4">It is increasingly recognized that multiple alleles can be generated in a single founder mouse (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Oliver <em>et al.</em> 2015</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Zhang <em>et al.</em> 2016</a>). Engineered mutations created by donor DNA recombination include a restriction-based strategy for genotyping, whereas indel mutations are commonly genotyped by direct Sanger sequencing (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Dehairs <em>et al.</em> 2016</a>) and, once identified, by probe competition assay (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Zentilin and Giacca 2007</a>) or by singleplex qPCR strategies (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Yu <em>et al.</em> 2014</a>). Direct Sanger sequencing is nonquantitative and relies on high-quality sequence to detect up to three alleles; however, more alleles can be generated in a founder animal, causing the least prevalent sequences to be missed by this approach. PCR-clone sequencing increases the number of variants detected, but the precision of quantitation is limited by the number of clones sequenced and requires significant time and labor. CRISPR-Cas9 editing generates diverse NHEJ alleles, making probe competition assay development for all possible variants impractical. Several recent techniques assume a range of sequence variations (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Kc <em>et al.</em> 2016</a>) or size differences (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Yang <em>et al.</em> 2015</a>). These prescriptive approaches do not embrace the reality of broad allele composition in founder animals, where many of these alleles are useful for gene disruption. Rapid detection of undefined editing events at CRISPR-Cas9 target sites is crucial to expedient mouse colony management. Singleplex qPCR has been employed to detect sequence deviation at the CRISPR-Cas9 cut site in zebrafish (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Yu <em>et al.</em> 2014</a>) and served as the starting point for this study. In this work, we show the genetic complexity that derives from CRISPR-Cas9 genome editing in mouse. Importantly, we demonstrate significant oligo probe insensitivity to sequence mismatching, preventing accurate detection of CRISPR editing. To resolve this limitation, we developed qPCR and dPCR methods that increase editing detection sensitivity to enable the quantification of total CRISPR-mediated editing or to detect HDR alleles. The approaches developed are rapid and applicable to sgRNA testing <em>in vivo</em>, founder mice characterization, and colony genotyping.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Production of sgRNA and generation of T1079A, T1079D LATS1, and R457Q CREBRF mice</h3><p id="p-5">The methods for the targeting strategy and the generation of CRISPR-Cas9 edited mice by recombination-mediated mutagenesis have been described (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Pelletier <em>et al.</em> 2015</a>). Briefly, for the synthesis of the sgRNA, a double-stranded linear DNA template was created by annealing a target-specific primer (TSP) with a common primer containing the full tracrRNA sequence (sgRNA-Scaffold-Primer) and then PCR amplified (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bassett and Liu 2014</a>). The TSP contains a T7 promoter, the target sequence [without protospacer-adjacent motif (PAM)], and part of the tracrRNA sequence, while the common primer contains the full tracrRNA sequence (sgRNA-Scaffold-Primer). The purified PCR product was used as template for the synthesis of the sgRNA using MEGAshortscript T7 kit (ThermoFisher Scientific). The Cas9 mRNA was produced using a linearized plasmid as a template for the <em>in vitro</em> transcription mMESSAGE mMACHINE T7 Ultra kit (ThermoFisher Scientific) as previously described (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Pelletier <em>et al.</em> 2015</a>). Both sgRNA and Cas9 mRNA were purified using a MEGAclear kit (ThermoFisher Scientific) following the manufacturerâs instructions, and then RNA integrity was confirmed using a Bioanalyzer or Tape station (Agilent Technologies).</p><p id="p-6">C57BL/6J pronuclear-stage zygotes, obtained by natural mating of superovulated females, were microinjected with guide RNA (10 ng/Î¼l), Cas9 mRNA (20 ng/Î¼l), and with (mouse production) or without (sgRNA testing <em>ex vivo</em>) single-stranded oligodeoxynucleotide (ssODN) Ultramer (0.2 pmol/Î¼l). Injected zygotes were cultured either overnight and transferred to pseudopregnant CD1 receipted females to obtained founder mice, or cultured for 4 d to the blastocysts stage and then harvested.</p></div><div id="sec-3" class="subsection"><h3>Design of drop-off probes and primers</h3><p id="p-7">The murine Large Tumor Suppressor Kinase 1 (LATS1) Oligo and LNA Target Probes were designed to detect the editing events at the Cas9 cleavage site, using three consecutive LNA bases in the probe to destabilize binding to non-wt sequences. The wt and R457Q LNA Target probes for the CREBRF genotyping assay were designed to have two consecutive locked bases on the R457Q mutation site, to improve their hybridization specificity to the target sequence. Thermodynamics and hybridization profiles for all probes were determined using the Biophysics tool from Integrated DNA Technologies (IDT; Coralville, IA: biophysics.idtdna.com) with default settings adjusted to match standard qPCR experimental conditions (Mg<sup>2+</sup> = 3.0 mM and dNTPs = 0.8 mM). To ensure the probes bind to the template prior to the DNA extension, the LNA Target and Reference Probes were designed with similar melting temperatures, both 3â6Â° higher than the T<sub>m</sub> of the primers.</p></div><div id="sec-4" class="subsection"><h3>PCR and cloning of individual allelic variants</h3><p id="p-8">Total DNA was extracted from the toe/tail biopsy from 20 LATS1 and 6 CREBRF founder mice as described (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Kolesar <em>et al.</em> 2013</a>) and amplified with Taq DNA Polymerase (New England Biolabs) using the primer sets reported in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300123/-/DC1/TableS1.eps;">Table S1</a>. The PCR was performed in a ProFlex thermal cycler (ThermoFisher Scientific) with the following PCR amplification profile: one cycle of 94Â° for 2 min; 35 cycles of 94Â° for 20 sec, 55Â° for 20 sec, and 68Â° for 60 sec; and one cycle of 68Â° for 5 min. The PCR products were cloned using the TOPO 4 kit with <em>Escherichia coli</em> DH5Î± cells (ThermoFisher Scientific). For each LATS1 or CREBRF founder mouse, 48â96 bacterial colonies were picked for colony PCR, amplified, and sequenced individually after PCR product purification with LATS1 Fw Primer and CREBRF Fw Primer 2, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300123/-/DC1/FigureS1.eps;">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300123/-/DC1/FigureS2.eps;">Figure S2</a>). The CREBRF blastocysts were amplified using the primer sets reported in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300123/-/DC1/TableS1.eps;">Table S1</a> and Sanger-sequenced directly.</p></div><div id="sec-5" class="subsection"><h3>qPCR assays</h3><p id="p-9">The qPCR multiplex assays were performed on a StepOnePlus thermo cycler (ThermoFisher Scientific) using PerfeCTa MultiPlex qPCR ToughMix (Quanta Biosciences) or TaqMan Fast Advanced Master Mix (ThermoFisher Scientific), 30 Î¼M of primer, 4.6 ng/reaction DNA, and 5 Î¼M of reference and target probes in a 10 Î¼l final reaction volume. For PerfeCta MultiPlex qPCR ToughMix, the PCR amplification profile was: one cycle of 50Â° for 2 min and 95Â° for 10 min; and 40 cycles of 95Â° for 15 sec and 60Â° for 1 min. Using TaqMan Fast Advanced Master Mix, the profile was: one cycle of 95Â° for 20 sec, and 40 cycles of 95Â° for 1 sec and 60Â° for 20 sec. For the <em>trans</em>-Multiplex Assay (ToughMix), the optimized PCR amplification profile was: one cycle of 50 for 2 min and 95Â° for 10 min; and 40 cycles of 95Â° for 15 sec, 59Â° for 1 min, and 60Â° for 1 min). All reactions were run in triplicate and the levels calculated using the ÎÎC<sub>q</sub> method (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Livak and Schmittgen 2001</a>). To assess the compatibility of the probes used in each qPCR assay, a calibration curve was built and the ÎC<sub>q</sub> slope approach was used.</p></div><div id="sec-6" class="subsection"><h3>dPCR method and parameters</h3><p id="p-10">The dPCR analysis was performed on a QuantStudio 3D dPCR System using the manufacturerâs procedure and reagents (ThermoFisher Scientific). Data analysis and chip quality were assessed using the QuantStudio 3D Analysis Suite software online. Two different dilutions were tested to ensure that all chips were suitable and yielded reproducible quantification.</p></div><div id="sec-7" class="subsection"><h3>Data analysis</h3><p id="p-11">Multiple sequence alignments were obtained using MultAlin Software (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Corpet 1988</a>). Statistical significance of the data was determined using the Studentâs <em>t</em>-test in Microsoft Excel; all <em>P</em> values correspond to two-tailed two-sided sample <em>t</em>-test. The reported error bars represent the SEM. Pearson and Spearman correlation coefficients were calculated for 95% C.I.s using GraphPad Prism 7.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-12">All primer sequences used are reported in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300123/-/DC1/TableS1.eps;">Table S1</a>. All primers were from IDT.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Genetic complexity at the target site after CRISPR-Cas9 genome editing</h3><p id="p-13">To assess the genetic complexity derived from gene editing, we used a set of 20 founder mice generated by CRISPR-Cas9 mutagenesis (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) of the LATS1 gene. One 20 nt sgRNA was designed to target a unique genomic site on the murine LATS1 locus (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). To generate HDR alleles, a ssODN was synthesized, carrying either the LATS1 T1079A or T1079D mutation, along with PAM mutations and <em>Eco</em>RI silent mutations (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). To generate founder animals, sgRNA, Cas9 mRNA, and ssODN were microinjected into single-cell zygotes (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). After overnight culture, two-cell zygotes were transferred into pseudopregnant females and pups delivered at full term. At weaning, tail or toe snips were processed for total DNA extraction and a 476 bp PCR product generated. For each animal, the PCR product was cloned and â¼15 plasmid isolates per mouse were Sanger sequenced. PCR-cloned sequences allowed the determination of all variants on the same allele.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Workflow of CRISPR-Cas9-mediated production of founder mice. In vitro-fertilized oocytes from superovulated females generate single-cell zygotes, which are injected with the prepared sgRNA, Cas9 RNA, and ssODN and incubated overnight. The developed two-cell stage embryos are then implanted into a pseudopregnant female mouse to produce the founder mice, which are then characterized for CRISPR-Cas9 editing and bred to maintain the mouse line. CRISPR-Cas9, Clustered Regularly Interspaced Short Palindromic Repeats-CRISPR-associated protein 9; sgRNA, single guide RNA; ssODN, single-stranded oligodeoxynucleotide." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1162861369" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Workflow of CRISPR-Cas9-mediated production of founder mice. &lt;em&gt;In vitro&lt;/em&gt;-fertilized oocytes from superovulated females generate single-cell zygotes, which are injected with the prepared sgRNA, Cas9 RNA, and ssODN and incubated overnight. The developed two-cell stage embryos are then implanted into a pseudopregnant female mouse to produce the founder mice, which are then characterized for CRISPR-Cas9 editing and bred to maintain the mouse line. CRISPR-Cas9, Clustered Regularly Interspaced Short Palindromic Repeats-CRISPR-associated protein 9; sgRNA, single guide RNA; ssODN, single-stranded oligodeoxynucleotide.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3533/F1.medium.gif" width="393" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/10/3533/F1.medium.gif" width="393" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486828" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-14" class="first-child">Workflow of CRISPR-Cas9-mediated production of founder mice. <em>In vitro</em>-fertilized oocytes from superovulated females generate single-cell zygotes, which are injected with the prepared sgRNA, Cas9 RNA, and ssODN and incubated overnight. The developed two-cell stage embryos are then implanted into a pseudopregnant female mouse to produce the founder mice, which are then characterized for CRISPR-Cas9 editing and bred to maintain the mouse line. CRISPR-Cas9, Clustered Regularly Interspaced Short Palindromic Repeats-CRISPR-associated protein 9; sgRNA, single guide RNA; ssODN, single-stranded oligodeoxynucleotide.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CRISPR-Cas9 genome editing creates significant allelic distribution in the LATS1 founder mice. (A) Diagram of the CRISPR-Cas9-mediated DNA cleavage and subsequent HDR to form LATS1 T1079D or T1079A mutation (data not shown). (B) Venn diagram showing the frequency of HDR-associated events based on the sequencing data of plasmid-cloned PCR products. (C) Distribution of 304 sequence variants by allele type. For this analysis, the sequence from GRCm38 chromosome 10: 7,712,787 to 7,712,856 nt was considered. (D) Allelic composition of 20 founder mice based on sequencing of PCR-clones. *inside or outside the selected type sequence; **ins/del, del/mut, mut/ins, or del/mut/ins (inside or outside the target sequence). CRISPR-Cas9, Clustered Regularly Interspaced Short Palindromic Repeats-CRISPR-associated protein 9; del, deletion; HDR, homology-directed repair; ins, insertion; LATS1, Large Tumor Suppressor Kinase 1; mut, mutation; nt, nucleotide; PAM, protospacer-adjacent motif; PCR, polymerase chain reaction." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1162861369" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;CRISPR-Cas9 genome editing creates significant allelic distribution in the LATS1 founder mice. (A) Diagram of the CRISPR-Cas9-mediated DNA cleavage and subsequent HDR to form LATS1 T1079D or T1079A mutation (data not shown). (B) Venn diagram showing the frequency of HDR-associated events based on the sequencing data of plasmid-cloned PCR products. (C) Distribution of 304 sequence variants by allele type. For this analysis, the sequence from GRCm38 chromosome 10: 7,712,787 to 7,712,856 nt was considered. (D) Allelic composition of 20 founder mice based on sequencing of PCR-clones. *inside or outside the selected type sequence; **ins/del, del/mut, mut/ins, or del/mut/ins (inside or outside the target sequence). CRISPR-Cas9, Clustered Regularly Interspaced Short Palindromic Repeats-CRISPR-associated protein 9; del, deletion; HDR, homology-directed repair; ins, insertion; LATS1, Large Tumor Suppressor Kinase 1; mut, mutation; nt, nucleotide; PAM, protospacer-adjacent motif; PCR, polymerase chain reaction.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3533/F2.medium.gif" width="440" height="399"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/10/3533/F2.medium.gif" width="440" height="399"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486792" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-15" class="first-child">CRISPR-Cas9 genome editing creates significant allelic distribution in the LATS1 founder mice. (A) Diagram of the CRISPR-Cas9-mediated DNA cleavage and subsequent HDR to form LATS1 T1079D or T1079A mutation (data not shown). (B) Venn diagram showing the frequency of HDR-associated events based on the sequencing data of plasmid-cloned PCR products. (C) Distribution of 304 sequence variants by allele type. For this analysis, the sequence from GRCm38 chromosome 10: 7,712,787 to 7,712,856 nt was considered. (D) Allelic composition of 20 founder mice based on sequencing of PCR-clones. *inside or outside the selected type sequence; **ins/del, del/mut, mut/ins, or del/mut/ins (inside or outside the target sequence). CRISPR-Cas9, Clustered Regularly Interspaced Short Palindromic Repeats-CRISPR-associated protein 9; del, deletion; HDR, homology-directed repair; ins, insertion; LATS1, Large Tumor Suppressor Kinase 1; mut, mutation; nt, nucleotide; PAM, protospacer-adjacent motif; PCR, polymerase chain reaction.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">The hundreds of sequenced alleles provide insight into allelic complexity that can arise from <em>in vivo</em> CRISPR-Cas9 gene editing. Of 304 alleles analyzed, 21 contained the correct designed HDR (herein designated HDR), which is comprised of LATS1 (T1079A or T1079D), PAM, and <em>Eco</em>RI mutations. We also observed partial recombination occurring in 31 of 52 HDR-related sequences (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>), suggesting that incomplete HDR events can occur with high frequency and contribute to the genetic complexity induced by the CRISPR-Cas9 system. When examining HDR- and NHEJ-derived variants together, we observed numerous sequence types, which we categorized into unedited alleles (wt), LATS1 HDR mutation, deletions, point mutations, insertions, and mixed alleles (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). Importantly, we noticed significant genetic mosaicism, ranging from homozygous to five different alleles within a founder DNA sample (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2D</a>). The occurrence of editing was not binary, with eight animals having variable abundance of wt alleles remaining. This degree of complexity would not be observed in cultured diploid cells, and would be difficult to quantify using T7 or surveyor methods. This observation motivated the development of faster gene editing quantification methods that can be planned concomitantly with sgRNA and ssODN design.</p></div><div id="sec-11" class="subsection"><h3>Quantification of wt allele abundance in vivo after LATS1 CRISPR-Cas9 editing</h3><p id="p-17">Prior work identified priming interference in singleplex SYBR qPCR assays as a valid approach to detect NHEJ editing (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Yu <em>et al.</em> 2014</a>). Rather than using PCR primer mismatch for detection as in that study, we used TaqMan probe mismatch for assessing mutations. For normalization, we used multiplex assays to reduce the number of reactions and the influence of pipetting error. We next designed and tested two different strategies, here named <em>trans</em>-Multiplex Assay and Drop-Off Assay (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The difference between these assays lies in the normalization method. The <em>trans</em>-Multiplex Assay uses an External Reference Assay for the murine Transferrin Receptor (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>), while the Drop-Off Assay uses an Internal Reference Probe within the LATS1 amplicon yet distal to the mutation site (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). We used the same amplification primers for the LATS1 region in all assays (amplicon size 134 bp). The Target probes base paired with the CRISPR-Cas9 guide sequence inclusive of the cleavage site. For this experiment, we tested both normal oligonucleotides (Oligo Target Probe) and LNAs (LNA Target Probe). The LNA Target Probe contains three locked nucleotides, connecting the 2â oxygen and 4â carbon within the same nucleotide, which increases the discrimination of mismatch sequences through improved specificity of probing when compared to an oligo probe (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">You <em>et al.</em> 2006</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic representation of three qPCR strategies for wt allele quantification after CRISPR-Cas9 editing in LATS1 mice. (A) trans-Multiplex Assay combines LATS1 Oligo Target qPCR with an External Reference qPCR (TFRC). The target probe (orange) binds at the Cas9 cleavage site (scissors). (B) Drop-Off Assay with both reference (blue) and target (orange) probes within the same amplicon. The LNA Target Probe (top) contains locked nucleotides (underlined), which increase detection specificity. Both Oligo and LNA Target Probes bind to wt sequence at the Cas9 cleavage site (scissors). CRISPR-Cas9, Clustered Regularly Interspaced Short Palindromic Repeats-CRISPR-associated protein 9; Fw, forward; LATS1, Large Tumor Suppressor Kinase 1; LNA, locked nucleic acid; PAM, protospacer-adjacent motif; qPCR, quantitative polymerase chain reaction; Rv, reverse; TFRC, Transferrin Receptor; wt, wild-type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1162861369" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic representation of three qPCR strategies for wt allele quantification after CRISPR-Cas9 editing in LATS1 mice. (A) &lt;em&gt;trans&lt;/em&gt;-Multiplex Assay combines LATS1 Oligo Target qPCR with an External Reference qPCR (TFRC). The target probe (orange) binds at the Cas9 cleavage site (scissors). (B) Drop-Off Assay with both reference (blue) and target (orange) probes within the same amplicon. The LNA Target Probe (top) contains locked nucleotides (underlined), which increase detection specificity. Both Oligo and LNA Target Probes bind to wt sequence at the Cas9 cleavage site (scissors). CRISPR-Cas9, Clustered Regularly Interspaced Short Palindromic Repeats-CRISPR-associated protein 9; Fw, forward; LATS1, Large Tumor Suppressor Kinase 1; LNA, locked nucleic acid; PAM, protospacer-adjacent motif; qPCR, quantitative polymerase chain reaction; Rv, reverse; TFRC, Transferrin Receptor; wt, wild-type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3533/F3.medium.gif" width="440" height="321"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/10/3533/F3.medium.gif" width="440" height="321"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486794" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-18" class="first-child">Schematic representation of three qPCR strategies for wt allele quantification after CRISPR-Cas9 editing in LATS1 mice. (A) <em>trans</em>-Multiplex Assay combines LATS1 Oligo Target qPCR with an External Reference qPCR (TFRC). The target probe (orange) binds at the Cas9 cleavage site (scissors). (B) Drop-Off Assay with both reference (blue) and target (orange) probes within the same amplicon. The LNA Target Probe (top) contains locked nucleotides (underlined), which increase detection specificity. Both Oligo and LNA Target Probes bind to wt sequence at the Cas9 cleavage site (scissors). CRISPR-Cas9, Clustered Regularly Interspaced Short Palindromic Repeats-CRISPR-associated protein 9; Fw, forward; LATS1, Large Tumor Suppressor Kinase 1; LNA, locked nucleic acid; PAM, protospacer-adjacent motif; qPCR, quantitative polymerase chain reaction; Rv, reverse; TFRC, Transferrin Receptor; wt, wild-type.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">Next, we compared the extent to which the <em>trans</em>-Multiplex Assay, the Drop-Off Oligo Assay, or the Drop-off LNA Assays described the editing in the founder mouse DNA samples (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). For these qPCR analyses, we used the ââC<sub>q</sub> method (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ribot <em>et al.</em> 1998</a>) and expressed the data as a percentage of unedited events (wt). Each assay was optimized and calibrated (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300123/-/DC1/FigureS1.eps;">Figure S1</a>). The three qPCR assays detected unedited alleles in founder mice (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, AâC</a>), although we observed a high rate of false-positive unedited events with both <em>trans</em>-Multiplex (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>) and Drop-Off Oligo Assays (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>) relative to the Sanger sequencing data. The underestimation of editing was more prominent in those mice characterized by a low percentage (&lt; 8%) of wt as estimated by sequencing. To model the direct relationship between qPCR and sequencing data, linear regression analyses (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4, DâF</a>) showed that the Drop-Off LNA Assay (<em>r</em><sup>2</sup> = 0.968) produced the best correlation with Sanger sequencing data. The <em>trans</em>-Multiplex and Drop-Off Oligo Assaysâ underestimation of editing was reflected by the numerous data points located above the idealized slope (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4, D and E</a>). To further describe the association between qPCR results and sequencing data, we performed Pearson linear correlation and Spearman rank-order correlation analyses on the results obtained from each method. The Drop-Off LNA Assay (Pearson <em>r</em> = 0.984 and Spearman <em>r</em> = 0.905) showed the highest degree of correlation relative to the <em>trans</em>-Multiplex Assay (Pearson <em>r</em> = 0.926 and Spearman <em>r</em> = 0.842) and Drop-Off Oligo Assay (Pearson <em>r</em> = 0.914 and Spearman <em>r</em> = 0.875). Importantly, the Drop-Off LNA Assay showed significantly higher sensitivity to all mutations and better accuracy for measuring editing when compared to the other two strategies. Our data highlight the utility of LNA-based over unmodified oligonucleotide-based methods for detecting and quantifying sequence mismatch.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Drop-Off LNA Assay quantitatively detects CRISPR-Cas9 editing in LATS1 founder mice and shows higher sensitivity to editing on isolated target sequences. (A&#x2013;C) Comparison between the extent of unedited alleles (wt) detected by sequencing (open bars) and qPCR analyses (filled bars) in individual founder transgenic mice. (D&#x2013;F) Correlation of qPCR and sequencing data across founder animals using trans-Multiplex (A and D), Drop-Off Oligo (B and E), or Drop-Off LNA (C and F) Assays. The linear regression fit (solid line), ideal slope of 1.0 (dotted line), Pearson correlation coefficient, and Spearman correlation coefficient are shown for each assay. qPCR replicate data expressed as mean &#xB1; SEM (G) Amplification efficiency of Drop-Off Oligo and Drop-Off LNA qPCR Assays using templates that contain target sequence variants. Target (orange) and reference (blue) sequence regions are shown. CRISPR-Cas9, Clustered Regularly Interspaced Short Palindromic Repeats-CRISPR-associated protein 9; oligo, oligonucleotide; LATS1, Large Tumor Suppressor Kinase 1; LNA, locked nucleic acid; qPCR, quantitative polymerase chain reaction; wt, wild-type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1162861369" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Drop-Off LNA Assay quantitatively detects CRISPR-Cas9 editing in LATS1 founder mice and shows higher sensitivity to editing on isolated target sequences. (A&#x2013;C) Comparison between the extent of unedited alleles (wt) detected by sequencing (open bars) and qPCR analyses (filled bars) in individual founder transgenic mice. (D&#x2013;F) Correlation of qPCR and sequencing data across founder animals using &lt;em&gt;trans&lt;/em&gt;-Multiplex (A and D), Drop-Off Oligo (B and E), or Drop-Off LNA (C and F) Assays. The linear regression fit (solid line), ideal slope of 1.0 (dotted line), Pearson correlation coefficient, and Spearman correlation coefficient are shown for each assay. qPCR replicate data expressed as mean &#xB1; SEM (G) Amplification efficiency of Drop-Off Oligo and Drop-Off LNA qPCR Assays using templates that contain target sequence variants. Target (orange) and reference (blue) sequence regions are shown. CRISPR-Cas9, Clustered Regularly Interspaced Short Palindromic Repeats-CRISPR-associated protein 9; oligo, oligonucleotide; LATS1, Large Tumor Suppressor Kinase 1; LNA, locked nucleic acid; qPCR, quantitative polymerase chain reaction; wt, wild-type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3533/F4.medium.gif" width="440" height="397"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/10/3533/F4.medium.gif" width="440" height="397"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486812" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-20" class="first-child">Drop-Off LNA Assay quantitatively detects CRISPR-Cas9 editing in LATS1 founder mice and shows higher sensitivity to editing on isolated target sequences. (AâC) Comparison between the extent of unedited alleles (wt) detected by sequencing (open bars) and qPCR analyses (filled bars) in individual founder transgenic mice. (DâF) Correlation of qPCR and sequencing data across founder animals using <em>trans</em>-Multiplex (A and D), Drop-Off Oligo (B and E), or Drop-Off LNA (C and F) Assays. The linear regression fit (solid line), ideal slope of 1.0 (dotted line), Pearson correlation coefficient, and Spearman correlation coefficient are shown for each assay. qPCR replicate data expressed as mean Â± SEM (G) Amplification efficiency of Drop-Off Oligo and Drop-Off LNA qPCR Assays using templates that contain target sequence variants. Target (orange) and reference (blue) sequence regions are shown. CRISPR-Cas9, Clustered Regularly Interspaced Short Palindromic Repeats-CRISPR-associated protein 9; oligo, oligonucleotide; LATS1, Large Tumor Suppressor Kinase 1; LNA, locked nucleic acid; qPCR, quantitative polymerase chain reaction; wt, wild-type.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">To identify the differences in sensitivity between Oligo and LNA Target Probes, we tested the two Drop-Off Assays on representative isolated mutant target sequences. For this analysis, 14 target sequences from our Sanger sequencing data of the plasmid-clone PCR products were selected, including wt, PAM mutation, 1 nt insertion, 3 nt deletion, and &gt; 3 nt deletion sequences (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4G</a>). We performed both Drop-Off Oligo and LNA qPCR Assays on variant and wt templates and expressed the amplification efficiencies relative to that of the wt sequence. Both Drop-Off methods detected deletions &gt; 3 nt; however, the Drop-Off Oligo Assay moderately amplified both the PAM mutation and the 1 nt insertion sequences, indicating that this strategy is not sufficiently sensitive to detect all allelic variants and explaining why this assay underestimates editing in the founder mice.</p></div><div id="sec-12" class="subsection"><h3>Combination of drop-off strategy with dPCR method to quantify on-target editing</h3><p id="p-22">qPCR assays, in general, show the strongest response in the midrange of the assay; as such, we observed decreased precision at low (&lt; 15) and high (&gt; 85) percent editing. Other technologies have improved sensitivity near these limits, such as dPCR. For most dPCR approaches, thousands to hundreds of thousands of single-molecule PCR reactions are performed to accurately measure the number of PCR-positive events independent of amplification efficiency. Using a chip-based dPCR system, we performed the Drop-Off LNA Assay on the 20 LATS1 founder mice (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Example chip images (left) and scatterplot quantitation of probe results (right) showed clear differences in the extent of editing (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5, AâC</a>). Unedited samples were target and reference probe-positive (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>). Intermediate edited samples showed a mix of reference-positive and target-plus-reference-positive wells, and alleles with different amplification efficiency than wt (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>). Completely edited samples were only reference-positive (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C</a>). We found that the dPCR Drop-Off LNA Assay improved measurement precision of unedited and highly edited DNA (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5D</a>), yielding slightly stronger correlation with the Sanger sequencing data (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5E</a>) compared to the qPCR data.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="dPCR enhances the Drop-Off LNA Assay precision of detection of unedited alleles in founder mice. (A&#x2013;C) Representative chips and scatter plots showing the distribution of Reference (red) and LNA Target (blue) probe signals in LATS1 founder mice samples that contain: (A) unedited DNA, (B) partially edited DNA, or (C) completely edited DNA. (D) Comparison of unedited alleles detected by sequencing (open bars) and dPCR (filled bars) analyses. (E) Correlation of Drop-Off LNA dPCR and sequencing data across founder animals. The linear regression fit (solid line), ideal slope of 1.0 (dotted line), Pearson correlation coefficient, and Spearman correlation coefficient are shown for each assay. dPCR replicate data expressed as mean &#xB1; SEM. Amp, amplification; dPCR, digital polymerase chain reaction; LATS1, Large Tumor Suppressor Kinase 1; LNA, locked nucleic acid." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1162861369" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;dPCR enhances the Drop-Off LNA Assay precision of detection of unedited alleles in founder mice. (A&#x2013;C) Representative chips and scatter plots showing the distribution of Reference (red) and LNA Target (blue) probe signals in LATS1 founder mice samples that contain: (A) unedited DNA, (B) partially edited DNA, or (C) completely edited DNA. (D) Comparison of unedited alleles detected by sequencing (open bars) and dPCR (filled bars) analyses. (E) Correlation of Drop-Off LNA dPCR and sequencing data across founder animals. The linear regression fit (solid line), ideal slope of 1.0 (dotted line), Pearson correlation coefficient, and Spearman correlation coefficient are shown for each assay. dPCR replicate data expressed as mean &#xB1; SEM. Amp, amplification; dPCR, digital polymerase chain reaction; LATS1, Large Tumor Suppressor Kinase 1; LNA, locked nucleic acid.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3533/F5.medium.gif" width="264" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/10/3533/F5.medium.gif" width="264" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486863" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-23" class="first-child">dPCR enhances the Drop-Off LNA Assay precision of detection of unedited alleles in founder mice. (AâC) Representative chips and scatter plots showing the distribution of Reference (red) and LNA Target (blue) probe signals in LATS1 founder mice samples that contain: (A) unedited DNA, (B) partially edited DNA, or (C) completely edited DNA. (D) Comparison of unedited alleles detected by sequencing (open bars) and dPCR (filled bars) analyses. (E) Correlation of Drop-Off LNA dPCR and sequencing data across founder animals. The linear regression fit (solid line), ideal slope of 1.0 (dotted line), Pearson correlation coefficient, and Spearman correlation coefficient are shown for each assay. dPCR replicate data expressed as mean Â± SEM. Amp, amplification; dPCR, digital polymerase chain reaction; LATS1, Large Tumor Suppressor Kinase 1; LNA, locked nucleic acid.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Measuring CRISPR-Cas9 editing efficiency in blastocysts</h3><p id="p-24">To measure editing efficiency without generating mice, injected single-cell embryos can be cultured until blastocyst stage <em>ex vivo</em> (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Ran <em>et al.</em> 2013</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Wang <em>et al.</em> 2013</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Horii <em>et al.</em> 2014</a>). The process is similar to that of transgenic mouse production; in this case, sgRNA and Cas9 mRNA are injected into single-cell embryos that are matured to blastocysts in culture (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). Blastocysts are then harvested, DNA isolated, and the region of interest amplified for sequencing to detect editing. This procedure is effective for optimizing CRISPR-Cas9 reagents or selecting guide RNAs that will ultimately be used for transgenic mouse generation. Here, we tested whether a Drop-Off LNA Assay could be used for rapid qPCR detection of editing to substitute for sequencing. We designed a sgRNA guide that targeted the CREB3 Regulatory Factor (CREBRF) gene for NHEJ editing (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). In parallel, a Drop-Off LNA Assay was developed and validated (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300123/-/DC1/FigureS2.eps;">Figure S2A</a>) for detecting loss of wt amplification. We obtained 46 PCR products from 51 blastocysts, which were then subjected to direct Sanger sequencing and Drop-Off LNA qPCR (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6C</a>). The frequency of editing by Sanger sequencing and Drop-Off LNA qPCR was identical, extending the applicability of this method.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Drop-Off LNA qPCR Assay detects editing events in blastocysts. (A) Workflow for validating sgRNA guides in single-cell zygotes. (B) Schematic representation of the CREBRF Drop-Off LNA Assay and the CRISPR-Cas9 cleavage site (scissors). (C) Comparison of unedited events in ctr samples or sgRNA-injected blastocyst DNA (guide) as detected by sequencing and Drop-Off LNA qPCR analysis. CREBRF, CREB3 Regulatory Factor; CRISPR-Cas9, Clustered Regularly Interspaced Short Palindromic Repeats-CRISPR-associated protein 9; ctr, control DNA samples; Fw, forward; LNA, locked nucleic acid; qPCR, quantitative polymerase chain reaction; Rv, reverse; sgRNA, single guide RNA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1162861369" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Drop-Off LNA qPCR Assay detects editing events in blastocysts. (A) Workflow for validating sgRNA guides in single-cell zygotes. (B) Schematic representation of the CREBRF Drop-Off LNA Assay and the CRISPR-Cas9 cleavage site (scissors). (C) Comparison of unedited events in ctr samples or sgRNA-injected blastocyst DNA (guide) as detected by sequencing and Drop-Off LNA qPCR analysis. CREBRF, CREB3 Regulatory Factor; CRISPR-Cas9, Clustered Regularly Interspaced Short Palindromic Repeats-CRISPR-associated protein 9; ctr, control DNA samples; Fw, forward; LNA, locked nucleic acid; qPCR, quantitative polymerase chain reaction; Rv, reverse; sgRNA, single guide RNA.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3533/F6.medium.gif" width="440" height="286"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/10/3533/F6.medium.gif" width="440" height="286"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486913" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-25" class="first-child">Drop-Off LNA qPCR Assay detects editing events in blastocysts. (A) Workflow for validating sgRNA guides in single-cell zygotes. (B) Schematic representation of the CREBRF Drop-Off LNA Assay and the CRISPR-Cas9 cleavage site (scissors). (C) Comparison of unedited events in ctr samples or sgRNA-injected blastocyst DNA (guide) as detected by sequencing and Drop-Off LNA qPCR analysis. CREBRF, CREB3 Regulatory Factor; CRISPR-Cas9, Clustered Regularly Interspaced Short Palindromic Repeats-CRISPR-associated protein 9; ctr, control DNA samples; Fw, forward; LNA, locked nucleic acid; qPCR, quantitative polymerase chain reaction; Rv, reverse; sgRNA, single guide RNA.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Competitive qPCR with LNA probes to identify founder pups carrying recombinant alleles</h3><p id="p-26">The detection of recombinant alleles in founder mice is frequently built into the design of the donor sequence. A common approach includes silent mutations that add a restriction site within a diagnostic amplicon to generate a restriction fragment length polymorphism (RFLP), which can be used to demonstrate the presence of an allele after restriction and agarose gel electrophoresis. This PCR/RFLP approach is usually adequate for genotyping an established mouse line, where 0, 1, or 2 restriction-positive alleles can be easily distinguished. However, detection of low-abundance RFLP alleles in founder mice can prove to be challenging. To address recombinant allele detection in mosaic founder mice, we used an existing method for detecting single nucleotide polymorphisms: Competitive LNA PCR (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Mouritzen <em>et al.</em> 2003</a>). In contrast to the Drop-Off Assay, the fluorescent probes for WT or the CREBRF recombinant allele (R457Q) are designed for the same region and differentially labeled (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>). To detect the recombinant alleles, the terminal probe fluorescence is quantified for each sample and compared with appropriate controls in an allele discrimination plot (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>). To calibrate the assay, we used RFLP- and sequencing-validated wt, heterozygous, and homozygous offspring DNA samples and ensured linearity of dose response (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300123/-/DC1/FigureS2.eps;">Figure S2B</a>). When the original six founder DNA samples were analyzed, the correct founder was identified. These data demonstrate the ability of LNA-based probes to rapidly identify the subset of edited mice carrying targeted mutant alleles after CRISPR-Cas9 editing.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CREBRF genotyping assay using LNA Target Probes to detect the recombinant R457Q allele in founder mice. (A) Schematic representation of the allele competitive qPCR assay to detect CREBRF WT and R457Q sequences. The WT LNA and R457Q Target Probes (orange) contain locked nucleotides (underlined) at the PAM and R457Q mutation sites. (B) Allelic discrimination plot showing CREBRF founder mice (purple), and heterozygous (red), homozygous (green), and WT (blue) offspring controls that were validated by RFLP and sequencing analysis. CREBRF, CREB3 Regulatory Factor; HET, heterozygous; HOMO, homozygous; LNA, locked nucleic acid; PAM, protospacer-adjacent motif; qPCR, quantitative polymerase chain reaction; RFLP, restriction fragment length polymorphism; WT, wild-type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1162861369" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;CREBRF genotyping assay using LNA Target Probes to detect the recombinant R457Q allele in founder mice. (A) Schematic representation of the allele competitive qPCR assay to detect CREBRF WT and R457Q sequences. The WT LNA and R457Q Target Probes (orange) contain locked nucleotides (underlined) at the PAM and R457Q mutation sites. (B) Allelic discrimination plot showing CREBRF founder mice (purple), and heterozygous (red), homozygous (green), and WT (blue) offspring controls that were validated by RFLP and sequencing analysis. CREBRF, CREB3 Regulatory Factor; HET, heterozygous; HOMO, homozygous; LNA, locked nucleic acid; PAM, protospacer-adjacent motif; qPCR, quantitative polymerase chain reaction; RFLP, restriction fragment length polymorphism; WT, wild-type.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/10/3533/F7.medium.gif" width="440" height="345"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/10/3533/F7.medium.gif" width="440" height="345"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/10/3533/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486975" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-27" class="first-child">CREBRF genotyping assay using LNA Target Probes to detect the recombinant R457Q allele in founder mice. (A) Schematic representation of the allele competitive qPCR assay to detect CREBRF WT and R457Q sequences. The WT LNA and R457Q Target Probes (orange) contain locked nucleotides (underlined) at the PAM and R457Q mutation sites. (B) Allelic discrimination plot showing CREBRF founder mice (purple), and heterozygous (red), homozygous (green), and WT (blue) offspring controls that were validated by RFLP and sequencing analysis. CREBRF, CREB3 Regulatory Factor; HET, heterozygous; HOMO, homozygous; LNA, locked nucleic acid; PAM, protospacer-adjacent motif; qPCR, quantitative polymerase chain reaction; RFLP, restriction fragment length polymorphism; WT, wild-type.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-28">We observed that CRISPR-Cas9-injected RNAs can generate broad allelic diversity even in the context of a donor ssODN recombination template. Importantly, we detected a range of alleles, from a homozygous single edit to five different edited variants in individual mice. Simultaneously, we also identified animals with no detectable CRISPR-Cas9 editing events. Although sgRNA, Cas9 RNA, and ssODN templates are injected into single-cell embryos, our data suggest that the observed <em>in vivo</em> allelic diversity may arise from target restriction occurring in later multicellular stages of development. Consistent with this notion, some delay would be expected due to translation, folding, and assembly of the sgRNA-Cas9 complex prior to endonuclease activity. Therefore, unlike in clonal populations of cells that carry at most two different alleles, measuring editing extent in founder animals benefits from a sensitive, quantitative method.</p><p id="p-29">In this study, we introduced three important refinements in our PCR-based approach: the use of multiplex detection, the use of LNA probes, and dPCR. We did not observe significant differences in editing detection between the <em>trans</em>-Multiplex and Drop-Off Oligo approaches, which differ in the method of normalization, but we noted the convenience of an internal reference probe that is insensitive to differences in amplification efficiency between experimental and housekeeping targets. Detection of predicted or prescribed alleles by other methods, such as ligation-mediated or allele-specific PCR, may overlook editing events in founder lines that are potentially useful for gene disruption studies. Our data show that weak discrimination of some mutations can occur when standard oligonucleotides are used, but mutation detection is greatly improved with LNA probes. For dPCR, the limited quantitative differences between the qPCR and dPCR correlations with the PCR sequencing might suggest that the qPCR is sufficient for editing detection. However, because dPCR performs tens of thousands of reactions relative to the limited number of sequences per sample, we suggest that the correlation might be limited by the power of the sequencing data. For careful quantitation, or unambiguous exclusion of founders as unedited, we recommend dPCR.</p><p id="p-30">In the current study, the internally referenced LNA probe system described here can be used as a founder generation screening tool, and based on our isolated allele data, would work for rapid genotyping of both recombinant and indel alleles in wt, heterozygous, and homozygous animals. While <em>cis</em>-sequencing remains necessary to describe the precise sequence of individual alleles produced by CRISPR-Cas9 editing <em>in vivo</em>, elimination of unedited founders and detection of sub-Mendelian levels of mutation will enable better colony management. The need for sequencing is further underscored by our detection of incomplete conversion of ssODN into targeted alleles, suggesting that even RFLP analysis is inadequate as a first-pass confirmation of HDR success.</p><p id="p-31">In conclusion, our data demonstrate the high sensitivity of the Drop-Off LNA Assay for quantitative detection of CRISPR-Cas9-mediated gene editing. Based on the results presented herein, the Drop-Off Assay provides a rapid approach for initial identification and selection of high-fidelity HDR edits in founder mice.</p></div><div class="section" id="sec-16"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-32">We would like to thank Rachael A. Gordon for assistance with initial CRISPR guide selection, and Chunming Bi and the University of Pittsburgh School of Medicine Transgenic and Gene Targeting Core for microinjection of zygotes and production of mice used in this study. This work was supported by the Center for Metabolism and Mitochondrial Medicine and the Vascular Medicine Institute at the University of Pittsburgh. B.A.K. and M.F. were supported by National Institutes of Health (NIH) grant R01GM110424. A.Z.P. and E.A.G. were supported by NIH grant R01HL130261.</p><p id="p-33">Author contributions: M.F. and L.S. performed all experiments, interpreted data, generated figures, and wrote the manuscript. A.Z.P. maintained mouse lines. E.A.G. and E.E.K. initiated generation of the mice. J.B. designed primer probe sets and advised on methods. S.G. advised on CRISPR-Cas9 transgenesis and generated figures. B.A.K. conceived the overall idea, designed experiments, interpreted the data, and wrote the manuscript. J.B. is an employee at Integrated DNA Technologies and played no role in the interpretation of the data. No authors received a financial incentive for this work.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-34">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300123/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300123/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-35"><em>Communicating editor: B. Gregory</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-01-01"><span class="received-label">Received </span>January 1, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-08-28"><span class="accepted-label">Accepted </span>August 28, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2017 Falabella <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.1" data-doi="10.1016/j.ymeth.2014.02.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bassett</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">CRISPR/Cas9 mediated genome engineering in Drosophila.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">128</span>â<span class="cit-lpage">136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%26rft.volume%253D69%26rft.spage%253D128%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2014.02.019%26rft_id%253Dinfo%253Apmid%252F24576617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2014.02.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24576617&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.2" data-doi="10.1016/j.cell.2015.02.038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanjana</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shalem</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Genome-wide CRISPR screen in a mouse model of tumor growth and metastasis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">1246</span>â<span class="cit-lpage">1260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D160%26rft.spage%253D1246%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.02.038%26rft_id%253Dinfo%253Apmid%252F25748654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.02.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25748654&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.3" data-doi="10.1093/nar/16.22.10881"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corpet</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span>â<span class="cit-article-title">Multiple sequence alignment with hierarchical clustering.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">10881</span>â<span class="cit-lpage">10890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DCorpet%26rft.auinit1%253DF.%26rft.volume%253D16%26rft.issue%253D22%26rft.spage%253D10881%26rft.epage%253D10890%26rft.atitle%253DMultiple%2Bsequence%2Balignment%2Bwith%2Bhierarchical%2Bclustering%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F16.22.10881%26rft_id%253Dinfo%253Apmid%252F2849754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/16.22.10881&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2849754&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988R135700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.4" data-doi="10.1038/nm.3793"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">D. B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Therapeutic genome editing: prospects and challenges.</span> <abbr class="cit-jnl-abbrev">Nat. Med.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">121</span>â<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMed.%26rft.volume%253D21%26rft.spage%253D121%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm.3793%26rft_id%253Dinfo%253Apmid%252F25654603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm.3793&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25654603&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dehairs</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talebi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherifi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swinnen</span>  <span class="cit-name-given-names">J. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">CRISP-ID: decoding CRISPR mediated indels by Sanger sequencing.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">28973</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D28973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heidenreich</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Applications of CRISPR-Cas systems in neuroscience.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Neurosci.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">36</span>â<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BNeurosci.%26rft.volume%253D17%26rft.spage%253D36%26rft_id%253Dinfo%253Apmid%252F26656253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26656253&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.7" data-doi="10.1038/srep04513"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horii</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamazaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morita</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Validation of microinjection methods for generating knockout mice by CRISPR/Cas-mediated genome engineering.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">4513</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D4%26rft.spage%253D4513%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep04513%26rft_id%253Dinfo%253Apmid%252F24675426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep04513&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24675426&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.8" data-doi="10.1126/science.1225829"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jinek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chylinski</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fonfara</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doudna</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">816</span>â<span class="cit-lpage">821</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1225829%26rft_id%253Dinfo%253Apmid%252F22745249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzcvNjA5Ni84MTYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEwLzM1MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.9" data-doi="10.1038/s41598-016-0001-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kc</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkowski</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shavit</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Detection of nucleotide-specific CRISPR/Cas9 modified alleles using multiplex ligation detection.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-016-0001-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-016-0001-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.10" data-doi="10.1093/nar/gks1324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kolesar</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taguchi</span>  <span class="cit-name-given-names">Y. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chou</span>  <span class="cit-name-given-names">S.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaufman</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Two-dimensional intact mitochondrial DNA agarose electrophoresis reveals the structural complexity of the mammalian mitochondrial genome.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">e58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1324%26rft_id%253Dinfo%253Apmid%252F23275548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks1324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23275548&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.11" data-doi="10.1006/meth.2001.1262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Livak</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmittgen</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Analysis of relative gene expression data using real-time quantitative PCR and.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">402</span>â<span class="cit-lpage">408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DLivak%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D25%26rft.issue%253D4%26rft.spage%253D402%26rft.epage%253D408%26rft.atitle%253DAnalysis%2Bof%2Brelative%2Bgene%2Bexpression%2Bdata%2Busing%2Breal-time%2Bquantitative%2BPCR%2Band%2Bthe%2B2%2528-Delta%2BDelta%2BC%2528T%2529%2529%2BMethod.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fmeth.2001.1262%26rft_id%253Dinfo%253Apmid%252F11846609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/meth.2001.1262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11846609&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173949500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.12" data-doi="10.1038/nmeth.2649"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mali</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esvelt</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Cas9 as a versatile tool for engineering biology.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">957</span>â<span class="cit-lpage">963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D957%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2649%26rft_id%253Dinfo%253Apmid%252F24076990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2649&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24076990&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325073800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.13" data-doi="10.1586/14737159.3.1.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mouritzen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfundheller</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choleva</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kongsbak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Single nucleotide polymorphism genotyping using locked nucleic acid (LNA).</span> <abbr class="cit-jnl-abbrev">Expert Rev. Mol. Diagn.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">27</span>â<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExpert%2Breview%2Bof%2Bmolecular%2Bdiagnostics%26rft.stitle%253DExpert%2BRev%2BMol%2BDiagn%26rft.aulast%253DMouritzen%26rft.auinit1%253DP.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D38%26rft.atitle%253DSingle%2Bnucleotide%2Bpolymorphism%2Bgenotyping%2Busing%2Blocked%2Bnucleic%2Bacid%2B%2528LNA%2529.%26rft_id%253Dinfo%253Adoi%252F10.1586%252F14737159.3.1.27%26rft_id%253Dinfo%253Apmid%252F12528362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1586/14737159.3.1.27&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12528362&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188437500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-book" id="cit-7.10.3533.14" data-doi="10.1371/journal.pone.0129457"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McSwiggin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Pervasive genotypic mosaicism in founder mice derived from genome editing through pronuclear injection.</span> <span class="cit-source">PLoS One</span> <span class="cit-vol">10</span>: <span class="cit-fpage">e0129457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0129457%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0129457%26rft_id%253Dinfo%253Apmid%252F26053263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0129457&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26053263&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.15" data-doi="10.1016/j.immuni.2015.01.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pelletier</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gingras</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Mouse genome engineering via CRISPR-Cas9 for study of immune function.</span> <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">18</span>â<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D42%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.immuni.2015.01.004%26rft_id%253Dinfo%253Apmid%252F25607456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.immuni.2015.01.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25607456&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.16" data-doi="10.1016/j.cell.2013.08.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ran</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gootenberg</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konermann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Double nicking by RNA-guided CRISPR cas9 for enhanced genome editing specificity.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">1380</span>â<span class="cit-lpage">1389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D154%26rft.spage%253D1380%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.08.021%26rft_id%253Dinfo%253Apmid%252F23992846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.08.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23992846&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000324239300024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ribot</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quinn</span>  <span class="cit-name-given-names">F. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murtagh</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Comparative PCR: an improved method to detect gene amplification.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">22</span>â<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DBrass%26rft.auinit1%253DN.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D22%26rft.epage%253D4%252C%2B26%26rft.atitle%253DComparative%2BPCR%253A%2Ban%2Bimproved%2Bmethod%2Bto%2Bdetect%2Bgene%2Bamplification.%26rft_id%253Dinfo%253Apmid%252F9454944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9454944&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071414300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.18" data-doi="10.1016/j.molcel.2016.03.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruiz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayor-Ruiz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lafarga</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murga</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vega-Sendino</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">A genome-wide CRISPR screen identifies CDC25A as a determinant of sensitivity to ATR inhibitors.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">307</span>â<span class="cit-lpage">313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D62%26rft.spage%253D307%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2016.03.006%26rft_id%253Dinfo%253Apmid%252F27067599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2016.03.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27067599&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.19" data-doi="10.1016/j.molcel.2015.02.032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sternberg</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doudna</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Expanding the biologistâs toolkit with CRISPR-Cas9.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">568</span>â<span class="cit-lpage">574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D58%26rft.spage%253D568%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2015.02.032%26rft_id%253Dinfo%253Apmid%252F26000842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2015.02.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26000842&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.20" data-doi="10.1016/j.cell.2013.04.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shivalila</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawlaty</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">One-step generation of mice carrying mutations in multiple genes by CRISPR/Cas-mediated genome engineering.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">910</span>â<span class="cit-lpage">918</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D153%26rft.spage%253D910%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.04.025%26rft_id%253Dinfo%253Apmid%252F23643243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.04.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23643243&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318844000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.21" data-doi="10.1093/nar/gkv126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steentoft</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauge</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomsen</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Fast and sensitive detection of indels induced by precise gene targeting.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">e59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv126%26rft_id%253Dinfo%253Apmid%252F25753669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25753669&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.22" data-doi="10.1093/nar/gkl175"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreira</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Behlke</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Owczarzy</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Design of LNA probes that improve mismatch discrimination.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">e60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DYou%26rft.auinit1%253DY.%26rft.volume%253D34%26rft.issue%253D8%26rft.spage%253De60%26rft.epage%253De60%26rft.atitle%253DDesign%2Bof%2BLNA%2Bprobes%2Bthat%2Bimprove%2Bmismatch%2Bdiscrimination.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl175%26rft_id%253Dinfo%253Apmid%252F16670427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl175&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16670427&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-book" id="cit-7.10.3533.23" data-doi="10.1371/journal.pone.0098282"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">A PCR based protocol for detecting indel mutations induced by TALENs and CRISPR/Cas9 in zebrafish</span> <span class="cit-source">PLoS One</span> <span class="cit-vol">9</span>: <span class="cit-fpage">e98282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De98282%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0098282%26rft_id%253Dinfo%253Apmid%252F24901507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0098282&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24901507&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.24" data-doi="10.1038/nprot.2007.299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zentilin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giacca</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Competitive PCR for precise nucleic acid quantification.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">2092</span>â<span class="cit-lpage">2104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DZentilin%26rft.auinit1%253DL.%26rft.volume%253D2%26rft.issue%253D9%26rft.spage%253D2092%26rft.epage%253D2104%26rft.atitle%253DCompetitive%2BPCR%2Bfor%2Bprecise%2Bnucleic%2Bacid%2Bquantification.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2007.299%26rft_id%253Dinfo%253Apmid%252F17853864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2007.299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17853864&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F10%2F3533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.10.3533.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Efficient production of gene-modified mice using Staphylococcus aureus Cas9.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">32565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D32565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/10/3533.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/10/3521" title="Overexpression of miRNA-9 Generates Muscle Hypercontraction Through Translational Repression of Troponin-T in Drosophila melanogaster Indirect Flight Muscles" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/10/3543" title="Genomic Relatedness Strengthens Genetic Connectedness Across Management Units" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/10"><span>Volume 7 Issue 10, October 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/10.cover-source.jpg?itok=lurMO2QE" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/10/3533" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-gvqnklfZJ32eKvxxrRkfz13xbGd9Ni098QBcADgv_DI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/10/3533" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/10/3533" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/10/3533" class="active" data-icon-position="" data-hide-link-title="0">Single-Step qPCR and dPCR Detection of Diverse CRISPR-Cas9 Gene Editing Events in Vivo</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/486761" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-s-dbuJvpys4Iw5kO5fN_Uz8Sovp8FHVTvzCrSUNIUJY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/10/3533.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/10/3533" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Bv5SmourYlTb3ncLsMscLsUxh6cJvs56daltyWr6Sfc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/10/3533" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/10/3533.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486761" id="node486761--2" data-pisa="ggg;7/10/3533" data-pisa-master="ggg;g3.117.300123" data-apath="/ggg/7/10/3533.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Single-Step qPCR and dPCR Detection of Diverse CRISPR-Cas9 Gene Editing Events <em>in Vivo</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Micol Falabella</span>, <span class="highwire-citation-author" data-delta="1">Linqing Sun</span>, <span class="highwire-citation-author" data-delta="2">Justin Barr</span>, <span class="highwire-citation-author" data-delta="3">Andressa Z. Pena</span>, <span class="highwire-citation-author" data-delta="4">Erin E. Kershaw</span>, <span class="highwire-citation-author" data-delta="5">Sebastien Gingras</span>, <span class="highwire-citation-author" data-delta="6">Elena A. Goncharova</span> and <span class="highwire-citation-author" data-delta="7">Brett A. Kaufman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3533-3542;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300123 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486761--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Micol Falabella </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Pittsburgh School of Medicine, Department of Medicine, Division of Cardiology, Center for Metabolism and Mitochondrial Medicine and Vascular Medicine Institute, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Micol%2BFalabella%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Falabella%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMicol%2BFalabella%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Linqing Sun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Pittsburgh School of Medicine, Department of Medicine, Division of Cardiology, Center for Metabolism and Mitochondrial Medicine and Vascular Medicine Institute, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linqing%2BSun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sun%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinqing%2BSun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Justin Barr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Integrated DNA Technologies, Coralville, Iowa 52241</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BBarr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barr%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BBarr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Andressa Z. Pena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>University of Pittsburgh School of Medicine, Department of Medicine, Vascular Medicine Institute, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andressa%2BZ.%2BPena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pena%20AZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndressa%2BZ.%2BPena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Erin E. Kershaw </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>University of Pittsburgh School of Medicine, Department of Medicine, Division of Endocrinology, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erin%2BE.%2BKershaw%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kershaw%20EE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErin%2BE.%2BKershaw%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sebastien Gingras </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>University of Pittsburgh School of Medicine, Department of Immunology, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastien%2BGingras%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gingras%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastien%2BGingras%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Elena A. Goncharova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â â </span>University of Pittsburgh School of Medicine, Department of Medicine, Division of Pulmonary, Allergy and Critical Care Medicine, Vascular Medicine Institute, and Department of Bioengineering, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elena%2BA.%2BGoncharova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goncharova%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElena%2BA.%2BGoncharova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Brett A. Kaufman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Pittsburgh School of Medicine, Department of Medicine, Division of Cardiology, Center for Metabolism and Mitochondrial Medicine and Vascular Medicine Institute, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brett%2BA.%2BKaufman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaufman%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrett%2BA.%2BKaufman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bkauf@pitt.edu" class="" data-icon-position="" data-hide-link-title="0">bkauf@pitt.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/10/3533" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/10/3533&amp;title=Single-Step%20qPCR%20and%20dPCR%20Detection%20of%20Diverse%20CRISPR-Cas9%20Gene%20Editing%20Events%20in%20Vivo" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/10/3533&amp;title=Single-Step%20qPCR%20and%20dPCR%20Detection%20of%20Diverse%20CRISPR-Cas9%20Gene%20Editing%20Events%20in%20Vivo" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/10/3533&amp;title=Single-Step%20qPCR%20and%20dPCR%20Detection%20of%20Diverse%20CRISPR-Cas9%20Gene%20Editing%20Events%20in%20Vivo" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/10/3533&amp;text=Single-Step%20qPCR%20and%20dPCR%20Detection%20of%20Diverse%20CRISPR-Cas9%20Gene%20Editing%20Events%20in%20Vivo" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/10/3533&amp;title=Single-Step%20qPCR%20and%20dPCR%20Detection%20of%20Diverse%20CRISPR-Cas9%20Gene%20Editing%20Events%20in%20Vivo" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/10/3533&amp;t=Single-Step%20qPCR%20and%20dPCR%20Detection%20of%20Diverse%20CRISPR-Cas9%20Gene%20Editing%20Events%20in%20Vivo" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/10/3533&amp;title=Single-Step%20qPCR%20and%20dPCR%20Detection%20of%20Diverse%20CRISPR-Cas9%20Gene%20Editing%20Events%20in%20Vivo" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/10/3533&amp;title=Single-Step%20qPCR%20and%20dPCR%20Detection%20of%20Diverse%20CRISPR-Cas9%20Gene%20Editing%20Events%20in%20Vivo" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/486761/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486761" id="node486761--3" data-pisa="ggg;7/10/3533" data-pisa-master="ggg;g3.117.300123" data-apath="/ggg/7/10/3533.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Single-Step qPCR and dPCR Detection of Diverse CRISPR-Cas9 Gene Editing Events <em>in Vivo</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Micol Falabella</span>, <span class="highwire-citation-author" data-delta="1">Linqing Sun</span>, <span class="highwire-citation-author" data-delta="2">Justin Barr</span>, <span class="highwire-citation-author" data-delta="3">Andressa Z. Pena</span>, <span class="highwire-citation-author" data-delta="4">Erin E. Kershaw</span>, <span class="highwire-citation-author" data-delta="5">Sebastien Gingras</span>, <span class="highwire-citation-author" data-delta="6">Elena A. Goncharova</span> and <span class="highwire-citation-author" data-delta="7">Brett A. Kaufman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3533-3542;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300123 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486761--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Micol Falabella </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Pittsburgh School of Medicine, Department of Medicine, Division of Cardiology, Center for Metabolism and Mitochondrial Medicine and Vascular Medicine Institute, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Micol%2BFalabella%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Falabella%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMicol%2BFalabella%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Linqing Sun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Pittsburgh School of Medicine, Department of Medicine, Division of Cardiology, Center for Metabolism and Mitochondrial Medicine and Vascular Medicine Institute, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linqing%2BSun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sun%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinqing%2BSun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Justin Barr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Integrated DNA Technologies, Coralville, Iowa 52241</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BBarr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barr%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BBarr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Andressa Z. Pena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>University of Pittsburgh School of Medicine, Department of Medicine, Vascular Medicine Institute, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andressa%2BZ.%2BPena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pena%20AZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndressa%2BZ.%2BPena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Erin E. Kershaw </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>University of Pittsburgh School of Medicine, Department of Medicine, Division of Endocrinology, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erin%2BE.%2BKershaw%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kershaw%20EE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErin%2BE.%2BKershaw%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sebastien Gingras </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>University of Pittsburgh School of Medicine, Department of Immunology, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastien%2BGingras%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gingras%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastien%2BGingras%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Elena A. Goncharova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â â </span>University of Pittsburgh School of Medicine, Department of Medicine, Division of Pulmonary, Allergy and Critical Care Medicine, Vascular Medicine Institute, and Department of Bioengineering, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elena%2BA.%2BGoncharova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goncharova%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElena%2BA.%2BGoncharova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Brett A. Kaufman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Pittsburgh School of Medicine, Department of Medicine, Division of Cardiology, Center for Metabolism and Mitochondrial Medicine and Vascular Medicine Institute, Pennsylvania 15261</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brett%2BA.%2BKaufman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaufman%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrett%2BA.%2BKaufman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bkauf@pitt.edu" class="" data-icon-position="" data-hide-link-title="0">bkauf@pitt.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3533&amp;rft.epage=3542&amp;rft.atitle=Single-Step%20qPCR%20and%20dPCR%20Detection%20of%20Diverse%20CRISPR-Cas9%20Gene%20Editing%20Events%20in%20Vivo&amp;rft.volume=7&amp;rft.issue=10&amp;rft.date=2017-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Falabella%2C+Micol&amp;rft.au=Sun%2C+Linqing&amp;rft.au=Barr%2C+Justin&amp;rft.au=Pena%2C+Andressa+Z.&amp;rft.au=Kershaw%2C+Erin+E.&amp;rft.au=Gingras%2C+Sebastien&amp;rft.au=Goncharova%2C+Elena+A.&amp;rft.au=Kaufman%2C+Brett+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/486761/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/486761/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/486761/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/486761/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/486761/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/486761/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/486761/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/486761/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/486761/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/486761/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/486761/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/486761/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/10/3533" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/10/3533.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/10/3533.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/10/3533.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/10/3533.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
