<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/1/57" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/1/57.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/1/57.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/1/57.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1.cover.gif" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/1/1/57/embed/mml-math-8.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/1/57.atom" />
<meta name="HW.pisa" content="ggg;1/1/57" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="An Improved Method for Estimating Chromosomal Line Origin in QTL Analysis of Crosses Between Outbred Lines" />
<meta name="DC.Identifier" content="10.1534/g3.111.000109" />
<meta name="DC.Date" content="2011-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2011 Crooks et al.. This is an open access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Estimating the line origin of chromosomal sections from marker genotypes is a vital step in quantitative trait loci analyses of outbred line crosses. The original, and most commonly used, algorithm can only handle moderate numbers of partially informative markers. The advent of high-density genotyping with SNP chips motivates a new method because the generic sets of markers on SNP chips typically result in long stretches of partially informative markers. We validated a new method for inferring line origin, triM (tracing inheritance with Markov models), with simulated data. A realistic pattern of marker information was achieved by replicating the linkage disequilibrium from an existing chicken intercross. There were approximately 1500 SNP markers and 800 F2 individuals. The performance of triM was compared to GridQTL, which uses a variant of the original algorithm but modified for larger datasets. triM estimated the line origin with an average error of 2%, was 10% more accurate than GridQTL, considerably faster, and better at inferring positions of recombination. GridQTL could not analyze all simulated replicates and did not estimate line origin for around a third of individuals at many positions. The study shows that triM has computational benefits and improved estimation over available algorithms and is valuable for analyzing the large datasets that will be standard in future." />
<meta name="DC.Contributor" content="Lucy Crooks" />
<meta name="DC.Contributor" content="Carl Nettelblad" />
<meta name="DC.Contributor" content="Örjan Carlborg" />
<meta name="article:published_time" content="2011-06-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="An Improved Method for Estimating Chromosomal Line Origin in QTL Analysis of Crosses Between Outbred Lines" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Estimating the line origin of chromosomal sections from marker genotypes is a vital step in quantitative trait loci analyses of outbred line crosses. The original, and most commonly used, algorithm can only handle moderate numbers of partially informative markers. The advent of high-density genotyping with SNP chips motivates a new method because the generic sets of markers on SNP chips typically result in long stretches of partially informative markers. We validated a new method for inferring line origin, triM (tracing inheritance with Markov models), with simulated data. A realistic pattern of marker information was achieved by replicating the linkage disequilibrium from an existing chicken intercross. There were approximately 1500 SNP markers and 800 F&lt;sub&gt;2&lt;/sub&gt; individuals. The performance of triM was compared to GridQTL, which uses a variant of the original algorithm but modified for larger datasets. triM estimated the line origin with an average error of 2%, was 10% more accurate than GridQTL, considerably faster, and better at inferring positions of recombination. GridQTL could not analyze all simulated replicates and did not estimate line origin for around a third of individuals at many positions. The study shows that triM has computational benefits and improved estimation over available algorithms and is valuable for analyzing the large datasets that will be standard in future.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/06/01" />
<meta name="citation_mjid" content="ggg;1/1/57" />
<meta name="citation_id" content="1/1/57" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/1/57" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/1/57.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/1/57.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/1/57.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.000109" />
<meta name="citation_pmid" content="22384318" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="1" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="57" />
<meta name="citation_lastpage" content="64" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Lucy Crooks" />
<meta name="citation_author_institution" content="Department of Cell and Molecular Biology, Uppsala University, SE-751 24 Uppsala, Sweden" />
<meta name="citation_author_email" content="lucy.crooks@icm.uu.se" />
<meta name="citation_author" content="Carl Nettelblad" />
<meta name="citation_author_institution" content="Department of Information Technology, Uppsala University, SE-751 05 Uppsala, Sweden" />
<meta name="citation_author" content="Örjan Carlborg" />
<meta name="citation_author_institution" content="Department of Cell and Molecular Biology, Uppsala University, SE-751 24 Uppsala, Sweden" />
<meta name="citation_author_institution" content="Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=S. Sen,;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=P. M. Corva;citation_author=J. F. Medrano;citation_title=Quantitative trait loci (QTLs) mapping for growth traits in the mouse: a review.;citation_pages=105-132;citation_volume=33;citation_year=2001;citation_issue=2;citation_pmid=11333830;citation_doi=10.1051/gse:2001112" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Frary;citation_author=T. C. Nesbitt;citation_author=A. Frary;citation_author=S. Grandillo;citation_author=E. Van Der Knaap;citation_title=fw2.2: A Quantitative Trait Locus Key to the Evolution of Tomato Fruit Size;citation_pages=85-88;citation_volume=289;citation_year=2000;citation_issue=5476;citation_pmid=10884229;citation_doi=10.1126/science.289.5476.85" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. A. M. Groenen;citation_author=P. Wahlberg;citation_author=M. Foglio;citation_author=H. H. Cheng;citation_author=H.-J. Megens;citation_title=A high-density SNP-based linkage map of the chicken genome reveals sequence features correlated with recombination rate.;citation_volume=19;citation_year=2009;citation_pmid=19088305;citation_doi=10.1101/gr.086538.108" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. S. Haley;citation_author=S. A. Knott,;citation_author=J.-M. Elsen;citation_title=Mapping quantitative trait loci in crosses between outbred lines using least squares.;citation_pages=1195-1207;citation_volume=136;citation_year=1994;citation_pmid=8005424" />
<meta name="citation_reference" content="citation_journal_title=Worlds Poult. Sci. J.;citation_author=P. M. Hocking;citation_title=Review of QTL mapping results in chickens.;citation_pages=215-226;citation_volume=61;citation_year=2005;citation_doi=10.1079/WPS200461" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=S. Kerje;citation_author=Ö. Carlborg;citation_author=L. Jacobsson;citation_author=K. Schütz;citation_author=C. Hartmann;citation_title=The twofold difference in adult size between the red junglefowl and White Leghorn chickens is largely explained by a limited number of QTLs.;citation_pages=264-274;citation_volume=34;citation_year=2003;citation_issue=4;citation_pmid=12873214;citation_doi=10.1046/j.1365-2052.2003.01000.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. S. Lander;citation_author=D. Botstein;citation_title=Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=E. S. Lander;citation_author=P. Green;citation_title=Construction of multilocus genetic linkage maps in humans.;citation_pages=2363-2367;citation_volume=84;citation_year=1987;citation_issue=8;citation_pmid=3470801;citation_doi=10.1073/pnas.84.8.2363" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. D. Long;citation_author=R. F. Lyman;citation_author=A. H. Morgan;citation_author=C. H. Langley,;citation_author=T. F. C. Mackay;citation_title=Both Naturally Occurring Insertions of Transposable Elements and Intermediate Frequency Polymorphisms at the achaete-scute Complex Are Associated With Variation in Bristle Number in Drosophila melanogaster;citation_pages=1255-1269;citation_volume=154;citation_year=2000;citation_issue=3;citation_pmid=10757767" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. F. C. Mackay;citation_title=Quantitative trait loci in Drosophila.;citation_pages=11-20;citation_volume=2;citation_year=2001;citation_issue=1;citation_pmid=11253063" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=J. N. Maloof;citation_title=QTL for plant growth and morphology.;citation_pages=85-90;citation_volume=6;citation_year=2003;citation_issue=1;citation_pmid=12495756;citation_doi=10.1016/S1369526602000080" />
<meta name="citation_reference" content="citation_journal_title=Proc. IEEE;citation_author=L. R. Rabiner;citation_title=A tutorial on hidden Markov models and selected applications in speech recognition.;citation_pages=257-286;citation_volume=77;citation_year=1989;citation_doi=10.1109/5.18626" />
<meta name="citation_reference" content="citation_journal_abbrev=Int J Biol Sci;citation_author=M. F. Rothschild;citation_author=Z.-L. Hu,;citation_author=Z. Jiang;citation_title=Advances in QTL mapping in pigs.;citation_pages=192-197;citation_volume=3;citation_year=2007;citation_issue=3;citation_pmid=17384738" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. Rönnegård;citation_author=F. Besnier,;citation_author=Ö. Carlborg;citation_title=An Improved Method for Quantitative Trait Loci Detection and Identification of Within-Line Segregation in F2 Intercross Designs;citation_pages=2315-2326;citation_volume=178;citation_year=2008;citation_issue=4;citation_pmid=18430952;citation_doi=10.1534/genetics.107.083162" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A.-S. Van Laere;citation_author=M. Nguyen;citation_author=M. Braunschwieg;citation_author=C. Nezer;citation_author=C. Collette;citation_title=A regulatory mutation in IGF2 causes a major QTL effect on muscle growth in the pig.;citation_pages=832-836;citation_volume=425;citation_year=2003;citation_issue=6960;citation_pmid=14574411;citation_doi=10.1038/nature02064" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="An Improved Method for Estimating Chromosomal Line Origin in QTL Analysis of Crosses Between Outbred Lines" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1.cover.gif" />
<meta name="twitter:description" content="Estimating the line origin of chromosomal sections from marker genotypes is a vital step in quantitative trait loci analyses of outbred line crosses. The original, and most commonly used, algorithm can only handle moderate numbers of partially informative markers. The advent of high-density genotyping with SNP chips motivates a new method because the generic sets of markers on SNP chips typically result in long stretches of partially informative markers. We validated a new method for inferring line origin, triM (tracing inheritance with Markov models), with simulated data. A realistic pattern of marker information was achieved by replicating the linkage disequilibrium from an existing chicken intercross. There were approximately 1500 SNP markers and 800 F2 individuals. The performance of triM was compared to GridQTL, which uses a variant of the original algorithm but modified for larger datasets. triM estimated the line origin with an average error of 2%, was 10% more accurate than GridQTL, considerably faster, and better at inferring positions of recombination. GridQTL could not analyze all simulated replicates and did not estimate line origin for around a third of individuals at many positions. The study shows that triM has computational benefits and improved estimation over available algorithms and is valuable for analyzing the large datasets that will be standard in future." />
<meta name="og-title" property="og:title" content="An Improved Method for Estimating Chromosomal Line Origin in QTL Analysis of Crosses Between Outbred Lines" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/1/57" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Estimating the line origin of chromosomal sections from marker genotypes is a vital step in quantitative trait loci analyses of outbred line crosses. The original, and most commonly used, algorithm can only handle moderate numbers of partially informative markers. The advent of high-density genotyping with SNP chips motivates a new method because the generic sets of markers on SNP chips typically result in long stretches of partially informative markers. We validated a new method for inferring line origin, triM (tracing inheritance with Markov models), with simulated data. A realistic pattern of marker information was achieved by replicating the linkage disequilibrium from an existing chicken intercross. There were approximately 1500 SNP markers and 800 F2 individuals. The performance of triM was compared to GridQTL, which uses a variant of the original algorithm but modified for larger datasets. triM estimated the line origin with an average error of 2%, was 10% more accurate than GridQTL, considerably faster, and better at inferring positions of recombination. GridQTL could not analyze all simulated replicates and did not estimate line origin for around a third of individuals at many positions. The study shows that triM has computational benefits and improved estimation over available algorithms and is valuable for analyzing the large datasets that will be standard in future." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1.cover.gif" />
    <title>An Improved Method for Estimating Chromosomal Line Origin in QTL Analysis of Crosses Between Outbred Lines | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"z5oQOV-d68Xm2Y_zdjKUPP3LfLeLhwhCWfIWz_x7SK0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459157","apath":"\/ggg\/1\/1\/57.atom","pisa":"ggg;1\/1\/57","ac":{"\/ggg\/1\/1\/57.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/1\/57.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/1\/57"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/1\/57"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/1\/57.figures-only":true,"\/content\/1\/1\/57":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MMR5i1YxKy2UaZBP9GzrJsI45Djn4TAJ3iJxCcjfs0N\\\\\\\/aPbvmp6f0875RRckJWP486vAA7tUqMWDzYHMeceCbrWbNHAOXD++ZyLnHE9iZTSoS8BRkzLEhR\\\\\\\/SJIJ\\\\\\\/8ut4qK6I5hUqx3JgwWDlDhc9qBmVnsVJNOkNBXo75irAa5dr89M8S+2bGaw3gvP+pKqz01KrRDrro0IczraPkOhub7mbNP\\\\\\\/mBZmiiocLS2dSu4ozaXXzpzyuDalOinwwTIXMJlGVq2oAHTtLyHCT3J1UURxToO21x7Uu3KXdawoOFFk2VtiSCuFjMROpLlIsaZvgm8jFCg06OStT6X01Rc4X\\\\\\\/MNbmV7dD+9NXsBVolZNMrkH1De9G+mfuk5vW0YggIxhVk\\\\\\\/H5A63WAvSMO\\\\\\\/3wpnWq\\\\\\\/nbytXZ37v8ZpqtNxRDHg8mYjynvZfPLReG+Liy5aQ0A1Z0uFMN\\\\\\\/mpieOPKC1accKMeROpexR8WkSGNOhAvp6J9wBCWhblVi6CaqxRrBa7rZCbVrHSdldp\\\\\\\/kEFN0SxUpJrAFhxx3OW3X1vhtsu6IEQQAL546C5+SbTgM2ARIE7raOwH54NIPblZPQ50OaAuu6F61e86N+HPFS3wOPExM\\\\\\\/CVbLZzCw0d86k\\\\\\\/CflB0AOQRIFX36H\\\\\\\/blpRemBkfHzTTQUc1dmgWdNyvnpZQvwDxGsUVaq4eIJf45rJFvIHcDtv2S961xaUbRqLCeBZRuXYx3dweTo+ABYxUhm\\\\\\\/EER7G1RIvf0sMI8WtdTLAKe13b9b6qeHGwwTZD1Gelq3hj5D6IOf7ofAZM9r9Wjz1NuHkVkYx1+cuTSjMg+D+2c0J+9XjJ7wKy03nCMGQOj044hjB5HBQ9LwXUdnkAyLn3RX9aME4ayOOvHUNRYoJaRbRZ9l2EqN3O\\\\\\\/dBXfNne1AQUA3OOzW75JlRpwl+d56yUS7Pb7gcrCOz4gktdxYXn3plYzCHmQBcYaSaARMUXu5nj\\\\\\\/gTzztk4FlqdSwp8MhqJOlGe2iM+GJ0yMQvEgqIsU+VYgWNalrd83QT2W3zjUiTNHCEdSPbDCM2Oi+wSWYj9W7p3Jo0tnoK3RMc6nu2UO1qV0oigkx\\\\\\\/cGuXRppwfbdA5dgz7cPG0vZnQTiT25DdCAvCvjdG8NZ0bcK4W\\\\\\\/GDqtkGFQpKC3WiQ15n2wgsTbYWo6NdcsGBHkRF4cEnga5nyLoON9P2eccQD8EVEOV6JXDeZMYKX+zs+KcXUvBLQ\\\\\\\/gE7PByBN8VPNkjLDAryKhH\\\\\\\/Yxmai0PBQOwHxWQFbQ+u9T7mUB\\\u0022,\\\u0022iv\\\u0022:\\\u0022bG4EyBTvIjxO6xGGhQIFZw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c2a1a4f2b66cb4d5112aab7d0132122bc85bdf443ae9b13df011d6fde1d45354\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Bm\\\\\\\/LaxfrepXB0ztB6yW6JrIeF04X+708mi5jSDXt3RMUH5MH3slWTaQvDxTQFHPsQWHywtJeeEZx\\\\\\\/xML8gdLmg1\\\\\\\/mJ2FNBrNzIdwNmU1\\\\\\\/9zlGhQMK+WpzoSqj6idohExQI9Ah6Brpc0+0iTwJyD5zzkkrHPz12ucOWrO2fxKT+SvC5Exm+i6Aochf4jNfZNGFbsq59WrbnVUQ5NWTQhbSUuZIti+kekYK8Dg6t7pYeQfDa5xdOzl8O2iw6CieAgHHNcqny1wABqnpBxmcpWRTYmbXWrdj8bddnYe7g7LkhzJ\\\\\\\/qsl+ZVJJOJEA9JWq0OiYlYHdUaLIQafps\\\\\\\/6RO5g7InGFbfhaRUogyE56l3BBbddZIdgXFxiuG99Q737gA6t1wQTX+8RQR9TrwzO2Jxk2Z69s1XVor73SmsjOPR++kBAoQTWk2+gXTPiy88CVd8FKr7zoCs2ggVjQqf13K9NoP\\\\\\\/82Lvmdok5GhxqRzFpfm8Bb4Nzt0NxtAR9ekJjTmxb+r6jTZPMR44Q2bOM5LFRwd4L2B1Z0x7csrbQmf7i0Xet5XIXlc+Pg128vmcYM7BTJJhLifrA\\\\\\\/LdnX3+scqgrfpzJeTfc9b4qkKwxk\\\\\\\/rVxdjVs\\\\\\\/GuIICTJfVKFy7IbitzgcCxzW4zMYI4SBtkRxMvHl4yhMFr+a5WPdv8u4Zs5i4d9pGrhcG\\\\\\\/1+w4t2sBAksnDo8EFXtoJ+tm\\\\\\\/hlTehQ80WBoe0lAxNJalSuUDUMJ0Dt7RMCsfcHQkhcFJEUFC79sDn+qLcXUzXtEcsngA7uZYe7NRbEYeD1YH+\\\\\\\/+tnaXYkHReMC3gAzBbfl8FXDvatJHtIVibw3MTllvq\\\\\\\/4fHr8F2QdPcny7WpwA3r1aN9M9QBeoGnR5j9cIpP0OwpNfJS5\\\\\\\/o6HH2MY2uVH1EUoMzLARNva5D8Su0MqcqeKH697npFvXKs84V\\\\\\\/YSdSPVswQick+XOJ1aeI++isoh\\\\\\\/A52JGjXOZxtxB3ev9NyrA+MdzYIVrGcMnEOIiG+2R6uqawRvkRCqEuYnob09OlV4VkfDXPRdBtisgybt6YS0DlZQnCaqlLV2dD3wXlhhakq1tFzsjHNCh5v1CQqTSezGKaNaW5GlUKdOMbSp\\\\\\\/Mr0Pc98zaO1wr4w8Az1MXPE5zsrdQAN\\\\\\\/a4RkHSw4HYaSSLbvjx5of4yMuBHXudBH5oC4PPa0vCSrZ35XTdYrMhtLEWx9a8AzuSydUr46gH4jodXp7qDjkq38qzSaRlpj1rmotbWmVqlbBaqM2ODrK5w3BPSe44cTwtn4LoliJvU40EvuF7Ucmq3BnhcJMEYGDl615H6txz98QXeybibBbGAG91TwRRYz0AzVQnpZxfAZ7p\\\\\\\/xu\\\\\\\/uDLsYz89CyW2bKnAk\\\\\\\/IAukBL73Y27u5IHpMWMl\\\\\\\/el3S7QolxTlXSf1Ld80vB7j13PC90eEifR3sQzT0MAxGOrsCroxu7uhxQQzY1YNSqC++785efZHDKeSsELYn13bi+CmyGX5KsWo0w2QrrF+VFPlIJYsUf7biAfK2Mi7DDxVvpj2QsqyUFmMvTUPzrZtVggsgzyRBBe0nsAdVM0xNMIDD6dcYzv35eOsIsW8Ak9JZ7KEVV+Zli1oOMRRpc8r2C6GBnfJkj+RBeCq8QUh+0jW9F+3a9kdBntABv0efNLblXVbD\\\\\\\/BnbeFVoQojw+pRGME\\\\\\\/LPJmcOtvqPXyUvTgNMawXDZkeHJZwy3U24cATn8vKCaQZgoO5kMdzfF9SP2pke5lgSTxMsp5PYRvMY3z8voYnX2yYhyR8vSs0y\\\\\\\/41\\\\\\\/zl\\\\\\\/3YX7pt1vjr1LwtVDKyiSWsHNodW6tkdiusYorewspBl1q8\\\\\\\/pP97GIzgrM+0rgvdT6Ni3yl6K7WBxy0OOgF4whJDYncdTxR8ckUD72OTszbkyY8GrcbT2zo7i6cpqTC9XXT7I5tcxFlXPkAU\\\\\\\/Fw+4mbzfClR0uqLHa6AEXXTBdn1FsrZzDXExLwscOsaau4nWPFx+u6xqwAoXdxbxukdn4NCG6Jxw9r4b60Hn1DbcIRDn7q7VFoza\\\\\\\/f1\\\\\\\/6y1IEXcvl9NgmB87h5qJSn9aslZJjX+naYnSrMIn6uvvNc9Gx83Sw4gCd2mratV2vAlLm0SlYS9H+sfRuf\\\\\\\/d948k\\\\\\\/edV\\\\\\\/vjVbgv8hmSOP3tRiPsMs+Bdx1qMc7TnEMflu1JermZtsV01eYrAcZW+u3nXyBakzjWlNbajjNt99ZAr+hNXDdtGmpaBm7qGYIiYQUrFMdqXGYJdlM4QB9fXL26ZJjMNJPopcDK5t+Hgf6i1H8JIG4edzkyOCynXO40Hb1ecSwlNkWUkvHir9vN7bMGL3CAaUxV+So1bzR8Yqf53hBBeqH3I\\\\\\\/0cRsFZW7Pgf6krkbVhpXAkZGAAMxWObzpZCXIYhoz7HiQWUVTXkuZHgPkqefp5AdUR\\\\\\\/uciPpHL0yjzSwE+9GddibXo76oX70APeb7UWnbf+n2l486woqCrpl4Vk0dO5g\\\\\\\/eGWg+SbKIeAlmaROoyLF0lV+I87goer2g0iikXfJ9qaYF\\\\\\\/NcHyigideZ7IUsys2bCZkM8EZz67KlD9tcPhfRSh5YDRiXhrJipq2WkNGdS6DyiIAjh7o01hrAIjFAv9ZN\\\\\\\/OuHhZtbyUYTeErSTtYBNwvOCaM0d4gnlc8KkvNTL2Sh4E1tXiElWbOojJG812AFaX0\\\\\\\/3wBvwAmrNsh1sOVDU\\\\\\\/J6gPG\\\\\\\/X9QgwsKwdUyMeyZ9FTOBalP\\\\\\\/591QiSxotOojwMghdYY7ZkjdHFvenqvyxKR7QidtwOrePuAukRmJRJ0WYfsNbSj2b9of0FSZNNLrWFfJMIDZ5fPn34YhjKwoGNJiEMOksNTEsoUm5nrWotanI7PtRNUuxSArJqcUQpgsYpkBAPhi40xo1umssAH3Q\\\\\\\/yLtlk9EOIA11moKZkrUalldz1tST+WBDMnz\\\\\\\/s2dc1S1ux29rRPi8QIAiYS4N4FlGvNlxbv1TKQJnk2fKvcGokdbs+hqWVmyELw4Jvz36QF\\\\\\\/uMHSyd8OLQYbsZIIxFzXALpyXf9mOgZG\\\\\\\/fHmGNFHY1P3X2qfEWnGCxTbE0hgV\\\\\\\/figWUoVT9n6ZAWgBjKTU+tMYArK+EP0ai4fVrv5h0vFqgIMu3NZZbD6PcC\\\\\\\/q0i683pXN2K9Dc67FLPbAOKIBoXthg35718GdUYHLyUiWc9JqoU1GhyYwAZTvKpW+WjbLF62qxmrV1VGh3nM5LWTFOm0tKPY1bCAAFi2XswNiUnBXKEKa9AWJkNWMWeY5xI+KZ1Tj+NIocpBFROl0kEcETtAsCpXpYxMfg5mk3FF7cQzjm9Ee1T9iyCQXQ3wjqg7tjY6T\\\\\\\/1\\\\\\\/1zuYsjF3typzP7fin3QkQe\\\\\\\/RS4mOYaC9Gesj\\\\\\\/2PmJPy44QZt9yFt7henuWCONokfV7CAj637nzrpkbfa\\\\\\\/eHMErdGS4PGcVXBQlFBd86n8jkSzHpv2r4Ov5C1vQHvDp3f7MxNeWUQhs0VVTFhaawnYqVQ2r4+CwoXExmPIFMEw8bZ6mwuPngZv3D61kV1Y5DjWUhNLOQ5+ogw0Wx\\\\\\\/+IPDGECtLGqcDK3bF0BzTUJbHhfiNRqfiB9plQ6UWm6WMBlX6R75C86AA2XaSvP2CO1OLy1un4iacpeW\\\\\\\/NzO5YuHtfiwOLiLv2vG5U25tuNFQ5dkOlp0hfFsP3HzoCxhMD9QPGEhke+YVLWNUFOZnojIL3qXfCzsT9LGtfrf7FFkyDGTzYByyibMrTLrmQZv1vMb3ZBo\\\\\\\/SqPNq2vxqPL55AwtLKFbKF\\\\\\\/014RzOXb8aI4hEm6SJFVqB0QD82oSBdhHTDODmgmAwmfkSJco7a+K4zNIN+S2T\\\\\\\/YVpRs+PuqdOzecnVTGthqkEQczEvoerGaZxVfWiJYAuGjW02E1222Ki9DjDfiplrtLQ9prbg+e6kaGdQpenzP5OJirpulDdwiDdYPkjjSIENn61YolMR8CqwIrHY3g0BDszSHZttaQcb5aW7CE3EJiNUCQuhrVr4gGpKNMiJ9nUN+Kj2r8tOQS5Jyf+yibIVnBYlDp3\\\\\\\/kKoeuH6iOIvkKUr3NY5RHVPKJgNEsfHXLawMajp1hFOD3\\\\\\\/SzBSMeObm9x4nqxcF5Mllmxd6MKqL6iIm5vAxbNiSz+bmd8XvwvRSlmJz5rv0r3Rm8VUES5TvXUOdi3T31WltWooZ0DI9vFlSZZzxOKll7b17U+QB5eGGWJHZ97YR3SyjWJF96FJ\\\\\\\/HZxaKPc6vrnvJOfUdYbs31ue\\\\\\\/IYy4982P3JSQFGN4wzI0xgxhnpo29ShmgzLjiQBSlEJtsBkCH9jRQr\\\\\\\/SJBGcxSWE9RxPoHzWTog9lPC9OaxM\\\\\\\/4QGmWThLcv2eN76+vP5H4p4QM70p2zkJBCj126r6Xk6ewmenPkFsP9XIIwzGjvUS\\\\\\\/a1PYqn2uxIV6\\\\\\\/h1YJcSi\\\\\\\/RbFpt0OnCrvLyQ3H55MiigwsXcw1aKM+ZGA+XdpXdjamiTgsQ4nzPulrdDKAg+DW5w6WraqTd11+OuSr2lNFeGIv+n4NC8WbW28JVo6dxCZzOv+7IYqhRe2GoXNefjsRJYE1uIvWrVvojGyOr6IsyhP5ASDmnbKc\\\\\\\/NePbIfXLC4L1hrL+Q7MN2Lp+fwB8z37b7u+00AobtREYljVlqUTmR+764zH1+qpFGZ0+Y4EDMQN4EtxltU3SfqOlyhFuc6aMVjhGixaMGX1GNCbuiVmZ1C8GmmPcV8j80cKR4iVJ8Zsa13Ijco5ZGlSq87\\\\\\\/bDUZCXX2wl7Vzu0ZrN3houL72AodFRmTaMrm637drVqC6bXnDUsHeqJekr0gtljjP+py7My341Jjd+D41cuSPblFNkd0nv2YIU93nDdbiOG3kNxlcElvklbYXhVblxEl1qp9QDbghVQyCilvTbCc\\\\\\\/\\\\\\\/TYzon0jQEbFV0reXpCztuaZzpn2OyFHwrEUymdP+31A3VsfeG\\\\\\\/TXj+btyvUzhvTArzzm9HxOAzf6cTdK331h2mzMU\\\\\\\/F5doMylFxsIK2IZ9DJhfX5FCuFReelkZ4bAfbScjTBWW6DAo7em399KiYDjnQ+7fxquTFta\\\\\\\/6bLuwQsiFYBmqflv9\\\\\\\/VPTWRElXRi0i8H8xuXAVpaV04bfZkBE3TL0TnWOa3MCCmsW3CK38mzmz2pgNo0Geuq\\\\\\\/dRZ4n4TdF\\\\\\\/gSpa0PYtt+FhAQbdwmcXgaPgSHUrcv4UUZ7AQovjiXwjwGBlZ5lgDPYl1sKP\\\\\\\/tkZ3KG7mlzsyBt9A3wSnusPLaAihVp56i9LZDScudty1+0PX3KKsWnrBwaUzgIfS+iV7nM80GqEnGAdd31vAcAMpCfmkKEMgC4O24aXrgcxPs4k6PxLoR9fuGvK4naY1XYl\\\\\\\/kwDGJ\\\\\\\/hyo6Bc7LIM\\\\\\\/4o6G\\\\\\\/hrEmgACcV5UsqJI\\\\\\\/pcnavJIc7pp\\\\\\\/hPeRU98TnFuOpW5BjJtVirI\\\\\\\/GJ6OKtqGpWYJMmK2Asn2JNNVLrCmaRGN+1j8t+BtCa6gjVmewwgq1BbhfX1HUgbWKCU5Qq23oc6fbsshWoq1T4DJR0dryTr2XdujK5Ob7NNbnHhrG1qP3ILFnTgwr+P8SScENu6ZD91KPpOhxomdoVCEtFllURU4Yhb+V7cvP6A7K3q9NCFcQKkyNk12yzc8dWRZqGuF0UhffLP1jc+2+eijOsBr9RNL8MAucN39HOpnMpdqomq3A2w5nyClBFUPLTPS0cjHhg5x25WLRSC0tIJkT4AaSEzBfbAra\\\\\\\/9HUqI1HGrx8HoSjzeLRRM90B\\\\\\\/umiOTEKqRVo6Xi0YpWqyqvBA8OkrZcF4Jpei0pQ9PQ40IVSU0p0B96iG2cIuBhyfg6XJW+teWYdNRTyuuATcTHUNZn15GeMpLG670JAjiLEUvcpj3NiHTZ03\\\\\\\/2rz5xXljhJ0lnkdTMqUQ3e0AXGXm2GmGeTFx5s6lDDcHvigcb4D3Lt+gpNmuRXUVZ2C+rRsGsOJ6Lm6i2NHcDtBhWvbgGib+1T\\\\\\\/TpDp8acltqqH1GzSPVkxEuABgBvBq3gR8JCByJZ+JaVY63O9xtSibFE+P0r6Dre6h6WFaHFHePyzuVVC0R0\\\\\\\/YpOTdc2ZfL9DAhBlp+mHUYlgD18nVQQpuFLkqAFU6HsRwJEgRu6veQ4x3gKNR+F6OIzKsOEVtnphdlJ9MYDhNIR\\\\\\\/wTDo5XqfAfcWZ7eHbcVa0kfpzg4g672yXUvnWQgOJAJyVvK0M9Uao9wS2J6YUd2y\\\\\\\/2yQNulOhzEUPRHyUnS4TEOqljlrlx8Q4vuT7rT7ZfLJu2VY2YjWSintcISJgsooTudMGEGNVhRpZKyg73aMkJsmmhq6UwbV50JiNDxn+HAY9HiuSqIkRxjx1\\\\\\\/jhfgB0Jku8L7x3xzoydRJQJN4iUaEEokY7oMdzAA\\\\\\\/mnO2sHlIwgxB2iY54M3w++Rx\\\\\\\/qZBJV0ltXaoTZTZDJAp3imGcff2Rk7wFCoN2qC\\\\\\\/Lsy2w2pTxsEe302UA+n19z5GdXT4x6Je2d7BRgwrY2uw5gUlrd3X5+2TUxf6OoGzWRhypTerNAEKy+n2mz4i28RcEHTYDel8EFtywTTy8Tc4LDLsfmELFXt0RnIXVeHnGAi\\\\\\\/Rsdq6OGqUiROXUtcOv3kv4bXGiXow63nsVKIzxlQT8iEYv8iJrUIrImN8gfxWQGgfG32Z\\\\\\\/E3Zmh5O4r7Tcn+ZGn5t+Gqfiv30Ql7chQZTvZAwZtlsLftvScc1OUc3\\\\\\\/6bi9uR765jH51eH+CXYnqxdLsvL7nxo0q4cjZNKthwT75LMxg2F+SKDEuSGQuTzMIJK29ikdn2V5WMYhaHTP9F5jC\\\\\\\/JBOX7fKg8ANSPBoyBwPoVkbIfSEElyHHq3AgN4Ijl+1vjMZYB4gzrFMMY7TLCkOwoayeialD4wb\\\\\\\/IhRMzUnvYk4ABRgL+VIVw6owkhHGUYSndmCdgnf3S9U6RwXgqnHk4sHiCOzzXjuCVIDOlTQIxrcqxxPTTf0EC0iAYDR1gAtyKt0\\\\\\\/qm6v67z9hZNVHZBP012JgmJFnFfI4WIZgMoEWYo\\\\\\\/qAMUCuyrVj2XUuPW3mo6YSZGNhaoy+oQlr5khFJOKN15nua1wNHX1Clxd7yn9wvO+04sWzJHLcOuxv9V0Z5fXX3mwNM1lwfcM2IBZnd13i86SI+3Fdquyp1V9K0uDqWgBLFCFSB20FgVTRA9qMXfK9e14ZmSTmWn70nkIIsUcvhlFJr43iebgxW+PIdsgb2FICnPcQSYbeGVN4ZzH0hC7sOAgwImS0cCsLS0+7\\\\\\\/eIvFmrEj7u1gXOfw7VF7WnY4+g6jVdYHaGC6hSMQZPmRRc5yqSH8DMDThFwZ+NyjQiDkyHlmEHuRNWcYvhwKu8IX205m5xfvGjdoTjNjD+titZSgCk5XktDq01z2dD4WKCGTSMinBjom4MTgJT8G1B30eQujTXo7cW+C9+p+mgGrx8xEhm1bmRLzFvqXD\\\\\\\/xAKpv4jhvsOYHJ+skXgTkgu0fsDo1P647J0SfHivxXWal5gpi8rohVF2Z3rUU7VWNUHRlCW7RWXAxmjrMACv0d0USCG5TZ15+sM4m9LMdbzX2UH+rbN5\\\\\\\/aZfjMr8gzvfHqhpNMPN62vPhp7hqAl8bXrQTSQk98gZ87oV+EnEmaE0o3yFM5l3RuMh04ebQswTkOkJe\\\\\\\/KR4agJanAHWzbAfMnWXPretfYZV5hIYamBejqZdQQee757YexTQphbl3FRM2VzRx7tOhNSdSC7ALlVQeUZGDg2ian1hEIeiZj7wBz6\\\\\\\/mrbH21BM9dC39ee1ZexnmM7FjzDNW1SGPHrrxTLCS7gHY4xJc2Z9Err2ziUQqim97+ep2ZAOrh5PQYqD6sIoMpgcVp+YkWMS1oHMyj97l05zOPYVgDOH+Yzbzh0FzTO\\\\\\\/MslgfiTLGQMfwya6bomuERBQC526du8WOzp9epr+aNbViARievh+kJzhBZe17dhjkttfnHonKW+tUkjab6wfpY3Hysr+24FI8kCtmhFuVx8WHElIzOzgMrud4BDMPAMyd1X4SFiQm51ehhaqcxXbn4REpc8FLQtyMdb7fOXcMQ0jPslmFHh5btHpnAKXVP8XsTsWnxpfvmS40l+pWHTD4iW9wvp6AqzxHj\\\\\\\/QongQVB6HKtC\\\\\\\/B\\\\\\\/NtBJq0T\\\\\\\/2Mj3rmw5u4It1osgK04AMts72rziDfMffPl7YV\\\\\\\/NFoIZhX5tH9kcbZEL4Pv4tNCJtpAg+QbJHtUArkeKlwU2rQfXoSX38W1DfmqBXUxu\\\\\\\/ROgpsTAGPnZnKcnCFkCIlFSWbvdoIRo1+xxsmChfZVDzAM9pwPiLcyKc6h+Dawg97M5X51iB643JlZShz\\\\\\\/jaz3Y3cLTAwHeWnWNDa8j76jh+U7YRvesdQJRleAsP8PJfXA4CNjtLRuKH0MnOm2pF\\\\\\\/7iSEu3CQI7euD4Xdm2e1W6XAwJNSV9+0Jlold5oNIMutNQ+e0GppdJOthJM\\\\\\\/EbHfi2cOCL0qowRzQm1MNeqW8Oxy6NqH9SPi+wvLiEAew5TuX2XNyoLe1Eqv7dHzbXzRbSCoToNmbyGByLKeoxVvLKw3cDVFLOT32h2MIxHe7JwSIEDSDwCzpUuVy2jRKrZpzHyaD1l0JW9W\\\\\\\/5dF7D53koi9LoJgcNhtl+UMJ49HzqnXjTpdp9D+pyQZNObXHrNtrk1+1uSHk7b3wkW1d3GFoGM9JO1EEDDn0c0RSe1gvv+vlM3FSnKI7yxIrO7Xrj\\\\\\\/LoQMT\\\\\\\/Z6eQT1jqct72H\\\\\\\/Rp7gv93gOKQiKUEvaYAiv429zgqQXFRrtNumb5PCeF7MmIOmO1KSmYC7FK8t7CC\\\\\\\/A6ONMmQur\\\\\\\/leupdM4Sg4yyELj115XaUFaYTtRA\\\\\\\/hcSJ6voMTUQwEFDJjxdgR67fAG+e6SajHTdiqQY+fpZ\\\\\\\/XnSgKj+eLiY\\\\\\\/mrQg1h3XUar7629cTB4ktVWBSgTcSPheq2oih9ISmzZBjMpGMWfwRsok4fPs2Xw+5peQ\\\\\\\/PibpipJUjGqfROu\\\\\\\/nb\\\\\\\/6W0Lrch5KHT\\\\\\\/W5+bXCQwC9\\\\\\\/4d9wxYvo3RP0BWGFNV1jRlETfb95TPOBnV6kwk9An9GJfca59M8USkffB9h2NU9Cffw10x0GbtdTDZNQYcjwp2f+poOy+N2QbqBuBqLuPURCth3Zut1+umrT7e9eZGDePA\\\\\\\/H7QD0f5jEQ3QnSEwB02cNKnGmMjU0O632Ob6sc6zB8Nv0rJX44K37MTgzY6bnh6IVNTamnIeBOOZUF+5sMAv0A07Ittm0Wy2DOB9LWm4tR0ep0CBqz6up5ZnuQyXDn8mA5PNBFPiUwgfBpg5HZrPH9vppt7hqa7Ss48U0R7o5MqJt7TD9EnIzDfx854l1GTwpCLED7zMBI3B1w57Yo219fs7ej5RKQremWrzAsweVO0mc+xA3tkfpPq4i7mXpVKQgH35VbdDW6iKDAGfEOqq4cCW5xX9bLQUxzlQSqhAdN4vU3uqGrMLGrMgQBcRllQEzIAAkHnzcARoVeGihq2tlyOdAeDDWDUgzYMnEJEVY9LSHkLiGGrb9EjDYBtu8kFLh833ZZ2GABv9aeYHEfl2LcjtIU2NTi6ciWF6GJ9tguwrWsoIzhKkHdWbLhkVH\\\\\\\/mJ7G1ULBDztfBG5u+5fGkOrUVUQtW1uDec6VsqhpoNtEXpn9aPsvTgy\\\\\\\/zrcXlUdNq55g41Gng6h+GWrPuJuwmV\\\\\\\/BRqIFUw5I4x4jPOnFOb4z0fKSiADJEzJGSKTfsEnD0+euVmA3xeYsZIvzyorNPuKOa6\\\\\\\/cdnfPyhiE7MN3rZyvPXJCWBtv+wzwUAxHRU8uuW+JFu3EUMHGWZ0HCk1KQGZZb85djpHLj4VoqpwMy0onUnLVxdnlYu1cIKuHfFF93gmB7HQPKNMGrpbEtuERqE0dELAIC8CkuXJTNRU0r7js9QQ2X60+0TAZ8by5sYs+VPSESKASf0GCJMe4wdNo5KmOj9bzyPUzrRpiLsI9csIjLn+6E4FTD8IJ4r1SumuJKSLXTwdOLezd4GWQLkAqa8b\\\\\\\/GK5OVWlr8nnM1c05hSEKl+efVpuvitiIiu2dK9KULyJwED7Cx\\\\\\\/c64qI\\\\\\\/9qKf150LGVcZb\\\\\\\/RZFj6bEcleJGx+QaazRsAYRM3051bpkrQCgXyKXKEvf8km5kpkEHRrcuoILjTRCndpkGGX+NHFbZc4A6rjYclDsIEFlZcfXqyLwcZpj6GT0tEndgdrQQxksarN\\\\\\\/nJQI3EtFYWMuWWwhVNI3zcZe5QPnlpWaOaxEbRPEy3VBMswgMiMJ6iVPxqo+VKkBhOa0YY1iKGn6cTu1N2KskIZj79gpgmbyXzFaz2Pd+hUW2F8U1GV2\\\\\\\/KWPvNNm\\\\\\\/uPdIEConhRorJFMDpgwxkO5hK9rFxFZwpzSNiEw0X6\\\\\\\/w79Caf44KQNw7fIDPvQHiEBllgHlVv6xtPmMMmX1ISCt5xzm2j6AfJWId6iLbR2wG\\\\\\\/+jXNQaKZFLBLoOuZZmUr4Y2jwDI7YDuGfR+Nj2k9Hmk41bVaL6SqjnHIZQ2BtuAS7i9MH9vQxjSoQNCrzF34+MIIuqVwNUHD6RFxQ2deuI3GV8oVvnyuScrqZpmSCLGBRQyTkum7X2UBAORDR6y0RNHO\\\\\\\/lve1dplxOCqsc54w80xpWFRp2i27YnuCk\\\\\\\/z4n5d056rHDTuO2\\\\\\\/nfwU\\\\\\\/DV2jBORk4CtgcXk5PDWcrdVBkVUF273OHkH7DnkoPI85Oa0EJJNp\\\\\\\/VE+G0RWXP0yg9MncTk+LuCSCauS+ZaHpumlQG9x9njq1hizxuy\\\\\\\/MHtwo5DhGAYdRh2kjrQnC47OwdghclTNVF2PCtRqn3r2AHH4ta\\\\\\\/KbwNM9\\\\\\\/KPyHx9ZK7O3x1rF4I06nfVr9pfvgbZ4U70Ggy1BSB4zSH8lm6zb20Fkgw6LPr\\\\\\\/igbwxgRoVbgHf+87qfp67LKR5Mgid7Nz6H7HMnNHWUprQHxTgAKTle3WhotIqri2f9E8hezIqEHPm6ZSkv79BvSlwvoKm9eRpMHF4PGelbPG8SDmRT70R\\\\\\\/h9Qzb8KSBAxWAVJ0s7VAML+g4hP+Vfdajv4j6plIkqi15ST82bf5klMhgi4xsmTWqQs3vog\\\\\\\/DpQc0EjZqf3AjwVDuu6\\\\\\\/rH6\\\\\\\/jBczkpY7cDLag7VrLVbWlEyCNV9eKkf8dm3kvY7HmP6jcdTWU9qCdRo+rg8IhNuKrfIEA\\\\\\\/jPnTc37J3LVEa\\\\\\\/Zag+sooGBGdCZaD7vqRX49b+TDxeKnVCxuUYeInuBTMc0m95SBVjgRoamw1W4c9kHAOUPyneJtyGpnBSGeq6F0QE9exzUknuJGGrr5xX\\\\\\\/+IxhdqqyhZeIOSezTl6eoDJZ95HEw3bci\\\\\\\/ce\\\\\\\/gbuA4j5UTnNPeAqYkVbfgdsv3DYRVdvZ8d5zyo0egoEyqSFaqNxJiwAcNg\\\\\\\/3dcC25\\\\\\\/su6p8t56ISnzKKiOd6fY0CVGRY3h2iZETiN\\\\\\\/D\\\\\\\/Lbc6KaHPsv8Cm\\\\\\\/r6\\\\\\\/DCdINRQRa\\\\\\\/9Q6Vo3UWsSLyCkPp536ZGjbJkuF9Ka5+SIsdm6rMGFgf4HGwlfqpjeDDF9BOLpe5zvIgwoRF2KdhsK09ECak4AJYEXoW+UkkZqCH9JKy7WZDrG0L+qEw9RKY5\\\\\\\/mPYZx7YervLG+fNpsHKahIeKseTMm7per9Emhs+M3kqJnRSfRdMLK5mQXIUZDmzsJPh0KzZn+y4+C4UDAAivgRAh\\\\\\\/WbrKkCDcsuj9rIzRKNRiiS\\\\\\\/r7AJrJaQmhm66NnO0yv9Z3ngnaNW7JgbAux1RLtt6XNtwti3KKX8S5BVVxrxZg1lbDGgFMZzXd\\\\\\\/oWXFfYZRUvP0rvDUJDO0vl9WxQAr24ctzUxnE3B0bghvsvEYo+SDDNXw2gFj79a2xwoszH2+6HDfBPyqHPlGqirxSpqJLugMfC6v4gA4KkXpilrpvzz618uYxxGutBiS9HX3jgVHxZhq\\\\\\\/SGTncxt\\\\\\\/qltqUmYW4diWKnnqW9FjDZy4nvkdfDSWFzFEBDX2nMLaSfPqLWCztQIFWz0ylJV8Phuk+W5Ap6aKE+Y7xg47nyVm2jyaU\\\\\\\/mkNE7uQPqST++5\\\\\\\/cG7OOilvRGBGGv5toA+K+zASPELw6sngoFr1zZMmEtXSmxdvY+UwvO9wOamDLXMAOFOWDyScgR7IsL6hJUzqhPbGBKP2ax0ihsHMdwpNaHZ23PxtFxskJa4QGmXAernYxt2P02fGYGhROWtqHkrjk0YZz+lrw405h6pSe\\\\\\\/hmWiMMeXkctqd0ZfqHdjy0Xq3uz4CuYZyx0\\\\\\\/9DmPO2OkjB\\\\\\\/NaYtMxoYBp431kyB2dD3KfgyyaHBD3d3TVog5pCkz4W3SdlgcP6imYFNW9IZfFqN3pDfhAYtbiXa+j\\\\\\\/cdqUxtvA0RbJ90YaPrypjcmMWAUzvCz43G8S8nds7bij2v8rsxl1K6IRwtPVhuL842PZXBbZGRWkejg4mBSXN+7cJZvBEvoEUjMdu2ycDMtw3HPc5B8Hm4b\\\\\\\/GHxavVU3evT63EF5+Jku3LCik88+ofWOS24eXHXCrJVpzx2JklI04Fl3OZUwlEnaeP3dUPujBu7tpR0nGYgPbNqdjigxqngZ85hUlAQGJlBPZGl7HwhUySK+1APTrvsTvjY4yXByUhSzLLxBuR4E4bBkv3IHZjPH7pRyfa2jKVgmGm\\\\\\\/DFl5z4OP\\\\\\\/pgz74LZrd3R5IJx1EasWGf8bNYlgu343eekjvnQuMW50a9XArkpRa2ELq7BYMh+T7IvD3FeDw5hKj7NdxPG0sa35VvmKmP67uMYXS9bS0n3eMtSmxphlsyn98zrPIY2hEKmt6pS0yM0RNT5\\\\\\\/DXkn+BXiU2Yd0lRrP5Pnwwv\\\\\\\/VB45fdx9eU9ELGEJ8wtjNVmrc1\\\\\\\/VXYLWC7x7yad+U6UvYRmzNN7cT1WNZgNCnlFjL5RQIdoyP+wmcGbByLrkedLShCB2Oc3hm\\\\\\\/aL\\\\\\\/0mEk761pjPvCwRgvJ5iYBEA1V0LnWEb8OA06t6gN8PxK50+ZGoyn2sWvroOBkHcZHS5C+MPR45PVqM2+JGlp9hWPb1f\\\\\\\/mUn5r1YjtXnYmXROOgiofmWyb++v6Cac03Z48HJG1LL2waxaW\\\\\\\/LBq2CiZNpazsmZ4uhsRdXfRWmsq1wLvU\\\\\\\/pYkldTHO3Q\\\\\\\/Q\\\\\\\/CsIu5CEMINImMa5\\\\\\\/SbZb0qNFEWzA57cgk54Ofy\\\\\\\/3w3S7C6KIbbhgBnGlfLTkG8lLYCu2DBRQIjDEhBEkbRXIA9Vzj8V5+h3X6SfT4Qjdls\\\\\\\/ywRAtR8BaCiHpvdXqXTM+B9JYCSrAC+g9EOj+EJ4aV9eYg79K+uoFXKBaEQP5AkkX95GrFQebfNv6jYmvbKaHOzAK8zEpWV1GVYw7OAKKjs4cs+n+PPSIHUnbd9jsuZGApKnj9KmC0wjVgkujsAee1EqH2f1milWctaBeLUzx+s9gybCLbZEMr2POI9RsyFcWbPdvOFIKdg2qEt4umZEuBSX99TiGZyE1Oi4l6Wk\\\\\\\/7HU6Ngs8HzeRFc\\\\\\\/yV5NO6h6iP6gH7BAUZAWQURM7cZp16OOtWgj\\\\\\\/5XblGvHzkoqzK\\\\\\\/Stz5Ln9z2zJylPoxxfyuXxJqKrCpOboDIJCdEANipOexWnvTCai7DzpByQKJrkGksOM+yyXvWh9Xt\\\\\\\/AmiEOxCLoywJVU3jSljBC2zuxHQA5IyZF55ZJTX0PpLBNcgrxIhXQ1g5W4XpRxcCf7\\\\\\\/c7q5XDQ7AqwoSRerLyc0ecO49WX2lVy\\\\\\\/ocJ1jPlhn3CROmOllZbXQstqo5OeZjXWhAnSXCBgXeNfYJo8RSthhaIWGNyAFCHseCPElnMn6bsZCUe2BKzarljVPFgm+AfEj4sJziXz8AmAbTpTaSYb7YitpR60dvAbi5AKv9fIET6M8a3UctkdJ1yiWPIeg2EVEfDpNBt4Nx\\\\\\\/1YS3FZ4zpe\\\\\\\/JHbfm5\\\\\\\/ai+GjNLHHKIRb6q4090T\\\\\\\/tP\\\\\\\/XEEPUFoxv2BVzvfxiYwu47hmWgD75nPG9RRwY8OgBq2KiMAz2ARLF8VQ58ohpGlucomjO22Lg5gKJss6rndCuCyHXXIVCBneXopKxb46AuasyqROkSZZ3tyYeUZ38h\\\\\\\/j02idl5hUlIpuOzyFv6AEVw\\\\\\\/OwyjbWqW3CGMi6QxMZcL\\\\\\\/lkDj5t89FWRvOSnM1Ke1QOLgoT5wXmDxQiJJusQpYNYUuRscdcTgyQAohvGZUXDZtrg43sOyfC18\\\\\\\/rS2pXc5doAfXdPFp9z6O0l5kY0XMwWVYG2a4Aw9PTGJuiIrI3xM3Rk6FiKQgQL20BHt\\\\\\\/zBSVtJ1Sahqn98KeOStQSOBsC\\\\\\\/V4ep9fuQFuvO3jaiHeYiEkWqigfgUrcHMEQFoBvHY7ttuUgRQotslYG6OtCqX+PnYT5OHNstjlRuJiCm6YkbRkoTg+mXqVvOgp+B1+wbpFLEqo2PrNuuYVha9Jt97BIzE+PRcgmFJgV24gSxQBdDMycyGWJwl6AN\\\\\\\/UgIHh+wZEeMZiJ2IlG+n9LVv1iZGlqfV3nAoB4QT25sV8mOOQkLgbDL1\\\\\\\/Kkos0nYSbCPS0o53jpTvGVetxCyyRlV0EGJjth4iU6bHoh3VLO\\\\\\\/V2h\\\\\\\/Nq9nrFcF+BueGrang2FM\\\\\\\/gyXVYti3cFyYYj9lrb2dKZbrcccCEG8JHibA5NMyN8cCwfzCxmIs5NrGW1qc\\\\\\\/R\\\\\\\/m32tedciwxS2DG7RjGghOFfvewARWvkwZo9y7hL1URkyHAeDrkvJ36DNvzAQdNFoKLhkPsqacYg+tSfZfawee2hbOM2uuQRLsLJ2\\\\\\\/37OXMR39ek2nGE6GuFX9yvXxUiSQqJikAZzqDg02ANcB73ePE74YIvPtBm7EIfRdaSwriQK\\\\\\\/6fDrOFEhynORhcZj+fKAqV23jn4X+Kh+p\\\\\\\/VVv6VoBSH1nyelp42CDCgvc09vM4ahOF8KsuSidlQ\\\\\\\/9+VMMoGZnghHVDCEvMyDWETyTM4M9jS+1hCf9GmIqVwSWfIWo\\\\\\\/KsLYe8cos1+UyI2oGgQOnhuipG65cMmxUzmvtUPjquCvRRdfxZ9rBylaXUN2NAaPuZx1OhEA9+qScqkvT4odPyMIwhI2jcMV5QYaeOW7mjh90eLamaw0JtYTLwnSeSlZRkK4QMyoYwL4CcysLcyAyFKaTUtpY8fesTTL8okmgovMr5kVPoDmDoXoX9lVUGpRILEGLU8aYN0zLBcXk4eCYuKBW6Y2WthNVjCBW2I8x49WKZQ\\\\\\\/5hXQ10lwclULWNF7DkwN3Q\\\\\\\/za4IAI74TAQTh1mFd5TmtWsVfqO8Fgs\\\\\\\/3HwmaOW4\\\\\\\/IScv8XG9A3dsZGNta6CGWpkZdJ0veXNGvxTk\\\\\\\/IsynW73BP0lzb12l6JP7Hg0FYHLbAkjciRAVOONos\\\\\\\/JbNZN4edfpTDWr4pflcFGivmYJGAX+Yuue5Mf1u0zMsGet3Hz+9iWwgH0sj98wOeryjpe+3j6FzdINQaqaUC3qVUZyU+23+fnuR3Kf5BqqwzSTTRpdFpdEOGPqqxBORsIt\\\\\\\/eJ\\\\\\\/F94cbi+86boZMMWa4\\\\\\\/4\\\\\\\/SHr5EsDVKpnpyJPAuUVptx5ZqmNI20bvKv5zLB5RDu8kH3YpQpU37FZeJSHiW3oP5lOwgKUInEwTD07RI2vLbcwo0ndKnp5JuRvNs887JyNJe26dhmw4FvXnJzs3c45XYhZ6yfqh3SGa3oBeURTurWuD2TPX9Me3eo0JrxvRqyt\\\\\\\/\\\\\\\/eCTlBHfmWQMPwpwLjdOgEFN6x1KDjtEKtQC4EckpUB+0JjTI\\\\\\\/ta9VIKvEtDRFLf4j0evbuaNsLic0402O87vMDS4vc4zIylvMO2Lht8NOEOyDBwu0fCL7NtBib+Yxrg87Onv9cvBSjvR966uO1tzd\\\\\\\/N8VO0kKLVfyjyHuUm2tlJTXAbZWtaZxJSUctOVooXRu\\\\\\\/9Eib59dA2Qs1rj2Ixr5S6vax1zPYD2yINLfUBsK5NpU69hDQRSpni0IX3nGr8bKfJfUKcj2J3dlRMn9YHk6GHZPgmfkC\\\\\\\/9gRbkY9G51WMNAkJouE6grrXyvTfUukRXiVTae\\\\\\\/uKTsfy0cWc3cSCD9vjryTzsiFkDpDZFsNYE8polSUPenT+5lplv3ruXWjpXggNzjaCXtvjaPEhknARhU+3c2uxEhYSkMJC01l\\\\\\\/+\\\\\\\/3WbqO8G21aKU3rpxhPTZ65Yso97yzPo7SWb2N+yJPXz5NlmV\\\\\\\/T7kuFsRZ5QoDS\\\\\\\/eN3V9jP7Htu7KJSjyG3QlyB1SOftnzbff\\\\\\\/ZTQJeY+wZs5j+NtDlTHeWjvcResZpPAhlWMeo0QaQIC3d31ogtpb\\\\\\\/RTNU4nJFIAHDqpqLuGbLy+T2X7WrpXQLqpm+V5ga2+VQd87zeyCWCZgs6EznSMnsgr77KO65jnOHywUxPw7A\\\\\\\/vFF6XCnneDH9UWwFZnUZRaKXBJeVeTHZb16e8SprUUs4\\\\\\\/oH8GYgFY8h1AxejPLUDVFFibBiP+81yOyOEpM\\\\\\\/8FBBvuE2qcczI77F3y0E4wlEUmz9V2l6vRyOppEZVm0SzTvEdOXYkUbEoC8DovvjfdZE9uFuwB0MkXetwbbSgeeUsH5qUvzGQhPqfxhZBucV95e1G+AXPfiT3v7E\\\\\\\/iEgVt7tbPadFqRFgQNtgu71rzRFCKaxlQkh+HAgjlqZCINNOFuVxTqivf3Ol6gv\\\\\\\/ypkwwiONID4M2nuvk1uLCNAzIDYLu7b6SBGpL8MuJcYb6qSEDyCAX4y038E+s0UqPlMDYVAXsziO1d7SybB9S\\\\\\\/995QW24Y9thSjzgxmliBZR\\\\\\\/b\\\\\\\/K+eKt+91oRESyi3nTb20Gtq9LotnRKX3cUBTqnUuopZWIVIow+g6DCMk1j4EuLzcOwrlTxrYk2T39w\\\\\\\/A9FyBtPwmxh6yyhqP4KmlHvSQTn+AOjWsOSg4+D0fl1XgrP5OqVxaWUaROkqcNW5Y5+NVNgWsctwCR64DWAFEv2xdxIXdm0GBEHQQk4YpGUvHD0TUEZbQKw5zYs7PFpW6eDb00oE6mjCAObVQkohYWndFbZI2UzkaNaWXf+sQJn8ylr5UEg88Z6uYKqmk1q6AhXdOIYkR1h7VCY1crwf13wkKDvIgaFeIgYHUsUXNCul7VDruealPFAAsGy0zr96o2FLSHeSnvppotERl+lqzTkTpu2Zbj1AkqG5NhYkaNiyZeUYmrhbB3ESvNiqb5mnr+zbkA6J+SmBq\\\\\\\/rFyr4UUFAzqABhqcHKd4a00PV3Zxa51UxWB+oYBs5MrSIufyz16Eg9WtX77KNkHR4d5kIEcOWG6prPh9JTnAO\\\\\\\/PD4S9z7QvHihon8GnIhMSf7reAVAoog15fyWdTtg9arWXKA26I+fuqdt8\\\\\\\/XqRnXeBqOHQO0W3K0PxeJC6fAhnkuE3NldYy32ss4Q6KQ0X5kjCCJ2ikPiOMbk3SHscVLA8aq74G8jQ2ebXm+zAHR82947WWGO10281ek2ACQ+PQlb8mbEBVJvm9wWDdHGyU0QIXUpZT\\\\\\\/iid+geFsV53CxOV5lHol8LUXZ7MoStD7Px3F17WEGHSn1\\\\\\\/+BExw1ob3\\\\\\\/6KtsY+\\\\\\\/m1kLgnkf\\\\\\\/v5r3e0plj1jtxXc8KcB0a4zHJDlFARs+eXmj5K2eS2+XmD\\\\\\\/jobOWWnn2x2Pfe4hYMNkViuyd10szVJ0II7a2bhY\\\\\\\/JYjKPvu69QBDTYLzTcSu6siDNPdwxnmc+JgCdsFp9rlw988a5zN3RuWIFEEnMBWKNtSFJMKvkynCddUF56cZuuXbl4ejGNq\\\\\\\/SGCXtTXCuSs\\\\\\\/GgOLGDDLWpZ2evZCe7GuDOJre52\\\\\\\/GXkJCJSmeoRS6ppS8z1JDMoZ\\\\\\\/dD3I7psoT81WKDN8nQo5QBN8nd1ZBpd2BhBsFZENTuiDsu5VYvsGSS6dDgRmyZMpuowBXoYJxMrmmnbWhnI6+nj2rRyESdoEaxxzfq47upTnFgK5q\\\\\\\/fi4V1IqMeo26v\\\\\\\/P9E3sh6500fvqFN1rpBAT8HfhmXuHl\\\\\\\/NV3vl70ZqHRri7GOvf4VVxHMnw1kgg9wrw2kRQe0foU+AejFe6OHrizPi1jaoi3ydsd9z9c73WwglFC5uGvZ0mp2eEu0uyGWiaLenUwaVJL8WU7SOhN2\\\\\\\/vIHwP\\\\\\\/j9vBsfs\\\\\\\/I4CGpHu0JOIyWLcKKrlea+Z0H0\\\\\\\/kGs9AQvtSujGViGdQHByIm7pq0uNWop20WS\\\\\\\/IRPdMY02NOEPVDaW0kHFk+ET+9QelK8J5EOA8TC7rsTF2fKNP3IZVNHmdDrBg4SS64Zdj5B4N48Hws6ecKQNh42pjMxHFMvrnmaib9vWdOLFlrcab9yOCqbGIJFwxRyIUgvOQJbOzw1NNG1AUBueYR5c8vkq4csvRAoysC6u+QOgtsqgT5S0h8iGb1ZWv6yq81N\\\\\\\/+3U1XOkvCUy3rMO+Fia+uyZSZpLz\\\\\\\/z0pN\\\\\\\/fM1pm8pwVXkvL0sUnFZfqKhKlidRf\\\\\\\/1tKqeGWt6OGIfpuOcr9uW21xhpOGLDG8K2mztuHxGEM3eHqi4tUgpTT8nX5VqOKFe4R4a4UHACY0Qo0U8ZWXMhl3rphjH\\\\\\\/dUXzIQvRpxNnGWya3aBnN5DtePiZZ9SOghyUm97YdU7vZMVd3DmV5pPfEuI4em4bG7fNwhJcKcsbASKUo5X76y9N1tsy2skCm19UrAc8Fj6JQErz7WA02GE050dH9Ykajyf8RrdJduvWZ1f169NJIkjQRLIPyz2UWUARcw5tyM7z14UtYlBtfqIsuoXeMmddQn85f2PMoA3LIHvB9XpDLubvRYIc985gkyQYcvlrCD1RxHwYxkSW2yrNlu1Jj0wLb1h1ceRf0Bg2ssdwRUkAypULpzSnZTLCSbapaEcstvbxy41P6WGa79ZE+I6lmtZLCtUzqQepKM943XqD4sfCpLpULL87xX6RIySa5b3FYu8isLK84fF82cH6lYz24FWJFYoQVePxPhD0YVuIvLSiZTAkI0V1iMVWm5w1VCGilyS2o1ueDsTbBNPa+YHuEUUYQwq\\\\\\\/hvXgir5n+\\\\\\\/exyYVrX\\\\\\\/OZtN5sRxg5+8lPOJlLjxl4mjPnBp4dfn6T2yQyx3k9IwDIR3\\\\\\\/q8dK78v2N95iJgiSWbuJ4nKoEx+M\\\\\\\/ojs6Kgc5RRRqiDm\\\\\\\/V9OqhK5AMZPkAyB2363nKSUVDxfMNw9ybLWKIbHmovcGEJ0EhQxYOcA6PCoF98nmeOAJJadZkpoHE04pTKYqi3TVxhesixv77NZ\\\\\\\/CPXLAdmUL9cELw7hkmFxdVIk66HE85rMTPTuHlYmGrHydEHD8AHrg1glhuqsdhM7\\\\\\\/8QIiUo78OuACunddPVDAMV6235RV46VyPAcK8tCSkcv31cyxa7Il8I8DHsuhAJOdMAGmBOa0fAuBjL9jhuOx\\\\\\\/WhakIxlpZNkG5ZXSwKhDiSz+M2yh0+dgFI4wckz8bzRH2ujpBqQIRABSDO6Y8CRennERbNrGkuabHbl3+MdgB3LK32wPxX5RCei8mKwQiYzT6Xd81KWA4qypodGGu\\\\\\\/8Br5iBqMj\\\\\\\/BY9BaBrI3psfqKPXFzRuA7lUK1w46k4Dy++SFxfuR\\\\\\\/+LIS0QEcO0Kus8H0GsAKhyNpUsQQw4FDDKvdIDqnK0N0r5AIIv+r6Y1YwSmqQUtvoGzCQQWrjnJi\\\\\\\/RANSO7Sq0zfvr40Eoty2upvnqVdeEnJ+cXbklFhECNGMGLycRmpOQLp0VT4UTdF2\\\\\\\/QmM7V5F9cyXdg7Kdw7e\\\\\\\/h0P8r1Va5VvmJFfYEh34XrYUyO+JVeFyVNzM0zxGhl7yjOlWZoWbWV2CXGMbKu9AYgJGEAeDsGC\\\\\\\/HCpfDpeEZ9QUaoCxTiqJcFYLcn5QZUhF4zCVWq9Z+6gj5UJi5I5S7E+3dwisw1LBNf5ws35EZ3T5rD\\\\\\\/12SYFSIDRvGQX+nZ7pA7m\\\\\\\/dmNfqF6rUjKfgQ9AfEF8i1PS8u5aJSLu6SS6mG7d+eQLdbRvDWHBrQN9HJhCqlq1kHQdiVJaaZUxnPL0Re766ZPYkVAJxE1AQC5JeICEgrieaX6zhn3tU1kbO3s5iYFT51xN5YxPZ1VKdjumIRZvZ6A\\\\\\\/kZUWWkASp+Piwi0\\\\\\\/pLIBZluUN2jl\\\\\\\/EurnHhXFujj0pBeFUoAkdfBvuYIokm++o6ZoiVO1n3F32yiXjgpPL3vgGAP5F+KVWNds40C\\\\\\\/Y1NpnFPh0PtwNPnDsQ6jHdlA6kIPK+hNlP0aMlbNWAykOCEhJ5Eb2hVg3JWtXM4pPWDWePd1xYUI27haGBBSVGLWMsenCqUXJS+Pjt52iVZwHt8a9gI2oj7vQtCFhPOXqbgm\\\\\\\/rjVKlNsZpXatd2wLV8huGTiM3lPKpAZJDMhK\\\\\\\/awc9Ul4oHfh+Ng0gXu2PCxpB6WKh89pNkDcBL3JzEwJaRezPgug\\\\\\\/2FOsmvsqETipap4LZlsaPiA0NxtJbRBVr+K2hzTIpweDD2ZFLuZoDmzXQmu7H14ToSQBP3EL++NdXtTfisSlrmAWqN7bOU6iCQme1gnuNJcs0UDDze4EFynfDn+Iu85Cyh+rc2V5eg2MoKSNFPPrluZLlWrpzayRfYj2mYEV9JZSmE+tcX4bL4SG8BJW0zdG7bjdJKEBl4cUELSVvG618jmvfnxUKPGI3Jr502POT+fsAsp1mSvpaEpB\\\\\\\/rM8B\\\\\\\/\\\\\\\/UIEuMUrK5JvcuCaAHxL+3yK\\\\\\\/lCRufJsAzxL8EXQmlIFcrT6kgbHGwQQ8NUw0pnt\\\\\\\/f1ynhQvtOMba5clsGQXg\\\\\\\/83Gxr3H3klU5bRhWWbmT9xH+wbHjd7hRJC7PXT3zpjZGjmd46LDIBV6sI0bMNLhuL3IJxGyyfBajfvqqHNxiz1usRBgGi2+vkfIC2t5YXhgaajCitWI6MDRtv0xPQK7fVnx29UyRwGThwosY4Ug\\\\\\\/Z1ZNVuVz07BSU1fRd5gI\\\\\\\/0DyJJxP1cs\\\\\\\/5+nZySyhmoPt5qhh8uEFZKc+nw4GSYzQU\\\\\\\/\\\\\\\/XAqlu16dhElTzTMp3hwQevXGm5uh3zcPIeYL0IRotYX+AkAdp7OeXhyGSUmIxw9kElXd837EpsDTWP3xeaHg1FdS9o7hITFFm9XZr0\\\\\\\/Yrg41jcfb\\\\\\\/M78aDm\\\\\\\/0C807ujAOxiJ6AoXYRmoOxgKLGPQM9Elp82TZOKmJAYVCEiP9DndhaCxTri5rOM5Xjd27Un6QRSWwicqNqdjuiUaSvgWH7mygkDQzubMEEK3IJ9Cgp0JPo4yoQdzsKI2B51g00aDFVKy9T5Nkgs56poymk8AN4qOHglr4VViFTWWItb01I8pN\\\\\\\/Qkz\\\\\\\/scARcQH2EgJUl75A7E08rcu+EIEfuAZPJclnkGpY2xGWE7vpBfW5Z3+7+qV3zF7Sxkhlai8N5DoYvFHebAxlbXvV1VkIo5wrICrJ2\\\\\\\/um1x0PvG1f2cKd6dZ3r7E2qyLtGGBBbZX1nhhsCcYupEXQW\\\\\\\/lGv5Ur\\\\\\\/yET4+DeudYizgzQAGoys6AuzcOxTIguiLAiIAkgn3NQRFAOkhhI+MdRBeqbdP0B0kiTirsa0cgyJxNm4c3GkchxuDx6uJgvu4YV5YP0ntUWfN5I8kwvVt4Hd22kKLKeJ\\\\\\\/KeDBKA2KH2ffxpPbnqYM6aiTZaPXFVQKw4+oxjVrt+JoQVST\\\\\\\/GcCupFdAAddE23IHtfyjZClkDGpSOBSxeKB37PYTJMOgTwnGZ+wCweTSi9o7znn2H3iVtufzBBIJ\\\\\\\/WYAkyPkYjNrrrS8tqYVxUsF2TPmidmzistHKs5xk6iuyh+8eKnZOYkn7Vj3NJr2UWKrNwSHI52RGPOB4G\\\\\\\/DMd5Anbb0JKAOK3r9iHj\\\\\\\/V1NSntUYVfP+2g\\\\\\\/iPGNbm\\\\\\\/PSiD94rOMNZxZ597HcmieU6Oh1k8FIW8kJkxs2m47ZGGi\\\\\\\/o6UjWj+40UfcrHBymOWWBDZ2mgiXXlGaUzdT59eonFQyUyNyz2EYS+ALIM\\\\\\\/Db+cVXe3FfWTIY+qI3T2MKRBl68IuQlxk+Jg0VoRWtf51OmLtxiBj32X0xPm+a8N4O6c9mRaU4yEJbmcCkthZLLzkvnWOz1sxLFB4zET+nf1qsarPLW3xck90F123KBv\\\\\\\/vk6of9hDm1DFEYc66gwSPP8aDHMPy0MpLfYbETcblug9\\\\\\\/YedB1jTtaPE3bd1pK8+fMuo9DJk8zsWNqbh0Js+ybqDaxWE57NLb0OsGIsJZd\\\\\\\/OFUyOeTZ12CAZEP7ruFnq7bKxQNWznYRBY0MZSZPysYn\\\\\\\/4lW5b7sAyFnjaW9dYyykx7MnK3zsK3SIzkjHuWhCe5NY+5Ow\\\\\\\/8ykdDHcj6bN1m6GjoVPjEy7VcGc2B9MwubXxP27GLFHqKNfsQX6a9PHQkv4aiFYp7tKMco9IzXQ\\\\\\\/f5dP4I40rP9JUso7FKaP+ykYw4eB5\\\\\\\/ljwtE80qlGsQUnxfRkn+ZWkojuq82xCxD9+4g9to\\\\\\\/ZChFuDzD4LfJRiCuvPCDsKgChs58mjm5UM\\\\\\\/JMR2jgLrUtMxl0LfYz6viJEsgrS0yzvjbzk5fCTkedDmpXi\\\\\\\/6P7zlenDURJ3OWdGl07UbYVWOLCPxTj2JtGPu+WLb\\\\\\\/VanvyGV9PP0dgkmXvHznNpmZqJgYDvVCX8u16Fo\\\\\\\/pdHKFZefCIsdjIY1i4Zwa12boVGr1vmOvBJv8vntHKsK0q35E7yosCJqWsH9zJxaM3PQhFqp4zFkCMsel52tJzG8chL5KJFGPhB7wMo9a7OiC19veoXYkFMoECFfS6q+fE1x\\\\\\\/9YXpYGSG1CnG991gWh5SaW1daKl8Ckx55B5RhEl74aSfXJfdNSH4YJLFHHyiJC719pOP9qSBmwm7x0RjDrqXLzNOMNIdRxn92VBgUrMWMnV2o0YfeZil02XsbyjDbFhupYGQx4wm+yn5ZTAY+FLsCYFXAHipifHEaPBQHKn5s7n0ykY1XxqXskAmkl6HcqdUEO4pQjXXrS0GZLi98xNNyAgytzqFtyv0OLMBDg0xlPCl2\\\\\\\/7I46u6nkowT6fCL8IGe32NoiEru\\\\\\\/SLvDkLesjo6xz5qNYOQ48N6PUnvLfYCblkhFKnuar+yOumJO75DrpXhQ\\\\\\\/qXAOtP\\\\\\\/bVSHmXNdA8gB5oOa4dclN3vBrO3p3hI6Io2YCL95XiD\\\\\\\/ZSYxr8oUuEYmG0bYrazW3CrnGLIo5U\\\\\\\/pdPMj0B54v+wBEj4rsQEK4GX6PdxEa3ZfLjvdKAY1nIkaHx9jI6PnFYsYfsTfZgkwCUYPVS4FfjW5tmPNXVGVvSzvV6HW61Uiw2642L2b1hpAkTomvE5gmXuVo1QBqyoJMdeFhpNrMEM\\\\\\\/R\\\\\\\/tUx3HtQRHmBRXq11BCSa6CtZFyMYaZjguK\\\\\\\/UFubZ+ssJm45PHUdbqGMkO25TFKmGBTCkyvY2F5ts+C0rOv1HtIgVTyJquEvntMwCKU1s2adZauBGfgymh9Etw0oK90SoEa+nD\\\\\\\/m1d+uAy2oZptEgUmcDrkNrGPmtWX\\\\\\\/\\\\\\\/8lrH3kBkfg9ikwedcMaggvBjKt\\\\\\\/TeMYUIVfxQgFVsw5QSC8pRLjLmtssE5QRFkj0lpeYSv+26lgcPZvH8q+eojj1TbeS8dbfPFbGwLXPj6wm5dGvwHIsi8IEUPJdumYkprJs4tuy+emXdGpu1mAhznuKRAP6Vw5tjDgWpR+5ZmTvDExMekL92fQ37kTwsdB+Y18DoEHOU\\\\\\\/YfS0y1+2uggKhIQLUnVd8ViDGydGttGV3DtAsnVzNxObU8UMhDk1ad3F\\\\\\\/qcKRg\\\\\\\/XVfIYXpNa\\\\\\\/X22k+ADauxaZg+5v0zMFpUus9D+K+GtjwLSKkyHwlYjTZnQ6EYKByejR\\\\\\\/NH8ZLmEB9dlxMck+U02\\\\\\\/84dDBFNJTKXvz0zW6JBiHtWKzRp0WXa4x98GQQI1nxqaVr2+8sxQ2FD2A9AtWMJxaY1R2JsJWKvNv9wKRptTS3j\\\\\\\/CAQiCK8GP\\\\\\\/x4om1CrzV6doUIv0Zh8kEnJdpOkxrm5KTtGR5+ojgltq7P3glCoNn6UuFC6DC64dhQhuL7pj7oIZ8TjWHAsv5\\\\\\\/6Ba\\\\\\\/qVIstZp2IOTqP42kkZe4jG5aZnIxtVGdJ\\\\\\\/iJd5tqC9o90tMJ2F+rCuWHh85Sho2eIa1H7K86ANsdEH2h0p2sYsxJ349ZE83pjdw8oJqSGqTc2ASFagN9FE83fZT6WaWnPx2H6bDxakpqw5W2BbNleLZ1jkzseEhfAqv5yKW9PCZsFmE8GCdOiXBIUfp3KEZxSyHlreJ1p+lEbN+tAg42pJlxPmbjpOgTffRaaD8tV+O8hWWyWxHaYcQAQI2Cxan47w9VxgQga+K8oNOUXF8N0fzkiaGdB9yLToenwTpETNL3oHMq5wq2cEAAMYFhh8TzV2LnJXz4HON5UjCL+AFhnoeRJovzZWbPvRz4bOQUDUZN\\\\\\\/GyZ0rtZJLJz7coZTRM4WB\\\\\\\/L6430umdZPw3jmc1BOpv7IyKSTrX\\\\\\\/9k\\\\\\\/hf5z92qVL3PA7CfIEyJ\\\\\\\/rtw8ujv39k3y2qzFklOI02m9CvCy4Tvus1ObaRy3qvw3ar6V8gtGe2GlGz\\\\\\\/9Yaxm\\\\\\\/ncVvO1of4pcK4sYaqyzLM5JgNScwtjfOXSKdtSynyIvkyj2ZRbbRpL6FIAuUcGPUfjyR+76GxTmnvnckHqsJSS80jVAu1KyUCNLTNkKHjudWpc72PffRBFGv6XFhrIuUNtfEoEr6QHyM39gIQ0k6AnYwyXpBBDvohC\\\\\\\/9QvE5hbxWnNbgiC1hU\\\\\\\/ithrUeNLuTo5qL3VPtgIXs+kHpgAY4+V\\\\\\\/JCjsvIrSoCJn7rDXjspoKSHpnDh5\\\\\\\/i9rCnEKRjTIVfFfuqBQC4vPN2KRr66YzOSgwRaN6MzK\\\\\\\/NQmjkbTHPSkqTXVf7f\\\\\\\/+nSAqGoWvhJkyI8OIilqxdDqjKpFRIekjEfejb8B1bcYG9XqDLYuX+IJktbLtnzDuRPdnUT7oWnyiFE2fDblRpLg4Q3JdoeGwNLVj++s5Uw0m3o+R2smvPrCaQkad2igJNvHxKScEQdRJ\\\\\\\/cuXq5G1p364xjdJqR7mlRXcrT6VTwbfq9R9P9nK2BfBdyL1OM0TVq54jD4GOvYzjo\\\\\\\/Lxz\\\\\\\/jqwfFVYAdsJMf0fimdUHcx385AT9yIoNYquYG8GAeV85s3y6hc9glBnjZYrkmwJMUs+y2VQhZiBzYPRhKNd9A9ymsUD0R7vMDqVWfsX0T0r\\\\\\\/aEMAxIId3PH2wmZCriavg77JRt2b2AdxaA9iFeZJkIphkMXlnEbCN713pGPmDmzbLGbIJPAe\\\\\\\/47wzkOuBDJNEpJdPmchUKHBlQJe9ugaR\\\\\\\/9O9ihAleyaTPjask1lkIgcIYvHhpRsovcoiaff7x38pH5bHJwkf7rAC1SAcsSuu8XC9Xvn5DrfBeLMl6rXR+Fp+e1vyJadpazoCBKCHs5wmJRszYfm7Z30EnWMYMVIkg5QBtm8s4khnQ1xzpBvJ+h9R8E\\\\\\\/5YMzwWuq2B783h7VLwgFlFgeK7vN85L\\\\\\\/FMZC11re45hCEeosZYD3XKYYHhecdof2qAwvd3fCKripncVxoPQIJ0eUHZU\\\\\\\/Yiy9fxdq+8apt8O6XpW7MVSN\\\\\\\/zHlmDQn2Yc\\\\\\\/0m5a69y0zV0R9dO3uD3PGBEXauilZWJaGeYPW5VK27TbR4\\\\\\\/YGhrejM2+9uhOrnjMUKKPeqlvF9hxnuqsnntnw2K7D+UFki1wS5Y6xJ33PRIMfXDA2F8kANTkmOQbVB\\\\\\\/1hnCeBErIpaJoxkt4UJl4n9U+bBXgHDfw2p1pbIpiisFLCxeTM\\\\\\\/C78Tzy6zsXaE7W2yuLujDT6+VfntPsYz0N5jsnW1JvRyHfmBqEdT9PoEeuDzvp\\\\\\\/K4iyKOZFJtpzyZLLN6ujOsXD2+N6wTJvc9UkMEnTB1lEqKZb3HI3MdeaT3KNaAHCLUxWJUeaFdUMc6gCCCs1vmtQQQgfeWIYwLuOLQ95C13wYVPkZD2SsfOp9Okq8i7+67PSLIdeG8XQP6ZIs8+icJAoXQckx4r2IrY8qwMVGNvkY5nGrmwN2fYxR0TE6PMsQ5RFE9VTn6imrCo\\\\\\\/l0TlRmSG74n\\\\\\\/axxVD2yeqj6l6fEGdMGsZrgORDuDSfFYjCtjFdxmUGWYZ5SzBlJOhkhPLEL4GxcAKyt6Tp9x5BGEVA05c1SjfGvLcc56krhbtEVS93l\\\\\\\/p2b5kbc3xEmyarZ11oXNCVkQuTkYnumbNIY2TI2QqfN\\\\\\\/gfTNizotIalNMEbvu+nsLSGmR\\\\\\\/vhcz6gYsiPHCQ5cEaamz9KZeHciAzmFCHkuk7BJQVWDmyQGY0AyDi\\\\\\\/z4ozg9qVo+3OXN9zjLPTP5M7D8dnmePKjPt0eKVJk3C1op2fw9meaWExN733Fb8DQRLnOKvnMyWg+WUznero\\\\\\\/rmK94HR6ommcnSVJ\\\\\\\/ZiIHffIm7e2tuvZrGeq19\\\\\\\/qo1y3A4Ph7Iyd5B8SkICzSRSb04TvskCFUIIzDwT54Yk78dou4Nk1dEwdj8HvI5g2Onud39B0Ao2gdn4Hfs2UxZ4\\\\\\\/htlw5dykKEkb\\\\\\\/JGvsMgli65CgOevAYQaQbeUcnO5d0v+uqYloQTcB\\\\\\\/TcC\\\\\\\/E8C9VsP5CjYlqrUBCWdopFgEVTNV2ze7HN4mQGTysZgMw7fUY\\\\\\\/FlaCyKP\\\\\\\/tvK43A\\\\\\\/wW4toyAQfbA7qUfB+3nTarLKjScTo9Tyhv8Ik8pN1J1e9L\\\\\\\/ncZQThOm1m83Y2USZ+VQaPLLtrGgsshQOlRaVDjCp\\\\\\\/U\\\\\\\/vM2AtmVcZkQTRXVrXEQYBOTd8vzgZZhOVeBV1C5CTAnjBAKj9+\\\\\\\/i8iFcM3BCG3lcb7NyfE5J1PlDbBwMoaL6pAJCHdlmz0467Lad6OitmXb7MQyh7UWGhkXaWBSvouvxjpv7hQfsj+BrGnuG6WITp1zcpHvDxyJtYVv7SmfqH1Ur\\\\\\\/62A\\\\\\\/DNl1fZn0RWoaSSVKHktdFPoexcEy+I+ofQROjeqlQ2Mxr29d5cMnpP\\\\\\\/SLWJkf6vdNp31DaMmBT7Rhwfz8DILrydL4hCUJg\\\\\\\/XcBikJiRzKierIW6mcfVxNYcTsyhtBsyVftNJyUUHm\\\\\\\/jvT7JrW\\\\\\\/yi2D6EmUxz3DrczAoDblnJ4dESxBncmPxxLGxzTkRXhixlmZcH14jhIm4N1vGsAdZDWxGdPNWL3lCyZxlGO7cj9w9eckw4zApSXlD+WdhsZTvE9yPB\\\\\\\/81f71KgL1Qi3FMd1yQdlaXq0rjyYj6rR+dHxY2Z2DCUC+UUhW0x\\\\\\\/VULWRVv9W9vb5r+WgDmKqE\\\\\\\/Jy11wEQ0eDbqoNNrBvmsIrHklQrE5KNjT+bRfoEzLrkIzXUNVcDXrYSKggRAo2UiLbZjynzDhrxH2ptm1RyFqQc+v+sl4JbN4cnOsLWpedaTcXQEBz7mXcZf0G8Fc5wIWFnR\\\\\\\/+NQMk7jBuTkdaUZFo5GAMKnLLR6n6kTEwX2zbMbClKNpd\\\\\\\/S+b2noTXGPBV\\\\\\\/athdzV70Ald7pEzjfbtJGipU8rV5rmy3gNboZKJM1PJbqo3LILI7H9p6bN8XJZAsHVk3+rARmnNkgPlGTJNW9HOS7BgBzu\\\\\\\/eYgaYEgQy5k3qVSdE0biWKYY73JxN49g7QjRFJnsCdSum1BZ+7ag0kEhPLyTC5xidpEOgIe3tTBR4Ze6K4+2DKs3I8hDXxstEVHRE5XhwrZW9jouegA\\\\\\\/zu5rjQ5guLrsKh44RBykDd2nXIRBhmJiQIa0kZp2lY1lds81g4le7vYvCgCtA9DEogOMBv9Fh6fiZsFyljKtLWV7DP0yAGQGFTfO5NMrlgSHOIu6nmjrKUvM798Ji5NWryg6HrDAW8NklmQfqOiEV+WIloFbEyFL9O+OlqcL74v4sEK+ohsNE1LX66rcC5QZzIk8okdzOECLnAEzE\\\\\\\/9iVdVnc1LYoDWaY+5OPozcNzUw\\\\\\\/rlBKphNloCs7kGJLTjWW37PlCDnYKkkUynJ8ymLieyKaolf\\\\\\\/jffRJD1GdtMjX0SgTblyheUqIjm7OcSEO0LJ11ZQophDffpK09xrltzRUOivKSa3Z8ILHiOFxJZ3pA8VZe5a5LG9WTiXeGxGMtGghnNh6hfrZTsLl0MbY8rr+nuHb0ZA5nOPD98MbExaCGLt6HC\\\\\\\/YAE7pxccK7Rt++2nXt+U+HkOGL7IlVB5ggYxYYeEubKjgnlpU9CWPUDfen\\\\\\\/DgZmaggz3rVPxdGl29GrkwPYe2i9qiV8Qd+RgCrds2vuT5ERYsdvJW9L0BKFd7iRuU\\\\\\\/vAwdUgqfsHFCXQZEUukJfH3cElXEq3ay9g84GHitz9Nycrklg7RuOTasbc6UtzZihwMmsv95TbMpeIos7HNsaNQb9HR44DgPPvrmyxTlW06NrI8EsDkbnTqEALjGhrTEY9Zz2IM7fUEVl1xcjfZ+62WEJG0ty+Vl5IFfL6pWN+aL7wPIujgFveP6h3cG3SpbVu2OQiPJ+f2SrsgA22Bi306xhHcBguxuFfwXmlC2FRjbzdYBxLII\\\u0022,\\\u0022iv\\\u0022:\\\u0022sanHP8wfL\\\\\\\/GTw3j3zdSoyg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f40cb0ba9b0cb28bc5dbc6a905a24cfae23b6a59c51397c28f7aae7aa1e8d322\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RKcA9JC8OyfsU1ekA4ZFPwt5e5f9STFF9v3u13LQNY4I8tnZ5rvl0ehCUz6OGYYLgx3B4K1FQbdfa2g\\\\\\\/2P3AXJaObGka3bt3+nGypkZ4ouPDcXTaHsZPAvt3D4OVo+5hcAWFODebgZob2O4T3KjpQhB\\\\\\\/9g6RA5ESdzgLGWE4n8vW7Ni5kKWLbwoYlkeg\\\\\\\/\\\\\\\/c6Oyw4iwyo2GmtN+cStVW6Y4cXQ11FFMu4HSy7YtgLAcYwjBEyoa6FqcLgWTvl1C1JiU29ZK5H\\\\\\\/y2xSy3weI2DX7xAIGgK4be4JS98HbTJztLXvga+wL0A6wmC8JY681HMCHmB5k1ZAcFEDyGz95HxaFlSDs3biPnXyI6QwhiVbfn4BmN7lyJM5o4\\\\\\\/NcZpdLC0yniGt8sVnErL556DPP3YIgq2DSKUo\\\\\\\/00ocHcULbS9db3uSuLhteYJup1xWlMDAsG3TWKeplPfesDxoSEovvls4EnfC2MGzDOCO22Sy6WbbPPklQD8exDVeYBrNJ8JpqOEr1yZwjQ6Ltv1BLXlt1q9g7QCgQS4cQjLldSRvF5hrNXZztZZANGa3M\\\\\\\/2eHUQlKJclxxnxRJN4Fz3cB3rr+gBiP6+3CVc4DtlpiBkLnyvvh0xnCt\\\\\\\/iiHsyHTHU2W1YDF5Z6HQl\\\\\\\/l3IHxMTIwoHKhBfn3SCOfHbNRNnZHptdRYR3y78GT5fEDub55q+Z1AAIQ+pwfJKp0Cs5FA1SbWRuaYF9DAOyqQIhGTpbc\\\\\\\/vSFurL22rZRtmlp7vH\\\\\\\/mD5CDhEf2LM8ro63i6WYff8dsEvOoaIJKnI4ay6o9omCwYTEcyY8ZRCEH6d8RmE5e9rNWhWiyH2XsdPQKtJ0+gOZ8ajvaQWvVCFt0MMrcHKTxWO0fXP0b+dpGvv8ksfP2e\\\\\\\/SXf0KioCJk8pf0PHie+66semlqNXi8xtI2aYIZyVtOPt91L5c0\\\\\\\/YvAa77Cf8ZujP4cEjHQ35sADUi6V5LF3OsKh+Px4BijFB7DJ5LbOzUd+cr0ZHfpcVWLnlfHfK+RHEDR5MSk8syqvHtM2A2O9YBA7WN7q4A2ia+lqvsll9+dUXOEugHXN2h\\\\\\\/5rIVPsLAr610rlmK853QXyrk+KGtnK6rhAvtERjQWuWnWR6UclCbpS11dnGgEvpQ4LeXj44Sn7ZGFZKynr9PHWGeUae5w2uP1odbXxAb5JNw\\\\\\\/AAAc5N8\\\\\\\/ibBHTAKR\\\\\\\/oq7fSolfglwxTKNRbVyG0W5NLx7ySfSqyC2AlNNC\\\\\\\/fWUCrsMxbyjTozLpxTJPunGF4IUlOB8Rj1Tm1uqhcIi2kabd0e1N1qxk+oEdmp6ekI1Zuwh2Dspm9FMe5F9L+dQdmOS5TtwnbRk1Um2SiMszQSraZGWPgXSkL+KrzASIaMNukBpY8kMqvjdAHKDXndMGGJTNreqk2+GGWtpttze4onpdqAtvDhFB4hROk97\\\\\\\/IWTvK2mQ7jNBv8USaaYUtcy+UG09UvC5oGXaEvJGh1+OSqAxiuYDQzYuSJn0yGB0HSW4Uv71Wx7oCInSxO4H1nap+KiY0cx4mW+\\\\\\\/HzTuRf92Yn8Ud764oPdFSUWooFF\\\\\\\/0iswEKfJojlsMNUJXQ6Ou1zBnkQghTombvguPW3ZXmn4Ag8o0dyPbIPsSPW9wlNMqZ6fM\\\\\\\/BxrwreHm5KO5R6fRJfpEeysBCruz7MF8wW1AD+jwTvw49k77U74mbdGaF9iZHg\\\\\\\/3XTPn059JiX\\\\\\\/k\\\\\\\/gMXjXxc7ZLDdGmkQ5f6QVhVaIrx6JG7fSZ+eTiyO0tEeQaZHJkIibeaqFvKwCrMa0ZLgvW15yWIqHgudH+24XlzTeGkED2B7crttBLrH23ASzHjUpOZgN\\\\\\\/p8KYb7NZZa6HQtE+JSOv9r4ZOnKsKG3lXwfK7QAAXTJ+IQrxhSCXI3aJWCwkyM3sYTlRSDcMpCzZAWA2GCQhz9pmYulhgWe6sbnGZFg4RV8rQMZmHixas8b1xYxNW\\\\\\\/t6D1Vh53Vtqij6aQ9e1ahxaJnJR3T7hjRJMrcLWaFvNU+3PQvUCe\\\\\\\/lSRgxi8CFTJOcEeRTCaGZB4=\\\u0022,\\\u0022iv\\\u0022:\\\u0022NxXb\\\\\\\/Y+A2nzIYSAZeGMSiA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226aff8ace2ac7c51e415a93c1e3853bb10aa3161271b5aeb1054168504d3fc925\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459157 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/1/57" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9-BgGzzMxlqru1R9RLiTV5ADy3oLYG8i6Sf4mOm9RWw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/1/57" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XRj4HG1gybUvcmltzJMU9UgRdXwOqq7MRpv8dQxK1ZY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/1/43" title="Identification of Yeast Genes Involved in K+ Homeostasis: Loss of Membrane Traffic Genes Affects K+ Uptake" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/1/65" title="Fine Mapping of a QTL for Fertility on BTA7 and Its Association With a CNV in the Israeli Holsteins" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459157" id="node459157" data-pisa="ggg;1/1/57" data-pisa-master="ggg;g3.111.000109" data-apath="/ggg/1/1/57.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">An Improved Method for Estimating Chromosomal Line Origin in QTL Analysis of Crosses Between Outbred Lines</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lucy Crooks</span>, <span class="highwire-citation-author" data-delta="1">Carl Nettelblad</span> and <span class="highwire-citation-author" data-delta="2">Örjan Carlborg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">57-64;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000109 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459157" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lucy Crooks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Cell and Molecular Biology, Uppsala University, SE-751 24 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucy%2BCrooks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crooks%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALucy%2BCrooks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lucy.crooks@icm.uu.se" class="" data-icon-position="" data-hide-link-title="0">lucy.crooks@icm.uu.se</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Carl Nettelblad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Information Technology, Uppsala University, SE-751 05 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carl%2BNettelblad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nettelblad%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarl%2BNettelblad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Örjan Carlborg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Cell and Molecular Biology, Uppsala University, SE-751 24 Uppsala, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C3%96rjan%2BCarlborg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carlborg%20%C3%96&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3A%25C3%2596rjan%2BCarlborg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/1/57" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459157" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459157/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/1/57.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459157" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459157/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/1/57.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459157" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459157/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/1/57.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459157" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459157/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Estimating the line origin of chromosomal sections from marker genotypes is a vital step in quantitative trait loci analyses of outbred line crosses. The original, and most commonly used, algorithm can only handle moderate numbers of partially informative markers. The advent of high-density genotyping with SNP chips motivates a new method because the generic sets of markers on SNP chips typically result in long stretches of partially informative markers. We validated a new method for inferring line origin, triM (<span class="underline">tr</span>acing <span class="underline">i</span>nheritance with <span class="underline">M</span>arkov models), with simulated data. A realistic pattern of marker information was achieved by replicating the linkage disequilibrium from an existing chicken intercross. There were approximately 1500 SNP markers and 800 F<sub>2</sub> individuals. The performance of triM was compared to GridQTL, which uses a variant of the original algorithm but modified for larger datasets. triM estimated the line origin with an average error of 2%, was 10% more accurate than GridQTL, considerably faster, and better at inferring positions of recombination. GridQTL could not analyze all simulated replicates and did not estimate line origin for around a third of individuals at many positions. The study shows that triM has computational benefits and improved estimation over available algorithms and is valuable for analyzing the large datasets that will be standard in future.</p></div><span class="kwd-group-title">Keywords</span><ul class="kwd-group"><li class="kwd"><a href="/keyword/interval-mapping" class="hw-term hw-article-keyword hw-article-keyword-interval-mapping" rel="nofollow">interval mapping</a></li><li class="kwd"><a href="/keyword/outbred-line-cross" class="hw-term hw-article-keyword hw-article-keyword-outbred-line-cross" rel="nofollow">outbred line cross</a></li><li class="kwd"><a href="/keyword/line-origin-probabilities" class="hw-term hw-article-keyword hw-article-keyword-line-origin-probabilities" rel="nofollow">line origin probabilities</a></li><li class="kwd"><a href="/keyword/hidden-markov-model" class="hw-term hw-article-keyword hw-article-keyword-hidden-markov-model" rel="nofollow">hidden Markov model</a></li><li class="kwd"><a href="/keyword/snp-chip" class="hw-term hw-article-keyword hw-article-keyword-snp-chip" rel="nofollow">SNP chip</a></li></ul><p id="p-3">Analysis of experimental line crosses using genetic markers has been successful in identifying quantitative trait loci (QTL) for a wide range of physical, behavioral, and disease traits in plants, model organisms, and livestock species (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Corva and Medrano 2001</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Mackay 2001</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Maloof 2003</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Hocking 2005</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Rothschild <em>et al.</em> 2007</a>). Ultimate aims of QTL analyses include determining the number of genes influencing a trait and the nature and size of their effects, identification of important pathways in disease, and discovery of genes that may be useful in breeding programs. Resolving QTL to the level of the causative gene has been challenging, although there are some notable examples: <em>IGF2</em> affecting muscle growth in pigs (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Van Laere <em>et al.</em> 2003</a>), <em>ORFX</em> for fruit size in tomatoes (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Frary <em>et al.</em> 2000</a>), and <em>achaete-scute</em> influencing bristle number in <em>Drosophila melanogaster</em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Long <em>et al.</em> 2000</a>). An obstacle to identifying the underlying genes has been the relatively large size of QTL regions, which can encompass hundreds of genes. For line crosses, ultimately the lower limit on the size of QTL regions is set by the number of recombinations that have occurred in the pedigree. However, to achieve the best resolution and statistical power within this constraint, it is important to extract the maximal possible information from the genetic markers.</p><p id="p-4">In linkage-based QTL analyses, such as with line crosses, marker genotypes are used to infer the inheritance of chromosomal segments through the pedigree. For line crosses, the key factor is the line origin of segments (<em>i.e.</em> whether they are inherited from a founder individual of line 1, or a founder individual of line 2). Most analyses apply interval mapping (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Lander and Botstein 1989</a>) where an association between line origin and phenotype is tested at regularly spaced positions along the genome. Many of these positions will not coincide with markers that unambiguously indicate the line origin (fully informative markers), in which case the probability of having each line origin is estimated based on the genotypes of available markers and the expected frequency of recombination between these markers and the test position. With inbred lines, all selected markers are fully informative, and only the genotypes of the two markers flanking the test position are needed. For outbred lines, however, many markers are likely to be only partially informative (more than one line origin is compatible with the genotype although at least one line origin can be excluded) because the same marker alleles are segregating in both lines. <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Haley <em>et al.</em> (1994)</a> showed that for outbred lines, the genotypes of all partially informative markers up to the nearest fully informative marker on each side should be considered. They proposed an algorithm that separately evaluates the probability of each possible combination of line origins between fully informative markers, then sums and normalizes the results. To date, this algorithm, or variants of it, has been the most commonly used method for estimating line origin probabilities in outbred line crosses.</p><p id="p-5">The use of SNPs has become standard in QTL mapping, and because of their high frequency in the genome, the number of scored markers has increased considerably from older technologies. A recent advance is the SNP chip, which allows parallel genotyping of hundreds of thousands of selected SNPs. These SNPs are typically chosen from a few populations of interest. As a result, some will not be fully informative in studies involving other populations. However, because so many SNPs are included, there will still be many that are at least partially informative. The advantage of the chips is that they remove the need for time-consuming development, selection, and testing of markers for each study and provide a standard set of markers with known location, allowing direct comparisons between studies. Chips of at least 50,000 SNPs are currently available for cow, pig, dog, chicken, sheep, mouse, and maize, for example. The <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Haley <em>et al.</em> (1994)</a> algorithm worked well for the small numbers of markers that were used at the time. However, the maximum number of calculations required by the algorithm scales exponentially with the number of partially informative markers in stretches with no intervening fully informative markers. On most computers, the algorithm crashes when there are more than 20 partially informative markers in such stretches (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Nettelblad <em>et al.</em> 2009</a>). Therefore, it is not suitable for handling SNP chip datasets. To utilize all the information available from SNP chips, a new method is needed.</p><p id="p-6">We recently developed a new tool for estimating line origin probabilities. The algorithm scales logarithmically with the number of markers included, making analysis of very large datasets feasible. We have designated this method triM (<span class="underline">tr</span>acing <span class="underline">i</span>nheritance with <span class="underline">M</span>arkov models), which is implemented in the existing codebase cnF2freq (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Nettelblad <em>et al.</em> 2009</a>). Here we evaluate the performance of triM on simulated datasets of approximately 1500 SNPs of mixed information content. Although the original <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Haley <em>et al.</em> (1994)</a> algorithm cannot deal with such datasets, they can be analyzed if the algorithm is modified to include information from only a subset of markers. We compare the behavior of triM to one such modified version of the <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Haley <em>et al.</em> (1994)</a> algorithm incorporated in the web-based tool GridQTL (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Seaton <em>et al.</em> 2006</a>). The size of these datasets is at the reported upper limit for GridQTL, but we emphasize that triM can easily analyze much larger datasets than this.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>triM</h3><p id="p-7">triM employs an algorithm developed for hidden Markov models (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Rabiner 1989</a><span class="sc">)</span>. Using these models to track the parental origin of alleles in linkage mapping was introduced by Lander and Green<span class="sc"> (</span>1987). Specific application of hidden Markov models to QTL analysis in intercrosses was implemented in R/QTL (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Broman <em>et al.</em> 2003</a>) but limited to the cases of inbred or haplotyped data. The underlying genetic model assumed by triM is the same as for the <a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Haley <em>et al.</em> (1994)</a> algorithm, but the advance in triM is full exploitation of the Markov property. Results from triM are expected to be identical to those that would be obtained from the <a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Haley <em>et al.</em> (1994)</a> algorithm, were the calculations possible.</p><p id="p-8">A Markov model consists of a series of states where the state at any point depends only on the immediately preceding state and not any of the states before that. Line origin along a chromosome is a Markov process if the frequencies of recombination in each interval are independent of each other. triM therefore applies Haldane’s mapping function, which fulfills the Markov property by assuming no interference between crossovers. The extension to a hidden Markov model is that the states themselves are not observable but influence the values of a second variable, which is observed. In this context, the state is the line origin and the output variable is the marker genotype. The line origin probability at a test position is calculated by iteratively building up the probability of each line origin at successive marker positions, working forward and backward from both ends of the chromosome. For each marker position, only the probabilities of moving from the possible line origins at the preceding marker are needed; all the previous transition probabilities are contained in the line origin probabilities at the preceding marker. The calculation therefore collapses back to a limited number of operations at each marker, which is the basis of its efficiency. The forward and backward probabilities for each test position are multiplied together and normalized by the product summed over all origins. The principle of the algorithm is illustrated in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, and additional details are given in Nettelblad <em>et al.</em> (2009). triM is available for download at <a href="http://www.computationalgenetics.se">http://www.computationalgenetics.se</a> under “Software.” We are currently developing a software package in the statistical environment R, incorporating triM, that will perform the complete process of QTL analysis for crosses between outbred lines.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/1/57/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Outline of the forward-backward algorithm used in triM. (A) Schematic of line origin states along a section of chromosome flanked by two fully informative markers. Positions 1-5 are markers, and t is the position being tested as a QTL. Circles represent the possible line origins, where i,j symbolizes that the maternal haplotype originates from line i and the paternal haplotype from line j. Black filled circles show line origins that are compatible with the marker genotypes and pedigree. At t, all line origins are treated as compatible, shown by the gray filled circles. (B) Calculation of , the combined probability of having line origin i,j at position p and the observed marker genotypes from positions 1 to p, in the forward step. rp is the recombination frequency between position p and the next position. Only  and rp&#x2212;1 are needed in each iteration. For clarity, calculations are only shown for one line origin. Although only compatible line origins are included in the example, triM considers incompatible origins with a low probability to allow for genotyping errors. (C) Calculation of , the probability of having the observed marker genotypes from position p +1 to the end, given that the line origin is i,j at position p, in the backward step. Only  and rp are needed in each iteration. Again, calculations are shown for only one line origin. Having obtained  and , the probability of line origin i,j at the test position is given by . (D) Representation of the number of operations needed for the forward-backward algorithm in this example. (E) For comparison, a representation of the number of operations required under the original algorithm for the same example." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1270635032" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Outline of the forward-backward algorithm used in triM. (A) Schematic of line origin states along a section of chromosome flanked by two fully informative markers. Positions 1-5 are markers, and &lt;em&gt;t&lt;/em&gt; is the position being tested as a QTL. Circles represent the possible line origins, where &lt;em&gt;i&lt;/em&gt;,&lt;em&gt;j&lt;/em&gt; symbolizes that the maternal haplotype originates from line &lt;em&gt;i&lt;/em&gt; and the paternal haplotype from line &lt;em&gt;j&lt;/em&gt;. Black filled circles show line origins that are compatible with the marker genotypes and pedigree. At &lt;em&gt;t&lt;/em&gt;, all line origins are treated as compatible, shown by the gray filled circles. (B) Calculation of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-1&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, the combined probability of having line origin &lt;em&gt;i&lt;/em&gt;,&lt;em&gt;j&lt;/em&gt; at position &lt;em&gt;p&lt;/em&gt; and the observed marker genotypes from positions 1 to &lt;em&gt;p&lt;/em&gt;, in the forward step. &lt;em&gt;r&lt;sub&gt;p&lt;/sub&gt;&lt;/em&gt; is the recombination frequency between position &lt;em&gt;p&lt;/em&gt; and the next position. Only &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-2&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;em&gt;r&lt;sub&gt;p&#x2212;1&lt;/sub&gt;&lt;/em&gt; are needed in each iteration. For clarity, calculations are only shown for one line origin. Although only compatible line origins are included in the example, triM considers incompatible origins with a low probability to allow for genotyping errors. (C) Calculation of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-3&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, the probability of having the observed marker genotypes from position &lt;em&gt;p&lt;/em&gt; +&lt;em&gt;1&lt;/em&gt; to the end, given that the line origin is &lt;em&gt;i&lt;/em&gt;,&lt;em&gt;j&lt;/em&gt; at position &lt;em&gt;p&lt;/em&gt;, in the backward step. Only &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-4&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;em&gt;r&lt;sub&gt;p&lt;/sub&gt;&lt;/em&gt; are needed in each iteration. Again, calculations are shown for only one line origin. Having obtained &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-5&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-6&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, the probability of line origin &lt;em&gt;i&lt;/em&gt;,&lt;em&gt;j&lt;/em&gt; at the test position is given by &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-7&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. (D) Representation of the number of operations needed for the forward-backward algorithm in this example. (E) For comparison, a representation of the number of operations required under the original algorithm for the same example.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/1/57/F1.medium.gif" width="248" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/1/57/F1.medium.gif" width="248" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/1/57/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/1/57/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460692" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-9" class="first-child">Outline of the forward-backward algorithm used in triM. (A) Schematic of line origin states along a section of chromosome flanked by two fully informative markers. Positions 1-5 are markers, and <em>t</em> is the position being tested as a QTL. Circles represent the possible line origins, where <em>i</em>,<em>j</em> symbolizes that the maternal haplotype originates from line <em>i</em> and the paternal haplotype from line <em>j</em>. Black filled circles show line origins that are compatible with the marker genotypes and pedigree. At <em>t</em>, all line origins are treated as compatible, shown by the gray filled circles. (B) Calculation of <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/F1/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/F1/embed/mml-math-1.gif"/></noscript></span></span>, the combined probability of having line origin <em>i</em>,<em>j</em> at position <em>p</em> and the observed marker genotypes from positions 1 to <em>p</em>, in the forward step. <em>r<sub>p</sub></em> is the recombination frequency between position <em>p</em> and the next position. Only <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/F1/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/F1/embed/mml-math-2.gif"/></noscript></span></span> and <em>r<sub>p−1</sub></em> are needed in each iteration. For clarity, calculations are only shown for one line origin. Although only compatible line origins are included in the example, triM considers incompatible origins with a low probability to allow for genotyping errors. (C) Calculation of <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/F1/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/F1/embed/mml-math-3.gif"/></noscript></span></span>, the probability of having the observed marker genotypes from position <em>p</em> +<em>1</em> to the end, given that the line origin is <em>i</em>,<em>j</em> at position <em>p</em>, in the backward step. Only <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/F1/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/F1/embed/mml-math-4.gif"/></noscript></span></span> and <em>r<sub>p</sub></em> are needed in each iteration. Again, calculations are shown for only one line origin. Having obtained <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/F1/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/F1/embed/mml-math-5.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/F1/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/F1/embed/mml-math-6.gif"/></noscript></span></span>, the probability of line origin <em>i</em>,<em>j</em> at the test position is given by <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/F1/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/F1/embed/mml-math-7.gif"/></noscript></span></span>. (D) Representation of the number of operations needed for the forward-backward algorithm in this example. (E) For comparison, a representation of the number of operations required under the original algorithm for the same example.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Simulation</h3><p id="p-10">To evaluate triM with a realistic pattern of marker information, the simulation was based on data from a three-generation chicken intercross (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Kerje <em>et al.</em> 2003</a>) where dense SNP genotypes were available for two families (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Groenen <em>et al.</em> 2009</a>). These families originated from one Red Junglefowl male and three White Leghorn females and contained 23 and 25 F<sub>2</sub> individuals, respectively. We simulated a single chromosome using the data from chromosome 1. Markers with missing genotypes in the parents or grandparents were removed. Some additional markers were also removed because of inheritance errors or uncertainty in their order, and a small number of genotype corrections were made, leaving a total of 1508 markers covering 451 cM. Genetic positions of markers were taken from the sex-averaged consensus linkage map (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Groenen <em>et al.</em> 2009</a>) where possible. The linkage positions were not used in a few cases where the order of one or two markers differed between the linkage map and the genome assembly. Remaining genetic positions were estimated from the nearest flanking markers in the linkage map, assuming a linear relationship between genetic and physical positions over the interval. Markers that had the same genetic position <em>x</em> were separated by spacing them evenly from <em>x</em>-0.05 to <em>x</em>+0.049 cM. The average distance between markers was 0.3 cM. The biggest gap between markers was 8 cM, between the second and third, and the third and fourth markers. There were 28 gaps of 1-4 cM; the remaining distances between markers were less than 1 cM.</p><p id="p-11">The pattern of marker information in an F<sub>2</sub> individual depends on both the genotypes of the grandparents and how often successive markers that are heterozygous in a grandparent or parent, are heterozygous in their offspring. Essentially, it is determined by which alleles occur together on the same copy of a chromosome (the haplotypes) in the grandparents. It was not possible to unambiguously haplotype the grandparents because they had too few genotyped offspring. Therefore, we instead haplotyped the parents, and treated the haplotype transmitted to the parent as one haplotype in the grandparent, with the remaining alleles forming the second haplotype. When there were several parents that could be haplotyped for a given grandparent, one was chosen at random. Haplotyping was performed by the following procedure. First the allele originating from each parent was determined for the straightforward cases of markers that were homozygous or heterozygous with at least one parent homozygous. For the other heterozygous markers, the parental origin was inferred by looking at the combination of F<sub>2</sub> genotypes at the marker and a second heterozygous marker where the origin was known. The second marker was chosen so that the set of F<sub>2</sub> genotypes would differ according to which alleles were inherited together from the same parent.</p><p id="p-12">Genotypes of parents and then F<sub>2</sub> individuals were simulated from the grandparent haplotypes. The number of crossovers was sampled from a Poisson distribution with mean equal to the chromosome length in Morgans. The positions of each crossover were then sampled from a uniform distribution covering the length of the chromosome. A starting haplotype was chosen at random. Moving along the chromosome, alleles were taken from this haplotype until a recombination position was reached. Then alleles were taken from the second haplotype. This continued, with the sampled haplotype changing after each crossover, to the end of the chromosome. Individuals were generated using the pedigree structure of the original three-generation intercross (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Kerje <em>et al.</em> 2003</a>). There were four male and 37 female parents and 773 F<sub>2</sub> individuals.</p></div></div><div class="section other1" id="sec-4"><h2 class="">Analysis</h2><p id="p-13">Line origin probabilities were estimated at 1-cM intervals. Both methods estimate four probabilities for each F<sub>2</sub> individual and position, the product of two possible line origins for each haplotype. triM was used to obtain probabilities for 1000 replicates of the complete dataset. Attempts were then made to analyze the data using GridQTL, version 1.4.1. We were unable to obtain results from GridQTL for several tested replicates although the online documentation stated that GridQTL could handle around this number of markers. Therefore, we experimented with datasets of reduced size, produced by successively removing the last 100 markers. With 1200 markers, GridQTL produced results for nine of the first 10 replicates. Because of the difficulties in obtaining results from GridQTL, we compared triM and GridQTL for these nine replicates. To avoid possible artifacts at the end of the chromosome, as triM estimates were based on all 1508 markers, we compared only the region covered by 1150 markers, excluding the last 50 markers available to GridQTL, which was 333 cM. For many individuals, there were regions where GridQTL reported all line origin probabilities as zero (<em>i.e.</em> where probability estimates were missing). Even though these are the cases that have the largest impact on the power and precision for subsequent QTL analyses, we have not included these probabilities in our comparison. Only data points for which there were estimates from both GridQTL and triM were compared. Both methods were timed. For GridQTL, timing estimates were less accurate, because the viewer window in GridQTL had to be manually refreshed to check if an analysis had finished. The following two aspects of the results were considered.</p><div id="sec-5" class="subsection"><h3>Line origin error</h3><p id="p-14">From the probabilities estimated by each method, we calculated the overall probabilities that each haplotype originates from line 1. The probabilities given by triM are, in order, (i) both haplotypes originate from line 1, (ii) the maternal haplotype originates from line 1 and the paternal haplotype from line 2, (iii) the maternal haplotype originates from line 2 and the paternal haplotype from line 1, and (iv) both haplotypes originate from line 2. In the results from GridQTL, the order of probabilities (ii) and (iii) are switched. Probabilities (i) and (ii) were summed for the maternal haplotype, and (i) and (iii) were summed for the paternal haplotype. The true line origins are specified in the simulation. For each haplotype, we define the line origin error at position <em>i</em>, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/embed/mml-math-8.gif"/></noscript></span></span>, as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/embed/mml-math-9.gif"/></noscript></span>,</span><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/embed/mml-math-10.gif"/></noscript></span>,</span>where <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/57/embed/mml-math-11.gif"/></noscript></span></span> is the probability of line 1 origin at position <em>i</em>. The average and standard deviation in line origin error was calculated over both haplotypes for all individuals.</p></div><div id="sec-6" class="subsection"><h3>Positions of recombination</h3><p id="p-15">Inferred recombination events in the parents can be seen as changes in haplotype line origin probabilities from close to zero to nearly one and vice versa. Only such switches that covered the range of probabilities from 0.025-0.975 were considered; the positions where the probabilities exceeded 0.975 and fell below 0.025 were called the end points of the switch. Switches from GridQTL were not included if there were missing probability estimates within the switch. We defined two summary measures: imprecision and inaccuracy. Imprecision in an estimated recombination position is the distance between the end points of a switch. It quantifies the amount of uncertainty in the estimated position; high imprecision means the method locates a broad region in which recombination has occurred, low imprecision that an interval or short stretch has been pinpointed. Inaccuracy in an estimated recombination position is the distance between the true recombination position and where the line origin probabilities cross 0.5. Occasionally, several consecutive positions had a probability of 0.5; in these cases, the distance to the central position was used. The minimum possible imprecision is 1 cM; the minimum inaccuracy is 0 cM. Most of the largest line origin errors are likely to be where recombination positions have been estimated with high imprecision or inaccuracy.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><p id="p-16">The average line origin error for triM over 1000 simulation replicates ranged from 0.0009 to 0.04 for most of the simulated chromosome (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The error was higher at the start of the chromosome, where the distance between the markers was largest, reaching an average of 0.12 at 12 cM. This position is halfway between one of the pairs of markers that are separated by 8 cM, with the second marker being homozygous for the same allele in all four grandparents, and the low marker information limits how accurately line origin can be estimated. The average line origin error across all positions was 0.017. The actual haplotype line origin corresponds to a probability of either 1 or 0; therefore, the average error is 2% of the range of possible values. The average standard deviation in line origin error within a replicate was 0.08 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). As with the average error, the average standard deviation in the error was higher at the start of the chromosome.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/1/57/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Line origin error for triM. (A) Average line origin error at each position over both haplotypes and all individuals. Dots show the average for 1000 simulation replicates, and gray bars indicate the minimum and maximum average line origin error obtained for a single replicate. (B) Standard deviation in line origin error at each position. Dots show the average standard deviation for 1000 simulation replicates, and gray bars indicate the minimum and maximum standard deviations obtained for a single replicate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1270635032" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Line origin error for triM. (A) Average line origin error at each position over both haplotypes and all individuals. Dots show the average for 1000 simulation replicates, and gray bars indicate the minimum and maximum average line origin error obtained for a single replicate. (B) Standard deviation in line origin error at each position. Dots show the average standard deviation for 1000 simulation replicates, and gray bars indicate the minimum and maximum standard deviations obtained for a single replicate.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/1/57/F2.medium.gif" width="276" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/1/57/F2.medium.gif" width="276" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/1/57/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/1/57/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460657" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-17" class="first-child">Line origin error for triM. (A) Average line origin error at each position over both haplotypes and all individuals. Dots show the average for 1000 simulation replicates, and gray bars indicate the minimum and maximum average line origin error obtained for a single replicate. (B) Standard deviation in line origin error at each position. Dots show the average standard deviation for 1000 simulation replicates, and gray bars indicate the minimum and maximum standard deviations obtained for a single replicate.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">GridQTL had a higher average line origin error over nine replicates than triM at all positions (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The difference varied from 0.00001 to 0.007, with an average across all positions of 0.0019. This is equivalent to a 10% higher average error for GridQTL than triM.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/1/57/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average difference in line origin error at each position between GridQTL and triM. Only values from triM that were also estimated by GridQTL are included. The end of the chromosome was excluded to avoid artifacts due to the additional marker information used by triM. Positive values mean that the average line origin error was higher for GridQTL than triM; dashed line indicates zero. Dots show averages for nine replicates. Gray bars indicate the minimum and maximum difference in line origin error obtained for a single replicate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1270635032" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Average difference in line origin error at each position between GridQTL and triM. Only values from triM that were also estimated by GridQTL are included. The end of the chromosome was excluded to avoid artifacts due to the additional marker information used by triM. Positive values mean that the average line origin error was higher for GridQTL than triM; dashed line indicates zero. Dots show averages for nine replicates. Gray bars indicate the minimum and maximum difference in line origin error obtained for a single replicate.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/1/57/F3.medium.gif" width="440" height="395"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/1/57/F3.medium.gif" width="440" height="395"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/1/57/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/1/57/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460761" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-19" class="first-child">Average difference in line origin error at each position between GridQTL and triM. Only values from triM that were also estimated by GridQTL are included. The end of the chromosome was excluded to avoid artifacts due to the additional marker information used by triM. Positive values mean that the average line origin error was higher for GridQTL than triM; dashed line indicates zero. Dots show averages for nine replicates. Gray bars indicate the minimum and maximum difference in line origin error obtained for a single replicate.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">The percentage of individuals missing probability estimates from GridQTL at each position is shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. It was between 15 and 52%, with an average over all positions of 35%. For every replicate, over 30% of individuals had missing probabilities in at least 2/3 of the positions.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/1/57/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Missing probability values at each position from GridQTL. The percentage of individuals given zero probabilities for all four line origins is shown. Dots show the average for nine replicates. Gray bars indicate the minimum and maximum percentage of individuals with missing values for a single replicate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1270635032" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Missing probability values at each position from GridQTL. The percentage of individuals given zero probabilities for all four line origins is shown. Dots show the average for nine replicates. Gray bars indicate the minimum and maximum percentage of individuals with missing values for a single replicate.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/1/57/F4.medium.gif" width="440" height="399"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/1/1/57/F4.medium.gif" width="440" height="399"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/1/57/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/1/57/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460726" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-21" class="first-child">Missing probability values at each position from GridQTL. The percentage of individuals given zero probabilities for all four line origins is shown. Dots show the average for nine replicates. Gray bars indicate the minimum and maximum percentage of individuals with missing values for a single replicate.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">The analysis times for triM and GridQTL are shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. triM completed each analyses in seconds whereas GridQTL took several hours. <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> also shows the number of replicates that GridQTL was able to analyze for different numbers of markers.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460712/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460712/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title">Timing comparisons for triM and GridQTL</span><div class="sb-div caption-clear"></div></div></div><p id="p-24">In total, 46,520 recombinations were simulated over the first 333 cM in the nine simulations. We found switches in line origin probabilities for 88% of these using triM and 53% using GridQTL. Most of the recombination positions not found by GridQTL were in regions with missing probability estimates. GridQTL only identified 14 recombination events that were not found by triM. Imprecision in estimated recombination positions was nearly always higher for GridQTL than triM (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>), and inaccuracy was also higher on average for GridQTL (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). triM estimated more recombination positions with minimum imprecision than GridQTL and more with zero inaccuracy (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/1/57/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimation of recombination positions by triM and GridQTL. Imprecision and inaccuracy were measured as described in the METHODS. The size of the circles is in proportion to the number of data points at each pair of values. Points above the diagonal represent cases where imprecision or inaccuracy in the estimated position of recombination was greater for GridQTL than for triM; points below the diagonal, cases where imprecision or inaccuracy was greater for triM than GridQTL; and points on the diagonal, cases where imprecision or inaccuracy was the same for both methods. The further the points are from the diagonal, the bigger the difference between the results. (A) Imprecision in estimated positions of recombination. (B) Inaccuracy in estimated positions of recombination." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1270635032" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimation of recombination positions by triM and GridQTL. Imprecision and inaccuracy were measured as described in the METHODS. The size of the circles is in proportion to the number of data points at each pair of values. Points above the diagonal represent cases where imprecision or inaccuracy in the estimated position of recombination was greater for GridQTL than for triM; points below the diagonal, cases where imprecision or inaccuracy was greater for triM than GridQTL; and points on the diagonal, cases where imprecision or inaccuracy was the same for both methods. The further the points are from the diagonal, the bigger the difference between the results. (A) Imprecision in estimated positions of recombination. (B) Inaccuracy in estimated positions of recombination.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/1/57/F5.medium.gif" width="229" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/1/1/57/F5.medium.gif" width="229" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/1/57/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/1/57/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460767" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-25" class="first-child">Estimation of recombination positions by triM and GridQTL. Imprecision and inaccuracy were measured as described in the METHODS. The size of the circles is in proportion to the number of data points at each pair of values. Points above the diagonal represent cases where imprecision or inaccuracy in the estimated position of recombination was greater for GridQTL than for triM; points below the diagonal, cases where imprecision or inaccuracy was greater for triM than GridQTL; and points on the diagonal, cases where imprecision or inaccuracy was the same for both methods. The further the points are from the diagonal, the bigger the difference between the results. (A) Imprecision in estimated positions of recombination. (B) Inaccuracy in estimated positions of recombination.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460760/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460760/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2 </span> <span class="caption-title">Differences between triM and GridQTL in estimating recombination positions</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460772/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460772/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3 </span> <span class="caption-title">Best estimates of recombination positions by triM and GridQTL</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">Examples of cases where triM and GridQTL substantially differed in estimating recombination positions are given in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. In the few cases where the imprecision was much higher for triM than GridQTL, it was often when there were recombinations from line 1 to 2 in one parent and line 2 to 1 in the other, within a few cM, and although GridQTL was less imprecise in the estimate of one recombination, it failed to detect the other at all (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6D</a>). In contrast, cases where imprecision was much higher for GridQTL seemed to be when there were recombinations in the same direction in both parents (<em>i.e.</em> both line 1 to line 2 or vice versa), around 40 cM apart (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6A and C</a>). There were other times when the parental line origin probabilities estimated by GridQTL changed more slowly than the estimates from triM (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6B</a>). When inaccuracy was higher for triM than GridQTL by more than 2 cM, either imprecision was also higher for triM (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6D</a>), or triM was less imprecise, so that the number of surrounding positions with a high line origin error was lower for triM (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6E</a>). Cases where GridQTL estimates were much more inaccurate tended to be cases where they were also more imprecise (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6C</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/1/57/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Examples of haplotype line origin probabilities. Estimated probabilities that the paternal and maternal haplotypes originate from line 1 are illustrated for examples where there were large differences between triM and GridQTL in estimating recombination positions. The left two graphs show results from triM, and the right two graphs show results from GridQTL. The paternal haplotype is shown on the left, and the maternal haplotype is shown on the right. Dashed lines show the simulated line origins. (A) The two cases where GridQTL was most imprecise in estimating the recombination position in comparison to triM; the simulated recombination points were at 192 cM in the father and 152 cM in the mother. (B) Another case where GridQTL had much higher imprecision than triM; the simulated recombination point was at 303 cM in the mother. (C) The two cases where GridQTL was most inaccurate in estimating the recombination position in comparison to triM; the simulated recombination points were at 186 cM in the father and 155 cM in the mother. (D) The case where triM was most imprecise and inaccurate in estimating the recombination position in comparison to GridQTL; the simulated recombination position was at 166 cM in the father. (E) One of the cases where triM was most inaccurate in estimating the recombination position in comparison to GridQTL; the simulated recombination position was at 210 cM in the father." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1270635032" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Examples of haplotype line origin probabilities. Estimated probabilities that the paternal and maternal haplotypes originate from line 1 are illustrated for examples where there were large differences between triM and GridQTL in estimating recombination positions. The left two graphs show results from triM, and the right two graphs show results from GridQTL. The paternal haplotype is shown on the left, and the maternal haplotype is shown on the right. Dashed lines show the simulated line origins. (A) The two cases where GridQTL was most imprecise in estimating the recombination position in comparison to triM; the simulated recombination points were at 192 cM in the father and 152 cM in the mother. (B) Another case where GridQTL had much higher imprecision than triM; the simulated recombination point was at 303 cM in the mother. (C) The two cases where GridQTL was most inaccurate in estimating the recombination position in comparison to triM; the simulated recombination points were at 186 cM in the father and 155 cM in the mother. (D) The case where triM was most imprecise and inaccurate in estimating the recombination position in comparison to GridQTL; the simulated recombination position was at 166 cM in the father. (E) One of the cases where triM was most inaccurate in estimating the recombination position in comparison to GridQTL; the simulated recombination position was at 210 cM in the father.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/1/57/F6.medium.gif" width="440" height="399"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/1/1/57/F6.medium.gif" width="440" height="399"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/1/57/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/1/57/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460744" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-29" class="first-child">Examples of haplotype line origin probabilities. Estimated probabilities that the paternal and maternal haplotypes originate from line 1 are illustrated for examples where there were large differences between triM and GridQTL in estimating recombination positions. The left two graphs show results from triM, and the right two graphs show results from GridQTL. The paternal haplotype is shown on the left, and the maternal haplotype is shown on the right. Dashed lines show the simulated line origins. (A) The two cases where GridQTL was most imprecise in estimating the recombination position in comparison to triM; the simulated recombination points were at 192 cM in the father and 152 cM in the mother. (B) Another case where GridQTL had much higher imprecision than triM; the simulated recombination point was at 303 cM in the mother. (C) The two cases where GridQTL was most inaccurate in estimating the recombination position in comparison to triM; the simulated recombination points were at 186 cM in the father and 155 cM in the mother. (D) The case where triM was most imprecise and inaccurate in estimating the recombination position in comparison to GridQTL; the simulated recombination position was at 166 cM in the father. (E) One of the cases where triM was most inaccurate in estimating the recombination position in comparison to GridQTL; the simulated recombination position was at 210 cM in the father.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section discussion" id="sec-8"><h2 class="">Discussion</h2><p id="p-30">Our study clearly illustrates the need for new algorithms to calculate line origin probabilities in outbred line cross experiments. The algorithm that is the basis of the most commonly used methods today (<a id="xref-ref-5-7" class="xref-bibr" href="#ref-5">Haley <em>et al</em>., 1994</a>) was not designed to handle dense SNP marker datasets and cannot be used for these in its original formulation. The GridQTL project (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Seaton <em>et al.</em> 2006</a>) provides a modified implementation of the <a id="xref-ref-5-8" class="xref-bibr" href="#ref-5">Haley <em>et al.</em> (1994)</a> algorithm that is reported to be useful for analyses of intermediate density SNP markers (up to 1500 markers per chromosome). Our comparisons however, illustrate that such extensions are not optimal, because for the GridQTL implementation (i) there are problems with the stability of the algorithm leading to either a significant portion of the analyses terminating before results are obtained or probabilities not being calculated for many individuals in large parts of the genome; (ii) the computational load, despite considerable computational resources, is approximately two orders of magnitude greater for intermediate size datasets and increases rapidly with the number of additional markers included in the analyses; and (iii) accuracy in estimates of line origin probabilities and inference of recombination positions is lower as a result of not using all the available genetic information. Although the stability of the algorithm might be resolved and it is possible that the time requirements could be somewhat improved, further development based on the <a id="xref-ref-5-9" class="xref-bibr" href="#ref-5">Haley <em>et al.</em> (1994)</a> algorithm is not recommended, as the algorithm is inherently less efficient than alternative algorithms and discarding marker information will inevitably give lower accuracy in estimates of line origin probabilities.</p><p id="p-31">This study involved datasets of 1200-1500 markers, which was close to the maximum that could be handled by the modified <a id="xref-ref-5-10" class="xref-bibr" href="#ref-5">Haley <em>et al.</em> (1994)</a> algorithm implemented in GridQTL. This is, however, much lower than the number of markers that can currently be genotyped with SNP chips, for example, the chicken 60K SNP chip has approximately 8600 markers on chromosome 1. triM will be able to use all the marker information in such large datasets and complete the analyses rapidly.</p><p id="p-32">A result that is of major concern for those attempting to analyze moderately dense SNP datasets with GridQTL is the large number of missing probability estimates; over 30% of the individuals had no estimates for the majority of the genome. Using GridQTL estimated probabilities rather than a complete set of probabilities to test for an association with phenotype will substantially reduce the power to detect QTL.</p><p id="p-33">Data were simulated to closely reflect the pattern of marker information in real intercrosses. This was achieved by using founder haplotypes that were as near as possible to those in a well-studied chicken intercross and replicating the marker distances and pedigree structure. Therefore, the linkage disequilibria observed should closely resemble those that can be seen in real data. A series of replicates were created to ensure that findings were not due to peculiarities in specific runs; results showed relatively little variation across replicates. Hence, we believe that the findings in this study are representative of other large datasets.</p><p id="p-34">triM was able to analyze the full set of markers and all replicates and give probability estimates for every position. Theoretically, the number of calculations performed by triM scales logarithmically with the number of markers, not exponentially as in the <a id="xref-ref-5-11" class="xref-bibr" href="#ref-5">Haley <em>et al.</em> (1994)</a> algorithm. This manageable increase in memory required means that triM should be capable of analyzing much larger marker datasets than those simulated here, including those that might become available in the future as SNP genotyping technology develops. There is no need for extensive preprocessing of the data; all the marker genotypes can be submitted so that no information is lost. In this study, triM was shown to be considerably faster than the modified <a id="xref-ref-5-12" class="xref-bibr" href="#ref-5">Haley <em>et al.</em> (1994)</a> algorithm implemented in GridQTL, taking seconds to complete the analyses rather than hours. The gain in speed was at least a factor of 80. The online documentation for GridQTL gives an estimated runtime of 48 hr for data from 1500 markers on one chromosome in 1000 individuals. As with memory, the time required by triM should increase moderately as more markers are included, so that data from large SNP chips could still be completed in a reasonable time.</p><p id="p-35">Our simulations show that the average line origin error is low, around 2%, for both methods. The error was 10% larger for the GridQTL algorithm than for triM. Although the difference is small in absolute terms, triM did perform better and therefore should be preferred. It should also be noted that the observed difference in accuracy probably underestimates the true difference between the methods because only values that were estimated by GridQTL were compared. The points that GridQTL failed to estimate presumably included those where there were the longest stretches of partially informative markers, and therefore where the difference between the two methods could be greatest.</p><p id="p-36">To gain a more detailed understanding of the differences between the two methods, we explored how well they inferred positions of recombination in the parents using measures of imprecision and inaccuracy. High imprecision or inaccuracy in estimating recombination positions is likely to explain most of the largest errors in line origin. GridQTL was nearly always more imprecise in estimating recombination positions than triM. There was less difference between the methods in the accuracy of the estimated positions, although GridQTL was more inaccurate than triM considerably more times than the other way round.</p><p id="p-37">There is no detailed documentation of the changes made to the <a id="xref-ref-5-13" class="xref-bibr" href="#ref-5">Haley <em>et al.</em> (1994)</a> algorithm in GridQTL. It is likely that it uses a similar algorithm to that of QTL Express, the predecessor of GridQTL, where partially informative markers were discarded if necessary, so that there were no more than 15 partially informative markers in a stretch without intervening fully informative markers. Although these details are not available, we do not feel this affects our conclusions, as the differences between GridQTL and triM are in line with theoretical expectations.</p><p id="p-38">We evaluated triM on a simulated F<sub>2</sub> population. However, triM can equally be used for backcross populations or full or half sib family analyses. The codebase in which triM is implemented, cnF2freq, has additional capabilities. First, the grandparental chromosome that a region originates from can be estimated. This information can be used for more general analyses in a variance component framework without assuming QTL are fixed for alternate alleles in the two lines (Rönnegård <em>et al.</em> 2008). The functionality can also be extended into a full scheme for haplotyping. Inferring parental haplotypes can provide additional information on line origin by considering inheritance at linked markers. Second, different recombination rates can be applied for the two sexes. Although we simulated the same recombination rate in both sexes, sex differences in recombination rates of autosomes have been found in chicken (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Kerje <em>et al.</em> 2003</a>, <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Groenen <em>et al.</em> 2009</a>) and other species. Use of the appropriate recombination rates for each sex should improve the probability estimates in general and is essential for correct analysis of the sex chromosomes. When the physical location or ordering of markers is known, a maximum likelihood fit of sex-specific recombination rates can also be computed, where the sex-averaged distances can be used as a starting value. Third, joint line origin probabilities, required for fitting models of more than one locus per chromosome (for example, in testing for epistasis) can be estimated. The line origin probabilities at two such positions are not independent if there is no fully informative marker between them and these dependencies have been included.</p><p id="p-39">In conclusion, we have validated that triM provides accurate line origin probabilities from large SNP datasets and is extremely fast. A variant of the <a id="xref-ref-5-14" class="xref-bibr" href="#ref-5">Haley <em>et al.</em> (1994)</a> algorithm that has most commonly been used for this purpose was shown to be inferior for such data. triM was faster, more stable, provided more accurate line origin probabilities, and was better at inferring recombination positions. triM is therefore recommended as the method of choice for estimating line origin probabilities in outbred line crosses. triM is under continued development, and several useful additional functionalities are already available that will enable QTL analyses to keep pace with advances in genotyping technology.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">Ö.C. acknowledges funding from The Swedish Foundation for Strategic research and a EURYI Award from ESF. C.N. is funded by The Graduate School in Mathematics and Computing.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-41"><em>Communicating editor: Brenda J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-01-07"><span class="received-label">Received </span>January 7, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-03-30"><span class="accepted-label">Accepted </span>March 30, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2011 Crooks <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-1">This is an open access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.57.1" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen,</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span><strong>:</strong> <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12724300&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F57.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182634800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.57.2" data-doi="10.1051/gse:2001112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corva</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medrano</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Quantitative trait loci (QTLs) mapping for growth traits in the mouse: a review.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">33</span><strong>:</strong> <span class="cit-fpage">105</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DCorva%26rft.auinit1%253DP.%2BM.%26rft.volume%253D33%26rft.issue%253D2%26rft.spage%253D105%26rft.epage%253D132%26rft.atitle%253DQuantitative%2Btrait%2Bloci%2B%2528QTLs%2529%2Bmapping%2Bfor%2Bgrowth%2Btraits%2Bin%2Bthe%2Bmouse%253A%2Ba%2Breview.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A2001112%26rft_id%253Dinfo%253Apmid%252F11333830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:2001112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11333830&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F57.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168433600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.57.3" data-doi="10.1126/science.289.5476.85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frary</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nesbitt</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frary</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grandillo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Der Knaap</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title"><em>fw2.2</em>: a quantitative trait locus key to the evolution of tomato fruit size.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">289</span><strong>:</strong> <span class="cit-fpage">85</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFrary%26rft.auinit1%253DA.%26rft.volume%253D289%26rft.issue%253D5476%26rft.spage%253D85%26rft.epage%253D88%26rft.atitle%253Dfw2.2%253A%2BA%2BQuantitative%2BTrait%2BLocus%2BKey%2Bto%2Bthe%2BEvolution%2Bof%2BTomato%2BFruit%2BSize%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.289.5476.85%26rft_id%253Dinfo%253Apmid%252F10884229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIyODkvNTQ3Ni84NSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzEvMS81Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.57.4" data-doi="10.1101/gr.086538.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Groenen</span>  <span class="cit-name-given-names">M. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wahlberg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foglio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megens</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A high-density SNP-based linkage map of the chicken genome reveals sequence features correlated with recombination rate.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span><strong>:</strong> <span class="cit-fpage">510</span>–<span class="cit-lpage">519</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.086538.108%26rft_id%253Dinfo%253Apmid%252F19088305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzMvNTEwIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8xLzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.57.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knott,</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elsen</span>  <span class="cit-name-given-names">J.-M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Mapping quantitative trait loci in crosses between outbred lines using least squares.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span><strong>:</strong> <span class="cit-fpage">1195</span>–<span class="cit-lpage">1207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D136%26rft.spage%253D1195%26rft_id%253Dinfo%253Apmid%252F8005424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8005424&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F57.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994MZ47400043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.57.6" data-doi="10.1079/WPS200461"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hocking</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Review of QTL mapping results in chickens.</span> <abbr class="cit-jnl-abbrev">Worlds Poult. Sci. J.</abbr> <span class="cit-vol">61</span><strong>:</strong> <span class="cit-fpage">215</span>–<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWorlds%2BPoult.%2BSci.%2BJ.%26rft.volume%253D61%26rft.spage%253D215%26rft_id%253Dinfo%253Adoi%252F10.1079%252FWPS200461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1079/WPS200461&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000229873100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.57.7" data-doi="10.1046/j.1365-2052.2003.01000.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kerje</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobsson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schütz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartmann</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The twofold difference in adult size between the red junglefowl and White Leghorn chickens is largely explained by a limited number of QTLs.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">34</span><strong>:</strong> <span class="cit-fpage">264</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DKerje%26rft.auinit1%253DS.%26rft.volume%253D34%26rft.issue%253D4%26rft.spage%253D264%26rft.epage%253D274%26rft.atitle%253DThe%2Btwofold%2Bdifference%2Bin%2Badult%2Bsize%2Bbetween%2Bthe%2Bred%2Bjunglefowl%2Band%2BWhite%2BLeghorn%2Bchickens%2Bis%2Blargely%2Bexplained%2Bby%2Ba%2Blimited%2Bnumber%2Bof%2BQTLs.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2052.2003.01000.x%26rft_id%253Dinfo%253Apmid%252F12873214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2052.2003.01000.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12873214&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F57.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184331200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.57.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span><strong>:</strong> <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D121%26rft.spage%253D185%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2563713&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F57.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989R676700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.1.57.9" data-doi="10.1073/pnas.84.8.2363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Construction of multilocus genetic linkage maps in humans.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">84</span><strong>:</strong> <span class="cit-fpage">2363</span>–<span class="cit-lpage">2367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D84%26rft.issue%253D8%26rft.spage%253D2363%26rft.epage%253D2367%26rft.atitle%253DConstruction%2Bof%2Bmultilocus%2Bgenetic%2Blinkage%2Bmaps%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.84.8.2363%26rft_id%253Dinfo%253Apmid%252F3470801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4NC84LzIzNjMiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzEvNTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.57.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyman</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langley,</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Both naturally occurring insertions of transposable elements and intermediate frequency polymorphisms at the <em>achaete-scute</em> complex are associated with variation in bristle number in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span><strong>:</strong> <span class="cit-fpage">1255</span>–<span class="cit-lpage">1269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLong%26rft.auinit1%253DA.%2BD.%26rft.volume%253D154%26rft.issue%253D3%26rft.spage%253D1255%26rft.epage%253D1269%26rft.atitle%253DBoth%2BNaturally%2BOccurring%2BInsertions%2Bof%2BTransposable%2BElements%2Band%2BIntermediate%2BFrequency%2BPolymorphisms%2Bat%2Bthe%2Bachaete-scute%2BComplex%2BAre%2BAssociated%2BWith%2BVariation%2Bin%2BBristle%2BNumber%2Bin%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Apmid%252F10757767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NC8zLzEyNTUiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzEvNTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.57.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Quantitative trait loci in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">2</span><strong>:</strong> <span class="cit-fpage">11</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D20%26rft.atitle%253DQuantitative%2Btrait%2Bloci%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Apmid%252F11253063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11253063&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F57.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166056000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.57.12" data-doi="10.1016/S1369526602000080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maloof</span>  <span class="cit-name-given-names">J. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">QTL for plant growth and morphology.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">6</span><strong>:</strong> <span class="cit-fpage">85</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DMaloof%26rft.auinit1%253DJ.%2BN.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D85%26rft.epage%253D90%26rft.atitle%253DQTL%2Bfor%2Bplant%2Bgrowth%2Band%2Bmorphology.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1369526602000080%26rft_id%253Dinfo%253Apmid%252F12495756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1369526602000080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12495756&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F57.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180195000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-confproc" id="cit-1.1.57.13"><div class="cit-metadata unstructured">Nettelblad, C., S. Holmgren, L. Crooks, and Ö. Carlborg, 2009 cnF2freq: efficient determination of genotype and haplotype probabilities in outbred populations using Markov models, pp. 307–319 in <em>Bioinformatics and Computational Biology</em>, <em>1st International Conference BICoB</em>, April 8–10, New Orleans, edited by S. Rajasekaran. Springer-Verlag, Berlin, Heidelberg.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.57.14" data-doi="10.1109/5.18626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rabiner</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A tutorial on hidden Markov models and selected applications in speech recognition.</span> <abbr class="cit-jnl-abbrev">Proc. IEEE</abbr> <span class="cit-vol">77</span><strong>:</strong> <span class="cit-fpage">257</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BIEEE%26rft.volume%253D77%26rft.spage%253D257%26rft_id%253Dinfo%253Adoi%252F10.1109%252F5.18626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/5.18626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1989U374600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.57.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rothschild</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu,</span>  <span class="cit-name-given-names">Z.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Advances in QTL mapping in pigs.</span> <abbr class="cit-jnl-abbrev">Int. J. Biol. Sci.</abbr> <span class="cit-vol">3</span><strong>:</strong> <span class="cit-fpage">192</span>–<span class="cit-lpage">197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DInt%2BJ%2BBiol%2BSci%26rft.aulast%253DRothschild%26rft.auinit1%253DM.%2BF.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253D192%26rft.epage%253D197%26rft.atitle%253DAdvances%2Bin%2BQTL%2Bmapping%2Bin%2Bpigs.%26rft_id%253Dinfo%253Apmid%252F17384738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17384738&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F57.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246980700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.1.57.16" data-doi="10.1534/genetics.107.083162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Besnier,</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">An improved method for quantitative trait loci detection and identification of within-line segregation in F<sub>2</sub> intercross designs.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span><strong>:</strong> <span class="cit-fpage">2315</span>–<span class="cit-lpage">2326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRonnegard%26rft.auinit1%253DL.%26rft.volume%253D178%26rft.issue%253D4%26rft.spage%253D2315%26rft.epage%253D2326%26rft.atitle%253DAn%2BImproved%2BMethod%2Bfor%2BQuantitative%2BTrait%2BLoci%2BDetection%2Band%2BIdentification%2Bof%2BWithin-Line%2BSegregation%2Bin%2BF2%2BIntercross%2BDesigns%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.083162%26rft_id%253Dinfo%253Apmid%252F18430952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC80LzIzMTUiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzEvNTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-confproc" id="cit-1.1.57.17"><div class="cit-metadata unstructured">Seaton, G., J. Hernandez, J.-A. Grunchec, I. White, J. Allen <em>et al.</em>, 2006 GridQTL: a grid portal for QTL mapping of compute intensive datasets, <em>Proceedings of the 8th World Congress on Genetics Applied to Livestock Production</em>, August 13–18, Belo Horizonte, MG, Brasil.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.57.18" data-doi="10.1038/nature02064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Laere</span>  <span class="cit-name-given-names">A.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braunschwieg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nezer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collette</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A regulatory mutation in <em>IGF2</em> causes a major QTL effect on muscle growth in the pig.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span><strong>:</strong> <span class="cit-fpage">832</span>–<span class="cit-lpage">836</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DVan%2BLaere%26rft.auinit1%253DA.%2BS.%26rft.volume%253D425%26rft.issue%253D6960%26rft.spage%253D832%26rft.epage%253D836%26rft.atitle%253DA%2Bregulatory%2Bmutation%2Bin%2BIGF2%2Bcauses%2Ba%2Bmajor%2BQTL%2Beffect%2Bon%2Bmuscle%2Bgrowth%2Bin%2Bthe%2Bpig.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02064%26rft_id%253Dinfo%253Apmid%252F14574411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14574411&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F57.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186118500044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/1/57.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/1/43" title="Identification of Yeast Genes Involved in K+ Homeostasis: Loss of Membrane Traffic Genes Affects K+ Uptake" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/1/65" title="Fine Mapping of a QTL for Fertility on BTA7 and Its Association With a CNV in the Israeli Holsteins" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/1"><span>Volume 1 Issue 1, June 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/1.cover.gif?itok=pV8lCSGW" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/1/57.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0lqXoj-8nguS1MHo2UUM2tHt-AhHO-UydQPtV_CqwKM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/1/57" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/1/57" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/1/57" class="active" data-icon-position="" data-hide-link-title="0">An Improved Method for Estimating Chromosomal Line Origin in QTL Analysis of Crosses Between Outbred Lines</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459157" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-LsRduLeB1Swotp1lyOq3-HKCWV0-yuR6Olu0z8tb7bI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/1/57.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/1/57" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-9k3Q4pBSVH6dNJWGC4URpSxoR6cCQJQMhhc33DkFxbM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/1/57" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/1/57.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459157" id="node459157--2" data-pisa="ggg;1/1/57" data-pisa-master="ggg;g3.111.000109" data-apath="/ggg/1/1/57.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">An Improved Method for Estimating Chromosomal Line Origin in QTL Analysis of Crosses Between Outbred Lines</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lucy Crooks</span>, <span class="highwire-citation-author" data-delta="1">Carl Nettelblad</span> and <span class="highwire-citation-author" data-delta="2">Örjan Carlborg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">57-64;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000109 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459157--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lucy Crooks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Cell and Molecular Biology, Uppsala University, SE-751 24 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucy%2BCrooks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crooks%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALucy%2BCrooks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lucy.crooks@icm.uu.se" class="" data-icon-position="" data-hide-link-title="0">lucy.crooks@icm.uu.se</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Carl Nettelblad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Information Technology, Uppsala University, SE-751 05 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carl%2BNettelblad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nettelblad%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarl%2BNettelblad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Örjan Carlborg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Cell and Molecular Biology, Uppsala University, SE-751 24 Uppsala, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C3%96rjan%2BCarlborg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carlborg%20%C3%96&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3A%25C3%2596rjan%2BCarlborg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/1/57" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/1/57&amp;title=An%20Improved%20Method%20for%20Estimating%20Chromosomal%20Line%20Origin%20in%20QTL%20Analysis%20of%20Crosses%20Between%20Outbred%20Lines" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/1/57&amp;title=An%20Improved%20Method%20for%20Estimating%20Chromosomal%20Line%20Origin%20in%20QTL%20Analysis%20of%20Crosses%20Between%20Outbred%20Lines" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/1/57&amp;title=An%20Improved%20Method%20for%20Estimating%20Chromosomal%20Line%20Origin%20in%20QTL%20Analysis%20of%20Crosses%20Between%20Outbred%20Lines" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/1/57&amp;text=An%20Improved%20Method%20for%20Estimating%20Chromosomal%20Line%20Origin%20in%20QTL%20Analysis%20of%20Crosses%20Between%20Outbred%20Lines" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/1/57&amp;title=An%20Improved%20Method%20for%20Estimating%20Chromosomal%20Line%20Origin%20in%20QTL%20Analysis%20of%20Crosses%20Between%20Outbred%20Lines" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/1/57&amp;t=An%20Improved%20Method%20for%20Estimating%20Chromosomal%20Line%20Origin%20in%20QTL%20Analysis%20of%20Crosses%20Between%20Outbred%20Lines" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/1/57&amp;title=An%20Improved%20Method%20for%20Estimating%20Chromosomal%20Line%20Origin%20in%20QTL%20Analysis%20of%20Crosses%20Between%20Outbred%20Lines" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/1/57&amp;title=An%20Improved%20Method%20for%20Estimating%20Chromosomal%20Line%20Origin%20in%20QTL%20Analysis%20of%20Crosses%20Between%20Outbred%20Lines" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459157/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459157" id="node459157--3" data-pisa="ggg;1/1/57" data-pisa-master="ggg;g3.111.000109" data-apath="/ggg/1/1/57.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">An Improved Method for Estimating Chromosomal Line Origin in QTL Analysis of Crosses Between Outbred Lines</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lucy Crooks</span>, <span class="highwire-citation-author" data-delta="1">Carl Nettelblad</span> and <span class="highwire-citation-author" data-delta="2">Örjan Carlborg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">57-64;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000109 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459157--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lucy Crooks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Cell and Molecular Biology, Uppsala University, SE-751 24 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucy%2BCrooks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crooks%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALucy%2BCrooks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lucy.crooks@icm.uu.se" class="" data-icon-position="" data-hide-link-title="0">lucy.crooks@icm.uu.se</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Carl Nettelblad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Information Technology, Uppsala University, SE-751 05 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carl%2BNettelblad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nettelblad%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarl%2BNettelblad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Örjan Carlborg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Cell and Molecular Biology, Uppsala University, SE-751 24 Uppsala, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C3%96rjan%2BCarlborg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carlborg%20%C3%96&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3A%25C3%2596rjan%2BCarlborg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=57&amp;rft.epage=64&amp;rft.atitle=An%20Improved%20Method%20for%20Estimating%20Chromosomal%20Line%20Origin%20in%20QTL%20Analysis%20of%20Crosses%20Between%20Outbred%20Lines&amp;rft.volume=1&amp;rft.issue=1&amp;rft.date=2011-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Crooks%2C+Lucy&amp;rft.au=Nettelblad%2C+Carl&amp;rft.au=Carlborg%2C+%C3%96rjan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459157/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459157/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459157/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459157/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459157/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459157/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459157/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459157/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459157/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459157/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459157/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459157/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496048" id="node496048" data-pisa="ggg;g3.119.400937v1" data-pisa-master="ggg;g3.119.400937" data-apath="/ggg/early/2020/03/25/g3.119.400937.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/25/g3.119.400937" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tunable Transcriptional Interference at the Endogenous Alcohol Dehydrogenase Gene Locus in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496046" id="node496046" data-pisa="ggg;g3.120.401048v1" data-pisa-master="ggg;g3.120.401048" data-apath="/ggg/early/2020/03/23/g3.120.401048.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/23/g3.120.401048" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tools for the Genetic Manipulation of <em>Herpetomonas muscarum</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496045" id="node496045" data-pisa="ggg;g3.119.400884v1" data-pisa-master="ggg;g3.119.400884" data-apath="/ggg/early/2020/03/17/g3.119.400884.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400884" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study for Maize Leaf Cuticular Conductance Identifies Candidate Genes Involved in the Regulation of Cuticle Development</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/1/57" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Analysis</a></li><li class="even"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/1/57.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/1/57.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/1/1/57.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
