<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/1/11" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/1/11.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/1/11.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/1/11.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1.cover.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/1/11.atom" />
<meta name="HW.pisa" content="ggg;1/1/11" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Awesome Power of Yeast Evolutionary Genetics: New Genome Sequences and Strain Resources for the Saccharomyces sensu stricto Genus" />
<meta name="DC.Identifier" content="10.1534/g3.111.000273" />
<meta name="DC.Date" content="2011-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2011 Scannell et al.. This is an open access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="High-quality, well-annotated genome sequences and standardized laboratory strains fuel experimental and evolutionary research. We present improved genome sequences of three species of Saccharomyces sensu stricto yeasts: S. bayanus var. uvarum (CBS 7001), S. kudriavzevii (IFO 1802T and ZP 591), and S. mikatae (IFO 1815T), and describe their comparison to the genomes of S. cerevisiae and S. paradoxus . The new sequences, derived by assembling millions of short DNA sequence reads together with previously published Sanger shotgun reads, have vastly greater long-range continuity and far fewer gaps than the previously available genome sequences. New gene predictions defined a set of 5261 protein-coding orthologs across the five most commonly studied Saccharomyces yeasts, enabling a re-examination of the tempo and mode of yeast gene evolution and improved inferences of species-specific gains and losses. To facilitate experimental investigations, we generated genetically marked, stable haploid strains for all three of these Saccharomyces species. These nearly complete genome sequences and the collection of genetically marked strains provide a valuable toolset for comparative studies of gene function, metabolism, and evolution, and render Saccharomyces sensu stricto the most experimentally tractable model genus. These resources are freely available and accessible through [www.SaccharomycesSensuStricto.org][1].

 [1]: http://www.SaccharomycesSensuStricto.org" />
<meta name="DC.Contributor" content="Devin R. Scannell" />
<meta name="DC.Contributor" content="Oliver A. Zill" />
<meta name="DC.Contributor" content="Antonis Rokas" />
<meta name="DC.Contributor" content="Celia Payen" />
<meta name="DC.Contributor" content="Maitreya J. Dunham" />
<meta name="DC.Contributor" content="Michael B. Eisen" />
<meta name="DC.Contributor" content="Jasper Rine" />
<meta name="DC.Contributor" content="Mark Johnston" />
<meta name="DC.Contributor" content="Chris Todd Hittinger" />
<meta name="article:published_time" content="2011-06-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="The Awesome Power of Yeast Evolutionary Genetics: New Genome Sequences and Strain Resources for the Saccharomyces sensu stricto Genus" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;High-quality, well-annotated genome sequences and standardized laboratory strains fuel experimental and evolutionary research. We present improved genome sequences of three species of &lt;i&gt;Saccharomyces sensu stricto&lt;/i&gt; yeasts: &lt;i&gt;S. bayanus&lt;/i&gt; var. &lt;i&gt;uvarum&lt;/i&gt; (CBS 7001), &lt;i&gt;S. kudriavzevii&lt;/i&gt; (IFO 1802&lt;sup&gt;T&lt;/sup&gt; and ZP 591), and &lt;i&gt;S. mikatae&lt;/i&gt; (IFO 1815&lt;sup&gt;T&lt;/sup&gt;), and describe their comparison to the genomes of &lt;i&gt;S. cerevisiae&lt;/i&gt; and &lt;i&gt;S. paradoxus&lt;/i&gt;. The new sequences, derived by assembling millions of short DNA sequence reads together with previously published Sanger shotgun reads, have vastly greater long-range continuity and far fewer gaps than the previously available genome sequences. New gene predictions defined a set of 5261 protein-coding orthologs across the five most commonly studied &lt;i&gt;Saccharomyces&lt;/i&gt; yeasts, enabling a re-examination of the tempo and mode of yeast gene evolution and improved inferences of species-specific gains and losses. To facilitate experimental investigations, we generated genetically marked, stable haploid strains for all three of these &lt;i&gt;Saccharomyces&lt;/i&gt; species. These nearly complete genome sequences and the collection of genetically marked strains provide a valuable toolset for comparative studies of gene function, metabolism, and evolution, and render &lt;i&gt;Saccharomyces sensu stricto&lt;/i&gt; the most experimentally tractable model genus. These resources are freely available and accessible through www.SaccharomycesSensuStricto.org.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/06/01" />
<meta name="citation_mjid" content="ggg;1/1/11" />
<meta name="citation_id" content="1/1/11" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/1/11" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/1/11.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/1/11.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/1/11.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.000273" />
<meta name="citation_pmid" content="22384314" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="1" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="11" />
<meta name="citation_lastpage" content="25" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Devin R. Scannell" />
<meta name="citation_author_institution" content="Department of Molecular and Cell Biology and California Institute for Quantitative Biosciences, UC Berkeley, Berkeley, California 94720-3220" />
<meta name="citation_author" content="Oliver A. Zill" />
<meta name="citation_author_institution" content="Department of Molecular and Cell Biology and California Institute for Quantitative Biosciences, UC Berkeley, Berkeley, California 94720-3220" />
<meta name="citation_author_email" content="zill@berkeley.edu" />
<meta name="citation_author_email" content="cthittinger@wisc.edu" />
<meta name="citation_author" content="Antonis Rokas" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235" />
<meta name="citation_author" content="Celia Payen" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Maitreya J. Dunham" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Michael B. Eisen" />
<meta name="citation_author_institution" content="Department of Molecular and Cell Biology and California Institute for Quantitative Biosciences, UC Berkeley, Berkeley, California 94720-3220" />
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, University of California, Berkeley, Berkeley, California 94720" />
<meta name="citation_author" content="Jasper Rine" />
<meta name="citation_author_institution" content="Department of Molecular and Cell Biology and California Institute for Quantitative Biosciences, UC Berkeley, Berkeley, California 94720-3220" />
<meta name="citation_author" content="Mark Johnston" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, Colorado 80045-2530" />
<meta name="citation_author_institution" content="Center for Genome Sciences, Department of Genetics, Washington University in St. Louis School of Medicine, St. Louis, Missouri 63108-2212" />
<meta name="citation_author" content="Chris Todd Hittinger" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, Colorado 80045-2530" />
<meta name="citation_author_institution" content="Center for Genome Sciences, Department of Genetics, Washington University in St. Louis School of Medicine, St. Louis, Missouri 63108-2212" />
<meta name="citation_author_email" content="zill@berkeley.edu" />
<meta name="citation_author_email" content="cthittinger@wisc.edu" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=E. M. Airoldi;citation_author=C. Huttenhower;citation_author=D. Gresham;citation_author=C. Lu;citation_author=A. A. Caudy;citation_title=Predicting cellular growth from gene expression signatures.;citation_pages=e1000257-e1000257;citation_volume=5;citation_year=2009;citation_issue=1;citation_pmid=19119411;citation_doi=10.1371/journal.pcbi.1000257" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schaffer;citation_author=J. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Atwell;citation_author=Y. S. Huang;citation_author=B. J. Vilhjalmsson;citation_author=G. Willems;citation_author=M. Horton;citation_title=Genome-wide association study of 107 phenotypes in Arabidopsis thaliana inbred lines.;citation_pages=627-631;citation_volume=465;citation_year=2010;citation_issue=7298;citation_pmid=20336072;citation_doi=10.1038/nature08800" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=R. K. Bradley;citation_author=A. Roberts;citation_author=M. Smoot;citation_author=S. Juvekar;citation_author=J. Do;citation_title=Fast statistical alignment.;citation_pages=e1000392-e1000392;citation_volume=5;citation_year=2009;citation_issue=5;citation_pmid=19478997;citation_doi=10.1371/journal.pcbi.1000392" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. H. Bullard;citation_author=Y. Mostovoy;citation_author=S. Dudoit;citation_author=R. B. Brem;citation_title=Polygenic and directional regulatory evolution across pathways in Saccharomyces.;citation_volume=107;citation_year=2010;citation_pmid=20194736;citation_doi=10.1073/pnas.0912959107" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Butler;citation_author=M. D. Rasmussen;citation_author=M. F. Lin;citation_author=M. A. Santos;citation_author=S. Sakthikumar;citation_title=Evolution of pathogenicity and sexual reproduction in eight Candida genomes.;citation_pages=657-662;citation_volume=459;citation_year=2009;citation_issue=7247;citation_pmid=19465905;citation_doi=10.1038/nature08064" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. P. Byrne;citation_author=K. H. Wolfe;citation_title=The Yeast Gene Order Browser: combining curated homology and syntenic context reveals gene fate in polyploid species.;citation_volume=15;citation_year=2005;citation_pmid=16169922;citation_doi=10.1101/gr.3672305" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=K. P. Byrne;citation_author=K. H. Wolfe;citation_title=Visualizing syntenic relationships among the hemiascomycetes with the Yeast Gene Order Browser;citation_pages=D452-D455;citation_volume=34;citation_year=2006;citation_issue=suppl_1;citation_pmid=16381909;citation_doi=10.1093/nar/gkj041" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=H. H. Chou;citation_author=M. H. Holmes;citation_title=DNA sequence quality trimming and vector removal;citation_pages=1093-1104;citation_volume=17;citation_year=2001;citation_issue=12;citation_pmid=11751217;citation_doi=10.1093/bioinformatics/17.12.1093" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=J. Y. Chou;citation_author=Y. S. Hung;citation_author=K. H. Lin;citation_author=H. Y. Lee;citation_author=J. Y. Leu;citation_title=Multiple molecular mechanisms cause reproductive isolation between three yeast species.;citation_pages=e1000432-e1000432;citation_volume=8;citation_year=2010;citation_issue=7;citation_pmid=20652018;citation_doi=10.1371/journal.pbio.1000432" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=J. Y. Chou;citation_author=J. Y. Leu;citation_title=Speciation through cytonuclear incompatibility: insights from yeast and implications for higher eukaryotes.;citation_pages=401-411;citation_volume=32;citation_year=2010;citation_issue=5;citation_pmid=20414898;citation_doi=10.1002/bies.200900162" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. G. Clark;citation_author=M. B. Eisen;citation_author=D. R. Smith;citation_author=C. M. Bergman;citation_author=B. Oliver;citation_title=Evolution of genes and genomes on the Drosophila phylogeny.;citation_pages=203-218;citation_volume=450;citation_year=2007;citation_issue=7167;citation_pmid=17994087;citation_doi=10.1038/nature06341" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=P. Cliften;citation_author=P. Sudarsanam;citation_author=A. Desikan;citation_author=L. Fulton;citation_author=B. Fulton;citation_title=Finding functional features in Saccharomyces genomes by phylogenetic footprinting.;citation_volume=301;citation_year=2003;citation_pmid=12775844;citation_doi=10.1126/science.1084337" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. F. Cliften;citation_author=R. S. Fulton;citation_author=R. K. Wilson;citation_author=M. Johnston;citation_title=After the duplication: gene loss and adaptation in Saccharomyces genomes.;citation_pages=863-872;citation_volume=172;citation_year=2006;citation_pmid=16322519" />
<meta name="citation_reference" content="citation_journal_title=FEM. Yeast Res.;citation_author=F. A. Cubillos;citation_author=E. J. Louis;citation_author=G. Liti;citation_title=Generation of a large set of genetically tractable haploid and diploid Saccharomyces strains.;citation_volume=9;citation_year=2009;citation_pmid=19840116;citation_doi=10.1111/j.1567-1364.2009.00583.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=F. S. Dietrich;citation_author=S. Voegeli;citation_author=S. Brachat;citation_author=A. Lerch;citation_author=K. Gates;citation_title=The Ashbya gossypii genome as a tool for mapping the ancient Saccharomyces cerevisiae genome.;citation_volume=304;citation_year=2004;citation_pmid=15001715;citation_doi=10.1126/science.1095781" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=I. A. Drinnenberg;citation_author=D. E. Weinberg;citation_author=K. T. Xie;citation_author=J. P. Mower;citation_author=K. H. Wolfe;citation_title=RNAi in budding yeast.;citation_volume=326;citation_year=2009;citation_pmid=19745116;citation_doi=10.1126/science.1176945" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=A. J. Drummond;citation_author=S. Y. Ho;citation_author=M. J. Phillips;citation_author=A. Rambaut;citation_title=Relaxed phylogenetics and dating with confidence.;citation_pages=e88-e88;citation_volume=4;citation_year=2006;citation_issue=5;citation_pmid=16683862;citation_doi=10.1371/journal.pbio.0040088" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=A. J. Drummond;citation_author=A. Rambaut;citation_title=BEAST: Bayesian evolutionary analysis by sampling trees.;citation_pages=214-214;citation_volume=7;citation_year=2007;citation_pmid=17996036;citation_doi=10.1186/1471-2148-7-214" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=B. Dujon;citation_title=Yeast evolutionary genomics.;citation_pages=512-524;citation_volume=11;citation_year=2010;citation_issue=7;citation_pmid=20559329;citation_doi=10.1038/nrg2811" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=B. Dujon;citation_title=Yeasts illustrate the molecular mechanisms of eukaryotic genome evolution.;citation_pages=375-387;citation_volume=22;citation_year=2006;citation_issue=7;citation_pmid=16730849;citation_doi=10.1016/j.tig.2006.05.007" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. Dujon;citation_author=D. Sherman;citation_author=G. Fischer;citation_author=P. Durrens;citation_author=S. Casaregola;citation_title=Genome evolution in yeasts.;citation_pages=35-44;citation_volume=430;citation_year=2004;citation_issue=6995;citation_pmid=15229592;citation_doi=10.1038/nature02579" />
<meta name="citation_reference" content="citation_journal_title=Genome Inform;citation_author=S. R. Eddy;citation_title=A new generation of homology search tools based on probabilistic inference.;citation_pages=205-211;citation_volume=23;citation_year=2009;citation_pmid=20180275;citation_doi=10.1142/9781848165632_0019" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. R. Engel;citation_author=R. Balakrishnan;citation_author=G. Binkley;citation_author=K. R. Christie;citation_author=M. C. Costanzo;citation_title=Saccharomyces Genome Database provides mutant phenotype data.;citation_volume=38;citation_year=2010;citation_pmid=19906697;citation_doi=10.1093/nar/gkp917" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Ewing;citation_author=P. Green;citation_title=Base-calling of automated sequencer traces using phred. II. Error probabilities.;citation_volume=8;citation_year=1998;citation_pmid=9521922;citation_doi=10.1101/gr.8.3.186" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. C. Fay;citation_author=J. A. Benavides;citation_title=Hypervariable noncoding sequences in Saccharomyces cerevisiae.;citation_pages=1575-1587;citation_volume=170;citation_year=2005;citation_pmid=15956668;citation_doi=10.1534/genetics.105.042283" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Fidalgo;citation_author=R. R. Barrales;citation_author=J. I. Ibeas;citation_author=J. Jimenez;citation_title=Adaptive evolution by mutations in the FLO11 gene.;citation_volume=103;citation_year=2006;citation_pmid=16844788;citation_doi=10.1073/pnas.0601713103" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Fischer;citation_author=S. A. James;citation_author=I. N. Roberts;citation_author=S. G. Oliver;citation_author=E. J. Louis;citation_title=Chromosomal evolution in Saccharomyces.;citation_pages=451-454;citation_volume=405;citation_year=2000;citation_issue=6785;citation_pmid=10839539;citation_doi=10.1038/35013058" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=M. Gaisne;citation_author=A. M. Becam;citation_author=J. Verdiere;citation_author=C. J. Herbert;citation_title=A 'natural' mutation in Saccharomyces cerevisiae strains derived from S288c affects the complex regulatory gene HAP1 (CYP1).;citation_pages=195-200;citation_volume=36;citation_year=1999;citation_issue=4;citation_pmid=10541856;citation_doi=10.1007/s002940050490" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. E. Gallagher;citation_author=J. E. Babiarz;citation_author=L. Teytelman;citation_author=K. H. Wolfe;citation_author=J. Rine;citation_title=Elaboration, diversification and regulation of the Sir1 family of silencing proteins in Saccharomyces.;citation_pages=1477-1491;citation_volume=181;citation_year=2009;citation_pmid=19171939;citation_doi=10.1534/genetics.108.099663" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=A. P. Gasch;citation_author=A. M. Moses;citation_author=D. Y. Chiang;citation_author=H. B. Fraser;citation_author=M. Berardini;citation_title=Conservation and evolution of cis-regulatory systems in ascomycete fungi.;citation_pages=e398-e398;citation_volume=2;citation_year=2004;citation_issue=12;citation_pmid=15534694;citation_doi=10.1371/journal.pbio.0020398" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Gerke;citation_author=K. Lorenz;citation_author=B. Cohen;citation_title=Genetic Interactions Between Transcription Factors Cause Natural Variation in Yeast;citation_pages=498-501;citation_volume=323;citation_year=2009;citation_issue=5913;citation_pmid=19164747;citation_doi=10.1126/science.1166426" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Goffeau;citation_author=B. G. Barrell;citation_author=H. Bussey;citation_author=R. W. Davis;citation_author=B. Dujon;citation_title=Life with 6000 Genes;citation_pages=546-567;citation_volume=274;citation_year=1996;citation_issue=546;citation_pmid=8849441;citation_doi=10.1126/science.274.5287.546" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=A. L. Goldstein;citation_author=J. H. McCusker;citation_title=Three new dominant drug resistance cassettes for gene disruption in Saccharomyces cerevisiae.;citation_pages=1541-1553;citation_volume=15;citation_year=1999;citation_issue=14;citation_pmid=10514571;citation_doi=10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=S. S. Gonzalez;citation_author=E. Barrio;citation_author=A. Querol;citation_title=Molecular characterization of new natural hybrids of Saccharomyces cerevisiae and S. kudriavzevii in brewing.;citation_volume=74;citation_year=2008;citation_pmid=18296532;citation_doi=10.1128/AEM.01867-07" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. L. Gordon;citation_author=K. P. Byrne;citation_author=K. H. Wolfe;citation_title=Additions, losses, and rearrangements on the evolutionary route from a reconstructed ancestor to the modern Saccharomyces cerevisiae genome.;citation_pages=e1000485-e1000485;citation_volume=5;citation_year=2009;citation_issue=5;citation_pmid=19436716;citation_doi=10.1371/journal.pgen.1000485" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=D. Greig;citation_title=Reproductive isolation in Saccharomyces.;citation_pages=39-44;citation_volume=102;citation_year=2009;citation_issue=1;citation_pmid=18648383;citation_doi=10.1038/hdy.2008.73" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. Greig;citation_author=E. J. Louis;citation_author=R. H. Borts;citation_author=M. Travisano;citation_title=Hybrid Speciation in Experimental Populations of Yeast;citation_pages=1773-1775;citation_volume=298;citation_year=2002;citation_issue=5599;citation_pmid=12459586;citation_doi=10.1126/science.1076374" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=D. Greig;citation_author=M. Travisano;citation_author=E. J. Louis;citation_author=R. H. Borts;citation_title=A role for the mismatch repair system during incipient speciation in Saccharomyces.;citation_pages=429-437;citation_volume=16;citation_year=2003;citation_issue=3;citation_pmid=14635842;citation_doi=10.1046/j.1420-9101.2003.00546.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=Y. Guan;citation_author=M. Dunham;citation_author=A. Caudy;citation_author=O. Troyanskaya;citation_title=Systematic planning of genome-scale experiments in poorly studied species.;citation_pages=e1000698-e1000698;citation_volume=6;citation_year=2010;citation_issue=3;citation_pmid=20221257;citation_doi=10.1371/journal.pcbi.1000698" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Guan;citation_author=M. J. Dunham;citation_author=O. G. Troyanskaya;citation_title=Functional analysis of gene duplications in Saccharomyces cerevisiae.;citation_pages=933-943;citation_volume=175;citation_year=2007;citation_pmid=17151249;citation_doi=10.1534/genetics.106.064329" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=U. Guldener;citation_author=S. Heck;citation_author=T. Fielder;citation_author=J. Beinhauer;citation_author=J. H. Hegemann;citation_title=A new efficient gene disruption cassette for repeated use in budding yeast.;citation_volume=24;citation_year=1996;citation_pmid=8692690;citation_doi=10.1093/nar/24.13.2519" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Hall;citation_author=F. S. Dietrich;citation_title=The reacquisition of biotin prototrophy in Saccharomyces cerevisiae involved horizontal gene transfer, gene duplication and gene clustering.;citation_pages=2293-2307;citation_volume=177;citation_year=2007;citation_pmid=18073433;citation_doi=10.1534/genetics.107.074963" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. T. Hittinger;citation_author=P. Goncalves;citation_author=J. P. Sampaio;citation_author=J. Dover;citation_author=M. Johnston;citation_title=Remarkably ancient balanced polymorphisms in a multi-locus gene network.;citation_pages=54-58;citation_volume=464;citation_year=2010;citation_issue=7285;citation_pmid=20164837;citation_doi=10.1038/nature08791" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. T. Hittinger;citation_author=A. Rokas;citation_author=S. B. Carroll;citation_title=Parallel inactivation of multiple GAL pathway genes and ecological diversification in yeasts.;citation_volume=101;citation_year=2004;citation_pmid=15381776;citation_doi=10.1073/pnas.0404319101" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Kellis;citation_author=B. W. Birren;citation_author=E. S. Lander;citation_title=Proof and evolutionary analysis of ancient genome duplication in the yeast Saccharomyces cerevisiae.;citation_pages=617-624;citation_volume=428;citation_year=2004;citation_issue=6983;citation_pmid=15004568;citation_doi=10.1038/nature02424" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Kellis;citation_author=N. Patterson;citation_author=M. Endrizzi;citation_author=B. Birren;citation_author=E. S. Lander;citation_title=Sequencing and comparison of yeast species to identify genes and regulatory elements.;citation_pages=241-254;citation_volume=423;citation_year=2003;citation_issue=6937;citation_pmid=12748633;citation_doi=10.1038/nature01644" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Research;citation_journal_abbrev=FEMS Yeast Res;citation_author=C. P. Kurtzman;citation_author=C. J. Robnett;citation_title=Phylogenetic relationships among yeasts of the 'Saccharomyces complex' determined from multigene sequence analyses.;citation_pages=417-432;citation_volume=3;citation_year=2003;citation_issue=4;citation_pmid=12748053;citation_doi=10.1016/S1567-1356(03)00012-6" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. J. Kvitek;citation_author=J. L. Will;citation_author=A. P. Gasch;citation_title=Variations in stress sensitivity and genomic expression in diverse S. cerevisiae isolates.;citation_pages=e1000223-e1000223;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18927628;citation_doi=10.1371/journal.pgen.1000223" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=H. Y. Lee;citation_author=J. Y. Chou;citation_author=L. Cheong;citation_author=N. H. Chang;citation_author=S. Y. Yang;citation_title=Incompatibility of nuclear and mitochondrial genomes causes hybrid sterility between two yeast species.;citation_pages=1065-1073;citation_volume=135;citation_year=2008;citation_issue=6;citation_pmid=19070577;citation_doi=10.1016/j.cell.2008.10.047" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=P. Lefrancois;citation_author=G. M. Euskirchen;citation_author=R. K. Auerbach;citation_author=J. Rozowsky;citation_author=T. Gibson;citation_title=Efficient yeast ChIP-Seq using multiplex short-read DNA sequencing.;citation_pages=37-37;citation_volume=10;citation_year=2009;citation_pmid=19159457;citation_doi=10.1186/1471-2164-10-37" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. Li;citation_author=H. Zhu;citation_author=J. Ruan;citation_author=W. Qian;citation_author=X. Fang;citation_title=De novo assembly of human genomes with massively parallel short read sequencing.;citation_volume=20;citation_year=2010;citation_pmid=20019144;citation_doi=10.1101/gr.097261.109" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. Liti;citation_author=D. B. Barton;citation_author=E. J. Louis;citation_title=Sequence diversity, reproductive isolation and species concepts in Saccharomyces.;citation_pages=839-850;citation_volume=174;citation_year=2006;citation_pmid=16951060;citation_doi=10.1534/genetics.106.062166" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Liu;citation_author=C. A. Styles;citation_author=G. R. Fink;citation_title=Saccharomyces cerevisiae S288C has a mutation in FLO8, a gene required for filamentous growth.;citation_pages=967-978;citation_volume=144;citation_year=1996;citation_pmid=8913742" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=M. S. Longtine;citation_author=A. McKenzie;citation_author=D. J. Demarini;citation_author=N. G. Shah;citation_author=A. Wach;citation_title=Additional modules for versatile and economical PCR-based gene deletion and modification in Saccharomyces cerevisiae.;citation_pages=953-961;citation_volume=14;citation_year=1998;citation_issue=10;citation_pmid=9717241;citation_doi=10.1002/(SICI)1097-0061(199807)14:10&lt;953::AID-YEA293&gt;3.0.CO;2-U" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=E. J. Louis;citation_title=The chromosome ends of Saccharomyces cerevisiae.;citation_pages=1553-1573;citation_volume=11;citation_year=1995;citation_issue=16;citation_pmid=8720065;citation_doi=10.1002/yea.320111604" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. M. Lowe;citation_author=S. R. Eddy;citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence.;citation_volume=25;citation_year=1997;citation_pmid=9023104;citation_doi=10.1093/nar/25.5.0955" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Lynch;citation_author=J. S. Conery;citation_title=The Evolutionary Fate and Consequences of Duplicate Genes;citation_pages=1151-1155;citation_volume=290;citation_year=2000;citation_issue=5494;citation_pmid=11073452;citation_doi=10.1126/science.290.5494.1151" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=I. Masneuf;citation_author=J. Hansen;citation_author=C. Groth;citation_author=J. Piskur;citation_author=D. Dubourdieu;citation_title=New Hybrids between Saccharomyces Sensu Stricto Yeast Species Found among Wine and Cider Production Strains;citation_pages=3887-3892;citation_volume=64;citation_year=1998;citation_issue=10;citation_pmid=9758815" />
<meta name="citation_reference" content="citation_journal_title=Cell research;citation_journal_abbrev=Cell Res;citation_author=O. C. Martin;citation_author=C. G. DeSevo;citation_author=B. Z. Guo;citation_author=D. E. Koshland;citation_author=M. J. Dunham;citation_title=Telomere behavior in a hybrid yeast.;citation_pages=910-912;citation_volume=19;citation_year=2009;citation_issue=7;citation_pmid=19506581;citation_doi=10.1038/cr.2009.65" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=Y. Nakao;citation_author=T. Kanamori;citation_author=T. Itoh;citation_author=Y. Kodama;citation_author=S. Rainieri;citation_title=Genome sequence of the lager brewing yeast, an interspecies hybrid.;citation_volume=16;citation_year=2009;citation_pmid=19261625;citation_doi=10.1093/dnares/dsp003" />
<meta name="citation_reference" content="citation_journal_title=International Journal of Systematic and Evolutionary Microbiology;citation_journal_abbrev=Int. J. Syst. Evol. Microbiol.;citation_author=G. I. Naumov;citation_author=S. A. James;citation_author=E. S. Naumova;citation_author=E. J. Louis;citation_author=I. N. Roberts;citation_title=Three new species in the Saccharomyces sensu stricto complex: Saccharomyces cariocanus, Saccharomyces kudriavzevii and Saccharomyces mikatae;citation_pages=1931-1942;citation_volume=50;citation_year=2000;citation_issue=Pt 5;citation_pmid=11034507;citation_doi=10.1099/00207713-50-5-1931" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. A. Orr;citation_title=Fitness and its role in evolutionary genetics.;citation_pages=531-539;citation_volume=10;citation_year=2009;citation_issue=8;citation_pmid=19546856;citation_doi=10.1038/nrg2603" />
<meta name="citation_reference" content="citation_journal_title=Developmental dynamics :  an official publication of the American Association of Anatomists;citation_journal_abbrev=Dev Dyn;citation_author=C. L. Peichel;citation_title=Fishing for the secrets of vertebrate evolution in threespine sticklebacks.;citation_pages=815-823;citation_volume=234;citation_year=2005;citation_issue=4;citation_pmid=16252286;citation_doi=10.1002/dvdy.20564" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=J. Piskur;citation_author=R. B. Langkjaer;citation_title=Yeast genome sequencing: the power of comparative genomics.;citation_pages=381-389;citation_volume=53;citation_year=2004;citation_issue=2;citation_pmid=15228521;citation_doi=10.1111/j.1365-2958.2004.04182.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. Prud'homme;citation_author=N. Gompel;citation_author=S. B. Carroll;citation_title=Emerging principles of regulatory evolution.;citation_volume=104;citation_year=2007;citation_pmid=17494759;citation_doi=10.1073/pnas.0700488104" />
<meta name="citation_reference" content="citation_journal_title=Studies in Mycology;citation_journal_abbrev=SIM;citation_author=A. Rokas;citation_author=G. Payne;citation_author=N. D. Fedorova;citation_author=S. E. Baker;citation_author=M. Machida;citation_title=What can comparative genomics tell us about species concepts in the genus Aspergillus?;citation_pages=11-17;citation_volume=59;citation_year=2007;citation_issue=1;citation_pmid=18490942;citation_doi=10.3114/sim.2007.59.02" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Rokas;citation_author=B. L. Williams;citation_author=N. King;citation_author=S. B. Carroll;citation_title=Genome-scale approaches to resolving incongruence in molecular phylogenies.;citation_pages=798-804;citation_volume=425;citation_year=2003;citation_issue=6960;citation_pmid=14574403;citation_doi=10.1038/nature02053" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=J. P. Sampaio;citation_author=P. Goncalves;citation_title=Natural populations of Saccharomyces kudriavzevii in Portugal are associated with oak bark and are sympatric with S. cerevisiae and S. paradoxus.;citation_volume=74;citation_year=2008;citation_pmid=18281431;citation_doi=10.1128/AEM.02396-07" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=D. R. Scannell;citation_author=G. Butler;citation_author=K. H. Wolfe;citation_title=Yeast genome evolution--the origin of the species.;citation_pages=929-942;citation_volume=24;citation_year=2007;citation_issue=11;citation_pmid=17621376;citation_doi=10.1002/yea.1515" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. R. Scannell;citation_author=K. P. Byrne;citation_author=J. L. Gordon;citation_author=S. Wong;citation_author=K. H. Wolfe;citation_title=Multiple rounds of speciation associated with reciprocal gene loss in polyploid yeasts.;citation_pages=341-345;citation_volume=440;citation_year=2006;citation_issue=7082;citation_pmid=16541074;citation_doi=10.1038/nature04562" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. R. Scannell;citation_author=A. C. Frank;citation_author=G. C. Conant;citation_author=K. P. Byrne;citation_author=M. Woolfit;citation_title=Independent sorting-out of thousands of duplicated gene pairs in two yeast species descended from a whole-genome duplication.;citation_volume=104;citation_year=2007;citation_pmid=17494770;citation_doi=10.1073/pnas.0608218104" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. R. Scannell;citation_author=K. H. Wolfe;citation_title=A burst of protein sequence evolution and a prolonged period of asymmetric evolution follow gene duplication in yeast.;citation_volume=18;citation_year=2008;citation_pmid=18025270;citation_doi=10.1101/gr.6341207" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=H. S. Seidel;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_title=Widespread genetic incompatibility in C. elegans maintained by balancing selection.;citation_volume=319;citation_year=2008;citation_pmid=18187622;citation_doi=10.1126/science.1151107" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=B. Shapiro;citation_author=A. Rambaut;citation_author=A. J. Drummond;citation_title=Choosing appropriate substitution models for the phylogenetic analysis of protein-coding sequences.;citation_volume=23;citation_year=2006;citation_pmid=16177232;citation_doi=10.1093/molbev/msj021" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=G. S. Slater;citation_author=E. Birney;citation_title=Automated generation of heuristics for biological sequence comparison.;citation_pages=31-31;citation_volume=6;citation_year=2005;citation_pmid=15713233;citation_doi=10.1186/1471-2105-6-31" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. L. Souciet;citation_author=B. Dujon;citation_author=C. Gaillardin;citation_author=M. Johnston;citation_author=P. V. Baret;citation_title=Comparative genomics of protoploid Saccharomycetaceae.;citation_volume=19;citation_year=2009;citation_pmid=19525356;citation_doi=10.1101/gr.091546.109" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=L. D. Stein;citation_author=Z. Bao;citation_author=D. Blasiar;citation_author=T. Blumenthal;citation_author=M. R. Brent;citation_title=The genome sequence of Caenorhabditis briggsae: a platform for comparative genomics.;citation_pages=E45-E45;citation_volume=1;citation_year=2003;citation_issue=2;citation_pmid=14624247;citation_doi=10.1371/journal.pbio.0000045" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=F. Storici;citation_author=L. K. Lewis;citation_author=M. A. Resnick;citation_title=In vivo site-directed mutagenesis using oligonucleotides.;citation_pages=773-776;citation_volume=19;citation_year=2001;citation_issue=8;citation_pmid=11479573;citation_doi=10.1038/90837" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=F. Storici;citation_author=M. A. Resnick;citation_title=The delitto perfetto approach to in vivo site-directed mutagenesis and chromosome rearrangements with synthetic oligonucleotides in yeast.;citation_pages=329-345;citation_volume=409;citation_year=2006;citation_pmid=16793410;citation_doi=10.1016/j.ymeth.2006.06.011" />
<meta name="citation_reference" content="citation_journal_title=Mycologia;citation_journal_abbrev=Mycologia;citation_author=J. W. Taylor;citation_author=M. L. Berbee;citation_title=Dating divergences in the Fungal Tree of Life: review and new analyses;citation_pages=838-849;citation_volume=98;citation_year=2006;citation_issue=6;citation_pmid=17486961;citation_doi=10.3852/mycologia.98.6.838" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=I. Tirosh;citation_author=S. Reikhav;citation_author=A. A. Levy;citation_author=N. Barkai;citation_title=A Yeast Hybrid Provides Insight into the Evolution of Gene Expression Regulation;citation_pages=659-662;citation_volume=324;citation_year=2009;citation_issue=5927;citation_pmid=19407207;citation_doi=10.1126/science.1169766" />
<meta name="citation_reference" content="citation_journal_title=International Journal of Systematic and Evolutionary Microbiology;citation_journal_abbrev=Int. J. Syst. Evol. Microbiol.;citation_author=S. A. Wang;citation_author=F. Y. Bai;citation_title=Saccharomyces arboricolus sp. nov., a yeast species from tree bark;citation_pages=510-514;citation_volume=58;citation_year=2008;citation_issue=2;citation_pmid=18218959;citation_doi=10.1099/ijs.0.65331-0" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=I. Wapinski;citation_author=A. Pfeffer;citation_author=N. Friedman;citation_author=A. Regev;citation_title=Automatic genome-wide reconstruction of phylogenetic gene trees;citation_pages=i549-i558;citation_volume=23;citation_year=2007;citation_issue=13;citation_pmid=17646342;citation_doi=10.1093/bioinformatics/btm193" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=I. Wapinski;citation_author=A. Pfeffer;citation_author=N. Friedman;citation_author=A. Regev;citation_title=Natural history and evolutionary principles of gene duplication in fungi.;citation_pages=54-61;citation_volume=449;citation_year=2007;citation_issue=7158;citation_pmid=17805289;citation_doi=10.1038/nature06107" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. Wernersson;citation_author=A. G. Pedersen;citation_title=RevTrans: multiple alignment of coding DNA from aligned amino acid sequences;citation_pages=3537-3539;citation_volume=31;citation_year=2003;citation_issue=13;citation_pmid=12824361;citation_doi=10.1093/nar/gkg609" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. R. White;citation_author=M. Roberts;citation_author=J. A. Yorke;citation_author=M. Pop;citation_title=Figaro: a novel statistical method for vector sequence removal.;citation_volume=24;citation_year=2008;citation_pmid=18202027;citation_doi=10.1093/bioinformatics/btm632" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. L. Will;citation_author=H. S. Kim;citation_author=J. Clarke;citation_author=J. C. Painter;citation_author=J. C. Fay;citation_title=Incipient balancing selection through adaptive loss of aquaporins in natural Saccharomyces cerevisiae populations.;citation_pages=e1000893-e1000893;citation_volume=6;citation_year=2010;citation_issue=4;citation_pmid=20369021;citation_doi=10.1371/journal.pgen.1000893" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. H. Wolfe;citation_author=D. C. Shields;citation_title=Molecular evidence for an ancient duplication of the entire yeast genome.;citation_pages=708-713;citation_volume=387;citation_year=1997;citation_issue=6634;citation_pmid=9192896;citation_doi=10.1038/42711" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=V. Wood;citation_author=R. Gwilliam;citation_author=M. A. Rajandream;citation_author=M. Lyne;citation_author=R. Lyne;citation_title=The genome sequence of Schizosaccharomyces pombe.;citation_pages=871-880;citation_volume=415;citation_year=2002;citation_issue=6874;citation_pmid=11859360;citation_doi=10.1038/nature724" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_title=PAML 4: phylogenetic analysis by maximum likelihood.;citation_volume=24;citation_year=2007;citation_pmid=17483113;citation_doi=10.1093/molbev/msm088" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. R. Zerbino;citation_author=E. Birney;citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs.;citation_volume=18;citation_year=2008;citation_pmid=18349386;citation_doi=10.1101/gr.074492.107" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=O. A. Zill;citation_author=J. Rine;citation_title=Interspecies variation reveals a conserved repressor of {alpha}-specific genes in Saccharomyces yeasts;citation_pages=1704-1716;citation_volume=22;citation_year=2008;citation_issue=12;citation_pmid=18559484;citation_doi=10.1101/gad.1640008" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=O. A. Zill;citation_author=D. Scannell;citation_author=L. Teytelman;citation_author=J. Rine;citation_title=Co-evolution of transcriptional silencing proteins and the DNA elements specifying their assembly.;citation_pages=e1000550-e1000550;citation_volume=8;citation_year=2010;citation_issue=11;citation_pmid=21151344;citation_doi=10.1371/journal.pbio.1000550" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Awesome Power of Yeast Evolutionary Genetics: New Genome Sequences and Strain Resources for the Saccharomyces sensu stricto Genus" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1.cover.gif" />
<meta name="twitter:description" content="High-quality, well-annotated genome sequences and standardized laboratory strains fuel experimental and evolutionary research. We present improved genome sequences of three species of Saccharomyces sensu stricto yeasts: S. bayanus var. uvarum (CBS 7001), S. kudriavzevii (IFO 1802T and ZP 591), and S. mikatae (IFO 1815T), and describe their comparison to the genomes of S. cerevisiae and S. paradoxus . The new sequences, derived by assembling millions of short DNA sequence reads together with previously published Sanger shotgun reads, have vastly greater long-range continuity and far fewer gaps than the previously available genome sequences. New gene predictions defined a set of 5261 protein-coding orthologs across the five most commonly studied Saccharomyces yeasts, enabling a re-examination of the tempo and mode of yeast gene evolution and improved inferences of species-specific gains and losses. To facilitate experimental investigations, we generated genetically marked, stable haploid strains for all three of these Saccharomyces species. These nearly complete genome sequences and the collection of genetically marked strains provide a valuable toolset for comparative studies of gene function, metabolism, and evolution, and render Saccharomyces sensu stricto the most experimentally tractable model genus. These resources are freely available and accessible through [www.SaccharomycesSensuStricto.org][1].

 [1]: http://www.SaccharomycesSensuStricto.org" />
<meta name="og-title" property="og:title" content="The Awesome Power of Yeast Evolutionary Genetics: New Genome Sequences and Strain Resources for the Saccharomyces sensu stricto Genus" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/1/11" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="High-quality, well-annotated genome sequences and standardized laboratory strains fuel experimental and evolutionary research. We present improved genome sequences of three species of Saccharomyces sensu stricto yeasts: S. bayanus var. uvarum (CBS 7001), S. kudriavzevii (IFO 1802T and ZP 591), and S. mikatae (IFO 1815T), and describe their comparison to the genomes of S. cerevisiae and S. paradoxus . The new sequences, derived by assembling millions of short DNA sequence reads together with previously published Sanger shotgun reads, have vastly greater long-range continuity and far fewer gaps than the previously available genome sequences. New gene predictions defined a set of 5261 protein-coding orthologs across the five most commonly studied Saccharomyces yeasts, enabling a re-examination of the tempo and mode of yeast gene evolution and improved inferences of species-specific gains and losses. To facilitate experimental investigations, we generated genetically marked, stable haploid strains for all three of these Saccharomyces species. These nearly complete genome sequences and the collection of genetically marked strains provide a valuable toolset for comparative studies of gene function, metabolism, and evolution, and render Saccharomyces sensu stricto the most experimentally tractable model genus. These resources are freely available and accessible through [www.SaccharomycesSensuStricto.org][1].

 [1]: http://www.SaccharomycesSensuStricto.org" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1.cover.gif" />
    <title>The Awesome Power of Yeast Evolutionary Genetics: New Genome Sequences and Strain Resources for the Saccharomyces sensu stricto Genus | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"nV__wXqzZuGD0y-gpEaCPDGIaj3GuihRRkhwjvEtsRE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459154","apath":"\/ggg\/1\/1\/11.atom","pisa":"ggg;1\/1\/11","ac":{"\/ggg\/1\/1\/11.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/1\/11.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/1\/11"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/1\/11"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/1\/11":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022W6aK0Oezs4sxPZzuQH4T77QWyWQUBpNhjh3BP3r7FkR8ZT0YNBsTlbEsVuXgx78KSr6XDCARgKK5Br\\\\\\\/XG2E+eA3lVeiSIyGwi0gF8RBjz6a7fO5gQrGIWZx1dLfY2RQzAVVHeOfCMBEsk2Fr2qkIc7yBwkuSKBGPT7JH7Y9sZ1chXcF6mIJOYhcRObza50aRqn5O\\\\\\\/WqJGbCEgBeeLzodXcf0LabX3U9FbwtqOag6Vyn2YHkClw67MSXTD2y6DOjI2QLVYpVihiVqS3YOBTzFWquBmYJR7dtgUfnNvWIyrYOULa\\\\\\\/oHgIOgyXsXkc0N0gStWEHIs10YDZRqNsF6Rk+nJbCUZGkYvr1Xi3OWO+BXlOaIIHg\\\\\\\/4+XLUsTHl3bbcb+8n\\\\\\\/sLmm+i72WEVssmcVMMQgGKiuDXwlQFrhIQyzQpRHgLCILRJzd2jmltb11Ek3empRApzxil8mnNXQOeo1xxM2JmGEeHR1EDJeFPfp9PJVzEiH72Fii7JGl3Zz7mM8ODGXynGYhcHm4elzP+cn\\\\\\\/tbr1AJgztppmjhqLR7PRkBFQPu986+w4Nlg7cR\\\\\\\/ohgRgDkkO++EQIZHSIP4vkKfUU0RL459mtbjOH3vAgCwjo2AsOq80YiSzImC3z+MWlgv1KsPPL5HUZta0i00o3h95FQfZvelQbLtSYSdtQRpBaMcgPU17HvkDkkXzWhgT3gb8l0fd2abvn1d9\\\\\\\/Ng+45FnGlsuq+p8Z9\\\\\\\/Nqzib0qXA5Nr4VjEk81Ep9rwgRKQoWzueRLszlmC2\\\\\\\/cOISzxKScY0\\\\\\\/CbWCnosK4Wn7cIIaQ+oxdhxeGWMgMzfleHJaJKdqwOgulX\\\\\\\/Wx03ijTr9q8CwcrtUVTcMThKKEDpiAFXoYbyAlL7dvmQ1TyRmdr3C+csjTb9Bjv7cSxSBriXUq3KlXxUEewfKZOfqE4SD9\\\\\\\/vGYaB5a+57Ewzs8wBS0oYWfhI60lB8jtRA\\\\\\\/5yfMU7Gu2ZXYBuxem8noR\\\\\\\/Be\\\\\\\/yfigrVBShFhmiFP18nC7UBww+S+m9Ihh4i4iOdhXhaJtKxyAe3+wRoSJ4Omtpp2hX0SF9K2g8hUM7Sr9mNV5sJNhJAkKZufqq5KQDPipGP7nLa4ZmRUoLAuZHCuepGgLtvV3DmI2\\\\\\\/aaLVxDYnRznjmQNYAVwtezcUV+iMQEu4xNH+MYl1aw4NG7gYkbFLsSy3rAWHrWKkMJs4XgPEFJwa1FkmoBHqII5Iwd5+Pio0E7rtAkQ7m92xLMmHD\\\\\\\/EOjwIgGFw5wH7fQq2pljcvCYZZQkwhwemNdPYd\\\u0022,\\\u0022iv\\\u0022:\\\u0022mCm6nMAP88us3\\\\\\\/1A\\\\\\\/FnAOg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002270b8a57568b6f75017fd7c133798c8d18cf0d242deda7bd99fbf3438ea192994\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Ij\\\\\\\/kMNNAtG0qnX1R2mJB31xGZvNpw2hN6oCrClLWfL9nNvj4CSW19JbNXyUUt+s8Nh\\\\\\\/oUUSOSBAeTJmadHQE359RAba\\\\\\\/vO\\\\\\\/MV8lNuv4vXIuvRC\\\\\\\/HWlP3kRj9dPdR6HOiuhjLmzL2AzXTvfBOI98sja0Q\\\\\\\/aT6fn53gZMzJa9uhU1iTVEuQFycCNia3AqamY4n5OHRY+VY0eVoSACbUIlY28edvfF04uRZUDbMfOi9vOwC9vpGAl54QutxL9Ydj3aSXoRjAA2MeS3WDhYJmifTdpLTHh9yIQchUsc1wYNZcXZapbnoW0FJ2HJLjEoYnBqc5lvqUwl3KJXYP9YQdP3jUlZTAP0BHDrJ8PwxBTVo1dXiucK0auRSn5khpzskjNWcTP3ljfAvdkvo3MTz+qk4InPlBJHaflx4YHCrYL82HOqYGSJm5N6iD+2ThifLiQOClc70x3AjStZMe48+OKcewnI2mIuQgow8zSoiayTDfdeHzllkl6+aoaCNS\\\\\\\/Lnuw63jdm4qlns1gEKRFUc8kogstXzePzqG8LKDNg8sZ34v1P+f4dsvqEaDcwEEuVKoz9+v4qAjqrtLeCPKPp73MpBMW8V2ZNo\\\\\\\/\\\\\\\/e0idWSGiViVwumVoKKgpv228LbddzpdXdDQCje8+6PWMB+YdXF2iWzmDmv0sS5\\\\\\\/d4hDsDYPYduAzbtvZWBX83hw54ALHT5ZWgRrPLjrrkn8eIUBJDEPD3Q+bYsebWjtwWt3yiy\\\\\\\/uAtph2Y1pkFyoe1+H9IBMzKiLW2nsW+XuIyXCjazo4LCSctSj1EB1tGLcxfpswooMjfi7dEu\\\\\\\/TjFO73I44GEtvTnwamGjY36oHEcDNpmSQxixQiSoKbgBYrdqJ+EetXxjb0lq0ileK239ywJyyiZUm0QNdDifgSBpCPwb+h+w6YllXn9ESADOBhyplAvf8dNzyJCbQgWNgBAtTA+cM\\\\\\\/cbi+ZIG+JsSrKwe+WFL5pUAx72q8dcd1YC0ayWtvAlNNTuQ3IHqnhjYde1bZufC6du9akq+pWAIdA9xyti0wynZDR0bv\\\\\\\/lV9bBQLbnP+EbLf92S7pNnbWDdbHvco3YcruNOXlNtmtiZohj5rh2WcJrOOub01jmZBXZqy5V9B+QRFvyl05Pb8HqUxufVzjGQ5VV\\\\\\\/paSlDXQspeBDRx\\\\\\\/LEdN0G50XQyj5iUJ1DsSMGfLeM2wFYBhV8AIZiv68+j3cJ0jhTvC9LaS6Ky3t9nFo3WgxDd4BxqFs4UJjGnVuWwFAVSG6tRN5WK96va8CFgGxkNA5jfg+PuDsZ3ZtSklnjNX62gXalfCkhOcMYsqWtkEnpS6Nv6fSjPhRAnBbqFe3wnZxxYEz52S0dlJcE26WXVDI7nKVPVGhuDYbNR77S7xGfxoeh\\\\\\\/Jj+DYTtodYSv+TbmmSEKwGf8P4rJ\\\\\\\/cbFiYId9z+7xxf0yW27JYvwhfF\\\\\\\/dtDJVm3OfCVmRsZAGTNeeO0wT0Y4q3mhZjE5QtdMBuoaV9UQkmq7fsIMqhEFHRvHYbgh2h3igyrbFcB8sN3jH1z24bg770fKnyrRO528CEa93+ykrIKb8t7NZztV2Yu9W+rQDgHYxi0stUpMfEkX3sw4cgrVRme0avINX\\\\\\\/WTmrz5p6MknswMddJ7uLLHzK7mXuotdbHpVc+vM585+Be0\\\\\\\/VVN3gdg3gTQMqkEYyKO2hEL4QtUXNfmwznxiHDsoEAMxRdy0bG6mvag2OnVu5yYR0qE3NV9i3GxfR5Z0d2RNHDnj1DUF2i1IEtncLP+5SAHj3O7dy\\\\\\\/ThxdvOut0DCJFhK8RtAvuJp4wVQqwwP7vmJbveOZ4m+mTV6TYBMhSJHsx\\\\\\\/JMaeOeunDaHmZNluhfcurCva+N5SigBGvsSiGboF+9lcp244r1hI+1UqlHtWyhe9fTVSd8J3CNBPQbST41xZC2UYYhEuJHHdq\\\\\\\/zz2J91yjn5sj86waYtKMygH3RuDJfXzwCx09s1QoI9fkLq5J6zLiFFfb78YRly4EdwiI\\\\\\\/3wQ6ofPCBkvE8V9KyuXGzzHkC5UA\\\\\\\/MYernv9D3cARfgqCPJynyVZY6WRmDJC5IqU7qFcs3khd3W8SRtNxejM0W\\\\\\\/yJFtr7TYjzLWYfz3QAjrhNWh9heShZMOecyv49XNXtuCn3jWIlG20YbUCkfTtflVyZWFnhO\\\\\\\/o2IV1\\\\\\\/xWE90+dEUV9\\\\\\\/s53Y9mCTrMawPmynNu9xFpoCxf3WuR9OnEhioO3q5v8riUVBQruRDFvDVfUjTRUKurToSDyJs3k0A7l0gHzmxOVLyNW27Zmb9pMPxC60xTaAr89wNe68lMaOVIkKjcJ6TGNELT7fhUJh5c\\\\\\\/TbDo3xnQC48egYNeyHXyfIpIvs30CUsKH8nUM73IE12X+VEMzM1YZzLIGMCXxrzsKWk73xICDl5uVLAoqiZQaRwJScpkk2+9h5x+7HcAGEMjiWTvt7YvqZfM6bYRi0\\\\\\\/z9KAHTffpd2PGuZuVDVA0wjPZh1vj3RorXQOQBdMROM\\\\\\\/5RO63VUUzrMiqFMDmAu+jo1GX5XK09y6CgUQvxkSdsm96QwkyIVJYdAQ5MORUSaK9QrC\\\\\\\/OoufrA5yahKQph78FlZNcDvN79jZ278JpMfbQB+qXaq3OpFXk3mwbbzIeUwZadA6wkJo1cma7YMUvz7WljE8rJf96UjHGS2vzQYtl9TTCchVVS85v3T4KV\\\\\\\/KPD7HjWwqWodDGym65vKnOUUTcePPCugl0ojxCBvV0wDWvMiNKS7g+ZF2Akj0eJV0Y9fUfxEqKsy1S+npg9hb3xAWa2f\\\\\\\/jaBXyWamXoUXU2C1m+Uvn+QWjFQSukLqvRaJD0SKrWLqMgB2GSFMA5NhzgI2lBW52jTyCupubAUvQNO4CscoaG9LIE342jtpkT0V5Cqk1uMpkjV+UZaR\\\\\\\/iy3YgqzQIm415Ehs\\\\\\\/Gs6zdghTOl75UWm1gydH9uFmpK53lPspfcpHHUX70Rl5QctjNszA5\\\\\\\/Io42n73ChtA\\\\\\\/RSV6TnmbZcihHtPFBPNX9I2een8r6QRB3\\\\\\\/TdPYAkD3UvmIQjRDQI14Ch0vXy0JG6IkjFDRxTIZ3R24F83p7Jt5raregtP9l3KhQG0mOJViJWPgSMuVY+szsnFN63dpH5vGhDfqMZ5j+6S9mGQ0N+aasAFdgoX6cd8YOIVEdPRbJun\\\\\\\/JT2d8W8rDpYikBFn0vrunlV480Aw\\\\\\\/6SowvnVrYQRUxdRYu2FTAi1OH0tM1aadtqxipiNPyzjEXi+qb+1zLVpqZ4SSSOCgPBx0ti\\\\\\\/VAfn9yZDjP20d40gA7\\\\\\\/EVbx++RypfHFivuCNdGZ\\\\\\\/8v7PT9LGBmzdh\\\\\\\/lhM53SIkxXtqV5UzmxYMPdb70He74JFAPdR2U92XFgu0W3UuzE9RpJlN3YFQM5GvGbemkjOovBcucV+ucnLVYsxRYjG0Xx5IAXeHpVylhI0rvAy20NtBQhdu+RU5BmMN4TfZ2QezCytYc4n+FBANFRJTN6h8+kuMeZOtGO4ABxJfRxZggpKbaADS+q+zstjhatwaWdJ5PmZ2GR7YJI7CYhdgDMnsWBVlC\\\\\\\/QLD7KDg4ih3WmPahNujXh8gpPblQXw+lO\\\\\\\/C5hFX9MgxsDfbctBJv33+Wgou+U5Cpozs\\\\\\\/gx8oNR1JjrTx9nYS7yfEev+L613F1ArvIzhlHp8h0crcJ+RnOdRqkHsD\\\\\\\/taLXNkCL4myZsFpLjl9oqDOC4z89rQ28YFLsfPxImPo5VZTmQwD7iqNADMh\\\\\\\/TE29KsRUGRLP9AEiOeh31NLPr439d3CsLFrleY8YBr\\\\\\\/qQwMT5vTr03XuFLWB4RcpiflW+a0oBKzCp8gBjS21ltiwBDVBzh1rXgYdOd0G0NX+JdzCAq4O00cwnWjew4Wl4K6M9IiY3i\\\\\\\/U9lOgntj5Yk6XI4HeaA0oqs\\\\\\\/8jysTbcijQ2hJtIxndTSCcxZ0bxZbGnnZfuxeeq41UeqJAEBKhGP5N9apXo9KjTxD+vOUTbxLjOBbcy2WSBUpIkkVf7vdPi9ZX30HeRcdF+t+iRR2JJb9HrFl+rW8GcA3MFTXV5GVN6JUBbGkKC4HreSDhhdyoEjF+B8PdJ6xnuOtZf51ewoD8HRw\\\\\\\/vqWy+0BrwmnsAlOtpqvnobum4U+QdaGVqBCvQ7nSBZ3\\\\\\\/4TdmkS0R8oJVA325mylzNQzPOyVy7jxErZIqVfRXNtO\\\\\\\/H6FMfuKveE+dI5r87b65db467lvSSKw3WhcuhTchOuZBHXQPe2mo1oRtStuuSrGAoFFnyzO5ycFIwc38Qn3ESVdNrSgvAx0Z3YRph58n2wCj3YwOeFyi1HvvBdPPQ9ygKuuYJ\\\\\\\/D6232ep8HSIMjYBcJNQSXG9ep7c89+Gug6\\\\\\\/kIJeu\\\\\\\/jrGPdCLTyC9U8\\\\\\\/VNT7pyQAHZOa4cCF\\\\\\\/qhCq3knHmHAdv+eqDdFcubjbE1bJiGCuCbrD7\\\\\\\/EwCEPbWFtdcK9nv7l15Nb5b33UJG5o8gmg6hlWlVp2iLWFGXa23yfR2z+zAHFdvSWxyagd4GE9udFumFsupUSWKg2kDffqAZWrAolc4LKVbTSUcFRtYbO1bZ+12+M\\\\\\\/eS7vcVnGqI9fTEN6AAjwcrUzcXigaU9kgIjMWW1dZlkG2dKHx57FopEhMstdqcbsANAcsDfCeQS\\\\\\\/O4GTW1TlEsC4umvOhWm4GFdgdR4N7ZcjVEWytAS8xtMMMfbd35N55ia0vjBmZPZMVsw5mgBx24xu1tgoNC15snHb7BagulD4zf6M554I0sJ1vsnHCV25hIJSEk38ZBpBulmea+qCd4IgaCEBDNbd\\\\\\\/Lj2Ljb5dJlo2zG5BclZ6JNbnpSRMrYR2Q8uLvEIxm9UYCQ17l+6DoNTa7rV5eBJ3seBxcDb88KYfq2i3gG\\\\\\\/PrHU3OxltRxGKnFdSc6r917GQ9wujWvp\\\\\\\/PorzJcwTNfgQZcY\\\\\\\/dpefHBclgGe1RmWQaN4VqNIIoetDPMwublt5ajbQMKxWDdtp3WURZ\\\\\\\/qTwhJ2tFH2nG6lF79gYV+EBg7xdEMgcy3\\\\\\\/vsjIX6jw4mVfPDHFndUPs7GVn3Y5eSmgdCmrYTopqO\\\\\\\/8eFKYt1ANAbhIqcGD6zLfoxgnnDPlUBq9vaWoJXN7bjZnCbItU6Gxo+ZIFfUVlGq93xbJBmzPQ7VcjcXRWDHpeufEJgFeD7TBmO1e6NtjYgb45qzcCoplTVjElKu9ccTXwS8w+ep+O6F3pvx8nkuLeTE1\\\\\\\/pJJZrOHEOZnL7CGmm8Zvbaf1L6uLvuL0\\\\\\\/Wcje3rt169pUhsohXtNchyKjwgiKPB+ySjkaiFen2MpQuhm4l5s88qCsL24dJA66tRm+hCZqIJ7Vm0+dQ4NRGTnYbMKnRI7FWjNoPH2Qv40c2\\\\\\\/oib5qd8eNb\\\\\\\/O1Cy7fhx603NBSfnRyemLqm74PXd6xsNSFEMizDvyxI8MA4oWNtC+IU7L8TXJEUrReEXv0rkzsuVdEko83crtXSOEKDaFpjwlRQENsDwS73+QABj7Kd0X0jzAJwDEC5iB7GO3CXOZSLJ3yXVzFCqkwWfNljWY51toZ3dhx1\\\\\\\/TNHtili2NI7PISm3Bya0A0Q5bdpdsqASTndvuGhCFCq6xTMQQWtMLnjyGdZ+pCWj9t2nMfe\\\\\\\/MhUiLtCIi1pguSXH5g1HL6VVxmhDBLfjF1qtiEyN8f0u5n2gbRMKR826L+8DyUQMjcpmJUp38WBT9ln5RkioOHHkfS2VX+\\\\\\\/gPQSFO0s9bVAHgBA60A4ghZA5fE9zUs0de4bRjw1hakzvk\\\\\\\/w8ozCpS2u3XhNW9aDm\\\\\\\/AM4XDKYTezer8S\\\\\\\/5CHetTLb1eDSxz0piH8sG+kGhj97Ftisrx0LqsDv40cWOeIjXl6clt4OG29QQxXmjavUunZLMLtEdQ8ib5lwsmPHJMYGSYVsgKP7+EczZmK1l2PptzdURsSaFbzxP9if03FgTsujP6WjiNaf+cNSPVmkwsMNJgMQN1j8MJ89hDLXeqgHv7DdyDP9zadnDq2q51Y\\\\\\\/DuJkA50mkhxi6tuMBha0Za2CvzUaQyFCyCGw\\\\\\\/djHSPL7ZldDsFZ\\\\\\\/\\\\\\\/Dz7mYsEEKAK84prQjopfVJehv\\\\\\\/86GjNNIq0BEcmveqYSVKbyBrTiV5RqKtru7lxSFJ\\\\\\\/cC24ficRyzDdNtc3w9DYncSt1dwlFOFHu7es8l\\\\\\\/kgJke8woIlPaV7A5Kgkidi5XMQ2hWnldQ1O71LnXvs2Xy0XXN0r4UzSQvUfw1jOsG1uk0sq5f5uhd7HZ6xU4uJNnEIXTU5HnYBcJiHvmtms\\\\\\\/3T8hbGMV7LoUFpSByxWW1LvT63HEBylQbLzjlcaBOLAlSMWpUf+4i3pNWTZPg1CEUKqoKRdsf91C9nVWcMnDP1MoFpfuj1Y2C1fnBKFxH9VPx5KSWyUIRwe9Sxl9b\\\\\\\/v39k6lfiv3t9r8QEbai0CisWyEpwGZdU34hBqT+TXE\\\\\\\/dzxbS02uii0jVaurhkyXi6dBTjJfjJrCR6mgPcWEXo7QaZprXFbHyRi0p0HC1E4nVYX30bPVZw3vlGOLcskcGYc7kLfeW\\\\\\\/DApHjQUcW3sJL6GklfBDkQg3nZZXfHHlmqSuVSCwlfmkSbp71qJE94vzgBzPfuKdmnn5XlcnLvLaaHWqm\\\\\\\/SF2FsSQwIA24l4r6HbHH\\\\\\\/eonTxlBc4xu3Messo8AUII+rGfZ3vXb8M7JOgzq3baq91FGC1DnLRnQGoFpvVPa1soIuff7Y4cJJ7YAGT7XsaMoagCuDwfuk30jqh1AEQKfZ16tK\\\\\\\/RUDJQv7ei2QFlYvR\\\\\\\/2qqG7tAxKdLaq9B4LVogFvTxoAxNtbyT9ZiEdj+eAU9aBcyxeyKYuqMAuAz0xpM\\\\\\\/LTwvGscLejgm7pnUuS+q5wAuNy53ZydT+EZGVQuWK2PlpgjXsimlPinF2cY\\\\\\\/TP788P9B4\\\\\\\/tG8hAKJywrYgtiqmPaasodi4owD\\\\\\\/993EfvyTbbdkI7pBpcN1NnR5sDw2wqY+FX6SLWUhDVJIptLVVNHZo3tazNlOXbdWsBDVxq87dPvar8XEOFz0Pzp0LWy40qA4tNA8Xr+aSjNSlCdhykkDR5qrSog0xJTF39XiZwb0asiiHII1okni7Zr0wtMo847PwHPJgyESOfGn7GSCBKhwRFguuCyPgsg3tiTQPzNgh5CpOebNwOLi\\\\\\\/ViAfexLbn2I1WooK91Ecmj1PmKpsPkjjzYCL2EYA\\\\\\\/qfyyOlmwSAIRq2+qR5axY8mbblp7ImdT4G0z5OOLRctalC7VwsoKKPSuSXoaWQvMD3GsO2+m5fIBPund4\\\\\\\/hWQ587RUIJFLAgOlrhlZo3YgBCh34cv8BIFptTS3fj3fqbwHMXS2TJrnBO69Zirvf9HHlFTPuCHHCP+ff\\\\\\\/YRbo2FyITSW7egY3vQcUmwP9E0Y5G2T8t9vcKjTUqnKsySoSPgos1kvo+uTuN\\\\\\\/uRLRznI\\\\\\\/+kFaJxyb1Rhj+HuyIAmcF3\\\\\\\/ysL9cI5iBcdQUJe\\\\\\\/X9Th0JEgw9qxPUlpMAA84MG5A6yZqJ2IpFQ8ztJbnimu\\\\\\\/HKrVLM7lH\\\\\\\/Nsm8TwkJ\\\\\\\/AWdX1pNWKL5\\\\\\\/PBhg7v8sXYVadEoiZpzT9gKTx3KSxijreWPOJbguilelMn6RzE63XdaXogSv1QY9UcJjmBC0YVHIrOV5\\\\\\\/kZNJqSaOtxqWh9\\\\\\\/b6cmhX1mYcN7CJgzn7NT5FHbW1ZTLaWBu61QgtMk315NzljMHke\\\\\\\/3aQY0U1dFSi2CG\\\\\\\/LgzkX0y6OcpyA4v0B7rONS1N\\\\\\\/zPRF2JtHdNPc\\\\\\\/c7E\\\\\\\/bPLX1ddVVmIz9FLHj0yCDlF9JzdFJtRvPvz8CbcUpjDPgoBgEOz9XAj+eKwU7TdN50Zi\\\\\\\/J+aZpNtykXvgVq2sOZ03o\\\\\\\/H25xa9yPFi9ECqeNEdj3Epx8zmQaR7oTe52PWWBrcTOF+d4aDrsM22Nnh0ObQCHNkxC2x0zjahZizShPHWeg6MthptKXZiqA\\\\\\\/Z7s8Oqz8RKpcdu6qJwuPe2dixrrL+hUjOKnItEdta6NkQSh86SgOdMQC7V+rs5\\\\\\\/XDH7cC00dMGXWyvzZkMHQHW8RVcUeg0wKQUI3z4eJXq56pxF15Byq6FsPF1qKiErvxgh\\\\\\\/T+n3hLgs+Vrp33fy+x6sxQYeKfC+n9vNMOA9mump9wo6zzmvL0YZY69UUNgPL6rSsnZtQzNsBU1iEUVajs6ivsLdnIs3jQ+bCMxficWN3uF0Be0L8\\\\\\\/Av2XdZtKBhawDe3xyGoxTcOvc6\\\\\\\/JpWOQfXGQrojupqADHEo+UkLXg4Djg7msGcxkGSz7wMdSu+e5j2Jq0mHmEfc8wALOMVyfR8BunmfYd+wl4cCr6prYcNfhEh9xBPxiCkE7og5UDt69kQOiW9vuMzk090Ni\\\\\\\/W2QGTFrnjoR1olvyTECdGMDhlMvPqrDW24g4tmDEWKVj5u1kWAFO8eNpnDrDTy2Rf+9HQ2KWI3yKwmV3yG\\\\\\\/GEiEQ4anfG6kWcSMeAPPHHHWL9NA3u24FZudY1OKFpUpbb3MmgjmpOGK7m25SzFup1iozDl3vH72ejHIQCK5dKIT6HtPfAhnoZbOf5DZfCGIamJgovN16m3rKo6JEJjyxJoFjAEVxFTs1HGLS0ZDgl7Q4Le\\\\\\\/DH91ZCr+UI\\\\\\\/FZRQIpdXpyyi6e8DhtBZxaJa9LHMH7X\\\\\\\/ikF9KDXew2k3+O\\\\\\\/E6C6wDwcAPeu3Cl760zhtaD5S2HhlKYhSHzwHBvqwJoLTYXd96wdTQlh\\\\\\\/n82Z4+ruTv3PoVkGSbQO4RuZem9kBYSJLBetH5giT\\\\\\\/OpXssZwdzCvLg\\\\\\\/94lrm9pkwqyBhXnxVKMo3t9UuEPXYJ\\\\\\\/wb7oe+2A2X2bhf9puiAQc238INU+ltmYqV7TC0Yng8hZa544OTELaWXoGNYGsUwMHYpT8aKMzCranNqFIoEqDAVCecJGAYoaZ77CTJf\\\\\\\/I4H6TThjXNxZHeL8y+qqtsqzSB+Izbbnjg8zSgsG1A5NsalBtWFto9xjagLafFEtnVGZ9sAmvPPTpxfU1UDqVo1dMveTx3GKWbajawzTM95SKjmrAy6wl0hupIa6jSyYorsSoNz+sMvxPegNYlwXZtsVAtfynZi5YPL6y8T4G+6EQ7t5Kty\\\\\\\/rmqEIy2m7I8RbjGHIRUhtMALwikWFfEbjlnYEhFItByH+Mu9C4WHiDnD4CwryznIYdEHaLQ0IiPUCPrEXGFE4Ohw7qiMbi40KGfx89s32+wtADguR848ZfZAoTkfaxUsXWN+KtKlDHv2kTBL\\\\\\\/ucqE8jAV+2ltBbQz\\\\\\\/b9APvrgLGGWD5JhZCuGzXtBOi80KyB6V3YmCj1KO7z3o93QCjfqoVpQAZspWaQAx0f0dKGA8imdScP0eDakN7zkZL+bYtJbbqa\\\\\\\/U7K81WYhtfQ3AvrS+IHWZmMQJZ0N1FGILMSsTj+SOcaDeLX0\\\\\\\/u5syq1r1f56BWgz8BGD6HQ6xFaS1Su0M8bQn4O7JG260+PgoqqzkY88pdW8DCcWiLPw7M19sRN28tPouYez4clyO3xNYwSvlVO\\\\\\\/lBJm0WXlWItUZgLz6wR1q9XrFxC73ZzmvG+qw7ZK7CQxqSwdab2\\\\\\\/Aud8ECusz+W0O9WdoLHLaRwYrIQZ0tc7rGvqE5zO+gBR2cBnDOnkI2qE4zECRn06OoXqfj73\\\\\\\/a+6fuqHovfJhvCs\\\\\\\/1p4iI28IFNUalTBsdFJVWib7I8\\\\\\\/hpViA2P5prYSCuKs07If4i3Ka\\\\\\\/ZDy6dMWAPyKQY\\\\\\\/Aw9I3RelneebkRAABTPTCkHTwIHinHF92GvJv9H\\\\\\\/QfFO3g5Nb7ouVNfTD2U5+NxmocXdwuV6aqff\\\\\\\/Y33NTT9jp48hRvjedZsUxZlrLrwM\\\\\\\/aLH\\\\\\\/dny+AYpzk4cYX\\\\\\\/v3ZJ9eridrRRjVLVBS9ttZkFL4WS+AiJJpCb+5orIWtse4YubNyFkHPCGFLIYaFssMuQDGAeGaxCqVHLJKNADjfUWivs7sNawOJzBizcXNbEhJJxSwbQ\\\\\\\/+Qf6K5J19unTiwTIhEewP\\\\\\\/LGAGarzIc3\\\\\\\/QbCYOYFgfhmm9DbnvSpiytYAMCqhdflaUqj916f6QTENlTyDem7tQNnIMOBnZYJEBmdYJ+x1yHEwWVnzFOlogF0b5duhJXFVIKpX9GC1FicE5MHuFbIgtXTfJeGpEypo1g9kPYx60X0beHmMy0b5F2xDhCTrWgA8xcXdiXuT\\\\\\\/7WpJekB4W9PsoJRvlHkJnlCeKGZaFXzLARjiu97bH4UuFsL1XC+vuGP+wLvxrsN8AoqcZjSv\\\\\\\/6bjp2vN9bVUYX5d1TV90rYUFj3yIykzsyh2ZBgZT2WGGdotTTXl6QbFF5H\\\\\\\/x7jcmaQt3znueSqKfUZspzLCpQjd1czDkSJV3P0oMUUgEiQDuoWt9DXNrfKY0OK3odAmxZPfrLybeZgOEumsX7QJlXCM6J0qPuAe1CSJwp5d6KpnSB7XoEUvJAjqybgg1DpPLFDk8XLW\\\\\\\/emJ7668I0sJ67lAuphiz\\\\\\\/3hBsWFreqky8HGUOLSSND\\\\\\\/rVJe2v2xy9iu6GQGccx0KFbYmBcE1\\\\\\\/KV5tx1b5YukhCCfdP6kKqlCOhmLB\\\\\\\/YW3Kboez6PseqIY\\\\\\\/U\\\\\\\/MJGmh9\\\\\\\/DcDJtTLb7NtwucGr3h\\\\\\\/uom+YDNXYLxDRP2n2h7Yh1yWviWMJekmw2FkGUZFeXIaT\\\\\\\/zvcSBG419L\\\\\\\/JrowEL2V6opTIAs5ZVGiAXGm7cqwKfjBIUwav6TArkgwfuqVRbp9nrd7z\\\\\\\/Ro+NaxWaswiIfwyi+T7KdS7nd\\\\\\\/Mo\\\\\\\/lLjQyxZ4RnwZUnxikmzLR7Fc8slo9HzZzS4tPIfHQGa6Lc1Lc4XQktZV5p4uXlieMVC3xpaWqtI20zhF27Z3ew1wtcs0mKF8WMANvFp\\\\\\\/58CAXkejRCCyI\\\\\\\/TMBRvUAHF5tFKSz4eqMsh8t4lMv2\\\\\\\/yoW62oVgHyD6waSayZDSBUcgheSHgvAz\\\\\\\/9DMcCHA0wxutR5s924MnOFmMC29KqCjuz7jl07eZdNZKmqlx7yyk4SwM\\\\\\\/WW1y\\\\\\\/2SrqTlqdUZhgTRJzBObKezr0TtigYXXPmWQ99izIGyyF0DrQR+vyVs8VugW\\\\\\\/8aqU61mISk9UUESNpiRvFkcXXlewJbTB4voKoZVSIW\\\\\\\/Kmttp8Hba4f2kc\\\\\\\/j71Ajr+KHXWPf+Wrvp+Yz+f8l2SxvRrVQU6roVaQPE1MSzhfHpEmHAHqqr7MmautJVkk6y0NGz1BkYrGDQhhzRfaYF57YtsrJl7yucGtKoIHNgOl5g50un84p9mFV1Xj0Kgc2F8UHfPC80t6WwDCeaXfif6UZOsmNcaaE3BG9zTV\\\\\\\/XQUZjaaqyQLxjtr8fd2va9zFV6AVoquaNPZ1a2sJhSOne2F2ZASZkbADoeTE975MAGBXAsb9KT\\\\\\\/ST5PVcvKkmNbqJxKnbDZojlScu30ro6QElGjJCcL19FXMtnfHBGVYenmb9IzHqyXERCkK6Hf+yFF6gA+Y1xKbv\\\\\\\/gAdrL2tndlrDhBm3i6vSyFH7U4ygCFbMYy4XQopt898YglALIDoSddwMClwT4vsJQ1tAqIHlC5x86PV1Kr3FP54iZPSyZYqKxFnfD4b2d0yHoyXOVlfACSuBSAott38DCMxvqEtZFxhjvNhNxxjG7KBiBvgrphMteSLL1aVvnouEr9dmHGCVuo2K1pMojgxpxnN\\\\\\\/rMAn8falTVbcaGPHgLPHxwiXBar\\\\\\\/z7WvQALzJbGWY0KMTjhYjcfQtYVAi1rP8jG2ghc3y2grZVnI\\\\\\\/ltE0uFK5rXM0KKpm\\\\\\\/VXIeW+iEU2dbW44Y9pypCDrfTWkLIF7UzfyYq71uSQ9FjxkCkOViEtM0I7bCnTN3mV3bcU1ge8oTzcLGgOSQIUj3CY89xDa9jRSKeTx67nHUl7oxaIfIjo+aZYHB4pKHox2xEop1zcB7Cy2hFZJ\\\\\\\/Xgi9L8OHWPe3BvgnQqS1uAKzq0yE5LPIP8XJQmUpWjlZ0UaRfmJqgJYnH5hdHn1YjtMOtnMUBVGe465KFQD29FYEq2lwySF9WiNfvFZNODyzmQxKGw1T8zemjcBZmlnr85tb2xu+ymOxlOnBXk2XveTrX42A+bMXboR39XoFTRqACmiPnVVmGeNf89TjH4JKJL6At5eo0KM1LAgD+UKh08pXUc1vwzh8nc2vNTU\\\\\\\/ikHJRFG6aUl8zCYN+PCyRwTtML2JMnt5C4azpPcmLakW38fO7VWnIFkItrVAhzEg7ZSpYNEwTrK8nXzJwabSvMW6uZOHnhExSAunx\\\\\\\/lBBQnsG\\\\\\\/Bu6Mg+byByWxniZJ0BnmGyjtOM4O\\\\\\\/MeVd2OfEAFuoi0MM34dJ+nloG0hPiRxzT6UPxObo6LdLWsh4sTMSQdLQRmTcw7UjIvJPgXQR90Wka4BwPlNZSaK0Mdk+PMNTJQEw6QsjGRdrkuD9n4TLuNeF9Dlqof1XtFfy74cCdTAdShXVl5cb\\\\\\\/XTN39ahgnNM8QFBcIpJE1eCZWX\\\\\\\/kXR7mOAycm2HM8VFfoYJtgX8FMQJwutCrZLsa4WJZf8M2nTAw9kg9XqcrzAeP1ZsoW17DLXXvygQofE+S8lgXZ9AwL2yJ06O5YyujEdpxOwrEolRzUHN9KayKq432nFgwmdUUxKK1FNaIY\\\\\\\/GFMggG1Y6VGXmE8MmiJJoSyJ57c8Iv2UIwh3x8AhrSA9v+A9xM3W1oznnY0WKy\\\\\\\/yJNVgIjf2VTbYW\\\\\\\/9ZMv04tTYvV+\\\\\\\/BDrT3gUF9vbtOAQInDIPsH8bB8IvBAarI\\\\\\\/yNag+MQv36YnPv0bVXvo36H94VStBTk\\\\\\\/ZQde4\\\\\\\/lA1EW5GbbEOj63gmYldoG7eZMzctohdVmTfP0dshxaaiN0ULbbWLkGwgjwKb5K1YYiZZEmnPbImCiNPJ2b7VWQIirNDwDq4wYsmjhLXZWeMHE6cHDGYGk\\\\\\\/USJSSjy+5KnRc5HDCIOSPPtFPwwCXOuwJNL\\\\\\\/GhwkH0XA120JgvApeucsmCs8H8RWMj9kTZZCfuRU3IMVali2a8cKXyhU2eJrzwEWo8PoWon\\\\\\\/IC5ksdKloIWGwkhZe2H8CpDsLiDKWJCDSuAHyiZ164M4fQ10BkBUdtERHkeYiLOctBWlBgzmYbJxNO1jQhKypNJsZTV60SvWMl0+kyDfCAFLN31WT7wS4e35mbMrHUO7hXR99MpPMJ8uHKKQXdF74OEc7o5+1SNEQDaKWxKigC81C+pARlWxmXn2wm3BnD0ECpA7aT\\\\\\\/K+s\\\\\\\/tz3AUo89WnVEHqSwe3PhxMJ3Aaft\\\\\\\/lCUNSDYQAT2ZWQgIFhOpcKd2L7W32kW7xHzXXRdhELaCe5LguTgk8\\\\\\\/9io0lO+hnd3iKE92iOoTD5Rko10a+BNIW1P5PwvHmoqmsgu4FikB65CXfBSmujFlTD04OOAhX0719VwUqBpD5ruM0PhNSB2AXcl+aFBRZgKPfvVrmggbV2Pt75Dp1E8ioy3ah4HqsEzojCf43BsasG6UG3eEiRecgy+xtGwOdtTpzcn0mtejdR\\\\\\\/pl872p+kAgFw1hbB6vGzrHOIHkwxD3BbjtlBWbDt61rKH6ETBfY1fj23mK5K1v5+t8LlV0NCdxoE0lF6wTRZnqFxkZNEmowViBh7+MFL5nNvmHvGT1tf\\\\\\\/7K5rEeebZ6hlln6mKbyIXuHRYWuaR81rGpIHPfpoYReakA3X+Hr0fJ2FyNstWCM+vJYBn2SRIaiG3C\\\\\\\/0DjVLCZQ+w8duVr1bIcaOLKYp9TkvGmPI18\\\\\\\/q2rlRzQQ\\\\\\\/atjZHguKhHan\\\\\\\/BwPxjegqyPGYz6x7ZQwCxVAZKmbYy3HFUK9BXS9bhI95a\\\\\\\/tH3cLHxTJeoYjg6ttpYV5q0ujGt4yXbtJ8PXvXSNKMMtTUs+hXfVznxQJvLKgungvRgX8EKdAeMV2aL\\\\\\\/i1Ov2FZaei+cUlCpD6WgLf2UXQc7BYfPqiU3ECAHkABVIbeEolmyJ7yXKXpqecWFG9aDQa\\\\\\\/3Vy33xryqrx56xCuneS9iuiGw5fowCoWStpW1B5nl76cliQNlZ38R4iWgFs4hrWny35txOIdBmd9KYXV1OuPFV+M2zpTNdjF5qTKXEL4ZTnrUE6YB5Wmwajn5pV2hMpfDwf7rx3m8YgVTQFT\\\\\\\/w0Gag+nKQVqQWjtZzHqPRB5UIyQt24iyQrAZSxvu92KXr29clffWzP7XgQFfhvfXkRbW18C89WkkDF+7Rs7qyB2V0mb4dqH57unRNuYqUmfQ5X2q5NUcg7dAFzWErYxpZ0JnKCGsuR7W8F8CwRGhgJBIKnUIeV0jARGTbutFPafZEO59IaFYiFdVimn5kKhAD2BfU+4AZN1MWTWUuJOStRhGt47btMWn4HrVofsGNqq6TMBJLyT9aQ1o8mlzMPIhd6rU6yOTx0gq\\\\\\\/LQrrZ\\\\\\\/rROWHh6+t3o24JBtWWwucaqgirjrSn3I4XNbynEZMOHMmz8PRMLXSDLUXzcoh1yW6w9nLsU6Uzoha8VGanRE1Cv+pFinyEA1bGypnMLRBbRo\\\\\\\/rZMw8IZE2AT90he7gLuW7oQq8WiZ2XJahllDA2owPv7\\\\\\\/3dBulyGOlj+lLYj+BQmr\\\\\\\/rdqTvSpQ+7xRIe7SfPqW+bbtSEW\\\\\\\/0fTGfro5+WRynq7+1\\\\\\\/+DNfRCh2hZvaX3wJDX937a1iwPdNy7XlEP2N7t4C8e4csUOznrXCkk6HC+0uFD5E7oV49HSOyNwRWvitEXeub8LQmDkwuY\\\\\\\/1SWVE1sl5gLGYFtdSvRW3iPklS\\\\\\\/Z+Yg8rUyBHHoBxvPk+5XU9n3CwwqiMLwswxi4K8mYYFARfEqqT5PsNtgkDImAmnY6wJPidKbcJaKF8mNEezsmP8ij9GAZp+GooyFTndrNka8l5yuG8CfK6NPU8KRONUoCDOkiDKzH3y7yvUW08UHd4uN+Qv+m12WQ0kwjN4rTgyKPIndCdbMVPc\\\\\\\/cQTsJam57isko4zb6X3WAil6rQXb3yjB1hUPA3wI5l1mhUhxQ0AAm89yr3TDQ+bjwYXbwwDzdA5SLhdt88HUAdIxkzVIQAIQ5u99i+8Z2ZvDQxxttsh4SWINJD4jJhcsAOwrqJPyKAERqQIPv0YIaQRy4R9M8gd8VSueqTufP5f72D0yPkMdBpsG91daWInQhvLm6TkxgwiA8qsgCW6K3OsuonsdoqSchzTEdUJWGBqUpRUqJ\\\\\\\/g2RMKLPTGN8VRLX7o81RUtVfu4pEFxEXWdxHlN1oS5hHLE\\\\\\\/l\\\\\\\/Ba\\\\\\\/R+qTZGgi8Y9dxWU0fC\\\\\\\/u4IsHA2empZsgwwt4cVLUHw\\\\\\\/muvOsniMRYHv93C7q2SKHm\\\\\\\/57n\\\\\\\/LJNksJJzPMt8\\\\\\\/P2KhyYGLkJlHCcfiTX7rjmgSqm2HR12UaO5f28Udky7sT8DDKfZQx1k6iupTzRHWFNYdd6eZ0q4yPtfKpNrk9QxVfjecGcO+xOWGpDKQrHEj9I085CptTEtu6RYTc4sWb+zEn\\\\\\\/3jLzKBmGjVpAMkR8Li\\\\\\\/IM\\\\\\\/mWaP+oNvL6AviBTMZ20frJ4\\\\\\\/owDtZQlYWZR5d0MhaD8FuoaVDhI7F5xhjmDH3WVuPiUslkvka28m25xbLPPwqvK9Slhxd7gs1cpttlkYk04cVI7YdEf+Q2K5BJdrVZTXnlj8ezM5NZwy01QktINqyVPCaBNkR6Y\\\\\\\/eFEyTAE0hO7trNRDna3tq1W4SKBNSxpqUn+Znq3uWVEXbCY9PhklE8F8VnaGYncdAeii9HFJj9GnQOQoHvlcjrYEcM9Cu5xYDCV+p7kJ1DnVTRHUYJNAvJ753bxW2TnOUdaoQ67Z+vgSi1exhRWDhNOGGSpYAJZ3cGjWcjXD9xwRii+pEm04IRhzFJ5A6JCSkSRAnMI20tCqU9qKizFSFX0nTmA\\\\\\\/F54Hjsu0o7aZDhXuYgzj+umjcrNJWggTIOgCPk2g9lZRB7o8hPOs+8pvceeuI7O0y3DdbD0\\\\\\\/FgcnCOlN9Jz8kfMPU8UunqwaQq4qPwqEa\\\\\\\/b78h8S5tabZXQdk7PS1WhmAzHrtdb6IEedS4hWem7BQoK6iEXZbz6vE3xhEei73XbdVyLn0eTaQTFB3XD\\\\\\\/ra7QZGT8HQVwHzLli8Ki16O9hrKE9\\\\\\\/Jygda5UjQuJWhrE2W1xBvUDxfDoSdSq4HmpDEiDfhV8VjPHFUDolaE1t6MX2SEw2+X+kKLhyjve7kGHp1XR999g8kU6TjlqZwjvmbUjlJWfFKCz7Sp4OZEUf6oyrJwNW1ASge+GtisyXgbw4VweUAXxJ9yD0WIOyZwWcSfcmDHaQ+wPA63iBvuRWWKS0\\\\\\\/gu6AniDTSBNo7q0eiOZAv\\\\\\\/ezc4P80el6MwRB7Eazw7uub4Q2X4KpsT\\\\\\\/ozlEWpIK57DQQy7F4pN+Zpd56+3KtxrpJD\\\\\\\/9zqO4Uv86\\\\\\\/dUr72LUlQdDiO0n0sfKUs7tIdqyxfRaEICjuuUrtPNisJ07HEDQipfEVTcYwfdeze8+Im0pdo+00EpEcK5wZi3LazMD9KMG5Wg8LmQS2tGLP0pLwfHmxBUAG6BtiXWTL3GmEG00XkAR8btpzj30wg7tb11eDUhJKgJRbw4\\\\\\\/6ZX5JhuKCIsXCSkQYxE5QleP5Mgg5\\\\\\\/K\\\\\\\/8bwEs\\\\\\\/esxaKbHWhg9PcZcTfR4bGGItYwJGYdU5maeLKD6YR3cy8pGrrbnvdGm32gw3OlifciismkrImuSRk6BITuhFsQt+aB77+6zpOXBXFNeZd1Qh4m5Qov9wO+ZWQ\\\\\\\/W7F7R\\\\\\\/eQQPbqZ8X7ksQuB7b+VurpnJz5kkZSKI\\\\\\\/mZR4Oz7aJYRHZU+zJGzJEu5zBKPpk1t1gs+yHEuhsWhWq9ryZYxjURqHvRlvFeezaAMIbe6tg2ytkZGhGU+Cj2PUFDS1DgKcTFSFJsCUFmnhbrRsISsHN3lGPdoz9lCm0LnNigbqE11BZ1\\\\\\\/ChMp5iWtW7258zmaaZAqdak81KyFHJlBa6T+9sgw7rH3ygOLrQDNJtM+lQz9wiFOWUVoWXgVz\\\\\\\/31spr3OMI3mg2KniFFauCl5NDu5p7zS6IiJMU0HtVVRoqU2vmT1jjTGVTAoTmfwA0PXc4E2DofZAZ+pRhW7JL8L47o+fn3E81zIYM8cIlmvq3HEZXmk1MKyqWy6CNOTTP2eaGmlnVkL37gnI0ununt5vGdjaAh8to02Slqr6qWsnMPCTUiw6TvrUrnGI+d3+gALuhQQ6vSJJ7XIW+NZKlTxCwuN\\\\\\\/JV55ZnDHcAx4XyKshvodVg8thHCYFQrK6KytcV+zYRmC\\\\\\\/uzbPzNbuBUCeRwcwm7nWPcklDdXqxyJlrJFP1c3VVgwf6NyAXQwgjHNN+tHfsFqMKl83cPm3g6doM\\\\\\\/z4XyoLCpxO\\\\\\\/y\\\\\\\/b+j54N7LIX4ZFf7G2LWLRQkOJvJUc5m244SlrT5+Ly35MpWAdNlTR4U1mNDwMC0z3mfNpCFV4cntyjLgk+rYRBrblRPNkAFaYb\\\\\\\/\\\\\\\/L4iCsxVymWY9kKkv9klxZbI2JOlbR1iqVtx9PPcTZ8VUwFQ\\\\\\\/nelR0qDl\\\\\\\/ArcDWo1r+20nBk71j99MpEjM5XzABA+3soB\\\\\\\/uw+zz9CQd6jRu4SS49wBUMQAZJUi5H8Hzm285UBnzu+bpUGzPBruSo213R5cr0qguoUvAqT7KBjR5HFkRwk8h2gkiGE10kZNoDmEp3A02Q6OHcrgeUR4xW8vIsg2UFRlaD1maMt7p+bT7XYoKcJ7idiR51zwP\\\\\\\/kqx9yAiCe8ph5QXyG3ooN\\\\\\\/CyIQQBhIpxW5tUqQ3I3sN9MSFHiQoK3e6fy3cY6xOkvtqVfhp1cTGoTPMpFMkPrqZXLEWbEeCFhZAQUxbjs5l7tY4RbXiuUDPVZwZIm4uRhWihlxKmANJExAPMeKEgqRBUgHevXIhdTMkiNGVcEDgSrYt6H0PTcGRNk1ZjzeUMUHE1p70cjwulQmvRTfnY3DXMECmDJZMgg5LHU87PK\\\\\\\/\\\\\\\/KYS+qGHrTgNZ+PDnDLsudl50\\\\\\\/HyRm1KhR076F\\\\\\\/mbnhaRzifiau5Vm4LhdDu3ueSJE7Gwo0+S3jOQ\\\\\\\/yaYODDrCJc0Mub51q\\\\\\\/XE8ezi2RlNQI2fvBJmI\\\\\\\/EqC4rXSm5aVURobNvdjUmnAtS1iSJDHoD43LQ\\\\\\\/SGAt70WZsogyQCqGCDBc7ArRfO6y5w8KCbjd82Gz2ep9\\\\\\\/ATqQhr9CPvl32bh\\\\\\\/JmB2fYytKyL91MmK+6jb6S3DHeOOVxujnJbKkRriM\\\\\\\/1oysOG0esBfQmqOUWjVf3QpuSp9AAg5V19pSzYXicfQmucoFDJSU02WY5X1EQXRSY9QpQWk2yr0zqMlMxMvUs6gkberULHUr9vIaCH7y3rtq4oxHr58\\\\\\\/hhYqc+WA+vML7eyL+6uk1QBLEprB1UsgdvYyRELfQUIbUzrKeoU3D5qg2a84NOn5FhzC4XkX3R1kzyrXJK8QsS0wjgO2TFoFCSlWZ3hR41TpNGRGtphUGJpZlvaK6+Nop0LDUfdxy4iqE8QVxjF8enGSX9+a\\\\\\\/Y079sM7u1KWCPV31WO3otkgBMjq\\\\\\\/kZxR0XljvKiN1pJ\\\\\\\/jgtFghCQq52QeiIz2LGN1ip77e7j72koxVYbi\\\\\\\/tIkBilteV3O1vQlCgV1BkOGohbF\\\\\\\/zknde5FcSPw1HYbNLFBfAAQh2V8GzVy24wmrO8cNXMk4zEEA6ZIQ9Pw02Mfx45Xsv1tktdkTkDWwUe2r2dhNqB1kdMFPvtxXOSNl1RFZMgqDmf7PjELtHYE0MAqmn1fJcnEHT6UXM3FxG3rYlae+f3YZHPa0xsMUclalbK3Jepql+90EHX0A6Us7UNWYf10vM+c0J4AKE5r2FaCwHGZ+VR4dmzMAwLNQjTbY0zpXGM16697tm7x0\\\\\\\/WxldnzGOXDSLXZ8lvyhSzSp6jT5TMCg2JRkJqgxSbewvBwJCl+V0cICe9gG35JE8+1ZtJYF2zAF6zfpfOmC6adzl3P0I9s1KqeYzfoRL3G6X1mJntnlSwPnhLJxyFymGbBCtd4JL7mkD9WeGBmB+yRIHoCiamrz9xEExBTf4\\\\\\\/eV9sT\\\\\\\/u8\\\\\\\/1swfFg94xCrqmiP+9Ikl7b3zik0yYKei3G6LWUbkKfwg781aaABMMZi7KPP+GNZyj8lweIlAi9RcY4C5etlaChYNdcPLNq\\\\\\\/vWtLTAKCLOgtxb4vSxU8NJxbyNgMdwh2CN4D6jwN3L8g8DA2BqdvuN96rw71iTiIpzpxjWM+kvXT+Bohkdz3NalDOU5SIsL7TaJmaOSw2CAdo99u7QAx+TlcTYZW8EUCGsCJ+euNUbIAPiXDy7jVy++1veocbtWABm2Dxatw2nFrhMQq\\\\\\\/iCZSiXOQ15OCXkB\\\\\\\/t0NdpUgLZHkPtSzsOF\\\\\\\/DRCs6yCTZJ7Xa\\\\\\\/YtVK0GD1CcZgnt9FJmWrtcsHQUrgzn0I77gTzKfv8JoXn5fEgxS4WApYZL77QiWKodVG\\\\\\\/gWcv48fZgnxriEZlOIJM7uAY7sQ8gGQImnfhp9+VMxfDFLyTLjd0cn5vDGOQ2b77BJl8fTYjtKF\\\\\\\/nyC0FsV\\\\\\\/67M1Ek73BDMeplOj5jt2hNwp7QdnKZR\\\\\\\/TV7l+7FYeM0nJEmLeYimEAxc\\\\\\\/35qNPb+nw2ZQsMG6NlTVC4ALS7fG3fhPN+gHupdqRRX1LSZqM\\\\\\\/SHBP90Kg2yIcQeFn5qasgY0RyB1Z6r7\\\\\\\/UBX5GtvdNDLD04SaydT1eq5VtwivzSfzAZnVTvGbA9kuAdCAz8LElN53bJ8c\\\\\\\/w0DAZ3Hx2YvREnopwOD53yHqZKMwl4WLAOCQsINOsoQy8Zd+bwYQtxBK9rNTjLjev5BbX2TOFgjaC4XgsfYfFgePO2C+0m6SASQjH16FO2y8q3VFaBPq1E3OPQHk8YNOjY+M7zh4tZlVOyOpx8Y8jBgJbso5StcuNHOHSvVysPNA1DBbrGAHe\\\\\\\/JzvINwCx0j6vT6GDdIFE1uDIQLnqVMNFTBCh5Ao3rlkJuphGaXrPumQ\\\\\\\/UcmMsOzsjp9T\\\\\\\/zbyHKT7Hi06lrN7K717tmF5bCtjGb1Oip+Q9CtZckH\\\\\\\/2X4WxUGgyLwMz106DsMTbYfwctvLvG3EyLggCKG4Lc8jGLu2WbWRqRE8fSSOnEuLjwvI5J0gXpWvrl5mwW4Dxr+TBJm2indUU486CkNXAJaYjkrpTwphlehrkoq3EKlNvTLRPlafsSf5EvVrGPMR64cbe1RaP983bzZ34OdCZvhj6lyfjXHAiL7HqfCbF6JwAtOilA3b6N5qwffFZ+rzQLH0BXRF9oCEIQ1AfHZ3nNtUy1KFsx5vWtvoNCV8L5SRSXXKo6MYDhk3g0Gp+PBGjVbhcYNkygk5DW1Cmfk7IHko9ztZUmhiNXHJARgxmTHN2VJOefpsif2cdTKBMO3Xjk4+m1Ph2N8pgIoE7fNhEylnK+5GUXT9hrd5zDNJsnEF9IoW7AvSIatil0q9uNXWRn7bzc1ccOTt3vaJvelqB7+rtM96Vk7DRSs0S49lAMFfA8nIqiMAdBn7VJ\\\\\\\/CCJdS9v5g3OKqWBfBWfV2fYCRRnsCI7Aik9SY6VcBOWwbQnCIwVUYvvF36zQu4jhwpIpUBAd1uKrACxbB9\\\\\\\/E7eQlCty+2QmltnMvoURKVHlpNnQT6qcUwVjoPiPwNutZUl+vQFIF5WgFB+07g9MyXfBYE3avNX\\\\\\\/AnzTTWgWFIt3\\\\\\\/kMj5QWwf0gYav7E50lgZ8uhaxK73CSeBI+6eDvf6Nkys8AiQVVnIkOAZ7twWLA9lCMIwVi5dyUEIraHAWe9MFhU0N3CCHzagf68xxi2Nr0OLhvcpFtTs6SJiI\\\\\\\/0mhgBO1bLk6GgCn59UD8PUzcurASMQBfiSrWnSTZKKWSK9uE62xiQOFfUxSmGuf2ZgxnQN0nDTmw7m+RrOmRdyDImLPjnp02tlYpemnjrgORGj2eUFu1Gar+C9Kid4paEfueYZ32tKSViafx3Hu7xmwDuHDQAVnQb1vxdyR4oCbT8B59t8xV2Xr+9yvuZ\\\\\\\/4KKycLusikPvipfKGuSnBU7LyGd8r8OToNKwmVrQAnRxsrfDDA7X6QbyWeFkIzKO2zd753K6dqwIXvUyOInQ97v0pleNR2WHmPupBxYOkQaOCOH7+0tc0fu5FuC4TMo3L6s5FusEAnN+Cr8zbqjE6rvCb+hDXI0nvrD6cZlSELjiUFFvnAPrsr1NUefWAmNb8imhneMBSbR0+NxEdMB0D\\\\\\\/1sAaV\\\\\\\/88LzcBiMMV9XCjmBoSan4c7+L3HuqOqE4BDYY2EJK22FwDu4coxKf+oOYNj2OKhontrzbT6BAZk8jKXjGpYUB99GlankC3He+Mzg+LOrw8lq2JjXJDkw5Pe695FbdslP\\\\\\\/AvSJzEudflLHAMqnr48xYkkJHvy5lV62cXP\\\\\\\/nofxtfpqriZz2s9+TKzLhk\\\\\\\/Zw6fYxxlaDGExUHkg99Nl6VrKLnLnIC4VF0SnhsT78qnqpwcyW0lZWkdzAP1bCv97\\\\\\\/gkrIpCTkGezSSYC\\\\\\\/Xk8hXjKeCVf\\\\\\\/JQflmo6aNOPhD43iTVE8oGuKmmpDLr0TnpvUI\\\\\\\/LEpNWC5Jxk9+AgTd7LrNTFvoGdolS7VnZAn0JWuq\\\\\\\/SyHq+aOyG4Hn2e\\\\\\\/U9V4PLokePyahU8dl48zY+Vp3E64UEJWN+rkctxreiIhUzUoU8tSGXe79y6hsVyoRIWZ3sm+kQHk4edv9Yvw3Vg\\\\\\\/pFwk33uXV29UIqLomKXF4eQc4wbaeHOcRM9\\\\\\\/1JZaAfF9mbIo2ecwWtUYlR+lbqRXssHn3WSekVs4qwBKBzHiZJ6Z1qKlAHo5XDkknC2\\\\\\\/T3+ic402oGOlEIOq1fYlAn6bGZr+yy0lHvtTvLFo5IW0gtscCWwPqu39Zq+oDBCPemDk7+yqvOZUtjWni0+MpUn2BcNqAJVTUd2SkyOUAyFfRdv1mY1F4ix8G9BO0kpyVsv0dpm+7IwRib090So+p8OryA6HuNoYxlanrrA5KIplJKViu3l80+1YHy110yDmMFKEmvcHjAwHS2P5GOsfGcdHPq3xcTwVnzLSG0HiuT8UGK6\\\\\\\/F7De4mRYtHnnp1XjrF6ccpOIGpPccTjBHa3ohEVrl2bhVeSfIVnqPYfTsiubKM2ezZKTAY0Oz1k3sAyWEbSZ40bQfDeCvafLhmBvFGSRaalDUspEo8V0nSEvw4D3JRhRxwZihWLkXcaIjzs\\\\\\\/V38qu49JffJ4FvJPPNDAt+dKse2Ob4+5JinrNSYrHzW\\\\\\\/pyzNgvdTaBV1ATq2YDZWvWwbDk9SONmISLCq162zkI0+3QrhB6M4\\\\\\\/gw8dWsuPHH1Cqq9wpGNsBEiTBMhzIHEzOvjxwOLlKqao4Fx+YjAE6HUC+thoLD61iW3Wp9AxMe\\\\\\\/fFLULhwQZ6TYBNh4d+5poEjYKE0SSs6ZWZuPzRbfQbq+9LYpaRebq5AAir3Gb6CXCM895sMkhbrWK5wSwHo\\\\\\\/sO6Rv819qgBX+CE4g3IcTceFKVeWjkT+EytCNLz45ZM9lSu50OS+3sgN3uWpzdAYXb5eb1DoWDTzWqp8+p6w1Coo+SNMzSnOeYcmH2\\\\\\\/3OP40\\\\\\\/NUUTu6tiOvm\\\\\\\/g0MRBn2FUfSNHT3us+gN3cBP\\\\\\\/97jfMyL2joUOtF9YMh+TJLAx4EgA+jKy3BNCd7YYLt1lXEQONb+AUusE9vqfGsg4RaCWtJkk+SyZ2Lp8K0n0zUB23LhuiGNWYXHEOH7Gajb3ApqR8rRzwrGatTS27E3HK3qeRMXHDPSrkVtOE+jMJol25uTBPKYMj4YvqqIB5ruzywnWwQj77seyapejgwQmykRnQrqlXSLshajdTLQC7dDJm7C6OsilY3AefrpHXeL+hetnfGcb+c0D67PSpAsW8PDhkforrxuMqphhgX2QiE\\\\\\\/J\\\\\\\/OWGJI\\\\\\\/KYKOdTBWw8AEJkG4wqfX4Ws6km0pHaPxcXc2dHLoMxhgYebj3DMrLhfuujRMGOhGamJSni1P6yYK7KupHNkRDKN1y30PEJunqvvKIb0uVZhFcD1gWCtynhUZEfDcJVOkOjx0vyooIZfrgs1ukW1x+uUxFiX8dHddemYksxNZIFmu9T5FAYB\\\\\\\/vEszDINvVl9KFYFsy0WbuE3QUfANP3\\\\\\\/HBEzbbUF9wQsb0gL3\\\\\\\/IvCCsxEFbSJT6A\\\\\\\/txWj29al1MmqDegD04qhfLnQhxX5alhZhcoNVl4wXq\\\\\\\/8I6B2O8zynd34juUNi5\\\\\\\/SSTzx+Cz8GtKc1CLiXqa+BDxfHQO6rx+ze1ZsDSBdYsGwRXKu+8QPFCaJdAxkYyWo7uHhmIGcaER3zqDi+h++dgMsBP1Cqc1lzSz3BT\\\\\\\/gkUUzOrZjlHZ5juVumB1Zmw\\\\\\\/KyuWr\\\\\\\/4CDmGf7nNzSrPSJ7nMJxttQJtvO150Y\\\\\\\/M+Jv+oi5strJ5QzOyhjjQt3oNooMiaJF1XXNFmkhtN7Jahn1XQ9iwCaCZtQZAP2s+oijnYMTqT05kzYReJ3af5xQgUEbBjLuvFTyZtNCK6Tg1M2GM7rSdwAyVxf8kwBR3auftN1Q0dLcM53MRuY1uw1OV5Mty0QE8F0CuuBJF1CP\\\\\\\/\\\\\\\/ji9a2luI6hEYXTqA\\\\\\\/d8PjVsFBNXR+2XoDzIFwlIehvkTKmxD1N0sZ5QfQO\\\\\\\/wbRvXoIZ81L7w8uOeAM49WvQtp8FQH\\\\\\\/iup1BYj\\\\\\\/pA3hyalObH21fdVMgp56tVtczrhGiZsGjYf2p59QFAhHH0a7+o+AIlfVVudU0QZmZMQJWqiES4wJXLedc++5iqnjlyUc0qm6RN7y5Rhgbtg58n6IUs07aL63e9HG3kLP7MHL6Gv3z8bXplG2XQd43OFABoXbNTS7h3XY5fEa\\\\\\\/IDNKyHttnRPNfiUvCtOq8FaWbGi\\\\\\\/Pjmj0jQxJAIj+DXG\\\\\\\/V8I5cfcV9HrxcJ83tnHaKlyHoRd03GxeTPr43IDfdLgAwF8Mc1bKnmapk3bY4SvbnzQ9iLf7\\\\\\\/Vqj0+f2MjLf5JJCIAGiHj9hRDm10VhJHiWBWMXi4mLrVBhN8CPtFQT9cujzHtl5\\\\\\\/Tqc+Z60sMIkCCQOpD5BegUTwpa5RslUs2rQ8ewIvo2bLQud2tv+onBmA+zOdsfq1BDTiirH6vvPAtt3+cRMWS\\\\\\\/cqbywMs7Hd3xqglgVBVzXP2ovlMk22D\\\\\\\/TzoqevsKiF1dqaKtJNjGqTu2JoE80xbkMqgs2DgjZZLCR+wVHKTlUgSdSXEJ4+BSzgLgWmmVHkIXH7+CQmG87Bv7NIK5srAFcGdEnwZwhGRrGnsvRwEXIE9BJfU\\\\\\\/gFEZS8AaE5TaQCe3RsiI1uKfG+pwVE5IXjfgAAZq1KxDOZOlHCwy9I6o7z5iAQHmXZPI5pGcjQ\\\\\\\/i+p7sw4l+hks+pvZepybOKg1ou8PmXVYexJWBXaGasPTOOjC3NduIG7L7ppLi0X\\\\\\\/eIrI2ih0GWlgf+ln4gmC60O8L0NwwXqiX8XYivUhOK5NbTmsOH58aVYn7+kdQr4Yx9DKZYVF1SIe12iY\\\\\\\/siAiYS0TQ0vywCzU7zOW+wUi8touemhVA9OJTky2JBnGTy\\\\\\\/SZlZYJJkcYEdiwmmACnfsQEVzQOmDFB2xVrLzNMbPx5Z+mZnfrt2KFY7tGCdUtgD\\\\\\\/ZzJVH6I7AeuJw7BbNLz4DQteHSTe+1umqu6P11Tk3Cr4Ep6nAWoyAeWky4LhDdkuRkguH22dDdLc6xcrXvB4VMv70CAkdvu9EwcfySdZnbsTgMvBXoZYk7yC6Mp455Cs0oRoncRlgO5\\\\\\\/Kf2tUg+dnKFiKocLZha5VIN4f1OPRQ29AzCF6L3N9h2RX\\\\\\\/WZ27JhaGz4bE6CW3CV8XaXLe0ANpP31pMO9JxvifKRV0KJiwXzMDBQpY1KOhtsBr\\\\\\\/GztJYLIMqPPmCEwP0eFccrnF+yysEqcoFxz0hEYwHMjxl2prBt2TDccXhvPa38CPi8GsPZvmpNreXoyQriZ4F9YbVw7utpDXNUy0y4hgVJEE8bMGM85ptglV+H+uO9e4ExoM3ynp09L\\\\\\\/0+t7\\\\\\\/g2Tmj6nB5ccxQq\\\\\\\/Y2sNetfNlz1KFjXh7JsWzuQ+XnFjUCbO+e2GIJKWgKvBoWYxYVDp6Gk3nvqff95X\\\\\\\/aPntEtxXp83MiTHohRcSTtc3tiNN+QZToKcR4UOZn1rtNBKvwhbCXTbGGfsJv\\\\\\\/38hmCiEadulhkrRwmA1GXoMcZXaDYQvGrrfvp4qU6EPRImnr9hFcOQFYnY2lpSzVrLbz9Uee1OMXEFHGNSg8n0GzyoSaaA6Vt1eO4Jd8nLVMO\\\\\\\/IUt1C20wGMJfvt2HrOXKgg\\\\\\\/u55TXCvSsWLYSDKuX\\\\\\\/b+VHdhzaMUEt6sYAaWBZ4HAd9+5qG7VRC9YHwW2NrFd4W0WkwGJLNqTDqJxTRqkKpeNtdX3epL3FtcbHMvtQWDfxiEkFWplJjzooShSty4iYJVpvIjPR3pH\\\\\\\/8Eo9GcL7MYzNFSgg8tP219ukR6eKLdLZis4180xkCgIacjFsLswHHr6lMFZyRgfl8RPLdGBJBXpARpQ6nZ0Zac4bkmTM604vzZ7tVbZFI1EVAjUqYIX\\\\\\\/TXerYA7xB19iFkjCQWBo5kEeayJ2G6FyNUWA+oKhAqa7efYnGTs8ifZ+FwpLVYcAaKFYL6IMn39xJhQFVme8yZQDhfPfTkZF7zKT6UFtIiQ420WK9vXV8\\\\\\\/Dt0PO5isPlg7nsX\\\\\\\/rI7E2a0eChBu\\\\\\\/JNiJjAlEKuA\\\\\\\/kDAUrm5mOngbsC4k6N6sMv\\\\\\\/6roO1UMP0aUkTKy5ERqXL3lSX\\\\\\\/P8iHYkjVjG+EJxv0WPJS1tzoQ93BXbE3a9XW20g\\\\\\\/TwV9SmSQGnaEvRiy8LCIFq4pKMgoJy0ix1Sop90tOQndq0s8ZIuELEvCpeabgv5twQYKOmQHEe5eX4lt\\\\\\\/Qi8PEQ6BbMS+8JowbDLvjj4JEniv3JrD3\\\\\\\/xubwvhDXrdQnPyxKQVPnBlZnTQV3+v7+2FfatQ+Z4rOVFlUaHXiZ0k7HSbH2FI2wlfjLF4+NKdkCs8R\\\\\\\/ABHNknpCeq85Eom8WXGP9txG7eLRC1bcxNsprFIO1e7GD2Tbrm+wCghrHBkA+9x\\\\\\\/V4BSe1fbU\\\\\\\/r0pOcpfHlV0B5gUoAB826\\\\\\\/Fieo7gk5ALQ0mMbZ84Qu2dwpHQBEQBAjazooViqhSV61RvfYEeq+KK8ks2OC55ebddMatxbeiWhPLwEdS66xz\\\\\\\/S7m1olIQsjphM8VsAogMt6kOatvW54w+ZS5JPjr9yuxHkrv4bVlpGeJAK5cPKaZkBQUtuX\\\\\\\/RSdnB2UrdOan7EKG0ui2hyTmsjC3RAihbQKZ1aFMR8p2zkXj2WgPrAsD80U04baUvQZHc1rCubjVQAD1ijXbQd\\\\\\\/vssuJnB2brUDC+Y1mKQ9E9A5Ai\\\\\\\/fDXDEWR7z0lQM+pZK+zpPnF4WplKNBBjzGpf42VTdvzDvfdjwrVY4e83q+vLZjNebbzqOgkRah14asa3w+cUwFZNPr2yJWPVyA637wOpNdfD4Z3PBkiQXagnpSCD4ImNBvdGNjZ6rkMPWK982X19WhbeL\\\\\\\/L1YKvMOjjdspYuDTLr4mE14d8PAVwz+bEQAUrRa2QF03RBuP9fLtUFv1iZvLu7nessTfl54Ee4A64pwG9m9qxCSS+Uw2nAPPxV6+trth0WeZdoJxNgbQ+9VZmrHU2QBqjsV6g0Yp92sbwnn1BZlUniqdfBfqo0A+M3BMUjBoAJJXa8QyvXei9sL+Xg4SweWRdC3FadR2Cixmb2wcadEZvgsX5bkNo\\\\\\\/dIN7Yp2+EF0jDVqSOdAZXN\\\\\\\/djJ7PfZ85PiB7XP9i1LdBuxIf8isxn02NY5iVgEaO6mUWall2cFbR0bL7L4+JKPnY5vJEfJcI2RwnmrGCOpS3GpworiOVEWesO+2Pvckxd4\\\\\\\/O\\\\\\\/m7u90MU\\\\\\\/P96xMedWLRRJkXxGY+VFmHGNpJwDzTjcCZSmmN1mtO\\\\\\\/uKwqZougrK03a\\\\\\\/0yR7OOOdaOHJjK9Qvze1bMlFU3PRLs7McQrkSzGNvmYPBj54NJs\\\\\\\/XhXcV3PMq1Gj6nFzY4nQVrf5OuTMZM8Uz7VXS9QF+zh2Mt\\\\\\\/f0v\\\\\\\/v8u17sDehh20UrqnQjvWBSLTfrO160S20pkgcLJPO8+kMjgqpPVAclmLLh8j7vKoUBA+AE7OKjjc7rkGDB0FdjVTr7mCTGo2OE\\\\\\\/l57ZdXy9SKQL7kGh9kDEHx84KvN\\\\\\\/TXhxUKn3tzosSiC+2yGzsAGKxrTFOo0ndQ+dyXKnalZbP3VHrjaEOACrOyJTXnggpAR5lDjFvbygmfDx\\\\\\\/ac9z+uAm\\\\\\\/uP1Q1Xxe3Tk7W7z13zT3oXSEXMLzBNDPRo6vYza4XMutWuurI6rE2aBmWnC3x1xDIJ98KtBZ72WK3ubPjS6IPQmxheT2KtzJRituPrEjLuS6a2vm3Q42TVLHg\\\\\\\/2jg5KQQSUzTnmCa9KDbgtLzXDA03U+twcVNJJbQtipFjRjOLu37Xn0fjBnGEkxMpgoeLBW4pteyHGfh4ExKSOIMtQwpiHAdA19U5ZRApVlYDQx\\\\\\\/gBtmpUULC89G58fzIKjMq0pxhE2IsAaPqK4ZXopwKSgFl0McYGrWuFW5E9WKr7dTYxLKym02tnfHAJgTz8eBnsF7F06Rhh9erIhiowzLjGRTdHd\\\\\\\/xBFmBYWcw3Ui1t3co\\\\\\\/DpCrA7w9tDFPdQRgMLUNUis1dMnhtbaH6CFh8dUWAp3dVqGC8yA7JHLJEHRsXSit7cSE2QvJdtoqTqf4srh8waDC5wanWfniTb2I5W2S8cz6oT0jE4T2m6QViVVVG08rOuhCb3a3bXK0b76A6Q47NmFc\\\\\\\/bq0ui5XppLcYKJqU2eZ7XGlHld+k7AjcrpNSMm8qBu+0T\\\\\\\/HEqBeTNIAEWFNGlaxbWsV6zF2b6BwXpkQpUPmmfqi4CY8FBJDa6jTXyrO5zt5fzwFgUmciKK+JUXaPU+wE2\\\\\\\/doQLYkXr8cyFBkWda3U8w3\\\\\\\/HkQJuk81YU2t54tTIDv\\\\\\\/NDLRNsHIhs99uMf7GQgXjGNMR4wubnX43s4R1KDaakxVhIdIEWWdEAFFX8FWJtGUPlXAxZ0MjfOIgouYyz4x8QsyDQ0jxRuz7ykYTBAlV3XlkfaGchrqGcJoLq9wIuTnzGf+yN6zlZRTA9G1pUyQ4SS3ciYAc1Nd5hZ96ph8xOmC6Z7xvSJw4Ln7GK\\\\\\\/BwNb9v9W9rrT0ZWMdtOr09iCoPSnLkEyja0e9oReaAkQbPUJczGhORYQzeA4YklhO2YBR86iLAzZSn61wK+UmNYlnkkdgZZkcY+YTgXw\\\\\\\/DA56LGx5OpE7t6UkKBPOvL4Lg+s3DsWciKGVBbkWkvgbyDR6dKaIhrPwR+PeRbZNxUOQV286ShKVdbMNghjLGeUSB9Pa\\\\\\\/RkwSRfNgKNyO1icDWCbNaPqYFRJXQ5lUeqYDlc7jgz\\\\\\\/yQWxQ8RHFvZ3208wYPnFp3twjLjatJ4PQBAT\\\u0022,\\\u0022iv\\\u0022:\\\u0022jKwTGOyWe0qMp+5WJM5HMg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227680cc29f02e0a2338f16201e5ca385dcd1aa49c91f0bf23779ea0752b349a60\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ynGg09TrTSKmBIujTiO\\\\\\\/C8Tgaopgl2qmU+XzM+cuy9g2XqofXmc7upeyNdDgd7V6B4h1405YYIVzU1zCiPyaCuPEa0G6oYI50nq2BtpLfLzjWuZmfSoE3+E4FKtR1KY9vfhhvnRI\\\\\\\/zOwofDMlVZQWMn\\\\\\\/dyahB1CrrUejseuVrajvttbcyyTeDPhMnY5dtkK0gUuKOKLVWjcsxsneADRHHM5LX4wXjgUHbwDpF+dvIPZldq7orwDyz1fcUPu9L0\\\\\\\/3yjVrp60IBQkoRelH6Ku95GUt+P\\\\\\\/\\\\\\\/htvUvHEDph2TK04VXVa52LBgSsoe62n+yJioE5sJCns5FERtEjK5U3+AjgQz9dJFlAop621eI1MURyrGJm98S0TmgLrm01EPyYxC2oCT2UphFTrnXiHmPdLHy2WDqoFey5b9cvQprq2AFb1V5Pk2s\\\\\\\/PhkOG3gZCKXzL6o64zrcuHStydAKK2Z05po4iK178mmiPP6I811czo34+p8\\\\\\\/Mux325mIFcTnUodLzZ6M4Pp+RYTJuLiw8CqI0L6c4lqX1prTLatXnU1ib\\\\\\\/OWKbxR+fBIp2T6Y1nMjFg9qyXHflkDjkBOtFbhsy0TocxtIzDx9GRWJQ7O67OuMWDlq\\\\\\\/Onz55fMkMc\\\\\\\/4MXetXuv9sDWFYAvvcS0rDRqYEppNy+RYDgwOEo0aeN97WlijNCkO91zsFtDjtvAD50fYJIDh\\\\\\\/sZOMN\\\\\\\/w169UwmYuAUsmyWX8ndUsGUNYrsW821y7OanZF8NnCTbul+zuuP2qLx3i2DlikyyQ3kqpXAmpMZsQhlGu7VJuCjoSO8Lg8Q6YqqXbcWWtV23WvnT4iqqej1nnbpqgWStbMVcKe4LBwgCXSvRViB5bnfHPqmw2DVhSq31XhK5epPr5ucUtjLTDvy42YXQ\\\\\\\/9\\\\\\\/Dden5Onw7mQBughfyRJsYEvVI4xkoNcp2go73\\\\\\\/muLF7On0JNnl4LU7ekma4ivVmq8T09KV82LsN77kc2Djw7gboyxdxeS6emJGg1aonNz1rlafBdWzCmqMllBXNoc3NKrMeRLkraFZyhHXmfH02BZz9KTNkpNHt57tVlgSBt2SIVDy7KlhPOvQVcOhTS0EJSa7z84k5Q8K1le7xyIWjWXY\\\\\\\/SCLxYSYPlNCOJSKEtjtOIq5Q7fBfEFQRzLjN6dR+XiqOOilc4RffWi1VIGKEOzyLwrIDxob2LiQ7J4+jKTKwsutBO58kD2ECnquTZUrinRivQazZa5dJTbfhtr5JwsL6MdRpb6gf+VxXVek690n1kaqPCrLEloUIpjbHzWIBkmdWGFJ37NkFA9J8sSnTyZHvib\\\\\\\/fMV0KY+ORzx7CM9y5BmeHQKCqVVezfXfWJ3VxV+g6\\\\\\\/km3eXRo\\\\\\\/SwM4Y9QGQB3QKQg8wh9ufK+MuWh56Hek9GCNCrEiGGhtnrh72hQvbeLXYPi\\\\\\\/fsHEf6vSmRAVoK4UBgdwLz3qrdLn9TtrwJr5KgEek0jjE26U7gETDfv2vAEieTZuSp7Y7rT7mp72KU4s\\\\\\\/J7O\\\\\\\/l2fFJ5CFpA6RA7935JUDwOjovMIAcvOEW8RrfRtCldlPdfOLMEYZXF9vOEHTk\\\\\\\/\\\\\\\/inIVfD+DQnvxtxBCVdBFLqm+0Vi\\\\\\\/dAa0c\\\\\\\/gvxpg44FpadLY+pqnO7Xab3W76yH8DTZX+DwuTNjcfglgj690xxgVs9Ug9E6Uts7HThFZvzoS2Af1ufJ5IrVQxe0KV9UODAciK8XZ8zM2fKo8MvKrOOUHoMvxp9kMTDI01jMQpuOu8B\\\\\\\/d8ddQKExB2cuUhFD1Vlc7OpiIiQ6b0cp4nK23rlpCAMGZZyLhG5hq5StQRTr4CWsIDxiPXU+Bu2laqnZtkHrWHz1bd59ogKokef2L+1yEFuLThaoEc9nea3u3APSi\\\\\\\/EGILr3JqbllunJf5GKoVMIKBSelosMNnuME28H5hfE2AuncKSBfvpzzezd0RylVN4sRI8s6t5+9ScR7EfstQC82OL9xIimQcYxZU4FC2Z4YlolqKdQ0yRBN4wYOhfA9kcAGw7dnc9ZWH4JYD1RYieV19wKCic=\\\u0022,\\\u0022iv\\\u0022:\\\u0022hSqhMCbrV5GyH5OBnPOIrQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b40f275fbe7068a98561dee58dbfaf7311830ee51d24dd91df5a25c3fda2b7dd\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459154 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/1/11" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rJABirOmv_YemyaFGxVypy9Jh2xSakJMKrR83F22wuo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/1/11" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7yMSmPrMVlk9gs2gpefpB9Om95Kt59eIvKhQMD4eUMI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/1/3" title="An SNP-Based Linkage Map for Zebrafish Reveals Sex Determination Loci" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/1/27" title="Variability of Gene Expression After Polyhaploidization in Wheat (Triticum aestivum L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459154" id="node459154" data-pisa="ggg;1/1/11" data-pisa-master="ggg;g3.111.000273" data-apath="/ggg/1/1/11.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Awesome Power of Yeast Evolutionary Genetics: New Genome Sequences and Strain Resources for the <em>Saccharomyces sensu stricto</em> Genus</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Devin R. Scannell</span>, <span class="highwire-citation-author" data-delta="1">Oliver A. Zill</span>, <span class="highwire-citation-author" data-delta="2">Antonis Rokas</span>, <span class="highwire-citation-author" data-delta="3">Celia Payen</span>, <span class="highwire-citation-author" data-delta="4">Maitreya J. Dunham</span>, <span class="highwire-citation-author" data-delta="5">Michael B. Eisen</span>, <span class="highwire-citation-author" data-delta="6">Jasper Rine</span>, <span class="highwire-citation-author" data-delta="7">Mark Johnston</span> and <span class="highwire-citation-author" data-delta="8">Chris Todd Hittinger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">11-25;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000273 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459154" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Devin R. Scannell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cell Biology and California Institute for Quantitative Biosciences, UC Berkeley, Berkeley, California 94720-3220</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Devin%2BR.%2BScannell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scannell%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADevin%2BR.%2BScannell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Oliver A. Zill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cell Biology and California Institute for Quantitative Biosciences, UC Berkeley, Berkeley, California 94720-3220</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oliver%2BA.%2BZill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zill%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOliver%2BA.%2BZill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zill@berkeley.edu" class="" data-icon-position="" data-hide-link-title="0">zill@berkeley.edu</a>
<a href="mailto:cthittinger@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">cthittinger@wisc.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Antonis Rokas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonis%2BRokas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rokas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntonis%2BRokas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Celia Payen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Celia%2BPayen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Payen%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACelia%2BPayen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Maitreya J. Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maitreya%2BJ.%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaitreya%2BJ.%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michael B. Eisen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cell Biology and California Institute for Quantitative Biosciences, UC Berkeley, Berkeley, California 94720-3220</div><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Howard Hughes Medical Institute, University of California, Berkeley, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BB.%2BEisen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eisen%20MB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BB.%2BEisen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jasper Rine </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cell Biology and California Institute for Quantitative Biosciences, UC Berkeley, Berkeley, California 94720-3220</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jasper%2BRine%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rine%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJasper%2BRine%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Mark Johnston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, Colorado 80045-2530</div><div class='author-affiliation'><span class='nlm-sup'>â â </span>Center for Genome Sciences, Department of Genetics, Washington University in St. Louis School of Medicine, St. Louis, Missouri 63108-2212</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BJohnston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnston%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BJohnston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Chris Todd Hittinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, Colorado 80045-2530</div><div class='author-affiliation'><span class='nlm-sup'>â â </span>Center for Genome Sciences, Department of Genetics, Washington University in St. Louis School of Medicine, St. Louis, Missouri 63108-2212</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BTodd%2BHittinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hittinger%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChris%2BTodd%2BHittinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zill@berkeley.edu" class="" data-icon-position="" data-hide-link-title="0">zill@berkeley.edu</a>
<a href="mailto:cthittinger@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">cthittinger@wisc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/1/11" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459154" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459154/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/1/11.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459154" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459154/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/1/11.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459154" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459154/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/1/11.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459154" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459154/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/1/11.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459154" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459154/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">High-quality, well-annotated genome sequences and standardized laboratory strains fuel experimental and evolutionary research. We present improved genome sequences of three species of <em>Saccharomyces sensu stricto</em> yeasts: <em>S. bayanus</em> var. <em>uvarum</em> (CBS 7001), <em>S. kudriavzevii</em> (IFO 1802<sup>T</sup> and ZP 591), and <em>S. mikatae</em> (IFO 1815<sup>T</sup>), and describe their comparison to the genomes of <em>S. cerevisiae</em> and <em>S. paradoxus</em>. The new sequences, derived by assembling millions of short DNA sequence reads together with previously published Sanger shotgun reads, have vastly greater long-range continuity and far fewer gaps than the previously available genome sequences. New gene predictions defined a set of 5261 protein-coding orthologs across the five most commonly studied <em>Saccharomyces</em> yeasts, enabling a re-examination of the tempo and mode of yeast gene evolution and improved inferences of species-specific gains and losses. To facilitate experimental investigations, we generated genetically marked, stable haploid strains for all three of these <em>Saccharomyces</em> species. These nearly complete genome sequences and the collection of genetically marked strains provide a valuable toolset for comparative studies of gene function, metabolism, and evolution, and render <em>Saccharomyces sensu stricto</em> the most experimentally tractable model genus. These resources are freely available and accessible through <a href="http://www.SaccharomycesSensuStricto.org">www.SaccharomycesSensuStricto.org</a>.</p></div><span class="kwd-group-title">Keywords</span><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/saccharomyces-genome" class="hw-term hw-article-keyword hw-article-keyword-saccharomyces-genome" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Saccharomyces</em> genome</a></li><li class="kwd"><a href="/keyword/genome-assembly" class="hw-term hw-article-keyword hw-article-keyword-genome-assembly" rel="nofollow">genome assembly</a></li><li class="kwd"><a href="/keyword/evolutionary-genetics" class="hw-term hw-article-keyword hw-article-keyword-evolutionary-genetics" rel="nofollow">evolutionary genetics</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/sensu-stricto" class="hw-term hw-article-keyword hw-article-keyword-sensu-stricto" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">sensu stricto</em></a></li><li class="kwd"><a href="/keyword/genomics" class="hw-term hw-article-keyword hw-article-keyword-genomics" rel="nofollow">genomics</a></li><li class="kwd"><a href="/keyword/yeast-species" class="hw-term hw-article-keyword hw-article-keyword-yeast-species" rel="nofollow"> yeast species</a></li></ul><p id="p-4">Hemiascomycete yeasts (subphylum Saccharomycotina) have emerged as a preeminent phylogenetic clade for comparative genomics due to their small, streamlined genomes, a wealth of functional data, and genetic diversity spanning 500â1000 million years of evolution (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Dujon 2010</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Dujon <em>et al.</em> 2004</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Piskur and Langkjaer 2004</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Taylor and Berbee 2006</a>). Although low-to-medium-coverage genome sequences of many species in this group have been determined (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Scannell <em>et al.</em> 2007a</a>), relatively few are complete and well-annotated (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Dujon 2010</a>). Most studies have focused on large-scale evolutionary changes, such as the whole-genome duplication that occurred within the <em>Saccharomyces</em> complex of species (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Wolfe and Shields 1997</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dietrich <em>et al.</em> 2004</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Kellis <em>et al.</em> 2004</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Scannell <em>et al.</em> 2006</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Scannell <em>et al.</em> 2007b</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Wapinski <em>et al.</em> 2007b</a>). Broad comparative analyses have been critical to our understanding of how genomes evolve over long time scales, and for describing what makes fungi distinct from plants and animals. Determining the genetic bases for more recent and rapid evolutionary changes within and between species remains an area of active research across many phyla (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Atwell <em>et al.</em> 2010</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Peichel 2005</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Prud'homme <em>et al.</em> 2007</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Seidel <em>et al.</em> 2008</a>), for which unfinished genome sequences have proven inadequate.</p><p id="p-5">Comparative genomic analyses of entire genera greatly facilitate evolutionary research, but few genera have the resourcesâboth genetic and genomicârequired to support such work (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Clark <em>et al.</em> 2007</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Butler <em>et al.</em> 2009</a>). Smaller-scale comparative studies in yeast have already provided mechanistic insights into key evolutionary concepts, such as speciation (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chou <em>et al.</em> 2010</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Greig <em>et al.</em> 2002</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Lee <em>et al.</em> 2008</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Greig 2009</a>), life history variation (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Gerke <em>et al.</em> 2009</a>), <em>cis</em>-regulatory evolution (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Fay and Benavides 2005</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Fidalgo <em>et al.</em> 2006</a>), conditional-fitness tradeoffs (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Will <em>et al.</em> 2010</a>), and the long-term maintenance of complex genetic variation (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Hittinger <em>et al.</em> 2010</a>). Further, comparative analyses of species closely related to a classical model organism can reveal regulatory pathways not readily discoverable in a single âmodelâ species (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Zill and Rine 2008</a>), provided genetic tools exist in the ânonmodelâ species. Next-generation genomics technologies make the sequencing of entire genera labor- and cost-efficient, bridging the gulf between research on an established model organism and comparative research on its relatives. Endowing several con-generic species with the genetic prowess of their classical model relative would revolutionize the study of the genetic basis of evolution by allowing reciprocal experiments across a model genus.</p><p id="p-6">Although the genome sequences of several species within multiple eukaryotic genera have been determined (<em>e.g.</em>, <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Stein <em>et al.</em> 2003</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Clark <em>et al.</em> 2007</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Rokas <em>et al.</em> 2007</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Butler <em>et al.</em> 2009</a>), none of these are amenable to nucleotide-level targeted reciprocal genetic analyses between a classical model organism and multiple close relatives. The <em>Saccharomyces sensu stricto</em> genus, which includes the model organism <em>S. cerevisiae</em>, offers a unique opportunity. This clade includes at least five other natural speciesâ<em>S. paradoxus</em>, <em>S. mikatae</em>, <em>S. arboricolus</em>, <em>S. kudriavzevii</em>, and <em>S. bayanus</em>âand one hybrid speciesâ, <em>S. pastorianus</em> (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Naumov <em>et al.</em> 2000</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Wang and Bai 2008</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Nakao <em>et al.</em> 2009</a>). (We note that a recent study provides strong evidence that <em>S. bayanus</em> var. <em>bayanus</em> and <em>S. bayanus</em> var. <em>uvarum</em> are genetically and ecologically isolated sister species from two distinct lineages (Libkind, Hittinger, <em>et al.</em>, unpublished data). The genomics and genetics communities have used <em>S. bayanus</em> to refer to <em>S. bayanus</em> var. <em>uvarum</em>, and we continue that convention here.) The <em>Saccharomyces sensu stricto</em> genus is thought to have evolved â¼20 million years ago, and its species have a level of nucleotide divergence similar to that found between birds and humans (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Dujon 2006</a>). However, because yeasts lack a fossil record, the estimation of absolute divergence times for any set of yeast species is imprecise (<a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Taylor and Berbee 2006</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/1/11/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Resequencing and assembling the genomes of three Saccharomyces species. (A) Schematic showing phylogenetic relationships among nonhybrid members of the Saccharomyces sensu stricto genus plus the outgroup Kluyveromyces lactis based on (Kurtzman and Robnett 2003), (Nieduszynski and Liti 2011), and (Libkind, Hittinger et al., unpublished data). Branch lengths are not proportional to sequence divergence. The branch on which the whole-genome duplication occurred is marked. (B) Schematic depicting co-assembly of genomes from Illumina short-insert paired-end reads and mate-pair Sanger shotgun reads. Illumina reads were used to build contigs, which were stitched into scaffolds using mate-pair reads from the longer-insert Sanger libraries. Scaffolds were then joined into ultra-scaffolds (contiguous with chromosomes) using MEGABLAST and manual scaffold ordering." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2136499342" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Resequencing and assembling the genomes of three &lt;em&gt;Saccharomyces&lt;/em&gt; species. (A) Schematic showing phylogenetic relationships among nonhybrid members of the &lt;em&gt;Saccharomyces sensu stricto&lt;/em&gt; genus plus the outgroup &lt;em&gt;Kluyveromyces lactis&lt;/em&gt; based on (Kurtzman and Robnett 2003), (Nieduszynski and Liti 2011), and (Libkind, Hittinger &lt;em&gt;et al.&lt;/em&gt;, unpublished data). Branch lengths are not proportional to sequence divergence. The branch on which the whole-genome duplication occurred is marked. (B) Schematic depicting co-assembly of genomes from Illumina short-insert paired-end reads and mate-pair Sanger shotgun reads. Illumina reads were used to build contigs, which were stitched into scaffolds using mate-pair reads from the longer-insert Sanger libraries. Scaffolds were then joined into ultra-scaffolds (contiguous with chromosomes) using MEGABLAST and manual scaffold ordering.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/1/11/F1.medium.gif" width="440" height="155"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/1/11/F1.medium.gif" width="440" height="155"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/1/11/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/1/11/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460678" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1â</span> <p id="p-7" class="first-child">Resequencing and assembling the genomes of three <em>Saccharomyces</em> species. (A) Schematic showing phylogenetic relationships among nonhybrid members of the <em>Saccharomyces sensu stricto</em> genus plus the outgroup <em>Kluyveromyces lactis</em> based on (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Kurtzman and Robnett 2003</a>), (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Nieduszynski and Liti 2011</a>), and (Libkind, Hittinger <em>et al.</em>, unpublished data). Branch lengths are not proportional to sequence divergence. The branch on which the whole-genome duplication occurred is marked. (B) Schematic depicting co-assembly of genomes from Illumina short-insert paired-end reads and mate-pair Sanger shotgun reads. Illumina reads were used to build contigs, which were stitched into scaffolds using mate-pair reads from the longer-insert Sanger libraries. Scaffolds were then joined into ultra-scaffolds (contiguous with chromosomes) using MEGABLAST and manual scaffold ordering.</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">The genomes of <em>S. paradoxus</em>, <em>S. mikatae</em>, <em>S. kudriavzevii</em>, and <em>S. bayanus</em> were originally sequenced to low-medium coverage (3-8Ã) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cliften <em>et al.</em> 2003</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Kellis <em>et al.</em> 2003</a>). These sequence assemblies were far from complete with N50 values (<em>i.e.</em>, the minimum contig/scaffold length above which 50% of the entire assembly is contained in contigs or scaffolds equal to or larger than this value) well below 100 kb (as low as 11 kb for <em>S. kudriavzevii</em>). Due to the large number of gaps in each genome sequence, fewer than half of the potential orthologs of <em>S. cerevisiae</em> genes (2742/6615) were fully assembled and annotated across all four con-generic species. The missing data have both limited comprehensive sequence-based evolutionary analyses, and forced individual investigators to perform targeted resequencing to support sequence and genetic analyses of specific genes (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Guan <em>et al.</em> 2010</a>; <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Zill <em>et al.</em> 2010</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Hittinger <em>et al.</em> 2004</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Airoldi <em>et al.</em> 2009</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Gallagher <em>et al.</em> 2009</a>).</p><p id="p-9">To facilitate evolutionary genetic and genomic analyses within the <em>Saccharomyces sensu stricto</em> genus, we resequenced to high coverage and reassembled the genome sequences of <em>S. mikatae</em>, <em>S. kudriavzevii</em>, and <em>S. bayanus</em>. With these new genome sequences, an improved assembly of the <em>S. paradoxus</em> genome (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Liti <em>et al.</em> 2009</a>), and the reference genome of <em>S. cerevisiae</em> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Goffeau <em>et al.</em> 1996</a>), we determined the average and branch-specific evolutionary rates for a revised set of 5261 complete, annotated protein-coding orthologs across five <em>Saccharomyces</em> species, and identified 123 genes that may have been targets of positive selection. Through a relaxed-clock phylogenetic analysis, we obtained more accurate and precise relative estimates of interspecies divergence. Finally, we derived marked laboratory strains of the three species, permitting comparative genetic experiments at an unprecedented level of phylogenetic resolution and power within the <em>Saccharomyces</em> genus.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genome sequencing</h3><p id="p-10">Paired-end Illumina sequencing libraries were prepared from sonicated or nebulized genomic DNA according to manufacturer protocols with certain modifications (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Hittinger <em>et al.</em> 2010</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Lefrancois <em>et al.</em> 2009</a>). For <em>S. bayanus</em>, a <em>MAT</em><strong>a</strong> <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">hoÎ</a>::NatMX</em> derivative of CBS 7001 was sequenced. For <em>S. mikatae</em>, sheared DNA isolated from strain IFO 1815<sup>T</sup> was processed by an IntegenX robot. For <em>S. kudriavzevii</em>, haploid derivatives FM1097 and FM1109 were sequenced. Mean insert sizes (Â±SD) of Illumina libraries, as determined by SOAPdenovo, were as follows: IFO 1815<sup>T</sup>, 259 bp (Â±76 bp); IFO 1802<sup>T</sup>, 203 bp (Â±20 bp); ZP 591, 226 bp (Â±23 bp); CBS 7001, 437 bp (Â±45 bp).</p><p id="p-11">Sequencing was performed on Illumina Genome Analyzer II or II<sub>x</sub> machines at the Vincent Coates Genome Sequencing Lab, QB3, Berkeley, CA and at the University of Colorado School of Medicine. Read lengths varied for each strain as follows: <em>S. bayanus</em>, 51 bases; <em>S. mikatae</em>, 80 bases; <em>S. kudriavzevii</em>, 114 bases. All raw read data have been deposited in the SRA at NCBI (<a href="http://ncbi.nlm.nih.gov/sra">http://ncbi.nlm.nih.gov/sra</a>) in SRP006340 of SRA034902. Reads, assemblies, and annotation files are freely available at <a href="http://www.SaccharomycesSensuStricto.org">http://www.SaccharomycesSensuStricto.org</a>.</p></div><div id="sec-3" class="subsection"><h3>Co-assembly of Illumina and Sanger reads</h3><p id="p-12">Sanger reads were quality trimmed using LUCY (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chou and Holmes 2001</a>) (default parameters, except âminimum 60). Vector sequences at the 5â² end of reads were masked using Figaro (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">White <em>et al.</em> 2008</a>) (default parameters). Reads where more than 20% of bases were determined to be of vector origin were discarded; all others were 5â² trimmed and retained. Reads with remaining significant homology to the NCBI UniVec database (downloaded June 12, 2009) detected by Crossmatch (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ewing and Green 1998</a>) (default parameters) were discarded. Reads shorter than 60 bp and unpaired reads were discarded. Reads were 3â² trimmed to a maximum of 180 bp.</p><p id="p-13">Illumina reads were quality trimmed using fsq2fsa (available from D. Scannell on request), which trims bases from the 3â² ends of reads based on the Illumina quality score in small windows. The quality score was optimized for each dataset by assembling all reads with SOAPdenovo (<a href="http://soap.genomics.org.cn/soapdenovo.html">http://soap.genomics.org.cn/soapdenovo.html</a>; Version 1.05; July 29, 2010) (default params; âK = 31) and selecting the assembly with the best N50 (other metrics produced similar results). In addition, we used fsq2fsa to eliminate reads with significant matches to Illumina adapters and used the SOAP Corrector (default parameters) tool to correct errors in reads. We did not hard trim bases from the 5â² ends of reads because doing so did not improve assemblies.</p><p id="p-14">Assemblies were generated using SOAPdenovo (default parameters except âK as described below) using Illumina reads for both contig generation and scaffolding (rank = 1, pair_num_cutoff = 3, asm_flags = 3, map_len = 32) and Sanger reads for scaffolding only (rank = 2, pair_num_cutoff = 3, asm_flags = 2, map_len = 32). Sanger libraries of different origins were supplied to SOAPdenovo separately and the insert sizes for each determined by BLASTing against contigs longer than 10 kb. We optimized the âK parameter (Kmer) separately for each assembly by examining a range of values in the range 17â61. Finally, we used the SOAP GapCloser tool (default parameters) to fill assembly gaps using Illumina reads only.</p></div><div id="sec-4" class="subsection"><h3>Genome annotation</h3><p id="p-15">Our new assemblies as well as previously published sequences for <em>S. cerevisiae</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Engel <em>et al.</em> 2010</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Goffeau <em>et al.</em> 1996</a>) and <em>S. paradoxus</em> (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Liti <em>et al.</em> 2009</a>) were all annotated with a common pipeline. We used HMMER 1.8.4 (<a href="http://hmmer.janelia.org/software/archive">http://hmmer.janelia.org/software/archive</a>) to detect high scoring matches to an HMM created from an alignment of <em>S. cerevisiae</em> introns and flanking sequences. We then generated all ORFs above a context-dependent minimum size (60 bp at contigs ends and intron splice-sites, 300 bp at other locations), combined these with the putative splice-sites and selected the most upstream ATG as the putative start codon. Initially, we retained only possible gene models longer than 300 bp. Shorter genes were later recovered by extending BLAST HSPs (described below). Following this procedure, potential ORFs that spanned high-scoring matches to the splice site model were represented by multiple alternative gene models. At each locus, we aligned alternative models to the best available homolog (typically from <em>S. cerevisiae</em>) using Exonerate (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Slater and Birney 2005</a>) (âmodel affine:global âexhaustive 1 âbestn 1) and selected the model with the highest score. We sought positive evidence for all remaining gene models by BLASTing (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em> 1997</a>) against a local database of yeast proteins assembled from completely sequenced yeast genomes (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Wood <em>et al.</em> 2002</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Cliften <em>et al.</em> 2003</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Dujon <em>et al.</em> 2004</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Kellis <em>et al.</em> 2004</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Souciet <em>et al.</em> 2009</a>), the NCBI nr protein database, and <em>S. cerevisiae</em> Ty and LTR sequences downloaded from <em>Saccharomyces</em> Genome Database (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Engel <em>et al.</em> 2010</a>) (SGD, <a href="http://www.yeastgenome.org">http://www.yeastgenome.org</a>). In addition, we used HMMER3 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Eddy 2009</a>) to search our gene models against a database of 4704 protein HMMs derived from the Yeast Gene Order Browser (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Byrne and Wolfe 2006</a>) (YGOB, <a href="http://wolfe.gen.tcd.ie/ygob">http://wolfe.gen.tcd.ie/ygob</a>). In order to recover small genes or rapidly evolving genes, we subjected all presumed intergenic regions to low stringency BLAST searches against our local database of yeast proteins and searched genomic regions predicted by synteny to contain YGOB matches with our YGOB-HMM library. To detect potential novel genes, we used linear discriminant analysis to assign ORFs lacking homology to genes in the YGOB or yeast ortholog databases a âcoding probabilityâ based on whether their codon usage frequencies more closely resembled confirmed protein-coding genes or confirmed intergenic regions. Finally, we eliminated poorly supported gene models that overlapped well-supported models and merged neighboring models that were likely to be fragments of the same gene. The fragmented gene models were typically created by scaffold gaps or frameshifts (either real or due to sequencing errors). The remaining models were stratified according to the quality of the supporting evidence and comprised our initial protein-coding gene set. We used tRNA-ScanSE to identify tRNA genes (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Lowe and Eddy 1997</a>).</p><p id="p-16">As described in more detail below (<em>Ortholog Detection and Assignment</em>), we identified putative orthologs among the genomes of five speciesâ ârepresentativeâ strains (<em>i.e.</em>, <em>S. mikatae</em> IFO 1815<sup>T</sup>, <em>S. kudriavzevii</em> IFO 1802<sup>T</sup>, <em>S. bayanus</em> CBS 7001, <em>S. cerevisiae</em> S288C, and <em>S. paradoxus</em> CBS 432<sup>T</sup>). We used interspecies comparisons to improve annotations for these five strains in two ways. First, where intronâexon structures differed among species, we used Exonerate (âmodel protein2genome âexhaustive 0 âbestn 1) to perform a multiframe alignment of the closest homologous protein to the genomic region around each gene. Resulting models were preferred if they improved the consistency among species and typically contained small (2-7 bp) first exons or multiple frameshifts. Second, we aligned the putative 5â² termini and upstream regions of genes and selected the start codon that minimized the variation among species while maximizing overall gene length.</p></div><div id="sec-5" class="subsection"><h3>Ortholog detection and assignment</h3><p id="p-17">We detected single-copy orthologs among the five strains using a two-step procedure that incorporates both homology and synteny at both steps. First, we grouped genes across all species into families according to the best match in our YGOB-HMM database. Within each family, we selected the species with most representatives and founded an orthology group (or âorthogroupâ) with each copy. We then assigned genes from other species to orthogroups if they exceeded the specified minimum level of synteny support and the level of support for the next best orthogroup was significantly less. To compose a synteny statistic, we counted the number of YGOB-HMM families that were shared between a ten-gene window centered on the query gene and one centered on the focal orthogroup. We computed the hypergeometric probability of this observation assuming a total genome-size of 4704 genes (the number of YGOB models that can be distinguished) and used âlog<sub>10</sub>(P<sub>hyper</sub>) as our synteny statistic. Once all possible genes in a family were assigned to orthogroups, five-membered orthogroups were considered complete and removed. Remaining family members were then assigned to the remaining incomplete orthogroups and the procedure iterated until steady state.</p><p id="p-18">Once our initial set of orthogroups was defined we looked in the genomic regions between orthogroups for additional sets of syntenic orthologs that were not related to any of the YGOB-HMM families or had been missed for lack of synteny or other reasons. If orthologs were present at a genomic location in only a subset of species, we reannotated the syntenic region in the un-represented species to recover any missed orthologs. Finally, we re-examined all orthogroups and rejected those that exhibited either weak synteny or weak homology support. Orthologous genes were aligned in protein-space with FSA (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bradley <em>et al.</em> 2009</a>) and back-translated to DNA using RevTrans (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Wernersson and Pedersen 2003</a>). A comparison of gene structures and lengths suggested at least 4792 (â¼87%) of our orthologous gene sets were of high quality (<a href="http://www.g3journal.org/content/vol1/issue1/images/data/11/DC1/TableS1.xls">Table S1</a>).</p></div><div id="sec-6" class="subsection"><h3>Evolutionary analyses</h3><p id="p-19">To assess how improved genome sequences facilitated thorough evolutionary analyses, we determined the number of the single-copy orthologs present in the original 2003 genome annotations, which were not published jointly (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Kellis <em>et al.</em> 2003</a>, <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Cliften <em>et al.</em> 2003</a>). We considered the more complete Kellis <em>et al.</em> 2003 annotations for <em>S. bayanus</em>, <em>S. paradoxus</em>, and <em>S. mikatae</em>; the Cliften <em>et al.</em> 2003 annotations for <em>S. kudriavzevii</em>; and the <em>S. cerevisiae</em> annotations as downloaded from SGD on February 13, 2008. Annotations were taken to designate orthology if the original authors used the systematic name for <em>S. cerevisiae</em> gene along with the systematic name for the species in question in the versions deposited in SGD. As before, this analysis yielded 2805 genes with orthologs previously designated in all five <em>Saccharomyces sensu stricto</em> species (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Hittinger <em>et al.</em> 2010</a>). However, the Kellis <em>et al.</em> nomenclature allowed multiple genes to be named for the same <em>S. cerevisiae</em> gene (<em>e.g.</em>, lineage-specific duplicates that are all orthologous to a single <em>S. cerevisiae</em> gene). Excluding these genes, only 2742 genes were actually annotated as single-copy (1:1:1:1:1) orthologs in the 2003 genomes.</p><p id="p-20">To construct a data matrix for codon-based analyses, we first filtered out all ortholog groups from the new assemblies that did not contain orthologs from all five species as well as any ortholog groups in which more than 75% of alignment columns had missing data or gaps. The resulting data matrix was composed of 5152 orthologs and contained 7,880,523 nucleotide columns (the average length of a coding sequence alignment was â¼1530 base pairs).</p><p id="p-21">We examined variation in selection pressure along branches of the species tree and tested each gene for evidence of positive selection using the CODEML module from PAML (<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Yang 2007</a>). To examine selection pressure variation along branches of the species tree for each gene, we evaluated the log likelihood of two alternative hypotheses relative to the null hypothesis H<sub>0</sub>, under which all branches of the phylogeny exhibited the same Ï ratio of nonsynonymous (dN) to synonymous (dS) substitutions (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The first set of alternative hypotheses (H<sub>1</sub>) stated that the Ï ratio along the external branch for a given species was different from that in the rest of the branches of the phylogeny (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). To discriminate between genes that were consistent with a different Ï ratio only along a particular species from genes that were consistent with distinct Ï ratios in all branches of the phylogeny, we also tested the second alternative hypothesis (H<sub>2</sub>), in which each lineage exhibited its own Ï ratio against the H<sub>0</sub> hypothesis (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). To test for positive selection in each gene, we first evaluated the log likelihood of the null M7 model. Under M7, Ï values at different codon positions in a gene follow a beta distribution, where Ï is constrained to fall between zero and one. We then compared the log likelihood of the M7 model relative to that estimated by the alternative model M8, which, in addition to the zero to one beta distribution for Ï values, also allows for a subset of codon sites to have Ï values above one. We excluded all genes with dS values of zero. All tests were done at <em>P</em> = 0.01 significance. In <a href="http://www.g3journal.org/content/vol1/issue1/images/data/11/DC1/FileS1.xls">File S1</a>, we have also provided an optional filter to remove genes from these screens whose high Ï values were driven by abnormally low denominators (<em>i.e.</em>, dS values two standard deviations below the mean). Depending on the specific alternative hypothesis, application of the filter removes â¼5â10% of the genes rejecting H<sub>0</sub>, including many genes encoding ribosomal proteins and other translation factors that are not likely to be experiencing lineage-specific selection.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/1/11/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genes exhibiting lineage-specific rates of evolution in the Saccharomyces sensu stricto genus. (A) The three alternative hypotheses designed to test whether genes are evolving at a different rate in each of five species of the Saccharomyces sensu stricto genus. Under hypothesis H0 all branches of the tree exhibit the same &#x3C9; ratio of nonsynonymous to synonymous substitutions. Under the set of H1 hypotheses, the &#x3C9; ratio along a given species&#x2019; branch is different from that along all other branches of the tree. Under the H2 hypothesis, each branch exhibits its own &#x3C9; ratio. (B) Numbers of genes with lineage-specific rates of evolution in the Saccharomyces sensu stricto genus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2136499342" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genes exhibiting lineage-specific rates of evolution in the &lt;em&gt;Saccharomyces sensu stricto&lt;/em&gt; genus. (A) The three alternative hypotheses designed to test whether genes are evolving at a different rate in each of five species of the &lt;em&gt;Saccharomyces sensu stricto&lt;/em&gt; genus. Under hypothesis H&lt;sub&gt;0&lt;/sub&gt; all branches of the tree exhibit the same &#x3C9; ratio of nonsynonymous to synonymous substitutions. Under the set of H&lt;sub&gt;1&lt;/sub&gt; hypotheses, the &#x3C9; ratio along a given species&#x2019; branch is different from that along all other branches of the tree. Under the H&lt;sub&gt;2&lt;/sub&gt; hypothesis, each branch exhibits its own &#x3C9; ratio. (B) Numbers of genes with lineage-specific rates of evolution in the &lt;em&gt;Saccharomyces sensu stricto&lt;/em&gt; genus.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/1/11/F2.medium.gif" width="380" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/1/11/F2.medium.gif" width="380" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/1/11/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/1/11/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460768" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2â</span> <p id="p-22" class="first-child">Genes exhibiting lineage-specific rates of evolution in the <em>Saccharomyces sensu stricto</em> genus. (A) The three alternative hypotheses designed to test whether genes are evolving at a different rate in each of five species of the <em>Saccharomyces sensu stricto</em> genus. Under hypothesis H<sub>0</sub> all branches of the tree exhibit the same Ï ratio of nonsynonymous to synonymous substitutions. Under the set of H<sub>1</sub> hypotheses, the Ï ratio along a given speciesâ branch is different from that along all other branches of the tree. Under the H<sub>2</sub> hypothesis, each branch exhibits its own Ï ratio. (B) Numbers of genes with lineage-specific rates of evolution in the <em>Saccharomyces sensu stricto</em> genus.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">We inferred relative divergence times for the yeast phylogeny using the BEAST software, version 1.6.1 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Drummond and Rambaut 2007</a>) on a data set of 106 genes spanning the yeast genome (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Rokas <em>et al.</em> 2003</a>). Because the fungal and yeast fossil records are sparse and reliable fossil calibration points unavailable, we estimated all branches in units of substitution/site. For all phylogenetic analyses using BEAST, we assumed the SRD06 model of sequence evolution (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Shapiro <em>et al.</em> 2006</a>), allowing for rate heterogeneity across sites through the gamma distribution, and the uncorrelated log-normal relaxed clock model. We chose the Yule process as our tree prior. We ran three independent runs for 10,000,000 generations. We verified the convergence of runs by examining the effective sample size of the likelihood and posterior probability parameters for each analysis (&gt;100 parameters) and by visually inspecting the likelihood and posterior probability distributions across independent runs. We discarded the first 10% of sampled data points from each run as burn-in.</p></div><div id="sec-7" class="subsection"><h3>Identifying potential gene gains and losses</h3><p id="p-24">We used a variety of approaches to identify gene gain and loss candidates that we subsequently manually inspected. First, we took a synteny-based approach to identify genes that were lost or gained in internal chromosomal regions with well-conserved synteny. Briefly, we âwalkedâ along the genome of <em>S. cerevisiae</em> and checked the region between each orthogroup (conserved 1:1:1:1:1 syntenic orthologs) and the previous orthogroup for the presence of genes in one or more species. We excluded regions that contained assembly gaps in any of the five species or where there were more than three intervening features in any species. As these criteria were restrictive, we also took a homology-based approach, grouping genes by their homology to YGOB-HMM families and selecting families that differed in size among species (but excluding the small number of very large families). In addition, we examined any genes that had evidence of function (see <em>Genome annotation</em>) but which had no detectable homology to either a YGOB-HMM or to a gene in SGD. Finally, we also sought to identify cases where although a gene was detectable by homology, the reading-frame had been disabled. To do this, we sorted genes by the number of frameshifts required to reconstitute a full-length gene during the annotation process and examined any gene with three or more disruptions. We also sorted orthogroups by the standard deviation of gene length divided by the mean gene length and examined the top 200 outliers to detect genes that had been severely truncated.</p></div><div id="sec-8" class="subsection"><h3>Strain construction</h3><p id="p-25">Prototrophic diploid yeast strains CBS 7001, IFO 1802<sup>T</sup>, ZP 591, and IFO 1815<sup>T</sup> were made heterothallic by inactivating the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">HO</a></em> gene. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">HO</a></em>/<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">hoÎ</a></em> heterozygous diploids were sporulated and tetrads were dissected to isolate <em>MAT</em><strong>a</strong> and <em>MAT</em>Î± <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">hoÎ</a></em> haploids. For <em>S. mikatae</em> and <em>S. kudriavzevii</em>, auxotrophic markers were generated by gene targeting. For <em>S. bayanus</em>, auxotrophic markers were introduced by EMS mutagenesis as described previously (<a id="xref-ref-95-2" class="xref-bibr" href="#ref-95">Zill and Rine 2008</a>). To generate the <em>S. kudriavzevii <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">trp1Î</a>0</em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3Î</a>0 delitto perfetto</em> alleles, prototrophic strains were transformed with DNA oligos and/or PCR products encoding a direct junction of the sequences immediately upstream of the start codon and downstream of the stop codon (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Storici <em>et al.</em> 2001</a>). Transformed pools were grown on 5-FAA and 5-FOA media to select for strains that lack functional <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em>, respectively. All other homologous-recombination-based gene targeting was performed by one-step gene replacement using standard drug-resistance cassettes (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Guldener <em>et al.</em> 1996</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Longtine <em>et al.</em> 1998</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Goldstein and Mccusker 1999</a>) or standard two-step replacements using <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Storici and Resnick 2006</a>). All gene disruptions were confirmed using PCR and/or sequencing to examine the 5â² and 3â² ends of targeted ORFs.</p><p id="p-26">Transformation protocols for <em>S. bayanus</em> (<a id="xref-ref-95-3" class="xref-bibr" href="#ref-95">Zill and Rine 2008</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Gallagher <em>et al.</em> 2009</a>) and <em>S. kudriavzevii</em> (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Hittinger <em>et al.</em> 2010</a>) have been described. Here we again summarize the relevant modifications to the standard PEG/LiAc heat-shock protocol used for <em>S. cerevisiae</em> transformation. All of the nonâ<em>S. cerevisiae</em> species appear to be quite sensitive to prolonged heat shock at 42Â°C. For <em>S. bayanus</em>, heat shock was performed for five minutes at 42Â°C after a 10-minute incubation at room temperature in the transformation mixture. For <em>S. kudriavzevii</em>, heat shock was performed for 30 min at 34Â°C. For <em>S. mikatae</em>, heat shock was performed for five minutes at 37Â°C after a 10-minute incubation at room temperature. For all three species, subsequent outgrowth and culture were performed at room temperature (â¼23Â°C). For <em>S. bayanus</em>, gene disruption primers contained 40nt homologous to the sequences immediately flanking the targeted ORF. For <em>S. mikatae</em> and <em>S. kudriavzevii</em> the primers had 50 nt and 70 nt of homology, respectively.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Improved genome assemblies for <em>S. mikatae</em>, <em>S. kudriavzevii</em>, and <em>S. bayanus</em></h3><p id="p-27">The taxonomic type or other representative strains of <em>S. mikatae</em>, <em>S. kudriavzevii</em>, and <em>S. bayanus</em> were previously sequenced to 3-8Ã coverage with Sanger sequencing technology (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Cliften <em>et al.</em> 2003</a>; <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Kellis <em>et al.</em> 2003</a>). Though this depth of sequencing provided value in terms of genome coverage, and where coverage was highest provided good long-range continuity (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cliften <em>et al.</em> 2006</a>), the resulting assemblies had many gaps and a moderate number of errors. To obtain more complete assemblies that would support base-pair level analyses of these three species, we sequenced short-insert (203-437 bp) Illumina paired-end libraries to greater than 100Ã coverage (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) and assembled these together with the available sequences (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). The high raw coverage afforded by the short-read technology minimized erroneous base calls and gaps in unique regions, whereas the longer inserts from the shotgun sequencing projects (3198-4789 bp inserts; trimmed reads averaged 179 bp in length) helped bridge repetitive regions and establish long-range scaffolds. In addition, we sequenced and assembled a genetically and phenotypically diverse <em>S. kudriavzevii</em> strain (ZP 591) (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Sampaio and Goncalves 2008</a>) from Illumina reads only, as there were no Sanger shotgun reads available.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460695/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460695/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1â</span> <span class="caption-title">Short-read library statistics</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">We aggressively trimmed, corrected, and discarded lower quality reads and assembled those satisfying our quality control criteria using SOAPdenovo (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Li <em>et al.</em> 2010</a>) and a custom parameter optimization strategy (see <em>Materials and Methods</em>). The resulting assemblies exhibited comparable total base counts (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), suggesting that each had converged on the similar physical genome sizes that were predicted by karyotyping and other studies (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Fischer <em>et al.</em> 2000</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Naumov <em>et al.</em> 2000</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Gonzalez <em>et al.</em> 2008</a>). Indeed, the assembliesâ size range of 11.6â11.9 Mb was close to the completed <em>S. cerevisiae</em> genome size (12.1 Mb) and bracketed that of the ostensibly completed <em>S. paradoxus</em> (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Liti <em>et al.</em> 2009</a>) (11.7 Mb) assembly, suggesting that our assemblies were essentially complete.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460694/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460694/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2â</span> <span class="caption-title">Genome assembly summary statistics, before and after manual ordering of scaffolds</span><div class="sb-div caption-clear"></div></div></div><p id="p-33">The new assemblies also had many fewer gaps and greatly improved continuity compared to the originally published versions. For instance, our unordered <em>S. mikatae</em> assembly consisted of 1220 scaffolds, of which 159 were longer than 500 bp, and many of the shorter ones were likely to be spurious byproducts of the short-read assembly process. Notably, those 159 scaffolds accounted for &gt;98% of the assembly and contained just 18 gaps (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). By contrast, the previous assemblies consisted of more than 300 scaffolds longer than 500 bp, and these contained &gt;1300 gaps averaging &gt;600 bp. Indeed, the contig N50s were on the order of 20 kb compared to our scaffold N50 of &gt;360 kb (a fair comparison given the paucity of gaps in our assembly). Though the relative improvement varied by species, the N50 range (151â445 kb) of our unordered assemblies represented significant improvements in all cases (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-34">Excluding sub-telomeric regions, five <em>S. bayanus</em> chromosomes (I, VI, XI, XII, XIII) were each represented by a single large scaffold. The same was true for five <em>S. mikatae</em> chromosomes (II, VII, VIII, XI, XVI), and because of the paucity of gaps in this assembly the sequences were almost completely contiguous (note that in all species other than <em>S. cerevisiae</em> a gap persists at the rDNA locus on ChrXII). These observations, and the small number of rearrangements believed to distinguish the karyotypes of <em>Saccharomyces sensu stricto</em> yeasts (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Fischer <em>et al.</em> 2000</a>), prompted us to use sequence similarity to organize our initial assemblies with respect to the <em>S. cerevisiae</em> genome. By ordering and orienting 46â154 scaffolds per assembly from MEGABLAST results, we were able to organize 96â98% of bases in each species into 16 ultra-scaffolds, which were likely collinear with chromosomes (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). As <em>S. bayanus</em> and <em>S. mikatae</em> have translocations relative to <em>S. cerevisiae</em>, we numbered all ultra-scaffolds/chromosomes according to which centromere they contained (defined largely by flanking synteny with <em>S. cerevisiae</em>; <a href="http://www.g3journal.org/cgi/data/1/1/11/DC1/1">supporting information</a>, <a href="http://www.g3journal.org/cgi/data/1/1/11/DC1/2">Figure S1</a>). This nomenclature differs from that proposed by Fischer <em>et al.</em>, but is simpler and hopefully acceptable to the community.</p><p id="p-35">In creating ultra-scaffolds for each species, we had little difficulty observing the two known translocations in <em>S. mikatae</em> IFO 1815<sup>T</sup> and four translocations in <em>S. bayanus</em> CBS 7001 (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Fischer <em>et al.</em> 2000</a>). Indeed, most translocation breakpoints were spanned by large scaffolds, and were immediately visible in the scaffold alignments to the <em>S. cerevisiae</em> genome. As previously observed (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Fischer <em>et al.</em> 2000</a>), the genomes of both <em>S. kudriavzevii</em> strains appeared completely collinear with the <em>S. cerevisiae</em> genome. These observations suggested that the new assemblies were free of gross assembly errors. To test whether our proposed ultra-scaffolds had erroneously linked genomic segments, we designed PCR primers spanning 32 proposed junctions in <em>S. bayanus</em>. Twenty-nine of these primer sets (91%) yielded PCR products of appropriate sizes (0.5â4 kb), and the remainder likely failed due to the length of the intervening gap (data not shown).</p><p id="p-36">Despite the overall improvements in the genome assemblies, several systematic differences were evident. Notably, the unordered <em>S. kudriavzevii</em> and <em>S. mikatae</em> assemblies contained many fewer gaps than the <em>S. bayanus</em> assembly whereas the unordered <em>S. kudriavzevii</em> assemblies had considerably smaller N50 values (though still an order of magnitude greater than the original 2003 assemblies). These results were direct consequences of the mixture of read types used to assemble each strain. For instance, the greater N50 obtained for <em>S. bayanus</em> and <em>S. mikatae</em> than for <em>S. kudriavzevii</em> IFO 1802<sup>T</sup> was due to the availability of two long-insert libraries in the first two cases (â¼90,000 read pairs) but only one library in the latter (â¼27,000 read pairs). On the other hand, the higher number of gaps in the unordered <em>S. bayanus</em> assembly relative to <em>S. mikatae</em> and <em>S. kudriavzevii</em> IFO 1802<sup>T</sup> is likely due to the lower Kmer size used to assemble the Illumina reads (31 bp), and the somewhat lower Kmer coverage (45.1Ã; <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). [Kmer is the length of DNA that is used to construct the de Bruijn graph during assembly, and is the minimum number of identical bases required to join two overlapping reads (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Zerbino and Birney 2008</a>).] By contrast, in the case of both <em>S. mikatae</em> and <em>S. kudriavzevii</em> our raw short-read coverage of 262.1â362.3Ã translated to 67.0â98.4Ã Kmer coverage at a Kmer size of 61 bp. Given the close tracking between Kmer size and gap number, we believe that many of the remaining gaps in the <em>S. bayanus</em> assembly were likely to be between 31 and 61 bp in length and thus to represent minimal obstacles to routine use. Indeed, in annotating the new <em>S. bayanus</em> assembly it was clear that most gaps caused small interruptions in gene sequences rather than gene absences.</p></div><div id="sec-11" class="subsection"><h3>Telomeres, transposons, and other repeats: an assembly challenge</h3><p id="p-37">Telomeres, transposons, and other long and highly repetitive sequences such as mammalian centromeres remain one of the main stumbling blocks to the assembly of truly complete genomes. To assemble and place any given repetitive sequence correctly, one must have library inserts of a length greater than the length of that sequence as well as unique DNA sequence on at least one side. Our Illumina libraries had mean insert sizes of 200â440 bp, which is considerably shorter than full-length transposons in <em>S. cerevisiae</em> (<em>e.g.</em>, full-length Ty1 elements are â¼6 kb long). In <em>S. cerevisiae</em>, sub-telomeric duplication blocks and repeats that possess few sequence differences can reach 30kb in size (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Louis 1995</a>). The combination of the high-coverage Illumina reads with the lower-coverage Sanger shotgun reads (4â5 kb insert sizes) allowed us to assemble many subtelomeric sequences, but they remained much more fragmented than the rest of the genome. Indeed, although all of our assemblies extended into the subtelomeric regions of most chromosomes, few scaffolds appeared to reach the telomeric terminal repeats. The two best examples, Skud_66 (IFO 1802<sup>T</sup>) and Skud_52 (ZP 591), were large <em>S. kudriavzevii</em> scaffolds (&gt;50 kb) that corresponded along most of their lengths to <em>S. cerevisiae</em> ChrIV (right sub-telomere) and ChrVII (left sub-telomere), respectively. However, near the telomere end of each scaffold, the synteny with <em>S. cerevisiae</em> chromosomes is broken, leaving multiple apparent rearrangements with other telomeres. In general, synteny was poorly conserved in the sub-telomeric regions of all three species.</p><p id="p-38">The ability to assemble the telomeric regions correlated with Illumina read length, even when the Kmer length used for two of the assemblies was identical. <em>S. kudriavzevii</em>, with a read length of 114 bases and Kmer size of 61, had nine scaffolds with terminal-repeat sequences, including the two completely assembled telomeres described above. <em>S. mikatae</em>, with a read length of 80 bases and Kmer size of 61, had only one small scaffold containing terminal-repeat sequences. <em>S. bayanus</em>, with a read length of 51 bases and Kmer size of 31, had no terminal-repeat sequences in its genome assembly. These data suggest that read length and library insert size were both limiting factors to assembling repetitive regions with current genome assembly software. All of our raw reads are publicly available (<a href="http://ncbi.nlm.nih.gov/sra">http://ncbi.nlm.nih.gov/sra</a>; <a href="http://www.SaccharomycesSensuStricto.org">http://www.SaccharomycesSensuStricto.org</a>), and we urge interested members of the genomics community to use them to develop methods to improve the genome assemblies of these and other problematic regions.</p></div><div id="sec-12" class="subsection"><h3>Updated genome annotations and identification of syntenic orthologs</h3><p id="p-39">To provide a basis for future comparative functional studies, we annotated tRNAs, repeats, centromeres, and protein-coding genes in the new genome assemblies. We anticipate the addition of more classes of functional elements in the future. For the present annotation, we took two steps to generate a robust set of protein-coding gene predictions. First, we used a yeast splice-site model as well as homology-assisted gene prediction to ensure correctly delimited genes with introns or with reading-frame interruptions. This step allowed us to correctly recover genes that have traditionally not been well annotated. For example, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005040">RPS7B</a></em> (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001063">YHR021C</a></em>) and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004068">BOS1</a></em> (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004068">YLR078C</a></em>) both have first exons that are just 3 bp long, an intron (550 bp and 87 bp respectively) and a larger second exon. In the case of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005040">RPS7B</a></em> the intron is more than twice as long as the reading frame, which is only 249 bpâbelow the 100 AA (300 bp) length minimum that is a common threshold. In both cases we detected a strong match to our splice site model that allowed us to recover homologous gene structures in all five species. Similarly, we predicted homologous structures consisting of three exons and two introns for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000028510">SUS1</a></em> (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000028510">YBR111W-A</a></em>) in all five species.</p><p id="p-40">To distinguish spurious open reading frames from biologically meaningful ones, we developed a database of HMMER3 protein hidden Markov models (HMMs) based on the Yeast Gene Order Browser (YGOB) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Byrne and Wolfe 2005</a>). HMMER3 exhibits significantly increased sensitivity and specificity compared to older tools such as BLAST and combined with the gold standard human-curated YGOB database (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Byrne and Wolfe 2005</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Wapinski <em>et al.</em> 2007a</a>) provides a powerful basis for recognizing small or fast-evolving genes and for distinguishing closely related ones. Using this approach, we could detect small genes that can easily go undetected such as the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002869">MFA1</a></em> gene (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002869">YDR461W</a></em>, which encodes a 37AA-long mating pheromone <strong>a</strong>-factor protein) and the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002103">PMP2</a></em> gene (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002103">YEL017C-A</a></em>, which encodes a 40AA protein). We also detected rapidly evolving genes such as <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002635">SIR4</a></em> (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002635">YDR227W</a></em>) (<a id="xref-ref-96-2" class="xref-bibr" href="#ref-96">Zill <em>et al.</em> 2010</a>) and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000352">YSW1</a></em> (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000352">YBR148W</a></em>) (<a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Kellis <em>et al.</em> 2003</a>) with high confidence in all species. Altogether, we detected between 5440 and 5559 genes with homology to one of our YGOB-HMMs (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, <a href="http://www.g3journal.org/cgi/data/1/1/11/DC1/3">Table S2</a>) and no more than 100 of the 4704 families represented by a YGOB-HMM were absent from any genome (the one exception was <em>S. paradoxus</em>, which had some remaining large gaps). Thus, these genes defined the core yeast proteome.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460690/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460690/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3â</span> <span class="caption-title">Counts of annotated tRNA and protein-coding genes across representative strains of five Saccharomyces species</span><div class="sb-div caption-clear"></div></div></div><p id="p-44">We also detected several hundred genes in each genome with sequence similarity to a gene in the <em>Saccharomyces</em> Genome Database (SGD, <a href="http://www.yeastgenome.org">http://www.yeastgenome.org</a>) but not to a YGOB-HMM (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). In contrast to the consistent number of genes with homology to YGOB-HMMs, the recovery of genes with homology to an SGD-only gene declined rapidly with evolutionary distance from <em>S. cerevisiae</em> (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). Although some of these are located in subtelomeres, many are annotated as dubious by SGD suggesting they are not biologically relevant (<a id="xref-ref-47-5" class="xref-bibr" href="#ref-47">Kellis <em>et al.</em> 2003</a>). The best sequence-based method to determine whether a gene is real is to test if dN/dS (Ï) is significantly different from 1; however, we were able to recover complete ortholog sets for only a small fraction of these genes (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>) and hence this test was not performed. Finally, we also predicted a small number of open reading frames that we could associate neither to YGOB-HMMs nor to a known SGD gene but for which we found some evidence of function (see <em>Materials and Methods</em>). On closer inspection, the majority of these were related either to transposable elements or derived from YÂ´ elements, but some appeared to be species-specific genes (discussed further below). Taken together, our reannotation suggested that <em>Saccharomyces sensu stricto</em> yeasts share a large common proteome with other yeast species (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Dujon <em>et al.</em> 2004</a>; <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Souciet <em>et al.</em> 2009</a>) but also possess a much smaller set of genes that distinguish them from other yeasts and from each other.</p><p id="p-45">To facilitate future comparative studies, we used our revised gene annotation to identify sets of genes that are orthologous across the representative strains of all five species. By making extensive use of homology and synteny (see <em>Materials and Methods</em>), we identified 5261 sets of orthologous proteins (82â87% of predicted protein-coding genes; <a href="http://www.g3journal.org/content/vol1/issue1/images/data/11/DC1/TableS1.xls">Table S1</a>) as well as 229 tRNAs at syntenic locations, for a total of 5487 complete orthologous gene sets (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). (We note that 5141 of the 5261 protein-coding orthogroups had YGOB support.) This analysis approximately doubled the number of five-species syntenic orthologs that were previously available (2742) (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Cliften <em>et al.</em> 2003</a>; <a id="xref-ref-47-6" class="xref-bibr" href="#ref-47">Kellis <em>et al.</em> 2003</a>), and also dramatically expanded the previous Kellis <em>et al.</em> dataset that identified 4180 orthologs but did not include <em>S. kudriavzevii</em>. The nearly complete genomes and improved annotations can be visually perused on a Gbrowse site (available at <a href="http://www.SaccharomycesSensuStricto.org">http://www.SaccharomycesSensuStricto.org</a>).</p></div><div id="sec-13" class="subsection"><h3>Evolutionary analyses of a nearly complete set of Saccharomyces orthologs</h3><p id="p-46">Our expanded ortholog dataset (5261 orthogroups) allowed examination of the selection pressures operating on yeast protein-coding genes on a genome-scale, painting a broad-brushstroke picture of how yeast genes evolve. This portrait should be considered conservative because of the absence of some subtelomeric genes from our ortholog set, and because of the filters we applied, which removed orthologs with problematic alignments (see <em>Materials and Methods</em>). We calculated the average Ï ratio of nonsynonymous (dN) to synonymous (dS) substitutions (dN/dS ratio) for each of 5152 unique (1:1:1:1:1) orthologs conserved across all five species. No high-quality ortholog achieved an average Ï value of 1, with the fastest evolving gene (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000388">YBR184W</a></em>) having Ï = 0.58 (<a href="http://www.g3journal.org/content/vol1/issue1/images/data/11/DC1/FileS1.xls">File S1</a>). The average across all high-quality orthologs was Ï = 0.10, consistent with previous studies (<a id="xref-ref-47-7" class="xref-bibr" href="#ref-47">Kellis <em>et al.</em> 2003</a>), and suggesting that most yeast genes are subject to strong purifying selection.</p><p id="p-47">To examine yeast gene evolution on a finer scale, we considered variation in selection pressure across the <em>Saccharomyces sensu stricto</em> genus by comparing three alternative hypotheses of the distribution of the Ï ratio (dN/dS) along the phylogeny (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>, <a href="http://www.g3journal.org/content/vol1/issue1/images/data/11/DC1/FileS1.xls">File S1</a>). Specifically, for each set of unique orthologs conserved across all five species, we evaluated the hypothesis that all branches of the phylogeny exhibited the same Ï ratio (H<sub>0</sub>) against a set of alternative hypotheses (H<sub>1</sub>) under which the Ï ratio along the external branch for a given species was different from that in the rest of the branches of the phylogeny (H<sub>1</sub> hypotheses; <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A</a>). To discriminate between genes that exhibited a different Ï ratio only along a branch leading to a particular species and genes that had distinct Ï ratios for all branches of the phylogeny, we also tested the set of H<sub>1</sub> hypotheses against the hypothesis in which each lineage exhibited its own Ï ratio (H<sub>2</sub>; <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>). As expected, the overwhelming majority of genes did not reject the null hypothesis of a uniform Ï ratio across the phylogeny, but 107â251 genes in each species exhibited statistically significant lineage-specific Ï ratios (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2B</a>). For example, our results indicated that 152 <em>S. cerevisiae</em> genes showed lineage-specific Ï ratios. Of those 152 genes, 65 supported a higher Ï ratio in the <em>S. cerevisiae</em> branch relative to the rest of the phylogeny, whereas 87 genes supported a lower Ï ratio. The complete list of candidate genes that exhibit lineage-specific Ï ratios is provided in <a href="http://www.g3journal.org/content/vol1/issue1/images/data/11/DC1/FileS1.xls">File S1</a>.</p><p id="p-48">We also examined each of the 5152 orthologs for evidence of positive selection by comparing two alternative models (called M7 and M8) of the distribution of Ï that differ with respect to the allowance of a subset of codon sites to be under positive selection (with Ï &gt; 1). We found 123 genes (after filtering, see <em>Materials and Methods</em>) whose sequence evolution fit a model of codon evolution in which a detectable fraction of sites has been under positive selection (M8) better than it fit a model where sites evolve neutrally (M7) (<a href="http://www.g3journal.org/content/vol1/issue1/images/data/11/DC1/FileS1.xls">File S1</a>). Rapidly evolving genes are more likely to have functionally diverged, potentially contributing to genetic incompatibilities between species (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Orr 2009</a>). Among the 123 genes found in our analysis was <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002635">SIR4</a></em>, which has previously been shown to be under positive selection by multiple measures (<a id="xref-ref-96-3" class="xref-bibr" href="#ref-96">Zill <em>et al.</em> 2010</a>; Zill <em>et al.</em>, unpublished data), supporting the view that codons within these 123 genes were evolving faster than neutral, and were not simply misaligned. Several genes involved in mitochondrial maintenance and inheritance (<em>e.g.</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002262">QRI7</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000047">AIM2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003128">AIM14</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001442">AIM21</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006020">AIM43</a></em>) fit this pattern, consistent with suggestions that divergence in nuclear genes with mitochondrial functions have contributed to speciation in <em>Saccharomyces</em> (via cytonuclear incompatibility) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chou and Leu 2010</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Lee <em>et al.</em> 2008</a>). Intriguingly, we also found several meiotic genes involved in homologous chromosome interactions, and/or DNA-repair-coupled chromatin modifications during meiosis, that were undergoing positive selection (<em>e.g.</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003218">ZIP2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004681">PDS5</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003628">SRS2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002848">DOT1</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002771">ESC2</a></em>). <em>Saccharomyces sensu stricto</em> species are post-zygotically isolated due to a failure of inter-species homologous chromosomes to segregate properly in meiosis I, which is caused in part by nucleotide sequence divergence acted on by the mismatch repair machinery (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Greig <em>et al.</em> 2003</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Liti <em>et al.</em> 2006</a>). It is possible that these rapidly diverging chromosome-biology genes play a role in the meiotic barrier between species.</p></div><div id="sec-14" class="subsection"><h3>A relative timescale of interspecies divergence</h3><p id="p-49">To establish a quantitative framework for interspecies divergence in the <em>Saccharomyces sensu stricto</em> genus, we used a relaxed molecular clock approach to estimate the relative divergence times among lineages (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Drummond <em>et al.</em> 2006</a>). However, because we were unable to consistently estimate the necessary parameters with our complete set of orthogroups (data not shown), we analyzed a smaller dataset of 106 genes spanning the yeast genome (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Rokas <em>et al.</em> 2003</a>). As the origin of the genus coincides with the divergence of <em>S. bayanus</em> from the rest of the <em>Saccharomyces sensu stricto</em> lineage, our results showed that the divergence of <em>S. kudriavzevii</em> was 78% as old as the lineage and the divergence of <em>S. mikatae</em> was 53% as old, whereas the divergence of <em>S. cerevisiae</em> and <em>S. paradoxus</em> was 33% as old as the lineage (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/1/11/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relaxed molecular clock estimation of relative species divergence within the Saccharomyces sensu stricto genus. The top scale bar and the values above branches denote estimated substitutions per site. The bottom scale bar expresses species divergence in percentage points relative to the origin of the genus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2136499342" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relaxed molecular clock estimation of relative species divergence within the &lt;em&gt;Saccharomyces sensu stricto&lt;/em&gt; genus. The top scale bar and the values above branches denote estimated substitutions per site. The bottom scale bar expresses species divergence in percentage points relative to the origin of the genus.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/1/11/F3.medium.gif" width="440" height="257"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/1/11/F3.medium.gif" width="440" height="257"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/1/11/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/1/11/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460750" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3â</span> <p id="p-50" class="first-child">Relaxed molecular clock estimation of relative species divergence within the <em>Saccharomyces sensu stricto</em> genus. The top scale bar and the values above branches denote estimated substitutions per site. The bottom scale bar expresses species divergence in percentage points relative to the origin of the genus.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Species-specific gene gains and losses</h3><p id="p-51">The improved assemblies and consequent improved ability to detect orthologs in all five species allowed us to revisit which genes had been gained and lost in specific lineages (see <em>Materials and Methods</em>). Below we present the results of our initial five-genome survey as a set of hypotheses, in which each candidate genetic difference (such as a species-specific gene loss) predicts an alteration of the ancestral <em>Saccharomyces</em> genetic network. To organize lineage-specific gene-level changes into a simple, logical framework, we first considered any genes that were present in the common ancestor of these yeasts but were not found in all five of the modern genomes as âlosses.â (We note that in some cases we were able to detect pseudogenes or truncated genes but in other cases the genes had essentially vanished, consistent with either large deletion events or the accumulation of many smaller changes.) We next divided the losses into three sub-categories: âlineage-restricted lossesâ (lost in one or two species), âwidespread lossesâ (absent from more than two species, but may have involved more than one loss event), and losses of one paralog of a duplicate gene pair descended from the whole-genome duplication (âduplicate gene lossesâ). Any genes present in one or more of the five <em>Saccharomyces</em> species but not in the <em>sensu stricto</em> ancestor we considered to be âgains.â We also analyzed tRNA variation in each of the five species.</p><div id="sec-16" class="subsection"><h4>Lineage-restricted losses:</h4><p id="p-52">We identified at least 44 examples of genes lost from only one or two lineages (<a href="http://www.g3journal.org/content/vol1/issue1/images/data/11/DC1/FileS2.xls">File S2</a>). One interesting example of a gene that appeared to be completely missing was the loss from <em>S. cerevisiae</em> of a GATA family transcription-factor gene [Anc_2.395; we use the nomenclature proposed by YGOB to identify conserved yeast loci to also refer to the derived YGOB-HMM (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Gordon <em>et al.</em> 2009</a>)] related to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004003">GAT3</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001452">GAT4</a></em>. This loss suggested that a suite of target genes may have experienced regulatory changes relative to the <em>Saccharomyces</em> ancestral circuit. In another example, <em>S. mikatae</em> has lost <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003319">PDC6</a></em>, which encodes a minor pyruvate decarboxylase expressed under sulfur limitation. This gene has additionally been pseudogenized in <em>S. kudriavzevii</em>, suggesting that these species may have experienced selective pressure to alter their alcohol metabolism.</p><p id="p-53">In addition to identifying genes that were entirely missing, we detected many pseudogenes with varying numbers of reading-frame disruptions. A well-known example of this sort of mutation is the parallel inactivation (as pseudogenes) of all the <em>GAL</em> genes in <em>S. kudriavzevii</em> IFO 1802<sup>T</sup> but not ZP 591 (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Hittinger <em>et al.</em> 2004</a>; <a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">Hittinger <em>et al.</em> 2010</a>). Our analysis recovered all of these and a previously described mutation in the <em>S. bayanus</em> CBS 7001 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001277">BAR1</a></em> gene (<a id="xref-ref-95-4" class="xref-bibr" href="#ref-95">Zill and Rine 2008</a>), where a single base-pair deletion leads to a frameshift and truncated coding sequence. We found several additional likely pseudogenes in each of the five species (<a href="http://www.g3journal.org/content/vol1/issue1/images/data/11/DC1/FileS2.xls">File S2</a>). Notable among these were several metabolic genes such as <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003386">GTO1</a></em>, which encodes an omega-class glutathione transferase, lost in <em>S. bayanus</em> and <em>S. mikatae</em>. Similarly, <em>S. mikatae</em> lost the genes <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006092">OYE3</a></em>, which encodes an NADPH oxidoreductase, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003488">GND2</a></em>, which encodes the minor isoform of 6-phosphogluconate dehydrogenase. Additionally, a few losses involve genes important for stress responses or environmental interactions such as <em>S. bayanus <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002831">CAD1</a></em>, which encodes an AP-1-like basic leucine zipper (bZIP) transcription factor involved in stress responses and iron metabolism.</p></div><div id="sec-17" class="subsection"><h4>Widespread losses:</h4><p id="p-54">Many differences among species that initially appeared to be lineage-specific gains were in fact losses of ancestral genes in multiple lineages. For example, it was clear from the presence of a syntenic homolog in <em>Naumovozyma castellii</em> (syn. <em>Saccharomyces castellii</em>) and other yeast species that the budding-yeast <em>Dicer</em> homolog in <em>S. bayanus</em> (Anc_8.880) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Drinnenberg <em>et al.</em> 2009</a>) has been lost in the other <em>sensu stricto</em> species, and not gained on the <em>S. bayanus</em> lineage. The same conclusion applied to several genes with sequence similarity to a YGOB-HMM but for which we had no functional data (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The rapidly evolving gene <em>Sbay_15.267</em> also fell into this category; orthologs were found in <em>Candida glabrata</em> (<em>Nakaseomyces</em> clade) <em>N. castellii</em>, and <em>S. bayanus</em> but not other sequenced species. The average dN/dS value was estimated as 0.33, but only about half of the codons could be aligned in all three species, indicating that despite the conservation of an intact open reading frame it was likely one of the fastest-evolving genes in yeast (<a id="xref-ref-96-4" class="xref-bibr" href="#ref-96">Zill <em>et al.</em> 2010</a>). We also detected the multiple paralogs of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001809">SIR1</a></em> in <em>S. kudriavzevii</em> and <em>S. bayanus</em>, which function in transcriptional silencing in these species (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Gallagher <em>et al.</em> 2009</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460707/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460707/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4â</span> <span class="caption-title">Genes not previously reported in the <em>Saccharomyces sensu stricto</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h4>Duplicate gene losses:</h4><p id="p-57">As a whole-genome duplication (WGD) event occurred in the ancestry of the <em>Saccharomyces sensu stricto</em> yeasts, we paid special attention to detecting duplicate pairs originating from this event. In total, our automated procedure detected 1044â1084 WGD duplicate genes (522â542 pairs) in each species (see <em>Materials and Methods</em>). 98% of those predicted in <em>S. cerevisiae</em> agreed with those listed in the Yeast Gene Order Browser (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Byrne and Wolfe 2005</a>). From these data, it was apparent that many of the potential gains and losses we detected were actually the result of differential loss of ancestrally duplicated genes from the yeast whole-genome duplication (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>), a process that contributes to reproductive isolation (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Lynch and Conery 2000</a>; <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Scannell <em>et al.</em> 2006</a>). For example, it was clear from the syntenic context that the presence of two copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004515">GAL80</a></em> in <em>S. bayanus</em> was not due to a recent duplication but to loss of one of the two ancestral copies derived from the WGD on the shared <em>S. cerevisiae/S. kudriavzevii</em> lineage and its retention on the <em>S. bayanus</em> lineage (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Hittinger <em>et al.</em> 2004</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Cliften <em>et al.</em> 2006</a>). Indeed, both <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004515">GAL80</a></em> WGD duplicates were also retained in other yeasts such as <em>N. castellii</em>. Interestingly, not all of the cases in <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a> can be explained by a single loss event. For example, the gene <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004482">YML020W</a></em> was present in two copies in <em>S. paradoxus</em>, indicating that it was present in duplicate until relatively recently. Therefore, one copy has become a pseudogene independently in each of the other <em>Saccharomyces sensu stricto</em> lineages. The same logic applied for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006013">SSU1</a></em>. These observations were in line with previous predictions (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Scannell <em>et al.</em> 2007b</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460721/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460721/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5â</span> <span class="caption-title">Loss of duplicate genes from the ancient whole-genome duplication in the <em>Saccharomyces sensu stricto</em> clade</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h4>Gene gains:</h4><p id="p-59">Although gene gains are quite rare in hemiascomycetes (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Hall and Dietrich 2007</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Gordon <em>et al.</em> 2009</a>), we found three candidates for horizontal gene transfer events. For example, <em>Smik_18.9</em> is 849 bp long and has codon usage typical of other <em>S. mikatae</em> genes but has no detectable homology to any gene in SGD. On closer inspection, it shows weak sequence similarity to the YGOB-HMM Anc_8.241 (homologous to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001252">CRG1</a></em>/<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001252">YHR209W</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000977">TMT1</a></em>/<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000977">YER175C</a></em>) and is clearly related to the <em>Lachancea thermotolerans</em> gene, KLTH0F00110. Both KLTH0F00110 and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001252">CRG1</a></em> are annotated as <em>S</em>-adenosylmethionine-dependent methyltransferases, indicating that <em>Smik_18.9</em> is a real gene that likely contributes to species-specific biology. Of the two other possible horizontal gene transfers in <em>S. mikatae</em>, one, CGSSp3BS71_00010 (an uncharacterized protein from <em>Streptococcus pneumoniae</em>), was apparently also present in <em>S. paradoxus</em>. These bacterial sequences were not merely contaminants introduced during Illumina library preparation, as we found identical sequences using BLAST in the Sanger shotgun reads. However, as there was only a single hit in NCBI, we could not construct gene trees to test whether it was a true horizontal transfer. Similarly, the <em>S. mikatae</em> gene <em>Smik_10.15</em> was a candidate horizontal gene transfer, because it displayed strong sequence similarity to a gene with an <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000811">NTF2</a> domain from the bacterium <em>Ralstonia solanacearum</em> (E-value of 6E-18), but no fungal homolog was detected. However, because <em>Smik_10.15</em> could not be placed within a specific bacterial clade (data not shown), we have not yet confirmed this as a horizontal transfer event.</p><p id="p-60">Interestingly, our analysis led to the discovery of a novel <em>S. cerevisiae</em> gene homologous to Anc_7.495. This gene is conserved across all <em>Saccharomyces sensu stricto</em> (as well as more diverged) yeasts but was previously not observed, presumably due to its short length (237 bp; <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). A dN/dS ratio of 0.29 confirms that this gene is subject to codon-level evolutionary constraint within the <em>Saccharomyces sensu stricto</em> and, based on the established naming convention, we propose that it be named <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003868">YJR107C</a>-A</em>.</p></div><div id="sec-20" class="subsection"><h4>tRNA Variation:</h4><p id="p-61">Using tRNA-ScanSE, we identified all 275 SGD-annotated tRNA genes in the <em>S. cerevisiae</em> genome with no false positives. Given the reliability of this procedure, we were surprised to observe that the number of Ser:AGA tRNA genes varied from 8 in <em>S. kudriavzevii</em> to 14 in <em>S. bayanus</em> (<a href="http://www.g3journal.org/content/vol1/issue1/images/data/11/DC1/TableS3.xls">Table S3</a>). Based on synteny, we estimate that there were 9â10 copies in the ancestor of all the species presented here and that whereas <em>S. kudriavzevii</em> has sustained a net loss of Ser:AGA tRNA genes, <em>S. bayanus</em> and <em>S. paradoxus</em> have gained copies. Variation in Ser:AGA tRNA copy number was not compensated by variation in the copy number of other serine tRNA genes (<a href="http://www.g3journal.org/content/vol1/issue1/images/data/11/DC1/TableS3.xls">Table S3</a>).</p></div></div><div id="sec-21" class="subsection"><h3>Genetically tractable strains for <em>S. mikatae</em>, <em>S. kudriavzevii</em>, and <em>S. bayanus</em></h3><p id="p-62">Functional tests of the genetic rewiring and other hypotheses presented above would require genetically tractable and marked strains from across the <em>Saccharomyces</em> genus. The four strains whose genomes we sequenced (IFO 1815<sup>T</sup> and derivatives of IFO 1802<sup>T</sup>, ZP 591, and CBS 7001) were originally prototrophic and homothallic. To enable genetic experiments to be conducted with similar ease to experiments in <em>S. cerevisiae</em>, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">HO</a></em> gene was inactivated and auxotrophic markers were introduced into the reference or type strains for <em>S. mikatae</em>, <em>S. kudriavzevii</em>, and <em>S. bayanus</em>. The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3â</a>0</em> strains are of particular utility because they enable the two-step procedure necessary to introduce precise changes in individual nucleotides (<a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Storici <em>et al.</em> 2001</a>). Although some of these strains have been described previously (<a id="xref-ref-44-6" class="xref-bibr" href="#ref-44">Hittinger <em>et al.</em> 2010</a>; <a id="xref-ref-96-5" class="xref-bibr" href="#ref-96">Zill <em>et al.</em> 2010</a>; <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Gallagher <em>et al.</em> 2009</a>), for the communityâs benefit, we briefly summarize below how each set of strains was generated (see <em>Materials and Methods</em>). A convenient collection of the most useful strains can be obtained from a single central repository (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>). Some heterothallic, marked <em>S. paradoxus</em> strains are already available (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cubillos <em>et al.</em> 2009</a>). Collectively, laboratory-ready strains are now available for genetic experiments in every <em>Saccharomyces</em> species whose genome sequence has been published.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460680/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460680/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6â</span> <span class="caption-title">Construction of heterothallic haploid strains with auxotrophic markers for <em>S. mikatae</em>, S. <em>kudriavzevii</em> and <em>S. bayanus</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h3>Characteristics of the <em>Saccharomyces sensu stricto</em> species that distinguished them from <em>S. cerevisiae</em> laboratory strains</h3><p id="p-64">Nonmodel species can often pose unanticipated challenges when brought into the laboratory. However, they also offer the great benefit of phenotypic diversity. To highlight some potentially useful characteristics of these newly laboratory-adapted yeast species described above and to alert investigators to potential practical problems, we offer several anecdotal observations culled from our collective experiences working with these nonmodel yeasts. We note that these comparisons are made to standard <em>S. cerevisiae</em> laboratory strains such as S288C. Hence some of the trait differences described below almost certainly resulted from the selections imposed on <em>S. cerevisiae</em> strains during laboratory adaptation, as others have noted (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Liu <em>et al.</em> 1996</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gaisne <em>et al.</em> 1999</a>).</p><p id="p-65">As with most wild strains, <em>S. paradoxus</em>, <em>S. mikatae</em>, <em>S. kudriavzevii</em>, and <em>S. bayanus</em> diploids showed very high levels of sporulation (approaching 100%) when placed on standard potassium acetate medium. These <em>Saccharomyces sensu stricto</em> strains also sporulated with striking efficiency (25â50%) after about 1 week on YPD plates stored at room temperature. These strains also sporulated on YPD at 4Â°C over a somewhat longer period of time. The tetrads of each of these species were of a greater size range than those of <em>S. cerevisiae</em>, but were smaller on average. As previously noted, these species preferred to grow at 18â23Â°C, and did not grow well at 30Â°C (<a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Sampaio and Goncalves 2008</a>). Indeed, they appeared more sensitive to heat shock than <em>S. cerevisiae</em>, but tolerated cold and freeze-thaw cycles better than <em>S. cerevisiae</em> (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Kvitek <em>et al.</em> 2008</a>).</p><p id="p-66">In liquid culture, flocculence was readily apparent in <em>S. paradoxus</em> and one of the <em>S. kudriavzevii</em> strains (IFO 1802<sup>T</sup> and derivatives). In fact, IFO 1802<sup>T</sup> was so flocculent that in overnight liquid culture it grew into spherical, 2-3mm pellets. Flocculence was less pronounced in <em>S. mikatae</em>, <em>S. bayanus</em>, and <em>S. kudriavzevii</em> strain ZP 591, with haploids of these strains more closely resembling the mild clumpiness of <em>S. cerevisiae</em> vineyard strain RM11 (R. Brem, personal communication). Interestingly, flocculence in <em>S. paradoxus</em> and <em>S. kudriavzevii</em> IFO 1802<sup>T</sup> appeared to be regulated by mating type. In both species, <em>MAT</em><strong>a</strong>/Î± diploids were less flocculent than both <em>MAT</em><strong>a</strong> and <em>MAT</em>Î± haploids. This regulation was likely due to mating-type control, rather than diploidy <em>per se</em>, as haploid <em>sirÎ</em> mutants (which express <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029214">HML</a></em>Î± and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029655">HMR</a></em><strong>a</strong>) of both species showed reduced flocculence (O. Zill, unpublished data). Cells of all four species displayed obvious differences in size and shape between haploids (small and round, often growing in clusters) and diploids (larger and ovoid, with polar budding). Diploids often appeared bulb-shaped, with a rounded apical tip and a flat base defined by the cellâs site of budding from its mother.</p><p id="p-67">All four species propagated <em>S. cerevisiae CEN</em>/<em>ARS</em> and 2Î¼ plasmids well enough to conduct complementation experiments (<a id="xref-ref-30-11" class="xref-bibr" href="#ref-30">Gallagher <em>et al.</em> 2009</a>). However, in <em>S. kudriavzevii</em> and <em>S. bayanus CEN</em>/<em>ARS</em> vectors segregated with lower fidelity than in <em>S. cerevisiae</em>, which was likely due to divergence in the <em>CEN</em> element (<a href="http://www.g3journal.org/cgi/data/1/1/11/DC1/2">Figure S1</a>; C. T. Hittinger and J. Gallagher, unpublished data). Notably, Japanese <em>S. kudriavzevii</em> (IFO 1802<sup>T</sup>) lack functional galactose metabolic and regulatory genes, while Portuguese <em>S. kudriavzevii</em> (ZP 591) are Gal<sup>+</sup> (<a id="xref-ref-44-16" class="xref-bibr" href="#ref-44">Hittinger <em>et al.</em> 2010</a>). Thus, the standard <em>GAL</em> induction plasmids would only work in the Portuguese strains, in which there was still a delayed response to galactose (<a id="xref-ref-44-17" class="xref-bibr" href="#ref-44">Hittinger <em>et al.</em> 2010</a>). <em>S. kudriavzevii</em> was originally reported to utilize the fructose-based complex carbohydrate inulin (<a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Naumov <em>et al.</em> 2000</a>). However, others and we were unable to replicate this result (C. T. Hittinger and Gregory I. Lang (Princeton University), unpublished data).</p></div></div><div class="section discussion" id="sec-23"><h2 class="">Discussion</h2><div id="sec-24" class="subsection"><h3>High-quality Saccharomyces genome assemblies for evolutionary analyses</h3><p id="p-68">A vibrant community of geneticists, genomicists, and computational biologists has made <em>S. cerevisiae</em> into a model species whose genome is arguably the best described, most easily manipulated, and best understood at all functional levels. Here we have provided a set of genetically tractable laboratory strains and vastly improved genome sequences that make <em>Saccharomyces sensu stricto</em> a model genus for evolutionary and comparative analyses and experiments.</p><p id="p-69">Deep paired-end Illumina sequencing allowed us to determine nearly complete genome sequences of <em>S. mikatae</em>, <em>S. kudriavzevii</em>, and <em>S. bayanus</em>, and to assign over 96% of base pairs to specific chromosomal locations. The closure of most gaps and the creation of ultra-scaffolds allowed us to provide a user-friendly genome browser (available through <a href="http://www.SaccharomycesSensuStricto.org">http://www.SaccharomycesSensuStricto.org</a>) for each species that will facilitate rapid experimental design, visualization of data, and further analyses. The ultra-scaffolds should be of particular value to genetic mapping studies. Draft genome sequences of <em>S. arboricolus</em> and <em>S. bayanus</em> var. <em>bayanus</em> (G. Liti, E. Louis, and C. Nieduszynski, personal communication; Libkind, Hittinger <em>et al.</em>, unpublished data) are also available, completing the catalog of known species-level diversity for the <em>Saccharomyces</em> genus (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>).</p><p id="p-70">Among model organisms, the genome of <em>S. cerevisiae</em> is uniquely well described both in terms of its functional elements and the relationships among those elements. The new assemblies and genetic tools presented here permit the same level of knowledge to be attained in its con-generic species. However, they also open the door to understanding how functions and interactions change over time by studying the same (orthologous) genes in multiple species. Such an evolutionary approach is becoming ever more common and has proven powerful even when applied to pathways with a long history of study in <em>S. cerevisiae</em> (<a id="xref-ref-44-18" class="xref-bibr" href="#ref-44">Hittinger <em>et al.</em> 2010</a>; <a id="xref-ref-96-7" class="xref-bibr" href="#ref-96">Zill <em>et al.</em> 2010</a>; <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Lee <em>et al.</em> 2008</a>). To facilitate such studies, we have annotated 5261 sets of genes that are orthologous among all five species (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>, <a href="http://www.g3journal.org/cgi/data/1/1/11/DC1/2">Table S1</a>). Thus, the vast majority of <em>Saccharomyces sensu stricto</em> genes are now available for systematic comparative and evolutionary study.</p></div><div id="sec-25" class="subsection"><h3>The susceptibility of comparative genomics to errors and missing data</h3><p id="p-71">The annotation of nearly complete genomes for five <em>Saccharomyces</em> species approximately doubled the number of orthologous gene sets available when compared with the sequences available in 2003 (previously 2742 orthogroups). This comparison provides an important reminder of the relationship between assembly completeness, annotation accuracy, and the downstream comparative analyses that rely on complete datasets (<em>e.g.</em>, phylogenetics). For example, even if 95% of genes were present and correctly annotated in each assembly, we would only expect 77% (0.95<sup>5</sup>) of genes to be present and correctly assigned to sets of orthologous genes in all species (assuming assembly biases are uncorrelated). Indeed, with five species, genome annotations that were 80% complete and accurate would yield full ortholog sets for just 33% of genes. To obtain orthogroups for 90% of genes would require an average per-genome completeness and accuracy of 98%. Thus, relative to single-genome studies, comparative studies are disproportionately sensitive to missing data and to the quality of the underlying annotations. Because this problem scales exponentially with the number of species, it will become drastically more severe as more species are considered. Therefore, designing procedures and analyses that are robust to missing data must be a key component and priority of future large-scale comparative genome sequencing projects.</p></div><div id="sec-26" class="subsection"><h3>Ancient whole genome duplication still impacts modern yeast evolution</h3><p id="p-72">Although we identified orthologs across all five species for most genes, we came across many examples of genes that had been lost in one or more lineages. For example, the losses of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003319">PDC6</a></em> on the <em>S. mikatae</em> lineage and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002831">CAD1</a></em> on the <em>S. bayanus</em> lineage immediately suggest potential species-specific biology. Interestingly, many of these losses appeared to involve members of duplicate pairs derived from the yeast whole-genome duplication (<a id="xref-ref-91-2" class="xref-bibr" href="#ref-91">Wolfe and Shields 1997</a>). The period after the WGD was characterized by rapid protein evolution (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Kellis <em>et al.</em> 2004</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Scannell and Wolfe 2008</a>) and gene loss (<a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">Scannell <em>et al.</em> 2006</a>), but our data suggest that duplicate genes continue to experience an elevated rate of loss even 100 million years after the WGD event. Though initially surprising, this conclusion is in line with previous analyses predicting that a small fraction of redundant duplicates remained to be resolved (<a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Scannell <em>et al.</em> 2007b</a>). Further, our observations of multiple orthologous losses (<em>e.g.</em>, Anc_5.554, <a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>) are consistent with the proposal that if one copy is capable of supplying all the required functions, then the second (âminorâ) copy will be convergently lost in all lineages (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Scannell and Wolfe 2008</a>). This idea is also supported by the loss of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004884">URA10</a></em> rather than <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004574">URA5</a></em> from <em>S. mikatae</em>. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004574">URA5</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004884">URA10</a></em> encode phosphoribosyltransferase genes that catalyze the fifth step in the pathway of <em>de novo</em> synthesis of pyrimidine ribonucleotides, but in <em>S. cerevisiae <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004884">URA10</a></em> supplies less than 20% of the activity and is conditionally expressed. Taken together, these observations point to the fascinating conclusion that the consequences of the WGD are still felt by modern yeast and contribute significantly to genomic and potentially phenotypic differences among <em>S. cerevisiae</em> and its con-generic species. Moreover, a specific line of research that emerges from this observation is to compare the biological functions of WGD duplicates that differ in copy number among these five yeasts using the strains that we have generated.</p><p id="p-73">In addition to lineage-specific losses, we also identified a number of candidates for lineage-specific gains, including two possible horizontal gene transfers from bacteria. These genes differentiate the <em>Saccharomyces sensu stricto</em> yeasts from one another (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>) and thus may play important roles in ecological specialization. Interestingly, among the putative gene gains we identified, was the discovery of a novel gene in <em>S. cerevisiae</em> that resides between <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003868">YJR107W</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003869">YJR108W</a></em> (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>). The biological functions of this rapidly evolving gene are unknown but its deep conservation outside the <em>Saccharomyces sensu stricto</em> clade leaves little doubt that it has a function. The discovery of a novel gene in the well-studied <em>S. cerevisiae</em> genome validated our goal of producing high-quality annotations for five <em>sensu stricto</em> yeasts.</p></div></div><div class="section conclusions" id="sec-27"><h2 class="">Conclusions</h2><p id="p-74">Along with creating stably marked haploid strains, we have ported routine techniques for manipulating <em>S. cerevisiae</em> genetically to the other <em>Saccharomyces</em> species. These include the powerful tools of targeted-gene knockouts, plasmid-driven expression, and altering single nucleotides within genomes. Species within the <em>Saccharomyces sensu stricto</em> genus also readily hybridize (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Masneuf <em>et al.</em> 1998</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Greig 2009</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Martin <em>et al.</em> 2009</a>). The complementary markers and mating types in the collection make these experiments especially convenient. Interspecies hybrids can be used in complementation tests to identify mutated genes in species closely related to a model organism (Zill <em>et al.</em>, in preparation), and to study the evolution of genetic regulatory circuits (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bullard <em>et al.</em> 2010</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Gasch <em>et al.</em> 2004</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Guan <em>et al.</em> 2007</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Tirosh <em>et al.</em> 2009</a>). This expansion of genetically tractable species opens the <em>Saccharomyces sensu stricto</em> genus to comparative analysis, and provides molecular biologists with an unrivaled set of tools to explore this model genus.</p><p id="p-75">With near-complete genome sequences, geneticists now have essentially complete experimental access to the genomes of each species. The candidate changes in gene content and selection pressures we found within each <em>Saccharomyces sensu stricto</em> species present excellent opportunities to study species-specific biology, and to use these genetic differences to learn how genetic networks have been rewired during the evolution of this genus. At the same time, the large set of orthologs we have identified provides a strong foundation for comparative genetic studies, and should lead to a wealth of discoveries that are refractory to sequence-based analyses. Together, these new genomic resources and universal genetic techniques provide an infrastructure for an unprecedented integration of evolutionary and experimental biology, enabled by the Saccharomyces model genus and the awesome power of yeast genetics.</p></div><div class="section" id="sec-28"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-76">We thank Ed Louis for providing strains IFO 1815<sup>T</sup>, IFO 1802<sup>T</sup>, and CBS 7001, and for invaluable discussions. We thank Paula GonÃ§alves and JosÃ© Paulo Sampaio for strain ZP 591. We thank Leath Tonkin at the Vincent Coates Genome Center UC-Berkeley and Jim Dover at the University of Colorado School of Medicine for Illumina sequencing. We thank Leonidas Salichos for providing sequence-analysis computer code. We thank Skylar Thompson for providing technical support with the browser. We thank Logan Fink for help with testing <em>S. bayanus</em> scaffold ordering. We thank Ken Wolfe for critical reading of the manuscript. This research was supported by an NSF predoctoral fellowship and NIH Genomics training grants NIH-T32-HG00047 (to O.Z.), NSF-DEB-0844968 (to A.R.), NIH-P41-RR011823 (to M.J.D.), NIH-NHGRI-HG002779 (to M.B.E.), NIH-NIGMS-GM31105 (to J.R.), NIH-NIGMS-GM032540 (to M.J.), and the James S. McDonnell Foundation (to C.T.H., M.J.). C.T.H. is the Maclyn McCarty Fellow of the Helen Hay Whitney Foundation, M.J.D. is a Rita Allen Scholar, and M.B.E. is an Investigator of the Howard Hughes Medical Institute.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-13"><p id="p-77"><em>Communicating editor: Brenda J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-04-18"><span class="received-label">Received </span>April 18, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-05-01"><span class="accepted-label">Accepted </span>May 1, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2011 Scannell <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-2">This is an open access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.1" data-doi="10.1371/journal.pcbi.1000257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Airoldi</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huttenhower</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caudy</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Predicting cellular growth from gene expression signatures.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DAiroldi%26rft.auinit1%253DE.%2BM.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253De1000257%26rft.epage%253De1000257%26rft.atitle%253DPredicting%2Bcellular%2Bgrowth%2Bfrom%2Bgene%2Bexpression%2Bsignatures.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000257%26rft_id%253Dinfo%253Apmid%252F19119411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000257&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19119411&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.2" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>â<span class="cit-lpage">3402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.17.3389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9254694&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU79300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.3" data-doi="10.1038/nature08800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Atwell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjalmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willems</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Genome-wide association study of 107 phenotypes in Arabidopsis thaliana inbred lines.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">465</span>: <span class="cit-fpage">627</span>â<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAtwell%26rft.auinit1%253DS.%26rft.volume%253D465%26rft.issue%253D7298%26rft.spage%253D627%26rft.epage%253D631%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2B107%2Bphenotypes%2Bin%2BArabidopsis%2Bthaliana%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08800%26rft_id%253Dinfo%253Apmid%252F20336072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20336072&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278249000044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.4" data-doi="10.1371/journal.pcbi.1000392"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradley</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smoot</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juvekar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Do</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Fast statistical alignment.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DBradley%26rft.auinit1%253DR.%2BK.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253De1000392%26rft.epage%253De1000392%26rft.atitle%253DFast%2Bstatistical%2Balignment.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000392%26rft_id%253Dinfo%253Apmid%252F19478997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000392&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19478997&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.5" data-doi="10.1073/pnas.0912959107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bullard</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mostovoy</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudoit</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brem</span>  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Polygenic and directional regulatory evolution across pathways in Saccharomyces.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">5058</span>â<span class="cit-lpage">5063</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0912959107%26rft_id%253Dinfo%253Apmid%252F20194736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA3LzExLzUwNTgiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzEvMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.6" data-doi="10.1038/nature08064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakthikumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Evolution of pathogenicity and sexual reproduction in eight Candida genomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">459</span>: <span class="cit-fpage">657</span>â<span class="cit-lpage">662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DButler%26rft.auinit1%253DG.%26rft.volume%253D459%26rft.issue%253D7247%26rft.spage%253D657%26rft.epage%253D662%26rft.atitle%253DEvolution%2Bof%2Bpathogenicity%2Band%2Bsexual%2Breproduction%2Bin%2Beight%2BCandida%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08064%26rft_id%253Dinfo%253Apmid%252F19465905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19465905&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266608600034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.7" data-doi="10.1101/gr.3672305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">The Yeast Gene Order Browser: combining curated homology and syntenic context reveals gene fate in polyploid species.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1456</span>â<span class="cit-lpage">1461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3672305%26rft_id%253Dinfo%253Apmid%252F16169922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMC8xNDU2IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8xLzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.8" data-doi="10.1093/nar/gkj041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Visualizing syntenic relationships among the hemiascomycetes with the Yeast Gene Order Browser.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">D452</span>â<span class="cit-lpage">D455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DByrne%26rft.auinit1%253DK.%2BP.%26rft.volume%253D34%26rft.issue%253Dsuppl_1%26rft.spage%253DD452%26rft.epage%253DD455%26rft.atitle%253DVisualizing%2Bsyntenic%2Brelationships%2Bamong%2Bthe%2Bhemiascomycetes%2Bwith%2Bthe%2BYeast%2BGene%2BOrder%2BBrowser%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkj041%26rft_id%253Dinfo%253Apmid%252F16381909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkj041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16381909&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239307700099&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.9" data-doi="10.1093/bioinformatics/17.12.1093"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chou</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">M. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">DNA sequence quality trimming and vector removal.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1093</span>â<span class="cit-lpage">1104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DChou%26rft.auinit1%253DH.-H.%26rft.volume%253D17%26rft.issue%253D12%26rft.spage%253D1093%26rft.epage%253D1104%26rft.atitle%253DDNA%2Bsequence%2Bquality%2Btrimming%2Band%2Bvector%2Bremoval%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.12.1093%26rft_id%253Dinfo%253Apmid%252F11751217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/17.12.1093&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11751217&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173050500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.10" data-doi="10.1371/journal.pbio.1000432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chou</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hung</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leu</span>  <span class="cit-name-given-names">J. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Multiple molecular mechanisms cause reproductive isolation between three yeast species.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1000432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DChou%26rft.auinit1%253DJ.%2BY.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253De1000432%26rft.epage%253De1000432%26rft.atitle%253DMultiple%2Bmolecular%2Bmechanisms%2Bcause%2Breproductive%2Bisolation%2Bbetween%2Bthree%2Byeast%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000432%26rft_id%253Dinfo%253Apmid%252F20652018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000432&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20652018&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.11" data-doi="10.1002/bies.200900162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chou</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leu</span>  <span class="cit-name-given-names">J. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Speciation through cytonuclear incompatibility: insights from yeast and implications for higher eukaryotes.</span> <abbr class="cit-jnl-abbrev">Bioessays</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">401</span>â<span class="cit-lpage">411</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DChou%26rft.auinit1%253DJ.%2BY.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D401%26rft.epage%253D411%26rft.atitle%253DSpeciation%2Bthrough%2Bcytonuclear%2Bincompatibility%253A%2Binsights%2Bfrom%2Byeast%2Band%2Bimplications%2Bfor%2Bhigher%2Beukaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.200900162%26rft_id%253Dinfo%253Apmid%252F20414898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.200900162&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20414898&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277577200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.12" data-doi="10.1038/nature06341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Evolution of genes and genomes on the Drosophila phylogeny.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">450</span>: <span class="cit-fpage">203</span>â<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DClark%26rft.auinit1%253DA.%2BG.%26rft.volume%253D450%26rft.issue%253D7167%26rft.spage%253D203%26rft.epage%253D218%26rft.atitle%253DEvolution%2Bof%2Bgenes%2Band%2Bgenomes%2Bon%2Bthe%2BDrosophila%2Bphylogeny.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06341%26rft_id%253Dinfo%253Apmid%252F17994087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17994087&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250746200042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.13" data-doi="10.1126/science.1084337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cliften</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sudarsanam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desikan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Finding functional features in Saccharomyces genomes by phylogenetic footprinting.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">301</span>: <span class="cit-fpage">71</span>â<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1084337%26rft_id%253Dinfo%253Apmid%252F12775844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMDEvNTYyOS83MSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzEvMS8xMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cliften</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">After the duplication: gene loss and adaptation in Saccharomyces genomes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">863</span>â<span class="cit-lpage">872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D172%26rft.spage%253D863%26rft_id%253Dinfo%253Apmid%252F16322519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16322519&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236178800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.15" data-doi="10.1111/j.1567-1364.2009.00583.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Generation of a large set of genetically tractable haploid and diploid Saccharomyces strains.</span> <abbr class="cit-jnl-abbrev">FEM. Yeast Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1217</span>â<span class="cit-lpage">1225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEM.%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2009.00583.x%26rft_id%253Dinfo%253Apmid%252F19840116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1567-1364.2009.00583.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19840116&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271264400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.16" data-doi="10.1126/science.1095781"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dietrich</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voegeli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brachat</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lerch</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gates</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">The Ashbya gossypii genome as a tool for mapping the ancient Saccharomyces cerevisiae genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">304</span>: <span class="cit-fpage">304</span>â<span class="cit-lpage">307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1095781%26rft_id%253Dinfo%253Apmid%252F15001715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDQvNTY2OC8zMDQiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzEvMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.17" data-doi="10.1126/science.1176945"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drinnenberg</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinberg</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mower</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">RNAi in budding yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">544</span>â<span class="cit-lpage">550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1176945%26rft_id%253Dinfo%253Apmid%252F19745116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjYvNTk1Mi81NDQiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzEvMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.18" data-doi="10.1371/journal.pbio.0040088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Relaxed phylogenetics and dating with confidence.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDrummond%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253De88%26rft.epage%253De88%26rft.atitle%253DRelaxed%2Bphylogenetics%2Band%2Bdating%2Bwith%2Bconfidence.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040088%26rft_id%253Dinfo%253Apmid%252F16683862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16683862&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.19" data-doi="10.1186/1471-2148-7-214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">BEAST: Bayesian evolutionary analysis by sampling trees.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DDrummond%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D7%26rft.spage%253D214%26rft.epage%253D214%26rft.atitle%253DBEAST%253A%2BBayesian%2Bevolutionary%2Banalysis%2Bby%2Bsampling%2Btrees.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-7-214%26rft_id%253Dinfo%253Apmid%252F17996036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-7-214&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17996036&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.20" data-doi="10.1038/nrg2811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Yeast evolutionary genomics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">512</span>â<span class="cit-lpage">524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDujon%26rft.auinit1%253DB.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D512%26rft.epage%253D524%26rft.atitle%253DYeast%2Bevolutionary%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2811%26rft_id%253Dinfo%253Apmid%252F20559329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20559329&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.21" data-doi="10.1016/j.tig.2006.05.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Yeasts illustrate the molecular mechanisms of eukaryotic genome evolution.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">375</span>â<span class="cit-lpage">387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DDujon%26rft.auinit1%253DB.%26rft.volume%253D22%26rft.issue%253D7%26rft.spage%253D375%26rft.epage%253D387%26rft.atitle%253DYeasts%2Billustrate%2Bthe%2Bmolecular%2Bmechanisms%2Bof%2Beukaryotic%2Bgenome%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2006.05.007%26rft_id%253Dinfo%253Apmid%252F16730849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2006.05.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16730849&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239684500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.22" data-doi="10.1038/nature02579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durrens</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casaregola</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Genome evolution in yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">430</span>: <span class="cit-fpage">35</span>â<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDujon%26rft.auinit1%253DB.%26rft.volume%253D430%26rft.issue%253D6995%26rft.spage%253D35%26rft.epage%253D44%26rft.atitle%253DGenome%2Bevolution%2Bin%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02579%26rft_id%253Dinfo%253Apmid%252F15229592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15229592&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222356800036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.23" data-doi="10.1142/9781848165632_0019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A new generation of homology search tools based on probabilistic inference.</span> <abbr class="cit-jnl-abbrev">Genome Inform</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">205</span>â<span class="cit-lpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BInform%26rft.volume%253D23%26rft.spage%253D205%26rft_id%253Dinfo%253Adoi%252F10.1142%252F9781848165632_0019%26rft_id%253Dinfo%253Apmid%252F20180275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1142/9781848165632_0019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20180275&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.24" data-doi="10.1093/nar/gkp917"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engel</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balakrishnan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binkley</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christie</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Saccharomyces Genome Database provides mutant phenotype data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">D433</span>â<span class="cit-lpage">D436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp917%26rft_id%253Dinfo%253Apmid%252F19906697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkp917&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19906697&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276399100068&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.25" data-doi="10.1101/gr.8.3.186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Base-calling of automated sequencer traces using phred. II. Error probabilities.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">186</span>â<span class="cit-lpage">194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.8.3.186%26rft_id%253Dinfo%253Apmid%252F9521922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/gr.8.3.186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9521922&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.26" data-doi="10.1534/genetics.105.042283"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benavides</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Hypervariable noncoding sequences in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">1575</span>â<span class="cit-lpage">1587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D170%26rft.spage%253D1575%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.042283%26rft_id%253Dinfo%253Apmid%252F15956668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.105.042283&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15956668&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232033300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.27" data-doi="10.1073/pnas.0601713103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fidalgo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrales</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ibeas</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jimenez</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Adaptive evolution by mutations in the FLO11 gene.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">11228</span>â<span class="cit-lpage">11233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0601713103%26rft_id%253Dinfo%253Apmid%252F16844788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzMwLzExMjI4IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8xLzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.28" data-doi="10.1038/35013058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">I. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Chromosomal evolution in Saccharomyces.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">405</span>: <span class="cit-fpage">451</span>â<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFischer%26rft.auinit1%253DG.%26rft.volume%253D405%26rft.issue%253D6785%26rft.spage%253D451%26rft.epage%253D454%26rft.atitle%253DChromosomal%2Bevolution%2Bin%2BSaccharomyces.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35013058%26rft_id%253Dinfo%253Apmid%252F10839539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35013058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10839539&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087212000047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.29" data-doi="10.1007/s002940050490"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaisne</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becam</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verdiere</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herbert</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">A 'natural' mutation in Saccharomyces cerevisiae strains derived from S288c affects the complex regulatory gene HAP1 (CYP1).</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">195</span>â<span class="cit-lpage">200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DGaisne%26rft.auinit1%253DM.%26rft.volume%253D36%26rft.issue%253D4%26rft.spage%253D195%26rft.epage%253D200%26rft.atitle%253DA%2B%2527natural%2527%2Bmutation%2Bin%2BSaccharomyces%2Bcerevisiae%2Bstrains%2Bderived%2Bfrom%2BS288c%2Baffects%2Bthe%2Bcomplex%2Bregulatory%2Bgene%2BHAP1%2B%2528CYP1%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002940050490%26rft_id%253Dinfo%253Apmid%252F10541856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s002940050490&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10541856&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083336500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.30" data-doi="10.1534/genetics.108.099663"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallagher</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babiarz</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teytelman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rine</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Elaboration, diversification and regulation of the Sir1 family of silencing proteins in Saccharomyces.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">1477</span>â<span class="cit-lpage">1491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D181%26rft.spage%253D1477%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.099663%26rft_id%253Dinfo%253Apmid%252F19171939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.108.099663&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19171939&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270213700026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.31" data-doi="10.1371/journal.pbio.0020398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gasch</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">D. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">H. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berardini</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Conservation and evolution of cis-regulatory systems in ascomycete fungi.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DGasch%26rft.auinit1%253DA.%2BP.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253De398%26rft.epage%253De398%26rft.atitle%253DConservation%2Band%2Bevolution%2Bof%2Bcis-regulatory%2Bsystems%2Bin%2Bascomycete%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0020398%26rft_id%253Dinfo%253Apmid%252F15534694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0020398&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15534694&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.32" data-doi="10.1126/science.1166426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Genetic interactions between transcription factors cause natural variation in yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">498</span>â<span class="cit-lpage">501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGerke%26rft.auinit1%253DJ.%26rft.volume%253D323%26rft.issue%253D5913%26rft.spage%253D498%26rft.epage%253D501%26rft.atitle%253DGenetic%2BInteractions%2BBetween%2BTranscription%2BFactors%2BCause%2BNatural%2BVariation%2Bin%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1166426%26rft_id%253Dinfo%253Apmid%252F19164747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjMvNTkxMy80OTgiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzEvMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.33" data-doi="10.1126/science.274.5287.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goffeau</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bussey</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Life with 6000 genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">274</span>(<span class="cit-issue">546</span>): <span class="cit-fpage">546</span>, <span class="cit-fpage">563</span>â<span class="cit-lpage">567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGoffeau%26rft.auinit1%253DA.%26rft.volume%253D274%26rft.issue%253D5287%26rft.spage%253D546%26rft.epage%253D567%26rft.atitle%253DLife%2Bwith%2B6000%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.274.5287.546%26rft_id%253Dinfo%253Apmid%252F8849441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvNTI4Ny81NDYiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzEvMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.34" data-doi="10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Three new dominant drug resistance cassettes for gene disruption in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1541</span>â<span class="cit-lpage">1553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DGoldstein%26rft.auinit1%253DA.%2BL.%26rft.volume%253D15%26rft.issue%253D14%26rft.spage%253D1541%26rft.epage%253D1553%26rft.atitle%253DThree%2Bnew%2Bdominant%2Bdrug%2Bresistance%2Bcassettes%2Bfor%2Bgene%2Bdisruption%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199910%252915%253A14%253C1541%253A%253AAID-YEA476%253E3.0.CO%253B2-K%26rft_id%253Dinfo%253Apmid%252F10514571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10514571&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083104100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.35" data-doi="10.1128/AEM.01867-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrio</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Querol</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Molecular characterization of new natural hybrids of Saccharomyces cerevisiae and S. kudriavzevii in brewing.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">2314</span>â<span class="cit-lpage">2320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.01867-07%26rft_id%253Dinfo%253Apmid%252F18296532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6Ijc0LzgvMjMxNCI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzEvMS8xMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.36" data-doi="10.1371/journal.pgen.1000485"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Additions, losses, and rearrangements on the evolutionary route from a reconstructed ancestor to the modern Saccharomyces cerevisiae genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGordon%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253De1000485%26rft.epage%253De1000485%26rft.atitle%253DAdditions%252C%2Blosses%252C%2Band%2Brearrangements%2Bon%2Bthe%2Bevolutionary%2Broute%2Bfrom%2Ba%2Breconstructed%2Bancestor%2Bto%2Bthe%2Bmodern%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000485%26rft_id%253Dinfo%253Apmid%252F19436716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000485&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19436716&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.37" data-doi="10.1038/hdy.2008.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greig</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Reproductive isolation in Saccharomyces.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">39</span>â<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DGreig%26rft.auinit1%253DD.%26rft.volume%253D102%26rft.issue%253D1%26rft.spage%253D39%26rft.epage%253D44%26rft.atitle%253DReproductive%2Bisolation%2Bin%2BSaccharomyces.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2008.73%26rft_id%253Dinfo%253Apmid%252F18648383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2008.73&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18648383&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261774400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.38" data-doi="10.1126/science.1076374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greig</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borts</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Travisano</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Hybrid speciation in experimental populations of yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">298</span>: <span class="cit-fpage">1773</span>â<span class="cit-lpage">1775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGreig%26rft.auinit1%253DD.%26rft.volume%253D298%26rft.issue%253D5599%26rft.spage%253D1773%26rft.epage%253D1775%26rft.atitle%253DHybrid%2BSpeciation%2Bin%2BExperimental%2BPopulations%2Bof%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1076374%26rft_id%253Dinfo%253Apmid%252F12459586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTgvNTU5OS8xNzczIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8xLzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.39" data-doi="10.1046/j.1420-9101.2003.00546.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greig</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Travisano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borts</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">A role for the mismatch repair system during incipient speciation in Saccharomyces.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">429</span>â<span class="cit-lpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DGreig%26rft.auinit1%253DD.%26rft.volume%253D16%26rft.issue%253D3%26rft.spage%253D429%26rft.epage%253D437%26rft.atitle%253DA%2Brole%2Bfor%2Bthe%2Bmismatch%2Brepair%2Bsystem%2Bduring%2Bincipient%2Bspeciation%2Bin%2BSaccharomyces.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1420-9101.2003.00546.x%26rft_id%253Dinfo%253Apmid%252F14635842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1420-9101.2003.00546.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14635842&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182322200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.40" data-doi="10.1371/journal.pcbi.1000698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caudy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troyanskaya</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Systematic planning of genome-scale experiments in poorly studied species.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000698</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DGuan%26rft.auinit1%253DY.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253De1000698%26rft.epage%253De1000698%26rft.atitle%253DSystematic%2Bplanning%2Bof%2Bgenome-scale%2Bexperiments%2Bin%2Bpoorly%2Bstudied%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000698%26rft_id%253Dinfo%253Apmid%252F20221257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20221257&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.41" data-doi="10.1534/genetics.106.064329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troyanskaya</span>  <span class="cit-name-given-names">O. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Functional analysis of gene duplications in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">933</span>â<span class="cit-lpage">943</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D175%26rft.spage%253D933%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.064329%26rft_id%253Dinfo%253Apmid%252F17151249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.106.064329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17151249&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244689600043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.42" data-doi="10.1093/nar/24.13.2519"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guldener</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heck</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fielder</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beinhauer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hegemann</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">A new efficient gene disruption cassette for repeated use in budding yeast.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2519</span>â<span class="cit-lpage">2524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F24.13.2519%26rft_id%253Dinfo%253Apmid%252F8692690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/24.13.2519&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8692690&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UX27000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.43" data-doi="10.1534/genetics.107.074963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dietrich</span>  <span class="cit-name-given-names">F. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">The reacquisition of biotin prototrophy in Saccharomyces cerevisiae involved horizontal gene transfer, gene duplication and gene clustering.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2293</span>â<span class="cit-lpage">2307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D177%26rft.spage%253D2293%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074963%26rft_id%253Dinfo%253Apmid%252F18073433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.107.074963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18073433&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251949800028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.44" data-doi="10.1038/nature08791"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hittinger</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goncalves</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sampaio</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dover</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Remarkably ancient balanced polymorphisms in a multi-locus gene network.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">54</span>â<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHittinger%26rft.auinit1%253DC.%2BT.%26rft.volume%253D464%26rft.issue%253D7285%26rft.spage%253D54%26rft.epage%253D58%26rft.atitle%253DRemarkably%2Bancient%2Bbalanced%2Bpolymorphisms%2Bin%2Ba%2Bmulti-locus%2Bgene%2Bnetwork.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08791%26rft_id%253Dinfo%253Apmid%252F20164837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08791&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20164837&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275117500032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.45" data-doi="10.1073/pnas.0404319101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hittinger</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Parallel inactivation of multiple GAL pathway genes and ecological diversification in yeasts.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">14144</span>â<span class="cit-lpage">14149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0404319101%26rft_id%253Dinfo%253Apmid%252F15381776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzM5LzE0MTQ0IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8xLzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.46" data-doi="10.1038/nature02424"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birren</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Proof and evolutionary analysis of ancient genome duplication in the yeast Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">428</span>: <span class="cit-fpage">617</span>â<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKellis%26rft.auinit1%253DM.%26rft.volume%253D428%26rft.issue%253D6983%26rft.spage%253D617%26rft.epage%253D624%26rft.atitle%253DProof%2Band%2Bevolutionary%2Banalysis%2Bof%2Bancient%2Bgenome%2Bduplication%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02424%26rft_id%253Dinfo%253Apmid%252F15004568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02424&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15004568&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220697200034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.47" data-doi="10.1038/nature01644"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endrizzi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birren</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Sequencing and comparison of yeast species to identify genes and regulatory elements.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">423</span>: <span class="cit-fpage">241</span>â<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKellis%26rft.auinit1%253DM.%26rft.volume%253D423%26rft.issue%253D6937%26rft.spage%253D241%26rft.epage%253D254%26rft.atitle%253DSequencing%2Band%2Bcomparison%2Bof%2Byeast%2Bspecies%2Bto%2Bidentify%2Bgenes%2Band%2Bregulatory%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01644%26rft_id%253Dinfo%253Apmid%252F12748633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12748633&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182853100033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.48" data-doi="10.1016/S1567-1356(03)00012-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurtzman</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robnett</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Phylogenetic relationships among yeasts of the 'Saccharomyces complex' determined from multigene sequence analyses.</span> <abbr class="cit-jnl-abbrev">FEM. Yeast Res.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">417</span>â<span class="cit-lpage">432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BResearch%26rft.stitle%253DFEMS%2BYeast%2BRes%26rft.aulast%253DKurtzman%26rft.auinit1%253DC.%2BP.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253D417%26rft.epage%253D432%26rft.atitle%253DPhylogenetic%2Brelationships%2Bamong%2Byeasts%2Bof%2Bthe%2B%2527Saccharomyces%2Bcomplex%2527%2Bdetermined%2Bfrom%2Bmultigene%2Bsequence%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1567-1356%252803%252900012-6%26rft_id%253Dinfo%253Apmid%252F12748053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1567-1356(03)00012-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12748053&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182999800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.49" data-doi="10.1371/journal.pgen.1000223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kvitek</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Will</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasch</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Variations in stress sensitivity and genomic expression in diverse S. cerevisiae isolates.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKvitek%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000223%26rft.epage%253De1000223%26rft.atitle%253DVariations%2Bin%2Bstress%2Bsensitivity%2Band%2Bgenomic%2Bexpression%2Bin%2Bdiverse%2BS.%2Bcerevisiae%2Bisolates.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000223%26rft_id%253Dinfo%253Apmid%252F18927628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18927628&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.50" data-doi="10.1016/j.cell.2008.10.047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chou</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Incompatibility of nuclear and mitochondrial genomes causes hybrid sterility between two yeast species.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">1065</span>â<span class="cit-lpage">1073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLee%26rft.auinit1%253DH.%2BY.%26rft.volume%253D135%26rft.issue%253D6%26rft.spage%253D1065%26rft.epage%253D1073%26rft.atitle%253DIncompatibility%2Bof%2Bnuclear%2Band%2Bmitochondrial%2Bgenomes%2Bcauses%2Bhybrid%2Bsterility%2Bbetween%2Btwo%2Byeast%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.10.047%26rft_id%253Dinfo%253Apmid%252F19070577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.10.047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19070577&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261642800022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.51" data-doi="10.1186/1471-2164-10-37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lefrancois</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Euskirchen</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auerbach</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozowsky</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Efficient yeast ChIP-Seq using multiplex short-read DNA sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLefrancois%26rft.auinit1%253DP.%26rft.volume%253D10%26rft.spage%253D37%26rft.epage%253D37%26rft.atitle%253DEfficient%2Byeast%2BChIP-Seq%2Busing%2Bmultiplex%2Bshort-read%2BDNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-37%26rft_id%253Dinfo%253Apmid%252F19159457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19159457&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.52" data-doi="10.1101/gr.097261.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">De novo assembly of human genomes with massively parallel short read sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">265</span>â<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.097261.109%26rft_id%253Dinfo%253Apmid%252F20019144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzIvMjY1IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8xLzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.53" data-doi="10.1534/genetics.106.062166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Sequence diversity, reproductive isolation and species concepts in Saccharomyces.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">839</span>â<span class="cit-lpage">850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D174%26rft.spage%253D839%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.062166%26rft_id%253Dinfo%253Apmid%252F16951060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.106.062166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16951060&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241798700024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.54" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Population genomics of domestic and wild yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">337</span>â<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styles</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Saccharomyces cerevisiae S288C has a mutation in FLO8, a gene required for filamentous growth.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">967</span>â<span class="cit-lpage">978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D144%26rft.spage%253D967%26rft_id%253Dinfo%253Apmid%252F8913742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8913742&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VP82600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.56" data-doi="10.1002/(SICI)1097-0061(199807)14:10&lt;953::AID-YEA293&gt;3.0.CO;2-U"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Longtine</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKenzie</span>  <span class="cit-name-given-names">A.</span>  <span class="cit-name-suffix">3rd</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demarini</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wach</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Additional modules for versatile and economical PCR-based gene deletion and modification in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">953</span>â<span class="cit-lpage">961</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DLongtine%26rft.auinit1%253DM.%2BS.%26rft.volume%253D14%26rft.issue%253D10%26rft.spage%253D953%26rft.epage%253D961%26rft.atitle%253DAdditional%2Bmodules%2Bfor%2Bversatile%2Band%2Beconomical%2BPCR-based%2Bgene%2Bdeletion%2Band%2Bmodification%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199807%252914%253A10%253C953%253A%253AAID-YEA293%253E3.0.CO%253B2-U%26rft_id%253Dinfo%253Apmid%252F9717241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199807)14:10&lt;953::AID-YEA293&gt;3.0.CO;2-U&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9717241&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075005100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.57" data-doi="10.1002/yea.320111604"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-article-title">The chromosome ends of Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1553</span>â<span class="cit-lpage">1573</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DLouis%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D11%26rft.issue%253D16%26rft.spage%253D1553%26rft.epage%253D1573%26rft.atitle%253DThe%2Bchromosome%2Bends%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320111604%26rft_id%253Dinfo%253Apmid%252F8720065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320111604&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8720065&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TQ24100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.58" data-doi="10.1093/nar/25.5.0955"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">955</span>â<span class="cit-lpage">964</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.5.0955%26rft_id%253Dinfo%253Apmid%252F9023104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.5.0955&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9023104&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.59" data-doi="10.1126/science.290.5494.1151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conery</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">The evolutionary fate and consequences of duplicate genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">290</span>: <span class="cit-fpage">1151</span>â<span class="cit-lpage">1155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D290%26rft.issue%253D5494%26rft.spage%253D1151%26rft.epage%253D1155%26rft.atitle%253DThe%2BEvolutionary%2BFate%2Band%2BConsequences%2Bof%2BDuplicate%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.290.5494.1151%26rft_id%253Dinfo%253Apmid%252F11073452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTAvNTQ5NC8xMTUxIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8xLzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masneuf</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groth</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piskur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubourdieu</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">New hybrids between Saccharomyces sensu stricto yeast species found among wine and cider production strains.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">3887</span>â<span class="cit-lpage">3892</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DMasneuf%26rft.auinit1%253DI.%26rft.volume%253D64%26rft.issue%253D10%26rft.spage%253D3887%26rft.epage%253D3892%26rft.atitle%253DNew%2BHybrids%2Bbetween%2BSaccharomyces%2BSensu%2BStricto%2BYeast%2BSpecies%2BFound%2Bamong%2BWine%2Band%2BCider%2BProduction%2BStrains%26rft_id%253Dinfo%253Apmid%252F9758815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI2NC8xMC8zODg3IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8xLzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.61" data-doi="10.1038/cr.2009.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">O. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeSevo</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">B. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koshland</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Telomere behavior in a hybrid yeast.</span> <abbr class="cit-jnl-abbrev">Cell Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">910</span>â<span class="cit-lpage">912</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bresearch%26rft.stitle%253DCell%2BRes%26rft.aulast%253DMartin%26rft.auinit1%253DO.%2BC.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D910%26rft.epage%253D912%26rft.atitle%253DTelomere%2Bbehavior%2Bin%2Ba%2Bhybrid%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2009.65%26rft_id%253Dinfo%253Apmid%252F19506581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2009.65&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19506581&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.62" data-doi="10.1093/dnares/dsp003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanamori</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Itoh</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kodama</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rainieri</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Genome sequence of the lager brewing yeast, an interspecies hybrid.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">115</span>â<span class="cit-lpage">129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsp003%26rft_id%253Dinfo%253Apmid%252F19261625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/dnares/dsp003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261625&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265770800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.63" data-doi="10.1099/00207713-50-5-1931"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naumov</span>  <span class="cit-name-given-names">G. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naumova</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">I. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Three new species in the Saccharomyces sensu stricto complex: Saccharomyces cariocanus, Saccharomyces kudriavzevii and Saccharomyces mikatae.</span> <abbr class="cit-jnl-abbrev">Int. J. Syst. Evol. Microbiol.</abbr> <span class="cit-vol">50</span>(<span class="cit-issue">Pt 5</span>): <span class="cit-fpage">1931</span>â<span class="cit-lpage">1942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BSystematic%2Band%2BEvolutionary%2BMicrobiology%26rft.stitle%253DInt.%2BJ.%2BSyst.%2BEvol.%2BMicrobiol.%26rft.aulast%253DNaumov%26rft.auinit1%253DG.%26rft.volume%253D50%26rft.issue%253D5%26rft.spage%253D1931%26rft.epage%253D1942%26rft.atitle%253DThree%2Bnew%2Bspecies%2Bin%2Bthe%2BSaccharomyces%2Bsensu%2Bstricto%2Bcomplex%253A%2BSaccharomyces%2Bcariocanus%252C%2BSaccharomyces%2Bkudriavzevii%2Band%2BSaccharomyces%2Bmikatae%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00207713-50-5-1931%26rft_id%253Dinfo%253Apmid%252F11034507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00207713-50-5-1931&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11034507&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089717800030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">âµ</a><div class="cit ref-cit ref-other" id="cit-1.1.11.64"><div class="cit-metadata unstructured">Nieduszynski, C. A., and G. Liti, 2011âFrom sequence to function: insights from natural variation in budding yeasts. Biochim. Biophys. Acta. DOI:10.1016/j.bbagen.2011.02.004</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.65" data-doi="10.1038/nrg2603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Fitness and its role in evolutionary genetics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">531</span>â<span class="cit-lpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DOrr%26rft.auinit1%253DH.%2BA.%26rft.volume%253D10%26rft.issue%253D8%26rft.spage%253D531%26rft.epage%253D539%26rft.atitle%253DFitness%2Band%2Bits%2Brole%2Bin%2Bevolutionary%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2603%26rft_id%253Dinfo%253Apmid%252F19546856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19546856&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268038200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.66" data-doi="10.1002/dvdy.20564"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peichel</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Fishing for the secrets of vertebrate evolution in threespine sticklebacks.</span> <abbr class="cit-jnl-abbrev">Dev. Dyn.</abbr> <span class="cit-vol">234</span>: <span class="cit-fpage">815</span>â<span class="cit-lpage">823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bdynamics%2B%253A%2B%2Ban%2Bofficial%2Bpublication%2Bof%2Bthe%2BAmerican%2BAssociation%2Bof%2BAnatomists%26rft.stitle%253DDev%2BDyn%26rft.aulast%253DPeichel%26rft.auinit1%253DC.%2BL.%26rft.volume%253D234%26rft.issue%253D4%26rft.spage%253D815%26rft.epage%253D823%26rft.atitle%253DFishing%2Bfor%2Bthe%2Bsecrets%2Bof%2Bvertebrate%2Bevolution%2Bin%2Bthreespine%2Bsticklebacks.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fdvdy.20564%26rft_id%253Dinfo%253Apmid%252F16252286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/dvdy.20564&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16252286&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233715500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.67" data-doi="10.1111/j.1365-2958.2004.04182.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piskur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langkjaer</span>  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Yeast genome sequencing: the power of comparative genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">381</span>â<span class="cit-lpage">389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DPiskur%26rft.auinit1%253DJ.%26rft.volume%253D53%26rft.issue%253D2%26rft.spage%253D381%26rft.epage%253D389%26rft.atitle%253DYeast%2Bgenome%2Bsequencing%253A%2Bthe%2Bpower%2Bof%2Bcomparative%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2004.04182.x%26rft_id%253Dinfo%253Apmid%252F15228521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2004.04182.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15228521&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222423000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.68" data-doi="10.1073/pnas.0700488104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prud'homme</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gompel</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Emerging principles of regulatory evolution.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">8605</span>â<span class="cit-lpage">8612</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0700488104%26rft_id%253Dinfo%253Apmid%252F17494759%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxNjoiMTA0L3N1cHBsXzEvODYwNSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzEvMS8xMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.69" data-doi="10.3114/sim.2007.59.02"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payne</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fedorova</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Machida</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">What can comparative genomics tell us about species concepts in the genus Aspergillus?</span> <abbr class="cit-jnl-abbrev">Stud. Mycol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">11</span>â<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStudies%2Bin%2BMycology%26rft.stitle%253DSIM%26rft.aulast%253DRokas%26rft.auinit1%253DA.%26rft.volume%253D59%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D17%26rft.atitle%253DWhat%2Bcan%2Bcomparative%2Bgenomics%2Btell%2Bus%2Babout%2Bspecies%2Bconcepts%2Bin%2Bthe%2Bgenus%2BAspergillus%253F%26rft_id%253Dinfo%253Adoi%252F10.3114%252Fsim.2007.59.02%26rft_id%253Dinfo%253Apmid%252F18490942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3114/sim.2007.59.02&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18490942&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.70" data-doi="10.1038/nature02053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Genome-scale approaches to resolving incongruence in molecular phylogenies.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">798</span>â<span class="cit-lpage">804</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRokas%26rft.auinit1%253DA.%26rft.volume%253D425%26rft.issue%253D6960%26rft.spage%253D798%26rft.epage%253D804%26rft.atitle%253DGenome-scale%2Bapproaches%2Bto%2Bresolving%2Bincongruence%2Bin%2Bmolecular%2Bphylogenies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02053%26rft_id%253Dinfo%253Apmid%252F14574403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14574403&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186118500036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.71" data-doi="10.1128/AEM.02396-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sampaio</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goncalves</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Natural populations of Saccharomyces kudriavzevii in Portugal are associated with oak bark and are sympatric with S. cerevisiae and S. paradoxus.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">2144</span>â<span class="cit-lpage">2152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.02396-07%26rft_id%253Dinfo%253Apmid%252F18281431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6Ijc0LzcvMjE0NCI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzEvMS8xMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.72" data-doi="10.1002/yea.1515"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scannell</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">a</span>â<span class="cit-article-title">Yeast genome evolutionâthe origin of the species.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">929</span>â<span class="cit-lpage">942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DScannell%26rft.auinit1%253DD.%2BR.%26rft.volume%253D24%26rft.issue%253D11%26rft.spage%253D929%26rft.epage%253D942%26rft.atitle%253DYeast%2Bgenome%2Bevolution--the%2Borigin%2Bof%2Bthe%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1515%26rft_id%253Dinfo%253Apmid%252F17621376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1515&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17621376&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251549800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.73" data-doi="10.1038/nature04562"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scannell</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Multiple rounds of speciation associated with reciprocal gene loss in polyploid yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">440</span>: <span class="cit-fpage">341</span>â<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DScannell%26rft.auinit1%253DD.%2BR.%26rft.volume%253D440%26rft.issue%253D7082%26rft.spage%253D341%26rft.epage%253D345%26rft.atitle%253DMultiple%2Brounds%2Bof%2Bspeciation%2Bassociated%2Bwith%2Breciprocal%2Bgene%2Bloss%2Bin%2Bpolyploid%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04562%26rft_id%253Dinfo%253Apmid%252F16541074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04562&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16541074&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235997600054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.74" data-doi="10.1073/pnas.0608218104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scannell</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frank</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conant</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolfit</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">b</span>â<span class="cit-article-title">Independent sorting-out of thousands of duplicated gene pairs in two yeast species descended from a whole-genome duplication.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">8397</span>â<span class="cit-lpage">8402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0608218104%26rft_id%253Dinfo%253Apmid%252F17494770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzIwLzgzOTciO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzEvMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.75" data-doi="10.1101/gr.6341207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scannell</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">A burst of protein sequence evolution and a prolonged period of asymmetric evolution follow gene duplication in yeast.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">137</span>â<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6341207%26rft_id%253Dinfo%253Apmid%252F18025270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzEvMTM3IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8xLzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.76" data-doi="10.1126/science.1151107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Widespread genetic incompatibility in C. elegans maintained by balancing selection.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">589</span>â<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1151107%26rft_id%253Dinfo%253Apmid%252F18187622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTkvNTg2My81ODkiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzEvMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.77" data-doi="10.1093/molbev/msj021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Choosing appropriate substitution models for the phylogenetic analysis of protein-coding sequences.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">7</span>â<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsj021%26rft_id%253Dinfo%253Apmid%252F16177232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msj021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16177232&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233843900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.78" data-doi="10.1186/1471-2105-6-31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slater</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Automated generation of heuristics for biological sequence comparison.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DSlater%26rft.auinit1%253DG.%2BS.%26rft.volume%253D6%26rft.spage%253D31%26rft.epage%253D31%26rft.atitle%253DAutomated%2Bgeneration%2Bof%2Bheuristics%2Bfor%2Bbiological%2Bsequence%2Bcomparison.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-6-31%26rft_id%253Dinfo%253Apmid%252F15713233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-6-31&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15713233&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.79" data-doi="10.1101/gr.091546.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Souciet</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaillardin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baret</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Comparative genomics of protoploid Saccharomycetaceae.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1696</span>â<span class="cit-lpage">1709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.091546.109%26rft_id%253Dinfo%253Apmid%252F19525356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMC8xNjk2IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8xLzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.80" data-doi="10.1371/journal.pbio.0000045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blasiar</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blumenthal</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brent</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">The genome sequence of Caenorhabditis briggsae: a platform for comparative genomics.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">E45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DStein%26rft.auinit1%253DL.%2BD.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253DE45%26rft.epage%253DE45%26rft.atitle%253DThe%2Bgenome%2Bsequence%2Bof%2BCaenorhabditis%2Bbriggsae%253A%2Ba%2Bplatform%2Bfor%2Bcomparative%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0000045%26rft_id%253Dinfo%253Apmid%252F14624247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0000045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14624247&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.81" data-doi="10.1038/90837"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storici</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resnick</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">In vivo site-directed mutagenesis using oligonucleotides.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">773</span>â<span class="cit-lpage">776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DStorici%26rft.auinit1%253DF.%26rft.volume%253D19%26rft.issue%253D8%26rft.spage%253D773%26rft.epage%253D776%26rft.atitle%253DIn%2Bvivo%2Bsite-directed%2Bmutagenesis%2Busing%2Boligonucleotides.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F90837%26rft_id%253Dinfo%253Apmid%252F11479573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/90837&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11479573&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170188600031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.82" data-doi="10.1016/j.ymeth.2006.06.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storici</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resnick</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">The delitto perfetto approach to in vivo site-directed mutagenesis and chromosome rearrangements with synthetic oligonucleotides in yeast.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">409</span>: <span class="cit-fpage">329</span>â<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DStorici%26rft.auinit1%253DF.%26rft.volume%253D409%26rft.spage%253D329%26rft.epage%253D345%26rft.atitle%253DThe%2Bdelitto%2Bperfetto%2Bapproach%2Bto%2Bin%2Bvivo%2Bsite-directed%2Bmutagenesis%2Band%2Bchromosome%2Brearrangements%2Bwith%2Bsynthetic%2Boligonucleotides%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2006.06.011%26rft_id%253Dinfo%253Apmid%252F16793410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2006.06.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16793410&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238354800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.83" data-doi="10.3852/mycologia.98.6.838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berbee</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Dating divergences in the Fungal Tree of Life: review and new analyses.</span> <abbr class="cit-jnl-abbrev">Mycologia</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">838</span>â<span class="cit-lpage">849</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycologia%26rft.stitle%253DMycologia%26rft.aulast%253DTaylor%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D98%26rft.issue%253D6%26rft.spage%253D838%26rft.epage%253D849%26rft.atitle%253DDating%2Bdivergences%2Bin%2Bthe%2BFungal%2BTree%2Bof%2BLife%253A%2Breview%2Band%2Bnew%2Banalyses%26rft_id%253Dinfo%253Adoi%252F10.3852%252Fmycologia.98.6.838%26rft_id%253Dinfo%253Apmid%252F17486961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToibXljb2xvZ2lhIjtzOjU6InJlc2lkIjtzOjg6Ijk4LzYvODM4IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8xLzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.84" data-doi="10.1126/science.1169766"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tirosh</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reikhav</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barkai</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A yeast hybrid provides insight into the evolution of gene expression regulation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">659</span>â<span class="cit-lpage">662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTirosh%26rft.auinit1%253DI.%26rft.volume%253D324%26rft.issue%253D5927%26rft.spage%253D659%26rft.epage%253D662%26rft.atitle%253DA%2BYeast%2BHybrid%2BProvides%2BInsight%2Binto%2Bthe%2BEvolution%2Bof%2BGene%2BExpression%2BRegulation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1169766%26rft_id%253Dinfo%253Apmid%252F19407207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNy82NTkiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzEvMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.85" data-doi="10.1099/ijs.0.65331-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">F. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Saccharomyces arboricolus sp. nov., a yeast species from tree bark.</span> <abbr class="cit-jnl-abbrev">Int. J. Syst. Evol. Microbiol.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">510</span>â<span class="cit-lpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BSystematic%2Band%2BEvolutionary%2BMicrobiology%26rft.stitle%253DInt.%2BJ.%2BSyst.%2BEvol.%2BMicrobiol.%26rft.aulast%253DWang%26rft.auinit1%253DS.-A.%26rft.volume%253D58%26rft.issue%253D2%26rft.spage%253D510%26rft.epage%253D514%26rft.atitle%253DSaccharomyces%2Barboricolus%2Bsp.%2Bnov.%252C%2Ba%2Byeast%2Bspecies%2Bfrom%2Btree%2Bbark%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fijs.0.65331-0%26rft_id%253Dinfo%253Apmid%252F18218959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/ijs.0.65331-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18218959&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253563800036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.86" data-doi="10.1093/bioinformatics/btm193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wapinski</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfeffer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regev</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">a</span>â<span class="cit-article-title">Automatic genome-wide reconstruction of phylogenetic gene trees.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">i549</span>â<span class="cit-lpage">i558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DWapinski%26rft.auinit1%253DI.%26rft.volume%253D23%26rft.issue%253D13%26rft.spage%253Di549%26rft.epage%253Di558%26rft.atitle%253DAutomatic%2Bgenome-wide%2Breconstruction%2Bof%2Bphylogenetic%2Bgene%2Btrees%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm193%26rft_id%253Dinfo%253Apmid%252F17646342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17646342&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248620400088&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.87" data-doi="10.1038/nature06107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wapinski</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfeffer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regev</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">b</span>â<span class="cit-article-title">Natural history and evolutionary principles of gene duplication in fungi.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">449</span>: <span class="cit-fpage">54</span>â<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWapinski%26rft.auinit1%253DI.%26rft.volume%253D449%26rft.issue%253D7158%26rft.spage%253D54%26rft.epage%253D61%26rft.atitle%253DNatural%2Bhistory%2Band%2Bevolutionary%2Bprinciples%2Bof%2Bgene%2Bduplication%2Bin%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06107%26rft_id%253Dinfo%253Apmid%252F17805289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17805289&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249233500034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.88" data-doi="10.1093/nar/gkg609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wernersson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">RevTrans: Multiple alignment of coding DNA from aligned amino acid sequences.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3537</span>â<span class="cit-lpage">3539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DWernersson%26rft.auinit1%253DR.%26rft.volume%253D31%26rft.issue%253D13%26rft.spage%253D3537%26rft.epage%253D3539%26rft.atitle%253DRevTrans%253A%2Bmultiple%2Balignment%2Bof%2Bcoding%2BDNA%2Bfrom%2Baligned%2Bamino%2Bacid%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg609%26rft_id%253Dinfo%253Apmid%252F12824361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg609&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12824361&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183832900053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.89" data-doi="10.1093/bioinformatics/btm632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yorke</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Figaro: a novel statistical method for vector sequence removal.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">462</span>â<span class="cit-lpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm632%26rft_id%253Dinfo%253Apmid%252F18202027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18202027&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253246300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.90" data-doi="10.1371/journal.pgen.1000893"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Will</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Painter</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Incipient balancing selection through adaptive loss of aquaporins in natural Saccharomyces cerevisiae populations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000893</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWill%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De1000893%26rft.epage%253De1000893%26rft.atitle%253DIncipient%2Bbalancing%2Bselection%2Bthrough%2Badaptive%2Bloss%2Bof%2Baquaporins%2Bin%2Bnatural%2BSaccharomyces%2Bcerevisiae%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000893%26rft_id%253Dinfo%253Apmid%252F20369021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000893&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20369021&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.91" data-doi="10.1038/42711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shields</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Molecular evidence for an ancient duplication of the entire yeast genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">387</span>: <span class="cit-fpage">708</span>â<span class="cit-lpage">713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWolfe%26rft.auinit1%253DK.%2BH.%26rft.volume%253D387%26rft.issue%253D6634%26rft.spage%253D708%26rft.epage%253D713%26rft.atitle%253DMolecular%2Bevidence%2Bfor%2Ban%2Bancient%2Bduplication%2Bof%2Bthe%2Bentire%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F42711%26rft_id%253Dinfo%253Apmid%252F9192896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/42711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9192896&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XD86900055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.92" data-doi="10.1038/nature724"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gwilliam</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajandream</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyne</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyne</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">The genome sequence of Schizosaccharomyces pombe.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">415</span>: <span class="cit-fpage">871</span>â<span class="cit-lpage">880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWood%26rft.auinit1%253DV.%26rft.volume%253D415%26rft.issue%253D6874%26rft.spage%253D871%26rft.epage%253D880%26rft.atitle%253DThe%2Bgenome%2Bsequence%2Bof%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature724%26rft_id%253Dinfo%253Apmid%252F11859360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature724&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11859360&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173941000036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.93" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">PAML 4: phylogenetic analysis by maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1586</span>â<span class="cit-lpage">1591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17483113&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248848400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.94" data-doi="10.1101/gr.074492.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zerbino</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Velvet: algorithms for de novo short read assembly using de Bruijn graphs.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">821</span>â<span class="cit-lpage">829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.074492.107%26rft_id%253Dinfo%253Apmid%252F18349386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzUvODIxIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8xLzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.95" data-doi="10.1101/gad.1640008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zill</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rine</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Interspecies variation reveals a conserved repressor of alpha-specific genes in Saccharomyces yeasts.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1704</span>â<span class="cit-lpage">1716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DZill%26rft.auinit1%253DO.%2BA.%26rft.volume%253D22%26rft.issue%253D12%26rft.spage%253D1704%26rft.epage%253D1716%26rft.atitle%253DInterspecies%2Bvariation%2Breveals%2Ba%2Bconserved%2Brepressor%2Bof%2B%257Balpha%257D-specific%2Bgenes%2Bin%2BSaccharomyces%2Byeasts%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1640008%26rft_id%253Dinfo%253Apmid%252F18559484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIyLzEyLzE3MDQiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzEvMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.1.11.96" data-doi="10.1371/journal.pbio.1000550"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zill</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scannell</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teytelman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rine</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Co-evolution of transcriptional silencing proteins and the DNA elements specifying their assembly.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1000550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DZill%26rft.auinit1%253DO.%2BA.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253De1000550%26rft.epage%253De1000550%26rft.atitle%253DCo-evolution%2Bof%2Btranscriptional%2Bsilencing%2Bproteins%2Band%2Bthe%2BDNA%2Belements%2Bspecifying%2Btheir%2Bassembly.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000550%26rft_id%253Dinfo%253Apmid%252F21151344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000550&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21151344&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F11.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/1/11.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/1/3" title="An SNP-Based Linkage Map for Zebrafish Reveals Sex Determination Loci" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/1/27" title="Variability of Gene Expression After Polyhaploidization in Wheat (Triticum aestivum L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/1"><span>Volume 1 Issue 1, June 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/1.cover.gif?itok=pV8lCSGW" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/1/11" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_-yfEoMu5ynHoirVCHtQCAl9szdYJHVkXszfvJvrMd8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/1/11" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/1/11" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/1/11" class="active" data-icon-position="" data-hide-link-title="0">The Awesome Power of Yeast Evolutionary Genetics: New Genome Sequences and Strain Resources for the Saccharomyces sensu stricto Genus</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459154" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-wRb0J2RmIEhrnXUkjoN74ilxuQ01vS0P5K6ymw1RwNM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/1/11.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/1/11" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-U1_zhhcxcEPg1yBVJ6YXZS_-mTGC9kqVFXA1U08NUiU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/1/11" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/1/11.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459154" id="node459154--2" data-pisa="ggg;1/1/11" data-pisa-master="ggg;g3.111.000273" data-apath="/ggg/1/1/11.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Awesome Power of Yeast Evolutionary Genetics: New Genome Sequences and Strain Resources for the <em>Saccharomyces sensu stricto</em> Genus</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Devin R. Scannell</span>, <span class="highwire-citation-author" data-delta="1">Oliver A. Zill</span>, <span class="highwire-citation-author" data-delta="2">Antonis Rokas</span>, <span class="highwire-citation-author" data-delta="3">Celia Payen</span>, <span class="highwire-citation-author" data-delta="4">Maitreya J. Dunham</span>, <span class="highwire-citation-author" data-delta="5">Michael B. Eisen</span>, <span class="highwire-citation-author" data-delta="6">Jasper Rine</span>, <span class="highwire-citation-author" data-delta="7">Mark Johnston</span> and <span class="highwire-citation-author" data-delta="8">Chris Todd Hittinger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">11-25;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000273 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459154--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Devin R. Scannell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cell Biology and California Institute for Quantitative Biosciences, UC Berkeley, Berkeley, California 94720-3220</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Devin%2BR.%2BScannell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scannell%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADevin%2BR.%2BScannell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Oliver A. Zill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cell Biology and California Institute for Quantitative Biosciences, UC Berkeley, Berkeley, California 94720-3220</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oliver%2BA.%2BZill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zill%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOliver%2BA.%2BZill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zill@berkeley.edu" class="" data-icon-position="" data-hide-link-title="0">zill@berkeley.edu</a>
<a href="mailto:cthittinger@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">cthittinger@wisc.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Antonis Rokas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonis%2BRokas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rokas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntonis%2BRokas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Celia Payen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Celia%2BPayen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Payen%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACelia%2BPayen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Maitreya J. Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maitreya%2BJ.%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaitreya%2BJ.%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michael B. Eisen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cell Biology and California Institute for Quantitative Biosciences, UC Berkeley, Berkeley, California 94720-3220</div><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Howard Hughes Medical Institute, University of California, Berkeley, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BB.%2BEisen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eisen%20MB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BB.%2BEisen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jasper Rine </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cell Biology and California Institute for Quantitative Biosciences, UC Berkeley, Berkeley, California 94720-3220</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jasper%2BRine%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rine%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJasper%2BRine%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Mark Johnston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, Colorado 80045-2530</div><div class='author-affiliation'><span class='nlm-sup'>â â </span>Center for Genome Sciences, Department of Genetics, Washington University in St. Louis School of Medicine, St. Louis, Missouri 63108-2212</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BJohnston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnston%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BJohnston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Chris Todd Hittinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, Colorado 80045-2530</div><div class='author-affiliation'><span class='nlm-sup'>â â </span>Center for Genome Sciences, Department of Genetics, Washington University in St. Louis School of Medicine, St. Louis, Missouri 63108-2212</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BTodd%2BHittinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hittinger%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChris%2BTodd%2BHittinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zill@berkeley.edu" class="" data-icon-position="" data-hide-link-title="0">zill@berkeley.edu</a>
<a href="mailto:cthittinger@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">cthittinger@wisc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/1/11" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/1/11&amp;title=The%20Awesome%20Power%20of%20Yeast%20Evolutionary%20Genetics%3A%20New%20Genome%20Sequences%20and%20Strain%20Resources%20for%20the%20Saccharomyces%20sensu%20stricto%20Genus" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/1/11&amp;title=The%20Awesome%20Power%20of%20Yeast%20Evolutionary%20Genetics%3A%20New%20Genome%20Sequences%20and%20Strain%20Resources%20for%20the%20Saccharomyces%20sensu%20stricto%20Genus" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/1/11&amp;title=The%20Awesome%20Power%20of%20Yeast%20Evolutionary%20Genetics%3A%20New%20Genome%20Sequences%20and%20Strain%20Resources%20for%20the%20Saccharomyces%20sensu%20stricto%20Genus" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/1/11&amp;text=The%20Awesome%20Power%20of%20Yeast%20Evolutionary%20Genetics%3A%20New%20Genome%20Sequences%20and%20Strain%20Resources%20for%20the%20Saccharomyces%20sensu%20stricto%20Genus" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/1/11&amp;title=The%20Awesome%20Power%20of%20Yeast%20Evolutionary%20Genetics%3A%20New%20Genome%20Sequences%20and%20Strain%20Resources%20for%20the%20Saccharomyces%20sensu%20stricto%20Genus" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/1/11&amp;t=The%20Awesome%20Power%20of%20Yeast%20Evolutionary%20Genetics%3A%20New%20Genome%20Sequences%20and%20Strain%20Resources%20for%20the%20Saccharomyces%20sensu%20stricto%20Genus" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/1/11&amp;title=The%20Awesome%20Power%20of%20Yeast%20Evolutionary%20Genetics%3A%20New%20Genome%20Sequences%20and%20Strain%20Resources%20for%20the%20Saccharomyces%20sensu%20stricto%20Genus" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/1/11&amp;title=The%20Awesome%20Power%20of%20Yeast%20Evolutionary%20Genetics%3A%20New%20Genome%20Sequences%20and%20Strain%20Resources%20for%20the%20Saccharomyces%20sensu%20stricto%20Genus" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459154/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459154" id="node459154--3" data-pisa="ggg;1/1/11" data-pisa-master="ggg;g3.111.000273" data-apath="/ggg/1/1/11.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Awesome Power of Yeast Evolutionary Genetics: New Genome Sequences and Strain Resources for the <em>Saccharomyces sensu stricto</em> Genus</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Devin R. Scannell</span>, <span class="highwire-citation-author" data-delta="1">Oliver A. Zill</span>, <span class="highwire-citation-author" data-delta="2">Antonis Rokas</span>, <span class="highwire-citation-author" data-delta="3">Celia Payen</span>, <span class="highwire-citation-author" data-delta="4">Maitreya J. Dunham</span>, <span class="highwire-citation-author" data-delta="5">Michael B. Eisen</span>, <span class="highwire-citation-author" data-delta="6">Jasper Rine</span>, <span class="highwire-citation-author" data-delta="7">Mark Johnston</span> and <span class="highwire-citation-author" data-delta="8">Chris Todd Hittinger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">11-25;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000273 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459154--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Devin R. Scannell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cell Biology and California Institute for Quantitative Biosciences, UC Berkeley, Berkeley, California 94720-3220</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Devin%2BR.%2BScannell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scannell%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADevin%2BR.%2BScannell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Oliver A. Zill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cell Biology and California Institute for Quantitative Biosciences, UC Berkeley, Berkeley, California 94720-3220</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oliver%2BA.%2BZill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zill%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOliver%2BA.%2BZill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zill@berkeley.edu" class="" data-icon-position="" data-hide-link-title="0">zill@berkeley.edu</a>
<a href="mailto:cthittinger@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">cthittinger@wisc.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Antonis Rokas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonis%2BRokas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rokas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntonis%2BRokas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Celia Payen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Celia%2BPayen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Payen%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACelia%2BPayen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Maitreya J. Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maitreya%2BJ.%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaitreya%2BJ.%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michael B. Eisen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cell Biology and California Institute for Quantitative Biosciences, UC Berkeley, Berkeley, California 94720-3220</div><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Howard Hughes Medical Institute, University of California, Berkeley, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BB.%2BEisen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eisen%20MB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BB.%2BEisen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jasper Rine </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cell Biology and California Institute for Quantitative Biosciences, UC Berkeley, Berkeley, California 94720-3220</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jasper%2BRine%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rine%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJasper%2BRine%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Mark Johnston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, Colorado 80045-2530</div><div class='author-affiliation'><span class='nlm-sup'>â â </span>Center for Genome Sciences, Department of Genetics, Washington University in St. Louis School of Medicine, St. Louis, Missouri 63108-2212</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BJohnston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnston%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BJohnston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Chris Todd Hittinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, Colorado 80045-2530</div><div class='author-affiliation'><span class='nlm-sup'>â â </span>Center for Genome Sciences, Department of Genetics, Washington University in St. Louis School of Medicine, St. Louis, Missouri 63108-2212</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BTodd%2BHittinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hittinger%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChris%2BTodd%2BHittinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zill@berkeley.edu" class="" data-icon-position="" data-hide-link-title="0">zill@berkeley.edu</a>
<a href="mailto:cthittinger@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">cthittinger@wisc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=11&amp;rft.epage=25&amp;rft.atitle=The%20Awesome%20Power%20of%20Yeast%20Evolutionary%20Genetics%3A%20New%20Genome%20Sequences%20and%20Strain%20Resources%20for%20the%20Saccharomyces%20sensu%20stricto%20Genus&amp;rft.volume=1&amp;rft.issue=1&amp;rft.date=2011-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Scannell%2C+Devin+R.&amp;rft.au=Zill%2C+Oliver+A.&amp;rft.au=Rokas%2C+Antonis&amp;rft.au=Payen%2C+Celia&amp;rft.au=Dunham%2C+Maitreya+J.&amp;rft.au=Eisen%2C+Michael+B.&amp;rft.au=Rine%2C+Jasper&amp;rft.au=Johnston%2C+Mark&amp;rft.au=Hittinger%2C+Chris+Todd"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459154/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459154/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459154/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459154/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459154/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459154/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459154/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459154/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459154/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459154/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459154/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459154/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496048" id="node496048" data-pisa="ggg;g3.119.400937v1" data-pisa-master="ggg;g3.119.400937" data-apath="/ggg/early/2020/03/25/g3.119.400937.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/25/g3.119.400937" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tunable Transcriptional Interference at the Endogenous Alcohol Dehydrogenase Gene Locus in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496046" id="node496046" data-pisa="ggg;g3.120.401048v1" data-pisa-master="ggg;g3.120.401048" data-apath="/ggg/early/2020/03/23/g3.120.401048.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/23/g3.120.401048" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tools for the Genetic Manipulation of <em>Herpetomonas muscarum</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496045" id="node496045" data-pisa="ggg;g3.119.400884v1" data-pisa-master="ggg;g3.119.400884" data-apath="/ggg/early/2020/03/17/g3.119.400884.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400884" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study for Maize Leaf Cuticular Conductance Identifies Candidate Genes Involved in the Regulation of Cuticle Development</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/1/11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-27" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/1/11.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/1/11.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/1/1/11.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/1/1/11.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
