<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/1/35" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/1/35.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/1/35.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/1/35.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1.cover.gif" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/1/1/35/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/1/35.atom" />
<meta name="HW.pisa" content="ggg;1/1/35" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Inferring Haplotypes of Copy Number Variations From High-Throughput Data With Uncertainty" />
<meta name="DC.Identifier" content="10.1534/g3.111.000174" />
<meta name="DC.Date" content="2011-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2011 Kato et al.. This is an open access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Accurate information on haplotypes and diplotypes (haplotype pairs) is required for population-genetic analyses; however, microarrays do not provide data on a haplotype or diplotype at a copy number variation (CNV) locus; they only provide data on the total number of copies over a diplotype or an unphased sequence genotype ( e.g. , AAB, unlike AB of single nucleotide polymorphism). Moreover, such copy numbers or genotypes are often incorrectly determined when microarray signal intensities derived from different copy numbers or genotypes are not clearly separated due to noise. Here we report an algorithm to infer CNV haplotypes and individuals’ diplotypes at multiple loci from noisy microarray data, utilizing the probability that a signal intensity may be derived from different underlying copy numbers or genotypes. Performing simulation studies based on known diplotypes and an error model obtained from real microarray data, we demonstrate that this probabilistic approach succeeds in accurate inference (error rate: 1–2%) from noisy data, whereas previous deterministic approaches failed (error rate: 12–18%). Applying this algorithm to real microarray data, we estimated haplotype frequencies and diplotypes in 1486 CNV regions for 100 individuals. Our algorithm will facilitate accurate population-genetic analyses and powerful disease association studies of CNVs." />
<meta name="DC.Contributor" content="Mamoru Kato" />
<meta name="DC.Contributor" content="Seungtai Yoon" />
<meta name="DC.Contributor" content="Naoya Hosono" />
<meta name="DC.Contributor" content="Anthony Leotta" />
<meta name="DC.Contributor" content="Jonathan Sebat" />
<meta name="DC.Contributor" content="Tatsuhiko Tsunoda" />
<meta name="DC.Contributor" content="Michael Q. Zhang" />
<meta name="article:published_time" content="2011-06-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="Inferring Haplotypes of Copy Number Variations From High-Throughput Data With Uncertainty" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Accurate information on haplotypes and diplotypes (haplotype pairs) is required for population-genetic analyses; however, microarrays do not provide data on a haplotype or diplotype at a copy number variation (CNV) locus; they only provide data on the total number of copies over a diplotype or an unphased sequence genotype (&lt;i&gt;e.g.&lt;/i&gt;, AAB, unlike AB of single nucleotide polymorphism). Moreover, such copy numbers or genotypes are often incorrectly determined when microarray signal intensities derived from different copy numbers or genotypes are not clearly separated due to noise. Here we report an algorithm to infer CNV haplotypes and individuals’ diplotypes at multiple loci from noisy microarray data, utilizing the probability that a signal intensity may be derived from different underlying copy numbers or genotypes. Performing simulation studies based on known diplotypes and an error model obtained from real microarray data, we demonstrate that this probabilistic approach succeeds in accurate inference (error rate: 1–2%) from noisy data, whereas previous deterministic approaches failed (error rate: 12–18%). Applying this algorithm to real microarray data, we estimated haplotype frequencies and diplotypes in 1486 CNV regions for 100 individuals. Our algorithm will facilitate accurate population-genetic analyses and powerful disease association studies of CNVs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/06/01" />
<meta name="citation_mjid" content="ggg;1/1/35" />
<meta name="citation_id" content="1/1/35" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/1/35" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/1/35.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/1/35.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/1/35.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.000174" />
<meta name="citation_pmid" content="22384316" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="1" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="35" />
<meta name="citation_lastpage" content="42" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Mamoru Kato" />
<meta name="citation_author_institution" content="Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724" />
<meta name="citation_author" content="Seungtai Yoon" />
<meta name="citation_author_institution" content="Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724" />
<meta name="citation_author" content="Naoya Hosono" />
<meta name="citation_author_institution" content="Center for Genomic Medicine, RIKEN, Yokohama, Kanagawa 230-0045, Japan" />
<meta name="citation_author" content="Anthony Leotta" />
<meta name="citation_author_institution" content="Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724" />
<meta name="citation_author" content="Jonathan Sebat" />
<meta name="citation_author_institution" content="Department of Psychiatry, University of California, San Diego, La Jolla, CA 92093" />
<meta name="citation_author" content="Tatsuhiko Tsunoda" />
<meta name="citation_author_institution" content="Center for Genomic Medicine, RIKEN, Yokohama, Kanagawa 230-0045, Japan" />
<meta name="citation_author" content="Michael Q. Zhang" />
<meta name="citation_author_institution" content="Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724" />
<meta name="citation_author_institution" content="Bioinformatics Division, TNLIST/Department of Automation, Tsinghua University, Beijing, 100084, China" />
<meta name="citation_author_email" content="mzhang@cshl.edu" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Alkan;citation_author=J. M. Kidd;citation_author=T. Marques-Bonet;citation_author=G. Aksay;citation_author=F. Antonacci;citation_title=Personalized copy number and segmental duplication maps using next-generation sequencing.;citation_pages=1061-1067;citation_volume=41;citation_year=2009;citation_issue=10;citation_pmid=19718026;citation_doi=10.1038/ng.437" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=B. L. Browning;citation_author=Z. Yu;citation_title=Simultaneous genotype calling and haplotype phasing improves genotype accuracy and reduces false-positive associations for genome-wide association studies.;citation_pages=847-861;citation_volume=85;citation_year=2009;citation_issue=6;citation_pmid=19931040;citation_doi=10.1016/j.ajhg.2009.11.004" />
<meta name="citation_reference" content="citation_title=Computational Methods for SNPs and Haplotype Inference, Lecture Notes in Bioinformatics 2983;citation_title=Trisomic phase inference;citation_author=A. G. Clark;citation_author=E. T. Dermitzakis;citation_author=S. E. Antonarakis;citation_pages=1-8;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=L. J. Coin;citation_author=J. E. Asher;citation_author=R. G. Walters;citation_author=J. S. Moustafa;citation_author=A. J. de Smith;citation_title=cnvHap: an integrative population and haplotype-based multiplatform model of SNPs and CNVs.;citation_pages=541-546;citation_volume=7;citation_year=2010;citation_issue=7;citation_pmid=20512141;citation_doi=10.1038/nmeth.1466" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. F. Conrad;citation_author=M. E. Hurles;citation_title=The population genetics of structural variation.;citation_pages=S30-S36;citation_volume=39;citation_year=2007;citation_issue=7 Suppl;citation_pmid=17597779;citation_doi=10.1038/ng2042" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. F. Conrad;citation_author=D. Pinto;citation_author=R. Redon;citation_author=L. Feuk;citation_author=O. Gokcumen;citation_title=Origins and functional impact of copy number variation in the human genome.;citation_pages=704-712;citation_volume=464;citation_year=2010;citation_issue=7289;citation_pmid=19812545;citation_doi=10.1038/nature08516" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=A. J. de Smith;citation_author=A. Tsalenko;citation_author=N. Sampas;citation_author=A. Scheffer;citation_author=N. A. Yamada;citation_title=Array CGH analysis of copy number variation identifies 1284 new genes variant in healthy white males: implications for association studies of complex diseases.;citation_volume=16;citation_year=2007;citation_pmid=17666407;citation_doi=10.1093/hmg/ddm208" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=L. Excoffier;citation_author=M. Slatkin;citation_title=Maximum-likelihood estimation of molecular haplotype frequencies in a diploid population;citation_pages=921-927;citation_volume=12;citation_year=1995;citation_issue=5;citation_pmid=7476138" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. L. Freeman;citation_author=G. H. Perry;citation_author=L. Feuk;citation_author=R. Redon;citation_author=S. A. McCarroll;citation_title=Copy number variation: new insights in genome diversity.;citation_volume=16;citation_year=2006;citation_pmid=16809666;citation_doi=10.1101/gr.3677206" />
<meta name="citation_reference" content="citation_journal_title=Clin. Chem.;citation_author=N. Hosono;citation_author=M. Kato;citation_author=K. Kiyotani;citation_author=T. Mushiroda;citation_author=S. Takata;citation_title=CYP2D6 genotyping for functional-gene dosage analysis by allele copy number detection.;citation_volume=55;citation_year=2009;citation_pmid=19541866;citation_doi=10.1373/clinchem.2009.123620" />
<meta name="citation_reference" content="citation_journal_title=Human mutation;citation_journal_abbrev=Hum Mutat;citation_author=N. Hosono;citation_author=M. Kubo;citation_author=Y. Tsuchiya;citation_author=H. Sato;citation_author=T. Kitamoto;citation_title=Multiplex PCR-based real-time invader assay (mPCR-RETINA): a novel SNP-based method for detecting allelic asymmetries within copy number variation regions.;citation_pages=182-189;citation_volume=29;citation_year=2008;citation_issue=1;citation_pmid=17879341;citation_doi=10.1002/humu.20609" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. J. Iafrate;citation_author=L. Feuk;citation_author=M. N. Rivera;citation_author=M. L. Listewnik;citation_author=P. K. Donahoe;citation_title=Detection of large-scale variation in the human genome.;citation_pages=949-951;citation_volume=36;citation_year=2004;citation_issue=9;citation_pmid=15286789;citation_doi=10.1038/ng1416" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=H. Kang;citation_author=Z. S. Qin;citation_author=T. Niu;citation_author=J. S. Liu;citation_title=Incorporating genotyping uncertainty in haplotype inference for single-nucleotide polymorphisms.;citation_pages=495-510;citation_volume=74;citation_year=2004;citation_issue=3;citation_pmid=14966673;citation_doi=10.1086/382284" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=M. Kato;citation_author=T. Kawaguchi;citation_author=S. Ishikawa;citation_author=T. Umeda;citation_author=R. Nakamichi;citation_title=Population-genetic nature of copy number variations in the human genome.;citation_volume=19;citation_year=2010;citation_pmid=19966329;citation_doi=10.1093/hmg/ddp541" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. Kato;citation_author=Y. Nakamura;citation_author=T. Tsunoda;citation_title=An algorithm for inferring complex haplotypes in a region of copy-number variation.;citation_pages=157-169;citation_volume=83;citation_year=2008;citation_issue=2;citation_pmid=18639202;citation_doi=10.1016/j.ajhg.2008.06.021" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Kato;citation_author=Y. Nakamura;citation_author=T. Tsunoda;citation_title=MOCSphaser: a haplotype inference tool from a mixture of copy number variation and single nucleotide polymorphism data.;citation_volume=24;citation_year=2008;citation_pmid=18492685;citation_doi=10.1093/bioinformatics/btn242" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. M. Korn;citation_author=F. G. Kuruvilla;citation_author=S. A. McCarroll;citation_author=A. Wysoker;citation_author=J. Nemesh;citation_title=Integrated genotype calling and association analysis of SNPs, common copy number polymorphisms and rare CNVs.;citation_pages=1253-1260;citation_volume=40;citation_year=2008;citation_issue=10;citation_pmid=18776909;citation_doi=10.1038/ng.237" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. A. McCarroll;citation_author=D. M. Altshuler;citation_title=Copy-number variation and association studies of human disease.;citation_pages=S37-S42;citation_volume=39;citation_year=2007;citation_issue=7 Suppl;citation_pmid=17597780;citation_doi=10.1038/ng2080" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. A. McCarroll;citation_author=F. G. Kuruvilla;citation_author=J. M. Korn;citation_author=S. Cawley;citation_author=J. Nemesh;citation_title=Integrated detection and population-genetic analysis of SNPs and copy number variation.;citation_pages=1166-1174;citation_volume=40;citation_year=2008;citation_issue=10;citation_pmid=18776908;citation_doi=10.1038/ng.238" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=A. K. Merikangas;citation_author=A. P. Corvin;citation_author=L. Gallagher;citation_title=Copy-number variants in neurodevelopmental disorders: promises and challenges.;citation_pages=536-544;citation_volume=25;citation_year=2009;citation_issue=12;citation_pmid=19910074;citation_doi=10.1016/j.tig.2009.10.006" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. E. Mills;citation_author=K. Walter;citation_author=C. Stewart;citation_author=R. E. Handsaker;citation_author=K. Chen;citation_title=Mapping copy number variation by population-scale genome sequencing.;citation_pages=59-65;citation_volume=470;citation_year=2011;citation_issue=7332;citation_pmid=21293372;citation_doi=10.1038/nature09708" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=T. Niu;citation_title=Algorithms for inferring haplotypes.;citation_pages=334-347;citation_volume=27;citation_year=2004;citation_issue=4;citation_pmid=15368348;citation_doi=10.1002/gepi.20024" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=T. Niu;citation_author=Z. S. Qin;citation_author=X. Xu;citation_author=J. S. Liu;citation_title=Bayesian haplotype inference for multiple linked single-nucleotide polymorphisms.;citation_pages=157-169;citation_volume=70;citation_year=2002;citation_issue=1;citation_pmid=11741196;citation_doi=10.1086/338446" />
<meta name="citation_reference" content="citation_journal_title=Journal of human genetics;citation_journal_abbrev=J Hum Genet;citation_author=J. Ohashi;citation_title=A practical case-control association test for detecting a susceptibility allele at a copy number variation locus.;citation_pages=169-173;citation_volume=54;citation_year=2009;citation_issue=3;citation_pmid=19197338" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=Z. S. Qin;citation_author=T. Niu;citation_author=J. S. Liu;citation_title=Partition-ligation-expectation-maximization algorithm for haplotype inference with single-nucleotide polymorphisms.;citation_pages=1242-1247;citation_volume=71;citation_year=2002;citation_issue=5;citation_pmid=12452179;citation_doi=10.1086/344207" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. Redon;citation_author=S. Ishikawa;citation_author=K. R. Fitch;citation_author=L. Feuk;citation_author=G. H. Perry;citation_title=Global variation in copy number in the human genome.;citation_pages=444-454;citation_volume=444;citation_year=2006;citation_issue=7118;citation_pmid=17122850;citation_doi=10.1038/nature05329" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=C. Sachse;citation_author=J. Brockmoller;citation_author=S. Bauer;citation_author=I. Roots;citation_title=Cytochrome P450 2D6 variants in a Caucasian population: allele frequencies and phenotypic consequences.;citation_pages=284-295;citation_volume=60;citation_year=1997;citation_issue=2;citation_pmid=9012401" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Sebat;citation_title=Major changes in our DNA lead to major changes in our thinking.;citation_pages=S3-S5;citation_volume=39;citation_year=2007;citation_issue=7 Suppl;citation_pmid=17597778;citation_doi=10.1038/ng2095" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Sebat;citation_author=B. Lakshmi;citation_author=J. Troge;citation_author=J. Alexander;citation_author=J. Young;citation_title=Large-Scale Copy Number Polymorphism in the Human Genome;citation_pages=525-528;citation_volume=305;citation_year=2004;citation_issue=5683;citation_pmid=15273396;citation_doi=10.1126/science.1098918" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=F. Speleman;citation_author=C. Kumps;citation_author=K. Buysse;citation_author=B. Poppe;citation_author=B. Menten;citation_title=Copy number alterations and copy number variation in cancer: close encounters of the bad kind.;citation_pages=176-182;citation_volume=123;citation_year=2008;citation_issue=1-4;citation_pmid=19287153;citation_doi=10.1159/000184706" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Stefansson;citation_author=D. Rujescu;citation_author=S. Cichon;citation_author=O. P. Pietilainen;citation_author=A. Ingason;citation_title=Large recurrent microdeletions associated with schizophrenia.;citation_pages=232-236;citation_volume=455;citation_year=2008;citation_issue=7210;citation_pmid=18668039;citation_doi=10.1038/nature07229" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Y. Su;citation_author=J. E. Asher;citation_author=M. R. Jarvelin;citation_author=P. Froguel;citation_author=A. I. Blakemore;citation_title=Inferring combined CNV/SNP haplotypes from genotype data.;citation_volume=26;citation_year=2010;citation_pmid=20406911;citation_doi=10.1093/bioinformatics/btq157" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Rare chromosomal deletions and duplications increase risk of schizophrenia.;citation_pages=237-241;citation_volume=455;citation_year=2008;citation_issue=7210;citation_pmid=18668038;citation_doi=10.1038/nature07239" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Yoon;citation_author=Z. Xuan;citation_author=V. Makarov;citation_author=K. Ye;citation_author=J. Sebat;citation_title=Sensitive and accurate detection of copy number variants using read depth of coverage.;citation_volume=19;citation_year=2009;citation_pmid=19657104;citation_doi=10.1101/gr.092981.109" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=J. Zhang;citation_author=L. Feuk;citation_author=G. E. Duggan;citation_author=R. Khaja;citation_author=S. W. Scherer;citation_title=Development of bioinformatics resources for display and analysis of copy number and other structural variants in the human genome.;citation_pages=205-214;citation_volume=115;citation_year=2006;citation_issue=3-4;citation_pmid=17124402;citation_doi=10.1159/000095916" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Inferring Haplotypes of Copy Number Variations From High-Throughput Data With Uncertainty" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1.cover.gif" />
<meta name="twitter:description" content="Accurate information on haplotypes and diplotypes (haplotype pairs) is required for population-genetic analyses; however, microarrays do not provide data on a haplotype or diplotype at a copy number variation (CNV) locus; they only provide data on the total number of copies over a diplotype or an unphased sequence genotype ( e.g. , AAB, unlike AB of single nucleotide polymorphism). Moreover, such copy numbers or genotypes are often incorrectly determined when microarray signal intensities derived from different copy numbers or genotypes are not clearly separated due to noise. Here we report an algorithm to infer CNV haplotypes and individuals’ diplotypes at multiple loci from noisy microarray data, utilizing the probability that a signal intensity may be derived from different underlying copy numbers or genotypes. Performing simulation studies based on known diplotypes and an error model obtained from real microarray data, we demonstrate that this probabilistic approach succeeds in accurate inference (error rate: 1–2%) from noisy data, whereas previous deterministic approaches failed (error rate: 12–18%). Applying this algorithm to real microarray data, we estimated haplotype frequencies and diplotypes in 1486 CNV regions for 100 individuals. Our algorithm will facilitate accurate population-genetic analyses and powerful disease association studies of CNVs." />
<meta name="og-title" property="og:title" content="Inferring Haplotypes of Copy Number Variations From High-Throughput Data With Uncertainty" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/1/35" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Accurate information on haplotypes and diplotypes (haplotype pairs) is required for population-genetic analyses; however, microarrays do not provide data on a haplotype or diplotype at a copy number variation (CNV) locus; they only provide data on the total number of copies over a diplotype or an unphased sequence genotype ( e.g. , AAB, unlike AB of single nucleotide polymorphism). Moreover, such copy numbers or genotypes are often incorrectly determined when microarray signal intensities derived from different copy numbers or genotypes are not clearly separated due to noise. Here we report an algorithm to infer CNV haplotypes and individuals’ diplotypes at multiple loci from noisy microarray data, utilizing the probability that a signal intensity may be derived from different underlying copy numbers or genotypes. Performing simulation studies based on known diplotypes and an error model obtained from real microarray data, we demonstrate that this probabilistic approach succeeds in accurate inference (error rate: 1–2%) from noisy data, whereas previous deterministic approaches failed (error rate: 12–18%). Applying this algorithm to real microarray data, we estimated haplotype frequencies and diplotypes in 1486 CNV regions for 100 individuals. Our algorithm will facilitate accurate population-genetic analyses and powerful disease association studies of CNVs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1.cover.gif" />
    <title>Inferring Haplotypes of Copy Number Variations From High-Throughput Data With Uncertainty | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"DY89S4wG-eeNXj6U_nwILCL-O8ciC-3l37IHnZFQvx8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459153","apath":"\/ggg\/1\/1\/35.atom","pisa":"ggg;1\/1\/35","ac":{"\/ggg\/1\/1\/35.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/1\/35.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/1\/35"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/1\/35"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/1\/35.full?utm_source=twitterfeed\u0026utm_medium=twitter":true,"\/content\/1\/1\/35":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AhwAcZBIMmkRxd6ksaMyspaGsIOzznprcMqpxMo4rfgeYOFIAt5cxGWH15qUTiCr2uDhWRhiAMfXtUDyLyLjH06Li0fvmhKbuo48c2iswbqLKriOF4DFu2pu9Ohbj4tn0WtHbH1aHEJoRYs+gNFcQm+HJZ+F\\\\\\\/3bJHS1pxVxq4eLQfXFUgmdmNLy5AKPf9bVWlqEHyAVO3S28kWSc+wU+Hes\\\\\\\/8mDjVzDlrAsbRL+loSsW5as33fO26Pcu+yluwCs3tFI\\\\\\\/YnEorXxFT\\\\\\\/D9bXflhct3fV3jd9j4zCYJq5bkjmd7+C54wmD946XOFzxkzOmDFqt+68zjAXmWqRnUJFtRM2nsE84\\\\\\\/wWlSiesTQWELQorza84ybkWtrQRs184Hz2p5SKtVcclStOKCVnfJFgwSD6S+gyb2S7L9Aks7rFM0tOwBnVBHvXDixyUEkcAXk3QgiDGokQFTyogzLa5vm9trv1jVz9jN++10JQDjICEpSSzgTFgCxhXQzDHmH4iAeZaUs9mnCNWNFt6G7OPqLSuvNDS4BQh\\\\\\\/tuDB9zn9cJnxn+gkTQ47CT\\\\\\\/RyGoI0Gam3X24nANW\\\\\\\/kTmaB35SCuKiuYsUs2eRtyd7KjwJHtwlTVia1RfGS78MSZmhmWQgkFhBZkTlLKc+a\\\\\\\/fcuMnDMIJKDN76t5pM+D0AvEIgzBxTnjHlaNYveSHX90hWodOEr8tznKgYn0yllRcq9qhiS4lnUmsGpXVXY1ZnjXU+p5USgBdwlSIrRH0HYgu9X8NpJ2afGxByk22YPvJgfEo\\\\\\\/4iCBkIZCyGCtuo7Dv61GG2p1SECIato2Dh47+DgcEjJZLusf9uUbwk89XYMKh2CKGDpELmTnt4JaoGiyJ71pRwmv2f63ul8FptRxxjJAP0wckPus4LJI0sHiZQMsAL2kug8UbemDwDN6nFbqXSyi\\\\\\\/FAoM4WVDDQ44RwFp39Wsp4FUXp0jpa3qFR\\\\\\\/nsYcb7vHPwh0Y3n4v4nNO2KpB4t+2j90sbz4KZhfH\\\\\\\/ti3wUv9pIgx5QpOUeuowr7Jli5xEP3OZbzWrNhgNrrJ29XpQjWGie0R9RWIjzCkryR9bvmKvncoYx3ZC1jm8uhNVdD6lcy7Oeq+JN+2ixMa9bGM2BXqIFeqjA4OaJYYCmWF9LW90Xgl9Z8ZXcf1+ieBNv+\\\\\\\/ukBZfS\\\\\\\/loYuFSPmWwYDxpQ9iEqVGZbg94HGHJdjBFH+rGH1rRsCo2mbanDjrOPeeT9Xs59YCCAUGqsLwf4Ps8R140MUkuyBn7le9M2IT61jIRioQSdhdV7\\\u0022,\\\u0022iv\\\u0022:\\\u0022uw3wCV39lYt7OU1iqX5nmA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022757f508dfb5785261626498d6e8b7ff94322500d8fa26a29646f3a8a3db5754c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DHPvXyUcI0TJsivgknvDCkdNHNx8GQ5gx42sawMyywoLCC2Zrdp3izPBVwIhqM3lIlXtpQhEiKpufGlFRd5zNwi+sbdY73iMfYoKlT8Vj4qaivtV2vhrQe\\\\\\\/qBKKX3d8kovxANwVwfCpiorYNyLrR95n4kFACl6wfX3F2gUGRJUeKrML0f4\\\\\\\/Zz\\\\\\\/qC16n05OxWNaZxmAFY7mToMrBW\\\\\\\/9l\\\\\\\/P3UzyUy\\\\\\\/uYhLiUyacYCcnE2KrNpfEh03VyVn5aWX30nQVblmyet8olmrEDNn\\\\\\\/92EZ+05WdNFu7HiLfm3389MOaz1uVoybHox0ajHub\\\\\\\/kAx2Fg4oj3fJkORguVwRyq4RVdCNIOEpaGnkMQxQhoEsY9wu2Kt3E85KJwQB5CuOWWEk0t1kK4BpjpWdG3XbgH9lmCap5pxx1Vfl4JM6dzSszcqr87ymuiIbgwb\\\\\\\/qnLXnOdU9vIujvP0QTKNeDb\\\\\\\/568uZY849QdvEo7aVoTTutEb2psSw4viTCEu548vK306BLKYem02bbQG9KX500oOBlmL+ulTTbuK2ZNQCOh8PEZq0nj\\\\\\\/bStS1+P2ygh6+yKDMFUNwkzL8JT7i6bLnhv9fR4iOMqlUQO2B4fhJAF5Rml+8Rtj5wQ9yXZ0trRlZfiCuxB60HmP+jc1UJh255YsKGyF7PjOT1cWa8+DMUgXGUFA1JngA3zEhmWnDKmPk8qIktsp5tcifYUVomQv8rxSkmAttygqJIkqPyUQY\\\\\\\/RjCb7YRaWnVnhAP\\\\\\\/y4bR79APYj8O90dVZubcKEv14JJDWuUoO\\\\\\\/01wzCXCBRxsvvSUWS82hjqTTN5bHBR4+Tbk6DyJkrWDinuoQtEw\\\\\\\/nN9FiU5lt\\\\\\\/7Me8HLlNQETDUHvQeHzkfnRJoIki4DVkV8vOeYuU0z4jzperUZ+3c5ERCPkEVVAWmbiujyTnkLpFRwwydSdkJrUhVQou1EFzbbI5M7FimMtINgFqSId5ZBGQhPztCngvp4iMt+QqlG71mFA07m11LKEPdViGNi0fYvlrJSZix2jq3nDXO1jJbK69bDyl89kInUv3M60MkWCJhfqPNQv\\\\\\\/rn8WGCkkwrX8MnpPQXwRWnD15cdM87ldpaFUm9ft0TDLCARjtPseDVByYW4edXF0w5kPktdc6efsiU6hBheWQz8ECra\\\\\\\/gcgM1gChWgcZAozVYftjQKfD1Dd+yRl9dBc6Ha0T5aIEiP8isbnm4jk40hCMI6uvvWw7tX+MbD2DlBupTNMsDqvC+OiQScyXA1QbjKAcJ6TTTYhd+sV\\\\\\\/AQcl++Sr9EcQBsI4RxpZKA\\\\\\\/THOTVcLh9qTjMrbrd8csNbmNiEEcaXLfV1onk5NMT2gXx0QMVYYvbexWIhIx3FIRnvQnDuIodYVD6uDEApRsgUQC6Bv1uh\\\\\\\/OzDzwuzZL5mski0TNrRl0kyIagvwuXsfOrdPQQiO8LMLMOuquE\\\\\\\/SRORr9984P6F0lfdWkNU\\\\\\\/ky7KyhommOXfJvPym4RyZ2VU9GOUqMjhzo2hxOQmJuzK+6c1t6Gjtd4MW\\\\\\\/o6IwJqWISWcfTx2hDreRXUvQntcbSRY3oxViuBAV1yrtEf+F6Xe6bOCGoZUMeH+DRcWiM1Dya8tSQOIAje3tW\\\\\\\/ustMz+L\\\\\\\/2DopROl7ZUDf5MzdFymXEXVOhEMlterNjQaBSYxzN+ZVbHu6zY8AzvAbCAk84vJB0ETrw0F0uGW5fihEbOVoSeYKENeiioaf5iGUho07F063mUzsawoSexaB8zXhP9YuY2GutId93NXvYBjuEETpKvwFO7lP4IXnYP1wkRnIk1pvqzWqc\\\\\\\/Roeo7cXYnLSmLSJfKbq5PHlGP9q1iS70p+7flP1KrhO2i\\\\\\\/duUWG4P579ehljKNdnKyak7tydjBcJiYauLXdpvoEA2n7LJtH5aEdE0GQEEhjFIkPRFlRo6MK5yE7c9q8KpWRxH6yoxnpdcNGsyV9xZMyS\\\\\\\/1dPmUsCO6VePfxI3YjQ9WRLAxiLed5h5xTU51+E2aoLZZP+T5yNDn57RxCJHW5R\\\\\\\/f6aMK24GWrXqEm\\\\\\\/992T1eT4YD2WbU30NOr8kxQt7viG\\\\\\\/pPmd0poKtz6JvO91vWPe4RS1dRhoHwdzrqGn73prtYA1FAYQv+puZCp3iUlzq3xd\\\\\\\/xKcG79SDRQk9VCHHAS1VRWD26hpy83r2GF3llIBBCMper32XjDVF\\\\\\\/yKl1ZMcDU4DP0pbA9ZEmpXRbXnc87Zy\\\\\\\/6ppWdioicu0708KqJM7+Qh0NZfNUvQn6oCvGDOYm4CvMvtKOr4TpFes1LpXlqAqzla6q6emXnCyvZpWn8Th+4fT6QSQyb6rffiw8eZYjWOunhG0SCsPkeyAWLXET7rCh6TsuICUo5FWHdEtY5rCf4KR8KSNvfY5oxCuahyRkhG3RgdffjT1gKNQYDryfZPT4p2NkMe2XNUPhocZtyTInXRN2l2MGcCGXtF5vSGc9CqboNrdeBemZyJ5DCvfP+AtEt+6jRYFzf\\\\\\\/PF4vaNSRL7tPaZ\\\\\\\/n18+U9cVrhtm6OHwlmu7jhiuUKWZIXUOrVU\\\\\\\/cXlPZkKvs36zwhExLeynL0N1D8VptY4wjB40\\\\\\\/ms20iV\\\\\\\/hmHun2C2usw3dD34y5PL\\\\\\\/klGyD1Nk\\\\\\\/Qpz9\\\\\\\/iQRZAP6iqH8O0J4wA3\\\\\\\/KpZHaeln42jdaEFtNAN0xPhLzlU33OIRzKyCT8flzfQpEfZmJRTs2rWNmiua5Cula8Cfn\\\\\\\/V7pZJsFZypMHK4XCLSJf63Gy48wSXJIajLACRNLOZoRdVx5OZ4fAttQGkANgA98qONrntC7\\\\\\\/4a4Gdb7ulKSIfzOPQXt1YWhVTOsVgKrcWgQAd3FuReKl+zKCHzSOhQW7M7s2H63G\\\\\\\/7hyz3kI0qLg6iOUzZg9mF88BRG8+uZQnBZwBX+T3yqMrBUd\\\\\\\/l8onkap3i6DABXImkgDu5roU1uRjlqgAItFyrO7plKPcwf7vfyaS1LOSSA4ryoK5qB159yW0lZSuTEXE23ynz65u5AlV+VvPxVMChK6kqTu6w8YBbixewgRUn1yKqyHizevb47UYKLAobaXl1W70YP0GD4yqpexgFaw1pC6hJMYXVat70D1xv0MmGkdF5DhXdzRpupJj1v\\\\\\\/V5NNdeGyyiN8QjEhECcm+ksdIIfrGy9+p4ALVJZMfSpJ3+nXdM4Sv5hOg6WgcRS4clCfqIMEKAbAk6Zh\\\\\\\/wCBQmBNOkncCa757X3uE0DHtzUbVtKjAgP7ghDwsx4oIJQunPb+Xb3Ie3s1K2Yo67tEs7W3btD1omQNNKJqF4g9sn52BzikOsDuF58M9Ohc0yKhBJrGlGPk4RHrt3wniS\\\\\\\/syDCKx4osvMMFfJoWXI2f4gNrxgKzziqgXMXZgDL3gfrMZOPCW05\\\\\\\/PY8f9\\\\\\\/1USxW2Q+eTGmyghgiXhndAbsbJNTQEUYZtEY2cCfI3Z+H2ep1k+eo6BYxj2QXSb63emmMGR9d80ri96A16Owe5TdB5MTzEAXRaTHM7B+4b4jIqfEKf972WKadgz2JhSdJwz9gInjSKFSBckHRalUTuWpJArRZen9Szar2BckS4lA3EmZ5VIGtoTw2uQi+L\\\\\\\/X8ysFQmVPAGJ75bqYdMidDqoPWEhogEMCptnUGLOMXf3AzmeU45pUGct+KZ5HVqdNJGH8dPOdO7njzFeZHPKD3TfnyVMTEjr3qfkZbZgcCfhB7TJKtN5gxKMhJrg8h9GheKKryoYSDhH2FnrU00MoLuGC81hbjTGto\\\\\\\/W2m2ermM7ewLdYUNbd319tCenvNYC1wRsYLAficpXe9Ult6OOrLQXbSwsHAlTLahiHPOMNAmDhy1SKsQ9Rs+xsvyOn+neqoPjRKc5L58fouC4P5GQ\\\\\\\/sMRyYBTRZFCNCT5k8Yh6uD6e0gnq45y5nuZFDJfYgNfgO4BPqbEXtReTPCKDxHkm5tDByrL2iIOGH\\\\\\\/n6v0r\\\\\\\/LMh3Ru6IMmI8vCwvYknUNSWv16x3+7P+HHAFUhPq7UNy1EEg\\\\\\\/fq29R\\\\\\\/wSdscbMsfeHpXZlsn+Ay3Rcw5h6QSKOdJtbXvl0KF5HKx0+SO9Yxq2zny6Ui8ClgRyezyjAPge5m30ncK5bE5kPB4NpympxAcSk6iR5G8uY94ksaweGi8Og\\\\\\\/DN40sPs6ejV9UkzgA8iCf8DFugMtI+O8l\\\\\\\/6gQtQjHCguFqXpmvwxiwIwNKTXuMoCrzxZpJuq\\\\\\\/QjTyxkXaKPRLwiPsa2Q2E2F2Jry+erlDrFUo+n0PIcL458cZ1AL378aNInD8nTfm4OpH9Wqa8Tau5AAm2HxLbN\\\\\\\/z30axiBlFQqdyujk25v6poZtSSgHQ1+DxCxZ2B3StQ34Nl0gGqOcOxzk2BHPCmIOxEFG46\\\\\\\/wWQ2aZYhTVTTd0EHEbpFbfBTE6YkoANWP+hN8lb\\\\\\\/nmDWIt1W+I\\\\\\\/VqNekPfq3Sj5C2OriFfqolzWO\\\\\\\/1nsu8cNUxFJbTkrMirMlZJOiFt1Tk2jq6MgrEVYi7\\\\\\\/laz6BfujZRpY1Bs9CG39PzzddPm6brqEhXPsjakyvXwvFyuSe0HlS95F61K6EUAS9\\\\\\\/3CjJTmqYEoFKtJZ\\\\\\\/NNu4YFFfAo8ksvH6toDQYHcMMHiKtBAHR8kMdjsN4Ed4tNzcS9aBPOlA07ipNbZT\\\\\\\/d8rugaEg67CDDcQyjJALgX9NIe1hZAxZGxHZlNQoWlwMsn+xoxiZbKWLoYnYStXu9YAd\\\\\\\/v0Mly0VAzRmtVvCs+2p+RtgYermrdGp10wKSMcM4ZMgE6BUgBktAPMV9K+ksUcArEK+d69NWBNmAEFaqjoft1ZgaImNf5nKvZ+CZe5kwtMVB7aipDRUhbNCLzf\\\\\\\/yi2qWqbt3bPh+NFGFlCfZjBLKd7sskt7SaU3u2DJCmJFBAb2SywJ22Lg8AQpsIg+9qiqafVZ2zjnlHxsrK+XzpOoUpi1WLnojvt8o\\\\\\\/zayk711b3phas7+dwm3Bcbq5hdU23wDZsuPgX++x4wZmCF45oRC3VL6DBhjj8IoqSzxgElDK\\\\\\\/HuBz3h0YBNCaykc\\\\\\\/uIm5Wm5i76lYkNfF7WXG3B6wUmDcNukZoyVkQGx+ofTu+jA9FAg7zXQB+hugNVbZx\\\\\\\/888f337\\\\\\\/yvN8yMWGsuh\\\\\\\/qaujWHZfvApwu+U1kvG20Sqa9FWBQhqYmpK8VOH1o51yMnRyweO2\\\\\\\/6nUKqIYsJYPnTJdCGoqlHz791q50amcLMgcWDl4HX6Kqug+Jqmecpi9JZI6s\\\\\\\/rtY5IG8D+NPLRPTJdEjqEwxZs2WM7JCeDyQq+TnWArYE2OQgmJjqvjeSjVyPQB0ksDobdowTxQoRfjRurJWMVdrbrEI38wIerK50QnmU\\\\\\\/sNEUEmLaR7dKhqeE26hGuUbBYpA0AhQxZetaigropcpR7itJIWuO\\\\\\\/gte0LhpVsWHQQXa+TyrpH5C6pSQjCRSdxKIyZ1AE9DKn4Pc+JhPiR\\\\\\\/E7r93JIgelQuHvi\\\\\\\/6pNqzpGVisGPiAUYakUy\\\\\\\/PRpjTix23MS98LNfwD9EmQiYTFtoUhvMATUo+TwneKMhOxu6SQe4MAnvm\\\\\\\/9woB5Pvgcjd9Utr5t7DyU0p8\\\\\\\/4enGp\\\\\\\/IaXvosCrfqCHbl0c1iQXUlu21tVgZ4bjVIsIYMkaQojuaD5\\\\\\\/ddIC2+\\\\\\\/hNp7XGg63QpPnDOLi7rO5JkrN+b3P7nZF7YUGUIjO8ignWqHvue4+W5x3YtmYYK\\\\\\\/vX\\\\\\\/5GN2pjQlBu3si1BGRXojL2ENHW820LxizKOQN\\\\\\\/W9Q\\\\\\\/z8u3vS0grE3wL1sBV51ngX5c4X924ITEN7GUwu2q4yivlv\\\\\\\/H+R2KVVv2kBdlBhixhh9nSziAqgC\\\\\\\/d6PfU9oZp\\\\\\\/ts0TAJEMqHIhkBLboitw3nnqx3jC0lC0k86j8JpCRj1NWwjnTNZSTb902nr+mXGlUFJouLNnMdFpvpRV1IU3BmcSvO9BHhhjUQoUwVzrFyTaPAJPVorryKlNMYKHjIanyrmH3xX12E\\\\\\\/+HZQ3zCBDw4eQrGfcwczfP5rD6ikBx8Y2AdYinAJkOWSHmRxUuA4nV3TNIADbX2qrJDZwEU\\\\\\\/VM+7snnXVhGsAC7kHjUcy+5as+wDfvRcHQDg5cmo5xLWgW+kDnppZVGZ8JQ41t4pcaP4cSIjGm6QGp6lo\\\\\\\/fpntFkbFPKp83sNtRDFNWKhsbBWDvVSMFyEBtQN8k7NdS\\\\\\\/4051v7DFsw2zCrFcXzBTKmRXhpD44+zyDepLjBxfqVpxd4xjAPumw6S\\\\\\\/QtwmCRj+z\\\\\\\/aJaf1R89LGRlQKiccnGRkvK7RPP7ih4Gz83XQDSc6WKkOiCpQClUGB6X8EFz8h7xNLV+W\\\\\\\/D09ZYoEHdGao\\\\\\\/JZQjNuA\\\\\\\/BFeb6xWOoETY7d9dghMJelNEyMCnYsazGErGcDKzLVUsCSioY1dn51nkjcQ8Ly+J\\\\\\\/BU\\\\\\\/SaBWUahK26TCKbLVCj3qVjzr6OGvHCBnrgJBzi2dii9HdTOkucQSQamak8Jnzk3rATdNP9jNDB2FjTv2mBr4BSwXzfAP99sfuy3DvoCHSzyq989lEMutD73ZX4ZGz9Ib32dtOiV8KhUWzbWtY5+eyEqnQGZebdqnUVrgfkb6ZiaFSJ8ioBjz6bAjeS3swPpoMTPQt8avfzisviJJwGUgOEGtG7kjZkj6ZT+YY0RiHYZbvtQyNh6EvRgwf3MMgMlYLqmMJdo\\\\\\\/035fVrR0vorvAUfoIllXCqqaR06SsLOXjKb0Leq2r6chf4uSzBAr0fohiWkLtuLPpo4+5w2+Q5ATvFUiLGbN0iRtur7oIxlQSZXDFctbuyWoyOFljzORMvJbfU4g\\\\\\\/ThVGWANqiCXlEJTM8Cnjeix\\\\\\\/efaMmDnJOJXXcr6on2oMnrZTJrewb1PDOgaP2GAlMIkctjS2STVEff2ULeOMebxjIuOCdS5zuWcf+aZI9AU7QPXu1w0KCrtZfz9KUDN\\\\\\\/Unnzx8qrnWBi04HQulhV70Ec7z5dNs7smxXAjozKv+00GiIndhjQwhcpnicFwJ+oUk93h7C1ZSEaIJuMQBgUezO913\\\\\\\/rkYQQc6nTN8quiLayeJYgP4MQ+Ji8YnPwrYdDRnDVPWIRoZ6\\\\\\\/d8D57ktIK1z\\\\\\\/rS2ae9Nl6rwHAt7880giN\\\\\\\/viYv3D9+UIE7j4a1q6CJMvfqf6GLY7GkblyfJ00O9OOcER8YE69rqLwdT4hJwoEaBaoedzDlh3E5gbu\\\\\\\/GPWm9P6ANlfUiNEJRsh\\\\\\\/NFZ4AqJvpzRRvS0M+gghLVJ4KBlyCB+yvTb57hI7Q6+KouMl+UkTbyv0R\\\\\\\/ubc0WWBKZq1zkkWkUArOEKTS\\\\\\\/ah6mgyF0P0pYxah5G3CAtfLJtuOUANIF+0oe8h7JMszia4l6FelA5a692QM8csdQeECI35NNPb2nKKIccxmvNtHzEEYp7yUqeu3ET2wdGhzs8ALCTIS12IVHZkaE1Ab4Zip\\\\\\\/19ftZdQAUWOlMrCeE9ZlWzB95KjRF0juNTfP3TYXcgWeURKPK3HhceD76GPBzctwLoqtcBt0mczZecziGAvNMqyXTQfnJ33lte0R8H4PMJy9vTzQJUNySomvVhp4tfEYitY\\\\\\\/WB7HBbNkt2R2EZzriPmFy4fALuGYHmkk23XepKHqCAB0avyqGp48mP0kSa9W8HnHbwlgUHec7QHHltrqyyoYvxyhYoGhj3TSdVzqjhBP30Gv5QNoJPrvESvwdguDIzO6NG4A\\\\\\\/DJo6nTS+CooLbVWC0EzHWvodNO+p0Va\\\\\\\/Uo1VGePP0qqPwga75GNtzxJbqtmi5juxU+mKTlrKsM76Zp\\\\\\\/ndoQw0fSoyEN002FJrvyLIHzXFKuLGkrJs850sizYQqeyxl5e+pDhHbCNZ6fcjdyKAuCWmGwZUw2ZNkU7j9rFuyCHH\\\\\\\/g5AV4b7k5NLLzn5f\\\\\\\/1fJw6QdNc\\\\\\\/ONUxdTBI+Pgc2Cbg+BjjQl+cOBrvsHuWXgnUcul5ZCVSexo86Mmjt7LNtBnFQjNXYPa+Ik3RZND6k6959de7CTqoJe5ML0C8U3VZdy3DRenYozbO4WUhfRlKAkTnghcZ0hnWaXXCI+Pm7md5pmBt7TR\\\\\\\/PgkVZNiIJLH9FwD1+pg+t4WLNTVsAzfK10xBjUhbVJ96HLUGA0FedZmFrUnp60gHwlCv8yonSFe3ZD6iLxX6mj1HvmBqdLmlMH4ceU9vnVtAXLwArJZjC3Wc4Z\\\\\\\/hdhTOUgkyUugxKorOb8E8OIilLFaGMBSIpRb0a2UWLQtVI90+zBvO4JRBcpJamiDLTHisDmyt6TAG0yY8sZ31RZyYxiE2cLmm3X4wlDEXvh7Mfuy3litWpcTpkBWCu+LkgTMuakVIPMJcjFSsN3oBXlxBFzU62pqjfYzrsMGwGVyWwBOdlyWXiZA+HulK8P3EBguP8agVUUMuPheXdVHFhnoH42Dr8RJWvj3eYDCcJ3HKewQeKM6sdF42foVg8DTS2bhTkXsxc5RZbmGPck7AAyeF8CBY0gpi\\\\\\\/SO6yKq9kxyqVLYJ1qRmWNn2VFNz+875\\\\\\\/JAUJ6Tp233EvkYGRp6a9U8OXHr9P3ACtx0kHpAiA2ylJLwyApFV13zissKlym9IcLcNENekB4wD+jKBLVmSI5ZC1oux1hIg4Hx5nglMLu3CWC3OhmoYMocVTrE\\\\\\\/uO7u1rdrbh858SD4VLWQffpfPlodhGDgVDghibAaMBfFtCkL5nc\\\\\\\/quub5NK1JsChU9KcZRYKL4sflpl8ekHNQXQAHAEzupgOrUKWQnIRvqJUHU3CkkFlAPK12yWFRHQwQhVpbD51Hld0r6rIc+SEVpXv\\\\\\\/3U1ynUGOnzx5f6\\\\\\\/EEQ6oEskvGmo47CHy3CZ\\\\\\\/UZtQp+RZkiIv+Htz8+wHyIq0cleYcw5AunmyJZl15D6qRvYpp7oGfc1dSlVh6Qgu7i5aU80Ed1A9A0ODBO1RmVFCqC9hmfZgvwESCECaG0T1yLY\\\\\\\/mWbzIzIShWcj59bWF\\\\\\\/ypt0ms1WiVBxu01xDRWI9v50UwjK+Bnq58+knn5M240Z7AF3fTEMwZF3sz1VH0r+pcUTJJNK0+3IBFSAq1CW\\\\\\\/YI3\\\\\\\/Xr63gj4XrLfa73GRWIGyZFIaanN8tUtZ+4TTwgQhCNLj8rr4JLfbF2hVa9824NeYN34H8zM0xnQ5pdgY34ZjQ36mGjXtW\\\\\\\/TKxW2dBvMpX9GAGwirXwH+obSrKmZmXlggj1jYfg0DTj8wJimzROuWSfKOY977g9tLYsOw5r4IWYsP8Leg8xMESX4GcuFBGhr\\\\\\\/OnFIFsRcTylwSrIMhXqWkuGQKWQDO0FB\\\\\\\/7MdCzEPL5sgseOvdlOFfAfOlrP5g4Cd3PuST9H1a957Ki42ev4ita\\\\\\\/BRtHz5XMkH2K5TG84O0F1RonLoGmHJSi1x0oXknD8ZuCMv5\\\\\\\/ugSVM5JgvYx\\\\\\\/9zDHFibGVxmfLCWD5Mz5jY\\\\\\\/9BbhTBtgNtKfLeEhQIAaYpZ3Yd\\\\\\\/9i7GE5PnUKAKy6J+S6v014\\\\\\\/Eg84DaMmpFoSOgIROSyjAvXWIZr4vArNZJbOJoM8j2CI4kGZgnjRNkMWrY+e8wQys325SuM+08Jczn+xMrEVM3U21t5YKP8KLY4c1pebS9XoX5CoDy1GsMPORp0E\\\\\\\/YjJYT6m\\\\\\\/ZpUNa\\\\\\\/fvoLS59kKD9vwNpX30h8v5DITL3eoyKc5vOm5YrPtFc37txMwPsqEtPT9D7gmK276Mj0Ftz2FyhA15RzvD0wedPnJ83rrYW68Avr0jkYx1VZ8fwx\\\\\\\/k0Z3\\\\\\\/8G77+vy4mjB47\\\\\\\/a9lGA+Q3bgWMEI70Z5OmOJH+TOO9s5b4Tx99yPz2ExXc4P6pcqCSI78SYe43tPTTL8Q2OauyQSAIv9pqtMojJ4qpCw\\\\\\\/u5Iv+xBEhgCz81VSb6k4C+s+lq4MSKt9f4Pzm92ysvh8UKrUSKhLOSA\\\\\\\/krSdQje+q5\\\\\\\/sYCaS04yiUbtV5bvot8KjiuyWBWr7VSd8hj0SeWCAeHS6Vt92VJ99h3DYbC9cnVObiMlpcEOhnWMB8gNs0ZdwhOCuOBhxjx16j6KBGeVwny\\\\\\\/a4SjQHG08ixuBCrFtlivHiqmOFeJCJcMDoLW6248H1wc0Z3f1K9Qda\\\\\\\/3HJm84RxABcXrX56uVkqpf7b4JcwjY5ISsGd384VSA1sjNdpjT3IiGyvSTZt+dIz+v7JTkjp5o5Gw+fCNKoAhCCRVNU0O9HnEfsT3I1U\\\\\\\/7+0rvxDWpk9wB5NXCerxoC5OMef7H0+cmdzSUXUk3dbLuvAM4Kr9GdFixPuhKHBiLbrBPPgab+PAhxSHN9LjHtzWGR3a7MqFgdLzBxd4YAHTgYpw0d\\\\\\\/pGr3b6P5Gtlv785WB6Iy1lszSEXP8c5MQOcO2tRPL7phQZG1vKaQdJLmeQoybkrCgcqRKHimAddploOiO+yxcrNel3x\\\\\\\/Z12x3+I0HFU49mDDGeu+N5Wl4eLzIUpCajPgTZD5DuqEEpbF2QiPh+OqcGvK2mYJW4xeUIGGRbSO1xdtm9Joo8ISuATLTajtn6546MV9CFGOCbdl4UYKl+qXFsJwQPFVsa7MWua1bnjDRe9Lc4dwyxsMB1y3s+YEze1Kd0RgzCXaprXgTOyqDygsDz7Yq9RM+z7AwPAQTh2wp1yCXqka4o6ufWbSkOX1Je0zpvKYDazNKQvMvIPpa+7HRtkoXEuebDnbVxkyzOXN6lAVgPCNzdeCHOcR+PEyrI6hglXTZgptBM+2xRQOSO9i5ewSzObxuDrJ3HO2PUoQZ83yOwyeJc\\\\\\\/akN4O3laln6RAYDAvhCdbs8KCd9qyoRdDuoKJPE3\\\\\\\/ovTB5ma71W9vNtIwYiQ5COPnAkxQPCNM1F+Q5ASeVAdpFvDPlonR6HA05uZfyIa4Tp+\\\\\\\/U1yZl9AcAK5o8c1LxVCvjWYEi4lG7A4xmv1TB+GtCqDc7zHOdYtN6smvPoCmPDEtBTHEt5h7n4ix2H\\\\\\\/7n\\\\\\\/be6CxU7qCEHIRXOHa32WQS84ZIS3OkDhWZ6ortdwYH+x8x6oNJ6+OTvZ6laaX86iR0FClAyb9ch33V+W+w7EIOOqq7HUc0BgpUsFO3pBSbfRnoIVMx1TJAq9Vi0vY34Ilz1xJVP74vU5vPKmiqTm\\\\\\\/K\\\\\\\/q1ZLkvjxjd06gqNZFI\\\\\\\/U88B7TBxvCNFmHiuXrgiyw+hoWXF\\\\\\\/6TLFGMJESHj345BmnfwQWWMQJxV7ReaN+oQIBF1HTn0GrbeKvntYyJ1NTqwCsqxbpXfBsFNyV4pxwD1gAGayHPdGBHt9hUgmBg7vOxyTHCcKl78tfK641n1oKMI\\\\\\\/vZELozpA+dE3410FM2O+SeQVfkuGaZ+ug58MI3FBs2TGsmmO3UrmwsKbTXZ1oF8byBkvDqfyT5ZUFw0Hx3Y0WNn7H38BIe+SRTwV64A78seCn7h5Gn+2nfaLHTRD7UjIO\\\\\\\/cUHBf7u8oiefqfmN1NON5xbzGHtXMCZyJ\\\\\\\/bMtXDWzeTvA7XvMeIpoA3U4wyfolzzTQ69NcmgABcC7yy6yMScj+2m2XM5qWj7EYM4HRssnl6taIHVaOIn6ZpunEloAikg+cQmNzsAU0MT35CKJNi8SLRwT4pYCzNyKlUVXYzmuhSLjKxSxhNc83WnCjlATgNKCRmhZWfsL\\\\\\\/bMH5kSROviaeMpBlTZl1RIjB5Vc6lFDkl9leCHJpuuJgBlP0q3U7jUKpf2qFZRF8gvwulbIMQYvpQeSeBd1W1F0A8deUYH76FNJqAVR5HAhdFLidBcj+2jUkMvK5dYDtAGzDqGj675JMG17uLhYfw7u9yiA1ih4gL2pas2iJBD1MuVcieN6qGEdC9OoGp7cyVA0aDIW06tujNLkJY\\\\\\\/mFwme\\\\\\\/eE\\\\\\\/f5yF0ZS1ab\\\\\\\/B3\\\\\\\/6SJKeu7JhMGgElTheTkAa3EndAVN6Xqql8hTfrsyb6dJ5GPCBZBIFDn5mA4Tmr3QO2TnHR1wt\\\\\\\/dUQ\\\\\\\/nlpusvREAqxSNblg1MhqhcEnDowG3Zb27Z+Lu6vlBcXT+sSpbDt3dcZBTCY+4+LCLZWrsFfBBtBYwhOO38l76wLOxujCWAt+6NIpL0j8hhnEBmouLjvrivwexf8TiTmmU76Vtrbe2zClKLgncLF2F3i1HYd7zFF\\\\\\\/7fneFwfMJ+EypjizLf6h0Ir8h8g0sU9IuN963RXzGuPScUQOMssBISAfilAKE5IQlY+pV34+2mbxvx9rnRb1nlPvb72Fppg2qSp0TtuzJkiPg\\\\\\\/G3EYME3uil+KCXIELQW829cuTm3HhGHMfO\\\\\\\/DCcHYTQkdKTZc7UFFkwIdZGEyGNVYkEFZUWE85VuL1jU+K708ohVAT72FGrJL+O+eNWe4DHyKNOnTDG8aFsUvBgiwUINTYGUOyYHd6epClu04p\\\\\\\/U6j3B3YG1zhwMqCFfIJBOMPQrTji8KpaH2nahSfzdqa6StHnLzcv2QbED6dq1fdsxpANZtlv9wQr6dGCjTjgS5qrp5LiT7ibLGiCX\\\\\\\/ZV4H3DOre\\\\\\\/\\\\\\\/YLkqtzj8HyIGfi7PcF1\\\\\\\/yFBDXvE1iNY4FaxhhqcqfOSFG63Sbbnv6wB\\\\\\\/msnI4TXF1wtU\\\\\\\/Z13VWj8o1k1\\\\\\\/Eqw7SbinzAttd+2F2YDErG56GAvd7kddHzhb2HfAoHajH9GsxZkgLWCQ0faCV6jLB9I8ktp71YlM3Tv\\\\\\\/AhXtk5tKMeRgyQjKiGm8NwwW1luvV+ZmWooiCGQjJ+lYqvu+C0iOJ2B9EaLsK++cdWjxbTesWsTOjAfNKaZsgVXiFadqtw7fhE9DDKpdGglB2V0P6WsqeBEtT6iZ7wHZfEH2YbPLIrl3SOsK1S8xwNqvvR1tiNtXBRmsmABKIH+84G\\\\\\\/E0oQs0FW\\\\\\\/n7OZtqL5ghc7YYlSHNU1xnZs2l9GUtH9iWmyrEcXyNA1TRjj+ya5CHDdrt0bjuhvijp63sdOqNUGnlC5zGDV2M6prjS+A\\\\\\\/OpnjpPsNHPxIz7Cwo4bqE15z\\\\\\\/lfZjWcqPH\\\\\\\/PWcypDiWFH+ZC108lMgFphOzrM9iy0pIFzTScjo8BIlByakysIT6U5BgHZSOR9I57UwdGmyvjxWGlqSMJHltb6I1\\\\\\\/Y6Fn53uwEIqBlDfZp99EDKPKmnvrww62he5eTKV3vAln3YFqQAcEYYdEGH7YENNoCgipCNXcSgGUuIzpeYZnFUcJGQ1lKLq2bzwnTGa1FZpSzgsucdr+JYVQGx7E0hdvxLn86o0u2NYRAA8A5Cvya3hTk1+5SFwGUL30cm4t4WubZ24r8WxkiEDOQTk3C0Kklv3pviAwRyLdPeJTa6oOocI\\\\\\\/1sVkRVeHyzA0XcIbCSK7fNaGykDQ0e+yCU5Z2+ThWVB7ae4ZWoRzYrG0Xv\\\\\\\/TBI3Irn9MQv\\\\\\\/O8oVe23h\\\\\\\/vEKcM+mUbODp2LMmHGXDNG5ANTDcjg+LOYfnRFaf8ufdgu4YpD4s6Ga5UacpOeJWSH9N\\\\\\\/4ItllldxYRHA+QT7Z+jScvUgagdWQoh2BwQZqSCex39prmbDhqm7snbEsfin8Sb02NEfdvpATEf+bhDqg3jtpiBgBZvrg6LKBcMDZVoodZxe82RxXVR1l31D0LgObrWixtq8IpYS5cQF691rQTqVrOm\\\\\\\/OWVqAhtsyZ6gMDobMj8lZ8bzQm3pHAtqqaSOM8JbNtfXpqTzanzuElC7PegkI\\\\\\\/HQ6G5AmgVp\\\\\\\/Dt3o7nKzCKbInkhJk7XtJnSoQ0wvC2ZaPiFPssJ0LxP0lk6V27OvHPj\\\\\\\/UaR+4I07bw79A3Sc3Q1w5S2jwNQoybhYiQ9mdGFnxDufC2tBp+K5b\\\\\\\/7\\\\\\\/G109aAdF3JE7QN3IMsmrHduHo\\\\\\\/antpC\\\\\\\/YQQHAWJSCg+dRbaOliTwr5R66FP5piQ96tOt8bcAfgBRu1Fu1ApGs3bcjvwE2NCGZ1+nVRkaGEaABjvTPWXCmgMnCfAypsUtwwh69Uj5ywquzSgAd7xtm\\\\\\\/MNc0b6+2N+FQC0JlvtItgyldUf4gGG9YHujRKQSAttdH1qylCg4wqEbHEUspEiw8x4d+3psnDiAlMALoMSnnDW+qM\\\\\\\/EgNNQK6ZLGOH4mTu6KTeaSeIh1vy7d+rkVZL\\\\\\\/74h5qBSfsZblfvXbWL4df6E1IE0Qt8yXqk1BU4K7dDTDu2YIrd59G8WN7MmEIV+vGGX2zOySmJwB2s\\\\\\\/W2aIbn0NucMTzzKhP2mWs3or+MOFGzGb\\\\\\\/3x8QpJqNZwDjbJpwFdijCPeOKeKybo8w5goWObn4BJ\\\\\\\/myM++zrRoXfunIPOWk4ocnx7vb\\\\\\\/n7JDcVlCnmcNUR7PZdfVMeJn25wxe+gbkSyDdc6tFNe+aYk+LIryP2NSTd7J8Jlgi8q5EOhQYxlUF2FuFk0Z\\\\\\\/3iCZar5IFtufO5iD\\\\\\\/GIb57Z5\\\\\\\/67E1Y8w9aOwe4Uj51EcVzvRvc5vmYn2sLGC0l0v8w7s3A5pdIeanOTveYP\\\\\\\/oIMkMi7u9Whg7VRcmDH2nPwkst5fIDb86R7i7nXiT2fRIlpVXvIh6SUdgJxwV+JiMOWEG+OYW0XxnmwEkqWJ+rKf\\\\\\\/rwH\\\\\\\/FHTdKJbTkDEraGeVkJeMF7cINUVijFohTydODxAlOCSPKA2DNVIIqaf6\\\\\\\/wzyBcq4bCnWOIXIB0Rl0SxkuuDHXOQF7D6fCaTM1Z6fyWeLg6dfhab9JbzEW601TdBErbRqebayehK1lH35qp6cFgVlU\\\\\\\/MiZFbyKJr6tFnSKAmrLeqMWuBvlbib1a+dcwruh3s0PwUBhLMRTxsrCDPeUzE51ZGmvq0SRpClAvbxaSil4ChO64eu5YVpwEcntAIG\\\\\\\/HtWHETugIqUTb\\\\\\\/yJxrd0ZJjZkSXmH\\\\\\\/6fQ2EYW2M3fin+wpsbv5ub2dqEBFXpoqP4jqI7EgnDUzZk5WCBLkQspgGVdvKoR+AUFpnZISjIG4lvPqzpDO6gnHdL79KXR+TjkCqUG7Td6GxJTJNnT0Y5H0cw1Vg44ptsS8A5tiuDXJYpE+PEbOlrsR5SoJDnneA5Shl4OOFJqOtpJzH5VZ16VZEQ+asy\\\\\\\/Tv4emsSTv9SWgggIeGR8IAme5C8imn7BT0fSEKyIXLV4Ff7jV3PmRyT+2MNap8MBEgE91oc2T7dUBsWDeLCa8q4cIKhBr5S3cAUj0ifS3OLnQpW6ohxJzo+e+W5zwnmzPc3XexvP4Kt5RtUW3Sxhc5DiSGUseT0ixcFxf32cXGu3dg+iP6MzyY6d2VGyDEn3mPqW6nwpTPsJTfiyOujAcHAAvz6LWHKGF9YnS3ro2zKB4st9Jdftbs5tBElm6YtQYvyeCtoUijD7+7s1xfPUaveDHLWb+ORB07J98zfYkEJs+1RFeb1QA9ly4MflXHJ49W+RbGLeMPVNBluixvNHnyQKqsudd7T2ddP1X3cZPMLoOQMQ1T2f9Fp6q3wADXX1iysdLxt1K7gCeWFBwZeqZ6wOn43SPSqynjn4girBksd0PZ6QWTIE4cn65lOKOkFIisUMwZCE1Qsgsp2\\\\\\\/wkeLxGAB90LyANGDWzUJ5FjnH4RhEQXseBxA5oaVfLcda9XhWXyZuVPV5gIM8mATF158cHye1mIaGvbP3NGJ89cQA5enO\\\\\\\/4YCBDxWBZLZuglly3Kvx5T1VlfhUw2FPOIgamGwfnDK44S0Xk32PXEWT\\\\\\\/bdL\\\\\\\/V+8iSL7mTz1RnOYpJpmkZUwRF6BZ28eXTqM0UzK\\\\\\\/9lWMl9NQmMEWM3EJ0daDccmx\\\\\\\/D2s2+IuLeci+eB5BpcJqPeFUgwXxvCSB5cyA9UybGnPd3rDcQJ5UO6LzII2ZWpc\\\\\\\/B3sUrOGfHXRuxysQI0msV1SZGFgbKQaF8aDGuxbBHm6bTlhqj0Wgt7YiQPIT1vjlHNGT91ADuS+TPzUTfuLhjBygIi8BdJ8MUzYUgZGzwN5BTFakmtR8GWq5ebbD3AdURIl3r1VqbMaApWYe+82vF0jt0e5slhpLXZOLfWHdO+T1bXSSK+fMdeixCkBeA8gWYFdxQEK5HpHcrzolDR6sJIcFgdI8f3Xa68TyLnG6Oq\\\\\\\/pkpAArl0+MBtO9ajaFu2LtB4G5HpQ0o8aQRluKCz+v+ca7n8z84SJUdmbRxtFSodjvUbWMsfcyekv\\\\\\\/k7DvcGlf4eNIGjFb753TuBR4MYhgQoK8O8sbxVt+86J5tx2k5s4l\\\\\\\/yeyP1xBVIEK4a6+PUria1hSw+AVvHtJ\\\\\\\/7Z8uY5crsPsORO9wf1NgWIFneES2uWbi0fLa32UW7RLqNrMsVMuHsqtsw0fmB\\\\\\\/fFno9AMgvs3um0VwlJx3gqJyU\\\\\\\/oUKRVfHCT5upc2w0pOTeTqFCls4qjeK4UQumEco7GZMnepy+sESAzMvkfW1u8vhW7xhNGvHzLbhz72\\\\\\\/YrKI\\\\\\\/ATuhOoKIVLzEtemffC+5eZbEzSq0jyy4iDP+4\\\\\\\/OV\\\\\\\/V9YjKnIdj5fllZ4IhAwwLEX7U+sBVKOfI1EJY7XJBxJBWL\\\\\\\/MpF+fGDPn\\\\\\\/TEuCiNE24v2QZDVR4EkHeJADIje9eUOwk\\\\\\\/TeAUNBZxX14vKT4dXlagyIsQUUaw624r49apuxBrqCPoPY3hJnZTAtOWd5YMfk45q8Ip4C\\\\\\\/fEOVr6p81+8swzO\\\\\\\/OUgxe6T80Ry4MfKxvBQausX3XwI3dNGBHuq2sRx3f6vhBe1pwXtP10nnrFNCtNgut9hcTmhZQePllo2qGjgD2ihKcRRg6a8JlFSy9SC3NpDIJ1dfuIO\\\\\\\/bnVZnqN2VJWcJbsyXIj\\\\\\\/aeTvpd1yNGuP9aA1TEnteZ3awgDw9ltb2rv7grGaa2pfGqAyqHz72IEZG3tLcX1eyt68w1dg5LAUkPUUgnUqDAnRixpl7zEXnun0fpBNS65A5UhA7dSrOGzrDZ7pfACvZrSUMMqXzJ7cf2qolWjf89xnYzqA9D5GfRpyOHdn1Vx5C7bYYDkGvRKvb1jDqmqriOpEk0Mzb1T\\\\\\\/Yz7JSpaCgRMbRCDF945h5e4LPT32ZZMRSuGKsxDcNWOsR8PbG6x7j6kgNESa4aZxsZ67lmgwusYV7BVp8b6ylVcVJrUmA3FNM3D94QaUU1\\\\\\\/Ie+Bnf5qkHaJloc9YgeQm9hd3qo2l8qElitkvk1NkaAiulbN05OPRXugpNSEbDB\\\\\\\/wijj\\\\\\\/+9qnOcE6pTnTujl5xM0skgHOKWKzXWWynIfnWSkvfoP\\\\\\\/45bf2N7UFzmJSMWp05KbYtoVU7OWyDRRPzqKmSp0337zAEk3fsTTFOtfmbbtsOansJUF5pSZA2j1pFGkoN0z9+1u6P44telZ3Yx2oTcqDxOXcs0lPy2MQZ3Ax229LhTZ7JBuaFcZg1gT8HXLrVxaQ4\\\\\\\/IT30wp+Diym1JGjZkKvJxqxE8LOBSY\\\\\\\/0PYw9APfogOf6KhW9upM27HU8W0FtGbHo1EVGWONB0qPnhx3xfKXczi9US4XvGwWzDNDJYrsNaLET5HK0xnKWV1NEaI14T52dvIzMvKe4UM0Vt9jecO2zlOQ0CSZV4xr0bvnnNFk3S5hCFsJ6+LSFUKSzhSVhZ9S8q+KCr1k41cFP+xmn5oJdV\\\\\\\/IEX1n1f0SDou5uzjVIkuZjq\\\\\\\/F49rRFTQIE\\\\\\\/YVT\\\\\\\/ukCCBDLzZtohPSYWAVkRZWyYxpm2UpyWMo0q2KDHb0PC8LUulw3XR1Y05\\\\\\\/xtsPuHPb0NxvsmnDd3gOS\\\\\\\/AlTBWB6k5XxeTH7LN21jb+riIpQHIZabcIOfeeg+Xa4y1DZJAsiIPVZ3JsN6VhZROzuBLynDEBKd0aRSRi1DCorgZw7fS5i5Jlv9msLPmk9XrP73K5DdrHYDifmFVuzn3M7hcCqpCFquBjAPMmR7UBhNRGLjB0hv5AeiP+E82+B7GUcKN5vkStcbuYvDg+5E9BW4mvv6l8ri3UTXNkQkRscHieI42TYbwxVQQikMKVrlq3LmBo4Y165KBWSLeracjkgIh3oNlIHpcxDICOUpTRPmTOxsRUOqoTbm3FzWWDgUnCbBXinb6QrmySEZ653PrUujBzO9Iin2fmOSEehBjZ7TdFnNW\\\\\\\/11AOgiUkLywi1+GEfBOdPowAkTE49RXNGlruuTHiDL1L4ytkUAhqIQEZMlRSwSzf++MLTjV9LTdEcCCkCVYkDvMd+9DTrkMRk1UbzqiZHQDlt+6dOgqJfAcdheAV1lxdlLtez+qW2UpscbjWm4W0db3MR4Q9iWSf\\\\\\\/YmqtV5mV7v7OTMXc2jVPvoMNCISDbxjsWHQwbRAVSQvv5qOqEvQpqjQBvjfw2a55wU49hLz1XcvDYFbqmc1xjq2Mz7Wz0ADDen3GVss54iJ3hu7LoKHF3vLL5d3i4QHs24N6D6IG9ikti\\\\\\\/BhxYHbOqtMDzViPdj3PsrwDP8UwWifullygSTLpAF59Eaky551CMOhrydSkhrmR1My7EdSPYtQLVLlO6KsQVxFclvP1u5chPZqTuDrZZAcMLNJifEVJiUtaoc5diApxYwCnu1D2z\\\\\\\/0Ql2i9v4ZMFsH5EB1sD+9iEd77zXGRRttMzPJxlwfMOBq\\\\\\\/T1RrTXpZjS3fIEEfu4g1psxFsRp0iD445vIrxgjFp8scyAcsYy2uiV42DncCSpFMDJzsZ0wIBgeNdJnLR5UpzVwufBI5Jqd7BNbti4\\\\\\\/ZMXHNLVu3NlgTnoe8NaKxyLm2\\\\\\\/aIZjMLnNrijvoHzFAjhmmImHd3y1M4hGfNGypeALAxMtP+BiaXNzJkQ0EXwdT9MbvdbfX4bM39eexCZ0HkRBrI\\\\\\\/hXTxRJ3HD5MLSmCV7ZXxawAz9YrhF6Mmm6p1mhXmGGQrTwPV4VCvRvu8EB+e2ye5aIhf26Hav04xkyhQAEJgc63M39+V0dvPdddGgGr6FcTvUzWaYLwqGZD5vCb\\\\\\\/8FxR5qbdOaI\\\\\\\/skFjFBvVKrN7DnnT2OkLdOAfa3LI0FKIuAzbvaSqnV0K0ntldkEyCtwcDrGKYUkPQvGcMG8KaC4QZ5pIKk97U\\\\\\\/0MxWyZ1W2IhHlhMiCZNDhBoG1AvIP4+I6gomeJAI+7gnFzViZW8qgAHi5X1ev2Ocpn7doPTfChPs79n4eqW2+RF0BqrjaynfLFkPf4ji2hmL37UZuloqN\\\\\\\/vjMiCbIuilXFdijwNxRLCr1qBBdR08wv7G6FiwRMcXOOubmc17gBCL5RyAMfu42Fjs2S3A8GGxUIO7GnEUCs2TcOE1+FzAN\\\\\\\/OQtuXdoGir\\\\\\\/9yPkw+DVnjUjGGUiTrq84UMXiTRQL72fg8X1yMNSB1clJS40uX28uHwMIjE2gc6k9a7S4AD2kpZHiM2u9eUk4B4cTkvmO4dda8rFKYhGM9pMPCEcfCq+d4v1J36+Xy7LBJvSuXmyTS2s35OFZWb9tt1I8NmuUIDhSevX0xHnzWmLDWme8bmouZ0VhBEfRjQ95ZkkXFe83pxj6+k2twKR3ii2xSas9zjwpvME+LC4MvRmnezExh+gla4JNCoe3x0AlmCvxLUZ4zzbH5UoMQJ0MpxCuVVhdbvSLFng3h\\\\\\\/ZVStlKopYxRwT8UFwXLxqhN5\\\\\\\/vIORy5tydA5zLQBTDNmdodN25hQqRE21w+ZxKxhRP4rs1Wt\\\\\\\/V+G3fg1tdrbeI1aKF2C0VMkMg1BNCpjftdNz8UF7YA7AFM+4G21gizHTgy4XGFIiRk3SdIleH6m5sYzY7D6ETwWzLSEPMH1TKAsRpXJk1M5yiFoi9nR\\\\\\\/MtFXvQQuDWZI5AG52l5ug93TqNpiyW2F5101yszGIpLX29FOrO1KrChgXeDCujwgOsQpBNlhAmAvpCKUPdzEMQSeu8LJyLoP0oZ6KMHYCxL4il5Ar5A88fAax1N8GfM5DWv98sN4iKoFoXGemcQphJ0qpF3dx63fuNbSF4T7LZq56mu3sJSuzbWXWJoT3tGQOpblR6C9zTFedJRGFgsjPLFlzq9JsviBCgUg5sie\\\\\\\/6rytUqKjXNtkWdsHcOgFYydecZF\\\\\\\/IVS4ji\\\\\\\/P5a6\\\\\\\/0dZPB1kBfNC0HXkvJLW84EZLN3MzDkir31+sBcfQkE0Vf2uD7qZNV+AyU96ZnpoAExEMZSqRrN\\\\\\\/FyLcjP3hBfAZAcoB\\\\\\\/OQVb0yi6\\\\\\\/aJnXUqlBC7O8vSiX4Nkdn\\\\\\\/HVvdr2ooc43A3rGpsWLJflg3Hz4xziPZxNRZUNrqVG8HTGggLCmpqqKjT1NxD1dqkf6ihQzPtLAnuzb9SuJUFfTZLwCIeGSQiZO2W2iKbxC\\\\\\\/gMsnYGl38dKkvOy7sQvvcucCSgvtN7Grdh7iHIQpCDKYYGUw5Np2r29uSfY1PksGYyvNAGCMR4NhVKo6pgyuTz1CYrQczULlSapZv2rnzB\\\\\\\/knLlEwf3ZWm1LN+X8GBWsShuqZ0qssyADI6bwFwN46VzgRM4n3unjn27mBuPrBd4UZ2nenK54hT6pg6fw5aOpfdp0bxotLVjSYI4B3V2F+W4GvJpFKKgetLBVsznUAe5Ax4+v80sTuLH1tx0NaT93ZJPvzaJlmsmt894TFD4\\\\\\\/1215SchkuhIkqSsBg6ToZylv\\\\\\\/bQ5Nh3PC8a2VVD+qZAMRO\\\\\\\/UhEVJgSLg7M87d4kLSIxLlNUM6oyzeKw4YZBmnkAVGKPuwr33Q5bj4JyprMK5LNt8TlfaJlv9EGv1M9Vdop0qHv1P0TKGZwuW40cBheBATm28ubhxYwCKLM12dZNQLLSo2AoaJrKU01Bhm0lLnJ4qeX7q7xHt0YNjdX3ptr1\\\\\\\/XJDG59y7dFWFgcKC9TWBZ2jjObfFlFokOaj6GSjcuGNOc6iougO3pLG9XNNJ1UaK\\\\\\\/10UtsVAf95ugBuA9XnYjrdtv8N2XYk2Kpxzs349SDrsLSP4N7k6BeXPnpA4g3ORYjdcGpbel5HjrF0JyApYN3CPqQ2HzF5Lh6uPiZeAIpMo0LvU40j8B39w2O5j8KHW\\\\\\\/IVX+9PZ0mbiXPQlsrZSxz4BE4XwWNbJlm8UkJ5IiNv3JuaPmy9rqLtySjueDrJvB\\\\\\\/QIzWknm3qE6gYcBPjIOG5M1OplSQSWVOwQ9oPvSQWihvYvXWUAYeYJbVWzOPZGnQHlw6tYTwp2uBJAsyfNPAU1EgfKQhjSY2nRFI1g2SoaGXtAmQSOHM40BhQ+iCgtaj+jFdZLNJLv5v\\\\\\\/A3L+wsmAOnV2IT1erptH5SgYeyG6cvyR32GNijEgSGD\\\\\\\/QihsG5mf8frkyWQC7FXYFIMlGe8k+o8+qE+WL+oMOpI2jUN6PDICHtayM2FQ5MCYdEx7cnMFzdaruv+KDcLsfPgC8Qqs5OTtvcApetoVzS1liIaKL1o5akQBh57DxQzrU70NBFTcE\\\\\\\/Ixk3UyxgAIzyuHCCEa7htDxdeTJ9izkgrtJmN\\\\\\\/U22qTOCeQ5CuLers8PTyvJZ8sj0e5roU4PfXDZyBf1Q0yVE4TWuYKDAB9K1If9QekvQl8BD66lTtVyAihAdAaYMhzcghGy0XkVsR5mV5mJ3O99wrECrRvmNPzvGlzbk0AID68u7gJWEQyuD3Rl\\\\\\\/vxB0Od6QaRKW1Bh3rc1uTpkntvlRsXI\\\\\\\/wtl5n8qnCGd7CUxaWhcsIxvFFf5Jd7ps8O37dG9UnbbhluJhISzs11Uc5UqGgD\\\\\\\/vZi2xOkhUQGXBJ6C65wbUFnUOSOtFNLBXBmUlID8qqxuUoeAZ\\\\\\\/wVGdaxN09+m70skHDk7Pl\\\\\\\/nTotvIOfjihSyCwlMype1zObS3dEuq88Yci7g6SpDru6QiB3zmNb0RDehHVxW1OcpQ6p+8uVlTuRnHGurcAo6JzFEgdVWK3okgm5Uuixr7lRxurNq4jJtfzG0pUJBCyETI\\\\\\\/INac6SIFdKZAQCTmzrnaeJCONVYByYQ7uvaOXj0bSH+Rp84Ub7HSVqL242udWuS\\\\\\\/FH8pUgjr1eyJ2efzZQXXKLgS3g6axsbwVFxn4cCjqZz3BhlVk1kWAFyN0AP6UuhO2UaJJVzXNHkjLj5MBaB2h1CzSto8\\\\\\\/OUo28ywHnWFA9zv7mEZYoZe3sOHCBGJmOShlOq+pYQg312486OISthga2lwxPdRhfJQzomUleBFd3vJAHDoYRDmbpjAw35wU\\\\\\\/H1CptNFKb+ZI9RVU+lqCUKPEW5bqU+TyOBb\\\\\\\/rKIBOCv9VP1xAM09dNJkKJ+DUl79WEklj6zRsFYfHDSFjfWa0uTZh98qMwBhyEe68hMkKS0HSA1G6Q\\\\\\\/mq0qTiKxH+qRl45jNCIsxROzZmg+slN+4wvlArzeoQo9xQf95KfCeb12\\\\\\\/yvTgyTGSazEEVYg9AxkWRS+uY5cxcXov9fe02ZI6+10FNnS\\\\\\\/SvEsL5htyqpFdfNCnPPWybGugBolkkq6IsnDFcDhRwHbd8oaToJWAqSD3H9GBHjKpkQPQvAfBu\\\\\\\/Y0ezqSPHow3P982iD3\\\\\\\/kXprV9iVXy7UVkqMNQw1TI4iRDgMDPIk6rkEypA44Ba2IhymBuCEJufWbTSvqiPWOMQgsM+Tx7UDPMbht1+33FAfpZrt12lrdaSokZr+bXiDq9xPC8AFhfaOzWAzadCS6+3LlXd3J2omBY16jcy\\\\\\\/h8K5+YFjjvQhz9vgbQwlZ0jnCNTFaXFvn5KWCqnvONoXmzmkNzmMn+KstDj3tmm0S3z84tvJDZRCnaX\\\\\\\/fuQsJ7mso5ZqpDwL4RgvPvMC1L6A3JQbIF8yLlP7wIF9U2Byx05GEaHI7lQHWirTmyy0YBwdt+vnfh0zNbaTnj3PT0KQNdaA6GteTHPBoQSMM2zC\\\\\\\/1bHJMylp\\\\\\\/nCeesZTUuywzzzuVtM6N\\\\\\\/7LGrA1b8nxyWpROjFLswWZsKJktruBwIFr5i74eo1zoRDVg9S48kinbsX9JRXGcu1PkFCruJSMn\\\\\\\/7woSOrJLsVzoe0WCCRtnMo\\\\\\\/K9ns7VYwBXmtuq92HBaaJubv7jffzM9Q99aMi3enY6iXC2gxj+CfLOwC4stds3nLReONLKAd8jJpsyHWfTzM\\\\\\\/Lu7kKjAUqoYkyD4ZbQnH3ad1\\\\\\\/M58ahqK3aNotSUwQeMTPUKMgLGo1eF9kBBeHM\\\\\\\/gNb2fXNNMSyHFZxnHSMK+Cowm47GlQBSjQ+onqh78mjDXY2LmgLoT\\\\\\\/zi6XK3OCnZa28pi5f+ltLfglCmQ0T9J4OyFtd2y\\\\\\\/IcEfOrnwPUk0kB0IwOZRF3JfCBtBjVA2+GOHlvWmxhMNLScVT9BXCJ0pJjargeOVRKTNfmwrtzvDiNVOZqmSYzy9h3whXOOxHjI3k9kgndjHOxb0f42Dn61xXjWztPKFJTVPMxRBLgCJky\\\\\\\/t3rXg0VAy4IXtkPaiuni7AS6c2ZQg1zm54RGj+bSAYZWit2lydU8qwFSTBUKloQayD1qGDP0i7p0XywKHoerE67dE40elAdzZCnVWWLvwv7XQCj7or+H8scUmF4kSuJLFqjbC6+j9UUok9AZrHXjZZuG\\\\\\\/VznD7mtpBpQEMxx6z\\\\\\\/i0RjDRpCdFPEGwPwF3AlAABpMuCfBD4bEqCAHpzIgm5mA+PEAFFJgSRwT8TZ4ClSMqFybDzWQmgnYOr\\\\\\\/7RBBy84A992V\\\\\\\/0dpvBXSNyds582ufXU8yxvuEOARw+sMzNsu0NcZ6vA4Yr1lSvoBS6ovHZOwO6GlcoeZyrz1toYi0D2grK5Y41Kujd4Hjkq\\\\\\\/Hb8CuUBQ8uWwWctBZxVLkN\\\\\\\/vGdOLUBls2wGO2a4gQ7DeNdhJV4b8b2fEkUoqFhR5pt0aBdi8pF1uMWnX2zyT\\\\\\\/RAo8QiCbl8DIfO9cWpRJBD4dCNUwv4OKZnLGqywzGFtu9J4gS71vOKoDJpNEqsEUKMutP8xpT79TNdBAjD3S+1rG0LfqfQGXT3NNTywuK6+0k1ZzYNeroSgY6aOMtfWFxdlCvHke+I3Xp9YVOU9IxopYU9j6MZd32I6m8JklBmBLp11Y\\\\\\\/Oj+TCv83Ca6lBAb0YDwNCQs872S4ILr58y9MciYTaA12AUZeQtQ2KrcSHBHGI9IuaPqZTH4YuEMZbTpa5VhDhDEtQ+PO9xC4wk60lSYvx0wOWBATJW+wYA\\\\\\\/k5zVSi463CU6aV3rwPQqGLkRzDeyS6B5N6DzgowGvClVaesD2iMeEem9Fzpvx0QXi3guvULdVCb0D6AYegNm8t2D7ZUSDrxS3ndy8EmKxB\\\\\\\/lKp8KhEpSBD+4GmHY\\\\\\\/j3W+tXnegwX\\\\\\\/VKWaRxdYMc4+dhOeM782cKa6BOTC3YHIh6J8ywhQyTZRxgo1m7\\\\\\\/p7xnubu1SrTV5Ng2tMDiwEGEEh4PfUbeT5q9jsNK0RBG6paVodbOSzCosbffwyemrwtxGqtezLFHRZUc1gP9rX4JI7yv+UJDPb3NUS6m3xdMlgnYfFtgP0ReRttYvS36uQsbR+ZUcTX0M244GeWhqUz3OQx1EhtCXbFXL0xITNg9IqCWJuA7DfjsLG4m9WsWznKe\\\\\\\/dveUdMahfJ1hhcnHwXG0m7H+6tuKDf9B0nEXYTEPZYa0Zs4fcxYb35AJ2VXenKu4BMhiR2ODFTXWOYUfPiGkp7Ab8\\\\\\\/cZpWxzjBo845DVsIKGTNJThQj\\\\\\\/gYg5r3pGW5dpCx0RnOMe+r6sGTtocEUyewA4wyh\\\\\\\/ifpdv7j8zw7W\\\\\\\/gYeT56vFYi+0mAeoohVUNkolqbHiLecWFtzVgilJIRlB\\\\\\\/okpwbr9+3OZL0iOZVMLfQ2JfaFOwYFhfqU1rIgwrjakTpB9WwH0vhRpxLNdxFmaSt5Oh\\\\\\\/wyz0bJ8WZe5+iCttqGwFI\\\\\\\/MNvK\\\\\\\/ZJ9kgz8pYcZObPt5LdkuLMT8BRRdQX9lw4im4j2yN\\\\\\\/iWSH2vQGsZ8p2NOwomw9FrCXZfqYkjWDZ1b9HJOe8L6796kLodUuBO5KwAC0DOI199af2SSNzJ8y\\\\\\\/hA6NhUF+MI2y3tMbJhYVcNCvNvlU45HDhyDmXaH6k6jMAHrcDbWZLVDFKsWK1L70QajY8fZBc0KKh\\\\\\\/reYkrmS7zHlW2qSmiQyYsfNWtOa3asylh88GrTon1qwtATp1NtUZhslSkX272OySDo6oRfRCw+KMbliMIsY2aAvAm6xJ4Jf6q0kjwPrOLEytDXzLiESWTXTQEiMT+0xuBWiGpjIH+FomdUIO3yYhQD8znFS5Z89YNh95b\\\\\\\/GDzhze6xYSxgxLM5nY9KEGP7ARDnSXBL6OpSIL\\\\\\\/KrPpvhXAIRQjqCHuOrvhiOw9p8pjBhf+TbMEA3My54VABuDaUvegsF\\\\\\\/0dqFQuYSiNc3wrktB0fvat6WsMVxjBznZdrycb3DzgUy9rc0mlIz3By3fsv5I8t695DWBmNIBYFRPq1khWlbDXZUfXM5rJX9VYaTeKl5Q\\\\\\\/ieo4rwUvVj1R+aqwOmZCRu1dmc+UCYP6N+9suPjdyjY2WFks2XLIZ64+bwRESeQIJbDWtan3FbVK36gj8wRTCGYlhhqnpIigs8MezrqexOfDtZVn7s\\\\\\\/7B\\\\\\\/QbtV8j0w0C1FCmtPp31LFWxwNeLOVKIKYwSdonM3MF1Dl\\\\\\\/VjJ8PlI\\\\\\\/rjbN+oXwfTEzJL6ssQ+oF9scNAFNpeVcH\\\\\\\/j5fJNTch1cj15FuIHFJKEwwMK3TKgYEwp8JfL4a9kdgorNFyOiDjojxq6bRTg3Zyv\\\\\\\/T9t3Id\\\\\\\/dBwHHsU+zp22I+EdYMAEDQKdgVlF06nopUj88LZ8SWpjXDYVSzkgMe3yitQ2ZzK45QEg2ckqXLN3zmxKe1aSCCHaD9X9IoLrlLl7o2v4NcpoSutwElC2Qu6p1e43vuz4DjvWCVNL9+TUraESXRPr\\\\\\\/ZGq1FidhkiDOC3lp8PmqmgBr+l3wwgprVo8iNIf8RLI12Q9yHh\\\\\\\/W3hal8F1xiXUpddE8jj2UU4cAUokSnnNAO+41x9QYOGvrF6cfD1er4ZWVsPw6DEGsCO0CD2n3aIQ+JeVG34CGMXiUKe1BZ6JjnGaVMQ+yDThC930TvVC0EGLckoUdwNwc29maJu\\\\\\\/uV2gyX+9W7BoiD4PpSFJpffN8W+VJrFPv6jLzBM0BtJcUmaZv5WJ7sNpc8GeXLDyzsH4\\\\\\\/PlNwYY4xuuh7hQKK5ayN0xKAfb5mzhiIu1Lv4iXjQY3\\\\\\\/rew\\\\\\\/p3a81ICO32Hu5rkakNM9jtEcRaltSh0gHa84O1Apml\\\\\\\/JNCnZBT15C+ZrDYVqQxd9Iau5ngPG8oUNFK+yKvGFMm9A0F5d\\\\\\\/yLDlI06nwnGk3aSO5V2VcjezVjo8i9LOVcg4SAZ1hQCUveyTnwusfVsh4f1oF8bViwSAuTETbNnGxBbCHHhzABcJBnMbLhj6Apty9yToeVqM4uFoHbxocRKNafmQOe05ROW3bKmOVoZCaUruux8ZnKV7\\\\\\\/2hqSFczPo62K8vB1p4aD+GltzOUXmgMZnbo2dLPOlJfi1EV3hBUXFtGhAJuux4YMBuXgkWWd+OVrKWJrCoOCoPOVY3fLN53xSG9kag0sOGp7l7PNx1xytq9fvVNcwenlIhbns2s9poUbmfV0DiQkf\\\\\\\/cq\\\\\\\/xgwlftnlejrz22LlakF7bTtfKVnjeiU3gSWT7SYDbgbJs+84wMB5JiCBcAvbKciO+IBPKEIRJsvi70UkPByBT9SCLeKe9mfPvizBdqm62PjfGpUU+fd8MraZDnexOVF7RlwDsqRanTKZ+oWpsYMk6sSMwMHgDpNJ8QMBfVo1+fyFZq5dCurYiaBrBpwOkKRnqxFNaj4tvNW8P4nq72lOXzU1dMwbxd7eEPXCHN1H9dd8H4owtqda64BIJms+zCA5ADAydmw+klKnVp1Sa4YLc5upWE2pxHFKvnzsGm3g0slgd9TLuXH0A43s2xhDgE7hhfedp1fn\\\\\\\/4RKu7mIJqsBGFLq7j4rPNSwpA44bvrvP0P4ObeGG7R9S3RrleiFuQPMpt2fiVrSnYt1MFfrlQuLGY3e8TW63UwCkKq215UPkeaITCatftv4TZEeJA1KQ2fMR6c\\\\\\\/SGRDqvzrzRlg6LDWVnH8pVEcWjksxB5ZtfrYGofm2fxZZO\\\\\\\/oRTpGpR3D7qwIAyJCp2Crbs+\\\\\\\/HcaRtu\\\\\\\/cST4\\\\\\\/4tL+rZQgKooGZsjLz3b+\\\\\\\/rfHFxysPf1MzTLMQvGdjJPQtZI02zPlgaERv2xrWBwfvcmGvJm28Bvj5swNzfEZcH64RFKqF0xqO12iLg+AjAwqLjbyXjAcuO\\\\\\\/WjOFQzSjjtm6qWX3n5XQVFaWvryHYztbX94iii7\\\\\\\/QVNdvgu2pHC8mwos2+sfEG3iPpd3wlQnCrotA4Krh8rUm6e4\\\\\\\/MJ6FPPoA7XnDhp0Ql3\\\\\\\/0XkIpxDEmtiF6lxjcrnmY0XZPfub4OiGmyyavZz4AzZqbUVmUtQG89q7uYfEZ1htXedvOrWM658hj1nFXAbWVcB8tt2GPnIZJbo7KmHh1bylR0WGGOjpGLnwO4RuO7AsRunfPrADyXEpMViCEHNuuc5qxX5lcAyik7sT\\\\\\\/+F2Kp+OUuUZzniEkRZp8nu9TbkNfY4wjiiVWOwk\\\\\\\/e\\\\\\\/0\\\\\\\/6S\\\\\\\/u51FCKps8x6Zs++ZD0eh6rQP0JwtaSB4aW0rzoQad3kScyJt5HOITzNMeCazDnjHAFzgTWLMmplqbKq84Z47HLnA1fCtleRJbETGgOG8++xnnRM85KfCBuhOTh+Zs39CD9tn83FiymXURm9mJYSk561mSCOPvjD7PF69posMcYcDBWI1LYBRo3N0rPKTOK8rxZrQ0bKHOQb9IaMxrlSbrmQUeTmqc8OitVcxMhgBYPtq2KrZTDmaPWCHKRsXKt24V2wdIReoPGKJ4EMu9SGTyGI2k6g6VKJyGAWD9Ie9Lv1Y9UuXQ9zydeSppt1Loso\\\\\\\/xGDcI2SXM8JIiRSLwzVmARw5MirxZAhX5+Zm7tbnCLOLljrpg7VQDtIbvCp8nhBQyqfgvMY5mfgbpcItnJOr04X5XRhlfDWIs5EYsTt9czp2KJgKeSEd7F+sR9HhSz3gKB\\\\\\\/MbbeFXgXMbDrWaixxE7xUtAf17fxlClGPX8SPobz3zv6qtX98HzpqYMtQBPwPUNzDGCs4QEFE0urQBQfY5uyobFfvnsdnEG6ZpNVgdeBCN+bcRQFxJSK0IO9a9LmN4NDzq+8Lh5NLxOfc8NtXTFT3LIFDt2hVCsEZ4SYYY2WDsgz5rtSqVXCX4AVHKVOBEqM9gIs\\\\\\\/1tcr\\\\\\\/Uq769wBlrH9sZF2Pg+J\\\\\\\/Kh\\\\\\\/cWkDTUfNh\\\\\\\/0e\\\\\\\/6iYIzH2GdEchIcooxXAUYhTcht\\\\\\\/Dy3\\\\\\\/R\\\\\\\/jt+PWpV9eMKqxBHe5CB4Yt1g+gqvGhEWIUz1z+7P5dOlwbwTcPtgoOSoBS9IQnYGOQVeAT6cEFX0ZlE\\\\\\\/oQ37vq7dFsKTvb7NdXu5NWB6wm6lbZ3QiurUG4Gxg8jXW\\\\\\\/XmbWj2CIs\\\\\\\/dPQt4vj5ExlZXhLt048uiCu4JBBiT3eQ7BA0dOeLDxuMNpuJx3TN\\\\\\\/fU9DLoXApZfdA5OcPaqHNMVm5fEzrbR+n8d32u8LZvsixpvUkYST3aPgIX44kx4hruyiwA2OcHUMVlPoJhXPU6ilDKwB92oBAcd4t76ooTtCUZv\\\\\\\/Z6VQyPvEpnzH4BJsOR7DBls\\\\\\\/HbO\\\\\\\/taeWzn2+uNWhSxSP5HK2G7OeRmLwFP4JXivSDUzerPPWqa9WbMtr+P3DpSP0MPEtgivMCloT6gZ6DCSnuzv0w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022lJ80rC4StIr2x8C9cinTAw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b95522e570b5ffd6f3b4c7f372f476ab7181a89268b7732b859832365dcc047d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022djdhT4dBAhfjrZpzHE84Vxk75FRW91x3U7eN\\\\\\\/cWtdIIRgwLH1cxj8hjiZQ1JoDO2\\\\\\\/ALZPCyAyT46xrrKnNBzRvlx+sx1gCVtcYaKYMRB+YNTp7dI+80Fn3DV8vHJgYctC03xNMIceDurQXNJvCJGMFBT+77zriJhGoqGLO9AvhMykCS\\\\\\\/veXQoPv1S7baiG9a3g0QZ80MKi3lDxqBUBZhXMzLcHcGUUb\\\\\\\/q+fylTlzhPY69zDu485VgKRAlA5BLj4MMtOvZbC46FM2l13NAmu3mnVrB8nTXgl7gm0WS71m3tWtzu8KNCSpGIBe+UvcT5lPf12rE8aPdoCNh7Ji5g8hZ2IXNK\\\\\\\/TmFpdusY3LSg1QDuSQbsmhooN91+VIawbWgTRSa0PQ7DCSk24HBFLPubYRebirrkyerSOLlDhFyrTKm\\\\\\\/ZnHvAcOU+JMgaUz10PGriPldmlBBVK7bMupiSARopB2Ch+7eoOvvdyMU8JEsQegDRZBG3cjlBlweWyAdMWP11CMXp5mG2vuUbSJhMLCXRUA5YK4h5APXqV0oj5RRxzjdjWpHJMnPtIp1OeDFVZCnr8WksU2ob2mMnzzeb4H4A99oPWzBOJltb2LeRq2JBATzMhydZzV08gXrL4p0qcIbq8RtB7g3ylgga+euOGJQq33uF5jMm1gz\\\\\\\/dL9r+JVUc8yvwmgiPnSypjjNd79VbgK8Ii6MxiK1DTvyKdg1wkXpK43BViK5Rbwu1rj8mwjO4JbOuEHvMkquU0irosKFB6XXCivKgtUqqkGCbnGMA3hshAjCgqSZB+siYtrv2tEooijOzSBSWOwK4KbQukx1aHtaAQ9xMMgcN65XxdJ+0zscjDEmRZDObr5LaH0c119VLE3UM84SfO4QE2yXUp1Zn7gINtV0ldfDoKl9bMuyZ5zBHj7qPEbGXUhEOG9zqi+LSO8IyJVvC\\\\\\\/rjGi03zkcxIgqcvr2tZWfY64t8J10Cj9+IlJbNHJJljEY64AbQFkCF6tx+Hto4Ookf6ljzTly9f17+ZWvUyo\\\\\\\/Wc+Iidcc\\\\\\\/DBoRmGSMBOlIGk9BiaTgOkHElPeqUu5yRHNLPSnEtCHLM9KoXjOersuJwmNQQNBSdskUyEHoOmldTBs25vYTTytSel19xr778iJ8hqIlQoHVW8QxpR+HPmOlCZCJ0oYI7y7Re8ZvO2pAZBMAVoEi7aD\\\\\\\/p0zraBfBEhc5OwBSgcVnO6\\\\\\\/Jjtiv20DtpCN6+Hk7Oq5t9OufuKAYOVfRc3Q5HleoVB7iIIew9Ff29eK1bNdLNng7TQI8PzXmo5BtaQh1QfRu3mZItWls6Eoe2U3ai6XmJ6\\\\\\\/KbkRHSKrbjy+7284+OKMQi526NB0kp8wNBOMf6Pf7cj6opcp7Nnhb5VVG04jXbxAKseHLoLXMmFW3eZ\\\\\\\/r57pI9zXrp3j3z8wYMRc+HI5krCEGQ15\\\\\\\/\\\\\\\/8R7M5CjCSe7H3IVqPYaysOETka4S5f9rho8DaaHHdEEg6C69\\\\\\\/+JlGCS+YjWuPR56xqR8i7EDmulHEwkUeMwZvZFERH8cXmP\\\\\\\/D1HR+Z8oL\\\\\\\/ps88S4egB21QCEykPXhwlMh+S0pqhFPKquGm4XSAZpUUCcTnYDdqWwpofCPNXRs1Y6tbhBDflVIHmzM21nIIL3c8cUhSC+eL24Qziv9MYyvNEQ9SF0\\\\\\\/cNxHcM5P8UtIZti+x18zGpI3CqD45w0scipvOlOSAcgQzDXcjSWF\\\\\\\/LV5JXalvsP74JViDywjnUja6idp+6fuhqMyvI8iDV7fcekZNYCkuTFnjp0ymLIeJMAApJZdCu2H+RZW6lSKdMxrR2Zgl8X0wkJrsfv6hjVKMN5pJ5dZBsOS+C2HV6NF2q\\\\\\\/izkg1ga8Ckz9pYGEgBSjl83H2lgyTCLRpROlcyuPnei9OWwtWUHxgPDZF+NptLtjiM07lY2mVe0QAmGrqovqi21huKS6tj6tZEaUuslGNlbs4i2CK2K5sSlVS4qQmU1LSaTkE069FdeOnzabLl2021+Qolc6ZXWlyC8dT\\\\\\\/xT80rX2O2\\\\\\\/ZZHREc1eC7a05aBbR4Le2NyIyM7nv8GkGP0Sxod0GUKXh9DQfSPcOhm7tc4G81aJHVpwMoloAOM\\\\\\\/J6gY5W0fovT1PzXAbhNOCkPXERfNVMMV64mQmPkuQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022V6F+cMiEH2Aft73XiFKkSg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220b8df7ff0fcc2abe3953194720aa63395f4b7b1887877251ca6fe2c9891e88b0\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459153 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/1/35" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AQ8zPTQPli22nRmmz97Ahb8tom8V4UYYSy6u_Sg93-s" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/1/35" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nxgGexNADLj0n7hWkpb7wSOyQy7DerodoZxsQET6XGo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/1/27" title="Variability of Gene Expression After Polyhaploidization in Wheat (Triticum aestivum L.)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/1/43" title="Identification of Yeast Genes Involved in K+ Homeostasis: Loss of Membrane Traffic Genes Affects K+ Uptake" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459153" id="node459153" data-pisa="ggg;1/1/35" data-pisa-master="ggg;g3.111.000174" data-apath="/ggg/1/1/35.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inferring Haplotypes of Copy Number Variations From High-Throughput Data With Uncertainty</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mamoru Kato</span>, <span class="highwire-citation-author" data-delta="1">Seungtai Yoon</span>, <span class="highwire-citation-author" data-delta="2">Naoya Hosono</span>, <span class="highwire-citation-author" data-delta="3">Anthony Leotta</span>, <span class="highwire-citation-author" data-delta="4">Jonathan Sebat</span>, <span class="highwire-citation-author" data-delta="5">Tatsuhiko Tsunoda</span> and <span class="highwire-citation-author" data-delta="6">Michael Q. Zhang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">35-42;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000174 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459153" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mamoru Kato </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mamoru%2BKato%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kato%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMamoru%2BKato%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Seungtai Yoon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seungtai%2BYoon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yoon%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASeungtai%2BYoon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Naoya Hosono </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Genomic Medicine, RIKEN, Yokohama, Kanagawa 230-0045, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Naoya%2BHosono%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hosono%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANaoya%2BHosono%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Anthony Leotta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anthony%2BLeotta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leotta%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnthony%2BLeotta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jonathan Sebat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Psychiatry, University of California, San Diego, La Jolla, CA 92093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BSebat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sebat%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BSebat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Tatsuhiko Tsunoda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Genomic Medicine, RIKEN, Yokohama, Kanagawa 230-0045, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tatsuhiko%2BTsunoda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsunoda%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATatsuhiko%2BTsunoda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael Q. Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Bioinformatics Division, TNLIST/Department of Automation, Tsinghua University, Beijing, 100084, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BQ.%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20MQ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BQ.%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mzhang@cshl.edu" class="" data-icon-position="" data-hide-link-title="0">mzhang@cshl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/1/35" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459153" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459153/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/1/35.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459153" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459153/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/1/35.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459153" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459153/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/1/35.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459153" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459153/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/1/35.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459153" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459153/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Accurate information on haplotypes and diplotypes (haplotype pairs) is required for population-genetic analyses; however, microarrays do not provide data on a haplotype or diplotype at a copy number variation (CNV) locus; they only provide data on the total number of copies over a diplotype or an unphased sequence genotype (<em>e.g.</em>, AAB, unlike AB of single nucleotide polymorphism). Moreover, such copy numbers or genotypes are often incorrectly determined when microarray signal intensities derived from different copy numbers or genotypes are not clearly separated due to noise. Here we report an algorithm to infer CNV haplotypes and individuals’ diplotypes at multiple loci from noisy microarray data, utilizing the probability that a signal intensity may be derived from different underlying copy numbers or genotypes. Performing simulation studies based on known diplotypes and an error model obtained from real microarray data, we demonstrate that this probabilistic approach succeeds in accurate inference (error rate: 1–2%) from noisy data, whereas previous deterministic approaches failed (error rate: 12–18%). Applying this algorithm to real microarray data, we estimated haplotype frequencies and diplotypes in 1486 CNV regions for 100 individuals. Our algorithm will facilitate accurate population-genetic analyses and powerful disease association studies of CNVs.</p></div><span class="kwd-group-title">Keywords</span><ul class="kwd-group"><li class="kwd"><a href="/keyword/copy-number-variation" class="hw-term hw-article-keyword hw-article-keyword-copy-number-variation" rel="nofollow">copy number variation</a></li><li class="kwd"><a href="/keyword/em-algorithm" class="hw-term hw-article-keyword hw-article-keyword-em-algorithm" rel="nofollow">EM algorithm</a></li><li class="kwd"><a href="/keyword/haplotype-inference" class="hw-term hw-article-keyword hw-article-keyword-haplotype-inference" rel="nofollow">haplotype inference</a></li><li class="kwd"><a href="/keyword/phasing" class="hw-term hw-article-keyword hw-article-keyword-phasing" rel="nofollow">phasing</a></li></ul><p id="p-5">Large-scale studies employing high-throughput experimental technologies have recently revealed the genome-wide nature of copy number variations (CNVs) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Conrad <em>et al.</em> 2010</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Freeman <em>et al.</em> 2006</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Iafrate <em>et al.</em> 2004</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Kato <em>et al.</em> 2010</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">McCarroll <em>et al.</em> 2008</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Redon <em>et al.</em> 2006</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Sebat <em>et al.</em> 2004</a>), which are variations of the number of DNA segments &gt;1 kilobase in size, and it is estimated that CNVs occupy as much as 4–6% of the human genome. The importance of CNVs in phenotypic traits and disease susceptibility has been revealed in their associations with HIV infection, autoimmunity, autism, schizophrenia, and cancer (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Merikangas <em>et al.</em> 2009</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Sebat 2007</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Speleman <em>et al.</em> 2008</a>). Currently, high-throughput experimental technologies for CNVs cannot provide data on a haplotype or diplotype (haplotype pair) of CNVs; instead, they provide data on the total number of copies over a diplotype or an unphased sequence genotype (<em>e.g.</em>, AAB at a CNV locus, where A and B represent alleles) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Conrad and Hurles 2007</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kato <em>et al.</em> 2008a</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kato <em>et al.</em> 2008b</a>). This unknown state of haplotypes hinders precise population-genetic analyses such as analyses of allele frequencies, linkage disequilibrium (LD), and population differentiation, as well as the development of efficient strategies for disease-association studies (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">McCarroll and Altshuler 2007</a>).</p><p id="p-6">To obtain information on haplotypes, algorithms and computational tools have been developed to infer haplotypes from total copy numbers or unphased sequence genotypes (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Kato <em>et al.</em> 2008a</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Kato <em>et al.</em> 2008b</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Shindo <em>et al.</em> 2009</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Su <em>et al.</em> 2010</a>). In this regard, algorithms using similar procedures have also been developed to infer haplotypes from the total copy numbers or unphased genotypes of &gt;2 DNA copies in polyploid chromosomes (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Clark <em>et al.</em> 2004</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Neigenfind <em>et al.</em> 2008</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Su <em>et al.</em> 2008</a>). These algorithms are based on the implicit assumption that total copy numbers or unphased genotypes are measured with good accuracy. While reasonable for data from target-specific assays such as quantitative PCR and the RETINA technique (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hosono <em>et al.</em> 2008</a>), as was previously demonstrated (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hosono <em>et al.</em> 2009</a>), it is not always true for data from high-throughput genome-wide platforms such as microarrays. Microarray data are noisy overall (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">McCarroll and Altshuler 2007</a>) and replicate experiments to correct these inaccuracies often difficult to repeat. Noise in microarrays causes uncertainty in the determination of total copy numbers or unphased genotypes. For example, when greater noise is present in a signal intensity deriving from a total copy number of 1, the observed measurement may get closer to a signal intensity expected from a total copy number of two rather than from one copy. Even though there might be a considerable degree of likelihood of the observed measurement deriving from a single copy, one might discard this possibility and errantly conclude two copies were present. Since previous tools cannot handle multiple possibilities with likelihood values, the use of wrongly determined total copy numbers as the input would lead to incorrect haplotype inference.</p><p id="p-7">To overcome this problem, we developed an algorithm that handles such likelihood values for CNV haplotype inference. This algorithm is designed as a post-process to be performed after a calling method, such as Birdsuite (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Korn <em>et al.</em> 2008</a>), determines the boundaries of CNVs and computes the parameters of a probabilistic model that determines the most probable total copy numbers or unphased genotypes in its framework. Likelihood values as inputs of our algorithm are given through such a probabilistic model. We implemented this algorithm in a computational tool called <em>CNVphaserPro</em> (<span class="underline">CNV</span> <span class="underline">phasing</span> tool using <span class="underline">pro</span>babilistic information). This tool can infer haplotypes composed of not only integer copy numbers (ICNs) but also single nucleotide variations in a CNV (SNVCs) (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Kato <em>et al.</em> 2008a</a>) and SNPs, whose relationship is illustrated in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. As a by-product, this tool can also estimate individual’s diplotypes. We performed simulation studies based on known haplotypes and real microarray noise, demonstrating that this tool successfully inferred haplotypes and diplotypes from noisy microarray data, and that the tool even had an ability to correct total copy numbers and unphased genotypes that are wrongly determined due to noise. Encouraged by the success in simulation studies, we applied the tool to real microarray data for individuals of European descent from Utah (CEU) and obtained the estimation of haplotype frequencies and individuals’ diplotypes in 1486 CNV regions along the human genome.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/1/35/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustration of ICN, SNP, and SNVC sites. (Upper part) &#x201C;ICN&#x201D;, &#x201C;SNP&#x201D;, and &#x201C;SNVC&#x201D; represent integer copy number, single nucleotide polymorphism, and single nucleotide variation in a CNV, respectively. A &#x201C;copy unit&#x201D; represents the unit of DNA sequence that is duplicated in a CNV region (Kato et al. 2008a). In this illustration, most invariant bases in copy units are omitted for the purpose of simplicity. Lower part: High-throughput experimental technologies give data on the total number of copies over a diplotype or an unphased sequence genotype." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-425633437" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Illustration of ICN, SNP, and SNVC sites. (Upper part) &#x201C;ICN&#x201D;, &#x201C;SNP&#x201D;, and &#x201C;SNVC&#x201D; represent integer copy number, single nucleotide polymorphism, and single nucleotide variation in a CNV, respectively. A &#x201C;copy unit&#x201D; represents the unit of DNA sequence that is duplicated in a CNV region (Kato &lt;em&gt;et al.&lt;/em&gt; 2008a). In this illustration, most invariant bases in copy units are omitted for the purpose of simplicity. Lower part: High-throughput experimental technologies give data on the total number of copies over a diplotype or an unphased sequence genotype.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/1/35/F1.medium.gif" width="440" height="286"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/1/35/F1.medium.gif" width="440" height="286"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/1/35/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/1/35/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460717" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-8" class="first-child">Illustration of ICN, SNP, and SNVC sites. (Upper part) “ICN”, “SNP”, and “SNVC” represent integer copy number, single nucleotide polymorphism, and single nucleotide variation in a CNV, respectively. A “copy unit” represents the unit of DNA sequence that is duplicated in a CNV region (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Kato <em>et al.</em> 2008a</a>). In this illustration, most invariant bases in copy units are omitted for the purpose of simplicity. Lower part: High-throughput experimental technologies give data on the total number of copies over a diplotype or an unphased sequence genotype.</p><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>The algorithm</h3><p id="p-9">The aim of the algorithm is to estimate the frequencies of haplotypes from a group of total copy numbers and/or unphased genotypes with likelihood values for randomly sampled individuals. Here we call a total copy number the total number of allelic copies over a diplotype (a pair of haplotypes) at an ICN site. For example, allelic copy number 1 in one haplotype and allelic copy number 2 in the other haplotype at an ICN site result in the total copy number of 3. We call an unphased genotype a nucleotide sequence for which the phase is unknown but the total number of each allelic nucleotide base over a diplotype at an SNVC site is known. For example, regarding the unphased genotype AAG at an SNVC site, the phase is unknown (it may be A in one haplotype and both A and G in the other haplotype, or both A and A in one haplotype and G in the other haplotype, or another) but the total numbers of the alleles A and G are known as two and one, respectively (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-10">For that aim, we combined an algorithm of Kang <em>et al.</em> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Kang <em>et al.</em> 2004</a>) with the algorithms of CNVphaser (<a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Kato <em>et al.</em> 2008a</a>) and MOCSphaser (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Kato <em>et al.</em> 2008b</a>) so that the proposed algorithm could incorporate likelihood values for total copy numbers and unphased genotypes into the framework of the expectation-maximization (EM) algorithm for CNV phasing. The algorithm by Kang <em>et al.</em>, which is called the GenoSpectrum-EM algorithm, uses the EM algorithm that considers likelihood values for SNP genotypes to infer SNP haplotypes, not CNV haplotypes. CNVphaser and MOCSphaser use the EM algorithm to infer CNV haplotypes from total copy numbers and unphased SNVC genotypes, but do not consider likelihood values for total copy numbers or unphased genotypes.</p><p id="p-11">The proposed EM algorithm can be divided into two procedures: enumeration and iterative calculation. In the initial procedure, we enumerate all possible diplotypes that are consistent with total copy numbers or unphased genotypes across multiple sites for every individual. More specifically, we first enumerate all possible diplotypes per site. For example, if a total copy number at an ICN site is 3, all possible diplotypes we enumerate are [0/3] and [1/2], where a slash (/) represents the separator between haplotypes (we do not distinguish the order of haplotypes). Similarly, if an unphased genotype at an SNVC site is AAG, all possible diplotypes are [-/A, A, G], [A, A/G], and [A/A, G], where a dash (-) represents a deletion and a comma (,) represents the separator between copy units (we do not distinguish the order of copy units in a haplotype). Here, we call a copy unit the unit of DNA sequence that is duplicated in a CNV region (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Then, we enumerate all possible diplotypes across multiple sites. For example, if we have the one-site diplotypes [0/3] and [1/2] at an ICN site, and the one-site diplotype [a/t] at an SNP site, all the possible diplotypes across these two sites are [0_a/3_t], [0_t/3_a], [1_a/2_t], and [1_t/2_a], where an underscore (_) represents the separator between sites.</p><p id="p-12">In this procedure, we also calculate a likelihood across multiple sites for each diplotype, following the assumption of Kang <em>et al.</em> (2004) that the occurrence of an experimental measurement, such as a signal intensity, at one site given a diplotype is independent of that at another site (also see Equations 1 and 6 below). Note that this assumption does not require linkage equilibrium – copy numbers or genotypes at multiple sites are independent of each other. Also note that, for microarray probes that are determined to be in the same state by a segmentation tool, it might be necessary to aggregate the correlated measurement values (<em>e.g.</em>, through median) into one value to treat the probe locations as one site.</p><p id="p-13">Under the assumption, the likelihood <em>λ<sub>j,k</sub></em> is calculated as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/35/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/35/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>k</em> denotes the index for the diplotype of individual <em>j</em>, and <em>s</em> and <em>S</em> denote the index for a site and the total number of sites in a given dataset, respectively. The variable <strong>x</strong><em><sub>j,k</sub></em> denotes the observed measurements such as the signal intensities or the log ratio intensities across multiple sites, and <em>x<sub>j,k,s</sub></em> denotes the observed measurement at site <em>s</em> in <strong>x</strong><em><sub>j,k</sub></em>. The symbol <em>d<sub>j,k</sub></em> denotes the diplotype indexed by <em>k</em> for individual <em>j</em> across multiple sites, and <em>d<sub>j,k,s</sub></em> denotes the one-site diplotype at site <em>s</em> in <em>d<sub>j,k</sub></em>. The last term Pr(<em>x<sub>j,k,s</sub></em> | <em>d<sub>j,k,s</sub></em>) is equal to the probability of <em>x<sub>j,k,s</sub></em> given the total copy number or unphased genotype of diplotype <em>d<sub>j,k,s</sub></em> (because we do not distinguish the signal intensities of, for example, [0/3] and [1/2]), and this probability is calculated from a statistical model of signal intensities or log ratio intensities in microarrays. The choice of a statistical model and a method to estimate its parameters is left to a user. For example, in simulation studies, we used those of Birdsuite (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Korn <em>et al.</em> 2008</a>), which employs a Gaussian mixture model and parameter estimation by the expectation-maximization algorithm. We obtained the last term probability, or likelihood, directly from the probability density of a signal intensity in a normal distribution of the model.</p><p id="p-14">After enumeration, we perform the iterative calculation, in which we iteratively repeat the expectation (E) and the maximization (M) steps. In the E step, we calculate the proportion of the frequency of a diplotype to the sum of the frequencies of all diplotypes in an individual, using likelihood <em>λ<sub>j,k</sub></em>. The equation at the E step is:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/35/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/35/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <em>w<sub>j,k</sub></em> denotes the diplotype proportion, <em>D<sub>j</sub></em> denotes the number of possible diplotypes of individual <em>j</em>, and <em>P</em> denotes the population frequency. <em>P</em>(<em>d<sub>j,k</sub></em>) is calculated from Hardy-Weinberg equilibrium (HWE):<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/35/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/35/embed/mml-math-3.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where the diplotype <em>d<sub>j,k</sub></em> consists of the haplotypes indexed by <em>l</em> and <em>m</em> (<em>h<sub>l</sub></em> and <em>h<sub>m</sub></em>). At the M step, the frequency of a haplotype is calculated from the number of the haplotype in consideration of the diplotype proportion calculated at the E step. The equation at the M step is:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/35/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/35/embed/mml-math-4.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/35/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/35/embed/mml-math-5.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <em>N</em> denotes the number of individuals in the dataset. After this M step, the iteration goes back to the E step to update the diplotype proportion, and in turn goes to the M step to update the haplotype frequency until the log-likelihood converges. The log-likelihood ln <em>L</em> is:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/35/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/35/embed/mml-math-6.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <strong>θ</strong> denotes (<em>P</em>(<em>h<sub>1</sub></em>), <em>P</em>(<em>h<sub>2</sub></em>), ...).</p><p id="p-15">The main difference of this EM algorithm from those of CNVphaser (<a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Kato <em>et al.</em> 2008a</a>) and MOCSphaser (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Kato <em>et al.</em> 2008b</a>) is the inclusion of the likelihood term <em>λ<sub>j,k</sub></em> in equations 2 and 6. By incorporating this term, total copy numbers and/or unphased genotypes are probabilistically represented. Equations 3, 4, and 5 are the same. The main difference of our algorithm from the GenoSpectrum-EM algorithm of Kang <em>et al.</em> (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Kang <em>et al.</em> 2004</a>) lies in the enumeration procedure, in which our algorithm can enumerate diplotypes composed of any combination of ICNs, SNVCs, and SNPs, whereas Kang <em>et al.</em>’s algorithm enumerates diplotypes composed only of SNPs. The equations in our EM algorithm are essentially the same as those in Kang <em>et al.</em>’s EM algorithm.</p></div><div id="sec-3" class="subsection"><h3>Software</h3><p id="p-16">We implemented this algorithm into a computational tool called CNVphaserPro, which is available online (<a href="http://rulai.cshl.edu/people/kato/">http://rulai.cshl.edu/people/kato/</a>). The advancements of this tool from the previous tools (<a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">Kato <em>et al.</em> 2008a</a>; <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Kato <em>et al.</em> 2008b</a>) are shown in the <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000174/-/DC1/000174SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000174/-/DC1/TableS1.pdf">Table S1</a>. The tool can handle missing calls and use multiple sets of randomly-generated initial values for multiple EM runs.</p></div></div><div class="section results" id="sec-4"><h2 class="">Results</h2><div id="sec-5" class="subsection"><h3>Simulation tests</h3><p id="p-17">We tested whether the algorithm could correctly restore haplotype frequencies in simulated datasets, which were made as follows (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). We first obtained unphased genotypes from the known haplotypes (listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000174/-/DC1/FileS1.zip">File S1</a>) of 588 individuals at 14 sites (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Sachse <em>et al.</em> 1997</a>), and then randomly generated signal intensities for the unphased genotypes at each site, using two-dimensional normal distributions for the genotypes. Their means and variances were derived from real microarray data (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Korn <em>et al.</em> 2008</a>). Next, we used these normal distributions to calculate the probability densities of the signal intensities for all possible unphased genotypes and used them as likelihood values. For total copy numbers, we took the summation of the likelihood values across unphased genotypes with the same copy number to obtain the likelihood values of total copy numbers. See <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000174/-/DC1/FileS2.pdf">File S2</a> for details on generating the simulation data. We input the likelihood values separately at each site into CNVphaserPro for one-site inference. We used the known haplotypes directly as the answer haplotypes.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/1/35/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustration of simulation. &#x201C;Ind.&#x201D; is short for individuals. The symbols &#x201C;-&#x201D;, &#x201C;/&#x201D;, and &#x201C;,&#x201D; in the top left table represent a deletion, the separator between haplotypes, and the separator between copy units in a duplication. (1) Using known diplotypes (Sachse et al. 1997), we made unphased genotypes. (2) We randomly generated signal intensities for the unphased genotypes, using normal distributions with the means and variances taken from real microarray data (Korn et al. 2008). (3) We calculated the likelihoods of the signal intensities for all possible unphased genotypes (or total copy numbers), based on the normal distributions above." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-425633437" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Illustration of simulation. &#x201C;Ind.&#x201D; is short for individuals. The symbols &#x201C;-&#x201D;, &#x201C;/&#x201D;, and &#x201C;,&#x201D; in the top left table represent a deletion, the separator between haplotypes, and the separator between copy units in a duplication. (1) Using known diplotypes (Sachse &lt;em&gt;et al.&lt;/em&gt; 1997), we made unphased genotypes. (2) We randomly generated signal intensities for the unphased genotypes, using normal distributions with the means and variances taken from real microarray data (Korn &lt;em&gt;et al.&lt;/em&gt; 2008). (3) We calculated the likelihoods of the signal intensities for all possible unphased genotypes (or total copy numbers), based on the normal distributions above.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/1/35/F2.medium.gif" width="440" height="300"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/1/35/F2.medium.gif" width="440" height="300"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/1/35/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/1/35/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460661" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-18" class="first-child">Illustration of simulation. “Ind.” is short for individuals. The symbols “-”, “/”, and “,” in the top left table represent a deletion, the separator between haplotypes, and the separator between copy units in a duplication. (1) Using known diplotypes (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Sachse <em>et al.</em> 1997</a>), we made unphased genotypes. (2) We randomly generated signal intensities for the unphased genotypes, using normal distributions with the means and variances taken from real microarray data (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Korn <em>et al.</em> 2008</a>). (3) We calculated the likelihoods of the signal intensities for all possible unphased genotypes (or total copy numbers), based on the normal distributions above.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">We first show an example of our results on one-site ICN inference (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The estimated frequencies were all close to the answer frequencies; hence, we concluded that the algorithm succeeds in the estimation. For comparison, we estimated haplotype frequencies by the previous algorithm [of MOCSphaser (<span class="sc">K</span>ato <em>et al.</em> 2008b) and CNVphaser (<a id="xref-ref-15-8" class="xref-bibr" href="#ref-15">Kato <em>et al.</em> 2008a</a>)], using the most likely total copy numbers as the input, which were defined as the one with the largest likelihood value of all possible total copy numbers. As demonstrated previously (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Excoffier and Slatkin 1995</a>; <a id="xref-ref-15-9" class="xref-bibr" href="#ref-15">Kato <em>et al.</em> 2008a</a>; <a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">Kato <em>et al.</em> 2008b</a>), we quantified the degree of estimation error by the total variation distance <em>TV</em>:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/35/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/35/embed/mml-math-7.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/35/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/35/embed/mml-math-8.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/35/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/1/35/embed/mml-math-9.gif"/></noscript></span></span> are the true and estimated frequencies of the haplotype <em>i</em>. As a result, the estimation error of the current algorithm was much less than that of the previous algorithm (the error rate 2% <em>vs.</em> 12%) (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). When we compared the most likely total copy numbers with the answer total copy numbers, 22% were incorrect, on average. This indicates that total copy numbers determined with the largest likelihood value are often incorrect due to microarray noise, which would result in the poorer estimates by the previous algorithm.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460715/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460715/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title">Examples of haplotype frequency estimation</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460739/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460739/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2 </span> <span class="caption-title">Error rate of haplotype frequencies estimated by the current and previous algorithms</span><div class="sb-div caption-clear"></div></div></div><p id="p-23">We also inferred individuals’ diplotypes of ICNs by selecting the diplotype with the largest diplotype proportion value in equation 2 (we did not use a threshold), and then evaluated inference accuracy by the proportion of inferred diplotypes that were the same as the answers. The result showed a good accuracy, 95% on average (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). We also used this proportion as the rate of total copy numbers correctly inferred by the current algorithm, because total copy numbers can be obtained simply from diplotypes. Thus, we compared this rate with the rate of correct ones of the most likely total copy numbers. As a result, the former rate (95%) was clearly larger than the latter rate (78%) (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>), which indicates that the current algorithm has an ability to correct total copy numbers incorrectly determined with the largest likelihood value. In fact, there were total copy numbers that were incorrectly determined with the largest likelihood value but were correctly inferred by the current algorithm; though we did find some, albeit far fewer, of the opposite case (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460729/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460729/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3 </span> <span class="caption-title">Inference accuracy of individuals’ diplotypes and total copy numbers or unphased genotypes</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">We also estimated the frequencies of haplotypes at one- and two-SNVC sites. In the two-site case, we chose pairs out of the 14 sites randomly a total of 14 times and input simulated likelihood values at the pairs into our algorithm. We show examples of results in both cases (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), which demonstrates that the estimated frequencies were almost the same as the answer frequencies. The <em>TV</em> index revealed that the estimation error of the current algorithm was much less than that of the previous algorithm (the error rate 1–2% <em>vs.</em> 12–18%) (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Regarding the inference of individuals’ diplotypes, the inference results also showed a good accuracy: on average, 95–96% were correct in both cases (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). As with ICNs, the current algorithm corrected unphased genotypes that were incorrectly determined with the largest likelihood value (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). Most importantly, improvement by the current algorithm from the previous algorithm was greater in the case of two SNVC sites, which is more complicated than the single ICN and SNVC cases.</p></div><div id="sec-6" class="subsection"><h3>Influence of sample size</h3><p id="p-29">We examined the influence of sample size on algorithm performance. For that purpose, we used population frequencies calculated from the known haplotypes of the 588 individuals to randomly sample haplotypes, and then randomly paired them on the basis of HWE to make answer datasets with different simple sizes. We made 10 replicate answer sets for each simple size. For each answer set, we used the same procedures as in the first simulation to make an input dataset.</p><p id="p-30">As a result, with the increase of the sample size, the current algorithm showed a slight but constant improvement in both the <em>TV</em> performance and the accuracy of inferred genotypes (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) or inferred copy numbers (data not shown), though the values of both indices were nearly saturated. In contrast, the previous procedures (the previous algorithm and the method to determine unphased copy numbers or genotypes by the largest likelihood value) did not always show a constant improvement. The current algorithm showed a better <em>TV</em> performance and a better genotype or copy-number inference at all sample sizes as compared to the previous procedures. In the two-SNVC case, the current algorithm showed the largest difference in <em>TV</em> and the accuracy from the previous procedures. In all the cases, the accuracy of diplotypes inferred by the current algorithm was greater than 90% even at a sample size of 50 and greater than 95% at a sample size of 100 or more. The <em>TV</em> values were extremely low at any sample size in all the cases (6% at a sample size of 50 at two-SNVC sites; less than 4%, otherwise).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/1/35/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Influence of sample size on performance. The left y-axis represents the deviation (the TV index) of estimated haplotype frequencies from answer frequencies for the current (red line with circles) and the previous (red line with crosses) algorithms. The right y-axis represents the proportion of diplotypes correctly inferred by the current algorithm (blue line with circles) or unphased genotypes correctly determined with the largest likelihood value (blue line with crosses). Points in the y-axes at each sample size indicate the mean of values over 10 different answer sets &#xD7; 14 different site sets. (A) For one SNVC site. (B) For two SNVC sites. The result of one ICN site is not shown, because this result was similar to that of one SNVC site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-425633437" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Influence of sample size on performance. The left &lt;em&gt;y&lt;/em&gt;-axis represents the deviation (the &lt;em&gt;TV&lt;/em&gt; index) of estimated haplotype frequencies from answer frequencies for the current (red line with circles) and the previous (red line with crosses) algorithms. The right &lt;em&gt;y&lt;/em&gt;-axis represents the proportion of diplotypes correctly inferred by the current algorithm (blue line with circles) or unphased genotypes correctly determined with the largest likelihood value (blue line with crosses). Points in the &lt;em&gt;y&lt;/em&gt;-axes at each sample size indicate the mean of values over 10 different answer sets &#xD7; 14 different site sets. (A) For one SNVC site. (B) For two SNVC sites. The result of one ICN site is not shown, because this result was similar to that of one SNVC site.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/1/35/F3.medium.gif" width="440" height="210"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/1/35/F3.medium.gif" width="440" height="210"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/1/35/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/1/35/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460763" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-31" class="first-child">Influence of sample size on performance. The left <em>y</em>-axis represents the deviation (the <em>TV</em> index) of estimated haplotype frequencies from answer frequencies for the current (red line with circles) and the previous (red line with crosses) algorithms. The right <em>y</em>-axis represents the proportion of diplotypes correctly inferred by the current algorithm (blue line with circles) or unphased genotypes correctly determined with the largest likelihood value (blue line with crosses). Points in the <em>y</em>-axes at each sample size indicate the mean of values over 10 different answer sets × 14 different site sets. (A) For one SNVC site. (B) For two SNVC sites. The result of one ICN site is not shown, because this result was similar to that of one SNVC site.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Real data application</h3><p id="p-32">We applied the phasing tool to real data taken with NimbleGen HD2 comparative genomic hybridization (CGH) arrays for the HapMap Phase3 CEU population. We identified 1486 CNV regions for 106 unrelated individuals, and then used the median log-ratio intensities across probes present in the intersections of CNV segments to obtain their likelihood values for the total copy numbers of zero to four, based on the Gaussian mixture model. See <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000174/-/DC1/FileS3.pdf">File S3</a> for details on processing the real data. We next applied the phasing tool to the likelihood values and estimated haplotype frequencies (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000174/-/DC1/FileS4.zip">File S4</a>) and diplotypes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000174/-/DC1/FileS5.zip">File S5</a>) in the CNV regions along the genome. We compared the predicted diplotypes (with the diplotype proportions &gt; 0.98) of two individuals to deletions that were extensively detected by next-generation sequencing at a high coverage (42×) for the same individuals (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Mills <em>et al.</em> 2011</a>). We found that eight out of the nine of our deletions were consistent with theirs and only 3.3% (65/1983) out of our 1/1 diplotypes were inconsistent.</p><p id="p-33">Using all the CNV regions, we drew an allele frequency spectrum (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>)—a basic graph that summarizes their population-genetic nature. The graph shows that the zero-copy (A0) and two-copy (A2) alleles tended to have small frequencies and that the number of the zero-copy allele was somewhat larger than that of the two-copy allele. We found 15 CNV regions where both zero-copy and two-copy alleles had the population frequencies of more than 1%, which indicates that these are tri-allelic. Twelve out of the fifteen were overlapped with CNVs in the Database of Genomic Variants (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Zhang <em>et al.</em> 2006</a>) and eight out of the twelve had been defined as multi-allelic CNVs (having both gains and losses) in CEU or a French population (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Conrad <em>et al.</em> 2010</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">De Smith <em>et al.</em> 2007</a>). We show examples of such CNV regions overlapped with genes (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The CNVs overlapped with <em>ACOT11</em> and <em>CWF19L2</em> were previously reported as multi-allelic in CEU (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Conrad <em>et al.</em> 2010</a>), but the CNV overlapped with <em>EYA2</em> was not reported. A homolog to <em>ACOT11</em> in mouse has been associated with obesity, but the function of <em>CWF19L2</em> is unknown. <em>EYA2</em> may play a role in eye development.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/1/35/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequency spectrums. The width of each bin is 2%. Alleles with a very small [1 &#x2212; 1 / (2 &#xD7; the number of individuals)] frequency are excluded from the counts. (A) The frequency spectrum of alleles. An (n = 0, 1, and 2) in the box represents an allelic copy number. (B) The frequency spectrum of total copy numbers derived from the allelic copy numbers. Tn (n = 0, 1, ..., 4) in the box represents an total copy number." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-425633437" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Frequency spectrums. The width of each bin is 2%. Alleles with a very small [&lt;1 / (2 &#xD7; the number of individuals)] or large [&gt;1 &#x2212; 1 / (2 &#xD7; the number of individuals)] frequency are excluded from the counts. (A) The frequency spectrum of alleles. A&lt;em&gt;n&lt;/em&gt; (&lt;em&gt;n&lt;/em&gt; = 0, 1, and 2) in the box represents an allelic copy number. (B) The frequency spectrum of total copy numbers derived from the allelic copy numbers. T&lt;em&gt;n&lt;/em&gt; (&lt;em&gt;n&lt;/em&gt; = 0, 1, ..., 4) in the box represents an total copy number.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/1/35/F4.medium.gif" width="440" height="210"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/1/1/35/F4.medium.gif" width="440" height="210"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/1/35/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/1/35/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460755" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-34" class="first-child">Frequency spectrums. The width of each bin is 2%. Alleles with a very small [&lt;1 / (2 × the number of individuals)] or large [&gt;1 − 1 / (2 × the number of individuals)] frequency are excluded from the counts. (A) The frequency spectrum of alleles. A<em>n</em> (<em>n</em> = 0, 1, and 2) in the box represents an allelic copy number. (B) The frequency spectrum of total copy numbers derived from the allelic copy numbers. T<em>n</em> (<em>n</em> = 0, 1, ..., 4) in the box represents an total copy number.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460670/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460670/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4 </span> <span class="caption-title">Examples of CNV regions that overlapped with genes and that had substantial amounts of estimated frequencies for both zero-copy and two-copy alleles</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-8"><h2 class="">Discussion</h2><p id="p-37">We used simulated datasets that were as close to real haplotypes and microarray noise levels as possible, and we demonstrated successful estimation by our algorithm. The reason for this success is the use of likelihoods to quantify fluctuations caused by noise. For example, even when a signal intensity is observed far away from the center of a distribution of the original total copy number, our algorithm does not discard the possibility that the signal came from this copy number, but considers the signal as a rare outcome with a small likelihood value. The usefulness of this strategy of incorporating genotype likelihoods (instead of called genotypes) into haplotype frequency estimation was demonstrated in SNP cases (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Browning and Yu 2009</a>; <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Kang <em>et al.</em> 2004</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Yu <em>et al.</em> 2009</a>). For example, in one recent study using BEAGLE version 3.1 (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Browning and Yu 2009</a>), this strategy was incorporated into SNP haplotype phasing based on an hidden Markov model (HMM). In our study, combining likelihoods with the EM algorithm based on a multinomial model (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Excoffier and Slatkin 1995</a>), we demonstrated its usefulness in CNV haplotype inference.</p><p id="p-38">As a by-product, our algorithm also inferred individuals’ diplotypes. The accuracy of the diplotype inference went over 90% even at a sample size of 50 and over 95% at a sample size of 100 or more, which indicates that even a sample size around 100 is enough for a good inference in typical cases. By accurately inferring diplotypes, the algorithm even corrected total copy numbers or unphased genotypes that were incorrectly determined with the largest likelihood value. Substantial amounts of 21–34% incorrect rates were decreased to only 4–5% through correction by the current algorithm.</p><p id="p-39">Most importantly, as the sample size increased, the estimation error index <em>TV</em> and the accuracy of estimated total copy numbers or unphased genotypes were improved in the current algorithm, but not always in the previous procedures. This is because there are always incorrect unphased copy numbers or genotypes determined with the largest likelihood value regardless of increasing a sample size (as shown by the blue lines with cross in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), as long as signal intensities coming from different unphased copy numbers or genotypes are not clearly separated with each other. Thus, increasing a sample size does not always improve estimation accuracy in the previous algorithm, but does so in the current algorithm—this is another advantage of the current algorithm.</p><p id="p-40">There would be several algorithmic issues to be addressed in the future. Our algorithm assumes HWE; hence, if a population does not satisfy HWE, the estimation would be worse. Regarding the EM algorithm used in SNP haplotype inference, it is known that deviations from HWE do not greatly impact the accuracy (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Niu <em>et al.</em> 2002</a>). In addition, a large-scale CNV study has found that most CNVs (98% of the bi-allelic CNVs) are in HWE (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">McCarroll <em>et al.</em> 2008</a>). Furthermore, in the first type of simulation, we demonstrated accurate estimation by our algorithm for known haplotypes in a natural population, in which HWE was not artificially introduced. Nevertheless, it is necessary to know how much impact deviations from HWE have on the accuracy.</p><p id="p-41">Another issue is to handle haplotypes composed of a number of sites. Currently, to analyze haplotypes composed of a few sites meets the needs for practical applications in CNV research, because urgent issues in this field are, first, association studies of CNVs, where current studies handle CNV regions as single sites (loci) and perform statistical tests separately per region (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Stefansson <em>et al.</em> 2008</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">The International Schizophrenia Consortium 2008</a>), and second, to find CNVs tagged by SNPs, where two-site haplotypes composed of an ICN and a SNP sites are phased to calculate a two-site LD index such as <em>R<sup>2</sup></em> (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Conrad <em>et al.</em> 2010</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Kato <em>et al.</em> 2010</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">McCarroll <em>et al.</em> 2008</a>). Our algorithm can fulfill these needs. In fact, our algorithm finished calculations for one and two sites across 588 samples within a short time (13 and 187 sec on average for one and two sites, respectively) on our machine with eight Xeon 2-GHz CPUs and 8 GB RAM, and also for three sites within a reasonable time (49 min on average). In addition, the algorithm inferred allelic copy numbers in all the 1486 CNV regions in the real data application within 18 hr on the same machine. However, our EM-based algorithm may not be suitable for applications beyond the current needs, since it takes significant time to handle a large number of sites due to the exhaustive enumeration. For example, our algorithm did not complete calculations within a reasonable time (24 hr) for four sites in some datasets, though the computational time can be reduced to some degree by cutting off states with the likelihood value of zero or nearly zero (after normalizing likelihood values, since only the relative values are meaningful).</p><p id="p-42">For applications beyond the current urgent needs, the algorithm will have to be improved. For example, recently the GrEM algorithm (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Shindo <em>et al.</em> 2009</a>) and the algorithm implemented in polyHap v2.0 (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Su <em>et al.</em> 2010</a>), which uses an HMM, have been proposed to deal with a larger number of sites for CNV haplotype inference from data unambiguously determined. They could be extended or modified to accommodate data with noise. In addition, a computational tool called cnvHap has recently been developed in order to call (unphased) copy numbers or sequence genotypes based on a haplotype HMM (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Coin <em>et al.</em> 2010</a>). Although the aim of cnvHap is not to estimate haplotype frequencies, information on haplotypes might be extracted from this approach. Other possible approaches include partition-ligation EM (<a id="xref-ref-15-10" class="xref-bibr" href="#ref-15">Kato <em>et al.</em> 2008a</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Qin <em>et al.</em> 2002</a>), Gibbs sampling, and coalescent-based sampling (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Niu 2004</a>).</p><p id="p-43">We applied our tool to real ICN data. Although we demonstrated accurate inference of SNVC haplotypes in simulation studies, it will be worth trying real SNVC data when such data in a large sample size (&gt;100 individuals) are accessible. Regarding association studies of CNVs, current studies using microarrays focus on ICNs or ICN groups known as gains (&gt;2 copies) and losses (&lt;2 copies). Recently, a theoretical study examined statistical power in association studies based on ICN haplotypes (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ohashi 2009</a>). Because SNVCs, which are a form of variation mixed of SNPs and ICNs, are more likely to have a phenotypic effect than only SNPs or only ICNs (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Hosono <em>et al.</em> 2008</a>; <a id="xref-ref-15-11" class="xref-bibr" href="#ref-15">Kato <em>et al.</em> 2008a</a>), it would be important that these studies are extended for SNVC haplotypes, in which case, our algorithm will be of essential use.</p><p id="p-44">Because microarrays are currently less costly than next-generation sequencing in the same sample size in genome-wide CNV studies, our haplotype phasing based on microarray data are cost effective for rapidly investigating hundreds or thousands of individuals. Recently, a study has developed a successful method to estimate total copy numbers from read depth in next-generation sequencing and applied it to three individuals (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alkan <em>et al.</em> 2009</a>). Taking account of the fact that read counts fluctuate by noise (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Yoon <em>et al.</em> 2009</a>), our likelihood method would be applicable also for such copy numbers when sample size is sufficiently obtained in the future.</p></div><div class="section" id="sec-9"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">We thank Takashi Morizono for help with coding programs in the simulation tests and Tetsuo Abe for help with coding programs to process real data. We thank Michael Wigler for information on real data. We acknowledge Willey Liao and Paloma Anderson for English correction. This work was supported by the National Cancer Institute (P30CA045508) and the National Institutes of Health (HG004222 and ARRA 1RC2 CA148532-1).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-12"><p id="p-46"><em>Communicating editor: Brenda J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2010-12-31"><span class="received-label">Received </span>December 31, 2010.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-03-14"><span class="accepted-label">Accepted </span>March 14, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2011 Kato <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-3">This is an open access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.1" data-doi="10.1038/ng.437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alkan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marques-Bonet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aksay</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antonacci</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Personalized copy number and segmental duplication maps using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span><strong>:</strong> <span class="cit-fpage">1061</span>–<span class="cit-lpage">1067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAlkan%26rft.auinit1%253DC.%26rft.volume%253D41%26rft.issue%253D10%26rft.spage%253D1061%26rft.epage%253D1067%26rft.atitle%253DPersonalized%2Bcopy%2Bnumber%2Band%2Bsegmental%2Bduplication%2Bmaps%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.437%26rft_id%253Dinfo%253Apmid%252F19718026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19718026&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270330400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.2" data-doi="10.1016/j.ajhg.2009.11.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Simultaneous genotype calling and haplotype phasing improves genotype accuracy and reduces false-positive associations for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">85</span><strong>:</strong> <span class="cit-fpage">847</span>–<span class="cit-lpage">861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DB.%2BL.%26rft.volume%253D85%26rft.issue%253D6%26rft.spage%253D847%26rft.epage%253D861%26rft.atitle%253DSimultaneous%2Bgenotype%2Bcalling%2Band%2Bhaplotype%2Bphasing%2Bimproves%2Bgenotype%2Baccuracy%2Band%2Breduces%2Bfalse-positive%2Bassociations%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.11.004%26rft_id%253Dinfo%253Apmid%252F19931040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.11.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19931040&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272797100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-1.1.35.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dermitzakis</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antonarakis</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Trisomic phase inference</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span> in <span class="cit-source"><em>Computational Methods for SNPs and Haplotype Inference</em>, <em>Lecture Notes in Bioinformatics 2983</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Istrail</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Waterman</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.4" data-doi="10.1038/nmeth.1466"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coin</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asher</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walters</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moustafa</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Smith</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">cnvHap: an integrative population and haplotype-based multiplatform model of SNPs and CNVs.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span><strong>:</strong> <span class="cit-fpage">541</span>–<span class="cit-lpage">546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DCoin%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253D541%26rft.epage%253D546%26rft.atitle%253DcnvHap%253A%2Ban%2Bintegrative%2Bpopulation%2Band%2Bhaplotype-based%2Bmultiplatform%2Bmodel%2Bof%2BSNPs%2Band%2BCNVs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1466%26rft_id%253Dinfo%253Apmid%252F20512141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1466&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20512141&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279343300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.5" data-doi="10.1038/ng2042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurles</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The population genetics of structural variation.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span><strong>:</strong> <span class="cit-fpage">S30</span>–<span class="cit-lpage">S36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DConrad%26rft.auinit1%253DD.%2BF.%26rft.volume%253D39%26rft.issue%253D7%2BSuppl%26rft.spage%253DS30%26rft.epage%253DS36%26rft.atitle%253DThe%2Bpopulation%2Bgenetics%2Bof%2Bstructural%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2042%26rft_id%253Dinfo%253Apmid%252F17597779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17597779&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.6" data-doi="10.1038/nature08516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinto</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feuk</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gokcumen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Origins and functional impact of copy number variation in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span><strong>:</strong> <span class="cit-fpage">704</span>–<span class="cit-lpage">712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DConrad%26rft.auinit1%253DD.%2BF.%26rft.volume%253D464%26rft.issue%253D7289%26rft.spage%253D704%26rft.epage%253D712%26rft.atitle%253DOrigins%2Band%2Bfunctional%2Bimpact%2Bof%2Bcopy%2Bnumber%2Bvariation%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08516%26rft_id%253Dinfo%253Apmid%252F19812545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08516&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19812545&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276205000035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.7" data-doi="10.1093/hmg/ddm208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Smith</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsalenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sampas</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheffer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamada</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Array CGH analysis of copy number variation identifies 1284 new genes variant in healthy white males: implications for association studies of complex diseases.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">16</span><strong>:</strong> <span class="cit-fpage">2783</span>–<span class="cit-lpage">2794</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddm208%26rft_id%253Dinfo%253Apmid%252F17666407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/hmg/ddm208&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17666407&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251036400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Maximum-likelihood estimation of molecular haplotype frequencies in a diploid population.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">12</span><strong>:</strong> <span class="cit-fpage">921</span>–<span class="cit-lpage">927</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DExcoffier%26rft.auinit1%253DL%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D921%26rft.epage%253D927%26rft.atitle%253DMaximum-likelihood%2Bestimation%2Bof%2Bmolecular%2Bhaplotype%2Bfrequencies%2Bin%2Ba%2Bdiploid%2Bpopulation%26rft_id%253Dinfo%253Apmid%252F7476138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7476138&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RQ66400021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.9" data-doi="10.1101/gr.3677206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freeman</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feuk</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarroll</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Copy number variation: new insights in genome diversity.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span><strong>:</strong> <span class="cit-fpage">949</span>–<span class="cit-lpage">961</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3677206%26rft_id%253Dinfo%253Apmid%252F16809666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzgvOTQ5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8xLzM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.10" data-doi="10.1373/clinchem.2009.123620"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hosono</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiyotani</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mushiroda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takata</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">CYP2D6 genotyping for functional-gene dosage analysis by allele copy number detection.</span> <abbr class="cit-jnl-abbrev">Clin. Chem.</abbr> <span class="cit-vol">55</span><strong>:</strong> <span class="cit-fpage">1546</span>–<span class="cit-lpage">1554</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1373%252Fclinchem.2009.123620%26rft_id%253Dinfo%253Apmid%252F19541866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiY2xpbmNoZW0iO3M6NToicmVzaWQiO3M6OToiNTUvOC8xNTQ2IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8xLzM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.11" data-doi="10.1002/humu.20609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hosono</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kubo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuchiya</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Multiplex PCR-based real-time invader assay (mPCR-RETINA): a novel SNP-based method for detecting allelic asymmetries within copy number variation regions.</span> <abbr class="cit-jnl-abbrev">Hum. Mutat.</abbr> <span class="cit-vol">29</span><strong>:</strong> <span class="cit-fpage">182</span>–<span class="cit-lpage">189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bmutation%26rft.stitle%253DHum%2BMutat%26rft.aulast%253DHosono%26rft.auinit1%253DN.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D182%26rft.epage%253D189%26rft.atitle%253DMultiplex%2BPCR-based%2Breal-time%2Binvader%2Bassay%2B%2528mPCR-RETINA%2529%253A%2Ba%2Bnovel%2BSNP-based%2Bmethod%2Bfor%2Bdetecting%2Ballelic%2Basymmetries%2Bwithin%2Bcopy%2Bnumber%2Bvariation%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fhumu.20609%26rft_id%253Dinfo%253Apmid%252F17879341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/humu.20609&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17879341&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252143300025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.12" data-doi="10.1038/ng1416"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iafrate</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feuk</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rivera</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Listewnik</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donahoe</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Detection of large-scale variation in the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span><strong>:</strong> <span class="cit-fpage">949</span>–<span class="cit-lpage">951</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DIafrate%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D36%26rft.issue%253D9%26rft.spage%253D949%26rft.epage%253D951%26rft.atitle%253DDetection%2Bof%2Blarge-scale%2Bvariation%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1416%26rft_id%253Dinfo%253Apmid%252F15286789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1416&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15286789&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223658100018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.13" data-doi="10.1086/382284"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">Z. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Incorporating genotyping uncertainty in haplotype inference for single-nucleotide polymorphisms.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">74</span><strong>:</strong> <span class="cit-fpage">495</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DKang%26rft.auinit1%253DH.%26rft.volume%253D74%26rft.issue%253D3%26rft.spage%253D495%26rft.epage%253D510%26rft.atitle%253DIncorporating%2Bgenotyping%2Buncertainty%2Bin%2Bhaplotype%2Binference%2Bfor%2Bsingle-nucleotide%2Bpolymorphisms.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F382284%26rft_id%253Dinfo%253Apmid%252F14966673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/382284&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14966673&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220118500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.14" data-doi="10.1093/hmg/ddp541"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawaguchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishikawa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Umeda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakamichi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Population-genetic nature of copy number variations in the human genome.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">19</span><strong>:</strong> <span class="cit-fpage">761</span>–<span class="cit-lpage">773</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddp541%26rft_id%253Dinfo%253Apmid%252F19966329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/hmg/ddp541&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19966329&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274341400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.15" data-doi="10.1016/j.ajhg.2008.06.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsunoda</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span><span class="cit-comment">a</span> <span class="cit-article-title">An algorithm for inferring complex haplotypes in a region of copy-number variation.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">83</span><strong>:</strong> <span class="cit-fpage">157</span>–<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DKato%26rft.auinit1%253DM.%26rft.volume%253D83%26rft.issue%253D2%26rft.spage%253D157%26rft.epage%253D169%26rft.atitle%253DAn%2Balgorithm%2Bfor%2Binferring%2Bcomplex%2Bhaplotypes%2Bin%2Ba%2Bregion%2Bof%2Bcopy-number%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2008.06.021%26rft_id%253Dinfo%253Apmid%252F18639202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2008.06.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18639202&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258418200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.16" data-doi="10.1093/bioinformatics/btn242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsunoda</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span><span class="cit-comment">b</span> <span class="cit-article-title">MOCSphaser: a haplotype inference tool from a mixture of copy number variation and single nucleotide polymorphism data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span><strong>:</strong> <span class="cit-fpage">1645</span>–<span class="cit-lpage">1646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn242%26rft_id%253Dinfo%253Apmid%252F18492685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18492685&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257576000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.17" data-doi="10.1038/ng.237"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korn</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuruvilla</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarroll</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nemesh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Integrated genotype calling and association analysis of SNPs, common copy number polymorphisms and rare CNVs.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span><strong>:</strong> <span class="cit-fpage">1253</span>–<span class="cit-lpage">1260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKorn%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D40%26rft.issue%253D10%26rft.spage%253D1253%26rft.epage%253D1260%26rft.atitle%253DIntegrated%2Bgenotype%2Bcalling%2Band%2Bassociation%2Banalysis%2Bof%2BSNPs%252C%2Bcommon%2Bcopy%2Bnumber%2Bpolymorphisms%2Band%2Brare%2BCNVs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.237%26rft_id%253Dinfo%253Apmid%252F18776909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.237&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18776909&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259651000028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.18" data-doi="10.1038/ng2080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarroll</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Copy-number variation and association studies of human disease.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span><strong>:</strong> <span class="cit-fpage">S37</span>–<span class="cit-lpage">S42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMcCarroll%26rft.auinit1%253DS.%2BA.%26rft.volume%253D39%26rft.issue%253D7%2BSuppl%26rft.spage%253DS37%26rft.epage%253DS42%26rft.atitle%253DCopy-number%2Bvariation%2Band%2Bassociation%2Bstudies%2Bof%2Bhuman%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2080%26rft_id%253Dinfo%253Apmid%252F17597780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17597780&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.19" data-doi="10.1038/ng.238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarroll</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuruvilla</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korn</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cawley</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nemesh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Integrated detection and population-genetic analysis of SNPs and copy number variation.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span><strong>:</strong> <span class="cit-fpage">1166</span>–<span class="cit-lpage">1174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMcCarroll%26rft.auinit1%253DS.%2BA.%26rft.volume%253D40%26rft.issue%253D10%26rft.spage%253D1166%26rft.epage%253D1174%26rft.atitle%253DIntegrated%2Bdetection%2Band%2Bpopulation-genetic%2Banalysis%2Bof%2BSNPs%2Band%2Bcopy%2Bnumber%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.238%26rft_id%253Dinfo%253Apmid%252F18776908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18776908&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259651000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.20" data-doi="10.1016/j.tig.2009.10.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Merikangas</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corvin</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallagher</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Copy-number variants in neurodevelopmental disorders: promises and challenges.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">25</span><strong>:</strong> <span class="cit-fpage">536</span>–<span class="cit-lpage">544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DMerikangas%26rft.auinit1%253DA.%2BK.%26rft.volume%253D25%26rft.issue%253D12%26rft.spage%253D536%26rft.epage%253D544%26rft.atitle%253DCopy-number%2Bvariants%2Bin%2Bneurodevelopmental%2Bdisorders%253A%2Bpromises%2Band%2Bchallenges.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2009.10.006%26rft_id%253Dinfo%253Apmid%252F19910074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2009.10.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19910074&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272808900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.21" data-doi="10.1038/nature09708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mills</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mapping copy number variation by population-scale genome sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">470</span><strong>:</strong> <span class="cit-fpage">59</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMills%26rft.auinit1%253DR.%2BE.%26rft.volume%253D470%26rft.issue%253D7332%26rft.spage%253D59%26rft.epage%253D65%26rft.atitle%253DMapping%2Bcopy%2Bnumber%2Bvariation%2Bby%2Bpopulation-scale%2Bgenome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09708%26rft_id%253Dinfo%253Apmid%252F21293372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09708&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21293372&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286886400033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-other" id="cit-1.1.35.22"><div class="cit-metadata unstructured">Neigenfind, J., G. Gyetvai, R. Basekow, S. Diehl, U. Achenbach <em>et al.</em>, 2008 Haplotype inference from unphased SNP data in heterozygous polyploids based on SAT. BMC Genomics <strong>9:</strong> 356.1–356.6.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.23" data-doi="10.1002/gepi.20024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Algorithms for inferring haplotypes.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">27</span><strong>:</strong> <span class="cit-fpage">334</span>–<span class="cit-lpage">347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DNiu%26rft.auinit1%253DT.%26rft.volume%253D27%26rft.issue%253D4%26rft.spage%253D334%26rft.epage%253D347%26rft.atitle%253DAlgorithms%2Bfor%2Binferring%2Bhaplotypes.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20024%26rft_id%253Dinfo%253Apmid%252F15368348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15368348&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225524800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.24" data-doi="10.1086/338446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">Z. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Bayesian haplotype inference for multiple linked single-nucleotide polymorphisms.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">70</span><strong>:</strong> <span class="cit-fpage">157</span>–<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DNiu%26rft.auinit1%253DT.%26rft.volume%253D70%26rft.issue%253D1%26rft.spage%253D157%26rft.epage%253D169%26rft.atitle%253DBayesian%2Bhaplotype%2Binference%2Bfor%2Bmultiple%2Blinked%2Bsingle-nucleotide%2Bpolymorphisms.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F338446%26rft_id%253Dinfo%253Apmid%252F11741196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/338446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11741196&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172571900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohashi</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A practical case-control association test for detecting a susceptibility allele at a copy number variation locus.</span> <abbr class="cit-jnl-abbrev">J. Hum. Genet.</abbr> <span class="cit-vol">54</span><strong>:</strong> <span class="cit-fpage">169</span>–<span class="cit-lpage">173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DJ%2BHum%2BGenet%26rft.aulast%253DOhashi%26rft.auinit1%253DJ.%26rft.volume%253D54%26rft.issue%253D3%26rft.spage%253D169%26rft.epage%253D173%26rft.atitle%253DA%2Bpractical%2Bcase-control%2Bassociation%2Btest%2Bfor%2Bdetecting%2Ba%2Bsusceptibility%2Ballele%2Bat%2Ba%2Bcopy%2Bnumber%2Bvariation%2Blocus.%26rft_id%253Dinfo%253Apmid%252F19197338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19197338&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.26" data-doi="10.1086/344207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">Z. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Partition-ligation-expectation-maximization algorithm for haplotype inference with single-nucleotide polymorphisms.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">71</span><strong>:</strong> <span class="cit-fpage">1242</span>–<span class="cit-lpage">1247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DQin%26rft.auinit1%253DZ.%2BS.%26rft.volume%253D71%26rft.issue%253D5%26rft.spage%253D1242%26rft.epage%253D1247%26rft.atitle%253DPartition-ligation-expectation-maximization%2Balgorithm%2Bfor%2Bhaplotype%2Binference%2Bwith%2Bsingle-nucleotide%2Bpolymorphisms.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F344207%26rft_id%253Dinfo%253Apmid%252F12452179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/344207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12452179&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178884300030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.27" data-doi="10.1038/nature05329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Redon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishikawa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitch</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feuk</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Global variation in copy number in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">444</span><strong>:</strong> <span class="cit-fpage">444</span>–<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRedon%26rft.auinit1%253DR.%26rft.volume%253D444%26rft.issue%253D7118%26rft.spage%253D444%26rft.epage%253D454%26rft.atitle%253DGlobal%2Bvariation%2Bin%2Bcopy%2Bnumber%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05329%26rft_id%253Dinfo%253Apmid%252F17122850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17122850&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242215700038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sachse</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brockmoller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roots</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Cytochrome P450 2D6 variants in a Caucasian population: allele frequencies and phenotypic consequences.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">60</span><strong>:</strong> <span class="cit-fpage">284</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DSachse%26rft.auinit1%253DC.%26rft.volume%253D60%26rft.issue%253D2%26rft.spage%253D284%26rft.epage%253D295%26rft.atitle%253DCytochrome%2BP450%2B2D6%2Bvariants%2Bin%2Ba%2BCaucasian%2Bpopulation%253A%2Ballele%2Bfrequencies%2Band%2Bphenotypic%2Bconsequences.%26rft_id%253Dinfo%253Apmid%252F9012401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9012401&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WE93100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.29" data-doi="10.1038/ng2095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sebat</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Major changes in our DNA lead to major changes in our thinking.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span><strong>:</strong> <span class="cit-fpage">S3</span>–<span class="cit-lpage">S5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSebat%26rft.auinit1%253DJ.%26rft.volume%253D39%26rft.issue%253D7%2BSuppl%26rft.spage%253DS3%26rft.epage%253DS5%26rft.atitle%253DMajor%2Bchanges%2Bin%2Bour%2BDNA%2Blead%2Bto%2Bmajor%2Bchanges%2Bin%2Bour%2Bthinking.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2095%26rft_id%253Dinfo%253Apmid%252F17597778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2095&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17597778&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.30" data-doi="10.1126/science.1098918"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sebat</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lakshmi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troge</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Large-scale copy number polymorphism in the human genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">305</span><strong>:</strong> <span class="cit-fpage">525</span>–<span class="cit-lpage">528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSebat%26rft.auinit1%253DJ.%26rft.volume%253D305%26rft.issue%253D5683%26rft.spage%253D525%26rft.epage%253D528%26rft.atitle%253DLarge-Scale%2BCopy%2BNumber%2BPolymorphism%2Bin%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1098918%26rft_id%253Dinfo%253Apmid%252F15273396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDUvNTY4My81MjUiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzEvMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-confproc" id="cit-1.1.35.31"><div class="cit-metadata"><cite>Shindo, H., H. Chigira, T. Nagaoka, N. Kamatani, and M. Inoue, 2009 Grouping preprocess for haplotype inference from SNP and CNV data. Journal of Physics, Conference Series, Kyoto, Japan. DOI: 10.1088/1742–6596/1197/1081/012009.<span class="cit-pub-id cit-pub-id-doi">10.1088/1742–6596/1197/1081/012009</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.32" data-doi="10.1159/000184706"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Speleman</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumps</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buysse</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poppe</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menten</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Copy number alterations and copy number variation in cancer: close encounters of the bad kind.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">123</span><strong>:</strong> <span class="cit-fpage">176</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DSpeleman%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D1-4%26rft.spage%253D176%26rft.epage%253D182%26rft.atitle%253DCopy%2Bnumber%2Balterations%2Band%2Bcopy%2Bnumber%2Bvariation%2Bin%2Bcancer%253A%2Bclose%2Bencounters%2Bof%2Bthe%2Bbad%2Bkind.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000184706%26rft_id%253Dinfo%253Apmid%252F19287153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000184706&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19287153&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.33" data-doi="10.1038/nature07229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stefansson</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rujescu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cichon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pietilainen</span>  <span class="cit-name-given-names">O. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingason</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Large recurrent microdeletions associated with schizophrenia.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">455</span><strong>:</strong> <span class="cit-fpage">232</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DStefansson%26rft.auinit1%253DH.%26rft.volume%253D455%26rft.issue%253D7210%26rft.spage%253D232%26rft.epage%253D236%26rft.atitle%253DLarge%2Brecurrent%2Bmicrodeletions%2Bassociated%2Bwith%2Bschizophrenia.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07229%26rft_id%253Dinfo%253Apmid%252F18668039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07229&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18668039&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259090800049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.34" data-doi="10.1093/bioinformatics/btq157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asher</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarvelin</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Froguel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blakemore</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Inferring combined CNV/SNP haplotypes from genotype data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span><strong>:</strong> <span class="cit-fpage">1437</span>–<span class="cit-lpage">1445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq157%26rft_id%253Dinfo%253Apmid%252F20406911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20406911&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277985500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-other" id="cit-1.1.35.35"><div class="cit-metadata unstructured">Su, S. Y., J. White, D. J. Balding and L. J. Coin, 2008 Inference of haplotypic phase and missing genotypes in polyploid organisms and variable copy number genomic regions. BMC Bioinformatics <strong>9:</strong> 513.1–513.9.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.36" data-doi="10.1038/nature07239"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The International Schizophrenia Consortium</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rare chromosomal deletions and duplications increase risk of schizophrenia.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">455</span><strong>:</strong> <span class="cit-fpage">237</span>–<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DStone%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D455%26rft.issue%253D7210%26rft.spage%253D237%26rft.epage%253D241%26rft.atitle%253DRare%2Bchromosomal%2Bdeletions%2Band%2Bduplications%2Bincrease%2Brisk%2Bof%2Bschizophrenia.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07239%26rft_id%253Dinfo%253Apmid%252F18668038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07239&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18668038&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259090800050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.37" data-doi="10.1101/gr.092981.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xuan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makarov</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sebat</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sensitive and accurate detection of copy number variants using read depth of coverage.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span><strong>:</strong> <span class="cit-fpage">1586</span>–<span class="cit-lpage">1592</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.092981.109%26rft_id%253Dinfo%253Apmid%252F19657104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTU4NiI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzEvMS8zNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-other" id="cit-1.1.35.38"><div class="cit-metadata unstructured">Yu, Z., C. Garner, A. Ziogas, H. Anton-Culver and D. J. Schaid, 2009 Genotype determination for polymorphisms in linkage disequilibrium. BMC Bioinformatics <strong>10:</strong> 63.1–63.8.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-1.1.35.39" data-doi="10.1159/000095916"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feuk</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duggan</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khaja</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scherer</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Development of bioinformatics resources for display and analysis of copy number and other structural variants in the human genome.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">115</span><strong>:</strong> <span class="cit-fpage">205</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DZhang%26rft.auinit1%253DJ.%26rft.volume%253D115%26rft.issue%253D3-4%26rft.spage%253D205%26rft.epage%253D214%26rft.atitle%253DDevelopment%2Bof%2Bbioinformatics%2Bresources%2Bfor%2Bdisplay%2Band%2Banalysis%2Bof%2Bcopy%2Bnumber%2Band%2Bother%2Bstructural%2Bvariants%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000095916%26rft_id%253Dinfo%253Apmid%252F17124402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000095916&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17124402&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F1%2F35.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242391500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/1/35.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/1/27" title="Variability of Gene Expression After Polyhaploidization in Wheat (Triticum aestivum L.)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/1/43" title="Identification of Yeast Genes Involved in K+ Homeostasis: Loss of Membrane Traffic Genes Affects K+ Uptake" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/1"><span>Volume 1 Issue 1, June 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/1.cover.gif?itok=pV8lCSGW" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/1/35.full?utm_source=twitterfeed&amp;utm_medium=twitter" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-COjoA2CWGEQ1fSnKDlEfspc5lFaLp_GV9Wju3Gzu-jg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/1/35" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/1/35" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/1/35" class="active" data-icon-position="" data-hide-link-title="0">Inferring Haplotypes of Copy Number Variations From High-Throughput Data With Uncertainty</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459153" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-z9HIxbTalVDz8sD0xr_XURByCyJwFMa-oZ3QfWMH-R4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/1/35.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/1/35" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-_bo_DzQL6a5CCuhwB6DyXRnCxL-BzUaLDOZgscPuRmY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/1/35" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/1/35.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459153" id="node459153--2" data-pisa="ggg;1/1/35" data-pisa-master="ggg;g3.111.000174" data-apath="/ggg/1/1/35.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inferring Haplotypes of Copy Number Variations From High-Throughput Data With Uncertainty</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mamoru Kato</span>, <span class="highwire-citation-author" data-delta="1">Seungtai Yoon</span>, <span class="highwire-citation-author" data-delta="2">Naoya Hosono</span>, <span class="highwire-citation-author" data-delta="3">Anthony Leotta</span>, <span class="highwire-citation-author" data-delta="4">Jonathan Sebat</span>, <span class="highwire-citation-author" data-delta="5">Tatsuhiko Tsunoda</span> and <span class="highwire-citation-author" data-delta="6">Michael Q. Zhang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">35-42;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000174 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459153--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mamoru Kato </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mamoru%2BKato%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kato%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMamoru%2BKato%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Seungtai Yoon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seungtai%2BYoon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yoon%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASeungtai%2BYoon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Naoya Hosono </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Genomic Medicine, RIKEN, Yokohama, Kanagawa 230-0045, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Naoya%2BHosono%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hosono%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANaoya%2BHosono%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Anthony Leotta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anthony%2BLeotta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leotta%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnthony%2BLeotta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jonathan Sebat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Psychiatry, University of California, San Diego, La Jolla, CA 92093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BSebat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sebat%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BSebat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Tatsuhiko Tsunoda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Genomic Medicine, RIKEN, Yokohama, Kanagawa 230-0045, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tatsuhiko%2BTsunoda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsunoda%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATatsuhiko%2BTsunoda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael Q. Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Bioinformatics Division, TNLIST/Department of Automation, Tsinghua University, Beijing, 100084, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BQ.%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20MQ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BQ.%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mzhang@cshl.edu" class="" data-icon-position="" data-hide-link-title="0">mzhang@cshl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/1/35" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/1/35&amp;title=Inferring%20Haplotypes%20of%20Copy%20Number%20Variations%20From%20High-Throughput%20Data%20With%20Uncertainty" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/1/35&amp;title=Inferring%20Haplotypes%20of%20Copy%20Number%20Variations%20From%20High-Throughput%20Data%20With%20Uncertainty" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/1/35&amp;title=Inferring%20Haplotypes%20of%20Copy%20Number%20Variations%20From%20High-Throughput%20Data%20With%20Uncertainty" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/1/35&amp;text=Inferring%20Haplotypes%20of%20Copy%20Number%20Variations%20From%20High-Throughput%20Data%20With%20Uncertainty" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/1/35&amp;title=Inferring%20Haplotypes%20of%20Copy%20Number%20Variations%20From%20High-Throughput%20Data%20With%20Uncertainty" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/1/35&amp;t=Inferring%20Haplotypes%20of%20Copy%20Number%20Variations%20From%20High-Throughput%20Data%20With%20Uncertainty" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/1/35&amp;title=Inferring%20Haplotypes%20of%20Copy%20Number%20Variations%20From%20High-Throughput%20Data%20With%20Uncertainty" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/1/35&amp;title=Inferring%20Haplotypes%20of%20Copy%20Number%20Variations%20From%20High-Throughput%20Data%20With%20Uncertainty" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459153/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459153" id="node459153--3" data-pisa="ggg;1/1/35" data-pisa-master="ggg;g3.111.000174" data-apath="/ggg/1/1/35.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inferring Haplotypes of Copy Number Variations From High-Throughput Data With Uncertainty</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mamoru Kato</span>, <span class="highwire-citation-author" data-delta="1">Seungtai Yoon</span>, <span class="highwire-citation-author" data-delta="2">Naoya Hosono</span>, <span class="highwire-citation-author" data-delta="3">Anthony Leotta</span>, <span class="highwire-citation-author" data-delta="4">Jonathan Sebat</span>, <span class="highwire-citation-author" data-delta="5">Tatsuhiko Tsunoda</span> and <span class="highwire-citation-author" data-delta="6">Michael Q. Zhang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">35-42;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000174 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459153--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mamoru Kato </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mamoru%2BKato%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kato%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMamoru%2BKato%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Seungtai Yoon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seungtai%2BYoon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yoon%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASeungtai%2BYoon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Naoya Hosono </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Genomic Medicine, RIKEN, Yokohama, Kanagawa 230-0045, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Naoya%2BHosono%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hosono%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANaoya%2BHosono%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Anthony Leotta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anthony%2BLeotta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leotta%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnthony%2BLeotta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jonathan Sebat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Psychiatry, University of California, San Diego, La Jolla, CA 92093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BSebat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sebat%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BSebat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Tatsuhiko Tsunoda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Genomic Medicine, RIKEN, Yokohama, Kanagawa 230-0045, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tatsuhiko%2BTsunoda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsunoda%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATatsuhiko%2BTsunoda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael Q. Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Bioinformatics Division, TNLIST/Department of Automation, Tsinghua University, Beijing, 100084, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BQ.%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20MQ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BQ.%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mzhang@cshl.edu" class="" data-icon-position="" data-hide-link-title="0">mzhang@cshl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=35&amp;rft.epage=42&amp;rft.atitle=Inferring%20Haplotypes%20of%20Copy%20Number%20Variations%20From%20High-Throughput%20Data%20With%20Uncertainty&amp;rft.volume=1&amp;rft.issue=1&amp;rft.date=2011-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kato%2C+Mamoru&amp;rft.au=Yoon%2C+Seungtai&amp;rft.au=Hosono%2C+Naoya&amp;rft.au=Leotta%2C+Anthony&amp;rft.au=Sebat%2C+Jonathan&amp;rft.au=Tsunoda%2C+Tatsuhiko&amp;rft.au=Zhang%2C+Michael+Q."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459153/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459153/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459153/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459153/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459153/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459153/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459153/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459153/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459153/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459153/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459153/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459153/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496048" id="node496048" data-pisa="ggg;g3.119.400937v1" data-pisa-master="ggg;g3.119.400937" data-apath="/ggg/early/2020/03/25/g3.119.400937.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/25/g3.119.400937" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tunable Transcriptional Interference at the Endogenous Alcohol Dehydrogenase Gene Locus in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496046" id="node496046" data-pisa="ggg;g3.120.401048v1" data-pisa-master="ggg;g3.120.401048" data-apath="/ggg/early/2020/03/23/g3.120.401048.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/23/g3.120.401048" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tools for the Genetic Manipulation of <em>Herpetomonas muscarum</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496045" id="node496045" data-pisa="ggg;g3.119.400884v1" data-pisa-master="ggg;g3.119.400884" data-apath="/ggg/early/2020/03/17/g3.119.400884.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400884" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study for Maize Leaf Cuticular Conductance Identifies Candidate Genes Involved in the Regulation of Cuticle Development</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/1/35" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/1/35.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/1/35.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/1/1/35.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/1/1/35.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
