<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/1/13" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/1/13.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/1/13.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/1/13.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/1/13.atom" />
<meta name="HW.pisa" content="ggg;9/1/13" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="OCMA: Fast, Memory-Efficient Factorization of Prohibitively Large Relationship Matrices" />
<meta name="DC.Identifier" content="10.1534/g3.118.200908" />
<meta name="DC.Date" content="2019-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Xiong et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7384973" />
<meta name="DC.Description" content="Matrices representing genetic relatedness among individuals ( i.e. , Genomic Relationship Matrices, GRMs) play a central role in genetic analysis. The eigen-decomposition of GRMs (or its alternative that generates fewer top singular values using genotype matrices) is a necessary step for many analyses including estimation of SNP-heritability, Principal Component Analysis (PCA), and genomic prediction. However, the GRMs and genotype matrices provided by modern biobanks are too large to be stored in active memory. To accommodate the current and future “bigger-data”, we develop a disk-based tool, Out-of-Core Matrices Analyzer (OCMA), using state-of-the-art computational techniques that can nimbly perform eigen and Singular Value Decomposition (SVD) analyses. By integrating memory mapping (mmap) and the latest matrix factorization libraries, our tool is fast and memory-efficient. To demonstrate the impressive performance of OCMA, we test it on a personal computer. For full eigen-decomposition, it solves an ordinary GRM ( N = 10,000) in 55 sec. For SVD, a commonly used faster alternative of full eigen-decomposition in genomic analyses, OCMA solves the top 200 singular values (SVs) in half an hour, top 2,000 SVs in 0.95 hr, and all 5,000 SVs in 1.77 hr based on a very large genotype matrix ( N = 1,000,000, M = 5,000) on the same personal computer. OCMA also supports multi-threading when running in a desktop or HPC cluster. Our OCMA tool can thus alleviate the computing bottleneck of classical analyses on large genomic matrices, and make it possible to scale up current and emerging analytical methods to big genomics data using lightweight computing resources." />
<meta name="DC.Contributor" content="Zhi Xiong" />
<meta name="DC.Contributor" content="Qingrun Zhang" />
<meta name="DC.Contributor" content="Alexander Platt" />
<meta name="DC.Contributor" content="Wenyuan Liao" />
<meta name="DC.Contributor" content="Xinghua Shi" />
<meta name="DC.Contributor" content="Gustavo de los Campos" />
<meta name="DC.Contributor" content="Quan Long" />
<meta name="article:published_time" content="2019-01-01" />
<meta name="article:section" content="Software and Data Resources" />
<meta name="citation_title" content="OCMA: Fast, Memory-Efficient Factorization of Prohibitively Large Relationship Matrices" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Matrices representing genetic relatedness among individuals (&lt;i&gt;i.e.&lt;/i&gt;, Genomic Relationship Matrices, GRMs) play a central role in genetic analysis. The eigen-decomposition of GRMs (or its alternative that generates fewer top singular values using genotype matrices) is a necessary step for many analyses including estimation of SNP-heritability, Principal Component Analysis (PCA), and genomic prediction. However, the GRMs and genotype matrices provided by modern biobanks are too large to be stored in active memory. To accommodate the current and future “bigger-data”, we develop a disk-based tool, Out-of-Core Matrices Analyzer (OCMA), using state-of-the-art computational techniques that can nimbly perform eigen and Singular Value Decomposition (SVD) analyses. By integrating memory mapping (mmap) and the latest matrix factorization libraries, our tool is fast and memory-efficient. To demonstrate the impressive performance of OCMA, we test it on a personal computer. For full eigen-decomposition, it solves an ordinary GRM (&lt;i&gt;N&lt;/i&gt; = 10,000) in 55 sec. For SVD, a commonly used faster alternative of full eigen-decomposition in genomic analyses, OCMA solves the top 200 singular values (SVs) in half an hour, top 2,000 SVs in 0.95 hr, and all 5,000 SVs in 1.77 hr based on a very large genotype matrix (&lt;i&gt;N&lt;/i&gt; = 1,000,000, &lt;i&gt;M&lt;/i&gt; = 5,000) on the same personal computer. OCMA also supports multi-threading when running in a desktop or HPC cluster. Our OCMA tool can thus alleviate the computing bottleneck of classical analyses on large genomic matrices, and make it possible to scale up current and emerging analytical methods to big genomics data using lightweight computing resources.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="ggg;9/1/13" />
<meta name="citation_id" content="9/1/13" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/1/13" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/1/13.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/1/13.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/1/13.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200908" />
<meta name="citation_pmid" content="30482799" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Software and Data Resources" />
<meta name="citation_firstpage" content="13" />
<meta name="citation_lastpage" content="19" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zhi Xiong" />
<meta name="citation_author_institution" content="Department of Computer Science, Shantou University, China" />
<meta name="citation_author" content="Qingrun Zhang" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, University of Calgary, Canada" />
<meta name="citation_author_institution" content="Annie Charbonneau Cancer Institute, University of Calgary, Canada" />
<meta name="citation_author" content="Alexander Platt" />
<meta name="citation_author_institution" content="Center for Computational Genetics and Genomics, Temple University, USA" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3783-4781" />
<meta name="citation_author" content="Wenyuan Liao" />
<meta name="citation_author_institution" content="Department of Mathematics and Statistics, University of Calgary, Canada" />
<meta name="citation_author" content="Xinghua Shi" />
<meta name="citation_author_institution" content="Department of Bioinformatics and Genomics, University of North Carolina at Charlotte, USA" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4662-3177" />
<meta name="citation_author" content="Gustavo de los Campos" />
<meta name="citation_author_institution" content="Department of Epidemiology &amp; Biostatistics, Statistics &amp; Probability and Institute for Quantitative Health Science and Engineering, Michigan State University, USA" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5692-7129" />
<meta name="citation_author" content="Quan Long" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, University of Calgary, Canada" />
<meta name="citation_author_institution" content="Department of Medical Genetics, University of Calgary, Canada" />
<meta name="citation_author_institution" content="Department of Mathematics and Statistics, University of Calgary, Canada" />
<meta name="citation_author_institution" content="Alberta Childrens Hospital Research Institute, University of Calgary, Canada" />
<meta name="citation_author_email" content="quan.long@ucalgary.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3795-1873" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=F. Buettner;citation_author=K. N. Natarajan;citation_author=F. P. Casale;citation_author=V. Proserpio;citation_author=A. Scialdone;citation_title=Computational analysis of cell-to-cell heterogeneity in single-cell RNA-sequencing data reveals hidden subpopulations of cells.;citation_pages=155-160;citation_volume=33;citation_year=2015;citation_pmid=25599176;citation_doi=10.1038/nbt.3102" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=F. P. Casale;citation_author=B. Rakitsch;citation_author=C. Lippert;citation_author=O. Stegle;citation_title=Efficient set tests for the genetic analysis of correlated traits.;citation_pages=755-758;citation_volume=12;citation_year=2015;citation_pmid=26076425;citation_doi=10.1038/nmeth.3439" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=H. Chen;citation_author=M. Iqbal;citation_author=R.-C. Yang;citation_author=D. Spaner;citation_title=Effect of Lr34/Yr18 on agronomic and quality traits in a spring wheat mapping population and implications for breeding.;citation_pages=53;citation_volume=36;citation_year=2016;citation_doi=10.1007/s11032-016-0478-7" />
<meta name="citation_reference" content="citation_title=Mol. Breed.;citation_author=Y. Chen;citation_author=X. Ren;citation_author=Y. Zheng;citation_author=X. Zhou;citation_author=L. Huang;citation_title=Genetic mapping of yield traits using RIL population derived from Fuchuan Dahuasheng and ICG6375 of peanut (Arachis hypogaea L.).;citation_pages=17;citation_volume=37;citation_year=2017;citation_doi=10.1007/s11032-016-0587-3" />
<meta name="citation_reference" content="citation_journal_title=Methods Mol. Biol.;citation_author=S. A. Clark;citation_author=J. van der Werf;citation_title=Genomic best linear unbiased prediction (gBLUP) for the estimation of genomic breeding values.;citation_pages=321-330;citation_volume=1019;citation_year=2013;citation_doi=10.1007/978-1-62703-447-0_13" />
<meta name="citation_reference" content="citation_journal_title=Lancet;citation_journal_abbrev=Lancet;citation_author=R. Collins;citation_title=What makes UK Biobank special?;citation_pages=1173-1174;citation_volume=379;citation_year=2012;citation_issue=9822;citation_pmid=22463865;citation_doi=10.1016/S0140-6736(12)60404-8" />
<meta name="citation_reference" content="de los Campos, G., and A. Grueneberg, 2017 A Suite of Packages for Analysis of Big Genomic Data [R package BGData version 1.0.0]. https://github.com/QuantGen/BGData" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. de los Campos;citation_author=D. Sorensen;citation_author=D. Gianola;citation_title=Genomic heritability: What is it?;citation_pages=e1005048;citation_volume=11;citation_year=2015;citation_pmid=25942577;citation_doi=10.1371/journal.pgen.1005048" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=K. J. Galinsky;citation_author=G. Bhatia;citation_author=P.-R. Loh;citation_author=S. Georgiev;citation_author=S. Mukherjee;citation_title=Fast principal-component analysis reveals convergent evolution of ADH1B in Europe and East Asia.;citation_pages=456-472;citation_volume=98;citation_year=2016;citation_doi=10.1016/j.ajhg.2015.12.022" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jia;citation_author=J.-L. Jannink;citation_title=Multiple-trait genomic selection methods increase genetic value prediction accuracy.;citation_volume=192;citation_year=2012;citation_pmid=23086217;citation_doi=10.1534/genetics.112.144246" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. M. Kang;citation_author=J. H. Sul;citation_author=S. K. Service;citation_author=N. A. Zaitlen;citation_author=S.-Y. Kong;citation_title=Variance component model to account for sample structure in genome-wide association studies.;citation_pages=348-354;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208533;citation_doi=10.1038/ng.548" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Kim;citation_author=A. Grueneberg;citation_author=A. I. Vazquez;citation_author=S. Hsu;citation_author=G. de los Campos;citation_title=Will big data close the missing heritability gap?;citation_volume=207;citation_year=2017;citation_pmid=28893854;citation_doi=10.1534/genetics.117.300271" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. Koivula;citation_author=I. Strandén;citation_author=G. Su;citation_author=E. A. Mäntysaari;citation_title=Different methods to calculate genomic predictions--comparisons of BLUP at the single nucleotide polymorphism level (SNP-BLUP), BLUP at the individual level (G-BLUP), and the one-step approach (H-BLUP).;citation_pages=4065-4073;citation_volume=95;citation_year=2012;citation_issue=7;citation_pmid=22720963;citation_doi=10.3168/jds.2011-4874" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Korte;citation_author=B. J. Vilhjalmsson;citation_author=V. Segura;citation_author=A. Platt;citation_author=Q. Long;citation_title=A mixed-model approach for genome-wide association studies of correlated traits in structured populations.;citation_pages=1066-1071;citation_volume=44;citation_year=2012;citation_issue=9;citation_pmid=22902788;citation_doi=10.1038/ng.2376" />
<meta name="citation_reference" content="citation_journal_title=Proc. IEEE Int. Conf. Big Data;citation_author=Z. Lin;citation_author=M. Kahng;citation_author=K. M. Sabrin;citation_author=D. H. P. Chau;citation_author=H. Lee;citation_title=MMap: Fast billion-scale graph computation on a PC via memory mapping.;citation_pages=159-164;citation_volume=2014;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. Lippert;citation_author=J. Listgarten;citation_author=Y. Liu;citation_author=C. M. Kadie;citation_author=R. I. Davidson;citation_title=FaST linear mixed models for genome-wide association studies.;citation_pages=833-835;citation_volume=8;citation_year=2011;citation_issue=10;citation_pmid=21892150;citation_doi=10.1038/nmeth.1681" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. Listgarten;citation_author=C. Lippert;citation_author=D. Heckerman;citation_title=FaST-LMM-Select for addressing confounding from spatial structure and rare variants.;citation_pages=470-471;citation_volume=45;citation_year=2013;citation_pmid=23619783;citation_doi=10.1038/ng.2620" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=J. Listgarten;citation_author=C. Lippert;citation_author=C. M. Kadie;citation_author=R. I. Davidson;citation_author=E. Eskin;citation_title=Improved linear mixed models for genome-wide association studies.;citation_pages=525-526;citation_volume=9;citation_year=2012;citation_issue=6;citation_pmid=22669648;citation_doi=10.1038/nmeth.2037" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Q. Long;citation_author=Q. Zhang;citation_author=B. J. Vilhjalmsson;citation_author=P. Forai;citation_author=Ü. Seren;citation_title=JAWAMix5: an out-of-core HDF5-based java implementation of whole-genome association studies using mixed models.;citation_volume=29;citation_year=2013;citation_pmid=23479353;citation_doi=10.1093/bioinformatics/btt122" />
<meta name="citation_reference" content="citation_journal_title=Comput. Phys. Commun.;citation_author=T. P. Lou;citation_author=B. Ludewigt;citation_title=MMAPDNG: A new, fast code backed by a memory-mapped database for simulating delayed γ-ray emission with MCNPX package.;citation_pages=10-17;citation_volume=194;citation_year=2015;citation_doi=10.1016/j.cpc.2015.04.005" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=Y. Masuda;citation_author=I. Misztal;citation_author=S. Tsuruta;citation_author=A. Legarra;citation_author=I. Aguilar;citation_title=Implementation of genomic recursions in single-step genomic best linear unbiased predictor for US Holsteins with a large number of genotyped animals.;citation_pages=1968-1974;citation_volume=99;citation_year=2016;citation_pmid=26805987;citation_doi=10.3168/jds.2015-10540" />
<meta name="citation_reference" content="citation_title=The Design and Implementation of the FreeBSD Operating System;citation_author=M. K. McKusick;citation_author=G. V. Neville-Neil;citation_author=R. N. M. Watson;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=O. A. Montesinos-López;citation_author=A. Montesinos-López;citation_author=J. Crossa;citation_author=F. H. Toledo;citation_author=O. Pérez-Hernández;citation_title=A Genomic Bayesian Multi-trait and Multi-environment Model.;citation_volume=6;citation_year=2016;citation_pmid=27342738;citation_doi=10.1534/g3.116.032359" />
<meta name="citation_reference" content="citation_title=bioRxiv;citation_author=R. Moore;citation_author=F. P. Casale;citation_author=M. J. Bonder;citation_author=D. Horta;citation_author=B. Consortium;citation_title=A linear mixed model approach to study multivariate gene-environment interactions.;citation_year=2018;citation_doi=10.1101/270611" />
<meta name="citation_reference" content="citation_journal_title=BMJ;citation_author=M. Peplow;citation_title=The 100,000 Genomes Project.;citation_volume=353;citation_year=2016;citation_pmid=27075170;citation_doi=10.1136/bmj.i1757" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez;citation_author=G. de los Campos;citation_title=Genome-wide regression and prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_journal_title=ACM Trans. Math. Softw.;citation_author=J. Poulson;citation_author=B. Marker;citation_author=R. A. van de Geijn;citation_author=J. R. Hammond;citation_author=N. A. Romero;citation_title=Elemental: A new framework for distributed memory dense matrix computations.;citation_volume=39;citation_year=2013;citation_doi=10.1145/2427023.2427030" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=E. Rahmani;citation_author=N. Zaitlen;citation_author=Y. Baran;citation_author=C. Eng;citation_author=D. Hu;citation_title=Sparse PCA corrects for cell type heterogeneity in epigenome-wide association studies.;citation_pages=443-445;citation_volume=13;citation_year=2016;citation_pmid=27018579;citation_doi=10.1038/nmeth.3809" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. Ringnér;citation_title=What is principal component analysis?;citation_pages=303-304;citation_volume=26;citation_year=2008;citation_issue=3;citation_pmid=18327243;citation_doi=10.1038/nbt0308-303" />
<meta name="citation_reference" content="citation_journal_title=Curr. Bioinform.;citation_author=J. Salavert;citation_author=A. Tomás;citation_author=I. Medina;citation_author=K. Sadakane;citation_author=I. Blanquer;citation_title=Pair-end inexact mapping on hybrid GPU environments and out-of-core indexes.;citation_pages=459-469;citation_volume=11;citation_year=2016;citation_doi=10.2174/1574893611666160212235359" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=J. Salavert;citation_author=A. Tomás;citation_author=J. Tárraga;citation_author=I. Medina;citation_author=J. Dopazo;citation_title=Fast inexact mapping using advanced tree exploration on backward search methods.;citation_pages=18;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12859-014-0438-3" />
<meta name="citation_reference" content="citation_journal_title=ACM Trans. Storage;citation_author=N. Y. Song;citation_author=Y. Son;citation_author=H. Han;citation_author=H. Y. Yeom;citation_title=Efficient memory-mapped I/O on fast storage device.;citation_pages=19;citation_volume=12;citation_year=2016;citation_doi=10.1145/2846100" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=D. Speed;citation_author=D. J. Balding;citation_title=Relatedness in the post-genomic era: is it still useful?;citation_pages=33-44;citation_volume=16;citation_year=2015;citation_pmid=25404112;citation_doi=10.1038/nrg3821" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=O. Stegle;citation_author=L. Parts;citation_author=M. Piipari;citation_author=J. Winn;citation_author=R. Durbin;citation_title=Using probabilistic estimation of expression residuals (PEER) to obtain increased power and interpretability of gene expression analyses.;citation_pages=500-507;citation_volume=7;citation_year=2012;citation_issue=3;citation_pmid=22343431;citation_doi=10.1038/nprot.2011.457" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=S. Tsuruta;citation_author=I. Misztal;citation_author=I. Aguilar;citation_author=T. J. Lawlor;citation_title=Multiple-trait genomic evaluation of linear type traits using genomic and phenotypic data in US Holsteins.;citation_pages=4198-4204;citation_volume=94;citation_year=2011;citation_issue=8;citation_pmid=21787955;citation_doi=10.3168/jds.2011-4256" />
<meta name="citation_reference" content="citation_journal_title=Cluster Comput.;citation_author=B. Van Essen;citation_author=H. Hsieh;citation_author=S. Ames;citation_author=R. Pearce;citation_author=M. Gokhale;citation_title=DI-MMAP—a scalable memory-map runtime for out-of-core data-intensive applications.;citation_pages=15-28;citation_volume=18;citation_year=2015;citation_doi=10.1007/s10586-013-0309-0" />
<meta name="citation_reference" content="citation_title=High-Performance Computing on the Intel Xeon Phi;citation_author=E. Wang;citation_author=Q. Zhang;citation_author=B. Shen;citation_author=G. Zhang;citation_author=X. Lu;citation_title=Intel math kernel library;citation_year=2014;citation_doi=10.1007/978-3-319-06486-4" />
<meta name="citation_reference" content="citation_author=A high performance level-3 BLAS library for heterogeneous multi-GPU computing. Proceedings of the 2016 International Conference on Supercomputing 2016:20;citation_year=2016" />
<meta name="citation_reference" content="citation_author=HSPT: Practical implementation and efficient management of embedded shadow page tables for cross-ISA system virtual machines. Proceedings of the 11th ACM SIGPLAN/SIGOPS International Conference on Virtual Execution Environments;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. Yang;citation_author=S. H. Lee;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=GCTA: a tool for genome-wide complex trait analysis.;citation_pages=76-82;citation_volume=88;citation_year=2011;citation_issue=1;citation_pmid=21167468;citation_doi=10.1016/j.ajhg.2010.11.011" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="OCMA: Fast, Memory-Efficient Factorization of Prohibitively Large Relationship Matrices" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1.cover-source.jpg" />
<meta name="twitter:description" content="Matrices representing genetic relatedness among individuals ( i.e. , Genomic Relationship Matrices, GRMs) play a central role in genetic analysis. The eigen-decomposition of GRMs (or its alternative that generates fewer top singular values using genotype matrices) is a necessary step for many analyses including estimation of SNP-heritability, Principal Component Analysis (PCA), and genomic prediction. However, the GRMs and genotype matrices provided by modern biobanks are too large to be stored in active memory. To accommodate the current and future “bigger-data”, we develop a disk-based tool, Out-of-Core Matrices Analyzer (OCMA), using state-of-the-art computational techniques that can nimbly perform eigen and Singular Value Decomposition (SVD) analyses. By integrating memory mapping (mmap) and the latest matrix factorization libraries, our tool is fast and memory-efficient. To demonstrate the impressive performance of OCMA, we test it on a personal computer. For full eigen-decomposition, it solves an ordinary GRM ( N = 10,000) in 55 sec. For SVD, a commonly used faster alternative of full eigen-decomposition in genomic analyses, OCMA solves the top 200 singular values (SVs) in half an hour, top 2,000 SVs in 0.95 hr, and all 5,000 SVs in 1.77 hr based on a very large genotype matrix ( N = 1,000,000, M = 5,000) on the same personal computer. OCMA also supports multi-threading when running in a desktop or HPC cluster. Our OCMA tool can thus alleviate the computing bottleneck of classical analyses on large genomic matrices, and make it possible to scale up current and emerging analytical methods to big genomics data using lightweight computing resources." />
<meta name="og-title" property="og:title" content="OCMA: Fast, Memory-Efficient Factorization of Prohibitively Large Relationship Matrices" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/1/13" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Matrices representing genetic relatedness among individuals ( i.e. , Genomic Relationship Matrices, GRMs) play a central role in genetic analysis. The eigen-decomposition of GRMs (or its alternative that generates fewer top singular values using genotype matrices) is a necessary step for many analyses including estimation of SNP-heritability, Principal Component Analysis (PCA), and genomic prediction. However, the GRMs and genotype matrices provided by modern biobanks are too large to be stored in active memory. To accommodate the current and future “bigger-data”, we develop a disk-based tool, Out-of-Core Matrices Analyzer (OCMA), using state-of-the-art computational techniques that can nimbly perform eigen and Singular Value Decomposition (SVD) analyses. By integrating memory mapping (mmap) and the latest matrix factorization libraries, our tool is fast and memory-efficient. To demonstrate the impressive performance of OCMA, we test it on a personal computer. For full eigen-decomposition, it solves an ordinary GRM ( N = 10,000) in 55 sec. For SVD, a commonly used faster alternative of full eigen-decomposition in genomic analyses, OCMA solves the top 200 singular values (SVs) in half an hour, top 2,000 SVs in 0.95 hr, and all 5,000 SVs in 1.77 hr based on a very large genotype matrix ( N = 1,000,000, M = 5,000) on the same personal computer. OCMA also supports multi-threading when running in a desktop or HPC cluster. Our OCMA tool can thus alleviate the computing bottleneck of classical analyses on large genomic matrices, and make it possible to scale up current and emerging analytical methods to big genomics data using lightweight computing resources." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1.cover-source.jpg" />
    <title>OCMA: Fast, Memory-Efficient Factorization of Prohibitively Large Relationship Matrices | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"tcISgJ4ueR4yQ7s3UBcUlLwiwr5N35khqvjJ6yhYp_A","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491784","apath":"\/ggg\/9\/1\/13.atom","pisa":"ggg;9\/1\/13","ac":{"\/ggg\/9\/1\/13.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/1\/13.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/1\/13"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/1\/13"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/1\/13":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NvO0g\\\\\\\/QvU94EgZTwomXkQQapYc\\\\\\\/wSOerjDrq03EaAvjNl+hlLIgwk3NVSXpunZwCUucvnqCGMsdSD46It1uyodQBKj3LK8q6dwcI3DypCtDdrf82V\\\\\\\/S3jAMNoobKAMV+Ne4C1Yhhl7PUy8Xeh2zVyDezLOGcXdhFLf2yL+5+gJ8traaaLVqvrdsxUOYAuUGKUkI5Wz8t9nrLdmMYmdIUMd9avpQXsDEwnyOffiqlhb4IudN6dlsCKRUEIhvDkaZre0UnLwOL+\\\\\\\/mpR4laIi\\\\\\\/shRwvBx3s7mUX9CT4UTLEzmGcARoMTznmvaUMZRciPNVJX4SVazTxYFKzh41MZ\\\\\\\/86TraAahPNha2lumCZ0eq8NqTl5W0Xq2pgj\\\\\\\/R7c4O3xHHAHXfOnpXDVynZBXCma8T\\\\\\\/oDEsMTfRNYOBnavZilqKmjdcUvwuDWWCxrK0GJQtVPmLNMqgLOkXLs3\\\\\\\/v8jLjLrkJrMgh1BUn30yFEkl30X1d0lVe9O9nRv+8\\\\\\\/zYKVbePdpfhkT+VO+hPuHhtEqQzQ8FR+WWH4Uofbffatv0G7+j4QJLTfyGrGNeha6xaU6sqjlGWoALDyRt+WAo8K4ICF25a6VBN+3W+5nPxTnYBV4WNqC209YnmnVeUVizl3+JDX0gXWBO9XNQPoEzTdsgzRTfPdc68OwOn+026DLTN7LjynJqxYbwBdAd+tuW14GJ0tQOKNkvFI2JQTYJ7+lfjucJBLYHEW+PLBSesjtH3j+p3dpVYniPj\\\\\\\/bb67o4UUKRbERe1HsqCogSTn9ZHPO\\\\\\\/YeW3mmfuVWt4BerdSgMQ9RSBSLnstAEyYTNKuOVBzVf2ssvuRAmUt37dQ684S5UdVU2PI0Twaloh39daFGYiSIaKJlaD1zQ1zz62z4Jc\\\\\\\/xAkAfxbYIkn\\\\\\\/IqRABlt688v0LhOB6cbzq+pmBURC63fpHIy3nfpeIJhna5A3WDD+vhRQBoVNtyiZY4N4XBGD33JPXp7pHc\\\\\\\/GqIGvP+4ofJKaZjU2XhspwrFmMsSTXN\\\\\\\/SWmoQ6i13ssrzeCjpIVGn6a0pCcEWhM4F5ChQvS96d4P89qDz9XyI+b6EpnOU7Z2VLOWXNBKp\\\\\\\/\\\\\\\/o4QpO4B1Px7NBKKUyjSGs+vqXzgpp+jwc0QpBYa7qVgrDW9AS5v915qSestghPFZ6Zlshu2En+DxT+BwwwTyN0onaueXHeX1P8s5WR9fuc9S49SLATFZm7GA8HGDRvK8eaVIwhiqbtnsTn27pc4C7AJVY+HhVbCJ+cxKDtY4zfju\\\\\\\/kMjwZFNO6r3rrjUp\\\u0022,\\\u0022iv\\\u0022:\\\u002269gjv3rb9WMvFjBmInhxeg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227c303dd3e31823fdaf57842ae323d0fb5a69e49ff0cd77e3d403e7032c673ea8\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LM\\\\\\\/\\\\\\\/JR06IegbHELedSSuU5sVa3\\\\\\\/xxZT\\\\\\\/g7TOhZq++PC03DkJVSuo1r5+PfdXjHjbcUgFnN1QjkQ2cFlgHA59xZIThQRnAcb+2jiLpTfZ9JsnZ8lxLmsT291uqeuaEY+ZpENlrvDu0tIgOrkk2ZrANnM8+Zj\\\\\\\/qHTW\\\\\\\/Xh4F7ADt3SYeLWENq5UUQ4cK+YbN3tfGJpXasts\\\\\\\/25obDJj3p+iQXWABNY5DGPZBOa1M87UJFISzjTGBB05zImp7vAx1kgeNovDCLUwIHc7crhnVJcBzbHP+e4Q3Xb1lxk2aUaU2PPNZQbRIp9oJR96TpQ36rNzyjbMOZRptzLbPMSjOILYiMMpG+881zVtOHGP7xm3ZPnAphVEJayjtXcVKvLTcs5eqZYMh0V2jFUEEmF71zyINQRidVGwh+oaLz6QaYwT7xh7F8RKOI6n8BmB9\\\\\\\/kRMAU5wYjW5rgv20CtT8L02TYWAztjjcw82wYWWwFdEkHn9F20l7KPXttT\\\\\\\/2VQvi2JwESoae5CMxqVCOtK5iXXZK\\\\\\\/RiYD\\\\\\\/6wGXtVbW1BkVgnHgS\\\\\\\/V1ctppb\\\\\\\/r0aS7Mc9FOm1pjOvNmLqhSUhNWh3q3gUEn6iJIBAcg5Yq7t+0u3hlgalwTvA9hxLTgyArB40t61a+aGHe42RCxPgyJqBLRf2LFkNqorSFAULjSkuPJtbhS70XEQTNwpaGStcKtB6tSodN1cUuZ6EDizGyYlS4hE7x8DdcZMxhJFrHNetq6euUsoZsfWDA0MHYOcuCfkH67Q7cO0evnpbpeFGgzueFqzv2yE2M6ejTOCx3ZNANk0NvIsMnFR7Wvh\\\\\\\/995YrXgC3NE93WupE3HtXGtPYv1F\\\\\\\/YdZn2HKY6kc\\\\\\\/OB2vzWYYBNBe6HGt48txb+i33KgHOd+AUNCVksdxSkLNBqPGYNndJEjRmeWAzi5e7nQFzAicdQev05+N\\\\\\\/Wul1egm\\\\\\\/1gL+PZxsfoGYR56zFlPlKE05CTP81gqlig1cHG6Mt3H4EMaea2Uas0Fn7pCTt1aDGejz5EaSac5gi\\\\\\\/ezNd0\\\\\\\/OScCzNfOlVTis0RLbYYmmNPYVfZAHhX3g6MbHCYSXI6GDQ6cPHax5d6ITG7Ww4bYyJcrruprRUe3CQQ0OKEKhsytvQPV2eomS6IbQaJA7SoGc08IgQCq9Iuw+A5zk3XoHEQ320EoHP55in+rhlUCNDyk+uoQ7pD9wbOe06CVCKnmg6jpAOQ\\\\\\\/PykA\\\\\\\/hk0p0vnD50Qb\\\\\\\/yhzaoxcEYhvD5kxHiN52QTOQFduKfLfPfHFoeXqqual6JVljLPkzCDxNcpu0PiLE0eJBWwrY1oyF9NWwlbtmraVRJUUwU51ABi43YGoHToZccrX5z0D25DM7mK2upGdiGY5h\\\\\\\/K1LPqv8aG7NPYIwC5ERaEqadIe0lHO7v879Hx44ArfRSy5BYwfjTN\\\\\\\/kUgJUlQf\\\\\\\/UdfGFQoFkpvRpvPXXD5LAEMNH2BPpDhzLeTWKTO3kag63ZbTAhwR+yFumbUPCti6NCQPcLRlVpP41MWebIuG7OBXO4pCdFvJ\\\\\\\/p2ecf9Q+Rymwumst3xKCEdnDLdxn0l7y6V2Vgp69G0l0TjD5iJ3UyAFjalGr+rEb+bEQQsAb2lmINuovTVkm6ZUaa\\\\\\\/CzXHvp+94EfgF\\\\\\\/BXsPUkvDQpA9RD58+koM+D1lfeWm3v5d91ryVuy6rAAxOFVd4G+zoHshRc9KK7JRXBD46Lbe7NUTB8gw\\\\\\\/mVp8jrqpirPoW508nSVEaEQg30BfWNDCXbK+2IlfHQpU2ccwTXUG9S0oAJKmkf9QCT8NbbkainjtFBhJQ2nom5qn0dEBBwzixh0dvouh4V\\\\\\\/rqFBipSsq239gzutW6mVDZ1F+\\\\\\\/jIAabXh7TZLyLcN1LtIiOt0mZ2YC7vIqvLMJGUIlzD0o4akNyVukLpExBfYBdQDtW69O+JbCdd1eqitdhHEcEos\\\\\\\/Mndsn11Jvr+AzI1GGrezAwNFpPW5KashOtZEQ3fatAQjjoInzbo3eaav3msRRS77kdMS7qAqdlOvjmVLSCDlt1F0JB9rzgvNwM6TyWT4b\\\\\\\/QtkFc202uMzE1O1w3ONraskKrmOfBsPBBCeGXASuYI86YLLA\\\\\\\/z8FdTpvjkA+NYYUv\\\\\\\/nFvvZmx2iCszMEheCy\\\\\\\/+2xsXI9w8fuk4H7XdVpsfo8DbCSO0ORzitFqQr2hJ3I6zudTcYEwrAxCyVNyS1IHfffzoSbb6tN2ofF0GuXXr31\\\\\\\/vZg0kv\\\\\\\/kRAnAM53eR+9P1PueZnHCNx+4CDpUJxx4CaWbuCa4YcFiZixLiaDGLII0a84jV8bpLfbRNBLTAzolYmd\\\\\\\/FV6iBEbYfBIMbcygIxdJxxtAcYNUYFwAqyx45i3Ax5d89CSRqwazbf989ftVM4+X9dPGeHpqVioVbJEjFy\\\\\\\/RZHpD7huIWqqhQ30vLNBv33wBkxH\\\\\\\/zH0BrbkArLpveg8rZQnZVxMDZnYbMpsuTCA0lU24DfO4CK83p\\\\\\\/aU07fuOUGuzqYKq7F8rnnYPrnvuJLvKBeLL37O9FMBCJuyhpR2sESLQoEDgZ0JrBQU2D2Crv0y3YVUmVpfjUcLaDGdePFFfW0OM14WqK8fba+t\\\\\\\/2GatLzQ0AeHDRwDN9wbBsqwxIEpINS0riWXHkrpMMoDjb8OAX+a2ekEr30iO8qSz4DuZIKxMrEyoFrtmJIQuT8F5mHHtEalQhXT5Nm2VjC0dNfqYPMDdmCxumdnamaDi2HrhlZNHfhX3QPVbEcpB2Tx\\\\\\\/ShJCOV+C10b1+S4DF0QiNrzKheSj4u0jvxfEXd6XDJB7BqHMWKDfr2D7KGzYJJtLgCm5\\\\\\\/lR94ItG8DTNpmv0nVwN0WKiOah5rcW8esAw4A5K080XBmPE4mZyEMyJ0b50ls+m+9als1tmGBaFRjebZFW\\\\\\\/NRfHgjUzelqr\\\\\\\/6uFw3IJ1hNyIQ5CC2oSZKtfSbGGn1\\\\\\\/OJljIBl7zaypLMfzc4cPACqqyZIR1Ac94bRzKqv9cxgfgevZfT0JUEUPqvL9r1DIdxVkh\\\\\\\/6RV8zCMIlYXTuUphtDnNItO0DY6Hv1d960L3hbYed4onFBG0ceHeq1caBWF22wmCFNIJNI2jW7zxQa1AMgPBcAQjbncEiKtfFaA6OrQaCRKwmEOuf839ifJ5CJWq6tsudaGZL1Z3BH8wltOMsIp8o3zPaVMl5fwa76sWTvhoUn52AZjZbfYUeRozR5QC8c2X5isHfy3m+2y9bxbfNJxM4OVYlrZyzRl09vvxjQJ39DikputT2BVSwzyWJUASh6Kl7\\\\\\\/vHK46+OCmbbBeCW0B+nsFgMO+CDx+FH5zr3HPcp2OCXU0PBQvtfdhXIop15dVX1IzY9LK9KBlcghpk+mubFLELiYyQY432+Q83wUPACF1yT1HcQY2YJnk0nTF\\\\\\\/uFafM1AVVbtD1LlRG8nWdE71vBsHUdTBRJ5gVXrnvGulrgXKwqolXTTzCrM7o2RsGVtNYmSZMkjfb36PWcuOjgOxiweB4ZlOviemsvfLJN4TDt3QUNdiqvalGQFsIU4EmC6qgPAAmhbx1QCB69t4x\\\\\\\/sQEI7ibiryZ2cq7tWRiF6xSLVJAwnzG1IJ50KRPkOXz2KKPPELweoHT2KeyFu+n9KC2HGtslDsFmxUYJ\\\\\\\/tQ+74ECpyHZaLrR\\\\\\\/BJ+AOumu1fOFggLQQkb2eRP08KvxDGn68yJCAldtdPnJUSlhwr5qePxKfyoZ6Z9j2Mmhpsb9JOXWmgQV0GD4PFySM0+b5kllBTxAEfsfiXov7nb+7XL9ppedMson+VwKWRPDkkGmytDNvKkJWjLvyyrk4+2B4YatHvnGZS1\\\\\\\/C5Ng\\\\\\\/2XWNqhS1rnage2C4u6v8MbQNi2vPMe8Ef7nmxFyR79ZRI8owRyMHABtloG\\\\\\\/saHOQsFIeQL7ReFVR+r\\\\\\\/unOq4vc9tbgTjO2TH4VdlZDDufmraKIcuph\\\\\\\/z7uIspxoH3hPerD8fU\\\\\\\/S8AWujarx1UGfB9Ty05Qm2ECT4jvnLOLvVl7AoJrb3kVgV6HAhJZiej\\\\\\\/SCrSOFIYJokGrvuqzbkySsdawe0cP6vbAf\\\\\\\/9tYPst6o6HWmU+KB1hBoG11n3UR6Z2GMskHpuLFNzSANWd\\\\\\\/RkVx8vZIL\\\\\\\/QAcKdqO532OvUNahZ82L5rcBxHghpFdXDRh1STYMDq0YmdJNsxLeUtrqF425Oia58Jibizpy5pK4TlKm7zDGN\\\\\\\/rHy0SiM0wsZ6QBAIPDwaLEzTLYLmkhhJ+iqP4d9Wd+QJ1JFWKGtgvrXIQMj3yJeuH2Eb1Jq\\\\\\\/gm9rBeR3lKaGhFMB+cNkdZOsboTJSU9KxU7l3W2Aw4idAJgI3lYcsZxbnQohv952KExAbd4gsDC1PEHDsOoUcZch8NxvAhJ5lgHcEoaDOjNTHAvORi4Sk9Z3Y9NqIsOnyKtcsj8qWb5ta4FcS8nnbYBJa3lOxOSDy3451Eavrql0T+gunrxle6G\\\\\\\/FL+e+3rNKAY5ppiRmIywmAwLLQtkjuAZ1LOJBGdBtC6s7KM6B\\\\\\\/juRM2XAjQOD6fl5r2Cv0O67GnqQFJw8ErN68kusLjM6+qyTEf\\\\\\\/LCxcYx4xmdLpfXWq880+Jp1d3DrwMr2vCl5kAKl+c57WiKDVD8uKZC2xSisPT0zO6c5PTda8eP11zfIgT2KiP3mi0JvL2dppzMYwbsMSCJpJg3QLbLD+cu4o9d+ltFRwzRQ4tSbsPkgfIJosqV0EtrdIJgERFKEdS09HxiN2cHDg+sVYjo72A6ZHbL6On371s9p3Oac5Vp7PoOlBHuMWCWSqPU9iwN5d12F\\\\\\\/TVnfdtHIam+0eDt\\\\\\\/Ts3W9jH+Mykz93l4fcW83lTQISLG0Sw16F0EkZCaoL5E5HZ5HqDIGe7lLgKSlsN2++Abr+Hea3xXSkL9zzqCFbhJyOwskElL9pmOYnZwKV4ahAwEqGMqcCLPe9cHfxdpVu+nQDScLuMrsRFPpMrr85fx5Xd25CiLnn2hrPO91P8L+0wK3dmmoVuDDvUyU3SIYLjMY9F+5GB6CLY\\\\\\\/S8UkjnaDRnysFUBg3wOzw\\\\\\\/+1mQvXsHnKaM6\\\\\\\/bgPO+eJth6ZP6SOHrqbJ1wrGJYnq3smtNxEiyTrs4RNm+Go19aZ0eEoitAB44FvwaNFp6biDwLLCq8G3e36ZE1BZVkK6Ke1tX0uDx2PkegI4PdxhymHmf7GPWJsNpFImKXsWQv0yPDBQwEqWpBr9B7tFcCmhyXiP+CVqYNs6KPVGm2UwlLAxQR++ctSTWF4b6m8Bz\\\\\\\/Z4MZSks3IM8V9TIKSZWOcePSi3XmgehCe9Xuwb1DVBFgoTe20Fe5cSSQ6NZnSp\\\\\\\/zSpcaQF3iiQhrMBHkQzxcu8MJzjVcP5XnuGb+X0G\\\\\\\/zpjr\\\\\\\/EgdEgwLXqiC0r55EXNb1v\\\\\\\/OOMyQe21oA2ZcZQqlqvAs5YuhPsHThYzKYtSDhm5G4cklFTMIS6KlPkIwmx0WA0ExGp0jBytzQlOUt5cqbAsUeSqcs+mzi2NonELqM0Zke1\\\\\\\/QAu7BXjMrJEVPz4NV9wiCAgfP+Tv8djrnzvJDbneiQFPZoMWH41Wq8jJVAt5P9Ej1WTJevy3ooV\\\\\\\/8zVRz79HsVlNUB\\\\\\\/5JBPafIrPBwfwzNXPP8QZLn\\\\\\\/uu6Vg\\\\\\\/Dw+nSt7kbj2C9ApbzBEPXniECfa7COmPl\\\\\\\/EdWbBEH6k\\\\\\\/5cjo0nBilfH6iTWVNHxCN+WUqIkBKLT+oAtOOpJNHRuYNbl3bHM9\\\\\\\/FS8SRHNBPUy+hwVJ4lurFd542fTxLblKDUvGtzQGZAQIrhQky61ySCkgokrhZLXktZbtA1587RBSU6a0aacK0TGhYqxFxAFXcat2kzS0MUiKg7IrTULkuwYI3SdHG+Xeh4sjG45b2CVLFUug+00x8fMcUTYTvtJjBX0qDA49BmZ2iH\\\\\\\/dY8oEJHpRKa9yrC3KFbAV8aWKaaJHRDDehAbzsTSRoVHzE7Vu5C6UMBzeVtocGWYLkKE+IcBKgKa4XG5QHm9+zhmXOLOCASnlG1TwaxVnV7BduAfhkZyQNDa9rdmZDXcRdkiRpSm8p6WoSNEXFQw3akcydDpQvg4Zm6f2uOEva17BYwjcrV+iA6wZTlhqxmeMnCx42YU774s0nxnl1+4bTD8vhPvnOx33eG1GxZftYntwWyTIG0OjhIpk3ewG+zxGxKfUZzupigi52cSoCp2wc313SrCpGtdUGdsFHF5uWIk5R\\\\\\\/4kDzOyJVHwZ7simwewOwpK8JeP5h3Ci1sXBIUmrUDfcReF4k4UK7iY8rl2K3LNuirIYji7zAEQNA4OxTnIl7KoqAo5SLcG0AqlwEZ6OLZ5h\\\\\\\/AtWT2pUn5iwVhGJpbED7hWRJMlJVtNfDrA0FhMaeZ6xd94fK6maQfOSWDcRjNAOeF4ld7FXpiV+QTDwEG49gwTHM+9XVl21eIFZbYqnodZt3aAjGUZ008DrdsugQtLimW70kL8\\\\\\\/bh9wrxYgoSLN\\\\\\\/NddCHT4n6e90HATEzNqTgo6sZVRsIgTBWM4yKl0arZbbeMWAJiTY7R1IbG11M3WyfLIpq1UepQy1lEIdDuBt6rUACt64HNnV5rUTp5dsfB4oGSXk2hVlSZiSoRabs\\\\\\\/X4atnpW2KwB7hm29cN7h\\\\\\\/ZTDXIfaZnfhR78Ge0FLoXUW4SEuMjnbpLyvTwK4u2S2DZhj09BINy1Uev8e+JnIGF1Mydi4z7lnBvOWVgRSbYgROGRT8oJCbcYb0uh\\\\\\\/jFN9gpg7HggbYnc5ITsY7kFz70eD\\\\\\\/lTJmH\\\\\\\/0R9W2DD4FUoK33hzw841KjMgIDfIrt5q\\\\\\\/jX2K2w1iM5T1xZNb4iGypA\\\\\\\/BZRECU5zg3Ap2TSJouXv1Vp7GsrGnew4lSJog5+Tp6bWHvwFx5p8gAkjWsOcDI7C+nTcUnh0VVq49ppIWyK6TE0iOF9z6KuBP5s9dSpyI6\\\\\\\/W5dkqElL5geN\\\\\\\/WkTX0op5Ry0yHK8a3vA5NPMAaozzoQW3XDh5wkNKMwF2uKtZNoHjf1Hso7LL0Y\\\\\\\/Mh+E8VwOqHSFCLMsEVs8Ks\\\\\\\/hJRDA80eRTNhgqAtKPsAdIZ\\\\\\\/dcw3Kxs1xaoLs4PwMjIM\\\\\\\/jfl8wT7pebVHGjSAEj40M5+2vQBRO1Gicn0ClxQ7CYs9bXXSq4jOMG2UjMaC5JflUrDDqZ\\\\\\\/6+SEAXyarTvVhetju6gCNbZpBhUwdvxyds+c13by3KFdcJpLcU2JRGP8kCeX\\\\\\\/6Dt29EB9d674D+vOD4HjbW1O6LKtbT\\\\\\\/fenqUrXZ28r7q6x\\\\\\\/bmkEbnxg+kkDbm8Ux4w4w4ZWNQIsuSqqN62cZBOykjDePT+t3kHviX09RQInI5+eF7gvTFUTF0lD4\\\\\\\/ii\\\\\\\/wdF+25mqIWSwqQEtmAtPeLGeMNs\\\\\\\/TyBLrsxSHHU8s1TFOBOx\\\\\\\/GsCxI22Gcg++AQPHzy4s3UFpP9fuAPlyhxuO1Bkj9oghpDZnI\\\\\\\/hD1RMHUj+Bw8phgdPKARK01kG2T7ZBx1xQJCbijzvBPnaEyWVjITlyrJBfGyg\\\\\\\/j46h9c3m\\\\\\\/kJFAwAqpazYJENcvYIhHZdKB+DNavx45Se2STk9YhDCm6a678I6XMcOrFmutXt0pLwTEg5ggEigKRAGcOOXDX3vec4yucyekFR4P7yWvkCQwAT9OCN8UvdC9wU81gh5EOL3Uxaj0gGVeiu2gZqxl7nJjEt2O7XvXva6ojzxUU8hpiC+WAN0pP7XN1dXH7mENs2GEWl7WSSNCRVOxN5kKN1sw9p4DGUrdEWwF1\\\\\\\/Hooz7DiEf3\\\\\\\/ArovIlwzpZS99fdaSZCNy0TLaIJvGhX7fkQ\\\\\\\/aDdw9IC2TH5Frs4dCXPhcTl9w2QzYgVe\\\\\\\/E2pmxQEJg\\\\\\\/u+YRsGW96Z0qWcFLUw8vLgDGY5wpGKmTyd+hZyL58ggg0zMZPcSpDf2rzsr0Bf9NYOFLkGTNLbI2p8hGZY+FxCbLjW5wvTWcNvUByuwGO7blwFRT9pCDyTd\\\\\\\/WSdHJ4v07RAdv8iVY9YDbgZTuGt5tWbFsgP7HDY7LSJMdwZRkKA0SSWTc6yeddg582wmOXEVHJ3S46NFKH3aQqgeHJfwHULvPu5HZPnYFRmOSAhLf8vjAoF0j5zOkILDUhb1N+iso5bwziBo8RG952HML0Qc3Or0yNvPqwfijca95Em\\\\\\\/f1rOSbue96wSUQcbyXbklVoKLgXL37ltWPoIPVTzfO6peYRWpQ0uh7URbmwazdIQrYRbi6H5ThTlO4QPA9FH4U1Yp1aW7l7svFNR20gkT5EjQWAXikjQL1uIdEqIXdZUjPoKz52+3HvY16VrCmD2MNA\\\\\\\/Cvjhn4kg2KWjFkKvHNIFU+abSqnnHoBimATHYNjAaJnerfrhyczANaKu12oK1A0JSusUJ6w++toTcfh4UCA7a8egUmyBbPoA+UoG8RTW2UPbJHIJA2T1XItPjvAA94u9V11qqx\\\\\\\/nPJ6h1W1vH1vOqgjflR02GRPqZbfL+bl+9C+W89xhbEzanLbPnq2bfAF2BOVZ\\\\\\\/YCao0iyK4718Set7+jwuPhFm0MblaaS1Ipj4DvcWl9P7v3uLJIf+fuC6+EcDiV+wB\\\\\\\/bbAQK1JtS3CuYbdWuJ76xpGQEVcdU7s2Z4ppPAfXKWUM4xA9fTt8fP9+bUZNil+qOAj9Zpfw65wmkr2yEHd0S+hs4JcJrc3jcXe+H+sUSGaqhDRsXjgu\\\\\\\/HhAKQJYK7eADtDZmhKIY3Bsr1PsQT3XSYepL7FVBF4xmRdYWovranT9Oq8h99M\\\\\\\/VdjAXhF0ev1\\\\\\\/GWZMScBAcStlJWuBZDk+FlA25aetoyz4xhADFeRq5CCx2pIbPIdZU8i5k\\\\\\\/5ySIRFRLWsAx+z5EW+wQPQZxqEUNWOqMeWu+WCxHXIYJGdXrAxvsaoSdQok+ZYgl3Nv7b02R4jUskExnnKGm6v2qE1kZINgB6wcGxziQItGA6XAaMRJIbfrVjz8A2RZ52pNNakru726kQDhuG4tDVO1Q661SDgQy7ceso1mrvLmeu1qWY1eC+seZMk4FNDJAR2rlo9saHtcLGM4c\\\\\\\/p2V6\\\\\\\/v7uCvJ5VFBEUylH11CSD3mslJ0LR7JLO67\\\\\\\/Kak7AVdxFM4NNUi2N9lzPyIjC45Pp6+ZyC8TbfRH\\\\\\\/1xQjYn8k\\\\\\\/AIrRflrAmxEryOkupwJUAv03eX\\\\\\\/D+l7KNh4VIDjqb2w3CinsvxnRdDZl337ab1nXBt1cx6f2IMDvtxY8i3ujwV4t1dIdZXQVmzTefSDbiClg80AnmXXFclPNNxmbUXu3lKcBXjfUo2Z+ovrHMbnZjKRrcZaCkVv+CVvOiGTP5JWuyXsiERTHCmuEeALeagFHI7a7BZqk1HvtWWtYS4bhq6dEO4gaWyOgPTM1r9QqJDb6AQoyqR0YJ5KiKUUnVFb2KF7l1QDsBnG18+gq9zzJQrH6c25L1iKU8u0gU0jFgw3vuUwZf\\\\\\\/14GgSjgs8vmEIBbcFBBsMZB+Voa48DZRrt0tYKQucvydRBQsCs+AJLTICzG9LcvYu\\\\\\\/Ain3dblZk3hf2DDroptf3BUs9PluiT7uSi6JnUiOxDkyyFo+4qEZJe+XyCbYvQLHoWYdlom3Jku2S3gpFN0ZLyqOxHTuUVDlnd9xlXKk49T2JvwTFEtCTxtQcBMRzoHDBppbKfBqFdwOxKmgYTZOShBJnDe7KWa8YXK+YM4QFqJ5XR6gjSfb0gD2g2a20mdUNFqdAS7CEkWisrK25EumZ51bLSFt6mgDMkWxCZI1Mg\\\\\\\/pCPcbNbgZoLp3gGv5BT6i9TW+ETAUed+TkAj3kCHH3+PAIQa2tZ57XmgyraXGfFrCD7KJ9X9mvQlhTVsDP28fSuhQV1yuGcSxcOPMRI5QzXQUefPrXLy2xbGAyPYseDiftcp100l\\\\\\\/GoN9JqciIV3vIuMYa1rmksGB+57+T2U2mwOyrDHcjgsmFwQ22Rpj4pL+\\\\\\\/UQlXwcRycKOxN+wWOLTV6uTMEnjPcEq5NawXfVVfSU4fijTtPazeQIewDOxAR\\\\\\\/RPCE6mJyMntVsYeCOovRfdXI+lrm3sZ89YXaz7vJj8qZQeSfsLovhH+6Zr+PnLWtvyIF8jLhpWpovAUJ12lwKA5y6rJyN85a\\\\\\\/bk8UoM+EIr0+8VHa5QHUEH0YJG7oj4RRJcr4PzmJpqp8QPZg0urnkzc7eKLu3CN\\\\\\\/EJwhyPBQBC2iBLK9CJCQF4TUn0ClP1cWdb9cUzvrdlG+stdvqfTWAsMwsNfsCld\\\\\\\/nmoOT1EL0fOueTwmsSBmheYFhdwgm4vq2O32wIDw+hn4A27KvYbaJdPpEy\\\\\\\/uKX0IP65zTlHR1sRTsOCU96bcRNdmXqlAEe2zJT4c4l0beFgia4rchFA6a+G6Kzw7wOlKiAJhk4ye1\\\\\\\/cW+dPNtK5k7y\\\\\\\/IX\\\\\\\/frLB6wAUG4PhBh6o77ve1C2TpukNygnZVSx90Z0Z6Su459k2Gdw8jQiRVqdBc0YUqMv9CsMid4DTQ7L1oRQEnbaD6JLXKjJWVGqVV+6bcguNjA4el6ZY9qqA7oDya1G5RX41ZQ2Qbq27LtPO5dJR70x22aWVGv1d1\\\\\\\/e8GBTxgdSZyF\\\\\\\/g28yD0WIAiW4uzuE968ZFbxxwTrFFuG1DSwpG25SxSXMWSp74unWuwEmadXLTBMR0TgPr2NwGMzdlBn8HsnQ3W9zBx9ZeS43+9XNpq0v1Zg3QLJHMr3lyxN8Edboop+7ZbkNMsiu7Ocy8G57bfDrcbcArlnchcaYHSgJbMmW2Nzo9jR9yo9P6v\\\\\\\/tlIfqE\\\\\\\/3n46oTj1OCS\\\\\\\/fbvambg1EBd9Gr1hsWGD52BqtJWT1hQfKk8ievr9NyyOj6ilmb30HDo42d1FAAd9sI3iw\\\\\\\/5OwMiekjwdge+dqo37aGJ2u7G7mfRbklQfCmYKa2+KqWWLMzucXO7G0JL\\\\\\\/AicsV45wbgkd5\\\\\\\/kgIAS61JCR4qN5iK1ZAruY7tHPK9uZAXRRdL0\\\\\\\/q+h37zAh\\\\\\\/lAVARds9yOJ+dkzDVloCR2qh0oXgDuk2KxNlPKWrUq2XdTEcD1afDLFYncLQW+vHU5ITuHfN+M2NuL3DSpKMZfjW8D7KAjuV0quT2jzzDnjtQ0K7elsnKsBdFc4pNQ0OlHGsg0RSD\\\\\\\/gv98JWPef5xYSUZujMGMCg8sEFeUchVaIG1Jsw90IfX8Y7CN5\\\\\\\/+XkfdqufJ60U48w2OjqLPslU1450MZjI+bk3wEgR0UC5aUp7bMS+wdNmHLkShdXgBEFLtkVcgCz5c3xbaPh9y472FhEbICatE58CHo8VeYk5eeRzoFlaL\\\\\\\/jRTOtpFgONAmow5C63yanCnBwODyTe310uKUj4x+rjl2CjCNG\\\\\\\/9pny7sNV0WkuD0Dc8rRNahhosw0q2ZcnmffeByIUqnen1rhf2GaOvVkEN2XEMwpcjCo\\\\\\\/Co8je5NeesjHFtXnE4CtaoOT1W0OA9P+2x7i7t2F4p+DqhduJkCcG2ht7wWcyrVFGBm+z8Jd5IOMJka8LTB+9Tt23i0mXdOtfxu7PjEgEc5D\\\\\\\/Jo87zL0AONk1KSMl40A04+odWlXn0gn1vQZZOmFlXxAjQP3bhXD9UY5TjKSwGPcsIlmY\\\\\\\/COlOgb21xOc7pNibCTpdbqu81m9G5jGTLaAT6JgBwOzQtO8TxCswQNUDStiQpg82g7gWHQFg8Ym4VpKB\\\\\\\/OVHEzlFYj0MJN2sb0s9Kn+aYVLf91uZO950QtQL2cyF\\\\\\\/AYLWuf\\\\\\\/eqGbweakTuN9DiU+q4nJN8jzqyL+O6IWXYuxxnTV1VXDUTO7quMyaJ3Viz\\\\\\\/to7qWxr6jPt2BbgHu0y4N0yngZrEKYeeoOPiPsxSKcnuAj4pS8L\\\\\\\/yDHduYOGb9guRYhRnXxoyyA8XXU1Nbttr5853IqnPC1qg5d\\\\\\\/pYGIeeDPQ2rE0xBnh31MThGKAfjqipy+c4\\\\\\\/rFgTvv97zxrArw7PRaRaqBwJN3TKhtWadKPOPf2EP03yemhkWHvqkBP1N+muFRy4qpujpPWg5frCwHHso5pmS\\\\\\\/2Ir98p4vEz7M3C\\\\\\\/pjLUYv1Tu2MnEFBY1PQywQv9JmjXwzI6pcvR0AOOm43TN9N4BY\\\\\\\/X+siZjAARmZdD5I4i1H5h5uNL9LtY8pv3uSL6pZyyFcUOZvroEXAn+kcPHDxTCwCZdtv2R2B0fsYTyOw99vbnESuD0DGFJj\\\\\\\/Lea\\\\\\\/LIYjbXOQHOWI0nYkSfuSrQ0HhBz5dP8HXgTdfMy+x1dRSXxHT5Q9Vi1nq9YaQlF790kweHEnUuP4p\\\\\\\/ZtI2grJunXuXuvj+sx5Je1F8dwZEEtgzXIXG8qnl4hGfXjRZjFDQmKKVJMuoay53FL7u365qc+3ES5iyeyOEk\\\\\\\/SVqdX1i5Z26MM7db+euylhP5SxEBNSVgrL\\\\\\\/NGUneijO\\\\\\\/l9tKV8cDBUwaOIZXaTTtQ\\\\\\\/7mSkpZLS2Q++VfEo06OZ8uHrkxrKfrfO6l0e\\\\\\\/lF+\\\\\\\/ZE\\\\\\\/G2+bhO36M\\\\\\\/eY+\\\\\\\/9mOQRfiCHiP8NjoKPfR+Vac3HsvxP4hYqXJTDUjEPzjQxB9XLLVVHjSCkR0nyE7k9dbruo9TuCS7jhwEmVseOB\\\\\\\/JBqxGtJUTJRu8lHdh35qlNjomvdc\\\\\\\/Pqrcb5JbiPGyo5hzWrvmdFLgzCYGqp\\\\\\\/PmyLJ8jH2KXtvzupuhblHqOrYztyzb9eJ8\\\\\\\/gBCpukYO+XFMdbCFafK72vdnZ2q5Ymmzo7QvHuBqqF+B+jMGtUdj5o\\\\\\\/J9LVC0FVO+fl+kZptE1hm8r5IKbn\\\\\\\/IB7PVsSczfDL0seARooGjej2L2e\\\\\\\/37QNkW\\\\\\\/LXGGbL6+3OUHf8GM8MzyzojXLbnpXLzIihwnTeiGnz3nkUzjlDs18sMyWGFqaDUZ3YNm1\\\\\\\/2XFUM1OkM+LfvXSogMAsBAoX\\\\\\\/GcFCD7wbvGnwXVqcMh9Rbj2tLoV\\\\\\\/DI9dvSaL0FfXnTXTZ93l31ZmeqrHcymQU+Qxa0nNolm5Ea3N3YIK45VrbbEO\\\\\\\/p8pNknaCjTe2N7VPQwZ7VMtUYzS2KPnxn3SuLojmdmHx6vdxt0vJzUF3Q8p7aXH3kGpQ4mIFoF3rHgY+lQFUQ4jRpGcPeEgFbAgTricp6x7c61ju7OwMs7ew\\\\\\\/L06hKEP8nEYIpR+1Ontx3NwyfDiXj5tANh\\\\\\\/smJnVKOMUFupvKtGdR5E4ErbJVZRyeJPr8rsYxjjk1Ox8Dzn5xpweD1etyRlwePluFcJm5lim32Iko5W7Mrs6iKEAX94s1nFD8wsRkSfTU8KeaBg6YaP2ShM1Vv6YxVJvI01DbkG3DsRsaZmWp2VZlVpGClz+HKz2nvqGDCMNrClg0ZP\\\\\\\/ScpgTZkeSE3+p467+h+KR\\\\\\\/f31MF8W8v9ZbbuLmJ+1qiyTjLJdHjB7Jy987YYwq39AZ1hl3EIVJnwcleDDGsnw9f3E7cVcWpPQR7NUwb5p2cBfGr1sg6j6Q5c0c+M2dd3wG4\\\\\\\/P2Ui7I+YbqAKiZt1A5bUifdIW7hWgKIQT8Ez7mxmASVFtmu42h5aJAM\\\\\\\/\\\\\\\/CKctOEqXqZqTbajJOUecD1tocHUWNPoOHRtdQVvTmcyM\\\\\\\/baVeGo3RVl+R6T0Ep2EQyS5Sv5ymXeNtYzH\\\\\\\/aCazvqLuEm4Uoso4WbSW8vmJf9B86kDI7e52uzAtYD\\\\\\\/WTb1TF879CJ3OmWaafili41GiXnVrDv\\\\\\\/zIpxgDNurNWthH7IkGKf69rp1iiTZYiYEhGoD2GiCgjxLI5nzltMXxaDS7exCPYpZ25aQJM4GnWLIMNhKnpGHWJcjxSEE6bWv7mV4fOUq1V9z8QiRKUOGC8QjmzQC1vzXH77DPE01+I0I83VcLPwp879fipncfrLjILj6GDS+Aaw6n36vy4VJwVe74HCWNObCm3Rrc4Gur95bV8MGrQJqWV2HsgW9wPNfM9W\\\\\\\/xwU9VzwcWqFeulGeAAq9t5zQJ7SgKPIiHVrU+vS6RyTGY2\\\\\\\/twSpV5lGY7aWD5Q7mOUo7uzf27G1rwAtMqSLyMDRu9PPinPDr8aXpvBkNVoiSvATpxf3PZ9pTQDMB7ysQkhEYk0SrLqaoTCFiunXQ7g5hIC2AqxFaw6ZKu4V5nR2UVk6QZmo+Fef+\\\\\\\/+4+\\\\\\\/yiicvPw2uBGmN3b9uu\\\\\\\/i9MZsDm9\\\\\\\/a\\\\\\\/BCeNNi3hbQz+ZtMD31XV\\\\\\\/JzjkveIuvc18WDfIkUl6XAlwa2BlBgBikl6wx+3zWbuQ0e9RSWFkGBuQZR1Gz\\\\\\\/st2CwGleWwDAOgaZB5RQ+ZGaYZeGxogUJi9\\\\\\\/ZG7uPAY4iQQ\\\\\\\/V7za2U6OpkAXzq7+EOpBebYEuXcYGPYIDhAVxGXtsD3LbbvYNE2kTfsBiiw2l\\\\\\\/Uislb3Si3mv\\\\\\\/bhq1lOA2hJjLnsQBO+ybbrnxRcRoSCgFaFxOWFimyvIyLEQKQoedPdF04w2LT2FIapfVEa3usVpV7FpAbISd3C9NvEVQOTkdpU6aIwtkLO0oEHMsXfMn8gi\\\\\\\/Qdk9WibpyYLSGS\\\\\\\/j7Q3Io6L6XE3pRjzQXpYsijenRUv8Qg0NgugbQzw3rkR2O33dE7E2RrzjIk+WrShwWNTv\\\\\\\/YHCc+OLOwAn9zV8wyvzAaekD\\\\\\\/rTw33OF1V\\\\\\\/yfyK\\\\\\\/N5vds4wdclhA+TDVl5cSGPRIw4wxAc0gDlTs5gN8KlOZu+FJEwtnk+Olma+NBcboABQrQIlPINNC09c+Sgeyc+efpPNrjC1dHF88AHifG0Y3ddeGcgplyaaFF9vBAFQB5PS5AL1BkZEaLr7KbSorFIJJuoKP6Ti4G3w0hMYVa1EQVELs2Wd0eUhAOg7uXPDJttV9eKfeEMZCwaVYGAHWcnAEkDvU3jZ2COgc9evkn2EycpWxreEnp08MR55Uu4nKzDQ2lJN6IquktOTXOmqZKwp+iakf2YOSTW1b\\\\\\\/iYxaS6\\\\\\\/wHrAt8OPfTskyBF9xoNc\\\\\\\/iHtzF6vBIQfD2a0TZYFi3\\\\\\\/uYGBM4WeffszHLmLnXVnExwSNkn3vknShqwYAnaWMYO\\\\\\\/3YMMeizbeKrWOFJc7SfETLr6BkOUg2KFe1rr7L1ckIZHSTW3+Ey2a634L\\\\\\\/smsvfatT+ClccDN+Rvn3F59P3XnGejEWHpU28V8Hb\\\\\\\/sdHySlNmDIZ\\\\\\\/ZQL+EgnCPmFs+WS44iILuDr02GR6L67G4s\\\\\\\/0+j1IM1nIw5bT7bYz8xKpGCyls1HTGvhhaYJIW79vqbjWc8ABcJkz4lrJvh7MbweEcKD6O0qFwFAPdTjx88wO4E9acTM5YD0Lj7yyp\\\\\\\/lQe2eao4682NCHk4Eudq7\\\\\\\/a8t+xoVgxStNO6GuPauCw+zB4xqLOQXIzejQGlyNA1ZLvncjeWwtq3pixVwQBPZeTpIH9Iq\\\\\\\/oWVWuTIIvrxgln0PKDre0wc4RnzJUH6fStC6oRW+gb8UkkaR3t\\\\\\\/SL+RoVUQ\\\\\\\/ayEZ5\\\\\\\/9t0tghLEJPiI0UeX1eK070kMREPLmVunk+5dXyleIp6by357woJbrARoej\\\\\\\/aHnrqy4giULkEp2pjlwkig+ZAPPeGgU4f\\\\\\\/wQeBRw09Hk1pXkEyUWPjAJopJ3L1yFDmO3uK3bqa1KjS\\\\\\\/d6WEFiB1MCZK3A4Q6OHO59U6CYnGzag1ueawCTMaHM2orAj6o27YcpSOsjwlC0jjNlEFQAWXFr1K38l02zHfPUeAGQ9FajSr4Rh7Hso1xgErndrLXiAUDSDEfDDRynzHGYlM+Pa6LsW61aICGFIO+FbBpVhY6\\\\\\\/fHenjHK+PFh04tJWww7jtoiz6dkGKEQJszmfPsW\\\\\\\/SkxhdV0pK3uhYPq8HXR8DMVOFJL4LOQKHtcaXBwgxIVykOjv\\\\\\\/4AV4jdGh9WDSlFcKlmd3D91gbqt0CVaxnYF8JF6cSBZjS9XHToJZ87OGnqpr2HZwTOenHH27VtrCTNmQ83EM9F9utxyrEgarrpIEkTCiTlw\\\\\\\/LqbC+EquyVZSAcXOnegaUfXYJocXttTlQodXtj1ivA4FSdtkEEeWIOqIiHGI+ikakpu0lWIjKxX3HtLUV91O0XKX4ao2jZPViMsXTv0BAtIzuckb4LfJZDTc2\\\\\\\/yjyTa85dNh+oa\\\\\\\/ryS0kJNCf+wZAXHrG9zmxkV2CMQhpWpj7tiwRtqBY4l+muz+7Wo2Gy\\\\\\\/IhV6uq5FMb0PibLte7Ecf7sHH8tlLoMbRfx1OCIUWVp+o54OaEob4JmYehdnbXpQi25PhzgGJqR0Wv7ge1OOJXolmDyfVm7Or4CUrgVi00RL3phq8MmZX2\\\\\\\/4fbSyi13wwNrC2diJX\\\\\\\/iVtAeukJBhYpZVH0ogs1aWh4h4Y3msoP1kJuLspWYl6xpJCnPaK1tK7sf50\\\\\\\/NXLLLIq6WQAErEsdOEuVGdEjtKVY+d8xvVFHIXRNjq8AoNvkFu\\\\\\\/Wbh3HG\\\\\\\/rMmInKI+QQiw2iiH49LI9f9UOvnefZpK0BLAiEpCr+q7qCHSzZDFaKOTdAATc26kLgqFvMoHulqRvH4GpJvtpC9\\\\\\\/EG2GBDw\\\\\\\/2IsN+nGuMXKRs6OYq0o+WQlkWBz3AjAefd8g7d31ADbMt42z0myzxI1latwNYgtlYLZHFRVI2huw3icBsUYlqQ1b\\\\\\\/HRCv\\\\\\\/ZtxW+GqyqWJSkiue2dhoUgv++SqA96pSXFCo2ou7jBw6TLGPV4MdGX\\\\\\\/Qmc8\\\\\\\/oMGVIDPTRM0I5Z5nfOvgxTiDeuRDxy+b\\\\\\\/Vt2dOU9xIMg7\\\\\\\/Vb+toOurLz8hMHWhWiiZl8wlnR08JgsPY1MRrAIkMBGoJnDjEyTgwcqCynPzxtoNWTWHt+GL9Iz8soTT6vrNsXTXriFwrCqiE0P1M4xg4+xGcOHMBrr\\\\\\\/pS7DNJNaSpKCMw4hr8Iq\\\\\\\/Vc6OtF8FPycGqLhKTA24z2sZpHHrKu6MtERsSXbJldgt\\\\\\\/3JGqtgV0ScI4XUpsmU\\\\\\\/YcufvyQYLgOeWS7SYKz0wG46Hr0j\\\\\\\/T9pBoa6FFCK7dyqvQGKZYgDc9SGFsPqlrG9lE7N6Eg45+krJSeDWuiaUNxhBi23yn\\\\\\\/Kg53GtvNub5p7drTXFzE1k9vcO4pk4ByLOSdGKu0KvP5emX5USXTPggfBWdX9A6lgPHbhb6gqO5t0dTx9QVtsGHIJxxjAedjqNG9PRf7pb3ay+wZ02T1ZHPwrz6mG0CThYNBpkH6iG+r9QXR1uLBbrHrNK9DYUq8s0V8GVjUWq1rNsBqa+iRGcrp9hSqNEgDjFeaikCp7Ic9URTpm\\\\\\\/sAqkTFcNjfs1PkgRxzKF1f6h13MwOgk91cf4ROkgMDMQe5fjZbktxGfr73q7oQCsrvYG9EfUQWcwrVMaMSulAmaTFr\\\\\\\/f7BGZ\\\\\\\/alWsOEB\\\\\\\/h\\\\\\\/aY5ptMoYKnbgIsHlf3LVGK11tZy8ZyETk+ZPdFilVc1bJsXQWv5sgwYVKP6+UBmH1tNufAHEY1m\\\\\\\/sdbxr6rIpQDBftrHY46\\\\\\\/\\\\\\\/I+FEtTrWwo+Q1LpwwTYe481IlB78WUFWuqr42mUlCTuulzoijkXd2RwxXmnkaVjbKaT0PN8asuikH\\\\\\\/a4zwecPauTssWQuQa+wVtYyIva7OkbXG5QHuSgjex+97heU2qn2EJt1bvzMHGGDNXzbspjmBL29hvJK2j\\\\\\\/E8KE5wwHWKYGesieAfWmlem9rnTD2dbFC8KMfbjZV2i0ajBkdPkBarSJz3eThaKXuWnG2OH01OBis2Ts+mRMianx4DSUn+hjDFgCxQYlFvSnofuHPSuNtm38NZyxTOwKIGfGkEW9jgSuz2t8e9jL2GUTqqNHpxy4bftQj2ck1AsqgGOkEZ\\\\\\\/RRLOa5+vAg9q3E5hXIJkRGmKFPu4OIMIiAWwCWrDPfwfqRciDhrJQ0FsekrCteb9YHn4M188dDeMtds6NHl6lW5HPrKz\\\\\\\/ShPKenG7reahkUJpQf+S1TCgSncjIW\\\\\\\/tPlqArEWm5QJb8gKLuZJnfNbCQ4sXS+PqX4LBNx4RRsBTz2DG1s3sLNp9leZKlm8sLaCqdAN8nB+V05IG7gtXTT\\\\\\\/DDrr046wrS4vhZT+nNS+\\\\\\\/O0FU2jjxU4VoB6XEdaY+P6G8ACn1gDiR43JCqx5sJ7wQ38a2H8oj47cWo0AxLJ79mqQHF\\\\\\\/mF35Tb7eQblMXgBW\\\\\\\/3MzG8HNXWN75tlo9E8oQJAu3+F19ZcVLLWfDnDEmTqvt1BE4UWurFz2yf5i+eW2HE7tY3Kbw0URVlzjCQnZ2OO52nAvD+Fs2zg6qRk+o1kuek1uUaD++KnUedxkSDnWpuZ2NHJRV84EePI0JNZ6pUh+N\\\\\\\/T9necJbGiviJsuBnP8tsO4QebghjbcT3Cn1oebcrShHPNYvukMovYibWbMXOwu\\\\\\\/2BvN8vElPJBa\\\\\\\/sJNt4iD5Xy9OM3+RL9Mcur3y8nK1sY9jNHndbpV\\\\\\\/P2EEZjIysPf\\\\\\\/7\\\\\\\/4uPiL6xxf72MbFjJeBB4IagUZYuomI5dVZNsjKt\\\\\\\/m3mXem5mVFokdfPUdK4LbqHoX6Dk4G4M\\\\\\\/EiDtZ0In6bBm5C2Cwvo3UBsl1wXwye2n1SvPHGaKUOG+xaNDAlOqGqyxyAa8mZBP4e9psx8A4Nmkc2oZuGNmLxPxnIWSHqMFRJSTlrYv69OBstfgKkCum5MkinpLYxOz8ORwYMmVdepiTm1wYv9iSfAPG3x4\\\\\\\/RSvHIMc+MqAfskgO3LxSePICvRniv91bDdCrtjfHynTyudwE44fb0VgroL8DCPKflRQinmqkByfv8GnrNqoOWe\\\\\\\/Vs+UA1CxqqSpG4mjuQDMsRpmNdrbhdkxPTxlzhyr2d+3azj54VmfkSA5lLTbre2PQwYwK9yOsSzz7\\\\\\\/9\\\\\\\/QW9qb9LxN\\\\\\\/G2UszMLKS4jRZpxC6QoHIPTMszL764goJNqdYfkm1a4GZTgKtBpUPjlOYpIt30i20Y+d8+binY\\\\\\\/CdruCZ0+XfC8uiTKsiQJh6atkqbP2rRIFj9aoIk3\\\\\\\/QZ4l9213W0sU\\\\\\\/ws5a87deVFeMVt5neTFjbF5VAg3zHIXcPZcDGmLD2j\\\\\\\/sBJhtS7f\\\\\\\/+BzSH2RTZiv48+elLKGbJYm\\\\\\\/RMHKU60RvImDH4cITq50PaEngtTtHIsfLvYRRi0e\\\\\\\/JpbpZ6ShRjJbrjZ+iiNwMSHqcuvV+aNNDUcukrLcBJUeCZ4MtatN3kRT4K13EjhIR4njlwGZPhCQunnO6mAf7BRILLCUCNj7FqpuHi\\\\\\\/KVs5SQ54f+aVXHAsTavcqrdeKtyR03y6GTM54iWcejH+o\\\\\\\/dheix0zgF2CIIpobCEV6Kf6\\\\\\\/wK0fz9IpIR2DUc5+LZ+Z8fY00X5tq4DasKVzyq2nxXe4hP93v116PX\\\\\\\/II19\\\\\\\/grCPR+P79Tvy987ALi4BTbSvp1\\\\\\\/Eo2v0SoJHNvPJO7gZ4+0J0gBZCB07PQ1iRQVBJbsJHOZh6AmiIqWnhWWmtWWAOmA\\\\\\\/o6WoDCeVlxNUg5XFpZE71Sm3BBG4U7Gc6kDQPAK3nL1cEjyLal6NU\\\\\\\/uF5lgjgvy3Z\\\\\\\/mV98GDgvPnq\\\\\\\/PbNcfoLx+zd53B\\\\\\\/L3uR4RC+I9MY+UmoELaf7Qn79ehJcFaFQBkX6pLSNexmOcGqRDAnt3ORGoj1c\\\\\\\/GiRBalDx5ZHGpJQWdKOo013kEH4smit6\\\\\\\/l+IlXEKkCDYk3eERon2z2Cgm9tXXDqOsh9rVKlzY6u0nNqIz0W9KASHq6bOarBwQdAXihPASLwJcrD1BLB8RQ0Yo7zVQ5J6wE3CPFfnNpB9fcw7q6AJLa4\\\\\\\/dU9JCo0\\\\\\\/MoMTckLYx3823\\\\\\\/RlUUXKMUiRg8hH7CboxM4Q0dPgYUQtToKWfDJ1Eg0OUHeDs1T6UsC98madFyn5TLIDjJPtVql3hUnj4nWSOt1WDG9\\\\\\\/Wqi3BoEiwD1n95HTFkbzrGlVpP0XIKiLef2CctuAaTObShiDIIIaikrD4lLSd+MieY3XXJ+p5+4bYxZTxqBdrt4S9Osumb8TcM9fL7wvjaSTgG4I5RR2eNNSFoFcFNtanRI0A7SVTDlVIPIkzl9aRAefqy9uEyPV8zdXCFA5LeaOwKDLGjC0DPxZ6NoqqfI34vr7yAe7+snRZffYmk3UEusfAHcVPFgjREXVZik8z3yh6RdwL3FSH804pQeg5O2hKMRTLq8chiCg5FvSo63om0AmjAmgh9NLjB9q+SFc+2JHPYNvgm\\\\\\\/ld9bjV82zavWvCByCxN2r\\\\\\\/2amrKmeILjVy72i5aPU+mUoON2\\\\\\\/ILtTdb93X88duizlQyj+EMwK56+qtV0P16g\\\\\\\/lk7TOyLl8OnC9QQZNOkozXdvbm+7OlBzyFTljpXbf+J2g4xmTyn5HYF3Q1xT5G\\\\\\\/\\\\\\\/lQPDrGKL4KNuc+V6k1WaOsANSWUzxgzIOrVJkkEtwOLqUOn4XvygbxKeAtDKJwUjhIXYhHSrNLjYzO4Y\\\\\\\/Lr7IcMUGN0auSgzUgvzkf8wcQ0DILuqeeWRHTTCLfLZLc6N7pR\\\\\\\/HyMNkTeiyLlMcfXIB5Ns+FPSDmUVWNY1i9X1lv4TCrOnGUhHXS42y56yUQoXF6dY8ZQT5QSFoFbILoDmfM+S0WiPCrpcHAUhqhVB\\\\\\\/DJtJQjcKqfU9ytZqaOGEVGb9whWTMb7juKlOJjRB\\\\\\\/nsqfsOUXHMAZ3WjvyCZAMcWbqfA+IAxIb7TGKLlPouA1PEsYBAC0If4lzqkTcfW2gf3wUhRljHU8m549hq8SFob+LGpQXoJrAN9jliR9WPpRSIjP+vEMyitRmIxu3Q1gUpZs1Z8elGPQoTnjRBCpz4NwMwVFcy5YkreXSUbmOi2f6hsT6diUQf9XNYOJRzM8qd2miMO9K1Nfy8B3gyflCjIcrry0jTE8dr6c0n0alQ44\\\\\\\/fEEgiEpFrQtO2XU7yf4cd50eYHSwTxt1hS9xGTXGTHKPWP58MvowXwQ1DLbaeL4a+mfQpO\\\\\\\/zkzcsQNSZtDPI30HbR1wzI2VXMvTXwcSqNkQGxHb+IJlf2ExZkfB+xZXInY3Lio\\\\\\\/GnxeKEaE1a0ctIwkNgRviRWH37ump4osa3aWHSkrXqHMGYG7DprThu\\\\\\\/eHMt0mc0MIoOh805jIWPCt3NF+VdwayrFEaD7TumfHnYza2+cAllTzRkUhkFfsgPoI1R4gQGPrgd8OvhzuDjNiwyqMICgkhxRNinR8ZoF2Dnl2\\\\\\\/L2ZZ89chHp3cUUiLwDCRMiwHigVXvrm3njtEUzTC9smXb3GFV2WB1AfjijLtVtcaSkG7kER\\\\\\\/G5XVttVaoR1jYRrkJKg97lRvDDf1\\\\\\\/FedxL6e5u+0Gb9EiYr830AHYJHk2Hhc3rTYRgvPMoegmxmGIkszSCz3BXBz5ugfd30goyzNFoMFRozF5WAJND7mZUNc\\\\\\\/gbokHuQUH6yGv7\\\\\\\/N9bDT4qPepYVrm5qyxON+tNjH\\\\\\\/5gjqzsDpNSdTnh81fGdVIj6sqHPUDfi7IRe66B1lo\\\\\\\/Rwd9Snwmg+nGHY86W5bMA1rdq0PqkOEMnhXxzFBHyGTeAvV1FmDl2izJ1RkblZQzRurcMwRtYhdVXMwPxdVakGd9uP8OeORL2AzZOFMFSG0IOouS\\\\\\\/6sgBs+VagqXf47nMkrJzz6weCwILGntefXRnSJDHEdcpnJZEoFCEU0EWpds9vg08+Z41V1Z97ZhSYRdJ2Td8jRo\\\\\\\/EC86brPED4UxsaabQavoTn8fIbyvdxlJ+vbh1W1SCRriAITBuOEREvjgqKixxoW67DVcT7RmeZyKPmC2tESxFERBijo+TPwZnE014Cj2dATLHrNgnoC1eN+TvSld\\\\\\\/FUhK1W\\\\\\\/v0YIeffv+lLCO6Wg\\\\\\\/xv3e+X1AOl1qCES6G9LwpVi3gqe04JJd1\\\\\\\/xNAB5GE7UxNFcBhkf29kgSMW93yZcvKMiBy3SCtF+XXSKTW5qTtL57kywlhwhCBohAg1XraDEOO5JbfXc9ZslFHiSOKZAPofMhK0JY0jRRIOats6cyAIOc50Wq6\\\\\\\/2tTOLsRbtQ2au2aOYKqqf\\\\\\\/0e5GNpCKpfcHft1VelghaND874iDNRAZQdzTpBoAd03ZSakz1MjIOfiElZelx+w4xwcXeQNg6PgGKbKHCJsdOoYw73QpGRNGbFj5wOux+ZF9oeQDVaKlzE8zp3uJ6mLtKU\\\\\\\/LJdMTq19YEmvH\\\\\\\/r\\\\\\\/l67hxYTSzUSQq0VizOF2G\\\\\\\/spiYTyU+txENlpdbPdsFPQ3uHf\\\\\\\/WJEZ3N7BkO6bTugjbRAumYBrOZ+60UBSPpPmc7YHfR3nqfr7jJcvymyp1sunQgxvuEwADO610BSInubwFLRzgj7wnQYpSL3UI9jeDJe+wT7ye6D5eLJbnIFBHScEpRhwdLmO8tX4BMZfrbyDCZYxKS3xKrJwFCvL+KkbHSBLlfFPyiibzw9RD8jyfO6Hwy59bCmLpp7A+uQua7h5HS8nTlSrWBfkrKo\\\\\\\/LIwIefvoTmS0D9z5RF+TVUQ434o0VC5OkPOt73LOmdnDGE99LUzvj70uQvkemIOwUrrIu8qjnZgbvZTA7GZ8GUvQMpjWkth95trTUv0ukZ\\\\\\\/h9Zcj+BJVTX8++MK3+Q+JN6LjcUlI9o4yOL9SWioS7ir25Vwsh+PKDUjvevUJc9lGeOEga9ly3hcL3IUmdir0KsNmhrnXoAJU2Jk+Uce9lp1a6xUJfAzsYi4RlKRpzhxJ742R54yCGbMr4bpFmjYA\\\\\\\/IFF3uwRJFWzt0oII2yLeRSKNzPstJdKGGIOU05T\\\\\\\/k\\\\\\\/mkGjqygkpmEaPKgZTehbTK6WZZXZYiRwR5rklC3h0XRG1P5NZ9233dGymy8cBhfwBQFiVG1vnfVIkyvcqrqThioMVkPzIasrA\\\\\\\/4T1zeldqC5jKE87ePpgkxlIo4Hp+QjaYA3Ug9DG8zW4xNfOM0BQxM9GuYDF76gZXczCtvoYVEM\\\\\\\/lqXMhTlV9g9bCL9DA8bCwf606LfE06GLHrJ7lMZo2TYSrzO0MWow2xatlE6CYrn2bDURn6p+fi8d4ySDwuR7jhn2QD6Uic6cp7bOl8rCoyOgSZ3iBY+PsSITzubgqPaTDFqiBjwDDt4R+fU9N2o4m1ULMcPA8hvDLUh41Pf79L3tSuDMGMDQIgPLC0LHZrHSyCh2q4CLtmC5qSQ7F2dbOz1j4EhEUkkhDVyQG6x4rbYU97LakkAMJsIt9wmJRcTnf6cF+iX7jJSHjUh0zvXsRQByupGQLk6Q1ddHoI1WQ5jxwUaFt128B2zK+I+wF\\\\\\\/47T0hD\\\\\\\/p+IPIz4ooPqBLcLCYwubdpOkgyTBV0HTMDcziPPCENTR6EyarJNmIm+Jcpad1sIs427jPC0b73\\\\\\\/RLLW4tep2TMQ8887SM4OhskE+52G7ThiLK+PeOpvCVSCdV4ori0XrIND8lZtm6c0C6wz22xvXzPaNbdrr8DDPs8gfmPLfMabQQj3mpaj8f\\\\\\\/sHvDIaqjDqJlnWIZmHd6YUvVbBXd0ggpAlHXpdUeHsMVrbKO\\\\\\\/xK9sByjwidqRn0cObxl9rdv\\\\\\\/nJHh+BCG3U95nr6if+9J6e1Kal+Upr11baOJEedcpy3NyRzFf8tiOxqDkBNP7G6lmw7NorJJYgrHzL3aF2Z09sJK9BK3PaCAV1jd59yg6fAbI5ubD5kCmq11rTqNFi\\\\\\\/RvtLlufJJKi5TTwo2UKDKDTBr0Un2y5A2xMlso0T\\\\\\\/xULFoY3SnsVR0HVslfuNOYpAUpwqlykVtM29z+jBqMS1vPWF6V0YaS7kfCejwEkTHjujXxQWb5FTHdfDWUd5bbqkmjPHbqWsGnTSY43BHlGrEd7NWpyizkF19O5BwAyWZkEAHSX3wkB18yxY4dVgouK\\\\\\\/+hZBDFHUFsamUVObfgqZAgQziOuUYUbZxCOzO\\\\\\\/PrVyXDgfMhV\\\\\\\/TZK4Q5FWNK1dDv3bCLLdEfXceHZHxkBhRLwSoe6Njygil2eqOXT+PlFyxbEOdagBjqq5InfmKtF+ZH+vIop\\\\\\\/umkjCJo\\\\\\\/BkAY++EOY4smkfIPGAELHrAYQB\\\\\\\/epOJPodg71e+ldlA0zmOR3n7IApZLxMoReVbP29oW7MasVipQKb\\\\\\\/2uvKM0q\\\\\\\/DSd09koDmLvwgxlnDpc9vLqH9SqIvBIPdJYIRz2s61by8B\\\\\\\/K3GJoBoXkOA+0Hm9D6LX1zdQDJGvDTyOor5XuPYS3WxPpbwmCnbm17EuLRBglxKPdqnqjYr82wKd0LZ7wJmI0riIDZRytw0WVdxXYdGPZBUMftfZgqdcgnFfg09RtgViMzq1AujWo0n7h3\\\\\\\/wsW6wQbOHb4bTD9\\\\\\\/mqQcXwEoHEC\\\\\\\/PLFKAsRSb905a1tUISp3IZxpUlvzjOhPxDMnTRSrfOWQz2Px2pJOkL4\\\\\\\/Jhdn8q3RWiatFpUzrEe5PIt2rS3xGh6TLPdc8BdFFMvHul9GF4+irlXp5QAB2O93GHuOErsUqIstSFfdV59U\\\\\\\/bIDtHl2IKGrwgc1ZfE1nhPE532RMNqdBNydf3BBnXzWD6KZs\\\\\\\/mSVVlK5HWv1aVI4YycKEVzUySpAeEDWHoAr\\\\\\\/ey8biWAOVF9P0HRIdWUBsEKk\\\\\\\/rPgiN+Y2zisZPJ9rjhDZL+5sNz9+3U2vraJ6Cy7QI1JRe1mO\\\\\\\/rcLDroty1LzYQzL6i7EmabthWX+RkStFKkFy3itwSYotyrtyGe78Iz\\\\\\\/Bhd56PV4UNkuC4xT3NbJ3aoxWMHYU+Lr2RWkmfjYi+ljBIwrE63LgIQcJ+4a60TrTOFnvYrVThReFfvOnuuzsXud26ZXDsQn3ewPKtyNwyva4CFqA2tVKNXWi5F8ta3GFHqZk1u09bxASmLd646MwPFl7wluMFGAbBYabZ3SpkXO3+97KO86pgELLPC+RdFOb6UQ5ovxecptNe64cixMa6WXXTdtXikczwPquggGNz0D3AHN\\\\\\\/zJTrzKGnb0FNyVnYokWT6UJEU7iKdFWBlGnfypwjSdEy5FKGOudux1OF87JV+qcU6ni8+CpkL+sA3RJVDXCP1WgEXeseoVEDOn0aZa+yyuQUknRP6p1brEuSwCaS+nMtmVcfc5ZO3XGiRykjkhLARdwG7HE+\\\\\\\/7OXYlda05oehKA849u5EZv9\\\\\\\/zqQGibyEbKk+cTE\\\\\\\/\\\\\\\/lRDrVOnUVjHNpH1ipKdVaEESRindDb6bi9y+YZbYwiHZ0WdFJGiNggM6EabfuapgOLGZPjbhJ0aFsEcxezwaE0e0oOyyK8fc8CkrCmL3AfhJtOGE6dZmvf4yo7JeYkcrfTc\\\\\\\/1KxgYDrmRiwiGj3KQ9Kfjg1UwRMK\\\\\\\/i0UJqLzmKX23ZGs0wlMLI+K\\\\\\\/vu\\\\\\\/6nW6X2TqCudVey280Rg\\\\\\\/HqqojmVsMUxF65Ao7cuYVCirWucRMH88RC7CNpnOcoctH\\\\\\\/1zyH6tV8607NvJMWBx1Y74YUMItweQRz1Wuq11VXScSxES047tdJ6o2hNfSggvdnBe58o9659lfYeuJ7cONhrwM03ELn3mfkgoMjeyLrBkk6F31p7Np\\\\\\\/iEzDSYxd7nly3D2HfGd625o9W9qF1F6CBtfQ+vAGOx16KBnva5N7CgNneP\\\\\\\/DErTKsLhR7jWPBQTjOk8dmWswerUcr8L31ojbXoWaBC9CKJj8\\\\\\\/mK3ojqw+etLnWiqFBTh\\\\\\\/OuvHHGIidCZPAztG87FOfZEauLM7nkSsJXcCC7Z1rBvxwQHj8CnwWbfHnGpZf3MTEMpL42d9Qpv8007ueL8FCF04X2VD5YXWbXdmSkwxClU5XkEDgj5jzG3YDRe85T8j\\\\\\\/Ue8ZLnXiDvBw8H\\\\\\\/WHJkKb6nN0IhaFTSO+uRTLfCCQ6RuogSaF7DHfn4KaH59WWUMX0tckb1mtRAuSlEWY8Cu0LIu2TfAAGAfSGb2x\\\\\\\/P2rM1Tb1meg4zzQVTi8yU+hsk5A6K8Xr+d0+BHiKk2fOetExSlv5rBXZn6fx7+Qq0BfduOBfJIe7WmHNA9yfYyaTS9Rpo2WLeYveCmq0CQgUwBgn9s2SFukteaOQgm7pKA\\\\\\\/mCCoLJ+5D5lB\\\\\\\/hG+9Xm6x7+p\\\\\\\/BqsEu1I2MPE69COCP1kw0BWjUfzKHasHJpllcE5tgCkw2i3HiV3mOGvIhJZKC3cQ5jWRkfVKxyzXkeXoztMFAFX58cCfyxIJOH8rul+vF0cEmOQ7cbCxWWwSy1r3pOPRcScTMNQ8tOiYj7ufBsXRxCwtZBLV7XqM21zZCJEfs+csBGnRvRCzKkjcto9upHNbcdRt3oOUtuuQk4VeC1m6AKyOKxPV\\\\\\\/WtaXRGs\\\\\\\/r5iiMVVXK2OgMazRbHKXvKUHGt5Sn8ewtwTB291faVOLDbrWKhdLZzWGJNk4TobCv0sr8XdW8R8++9XwhY+QCrKcclwLQgPLNZ4qdd88yre4jMQcmIKHNTWhMC88EurJTXpgvad3YSwH8em\\\\\\\/AdNdQg4dRgqHCH2anxpV6atgRDnajLn0ouTVr9JjGPpH2oQNXhl7u+MHwKsRA8ebmR6flHv9BX6y8mPMts+s+jqr0zn4KGq9XqNrA0g0446nXOpMVKPXdHF7V6l3hiD4CDNzYYqICsiaS65KEmORxGCyZtKMEudq3tCJLCPAnEJ7DK7C7i0js18Tzk+Tm127eoo2uJp5vMpFvUIk5U5x+JZkC3vbOwXjRUC+7EWQzOdzNvUEMfsvNZsklZ87xLrkaPgsIxvTLVK7ijuHUmcsRFcMZb6NnyB\\\\\\\/NgmVHnh9I8u8XXio7u7xWvGHosvjZ5zjEWrpb9z9kVYLEewcxLqWDoOr+FjK1yiwWO9sJYVcGAOOqH5USGP8deFlChkmBdcA8ZXWvvR8HGLIjOU09wwRrB5aWo8rOWbZkV\\\\\\\/\\\\\\\/7yx4jf51VyFsmGx9B2BtEGrDgbs9CD9JeTxQd7EqlrO64kNAy7ECla0fCCzIr+wvU1ZrL72t6cDUOFplMwXGayOW6jIJZFvYP38Vva6cqq2TEUFZ+fiiZw+Fwh7lvvd2PVoEEJw27glND\\\\\\\/sRwNSufygVZbH8S\\\\\\\/8HvqO0s7LIShpflPmM9Hcdrn3GQfmRnvxEMgbNW0qqpyVgZoQHBHXSPFMicSDn2CeucQe3FHB4bP1dYSoJjgkkZr49bQmxvdxApNp5hk7HiH0xJ3p19jDaCZ\\\\\\\/OEE0hKh3sgvf9UOyXuwM+TirTjPpmWraBmyrbtHJ5yRKG8SI8wlXeMFQIC\\\\\\\/xJQ9VqbRyae2L5HJ98DWFMPlzzbumUXLS3niY4uOkbyqhdobIWzHnHTHxQAbTo7Z8mEpFNQUgld9HClkiJbSQBXkGeJ+hdcMCJmViYWP15HrIo4rC3ClsbjdLCOQpX8m\\\\\\\/HKNYjN0A2e2Nr+zaJfhfrfR7KfUGrSaMYUB6iwlOsdTMVgbe\\\\\\\/PlqAhyL3EYabANpYZ6bQ6HTjDQVbNSjk9T80lanylnF9W12Xy1GBgECFVc7cnyRYbLBIs8NqvMEmKYE6qsm9MBsCTOuWgzw21VjLRPn+e6ETCyuEcsAOKRy28fC4KpGXSO8hfbl2YgprvA6rGZmQraG9lDa4uEwB87EHuzq\\\\\\\/dR+PFCdjGjz3NH1LhIZa5azMsgx9qUhItLRsu14ZtZPduliL4qBItvte6uFqd2KyDTBglKVEKPd+MNTzBDgrWGfBQBDXq4gLxuA4pQ+nTEq+LyA2UhI7r63W3W+el+YAMcL4UmE8op320pNaxaHMJ6yNcZHwFDPS2AXlMFvH5W7482l3o\\\\\\\/qE9M3+028ZYOJeiQNM\\\\\\\/nxtvCV1cnMM6rDb7NYfgH\\\\\\\/RocAcQbEhVYIMh\\\\\\\/T6BVm1FxDPu3\\\\\\\/izenOnwRlNIyPWbnkOueC+jQ+xXfwYmAGz3N7xqkWGk8CBLslc\\\\\\\/tlXYoYxVV3gGaDlHXg5p1WW5AuDxJbx2z4SSKYX990cHXxpDhJ5fR6saRMkDQbUtvV5alBlUIdcgCYiIoCdTGv1igBmpDXpgfka91k874lH\\\\\\\/j8LCultVhzxN+2sZ3EQBWH1YGECzL\\\\\\\/4VH\\\\\\\/ba6MvNZcu43bS4r9G\\\\\\\/Wt4QK2ObHnoitxwc0WgbztcgkVojFyptitjQYpLSk557Bi6hNtzSOvqwUWNknb3eCosBvECpVjg7PEIEnqnuhcxEwSeJgvzH54Q0Zh0HTahnmSRsDHAwtsTRkeJk3xon5+sJHeeb2bB2OsAtyd9W6XhjYjbpr876SRsaNOvjeepPz1uFap3QD5mjuZ+dfKPF3xK46rKU9iZe15mauMFaEPc5pOk+x6WzCLKDA0FjA\\\\\\\/Q0G2LjyDBdKfQ70Mj8v45cVe1Yk8AkyizRDtWsnhIQ1DFbSBnQHl5xfCT4boWyNPjr8IQlhd9gQWTixoru91N5sHDxRq5V0NHpwnNUJWCNHZlqvzsIrDUfzMt40tDae8DbUsTAbIL9NOEIb+8atyNt5cOO63YQBi1cyIzmHApZIfW2Zn2kX+vkoxH4T2534WSN\\\\\\\/mjQomVOv5FSPCdLFkaUBLDXFE85hvZN4K1iFtAW8S+LfHfGfkf+rZO4A5tuy4\\\\\\\/PYXUUxUjX8iad\\\\\\\/0\\\\\\\/KyZjmnyUg4EIbm1g+Lg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022EcqJfwOmULZ7F3zJISV3uA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022da774b5b290fc52420f20d5ab8abcde290f0e02e2ff69e89f56386e6397b3549\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yBznp7YrpyOMRuwrJJX8kopj5iQ\\\\\\\/UZxZDXh2qLNt55WRepuJr8I60W\\\\\\\/TzA9hgrIlsEekt4gLcQRamVsnb4I2ZdySiva1z4F1U7Hs13pfyz2zktL9+mq1ifdzaTde9ugom549lKKkeV20wAcaJcU146cwLGEFcbt+91cQZ3mOmyaPAwgxQqh0t9bbHCV3ux6FtlHDO3NxMjbC0mrxCISTyeUaV8Nw3Cpwyb1aXOUbMI3C8VeFLNwoOHXrsyiQ4XDLzDUmPrjfD4aF8xA00XkjgmO8Wqf4zfbFVv16+8456qYhZ\\\\\\\/kFG4Mp\\\\\\\/4Jj2Dyc+b3XewREieY8OxxGpGYGVAzzvK9ZUNvToFZPr65p1B\\\\\\\/OSYnqz6WLZsWYHKuMBK8I6frKYu8Tubo6wll3xYpTF42oLvx1NXUvQ\\\\\\\/jsWpsflKQqexiD41GMMpx7B4KR8WIcmwc6D8\\\\\\\/O\\\\\\\/fM868rh2pyazHgqQe2Vj2A0clvGZ2BxlOtHxc7acYpqidMUcbcRkIpulxk6vj0w3Vhxdrj6ZziASnqYY\\\\\\\/ZuSasbX0EdTRZ9ddyrUB5av\\\\\\\/fUMVRSi1wbZCAKl4ttLaFI8BqgGTOM8+JRxKYBRSsZ4It9fHddo+BgUXQGWprWUg1LxO7fadDu7Nyn5xhzgwPAwEkXBlv5ULScBuFSZaZPbJMpvwCByORH+9ABc3GsOVw\\\\\\\/s1baN9VtMABgZjkNDESyglcGrbIn4T\\\\\\\/q7HxnXCkMLbQTwi0f\\\\\\\/hvv9Ay5tyK9Vm1bWx7R\\\\\\\/nUG4crmmeem99U\\\\\\\/MIkTEz4gGchn3z2KWAjDyr0lJVKm8wW2+lf8c5EY+aXT\\\\\\\/F1Msf4TuMZwSCrN6mf7pOPclg2coPRlmObTvqqc3rAAzdgE9JNRAjIgWBiUOF1008vp9hESg8nHbmtHF0cetFRvIPL2CenmL2sz6MR5cL3I84MQzFHNKLW2InShplzT1\\\\\\\/Nl2kAY6mva9I1HHO8hXt9ySL2UUsHi7fO3I260pvuJXdMq5sec+ygk3tEV1QQBKkNccJTU6cbSoWFWPw\\\\\\\/lwl5midQQBKLpe00oqPQfdyMG+Pk601iNbL43dqO+a+ulSD6tBjbHCgmMuFevz6mCqrv0mGtMrgsSuozioQ\\\\\\\/cwzI9inVOrnGsqk22O5\\\\\\\/wdpAXluzgF9BFXCQz81pClwvAa8rOcRJH189IOEvmdp8E1xks95CzE2fEJI9so5p+663zUcdciTVXZ9h1h+wgt+2xdV8t0rH0DN+bt2KnYsUo+P3Q2Pqh\\\\\\\/3271JeHHiqT42VRzgUXFtC04sLuRoTXs8Lh+klD5Aigj1EyZUK5\\\\\\\/OaaMGwZK+9Tyt9Jc+TZst9TVdL914SOdA8c7BARht8LhizOXMVDsUzne3Rc0aEXbqKbGb0BNlF\\\\\\\/yPb+e\\\\\\\/YEkCVGS+1LsGDocEgDtfV5gakna+\\\\\\\/vpF0sgPQem\\\\\\\/X4GSqEvheRRsBYdrDTeRnDj+l4ZfJ8C8SimnyatL6P87wc6tcGiyW980kCBrnjTVBTRaKPdhjjzUXMdg5X3iLpjTBlIZSfX269PXQVNUPQuak8\\\\\\\/WeYVAxuAIqUb9KdJQeuLZvjTrRV4emGRw577TarkfgXsukCEVJ\\\\\\\/J1fl3MiOcwD2GJWYAS+Fw6uyS3iHYuxw6Rr86XljKKABN3gAm6Mv8k0NhXwcemcO795QobWqvpNuJNdWgHTHt3LRWGIuTGlE4IGB6AKJFBVea6MaLo91mKcVXetPDLB7YZIIrSEH8xy3eYVD6MMdkL4Y\\\\\\\/qXog5\\\\\\\/x4yc5aMID\\\\\\\/r+ttTtxaBHMaQT4m\\\\\\\/YW61M9itfOgiD4TeMXL71jX\\\\\\\/8clqATbS4U084OVTWzsCeovw0vlsz7ej\\\\\\\/TC+RmJog4eAbKct10W6mZd6KD\\\\\\\/kEgq5HsFU6Dm5Yfib2l4UOHx2GBdJLBlt79jcR6riFZsZ0VPiYz1WLIbJvGFPq8IEnRwqmY7+RWHg08MarcBTY8LhaN\\\\\\\/ayaxIBJVqUgTuXyjrQEZC9pA\\\\\\\/OGCBeVRyDbPXkqMEFSQeI9jtmWoHpFI9Gk3A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022QgHM68yfVmO1\\\\\\\/ZtGBulGmw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229eb9253c08d86c937fd832fd3cfc405e7acf14708743036d7919e59649e6d8e8\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491784 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-software-and-data-resources">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/1/13" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9WkBUZz89zcnlIvLZ68WDgCbaMQ937MY25F5FAWkaQ0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/1/13" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rkY7Wgo4vylyC0xn-g0LTEN_wkNNNP_4CwXZBZsue4Y" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/1/1" title="iProteinDB: An Integrative Database of Drosophila Post-translational Modifications" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/1/21" title="Deciphering Hybrid Larch Reaction Norms Using Random Regression" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491784" id="node491784" data-pisa="ggg;9/1/13" data-pisa-master="ggg;g3.118.200908" data-apath="/ggg/9/1/13.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">OCMA: Fast, Memory-Efficient Factorization of Prohibitively Large Relationship Matrices</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhi Xiong</span>, <span class="highwire-citation-author" data-delta="1">Qingrun Zhang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3783-4781" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alexander Platt</span>, <span class="highwire-citation-author" data-delta="3">Wenyuan Liao</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4662-3177" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xinghua Shi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-5692-7129" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gustavo de los Campos</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-3795-1873" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Quan Long</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">13-19;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200908 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491784" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhi Xiong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Computer Science, Shantou University, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhi%2BXiong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiong%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhi%2BXiong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Qingrun Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Molecular Biology, University of Calgary, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Annie Charbonneau Cancer Institute, University of Calgary, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qingrun%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQingrun%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alexander Platt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Computational Genetics and Genomics, Temple University, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BPlatt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Platt%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexander%2BPlatt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3783-4781" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alexander Platt</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Wenyuan Liao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Mathematics and Statistics, University of Calgary, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenyuan%2BLiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liao%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWenyuan%2BLiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Xinghua Shi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Bioinformatics and Genomics, University of North Carolina at Charlotte, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinghua%2BShi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shi%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXinghua%2BShi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4662-3177" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xinghua Shi</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Epidemiology & Biostatistics, Statistics & Probability and Institute for Quantitative Health Science and Engineering, Michigan State University, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5692-7129" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gustavo de los Campos</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Quan Long </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Molecular Biology, University of Calgary, Canada</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Medical Genetics, University of Calgary, Canada</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Mathematics and Statistics, University of Calgary, Canada</div><div class='author-affiliation'><span class='nlm-sup'>***</span>Alberta Childrens Hospital Research Institute, University of Calgary, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Quan%2BLong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Long%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQuan%2BLong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3795-1873" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Quan Long</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:quan.long@ucalgary.ca" class="" data-icon-position="" data-hide-link-title="0">quan.long@ucalgary.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/1/13" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491784" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491784/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/1/13.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491784" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491784/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/1/13.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:491784" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:491784/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/1/13.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491784" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491784/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/1/13.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491784" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491784/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Matrices representing genetic relatedness among individuals (<em>i.e.</em>, Genomic Relationship Matrices, GRMs) play a central role in genetic analysis. The eigen-decomposition of GRMs (or its alternative that generates fewer top singular values using genotype matrices) is a necessary step for many analyses including estimation of SNP-heritability, Principal Component Analysis (PCA), and genomic prediction. However, the GRMs and genotype matrices provided by modern biobanks are too large to be stored in active memory. To accommodate the current and future “bigger-data”, we develop a disk-based tool, Out-of-Core Matrices Analyzer (OCMA), using state-of-the-art computational techniques that can nimbly perform eigen and Singular Value Decomposition (SVD) analyses. By integrating memory mapping (mmap) and the latest matrix factorization libraries, our tool is fast and memory-efficient. To demonstrate the impressive performance of OCMA, we test it on a personal computer. For full eigen-decomposition, it solves an ordinary GRM (<em>N</em> = 10,000) in 55 sec. For SVD, a commonly used faster alternative of full eigen-decomposition in genomic analyses, OCMA solves the top 200 singular values (SVs) in half an hour, top 2,000 SVs in 0.95 hr, and all 5,000 SVs in 1.77 hr based on a very large genotype matrix (<em>N</em> = 1,000,000, <em>M</em> = 5,000) on the same personal computer. OCMA also supports multi-threading when running in a desktop or HPC cluster. Our OCMA tool can thus alleviate the computing bottleneck of classical analyses on large genomic matrices, and make it possible to scale up current and emerging analytical methods to big genomics data using lightweight computing resources.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/eigen-decomposition" class="hw-term hw-article-keyword hw-article-keyword-eigen-decomposition" rel="nofollow">Eigen decomposition</a></li><li class="kwd"><a href="/keyword/singular-value-decomposition" class="hw-term hw-article-keyword hw-article-keyword-singular-value-decomposition" rel="nofollow">Singular value decomposition</a></li><li class="kwd"><a href="/keyword/genetic-matrices" class="hw-term hw-article-keyword hw-article-keyword-genetic-matrices" rel="nofollow">Genetic matrices</a></li><li class="kwd"><a href="/keyword/memory-virtualization" class="hw-term hw-article-keyword hw-article-keyword-memory-virtualization" rel="nofollow">Memory virtualization</a></li><li class="kwd"><a href="/keyword/gene-mapping" class="hw-term hw-article-keyword hw-article-keyword-gene-mapping" rel="nofollow">Gene mapping</a></li><li class="kwd"><a href="/keyword/genotype-based-phenotype-prediction" class="hw-term hw-article-keyword hw-article-keyword-genotype-based-phenotype-prediction" rel="nofollow">Genotype-based phenotype prediction</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic selection</a></li></ul><p id="p-3">Supported by recent advancements of high-throughput sequencing technology, several organizations have developed very large biobanks comprising DNA sequence linked to phenotypic and demographic data, <em>e.g.</em>, Genomics England The 100,000 Genomes Project (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Peplow 2016</a>). Some of these resources have been shared to worldwide scientists and practitioners (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Collins 2012</a>). This abundance of data brings unprecedented opportunities to develop a broad range of applications in precision medicine such as predicting drug response and disease risk for new patients. The same trend is evident in agriculture, especially with the recent trend of “high-throughput phenotyping” that generates high dimensional phenotypes. However, on the way to enacting this vision, a pressing roadblock is that the datasets are typically too large to be analyzed in the main memory of many existing computing infrastructure, including some High-Performance Computing (HPC) clusters or Cloud computing. As such, effective use of these biobanks is being relegated to the use of HPCs or computing nodes equipped with huge amount of memory (at the level of terabytes). To pave the path to the routine use of precision genomics in all clinics in the near future, it is preferable to carry out most statistical inferences of large genomic matrices involving hundreds of thousands individuals on personal computers, HPC or Cloud computing nodes of limited memory within tens of GB.</p><p id="p-4">Previously, we have developed JAWAmix5, a hard disk-based solution to large data problems to allow for quick analyses of biobank-sized genotype data on computers with limited memory (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Long <em>et al.</em> 2013</a>). JAWAmix5 offers scalable functions for genotype-phenotype association mappings by storing the genotype data in disk using HDF5-based data structure. However, the continued growth of biobanks has given rise to new challenges that require further development and extension of these memory-efficient approaches. Now, in addition to large amounts of genomic data, the matrix that represents pair-wise genetic similarities between all the participants in a cohort of a biobank (the Genetic Relationship Matrix, or GRM) may require tens of billions of entries, becoming as sizable as the genotype data itself. These GRMs often play central roles in genomic analyses (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Speed and Balding 2015</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Kim <em>et al.</em> 2017</a>). In particular, the eigen-decomposition of GRMs is the most time-consuming step that is required in many routine analyses such as Principal Component Analysis (PCA), a default model for high-dimensional data visualization (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ringnér 2008</a>) and gene expression analyses (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Stegle <em>et al.</em> 2012</a>); estimation of genomic-heritability using seemingly unrelated subjects (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Yang <em>et al.</em> 2011</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> 2015</a>), a first step to understand the genetic architecture of a trait; linear mixed models (LMM) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Kang <em>et al.</em> 2010</a>), a popular approach in genotype-phenotype association mapping; Genomic Best Linear Unbiased Predictors (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Clark and van der Werf 2013</a>), and phenotype predictions (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">de los Campos <em>et al.</em> 2013</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Pérez and de los Campos 2014</a>). Although some of the methods such as SNP-BLUP can avoid explicitly factorizing a GRM (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Koivula <em>et al.</em> 2012</a>), factorizing a matrix is still a main technique needed by most analytic models.</p><p id="p-5">The frequently used matrix-factorization as an alternative to eigen-decomposition is the Singular Value Decomposition (SVD) of a rectangular matrix (<em>e.g.</em>, a genotype matrix) which avoids heavy computations but approximately achieves similar precision. For instance, in the setting of a LMM, David Heckerman’s group has developed a method that selects only a subset of genomic variants to form a genotype low-rank matrix that can be solved faster and lead to fewer false positives (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lippert <em>et al.</em> 2011</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Listgarten <em>et al.</em> 2012</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">2013</a>). Many tools have soon adopted this approach as an alternative of solving eigen-decomposition of a full GRM.</p><p id="p-6">Additionally, many emerging novel analytical procedures, such as a recent proposal for correcting the confounding effects of cell-type compositions (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rahmani <em>et al.</em> 2016</a>) in epigenome-wide association studies and the correction of confounding effects in the analysis of single-cell RNA-Seq data (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Buettner <em>et al.</em> 2015</a>), need factorization of large matrices.</p><p id="p-7">Researchers have also resorted to approximations to evade the computational burden of calculating very large matrices. For PCA analyses, Alkes Price’s group developed an approximate algorithm using only the top few PCAs (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Galinsky <em>et al.</em> 2016</a>). For calculating the inverse of a GRM of many related animals, one may start with “Core Animals” to approximately solve the whole matrix (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Masuda <em>et al.</em> 2016</a>). In the R statistical programming language community, chunking algorithms and memory-mapping techniques are utilized to reduce the memory use for simple functions such as reading data and conducting linear regression (<em>e.g.</em>, The Bigmemory Project, <a href="http://www.bigmemory.org">http://www.bigmemory.org</a>). Parallelization strategies using “split-apply-combine” have also been implemented such as in the BGData suite of packages (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">de los Campos and Grueneberg 2017</a>). There are many efforts using Graphics Processing Units (GPUs) to speed up computations, for instance BLASX (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2016</a>) for numerical computations. Additionally, some researchers used distributed memory to solve the problem of limited memory on a single server, such as Elemental (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Poulson <em>et al.</em> 2013</a>). Although these approaches address individual problems on a case-by-case basis, researchers need universal and exact solutions for eigen-decomposition as well as singular value decomposition of very large GRMs and genotype matrices for various existing and future applications.</p><p id="p-8">The above applications are evidences that the eigen-decomposition (and its alternative, SVD) in genomic studies is indeed the first-line characterization of the relationship between participating individuals and the solution implementing these on ubiquitously available computer hardware is needed. In this work, we present a novel tool, OCMA, to factorize very large matrices using disk-based solutions. This tool, in conjunction with the other tools developed by us (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Long <em>et al.</em> 2013</a>) and others, will allow researcher to carry out association mapping and phenotype predictions nimbly using limited computational resources.</p><p id="p-9">In computer science, “<em>mmap</em>” (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) is a function (or, precisely, a “system-call”) that maps files or devices into memory (precisely, maps to the virtual address space of a computing process). “<em>mmap</em>” was originally designed as part of Berkeley Software Distribution version of Unix (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">McKusick <em>et al.</em> 2014</a>) and is now universally supported by all major operating systems (<em>i.e.</em>, Mac OS, Windows and Linux). This function efficiently handles data exchange between memory and disk, allowing program developers to carry out computational tasks as though the data in the disk were in memory without taking care of implementation details such as allocating memory, looking up addresses, reading data, writing used data, and de-allocating the memory again when it is no longer needed (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Song <em>et al.</em> 2016</a>). As a result, it appears that the calculations happen “out of the memory”, a technique called “<em>out-of-core</em>” in computer science. Mmap has been extensively used in many computing fields such as high-performance computing (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Van Essen <em>et al.</em> 2013</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Song <em>et al.</em> 2016</a>), graph computation (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lin <em>et al.</em> 2014</a>), system virtual machine (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wang <em>et al.</em> 2015</a>), and databases (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lou and Ludewigt 2015</a>). Mmap has also been used in bioinformatics in recent years, including the indexing of next-generating sequencing reads (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Salavert <em>et al.</em> 2016</a>) and tree-based backward search (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Salavert <em>et al.</em> 2015</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/13/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustrating schema of mmap. The horizontal bar represents the physical address in the disk and the vertical bar represents the virtual address from the perspective of a computing process in the memory." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-307340739" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Illustrating schema of &lt;em&gt;mmap&lt;/em&gt;. The horizontal bar represents the physical address in the disk and the vertical bar represents the virtual address from the perspective of a computing process in the memory.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/13/F1.medium.gif" width="440" height="318"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/1/13/F1.medium.gif" width="440" height="318"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/13/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/13/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491804" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Illustrating schema of <em>mmap</em>. The horizontal bar represents the physical address in the disk and the vertical bar represents the virtual address from the perspective of a computing process in the memory.</p><div class="sb-div caption-clear"></div></div></div><div class="section methods" id="sec-1"><h2 class="">Methods</h2><p id="p-11">The <em>Intel Math Kernel Library</em> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wang <em>et al.</em> 2014</a>) is a collection of optimized functions that represent the state-of-the-art for numerical computation. Using the core algorithm supplied by the Intel MKL and the memory-disk mapping enabled by mmap, here, we develop a tool called Out-of-Core Matrices Analyzer (OCMA) that solves eigen-decomposition of GRMs and SVD in the disk (<em>i.e.</em>, out-of-core).</p><p id="p-12">In order to integrate Intel MKL with mmap, we have thoroughly assessed various options of implementation to optimize the performance of OCMA. In the current version, OCMA adopts the ILP64 library that needs the support of 64-bit machines to accommodate very large indexes of matrices. We have selected the “ssyevd” subroutine that computes all eigenvalues and, optionally, all eigenvectors of a real symmetric matrix, using divide and conquer algorithm. We have re-engineered the interface of “ssyevd” so that the large matrices used in it can be disk-based using mmap (Supplementary Note I). Similarly, we have identified “sgesdd” that computes the SVD of a general rectangular matrix using a divide and conquer method. Additionally, to meet many applications that one only needs a small number of singular values and eigenvectors, we have also implemented a function that calculates the top <em>k</em> singular-values (<em>k</em> &lt; min (<em>N, M</em>) where <em>N</em> and <em>M</em> are the dimensions of a matrix, typically standing for the number of individuals and the number of genetic markers in a genotype matrix). This function is supported by the “sgesvdx” in Intel MKL. These subroutines require more workspace but are faster in some cases, especially for large matrices. Intel MKL provides a two-level C interface to LAPACK, consisting of a high-level interface and a middle-level interface. The high-level interface handles all workspace memory allocation internally, while the middle-level interface requires the user to provide workspace arrays as in the original FORTRAN interface. To facilitate mmap-based solutions, we have used the middle-level interface that is similar to its original FORTRAN interface. The details of the implementation are presented in Supplementary Note I.</p><p id="p-13">OCMA supports parallel computing. By default, OCMA will automatically decide how many threads to use based on the available hardware. (In our tests to be presented in Results, the OCMA process uses four threads, which is based on the hardware of the desktop computer we use.) Additionally, the users can also utilize environment variable (MKL_NUM_THREADS) to specify the number of threads (OCMA Users’ Manual).</p><div id="sec-2" class="subsection"><h3>Data Availability</h3><p id="p-14">We used the UK Biobank data that is available through application to the data generator (<a href="https://www.ukbiobank.ac.uk">https://www.ukbiobank.ac.uk</a>). Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7384973">https://doi.org/10.25387/g3.7384973</a>.</p></div></div><div class="section results" id="sec-3"><h2 class="">Results</h2><p id="p-15">OCMA provides two main functions: eigen-decomposition for square symmetric matrices (<em>e.g.</em>, GRMs) and singular value decomposition (SVD) for rectangular matrices (<em>e.g.</em>, genotype matrices). The users can also select to calculate a subset of top singular values/vectors. The input files can be prepared using plain text or the mainstream binary format (detailed in the Users Manual). OCMA also supports the format transformation between text and binary files. The output files will be the eigenvalues/vectors (or singular values/vectors) in the same format as the input. All three operating systems (Linux, Mac OS, and Windows) are fully supported.</p><p id="p-16">An example command using OCMA may be:</p><p id="p-17">$&gt; ocma eigen single disk n A E Q</p><p id="p-18">where ‘<strong>ocma</strong>’ is the name of the executable; ‘<strong>eigen</strong>’ stands for eigen-decomposition (alternatively one may use ‘singular’ for SVD); ‘<strong>single</strong>’ stands for single-precision (alternatively one may use ‘double’ for double precision); ‘<strong>disk</strong>’ specifies the function that uses disk-based solution (alternatively one may use ‘memory’ for smaller matrices); ‘<strong>n</strong>’ is the dimension of the matrix; ‘<strong>A</strong>’ is the filename of the input matrix; ‘<strong>E</strong>’ is the filename of the output file that stores the eigenvalues of matrix A; and ‘<strong>Q</strong>’ is the filename of the output file that stores the eigenvectors of matrix A.</p><p id="p-19">We first compared the results of an eigenvalue decomposition and an SVD computed using OCMA with the same factorization carried out using MATLAB functions “eig” and “svd” (version 2012b) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). As expected, the outcome of OCMA and MATLAB is highly consistent as quantified by the three standard distances (that is called <em>norm</em>) used in numerical computing (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Additionally, these initial comparisons also show that, for relatively small matrices that MATLAB can handle, OCMA is 4 – 11 times faster than MATLAB for the computation of eigen-decomposition and around 2 – 3 times faster than MATLAB for the SVD (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491992/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491992/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The algorithm of OCMA calculates the right outcome. Eigenvectors (or singular vectors) are compared using Infinite norm <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/13/T1/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/13/T1/embed/mml-math-1.gif"/></noscript></span></span>, Taxicab norm <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/13/T1/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/13/T1/embed/mml-math-2.gif"/></noscript></span></span>, and Euclidean norm <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/13/T1/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/13/T1/embed/mml-math-3.gif"/></noscript></span></span>. We use <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/13/T1/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/13/T1/embed/mml-math-4.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/13/T1/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/13/T1/embed/mml-math-5.gif"/></noscript></span></span> to denote the vectors calculated by OCMA and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/13/T1/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/13/T1/embed/mml-math-6.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/13/T1/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/13/T1/embed/mml-math-7.gif"/></noscript></span></span> to denote the vectors calculated by MATLAB (version 2012b). The comparison for eigen-decomposition is presented in the upper table, and the comparison for singular-value decomposition is presented in the lower one. In the upper table, <em>N</em> denotes the number of individuals. In lower table, <em>N</em> = 1,000,000, and <em>M</em> denotes the number of genetic markers. The configuration of the personal computer: Intel Core i7-6700 CPU (4 cores), Memory = 24GB. Disk = Samsung SSD 850 EVO 250GB. The operating system is Windows 7. Time is measured by wall-clock (instead of CPU time)</span><div class="sb-div caption-clear"></div></div></div><p id="p-20">To assess memory usage and computational time for analyses involving genomic big-data we used GRM describing additive genetic relationships among 100,000 distantly related (GRM between pairs of individuals smaller than 0.03) from the interim release of the UK-Biobank. This GRM was computed using 589,028 SNP genotypes from the Affymetrix UK BiLEVE Axiom and Affymetrix UK Biobank Axiom arrays (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Collins 2012</a>). The GRM was computed using the BGData R-package (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">de los Campos and Grueneberg 2017</a>). Further details about the criteria used for subject and SNP inclusion can be found in (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Kim <em>et al.</em> 2017</a>) and the procedure to compute GRM is in Supplementary Note III.</p><p id="p-21">When the GRM is small enough to be fully loaded in memory, we can directly compare the disk-based solution to the memory-based one (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, <em>N</em> = 10,000 to 50,000). In most cases, the overhead incurred by disk-memory mapping only causes a modest slow-down in the disk-based solution compared to the memory-based one. Notably, taking advantage of the efficient implementation offered by Intel MKL, OCMA can decompose a regular matrix of <em>N</em> = 10,000 in less than a minute. More importantly, OCMA rapidly solves much larger GRMs that are impossible to load in memory (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, <em>N</em> = 60,000 to 100,000). In particular, it solves a GRM of 100,000×100,000 in only 5 days in our personal computer that has only a memory of 24GB (Supplementary Table S4).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491949/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491949/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Runtime and memory consumption of OCMA in a Linux personal computer for eigen decomposition. The same hardware in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> is used. Operating system is CentOS Linux release 7.3.1611. Four threads are used for the test. The three columns in the “Calculation time” stand for the runtime for three tools: GCTA (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Yang <em>et al.</em> 2011</a>), OCMA using memory only, and OCMA using disk (based on mmap technique). The careful interpretation of the comparison between GCTA and OCMA is explained in the main text and Supplementary Notes II. *The usage is slightly larger than the physical memory because of the swapping by the operating system. **GCTA was tested in an HPC cluster when the memory of a personal computer is insufficient. Time is measured by wall-clock (instead of CPU time). Memory consumption is estimated using the formula on the GCTA website and the Intel MKL specification (detailed in the Supplementary Notes). The sign “/” indicates that the system does not allow the calculation to happen due to limited memory and swap spaces or it exceeds the maximal runtime in the HPC</span><div class="sb-div caption-clear"></div></div></div><p id="p-22">Although the core innovation here is that OCMA can handle very large arrays in disk without the needs of large memory, OCMA’s memory-based algorithm by-itself is at least up-to-date with, if not faster than, state-of-the-art GRM analysis tools. Two of the most notable tools using GRM for association mapping and heritability analysis are GCTA (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Yang <em>et al.</em> 2011</a>) and EMMAX (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Kang <em>et al.</em> 2010</a>). EMMAX is not actively maintained and is orders of magnitude slower than OCMA. The most recent release of GCTA (1.9x.beta) is much faster and less resource intensive than its previous versions, but doesn’t provide a single function for only solving eigen-decomposition for the purpose of a comparison. However, the GREML (genomic-relatedness-based restricted maximum-likelihood) function that involves matrix inverse is a frequently used function estimating heritability etc. in genetic analyses, which provides a benchmark for comparison, albeit indirect. While the runtime and memory usage of GREML are higher than the use of matrix-factorization alone, we have derived formulas to use these results to estimate the actual use of resources (detailed and justified in the Supplementary Note II) in matrix-factorization. These estimates (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>, <em>N</em> = 10,000 to 50,000) show that OCMA is more efficient than GCTA when the memory is sufficient for GCTA to perform.</p><p id="p-23">The above comparisons are based on the Linux version of OCMA. As mmap is a call at the level of operating system, the implementations of OCMA are quite different under different platforms. We have implemented the Mac OS and Windows version of OCMA. The performance of OCMA in the Windows system is presented in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. The performance on Mac OS is similar to that on Linux. This is to be expected due to their shared Unix kernel. The performance on Windows is even faster than the Linux version, presumably due to the extra optimization by the operating system.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491887/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491887/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Comparison of the runtime of OCMA under the Windows and Linux operating systems for eigen decomposition. The same machine described in the <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">2</a> was used. Four threads are used in the test. Time is measured by wall-clock (instead of CPU time). The sign “/” indicates that the system does not allow the calculation to happen due to limited memory and swap spaces</span><div class="sb-div caption-clear"></div></div></div><p id="p-24">The performance of OCMA calculating SVDs (using the options “single” and “singular”) for very large genotype matrices (1 million simulated individuals times thousands of markers) is presented in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>. Evidently, the calculation of SVD is much faster and even less memory-intensive. Further, the performance of OCMA calculating a subset of SVs is displayed in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>. The runtime of calculating a small subset of SVs is not significantly faster than calculating all the SVs. This is because, for SVD computations, a significant proportion of runtime is spent on calculating the product of the matrix and its transpose (<em>i.e.</em>, A’ × A, where A is the <em>N</em> × <em>M</em> matrix). As this is a necessary step in the “sgesvdx” function of Intel MKL, calculating a subset of SVDs doesn’t save much time in this implementation.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491990/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491990/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Runtime of OCMA for Singular Value Decomposition (SVD). Sample size <em>N</em> = 1 million. <em>M</em> is the number of selected genetic markers. The same machine described in the <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Tables 1</a> and <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">2</a> is used. Four threads are used in the test. Time is measured by wall-clock (instead of CPU time). The sign “/” indicates that the system does not allow the calculation to happen due to limited memory and swap spaces</span><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491889/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491889/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Runtime of OCMA for a subset of singular values. Sample size <em>N</em> = 1 million. <em>M</em>, the number of selected genetic markers, is 5,000. <em>K</em> is the number of top SVs. The same machine described in the <a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">2</a> is used. Four threads are used in the test. Time is measured by wall-clock (instead of CPU time)</span><div class="sb-div caption-clear"></div></div></div><p id="p-25">In addition to laptop/desktop-based data analyses, OCMA also supports HPC-based parallel computations that use many threads. We have tested OCMA on an HPC server and the results are presented in <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>. As a comparison of the effectiveness of parallelization, we also have tested OpenBLAS, an optimized BLAS library using multi-thread. The same method, “ssyevd” in OpenBLAS is used for this test. Evidently, the parallelization in Intel MKL, which is adopted by OCMA, is more effective than the OpenBLAS alternative (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491911/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491911/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Comparison between OCMA and OpenBLAS, a multi-thread package for matrix factorization. The same function (“ssyevd”) in OpenBLAS is used. The computing node has two E5-2690V4 CPUs with 28 cores in total. The total memory available is 192GB. Environmental variables MKL_NUM_THREADS and OPENBLAS_NUM_THREADS are used to specify the number of threads in OCMA and OpenBLAS respectively</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">As mmap involves complicated disk-memory data exchange, the memory usage of OCMA in practice is a dynamic function depending on the system status. However, based on the specification of Intel MKL and our implementation, we can provide a static analysis of the memory and disk usage. Quantitative analyses of the memory and disk usage of <em>mmap</em> are discussed and presented in Supplementary Tables 1-3. Intuitively there is an additional overhead incurred in the operating system, as the CPU must wait for data exchange between memory and disk. Under the desktop with Windows system, we monitored the CPU usage during the test and find this effect does exist but falls in a reasonable range. Supplementary Figures 1 and 2 show this effect for Eigen-decomposition and SVD respectively.</p><div id="sec-4" class="subsection"><h3>Software Availability</h3><p id="p-27">The source code and executable are freely available at GitHub: <a href="https://github.com/precisionomics/OCMA">https://github.com/precisionomics/OCMA</a>. The repository provides OCMA for Windows (ocma-0.1.zip) and Linux/Unix systems (ocma-0.1.tar.gz). Upon decompression, one will see two folders: src and bin. The folder src contains all the source code, and the folder bin contains the compiled binary executable. Under the folder bin, two small matrices, SM10 and M3_4 for testing purpose are also provided.</p><p id="p-28">The users can also build their own binary executable. The program is written in the C language, and it uses Intel MKL. Thus, in order to compile the source code, one needs to install C compiler, <em>e.g.</em>, the Intel C/C++ Compilers, the GNU C Compiler or Visual Studio C Compiler. Additionally, one needs to install Intel MKL. When C compiler and Intel MKL are available in the system, one can enter the src folder and modify the makefile based on the C compiler and the Intel MKL paths; then type make or nmake to compile. After that, the ocma executable will be generated in the bin folder.</p></div></div><div class="section discussion" id="sec-5"><h2 class="">Discussion</h2><p id="p-29">OCMA allows researchers to factorize symmetric and non-symmetric of very large real-valued matrices, including GRMs, genotype matrices and any other real-valued matrices. Thus, OCMA enables many Big-Data applications that use distance and similarity matrices that do not use GRMs. Furthermore, the recent trend of analyzing multiple traits together in association mapping (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Korte <em>et al.</em> 2012</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Casale <em>et al.</em> 2015</a>) and genomic selection (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tsuruta <em>et al.</em> 2011</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Jia and Jannink 2012</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chen <em>et al.</em> 2016</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">2017</a>) involves matrices yet larger. The OCMA algorithm will be helpful in solving such larger matrices. Moreover, the study of Gene-Environment Interaction (GxE) in both health and agriculture also involves large matrices (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Montesinos-López <em>et al.</em> 2016</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Moore <em>et al.</em> 2018</a>), which can be solved nimbly by OCMA.</p><p id="p-30">As one may figure out from Figure S1, from the perspective of CPU usage, the swap within the operating system outperforms mmap for a matrix that is a little bit larger than the physical memory, especially in the case of calculating SVD. This might be because of the fact that the swap is supported by continuous and dedicated disk spaces designed to meet the specific requirements of memory virtualization. In contrast, mmap uses general file system that usually does not provide continuous space. So the mmap may consume more time than swap when the space for swap is sufficient. However, swap cannot meet the requirements of analyzing very large files. This is because (1) the performance of swap heavily relies on the setting of the operating system and how many other processes are sharing it; (2) swap requires administrators to set the parameters which is not available for the end users of an HPC; and (3) swap cannot be very large as it requires continuous and dedicated spaces in the disk. In contrast, mmap offers a scalable and flexible file mapping between the memory and the general file system, which does not suffer from the above limitations, therefore can be applied to very large files.</p><p id="p-31">A trade-off in the design of OCMA is how to strike a good balance between space/speed and the nature of its memory visit. For instance, based on the Intel MKL website, it is recommended to use “ssyevr” for eigenvalue problems because “<em>its underlying algorithm is fast and uses less workspace</em>”. We have tested the function “ssyevr” against “ssyevd” (the function we adopted), and indeed we find that “ssyevr” uses around 65% memory that is required by “ssyevd”. However, “ssyevr” is slower when solving large matrices sized at the level of tens of thousands. In case of a very large matrix for which mmap is needed, the performance of “ssyevr” decreased dramatically. For instance, for a GRM of 80,000 that “ssyevd” can solve in 18.31 hr using a desktop with 24GB memory and 4 threads, “ssyevr” cannot be finished in 5 days. We expect this is because of the pattern of memory visit of “ssyevr”, which requires intensive workspace visits that needs more frequent data exchange between memory and disk. So although “ssyevr” needs less workspace (in terms of either memory or disk), its more frequent workspace visits leads to very poor performance in the context of an out-of-core implementation.</p><p id="p-32">As an immediate future development, we plan to implement more factorization functions such as QR decomposition and Cholesky decomposition. Moreover, we will integrate OCMA with various tools especially the ones we have developed and maintained for years. For instance we will integrate OCMA with BGLR, a tool that we have developed for phenotype predictions using genomics data (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">de los Campos <em>et al.</em> 2013</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Pérez and de los Campos 2014</a>).</p><p id="p-33">The initial release of OCMA is based on C language that is the default for <em>mmap</em>. We will develop APIs (Application Programming Interfaces) for R, Java and Python to accommodate broader communities. The extremely fast function in Intel MKL is optimized for Intel architectures that are supporting most current personal computers in the current market. Therefore, OCMA may be sub-optimal if running on other machines. However, other vendors, such as AMD, provide similar libraries. We are extending OCMA using the AMD Core Math Library, to cover the non-dominant architectures.</p><p id="p-34">Another non-trivial extension will be the integration of OCMA with our existing tool JAWAMix5, a tool that enables out-of-core association mapping at the level of population genotype files (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Long <em>et al.</em> 2013</a>). The JAWAmix5 (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Long <em>et al.</em> 2013</a>) package uses HDF5 (<a href="https://www.hdfgroup.org">https://www.hdfgroup.org</a>) for accessing genotype data stored in disk. HDF5 builds indexes of genotype data to allow fast random access to data. JAWAmix5 offers scalable functions for the generation of GRM and association mappings using standard linear models as well as linear mixed models (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Long <em>et al.</em> 2013</a>). However, HDF5 is not efficient for complicated matrix operations because the index-based techniques are efficient only if the data are in a natural order and not frequently updated. So, JAWAmix5 loads matrices into the memory and conduct matrix factorization in memory. This was fine at that time (2013) however becomes increasingly a bottleneck when analyzing a sample with tens of thousands individuals. Although mmap is not as efficient as HDF5, it does allow users use memory mapping without concerning the properties of the composition of the underlying data.</p><p id="p-35">HDF5 and mmap are two fundamentally different techniques. HDF5 facilitates a hierarchical indexing of the data so that they can be stored in the disk but randomly accessed by the computing processes as though they are in the main memory. This is convenient for some data that are naturally straightforward to index. However, if the reading and writing of the files cannot be well predicted, the HDF5-based solution could be very slow even if the file size is small. In contrast, mmap, despite its lower efficiency for well-formatted data (comparing with HDF5), offers a full “memory virtualization.” That is, one can just use disk as if it were memory regardless the type and organization of data, as well as how the users are going to read and write them. The operating system will handle which part of the data will be loaded and swapped. In statistical genomics, the genotype data (<em>e.g.</em>, a VCF file) meet the requirements of HDF5 perfectly because the genomic coordinate system offers a natural index and it is rarely updated during the statistical analyses. However, when there are many other intermediate quantities, <em>e.g.</em>, metrics to assist the decomposition of a matrix, for which one needs to generate and update frequently, the perfect choice may be mmap, which does not rely on the order of data. HDF5 and mmap nicely complement the advantages of each other. For the moment, based on JAWAmix5’s functions of association mapping and calculation of GRM and OCMA’s function of factorization of matrices, one can carry out GWAS using our out-of-core technologies. We plan to develop more functions for genomic predictions, <em>e.g.</em>, GBLUP and ssGBLUP (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aguilar <em>et al.</em> 2014</a>) that have been extensively used in the fields.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-36"><em>Funding</em>. This work was supported by Natural Science Foundation of Guangdong Province (2018A030313438) and Special Funds for Discipline and Specialty Construction of Guangdong Higher Education Institutions (2016KTSCX040) to Z.X.; University of Calgary Startup and URGC Seed Grants, Canada Foundation for Innovation, and NSERC Discovery Grant to Q.L.; NSERC Discovery Grant to Q.Z.; and NIH to G.D.L.C. (R01 NIGMS 101219) and X.S. (R15HG009565).</p><p id="p-37"><em>Dataset</em>. G.D.L.C. has access to the genotype data of the UK Biobank and all the data analysis related to the UK Biobank data were done by G.D.L.C.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-38">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7384973">https://doi.org/10.25387/g3.7384973</a>.</p></li><li class="fn" id="fn-2"><p id="p-39"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-10-22"><span class="received-label">Received </span>October 22, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-11-26"><span class="accepted-label">Accepted </span>November 26, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Xiong <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-other" id="cit-9.1.13.1"><div class="cit-metadata"><cite>Aguilar, I., I. Misztal, S. Tsuruta, A. Legarra, and H. Wang, 2014 PREGSF90 – POSTGSF90: Computational Tools for the Implementation of Single-step Genomic Selection and Genome-wide Association with Ungenotyped Individuals in BLUPF90 Programs. Proceedings of 10th World Congress of Genetics Applied to Livestock Production 2014: 680.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.13140/2.1.4801.5045</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.2" data-doi="10.1038/nbt.3102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buettner</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Natarajan</span>  <span class="cit-name-given-names">K. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casale</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proserpio</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scialdone</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Computational analysis of cell-to-cell heterogeneity in single-cell RNA-sequencing data reveals hidden subpopulations of cells.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">160</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D155%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3102%26rft_id%253Dinfo%253Apmid%252F25599176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25599176&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F13.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.3" data-doi="10.1038/nmeth.3439"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casale</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rakitsch</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lippert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stegle</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Efficient set tests for the genetic analysis of correlated traits.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">755</span>–<span class="cit-lpage">758</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3439</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D755%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3439%26rft_id%253Dinfo%253Apmid%252F26076425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3439&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26076425&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F13.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.4" data-doi="10.1007/s11032-016-0478-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iqbal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">R.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spaner</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Effect of Lr34/Yr18 on agronomic and quality traits in a spring wheat mapping population and implications for breeding.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">53</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-016-0478-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D36%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-016-0478-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-016-0478-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-other" id="cit-9.1.13.5" data-doi="10.1007/s11032-016-0587-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genetic mapping of yield traits using RIL population derived from Fuchuan Dahuasheng and ICG6375 of peanut (<em>Arachis hypogaea</em> L.).</span> <span class="cit-source">Mol. Breed.</span> <span class="cit-vol">37</span>: <span class="cit-fpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-016-0587-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D37%26rft.spage%253D17%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-016-0587-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-016-0587-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.6" data-doi="10.1007/978-1-62703-447-0_13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic best linear unbiased prediction (gBLUP) for the estimation of genomic breeding values.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">1019</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">330</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-62703-447-0_13</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%26rft.volume%253D1019%26rft.spage%253D321%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-62703-447-0_13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-62703-447-0_13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.7" data-doi="10.1016/S0140-6736(12)60404-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">What makes UK Biobank special?</span> <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">379</span>: <span class="cit-fpage">1173</span>–<span class="cit-lpage">1174</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0140-6736(12)60404-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.stitle%253DLancet%26rft.aulast%253DCollins%26rft.auinit1%253DR.%26rft.volume%253D379%26rft.issue%253D9822%26rft.spage%253D1173%26rft.epage%253D1174%26rft.atitle%253DWhat%2Bmakes%2BUK%2BBiobank%2Bspecial%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252812%252960404-8%26rft_id%253Dinfo%253Apmid%252F22463865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(12)60404-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22463865&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F13.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302230400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-other" id="cit-9.1.13.8"><div class="cit-metadata unstructured">de los Campos, G., and A. Grueneberg, 2017 A Suite of Packages for Analysis of Big Genomic Data [R package BGData version 1.0.0]. <a href="https://github.com/QuantGen/BGData">https://github.com/QuantGen/BGData</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.9" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOS8xLzEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.10" data-doi="10.1371/journal.pgen.1005048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic heritability: What is it?</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005048</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005048</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005048%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005048%26rft_id%253Dinfo%253Apmid%252F25942577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25942577&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F13.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.11" data-doi="10.1016/j.ajhg.2015.12.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galinsky</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhatia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loh</span>  <span class="cit-name-given-names">P.-R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Georgiev</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mukherjee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Fast principal-component analysis reveals convergent evolution of ADH1B in Europe and East Asia.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">456</span>–<span class="cit-lpage">472</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2015.12.022</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D456%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.12.022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.12.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.12" data-doi="10.1534/genetics.112.144246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multiple-trait genomic selection methods increase genetic value prediction accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.144246</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144246%26rft_id%253Dinfo%253Apmid%252F23086217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MTMiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy85LzEvMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.13" data-doi="10.1038/ng.548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sul</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Service</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">S.-Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Variance component model to account for sample structure in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">354</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.548</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D348%26rft.epage%253D354%26rft.atitle%253DVariance%2Bcomponent%2Bmodel%2Bto%2Baccount%2Bfor%2Bsample%2Bstructure%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.548%26rft_id%253Dinfo%253Apmid%252F20208533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208533&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F13.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.14" data-doi="10.1534/genetics.117.300271"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grueneberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Will big data close the missing heritability gap?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">207</span>: <span class="cit-fpage">1135</span>–<span class="cit-lpage">1145</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300271</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300271%26rft_id%253Dinfo%253Apmid%252F28893854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNy8zLzExMzUiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy85LzEvMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.15" data-doi="10.3168/jds.2011-4874"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koivula</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strandén</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mäntysaari</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Different methods to calculate genomic predictions–comparisons of BLUP at the single nucleotide polymorphism level (SNP-BLUP), BLUP at the individual level (G-BLUP), and the one-step approach (H-BLUP).</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">4065</span>–<span class="cit-lpage">4073</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2011-4874</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DKoivula%26rft.auinit1%253DM.%26rft.volume%253D95%26rft.issue%253D7%26rft.spage%253D4065%26rft.epage%253D4073%26rft.atitle%253DDifferent%2Bmethods%2Bto%2Bcalculate%2Bgenomic%2Bpredictions--comparisons%2Bof%2BBLUP%2Bat%2Bthe%2Bsingle%2Bnucleotide%2Bpolymorphism%2Blevel%2B%2528SNP-BLUP%2529%252C%2BBLUP%2Bat%2Bthe%2Bindividual%2Blevel%2B%2528G-BLUP%2529%252C%2Band%2Bthe%2Bone-step%2Bapproach%2B%2528H-BLUP%2529.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2011-4874%26rft_id%253Dinfo%253Apmid%252F22720963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2011-4874&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22720963&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F13.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.16" data-doi="10.1038/ng.2376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjalmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segura</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A mixed-model approach for genome-wide association studies of correlated traits in structured populations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1066</span>–<span class="cit-lpage">1071</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2376</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKorte%26rft.auinit1%253DA.%26rft.volume%253D44%26rft.issue%253D9%26rft.spage%253D1066%26rft.epage%253D1071%26rft.atitle%253DA%2Bmixed-model%2Bapproach%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bof%2Bcorrelated%2Btraits%2Bin%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2376%26rft_id%253Dinfo%253Apmid%252F22902788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22902788&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F13.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kahng</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabrin</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chau</span>  <span class="cit-name-given-names">D. H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">MMap: Fast billion-scale graph computation on a PC via memory mapping.</span> <abbr class="cit-jnl-abbrev">Proc. IEEE Int. Conf. Big Data</abbr> <span class="cit-vol">2014</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BIEEE%2BInt.%2BConf.%2BBig%2BData%26rft.volume%253D2014%26rft.spage%253D159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.18" data-doi="10.1038/nmeth.1681"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lippert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Listgarten</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadie</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">R. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">FaST linear mixed models for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">833</span>–<span class="cit-lpage">835</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1681</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLippert%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D833%26rft.epage%253D835%26rft.atitle%253DFaST%2Blinear%2Bmixed%2Bmodels%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1681%26rft_id%253Dinfo%253Apmid%252F21892150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1681&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21892150&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F13.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295358000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.19" data-doi="10.1038/ng.2620"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Listgarten</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lippert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">FaST-LMM-Select for addressing confounding from spatial structure and rare variants.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">470</span>–<span class="cit-lpage">471</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2620</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D470%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2620%26rft_id%253Dinfo%253Apmid%252F23619783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2620&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23619783&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F13.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.20" data-doi="10.1038/nmeth.2037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Listgarten</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lippert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadie</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">R. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Improved linear mixed models for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">526</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2037</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DListgarten%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.issue%253D6%26rft.spage%253D525%26rft.epage%253D526%26rft.atitle%253DImproved%2Blinear%2Bmixed%2Bmodels%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2037%26rft_id%253Dinfo%253Apmid%252F22669648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22669648&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F13.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304778500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.21" data-doi="10.1093/bioinformatics/btt122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjalmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forai</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seren</span>  <span class="cit-name-given-names">Ü.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">JAWAMix5: an out-of-core HDF5-based java implementation of whole-genome association studies using mixed models.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1220</span>–<span class="cit-lpage">1222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt122</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt122%26rft_id%253Dinfo%253Apmid%252F23479353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt122&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23479353&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F13.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318573900020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.22" data-doi="10.1016/j.cpc.2015.04.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lou</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ludewigt</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">MMAPDNG: A new, fast code backed by a memory-mapped database for simulating delayed γ-ray emission with MCNPX package.</span> <abbr class="cit-jnl-abbrev">Comput. Phys. Commun.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cpc.2015.04.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BPhys.%2BCommun.%26rft.volume%253D194%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cpc.2015.04.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cpc.2015.04.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.23" data-doi="10.3168/jds.2015-10540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masuda</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuruta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Implementation of genomic recursions in single-step genomic best linear unbiased predictor for US Holsteins with a large number of genotyped animals.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">1968</span>–<span class="cit-lpage">1974</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2015-10540</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D99%26rft.spage%253D1968%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2015-10540%26rft_id%253Dinfo%253Apmid%252F26805987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2015-10540&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26805987&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F13.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-9.1.13.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKusick</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neville-Neil</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">R. N. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">The Design and Implementation of the FreeBSD Operating System</span>, <span class="cit-publ-name">Addison-Wesley Professional</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.25" data-doi="10.1534/g3.116.032359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toledo</span>  <span class="cit-name-given-names">F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Hernández</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A Genomic Bayesian Multi-trait and Multi-environment Model.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2725</span>–<span class="cit-lpage">2744</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.032359</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.032359%26rft_id%253Dinfo%253Apmid%252F27342738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvOS8yNzI1IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOS8xLzEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-web" id="cit-9.1.13.26" data-doi="10.1101/270611"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casale</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonder</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horta</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Consortium</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A linear mixed model approach to study multivariate gene-environment interactions.</span> <span class="cit-source">bioRxiv</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/270611</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F270611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIyNzA2MTF2MSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzkvMS8xMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.27" data-doi="10.1136/bmj.i1757"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peplow</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The 100,000 Genomes Project.</span> <abbr class="cit-jnl-abbrev">BMJ</abbr> <span class="cit-vol">353</span>: <span class="cit-fpage">i1757</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1136/bmj.i1757</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMJ%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fbmj.i1757%26rft_id%253Dinfo%253Apmid%252F27075170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYm1qIjtzOjU6InJlc2lkIjtzOjE3OiIzNTMvYXByMTNfMy9pMTc1NyI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzkvMS8xMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.28" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression and prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOS8xLzEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.29" data-doi="10.1145/2427023.2427030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poulson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van de Geijn</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammond</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Elemental: A new framework for distributed memory dense matrix computations.</span> <abbr class="cit-jnl-abbrev">ACM Trans. Math. Softw.</abbr> <span class="cit-vol">39</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">: </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1145/2427023.2427030</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACM%2BTrans.%2BMath.%2BSoftw.%26rft_id%253Dinfo%253Adoi%252F10.1145%252F2427023.2427030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1145/2427023.2427030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.30" data-doi="10.1038/nmeth.3809"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rahmani</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baran</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Sparse PCA corrects for cell type heterogeneity in epigenome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">445</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3809</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D13%26rft.spage%253D443%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3809%26rft_id%253Dinfo%253Apmid%252F27018579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3809&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27018579&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F13.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.31" data-doi="10.1038/nbt0308-303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ringnér</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">What is principal component analysis?</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">304</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt0308-303</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRingner%26rft.auinit1%253DM.%26rft.volume%253D26%26rft.issue%253D3%26rft.spage%253D303%26rft.epage%253D304%26rft.atitle%253DWhat%2Bis%2Bprincipal%2Bcomponent%2Banalysis%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt0308-303%26rft_id%253Dinfo%253Apmid%252F18327243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt0308-303&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18327243&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F13.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254123400032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.32" data-doi="10.2174/1574893611666160212235359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salavert</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomás</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medina</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sadakane</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanquer</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Pair-end inexact mapping on hybrid GPU environments and out-of-core indexes.</span> <abbr class="cit-jnl-abbrev">Curr. Bioinform.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">469</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2174/1574893611666160212235359</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBioinform.%26rft.volume%253D11%26rft.spage%253D459%26rft_id%253Dinfo%253Adoi%252F10.2174%252F1574893611666160212235359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/1574893611666160212235359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.33" data-doi="10.1186/s12859-014-0438-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salavert</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomás</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tárraga</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medina</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dopazo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fast inexact mapping using advanced tree exploration on backward search methods.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-014-0438-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D16%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-014-0438-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-014-0438-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.34" data-doi="10.1145/2846100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">N. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Son</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeom</span>  <span class="cit-name-given-names">H. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Efficient memory-mapped I/O on fast storage device.</span> <abbr class="cit-jnl-abbrev">ACM Trans. Storage</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1145/2846100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACM%2BTrans.%2BStorage%26rft.volume%253D12%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1145%252F2846100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1145/2846100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.35" data-doi="10.1038/nrg3821"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balding</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Relatedness in the post-genomic era: is it still useful?</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">44</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3821</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D16%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3821%26rft_id%253Dinfo%253Apmid%252F25404112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3821&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25404112&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F13.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.36" data-doi="10.1038/nprot.2011.457"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stegle</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piipari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Using probabilistic estimation of expression residuals (PEER) to obtain increased power and interpretability of gene expression analyses.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">500</span>–<span class="cit-lpage">507</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2011.457</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DStegle%26rft.auinit1%253DO.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D500%26rft.epage%253D507%26rft.atitle%253DUsing%2Bprobabilistic%2Bestimation%2Bof%2Bexpression%2Bresiduals%2B%2528PEER%2529%2Bto%2Bobtain%2Bincreased%2Bpower%2Band%2Binterpretability%2Bof%2Bgene%2Bexpression%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2011.457%26rft_id%253Dinfo%253Apmid%252F22343431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2011.457&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22343431&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F13.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.37" data-doi="10.3168/jds.2011-4256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsuruta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawlor</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multiple-trait genomic evaluation of linear type traits using genomic and phenotypic data in US Holsteins.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">4198</span>–<span class="cit-lpage">4204</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2011-4256</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DTsuruta%26rft.auinit1%253DS.%26rft.volume%253D94%26rft.issue%253D8%26rft.spage%253D4198%26rft.epage%253D4204%26rft.atitle%253DMultiple-trait%2Bgenomic%2Bevaluation%2Bof%2Blinear%2Btype%2Btraits%2Busing%2Bgenomic%2Band%2Bphenotypic%2Bdata%2Bin%2BUS%2BHolsteins.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2011-4256%26rft_id%253Dinfo%253Apmid%252F21787955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2011-4256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21787955&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F13.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293011700047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.38" data-doi="10.1007/s10586-013-0309-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Essen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsieh</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ames</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearce</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gokhale</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">DI-MMAP—a scalable memory-map runtime for out-of-core data-intensive applications.</span> <abbr class="cit-jnl-abbrev">Cluster Comput.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">28</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10586-013-0309-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCluster%2BComput.%26rft.volume%253D18%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10586-013-0309-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10586-013-0309-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-9.1.13.39" data-doi="10.1007/978-3-319-06486-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Intel math kernel library</span>, pp. <span class="cit-fpage">167</span>–<span class="cit-lpage">188</span> in <span class="cit-source">High-Performance Computing on the Intel Xeon Phi</span>, <span class="cit-publ-name">Springer International Publishing</span>, <span class="cit-publ-loc">Cham, Switzerland</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-319-06486-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHigh-Performance%2BComputing%2Bon%2Bthe%2BIntel%2BXeon%2BPhi%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-319-06486-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-319-06486-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-9.1.13.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-auth cit-collab">A high performance level-3 BLAS library for heterogeneous multi-GPU computing. Proceedings of the 2016 International Conference on Supercomputing 2016:20</span> <span class="cit-publ-name">ACM</span>, <span class="cit-publ-loc">New York, New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1145/2925426.2926256</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-9.1.13.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-auth cit-collab">HSPT: Practical implementation and efficient management of embedded shadow page tables for cross-ISA system virtual machines. Proceedings of the 11th ACM SIGPLAN/SIGOPS International Conference on Virtual Execution Environments</span>. <span class="cit-pub-date">2015</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">64</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1145/2731186.2731188</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.13.42" data-doi="10.1016/j.ajhg.2010.11.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">GCTA: a tool for genome-wide complex trait analysis.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">76</span>–<span class="cit-lpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2010.11.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D88%26rft.issue%253D1%26rft.spage%253D76%26rft.epage%253D82%26rft.atitle%253DGCTA%253A%2Ba%2Btool%2Bfor%2Bgenome-wide%2Bcomplex%2Btrait%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2010.11.011%26rft_id%253Dinfo%253Apmid%252F21167468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2010.11.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21167468&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F13.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/1/13.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/1/1" title="iProteinDB: An Integrative Database of Drosophila Post-translational Modifications" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/1/21" title="Deciphering Hybrid Larch Reaction Norms Using Random Regression" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/1"><span>Volume 9 Issue 1, January 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/1.cover-source.jpg?itok=uUntglLo" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/1/13" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FvACdToXCWG5WipRzfSe0_F8OzqVf3_aE_pO0cLZwZw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/software-and-data-resources" class="" data-icon-position="" data-hide-link-title="0">Software and Data Resources</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/1/13" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/1/13" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/1/13" class="active" data-icon-position="" data-hide-link-title="0">OCMA: Fast, Memory-Efficient Factorization of Prohibitively Large Relationship Matrices</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491784" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-pAs6aLpTqRawRVxM9DgbRrQVyFaG9jHYNESn0D0hzJY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/1/13.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/1/13" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-BYNSd-IAZXHqibYPTVLD7hvTf7R9oehobMBp0xm2ZZA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/1/13" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/1/13.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491784" id="node491784--2" data-pisa="ggg;9/1/13" data-pisa-master="ggg;g3.118.200908" data-apath="/ggg/9/1/13.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">OCMA: Fast, Memory-Efficient Factorization of Prohibitively Large Relationship Matrices</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhi Xiong</span>, <span class="highwire-citation-author" data-delta="1">Qingrun Zhang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3783-4781" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alexander Platt</span>, <span class="highwire-citation-author" data-delta="3">Wenyuan Liao</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4662-3177" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xinghua Shi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-5692-7129" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gustavo de los Campos</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-3795-1873" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Quan Long</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">13-19;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200908 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491784--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhi Xiong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Computer Science, Shantou University, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhi%2BXiong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiong%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhi%2BXiong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Qingrun Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Molecular Biology, University of Calgary, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Annie Charbonneau Cancer Institute, University of Calgary, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qingrun%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQingrun%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alexander Platt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Computational Genetics and Genomics, Temple University, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BPlatt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Platt%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexander%2BPlatt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3783-4781" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alexander Platt</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Wenyuan Liao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Mathematics and Statistics, University of Calgary, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenyuan%2BLiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liao%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWenyuan%2BLiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Xinghua Shi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Bioinformatics and Genomics, University of North Carolina at Charlotte, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinghua%2BShi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shi%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXinghua%2BShi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4662-3177" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xinghua Shi</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Epidemiology & Biostatistics, Statistics & Probability and Institute for Quantitative Health Science and Engineering, Michigan State University, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5692-7129" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gustavo de los Campos</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Quan Long </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Molecular Biology, University of Calgary, Canada</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Medical Genetics, University of Calgary, Canada</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Mathematics and Statistics, University of Calgary, Canada</div><div class='author-affiliation'><span class='nlm-sup'>***</span>Alberta Childrens Hospital Research Institute, University of Calgary, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Quan%2BLong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Long%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQuan%2BLong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3795-1873" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Quan Long</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:quan.long@ucalgary.ca" class="" data-icon-position="" data-hide-link-title="0">quan.long@ucalgary.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/1/13" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/1/13&amp;title=OCMA%3A%20Fast%2C%20Memory-Efficient%20Factorization%20of%20Prohibitively%20Large%20Relationship%20Matrices" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/1/13&amp;title=OCMA%3A%20Fast%2C%20Memory-Efficient%20Factorization%20of%20Prohibitively%20Large%20Relationship%20Matrices" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/1/13&amp;title=OCMA%3A%20Fast%2C%20Memory-Efficient%20Factorization%20of%20Prohibitively%20Large%20Relationship%20Matrices" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/1/13&amp;text=OCMA%3A%20Fast%2C%20Memory-Efficient%20Factorization%20of%20Prohibitively%20Large%20Relationship%20Matrices" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/1/13&amp;title=OCMA%3A%20Fast%2C%20Memory-Efficient%20Factorization%20of%20Prohibitively%20Large%20Relationship%20Matrices" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/1/13&amp;t=OCMA%3A%20Fast%2C%20Memory-Efficient%20Factorization%20of%20Prohibitively%20Large%20Relationship%20Matrices" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/1/13&amp;title=OCMA%3A%20Fast%2C%20Memory-Efficient%20Factorization%20of%20Prohibitively%20Large%20Relationship%20Matrices" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/1/13&amp;title=OCMA%3A%20Fast%2C%20Memory-Efficient%20Factorization%20of%20Prohibitively%20Large%20Relationship%20Matrices" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491784/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491784" id="node491784--3" data-pisa="ggg;9/1/13" data-pisa-master="ggg;g3.118.200908" data-apath="/ggg/9/1/13.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">OCMA: Fast, Memory-Efficient Factorization of Prohibitively Large Relationship Matrices</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhi Xiong</span>, <span class="highwire-citation-author" data-delta="1">Qingrun Zhang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3783-4781" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alexander Platt</span>, <span class="highwire-citation-author" data-delta="3">Wenyuan Liao</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4662-3177" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xinghua Shi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-5692-7129" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gustavo de los Campos</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-3795-1873" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Quan Long</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">13-19;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200908 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491784--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhi Xiong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Computer Science, Shantou University, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhi%2BXiong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiong%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhi%2BXiong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Qingrun Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Molecular Biology, University of Calgary, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Annie Charbonneau Cancer Institute, University of Calgary, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qingrun%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQingrun%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alexander Platt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Computational Genetics and Genomics, Temple University, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BPlatt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Platt%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexander%2BPlatt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3783-4781" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alexander Platt</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Wenyuan Liao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Mathematics and Statistics, University of Calgary, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenyuan%2BLiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liao%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWenyuan%2BLiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Xinghua Shi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Bioinformatics and Genomics, University of North Carolina at Charlotte, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinghua%2BShi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shi%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXinghua%2BShi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4662-3177" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xinghua Shi</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Epidemiology & Biostatistics, Statistics & Probability and Institute for Quantitative Health Science and Engineering, Michigan State University, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5692-7129" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gustavo de los Campos</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Quan Long </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Molecular Biology, University of Calgary, Canada</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Medical Genetics, University of Calgary, Canada</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Mathematics and Statistics, University of Calgary, Canada</div><div class='author-affiliation'><span class='nlm-sup'>***</span>Alberta Childrens Hospital Research Institute, University of Calgary, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Quan%2BLong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Long%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQuan%2BLong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3795-1873" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Quan Long</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:quan.long@ucalgary.ca" class="" data-icon-position="" data-hide-link-title="0">quan.long@ucalgary.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=13&amp;rft.epage=19&amp;rft.atitle=OCMA%3A%20Fast%2C%20Memory-Efficient%20Factorization%20of%20Prohibitively%20Large%20Relationship%20Matrices&amp;rft.volume=9&amp;rft.issue=1&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Xiong%2C+Zhi&amp;rft.au=Zhang%2C+Qingrun&amp;rft.au=Platt%2C+Alexander&amp;rft.au=Liao%2C+Wenyuan&amp;rft.au=Shi%2C+Xinghua&amp;rft.au=de+los+Campos%2C+Gustavo&amp;rft.au=Long%2C+Quan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491784/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491784/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491784/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491784/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491784/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491784/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491784/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491784/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491784/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491784/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491784/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491784/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496031" id="node496031" data-pisa="ggg;g3.120.401086v1" data-pisa-master="ggg;g3.120.401086" data-apath="/ggg/early/2020/03/09/g3.120.401086.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/09/g3.120.401086" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The UCSF Mouse Inventory Database Application, an Open Source Web App for Sharing Mutant Mice within a Research Community</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495860" id="node495860" data-pisa="ggg;10/3/925" data-pisa-master="ggg;g3.119.400983" data-apath="/ggg/10/3/925.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/925" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">An Automated Method To Predict Mouse Gene and Protein Sequences Using Variant Data</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495845" id="node495845" data-pisa="ggg;10/3/945" data-pisa-master="ggg;g3.119.400945" data-apath="/ggg/10/3/945.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/945" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Software%20and%20Data%20Resources" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Software and Data Resources</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/1/13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/1/13.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/1/13.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/1/13.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/1/13.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
