<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/1/117" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/1/117.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/1/117.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/1/117.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/1/117/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/1/117.atom" />
<meta name="HW.pisa" content="ggg;9/1/117" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Fine-Scale Resolution of Runs of Homozygosity Reveal Patterns of Inbreeding and Substantial Overlap with Recessive Disease Genotypes in Domestic Dogs" />
<meta name="DC.Identifier" content="10.1534/g3.118.200836" />
<meta name="DC.Date" content="2019-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Sams, Boyko. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7330151" />
<meta name="DC.Description" content="Inbreeding leaves distinct genomic traces, most notably long genomic tracts that are identical by descent and completely homozygous. These runs of homozygosity (ROH) can contribute to inbreeding depression if they contain deleterious variants that are fully or partially recessive. Several lines of evidence have been used to show that long (&gt; 5 megabase) ROH are disproportionately likely to harbor deleterious variation, but the extent to which long vs. short tracts contribute to autozygosity at loci known to be deleterious and recessive has not been studied. In domestic dogs, nearly 200 mutations are known to cause recessive diseases, most of which can be efficiently assayed using SNP arrays. By examining genome-wide data from over 200,000 markers, including 150 recessive disease variants, we built high-resolution ROH density maps for nearly 2,500 dogs, recording ROH down to 500 kilobases. We observed over 678 homozygous deleterious recessive genotypes in the panel across 29 loci, 90% of which overlapped with ROH inferred by GERMLINE. Although most of these genotypes were contained in ROH over 5 Mb in length, 14% were contained in short (0.5 - 2.5 megabase) tracts, a significant enrichment compared to the genetic background, suggesting that even short tracts are useful for computing inbreeding metrics like the coefficient of inbreeding estimated from ROH ( FROH ). In our dataset, FROH differed significantly both within and among dog breeds. All breeds harbored some regions of reduced genetic diversity due to drift or selective sweeps, but the degree of inbreeding and the proportion of inbreeding caused by short vs. long tracts differed between breeds, reflecting their different population histories. Although only available for a few species, large genome-wide datasets including recessive disease variants hold particular promise not only for disentangling the genetic architecture of inbreeding depression, but also evaluating and improving upon current approaches for detecting ROH." />
<meta name="DC.Contributor" content="Aaron J. Sams" />
<meta name="DC.Contributor" content="Adam R. Boyko" />
<meta name="article:published_time" content="2019-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Fine-Scale Resolution of Runs of Homozygosity Reveal Patterns of Inbreeding and Substantial Overlap with Recessive Disease Genotypes in Domestic Dogs" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Inbreeding leaves distinct genomic traces, most notably long genomic tracts that are identical by descent and completely homozygous. These runs of homozygosity (ROH) can contribute to inbreeding depression if they contain deleterious variants that are fully or partially recessive. Several lines of evidence have been used to show that long (&amp;gt; 5 megabase) ROH are disproportionately likely to harbor deleterious variation, but the extent to which long &lt;i&gt;vs.&lt;/i&gt; short tracts contribute to autozygosity at loci known to be deleterious and recessive has not been studied. In domestic dogs, nearly 200 mutations are known to cause recessive diseases, most of which can be efficiently assayed using SNP arrays. By examining genome-wide data from over 200,000 markers, including 150 recessive disease variants, we built high-resolution ROH density maps for nearly 2,500 dogs, recording ROH down to 500 kilobases. We observed over 678 homozygous deleterious recessive genotypes in the panel across 29 loci, 90% of which overlapped with ROH inferred by GERMLINE. Although most of these genotypes were contained in ROH over 5 Mb in length, 14% were contained in short (0.5 - 2.5 megabase) tracts, a significant enrichment compared to the genetic background, suggesting that even short tracts are useful for computing inbreeding metrics like the coefficient of inbreeding estimated from ROH (&lt;i&gt;F&lt;sub&gt;ROH&lt;/sub&gt;&lt;/i&gt;). In our dataset, &lt;i&gt;F&lt;sub&gt;ROH&lt;/sub&gt;&lt;/i&gt; differed significantly both within and among dog breeds. All breeds harbored some regions of reduced genetic diversity due to drift or selective sweeps, but the degree of inbreeding and the proportion of inbreeding caused by short &lt;i&gt;vs.&lt;/i&gt; long tracts differed between breeds, reflecting their different population histories. Although only available for a few species, large genome-wide datasets including recessive disease variants hold particular promise not only for disentangling the genetic architecture of inbreeding depression, but also evaluating and improving upon current approaches for detecting ROH.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="ggg;9/1/117" />
<meta name="citation_id" content="9/1/117" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/1/117" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/1/117.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/1/117.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/1/117.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200836" />
<meta name="citation_pmid" content="30429214" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="117" />
<meta name="citation_lastpage" content="123" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Aaron J. Sams" />
<meta name="citation_author_institution" content="Embark Veterinary, Inc., Boston, Massachusetts 02111" />
<meta name="citation_author_email" content="asams@embarkvet.com" />
<meta name="citation_author_email" content="adam@embarkvet.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3800-4459" />
<meta name="citation_author" content="Adam R. Boyko" />
<meta name="citation_author_institution" content="Embark Veterinary, Inc., Boston, Massachusetts 02111" />
<meta name="citation_author_institution" content="Department of Biomedical Sciences, College of Veterinary Medicine, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_email" content="asams@embarkvet.com" />
<meta name="citation_author_email" content="adam@embarkvet.com" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=W. Amos;citation_author=J. W. Wilmer;citation_author=K. Fullard;citation_author=T. M. Burg;citation_author=J. P. Croxall;citation_title=The influence of parental relatedness on reproductive success;citation_pages=2021-2027;citation_volume=268;citation_year=2001;citation_issue=1480;citation_pmid=11571049;citation_doi=10.1098/rspb.2001.1751" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Awano;citation_author=G. S. Johnson;citation_author=C. M. Wade;citation_author=M. L. Katz;citation_author=G. C. Johnson;citation_title=Genome-wide association analysis reveals a SOD1 mutation in canine degenerative myelopathy that resembles amyotrophic lateral sclerosis.;citation_volume=106;citation_year=2009;citation_pmid=19188595;citation_doi=10.1073/pnas.0812297106" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=F. Balloux;citation_author=W. Amos;citation_author=T. Coulson;citation_title=Does heterozygosity estimate inbreeding in real populations?;citation_pages=3021-3031;citation_volume=13;citation_year=2004;citation_issue=10;citation_pmid=15367117;citation_doi=10.1111/j.1365-294X.2004.02318.x" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=A. R. Boyko;citation_title=The domestic dog: man's best friend in the genomic era.;citation_pages=216-216;citation_volume=12;citation_year=2011;citation_issue=2;citation_pmid=21338479;citation_doi=10.1186/gb-2011-12-2-216" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. G. Cassell;citation_author=V. Adamec;citation_author=R. E. Pearson;citation_title=Effect of Incomplete Pedigrees on Estimates of Inbreeding and Inbreeding Depression for Days to First Service and Summit Milk Yield in Holsteins and Jerseys;citation_pages=2967-2976;citation_volume=86;citation_year=2003;citation_issue=9;citation_pmid=14507033;citation_doi=10.3168/jds.S0022-0302(03)73894-6" />
<meta name="citation_reference" content="citation_journal_title=GigaSci;citation_author=C. C. Chang;citation_author=C. C. Chow;citation_author=L. C. Tellier;citation_author=S. Vattikuti;citation_author=S. M. Purcell;citation_title=Second-generation PLINK: rising to the challenge of larger and richer datasets.;citation_pages=7;citation_volume=4;citation_year=2015;citation_pmid=25722852;citation_doi=10.1186/s13742-015-0047-8" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=D. Charlesworth;citation_author=J. H. Willis;citation_title=The genetics of inbreeding depression.;citation_pages=783-796;citation_volume=10;citation_year=2009;citation_issue=11;citation_pmid=19834483;citation_doi=10.1038/nrg2664" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=D. W. Coltman;citation_author=W. D. Bowen;citation_author=J. M. Wright;citation_title=Birth weight and neonatal survival of harbour seal pups are positively correlated with genetic variation measured by microsatellites;citation_pages=803-809;citation_volume=265;citation_year=1998;citation_issue=1398;citation_pmid=9628038;citation_doi=10.1098/rspb.1998.0363" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=T. N. Coulson;citation_author=J. M. Pemberton;citation_author=S. D. Albon;citation_author=M. Beaumont;citation_author=T. C. Marshall;citation_title=Microsatellites reveal heterosis in red deer;citation_pages=489-495;citation_volume=265;citation_year=1998;citation_issue=1395;citation_pmid=9569667;citation_doi=10.1098/rspb.1998.0321" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=Y. D. DeWoody;citation_author=J. A. DeWoody;citation_title=On the Estimation of Genome-wide Heterozygosity Using Molecular Markers.;citation_volume=96;citation_year=2005;citation_pmid=15618305;citation_doi=10.1093/jhered/esi017" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Gusev;citation_author=J. K. Lowe;citation_author=M. Stoffel;citation_author=M. J. Daly;citation_author=D. Altshuler;citation_title=Whole population, genome-wide mapping of hidden relatedness.;citation_volume=19;citation_year=2009;citation_pmid=18971310;citation_doi=10.1101/gr.081398.108" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=B. S. Weir;citation_title=Variation in actual relationship as a consequence of Mendelian sampling and linkage.;citation_pages=47-64;citation_volume=93;citation_year=2011;citation_issue=1;citation_pmid=21226974;citation_doi=10.1017/S0016672310000480" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=D. P. Howrigan;citation_author=M. A. Simonson;citation_author=M. C. Keller;citation_title=Detecting autozygosity through runs of homozygosity: a comparison of three autozygosity detection algorithms.;citation_pages=460-460;citation_volume=12;citation_year=2011;citation_pmid=21943305;citation_doi=10.1186/1471-2164-12-460" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. C. Keller;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Quantification of Inbreeding Due to Distant Ancestors and Its Detection Using Dense Single Nucleotide Polymorphism Data.;citation_volume=189;citation_year=2011;citation_pmid=21705750;citation_doi=10.1534/genetics.111.130922" />
<meta name="citation_reference" content="citation_journal_title=Hum. Genet.;citation_author=C. S. Ku;citation_author=N. Naidoo;citation_author=S. M. Teo;citation_author=Y. Pawitan;citation_title=Regions of homozygosity and their impact on complex diseases and traits.;citation_pages=1-15;citation_volume=129;citation_year=2010;citation_pmid=21104274;citation_doi=10.1007/s00439-010-0920-6" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Lencz;citation_author=C. Lambert;citation_author=P. DeRosse;citation_author=K. E. Burdick;citation_author=T. V. Morgan;citation_title=Runs of homozygosity reveal highly penetrant recessive loci in schizophrenia.;citation_volume=104;citation_year=2007;citation_pmid=18077426;citation_doi=10.1073/pnas.0710021104" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=P. R. Loh;citation_author=P. Danecek;citation_author=P. F. Palamara;citation_author=C. Fuchsberger;citation_author=Y. A. Reshef;citation_title=Reference-based phasing using the Haplotype Reference Consortium panel.;citation_pages=1443-1448;citation_volume=48;citation_year=2016;citation_pmid=27694958;citation_doi=10.1038/ng.3679" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=R. McQuillan;citation_author=A.-L. Leutenegger;citation_author=R. Abdel-Rahman;citation_author=C. S. Franklin;citation_author=M. Pericic;citation_title=Runs of homozygosity in European populations.;citation_pages=359-372;citation_volume=83;citation_year=2008;citation_issue=3;citation_pmid=18760389;citation_doi=10.1016/j.ajhg.2008.08.007" />
<meta name="citation_reference" content="citation_journal_title=Poljoprivreda (Osijek);citation_author=G. Mészáros;citation_author=M. P. Stückler;citation_author=M. Ferenčaković;citation_author=J. Sölkner;citation_title=GENOMIC BACKGROUND OF ENTROPION IN FLECKVIEH CATTLE.;citation_pages=48-51;citation_volume=21;citation_year=2015;citation_doi=10.18047/poljo.21.1.sup.10" />
<meta name="citation_reference" content="citation_journal_title=Neurogenetics;citation_journal_abbrev=Neurogenetics;citation_author=M. A. Nalls;citation_author=R. J. Guerreiro;citation_author=J. Simon-Sanchez;citation_author=J. T. Bras;citation_author=B. J. Traynor;citation_title=Extended tracts of homozygosity identify novel candidate genes associated with late-onset Alzheimer's disease.;citation_pages=183-190;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19271249;citation_doi=10.1007/s10048-009-0182-4" />
<meta name="citation_reference" content="citation_journal_title=Canine Genet. Epidemiol.;citation_author=N. C. Pedersen;citation_author=A. S. Pooch;citation_author=H. Liu;citation_title=A genetic assessment of the English bulldog.;citation_pages=6;citation_volume=3;citation_year=2016;citation_doi=10.1186/s40575-016-0036-y" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=E. Peripolli;citation_author=D. P. Munari;citation_author=M. V. G. B. Silva;citation_author=A. L. F. Lima;citation_author=R. Irgang;citation_title=Runs of homozygosity: current knowledge and applications in livestock.;citation_pages=255-271;citation_volume=48;citation_year=2016;citation_doi=10.1111/age.12526" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. R. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=J. Slate;citation_author=J. M. Pemberton;citation_title=Comparing molecular measures for detecting inbreeding depression.;citation_pages=20-31;citation_volume=15;citation_year=2002;citation_doi=10.1046/j.1420-9101.2002.00373.x" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=J. Slate;citation_author=P. David;citation_author=K. G. Dodds;citation_author=B. A. Veenvliet;citation_author=B. C. Glass;citation_title=Understanding the relationship between the inbreeding coefficient and multilocus heterozygosity: theoretical expectations and empirical data.;citation_pages=255-265;citation_volume=93;citation_year=2004;citation_issue=3;citation_pmid=15254488;citation_doi=10.1038/sj.hdy.6800485" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=Z. A. Szpiech;citation_author=J. Xu;citation_author=T. J. Pemberton;citation_author=W. Peng;citation_author=S. Zöllner;citation_title=LONG RUNS OF HOMOZYGOSITY ARE ENRICHED FOR DELETERIOUS VARIATION.;citation_pages=90-102;citation_volume=93;citation_year=2013;citation_pmid=23746547;citation_doi=10.1016/j.ajhg.2013.05.003" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Vaysse;citation_author=A. Ratnakumar;citation_author=T. Derrien;citation_author=E. Axelsson;citation_author=G. Rosengren Pielberg;citation_title=Identification of genomic regions associated with phenotypic variation between dog breeds using selection mapping.;citation_pages=e1002316-e1002316;citation_volume=7;citation_year=2011;citation_issue=10;citation_pmid=22022279;citation_doi=10.1371/journal.pgen.1002316" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=S. Wright;citation_title=Coefficients of Inbreeding and Relationship.;citation_pages=330-338;citation_volume=56;citation_year=1922;citation_doi=10.1086/279872" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Q. Zhang;citation_author=B. Guldbrandtsen;citation_author=M. Bosse;citation_author=M. S. Lund;citation_author=G. Sahana;citation_title=Runs of homozygosity and distribution of functional variants in the cattle genome.;citation_pages=542;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-1715-x" />
<meta name="citation_reference" content="Online Mendelian Inheritance in Animals, OMIA. Faculty of Veterinary Science, University of Sydney, January, 1 2018 World Wide Web URL: http://omia.angis.org.au/" />
<meta name="citation_reference" content="Online Mendelian Inheritance in Man, OMIM. McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University (Baltimore, MD), January, 1 2018. World Wide Web URL: https://omim.org/" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Fine-Scale Resolution of Runs of Homozygosity Reveal Patterns of Inbreeding and Substantial Overlap with Recessive Disease Genotypes in Domestic Dogs" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1.cover-source.jpg" />
<meta name="twitter:description" content="Inbreeding leaves distinct genomic traces, most notably long genomic tracts that are identical by descent and completely homozygous. These runs of homozygosity (ROH) can contribute to inbreeding depression if they contain deleterious variants that are fully or partially recessive. Several lines of evidence have been used to show that long (&gt; 5 megabase) ROH are disproportionately likely to harbor deleterious variation, but the extent to which long vs. short tracts contribute to autozygosity at loci known to be deleterious and recessive has not been studied. In domestic dogs, nearly 200 mutations are known to cause recessive diseases, most of which can be efficiently assayed using SNP arrays. By examining genome-wide data from over 200,000 markers, including 150 recessive disease variants, we built high-resolution ROH density maps for nearly 2,500 dogs, recording ROH down to 500 kilobases. We observed over 678 homozygous deleterious recessive genotypes in the panel across 29 loci, 90% of which overlapped with ROH inferred by GERMLINE. Although most of these genotypes were contained in ROH over 5 Mb in length, 14% were contained in short (0.5 - 2.5 megabase) tracts, a significant enrichment compared to the genetic background, suggesting that even short tracts are useful for computing inbreeding metrics like the coefficient of inbreeding estimated from ROH ( FROH ). In our dataset, FROH differed significantly both within and among dog breeds. All breeds harbored some regions of reduced genetic diversity due to drift or selective sweeps, but the degree of inbreeding and the proportion of inbreeding caused by short vs. long tracts differed between breeds, reflecting their different population histories. Although only available for a few species, large genome-wide datasets including recessive disease variants hold particular promise not only for disentangling the genetic architecture of inbreeding depression, but also evaluating and improving upon current approaches for detecting ROH." />
<meta name="og-title" property="og:title" content="Fine-Scale Resolution of Runs of Homozygosity Reveal Patterns of Inbreeding and Substantial Overlap with Recessive Disease Genotypes in Domestic Dogs" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/1/117" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Inbreeding leaves distinct genomic traces, most notably long genomic tracts that are identical by descent and completely homozygous. These runs of homozygosity (ROH) can contribute to inbreeding depression if they contain deleterious variants that are fully or partially recessive. Several lines of evidence have been used to show that long (&gt; 5 megabase) ROH are disproportionately likely to harbor deleterious variation, but the extent to which long vs. short tracts contribute to autozygosity at loci known to be deleterious and recessive has not been studied. In domestic dogs, nearly 200 mutations are known to cause recessive diseases, most of which can be efficiently assayed using SNP arrays. By examining genome-wide data from over 200,000 markers, including 150 recessive disease variants, we built high-resolution ROH density maps for nearly 2,500 dogs, recording ROH down to 500 kilobases. We observed over 678 homozygous deleterious recessive genotypes in the panel across 29 loci, 90% of which overlapped with ROH inferred by GERMLINE. Although most of these genotypes were contained in ROH over 5 Mb in length, 14% were contained in short (0.5 - 2.5 megabase) tracts, a significant enrichment compared to the genetic background, suggesting that even short tracts are useful for computing inbreeding metrics like the coefficient of inbreeding estimated from ROH ( FROH ). In our dataset, FROH differed significantly both within and among dog breeds. All breeds harbored some regions of reduced genetic diversity due to drift or selective sweeps, but the degree of inbreeding and the proportion of inbreeding caused by short vs. long tracts differed between breeds, reflecting their different population histories. Although only available for a few species, large genome-wide datasets including recessive disease variants hold particular promise not only for disentangling the genetic architecture of inbreeding depression, but also evaluating and improving upon current approaches for detecting ROH." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1.cover-source.jpg" />
    <title>Fine-Scale Resolution of Runs of Homozygosity Reveal Patterns of Inbreeding and Substantial Overlap with Recessive Disease Genotypes in Domestic Dogs | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"2bM7w8Oy-IwizamEcGnrCtpJvgu9ysCgEzb46O_jAQU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491828","apath":"\/ggg\/9\/1\/117.atom","pisa":"ggg;9\/1\/117","ac":{"\/ggg\/9\/1\/117.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/1\/117.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/1\/117"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/1\/117"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/1\/117":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022I9onvwEZ5NBRDAteaCjhXZBcHHm0W3xRMUWfW4sbQmXI03SDxc922X8JiY4FPHL29pLwIfOpbesvkrn250Tm809y5qtZVFyfgeiUeaIn996xoEGWGqW2\\\\\\\/nsmdXJxNmWqB9ogSZmZLs\\\\\\\/dIB7mjWNzmS1rx1zNNXayBJWTUhyZVOCKahwa6KZN3NqI\\\\\\\/LvlaxWxOFf6F642tPYBgNVNRJmjz62t4T696MpVw09o4ldKXX\\\\\\\/MYChUJMLlvDS3NILvZAu0FMtAhPWqRGk4rsAaA7U6mdw2KE4uDV8okQS0WbA9bZNWnTkkXXP1JM09hXEXglUdoWBwRLAPO1s0tYd6+A\\\\\\\/CNlbQLQGPKxbjTZF\\\\\\\/JCtNjU5XIEdHWEw6YkNf8qTrazhUZvbiSUfT8nKEfezhPDwb+MXbG8fM6WhlpOWl0TyECS5RQIGYfym7+kyqBAKdX3ILVi7iYy0a1MS936TjqfqCSzCjzdWcbWMhMY8C5KklNJAc45CjSWXIpBUXoCFb\\\\\\\/poRQe4WSoBQabnmFgCj1LiMBIvXG7mHBgMNSo8FhZtyc6HYvzHTkkL7UK6JCE0ZNYtP2Rk1+ZFZZCUF++uOrbcMfO6S2EgmSPNDPmjTccLTJr1Fehzf3oF2vPXp3F6hVEnaTECHgLN0GzolVzYYhKnshewnIsroRapshXIwUJc6AAhgAFtDHcol0d7iPoQpMumO9TKKDhpkOHugvRPhWYpIPp0Xz\\\\\\\/p+40ylL7lUmAelOY0F1okOBiqCtRY9J+MQcrwXP86nBo95N\\\\\\\/olrmKzCYkXLZ07t4EzFVhTRn\\\\\\\/qezo+Wd7q6mPyVBpSdmW2zCSz+4wfpuCuCyhPtDaMpCjd5ZMy9JMRdSJT6cIKFvI\\\\\\\/JJ7RrzhWpneu\\\\\\\/3vDUd9CQwWe4y2IjNElqrT4TDt0HcBZBoe7fIMyHp7FL0zpA4O+YdHgoqR\\\\\\\/vZZxSj+jyUND8P47uSVoI\\\\\\\/uiVqs7p0XxH43hfECOlsUVgr7dvLpDk9Iwmq5Z8s+ChhYfOO9mAhgzkJ5lfekb5UoTx29OvL4erIE0DD\\\\\\\/XWIrS2vbkYLBprROAGU\\\\\\\/4rZH7Xku3Ig+Qu\\\\\\\/52TZiHPbu4yuPu2OfSejbVKD1Qrg9kdcluTaHwMBDM8QicCxAFDiKjxqxFmGgM2yylF2Ey9gQ4FLSz\\\\\\\/nMiDSuQWPb48ar7fuVuPwSSfMCkO1llwQh17bschEawyNabKC11Tig3Uha4dAM2sqUikJJB8qnMAziLlYf6TDcWdOeCoTAe\\\\\\\/E9pDO1IZGHX0shO6q8BM11dUWPE\\\u0022,\\\u0022iv\\\u0022:\\\u0022lgm89QdYBClYs1fgKmz1zQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227bfafafdae35e2d4a62b6eb3997823f90d7dce944c42884ef51d7e46aa5106aa\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222LvO2hodhJRqzOUd\\\\\\\/F6ekPuPUaNfvgF8TmO8YywxEHehKiMs3JZOXGhc4pPftYw23Mn\\\\\\\/h2pw9aLou2Oq2Vf2KgSsa31OPPqH6sSEaZtnn9Ion8EpaKJ8hMMTIHyVFuy+8K7FuXRPzarWcYXIYhMEZ0z4eKbs8FbeIvUxL4R6JSPZgJ6\\\\\\\/UjUCXD3Hi+JGpAlik6YkAW6+EKYjbhEjEsjMO4K9a5UMo8EUo+n1E3fr4d0FRLIAXOLs4itbMAncJpyx\\\\\\\/dKNgD8g\\\\\\\/CaFKjjSC5U2fsTY05qjAflDoiFI6z+P+rqN3TuT0U6rOrpsPR6DoYyCImha646FlipQneZ31Obz68KZH4JaOxQ9ddbPCJisGsPXnWvD79aT79sxjuFAn8Y+FLI3XKUC+\\\\\\\/h3f0sYGAxLJEtDKWlfbdCWQ0Emp0jmBlSEk\\\\\\\/ysNPrnjS7E3Dfgu50m7moSWkd8QRp98621meSodo3JufeJVq9wdzXtcmI51yZBj554jP03qWRrSxlJQ8VXQUqPQv60jfJZlbrKnS\\\\\\\/jUdHJJWvhYHxn7X5xyrNfuC5P7b4eJV4SVUZiS7w9FMzGPzJjxQhI\\\\\\\/2o50EzMt62w\\\\\\\/HgGsCbq2tkHd5BMn6rsMet\\\\\\\/pQYJYY8cFAdclLe4q7RsgtPERAjjgUftBq62xdeAEf44IG9mvECx\\\\\\\/MD3Cbik9pDIXJrMNWog+rB5habsBQpV9ZAq6MobYx6T6aoRZYS2qxYRIWXP4\\\\\\\/pcwSf+R5HnNS7dLpIe7Fr+WeEDPSuXSUz5RuPZvTkRiSYBCOlJABjy5XOgPK3zjL\\\\\\\/ppAmJUimLDBP1voOUt\\\\\\\/O+W6PumGoDMDphzEE4pqYrj+1bD6T0cMCJOshiL4c+eZx9aMQOdEmzizGvX2XSMLmCKUYsn+csDE5ob1wEC\\\\\\\/uNfm1cNL+l5HPqxyehHNCa6lOVosEmcm2IWpZ1kQBCJ2tIDQVBXnzcEkD3ueZRcZtLcpzmxzYepmmGR3qvbFLUoE1RbXAs9FNHv2JnqIdCQhPzchiCw412FEqtE\\\\\\\/Negyym2kZPOKfR6ZN\\\\\\\/PcLer5h3lCUXxE45mo9AgnfvsmxIeL9nehJ7ZnhqII9MxIv1hOJuQQSQwETXa6ElLjThOx0vkADJav+Wxb20gf7NRuMP9HG2lt9RRZok9d44D0xtCcjTJEscvgxt4pxGftrw4RS9hSRVybq8Sel+GHFva6PAKmhk6jI5fSr+rh5lzcjvVocJa+uFCmG1dA8GopxKTQwGL+QCzRl\\\\\\\/7P35Gf6ivCpNRxv07Flbb9XLx4qswN5siVJRY\\\\\\\/i\\\\\\\/n59+x1m8s8zoVCqsEaSCDD2\\\\\\\/6WmdhiR2Um9ocKZK3u9dDAE2HVtKqnaWY+oe1Kv3ahWiqGgDa+7\\\\\\\/Y\\\\\\\/fa6jXNUZrxpRk3VHH0V7u14LjkrNXQ+N76wnVTok+XHDCDXGcMbZRckODEUO\\\\\\\/H0KV3OfLqROg\\\\\\\/5CFSE4r7wRtQHMwI+wETo1Ty72XAcp4qZPvYw0RDj0Lv5Z6Qadn9fCENmXjmQWJW1CCZnxDwm74V2jfvYiG8TXnV\\\\\\\/4HVOmRTZEMO2iNsmRzQMJCbalZrf\\\\\\\/rUuo3\\\\\\\/gCsYKiqm9dgzBzYiBx+s4ZhRYeZtN6KwRwsWq6PEZMN+yH9KAPuSIFpeEe+ZMhrOBTwEYvq4Xup0fNQWrAsPvBrFYZ+J\\\\\\\/ylibkkCyVEtiKi3GHO797ZUJjvrHEBllvX0te71\\\\\\\/00+XVxr+o\\\\\\\/2vikOgrYkIpujdMtzX2Cb05oPxbOOxsm\\\\\\\/d1DbnOMVeM\\\\\\\/RL\\\\\\\/0CzG97uuwgN2yfbNZZZGaDdy5JcaYq76kqMWV4rfZjZ6xs86R\\\\\\\/7jPtUUJsfbcQ5Y1Y4UgvEvgtB1gWsK17FIiay9kLUvlzV7E4LSrvuL8t8TBNtqGY6QXzC5bMkk1gsLTxMUhdNSaNUukGSGZ5bVMwoQJeKs0kxL2xDLzTe7yOSx2vbSllYFmOaDnMLLVlRtMD9tF5Z6mnGBSGqigVYBN+Au4G56sR+krm96\\\\\\\/rDw9Ua0MhX5XGn4jTBeRb5fdA2S4eDVQQydHMWJ1QTWwGdBFn9m+crTOIQH8rXrPvRmSHyxAaZclR6FheLO8oKUAN+oWK3VZEIpetD9\\\\\\\/aaurT1lpOtSb31gmFF6sBn81f7lqGRBwO4VBDjoAYw3\\\\\\\/QCS2\\\\\\\/TUqPrU2K3P\\\\\\\/g9w7bI1jULAhyGaoz9IQ\\\\\\\/wSsDiXSKzE4plumCbOrfikW1tOJzUf3Zt7aHDwxryUbO\\\\\\\/QrOvvLJPqYBY76Ar3nnYvB81Y8C4UH2lDsulE2OJGcPz+V8mWQGg\\\\\\\/7r\\\\\\\/e+80etmtQGnhbrJdhbcizC+wMNhTiig2zQG0nKTw1JlOPhsBwHBXAKmFLOGnn\\\\\\\/dVBZZahJduX6fIxeaa3SVGdgayldPyNDsUI8xWcmczUfWuft8q64mk2bIGW5CJ50hq9XXYSWnP6DSXbmIcMbdcp8XyqCcW1k5Vi32ofFAwzDSz\\\\\\\/2emkyDI4wXbVWBaGL0e7eesnn\\\\\\\/+U9zUiej9oKVKs2g5034fCS9kpKMT\\\\\\\/ZKELiPGJVAxnxgqIVBbUgKojbRUr848HC\\\\\\\/R9EpAW0pt7exllMnbCkfMfPh39MuS4ongmpyNVXs\\\\\\\/XGVM0hiFjHooQ34Bu2SwMRca3CUW79NdgpLLE+g6Qu6q4Bujq4JMwp9\\\\\\\/bGkZJUY4h7QcFeR5K9sZeQI0Yx6aoI6j2VPqxRm1xoyeSVo\\\\\\\/5nvPFgbqwqMArDJY5KPp8+pbqM4E5Yi4FbRN0vNbvOlc7jvt1iss7PVOkT0zoEyuPIE0+k8Iz\\\\\\\/7fd55aLdZ5JXYTNtubpCF+siCBbMaOseZZJmqGgXWjGhGynLWPCHMW+YEe7TR9Jd3KjhHalujGacQUUygmTWekWWQtvrtMY6Wq2E4ArVHaGhiyw5aHgsYNspGn3Y5J3+ZnhIEEJiL4PgLE23OzYpalQ40mXI9Y8xJEioPHNMEFHhhgN3\\\\\\\/JCNmhsytja\\\\\\\/1xbYE37wYxHilJ4nrpwauqNLYDwhDcUSgUoZdpOT4\\\\\\\/S0qyliZI++81ExBzDvzCsQhimErvrRnc2a+HUbBy8Ep8DuXsW1LoECOhyvZtSKskMu6QyWFkBeBpT3s4qId6rqlxqYaIRqi4iulNFkgmfjNq5Z6OdFYW3oPOgwEW3B8owVADa3k6k07eL99rFwyAd1SdivXSzzx7ncec039a1MhHMNZwf0yC\\\\\\\/n35Si3LWNY7aporP0yzHY73UJ70nfUFXLw6cydgnuwOkekjhtre4TWztwzNgEDNz7McIxFVvarjNKgGFO9r1kjrZm2M4KlD\\\\\\\/GFnsA0+DRVOYpm4H2BcxLw+HXlXvHDk7+DfCP5bR8vWZhYn+JO5DagE38wMGXMNvqypYnCmHVyZ6QRBdJNjD26rALo7OjrciunWcOvH5JQ4LpQcPGGE3+v0b0+sHdcN44audIo5GvlnBUe6Dn8QHfiYGSHN9NDx\\\\\\\/vPH69xw2pq0d8HPJndkOaC117uYXy8n495v3eJAJlHgLSh5HjDFgdUs2bYc2L4XAGHYL3HFJnsT+7KExJ\\\\\\\/op3jYtOhTYnAS6jxZJZ+dd6JjSXYaFyahbWKrbwHNiRotgEKQxve2+kWu+IP10UVLcGWYh1DyexccIrV8qPeGOhOYkvPK\\\\\\\/tOpS6yiyrLOcK6hDUVYO6Sv1H8TkWnIUbZOFTvEqZ8x9FR7kp4S2yzqs+iQ1ymStNLeZdXjB5VK4WRrwO6\\\\\\\/yitoHw6rcMgRNkSRFoOMOD1zLrJCOMnAwWZ7ShQnh\\\\\\\/Ku4odHusj3ypVXW36ucgoirzp6ESry9oTFE9mGPyIdCJlFaU0Co+risjujCPapg04T3eu9vzR8kxjWXaHui0UWUz+qxiU9FAgyWgqX3Ggmr52Z6cABAOSjSX2xQkk49ilPowJD4OFBzKgPIRyGc+Vb5cDyXlt8gk+fUKzSADa2Isy\\\\\\\/BQe739kkk5mrbh52nIddnChg59p2uxZLbmZtpWeHakCHJKBs4MIHitogDkxPUCk0U2LADudZFzZ4rzeIyRvgL3RoVNYHWClP5Tln54xzcdiRtcYCBPBkrNiUCd\\\\\\\/ZD+mwBiA2+XwSp7DKSultERQUGt18OD7BGIoxpNCrb5bJq2dJmowIBlnomlotpiaq4Kg43NIrO71nSA\\\\\\\/lwQyfm+bdSn9w1qAgA\\\\\\\/Dl5nu5fowlgE26QH\\\\\\\/1qd3UuIi1tPyXA6VA5hb4ISgrRZYwndf\\\\\\\/Nuic2sw1TBi8GNQ3mvTZ6lTu4Fjj5icVESHWPJzWW6jGLxZbk5LZiCu\\\\\\\/F90hJn5Q6A+ot85B7XDGS\\\\\\\/4ZBvTqpJ7GDGnLFjG3yyWhLt00Yc69krg6+JP+C3m7M7Us6WHrLWCi7WuDQG0sV5fyXBFnjzx5nbJxVa+iRsW8KbCSgdmFt4mDMW3N71ZYTypEIlJW+5ld9LQC\\\\\\\/2pWsiBv6obwWkNqnDQaG0nIlw08JlHbchA2BQtvRIkbNFq5hZzXhlm+BxeY7NHxYVpm8pCWloR8llrDWcaDCWXlNUKriYIK8Ks7SO82Qxa6BA397x+TWnb3Pvtl25V3i31iAXLOFsrSeGAk7+NtqY\\\\\\\/vfg4XgSM0L9nsZoZKaf6nszrvzpdfYg1uYSi+09N+Kq1hd\\\\\\\/bPU2G+5l2D7qZ2LcYx+b4r\\\\\\\/AsgRGOCjOxtMjknWtJX0gsXva0oPbqTC7I4xCIen66AKhAqWJjxHLbafpA5Wl\\\\\\\/OiG5+mYXccD26uf7RPEa8jnNdJjJwH0m2z7qWxnqnc+T2TlRhBUfGCz\\\\\\\/JV2m+5x0DR8clSNDFTQaKYruOYCD5n9eG3Z436\\\\\\\/r238r9q3+YO3pP9av6QdNpdboLEBOIILikhURuCb4VSzIjVk8rj30MV15MsdW6C2BHXIavjipNgoUHu+6ioqnZq2UPWhuLwukoJVp05twRYZP6buV63Z0ve3IR50j1bA2xggVvJ55qtDvbabGetBJnd7rZgLTXQ4ZPBW3XjYKe743veJ\\\\\\\/W53VOW3DHIMvCJ8KuDOsZNl67bXDohYjcJ0y15xcwKYyHICZh\\\\\\\/ttvO8N6zx2CSS7X6Yo1ZNKNlOon7gJTnM\\\\\\\/xipczh1QEnFttKYDOqQ4eo2XmksNw5qAPSL9L4+6kP+NLQbXCNx60u9JogbDytN\\\\\\\/Auy10mZ+35KdHL8NSAc76JPZGMVPQhBX9CS2pCzYyAfYiSe319Y2lcfLNz0ERFUw60Je8M3Jo7AGrhMZRU79o\\\\\\\/yV3ns8zRaL6hclCTsx72d1pXPfMAnX3jdKW+okcBYGpnVmDIGn5PMl+IBs5Zu61KEFpTxsUKxDazBU+EKeVksOu3ldALS57WqwE1oad97U+Z\\\\\\\/TyP85\\\\\\\/piDcUPBiImkQfiwzCg6tXd\\\\\\\/EJQECqAiIDaW9copXUwK+roy0PpfI\\\\\\\/gTzZMnHTtLb1Sb086BqVHkzponPnW5mpYjmt03lQIDfitkjhjhw+RKbePcc9E4PmoRS2Zbyg8Ue6T2ncoAI1AaTO2GhHD2AOf9wmFAvia4vFv8pFLDLuHxc+vy2RHYTp1T1z6h1wueD7RpqYCjGiexaYWgay4bI0Ekf0mPA1uVWBGtRZcsWGuYRste2PTYCCS8P9KgvG7OYBtI03N9ZeKCikmYtQXVAZDQ9o2BmLldtEgi9zjJtVTMvmGfjS1an\\\\\\\/Kr3SZVZVrTh2W\\\\\\\/Mmf4fU66wrcM1XbdkQQJHgqRmuJpQaVGea7NF8ECVpqy99x8kVbR4md99Z2WQ2Dnvl+M0BVHAwVMGlQJEoBbyEx32de5cAlIBpzhjjInXBE\\\\\\\/tG7jM6InGLqWblaOkjLhmUPHdQWu\\\\\\\/wTEtZ9ACJq\\\\\\\/iLBNC\\\\\\\/NqQg60JvRCwn8YZqv6SXfC0kLxY+ZyAEuQY\\\\\\\/mWAWrj1BejQqxtXEB90csH6cZrENQYkjfhBIfO5XeynzgXBeNou6wdKgoCqIKtHVdO3E0fl47lCBwAIdiay+bhlJxqZgttCJAB+wDSovwsSc0xoXEsJRZnjCApDiRokORuQ3HoudJQyp4i3rxzGPoYr36eXli\\\\\\\/5Q23EluPPhZmO9AS4q5kcU6a0QfS4oipeJDXGaiS34Yq2oriltnu9PJ0L9TSnIgpWTF62vyRLJaFKwPfroAT8js1+2\\\\\\\/lr1Yp9dJXWLmXvu23DHye6ePcmHg79pad2mmEEEeMDFDwW9LIBWMb3UDxIvV1v7qciIPgz\\\\\\\/M+i55rumok4coO0L+uN9Of0AlNKyANhK7yZ9MThflF0hzVQojThKj64hX0XdiDcgd8So508AZUC6RMSYDi5ajWQw3eUGsN6QuN3e+95kM408T2APGpQquHT\\\\\\\/z1qPtn414Zmk7fstLr1d3amKXLWPNQ1O6dE0kWrnseQE+KO8W2HyCp09vv0uQ3FjLE8+9E3AUSBCHhC1bfnZEKBAqHOs3Ydn0QimgfbELRSnjByRbs5PUcyjVE5GvS\\\\\\\/YZgmtvBZlVMkpDw3JpEnJu9266T9wmon6NMsBGDUTCu0HAi3Dy9hwD33ibw5cOst0qJaOrrP0I4631Bib9Fvue5LzLkgNlJIERWAxlb+NuVTiM7kUq0hF0SB\\\\\\\/D7RDbt4NreYjbSGM9YGUEsRuEZkajcwPZziubnrKZgSa93m8x4Bn93Zyyb8iV3kAf8c05wpiUrsutuUYpRHT7LwoWTFVi0fHqGce4Vmrw48CChtvNFTNQ9Y4lEre\\\\\\\/QzT4e8\\\\\\\/cULnYAPy5NkZTdi9o3jT7HMyvTXKiyVcl1UbdRguTUjgyzLZulaNv6lKOSSJA4Ixg9gSNg1Bkp03YfotbnltPCSKYM0vEjctH+GexrNsoXq9Oyl4432HzyzPL6g+jvZYLexFdpLvZF+D3JTpwr6Mf1WQSfMmAFTSDKG9uVEN9FxqrvmSsCbNSlItqz9wWnl0A\\\\\\\/xPt\\\\\\\/qnzyomIH23It+z\\\\\\\/FueAVb6q24HN0LoxOPUFgMHWUQ4RlDWFaDN0pcoRyTiwEgyvBSnXbMcL4BvVdstpgf70X+QSTdbj4Okw1yIOx7jT4NaPNbJygDfzLImjx1PWrDPqvQA\\\\\\\/GmbdRn6rQCERd4WGkS4zej\\\\\\\/utDu0Lcm5waamveH\\\\\\\/Lpjnn\\\\\\\/TCSBKWGybRJZ2slfuTHHHbfWmf0B63nNCS0+4pQsKAus4n4NJmM991r89E2WEsLark0KJk\\\\\\\/ITpdnOLh2pc3VH2qbcGn9HL5k4HAdABN8OHrfkLUbD0KXIbK0tHkBg3+WhrxGag3lPZtC3GJ90afEhWRPmyI1sOLg7OxpIgzm2x43P2CF3c9UTi\\\\\\\/yWEhj0gxL4f8D8LYyhMzUIcV7yVpQz9NR+jSEYrQiWDsp2K4b5SCHG9RSMTMKvZiEfANA4xcrfO6QQF2W1LRsW9RFuH4vHDagbnGYe2v3Vt8kxHQ9U7pKhrSLRM4HmiGF2PeTxpiLszfzjToAI4k89drsfAWGNh0kbQn56uSuadXEMIaR+8Dx+\\\\\\\/htgQ0bEc8WXS1cnElS6KhzMVmnKzxU7tlLhMRXBThLhKhpXNDdmEdFUbwIKLoHpfCgE2ehxq+TVyFAEjhDlJPAfEQbe9JNWY+h7fUjvLYS46uOPi64pCpvirPuLyZWNDH4QwowbVWs+3BL1\\\\\\\/qKEyPSl3wksx5lXrVeoCBgv\\\\\\\/T4digL8g9C0YG50hdiYEMAhvo8EAreFtRVWUGqnhZg+esfa6Q+Kiy50a+RyKqk78tarKZn+HTCyppEuahLAqwgZRlE6GLS3V0+9Eky1V6aLfglEaCtN1X7AaE7Fc9W5zN127qKkM41\\\\\\\/HjlDcYAs5gWKV1V1yinb1x7ET\\\\\\\/b278Pjn6Gk8i\\\\\\\/pygfaVO8ZRPNwR9v5qvDo6U1fXJuW+hsCfPvbKktZVLoBJiTcQcOjtp5\\\\\\\/JZU2xis3HVfurNLO1F+h9VJHhGIrj1X8PKX8\\\\\\\/26Y3OfqUam6Ojrb+iUnpOfoR8bDBxmALXI0dsGP2KRw8DDH9DX906OwADHttEMGnUR4u4v3QBFj+h88s+SgF9NpRkcN1B\\\\\\\/b0YryHwBx754xtkcgeUCBWzI+K3du1oqajHnhIQ893ltGjrCBDmC\\\\\\\/z54lIJene6qPXFDWScVtL7XXidG2hGV+tsS++KFCldTFrv9mPhRr0qGrHRvA40PgNswr6cSdNDVWi7zSX6Xw2x99wO5hFke4sF9A6VrN3MDSrlJxOE5zEORZAK8byX7nrsSVkQQFXiOdZDW0uH801K7K4aqP+Aukjlk0A\\\\\\\/t1APSCyBAywfUfpK3KfBJxZQPKxzae2mCRamQUx4EpltJNEeD+gn+5HSMakLbOntGp1EchGtb6SAKWsj0y19oZyueedfD4H51Oj2e2JVcR2NbCZrQlQU9PNsitdPUJKTZORgItQU7e2NbN1E0GW4jk3rw3KxXvh0MUDMcC3AqAcXwNP1wGUXgmur\\\\\\\/Xrhf+PpbXVQGit+YCB\\\\\\\/nLyZ4lJKSfyCAH\\\\\\\/19O1Icc9Z4W3o785GZzssgcp8koLvxMi5CrDl52wXEdAKuaU3NqwDc\\\\\\\/0B65KNYzYzJODkDKBarEyNysc4ThbmshCSaMWCxmG\\\\\\\/2pD2LR8z+oEKgUDqEXxpB3hQ7eiQvTUd1pueF+8KI1kS1IoIbr\\\\\\\/zjZoCjPzS+cIBQPYkTuCS+ZhCfeDLZZdn8FdZW+lDHIFHQJU+w9z9ryiedHnqWya\\\\\\\/wPgEN2cSRHtMzmKlfcr6EblR465ammtBowwA2FnJwRSINZJMJUpDa5VpnEjn230MvthLukBRNIvJViV8fC3rCHlm5IQvQ5o3MkNJeE7cn0f4y8iLbb9eY5hSyU2bBVsDOYyb5FgfPXszg0+gUob0OF7I0Wr32P0LfqD741OmjmMs9\\\\\\\/Biz9hyZM4xfssiGksFm\\\\\\\/cQbflcJTZlqFJhYW\\\\\\\/\\\\\\\/AoIkYbyRht6Qw3OADPIb0Jr4r3fhHuvB8oH\\\\\\\/uhowDIQSVig9VX8aDh9PNGI26m+HlAWzlCcg2AsUZ7E33\\\\\\\/wSQ2QyJkJXm9\\\\\\\/gxIjsDKehuMlJbiZMyaFPvU4gLSsG6CVKS6Vcu19WRB0qS1XDx6dsMyIn0YBq8eGjCjiOZsAbIAVfK65eNZI4Eaj0hOoePHuoJdYeho4QfQUaRpwUs9uW+mMrJy2QBj+xlBe8H91S+0qIWyHDAIyC4rHWNip8HimJYo0I\\\\\\\/KSA0nOqarleWZdSr6LomFUvnAkbipJXZOBH5g8vw8\\\\\\\/4J8me19LcrYv4L3F0NDRD214T41otyxUtSHFOOUd+upQceqdJ1vKtMGPfQz\\\\\\\/FZv2e2tMijz\\\\\\\/LsvHyZD6FL519rTDr6TZT6W0LUCiJslwxVc2qoUdMmPc1xQzJ9BmSun3DFGRreKi1eMLXp3N4RIIrPocwh97+e3U3OHskv556xdzZlmnVVSMGGU8cnWOb\\\\\\\/WaagLG6gvZtriCv1t7CH0HGMSHcpzO6NS8KvslleIhwvoFyJ\\\\\\\/kcFDLVYPzPdvsGJ1cQpby3B281s16vu42Mqh7uDAjh\\\\\\\/CzojOM\\\\\\\/uEZDw3byW2+bfaQk7IX\\\\\\\/RksicgEziQWUNakr1n59BoKLEIqJj4kKMie6rynwV27OuTJIzKYfeNj2IdseXQTHmxH+JUA4iUQcjmwZuuafAE2CFFYqpXdpWam+aaOZ1e3JehC07JqOFxXgWGcwJQ3E8Fg3updMqEbHVRRUSDcF1JKhxd48JgRTAKShm0C8lL4WtY53Og9Rkp+LKeRp7SnP05gDW7sp+7EnZLV9yNFQfTqjI7qviirTFMnTNAIf\\\\\\\/5IhmG9G0tFpSf3p6a+RkVglT4e6TRYxZVip6XL89oIe4g+LwiCJAaTDMdoBoZYEpm4IWJRBxOyRdJeURpZrs2B+mHD00c6JtTgabnGwf6q+jDsoGenT0MOIdydxBCQ0w1q4NRr8CuRIdvaPU0Ii4hLg78VxOJnxb2Lv+UVj6XK1zY2ivr7gQp5oO5KeCbo8QDEX5GbtabDJrOaNrLViBtjrcrNcRBiW6WfSs4qHkJpMpaPsBmgvILz1A+pMmWyzIRtv25ydiqDMoIoj89KT810Br7rG3nXmKCWCq\\\\\\\/7zN6vYk7tuv\\\\\\\/Z5pmYAt6ZAHwCad0WAUHzOc9Ttkp5LH9zHsQXI5QDsYsz016R8oqLmJO6+UI5v7eLVJpuIMRv\\\\\\\/Z14ZjDr+EHqH6KU\\\\\\\/\\\\\\\/XRyEld+ZY+jQ6Zbyh+Ud2QtvjbsT2rAc9EMCCYF0hHvzOJSpw34E4dcHrH\\\\\\\/A\\\\\\\/pAQ0+qQhM8psB9z0bTK5Vozvnf1kU4k7wAbwrY\\\\\\\/pV1iWX1O9mbDc3kB2jrfcwJNfdRkTc9Tl97obLFZEfGUbGBI7oYdqJZQTrR5Z6Slowl4HAfRy\\\\\\\/\\\\\\\/YssUOm4zRmPg79E4PY7PpcYpARSvCMpavZaDFeCqpwutrO3u9PPTa3F2BjEcrfGqrpzlz9g2J2csZf8T75BjTynBIXfd9aLU+CbU1PUi\\\\\\\/7paokarUohDdCIcAPQwwhIPJtRXq40wLU9lLGUANnMS5uWNQugu9pRECpNquKnofZSLQE6eUxaItKBLtU3K4itGJq2b0kNgJ2nqTQKUPB\\\\\\\/uKpO8Xm6kTyKUYRghrkuwfYF\\\\\\\/mbG1suoh30knUVTu523HHUno4Tus58Sw72mveCAyXYcDh4a+l9IMI5qHtR03IchFHhgimAZoIURL8G6IfxDSg9zqzGirlXaMUB6OvlXzsj6QcNoVeNS+QYtyXghTLbKVhbPhDYvdz6Km1sx8\\\\\\\/4P\\\\\\\/O0WjN1mblGDd7mZ0nQKjcL5jyBsKTHeX4r4NBke5pwrFy6DvIkFV94ms\\\\\\\/KGn+t9HLioHXQf5MrE\\\\\\\/fOs8nW493fEFvcdMpb47tlcaA4+6UHGF4ZjowWD968a1\\\\\\\/Bur6EM2uG8zNpOGZbTgZimBd4GFA66id11hR6EyesC82Nf4\\\\\\\/ZN7jZZdNzuZ3pJBbprmPgj+LOhUOtH25ElMtu4UiPBJv3QDLkI1sJRCBedanryz6AI8gZWL66I\\\\\\\/Rpdv2VEckHIh7QHkFJsvMB5jluf1E\\\\\\\/vwdBEQbISfpbvq1nHbB0VYdHxAb4dRWHdvpl8tfWqBQTU7LTefZBB+1mpZNbRqjNY6SV2dZz9+w0YISbw82jLI9D1NkBQHoxUhVONcn4pPGyYtkXT0Bwrg9h807CqCEkWywAX4ONxczO0vQPfGC\\\\\\\/kAHIBHMviLZF+QeMQlQT0wdDyHKtruwKGHfIBbY9w9CICoqTuIEDfP68qRnpQcNVR10vlFvPDQm\\\\\\\/PC3HNyXg9Exh9lc0\\\\\\\/cQcE88MUVW7Y8XO1VdO+isQyCMQZK3fK1g6Jn3Ia98R0GQh5UULCQGkF+2SDahng7kEG0fpQgzvD09iZ9O7kTjOSBo7JxFLvfBHeSTgfCIzpHJhDyK1uDy6k3NvQAT4dIMVZBkLZ7g5LJmCzBz7jKTZrzuyq1DnnQvLnfLR+xTAESEFuG\\\\\\\/XQzxI4XTWOo+vaeWlnPOFnRHpMYCl45lHLrZTSaHvdwTtP1GrJUUeOyzpQ6iIDPUwWdOC1Ecjv8SvHuVsxq61dZnitB4pjW0hjIJ3ZBcRN2WhCDSyq3f+oNPiRlb\\\\\\\/b9OpeUgxjwytPESyGKTSpE\\\\\\\/rQ4LYxHTCj6ic4i2SgYiuc\\\\\\\/16bSLnc\\\\\\\/E+9ZJEXqt\\\\\\\/\\\\\\\/lQGz6u+l36CFVTyWPiDZe2vCbaPRcZcZM6wsyBlZfA2lnpE3o26mBQ9LeUCCS91bHmGh\\\\\\\/X1ONdkgARu+bphk6k15OIFER\\\\\\\/mA90mb7S7F7xLhTT07OvgiboaEsbWTpPV29UQ9y6s7r5+LhUWMUKtgPH0HkWrR+mT0bAePL1kyvhxHnQZ5aYQf1MSv0NVznj\\\\\\\/YEJErLr1RI4kaJ7AuVyz1iCDj14++OODdG0jQKloHXIoD5GZXQQ829jeSo0SvrUlmPLwiruImwyuzlH7BkdR95A4LeL2vMpMd3GYMFT69wQh13ooGypqrDG+MVmh+idPYSLYmMMRV7cT43jop4iFhPJnK+jEWuypQ80FW5sLp3VC3F9yjPC10ggU2ZlNXRTAgl9uU4eOn6MmIfV4LM0AkJsM9hVxZsH3eDjJ+WU+Rke8qoP1ZvcROu+CekUt164Euny4LLBU9oWVs6ziO2nMkxJ3ADWTc\\\\\\\/T20qz8XTJro+zvybgR5PC5A7w0XW1fmkOT7Nw\\\\\\\/VYGY9sP4akT0Pt6K6XfE17haB\\\\\\\/5vJHb2Q9vGTY6RC0Q4AEtrGqlZuusA\\\\\\\/b5sLJI76bDmckUZRxVDWsfXBlsGJZhnvNbI9y92CdC+RUVDw3EpgQoiNJFSd2Y6hd5pmte\\\\\\\/8d1ZrfMzgnrauZ7bbPtcmhBXuz5XZigPuj1EV\\\\\\\/qTOb75ix+R6f6xh5bAmaDu+mYdX2wkZWTcRW6rLrhF2hWiqyhidoJ1OXdkdStZ4NY2DI3bov+TaqctGx9aV6bV4kDhYXfMoCZC6XWZ1+EVXQQHltWc\\\\\\\/8iELon+aEfElUSH9Zk42yyWTS1J8qXmY63ith0rHC5EGAkoOfjZxWqo\\\\\\\/o5BjE5znheYuGPQyt00dPJZCT7TPG\\\\\\\/qHS6lRCMzB\\\\\\\/bi3gxn4gSRZbylUiRSqcjmA9zAmLyjwP9SBxIknzrGuK6P9n5DETxi5QnFnmKozp+4UEnhozU2yR6TJh5cEcfqy2NYyQ7Uj82hvUUOFLMka3lo\\\\\\\/9dfggx09tlXnGdHRLoOJrqz1hM6EW5IbNJVVSugzjVpB2y1Z63tI+BeJ25tqmbIVslZiZfWN\\\\\\\/pl0M9I7W9\\\\\\\/bSng+wHSagk1rBbjyZntikDdgA5tvQqUPMFsdMYqNoja5VV7fQLouGxgzy9LUg+\\\\\\\/3\\\\\\\/M\\\\\\\/wBjQ3i3RTKi93VroLhwoLtfN2dB3\\\\\\\/jMl6N7VmgtMD8t+o7D6JQnl6dpcBRGnR6bXehioosVndRElSjs8donDquCWVYyTwQ\\\\\\\/XIbPvgHTDc7J6qzq4QqKKcLKZdS0PgTedzzgA5JfLSLLuanhwcLoPJY4hWDpgzg2ARd0KrxU2YNcj\\\\\\\/3obldzIEb\\\\\\\/fhToGkDXngW6W6pKrJDWV9DcwAky9R4TfVdL8tp4zc6lvualyZ3tovPIIWWLVIEj9SHNeDqV1MRRgFtzBa9bu9IHIP9hHfVvOQrtgB4fQSQbxbeQBu3wZBJn5x+y7JjquuegBuGuMkRgoBZ7AM8fAK4UXmlABkT3QwJZG5lO+0W0LtL7tj7eU8PSqOmapW336jn2wAtNYJLto78i+DbJDhKIV03zrKTV7HQgV+j++WSV7LxLLe1Z99TBDdPrlfmX7+\\\\\\\/DPDUVuSVK8RxOzyKRhaBvaVyOZ\\\\\\\/o0IbecyYw2HUGsyWNXX21z8xOa4ztD9QmIwEIAUbrz0kZLUEG\\\\\\\/SMLtQlhfXm5p7iYs0Nwd6V1eJ1juITBjp1\\\\\\\/3gJAhxJ2RriQaLjrONVsg9jGUIyHtD27HOwPnl+dH6N7+dokD+lL4z+BcIBKsLmf+Gha4WiADEoLt3LSJkhfOcILVOQ1fRjsRqAIv1u14Qt\\\\\\\/zJJ7gLW0El6qjk4onn\\\\\\\/UNstC6Ej4SDUf8aJP44F1g7m8mQKYda6qbg10DPA17QX\\\\\\\/+l70KutR1UBQYrzy8Id78PTTbOMyjUcIauwW06pTvFa2eSCNbypjDZ0c6p720ocT5aUBD+dak2Ms22FBItUMN\\\\\\\/TfEwASqXY0ENQm6nnOiOgsoMthbkiDW\\\\\\\/cPr+KEs8Kgk\\\\\\\/y5seerBSUQM6U6cWiqy7Dqp1vKy6kHPqOp9QozfRQ9Ic4gQ5vct5FuutEpfp7bz6rgzkI6uNF+LkIZhHEO+qLCXo6HA7kyBKsQOhL+\\\\\\\/CpNkcpf6nIVGip62q9l\\\\\\\/uD\\\\\\\/VvR6QcPvHzCUQkzkSHR3+0S3hkKQutqJUVT1To39wZAFAPRXe0NqdgDILMTI7k1Cm4Os4L8wWWiio\\\\\\\/3IlcaH+1lvs\\\\\\\/Pfhe3pdaUjnC08Gm6UZgShYCesXnTzmpYgshA\\\\\\\/L0VxXhqGVCevrt208zXSLLlecyDJPY0DmdxhH2bNqDe9mEORr5L1wARTPUkOtOzgf\\\\\\\/fpIkgP1SYoWBEJqtTqQfVy9d\\\\\\\/z3EnrmMqqab3FXK+CaBCvNoZzj1hV5nVu5qdg0FbVsVoZhBC3AxM0kNcfzXVrXowAZgsmyWrNb4wZymIpyY86TWmUz6C4BwUci8uMXIwu9Yy+G6GzE9RQcVYaCWLQxTsN78cQTyI8DPvG3CktpXHUA5R8bGvXZz7ubch9i2rMJkhVjEaB9XMDhJf+QEQllIbrGqQ10DCVcZo8N2TNgwTcKi425C+YDnkxXRIy8loXZ2+JD1gDja4XRFaLoZmToorQJUJqnqmI5Hg0gyEGeucb13RjfE6P7F1vUmtofKWZNXHK2qtWFcJ0laHi5UOr2608CLc0mOLy0W++WMZR8RzOPt0KNQhiMR9xuJvr4QeTlN8ea19sXPZtL3AYOQaeKljJ5l883pV\\\\\\\/hJgzFlwJjT7hgC6wCuuDsSB7LWnI\\\\\\\/6k6mM\\\\\\\/YaZRVgrL1tzgMzMf6s0ITDKq5tJxhRz4oxyytCY9q5\\\\\\\/6gvP61C1MhvbeKkGFVrBazYfVUlzM3vNnp\\\\\\\/IWHZVyTOGEiAzSzxYzoXODA\\\\\\\/9kA\\\\\\\/5gZ1HFYjL0hDHg0thu+NEs8tMqvEDFt8sHUd9jmiHU2YBnO3cIU9X\\\\\\\/E4j+LJAMdlPtDpg+VeI5O3ZnhUMZv5vZv59WYXtMRC\\\\\\\/HfAPUK4yyLHNYy23jOi7FoO\\\\\\\/ZjkUUdwXaHEJ4c\\\\\\\/uIsFZhstPTqt3JdPY3RRM5Hh3Byz8wkclK3P0vzFVRo5U4GHXATxWIgJt7jSrIJvyxQj4gnPAwZgolwM3CNiV\\\\\\\/tarhF8evTRt0qqVNPprvJi6b9WfANcjQVl62MbHhJMElEU66A0YUxooRc+dEy\\\\\\\/4EiSasLM5T99kyRxoKRMGP7f1XPlxXqvzZafYWvNqsaowwjUofqv8gXaJFw4ljM8\\\\\\\/nPng5Ip0YB4o57OPA2R\\\\\\\/koqp4dpdqJrFazY7GWmyMGipWfBGkK6o+kjwSP959CNWf9jOvaF9LE5ahqSt7Ad1qwUV3a10ICD+q45TSOiwdkhdBPljOMgIwz3fu78xK8cJH3p\\\\\\\/HE2iwWEBrcggIWjylzd51QqTYEe\\\\\\\/6qYIGAHQ0xxmmI7htmsQryxWRpZr0XCYnWxYqg5loRAP5mNcSgiJ3lPI17zo8U0\\\\\\\/7x5Unp8wXWwPCoZP0+kEUO41jbHbg4yPtdAPc5NoaYZWtCSr68QLQHO\\\\\\\/EL4qZ5CGE1WbV2JfZUTdpPjvUSSJY1IVY1Yaix9nscWgK7slfEZtBTe+9BtDgA42h774f4\\\\\\\/kcItf9XOUoN4h5vjHENJ1LvojN6AYr16oxezLW9TlN2WKVmbh0GtVs90CjeAa4xqJqzQr2\\\\\\\/2HsNHPbvmW8uDbIJIt3dNjUWofb5twdGndAhQ60qc0mOjlFB7zXAZLZg8dH0TVNPhrDFHrrNMDLEZ5TBfGG6eL8bLm30m0\\\\\\\/e9JMF7vDCa3sVdsdUHFJe6R8VFmb+9Jl76vrRMEXHl2Um2Y7\\\\\\\/pZP9cXTCFMtFP23\\\\\\\/gxTrYc\\\\\\\/AeW9ETHDX0Y1qAyqKWuKy\\\\\\\/7f2F7Xjx+5seQ2DrXZDS951VVEGymkeBjAqb0r6QuOlhzH2xIPq1krHFg0wpD98GJpTpvqehOwguoCgFt8wM+15PBUAsEk7CuVi\\\\\\\/jzcXZSvNVLwP6JidTw9txUrn2R\\\\\\\/suMI7T3lg1woMkaI0yR4qoOZtYOyeAhcdKcGEuEo+UB2cJvAco+rknh08Y9SWkv0pLdVE4LrttwtaRWMTsCU+ZkDMsORLaa\\\\\\\/XVo4EfGIf1pLmtLRTieB1BX18u0A33qDKEeXTtpumUOMC39rkBOTK2Vx3JWv2QvvIJ3HZ5iLjlQeEX8PzOQWLnywT8jaEv\\\\\\\/c4e0ULFLzk1eP0w3yFUfixJZjjOPne6BQJbVx3cFAsewyO3TTumWuxxTCwg7PaXfg9wBMgVc98rk+sulaOWXNV3nc2Jy68v4y62ngLKegFioxxrlZxm5EibZnvGeDtLreem2zpRe1aOuQsWKU25hglROAg3cEpx96DZP0K\\\\\\\/2xLmsVN+aIvfw630m4Rtt5CBP0XfL9mUxNptUVXipgyttQtT8Vn\\\\\\\/EwILE0IL5iGuQr9su8\\\\\\\/BmOACLVgLPwRpYO1sN6TXewXisnnQ4eh2idmTzK4lQJB7iE9OxTHO91kgN+qErLKsOyrOsL1n\\\\\\\/xS0y6DI6UOjyehhshscp6zVB5Tjn\\\\\\\/F6hnX\\\\\\\/hZMpBlHlBa99SWfkXB2MPGfH5aG7zVRNb3rxLHvj6tOgnqQQ6ydyP1dALdFrV4+lr4q8y7gVtPTQBHQEUu9aQwwXVG\\\\\\\/ckgAbG9QByK1zC8iWrnGeL1patSGZsk+A1j0CqHVMlJHbU2V6trVLzI7Q2OfrHut5SPhvsASNdKPy9Qj9\\\\\\\/5u5pDQGp9Yu8BA1zlikfe6z\\\\\\\/9FDDwynJHmkBkWr5LUKHGTZMTIaJFNgn+WE5H8dD7\\\\\\\/TJ4759JezeZyfPL9t7hw93qpqANaNcB3Oo7xHViVo0m3ZzxoEhhaMM4BNgvdrwz9\\\\\\\/kAcNPworhLhhepKPqf4dpbE02RTryye\\\\\\\/r6ic6YPv4gcEnk+Kbu2LsHloAJEZB7LakgMgNYtXyouAJwpxyE1oZEuJd5kAnR433Az1Ibdw2x6JEAjz1c3tMW8x+wq0z+MUNiTC14JDY7Ct0xxAcK\\\\\\\/UdO0ihYCYBUrV0KP9e4a4KKbhiKQEe9BKAByokQCj5eIad\\\\\\\/7a3xdUgwzBIIx1L71wNK8VFK8+NH2ntJ9WZqHS94Hk0wcA5Tf0nY+ztEqig+rKtX0lYgvRgPnENdB3wH2VKO1hlxzaY2\\\\\\\/KVHn6Hz+jAGBGI4GbeQTPQy3PpYqzA\\\\\\\/F+wY8oUEPS2ne53p69Wwb5X0W+idQ8\\\\\\\/4xV7l6OVK39brmqwjidVzi0XruUV3CcxBk\\\\\\\/tFy7Xjsk54Fyd7hWQgOJ1OR0duPASxJg4OrQYloEXqx+kVSIONeyssZF4x+dHW7JoOZlc1cNwUHakbGH9WmdUGH2g0N6Y2E4ZE4kQH\\\\\\\/xSk1hjCdXLE5xgUoWHKDCu+QePl\\\\\\\/lP1hYVo1W\\\\\\\/N2xXTCKTjLTVkftVL4uZxX4D5QWbLxeRDkufqbHzA\\\\\\\/hz\\\\\\\/dA5J+ql7ZNDxO63G6Vem3NpHSbVv0CbqFMnJoWBmJp5HpxNSntI24p+66\\\\\\\/C3G9q\\\\\\\/HpsgJe3UnVs9jxE2SeSD4HnIoTsWoOWsIE69Pd0vNi7fnPn19BYvNB31+TWIDPPZ4xaS+F9HjabxDxm9S8fCGfQtJeShB8PbaD2jisZiPYtrNx98PFacMYnr8vTwlq7qzEokTzWUeJ4zRrR9WT1g0H33JMgiGb4bZ7H4jHsqRxw1pNWRZSvFCc7L9ptY0PwU8v+R31eLfc+ifeRzQRjraxta4xV30A\\\\\\\/bTxcx2mxElq609Kmn\\\\\\\/ZOrlsk8sLCqaGrwC8Y+GS+hWvECSOC9xkgtuh7vpnEluGtC266zyC4d3fKY+F2ifD6T58c1HfBBTVsjeLOaFpm9z9hH9UOpunsrG0+qN0lLgGs+Wtwd0he3CzLJcSdxxp1oMB3GFgAjhgYduYds2vo6P03YI2Ht2r\\\\\\\/qHsuAo5mRTBD3tPFN3p1vctO1oC3Fzc5UgnymD+QzAt1TLeQ+10lUSvTZ2JAxpzh6nIoUZq6AJzYRA90\\\\\\\/AlI6OE5cxw52w0o109AU1DyuI0\\\\\\\/Wjjwot8X+Meph3\\\\\\\/n0lBbNgwqjQYv1zqvZlzOjfo\\\\\\\/UUwIyQ4glSJMwP4VzFnRnOt\\\\\\\/ojkS4pBDIcU5M8pLGNqQl5nhdF3gqR4nLHcF3+LUn+QeGV+jsthC2lhWiHWHIld9u+hDU5eggcXqck\\\\\\\/Cq77YPgR++OVGm\\\\\\\/GIjYU+GGyRI\\\\\\\/0uUE9kzr1ayZsqnoJVNqFCq5ofjTNWW3NTMhGeX\\\\\\\/YyghomfpdtZwhmAS9pWTHqZV0wX4OLjTUIBx0SKQBqgaaXWRk8jBB53U9BnyZJf+5mpwB5IS+oaHjndsn79Ycw8UafQWgKNBTQm7A+Sz8YL7V1MC9z01+EF2Cn93YAWWckad3r022ukG2apdBWrkmj1Y9yGilofVjgOOboNifPhpn6DjlB7tOYNdEoSqYF8Pwyzjgf9g+5oZpkopdqztLjFEkmy8mI0O9vypWt5LxSjYatBDLoNMlLHXRFORp\\\\\\\/3IXG1VYt\\\\\\\/Kub7SJIhYuGWH52mstjQvvur4u1OB42sqGiwGcmrgPs8lo969+gCk4eWYdpeEesqpvEz7A\\\\\\\/koj2qENP2vjQrVlL80fsddNIuNVT6IiYpcyKiJj1wRc+oTTbYnjPoXhs4RXzHrsav0R8pEVP9o8+mswCcnh7aA5mKCVzBdsbfPQMps2zMtWTwFwG5nDed7Ihxi7Y\\\\\\\/QQ9GalucUHkrL+nb5ysu\\\\\\\/8h6kwmTwm8nnWnb6jbJoL7QpZxvydHko7UFERPZDTjMicoFX7iRSSoUExI5DN8vOVhaROPd4P8eAm270TaszPdOaz29y2GaSomAu1xW3dASsjL4hdMZBqbh9XN+N\\\\\\\/ID2hWF4Pc\\\\\\\/JnhI\\\\\\\/jG91ELFmhraHWrPYcNAe3VbC+hjDkAFakg6szmUZy7ioDXhiawj\\\\\\\/F16FasBq4fsgdDrY+s\\\\\\\/r\\\\\\\/CZmh4A6bTxRFEhx+a56Wl8DPIHDvBFSGrOxFiRXOPcht7cg3OqkzLiUpkj+10zntLaYyITkrLV5Wr9IhIZHrgwlrNDfRVXpBzs+kW8\\\\\\\/wUaQkMb1hjtS2nRO3fPzxlEFbzj09f+sdg3xsmC6GWQbBpuhgJiRN6SBfV8UDW9TziXpyTTUyk4T4PwPUFx7TgIFdUs7AdCVFfq+UnSJqXKxFqv7AohjTjxuOkfz2NGO+7SZelhyJZFS5+QwjMwPyEqTUmWsOpVrFMVQtBcfzxDuXPMENl3+ZHKdV7EV5X5uyO9bAIBW6+EGHd3cTe42u4jijbiMrFWFb+FMCY\\\\\\\/ts\\\\\\\/xcwP69gdL91yZ1xqS1zGSbpo7ZJMRkR7Ck5OImHziS0egPR\\\\\\\/zHGp6sllxjk6\\\\\\\/0qgEGeJQVZMs\\\\\\\/XwG0pQGsVGteROGC7pqQH6bSpO+S5DaloUHy6kDmgxDP1x7FVJNBKqj96L5EXQT7r705t+\\\\\\\/wlqqLgfgHQRGN98zwRoIEOZsi5n3UTqrUp0gsJ0TrYmuiHooYiSL5YZtxGFnddXChoM56dZQGXkb\\\\\\\/X+C+2QhyukYgvdVM54d8fKYZW8PUcnpVygwjJFz+EkFVBr\\\\\\\/K9WACuFwIg2FAWsw6BDWicLuGvhYq7CzbHg7kSMSBqD1X8t1r3crHKOfbTpCnMh3cjUfrG0rdZpcFARE9wdU0hq+4QdmNdhQFJd6LihOPqXf4nbyI5zybUh3xgFhBUjXOz0w5uQntabm23U2dXmo8iNXzohZnIusxblZ7cDjY2a5Aj4TacZ3WIhc5se+NcwpXKT5ERHqllHwbzWL+E67TBfTEcnCGi7VktbOntoD2onSiHy8f2KlBNooxJpKrpH8dgTbTZ7Y54TX7L2UwuB\\\\\\\/DoqCncihuVqkoaZX2oYdxL2Oq+98Oc+HwkJ9r7A0iFR9oXeWptY1ORLyM\\\\\\\/sy8h3W6ePP5ynQ3HwKiigrb9icwRSWn8kyh1WBV2vfpe7TXqm5KNNKYO\\\\\\\/pdMxaMmGU2P2s6DZY2He2Kz3Dec1LtFFEdLuqQKr0Gx8vZk4a2Vdvc0JcBpFzL+sUYx0unQ6sf1GDl6gh\\\\\\\/S5iZ0VINAWkLSVOtRnk0aIQfIzyO2RhdwUQIIdWK4Lur35bTMnMyGeY94\\\\\\\/fIeTfS4\\\\\\\/CdsROajHwPhYsffjZLAr1AZLIsipEKnM8GFE6j1MsXtxqJW9tw0HY33pn\\\\\\\/i2RXvyx46InKmXvNVCd0sNmFUEp1Ft3eInQe1bMF7ogNYh2\\\\\\\/M0IK9q858B9SvdK6NWLIthrDYOKdIJta8HVVFQ7TUp4b\\\\\\\/oFyfU+teGxxz9EmEsOOI13xIVkxbwb17l+nQvZuzU7FdsbV6dJRQtLFN1ovEKhZb\\\\\\\/CuJGNLRox1wmqFlKNgdWKbdnAjyjQSbdlpYzVw4Po9KDaym4hsAgFyRe3Vy\\\\\\\/mlajZ0p+7VFGc+en4hmJk51yipNdwSlt3a2FFVh7UQO+AMhquxRSmA1IPBuDT94v8l0uuQ1NSq8YOx7LYXrflfPCmPZsknELL5p0\\\\\\\/5nKRnHpQgPb9J57zemFZk4fj4XxpzHbtiQa\\\\\\\/ngSnqfzDHQRvwM62xuHXHlXO7lRwaMTm9q2WMJCAI18cBNYFtOzrgaqFStj2eyzG6Qcqo6PT0dVrtDDQgv0oWBSB7IHLBsvVfsIhwHBvOXTDA29DGm3BdGr69pPKNt7IcRoRZyNT5YMJwwrliPK3a5T46DtYCROUt31f8GScuo0vx\\\\\\\/y2srVJP83flnNB\\\\\\\/LLIL\\\\\\\/s1c\\\\\\\/1g64MC3zSOd\\\\\\\/0cpGO0fDCCI+P5eDYmpUvSiUWkYyPh8QUGzVlDTgt2\\\\\\\/Zf0xtb5DdpyeMF9mRPi9nnlqCQwxUxBGTRRnIDoJ3eYL46d\\\\\\\/ESW9QuzK7GyXJEVNkwPmXyjtjtYNVudXn5ysQUs0VWc3NKM1pdxpCoJnm7Ba+ryy8bSaVdlub9DY1kf\\\\\\\/MrtfDyjq5Jexw9eotFtpU\\\\\\\/+wENET2DRNe+xkQgV0EY7DWbvOXdnxtUUgo2vzPRlT7sSiS58luGjTPzMO\\\\\\\/zUQFkhUNnrqV4TuKvFzVvn5EjZ67nARD0jCRZWiFxBwRAmD3S+Nj1bGRWA59vYI\\\\\\\/bybJGioji0dM76UNiSCrSOxT1isC9l3qfv0P2VlSlzw6vXJJfCdIPNCRYt8fDCDY57BAa8DUO2+4ZqmOZ+zjIb1KCSKK0+HV7kj6cPJlg3raKxuWPsuOChc1GV52DW0Ty56re1te5yoSc94jf7Evu+drbvrSPDBbFBP6A\\\\\\\/0rIkOeXKcS83jY7mPB3yGVAAZYqAzxVGagDKF3eL9S\\\\\\\/tbOA0ixcpW\\\\\\\/hoH1qRlSYgBtGlOwL27BtQwOjk60+b9ULBpkqjVVy0RIN2A07OcGMEC6RFjKRSVxFzft9k2Hnicy3IQsARMn8ME94Pd1xDGIKErmImr2vzkXgNdySPD6Ic5wHmbEzrLVollHiCJBr417wVnEawQsksIAGPoaPQCqcB+VEiB9yL3brTWpsX46YqYX8wZdDxgFMcMb8daFScILY7IhcMH5W3E5eLAY5poOR29yvAPU9+P9nb2h1RhHOL2Okq8TgD4v+MyE86T0lcHGnAQ6iPRNkKFfI+izoza5hSU8WpkKAe8Dhd8C9nFHn9uO6TpbfeTEwCWmuYUPboQMAdbCE08LOzfUD2zNxThjwAhugheGdfN5a7oq9Hcz8qIK5iVUXJEEHc31AmsT7APMTlE319F1+DcIvKDYhS6vZinWRIFRzDip6CpOSJx1vttEGEFsPidwht4HZsnJCXb\\\\\\\/h5Gr1Hm8vDJ\\\\\\\/jfIx1mkPJCuu24NWLbVSQxGQESh4MiTU9Z9rH\\\\\\\/oB1b3ncLLCivxG3Sp4tfzAuMAbdFZ8Nh3+H1+IWJMtfHtEhDxErzn1aE2EJSfQT\\\\\\\/dJ6nANNRfnG\\\\\\\/tPmkHmRmx6DsYzHHIdgtqUyW4rDQKsXSHU6cI+tMBF8MXrmyzBL4rXiyAfqP7XI\\\\\\\/4XHyx7UxvB1Vm+Spggl0RbA90N1qp8wkkC64Mvy13ne\\\\\\\/q8vlq3PUrdPI7DB1fIS3xzM3vcg1Ws0eoYKbv7ih\\\\\\\/s6s0bMVrvMT0dNl0CiQYOqgr5iQbjMBEm0V0wpHViNxOAIZThHXluh2gRPW25eajuGqdTyi5asU6\\\\\\\/a8qzGo\\\\\\\/PsRE28EiyhIywZsb\\\\\\\/5OqWubFd+Yvaniwz\\\\\\\/vmB9Q6qPoh2XZm9c4+Cd6YT\\\\\\\/8k9ahMO1R9Tz8+7jx5AefsGf9RO28gks1\\\\\\\/j5OrSEKf7kfGIYYdrNkrVV0lR30siSW+67pMJo4Pple\\\\\\\/YLU6pqPjJ0k2597FLBEyRfNsuihOEVhEOQM6\\\\\\\/vWYjIhRXX0RBzCw4ajWa7zHFxp3fbl1Y6w+Yazsupd7GEuL2kXkRiGGLEBZtZZdOBMDtWnpPDUOMO7tT0079tRNHNkL8udZk7N2o2sVMLJcosHQA3rxJNz8oTuXiA\\\\\\\/7QOeXRO4LhJFttlBx07cW7rpcrYpHHZ3Ywbj+OFQ5j5tCL0LU1ILfwOfo7qQdHBqQBcVBSIyHkIPCs7DDrprxybKS6CfXj1zk0F6VzeKOlAI3yTdE74xQUUyVa6WJDnN2aPBOJTvf+CXZUcM2aIcfB\\\\\\\/4BtQGrTRw3ZoGUJatcqPKU9uoJAmvwTZsywLRBNy5QxRVwdLH0ZMQ0jTUZsp+WB6V0QDTyvWoi9oEJ22GU4RCT9B1WMYQ5eD6tGJ+T252pb+Fxu+J1lB5Im8Vp8uGJo8sJOUAdHsDB0PYmLJTgHtPlEeig4\\\\\\\/GKXL7e6kt+XbHD2fllYoddo7Ow4Qqvt+Z5bOz8YADLJVLdLVV+wBrF5qWZ3PQR0UKFL3MiPzg1F+QIikvb7zqu1LzpwNvzdmrRXBe2\\\\\\\/Zlsm\\\\\\\/hbV2TWywinMmUUv7o45WD+bWqup8c6SIUlwZ+yEysDU9pB2vh3lAb\\\\\\\/CSn3gtRJ0mFimEbtRM4lyoqttOJOgy90aDmmQCNhZOE0LLTkSrUpf7afknfTqybOM8z88COicT63GXz2WRR6HPGJkfsvpXUFct+YrJi8ssvmzn8M823haMW6FTVy10cRLserSk4s7FIFkeedFN+fY7lZyvEcncLGwXCkR1iOYdoaUhF2PYyidcfPLlkpQsf1CkYotlWfWcCa4J4Tws2N39DQXO2LDVUNKlLCiSJA5hkGr8FNmrCTQY0P2ZDqzkMEVohVqP2DpPM\\\\\\\/z9mZsGxmGmpN6oDBvbFZ9uneJvDxFxP+rk9PZ\\\\\\\/28aqtuW1QPmWUUvcfgQaHf1WqDywkUQiFeujI+iThxOVF0oZqD3C2bYBXmKlw61lXX9mtkKyqbwdYTJun87Z38js9hpKhVStFrDFxoQT54z92iKM\\\\\\\/ezonqbW8XFXHTDj2XDEVN0BhWsOvcQ3YgxQKLcULkYhhfI2MGLfIyiIqZK0xE6TsqMy4or68tz2RE2DlK7tnTZDsg2\\\\\\\/eulHX22XGX8DqpgSgtS4u0TF9eOh4pGn32773sNL3Ymaw+kj5YW90BwsMZ3wsvJyci8q7cA\\\\\\\/mloMYt4I1VygflQQhyZxv+qo8B+6rEovLAQbrLZCd1q3+bJfMROoXIZwDfFXDzhqWoWf9SHAQtrZKox0As\\\\\\\/jPrEoFWIzL1prRthgLaG4jFrQC9VQr1r7yVqK5KoJfpLlFTksVCN2RHLH+oW9s\\\\\\\/97DdJ2D6wsGv\\\\\\\/zTZZHH7G5\\\\\\\/w96WThIE8J\\\\\\\/jASH4ptIElHNMxl5um9qe+97tgjCmXmj7jhzfBGPfptue6AyG7o+eJ7wWkZIVQk6KDfOueSJVR3PgFlV4JOBKrdZl4Dq6SxY0cJuRqTFpUlhptSla2BEv3xsxQknlLKvfqiFHw5xzcUhEwxIMJ5kelfMPk\\\\\\\/FkAC6BSeVdVe\\\\\\\/LnIRTSXiqDuUUALhDfeEhrF7lJ+DevBwdTPb4rdLDR4N7iz+AUizxILzWx+tf0xZG+QdnFxrh2stn33F474XF1a5xQWsHbIbvq5T4CPip52j3aZeasRnrH3j7dCXx7c0mBut3JICub4XYbsnizNvXTSL7xs+HYmw7ou9FvX1VQQhjC5jSWX3OHY7Dt3P8u4r8Q9M62xJuVYy6EYwOsTnjxERoHMzb6viOvm2gLa9jfEcU2J0dEH31jCtekc+awnNXj6FuQAeafegwg\\\\\\\/yVNk0U16HYgBMxaV5HuH+v598GYvCNuhtimrAN\\\\\\\/7531ug80VXZpblo8csNis64omjdauj9znqYd3KxMEGKR732oToGIQihl7WRt9WSk2LH5p9F2E51qndVmqeJAHiSijoYQj86dK7QwS2RV+51gf+QtLRFWhi3WqXUE+IqiuujHjTk9WomfdrVdJNEe+HRISI7pC8hcP2CjZfalrL74D8AUkvV4kbgN6aujYYn5ThFmIfZT9zwE+nJsqVa00XlZT0Iz3rVbcMbQ1RPGs7PX\\\\\\\/1Cot55OVRo8No1U6QIppGmDYlZvJmZgvidcIjAHwqOmtEgIPtxVVZLXH6M1rssjQgw294kIfTB9FqBf4ORGMv+892hiRV8IBc3vBZzEpTE4pwZjkzQ10uYLecw1T\\\\\\\/AJHqfwSdU6c09fowW\\\\\\\/b4d7u82PwLII7HOLw6+4UOR\\\\\\\/kMdknYBVsH1DBwh\\\\\\\/B6ua\\\\\\\/tIm\\\\\\\/ISZMt6VwzEGLVaUyhF8Gm5VssK2cfITPYgwBcoj6Q\\\\\\\/U6ZhC36oGR6IyNwvWDFVu6SDPT0hVnTSz5CWZCCHOO3yHiGImOWyyWufMHpKQtD0ZsjIyflPif1hBENrACNH9W45n8GGug812QcG2RZHKZeT1tmlRsL3tmOY7qc1OC+vYyPTXED7dHSIlXhKm\\\\\\\/ny5\\\\\\\/BseQazbsxsZNN1XLlbmuVFDtHvFWVuWBgeVxKpoOxZf1\\\\\\\/bXCGZwepjhzBDvXq+bF\\\\\\\/wOdkhZjJzUfQTxBdghUpq1ReDoNU1rdmp+rh5ahKGsNBWvi9n6n5V+B2MFUvo7Uf8sOViaFmVoCR3w5qwoo2Lx+eMCYgWW2p0POmd6lSxPfmUxa9\\\\\\\/5udCpHs+KBWK6SWJ4q3Tcox3bzqboexJUPDsT2ze157sLdkOiXBU6uPT8pv8RI9pLsP6lHEyVaTNmEikOQ3QIJ6iP8Vj6\\\\\\\/gwgGFQ7acY00zGfrV5TGO2nZfjn\\\\\\\/4H1ORBEzPOn6RuHGX1B2HaSRp\\\\\\\/fG69Y+A\\\\\\\/t1lJr9lzDkdk7j0wpNg8z0peMaXhAdLSKdo1dtHzkE3lLUQS57vapJe6OVCgseXYV\\\\\\\/kpfNYOWpioDwNhQnNmbo5Sv2XuHMNnBOexQ0aOP2hY1cq1Hn5J\\\\\\\/G+YR4GcRla5qvYWHyCIXly9AzWV4HREozTf3FJRQ\\\\\\\/ulNEZ4guJVgOz5n3J5iz3VxnCVLMuHKPl0khW3MAH90q8leGMOUKWYqRAZa20VwUxi1f0jH8Gi9ooiRXmPwi\\\\\\\/LEcS6xQRGDGboz5LitXbFbf8RG4Tem6TghyFhc73x2wD\\\\\\\/d4P52cOiGpOQ208tDGbYh99Qba4\\\\\\\/Ex\\\\\\\/KOSmG4ItKMgp\\\\\\\/I6lK4GkoDWIdDyNoWrMgl8chrpkvy9SwecYo2P9jkmDFmoEXqxp7zj0h8gZksbP4eqCAOsKb4Hkhup6l8mu2ruybwPN3gq7nZ8ft\\\\\\\/QI9plaEH978MAAoWZJkDzbz+X9SJknSzhPNgMbxyMEXg11B9Q7P\\\\\\\/YVS5boYOzon7SQdg+KWrnIcADFRSju+rV0P\\\\\\\/utJi1amUigwU6BU+SGh8kgzGAms7Bzpsylu1Sy1fXfm6R2gJdJarwjL64KcSoouqrmJ6VQw4uf7efuKeOeAw9w5efWhpfyZNX2GVjJKGst\\\\\\\/hxU0\\\\\\\/9sbqhhSj5WfMCKA\\\\\\\/wEY3f2iaggRAz38tdrOLD7yCzjy3WJgkuxU7njUja5q8iwpA10eA6u0QeuwzoV8k7A3+4AQW7Cs8heeefniZ0Qizy3dTE+f2QHeQ4LAIQh35Dtz2I1mQq7rkfiigX\\\\\\\/eIPXwPewPNOll8D\\\\\\\/XhKmzdbw6T7FwOFbmD5BeNcPNTTKnT5ioQeEkk9aboXSerI5RCj4BrUkxoBpmimTWVVemxJiNEcyF9jVYJXmpdqnmIAm9jjrPOHXeVwRqUsxOaNvSKwajYcNEdMh677\\\\\\\/8zr6s4rZikzWuFW9UF0gZjYvMDvc6wbgM3BhDWM+YmC8ajgXDsWS3m9dDsUUZAH3qMRxOpr6IS9EEsQlI4UtoPSGH0i6H+16VeCZWsEEiOU7DS1elTnoZpOS1gcRIBEPv\\\\\\\/wYz8DDEltQweL\\\\\\\/LCfHFV6L69gAZjuTnwwtuYXrsYAf\\\\\\\/GrIeFIXlgo5PmyRRKCADIKibmOyo0DSoSOZjDIuu5gsLH4usznu5BRETl4p7YsMKAvVtKP+Wf1pQSfoddcrUDBqcLfsaZOp28HiN0lAcWg2\\\\\\\/OVa2ykc\\\\\\\/l1579tk04yDyHj9Zu3x35Tjby4xNVUIpkTmOf7ysqho12U73ajcBMu\\\\\\\/23mSAFGyAITnNqFaT7YeV0XdKaAxVozD5u4FixzdnL9O2b2yA35cb59fHiFzyYmS0wKzp48v5I7omJ9aPIxG6mT2MN6vMKrz4hBH0yRtYteeSvUxo0t6uNoznJ4DaPJTnBZi4tcG\\\\\\\/OlJUY2Alux588nHqs82QO4hG+CuVehEGUXcDi+WT7x3lnntwlW0F+vQs1bO5F1PTlVUb4HeJGOwhUMa7Si7ZE3Hij4DVuLd8RL9\\\\\\\/YGXi\\\\\\\/wjeAq6PfA2WonPbKS4QuIVR1QEy5KLOyQAN+sgMCeP3b+yzLLKbVArH1wI2Gnnv1arIl8IXCrUu4HvsosIu6LknNfL7tFqwYp+oLuC7iSxYeZ3r1uMTj94CI7wj0HN0MEYe6Xrijh2f4OfwCcd\\\\\\\/N+sN2THJlhMlzDLPLYyClLai\\\\\\\/P+vdqEu43YAwzQX+KUlcC87FA+XxP+qs\\\\\\\/LDdxu0qHLyHbW4Ya6bQlCBFXt8Wc5hzwO8gkPLY4AtpfHsG9rBfwrwyeDdnloQkDPGG0s6Lj3kdQJQSFroUBg032jEbz3E7HABaWdZKq2iN7GuuMTcRyb8l9LfZPwoe\\\\\\\/BO\\\\\\\/wjcPXf6h5d5+I5NtIOLL+sfJH85JOwHQUsyHXfAb0GLEk4FiAbtqImQ\\\\\\\/LIgQuvSO9iSgYKnB09AHR4weH6bHVKmwFisKYgNDDBNNVruTMXSwyshaZukOn+Tke1viQ0XRqa7V5GeRmM6XFC\\\\\\\/Bk5auzhuonlJ+BbqpYZbYULkSMfUCvi1qH8\\\\\\\/PmSAFFOsAv0q4OGtbzLFitweReA463KNP8Je7SHisNqu0S2rMsfKfm5gwADRB29GObQfG+S+rvf\\\\\\\/u75om0fMeq1SALF2Dtn39ZFJNFf74s5whJSA7KQ1UU3LPsyUxcQNsUbhUQi+yeoa+8CzH\\\\\\\/FHGPnGJFpOaYULKCiBWFR+vrzHwNQZwVCYG2kr6HrRSc18m+NvGntpGV8E2725GiE1T+EEtVNMDNOAy1GQxA90CgFOTgxqSF7Y8nn78JIuiPYgEmz7YNHPlNu\\\\\\\/QoZF8DpgPM1sXALNUD2j5GsAoc\\\\\\\/hVitQ6mTyx2fWT8fgHdQs0ziVO5Az\\\\\\\/LLPduDLutAc4Hd+0O\\\\\\\/cadySnBqjYvOsK9CfBEus\\\\\\\/C0wndoyO8k6bsvx1mrrR2du2SbXCUwH1+wjjin8\\\\\\\/BYFVykpmIyRqjnvPS+O\\\\\\\/E2cTaIc3dncsthGYA50KMqp1mhiOYJ6trDKTVPv7osObPpsVxjOncacwJaHk89usNaR9d6AL9J3mqZNa2rsTZScZYemLVNz+Jord+12jOQGEkTM1Ak7m1eyd3TDyVinf4a7X4qTEzKiHCD6cQ+Ks6g2QKqc1ae9yD3xCIjvnvl5Dp2su2ker8jQVpTCDcO8Ty518k4cInz2rm4TmcbhR+VcVD3LEjfeCdXQYLY\\\\\\\/BWjPBuMaWP5X0kNg98R9MMREHOyV4XiVZkh8LvYa7TjodOwaXT8VrBlNUyq6\\\\\\\/XqzQLguU9tYBFhBrqpBaEDnIgSZHNvvNHw2IsTs8kYpSGCowxCo0EBqRSb3DtaQogxGtEmRusbkWnHs1mK+ne5upTv0LMY2oURBYFx1Pcoc1lyJ\\\\\\\/\\\\\\\/QjZNGqBSzkVZufjPpYalpSAMyI9YpIc6OvqY1n2Wyf\\\\\\\/7gYuwlQyscypKA7twBjbIGc+4IPzvStXpdaxKpwc2Xuyo6eDEXDw0iGTo1K4M2mTqagGyTtR788aDwjKTMruaPs1yVfwCUyGfnQ80pgIQ8En5p4fCHOft86vqY1oUyVbqgPLLNh3xy3pZAwbqbIDCHypj\\\\\\\/GxPiAO9jQQWBfBpQ\\\\\\\/slz2pUkPtfrXYHCU1xlR6Ui+6jxWqmxR63\\\\\\\/4gQAYL+leczqtFqNCAAasRZJx9Hr3956wimHA+sjOmbZ5fj4bA9sCs\\\\\\\/YrmqXFSAIcAizUFbK\\\\\\\/+m\\\\\\\/PSHktC0UTvGE2BmAFLY6b2ryF2GxD3sVUzhSInxzzgUk3Qb64rV5qiwE0Amt\\\\\\\/aYQC\\\\\\\/9my97knUEvUqM138aHAwsSJyjCBXlKvaeaGK43ddFGR4WnF20XQkjsuJQ5+sfEh9j09xBX8LFqC5LIQQXeIB4Iu3BdtD+PHKmw6V6oWUGcP+jmMaV7OVNEXZeBMu1M2bP9Q4jUsOphKG64iLJbYKnWMi29WY4G30xWkA\\\\\\\/MwkJBJEyQLHVhWCvdIdoJAgCbcCe1oPvUjR7tjlD+0dQ7FPr01TzGVQ9oUG4Jju1N0fIdX1cIRDw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022LZC9bBJXufCuejCUukGYgA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226742db1151066b39b17f0a0aa44ff298303cc80c89d58641e9472fa466e9fae0\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZQ0VBS2u3jV6A3Bc5hD62EkN2M\\\\\\\/5UE9eg2cJ+VrSxFxmX0mu1p0ITDE\\\\\\\/sz5xt5RDH\\\\\\\/6dmgsFxcYIcl8rpvUN0XuQIX4UbK175KUupFU1wzhMR9kOR3MAeYl\\\\\\\/FDqeENqNbhJj3wD2HHfKZvZOzhSNf350+xH6hql384rEyMQQ9W+LpYcusqM3tIPCPUYaWJTszcQo\\\\\\\/9PkNq21bzhham0OOElg0H+fjdzq296Js+kBhsK5b46wXepKqjLym8lyvAbe0y4\\\\\\\/Y2B\\\\\\\/rdUhfHJCumW8f+NY\\\\\\\/sNzuJyLIJeComy5al7Ds4iJnZFeTtF7UHq3MLG3R3pIgUBgWR7N0VZ3A8KGUS09cBocUHSVF+\\\\\\\/2obb\\\\\\\/qi05gkBiwW80zkGVMNGGBnORDa5rUmRvjKuJgKpm3PE5Ai+CKgBzYPL8w2w1PhJ7JWdxLyGPUaApHxouPEv9QOPUT5J95xvE6avDoLUjVZj5Rh61yWTqeZMRpYQp7mSqfOnWY+VgaMB5J3kT3ZzY02+xlMi+kBU6KTpyfnagczCkoFqvJ0CqyRotMiuh8cMpexrpDRiKwCzMR7X1JIYTewj+GfuKpb1TK78r1ZHm0S0ZFfKRJ3IuNOGK+BPXmhKl+pE590W01W0g9v5Ynr\\\\\\\/4zdySTs4W28CD3tBa4x5WgVaQ3jF2Zz2xSPqF2iNOlsWSuh83DDhixmpKdntmikQlFejVrY7TBYxSg8b9jGS+Ku8mTsCpcPSfqX9FPrgU6oycH6m\\\\\\\/FyT35ogJGVF+AGJcwJE5qKlQI9w+yzIAeo9tengcFEVbPTLDIZFn3putHvV5O5KELL0lP4IbnqW7kSnZ1Jxsm7YcMRi2ozEYBZPekKw3\\\\\\\/PHtnS9EDAqI72ScINsIj9\\\\\\\/zdu60vJwq\\\\\\\/C86S\\\\\\\/SLKzpqAWrTh04JMvB9UYC9XQlQullIvlk2QIAOTE1nZ7B\\\\\\\/u0lKklZ+lRkvc1WJgp7h9KbZPOKC7kMqH\\\\\\\/BpzqZc0hdlwgjR+ZktfaxQ9VlgmB2CK8ceWRzH8G1ZwmicvICN+07uye1rya+6SbApaV7InhtMMwqCf5IkukBaAjvhHFUG9saLt8r7A7Ge2PLpH2u0vz6k0aj8QQ+DMhKq1hGDrqPC754ijObOgfwjAKk\\\\\\\/RHS4mXvdRhSy6X9ai1jLFzhmVjjnuTCnGJLphiQxOj4zRirLQei+WS13Nqk5AwNvBAqCsjzLN8dfg42ryHVtnwlWBNUZK+UfqUPiOcIq\\\\\\\/nTHilZDSMaE0lrbCiVYsbBr+XD1ep0oT8LD\\\\\\\/fTQyho9Lf23zLBUgOhKq7uVI4Oombr8TS8qF2fYWUjVluvja1nIJAKAIGFr\\\\\\\/GCbEY\\\\\\\/ryuIRrj50k4ShpaoGKz52M3crmU++lPwiHgqjXXYeMmylmge+E2iEOoEREB8R8LWNhsfdsdtl02iQxG0rXcEgpPFMkAJVh0damwUVOu8ogztc6SGDQ5Cq7EgNjK2SbJMjAS8PtEFAN2H5Wx\\\\\\\/clLX6\\\\\\\/y4IAhP6ewzh6O\\\\\\\/Ub0o5E7aCT7PTCcf2nnLDcpJJ8g+nOPFp\\\\\\\/EyHZrimh3jANaGnY8LM6cy1do2\\\\\\\/Per\\\\\\\/Ac59NVcRoKGHzTpFL+aEvSJcxXCRLamfku+oVtxhFurvgFBGxpJFExxs1zHFq7EpTNqmUsaMwJmAI5+ZuvxYx3VR2Hds5C1cCCblRY+7eWYfUrXct3LDwlGB5q8SIwEPNsB965Sfg3ymqMzN+WiubUqzka7Ldej7jteDrv+whnplmx3uz9xsbkD5RBHKHd+YaTMr8L9ymo1vy3Ac95j0Qw+oiBWgoCAnnk4t4GFcie+qdpIbk5QkNKiasU6wEXpB91CnON11NcvKalurA4qn917RYE3n9rqn8aURsHOCUlz0fASuDBuyXcZIRnkN6P6uQqpIjmvUBMJeA0m01hg1FfUB8QUP4I7j6URc9f7DSa0EB5tY\\\\\\\/9zMNlrqN+KXIB8kYxUpb1brJ3C38ofl2U2h1DcD2VSp12DBDtOfdojr0xtbXm05PGXJdQeHLG2ZVA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022hb9x0Wa5UQ0szg41orXgPQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c40e96163c96e4c91b5c34a641f95509e806d2c3c59568a94bb7ff35e16489f8\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491828 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/1/117" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-A21knhU0sQ0y5RXWZi-sOn1YDKp9k2C2N1xrPWXx4JA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/1/117" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-dtEbVlLSHTWvjtEduGiSxSCGkLv9XZfSqjnQ6m3CYkI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/1/105" title="Domestication and Temperature Modulate Gene Expression Signatures and Growth in the Australasian Snapper Chrysophrys auratus" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/1/125" title="Genotype Imputation in Winter Wheat Using First-Generation Haplotype Map SNPs Improves Genome-Wide Association Mapping and Genomic Prediction of Traits" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491828" id="node491828" data-pisa="ggg;9/1/117" data-pisa-master="ggg;g3.118.200836" data-apath="/ggg/9/1/117.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fine-Scale Resolution of Runs of Homozygosity Reveal Patterns of Inbreeding and Substantial Overlap with Recessive Disease Genotypes in Domestic Dogs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3800-4459" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aaron J. Sams</span> and <span class="highwire-citation-author" data-delta="1">Adam R. Boyko</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">117-123;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200836 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491828" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aaron J. Sams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embark Veterinary, Inc., Boston, Massachusetts 02111</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BSams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sams%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BJ.%2BSams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3800-4459" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aaron J. Sams</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:asams@embarkvet.com" class="" data-icon-position="" data-hide-link-title="0">asams@embarkvet.com</a>
<a href="mailto:adam@embarkvet.com" class="" data-icon-position="" data-hide-link-title="0">adam@embarkvet.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Adam R. Boyko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embark Veterinary, Inc., Boston, Massachusetts 02111</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biomedical Sciences, College of Veterinary Medicine, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BR.%2BBoyko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boyko%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAdam%2BR.%2BBoyko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:asams@embarkvet.com" class="" data-icon-position="" data-hide-link-title="0">asams@embarkvet.com</a>
<a href="mailto:adam@embarkvet.com" class="" data-icon-position="" data-hide-link-title="0">adam@embarkvet.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/1/117" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491828" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491828/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/1/117.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491828" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491828/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/1/117.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:491828" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:491828/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/1/117.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491828" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491828/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/1/117.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491828" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491828/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Inbreeding leaves distinct genomic traces, most notably long genomic tracts that are identical by descent and completely homozygous. These runs of homozygosity (ROH) can contribute to inbreeding depression if they contain deleterious variants that are fully or partially recessive. Several lines of evidence have been used to show that long (&gt; 5 megabase) ROH are disproportionately likely to harbor deleterious variation, but the extent to which long <em>vs.</em> short tracts contribute to autozygosity at loci known to be deleterious and recessive has not been studied. In domestic dogs, nearly 200 mutations are known to cause recessive diseases, most of which can be efficiently assayed using SNP arrays. By examining genome-wide data from over 200,000 markers, including 150 recessive disease variants, we built high-resolution ROH density maps for nearly 2,500 dogs, recording ROH down to 500 kilobases. We observed over 678 homozygous deleterious recessive genotypes in the panel across 29 loci, 90% of which overlapped with ROH inferred by GERMLINE. Although most of these genotypes were contained in ROH over 5 Mb in length, 14% were contained in short (0.5 - 2.5 megabase) tracts, a significant enrichment compared to the genetic background, suggesting that even short tracts are useful for computing inbreeding metrics like the coefficient of inbreeding estimated from ROH (<em>F<sub>ROH</sub></em>). In our dataset, <em>F<sub>ROH</sub></em> differed significantly both within and among dog breeds. All breeds harbored some regions of reduced genetic diversity due to drift or selective sweeps, but the degree of inbreeding and the proportion of inbreeding caused by short <em>vs.</em> long tracts differed between breeds, reflecting their different population histories. Although only available for a few species, large genome-wide datasets including recessive disease variants hold particular promise not only for disentangling the genetic architecture of inbreeding depression, but also evaluating and improving upon current approaches for detecting ROH.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/inbreeding" class="hw-term hw-article-keyword hw-article-keyword-inbreeding" rel="nofollow">inbreeding</a></li><li class="kwd"><a href="/keyword/inbreeding-depression" class="hw-term hw-article-keyword hw-article-keyword-inbreeding-depression" rel="nofollow">inbreeding depression</a></li><li class="kwd"><a href="/keyword/identity-descent" class="hw-term hw-article-keyword hw-article-keyword-identity-by-descent" rel="nofollow">identity-by-descent</a></li><li class="kwd"><a href="/keyword/canid" class="hw-term hw-article-keyword hw-article-keyword-canid" rel="nofollow">canid</a></li></ul><p id="p-3">Chromosomal segments that are homozygous by descent (autozygous) are a hallmark of inbreeding. Close consanguineous matings typically result in offspring with several long runs of homozygosity (ROH), while matings between more distant shared relatives (as often occurs in small or bottlenecked populations) produce a distribution of ROH skewed toward shorter tract lengths (see, for example Figure 2 in (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Howrigan <em>et al.</em> 2011</a>)). Most organisms of interest contain a wide array of segregating, often rare, recessive or partially recessive variants that can produce a deleterious phenotype when exposed as homozygous on genomic segments of autozygosity. Therefore, the efficient and accurate identification of ROH is of immense interest in the field of genetics, particularly in conservation biology and plant/animal breeding where avoidance of inbreeding depression is of critical importance.</p><p id="p-4">Because ROH are a direct consequence of inbreeding, the two concepts are closely related. Inbreeding is often estimated from pedigrees, where the coefficient of inbreeding (<em>F</em>) is calculated as half the coefficient of relatedness (<em>r</em>) between the parents of an individual (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Wright 1922</a>). However, a pedigree-based estimate of <em>F</em> merely measures the mean expected autozygosity of an individual and not the true inbreeding level for an individual—the actual proportion of the genome that is identical by descent—which depends on the actual segregation and transmission of chromosomal segments (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hill and Weir 2011</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Keller <em>et al.</em> 2011</a>). Furthermore, in many populations, pedigrees may be inaccurate, incomplete, or missing, leading to incorrect or biased estimates of inbreeding (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cassell <em>et al.</em> 2003</a>).</p><p id="p-5">Genetic marker-based <em>F</em> estimates can be more accurate than pedigree-based estimates, but estimates based on only a handful of markers are typically less precise than pedigree-based estimates. For example, early molecular approaches to indirectly estimate <em>F</em> from microsatellites involved calculations of multi-locus heterozygosity (MLH), <em>d<sup>2</sup></em>, and internal relatedness (IR) (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Coulson <em>et al.</em> 1998</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Coltman <em>et al.</em> 1998</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Amos <em>et al.</em> 2001</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Slate and Pemberton 2002</a>). However, several studies later demonstrated that small microsatellite panels are ineffective at accurately estimating <em>F</em> (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Slate <em>et al.</em> 2004</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Balloux <em>et al.</em> 2004</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">DeWoody and DeWoody 2005</a>).</p><p id="p-6">Dense genotyping from either whole-genome sequencing or array-based genotyping allows for the detection of ROH and the inference of autozygous segments of the genome. Long ROH are indicative of recent identity by descent (IBD) and the sum of these tracts is, in theory, the exact inbreeding level of an individual. However, what constitutes a “long” ROH is unclear (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Peripolli <em>et al.</em> 2016</a>) and establishing an optimum length threshold is a challenge (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">McQuillan <em>et al.</em> 2008</a>).</p><p id="p-7">The two parental chromosomes within a diploid individual can be considered IBD at any point, as IBD is ultimately determined by coancestry back to a coalescent event. Such a definition of IBD is unhelpful, of course, because every portion of the genome has a finite time to the most recent common ancestor and would be considered IBD under this definition. Defining a minimum ROH threshold length is thus necessary to create a meaningful IBD estimate and may be particularly important for studying genetic load since some evidence exists that longer tracts of homozygosity carry disproportionately more deleterious variation than shorter tracts (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Szpiech <em>et al.</em> 2013</a>). However, the threshold at which a ROH is considered as an IBD segment has generally been determined empirically (if not arbitrarily) as a tract length that is long enough to likely have been inherited from a recent common ancestry (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ku <em>et al.</em> 2010</a>) and has not been determined by asking which length threshold is actually most useful for detecting ROH harboring deleterious recessive loci.</p><p id="p-8">Estimates of inbreeding are correlated with negative fitness consequences and reduced breeding value of a wide variety of traits of interest in many populations (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Lencz <em>et al.</em> 2007</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Nalls <em>et al.</em> 2009</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Szpiech <em>et al.</em> 2013</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Zhang <em>et al.</em> 2015</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Mészáros <em>et al.</em> 2015</a>). Precise estimation of inbreeding is particularly important for understanding inbreeding genetic load (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Charlesworth and Willis 2009</a>). In particular, inbreeding genetic load must be traced to recessive deleterious variants harbored on actual ROHs, but the degree to which this load is a consequence of ROH of various sizes, and thus the thresholds by which biologically relevant IBD tracts should be inferred, is not well understood. Previous studies have used functional predictions rather than known deleterious mutations and have led to different conclusions as to whether short or long ROH harbor more deleterious genetic variation (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Szpiech <em>et al.</em> 2013</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Zhang <em>et al.</em> 2015</a>).</p><p id="p-9">Domestic dogs are an ideal organism in which to study the phenotypic effects of recent inbreeding. Hundreds of dog breeds, each with their own unique genetic history, form closed populations usually characterized by significant levels of autozygosity owing to founder effects, bottlenecks, popular sires, and artificial selection for conformation or performance. Approximately 200 known Mendelian recessive disease variants have been identified in dogs, the majority of which are potential models for human disease (Online Mendelian Inheritance in Animals) and can be tested efficiently with genotyping arrays. While a few of these variants, like the <em>SOD1</em> mutation, which predisposes dogs to degenerative myelopathy (a relatively late-onset disorder), are ancient mutations segregating in dozens of breeds, most of these disease variants are found in only one breed, or at most a few related breeds, suggesting they are relatively recent mutations (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Awano <em>et al.</em> 2009</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Boyko 2011</a>).</p><p id="p-10">If a method to detect ROH is highly accurate, the vast majority of homozygous recessive disease genotypes should occur in regions classified as ROH (and essentially all homozygous genotypes for diseases caused by recent mutations). This is because almost every disease variant is a consequence of a single mutation and thus occurs on a single genetic background. Therefore, all homozygous genotypes for that variant must occur in an ROH, at least until recombination breaks down the background haplotype so it is no longer detectable as ROH by genomic methods. Furthermore, the distribution of tract lengths for tracts overlapping these genotypes will enable a direct test of whether longer or shorter ROH tracts disproportionately harbor known recessive disease mutations. Even relatively long tract length thresholds (<em>e.g.</em>, 5 cM or 5 Mb) can capture shared ancestry ∼20 generations back which is substantially better than most pedigrees. However, even shorter thresholds capable of detecting coancestry even further back might be desirable. By incorporating knowledge of known recessive disease variants along with dense genome-wide data, we can better refine and evaluate methods of ROH detection, and more precisely investigate patterns of ROH across populations and across genomic regions.</p><p id="p-11">Here, we estimate ROH using two popular methods, PLINK (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Purcell <em>et al.</em> 2007</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chang <em>et al.</em> 2015</a>) and GERMLINE (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gusev <em>et al.</em> 2009</a>), to examine the association between ROH and known at-risk genotypes (observed cases of homozygous recessive deleterious genotypes) in domestic dogs. We hypothesize that at-risk genotypes will be highly enriched in ROH regions compared to the non-ROH genomic background. This enrichment can be used to evaluate the sensitivity and specificity of ROH-calling methods and can provide a direct test of whether longer ROH tracts are more or less enriched for these recessive disease variants. We additionally characterize the distribution of ROH in 11 common dog breeds as an example of how the distribution of ROH is influenced by the timing and extent of artificial selection in a breed.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>At-Risk Dog Dataset</h3><p id="p-12">We queried Embark’s customer database on April 4<sup>th</sup>, 2018 for all dogs whose owners consented to participate in research that were homozygous (at-risk) for recessive deleterious conditions assayed by Embark’s platform. In total, we identified 678 at-risk cases in 670 dogs (some dogs were at-risk for more than one condition). We separated these at-risk cases into two categories: 1) at-risk for <em>SOD1</em>-based degenerative myelopathy (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Awano <em>et al.</em> 2009</a>), of which we observed 283 at-risk dogs, and 2) at-risk for all 28 other recessive deleterious conditions assayed by Embark, of which we observed 395 at-risk cases across 393 dogs (Tables S1, S2).</p></div><div id="sec-3" class="subsection"><h3>Breed Dog Dataset</h3><p id="p-13">We queried Embark’s customer database on January 23<sup>rd</sup>, 2018 for all customer dogs identified as purebred by Embark from the most common 11 breeds and whose owners consented to participate in research. We then used the ‘–genome’ flag in PLINK v1.9 (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chang <em>et al.</em> 2015</a>) to identify pairs of dogs for which the proportion of IBD (PI_HAT) was greater than 0.45 and used these pairs to remove dogs that were potentially related as parent-offspring or full siblings. In total, our final dataset included 1,792 dogs from 11 breeds (Table S3).</p></div><div id="sec-4" class="subsection"><h3>Genotyping &amp; Quality Control</h3><p id="p-14">Customer dogs were genotyped on Embark’s custom high-density genotyping platform containing approximately 220,000 markers including all 173,000 markers found on the Illumina CanineHD platform and probes to detect over 160 Mendelian disease variants. SNP filtering using PLINK 1.9 (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Chang <em>et al.</em> 2015</a>) was done to ensure genotype concordance rates above 99.99% and missingness rates below 0.1%. Genotype data were phased against a proprietary reference panel and missing data imputed using Eagle2 (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Loh <em>et al.</em> 2016</a>). SNP data were also pruned with PLINK to remove markers in close linkage disequilibrium using “–indep-pairwise 200 100 0.90”. After pruning, 170,728 autosomal and 4,395 chrX markers remained, for an average of one marker per 12.8 kb for autosomes (one marker per 28.2 kb on chromosome X).</p></div><div id="sec-5" class="subsection"><h3>Defining Runs of Homozygosity with PLINK</h3><p id="p-15">We generated ROH for at-risk dogs in PLINK using software version 1.9 (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Chang <em>et al.</em> 2015</a>) (which uses the algorithm from software version 1.07 (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Purcell <em>et al.</em> 2007</a>)).</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-16"> –homozyg-window-het 0</p></li><li id="list-item-2"><p id="p-17"> –homozyg-snp 41</p></li><li id="list-item-3"><p id="p-18"> –homozyg-window-snp 41</p></li><li id="list-item-4"><p id="p-19"> –homozyg-window-missing 0</p></li><li id="list-item-5"><p id="p-20"> –homozyg-window-threshold 0.05</p></li><li id="list-item-6"><p id="p-21"> –homozyg-kb 500</p></li><li id="list-item-7"><p id="p-22"> –homozyg-density 5000 (set high to ignore)</p></li><li id="list-item-8"><p id="p-23"> –homozyg-gap 1000 (set high to ignore)</p></li></ul></div><div id="sec-6" class="subsection"><h3>Defining Runs of Homozygosity with GERMLINE</h3><p id="p-24">Initially, we attempted to use GERMLINE’s internal filtering to identify ROH &gt;= 0.5 Mb and consisting of at least 41 markers using the following parameters:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/117/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/117/embed/mml-math-1.gif"/></noscript></span></span>where -min_m = 0.5 is in units of Megabase-pairs (all measurement in this study was computed in physical distance).</p><p id="p-25">However, we noticed an issue with the germline software in which using the <em>-w_extend</em> flag in conjunction with the <em>-homoz-only</em> whereby all tracts are extended beyond the first mismatching marker to the end of the next slice (or beginning of the previous slice).</p><p id="p-26">As an alternative, we used the following command in germline to generate preliminary homozygosity tracts for all dogs in this study:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/117/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/117/embed/mml-math-2.gif"/></noscript></span></span>This identified all segments of the genome &gt;500 kb with no heterozygous markers. We then merged all such segments separated by &lt;50 kb from a neighboring autozygous segment and subsequently removed all merged segments containing fewer than 41 markers (to avoid spurious inference of ROH in regions with few markers). We found this approach superior to allowing a certain set number of heterozygous markers within an ROH for two reasons: (1) requiring no heterozygous variants for at least 500kb vastly improves the specificity for detecting short ROH (500kb - 4000kb), and (2) allowing one or a number of tightly clustered (&lt;50kb) heterozygous variants between two ROHs improves the sensitivity for detecting long ROHs that would otherwise be broken up by genotyping error or copy-number variation (deletions or duplications, most of which are &lt;50kb, can lead to clustered heterozygous at the markers within the structural variant).</p></div><div id="sec-7" class="subsection"><h3>Defining <em>F<sub>ROH</sub></em></h3><p id="p-27"><em>F<sub>ROH</sub></em> was computed as in previous studies (<em>e.g.</em>, <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">McQuillan <em>et al.</em> 2008</a>) as:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/117/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1/117/embed/mml-math-3.gif"/></noscript></span></span>where <em>ROH<sub>k</sub></em> is the <em>k</em>th ROH in individual <em>j</em>’s genome and <em>L</em> is the total length of the genome (or X-chromosome).</p></div><div id="sec-8" class="subsection"><h3>Data Availability</h3><p id="p-28">File S1 contains a list of all supplemental files. File S2 contains supplemental figures, tables, and references. File S3 contains phenotype information for at-risk dogs in this study. File S4 contains breed and sex information for breed dogs in this study. Files S5 and S6 contain genotype and marker location information in <em>PLINK</em> .ped/.map format for at-risk dogs in this study. Files S7 and S8 contain genotype and marker location information in <em>PLINK</em> .ped/.map format for breed dogs in this study. File S9 contains a python script for postprocessing germline homozygosity tracts generated using the flags presented in Materials &amp; Methods. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7330151">https://doi.org/10.25387/g3.7330151</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Distribution of overlaps of ROH with known homozygous at-risk alleles</h3><p id="p-29">In total, 670 dogs whose owners consented to participate in research were homozygous for at least one of 29 Mendelian disease alleles assayed by Embark and considered in our analysis. Of those, we measured the genome-wide distribution of ROH in order to compare it to the set of ROH overlapping a homozygous at-risk allele in our sample. We identified the set of ROH overlapping each occurrence of an at-risk genotype and summarized these tracts in four length categories (&lt; 0.5 Mb–below ROH detection threshold; 0.5 Mb - 2.5 Mb–short; 2.5 Mb - 5.0 Mb–medium; &gt; 5Mb–long).</p><p id="p-30">We analyzed ROH generated from PLINK and GERMLINE using similar parameters for identifying ROH. In short, we considered ROH &gt;= 0.5 Mb as long as they consisted of at least 41 markers. The results are broadly consistent between these two analyses (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), although GERMLINE was modestly more sensitive, in terms of at-risk genotypes overlapping GERMLINE ROH tracts slightly more often than PLINK-generated ROH tracts, so we focus on results from the former here.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491967/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491967/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Analysis of runs of homozygosity (ROH) in dogs carrying homozygous recessive deleterious mutations. ROH &gt; 0.5 Mb (detected by our ROH analysis) harbor recessive deleterious alleles at minimum 29.8X more than ROH &lt; 0.5 Mb</span><div class="sb-div caption-clear"></div></div></div><p id="p-33">For all at-risk cases excluding <em>SOD1</em>, 92% of at-risk genotypes had an ROH overlapping the at-risk allele. While the longest ROH in our analysis harbor the majority (66%) of deleterious recessive alleles in this panel of at-risk dogs, short ROH nonetheless harbor known recessive disease homozygous genotypes at a rate nearly 30x higher than stretches of DNA that are not considered ROH. Across all tract lengths that we considered, the relative risk of a ROH carrying a deleterious mutation was similar across classes, suggesting that ROH of all lengths may contribute to inbreeding depression in dogs (see <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). To ensure that the deviation of the tracts overlapping at-risk genotypes is not a product of random sampling, we resampled the full distribution of homozygosity tracts from at-risk dogs to mimic the sampling of tracts from at-risk dogs (see <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/117/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cumulative density plot of runs of homozygosity (ROH) by length. Tracts are ordered from longest to shortest. Yellow line is all ROH in all dogs at-risk for a deleterious recessive disease, excluding the SOD1 Degenerative Myelopathy (DM) allele. Red line is the distribution of ROH harboring homozygous recessive deleterious genotypes (excluding SOD1) in the same set of dogs. Blue is the set of ROH harboring homozygous genotypes of the SOD1 DM allele only. Gray lines are 1000 sets of ROH, with the same sample size as the red line, sampled randomly from the set of all ROH and illustrate that the blue and red sets of ROH are highly non-random samples from the full set of ROH." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-26071982" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Cumulative density plot of runs of homozygosity (ROH) by length. Tracts are ordered from longest to shortest. Yellow line is all ROH in all dogs at-risk for a deleterious recessive disease, excluding the &lt;em&gt;SOD1&lt;/em&gt; Degenerative Myelopathy (DM) allele. Red line is the distribution of ROH harboring homozygous recessive deleterious genotypes (excluding &lt;em&gt;SOD1&lt;/em&gt;) in the same set of dogs. Blue is the set of ROH harboring homozygous genotypes of the &lt;em&gt;SOD1&lt;/em&gt; DM allele only. Gray lines are 1000 sets of ROH, with the same sample size as the red line, sampled randomly from the set of all ROH and illustrate that the blue and red sets of ROH are highly non-random samples from the full set of ROH.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/117/F1.medium.gif" width="440" height="307"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/1/117/F1.medium.gif" width="440" height="307"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/117/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/117/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491928" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-34" class="first-child">Cumulative density plot of runs of homozygosity (ROH) by length. Tracts are ordered from longest to shortest. Yellow line is all ROH in all dogs at-risk for a deleterious recessive disease, excluding the <em>SOD1</em> Degenerative Myelopathy (DM) allele. Red line is the distribution of ROH harboring homozygous recessive deleterious genotypes (excluding <em>SOD1</em>) in the same set of dogs. Blue is the set of ROH harboring homozygous genotypes of the <em>SOD1</em> DM allele only. Gray lines are 1000 sets of ROH, with the same sample size as the red line, sampled randomly from the set of all ROH and illustrate that the blue and red sets of ROH are highly non-random samples from the full set of ROH.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">Although all the recessive diseases we studied had at-risk genotypes that were highly enriched in ROH tracts, the enrichment was not uniform across these disease variants. Notably for the <em>SOD1</em> mutation leading to canine degenerative myelopathy (DM) (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Awano <em>et al.</em> 2009</a>), an ancient mutation found in dozens of breeds, we observe a weaker enrichment of at-risk genotypes in long ROH (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p></div><div id="sec-11" class="subsection"><h3>Genome-wide distribution of ROH in 11 common dog breeds</h3><p id="p-36">We estimated and analyzed the distribution of ROH in 1,792 dogs from 11 common dog breeds. First, we calculated <em>F<sub>ROH</sub></em> for all breed dogs and assessed the distribution within breeds for both autosomes and chromosome X. Of the breeds we analyzed Doberman Pinscher had the highest overall levels of <em>F<sub>ROH</sub></em> and Beagle the lowest. In general, <em>F<sub>ROH</sub></em> of chromosome X varied in concert with the autosomes, although some breed (<em>e.g.</em>, Doberman Pinscher and Golden Retriever) had somewhat elevated <em>F<sub>ROH</sub></em> on X compare to autosomes while others (<em>e.g.</em>, Bulldog) had lower <em>F<sub>ROH</sub></em> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/117/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of FROH in 11 common dog breeds. Breeds ordered from highest mean autosomal (yellow) inbreeding to lowest. Inbreeding in females only for chromosome X plotted in blue." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-26071982" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of &lt;em&gt;F&lt;sub&gt;ROH&lt;/sub&gt;&lt;/em&gt; in 11 common dog breeds. Breeds ordered from highest mean autosomal (yellow) inbreeding to lowest. Inbreeding in females only for chromosome X plotted in blue.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/117/F2.medium.gif" width="440" height="324"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/1/117/F2.medium.gif" width="440" height="324"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/117/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/117/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491846" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-37" class="first-child">Distribution of <em>F<sub>ROH</sub></em> in 11 common dog breeds. Breeds ordered from highest mean autosomal (yellow) inbreeding to lowest. Inbreeding in females only for chromosome X plotted in blue.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">For each breed, we also assessed the distribution of ROH by length across all dogs in the breed (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). These distributions illustrate variation in the timing of diversity loss via inbreeding across breeds. For example, while today Doberman Pinschers have the highest average <em>F<sub>ROH</sub></em> of all breeds in our analysis, the relatively higher fraction of inbreeding in short tracts in Bulldogs reflects the tremendous bottleneck that occurred in that breed after bull-baiting was banned in the 1830s and the breed was driven to the brink of extinction (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Pedersen <em>et al.</em> 2016</a>). Finally, for each breed we calculated a map of the local density of ROH, in other words the fraction of dogs in the breed sample carrying a ROH at each position (Figures S1-11). These maps highlight the deterministic loss of diversity (ROH islands) within breeds, and variation in ROH islands across breeds. Regions of homozygosity associated with fixation of certain variants (<em>e.g.</em>, the chr13 <em>RSPO2</em> locus in poodles) are clearly evident and are concordant with previously identified homozygosity regions in these breeds (c.f. (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Vaysse <em>et al.</em> 2011</a>). However there is also marked variation in rates of homozygosity outside of these fixed haplotype windows, demonstrating that drift and selection have led to non-uniform diversity loss across the genome in these breeds.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/117/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cumulative density of runs of homozygosity (ROH) in 11 common breeds. A) Distribution of autosomal ROH across breeds. B) Distribution of chromosome X ROH in females only across breeds." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-26071982" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Cumulative density of runs of homozygosity (ROH) in 11 common breeds. A) Distribution of autosomal ROH across breeds. B) Distribution of chromosome X ROH in females only across breeds.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/117/F3.medium.gif" width="287" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/1/117/F3.medium.gif" width="287" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/117/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/117/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491993" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-39" class="first-child">Cumulative density of runs of homozygosity (ROH) in 11 common breeds. A) Distribution of autosomal ROH across breeds. B) Distribution of chromosome X ROH in females only across breeds.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-40">Our study is the first to examine patterns of ROH in dogs to a resolution of 500kb. While undoubtedly the sensitivity and specificity of detecting ROH is lower in short (0.5 - 2.5 Mb) tracts compared to long ones, particularly in regions of the genome with low marker density and/or high recombination rates, we observe a clear signal of enrichment of known recessive deleterious homozygous genotypes in these regions (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Homozygous recessive deleterious genotypes are consistently enriched 30-37-fold in ROH compared to the non-ROH genetic background regardless of ROH tract length. Thus, short tracts on aggregate represent a real signal of autozygosity, and furthermore these short tracts, often overlooked in studies of ROH, may be important and measurable contributors to inbreeding depression in dogs and other species. Indeed, 4.8–10.7% of the genomes of these 11 common breeds were covered by ROH tracts between 0.5 to 2.5 Mb (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>) and these tracts contained 18.7% of the <em>SOD1</em> and 14.7% of the non-<em>SOD1</em> known deleterious recessive homozygous genotype calls. Only 34.3% and 7.8% of the <em>SOD1</em> and non-<em>SOD1</em> homozygous genotype calls were not detected in ROHs, respectively (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). To determine whether gaps in ROH introduced by genotyping error may explain some cases of these false negative cases, we repeated our analysis, doubling the gap filling parameter from 50 to 100kb, and found that no additional homozygous recessive deleterious genotypes were found in ROH. Because these false negative calls were usually flanked by many homozygous markers (but for less than 500kb) rather than heterozygous markers, we believe they are due to ROH below our detection threshold and not due to point mutations arising on different haplotype backgrounds.</p><p id="p-41">Knowing the extent to which long <em>vs.</em> short ROH contribute to inbreeding depression would enable an assessment of the risk posed by recent consanguineous matings, which are often avoidable in dog breeding, and the risk from matings between more distant relatives which is generally unavoidable for purebred dogs. Because large-effect deleterious loci are purged by selection over time, segregating large-effect variants tend to be younger than neutral or weakly deleterious variants, and thus are likely to be breed-specific like most of the Mendelian disorders in this study). Therefore, if inbreeding depression in dogs is mainly caused by rare, large-effect recessive (or partially recessive) variants, the contribution of long <em>vs.</em> short tracts to inbreeding depression is likely well approximated by the contribution of long <em>vs.</em> short tracts to recessive homozygotes at known Mendelian disease alleles, most of which are rare on aggregate, although possibly common in the breed(s) they affect. In contrast, if inbreeding depression is mainly caused by common, small-effect recessive or partially recessive variants, the contribution of short tracts to inbreeding depression will be much greater, possibly even greater than the contribution of short tracts to DM risk at the <em>SOD1</em> locus. Further studies examining the distribution of ROH for different types of deleterious recessive loci (<em>e.g.</em>, Mendelian <em>vs.</em> complex, or strong <em>vs.</em> weak) in actual or in silico populations are needed to address this, and the results may depend strongly on the demographic history of the population and the specific phenotype(s) being used to measure inbreeding depression.</p><p id="p-42">Within dogs, we see substantial variability in levels of autozygosity between breeds as well as across the genome within a breed. These differences represent the unique history of each breed, and the effect of drift and selection for particular traits over time. At one extreme, many breeds have complete autozygosity in certain windows of the genome. However, genomic regions of high autozygosity that are not completely fixed are also evident in every breed, and efforts to preserve breed diversity should focus on preserving rare haplotypes in these regions rather than rare markers in randomly selected genomic regions. Where fixed haplotypes harbor deleterious variation, marker-assisted crossings and backcrossings to introduce new diversity at the locus or gene-editing techniques like CRISPR/Cas9 to remove the deleterious variant(s) are required to rid a breed population of harmful variants.</p><p id="p-43">Given the intense interest in developing and comparing methods to detect ROH, it is somewhat surprising that these methods are not typically evaluated for their sensitivity and specificity to detect known deleterious recessive mutations (indeed we are aware of no other genomic studies that have done so). While several studies have examined predicted deleteriousness of variants (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Szpiech <em>et al.</em> 2013</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Zhang <em>et al.</em> 2015</a>), current methods for predicting deleteriousness are almost certainly less accurate than current methods to detect ROH (and predicting recessiveness is even more fraught), making them a poor way to evaluate the sensitivity and specificity of ROH detection methods. Being able to evaluate ROH methods in this way, however, is extremely valuable for comparing ROH detection methods and fine-tuning parameters to optimize accurate ROH inference for a population and genomic dataset of interest. Accurate ROH tract detection is invaluable not only for inferring the coefficient of inbreeding and other genetic parameters of interest, but also for accurate reconstruction of population history and identification of relatives.</p><p id="p-44">The dog is an excellent genomic model for evaluating ROH methods as over 200 Mendelian, mostly recessive, variants are known and the requisite genomic resources (<em>e.g.</em>, high-quality reference genome and high-density SNP arrays) are available, although at present only the canine array platform used in this study includes both dense genome-wide coverage and probes to assay most of the Mendelian variants known in dogs. Although humans generally have much lower levels of inbreeding than purebred dogs, many more recessive disease variants are known in humans (Online Mendelian Inheritance in Man), and even denser arrays (including probes for many Mendelian variants) have been used to investigate over 10 million humans to date, largely on commercial DNA testing platforms (<a href="https://isogg.org/wiki/Autosomal_DNA_testing_comparison_chart">https://isogg.org/wiki/Autosomal_DNA_testing_comparison_chart</a>). Thus, high-powered studies looking at many more at-risk loci and many more at-risk individuals with even denser marker panels could potentially be done to further investigate the sensitivity and specificity of various ROH methods and the contribution of different size ROH tracts to inbreeding risk if privacy concerns could be managed and data access granted to researchers in the field. Until then, researchers are encouraged to use this publicly available canine genetic database and to develop similar databases in other model genetic species to improve both the methodology by which ROH is computed and our insights into the genetic architecture of inbreeding depression in these species.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">We thank customers of Embark whose participation made this work possible, in particular the Doberman Diversity Project. We also thank all of the Embark employees who made this work possible, particularly Ryan Boyko, Matt Barton, Erin Chu, Adam Gardner, Tiffany Ho, Andrea Slavney, and Samuel H. Vohr. We also thank the Embark scientific advisory board (in particular Carlos Bustamante for comments on this manuscript), Cornell University, and the Kevin M. McGovern Family Center for Venture Development in the Life Sciences, for their guidance and encouragement. This study was funded by the participants and by Embark.</p></div><div class="section ack" id="ack-2"><h2 class="">Author Contributions</h2><p id="p-46">AJS analyzed the data. AJS and ARB jointly wrote the paper.</p></div><div class="section ack" id="ack-3"><h2 class="">Author Information</h2><p id="p-47">Conflict of interest statement: AJS and ARB are employees of Embark Veterinary, a canine DNA testing company which offers inbreeding testing using methods similar to those described in this study. ARB is co-founder and part owner of Embark. Correspondence and requests for materials should be addressed to AJS (asams@embarkvet.com) and ARB (adam@embarkvet.com).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-48">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7330151">https://doi.org/10.25387/g3.7330151</a>.</p></li><li class="fn" id="fn-4"><p id="p-49"><em>Communicating editor: D. Bannasch</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-06-12"><span class="received-label">Received </span>June 12, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-11-05"><span class="accepted-label">Accepted </span>November 5, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Sams, Boyko</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.1" data-doi="10.1098/rspb.2001.1751"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilmer</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fullard</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burg</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Croxall</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The influence of parental relatedness on reproductive success.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">268</span>: <span class="cit-fpage">2021</span>–<span class="cit-lpage">2027</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.2001.1751</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DAmos%26rft.auinit1%253DW.%26rft.volume%253D268%26rft.issue%253D1480%26rft.spage%253D2021%26rft.epage%253D2027%26rft.atitle%253DThe%2Binfluence%2Bof%2Bparental%2Brelatedness%2Bon%2Breproductive%2Bsuccess%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2001.1751%26rft_id%253Dinfo%253Apmid%252F11571049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2001.1751&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11571049&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171566100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.2" data-doi="10.1073/pnas.0812297106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Awano</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katz</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide association analysis reveals a SOD1 mutation in canine degenerative myelopathy that resembles amyotrophic lateral sclerosis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">2794</span>–<span class="cit-lpage">2799</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0812297106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0812297106%26rft_id%253Dinfo%253Apmid%252F19188595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA2LzgvMjc5NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMS8xMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.3" data-doi="10.1111/j.1365-294X.2004.02318.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balloux</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulson</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Does heterozygosity estimate inbreeding in real populations?</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">3021</span>–<span class="cit-lpage">3031</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2004.02318.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBalloux%26rft.auinit1%253DF.%26rft.volume%253D13%26rft.issue%253D10%26rft.spage%253D3021%26rft.epage%253D3031%26rft.atitle%253DDoes%2Bheterozygosity%2Bestimate%2Binbreeding%2Bin%2Breal%2Bpopulations%253F%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2004.02318.x%26rft_id%253Dinfo%253Apmid%252F15367117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2004.02318.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15367117&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223863000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.4" data-doi="10.1186/gb-2011-12-2-216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The domestic dog: man’s best friend in the genomic era.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">216</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2011-12-2-216</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D216%26rft.epage%253D216%26rft.atitle%253DThe%2Bdomestic%2Bdog%253A%2Bman%2527s%2Bbest%2Bfriend%2Bin%2Bthe%2Bgenomic%2Bera.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-2-216%26rft_id%253Dinfo%253Apmid%252F21338479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-2-216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21338479&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.5" data-doi="10.3168/jds.S0022-0302(03)73894-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cassell</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adamec</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Effect of Incomplete Pedigrees on Estimates of Inbreeding and Inbreeding Depression for Days to First Service and Summit Milk Yield in Holsteins and Jerseys.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">2967</span>–<span class="cit-lpage">2976</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.S0022-0302(03)73894-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DCassell%26rft.auinit1%253DB.%2BG.%26rft.volume%253D86%26rft.issue%253D9%26rft.spage%253D2967%26rft.epage%253D2976%26rft.atitle%253DEffect%2Bof%2BIncomplete%2BPedigrees%2Bon%2BEstimates%2Bof%2BInbreeding%2Band%2BInbreeding%2BDepression%2Bfor%2BDays%2Bto%2BFirst%2BService%2Band%2BSummit%2BMilk%2BYield%2Bin%2BHolsteins%2Band%2BJerseys%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252803%252973894-6%26rft_id%253Dinfo%253Apmid%252F14507033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(03)73894-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14507033&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185137900026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.6" data-doi="10.1186/s13742-015-0047-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chow</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tellier</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vattikuti</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Second-generation PLINK: rising to the challenge of larger and richer datasets.</span> <abbr class="cit-jnl-abbrev">GigaSci</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13742-015-0047-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigaSci%26rft.volume%253D4%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13742-015-0047-8%26rft_id%253Dinfo%253Apmid%252F25722852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13742-015-0047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25722852&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.7" data-doi="10.1038/nrg2664"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetics of inbreeding depression.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">783</span>–<span class="cit-lpage">796</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2664</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCharlesworth%26rft.auinit1%253DD.%26rft.volume%253D10%26rft.issue%253D11%26rft.spage%253D783%26rft.epage%253D796%26rft.atitle%253DThe%2Bgenetics%2Bof%2Binbreeding%2Bdepression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2664%26rft_id%253Dinfo%253Apmid%252F19834483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2664&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19834483&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270915400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.8" data-doi="10.1098/rspb.1998.0363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coltman</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Birth weight and neonatal survival of harbour seal pups are positively correlated with genetic variation measured by microsatellites.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">265</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">809</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.1998.0363</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DColtman%26rft.auinit1%253DD.%2BW.%26rft.volume%253D265%26rft.issue%253D1398%26rft.spage%253D803%26rft.epage%253D809%26rft.atitle%253DBirth%2Bweight%2Band%2Bneonatal%2Bsurvival%2Bof%2Bharbour%2Bseal%2Bpups%2Bare%2Bpositively%2Bcorrelated%2Bwith%2Bgenetic%2Bvariation%2Bmeasured%2Bby%2Bmicrosatellites%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1998.0363%26rft_id%253Dinfo%253Apmid%252F9628038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.1998.0363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9628038&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073580200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.9" data-doi="10.1098/rspb.1998.0321"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coulson</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albon</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Microsatellites reveal heterosis in red deer.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">265</span>: <span class="cit-fpage">489</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.1998.0321</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DCoulson%26rft.auinit1%253DT.%2BN.%26rft.volume%253D265%26rft.issue%253D1395%26rft.spage%253D489%26rft.epage%253D495%26rft.atitle%253DMicrosatellites%2Breveal%2Bheterosis%2Bin%2Bred%2Bdeer%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1998.0321%26rft_id%253Dinfo%253Apmid%252F9569667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.1998.0321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9569667&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072731600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.10" data-doi="10.1093/jhered/esi017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeWoody</span>  <span class="cit-name-given-names">Y. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeWoody</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">On the Estimation of Genome-wide Heterozygosity Using Molecular Markers.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">88</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/esi017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesi017%26rft_id%253Dinfo%253Apmid%252F15618305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esi017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15618305&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227130600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.11" data-doi="10.1101/gr.081398.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gusev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Whole population, genome-wide mapping of hidden relatedness.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">318</span>–<span class="cit-lpage">326</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.081398.108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.081398.108%26rft_id%253Dinfo%253Apmid%252F18971310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzIvMzE4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8xLzExNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.12" data-doi="10.1017/S0016672310000480"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Variation in actual relationship as a consequence of Mendelian sampling and linkage.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">64</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672310000480</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D64%26rft.atitle%253DVariation%2Bin%2Bactual%2Brelationship%2Bas%2Ba%2Bconsequence%2Bof%2BMendelian%2Bsampling%2Band%2Blinkage.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000480%26rft_id%253Dinfo%253Apmid%252F21226974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000480&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21226974&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.13" data-doi="10.1186/1471-2164-12-460"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howrigan</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simonson</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Detecting autozygosity through runs of homozygosity: A comparison of three autozygosity detection algorithms.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">460</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-12-460</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHowrigan%26rft.auinit1%253DD.%2BP.%26rft.volume%253D12%26rft.spage%253D460%26rft.epage%253D460%26rft.atitle%253DDetecting%2Bautozygosity%2Bthrough%2Bruns%2Bof%2Bhomozygosity%253A%2Ba%2Bcomparison%2Bof%2Bthree%2Bautozygosity%2Bdetection%2Balgorithms.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-460%26rft_id%253Dinfo%253Apmid%252F21943305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21943305&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.14" data-doi="10.1534/genetics.111.130922"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantification of Inbreeding Due to Distant Ancestors and Its Detection Using Dense Single Nucleotide Polymorphism Data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">249</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.130922</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.130922%26rft_id%253Dinfo%253Apmid%252F21705750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzEvMjM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8xLzExNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.15" data-doi="10.1007/s00439-010-0920-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ku</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naidoo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teo</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pawitan</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Regions of homozygosity and their impact on complex diseases and traits.</span> <abbr class="cit-jnl-abbrev">Hum. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00439-010-0920-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BGenet.%26rft.volume%253D129%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00439-010-0920-6%26rft_id%253Dinfo%253Apmid%252F21104274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00439-010-0920-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21104274&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.16" data-doi="10.1073/pnas.0710021104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lencz</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lambert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeRosse</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burdick</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">T. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Runs of homozygosity reveal highly penetrant recessive loci in schizophrenia.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">19942</span>–<span class="cit-lpage">19947</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0710021104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0710021104%26rft_id%253Dinfo%253Apmid%252F18077426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzUwLzE5OTQyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8xLzExNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.17" data-doi="10.1038/ng.3679"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loh</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palamara</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchsberger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reshef</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Reference-based phasing using the Haplotype Reference Consortium panel.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1443</span>–<span class="cit-lpage">1448</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3679</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D48%26rft.spage%253D1443%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3679%26rft_id%253Dinfo%253Apmid%252F27694958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3679&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27694958&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.18" data-doi="10.1016/j.ajhg.2008.08.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McQuillan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leutenegger</span>  <span class="cit-name-given-names">A.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abdel-Rahman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franklin</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pericic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Runs of Homozygosity in European Populations.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">372</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2008.08.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DMcQuillan%26rft.auinit1%253DR.%26rft.volume%253D83%26rft.issue%253D3%26rft.spage%253D359%26rft.epage%253D372%26rft.atitle%253DRuns%2Bof%2Bhomozygosity%2Bin%2BEuropean%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2008.08.007%26rft_id%253Dinfo%253Apmid%252F18760389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2008.08.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18760389&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259307200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.19" data-doi="10.18047/poljo.21.1.sup.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mészáros</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stückler</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferenčaković</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sölkner</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">GENOMIC BACKGROUND OF ENTROPION IN FLECKVIEH CATTLE.</span> <abbr class="cit-jnl-abbrev">Poljoprivreda (Osijek)</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">51</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.18047/poljo.21.1.sup.10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPoljoprivreda%2B%2528Osijek%2529%26rft.volume%253D21%26rft.spage%253D48%26rft_id%253Dinfo%253Adoi%252F10.18047%252Fpoljo.21.1.sup.10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18047/poljo.21.1.sup.10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.20" data-doi="10.1007/s10048-009-0182-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nalls</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guerreiro</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon-Sanchez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bras</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Traynor</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Extended tracts of homozygosity identify novel candidate genes associated with late-onset Alzheimer’s disease.</span> <abbr class="cit-jnl-abbrev">Neurogenetics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">190</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10048-009-0182-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeurogenetics%26rft.stitle%253DNeurogenetics%26rft.aulast%253DNalls%26rft.auinit1%253DM.%2BA.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D183%26rft.epage%253D190%26rft.atitle%253DExtended%2Btracts%2Bof%2Bhomozygosity%2Bidentify%2Bnovel%2Bcandidate%2Bgenes%2Bassociated%2Bwith%2Blate-onset%2BAlzheimer%2527s%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10048-009-0182-4%26rft_id%253Dinfo%253Apmid%252F19271249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10048-009-0182-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19271249&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.21" data-doi="10.1186/s40575-016-0036-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pooch</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A genetic assessment of the English bulldog.</span> <abbr class="cit-jnl-abbrev">Canine Genet. Epidemiol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s40575-016-0036-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCanine%2BGenet.%2BEpidemiol.%26rft.volume%253D3%26rft.spage%253D6%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs40575-016-0036-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s40575-016-0036-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.22" data-doi="10.1111/age.12526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peripolli</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munari</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">M. V. G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lima</span>  <span class="cit-name-given-names">A. L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irgang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Runs of homozygosity: current knowledge and applications in livestock.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">271</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/age.12526</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D48%26rft.spage%253D255%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fage.12526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/age.12526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.23" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: A Tool Set for Whole-Genome Association and Population-Based Linkage Analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/519795</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.24" data-doi="10.1046/j.1420-9101.2002.00373.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Comparing molecular measures for detecting inbreeding depression.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">31</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1420-9101.2002.00373.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D15%26rft.spage%253D20%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1420-9101.2002.00373.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1420-9101.2002.00373.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000173865600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.25" data-doi="10.1038/sj.hdy.6800485"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dodds</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veenvliet</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glass</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Understanding the relationship between the inbreeding coefficient and multilocus heterozygosity: theoretical expectations and empirical data.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">265</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.hdy.6800485</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DSlate%26rft.auinit1%253DJ.%26rft.volume%253D93%26rft.issue%253D3%26rft.spage%253D255%26rft.epage%253D265%26rft.atitle%253DUnderstanding%2Bthe%2Brelationship%2Bbetween%2Bthe%2Binbreeding%2Bcoefficient%2Band%2Bmultilocus%2Bheterozygosity%253A%2Btheoretical%2Bexpectations%2Band%2Bempirical%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800485%26rft_id%253Dinfo%253Apmid%252F15254488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800485&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15254488&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223526900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.26" data-doi="10.1016/j.ajhg.2013.05.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szpiech</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zöllner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Long Runs of Homozygosity Are Enriched for Deleterious Variation.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">90</span>–<span class="cit-lpage">102</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2013.05.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D93%26rft.spage%253D90%26rft.atitle%253DLONG%2BRUNS%2BOF%2BHOMOZYGOSITY%2BARE%2BENRICHED%2BFOR%2BDELETERIOUS%2BVARIATION.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2013.05.003%26rft_id%253Dinfo%253Apmid%252F23746547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2013.05.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23746547&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.27" data-doi="10.1371/journal.pgen.1002316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vaysse</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ratnakumar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derrien</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Axelsson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosengren Pielberg</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of genomic regions associated with phenotypic variation between dog breeds using selection mapping</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002316</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002316</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DVaysse%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002316%26rft.epage%253De1002316%26rft.atitle%253DIdentification%2Bof%2Bgenomic%2Bregions%2Bassociated%2Bwith%2Bphenotypic%2Bvariation%2Bbetween%2Bdog%2Bbreeds%2Busing%2Bselection%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002316%26rft_id%253Dinfo%253Apmid%252F22022279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002316&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22022279&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.28" data-doi="10.1086/279872"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1922</span> <span class="cit-article-title">Coefficients of Inbreeding and Relationship.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">330</span>–<span class="cit-lpage">338</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/279872</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D56%26rft.spage%253D330%26rft_id%253Dinfo%253Adoi%252F10.1086%252F279872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/279872&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200856800029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.117.29" data-doi="10.1186/s12864-015-1715-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guldbrandtsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bosse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sahana</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Runs of homozygosity and distribution of functional variants in the cattle genome.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">542</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1715-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D542%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1715-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1715-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-9.1.117.30"><div class="cit-metadata unstructured">Online Mendelian Inheritance in Animals, OMIA. Faculty of Veterinary Science, University of Sydney, January, 1 2018 World Wide Web URL: <a href="http://omia.angis.org.au/">http://omia.angis.org.au/</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-9.1.117.31"><div class="cit-metadata unstructured">Online Mendelian Inheritance in Man, OMIM. McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University (Baltimore, MD), January, 1 2018. World Wide Web URL: <a href="https://omim.org/">https://omim.org/</a></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/1/117.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/1/105" title="Domestication and Temperature Modulate Gene Expression Signatures and Growth in the Australasian Snapper Chrysophrys auratus" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/1/125" title="Genotype Imputation in Winter Wheat Using First-Generation Haplotype Map SNPs Improves Genome-Wide Association Mapping and Genomic Prediction of Traits" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/1"><span>Volume 9 Issue 1, January 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/1.cover-source.jpg?itok=uUntglLo" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/1/117" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-og6U1n2hCswa4Y8XlwAcOKcFpIOoVo70ke3AdfwN4xA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/1/117" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/1/117" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/1/117" class="active" data-icon-position="" data-hide-link-title="0">Fine-Scale Resolution of Runs of Homozygosity Reveal Patterns of Inbreeding and Substantial Overlap with Recessive Disease Genotypes in Domestic Dogs</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491828" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-FOOW8TVBy7YsFAnsh4uU2J7631ohR06Xoi6MCipT6wc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/1/117.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/1/117" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form--IHYzw3VnEh3gfU30g4STx_3odUrqeS6UNIu2IEMTkM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/1/117" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/1/117.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491828" id="node491828--2" data-pisa="ggg;9/1/117" data-pisa-master="ggg;g3.118.200836" data-apath="/ggg/9/1/117.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fine-Scale Resolution of Runs of Homozygosity Reveal Patterns of Inbreeding and Substantial Overlap with Recessive Disease Genotypes in Domestic Dogs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3800-4459" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aaron J. Sams</span> and <span class="highwire-citation-author" data-delta="1">Adam R. Boyko</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">117-123;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200836 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491828--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aaron J. Sams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embark Veterinary, Inc., Boston, Massachusetts 02111</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BSams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sams%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BJ.%2BSams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3800-4459" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aaron J. Sams</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:asams@embarkvet.com" class="" data-icon-position="" data-hide-link-title="0">asams@embarkvet.com</a>
<a href="mailto:adam@embarkvet.com" class="" data-icon-position="" data-hide-link-title="0">adam@embarkvet.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Adam R. Boyko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embark Veterinary, Inc., Boston, Massachusetts 02111</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biomedical Sciences, College of Veterinary Medicine, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BR.%2BBoyko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boyko%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAdam%2BR.%2BBoyko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:asams@embarkvet.com" class="" data-icon-position="" data-hide-link-title="0">asams@embarkvet.com</a>
<a href="mailto:adam@embarkvet.com" class="" data-icon-position="" data-hide-link-title="0">adam@embarkvet.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/1/117" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/1/117&amp;title=Fine-Scale%20Resolution%20of%20Runs%20of%20Homozygosity%20Reveal%20Patterns%20of%20Inbreeding%20and%20Substantial%20Overlap%20with%20Recessive%20Disease%20Genotypes%20in%20Domestic%20Dogs" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/1/117&amp;title=Fine-Scale%20Resolution%20of%20Runs%20of%20Homozygosity%20Reveal%20Patterns%20of%20Inbreeding%20and%20Substantial%20Overlap%20with%20Recessive%20Disease%20Genotypes%20in%20Domestic%20Dogs" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/1/117&amp;title=Fine-Scale%20Resolution%20of%20Runs%20of%20Homozygosity%20Reveal%20Patterns%20of%20Inbreeding%20and%20Substantial%20Overlap%20with%20Recessive%20Disease%20Genotypes%20in%20Domestic%20Dogs" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/1/117&amp;text=Fine-Scale%20Resolution%20of%20Runs%20of%20Homozygosity%20Reveal%20Patterns%20of%20Inbreeding%20and%20Substantial%20Overlap%20with%20Recessive%20Disease%20Genotypes%20in%20Domestic%20Dogs" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/1/117&amp;title=Fine-Scale%20Resolution%20of%20Runs%20of%20Homozygosity%20Reveal%20Patterns%20of%20Inbreeding%20and%20Substantial%20Overlap%20with%20Recessive%20Disease%20Genotypes%20in%20Domestic%20Dogs" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/1/117&amp;t=Fine-Scale%20Resolution%20of%20Runs%20of%20Homozygosity%20Reveal%20Patterns%20of%20Inbreeding%20and%20Substantial%20Overlap%20with%20Recessive%20Disease%20Genotypes%20in%20Domestic%20Dogs" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/1/117&amp;title=Fine-Scale%20Resolution%20of%20Runs%20of%20Homozygosity%20Reveal%20Patterns%20of%20Inbreeding%20and%20Substantial%20Overlap%20with%20Recessive%20Disease%20Genotypes%20in%20Domestic%20Dogs" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/1/117&amp;title=Fine-Scale%20Resolution%20of%20Runs%20of%20Homozygosity%20Reveal%20Patterns%20of%20Inbreeding%20and%20Substantial%20Overlap%20with%20Recessive%20Disease%20Genotypes%20in%20Domestic%20Dogs" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491828/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491828" id="node491828--3" data-pisa="ggg;9/1/117" data-pisa-master="ggg;g3.118.200836" data-apath="/ggg/9/1/117.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fine-Scale Resolution of Runs of Homozygosity Reveal Patterns of Inbreeding and Substantial Overlap with Recessive Disease Genotypes in Domestic Dogs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3800-4459" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aaron J. Sams</span> and <span class="highwire-citation-author" data-delta="1">Adam R. Boyko</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">117-123;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200836 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491828--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aaron J. Sams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embark Veterinary, Inc., Boston, Massachusetts 02111</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BSams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sams%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BJ.%2BSams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3800-4459" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aaron J. Sams</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:asams@embarkvet.com" class="" data-icon-position="" data-hide-link-title="0">asams@embarkvet.com</a>
<a href="mailto:adam@embarkvet.com" class="" data-icon-position="" data-hide-link-title="0">adam@embarkvet.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Adam R. Boyko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embark Veterinary, Inc., Boston, Massachusetts 02111</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biomedical Sciences, College of Veterinary Medicine, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BR.%2BBoyko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boyko%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAdam%2BR.%2BBoyko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:asams@embarkvet.com" class="" data-icon-position="" data-hide-link-title="0">asams@embarkvet.com</a>
<a href="mailto:adam@embarkvet.com" class="" data-icon-position="" data-hide-link-title="0">adam@embarkvet.com</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=117&amp;rft.epage=123&amp;rft.atitle=Fine-Scale%20Resolution%20of%20Runs%20of%20Homozygosity%20Reveal%20Patterns%20of%20Inbreeding%20and%20Substantial%20Overlap%20with%20Recessive%20Disease%20Genotypes%20in%20Domestic%20Dogs&amp;rft.volume=9&amp;rft.issue=1&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Sams%2C+Aaron+J.&amp;rft.au=Boyko%2C+Adam+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491828/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491828/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491828/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491828/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491828/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491828/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491828/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491828/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491828/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491828/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491828/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491828/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/1/117" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#ack-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Author Contributions</a></li><li class="odd"><a href="#ack-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Author Information</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/1/117.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/1/117.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/1/117.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/1/117.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
