<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/1/217" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/1/217.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/1/217.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/1/217.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/1/217.atom" />
<meta name="HW.pisa" content="ggg;9/1/217" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Medaka Population Genome Structure and Demographic History Described via Genotyping-by-Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.118.200779" />
<meta name="DC.Date" content="2019-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2019 Katsumura et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7358042" />
<meta name="DC.Description" content="Medaka is a model organism in medicine, genetics, developmental biology and population genetics. Lab stocks composed of more than 100 local wild populations are available for research in these fields. Thus, medaka represents a potentially excellent bioresource for screening disease-risk- and adaptation-related genes in genome-wide association studies. Although the genetic population structure should be known before performing such an analysis, a comprehensive study on the genome-wide diversity of wild medaka populations has not been performed. Here, we performed genotyping-by-sequencing (GBS) for 81 and 12 medakas captured from a bioresource and the wild, respectively. Based on the GBS data, we evaluated the genetic population structure and estimated the demographic parameters using an approximate Bayesian computation (ABC) framework. The genome-wide data confirmed that there were substantial differences between local populations and supported our previously proposed hypothesis on medaka dispersal based on mitochondrial genome (mtDNA) data. A new finding was that a local group that was thought to be a hybrid between the northern and the southern Japanese groups was actually an origin of the northern Japanese group. Thus, this paper presents the first population-genomic study of medaka and reveals its population structure and history based on chromosomal genetic diversity." />
<meta name="DC.Contributor" content="Takafumi Katsumura" />
<meta name="DC.Contributor" content="Shoji Oda" />
<meta name="DC.Contributor" content="Hiroshi Mitani" />
<meta name="DC.Contributor" content="Hiroki Oota" />
<meta name="article:published_time" content="2019-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Medaka Population Genome Structure and Demographic History Described via Genotyping-by-Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Medaka is a model organism in medicine, genetics, developmental biology and population genetics. Lab stocks composed of more than 100 local wild populations are available for research in these fields. Thus, medaka represents a potentially excellent bioresource for screening disease-risk- and adaptation-related genes in genome-wide association studies. Although the genetic population structure should be known before performing such an analysis, a comprehensive study on the genome-wide diversity of wild medaka populations has not been performed. Here, we performed genotyping-by-sequencing (GBS) for 81 and 12 medakas captured from a bioresource and the wild, respectively. Based on the GBS data, we evaluated the genetic population structure and estimated the demographic parameters using an approximate Bayesian computation (ABC) framework. The genome-wide data confirmed that there were substantial differences between local populations and supported our previously proposed hypothesis on medaka dispersal based on mitochondrial genome (mtDNA) data. A new finding was that a local group that was thought to be a hybrid between the northern and the southern Japanese groups was actually an origin of the northern Japanese group. Thus, this paper presents the first population-genomic study of medaka and reveals its population structure and history based on chromosomal genetic diversity.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="ggg;9/1/217" />
<meta name="citation_id" content="9/1/217" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/1/217" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/1/217.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/1/217.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/1/217.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200779" />
<meta name="citation_pmid" content="30482798" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="217" />
<meta name="citation_lastpage" content="228" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Takafumi Katsumura" />
<meta name="citation_author_institution" content="Department of Anatomy, Kitasato University School of Medicine, Sagamihara, Kanagawa 252-0374, Japan" />
<meta name="citation_author_institution" content="Graduate School of Natural Science and Technology, Okayama University, Okayama, Okayama 700-8530, Japan" />
<meta name="citation_author_email" content="tk@okayama-u.ac.jp" />
<meta name="citation_author_email" content="hiroki_oota@med.kitasato-u.ac.jp" />
<meta name="citation_author" content="Shoji Oda" />
<meta name="citation_author_institution" content="Department of Integrated Biosciences, Graduate School of Frontier Sciences, University of Tokyo, Kashiwa, Chiba 277-8562, Japan" />
<meta name="citation_author" content="Hiroshi Mitani" />
<meta name="citation_author_institution" content="Department of Integrated Biosciences, Graduate School of Frontier Sciences, University of Tokyo, Kashiwa, Chiba 277-8562, Japan" />
<meta name="citation_author" content="Hiroki Oota" />
<meta name="citation_author_institution" content="Department of Anatomy, Kitasato University School of Medicine, Sagamihara, Kanagawa 252-0374, Japan" />
<meta name="citation_author_email" content="tk@okayama-u.ac.jp" />
<meta name="citation_author_email" content="hiroki_oota@med.kitasato-u.ac.jp" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0746-2729" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. H. Alexander;citation_author=J. Novembre;citation_author=K. Lange;citation_title=Fast model-based estimation of ancestry in unrelated individuals.;citation_volume=19;citation_year=2009;citation_pmid=19648217;citation_doi=10.1101/gr.094052.109" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=K. R. Andrews;citation_author=J. M. Good;citation_author=M. R. Miller;citation_author=G. Luikart;citation_author=P. A. Hohenlohe;citation_title=Harnessing the power of RADseq for ecological and evolutionary genomics.;citation_pages=81-92;citation_volume=17;citation_year=2016;citation_pmid=26729255;citation_doi=10.1038/nrg.2015.28" />
<meta name="citation_reference" content="citation_journal_title=Biol. Open;citation_author=S. Ansai;citation_author=M. Kinoshita;citation_title=Targeted mutagenesis using CRISPR/Cas system in medaka.;citation_volume=3;citation_year=2014;citation_pmid=24728957;citation_doi=10.1242/bio.20148177" />
<meta name="citation_reference" content="citation_journal_title=Ichthyol. Explor. Freshwat.;citation_author=T. Asai;citation_author=H. Senou;citation_author=K. Hosoya;citation_title=Oryzias sakaizumii, a new ricefish from northern Japan (Teleostei: Adrianichthyidae).;citation_pages=289-299;citation_volume=22;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. A. Beaumont;citation_author=W. Zhang;citation_author=D. J. Balding;citation_title=Approximate Bayesian Computation in Population Genetics;citation_pages=2025-2035;citation_volume=162;citation_year=2002;citation_issue=4;citation_pmid=12524368" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Catchen;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_title=Stacks: an analysis tool set for population genomics.;citation_pages=3124-3140;citation_volume=22;citation_year=2013;citation_pmid=23701397;citation_doi=10.1111/mec.12354" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. M. Catchen;citation_author=A. Amores;citation_author=P. Hohenlohe;citation_author=W. Cresko;citation_author=J. H. Postlethwait;citation_title=Stacks: building and genotyping Loci de novo from short-read sequences.;citation_pages=171-182;citation_volume=1;citation_year=2011;citation_doi=10.1534/g3.111.000240" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=C. C. Chang;citation_author=C. C. Chow;citation_author=L. C. Tellier;citation_author=S. Vattikuti;citation_author=S. M. Purcell;citation_title=Second-generation PLINK: rising to the challenge of larger and richer datasets.;citation_pages=7;citation_volume=4;citation_year=2015;citation_pmid=25722852;citation_doi=10.1186/s13742-015-0047-8" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=O. Chernomor;citation_author=A. von Haeseler;citation_author=B. Q. Minh;citation_title=Terrace Aware Data Structure for Phylogenomic Inference from Supermatrices.;citation_volume=65;citation_year=2016;citation_pmid=27121966;citation_doi=10.1093/sysbio/syw037" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J.-M. Cornuet;citation_author=P. Pudlo;citation_author=J. Veyssier;citation_author=A. Dehne-Garcia;citation_author=M. Gautier;citation_title=DIYABC v2.0: a software to make approximate Bayesian computation inferences about population history using single nucleotide polymorphism, DNA sequence and microsatellite data.;citation_volume=30;citation_year=2014;citation_pmid=24389659;citation_doi=10.1093/bioinformatics/btt763" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=J. Felsenstein;citation_title=Confidence limits on phylogenies; an approach using the bootstrap;citation_pages=783-791;citation_volume=39;citation_year=1985;citation_issue=4;citation_pmid=7288891;citation_doi=10.2307/2408678;citation_georef=1996015960" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=S. Guindon;citation_author=J.-F. Dufayard;citation_author=V. Lefort;citation_author=M. Anisimova;citation_author=W. Hordijk;citation_title=New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0.;citation_volume=59;citation_year=2010;citation_pmid=20525638;citation_doi=10.1093/sysbio/syq010" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=M. Hirayama;citation_author=T. Mukai;citation_author=M. Miya;citation_author=Y. Murata;citation_author=T. Yamashita;citation_title=Intraspecific variation in the mitochondrial genome among local populations of Medaka Oryzias latipes.;citation_pages=13-24;citation_volume=457;citation_year=2010;citation_doi=10.1016/j.gene.2010.02.012" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=K. Igarashi;citation_author=J. Kobayashi;citation_author=T. Katsumura;citation_author=Y. Urushihara;citation_author=K. Hida;citation_title=An Approach to Elucidate NBS1 Function in DNA Repair Using Frequent Nonsynonymous Polymorphism in Wild Medaka (Oryzias latipes) Populations.;citation_pages=e0170006-e0170019;citation_volume=12;citation_year=2017;citation_doi=10.1371/journal.pone.0170006" />
<meta name="citation_reference" content="citation_journal_title=Ichthyol. Res.;citation_author=Y. Iguchi;citation_author=K. Kume;citation_author=T. Kitagawa;citation_title=Natural hybridization between two Japanese medaka species (Oryzias latipes and Oryzias sakaizumii) observed in the Yura River basin, Kyoto, Japan.;citation_pages=405-411;citation_volume=65;citation_year=2018;citation_doi=10.1007/s10228-018-0623-x" />
<meta name="citation_reference" content="citation_journal_title=Zoological science;citation_journal_abbrev=Zoolog Sci;citation_author=K. Inoue;citation_author=Y. Takei;citation_title=Diverse adaptability in oryzias species to high environmental salinity.;citation_pages=727-734;citation_volume=19;citation_year=2002;citation_issue=7;citation_pmid=12149572;citation_doi=10.2108/zsj.19.727" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Jeong;citation_author=S. Nakagome;citation_author=A. Di Rienzo;citation_title=Deep History of East Asian Populations Revealed Through Genetic Analysis of the Ainu.;citation_volume=202;citation_year=2016;citation_pmid=26500257;citation_doi=10.1534/genetics.115.178673" />
<meta name="citation_reference" content="citation_journal_title=MAMMALIAN PROTEIN METABOLISM;citation_author=T. H. Jukes;citation_author=C. R. Cantor;citation_pages=21;citation_volume=2;citation_year=1969;citation_doi=10.1016/B978-1-4832-3211-9.50009-7" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Kasahara;citation_author=K. Naruse;citation_author=S. Sasaki;citation_author=Y. Nakatani;citation_author=W. Qu;citation_title=The medaka draft genome and insights into vertebrate genome evolution.;citation_pages=714-719;citation_volume=447;citation_year=2007;citation_issue=7145;citation_pmid=17554307;citation_doi=10.1038/nature05846" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=T. Katsumura;citation_author=S. Oda;citation_author=S. Mano;citation_author=N. Suguro;citation_author=K. Watanabe;citation_title=Genetic differentiation among local populations of medaka fish (Oryzias latipes) evaluated through grid- and deme-based sampling.;citation_pages=170-177;citation_volume=443;citation_year=2009;citation_pmid=19397956;citation_doi=10.1016/j.gene.2009.04.017" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=T. Katsumura;citation_author=S. Oda;citation_author=S. Nakagome;citation_author=T. Hanihara;citation_author=H. Kataoka;citation_title=Natural allelic variations of xenobiotic-metabolizing enzymes affect sexual dimorphism in Oryzias latipes.;citation_volume=281;citation_year=2014;citation_pmid=25377463;citation_doi=10.1098/rspb.2014.2259" />
<meta name="citation_reference" content="citation_journal_title=Anthropol. Sci.;citation_author=T. Katsumura;citation_author=S. Oda;citation_author=K. Tsukamoto;citation_author=Y. Sekiya;citation_author=T. Yamashita;citation_title=A population genetic study on the relationship between medaka fish and the spread of wet-rice cultivation across the Japanese archipelago.;citation_pages=81-89;citation_volume=120;citation_year=2012;citation_doi=10.1537/ase.110525" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Zool. Sci.;citation_author=M. Matsuda;citation_author=H. Yonekawa;citation_author=S. Hamaguchi;citation_author=M. Sakaizumi;citation_title=Geographic variation and diversity in the mitochondrial DNA of the medaka, Oryzias latipes, as determined by restriction endonuclease analysis.;citation_pages=517-526;citation_volume=14;citation_year=1997;citation_doi=10.2108/zsj.14.517" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=B. Q. Minh;citation_author=M. A. T. Nguyen;citation_author=A. von Haeseler;citation_title=Ultrafast approximation for phylogenetic bootstrap.;citation_volume=30;citation_year=2013;citation_pmid=23418397;citation_doi=10.1093/molbev/mst024" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Miya;citation_author=M. Friedman;citation_author=T. P. Satoh;citation_author=H. Takeshima;citation_author=T. Sado;citation_title=Evolutionary origin of the Scombridae (tunas and mackerels): members of a paleogene adaptive radiation with 14 other pelagic fish families.;citation_pages=e73535;citation_volume=8;citation_year=2013;citation_pmid=24023883;citation_doi=10.1371/journal.pone.0073535" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=S. R. Narum;citation_author=C. A. Buerkle;citation_author=J. W. Davey;citation_author=M. R. Miller;citation_author=P. A. Hohenlohe;citation_title=Genotyping-by-sequencing in ecological and conservation genomics.;citation_pages=2841-2847;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12350" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=L.-T. Nguyen;citation_author=H. A. Schmidt;citation_author=A. von Haeseler;citation_author=B. Q. Minh;citation_title=IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies.;citation_volume=32;citation_year=2015;citation_pmid=25371430;citation_doi=10.1093/molbev/msu300" />
<meta name="citation_reference" content="citation_title=Medaka: A Model for Organogenesis, Human Disease, and Evolution;citation_title=Human Population Genetics Meets Medaka;citation_author=H. Oota;citation_author=H. Mitani;citation_pages=339-350;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genes Genet. Syst.;citation_author=N. Osada;citation_title=Genetic diversity in humans and non-human primates and its evolutionary consequences.;citation_pages=133-145;citation_volume=90;citation_year=2015;citation_doi=10.1266/ggs.90.133" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=N. Saitou;citation_author=M. Nei;citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees;citation_pages=406-425;citation_volume=4;citation_year=1987;citation_issue=4;citation_pmid=3447015;citation_doi=10.1099/ijs.0.01472-0" />
<meta name="citation_reference" content="citation_journal_title=Zool. Sci.;citation_author=M. Sakaizumi;citation_title=Rigid Isolation between the Northern Population and the Southern Population of the Medaka, Oryzias latipes (Genetics).;citation_pages=795-800;citation_volume=1;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=M. Sakaizumi;citation_title=Genetic divergence in wild populations of Medaka, Oryzias latipes (Pisces, Oryziatidae) from Japan and China.;citation_pages=119-125;citation_volume=69;citation_year=1986;citation_doi=10.1007/BF00115131" />
<meta name="citation_reference" content="citation_journal_title=Proc. Jpn. Acad. Ser. B;citation_author=M. Sakaizumi;citation_author=N. Egami;citation_author=K. Moriwaki;citation_title=Allozymic Variation in Wild Populations of the Fish, Oryzias latipes.;citation_pages=448-451;citation_volume=56;citation_year=1980" />
<meta name="citation_reference" content="citation_journal_title=Copeia;citation_author=M. Sakaizumi;citation_author=K. Moriwaki;citation_author=N. Egami;citation_title=Allozymic variation and regional differentiation in wild populations of the fish Oryzias latipes.;citation_pages=311;citation_volume=1983;citation_year=1983;citation_doi=10.2307/1444373" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Zool.;citation_author=M. Sakaizumi;citation_author=Y. Shimizu;citation_author=S. Hamaguchi;citation_title=Electrophoretic studies of meiotic segregation in inter- and intraspecific hybrids among East Asian species of the genus Oryzias (Pisces: Oryziatidae).;citation_pages=85-92;citation_volume=264;citation_year=1992;citation_doi=10.1002/jez.1402640113" />
<meta name="citation_reference" content="citation_journal_title=Biol. Lett.;citation_author=D. H. E. Setiamarga;citation_author=M. Miya;citation_author=Y. Yamanoue;citation_author=Y. Azuma;citation_author=J. G. Inoue;citation_title=Divergence time of the two regional medaka populations in Japan as a new time scale for comparative genomics of vertebrates.;citation_volume=5;citation_year=2009;citation_pmid=19586967;citation_doi=10.1098/rsbl.2009.0419;citation_georef=2013074188" />
<meta name="citation_reference" content="citation_journal_title=Zoology;citation_author=A. Shima;citation_author=A. Shimada;citation_author=M. Sakaizumi;citation_author=N. Egami;citation_title=First listing of wild stocks of the medaka Oryzias latipes currently kept by the Zoological Institute, Faculty of Science, University of Tokyo. Journal of the Faculty of Science, Imperial University of Tokyo Sect IV;citation_pages=27-35;citation_volume=16;citation_year=1985" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=T. Shimmura;citation_author=T. Nakayama;citation_author=A. Shinomiya;citation_author=S. Fukamachi;citation_author=M. Yasugi;citation_title=Dynamic plasticity in phototransduction regulates seasonal changes in color perception.;citation_pages=412;citation_volume=8;citation_year=2017;citation_doi=10.1038/s41467-017-00432-8" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. Spivakov;citation_author=T. O. Auer;citation_author=R. Peravali;citation_author=I. Dunham;citation_author=D. Dolle;citation_title=Genomic and phenotypic characterization of a wild medaka population: towards the establishment of an isogenic population genetic resource in fish.;citation_volume=4;citation_year=2014;citation_pmid=24408034;citation_doi=10.1534/g3.113.008722" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=EVOLUTIONARY RELATIONSHIP OF DNA SEQUENCES IN FINITE POPULATIONS;citation_pages=437-460;citation_volume=105;citation_year=1983;citation_issue=2;citation_pmid=6628982" />
<meta name="citation_reference" content="citation_journal_title=Zoological science;citation_journal_abbrev=Zoolog Sci;citation_author=Y. Takehana;citation_author=N. Nagai;citation_author=M. Matsuda;citation_author=K. Tsuchiya;citation_author=M. Sakaizumi;citation_title=Geographic variation and diversity of the cytochrome b gene in Japanese wild populations of medaka, Oryzias latipes.;citation_pages=1279-1291;citation_volume=20;citation_year=2003;citation_issue=10;citation_pmid=14569151;citation_doi=10.2108/zsj.20.1279" />
<meta name="citation_reference" content="citation_journal_title=Zool. Sci.;citation_author=Y. Takehana;citation_author=M. Sakai;citation_author=T. Narita;citation_author=T. Sato;citation_author=K. Naruse;citation_title=Origin of Boundary Populations in Medaka (Oryzias latipes Species Complex).;citation_pages=125-131;citation_volume=33;citation_year=2016;citation_doi=10.2108/zs150144" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Tamura;citation_author=D. Peterson;citation_author=N. Peterson;citation_author=G. Stecher;citation_author=M. Nei;citation_title=MEGA5: Molecular Evolutionary Genetics Analysis Using Maximum Likelihood, Evolutionary Distance, and Maximum Parsimony Methods.;citation_volume=28;citation_year=2011;citation_pmid=21546353;citation_doi=10.1093/molbev/msr121" />
<meta name="citation_reference" content="citation_journal_title=BMC Dev. Biol.;citation_author=T. Watanabe-Asaka;citation_author=Y. Sekiya;citation_author=H. Wada;citation_author=T. Yasuda;citation_author=I. Okubo;citation_title=Regular heartbeat rhythm at the heartbeat initiation stage is essential for normal cardiogenesis at low temperature.;citation_pages=12;citation_volume=14;citation_year=2014;citation_doi=10.1186/1471-213X-14-12" />
<meta name="citation_reference" content="citation_journal_title=Wiley Interdiscip. Rev. Comput. Stat.;citation_author=H. Wickham;citation_title=ggplot2.;citation_pages=180-185;citation_volume=3;citation_year=2011;citation_doi=10.1002/wics.147" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=X. Zheng;citation_author=D. Levine;citation_author=J. Shen;citation_author=S. M. Gogarten;citation_author=C. Laurie;citation_title=A high-performance computing toolset for relatedness and principal component analysis of SNP data.;citation_volume=28;citation_year=2012;citation_pmid=23060615;citation_doi=10.1093/bioinformatics/bts606" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Medaka Population Genome Structure and Demographic History Described via Genotyping-by-Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1.cover-source.jpg" />
<meta name="twitter:description" content="Medaka is a model organism in medicine, genetics, developmental biology and population genetics. Lab stocks composed of more than 100 local wild populations are available for research in these fields. Thus, medaka represents a potentially excellent bioresource for screening disease-risk- and adaptation-related genes in genome-wide association studies. Although the genetic population structure should be known before performing such an analysis, a comprehensive study on the genome-wide diversity of wild medaka populations has not been performed. Here, we performed genotyping-by-sequencing (GBS) for 81 and 12 medakas captured from a bioresource and the wild, respectively. Based on the GBS data, we evaluated the genetic population structure and estimated the demographic parameters using an approximate Bayesian computation (ABC) framework. The genome-wide data confirmed that there were substantial differences between local populations and supported our previously proposed hypothesis on medaka dispersal based on mitochondrial genome (mtDNA) data. A new finding was that a local group that was thought to be a hybrid between the northern and the southern Japanese groups was actually an origin of the northern Japanese group. Thus, this paper presents the first population-genomic study of medaka and reveals its population structure and history based on chromosomal genetic diversity." />
<meta name="og-title" property="og:title" content="Medaka Population Genome Structure and Demographic History Described via Genotyping-by-Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/1/217" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Medaka is a model organism in medicine, genetics, developmental biology and population genetics. Lab stocks composed of more than 100 local wild populations are available for research in these fields. Thus, medaka represents a potentially excellent bioresource for screening disease-risk- and adaptation-related genes in genome-wide association studies. Although the genetic population structure should be known before performing such an analysis, a comprehensive study on the genome-wide diversity of wild medaka populations has not been performed. Here, we performed genotyping-by-sequencing (GBS) for 81 and 12 medakas captured from a bioresource and the wild, respectively. Based on the GBS data, we evaluated the genetic population structure and estimated the demographic parameters using an approximate Bayesian computation (ABC) framework. The genome-wide data confirmed that there were substantial differences between local populations and supported our previously proposed hypothesis on medaka dispersal based on mitochondrial genome (mtDNA) data. A new finding was that a local group that was thought to be a hybrid between the northern and the southern Japanese groups was actually an origin of the northern Japanese group. Thus, this paper presents the first population-genomic study of medaka and reveals its population structure and history based on chromosomal genetic diversity." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1.cover-source.jpg" />
    <title>Medaka Population Genome Structure and Demographic History Described via Genotyping-by-Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"B-13ZnXqjUe13L4BDkEg4sW7ROLDG45muM1klGk2Doc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491872","apath":"\/ggg\/9\/1\/217.atom","pisa":"ggg;9\/1\/217","ac":{"\/ggg\/9\/1\/217.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/1\/217.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/1\/217"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/1\/217"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/1\/217":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022F6VMGZ6ioTM8DJMlqFX7bhwXFnqBCBe1fH\\\\\\\/MKe4BqmdEOVPXoQD3Gs1VbowyVDM6zkyfigrwz9aW5JMs42VKWoxx5lellDAD6V3REqf7xKA62bowoK+xT9r68J1zKMQUunSCd9\\\\\\\/rTClOlRPuo2VWHnfEuowDd7Bkib5v1Yc6gbjz2jsBCCiq1vtQ8DGGufkPTZfkTeBmvi\\\\\\\/3dSLBowMvRWx3csNYqiSiIcH5eo\\\\\\\/vS25nMLFGYJWsUije+eJY7pl8sYANM8lfFQIcKFiUifzSTz0BOQtkHsZ7T129\\\\\\\/ZfRSfsYWgW3PN11BguBR9auzAdEHz\\\\\\\/1AEgmcv+2saVqGp5tChSRvc12b88Njq7Ix1T1qNTJ9NPcfcqtnvR9xRtVWqOQQhdT+AQQXr+s7oT3TLzj6ddPeDaBprlATjciqAUz9xuk\\\\\\\/QWAzcz1Z\\\\\\\/5x+B5dA6eu0rGfE9WZlpQAcIu69hQV77679KqocLBQ3kdLewgHL\\\\\\\/IX+R70DJaLm0vKY6vRu\\\\\\\/srN0whIFhw4nflbB+2lXFbtf0VkHI0hmUDrzJpdKjF93ey9BKbw\\\\\\\/f5acD5uKqrh3KxBg9xFaIx4ne\\\\\\\/w5LKOZMLSsFjnWUP2BMMTYEjvhnuBeRW5x9WDOvafhg\\\\\\\/NKIbG0Kj3hIiNBeqrhBbdoi8pALY6lHUUDggphloPYEdj5+yxkh0xVoW8LNVjlqnjU06+YalB3k0dMD4XbUeQlnLMgD7oNeX+cxd\\\\\\\/Aw6qUfnlEqZ4SM35byqK26n\\\\\\\/EZpfJtZIad7CmPsxlfe0QYD0MbjnSOOrfIlUx+d+MMT+xxLNO1UyLTNuEfQsOb3eBzSUuIE56nYHtzU2F7rVAPGTWJOxeOL1PEe3rgCO9nmMlvWduAGXmSR2sogARknxElU4CO+oyIwh24L0Ph3MsL5XxQg2WcU7mv8d\\\\\\\/v0z0UW6MHjFQrWZLQnFqpTYcv+b2wyNzYGM1UnydVjH7veLqPKFyEPh5g8J1Rpt16bBxOEs7kzeIpiaJzTg\\\\\\\/pXbhHI0zJisnrVMm32+S2125p5924ysf4aQgV34iF2ftlbtFR3EfA1rW1UJYEHJTrI1uH2wwiw3wCeSpyZTMbqOA\\\\\\\/TM+pcpB+jsOZXsU1zJNA+FTNWk7g3Rw9rEPEK0SbAOXsx+Qan4j\\\\\\\/fJyKa4FamZZKIdxLTPJALtlVaARyGEz7utlHvzvMO4YnacpJPUjIWi11FE3GMnNq0V7uTwMECqCH1K3cxqPclnz1a8ENJ7DkcIg75wCUj+5MqlOntKZpV2RoaRNuHVw57OYEi\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/n73E+DtRPGngcB8sN4Azw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f07892b4370cc3b9c6f4f127bbf08ec892d398bec8d4fdc25f423b320c5a8ba5\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mTRwsz5HnhxeXRIL4n2ims01rQ2917wK5F7h23l9rvwC8hkrpuxs51ST0NFwzsTWnSYMxvTMSwlR++xm0UCRwqAyq\\\\\\\/eQsFHtyMrwCK8B\\\\\\\/als\\\\\\\/\\\\\\\/L9NVs45TfxN\\\\\\\/ZczWwaz\\\\\\\/\\\\\\\/PWy6o7PA66JsF5mc1xxrrjrkYHbMPPEBKpAclFkZbKOqUgs5hv4qyMD76\\\\\\\/YFKnNfumOSym85jYbPBZW\\\\\\\/ueJOSenr\\\\\\\/+jc8+bZ6altWPexwVt0JAzptM0D7R4C60j5JcEiZEESM2cT\\\\\\\/Kdla6c6lUoft2EDf86qTxvSPtCsbAebJnlb48pU+vRLpYc+3jHDeJc+uawfMBC6MsgzJgzH2\\\\\\\/Ckczz+0xutJ2a8nel7vd1nvA2xHQ+xWAU\\\\\\\/\\\\\\\/s1RmE9xpF2\\\\\\\/QvhSWWBl3lokPyXRgdMLEi9HvzSjtJFJ+QsRSNlQfxmGPPNQbWjvhtpfE0DyxEt3DL3jObIHjVTyaODd5Ydyua0KPzU9t8EHFJAky9knveTQrwBcyNCdyTL4Vv\\\\\\\/UUdCD2CN2MTHNw2GzrhVhBUnb4bZLvlZ9GPJ06wVpcGWxMpCZNuu5mCrkyiyDJj8NzBsDsD+T9SwG2qeQlU0N2fVcTulSMiq1ZlwZfIQqSSDXkOFbVMhIiTWpF+cJHYJGccRc2kcxSby1qp3PUMusbZ0ZG3r9JQekVJAjh4bz5OwCNscP7sxVArlvfaKz12H6d1IhymZTSQWEsd4lsC4qge2XRA6pxBrOS04rfzlReg3o3Ro+v8G5nR7x28GlDVEuGZgSoDNwEfSz35HGasyKpumbkApMuQkU3jn4qyL1QsGrlnxcX44GU6L7zg8oBSXfyfp1kRUUELSIort2g0b81yFK8GLs0pVCqBasLkslvKegL2FoReYlUZyade0U+ehGkBbDpmP8UZWVtCrCIvjSuek8Zmf0CjWnC+E4nOKTg2zWykiLzZC8DGeggbMKzZTBFzDenKiz+bEyKTpl380APYd65eU7DETwuCPSzkV3UHx2nhxcjWvJnaLXzSR3Jk8qo5a926FQ4Air\\\\\\\/tKw2huPY5Cba+ebG\\\\\\\/C4hXj+jTcPVJSFpchXLS1hCXFjWeffHAN77z9\\\\\\\/cYms0SV79TgtMIUBqt6OAZepYoopghoPv6WS5mTGOeN4wkm2X9\\\\\\\/T\\\\\\\/0IHGTaMmzKV93mFXbt0sZxI+DyR7AZqpFwagHavrffhm21ND67nOoT1kFsCMylmujSYMHjph6DRvq+cwBlKfflHLI46gejxPWXm64tzPtpAuiKkX\\\\\\\/owioTNqXi\\\\\\\/QnondDvGDHXVNjyyQ3z\\\\\\\/EC1Cw1WTF3Dk1fzhChULUgWk6TaSG3BOMyHovrLuD69sS9Zp6XVmHrRAK6srRqqWLNCdQJ++TeAhvEOTogGFVcgAd0O2VFu2deH77jTNzsosZhe2Pi75yrt4EaLit7qGZXlKIFYeOMMf10vKbWGtyaTjc8Q74jymFB0I+s3QjVaBdOkaZ8CB2ozK2FavtSiZpCKZsZRJBkshJimhlw7bvObY86Vlxq1LfF0y\\\\\\\/QsZZ79xhOAJtwfNNSHqXpKhKWJ6EdW6QHbPqHogH5qyA\\\\\\\/PtgBepLZ9iiCA7zICdLjwslKdN6KcB3XnT4JFLyI\\\\\\\/NYh4BZFov3dsZiFrrdvShGTFa3ytyY8NkvbcB3EDtB34gSeuDFs7oHF3iFM6kMnl\\\\\\\/w6gflOVMOoWFJhYo63guhCVYeFhMNQoHQqaAScQFerQF6TjkyWKbB46EIn4u4KcXkq2k1udEHy+NvlCrSG1xW5swN7E5vu3Q+wrVJ9UJEh11mHsxYJbjqxHLiRPmiAFvIB78fFkv1KoNPvhsfeOJVC9KFjItp3Opqj+iVONpYle0GjEl4Yy3TX+Tvicmm8T7O7N8HHK4zuTomLjvrSXvnlpP\\\\\\\/jWCm90ZtOqJqhKU\\\\\\\/pRTYc8ucr6+rMJ+CLItEto9FsVdWKAxp3XhGtVhVbdQ15vHliC63z9lhc0CUJBt9\\\\\\\/jVj4y1dNWo6gedQiufi08nvLvhoT45lI\\\\\\\/+QpsaqXJu3C1spmbRsIby6OukDUy\\\\\\\/w1Xfmn29cBrIL0sdVejZ2rbHrL02JgsiqljLBwOXe2NTQvT5XlI0bhjfW89OMxQXVexap+khCdnOZgkrGInaDvsI4n0bQ3x02K2Ez3t92zmlYP\\\\\\\/wUR1CgfM\\\\\\\/+\\\\\\\/1oCv+BfhTLTFmqAwfXNP12\\\\\\\/U3NJc34e\\\\\\\/snCnK6fjDG+QiYaJbnMfF4trFa7N\\\\\\\/Jqgty8RT61xdmtOkUe\\\\\\\/zojGN0XQyCP3z3j8Rv4d8zz7FiVe30NWVhI+Zl07+o+VSi\\\\\\\/oARRJI5Ognh+UCd2SrarRDOgztdqczB8ChnQKmLCmF1bmps7ZEt689hWEnDG+ZVVuBLpA6SA++f60kVNHdGsZyINz2L5Cq7Yt2ZcDZaUW7owUKiAQ8IishOk\\\\\\\/SFq2Q8ewUFsnv1aQVljcPv3u2BegSSiMiI7TczltRuXdcB+zxcvkM3G8D0rAw3WI5KANUbccfIoIGjJZ5AU119n8y5ej3H2S521BKNP0fQnh1Cb\\\\\\\/BZKE8IzvbmGwYBzWLNUktfL0mYb+Kr64eXgonogAjDimFYR7Nva3QXAsu0k3aKW7yb6r1ps7rJ2pwsEpPPp7AIzdA06gtMvBa+hGgr7iIKZ5HGySqQ7iXUbLq7GaSxB7r6SJdFHbkGChFbrwMDYk+z9PH\\\\\\\/9Rw4NMSycpas012a\\\\\\\/G+iCiB7FQyKkeVpp\\\\\\\/UK7kD5gFjpGrn+dritSW1bZetetTnu+HrqAIrII5cM1hSL\\\\\\\/E1j2B82LFB5d9vJKEq373S3UFQ2MUxM+yrI8TtZZm8WIU89djiwWRxpvpxwdqcIQyiGTB94BGzulBM5C588Kom7yFCH\\\\\\\/tfuuhjaPQCfdjxwc1l\\\\\\\/Ung2NLBhWKZVyDaCJBQSSr50XmphshOt9bqjKsKVs9T+0HUHaH+PxSEu\\\\\\\/xNAfx7Bl4GKsl91tHeOfYW4icW\\\\\\\/7WCs0jNOZjXDttQmwJPMDnzqhGTW6oT9kb5DUzfu2Ipty4GZjdtoJtlaETFUDNNpQTQLWmPk54l3NCtEoy1aCrGrYKUgsCz2IZDG6wbJlwefPw8zVKsOdKIvcwuOXxe6jMbNy3BK\\\\\\\/yMrsRFuOKlvPjlP9QykgEaPMa1PvN4C2WYATFfpg+LNlgUrxwS5ESAbDLStamvPsGjDjK67rjjZVdJ4FVy3Q8t2Q1eYp9oTYxJhIYIYoRlk65v2gPBHOiyYXJ3+C4+46Aaip1nu01f\\\\\\\/ip5YweBAMFPSkdxPcBfBVJKdu6qUIYRmCqVG8ZoNpgG8tF4+6DlEmNfccAatirv3qb6SH9JzbJtsvTrhOEZ76G7ize2pxLrAiphWdY0hQOsr8yMEr\\\\\\\/R+NcZNbYWzkPO\\\\\\\/vlc+3Yq4eV98VXLLdqmRRUfKzJDfsS7Ey8wKy9M2FPYeWgMM6X3ztinEeGjyYhn3EK8rkASa3YKILZEzUGfgGNh8iXPpBAkJMaSv01E+Ne3XTSOu3ZimS3GJRXO8iA9T0O2mHkkKhqItCMys5MPf\\\\\\\/0eLkvTzJhEOwWCjiQnOZe6pteSf7mOeIYPKjhTO94rqdwqoYTxifKPS\\\\\\\/lwK2NWF2Sq83Of4unFJf4WuDTxpUoa3oHlRzduvVVKCGWYko0a\\\\\\\/Rx76Sr6EAT1qKHT5OJC1ewLYfbcK9HSW4AU+xImY4czKFo0oSpvN2kGqiNVzLT1iCT33Em0OQ6xpHgFOznvwQQ6nBdL7Xh67yd1AR9BNk4a\\\\\\\/QLtuLdikMT5tNYSzOIHA0wze4RnvyvdRvkQa5EtU\\\\\\\/nvrnzpvwlmt8BqjqdiXhrGyvBtiJNZncSf4C+RV\\\\\\\/qrouIKkp6g7y8ryHHT3uQEhYEwtnbQpS2aibSKMaq\\\\\\\/pjYtn6mVdOrskJpbom5K3wiITAgjh4oUJisV0HPXvBQbqDbSuhOUmojrMjywVxYgQFpGbXkRxUv3Y+2ki2UVRIILQhmuoQnd06HY\\\\\\\/1i7bXNxQHkX9dXWkzn+c2xMIrIdD21yV\\\\\\\/n0fWoHv6xo7NIHCnU6NgX1rPJC7nKFY6OqtqRPGP97JRVoDd42tmeC18qISRfsGBsunaV\\\\\\\/mqmaStatgbmNCIATknVBLTx5FWG0N3W24SjG61uSadZRBD1mhUlJybb9c4bqyQl8fLtEWYJF6jwb3GPI\\\\\\\/+58r5uM1ZmTwe1vA34m1vp6BBJ\\\\\\\/jFTcUfEAk3m7zdGnrOabhXgwu\\\\\\\/gPRGCFgxej8NzcbzGo52PDj32615M1JWHAFyws3U7\\\\\\\/Qwgo7Z4z7klWXStEWkYbqvIhRXV\\\\\\\/iXlTY\\\\\\\/rtFqzIxyb5XATNOh0nMRiYsZaVznHxH2Vr3rYN5ygROnxjNCI3o4aqAyzS03YhylYi\\\\\\\/PxI\\\\\\\/frLtpY8HCO4cOQaYU4OkNRZLqsjWKdeSuullr79em0NBQlKLZ3uuX\\\\\\\/9ndnDLVj6LvyNPqT\\\\\\\/V+TquCd3irhthRPfQjpmx6+Jk5zKvI5jidjJV1kkLYdt0pVjZdk1lfbmmEu54zMtToM5QLTjQAgc9V2aU\\\\\\\/1fEg4K7nTH5vCiFrIqw6aBmzbwxOgZovgN4Pbm5OZIDJmejz6ou2dHqaxFJgHQ85GHcwzpeUdNwY\\\\\\\/FQRwbitC468Ghp1tdndPi8W42yzQRFW9etnVm5qSpa6PYMEoRyrEwAWibjshufotTc2iF2oOawAvXzj1dhJB9avdXvwiShhFSOsAX3SQG7jybcshNvtw9bR+mlL1QNUpbtHdRoFz3c8r3alH5eiJAJFTUhIrV2Mvm8znoPVP0GMbJ5K46mh5p1KPWImAElTbF6Y1mUJ\\\\\\\/hmgNibMnEcaa+ChmAiCXP1siAF7d0K7r4T66DuzCdEVo+iCYJ+vnqgXY5SXDRnSv8954pi7UG6gEmDmXxS8ax83KpeV2API\\\\\\\/Eo8WGItdHWvYNArHnNV9erb08pWhxrP7H27bN9wVHEqoOMWDOL8roH634srzqif5wzwFz26KFDIFn2zr7XR9YQNtWoK0rV39HADJBsM1OZAfBgOtB\\\\\\\/o5QtBbh5eyC4wS9ulr2WB7X8fsVddyZkwlsrzgg\\\\\\\/eCGvuYA+iOBq6AxzeRCeiXlGevsnOdlrO3\\\\\\\/QxLMV0SFQgrduKZpTxUuEGBOuQxBfVmHuRwrIgV2wNBp4RmsMhTRfHAoC0SImzIXt+qk3vxNADPHTkwwA9KHJAu7KxAwLQymAozUXNzhOFkgrtDHJ7GCoskLMYD1tPkcHu5bcPnT7OJqNBdtfd2FQBSCUQdwaOniEjYD9gEzIAPqrZIPnK5SJn20eu\\\\\\\/pUlwNl4ny8DMsMI9OtBQD7HDJBZQUzTWl7QMIe66h8QnT4FQEwak36JFc\\\\\\\/5qtepmcvRGkf\\\\\\\/HA3acNyeERMfVi3NWD0rhXBw7RL8bhHLJGB7NENFRzOTUBALg+uE+mX43GVCWP8jppIr\\\\\\\/bzN55+lKSvOJH8lfM5vPqH1YIGw9WJv3dMoDq8uP1AiNZQCgY0L10uVXoNdtgoQwCcW3Im+KGfEfTVF7ImAO3YoIQrEUdcAfgLWzK2vSKvXDCqI7jim4O\\\\\\\/u0fFucNmKGCyk7jf4dijjRP8yds3GacAZuiIdJJKuBmk2YLmsE5jQ24kCRmoEWc93DXFcKdAjqzMcy0we8cE5kkXox0deQcEvg1Nius4w9AzIiL8RdQ0SWB1kUDIoYBGjqG8QxsQmwbC5Xz0hxGjKLZrPlB7s6FgAwUTWFfbJS3lLKJGNr6T3P0oQxXTN7FCDnOyWPoX6CyCcXHubwG3TZ2Yu+ulvB2MsNWAPoiuYT34PWdN0nA\\\\\\\/khg9Jc6A\\\\\\\/C+jNlPMojx5YyE2tMMoz3jjKVQQS5H0bJhsmu6I7STsK0EUqP6ySDT8idkyF\\\\\\\/TeinHIerCtnfT7qPRvKbuU8MfyzXIsi3NDVJEyzzP9eOjDyK5aAsmqRSd17jZhXiHwkguvOFnjvJfnLGJ8I\\\\\\\/YoAGcrxw3\\\\\\\/MMecUvmYhhV72qhCR5+AM6hzd1nlKN+E\\\\\\\/aGil41JH7vq44sUkCCz7luL0pjGpYiCpJyNVDR6jwqog2bBRZfs9uFea4ZU2E2LfNpW1ID7uik03m9ww0mrkWbBquqs+3QPyKNPx\\\\\\\/+sfjxZolZx681BiMHtw6vsL+BpPAugbwFXA\\\\\\\/WAD+ZajT4Zto1G7wkpren8LQpQ5Q1qBs1E6SJQnF05H2gDifzuNHR8g9jZ7dfBlv\\\\\\\/0bMaLrFKlSW5Gl41J9AJCPTwuvfrQL3IBKTeEpTktPD5sI1TrI\\\\\\\/mS6Mn5MyZNVdYR\\\\\\\/MhCbiaDNHipkrpVpox5YKRFVajFGyGdeO6g6DXQ6vKQpfvJ70f73BvMEsMOOk5+ia7VD70zDB8V3V9FGeqbCgF8KBIiz\\\\\\\/46Nd6c36WSbi8wFUtzPJl\\\\\\\/IBatBBMXKlDFCwPeIDa7lCoQONWVjlHuQXqwyeoqCshl30mXHq6QTZGfe1UEIzry7Wxf2cgxC\\\\\\\/gdL8b58G9U2WkuLntmSQLPM90wqrSRUoVo7P7Oulnqy3HzdDIcBH1SO9Zp5y3wlAKJvH0j9QKOqe5P5AR4OhjFcxhNuuiPkRaTKO9yOfAhcT8MLBdXqiXZ56CcP7wI7FvLpWhfHwD01nTX7SOo3bCl7M85jDfvDNGDVfvoqDJWzBUyyOC5vJw1Z4wWaR3ExFEpXuon95k+CAbB3ssGuQc9IlRVfm5qUAtvBTo6r5x\\\\\\\/bBsm912uBJWr8M7tKr3M\\\\\\\/\\\\\\\/fGShiA9Bab0l2CbwBDKDvyuBwd9ht517UVly3BlWqJGQagBX+u2VBPe1VVT8qmakB8LLabCCdQlkcjZFVe9zWaRMfVAq0BuLfBWpcwR\\\\\\\/XJi8n4jd14ifIS9b9+LQ2Q9R8s4M2dgrapp8qSLRD1hP9dYLOjkEv3UJPPngYubjeK6+pYGIoFv6j9WULdNosyqZCTtn3UncPP4bJ7lBJilB1OMaQbAS2Y8dtpnYlg9Wd4BS26MvyvszJ8cmSIrNzjrHb+Mdd4kZm26rvHd4E3uk7ferpm3mlclW4zYujp3WHh3Wk0ZXpYqSROxW5xpa7xN\\\\\\\/C5sXmmYW53zXwKtGwCqlkwwIVAW8yM4Y\\\\\\\/gkWLWcOlZlcatK6j0ewqvsAvCA0qZUhm+JLNwUpgdF1KVvlhvYTppEpLpl8t55zoUg\\\\\\\/SsqdY+Lt+3RVtrXaxb3NFrZTnZ19o3w8TUNhXyRRmSgoRxJahSboKeomvnDUmm+jG4HRYqCFrQqrv\\\\\\\/z1ViZlkUQp4MDlUwzjLHeX9XfaHsx7PwTTuL9QevQcg5hGlcuMcYKoq\\\\\\\/hc74qo7sGbOg7GB39jAPmah2aGLKWebgMtnAPlzeFcu+qqNB5vqguAyw4wJBzZXpqSABGbX4nzPotf0VIrdu+iAXzp\\\\\\\/HyeZF\\\\\\\/5nzlwID8OOSpEYYTMZn1gKQOlO8Q9tDQadVepDmgqWeA3Yvfox3NCGTVvOsaOQS9O2ackmVpBPRdwteBD0\\\\\\\/dCQ9tTGkel19hmp1mo1AOzuo1WmW4C5D5rWqsFyY0etmQWqCOJZMr5ERbGSR+pNuIpz90cjecPqOBcQKlSzeqCASiZ4+LFyc3kr\\\\\\\/g0YGqkhUCxfdFtrgi03XT\\\\\\\/7LnjsvuwbCJYFPyrO0OrsbtRKLRlIS043uqYrr+lrG+5GmIr+GTfSGslamEtvRP4tuxJmBM8m5732u35eRr8Zl3OWdwno9rYx\\\\\\\/frljIBBjA2Vs5I320gfOddEgf+VhemwuSBTT5GWCpVqkVpoAIzL7dx\\\\\\\/S47mPAq0ObRGbcEOoH6WcmLsC\\\\\\\/TjwhYNtJEXsZo3cHP95fbn\\\\\\\/8foXgIFwPp6la9gVE\\\\\\\/kKeOrbQOIu48LazSHaccv5fx5E9htJAPbnes+uypUIEdK+WjuAuJWRpUUgayo8xEdqPY\\\\\\\/\\\\\\\/jTQ41ooWLkOFsoMysJIN6nY4vKlM4b1dbM6270Fb4Qg4x91iKAwD6KZ0W97qlDvKCQoxNrfUMdAg2shh11b5akSzzt7ix6Z4DJN7UMI6SOFHnMNgXPTM243pSO5iNjPdyemcI7E4NwVHNNvpfgduyMP8Y8uyIJJ14ecFQnSbgAlzskIVSo8qJXw0Y+SYxH+T3WkSm4GLUkTTY8xp+sO+OJoPuHkd1akUDe+0+JgAIU4FcT5c\\\\\\\/0KLcXySn36U7Rkd9NT\\\\\\\/rcCZ465qhJ+Jp+uvWl\\\\\\\/Pqkt691WIctB5eIxanqje+J6j1cmwO4cijttiGS9ymK9WjKLG5juHs\\\\\\\/VKLL0unHe+N8gS9B7J6axhn8oocILHu0OQqohs\\\\\\\/stWot2Ymph90mp+EWA7cmn+PFYtSo8mHi+hQ6SN68Q8YBJNMqvROjAWep3Y96pVRMNBDt0t+VWrO2kTPXX+B3pGckd1vXIJFxb9rHhmFxLiQw0mwGbZt6oLxvh5Eu\\\\\\\/l9otQQN1HhViYdvF7wDMrDiYQuiXMJBviAk13ElPfrN6dBmkz0zsOTBrtGduTfIRw2Wh9Zs7osxlFYAw+WsN+ru7J62nopRMma\\\\\\\/wsNIEMQ2pEAdjmVq7l\\\\\\\/uwqBkPj65W8M7sYblxetSoB5l9qiwqxIMY4FOFrQfppTo1FCf17MuqYYWsx\\\\\\\/qRe0bNhT8SAJl9F4lwEv7GrzhbFQFrQnFVVWttD8dDNXHQJVmzXvCMCDjsnY5xs7UsH974VWf5PgIE49Jl4fA1KAW3\\\\\\\/HWSMUwIYJ+u4\\\\\\\/FrdqqfCck4A+nTorydlUCwJuH0pHFmyMRVq33x5Hev1tKF6+Z1qxEkTC6epPGV1XuVoH+4D1R40rLPwet9Km94gs3CRl8YKM9MKOWeTYDrASyAytmdXOkOXH1hEnVUbe8GevEpSMEtaz2HTv+n\\\\\\\/7VNBD6Ob\\\\\\\/moGYBLMJSJspOoTh0QIq\\\\\\\/pql8AF4Jn7p2bU\\\\\\\/L1+VbDgOmLdWGR5Hwrours8WScDYlyODVa68RsKxrDJCcO7fe0iXMPwGXWVCjiXcIS7AmELpWmRGqYvVNb9IZz0nrvSIG4b4ivaYBxAyY\\\\\\\/zoyDMUoXPldmseMenefQZSFLL3+0as21vKkf7ceAQZWZY7xxf3gtKPf\\\\\\\/sJ+FM8MVHfb4fYU5nIsfYPi0GLe8aVqpS8M0s4CUJiCUQvkQliRtl9+d+8IAfkFc6I2ZCBYAd0f0mjMUMNi67EI\\\\\\\/2b32lkCV7V2YzpScs44OxVkiuo+OR1umAXjGF4HdQ104TTGqHmq591lBdYJHvyH45aa8hK1rY1uQ6k\\\\\\\/aUN3zE876D5fMkmLBXfAxrfvRvAAL3kiIYcKPe2bZ1PWwTjxtwVoyZRh+TwNrbzYIUsSmg+LzFy4uU89pZVbIbxr2M9i6z4\\\\\\\/VHDmORKtFbqRar1ccRctZtV23sEPffOjN\\\\\\\/8nA5u0L9t\\\\\\\/hShXxyYmd6oSo2blnR0Z7\\\\\\\/nSDJasQ8CmGORBjb5oKfZBKtPfaNR8WZKoZH9B0PfVx2YqQn0ySbl8cv\\\\\\\/2Oi3jxu4Grfsg9ufWQEAGA1TWmWMl1a43hRNLBySjfDP5KwubVwI8rZFKPXOfSQx87hIiJsUWcOUS+4r\\\\\\\/FUyaz8CCnKjCsN3I1KosXZxEb9CHPM2W51QyTNDxsOgzTEnkJS62pPFkcFPcmwxgIRPBXt\\\\\\\/ZIPbqJxnIZwXU72UPc464ctWEiTxQj1Dk4i+mX9I5c4gft60QjSKhvuQ5VTjWWZqbZnVoiNtOl0Yz2InvOEQudnpknPtNFRhSn4bId4KUWkKYXxN7OCgm4oWkm\\\\\\\/cmpHgOapB2TKjJHFbCqiJAOtB8dpcKze1qSIZhir+vrQClpLNFaEcpNV76C0m2n38YEwaVLFbETw3bBUt01eHjOKwQKSU148Ey\\\\\\\/nTLQ4BSZdbKmPgx5D\\\\\\\/Mz\\\\\\\/EXHEKiKQAh4poOtMZlaowgXkRK8zIwAFbcmRDNZgop7Ql2cOxoHkYKwLQIF7NZqiUZp3Un+W6T5SxxEbjqqYiIEIc3LYDuX\\\\\\\/iCziRL6Dzq7GkcmXV\\\\\\\/yAoG88jU3+L4JFjm0Yekvgct+SkiqCA4qL2ef\\\\\\\/fSvEmUC5S5JH3rDGEBSGwosgMQ+W3XKk3e+CkKLFEYJtee1wV+16GVpLwasRwLnrNC4Q5Sit8OL+pQoaYk8S+8h6XqsHuJkiz5bQsWHLAc+u1HwYoZQvsyknJJVMVGBKyaCzhqbRSekFXnWKmjO0WodM0IDYPZVY+SN5OUHI0ABohTZP5rwoNHAgQ4VYXHuowIbJdtQqO1\\\\\\\/pu33lDhEK7goHo8O8JvS7AM0VNLQ6akygwEeu8G7UP5GmsGO8bsqkvWF4qPlKPyLYzMpUaOHwF9DMT+5V8GP7zWSPcjp0iFfe9iyfubNAekTeXsp\\\\\\\/BbhoHJjq7aFQYVhy03wwTo\\\\\\\/2ybl0M8bzd7n2PrPdhdybMSVy\\\\\\\/c3HL5UHP6AVxemzuyyB\\\\\\\/NTgg+84fEEAtyYvXkH8ca4rJjX9+85SYw3B9Xp0erFZt7gQAVd59xBtffoWpMzX6LTt4214wojyzJz5QhA786Ha1UUe7p\\\\\\\/vWi468v9k+wRkw3nJH9pliKb+JpSaaeDdsbYQpTrWJu1EsQVhZ0EXVfvthzOuyPyZ3DzGY9pu9qKcwFFaDkVdZt4jBKjwmhN1SDOAAL5awSS0G6MyQhBAr9p2WCwGh\\\\\\\/IzWJnna8qXQ6IPgVHZFKNnLU8APyoQdrbPddrxuJXK2H7Vpjr\\\\\\\/P+1w2r+DKG38FLhcA4BTMYCBz0Stnk3osZh\\\\\\\/ddNn86PB0L3zu5LUUDbEnyyGlTQFR6ClB4Jk0ItTbXa7FNich9FKzgXy3BlCk4pnlltiGUQg7aanrE4B1d5mQ9JU+k1Mn3NIZ+ax8V4zVEzLimaYBjV\\\\\\\/QniXP7mdH5t+LLvsK\\\\\\\/cJf9MZIyam155TIYzm39cziYbWjfsC\\\\\\\/x8uExeiKZe++y4uEQuDRefW59DoXqkDlA030ok2G5t24HsP24fAfujMwHg9gXX0fTW5cKD3RqV5UoJKHhwM9URjw1qEU0jCa9NjJ6ReMNNOIOh3YS2CQnYNwENpv5chC9NEitk95qB3\\\\\\\/i1poqjerwPFC+DAqKQwkAeBAJHWeDKvl0dK3QsB9+O72WZCt1U3yRwUsK8QnWRMQky+5yYMKFuyNHPbn9tKOvdO08FQMG37rBcZLg\\\\\\\/SdarOUGM+W6LzKAM6CzlKowW4XkfdNP5lJbuNW70Ch8CTB+KR33uHl4qGL8hzQYSSaHTjAKFN7ywUwstX7ctzf6urA89OjlPqKOFxRwJNPOOewcq3ffy5Z2OjiLMxp0h\\\\\\\/Wow\\\\\\\/wHxRHa6cxkgFgBWeheRt3fisA5aLZHsH\\\\\\\/UFwry7JwLTsWjcuccTTOERJvWAIAUvoxsAPq4+EOBgHWw4I6H8pwtwE02Mo081Agvap2CUFLX+rvr0MtLJTSkM6PQFPybOBpHtpdbtA5MDyTj2pRXX3t4qxT47GnWir2ZENvQv92EoQ1Hjr+vWGW4AvDPP9HohOzFeQxxlBrrcQ3Oc576pxcF7A5WW4QtL2wJBr3pqAY2t5noEL5FWCzi6SaCLtLwkBzAa3kIlKIsw7ls1SUsndGHpXKvvvKHmJbA+yGySqmbiWToh2i9kfNi4YgXN734Q0mNfiIG\\\\\\\/aChJDz29wBY2LmNtbOkrM7YMzDSBXkcxMYEpFqPIR1OkgLyZgWEIqVzSFEwJo\\\\\\\/hE029rQBhXdICpSVjarUbiYNu4Hw3cBdXhe9GW6AF1qE2r0raDQ0U8sEE\\\\\\\/RLYj\\\\\\\/VLX3V0g+iq7pl5AF7mLZRQ5ko1IimJ8SrHMpUO4rIHe1WDoxWvdi9kty7lFPfdHJXTZvBYkNfocUiafqE4\\\\\\\/pg7VXBIvbJb6tCcYTsOzUCmjSkainQsE3r5eXPuQfjk3OfbNR1KROV3vLBIqB3qyt9vvYhRQbk6A3r\\\\\\\/Ati2KawcP2IOKYo3fogNQMzKhbu\\\\\\\/SVnc1W9WzMUvN0dHKlFAbeNu\\\\\\\/u3XtDKhSbPVO13+E+8kuVDPHpxQ6N5eBVILP04zjyM9h6f5r1cRB5HeCjCUb5I6CzUQZSmZJ6HK5LA4igzpjYqWWzca+tz8BeSqKx9dmzyTxOxWD2kBnV+TtodIG3pCeFY9j58OZsI\\\\\\\/o5ciSjvxmzaZqc0xgpyQPGAi7F2qfnptMsuimysJiGDFRZ6aGNvIFICcC6WMQcaerxy6AW+\\\\\\\/LOWZgWiYpa\\\\\\\/wXX5kSnb8HREkjcyYSWqWc2GKUXv6VP4J\\\\\\\/gXJDn7qG+kAMwmzxGYEjzzCRV+lvuMzyoFKUfTt2eY9OYhKduTu3V5GqcvEXwFEurakoMt+CyMrf3F80mcWCJgZcBHdj16LgYhyJdx5CD3yuWB9RNwFxEa4QVacmpBjvN3gxCmNcW+EVcppDJnQ7fAZgtmcV2zpH0TMnG5eJVeemY279THGVQqC713eccCPpGsvMqs+ShD3i0DWL\\\\\\\/omCWjGYyHVG8j9aCIPryE9pGgfc5jGYyGZJvrS6BJX4LFgw5YSppe797ykBa0TTutyeGF21YnKpT7y04yVGTSpgjdxqrWnC8A1KXiAozl4jhV55cKM+IkUTBMg+uTt4UjWA5jKpE+li+41c1\\\\\\\/nzrfuth7qRAN7bkmSh4hRtEHOAOYFT4IIELzMnnQrymOEF0QasOJKzPlH8JA\\\\\\\/PsOYzKN6rs\\\\\\\/6ibJLLmnm6wSY+B6dVQFyaHfOM7S9ffiQFDFe3LJ85baWTvUcz9eIlfSkpT+lrCF1TNYBxaVcWjGBmzkaDv1CWtrXIGv3s7GdNykkc9yltZwHHnRv06WUXZkUkPD7URCj0JvP4JC\\\\\\\/HL6d3g1YQ0pRwYoMPGft45EXuwObf\\\\\\\/XpygbB8x5\\\\\\\/Acps9Gyjmgg\\\\\\\/xA3Lr72Jutj5tng1kGJG5BmUhmJPKIam46fWFzQWSCr0aRScgctbldunTth7UrKzBf6pJwIOCtBYQ2SAU\\\\\\\/b5wWiZjBPHWLnbxj2WEP+u9ypB7\\\\\\\/qeO14qfiOJD74gz\\\\\\\/rCOUhty68I+FQFlnPh5VCFgKBNRmuq9gW9OCB6F1YHAnXm5OSc9zAjTTrV15rstyXkyzpieWS81FwVocBIp\\\\\\\/bicnKj7EI1sTgmmZLZT2RbK5Wz1T77lcIbQkB2Ysj0o231OIw5rQnwDZuqJAR1GyFs3+l8FQv7xZOaeO9xnvLtx732zfTkJ+sfT7X43JGEpNR+OMeGv9SyU3j8YJExg1DyaJaIB\\\\\\\/Nc0ZfJHq1VFbxXHslwZlszw0GaiF6zZGDOpaDGjTcGzhg9ZTbFqv+n4snmhgJ26l7jH72vnu7evPUGKTEytM214yd8MUxYoCh9\\\\\\\/g+pwQyX8LHEH+yERNCiPwxuGM16RYSMa17BV+sUek4rEWVDOt5LENir3W0waPZhHMs7142m+I7pZ9vaOO5Krqwzf\\\\\\\/jm1sm1eHOsM9BCd++BE3bldiDYMdHs+A8jLTuY4TvkHPzMPWmbcqDni\\\\\\\/Xgo2HpI+ojmCAL9ZG6Bp9KGHMBvmnRI0txzXe8KE3MYkfEAnSh3s3QItt4QVtOSyMmHVIlh+qCwTSkkxuvPEYfMh9sPtn0An4Noog5Q4x1DEIxeH7zXb6k9EA3H3Ab6hfiEJGzXxSeSIuAoRiUTB86OrwrJyp5hrFCw+NNd+O1q+R1GBNuQfWF2NJnIDURsQjCjvkp3Do78l9LxsXbl2n9ipl\\\\\\\/35q67h+XlQmXNFUZqXHMMu8Ji5QuLkbXlnUl9YDPmGRUbOmAB3yDXocgK3fOX2FdeF\\\\\\\/ubKKmztIKXl6BsAe7fDfTHBf3efDbTr3OjfnBQoHB0DFMkbRTnrLq+2T4y0Aed68xKliu0wJI\\\\\\\/tgFScqv+bVDkkeYbBnRVTu7ZtdG\\\\\\\/Q9scGmDRPKdvzS3dCP3pAKkWnhl0ephNI7fcKwvXkr90NSPaGRrVxow+ILUYYusbxRuGjJsumuaqB2loz7mYd8zUpcPwB60+xaayZLLH1SlyUhrMhbeVMKyPsuRSXo2FGuXwR0tSrX7f5xmuLiNT+Hs9mlEsboXq3lxugUpyv8E2sj8lPDwj0yasTnhl1pvpyvpOPDKEDEbGqtffw+hS5mGD3BSKXJi6dJseI\\\\\\\/WTypA6o5r7saR6bLcOg1ZhK3pnNiQmFcJa5OTQ2R5TWuCUUlD5w+0czdlt82AwF+zm+eiE6DBwXakkOsmraim5qE2Yn\\\\\\\/\\\\\\\/IGpDQBzQ4pd1D+3oYM+cUhTCi9qMbveequgwD5MuQscifQ\\\\\\\/94xAzD8iWnjJYdvT3GkNQ8K21N1dVoBX9FVRDkeWAWcJZDyJhQmwzvt8U+RMb38fDEnPc+3lsUq4qD6mk0Ux0If0pXXfb8TE16w1OKKkDjNxNUmNtGKZmfE25EPcBzZb2jecyVGN3cer3qS7PLgtb0gsXAGSPv3p3DgpSRBg8uuAydtKGNEVVznAHT9bguKZqQU5aO+7+5PNiA7exkjzw6i1E16ZKh5+ark1gtlJxyR1aI5fiSZ47XB9rlfvghyEFpzniREnEjjef4KvfsRNT+PwrNEWrO1B5jHkpymLY01i8nJKTj78mQ71OOO3xbRVCFe6HtJYbVrRL39aS7KsdNLvRAGQ9gdVu7EtGML6fnlxKCruel5sU9LOEQeH3NnBPgjFP5E1c2siuy31KvU3VcjMjxM+EbsTo\\\\\\\/avzCb0woioW4\\\\\\\/c9ZMrUTNCRP9\\\\\\\/bsCakkFJ5yPC\\\\\\\/Mwk\\\\\\\/Xb\\\\\\\/AN95iYkNEl4mRwbZE1TiP1klp4NeRkixmnjXNUbftrYrujtdcm+4bE\\\\\\\/K4ggr\\\\\\\/8jAuzbgVyIj3S4wWvZf7WcaDSBcWyfb+vuBPA9lBHuOk9CKCOB\\\\\\\/V3COl+FH3EOehHSe5gIZN4g6UeQ9b5Oe25\\\\\\\/vF\\\\\\\/SobHIerdjV9DmtjyYU3ftjvnmbhyu26gCV5bMEdmCCrjie5nXUn6R5OSSVC4iwf9dLDp\\\\\\\/jIagvK+2wrVROwt+IPg\\\\\\\/HWnYpnNqAjSjSH8KOoC6HVYHTMjwu8VB0OgICBU0ED\\\\\\\/KhvMA2k7T8k9oScUb5Hhhw05TlPE5siSi+v97ODWRvpOwgO631u33JzD+LHicIBpmcoyUMF1KgMOlqayokkUaMuY664jSTz+jsADcMY6mP5M2wO157gzOXAOeoDkeBVIVluWqmJGucbZVPilm24+AYVXhfZzM6ioHM1wyYby+5pWR2I4+JZ\\\\\\\/13KexVmiedOdi8xCYmx2elKgN1fQiWPJtNSkzXLl9SoLI+l++YN6dWO3T1X7WVudeJfmLlGnFa4iWgbf\\\\\\\/z\\\\\\\/eRte2l0YC7WY63fsuYTHBTYnN9tDMiu51CqZHLAeK2bEbvWQ7UyTjL2XfHkQC92AovHOtJC0Dj1pvqsD3gTVzNkYjMP4SaL3BolC3im\\\\\\\/+RFREZizAD3kq2TncmbnJ2lrcFpHYooYcQDchwnz72OQuF0WTnJSh1hgEMJMbFOQyLVDqqB7WPC4WzBIk4\\\\\\\/SQQDAggKElU6D71bbAPCIqy2dOokirQrcjNje5KO+G\\\\\\\/4tZh1Yoqb1zhM0OQY6aIE2xwJiyyKfOVd1FLp7pBTURt16jGh3f2Xe\\\\\\\/DkMChy0y+2BQaAMhoBQvhUfr2u4HIIIXgWL4bv+WvxfcCmdV6GatO0GB72NoPbtji73psS1m5qF8dnArBLa2TVXuZesxS8oXkDjdZ8sVkYxskXIWPKhEdQllcg5zJG+07YtUOgmYrnjjo20NbYBSaY2KXlBLC7\\\\\\\/wIzobz6ZWpt1pe7R8IySWCk6XxuuYb0abfgfrAXsZWi3nhd8SMFVfo0D0\\\\\\\/8Dq6Ylipobr2QscAMR1tJa9UiPoSPYNxtt7ffTvJJDUycpiRJAeyrzbNo70aE8RVQ+hZnU4vxalj\\\\\\\/Dr2lZLvGxQEwn7dBKHlEfBOmv5rtrB3u\\\\\\\/eE5W9AHA1kAxxmrv8awimIEXgKWkngWlKDduSLzVB14RogR3sOzlVcwK2V31We\\\\\\\/vQPKAR6iyhdcPEMh3dhW7tch2q36BJ5RtoWPqPoCxviBMwDClNQ6FCt+F+UKb0dB1qCfacJINA0nH5vQAo2ySUgR\\\\\\\/rCBKULFshzlHzCjoRLcH4e6IGx530bTGWB0iD0Huzer+P2qAbqWDB4S9jU7rglo+yrGX8tt+sRuwOyIH9xWQout9pIPgVI3wPvpI2YEHRtqOy0NBEfXU1QV\\\\\\\/1fKJ1Zj3VQyQpAq8yi+lEYUNba1l+ZO69pXVSynSAIgwhBpc\\\\\\\/62Sa1JE7pTATBpI5tdr\\\\\\\/RDFlOr3Msqz0fDrpKdlh+UOphm9CNSIc5YOBfMhPP5+VAt4Cy+cIDcHMFx3IsYNLtKimXk5eG57nSDBKVuhsDcC9fxXBslt7JCeCzLSqvji6UeX4luFwpTlYFLAxxbCsqUeFI4xGJTPgBLH1T2e+JnPkTcOSkFo0viquEYSHC+p3rzl40O3S6ihdtY7JBMB\\\\\\\/WFR3im5CcacV5+3gg43ZOaikSbboRNkddtJa\\\\\\\/HviflAlrTzBXzK6ONy8SJ66hX45cUiI0MMPqMfAxxU0dZVWlCnMrn+TwyztHmE2\\\\\\\/ZWs0Act4xWx6pcAZZvr1OmDXLEU7IJHVkAqWMe0vj5rXbtNkL6gbv9ks5YReIsVb4VXSy0uHew48GlO2WhaZbftoDT53sbDIqRntXK0ZHuKRZO1ovo+BNQMFYeNg2Rh\\\\\\\/u\\\\\\\/QfRAiUo9joTOiL2JUVpdS0MKj78yFL7B31rnTr3FPpdYeFFiKp86hM\\\\\\\/w3pzY5UvEUB81d7xowUnGD530BqFT7hxMGahi0yTryPfXuB0ha4SS+72LuAnsngancY6TE6BWSgGzGVUAwRvzWJWmAsF1ld3GqFsOX3UrWTBBpHdAnOHhfjWsuRWhInduxD1ZI7yZW9cZlRK2XRQxzG3CeiOs6KPQBlieqj3AZSOrAeXT6ejYJkLMrPh14bpOqVnhL7nYW+IWLVZmtk7AA9HwL+DvdnvbiDr2kGUI4QwqYQADMdZOvJkTShET3JL8v7QFA4D5nM0KqDUfOaTkuKHfaCaTuJ2\\\\\\\/VJyebaDu12p9NYMIi5LvZ1TicuX4O\\\\\\\/LnqFBUh0I+nec+g9qvFR+mqV\\\\\\\/mdfLphxDx2e\\\\\\\/3VcgS9LZvrflt72MVUmSOLBn7ZjnUw3QsNfGv1XH0aPS7KiZpaF6P\\\\\\\/+zvua\\\\\\\/IfV9\\\\\\\/xoI4Ac\\\\\\\/Z1P7exwbK6xRizYjPNshAGJLK9PBO1qw3T1pKXEwXv9okEEAQMNxQMIneZKzgNJQD4h1YuDBwKNzBzGbAnnSAV2gI+HTEdEZ38bSsF4Xl6hEY1VWv5Qa\\\\\\\/hBrJtRIOeQpWQNE7H\\\\\\\/t1xmi8F5mCYSnRHnedCfWrQEzp8fHPFXbY07Uxo+zB73zqk2ujpD+uJhV2ZLMtsoBKeDBne6JUYV0JpelxuY4vQio1\\\\\\\/5KcyVqjIEqZ8SYWh\\\\\\\/\\\\\\\/vCrzV\\\\\\\/LH00Nm4UzUOZ1a2Cp\\\\\\\/5SZrTx4oem7EsbA4L5VCBTBOuFXlMkiryFwTdFxwamc7NBZ0cp7x7wt+WG0WT070LlUo1w+fXyDZZoCEPgpawPg9BKfXl5k0gRazR2iwEUsZ57kYuMcX6GymlQMw66yEgeP0yeV5GUABJBAo628vlVa5b1\\\\\\\/3ZkmfhSKA4Od1Wlt6QuyRFJAmQziE2z0MILM+pCV0lrx0BoXYcitPHxnzJRmlMy6hLLnrVt8nOe5OaU4ZLDCGxRleQC+lPwgtzb0GdARmJggpgDrYdb20W2qy3H0BwN8LBTtRZ6aA1+Wz9QicJDfUamZS1ZXbcOIGKAHShoRl8LD9h07tDhrV21TNfAnHq2eKsC+uwboxwDUOUYLY9sXbDg+ivywn6TQqxOApmdeLPyXgWECRh4jOHl8fxldxN41p1B7dpd8ZPJSw9RcJn+IHHMGzz5Q6zWqOiu3p2bK7UtGHX+6RqCB1GcfeydCE5nLKrCrN6bJErku6CvAacoGBfv3nJeXTm\\\\\\\/2pWS4wHWTeFxciZOzWxxHzgeTPegJpxT8q2QwQ\\\\\\\/gPlcYlr3u82ASgNyZN1R7e7UBFUKoi7w3aP9YRI3LTn\\\\\\\/N7Vk45akoEp9pxKSJtgZWUdb\\\\\\\/OFivIj3kjs6KTc4KO18asLPEm21cdun0eYRx91sNy\\\\\\\/NzhKply3kjE04kWy\\\\\\\/IlapUz1rgRSeUrfx6N99FLQrFaXEZlYVRuWvnbDeqXNWZQbE9wKclh5Cow7g6C7bOFKwGbvd3HvexpXCW93a5dpg8+eWIexCwwhAmN3ZzfOucs6PBSofgUAX3LXiXadMCOSjM6Y5fG\\\\\\\/Yys2Pdb50TitWoQpq9AAL8aNgzl3Cs6PFlix1i6DvhSUGiMm74g6MBNpByN5C6YKq08QTr8y\\\\\\\/TROq9L21MGvZOqP59Haf8NJ+qyWfwkKWn87zPLI3XMWo58jnuNC7PgkPNiuyZS5+le9+euClNjUz6Dei3oKpsCeRmHlUjDN4R2dP\\\\\\\/HEp+HweznR1pubLpH9wZvRGJyPAR1tdYyG6ojXWJ9sHE2pnVi2QpdqAlStGn4ZgGOdgLHk72\\\\\\\/WSP6IDlfs0L+HqrsTnY9diK9Z0fXqzCtQdD1fTHumN6E56De5DpHHZGSJeVmOY5Rt4GENEZqEurt9fQFwap3M4QDKPQNZ7wxm\\\\\\\/H2s01BCnFn0Mv5sHFAlcnqNp\\\\\\\/p54Yzg3xDVJ17UqAosalxVKe5I5hGTaOs+AXws8J8edt0+WklMsHm+KvlYNEjLEZubX78XrtqC9ppS5FqT5VaxIJTz0uZWDY1Pwgx5Xpfjd1oTWZiG6e9LHRJ9qNTuom9N3FRhULES1GlREKv5y5ypMUSyDqC5LffoYeVMk1QzO92ut8MUlYoAELbTmUSNztsQcfHAB659JqE5GE9BrnmgiKp61eEplVjkUmIUXoOjTEjcBHKpKIxEX+1Z7r7eIKRLdR\\\\\\\/goZpcgo9yJOX6nBH80s50gbS8FPpUTQhbrzBiIWi9rXVKUvI7tjnzIBcR65cW9LrPUTdvb4Uz\\\\\\\/ubtCg+n1vhjurug86jtv8qO5X7YWt\\\\\\\/Y8+08k4EwQLTLjirEp6JiGkxFeDT85Lj\\\\\\\/B+bV\\\\\\\/V8XboTPZBZPLfzehEJ8Kwb4K+EAnZ3y+rC5G8voaET8J2ET+dlOhs4JQXjA3w9EP+PskCITXkSrKdCUXKn2\\\\\\\/a7Xzvb2BbJGSD2BxQJQyyM\\\\\\\/yaCN6jidZoRr\\\\\\\/XnUDS8zs0GzxICElj7IAf2e7Ol4TZly4ji25wQWwD5tQGil+FghzWzQ+MG0QXpfU31uio5\\\\\\\/VtlLhudGzHt6FsqgewrBtr5+D7441Du2rcaviE\\\\\\\/SbPirXYXwL+mSbLgRjg4cwe9Opz9H4hHlUKAs\\\\\\\/0IhITnG1JmBpREWNeGSvWdfCAdxCpEydBgEIZ5PllQmcVA6+23NlkhYiL03+uUPtRI5129\\\\\\\/0Yn7jSsDRqAlezXVxLMzmFLZVX5+1BZN165hSxHXHoj4oy2sQ2PbKThSGzWesFJeMrjmAxMnScRCdT1iGqxKujQo3ATgMS0HnLtwCxUC6AtvkQ+qpKe9WY7rLF6ch+nKNF8qmVYgsgCceKCj2wrPeddK0HfnbilIADVD9iq5k1Zf2WinwpazHu8fGhdpTb2OEND+vpn+6kv4TFJysQiLqR9xmH1yvzDaISoOYXJsqoaeBNNHsA+6f6j4fsyzMMQyJHJ+XiilhCCt40AKTwEX7x+mofSkz051NwyHiPVLW8ZSw0vo+8BbXVb5AMQGQQ315OeWVk36O+raiOQoVtBMAx\\\\\\\/sUKcw0iK5I7J5ffdWgqabcR08Ib6nS0yWJcfCvWOvi4Oj539mHTS6jVVFQz9U1RcCWvrDQ58oUMm3RsSlZXGIJlj9n41eXKDWQnKkK\\\\\\\/eq80clbP7PlmymXbMcZcVgKKoL+7yVnOAMn3GuXNj7VWa1ap2iytTIbU9whlmGNP7vsfP5SgElTQFAzJQzT8PZks\\\\\\\/xdYTil2HnBf1Y+3cyf5NJCcqbKC9VhS1SwO0adnKfW7POI9gOOFVqJxkleWa2FMQjHz34bJfub2NUB7Ciy7sYXRRScq3DysLF\\\\\\\/wVZwXz\\\\\\\/Gu1Tw6nwog9YsCQTW\\\\\\\/6D+4z2X4BBBPkpqnF5HJQg7MJcADucQhY1AUWZCBFYQdm8mYnONWrmpZi3hXTe5ALsz5Ozojc2oEjjEfxjlp0Sr2bY\\\\\\\/Dsbw2sCEsG20gobL0oIG\\\\\\\/2ZK6uFNOVyJ1y4SsdhnAt19Ukw9oIkaIgks6P133PiSG1RD77W65vrF\\\\\\\/adXWoWF2JuK4K64wXKOME8krT2VXjJT4sjjjY5z0TPCkC0nvYPciCcXdecAUocxG7kPm75pvlmxVuEUescvg7hOj4cU57KeoeiLGnGr6VMOj5TDfeF4iGIjHYcixF5z1L+XZdqP6XBu0fDswVm38X8XtO0PVyy2BPFHc8848VxcO5TpplGv9mI9dQOTnWm7G00GULbTcylzQNRIGv7z7PPsuvj2AwJGNV30od\\\\\\\/wNhPWw6UMZSjVlZEw1CPZG5b\\\\\\\/nrFpH2ns8rYC74Y0ebGhqXMmweFtqSRDgY7qTPlFgza4VrV9hxq+PlLSjCjCubZRAME1pc9LyX8hTTJqQEqyjH31+PQt+i5q0wqiHkbEVmAgLPTTMzeT87+V9VEFGoKBTJRRp6ZMsjo\\\\\\\/\\\\\\\/4mUm9ozrjo3BAyM3vXM87ec3GY3uujyx1ohRScPnTHZK\\\\\\\/EtTT6PjUXSM7rausnw9252dwcDoBRaZh3PRhR80GYiV+A1PwyCV5vImGCjWOvULRE91\\\\\\\/9CgsYpSvGOtYOj0C\\\\\\\/sAfCpXFXfV\\\\\\\/HIimrVfevpb4mmqT2lvXFtBVl1oyxSzMgbLLKvphlCEqXm4tBdk6TGooteXSd9xRaxBIKXNoMb\\\\\\\/0jBqZHy8iTfdv6pVahWTWeK9N4o0RcSVLSoEfTc0C2BbVdde8SnabTkkgKnHxWb2rKlp7FXu6uPBE2t6TbdsOKa3d8Ual3Q5yGgHpeUR2VKIQqyeWeVk17PvuPBqKiVQ7oQE2OEn5pREFuzkT5r3Nmfki5xNKlIIstBZp3fzBXFU8hiaSZDnwCY2JfogBi7EZjZ7gpGr7jyoq68tAG\\\\\\\/0nAi3gjyRFsTFnrPzD5fwXfEV8aZDLmvXHXxKGb5N3Epch8kXkgG9Hsl7WIF4otU3OGprc3w6+kb5K36pqDc8r8jeKT82f4FvuNnl7a7ZlntLMTvoYjjH4HWjHDxY8MZFzTd28xAoHnzM6n5p8PibYdSEcI0KslQV7sOGx6yaDfPabHBO2z3QytuLcXziWVT1myOhRkEMNjAHufa8ftmoi5RZqwgQPKRgyBrH5MsrZoMlRjcl8h6XOMYMwQiMfZWh5EfgY5LBtiRGMymz1rQ03EHyG1kdWrAf3gGcOkLXuVEZZKd2uFrIHMw39jZySO9N1aR4L3JwNfFNmf\\\\\\\/bYesbZJBlH4OHNpBIeBb\\\\\\\/OxHN\\\\\\\/hPQTpSzqxcRZc4XCytViizbsusH\\\\\\\/dEMv0kJPEjy2LMN\\\\\\\/OqH\\\\\\\/v5w8+N8DytX9i5zT2X8aeGZiZt5GlBJCX2jfT0q9X+80XryNSK8cuxJG4PsF0zVX+bkWUx1+740mqmB+33qAX2YwRa4fpK51vuxvD6Ca0iUn11hmT\\\\\\\/U9OAb6NxiMLlcXYdkcxE\\\\\\\/Iq9wK9ltY0lK\\\\\\\/zPqfFBHJiSOFVExWYi4W9GA2kZSV\\\\\\\/24keSxsoSToYPLiykelLPlfzHswY+loB570UbjitCQ\\\\\\\/zB\\\\\\\/BNSKYrAU0w7E7dVts\\\\\\\/70VLjtn2nBoNhA45uNAG0d837UhCGezvDnAnbrTGrgBhZ6Xpuy8ggWZbvBvgwWANlHtCMqUpWOpa2OgsqyM6ECeG4DGtaumSgH5KQ3t5NstD5+GsCGcrW7ofmpTMZCVrUlyoecVH4rdyRJfGnoheFVCDJ3Eh1jHjErH\\\\\\\/6XboEZnVdSpKPlm0dhwYV6qbGK5+mW8VuGEcYZmj60EVDe+3mFSQl\\\\\\\/uvBx5fke1neQm4BsDiOywEwcqVz3fAwCNzntyOc9hjTN9\\\\\\\/bpYaBO4k2RWucj7jqRuPAflUAbpowFP9EEjqWWGmIlutakVAzWIgkLeu8NrZrGUBn2ymihKbZZ+9SBAYTsdDXbLcN4IPUqDIp+Mt3dyiCv745+JN4tk6zWhgExp2x6R5hvzUaYa8CUQ9A9JzYDYTN5XHX59Ac8RqHKBe8yyHuYl12bSrhA3npSBVx8xpfjXnBnC0C0tFa+h2QTj41Om6DgJ+KwS30xUepfThbBpv85uLNaIGN\\\\\\\/VR7RWuuxi7gR+3G9wtWwb7GgQFbvO5Yy0K33sQ+Vqtlj1eVm\\\\\\\/2phqiTfcln9qsM3DuKA8ujQ3+EYkNCZ23eX1qpn5X0s9Lkgvo55Rk67qPiCkf9dBDR58qn\\\\\\\/ftCWGY2hdJDynp19UjTUen+Xgv6WBDDgHRWi1WJfSipGXOraEjUD2uRZao+PwpMAIDdX\\\\\\\/iMdkMkKcsiFw+Mkr5I8xSMVDduXn+eRhWeNsAqHQ7BWkhMkxgGti8nD4HXrAE65X0duVen13U8Kc7P5zEew4O9j853vXjK3nTM7fSYC5NZfYQQ83fpC1MZKe9Ur1bR0WyUOIilmKnt8NsMxV4CQzag8EEesWIgFyP\\\\\\\/nrn6NHXmOXuzrE+DmtM+0UppJSqCtdhgeGzBhenINqH31P1W6s1tMw43O\\\\\\\/OitiwFhQvurOztz5jJO\\\\\\\/N+ADH7jjrRIs1V2Y2ksBAa\\\\\\\/V9aeih66Q9ZcJyuB31SNuitDhKXpPjdzYSLdxbqvyyaUH9Pc+HBNSo0Z7lM1hnVslR0fQeGdYN+KEZdisHwrYXlcZMpOYspubKq7TyoIRWIIe02rrs1GcsKXKAegy\\\\\\\/Xr+zNZ7S7oOJWFC6oG3To1ZYvMujEjrETzcTQta2e6GsyGCPrYNgj48Kuk2pqaB2qTh2wh\\\\\\\/9q6x9QXTlIJEr\\\\\\\/QwWUxT0i4rLZT7Od9ZjWm98bATfVMZVWdBPzVudVFDMo9IcbSxogVqXHxX\\\\\\\/Ei+\\\\\\\/QFgSvS0NZP0o0j6L\\\\\\\/VQuSQKoHMw8LcXOI48Sj3\\\\\\\/1bsdTgQUSBu+ld5H65BWVUvpy\\\\\\\/3fPXqmfnqXxI5L0G7+gUPmjtxCbQ44hNNNTvtORmdInQ54PxLBbDmOa9RKJiNpuTHkQR6k7xx+zMjP2cxI7xp50\\\\\\\/1J2Lm4qMGLQqexpplTQSemIlnkXc7jOLtN4WAK8ijx1Vb8fTUYLdYHQoSYatO6lxObSQj\\\\\\\/tSjUzP58a6hVtj6iAlJHIp8Vk0tZnvK95Y0ubuWdvsUuXMxtESlUKoWFixfiwh5FlIVEIjOdwEVuATaVu\\\\\\\/Aa3KyfCo+md1pPUv\\\\\\\/9U98gSIrO3019UfWsjUpiUvSmDkZSp9wYmtbY8Au0aHvCi0rdb63cA4eKYVfzpRlfBouYoOJ+7RKbndekqbaHQCNpIw0VUbrKb6QD3b5siCV8G80dJVuSc1V4tdhYU+gdn9fn3Nq1NCHzlaYCRp7MFKOEgFbSFYt\\\\\\\/MCBgq5+ambxHPbBJ4OHd6w\\\\\\\/qz9YsZmhNNob8sPx55wUNRwytzSBo+xgSJbKqvXiY2P5RlrywKaI4Yz86mmuL+RlWB70P+qVS\\\\\\\/fhmwhjeZw0yUZGbLbe\\\\\\\/yP6KLBQYmzDCp1AbMhvtU1OsuEftNoV8DL3UsMa17PQDfyfvAr3azW0kACistUqb10SDXQ9Q3gBb2oBHPvNBsvEWKcKKxKw5yY\\\\\\\/psGbA5U\\\\\\\/JeViW1Hc5ZzD\\\\\\\/AUYInixVTmwWH9WBpShQhQexOBKD86LPrkH0\\\\\\\/WFG4DlTRxD8KGeuacRTV5+NfaHVT21Pq4ORbMmfA6SAk5IgGFs+EPEq2tCKdPtSubyXp2+WeUHZjcxM\\\\\\\/05apcIr2Ozd3K7SZZWqv7xGg6rUzZ9CWPV0oYJUO3BdKLs4S5I6EyWbSVtHzRKnit5soeWu6vOygot4wZ6kr8eIX2uVSfyWbH9LRUrUZkeTOqGxlUcZNrlHyJrQ3BK7Gg12GI9w4++RfjA5OjeoJtmzi5pQC7RJn6gf2soUjegYmpWbvMi6dmwwZMvnswLXICqICw+WvCQ\\\\\\\/1q1VAOmmZ0w7XA2UKij9wqoIVPrORzQ6V6kjdSR5bIqSi9xhMtqZ33LGcSoHo5MD9q4pWrgtzY+JYuf3KicMAGdZFZelLFmguXZrcbGzMGemIZDn94V2+D0zq+e8aFX7Ju9j3DBgWuGO+8jEX3UhtFFF5fCvMMnsrJykI3o3aHJAVyD1e+5nhREFeFIQwiR6XHE3nwRwK4uJ4N2orhT3VMp5+wOghaaGxIBprjh79jdDOaAYJ0737L3COUlyZCgktwaV1JprshE3qmXmE9xab8HN9DS\\\\\\\/GxE0kN76xltAIxd4Zvx0OGd6O579IR1rtQZMuWrxk4R2DhYKWB8vUagEpAIhMMYrX4Rg6TxlYHHY8K++qzZAZoN\\\\\\\/yhQgu9WI3EfER7BApWX4\\\\\\\/HI3n5MFAJicEQKFEd9gtSfhZpWC9yJ\\\\\\\/NyteNq6z7YIb5vwPlBrpsoaO+bNZV8r0W5E5n9ePrfzT5N3Sa4L5parkXMrVV1aLSacbohxh\\\\\\\/vsA7phNgvWDf9ATIHlTWnkkfT6LhOhQhaJSQILXY30EdNPR\\\\\\\/mlyWWZLrCsyIBgORnjYnIu+d3hyKdYmiRLccUGO5w2lYsFfr7UQFwIPeCTL8MRM0olzIVqh5LLYMaVkeQ3rfBC119Y7PlaVJxnagM3KRMRPi52vSd+2mSwNapcsmmv9kFyPm7PUjFF5zwUzCkpHcN7qnCVa2WNKgVpnivHB1XHXPck6BXIwuRFiyMoqVUizLwrNsKZqSmFOceN6Vx4mFBvC8WK5wDRR0g3YuHaqg7i98dt9ZFLgu6K\\\\\\\/oUL1ZNTUOd+SRNraSe+BLOw6wzpQHIVqjFYwV7sZx0a+5kpk7FGux7ssAGBPb9JwdidSOQzLXieQGaLXvjPnN58W355U9z4hqpEzNlIJllQWScDE+skqjW7hHFp2GJKQqBx8B5Wzhp8an8bsXUZZQrDeHWw699CDOI0nEu5DPYGBRLwI7vSvdRMguO0ez9eflQ6yMDBDGRFOlaAjOfzdcBSOkrOnV6g+firZ0bWRaj1f6MrQnBVmDmCxDJ4bj8w\\\\\\\/oAeQ6Mrj6UZrbMpx7r2RG0cne2m6CmjGr6ON\\\\\\\/TXe23JmMlRABCunL+xRVGcYRJY1vxGIpKHHItulURfCFnxt+Vg59E82TeCJg2\\\\\\\/vEOTd5Usdvf2tpzzwVG5z99ovDO4XslEeKJ9NcOAFeilou7H6Aw9AFzehXSOM4KakGjy\\\\\\\/H4YZGbI59GcWLHeTyhW9ZQxSstwlGrtN7mjwEtN9PNhmLRSqlPJbjCVjxwm+MN3OdtVugdhRswYrD13p61REdbTsvgzszyrDuHCdPjGSsOiYNNV+e3EjE7xQac6qg0GLAXXNeaWF16mMz2oVAdkfmzosZeYlYvybpH6j300SOpZUuMrGQDaCnj6EfPaiDBuLMKZwuoKnF2Rf\\\\\\\/NDsayoRq\\\\\\\/CIchJgNDPBOWUlAU9mayDr6AN4oYYpaIZzVAlyp2BZZ3xoUAI8z9h0BkqonZCJUcWPT5VBXoXS8oCC3Awn0H\\\\\\\/CltTWN4LsM7s2PoQOzl6sUn0Q\\\\\\\/ZHy2Mu2jZ9A5njGv+s1PXMGYzoyPiJW0CdZ16LRXs5hvohbozYSjkzZ4xMOsbvOBp64kbTHg+gI9cWXgUxCnCADPh4WCXFdT6aFRCv\\\\\\\/vRucY\\\\\\\/GWJ6EGDdDg+t0llWragMcDbSDi9LWRY38XbNAkDxo7iDGjLP\\\\\\\/hkU2Qtl4h10ngfwV4jn6IODnALt3Lle4\\\\\\\/JCLK9lGH860lmKpuJvmBoNvmv5MMS2Y2rUrl6dGQ92q6ZItLToVNnEle+io\\\\\\\/I3ZoeVlrDXhX9Cv0NAsNOjKun53x7RFc0sqdzH4oIasy4sOhzcQUMYw\\\\\\\/QLG15lcVweHnF0vqa7g1tQUGEJEtfseFdTMo+B\\\\\\\/N1Oc9WAEDWLHWsuzNwHWrk6lNM+FG3GIEKRYNwC11XBK\\\\\\\/rnS9vgXuXOW1W9IhyZoC0t1MSskznObP6HUdIsItZNaq7z\\\\\\\/pFQwDiBH3pAxEn+m092gYC9f4T7SZ6XcDC16Bj2SqT+8MwfXeKdkFAhnd2j7su4GPGHIijF7gnOrJ66kNvUlRnoNYTbmk6tLDdbLLz56ODdmeY9Omuwk08IreJFWWT3alOVx\\\\\\\/dbvoaq7FtTJIF+ckqwi7FDC55No4uLIPBdINNUpY6YDUlTqLFqqYT1LZnqE+mheSNwXl5a8VuuVgJnVBGdI\\\\\\\/g2yiIbu+YhM3zJR7DQXfE1+KMH5F53i6dCDyF021UQ9zG2nPZIYq8XQwHtljPfuumPjD8JLgvXcM89NWbyKCloo+aqi5XFs7XbcZaRH0Op0DfFZEocKMaDrZUv5c4SmgCLqRP\\\\\\\/O+j2VXKOKFR6drzCEe5LEqHGVVenHGkn20y\\\\\\\/GfOkL62c2rUlKBbSPcR0TRjqViGJs6n1NwoxXfTtxAAAGEXoSaNhaJc2x\\\\\\\/UPWEUNTAPUjxDR+u+X4B90crZ5nUazaFvsRM4I8JBcvFKNPjWO8KJxpuMZenol\\\\\\\/z5\\\\\\\/hug6bbsnFVxqYS6hTsqfvJ2iViCWVBOa0nPTZfC5OCmhbmDDdPnQdY7bMgn01qCbJ5T+0WKeJbHMGypiju6hOzN1y67IB7y69fKiOvdLJ9ujcA4FJY46GJR2hTTDYHh\\\\\\\/8sQ6aKNJxuOoKfgE9pLlYkz+Tuat6UFP9TTaIcnnLtQfGtzCTHJ0s1gHFi8NML3h6EdPl2fZy1CsRgeiEAYbjozBd\\\\\\\/dci9YVtO34heHh9Rbva\\\\\\\/FZHG+BzUGFeBrr+l1JdkOEq8EG8AiAZ1EwOzf+\\\\\\\/T5Zs8ffbvocHqWoPgxJxJ6Q79\\\\\\\/6IMj9wrPsrRv9NCWQ9smBzH2NWhjvoLuldIWvU7Rsf4Th+jLj6nQOZjjJqY8Izj3FELYwO10WA\\\\\\\/In+710zD6mxH7DfIIRxZA6sGG2r+Y45h+NhX5jQpHFl4180i\\\\\\\/fUEfALgUhOjKpLu+\\\\\\\/LGZy13EjVoDxPXcz7JrNg11DOxO2JeIFeNElGY2BWbOXNvGdeGft\\\\\\\/O8HjUHXbV1LDy9aXEV9qltUMwWYujyLuFXbgqxhCxo18Oh0LWrwGwTq10Em4FF2rR88HhDpUp3hT9IWWxHxx\\\\\\\/5gdv2ROyCMxH7GfMhVI7DokWL+UXw5ffE9vZt3DHA7XY+onfSi1i8VJeU\\\\\\\/uFcmEDWpMY+1yKIyAMQgQadkLhK0vDp7pU9tObkP6LRc1dCoJQVKg7y6LM7b8ecXsuuZFA+UmyYspRi6aParZvtr5sLZNWpVhKMT+sKiYX2aXdboR7XUgYFafAysC4yGZ0kjLSpDYnjZeKHaajveRaU+rpd4\\\\\\\/olr\\\\\\\/U1fgjPB0dDpdUL5c4Ih1Q\\\\\\\/BbFalOHvVe7F7az6ZE1aabZhX7Wgc9fsZ7uWdlewCo+DTIiBbtU+drCvcUwkrvlRsXSuMCgURUj\\\\\\\/thNLY5XQrEaeAqeerx1XEpzNCeK8gwXxDpaLhI+l5RusgOtJu3hmRHmnXpsmgjTsujAYO12O3gQJsQyzgnuxa56baKxlDGUznLtkVcN1P7C2prwFX64gYATk2nCzu0kbtrZ3lXx1hDTiG\\\\\\\/3LDuAS\\\\\\\/jox4PDEZIGSm+cl\\\\\\\/IFRQfYkaOxseanKD4vGc45q7YgBEyRiqm2ScHOTDt7OGs+46ATEXSq6clEl3gEa8hIt9dMeTLBlxnkn0w4Pw6qQOneSM4k0XhDr4STGOIBGLJlA0cvwcveJT3oO3FvDGbI8w2JXykowpErigaRrJsQ2mHqyz66eId3+kfzm7BVusm\\\\\\\/PmsadFLZeLR2k1OGLjqI1GGrGWLMxqLihvHiElbovSehtCYwA21Sfgk6RDFEC+LIn042rJgLv0QLH80agJ9qKtu\\\\\\\/RKl\\\\\\\/KhNc5OVMGcGPvDdzJaLaT1ZsBGWRXXKSYwGnbpW11raNBF38GWwcb\\\\\\\/72uSdI0Ng0GFvUsUPxdItTh3tCDRC7UELbvl1AyE\\\\\\\/AZCIL0ciAaNAmMUMh728vmauoHvzxLPvGTL5kY7XOE9A+7Xzo3q1EFq0iA6crZueGEHNuVI0nVPuI2lp0xbRHGJxwWb5Il5OH7HeYQvV+x2zoTImMFFtih4HdULiVNoAxYgKdMLlvWwZ5SFHiZiC+X26ciMjH+0Y+YbsgYu5ElHOCqEQt8vgE64+piZpjqvkhD0gTxeANE2yNmYoo65x606BLGIK9VQqyil7KbJWplocpH8kkzX66smfOOkDrJfJ1tCrFbYBj3KX18z4eKFxyXBn5c9cLep6FVN7hVI2nPlNwPSJQG+\\\\\\\/opcmUhnx8GLSxTLoFY+BHNYVB8lNhdZgpUG\\\\\\\/qwxyaRujUxcA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022QhMcW9jDZpNVEOvSBHTsPA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227206f5b565d301c4178c5127f4446aabeb8e0471f7737c8219746527e82bd633\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WWGcJhxcMHsOKcRK97g4CaesQHQL38NJvg8ikn0Nk1uwsH7mcnE2Vogzd1asJS\\\\\\\/gSBJAe1TjnnlBMYuqHCUMJEh\\\\\\\/v3V1a5yfEidy2heZU3xOCIHK5IKYov\\\\\\\/3sa4S9JTX692Ym60DONGsx5HtjNzgimv8bAv4wwI9EIv9Tl3eExorj4rVEpE4A5eV+DMAYX\\\\\\\/fGut3iAIFHai1K+M7NaZfakr4yncrMURRQ\\\\\\\/XnC62Q8XUZrkKv3V\\\\\\\/2OWhGnX8l6tplo6LI4AVawchkF9TxTWBt1IpuwVL1rPPZLLWdJDfsV1BuQRd7sRB8+pzTiE7swPqMWM5\\\\\\\/9IqSlNIXfueho0T2xqVEbOjlTsNRWFKel0vtiqsE8nWdQmOZ3wgwc2afzJ8T5aZPGV0T7c2P3qPmJQ6UTeAK0O30lGuefu7M4KnasZTiasRoswIGQl5QgkxnybFu6FFVfCAzKNfDbJ7TYuoqybUTiES1NpevG6GKV7B1wNAkJoCf73ZZDaTvSPbjKepGQoJMToyWVX7Cen4mQqJISrs5FPHmUfLI4OYOZFMpOaVIYUVAzA8r4jiE8Bu6yrXhI0rgOvT62Fz+Tug9thjvltayVUE4setRr9jB41TeIxbmooyzSV2JJ1YY43xS9s7MyONQKUfk9Qc1VQnSplYpHNMGHGILUBR9zMXa6NlEhJnUMqwIXsJ2q6hr8h4R64DaQX+oJcLjyNgXFeVYyhO+d1yCgAW3o0itJJ12rAqYixbOLFw\\\\\\\/rjPyaTgr0IZRzIomFdqLIwhbtgiKym5DOysSpw7J1Etu3x5gvth5xWEzXcaNSmfzJNqpjpfr4v6kPstoXwJZJgNplWYjGirJDKSmuzOsncYDYeG5WEqs68gjI1VGV58M2yw\\\\\\\/iaI7dlRJTESd2YxpX8405oZ01YhAch0tQ+v3eLx68sZx2kqrBU30+oSG62v1xL7QB8LfJCW63ErsALvFyjWZIVSPC+JuFvoq\\\\\\\/toMOEVdWFrgl3oBVnGzTx1h8dE3oW39RTTajJEdA345IJQSNch7Yox\\\\\\\/HQmeQkqfD8A4b\\\\\\\/tzvumV0OlSqO\\\\\\\/q3OwhGxtoeBQCwMFce206WK5ZCNimKPNCz\\\\\\\/r2CvvcDXULDC3Du78\\\\\\\/NCOwNPywmpmtPsHCnI8FurGW5jCdQtbagJ8DRrNwwaHPiC+ulko74pNV5zbktFkn2IpM3ABzWhou1aCep8r5IFs8XYA6J1zmH\\\\\\\/PdGQCrIR2uPyXUuTFRCfqjASp\\\\\\\/IyeC8GIjhEke\\\\\\\/0lI9gTA36Z6U\\\\\\\/VJn94J4UEoEumcyRZeDzcuC1Vc5WvOEpv53AYuqT1qnqI9Z8Oqf6YAQR2RB0sck1gyj6QyTUYgh8R8ETNvYce5XyUf8xwyAgrYS20aKyg\\\\\\\/sH+tYvWDMvYKGtZAdiHmbfdd0qrWJObtGHTOIvJACmT9+hyytxcmtWLYTB87AAUqQoCkvstX85L9aFtogXWwW9UEfVDSMGigjI6gH6QRuEzmNq8hV40EVW5zrIFucLtJEgFd3pyug3py\\\\\\\/dDQj4pp21v1X6vzEZXvpJNvX4iHOhQmZkGlbCLyi3MEVv5aiubAc8QkaMAE7+jt03OvZ05S0qPXRGKzDjCShvpSx4NcQdOvtpzsMQPe51H5BzFu5di1HiTZVhONkifqvZWZ78Cf+rYEwe8TuvR\\\\\\\/IzhXm66dneXbZ3hBhFpsc6I58V2Cma6WSP7UECx4iDB6P+U+uinIe2PZ8W88oOBvRQRoqssZ4F\\\\\\\/1SqFnJG9ecj\\\\\\\/IzNopnYMZk67zNLWmx1HOpoJyblnm0ZzLDpVoutyX9xpqdkUb3+mUByjYPhwk9AVZmzpR6aSlLCk9Eliau1J9pvS+Pdpu9WokHO0XWXesDHU4dcqRFiM6gSlnSs0jxe42PK7N1o7Nv11eUQQnN\\\\\\\/TwnQqLBgTaALa2SCDHR6wnoNnWji9uHhiOdSqbdWzh4o8JprxDY\\\\\\\/XlIolBQJF4uz7nOFPeuSW2jWQVnwNB5aMvPeQTzCr7GhFvGXDE5BC+yWdDvg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022gD3\\\\\\\/AaLBGmQtFhBpFeiJbQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227942866d03588927f1746e07fd8c3de8ce6f54f096783788fe93f2b3367a4908\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491872 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/1/217" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-J1ZAI_LeMvOnklVr_nv8oPtfIkxIVZbSiD7hejkmeoM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/1/217" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pAd2w0YnDvAZkh1tcnanaPGt0POZCVqo5L6v6y0m77E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/1/203" title="A Strategy To Exploit Surrogate Sire Technology in Livestock Breeding Programs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/1/229" title="Wolbachia Infection Associated with Increased Recombination in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491872" id="node491872" data-pisa="ggg;9/1/217" data-pisa-master="ggg;g3.118.200779" data-apath="/ggg/9/1/217.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Medaka Population Genome Structure and Demographic History Described via Genotyping-by-Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Takafumi Katsumura</span>, <span class="highwire-citation-author" data-delta="1">Shoji Oda</span>, <span class="highwire-citation-author" data-delta="2">Hiroshi Mitani</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0746-2729" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hiroki Oota</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">217-228;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200779 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491872" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Takafumi Katsumura </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Anatomy, Kitasato University School of Medicine, Sagamihara, Kanagawa 252-0374, Japan</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Graduate School of Natural Science and Technology, Okayama University, Okayama, Okayama 700-8530, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Takafumi%2BKatsumura%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Katsumura%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATakafumi%2BKatsumura%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tk@okayama-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">tk@okayama-u.ac.jp</a>
<a href="mailto:hiroki_oota@med.kitasato-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">hiroki_oota@med.kitasato-u.ac.jp</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Shoji Oda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Integrated Biosciences, Graduate School of Frontier Sciences, University of Tokyo, Kashiwa, Chiba 277-8562, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shoji%2BOda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oda%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShoji%2BOda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Hiroshi Mitani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Integrated Biosciences, Graduate School of Frontier Sciences, University of Tokyo, Kashiwa, Chiba 277-8562, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hiroshi%2BMitani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mitani%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHiroshi%2BMitani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hiroki Oota </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Anatomy, Kitasato University School of Medicine, Sagamihara, Kanagawa 252-0374, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hiroki%2BOota%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oota%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHiroki%2BOota%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0746-2729" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hiroki Oota</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tk@okayama-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">tk@okayama-u.ac.jp</a>
<a href="mailto:hiroki_oota@med.kitasato-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">hiroki_oota@med.kitasato-u.ac.jp</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/1/217" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491872" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491872/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/1/217.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491872" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491872/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/1/217.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:491872" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:491872/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/1/217.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491872" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491872/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/1/217.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491872" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491872/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Medaka is a model organism in medicine, genetics, developmental biology and population genetics. Lab stocks composed of more than 100 local wild populations are available for research in these fields. Thus, medaka represents a potentially excellent bioresource for screening disease-risk- and adaptation-related genes in genome-wide association studies. Although the genetic population structure should be known before performing such an analysis, a comprehensive study on the genome-wide diversity of wild medaka populations has not been performed. Here, we performed genotyping-by-sequencing (GBS) for 81 and 12 medakas captured from a bioresource and the wild, respectively. Based on the GBS data, we evaluated the genetic population structure and estimated the demographic parameters using an approximate Bayesian computation (ABC) framework. The genome-wide data confirmed that there were substantial differences between local populations and supported our previously proposed hypothesis on medaka dispersal based on mitochondrial genome (mtDNA) data. A new finding was that a local group that was thought to be a hybrid between the northern and the southern Japanese groups was actually an origin of the northern Japanese group. Thus, this paper presents the first population-genomic study of medaka and reveals its population structure and history based on chromosomal genetic diversity.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/local-population" class="hw-term hw-article-keyword hw-article-keyword-local-population" rel="nofollow">local population</a></li><li class="kwd"><a href="/keyword/freshwater-fish" class="hw-term hw-article-keyword hw-article-keyword-freshwater-fish" rel="nofollow">freshwater fish</a></li><li class="kwd"><a href="/keyword/demography" class="hw-term hw-article-keyword hw-article-keyword-demography" rel="nofollow">demography</a></li><li class="kwd"><a href="/keyword/rad-seq" class="hw-term hw-article-keyword hw-article-keyword-rad-seq" rel="nofollow">RAD-seq</a></li><li class="kwd"><a href="/keyword/bioresource" class="hw-term hw-article-keyword hw-article-keyword-bioresource" rel="nofollow">bioresource</a></li></ul><p id="p-3">Medaka (<em>Oryzias latipes</em>) is a small fresh-water fish native to East Asia that has attracted attention as a vertebrate model for population genetics (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Oota and Mitani 2011</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Spivakov <em>et al.</em> 2014</a>). Wild medaka populations have been maintained in certain universities and research institutes as a bioresource (hereafter, wild lab stocks) with funding from the Japanese government since 1985 (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Shima <em>et al.</em> 1985</a>). These populations consist of more than 100 local-wild populations that have various phenotypic traits (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Watanabe-Asaka <em>et al.</em> 2014</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Igarashi <em>et al.</em> 2017</a>) and abundant genetic diversity (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kasahara <em>et al.</em> 2007</a>). Geographical features affect the population structure of organisms. Seas and mountains restrict the movement of terrestrial animals and freshwater fish, and the climate also changes the breeding timing and foraging environment. Particularly, noticeable climate differences are observed within island groups, such as the Japanese archipelago, where a large latitudinal difference exists between the southern and northern ends. Therefore, animals are exposed to various selective pressures according to the geographic environment. The local populations have differentiated into local groups by genetic drift, resulting in genetically divergent groups. Medakas stocked as a bioresource are thought to have retained the genetically adapted traits they acquired from various environments of the Japanese archipelago.</p><p id="p-4">Our ultimate goal in exploiting medaka characteristics is to establish an experimental system for testing the functional differences between alleles detected by, for instance, genome-wide association studies. Once genetic polymorphisms related to phenotypic traits in medaka are detected, <em>in vivo</em> experiments, such as genome editing experiments (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ansai and Kinoshita 2014</a>), can be conducted to understand the functions of the genetic variants (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Shimmura <em>et al.</em> 2017</a>). Revealing the functional difference between alleles in wild populations allows us to infer the role of genetic polymorphisms in human homologous genes (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Igarashi <em>et al.</em> 2017</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Shimmura <em>et al.</em> 2017</a>).</p><p id="p-5">Most previous analyses of medaka genetic diversity and population structure have been conducted using mitochondrial DNA (mtDNA). Medaka is divided into four mitochondrial groups: the northern Japanese (N.JPN), southern Japanese (S.JPN), eastern Korean (E.KOR) and western Korean/Chinese groups (W.KOR) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sakaizumi <em>et al.</em> 1980</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">1983</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sakaizumi 1986</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Katsumura <em>et al.</em> 2009</a>). There is a claim that N.JPN and S.JPN are different species because of slightly different morphological characteristics (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Asai <em>et al.</em> 2011</a>) and deep divergence time based on mtDNA sequences (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Setiamarga <em>et al.</em> 2009</a>). However, it is still controversial, because the divergence time (approximately 18 MYA) is able to change using the different calibration points (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Miya <em>et al.</em> 2013</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Broughton <em>et al.</em> 2013</a>), N.JPN and S.JPN have no reproductive isolation (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Sakaizumi <em>et al.</em> 1992</a>), and fertile when mating even in wild environment (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Iguchi <em>et al.</em> 2018</a>), strongly suggesting that N.JPN and S.JPN are geographical subspecies. Furthermore, although previous allozyme-based studies show that an ambiguous group exists in the geographic boundary region known as Tajima-Tango area (see also <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figures 1</a> and S1), between N.JPN and S.JPN, which is thought to be a hybridization of the two groups (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sakaizumi 1984</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Takehana <em>et al.</em> 2016</a>), the process has not been fully verified. Further investigation of the local population structures of medaka in Japan and east Asia would provide evidences for more precise and deep understanding of natural history of medaka in the Japanese archipelago.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/217/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Map of the original locations of the wild lab stocks and wild-captured medakas. In the upper central map, the enlarged red frame shows the boundary region between S.JPN and N.JPN. Each color represents the mtDNA and allozyme-based groups shown in the left upper inset box. The numbers on the map are consistent with the population IDs, with the names on the right bottom inset box." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-64091323" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Map of the original locations of the wild lab stocks and wild-captured medakas. In the upper central map, the enlarged red frame shows the boundary region between S.JPN and N.JPN. Each color represents the mtDNA and allozyme-based groups shown in the left upper inset box. The numbers on the map are consistent with the population IDs, with the names on the right bottom inset box.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/217/F1.medium.gif" width="440" height="324"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/1/217/F1.medium.gif" width="440" height="324"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/217/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/217/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491805" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-6" class="first-child">Map of the original locations of the wild lab stocks and wild-captured medakas. In the upper central map, the enlarged red frame shows the boundary region between S.JPN and N.JPN. Each color represents the mtDNA and allozyme-based groups shown in the left upper inset box. The numbers on the map are consistent with the population IDs, with the names on the right bottom inset box.</p><div class="sb-div caption-clear"></div></div></div><p id="p-7">Based on mtDNA cytochrome <em>b</em> gene sequences, N.JPN and S.JPN are composed of 3 and 11 subgroups, respectively (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Takehana <em>et al.</em> 2003</a>). Each subgroup is composed of local populations, and the between-population genetic diversities are greater than the within-population genetic diversities, indicating substantial genetic differentiation between local populations (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Katsumura <em>et al.</em> 2009</a>). Their habitat environments are also largely different. For instance, there is a large amount of snowfall in the habitats of N.JPN, where the breeding season is short. The habitat of S.JPN is wider than that of N.JPN, and its climate environment is also diverse; <em>e.g.</em>, the differences of the annual average temperature and rainfall between the south end (Nago in Okinawa) and north end (Ichinoseki in Iwate) inhabited by S.JPN were 11.6Â° and 988 mm in 2017, respectively (<a href="http://www.jma.go.jp/jma/indexe.html">http://www.jma.go.jp/jma/indexe.html</a>). The phylogenetic data from these studies also suggested that N.JPN and S.JPN have been spreading in the Japanese archipelago at different times. Particularly, the origin of S.JPN, which has the largest habitat, has been suggested to be the northern part of Kyushu Island based on mtDNA (âOut of Northern Kyushuâ hypothesis) (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Katsumura <em>et al.</em> 2012</a>). However, mtDNA is insufficient to describe the population structure and history because of its single locus. Genetic diversity based on chromosomes is essential to understand the medaka population structure, but the comprehensive data of chromosomes are still limited.</p><p id="p-8">An inference of the population structure and history induced by genome-wide information is more robust than that induced by single loci. To unravel the population structure based on chromosomes, we comprehensively performed a population-genetic analysis based on chromosomal single-nucleotide polymorphisms (SNPs) using all 81 local populations maintained as wild lab stocks at the University of Tokyo. We examined an individual sampled from each population stock and 12 wild individuals captured from the northern part of Kyushu Island, where (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Katsumura <em>et al.</em> 2012</a>) the medakas currently distributed along the entire Pacific side of the Japanese archipelago originated, to estimate each populationâs time of expansion from Northern Kyushu. To obtain SNP data, we conducted genotyping-by-sequencing (GBS) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Narum <em>et al.</em> 2013</a>) using a high-throughput sequencer. This method, which allowed us to cost-effectively genotype tens of thousands of SNPs (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andrews <em>et al.</em> 2016</a>), resulted in an accurate enhancement of the population-genetic estimations because the variance in the demographic parameter estimates decreased when we used many SNPs. Eventually, we obtained more than ten thousand SNPs from eighty-one wild lab stocks and twelve wild-captured medakas. Here, we re-evaluated the medaka genetic diversity and population structures based on these SNPs and reconstructed the population history by assessing three demographic events. These data redefine the medaka local groups and provide a basis of the population history for discussing the role of phenotype-associated alleles in the context of adaptation.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Samples</h3><p id="p-9">We sampled 81 male medakas from 81 wild lab stocks (14 from N.JPN, 56 from S.JPN, 5 from Tajima-Tango in the geographic boundary region between N.JPN and S.JPN, 3 from E.KOR, and 3 from W.KOR; note that these groupings come from previous mtDNA sequences and allozyme patterns) at the University of Tokyo in 2014. The lab stocks originated from geographically distinct populations in East Asia and have been maintained since 1985 (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Shima <em>et al.</em> 1985</a>) as closed colonies in the Graduate School of Frontier Sciences, the University of Tokyo, Kashiwa City (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). These lab stocks maintain the genetic diversity originating from their habitat (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Katsumura <em>et al.</em> 2014</a>; <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Igarashi <em>et al.</em> 2017</a>) and show less diversity within populations than between populations (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Katsumura <em>et al.</em> 2009</a>). Because of these characteristics, we considered that one individual sampled from the lab stock would be adequate to represent its originating population. In addition, we used two wild-captured medaka populations from the Saga Prefecture in the northern part of Kyushu, Japan. One population was the Ogi (SO: Saga Ogi) population from Southern Saga (S.Saga), and the other population was the Umejiro (US: Umejiro in Saga) population from Northern Saga (N.Saga), both of which were captured in September 2010 (see <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Katsumura <em>et al.</em> 2012</a>). Six of 48 medakas from each wild-captured population were selected randomly and analyzed via GBS.</p></div><div id="sec-3" class="subsection"><h3>DNA extraction and genotyping-by-sequencing</h3><p id="p-10">One-third of the medaka body was dissolved in a 600 Âµl lysis buffer containing 1.24% SDS, 0.124 M EDTA and 0.062 mg/ml proteinase K (final concentrations). The total genomic DNA was extracted and purified using phenol-chloroform and isopropanol precipitation. After a 70% EtOH wash, an isolated DNA pellet was resuspended in 100 Î¼l TE buffer and then treated with RNase A (final conc. 1 mg/ml) for 1 hr at room temperature. Then, the DNA was purified again using phenol-chloroform and isopropanol precipitation. For ninety-three samples, the GBS process was outsourced to Macrogen Japan in Kyoto. The procedures for constructing libraries and performing Illumina HiSeq 2000 single-end sequencing were the same as those described by Poland <em>et al.</em> (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Poland <em>et al.</em> 2012</a>) except for the use of the restriction enzyme ApeKI instead of <em>Eco</em>RIâ<em>Msp</em>I. The sequence lengths were 51 bp and included each individual in-line barcode (4â9 bp) for the individual sample. The data have been submitted to the DDBJ Sequence Read Archive (DRA) database under project accession ID: DRA006353.</p></div><div id="sec-4" class="subsection"><h3>Quality filtering and SNP extraction</h3><p id="p-11">Our single-end reads were filtered using <em>FASTQ Quality Filter</em> in <em>FASTX-Toolkit version 0.0.13</em> (<a href="http://hannonlab.cshl.edu/fastx_toolkit/download.html">http://hannonlab.cshl.edu/fastx_toolkit/download.html</a>) using the following options: -Q 33 -v -z -q 30 -p 90. The draft genome of the medaka sequenced by the PacBio sequencer (Medaka-Hd-rR-pacbio_version2.0.fasta; <a href="http://utgenome.org/medaka_v2/#!Assembly.md">http://utgenome.org/medaka_v2/#!Assembly.md</a>) was used to align the reads using <em>BWA backtrack 0.7.12-r1039</em> (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li and Durbin 2009</a>) using the â-n 0.06â option. After the mapping process, the multi-mapped reads were removed using <em>Samtools v1.2</em> (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2009</a>) and the â-Sq 20â option. Following this pipeline, sequencing of the ApeKI-digested GBS libraries generated an average of 3.06 million reads per individual before any quality filtering. The read numbers ranged from 1.89 to 4.10 million reads per individual. After quality filtering, 2.76 million (90.2%) sequences per individual on average were retained, and 0.30 million (9.8%) sequences were eliminated. The retained sequences presented a mean quality score of 38.7 and a GC content of 47.8%. An average of 1.84 million of the retained sequences (66.7%) aligned to the medaka chromosomal genome, and 0.92 million sequences (33.3%) were not aligned and discarded because they mapped to non-chromosomal loci (mitochondrial genome and unanchored contigs) and multiple loci.</p><p id="p-12">The <em>Stacks</em> pipeline (version 1.35) and the Stacks workflow (<a href="https://github.com/enormandeau/stacks_workflow">https://github.com/enormandeau/stacks_workflow</a>) were used to generate SNPs and sequences for each individual separately (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Catchen <em>et al.</em> 2011</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">2013</a>). The selected <em>Stacks</em> parameters were as follows: minimum stack depth (-m), 3; and number of mismatches when building the catalog (-n), 1. In the ârxâ step of <em>Stacks</em>, we used the bounded SNP model set to 0.1, the Îµ upper bound set to 0.1 and the log likelihood set to 10. Following this pipeline, five datasets were constructed using the <em>population</em> program in <em>Stacks</em>.</p><p id="p-13">The first was the âPopStatâ dataset, which was generated using the â-p 5 -r 0.66â options and mitochondrial grouping (N.JPN, Tajima-Tango, S.JPN, E.KOR, W.KOR; see also <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>) to calculate the population-genetic statistics (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) using the loci shared across all groups and sequenced in one or more populations in each group.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491934/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491934/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary genetic statistics for five populations using âPopStatâ dataset. These statistics include the mean number of individuals genotyped at each locus, the number of variable sites in five populations (Variants), the number of polymorphic sites in each population (Polymorphic) and the number of variable sites unique to each population (Private). The number in parenthesis is Standard Error</span><div class="sb-div caption-clear"></div></div></div><p id="p-14">The second was the âGlobalâ dataset, which was generated using the â-p 58 -r 1.00â option and without the mitochondrial grouping, to examine the phylogenetic relationships between the geographic populations and the population structure within the species. Additionally, nucleotide diversity in each individual were calculated using this data set to compare genetic diversity between lab-stocks and wild-captured medaka.</p><p id="p-15">The third and the fourth were the âHZ-1â and âHZ-2â datasets (âHZâ is the abbreviation of âHybrid zoneâ), respectively, including the 15 boundary populations (N.JPN: Kaga, Maiduru, Miyadu, Obama, and Sabae; Tajima-Tango: Amino, Hamasaka, Kinosaki, Kumihama, and Toyooka; and Honshu: Ayabe, Iwami, Kasumi, Matsue, and Tottori) with or without the Kyushu populations (Kyushu: Fukue, Hiwaki, Izumi, Kadusa, and Kikai). These datasets were generated using the â-p 3 -r 0.70â (without Kyushu) and the â-p 4 -r 0.70â (with Kyushu) option, respectively, to assess the genetic population structure and the history of the boundary population for the Tajima-Tango group.</p><p id="p-16">The fifth was the âLocalâ dataset, including two Kyushu deme samples: one was Umejiro (US), which was sampled in the northern part of the Saga prefecture, and the other was Ogi (SO), which was sampled in the southern part of the Saga prefecture. These samples were used to estimate the time of Honshuâs population divergence from Kyushu to infer the timing of the âOut of Northern Kyushuâ event (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Katsumura <em>et al.</em> 2012</a>).</p><p id="p-17">Note that we define the terms âdeme samplesâ and ânon-deme samplesâ as âsamples from the local-wild populationâ and âsamples from the wild lab stocksâ, respectively (see details in <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Katsumura <em>et al.</em> 2009</a>).</p><p id="p-18">Against those datasets (except âPopStatâ and âHZ-1â), the SNPs with strong linkage disequilibrium were randomly removed, one from each pair of SNPs with r<sup>2</sup> &gt; 0.2, using Plink1.9 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chang <em>et al.</em> 2015</a>) and the ââindep-pairwise 12.5 5 0.2âautosome-num 24â option. These values were set based on a medaka population genomics study (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Spivakov <em>et al.</em> 2014</a>). Finally, the âGlobalâ, âHZ-2â and âLocalâ datasets contained 8,361 SNPs out of 13,177 SNPs, 1,014 SNPs out of 2,661 SNPs and 698 SNPs out of 2,246 SNPs, respectively.</p></div><div id="sec-5" class="subsection"><h3>Genetic clustering analysis</h3><p id="p-19">To obtain a genetic overview of the relationship of medaka geographic populations, we performed a principal component analysis (PCA) of the âGlobalâ dataset as implemented in the <em>SNPRelate</em> program (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zheng <em>et al.</em> 2012</a>) in R version 3.2.2. Additionally, to examine the genetic relationships between medakas in the Japanese archipelago, we used the subdataset without the E/W.KOR and Chinese populations, which included 7,126 SNPs. We also performed a model-based genetic clustering analysis using <em>ADMIXTURE v1.23</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alexander <em>et al.</em> 2009</a>) to estimate the proportions of ancestral medaka populations. We ran 50 replicates with random seeds for the number of clusters (K) from 1 to 9 and calculated the mean of the lowest fivefold cross-validation errors for each K (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jeong <em>et al.</em> 2016</a>). Values of K = 4, 5, and 6 showed the first-, second- and third-lowest fivefold cross-validation errors, respectively (Figure S2). The results of the genetic clustering analyses were visualized by a ggplot2 package in R (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wickham 2011</a>).</p></div><div id="sec-6" class="subsection"><h3>Reconstructing the phylogenetic tree using the maximum likelihood method</h3><p id="p-20">We considered an individual as representative of a population and generated the individual sequences using the <em>population</em> program with the â-p 58 -r 1.00âphylip_allâ option in the <em>Stacks</em> pipeline. The dataset included 4,638 partitions and 217,986 bp of nucleotide sequences to compensate for the loci that could not be sequenced by invariable sites across all samples and âNâ at the variable site. The dataset including 4,638 partitions and 217,986 bp nucleotide sequences was analyzed via the <em>IQ-TREE</em> program (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Nguyen <em>et al.</em> 2015</a>) to reconstruct a maximum likelihood tree with model selection for each partition (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chernomor <em>et al.</em> 2016</a>) and 1,000 SH-aLRT/ultrafast-bootstrappings (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Guindon <em>et al.</em> 2010</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Minh <em>et al.</em> 2013</a>). Then, we used the <em>FigTree</em> program (<a href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtree/</a>) to visualize the phylogenetic tree.</p></div><div id="sec-7" class="subsection"><h3>Inference of demographic parameters of the populations in and around Tajima-Tango and the time of âOut of Northern Kyushuâ based on the ABC framework</h3><p id="p-21">Using the HZ-2 dataset, which added five Kyushu populations to 15 boundary populations of the HZ-1 data set, the population history, including the possibility that the Tajima-Tango group occurred by the admixture of N.JPN and S.JPN, was inferred from 1,014 SNPs using <em>DIYABC ver2.1.0</em> (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cornuet <em>et al.</em> 2014</a>). We tested four evolutionary scenarios in which N.JPN and S.JPN diverged from an ancestral population at time t3: (I) Tajima-Tango originated in N.JPN: Honshu and Kyushu diverged at time t2, and then Tajima-Tango diverged from N.JPN at time t1 (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Takehana <em>et al.</em> 2016</a>); (II) Admixture of N.JPN and Honshu: Honshu and Kyushu diverged at time t2, and then Tajima-Tango occurred at time t1 by an admixture with rate r between N.JPN and Honshu (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Sakaizumi 1984</a>); (III) N.JPN originated in Tajima-Tango: Honshu and Kyushu diverged at time t2 and then N.JPN diverged from Tajima-Tango at time t1; and (IV) Honshu diverged from Kyushu and then Tajima-Tango occurred by an admixture with Honshu of rate r. For simplicity, all populations were assumed to have constant effective sizes in each lineage (<em>i.e.</em>, no bottleneck and expansion). The same prior parameters were defined for four scenarios based on previous studies, and the prior distribution of each parameter is presented in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. In addition, we set the conditional constraint as follows: t3 &gt; t2, t3 &gt; t1 and t2 â¥ t1. In total, four million simulations were run, which provided approximately one million simulations for each scenario.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491818/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491818/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Demographic parameters estimated by DIYABC under three scenarios. Performance of each estimation was evaluated by RRMISE (the square Root of the Relative Mean Integrated Square Error), RMeanAD (the Relative Mean Absolute Deviation) and RRMSE (the square Root of the Relative Mean Square Error), which are output from DIYABC option âCompute bias and mean square error.â The numbers with or without parentheses in the columns âPerformances for estimating posterior distributions of parametersâ are those of the statistics computed from the prior or posterior distribution of parameters, respectively</span><div class="sb-div caption-clear"></div></div></div><p id="p-22">The summary statistics for all SNP loci included the proportion of loci with null gene diversity, mean gene diversity across polymorphic loci, proportion of loci with null Neiâs distance between the two samples, variance across loci of non-null Neiâs distances between two samples, proportion of loci with null admixture estimates, mean across loci of non-null admixture estimates, variance across loci of non-null admixture estimates and mean across all locus admixture estimates. The 1% simulated datasets closest to the observed dataset were used to estimate the posterior parameter distributions through a weighted local linear regression procedure (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Beaumont <em>et al.</em> 2002</a>). Scenarios were compared by estimating their posterior probabilities using the direct estimation and logistic regression methods implemented in <em>DIYABC</em> (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Cornuet <em>et al.</em> 2014</a>). We also estimated the time for âOut of Northern Kyushuâ using the Local dataset that included 698 SNPs from <em>DIYABC</em>. We inferred the following time-based simple evolutionary scenario: N.Saga diverged from S.Saga at time t because this divergence event was consistent with the âOut of Northern Kyushuâ event (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Katsumura <em>et al.</em> 2012</a>). In this case, all populations were assumed to have unchangeable effective sizes in each lineage.</p><p id="p-23">The prior distribution of each parameter is presented in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, and the 1% simulated datasets closest to the observed dataset were used to estimate the posterior parameter distributions. We evaluated the accuracy of the demographic parameter estimation by calculating accuracy indicators (<em>Bias and mean square error</em> option) in <em>DIYABC</em> and accepted the parameters that were non-scaled and scaled by the mean effective population size for the HZ-2 and Local datasets, respectively. Using the infinite-sites model in the Wright-Fisher populations of a constant size, a rough value of <em>N</em><sub>e</sub> was estimated using the relationship <em>Ï</em> = 4<em>N</em><sub>e</sub><em>Âµ</em> (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Tajima 1983</a>), where <em>Ï</em> is genome-wide nucleotide diversity and Âµ is the mutation rate per site per generation (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Osada 2015</a>). We calculated <em>N</em><sub>e</sub> using the above formula with the following values: <em>Âµ</em> = 10<sup>â9</sup>/site/generation; generation = 1 year, which resulted in a mean effective population size from all presented demes in northern Kyushu of 1,275,000.</p></div><div id="sec-8" class="subsection"><h3>Reconstruction of the phylogenetic tree for the partial mitochondrial DNA sequence</h3><p id="p-24">To examine the mitochondrial introgression in the tested samples, we generated and analyzed the mitochondrial DNA (mtDNA) partial sequences from the GBS-read data. We mapped the reads to the complete mtDNA sequence of âClade Câ (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Matsuda <em>et al.</em> 1997</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Takehana <em>et al.</em> 2003</a>), which diverged genetically from S.JPN and N.JPN, to eliminate the mapping bias that occurs when genetically distant sequences are used as a reference. The genomic DNA of âClade Câ was extracted in <a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Katsumura <em>et al.</em> (2009)</a>. The complete mtDNA sequence was determined as follows. Using PCR, five DNA fragments of mitochondrial DNA were amplified: fragment 1â4,556 bp; fragment 2â4,527 bp; fragment 3-1â4,589 bp; fragment 3-2â4,504 bp; and fragment 4â4,546 bp (Figure S3). Table S1 describes the primers, which were designed based on the inbred strain Hd-rR complete mtDNA sequence (accession number: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AP008938&amp;atom=%2Fggg%2F9%2F1%2F217.atom">AP008938</a>). Approximately 20 ng of the genomic DNA was used as a template for the PCR assay in a 50 Î¼l solution containing dNTP at 0.2 mM, 0.2 Î¼M of each of primer, 0.75 U of EX Taq polymerase HS (TaKaRa Shuzo Co.), and the reaction buffer attached to the polymerase. The reactions were conducted in a TaKaRa PCR Thermal Cycler Dice (TaKaRa Shuzo Co.) using the following protocol: an initial denaturing step at 95Â° for 2 min, 40 cycles of denaturation at 95Â° for 30 sec, annealing at 60Â° for 30 sec, extension at 72Â° for 300 sec, and a final extension step at 72Â° for 5 min. The PCR products were diluted 20-fold and used as templates in the sequencing reaction (following the commercial protocol) with thirty-three primers (Table S1 and Figure S3) and then analyzed in an ABI 3500xL Genetic Analyzer (Life Technologies). The complete mtDNA sequence of âClade Câ was reconstructed using <em>SeqMan Pro 10.1.2.20</em> (DNASTAR) and deposited into the international DNA database DDBJ/EMBL/GenBank (accession number: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=LC335803&amp;atom=%2Fggg%2F9%2F1%2F217.atom">LC335803</a>).</p><p id="p-25">The âClade Câ complete mtDNA sequence was used to align the reads using <em>BWA backtrack 0.7.12-r1039</em> (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Li and Durbin 2009</a>) using the â-n 5â option. After the mapping process, to remove the multi-mapped reads, we used <em>Samtools v1.2</em> (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2009</a>) using the â-Sq 20â option. Then, we used <em>Stacks</em> with the â-m 3â option for minimum stack depth (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Catchen <em>et al.</em> 2011</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">2013</a>) and then obtained the 322 bp nucleotide sequences. The loci where the sequence was missing were filled in with âNâ. In addition, the nucleotide position showing the multi-allelic state was replaced by âNâ. Phylogenetic trees were constructed using the neighbor-joining (NJ) method (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Saitou and Nei 1987</a>) with the program MEGA5 (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Tamura <em>et al.</em> 2011</a>). The evolutionary distances were calculated using the Jukes-Cantor method (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jukes and Cantor 1969</a>). The analysis also involved 13 nucleotide sequences from the DNA database (See Figure S4). All ambiguous positions were removed for each sequence pair. The reliability of the tree was evaluated using 1000 bootstrap replicates (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Felsenstein 1985</a>).</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-26">Wild lab-stocks are available upon request (e-mail to Hiroshi Mitani: <span class="em-link"><span class="em-addr">mitani{at}edu.k.u-tokyo.ac.jp</span></span>). GBS data are available at DDBJ Sequence Read Archive (DRA) database with the accession number: DRA006353. The complete mtDNA sequence is available at the international DNA database DDBJ/EMBL/ GenBank with the accession number: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=LC33580.&amp;atom=%2Fggg%2F9%2F1%2F217.atom">LC33580.</a> Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7358042">https://doi.org/10.25387/g3.7358042</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Genome-wide genetic diversity of five groups of Oryzias latipes</h3><p id="p-27">To assess the genetic diversity based on chromosomal SNPs within known mitochondrial groupsâthe northern Japanese (N.JPN), southern Japanese (S.JPN), eastern Korean (E.KOR) and western Korean/Chinese (W.KOR) groupsâ and hybridization group which is Tajima-Tango group, their population-genetic summary statistics were calculated using the âPopStatâ dataset that included the alignment regions across all groups and contained approximately 45 kb of sequences and 2,453 SNPs (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). All groups showed major allele frequencies and heterozygosities of 0.940â0.983 and 0.023â0.054, respectively, indicating many rare alleles and homozygous sites in our wild lab-stocks. W.KOR showed the highest nucleotide diversity (0.0053 Â± 0.0003) in all lab-stocks originated from East Asia. S.JPN showed the highest nucleotide diversity (0.0036 Â± 0.0001) and N.JPN the lowest in the Japanese archipelago (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), which is consistent with the diversity based on mtDNA (<a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">Katsumura <em>et al.</em> 2009</a>). To evaluate the influence for had been maintained as closed colony, we calculated the nucleotide diversity in each individual (that is mean heterozygosity per site). Comparing these lab stocks with wild-captured medakas (Ogi and Umejiro populations) by nucleotide diversity, lab-stocks, excluding Yongcheon and Shanghai, showed the lower values than wild ones (Figure S5).</p></div><div id="sec-12" class="subsection"><h3>Genetic clustering based on genome-wide SNPs inferring medaka population structure</h3><p id="p-28">To reveal the medaka population structure based on the chromosomal SNPs, we performed genetic clustering analyses using the âGlobalâ dataset. First, we investigated the genetic relationship by a principle component analysis (PCA) using 8,361 SNPs on 24 chromosomes. The PCA showed that the wild lab-stocks were divided into two clusters, N.JPN/Tajima-Tango and S.JPN and others from E/W.KOR, which is similar to the grouping based on the mtDNA data (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Takehana <em>et al.</em> 2003</a>; <a id="xref-ref-22-7" class="xref-bibr" href="#ref-22">Katsumura <em>et al.</em> 2009</a>) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figures 2A</a> and S6). E/W.KOR individuals were plotted dispersedly compared with other clusters, and Yongcheon (E.KOR) was close to N.JPN/Tajima-Tango. As the third principle component (PC3) was divided into E.KOR and W.KOR, the chromosomes in these mtDNA groups were differentiated into each other.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/217/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of principle component analysis (PCA) using SNPs in East Asia (A) and the Japanese archipelago (B). Each plot shows PC1 vs. PC2. The population names of each point are described in Figure S7." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-64091323" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Results of principle component analysis (PCA) using SNPs in East Asia (A) and the Japanese archipelago (B). Each plot shows PC1 &lt;em&gt;vs.&lt;/em&gt; PC2. The population names of each point are described in Figure S7.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/217/F2.medium.gif" width="440" height="176"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/1/217/F2.medium.gif" width="440" height="176"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/217/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/217/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491807" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Results of principle component analysis (PCA) using SNPs in East Asia (A) and the Japanese archipelago (B). Each plot shows PC1 <em>vs.</em> PC2. The population names of each point are described in Figure S7.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Although Tajima-Tango was considered to be a hybridization group between N.JPN and S.JPN (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Sakaizumi 1984</a>), all individuals from Tajima-Tango were plotted with those from N.JPN as a cluster (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). Next, for a fine-scale mapping of Japanese medaka, we performed the PCA based on 7,126 SNPs, excluding E/W.KOR individuals (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). The PCA showed that the Kyushu populations (Kudamatsu, Ogi, Izumi, Hiwaki, Kikai, Nago, Gushikami, Hisayama, Umejiro, Ashibe, Arita, Kusu and Nobeoka) in S.JPN were dispersed along the PC2 axis (Figure S7), while that Tajima-Tango overlapped with N.JPN again, even though we used up to PC5 (Figure S6). This result indicated that Tajima-Tango was not considerably differentiated from N.JPN on chromosomes, suggesting that S.JPN was differentiated within the groups and the genetic diversity was especially high among the Kyushu populations (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>).</p><p id="p-31">To examine the phylogenetic relationship among 83 populations, we constructed a maximum likelihood (ML) tree based on 4,638 short fragment sequences (comparative nucleotide sequence length: 217,986 bp) using <em>IQ-TREE</em>. The maximum likelihood tree showed relatively high bootstrap values on each branch and almost the same topology as previous trees based on mtDNA (<a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Takehana <em>et al.</em> 2003</a>; <a id="xref-ref-22-8" class="xref-bibr" href="#ref-22">Katsumura <em>et al.</em> 2009</a>), except for Yongcheon (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figures 3</a> and S8; see also Discussion). Medakas from the Korean peninsula were genetically close to each other in both groups, whereas those from Shanghai, which have been classified into W.KOR based on mtDNA studies, diverged from the other individuals from W.KOR. The medakas in the Japanese archipelago were divided into two major clusters similar to mtDNA. One was the N.JPN/Tajima-Tango, and the other was S.JPN. The N.JPN/Tajima-Tango cluster was further divided into two submajor clusters with a 100% bootstrapping value in the ML tree based on concatenated sequences obtained from GBS, although N.JPN and Tajima-Tango overlapped in PCA, which reduced SNP information, <em>i.e.</em>, N.JPN and Tajima-Tango diverged on the whole chromosomal sequenced regions. S.JPN was also further divided into two sub-major clusters: Kyushu-only and Kyushu &amp; others, which included 14 Kyushu populations, represented in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> by an open and a closed red circle, respectively. While the Kyushu-only cluster diverged into the northern and the southern Kyushu, the Kyushu &amp; others cluster diverged to the Pacific side of eastern and the northern Honshu (main-island Japan), which included several clusters supported by high bootstrapping value (SH-aLRT â¥ 80% and UFboot â¥ 95%, Figure S8). Finally, we characterized the S.JPN ancestry in the context of East Asian genetic diversity of medaka by performing an <em>ADMIXTURE</em> analysis, which is a model-based unsupervised genetic clustering method. With the optimal number of ancestral components (K = 4), S.JPN medakas were assigned to two distinct ancestries (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figures 3</a> and S2). In suboptimal runs with more ancestral components (K = 5, 6), only S.JPN in Honshu (main-island Japan) was assigned to the other ancestral components found in the northern Kyushu populations. This analysis indicated that genetic diversities in the northern Kyushu populations were the highest among S.JPN. Thus, the genetic clustering analyses based on genome-wide SNP data strongly suggested that S.JPN spread from northern Kyushu to Honshu.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/217/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic tree using the maximum likelihood method and an ancestry barplot with ADMIXTURE analysis. Red closed and open circles represent the northern and southern Kyushu populations, respectively. &#x201C;Taj-Tan&#x201D; in the tree is the abbreviation for Tajima-Tango." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-64091323" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Phylogenetic tree using the maximum likelihood method and an ancestry barplot with ADMIXTURE analysis. Red closed and open circles represent the northern and southern Kyushu populations, respectively. &#x201C;Taj-Tan&#x201D; in the tree is the abbreviation for Tajima-Tango.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/217/F3.medium.gif" width="440" height="286"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/1/217/F3.medium.gif" width="440" height="286"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/217/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/217/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491824" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Phylogenetic tree using the maximum likelihood method and an ancestry barplot with ADMIXTURE analysis. Red closed and open circles represent the northern and southern Kyushu populations, respectively. âTaj-Tanâ in the tree is the abbreviation for Tajima-Tango.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Boundary population genomes similar to that of the northern Japanese group</h3><p id="p-33">To explore hybridization signatures on chromosomes, we examined allele-sharing between boundary populations and surrounding populations using the âHZ-1â dataset (see Materials and Methods). We classified the fixed alleles between the groups into three states: shared by N.JPN and Tajima-Tango, shared by S.JPN and Tajima-Tango, and shared by N.JPN and S.JPN. Then, we summarized each stateâs total numbers, as shown in table in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. We found 1,380 out of 4,661 SNPs that were common alleles between two of the three groups (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Regarding those alleles, the majority (81.4%) were common alleles between Tajima-Tango and N.JPN, which was a much higher frequency than that between Tajima-Tango and S.JPN (11.2%). The rest of them were common alleles between N.JPN and S.JPN (7.5%), <em>i.e.</em>, specific alleles in Tajima-Tango. These proportions are near those of a previous study based on 96 genomic regions (4 loci for each chromosome) (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Takehana <em>et al.</em> 2016</a>). Although the regions with neighboring common alleles between Tajima-Tango and S.JPN were observed for certain chromosomes, alleles on the Tajima-Tango genome were mostly shared with N.JPN, suggesting that Tajima-Tango is a subgroup of N.JPN with a short divergence time.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/217/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Shared allele distribution among boundary populations. The gray rectangle represents the total length of each medaka chromosome. The table in the figure shows the number of alleles shared between the groups observed on each chromosome. Mark represents the state whose group shares that allele. *1 and *2 are the total number from chromosomes 1 to 12 and from chromosomes 13 to 24, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-64091323" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Shared allele distribution among boundary populations. The gray rectangle represents the total length of each medaka chromosome. The table in the figure shows the number of alleles shared between the groups observed on each chromosome. Mark represents the state whose group shares that allele. *1 and *2 are the total number from chromosomes 1 to 12 and from chromosomes 13 to 24, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/217/F4.medium.gif" width="440" height="332"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/1/217/F4.medium.gif" width="440" height="332"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/217/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/217/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491953" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Shared allele distribution among boundary populations. The gray rectangle represents the total length of each medaka chromosome. The table in the figure shows the number of alleles shared between the groups observed on each chromosome. Mark represents the state whose group shares that allele. *1 and *2 are the total number from chromosomes 1 to 12 and from chromosomes 13 to 24, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">To investigate the mitochondrial introgression in boundary populations, we reconstructed the phylogenetic tree based on partial mtDNA sequences generated by mapping short reads from the GBS of the complete mitochondrial genome (Figure S4). The mtDNA sequences from Toyooka and Kinosaki in Tajima-Tango were clustered and closely related to those from Kaga, which was a root population in N.JPN. Considering that Tajima-Tango was a subgroup of N.JPN, which we inferred from the ML tree based on chromosomal sequences, these phylogenetic positions on the mtDNA tree would have also reflected an evolutionary history in which N.JPN diverged from the Tajima-Tango group. We confirmed that Amino, Kumihama and Hamasaka in Tajima-Tango had the mitochondrial genomes of phylogenetically distant populations classified into S.JPN (Figure S4). Additionally, the mitochondrial genome of Ayabe in S.JPN was classified into N.JPN but diverged from Tajima-Tango. These results suggest that mitochondrial genome introgressions occurred independently, meaning that they occurred not only from S.JPN to Tajima-Tango but also from N.JPN to S.JPN.</p></div><div id="sec-14" class="subsection"><h3>Inferring medaka demographic parameters</h3><p id="p-36">To infer the demographic parameters, effective population size (<em>N</em><sub>e</sub>), time (T), and proportion of the admix (r), we analyzed the âHZ-2â dataset based on the coalescent theory using an approximate Bayesian computation (ABC) framework. We performed the model selection to identify the best explanation scenario for the observed data from the four scenarios; (I) Tajima-Tango originated in N.JPN, (II) Admixture of N.JPN and Honshu, (III) N.JPN originated in Tajima-Tango, and (IV) Admixture of Tajima-Tango and Honshu (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). <em>DIYABC</em> has two different approaches (directional and logistic) for model selection. Based on each criterion, scenario III (Honshu diverged from Kyushu and then N.JPN diverged from the Tajima-Tango group without admixture with Honshu) and scenario IV (Honshu diverged from Kyushu and then Tajima-Tango occurred by an admixture with Honshu at r rate) were supported by the directional (posterior probability: 0.4720, 95% CI: 0.0344â0.9096) and logistic (posterior probability: 0.4477, 95% CI: 0.4313â0.4641) approaches, respectively (Figure S9). Our data could not distinguish between scenarios III and IV (see the Discussion section).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/217/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scenarios for estimation of the demographic parameters. Possible scenarios for the population history of the Tajima-Tango group (A) and the &#x201C;Out of northern Kyushu (NK)&#x201D; event (B) are shown. The TMRCAs estimated using ABC are in scenario III (C), scenario IV (D) and Out of NK (E)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-64091323" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Scenarios for estimation of the demographic parameters. Possible scenarios for the population history of the Tajima-Tango group (A) and the &#x201C;Out of northern Kyushu (NK)&#x201D; event (B) are shown. The TMRCAs estimated using ABC are in scenario III (C), scenario IV (D) and Out of NK (E).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/217/F5.medium.gif" width="440" height="277"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/1/217/F5.medium.gif" width="440" height="277"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/217/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/217/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491907" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-37" class="first-child">Scenarios for estimation of the demographic parameters. Possible scenarios for the population history of the Tajima-Tango group (A) and the âOut of northern Kyushu (NK)â event (B) are shown. The TMRCAs estimated using ABC are in scenario III (C), scenario IV (D) and Out of NK (E).</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">The posterior parameter estimates of scenarios III and IV shown in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, and <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figures 5C</a> and <a id="xref-fig-5-3" class="xref-fig" href="#F5">5D</a> were not scaled because various measures of accuracy (RRMISE, RMeanAD, and RRMSE in the <em>DIYABC</em> output) indicated that non-scaled parameters fit the observed data better than scaled parameters. These estimates showed that the time to the most common ancestor (TMRCA) between S.JPN and N.JPN in scenarios III and IV were dated to 4,560,000 (95% CI: 2,820,000â6,810,000) and 5,160,000 (95% CI: 3,080,000â7,780,000) years ago, respectively. However, although we inferred the TMRCA at three divergence events (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>), these times might be overestimated because the dataset for this estimation consisted of non-deme samples, <em>i.e.</em>, polymorphisms within populations were underestimated, and the random-mating assumption could not be satisfied. Therefore, we estimated the timing of the âOut of Northern Kyushuâ event using the âLocalâ dataset composed of the two deme samples (S.Saga and N.Saga), which were split at first on the S.JPN lineage (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). Based on a simple hypothesis, constant population size and no migration, we obtained the estimated time (510,000 years ago, 95% CI: 337,875â679,575) for the ancestral divergence of the two deme samples (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5E</a>), which was calculated from a scaled parameter by the mean effective population size because various accuracy measures indicated that scaled parameters fit the observed data better than non-scaled parameters. Thus, our population-genetic estimate using ABC suggests that medakas diverged and dispersed throughout Pacific-side Japan approximately 510 kyr ago.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><div id="sec-16" class="subsection"><h3>Redefined subgroups in Japanese archipelago based on the population structure inferred from genome-wide single-nucleotide polymorphisms</h3><p id="p-39"><em>Oryzias latipes</em> can be divided into five groups (N.JPN, S.JPN, E.KOR, W.KOR and Tajima-Tango) by mtDNA sequences and allozymic electrophoresis patterns (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Sakaizumi 1984</a>; <a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">Takehana <em>et al.</em> 2003</a>). In this study, based on chromosomal SNPs, the genetic clustering analysis showed that âK = 4â was the most supportive because it presented the lowest fivefold cross-validation error, indicating that N.JPN and S.JPN were divided into three ancestral clusters. When the K values increased, only S.JPN divided into more subgroups, which suggests that the S.JPN group was composed of more divergent groups than the other groups. Considering together with the results of the ML tree analysis, it is possible to redefine subgroups composed of each major group for our wild lab-stocks originated from the Japanese archipelago as follows. First, Tajima-Tango, which had been considered a hybridization group, should be included under the N.JPN group because it shows almost the same ancestral component as N.JPN, for which we propose the name âancestral northern Japanese (aN.JPN).â Then, the group called N.JPN should be assigned to a subgroup, for which we propose the name âderived northern Japanese (dN.JPN).â These two subgroups, aN.JPN and dN.JPN, compose the N.JPN group (Figure S8). Second, S.JPN can be divided into several subgroups, San-in, San-yo/Shikoku/Kinki and the Pacific Ocean side of Northeastern Honshu (PO.NEH), because the Kyushu &amp; others cluster was subdivided into three sub-clusters composed of geographically neighboring populations. Adding the two Kyushu subgroups: Northern and Southern Kyushu (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>), S.JPN is composed of five subgroups. Thus, medaka in Japanese archipelago could also be composed of the seven distinct subgroups based on chromosomal genetic diversity (Figure S8).</p><p id="p-40">S.JPN can be divided into finer subgroups based on the mitochondrial genome (<a id="xref-ref-46-6" class="xref-bibr" href="#ref-46">Takehana <em>et al.</em> 2003</a>), likely because the effective population size of mtDNA is one quarter that of a nuclear gene. This causes the intermingled branch pattern of the mtDNA tree, which is not associated with geographic distance. Although this intermingled branch pattern has been also observed our wild lab-stocks in a previous study based on mtDNA (<a id="xref-ref-22-9" class="xref-bibr" href="#ref-22">Katsumura <em>et al.</em> 2009</a>), it has been sometimes interpreted as artificial migrations accompanied by recent human activities (<a id="xref-ref-46-7" class="xref-bibr" href="#ref-46">Takehana <em>et al.</em> 2003</a>). If the branching pattern was formed by human activities, various ancestral components should appear independent of geographic relatedness in the result of the <em>ADMIXTURE</em> analysis based on chromosomal SNPs. However, no signal of recent migration was found in the <em>ADMIXTURE</em> result. Rather, the chromosomal SNP data support another hypothetical scenario in which the mtDNA tree topology reflects ancestral polymorphisms only and their local fixation is caused by a small effective population size (<a id="xref-ref-22-10" class="xref-bibr" href="#ref-22">Katsumura <em>et al.</em> 2009</a>). This result indicates that arguing the geographical origins of medaka based only on mtDNA may lead to false conclusions.</p><p id="p-41">From our data, it may be difficult to accurately evaluate the genetic diversities in E/W.KOR groups because the number of populations examined in each group was small. Therefore, though care must be taken in the interpretation, W.KOR showed the highest nucleotide diversity and E.KOR the second highest among the four major mitochondrial groups (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). The W.KOR group included the Chinese medaka in Shanghai, which could have elevated its value, while the latter group did not include any geographically distant populations. The clustering analysis showed that Shanghai from W.KOR and Yongcheon from E.KOR had an ancestry component from N.JPN (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). Additionally, Shanghai and Yongcheon showed higher nucleotide diversities than other lab stocks and wild-captured medaka. These suggest two possibilities: one is that the N.JPN ancestor was derived from the E/W.KOR ancestor, and the other is that contamination occurred through the maintained wild lab stocks (the latter possibility may be slightly high because the nucleotide diversities of Shanghai and Yongcheon were higher than those of wild-captured medakas). To investigate these possibilities and the origin of <em>O. latipes</em>, a population-based genome wide analysis must be conducted to increase the population numbers of E/W.KOR from other wild lab-stocks or wild medaka in Korean, and include the sister species <em>O. curvinotus</em> and <em>O. luzonensis</em>.</p></div><div id="sec-17" class="subsection"><h3>Reconstruction of the medaka population history in the Japanese archipelago</h3><p id="p-42">Our genome-wide analysis shows that medakas in N.JPN and S.JPN are deeply divergent and dispersed over the Japanese archipelago from different locations at different times. In particular, our ABC analysis indicates that dN.JPN originated in and diverged from aN.JPN. This inference about the history of N.JPN after divergence from S.JPN is not consistent with previous inferences from the allozyme, mtDNA and limited chromosomal SNP analyses, which have suggested that aN.JPN is a hybridization group between dN.JPN and S.JPN, or aN.JPN is a sub-group derived from dN.JPN. Our GBS data show that (i) the nucleotide diversity in aN.JPN is higher than that in dN.JPN (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>), (ii) S.JPN is genetically more closely related to aN.JPN than to dN.JPN based on the number of shared alleles (table in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>), and (iii) the aN.JPN branch is the root in the N.JPN clade of the phylogenetic tree based on partial mtDNA sequences (Figure S4), which is also shown by the whole mitochondrial genome analysis (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hirayama <em>et al.</em> 2010</a>). These data suggest that aN.JPN forms an outgroup to all present-day dN.JPN and it spread along the Sea of Japan side. Furthermore, the ABC frameworkâs estimation supports our scenario, although the analysis does not statistically distinguish between scenario III and scenario IV. Even if the admixture occurred, the inferred ratio of the admixture is too low (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5D</a>). However, we have to recognize the problem and limitation which derived from the use these lab-stocks in this estimation by ABC approach. The nucleotide diversity showed the low in lab stocks, indicating that the lab stocks experienced genetic bottlenecks or selection under the laboratory condition. This would affect the estimation of population history (especially in demographic events, admixture rate might be underestimated) by the ABC approach. We would have to sample the wild-capture medaka for more highly accurately estimation.</p><p id="p-43">Our findings strongly support the âOut of northern Kyushuâ model of S.JPN proposed in <a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Katsumura <em>et al.</em> (2012)</a> and have revealed the dispersal route of dN.JPN / aN.JPN in N.JPN. The genetic clustering analyses and phylogenetic tree based on the GBS data elucidate medaka history better than previous mitochondrial DNA analyses. In particular, the <em>ADMIXTURE</em> analysis shows that the two ancestry components (yellow and blue in <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>) were observed in northern Kyushu, suggesting that S.JPN dispersed in three different directions after dispersing out of Northern Kyushu. Geographically, Shikoku, San-yo, and Kinki are separated by sea, but medakas in the three local lands share the same ancestral component (blue in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>; see also Figure S1 for geographic information). Medaka is a freshwater fish, but survival and reproductive even in seawater (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Inoue and Takei 2002</a>). Because most of the rivers in the Japanese archipelago are steep and short, the rivers tend to flood after heavy rain. Although medakas are highly likely to drift to the sea on each occasion, medakas can have survived even in the sea and may have returned to the river because of their saltwater tolerance. Thus, the possibility of moving from river to river through the sea cannot be also ignored, although there were several subsidings of sea level during the Pleistocene. The results of this study suggest a history in which medakas migrated throughout the Japanese archipelago through the sea.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/217/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Map representing the ancestry proportions from ADMIXTURE analysis at K = 6. Solid and dashed lines represent the spreading patterns of S.JPN and N.JPN inferred by GBS data, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-64091323" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Map representing the ancestry proportions from ADMIXTURE analysis at K = 6. Solid and dashed lines represent the spreading patterns of S.JPN and N.JPN inferred by GBS data, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/217/F6.medium.gif" width="396" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/1/217/F6.medium.gif" width="396" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/217/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/217/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491981" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-44" class="first-child">Map representing the ancestry proportions from ADMIXTURE analysis at K = 6. Solid and dashed lines represent the spreading patterns of S.JPN and N.JPN inferred by GBS data, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">The different branching patterns of chromosomal and mtDNA trees suggest that the mtDNA introgression occurred not only from S.JPN to aN.JPN but also from dN.JPN to S.JPN. The Tajima-Tango region is surrounded by mountains (Figure S1); however, it contains the lowest watershed (sea level 95.45 m) in the Japanese archipelago. The medaka has possibly moved in both directions across the watershed. From the above, the most plausible scenario is as follows (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). Ancestral S.JPN and N.JPN diverged first and independently reached their current habitats in the Japanese archipelago. After S.JPN, which is an ancestor of San-in, San-yo/Shikoku/Kinki and PO.NEH, dispersed from northern Kyushu approximately 510 kyr ago, dN.JPN diverged from aN.JPN in N.JPN. Then, their descendant populations spread rapidly to northern Honshu on the Sea of Japan side. Meanwhile, S.JPN dispersed in as many as three different directions and then spread rapidly northeastward from the western part of Fossa Magna. In the process of dispersing across the main island of Japan, certain S.JPN populations infiltrated the Tajima-Tango region from the west and the south, and the resulting mtDNA introgression occurred independently.</p></div><div id="sec-18" class="subsection"><h3>Conclusion</h3><p id="p-46">Our genome-wide SNP analysis reconstructed the detailed population structure and reliable history of medaka that evolved in the Japanese archipelago. Since the distribution of the subgroups was highly consistent with the geographical features, several adaptive traits could have evolved in each subgroup. Furthermore, the boundary populations were not caused by a hybridization event but instead were the origin of the populations dispersed to a northeastern part of the Japanese archipelago on the Sea of Japan side. A better understanding of the population structure and history of medaka will support association studies for phenotypes and genotypes related to environmental adaptation.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-47">We thank Mrs. Shizuko Chiba, Mrs. Sumiko Tomizuka, Dr. Atsuko Shimada and Prof. Emeritus Akihiro Shima (the University of Tokyo) for maintaining the medaka stocks from wild populations. This study was supported in part by grants-in-aid for Young Scientists (B), no. 16K21352 to T.K., and by a grant-in-aid for Scientific Research (A) nos. 26251050, 25251046, 17H01453, 18H03590, and Scientific Research (B) no 17H03738 to H.O. T.K. was also supported by a grant-in-aid from JSPS Research Fellowships (16J07227).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-48">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7358042">https://doi.org/10.25387/g3.7358042</a>.</p></li><li class="fn" id="fn-2"><p id="p-49"><em>Communicating editor: D. Grunwald</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-10-05"><span class="received-label">Received </span>October 5, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-11-18"><span class="accepted-label">Accepted </span>November 18, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2019 Katsumura <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.1" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1655</span>â<span class="cit-lpage">1664</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.094052.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMS8yMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.2" data-doi="10.1038/nrg.2015.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Good</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Harnessing the power of RADseq for ecological and evolutionary genomics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">81</span>â<span class="cit-lpage">92</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg.2015.28</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D17%26rft.spage%253D81%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg.2015.28%26rft_id%253Dinfo%253Apmid%252F26729255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg.2015.28&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26729255&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.3" data-doi="10.1242/bio.20148177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ansai</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinoshita</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Targeted mutagenesis using CRISPR/Cas system in medaka.</span> <abbr class="cit-jnl-abbrev">Biol. Open</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">362</span>â<span class="cit-lpage">371</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/bio.20148177</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BOpen%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fbio.20148177%26rft_id%253Dinfo%253Apmid%252F24728957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiYmlvbG9wZW4iO3M6NToicmVzaWQiO3M6NzoiMy81LzM2MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMS8yMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Senou</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hosoya</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title"><em>Oryzias sakaizumii</em>, a new ricefish from northern Japan (Teleostei: Adrianichthyidae).</span> <abbr class="cit-jnl-abbrev">Ichthyol. Explor. Freshwat.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">289</span>â<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIchthyol.%2BExplor.%2BFreshwat.%26rft.volume%253D22%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balding</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Approximate Bayesian computation in population genetics.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">2025</span>â<span class="cit-lpage">2035</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBeaumont%26rft.auinit1%253DM.%2BA.%26rft.volume%253D162%26rft.issue%253D4%26rft.spage%253D2025%26rft.epage%253D2035%26rft.atitle%253DApproximate%2BBayesian%2BComputation%2Bin%2BPopulation%2BGenetics%26rft_id%253Dinfo%253Apmid%252F12524368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Mi80LzIwMjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzEvMjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broughton</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betancur-R</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arratia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">OrtÃ­</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>âMulti-locus phylogenetic analysis reveals the pattern and tempo of bony fish evolution. PLOS Curr. 5:ecurrents.tol.2ca8041495ffafd0c92756e75247483e.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/currents.tol.2ca8041495ffafd0c92756e75247483e</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.7" data-doi="10.1111/mec.12354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Stacks: an analysis tool set for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3124</span>â<span class="cit-lpage">3140</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12354</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.8" data-doi="10.1534/g3.111.000240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Stacks: building and genotyping Loci de novo from short-read sequences.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">171</span>â<span class="cit-lpage">182</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.111.000240</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D1%26rft.spage%253D171%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.111.000240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.9" data-doi="10.1186/s13742-015-0047-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chow</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tellier</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vattikuti</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Second-generation PLINK: rising to the challenge of larger and richer datasets.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13742-015-0047-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D4%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13742-015-0047-8%26rft_id%253Dinfo%253Apmid%252F25722852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13742-015-0047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25722852&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.10" data-doi="10.1093/sysbio/syw037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chernomor</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Haeseler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minh</span>  <span class="cit-name-given-names">B. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Terrace Aware Data Structure for Phylogenomic Inference from Supermatrices.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">997</span>â<span class="cit-lpage">1008</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/sysbio/syw037</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyw037%26rft_id%253Dinfo%253Apmid%252F27121966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syw037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27121966&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.11" data-doi="10.1093/bioinformatics/btt763"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cornuet</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pudlo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veyssier</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dehne-Garcia</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">DIYABC v2.0: a software to make approximate Bayesian computation inferences about population history using single nucleotide polymorphism, DNA sequence and microsatellite data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1187</span>â<span class="cit-lpage">1189</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt763</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt763%26rft_id%253Dinfo%253Apmid%252F24389659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt763&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24389659&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000335001000022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.12" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.13" data-doi="10.2307/2408678"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span>â<span class="cit-article-title">CONFIDENCE LIMITS ON PHYLOGENIES: AN APPROACH USING THE BOOTSTRAP.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">783</span>â<span class="cit-lpage">791</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1985.tb00420.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.volume%253D39%26rft.issue%253D4%26rft.spage%253D783%26rft.epage%253D791%26rft.atitle%253DConfidence%2Blimits%2Bon%2Bphylogenies%253B%2Ban%2Bapproach%2Busing%2Bthe%2Bbootstrap%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408678%26rft_id%253Dinfo%253Apmid%252F7288891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408678&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1996015960&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=7288891&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985APJ8100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.14" data-doi="10.1093/sysbio/syq010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dufayard</span>  <span class="cit-name-given-names">J.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lefort</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anisimova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hordijk</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">307</span>â<span class="cit-lpage">321</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/sysbio/syq010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyq010%26rft_id%253Dinfo%253Apmid%252F20525638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syq010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20525638&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276528300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.15" data-doi="10.1016/j.gene.2010.02.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirayama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mukai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miya</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murata</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamashita</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Intraspecific variation in the mitochondrial genome among local populations of Medaka Oryzias latipes.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">13</span>â<span class="cit-lpage">24</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gene.2010.02.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D457%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2010.02.012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2010.02.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.16" data-doi="10.1371/journal.pone.0170006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Igarashi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katsumura</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urushihara</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hida</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">An Approach to Elucidate NBS1 Function in DNA Repair Using Frequent Nonsynonymous Polymorphism in Wild Medaka (<em>Oryzias latipes</em>) Populations.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e0170006</span>â<span class="cit-lpage">e0170019</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0170006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0170006%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0170006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0170006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.17" data-doi="10.1007/s10228-018-0623-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iguchi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kume</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitagawa</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">Natural hybridization between two Japanese medaka species (<em>Oryzias latipes</em> and <em>Oryzias sakaizumii</em>) observed in the Yura River basin, Kyoto, Japan.</span> <abbr class="cit-jnl-abbrev">Ichthyol. Res.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">405</span>â<span class="cit-lpage">411</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10228-018-0623-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIchthyol.%2BRes.%26rft.volume%253D65%26rft.spage%253D405%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10228-018-0623-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10228-018-0623-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.18" data-doi="10.2108/zsj.19.727"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takei</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Diverse adaptability in oryzias species to high environmental salinity.</span> <abbr class="cit-jnl-abbrev">Zool. Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">727</span>â<span class="cit-lpage">734</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2108/zsj.19.727</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZoological%2Bscience%26rft.stitle%253DZoolog%2BSci%26rft.aulast%253DInoue%26rft.auinit1%253DK.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D727%26rft.epage%253D734%26rft.atitle%253DDiverse%2Badaptability%2Bin%2Boryzias%2Bspecies%2Bto%2Bhigh%2Benvironmental%2Bsalinity.%26rft_id%253Dinfo%253Adoi%252F10.2108%252Fzsj.19.727%26rft_id%253Dinfo%253Apmid%252F12149572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2108/zsj.19.727&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12149572&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178165300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.19" data-doi="10.1534/genetics.115.178673"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeong</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakagome</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Rienzo</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Deep History of East Asian Populations Revealed Through Genetic Analysis of the Ainu.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">261</span>â<span class="cit-lpage">272</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.178673</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.178673%26rft_id%253Dinfo%253Apmid%252F26500257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzEvMjYxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8xLzIxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.20" data-doi="10.1016/B978-1-4832-3211-9.50009-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jukes</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantor</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span>âEvolution of protein molecules. Mammalian protein metabolism 3: 132.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/B978-1-4832-3211-9.50009-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMAMMALIAN%2BPROTEIN%2BMETABOLISM%26rft.volume%253D2%26rft.spage%253D21%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-1-4832-3211-9.50009-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-1-4832-3211-9.50009-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.21" data-doi="10.1038/nature05846"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kasahara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naruse</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakatani</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">The medaka draft genome and insights into vertebrate genome evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">447</span>: <span class="cit-fpage">714</span>â<span class="cit-lpage">719</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature05846</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKasahara%26rft.auinit1%253DM.%26rft.volume%253D447%26rft.issue%253D7145%26rft.spage%253D714%26rft.epage%253D719%26rft.atitle%253DThe%2Bmedaka%2Bdraft%2Bgenome%2Band%2Binsights%2Binto%2Bvertebrate%2Bgenome%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05846%26rft_id%253Dinfo%253Apmid%252F17554307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05846&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17554307&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247030700044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.22" data-doi="10.1016/j.gene.2009.04.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katsumura</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mano</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suguro</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Genetic differentiation among local populations of medaka fish (<em>Oryzias latipes</em>) evaluated through grid- and deme-based sampling.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">443</span>: <span class="cit-fpage">170</span>â<span class="cit-lpage">177</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gene.2009.04.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D443%26rft.spage%253D170%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2009.04.017%26rft_id%253Dinfo%253Apmid%252F19397956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2009.04.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19397956&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268084100020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.23" data-doi="10.1098/rspb.2014.2259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katsumura</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakagome</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanihara</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kataoka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Natural allelic variations of xenobiotic-metabolizing enzymes affect sexual dimorphism in <em>Oryzias latipes</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">20142259</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.2014.2259</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2014.2259%26rft_id%253Dinfo%253Apmid%252F25377463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2014.2259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25377463&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.24" data-doi="10.1537/ase.110525"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katsumura</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsukamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekiya</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamashita</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">A population genetic study on the relationship between medaka fish and the spread of wet-rice cultivation across the Japanese archipelago.</span> <abbr class="cit-jnl-abbrev">Anthropol. Sci.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">81</span>â<span class="cit-lpage">89</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1537/ase.110525</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnthropol.%2BSci.%26rft.volume%253D120%26rft.spage%253D81%26rft_id%253Dinfo%253Adoi%252F10.1537%252Fase.110525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1537/ase.110525&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000305305500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.25" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>â<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.26" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>â<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.27" data-doi="10.2108/zsj.14.517"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matsuda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yonekawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamaguchi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakaizumi</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Geographic variation and diversity in the mitochondrial DNA of the medaka, <em>Oryzias latipes</em>, as determined by restriction endonuclease analysis.</span> <abbr class="cit-jnl-abbrev">Zool. Sci.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">517</span>â<span class="cit-lpage">526</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2108/zsj.14.517</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZool.%2BSci.%26rft.volume%253D14%26rft.spage%253D517%26rft_id%253Dinfo%253Adoi%252F10.2108%252Fzsj.14.517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2108/zsj.14.517&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.28" data-doi="10.1093/molbev/mst024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Minh</span>  <span class="cit-name-given-names">B. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">M. A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Haeseler</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Ultrafast approximation for phylogenetic bootstrap.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1188</span>â<span class="cit-lpage">1195</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/mst024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst024%26rft_id%253Dinfo%253Apmid%252F23418397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23418397&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318165700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.29" data-doi="10.1371/journal.pone.0073535"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miya</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satoh</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takeshima</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sado</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Evolutionary origin of the Scombridae (tunas and mackerels): members of a paleogene adaptive radiation with 14 other pelagic fish families.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e73535</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0073535</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De73535%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0073535%26rft_id%253Dinfo%253Apmid%252F24023883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0073535&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24023883&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.30" data-doi="10.1111/mec.12350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Narum</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buerkle</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Genotyping-by-sequencing in ecological and conservation genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2841</span>â<span class="cit-lpage">2847</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12350</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D2841%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.31" data-doi="10.1093/molbev/msu300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">L.-T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Haeseler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minh</span>  <span class="cit-name-given-names">B. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">268</span>â<span class="cit-lpage">274</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu300</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu300%26rft_id%253Dinfo%253Apmid%252F25371430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25371430&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-book" id="cit-9.1.217.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oota</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitani</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Human Population Genetics Meets Medaka</span>, pp. <span class="cit-fpage">339</span>â<span class="cit-lpage">350</span> in <span class="cit-source"><em>Medaka: A Model for Organogenesis</em>, <em>Human Disease</em>, <em>and Evolution</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Naruse</span>  <span class="cit-name-given-names">K.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Takeda</span>  <span class="cit-name-given-names">H.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Tokyo, Japan</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.33" data-doi="10.1266/ggs.90.133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osada</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Genetic diversity in humans and non-human primates and its evolutionary consequences.</span> <abbr class="cit-jnl-abbrev">Genes Genet. Syst.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">133</span>â<span class="cit-lpage">145</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1266/ggs.90.133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenet.%2BSyst.%26rft.volume%253D90%26rft.spage%253D133%26rft_id%253Dinfo%253Adoi%252F10.1266%252Fggs.90.133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1266/ggs.90.133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.34" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Development of High-Density Genetic Maps for Barley and Wheat Using a Novel Two-Enzyme Genotyping-by-Sequencing Approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32253</span>â<span class="cit-lpage">e32258</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0032253</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.35" data-doi="10.1099/ijs.0.01472-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saitou</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span>â<span class="cit-article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">406</span>â<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DSaitou%26rft.auinit1%253DN%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D406%26rft.epage%253D425%26rft.atitle%253DThe%2Bneighbor-joining%2Bmethod%253A%2Ba%2Bnew%2Bmethod%2Bfor%2Breconstructing%2Bphylogenetic%2Btrees%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fijs.0.01472-0%26rft_id%253Dinfo%253Apmid%252F3447015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/ijs.0.01472-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3447015&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987J406700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakaizumi</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span>â<span class="cit-article-title">Rigid Isolation between the Northern Population and the Southern Population of the Medaka, <em>Oryzias latipes</em> (Genetics).</span> <abbr class="cit-jnl-abbrev">Zool. Sci.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">795</span>â<span class="cit-lpage">800</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZool.%2BSci.%26rft.volume%253D1%26rft.spage%253D795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.37" data-doi="10.1007/BF00115131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakaizumi</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span>â<span class="cit-article-title">Genetic divergence in wild populations of Medaka, <em>Oryzias latipes</em> (Pisces, Oryziatidae) from Japan and China.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">119</span>â<span class="cit-lpage">125</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00115131</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D69%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00115131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00115131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakaizumi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egami</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moriwaki</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span>â<span class="cit-article-title">Allozymic Variation in Wild Populations of the Fish, <em>Oryzias latipes</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Jpn. Acad. Ser. B</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">448</span>â<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BJpn.%2BAcad.%2BSer.%2BB%26rft.volume%253D56%26rft.spage%253D448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.39" data-doi="10.2307/1444373"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakaizumi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moriwaki</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egami</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span>â<span class="cit-article-title">Allozymic variation and regional differentiation in wild populations of the fish <em>Oryzias latipes</em>.</span> <abbr class="cit-jnl-abbrev">Copeia</abbr> <span class="cit-vol">1983</span>: <span class="cit-fpage">311</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/1444373</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCopeia%26rft.volume%253D1983%26rft.spage%253D311%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1444373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1444373&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.40" data-doi="10.1002/jez.1402640113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakaizumi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamaguchi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span>â<span class="cit-article-title">Electrophoretic studies of meiotic segregation in inter- and intraspecific hybrids among East Asian species of the genus <em>Oryzias</em> (Pisces: Oryziatidae).</span> <abbr class="cit-jnl-abbrev">J. Exp. Zool.</abbr> <span class="cit-vol">264</span>: <span class="cit-fpage">85</span>â<span class="cit-lpage">92</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jez.1402640113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BZool.%26rft.volume%253D264%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjez.1402640113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jez.1402640113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.41" data-doi="10.1098/rsbl.2009.0419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Setiamarga</span>  <span class="cit-name-given-names">D. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miya</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamanoue</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Azuma</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Divergence time of the two regional medaka populations in Japan as a new time scale for comparative genomics of vertebrates.</span> <abbr class="cit-jnl-abbrev">Biol. Lett.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">812</span>â<span class="cit-lpage">816</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rsbl.2009.0419</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2009.0419%26rft_id%253Dinfo%253Apmid%252F19586967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsbl.2009.0419&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2013074188&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=19586967&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shima</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimada</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakaizumi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egami</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span>â<span class="cit-article-title">First listing of wild stocks of the medaka Oryzias latipes currently kept by the Zoological Institute, Faculty of Science, University of Tokyo. Journal of the Faculty of Science, Imperial University of Tokyo Sect IV</span>. <abbr class="cit-jnl-abbrev">Zoology</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">27</span>â<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZoology%26rft.volume%253D16%26rft.spage%253D27%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.43" data-doi="10.1038/s41467-017-00432-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shimmura</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakayama</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinomiya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukamachi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yasugi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Dynamic plasticity in phototransduction regulates seasonal changes in color perception.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">412</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-017-00432-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D8%26rft.spage%253D412%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-017-00432-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-017-00432-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.44" data-doi="10.1534/g3.113.008722"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spivakov</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auer</span>  <span class="cit-name-given-names">T. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peravali</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dolle</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Genomic and phenotypic characterization of a wild medaka population: towards the establishment of an isogenic population genetic resource in fish.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">433</span>â<span class="cit-lpage">445</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.008722</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.008722%26rft_id%253Dinfo%253Apmid%252F24408034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjQvMy80MzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzEvMjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span>â<span class="cit-article-title">Evolutionary relationship of DNA sequences in finite populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">437</span>â<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D105%26rft.issue%253D2%26rft.spage%253D437%26rft.epage%253D460%26rft.atitle%253DEVOLUTIONARY%2BRELATIONSHIP%2BOF%2BDNA%2BSEQUENCES%2BIN%2BFINITE%2BPOPULATIONS%26rft_id%253Dinfo%253Apmid%252F6628982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA1LzIvNDM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8xLzIxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.46" data-doi="10.2108/zsj.20.1279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takehana</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagai</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuchiya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakaizumi</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Geographic variation and diversity of the cytochrome b gene in Japanese wild populations of medaka, <em>Oryzias latipes</em>.</span> <abbr class="cit-jnl-abbrev">Zool. Sci.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1279</span>â<span class="cit-lpage">1291</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2108/zsj.20.1279</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZoological%2Bscience%26rft.stitle%253DZoolog%2BSci%26rft.aulast%253DTakehana%26rft.auinit1%253DY.%26rft.volume%253D20%26rft.issue%253D10%26rft.spage%253D1279%26rft.epage%253D1291%26rft.atitle%253DGeographic%2Bvariation%2Band%2Bdiversity%2Bof%2Bthe%2Bcytochrome%2Bb%2Bgene%2Bin%2BJapanese%2Bwild%2Bpopulations%2Bof%2Bmedaka%252C%2BOryzias%2Blatipes.%26rft_id%253Dinfo%253Adoi%252F10.2108%252Fzsj.20.1279%26rft_id%253Dinfo%253Apmid%252F14569151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2108/zsj.20.1279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14569151&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186945600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.47" data-doi="10.2108/zs150144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takehana</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakai</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Narita</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naruse</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Origin of Boundary Populations in Medaka (<em>Oryzias latipes Species Complex</em>).</span> <abbr class="cit-jnl-abbrev">Zool. Sci.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">125</span>â<span class="cit-lpage">131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2108/zs150144</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZool.%2BSci.%26rft.volume%253D33%26rft.spage%253D125%26rft_id%253Dinfo%253Adoi%252F10.2108%252Fzs150144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2108/zs150144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.48" data-doi="10.1093/molbev/msr121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">MEGA5: Molecular Evolutionary Genetics Analysis Using Maximum Likelihood, Evolutionary Distance, and Maximum Parsimony Methods.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2731</span>â<span class="cit-lpage">2739</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msr121</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr121%26rft_id%253Dinfo%253Apmid%252F21546353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msr121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21546353&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295184200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.49" data-doi="10.1186/1471-213X-14-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watanabe-Asaka</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekiya</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wada</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yasuda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okubo</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Regular heartbeat rhythm at the heartbeat initiation stage is essential for normal cardiogenesis at low temperature.</span> <abbr class="cit-jnl-abbrev">BMC Dev. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-213X-14-12</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BDev.%2BBiol.%26rft.volume%253D14%26rft.spage%253D12%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-213X-14-12%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-213X-14-12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.50" data-doi="10.1002/wics.147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wickham</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">ggplot2.</span> <abbr class="cit-jnl-abbrev">Wiley Interdiscip. Rev. Comput. Stat.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">180</span>â<span class="cit-lpage">185</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/wics.147</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWiley%2BInterdiscip.%2BRev.%2BComput.%2BStat.%26rft.volume%253D3%26rft.spage%253D180%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fwics.147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/wics.147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.1.217.51" data-doi="10.1093/bioinformatics/bts606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogarten</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">A high-performance computing toolset for relatedness and principal component analysis of SNP data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3326</span>â<span class="cit-lpage">3328</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts606</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts606%26rft_id%253Dinfo%253Apmid%252F23060615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts606&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23060615&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312105300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/1/217.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/1/203" title="A Strategy To Exploit Surrogate Sire Technology in Livestock Breeding Programs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/1/229" title="Wolbachia Infection Associated with Increased Recombination in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/1"><span>Volume 9 Issue 1, January 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/1.cover-source.jpg?itok=uUntglLo" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/1/217" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YLqok286CDULG4vu3DWyzOKy2RP6WIYKrPfWoHH01DM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/1/217" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/1/217" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/1/217" class="active" data-icon-position="" data-hide-link-title="0">Medaka Population Genome Structure and Demographic History Described via Genotyping-by-Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491872" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-0pP1ikpcLEGn42ErFailbpijrknErt1dnHrCKbrfVVc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/1/217.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/1/217" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-X0zW_H4i2w4lju8smsjZ-2m31Ze04uUokLuimCCFDrI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/1/217" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/1/217.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491872" id="node491872--2" data-pisa="ggg;9/1/217" data-pisa-master="ggg;g3.118.200779" data-apath="/ggg/9/1/217.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Medaka Population Genome Structure and Demographic History Described via Genotyping-by-Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Takafumi Katsumura</span>, <span class="highwire-citation-author" data-delta="1">Shoji Oda</span>, <span class="highwire-citation-author" data-delta="2">Hiroshi Mitani</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0746-2729" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hiroki Oota</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">217-228;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200779 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491872--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Takafumi Katsumura </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Anatomy, Kitasato University School of Medicine, Sagamihara, Kanagawa 252-0374, Japan</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Graduate School of Natural Science and Technology, Okayama University, Okayama, Okayama 700-8530, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Takafumi%2BKatsumura%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Katsumura%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATakafumi%2BKatsumura%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tk@okayama-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">tk@okayama-u.ac.jp</a>
<a href="mailto:hiroki_oota@med.kitasato-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">hiroki_oota@med.kitasato-u.ac.jp</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Shoji Oda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Integrated Biosciences, Graduate School of Frontier Sciences, University of Tokyo, Kashiwa, Chiba 277-8562, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shoji%2BOda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oda%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShoji%2BOda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Hiroshi Mitani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Integrated Biosciences, Graduate School of Frontier Sciences, University of Tokyo, Kashiwa, Chiba 277-8562, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hiroshi%2BMitani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mitani%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHiroshi%2BMitani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hiroki Oota </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Anatomy, Kitasato University School of Medicine, Sagamihara, Kanagawa 252-0374, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hiroki%2BOota%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oota%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHiroki%2BOota%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0746-2729" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hiroki Oota</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tk@okayama-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">tk@okayama-u.ac.jp</a>
<a href="mailto:hiroki_oota@med.kitasato-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">hiroki_oota@med.kitasato-u.ac.jp</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/1/217" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/1/217&amp;title=Medaka%20Population%20Genome%20Structure%20and%20Demographic%20History%20Described%20via%20Genotyping-by-Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/1/217&amp;title=Medaka%20Population%20Genome%20Structure%20and%20Demographic%20History%20Described%20via%20Genotyping-by-Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/1/217&amp;title=Medaka%20Population%20Genome%20Structure%20and%20Demographic%20History%20Described%20via%20Genotyping-by-Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/1/217&amp;text=Medaka%20Population%20Genome%20Structure%20and%20Demographic%20History%20Described%20via%20Genotyping-by-Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/1/217&amp;title=Medaka%20Population%20Genome%20Structure%20and%20Demographic%20History%20Described%20via%20Genotyping-by-Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/1/217&amp;t=Medaka%20Population%20Genome%20Structure%20and%20Demographic%20History%20Described%20via%20Genotyping-by-Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/1/217&amp;title=Medaka%20Population%20Genome%20Structure%20and%20Demographic%20History%20Described%20via%20Genotyping-by-Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/1/217&amp;title=Medaka%20Population%20Genome%20Structure%20and%20Demographic%20History%20Described%20via%20Genotyping-by-Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491872/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491872" id="node491872--3" data-pisa="ggg;9/1/217" data-pisa-master="ggg;g3.118.200779" data-apath="/ggg/9/1/217.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Medaka Population Genome Structure and Demographic History Described via Genotyping-by-Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Takafumi Katsumura</span>, <span class="highwire-citation-author" data-delta="1">Shoji Oda</span>, <span class="highwire-citation-author" data-delta="2">Hiroshi Mitani</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0746-2729" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hiroki Oota</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">217-228;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200779 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491872--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Takafumi Katsumura </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Anatomy, Kitasato University School of Medicine, Sagamihara, Kanagawa 252-0374, Japan</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Graduate School of Natural Science and Technology, Okayama University, Okayama, Okayama 700-8530, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Takafumi%2BKatsumura%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Katsumura%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATakafumi%2BKatsumura%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tk@okayama-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">tk@okayama-u.ac.jp</a>
<a href="mailto:hiroki_oota@med.kitasato-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">hiroki_oota@med.kitasato-u.ac.jp</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Shoji Oda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Integrated Biosciences, Graduate School of Frontier Sciences, University of Tokyo, Kashiwa, Chiba 277-8562, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shoji%2BOda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oda%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShoji%2BOda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Hiroshi Mitani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Integrated Biosciences, Graduate School of Frontier Sciences, University of Tokyo, Kashiwa, Chiba 277-8562, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hiroshi%2BMitani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mitani%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHiroshi%2BMitani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hiroki Oota </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Anatomy, Kitasato University School of Medicine, Sagamihara, Kanagawa 252-0374, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hiroki%2BOota%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oota%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHiroki%2BOota%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0746-2729" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hiroki Oota</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tk@okayama-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">tk@okayama-u.ac.jp</a>
<a href="mailto:hiroki_oota@med.kitasato-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">hiroki_oota@med.kitasato-u.ac.jp</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=217&amp;rft.epage=228&amp;rft.atitle=Medaka%20Population%20Genome%20Structure%20and%20Demographic%20History%20Described%20via%20Genotyping-by-Sequencing&amp;rft.volume=9&amp;rft.issue=1&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Katsumura%2C+Takafumi&amp;rft.au=Oda%2C+Shoji&amp;rft.au=Mitani%2C+Hiroshi&amp;rft.au=Oota%2C+Hiroki"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491872/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491872/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491872/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491872/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491872/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491872/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491872/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491872/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491872/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491872/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491872/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491872/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/1/217" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/1/217.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/1/217.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/1/217.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/1/217.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
