<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/1/135" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/1/135.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/1/135.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/1/135.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/1/135.atom" />
<meta name="HW.pisa" content="ggg;9/1/135" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="CRISPR/Cas9 Methodology for the Generation of Knockout Deletions in Caenorhabditis elegans" />
<meta name="DC.Identifier" content="10.1534/g3.118.200778" />
<meta name="DC.Date" content="2019-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Au et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The Caenorhabditis elegans Gene Knockout Consortium is tasked with obtaining null mutations in each of the more than 20,000 open reading frames (ORFs) of this organism. To date, approximately 15,000 ORFs have associated putative null alleles. As there has been substantial success in using CRISPR/Cas9 in C. elegans , this appears to be the most promising technique to complete the task. To enhance the efficiency of using CRISPR/Cas9 to generate gene deletions in C. elegans we provide a web-based interface to access our database of guide RNAs (&lt;http://genome.sfu.ca/crispr&gt;). When coupled with previously developed selection vectors, optimization for homology arm length, and the use of purified Cas9 protein, we demonstrate a robust and effective protocol for generating deletions for this large-scale project. Debate and speculation in the larger scientific community concerning off-target effects due to non-specific Cas9 cutting has prompted us to investigate through whole genome sequencing the occurrence of single nucleotide variants and indels accompanying targeted deletions. We did not detect any off-site variants above the natural spontaneous mutation rate and therefore conclude that this modified protocol does not generate off-target events to any significant degree in C. elegans . We did, however, observe a number of non-specific alterations at the target site itself following the Cas9-induced double-strand break and offer a protocol for best practice quality control for such events." />
<meta name="DC.Contributor" content="Vinci Au" />
<meta name="DC.Contributor" content="Erica Li-Leger" />
<meta name="DC.Contributor" content="Greta Raymant" />
<meta name="DC.Contributor" content="Stephane Flibotte" />
<meta name="DC.Contributor" content="George Chen" />
<meta name="DC.Contributor" content="Kiana Martin" />
<meta name="DC.Contributor" content="Lisa Fernando" />
<meta name="DC.Contributor" content="Claudia Doell" />
<meta name="DC.Contributor" content="Federico I. Rosell" />
<meta name="DC.Contributor" content="Su Wang" />
<meta name="DC.Contributor" content="Mark L. Edgley" />
<meta name="DC.Contributor" content="Ann E. Rougvie" />
<meta name="DC.Contributor" content="Harald Hutter" />
<meta name="DC.Contributor" content="Donald G. Moerman" />
<meta name="article:published_time" content="2019-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="CRISPR/Cas9 Methodology for the Generation of Knockout Deletions in Caenorhabditis elegans" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; Gene Knockout Consortium is tasked with obtaining null mutations in each of the more than 20,000 open reading frames (ORFs) of this organism. To date, approximately 15,000 ORFs have associated putative null alleles. As there has been substantial success in using CRISPR/Cas9 in &lt;i&gt;C. elegans&lt;/i&gt;, this appears to be the most promising technique to complete the task. To enhance the efficiency of using CRISPR/Cas9 to generate gene deletions in &lt;i&gt;C. elegans&lt;/i&gt; we provide a web-based interface to access our database of guide RNAs (http://genome.sfu.ca/crispr). When coupled with previously developed selection vectors, optimization for homology arm length, and the use of purified Cas9 protein, we demonstrate a robust and effective protocol for generating deletions for this large-scale project. Debate and speculation in the larger scientific community concerning off-target effects due to non-specific Cas9 cutting has prompted us to investigate through whole genome sequencing the occurrence of single nucleotide variants and indels accompanying targeted deletions. We did not detect any off-site variants above the natural spontaneous mutation rate and therefore conclude that this modified protocol does not generate off-target events to any significant degree in &lt;i&gt;C. elegans&lt;/i&gt;. We did, however, observe a number of non-specific alterations at the target site itself following the Cas9-induced double-strand break and offer a protocol for best practice quality control for such events.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="ggg;9/1/135" />
<meta name="citation_id" content="9/1/135" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/1/135" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/1/135.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/1/135.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/1/135.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200778" />
<meta name="citation_pmid" content="30420468" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="135" />
<meta name="citation_lastpage" content="144" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Vinci Au" />
<meta name="citation_author_institution" content="Department of Zoology and Michael Smith Laboratories" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3587-6200" />
<meta name="citation_author" content="Erica Li-Leger" />
<meta name="citation_author_institution" content="Department of Zoology and Michael Smith Laboratories" />
<meta name="citation_author" content="Greta Raymant" />
<meta name="citation_author_institution" content="Department of Zoology and Michael Smith Laboratories" />
<meta name="citation_author" content="Stephane Flibotte" />
<meta name="citation_author_institution" content="Department of Zoology and Michael Smith Laboratories" />
<meta name="citation_author" content="George Chen" />
<meta name="citation_author_institution" content="Department of Zoology and Michael Smith Laboratories" />
<meta name="citation_author" content="Kiana Martin" />
<meta name="citation_author_institution" content="Department of Zoology and Michael Smith Laboratories" />
<meta name="citation_author" content="Lisa Fernando" />
<meta name="citation_author_institution" content="Department of Zoology and Michael Smith Laboratories" />
<meta name="citation_author" content="Claudia Doell" />
<meta name="citation_author_institution" content="Department of Zoology and Michael Smith Laboratories" />
<meta name="citation_author" content="Federico I. Rosell" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, University of British Columbia, Vancouver, British Columbia, Canada, V6T 1Z4" />
<meta name="citation_author" content="Su Wang" />
<meta name="citation_author_institution" content="Department of Zoology and Michael Smith Laboratories" />
<meta name="citation_author" content="Mark L. Edgley" />
<meta name="citation_author_institution" content="Department of Zoology and Michael Smith Laboratories" />
<meta name="citation_author" content="Ann E. Rougvie" />
<meta name="citation_author_institution" content="Department of Genetics, Cell Biology, and Development, University of Minnesota, Minneapolis, MN 55455" />
<meta name="citation_author" content="Harald Hutter" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Simon Fraser University, Burnaby, British Columbia, Canada, V5A 1S6" />
<meta name="citation_author" content="Donald G. Moerman" />
<meta name="citation_author_institution" content="Department of Zoology and Michael Smith Laboratories" />
<meta name="citation_author_email" content="moerman@zoology.ubc.ca" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. A. Arribere;citation_author=R. T. Bell;citation_author=B. X. Fu;citation_author=K. L. Artiles;citation_author=P. S. Hartman;citation_title=Efficient marker-free recovery of custom genetic modifications with CRISPR/Cas9 in C. elegans.;citation_volume=198;citation_year=2014;citation_pmid=25161212;citation_doi=10.1534/genetics.114.169730" />
<meta name="citation_reference" content="citation_journal_title=Methods;citation_author=T. O. Auer;citation_author=F. Del Bene;citation_title=CRISPR/Cas9 and TALEN-mediated knock-in approaches in zebrafish.;citation_pages=142-150;citation_volume=69;citation_year=2014;citation_pmid=24704174;citation_doi=10.1016/j.ymeth.2014.03.027" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. Bottcher;citation_author=M. Hollmann;citation_author=K. Merk;citation_author=V. Nitschko;citation_author=C. Obermaier;citation_title=Efficient chromosomal gene modification with CRISPR/cas9 and PCR-based homologous recombination donors in cultured Drosophila cells.;citation_volume=42;citation_year=2014;citation_pmid=24748663;citation_doi=10.1093/nar/gku289" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Brenner;citation_title=THE GENETICS OF CAENORHABDITIS ELEGANS;citation_pages=71-94;citation_volume=77;citation_year=1974;citation_issue=1;citation_pmid=4366476;citation_doi=10.1051/medsci/20031967725" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. Chen;citation_author=L. A. Fenk;citation_author=M. de Bono;citation_title=Efficient genome editing in Caenorhabditis elegans by CRISPR-targeted homologous recombination.;citation_volume=41;citation_year=2013;citation_pmid=24013562;citation_doi=10.1093/nar/gkt805" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Chiu;citation_author=H. T. Schwartz;citation_author=I. Antoshechkin;citation_author=P. W. Sternberg;citation_title=Transgene-free genome editing in C. elegans using CRISPR-Cas9.;citation_volume=195;citation_year=2013;citation_pmid=23979577;citation_doi=10.1534/genetics.113.155879" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. W. Cho;citation_author=J. Lee;citation_author=D. Carroll;citation_author=J. S. Kim;citation_author=J. Lee;citation_title=Heritable gene knockout in C. elegans by direct injection of Cas9-sgRNA ribonucleoproteins.;citation_volume=195;citation_year=2013;citation_pmid=23979576;citation_doi=10.1534/genetics.113.155853" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=K. Dejima;citation_author=S. Hori;citation_author=S. Iwata;citation_author=Y. Suehiro;citation_author=S. Yoshina;citation_title=An aneuploidy-free and structurally defined balancer chromosome toolkit for Caenorhabditis elegans.;citation_pages=232-241;citation_volume=22;citation_year=2018;citation_doi=10.1016/j.celrep.2017.12.024" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. R. Denver;citation_author=P. C. Dolan;citation_author=L. J. Wilhelm;citation_author=W. Sung;citation_author=J. I. Lucas-Lledo;citation_title=A genome-wide view of Caenorhabditis elegans base-substitution mutation processes.;citation_volume=106;citation_year=2009;citation_pmid=19805298;citation_doi=10.1073/pnas.0904895106" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. E. DiCarlo;citation_author=J. E. Norville;citation_author=P. Mali;citation_author=X. Rios;citation_author=J. Aach;citation_title=Genome engineering n S. cerevisiae using CRISPR-Cas systems.;citation_volume=41;citation_year=2013;citation_pmid=23460208;citation_doi=10.1093/nar/gkt135" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=D. J. Dickinson;citation_author=J. D. Ward;citation_author=D. J. Reiner;citation_author=B. Goldstein;citation_title=Engineering the Caenorhabditis elegans genome using Cas9-triggered homologous recombination.;citation_pages=1028-1034;citation_volume=10;citation_year=2013;citation_pmid=23995389;citation_doi=10.1038/nmeth.2641" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. J. Dickinson;citation_author=A. M. Pani;citation_author=J. K. Heppert;citation_author=C. D. Higgins;citation_author=B. Goldstein;citation_title=Streamlined genome engineering with a self-excising drug selection cassette.;citation_volume=200;citation_year=2015;citation_pmid=26044593;citation_doi=10.1534/genetics.115.178335" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Farboud;citation_author=B. J. Meyer;citation_title=Dramatic enhancement of genome editing by CRISPR/Cas9 through improved guide RNA design.;citation_volume=199;citation_year=2015;citation_pmid=25695951;citation_doi=10.1534/genetics.115.175166" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Flibotte;citation_author=D. G. Moerman;citation_title=Experimental analysis of oligonucleotide microarray design criteria to detect deletions by comparative genomic hybridization.;citation_pages=497-497;citation_volume=9;citation_year=2008;citation_pmid=18940006;citation_doi=10.1186/1471-2164-9-497" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=B. X. Fu;citation_author=L. L. Hansen;citation_author=K. L. Artiles;citation_author=M. L. Nonet;citation_author=A. Z. Fire;citation_title=Landscape of target:guide homology effects on Cas9-mediated cleavage.;citation_volume=42;citation_year=2014;citation_pmid=25399416;citation_doi=10.1093/nar/gku1102" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=A. E. Friedland;citation_author=Y. B. Tzur;citation_author=K. M. Evelt;citation_author=M. P. Colaiacovo;citation_author=G. M. Church;citation_title=Heritable genome editing in C. elegans via a CRISPR-Cas9 system.;citation_pages=741-743;citation_volume=10;citation_year=2013;citation_pmid=23817069;citation_doi=10.1038/nmeth.2532" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. Gasiunas;citation_author=R. Barrangou;citation_author=P. Horvath;citation_author=V. Siksnys;citation_title=Cas9-crRNA ribonucleoprotein complex mediates specific DNA cleavage for adaptive immunity in bacteria.;citation_volume=109;citation_year=2012;citation_pmid=22949671;citation_doi=10.1073/pnas.1208507109" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=D. G. Gibson;citation_author=L. Young;citation_author=R. Y. Chuang;citation_author=J. C. Venter;citation_author=C. A. Hutchinson;citation_title=Enzymatic assembly of DNA molecules up to several hundred kilobases.;citation_pages=343-345;citation_volume=6;citation_year=2009;citation_issue=5;citation_pmid=19363495;citation_doi=10.1038/nmeth.1318" />
<meta name="citation_reference" content="citation_journal_title=Trends Biotechnol.;citation_author=A. Hendel;citation_author=E. J. Fine;citation_author=G. Bao;citation_author=M. H. Porteus;citation_title=Quantifying on- and off-target genome editing.;citation_pages=132-140;citation_volume=33;citation_year=2015;citation_pmid=25595557;citation_doi=10.1016/j.tibtech.2014.12.001" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=O. A. Itani;citation_author=S. Flibotte;citation_author=K. J. Dumas;citation_author=D. G. Moerman;citation_author=P. J. Hu;citation_title=Chromoanasynthetic genomic rearrangement identified in a N-ethyl-N-nitrosourea (ENU) mutagenesis screen in Caenorhabditis elegans.;citation_volume=6;citation_year=2015;citation_pmid=26628482;citation_doi=10.1534/g3.115.024257" />
<meta name="citation_reference" content="citation_journal_title=Curr. Issues Mol. Biol.;citation_author=M. Jamal;citation_author=A. Ullah;citation_author=M. Ahsan;citation_author=R. Tyagi;citation_author=Z. Habib;citation_title=Improving CRISPR-Cas9 on-target specificity.;citation_pages=65-80;citation_volume=26;citation_year=2017;citation_doi=10.21775/cimb.026.065" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=W. Jiang;citation_author=D. Bikard;citation_author=D. Cox;citation_author=F. Zhang;citation_author=L. A. Marraffini;citation_title=RNA-guided editing of bacterial genomes using CRISPR-Cas systems.;citation_pages=233-239;citation_volume=31;citation_year=2013;citation_pmid=23360965;citation_doi=10.1038/nbt.2508" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. Jinek;citation_author=K. Chylinski;citation_author=I. Fonfara;citation_author=M. Hauer;citation_author=J. A. Doudna;citation_title=A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity.;citation_volume=337;citation_year=2012;citation_pmid=22745249;citation_doi=10.1126/science.1225829" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=P. Kadandale;citation_author=I. Chatterjee;citation_author=A. Singson;citation_title=Germline transformation of Caenorhabditis elegans by injection.;citation_pages=123-133;citation_volume=518;citation_year=2009;citation_pmid=19085141;citation_doi=10.1007/978-1-59745-202-1_10" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I. Katic;citation_author=H. Großhans;citation_title=Targeted heritable mutation and gene conversion by Cas9-CRISPR in C. elegans.;citation_volume=195;citation_year=2013;citation_pmid=23979578;citation_doi=10.1534/genetics.113.155754" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Kim;citation_author=D. Kim;citation_author=S. W. Cho;citation_author=J. Kim;citation_author=J. S. Kim;citation_title=Highly efficient RNA-guided genome editing in human cells via delivery of purified Cas9 ribonucleoproteins.;citation_volume=24;citation_year=2014;citation_pmid=24696461;citation_doi=10.1101/gr.171322.113" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. W. Lo;citation_author=C. S. Pickle;citation_author=S. Lin;citation_author=E. J. Ralston;citation_author=M. Gurling;citation_title=Precise and heritable genome editing in evolutionarily diverse nematodes using TALENs and CRISPR/Cas9 to engineer insertions and deletions.;citation_volume=195;citation_year=2013;citation_pmid=23934893;citation_doi=10.1534/genetics.113.155382" />
<meta name="citation_reference" content="citation_author=N. R. Markham;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=FEBS J.;citation_author=S. E. Mohr;citation_author=Y. Hu;citation_author=B. Ewen-Campen;citation_author=B. E. Housden;citation_author=R. Viswanatha;citation_title=CRISPR guide RNA design for research applications.;citation_pages=3232-3238;citation_volume=283;citation_year=2016;citation_pmid=27276584;citation_doi=10.1111/febs.13777" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. D. Norris;citation_author=H. M. Kim;citation_author=M. P. Colaiácovo;citation_author=J. A. Calarco;citation_title=Efficient genome editing in C. elegans with a toolkit of dual-marker selection cassettes.;citation_volume=201;citation_year=2015;citation_pmid=26232410;citation_doi=10.1534/genetics.115.180679" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Paix;citation_author=Y. Wang;citation_author=H. E. Smith;citation_author=C. Y. Lee;citation_author=D. Calidas;citation_title=Scalable and versatile genome editing using linear DNAs with microhomology to Cas9 sites in Caenorhabditis elegans.;citation_volume=198;citation_year=2014;citation_pmid=25249454;citation_doi=10.1534/genetics.114.170423" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Paix;citation_author=H. Schmidt;citation_author=G. Seydoux;citation_title=Cas9-assisted recombineering in C. elegans: genome editing using in vivo assembly of linear DNAs.;citation_volume=44;citation_year=2016;citation_pmid=27257074;citation_doi=10.1093/nar/gkw502" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci.;citation_author=F. Port;citation_author=H. M. Chen;citation_author=T. Lee;citation_author=S. L. Bullock;citation_title=Optimized CRISPR/Cas tools for efficient germline and somatic genome engineering in Drosophila.;citation_volume=111;citation_year=2014;citation_pmid=25002478;citation_doi=10.1073/pnas.1405500111" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Ramakrishna;citation_author=A. B. Kwaku Dad;citation_author=J. Beloor;citation_author=R. Gopalappa;citation_author=S. K. Lee;citation_title=Gene disruption by cell-penetrating peptide-mediated delivery of Cas9 protein and guide RNA.;citation_volume=24;citation_year=2014;citation_pmid=24696462;citation_doi=10.1101/gr.171264.113" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. T. Robinson;citation_author=H. Thorvaldsdottir;citation_author=W. Winckler;citation_author=M. Guttman;citation_author=E. S. Lander;citation_title=Integrative genomics viewer.;citation_pages=24-26;citation_volume=29;citation_year=2011;citation_issue=1;citation_pmid=21221095;citation_doi=10.1038/nbt.1754" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=J. D. Sander;citation_author=K. Joung;citation_title=CRISPR-Cas systems for editing, regulating and targeting genomes.;citation_pages=347-355;citation_volume=32;citation_year=2014;citation_pmid=24584096;citation_doi=10.1038/nbt.2842" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=H. Thorvaldsdottir;citation_author=J. T. Robinson;citation_author=J. P. Mesirov;citation_title=Integrative genomics viewer (IGV): high-performance genomics data visualization and exploration.;citation_volume=14;citation_year=2013;citation_pmid=22517427;citation_doi=10.1093/bib/bbs017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. B. Tzur;citation_author=A. E. Friedland;citation_author=S. Nadarajan;citation_author=G. M. Church;citation_author=J. A. Calarco;citation_title=Heritable custom genomic modifications in C. elegans via a CRISPR-Cas9 system.;citation_volume=195;citation_year=2013;citation_pmid=23979579;citation_doi=10.1534/genetics.113.156075" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=R. van Schendel;citation_author=S. F. Roerink;citation_author=V. Portegijs;citation_author=S. van den Heuvel;citation_author=M. Tijsterman;citation_title=Polymerase Θ is a key driver of genome evolution and of CRISPR/Cas9-mediated mutagenesis.;citation_pages=7394;citation_volume=6;citation_year=2015;citation_pmid=26077599;citation_doi=10.1038/ncomms8394" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Waaijers;citation_author=V. Portegijs;citation_author=J. Kerver;citation_author=B. B. Lemmens;citation_author=M. Tijsterman;citation_title=CRISPR/Cas9-targeted mutagenesis in C. elegans.;citation_volume=195;citation_year=2013;citation_pmid=23979586;citation_doi=10.1534/genetics.113.156299" />
<meta name="citation_reference" content="citation_journal_title=Quant. Biol.;citation_author=X. Wu;citation_author=A. J. Kriz;citation_author=P. A. Sharp;citation_title=Target specificity of the CRISPR-Cas9 system.;citation_pages=59-70;citation_volume=2;citation_year=2014;citation_pmid=25722925;citation_doi=10.1007/s40484-014-0030-x" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=H. Yang;citation_author=H. Wang;citation_author=C. S. Shivalila;citation_author=A. W. Cheng;citation_author=L. Shi;citation_title=One-step generation of mice carrying reporter and conditional alleles by CRISPR/Cas-mediated genome engineering.;citation_pages=1370-1379;citation_volume=154;citation_year=2013;citation_pmid=23992847;citation_doi=10.1016/j.cell.2013.08.022" />
<meta name="citation_reference" content="citation_journal_title=Cell Res.;citation_author=P. Zhao;citation_author=Z. Zhang;citation_author=H. Ke;citation_author=Y. Yue;citation_author=D. Xue;citation_title=Oligonucleotide- based targeted gene editing in C. elegans via the CRISPR/Cas9 system.;citation_pages=247-250;citation_volume=24;citation_year=2014;citation_pmid=24418757;citation_doi=10.1038/cr.2014.9" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="CRISPR/Cas9 Methodology for the Generation of Knockout Deletions in Caenorhabditis elegans" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1.cover-source.jpg" />
<meta name="twitter:description" content="The Caenorhabditis elegans Gene Knockout Consortium is tasked with obtaining null mutations in each of the more than 20,000 open reading frames (ORFs) of this organism. To date, approximately 15,000 ORFs have associated putative null alleles. As there has been substantial success in using CRISPR/Cas9 in C. elegans , this appears to be the most promising technique to complete the task. To enhance the efficiency of using CRISPR/Cas9 to generate gene deletions in C. elegans we provide a web-based interface to access our database of guide RNAs (&lt;http://genome.sfu.ca/crispr&gt;). When coupled with previously developed selection vectors, optimization for homology arm length, and the use of purified Cas9 protein, we demonstrate a robust and effective protocol for generating deletions for this large-scale project. Debate and speculation in the larger scientific community concerning off-target effects due to non-specific Cas9 cutting has prompted us to investigate through whole genome sequencing the occurrence of single nucleotide variants and indels accompanying targeted deletions. We did not detect any off-site variants above the natural spontaneous mutation rate and therefore conclude that this modified protocol does not generate off-target events to any significant degree in C. elegans . We did, however, observe a number of non-specific alterations at the target site itself following the Cas9-induced double-strand break and offer a protocol for best practice quality control for such events." />
<meta name="og-title" property="og:title" content="CRISPR/Cas9 Methodology for the Generation of Knockout Deletions in Caenorhabditis elegans" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/1/135" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The Caenorhabditis elegans Gene Knockout Consortium is tasked with obtaining null mutations in each of the more than 20,000 open reading frames (ORFs) of this organism. To date, approximately 15,000 ORFs have associated putative null alleles. As there has been substantial success in using CRISPR/Cas9 in C. elegans , this appears to be the most promising technique to complete the task. To enhance the efficiency of using CRISPR/Cas9 to generate gene deletions in C. elegans we provide a web-based interface to access our database of guide RNAs (&lt;http://genome.sfu.ca/crispr&gt;). When coupled with previously developed selection vectors, optimization for homology arm length, and the use of purified Cas9 protein, we demonstrate a robust and effective protocol for generating deletions for this large-scale project. Debate and speculation in the larger scientific community concerning off-target effects due to non-specific Cas9 cutting has prompted us to investigate through whole genome sequencing the occurrence of single nucleotide variants and indels accompanying targeted deletions. We did not detect any off-site variants above the natural spontaneous mutation rate and therefore conclude that this modified protocol does not generate off-target events to any significant degree in C. elegans . We did, however, observe a number of non-specific alterations at the target site itself following the Cas9-induced double-strand break and offer a protocol for best practice quality control for such events." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1.cover-source.jpg" />
    <title>CRISPR/Cas9 Methodology for the Generation of Knockout Deletions in Caenorhabditis elegans | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"me7TrSFGDJ8BrDw_4F_uMPPMvecTUSyCBfYUi0qCFFA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491787","apath":"\/ggg\/9\/1\/135.atom","pisa":"ggg;9\/1\/135","ac":{"\/ggg\/9\/1\/135.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/1\/135.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/1\/135"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/1\/135"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/1\/135":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kZqS6E4TZEdBcOgabwCY3bC4pK2jrM\\\\\\\/eOtldH2PcOUi9NPWJHiHUzlhctEyl6QUhRQOyAEdgC0lMbnsARPVhI+KrjALDgAOqk1fX4THFeBK3JXi3dpbMytkXtB7auMVYD1bcMZV+jhsswuXGQ4sGqgiqi7uqOfMbdN6XPPX+BlbzpnV88gcnNPP5z+HeLpkTpQhfBWRVPmn7am6ea1OHSOwf10MtuS198dA32TlULCxMZ1KKP5EuBN4bkTy5faKPeMtCaYMfH1uZArdGDB6nYl3K85zSJhulZL\\\\\\\/Jk6DHMQZwv6JtUAEecRRNp8ykAyCXaBC\\\\\\\/qr3QdTqhcFR6o+7mso4pTQSOm8XtifFZeqwW6ixb6rSy9da0ySNdq9pUSgsM7ajJ+Sj15DOsuBsK1lTRZMQADad98zftKszpSKbSNg24MEpPKv\\\\\\\/k6KvoxyV7Xs1XXGYkmZeWAlz7RL\\\\\\\/B9jc2gKcwDoYyaMztORM4yW+dA+eP9qk3QbXbMreOKXhdCUjzYxcElRnY6x79mzwF2oTN0oTJ+kM4GF+q2VcYi6EJTSLi7CRQpi6dm0E9rWjyXV+VF5Aym0ADb265+E1LrM2OY+BaJVd5xjrTe7229cUvFrFrNbVBIWi0HA3C6pkeGvkPl+F+o129W+sn+734RIagdFR+VBkeIZMAZs8eHHPMjylR9neBmpsA9K8OXsVXX9IDBXLsb043l8Sva7P2r2eMppUxNMOzKaB6\\\\\\\/Hb3z95Q7Lecjapf1w62yeRLRqfOgZb8X9FOyvfcihgY5Zua2twLVpPl07t7FWaB5tXtaR5XQXks8O8cKKcEBFLExrOTzbCN9jlejQixmA+CraGEMLhFx5HDrltoBn3ZM6eQnLd7JnECT6\\\\\\\/IROWNAhzKQhBrMH98Yvn8JDeuvnKkjjpIYd7n3j5fRN8yUoln+wRm8diiPDslMrGSHZ+dIZAB\\\\\\\/McoIfzuD1o2kXE997GK38FMfxYuv4LA2odsqAFZFz\\\\\\\/Q\\\\\\\/Brn6Y9Hj+G11cbafwGE+Mq6Xyoqa3OQjFlV6y4KOCsdrODBKy8y6LPlNqBVny+Cmygt9YjNsXlKFywNc2ydoAvwYZ6npxagGJidtFH9r2bQbe8sjnKzNBs\\\\\\\/3m3zHCRQCTbLErvwOjbTruA91+0LHXSVMOvGBVhTnZHp3ilu4cZ4C8YSU6eS3ANrp95JJkjhG7UEss4fpJnSMWGQGxIvjsHkv+G6bwpWRYp7FDdUJ8ncFxynFh\\\\\\\/IOjErGX7Bj2RgW54L2onZ+ETSlK7EImEaZ743VnTU\\\u0022,\\\u0022iv\\\u0022:\\\u0022EXi\\\\\\\/IXImtgTweQosdEKqzA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ef9a54823d043c842ba9ea0e9614efb36fec52f3a1ae1bbae42eb6888d5105a2\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022d\\\\\\\/pjfE+KlZVBFB2MM5D7kAVjSOcxMKS08Ido5Z0psoo1TwN7gxxIRcPK8JekY81Wg3xkvOC4zegnFD22tlNUAGlpTt2Emgs+u\\\\\\\/2bHGmH5Zi6uwJG0v6VpbAiFThXdii1Mw22MpHaek8KK12GtvOqdMTCELA\\\\\\\/dFDAXHdqmRLGn5apJqTWpd0O+WTUuqtykFlQbDwE8DxAzCVhxhYjpuo7ADsp4MdZPt8VBuF0bZOxaG\\\\\\\/HGBbmU3GA54hipf1+3qelsyLxzx7WUUWdHOEd5rsI4bch2JHBE5BYOC3haasFzGM5WpfZ0HBjgpF1ocMSTUfBzbbswvrdcYdiD\\\\\\\/2OkMHKmVemERpSAdw\\\\\\\/hXuloLjnKzx\\\\\\\/eYdk5kvwbAu0\\\\\\\/dmmgRH5kZdE3wGSpPBtW8lOV7MNCwrSuqaFN108W\\\\\\\/AnKu0OeR7VrGuaLL6K8mUXIysJXZ6vm4G1\\\\\\\/WQD7aZylc7KlCFph1DxKmqr9VVyCqpId5dvtvpSiDW5ZRJQTPkpOV2kvtrrwp7fYm1QD0RAwuuK6tTItwK0aS4oVEKTLXKfq7piuZ+tWdMQ5OCqloQUftayykO4sJ4FOHpTkLQ7bBl52b3S1obwDtMD77u1qVFocWK27gSnk59Z7uzMD1gWx00hVy\\\\\\\/Yf1Rdt11Eo7ZIv3GZYchUYyNF\\\\\\\/EkvlvCK81OGjv6JvRSW5y73NkV8ADa6NDWwcXXcd++1LOKwPuLMkzbm9WjgM5VCDjwRFIxBa4t6GvrHj3N0K7d5bGm1G7HmyURnCekmUeJVPxmsl+iH+18NFqRNqS\\\\\\\/KvcX5sCSmD1NQof4U0xUoY86dIpSbT4iVS2Fu2QrKvedPuK4udoaAudhfLdNRD1VbXM0Ix8n4Yc0A0IcTKn5zKlysRI5mwoDyYRd7IddFcYvL0ETxkHJT1abi\\\\\\\/vUVNQdmKmcByjDq6BN8OiKHC8DLgp\\\\\\\/tcURqnmJ8Iw3\\\\\\\/f9N7GTM6Gk9X90X3o5uM24BbEu5HkD2ZD+MB9C4xkwTf2CsCqjIYUM0OcDU5QxiKDylHg7ljbkGt2wM09BCbThL2FgldU\\\\\\\/WZob+yPLuVbVKFVx4n18q1tLcbJopZIEYTqe4AXXD44\\\\\\\/SAdgdTiZl4iWLtu\\\\\\\/EWuh4ixPAWYawoG9hQdGSh1fWT+NdevvblNJ3mYO4d\\\\\\\/K+diTLlYYUSQInhATVrjnd6pdbP1qk+vTcbwDZxZeiB2WtUGViJ\\\\\\\/N4FWsrudMBCVIgdp0I2yj+pXy0meTSvHiouHNOKQlb0oMydksrhjn1ZCqncucfW5rZINziUEZ+qOqBC\\\\\\\/2swjndCCxARdFjMX1H7tWnM0alwc2CK7G\\\\\\\/5thf2KOi5J6SlDAyRwgA8Ef9aCUnAXWPdaVtIWq0kM0t7bOALWoF0n2s7MNmGQj8Qk\\\\\\\/WdPrifkUwtS7ySO6iu\\\\\\\/39GhEMjnkQIsHP9649wh5lDRIYUcHHlH60lU9A3DV6QNkoy2fYrGjLy5GOQ+BKJfPua7IMBCqD2vJ7fE+AkKxezRINxWGYe0IofL1jE1EAGGvbytK8MHs\\\\\\\/1iLIDT4oHzaxEuBOItGDS03I2WdhE3hOXgKbgMwzK8gKhX3uVdeuzLw5LWTu9JQ07ezLRXHDyJLQ9lUaR8iN0SzwcX9H4EJatlRNiSEtW9XT8Icz7Ci3v8tmqKOMD7Daop9nnxZWQLCfzXs2IA3oiFsRqE600DBxXDaFQzB6vVxgLlu45OAj3\\\\\\\/cywSGnOL12Y8xFvsQfMBbu7NjWZaMCC2L\\\\\\\/Xh5SPtwk00tDNczZwHdaRkX3Odd4l8axkrvjtBvM4UgMz7qJxDRdjx5qEJI\\\\\\\/0OiW\\\\\\\/dwFtk+ogbqQs+KIajq1Zk9fD\\\\\\\/mSYkhBdaMA\\\\\\\/8xmWq09uCSFYjM6jwIYIvTdXlCz3QszS88BBigUdXH+PcgmcS7y5rohxY\\\\\\\/7yXiaLQKCwPdIO1sG6O2ZmvJmBP6fUw2lNBgyliMSO00HviWGv\\\\\\\/whIWZrTZ9hSc\\\\\\\/srE8TPN2eFYp4DiCWiwTscersnwTTeBAhQfIWkd+LDOsM8j4T6ZxxMjGqgWE2Hx9UTG5V8eQCMojdZBYO8+oR4K5UJPtCNoKOViB36+PuKUC\\\\\\\/TFmqeRyVS+qVfB3LhjU4zQGr9bYdU\\\\\\\/j5QWDYkQbBGfH4VWVvFGMMYeRaLk6TSbk7ABEvVFtfaXKklKDS8QiVW8NRPHNOo2RQzSTM4fSQZQqrXY\\\\\\\/vc\\\\\\\/q3ZFTjs9sqvAXAOS7S\\\\\\\/5KiCUMRZxQp6mVlBgzXQBiCT6cEAItLk6uZVO0w9ALzFovUdJZJRCh0nyKiErTR+pxuhAws0fW+kKj8niC21BVKJ4nOoOH2clCo+ebWuHu5Oo3Wv8PyJHUN1tDZDTEMgOR5O6Fxvn\\\\\\\/Xxnlxw+hi30pV58wPW0Ompvq4qelkxGgSa+ijN9NOtz+4vTk7H3\\\\\\\/vXSJIUqAHMam2zFDIBaWG\\\\\\\/2NajaS\\\\\\\/i+Rg9EJJ3GCRkujoHq9ynj2bgvo9PFHNA5qU+Vbs\\\\\\\/kEQCfS8x0ILUfBcIXadKJHWD0nHPTJsybPoyTuQnG3tTf+Sap7lhkoEreoudw\\\\\\\/AMHIU4kNIN0sbuah7grP0prgVxMZ6NjWoiZCgZ8kWAQahabC7w8UK11Xna61XcII83DvC2zeNjXazbQChm11cwBN9zgnLjiloApxAKLoPMIsbex3EE2H561dDfSEO\\\\\\\/iXLNVEEVt0i8tNw9vh4JUreR3S7I5asJEkGUQkt\\\\\\\/j80EL\\\\\\\/S0Oi9ilu7C1jnMLFf\\\\\\\/wOrRd7mT4Dhq5MQmTaZTkAHaLrCOLL\\\\\\\/JY9TjcHacQMOwicnNCDoXrFGlxx0dpceb2D2ClwwAK702OxAy+fRwlJIPvh8c0U0N\\\\\\\/q7BfW\\\\\\\/BUSLQsK0Ix594yn0XpZkL2Bvzht61tixlrlMrSZYw75G7IIVfsyIn5g6AFcWtx35C5byCzG2rYhuMb2UcdHY66\\\\\\\/z56yxXiGIrE7dbAIRTyai2A16LejYId0gBhdoWbxSLZ5HHCZoNBros8WDYV9nUjY9esUUPI1BnXDOlv8gZi1TnrJK5LblIc6TIA8Iwo5NPL4Hsk0pr35Uj2fwTobKvCqbEEMRaamP9YahGciMonZJu+9LXnt3fxPiPiITLjkk4qfAUmLPm7DQKt6RegIJ5rme4\\\\\\\/2keUYBI8hbUa1EjNYrXpUWQx+dXmlNafdh8mBdq5mfYhZS7\\\\\\\/mYJ+MJ\\\\\\\/QJ7043Dt5KRmV4SSzlUHBX3YwSwYCvFRp+TfPRBytKnBx7ZYyI1OHCZKSLbgikHgsbFyoOsUJ2jh9lDhhDKQDnPQCHLt8OIt7yQ6DesD2KQ5dFvziLdFw9u\\\\\\\/8zrpp\\\\\\\/Z4CowjZ82yAVX5E6JPaKyzDVQukbclH\\\\\\\/T+jJgoJ+Dv9bvQGS5bdqibmpwdXFkdLg\\\\\\\/hXebv40McEHu51EEoIdrOTKoctqgywn+tXQh399nnW9ngEhh\\\\\\\/iyiDRdezaMpF9IuUcO1hZMBZBmxAEco1x4174t9w1vD2GBff6JKb7Etr8DNcc\\\\\\\/b82iLIHAD0DIjCrXrdxoC4ckmvn63RqfNc4zDjnxClAgzMbKlAqcTB9q9BAeCeuZ5u\\\\\\\/eC8l3aAS54kXHVkEL4aW6c8qd6Oof72JdNMEQTJ64VuP7bdWlcld9L2XxXbOJD3WbDVT\\\\\\\/z64naxqR93uElAvUjZmxdE05GRfCEIxc5u2DSCxxvyEvBsI9eLTQD20qjyy1rIQxjEe4ZTp\\\\\\\/PnWEumMR1hy0XjgVyQs0JVRacKaCHYPheSLvuoEkCuMsR015NOmIHVd+H8GapfVVC1kZcCK3NwZX1kQ4WB6QQKD1s2XGZbN3a24ehJhxieVJLEToi7heYp9WcQbhm62D6k\\\\\\\/Qp3MwppBITD6P2bmEaZHmSEZK2a5yTcsNh2g4gMxVEx3rplSVI2S3Y0cwAaRgrKSqOZPlMfJhrTd9uFjVQCwBpChWaaas6FMcBfgRYSMTt9XwHaNP6pvJD\\\\\\\/ajdbhFZKsj6YPnDFkbdMAiGpO9IyeB1AMcJVjP1nCsNfcTleQ\\\\\\\/iLdHnx9V5D\\\\\\\/xqpOHI9Kf6p+UB1abXP1dN0ODuLlERg6WMRiRtu\\\\\\\/Ez8fLKhizZ4mvAmod\\\\\\\/EhcnIvKr5XzCxbsLE2P5KiIyrIXuTNs2q+z2y3fep6PY39s6o\\\\\\\/FMSL7PcRJ34WACeAxIz58P+5gvgjfcjb41\\\\\\\/bl3PRCrEioaw3YaZXPewL\\\\\\\/tW6EY78qajuDWjCAf9yv2hcisDa2PV7tZRYNDZNLYdW0wsmK3QG14XuGeNIRU+8IuVStpLL4lsTbGoOYLvq4gTKB73hZvXbOcsJxEPB5\\\\\\\/jp15mXNCqC6n8nGlTT\\\\\\\/aWl5WDufAXLfMJjXGuo4IR0n1TLhhKG17HhudGjtnGrKDp1zb+QZSHNdbQRO760DmtPtXP6hvTUS4ChMxFQdHrHJsMKrReaxqEVo9HlPorWML0ZZHR9MV4FG5wYURpMsqIGZC4M5s5BKnumws1o6o1GOTu+mnznRolb6+WRlU3jEqRKoc6N317EnuAppUHgdkENWTVs0sH2Taw0waquysyGZmbA8QBA1i5R9m7mk698qX2LnaDl1MCoKOqQkc0tMt3yYHorCP83qfsJxFzBDWAWEaevDFKxi+7M5NgZorjC5ydvi+s\\\\\\\/I2\\\\\\\/Tz+GCW6AE2Z1mz87lSbUJ7UGFZXiJYTbYUFtnR2iOb4H1fDsFLeL8EHNTOxNZS\\\\\\\/LdkQHAgFH3bxRF2plyQuYqCAG8CVaMva5oPrrPHFOSKnyNK9dUCFbS6Sf7Dc3RU0B14bn4X3f\\\\\\\/ZvLGkjlKVKqR582wJkyn\\\\\\\/0snQwNt48LCXsSBFFBU8EY8kUCAJyhF+LB1ZHbAogkASGIJiXOeATePukqszzC5bcSvBZQbBvipUqVFHT\\\\\\\/3c8xqPL+IzoM\\\\\\\/UtAt66yb48BmzV7\\\\\\\/pLheX6rBuc5KsB+D9wj+cq\\\\\\\/2HxU8xVwfJ0Bh\\\\\\\/YLmTQmFNjsb0OSd2TCoi6Vaze1gNlwBUUw+jTPY\\\\\\\/gc2i7PCru8OdYCL1PygbJ2Gk\\\\\\\/KpR6XnTIyrVZQh40h3ovthz4hxeV8XUzE5vHuApXV28z7Q5\\\\\\\/9bF6De4jW5J\\\\\\\/BQWrMaUSGOQ8aCCUyTBVjxiz9NDoOdOIe20MRu7zKR3fuV+oguZLOmf7HqivmadGNHjJrXPM9GR773PHdFlYUnXzFK\\\\\\\/lO6c2j1ALfaJfJx6mRaWhQ9pLOAbuPgKIsrbcxHFVmWxUt1C7J7bv2b2mmrDCSbCLPkZd+FSsm\\\\\\\/tPh3N+8\\\\\\\/PkH4ytdl0xC8rYEnjMZfQmMvvnPtPqeBVXCV4v6PNAj+BeloH4lwVQm0GKBBHBED0pKmahMg9ipFNLyQI\\\\\\\/qEkXJxwrDrq5B6OIYl5v0ynRA+XyKNcvgQeGQzOke\\\\\\\/08kFatIWjzwc2f9agDAdrpYFi+5VIQ+lcil0PS9BtLDkC6meeTJEV6rDrV01e6c3bbs4H0rZziWrM9bt2c29hl1JWklthXbvVwI2eOvAzCvx4zCu8ilruyVjtsFat2P6sb4nbuUsM\\\\\\\/lCSq\\\\\\\/JDLnjvBrta4V5+woAvlheme\\\\\\\/dLaLvy8NL6TbeESJXgR7AtGMNuCAUFgI1temPMasVL0jYClkncZqX40Gg6LNDyXxiVNufBI9yuDcKjIfHb1HGLp7Tkz1iLOIG+55rpzY7kHztaP8KakotxJz8Ek16LBj5QLiClOn01XKoMPZXgs2clY4QHhWU6yDxkcruVQG3KYTaH0MO56h4d0lnvpdlb5OeeTcLG6f6WI4liTcHG1mQED3FE8kXQ2wtMQO3e8q\\\\\\\/tecSgQYDQFDbYTFD2h6Cpqk16mDj5NhCMOAetCKSlQy8qo\\\\\\\/jJJCYiqV1CDRqms4RSCbj8xD53UUAkWX8P9Eb6l4Slp5KPBQDj4FwxqJKpn4WctXhFNzIo43esTOk4Gp4DlXztaxDVh2DUwO9Ixby2V94wddMct2CTcZunu3cd2NCqGrNIHqrZP8QWtbSYnEsAsT0Z22dxxNHKk\\\\\\\/Y0OivkuODjbdLseXsNMEpv+soGpgYo2v7Ymh8Lm0CfZtJvKNDmNP2XwKzduDR+Q9D4JmdCYOmkBuYEuTNLpMB4BXZoG5E\\\\\\\/+YXG4dXabW+ctN0Y8TiH\\\\\\\/Vt9hx1ihnrBJW+Sh48nYobb2Nb6vJc7Cvw1AMHCeLq4LP1grYMsBHn3OiUwecN7SrnErRTyrExgIrHaDzQi3dHP+ks7FnutwoDJsrljrM\\\\\\\/SHXRn9OkuWZZG6uy5A029LNRKvnEGnqwseBn0GbhgrtSef3KBSE+KrYmpakrwV30VqmNenCqkMTH9J26HXYuJEKL2jJle57Cp+Ze9c1ey\\\\\\\/kuKItC7dzfCgvsUCjQbyWc6xAnGd+jZZh\\\\\\\/fEfDnp+2me1TJ8t2i\\\\\\\/ffQUNonrqodf7e48KfSjo7OKPEV+2a3VRs6P4PXKr0t0ormRh7jQ9E6STG3CDwlgUbtnHbvRMNPUqSUwTc92Y7BRb4zX0KS1pJRXdIye33wH2zoErpogi+Oy4LfgBZhB9agVnCvSrpA8Y951jEruRn9kvbEaibPHN3fwAu012dZXfI17\\\\\\\/VEK8WpW6K0xa9KSQYXSYGui\\\\\\\/ZumZbSQ5w1u3ib1bjPldsj3v7aF6lCEgFra4X2eI674AjKJn2IVUAbzG4uoqsuuiT0LkefkdP1wTTKFTDgQG2xbGlhjUKtDg9S+0eiI1oXp6cWEpFmmisuhI+A73YsC9tnOrfXL\\\\\\\/0kkRBRyGWFGpaZgO+5gMjn5YE9tesDLyjztFyaMOHkJlfBlDT9MqkTt4vOk5c6sDN0KgOzBA43ozvyPSOCIMDLlAlyHV0v1bUd\\\\\\\/RKssbzk3PGQb8M9FgW9n2AUI6Ylw+6ZfCPjeNUcPgwde3rmNFyqT0VDPOt\\\\\\\/GG8z8bWzNQ829rj9sqXScBDFCunPaAkWQJSYB1ep4TlKKXN0QJav0IOv+Svz\\\\\\\/bC9lapCps+qp1+6nzsYC1BS5hT8rjWv1xjo1r9L6HZafjjPSIvYttHSZZXlbZfaRg+AYsRuj4CpKWBUp24hsUfW6feBbq7MqD30ZhmvYCk07ny\\\\\\\/Yg+JPmb5Ujc2cJG7tzLAvHGNGRwj6h+hRwlwRIiGt+QJTgoUQUseQ4oLb13HnQMj4EIbh8weISnqWg+ZtLU+Y+EAico4rs4nCxCne636\\\\\\\/FDOd\\\\\\\/gyYFqjxWCFvKeDE6m8y5BdPbgjjZc+BlSIJkOXiotSK2w6Mf20Qz7\\\\\\\/2k2fRsNkg7IMvFp5dpopnyoTN5SsT4KCMRzeprhBN\\\\\\\/yN1htSff\\\\\\\/kbQlh2RJhoClOueBnj3HSAgT9PRSFfIMGCMtWfed6EcsNPoDo5hPAv2gkh1Nh\\\\\\\/tP9GNt5rwMdUTml+soVF6og3u7YgulcGgUnPQaHTOAMDpxH21CYGsZ4baS2vYAZFee+PESkSurM4uBifo26Le1vLWaEX2uPcWmIpSebDwDmqmiiVParJmd25UTCZgQg1+uY6KmYIbcf\\\\\\\/irfkEXaI7\\\\\\\/HMIdwIfZSgd\\\\\\\/xxM4QcTqOPO5QIi4dQzTs4HkuGajpjWhDwNeEdijbWBEgMzHi3wxW29dgsEw5mnyKVHyrcZIxEH4WCQu4vgdBQO0tzINVN6MPKaTJMkWS3rOzVYqNamzYXCf1UhQfBVrAipscuPa5BWCFOAhBDSpsICLT7dj+U8GlhAd5M8VAiEe9ZnK41nWewcpoRnRAXH+J0mPJnhDYur4FFbF9JnGXPvfD78kp47zm8WAb5VfnZsytfcWZai3oaMACr0BAMDPJ2t0cGc8cX2TT4W\\\\\\\/OTY85wWneHx\\\\\\\/7tA+MZBf\\\\\\\/gbLMSYbXw4F2KF47v\\\\\\\/jbuiYduCD+fX8F\\\\\\\/7gkc7ZrP0aqonahqDprBc2kLQhB66HTE9PL2JhgTKbvFkbn3fGSNzF4LawowBYhnm8h5S\\\\\\\/dOXqc6c5BRGIKBGOwJi7ZhGsbBRmOPLeaEVIQLsoU1Om0SyhjTcCv+ocqMxp+YAMQzsI8HNDsj7Zoqdxwu+X0nnD+OKLFs6P\\\\\\\/agAkF9CBNb6wQXE6Z1blNI1TIFgB53XOLNJIdy\\\\\\\/bRhS2swSb6scpdwbxJ49KnDr+6nQROekjX0+E7tGX49CbGJbbpr3CmdaaOwhi1bwWTJ\\\\\\\/O2wFpDMJO5fiLuiqIzX0qzxemGv0TKF3Hr0caQ\\\\\\\/dLZYSV71BVzbLmTLj4NkyngB73Wsbh1euDEOOwIwhjcx1C1Kdb057Hsbas0QGypFcVy7r7XcJAw2ZGOuCkrALCQw2i6FHX0PH4JIbEph43g16TzNfejvnWZbX3zUoNzhXuVbHOlo8Ki+twscFW+ZNq2l8lX5sNy\\\\\\\/dRQ8pQKJNl+4PCMtrsrePCyqAP0totALkUD5QZP2s8+re8hUohaccRC1ublk2RF61xNuEy6qhdNZpRgSrkOen\\\\\\\/5pX3m31D0huixxFpUUzG7Hw\\\\\\\/9fb6HQZkywqi6bwoo7wHoMeAc7ZQm6byL+u+bstQOk5u3RM5vUWAj1\\\\\\\/1nI5fNSlh8kXuAjzW+iXV7WQidyV\\\\\\\/QHjYY7uBh3s781DRZ5qSwgWLHiDCuAvvmGI\\\\\\\/OOErRfEjnMefknSUi6+AHocOZ9Js56Jud8EKQ8hEGylalk3pR5VzFdqrsL2zDfIKFGzp1dRQxeKXHdUtX1UqMhP3Dc06MvVBiU+visKiaeb+xTcvAdRZqLJg9k20pYsAgA1toP5x2jl4D1gEEeg20C0lLIagcJfhLKYhEODR36zvuxAfTWGC4CtMvDakPZz2KZLf0MZ5K9+ZGrqn8iWLicEp37HnTfKciVPlyF7ZK+Ese+LXUZQUxBxuoFrUTZ6drgA7+xAeCmzDsf6F5\\\\\\\/c7X0HVNbr3mBXcKwb87AKUhLjd0Hi1LG5WV6uykmt7dvtxcUKr6z+zYn76JFv1R4BHE1dRU7Ve0LIj5Fle70MJVLFDq7+2zzSbOyPb18LoYs5vH6oGCUqdsU2hAkipHWOKBTotJJBKHO+7m9ZtAN0wWr2bjchPIPLjOHhsYPhUJkiBq+XrLG4vFd5zzjOlzM2p6tIa86J3HnMUaywAyvsZfD2M\\\\\\\/YYrpo+f0wZ6uF4x2dHMqjJGNlxbAxybQx\\\\\\\/2gi6ZltCYdcKrTW0iBYjE9KdVt+DRWuBXgoJ04CdZwA4H6dsdAM4XaveovMYdkIXELcA5+mixPzlJI2+gpIt\\\\\\\/YS+c2L0BkprGPIsVO1MyluknsVjJeRQ3EGQOBnw+lpLjqTAPrOgPCB2w6sFHkw4m4rVq7rcq1+\\\\\\\/jrqYXUPpfXVDFshSU6iKCLQ0Q237CA3T+oga8zQ445d+DDHzMonsKrCWZUAdlL2ZR41id5Ri23a5vbWMqZPamgQxkHECo3Vb1qErjOUWd2TTE7lfP+D0BlayVbj39VoACN8hOkMYOh+jQ+Ok+AdoHXMajBktay0eZnCA5kIhDz+Rv4qNbhD7ULKUD9miHQaWMTJykKywGayoW45K\\\\\\\/I0szQoyrXWkXNhxJtDXCpov+nxFQlFm6ui2tGydcIwnH2QcZBFcKmjOOKbo3e8XWVVxGR2zF\\\\\\\/YDJ7jzMftatIP+PzAiOpRVRzUiaQAMbv+EBCAXbUqN8uVoAH11JccquRZ4yQoCfkScypchSpj0BMrIYSClyFTElEk79fgmySOjb\\\\\\\/k+OX5xgV8yMh8Zd9ANUdfqJnSy\\\\\\\/msu0K+UK+jNSZVjSPhRrMDOaeT1OTeeZlOy53TTA0mmm+n+ua0yVMgGoUzI5\\\\\\\/WSAMKDQyWRaMyqjOwpn1NvM5mzggw+VAei6DF9GDh7ZLVxtxqIdkkQpmNTcwdJAGJMItv2CFvsaz2TpzchKAcB0ubUgLwsvNmrT9cB63A0bvMNEFC15fv9wvcU0ttSQGQSx+rSdcb4zhmogsxGpgladhuJMM8myDfB8Vux8zJpNi7DI2sXlE7WZOY9FlPtZKf0hARnWX96Oq4Frsmcr\\\\\\\/YQaywRdR\\\\\\\/nWqPJYZ86zzrrQg7MAvT5EWeojT+a1O6QKMQ4KCnWn9CeqFopM46e+Q+rOSKD8TRLBLov9ZmfFOfLm4SodOrqRYdRAbuix4v5bzQN1JOhPJB5sMDqccijJnfE4hLH3ydZ3mVdt3YO3zZO7lA0h+4s78uEpZER1P7RKVZvEqRbJv2KXQuAPgf6eqnvAf\\\\\\\/\\\\\\\/UQTmaz6ZDDk6JDXmuqtgV9lyNd63T+LzbhkOQaRw04vCGouLH+Bo8N0NbqhJOdTXdmdLEuG2Fbh7X\\\\\\\/xeYxdmhdwl\\\\\\\/0iNlil4u\\\\\\\/H9ev+azNdBvVN8K\\\\\\\/Am2\\\\\\\/S8hIE00jzRBsFvVfC7x153rbbzo8QV8uGw+focD2fgNtLvWFPYpQSPQYeeSEZgk4lwxrMJ9iz+knZMSAVhn1tekiUarK+5pYic074LhXlI2O+7tf6x4hggpWbEwY2BZxIlt0tZy3JoNq4capqlQokumPM\\\\\\\/qatk76KYg6hPXUQXoIvoIoFBSZuXcgXeoz2IbnQpaIeceoUvw6k5zkaZX45DW5r5WBGU3QeLeu\\\\\\\/CFnYMW1hgLDqLXZRFHCnJz6YT8RKZfoprTsPvkJbAcM0c21f8QQFXVtcEhmXmH3s9yeibifoJwhzAutuvcsg2ydPyD+FQ\\\\\\\/Y9seX50DtrEUsFqz7\\\\\\\/DdcGmOLne51KmUlvqtVe5PQUzY7o+AIRVoA0RcIaY9QRdQIEF+NxXxdj2dFQDhl50xQAFDsCnaNr0PnKMOSn1L51sDoapUcG8gpudOo5OIHMaBGFyi9r8j3WhOfydbwqkCjKFuFFYr0p+mcDwOmsJKB3k7HJj7dz95VDSFxABRn\\\\\\\/wooWK8DOuIMcd9aQ7CGHQAiNxHXQv24sxDIaYfYhOvSFYNnI5pK\\\\\\\/1tYEYF08LpXgCS5F9xw82NM0xcSjbZswaQ0DL0xSQlISuNOWjLQIu5KnpWihWZmo0yL22Bk4ywRy7h4AhCoQQbEjvqpcGvp5stfhnlvPtdQq0y5LtjxHj8nF9iGlT64rr831mRQ+JdiPWSxa35Z42etbCrcYqrM7nbFenIlpyJueYBVu++VD++bbCHLUeBdUmGlnQsNx5EHanz3GLX4PyIIkzL735NBoxKaGA+CVBcuVAvASJn9oS+OXKT3TkeqaL1kf14kQ5+9zDnSU6K0wWFtnT61yM9BMvLKG7q4syHhmD9gB2W+UJAbmNX8YSr7oN5Iw2Nw7w5+UrXUqewQFa0WLRnCt2IRkz1\\\\\\\/+J528h+QbT0G1+ymFNCfUhp\\\\\\\/DLVnxZhCtGu46G6fKVvOiPGE0ApNpDvgIE8ub70y+CP3r515a7bmVTSwCNIYZ\\\\\\\/4ZLTAXEVShLOIAd6sysgOGv6u3yeKRyli0b51Zh2R1+zkylbRQ6vhUu\\\\\\\/Ge+1dJryv5iBPmENwlkXh8Bnkd1QBeVlWh\\\\\\\/Y3WIrNUztL\\\\\\\/zm6LOA9DvszcmL6tX3C\\\\\\\/+k86l2e8sg0XLUsAQValIdmYGCs34ofZDpXnwDGr5rhzFab1azLtXov5qL936qaWAQONgWu8Q98YlStTfQ5cIgngMgXzhJ4MVlFfvPigIQWoGs2Gdyqk2EarPPXJ+cODCYUgVGDI45jLxjignF9CLllPP0BJzJu05eBDNhSzCmhU5iPlwd4eNs2XOpjHf58ir74\\\\\\\/MqCc+XRAhFVJNt8dJrlwZmDzmPRh25IeLAHeicKgkptODMVcFVu1vd7fuqpM8wGK9sx\\\\\\\/b\\\\\\\/gt+2VI+QdiIujZOeuSumR7iSE5FMkTUUb484YBGV7apv1jfSVMr3QyP4S5cfiLk3d53A1fO2CA3mgr1vZLOpaFi77cL0gT3cUwbcuJBojilg\\\\\\\/4XsFsecDHREearI0IngfzlyhTf96SfIYl\\\\\\\/DgVwKWE+qjCMBV\\\\\\\/S1r7j4GFjwB12XMNVK1ozt4xjLJnVk+L\\\\\\\/J0DLOPJc8H7PQmm4nsX+a4+7TCrsBrKJat7CLf4z4g4YyltIYuCA7TDtdzXUpVyjRbH19+g84WnL9GHVqsuomqeEKGbI7y7grUnuqEv80\\\\\\\/JSMbKRBiNjL709aOA7sWylc0eM7KfH5jNoNXJT0IclzSRyVTEZnDWgOYFl6A0LEK+2EgdzotontEmQcyxb5EnfODFIhBXlDyHs+sTBInK19mvkvcIN6578zaS14E8E+UNLCJGrfO2yE2bhD8Rmu6WWnzy9zZomrgtg4RaHWwBy5rsidOoTh5NqoKtkZuETsZxaPT8R7RIFrrSAmQ9fdUo+bdeb+Y+GvGFbiRgVZQL9eWs\\\\\\\/gl1CidgnhelnCnmko\\\\\\\/KKkncpyZWxe2C3zSRKPJUP\\\\\\\/enyy0L8sl9NPgNJsrfzzlFYTqgpQ9VotmkGTchKjbGCKBRPo6aXhL9UgzNCSrRYyYOJlaYe1BQHbmNoAlUt+1CdUDTTc1xVMdsn0aKGp4KKXMgUtbCFndSdsT3c2zZYM\\\\\\\/4pjRZ7O8JNQjtTKeNKyvLTLyU3o6um4GuOetd+JRVHWxoys0UGzFvQqUpTBK6ng4GVFfFOVeoToi67nBjFrHG+USQCJn2Rle220z9taseBFgVGlQ9oo\\\\\\\/+OLrZmGWirtdZ+MPO1lY34C1IbDRBCQ00Ko1CBE5WZt7kamthQUebzXTN7KyLKrdd2UYpv683vw2a7\\\\\\\/DYg4SmRKh7bQnVFMc9zAFKpqiWXT8B7pkaXCZSUz3cP\\\\\\\/S787D\\\\\\\/zBlMQ1XjmCz+ZolwBgR\\\\\\\/uzIACzRW3Nmgk9Iy5e1cg5miSfdVk3cd\\\\\\\/+UoZ6smBmpawfWms8\\\\\\\/wHiylidzedxMh4VKm3jqNczn\\\\\\\/+zAfaGDQIc6xcCLFvbcURJqTpiS8b77kkTSzWj1ZYIuZBiYvRVLCP3d+4H0Bijy10vcYGgj+AGQJbccEYTFJgXMmYLQMlyf6fH7wOaWIl61HETWJHoZDLrQNCwGLkdtaaDsjHU8iY5yrnhvlB2kyStTLdp7WPej5aaFi66Q5wqVM98MXW9XPDvazR\\\\\\\/H0REq1SCHuDvZrQT2LbfoXgdwYZrnBA\\\\\\\/DjWVEfgJDsVUgeKQWI1TrlHX9e+daK\\\\\\\/qA\\\\\\\/jMUuLwP8IJJmSbcl3TlKFolS4GLVb\\\\\\\/ya2U8+SAsFfch8wDCs8G\\\\\\\/QLYu4z8Z5XXionvxNxzvRfHYWPZPIGvAlRR7+ppCADxSwwJpQS4aY62ImRHbWdtzFZB\\\\\\\/qi5bhD0itm7mxrxvloP6vZbmhGGe8MJV5s6ywERyzSi23lhMQKkWLGm65urCoRo5d7S0mX0WWSQ35gBzzJ6L9T\\\\\\\/\\\\\\\/dKtLolOIPee2eJObAPQS0\\\\\\\/qO4rN8JisqNv3IPlnehMYSBbwZWfkyDPUdC4bqmS4O1N0\\\\\\\/\\\\\\\/OSyGaA6WdQ76H3P6z0\\\\\\\/bUgvCF8p4W0O3lXfi\\\\\\\/+OmrSYARM4dP\\\\\\\/Zq4hrDe9teVamERiLdYRpDWhkJ1jPDk1YcritE3udmtq8I0QdhcEJOy3eRj5txn4jQ2Nctpola8qHIv98swACRQrCOePlQ5IBbDzhCREVPPUoG7C88pUv\\\\\\\/8IulhBVTRRVuCNyPsA\\\\\\\/Tfc35JZr+0YGm8u8REwY9utbMPZ607\\\\\\\/lMaKw2BJZvmE1n4P32T7ldBxaCdT2l6WPAIayhN0G1p3sTzrHiK0\\\\\\\/5gmlUwD3FIcbFO5r\\\\\\\/aV\\\\\\\/odQAeZt1rhVgGaQswzRUqS4YufxAzk+ipFXza73d4mW2VHL1fPGHhwIB9AFx1UBdmd4\\\\\\\/y303yKRcLQDRG4X2M7QYnQzQeg3DFyth0YnQsPN90oPNIxTPD7bM0Tb79EkejrtgD0tmS4xooglBCQBUvCGt6tk4ZFf4aMi8JaBjeLbcz918O2\\\\\\\/iswb4320mvKE2fJy19q+c\\\\\\\/OuIx7Hr8HJA4iL7aoRLLSCxaNAk9xVpyaPqfCCLD4ui\\\\\\\/Mh5oDp3bF5mCQVlSwKl3NYa\\\\\\\/8jLoK6RHR4MB2wxGVLDx9IViEX1EF2NLAn7KXUNLELxQrOjpJ+jHtTD0sNQG6oAZr9ApztDexIALD9\\\\\\\/f6eT9E5Nh3LDzyyqkMqyEOOXkW2wQ54ZqHkRhcz1CRWd13YHTCQWCUIdUzWM\\\\\\\/C9FxTxxJd1C6nfDT0cq0OX99hMOJRDRRUx+9w4XEiBw7T8mHfgkQdsUU\\\\\\\/Lkb6io68UZNCUQVvGj8K85NcXgGAy3Y48KtPrOtqxsu8HqcXcghqy3Jy8cSbFBfUU2gUsu09Tu0jzr+Phbgnpb7GkuWWOOTmzWR1\\\\\\\/IRjHyUUdA0hteUqB+9DPioEN0yaOATNe9odOHUShn6Mk+hSYKEIRlYyN+LvqZFkf4NP\\\\\\\/0yVR8ODmlo\\\\\\\/zVZTsAmRJL\\\\\\\/hGw6+0oiA8e4S\\\\\\\/AcLcs2tq++EMf4U9MFVrGXhgiiCW\\\\\\\/rIsItAXpzjTV1T+7zh88G1\\\\\\\/4ZvlbqMcv2PMte8OHvoPTYiKGv\\\\\\\/G2W3iLKw4jmWGumzCl+GgHzTj6v97vyQ3Nvi\\\\\\\/X0ogyIh7YX7mqiz2k7hXOmPzpSb\\\\\\\/ZES2XrSbsY+MC72znP9SXW8hj6rAe71Pwc3R4DNc\\\\\\\/U+SYAeQpQNoIVJtqPi9V06+AHthAHwiXzuBd9V6ks0rk7SpMxDUgdqb+U+ae4BUQZx0X\\\\\\\/TWemDYkaMo1WF7ji6glo98oiWzr3eiWfOSR7\\\\\\\/fBHmW3qwfB4PCL3qpCaa+gCY4a8VnbCsL0is0Fl+e3hf6nmKtm4z9eNrYXMU884PNe7cpGulqYS6bgrP8BldSXH8pjXX0IHPcZT17QT3NXbBgScoEdJqS2AoG+mmXpFJQMdmFGRBz3FNaxz2lXncYTB7jCSykDCZdXDpJaQ3SwGh6YwYhZA6bga\\\\\\\/cbgslj9SPF+qrkqj90DKmGwJqWxyMwphReZOz8sLH0XMBMCklkwdYfwwzTOmstjoFugPeOXX+txbA7J+Y4JWiViSyQXwQ5leZjTJGLjr9u5CBF0nnBFT7K3UyDLRbvaUOT70K1bp80Ia29edGo4xPZ+OLmVCBySJ2tY0aL2GL1v\\\\\\\/N9nv6v6DJz+DrYV8EWguLrz9S5q+1RmfWV0t1EfmGep288ZBUvvHM1lNr\\\\\\\/l4CvzVZCmdGOo7R4ZdK8nxg1o9RybFhX1zdlxpudC2xuRKmf2JoTe5B0CoYP2G5Wroo7KaMixq2\\\\\\\/wKmHhaARIJckmayPup4fqf4VL2ISKgAND4m4PrMESmzURKFVled9GZpUO1cXE+JBa9ZCgOIp6epEMC\\\\\\\/+DvTC8TZ6pwB0L0HEpKWGsJ6GMKGF4uemqWTsT\\\\\\\/s6w+HFb++a75kwQ3a4DxmOLeAelVMmwcwyT3obMXagZB3D8m8HhfxuxQG4sKeyq97TuzHQxV4fO8zhdjnUo\\\\\\\/UlN4PMvrjCVDNf\\\\\\\/V9kneLDmSy7HkK6ya2knPbvVy\\\\\\\/duFRZsriZv6K\\\\\\\/jMTNk6+SVpt83YT+\\\\\\\/hHoRaApb5Rg8euMK76Eye3UuEj1Hj5Q3Pe+jei0ka\\\\\\\/3quTYPI0CiSHi128p2UnIxuFdyXzRlHAh1mw+ye6Q2kA48r2rOqUztdZf2h+hZObyMd9L26RCHjY0VV8cZLS9Fzxyu8z73PfygXcRhZuB6Q5qIcdNfe0Pi9VrfI4Nq6DNAMOny0wKwM\\\\\\\/j4t8ygUpZz0h0jUNvx0NVkNv5Afb7vAl+jDNtb\\\\\\\/tbdaMvSCMxG6nV7XFusbcvVMjsVOgGCjjKfq5dEtv1NKD95d7rJ4g4c5xGLxH8P+VBld3LDhsGo\\\\\\\/B2Zezp5N901LJcnijRbc4U6NH4jHaB+zcVrK\\\\\\\/dKt5xYrjJgirOtC7PFcADivX6w0TKUlVQX0nM5v\\\\\\\/33t4ra3RPSMI2TccbWeopljEnEnAuNIk\\\\\\\/dBgH0f9otdXQfJqW1zW\\\\\\\/9Vgm\\\\\\\/0dRlDy0UsxI6l7lLudtVnCJBpdjoWy5StbG23wmgj17cwvj0SwLv4J9pnf0LZhNofJIPgOLnm7RvCsR9kXvoYNyL3He+Tzu\\\\\\\/lI2Mtp6H0CkzxDnf+QQ43o0viOiGVL3ek19YzVfMkQi++walcR\\\\\\\/fdXXy6QVgPlxOHkCaU9Zmtgt\\\\\\\/vLYyALdHhvVqOmx3DgZhSPSd\\\\\\\/dpNK+fbGHqCp1\\\\\\\/S+lN4L\\\\\\\/6H\\\\\\\/tigdg8njlBUaU+57+cvrrot7B71Da\\\\\\\/mYYRsLBT4eoOBt6ePiaJM6st8wZy\\\\\\\/aboiJoJkTXWBPu68ZPntDqtvWMk\\\\\\\/VOoM81tekWMHDsNq9hj1Ksp2atDtQCkKwW9aVJfOGYXpcj12Chae+dchVW7CAEz4xZMo7HzKO0B\\\\\\\/oHNvkYSbkTplsiYw7wsrVg4L\\\\\\\/BXal8HtLCyIY76OTHMFwz7AvySsbmmWF+JanjtxcEkICbnML9HefQHKzg1o0iiUf5pPU79+oQxwTQylL33NZ9k7F2+ANTsF\\\\\\\/dMzKCmeU5tgZaf\\\\\\\/04cjfY3ZuzyjFCfKjZSNTYAAR8KiLrEpcmlH0s6VSOFAHPvC5IZopstFOgR5SNlECP3DByFlLIzf\\\\\\\/CVVfrsM+Rp8DVxMagZ9M3CWHE0J9u9ON9THYjHlF+koFrPr9MXUxMsb9Eka8ZuFXL7YR\\\\\\\/ggmRZ5bHwcLQ03kosF7Xm60hp8KtKMTuP8zDHQx8rSSl4MtRdCgzHUq0bL1AyHbEGDh+WYA8lBAQNot5EuFjHDx1lJo3eJqDLp+5xE9AdLjUYbQpVz4R30r6DOTp70iTWiDXFWrFmooq9jM4UOIKX1H9zI4KfYhgzNjYmJaV7MI\\\\\\\/xHzFEA63Sd1gqq2JIgGzChFW2y9X3kSBQn8f4SQCjABYLL5Fd6yllaSE0STh1Ij98oTxlJu3vsTTAgQiRdFbIgm6wpXZSb30iWHlcAzEYQyUf\\\\\\\/mDxPQbpKInwE4sCrjRDruuv6QJQElBdE\\\\\\\/n2MWinZrrWp8uotVqyY3PdBm2mtJUn+HYSj\\\\\\\/KPugCVlhzLV40IOxOq38WaHQSwRTj\\\\\\\/+Q6Se4GBucLc0SRGU6IYQHejlOZaLtfot+S3Llxi\\\\\\\/ztSby9jzFCxP8CDNnW2reiJ7ZGMg6A2ggIQ5pMfuv14NDmoIw5I6K5yOXXP3KxqXSYi9Ho73+bvCtj1BFKijasg+pJDJfueLVf5igbWNtKHrAN89n6Hg7V2udKu1Zj87Qm0CCOniqpLJtKuNTT1yv+ru5TjHzKdhTfz4esn0XQ+\\\\\\\/0oL0y1Po+0ieSazxR4LRrmG6wGYVljemJEi\\\\\\\/\\\\\\\/ZoGBpaNHz2mxrbFgGzs4bQOtc97NHnZg4kpGaqSGdl5\\\\\\\/7FTyf\\\\\\\/3GnkIRhQQQ7ad6fxTcuk328XthjeG4ObcVpTSNtlbXBYAHDPieqPyFFe\\\\\\\/9zCweSZbQz8noFtLOhBFMDLq2zfe7VoSyr+q63Sm5ayUYJ4owYPG+mI0Y\\\\\\\/rytoC3wR1ofnUU4WakfwWIRACvwmwGM6SDG9toqjmyHXShVQnbIIlm4wlLKzHexEw\\\\\\\/8jOKekIRawaVRs\\\\\\\/OdfSv9QvTQdcoMgClRNJivPivPwgap74qTyXKQ3UsPSBlrNJ7OCAd5z1OlY393tb2veCpC3xF1FYZIIXOPh22Nd9EhCaZzgVUsxBpHhdMxYJy+e0sPy3xQetRL2lDjWP2xEua9NJdz8Z\\\\\\\/jZk+qqMI3KkFENf7xpSiGRlekatadSr5iZyZ2wc4OYzB+R\\\\\\\/QvDP7VvXLaUZ+rn+EzoHHM5KhTWZM3y4booIcZ+KIa3rMjnbkhDZJosmD0bxAZUHU02WKeZEPwb1G41FJmU7IO94cwZ27771mPMUAMcrMN6o0Z8RizZle+K0FMvOvWn8YZbvgCEFHPedCuAXxd32dJMRDw36TFLubyOYYsHTePvNoh8HqVsGYTyEfrpmFj0PTMMwVdqJkZYxUaYc\\\\\\\/\\\\\\\/PDkULHk5W6dWL+Iw4WkEO1rHvN9Wk+t2MXsQuqlZ5c1nBctJRSqql6x\\\\\\\/tHxfAXUHPjuij44tl1ZTDn3URPiqKKBzf94lyfPGIUwWofKSjmgFxvIZxRbFVvKQy7b23RYbB4bq+PG8CgOdT75jgSuNV1UFx1fWsVtcfwNGTH32Wgu54zYXuHSjgyuCsBOfwxBbms9kodhj+VJzURwCSqTefRmqeP\\\\\\\/EjxmSsUXAHLyELGHlj7K\\\\\\\/CVjkX06khgUXTYF7qpCjmiC3lAkR0U6w7II9MEE3dcDt95t2SY6yFFTAmo0En2yK7GeLjfoQfONMqdQ6jjRPMRH3IS1AV87vz+1nhh66mSoXLR9BMJ1CFwROcrlZOnHV+A6eNiCck+Wf7d5xlsJp13oP7ck81BJdkrGiEZ6L0SzwdW2v+5yiAwdwe+HABEWav+fV+IWKgZsDCr4bQbaZS4dYkuDn0YQTV+mMh\\\\\\\/v2fILeYDF9ZPrsQfRU9hCXYKCzMuRcIeUj4n1XhF+Lt1widJPIzseCT\\\\\\\/+iYRf94Alj4vESoCH1BBVTFJN5cVxMMU5l8zWLYtBdOJvm0APK2QbcAAnUBhRIMLBjLQA4A6394j9sHyuimWpQrqSm80qxow7hjiu7sX2cCW9AZUuM3k\\\\\\\/u\\\\\\\/ugW4VoddqpTXQMg2qENANbiVMrbUhJAklUquvtVpfMbgHMZlTcjqAesmjYDBqKUqTlrkUfrtI100DHjySKcOILBlnVIXmpeL3fb6fQzjVcImf2rlIbo+iDnMkgaHJKQ+tDBFmVY2xQsjxxpZsSA1IAbCgU5YIHRrgUE8dDLwhLEjWn6Hj7mtJADZ1Tc+tlhX20+NBqBLfyruMCKWGQ+yInHJqqjlIltn8av2El6vpX8QaQ5RAgqhc\\\\\\\/+8CHZQdnZawPUt7JuL+H7y28l00tFEdcTimkbuX18+yuWM2fK4yb7beLkl3qX8iP7ScOkbvaKHr5fYkZ3t48l32m5p04dSHwwGcgKNCO0lhCx5hPdGW9PSZNkpQmygYaH3UsEBpBawYowd1MME4rysPKuaaRFSK+etH3Log641FP3tzyCeD0D0Ca5XOtRyULPVPv9JcU+YSSKd\\\\\\\/vZsw7LPt8TZRzc0cwgEvDXUN\\\\\\\/11Yhtz0MMxsfxW3Bx1wBMUyTLLUk+hbLC9jxFibRVKy2xAVlNuSY1HIm1+NFuTQ3hHABEfzjKGyrVRcPcXslk201ob8C3zs0hZH6thODxr1ef\\\\\\\/oFM9aSN9RfAyZgyp\\\\\\\/cncT2fhNBk2iluUaefeMe4mQ7cdh2qMYav4q1qrOGLi3gIJUqnmxBABibQf7jpH\\\\\\\/dSL9q7oE+IS57rzVvZwjZqgALz6eOJUgbnjJQZSwoZgAyyxWhASyHUERqniyee9lrpjwXTZj1xxSloJeA2uZpDZnICepEq+1zsxBFUU9Qm1HeKueEGYVIpXa8ejR8lO8+AvIjyL2GBoFXiQZpEQJL62HYeDdVKjUsJ\\\\\\\/mMKOAtb9F9QIJO1qg5YR0novpK9ij1WvjVdqI9hlqk15YIkuVtleTLhDL14xEIdAz\\\\\\\/b3F45aLnhWQCaUnpBmn22NbA2YYzMNF1noRUVkAB2TddXx3t2AczdNA1vnVzfrTOWr7QIufRrPCBsT+9nRZJDF7QfXLPey7hIuDqvn0V8999qBnSN0OVzabgvMzmpdmA3AECVvUK1dPiudVSD6XNaw21cRCMqNdGAyseZw1hRFaIrY82yIIVCtk2LiKZw0GIt5G7YPFZYCR5WW2hXmN4OuoUbUQSW9mNai1wYjoAQUjSTg7ZERGjqE1wjDMdvLtGwqMFwFZ1bDD1MiREY6zttnTFYyWn8G1OW\\\\\\\/j87NQFeBv2cCQOKdenZnOQkOgSJMswzexW0JWhMPbS+QMPNZpbvh16L21fynw9oFnX2m85P1xzPestEn8t8XGkP\\\\\\\/BpITqrxS4yPWPP34yKCLdo3zd9SMKZ+7KIa9vjT8hu5rDFTG03PNj1ylZ6zZTn90gHj0wR8GY8nuCqgj6tch5KR91Zzd+aafRInydIQBGK0fi3l5z9dyTOECGGBaFNbVTfjoVtkILkWeH10Ww2eFDLwQ3tLSz1UJ7Axfi3TWe6kIEUDGWyZi+ut0AQAE5WCCXJpE6NqhMkEA+grOwhzTSp+OQF1EvGKt9VPbvZ8aPz9F4LC1bm7Tz4d+EzqYOkE5FaN7680pETpLOSUUaLauumvILz7rctyL6\\\\\\\/AwBF5IzMIpU7UG9n8kMKJKy3U35pff\\\\\\\/Y8FmWHsC\\\\\\\/FMvx4vlmT0hwS2U1KbRhcXCzU9POHFFENZLU5zVFgEgocZm35RZjDtXt4fYva1Kb1ecgp1XJDyrPnY\\\\\\\/TsgqY\\\\\\\/vtPBGAtuMDdLJXcwO\\\\\\\/HzrbfhEGe+ASEfHwWM3G9uI7tGZUGaycNwTnAmW\\\\\\\/jXuvoTjrRoUoXA0fHuoqm8Pr1I5pHycfve0WMXocy9MCtJuYsRKPfsGl\\\\\\\/O4LPK\\\\\\\/LSpstQLar1nVVSRc8G3TZOFu1KMmfVipnxUW4Z5uJVne4Z6sW5vEeASecNfRVMdsZ1+iJhE93irNzkyGmOPxTtVVNbyVfV2oQ6LHlRflUQ86K9S+swbzefMG4amiaUgDVXCK9JbYS5mGIyJrBdFBeI1Tv6W+A58WuQUIafYfJg5L02yHNkMygl3ueLzJhFymzBzQzHtV\\\\\\\/6iJ03CP3QjdahUm6HEhPmE5eslmHk+W8dPMiCyHwou0kAoXok+zEBI8higDjxxolUE5E0MuxLkbFcxZ\\\\\\\/cTfldCEXB5wkAzvg7A036AnvwolRBg8Dd9rgtR6GacmRVV16G9x7q3rT7X26yrTA\\\\\\\/JQDTyVssju6Jt5MjvAgBqsqCLd0pLzt06vDy8SR9ayBXHazEAT3YUTn3Mz4CqYq7M0fRGYdwa\\\\\\\/9IeY2nB6zSsucgtM8EkFOkCahP\\\\\\\/t6oOUzMme+qsFarh5+RTKW9GBgJcuByqzDc6c6wEa3+0yrNTXZ5N67Ptf7PSxF3XpVQHWieH9y2aT3fTSNeC\\\\\\\/kgGHx4yu3KLOkHYld3Pm9Ka4\\\\\\\/femYb\\\\\\\/IKiKqDAEnY5PTrP0ElNESWBPgJTGIit8BXe0S4Xxfu0vzN\\\\\\\/afjp+md47BkWlsbWrPX3LukQS6MSMgxtvIOEwdM+33GLNGTgN3mUvnVVp9yRJgNNmQpIUTuPxEEzMYCUeMeLVELG0GJz4v4HQZoDEvLwnfKVY13ECGOWlwBLMz84SwEo076kn0QWyXF+qgof+qfWNAl1DJkwrgfef9duCOIpTvYm1ybuwJLuTgVt5FOm9xzVuaNMBdXxcwrIZHhbkbYFQZp7s5Extm2czQ0n0\\\\\\\/94dy9+dlLQ4mY4Fa2OYYWSUeSlam66kt6uqIX0Df2+OWczUYeM+\\\\\\\/EFsAqX4RDPcDW1+g2Bx0FeKh9HpUu0tBlQp90hCENMldO0MS7gIfqYiNhI0pC4n7akk\\\\\\\/x0FsutRyk1qCBcs15XoEaG62VXc0rvmm2f0ymDaNcbdDfkdEjeUk1J3IzyOoXeMCeQVrUZZq\\\\\\\/ZOv5EkJEK7KS17haHlR6yDNoNkLP09tkuA9z12LwwBZIq4itq4jKcGRA+Yrf6W0B3OfMQNLKgMsQ1mpDL+ZDgxLoAQjapqIxgIGBdBrRdn583jjqSVOzhDrzch1zUGyoYfcBz\\\\\\\/lBRV5undqQjPDGUC1Uxh+rCWBMqfb5DLMwpguEV28wFx\\\\\\\/Nvg3OPAt\\\\\\\/leG7C01ePVSxemIFQwz4bhIubIkSjSV6xsLocPNv+gqDuD5XRHr0HPmMEl7rU5EXN2v06kMkQ7RCUcTVs3N7YZJov0SPvMlRqaB1qsW7kCKzFQ0gJqmBBbbwV2rP4Jyf+qm4yal9UsDh4vjy4tXYavQ58YyUY5igo+ubxY1RnrvanbSFpqQrfffBeotU40ATi1wDii\\\\\\\/dHQvolFxYXvgaJfwn3S5q8CghLBdAIoPCBAPIHh8idVkF6SCYkPoaO7tYDkz6EztTciyOImThlIdqddERbbrmMnKeoR9ZYnNL6CVsR7kPgEuYgHhiJucq1gih4czIw4mrFfMs3CyOFCUAToayM3Q+6jxg8bTNUvtF4fG+kuGygw5KckJD+oN1A0nKGD4corwl+6P9DnKROw2ax96EKiBgJ6GO8C\\\\\\\/b0PkRCMg1qVBVtEDuRZOvsQxpNjUUJ8vbLOMdIiCQQlYySXb8U7gnaAWbky8dfPZaFaVHU8aiGTuLU7XC1\\\\\\\/9knxXnKa+crf4bZ7rsdl+X6MXkV8BPXw4DZ33vebR0a+I3l\\\\\\\/kDA9fNOSwJdte7FE9kVUXPLkgaERge5DDpvi1b5xqRJC\\\\\\\/jMwBe4t4DQfxkg6hiWofTWd5mm0yL0QFwwDUxoSV8yNznkDIzXCvvbhiKYoxWkE8P+sjmkgaNcOwYv\\\\\\\/mN2ifw4owKTHEK3mMyG1BMFGRgwImwfI67yIuhyrtF5AEzJmjUxyNAxO0hDYCKn0yamCXdXXWNYAQhVGTpmjiqXFR5XiU6RKBnDZbhzf8G\\\\\\\/mJ4fFdcs3f7InChAW\\\\\\\/uwYUSzWYHaZltT0ScYREerHTsZKIQXR5EC7ehsmfM\\\\\\\/7DZT8RzMDlfgcU+Br9LTRl4Mn3mfNcJYA9J3LxG5fZp5iK60ubJ08UebANPSFI4vYOfmns0lv02w7naQBaYCYvNGC1CnJj1qb6TTCwAVNXR7QfQwk1s5VRgdUipozVYFs8BvgW1DhRRsxAPxUPrLsSdaPKbJjD9icuhWd41fwChBqWw7azl7L3ePz2ZjAlRpzUMLBZXdxEFIZL7BJZUhfuUIl5ku90cklYZ7XpmfXVnebUgFot0FL7wtYwiBSSNNLBDrN8yvnBO0d3SCqVF+PXCTqSdX9EEH5a\\\\\\\/jAsa0dkI1lrd4b6stOrkdFHcS02NnPDPmg6UQMAwhdNxCuv1vwvSr5sxFL3sFIVUxuozUUyO7y3es+MVOw4yRb06n0Xm373FKc74ee3SZ4HuDL93flyEKNdFCCHgZ+rnVZRsNa2lEjHYAWdc+f3myCLf69IFr2++ym4KGe1Gn1POSfe3vJGH3YsQCc\\\\\\\/mTDvJiIDPvjxz3f4IFmjMPbXlTBZ6H5vDmZiIULk57o5MiI4Tk4kl1Tul5DxjMsSCffwZi5y4mhreu593qugNW2OP+IdlHHaebSyp9q3GVpuFPZhrMTf29K\\\\\\\/AWsl9gbLmhPHeaCJvicgJ22ubGgfa3SUFMaS+NXS9Oa8wfZOwnUQdEOm\\\\\\\/jGwMdSw1mQdtgUPvmBjc66KiIS+5dWEObCzhHBz\\\\\\\/UMp3GkXC6UeTgQM4xfsxqAoLAmsAPBUSNHj8\\\\\\\/XRj6NYgx5VYBgM3GDLZVONZc3zaISvLVuawTIO17Pa\\\\\\\/SBUiTximl9Zet5RMsdQUVRAfcHliuy7DkyGlM\\\\\\\/s9wlWLoStUZH4JLhTw2lELF\\\\\\\/1lDr6b5nJTKQPblqdpH77Cd6+EDQh4GocmDG19RznmFyTGskIbfM10iL75I6rcCrBqSKgSZlr4v8E43zB1UFDg6CMwEF2czfn5+VexzmCfSfm2jzTjtnIGcs0rQ2S2gxooCjdyivTXVBDZ7\\\\\\\/sa0qnSXwDVT5mzc\\\\\\\/C7a33MEIIcxFkh6YE1rkoEjizGO8XioCpheisrdDQFIq8uZbknBeLprXFz6fSaAvnzXRSgl\\\\\\\/LvVYyeinNYfLS46IXVfw4V6kpijiE3jM5mBBmnEDv2XuEA2puiG8RLpZjhiEY6gd6pSxKKYQ4ER0f\\\\\\\/tP2yCG7gB6b0lsLV29Zzg2Q0xGnVJcW9RHszCBuzGBAPS5ei4sc\\\\\\\/t8NsFmDIZSwmt2\\\\\\\/guru\\\\\\\/PKEUoOXoh0MMD9zxr7HbC+XLxjayprt6UuBGpbOvnvAoW58ziPuOW3EkmsDHVEWlR8Qy0BGfcK7ihjn62uTItiDqLEH9va8PqfsVrCRFKsZYsVusR2RZ\\\\\\\/6b7ZCyrh5Lhogqa93iPdUtQt1kVg2FPnUk3p1PZ7XMrMxPGpRjHIj0Dg\\\\\\\/cqFBNFSggAqDaPTTJql5x8eypeBEVu0q2PwXFdLHwPBat0PDvYwWRGgnAkvXjx5GocOfPytGyh1xPQtHa8A+051j+KRMRn5cGypRfa8cKzTc98YbmIW72z02xu3aKqhLDMNCzCEfC0y8NPyuoxV8Wt8HJ53SF1fxFO\\\\\\\/+bwaEmKYhfNL30K9FviJsOj8oR6ChuCI3aeg6WAoY\\\\\\\/AKz6snDI+HFldMqz9bR9PNfMdwenkzghf3db3RXSxzQhjrNXC76Jd3dI147IELlvXreZm1n6uiyFRWWZ2syerZEWLTG5DYb8+CUY3ueu1ljkGc+qQeNal94D8e2E1ePsnZccRiyxm0KhoOcZH\\\\\\\/t5n1X\\\\\\\/Bwm3jc+1RG0hgAsQARTrZoiZhhHVYinvaws2vIp+bsE0L9JIaqYOS8uehEOwEVRQm5U5Xhhk8CM8ATG4vZO\\\\\\\/Zx3Lycq0qtLnBS53B1+Rg6NuQaEy6QVUHTDj3\\\\\\\/VwokwTQ0cJUyXZTtMlutqZFOdLUrVprOS+HYFOQrQfuy0SaZz8KULoaArALfnBEMp35dr0t62xkj4SqSIK9pwN8LMob6dNyfoHtecKJcDCe\\\\\\\/mnBWLOci8w4Yodx6WVWWE7H1KUfEC0xEy3C7yOiSLbPeSFzsh57TLtTcAV1znAA1pbRsIUn5ysuqMkbJVHrmCUS\\\\\\\/tRoRfdOkz0OgzlBufRbsS86eQGgSCqtQX4yekfKZVLmq4+lqFnQq\\\\\\\/VOD6v7qmF6H7k\\\\\\\/JyQgLf7Emo6Tyv7QgeCOStODL+ExQfIb6VMHUv+Svrb4EATrJyz27YXNpzqwd05vTYQYjN6U\\\\\\\/OYbGq4i3hH\\\\\\\/yeydUqB82VWPc8th1+nfg2Uv9GUdICHMZxNmtwwwyeU2qbsSaMhDBXqwEZpVr4RK+Z\\\\\\\/AQqK8OwuaGFs0xPiTR40V2wJgk3P8yCVXN6CgMm+WHIp3aYu84bOYNx5cJvKj38Y9spHam+aGNwd0Yc0raOQlDkr5Q671qUPTghjGAbD7NqTbtHesaKUg2J8tIGrrb1Y7lLt6SFoWwggyYJfTT9bhe\\\\\\\/\\\\\\\/JHdmAtU6u6oZqTodVuynj4KZSEhR4VZ9XiLBYkYeHS8r5IjwusM7Px9\\\\\\\/q8DX5tJpfKmpGgAfWYCxPHnDhuvX3PIRxP8tyyKWaCIDRXBeQAuCAVK3sNa3FKirZPz3U5A8LWnwEDKesUa0MChTqBbOAZFUJQxoFHZdV30IQaMpv1CWgNgB\\\\\\\/SHpbitMetp9Co6FJYbXK\\\\\\\/y6X+S6LEOlZbZDRjkbYyrH1hVgBCcTnBiM\\\\\\\/SY23AL1\\\\\\\/OOqDio51iVNIz5a7VDt627nieHDzli47\\\\\\\/4cspn5qKho9LHf6sS8k2r4fdbRyx5fneVrThlc2rcxBB4mFTxUMDaZWNfeJ1P51AkKMdRpR7mTYy\\\\\\\/9AeDg5X0BEZ+j1gg3r5FMtMdM\\\\\\\/DOvz9xjtKOP\\\\\\\/S6xMw+kGiYfHSW7YX+OkFgqVALwOUL91thkvH1lMGq6oQxDArXk5HXe0d2BUqAhH9qtd+jgBzK5UgXOtGTqtHuWpJsn9KFBPK8XPBf4LKjLlggOq\\\\\\\/9a9lfrJc06Cdve51BpG6Yu7LFlOh9DXJctD2+GaCxwzUBlUujzQzd37QLg3wS6oP\\\\\\\/ZrUI9imW03UBOb1BZUVPXV\\\\\\\/MmvWlRAt3cIDZod7MsX0fMUOiYDukmKLbL5tcaiKZMRzkricZvFVoCfSV\\\\\\\/tNORRIu+M6+9oUFdw0H88gPOW783Q+3JI\\\\\\\/itZeKUk2BNPVaHU8aZTXmw4e89i2lfWyMNw3XIGa6k9lSaMvDqsIIgxw74tcK4ksswV43opjp\\\\\\\/Wy+hppxzQZQWKq93Vl7zNF+eXvNme7EBOhAWKNHDU\\\\\\\/OQdkg51QntfYJFd9TDfQiLv\\\\\\\/1DkvfkEiDcVpGfgxK0SRp0jJgxBjgFiCqEc5CXI852DLbCgnPP6p2MDPr\\\\\\\/sxVQZo+6P2IP93kaMjwnVi9vfzC\\\\\\\/36qcKPX1XOMS+oyF2q154P3U0Xjjgil0jkaY35JtSmflKLsuCKLF3iDVKI9HTexDMo5KjUwC6+cQIIbgUTWJxZipMvHh\\\\\\\/eitpflpYavyH6VFtVlQD9X8bhJfVmIPaxvlv1N8hO24JQ3wlaGVtpmC7SpBx3VArud9so2tIamqEFA1hpoHJiHUODgpkKEboOc\\\\\\\/Q1biwsRFlNIYlsgoPMUpub0qlCMVFE9jt3dkvHHo7yIuPwJStrPRuP7CrQY\\\\\\\/3ZzzfPrUAqgsgnxts0mwkkf1BLDuRoMqzAyywMSnlOmyB1LpVrqklf7Ty0++P77EVf3hy09Nbl6vwTindRzUiBAfgKkKikl3k\\\\\\\/iOvGyIke\\\\\\\/M4kA5C4JwHaRvUuEXfXxvh2YUrJuFIl7H3ApnmRbDOCqtbO9D\\\\\\\/SJRgO8BKGuow92gYmsjV6EkIyjOjr9x0KVBRfZUF5htK6YLh4suoQgPwYcfv2A7GsUDHYP2LLhh5aFIDmX9H9MepJeYmzmyuvZ70p23WRFgjM4TnVDuMVsyCGTrYRnSe\\\\\\\/uidW2WSf9qe9N8x+dsJ3pNyTAytXyx96EN1Q7aHxF0uNM2R+qWhb4eKi5vL\\\\\\\/ukB1W11MGYurnSNZ2K1+EOjRKGWSpHOZx2QVkzhcOrG0hWcHwssjc9cRN2yyYsbK34pedlFGhKT2HXCWkvthJfm0hbXjPLEN5QOIzIEs8+TAuoDyCrI0OjiMts3bPbvolbboV1wypcy05748ej7fCF3Wo\\\\\\\/D4PeOpHWLng1TbMZXU\\\\\\\/JHJQjwiliYlAPCFmSx9U478tEBXeysPFAuZttjtvUU3oMZx0TNkpCQvwyPz4StdL7c+Eb9MZ12rcvL4XxD9AKMp4mz5x+HKgGJElwzheG6gmBHhmbxtWM0BrY+RfDJGah8jwETb05lEhYYs7Lz3Bi6\\\\\\\/cHrXu6BX50ybp9wMvJr\\\\\\\/2KNR5mRKQNAUgpGQ9q9IHGNZ6xyWfZZNrJ5zztUhTV5bDS23sAL1k2QiPmxLjzMm9cCEyNKRTmiuGofTH3wDZcnGJNZUKJ98r8DLYvBMoHCjBKb4pcL3KjGX3sh5GNDghnIA\\\\\\\/EG7FgtU56RqW73kGYWZ9OGFzofTGBfIc80ZIXdHLKKGzlVsQOkHA3R3pize6xK+YOqAH5gh533wjQk2aGp78FHoyXbdstE3WwfaSTU1KElCOLKPBLlMuyMx+1yIMOIgUfvcisGS50Ai5E1L1u99kJM3RHK+M4\\\\\\\/yGsTMrraV\\\\\\\/Y1ebWhV6+oEEJIpe7FRPkkvTnVc6uPc9IkdCIgO2QOrPvV9nChFUxA5Aw8IVnMaO5SK2ImMzki3V3N\\\\\\\/Cm\\\\\\\/lfk6aikYH\\\\\\\/OKgafuIZJYJoSDObp4vvJPPbksZagIAJEQ5ti8AIy10pCRpMTPninB6HPXs\\\\\\\/FhWlw7upXA4kCv+k7jefWHuzBG6ArPLDR+OKITLDB3eoj3b0EGP4Hjt4Me9nvVZ4dO9yFXcu9szvhSsXTQgNMbawoHpEkdBPSTgaAbBtPLA2y9dDzoX3GPzU9w4Wd6Zy6E5S5t5FeApWCYg+ulsT+55u9rAxVY\\\\\\\/Tr46WJD34\\\\\\\/B0UTnLxeJTyNmNTnuofhd4YAYVzWz2WFVOgt5KwMHj3F\\\\\\\/MuPDiANn52xnkBGmo7DxCAMJnbQlmHLbig2I5WVBH09haOkUXE1SENNdJzcPTOiEiJoKpJgdAWWdFBG2Dm1tsLOIW++fAEVkK06UEVcrzaiUxxKb4yTYCgTbXLdg8M7BtkJhbZyM9JGd7igZsZH7kl5cscnAHztkNAAXix+n5EJYAqEsmB14TMdzGVmGcZNS1ddwSowN9GoJiRVKf+Ra6r2TuPxhtduI8Qa3XdDf6b+9zXXJjB+eAmlEq5AesnihbtjU2hueI02hzjaFcOpUgiyiF3bJLF+G6fK0fiFibkrfkAJlE47O4FIzjDuE0S7KBWpB1c46MeZ+Jj5AdFJpEEV3Hyj18o7qlmz0M0qD5vR2Vel6QMEyrhZ70Xw9vyhR9qJA1zMf+yVRTKyAdHhkQJCk5jg2ady3cFbqAz7VNmh1oDC8BRTwaFFuR+d81iEE5TQ+RbT6IQzVo5ZCDeK6tX61KN5KAN+k+c85WJyf5CCu1qkAr3dFCA8vsME4uuSCyCP0u1Ks2C5jzVnRpEtPhJQC+q+I419AN7JRtb2GXcuj9K9WKU10rOuUfpR0kaSfRF98cLcvAJSI9TAGkWMU6X0R7v5xsf2hzJX1uUsrYpsOCPrtmEs6Wzq11zBvWqMZ0lV3IlarGa+D1ccuMFCtwNs+9ucZIEbaedb8TwnRQjrTQQILpD27IT89QNAPOQ5Qfecwyae1su4vkN1\\\\\\\/iFQ8tcc\\\\\\\/gvUn4amnMQrhAtWHVQxjkBThLovzse8wyz0DPKePQZxYbFarqycCROS+CSqX+EBMFgRMYriSiXeDgi+zS\\\\\\\/yEoSTNW8hfvKinGKsnxVhyYx1F8acwhTMBLlwuCCyFV74ooL\\\\\\\/Q8ZgE7CzSMv1LyclWi+XKdBb\\\\\\\/ae13u\\\\\\\/BuiIDrDnL4bw+SeT0PgB80l9HAi0AA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022NTWq099gh2fNGYcvdn0p+A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022884cd83ef37834a606a4348461e6b94600e6a98c4f4b30fd10d5823f509b0b35\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eWReoHT85aAAA9cCT3NhVBuSQNCwutuxfiJ6jx\\\\\\\/jZ8Xm0eijbn27YJBqFtfxy8EHmMLLLYMGH+KWJv7MOHqY2dkKRiQJsri5bvj2j+JaUtsIFhotB6\\\\\\\/EbLyOiaWYxagHe73x5YYkVK63OAeQ9VgFXF7W5OOGsOUJTr1ZBZKVGRNW37Zi8OORnkuuzRVKAkXoCjHxAjIiK3GViV4leFvnX73Rfhlrzr7I\\\\\\\/+Y2gh37MTstKiBPxPCPBB3w8KXX6OdNfEkjC3dT9VAVvxu21ezC4ojtwAJPt2+wXa9h0Psf\\\\\\\/S6e2My4ji5x0zZB8WUUgzoTPJGrAsfuocsc29zX\\\\\\\/fsf8hICRWFmyVDKOp5PK\\\\\\\/Pmr7K4v8I+BuYdREio9aquLtPBTDWVnZxKXyOfCXA6fPDidd37U4wgUbjw6r+hITMqPgCZHluaznD8xwuff4bD1S5qJMcUZ5gg\\\\\\\/q86+unDODpTumsQlCn9eaE5i5nYFxHb3oXgiRjDTqm5bCOiqfzRhM3OQe8i5jHydfTtmfOqu0d0Q31yazeDLetLcxt9GrKXYq8dPZ04zY56NBGWqPR+5JHcXA2mQyNKNWAJWGS+7BBeR7xeNKi1DR0co6As0VRrsxqRTtHw2gvfdrqLrqsAWNwJ\\\\\\\/aOeFilG1jXPbcICYnUkDkwwzMRdRgiRTz2AM6q\\\\\\\/O9Ahos0G3MKovkI3yJ0Vb0JpIXhRM7VxKTxJCtjuLn0wWdPkq8Ydj5a\\\\\\\/OHYRxDFxXCuNPdKbz+X8KwK1TCDY+ZWFK7q7jRNNb\\\\\\\/6ZTKxMffMd97X3IWJasQTYcU4+Dl\\\\\\\/ib2dr\\\\\\\/GY\\\\\\\/8BFeAFk7UUahG50nQce3WeLH0bdCXJBb9Uz3eJKwWsK51eF\\\\\\\/FYfuWockpYgOE2YwPsgkXngyRzPFDDGsB8PgMKWW1MSAFJICvoPDPH7p2L0Na+xdfispajLBKYJjezeeJmV7csegNyFXHtKTDVj\\\\\\\/LRII8k3qp3oztaUTKxGo+vZ8OrW0RifWFuEMYoHVNzN8VSS1WI4e0bpYE8QHzcKJfxZxc0Lsx5Cac2Qv93TO2lGVfiK9JY5Qdp9abWDqEL+0iuIsPPKs8mPT4D+l+knIqigQ7QZxz7zGMmJ53h512ltAm3aYLi6Ty2yTmlSpB3PzAP3Xj9pjr8ZrzbMYj+OT2BWHTRHCUFtuSZmgBiBvnIlOUZVORDmHE5kJ+i9ZDOfkVgLm9eRmi6h7Rv9uVyR3CWQcVI8G7smvkYHvEJgL6IYQxLc7ugeVutwByddReDhnyzMf2NSswDQUFzDWz8XpVwJD+\\\\\\\/NJ\\\\\\\/40tiFLlA6n9TzlP8JkXM6v4EqOhZcgkD9d3VjZSnSrlU1JRsKXrIJjf0vLrJSP1lQmgOxJ7wM6qHhei4ggAF5K4SiSF00GGNx0Kmg9Jgs3pEdL14pzKilUGu\\\\\\\/nIW+5ae3U1jD9DAClYnIAD6ryreKfwbkrGHauNC8FAxaIBJ\\\\\\\/713RxOzWmRMPUnRE+ACfgqgsk3LF+RTriFR+2JSi3zti3bvzg3uZXRJaDh7L\\\\\\\/jwPJjUlBW4UM+M0CrKiyHO2ezP+9RcGwdEvoz0cOcZoF5bLBbZRY70vbHSg4qGWz5NpJXtJPM4DMitI8NaiAUYQuIw4p+arfktELx9H8caBWuQdSd4kN4jeGHSqeTQB7zhUXn\\\\\\\/rT+q5RpbOK78692AqkAZidmsJ72TtA\\\\\\\/QSrMrHNgNWGZzYELh9Z8z+c7By67NMnDGlMopsc5Kzvtd3mJ1AmIQbI5mchBzGsHqNBzi9WwwX7vqvPxXH5IfidcrjVwL5sv6JH6ZBNueiKvz\\\\\\\/wtXPFaBnfG4A2DW9\\\\\\\/uCXqPLZFRBr0CC1ePexJPfGQf38n+nYT2NrZM2Qle13yodP\\\\\\\/pZ7\\\\\\\/c5b6DOd4eCtq27n+9fR2cv3v440z87NV+JKZvX4lQnUh3D6i\\\\\\\/3ZYNcIcwVyYSuwpttWH1zqXwB+jngftk20LHVtlRijuW8cOejS4rVkmKspFMYdIA\\\\\\\/ZlDmlVJq2XcbuSpFeFiDY9FAw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022o2VNalFgTAorIHWWQedFMQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227f61f71f55347c085d2e4d1c223d4f5d8d803b9664703100865a49f2dd17455c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491787 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/1/135" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-QzCznKDzxIJly_ZfBJqpC77pxK6j2tDqEqAdX_n1U0g" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/1/135" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zhk9r00cYPgR6KrpPwLn6qHPb4Ly7h8ycpFKffkhaNY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/1/125" title="Genotype Imputation in Winter Wheat Using First-Generation Haplotype Map SNPs Improves Genome-Wide Association Mapping and Genomic Prediction of Traits" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/1/145" title="Domain Requirements and Genetic Interactions of the Mud1 Subunit of the Saccharomyces cerevisiae U1 snRNP" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491787" id="node491787" data-pisa="ggg;9/1/135" data-pisa-master="ggg;g3.118.200778" data-apath="/ggg/9/1/135.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">CRISPR/Cas9 Methodology for the Generation of Knockout Deletions in <em>Caenorhabditis elegans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3587-6200" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vinci Au</span>, <span class="highwire-citation-author" data-delta="1">Erica Li-Leger</span>, <span class="highwire-citation-author" data-delta="2">Greta Raymant</span>, <span class="highwire-citation-author" data-delta="3">Stephane Flibotte</span>, <span class="highwire-citation-author" data-delta="4">George Chen</span>, <span class="highwire-citation-author" data-delta="5">Kiana Martin</span>, <span class="highwire-citation-author" data-delta="6">Lisa Fernando</span>, <span class="highwire-citation-author" data-delta="7">Claudia Doell</span>, <span class="highwire-citation-author" data-delta="8">Federico I. Rosell</span>, <span class="highwire-citation-author" data-delta="9">Su Wang</span>, <span class="highwire-citation-author" data-delta="10">Mark L. Edgley</span>, <span class="highwire-citation-author" data-delta="11">Ann E. Rougvie</span>, <span class="highwire-citation-author" data-delta="12">Harald Hutter</span> and <span class="highwire-citation-author" data-delta="13">Donald G. Moerman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">135-144;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200778 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491787" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vinci Au </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vinci%2BAu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Au%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVinci%2BAu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3587-6200" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vinci Au</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Erica Li-Leger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erica%2BLi-Leger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li-Leger%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErica%2BLi-Leger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Greta Raymant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Greta%2BRaymant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raymant%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGreta%2BRaymant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Stephane Flibotte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephane%2BFlibotte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Flibotte%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephane%2BFlibotte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">George Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=George%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeorge%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kiana Martin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kiana%2BMartin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martin%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKiana%2BMartin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Lisa Fernando </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lisa%2BFernando%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernando%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALisa%2BFernando%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Claudia Doell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claudia%2BDoell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doell%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaudia%2BDoell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Federico I. Rosell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Molecular Biology, University of British Columbia, Vancouver, British Columbia, Canada, V6T 1Z4</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Federico%2BI.%2BRosell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rosell%20FI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFederico%2BI.%2BRosell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Su Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Su%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASu%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Mark L. Edgley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BL.%2BEdgley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edgley%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BL.%2BEdgley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Ann E. Rougvie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics, Cell Biology, and Development, University of Minnesota, Minneapolis, MN 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ann%2BE.%2BRougvie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rougvie%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnn%2BE.%2BRougvie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Harald Hutter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biological Sciences, Simon Fraser University, Burnaby, British Columbia, Canada, V5A 1S6</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harald%2BHutter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hutter%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarald%2BHutter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Donald G. Moerman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Donald%2BG.%2BMoerman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moerman%20DG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADonald%2BG.%2BMoerman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:moerman@zoology.ubc.ca" class="" data-icon-position="" data-hide-link-title="0">moerman@zoology.ubc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/1/135" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491787" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491787/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/1/135.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491787" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491787/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/1/135.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:491787" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:491787/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/1/135.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491787" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491787/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/1/135.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491787" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491787/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The <em>Caenorhabditis elegans</em> Gene Knockout Consortium is tasked with obtaining null mutations in each of the more than 20,000 open reading frames (ORFs) of this organism. To date, approximately 15,000 ORFs have associated putative null alleles. As there has been substantial success in using CRISPR/Cas9 in <em>C. elegans</em>, this appears to be the most promising technique to complete the task. To enhance the efficiency of using CRISPR/Cas9 to generate gene deletions in <em>C. elegans</em> we provide a web-based interface to access our database of guide RNAs (<a href="http://genome.sfu.ca/crispr">http://genome.sfu.ca/crispr</a>). When coupled with previously developed selection vectors, optimization for homology arm length, and the use of purified Cas9 protein, we demonstrate a robust and effective protocol for generating deletions for this large-scale project. Debate and speculation in the larger scientific community concerning off-target effects due to non-specific Cas9 cutting has prompted us to investigate through whole genome sequencing the occurrence of single nucleotide variants and indels accompanying targeted deletions. We did not detect any off-site variants above the natural spontaneous mutation rate and therefore conclude that this modified protocol does not generate off-target events to any significant degree in <em>C. elegans</em>. We did, however, observe a number of non-specific alterations at the target site itself following the Cas9-induced double-strand break and offer a protocol for best practice quality control for such events.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/c-elegans" class="hw-term hw-article-keyword hw-article-keyword-c-elegans" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">C</em><em xmlns:default="http://www.w3.org/1999/xhtml">. elegans</em></a></li><li class="kwd"><a href="/keyword/crisprcas9" class="hw-term hw-article-keyword hw-article-keyword-crispr-cas9" rel="nofollow">CRISPR/Cas9</a></li><li class="kwd"><a href="/keyword/homology-dependent-repair" class="hw-term hw-article-keyword hw-article-keyword-homology-dependent-repair" rel="nofollow">homology dependent repair</a></li><li class="kwd"><a href="/keyword/mutagenesis" class="hw-term hw-article-keyword hw-article-keyword-mutagenesis" rel="nofollow">mutagenesis</a></li></ul><p id="p-4">CRISPR/Cas9 is the current technology of choice for genome editing (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sander and Joung 2014</a>). This is due to its versatility, as it is an RNA-guided system where a 20-base guide RNA (crRNA) directs a Cas9 nuclease (from <em>Streptococcus pyogenes</em>) to the target sequence, providing high specificity and minimal off-target site effects. The endonuclease makes a double-strand cut at the target site (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gasiunas <em>et al.</em> 2012</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jinek <em>et al.</em> 2012</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jiang <em>et al.</em> 2013</a>), which can then be repaired through Non-Homologous End Joining (NHEJ) or Homology-Directed Repair (HDR). CRISPR/Cas9 technology was first adapted for <em>C</em><em>. elegans</em> in 2013 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cho <em>et al.</em> 2013</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Friedland <em>et al.</em> 2013</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chiu <em>et al.</em> 2013</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> 2013</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dickinson <em>et al.</em> 2013</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Tzur <em>et al.</em> 2013</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lo <em>et al.</em> 2013</a>; Katic and Großhans 2013; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Waaijers <em>et al.</em> 2013</a>) and since then the community has produced increasingly sophisticated methods to mutate, delete and tag genes using this technology (for example, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dickinson <em>et al.</em> 2015</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Norris <em>et al.</em> 2015</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Paix <em>et al.</em> 2014</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">2016</a>).</p><p id="p-5">Our facility, along with several international laboratories, is attempting to isolate deletion alleles for the majority of genes in <em>C. elegans</em>. We have tested a number of the current methodologies for the CRISPR/Cas9 system and have successfully isolated several small deletions. We found that none of the CRISPR/Cas9 protocols presently available to the worm community were suitable as is for an ambitious high-throughput approach, which is what the <em>C. elegans</em> Gene Knockout Facility requires. In this paper we examine the details of several parts of the method with the aim of obtaining a more efficient experimental procedure suitable to the needs of the consortium that may in turn lead to a greater yield of deletions in a timely manner. These parts are (1) the repair mechanism and integrant selection, (2) homology arm length, (3) Cas9 delivery, and (4) guide RNA selection.</p><p id="p-6">We first examined repair mechanisms and selection procedures. While one can obtain deletions utilizing NHEJ, which is reportedly polymerase theta-mediated repair in the <em>C. elegans</em> germline (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">van Schendel <em>et al.</em> 2015</a>), the most versatile methodologies use HDR to introduce designer modifications at precise locations in the genome (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Yang <em>et al.</em> 2013</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Auer and Del Bene 2014</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Port <em>et al.</em> 2014</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bottcher <em>et al.</em> 2014</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">DiCarlo <em>et al.</em> 2013</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kim <em>et al.</em> 2014</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Arribere <em>et al.</em> 2014</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zhao <em>et al.</em> 2014</a>). HDR is homologous recombination generated at the Cas9 cut site that is facilitated by exogenous DNA with homology to the regions flanking the cut site in the genome. Two recent papers coupled HDR in <em>C. elegans</em> with the introduction of drug selection for either hygromycin (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Dickinson <em>et al.</em> 2015</a>) or G418 (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Norris <em>et al.</em> 2015</a>) as part of the screening process. This is an important modification as drug selection improves throughput by greatly reducing the number of animals that need to be screened. For the studies described in this paper, we have opted to use the G418 protocol described by <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Norris <em>et al.</em> (2015)</a>. Their protocol requires the introduction of a repair template containing a dual-selection cassette: a G418 resistance gene (<em>neoR</em>) and a pharyngeal GFP reporter (P<em>myo-2</em>::<em>GFP</em>), flanked by long homology arms (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). As described earlier, the drug resistance marker allows F1 selection on G418 and reduces primary screening efforts by eliminating animals that do not carry the repair template. Integration of the repair template via HDR allows the simultaneous deletion of a defined genomic region and introduction of the dual-selection cassette. Furthermore, the GFP marker makes it possible to discern recombinant animals from those harboring concatemer arrays. Recombinant animals display dim and uniform expression of GFP in the pharyngeal muscle, whereas concatemer array expression is often bright and mosaic. Additional pharyngeal and body-wall RFP-bearing plasmids (P<em>myo-2</em>::<em>RFP</em> and P<em>myo-3</em>::<em>RFP</em>, respectively) aid in the identification of non-integrants, since these independent plasmids should only be expressed from concatemer arrays. As many of the genes targeted in our facility are essential and thus homozygous inviable, the pharyngeal GFP marker acts as a dominant marker and allows us to track the deletion in heterozygotes. The GFP insert also allows us to track genes with no visible phenotype. For the research community this will be particularly useful for generating mutant crosses. Another convenient feature of this vector is the ability to easily excise the selection cassette using Cre recombinase as there are <em>LoxP</em> sites flanking the cassette (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>; <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Norris <em>et al.</em> 2015</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/135/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Generation of a deletion using the CRISPR/Cas9 protocol. (1) Guide RNAs direct Cas9 to create targeted DSBs in the gene of interest. (2) Through HDR, a portion of the ORF is replaced with the selection cassette containing pharyngeal GFP and G418 resistance (neoR) markers. (3) The dual-marker cassette, flanked by loxP sites, can be excised from the genome by injecting Cre recombinase. (Adapted from Norris et al. 2015.)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426239521" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Generation of a deletion using the CRISPR/Cas9 protocol. (1) Guide RNAs direct Cas9 to create targeted DSBs in the gene of interest. (2) Through HDR, a portion of the ORF is replaced with the selection cassette containing pharyngeal GFP and G418 resistance (neoR) markers. (3) The dual-marker cassette, flanked by &lt;em&gt;loxP&lt;/em&gt; sites, can be excised from the genome by injecting Cre recombinase. (Adapted from Norris &lt;em&gt;et al.&lt;/em&gt; 2015.)&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/135/F1.medium.gif" width="440" height="326"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/1/135/F1.medium.gif" width="440" height="326"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/135/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/135/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491850" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">Generation of a deletion using the CRISPR/Cas9 protocol. (1) Guide RNAs direct Cas9 to create targeted DSBs in the gene of interest. (2) Through HDR, a portion of the ORF is replaced with the selection cassette containing pharyngeal GFP and G418 resistance (neoR) markers. (3) The dual-marker cassette, flanked by <em>loxP</em> sites, can be excised from the genome by injecting Cre recombinase. (Adapted from <a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Norris <em>et al.</em> 2015</a>.)</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">Once we settled on an HDR procedure, accompanied by drug selection, we then examined the homology arm length required for efficient modification via HDR at the target site. The optimal length of homology arm required for efficient modification via HDR at the target site is still a matter for debate. <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Paix <em>et al.</em> (2014)</a> have shown that short, 30 to 60 nucleotide homologous linear repair templates can be highly efficient for gene editing over small target intervals. In contrast, two papers (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Dickinson <em>et al.</em> 2015</a>; <a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">Norris <em>et al.</em> 2015</a>) have shown that large inserts (&gt;1.5 kb) require longer homology arms (&gt;500 bp).</p><p id="p-9">We next examined how best to deliver Cas9 to facilitate efficient DNA cutting. There are at least three ways to deliver Cas9 to the target DNA: via plasmid, as messenger RNA or as purified protein. The Seydoux lab has shown that the third method, direct injection of Cas9 protein as a ribonucleoprotein (RNP) complex (Cas9 protein, the structural tracrRNA and the target-specific crRNA), is fivefold more efficient than injection of plasmids coding for these reagents in <em>C. elegans</em> (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Paix <em>et al.</em> 2014</a>). This advance could decrease injection time considerably and eliminate the need to generate guide RNA expression plasmids for every gene target since crRNAs could be synthesized.</p><p id="p-10">We recognized that the selection of effective guide RNAs is critical to efficiently obtain deletion mutations in <em>C. elegans</em>. Although there is no clear consensus for effective guide RNA design constraints, certain metrics for efficacy have been employed by various researchers (reviewed in <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Mohr <em>et al.</em> 2016</a>). We reasoned that our overall CRISPR/Cas9 success rate could be substantially improved by the application of a set of standard filters to the entire complement of all possible guide RNAs in the <em>C. elegans</em> genome, as a type of pre-selection to eliminate from consideration guide RNAs that could be expected to perform poorly. These filters are based on accumulated observations from many organisms. Our online <em>C. elegans</em>-specific guide RNA selection tool is available at <a href="http://genome.sfu.ca/crispr">http://genome.sfu.ca/crispr</a>.</p><p id="p-11">In this manuscript we present experimental data for each of these parts of the method, leading to an efficient standard protocol for using CRISPR/Cas9 to generate deletions in <em>C. elegans</em>, which is suitable for the requirements of our facility. We describe our new tool for the design and selection of guide RNAs specific to <em>C. elegans</em>, test the length of homology arms necessary and sufficient for engineering deletions of several hundred nucleotides while introducing a large (5.4 kb) selection cassette and compare the relative on-site efficiency of delivering Cas9 encoded in a plasmid to the direct delivery of Cas9 protein using a number of different gene targets. As our facility provides deletion strains to the Caenorhabditis Genetics Center (CGC), which then supplies the larger worm community, we want to ensure, if possible, that there are no additional mutations in these CRISPR/Cas9-generated mutants. To this end, we performed whole genome sequencing (WGS) on several of our deletion strains to determine the frequency of off-site mutations and to examine rearrangements at the target gene itself.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Designing a C. elegans-specific CRISPR guide RNA selection tool</h3><p id="p-12">Guide RNAs were designed for the whole <em>C. elegans</em> genome and made available to the research community (see Data Availability). Briefly, a series of filters on all potential guide sequences were applied with an in-house Perl script using the reference genome sequence and gene information from WormBase version WS250 (<a href="http://www.wormbase.org">http://www.wormbase.org</a>). In a first step, for each Protospacer Adjacent Motif (PAM) site in the genome we kept the corresponding adjacent 20-base guide only if its GC content was between 20% and 80% and no poly-T tracts of length five or longer were present. We annotated the presence or absence of the sequence GG at the 3′ end of each guide since guides ending with GG are expected to have higher efficiency for 3′ NGG PAMs (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Farboud and Meyer 2015</a>). Guides for which the seed region, defined as 12 bases at the 3′ end plus the PAM, was not unique in the genome were eliminated. The uniqueness of those 15-mers was assessed with an in-house C code modified from <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Flibotte and Moerman (2008)</a>. The guide plus PAM sequences were then aligned to the whole genome with bwa aln (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li and Durbin 2009</a>) allowing an edit distance of three and eliminating guides mapping to multiple locations in the genome. The minimum free energy in kcal/mol was calculated for each guide with the program hybrid-ss-min (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Markham 2003</a>) to help the researchers assess potential self-folding of the RNA. The location of the cut site associated with each guide was then annotated according to the gene feature being targeted, if any. All the guides were loaded into a database, which is searchable via a web interface. Users can search the database by entering a genomic interval or by searching a gene name. The database is also available as a genome browser track on WormBase (<a href="http://www.wormbase.org">www.wormbase.org</a>). Constraints can be applied to the GC content, folding energy, and the presence of GG at the 3′ end of the guides being returned.</p></div><div id="sec-3" class="subsection"><h3>CRISPR/Cas9 deletion procedures</h3><p id="p-13">CRISPR/Cas9 edits were generated using a protocol previously described by <a id="xref-ref-32-7" class="xref-bibr" href="#ref-32">Norris <em>et al.</em> (2015)</a>, with some modifications. Repair templates were assembled using the NEBuilder Hifi DNA Assembly Kit (New England BioLabs) to incorporate homology arms into a dual-marker selection cassette (<em>loxP</em> + P<em>myo-2</em>::<em>GFP</em>::<em>unc-54</em> 3′UTR + P<em>rps-27</em>::<em>neoR</em>::<em>unc-54</em> 3′UTR <em>+ loxP</em> vector provided by John Calarco). Homology arms ordered as DNA gBlocks from Integrated DNA Technologies (IDT), were approximately 500 bp in length including 50 bp of overhang adaptor sequences for the assembly reaction.</p><p id="p-14">Depending on the experiment, CRISPR/Cas9 edits were generated using one or two guide RNAs, and Cas9 was delivered via one of two methods: a Cas9 expression plasmid (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fu <em>et al.</em> 2014</a>) or purified Cas9 protein (Paix <em>et al.</em> 2014). For experiments using the Cas9 expression plasmid, a single guide RNA (sgRNA) vector was constructed by performing PCR on a p<em>U6</em>::<em>klp-12</em> vector (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Friedland <em>et al.</em> 2013</a>) to incorporate the 20 bp guide RNA sequence via the forward primer. Injection mixes consisted of: 100 ng/µL sgRNA plasmid, 50 ng/µL repair template, 50 ng/µL Cas9 expression plasmid (P<em>eft-3</em>::<em>Cas9</em>::<em>NLS <sub>SV40</sub></em>::<em>NLS</em>::<em>tbb-2</em> 3′UTR<em>)</em>, 5 ng/µL pCFJ104 (P<em>myo-3</em>::<em>mCherry</em>), and 2.5 ng/µL pCFJ90 (P<em>myo-2</em>::<em>mCherry</em>). For Cas9 protein injections, crRNA and tracrRNA (synthesized by IDT) were duplexed and then combined into a Cas9 RNP complex following the manufacturer’s recommendations. Cas9 protein was purified according to the protocol described by Paix <em>et al.</em> (2015) from plasmid nm2973 (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Fu <em>et al.</em> 2014</a>) and stored at -20° at a concentration of 34 µg/µL in 20 mM Hepes buffer pH 8.0, 500 mM KCl and 20% glycerol. Injection mixes consisted of 50 ng/µL repair template, 1.5 µM RNP complex, 5 ng/µL pCFJ104, and 2.5 ng/µL pCFJ90.</p><p id="p-15">For each CRISPR target, approximately 32 young adult VC3504 (Moerman lab derivative of <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>) hermaphrodites were injected as previously described by <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kadandale <em>et al.</em> (2009)</a> and screened according to the protocol in <a id="xref-ref-32-8" class="xref-bibr" href="#ref-32">Norris <em>et al.</em> (2015)</a>.</p></div><div id="sec-4" class="subsection"><h3>Whole genome sequencing (WGS) and analysis to measure off-target effects after CRISPR/Cas9 treatment</h3><p id="p-16">Off-target effects were assessed using WGS for eight CRISPR/Cas9 edited strains generated for two genes from the same parental VC3504 population. Two guide RNAs were used for each of the target genes, <em><a href="http://www.wormbase.org/db/get?name=WBGene00021146;class=Gene">lgc-45</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00016398;class=Gene">C34D4.2</a></em> (Table S1). To assess possible differences in off-target effects between Cas9 plasmid and Cas9 protein, we injected approximately 32 animals with either plasmid or protein for each set of CRISPR guides. All independently generated strains were sequence validated by PCR for correct insertion of the selection cassette. For each target, four independently generated strains (two from plasmid injections, and two from protein injections) were selected for WGS. We also sequenced the parental strain VC3504. All CRISPR strains were no more than three generations away from the parental strain when injected.</p><p id="p-17"><em>C. elegans</em> strains were grown as described by <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brenner (1974)</a>. Strains were allowed to grow on either 100 mm or 60 mm agar plates seeded with <a href="http://www.wormbase.org/db/get?name=OP50;class=Strain">OP50</a> until just starved. Worms were washed off plates with sterile distilled water into 15 mL centrifuge tubes, pelleted, and washed with several changes of dH<sub>2</sub>O to remove residual <em>Escherichia coli</em> and other possible contaminants. The supernatant was removed after the final wash and a dense pellet of worms was transferred into a 1.5 mL freezer tube. The dense pellets of worms were frozen at -80° and subsequently subjected to standard genomic DNA extraction. The extraction begins with Proteinase K digestion and treatment with RNAse A, followed by 6M NaCl protein precipitation. The DNA is then precipitated in isopropanol, washed in 70% ethanol, and finally resuspended in distilled water. The genomic DNA was run on 1% agarose gel and high molecular weight DNA was extracted using NEB’s Monarch Gel Extraction Kit. Sequencing libraries were made using Illumina’s Nextera XT library prep kit and run on an Agilent Bioanalyzer to check average fragment size. All strains were sequenced on MiSeq 2x300 or 2x75 runs. In one case, strain VC3823, the second read was unusable, so it was treated as a single run (<em>i.e.</em>, not paired) at the analysis stage.</p><p id="p-18">Sequence reads were mapped to the <em>C. elegans</em> reference genome version WS260 (<a href="http://www.wormbase.org">http://www.wormbase.org</a>) using the short-read aligner BWA version 0.7.16 (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Li and Durbin 2009</a>). For each sample, this resulted in an average sequencing depth ranging from 22x to 41x with a median of 32x. Single-nucleotide variants (SNVs) and small insertions/deletions (indels) were identified and filtered with the help of the SAMtools toolbox version 1.6 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Li <em>et al.</em> 2009</a>). Candidate variants at genomic locations for which the parental <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> strain VC3504 had a read depth lower than 10 reads or an agreement rate with the reference genome lower than 98% were eliminated from further consideration. Furthermore, for each of the eight CRISPR strains sequenced, only variants at locations where all of the other seven CRISPR strains agreed with the reference for at least 95% of the reads were kept. This last filter principally eliminated heterozygous candidates likely due to PCR artifacts (often in or adjacent to homopolymer stretches of A or T) or potentially present at low levels in the parental population but undetected by our sequencing of VC3504. Each variant was annotated with a custom Perl script and gene information downloaded from WormBase version WS260. The read alignments in the regions of candidate variants were visually inspected with Integrative Genomics Viewer (IGV; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Robinson <em>et al.</em> 2011</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Thorvaldsdottir <em>et al.</em> 2013</a>). Copy numbers were estimated from the alignments with a procedure analogous to that of <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Itani <em>et al.</em> (2015<em>)</em></a> using 1 kb wide sliding windows with the alignments from the parental strain as the reference in order to search for potential off-target genomic rearrangements.</p></div><div id="sec-5" class="subsection"><h3>Quality Control testing of CRISPR/Cas9 target deletions using PCR</h3><p id="p-19">WGS is an extremely informative method for determining the structure of a CRISPR mutation, but it is more labor-intensive, time-consuming and costly than PCR. We developed a straightforward PCR protocol as an efficient screening tool, which provides basic information about the selection cassette insertion and the desired deletion in four reactions per generated mutation. Two primer pairs are used to amplify the upstream and downstream portions of the insertion, each pair with one primer in the genomic sequence adjacent to the insertion and the other within the cassette sequence. Another pair of primers is used to amplify wild type (WT) sequence in the region of the deletion from both <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and mutant templates. The latter primer pair is sometimes designed as a pair flanking the deletion, and sometimes as a pair with one primer flanking and one internal to the deleted region. There are three possible results from these PCRs: (1) The WT product is present and of the correct size from <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and absent from the mutant, and both insertion-site products are present and of the correct size; (2) The WT assays are correct and one or both insertion-site product are missing or of incorrect size; and (3) The WT assay on the mutant is incorrect. For the first two cases, we conclude that the gene is disrupted and that the insertion sites are correct or rearranged. In the third case, where a WT product of the size predicted for <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> is produced from the mutant template, we conclude that the gene is not disrupted.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-20">The raw sequence data from this study have been submitted to the NCBI BioProject (<a href="http://www.ncbi.nlm.nih.gov/bioproject">http://www.ncbi.nlm.nih.gov/bioproject</a>) under accession number PRJNA 473363 and can be accessed from the Sequence Read Archive (SRA; <a href="https://www.ncbi.nlm.nih.gov/sra">https://www.ncbi.nlm.nih.gov/sra</a>) with accession number SRP149097. CRISPR guide RNA sequences designed for the whole genome are available using the search tool at <a href="http://genome.sfu.ca/crispr/">http://genome.sfu.ca/crispr/</a> and can be viewed interactively within JBrowse at WormBase (<a href="http://www.wormbase.org">http://www.wormbase.org</a>) by activating the track called “CRISPR_Cas9 sgRNA predictions”. All the guides can also be downloaded in bulk as a bed file at <a href="http://genome.sfu.ca/crispr/">http://genome.sfu.ca/crispr/</a>. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7312274">https://doi.org/10.25387/g3.7312274</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Guide RNA design and selection</h3><p id="p-21">In the context of a production laboratory like our gene knockout facility it is crucial to use the time of the laboratory personnel effectively and eliminate unnecessary tasks. With that in mind, we attempted to streamline the design of guide RNAs by pre-calculating, pre-filtering and annotating potential guide RNAs across the whole <em>C. elegans</em> genome using current information about what defines an optimal guide RNA. Briefly, we selected guides with a balanced GC content. We annotated the presence or absence of the sequence GG at the 3′ end of each guide, which has been shown to provide higher efficiency (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Farboud and Meyer 2015</a>). Guides for which the seed region was not unique in the genome and guides mapping to multiple locations in the genome were eliminated. Since such a resource is useful to the research community, we made our database of guide RNAs accessible via a public web site (<a href="http://genome.sfu.ca/crispr">http://genome.sfu.ca/crispr</a>). Our website provides direct links from each guide to JBrowse at WormBase (<a href="http://www.wormbase.org">http://www.wormbase.org</a>). Within JBrowse one can view the location, sequence and orientation of the guide. Zooming out reveals the location of other guides and provides context for the location of the guide within the gene. The guides can also be viewed within IGV (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Robinson <em>et al.</em> 2011</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Thorvaldsdottir <em>et al.</em> 2013</a>) after downloading a bed file available on our guide RNA website. Our website allows simple searches using either gene names or genomic intervals, with optional filters to limit searches to regulatory, intergenic, or non-coding regions.</p><p id="p-22">As expected, the use of the guide RNA selection tool significantly reduced the amount of time spent at the design stage by laboratory personnel, but most importantly, it also improved overall laboratory throughput by promoting the selection of more efficient guides. Our current success rate for obtaining deletions using our pre-selected guides is approximately 85%. Filtering and selecting for unique guide RNAs is most likely the major contributor to the negligible off-target background we observe after CRISPR/Cas9 use (see below).</p></div><div id="sec-9" class="subsection"><h3>Homology–directed repair (HDR) and homology arm length</h3><p id="p-23">HDR allows DNA fragments to be incorporated into a precise region of the <em>C. elegans</em> genome. Homology arms flanking these DNA fragments dictate the genomic region that will be simultaneously deleted (if so desired) and replaced. One of the two homologous regions must be immediately adjacent to the Cas9 double-strand cut site in the genome. To date, the homology arm length used in different protocols has varied from 35 bp (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Paix <em>et al.</em> 2014</a>) up to approximately 2 kb (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Dickinson <em>et al.</em> 2015</a>; <a id="xref-ref-32-9" class="xref-bibr" href="#ref-32">Norris <em>et al.</em> 2015</a>). <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Paix <em>et al.</em> (2016)</a> report optimal HDR efficiency with 35 bp homology arms on a single-stranded template for making small edits. From previous studies (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Dickinson <em>et al.</em> 2015</a>; <a id="xref-ref-32-10" class="xref-bibr" href="#ref-32">Norris <em>et al.</em> 2015</a>) we suspected we would need longer homology arms to make larger edits. We conducted a series of experiments to test the efficiency of generating a 536 bp deletion and inserting a large selection cassette (5.4 kb) at a Cas9-induced double-strand break using a single guide with different lengths of homology arms (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, Table S1). The lengths of upstream and downstream homology arms differed slightly due to synthesis constraints on the sequence of gBlocks ordered from IDT.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/135/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Assessing editing efficiency with homology arms of varying lengths. A 536 bp deletion was generated in rap-3 using a single guide RNA and various lengths of homology arms (Table S1). The proportion of individual injected P0s giving rise to animals with the selection cassette integrated at the desired location in the genome was determined by PCR validation. Error bars indicate standard error of the mean. Statistical significance between groups was determined using the Chi-squared test, without correction for multiple testing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426239521" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Assessing editing efficiency with homology arms of varying lengths. A 536 bp deletion was generated in &lt;em&gt;rap-3&lt;/em&gt; using a single guide RNA and various lengths of homology arms (Table S1). The proportion of individual injected P&lt;sub&gt;0&lt;/sub&gt;s giving rise to animals with the selection cassette integrated at the desired location in the genome was determined by PCR validation. Error bars indicate standard error of the mean. Statistical significance between groups was determined using the Chi-squared test, without correction for multiple testing.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/135/F2.medium.gif" width="440" height="308"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/1/135/F2.medium.gif" width="440" height="308"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/135/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/135/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491982" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Assessing editing efficiency with homology arms of varying lengths. A 536 bp deletion was generated in <em>rap-3</em> using a single guide RNA and various lengths of homology arms (Table S1). The proportion of individual injected P<sub>0</sub>s giving rise to animals with the selection cassette integrated at the desired location in the genome was determined by PCR validation. Error bars indicate standard error of the mean. Statistical significance between groups was determined using the Chi-squared test, without correction for multiple testing.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">For this experiment, we followed the plasmid-based CRISPR protocol described in Materials and Methods with one modification: injected P<sub>0</sub>s were singled onto plates in order to obtain an accurate measure of editing efficiency, which was defined as the proportion of P<sub>0</sub> plates bearing progeny with the selection cassette integrated at the desired location, as confirmed by PCR. The highest editing efficiency was obtained when utilizing homology arms over approximately 400 bp (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). When homology arms smaller than these were used, the editing efficiency dropped significantly (<em>P</em> &lt; 0.05). These results agree with previous findings for making larger edits (<a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Dickinson <em>et al.</em> 2015</a>; <a id="xref-ref-32-11" class="xref-bibr" href="#ref-32">Norris <em>et al.</em> 2015</a>), suggesting that the optimal length of homology for efficient genome editing is dependent on the particular application of HDR being used. As we observe no further gain in efficiency of HDR for larger deletions with homology arms over approximately 400 bp in length, we have adjusted our protocol accordingly. For our current production protocol, we order homology arms as 500 bp gBlocks from IDT with 50 bp of adaptor sequences included to enable Gibson assembly (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gibson <em>et al.</em> 2009</a>) into our repair vector.</p></div><div id="sec-10" class="subsection"><h3>Delivery of purified Cas9 protein <em>vs.</em> plasmid-borne Cas9</h3><p id="p-26">There are several reports in the literature from researchers using mammalian systems indicating that efficiency of Cas9 cutting improves if they use purified Cas9 protein rather than plasmid-borne Cas9 (reviewed in <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wu <em>et al.</em> 2014</a>). Results confirming these observations have been reported for <em>C. elegans</em> (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Paix <em>et al.</em> 2014</a>). There is some debate on how much Cas9 protein is required for making a double-strand break (DSB) in this organism. All results reported here were achieved using Cas9 protein at a concentration between 1.5 µM and 3 µM. Our results confirm that, as in other systems and as previously reported for this nematode, purified Cas9 is more efficient at inducing DSBs than plasmid-borne Cas9. In cases where we have tested the same gene with both purified protein and plasmid-borne Cas9 (three genes), using the purified protein can be nearly four times more efficient at inducing a double-strand break at the target site (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Note that this is not true for all genes, as we only see a twofold improvement for the gene <a href="http://www.wormbase.org/db/get?name=WBGene00016874;class=Gene">C52B11.5</a>. However, the overall difference for these three genes is highly significant (Chi-squared test, <em>P</em> = 2e-5). On average we observe a more than twofold increase in efficiency when comparing the 112 genes we have targeted using either plasmid (56 genes) or protein (59 genes) delivery of Cas9 (Table S2). There are reports using other model systems that suggest that the use of purified protein leads to fewer non-specific effects than using the plasmid. As discussed below, that is not the case for <em>C. elegans</em>, where the two delivery methods exhibit an identical low level of non-specific effects.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/135/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of Cas9 delivery methods. A direct comparison of editing efficiency between Cas9 plasmid and protein was done across three genes, using two guide RNAs for each gene (Table S1). Each P0 plate contained four injected animals, and F2 progeny were screened for selection cassette integration at the desired location using PCR. Overall, the average editing efficiency for the three genes targeted with Cas9 protein was significantly higher (Chi-squared test, P = 2e-5) than that of the same three genes targeted with Cas9 plasmid by more than a factor of three." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426239521" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of Cas9 delivery methods. A direct comparison of editing efficiency between Cas9 plasmid and protein was done across three genes, using two guide RNAs for each gene (Table S1). Each P&lt;sub&gt;0&lt;/sub&gt; plate contained four injected animals, and F2 progeny were screened for selection cassette integration at the desired location using PCR. Overall, the average editing efficiency for the three genes targeted with Cas9 protein was significantly higher (Chi-squared test, &lt;em&gt;P&lt;/em&gt; = 2e-5) than that of the same three genes targeted with Cas9 plasmid by more than a factor of three.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/135/F3.medium.gif" width="440" height="418"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/1/135/F3.medium.gif" width="440" height="418"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/135/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/135/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491884" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Comparison of Cas9 delivery methods. A direct comparison of editing efficiency between Cas9 plasmid and protein was done across three genes, using two guide RNAs for each gene (Table S1). Each P<sub>0</sub> plate contained four injected animals, and F2 progeny were screened for selection cassette integration at the desired location using PCR. Overall, the average editing efficiency for the three genes targeted with Cas9 protein was significantly higher (Chi-squared test, <em>P</em> = 2e-5) than that of the same three genes targeted with Cas9 plasmid by more than a factor of three.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Off-target effects</h3><p id="p-28">At present there appear to be conflicting results in the literature with regard to off-target events when using the CRISPR/Cas9 gene editing system. Whether such events are due to guide RNA homology, homology of the repair template, or unrelated spurious Cas9 cutting is unclear. As it is critical that we know of any such events in the strains we provide to the CGC and ultimately to the larger worm community, we undertook a series of tests to determine the frequency of off-target events after CRISPR/Cas9 treatment.</p><p id="p-29">We performed WGS on eight CRISPR/Cas9-edited strains and their parental strain (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). For this experiment we targeted two distinct genes using two different Cas9 delivery methods. We searched for small off-target mutations, both SNVs and indels, using strict filtering in order to eliminate variants already present in the parental population as well as potential technical artifacts (Materials and Methods). We found a total of eight mutations in addition to the desired edits (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), six homozygous and two heterozygous calls. None of the mutations appear to be related to either the guide RNAs or the homology arms. We also searched for off-target genomic rearrangements by analyzing copy number estimates (Materials and Methods), but none were found. The frequency of extraneous SNVs and indels detected is on the order of spontaneous mutations reported for this organism (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Denver <em>et al.</em> 2009</a>), which suggests the observed mutations are unlikely to be due to the CRISPR/Cas9 procedure. In our quality control studies we have used WGS to analyze an additional 30+ CRISPR/Cas9-generated deletion lines, and again, we found little or no evidence for off-target events (data not shown). We conclude that off-target mutations due to this method are rare and do not appear to be a serious concern in <em>C. elegans</em> when guide RNAs are carefully selected. We also conclude that in this nematode, unlike what has been reported for other systems, the delivery method (<em>i.e.</em>, plasmid <em>vs.</em> protein) does not appear to influence the occurrence of off-target mutations.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491811/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491811/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Whole-genome sequencing statistics for the parental strain and 8 mutant strains produced for the off-target mutation analysis</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Validating deletion events at the target site</h3><p id="p-30">While we detected virtually no off-target events, we now have several examples where target deletions are accompanied by local rearrangements, including local duplications, deletions of adjacent DNA, or insertions of multiple copies of the repair template. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figures 4</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">5</a> illustrate examples of what we observed. In <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, the upper panel shows a precise deletion of the interval between the homology arms encompassing the first two exons of the gene <a href="http://www.wormbase.org/db/get?name=WBGene00008491;class=Gene">F01D4.9.</a> However, in the same screen we also isolated a more complex event as illustrated in the lower panel of <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>. Here, the region we targeted to delete is accompanied by a large downstream deletion and multiple copies of the repair template, as evidenced by the increased coverage in the homology arm regions. <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> illustrates a more complex example of a rearrangement accompanying the deletion of a targeted interval in a gene. In this example, an additional eight genes are affected beyond the targeted gene. Several genes are deleted and there appears to be accompanying inversions and duplicated regions. These examples are from WGS of more than 30 CRISPR/Cas9 derived strains for several target genes.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/135/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Rearrangements at the CRISPR/Cas9 target site. Two examples of removing the target gene through CRISPR/Cas9 HDR visualized using IGV. In strain VC3671, there is a perfect deletion removing the first two exons of gene F01D4.9. In strain VC3674, there is evidence of a complex rearrangement. A large deletion encompasses the same two exons as well as the downstream region of F01D4.5 and a pseudogene, F01D4.3. This region likely harbors multiple copies of the repair template, since the average coverage of the homology arms are relatively high compared to the adjacent genomic sequence. At the bottom of the figure, the exons for the various genes are shown in blue, the homology arms chosen for F01D4.9 are shown in green, and the guide RNA is in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426239521" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Rearrangements at the CRISPR/Cas9 target site. Two examples of removing the target gene through CRISPR/Cas9 HDR visualized using IGV. In strain VC3671, there is a perfect deletion removing the first two exons of gene F01D4.9. In strain VC3674, there is evidence of a complex rearrangement. A large deletion encompasses the same two exons as well as the downstream region of F01D4.5 and a pseudogene, F01D4.3. This region likely harbors multiple copies of the repair template, since the average coverage of the homology arms are relatively high compared to the adjacent genomic sequence. At the bottom of the figure, the exons for the various genes are shown in blue, the homology arms chosen for F01D4.9 are shown in green, and the guide RNA is in red.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/135/F4.medium.gif" width="440" height="231"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/1/135/F4.medium.gif" width="440" height="231"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/135/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/135/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491892" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Rearrangements at the CRISPR/Cas9 target site. Two examples of removing the target gene through CRISPR/Cas9 HDR visualized using IGV. In strain VC3671, there is a perfect deletion removing the first two exons of gene F01D4.9. In strain VC3674, there is evidence of a complex rearrangement. A large deletion encompasses the same two exons as well as the downstream region of F01D4.5 and a pseudogene, F01D4.3. This region likely harbors multiple copies of the repair template, since the average coverage of the homology arms are relatively high compared to the adjacent genomic sequence. At the bottom of the figure, the exons for the various genes are shown in blue, the homology arms chosen for F01D4.9 are shown in green, and the guide RNA is in red.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/135/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Complex rearrangements at the CRISPR/Cas9 target site. Whole genome sequencing of VC3743, a CRISPR-generated strain, revealed an unintended 16 kb deletion that disrupts the target gene as well as eight other genes in the surrounding area. This deletion is also accompanied by a complex rearrangement that consists of fragments of genomic sequence from the local region and the repair template. The rearrangement joins (1) the 5&#x2032; portion of srh-281 to a duplicated fragment of oac-15, followed by (2) a duplicated inverted portion of oac-15, (3) an inverted intergenic region, and (4,5) the selection cassette flanked by homology arms. In blue are the exons for the various genes and in pink are the homology arms chosen for F11A5.3. The marker for the guide RNA is not visible at this scale." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426239521" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Complex rearrangements at the CRISPR/Cas9 target site. Whole genome sequencing of VC3743, a CRISPR-generated strain, revealed an unintended 16 kb deletion that disrupts the target gene as well as eight other genes in the surrounding area. This deletion is also accompanied by a complex rearrangement that consists of fragments of genomic sequence from the local region and the repair template. The rearrangement joins (1) the 5&#x2032; portion of &lt;em&gt;srh-281&lt;/em&gt; to a duplicated fragment of &lt;em&gt;oac-15&lt;/em&gt;, followed by (2) a duplicated inverted portion of &lt;em&gt;oac-15&lt;/em&gt;, (3) an inverted intergenic region, and (4,5) the selection cassette flanked by homology arms. In blue are the exons for the various genes and in pink are the homology arms chosen for F11A5.3. The marker for the guide RNA is not visible at this scale.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/135/F5.medium.gif" width="440" height="277"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/1/135/F5.medium.gif" width="440" height="277"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/135/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/135/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491937" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-32" class="first-child">Complex rearrangements at the CRISPR/Cas9 target site. Whole genome sequencing of VC3743, a CRISPR-generated strain, revealed an unintended 16 kb deletion that disrupts the target gene as well as eight other genes in the surrounding area. This deletion is also accompanied by a complex rearrangement that consists of fragments of genomic sequence from the local region and the repair template. The rearrangement joins (1) the 5′ portion of <em>srh-281</em> to a duplicated fragment of <em>oac-15</em>, followed by (2) a duplicated inverted portion of <em>oac-15</em>, (3) an inverted intergenic region, and (4,5) the selection cassette flanked by homology arms. In blue are the exons for the various genes and in pink are the homology arms chosen for F11A5.3. The marker for the guide RNA is not visible at this scale.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">The above observations on rearrangements led us to do PCR analysis of 330 CRISPR/Cas9-induced events for 81 genes (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> and Table S3). The PCR validation scheme described in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> was used to determine the proportion of strains belonging to each mutant class for each of the gene targets. On average, when calculated per gene target, we obtained precise editing events in 35% of the integrant strains. Notably, 37% of the events contained imprecise edits, with rearrangements at one or both, of the 5′ and 3′ deletion sites. Curiously, we found that 28% of the strains had at least a portion of the wild-type sequence retained along with integration of the drug and GFP selectable marker. For this latter group, we suspect that a partial deletion or duplication event accompanied the marker insertion during HDR. This is suggested by PCR results that show proper integration of the selection cassette at one or both ends of the intended deletion interval yet still exhibit a wild-type band of the correct size. What is important to note here is that there is a wide range in variability from gene to gene for the ratio of precise edits <em>vs.</em> the other categories (Table S3). For some sites, the majority of events lead to precise insertion/deletions, but for other sites it appears to be much more difficult to generate a precise deletion. Perhaps it should not be a surprise that a double-strand break can yield precise HDR, but can also be resolved in other ways.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/135/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Homology-directed repair resolves CRISPR/Cas9 double-strand breaks in unpredictable ways. Using the PCR validation scheme described in Table 2, analysis of 330 CRISPR/Cas9-derived mutants (Table S3) reveals that integration of the selection cassette via HDR does not always occur as predicted. These mutant strains were generated using up to two guide RNAs and either Cas9 protein or a Cas9 expression plasmid. The proportion of integrant strains belonging to each mutant class within each gene target was determined and the average proportions for 81 gene targets were calculated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426239521" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Homology-directed repair resolves CRISPR/Cas9 double-strand breaks in unpredictable ways. Using the PCR validation scheme described in Table 2, analysis of 330 CRISPR/Cas9-derived mutants (Table S3) reveals that integration of the selection cassette via HDR does not always occur as predicted. These mutant strains were generated using up to two guide RNAs and either Cas9 protein or a Cas9 expression plasmid. The proportion of integrant strains belonging to each mutant class within each gene target was determined and the average proportions for 81 gene targets were calculated.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/135/F6.medium.gif" width="440" height="273"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/1/135/F6.medium.gif" width="440" height="273"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/135/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/135/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491998" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-34" class="first-child">Homology-directed repair resolves CRISPR/Cas9 double-strand breaks in unpredictable ways. Using the PCR validation scheme described in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, analysis of 330 CRISPR/Cas9-derived mutants (Table S3) reveals that integration of the selection cassette via HDR does not always occur as predicted. These mutant strains were generated using up to two guide RNAs and either Cas9 protein or a Cas9 expression plasmid. The proportion of integrant strains belonging to each mutant class within each gene target was determined and the average proportions for 81 gene targets were calculated.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491905/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491905/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Quality Control PCR Interpretation</span><div class="sb-div caption-clear"></div></div></div><p id="p-35">As this will be a recurring problem and is virtually impossible to predict for each gene (Table S3), we have developed a Quality Control (QC) deletion validation protocol. Our initial selection is based on drug resistance and Mendelian segregation of weak GFP expression in the pharynx. This is strong evidence that we have targeted and disrupted the gene of interest, but as we state above it does not guarantee that we have not disrupted flanking DNA, or even indicate whether we still have an intact copy of the target gene. Our PCR-based QC protocol tests for these possibilities by verifying the integrity of the disruption junction on both sides of the insert and confirming that the wild-type sequence is indeed absent. Primer pairs chosen and expected results are shown in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>. A key test is to determine whether the wild-type sequence for the targeted deletion interval is indeed absent. For the wild-type PCR assay, we generate a genomic amplicon in wild-type and confirm its absence in the mutant. Our experience to date leads us to expect only a portion of drug-selected GFP positive animals to be correctly edited for any particular gene. It is for this reason that we inject approximately 32 worms to obtain at least five or six positive integrants per target gene.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-36">We have investigated several variables that may impinge on the effectiveness of the CRISPR/Cas9 procedure to generate large deletions in <em>C. elegans</em>. Our <em>C. elegans</em>-specific guide RNA selection tool, in conjunction with the selection vector designed by the Calarco group (<a id="xref-ref-32-12" class="xref-bibr" href="#ref-32">Norris <em>et al.</em> 2015</a>), and the use of purified Cas9 protein, as pioneered by the Seydoux group for use in this organism (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Paix <em>et al.</em> 2016</a>), yields a very efficient and effective protocol. We feel this protocol has sufficient robustness to allow our facility to tackle the remaining genes in this organism that lack null alleles.</p><p id="p-37">Among the parameters we explored in refining an HDR protocol appropriate for our purposes were homology arm length and Cas9 delivery methods, <em>i.e.</em>, delivery via plasmid or as a purified protein in a nucleoprotein complex. These parameters were previously explored (<a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Dickinson <em>et al.</em> 2015</a>; <a id="xref-ref-32-13" class="xref-bibr" href="#ref-32">Norris <em>et al.</em> 2015</a>; <a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Paix <em>et al.</em> 2014</a>, <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">2016</a>), but in somewhat different conditions than those reported in this study. The length of the homology arm required is at least partially dependent on the type of edit being done to the gene. For single nucleotide changes or small indels, as few as 35 nucleotides of homology are required when using single-stranded bridging oligonucleotides (ssODNs) (<a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Paix <em>et al.</em> 2016</a>). As our facility is looking to make larger deletions that remove most or all of an open reading frame (ORF), this approach is not feasible. For this reason, we were more attracted to protocols developed for replacing larger stretches of DNA. We also preferred protocols that replaced direct PCR screening of DNA from individual strains with a drug and/or visible marker for selection as we envision performing hundreds of screens using the protocol. As stated earlier, we examined the <a id="xref-ref-12-8" class="xref-bibr" href="#ref-12">Dickinson <em>et al.</em> (2015)</a> and <a id="xref-ref-32-14" class="xref-bibr" href="#ref-32">Norris <em>et al.</em> (2015)</a> vectors and settled on using the G418-resistant, pharyngeal GFP selection system of <a id="xref-ref-32-15" class="xref-bibr" href="#ref-32">Norris <em>et al.</em> (2015)</a>. We chose the latter system because it has a smaller selection cassette. The GFP marker is advantageous to the work in our facility as many of the genes we target are essential and thus a deletion allele will lead to homozygous lethality. The GFP acts as a dominant selectable marker and is quite useful for tracking the deletion and constructing balanced lines.</p><p id="p-38">Both the <a id="xref-ref-12-9" class="xref-bibr" href="#ref-12">Dickinson <em>et al.</em> (2015)</a> and <a id="xref-ref-32-16" class="xref-bibr" href="#ref-32">Norris <em>et al.</em> (2015)</a> groups agree that longer homology arms are required for making large insertions/deletions via HDR. They both experimented with homology arms of 1-2 kb but also pointed out that 500 bp arms would work. We concur with these findings and extend these studies to show there is no advantage to making arms longer than 500 bp. With homology length of only 450 bp, we were able to generate up to 20 kb deletions (using two guide RNAs – data not shown). From a recent paper on using CRISPR/Cas9 to generate inversions in <em>C. elegans</em> it appears there is no limit to the spacing of simultaneous cut sites (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dejima <em>et al.</em> 2018</a>).</p><p id="p-39">Similar to <a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">Paix <em>et al.</em> (2016)</a>, we find that purified protein is more efficient at generating on-target DSBs. In our hands, the purified protein is at least two times, and often four times more effective than plasmid-borne Cas9 when tested against numerous genes. There are a number of possible reasons for this difference but the most likely one is that there is substantial lag between injecting a plasmid and producing a protein, which then has to be combined with the guide RNA. This lag time is removed when injecting an RNP complex directly. Overall protein concentration between the two methods may also differ.</p><p id="p-40">We detect virtually no off-target events after using our CRISPR/Cas9 protocol. We suspect the major contributor to eliminating off-target events is the use of our <em>C. elegans</em>-specific guide RNA selection tool. Several reviews on the issue of off-target effects consider poor guide RNA design to be a major contributor to these effects (see, for example <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Wu <em>et al.</em> 2014</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hendel <em>et al.</em> 2015</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jamal <em>et al.</em> 2017</a>). Additionally, while we saw no difference in the frequency of off-target events between Cas9 delivery systems, others have shown that in mammalian cells, direct delivery of Cas9 protein and guide RNA as a nucleoprotein complex reduces off-target events (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kim <em>et al.</em> 2014</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ramakrishna <em>et al.</em> 2014</a>). Considering that the deletion lines we generate will go to the CGC for distribution to the nematode community, identifying and eliminating the sources of off-target events is imperative. It seems the protocol described here satisfies this criterion. Perhaps a more important issue to address at this time is on-target rearrangements that may occur during HDR. We have documented that these events happen at most target sites and therefore proper analysis has to be done when assaying any event generated using the CRISPR/Cas9 method.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">Dr. John Calarco (University of Toronto) and Dr. Geraldine Seydoux (Johns Hopkins University) kindly provided us with reagents and much valued advice as we refined the CRISPR/Cas9 system in our laboratory. We also thank Mei Zhen and Julie Claycomb, for their critical support during the early stages of these studies. This work was funded by CIHR grant PJT-148549 to DGM, CIHR grant MOP 93719 and NSERC grant 312498 to HH, NIH grant 5P40OD010440 to AR and NIH grant 5R240D023041 to AR, Paul Sternberg, Geraldine Seydoux and DGM.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-42">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7312274">https://doi.org/10.25387/g3.7312274</a>.</p></li><li class="fn" id="fn-3"><p id="p-43"><em>Communicating editor: K. Gunsalus</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-10-05"><span class="received-label">Received </span>October 5, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-11-07"><span class="accepted-label">Accepted </span>November 7, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Au <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.1" data-doi="10.1534/genetics.114.169730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arribere</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">B. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Artiles</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartman</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient marker-free recovery of custom genetic modifications with CRISPR/Cas9 in <em>C. elegans.</em></span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">837</span>–<span class="cit-lpage">846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.169730%26rft_id%253Dinfo%253Apmid%252F25161212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzMvODM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8xLzEzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.2" data-doi="10.1016/j.ymeth.2014.03.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Auer</span>  <span class="cit-name-given-names">T. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Del Bene</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">CRISPR/Cas9 and TALEN-mediated knock-in approaches in zebrafish.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">142</span>–<span class="cit-lpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%26rft.volume%253D69%26rft.spage%253D142%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2014.03.027%26rft_id%253Dinfo%253Apmid%252F24704174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2014.03.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24704174&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.3" data-doi="10.1093/nar/gku289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bottcher</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollmann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merk</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nitschko</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Obermaier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient chromosomal gene modification with CRISPR/cas9 and PCR-based homologous recombination donors in cultured Drosophila cells.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">e89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku289%26rft_id%253Dinfo%253Apmid%252F24748663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku289&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24748663&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.4" data-doi="10.1051/medsci/20031967725"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The genetics of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBrenner%26rft.auinit1%253DS.%26rft.volume%253D77%26rft.issue%253D1%26rft.spage%253D71%26rft.epage%253D94%26rft.atitle%253DTHE%2BGENETICS%2BOF%2BCAENORHABDITIS%2BELEGANS%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fmedsci%252F20031967725%26rft_id%253Dinfo%253Apmid%252F4366476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiNzcvMS83MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMS8xMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.5" data-doi="10.1093/nar/gkt805"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fenk</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Bono</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Efficient genome editing in <em>Caenorhabditis elegans</em> by CRISPR-targeted homologous recombination.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">e193</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkt805</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt805%26rft_id%253Dinfo%253Apmid%252F24013562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt805&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24013562&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.6" data-doi="10.1534/genetics.113.155879"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chiu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antoshechkin</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sternberg</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Transgene-free genome editing in <em>C. elegans</em> using CRISPR-Cas9.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1167</span>–<span class="cit-lpage">1171</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.155879</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155879%26rft_id%253Dinfo%253Apmid%252F23979577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS8zLzExNjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzEvMTM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.7" data-doi="10.1534/genetics.113.155853"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Heritable gene knockout in <em>C. elegans</em> by direct injection of Cas9-sgRNA ribonucleoproteins.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1177</span>–<span class="cit-lpage">1180</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.155853</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155853%26rft_id%253Dinfo%253Apmid%252F23979576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS8zLzExNzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzEvMTM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.8" data-doi="10.1016/j.celrep.2017.12.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dejima</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hori</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suehiro</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshina</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">An aneuploidy-free and structurally defined balancer chromosome toolkit for <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">232</span>–<span class="cit-lpage">241</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.celrep.2017.12.024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D22%26rft.spage%253D232%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2017.12.024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2017.12.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.9" data-doi="10.1073/pnas.0904895106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denver</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dolan</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilhelm</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucas-Lledo</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A genome-wide view of <em>Caenorhabditis elegans</em> base-substitution mutation processes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">16310</span>–<span class="cit-lpage">16314</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0904895106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0904895106%26rft_id%253Dinfo%253Apmid%252F19805298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzM4LzE2MzEwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8xLzEzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.10" data-doi="10.1093/nar/gkt135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DiCarlo</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norville</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mali</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rios</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aach</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome engineering n <em>S. cerevisiae</em> using CRISPR-Cas systems.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">4336</span>–<span class="cit-lpage">4343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt135%26rft_id%253Dinfo%253Apmid%252F23460208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23460208&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318167900046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.11" data-doi="10.1038/nmeth.2641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dickinson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reiner</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Engineering the <em>Caenorhabditis elegans</em> genome using Cas9-triggered homologous recombination.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1028</span>–<span class="cit-lpage">1034</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2641</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D1028%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2641%26rft_id%253Dinfo%253Apmid%252F23995389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23995389&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325073800029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.12" data-doi="10.1534/genetics.115.178335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dickinson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pani</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heppert</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Streamlined genome engineering with a self-excising drug selection cassette.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">1035</span>–<span class="cit-lpage">1049</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.178335</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.178335%26rft_id%253Dinfo%253Apmid%252F26044593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMC80LzEwMzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzEvMTM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.13" data-doi="10.1534/genetics.115.175166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Farboud</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Dramatic enhancement of genome editing by CRISPR/Cas9 through improved guide RNA design.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">959</span>–<span class="cit-lpage">971</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.175166</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.175166%26rft_id%253Dinfo%253Apmid%252F25695951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzQvOTU5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8xLzEzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.14" data-doi="10.1186/1471-2164-9-497"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flibotte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moerman</span>  <span class="cit-name-given-names">D. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Experimental analysis of oligonucleotide microarray design criteria to detect deletions by comparative genomic hybridization.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">497</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-9-497</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DFlibotte%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.spage%253D497%26rft.epage%253D497%26rft.atitle%253DExperimental%2Banalysis%2Bof%2Boligonucleotide%2Bmicroarray%2Bdesign%2Bcriteria%2Bto%2Bdetect%2Bdeletions%2Bby%2Bcomparative%2Bgenomic%2Bhybridization.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-497%26rft_id%253Dinfo%253Apmid%252F18940006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-497&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18940006&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.15" data-doi="10.1093/nar/gku1102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">B. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Artiles</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nonet</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fire</span>  <span class="cit-name-given-names">A. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Landscape of target:guide homology effects on Cas9-mediated cleavage.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">13778</span>–<span class="cit-lpage">13787</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gku1102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1102%26rft_id%253Dinfo%253Apmid%252F25399416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25399416&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.16" data-doi="10.1038/nmeth.2532"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedland</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tzur</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evelt</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colaiacovo</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Heritable genome editing in <em>C. elegans</em> via a CRISPR-Cas9 system.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">741</span>–<span class="cit-lpage">743</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2532</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D741%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2532%26rft_id%253Dinfo%253Apmid%252F23817069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2532&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23817069&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322453600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.17" data-doi="10.1073/pnas.1208507109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gasiunas</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrangou</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siksnys</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Cas9-crRNA ribonucleoprotein complex mediates specific DNA cleavage for adaptive immunity in bacteria.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">E2579</span>–<span class="cit-lpage">E2586</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1208507109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1208507109%26rft_id%253Dinfo%253Apmid%252F22949671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM5L0UyNTc5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8xLzEzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.18" data-doi="10.1038/nmeth.1318"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chuang</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venter</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutchinson</span>  <span class="cit-name-given-names">C. A.</span>  <span class="cit-name-suffix">3rd</span>.</span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Enzymatic assembly of DNA molecules up to several hundred kilobases.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1318</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DGibson%26rft.auinit1%253DD.%2BG.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D343%26rft.epage%253D345%26rft.atitle%253DEnzymatic%2Bassembly%2Bof%2BDNA%2Bmolecules%2Bup%2Bto%2Bseveral%2Bhundred%2Bkilobases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1318%26rft_id%253Dinfo%253Apmid%252F19363495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1318&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19363495&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265661600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.19" data-doi="10.1016/j.tibtech.2014.12.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hendel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fine</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porteus</span>  <span class="cit-name-given-names">M. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Quantifying on- and off-target genome editing.</span> <abbr class="cit-jnl-abbrev">Trends Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">132</span>–<span class="cit-lpage">140</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tibtech.2014.12.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D132%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2014.12.001%26rft_id%253Dinfo%253Apmid%252F25595557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2014.12.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25595557&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.20" data-doi="10.1534/g3.115.024257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Itani</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flibotte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dumas</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moerman</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Chromoanasynthetic genomic rearrangement identified in a N-ethyl-N-nitrosourea (ENU) mutagenesis screen in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">356</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.024257</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.024257%26rft_id%253Dinfo%253Apmid%252F26628482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjYvMi8zNTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzEvMTM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.21" data-doi="10.21775/cimb.026.065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jamal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ullah</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahsan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tyagi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Habib</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Improving CRISPR-Cas9 on-target specificity.</span> <abbr class="cit-jnl-abbrev">Curr. Issues Mol. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">80</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.21775/cimb.026.065</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BIssues%2BMol.%2BBiol.%26rft.volume%253D26%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.21775%252Fcimb.026.065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.21775/cimb.026.065&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.22" data-doi="10.1038/nbt.2508"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bikard</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marraffini</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RNA-guided editing of bacterial genomes using CRISPR-Cas systems.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">239</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.2508</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D233%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2508%26rft_id%253Dinfo%253Apmid%252F23360965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2508&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23360965&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.23" data-doi="10.1126/science.1225829"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jinek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chylinski</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fonfara</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doudna</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">816</span>–<span class="cit-lpage">821</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1225829</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1225829%26rft_id%253Dinfo%253Apmid%252F22745249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzcvNjA5Ni84MTYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzEvMTM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.24" data-doi="10.1007/978-1-59745-202-1_10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kadandale</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chatterjee</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Germline transformation of <em>Caenorhabditis elegans</em> by injection.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">518</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">133</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-59745-202-1_10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DKadandale%26rft.auinit1%253DP.%26rft.volume%253D518%26rft.spage%253D123%26rft.epage%253D133%26rft.atitle%253DGermline%2Btransformation%2Bof%2BCaenorhabditis%2Belegans%2Bby%2Binjection.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-59745-202-1_10%26rft_id%253Dinfo%253Apmid%252F19085141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-59745-202-1_10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19085141&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.1.135.25" data-doi="10.1534/genetics.113.155754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katic</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Großhans</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Targeted heritable mutation and gene conversion by Cas9-CRISPR in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1173</span>–<span class="cit-lpage">1176</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.155754</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155754%26rft_id%253Dinfo%253Apmid%252F23979578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS8zLzExNzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzEvMTM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.26" data-doi="10.1101/gr.171322.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Highly efficient RNA-guided genome editing in human cells via delivery of purified Cas9 ribonucleoproteins.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1012</span>–<span class="cit-lpage">1019</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.171322.113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.171322.113%26rft_id%253Dinfo%253Apmid%252F24696461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzYvMTAxMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMS8xMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.27" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.28" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.29" data-doi="10.1534/genetics.113.155382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pickle</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ralston</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurling</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Precise and heritable genome editing in evolutionarily diverse nematodes using TALENs and CRISPR/Cas9 to engineer insertions and deletions.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">348</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.155382</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155382%26rft_id%253Dinfo%253Apmid%252F23934893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzIvMzMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8xLzEzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-thesis" id="cit-9.1.135.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Markham</span>  <span class="cit-name-given-names">N. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> Hybrid: A software system for nucleic acid folding, hybridizing and melting predictions. Masters thesis. Rensselaer Polytechnic Institute, Troy, NY.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.31" data-doi="10.1111/febs.13777"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohr</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewen-Campen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Housden</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viswanatha</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">CRISPR guide RNA design for research applications.</span> <abbr class="cit-jnl-abbrev">FEBS J.</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">3232</span>–<span class="cit-lpage">3238</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/febs.13777</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BJ.%26rft.volume%253D283%26rft.spage%253D3232%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ffebs.13777%26rft_id%253Dinfo%253Apmid%252F27276584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/febs.13777&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27276584&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.32" data-doi="10.1534/genetics.115.180679"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Norris</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colaiácovo</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calarco</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Efficient genome editing in <em>C. elegans</em> with a toolkit of dual-marker selection cassettes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">458</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.180679</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.180679%26rft_id%253Dinfo%253Apmid%252F26232410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzIvNDQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8xLzEzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.33" data-doi="10.1534/genetics.114.170423"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paix</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calidas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Scalable and versatile genome editing using linear DNAs with microhomology to Cas9 sites in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1347</span>–<span class="cit-lpage">1356</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.170423</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.170423%26rft_id%253Dinfo%253Apmid%252F25249454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzEzNDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzEvMTM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.34" data-doi="10.1093/nar/gkw502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paix</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seydoux</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Cas9-assisted recombineering in <em>C. elegans</em>: genome editing using in vivo assembly of linear DNAs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">e128</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw502</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw502%26rft_id%253Dinfo%253Apmid%252F27257074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw502&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27257074&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.35" data-doi="10.1073/pnas.1405500111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Port</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bullock</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Optimized CRISPR/Cas tools for efficient germline and somatic genome engineering in Drosophila.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">E2967</span>–<span class="cit-lpage">E2976</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1405500111%26rft_id%253Dinfo%253Apmid%252F25002478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzI5L0UyOTY3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8xLzEzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.36" data-doi="10.1101/gr.171264.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramakrishna</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwaku Dad</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beloor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gopalappa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Gene disruption by cell-penetrating peptide-mediated delivery of Cas9 protein and guide RNA.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1020</span>–<span class="cit-lpage">1027</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.171264.113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.171264.113%26rft_id%253Dinfo%253Apmid%252F24696462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzYvMTAyMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMS8xMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.37" data-doi="10.1038/nbt.1754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winckler</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integrative genomics viewer.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.1754</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D26%26rft.atitle%253DIntegrative%2Bgenomics%2Bviewer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1754%26rft_id%253Dinfo%253Apmid%252F21221095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21221095&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286048900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.38" data-doi="10.1038/nbt.2842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sander</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joung</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">CRISPR-Cas systems for editing, regulating and targeting genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">355</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.2842</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D32%26rft.spage%253D347%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2842%26rft_id%253Dinfo%253Apmid%252F24584096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24584096&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.39" data-doi="10.1093/bib/bbs017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mesirov</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Integrative genomics viewer (IGV): high-performance genomics data visualization and exploration.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">192</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bib/bbs017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbs017%26rft_id%253Dinfo%253Apmid%252F22517427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbs017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22517427&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.40" data-doi="10.1534/genetics.113.156075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tzur</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedland</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nadarajan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calarco</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Heritable custom genomic modifications in <em>C. elegans</em> via a CRISPR-Cas9 system.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1185</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.156075</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.156075%26rft_id%253Dinfo%253Apmid%252F23979579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS8zLzExODEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzEvMTM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.41" data-doi="10.1038/ncomms8394"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Schendel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roerink</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Portegijs</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van den Heuvel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tijsterman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Polymerase Θ is a key driver of genome evolution and of CRISPR/Cas9-mediated mutagenesis.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">7394</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms8394</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D7394%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms8394%26rft_id%253Dinfo%253Apmid%252F26077599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms8394&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26077599&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.42" data-doi="10.1534/genetics.113.156299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waaijers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Portegijs</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerver</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemmens</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tijsterman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">CRISPR/Cas9-targeted mutagenesis in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1187</span>–<span class="cit-lpage">1191</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.156299</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.156299%26rft_id%253Dinfo%253Apmid%252F23979586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS8zLzExODciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzEvMTM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.43" data-doi="10.1007/s40484-014-0030-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Target specificity of the CRISPR-Cas9 system.</span> <abbr class="cit-jnl-abbrev">Quant. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">70</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s40484-014-0030-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuant.%2BBiol.%26rft.volume%253D2%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs40484-014-0030-x%26rft_id%253Dinfo%253Apmid%252F25722925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s40484-014-0030-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25722925&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.44" data-doi="10.1016/j.cell.2013.08.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shivalila</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">One-step generation of mice carrying reporter and conditional alleles by CRISPR/Cas-mediated genome engineering.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">1370</span>–<span class="cit-lpage">1379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D154%26rft.spage%253D1370%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.08.022%26rft_id%253Dinfo%253Apmid%252F23992847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.08.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23992847&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000324239300023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.135.45" data-doi="10.1038/cr.2014.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ke</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yue</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Oligonucleotide- based targeted gene editing in C. elegans via the CRISPR/Cas9 system.</span> <abbr class="cit-jnl-abbrev">Cell Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes.%26rft.volume%253D24%26rft.spage%253D247%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2014.9%26rft_id%253Dinfo%253Apmid%252F24418757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2014.9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24418757&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/1/135.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/1/125" title="Genotype Imputation in Winter Wheat Using First-Generation Haplotype Map SNPs Improves Genome-Wide Association Mapping and Genomic Prediction of Traits" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/1/145" title="Domain Requirements and Genetic Interactions of the Mud1 Subunit of the Saccharomyces cerevisiae U1 snRNP" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/1"><span>Volume 9 Issue 1, January 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/1.cover-source.jpg?itok=uUntglLo" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/1/135" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jttoGDRNr-I8AYyYPZLI5WmTtQ13Odf_x6-6gbAEeRg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/1/135" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/1/135" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/1/135" class="active" data-icon-position="" data-hide-link-title="0">CRISPR/Cas9 Methodology for the Generation of Knockout Deletions in Caenorhabditis elegans</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491787" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-yaefqIKQBhzFWCOzEt_ew3NNHnB4o4EdvC06nz9LReo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/1/135.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/1/135" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-hlC_92tGyPGy_cuwUqgPCOHTENGzZGjpSTgpL1I3IoU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/1/135" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/1/135.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491787" id="node491787--2" data-pisa="ggg;9/1/135" data-pisa-master="ggg;g3.118.200778" data-apath="/ggg/9/1/135.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">CRISPR/Cas9 Methodology for the Generation of Knockout Deletions in <em>Caenorhabditis elegans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3587-6200" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vinci Au</span>, <span class="highwire-citation-author" data-delta="1">Erica Li-Leger</span>, <span class="highwire-citation-author" data-delta="2">Greta Raymant</span>, <span class="highwire-citation-author" data-delta="3">Stephane Flibotte</span>, <span class="highwire-citation-author" data-delta="4">George Chen</span>, <span class="highwire-citation-author" data-delta="5">Kiana Martin</span>, <span class="highwire-citation-author" data-delta="6">Lisa Fernando</span>, <span class="highwire-citation-author" data-delta="7">Claudia Doell</span>, <span class="highwire-citation-author" data-delta="8">Federico I. Rosell</span>, <span class="highwire-citation-author" data-delta="9">Su Wang</span>, <span class="highwire-citation-author" data-delta="10">Mark L. Edgley</span>, <span class="highwire-citation-author" data-delta="11">Ann E. Rougvie</span>, <span class="highwire-citation-author" data-delta="12">Harald Hutter</span> and <span class="highwire-citation-author" data-delta="13">Donald G. Moerman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">135-144;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200778 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491787--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vinci Au </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vinci%2BAu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Au%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVinci%2BAu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3587-6200" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vinci Au</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Erica Li-Leger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erica%2BLi-Leger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li-Leger%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErica%2BLi-Leger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Greta Raymant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Greta%2BRaymant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raymant%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGreta%2BRaymant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Stephane Flibotte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephane%2BFlibotte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Flibotte%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephane%2BFlibotte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">George Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=George%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeorge%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kiana Martin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kiana%2BMartin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martin%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKiana%2BMartin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Lisa Fernando </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lisa%2BFernando%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernando%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALisa%2BFernando%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Claudia Doell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claudia%2BDoell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doell%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaudia%2BDoell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Federico I. Rosell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Molecular Biology, University of British Columbia, Vancouver, British Columbia, Canada, V6T 1Z4</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Federico%2BI.%2BRosell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rosell%20FI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFederico%2BI.%2BRosell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Su Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Su%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASu%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Mark L. Edgley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BL.%2BEdgley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edgley%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BL.%2BEdgley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Ann E. Rougvie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics, Cell Biology, and Development, University of Minnesota, Minneapolis, MN 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ann%2BE.%2BRougvie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rougvie%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnn%2BE.%2BRougvie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Harald Hutter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biological Sciences, Simon Fraser University, Burnaby, British Columbia, Canada, V5A 1S6</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harald%2BHutter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hutter%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarald%2BHutter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Donald G. Moerman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Donald%2BG.%2BMoerman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moerman%20DG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADonald%2BG.%2BMoerman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:moerman@zoology.ubc.ca" class="" data-icon-position="" data-hide-link-title="0">moerman@zoology.ubc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/1/135" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/1/135&amp;title=CRISPR/Cas9%20Methodology%20for%20the%20Generation%20of%20Knockout%20Deletions%20in%20Caenorhabditis%20elegans" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/1/135&amp;title=CRISPR/Cas9%20Methodology%20for%20the%20Generation%20of%20Knockout%20Deletions%20in%20Caenorhabditis%20elegans" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/1/135&amp;title=CRISPR/Cas9%20Methodology%20for%20the%20Generation%20of%20Knockout%20Deletions%20in%20Caenorhabditis%20elegans" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/1/135&amp;text=CRISPR/Cas9%20Methodology%20for%20the%20Generation%20of%20Knockout%20Deletions%20in%20Caenorhabditis%20elegans" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/1/135&amp;title=CRISPR/Cas9%20Methodology%20for%20the%20Generation%20of%20Knockout%20Deletions%20in%20Caenorhabditis%20elegans" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/1/135&amp;t=CRISPR/Cas9%20Methodology%20for%20the%20Generation%20of%20Knockout%20Deletions%20in%20Caenorhabditis%20elegans" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/1/135&amp;title=CRISPR/Cas9%20Methodology%20for%20the%20Generation%20of%20Knockout%20Deletions%20in%20Caenorhabditis%20elegans" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/1/135&amp;title=CRISPR/Cas9%20Methodology%20for%20the%20Generation%20of%20Knockout%20Deletions%20in%20Caenorhabditis%20elegans" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491787/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491787" id="node491787--3" data-pisa="ggg;9/1/135" data-pisa-master="ggg;g3.118.200778" data-apath="/ggg/9/1/135.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">CRISPR/Cas9 Methodology for the Generation of Knockout Deletions in <em>Caenorhabditis elegans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3587-6200" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vinci Au</span>, <span class="highwire-citation-author" data-delta="1">Erica Li-Leger</span>, <span class="highwire-citation-author" data-delta="2">Greta Raymant</span>, <span class="highwire-citation-author" data-delta="3">Stephane Flibotte</span>, <span class="highwire-citation-author" data-delta="4">George Chen</span>, <span class="highwire-citation-author" data-delta="5">Kiana Martin</span>, <span class="highwire-citation-author" data-delta="6">Lisa Fernando</span>, <span class="highwire-citation-author" data-delta="7">Claudia Doell</span>, <span class="highwire-citation-author" data-delta="8">Federico I. Rosell</span>, <span class="highwire-citation-author" data-delta="9">Su Wang</span>, <span class="highwire-citation-author" data-delta="10">Mark L. Edgley</span>, <span class="highwire-citation-author" data-delta="11">Ann E. Rougvie</span>, <span class="highwire-citation-author" data-delta="12">Harald Hutter</span> and <span class="highwire-citation-author" data-delta="13">Donald G. Moerman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">135-144;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200778 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491787--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vinci Au </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vinci%2BAu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Au%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVinci%2BAu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3587-6200" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vinci Au</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Erica Li-Leger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erica%2BLi-Leger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li-Leger%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErica%2BLi-Leger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Greta Raymant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Greta%2BRaymant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raymant%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGreta%2BRaymant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Stephane Flibotte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephane%2BFlibotte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Flibotte%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephane%2BFlibotte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">George Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=George%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeorge%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kiana Martin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kiana%2BMartin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martin%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKiana%2BMartin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Lisa Fernando </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lisa%2BFernando%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernando%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALisa%2BFernando%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Claudia Doell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claudia%2BDoell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doell%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaudia%2BDoell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Federico I. Rosell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Molecular Biology, University of British Columbia, Vancouver, British Columbia, Canada, V6T 1Z4</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Federico%2BI.%2BRosell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rosell%20FI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFederico%2BI.%2BRosell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Su Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Su%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASu%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Mark L. Edgley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BL.%2BEdgley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edgley%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BL.%2BEdgley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Ann E. Rougvie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics, Cell Biology, and Development, University of Minnesota, Minneapolis, MN 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ann%2BE.%2BRougvie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rougvie%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnn%2BE.%2BRougvie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Harald Hutter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biological Sciences, Simon Fraser University, Burnaby, British Columbia, Canada, V5A 1S6</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harald%2BHutter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hutter%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarald%2BHutter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Donald G. Moerman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology and Michael Smith Laboratories</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Donald%2BG.%2BMoerman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moerman%20DG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADonald%2BG.%2BMoerman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:moerman@zoology.ubc.ca" class="" data-icon-position="" data-hide-link-title="0">moerman@zoology.ubc.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=135&amp;rft.epage=144&amp;rft.atitle=CRISPR/Cas9%20Methodology%20for%20the%20Generation%20of%20Knockout%20Deletions%20in%20Caenorhabditis%20elegans&amp;rft.volume=9&amp;rft.issue=1&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Au%2C+Vinci&amp;rft.au=Li-Leger%2C+Erica&amp;rft.au=Raymant%2C+Greta&amp;rft.au=Flibotte%2C+Stephane&amp;rft.au=Chen%2C+George&amp;rft.au=Martin%2C+Kiana&amp;rft.au=Fernando%2C+Lisa&amp;rft.au=Doell%2C+Claudia&amp;rft.au=Rosell%2C+Federico+I.&amp;rft.au=Wang%2C+Su&amp;rft.au=Edgley%2C+Mark+L.&amp;rft.au=Rougvie%2C+Ann+E.&amp;rft.au=Hutter%2C+Harald&amp;rft.au=Moerman%2C+Donald+G."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491787/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491787/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491787/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491787/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491787/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491787/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491787/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491787/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491787/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491787/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491787/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491787/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/1/135" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/1/135.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/1/135.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/1/135.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/1/135.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
