<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/1/1" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/1/1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/1/1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/1/1.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/1/1.atom" />
<meta name="HW.pisa" content="ggg;9/1/1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="iProteinDB: An Integrative Database of Drosophila Post-translational Modifications" />
<meta name="DC.Identifier" content="10.1534/g3.118.200637" />
<meta name="DC.Date" content="2019-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Hu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.6986945" />
<meta name="DC.Description" content="Post-translational modification (PTM) serves as a regulatory mechanism for protein function, influencing their stability, interactions, activity and localization, and is critical in many signaling pathways. The best characterized PTM is phosphorylation, whereby a phosphate is added to an acceptor residue, most commonly serine, threonine and tyrosine in metazoans. As proteins are often phosphorylated at multiple sites, identifying those sites that are important for function is a challenging problem. Considering that any given phosphorylation site might be non-functional, prioritizing evolutionarily conserved phosphosites provides a general strategy to identify the putative functional sites. To facilitate the identification of conserved phosphosites, we generated a large-scale phosphoproteomics dataset from Drosophila embryos collected from six closely-related species. We built iProteinDB (&lt;https://www.flyrnai.org/tools/iproteindb/&gt;), a resource integrating these data with other high-throughput PTM datasets, including vertebrates, and manually curated information for Drosophila . At iProteinDB, scientists can view the PTM landscape for any Drosophila protein and identify predicted functional phosphosites based on a comparative analysis of data from closely-related Drosophila species. Further, iProteinDB enables comparison of PTM data from Drosophila to that of orthologous proteins from other model organisms, including human, mouse, rat, Xenopus tropicalis , Danio rerio , and Caenorhabditis elegans ." />
<meta name="DC.Contributor" content="Yanhui Hu" />
<meta name="DC.Contributor" content="Richelle Sopko" />
<meta name="DC.Contributor" content="Verena Chung" />
<meta name="DC.Contributor" content="Marianna Foos" />
<meta name="DC.Contributor" content="Romain A. Studer" />
<meta name="DC.Contributor" content="Sean D. Landry" />
<meta name="DC.Contributor" content="Daniel Liu" />
<meta name="DC.Contributor" content="Leonard Rabinow" />
<meta name="DC.Contributor" content="Florian Gnad" />
<meta name="DC.Contributor" content="Pedro Beltrao" />
<meta name="DC.Contributor" content="Norbert Perrimon" />
<meta name="article:published_time" content="2019-01-01" />
<meta name="article:section" content="Software and Data Resources" />
<meta name="citation_title" content="iProteinDB: An Integrative Database of Drosophila Post-translational Modifications" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Post-translational modification (PTM) serves as a regulatory mechanism for protein function, influencing their stability, interactions, activity and localization, and is critical in many signaling pathways. The best characterized PTM is phosphorylation, whereby a phosphate is added to an acceptor residue, most commonly serine, threonine and tyrosine in metazoans. As proteins are often phosphorylated at multiple sites, identifying those sites that are important for function is a challenging problem. Considering that any given phosphorylation site might be non-functional, prioritizing evolutionarily conserved phosphosites provides a general strategy to identify the putative functional sites. To facilitate the identification of conserved phosphosites, we generated a large-scale phosphoproteomics dataset from &lt;i&gt;Drosophila&lt;/i&gt; embryos collected from six closely-related species. We built iProteinDB (https://www.flyrnai.org/tools/iproteindb/), a resource integrating these data with other high-throughput PTM datasets, including vertebrates, and manually curated information for &lt;i&gt;Drosophila&lt;/i&gt;. At iProteinDB, scientists can view the PTM landscape for any &lt;i&gt;Drosophila&lt;/i&gt; protein and identify predicted functional phosphosites based on a comparative analysis of data from closely-related &lt;i&gt;Drosophila&lt;/i&gt; species. Further, iProteinDB enables comparison of PTM data from &lt;i&gt;Drosophila&lt;/i&gt; to that of orthologous proteins from other model organisms, including human, mouse, rat, &lt;i&gt;Xenopus tropicalis&lt;/i&gt;, &lt;i&gt;Danio rerio&lt;/i&gt;, and &lt;i&gt;Caenorhabditis elegans&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="ggg;9/1/1" />
<meta name="citation_id" content="9/1/1" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/1/1" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/1/1.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/1/1.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/1/1.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200637" />
<meta name="citation_pmid" content="30397019" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Software and Data Resources" />
<meta name="citation_firstpage" content="1" />
<meta name="citation_lastpage" content="11" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yanhui Hu" />
<meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115" />
<meta name="citation_author_institution" content="Drosophila RNAi Screening Center, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1494-1402" />
<meta name="citation_author" content="Richelle Sopko" />
<meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115" />
<meta name="citation_author" content="Verena Chung" />
<meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115" />
<meta name="citation_author_institution" content="Drosophila RNAi Screening Center, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115" />
<meta name="citation_author" content="Marianna Foos" />
<meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115" />
<meta name="citation_author" content="Romain A. Studer" />
<meta name="citation_author_institution" content="European Molecular Biology Laboratory (EMBL), European Bioinformatics Institute, Wellcome Genome Campus, Hinxton, Cambridge CB10 1SD, UK" />
<meta name="citation_author" content="Sean D. Landry" />
<meta name="citation_author_institution" content="Department of Bioinformatics, Cell Signaling Technology Inc., 3 Trask Lane, Danvers, MA 01923" />
<meta name="citation_author" content="Daniel Liu" />
<meta name="citation_author_institution" content="Drosophila RNAi Screening Center, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115" />
<meta name="citation_author" content="Leonard Rabinow" />
<meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115" />
<meta name="citation_author" content="Florian Gnad" />
<meta name="citation_author_institution" content="Department of Bioinformatics, Cell Signaling Technology Inc., 3 Trask Lane, Danvers, MA 01923" />
<meta name="citation_author" content="Pedro Beltrao" />
<meta name="citation_author_institution" content="European Molecular Biology Laboratory (EMBL), European Bioinformatics Institute, Wellcome Genome Campus, Hinxton, Cambridge CB10 1SD, UK" />
<meta name="citation_author" content="Norbert Perrimon" />
<meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115" />
<meta name="citation_author_institution" content="Drosophila RNAi Screening Center, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115" />
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, 77 Avenue Louis Pasteur, Boston, MA 02115" />
<meta name="citation_author_email" content="perrimon@receptor.med.harvard.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7542-472X" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. M. Altenhoff;citation_author=N. M. Glover;citation_author=C. M. Train;citation_author=K. Kaleb;citation_author=A. Warwick Vesztrocy;citation_title=The OMA orthology database in 2018: retrieving evolutionary relationships among all domains of life through richer web and programmatic interfaces.;citation_pages=D477-D485;citation_volume=46;citation_year=2018;citation_doi=10.1093/nar/gkx1019" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. M. Altenhoff;citation_author=A. Schneider;citation_author=G. H. Gonnet;citation_author=C. Dessimoz;citation_title=OMA 2011: orthology inference among 1000 complete genomes.;citation_volume=39;citation_year=2011;citation_pmid=21113020;citation_doi=10.1093/nar/gkq1238" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. M. Altenhoff;citation_author=N. Skunca;citation_author=N. Glover;citation_author=C. M. Train;citation_author=A. Sueki;citation_title=The OMA orthology database in 2015: function predictions, better plant support, synteny view and other improvements.;citation_volume=43;citation_year=2015;citation_pmid=25399418;citation_doi=10.1093/nar/gku1158" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=S. A. Beausoleil;citation_author=J. Villen;citation_author=S. A. Gerber;citation_author=J. Rush;citation_author=S. P. Gygi;citation_title=A probability-based approach for high-throughput protein phosphorylation analysis and site localization.;citation_pages=1285-1292;citation_volume=24;citation_year=2006;citation_issue=10;citation_pmid=16964243;citation_doi=10.1038/nbt1240" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=P. Beltrao;citation_author=P. Bork;citation_author=N. J. Krogan;citation_author=V. van Noort;citation_title=Evolution and functional cross-talk of protein post-translational modifications;citation_pages=714-714;citation_volume=9;citation_year=2013;citation_issue=1;citation_pmid=24366814;citation_doi=10.1002/msb.201304521" />
<meta name="citation_reference" content="citation_journal_title=Pharmacol. Ther.;citation_author=E. Beurel;citation_author=S. F. Grieco;citation_author=R. S. Jope;citation_title=Glycogen synthase kinase-3 (GSK3): regulation, actions, and diseases.;citation_pages=114-131;citation_volume=148;citation_year=2015;citation_pmid=25435019;citation_doi=10.1016/j.pharmthera.2014.11.016" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=B. Bodenmiller;citation_author=D. Campbell;citation_author=B. Gerrits;citation_author=H. Lam;citation_author=M. Jovanovic;citation_title=PhosphoPep--a database of protein phosphorylation sites in model organisms.;citation_pages=1339-1340;citation_volume=26;citation_year=2008;citation_issue=12;citation_pmid=19060867;citation_doi=10.1038/nbt1208-1339" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=B. Bodenmiller;citation_author=J. Malmstrom;citation_author=B. Gerrits;citation_author=D. Campbell;citation_author=H. Lam;citation_title=PhosphoPep--a phosphoproteome resource for systems biology research in Drosophila Kc167 cells;citation_pages=139-139;citation_volume=3;citation_year=2007;citation_issue=1;citation_pmid=17940529;citation_doi=10.1038/msb4100182" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=N. Casas-Vila;citation_author=A. Bluhm;citation_author=S. Sayols;citation_author=N. Dinges;citation_author=M. Dejung;citation_title=The developmental proteome of Drosophila melanogaster.;citation_volume=27;citation_year=2017;citation_pmid=28381612;citation_doi=10.1101/gr.213694.116" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=F. Diella;citation_author=S. Cameron;citation_author=C. Gemund;citation_author=R. Linding;citation_author=A. Via;citation_title=Phospho.ELM: a database of experimentally verified phosphorylation sites in eukaryotic proteins.;citation_pages=79-79;citation_volume=5;citation_year=2004;citation_pmid=15212693;citation_doi=10.1186/1471-2105-5-79" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=F. Diella;citation_author=C. M. Gould;citation_author=C. Chica;citation_author=A. Via;citation_author=T. J. Gibson;citation_title=Phospho.ELM: a database of phosphorylation sites–update 2008.;citation_volume=36;citation_year=2008;citation_pmid=17962309;citation_doi=10.1093/nar/gkm772" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=H. Dinkel;citation_author=C. Chica;citation_author=A. Via;citation_author=C. M. Gould;citation_author=L. J. Jensen;citation_title=Phospho.ELM: a database of phosphorylation sites–update 2011.;citation_volume=39;citation_year=2011;citation_pmid=21062810;citation_doi=10.1093/nar/gkq1104" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=L. Gabay;citation_author=R. Seger;citation_author=B. Z. Shilo;citation_title=In Situ Activation Pattern of Drosophila EGF Receptor Pathway During Development;citation_pages=1103-1106;citation_volume=277;citation_year=1997;citation_issue=5329;citation_pmid=9262480;citation_doi=10.1126/science.277.5329.1103" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Proteomics;citation_author=F. Gnad;citation_author=F. Forner;citation_author=D. F. Zielinska;citation_author=E. Birney;citation_author=J. Gunawardena;citation_title=Evolutionary constraints of phosphorylation in eukaryotes, prokaryotes, and mitochondria.;citation_volume=9;citation_year=2010;citation_pmid=20688971;citation_doi=10.1074/mcp.M110.001594" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=F. Gnad;citation_author=J. Gunawardena;citation_author=M. Mann;citation_title=PHOSIDA 2011: the posttranslational modification database.;citation_volume=39;citation_year=2011;citation_pmid=21081558;citation_doi=10.1093/nar/gkq1159" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=F. Gnad;citation_author=S. Ren;citation_author=J. Cox;citation_author=J. V. Olsen;citation_author=B. Macek;citation_title=PHOSIDA (phosphorylation site database): management, structural and evolutionary investigation, and prediction of phosphosites.;citation_pages=R250-R250;citation_volume=8;citation_year=2007;citation_issue=11;citation_pmid=18039369;citation_doi=10.1186/gb-2007-8-11-r250" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Proteomics;citation_author=M. Hilger;citation_author=T. Bonaldi;citation_author=F. Gnad;citation_author=M. Mann;citation_title=Systems-wide analysis of a phosphatase knock-down by quantitative proteomics and phosphoproteomics.;citation_volume=8;citation_year=2009;citation_pmid=19429919;citation_doi=10.1074/mcp.M800559-MCP200" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=H. Horn;citation_author=E. M. Schoof;citation_author=J. Kim;citation_author=X. Robin;citation_author=M. L. Miller;citation_title=KinomeXplorer: an integrated platform for kinome biology studies.;citation_pages=603-604;citation_volume=11;citation_year=2014;citation_pmid=24874572;citation_doi=10.1038/nmeth.2968" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=P. V. Hornbeck;citation_author=J. M. Kornhauser;citation_author=S. Tkachev;citation_author=B. Zhang;citation_author=E. Skrzypek;citation_title=PhosphoSitePlus: a comprehensive resource for investigating the structure and function of experimentally determined post-translational modifications in man and mouse.;citation_volume=40;citation_year=2012;citation_pmid=22135298;citation_doi=10.1093/nar/gkr1122" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=P. V. Hornbeck;citation_author=B. Zhang;citation_author=B. Murray;citation_author=J. M. Kornhauser;citation_author=V. Latham;citation_title=PhosphoSitePlus, 2014: mutations, PTMs and recalibrations.;citation_volume=43;citation_year=2015;citation_pmid=25514926;citation_doi=10.1093/nar/gku1267" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. C. Hsu;citation_author=N. Perrimon;citation_title=A temperature-sensitive MEK mutation demonstrates the conservation of the signaling pathways activated by receptor tyrosine kinases.;citation_pages=2176-2187;citation_volume=8;citation_year=1994;citation_issue=18;citation_pmid=7958887;citation_doi=10.1101/gad.8.18.2176" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=Y. Hu;citation_author=I. Flockhart;citation_author=A. Vinayagam;citation_author=C. Bergwitz;citation_author=B. Berger;citation_title=An integrative approach to ortholog prediction for disease-focused and other functional studies.;citation_pages=357-357;citation_volume=12;citation_year=2011;citation_pmid=21880147;citation_doi=10.1186/1471-2105-12-357" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Y. Huang;citation_author=M. G. Su;citation_author=H. J. Kao;citation_author=Y. C. Hsieh;citation_author=J. H. Jhong;citation_title=dbPTM 2016: 10-year anniversary of a resource for post-translational modification of proteins.;citation_volume=44;citation_year=2016;citation_pmid=26578568;citation_doi=10.1093/nar/gkv1240" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=T. Hunter;citation_title=Signaling--2000 and beyond.;citation_pages=113-127;citation_volume=100;citation_year=2000;citation_issue=1;citation_pmid=10647936;citation_doi=10.1016/S0092-8674(00)81688-8" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=L. M. Iakoucheva;citation_author=P. Radivojac;citation_author=C. J. Brown;citation_author=T. R. O’Connor;citation_author=J. G. Sikes;citation_title=The importance of intrinsic disorder for protein phosphorylation;citation_pages=1037-1049;citation_volume=32;citation_year=2004;citation_issue=3;citation_pmid=14960716;citation_doi=10.1093/nar/gkh253" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=K. Katoh;citation_author=K. Misawa;citation_author=K. Kuma;citation_author=T. Miyata;citation_title=MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform;citation_pages=3059-3066;citation_volume=30;citation_year=2002;citation_issue=14;citation_pmid=12136088;citation_doi=10.1093/nar/gkf436" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=L. Kockel;citation_author=K. S. Kerr;citation_author=M. Melnick;citation_author=K. Bruckner;citation_author=M. Hebrok;citation_title=Dynamic switch of negative feedback regulation in Drosophila Akt-TOR signaling.;citation_pages=e1000990-e1000990;citation_volume=6;citation_year=2010;citation_issue=6;citation_pmid=20585550;citation_doi=10.1371/journal.pgen.1000990" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=C. R. Landry;citation_author=E. D. Levy;citation_author=S. W. Michnick;citation_title=Weak functional constraints on phosphoproteomes.;citation_pages=193-197;citation_volume=25;citation_year=2009;citation_issue=5;citation_pmid=19349092;citation_doi=10.1016/j.tig.2009.03.003" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=T. Y. Lee;citation_author=H. D. Huang;citation_author=J. H. Hung;citation_author=H. Y. Huang;citation_author=Y. S. Yang;citation_title=dbPTM: an information repository of protein post-translational modification;citation_pages=D622-D627;citation_volume=34;citation_year=2006;citation_issue=suppl_1;citation_pmid=16381945;citation_doi=10.1093/nar/gkj083" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=I. Letunic;citation_author=P. Bork;citation_title=Interactive tree of life (iTOL) v3: an online tool for the display and annotation of phylogenetic and other trees.;citation_volume=44;citation_year=2016;citation_pmid=27095192;citation_doi=10.1093/nar/gkw290" />
<meta name="citation_reference" content="citation_journal_title=J. Alzheimers Dis.;citation_author=T. Ma;citation_title=GSK3 in Alzheimer’s disease: mind the isoforms.;citation_pages=707-710;citation_volume=39;citation_year=2014;citation_doi=10.3233/JAD-131661" />
<meta name="citation_reference" content="citation_journal_title=Trends in biochemical sciences;citation_journal_abbrev=Trends Biochem Sci;citation_author=G. Manning;citation_author=G. D. Plowman;citation_author=T. Hunter;citation_author=S. Sudarsanam;citation_title=Evolution of protein kinase signaling from yeast to man.;citation_pages=514-520;citation_volume=27;citation_year=2002;citation_issue=10;citation_pmid=12368087;citation_doi=10.1016/S0968-0004(02)02179-5" />
<meta name="citation_reference" content="citation_journal_title=Int. Rev. Cell Mol. Biol.;citation_author=O. Meyuhas;citation_title=Ribosomal Protein S6 Phosphorylation: Four Decades of Research.;citation_pages=41-73;citation_volume=320;citation_year=2015;citation_pmid=26614871;citation_doi=10.1016/bs.ircmb.2015.07.006" />
<meta name="citation_reference" content="citation_journal_title=Semin. Cancer Biol;citation_author=S. Nagini;citation_author=J. Sophia;citation_author=R. Mishra;citation_title=Glycogen synthase kinases: Moonlighting proteins with theranostic potential in cancer.;citation_year=2018;citation_doi=10.1016/j.semcancer.2017.12.010" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=J. C. Obenauer;citation_author=L. C. Cantley;citation_author=M. B. Yaffe;citation_title=Scansite 2.0: proteome-wide prediction of cell signaling interactions using short sequence motifs;citation_pages=3635-3641;citation_volume=31;citation_year=2003;citation_issue=13;citation_pmid=12824383;citation_doi=10.1093/nar/gkg584" />
<meta name="citation_reference" content="citation_journal_title=Drug Metab. Dispos.;citation_author=S. R. Pondugula;citation_author=C. Brimer-Cline;citation_author=J. Wu;citation_author=E. G. Schuetz;citation_author=R. K. Tyagi;citation_title=A phosphomimetic mutation at threonine-57 abolishes transactivation activity and alters nuclear localization pattern of human pregnane x receptor.;citation_volume=37;citation_year=2009;citation_pmid=19171678;citation_doi=10.1124/dmd.108.024695" />
<meta name="citation_reference" content="citation_journal_title=Epigenetics;citation_author=D. Sarkar;citation_author=E. Y. Leung;citation_author=B. C. Baguley;citation_author=G. J. Finlay;citation_author=M. E. Askarian-Amiri;citation_title=Epigenetic regulation in human melanoma: past and future.;citation_pages=103-121;citation_volume=10;citation_year=2015;citation_pmid=25587943;citation_doi=10.1080/15592294.2014.1003746" />
<meta name="citation_reference" content="citation_journal_title=Dev. Cell;citation_author=R. Sopko;citation_author=M. Foos;citation_author=A. Vinayagam;citation_author=B. Zhai;citation_author=R. Binari;citation_title=Combining genetic perturbations and proteomics to examine kinase-phosphatase networks in Drosophila embryos.;citation_pages=114-127;citation_volume=31;citation_year=2014;citation_pmid=25284370;citation_doi=10.1016/j.devcel.2014.07.027" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. A. Studer;citation_author=R. A. Rodriguez-Mias;citation_author=K. M. Haas;citation_author=J. I. Hsu;citation_author=C. Vieitez;citation_title=Evolution of protein phosphorylation across 18 fungal species;citation_pages=229-232;citation_volume=354;citation_year=2016;citation_issue=6309;citation_pmid=27738172;citation_doi=10.1126/science.aaf2144" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=S. Ullah;citation_author=S. Lin;citation_author=Y. Xu;citation_author=W. Deng;citation_author=L. Ma;citation_title=dbPAF: an integrative database of protein phosphorylation in animals and fungi.;citation_pages=23534;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep23534" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Waterhouse;citation_author=J. B. Procter;citation_author=D. M. Martin;citation_author=M. Clamp;citation_author=G. J. Barton;citation_title=Jalview Version 2–a multiple sequence alignment editor and analysis workbench.;citation_volume=25;citation_year=2009;citation_pmid=19151095;citation_doi=10.1093/bioinformatics/btp033" />
<meta name="citation_reference" content="citation_journal_title=Journal of proteome research;citation_journal_abbrev=J Proteome Res;citation_author=B. Zhai;citation_author=J. Villen;citation_author=S. A. Beausoleil;citation_author=J. Mintseris;citation_author=S. P. Gygi;citation_title=Phosphoproteome analysis of Drosophila melanogaster embryos.;citation_pages=1675-1682;citation_volume=7;citation_year=2008;citation_issue=4;citation_pmid=18327897;citation_doi=10.1021/pr700696a" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="iProteinDB: An Integrative Database of Drosophila Post-translational Modifications" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1.cover-source.jpg" />
<meta name="twitter:description" content="Post-translational modification (PTM) serves as a regulatory mechanism for protein function, influencing their stability, interactions, activity and localization, and is critical in many signaling pathways. The best characterized PTM is phosphorylation, whereby a phosphate is added to an acceptor residue, most commonly serine, threonine and tyrosine in metazoans. As proteins are often phosphorylated at multiple sites, identifying those sites that are important for function is a challenging problem. Considering that any given phosphorylation site might be non-functional, prioritizing evolutionarily conserved phosphosites provides a general strategy to identify the putative functional sites. To facilitate the identification of conserved phosphosites, we generated a large-scale phosphoproteomics dataset from Drosophila embryos collected from six closely-related species. We built iProteinDB (&lt;https://www.flyrnai.org/tools/iproteindb/&gt;), a resource integrating these data with other high-throughput PTM datasets, including vertebrates, and manually curated information for Drosophila . At iProteinDB, scientists can view the PTM landscape for any Drosophila protein and identify predicted functional phosphosites based on a comparative analysis of data from closely-related Drosophila species. Further, iProteinDB enables comparison of PTM data from Drosophila to that of orthologous proteins from other model organisms, including human, mouse, rat, Xenopus tropicalis , Danio rerio , and Caenorhabditis elegans ." />
<meta name="og-title" property="og:title" content="iProteinDB: An Integrative Database of Drosophila Post-translational Modifications" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/1/1" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Post-translational modification (PTM) serves as a regulatory mechanism for protein function, influencing their stability, interactions, activity and localization, and is critical in many signaling pathways. The best characterized PTM is phosphorylation, whereby a phosphate is added to an acceptor residue, most commonly serine, threonine and tyrosine in metazoans. As proteins are often phosphorylated at multiple sites, identifying those sites that are important for function is a challenging problem. Considering that any given phosphorylation site might be non-functional, prioritizing evolutionarily conserved phosphosites provides a general strategy to identify the putative functional sites. To facilitate the identification of conserved phosphosites, we generated a large-scale phosphoproteomics dataset from Drosophila embryos collected from six closely-related species. We built iProteinDB (&lt;https://www.flyrnai.org/tools/iproteindb/&gt;), a resource integrating these data with other high-throughput PTM datasets, including vertebrates, and manually curated information for Drosophila . At iProteinDB, scientists can view the PTM landscape for any Drosophila protein and identify predicted functional phosphosites based on a comparative analysis of data from closely-related Drosophila species. Further, iProteinDB enables comparison of PTM data from Drosophila to that of orthologous proteins from other model organisms, including human, mouse, rat, Xenopus tropicalis , Danio rerio , and Caenorhabditis elegans ." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/1.cover-source.jpg" />
    <title>iProteinDB: An Integrative Database of Drosophila Post-translational Modifications | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"gFUdN-SUT7sPRM43Hg0aWdL8-hWOa7jVq_Vx8z6ZbBc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491970","apath":"\/ggg\/9\/1\/1.atom","pisa":"ggg;9\/1\/1","ac":{"\/ggg\/9\/1\/1.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/1\/1.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/1\/1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/1\/1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/1\/1":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NGR3ltshcHMaxPt+ir0620VcB\\\\\\\/6bhDLwPo\\\\\\\/k7vbalF\\\\\\\/oGTVSAJPhZQ4elenCQfcA2DAjKhHSaG9Gqb9oXxq6D+ZGHcQ4rll2dofJb63lxQAhQHVf8nH6HIke0UVTzhNzQa6aEuqCknPT+QWtI5B5N4OhZbuJmVzsZk0Ouz47zHQOOyfyu4NzKTQAFpIol+BjAaPJ6xbRvY08M39oA3Jux\\\\\\\/VLoKB2nJ4KR9e\\\\\\\/\\\\\\\/9pQM0UJwEW7v47Cg6zILejp+GOmSIlSYWlyHW+PVmMyM89moIGHFMBWbULef0NMZKksBYTcvxftN586uMomyxQ+CmtmGw1KaDTi1swy5\\\\\\\/ZGzzMH74pIY8Yb\\\\\\\/x4E1C1gTOgyI5gsXpvNJVFc5XBR\\\\\\\/zMrMKhoPJieEg5y\\\\\\\/OVTnJAg9wmwzK4hez9rdqf32hbkOo95ltuJ4twdduE\\\\\\\/7ClAPnjdWVqsKcg6ovZ3OuwZU+l+V8+Y5qpn6pGHV3eHf9FZ6r69IgEFZ1VjdWJEisk09lJBXEe+bjlNaf8ffDTMi3xuwKaBaCoOtIYiq0NIJD8RgGRjh0KJb7Mhb4IlhfegUP+joVXcb+ijEIfEkugtbUY19CT3GqZGAPnPsrYkMIHRNi1U2R3k8rCUkmSC+ALMolfa7Kip5NAxsFloJDv7Kw5Oig2Igbq6ViZT6kRLxn5dRViHfh8ufwSICWeNdIgGU96Zc+IwC7D9TA6yoOggytdWKxoKYxOKzk8++uxXLvQS0Ri+fOcz\\\\\\\/gecEV1NpJspoitQyumT6dJKp2a+xOV2a9MjDY8Y1bl30xYMzov4zhVuRN3tLrjg0boMjCtb1+aPO4GPWJwhILa\\\\\\\/bmDMg0pXSpCoqAoVo1z+zuWPpy4RPwqniPIXpkTq2v\\\\\\\/wsRBD2NXaP12NT5mgMZaWoosSzxUZ20XqlW09SmjgKek4hmGff0psxWomz\\\\\\\/5+Ra6mG6Ig7pfvPCFgaLeWF384+63XUHAAT4uFjmyuQFNR\\\\\\\/HE+aLa6H4JGXfYsTgFAscJ7rBtQZvsIxDg5X1H3d0lE65dx6DJ+k4kYuRDHvDMjcYSnhCl0tcSfuTINy7rMizdmyTEE41VpY0iFRooPG1xT3vXZhVFi2mFTvc2nsH1j8OR\\\\\\\/Qlq2cJAyJDm8FFOflx1Mmj2I6I+1xS6SrBs3KUUvKk3M6BVNGud8QFGqmUee955yiCKmTxiCfsEqtTyBvKIqVJ0LXin6T0+LVuUmGF4lie+NotRRxpvuqmZFgP4CbbZZDNJN\\\\\\\/aYpMwXAV96+gPw1K2DZ8jqKaDHaOapn\\\u0022,\\\u0022iv\\\u0022:\\\u0022NM3HDNXnmHp5a+q6IdFRDA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d37c403a5ce432dd634dd8d4740923da141c4ac3de12fced88ddecd5d6539ee3\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229sSNPsknsbwnUQJmLFPFGRnbXcjPe0iAmMzFR75mZhMKR72XFgjMqLem4GzoVx4LLy\\\\\\\/GVAzYT6wAztOj\\\\\\\/SKSzYKwF4U49C5Tw\\\\\\\/HIzMvPNAUWaM1ahPOi2Y3JF5mzR9s8zyrd0HNbfuudgi74HUSvRk\\\\\\\/MwxFyU8DQKIsjAFmUAFaE9w1xCKBVDXrfCz49Wt5w3hZr6rtpg16jy9mUtb\\\\\\\/8vESzRl5DTHer9BF+XMiC9iTUcefbvJN3d+ZZdq\\\\\\\/1jSuZ4tGyJg61D8azpSHYRDUnvtmc1GtGPbz0geHaTXiiAeF4bzTIiux1z6d8X61H768zx+0OBtytKJ+suLsFAYgrsW72MVb6F\\\\\\\/OJoz8jeVBcdYlsgu5sTAzecnGMkFOdnWV+MShu+5oWQmHJCxz8p0Xzyw+sLXff+BYZIpGvwerg82S6k\\\\\\\/W\\\\\\\/Ykqnam1uUjS1RbNb7TjkbThV3NJQFQl9QPtI645KuR6BDgUBx9Zs5SGJkaJNgRNTg1VQORIP5qaL0gXOhNUm+fZWOvCgCwfKUrsPzxd0ATEDuiGzAtJ0T56IbWz1phH8HnCce0lmRjMBzGUfHKBxT1NlkOT3n0XV0i1zTL7CIA6g6ug73GPebmtSMh88aT0m5aorYUhphhPkr1+Mzoq4g32Ls6REhU\\\\\\\/YMC6mBhfZukCEiCzO+EArxxXIjjWkwcmQ\\\\\\\/y\\\\\\\/I12q4e8DNG\\\\\\\/5ExvNefEK70KHDezBEcKhU+qeTWsyYRAoIFjy27fV7P\\\\\\\/iFUeBIATuF8\\\\\\\/FLeTyJDPNR6sxOqHtoFMkQagI4dwe5bX7G3trE20kgcelJPgIlYhs0+ri4RkWiWglQNepKuC5XYmbVFSo+ytQqf\\\\\\\/oh34pqltJJWjouK2NeffFuSf5psV9mC\\\\\\\/jJVOIDSLRciRZr9tC7JSnhPH2+JVpRzavqNVZb5\\\\\\\/A8LxqI\\\\\\\/\\\\\\\/1e1ilRPkGiy0b2DPJRdabQn6TsFlwzg+5FVyNZqreepwDgF0xLJw2VnxTcTxA3wjrHZPlOfThY894+rS8SoQJRSpYyeeEwgTaT8jMfuP2IM3xF\\\\\\\/QgAPKoR9z68Ty2I0nfEgMsPzy5SIVgcaF7vdUYTwv\\\\\\\/BInvabpB63bHXgvtmnfqtAVX80U1HqqL61qEu46O+6MfvGSSyXkbxPmNLudnDmvXHgRPPJJ96o2CFMTv+Od2YSpf2uTBvmfVojT6cpKiD+FuR5PFQ4cODYa\\\\\\\/siWjv7M+5A2zPRqF7HTFWXAQ1aD3NUmzY4tSzhMn+Qna34\\\\\\\/SaP9c8ykg7Y7qGzXEB2w4j6GdzX2ev\\\\\\\/uoeV8H767jxHxBZEz2\\\\\\\/Z2ePFBS2GNhwgLtUwdNjmgwD7wnGga2RTkJqPVKiKneOAKVdSZnXSVUSb\\\\\\\/V0DldHW2\\\\\\\/vDQyiXuMKymSXisCBBtraaLhhZLz1vPjTMorBL1TKCQDpypWs9w0ho7JGB0DTTUTxbUpjc5y0qUGZZyL99iSpHNCP0BzDfbJ72WRDJuHnnXER58gLx+G++R3DKk9qjWC+LD9OA1r4xeDeUeZgszujQWVZLOP4BMp+dEV1ECJg+kdhkFGPPxlmrNTbzlR9BWyCxvMX\\\\\\\/8DqSE0ZypG1lwvsyZoEfKQ04ybLvQ8hKAc1qtzAR5VdcNpDrMH2iEIR2xijAgcTjndpHqen656nNcniKxLdc232x5\\\\\\\/kNK7KmzZKP8eE71gg3dEvVIzMfHii3+d64FE8lmgkhFzkUM1KUXJcZjVJIpImISfqdhTjbBNrLVil4+WJeDPD00MPlzb3HF16bC+tiDxQxAgs7tnlRMVQ6MSImuQXxvsF8SUbWkQpyPojnSu6AnhN6SPdcu38cpo4O59nCeHFEz5ZTXFWmH14bz4fwvX\\\\\\\/dyyENkFi\\\\\\\/Dx692ebT\\\\\\\/QX1F2H+nh53ze+Z2FXr0ZcMrUWs8G51HC1k8PAgGnWon0QgdeabsbVupv5barT\\\\\\\/zUpptIJ+hxjqRBU9QUeYliwIqYi57Yl3VLJ59WgdCjhtUMf5+Vio24kZYGdBNWtHY+t\\\\\\\/0wh1T5UJz2nNiByUdVZWkQe8w11mt8uNkKmRR2HJYVBV5dalHBNNd1IUYZ9tIFoFhwC68PeoPPxYUMpGRYocY+u32pcgvKuUAkl0YVOOUjpbaws6dH+Uidi5T9omdVp3ZZv8GKG\\\\\\\/g0rQWNjPiuMvSqI1\\\\\\\/1ltH\\\\\\\/JUpcNfYMruufsWUzxDDp5nrNurLVDdDRnjx752oMyuVNK1JcwLW\\\\\\\/U0j2Q0PxQuCgDKtxgNIGCv\\\\\\\/NA6KHOJQW2n6lsTroo14Xcrg2O0XLfuaFKia0hDWhqM8j7U\\\\\\\/hgl9OCF2JpXw51jMjX\\\\\\\/lFl4XQfSvwoOLDHh6JkNYZmfZ8L5B7ypFU\\\\\\\/4RiTDHCGUVVKt99kF2Xa30Qp\\\\\\\/uRYvaC2nHP1T\\\\\\\/Jq2w5vzZsEl22HfEn\\\\\\\/mHh3CSKfN8zgOvjrhRRh0NoFFz6nqA\\\\\\\/w00nzNtc4d\\\\\\\/S8nOblh\\\\\\\/AiSSDrnC6mj5NabKyRjNWW9bRgAfwPmd5BUQxTHqM89PfyhBV7DkmogVMXg9Czv8lF1r1pfaxGKfoEuBNkhPh7LQxpUlFNiivNLLvUmJb6+XmeK5ddS8o49d20rJhxZRpw5Qq\\\\\\\/MqS0NS4biB9cSvXYZbRYiEPExID1iptbVvqqURQWt2UgPhO+yne8Avs7SWmev0KdTLhgiXQw3Lt4m1FFsGOnPSy4yWKoihTCe7dsW3DTHJ+Giv2jj1QrgGznNlnlE\\\\\\\/NRpU4vL8hb1NLUQ2J1PDUNCc75lP65Zuv4zP9dVSvC4qOOARQdV8Lsq7QyH6hc8AEQb8y6Lc9ElV4DWueYwbHn8WT44vmpbUDcaOFdUKBuniuCKbOx6eGi3gupP9woQsNkNcoDH47izfBKjfI5u39RZsU3hkHADyh5kFNPMhagBoh3DrnVhArCEoX5LwlM2yNUKY6YTPoKGOKQrdLrfxeSjEk2VhmDkbWC7Ab\\\\\\\/lSxZCBGReygwsa1s4+BG0lEDR7dcvg883WXoDmcAObKIe0DdouDvh4Ozsj71ObR3DuwLs2lqCFtGu8DvPu1EL1XdPxYpHVC7RB+CgPKUn4rhqvYfLFYwXeaRULtMOoXe4xa0R6xIv3AlRFkfYrAIMciGdi1hifFjFdyonLVZsmiAljWKtxO7aAnU2rgyJpdU62OF92mVj\\\\\\\/e\\\\\\\/PQaIYIbrWzEW20WYf7cXmTtyy4zFi9scO2Ub0pLfXY50l\\\\\\\/hGBqBe0NYMHLF1RGB3Q\\\\\\\/l0TyOtILzv+yPlwvLrCXQaufmHq5Kpcm+l\\\\\\\/Rx47xjJolg8YNOz8n5WeI7sXM8w+I73QktfquUqv4mfqXfVe\\\\\\\/OlDoXVCd77Y2+0mTyjBrphM4lm09sDw\\\\\\\/QxrDTHQikUx1iP8cQBSCpLUTqMDdL65txPWse3IZ4hzEMT5NlpVFCasiEnvGO+vMfCX8p3lPhhWqMPt9\\\\\\\/Yjic+lDi5L4mhY1aevGQoI2ZnMSPKVR5yLgV7vj7NUy8ZDaZ60AX7vU1hJpAlzuLOnjg2t9WJ6VqM\\\\\\\/eYZM1lwPmDytnLspK5Mb3KfaseDRFE5EBZP+XUwfj+fcjHinSWRbG\\\\\\\/1ZZzKLNAGrs9bx14dQczyFwxnoQE5TrfCT\\\\\\\/oVjr52hFsGqvowD9X1tC2Se+6qfoaVgWTH1pLUoa4JY7SzrOnp\\\\\\\/LZbYv2v\\\\\\\/gomYhSf2GZGKWV\\\\\\\/xNIjirFPKZd7obt6i9YTKVKha65eaxoiMHCUmRZFY3\\\\\\\/V0vXSaEvrrAICfVumsdEM0nGxd4Gdg+vrWDxNQyw7UScLCmNd2e5mfxBNSFwrOOHrPtkEhWa4jPCvyiFL4WS4piI6ax+mOnnv8L39rVjj8xUUC1DUEQHBSuUdE6lsZF5wSODuJFWJ1dr7Bs923My\\\\\\\/aS2J+go\\\\\\\/pLhxpCr2wHY0FtWQB+A4+0h\\\\\\\/3nJn7g0j6WYg7ffzmdUNHHqJzIJawHvwK8lKpdkYcE66Pvsh4WQ+a\\\\\\\/Op1A+KE6baVb\\\\\\\/Rkwi2G05TFg0Rl+\\\\\\\/48iRB12A0IWP2IyxEbfysJT28ZM+7aeOXURNF9MM2rSfqao8No7wLShxH5JEN2p61IawGNoioBNRj2oDovWnuMiC+nTCn2Azde3UXqFb1THNF3fw+Emr53UBU6TZmmRxSwRCavMwhYxmKORCiH9aPXWwi7+zxASC5SGDJsnAoS7bwtK678f6bjnI06SXI9BTKLB3HLKzzv0L3\\\\\\\/0a\\\\\\\/ioQOnglHMdy4eUHxXopaEmB5+PWhpUkoEtLoj0LyjNpTKs5W57u\\\\\\\/noVGtz22tu9ETbTEiaK7BpjRQ4NkcOGdEUIp79KVtGxWwkviDkORZxiyv7g2XuuP\\\\\\\/ZnYXa1zxTcLw\\\\\\\/80fIVE47h+3EdZdCUVus65nr3Gi+Gb+5PoZzy\\\\\\\/m8GERAUSfYA8CzhLHSZ4mYH6HT6ZiEze3ueHpjNO9nOHv22JgYj5oaRvIq06cb8lUO7PnxrBwEaaPh4d+VA9V5NtcIwQ6VvFArdT9OlB8U1v7l5wrhtUDQ1zGXFb8zXEZCJvcgSkEpqh2583UmZRjiBa+Vh0kmoOczic1Hke7QSbpBvO\\\\\\\/xEPbXvZMgpbk\\\\\\\/5VHXvhpWygMvOhb5rQo3voFCCtk7E\\\\\\\/R1tr7UdU8Hd87g7XidzJ9BX7xobNEm8A5demCGbdwcOKPD+PCRTUUVdXkZ1LU3SHS6mZyKz2xcXUswzG+kHGbi42MI3YnTowwvn340PgMUtSuhHLAbUYj9SnXLIvQ8rPF\\\\\\\/LmjD9BpbDWKjij6Q7Mw4neyaVvyA1\\\\\\\/DwjwR7ZX58lVgCSVef6j3zYt84mqK0gr8UWW4KpEa2mlemg7uNlCLoRtnSFT3U5gnh9sOIiw5lW\\\\\\\/wgchVMfbdeqV7+Vj0F10JGjp2KN3aiiO9cm5N0zqcOoZXIYog232XiltmTpSnVAUJF7xJEWZ8nmYsR7WGCdFr8VZu0Df4jGl13VCdzWnevO6WuSN19Qgx2aafEVLNsnVmAVJs5pnqXQim2Ek07bkueJwUUlo1JJqx+2q4tUVhHX\\\\\\\/gEprXLA3UoAFHjaapR+8lS8JD8FrsDMAdG1BW8CMtPtgPoHLs1z9YzsIIyUjyddDwmu\\\\\\\/AGiIxPdDLQvbSc5pZLrEpiXYj3c+dnyHKHUKKLfQXynmS+twHN2kSYwqpyEzQsioiTztzK2sRACoGl68UK\\\\\\\/XEEJo\\\\\\\/R6URa5ni8jU9hbgvUsL\\\\\\\/WROOXRDPbylloUfSk4ctjAxIcMBEtbpj4fDc24g8Km1tuziXAIYs9zGJw0zPVA\\\\\\\/\\\\\\\/oCl0ZxtGUnx0Q63ZgsE5\\\\\\\/HUhAuectuYYJDl3sA1p5upXUtMFORrC1UFLHVFVYv0vBQDouMckTru7hsaXjxrctGk+Ajo5Y+oDD6b+6MU42OWCk4iR0Cbm9YoOwXA5NM+thNwUxE61SZPtNjlrl5AMvD\\\\\\\/GgneTQAAfP3dPJR0XeFnc221X64jiq5Xn\\\\\\\/LiuRAfxunCRTtPHHwnDTm6t4bhdZsX4+hLdGlQC4hu4GjgetI72jx4mZADpfQgPwMIoGBVVC6IDYJV2QUkXVKTeWAQiIJkyVBOIAPZWskYz1LDNR1MkQCoRUm8W7AnxdLBYg3s0VJ0TSzYTUaZixOkRu3A4mPmdsh60kK4fkVdXqYIc99DkFL3PrR1lzTf8Lo7XBkchZGziJM\\\\\\\/+qvzfSWivvBoxGBpDynLi7YW0WV3Whwz9hFAN4QuWfaEhkYItuPUOBNx6LnWpg56QlPhIYklnew9\\\\\\\/f\\\\\\\/\\\\\\\/yMl6E6c\\\\\\\/SjWyA20s0PQj0sF\\\\\\\/BQtsUhajOcUoAYo0jTMIBIWOtOgt7LNkkkT9r8kbnZZJ1tHsygrArfpcZD4MZaiytb2depLwhqRd+YQL+DoGhv\\\\\\\/iDaZZDvo8YSWRUKU76eYtgd3wWziVm6UYxBF6NKzNahyjj\\\\\\\/h8p1P5UkTZJ1Gz+Ai\\\\\\\/r38k9fQjcd3gJMfaiD0rrSwH8Wp7\\\\\\\/vqxE1z\\\\\\\/Ga0kZtyYwUKG6W+LnCAax6Do3f5kV0aNOqNQJPaYXvASu8G5uFtAw1+ZODJPL1dyOyJ1JY5QxEKGQgBDOrNKmavbJZmB0RhfyIarIlti18NyNI2FmPGtNn0VfCUO15xCt\\\\\\\/zB6zlIA2KoOx6c2fV8huAnpNY27R\\\\\\\/kcntnsBAMukOrrL6Gi+rvR5AMLSCo7lv+m59TG1osPTPH8JtNM5wYU6POXSG6zmLgAVfN0d1hVU8uk9Gp\\\\\\\/8SUredhSuWRL7aUs7xjY+rK2ktMlrawVxrr2ATVXviJzcofG4IPPRo+2eOJH84WxL3nkgmz9r5hixa\\\\\\\/SmgMkPdwZmvzO8ESXoyiRmFzK4QeaR\\\\\\\/Gqt1qvyLMoUMkhzQEIyHDhE10Lhrb\\\\\\\/Nsx7MzfnD73fIzbOI+36TInUVTkZ8sHVYvkkubsxrry36Ov0SIRsf+pen64O7oyOaTT67QWIcaQTedQ+yRNVaGCcPBYrqv4BLLBEbkuO5cDuE\\\\\\\/h+NKwdmpGYVCoiZaTL8rZO1esM9A54W9tAuIZfAh+KyQ\\\\\\\/9VFZJyhETzNPEQzyNbOXWqFj4oL6eR+jcOGXjqAtdSQkep1T7hZId6E9J0hfr6VudA0vropRyv75xtA0VeXaPgeeYUdymoHJdb3YTivohiOXNGu42oS1gKWlYWZ2RPwVMgqKlbKx6YW2pRDk3TeQpLasD27Zkr7tOELIKmEdTaswE7WwiHjTasOwFlsJ3Vdrgfv1eqx8Y\\\\\\\/H0URvtPfyONuDenje7sC6R0gNatS2oAdmx8IXF5jhlIC2lJlotqqs0i3jmAQ9vM3BsRLE7ahsaaXsoCtbUPNoBpbsu8hv\\\\\\\/tgX\\\\\\\/bJBxWZkNJorQb\\\\\\\/iG93M2UtEQaRomt+xBfnCWJO6BRVZkdNP6G15fy0iOQo1QKQ5PZvefYRcyLis+JuWdO5MwW8SvlBSzsN7E+edbJbWpV8Wh3EdQm6mUxPn8skxPfpHQoZTDf9NzY6fAaCdISTVBv2SieUfABOfNZ4fkQvBlW12XpwZTKFxju6d5cAV0f2X9KxAvETYtppyihqcd+ZrsfspKWm9CpMLDc6T4FPP\\\\\\\/dLnz4wVPJKDxHkMcmw8ZDC5zYnt\\\\\\\/df\\\\\\\/u8badu\\\\\\\/fwB91g9hMd3nrFbpdhZ5k8XjA14FfJwxpHBAZK3OTwJ+qBvxaO4p7IiAksnHtookckFz53tJQh+nGOUTT8m4hjJyREC3d1+jl9IIOEQKrLrvXazBcYTNdUifSD2J8qJBImpa9JTJBP8md95eK30iwsW8LvTaBbwHvhixJmeTQ3aulx7nMEqVG5iIy6QjhIIwwPpNwB34FhYFl19UOTvXX1mj2n4AvFYRsFwO9GV96maA+CJEkrLlcGSMdq1QlgwoghkGa\\\\\\\/IlLNILH+vC0mJgqAGsIDeOl30qyW3e23RSAVr0wGb\\\\\\\/IB+tLrqQxKdPrcN481SHZkogNaG\\\\\\\/8tK1e6rwRQPNOaE\\\\\\\/ESppvXLVul50qTva9vcUmOY7oR5XnJc93eRx7XaASEswVIb6XVlmbnOjECGr39Ev6RcHZoi3kk0a0wyapIVvGnkDdvgeKTFePCsbKGu+8njW5npfi\\\\\\\/TK4uhkJYPUgccHHAmfc7AP+Pt7mAZ83bin+jAlde8ctGVuRgyt6SbKB\\\\\\\/xw8ijxq7RfjDCcdc+uZJLXO+HHxkw4p+2GEOzUzx7j5junD0HAvlqZs5rainOeDHILXOjSrRP0YPep2nfDJQg7uWqTIZCyE5thSt9ffbrapOXNGz9Mqfs8hIlQd9ShfGgYou9SA\\\\\\\/WDMigpTJpEQIgcBbukBLaeOBDKQj0j5kuENAYDzp64TP5hrwhmsSsQ6LJhdb\\\\\\\/mq+WJK+w8nccCeGDvLyu7gqtbUvi9e8X9d3ASOX\\\\\\\/mDb3z9Mg1wJDGM\\\\\\\/SGaLdr9x3TgJ0\\\\\\\/C5KXwbm74BfdUXuDjNqVOyicpfKOM1esenGXiyflKhg4Ay8CRdWLP4oBQAx1krsv0d3ompP5ncrqC0xgviOxqN+dwmlBHBwLuwnaacDk+21wd3jQraIaf80D2veMwV7mLwJpNNRMZUIcrX\\\\\\\/h8TTaG7vX1pFV8aK+XgY3XtTxykm6PH24ti5fnRKQTGI4l0dAjTdWEqKK5WERt\\\\\\\/ws4Y2eRqUn5Blbuv3Vinp4wIv4OiJmgss\\\\\\\/PMtd8jGj3\\\\\\\/z6oqKfKFAl3LCaPVxdxXZuNuESuoSZcWmzje7BVAb9SKjPvnjm09Z6AVIYNoVY+JHbITpTGSCE1Z0Q2EP+Wh+UH+3I\\\\\\\/zjpA\\\\\\\/MdTMy9OQXfw\\\\\\\/mpsNX2Do+V7xNF4OqGR3PxVVNvjwxgde1qigDbV3TYvvpTV\\\\\\\/vCKUBB+qR3i4pjqYiZPW55Guf6O2zZSJ7iDLA04aqCyRLe9tiiWWqy5k\\\\\\\/WTlUlIJJvD9vPvlM4Pn7oKI79mih+DrG\\\\\\\/xt2CS7QiziCp\\\\\\\/EyJFdYVCN9+tYOnPJUnbZvPxz1H559j+xWmMOIeQzgWpn0q9QBN4FXz4fvK0sXt\\\\\\\/ECMVlZ7cIcb2AQ3qA4VvBrxlXF2KtgE5QPhbcT2KovtwH+HYPEI119t9CQLOb3jJt7WBY\\\\\\\/EAtD9B8N7ffta6AWA03oNl9c+mt\\\\\\\/\\\\\\\/eJ0yw\\\\\\\/m0\\\\\\\/RbAUYMBidyNgvLHFcXQYDmn0wJPkFPlajdRgXtn9teHtEpimQ0uXsfPxVNLIxMbtUPsGsFytGO0gSVrDA+8h0MGcdQjjgGTMotolS5+Vm2h1\\\\\\\/mpH4Mh5qP3Tqq5cCMdFUrg4B8F+q2M\\\\\\\/+HtX1pV3wPv2NHMoBCIsaVyOP1LzQyEQHmF9\\\\\\\/xHBLHrF+vsmfmoz\\\\\\\/6VnUY6pAeGjac2lo6\\\\\\\/DdWXxvK1GVkyHzzTba6zRsuYjtoIbAihbSyAo3\\\\\\\/hTOV4TN7P3mqZNSkWwqPkLtTTMPK4m+c8Hu0NygMNH\\\\\\\/qtwjNht0fZwQoB6DO9tqdWzXZqxI\\\\\\\/7prJh7T7E+UrXABleEK0b56KCR5zmmuBySHDHpmDOscM\\\\\\\/v7bjp6wUE10G+2C3eDaOMT09X+vRd0TwWd7q2dtz9JtRqIjIyWqw6OPcV9Q6XRmDMtcLIO02Qb8BhjGqIIFCv3+ke9yBbTKa6U6PDt91m3Go52bmHEPFVcDWgkdWZRjN6hc\\\\\\\/c8\\\\\\\/VR+r4AwOBw3pP8LAKsN773xcdOY0arGpUPvhbz4Fe9DX1uir\\\\\\\/oPJH932NYryGvmyKwEQqzQyQbKT3rUlAhWgqkRhX6+DqT3\\\\\\\/Bac015DPw65tIipStyGNawaolmkDXp7qoWxljucE4MT1SokHxQYevyNiTj5x58l0GVzppyQX6h1X3xNApi4RFxmnKpntP59QfiTlVoswGomF1Cd\\\\\\\/p4RBalVNq1kVR7KDMwGJy10k7b6HFuPKhwo3SJ3OY73fgEiXcLeFFgw8jOrscJB9KBxFwmgPNERl9X2tSoss27Aoo1pdAzmhNPPDNC\\\\\\\/8vu\\\\\\\/C0fPla\\\\\\\/\\\\\\\/jjvT5t212Szk7\\\\\\\/ARnpj8bS+WLuzJNqFzXFt07epziXpxPtmDtZZQVZg+fPoTgmU03HmP51jOErL2eQIrijq4rVNBtQR5+scIvPTn4CcgHjEtAyUdjk0N9NHOJtwRYUelUlUw7jlxGApcboapBmifqNbXMPA\\\\\\\/jZzRJvu12YwVeZdH8ocpAiEtp\\\\\\\/aAe+mI9XMeLoW828kKZJ5zkaUsgxan1rNE+g5D9qK7oBbTr2smlhFN0DpuotDqFMnoTOg1Zr0Cya01KyvmGEmKBe9V5mzCblHKz9EMNEm4w2\\\\\\\/\\\\\\\/Lvy4cpOv7OQBdBzFjqm46Tnm52tmHXoQJchVL1LzO2r5NBTjqgt8dWqtnEcKMl0XdTRp\\\\\\\/6nJsj53pN55kZTO4nSjBszkpgtyHcPEX7+6Oz8dfobcLbIOTJuMMrmtwV1L5zeiUnU6DXUzZ09K\\\\\\\/Diate0iOQGPsOsZi39uowJnfMUwUomHSDRtC04lXkBHVT7BD60tJKfiNJwQrNsP2Nh582fB8uXPfZfocCdS7WCIDyQFKj9HuUeRsWgZJJn2I3KkYeAckIx26C+mFnTWKzIlmRYbixVIYIYekulVHPNqvLZPzAKfv1l0tgLy0VoxoF5PmfyNWpFoiapvJty4gJB0fOOrh1hb1JXOkOfVEtM9A6PNebPwkmE8i6QbIRxXXr294ex3TXwIkEdx9OQTEb9P7jBAMgPEULvAVRg2BQ44wNThZaty7ovoKd+q0M9agL\\\\\\\/MPlPOkXf\\\\\\\/gW1xBlUiabhTn8aT\\\\\\\/+z0cBiOUODiVS940CPM7Yy\\\\\\\/E2OPQg5H07C0jKNapkZUCZdg3qGbGrs3BQC1Oi8X8dzSOFZ0ZHvstqz7+acTENPRSGXnHqRpF92WmjLox\\\\\\\/yO4jXPwxt3PjwHuIdavEyipriTmJQfRgonHTjn0CkpAYnCCwt9Vz+LIzlxMGK+WXwRgdKsFj4EIC5aC8LDLZ4ahLaVWjAwsH29atf1vMWsXh70WswVmuVZ8Lvz9cg33UkBno6T1Br\\\\\\\/Gh2nofyVJSoORaz8eAz4USPgp3AttkHOYJmPUo7RvIzH9ZqT9cVy68Hjc7Uc595OjMdbpchOKXq839Jon0cCHEvIMYWOV6+dc6tKM6faK781H3oZP+7i4x13qIstlSMpNCGNs6iWskWchW3TR7fqQMYH7ysI9DHNVMaC7V3\\\\\\\/bbf7el7bTc0\\\\\\\/eIkoYyaPiqmxgcZhN5+x12m79aESXDWi3Ucfiie2GtAHADQdIMTLzkrX424isvjDyc5jDEQibNw9ENL9h9BRAA7qAiybWfP+1Ejx3GH4KN64efLOC5IXWkISO7hZXa\\\\\\\/6GYu\\\\\\\/zfqEQSip9UpYDatAFqDTTqSxwyD+fQ6dqOKn2JrAP6uu9rBdhb2Wwbdek5hmla0x2PZq\\\\\\\/uAi80SzkeTGJX6vRP9UIWm0t06rmBmT52rdhx+C\\\\\\\/B+1MgQkf0uBXz8EYumYRRAjS3YNVP3ydQluvaEiPlOCy4CJek3wI79jbWli+sWyAvbF8nldwiVa77w7BTVDca8POsDeUiEZC9ifmOuMxYfN+acu7dRyCgwtNMyX8Xf7nYZir+OlHrhlr4W1iOi4LWZI59KLzNfmMx2e9TDGEvTDEotvksQ7M6jHz8kr5OsvjOf74YGUK+\\\\\\\/aHt3cHiJUCklthy2+z7PPeG0tFgPS6hEadlHu13q8o+3su\\\\\\\/PBi1a0Lk063SihwdULMpkA46MmFeXG0hpF1sBi\\\\\\\/nwuoKtI3w0BlmTNj+t974TtNc+DTYOupzQTi0R+9Yb0AMVtipDKWglaVY5Y+ZpMNFRVA2IL8nJXwpgcfbOSIOlDY3sRbZ3JauckRywtTy0\\\\\\\/UVnWNtyqTvBwXZLvpKZHTJl1EqqkE08Gv9sOxoqaqxBxcTonlCY+FM1SQjrFRibQHRdSfbOcMUiEynKr+Qc+dYd7wrpq65b\\\\\\\/kasiOwqKmKw1L8yg4YWvF+lHvz4OPDwk4YntfQi9iNhJcihVcllDcGWEk5NWbHW93DatODMs+8bjHtR00JLBPYWY00EZUiVuQJZcysDKlE2rlqE66AmJVBKIdTErpp0MpRO7EciVMK0pQFLSHfWKSjOpBVs4adXAD9CH7YgZsm0Hi0a06p880LIUPBKJPAM4QArLk6I5LuhWDGTZ9ixsDs2lZsYCh4if1uywcFlOtGHuunyNl3lTt9BGcaJu+KriHbGRXFdwW9D47vC5dEKiNFrt4g0SOaWkQHr9hHyTPh9FwbgqR1nKyK5SFhpZQEH7pOQ7aJmev\\\\\\\/92CimokRvWfEbQTQHNmYasO1TPJ3one5UP9hEexcVyUc5GAo2+TRKETDbUDS1nohNnW3BUgIxC8HC\\\\\\\/U0wmKkOLibNU1DgDOreI0iWdK141JzNNC8L1BEABBiDMI4lM8amHQSiWValpZnfkynsZnG6ySZdWihWnoMBVFS8+IFLpEGKh2uzik7UBGZP3RuqdIBrm+eT5NN5MrGZ6UEkIO5ZYDoCALZNiI9iFtdPdhHS\\\\\\\/8GZA8dkbI4duWco6gaEFf4y+yYh6qoi7evMgbdNayQB+jbT8l7oxYYYA0CZboruhEFkuZGnMCsOG6iRgLibKsVSaNZpp7EdTfgP2EGOKlLbb35ojXGFpZQa3Q0JHOZL8R96rjYG\\\\\\\/+y25lKsCJT3o4JnkLoZlqUzOF8pN7VoRPhDLWVADxwg2t3E1g+LieUz\\\\\\\/+ZJ6i4kb0NUrGGogMmg3rSUBRR6kkhvQtZrgPVWyIn4dXABzUTF2hG9xzUM7ptb\\\\\\\/cVkust1XAruRxls7to0INIXaafF45Dbtp2pRGdG7M4lhvEwn\\\\\\\/5pppgk0O0nj0A+lzzKCU1a8cvqAnr4XBjqL3FJtkHfyevyDzpf17sypiDveS4F4DS5yj7r2RcxbQjqz8n1tZf26UNByT0Aqzw+\\\\\\\/Ua66cQCVdNTgGLvKG+CLcfxpfm4s\\\\\\\/iZGx1\\\\\\\/FGevfATYyF8Vv2IMqcCh\\\\\\\/idFW6p4iu8h6GC8obl4op1VRdmWRbjuLJFFzNI5qJl5C65atVxLjXKbDJyXnIbofBblejXwNAI5\\\\\\\/9dvng9t2ZphTc1zVfpv5yGzKbYAu+lq6NeEJGk7ri5BpFDZac+nxb5PX5cS3H2b3ktSCnGE7W8XQeI1H2hOzKUP+w3gDMgIZohz7xZWAx3AAk\\\\\\\/bVltBk1gMf4XEW7hHcg2Tp1SdXILQROHeYKe2FSlJSdBuC0zGCRNMntqbEm2aNAlgX+M1d9S1lLwcJ477xnFlUe5xPp7+\\\\\\\/fD7c5qcwK+qUaeE2DFSXQFkx5ryB1UMfnMvoK5yLQyWVWrrF4NibZwJtvIDUeYKb81erhr5TDewlo\\\\\\\/hI8AE5JFgJDiOu\\\\\\\/\\\\\\\/t1ZEuOZ4f35+1btNte4RtLLSfvZM6hbL81B8Tzl2FaTfe3LaafbANDsJTICMEkdy0D2Vv9I2\\\\\\\/GaixTdiQiwAc07NvWeDabIEWaXpqO36ZozLmrLtAHySpRzBHfqYJXOMZ7gPMqwbMP9CmHkTYP01Nj10MGPTdN20+i7Kj3vYhvxtWoErBRZkX7znKANa7VSO7JUWPJyQhwj9CGpnhPwaBd7ZunZyRFoqlGqYk4bt+tbRl7h0cu0nZgI+6X\\\\\\\/0biTsMR5JHtucLwqQY0zUUbJ3H6D0wvqgh6cJkMQmO8XtozG7MtHEnC\\\\\\\/62HEo80alNPkj5fa5OF5a6fMteQryNCaiVeMZlQzYbW30bwFF7aG5caWYJokgEzg148Q\\\\\\\/0beFHw5A9tQrH6JUTZmuLH3\\\\\\\/DPWf\\\\\\\/jMkniC8UIAq45sp2RZ170NpDe5oMKCwW1d1MLYRq1c\\\\\\\/S\\\\\\\/JSFwPfgfPtQHJQdpgZLjuY8zx9qFwOtCWmy\\\\\\\/nrsd5HL7mGx6ZkHCgBybfR89sOk1aUF40t9qL+RxC2lYvqZiI7fnPMZShIg6b12MJ7WRdZqS6r1yLqb2Fin629PjZ+tE2FhldJpK5FkY1ko5vfI2b4b8qPIMG7Mb3EHB5jGZDwlskLiiNepOh7xvurWhltywBCSZvYag78AK7MNbgGEwpAyNNFISO3E\\\\\\\/EpMT2TGHoVhoNwC7AnY9hFpkBW13vwsF\\\\\\\/jHFXeAu7bG24L71xHPeLlVVR7\\\\\\\/eoiyervTc\\\\\\\/75R0x6G40nYZyuKJodxIUEd+5aM4PtQN2h893ZxXMhVd3u9FAGO\\\\\\\/BE8P9RZoh6DLUP8MNtP\\\\\\\/mD7H2FcT23Pr2B+NCA3G7iJ4LLRjx15UN3gBECrhz510yavQdfjSBQgEz6YoVUbwJUOUyEaYLPP9mpJ8arD\\\\\\\/CY7q7PLTfvRm979o+cqLL9n\\\\\\\/h253nnYBDiT08tufJCvQvZROOYgWpc8Hii6LtElY2VbiiB\\\\\\\/\\\\\\\/Vhh+cv5ZEDb0dIU3dTs1cKuxTL1lgZR2JHGzqVcdpmhM6sVHCEYdo6l9N1SFBZlJN1\\\\\\\/D\\\\\\\/Gy0ZXD6Ttd6P7cp+XnfRcflo33+73K3JH7E9DS6q8CL+JgxHhsLJYRxqHk4GfdFKECli++pcpE8wayted8cjo8Hr\\\\\\\/AfKwI2PiIGeMOuxBbf0EX50naS5c97BMlOzmg3v5l\\\\\\\/Pwjdvo9KCPo467EYGNVXI3Wv1UL\\\\\\\/i9DGx\\\\\\\/LWF40uemEjQt4k0mlrCXtfsmwXM6XHdoaguC33buWP6xmPZwODy1h2VznGpIyZf0BeqYvbx0MjcNnHy8B+2bdvYhgdFQCxSnppqjLrX2NGzz79KDxaiM14eIoMRhiLLXA+LBzdzdb0XdLxgCT5uq2Xb27SXvg5VfIjiEThgN6MvjDcPskCSVV3VzS2821+4An\\\\\\\/9HybEgDeXT5LeLBVDjrwHc7umSf8zl+xVMez\\\\\\\/nTCKIpxKNMjJws2x3dx\\\\\\\/\\\\\\\/CQLiiXNc6+ryMPEIknISFSF3dSH4QG9D5V3Q7tktYf2+ORRoxWnVvjMKhEoW1nZjFVG9u8Tsbjwt+xF\\\\\\\/VFy1OtH5cdEnFNWIyPh4S3\\\\\\\/J9GcHiviHd04BKJpNQqikSXT+4WofLacU8yCc944XZBTVZId1AaGS3SL4znkH4Lc8et+zqqEtrPMPRsCHf6vlPHZ8MbuqIA5vmEhd7+xwaICnV5E+daTOEcVbITO7D2eFDzntmimoV\\\\\\\/WzXxVpwycwU9G0fenFYQp8VYj93IjoQ9lTWNrBGWe4JzzmVmjg3Fw67loug00QlWlYt7xzCbnqr6sqnuRSqo3+uFETrB4cJpdaxyLsfp4VTjRMbIrVw\\\\\\\/0+MVGgg5hvHS\\\\\\\/JUncP0hg\\\\\\\/pgi+QQDuF6awvsRySewvj5Ci62Sx38WbKsWGBa6RpKGHoxMPDBPp7yVznNs33aCZdMX2UTKkb9s\\\\\\\/w7pTw+3SfW1aBCWLvkq9nmHMxm6eRLwo3lpIm2XQge1q9QlgTIq76Xol4Qr+qAodUAyfkjZDu3Q70CoydnzWrtOqEvMUUyXqsyAAj412YDCsKnZQdq1vM7nJp81E1kFQJvqlR820dZxrf65du0B\\\\\\\/WbocZzbFJID6AeV8FvvxDW1VnG58mblmpnEjnYKNzmqW4EEu6cRPZX+VLdNGEXmwywnWaSi2O4FeVkoHAVwDtd7XrzxxdeM4\\\\\\\/XmzO853VDaGtluqmp2B1Gnc6EGnuzcVt8EjanAUEOJU5T2ICeScHLmYyjJMJ9BEWX8JfojmjsGIqIdZR3Eus4pGPTru5srJkXGhHX1IJhs\\\\\\\/ipTrcum7x92LPOTSXwhQJUpG44wkeHHSiyLlbWZuKPr+QEUi+UtUHwk2g5BNiL7eZoufkyIn\\\\\\\/75j28x\\\\\\\/Zp2bTUwUrJiAd9o8ctnLKm5HkFM4v5H7f10HyNM8lyDR\\\\\\\/xSP7d3h28CSJjnQySYQy7m4tx2IxyZ1em7vaPd4pUV8fhR2LGzNmU6FcmHa30dZPkeQGCzjVbGGixIcJ1KCOXW4vaUfdUjcpumFzKNUvInjScoL5ovc0ILxLILOidVrqpPOOLrhLXzt6VTJQRuLLvMcSZBWatnX4D0AiOXW\\\\\\\/NKWeilPAG2WARHFFyejuS3WTgQ0ZEziF9dhqXbzslairyhj\\\\\\\/UpPGkpRIuXbwWG7JdG\\\\\\\/gzWILWrRxQYGGcoTMvAjxFPOsbcNGn3VArZd363sxMvysMt40gjGX55Pej2aEyfOYMFRHWkrCUvt5AP5S7ARrzNZEXe87Z7a\\\\\\\/KoqGJfIynl7tbImS5s3unoSyweZKwSIBH51Ipa4dIhqcktLraUD9KQsHn2m0dcXAQ8nBbCk6Fg88uxRPK\\\\\\\/AC9kXl\\\\\\\/gAqInTvaLYTJgx3hfOtVz8h\\\\\\\/3VDybvgQi8SEDST+25010wmfJt84p8KemLywGEkleoa21dOs6TkRURP1eRwc6ozllUYZMiPTvKgdTZ\\\\\\\/79eIo2KsLGnQy3Ym7lPTs6hHVoM+FkbuiMDG8F1O+W\\\\\\\/8rKO\\\\\\\/LkY+Z2OUc7mBY8VFB8ZNYP2jvhjpGi9Mpeol4mMc9F6gVm+xlQ4iP1Xeztzv+M9Y8nDQH4kFUmTh5Vk3ge+Muy2y6YJ5LoCvWNBUqyauGVDE4gMkW6D959p\\\\\\\/d9rIgxxK5Mr7LhCFOMJrrc4faPyEYAN8wvGH52C2pFVPNEPwnbOoUCtd2PY+U3eykdk3crCXaNiD7hDOoqsKTjIsR+seOzQHbpEF7g198GMoX4mkhJN6gyEtwfXui5N2w8AZaeV+nYmTw77H+SdMOfHzbLHvGab9TifgPa45+BWIFNtHNpHiIAA+CGo7F7D31zjzd7pPmTf34t2KxzJefee\\\\\\\/DwIkJXi0NIPpy8QlGPsnNJfDYpiwifroN5W8T6jjd0hM3x6PtAkXV19MfhZqE7pLmpWT2AtrAuM\\\\\\\/Ptr5NC+b+el1LHod9m+8dlQ7AxZvedUdchafBkbwkoac6M5cxctqkpO7y9gp6n+XsEIuX0FpMZ6OqfRNT5awh4d24HEujr9D0YgjWnOdNmw6IthjzcHJZt4ukWUzfp1mMbnfJPVOwcFer8ofLasKfT9n25J0cJoJzuifyk3kWhZs+KR4yGtb9fK1GmzkFtJu1hW0S+VNOgj3rmXGNHdo3jgjzK4Yj3GIx6Qxy8p\\\\\\\/XklUho91cMaNbsrRjlXGKPL3oLTujZsGylhN2NsfL292ikig+1WgYcFRN0\\\\\\\/p1A3r\\\\\\\/LhfDT+gNb58AYxDrOJQiSNKHOKa0xd+6CqTzGMsIc3cDNzKqa1gQjPFcV9JCpFZwB0NMHzSIyDawXd9WGSVcTuwKIxyRrTTjcRs9PdTBUekgne250YlF0HVe+O9yay1KYI7XdlLg666Iqa6nODsNYrN3gkye4olVimKcvSsxwWawZXrS8FtILSQs6kSBxCnbLyifU245F3JpzpewMMx9Mn71SSC8Et4r6z6xjrhRYZ9Rnv7smEgXKlwwJc57uRj55tPxfcKX3r04sAyHGwnSA+LOKj8ZFgIOk1YJNHKXL84hNo028ZFOemMhUuy2sg\\\\\\\/DyKopJhsLfno7fpj2JeOmV83C+slm8WwN0jNU3UDlz580eww7ay0HzU\\\\\\\/qt1Pm6G2QA9t7nZNkEcrsDg1r9AYg3M25HOQQeriLkd5\\\\\\\/EAoX4sFVvIBAXeHBRWPjSBLTfjFVljzEItD8BepGNs93URjqYKF7izLU6+t3U9PAEcaRq\\\\\\\/4EmC+bGThxUskgBCv5SWYefCFTq8U\\\\\\\/98q0JYLYAB6n9hIhVGgQjPet5ATkoZa\\\\\\\/Ykhew+xbmcv+okGQCYyCd\\\\\\\/VLRFPo2OSho8NQ3APZ+gIYWIgRZkdlxCzXtAGM4vH2s8OtX4MMBte0HAuFn6obyGq79A\\\\\\\/0dKD1J2wnABJ3exdpjgIYkGQadu6I4JSn8rZcbAss6Xoo71FfPp2DxQ4V5zPvFfnXXI+ef4VpyQMM9a9PRvjchrOEphlHu5EICRA5N3WYrMHHvE7NkcWmWavw+dICcldWKZSMGECbNpxbMJ\\\\\\\/r08ghbzMTXs\\\\\\\/x\\\\\\\/iDeg6poFavmsg0GUcX7yjbVDvtO+Uc0WQVYGyKrKIuXyHlJO5qKjromIirofhi3smXRfPXMpzPoBfE7cG9+ImQ1mGaDif8Gbkxnbb+q6O1IZEdOK6v8YD0MdVWQgB6k2Qn6MyyPnDkq+9qe65phCobk1mQsHmDpkxcT++4+Q2e6IGgpjcG03yrpRXHtBUpc1hgUCcKpQK7AlZZ6jpD\\\\\\\/SNB6+YlSXAUYwP8UIMXH8wTlDn5Z0cWaS84UNFDlbqq3yEtGCsMAQev9eU9E2v8+0m+1NhO4k35rzcNFfq\\\\\\\/F5HlOgSrr6nTm0Ht\\\\\\\/1JK9xZhk9qPUSz1tL3Emd2W5kEf3ZJ8CT7m+PIfJ4BF\\\\\\\/pQhtGR2kkMRnxBCn0702Obvg0yGx37LaArapSzpxigRfB0jG5qP6moBixBQJV1nLcj1iorWB\\\\\\\/IAMXis3nC5sc6GR5IWvjBb+9\\\\\\\/MQQJcw2t3AP858yEn8DfXxGkm48JA7t9jLZIW4PJ7LP3YEqcaNzkoE24VRwbtbJf1sviAgjRmer0YYV4hqwJlxyXjPPa2Q2gZCwcNe76RQ3vdHM1lGUmPbKWcUK91CvvsLwmUWlhRMjYiB53uKddRjl6sBwtl4t7UKzJf60mO1HDf+fPt\\\\\\\/lq7Ly6gBKSruIS5mxGWv7B9xjRz9Tl2WTSTWnYIQ2tMl9qptI4gAnbowPvfvQ2hxhnJ966FJfdVuX9me7O\\\\\\\/YOsLaRVXIweV8Ndh5\\\\\\\/Uq9+LLe\\\\\\\/lcl\\\\\\\/QXHeQsD0CiAAZZzvctDR6TUM3C5geuRTvl0iuL4xmp8N2wZLSzR2Crkvb6ur3RdWnDFlvKgzcsfV5jhBGmJn9COg2RM5RUJJLmJGQY\\\\\\\/3LFwQ6FsrJd4axTevZzbSPkDCeja0Rm+VbJvBIop3Z7LdFk9\\\\\\\/Os2+I+E2YRbTnwmJy+xm5RnIn4IOG4n\\\\\\\/g6CrwOKoKIaEdJGtbMiWKsO0ZAwZPcvyZt\\\\\\\/UCnnB7WrX\\\\\\\/1SLWjd9KKOuIXYWXxxYa8wGSzyS9cYcbEhkb+xqt5bCX\\\\\\\/8AYUhFXnr8Im2a95\\\\\\\/UW7CKOsFPOLzMUA\\\\\\\/OOqX27\\\\\\\/G0VeNwNYQCD9ojP6abKS9y7qk8lkExWUfwuDbyc92m\\\\\\\/YYTmTjOHneb8loNxamwAM01DdLWwg3rY9nspAwSQgw3ZymQ1NPKpBOqJD8KIrNc6GvE\\\\\\\/4ZcFXfUbED1v5aq5slMSCOZa5mqCgwmAhSWoYR0z66K5iX\\\\\\\/nUua8fGEY8VMFKxo1vWSlzat7xAhBFvlpNkBWKaVAB6XDiWQMIzcShGNHUAF24alzvtbR8TToex7ZHlFHG6V9dwZjlSqfC3leOfCrjDLSNUIRlbL582i3cuAi37Z+VecwsNb\\\\\\\/eKsG2SbMezNxbVBKE3S2sNz1+ghAGUKG3u\\\\\\\/doazPhYlzFrtJOTY7aeUXjixyOPMwDosTCZYElHSQcHX94k8A0XgFR3bvGrLDxFkR7B7I7+qV1cKi92EFBVAHH8uaUT0WRVnNsh2YSQY06yK110V\\\\\\\/APG5GuWIYdTwo4TuiyL5+TmgmQwc3mxHRLb9muWGNq2jaz7aIaafQLpfekQJnXuWkoETNRb9CEQuwuvffuxvt5BXlVQh0\\\\\\\/LtytuEdBWNsd\\\\\\\/GlfRnJ5wOXqxaKx5QbDRmur1MANUjXwbF8BiEhuHKG3Jb3mJkatLU7hRXsSmRHvQ2hbKPgpfeaGrhZ+B0fC5xEIfl6MwQbqm9GmNnrylw+zmC1HONchLTVpAogEAJ9\\\\\\\/GhmKpJrs6HgCKSeyxsHJBAJkbq8XnBVcuHAkR4clkxhOs3xg2r3LQLU6uO1uR\\\\\\\/uJUewOlNxFJ+QSMwADwJtihJuGsRCRUpLNRu9gWrpY2dZ3QkzW6ROQV0NruWMuPNBW69L\\\\\\\/awX+X7EjcXYJID8WrN0YBBPySLneszIAYv+9Nu\\\\\\\/3Kyzm1Z9qNqKtj9W3H2ySfFTDzOCtlM6hiTDA6bRi+dV1BeDncQEM5n8Jk+OENOVHOAj+X+BIFE4eJ5o39UHMdNYMxGhg5suBoBXOnLSRn82o0BJP1vHZKgRzZ9mgtIKQMUyFYEV8F0GJj9+Pcm7J\\\\\\\/sFSq9KCGroSgQRVf8gBnErC672\\\\\\\/GYjeDF0SPBKklSP7r0MYoj3X7SAJ5PYcwNnyMmk79cJPyH8vPcyEB6WmEQv0h84T9e6\\\\\\\/bLOtlzTQ2VzoRIGquObQt1XapeeRIIdc6YRJvI83deI\\\\\\\/eSBxf+K\\\\\\\/CBUgImd8hMexOi4qf4xGqkb5TC6OqAoEYwDPwdxLur2loI0UmCmV4MHNgYE0zWjizFjoWko4gHETm2vKmqdveoqYTFxARUQr5WXpR731+dc0sT+emK1iL1E\\\\\\\/rKJHbkUA1mfPGxBrLT8wh4XRyXV2BC+lB7kG7IaQW\\\\\\\/kLoQr\\\\\\\/z6\\\\\\\/zau2P+xhDWymlukWC\\\\\\\/xZ4eFq8xs8N\\\\\\\/kz3S1YYez4zAfs3usP9QNsk1zo4+tdMCnEBWuICkLtMC6FK4AABgXOleUZcka7+glIpkBYicRmNKX7aBZlIm82+UE\\\\\\\/1xMUM8rufYRv3Gv2FTUsCMzarMfV\\\\\\\/xCBtJOTpliPFuRmG8\\\\\\\/sKpakHdqeWa18UOoo0m70qT0IWb8WKgvUPhyFucrXalFBOWnvoCSqdP2g2UqO\\\\\\\/wZQO1sNoID8e0IIEM4rq5CunVZm0xmPT+K1+sRpGrIjAPFWTlWppqg30WDyDC9qPuSRgLA5yposYg+XRQJ29+wkSS4Y3Gs0m9uTAKxk8K7jb3h2A7reiIioliUs7SKM3kX076tavrsz3WQAqBWFhhXUVnm7WVhsCA\\\\\\\/t3+y01+L+9ddduFC668F1G5907hht28hNHAcla1aCM4Sq56m4y+T+xvhUjXrxVNS62SqRkEFDV2ZkDsrdNx1kYjJXCoOKJAP37iMrwsDImzyl3jFqwb+6YabZvIQmOJrkyZaKOx1YWMMe3XoIW+ShcN\\\\\\\/7c9EzaaoG1HRr0ALfQvNvmc+4WaKL\\\\\\\/cqLZgjG1nHGtlhE8t9g4iVgBcd8cMUsR21Jd+DpoyYd2LXi+GKwBnYU7KeZxAMhbeHkqbKUjqTnPulMpMmFcretV83MYOahFIFUpYWhi3ARl+8h42DpGX+U+6fV2GP1LAXIujzOGh\\\\\\\/xsvlKwtTAyrgIdiaSMHqQtnl5N3H4o+nEE7YtFw8POiCRDJJhoHclDjAUPG1VTrg0HOWNQP4qM6W2PSVUCsZQ8Oh1S6LPrGf\\\\\\\/AlMIT86F1W3gK2qAdVgjfWbajvgE11otGzzUdJvQwiCk1k371AS4fwjZkHjxWreIPZf4IPVFsj17n9CW\\\\\\\/i4\\\\\\\/jH4UouTwtxlgvafPZDZFykGhRIViljNPRFUfUmXB5hufhzAFTg9YSX93Ldsv6+8z9HPn+wqcG4efJO67L3BWeDwbMmSmgG2W2H3mlDsilDge0GJyBKL1bAOsUnzvqjuN+qiLE7jdq+hUqLql4PQoUfPQrcIjgg17mgUSG5Dp8AWBqlLa+OY853\\\\\\\/9JgZcLcZkKU3mqjiUyVCwhnLGMk0Al\\\\\\\/d2WutHoacBhakBe61mqQWgaHdZhC5Y7aISyuKRR72XWZwndz3Y7GagoOi2537tN1LhADO7YDlA+vz5OKZwkOjlG9W0Lw36Jevcl1C99qEfw0SGY3FaAfXK284yYXUFJAyjgOmtJQiJ6DBIwjTd4qTkMCdgjinMHyLutL5lftSA+LX\\\\\\\/GxOurL5GvDgpSvJ+fYR5z1rtULXulEvNSUU+fbudkXbHwAUgPChH7BVfsJK\\\\\\\/bZaJB0LfIW2mRgyDWb+6EYLbL4cbRluFYIqqFs9X4R2qQ+BE6\\\\\\\/Kabt8lDZL+vavkIGXi6tMxMxMDyzr2gfmqbrGkakiKAZQ4WU8IbTNJhbYwAElApR3j9dlgERZ0ihOMv1GdtnnbXrkivp+lSqYqI\\\\\\\/UjvU\\\\\\\/nETFixkK3yQt7ET19aMFwoCK5oRAUnTeYUfYupGJsYs04kFcplQdsuVLpJePYcpkhRUTlX2bKX2wTNDUHn4sC5fKgUUNFW9Z46wmlm9eJS39E7bx7ZKg+nJEOMzadxvRR5Iz586FCX2OAmEMGzS7KYPAHGWoKTcyDHbYljoX6NZRpKrnZ9nB+w+8uYey+XqgQt1f1txJRZbWOOpcxPqVxbNLQZK741XOWiSZqvrThdyPSqfeYTHQA7NeydIijoxU8zvhovk6aqHeccyuZxRPANVZ\\\\\\\/80vS2pkYKRw\\\\\\\/kEPo\\\\\\\/nspsgDHWIOA7+yJF8yCe7XxmwvUtHa5AgwUt0NOoD8Cn7N1+81IUgSS6NlfsbiiT7dhgLTGzTGkP3xy9JRnG1WeMA8lW7Ee0CwoBQwpaZ0Zl+gyDFf1I59A4x80jAo2SdyYTuMZ7dBy0Gqe6Mqy3Onu\\\\\\\/8yCDlKAWY18H1jQHHGvChLEmV4XdxayG7j6lXFHW4KdMIzrcTmF\\\\\\\/i3fJJH7SoDNBxfeHu+92cYZL1UJd6NJYm5Xb6auYWm2IqPR9mgow\\\\\\\/t0G41An8i+soDnT+y+Wf+QIrXV0gqiszizt+GCD930kQgIV+Z+zKSn1g3lzk7hON86al8b7M5S2js3QuX6ylnXk5pdNDotM1i5f7bjsNp5sMjxb91OLn56ro45zCB5fZM0ulg0TBGvgETmXLI30iHIVWdu1\\\\\\\/87weLz2zt7IW+8oJM2kH2L1xQJQbT22RpGA\\\\\\\/KTxdh9pOlISgrKThx2An7CiUhfCrZyw1h95JWGKEjwf3ylA5hra8Nv16S1iTIpZgAsnjfUK7NeMKvWyXR2DkAe+ebEPb1UY8UtQR+adxLk+ErKKWpsJ3LLUaNtwhiXG8eez\\\\\\\/PeQb\\\\\\\/nFW+vLTkpTAxvHyJis5aXckv9wgv2DtAOIlx6Jz2W7F9lbhy69ZN8x22lIt\\\\\\\/1Ti3Tk6KY4VM\\\\\\\/0BcOujKxUyTH+wSZj0PueaIsnQQw2pwGn1SIvXY0I9XNf5Wxj5mUDJAgYJqRLmallcy6Ct2z\\\\\\\/mNgtmXjPLR6OoJE0dv+uUT7uQBDiSy7tibLuH8DTV3Y61no35PhHnhox96GHqtq7+kWQS3azLYEnBQBSBx9UE5MmNlLKPnnQPkb8UlIJSu3VNaUU3gUqtSTfJR8LtjMH3Wk576MR4FNYFkHHKHRbGpsjAM8Fa82PARIozfi25uPGQY9NJ6c5Kk+wUVc36596NNz5LgyR6KAVCpX559v7E1ssS8wE8fD+UAEDMFWCwc9AxCvFHyxQSRg2Thgk0HTWs1TnsfidjqqcJe67K3Iz8NhAkzUWnOBDnCkoni+dO3fRB4dxcUo2TpFMk1BNdjIWBNl84p2IABderevosQBSb4s5oqQYJmt1vf2RcKShgl2Vc6kPAe4NhslKgYNogzkWdKuMClkpmW8HHm3qrZ+UJZrYeIRCJB6YD9hb+swgvh+jPOQFeBgAa+o1WNXCbzQE\\\\\\\/Va1Te2ufoY56jCNNqTcXAtdHlDU0ugzQCMvYaaRMtPcSi3KY3nIGSiqZ5Y867mzGfRa4QdlFqb8HrNwTrzychdeYASszVlYNT9K4COZka9Evtr4IYZSyn32E3Tw7hQGhMELn6kwDtyavk\\\\\\\/zWzIjIhv0eGnEsfi8hlp4mvLbm\\\\\\\/+mDOZqfzVCA4evzNTF7TnlKSOWGv3GPiWA0uY7GSBLxhG6pvHr8d5cl7WQgqWloueXMvJTR5B5zCcbYiIB\\\\\\\/Lr29EYEQ32tqU0f8x+1Rqbyge74na9N+p288FUyytXoEKhojkG4fpTsOeny1XQnU75B63icDrnHcPYhEKfWMOQqvMtxzCWeVc0+7C1WeL5j4Jljb5rL1Ga34H9KOmCyGBxZPqxHnc3cPnsVebLZ902s0VrXFezcJCwgl73H7dheaS193BtNZSHN5EwdQIjtCYsgZ94XK807rpO9G9AHJqz7DVkSQysYWhRkVrvryeGab8rAZmGW45Lom+8kZ53mmAaucwc4jzyV8zeyO5wzYqTy2kEgD2GPHaTsdO1BTGhhSIoOG4j7QsJdqQyuirKCIFqXs6yKxkb\\\\\\\/wwhXluxfOogTZq4jCZiemoOiya62p4pFucsraaV9aPRaLTKc\\\\\\\/gIi6x4ImuCzjnRuaXeDMqRgdpJyK0NPHZzk6U3bMV34RQxTXQLjhJezTJunIMZBxKKHGz08jh8KJ202b2a32E6f6MAmkCBAq4B7VFXoYSc94KdHpjjNK8CvT7aszPDFoGtHFA30XfBDnCzwwd8s4qhP+bm0mrFftsO5a2nd+GLl3Ut36FNGReD3VCOCsaR8ag7eJtKKnia0202wn6Fkzgjuv+h2CTjuRV+QQWOxg7AerIaSO1NbWru9Ly8HYLrsNn6MmRvxJT4iJh4+Nv+FDQnvnMkmz2wMLT+1Uf6cXb8QBNuHHDp\\\\\\\/wAnpTbe+Orzde4SZZUkf0aD10FvcdHwsvFjQSClBZevZfXEM8FfOhQGXiWnX6TBWRvKGiIwMATcy15\\\\\\\/+kLJFFbzBMsDrDZu0iMJOel8pz05uwjWbx42yapq6FvwQLB4or2J9ejsXHd2qeOc2dh\\\\\\\/0IRQPPSJmJfTaLYkt7S7MZvU7VhoFbOnt+e4zDKLgEmxg7xlhrBYXBo9KoLBC6QHP8oDgPQHi19v5610zDPiO5KeUXDJF13xGzbvqhH+Jn52wxZ+gBsNeLhRqAYhudmHHmvXEd+jZr5Jx6OJDEH9Rmwyu+jM36d4Ai65dVbLxPnzJEQUPCewnnKlHvLWIGVgb8\\\\\\\/YC5Z53zKoPQItCv9yHxioJq3XkaYj8+wOgaGsg1zrPuBYZ197iMqUkWbAeIM+DaKIGo7CqGi9PVLbroGf0BmNETTD3oZz5pjvULCdPPYWle3S0kstFaIPKgU8vhTcjFDhqmUTub5SuaCXmFsM4WJwiyJ57CsliI+49wMMy1fHrxxtfvB0PSh00INSfm39bluDj4uOpPaApqs6btObyJyEytlfB13fUCTgK+AVbw\\\\\\\/X7PoXqVyduSLtut7ZrIARw3gzWhagBEaEdnK+Likhx9iyx8llX7tJdDtFfy2\\\\\\\/LYWRiNWZLa5fGKT8SSD2EGO1ycabH2JYoXy3Xm12JK7lgANQcidvzVFnsYOcJ+wJEoIMi1+y\\\\\\\/9pkAM2WWWoeuoZRSYBliORUdBkfUDwtyuWVee0Hoki5icdvrMA+Y\\\\\\\/MRtXRJkKQEf0xT2j6oOnC+YBFIDrCdpEKxNd59u3npsHLnb0fveIuovqRB3UndxSa2Yc4dEzPuz+WgOtFUnC0ryOXsIN15opdfFiRFSoYr3zZB+wJ+UTxfsUqi\\\\\\\/gcxRa\\\\\\\/mXIBBgbJ70QbcE7BKYkLXMKj\\\\\\\/oQNeWfxcpBHod8OT2wGyY2vvjNuKSYO6sxlLjchjA+gEN55l3PD4fhA5B30U+yJbVpQEMg4T4hMN2bPaQA9NS3hH0\\\\\\\/\\\\\\\/EptFpT+ZH5NhbZWIWYPh9BBh+yJGg3o6RA+eK3w7p4Yzijj13tj1h3gdGAO39CvpuiWKv+fhgwZBAfMf+MgSHb3sQzi546u+8\\\\\\\/NMFU4y1bvPW6+ywYrlF2aEtDKNUcsP8r0sMIuVt+bvHMWrCnq\\\\\\\/ATSv+lCrU4Hyo6ywV32RKPybAoRU+2wghKUtTRVOBjkdPnk3S+IjsYa5oH4hs7uYTsFfxHIN0zdbiesoETLh0+bzPaTo3e+FxzFfEr7WJ34OJXkSBDoiIWvrdGxONYMPS2ul+htcYqMa88Sy8TtyfPNzSSLCkb\\\\\\\/GluSSY6J0iJV0WnSlXZM\\\\\\\/zUcQrNKM9WKxLI3UPYpciqYxyycHOnI7qtU+metkRqBZVmbDAVfdANtazJqcZxMzqZyCpUW+eZ10sqLxwOfOkQMunZPn09xh3RxiyAMsRDv\\\\\\\/nivxUAnzRUwnFY96ZYekh1uVmU+N9n2S4sFP8EFl3OQuAhtw+F+LP+nywHxY2Pc6F\\\\\\\/VBUfsY8QKTf5GUqUAO7NsOwDLlRfz3ckbrjk\\\\\\\/R3ZrWgdU7HDMmYQhKG8emb6jmXIwHK8KnpBwllDWbXqbj1jjeFgh9oLQN58a1ErxVLytgAXIeTZinzE\\\\\\\/fKnRTaUwmYFjBN9fH5smjSPUGgwlFV5Ls+FOAwgYSZH0BXZJ6LywZVZvYXnTSiIIQ6pdTuZehMhR6lRKkX4XVmQ0yNaa\\\\\\\/9nuw014su\\\\\\\/g2AsU7ayYq37WJk2Mo6PwRKPvD21nrLvkYw3U4jezyk74bW855b2eBTGXIwGPA3xgNgeP7uDe3v5ZGEeaTFZACdEpaNKllskTmablTbbVubs5H5QJzF0ZWrwBC0m202xsYHj4qDhJVc1y8BPQT9cdZCBdSzcq1W\\\\\\\/JrHCIpb1guLx2RyA+4hyjYt37pj9AIxluIrAGBTfAbWeRn87hlz6rSmFcqTZiCPh9H03QA\\\\\\\/FDCsEvpmBhIq9zu7rA4iSCZ69UB5QWHBclvTrOkiMax7uiwYbYGQSfh2vTsiKBBZlhaPFnHqI7fLX\\\\\\\/wvwGnrYqa2s2NtdX\\\\\\\/tpgX3vfC\\\\\\\/9MJmH+aBzPk9OyyaHvDdfb7w+MH7X\\\\\\\/WkZgUSGYXtPCbOatrgmYiglp\\\\\\\/bKYNbAZAhIN2XE7D2gRBfbv1L7HGa7vwHyvKxlY4Jp40ZlpDD4ocA+k1syln5Botbzmh8DFk6E6OKh9DmEuSpf0jIqeCxNVuj32Zy2Ak4VxpLBtmwTAGwgUE9eavQdtGPjOsFKgTAkhhY2+bwKMzpBWgkGEV3hpIsIvmJ+rbWzDDIBR0STJoZEsbWVtSzwUUIOwxabK1AmZKBGAeQMEtC6xoDKJESgvWDeVo6BJ3vcqVm3zMzu1xAWLzpVwHr5KJgKmYvtTGVly6OrSZuosACcHfTSNx+AowEy35V4vt17CU+VZ0ucTGIZZ1NfGYUHAoGMVm1VXQZVtCR+9W26uhZv+BfOvhSVtj2pqNA87cwzCv4ifIlH+oKzQJylBzcgiRTiw7+pX3NyL1rTLKVRo2\\\\\\\/mixICXp8k3PjDvw4Hq95gX1mOdLXmlXHYfXTqWqFY9qtHw3omR3V3tsuCgEZEc9xGBDzvoiTPpR6gKliIQOetHGAPXQOMpIevvYbOv8zoTB2vEjAmw9a3nEP7lmiz4QkRpETJ4gw6Ztro7SGKqaxYnjAYf6ZEsXrk8dEDf+Uh+bNQa4MnRNd2fJqCgZlEhjYJKQl0Bg13DVU29u41yiFh4u69z7BroKDGSJXlhvgY5ZFYygPRwQLXZvklChY0PGCIvgBBxxH9ISniWsdIH\\\\\\\/HlZ+GZ3IILNz1vHN5oMaL6FXIH2IuJIpBrHFF4PxvDGSfVX8Q7uUBEMbAYuKfQALwJoONp5qJESgRW21vXtSI8+4GysJDlP9YinGJjIxJP5bmPZJPSBk3hQYrXjpRnK4vbkHHtqpmCT96XECF57fwLmKMod5TmbLH1P9xeM\\\\\\\/V8UMe\\\\\\\/cS2HCnalGO8n+NYkriiffbqPVR1H8IOaUVLwuaBlLzpOxlpkG4qrkauDfh8zPEwISo3kcH+xMBYo9HTPJ2t\\\\\\\/ZKWsOZS0aseRvxxv97dW2Xu3B1lUp9ieT7lNSWnyw2K1HE5dm2E0YDtX9hdy8GnlUkk7kBR1sKexXtJq5WLJI2v8D+AJXfhIA6Sbs5j0zfvLIemmJfG7dffhLhKb2POvjgIUytAb5++UWS77+1fWwesAj32VC3ocsTqA8ydzR5KbHNcz+xIYZhYTT6x4DxZAdNtGf7weergt+2nuM4HLCG4TMoOyxtlVm3bcmgV0hq3cXq56fGIdbvQFRXAItc1nNf+JCut5Af64hehuOcvCtnqAa8VTMp4F0jPCbQFzaWCqKmZ1HjzNBtjHuyWnsAEU\\\\\\\/OP7qKLdB7Z4aniNs9eSmE988ExlZfvGBx3E8LjGT09NtKsQ2uPK2I1x2c71ZiMS9VqvA1jnTSbenVOsa8MNCsFZOYxxSZKgDCYonyyTa+jRZtxB0lvTEjJrlEmquyex439KFKh6sC363hUikPcbEzpRJWm4v6O9rXA8WI5Wq+r4SZzRp57x+FW4vSkSlJaZF7it8i5k9HHkl+OQuoXPB+G52curvpoyu1mRPoNKC9GqloxJ34mOszQmpWiPgevWvYDnUNuFZpw4X0H0zSmaIV4DksaVq0Mla84342gQfzbMJcgSffJL8CfQGrIDomt0c5KeBJMRdOOSoMV6BMUpUCjq5TXVFstmAvPdWtmzQdhVpyxe+I6Epw6IohCTReEOeiwbrPYELvud8fIuNklwG32fLkdFs5pdU61grbZzg2I16LaXbsu37lDrpYzdqOsZVLWE4MTFvuShTXlPq3krrf7db4zPYPzfMiaBaj5aw6R+MH3JQ99VRrrEY+jkVReO5pmkdFHGEj8GbxsaZfeReKHRCG3o1PJ0L49fKo3psurEAaLhMt9dtvjozh\\\\\\\/W01Ik7pTkWmIoxbNYtfbju6mhXfvjkAOyn7YilLxTFR1cfWrdMw4Zdp99A79UG39kCrzsRk50IAQczZODQGumEt+2RyugFdn7b\\\\\\\/5BeZtlKuwkpZKxqHWO4\\\\\\\/ilM2uC0DGBoyz19gc1fHRXv8J8mYpxwrDi+QPtKpyVwNDNHanpGkGBEwVLAGVwlKHFhli56nt1P6fSY9tZIT+aZKU\\\\\\\/NGiVz+0K0ZE0aASDoxvtWtKpun+polHUsF+uX0FMwmFZRa3OLQTP+0he9El5\\\\\\\/CfYNhx1sS6VQfJ\\\\\\\/MjoXZAEOzHhn3A+g8m\\\\\\\/gzobOiXTm8F4tll8sUsqChtCJIQ8MPwjW+UumPUjBjpqu78qEkct8zyqihd9L7NoLcp4VHOjDunbj4HcGFa7ZU\\\\\\\/HvWf2okcHqKbIxirCarqNtzh0hQ\\\\\\\/j7e3vOazeNLsN9DbOYFgOcF4pyXX5vvbsH\\\\\\\/9PIMhtrEuIpyXv5bndmVjAZfjOkneqnKliloS0p71MAQx+m+B1XV27YXJRcSwey9B\\\\\\\/GnYX1nwWCnoZ9l27b++VIycp6u5p+kAlmprlBtrSMIRv9qIwJsvIkwFw7pJ4rw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022EE6WIgk+mTSE6ueN2l3W9Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002218a13e27916da688fc2ce4376b0b22cec08336615f1bbafc3aa21ed17e0e0d1e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022iEP8LTPMPULyeqidr80soK7FWm8MQ1N7fIRDftGCEknICKtPuYapdewpdqATWoQmlE0w66WWP7AqD46iGL1I45wRrTLH8QQ7vXMfGqYLC48YWY64BGIL6SL6o+fb4LXCYuYnAi8kmwJw+Ru1mEPLeGBn4FGAD3vjUDF8zM5O79SpJ9yPPIBGzz5qwkMsL+Ekhbnms0h4\\\\\\\/poqtqUxzo3VmLT9whqhWcbROLDKyz5IMxdlPv\\\\\\\/JYYYGWUmDI8boloAar9DWvqdsMFbL97CgjBwj7yYiIiZfUXM0gg2p9TZt1iUZnVgzVaYZQs8b7Y4RvW9OCZ6eXM\\\\\\\/q1t1gSYIrOvbydL\\\\\\\/sguU720m9fi+hvQUvRVMmF4I6TvdXdNXzXUe7OibSD6vXwerVyQ1R0y88uracr12rpkSeB9ryI5YqeT\\\\\\\/Ns5wKJxKFtYTtNZN0mJGgj7OiEOPXmol67h16VGykaV8mcJf+AZXHsiQDYW20qxNlUIfoRo87KapIZc3bw7X616nlDEXbj+jR5s6M3dQ7KDqpgsxIR5FP609njVyleMEkDub8Ul\\\\\\\/EyuN4gTRqQ+ea5dUJNctuHdRaF9o1v9fBevuwcbnqSMETypkdK6eHvM6JbR\\\\\\\/TEeYfNaUEEuYC\\\\\\\/eXzh+a1wbL3ygJf9wdQkGzs5q0fPVqHvq8ZcW2grP+mLZTltu+tFqeEjMmem9i7\\\\\\\/+rl\\\\\\\/WWebq8oG1iqC\\\\\\\/Atru5YqpmSZ10FENgBYMXlub3kU5hveLgooOZYRwvlmwJLvMWamu5tPfPffn8Sdh2rql4FhHnqaAiywBw07luNDj4LXQS\\\\\\\/iTxyhBRbNN79fdEeLc6p6QjBuX9dd5y3va+a9Srp+FOiQF9EDg3Ei1EgC83BqLAdfayN0l9aqVtzFc4pF081RYE4hwmAkzOBn0kVKuiqhJs6Hn3q2kE71cRcA8Nh8VmK8NbF\\\\\\\/DgC4y+oxuUvLWDZPthtSAHdajf7DezpP56QjPej3icOeKGZoMQoiK7PUnEUaTRJTip8V+Wuyi5oGWLYdGkygPD5wZJCgm9IXfM3wH9dwmJk7fKBRg+DmurW2uZ2ZKJZa+Bf\\\\\\\/t3+tBNuniSxo0peElOkOEnv7XTrTLC1G+qYnZm9LHivRbCd0X+kNNHvbOarPHo1jHtu+wxI+8x0T9vxM9d2kt\\\\\\\/uI17C+pmI+gkkbMAsU2v45zNzmg25wdkabFoU0z1J0nNKY7rJ0KHwPQpN3G6e2SvUAd3\\\\\\\/C3X\\\\\\\/CDrTCVPmUvf8V3eneT2jIXo3nDc9ncrxh2oK5I6t8muXZFtQm1gqOogdN+HaSjKfmPloRp5NXGhwriTmyco5flExC8HUENpMpBmEOdXc6oVsSDyCrjKIUQNduDz7dNZ\\\\\\\/02A5iSX5K+P\\\\\\\/ZZAltAjWd\\\\\\\/Hlan\\\\\\\/l321eyZY3go37erDgEVEvp8k6E3J40U6hrdZWQUz6W2kZPcKQKBYv43z1o7IRJoLvERFA0JcoBtpGGYaJ8tnBlFLqOcauOLKsbO7tvKKYB1DQ5sxebq3yYExU2LigpMeakfELPcsfSenx3TM34JtJ40K3TsAR29BNtBg4N3V1uw4CX+I1l4XyOpz++Zu7Exjv4lnrR\\\\\\\/aXTsObDNX0cuSB2LEch8qb8JG44YGV7OiqZsFz85syRnLG+uybbkbRkCoIz1P3+N\\\\\\\/LW\\\\\\\/kFo4zIzgwdaQFy93LQI+1nOjFMkWKvQT+MjiJXMomXcCgZpuTYAdt2hl7SN5n8kT53RDfz7cxVTbu\\\\\\\/9Bo3ITWtD4QgANljSG4cKsABdM1CS1MRyqBhe3Vq31fSSgWy52h9ykNcOGt5k2hymLuZDk0pHgkKPXpQNLocnPSA+JPhCJSOrY0MzwqoSWBjX\\\\\\\/MEzequJ5ZIen+u8Ssny3lY4NxKB2pH5c1v2PvhmV7S4xftIIBhFLA27KxffhllfQ2NVVpBBCAsHCyJc2\\\\\\\/DTd7IeP0VIjr8ioXDotXfhTSvrLcnFHuB8klMuu0o4uYRWARup2O6O094EE9jKbO\\\\\\\/byFzYkrmwM09tbNhfQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022rbCXca5QFnTejP6Ezn8lJg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222a82d95f92491caa1cc9d2df22932a072878b4e5544c856c1697b3309c88207b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491970 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-software-and-data-resources">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/1/1" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Y8UrLDTEa4erUVLfn9Qj3yt2dKm2wwDWVs5piboqUY8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/1/1" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tjZj9R6_MNAWnAbRzfJoA8B22AAh4jKfOlHwK-lGh2Y" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3993" title="G3 Reviewer Index 2018" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/1/13" title="OCMA: Fast, Memory-Efficient Factorization of Prohibitively Large Relationship Matrices" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491970" id="node491970" data-pisa="ggg;9/1/1" data-pisa-master="ggg;g3.118.200637" data-apath="/ggg/9/1/1.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">iProteinDB: An Integrative Database of <em>Drosophila</em> Post-translational Modifications</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1494-1402" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yanhui Hu</span>, <span class="highwire-citation-author" data-delta="1">Richelle Sopko</span>, <span class="highwire-citation-author" data-delta="2">Verena Chung</span>, <span class="highwire-citation-author" data-delta="3">Marianna Foos</span>, <span class="highwire-citation-author" data-delta="4">Romain A. Studer</span>, <span class="highwire-citation-author" data-delta="5">Sean D. Landry</span>, <span class="highwire-citation-author" data-delta="6">Daniel Liu</span>, <span class="highwire-citation-author" data-delta="7">Leonard Rabinow</span>, <span class="highwire-citation-author" data-delta="8">Florian Gnad</span>, <span class="highwire-citation-author" data-delta="9">Pedro Beltrao</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0001-7542-472X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Norbert Perrimon</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1-11;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200637 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491970" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yanhui Hu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span> RNAi Screening Center, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yanhui%2BHu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYanhui%2BHu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1494-1402" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yanhui Hu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Richelle Sopko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richelle%2BSopko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sopko%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichelle%2BSopko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Verena Chung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span> RNAi Screening Center, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Verena%2BChung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chung%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVerena%2BChung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marianna Foos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marianna%2BFoos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Foos%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarianna%2BFoos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Romain A. Studer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>European Molecular Biology Laboratory (EMBL), European Bioinformatics Institute, Wellcome Genome Campus, Hinxton, Cambridge CB10 1SD, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Romain%2BA.%2BStuder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Studer%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARomain%2BA.%2BStuder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sean D. Landry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Bioinformatics, Cell Signaling Technology Inc., 3 Trask Lane, Danvers, MA 01923</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BD.%2BLandry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Landry%20SD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BD.%2BLandry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Daniel Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span> RNAi Screening Center, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Leonard Rabinow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonard%2BRabinow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rabinow%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeonard%2BRabinow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Florian Gnad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Bioinformatics, Cell Signaling Technology Inc., 3 Trask Lane, Danvers, MA 01923</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Florian%2BGnad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gnad%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFlorian%2BGnad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Pedro Beltrao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>European Molecular Biology Laboratory (EMBL), European Bioinformatics Institute, Wellcome Genome Campus, Hinxton, Cambridge CB10 1SD, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pedro%2BBeltrao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beltrao%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APedro%2BBeltrao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Norbert Perrimon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span> RNAi Screening Center, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Howard Hughes Medical Institute, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Norbert%2BPerrimon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Perrimon%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANorbert%2BPerrimon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7542-472X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Norbert Perrimon</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:perrimon@receptor.med.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">perrimon@receptor.med.harvard.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/1/1" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491970" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491970/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/1/1.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491970" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491970/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/1/1.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:491970" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:491970/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/1/1.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491970" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491970/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/1/1.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491970" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491970/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Post-translational modification (PTM) serves as a regulatory mechanism for protein function, influencing their stability, interactions, activity and localization, and is critical in many signaling pathways. The best characterized PTM is phosphorylation, whereby a phosphate is added to an acceptor residue, most commonly serine, threonine and tyrosine in metazoans. As proteins are often phosphorylated at multiple sites, identifying those sites that are important for function is a challenging problem. Considering that any given phosphorylation site might be non-functional, prioritizing evolutionarily conserved phosphosites provides a general strategy to identify the putative functional sites. To facilitate the identification of conserved phosphosites, we generated a large-scale phosphoproteomics dataset from <em>Drosophila</em> embryos collected from six closely-related species. We built iProteinDB (<a href="https://www.flyrnai.org/tools/iproteindb/">https://www.flyrnai.org/tools/iproteindb/</a>), a resource integrating these data with other high-throughput PTM datasets, including vertebrates, and manually curated information for <em>Drosophila</em>. At iProteinDB, scientists can view the PTM landscape for any <em>Drosophila</em> protein and identify predicted functional phosphosites based on a comparative analysis of data from closely-related <em>Drosophila</em> species. Further, iProteinDB enables comparison of PTM data from <em>Drosophila</em> to that of orthologous proteins from other model organisms, including human, mouse, rat, <em>Xenopus tropicalis</em>, <em>Danio rerio</em>, and <em>Caenorhabditis elegans</em>.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/drosophila" class="hw-term hw-article-keyword hw-article-keyword-drosophila" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Drosophila</em></a></li><li class="kwd"><a href="/keyword/post-translational-modification" class="hw-term hw-article-keyword hw-article-keyword-post-translational-modification" rel="nofollow">post-translational modification</a></li><li class="kwd"><a href="/keyword/phosphoproteomics" class="hw-term hw-article-keyword hw-article-keyword-phosphoproteomics" rel="nofollow">phosphoproteomics</a></li></ul><p id="p-5">Post-translational modification is essential for the regulation of many cellular processes. For example, phosphorylation can serve as a molecular switch for signal transduction (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Beurel <em>et al.</em> 2015</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hunter 2000</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kockel <em>et al.</em> 2010</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nagini <em>et al.</em> 2018</a>). Based on the annotation of PhosphoSitePlus (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hornbeck <em>et al.</em> 2012</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hornbeck <em>et al.</em> 2015</a>), the average number of phosphosites per protein is twelve for the human and seven for the mouse proteome. Evolutionary studies of protein phosphorylation have suggested that a significant fraction of these phosphosites may be non-functional (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Beltrao <em>et al.</em> 2013</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Landry <em>et al.</em> 2009</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Studer <em>et al.</em> 2016</a>) whereas evolutionarily conserved phosphosites are often highly relevant for function (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Studer <em>et al.</em> 2016</a>), as evidenced, for example, by the Mitogen Activated Protein Kinase (MAPK) or Extracellular Regulated Kinase (ERK) families (<em>i.e.</em>, ERK/MAPK, JNK, p38). Generally the activation of these kinases requires phosphorylation within the sequence, TxY, residing within the “T loop” of the catalytic domain by an upstream MAPK-K/MEK kinase. Upon phosphorylation, the activation loop moves away from the active site, allowing substrate entry and phosphorylation. The TxY motif is conserved in the vast majority of MAPK/ERK family members, from yeast to man, allowing, for example, the generation of an antibody specific for the phosphorylated, active form of MAPK/ERK (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gabay <em>et al.</em> 1997</a>). Another example of a highly conserved phosphosite includes ribosomal protein S6 (rpS6), which is conserved in essentially all eukaryotes, including yeast, plants, invertebrates, and vertebrates. The physiological roles of phosphorylation at Ser235/236 of rpS6 remained unclear until genetic approaches abolishing the phosphorylation sites were applied in model organisms (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Meyuhas 2015</a>). These examples highlight how conservation can illuminate phosphosite function and how model organisms can play crucial roles in the elucidation of their functions.</p><p id="p-6">Mass spectrometry (MS)-based proteomics is a powerful approach for large-scale identification and characterization of phosphorylation sites. Three large-scale <em>Drosophila melanogaster</em> phospho-proteomic datasets have been generated over the past years using MS. Two datasets were generated from cultured cells (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bodenmiller <em>et al.</em> 2007</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hilger <em>et al.</em> 2009</a>) and one was generated from embryos (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Zhai <em>et al.</em> 2008</a>). Because the coverage of each dataset is limited, and to further characterize the breadth of phosphorylation in <em>Drosophila</em>, we generated a new dataset for <em>Drosophila melanogaster</em> and five related species: <em>Drosophila simulans</em>, <em>Drosophila yakuba</em>, <em>Drosophila ananassae</em>, <em>Drosophila pseudoobscura</em>, and <em>Drosophila virilis</em>. To facilitate the use of this dataset, we built an online resource, iProteinDB, integrating our data with other large-scale PTM data (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Bodenmiller <em>et al.</em> 2007</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Hilger <em>et al.</em> 2009</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Zhai <em>et al.</em> 2008</a>) and curated PTM annotations for <em>Drosophila</em> and other model organisms. At iProteinDB, users are able to align PTM data for any protein of interest from multiple resources, including data from the six <em>Drosophila</em> species, other model organisms, and human cells. Additional relevant information, such as disease-related protein variants, sub-cellular localization, and protein abundance during <em>Drosophila</em> development, is also provided at iProteinDB.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Generation of phosphoproteomics data</h3><p id="p-7">Pre-larval embryos of mixed sex and age from each of the six <em>Drosophila</em> species were collected. Since different species develop at different speeds, the timing of collection was different for each species. Flies were enticed to lay eggs by incubating in the dark on grape juice plates. Proteins from embryos lysed in 8 M urea were digested with trypsin and separated into 12 fractions by strong cation exchange chromatography. Phosphopeptides were purified with titanium dioxide microspheres and analyzed via LC-MS/MS on either an LTQ-Orbitrap or Orbitrap Fusion instrument (Thermo Scientific). SEQUEST was used for spectral matching. Peptides were filtered to a 1% FDR. Proteins were filtered to achieve a 2% final protein FDR (final peptide FDR near 0.15%) and a probability-based scoring method was used to assign the localizations of phosphorylation events (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beausoleil <em>et al.</em> 2006</a>). The reference genomes used for initial analysis are <em>D. mel</em> r5.53, <em>D. ana</em> r1.03, <em>D. pse</em> r3.01, <em>D. sim</em> r1.04, <em>D. vir</em> r1.02 and <em>D. yak</em> r1.03 from FlyBase. The sites were re-mapped to <em>D. mel</em> r6.16, <em>D. ana</em> r1.05, <em>D. pse</em> r3.04, <em>D. sim</em> r2.02, <em>D. vir</em> r1.06, <em>D. yak</em> r1.05 at iProteinDB.</p></div><div id="sec-3" class="subsection"><h3>Predicting the probability of phosphorylation</h3><p id="p-8">We aligned the phosphorylation sites identified in our datasets from 6 <em>Drosophila</em> species along with other sequenced <em>Drosophila</em> and mosquito species based on orthologous relationships predicted by OMA (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altenhoff <em>et al.</em> 2018</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altenhoff <em>et al.</em> 2011</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Altenhoff <em>et al.</em> 2015</a>). For each proteome, we assign the probability of a phosphoacceptor (S+T (together) and Y) to be phosphorylated, using a two-step approach. First, we scan each proteome to find the kinase specificity of each phosphoacceptor, using NetPhorest (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Horn <em>et al.</em> 2014</a>). This provided 40 scores for kinase specificity for a given region. Then, a support vector machine algorithm (SVM-light) was trained on each of the six species, using all 40 scores. We extracted the surrounding region of sites that are detected to be phosphorylated, and they received an initial score of 1 (positive dataset). The regions surrounding non-detected phosphosites received an initial score of 0 (negative dataset). We sample the data (2000 for S+T and 800 for Y) to train the model, and then assign scores to unknown phosphosites based on the support vector machine output (detected phosphosites by MS always received a score 1, irrelevant of their prediction).</p></div><div id="sec-4" class="subsection"><h3>Comparison of PTM data across major model organisms and phylogenetic analysis</h3><p id="p-9">Orthologous relationships of <em>Drosophila melanogaster</em> proteins to major model organisms including human, mouse, rat, <em>Xenopus tropicalis</em>, zebrafish and <em>Caenorhabditis elegans</em> were obtained using DIOPT (release 7) with a DIOPT score of 3 or higher (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hu <em>et al.</em> 2011</a>). Protein sequences of the best orthologous genes based on DIOPT score and each non-redundant isoform of <em>Drosophila melanogaster</em> proteins were aligned using MAFFT (vs 7.305B) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Katoh <em>et al.</em> 2002</a>). Observed PTM sites, conserved phosphorylation sites, domain and disease-related protein variants are annotated on the aligned protein sequences (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). To compare specific PTM sites, the sequence of a sliding window of five amino acids surrounding the identified phosphosite was extracted and compared across species. The number of identical amino acids was counted and the percent of identity was calculated by dividing the number of identical amino acids over the window length. Phylogenetic trees for protein kinases were generated with Jalview 2.10 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Waterhouse <em>et al.</em> 2009</a>) and illustrated in iTOL (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Letunic and Bork 2016</a>). Notably, total protein sequences were used to assess sequence similarities between kinases for phylogenetic analysis.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/1/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Database content and statistics. Distribution of 168,997 observed PTMs in the proteomics dataset (A). Representation of different types of PTMs (B). Overlap of phosphorylation data for Drosophila melanogaster from five different sources (C). Distribution of phosphorylation sites observed at three phospho-acceptor residues (serine (S), threonine (T) and tyrosine (Y)) within protein domains and their conservation based on at least 50% similarity to human sequence, considering a sliding window of five amino acids (D)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-25006903" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Database content and statistics. Distribution of 168,997 observed PTMs in the proteomics dataset (A). Representation of different types of PTMs (B). Overlap of phosphorylation data for &lt;em&gt;Drosophila melanogaster&lt;/em&gt; from five different sources (C). Distribution of phosphorylation sites observed at three phospho-acceptor residues (serine (S), threonine (T) and tyrosine (Y)) within protein domains and their conservation based on at least 50% similarity to human sequence, considering a sliding window of five amino acids (D).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/1/F1.medium.gif" width="440" height="344"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/1/1/F1.medium.gif" width="440" height="344"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/1/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/1/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491879" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Database content and statistics. Distribution of 168,997 observed PTMs in the proteomics dataset (A). Representation of different types of PTMs (B). Overlap of phosphorylation data for <em>Drosophila melanogaster</em> from five different sources (C). Distribution of phosphorylation sites observed at three phospho-acceptor residues (serine (S), threonine (T) and tyrosine (Y)) within protein domains and their conservation based on at least 50% similarity to human sequence, considering a sliding window of five amino acids (D).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Source of other data sets or tools</h3><p id="p-11">Protein information of 6 <em>Drosophila</em> species was obtained from FlyBase (<a href="ftp://ftp.flybase.net/releases/FB2017_03/">ftp://ftp.flybase.net/releases/FB2017_03/</a>). Protein information of human, mouse, rat, <em>Xenopus tropicalis</em>, zebrafish and <em>Caenorhabditis elegans</em> were obtained from RefSeq (<a href="https://www.ncbi.nlm.nih.gov/refseq/">https://www.ncbi.nlm.nih.gov/refseq/</a>). Other high throughput datasets of <em>Drosophila melanogaster</em> were obtained either from online resources (Phosida: <a href="http://141.61.102.18/phosida/index.aspx">http://141.61.102.18/phosida/index.aspx</a> and Phosphopep: <a href="http://www.phosphopep.org/">http://www.phosphopep.org/</a>) or corresponding supplemental tables of relevant publications (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Zhai <em>et al.</em> 2008</a>). The protein annotation of Swiss-prot and TrEMBL was downloaded from the UniProt FTP site (<a href="http://www.uniprot.org/downloads">http://www.uniprot.org/downloads</a>). Orthologous relationships were obtained from OMA (<a href="https://omabrowser.org/oma/home/">https://omabrowser.org/oma/home/</a>) and DIOPT (<a href="http://www.flyrnai.org/diopt">http://www.flyrnai.org/diopt</a>). Protein domain annotation of Conserved Domain Database (<a href="https://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml">https://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml</a>) was extracted from the RefSeq release files (.gbff files). Kinase motifs were predicted using the API of Scansite3 (<a href="http://scansite3.mit.edu/#home">http://scansite3.mit.edu/#home</a>). PTM annotation of orthologous genes other than <em>Drosophila</em> was obtained from PhosphoSitePlus (<a href="https://www.phosphosite.org/staticDownloads.action">https://www.phosphosite.org/staticDownloads.action</a>).</p></div><div id="sec-6" class="subsection"><h3>Implementation of the online resource</h3><p id="p-12">There were several steps involved that process the information and populate the back-end database of iProteinDB. After downloading data from relevant sources, such as UniProt, PhophoSitePlus and various publications, the extraction of relevant information was accomplished with in-house parsers written in Perl and Python. The redundancy of protein sequences was consolidated and a collection of distinct protein sequences from each <em>Drosophila</em> species was assembled based on FlyBase genome release (<em>D. mel</em> r6.16, <em>D. ana</em> r1.05, <em>D. pse</em> r3.01, <em>D. sim</em> r1.04, <em>D. vir</em> r1.02, <em>D. yak</em> r1.03). Since different resources annotate data based on different genome releases, we synchronized the data from various sources by mapping the original data (peptides) to the non-redundant protein collection of recent FlyBase genome release (see above) using the SeqIO interface of BioPython. Once filtered and updated, the data were then uploaded into a MySQL database, which is currently hosted by the Harvard Medical School (HMS) Research Computing group.</p><p id="p-13">To display the data, we created a web-based application with PHP and a PHP framework called Symfony (version 2.6). Several client-side functions rely on JavaScript and AJAX, while some tabular displays use a jQuery plugin called DataTables.js, which allow for sorting and paging functionalities within the tables. This web application is also hosted by the HMS Research Computing group.</p></div><div id="sec-7" class="subsection"><h3>Data Availability</h3><p id="p-14">iProteinDB is available for online use without any restrictions at <a href="https://www.flyrnai.org/tools/iproteindb/">https://www.flyrnai.org/tools/iproteindb/</a>. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6986945">https://doi.org/10.25387/g3.6986945</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Data integration and quality of six Drosophila phosphoproteomes</h3><p id="p-15">Embryos from six <em>Drosophila</em> species were collected, proteins were extracted and digested, phosphopeptides were isolated, and these samples were then subjected to ionization and fragmentation for identification and phosphosite determination using a mass-spec based method described previously (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sopko <em>et al.</em> 2014</a>). The data coverage ranges from 14,915 to 21,750 sites per species (Supplementary Table 1) and motif analysis of the data (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Ullah <em>et al.</em> 2016</a>) shows that the most significant motifs of phosphosites in 6 <em>Drosophila</em> species are quite similar (Supplementary Figure 1). The orthologous relationships among the six <em>Drosophila</em> species, as well as other sequenced <em>Drosophila</em> species and mosquito species (Supplementary Figure 2), were predicted using the OMA algorithm (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Altenhoff <em>et al.</em> 2018</a>), which infers orthologous genes among multiple genomes on the basis of protein sequence. Based on the multiple-sequence alignment of each orthologous group, the aligned positions were selected, for which phosphorylation was observed in at least one of the six <em>Drosophila</em> species. Given that the mass-spec based identification of phosphosites is incomplete, we filled the gaps with machine learning predictions, using a similar approach as in (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Studer <em>et al.</em> 2016</a>). A support vector machine (SVM) algorithm was trained to assign a propensity score of 0-1 to each corresponding phospho-acceptor residue (serine, threonine, or tyrosine) for each species for which that residue was not identified as phosphorylated, based on the likelihood of phosphorylation. This information is available at the iProteinDB resource (see below) to help researchers interested in identifying evolutionary conserved phosphorylation sites. We next compared the propensity score with phosphoproteomics data from other sources. We found a strong correlation between the propensity score and the chance that a predicted site was phosphorylated, as supported by independent datasets (Supplementary Figure 3a). We also compared the frequency of phosphorylation among the six <em>Drosophila</em> species with experimental data for orthologous human proteins. Not surprisingly, phosphorylation sites conserved among the six <em>Drosophila</em> species were more likely to be reported as phosphorylated at the corresponding sites in orthologous human proteins. This correlation was more prevalent for those sites with greater than 50% amino acid similarity between <em>Drosophila</em> and human orthologs (Supplementary Figure 3b).</p><p id="p-16">We estimated the false negative rate for each of the six <em>Drosophila</em> species by selecting those sites that are 100% identical (considering eleven amino acid peptides comprising the phosphosite plus five amino acids upstream and downstream) among all six species and for which phosphorylation was observed in at least two species. The false negative rate is estimated to be the percent of the sites that are not covered by the data in each species. For example, 86% of these sites for <em>Drosophila melanogaster</em> are covered by at least one of the 4 datasets, and/or UniProt annotation so the false negative rate is about 14%, while there is only 1 dataset for each of the other 5 <em>Drosophila</em> species, and therefore, the false negative rate is relatively higher, 44–79% (Supplementary Figure 3c).</p></div><div id="sec-10" class="subsection"><h3>Integration of phosphoproteomes from other resources</h3><p id="p-17">We built the iProteinDB database to store phosphoproteomics data generated by our group and other large PTM datasets. Other datasets were obtained from the supplemental table of the original publications (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Zhai <em>et al.</em> 2008</a>) or the relevant websites (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bodenmiller <em>et al.</em> 2008</a>; <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Bodenmiller <em>et al.</em> 2007</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gnad <em>et al.</em> 2011</a>). Original data were mapped to the same version of the FlyBase proteome annotation (FB2017_03) and then integrated with our data in iProteinDB. The information of PTM sites and the score/peptide from the original source are stored and made available at the iProteinDB website. To compare PTM data across species, we integrated orthologous relationships of <em>Drosophila</em> species as predicted by OMA (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Altenhoff <em>et al.</em> 2018</a>), the orthologous relationships among major model organisms predicted by DIOPT (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Hu <em>et al.</em> 2011</a>), and PTM data for other species from PhosphoSitePlus (<a href="https://www.phosphosite.org">https://www.phosphosite.org</a>) (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hornbeck <em>et al.</em> 2012</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Hornbeck <em>et al.</em> 2015</a>). The subcellular localization annotation and human disease related protein variants were integrated from UniProt (<a href="https://www.uniprot.org/">https://www.uniprot.org/</a>), whereas protein domain annotation information was integrated from the National Center for Biomedical Information (NCBI) Conserved Domain Database (<a href="https://www.ncbi.nlm.nih.gov/cdd">https://www.ncbi.nlm.nih.gov/cdd</a>). Information about protein abundance during <em>Drosophila</em> development was also integrated from a recent publication (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Casas-Vila <em>et al.</em> 2017</a>).</p><p id="p-18">Altogether, iProteinDB covers 168,997 individual PTMs for <em>Drosophila</em>, of which 70,013 (41%) were observed in <em>Drosophila melanogaster</em> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1a</a>). 62,239 (89%) of the <em>Drosophila melanogaster</em> PTM data collected in iProteinDB are phosphorylation sites, covering 8,068 unique proteins and 3,937 genes (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1b</a>). Comparing our <em>Drosophila melanogaster</em> phosphoproteomics data with that from other sources, we find that 61% of our data overlaps with one other source and 36% of our data overlaps with at least 2 other sources (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Overall, 37% of the phosphorylation data are supported by multiple resources and thus can be considered high confidence (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1c</a>, <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491849/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491849/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Integrated phosphoproteomics data for <em>Drosophila melanogaster</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Online resource</h3><p id="p-20">Users can query <em>Drosophila</em> genes of interest, and choose one isoform if there are multiple non-redundant isoforms for the gene of the interest. There are three tabs from which to choose (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/1/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Features of iProteinDB user interface. Observed PTM sites are marked in red on the Drosophila melanogaster protein sequence. Predicted phosphosites based on phospho-proteomic data from five other Drosophila species are marked in blue. Sites observed in more than one Drosophila species are underlined. The protein domains are highlighted in green. The data sources of PTMs are summarized. At the &#x201C;Predicted Orthologs&#x201D; page, the multiple sequence alignment of orthologous genes of major model organisms and human are displayed with observed sites color-coded (red arrows), conserved sites bolded (brown arrow) and human disease variant mutations underlined (navy arrows)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-25006903" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Features of iProteinDB user interface. Observed PTM sites are marked in red on the &lt;em&gt;Drosophila melanogaster&lt;/em&gt; protein sequence. Predicted phosphosites based on phospho-proteomic data from five other &lt;em&gt;Drosophila&lt;/em&gt; species are marked in blue. Sites observed in more than one &lt;em&gt;Drosophila&lt;/em&gt; species are underlined. The protein domains are highlighted in green. The data sources of PTMs are summarized. At the &#x201C;Predicted Orthologs&#x201D; page, the multiple sequence alignment of orthologous genes of major model organisms and human are displayed with observed sites color-coded (red arrows), conserved sites bolded (brown arrow) and human disease variant mutations underlined (navy arrows).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/1/F2.medium.gif" width="440" height="348"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/1/1/F2.medium.gif" width="440" height="348"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/1/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/1/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491938" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Features of iProteinDB user interface. Observed PTM sites are marked in red on the <em>Drosophila melanogaster</em> protein sequence. Predicted phosphosites based on phospho-proteomic data from five other <em>Drosophila</em> species are marked in blue. Sites observed in more than one <em>Drosophila</em> species are underlined. The protein domains are highlighted in green. The data sources of PTMs are summarized. At the “Predicted Orthologs” page, the multiple sequence alignment of orthologous genes of major model organisms and human are displayed with observed sites color-coded (red arrows), conserved sites bolded (brown arrow) and human disease variant mutations underlined (navy arrows).</p><div class="sb-div caption-clear"></div></div></div><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-22">Protein detail tab. A user can view the protein sequence from any of the 6 <em>Drosophila</em> species in FASTA format. PTM sites are color-coded. The amino acid is displayed in red if the PTM is observed or blue if it was not observed but is predicted to be phosphorylated based on the data from one or more different <em>Drosophila</em> species. The amino acid is underlined if the phosphorylation event was observed in more than one <em>Drosophila</em> species. Protein domains are highlighted in green. A table summarizing all the PTM sites for a given protein, as well as the data sources from which the information was extracted, is provided, along with detailed information from the original sources, <em>i.e.</em>, the original scores and peptide sequences. A table summarizing all predicted sites based on data from closely related <em>Drosophila</em> species is provided with a link to detailed information and multiple sequence alignments. Also indicated in this tab is sub-cellular localization annotation from UniProt for each phosphoprotein and kinase predicted to act on individual sites, as identified using ScanSite3 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Obenauer <em>et al.</em> 2003</a>).</p></li><li id="list-item-2"><p id="p-23">Predicted ortholog tab. Users can find a table of the best ortholog candidates for major model organisms based on DIOPT ortholog predictions (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Hu <em>et al.</em> 2011</a>). Multiple sequence alignments were performed based on the protein sequences of orthologous genes. The sequences of all the aligned <em>Drosophila</em> phosphosites, over a sliding window of five residues, were compared to the corresponding sequences of each orthologous gene and a similarity score was calculated by pair-wise comparison. For example, if 10 of the 11 amino acids (phosphorylation site plus five amino acids upstream and downstream) are identical between <em>Drosophila</em> and human sites, the similarity score was assigned as 0.9 (10 divided by 11). Then, an average similarity score was calculated based on all pairwise combinations at a given site. All phosphorylation sites with an average similarity score of &gt;0.5 are listed and summarized as conserved sites. Human disease-related variants annotated at UniProt are also listed, along with sub-cellular localization annotation of all orthologous proteins from UniProt. Multiple sequence alignment (MSA) across major model organisms is displayed. For MSAs, observed PTM sites for all orthologous genes are color-coded, domains are highlighted, and disease variants are underlined. Conserved sites are bolded. As we hope that iProteinDB will lead to new discoveries and hypotheses on previously uncharacterized phosphorylation events, we further integrated information on availability of corresponding antibodies from Cell Signaling Technology for proteins and sites that are homologous between <em>Drosophila</em> and human to help users with experimental designs.</p></li><li id="list-item-3"><p id="p-24">Protein abundance tab. Protein expression levels from a comprehensive proteomic study covering the complete <em>Drosophila melanogaster</em> life cycle (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Casas-Vila <em>et al.</em> 2017</a>) are plotted. On this tab, a user can view the stages of the <em>Drosophila</em> life cycle during which a protein of interest is expressed.</p></li></ol></div><div id="sec-12" class="subsection"><h3>The Drosophila kinomes and their substrates show significant evolutionary conservation</h3><p id="p-25">The integration of six <em>Drosophila</em> phosphoproteomes along with ortholog information enabled us to determine the conservation of the <em>Drosophila</em> kinome and to assess the evolutionary selective pressure on its substrates.</p><p id="p-26">We found that virtually the entire <em>Drosophila melanogaster</em> kinome as defined by Manning and colleagues (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Manning <em>et al.</em> 2002</a>) shows orthologous counterparts in the other <em>Drosophila</em> species based on the OMA algorithm (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>: Tyrosine kinases, Supplementary Figure 4: Serine/Threonine kinases.). The few exceptions, such as the absence of an orthologous Abl tyrosine kinase in <em>Drosophila simulans</em>, might trace back to poor genome sequence quality or annotation. Additionally, and consistent with previous observations (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Manning <em>et al.</em> 2002</a>) nearly all <em>Drosophila</em> kinases showed strong evidence for orthologous counterparts in at least one of the six integrated model organisms. Among the 32 Tyrosine protein kinases, only Tie-like receptor, Ack-like, and Wsck, showed poor or no homology in higher eukaryotes other than additional <em>Drosophila</em> species, based on DIOPT prediction using a score of 3 or more as a cutoff. Tellingly, orthologs of these kinases were identified in <em>Caenorhabditis elegans</em>, suggesting that they may have been lost prior to origination and evolution of the vertebrate lineage. On the other hand, <em>Drosophila</em> Tyr kinase orthologs not observed in <em>Caenorhabditis elegans</em> but present in vertebrates may be due to evolution of the particular kinase only after divergence of its last common ancestor with <em>Caenorhabditis elegans</em>, as for example <em><a href="http://flybase.org/reports/FBgn0004864.html">hop</a></em>, the <em>Drosophila</em> Janus kinase. As for the Ser/Thr kinases, the vast majority of kinases possess recognized orthologs in other <em>Drosophila</em> species and major model organisms. “Missing” orthologs are again most often likely due to poor sequence annotation or quality. For example the lack of a recognized <em><a href="http://flybase.org/reports/FBgn0011739.html">wts</a></em> ortholog in <em>Xenopus tropicalis</em> is likely to be artifactual, given the importance of the <em>Hippo-warts</em> pathway in the growth and regulation of organ size and its presence in not only <em>Drosophila</em> but all vertebrate species.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/1/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evolutionary relationships among Drosophila melanogaster tyrosine kinases. The core of the plot illustrates the phylogenetic relationships among Drosophila melanogaster tyrosine kinases estimated by total sequence similarity. The outer circle reflects the presence of orthologs in other species. Relationships among Drosophila melanogaster Serine/Threonine kinases are presented in Supplementary Figure 4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-25006903" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Evolutionary relationships among &lt;em&gt;Drosophila melanogaster&lt;/em&gt; tyrosine kinases. The core of the plot illustrates the phylogenetic relationships among &lt;em&gt;Drosophila melanogaster&lt;/em&gt; tyrosine kinases estimated by total sequence similarity. The outer circle reflects the presence of orthologs in other species. Relationships among &lt;em&gt;Drosophila melanogaster&lt;/em&gt; Serine/Threonine kinases are presented in Supplementary Figure 4.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/1/F3.medium.gif" width="437" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/1/1/F3.medium.gif" width="437" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/1/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/1/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491862" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Evolutionary relationships among <em>Drosophila melanogaster</em> tyrosine kinases. The core of the plot illustrates the phylogenetic relationships among <em>Drosophila melanogaster</em> tyrosine kinases estimated by total sequence similarity. The outer circle reflects the presence of orthologs in other species. Relationships among <em>Drosophila melanogaster</em> Serine/Threonine kinases are presented in Supplementary Figure 4.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">The high conservation of the <em>Drosophila</em> kinome within flies and across other eukaryotes suggests that the corresponding substrates are also significantly conserved.</p><p id="p-29">To corroborate this hypothesis at the protein level, we determined the proportion of phosphorylated <em>Drosophila melanogaster</em> proteins that show orthologs in other species, and we used the conservation of <em>Drosophila melanogaster</em> proteins which have not been found to be phosphorylated as a control to assess significance. We assume that this control set is indeed enriched for proteins that do not present kinase substrates. While phosphorylated and non-phosphorylated <em>Drosophila melanogaster</em> proteins have the same proportion of orthologs (99%) as the close relative <em>Drosophila simulan</em><strong><em>s</em></strong>, the phosphoproteome showed significantly higher conservation in more distantly related species, ranging from <em>Drosophila yakuba</em> (phospho: 99%, control: 97%; <em>P</em> &lt; 0.01 based on two-sided Fisher Exact test) to <em>Caenorhabditis elegans</em> (phospho: 70%, control: 46%; p <sup>&lt;</sup> 0.01) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4a</a>). This suggests that not only the kinome but also its substrates are more conserved than other proteins. The significant conservation of the identified phosphoproteome might, however, also be partly driven by an enrichment of highly expressed proteins in the phosphoset and the presence of potential pseudogenes and predicted proteins in the control set. We therefore analyzed the conservation of phosphorylated <em>vs.</em> non-phosphorylated residues of identified phosphoproteins, and found that phosphorylated residues show significantly higher conservation within all <em>Drosophila</em> species (<em>P</em> &lt; 0.01) but not in more distant species (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4b</a>). Similar trends have been reported for other eukaryotic phosphoproteomes. For example, human phosphosites have significantly higher conservation in mammals and other higher eukaryotes than in comparison with distant species including <em>Caenorhabditis elegans</em> or <em>Saccharomyces cerevisiae</em>. (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gnad <em>et al.</em> 2010</a>). The prevalent localization of phosphorylation sites in fast evolving loop and hinge regions of proteins (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Iakoucheva <em>et al.</em> 2004</a>) might make it difficult to map the associated site in aligned disordered regions of distantly related species. In contrast, non-phosphorylated serines, threonines, and tyrosines are not restricted to localization on the protein surface, and therefore tend to occur in more structured and slower evolving regions on the protein (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gnad <em>et al.</em> 2007</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/1/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Conservation of phosphorylated proteins and sites. The line plot illustrates the proportions of Drosophila melanogaster phosphoproteins (blue) and non-phosphoproteins (orange) showing orthologs in other species (A). The line plot shows the proportions of conserved Drosophila melanogaster phosphosites (blue) and non-phosphorylated serines, threonines, and tyrosines (orange) across species (B)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-25006903" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Conservation of phosphorylated proteins and sites. The line plot illustrates the proportions of &lt;em&gt;Drosophila melanogaster&lt;/em&gt; phosphoproteins (blue) and non-phosphoproteins (orange) showing orthologs in other species (A). The line plot shows the proportions of conserved &lt;em&gt;Drosophila melanogaster&lt;/em&gt; phosphosites (blue) and non-phosphorylated serines, threonines, and tyrosines (orange) across species (B).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/1/F4.medium.gif" width="261" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/1/1/F4.medium.gif" width="261" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/1/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/1/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491866" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">Conservation of phosphorylated proteins and sites. The line plot illustrates the proportions of <em>Drosophila melanogaster</em> phosphoproteins (blue) and non-phosphoproteins (orange) showing orthologs in other species (A). The line plot shows the proportions of conserved <em>Drosophila melanogaster</em> phosphosites (blue) and non-phosphorylated serines, threonines, and tyrosines (orange) across species (B).</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">In summary, we found significant conservation of the kinome and substrate proteins across all species. Similarly phosphorylated sites are significantly conserved within <em>Drosophila</em>, but more difficult to trace back in distant species.</p></div><div id="sec-13" class="subsection"><h3>Conservation between the Drosophila and the human phosphoproteomes underlines the utility of using the former as a model system</h3><p id="p-32">To assess the utility of the <em>Drosophila</em> phosphoproteome as a model for human phosphorylation events, we examined the evolutionary conservation of their phosphorylated sites with a focus on localization in functional domains and association with diseases. Approximately 17% of identified phosphosites in <em>Drosophila melanogaster</em> proteins are located within annotated protein domains, and 82% of the identified phosphosites reside in proteins for which the corresponding <em>Drosophila</em> genes are conserved with human, based on DIOPT prediction using a score of 3 or more as cutoff (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Hu <em>et al.</em> 2011</a>). After analyzing the alignment of these Drosophila and human proteins, the corresponding human amino acid sequences for 23% of the Drosophila sites are also phospho-acceptors, 3201 of which have 50% or more sequence identity over the surrounding amino acids to Drosophila. Of the three phospho-acceptor residues, serine has the highest percentage of phosphorylation (Supplementary Figure 5) while phospho-tyrosine has the highest probability of residing within a defined protein domain and the highest sequence similarity with human orthologs (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1d</a>). Further analysis showed that the sequence identity of PTM sites between human and <em>Drosophila</em> melanogaster correlates with the probability that the associated phosphorylation event has also been observed in human cell phospho-proteomic datasets (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5a</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/1/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of the conservation of PTM sites of Drosophila melanogaster. Correlations of sequence conservation and observed phosphorylation in Drosophila melanogaster: 11,619 phosphosites identified in Drosophila melanogaster proteins can be aligned to phospho-acceptor amino acids of the human orthologs while 2601 acetylation sites identified in Drosophila melanogaster can be aligned to human orthologs. Considering a sliding window of five amino acids surrounding the identified phosphosite, the probability of the corresponding phospho-acceptor site having been observed as phosphorylated in human data correlates with the degree of sequence similarity. This correlation has also been observed with acetylation sites (A). Correlation of phosphorylation with disease related protein variants: The chance of the aligned human sites corresponding to the phosphosites identified in Drosophila locating within 10 amino acids distance to disease variants correlates with the sequence similarities between human and Drosophila sequences. The correlation is prevalent for phospho-acceptor sites in human (B)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-25006903" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of the conservation of PTM sites of &lt;em&gt;Drosophila melanogaster&lt;/em&gt;. Correlations of sequence conservation and observed phosphorylation in &lt;em&gt;Drosophila melanogaster&lt;/em&gt;: 11,619 phosphosites identified in &lt;em&gt;Drosophila melanogaster&lt;/em&gt; proteins can be aligned to phospho-acceptor amino acids of the human orthologs while 2601 acetylation sites identified in &lt;em&gt;Drosophila melanogaster&lt;/em&gt; can be aligned to human orthologs. Considering a sliding window of five amino acids surrounding the identified phosphosite, the probability of the corresponding phospho-acceptor site having been observed as phosphorylated in human data correlates with the degree of sequence similarity. This correlation has also been observed with acetylation sites (A). Correlation of phosphorylation with disease related protein variants: The chance of the aligned human sites corresponding to the phosphosites identified in &lt;em&gt;Drosophila&lt;/em&gt; locating within 10 amino acids distance to disease variants correlates with the sequence similarities between human and &lt;em&gt;Drosophila&lt;/em&gt; sequences. The correlation is prevalent for phospho-acceptor sites in human (B).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/1/F5.medium.gif" width="346" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/1/1/F5.medium.gif" width="346" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/1/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/1/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491803" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-33" class="first-child">Analysis of the conservation of PTM sites of <em>Drosophila melanogaster</em>. Correlations of sequence conservation and observed phosphorylation in <em>Drosophila melanogaster</em>: 11,619 phosphosites identified in <em>Drosophila melanogaster</em> proteins can be aligned to phospho-acceptor amino acids of the human orthologs while 2601 acetylation sites identified in <em>Drosophila melanogaster</em> can be aligned to human orthologs. Considering a sliding window of five amino acids surrounding the identified phosphosite, the probability of the corresponding phospho-acceptor site having been observed as phosphorylated in human data correlates with the degree of sequence similarity. This correlation has also been observed with acetylation sites (A). Correlation of phosphorylation with disease related protein variants: The chance of the aligned human sites corresponding to the phosphosites identified in <em>Drosophila</em> locating within 10 amino acids distance to disease variants correlates with the sequence similarities between human and <em>Drosophila</em> sequences. The correlation is prevalent for phospho-acceptor sites in human (B).</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">We observed an enrichment of UniProt human disease related variants located proximal to phosphosites conserved with <em>Drosophila melanogaster</em>. For example, the enrichment p-value of disease related variants is 4.3*10<sup>−10</sup> by Fisher exact test for the phosphosites with 80% or higher identity between human and <em>Drosophila</em> sites. To further analyze the intersection between phosphorylation events in <em>Drosophila</em> and disease variants in human, we calculated the percent of phosphosites that are within 10 amino acids of a disease variant at each identity cut-off. Our analysis indicates that more highly conserved sites tend to occupy positions proximal to residues variant in human disease (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5b</a>). For example, phosphosites with 50% or higher identity are about twofold more likely to be located within 10 amino acids of a disease variant than phosphosites with 20% or higher identity for the phospho-acceptor sites. Analysis of sites that are phospho-acceptor residues (serine, threonine, or tyrosine) in <em>Drosophila</em> but are not in human show a similar trend but this correlation was more prevalent for the phospho-acceptor sites, indicating that the correlation is driven by the phospho-acceptor as well as the conservation of the surrounding sequence.</p><p id="p-35">Finally we compared all phosphosites in <em>Drosophila melanogaster</em> with their human orthologous sites. We identified 370 sites that were observed as phosphorylated in <em>Drosophila</em> and have 100% identity with human phosphosites over a sliding window of five amino acids. These sites cover 146 human genes, many of which are kinases, including cyclin dependent kinases, glycogen synthase kinases, mitogen-activated protein kinases, ribosomal protein S6 kinases and the insulin receptor (InR) (Supplementary Table 2). For example, human glycogen synthase kinase 3A and 3B (GSK3A and GSK3B) auto-phosphorylate on a conserved tyrosine residue (Y279) for maximal activity, and play an important role in multiple signaling pathways (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Beurel <em>et al.</em> 2015</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Nagini <em>et al.</em> 2018</a>). Dysregulation of GSK3 has been linked to various diseases including cancer, in which GSK3 can function as a tumor promoter or suppressor in different contexts and with different phosphorylation status (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ma 2014</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Nagini <em>et al.</em> 2018</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sarkar <em>et al.</em> 2015</a>). The S278, Y279 and S282 sites within the protein-kinase domain of GSK3 have 100% identity with the <em>Drosophila</em> ortholog <em><a href="http://flybase.org/reports/FBgn0003371.html">sgg</a></em> (shaggy) and the phosphorylation of these sites has also been observed in <em>Drosophila</em> (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6a</a>). We further uncovered sites where the phospho-acceptor identity has changed, such as from serine to threonine (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6b</a>, Supplementary Table 3), suggesting that phosphorylation of these sites may also be conserved and required for regulation of the protein activity. We identified proteins for which the phospho-acceptor residues are conserved among <em>Drosophila</em> but absent in human despite the surrounding sequences being 100% identical (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6c</a>, Supplementary Table 3). These sites may regulate species-specific functions. In addition, 2977 PTM sites in <em>Drosophila melanogaster</em> collected in iProteinDB are acetylation sites. Approximately 89% of these acetylation sites are located within <em>Drosophila melanogaster</em> proteins which are conserved with human, based on DIOPT prediction using a score of 3 or more as cutoff (<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Hu <em>et al.</em> 2011</a>). Further analysis showed that the sequence conservation of <em>Drosophila melanogaster</em> acetylation sites to human correlates with the probability that the associated acetylation event has also been observed in human datasets, similar to what we observed for phosphorylation sites (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5a</a>). Altogether these results indicate the utility of using <em>Drosophila</em> as a model system to study the function of these sites in signal transduction and the regulation of associated proteins.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/1/1/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Examples of phosphosite conservation between human and Drosophila melanogaster. Examples of phosphosites identified in Drosophila melanogaster (red), also identified as phosphorylated in human (red), that share 100% identity with human (arrow) and indicated model organisms (A). Phosphosites where the observed phospho-acceptor residue has changed (B) and phosphosites where the phospho-acceptors have been lost but the surrounding sequences are 100% identical (C). The abbreviation of taxonomy name represents different model organisms (hs - Homo sapiens; mm- Mus musculus; rn - Rattus norvegicus; xt - Xenopus tropicalis; dr- Danio rerio; dm- Drosophila melanogaster; ce- Caenorhabditis elegans)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-25006903" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Examples of phosphosite conservation between human and &lt;em&gt;Drosophila melanogaster&lt;/em&gt;. Examples of phosphosites identified in &lt;em&gt;Drosophila melanogaster&lt;/em&gt; (red), also identified as phosphorylated in human (red), that share 100% identity with human (arrow) and indicated model organisms (A). Phosphosites where the observed phospho-acceptor residue has changed (B) and phosphosites where the phospho-acceptors have been lost but the surrounding sequences are 100% identical (C). The abbreviation of taxonomy name represents different model organisms (hs - &lt;em&gt;Homo sapiens&lt;/em&gt;; mm- &lt;em&gt;Mus musculus&lt;/em&gt;; rn - &lt;em&gt;Rattus norvegicus&lt;/em&gt;; xt - &lt;em&gt;Xenopus tropicalis&lt;/em&gt;; dr- &lt;em&gt;Danio rerio&lt;/em&gt;; dm- &lt;em&gt;Drosophila melanogaster&lt;/em&gt;; ce- &lt;em&gt;Caenorhabditis elegans&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/1/1/F6.medium.gif" width="440" height="381"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/1/1/F6.medium.gif" width="440" height="381"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/1/1/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/1/1/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491989" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-36" class="first-child">Examples of phosphosite conservation between human and <em>Drosophila melanogaster</em>. Examples of phosphosites identified in <em>Drosophila melanogaster</em> (red), also identified as phosphorylated in human (red), that share 100% identity with human (arrow) and indicated model organisms (A). Phosphosites where the observed phospho-acceptor residue has changed (B) and phosphosites where the phospho-acceptors have been lost but the surrounding sequences are 100% identical (C). The abbreviation of taxonomy name represents different model organisms (hs - <em>Homo sapiens</em>; mm- <em>Mus musculus</em>; rn - <em>Rattus norvegicus</em>; xt - <em>Xenopus tropicalis</em>; dr- <em>Danio rerio</em>; dm- <em>Drosophila melanogaster</em>; ce- <em>Caenorhabditis elegans</em>).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section conclusions" id="sec-14"><h2 class="">Conclusion</h2><p id="p-37"><em>Drosophila melanogaster</em> is one of the most-studied model organisms. Current PTM resources, such as PhosphoSitePlus (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Hornbeck <em>et al.</em> 2012</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Hornbeck <em>et al.</em> 2015</a>), dbPTM (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Huang <em>et al.</em> 2016</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lee <em>et al.</em> 2006</a>) and Phospho.ELM (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Diella <em>et al.</em> 2004</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Diella <em>et al.</em> 2008</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dinkel <em>et al.</em> 2011</a>), have comprehensive coverage for human, mouse, and rat, but have very limited coverage for <em>Drosophila</em>. Resources like PHOSIDA (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Gnad <em>et al.</em> 2011</a>), PHOSPHOPEP (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bodenmiller <em>et al.</em> 2008</a>; <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Bodenmiller <em>et al.</em> 2007</a>) and dbPAF (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Ullah <em>et al.</em> 2016</a>) provide large-scale PTM data for <em>Drosophila</em> genes but are focused on only one or, at most, a few datasets. We generated a large-scale proteomics dataset of six <em>Drosophila</em> species, made the data available, and integrated it with literature annotation and other large datasets for <em>Drosophila melanogaster</em>. This integrated resource allows researchers to obtain a more comprehensive view of the PTM landscape, taking into consideration all <em>Drosophila</em> proteomic data, and enabling comparison to orthologous proteins from other model organisms. Many of the conserved sites reside within kinases themselves, demonstrating that evolution has largely “optimized” protein kinase architecture and their operation within signaling pathways. We expect that iProteinDB will serve as a valuable resource to facilitate functional discovery. For example, iProteinDB can help a scientist identify sites that are critical for regulation that can be used for example to generate ‘activity-dead’ proteins that can serve as controls in rescue experiments with phosphomimetic (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Pondugula <em>et al.</em> 2009</a>) and temperature-sensitive mutants (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hsu and Perrimon 1994</a>).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-38">We would like to thank the members of Perrimon laboratory, Gygi laboratory, <em>Drosophila</em> RNAi Screening Center (DRSC) and Transgenic RNAi Project (TRiP) for helpful input on the project. Particularly we would like to thank Dr. Stephanie Mohr for helpful suggestions during manuscript preparation, Mr. Aram Comjean for advice during the resource implementation and Mr. Jonathan Rodiger for help with the data analysis. The DRSC is supported by National Institutes of Health (NIH) National Institute of General Medical Sciences grant R01 GM 067761 (to N.P.). The National Institutes of Health supported this work (5R01DK088718, 5P01CA120964, 5R01GM084947 and 5R01GM067761). R.S. is a Special Fellow of the Leukemia and Lymphoma Society. N.P. is a Howard Hughes Medical Institute investigator.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-39">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6986945">https://doi.org/10.25387/g3.6986945</a>.</p></li><li class="fn" id="fn-5"><p id="p-40"><em>Communicating editor: M. Boutros</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-08-02"><span class="received-label">Received </span>August 2, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-11-02"><span class="accepted-label">Accepted </span>November 2, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Hu <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.1" data-doi="10.1093/nar/gkx1019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altenhoff</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glover</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Train</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaleb</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warwick Vesztrocy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The OMA orthology database in 2018: retrieving evolutionary relationships among all domains of life through richer web and programmatic interfaces.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">D477</span>–<span class="cit-lpage">D485</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx1019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253DD477%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.2" data-doi="10.1093/nar/gkq1238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altenhoff</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonnet</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dessimoz</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">OMA 2011: orthology inference among 1000 complete genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">D289</span>–<span class="cit-lpage">D294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq1238</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq1238%26rft_id%253Dinfo%253Apmid%252F21113020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq1238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21113020&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285831700049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.3" data-doi="10.1093/nar/gku1158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altenhoff</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skunca</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glover</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Train</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sueki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The OMA orthology database in 2015: function predictions, better plant support, synteny view and other improvements.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D240</span>–<span class="cit-lpage">D249</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gku1158</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1158%26rft_id%253Dinfo%253Apmid%252F25399418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25399418&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.4" data-doi="10.1038/nbt1240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beausoleil</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerber</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rush</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gygi</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A probability-based approach for high-throughput protein phosphorylation analysis and site localization.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1285</span>–<span class="cit-lpage">1292</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt1240</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DBeausoleil%26rft.auinit1%253DS.%2BA.%26rft.volume%253D24%26rft.issue%253D10%26rft.spage%253D1285%26rft.epage%253D1292%26rft.atitle%253DA%2Bprobability-based%2Bapproach%2Bfor%2Bhigh-throughput%2Bprotein%2Bphosphorylation%2Banalysis%2Band%2Bsite%2Blocalization.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1240%26rft_id%253Dinfo%253Apmid%252F16964243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16964243&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241191700033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.5" data-doi="10.1002/msb.201304521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beltrao</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bork</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Noort</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evolution and functional cross-talk of protein post-translational modifications.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">714</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/msb.201304521</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DBeltrao%26rft.auinit1%253DP.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253D714%26rft.epage%253D714%26rft.atitle%253DEvolution%2Band%2Bfunctional%2Bcross-talk%2Bof%2Bprotein%2Bpost-translational%2Bmodifications%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fmsb.201304521%26rft_id%253Dinfo%253Apmid%252F24366814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjkvMS83MTQiO3M6NDoiYXRvbSI7czoxNToiL2dnZy85LzEvMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.6" data-doi="10.1016/j.pharmthera.2014.11.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beurel</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grieco</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jope</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Glycogen synthase kinase-3 (GSK3): regulation, actions, and diseases.</span> <abbr class="cit-jnl-abbrev">Pharmacol. Ther.</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">114</span>–<span class="cit-lpage">131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.pharmthera.2014.11.016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPharmacol.%2BTher.%26rft.volume%253D148%26rft.spage%253D114%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pharmthera.2014.11.016%26rft_id%253Dinfo%253Apmid%252F25435019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pharmthera.2014.11.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25435019&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.7" data-doi="10.1038/nbt1208-1339"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bodenmiller</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerrits</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jovanovic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">PhosphoPep–a database of protein phosphorylation sites in model organisms.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1339</span>–<span class="cit-lpage">1340</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt1208-1339</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DBodenmiller%26rft.auinit1%253DB.%26rft.volume%253D26%26rft.issue%253D12%26rft.spage%253D1339%26rft.epage%253D1340%26rft.atitle%253DPhosphoPep--a%2Bdatabase%2Bof%2Bprotein%2Bphosphorylation%2Bsites%2Bin%2Bmodel%2Borganisms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1208-1339%26rft_id%253Dinfo%253Apmid%252F19060867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1208-1339&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19060867&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261591300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.8" data-doi="10.1038/msb4100182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bodenmiller</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malmstrom</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerrits</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PhosphoPep–a phosphoproteome resource for systems biology research in Drosophila Kc167 cells.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">139</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/msb4100182</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DBodenmiller%26rft.auinit1%253DB.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D139%26rft.epage%253D139%26rft.atitle%253DPhosphoPep--a%2Bphosphoproteome%2Bresource%2Bfor%2Bsystems%2Bbiology%2Bresearch%2Bin%2BDrosophila%2BKc167%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb4100182%26rft_id%253Dinfo%253Apmid%252F17940529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjMvMS8xMzkiO3M6NDoiYXRvbSI7czoxNToiL2dnZy85LzEvMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.9" data-doi="10.1101/gr.213694.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casas-Vila</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bluhm</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sayols</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dinges</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dejung</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The developmental proteome of Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1273</span>–<span class="cit-lpage">1285</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.213694.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.213694.116%26rft_id%253Dinfo%253Apmid%252F28381612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI3LzcvMTI3MyI7czo0OiJhdG9tIjtzOjE1OiIvZ2dnLzkvMS8xLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.10" data-doi="10.1186/1471-2105-5-79"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diella</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gemund</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linding</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Via</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Phospho.ELM: a database of experimentally verified phosphorylation sites in eukaryotic proteins.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">79</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-5-79</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DDiella%26rft.auinit1%253DF.%26rft.volume%253D5%26rft.spage%253D79%26rft.epage%253D79%26rft.atitle%253DPhospho.ELM%253A%2Ba%2Bdatabase%2Bof%2Bexperimentally%2Bverified%2Bphosphorylation%2Bsites%2Bin%2Beukaryotic%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-5-79%26rft_id%253Dinfo%253Apmid%252F15212693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-5-79&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15212693&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.11" data-doi="10.1093/nar/gkm772"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diella</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gould</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chica</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Via</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Phospho.ELM: a database of phosphorylation sites–update 2008.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">D240</span>–<span class="cit-lpage">D244</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkm772</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm772%26rft_id%253Dinfo%253Apmid%252F17962309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm772&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17962309&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252545400044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.12" data-doi="10.1093/nar/gkq1104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dinkel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chica</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Via</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gould</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Phospho.ELM: a database of phosphorylation sites–update 2011.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">D261</span>–<span class="cit-lpage">D267</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq1104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq1104%26rft_id%253Dinfo%253Apmid%252F21062810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq1104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21062810&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285831700044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.13" data-doi="10.1126/science.277.5329.1103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gabay</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seger</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shilo</span>  <span class="cit-name-given-names">B. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">In situ activation pattern of Drosophila EGF receptor pathway during development.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">1103</span>–<span class="cit-lpage">1106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.277.5329.1103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGabay%26rft.auinit1%253DL.%26rft.volume%253D277%26rft.issue%253D5329%26rft.spage%253D1103%26rft.epage%253D1106%26rft.atitle%253DIn%2BSitu%2BActivation%2BPattern%2Bof%2BDrosophila%2BEGF%2BReceptor%2BPathway%2BDuring%2BDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.277.5329.1103%26rft_id%253Dinfo%253Apmid%252F9262480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzcvNTMyOS8xMTAzIjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvOS8xLzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.14" data-doi="10.1074/mcp.M110.001594"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gnad</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forner</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zielinska</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunawardena</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evolutionary constraints of phosphorylation in eukaryotes, prokaryotes, and mitochondria.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Proteomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2642</span>–<span class="cit-lpage">2653</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/mcp.M110.001594</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fmcp.M110.001594%26rft_id%253Dinfo%253Apmid%252F20688971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoibWNwcm90IjtzOjU6InJlc2lkIjtzOjk6IjkvMTIvMjY0MiI7czo0OiJhdG9tIjtzOjE1OiIvZ2dnLzkvMS8xLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.15" data-doi="10.1093/nar/gkq1159"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gnad</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunawardena</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mann</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">PHOSIDA 2011: the posttranslational modification database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">D253</span>–<span class="cit-lpage">D260</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq1159</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq1159%26rft_id%253Dinfo%253Apmid%252F21081558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq1159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21081558&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285831700043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.16" data-doi="10.1186/gb-2007-8-11-r250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gnad</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macek</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PHOSIDA (phosphorylation site database): management, structural and evolutionary investigation, and prediction of phosphosites.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">R250</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2007-8-11-r250</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DGnad%26rft.auinit1%253DF.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253DR250%26rft.epage%253DR250%26rft.atitle%253DPHOSIDA%2B%2528phosphorylation%2Bsite%2Bdatabase%2529%253A%2Bmanagement%252C%2Bstructural%2Band%2Bevolutionary%2Binvestigation%252C%2Band%2Bprediction%2Bof%2Bphosphosites.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-11-r250%26rft_id%253Dinfo%253Apmid%252F18039369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-11-r250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18039369&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.17" data-doi="10.1074/mcp.M800559-MCP200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hilger</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonaldi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gnad</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mann</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Systems-wide analysis of a phosphatase knock-down by quantitative proteomics and phosphoproteomics.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Proteomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1908</span>–<span class="cit-lpage">1920</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/mcp.M800559-MCP200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fmcp.M800559-MCP200%26rft_id%253Dinfo%253Apmid%252F19429919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoibWNwcm90IjtzOjU6InJlc2lkIjtzOjg6IjgvOC8xOTA4IjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvOS8xLzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.18" data-doi="10.1038/nmeth.2968"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horn</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoof</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robin</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">KinomeXplorer: an integrated platform for kinome biology studies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">603</span>–<span class="cit-lpage">604</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2968</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D11%26rft.spage%253D603%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2968%26rft_id%253Dinfo%253Apmid%252F24874572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2968&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24874572&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336899300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.19" data-doi="10.1093/nar/gkr1122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hornbeck</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kornhauser</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tkachev</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skrzypek</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">PhosphoSitePlus: a comprehensive resource for investigating the structure and function of experimentally determined post-translational modifications in man and mouse.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D261</span>–<span class="cit-lpage">D270</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkr1122</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1122%26rft_id%253Dinfo%253Apmid%252F22135298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr1122&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22135298&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.20" data-doi="10.1093/nar/gku1267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hornbeck</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kornhauser</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Latham</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">PhosphoSitePlus, 2014: mutations, PTMs and recalibrations.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D512</span>–<span class="cit-lpage">D520</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gku1267</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1267%26rft_id%253Dinfo%253Apmid%252F25514926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25514926&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.21" data-doi="10.1101/gad.8.18.2176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrimon</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">A temperature-sensitive MEK mutation demonstrates the conservation of the signaling pathways activated by receptor tyrosine kinases.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2176</span>–<span class="cit-lpage">2187</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.8.18.2176</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DHsu%26rft.auinit1%253DJ%2BC%26rft.volume%253D8%26rft.issue%253D18%26rft.spage%253D2176%26rft.epage%253D2187%26rft.atitle%253DA%2Btemperature-sensitive%2BMEK%2Bmutation%2Bdemonstrates%2Bthe%2Bconservation%2Bof%2Bthe%2Bsignaling%2Bpathways%2Bactivated%2Bby%2Breceptor%2Btyrosine%2Bkinases.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.8.18.2176%26rft_id%253Dinfo%253Apmid%252F7958887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiOC8xOC8yMTc2IjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvOS8xLzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.22" data-doi="10.1186/1471-2105-12-357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flockhart</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinayagam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergwitz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">An integrative approach to ortholog prediction for disease-focused and other functional studies.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">357</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-357</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHu%26rft.auinit1%253DY.%26rft.volume%253D12%26rft.spage%253D357%26rft.epage%253D357%26rft.atitle%253DAn%2Bintegrative%2Bapproach%2Bto%2Bortholog%2Bprediction%2Bfor%2Bdisease-focused%2Band%2Bother%2Bfunctional%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-357%26rft_id%253Dinfo%253Apmid%252F21880147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-357&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21880147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.23" data-doi="10.1093/nar/gkv1240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">K. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsieh</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jhong</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">dbPTM 2016: 10-year anniversary of a resource for post-translational modification of proteins.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D435</span>–<span class="cit-lpage">D446</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1240</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1240%26rft_id%253Dinfo%253Apmid%252F26578568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26578568&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.24" data-doi="10.1016/S0092-8674(00)81688-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Signaling–2000 and beyond.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">127</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(00)81688-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHunter%26rft.auinit1%253DT.%26rft.volume%253D100%26rft.issue%253D1%26rft.spage%253D113%26rft.epage%253D127%26rft.atitle%253DSignaling--2000%2Band%2Bbeyond.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981688-8%26rft_id%253Dinfo%253Apmid%252F10647936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81688-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10647936&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084722600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.25" data-doi="10.1093/nar/gkh253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iakoucheva</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radivojac</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Connor</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sikes</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The importance of intrinsic disorder for protein phosphorylation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1037</span>–<span class="cit-lpage">1049</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkh253</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DIakoucheva%26rft.auinit1%253DL.%2BM.%26rft.volume%253D32%26rft.issue%253D3%26rft.spage%253D1037%26rft.epage%253D1049%26rft.atitle%253DThe%2Bimportance%2Bof%2Bintrinsic%2Bdisorder%2Bfor%2Bprotein%2Bphosphorylation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh253%26rft_id%253Dinfo%253Apmid%252F14960716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14960716&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220490400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.26" data-doi="10.1093/nar/gkf436"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuma</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyata</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">3059</span>–<span class="cit-lpage">3066</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkf436</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKatoh%26rft.auinit1%253DK.%26rft.volume%253D30%26rft.issue%253D14%26rft.spage%253D3059%26rft.epage%253D3066%26rft.atitle%253DMAFFT%253A%2Ba%2Bnovel%2Bmethod%2Bfor%2Brapid%2Bmultiple%2Bsequence%2Balignment%2Bbased%2Bon%2Bfast%2BFourier%2Btransform%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkf436%26rft_id%253Dinfo%253Apmid%252F12136088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkf436&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12136088&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177154300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.27" data-doi="10.1371/journal.pgen.1000990"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kockel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melnick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruckner</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hebrok</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dynamic switch of negative feedback regulation in Drosophila Akt-TOR signaling.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000990</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000990</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKockel%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253De1000990%26rft.epage%253De1000990%26rft.atitle%253DDynamic%2Bswitch%2Bof%2Bnegative%2Bfeedback%2Bregulation%2Bin%2BDrosophila%2BAkt-TOR%2Bsignaling.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000990%26rft_id%253Dinfo%253Apmid%252F20585550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000990&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20585550&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.28" data-doi="10.1016/j.tig.2009.03.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michnick</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Weak functional constraints on phosphoproteomes.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">197</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2009.03.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DLandry%26rft.auinit1%253DC.%2BR.%26rft.volume%253D25%26rft.issue%253D5%26rft.spage%253D193%26rft.epage%253D197%26rft.atitle%253DWeak%2Bfunctional%2Bconstraints%2Bon%2Bphosphoproteomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2009.03.003%26rft_id%253Dinfo%253Apmid%252F19349092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2009.03.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19349092&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266524300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.29" data-doi="10.1093/nar/gkj083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hung</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">dbPTM: an information repository of protein post-translational modification.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">D622</span>–<span class="cit-lpage">D627</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkj083</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DLee%26rft.auinit1%253DT.-Y.%26rft.volume%253D34%26rft.issue%253Dsuppl_1%26rft.spage%253DD622%26rft.epage%253DD627%26rft.atitle%253DdbPTM%253A%2Ban%2Binformation%2Brepository%2Bof%2Bprotein%2Bpost-translational%2Bmodification%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkj083%26rft_id%253Dinfo%253Apmid%252F16381945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkj083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16381945&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239307700132&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.30" data-doi="10.1093/nar/gkw290"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Letunic</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bork</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Interactive tree of life (iTOL) v3: an online tool for the display and annotation of phylogenetic and other trees.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">W242</span>–<span class="cit-lpage">W245</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw290</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw290%26rft_id%253Dinfo%253Apmid%252F27095192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw290&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27095192&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.31" data-doi="10.3233/JAD-131661"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">GSK3 in Alzheimer’s disease: mind the isoforms.</span> <abbr class="cit-jnl-abbrev">J. Alzheimers Dis.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">710</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3233/JAD-131661</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAlzheimers%2BDis.%26rft.volume%253D39%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.3233%252FJAD-131661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3233/JAD-131661&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.32" data-doi="10.1016/S0968-0004(02)02179-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manning</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plowman</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sudarsanam</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Evolution of protein kinase signaling from yeast to man.</span> <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">514</span>–<span class="cit-lpage">520</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0968-0004(02)02179-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiochemical%2Bsciences%26rft.stitle%253DTrends%2BBiochem%2BSci%26rft.aulast%253DManning%26rft.auinit1%253DG.%26rft.volume%253D27%26rft.issue%253D10%26rft.spage%253D514%26rft.epage%253D520%26rft.atitle%253DEvolution%2Bof%2Bprotein%2Bkinase%2Bsignaling%2Bfrom%2Byeast%2Bto%2Bman.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0968-0004%252802%252902179-5%26rft_id%253Dinfo%253Apmid%252F12368087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0968-0004(02)02179-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12368087&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178603100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.33" data-doi="10.1016/bs.ircmb.2015.07.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyuhas</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Ribosomal Protein S6 Phosphorylation: Four Decades of Research.</span> <abbr class="cit-jnl-abbrev">Int. Rev. Cell Mol. Biol.</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">73</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/bs.ircmb.2015.07.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BRev.%2BCell%2BMol.%2BBiol.%26rft.volume%253D320%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fbs.ircmb.2015.07.006%26rft_id%253Dinfo%253Apmid%252F26614871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/bs.ircmb.2015.07.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26614871&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.34" data-doi="10.1016/j.semcancer.2017.12.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagini</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sophia</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mishra</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Glycogen synthase kinases: Moonlighting proteins with theranostic potential in cancer.</span> <abbr class="cit-jnl-abbrev">Semin. Cancer Biol</abbr>. S1044–579X(17)30179–7.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.semcancer.2017.12.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSemin.%2BCancer%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.semcancer.2017.12.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.semcancer.2017.12.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.35" data-doi="10.1093/nar/gkg584"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Obenauer</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantley</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yaffe</span>  <span class="cit-name-given-names">M. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Scansite 2.0: Proteome-wide prediction of cell signaling interactions using short sequence motifs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3635</span>–<span class="cit-lpage">3641</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkg584</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DObenauer%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D31%26rft.issue%253D13%26rft.spage%253D3635%26rft.epage%253D3641%26rft.atitle%253DScansite%2B2.0%253A%2Bproteome-wide%2Bprediction%2Bof%2Bcell%2Bsignaling%2Binteractions%2Busing%2Bshort%2Bsequence%2Bmotifs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg584%26rft_id%253Dinfo%253Apmid%252F12824383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg584&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12824383&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183832900075&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.36" data-doi="10.1124/dmd.108.024695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pondugula</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brimer-Cline</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuetz</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tyagi</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A phosphomimetic mutation at threonine-57 abolishes transactivation activity and alters nuclear localization pattern of human pregnane x receptor.</span> <abbr class="cit-jnl-abbrev">Drug Metab. Dispos.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">719</span>–<span class="cit-lpage">730</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1124/dmd.108.024695</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDrug%2BMetab.%2BDispos.%26rft_id%253Dinfo%253Adoi%252F10.1124%252Fdmd.108.024695%26rft_id%253Dinfo%253Apmid%252F19171678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZG1kIjtzOjU6InJlc2lkIjtzOjg6IjM3LzQvNzE5IjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvOS8xLzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.37" data-doi="10.1080/15592294.2014.1003746"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarkar</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baguley</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finlay</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Askarian-Amiri</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Epigenetic regulation in human melanoma: past and future.</span> <abbr class="cit-jnl-abbrev">Epigenetics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">121</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/15592294.2014.1003746</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEpigenetics%26rft.volume%253D10%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15592294.2014.1003746%26rft_id%253Dinfo%253Apmid%252F25587943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15592294.2014.1003746&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25587943&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.38" data-doi="10.1016/j.devcel.2014.07.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sopko</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinayagam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhai</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binari</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Combining genetic perturbations and proteomics to examine kinase-phosphatase networks in Drosophila embryos.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">114</span>–<span class="cit-lpage">127</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.devcel.2014.07.027</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BCell%26rft.volume%253D31%26rft.spage%253D114%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2014.07.027%26rft_id%253Dinfo%253Apmid%252F25284370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2014.07.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25284370&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.39" data-doi="10.1126/science.aaf2144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Studer</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez-Mias</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vieitez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evolution of protein phosphorylation across 18 fungal species.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">354</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">232</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aaf2144</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DStuder%26rft.auinit1%253DR.%2BA.%26rft.volume%253D354%26rft.issue%253D6309%26rft.spage%253D229%26rft.epage%253D232%26rft.atitle%253DEvolution%2Bof%2Bprotein%2Bphosphorylation%2Bacross%2B18%2Bfungal%2Bspecies%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf2144%26rft_id%253Dinfo%253Apmid%252F27738172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTQvNjMwOS8yMjkiO3M6NDoiYXRvbSI7czoxNToiL2dnZy85LzEvMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.40" data-doi="10.1038/srep23534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ullah</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">dbPAF: an integrative database of protein phosphorylation in animals and fungi.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">23534</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep23534</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D23534%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep23534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep23534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.41" data-doi="10.1093/bioinformatics/btp033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Procter</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clamp</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">G. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Jalview Version 2–a multiple sequence alignment editor and analysis workbench.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1189</span>–<span class="cit-lpage">1191</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp033%26rft_id%253Dinfo%253Apmid%252F19151095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19151095&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265523300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.1.1.42" data-doi="10.1021/pr700696a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhai</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beausoleil</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mintseris</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gygi</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Phosphoproteome analysis of Drosophila melanogaster embryos.</span> <abbr class="cit-jnl-abbrev">J. Proteome Res.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1675</span>–<span class="cit-lpage">1682</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1021/pr700696a</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bproteome%2Bresearch%26rft.stitle%253DJ%2BProteome%2BRes%26rft.aulast%253DZhai%26rft.auinit1%253DB.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253D1675%26rft.epage%253D1682%26rft.atitle%253DPhosphoproteome%2Banalysis%2Bof%2BDrosophila%2Bmelanogaster%2Bembryos.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fpr700696a%26rft_id%253Dinfo%253Apmid%252F18327897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/pr700696a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18327897&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254711000031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/1/1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3993" title="G3 Reviewer Index 2018" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/1/13" title="OCMA: Fast, Memory-Efficient Factorization of Prohibitively Large Relationship Matrices" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/1"><span>Volume 9 Issue 1, January 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/1.cover-source.jpg?itok=uUntglLo" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/1/1" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OHYgapIwt2B31Q0DBEbm8pzI9Y3ButC3Nd18rOQY_dE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/software-and-data-resources" class="" data-icon-position="" data-hide-link-title="0">Software and Data Resources</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/1/1" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/1/1" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/1/1" class="active" data-icon-position="" data-hide-link-title="0">iProteinDB: An Integrative Database of Drosophila Post-translational Modifications</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491970" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-4Qql_2sqHCLr2a3fOi_l176jN04nga0Ot8jK8H-m4KE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/1/1.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/1/1" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-MdXNYnI7FK8xePmUPEv73WULKwHuQDyACUclxKNXp_8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/1/1" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/1/1.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491970" id="node491970--2" data-pisa="ggg;9/1/1" data-pisa-master="ggg;g3.118.200637" data-apath="/ggg/9/1/1.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">iProteinDB: An Integrative Database of <em>Drosophila</em> Post-translational Modifications</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1494-1402" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yanhui Hu</span>, <span class="highwire-citation-author" data-delta="1">Richelle Sopko</span>, <span class="highwire-citation-author" data-delta="2">Verena Chung</span>, <span class="highwire-citation-author" data-delta="3">Marianna Foos</span>, <span class="highwire-citation-author" data-delta="4">Romain A. Studer</span>, <span class="highwire-citation-author" data-delta="5">Sean D. Landry</span>, <span class="highwire-citation-author" data-delta="6">Daniel Liu</span>, <span class="highwire-citation-author" data-delta="7">Leonard Rabinow</span>, <span class="highwire-citation-author" data-delta="8">Florian Gnad</span>, <span class="highwire-citation-author" data-delta="9">Pedro Beltrao</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0001-7542-472X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Norbert Perrimon</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1-11;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200637 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491970--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yanhui Hu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span> RNAi Screening Center, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yanhui%2BHu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYanhui%2BHu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1494-1402" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yanhui Hu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Richelle Sopko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richelle%2BSopko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sopko%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichelle%2BSopko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Verena Chung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span> RNAi Screening Center, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Verena%2BChung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chung%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVerena%2BChung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marianna Foos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marianna%2BFoos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Foos%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarianna%2BFoos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Romain A. Studer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>European Molecular Biology Laboratory (EMBL), European Bioinformatics Institute, Wellcome Genome Campus, Hinxton, Cambridge CB10 1SD, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Romain%2BA.%2BStuder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Studer%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARomain%2BA.%2BStuder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sean D. Landry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Bioinformatics, Cell Signaling Technology Inc., 3 Trask Lane, Danvers, MA 01923</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BD.%2BLandry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Landry%20SD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BD.%2BLandry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Daniel Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span> RNAi Screening Center, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Leonard Rabinow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonard%2BRabinow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rabinow%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeonard%2BRabinow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Florian Gnad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Bioinformatics, Cell Signaling Technology Inc., 3 Trask Lane, Danvers, MA 01923</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Florian%2BGnad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gnad%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFlorian%2BGnad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Pedro Beltrao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>European Molecular Biology Laboratory (EMBL), European Bioinformatics Institute, Wellcome Genome Campus, Hinxton, Cambridge CB10 1SD, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pedro%2BBeltrao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beltrao%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APedro%2BBeltrao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Norbert Perrimon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span> RNAi Screening Center, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Howard Hughes Medical Institute, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Norbert%2BPerrimon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Perrimon%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANorbert%2BPerrimon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7542-472X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Norbert Perrimon</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:perrimon@receptor.med.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">perrimon@receptor.med.harvard.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/1/1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/1/1&amp;title=iProteinDB%3A%20An%20Integrative%20Database%20of%20Drosophila%20Post-translational%20Modifications" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/1/1&amp;title=iProteinDB%3A%20An%20Integrative%20Database%20of%20Drosophila%20Post-translational%20Modifications" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/1/1&amp;title=iProteinDB%3A%20An%20Integrative%20Database%20of%20Drosophila%20Post-translational%20Modifications" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/1/1&amp;text=iProteinDB%3A%20An%20Integrative%20Database%20of%20Drosophila%20Post-translational%20Modifications" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/1/1&amp;title=iProteinDB%3A%20An%20Integrative%20Database%20of%20Drosophila%20Post-translational%20Modifications" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/1/1&amp;t=iProteinDB%3A%20An%20Integrative%20Database%20of%20Drosophila%20Post-translational%20Modifications" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/1/1&amp;title=iProteinDB%3A%20An%20Integrative%20Database%20of%20Drosophila%20Post-translational%20Modifications" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/1/1&amp;title=iProteinDB%3A%20An%20Integrative%20Database%20of%20Drosophila%20Post-translational%20Modifications" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491970/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491970" id="node491970--3" data-pisa="ggg;9/1/1" data-pisa-master="ggg;g3.118.200637" data-apath="/ggg/9/1/1.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">iProteinDB: An Integrative Database of <em>Drosophila</em> Post-translational Modifications</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1494-1402" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yanhui Hu</span>, <span class="highwire-citation-author" data-delta="1">Richelle Sopko</span>, <span class="highwire-citation-author" data-delta="2">Verena Chung</span>, <span class="highwire-citation-author" data-delta="3">Marianna Foos</span>, <span class="highwire-citation-author" data-delta="4">Romain A. Studer</span>, <span class="highwire-citation-author" data-delta="5">Sean D. Landry</span>, <span class="highwire-citation-author" data-delta="6">Daniel Liu</span>, <span class="highwire-citation-author" data-delta="7">Leonard Rabinow</span>, <span class="highwire-citation-author" data-delta="8">Florian Gnad</span>, <span class="highwire-citation-author" data-delta="9">Pedro Beltrao</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0001-7542-472X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Norbert Perrimon</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1-11;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200637 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491970--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yanhui Hu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span> RNAi Screening Center, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yanhui%2BHu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYanhui%2BHu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1494-1402" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yanhui Hu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Richelle Sopko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richelle%2BSopko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sopko%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichelle%2BSopko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Verena Chung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span> RNAi Screening Center, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Verena%2BChung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chung%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVerena%2BChung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marianna Foos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marianna%2BFoos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Foos%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarianna%2BFoos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Romain A. Studer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>European Molecular Biology Laboratory (EMBL), European Bioinformatics Institute, Wellcome Genome Campus, Hinxton, Cambridge CB10 1SD, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Romain%2BA.%2BStuder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Studer%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARomain%2BA.%2BStuder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sean D. Landry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Bioinformatics, Cell Signaling Technology Inc., 3 Trask Lane, Danvers, MA 01923</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BD.%2BLandry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Landry%20SD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BD.%2BLandry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Daniel Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span> RNAi Screening Center, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Leonard Rabinow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonard%2BRabinow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rabinow%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeonard%2BRabinow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Florian Gnad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Bioinformatics, Cell Signaling Technology Inc., 3 Trask Lane, Danvers, MA 01923</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Florian%2BGnad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gnad%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFlorian%2BGnad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Pedro Beltrao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>European Molecular Biology Laboratory (EMBL), European Bioinformatics Institute, Wellcome Genome Campus, Hinxton, Cambridge CB10 1SD, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pedro%2BBeltrao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beltrao%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APedro%2BBeltrao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Norbert Perrimon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span> RNAi Screening Center, Harvard Medical School, 77 Avenue Louis Pasteur, Boston, MA 02115</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Howard Hughes Medical Institute, 77 Avenue Louis Pasteur, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Norbert%2BPerrimon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Perrimon%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANorbert%2BPerrimon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7542-472X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Norbert Perrimon</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:perrimon@receptor.med.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">perrimon@receptor.med.harvard.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1&amp;rft.epage=11&amp;rft.atitle=iProteinDB%3A%20An%20Integrative%20Database%20of%20Drosophila%20Post-translational%20Modifications&amp;rft.volume=9&amp;rft.issue=1&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hu%2C+Yanhui&amp;rft.au=Sopko%2C+Richelle&amp;rft.au=Chung%2C+Verena&amp;rft.au=Foos%2C+Marianna&amp;rft.au=Studer%2C+Romain+A.&amp;rft.au=Landry%2C+Sean+D.&amp;rft.au=Liu%2C+Daniel&amp;rft.au=Rabinow%2C+Leonard&amp;rft.au=Gnad%2C+Florian&amp;rft.au=Beltrao%2C+Pedro&amp;rft.au=Perrimon%2C+Norbert"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491970/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491970/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491970/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491970/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491970/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491970/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491970/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491970/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491970/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491970/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491970/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491970/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496031" id="node496031" data-pisa="ggg;g3.120.401086v1" data-pisa-master="ggg;g3.120.401086" data-apath="/ggg/early/2020/03/09/g3.120.401086.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/09/g3.120.401086" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The UCSF Mouse Inventory Database Application, an Open Source Web App for Sharing Mutant Mice within a Research Community</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495860" id="node495860" data-pisa="ggg;10/3/925" data-pisa-master="ggg;g3.119.400983" data-apath="/ggg/10/3/925.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/925" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">An Automated Method To Predict Mouse Gene and Protein Sequences Using Variant Data</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495845" id="node495845" data-pisa="ggg;10/3/945" data-pisa-master="ggg;g3.119.400945" data-apath="/ggg/10/3/945.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/945" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Software%20and%20Data%20Resources" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Software and Data Resources</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/1/1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/1/1.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/1/1.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/1/1.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/1/1.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
