<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/10/1741" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/10/1741.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/10/1741.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/10/1741.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/10/1741/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/10/1741.atom" />
<meta name="HW.pisa" content="ggg;3/10/1741" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Global Linkage Map Connects Meiotic Centromere Function to Chromosome Size in Budding Yeast" />
<meta name="DC.Identifier" content="10.1534/g3.113.007377" />
<meta name="DC.Date" content="2013-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2013 Baryshnikova et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Synthetic genetic array (SGA) analysis automates yeast genetics, enabling high-throughput construction of ordered arrays of double mutants. Quantitative colony sizes derived from SGA analysis can be used to measure cellular fitness and score for genetic interactions, such as synthetic lethality. Here we show that SGA colony sizes also can be used to obtain global maps of meiotic recombination because recombination frequency affects double-mutant formation for gene pairs located on the same chromosome and therefore influences the size of the resultant double-mutant colony. We obtained quantitative colony size data for ~1.2 million double mutants located on the same chromosome and constructed a genome-scale genetic linkage map at ~5 kb resolution. We found that our linkage map is reproducible and consistent with previous global studies of meiotic recombination. In particular, we confirmed that the total number of crossovers per chromosome tends to follow a simple linear model that depends on chromosome size. In addition, we observed a previously unappreciated relationship between the size of linkage regions surrounding each centromere and chromosome size, suggesting that crossovers tend to occur farther away from the centromere on larger chromosomes. The pericentric regions of larger chromosomes also appeared to load larger clusters of meiotic cohesin [Rec8][1], and acquire fewer [Spo11][2]-catalyzed DNA double-strand breaks. Given that crossovers too near or too far from centromeres are detrimental to homolog disjunction and increase the incidence of aneuploidy, our data suggest that chromosome size may have a direct role in regulating the fidelity of chromosome segregation during meiosis.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006211
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001014" />
<meta name="DC.Contributor" content="Anastasia Baryshnikova" />
<meta name="DC.Contributor" content="Benjamin VanderSluis" />
<meta name="DC.Contributor" content="Michael Costanzo" />
<meta name="DC.Contributor" content="Chad L. Myers" />
<meta name="DC.Contributor" content="Rita S. Cha" />
<meta name="DC.Contributor" content="Brenda Andrews" />
<meta name="DC.Contributor" content="Charles Boone" />
<meta name="article:published_time" content="2013-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Global Linkage Map Connects Meiotic Centromere Function to Chromosome Size in Budding Yeast" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Synthetic genetic array (SGA) analysis automates yeast genetics, enabling high-throughput construction of ordered arrays of double mutants. Quantitative colony sizes derived from SGA analysis can be used to measure cellular fitness and score for genetic interactions, such as synthetic lethality. Here we show that SGA colony sizes also can be used to obtain global maps of meiotic recombination because recombination frequency affects double-mutant formation for gene pairs located on the same chromosome and therefore influences the size of the resultant double-mutant colony. We obtained quantitative colony size data for ~1.2 million double mutants located on the same chromosome and constructed a genome-scale genetic linkage map at ~5 kb resolution. We found that our linkage map is reproducible and consistent with previous global studies of meiotic recombination. In particular, we confirmed that the total number of crossovers per chromosome tends to follow a simple linear model that depends on chromosome size. In addition, we observed a previously unappreciated relationship between the size of linkage regions surrounding each centromere and chromosome size, suggesting that crossovers tend to occur farther away from the centromere on larger chromosomes. The pericentric regions of larger chromosomes also appeared to load larger clusters of meiotic cohesin Rec8, and acquire fewer Spo11-catalyzed DNA double-strand breaks. Given that crossovers too near or too far from centromeres are detrimental to homolog disjunction and increase the incidence of aneuploidy, our data suggest that chromosome size may have a direct role in regulating the fidelity of chromosome segregation during meiosis.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/10/01" />
<meta name="citation_mjid" content="ggg;3/10/1741" />
<meta name="citation_id" content="3/10/1741" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/10/1741" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/10/1741.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/10/1741.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/10/1741.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.007377" />
<meta name="citation_pmid" content="23979930" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1741" />
<meta name="citation_lastpage" content="1751" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Anastasia Baryshnikova" />
<meta name="citation_author_institution" content="Banting and Best Department of Medical Research, The Donnelly Center for Cellular and Biomolecular Research, University of Toronto, Toronto, Ontario M5S 3E1, Canada" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S 3E1, Canada" />
<meta name="citation_author" content="Benjamin VanderSluis" />
<meta name="citation_author_institution" content="Department of Computer Science and Engineering, University of Minnesota, Minneapolis, Minnesota 55455" />
<meta name="citation_author" content="Michael Costanzo" />
<meta name="citation_author_institution" content="Banting and Best Department of Medical Research, The Donnelly Center for Cellular and Biomolecular Research, University of Toronto, Toronto, Ontario M5S 3E1, Canada" />
<meta name="citation_author" content="Chad L. Myers" />
<meta name="citation_author_institution" content="Department of Computer Science and Engineering, University of Minnesota, Minneapolis, Minnesota 55455" />
<meta name="citation_author" content="Rita S. Cha" />
<meta name="citation_author_institution" content="Department of Life Sciences, Genome Damage and Stability Centre, University of Sussex, Falmer, BN1 9RQ, UK" />
<meta name="citation_author_email" content="charlie.boone@utoronto.ca" />
<meta name="citation_author_email" content="rc320@sussex.ac.uk" />
<meta name="citation_author_email" content="brenda.andrews@utoronto.ca" />
<meta name="citation_author" content="Brenda Andrews" />
<meta name="citation_author_institution" content="Banting and Best Department of Medical Research, The Donnelly Center for Cellular and Biomolecular Research, University of Toronto, Toronto, Ontario M5S 3E1, Canada" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S 3E1, Canada" />
<meta name="citation_author_email" content="charlie.boone@utoronto.ca" />
<meta name="citation_author_email" content="rc320@sussex.ac.uk" />
<meta name="citation_author_email" content="brenda.andrews@utoronto.ca" />
<meta name="citation_author" content="Charles Boone" />
<meta name="citation_author_institution" content="Banting and Best Department of Medical Research, The Donnelly Center for Cellular and Biomolecular Research, University of Toronto, Toronto, Ontario M5S 3E1, Canada" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S 3E1, Canada" />
<meta name="citation_author_email" content="charlie.boone@utoronto.ca" />
<meta name="citation_author_email" content="rc320@sussex.ac.uk" />
<meta name="citation_author_email" content="brenda.andrews@utoronto.ca" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=A. Bardhan;citation_author=H. Chuong;citation_author=D. S. Dawson;citation_title=Meiotic cohesin promotes pairing of nonhomologous centromeres in early meiotic prophase.;citation_volume=21;citation_year=2010;citation_pmid=20375150;citation_doi=10.1091/mbc.E09-05-0392" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=A. Baryshnikova;citation_author=M. Costanzo;citation_author=S. Dixon;citation_author=F. J. Vizeacoumar;citation_author=C. L. Myers;citation_title=Synthetic genetic array (SGA) analysis in Saccharomyces cerevisiae and Schizosaccharomyces pombe.;citation_pages=145-179;citation_volume=470;citation_year=2010;citation_pmid=20946810;citation_doi=10.1016/S0076-6879(10)70007-0" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=A. Baryshnikova;citation_author=M. Costanzo;citation_author=Y. Kim;citation_author=H. Ding;citation_author=J. Koh;citation_title=Quantitative analysis of fitness and genetic interactions in yeast on a genome scale.;citation_pages=1017-1024;citation_volume=7;citation_year=2010;citation_issue=12;citation_pmid=21076421;citation_doi=10.1038/nmeth.1534" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=F. Baudat;citation_author=A. Nicolas;citation_title=Clustering of meiotic double-strand breaks on yeast chromosome III;citation_pages=5213-5218;citation_volume=94;citation_year=1997;citation_issue=10;citation_pmid=9144217;citation_doi=10.1073/pnas.94.10.5213" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Bergerat;citation_author=B. De Massy;citation_author=D. Gadelle;citation_author=P. C. Varoutas;citation_author=A. Nicolas;citation_title=An atypical topoisomerase II from Archaea with implications for meiotic recombination.;citation_pages=414-417;citation_volume=386;citation_year=1997;citation_issue=6623;citation_pmid=9121560;citation_doi=10.1038/386414a0" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. K. Bishop;citation_author=D. Zickler;citation_title=Early decision; meiotic crossover interference prior to stable strand exchange and synapsis.;citation_pages=9-15;citation_volume=117;citation_year=2004;citation_issue=1;citation_pmid=15066278;citation_doi=10.1016/S0092-8674(04)00297-1" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=Y. Blat;citation_author=N. Kleckner;citation_title=Cohesins bind to preferential sites along yeast chromosome III, with differential regulation along arms versus the centric region.;citation_pages=249-259;citation_volume=98;citation_year=1999;citation_issue=2;citation_pmid=10428036;citation_doi=10.1016/S0092-8674(00)81019-3" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=H. G. Blitzblau;citation_author=G. W. Bell;citation_author=J. Rodriguez;citation_author=S. P. Bell;citation_author=A. Hochwagen;citation_title=Mapping of meiotic single-stranded DNA reveals double-stranded-break hotspots near centromeres and telomeres.;citation_pages=2003-2012;citation_volume=17;citation_year=2007;citation_issue=23;citation_pmid=18060788;citation_doi=10.1016/j.cub.2007.10.066" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=V. Borde;citation_author=W. Lin;citation_author=E. Novikov;citation_author=J. H. Petrini;citation_author=M. Lichten;citation_title=Association of Mre11p with double-strand break sites during yeast meiosis.;citation_pages=389-401;citation_volume=13;citation_year=2004;citation_issue=3;citation_pmid=14967146;citation_doi=10.1016/S1097-2765(04)00034-6" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=G. A. Brar;citation_author=A. Amon;citation_title=Emerging roles for centromeres in meiosis I chromosome segregation.;citation_pages=899-910;citation_volume=9;citation_year=2008;citation_issue=12;citation_pmid=18981989;citation_doi=10.1038/nrg2454" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=C. Buhler;citation_author=V. Borde;citation_author=M. Lichten;citation_title=Mapping meiotic single-strand DNA reveals a new landscape of DNA double-strand breaks in Saccharomyces cerevisiae.;citation_pages=e324-e324;citation_volume=5;citation_year=2007;citation_issue=12;citation_pmid=18076285;citation_doi=10.1371/journal.pbio.0050324" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=S. Y. Chen;citation_author=T. Tsubouchi;citation_author=B. Rockmill;citation_author=J. S. Sandler;citation_author=D. R. Richards;citation_title=Global analysis of the meiotic crossover landscape.;citation_pages=401-415;citation_volume=15;citation_year=2008;citation_issue=3;citation_pmid=18691940;citation_doi=10.1016/j.devcel.2008.07.006" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. M. Cherry;citation_author=E. L. Hong;citation_author=C. Amundsen;citation_author=R. Balakrishnan;citation_author=G. Binkley;citation_title=Saccharomyces Genome Database: the genomics resource of budding yeast.;citation_volume=40;citation_year=2012;citation_pmid=22110037;citation_doi=10.1093/nar/gkr1029" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. R. Collins;citation_author=M. Schuldiner;citation_author=N. J. Krogan;citation_author=J. S. Weissman;citation_title=A strategy for extracting and analyzing large-scale quantitative epistatic interaction data.;citation_pages=R63-R63;citation_volume=7;citation_year=2006;citation_issue=7;citation_pmid=16859555;citation_doi=10.1186/gb-2006-7-7-r63" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Costanzo;citation_author=A. Baryshnikova;citation_author=J. Bellay;citation_author=Y. Kim;citation_author=E. D. Spear;citation_title=The Genetic Landscape of a Cell;citation_pages=425-431;citation_volume=327;citation_year=2010;citation_issue=5964;citation_pmid=20093466;citation_doi=10.1126/science.1180823" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=C. A. Eckert;citation_author=D. J. Gravdahl;citation_author=P. C. Megee;citation_title=The enhancement of pericentromeric cohesin association by conserved kinetochore components promotes high-fidelity chromosome segregation and is sensitive to microtubule-based tension.;citation_volume=21;citation_year=2007;citation_pmid=17242156;citation_doi=10.1101/gad.1498707" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Fledel-Alon;citation_author=D. J. Wilson;citation_author=K. Broman;citation_author=X. Wen;citation_author=C. Ober;citation_title=Broad-scale recombination patterns underlying proper disjunction in humans.;citation_pages=e1000658-e1000658;citation_volume=5;citation_year=2009;citation_issue=9;citation_pmid=19763175;citation_doi=10.1371/journal.pgen.1000658" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. L. Gerton;citation_author=J. Derisi;citation_author=R. Shroff;citation_author=M. Lichten;citation_author=P. O. Brown;citation_title=Global mapping of meiotic recombination hotspots and coldspots in the yeast Saccharomyces cerevisiae;citation_pages=11383-11390;citation_volume=97;citation_year=2000;citation_issue=21;citation_pmid=11027339;citation_doi=10.1073/pnas.97.21.11383" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=E. F. Glynn;citation_author=P. C. Megee;citation_author=H. G. Yu;citation_author=C. Mistrot;citation_author=E. Unal;citation_title=Genome-wide mapping of the cohesin complex in the yeast Saccharomyces cerevisiae.;citation_pages=E259-E259;citation_volume=2;citation_year=2004;citation_issue=9;citation_pmid=15309048;citation_doi=10.1371/journal.pbio.0020259" />
<meta name="citation_reference" content="citation_title=An Introduction to Genetic Analysis;citation_title=Accurate calculation of large map distances;citation_author=A. J. F. Griffiths;citation_author=J. H. Miller;citation_author=D. T. Suzuki;citation_author=R. C. Lewontin;citation_author=W. M. Gelbart;citation_year=2000" />
<meta name="citation_reference" content="citation_title=Genetics. Analysis of Genes and Genomes;citation_author=D. L. Hartl;citation_author=E. W. Jones;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Human Mol. Genet.;citation_author=T. Hassold;citation_author=H. Hall;citation_author=P. Hunt;citation_title=The origin of human aneuploidy: where we have been, where we are going.;citation_pages=R203-208;citation_volume=16(Spec No. 2);citation_year=2007" />
<meta name="citation_reference" content="citation_title=Meiosis;citation_title=Chiasmata;citation_author=G. H. Jones;citation_pages=213-244;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. B. Kaback;citation_title=Chromosome-size dependent control of meiotic recombination in humans.;citation_pages=20-21;citation_volume=13;citation_year=1996;citation_issue=1;citation_pmid=8673097;citation_doi=10.1038/ng0596-20" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. B. Kaback;citation_author=V. Guacci;citation_author=D. Barber;citation_author=J. W. Mahon;citation_title=Chromosome size-dependent control of meiotic recombination;citation_pages=228-232;citation_volume=256;citation_year=1992;citation_issue=5054;citation_pmid=1566070;citation_doi=10.1126/science.1566070" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. B. Kaback;citation_author=D. Barber;citation_author=J. Mahon;citation_author=J. Lamb;citation_author=J. You;citation_title=Chromosome Size-Dependent Control of Meiotic Reciprocal Recombination in Saccharomyces cerevisiae: The Role of Crossover Interference;citation_pages=1475-1486;citation_volume=152;citation_year=1999;citation_issue=4;citation_pmid=10430577" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Keeney;citation_author=C. N. Giroux;citation_author=N. Kleckner;citation_title=Meiosis-specific DNA double-strand breaks are catalyzed by Spo11, a member of a widely conserved protein family.;citation_pages=375-384;citation_volume=88;citation_year=1997;citation_issue=3;citation_pmid=9039264;citation_doi=10.1016/S0092-8674(00)81876-0" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Kegel;citation_author=H. Betts-Lindroos;citation_author=T. Kanno;citation_author=K. Jeppsson;citation_author=L. Strom;citation_title=Chromosome length influences replication-induced topological stress.;citation_pages=392-396;citation_volume=471;citation_year=2011;citation_issue=7338;citation_pmid=21368764;citation_doi=10.1038/nature09791" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=B. M. Kiburz;citation_author=D. B. Reynolds;citation_author=P. C. Megee;citation_author=A. L. Marston;citation_author=B. H. Lee;citation_title=The core centromere and Sgo1 establish a 50-kb cohesin-protected domain around centromeres during meiosis I;citation_pages=3017-3030;citation_volume=19;citation_year=2005;citation_issue=24;citation_pmid=16357219;citation_doi=10.1101/gad.1373005" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. E. Koehler;citation_author=C. L. Boulton;citation_author=H. E. Collins;citation_author=R. L. French;citation_author=K. C. Herman;citation_title=Spontaneous X chromosome MI and MII nondisjunction events in Drosophila melanogaster oocytes have different recombinational histories.;citation_pages=406-414;citation_volume=14;citation_year=1996;citation_issue=4;citation_pmid=8944020;citation_doi=10.1038/ng1296-406" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=K. Kugou;citation_author=T. Fukuda;citation_author=S. Yamada;citation_author=M. Ito;citation_author=H. Sasanuma;citation_title=Rec8 guides canonical Spo11 distribution along yeast meiotic chromosomes.;citation_volume=20;citation_year=2009;citation_pmid=19439448;citation_doi=10.1091/mbc.E08-12-1223" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. Lacefield;citation_author=A. W. Murray;citation_title=The spindle checkpoint rescues the meiotic segregation of chromosomes whose crossovers are far from the centromere.;citation_pages=1273-1277;citation_volume=39;citation_year=2007;citation_issue=10;citation_pmid=17828265;citation_doi=10.1038/ng2120" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=M. Lichten;citation_author=A. S. Goldman;citation_title=Meiotic Recombination Hotspots;citation_pages=423-444;citation_volume=29;citation_year=1995;citation_issue=1;citation_pmid=8825482;citation_doi=10.1146/annurev.ge.29.120195.002231" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. Mancera;citation_author=R. Bourgon;citation_author=A. Brozzi;citation_author=W. Huber;citation_author=L. M. Steinmetz;citation_title=High-resolution mapping of meiotic crossovers and non-crossovers in yeast.;citation_pages=479-485;citation_volume=454;citation_year=2008;citation_issue=7203;citation_pmid=18615017;citation_doi=10.1038/nature07135" />
<meta name="citation_reference" content="citation_title=Encyclopedia of Genetics, Genomics, Proteomics and Bioinformatics;citation_title=History of genetic mapping;citation_author=N. E. Morton;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=H. Niu;citation_author=L. Wan;citation_author=B. Baumgartner;citation_author=D. Schaefer;citation_author=J. Loidl;citation_title=Partner choice during meiosis is regulated by Hop1-promoted dimerization of Mek1.;citation_volume=16;citation_year=2005;citation_pmid=16221890;citation_doi=10.1091/mbc.E05-05-0465" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. Pan;citation_author=M. Sasaki;citation_author=R. Kniewel;citation_author=H. Murakami;citation_author=H. G. Blitzblau;citation_title=A hierarchical combination of factors shapes the genome-wide topography of yeast meiotic recombination initiation.;citation_pages=719-731;citation_volume=144;citation_year=2011;citation_issue=5;citation_pmid=21376234;citation_doi=10.1016/j.cell.2011.02.009" />
<meta name="citation_reference" content="citation_journal_title=Trends in cell biology;citation_journal_abbrev=Trends Cell Biol;citation_author=B. A. Pinsky;citation_author=S. Biggins;citation_title=The spindle checkpoint: tension versus attachment.;citation_pages=486-493;citation_volume=15;citation_year=2005;citation_issue=9;citation_pmid=16084093;citation_doi=10.1016/j.tcb.2005.07.005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. Riles;citation_author=J. E. Dutchik;citation_author=A. Baktha;citation_author=B. K. Mccauley;citation_author=E. C. Thayer;citation_title=Physical maps of the six smallest chromosomes of Saccharomyces cerevisiae at a resolution of 2.6 kilobase pairs.;citation_pages=81-150;citation_volume=134;citation_year=1993;citation_issue=1;citation_pmid=8514151" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Rockmill;citation_author=K. Voelkel-Meiman;citation_author=G. S. Roeder;citation_title=Centromere-proximal crossovers are associated with precocious separation of sister chromatids during meiosis in Saccharomyces cerevisiae.;citation_volume=174;citation_year=2006;citation_pmid=17028345;citation_doi=10.1534/genetics.106.058933" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=L. O. Ross;citation_author=R. Maxfield;citation_author=D. Dawson;citation_title=Exchanges are not equally able to enhance meiotic chromosome segregation in yeast.;citation_pages=4979-4983;citation_volume=93;citation_year=1996;citation_issue=10;citation_pmid=8643515;citation_doi=10.1073/pnas.93.10.4979" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. Schwacha;citation_author=N. Kleckner;citation_title=Identification of joint molecules that form frequently between homologs but rarely between sister chromatids during yeast meiosis.;citation_pages=51-63;citation_volume=76;citation_year=1994;citation_issue=1;citation_pmid=8287479;citation_doi=10.1016/0092-8674(94)90172-4" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. A. Shonn;citation_author=R. Mccarroll;citation_author=A. W. Murray;citation_title=Requirement of the Spindle Checkpoint for Proper Chromosome Segregation in Budding Yeast Meiosis;citation_pages=300-303;citation_volume=289;citation_year=2000;citation_issue=5477;citation_pmid=10894778;citation_doi=10.1126/science.289.5477.300" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. W. Stahl;citation_author=H. M. Foss;citation_author=L. S. Young;citation_author=R. H. Borts;citation_author=M. F. Abdullah;citation_title=Does Crossover Interference Count in Saccharomyces cerevisiae?;citation_pages=35-48;citation_volume=168;citation_year=2004;citation_issue=1;citation_pmid=15454525;citation_doi=10.1534/genetics.104.027789" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Zool.;citation_author=A. H. Sturtevant;citation_title=A linear arrangement of six sex-linked factors in Drosophila, as shown by their mode of association.;citation_pages=43-59;citation_volume=14;citation_year=1913;citation_doi=10.1002/jez.1400140104" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Global Linkage Map Connects Meiotic Centromere Function to Chromosome Size in Budding Yeast" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10.cover-source.jpg" />
<meta name="twitter:description" content="Synthetic genetic array (SGA) analysis automates yeast genetics, enabling high-throughput construction of ordered arrays of double mutants. Quantitative colony sizes derived from SGA analysis can be used to measure cellular fitness and score for genetic interactions, such as synthetic lethality. Here we show that SGA colony sizes also can be used to obtain global maps of meiotic recombination because recombination frequency affects double-mutant formation for gene pairs located on the same chromosome and therefore influences the size of the resultant double-mutant colony. We obtained quantitative colony size data for ~1.2 million double mutants located on the same chromosome and constructed a genome-scale genetic linkage map at ~5 kb resolution. We found that our linkage map is reproducible and consistent with previous global studies of meiotic recombination. In particular, we confirmed that the total number of crossovers per chromosome tends to follow a simple linear model that depends on chromosome size. In addition, we observed a previously unappreciated relationship between the size of linkage regions surrounding each centromere and chromosome size, suggesting that crossovers tend to occur farther away from the centromere on larger chromosomes. The pericentric regions of larger chromosomes also appeared to load larger clusters of meiotic cohesin [Rec8][1], and acquire fewer [Spo11][2]-catalyzed DNA double-strand breaks. Given that crossovers too near or too far from centromeres are detrimental to homolog disjunction and increase the incidence of aneuploidy, our data suggest that chromosome size may have a direct role in regulating the fidelity of chromosome segregation during meiosis.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006211
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001014" />
<meta name="og-title" property="og:title" content="Global Linkage Map Connects Meiotic Centromere Function to Chromosome Size in Budding Yeast" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/10/1741" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Synthetic genetic array (SGA) analysis automates yeast genetics, enabling high-throughput construction of ordered arrays of double mutants. Quantitative colony sizes derived from SGA analysis can be used to measure cellular fitness and score for genetic interactions, such as synthetic lethality. Here we show that SGA colony sizes also can be used to obtain global maps of meiotic recombination because recombination frequency affects double-mutant formation for gene pairs located on the same chromosome and therefore influences the size of the resultant double-mutant colony. We obtained quantitative colony size data for ~1.2 million double mutants located on the same chromosome and constructed a genome-scale genetic linkage map at ~5 kb resolution. We found that our linkage map is reproducible and consistent with previous global studies of meiotic recombination. In particular, we confirmed that the total number of crossovers per chromosome tends to follow a simple linear model that depends on chromosome size. In addition, we observed a previously unappreciated relationship between the size of linkage regions surrounding each centromere and chromosome size, suggesting that crossovers tend to occur farther away from the centromere on larger chromosomes. The pericentric regions of larger chromosomes also appeared to load larger clusters of meiotic cohesin [Rec8][1], and acquire fewer [Spo11][2]-catalyzed DNA double-strand breaks. Given that crossovers too near or too far from centromeres are detrimental to homolog disjunction and increase the incidence of aneuploidy, our data suggest that chromosome size may have a direct role in regulating the fidelity of chromosome segregation during meiosis.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006211
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001014" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10.cover-source.jpg" />
    <title>Global Linkage Map Connects Meiotic Centromere Function to Chromosome Size in Budding Yeast | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"fU0poQytO5odaW6aLRyXe8M37GKqlhqwN85o35JjRoo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459547","apath":"\/ggg\/3\/10\/1741.atom","pisa":"ggg;3\/10\/1741","ac":{"\/ggg\/3\/10\/1741.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/10\/1741.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/10\/1741"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/10\/1741"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/10\/1741":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227y6qkbcFZDXQ4dcew7lPIgQnnFYdpmRW\\\\\\\/LZADypDJA4vW4zNohDA52i1lmFQmWHvTOx7s5PVxP6GYWlKRG3bj44NHx2I7ZwSQ0eIyQ82LEH3verU+Xrx7BI9CZ2xGr6vIawcAXykzGlCCoEagxy2p9s\\\\\\\/UmuKA5X8FwEzreif5x8csK3AyeZdBchI7gsLs1s0557ilvSm7ioYmF5sOQKNdGkjscoW4QOioZg\\\\\\\/YDiAxHoWmLhZmK1KUNZMeGPot0t7AJWG45JsX2NJ4a8VmyO0KygbVNaZMUvt17NvWGA8SUZmg9v0OwlRicKe4bHX7wcnLjp81iDgl4b8KPOhmIajzixpZOhun5KeCVWHMCLwg2iUU3ALdydKbGKfOz9q8GEJh\\\\\\\/s7Xz6ug00SzwRnUr9KgcKoVIJsPYBZkM50aTK\\\\\\\/rYef5tAMaK0s2b3gVL7+C6RyLyRxszaHvWeCqVedAt9yNCna\\\\\\\/R5tEaqaRc1qff9t6fExaOJs5KhWD7p1nO6UeJrrnRyK+\\\\\\\/CP3DRcPHOp0JHvisgOgEUwMH4xT6HolrGKQAgo26SO\\\\\\\/wVsW4eKHFQBMZ1EzhpyxCaLSUjM9Jm2kNZyz9vW0wlDcGkl5fOrPQ5sUoBQ4OyWA5Eh\\\\\\\/1W5L5SEP\\\\\\\/nu0D\\\\\\\/kKk\\\\\\\/Hf\\\\\\\/uLXP2iuwrr1Ib89yiOt482WgKOAtaPnRPs2PR+hiWUcCxH0WgkI8m2+L8ZfHRBd4WkoISJYpR2Cn5b0HTQyZdxhYSZxLJAWW7fsWy6ZAM1VcVSXxmldIpWSDbEgEnQSelDAAQKAQufu\\\\\\\/BiE+G+mx+enpcwg28144VakTAF2peaWD67+3g+nQNCRlINHLTq2SLq0WW771dPAYL4pkm121ENBiNzKgZ8yHDhya8s1umW56R7T33ICc1d0nt4QzaThzBcNCnjkvtR5ycp4Qa0q\\\\\\\/yLmi9TerjHJBe+yBBfpeNTJcXhJLg\\\\\\\/2DdQ5cFklLqQ9Q9+XdN0W3KqyIzlb9Y7Mygs1OLzpygCi0ZxMRUUmMkpSsKsYVkxjfJqqIOiP7FtQSraqcd0h4INDSpOx8dAFcLuAiR6WSFl\\\\\\\/hfbzzZF79vV0hYx8TdC9EPmCkHLLETJmigTKTfKCSQi8vYu5UnZAVPE7Pv\\\\\\\/ok4YjImnO4+K7sZc0GvzEQc0d0Eygi2tvCEwaXaVWZ3ljXzlBkQLIb1srhyFEBbUUJbAFDyMCZNAT0+U5On7IXAlItU91oSrCgKusAR69Mtxe6N0JpHbLe3YQY32OWNKK9m6EHAHAUN9VYm8KduhPp6MQS7z\\\u0022,\\\u0022iv\\\u0022:\\\u0022KPq8yOI0n0LgovyfTxiUKA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d77d3af86dc4afe93c17e146c41a0fedfeadc4d7d89097fb21dc94cd8bbd73a2\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002235o5ldQgIQgQ8YSKPV5Ccgz850hVlHUgc4sdhCQiY+pzrUVKLzXhpSOsfiqBmuR\\\\\\\/fQF2kgLIYLj1dXNEWnLaEcGTsDY5J+K3uw4beqPoySs7R3oChtOzf46nTe22xi0TBleWM92EXyUbfAOuhC3Si35OKaxupwbEVfEAAYB1nZ9f8\\\\\\\/xQrzhmYGuYiGUBsWO75svG50jfosLnzHKUFU3ZxpAEcjRA+xFhsz1NrpIAyiB1rRa50e0SojJvlSDGBesPUJlAH4+2ac7d2Js4uetMUYf0bIo9q1nkt4E\\\\\\\/i9VpPoyD7KhG4jk4LlllR3x6iB1uxIFzvcbtCzOWTkqcSfUqjy7L5V0Wn7oNGsVi1O+RYYicZiI\\\\\\\/bRqPlmB2fc6p3lbKblQoXeACu0gMNYcVnyNx6wQ+GxxmmihlbImDrOiIB+k4wtGylhbXd0DwaXf8el9GVk1T95xDhe7yDa73FyUb9Kb7wIVAlFxrCzT\\\\\\\/eCaK2cjqb1hDH55kTVX+21gQdO9s1dpv7r4xZdQGPu4jpk\\\\\\\/Tm2N9NMlhHA\\\\\\\/dffaHQo1cRdoN3ffoJz2MpGZ0k2VG0KgaHKKnnWzkLwXK5XmmbMLMFbwEGgrYH3A+7CuVa8+MxqItijZ0jxGIZOm+nimEcWVBHAHJeT7odo+qkbKkDedwU22YAOhQLmk+1BI7cMK\\\\\\\/OjicIqkAodwjmKvK8WPlMXXKfa9sXRO5xDUrlJNdqz+p+CaNBUyTPJXVoHj6WAs7FXII9z4TB9oYPl7QNYVHJhFsc647r15\\\\\\\/wwvVi7hf5b6T3U2i1vH1Yq\\\\\\\/1hV\\\\\\\/n\\\\\\\/Wmf1yBmP2WD8YVuYEaa0N4Pyx\\\\\\\/EOAXXhIXN9h67k8qqtp8\\\\\\\/8AKK+1LRhFVn5R\\\\\\\/IOFQsGIWygQpl3Z7RXbmXLZnVzTdldyceKvr3sdz40iG3lcmsBLTXvPoUPOrvmiGYzVn8Yh4UW+TtspYldMeVWMGCNBJjemzvpiFM\\\\\\\/wTrH92E3RFcyx19860pC6uRdjAv6XsgGOf59s4SVnYqntcIFEyZzynyT\\\\\\\/Ifw5ui7z2SZgQQLvRv4UXHtme1w5Iv5VNScoPWzrA6hKnwNjV+vKr2w0RUGrKpRV7EKr7s1d+dRoVXLpoW7dw1O2zNNz7xVz7BIJM8q3Ltl6D4SYfN+Tim8\\\\\\\/Aki1CcrEgKhWBpRLKuU7+0C+ABfuwJ7i\\\\\\\/kxrnB4eYPymylVClwHAQrByldcx3UP1e\\\\\\\/FEijebG60IM8jXsuawhwNDRiYMibruuFxIdzqpLucor133zZjMQU\\\\\\\/ceJu\\\\\\\/r4WVk5QuDurr2laGswbmebM+s5tjl0xgR4y9FMTYv9LhAGiGtStAACaRQmykl1d3HoT1fE4Xn5F+aa1WH6BOWYctEWBQuuCca2aQ2RUUzn8b+UoJ2qG1r4ngvGbCftu3c9uVflTFdurVG3ZVSB\\\\\\\/8sLgs1GYoIUwUXwLe3NABVTLJ1p9Z639+K7FkkCQ\\\\\\\/evYNsof2VfqLXoyUcWxjIdPy285\\\\\\\/C\\\\\\\/HQIp2bPnntsjjbDmwEivPT0x3hX9Pak\\\\\\\/SSTQ8BljrF75nT8k0DJLAo97lw9H0zOKjUGTH6GhiUcpv37DoPxua0NzlGUaMUCGtdCrOl8ueYaMLgooLoI2yyX0mOymI6mpg+7sJYNpf0CuE5ehl3HdQbFnYRkaQc73KJlU4nQrNAp2kjpVhGWzFigpjhmllo7Oth\\\\\\\/OnlWnIGmp1bnLva5m+lJahQA\\\\\\\/nPkxzgmHZBj15u1fY6wjn7shVZEL+uWHbV2ZQTze9x6f\\\\\\\/vOHqGFjQnUs+JJFYcI9oImakSarbSNYDjc8FVcsmADQ7hw3jSjiWqz8NJTqJJDPBLA5sWw9Wx48yvhLtJdB4HAbeW9OoGT0TcUN4eFYUb4ywoJ01XkLfSge7V7tHk25Lhu2uQvKplobdtTM647RDOnFZUweoTDJ4vQ2M81lC4sCZEj9LhOmK\\\\\\\/2y3R3\\\\\\\/b9l7d1itnI37G3kRI\\\\\\\/UhmYl1bPU03CneymVNu85GI5viyk6DShpKbzTTX4GjVbD2jyWwNSAzkfeNw9xYppmMxm6B9ssJOj0DZQ2BrmHmAT0LVWGIBok\\\\\\\/tHDKr5uDwFVRVuDx82U8Yz7PjNGqq3CiqKd6QTy3Ob1v6tk3XOIcw3ogETDX0cKJjfpaoA5y9m9olVC\\\\\\\/gv+E3eA+fPrJMGZ8zIfS4hMKoSy2feVnhn4z6drbLsKf8a\\\\\\\/kjCPwSZfMLxUj5S7PqDKzaIWch7us6zX2omHO2PMDSH+urv47fBYDRVSUKxpFJuPhzBEYsErA7kWm5NFFK6FEzw\\\\\\\/LI+E9s49R6nsC4afAG0vKTOxfmO4dJmt6cBOv1ROmeZzjlLxGAJNdN1V8YcFS4jkAe8IJ3o4pdhyc+Pv3VFnf3Jsu5V\\\\\\\/HR93Q+MAcrOAQd\\\\\\\/dieGTSAeGgThFdCRntQvyQ9iUk\\\\\\\/kl7w5nlUYXM6inKgUWyoU0Qr19Nt6\\\\\\\/WtGJ2zshfSV+BMaWxwIN7iSmHs2keikOLcAO+lH9Q+jQJX5ZgZQZl0xLW10\\\\\\\/cvbb+5TkUZWIkapdVEkcDg7e0zJY32uu3J\\\\\\\/9a+znrsJnpYFjH3osTMZOQnJKKeeKtoO5Yh+fhb30bf0BOM9E9kmsQ6+vbCW\\\\\\\/aI8aRqfLiWCF2dLxiWccBgIz2oMLH6M7hz+BJhUzL3KKn+4XGDD3oGEHmlCMyC\\\\\\\/EYFZ05ptjUekmpeLCgzauvvNsUNDqLUarn409MpxhmOEXilXNaO+e2gm+8ivCr51vVsuZIDGHShZyRV9EIH96ItRgvDTal2pW5ROPCF879+KeEYSkjKAaDSsnv7\\\\\\\/gmNHi9Derl6s8gahT2u37+rH4PFGLY8I3+7puGCJrc+rQ15UeZjQOsEBbLahjYLEXRhXMYo+CJpePEYK2bXTkRXsUYbmfJUVvc4dQOXqE+VJhatPE\\\\\\\/EDsj78ao6DGSuy6bAdeKBW31OXx43frxxtBm3y\\\\\\\/gNGQHt7jUibJm3VwCSkB5Do91P+vM+MMDCStMGjdf7AaUmJ2q4XMGfdQs4DL1Sb4y4jeJuJyuKWS\\\\\\\/d3oHl69RUq8tCXIUDnrMBiUYDmFs4I68iHU+WMMesYvg\\\\\\\/8K9xPiJBKj5CVypEB2YmJl9fy\\\\\\\/jkDv8xTSsM6Rh8sfCAn3I6ELHUgNC6kfHZJzgj5wIRmVBi9k+qBJASUCAqZJC8wdMTmeZ+\\\\\\\/e0cBqx4H10DmuLM1FUgWMOcb59YEPPCsPGotkgNFfpCYRhOZ0S9i7jZ7abR\\\\\\\/lqe5gIj3IjXXEUFOQahPoOs\\\\\\\/PCFyYjtB1\\\\\\\/1Ashst8MZXoXjp1A2bfVCgxqWMVeOYBvbxhYFLukeu11SRi4y2IEPl1B9JmDcM+9s2tF\\\\\\\/l0a1JRhv4vLf\\\\\\\/\\\\\\\/bBnL0CNQGV6Nwp+4jp80ike9nGBeo94iM5TxPglTeT34GyrlOyDGd4bnhC22fu9V9ycBTeva1zoZsoSvvXd2ea\\\\\\\/0CTG51wjfEIOfUiaVBMHCuVEHFAS4gSFBCJ1arl7p0DfUxjQlNdQNe57c9hmInjtuPc4qC7532I10qxUfkSJIsdQnvtyZ0VwuMIoevlBykTtqecU4WvDOnvODUDrG7IZbHCBSFxj9jo+ZbmB2N79sG9RDdHBBdTCULFdoOB2tgP56JOMLvUJLbixfnw6qZAuaylBnhvxZ+mL4mVNXiLnsn6HUiXDUqncUdb5SNuED+Ap8FzL0a7OXYkiCDzKXQ4Yz+kz8OB1JH9OV9LD9ZVO96oA\\\\\\\/77ZoawQsuQr3BKpLRB1hDeKef+k\\\\\\\/sgbtIpUdYhGokLXRSipxv4KAbjrg\\\\\\\/YQc\\\\\\\/uaKe+tH7GVdBzl+zQIPWSQ\\\\\\\/OtDq93WS9i7QeEeCe529lbjKjMbPM2ShgKWyqq2DMyQTvVHlGymIsITfbj0BROZn+Ckh8WNRXizGE1XjRPMfkNzyj37Ib7Sg4s7ypawsaskJMIYEheNvVZqWtm2XYcKPwbs9JiBvOCaexliI66P6OVYNhdAFw2af0u9MJG4jEs4KsB+IwpVZ9Fdx3KL7bcF7ayTxOaZOyd9+klLu0jqbef08ONpAQBO75dMylB4Or2427j2MuPYLlNA2MvlD98VwSzBIri\\\\\\\/WTHZH8VgiKVgABRbq31CG9E7FzgLjgsjVYagIX5pzWbmcvuwL9GTbG8Sy6lidhcacKT8Ahkc25lI2VPsvPA7HyuTr63+nZ9bmcGAP3wy6P0yCMrKQhFBY9qInGZ8KYf7RTxEoKvkmw\\\\\\\/mBbXO3tQ\\\\\\\/G6kKHUX2AdtaApvJ0X1M\\\\\\\/VJsUgXldWul4f\\\\\\\/\\\\\\\/G\\\\\\\/8Ke7VVKtmGNNxS8AiBnujrCbUAhrlToFG7jwTCxYUBadS8XbmUkF6a7q2aU4ZXWY2U2j38qsowq1nfX7oVtPqPyzANAmu++X7rh\\\\\\\/ZiMc6vpRvifhhFIpCmSBGYKlZrFI6N+vZSvBWnLLMyqvzdHmyvIsMZ1hcDLTqWWNMDgftsT9IU3Clb0xgZIJx+BBnUnMZFeIcLlRnL2ueK82H+9vc1MORVS57Tsq\\\\\\\/asV9J9s31bnc3RDEIi1dFfGac7AwOvuqUHAwEoMMyZ0JEjKhNgIualF+lB\\\\\\\/fwNLdE0tBStD34MrUQKoYw7N34EdBwqOxJ+ACLHwMI0DQQsyPVBlZ1xLyYkCto8FdzsNX9uq3opESi5ScX7SNmMa4acHuASruTQrZHdJXEz92gcj+G8qCXS9xXZ3IN6ljXjGoy+KSjHGOJ7eGNlGoK4yC3dhWP4N1nVMWZKljW0dzwzVhrQVWfdUWK276dxPkkIGa0tEhVXjZwmDDQTyJUqaE9GPr5GaJyX1wRzI0YaubdqDnn0+a3FKooRwHXelYih8uxPyiWqna3TOp9E2kSNCdDBwUUERik0iSANB2h+D9kxDjkJIYtUwuM9pKfeAYTHX8\\\\\\\/mbgc8ZKr7R2ysNRwhgBxSlw0EDTnIq2HTYUEIPdQo0vD6I8Nz1ZOR+YZz00ehQUXEA3Lxcmnb46zavbsh2gDo2eroXWpmNyWxa19z984nXCgYtIq0YeyDV6VgbsuK41EqjUQa4wksIHHSqCZoy\\\\\\\/3r3HRhYKxhWLCfmSWpIPNFIN2qjGxAc\\\\\\\/B\\\\\\\/UqSavsF2tM9ulmNnmAH9X6fY9E7ZImhQjhIwFOL7ntY773XfShVevcqAQl3BW+qSYF98ZiNxinS24wTs1lgDM51HRXut9LVBEVilkQgzSxxUTodm5la2i8PGuL0loPaSfULF+x\\\\\\\/dnFu2+3Pjxj7VE57kQjQ9DN7zZcaXBZYFjhFq4HGM8hnZKbli07eei+NNV1ljeveJcxB1D0w9O3J8H6iqIirZstgdFAH3KdkTpUwDT6ezE\\\\\\\/271HXnmqB1QS7Du7XuSEPuMOfQWnFsQiaN8DDVDzm34Cjw2A35HRo+Fm7RRyMJAvbkrvkOe7GOfpT6\\\\\\\/jn00RyooXw22oRHsn27SIkJ+Ao4LpDFsu\\\\\\\/BUKgIxIWg1ZvivnV54JsekqbOIesZwW7jS52Z3xA3KU9vc6MJQAPIcXodBR2i53+PZ+6AbC\\\\\\\/fsLJPna0HSRzZIa0VksOQ3h5nqi\\\\\\\/I1d+MA78GgEFHIwPYtFde8legBGLJ\\\\\\\/fPaMj4SmVkwLgaGkz7xz6\\\\\\\/eH6BbL7h0eI+PdMv0plduEcA71ya234TNZVMRAibDGEqWCliNXP0MwNmr4yUtoBU0Zg8AdpI6Lm0ok\\\\\\\/9V2oOBtTfuMW5gSAKULjw5xsST+7rJpLXKNvfZKmFVvck4zhtcAjfYtiADnrC24KEPVHw52gd2p7LoIA+Xe5Gc8LXHuLcjAfpV7BuIVrIZVoCvT08G3v6os4LfI\\\\\\\/21EL6tQKwaXzXvtSNnpWS1RQfptOV7tz\\\\\\\/KxOr8MWPxSWprBb9otkd5zEWVXDTFhzYxUaAdjHtt7Z5zZ688gaNE8UQwKgk38XKgoDX5bY06AZ3l\\\\\\\/6Cp+RMI00eCiVs1keePgTMitwG57Zprixs3pEt3EiCUG8yEnoeVQx7dKBUA43Sn\\\\\\\/n7YY9OLc4o\\\\\\\/\\\\\\\/Paem53pTziTfRVeDH6r3L0CgS\\\\\\\/xdWH+LSA8KlhB02C0OM3bMe9VrZx\\\\\\\/N+X9keTYOIpCW8WhwFZifXp353pkCAr0gSP5YtDzX9VpbyBQH49YUpjFRBIcXlhZ17UzOqms5RoQxgOnjyUEPcgd76JZOhPZSLQA6Kiy0jCwdyCH5zUA67bgmciL667dLlarlw967ofF6HS2sexOL094bTNvc4JQVac2V9Ie2fg0XA8ZrUc6iGKOvPylj664Tx\\\\\\\/GS3zwNJGqYARu6Qdqd0J3Coq3kQsKyCjsIOG3ZJzViBpizodWtHvEjEO9S+xuVPSkUF5kS5LU0am9UYRRD5Ur7M9XUJgKMmuRMvFu2\\\\\\\/of8Yn\\\\\\\/ZBZo8qRSDIlIgEr7UgkGvdZHbX2P4XtmwjGL+yP65zT2OAwtJWOLwQ2QxLJoMpIh6E9UHqK89EF\\\\\\\/qu8JklQRrA\\\\\\\/IjY9hx1j41SOVKPW3o878z4yrowEvvxIquscPxxvIbFON4tdTJKWAzTLwoxFK5WvUwL7Icc0ixddVpKXZJDz8QsgWaWzG0mH09NFtlR5EyEM4ReLtFpUZKkLgEDCmKndguknhk7IsbqaM09YkvYuGn0TB8xKTvT2PD+jJCmIdxpeY+g\\\\\\\/ZHKHDT7mBsY40XW43D152WS+ENQOd6Kn1EIrBiI8SDES74Ra9SPSZkgumeciVWZ8t8KUdcNzKb40aNjS+PTIkfDg9yN9DyqOR79eZJsxxB8h1Vx2i6KnVhr3VStzn4OPQn\\\\\\\/AIAl4MOLLbdzo1K3aThXdlptGOg08teXYQZOQ9sWPJo7TNWNNjnkbQvuUe5nyQptnkCub2wxhcsDWiGCxbGtbCiAHWS8sFfPqiwMU2RjGy1ZE7OuM14q36wAK8MHcvDqoizVPD\\\\\\\/gogFbiEPNPzFmfeMWfsbpeLl97igHcybFb6bug4ihKW+fwF5fhyK6tosgYMt18xDVoa34szPmbOPWa3WSE1A1L51vURBRUEV7gNrFFXtiQTeI8Q3mFzXmyipnZNRU+uyNRfbPPolD4DG0xJAxoS9P52Y39dlySEEkRvrI7R6v+CQBRsU77IJ6XKLilQOHItfFGblAxw2n+oSSIQuRu\\\\\\\/oVGZOWTqIfSx8vFIshFgjh0W2DaB+soyrlqlXAbN9XmMwqdrt+dUCAcwIEXnsUXU1NPAO8eZGD6GePs\\\\\\\/pbljI+JFQnAbYof1eWOvL3bZsz8ZOcgjV3ljDaAhbTthzI27aMSKRvVY9iZh4d6EUPx+E3ELOERu9DXB6ZOslTdYaqu3Fon4oGIKK1pwntL202ZJnBPB5OJ5Hmy7a8v8ae4A9iiQg8E60fQNXi2UrTgEutMwxYN4Y6GSapQfNEQrT1UMhWTvfAr3tT1I6usfckiPAD1ptzJMwyfSXorl2u8iEyJb6E9QtVMQ8k35LByeOfNmXALUzg+jJD1yh2DVTX\\\\\\\/IB2efOLO\\\\\\\/pzutHXDE8XocwFFv8OrQJgyECPvEuRL01Kfzvra0q5t8R+NNCX7EALG1J6l+5xw8RUbTsv7Ijw+Lu0SVPWK8\\\\\\\/xz7xjPJ5c085V7CEDhEcjI2x\\\\\\\/YsYPlPcWlpJBnzII+ElY1C8RCdEA8pLIDhfF\\\\\\\/UhM9\\\\\\\/LSmKgCwM6EGYyUZppubgpeu5vQmGS1AVQRIKyIT21tasAWNh4jPOdjlSzBViGBVL2mK4dvn+2aNNA1iHa4ViusfNjYk\\\\\\\/GrUthxKY0jVuyf2DsWLjUC+yrSwvQLXjkQzgR79JlqbiiqT71i83BKdwVVMSCdV\\\\\\\/CMvJue4qzC6oqgb\\\\\\\/QeEBeG\\\\\\\/RtJcl28acmCFQ7xfiwVk\\\\\\\/wTVa1uPJSeXHe8bBEBjp4mggMRo+0Al3dra\\\\\\\/Fg3MVYID8j\\\\\\\/lLY6\\\\\\\/yE7cy1V4FLf1PaLztWcVFZw5KkEbIu4xZRrXXmmuG\\\\\\\/V+LQOfsCQZKl1b0+8kexIxurpm7gCFMFIQEoT64EX+0VUC+2KgJHUyHyuSKI\\\\\\\/3zhy5R4+zWW0FaKWFR3k7JUowHwznV+qjQXjXSdIwTzmPgQ+bsV3G391eNWScsbbXDBZ9olSljdPRsFCjkthlrE1CxVPt4g2JbEiztNIIGbi6cgufqTrA1Hujia3ZcctXjwxmy2t0G23JH+pR8BeD\\\\\\\/s7N6f0fbUEdZZ1Itzgz9ea1CyleuSzE3GKtvzrITdSfax2kESrT6cFSqqQ2sGwN7schV1DwkEhgasb2cGeMUX7hhJ4aNC4dEmFhU63PJ7TTKQX5okiEh+RfVoUvBjWlNq9eCYJTzlWdGZV3a4CXrZdxI\\\\\\\/+RU\\\\\\\/AluDXKkJypuq4AnT9yHZEpzX1po1ydhVal1+pF59XOZFARVyTkADPUEcwpI427D1hrsIYdYM7hYIlAwRxCb8c4attieidV2ndT1YZEZfh7\\\\\\\/bWlPe3qY\\\\\\\/faT4rhwSImRf4wo+MGl6dMH4xVasuE3mKmr0g2fJbWryFYbaMQUOth0U3cjaTq3WkUBlo\\\\\\\/sDR\\\\\\\/+NQ2K4gmf9TXIt6jpxidGVRRbx4BhGnZ5buczxiWGxrmkmHeFbcbCvluzraBuqYhJxaZPsiUWgucs7sIhtXJz7NfsvNraNasFM4hhFbruf6hFIgfONUJ9TFYQlZ1aoYmNQKcO1kyzOFvWSeAsXNDZ9SZ8oRV\\\\\\\/JM6CkMsmRu9VByie95iN8tMK4U6Y7r9TSAFDuxMwKlB2gN6sJUiDCS3+mypLFoqQhKD5QCGdCzbx60Jh57\\\\\\\/+uiNkwCYO0ZfF4Mn2CKZ8LFei8LOcC1DR0kaFWhXAbBgd7bgw6RTmRhSEjlbsHIYbhS2gEvGxjj+Z7S4zQa8veA7+5ZCU51JSzOQTduI3EnoACf2M0+J05UvjDHmHToNjrkDHwvZ6nwYcFpB3tgXUzIKqujCPHk7DuH7WLfKPXC1dd9ALDuTaIKsfBEtn+zuYCJGy4QtmeFBhv8dnOF4vhlieCny3Ws1mA7XO+k5d7mCN2CWoFck5QkvGtHIay4dyAVIz4CCep3VM91UX\\\\\\\/Vn9UIVS6LPCr\\\\\\\/6rBnySymJTblMvr0T3a0\\\\\\\/aZXLc5cXdkdXqk0\\\\\\\/6r19aOTA7Si5mRNQcydYb02Mzfyu1etuQ5i+\\\\\\\/1iMY3X2kkWpOZ3vOdyTFtvlOusCvoixdrrtPMp+YKYPNIHietfNwG40KZd0vQQMC1lOi76RDzyyjV3H0fEtsghr5qSO5MH\\\\\\\/NvrbJTU0iWo13vFCgRXVMGHB6Pcco1NGxzc5BVtYCDSVgyJEE2j74ptRpLicSTvLOCHndcpfMVvUYyJ1+pNkiTkQUA6kI+NrU+OYijE9abdhYNhhbn5JzV7lgJZMgZarPnsiqzJOiZI2hVdPztFBJ9YJeT9GINEWau3GZGB3p4BVBF6dzVlC8XsAjTDyMI\\\\\\\/\\\\\\\/XATQp3w2q3zUZrDg3hbgWqfl+hQ3J\\\\\\\/1gew8ZxhtsN0bs7FxVafw02yokZLroTdaFWci7AwVMhD8JegLVHu+XinHpy+ulBV9OTOS4gehLB+lvE2TfE9wteLV4nRMh4JoqaB\\\\\\\/ELDfRtM0UgTh2D+Mq9fGA1RjLIbgLWpTZZ56tD\\\\\\\/dSyBJhqzj1GP6tBYcRJ6N6ktEJ9kg4qGad5yVTcOckoxuYuOB43YSfbZw+738\\\\\\\/bUGPtaQO+Ax+kjQUO+WcijIcrUcygdRhdicrzjx9NxGMqQRLDczqARoMsAVes8n\\\\\\\/AfuiDY11bnvV\\\\\\\/vmgRzB0OWJX3xfJQWmYesxskfQueEzWhp1HgjMRomzpQbb2nrepAP+ZZj9w06nh4yALnwuW7h8EI10FJrMZxppu7nzge8o0P0ZwftOXhw1oQzUfGgQveF4E0KCA+NeUdeo8InS6RZsytkaGB8um\\\\\\\/2csVAsxzOmz07KSu74921458TUKYzFPhPQHHKiC4jY+\\\\\\\/+kz+PFlUTNmGXGNm\\\\\\\/EDP4dVOTqlkTsVKrTNwO9DlgRTB\\\\\\\/s3j8Xr9iTLX+PSGn4xchHdqUddALOuPFrM8E8oUPi12MDW8Oft3GJgsX5yWdlILVG7e25MqyjV+ErQgFHaT7PKEcePGP\\\\\\\/Pt63qTPXpKr5F0MyQ+8gmulTWRMCVFW0AnhgtlD9Dj7f+OFrR+1fWW27sGiHs0Vtpos78ABANrbGUSHMwySdLql7JEUtgC8cQJ+B31vkrX5a2kIyftPmdOIIA5FswAzHO1JiyWuUPAD8s9HIYCudfMQxgBO1XIdeizbFSqOysQcmAnPoQPS+e0kvGOc5Dt5Q9r9l7Tj4f9bDTF56R0vJt2XSmfgBC\\\\\\\/XOsJwKDvowJ2hL\\\\\\\/HX3Kx5pBPgMCNI7TTBC3fDwHb6i665fhIHrdyYuifRIY1IDgYmaNqYDBV48\\\\\\\/Nf2Mh18vyZWhm5o9DVaaBUDBTTK+uYpTGALiC6+GyOCH0870ePsbZrmtVgYPu40iMVstruXpNDBo8qbuH9bP7xgUXCONDkVEcpvPy+CW20e\\\\\\\/VpTpBvDxXsI5CtjDUYLKBBjOsjz8Lw9XHchjAadLQYq4+BV5imOhWuvEEdabiDy9cLDqAiM1SUArsV8IAxKFxakV3pGVuC\\\\\\\/8tmhKocJmTZl608ouPmpxnE7yZtrT4+6qeV0TDfqm8zV5YNHVjJgJa++ml0ekNKaw6A0OFu1cWSNvn4yQV+XC0wOtywK+Cv3jzoTjAWuhJaXkih9Q3KeaHkPBLeecRgo8QocZK4GFdGQqmAr6Wm2CcV2dXDplpSEsOK5GAMBMu8n8SgjoEmqNAIn\\\\\\\/j622Cd2Lkb0P58r\\\\\\\/KgY2cwpKGrVSfhxrRSC5kJ1z2fIxSOMfwy937Y3fZMeUDOO\\\\\\\/X+X90Fl+k6u\\\\\\\/JR1niLFZIRmczfCgC70SUilRGvYj70qpeDXYHmTRwi4ZSCVefjBzaimWJSH1\\\\\\\/PogAIjtOqwtlL\\\\\\\/aaaJrJkckrueSp+\\\\\\\/gXbKVs+cP29gm+\\\\\\\/rb8y44o+0EShcbo3nUiGrrxXtPdvZ+EZWVKrTUYMx54K1VeDrWs5+Zk0XeseoJs2Qt0lf3YViJzp7PPSSYuT3MfPssEZCrLfZoBcscfCMhNarcZpOsa+EqvNkt8NN+DSBX2lK2neZM7NPoR6YTni1FJLRMD+VzQiCl3k7pYzfj31BngKOGZ6\\\\\\\/Zt1gmNknvbpHeFgURPnv\\\\\\\/GLMG3BrdbwA4T\\\\\\\/6JZLmNoy5ON+pyA3V0BVYn70sprJBk9esxes8A4WDfEC4kQpZRjLV5TEGl6JBjD1nnabmqJvhfjorc8l\\\\\\\/BuJHXvKq4V9ANkIYjGaFokF6f5RrDU\\\\\\\/s+nJ9n1RlRkX4sFxv\\\\\\\/9O+2sLZxUAP+Arm6pDr1lh\\\\\\\/HN9+Zj9tZnCpZk0yLzO1rMnCXNtLMiCivkGpfrixbIGQIXOBRnxDCi2gS3BmovTg8euJAXxriCKSYSuImdAFBkPBn7ESHf8Okq4eHH\\\\\\\/\\\\\\\/uOqZWkHWAdFbihZM+aSSF4azEXokU+VmdERexQLCBOutppx6nkPYHyGSRYMditIv\\\\\\\/mDEC5k850OB0bHDdyoA7cbtKwxMgE7V2ZenMK6gU1+jBTEhJZ2PIfejPK6mhRQEXe2zhPESm6h22e9+dpy8osB9qSo1PbxeOvInItJHhR7jZoJcFQTH1TrAloYJzV8KEWaXjsSpmd2Ag7dhVqacZj6\\\\\\\/iANsDak2Mw5mrZevWxQNAhCMWRqhl5PBbuo4AQOvT0D19IKrvJNDHQ4LRwfbR7QVmOllqmY4KUFt2u8aLmHR+XFwxGikM3k9PJbQYtA1TBVr2Ga\\\\\\\/rbBgCLOS1YfTXcS111hU4uSu2S5KCSiRhZFjvio+ijFwXussh4v0iyXMlC9CIcl88xqEkjgTsiVXGjJgICL85aQu3dX1n5s791Hg6DEp1mceorZ+x3C2Swz0o1F2Uym1NDM6RvFQ\\\\\\\/v6X5NCkkO+SC9O7kAXrHldPEwzmRLlWkSUXZqX1dYtAj+K00IEOSdkgtTIp59ugJNdIH6ichqPusnGXmdgmxC0wHlKX9CXvPxxkGqH0h2L8sdxr8yJ9kXRakVv2vJAeiEXC5B56IhdBJGqbMgv\\\\\\\/5d\\\\\\\/BInO8iHgAHHXyUB\\\\\\\/KLtmz4ldL5fH2ldxDcrndDLcq9rKttZ8\\\\\\\/ibKpcJzFwz2ad\\\\\\\/Rxqt9nZdESyLxs\\\\\\\/\\\\\\\/PTTcIwzIGPATHCmJzBo2fiFO1AwskZy+FPlzy5OgydmPKgCqQvAqJl0BCoNEdQ7MOpfxaN\\\\\\\/fAjhhjGWF7mSRC9vS1YE7Hiqe8pml2rh3GzZf47y2FcKkQBVPO4n1CBRzMhovIPVzQ4Xd8aZPgjlZOKz80GVd71vIL9PZfUA4plY4ScwROqnBDLBp2DM\\\\\\\/9VphMoh1L8AlEeg2gR2twKpjvftRt5dvFMM22C6wxSNnVIcxynUmvQO79QP807ucZxLUHXyh3f+6u7vHfeXwA+mh9JZl6EYUZE3ridTUoq10ywC1zaPdoeT114zm2jJ2x8eijgIXn9qJHFCv52FP9xFS\\\\\\\/eTK09LRgsKkRUmcOe+3L68NX19uTDi\\\\\\\/F4ycd9+DFQ3zi45QBTkfPlF+XeG5TgA21vbrlrUd5aLxWzPmdRTzq+SvzbTuZ1MsJoOLS9w1h5TNRc9uWES1mjb+\\\\\\\/UMWdJt7rMcyIvTVy9kj9meb5BPvmU2KblrxiNwvqLtiFJQqeHJgPVNADc5ud1JMH3sRqyDEDDG3HGjoatz1qN+qdxqFqMThNZLnZMpGggiE1CQ0\\\\\\\/iDPkt2U8tqE7Z\\\\\\\/wVBEn93D4jcgxuiSimMbitMhKOhBejgX4VyTQA9HWi9ZpfLWbBl9D9pukxkr7Qeneu+fk5Thw9RX6+m7BhO5zz\\\\\\\/WUveCsZR++M7792mXvZb3dpcCOFTpDeAOxquMXwYk\\\\\\\/SQtpEDyC973Bh5OT4ljQqZ1q3\\\\\\\/9emx1yBZWJ2UIvgBKyoJkFUyde97VANqLZYcoWpePAzcYcJpf38h26aqFJwIGw8wHVg6RIFzYb2EgK6rMOaTSWbl\\\\\\\/4Q08hvCIqG52GSQ3G0UsgKr7yTxoYd\\\\\\\/9DGXEtsKJ9Nmu7dTK1RBVGD42X3CDVO9lx8hvdzS46rKUXl3F0hgSl60Z8w8TgBoiJk2eLyLIge4fTrRipq\\\\\\\/jl7gjeEWJiVZFfh04HNllavLf3yhC0KZJ3omkKHl8GMUeZVxkxc7XzajdkrUJaeLNRijIEdMVc9+COptNDkODN41KuWLP7iIm56X3RqFB7\\\\\\\/5ZhQ\\\\\\\/+17gWlih2jqVqW8o6IsdMlzMJ90L4dhRpYVmZeQCIdf03RqPwHR0ALIb4jC8ZSVRifmJVxiJOJ\\\\\\\/7viTPq3JSCQzkbrgP9USYuAYQhBZTVWLJUvcre29OXluI4SXaxhI0nPi182oFEetQoFrNHaQsjwWVih4mLZVXLLvWUKoCVtR70Jid9+iuQtCfpk6VAKPkIaAMaSNVjDHIhqt3JufwNQ4VD5MoAH88u0k8DeQE2yFb91NSwPu06YDolvHCaYTQW2p22sAW7VxyHVvgp4SLfvMyO7lQmJ\\\\\\\/rTO8Se\\\\\\\/F+u\\\\\\\/RhBOekn8tgm9ebrwVYTjuJpMSizJnuK0YOMQAva4\\\\\\\/D8i3D2ioNBLfooHCLyWnCYCtLLH6\\\\\\\/gjoYtXdxhzCtF15ILGUkA2hCtyc5wPuwmTVawOTCOJaFs\\\\\\\/srklv4FIJeisoiaY1oPB\\\\\\\/b7ajfEzBK58RRjXHB4BPzKS9IPpI6lcWbsjIRtU33qZ0dNqQkRCqdTUPAZ1YonRTeZf32z9KWiWDPMR\\\\\\\/sN51to1NOY5leOrpY7kUWfgWYbUmtP4BWThIaUIauzGZNjryVDYu0ZKa2hgdXg97XvuMf1jcF3ZGJEdcJfJj\\\\\\\/BG0\\\\\\\/61DDWWAgO5sVbQA1oqoEhgj0B6g3KY1WapGTM5yhKCgPqcG8nswExZ06TZFM9B+HvHi5PEVO1BcBoPCHsKw3ELtvqwgNFpKO04bIgX9V41E\\\\\\\/e+pO6QMakROAw\\\\\\\/E2shIt3AzKQGizZTxA7ewPh2MAppeeMZ6t2SsX+g5Z4jJhb1KPSB3wXoGKyYhUKHRSCRTLJQ2JLhcWO90jjvfG\\\\\\\/yTD31U5DN00AvqSCCszZjpu0NcdO5Tgj63KcFErDZS\\\\\\\/ndcndWBVrIgukulyp4YEazIDapqubeyndSMb4l\\\\\\\/ZHg1\\\\\\\/ndqaQw30kjNGPCsFwkU8jj+5mdDwKPdBV8+0\\\\\\\/kvpYd2GabRhqIdCMeMwRguYY55g5pApNvePZ021VnVd1L5xXUh6atJKddmw0wN2iI9niFhjtgUY7V6mkZQV5AWnoPpscnTwWb2Que0IITGt4izkE8gqQRIPARj37KTZGBpgZtQHthJQpoiYFAiA2IcyalxVTXgnFeJlYwCaD1aV6Wj2y+CXmWQpxaa\\\\\\\/WxgZFEhVIQNIsIA+sfMchL5TRctx+mOhPu3lzgdbMWnhhurmhyetfNYZEBUU\\\\\\\/phOf0EJo78iO17nuhakZUH9H8IGiUw++zg9LgiQu8ylvoKK8aam3BB5N2s\\\\\\\/UQP48EuDwhdy29CZ97QzkxMDo2oefnPdTdy45dmukTGdJrXdMmhPwD\\\\\\\/pk8Ai\\\\\\\/QTEe3m8wpymcUxV6XjmqCE6FQbupui34gULq\\\\\\\/86z830nmtafU5P3ynytPBcITahY7IFZoXDfh0ImYgyLUecS+CxILg+CxqaKRbW3z2GPZ\\\\\\\/wDMTLqR0BWdlIqofq8EeGBxg06C3+ah3PZHP5+drEQTEUzhMnttdLy+f3Pxy4XhYFIQnXsvgMgIcIjk1PMA\\\\\\\/RaK7f4Pw\\\\\\\/wHWVTnWoExqaKIk9ndWDus7ZOhgWb\\\\\\\/fjKXTaoH9puMMgumZRGTgyLmV9dJd5\\\\\\\/oc2kebLgNSG2v46tz6R2PRbrSMNmqY9b1uFrYtEoU6WRcU+WXKp0qPV8lSTent2iT9rZ5jsa+uhTE97QaPsRVC+e6DkcKIciXKAoB2sZ9YuouSob7AmmmbIyfsikqsbhaotR\\\\\\\/t3RxXIIeNcgRICpVlJQR+4Ko4MmgZ146dLBWTE8eEg4K+JB7nGcIEKXxJob3DN9f5iqKRNSeYSpApSr3eMY76G5y+in3cFGpS7Yi0H2IwALLIJYOI\\\\\\\/jkq0jAsgPcWpdWpGUoot8lSg7QOUSSQ02Rw\\\\\\\/GBc+uCtb1lrLtmxLf2CVRe5Bvu4s6CJmfgpxia72l8lhpvilR9tCuRmg45tB86\\\\\\\/b7\\\\\\\/C+W9\\\\\\\/BDrdlZL+KBsia3mD0OEHewUn4aBNtAxO7ZFgCGn6PRnRU2oo3ItdD0ewhjhOVn8kTH3dYM+ZtgbcDKDwC2Q6Phk3igjTpccjSL6amC5SIvfAyccqCxfSWvjnhVdV5CfGROlSKZ0L6ENVV0L+aV7OppDp7oim1UkWDQ7A\\\\\\\/3Pk9L+EtOPgPVmQ+iKwSBXuYHvtJelylYSL3mG0Ddb08hJDqyzZhju3CfsOAzYQG9yQq7SUO9lOz7KdT0r\\\\\\\/laW\\\\\\\/dzR1GOIJc6Ar\\\\\\\/g44+KFRIDURsbq9Vveg6Xrc4BE4bptEpn9Z8c0VeaKiLLeuGix2NwIdtGYqoPlWlAJ0mgbju5er3cdtAUeaTd11cz\\\\\\\/43r0FyPVhpWllyL\\\\\\\/b1njURKu3byjc\\\\\\\/QmZavf5RIKHZTUNWNpB9R6XFEGQBBOiNgcdbRbLUOvFQOKhNHXqEQVqVOndTxwHWwdYkZ8ElhcuwBtwHgyXRUL6zQdhwPUbknWSMXoOYr8GLWvunEmRfuZSNL+Uztb\\\\\\\/URJCsoiVpgIMSAPlHs7fcbwjdFiNut1TpN7DcvXGozjTtaSQ6d3NPhysSa9nHZa1XHExiXdFGuLFvcxc4rMsuXwBB2QjpEu\\\\\\\/GrpXo\\\\\\\/u0y8rR+ML7hiek9UD3bICnqgtBBo4KAvodK0628zx9OJ8YZzQpC70nk9u0T3248i88i2sxSILy7VNTeRY1aarqg9BdpFxIsg1DgVVRqZZz04SS7Z92yquKbxKbuQQq85DspoH9OVL8PTB9Uon2qbl0L6N66iowJCNJCy\\\\\\\/jV4F0rnDi2GmWyioo8VgfNu1CQcD6R51rOLRrC3LdC5vxWXHnz78XsEKqZtx9PJtZC9P9v0\\\\\\\/7AKfsmacSSRZM3qlCClw8G5W86sCoxPN8UZNUJj\\\\\\\/E6CsdxY6GvNnCDQoiIdSXdi8etKyE9lVXTX54A7hBNfnTSVXyeKQdDVatloTv\\\\\\\/VoMT3JPP79N3s0Iz9uj4zGgSSVJ+koSQYnKmgtI1wQfI0cv6cy8BRCtbL71dy9L7g1MTBO6eVk3aznnn+gjcnapw6vgCiReHs\\\\\\\/kIzsch6iq9jJMCy3EEJP0P9x8qNv8o8WpHUfvSxLUsJcziajv6+j0ec5h\\\\\\\/P06s8wRHsIEWKWT2eybGYxxG1eyT2h31UTenkpR3oP8dwwCDtzlKdw8gX5\\\\\\\/pOAGcYCOYzF7UoekxyEVriMbVPRK+YSpFzhHzXCqAs90bANTggPia0v8d+1Q5lF4X2QXNViWAXhfWxfVJKi9uLln5GGE\\\\\\\/9xTZ17i1iVtZ0v5Uz3Mgnz\\\\\\\/cqO06Wrd0CKCVbYKLjLymSvpOXaQiLpjqkFnSpoJgdrS4XF8NrUrUJNqdK44tvuqLOg3WOORI1ThoYJTClNl02fkw6vrEuMy3WpyARzQOL2PTfTPzkEvd+J1jaU9iOtxNOkFoOCUsLeGHS9zYc5Ld2E4wnOZ31LdNru8WDRZMWD70AN0tt0Gn4V96AR3tYESnFpS3YNpXbcwkyabQgsy4LveTj1QCzzM9LpHoowxoBpwYUYddU\\\\\\\/8zxf8B\\\\\\\/GKMLMoPitKqqQf9bBEKSNNglyN1OuzZBZl\\\\\\\/9k\\\\\\\/++aZKha0fdAhFcJUDFA3NuWAdeg03bR0NY4RKyY3zJFN2nS3+GtTB08B\\\\\\\/ME3jPqyQFP6jroajn2xZZidydo89Px6uMKgTGLPzSIUaDAkv59hYzKAz00\\\\\\\/sGF3DgnSQhzuUM\\\\\\\/j24TBbBj+bFXaeGMI4ifhlONjnfdW7z\\\\\\\/IcBPNRFayVF9XafmEw1+t+F0ENwKp\\\\\\\/G5RavxMtwEufTcqB8R4XJf2\\\\\\\/nFEPQfxR\\\\\\\/4fDL+dR0ztLlLL1Hl0auQtvvPm4FS+OP6jRKkLDDFlVEigY0Vzp8pZf3WqFgWKW7E\\\\\\\/INFsuEMIRg95J5bfVvqSn\\\\\\\/T\\\\\\\/fuKgzp83A9wnyUusYiTOSQVOCNH7\\\\\\\/wSzHdWobY5Rts1XXBeDNh2cPSPGbq5RzuzCu3N2Wv0+\\\\\\\/dUsyjxMtQWyHqr\\\\\\\/QiPouOXcCf9wRgLoQNRN6sj68+5RytTfTxZSjLG2Y52L0JZ9lPxaOymgSrN7hvEQigQxhf\\\\\\\/VUAmOeMqLKatQb81dIVE6VkWoZTuFqswILrN6imvhntWM3gxe7EsB4+\\\\\\\/AIroExZNfdPZQYvMyKLyq+VaUoWzuXvcsqufgnkKWR339gJskqoxMFci5kG4CQyJ8AeOn7NphrRQ\\\\\\\/t3WGuE0LlWMd+JvbHh4HaAGUGObEQueq77KdhqVgQ0rcRk+\\\\\\\/YGGs80Jc7k52JRG1OWudjzsQe7PYHdYVAXVmOZjGYc3LOJDyf+CJqNHScBD9mRy1S5DtM41yBjYPWM8NXAG1nwbTG6cj9aB3KAxq4\\\\\\\/k6V53+UA2mFa4Uu2EBzHlctD9Q028SqGfEPKe13xRfrkj2vdj8ikukEn9wavyp7xR4nswh5Pcd7k56ytqwuOBZmdYW7eoJ8V1B1MJ0M2PiCqlL\\\\\\\/8V56PaLEKCOYDtMuLC\\\\\\\/jmCixIeRbOE9EZtuQsWvXUxKVfG+wcVfGgjMb7kXY6mjOu60k9MN+nGMQpE4MYeqZNOwkVVJ6uGHRgVDQCGxvDsP0FCYLeD5CyZIT3bJXNs1SOD0oYhiuCZ2UAs5BFers0YnswRilkOAdvny6tAHYMuH87VOsJrE1m9BgTe7ILwcFp28K3DSEMPmO2OjJDVR8Dmt1Y\\\\\\\/WqIg\\\\\\\/J5Bq7e3dEiLr3+HBe\\\\\\\/AGgdjD+5O+f9fFVbsMJjxFSBlnmZKnZI8vcf6Jpg9i0J\\\\\\\/xfAoyDVvMG1XWv8t0IcEyPVlqUCkMG09ouWch2z8IIMKJ9MsNFV5vMR\\\\\\\/vq0YK6S+yrdUKjlSYsf2SxIMFN287Z+6LbXW0asvzF9Sn7puVZy+qW5S+OOUQ7nZuNAhIB5VcuNmuFr8IO7Aa0KXHPU+3K\\\\\\\/KnJSOmo8UN6\\\\\\\/EOrDRTluMAhuqqH56dFNHfuohoonEh+2xYZhyGEZGU25dyPbjrgVGKZGD\\\\\\\/GBsCBaOrlNLEvbQkI97zonNoIY0pLijavaIlsh9wkhGkq52N4B83UVpJ9sWI7m8vHl3w+GZqNA\\\\\\\/LPySsaErrGPu8rCL4T2RXpFDXgDhjKuSnEBNuELj9QCrVVaLvUqSSRT3GByMnvFaF3hBGo5cO3gpRbRbrFCtx4O+PVdeM9UTCGaFSzoHdw\\\\\\\/UiQuIiXB5Pwg\\\\\\\/7pMzGeerY\\\\\\\/Am3iJYMGKIXjX3gcX9Zji4ryMwCgIqW1L3JkKMT6vuuGMdUrG2MjAdc\\\\\\\/uObAkD1IaoeNsh27iBB2Wp\\\\\\\/gskhYw16mxInuH0TaHxs+WLtE8lIc+CPDr4FfD5uP92HMCDiNmxlwSXNEmG+h9wYdIbqzsP+KGJk5xYTywtKAjmIdiD1s1KZe6dyE5BlZ4g2zBmfuWj\\\\\\\/pJ96krGq307jRjNoqSi89yw+jHAcYQMKQHA2ajY7yTXcEqn+3JF6ZrSBKAZNZrdm7iAOBatL8BxHvsHCFjE3TeXHCS41IjNQqVb2l5duj1cq0FHS\\\\\\\/qIJYqndcdQYYq0WvPSVEilzr4BvI0FkqVWB7j8bhhA\\\\\\\/nKVY2B8B11M9h6d0ZLmdzPekGGOHOtAk4LKlT1hnegvh7Dl9wdg9vrUxDPHNIqlPBdC5fOQM6zbV4LaM7tmoo+bqK4NrN6dBcyjllKIabOGZgRh4dqx\\\\\\\/2xyaDCQIWwNYPAr2NDGubErZGp43+Pflo3hjF60TZnbbkfvrRGs4mBHP71vO4gNsVhubtKYCc8nbOsZI5kAa10fsuxaODBuBcH611AYSnoyoTrtD2A\\\\\\\/+BDidqLw8LRflaoC2uBMvI+wJmDqGzXM8q0hVp7eKbokUCxW82dsR+PkCS82c03rHLEnSn09mTuQwblCDWN1aqNwWtBLsodtSdVwm6LnkjepDV3ojLCT63gU1F+GWu18O0agU3R7ckF8Q+mcqR7cq2xEKXbUKH05NSw4Ckm73Qo4OvSPycbJEm\\\\\\\/0de+hnQ+2eVjnuJxs9clv+yOpGjkARrkxIskihNsB\\\\\\\/KXQ0ZMD6vNdK09v1+rDgGNwHh0s6XrZXFKMZLLCGSvf7tRTLb65ue9FB+T2Q6MAaPkJz\\\\\\\/K+uMSE3pQE37s9+GdK69FBRvX4Syaj19k4HkIxEadN09am168gaib6V+q2KHSnbho+Cb2tBeEwZGjamJlwzWwkHm18pq6kxkWKbOOVsS96BIV8GAK70sgDA2HJu+RsnuCqTZ\\\\\\\/671DwgMLzVktzP8sL026kw0l8zScZiiyAdzDntnGBw9CjEISpKf2YPqVvrWWkHnL8mMz06kgJrBF8EFjXfFI4iCTCFjXZ6TVLnVUxWrN0jK8B6Pd9X2tR3G7S\\\\\\\/BXllEwK\\\\\\\/nkl30ppOgzfACXnIF1V2WomvTbVaDoMLS1AAZijNB539q3b+xTdWgYN0kaCdKEB92LKL9ZnK+MkEIAOL3PlxjAtEAsd2RuRIGHnAeaYZ6f+Ahi2yW8Icja33KGP0CddAPLniANADRTBMRkwVz6Yc1Z9rzxAzs0NXYOSCNPX41uUcJ9AlRkN4BSxIEjKvHobvftp+cFfuLsBH4yT4mF4GhDOWQrF0cDTgsf9lE54FGrfng2QbUmGQ2B+lQhcmDfytFGWMNNsIWOHM7AyAQV7I+2mDn8ADF2kQ5W0tmp\\\\\\\/9oHRumDiDjtoIcZ1l6mFeLMQcJTiEACP0+jpAl2wZ5YhQfnX8unO4ehZoenp+2VzsYGwFCJj0gKWmn0THHeEXUi3UsoFj6AKbhesXMS70o\\\\\\\/VSqGRZcivbb24mc8MO99KtASTDmO6jD2N8z+uHP5i2jOmzV9+jRHreJYiE+Uu31QYsyCxOpN\\\\\\\/CzZ5eG\\\\\\\/6tRLWO7soS1biqj4BF8hhxudKiOt3BxD2r7fMGLhh8APOP79EPICUc8eg\\\\\\\/1nKm+4sGTqpr3K\\\\\\\/syXuwN6Fxc53Mg+xIU9yCVQszcYAuOG1X+vzKdMODw5LXF2wHitx7VYZ16W4\\\\\\\/ipvaQqunYuIo4m+uhV+DxYWiQzxbT\\\\\\\/WXGO5uSENFI2K9rYpEw48ScT95MJ4\\\\\\\/TTNAbGB45SnmkXdbyjpQjW8vnfOuWs4bo1VombRySPmkZ47aM\\\\\\\/1Ai2Mu5z\\\\\\\/RXfRlc+iHw5i+1EFuyKJD4aZroU\\\\\\\/Pd+Y0p45w0ueiAO+6aYHydCe7a6Qhpf7BKnqn4P39NWcR7Hd4wV4veAsG6Evz7Bo7JIjymYnBmD5NDQ0vgJsKOwwKswrUif5aH0A9h8+O1l09ugef05afEc5XQZqzE5o5O+roukZyfJ6+oZoni+0tdDCTAuSzqCgDC9qMffyn0cuTdoWsCnWz5BUqrxpzLFI1YCyZt+oW9LL0k69Hbtko7t\\\\\\\/lCj0HQlhzCMOmbkUb9ZXD+YtWlGJZJMcOw7MRgjvlnqktlnW6MQw9J1zcWNQD59qGhnrr9iXk\\\\\\\/MiEIyIE3AyW4VfSy1lNiLiYHrFe2FY6PkmWxafctj4PPxiyGkdhOeAQgSjQjT5aqEJQXk+VrxCaArIJFM2otDYrHn8VQ\\\\\\\/gmbpAFDbSFegZxLxyP2KXampwrxNvguNY7FMMXExmvn2OpJNPBMXUn1RtK6MhRI4VJpCbdec+pIjjt58Ql3KlfioCvoGTmM1MCwY\\\\\\\/jN5ZA8vh4F\\\\\\\/AnuduEc6fDjsK26Y05xYp6gpTI+7hhHI4Il+L1dT+G9FM8LZr22llGzV0KA4XZo0dOJI7j1\\\\\\\/1wjP8VATuOPQzXwsagfGRaRFSYKIlkvdntL3Uvsak0FQ6yzHOCs6QDxfZxx5FBPHPoBbaqJLjlTrp0n9FEf7D8UmerNWlvcQrI7ould+RABms3bjJmycr37zmQ\\\\\\\/vsphkz3x8rd6Hw4jZE7iWxRghSO5XH4A5nohR9KWt3nnNii8\\\\\\\/sE0tgE\\\\\\\/4+8NqNCne6yDqQbTGKjLO18dYYNIVGhN0KxpfeK8JrHYnc37FiZEUuVstF+Y21OfNj19s\\\\\\\/e5wawMwKJUkbHZ1QRf5lFQTMPNQM7E6XVtlB2R9TRSgh4h5bLt3gAHg20gCuqQLF5MIUDz2o8EKjG4lHUo0IthV+zceAgZohMkaUUiZpFG1MntzAkSW+\\\\\\\/WaN0lkmbUy4Re2iQ\\\\\\\/26VWhqdZzCEAFm+6XTFZSk\\\\\\\/7GR4GneddRnNlCfOZJDA3RGUToprhxu1eNneMow3yz30DJ4QlGn9awgB2j7i9mMA8oox2KCwBzkroxXCEg83zQTwY\\\\\\\/LwWdQ3J1T5ROXywaM1B23VuDwm6CzolAaZxni\\\\\\\/jUXYoBmNU3qqlxGQOdzC+x5E9Bt733o\\\\\\\/s8YmqDM3gopkQYj\\\\\\\/njo4\\\\\\\/94ye4dFNnGFUn4aqOG7uvog4Yxcy0srF7ZfkdOUily6iaxQRt7yIX8Kt9p3If9nYgfkBpPaMmeWTLBkAziVkToIqV4Rz\\\\\\\/jRUgEYjo4HjxFQI21as7\\\\\\\/W8+FbLtry5Kz7uq2v4VWbHz+9iSPtyFJVZwlgCVV0z41u0QPUilLVS2LUg07MSHzOz2pdCNonJDt+YmCnk8aoS45oAtd2wO2LdLrSDx20kCHBVnT9F4CZ9s5N7by8tbrTFF7aiumV0Uaqk145YyM6m4jtSO8oQnahNAHRlUnkiSH5WX8miuALjWRk+Oo1TQa46z2lIh9R4O0jy\\\\\\\/vbqW2hk0hjz6ZTclZXC+bVERLuuidLIzcn8IUFS+XyeVmGr1TfBSm16wlk4L3+ScUGxhj0DVLzcdtlSsuTlOLUBhL\\\\\\\/GbKCzTNVxEUr5QTvOP\\\\\\\/A\\\\\\\/vg7if4OGd7iKLxC0SO\\\\\\\/ovncv9p2T2UkAbYZD1p3fZ3wNkNf\\\\\\\/6jdztJCFo4XOqVwga5Xdsrygf34uFhFcSbzKHTxwVbr4\\\\\\\/cL++YTuO3tBPLQ0AzbOop2vITk2fhlQD\\\\\\\/yxV0QOWR69B+IrdXrd2o6lF1GqHUkSgQ9VNVsQvvkU8Tkm08ZeHL2mfQ\\\\\\\/H+tTZksc7vdivdgi1AvGLNp4Bz4v\\\\\\\/k80f2\\\\\\\/DgurJcw0P9ssvrGORRw1wMxQplF3FBHVIGBY1U\\\\\\\/aPP0PvSt4MMPx6P\\\\\\\/Pjw5CoL5eIa0+2\\\\\\\/4fxCaz2MDlH3k3679WuIMAnVwhR3sAkApxyxIAnRG631BrImywhFr5e78nlwLTNgHxqlLP5ZeIrTTHQ7Z558\\\\\\\/bYLOSleBxmi03uvP3cBVrzyGSikuEJF4yHWzeOGRMWdEqwUdBSN65AcPDfwkXcoEFX8sE1L1N1PIM0tEXuPFUjb7bJvVXiYYJXnRP02JY\\\\\\\/YBA\\\\\\\/UPZ5cbgO4CrOoKg269p9Y3NIwel1xXW0JjhTKXomzVM4xMY5L2M6XJsHzEoLlTkomszkKniDAWtxC7L6MxXaUZoA9aaNxf4MeAoVw5NurlxLEdijqUOY22tu0C5IDya6+wkuMpdmzdZyjm\\\\\\\/YapSx2jE7Qjf6XC1\\\\\\\/R5N\\\\\\\/tikHVMfr3ZBRDNyHlyM\\\\\\\/W3kZeS8w+G7W1ZB2ST1R1yLpxA\\\\\\\/FkOu+UaJsrZ\\\\\\\/5aKgRlXwEMXTEnxSEBWPLlhYVnrMbkUpMbuaySx0emQcJYxYRseWjhqv02hKSd7q6gIdQuW\\\\\\\/qcw3DunTTcfLzlPpfpipsD9wIUj3L23B+7psDT\\\\\\\/p3bB\\\\\\\/PxI20ODn\\\\\\\/s9rIp+TeHjB0Jct7iFjYP0xYtkMH\\\\\\\/pNf3AvmApJoNQJNCNBgSeZbiRKLqoHq+vgywcGo3TBwouxfo4j9txS9rw0TX5ufhkJ6WUMPZO0+qxTNVzE02T4DjG0ZZzbTWlQXTGiFZTKvK\\\\\\\/Ma4BEPelpMpVVauoVXPQ2hCJdfgz\\\\\\\/nsTI4q2oftjSO9xQlWA\\\\\\\/91OigE88c47P+YdTtmFMBW5NK8lM2ThMLJVo\\\\\\\/HAU6AFMYzcXVsb\\\\\\\/phYx1FSbZnsVkSG7CI29XUtNcU5vC8DeKUza+FGcj4ZywegAeOV6nXjo4rs\\\\\\\/gQ6u\\\\\\\/fl1qoURTU9W1s62\\\\\\\/rpfIrrmP7T5q+pUrBWCXuZ1NLo5K9\\\\\\\/SO25o0wqc5SAPNqafi7owi6V7pAMM6JN0\\\\\\\/7xJMMU6J5+c1MHO4XvmdR0u6YYWriSzb5IxrNe36S+itrRkyl30cQYS7NZCeJ30xGaZI3yPJNgfiUlU9+R0UodrdXu1LK1A6RPwFPk0yR9jDQjjoAUJHR7NjQG5qOkzxFJtGB8CNOIWXutFeuNrFVGXkjmzlGnrEDNBpwcCSOYDdllyK55JTjrn\\\\\\\/x3Eb+z\\\\\\\/uFI6xfvchYLSzBVEpLpG\\\\\\\/mHQXtuv958APxMzavwKxE9dnRGeu\\\\\\\/4eWgLZB8WXlhFU15fC7ezPbI5XjAazrO7dIoozfnmnodmT9OwQc8MAsjD\\\\\\\/Wxmra\\\\\\\/M9vmhXluTFkHytm2BPBUSJsvu+Qzijm208hIBMNQX99p88dYXlFeZFyPAK1Y24Ip+H0lxM0T57WEjn7dgGInWDTtfpA5Sseo9jS0vZz6\\\\\\\/hWTSiAxgOK9fXgND3MHYUa1\\\\\\\/ndWzJOJVFGwgxAB+nd4ZuSSqCVi0h6e1yFT8P9qkmMn\\\\\\\/WhsEBmJGgRcw0kwBY2tDeSaniRAHjblCxR+IIdRtBwFUiHMfdzoK06hSxXhk9OljxnDM73\\\\\\\/HRFj9B2PcZeTlCUUENwbAOnU52jGqEyzFJLe6Nifswmp\\\\\\\/N4u\\\\\\\/njLG0DEgxfHE5wZUwx9\\\\\\\/elseNeIRZhWRIUObBzqTB04K92lSpJlcSymBDQ0Z+joI7w3XkcwxxnRLFh+j\\\\\\\/C5Q5dxhDh7iFiRWU956YcTRfbee5MNtSbj7Gey+ftyVYoOHaLtMj89YZh\\\\\\\/FOCxd17yskQm7DUusZ90RZ0CXaxQwOshA0m81RAR7nlzPzzMZj2L7MuIjx3MueBhjhr2loQx6WyDB7vF0fjhIrx\\\\\\\/\\\\\\\/fGsW0mP4Xunv7+EJM7jH\\\\\\\/\\\\\\\/py29\\\\\\\/vpeNhIYKB+SO79OELSgU8rNiwP1fiQgLCewiDmGW+CmXaQ2oR\\\\\\\/BBBB+3eH8ncQN4cEMxhZ\\\\\\\/RdDpI69uW91eqfmojY3AOKc8AIwn+J1ezr7pIfW2JIiV\\\\\\\/cCt+UrJx5b7hpck9vfxoBNfuFYkw6KsHuU58cuwsVQjeMR5v+PBdrlNeFkjpgkBdjowDtx9WzwPYoikQR\\\\\\\/CsqAX6+zpYwuenrTtsPMZHpV\\\\\\\/cLjt1850JZEbuMBUq9bkuco++lrO+g7tlSXPGMLRc0z70TAqFxp6WePsVAt\\\\\\\/bhsdqeyYO7Oo3n0lKz\\\\\\\/ajoAWjtSu9pSCy287lWIjY1TR3+tJrlJjoW2TEqcB8TeLqhSNrIG\\\\\\\/RYzpa5rb1j2mkrek0ibKvt0xGUehLd08ZazxnUV3EdtEe4dfrqX\\\\\\\/8ThfeacGhXvG1O4wMmyDhxbWUcBzMxyq8b7UQX\\\\\\\/xrDS4Hs+zim6vf9WQN1PvwziOlv0HErsXrElaiILSnYCtbrOKBMRGv6ItyeeSJQVgbAxQrYOfIeUIEunfPH4evHGujlN2TD3LPM7oyvHN2i11T0PG43l4GP4+bzIRTwyG8GAfqIi6Jf4Jkm9kHE4BXZZNthPEREVvfWXPHQk8bhleeKADB2a2adWGtyNBk1ejITCRn+YPShxcoa081JhR2zit7nAIRFPinO6GH\\\\\\\/xf22fby+oqWesJeywgckr1wwhePzjh7p5KDdmGfR3KHbma1FHhGzlvC38tyiTvt6z1QQUlx7P485N+wTvrrsVWuejjldjvAS6e9XlNo+plZbBtz3JZ3oGDJ0\\\\\\\/vuv5Esquk5bxP4H9NAs6CxD3o2JpkaPXgH+i6XVU7nYW2AcKKXDc99+x17i7D8rWIeeAh4PxYmZV0MytLn802EMPdDQlnfsNPRXC1Pzw5GRi6Wt3SQ5+MmwR\\\\\\\/sp8zktRI4ladPhnZ+Ji0wT9chXwvnMVbq0r5fYvKMsc6f8HOWQQMtlVUR15fS6rXo6+MVMFscIj7S6B\\\\\\\/LKGiFU+dv1Urk4v4eJj5RjGDuBroj8rPf45KDwIVl0sz\\\\\\\/g8q65+O65eXPq+z2yCgKES45MuhiTKYSuSm1kccMbj1u+MWPv8GZyfebRc0T8Im+vZLH2A7tZvV7entK9Bja5wBJyHcEao3khlY9DAXId7iZ4fWdT1J8UWSu8LKF1TbxSExS4x7vEnf0ecF\\\\\\\/v7aSvEfcvgSFXql++Gb\\\\\\\/x\\\\\\\/qyZu1FvcvUQZRVGLlnUSy3qnhVqTdAccL7X16KvhR0pwd8Yvd9uTPL+34bIn5aDIi2Iu8XgldWiZGv5wUhdM9Xpa7q95UwImz8CWncr5UMNhk7zyhnHa6gieMruyR7+gayxG3xnR7i0LUmqOIeznGKxpHTye\\\\\\\/dGNNxA16IaDMAplLsGs+HQgqxRpDiPYAHiNbp4hmRHJOkT1RtzlMRr2PGYqLzABiQlWwwCFenWiY2D9Tv6brOvVmjquBL6S1KC5KeATcpFSV190Ahkp5xfv8BQmyOPtWyEgcRuBMZ0BpIqYpmkRvf4hOco+JPocCc9FClq1U09ScGbVvaoPx0EqhjoqFXjrAL6r5BeJZmTU+bhhD603E4NGnOfru04UomJWxlLvu1DhsNhFaPiKNiOJhGgDq1SUrT40o8BRZed2QxtjV4fPfd2BDQQfNGTaaAg7bbwK7a8MIe7MTrNgpKrQN0SbKCogissPLNH9ulzT2ymREgX3RabODY00+plckcLXtJsqNaE2ahViJsYx5k7e83LwBfK4u6d1YfdwU5hU7ZTaGQaUM4YyQfjPgrnYEoE5cFsog4L3qzRCY7ppn+dCAHOlU7N8aUtk6Qwy\\\\\\\/YsnO\\\\\\\/94mR2RrnbwuC3XLtB3G8X3SJYqIZpEn8oGpF4fZmht5Q9klF\\\\\\\/mOMdgm8sLhG5A4e8DqVSUXoO0KBzE7yeUNM\\\\\\\/Rx9L\\\\\\\/\\\\\\\/gKs9ylV1TPtbQC9T1WSAyX7Z5N26KOdrDlskrVZELho4u3z0bL738mRd5euuR6Snw0hpyLzPm7LUQYvd4EdOB5n65A9ZO4xQAIZuuCmAxurbHDIbCp5Rgo4SLoZ25G\\\\\\\/+xqfmm6HyL0plOIBnokoNUZWa6syTwiA4voYvyCT2P\\\\\\\/zz0qYuBpf8hiF1oQWjEqgK4pEmch7z4fGUffY\\\\\\\/0E76aW3sT4\\\\\\\/iuuYIIMSRR837KGguIhLiSkp8zfTA1iv33mqcbxOdc\\\\\\\/upt6UL5TF1BFKqBAGEH1dvxjcjF7ShIWgivhYxWR97rCJWl5l+cXnnBRtJjTjm+U8mY+F1DDtU27i\\\\\\\/RJyuNPrL5\\\\\\\/2Z7cU7lpHPu0WHnIvmNPj5KMRzQxgXBhpYZdW8ewnuImvQp9EgPJZZnKb7Uqu5RF\\\\\\\/lFCHSNvQQ0o3R4fx1mKHC0zlv67A1uiY0HzCsV3G29erA6eK0\\\\\\\/qytyyXdAAkiGLFJb+1YjKVEPgnNVsy1SdI1K9LgLPoDq0MaE7CIx5mCQLQLCdbDvo34YK2matBQ5N+MflLFxHLa2irkuTCFD6kSLjt65M5O18wym6YbSj108Fq\\\\\\\/Qd9WPgAXKyEA0kj6Bnqj4eNpt00nXMyHEhn7wXTMBa5xJqB1Npxs8XLaf0NdcFl7HsZbjW+\\\\\\\/L8q8ELYySFtxdlGyYST2bqK7PbFZL3PE7CQfX3eYEo0I9sk1GS62M+wHxzqV2EcvTauwOMNcGrgLN0eKDMDQmjJN\\\\\\\/mW3FPto\\\\\\\/qPPiXhfjVmeAB7mooDRQ+g85tkXKopKkT17PPUBv8YZWHOuKplNrl4JDzCAP+koGvLhQarCdjK05EsFvkhMZEokDlq\\\\\\\/5AQEwORQSOnHQ7Xb0MniD6GGa7IeNs6BWI0QvHvgk0R9AmwuLpoHST1yqBQxcyXIrNxUU4vVj1iv8nXLVeJJAxu3FjqMnZ6trJss7xxeH8ZvpTXEauv81RoMDEec323ThyJXKUgRdmQnNlYMnNCuMeK01rUwb+txbYRhCnaFU8lQRggfx\\\\\\\/4JE9e0qdbbxXLrAewPMF3+dc\\\\\\\/vO\\\\\\\/J6yWJZAhC7XtpwKQbbJ\\\\\\\/ql\\\\\\\/jSu5smaVt9AY74TKq0xedtAVsVOV67l16YXlg23f1Ac1khbRDCpH8UlrSj+TkAKoIVJGpL4m74JU7A+S8rI5OEldIRZpSb8Cx4Nn9k8y6FkDIaPA1cGUw+mDG2JnYK0zZ5qI5lj1VFGuuJe5gaOskyW\\\\\\\/jVIZWqH+shogRDe8rzhSkWa4wZZcpq7nmDG2nGoWjBC4HEiWGtYJpzk\\\\\\\/6NTc3bEGqKmjohs\\\\\\\/ZjRJruEbMbKGkkyDImk3TI6UAbTp3glACibIXds4nxi6vdMzJeYAl5wbEZWB\\\\\\\/qBd6f\\\\\\\/KNAeS7Y2wK5SNejaybW8W9gj\\\\\\\/NERW6Gz3RCN91qvJDKErICtNZrJdnkIqwb7jMOtjbthf2QdUZ9uO3Bw\\\\\\\/o8K9NzTV2xVH+CLsPX0kWWsy2UTjaKUlp7JT7r1BJ7IUIRTvEyEm+6YpZyEYs2wI7Gs\\\\\\\/j48qhdS8IiKtAXRD\\\\\\\/BSrg0yVr8Y8UdQRzViLROydIAePtDlcSLaNhby9v2ELpRq5myGjqfGwjHlvc8oyIviEniPAVYznKfOrhvfF5puMJ0\\\\\\\/A5P4ipbFbn7pqZ54ta8skrQAFyeNuFGbBvP6sS17o6w4bThziIkfsRZQvgn8X6Ke64USVenlrXw\\\\\\\/Deve2D36dJPijqs50B0IZCNB7mJXKPtepOYa6pyae2coXlnA6dYA8mQWIhvhuh8JHBbCKiUb4JALS8JHeQplAU2sAc4ibNfnmsOAMO+hjRos\\\\\\\/4COYzdkpGUoLt\\\\\\\/993S3L1wLq+YXTJEMlM\\\\\\\/aZfPgHY223+5jffOPgG7vFEBMQOy+vjI+1\\\\\\\/eBeQcvNPFhnEZc47T\\\\\\\/DAwr2412wMSbvcyrNHxI+bDvL2O5gMBe4+KradHOY1j92bCXqV7wgs8Ssx6M5TPkCeUONvwy21wX\\\\\\\/0Z5Qg4Orc1cQwNaHV8MtBmrQRzv340Kg6rDqgt+CAdcYKvW\\\\\\\/hbg74Pdb6MRj92VDDC3YgzfBRoEOB0jWcflo\\\\\\\/QuxZPTEp7PVKwcCbXzqzedkFjoplxdviLUglWaLcFmIOrOk3XupR+hGY3u7IqrWnbJIygEqDjzGo\\\\\\\/5CQ52jp8NHtavXC9zs9tbF8NELvbVoAr981fRgZqvfzK0Iziqv\\\\\\\/FaHidVBtl14x5l+mCZI0P7OKfaWtDTI8NwGLdKeCBbN02rVyh2k3JiHydfvQ7IzVNWO\\\\\\\/8Kd7OIPLrLHl7OO9xSuOpLU9SYHdVG9Jyl3hcUrD4QunOtnKBTTWOkOAkTEtLycaMKfJ0OIrjLA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022NLsyh4LgX9TjI4OU07rv6Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b6503b18c26265c819ec94c7513ea983acd6a395c779e0fed23d1bdc645b6edb\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gpyyIh7fVRDI5pqN6w\\\\\\\/j93E78v2kFs1KfdZPktlEWUtdTkvSLVN9UDIMmcVFVVLIzemTRso3UsgNvZwD\\\\\\\/sxR7HVNQG6SnLRXrcED66XeDzXcmUCtTvjsq68IOa7jXgungVHQq3ujsPiuUMGktWSZGOPvoEML8ZQ0gVx47mYg+4G2jIVAfVDO+a8q9ZIhEc0OBnBHmkjrsfIAnZhCBWws8lYqwU3a1AtU1Ql8XKaJVu6iqVOvoKQh4HCuLE7P+VkTvLyOxku7iWAZdQrtYD17JxAlTsaje+tl5g4ARdPuGqyWnZ03Y252nD4bCYiPgwEwHQUH+Odiypf7MIxRlc3JcDp5EYOm7QLUvIHIxAN+v0nFmfxoi5Vi93A+T2TffqiKAjA6fWviW+A55Mr5ysun3LKGyTpQCpwX\\\\\\\/I3quXRuz+obkBut\\\\\\\/MuyyRDMTy07qWtMQ92oWBVz17yRJUD6dQfVUeaReUM89qxA2TIU3mg\\\\\\\/YUwRwFJsO0rBp5B5y6qfSny\\\\\\\/4lAfkrY0T1u79Z\\\\\\\/5NLPb6Bd2mgwgJD2QU1FPUICSE+cvq4Ii0VaTXrMSEbNC8byjupRGCQYkh5FPeZgQdDBph28IE0yzAqJCk\\\\\\\/v4gpS2tETtk\\\\\\\/04Z9MHWKeJ7FF9tMU0aamvRyR86SSXcPU4XUSnrCIF5r1E287iE0+zcaG42F\\\\\\\/QQWx4v8t4UMei1SpDZdxLXTVXFPSDSt\\\\\\\/FkFUwhuKbdIkrq4rHsVR+rwTq3OTNfgmpy\\\\\\\/ReGPmsl3XT0FDwdRkDagmcABn2BLKJiMyFj2aimYaO4jzULrhaRMcLoREF\\\\\\\/8cpkSyRdhfzUSq78B\\\\\\\/39RHLOQrwiyBBf\\\\\\\/VptlaxanR\\\\\\\/aO7ECcvMqJqoSSelaghBhX0oiZBqaw29Xkyj37xKKbT7KpTligI6cCwoLNNiqTXF6Eek36xkUXK83V9Mv8JFnctxFQHpXys\\\\\\\/Klr9sHFUD1ybiEkuXisJ7\\\\\\\/9a3N8BGihtHrxftht8ZD8VzqaALNynr0s9Fb\\\\\\\/CbzsQkudVWu4xmUoEEWtOH26X4s+wbSHW8ZzefL9teN\\\\\\\/mDIpm6\\\\\\\/1d0ns+ghcFDUdSmdja8\\\\\\\/mrFhklZSbbCjqnnUTZL3EfaQn3wOZNqTJWIWApWyHpP6rdfsJ9B67v+bee2Cck8E7TBE05VYBAyRkadenVrgnrEHiNM2iLC5cx9la9K+BOJbGGLe+5YwJLDovIXoRZTHSrNobNtGAxXD9guWoNPGr38OgC6yuNGpTctTnrtNJBJJDPMW\\\\\\\/0YnqfgTFjVc9R6N0FGIL2tzBRQAJnYJgp09HOfzX0wHrmLypGAwBDixx4QoH8Zkhzm4oOviVZVnN1LcY49d\\\\\\\/ABgndM5F1mB740oLV1it8yCl96qAkVz6n+k78RgRNLnbBdO1SSYfNoM5EvNnqylceVlSwWrzjIWc8xADgZdNW2\\\\\\\/\\\\\\\/tRZMXQTyh4SuU3LgQrXpYwex6RW21lf9shvg0yoL6vA2aMCP0P4jp5Kl3B8+ahmydGS21Y\\\\\\\/BsJxqmdnNnu5a4p1Jw7MLd3TXXi2YDTOgo65k2lKJgwv5jkmilz2sBZQYz5jHd0GSuIl66x834bCx+GKpVNh\\\\\\\/kfy0cjVYrToQ8L1scepho42FZYVMUXxU0nx7Ix121r4IuB+NZhlESj7wbLeH9gT687saMnJvzsZugVPvSElB8MImZHNXA5YymMIgKxp2nCNEircd10eALhu\\\\\\\/9p2mnjq0RBSUsbx0Wfo7rf+eXqSq50MT5JhBhFNS+SAlogmVx6kHPNpV+7D0MlmOez9y2UPDJNmPoKb7xJLwiQUCcb5xtecAYcYWZrYOBMbP83WunSvK2cKcQUgv\\\\\\\/9nnq3IBp4FwLtxsfhBmjBrcMm2+bmJsZ5E7IywZXooam74OV5u8oFR981\\\\\\\/QlOXJ6RLsBbe2L94psRKfhs7vfqk1zAnmd3HfB87KMWzrcJTVsTfNTDdzfllHqi4+0CZPZlVZapiNzOQ1PDist2hW9Dze5DfGVaI3dcaluVnFpg4EbOiKwGA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022y8zrjib+oyVVO01DesYkJg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226205c106942f79af11f8a157a5477b890e6fa460119231e0d63c5aad78d2e75f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459547 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/10/1741" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pIWoYSkhERi8xN8wVEl-0AftLdlASVhVPUdYPOw_6MA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/10/1741" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bsz6ZZtmDfaiffElLfQmSBhSI1pmvBUFPF6utUGT7cg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/10/1727" title="Cytoplasmic Male Sterility Contributes to Hybrid Incompatibility Between Subspecies of Arabidopsis lyrata" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/10/1753" title="Characterization of Two ENU-Induced Mutations Affecting Mouse Skeletal Morphology" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459547" id="node459547" data-pisa="ggg;3/10/1741" data-pisa-master="ggg;g3.113.007377" data-apath="/ggg/3/10/1741.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Global Linkage Map Connects Meiotic Centromere Function to Chromosome Size in Budding Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anastasia Baryshnikova</span>, <span class="highwire-citation-author" data-delta="1">Benjamin VanderSluis</span>, <span class="highwire-citation-author" data-delta="2">Michael Costanzo</span>, <span class="highwire-citation-author" data-delta="3">Chad L. Myers</span>, <span class="highwire-citation-author" data-delta="4">Rita S. Cha</span>, <span class="highwire-citation-author" data-delta="5">Brenda Andrews</span> and <span class="highwire-citation-author" data-delta="6">Charles Boone</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1741-1751;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007377 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459547" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anastasia Baryshnikova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Banting and Best Department of Medical Research, The Donnelly Center for Cellular and Biomolecular Research, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anastasia%2BBaryshnikova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baryshnikova%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnastasia%2BBaryshnikova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Benjamin VanderSluis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Computer Science and Engineering, University of Minnesota, Minneapolis, Minnesota 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BVanderSluis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=VanderSluis%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenjamin%2BVanderSluis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael Costanzo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Banting and Best Department of Medical Research, The Donnelly Center for Cellular and Biomolecular Research, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BCostanzo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Costanzo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BCostanzo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chad L. Myers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Computer Science and Engineering, University of Minnesota, Minneapolis, Minnesota 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chad%2BL.%2BMyers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Myers%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChad%2BL.%2BMyers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Rita S. Cha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Department of Life Sciences, Genome Damage and Stability Centre, University of Sussex, Falmer, BN1 9RQ, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rita%2BS.%2BCha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cha%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARita%2BS.%2BCha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a>
<a href="mailto:rc320@sussex.ac.uk" class="" data-icon-position="" data-hide-link-title="0">rc320@sussex.ac.uk</a>
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brenda Andrews </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Banting and Best Department of Medical Research, The Donnelly Center for Cellular and Biomolecular Research, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brenda%2BAndrews%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andrews%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrenda%2BAndrews%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a>
<a href="mailto:rc320@sussex.ac.uk" class="" data-icon-position="" data-hide-link-title="0">rc320@sussex.ac.uk</a>
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Charles Boone </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Banting and Best Department of Medical Research, The Donnelly Center for Cellular and Biomolecular Research, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BBoone%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boone%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BBoone%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a>
<a href="mailto:rc320@sussex.ac.uk" class="" data-icon-position="" data-hide-link-title="0">rc320@sussex.ac.uk</a>
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/10/1741" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459547" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459547/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/10/1741.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459547" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459547/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/10/1741.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459547" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459547/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/10/1741.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459547" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459547/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/10/1741.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459547" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459547/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Synthetic genetic array (SGA) analysis automates yeast genetics, enabling high-throughput construction of ordered arrays of double mutants. Quantitative colony sizes derived from SGA analysis can be used to measure cellular fitness and score for genetic interactions, such as synthetic lethality. Here we show that SGA colony sizes also can be used to obtain global maps of meiotic recombination because recombination frequency affects double-mutant formation for gene pairs located on the same chromosome and therefore influences the size of the resultant double-mutant colony. We obtained quantitative colony size data for ~1.2 million double mutants located on the same chromosome and constructed a genome-scale genetic linkage map at ~5 kb resolution. We found that our linkage map is reproducible and consistent with previous global studies of meiotic recombination. In particular, we confirmed that the total number of crossovers per chromosome tends to follow a simple linear model that depends on chromosome size. In addition, we observed a previously unappreciated relationship between the size of linkage regions surrounding each centromere and chromosome size, suggesting that crossovers tend to occur farther away from the centromere on larger chromosomes. The pericentric regions of larger chromosomes also appeared to load larger clusters of meiotic cohesin <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006211">Rec8</a>, and acquire fewer <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001014">Spo11</a>-catalyzed DNA double-strand breaks. Given that crossovers too near or too far from centromeres are detrimental to homolog disjunction and increase the incidence of aneuploidy, our data suggest that chromosome size may have a direct role in regulating the fidelity of chromosome segregation during meiosis.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/synthetic-genetic-array-sga" class="hw-term hw-article-keyword hw-article-keyword-synthetic-genetic-array-sga" rel="nofollow">synthetic genetic array (SGA)</a></li><li class="kwd"><a href="/keyword/genomics" class="hw-term hw-article-keyword hw-article-keyword-genomics" rel="nofollow">genomics</a></li><li class="kwd"><a href="/keyword/meiosis" class="hw-term hw-article-keyword hw-article-keyword-meiosis" rel="nofollow">meiosis</a></li><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li><li class="kwd"><a href="/keyword/centromere" class="hw-term hw-article-keyword hw-article-keyword-centromere" rel="nofollow">centromere</a></li><li class="kwd"><a href="/keyword/genetic-linkage" class="hw-term hw-article-keyword hw-article-keyword-genetic-linkage" rel="nofollow">genetic linkage</a></li><li class="kwd"><a href="/keyword/chromosome-size" class="hw-term hw-article-keyword hw-article-keyword-chromosome-size" rel="nofollow">chromosome size</a></li><li class="kwd"><a href="/keyword/double-strand-breaks" class="hw-term hw-article-keyword hw-article-keyword-double-strand-breaks" rel="nofollow">double strand breaks</a></li><li class="kwd"><a href="/keyword/rec8" class="hw-term hw-article-keyword hw-article-keyword-rec8" rel="nofollow">Rec8</a></li><li class="kwd"><a href="/keyword/spo11" class="hw-term hw-article-keyword hw-article-keyword-spo11" rel="nofollow">Spo11</a></li><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/saccharomyces-cerevisiae" class="hw-term hw-article-keyword hw-article-keyword-saccharomyces-cerevisiae" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Saccharomyces cerevisiae</em></a></li></ul><p id="p-5">Synthetic genetic array (SGA) analysis provides an automated method for constructing yeast double mutants and mapping genetic interaction networks (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Costanzo <em>et al.</em> 2010</a>). In a typical SGA experiment, a query strain, carrying a mutation in a gene of interest, is crossed to an input array of viable deletion mutants or conditional alleles of essential genes. Sporulation and a series of selection steps produce a corresponding output array of double mutants, which can be scored for various phenotypes, including fitness, by the use of quantitative colony size measurements.</p><p id="p-6">Single- and double-mutant fitness estimates can be used to identify positive and negative genetic interactions, in which the double mutant grows better or worse than expected from the combined effect of the two single mutations, respectively (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baryshnikova <em>et al.</em> 2010a</a>). However, in addition to fitness, SGA-based colony size also reflects the efficiency at which double mutants are formed. For example, for genetically linked gene pairs, SGA analysis produces fewer double mutant cells than for gene pairs that segregate independently (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Consequently, double mutants involving linked gene pairs tend to form smaller colonies (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, A and B</a>), which must be removed from genetic network analysis to prevent their misinterpretation as negative genetic interactions (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Collins <em>et al.</em> 2006</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Costanzo <em>et al.</em> 2010</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/10/1741/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Construction of genetic linkage map based on SGA analysis. (A) In an SGA experiment, a &#x201C;query&#x201D; strain mutated in a gene of interest (a&#x394;) is crossed to an array of ~5000 viable deletion mutants, including those located on the same chromosome as the query (b&#x394;). Both query and array mutations (empty blue and red squares, respectively) are linked to drug-resistance cassettes, which enable selection of the double-mutant meiotic progeny, after mating and sporulation, and act as markers of meiotic recombination. Physical distance between the loci and local frequency of CO events (dotted crosses) determine the efficiency of double mutant production, which, together with double mutant fitness, defines the size of the double mutant colony formed at the final stage of SGA selection (e.g., Figure 1B). Because genetic interactions are rare and are not expected to be influenced by physical distance between loci, quantitative double-mutant colony sizes can be used to estimate local recombination frequency. (B) Visual illustration of the relationship between physical distance and double-mutant colony size, which reflects recombination frequency. Images of double-mutant colonies corresponding to linked gene pairs were obtained from the SGA experiment involving the yhr030c&#x394; query strain and arranged based on their physical position on the chromosome (yhr015w&#x394;&#x2013;yhr047c&#x394;). Double-mutant colonies of unlinked gene pairs, derived from a control experiment, are indicated for comparison. Estimated fraction of recombinant progeny (Rec. fraction; Materials and Methods) and SGA-GD (Materials and Methods) are reported for each linked double mutant. (C) A typical genetic linkage profile. Black bars correspond to the estimated frequency of recombinant progeny (Materials and Methods) involving the query mutation (yhr030c&#x394;) and an array mutation located on the same chromosome at the position indicated on the x-axis. The red line represents the smoothed SGA-GD between the query and the array loci (Materials and Methods). (D) Relationship between physical distance and recombination frequency estimates, as measured by the fraction of recombinant progeny (black line) or SGA-based genetic distance (red line). The dotted line indicates the 50 SGA-GD threshold, which corresponds to the shortest genetic distance between two unlinked loci and, on average, is equivalent to a physical distance of ~35 kb (Materials and Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-141262518" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Construction of genetic linkage map based on SGA analysis. (A) In an SGA experiment, a &#x201C;query&#x201D; strain mutated in a gene of interest (&lt;em&gt;a&lt;/em&gt;&#x394;) is crossed to an array of ~5000 viable deletion mutants, including those located on the same chromosome as the query (&lt;em&gt;b&lt;/em&gt;&#x394;). Both query and array mutations (empty blue and red squares, respectively) are linked to drug-resistance cassettes, which enable selection of the double-mutant meiotic progeny, after mating and sporulation, and act as markers of meiotic recombination. Physical distance between the loci and local frequency of CO events (dotted crosses) determine the efficiency of double mutant production, which, together with double mutant fitness, defines the size of the double mutant colony formed at the final stage of SGA selection (&lt;em&gt;e.g.&lt;/em&gt;, Figure 1B). Because genetic interactions are rare and are not expected to be influenced by physical distance between loci, quantitative double-mutant colony sizes can be used to estimate local recombination frequency. (B) Visual illustration of the relationship between physical distance and double-mutant colony size, which reflects recombination frequency. Images of double-mutant colonies corresponding to linked gene pairs were obtained from the SGA experiment involving the &lt;em&gt;yhr030c&#x394;&lt;/em&gt; query strain and arranged based on their physical position on the chromosome (&lt;em&gt;yhr015w&#x394;&#x2013;yhr047c&#x394;&lt;/em&gt;). Double-mutant colonies of unlinked gene pairs, derived from a control experiment, are indicated for comparison. Estimated fraction of recombinant progeny (Rec. fraction; &lt;em&gt;Materials and Methods&lt;/em&gt;) and SGA-GD (&lt;em&gt;Materials and Methods&lt;/em&gt;) are reported for each linked double mutant. (C) A typical genetic linkage profile. Black bars correspond to the estimated frequency of recombinant progeny (&lt;em&gt;Materials and Methods&lt;/em&gt;) involving the query mutation (&lt;em&gt;yhr030c&lt;/em&gt;&#x394;) and an array mutation located on the same chromosome at the position indicated on the x-axis. The red line represents the smoothed SGA-GD between the query and the array loci (&lt;em&gt;Materials and Methods&lt;/em&gt;). (D) Relationship between physical distance and recombination frequency estimates, as measured by the fraction of recombinant progeny (black line) or SGA-based genetic distance (red line). The dotted line indicates the 50 SGA-GD threshold, which corresponds to the shortest genetic distance between two unlinked loci and, on average, is equivalent to a physical distance of ~35 kb (&lt;em&gt;Materials and Methods&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/10/1741/F1.medium.gif" width="371" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/10/1741/F1.medium.gif" width="371" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/10/1741/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/10/1741/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463479" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">Construction of genetic linkage map based on SGA analysis. (A) In an SGA experiment, a âqueryâ strain mutated in a gene of interest (<em>a</em>Î) is crossed to an array of ~5000 viable deletion mutants, including those located on the same chromosome as the query (<em>b</em>Î). Both query and array mutations (empty blue and red squares, respectively) are linked to drug-resistance cassettes, which enable selection of the double-mutant meiotic progeny, after mating and sporulation, and act as markers of meiotic recombination. Physical distance between the loci and local frequency of CO events (dotted crosses) determine the efficiency of double mutant production, which, together with double mutant fitness, defines the size of the double mutant colony formed at the final stage of SGA selection (<em>e.g.</em>, <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). Because genetic interactions are rare and are not expected to be influenced by physical distance between loci, quantitative double-mutant colony sizes can be used to estimate local recombination frequency. (B) Visual illustration of the relationship between physical distance and double-mutant colony size, which reflects recombination frequency. Images of double-mutant colonies corresponding to linked gene pairs were obtained from the SGA experiment involving the <em>yhr030cÎ</em> query strain and arranged based on their physical position on the chromosome (<em>yhr015wÎâyhr047cÎ</em>). Double-mutant colonies of unlinked gene pairs, derived from a control experiment, are indicated for comparison. Estimated fraction of recombinant progeny (Rec. fraction; <em>Materials and Methods</em>) and SGA-GD (<em>Materials and Methods</em>) are reported for each linked double mutant. (C) A typical genetic linkage profile. Black bars correspond to the estimated frequency of recombinant progeny (<em>Materials and Methods</em>) involving the query mutation (<em>yhr030c</em>Î) and an array mutation located on the same chromosome at the position indicated on the x-axis. The red line represents the smoothed SGA-GD between the query and the array loci (<em>Materials and Methods</em>). (D) Relationship between physical distance and recombination frequency estimates, as measured by the fraction of recombinant progeny (black line) or SGA-based genetic distance (red line). The dotted line indicates the 50 SGA-GD threshold, which corresponds to the shortest genetic distance between two unlinked loci and, on average, is equivalent to a physical distance of ~35 kb (<em>Materials and Methods</em>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">Here, we show that colony size data associated with linked gene pairs can be used to construct accurate maps of meiotic recombination throughout the yeast genome. Our analysis uncovered a previously unappreciated connection between chromosome size and centromere function during meiosis, revealing that crossovers (COs) tend to occur farther away from the centromere on larger chromosomes, which may influence chromosome segregation and aneuploidy events.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>SGA data</h3><p id="p-9">SGA experiments were conducted as described in <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Baryshnikova <em>et al.</em> (2010a)</a>. Quantitative colony size measurements were acquired and processed using the computational pipeline described in <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baryshnikova <em>et al.</em> (2010b)</a>. The <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/007377SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FileS1.zip">File S1</a>, containing the SGA-based genetic distances (SGA-GD), is available at <a href="http://boonelab.ccbr.utoronto.ca/data/baryshnikova_2013/">http://boonelab.ccbr.utoronto.ca/data/baryshnikova_2013/</a>. A set of 16 files, one for each chromosome, contains the SGA-GD between the indicated loci. Rows and columns correspond to query and array mutants, respectively. Multiple experiments, involving the same query locus, appear as duplicated rows in each file.</p></div><div id="sec-3" class="subsection"><h3>Calculating genetic distances</h3><p id="p-10">The recombination rate between two loci often is expressed in terms of their genetic distance, which is measured in centimorgans (cM) and refers to the average number of CO events occurring between the loci in a single meiosis (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Sturtevant 1913</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Griffiths <em>et al.</em> 2000</a>). Given a population of cells undergoing meiotic division, the genetic distance between two loci can be computed by using the frequency of recombinant progeny relative to the total number of meiotic products (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Griffiths <em>et al.</em> 2000</a>). Assuming complete CO interference, a genetic distance of 50 cM would correspond to a 50% frequency of recombinants and thus would be indicative of genomic loci that segregate independently. It must be noted that the observed fraction of recombinants may differ from the actual fraction of recombinants because multiple COs occurring between two markers often remain undetected. This issue is partially addressed by the Haldane mapping function (see herein).</p><p id="p-11">Using SGA data, we estimated the relative fraction of recombinant progeny for each double mutant as:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1741/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1741/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where <em>f<sub>i</sub></em>, <em>f<sub>j</sub></em>, and <em>f<sub>ij</sub></em> are the normalized colony sizes of the two single and of the double mutant, respectively, calculated as described previously (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Baryshnikova <em>et al.</em> 2010b</a>). We assume that double-mutant colony size can be used as an approximation for the number of double mutants produced by the original meiotic event and hypothesize that, in the absence of genetic linkage, just like in the absence of genetic interactions, the colony size of a double mutant, relative to wild type, should be equal to the product of the two single mutant colony sizes (<em>f<sub>ij</sub></em> = <em>f<sub>i</sub></em> Ã <em>f<sub>j</sub></em>). Any difference between the observed (<em>f<sub>ij</sub></em>) and the expected (<em>f<sub>i</sub></em> Ã <em>f<sub>j</sub></em>) double mutant colony size must be due to either genetic interactions or genetic linkage. Because genetic interactions are rare and do not normally depend on physical distance between loci, the average colony size of double mutants involving genes located in close proximity to one another should only reflect genetic linkage, whereas the contribution of genetic interactions can be considered negligible (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). A similar reasoning pertains to potential differences in sporulation efficiency, which might also affect double-mutant colony size. In addition to being rare and independent from physical positioning, sporulation defects are unlikely to be manifested in heterozygous diploids from which double mutants are produced in SGA. The ratio of observed and expected double-mutant fitness is multiplied by a factor of 0.5, such that R = 0.5 for genetically independent loci (<em>f<sub>ij</sub></em> = <em>f<sub>i</sub></em> Ã <em>f<sub>j</sub></em>).</p><p id="p-12">Recombinant fraction R counts the number of recombinant spores relative to all spores generated by meiosis and can be used as a measure of recombination among genomic loci. However, R often underestimates the true recombination rate because it does not account for recombination events that do not produce double mutants. For example, multiple CO events produce a parental arrangement of alleles and the resulting meiotic products do not contribute to recombinant frequency.</p><p id="p-13">Several normalization methods have been proposed to correct for this inaccuracy (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Morton 2004</a>). One of the simplest methods, known as Haldaneâs mapping function (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hartl and Jones 2009</a>), is based on the assumption that recombination events are distributed randomly within a given region and, thus, the probability of observing <em>i</em> recombination events follows the Poisson distribution:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1741/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1741/embed/mml-math-2.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>where <em>m</em> is the mean number of recombination events in the region per meiosis. Haldaneâs mapping function assumes that multiple CO events occur independently from each other and do not experience interference (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hartl and Jones 2009</a>). Although this is a fairly strong assumption, CO interference is unlikely to have a strong impact on our estimates of recombination frequency based on colony sizes. As a result, we believe that the adoption of the Haldaneâs mapping function is justified.</p><p id="p-14">Recombinants comprise 50% of the spores derived from meioses in which at least one recombination event occurred in the region. Thus, recombinant fraction R can be expressed as:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1741/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1741/embed/mml-math-3.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>where e<sup>â</sup><em><sup>m</sup></em> is the number of meioses with zero recombination events:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1741/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1741/embed/mml-math-4.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span>From <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">Equation 4</a>, we can derive <em>m</em>, which would equal to the probability of observing a recombination event in a given region:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1741/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1741/embed/mml-math-5.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span>Recombination rate between loci is often expressed in terms of their genetic distance, measured in Morgans, such that 1 Morgan equals 1 recombinant product per meiosis. Because each recombination event produces two recombinant products, genetic distance <em>M</em> is derived from <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">Equation 5</a> as:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1741/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1741/embed/mml-math-6.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span>1 centimorgan (cM) equals 0.01 Morgans.</p><p id="p-15">Compared with classical genetic studies based on small scale tetrad analyses (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cherry <em>et al.</em> 2012</a>), genetic distances derived from SGA double-mutant colony sizes seem to systematically overestimate recombination rates. To emphasize this quantitative difference and prevent confusion, we avoid the term âcentimorganâ and describe our data in terms of SGA-GD. A unit of SGA-GD is equivalent to a centimorgan in the context of SGA data and a genetic distance of 50 SGA-GD corresponds to a recombinant frequency of 50% and thus is indicative of genomic loci that segregate independently (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Griffiths <em>et al.</em> 2000</a>) (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1, C and D</a>).</p></div><div id="sec-4" class="subsection"><h3>Constructing genetic linkage maps</h3><p id="p-16">For each SGA experiment, we calculated the genetic distances between the query locus and all array loci located on the same chromosome. Because even inviable double mutants or double mutant failing to germinate form small residual colonies, raw genetic distances were normalized such that the genetic distance between the gene and itself is zero.</p><p id="p-17">Each chromosome was split into a set of consecutive nonoverlapping bins of 5 kb each. Query and array genes were assigned to bins based on their chromosomal position, as reported by the <em>Saccharomyces</em> Genome Database (SGD, <a href="http://www.yeastgenome.org">www.yeastgenome.org</a>; accessed in February 2011). A genetic linkage map G for chromosome C was generated as a n Ã n matrix, where n is the total number of bins in C, and the value G<em><sub>ij</sub></em> corresponds to the average of SGA-based genetic distances for query-array gene pairs where queries belong to bin <em>i</em> and arrays belong to bin <em>j</em>. The consolidated genetic linkage map Gâ² (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS1.pdf">Figure S1</a>) was calculated by averaging query-array and array-query genetic linkage maps for the same positions:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1741/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1741/embed/mml-math-7.gif"/></noscript></span> <span class="disp-formula-label">(7)</span></span>Averaging of query and array genetic linkage maps was applied to all chromosomes, with the exception of chromosome III, because of its peculiar role in the SGA selection process (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS3.pdf">Figure S3</a>). In addition, the left arms of chromosomes V and XIV, harbor two of the markers used during the SGA double mutant selection: <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000789">YEL063C</a></em> (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000789">CAN1</a></em>) and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005212">YNL268W</a></em> (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005212">LYP1</a></em>). As a result, all SGA screens show linkage in the corresponding areas of chromosomes V and XIV and gene-specific linkages cannot be reported (gray areas in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS1.pdf">Figure S1</a>, panels V and XIV).</p></div><div id="sec-5" class="subsection"><h3>Hotspot analysis</h3><p id="p-18">We estimated the recombination frequency at each position x along the chromosome by computing the average SGA-based genetic distance within the interval [x â 2, x + 2], corresponding to a 25-kb interval centered on position x (5 bins Ã 5 kb/bin). The resulting profile of recombination frequency was smoothed using a moving average filter over five consecutive positions.</p><p id="p-19">We identified 347 putative recombination hotspots defined as local maxima of recombination frequency, measured in SGA-GD per kilobase, that were greater than the average genome recombination frequency. The distribution of these peaks coincided well with known genomic loci associated with high levels of meiotic recombination (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lichten and Goldman 1995</a>). For example, we identified 8 of 9 most well-characterized recombination hotspots (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001921">HIS2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000535">HIS4</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001060">ARG4</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000010">CYS3</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001061">DED81</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000644">ARE1</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000642">IMG1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000036">CDC19</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000649">THR4</a></em>; data not shown).</p><p id="p-20">Meiotic recombination in yeast has been investigated on a genome-wide scale with the use of a variety of experimental approaches (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Baudat and Nicolas 1997</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Borde <em>et al.</em> 2004</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Blitzblau <em>et al.</em> 2007</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Buhler <em>et al.</em> 2007</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mancera <em>et al.</em> 2008</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Pan <em>et al.</em> 2011</a>). These studies focused primarily on mapping the initial phases of recombination, including <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001014">Spo11</a> DNA binding (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gerton <em>et al.</em> 2000</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Pan <em>et al.</em> 2011</a>) and DSB formation (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Baudat and Nicolas 1997</a>), as well as early recombination intermediates, such as single-stranded DNA produced by resection (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Blitzblau <em>et al.</em> 2007</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Buhler <em>et al.</em> 2007</a>). In addition, CO and non-CO recombination outcomes have been identified by genotyping single-nucleotide polymorphisms in parents and progeny of sampled meioses (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Mancera <em>et al.</em> 2008</a>).</p><p id="p-21">We compared the recombination profiles obtained from these published studies of meiotic recombination to those derived from our SGA-based genetic linkage analysis. The physical locations of recombination hotspots were downloaded from the supplementary material of each publication, as described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/TableS1.pdf">Table S1</a>.</p><p id="p-22">Our data showed good precision and sensitivity in detecting recombination hotspots reported in other studies (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS4.pdf">Figure S4</a>). For example, 65% and 77% of hotspots reported by <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Pan <em>et al.</em> (2011)</a> and <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Mancera <em>et al.</em> (2008)</a>, respectively, are located within 10 kb from a hotspot identified by our SGA-based dataset (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS4.pdf">Figure S4</a>).</p></div><div id="sec-6" class="subsection"><h3>Relationship between chromosome size and centromere linkage</h3><div id="sec-7" class="subsection"><h4>SGA data:</h4><p id="p-23">All gene pairs located on the same chromosome were divided into <em>cis</em>-pairs (located on the same side of the centromere) and <em>trans</em>-pairs (located on opposite sides on the centromere). Gene pairs in each group were then distributed into discrete 5-kb bins according to their physical distance. Within each bin, genetic distances were averaged. The extent of centromere linkage was computed as the physical distance of smallest (closest to the centromere) bin of <em>trans</em>-pairs where the genetic distance between loci was on average equal or greater than 50 SGA-GD.</p></div><div id="sec-8" class="subsection"><h4>Other studies of meiotic recombination:</h4><p id="p-24">Data relative to the location of recombination hotspots were downloaded from the supplementary websites of the respective publications, as indicated in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/TableS1.pdf">Table S1</a>. Linkage around the centromere was defined as the shortest physical distance between two hotspots located on opposite sides of the centromere. As a control for centromere-unrelated linkage, we calculated the average physical distance between any pair of consecutive hotspots.</p><p id="p-25">In addition, a genetic map was downloaded from the <em>Saccharomyces</em> Genome Database (<a href="http://downloads.yeastgenome.org/chromosomal_feature/genetic_map.tab">http://downloads.yeastgenome.org/chromosomal_feature/genetic_map.tab</a>). Linkage around the centromere was defined as the shortest physical distance between two genetically independent loci (50 cM) located on opposite sides of the centromere. As a control for centromere-unrelated linkage, we calculated the expected physical distance between two unlinked loci by using the average number of centimorgans per kilobase reported for each chromosome at <a href="http://www.yeastgenome.org/pgMaps/pgMap.shtml">http://www.yeastgenome.org/pgMaps/pgMap.shtml</a>.</p><p id="p-26">We computed Pearson correlation coefficients and the associated significance <em>p</em>-values between chromosome size and the estimated extent of centromere-related and centromere-unrelated linkage (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS5.pdf">Figure S5</a>). Given that a number of published studies reported an overall increase in meiotic recombination rates for the four smallest chromosomes (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kaback <em>et al.</em> 1992</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Gerton <em>et al.</em> 2000</a>; <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Blitzblau <em>et al.</em> 2007</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Cherry <em>et al.</em> 2012</a>), we also computed correlations between chromosome size and linkage extent following the removal of chromosomes I, III, VI, and IX. We found that any correlation observed between chromosome size and centromere-unrelated linkage, was completely abolished when the 4 smallest chromosomes were excluded from the analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS5.pdf">Figure S5</a>). In contrast, centromere-related linkage was generally unaffected (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS5.pdf">Figure S5</a>).</p></div></div><div id="sec-9" class="subsection"><h3>Measuring centromere-bound cohesin clusters</h3><p id="p-27">We obtained quantitative profiles of meiotic cohesin binding from two recent genome-wide studies (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Glynn <em>et al.</em> 2004</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kiburz <em>et al.</em> 2005</a>), as well as a third study in which the authors measured cohesin binding along four chromosomes at different time points (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kugou <em>et al.</em> 2009</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/TableS1.pdf">Table S1</a>). Profiles were smoothed using a moving average filter applied across 25 consecutive positions. The extent of centromere-bound cohesin clustering was measured as the width of the centromere-centered peak of cohesin binding. Peak width was defined as the distance between the two farthest chromosomal positions, on either side of the centromere, where cohesin binding was greater than global genomic average (a simple <em>b</em> &gt; <em>GGA</em> cutoff was applied, where <em>b</em> is cohesin binding and <em>GGA</em> is global genomic average).</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Construction of genetic linkage map based on SGA analysis</h3><p id="p-28">In SGA analysis, double-mutant colony size reflects both the efficiency of double mutant formation (<em>i.e.</em>, the fraction of double mutants among the meiotic progeny of a heterozygous diploid) and double-mutant fitness (<em>i.e.</em>, growth rate). While double-mutant fitness defects generally are rare and independent from chromosomal position (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Costanzo <em>et al.</em> 2010</a>), a reduced efficiency of double mutant formation is often observed for gene pairs located closely on the same chromosome (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1, A and B</a>). For linked gene pairs, formation of double mutants depends on meiotic recombination, whose frequency increases with physical distance at a rate that is specific to a given genomic region. Taking advantage of our growing genome-wide SGA dataset, we examined the potential to estimate region-specific recombination rates from the colony sizes of the double mutants carrying genes located close to one another on the same chromosome (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a> and the section <em>Materials and Methods</em>).</p><p id="p-29">From a collective set of ~3800 genome-wide SGA experiments, we obtained quantitative double mutant colony sizes for ~1.2 million gene pairs co-localized on the same chromosome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FileS1.zip">File S1</a>). Colony sizes were measured in pixel units from digital images of double-mutant plates (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1B</a>) and normalized for experimental systematic effects with the use of a computational protocol developed for genetic interaction analysis (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Baryshnikova <em>et al.</em> 2010b</a>). For each gene pair, we used single and double mutantânormalized colony sizes to estimate the relative frequency of recombination progeny for each double mutant (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1C</a> and the section <em>Materials and Methods</em>). Using this frequency, we calculated the genetic distance between the corresponding loci, which is referred to as SGA-based genetic distance, or SGA-GD. A unit of SGA-GD is comparable with a centimorgan, a widely used measure of genetic linkage and recombination frequency. A genetic distance of 50 SGA-GD corresponds to a recombinant frequency of 50% and thus is indicative of genomic loci that segregate independently (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Griffiths <em>et al.</em> 2000</a>) (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1C</a> and the section <em>Materials and Methods</em>).</p><p id="p-30">We found that SGA-based genetic distance correlated linearly with physical distance, a trend that is expected if double-mutant colony size truly reflects recombination frequency (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1D</a>). On average, genes located more than 35 kb apart tended to segregate independently, as their genetic distance was 50 SGA-GD or greater (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1D</a>). In contrast, early genetic tests, based on classical tetrad analysis, estimated that unlinked loci are normally positioned at ~139 kb (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Cherry <em>et al.</em> 2012</a>), a distance ~4-fold greater than our estimate of 35 kb. This difference suggests that SGA data overestimate the frequency of meiotic recombination. One potential explanation for this result is that, in SGA, double-mutant colony sizes are quantified 30â40 generations after the original meiotic event. At that stage, many colonies have reached saturation and small growth disadvantages, experienced by weakly linked gene pairs, might be diminished, thus causing an overestimation of local recombination rates. Despite this limitation, every chromosomal region in our dataset should be equally affected by this phenomenon, which therefore is expected to have a minor impact on the results of our comparative analyses.</p><p id="p-31">We used the set of SGA-based genetic distances between all tested gene pairs to construct a genome-wide, ~5 kb-resolution, genetic linkage map (<em>Materials and Methods</em>). This map revealed large blocks of linkage, where recombination is rare and neighboring loci are inherited jointly (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS1.pdf">Figure S1</a>). Linkage blocks were separated by relatively shorter stretches of loci that appeared to be genetically unlinked and thus likely to harbor recombination hotspots (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS1.pdf">Figure S1</a>; and <em>Materials and Methods</em>). Our genetic linkage data were reproducible (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS2.pdf">Figure S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS3.pdf">Figure S3</a>) and consistent with previous studies of meiotic recombination (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS4.pdf">Figure S4</a>; and the section <em>Materials and Methods</em>). In particular, we confirmed the relationship, previously observed in yeast and in other organisms, between recombination events and chromosome size whereby the total number of COs per chromosome follows a simple linear model <em>a</em>L + <em>b</em>, where L is chromosome size, while <em>a</em> and <em>b</em> are constants (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>) (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Riles <em>et al.</em> 1993</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kaback 1996</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Stahl <em>et al.</em> 2004</a>; <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Mancera <em>et al.</em> 2008</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fledel-Alon <em>et al.</em> 2009</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/10/1741/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pericentric linkage and chromosome size dependent recombination rates in linkage map based on SGA analysis. (A) The genetic linkage map of chromosome VI and part of chromosome XV. The horizontal and the vertical axes represent chromosomal coordinates; a third dimension indicating the SGA-GD between the corresponding chromosomal regions is represented by the intensity of color of each point in the image (Materials and Methods). The two chromosomes are drawn to scale. The green dot labeled as &#x201C;Recombination hotpot&#x201D; indicates the position of the most active DSB hotspot on chromosome VI, as reported previously (Pan et al. 2011). (B) Relationship between total physical and total genetic length of yeast chromosomes. Each data point corresponds to a chromosome. The total estimated genetic length of a chromosome (SGA-GD) was calculated from the physical length (kilobases) and the average SGA-GD/kb ratio of each chromosome. The definition of SGA-GD is described in the section Materials and Methods. The red line and the associated equation correspond to a linear fit to the data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-141262518" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Pericentric linkage and chromosome size dependent recombination rates in linkage map based on SGA analysis. (A) The genetic linkage map of chromosome VI and part of chromosome XV. The horizontal and the vertical axes represent chromosomal coordinates; a third dimension indicating the SGA-GD between the corresponding chromosomal regions is represented by the intensity of color of each point in the image (&lt;em&gt;Materials and Methods&lt;/em&gt;). The two chromosomes are drawn to scale. The green dot labeled as &#x201C;Recombination hotpot&#x201D; indicates the position of the most active DSB hotspot on chromosome VI, as reported previously (Pan &lt;em&gt;et al.&lt;/em&gt; 2011). (B) Relationship between total physical and total genetic length of yeast chromosomes. Each data point corresponds to a chromosome. The total estimated genetic length of a chromosome (SGA-GD) was calculated from the physical length (kilobases) and the average SGA-GD/kb ratio of each chromosome. The definition of SGA-GD is described in the section &lt;em&gt;Materials and Methods&lt;/em&gt;. The red line and the associated equation correspond to a linear fit to the data.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/10/1741/F2.medium.gif" width="440" height="404"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/10/1741/F2.medium.gif" width="440" height="404"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/10/1741/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/10/1741/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463340" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Pericentric linkage and chromosome size dependent recombination rates in linkage map based on SGA analysis. (A) The genetic linkage map of chromosome VI and part of chromosome XV. The horizontal and the vertical axes represent chromosomal coordinates; a third dimension indicating the SGA-GD between the corresponding chromosomal regions is represented by the intensity of color of each point in the image (<em>Materials and Methods</em>). The two chromosomes are drawn to scale. The green dot labeled as âRecombination hotpotâ indicates the position of the most active DSB hotspot on chromosome VI, as reported previously (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Pan <em>et al.</em> 2011</a>). (B) Relationship between total physical and total genetic length of yeast chromosomes. Each data point corresponds to a chromosome. The total estimated genetic length of a chromosome (SGA-GD) was calculated from the physical length (kilobases) and the average SGA-GD/kb ratio of each chromosome. The definition of SGA-GD is described in the section <em>Materials and Methods</em>. The red line and the associated equation correspond to a linear fit to the data.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Positive correlation between chromosome size and the size of pericentric linkage regions</h3><p id="p-33">In our genetic linkage map, the centromeres of all chromosomes were associated with extensive regions of genetic linkage (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS1.pdf">Figure S1</a>), consistent with centromeric suppression of meiotic recombination (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Brar and Amon 2008</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2008</a>; <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Mancera <em>et al.</em> 2008</a>; <a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Pan <em>et al.</em> 2011</a>). We examined pericentric linkage for each chromosome by measuring the shortest physical distance between two unlinked loci positioned on the same side (<em>cis</em>) or on opposite sides (<em>trans</em>) of the centromere (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A</a>âC, and the section <em>Materials and Methods</em>). The shortest distance between <em>cis</em>-pairs of unlinked loci generally was constant for all chromosomes and matched the global genomic average of 35 kb, with the exception of the four smallest chromosomes (I, III, VI, and IX), whose average recombination rate is greater, as documented previously (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Kaback <em>et al.</em> 1992</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">1999</a>; <a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">Pan <em>et al.</em> 2011</a>; <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Cherry <em>et al.</em> 2012</a>) and consistently with the <em>a</em>L + <em>b</em> model. In contrast, the shortest distance between <em>trans</em>-centromeric pairs of unlinked loci was notably greater than 35 kb for every chromosome (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, B and C</a>), consistent with pericentric suppression of meiotic recombination (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Brar and Amon 2008</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2008</a>; <a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">Mancera <em>et al.</em> 2008</a>; <a id="xref-ref-37-7" class="xref-bibr" href="#ref-37">Pan <em>et al.</em> 2011</a>). Furthermore, the shortest distance between <em>trans</em>-centromeric pairs varied from chromosome to chromosome and correlated significantly with chromosome size (<em>R</em> = 0.77, <em>p</em> &lt; 0.01; <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). This correlation indicates that, on larger chromosomes, COs tend to occur farther away from the centromere, resulting in the formation of larger linkage regions. We confirmed the relationship between chromosome size and pericentric linkage using several published datasets (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS5.pdf">Figure S5</a>), including large-scale mapping of individual CO events (<a id="xref-ref-34-7" class="xref-bibr" href="#ref-34">Mancera <em>et al.</em> 2008</a>) and the classical yeast genetic map compiled from traditional tetrad analysis (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Cherry <em>et al.</em> 2012</a>) (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3D</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/10/1741/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between chromosome size and meiotic recombination around centromeres. (A&#x2013;B) Distribution of estimated genetic distances for gene pairs located at a range of physical distances. Gene pairs located on the same chromosome (ordered by chromosome size and listed on the y-axis) were grouped according to their physical distance (x-axis). Within each group, genetic distances were averaged and visualized as a heatmap, where brighter colors correspond to shorter genetic distances and stronger linkage. Gene pairs located on the same side of the centromere (cis; A) or on opposite sides of the centromere (trans; B) were analyzed separately. Data for gene pairs spanning the centromere on chromosome V were not available due to the experimental constraints of the SGA method (Materials and Methods). The dotted line indicates the average genetic distance (35 kb) of unlinked loci (50 SGA-GD) (Figure 1D and the section Materials and Methods). (C) Relationship between chromosome size and the shortest physical distance between two unlinked loci, positioned on the same (cis) or on opposite sides (trans) of the centromere. Each data point corresponds to a chromosome. Data for trans-centromere gene pairs on chromosome V were not available due to the experimental constraints of the SGA method (Materials and Methods). R corresponds to Pearson correlation coefficient. (D) The shortest physical distance between two unlinked loci, positioned on opposite sides of the centromere, was calculated for each chromosome using the genetic map downloaded from the Saccharomyces Genome Database (Cherry et al. 2012), and plotted against chromosome length. R corresponds to Pearson correlation coefficient." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-141262518" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship between chromosome size and meiotic recombination around centromeres. (A&#x2013;B) Distribution of estimated genetic distances for gene pairs located at a range of physical distances. Gene pairs located on the same chromosome (ordered by chromosome size and listed on the y-axis) were grouped according to their physical distance (x-axis). Within each group, genetic distances were averaged and visualized as a heatmap, where brighter colors correspond to shorter genetic distances and stronger linkage. Gene pairs located on the same side of the centromere (&lt;em&gt;cis&lt;/em&gt;; &lt;strong&gt;A&lt;/strong&gt;) or on opposite sides of the centromere (&lt;em&gt;trans&lt;/em&gt;; B) were analyzed separately. Data for gene pairs spanning the centromere on chromosome V were not available due to the experimental constraints of the SGA method (&lt;em&gt;Materials and Methods&lt;/em&gt;). The dotted line indicates the average genetic distance (35 kb) of unlinked loci (50 SGA-GD) (Figure 1D and the section &lt;em&gt;Materials and Methods&lt;/em&gt;). (C) Relationship between chromosome size and the shortest physical distance between two unlinked loci, positioned on the same (&lt;em&gt;cis&lt;/em&gt;) or on opposite sides (&lt;em&gt;trans&lt;/em&gt;) of the centromere. Each data point corresponds to a chromosome. Data for &lt;em&gt;trans&lt;/em&gt;-centromere gene pairs on chromosome V were not available due to the experimental constraints of the SGA method (&lt;em&gt;Materials and Methods&lt;/em&gt;). &lt;em&gt;R&lt;/em&gt; corresponds to Pearson correlation coefficient. (D) The shortest physical distance between two unlinked loci, positioned on opposite sides of the centromere, was calculated for each chromosome using the genetic map downloaded from the &lt;em&gt;Saccharomyces&lt;/em&gt; Genome Database (Cherry &lt;em&gt;et al.&lt;/em&gt; 2012), and plotted against chromosome length. &lt;em&gt;R&lt;/em&gt; corresponds to Pearson correlation coefficient.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/10/1741/F3.medium.gif" width="369" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/10/1741/F3.medium.gif" width="369" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/10/1741/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/10/1741/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463300" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-34" class="first-child">Relationship between chromosome size and meiotic recombination around centromeres. (AâB) Distribution of estimated genetic distances for gene pairs located at a range of physical distances. Gene pairs located on the same chromosome (ordered by chromosome size and listed on the y-axis) were grouped according to their physical distance (x-axis). Within each group, genetic distances were averaged and visualized as a heatmap, where brighter colors correspond to shorter genetic distances and stronger linkage. Gene pairs located on the same side of the centromere (<em>cis</em>; <strong>A</strong>) or on opposite sides of the centromere (<em>trans</em>; B) were analyzed separately. Data for gene pairs spanning the centromere on chromosome V were not available due to the experimental constraints of the SGA method (<em>Materials and Methods</em>). The dotted line indicates the average genetic distance (35 kb) of unlinked loci (50 SGA-GD) (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1D</a> and the section <em>Materials and Methods</em>). (C) Relationship between chromosome size and the shortest physical distance between two unlinked loci, positioned on the same (<em>cis</em>) or on opposite sides (<em>trans</em>) of the centromere. Each data point corresponds to a chromosome. Data for <em>trans</em>-centromere gene pairs on chromosome V were not available due to the experimental constraints of the SGA method (<em>Materials and Methods</em>). <em>R</em> corresponds to Pearson correlation coefficient. (D) The shortest physical distance between two unlinked loci, positioned on opposite sides of the centromere, was calculated for each chromosome using the genetic map downloaded from the <em>Saccharomyces</em> Genome Database (<a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">Cherry <em>et al.</em> 2012</a>), and plotted against chromosome length. <em>R</em> corresponds to Pearson correlation coefficient.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Negative correlation between chromosome size and levels of pericentric DSBs</h3><p id="p-35">Meiotic recombination begins with programmed catalysis of DNA double-strand breaks (DSBs) by <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001014">Spo11</a>, an evolutionarily conserved type II topoisomerase-like endonuclease (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bergerat <em>et al.</em> 1997</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Keeney <em>et al.</em> 1997</a>). Genome-wide distributions of DSBs and COs exhibit a strong correlation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS4.pdf">Figure S4</a>) (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2008</a>; <a id="xref-ref-34-8" class="xref-bibr" href="#ref-34">Mancera <em>et al.</em> 2008</a>; <a id="xref-ref-37-8" class="xref-bibr" href="#ref-37">Pan <em>et al.</em> 2011</a>). Moreover, similarly to COs, DSBs are repressed near centromeres (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Gerton <em>et al.</em> 2000</a>; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Buhler <em>et al.</em> 2007</a>; <a id="xref-ref-37-9" class="xref-bibr" href="#ref-37">Pan <em>et al.</em> 2011</a>). To test whether the DSB repression might also be related to chromosome size, we examined the genome-wide DSB map based on high-throughput sequencing of short <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001014">Spo11</a>-bound oligonucleotides (<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001014">Spo11</a>-oligos), a byproduct of an early step in meiotic DSB repair and an established readout for the occurrence of a DSB (<a id="xref-ref-37-10" class="xref-bibr" href="#ref-37">Pan <em>et al.</em> 2011</a>). We found that, similarly to the suppression of recombination frequency, the extent of DSB suppression around centromeres appears to be directly proportional to chromosome size (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). For instance, in order to observe the same number of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001014">Spo11</a>-oligos between two <em>trans</em>-loci spanning the centromere, their physical distance must be greater on larger chromosomes compared to smaller ones (<em>R</em> = 0.75; <em>p</em> &lt; 0.001; <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, B and C</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/10/1741/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between chromosome size and frequency of DSB formation within pericentric regions. (A) The number of Spo11-bound oligos, a readout for meiotic DSBs, around centromeres was calculated from data reported in (Pan et al. 2011). The color scale represents the number of Spo11-bound oligos recovered between the centromere and loci within the physical distance indicated on the x-axis. The blue lines mark the DSB-repressed regions around the centromeres where less than 3000 Spo11-bound oligos were recovered. (B) Size of the DSB-repressed regions around the centromeres in A, represented in order of chromosome size. (C) Relationship between chromosome size and the size of pericentric DSB-repressed regions in panel A. R corresponds to Pearson correlation coefficient. Similar results were obtained with variable thresholds for the number of Spo11-bound oligos (data not shown). (D) Asymmetry of pericentric distributions of DSBs and meiotic recombination. An asymmetry score for pericentric recombination was calculated as the difference between the left and the right pericentric regions with at least 3000 Spo11 oligos detected [DSB (Spo11 cleavage) frequency] or presenting a genetic distance greater or equal to 50 SGA-GD from the centromere (SGA genetic linkage). Asymmetry scores, expressed in kilobases, are plotted for all chromosomes, sorted by their size." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-141262518" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship between chromosome size and frequency of DSB formation within pericentric regions. (A) The number of Spo11-bound oligos, a readout for meiotic DSBs, around centromeres was calculated from data reported in (Pan &lt;em&gt;et al.&lt;/em&gt; 2011). The color scale represents the number of Spo11-bound oligos recovered between the centromere and loci within the physical distance indicated on the x-axis. The blue lines mark the DSB-repressed regions around the centromeres where less than 3000 Spo11-bound oligos were recovered. (B) Size of the DSB-repressed regions around the centromeres in A, represented in order of chromosome size. (C) Relationship between chromosome size and the size of pericentric DSB-repressed regions in panel A. &lt;em&gt;R&lt;/em&gt; corresponds to Pearson correlation coefficient. Similar results were obtained with variable thresholds for the number of Spo11-bound oligos (data not shown). (D) Asymmetry of pericentric distributions of DSBs and meiotic recombination. An asymmetry score for pericentric recombination was calculated as the difference between the left and the right pericentric regions with at least 3000 Spo11 oligos detected [DSB (Spo11 cleavage) frequency] or presenting a genetic distance greater or equal to 50 SGA-GD from the centromere (SGA genetic linkage). Asymmetry scores, expressed in kilobases, are plotted for all chromosomes, sorted by their size.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/10/1741/F4.medium.gif" width="367" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/10/1741/F4.medium.gif" width="367" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/10/1741/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/10/1741/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463286" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">Relationship between chromosome size and frequency of DSB formation within pericentric regions. (A) The number of Spo11-bound oligos, a readout for meiotic DSBs, around centromeres was calculated from data reported in (<a id="xref-ref-37-11" class="xref-bibr" href="#ref-37">Pan <em>et al.</em> 2011</a>). The color scale represents the number of Spo11-bound oligos recovered between the centromere and loci within the physical distance indicated on the x-axis. The blue lines mark the DSB-repressed regions around the centromeres where less than 3000 Spo11-bound oligos were recovered. (B) Size of the DSB-repressed regions around the centromeres in A, represented in order of chromosome size. (C) Relationship between chromosome size and the size of pericentric DSB-repressed regions in panel A. <em>R</em> corresponds to Pearson correlation coefficient. Similar results were obtained with variable thresholds for the number of Spo11-bound oligos (data not shown). (D) Asymmetry of pericentric distributions of DSBs and meiotic recombination. An asymmetry score for pericentric recombination was calculated as the difference between the left and the right pericentric regions with at least 3000 Spo11 oligos detected [DSB (Spo11 cleavage) frequency] or presenting a genetic distance greater or equal to 50 SGA-GD from the centromere (SGA genetic linkage). Asymmetry scores, expressed in kilobases, are plotted for all chromosomes, sorted by their size.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">The aforementioned analyses suggest that chromosome size may regulate both frequencies of meiotic DSBs and COs around centromeres. Nevertheless, we observed some exceptional cases where a previously mapped DSB hotspot occurred within a pericentric linkage region (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS1.pdf">Figure S1</a>), consistent with the fact that not all meiotic DSBs are repaired via the CO-generating inter-homolog repair pathway (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Schwacha and Kleckner 1994</a>). We also found that pericentric linkage regions were often asymmetrical with respect to DSB frequency, as well as linkage intensity (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS1.pdf">Figure S1</a>; <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4D</a>), suggesting that chromosome size and distance from the centromere are not the only factors determining the frequency of meiotic DSB catalysis and/or the manner in which these breaks are repaired.</p></div><div id="sec-14" class="subsection"><h3>Positive correlation between chromosome size and pericentric association of meiotic cohesin subunit Rec8</h3><p id="p-38">Chromatin structure regulates both the formation and repair of meiotic DSBs. Of particular interest in the current context is the status of sister chromatid cohesion, which impacts both processes. Specifically, occurrence of meiotic DSBs is confined within the regions that are devoid of cohesin molecules, referred to as chromatin loops (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Blat and Kleckner 1999</a>). Sister chromatid cohesion also antagonizes COs, generating interhomolog DSB repair by favoring an alternative intersister repair pathway (<em>e.g.</em>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Niu <em>et al.</em> 2005</a>). Combining these considerations with the fact that cohesin molecules are enriched at centromeres (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Glynn <em>et al.</em> 2004</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Kiburz <em>et al.</em> 2005</a>) raised the possibility that the effect of chromosome size might be linked to cohesin association at centromeres. To explore this possibility, we examined the relationship between chromosome size and the extent of centromere-associated clustering of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006211">Rec8</a>, an evolutionary conserved meiotic subunit of the multiprotein cohesin complex. We found that the width of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006211">Rec8</a> centromere-binding peak, measured from two different genome-wide surveys of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006211">Rec8</a> distribution during meiotic metaphase/anaphase (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Glynn <em>et al.</em> 2004</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Kiburz <em>et al.</em> 2005</a>), correlates with chromosome size (<em>R</em> = 0.75 and <em>R</em> = 0.67, respectively, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5, A and B</a> and the section <em>Materials and Methods</em>). In contrast, for mitotic cohesion, the centromere association of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002161">Mcd1</a>/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002161">Scc1</a> does not show size dependent variation (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5C</a>), suggesting that the size dependence might be specific to meiotic centromere function.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/10/1741/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between chromosome size and the extent of the centromere-associated clustering of Rec8 and Mcd1/Scc1. (A, B) Positive correlation between chromosome size and the meiotic cohesin component Rec8. Quantitative profiles of Rec8 binding at 4 hr (A) and 5 hr (B) after the initiation of meiosis, corresponding to meiotic metaphase/anaphase, were obtained from Glynn et al. (2004) and Kiburz et al. (2005), respectively (Materials and Methods). Each data point corresponds to a chromosome. R corresponds to Pearson correlation coefficient. The correlation is apparent in the two independent studies despite the noticeable differences in the sizes of the centromere clusters of Rec8, likely reflecting differences in data collection and/or analyses. (C) Relationship between chromosome size and the extent of centromere-associated clustering of the mitotic cohesin component Mcd1/Scc1 in metaphase arrested cells (Glynn et al. 2004)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-141262518" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship between chromosome size and the extent of the centromere-associated clustering of Rec8 and Mcd1/Scc1. (A, B) Positive correlation between chromosome size and the meiotic cohesin component Rec8. Quantitative profiles of Rec8 binding at 4 hr (A) and 5 hr (B) after the initiation of meiosis, corresponding to meiotic metaphase/anaphase, were obtained from Glynn &lt;em&gt;et al.&lt;/em&gt; (2004) and Kiburz &lt;em&gt;et al.&lt;/em&gt; (2005), respectively (&lt;em&gt;Materials and Methods&lt;/em&gt;). Each data point corresponds to a chromosome. &lt;em&gt;R&lt;/em&gt; corresponds to Pearson correlation coefficient. The correlation is apparent in the two independent studies despite the noticeable differences in the sizes of the centromere clusters of Rec8, likely reflecting differences in data collection and/or analyses. (C) Relationship between chromosome size and the extent of centromere-associated clustering of the mitotic cohesin component Mcd1/Scc1 in metaphase arrested cells (Glynn &lt;em&gt;et al.&lt;/em&gt; 2004).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/10/1741/F5.medium.gif" width="402" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/10/1741/F5.medium.gif" width="402" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/10/1741/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/10/1741/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463466" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-39" class="first-child">Relationship between chromosome size and the extent of the centromere-associated clustering of Rec8 and Mcd1/Scc1. (A, B) Positive correlation between chromosome size and the meiotic cohesin component Rec8. Quantitative profiles of Rec8 binding at 4 hr (A) and 5 hr (B) after the initiation of meiosis, corresponding to meiotic metaphase/anaphase, were obtained from <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Glynn <em>et al.</em> (2004)</a> and <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Kiburz <em>et al.</em> (2005)</a>, respectively (<em>Materials and Methods</em>). Each data point corresponds to a chromosome. <em>R</em> corresponds to Pearson correlation coefficient. The correlation is apparent in the two independent studies despite the noticeable differences in the sizes of the centromere clusters of Rec8, likely reflecting differences in data collection and/or analyses. (C) Relationship between chromosome size and the extent of centromere-associated clustering of the mitotic cohesin component Mcd1/Scc1 in metaphase arrested cells (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Glynn <em>et al.</em> 2004</a>).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-40">Meiosis is a specialized cell division program, during which a single round of genome replication is followed by two successive rounds of genome segregation. During the first meiotic division (meiosis I) sister chromatids remain associated with one another, whereas homologs segregate to opposite poles. The accuracy of this process depends on the persistent centromeric cohesion and the formation of interhomolog COs, which facilitate homolog alignment, orientation and bipolar attachment to the meiotic spindle (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bishop and Zickler 2004</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Pinsky and Biggins 2005</a>).</p><p id="p-41">Frequency and positioning of COs impact chromosome segregation because COs that are too distal from centromeres are not as effective in facilitating bipolar attachment (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Koehler <em>et al.</em> 1996</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Ross <em>et al.</em> 1996</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lacefield and Murray 2007</a>), whereas COs that are too close to centromeres can lead to precocious separation of sister chromatids (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rockmill <em>et al.</em> 2006</a>). Indeed, centromere proximal CO formation is reduced in many organisms (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Brar and Amon 2008</a>) and may represent a general mode of regulating the fidelity of meiotic chromosome segregation (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Rockmill <em>et al.</em> 2006</a>; <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2008</a>). In humans, failure to form COs or their abnormal distribution cause aneuploidies and severe congenital conditions such as Down syndrome (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hassold <em>et al.</em> 2007</a>).</p><p id="p-42">Here, we show that colony size data associated with linked gene pairs can be used to build accurate maps of meiotic recombination throughout the yeast genome. Our study differs from previous genomic analysis of meiotic recombination in that we obtained our data from an isogenic strain, rather than a hybrid carrying alleles of two divergent genetic backgrounds (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2008</a>; <a id="xref-ref-34-9" class="xref-bibr" href="#ref-34">Mancera <em>et al.</em> 2008</a>). Our results confirm previous findings that: (1) the total number of CO events per chromosome increases with chromosome size, (2) the number of COs per unit length is greater in shorter chromosomes than longer ones, and (3) meiotic recombination occurs at lower frequencies in the vicinity of centromeres. We also present a previously unappreciated relationship between chromosome size and meiotic recombination, where chromosome size shows a negative correlation with pericentric CO density and DSB frequency, but a positive correlation with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006211">Rec8</a> association.</p><p id="p-43"><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001014">Spo11</a>-mediated catalysis of DSBs initiates meiotic recombination. Observations that sister chromatid cohesion antagonizes CO formation by down-regulating both DSB catalysis and CO-generating inter-homolog DSB repair (<em>e.g.</em>, <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Niu <em>et al.</em> 2005</a>; <a id="xref-ref-37-12" class="xref-bibr" href="#ref-37">Pan <em>et al.</em> 2011</a>) suggest that the effect of chromosome size on pericentric recombination might be mediated by size-dependent loading of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006211">Rec8</a>. Notably, however, because the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006211">Rec8</a> analysis was performed on cells in meiosis I (<em>i.e.</em>, after completion of meiotic recombination) (<a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Glynn <em>et al.</em> 2004</a>; <a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Kiburz <em>et al.</em> 2005</a>), it is possible that the observed association between chromosome size and centromeric enrichment of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006211">Rec8</a> is a consequence rather than the cause. For example, in mitosis, the extent of pericentric cohesin association is negatively correlated with tension between sister kinetochores (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Eckert <em>et al.</em> 2007</a>), possibly because mechanical tension acts toward displacing cohesin from pericentric regions. If a similar feedback system functions in meiosis, then perhaps the larger cohesin occupancy at the centromeres of larger chromosomes after meiotic recombination reflects the fact that their kinetochores experience reduced spindle tension, which may follow from more distal COs.</p><p id="p-44">The distribution of CO events per yeast chromosome can be described by the <em>a</em>L + <em>b</em> model (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Riles <em>et al.</em> 1993</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Stahl <em>et al.</em> 2004</a>; <a id="xref-ref-34-10" class="xref-bibr" href="#ref-34">Mancera <em>et al.</em> 2008</a>). This model is consistent with the hypothesis that yeast chromosomes undergo two different types of CO events: one type occurs at a constant rate per kb (parameter <em>a</em>), causing the total number of CO events to depend directly on chromosome size; the other CO type occurs at a constant rate per chromosome (parameter <em>b</em>) and thus is chromosome-size independent. Although the relative contribution of each type of CO events is still unclear, this model implies that, because of their small size, the shortest chromosomes would be relatively enriched for the <em>b</em>-type CO events and present a greater total recombination rate per kilobase, a trend that was indeed observed by earlier studies (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Kaback <em>et al.</em> 1992</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Riles <em>et al.</em> 1993</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kaback <em>et al.</em> 1999</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Stahl <em>et al.</em> 2004</a>; <a id="xref-ref-34-11" class="xref-bibr" href="#ref-34">Mancera <em>et al.</em> 2008</a>; <a id="xref-ref-37-13" class="xref-bibr" href="#ref-37">Pan <em>et al.</em> 2011</a>; <a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">Cherry <em>et al.</em> 2012</a>). It would be reasonable to speculate that the chromosome size-independent <em>b</em>-type CO may correspond to the so-called âobligateâ CO (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jones 1987</a>), which ensures that even the smallest possible chromosome, which would consist of just a centromere and two telomeres, undergoes at least one physical exchange with its homolog. This obligate physical exchange would be required to tether homologs together and facilitate their bi-orientation with respect to the spindle, thus ensuring proper chromosome alignment and segregation (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bishop and Zickler 2004</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Pinsky and Biggins 2005</a>). Because segregation is impaired when COs occur too close or too far from the centromere (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Koehler <em>et al.</em> 1996</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Ross <em>et al.</em> 1996</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Rockmill <em>et al.</em> 2006</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Lacefield and Murray 2007</a>), the centromere may play a role in determining the positioning of the obligate CO. This centromere function may be particularly relevant in humans where unregulated recombination near centromeres leads to aneuploidy and severe developmental abnormalities, including Down syndrome (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Hassold <em>et al.</em> 2007</a>). Importantly, a recent analysis of the human genome-wide recombination map showed that, similarly to yeast, the total number of CO events per chromosome follows the <em>a</em>L + <em>b</em> model, where <em>b</em> â 1 (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Fledel-Alon <em>et al.</em> 2009</a>).</p><p id="p-45">The relationship between chromosome size and the extent of pericentric linkage and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006211">Rec8</a> enrichment is likely to be linked to an essential requirement that centromeric cohesion remains intact during the first meiosis (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Brar and Amon 2008</a>). Indeed, budding yeast <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002693">zip1</a></em> mutants that are impaired in centromeric cohesin loading display promiscuous recombination in the vicinity of their centromeres (<a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2008</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bardhan <em>et al.</em> 2010</a>), suggesting a mechanistic link between meiotic centromere function and suppression of pericentric recombination. Here, we provide further evidence for such a link. In addition, we show that chromosome size itself might regulate both pericentric recombination and meiotic centromere function.</p><p id="p-46">Interestingly, in yeast <em>Saccharomyces cerevisiae</em> the inactivation of the spindle checkpoint causes larger chromosomes to missegregate more often than smaller chromosomes (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Shonn <em>et al.</em> 2000</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Lacefield and Murray 2007</a>). Because an essential function of the spindle checkpoint is to monitor bipolar attachment of chromosomes (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Pinsky and Biggins 2005</a>), this finding suggests that chromosome size may impact the efficacy of homolog biorientation. For example, COs may tend to occur further away from the centromere of larger chromosomes leading to a defect in spindle attachment (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Shonn <em>et al.</em> 2000</a>; <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Lacefield and Murray 2007</a>). This observation is particularly intriguing because all budding yeast centromeres appear to have the same point structure (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Brar and Amon 2008</a>).</p><p id="p-47">There is a precedent for size-dependent loading of chromosomal-bound proteins: the extent of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005394">Smc5</a>/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004375">Smc6</a> loading during mitotic S-phase is chromosome size dependent, and is proposed to be controlled by a chromosome structure-based mechanism (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kegel <em>et al.</em> 2011</a>). It is possible that the structural properties of larger chromosomes may differ from those of smaller chromosomes and may control <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006211">Rec8</a> loading at centromeres. Indeed, analysis of early <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006211">Rec8</a> loading at the onset of meiotic S-phase suggests a positive correlation between chromosome size and the initial pericentric clustering of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006211">Rec8</a> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1/FigureS6.pdf">Figure S6</a>). Regardless of the mechanism, our data points to the evolution of a balanced system, where the need to exclude COs from centromeres to ensure sister centromere cohesion and homolog centromere pairing creates a tendency for incorrect kinetochore-spindle attachment that is subsequently corrected by the spindle checkpoint (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Shonn <em>et al.</em> 2000</a>; <a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Lacefield and Murray 2007</a>).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">We thank Eric Foss for providing comments on the manuscript. This work was supported by National Institutes of Health grants 1R01HG005853-01 and 1R01HG005084-01A1 to C.B., B.A., and C.L.M, Canadian Institutes of Health Research grants MOP-102629, MOP-97939 and MOP-57830 to C.B., an Ontario Research Fund grant GL2-01-22 to C.B. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007377/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-49"><em>Communicating editor: N. Hunter</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-06-23"><span class="received-label">Received </span>June 23, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-08-02"><span class="accepted-label">Accepted </span>August 2, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2013 Baryshnikova <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.1" data-doi="10.1091/mbc.E09-05-0392"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bardhan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chuong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Meiotic cohesin promotes pairing of nonhomologous centromeres in early meiotic prophase.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1799</span>â<span class="cit-lpage">1809</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E09-05-0392%26rft_id%253Dinfo%253Apmid%252F20375150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMS8xNzk5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMC8xNzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.2" data-doi="10.1016/S0076-6879(10)70007-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vizeacoumar</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">a</span>â<span class="cit-article-title">Synthetic genetic array (SGA) analysis in <em>Saccharomyces cerevisiae</em> and <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">470</span>: <span class="cit-fpage">145</span>â<span class="cit-lpage">179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DBaryshnikova%26rft.auinit1%253DA.%26rft.volume%253D470%26rft.spage%253D145%26rft.epage%253D179%26rft.atitle%253DSynthetic%2Bgenetic%2Barray%2B%2528SGA%2529%2Banalysis%2Bin%2BSaccharomyces%2Bcerevisiae%2Band%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0076-6879%252810%252970007-0%26rft_id%253Dinfo%253Apmid%252F20946810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0076-6879(10)70007-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20946810&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275827900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.3" data-doi="10.1038/nmeth.1534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">b</span>â<span class="cit-article-title">Quantitative analysis of fitness and genetic interactions in yeast on a genome scale.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1017</span>â<span class="cit-lpage">1024</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DBaryshnikova%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253D1017%26rft.epage%253D1024%26rft.atitle%253DQuantitative%2Banalysis%2Bof%2Bfitness%2Band%2Bgenetic%2Binteractions%2Bin%2Byeast%2Bon%2Ba%2Bgenome%2Bscale.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1534%26rft_id%253Dinfo%253Apmid%252F21076421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21076421&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284686300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.4" data-doi="10.1073/pnas.94.10.5213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baudat</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Clustering of meiotic double-strand breaks on yeast chromosome III.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">5213</span>â<span class="cit-lpage">5218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBaudat%26rft.auinit1%253DF.%26rft.volume%253D94%26rft.issue%253D10%26rft.spage%253D5213%26rft.epage%253D5218%26rft.atitle%253DClustering%2Bof%2Bmeiotic%2Bdouble-strand%2Bbreaks%2Bon%2Byeast%2Bchromosome%2BIII%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.10.5213%26rft_id%253Dinfo%253Apmid%252F9144217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTQvMTAvNTIxMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTAvMTc0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.5" data-doi="10.1038/386414a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergerat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Massy</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gadelle</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varoutas</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">An atypical topoisomerase II from Archaea with implications for meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">386</span>: <span class="cit-fpage">414</span>â<span class="cit-lpage">417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBergerat%26rft.auinit1%253DA.%26rft.volume%253D386%26rft.issue%253D6623%26rft.spage%253D414%26rft.epage%253D417%26rft.atitle%253DAn%2Batypical%2Btopoisomerase%2BII%2Bfrom%2BArchaea%2Bwith%2Bimplications%2Bfor%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F386414a0%26rft_id%253Dinfo%253Apmid%252F9121560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/386414a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9121560&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WQ17000069&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.6" data-doi="10.1016/S0092-8674(04)00297-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zickler</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Early decision; meiotic crossover interference prior to stable strand exchange and synapsis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">9</span>â<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBishop%26rft.auinit1%253DD.%2BK.%26rft.volume%253D117%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D15%26rft.atitle%253DEarly%2Bdecision%253B%2Bmeiotic%2Bcrossover%2Binterference%2Bprior%2Bto%2Bstable%2Bstrand%2Bexchange%2Band%2Bsynapsis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252804%252900297-1%26rft_id%253Dinfo%253Apmid%252F15066278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(04)00297-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15066278&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221500300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.7" data-doi="10.1016/S0092-8674(00)81019-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blat</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Cohesins bind to preferential sites along yeast chromosome III, with differential regulation along arms <em>vs.</em> the centric region.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">249</span>â<span class="cit-lpage">259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBlat%26rft.auinit1%253DY.%26rft.volume%253D98%26rft.issue%253D2%26rft.spage%253D249%26rft.epage%253D259%26rft.atitle%253DCohesins%2Bbind%2Bto%2Bpreferential%2Bsites%2Balong%2Byeast%2Bchromosome%2BIII%252C%2Bwith%2Bdifferential%2Bregulation%2Balong%2Barms%2Bversus%2Bthe%2Bcentric%2Bregion.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981019-3%26rft_id%253Dinfo%253Apmid%252F10428036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81019-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10428036&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081632300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.8" data-doi="10.1016/j.cub.2007.10.066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blitzblau</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochwagen</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Mapping of meiotic single-stranded DNA reveals double-stranded-break hotspots near centromeres and telomeres.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2003</span>â<span class="cit-lpage">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DBlitzblau%26rft.auinit1%253DH.%2BG.%26rft.volume%253D17%26rft.issue%253D23%26rft.spage%253D2003%26rft.epage%253D2012%26rft.atitle%253DMapping%2Bof%2Bmeiotic%2Bsingle-stranded%2BDNA%2Breveals%2Bdouble-stranded-break%2Bhotspots%2Bnear%2Bcentromeres%2Band%2Btelomeres.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2007.10.066%26rft_id%253Dinfo%253Apmid%252F18060788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2007.10.066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18060788&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251482100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.9" data-doi="10.1016/S1097-2765(04)00034-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borde</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novikov</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrini</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Association of Mre11p with double-strand break sites during yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">389</span>â<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DBorde%26rft.auinit1%253DV.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D389%26rft.epage%253D401%26rft.atitle%253DAssociation%2Bof%2BMre11p%2Bwith%2Bdouble-strand%2Bbreak%2Bsites%2Bduring%2Byeast%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252804%252900034-6%26rft_id%253Dinfo%253Apmid%252F14967146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(04)00034-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14967146&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188959000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.10" data-doi="10.1038/nrg2454"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brar</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amon</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Emerging roles for centromeres in meiosis I chromosome segregation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">899</span>â<span class="cit-lpage">910</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBrar%26rft.auinit1%253DG.%2BA.%26rft.volume%253D9%26rft.issue%253D12%26rft.spage%253D899%26rft.epage%253D910%26rft.atitle%253DEmerging%2Broles%2Bfor%2Bcentromeres%2Bin%2Bmeiosis%2BI%2Bchromosome%2Bsegregation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2454%26rft_id%253Dinfo%253Apmid%252F18981989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2454&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18981989&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261037200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.11" data-doi="10.1371/journal.pbio.0050324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buhler</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borde</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Mapping meiotic single-strand DNA reveals a new landscape of DNA double-strand breaks in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DBuhler%26rft.auinit1%253DC.%26rft.volume%253D5%26rft.issue%253D12%26rft.spage%253De324%26rft.epage%253De324%26rft.atitle%253DMapping%2Bmeiotic%2Bsingle-strand%2BDNA%2Breveals%2Ba%2Bnew%2Blandscape%2Bof%2BDNA%2Bdouble-strand%2Bbreaks%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050324%26rft_id%253Dinfo%253Apmid%252F18076285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076285&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.12" data-doi="10.1016/j.devcel.2008.07.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsubouchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandler</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Global analysis of the meiotic crossover landscape.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">401</span>â<span class="cit-lpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DChen%26rft.auinit1%253DS.%2BY.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D401%26rft.epage%253D415%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bthe%2Bmeiotic%2Bcrossover%2Blandscape.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2008.07.006%26rft_id%253Dinfo%253Apmid%252F18691940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2008.07.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18691940&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259358100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.13" data-doi="10.1093/nar/gkr1029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cherry</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amundsen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balakrishnan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binkley</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Saccharomyces Genome Database: the genomics resource of budding yeast.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D700</span>â<span class="cit-lpage">D705</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1029%26rft_id%253Dinfo%253Apmid%252F22110037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC9EMS9ENzAwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMC8xNzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.14" data-doi="10.1186/gb-2006-7-7-r63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuldiner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">A strategy for extracting and analyzing large-scale quantitative epistatic interaction data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">R63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DCollins%26rft.auinit1%253DS.%2BR.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253DR63%26rft.epage%253DR63%26rft.atitle%253DA%2Bstrategy%2Bfor%2Bextracting%2Band%2Banalyzing%2Blarge-scale%2Bquantitative%2Bepistatic%2Binteraction%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-7-r63%26rft_id%253Dinfo%253Apmid%252F16859555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-7-r63&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16859555&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.15" data-doi="10.1126/science.1180823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spear</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">The genetic landscape of a cell.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">425</span>â<span class="cit-lpage">431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCostanzo%26rft.auinit1%253DM.%26rft.volume%253D327%26rft.issue%253D5964%26rft.spage%253D425%26rft.epage%253D431%26rft.atitle%253DThe%2BGenetic%2BLandscape%2Bof%2Ba%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1180823%26rft_id%253Dinfo%253Apmid%252F20093466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2NC80MjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE3NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.16" data-doi="10.1101/gad.1498707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gravdahl</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megee</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">The enhancement of pericentromeric cohesin association by conserved kinetochore components promotes high-fidelity chromosome segregation and is sensitive to microtubule-based tension.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">278</span>â<span class="cit-lpage">291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1498707%26rft_id%253Dinfo%253Apmid%252F17242156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjEvMy8yNzgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE3NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.17" data-doi="10.1371/journal.pgen.1000658"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fledel-Alon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Broad-scale recombination patterns underlying proper disjunction in humans.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DFledel-Alon%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253De1000658%26rft.epage%253De1000658%26rft.atitle%253DBroad-scale%2Brecombination%2Bpatterns%2Bunderlying%2Bproper%2Bdisjunction%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000658%26rft_id%253Dinfo%253Apmid%252F19763175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000658&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19763175&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.18" data-doi="10.1073/pnas.97.21.11383"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerton</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derisi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shroff</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Global mapping of meiotic recombination hotspots and coldspots in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">11383</span>â<span class="cit-lpage">11390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DGerton%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D97%26rft.issue%253D21%26rft.spage%253D11383%26rft.epage%253D11390%26rft.atitle%253DGlobal%2Bmapping%2Bof%2Bmeiotic%2Brecombination%2Bhotspots%2Band%2Bcoldspots%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.97.21.11383%26rft_id%253Dinfo%253Apmid%252F11027339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMjEvMTEzODMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE3NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.19" data-doi="10.1371/journal.pbio.0020259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glynn</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megee</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mistrot</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Unal</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Genome-wide mapping of the cohesin complex in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">E259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DGlynn%26rft.auinit1%253DE.%2BF.%26rft.volume%253D2%26rft.issue%253D9%26rft.spage%253DE259%26rft.epage%253DE259%26rft.atitle%253DGenome-wide%2Bmapping%2Bof%2Bthe%2Bcohesin%2Bcomplex%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0020259%26rft_id%253Dinfo%253Apmid%252F15309048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0020259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15309048&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-book" id="cit-3.10.1741.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">A. J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gelbart</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Accurate calculation of large map distances</span>, <span class="cit-source">An Introduction to Genetic Analysis</span>, <span class="cit-edition">Ed. 7</span>. <span class="cit-publ-name">W. H. Freeman</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-book" id="cit-3.10.1741.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">E. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-source">Genetics. Analysis of Genes and Genomes</span>, <span class="cit-edition">Ed. VII</span>. <span class="cit-publ-name">Jones and Bartlett Publishers</span>, <span class="cit-publ-loc">Sudbury, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hassold</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">The origin of human aneuploidy: where we have been, where we are going.</span> <abbr class="cit-jnl-abbrev">Human Mol. Genet.</abbr> <span class="cit-vol">16(Spec No. 2)</span>: <span class="cit-fpage">R203</span>â<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2BMol.%2BGenet.%26rft.volume%253D162%26rft.spage%253DR203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-book" id="cit-3.10.1741.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span>â<span class="cit-article-title">Chiasmata</span>, pp. <span class="cit-fpage">213</span>â<span class="cit-lpage">244</span> in <span class="cit-source">Meiosis</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Moens</span>  <span class="cit-name-given-names">P. B.</span></span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">Orlando, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.24" data-doi="10.1038/ng0596-20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaback</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Chromosome-size dependent control of meiotic recombination in humans.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">20</span>â<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKaback%26rft.auinit1%253DD.%2BB.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D20%26rft.epage%253D21%26rft.atitle%253DChromosome-size%2Bdependent%2Bcontrol%2Bof%2Bmeiotic%2Brecombination%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0596-20%26rft_id%253Dinfo%253Apmid%252F8673097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0596-20&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8673097&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UH43000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.25" data-doi="10.1126/science.1566070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaback</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guacci</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barber</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahon</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span>â<span class="cit-article-title">Chromosome size-dependent control of meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">256</span>: <span class="cit-fpage">228</span>â<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKaback%26rft.auinit1%253DD.%26rft.volume%253D256%26rft.issue%253D5054%26rft.spage%253D228%26rft.epage%253D232%26rft.atitle%253DChromosome%2Bsize-dependent%2Bcontrol%2Bof%2Bmeiotic%2Brecombination%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1566070%26rft_id%253Dinfo%253Apmid%252F1566070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNTYvNTA1NC8yMjgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE3NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaback</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barber</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamb</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Chromosome size-dependent control of meiotic reciprocal recombination in <em>Saccharomyces cerevisiae</em>: the role of crossover interference.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">1475</span>â<span class="cit-lpage">1486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKaback%26rft.auinit1%253DD.%2BB.%26rft.volume%253D152%26rft.issue%253D4%26rft.spage%253D1475%26rft.epage%253D1486%26rft.atitle%253DChromosome%2BSize-Dependent%2BControl%2Bof%2BMeiotic%2BReciprocal%2BRecombination%2Bin%2BSaccharomyces%2Bcerevisiae%253A%2BThe%2BRole%2Bof%2BCrossover%2BInterference%26rft_id%253Dinfo%253Apmid%252F10430577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Mi80LzE0NzUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE3NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.27" data-doi="10.1016/S0092-8674(00)81876-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keeney</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giroux</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Meiosis-specific DNA double-strand breaks are catalyzed by Spo11, a member of a widely conserved protein family.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">375</span>â<span class="cit-lpage">384</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKeeney%26rft.auinit1%253DS.%26rft.volume%253D88%26rft.issue%253D3%26rft.spage%253D375%26rft.epage%253D384%26rft.atitle%253DMeiosis-specific%2BDNA%2Bdouble-strand%2Bbreaks%2Bare%2Bcatalyzed%2Bby%2BSpo11%252C%2Ba%2Bmember%2Bof%2Ba%2Bwidely%2Bconserved%2Bprotein%2Bfamily.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981876-0%26rft_id%253Dinfo%253Apmid%252F9039264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81876-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9039264&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WG47900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.28" data-doi="10.1038/nature09791"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kegel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betts-Lindroos</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanno</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeppsson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strom</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Chromosome length influences replication-induced topological stress.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">471</span>: <span class="cit-fpage">392</span>â<span class="cit-lpage">396</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKegel%26rft.auinit1%253DA.%26rft.volume%253D471%26rft.issue%253D7338%26rft.spage%253D392%26rft.epage%253D396%26rft.atitle%253DChromosome%2Blength%2Binfluences%2Breplication-induced%2Btopological%2Bstress.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09791%26rft_id%253Dinfo%253Apmid%252F21368764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09791&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21368764&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288444000048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.29" data-doi="10.1101/gad.1373005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kiburz</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megee</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marston</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">The core centromere and Sgo1 establish a 50-kb cohesin-protected domain around centromeres during meiosis I.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3017</span>â<span class="cit-lpage">3030</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DKiburz%26rft.auinit1%253DB.%2BM.%26rft.volume%253D19%26rft.issue%253D24%26rft.spage%253D3017%26rft.epage%253D3030%26rft.atitle%253DThe%2Bcore%2Bcentromere%2Band%2BSgo1%2Bestablish%2Ba%2B50-kb%2Bcohesin-protected%2Bdomain%2Baround%2Bcentromeres%2Bduring%2Bmeiosis%2BI%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1373005%26rft_id%253Dinfo%253Apmid%252F16357219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE5LzI0LzMwMTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE3NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.30" data-doi="10.1038/ng1296-406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koehler</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boulton</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">French</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herman</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Spontaneous X chromosome MI and MII nondisjunction events in <em>Drosophila melanogaster</em> oocytes have different recombinational histories.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">406</span>â<span class="cit-lpage">414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKoehler%26rft.auinit1%253DK.%2BE.%26rft.volume%253D14%26rft.issue%253D4%26rft.spage%253D406%26rft.epage%253D414%26rft.atitle%253DSpontaneous%2BX%2Bchromosome%2BMI%2Band%2BMII%2Bnondisjunction%2Bevents%2Bin%2BDrosophila%2Bmelanogaster%2Boocytes%2Bhave%2Bdifferent%2Brecombinational%2Bhistories.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1296-406%26rft_id%253Dinfo%253Apmid%252F8944020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1296-406&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8944020&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VV73000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.31" data-doi="10.1091/mbc.E08-12-1223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kugou</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamada</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasanuma</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Rec8 guides canonical Spo11 distribution along yeast meiotic chromosomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3064</span>â<span class="cit-lpage">3076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E08-12-1223%26rft_id%253Dinfo%253Apmid%252F19439448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMy8zMDY0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMC8xNzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.32" data-doi="10.1038/ng2120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lacefield</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">The spindle checkpoint rescues the meiotic segregation of chromosomes whose crossovers are far from the centromere.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1273</span>â<span class="cit-lpage">1277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLacefield%26rft.auinit1%253DS.%26rft.volume%253D39%26rft.issue%253D10%26rft.spage%253D1273%26rft.epage%253D1277%26rft.atitle%253DThe%2Bspindle%2Bcheckpoint%2Brescues%2Bthe%2Bmeiotic%2Bsegregation%2Bof%2Bchromosomes%2Bwhose%2Bcrossovers%2Bare%2Bfar%2Bfrom%2Bthe%2Bcentromere.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2120%26rft_id%253Dinfo%253Apmid%252F17828265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17828265&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249737400025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.33" data-doi="10.1146/annurev.ge.29.120195.002231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldman</span>  <span class="cit-name-given-names">A. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-article-title">Meiotic recombination hotspots.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">423</span>â<span class="cit-lpage">444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DLichten%26rft.auinit1%253DM.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D423%26rft.epage%253D444%26rft.atitle%253DMeiotic%2BRecombination%2BHotspots%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ge.29.120195.002231%26rft_id%253Dinfo%253Apmid%252F8825482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ge.29.120195.002231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8825482&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TL71900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.34" data-doi="10.1038/nature07135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mancera</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bourgon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brozzi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">High-resolution mapping of meiotic crossovers and non-crossovers in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">454</span>: <span class="cit-fpage">479</span>â<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMancera%26rft.auinit1%253DE.%26rft.volume%253D454%26rft.issue%253D7203%26rft.spage%253D479%26rft.epage%253D485%26rft.atitle%253DHigh-resolution%2Bmapping%2Bof%2Bmeiotic%2Bcrossovers%2Band%2Bnon-crossovers%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07135%26rft_id%253Dinfo%253Apmid%252F18615017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18615017&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257860300043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-book" id="cit-3.10.1741.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morton</span>  <span class="cit-name-given-names">N. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">History of genetic mapping</span>, pp. X-X, in <span class="cit-source">Encyclopedia of Genetics, Genomics, Proteomics and Bioinformatics</span>. <span class="cit-publ-name">John Wiley &amp; Sons</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.36" data-doi="10.1091/mbc.E05-05-0465"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baumgartner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaefer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loidl</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Partner choice during meiosis is regulated by Hop1-promoted dimerization of Mek1.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">5804</span>â<span class="cit-lpage">5818</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E05-05-0465%26rft_id%253Dinfo%253Apmid%252F16221890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxNi8xMi81ODA0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMC8xNzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.37" data-doi="10.1016/j.cell.2011.02.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kniewel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murakami</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blitzblau</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">A hierarchical combination of factors shapes the genome-wide topography of yeast meiotic recombination initiation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">719</span>â<span class="cit-lpage">731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPan%26rft.auinit1%253DJ.%26rft.volume%253D144%26rft.issue%253D5%26rft.spage%253D719%26rft.epage%253D731%26rft.atitle%253DA%2Bhierarchical%2Bcombination%2Bof%2Bfactors%2Bshapes%2Bthe%2Bgenome-wide%2Btopography%2Bof%2Byeast%2Bmeiotic%2Brecombination%2Binitiation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.02.009%26rft_id%253Dinfo%253Apmid%252F21376234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.02.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21376234&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288007100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.38" data-doi="10.1016/j.tcb.2005.07.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pinsky</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biggins</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">The spindle checkpoint: tension <em>vs.</em> attachment.</span> <abbr class="cit-jnl-abbrev">Trends Cell Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">486</span>â<span class="cit-lpage">493</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bcell%2Bbiology%26rft.stitle%253DTrends%2BCell%2BBiol%26rft.aulast%253DPinsky%26rft.auinit1%253DB.%2BA.%26rft.volume%253D15%26rft.issue%253D9%26rft.spage%253D486%26rft.epage%253D493%26rft.atitle%253DThe%2Bspindle%2Bcheckpoint%253A%2Btension%2Bversus%2Battachment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tcb.2005.07.005%26rft_id%253Dinfo%253Apmid%252F16084093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tcb.2005.07.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16084093&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232333700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riles</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dutchik</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baktha</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mccauley</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thayer</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span>â<span class="cit-article-title">Physical maps of the six smallest chromosomes of <em>Saccharomyces cerevisiae</em> at a resolution of 2.6 kilobase pairs.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">81</span>â<span class="cit-lpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRiles%26rft.auinit1%253DL.%26rft.volume%253D134%26rft.issue%253D1%26rft.spage%253D81%26rft.epage%253D150%26rft.atitle%253DPhysical%2Bmaps%2Bof%2Bthe%2Bsix%2Bsmallest%2Bchromosomes%2Bof%2BSaccharomyces%2Bcerevisiae%2Bat%2Ba%2Bresolution%2Bof%2B2.6%2Bkilobase%2Bpairs.%26rft_id%253Dinfo%253Apmid%252F8514151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTM0LzEvODEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE3NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.40" data-doi="10.1534/genetics.106.058933"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voelkel-Meiman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Centromere-proximal crossovers are associated with precocious separation of sister chromatids during meiosis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">1745</span>â<span class="cit-lpage">1754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.058933%26rft_id%253Dinfo%253Apmid%252F17028345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC80LzE3NDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE3NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.41" data-doi="10.1073/pnas.93.10.4979"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">L. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maxfield</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Exchanges are not equally able to enhance meiotic chromosome segregation in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">4979</span>â<span class="cit-lpage">4983</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DRoss%26rft.auinit1%253DL.%2BO.%26rft.volume%253D93%26rft.issue%253D10%26rft.spage%253D4979%26rft.epage%253D4983%26rft.atitle%253DExchanges%2Bare%2Bnot%2Bequally%2Bable%2Bto%2Benhance%2Bmeiotic%2Bchromosome%2Bsegregation%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.10.4979%26rft_id%253Dinfo%253Apmid%252F8643515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTMvMTAvNDk3OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTAvMTc0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.42" data-doi="10.1016/0092-8674(94)90172-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwacha</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">Identification of joint molecules that form frequently between homologs but rarely between sister chromatids during yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">51</span>â<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSchwacha%26rft.auinit1%253DA.%26rft.volume%253D76%26rft.issue%253D1%26rft.spage%253D51%26rft.epage%253D63%26rft.atitle%253DIdentification%2Bof%2Bjoint%2Bmolecules%2Bthat%2Bform%2Bfrequently%2Bbetween%2Bhomologs%2Bbut%2Brarely%2Bbetween%2Bsister%2Bchromatids%2Bduring%2Byeast%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252894%252990172-4%26rft_id%253Dinfo%253Apmid%252F8287479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(94)90172-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8287479&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994MR49500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.43" data-doi="10.1126/science.289.5477.300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shonn</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mccarroll</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Requirement of the spindle checkpoint for proper chromosome segregation in budding yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">289</span>: <span class="cit-fpage">300</span>â<span class="cit-lpage">303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DShonn%26rft.auinit1%253DM.%2BA.%26rft.volume%253D289%26rft.issue%253D5477%26rft.spage%253D300%26rft.epage%253D303%26rft.atitle%253DRequirement%2Bof%2Bthe%2BSpindle%2BCheckpoint%2Bfor%2BProper%2BChromosome%2BSegregation%2Bin%2BBudding%2BYeast%2BMeiosis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.289.5477.300%26rft_id%253Dinfo%253Apmid%252F10894778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODkvNTQ3Ny8zMDAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE3NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.44" data-doi="10.1534/genetics.104.027789"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foss</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borts</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abdullah</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Does crossover interference count in <em>Saccharomyces cerevisiae</em>?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">35</span>â<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DStahl%26rft.auinit1%253DF.%2BW.%26rft.volume%253D168%26rft.issue%253D1%26rft.spage%253D35%26rft.epage%253D48%26rft.atitle%253DDoes%2BCrossover%2BInterference%2BCount%2Bin%2BSaccharomyces%2Bcerevisiae%253F%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.027789%26rft_id%253Dinfo%253Apmid%252F15454525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTY4LzEvMzUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE3NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1741.45" data-doi="10.1002/jez.1400140104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1913</span>â<span class="cit-article-title">A linear arrangement of six sex-linked factors in Drosophila, as shown by their mode of association.</span> <abbr class="cit-jnl-abbrev">J. Exp. Zool.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">43</span>â<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BZool.%26rft.volume%253D14%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjez.1400140104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jez.1400140104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000187376000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/10/1741.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/10/1727" title="Cytoplasmic Male Sterility Contributes to Hybrid Incompatibility Between Subspecies of Arabidopsis lyrata" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/10/1753" title="Characterization of Two ENU-Induced Mutations Affecting Mouse Skeletal Morphology" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/10"><span>Volume 3 Issue 10, October 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/10.cover-source.jpg?itok=h5LtrAoT" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/10/1741" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tPTLhsQWOqEgK3TX2Zg2mgJBH0CAbHCSsi-CC5x0-BM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/10/1741" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/10/1741" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/10/1741" class="active" data-icon-position="" data-hide-link-title="0">Global Linkage Map Connects Meiotic Centromere Function to Chromosome Size in Budding Yeast</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459547" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-MFYFiTaaXIEw9etNsZ6e_IvWcTo0rH0M3w4CFwadb98" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/10/1741.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/10/1741" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-YBbXJfQs28UGv-CdFbPx5PXt__umGXXas5zePqYUL3c" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/10/1741" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/10/1741.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459547" id="node459547--2" data-pisa="ggg;3/10/1741" data-pisa-master="ggg;g3.113.007377" data-apath="/ggg/3/10/1741.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Global Linkage Map Connects Meiotic Centromere Function to Chromosome Size in Budding Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anastasia Baryshnikova</span>, <span class="highwire-citation-author" data-delta="1">Benjamin VanderSluis</span>, <span class="highwire-citation-author" data-delta="2">Michael Costanzo</span>, <span class="highwire-citation-author" data-delta="3">Chad L. Myers</span>, <span class="highwire-citation-author" data-delta="4">Rita S. Cha</span>, <span class="highwire-citation-author" data-delta="5">Brenda Andrews</span> and <span class="highwire-citation-author" data-delta="6">Charles Boone</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1741-1751;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007377 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459547--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anastasia Baryshnikova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Banting and Best Department of Medical Research, The Donnelly Center for Cellular and Biomolecular Research, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anastasia%2BBaryshnikova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baryshnikova%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnastasia%2BBaryshnikova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Benjamin VanderSluis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Computer Science and Engineering, University of Minnesota, Minneapolis, Minnesota 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BVanderSluis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=VanderSluis%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenjamin%2BVanderSluis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael Costanzo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Banting and Best Department of Medical Research, The Donnelly Center for Cellular and Biomolecular Research, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BCostanzo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Costanzo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BCostanzo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chad L. Myers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Computer Science and Engineering, University of Minnesota, Minneapolis, Minnesota 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chad%2BL.%2BMyers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Myers%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChad%2BL.%2BMyers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Rita S. Cha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Department of Life Sciences, Genome Damage and Stability Centre, University of Sussex, Falmer, BN1 9RQ, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rita%2BS.%2BCha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cha%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARita%2BS.%2BCha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a>
<a href="mailto:rc320@sussex.ac.uk" class="" data-icon-position="" data-hide-link-title="0">rc320@sussex.ac.uk</a>
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brenda Andrews </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Banting and Best Department of Medical Research, The Donnelly Center for Cellular and Biomolecular Research, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brenda%2BAndrews%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andrews%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrenda%2BAndrews%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a>
<a href="mailto:rc320@sussex.ac.uk" class="" data-icon-position="" data-hide-link-title="0">rc320@sussex.ac.uk</a>
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Charles Boone </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Banting and Best Department of Medical Research, The Donnelly Center for Cellular and Biomolecular Research, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BBoone%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boone%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BBoone%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a>
<a href="mailto:rc320@sussex.ac.uk" class="" data-icon-position="" data-hide-link-title="0">rc320@sussex.ac.uk</a>
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/10/1741" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/10/1741&amp;title=Global%20Linkage%20Map%20Connects%20Meiotic%20Centromere%20Function%20to%20Chromosome%20Size%20in%20Budding%20Yeast" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/10/1741&amp;title=Global%20Linkage%20Map%20Connects%20Meiotic%20Centromere%20Function%20to%20Chromosome%20Size%20in%20Budding%20Yeast" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/10/1741&amp;title=Global%20Linkage%20Map%20Connects%20Meiotic%20Centromere%20Function%20to%20Chromosome%20Size%20in%20Budding%20Yeast" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/10/1741&amp;text=Global%20Linkage%20Map%20Connects%20Meiotic%20Centromere%20Function%20to%20Chromosome%20Size%20in%20Budding%20Yeast" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/10/1741&amp;title=Global%20Linkage%20Map%20Connects%20Meiotic%20Centromere%20Function%20to%20Chromosome%20Size%20in%20Budding%20Yeast" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/10/1741&amp;t=Global%20Linkage%20Map%20Connects%20Meiotic%20Centromere%20Function%20to%20Chromosome%20Size%20in%20Budding%20Yeast" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/10/1741&amp;title=Global%20Linkage%20Map%20Connects%20Meiotic%20Centromere%20Function%20to%20Chromosome%20Size%20in%20Budding%20Yeast" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/10/1741&amp;title=Global%20Linkage%20Map%20Connects%20Meiotic%20Centromere%20Function%20to%20Chromosome%20Size%20in%20Budding%20Yeast" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459547/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459547" id="node459547--3" data-pisa="ggg;3/10/1741" data-pisa-master="ggg;g3.113.007377" data-apath="/ggg/3/10/1741.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Global Linkage Map Connects Meiotic Centromere Function to Chromosome Size in Budding Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anastasia Baryshnikova</span>, <span class="highwire-citation-author" data-delta="1">Benjamin VanderSluis</span>, <span class="highwire-citation-author" data-delta="2">Michael Costanzo</span>, <span class="highwire-citation-author" data-delta="3">Chad L. Myers</span>, <span class="highwire-citation-author" data-delta="4">Rita S. Cha</span>, <span class="highwire-citation-author" data-delta="5">Brenda Andrews</span> and <span class="highwire-citation-author" data-delta="6">Charles Boone</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1741-1751;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007377 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459547--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anastasia Baryshnikova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Banting and Best Department of Medical Research, The Donnelly Center for Cellular and Biomolecular Research, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anastasia%2BBaryshnikova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baryshnikova%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnastasia%2BBaryshnikova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Benjamin VanderSluis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Computer Science and Engineering, University of Minnesota, Minneapolis, Minnesota 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BVanderSluis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=VanderSluis%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenjamin%2BVanderSluis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael Costanzo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Banting and Best Department of Medical Research, The Donnelly Center for Cellular and Biomolecular Research, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BCostanzo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Costanzo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BCostanzo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chad L. Myers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Computer Science and Engineering, University of Minnesota, Minneapolis, Minnesota 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chad%2BL.%2BMyers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Myers%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChad%2BL.%2BMyers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Rita S. Cha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Department of Life Sciences, Genome Damage and Stability Centre, University of Sussex, Falmer, BN1 9RQ, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rita%2BS.%2BCha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cha%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARita%2BS.%2BCha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a>
<a href="mailto:rc320@sussex.ac.uk" class="" data-icon-position="" data-hide-link-title="0">rc320@sussex.ac.uk</a>
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brenda Andrews </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Banting and Best Department of Medical Research, The Donnelly Center for Cellular and Biomolecular Research, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brenda%2BAndrews%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andrews%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrenda%2BAndrews%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a>
<a href="mailto:rc320@sussex.ac.uk" class="" data-icon-position="" data-hide-link-title="0">rc320@sussex.ac.uk</a>
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Charles Boone </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Banting and Best Department of Medical Research, The Donnelly Center for Cellular and Biomolecular Research, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S 3E1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BBoone%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boone%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BBoone%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a>
<a href="mailto:rc320@sussex.ac.uk" class="" data-icon-position="" data-hide-link-title="0">rc320@sussex.ac.uk</a>
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1741&amp;rft.epage=1751&amp;rft.atitle=Global%20Linkage%20Map%20Connects%20Meiotic%20Centromere%20Function%20to%20Chromosome%20Size%20in%20Budding%20Yeast&amp;rft.volume=3&amp;rft.issue=10&amp;rft.date=2013-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Baryshnikova%2C+Anastasia&amp;rft.au=VanderSluis%2C+Benjamin&amp;rft.au=Costanzo%2C+Michael&amp;rft.au=Myers%2C+Chad+L.&amp;rft.au=Cha%2C+Rita+S.&amp;rft.au=Andrews%2C+Brenda&amp;rft.au=Boone%2C+Charles"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459547/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459547/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459547/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459547/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459547/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459547/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459547/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459547/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459547/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459547/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459547/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459547/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/10/1741" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/10/1741.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/10/1741.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/10/1741.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/10/1741.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
