<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/10/1809" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/10/1809.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/10/1809.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/10/1809.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/10/1809.atom" />
<meta name="HW.pisa" content="ggg;3/10/1809" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Insights into the Evolution of Cotton Diploids and Polyploids from Whole-Genome Re-sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.113.007229" />
<meta name="DC.Date" content="2013-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Page et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Understanding the composition, evolution, and function of the Gossypium hirsutum (cotton) genome is complicated by the joint presence of two genomes in its nucleus (AT and DT genomes). These two genomes were derived from progenitor A-genome and D-genome diploids involved in ancestral allopolyploidization. To better understand the allopolyploid genome, we re-sequenced the genomes of extant diploid relatives that contain the A1 ( Gossypium herbaceum ), A2 ( Gossypium arboreum ), or D5 ( Gossypium raimondii ) genomes. We conducted a comparative analysis using deep re-sequencing of multiple accessions of each diploid species and identified 24 million SNPs between the A-diploid and D-diploid genomes. These analyses facilitated the construction of a robust index of conserved SNPs between the A-genomes and D-genomes at all detected polymorphic loci. This index is widely applicable for read mapping efforts of other diploid and allopolyploid Gossypium accessions. Further analysis also revealed locations of putative duplications and deletions in the A-genome relative to the D-genome reference sequence. The approximately 25,400 deleted regions included more than 50% deletion of 978 genes, including many involved with starch synthesis. In the polyploid genome, we also detected 1,472 conversion events between homoeologous chromosomes, including events that overlapped 113 genes. Continued characterization of the Gossypium genomes will further enhance our ability to manipulate fiber and agronomic production of cotton." />
<meta name="DC.Contributor" content="Justin T. Page" />
<meta name="DC.Contributor" content="Mark D. Huynh" />
<meta name="DC.Contributor" content="Zach S. Liechty" />
<meta name="DC.Contributor" content="Kara Grupp" />
<meta name="DC.Contributor" content="David Stelly" />
<meta name="DC.Contributor" content="Amanda M. Hulse" />
<meta name="DC.Contributor" content="Hamid Ashrafi" />
<meta name="DC.Contributor" content="Allen Van Deynze" />
<meta name="DC.Contributor" content="Jonathan F. Wendel" />
<meta name="DC.Contributor" content="Joshua A. Udall" />
<meta name="article:published_time" content="2013-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Insights into the Evolution of Cotton Diploids and Polyploids from Whole-Genome Re-sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Understanding the composition, evolution, and function of the &lt;i&gt;Gossypium hirsutum&lt;/i&gt; (cotton) genome is complicated by the joint presence of two genomes in its nucleus (A&lt;sub&gt;T&lt;/sub&gt; and D&lt;sub&gt;T&lt;/sub&gt; genomes). These two genomes were derived from progenitor A-genome and D-genome diploids involved in ancestral allopolyploidization. To better understand the allopolyploid genome, we re-sequenced the genomes of extant diploid relatives that contain the A&lt;sub&gt;1&lt;/sub&gt; (&lt;i&gt;Gossypium herbaceum&lt;/i&gt;), A&lt;sub&gt;2&lt;/sub&gt; (&lt;i&gt;Gossypium arboreum&lt;/i&gt;), or D&lt;sub&gt;5&lt;/sub&gt; (&lt;i&gt;Gossypium raimondii&lt;/i&gt;) genomes. We conducted a comparative analysis using deep re-sequencing of multiple accessions of each diploid species and identified 24 million SNPs between the A-diploid and D-diploid genomes. These analyses facilitated the construction of a robust index of conserved SNPs between the A-genomes and D-genomes at all detected polymorphic loci. This index is widely applicable for read mapping efforts of other diploid and allopolyploid Gossypium accessions. Further analysis also revealed locations of putative duplications and deletions in the A-genome relative to the D-genome reference sequence. The approximately 25,400 deleted regions included more than 50% deletion of 978 genes, including many involved with starch synthesis. In the polyploid genome, we also detected 1,472 conversion events between homoeologous chromosomes, including events that overlapped 113 genes. Continued characterization of the Gossypium genomes will further enhance our ability to manipulate fiber and agronomic production of cotton.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/10/01" />
<meta name="citation_mjid" content="ggg;3/10/1809" />
<meta name="citation_id" content="3/10/1809" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/10/1809" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/10/1809.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/10/1809.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/10/1809.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.007229" />
<meta name="citation_pmid" content="23979935" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1809" />
<meta name="citation_lastpage" content="1818" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Justin T. Page" />
<meta name="citation_author_institution" content="Biology Department, Brigham Young University, Provo, Utah 84602" />
<meta name="citation_author" content="Mark D. Huynh" />
<meta name="citation_author_institution" content="Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84602" />
<meta name="citation_author" content="Zach S. Liechty" />
<meta name="citation_author_institution" content="Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84602" />
<meta name="citation_author" content="Kara Grupp" />
<meta name="citation_author_institution" content="Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa 50011" />
<meta name="citation_author" content="David Stelly" />
<meta name="citation_author_institution" content="Department of Soil and Crop Sciences, Texas AgriLife Research, College Station, Texas 77843" />
<meta name="citation_author" content="Amanda M. Hulse" />
<meta name="citation_author_institution" content="Department of Soil and Crop Sciences, Texas AgriLife Research, College Station, Texas 77843" />
<meta name="citation_author" content="Hamid Ashrafi" />
<meta name="citation_author_institution" content="Seed Biotechnology Center, University of California-Davis, Davis, California 95616" />
<meta name="citation_author" content="Allen Van Deynze" />
<meta name="citation_author_institution" content="Seed Biotechnology Center, University of California-Davis, Davis, California 95616" />
<meta name="citation_author" content="Jonathan F. Wendel" />
<meta name="citation_author_institution" content="Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa 50011" />
<meta name="citation_author" content="Joshua A. Udall" />
<meta name="citation_author_institution" content="Biology Department, Brigham Young University, Provo, Utah 84602" />
<meta name="citation_author_institution" content="Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84602" />
<meta name="citation_author_email" content="jaudall@byu.edu" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=C. L. Brubaker;citation_author=A. H. Paterson;citation_author=J. F. Wendel;citation_title=Comparative genetic mapping of allotetraploid cotton and its diploid progenitors;citation_pages=184-203;citation_volume=42;citation_year=1999;citation_doi=10.1139/g98-118" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. L. Byers;citation_author=D. B. Harker;citation_author=S. M. Yourstone;citation_author=P. J. Maughan;citation_author=J. A. Udall;citation_title=Development and mapping of SNP assays in allotetraploid cotton.;citation_pages=1201-1214;citation_volume=124;citation_year=2012;citation_issue=7;citation_pmid=22252442;citation_doi=10.1007/s00122-011-1780-8" />
<meta name="citation_reference" content="citation_journal_title=Chin. Sci. Bull.;citation_author=C. Chaohua;citation_author=W. Xiyuan;citation_author=N. I. Xiyuan;citation_title=Observation of fiber ultrastructure of Ligon lintless mutant in upland cotton during fiber elongation;citation_pages=126-130;citation_volume=50;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J.-M. Chen;citation_author=D. N. Cooper;citation_author=N. Chuzhanova;citation_author=C. Férec;citation_author=G. P. Patrinos;citation_title=Gene conversion: mechanisms, evolution and human disease.;citation_pages=762-775;citation_volume=8;citation_year=2007;citation_issue=10;citation_pmid=17846636;citation_doi=10.1038/nrg2193" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J.-M. Chia;citation_author=C. Song;citation_author=P. J. Bradbury;citation_author=D. Costich;citation_author=N. de Leon;citation_title=Maize HapMap2 identifies extant variation from a genome in flux.;citation_pages=803-807;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22660545;citation_doi=10.1038/ng.2313" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Conesa;citation_author=S. Gotz;citation_author=J. M. Garcia-Gomez;citation_author=J. Terol;citation_author=M. Talon;citation_title=Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research;citation_volume=21;citation_year=2005;citation_pmid=16081474;citation_doi=10.1093/bioinformatics/bti610" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=G. Drouin;citation_title=Characterization of the gene conversions between the multigene family members of the yeast genome.;citation_pages=14-23;citation_volume=55;citation_year=2002;citation_issue=1;citation_pmid=12165839;citation_doi=10.1007/s00239-001-0085-y" />
<meta name="citation_reference" content="citation_journal_title=Cladistics;citation_author=J. Felsenstein;citation_title=PHYLIP–Phylogeny Inference Package (version 3.2);citation_pages=163-166;citation_volume=5;citation_year=1989;citation_doi=10.1111/j.1096-0031.1989.tb00562.x" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=L. E. Flagel;citation_author=J. F. Wendel;citation_author=J. A. Udall;citation_title=Duplicate gene evolution, homoeologous recombination, and transcriptome characterization in allopolyploid cotton.;citation_pages=302-302;citation_volume=13;citation_year=2012;citation_pmid=22768919;citation_doi=10.1186/1471-2164-13-302" />
<meta name="citation_reference" content="citation_journal_title=J. Bot.;citation_author=C. E. Grover;citation_author=J. F. Wendel;citation_title=Recent insights into mechanisms of genome size change in plants;citation_pages=1-8;citation_volume=2010;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Plant Syst. Evol.;citation_author=C. E. Grover;citation_author=K. K. Grupp;citation_author=R. J. Wanzek;citation_author=J. F. Wendel;citation_title=Assessing the monophyly of polyploid Gossypium species;citation_pages=1177-1183;citation_volume=298;citation_year=2012;citation_doi=10.1007/s00606-012-0615-7" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Han;citation_author=G. Fedak;citation_author=W. Guo;citation_author=B. Liu;citation_title=Rapid and repeatable elimination of a parental genome-specific DNA repeat (pGc1R–1a) in newly synthesized wheat allopolyploids;citation_volume=170;citation_year=2005;citation_pmid=15911583;citation_doi=10.1534/genetics.104.039263" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=M. V. Han;citation_author=C. M. Zmasek;citation_title=phyloXML: XML for evolutionary biology and comparative genomics.;citation_pages=356-356;citation_volume=10;citation_year=2009;citation_pmid=19860910;citation_doi=10.1186/1471-2105-10-356" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. S. Hawkins;citation_author=H. Kim;citation_author=J. D. Nason;citation_author=R. A. Wing;citation_author=J. F. Wendel;citation_title=Differential lineage-specific amplification of transposable elements is responsible for genome size variation in Gossypium.;citation_volume=16;citation_year=2006;citation_pmid=16954538;citation_doi=10.1101/gr.5282906" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. S. Hawkins;citation_author=S. R. Proulx;citation_author=R. A. Rapp;citation_author=J. F. Wendel;citation_title=Rapid DNA loss as a counterbalance to genome expansion through retrotransposon proliferation in plants.;citation_volume=106;citation_year=2009;citation_pmid=19815511;citation_doi=10.1073/pnas.0904339106" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=C.-H. Hsu;citation_author=Y. Zhang;citation_author=R. C. Hardison;citation_author=E. D. Green;citation_author=W. Miller;citation_title=An effective method for detecting gene conversion events in whole genomes.;citation_pages=1281-1297;citation_volume=17;citation_year=2010;citation_issue=9;citation_pmid=20874409;citation_doi=10.1089/cmb.2010.0103" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. Jacquemin;citation_author=C. Chaparro;citation_author=M. Laudie;citation_author=A. Berger;citation_author=F. Gavory;citation_title=Long-range and targeted ectopic recombination between the two homeologous chromosomes 11 and 12 in Oryza species;citation_volume=28;citation_year=2011;citation_pmid=21616911;citation_doi=10.1093/molbev/msr144" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. Krzywinski;citation_author=J. Schein;citation_author=İ. Birol;citation_author=J. Connors;citation_author=R. Gascoyne;citation_title=Circos: an information aesthetic for comparative genomics.;citation_volume=19;citation_year=2009;citation_pmid=19541911;citation_doi=10.1101/gr.092759.109" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Mondragon-Palomino;citation_author=B. S. Gaut;citation_title=Gene conversion and the evolution of three leucine-rich repeat gene families in Arabidopsis thaliana;citation_volume=22;citation_year=2005;citation_pmid=16120808;citation_doi=10.1093/molbev/msi241" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=H. Ozkan;citation_title=Nonadditive Changes in Genome Size During Allopolyploidization in the Wheat (Aegilops-Triticum) Group;citation_pages=260-264;citation_volume=94;citation_year=2003;citation_issue=3;citation_pmid=12816968;citation_doi=10.1093/jhered/esg053" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. H. Paterson;citation_author=J. F. Wendel;citation_author=H. Gundlach;citation_author=H. Guo;citation_author=J. Jenkins;citation_title=Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibres;citation_pages=423-427;citation_volume=492;citation_year=2012;citation_pmid=23257886;citation_doi=10.1038/nature11798" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=A. Salmon;citation_author=L. Flagel;citation_author=B. Ying;citation_author=J. A. Udall;citation_author=J. F. Wendel;citation_title=Homoeologous nonreciprocal recombination in polyploid cotton.;citation_pages=123-134;citation_volume=186;citation_year=2010;citation_issue=1;citation_pmid=19925554;citation_doi=10.1111/j.1469-8137.2009.03093.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. S. Senchina;citation_author=I. Alverez;citation_author=R. C. Cronn;citation_author=B. Liu;citation_author=J. Rong;citation_title=Rate variation among nuclear genes and the age of polyploidy in gossypium;citation_volume=20;citation_year=2003;citation_pmid=12679546;citation_doi=10.1093/molbev/msg065" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=H. Shaked;citation_author=K. Kashkush;citation_author=H. Ozkan;citation_author=M. Feldman;citation_author=A. A. Levy;citation_title=Sequence Elimination and Cytosine Methylation Are Rapid and Reproducible Responses of the Genome to Wide Hybridization and Allopolyploidy in Wheat;citation_pages=1749-1759;citation_volume=13;citation_year=2001;citation_issue=8;citation_pmid=11487690;citation_doi=10.1105/tpc.13.8.1749" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. W. Szostak;citation_author=T. L. Orr-Weaver;citation_author=R. J. Rothstein;citation_author=F. W. Stahl;citation_title=The double-strand-break repair model for recombination.;citation_pages=25-35;citation_volume=33;citation_year=1983;citation_issue=1;citation_pmid=6380756;citation_doi=10.1016/0092-8674(83)90331-8" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=J. A. Tate;citation_author=P. Joshi;citation_author=K. A. Soltis;citation_author=P. S. Soltis;citation_author=D. E. Soltis;citation_title=On the road to diploidization? Homoeolog loss in independently formed populations of the allopolyploid Tragopogon miscellus (Asteraceae).;citation_pages=80-80;citation_volume=9;citation_year=2009;citation_pmid=19558696;citation_doi=10.1186/1471-2229-9-80" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. A. Udall;citation_title=A global assembly of cotton ESTs;citation_volume=16;citation_year=2006;citation_pmid=16478941;citation_doi=10.1101/gr.4602906" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=A. Van Deynze;citation_author=K. Stoffel;citation_author=M. Lee;citation_author=T. A. Wilkins;citation_author=A. Kozik;citation_title=Sampling nucleotide diversity in cotton.;citation_pages=125-125;citation_volume=9;citation_year=2009;citation_pmid=19840401;citation_doi=10.1186/1471-2229-9-125" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=X. Wang;citation_author=H. Tang;citation_author=J. E. Bowers;citation_author=A. H. Paterson;citation_title=Comparative inference of illegitimate recombination between rice and sorghum duplicated genes produced by polyploidization.;citation_volume=19;citation_year=2009;citation_pmid=19372385;citation_doi=10.1101/gr.087288.108" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. F. Wendel;citation_title=New World tetraploid cottons contain Old World cytoplasm;citation_pages=4132-4136;citation_volume=86;citation_year=1989;citation_issue=11;citation_pmid=16594050;citation_doi=10.1073/pnas.86.11.4132" />
<meta name="citation_reference" content="citation_title=Advances in Agronomy;citation_title=Polyploidy and the evolutionary history of cotton;citation_author=J. F. Wendel;citation_author=R. C. Cronn;citation_pages=139-186;citation_year=2003" />
<meta name="citation_reference" content="citation_title=Polyploidy and Genome Evolution;citation_title=Jeans, genes, and genomes: cotton as a model for studying polyploidy;citation_author=J. F. Wendel;citation_author=L. E. Flagel;citation_author=K. L. Adams;citation_pages=181-207;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=J. F. Wendel;citation_author=P. D. Olson;citation_author=J. McD. Stewart;citation_title=Genetic diversity, introgression, and independent domestication of old world cultivated cottons;citation_pages=1795-1806;citation_volume=76;citation_year=1989;citation_doi=10.2307/2444478" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=S. Nacu;citation_title=Fast and SNP-tolerant detection of complex variants and splicing in short reads.;citation_volume=26;citation_year=2010;citation_pmid=20147302;citation_doi=10.1093/bioinformatics/btq057" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Xu;citation_author=T. Clark;citation_author=H. Zheng;citation_author=S. Vang;citation_author=R. Li;citation_title=Gene conversion in the rice genome.;citation_pages=93-93;citation_volume=9;citation_year=2008;citation_pmid=18298833;citation_doi=10.1186/1471-2164-9-93" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=P. S. C. Yong-Ling Ruan;citation_title=A Fiberless Seed Mutation in Cotton Is Associated with Lack of Fiber Cell Initiation in Ovule Epidermis and Alterations in Sucrose Synthase Expression and Carbon Partitioning in Developing Seeds;citation_pages=399-406;citation_volume=118;citation_year=1998;citation_issue=2;citation_pmid=9765525;citation_doi=10.1104/pp.118.2.399" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=Y. Zhang;citation_author=T. Liu;citation_author=C. A. Meyer;citation_author=J. Eeckhoute;citation_author=D. S. Johnson;citation_title=Model-based analysis of ChIP-Seq (MACS).;citation_pages=R137-R137;citation_volume=9;citation_year=2008;citation_issue=9;citation_pmid=18798982;citation_doi=10.1186/gb-2008-9-9-r137" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Insights into the Evolution of Cotton Diploids and Polyploids from Whole-Genome Re-sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10.cover-source.jpg" />
<meta name="twitter:description" content="Understanding the composition, evolution, and function of the Gossypium hirsutum (cotton) genome is complicated by the joint presence of two genomes in its nucleus (AT and DT genomes). These two genomes were derived from progenitor A-genome and D-genome diploids involved in ancestral allopolyploidization. To better understand the allopolyploid genome, we re-sequenced the genomes of extant diploid relatives that contain the A1 ( Gossypium herbaceum ), A2 ( Gossypium arboreum ), or D5 ( Gossypium raimondii ) genomes. We conducted a comparative analysis using deep re-sequencing of multiple accessions of each diploid species and identified 24 million SNPs between the A-diploid and D-diploid genomes. These analyses facilitated the construction of a robust index of conserved SNPs between the A-genomes and D-genomes at all detected polymorphic loci. This index is widely applicable for read mapping efforts of other diploid and allopolyploid Gossypium accessions. Further analysis also revealed locations of putative duplications and deletions in the A-genome relative to the D-genome reference sequence. The approximately 25,400 deleted regions included more than 50% deletion of 978 genes, including many involved with starch synthesis. In the polyploid genome, we also detected 1,472 conversion events between homoeologous chromosomes, including events that overlapped 113 genes. Continued characterization of the Gossypium genomes will further enhance our ability to manipulate fiber and agronomic production of cotton." />
<meta name="og-title" property="og:title" content="Insights into the Evolution of Cotton Diploids and Polyploids from Whole-Genome Re-sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/10/1809" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Understanding the composition, evolution, and function of the Gossypium hirsutum (cotton) genome is complicated by the joint presence of two genomes in its nucleus (AT and DT genomes). These two genomes were derived from progenitor A-genome and D-genome diploids involved in ancestral allopolyploidization. To better understand the allopolyploid genome, we re-sequenced the genomes of extant diploid relatives that contain the A1 ( Gossypium herbaceum ), A2 ( Gossypium arboreum ), or D5 ( Gossypium raimondii ) genomes. We conducted a comparative analysis using deep re-sequencing of multiple accessions of each diploid species and identified 24 million SNPs between the A-diploid and D-diploid genomes. These analyses facilitated the construction of a robust index of conserved SNPs between the A-genomes and D-genomes at all detected polymorphic loci. This index is widely applicable for read mapping efforts of other diploid and allopolyploid Gossypium accessions. Further analysis also revealed locations of putative duplications and deletions in the A-genome relative to the D-genome reference sequence. The approximately 25,400 deleted regions included more than 50% deletion of 978 genes, including many involved with starch synthesis. In the polyploid genome, we also detected 1,472 conversion events between homoeologous chromosomes, including events that overlapped 113 genes. Continued characterization of the Gossypium genomes will further enhance our ability to manipulate fiber and agronomic production of cotton." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10.cover-source.jpg" />
    <title>Insights into the Evolution of Cotton Diploids and Polyploids from Whole-Genome Re-sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"61k4SSwQDulO9I5mRWCamEb-L1HZkb99HoVe3d1gw8c","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459539","apath":"\/ggg\/3\/10\/1809.atom","pisa":"ggg;3\/10\/1809","ac":{"\/ggg\/3\/10\/1809.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/10\/1809.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/10\/1809"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/10\/1809"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/10\/1809.full":true,"\/content\/3\/10\/1809":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qLUHGTwdOmtHEjyU2FgrD3znzreyHhubVqsoflKfPm6w1T0nx8IJ5PiUfAneWVEhJ9nDSgK\\\\\\\/KkpOho9IBRJ+OSWzJaEarKiILdeCV0x5ZYwyijlq8kkSfU9OLrFKblYKeF+DgZj77zYhw4xaGQ6iUKv\\\\\\\/wnU35CK7FNM9k7YlvFWIDKKIERoCvkw+irSJm8sN+r3DVpHuCNe+Mlio0qieEnK+Twz2bS+j0dFALpGq+V0Ndjz+DHcaIReT74w+ZS41zgxz4vHSCyNtEY+9xoy8781UrBYR3mmIjfyFVyFbvVfyAo9ivLy8gGpXv6ntZRgS035fYWPVEN80LumazG\\\\\\\/IIhCyoHfejNz8X8q4CljeP+bmXfTiXxyx3uJLdG7yi9JGabSJx\\\\\\\/peyvxAW7Zcr6cigUbMX7sHPxS4n96MR5EkoWaEDI79RGcRGGzHUYcu+EH4ep0XEhqT15NufSZwCNY7Z+2epowt43RFgDIl6GXjPF+RMpg+Tc\\\\\\\/yhSyWLxGtxJhf6cIgjaobEwu8XgfhVX0UoI2Qb\\\\\\\/wug2oEoGnoCIuJWByXZOrxHbBwGTyYpAJbXSdnvuD3Gf8MXyAdAf6Q14LeXC4EvUaFqS2Bb7qfWDzOlFx9rZ4h2bmwy6XKAUS76udpR6+mMBWN3Fx6GXN3yEahwySBp8gTA7CDpYrqR+r3+DKNGBjHKEYp586czLa2b1tTNtevmr54xXSTj\\\\\\\/K1aSfLPmhdudy6pqcmixNtz4DjuUYOP6KqIQUUzr8jVO7uuNLghkGXjnvPGJ5gvW7JZYITWHPhUKsr0kutnjf0FZ22sk4GlnLg6KeZbmyr\\\\\\\/ZwV+fmOYMif6aJi50sC26Z1EqeZmwwcf9DQkx8S08P3TZjy6MOJ2ewFHjmnEpadVpMd0ZlnC0KAaqvrmMRGO1gqp2BUuug+rGZXTBQpeZKXR+GqIPBx8om4hmu5ek8FSajm0rOuN\\\\\\\/nj5kBiSWID6RDhobybKmgz+SRRkCpvi\\\\\\\/LoraQvlBsq1Qu8C5B9GZDwSHz7nfsIgPde4\\\\\\\/DhpQclLr0GhNm75Xve8JXnfFPBvSNmfWtWzCP2bvramvot8PBMGRYBu7+R4DyFeSgC64wim9cY0wfuP1rzvijG92\\\\\\\/VNqUJsmmJ2hiU3weCBEsCpoqZ9C\\\\\\\/23X8dfEmuJpj+HSO+dbRdmwoNDzupesNzM\\\\\\\/76U7J\\\\\\\/Zu4As8c5uZ2tBknfU1TIrNGBdzJ4NeJPrt+qtbuxbaw+TJbi0Np7IQINRwJUaeYeo9js7FFDl2OpLxK1I\\\\\\\/WJcpIj7CSw\\\u0022,\\\u0022iv\\\u0022:\\\u00227Bg3mOVy3TBiLQhIf0kfmg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226e7657ff97fd34bac95b8e66c3bf2b190d9259492e88f1cbf98658fdd16d1580\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022J27E0QqAVYc3LqyZsxiUS+oc70j1V9Z6GYrvBg+gK5aM49u3OiZOqZWZol4PNR6zh9CCdxE0q1fCG33BF7+A1YXDywjp5LTr8s1R4cBscJ9CMS4t6A4c24q1cbDVuN6DLfglWn6US5GJ1yvFIU2fSgBu3hDnMj1xDWow0AF68C85W+qK2RBEf2oGLdB+sddnNQEgFvc+GfhMEo082sfufbEP2CflZfeORUcdNZiMSt2ZyjsNCcJ3f+NNQ5piG1Z1D8NBOcM4j31K+yjy4muStUjJc0nyHzMlvguVy3L1Aenc2H0HwM8y+2Yb4cF3lIwxB+rwS7XNMrA1zozvznWFpDRCMGPrZszyDiaBXWuAy8dAvCJsvie5kI3VEtCNI0kuefF+Z4D7nUX50RDU9MB6gklqs5HR80FlA2V5qulozjetI5YzE1SzJHqTytrcgbvnvA1N3YYfhQY1RU+8ZQak6JcxJheZf14KY1ablmhIfXe7g5FDVKlCJFR+UI2m4Iamv9kIbYRq0BaDloY1cri62+FUHO\\\\\\\/tShg2rzaOO26pMdr7DaZFcPq9siPwe+36\\\\\\\/1LYRYhD5rwKi1odkhP5EFbl2KwAopJwoxGjahk684Oh00d4fUOwddyD94pKAxuMs492ja\\\\\\\/c8P8cZBXSMV3h3iJUNVS72LagqTeDFSJfcw9o25FWeQrRKhh4+wNi36df67GQMvj\\\\\\\/QX44m6Hv9BU9RqC6ZvcbpSPdss5OLmcg4gxid+J\\\\\\\/n+VPzssjeq0E825RMj1nI\\\\\\\/JMSYcAtfqu6LYlsXh5vtefCSBefe8XbJi98il+4E43vNgG9KlBBf61ef\\\\\\\/763e73VTe044w5u0Jy2vZiuWDosG7wzis6+tF5i+DdN\\\\\\\/U3lgX5T4q2UHScOT\\\\\\\/770T8q2jKXiRUNAJS5daMyducCgxYRPt+nRZ13hdx254dxgSfHSQHk\\\\\\\/X7Y3LEBSeZEi0BbFptgw2pa7id33eHJUSwSRoS4agbv6ZjwZFFAgWnSdOgcusrnaOQ2h0dhEJo80yZd+mmWx6pRnh7JLL0IDBk7Fc8FIExGy05GdFmWcw2RnH64lHIRSycFhoVVWXD1JMhis\\\\\\\/OQBS+B9yGNM8UmjBp4xHJJUIkFnrAgl7ktch2TLwBKLr1Cee4C+QzRQAuRjWOoE1Ou0k+yAUasU+3z+bcSF6sukheqksEz\\\\\\\/zSQLf+NI8O3JHbRBDO8IsQYktwofes4dsoFg42a96nA9ZnkDzd7jyJZgdi\\\\\\\/JrxvVsb0X43mbZp\\\\\\\/8GUfkxDisdvgxvOwWKoG4mLTxiLx1+0mBO5w\\\\\\\/AWSQPQ8Zrdud2LVrfed+h7miQkM3wxAiiiCd9WYKEnZkZqBw4RU+n04UeaRLRnuis1IUaTwF9Dw9MM6yCLHjbhMlZxKQ0SjeP4xyD639cWv\\\\\\\/UAaVfFZKBZcoM3rBIayO\\\\\\\/MTtbdIsPB5AF5TFezSylqU6q\\\\\\\/oZSHeOBK4U42uiMAOwc6b2OK7z9QJ5ViFRHTVC34vIdatUKeLm7alz1EtHruZ\\\\\\\/4BCvfYmkIELfinP2yLFoVW+xAufDLpYc0nfLcxvhOyWDFCltG0tnxM+\\\\\\\/lBYTv\\\\\\\/tv+XU2tnLGkllyRW7\\\\\\\/OXt6w5V2qT2f1aqbYFi\\\\\\\/oV2VPQZRtzvR\\\\\\\/7Kc2f0TERY5M\\\\\\\/E6NKnol2Aw4gVqMn4hj0M90SkwzrUZvyOmpyhOAbNM2lP5sbzOPETBw0L6uwjysk5scFsLfbTsCmYRHUfdz4R59w8p5r7UJtBjajplvehsdtdBUd6E4C2ysGinLIhqdxSaj1zMNEDIk4YvAjeODzqHeck9FSnOd5kiCTD+FB3PdXqDejVxC4R1\\\\\\\/LPoGLd7Jg1Dib9XtbN1r+xHHkvU61qAKk8\\\\\\\/m5sxjPPVn7wczA6vqddE3fUgBn736mgW32S6qXfXXaH3SqF+t3ZEUe9O665rAmFGZ2g+ZSjUuxWypaZsY51tg3DmAN3ZeesT9KGB9bqyVrID7h3Jm3XoV2FGJMPQs1Nf5Zta+FpmTBw\\\\\\\/LtolMoHHlStRh9aC68GXCv3aAhiJS5amsxuIcqEc9Mam+sE2hUARRl0s+1uyNM9f2ddLno9lZpDsDcvpZqWXt3daEWds5mpxD7E3CUqDWWqUAqRsH3lZmEMwAO0faYZNbQrPN\\\\\\\/R+LmtSS9vXRDMeIdDGJEwCL9qn10WNYU1dtmgiG7viX6pFgKw1vaZ2hGTOP9JEjfO1a37RncWL9xyperKNbwwNn5py9m\\\\\\\/y0uaJK\\\\\\\/mxOv9subtofsc617ooyh13FIi83glHPv\\\\\\\/V9oamWfYPIjVUnz1sklhwI8cwdFAJm9Q52tAITa5AcfyKmKZClsgWxDRKVhoecmBm2YdLd3GDJm3SaD9BFUZftnzlEnYMYM8n4uCjya2xvYwsjyuS\\\\\\\/m1QLqbUvUcCLiJG4Gh\\\\\\\/SqX9OA8or+C31M5dbyhsuwqCjiBlUJTaOUHT2R0\\\\\\\/Ob751b2kPeLnU8rv4Hum+vLgQGTGExnYu2zwuz6q0LNU3Lc01OHCd5p3NW46JKdq08xto+4qxdDT3Oi5qoVLUtilgU3jF5EvMRsyCx+uax5sLtns95t37enMOrcOPJv52biNszDFKyIDzuOi8xottCevsVBhFw\\\\\\\/2brqj+CqSLZX92JsGEl+PuJsv3Tw2E\\\\\\\/6vhTg9pPBmR7VRFMRmVbnnkN952mOuU4L7MnmkP\\\\\\\/7Jg6C7idM5r3mygXeaI\\\\\\\/Q2FZOce9+XHmArHsrC9\\\\\\\/KeHBn5VrS5FzLRWDNFIIT\\\\\\\/5a0G\\\\\\\/O6La1H3KVdjQ3mgpfNImXZSOvz7+arJcj9n0mFlPUr6pvOhM+6XxA\\\\\\\/2zd0bhctXqEBn2l5+QtvhhY4iMr\\\\\\\/tOM6k25aVfruavFOL6naXzIIzthgrDDKNq2jUGkmEJCGUXlNrXP+QK1yOhsj10Ecx60OKyJ9RxurSAVN2tl8KzIsqjP4r+IXXSmml0M0Ow2TyEx4+YfVd9YNumGEqplzoAlrvtmy248\\\\\\\/2hY+q4bqNfcGcP3ndFABeudEHToi\\\\\\\/6Bui8l4y6wvEZ8owpME5xvyhrrTSiga4Ctae24LNOZx9yXgHJ6NGzfJHEyZpXIvqF2QkC9oappW35brshglBm9\\\\\\\/JCYfUVgSU9XOMAk40lS1UkBD7GVnVnu3E4LIDgFz3\\\\\\\/WqpZgHYwPFaFiAGKUiHKX1uiFw8pwNd3p8Xas8jNdw+Lwa+RkjDLucSczp+Lj5opQpTJCOwTIheobciru5YbDZxAWUBJjfFzzlsW8bpMhIZknIEmzXO+1GOBW+QYyT1XFjTH5XePjkhCVISirbDl\\\\\\\/jYRmAtwi+9u4c+ujh75dVLQ3HM4iLA8XZ+JwI5U4+DCtEwTGxNpnnJQXJcSz+jP3rnJvNpV6JiP52WB5KdVF\\\\\\\/sZJRRz00bU86XwwUamywDfutIFaTrF94ZBxNP66bKlEI7Hp0cmOPMTp+WW4yqwrTDMbEc\\\\\\\/IGBBoYUEAmU\\\\\\\/RUVjsdhEdxDazn22L3umi6Dlg06+\\\\\\\/iG5cA24EhHn2E4+FaKXmFH38Sr41fCnx4tfMjOenA62HqhHNqblI86Y1JZ0ViuzEgH1dcS7dgqHoatqfLSq28uIfGQd78g9TSUTLLN4vH+pYatzzzvDX\\\\\\\/Vcu4aGkoM8hjtrvmGXpOlsJu65DsA\\\\\\\/rfwwLgXnEebnZgrcqQeK09YX17wfGljWwQ3xPgpq\\\\\\\/WJpRVdhxGxz9BlluDT+QiNEjTGhtacQSeHI4QN6jmvoXPugM2jJQHSv\\\\\\\/hyY25+1ced0GuptEoFxfWcMVRA5e0RZeRv7FsW8HWYKdcKvA2lQit+6HiAksGRHyed6NUpNoLXQnaq\\\\\\\/2rH8LFo7ILcO1U0sKeIPDpHOAbPuFxuhGONuO9nU76Q60KxI5QdcrNFpsXFTvs3KshL6KtON3hgfK7IXWP0BAbNNQH0E9RSxHbULv8GSnJfgRrQsZs5A3nRy5Fq8MuV9JVlAoLlVVkGHPsZdWeVvx81U2eFpVLmkTki+LLLZmDQJXI0X\\\\\\\/gkbVMgRvvhYGUk8xfFwA\\\\\\\/HWWD6sKntD+nqaiyq01vvuJ4f1t2TYKdiJqF\\\\\\\/MANDyX9j4SZPkqnb51M4tALrldRa+cCJLZZu59zgT+XLUkGBBgn74cTri4Opl6GEBQcMHzLrIcR1HU0CuuXA14M\\\\\\\/tE8GmTeooo3gvyrJfaCQHiAvhU6UJRfamFCB02mGx5zhs+njuMDkhrKWx63cfXw1cfPQIT5hf6lAiw1pTiGp8\\\\\\\/VXbETXnhaYhmanxaxRch5STIwuydw5GF0mkuWTJkUuJpg8dLPOco6scsCR3Fbl9HXCY2y9Ux21Qd9UG2aTwO4hOAPUTO0TTtHEonQTvPsyFWVjDfH98w\\\\\\\/uoJVW0IOMbqvENm1WkFq0K8pACVG0W4qs9guTS5bQTUJZhve1PlejssqCYKIb72IE3B4NjqP7w0+kLxY2ZK759OO7zjsphsOcpW\\\\\\\/xZCSE5Gj9w3m6BNgwjIHU4gdi\\\\\\\/eXanU7sgNd9vtZ8FBVKkWUJsM6mYJ2dNh\\\\\\\/8vbsuR6oVU518YiN7JNYac3uT+x\\\\\\\/cAiTBdqUsh\\\\\\\/guhfNn9sOuh4l+CabSeRl2\\\\\\\/dMBh0UapwdOCEOM6YtVGD6E0bnEZ33gflLZwuG2vYgHwauDqQiPapKu1bjwTfzYNfCJbfnOulPp3PzeGRr0V3XYfLEs0TImX5XXtg2VIJmc4CKmEfiZzBk4f8H2IZmH4Ddwc4HxeiybniBai5ndk3LuoGOlwLAstDDlMXLllqHdQ6z2TOBYesJa4MajboXwlhaNaRiPZiycz4XaUh7JxHyPCUCSXiSz+cKZOP0LCpXtFNjBya1GpLHgRURwx2zMRjvXmh4v8Taq57bQwg2aSftcrWX75jBttW5pnjoGYTse7mPdzgdNcedmGRLeOpEv1duldM10l61ismFD4ezMKC2inUDvp0OYOsZ5nIptMdPFeehCQj4VVbaTisxoQEhZTJs\\\\\\\/ia453Q5AqmNPGhljIwU4BZRCzNMHwvMxiI7Z+bHULaJPEPT9OauPpqA+a0R0FezLhyXIOpe0Cxu0xTw4DqOKPkSyBYKgUsG+UbKceeQL07RgBAUEL5gd2oSJa37zlofFQrAkWGkXe1l+HJ6YdDN8kQYniAul1TjOczOsE1+DdMwDzW81Pgo3M1gS2b6JuGtSJ0qSsHQ8qAH8zEdfYFhpyLdArTS5cX3+akSYz5txBEn6rtY2Ca5hQR82QeoSu7QeuMBcFDDXnf98qwEJKQxSMfh1fsWB3TbbjJklT\\\\\\\/7u1VTnbBB9K9ti7KVPOWbhUfDUhvDhAaFFXDMovG5LkDGfGHKOswdKv5LBN77sd5xq9qyPJEWXBUC71tGWeA9E3MAWjtcuKfBxfjOeUdZwAbbzN4MG89aYjgGFvfxfuPUa8Y1Q2z0mkovjH4sxGNI4OaDc5jeDRcfdqpVBNoiBfPf23I1OAY3nwYeVT4qHwt0435s7Z3V99J4xl9vcYOU\\\\\\\/Pk5nfcnQpGg6uSp5uB3Ctqg85eKMXcwwZ8sf2cF0Hg+Ue2RuNyCm8SgIHgK1ah3K8y7Fpc5bDcMget6cV3TSNHQC2rXDrr9jdgK92JZnbMdWfCIoXzvO8Dff5EddVRsuRrZnArDF5\\\\\\\/HzCAWeofQR8A9L+5XtuKal0TGKK26TQcEPDznOT9\\\\\\\/0UlvQZ0zkRHnwe2uCSsDP9ZHVxHL20iyGiQdg7MLTC6i0UWCqc351tFj3rHkgWmzVMjZn\\\\\\\/3JBjbMtFPes3mnsp9qn6BdaAJC6R88US+8P\\\\\\\/R4LYdfiR3QgbrHZi+tEJXqg3irpYlhIkITill506UnEYP2vLoj1CVyOn2H37LO04W27WvS9dy06R66uiisol3YlvNBwV1x7MJQnDBTDQOTFrzhQQsIw70okuPcWnfK+Ft0Pc+GkyyxsVjYLFfSUcYf7CapJCHA9KjaNxnlkEstnXybOyPAuhaUSwu9CrCVLq6raJH0D4FQC1dFA7+0bNlwg+qRoC77Yh9mJJO9uCuj9kiYDp\\\\\\\/L\\\\\\\/ZZYtmldkFm+ncQKy13p0A\\\\\\\/rzL1nfszNwPG7C+hUrkv8UEQzuP4RK\\\\\\\/Wp9\\\\\\\/XWm0ARpfwbBZ4YSDHqVN6yrnFbbnaePJMQpfwZqJS5RlsL45\\\\\\\/uBgmm2K3umIi2RFk4CaikJnU\\\\\\\/9CqNRHAU1hKl6fiUY67AMIyPVdp3lcIwpkn8ZJ\\\\\\\/d0z61oyFg1jVTageT9w7V\\\\\\\/9Kmsa4AoGjzKiu4CqhwNcuGHm8RxomSXaWZgpmXyR+W2H771LKhUuZo+vzZLiHgu4W1shhkbKZnorSos0SspuM8o6i064dN6V75SWQA7spCLOOQez05pXj6dpyTPShS3fzKjyjpnpXJPQ2WVII+5x73coKlR+bWt6VQcLsoKkq5yUU1SOoTgu1y3t6ZEeZUywvODJ0UUprUgFmWDs0ObvSFN+pWUctRtOu9DkAIHdFeAvtsnm2nIdvH4DmIGE3mhA6gQFzfBpgSG6sdkrDM1BqW\\\\\\\/keoMobd7zdDUxeHRGLsz8Z0BzOw1lqay\\\\\\\/52ioWqGUFbhymKmyM9XAVjf9cHPu2As3W7SpFOwAtYD\\\\\\\/RQWx5WcXvNA6sR3ckwKfjJeDnHfVs2dgwE\\\\\\\/zjicwCCix1hXU2kyx1yIqvcBnDRCYzPtGDFiaWZSVvmqe3hFGL\\\\\\\/4UdC1PIfdsDVRsm9I9qYANAV7ynZOcqaLhgw76H6hzdnfOIKrlFq3JipkM+IqUVtCpgWqRPyUcAqDWgdY\\\\\\\/4J4cOLttUSClKnZ7Zjmey8rL1fgEbnDz+a\\\\\\\/j2cMPN6ZyRE0bfh6t9jYGkfrjbOtpon\\\\\\\/LfJJymSTjGe53usbKP77YA8FywDxKM\\\\\\\/ClnBu253M1YF85webYsMogkgsbGCJCY3D5S6Rz1RuD8BeGwK5w9mFX8vMqEkjFw6VpCfTQfh09N4tONvF8shwh1n46PQ0CklMac1eEYYZ42q3Ax9KH6EQCsXW8Olu8vOtRPXTxYsLbj9fSdL6OUObIZFtZT\\\\\\\/lTOnM75ulnU5bQCtH+pOHalyc4OWIKYxbMzJMg0S151+1l9hZ7U58d0ahXI\\\\\\\/cXwrumO0A0TfR555uD\\\\\\\/c6Eg6p+oJOHyWOZmdOrBur67+lsMaw4SKJjoSL+jBmBT8rr4K0ApyVVTP2hnDut7ABYQewjA3UBjCY1iBDzPZCmtoAwe8LRDlXYabrWLARIQWHZqEY7zqlMKQyhzufnQyKOaai589E37pn6yll7m9usDBqGAcetydUaXzYHwrDZ6S6QbJzWRCqnbq13q94u1s\\\\\\\/ZiYgnJeGhqrCyTJCunhQIzKoG7\\\\\\\/cYqOMj5uM0alcWwXH1GhGvL65X3H+MmjmIqHDJhImOl9rmy+Cghpi\\\\\\\/m8gie9Iuq3eMIBokx4Qorf0+YSCEw7mJMPlgJsk6hzqBrptf0Y7MFB65BusOHPjeM1dOUpOE22TTiV0Shq4SrMksbJw0B\\\\\\\/Mb75Vrb1149zA4v5W+Cct1iWSY+fa+jHdAJIOf7J0CTYPFz5whSivO39I6GrR3RqcDmSBlF2qZ0zNaX3oasiqQO1zC5nNKmQ6KLhDjke2IYTiSzxfYGviq3iv\\\\\\\/3Cx9NqILOSziCBLuuenufv8vqQBKrGzRnSRNBvZxYwyPzR83GYBtFV+zh9iLkG9Lr6vVqoHi7aTz1YQAWfK7mmmQ1ObYBXe33cytRpY4PvorxQUbosPFlJ3h8CB9ZLaa2RVjSfDwyWkLheGvPP54ypRJXVdOFMcaq\\\\\\\/1IcsCO0YxjnpvQZ6w\\\\\\\/KwDhHxQtll3cKINLu0rzb7IsKvqdbh3HfJP4mpNP8BukRZEntCzgHJY31i079MsNsvobBkHH7vxh406wIKGLKrG\\\\\\\/sryhLWswlcbb0XQ4GunzOLR9whTRJetnSJLgFvFzexrZ22yo0Soazpy49di0z6\\\\\\\/dlUKPBx\\\\\\\/SVkGEtsjDI5O9Jwzmb3q\\\\\\\/TIhzcBtthxSvVZQ5LGYTE5F\\\\\\\/m\\\\\\\/X3m8ECYpNE4TA9cZoj4sPcPnKOHY8dspdXAqTbMe7US3XKZB2TLoBjxfEUI2Jk9xAqkFIm+5KE6LQIPWInSBus9yTNTqP+u1a2G\\\\\\\/EzkJrdvHDbTvDn+md1+HUou1zY\\\\\\\/medNx8M5m3P2C9OGkkdANWL0zNRllyKIUXUWFe53qBHHOmwG52R1j1h0tYG2BHbhPFPSdNH3OFgrDaqM0fWdM\\\\\\\/W71aqAfw6bqu9RFMMgINQMfNjgVCn+UteDiaMPEXPtd+o9cowamFtAXpXmZ5u6o\\\\\\\/cQ3dz1\\\\\\\/Xk647TrDnxeUZ6dVbW9y0trTglyEENosLTQoviFngIdTmgF98mHGRP9sKSzpa3TgaSQ7L1L4OfdM4FEXSAKrRDQ9sNIwAzPokHA9OuMXTIXlEkAHuE15\\\\\\\/fAUebsRfR3wt4lNO6SoAyLE5gIWjptbCMLHqI6d3QJEwFsNY9AbZfGjAsiG1gDPEuBMUkTmDHSKiK41vlICmN6LikFATojjYe4oauhaSty2F07XSrB7hkmN2W\\\\\\\/tVnSNvGiHFQy4edlz3Jdh2z4dg0y0mMCQaxsL+waPktIc1ruKJoHbnHf5He9\\\\\\\/X3BSEKubkzJKbByYNHvpAAa9XyOx2\\\\\\\/yWAfk0UWz+kTQQKxByoo38rG8jgVZ57d71HVH0qV\\\\\\\/r46vMHWX61v9Jb6Z2zc9T5+eFBn6Xt8qPuW3ueC\\\\\\\/6l9XoTJuwiVpx63DM015Ysi9SaCW59KhURpFte6UiB9JlaBdhSnxPNGh0gs2c\\\\\\\/LpFz1dhMc76zXP+veFkEKhnSEpY69gkCwtu8jAGwi19hDuh0idJt7JTiWS4gXGpm9dUEk9Ii3zxdvIM93jtmk2f57GP8mjrUWW06MjKgcp6HKh4Vl9aVo+acoYFcgNMt3yn7w+r+6uHT\\\\\\\/iik98kLuEZ8Lp3vmWubILuIiTBdybVteAuqMM7edzxKD6LB9BkH1a4hj4OxhBUQlO9V2rrWQsq0DmW6y5+vmvRcuWn6EdM4I\\\\\\\/1lCWNN5+gaxWCF4lALUF9D\\\\\\\/qgOV2C8XRYREZ6SiTUbELLVjjD7meju7cJOd7au6gMqEgjwtb8lg75OKGHGO+rjpGkwiDzHI2yBLS0zhFRPPdJ73XrIIVWlhMTTuPGGEtOoIEb1hsPpmkPMj1FdGn5giJFjPjfKEPQAF7sP+5rXAhtzr3eJp1w+vNF4kuGm9\\\\\\\/AqHHkgeQ6UfvlpvVfBS4bKcdbZAsNySBxQTpSB3pYE0lVadc\\\\\\\/OjPUjxSFYL5g5X\\\\\\\/q3RO4KXSBOPzDLacpvUA\\\\\\\/sKgNE6C48pzDc3aRqSGcvYYdtR8Z74rkPm1zRia9KEvsQ7xiltM9r8jvDfHhC3YBK8d3nujzzXwc4VLKSsAYoXNws9G8U4ryAXsuUkQ8mcQU6EZYYkLpShBTgZ1M6YdBMr04D2GKWFB6XyPIrGXvmFXaLDVI5N4+AJaYiBzIyg69Sz1BIrAihHHH74fXqwPFmQoB2OepvGVYipxtWTuCi9CVgj8BlNueiZhiCxVuAUaVj72zJZPxDvbyshhdUfzgOWC\\\\\\\/Vtsbw5LyUioPlblq7Rq741oGnNbvWramsBdjFe4KoPJCxyzaPe3VK6I+6Vm4XzBgfbBPR21MI1Sgqx0E\\\\\\\/HsUx6tuRi5Mah1SSYX75cQwv6tCcBSlS1kowRBBpGmLrmqDex86UjN1QE52L6ygMYsEHAmuhH6+74zxAIBvDOexYIOgUMgBd5fI6CNGSSwZjVngh1iNOUNoYV+DOMCk05+Q+97S\\\\\\\/\\\\\\\/Cc47cdFbGmrPRBRnGB497a9Fp9iEBy\\\\\\\/1csX5qEKJR88k0nGnDZT6c\\\\\\\/06AXmdRzEwRIiQWweWTIiPcELnWXTcCgyGNpijcceCycXpAh8nY2wyMWoO+3Rjwrfy8u3LVZFP0vpJNINd6Lh6UeWYkpo2wlWRPuG9pbXMITjooC5dJcC+ta9CyU4zirHE8UUoEnxeGWtqiVyVb1jQyVf27Yn7jTm+98nxu2mRM5vAvpZqsB8DvNWMJtT5cOVel4Z3xPbqDrfZWz87y1+hC\\\\\\\/6+DmoDPySKJY9j1sjY9j4374SFKkYEZijiC8pBx3ZqPxpkQ+g517Cq0PzjsAb+rjrZ\\\\\\\/GbO\\\\\\\/KJolrPRgevnXePFa75N0m1iBqkp004scJsJqp+KK5hkmWH0fv+HylW+GgPwj\\\\\\\/32MKUgf0tLpbMENHJlqYt10v5YHcbph5pDm3PXIxBtluLePeTmrNtBzCLNTiQ1L+cvHj+NLesm0mGL5m5DjMFg0gu1BTYT2A+jxoU5EPzidk1F0OsyxqQD7lQaRkP544U4SeprTXw8oZ6gIPezibwV9JVWJHwsOfahpxZ56Z487z+7KWHFQjdXjpO96fH+LpRXi1f8aP\\\\\\\/H5\\\\\\\/YgUa+xl1lkCiimiVl49FHC7rFchwqBn88dbZ17M38a1YcSXx5fI\\\\\\\/uG28tk1P1mxfqb\\\\\\\/W8ZZP1TyiotarcRzjHLCVKr6tciy7h9GFs\\\\\\\/cegZsr66CqRkkt70iZnSbUMzzB7ju3nvgh3GdntW9La69CJhNhIF05AaVNeod5WbO+dWISFdgAwPK9HrVkQAL+hsvAI0JxX9uieQHDSemDGw7bzlg71B\\\\\\\/DgzGivPRD6v7psQNT0kIUUG1CfX6d2egYtkBAkOg0aYn0l48KbZKCEUlwQlFSgdK69wMeThKHrZlqxtd2KBXUJQSGiBe0Ct+tTR1KRc4ZtcN0V6JLtoAN4urpzWlq66WvVdtjI7X3yQO4b62SsACcGhfd+MX4VdGeCNGzN8l9LJP\\\\\\\/8GfVzU5zCArqie5IFp5TgT4NLEiyGdw+BfkZ\\\\\\\/ahugjOkE9MhosTtlfMWCmc4MRRnOTtHxOwaO8z4bT4GQMp4MqOHCiCSm\\\\\\\/E5h5u48WW\\\\\\\/U1ng6tgIh5qm9jyTgdbrFPW4Mmhv3Xij1SEMQeposfcOl93rBULSMt1d0Ea1Ljf5BX61ubTu6Iq7AqD5rEhwx2o4OaUIt4ncofcO4x6sUShcpy3yWr2ew1uIkQgMITiJKJzhrynEIHBvetiUCs6yOy7zKk2HoRk6kj94kS8uztpf2vPd9qq0592BlJ3EMtN9+plX4IbZY9ipxJAiG8q3PyMLo3\\\\\\\/ORUBwmRset6wzGEtKTYiaBbmB6aqMr5llYkXETPNPq2Vy1\\\\\\\/R3nJWCgSMwmAHHuFJGneah3o6sYxfyDPTxGrL0nXnEKCqUsLfyplFQ32PBbx4ExRGTfbCAXpE3IxGjNn0Voj7s4bmBm93\\\\\\\/dWsgxVPqxbxWJPNj4BhJEMuzzQr8FJBKmUaiA\\\\\\\/W+4LZfvlj0kHzcOjsOgsoMsLp97BnJMG+xMIOkJVsNtfIYfPrpie8ouMTLgkqj90nDQLrhnuWrhc+nvvy3q\\\\\\\/uLRdCEnkM5Q37l1A0suUVrJkQ68ZYG0vADd30Z+WfCvhEiTRflNs4wUAegu+WSifFuOuhBQRB4DsyqVHx9VsuifZn954gmOOWSyKkOHfM8L9yl5pldoQ3KOmy6Yr98NZSqDyi+nXE6UGqvkzFTL2+az48KPN8o2sL17dOCxmSkHIgm5QxIB7CrCeQ6\\\\\\\/+rwS0nbMCdET+WMNg21J9U1TJ0ibLYSJV6A+EOohdMsJA7VTDRyFGljxWZyEIaD3HNWIVZoMjd09rdcCuPKwptxpZBVJzFxKmm32QmcPb7cnU+AFjHOaynfn70lXqfSwXIBSSMqekEFZfPs2wArry7D9yCAo6nZZoefJXME58dUR7Wf4Enp0jgEYQqmQflPxrdHyGkurKE13+Kogo1oMdFqrKJ5\\\\\\\/oHD6ivMlY1neklcXaJa3dH9Nak6I6sGOk+pnxs7s3TIVMNMMP9QDQfM34NzgSjsGcv4Z7FAyIVuluelXMREHfesRAk9\\\\\\\/lwZpjWP+AC0aDT5UDr+JutK4QhLuezTyWTdBOFEfRSzYBzxbE9hGCUnZ6f56JK+ja00QzUssRv3U9C8mY5cCVpcNjlj3FU0v3P\\\\\\\/BMyllBuPg97kDFFN6iNMllAXnls5TEv8ZmluYeJH8WauLc7JiPGifU9sLhZomOrg0H9oi5y6CRKYIZ0bBwyp2AMD+JtvObD1sEQm8vf5fHy9LVE1HO2SrhBh+FMwHRHDFJUY0hLgdMyrCLC98Xv\\\\\\\/iOh0tuKq04uwfLapB1TZdBMIwoL5AMI8Ac44D\\\\\\\/l8fe2z5bKNC1mrYhhgl9FEQ3Gy0DGnDw8WKBCRcixqjtTCmUiJ8d8nS66WXyjxkgV98+QqyyybXxXR+4Jd7Ie7nTAXgXQCA6ZijMVfvXMKJq5nx9009T4qGzJwiVFOSyJLg4Yfbm0z0yjMXwHdrHa7Ruovl8ODch1iVvAoGEYa+Bo3yys4e4PsCyhV7Wj8nd1cvPlfJyHym0f1P4prPXhlfzpzJYmAHXpDXgvtcDxz7xZHQorVzSDksCs20ocBNMzNPnYQ6PypGj7i\\\\\\\/WzviIFliKKWdREoigZClJQ87rD1QHUVhmDTtxpBrTo8\\\\\\\/c+7prLBeUniqUe8CSEyr1EvW9QUUm7JLO50jMAjn6GsCV1hpmAQu9bvvOOqYRxy1v08986uw27qjYJTly7x2QjOPQmVaDDWVL4Z6cyAMWTRANkjajWvqKcXmHEcjmcpcsTpMR6xPmBPWA5PCwA6R4AJ9RT77WUmRNER3Uo3Vt7y0UVNZ7mCsIS+PkAENTa92NGqbtqhr1b7wsOMfj0vKqVaYOO+5uZJlv4jph3ovQy\\\\\\\/9opIuGpF6Tu7wAB6T6fu9TXWQlVj+xNzQ7gWv8Mji87FwmDxsFxr1ndCkGBt5c5sFJKCqsbXTm94vO2Dl7Xeu7dBLwulFujmAUJGEHyNUlUh71Vb8iFiULwaRReEU8sO1QfL9xpiGnhtLNZPQs2vgX62L4FwbRHh+Ts2GBsMaSZ6K9dUXOV2HlPTOI+vHStt9gZC8nxLbJYHdGgCKLazZRGQbAhr7vSYUNcjBWPXmWyd+SXvK7+kpFIPCdWbbMPhcY+n7RgF6ESxmBgS3E3Ph2wXV9i1kIgWk4np0ThoaEdKzpOSX5jDKgi\\\\\\\/TW7ZXEVTjYeqzAK3nC7EY3DOAiNvuOshK6Z+GOwzWBKr3LkbzS0cTHxYKM6GrhxQxnIJJrds29edMj1ggtLRahdMvlHMZPHk3bsiwdTwmRh34qCYOYuSCjKU8UIm64Z48Zw6U6YB8nffN80Ibi1p5hF9q+fMEUxhgxrvG82TV4MWtCl2mebh6Uj0rCwez+vV+hI\\\\\\\/uxxCd4GcJAWNnV2FKuOfFnofuwBJuwg3N0jp7zm7DkfL+lsPTdW+OUqNpT\\\\\\\/D15a50jUzJMQXnt\\\\\\\/lhn+PMgv3xbYqTcTQmISZ7kQGbRSC\\\\\\\/+rbkdMIA4TSYSmW7oTOeQT0xK4DXP8vdDB7Scp4P351v5r0GIenHskH7uAh\\\\\\\/BwhJajrvo8iEbIMDlnCFDmjDK0qQ+3x9z8AhlrnMhv4cTbQhh6lGmiPaaqms3TzTaZWRKIOPKnjEphJqez+GibkQvD9ddd+jjX8c55zc+nqQk4\\\\\\\/is1FvPJHiv9mhGfpRJbtmrTLBmfF5fpvYHXUukgGC9xLgaZSZFpExjT+bDMAQ8Gx4c25OXUS0WFpZ8fNO30QnOxe6Z09kaU6\\\\\\\/I1Y5Uwj5yekngNYUzDk\\\\\\\/ArJanD9GdYePyHg8khInM7YqzlmauziJWOwDZDSm7\\\\\\\/pofoMVanxzN6xtHcSs6tjdf8GHu3s41oN00CIXh0rr2fGDMTqs7IRP65vQ1Q9nKdGlOK7Xw5GzgHlEtcX3uHkHMznv5lyu8FzOU9fAGJefAVeindr15MpJBVnwBEWPeNWOH\\\\\\\/5LofC7nPXeDlSO606Kl3E5flCfzYKPgo\\\\\\\/tQxgtWnPXLOkq9DY0zor7QNmrInNemzkH+BCG3yye45TdrdDeRmz143Q+DIVj\\\\\\\/r91ylQ22so\\\\\\\/c5mZRr2j7WqZ9alttBLQsBjh0swGgHGERjIqy0m5TOpuDhPywZp+nA\\\\\\\/0A37z3juBOllNSIfuxo3dHi8xU0UNes66FHhzSqkUdN6h44N1mPvdVZ2bwCDGbUa7oWLqH1gBVD0lgUlIUtblvuOG9WlDOBuNlXTK+g9uPKo1N3bySVzCozuCp9PIKrfw3By7I8RTWiZQr9sPAmDBiwmOlRogh0ElgZ7n8iTK8AyO9cQzu9LDozVkXslZ4z5bCHwA9by8Lhf4MKHuKuNFiSj+KbarId8qSbNkLjaYuid33WdaDs9m+fAvTnWl81wkezTJxoU6Nsx3npnDPuy5NqSNZOVZmKNCkjitzs2pjdKvEemFFJ6\\\\\\\/8mBYBO\\\\\\\/bMNZcf+1jB0UecNpM04GJSQ6guIuhY6SEjkOBzODfdCKpQmeQSopzJXc2f+Ry\\\\\\\/qgUD2SdSL4cB\\\\\\\/bOmbGzSBTTXhZe4LlrEfxKVX25BSDI\\\\\\\/YFu11zHgRvlRPpQ0ObBX4cBHOsrl+HFSCfT6zmMoo7AEAyKqXc3ThI6TK33JNfsC8cQRcFt7DE+t5UxzlfINzuCESymiVdS7PW9ITjN3HDNKl86R4EnpTubX0V4RnHnkOqRNYTPGywH59DTUJKV21qRcchjyaAvMEH2y3b3xiyrfPCN9R7MEN99IVWfJYE6dO4Gq\\\\\\\/IxVQceNfNr\\\\\\\/0r+cSYdoniFBwLCiTFZNoLBRqtKrvjFfm4nPid23fSwdhp5RSxtiGWoL8R0+7mVR4845xX4dckoy926op\\\\\\\/w74rKx1yDvqU+c7Bp4Umw4liHJluZQ1GYGu4EKb1eAV1UJaOByMlp3HXDI+SSY6QL3KIq5\\\\\\\/Nx4b1uF2Sc0vHd4nZPTQU54u8L5WsE\\\\\\\/yvuM0gZv4OZBf0uKYKWg\\\\\\\/dXl3TZdwzR3tqwaksaDA+6bRGi7Qlil1HX\\\\\\\/PqZ5QOxpz8wnhFdAu6mV3mKayQljZOhoRNqGDtlUGPIK+AhLcfuWZcy0st0OG7PkstHMUODAJ7c7lZU4ty8QGnOmuXkB815KDXe64Wqu73RxqXAovTw83FY2RbzJIBrZQOuoLEiMNAOk1lpHyVI4t8t5gkrEc\\\\\\\/lLaR5ofr19jpHHM1lLXvMVRb0Z52TT89tgVtH3hkQAcF24VUskQpMfRIgF8eF2QAI+2PSy03FN7BAK6bx1S4wwVso5uTDsewCuuvzEFgbsiKyw4uhLYftIFDr0eJvG02Zv8miZTC7O\\\\\\\/KvrOv5wePoDHTJN3+JqjjSDbgTKtv6Fd\\\\\\\/Gdcj4DTaRN4BE9l6H8uZDDtoJA9y+llFrUwK6DP77GybgRn0Ed0yo+BmXmMsgTGFco757E0ojOlCTTc2ZGoYpGba1w9laNe3e5DeK2TykUrF8heaFKuQi4jJc9t1s609ZjmmVvATUkljSLr04jV5srdB9fztbJMK0C7yOE90uZB5+XEqTPFyx5EmEb6gtHNQLW5eWnq7MhH0xbULrqRMeRLPs912BFyZoW97b7XkvkD3CkaOv1AXEI5bssbjtpQc7LwERUc5CcXS5a05C9Xh1wu0A8+auTId3R9ejo31XWHc4268jGSEHG8wgfwhS9Us7jsyuYiKaDUmRyWRawMYmvjfTpKMDI82zFxb2h53UKjWmVlOjunJsPO\\\\\\\/+xBbJgCk1ZHTTpOycZhkcPpM\\\\\\\/yVj+tklGzZKGv8pzTGYqYuW\\\\\\\/uc+7jr9hJx\\\\\\\/eAzunDEFs0li3NDc+tykJK2tGHHl8KuiVsbKlnsUCzF06eI2v0fLjhJvihTTOArHcHe7mjIzQlBMLG8PM47dUwXDCQ2BYoLEfrJznbHZhU+Y05MWpA4fKka29Q4D5q\\\\\\\/9CN\\\\\\\/NCMZ7aFmQwP4ynGCAcwom9tKAvepvmvVnuY7dKvZQX1KBgsoVeFhypbXMOPhkd2ShnKEglSFYNn+T4uNEDRJzrmhJmKwB5toJp2iKmRH0RDWUTa11cRH+OupTjNdiuGJRtVEVyn\\\\\\\/QMaRqn5loSY32OWxpxOu+m9muWHBuTe6zhWisnm5QE3JgE2IvW1uYVRpPnrFNDcvdnHLOvS+TtJIkq36NeCjMpl0dn67cBbjgRBER4CPtHulB\\\\\\\/PhFDzEA299J182giFbw9fQyv3OiB9LiOIAA8mQotrJoAARqnxHmIRiEolO03hm8dLDbB0dos8U3HhTY1xsmY8VHZZ3mPo04vieISqbQq517DJIiXL8oj3Yicv7+Jx+RSRslzWASxA\\\\\\\/qy+2nvK8k6KtbchOm6nNh+CYARYEX+EssKYmk2KIWjvJyPq4lhwiHHH6+7GpgZG4193jmddEqaMxpLIaFQoTNXn6cGUcpQI7kYJ\\\\\\\/qfXVay3kjrh\\\\\\\/9KgeNyBravenroY62T\\\\\\\/wM7Ii66dGXyIxEF11lonVeiS7DLFHl6MjM69YCrlmw9JS4sSZRqOBtUV6XY8WrKn1jN0+x4+ZP7Nk6RtRBrgGovhEcGAfqLUdQvpFzktgo7SEXkx9uEmKHvnYZJsZ8gi31KmGYfCTot495O4ukPbaeCvHpqjQw6UgSbDOnQQ41raX1wQiDNmhV3R6LTQSrDOG0ppGWPPpLUZOfguw0hef5jC\\\\\\\/1CJ19mElS6eALVEkRFHWrHws\\\\\\\/oeiNxioay0\\\\\\\/Gbe2zsL1BHdsPfIcr4mBapfh1BZTQSPx629HG3ObcMcaa20LiZ9VK3i\\\\\\\/cBaKA+nSTANYo5JAhbxePfqVHz0CUBICBoZkvdG1l8g1Gx41Q0039HhRxa1hG0z6WAeLyrS4ZMKTkknoWAjGlH0VoC+OBJGPiIOwuuDH1TuRbsmcKEcXOiYb2fRekpooYdrLw22DI+47v7Ah1zgxbMMDCZXr4\\\\\\\/PH9VdKSQKtY8xNGRY+gUDJEAG+7\\\\\\\/n+\\\\\\\/zE312SqmG+nqLKcF42b9+y3pAFm+z+12yjCUKKFSkDdb26egbUQ1idjAQno771\\\\\\\/a0wv3nfVYglS\\\\\\\/vwqXYgiBF953yLfRBlLlhME0Omst1lPYsed+38raiBrRA1zAQ7aV3nSEBgyNIzV0LGduV9f6Dd7CUo6FUR8s8eKyFACdBSHReSVCrFRxpsryCfeK927KLVGwetC94twkwkKUCe6faKO6FTRYVRAK+TyhiUIhVY28h4jdIT5I2qJKk3bAF+E5ooQ+Sx5OQWKz7G\\\\\\\/10lTOrt1Yew3sWVfAfbJFe2CPoreNzrgdfBV9CDkmF2hko14f\\\\\\\/1JuuWZOC4gTegkLJkewKtzM7wbVfa3HvdJ7TBP6EOQbMu0V9Rke\\\\\\\/CKKf54xk0Jr5JIzE1w0\\\\\\\/pJQIMDP\\\\\\\/KZP7q6D3k0izazU5C887szkPgMYc6r2nNvyaMZU0aGYiW4dpwRVXu9T\\\\\\\/UORSWN2daRlPs8OEUc2E3RNZs\\\\\\\/tTLrtuV1QdUgmL7eY7VpYFsbbibp69TZkPjCZtUzZk+8B+pBo0NyA+424ZHxTyRbmyoMTXLTZCwh+CFF\\\\\\\/ZJM8AShftbZDYfDRYmjFYKOfQCLMl9CLHcIver0yfOVDXtphwP2SuGGUhWsYcp0J157elDJyQtELuMqbM8d1EkvuGrWdGuqqWU16wBnTYxF0ML\\\\\\\/vp0P\\\\\\\/saKWpTPaS2o2nShjD5fRng784ix4UhRSkTsRn+kJ0LgcbcdXdi65aOlBwQS0v4MOodDuiIpAcsnjOa7UebWvBUAt2EfjOqTM1Agxtw\\\\\\\/6xWvO6qu8B3GGAo2L6v65yL6yGyI4vVKBeLxSZdqJWQePOWEsEqZwLrWLbS3eh0N5sC8Qz5owxfqoHMrLA1aT5pz776iTtZ+5KzsUWRyUUKCtiO7AgUdv\\\\\\\/ku4tY\\\\\\\/PQFnFs50A27DIDA5JMMB5g0gjzHqawJJ99MNqC\\\\\\\/JhI5qVE4ludWqaP2B7ENSj910sg\\\\\\\/y1uCXHVp4bZieh6RLgWd19pdhQTU7dXKDPoNz\\\\\\\/EpoVde0zQqMkcrDRDYpCISSbysWhmGz0h2ayz9Bbib6JSeks7zmtqOWZofjcQ6SrfveBlCX69wypmnrnQvHmY5OZA5A8h2W\\\\\\\/ZIOutj03Mfq3+HjMlZmG48cquuTUuTi1MWtpB1wyZU8X7pyvTPvYQJ75Dyqj0dZHx6pP4VETJT56DjPqCoCXbeoyTQ5bEVPI7EkI7sMfeD3ue61nOKI3UYSXppDL9UOYKyE5Nrq8nqiAXCVbWNVq5Tc3chxBQGhcGyOtCWHx50tJBQS9leEklrfF0Rbz8\\\\\\\/5NOuSEtZ55ykZ3BFwsx9j7rGe+Vd9Pc\\\\\\\/9AHyHskbKrc8b7FwPVpPkrx8Vj0kW5AOyzg3\\\\\\\/tYidHiLAgtz7Z3B5cw9sN4L+UFk8AeC2aRCFdBktHnJzGQFavraKaxbSr7haaqdzO00+kfG72FRDROEeRuRd3kw9uVCDRaaD\\\\\\\/Kh8BWoAe9e5gfuH23ypd5sbiXPHUaJPm+3p4ztD72CrB+gaG0tAwU6HOTWZ+hpzoNEvmDb1VWcKFvz953NbfxLKsyV5Dm8w9xtz3ZP869qxRXV43uJeldB4yt35uGCP3QO8JGqYarL6t3pEgRdW5oAvi+8MRge1zdePP1h0if1ydFW5PjSFXUBEgklVCc80MOLJvKVc5xuU1CB2ty2rNLpECnlxeaKiN3BGWCuqGsYaj+C82a5+4b5CeY\\\\\\\/qr8EsyfzD5z9LzwFaOuvne4wvlu91SIN2rNBzvZDcdYu5iITA0ToFqdtJsMtQQmtGGVOREc23tvK2enSnDzCIo1l9Mq7yHWVc5HUIElpdmAvBulFaqZSYXOvqhGn59ci7\\\\\\\/N2lPgZX+Wa61DENbgCDgzVF\\\\\\\/nMObNFuxS70cI+HE+ONVsrn8+JmE34ANSVTaZJZwZQ8zkUMoW+6ektT7fmnTOJaOqaX86IsLz38hEpbnY\\\\\\\/M5pC9I+3Fz5kgftqLipwwA9crcnqFn2IfDTuEptR+2eyz7ixcrJECVlNVTgP8IYKgGXq0K6J899jZ+ADIdQB6RenxgG6bU+eZtfNdDgdGsB4TC5LTiqaD\\\\\\\/43xAR1oLM2AkMTcKArLwQFjtefj7+Zptwy0WLGvf4RzYXUvZ6DaBa4VUmrPNn6S434V+3qkr3bARr7JVCCiygFOJlGKWvjLWfCDxve9JADEH0lhKxZUwPUBvlTFLX0GLmphRQvlPJKIaaYi6UawKs4pMO0hGsX2\\\\\\\/6fjgSeZrKsJxg2WWoAkUJIzX7HCD9lklfJj1WE\\\\\\\/NyloMIOY\\\\\\\/lvSiXeifYdnmbsc8TTd8IzTVR45H6\\\\\\\/uwJVI\\\\\\\/hiy3w7jTIYhi\\\\\\\/IkVQpv70lokIhqhKsVqQrdJ2EeUBWpJsAImwkmIRET\\\\\\\/jCTet3dg9I41rB7s4z7q4eE3BpylfcBRIn6bZFlBq7PVI4ykcqF6vrGNJxRRDfs2GYkBAmzmrkqgA\\\\\\\/lGxLQnQZ6iUHt9JWdehTCpRS9a3GeKH1FCMH9DyAZxHXnHVrp+qh9cYsBLksMuQhIE6e05BqNvcK\\\\\\\/dklsAdim71JPkjY7a3DjKIRANgUlbJRaBvPUnFCPycLwsiET0Yf7x7OzfCjC9gJIwtelx1x4xOynulYM5xFmRieyr9wxAzYdibIBgziL56Qa3m3iRn1\\\\\\\/cGnSsPcjDKH3o35bSBAdMxiy+\\\\\\\/kMu0+\\\\\\\/Pgi4uujKYNOUOF\\\\\\\/aAAyrA6RWXeqlysGlIsDsBHlu7scEEENjJeDJs6IMgqXW91LThRWyYvn0BcaO0uawjz6AYFN0MwP4ZaEomB4bzzBLW93Ming3fexKLrGlZijS9jkEuZUx8Nwo0TwD03Im+b1q7Fnm6tNyTTfxU+98CgsCIXMnVYo4Ky0SFCuh6LXv\\\\\\\/+leTF7+amIhLRPuUSrTIFiAIxo6vundEQojVXSyGJbW6eij9mvGv3vB47VcfJmbBCOZ560K6jOl1IRKUL7IALXxIRvqAOslRWhxr0HknM+lKpp2BOQoCO12DFF7kH0e2XVRhIJYRTO2xg09cLWL0ZuBckCUcpbQ\\\\\\\/n6fo48K+rQVDxXjU4apdVLyoaMp5D79tuldDzgY8mwKZO+WF9WwdLO12aQ4Q+lO\\\\\\\/SeRytG0aZhUaUPI8os7U2iHid9Oxonw\\\\\\\/+XUt6lcSpWA4CUO7xUq98TbeSExBlnFADz21jC\\\\\\\/+Rn0+mscjKm\\\\\\\/xf8oNmDVib84pCEa4vmG0ONCsCM4DqDOJ0\\\\\\\/17Jzd4SZ6RV3\\\\\\\/AJD0RamEJdSWrY\\\\\\\/9MU5t43tYSipZYPISzRXrkZkNrY5xCw3aNhfje0qTg9giw9H9vJCSc4moafe0YFRXSx0rxks2wcCOU2C3B6XH+c1JSPzjXf95Y2JkO11oViec7DT1nbRYo3O2Dtg0gUc4LhVoM5KS1qlgRPcozJIGEGh14OF00N3BMJRqXvOAaGz8tEbEpBld\\\\\\\/pg+AVSYpJWmpbSa872WvL0+P9MxFb+75tHM85d6ljNDU9mzZMO0P7pz4A5PCAalp4lf3KStcunfPiQckm7DxmgM\\\\\\\/7kn2G4JngClT4ehLxANJmlOIcSG6y+HHikFSDOXw0KAUQlA1wsl+Xx6rw614P+pT9hzJdsMZLI9f45VHFVBRbzgCGpDlSd8bHR0SnSrkuONznvZUuuYgoPpx\\\\\\\/yy\\\\\\\/AX8CUuhJb44GkC4+5+WjjX3DkNDU67j\\\\\\\/f6IMvL2OfrEGNztqEXz6Vj8KV6qi+h452hI4Df1wpoPjNXqhQFLMnSfwGtV1PCvctiUCx2qsTA4T+HuGDACWnyusNZRtuurmFiOp7NZgSeqH5RWHh1ht5DS3j9+qfAR7XUbPpMJ6oAhxNEYlNrROTo6QRsI8adekznuE9zMMP3HR4nglzUdr40MfVe0t\\\\\\\/U2XLduajPsa4PBDzOrktjGdq44M6hGox4UipQK0Vjr6lWqbp\\\\\\\/EBhBPw8PLODj82Gql+uPooNuPKQnpSYIBMCV2dL1KZ\\\\\\\/Q\\\\\\\/4JntbyRVDY11hkU6+r1Nf1hgt1WIpIOC7GvzcyY\\\\\\\/HFm8ILoPu8ckL3yKpSxgt8CVMrmdAJ8voFvCrBtWI5tt0J2PEnkto0WdyiChCGRanT+tYEJKMtRGi6LKcqbQyAIcu4TZ9St92BOX8uYCJwbKlu+TlpdBLLSod0GhIMD53bM8RYMgeIAhK4S3HnTNrFW11XkgOXW90LrRIrRPeEHEpsm2qCiL\\\\\\\/7xE6RGWCkeoZoD6H6CRWPBAk187cpaLSLz+bN9pnFnBlEvH82IQ\\\\\\\/LSEu+u++5TkeUrK2CF73aHA+UWIZbgo86gFU13SFAM9QbihNMmfjNOgIrFXmizIT+2+TjndAzKhYdnju3Axbh9s4a0UO799toQu5ZuDF4\\\\\\\/2wIB1t15d2Yp\\\\\\\/42gNyYM448fbxdZHBlA4qI4IXarLn7Tu4mnS4jwLAqwJALX7EYqqmzp7U\\\\\\\/syrB9lxQdydxd4C8cnvX0G56ygxe+TAgXTl11DTKfDIS+x6ofqYZhCjRlnoUzOGPb3bWA5gAaEZ2Vl6LHjzMi7RFELPn4Xam+aS1jTohn9DV56DFRgN7zyNdEkA90yhxvLbpI+Rgwdkl\\\\\\\/FGEZb1VbwpDynnDJeVZQjxG+y1UWehjHMuAdNUN0GvSZZRvjmP3Tu6iXVbdyNPP7+DfTJ2GfDMJZeqCRlCdSu2dvhgaNcgi7wWSGDbtzlvt1ZPOCstnG4huEmI56cNSfpeudc1bdvgWAgs7PZeVg5Ad27dpIb9SdeYgc1jdCvlAA\\\\\\\/CgQ5NBJh92QFGZBp0pIPDChTP89Hf7+q8f8Js7oibVoKogtAewYa4WJhq0cmUDtuCLGRAIf\\\\\\\/NNgRUfN8u0\\\\\\\/JO+5PEG4PdA4w0WkCCpRW6yEdv\\\\\\\/bO9QlVbi9tQ3F5nDDVQ2Dd0aEiDCCrtNssPI3RNciryuntwZ0L94diKBZun7t2027Dvk3jZKDgduha\\\\\\\/NGoJYzzbiZpBYXM46Xj7XBNTAByKHqtpj2BBrYN2\\\\\\\/sMPz4ZlSh2erl6xkeOfshJX6yCtZjTd+vnAELJV3gy0yjpaewSzk6mrfWTvAUl33h5mHcQP9SvwdxzZZoLpe5qnMfRCMke4EsJP6MUs8njH2VYoGgJ6LgqzTnCkyvYuKOxd82Mv+Go278N2fY3Q6Jn+pJ7iblRw92zNnbBf3Ir6AmOM9q3oKQ\\\\\\\/dZxw4b5SpcByljAsGF6JDUtrWiEVFk54J4ZngXjpGWFIPsdKV1lvWNazUmsOb7My9+g7hwnhBP8s2WgJhdxDIIRW0wzJ9o5Mnt29imY76qFxAPI9BQVSLRSd95Ej2rED\\\\\\\/z+bwsPWTwPha88hqC4DliC1DHJ2M3NhzwvZQIIBOYTyvagwxQmYH8J8+r9o80Y0gcTWaes9Hq+sqkmyCPoLs9PYMcX87FddMW21le6YFObFF+fzeewuUemiZ3UKjzFGsudx4MKJk5BfqjUn5DYBblyJtbxjmSwBSquOMWIdFYEZ8RmxBkkkctqgxb8xMZ4V1mUzl8dAI\\\\\\\/AZoRrg0Vo7DUOy1jdrVEhi1yaaG3aZ1IY6skDAjhsQgh2jp6Pi5rmCXJwCD69BM9v7G9gv7FIZ3InTgj8Uvsb\\\\\\\/ylr7sGZB1g6+8OZDfWeGcJoJ0iLISb2iHIdAtLBrU3gHQGg1LUIlFrDtfaGzMa8LE+mubZRaj2KjPGW7apqBYKky5brA4q8YBcKqv7J+MMohUvfCoF+\\\\\\\/AndFIxgsW6Wxx3xsRsMDo5LI41\\\\\\\/q7Lo+RgXQpa8hTbsedls3zxsui3ckLT9LDiQlepJs+9EGwtqOEyyfsJURFTORArX+dVSKZ5THY0wH21wwkmgDcQ39R3oamKzz7sFsopf9YjXyJw7YYJ\\\\\\\/b9m13Y1\\\\\\\/xHcW4iufRKuDgdo2SGI\\\\\\\/lgeJPG0w7yMRff3z9vylWqVqcMYlJZaKmlvphFXj+gv6GFoHOvG7cc6gjj31M9giK6q9K2+ZzhjWMCOQs2abxkRLeQwzFlcIKoqXOfae+SC\\\\\\\/oHwC1yEZhgthB5Ryg3ybK51J75kgQ5PCRzxceJaFtBmP8VZiN4w764YzEbx8XOm5mPDUyF6XO+K\\\\\\\/8+yRVXfE3N5kXmvf6hzFG5RYypLZGN9i2iy0KH97XOGC8Q\\\\\\\/Hsp3OQ9bGrTHM3tW1SgsxbldFSbNIP6bkQ97O3HD73S9sIgy\\\\\\\/yQ0Aw0+5gc5f\\\\\\\/0+CDyljxRWM02G43tmEfMSo5m729c0FprFY6VVjNofY4choSzngELgulRDnygj+59tsEJF\\\\\\\/ggS9Z9F405onMhvxbgLm+JAK6pkpnHuDI79cABISdcTvVClMUmaAFNSo3On+X1pbSnbc8\\\\\\\/BViECJGF12PV7XU5++\\\\\\\/9B8\\\\\\\/2g1UYQFOj7lpPuRvMm+YUA9DMAJkgKlFnp0uf2BIDMKjA5ylqZJQAYvkCshnIyh0UgS0Gj2N+Qiv66gJ3im1PtrkQ+\\\\\\\/+DsX\\\\\\\/uNN5aOeTVYJG6YsTYoMhTfrV1nbd3CwWzrjX3FfRggoQECRTUOYlk+OBZTiD6zfjBCDHSiRUQJuK9i3vAfM6acoC\\\\\\\/IZUaACzFfGZTe4ZQpw2MKBuprF42K3A04VdXREzNJdP6+3LMFBsgV7N9sLH9YT\\\\\\\/GutD8ypu4JwCQcYZdmiVjuCxl34sEvb5bhBognbekgqA0+ntTtiFNBLzfz41iqCPJddxYREOP25h\\\\\\\/l8awlFKNLCedK+2+hPGFnjOKZ3NxW3CRC7rV20Xnm9ovGMidzJfBzOcKNQchMAF9UEK1SwmHY7HghdtD8l6yaEm97ywFFmdqC+\\\\\\\/JoZEhgYbzjuu0HgfcsEXUw20mlNrhR3N\\\\\\\/yyCO6GAY2SYT\\\\\\\/4HPUnVSazzX25GDlYhz8JvtVUZ0l6YTbAD9xsLze2hhv5wKeJpxyoTZMLOUYSsuv9f3bg2nduljEsoP5p+yJvBupTmaFPpTzAYuo8xvVwd3c3V8UA45Gl\\\\\\\/fvq9SZ8aSkZ7ETSx151zMw\\\\\\\/EFLV976oNq7n3NUL6nmOg4mUdUSBQ\\\\\\\/8k1BB3ewEFpRFzJ8HTT\\\\\\\/oceREhUEj6ZlED2Ub5Qg9yFTX7c\\\\\\\/bwt67h8vMIKo0jB5XdmWCeV2I4H\\\\\\\/VS+1pN0dnlvHA+kXOUXvImvtjeOvMxdkNdDfeX9c7LfBb3c3XgESdbF\\\\\\\/gGtqGM70Q2xGuwQqDjawMM0fvHvMnrovcZOJqW0LC1Dg+wEbvFAL7p7s\\\\\\\/zYoxpNq6wUu3r5IkvKH1Sbtz9vKQqL8zjSQJhYa3FF5GWN8xAAUfVez\\\\\\\/3XMPtUS5RwrDkSqwxkrx0FIh\\\\\\\/AdnYceQcCgJPbNUwEdu6wX6EBN3GG1+UaGtYJ\\\\\\\/0BF4Fl7eUX10jz6C7\\\\\\\/oI8dcuE3HOVJmtwdk49PnEBT49ALRgr+jgeMmsmeRnc39xdZJ21fJXRICH7MtvBI4ugHUFzQ+R0R\\\\\\\/+7HPIPCckAuxGpHns4laBGH9ITOXMgTzSwj+c6NTkZOqvq8FbscvVFUDrc+RZZnr1Dk497nboi\\\\\\\/n2swXk7T74ZJI1rKwJcJg8JOxJ6zj019IcIY5OQ1S6xWGfaxNO59q8aLZWCuBTXFk31z4CwKnbMX9M579sL0e2Co9\\\\\\\/9epbfGFguHFwuUYBp0JcfBbGxieRw06ML4h\\\\\\\/+FTmdctjutLkj7fkT4GHQqv4u43xUxaVOwagTWZJDYGApL+mjxqtNhH+D09lGpu8h8CZi4dof071cP1XsjKKMQb3L6fb6YZbOfwIFpnSkQbw7qugwp08OEgcQ5xtQaBIT8e9qjWQft3zVEpU9h2Jl6fFih5hAWNGJJj18eIJfNm\\\\\\\/SYTlWDvW86poh5Ss7sA\\\\\\\/70kERp35\\\\\\\/mks4vWRtkk86SKujqXOE7V1y0UgADfgQnKsFhAjjmfp14VUYeEk4GyUhr4jAaLbQEgp4+h7zR4cgNHUwrmmjBZZ\\\\\\\/XprKwlya7PFPqpeg988RGShBoj8toK3IHdR2UcfSnx0d8jAleWenN0SBraGAVtJARga\\\\\\\/C8RFOJLr6TmT9tY38g4ZQMiS+Y3vjF6AP1d1m9yB54VNa0yp2bM3AW\\\\\\\/KLlJbPsIzf8Sp6NKe8dL1IpNU5EsCuWk7NLvSt1O8o8q6iuB4PY\\\\\\\/zwHrG4tQ5CGhVbnf9YnqQERRT9dtn3Lz15LWBjXQj2SfFUpc73k5AcFB9NKDXylFIMPTz1B7YUKvzqG5W+f8z8XeKMtgEcOpfNJtJOsGQg8GeZIDMaM1G3LtpMcyHf8TT\\\\\\\/bq1QvoOkwAmP6hehF1pm92JlE+aW3Pkr6gEO9fE+h\\\\\\\/8+hVfzo2uX2lazRgcYhwka1xP5F6fZoyL9RBCA3cF2im\\\\\\\/mhidh4ls\\\\\\\/gGfyDSF0DI1TAasAL3RNRYnStlI\\\\\\\/fp17sRHehIGsjgXdk03JphGdbUwhmlN6rMsdG51X3i6+rk\\\\\\\/IespoS3wk+DsMFuOMDjnCDpbPRX8F1cKvV6FOSylkODNG2BzLZ5YW3mLTOnQcAdykB37TEp7ByADBmtC15O1\\\\\\\/cAKkRH91vs+uhvoEPrjw8jfo4yNbn18GINPh7X3+XbEjhMkE8GWTiqA1cdC3KU8UsuVJHFl7JjEpOeGXMukuCtfHwxK26vX+AeWkign1PdVTQ6UfYnc0qjGm5r9NodxS\\\\\\\/RAtNzWdlNLr5U4jwU2U9d90l+V7pBthzbtjADKAHd2mLOHQ6c4mltXFCJY8ONt+ZzqIgJ2zVJ5ztyqfOCjGjwgZdGbQPO++WupViwAT2+0eGUhT\\\\\\\/c\\\\\\\/uoMOvQZsySu+3XNzpm0vYCKvLTcX2WZORy4pl3WL1zzawIPheS9lumLsclDVczxlEQ3Tgrn\\\\\\\/URZ2BCMDTs\\\\\\\/bfh\\\\\\\/wcByBl8iPInr3Dxj4cANTOPM3yiMwh1pKF4InyLZbcGYzYrxiPdK7A9G4tP\\\\\\\/DFbOhl+sVipSSdnJX7ZRwLDq13aFkwkDea0XeNVfhLLsJlz5MdYUg6nULFDKPdoxZQt3H4Uai8v4oebe7PVdrEU7xN\\\\\\\/0lmXodidTdDyXkkC5igwY2yy71SQBZrBy1UgH+iqGh4gfIVc8eFz\\\\\\\/xb9XBRGQaG9OYfzgxxO9hwds3H0Rm0hd8AtnTlsvXoaAT1MYZ2OfMgRnccW4eCO2D0yUmFUejZsXrYyP8i3RTm5O7y2U2T32hjk1FeXjiE3N052Rf94xtnRrXIEbkWjB42MdCG0buk4vKqmYrSqz\\\\\\\/dOnExNUwb82o\\\\\\\/cNAqrPDKbaGhve76wIhrQfTNBtFOKWIn2lcBHIjAIsUk507BLaQ0FgsDVO9PVxP3BcQGI1ul\\\\\\\/dK11\\\\\\\/MgzNbwFbNigUP45XkPrwX0z\\\\\\\/sqpi0gnBmEkTRXnDymzWM\\\\\\\/NYYIktzRpn+m\\\\\\\/uqX7Zep3Jk6VQq0gfQkjUrR6ZEVYoRRgwXyy2GFbvf\\\\\\\/J4isLETTBDCQ\\\\\\\/Dki6WbtOg16ON6Vp2upmS3vbFF7uxkJLh1t\\\\\\\/R\\\\\\\/G1D+Cm8QPNFl\\\\\\\/W5w5YEVEqNagaVmUiOs8MDBqqy7Xk3jX54idCRHqZe6n85EmHOcC\\\\\\\/byC5063Z9Js7mQLcUaC4FAKRxKpRQkV22QTf07FCA\\\\\\\/Tcz3fxOnsosYKhIZ4dXm2W32JB91vtG7ONqBab6ZSdU9Kz5gtcEyHWw2RBdQltKezhpCJ+9IzL0tukhnwQ33snpzSWt7Oc4\\\\\\\/jl4O0f37NC8AYCkpS+UJetArNQyV0qEmNm2WnBefzKxIiPd6rNCUvm599X+SeXoGW\\\\\\\/ZoPEztByM3TYXqxY1brSbyrESK1EUIGFpF9zyKgBnbQBfj7w\\\\\\\/kcO\\\\\\\/WwU55HYZAZJtnnw3I1A\\\\\\\/5sA+WoXqyRiRWspAI8b3Zziop+sKx8ZZ+uWw+WqZGkZVxOfHTILw2LkDDyB8SYOX6+KhvbREe\\\\\\\/kpANk7tqtcdjh4f6LubSeunRm1eLNcHShcCoeyFC4XH8hfqQk82HNYZxcJL0RHlp5O\\\\\\\/aYtTom02evDUkwwIeGAdnLd0UbAgPYB2lfBtPFD2pXbSdz8qpqdrgoTiF0za2Ys+Dkip1r8OxkNTkCdzRLWP5pFdgDJ\\\\\\\/wdn33vFTEno4OOYocxZRwyomHrH1XE4IAHNISNw5WeW6NrkLBDZOh6drHzU5Zf5J50DK3JhbJNxsvOsOcXBiE1FmbBJt0dvSBlEmu7s\\\\\\\/B814iLiSWa\\\\\\\/iMq9KF7JdryrXCPZiXxCuLl+\\\\\\\/4CzyxSrI0U6\\\\\\\/cXkMgq3LK3+Xs427EjO+k5cIM41Bvdctk2Cd1I08ad10XHrkA2f9B7hXKITP6l235cTxPYZi7NUZPYADAwpjo4en+UN0W4WDFf6IJ+6trRpLOJZLUEN6PggzHMwi+I96lD4zVPxlQNszoYkIZL+CINW\\\\\\\/A3ORgpyYJg13Zm2F2H81AQHjX9hQfragnQ65onFDRYuJRh7mMecayrmWyE80WclO2ma+cnWHzUI5zjC8l38022QsMJh8ASRdlMjzt2X7\\\\\\\/JyY+QHuvS7CfAui\\\\\\\/bOxlhTNR2ND+O40kV3VvgSUfqaf\\\\\\\/vHAia1Y5NuXBhXfCRrdbfeE5NFedo8do9lTyZQg+q5v\\\\\\\/ywHCQ8uC0S0M+87D7HjCIkkHN0Q8yK5Ep4VPrQqlO663ZUHo\\\\\\\/QmR1ZRw8TB3CTzK\\\\\\\/UtahZOf9gf0M0aJYqflpOiL0VZl3d+s6ULD3GAeA0nExp+qf5qZ1j5w2LI6eZcCvvGszITaVVWd4Nz5f7T5xGAo3syDxS7rpUSGPBudVR+5XBCC8K02VxpXnyPn1eT2HORX\\\\\\\/NLlxd6nlhNXL5EXPhKINYXCZJxAj6O3i1RmVnDpemOUdfhuampaDTIlkHhrSM4wYyNMqyNhxwKQ0sC7JOwoMwiHABtjqhzRpQem\\\\\\\/G\\\\\\\/jEWes44u4XSNCLYauOdv6hPI9W7r7bebRKvsUoEIkwtwVIIP4XhIVsVGP\\\\\\\/I1vuP50Fc0xnwMky5f1lVKVie4bWgy+EOLkc+7Qni\\\\\\\/4gxYG6nWKaMsbUocZ+NBt32m3rq0UW\\\\\\\/vvcSiVsmojw\\\\\\\/yUm5uBg26XekIR4uD3vAls0t1tFzyFqHxlh3mi8zgzU5H41xKn5BcZCEblOLK4cIfU2cctNcBUd451U7LZkO72zB7FiZy1WgmHi2VF9ULwpT2yiylHBGxWjS+foecSEbaZitFO6ZD9j8xLdi2kQFcxyS8E48m3WKvmE9l3JLUtPo0ncUinm039LAdlZ+\\\\\\\/mXqmDI6GOzRyi29z6jEXgEMb5AXca9pT5eEo0X8fSOW6\\\\\\\/r133VxHq6K\\\\\\\/ggFchDBNNw4YY6AsZwkNu4ND348G4spnS17H\\\\\\\/EscOQ\\\\\\\/kvhayot\\\\\\\/mBpJzUqjIDJXh+9GE+kC4EXTYhnMgKF\\\\\\\/iXH0yn0ZHT3X3Pw\\\\\\\/oelxe7mNYGXm5ob0V2NPudXCwVxmHCqdYVNAUWg4yDQXUopscG1RGGZpVUi1\\\\\\\/jRDDL8v6BQkDo1eZ9ID0TXiuXTXm203Rx7GcVpIzcr8QvzE5shgV90s5aN2Yd0ykwZ7Plvf9\\\\\\\/NjIHkWcPLHYGMFv7kBd1+pHDvKBLj\\\\\\\/nhAREK1PXNA5lJIT49OFp3\\\u0022,\\\u0022iv\\\u0022:\\\u0022m0fo6bAJEZxyHbl5VyxFQg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229a96519ac644c11096734303c7d5372a820063a3f8575010ddbd1e72a6033bb6\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/a9QvlO5a9a4x5U92tl\\\\\\\/h7CMfLFSneZRxATZy7\\\\\\\/k+gzziCYclJNCdyUW0HXO96x01zks3hXexFJCr1R0\\\\\\\/vD4q1LoUh0VMb10mpaA6SNG+pSU0+ie1g7QDlJwl0mdqu+FYvWQCn3gMgoS2iwwyHf0ynIcpjFRQ5xjcNroeZlmx\\\\\\\/5OP8bfCmp+ooeWeuQElk5+76bN3Xw1EWJgv+ybue1CBTQqUU+f9rkLF8nWhEXloSGor290GUjiBYQ8h\\\\\\\/mdAWNrH0zJSmZ5cl4gRlXzpZWO38EcaqElKIGCuai5nbYeeRdil2IM\\\\\\\/xCD26t5HgTzSSGtv9qaLr0V3v33RjLizZuqqhRUTNBAIN7c12D1k8ux+b7rRawdgk6IWOKb3Pafaf18U6tW8cK0JWg2T9S3AsBKXfOz6co+fhWYthCt9G5skLr95xI8828lCepKjoi0\\\\\\\/iiOuEUd8qFeJI7cIkVOb3\\\\\\\/u4L73g+yvKuXOT4JMOMN2BYORjfRMRLI0ysdgvsk3UKCEaSX7LPmqq\\\\\\\/uGnS7f+CdLy0Y6Xr+BfPg5aKd2tC8GZ6QFmVd7zLxJov1KQHRA70jHtfVz2zQRuam3g7b2uIs5sBXndJ+5jxvJBAlmoCCCgfiMobbj4aNcUvPUV1qxJKZTrZRlkQdRccQeaUhRSMR3nGJU\\\\\\\/sEw9Ninw3xcWtKcmoDiLK5rOH2mues7CxCD+MGhVcKWjjw1bX7AqDgHkkqFIQIDMafzZeG81pyRaW6OxXdsUcUdUWQtGEmJCpLP1ZKb\\\\\\\/jXYb\\\\\\\/P0odkEZ7xnxwZ2RQBm8fk8UwJKbEYK9CQBZ7G6WHOsnPAnBk7sLbhw9c9hDH7hso45MHz3LpX6zjlkfqewWmCbnze\\\\\\\/28BLj2PmJ\\\\\\\/UXEOp2iziHKxi4oP6gFRUYrEySEXqsey77OjTkkA+lDTV2x8RgrD232MxKLzh6btHAani1gkrXIpIwfPjHqdYp6t6eAgwR8aHgq3YhDI+pAwCmUSaWfcw0vuTkDgnvdFA+y4RZxM3f0YaV2XBR541kVHs5Bngrus92SgWifdAcxP92O2pkUhsE5A2ZbWiA2PYSdfu1Qtwkdb9TyOu\\\\\\\/7qEr0ym\\\\\\\/ee3FFefMryZ6zjMDn32gWtWoARPE47UV4yutgiNyAcQy8vWzC2c9V6VhXMyZZ8eXm3IKa8OTK0acKHg9expTViu7dId3DbyTInqfb3nyxyOSIa7tjyMPuTE+GObIhUO0zopDKon9IXeGY\\\\\\\/zycXXhnt7d7TwCVezJaL6RAZPPcqCIsEmFnhRWM3lQyjna+jcOxdCUS9AKd11ZkSIiLymI\\\\\\\/80IG4wbiHZuYNv5YuTztxJYN264wOu6RbvD+KgMCHJ76ZiUHfQTjE1lpEJWyIo7k30\\\\\\\/otYqFLa1w8kL7khnkPjJzQ2MmwX3cmKdhiAJiiNEZNxhkWom8F9jnzsUYTvDeAbWJYDR19ZGn+GDLiypE5kV5B3SqB5dEM9IlYBeSr4+9lJT2NQ3IS8BjJMfmCecBEy54rWZsNP6hsjDuk3xdbzd3ie33bxrLSoGN8VBbl\\\\\\\/JvSuj\\\\\\\/1h+nP4n0y4rf8+8WKy\\\\\\\/Rqr01\\\\\\\/NvPOIX8CU+EDNNVWbFlqPxSRIl67K85ArnynTNDhT7\\\\\\\/hwvNU4\\\\\\\/5FJxKiINd6hVBbQSVBeEmvdpocohMoCMXEPXJCA5pqe6\\\\\\\/DcmFIDFrvvjJ2N5Ye1XQFZzYyal+aCffF+\\\\\\\/Ts2ynxFO\\\\\\\/jmTny\\\\\\\/ff7dXyuZsrTQE0Tc0KGZaXzeG41myriVxaiGJo5BOVMBWrpY26kro6\\\\\\\/NITUetw6U+VJnhh3YPKuCwXDbS92oC6icyIJRxyTUd5nxM6lxHRiTydd+A3j4hSs6wvI52mpwZONChiTcWVFpd3iDCZSVZonkmFtq0yfPG5MYo6QttRta+UOu1zCudBf+Yo67nMuig4l4wjVlJIYWwmeR8PZLMhT+Jw8oRM2\\\\\\\/dfuaZgOnWBKZBBQuxBhVs5jM3DoAcJt3maTz12PQ9lsDNCupfVTNVnLp2qE7Iki+AglTvAMlMkmNKL1lXgYlAD78=\\\u0022,\\\u0022iv\\\u0022:\\\u0022lb+TU5i68kHZx4wDiFnnrQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229a562579c1c23ae26768f6d83a0580b334fea2db454690c78cf12c7977bbae72\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459539 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/10/1809" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7jnLeWxX8gi53g6insUoROYHIXi-OCqRyhYN8AiN8Ak" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/10/1809" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iPwK6oufRSO6zna0mN1wrP8Bd2F_7yluZ9pWuIQDA2A" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/10/1795" title="Imputation-Based Genomic Coverage Assessments of Current Human Genotyping Arrays" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/10/1819" title="Identification of a Novel Polymorphism in X-Linked Sterol-4-Alpha-Carboxylate 3-Dehydrogenase (Nsdhl) Associated with Reduced High-Density Lipoprotein Cholesterol Levels in I/LnJ Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459539" id="node459539" data-pisa="ggg;3/10/1809" data-pisa-master="ggg;g3.113.007229" data-apath="/ggg/3/10/1809.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Insights into the Evolution of Cotton Diploids and Polyploids from Whole-Genome Re-sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Justin T. Page</span>, <span class="highwire-citation-author" data-delta="1">Mark D. Huynh</span>, <span class="highwire-citation-author" data-delta="2">Zach S. Liechty</span>, <span class="highwire-citation-author" data-delta="3">Kara Grupp</span>, <span class="highwire-citation-author" data-delta="4">David Stelly</span>, <span class="highwire-citation-author" data-delta="5">Amanda M. Hulse</span>, <span class="highwire-citation-author" data-delta="6">Hamid Ashrafi</span>, <span class="highwire-citation-author" data-delta="7">Allen Van Deynze</span>, <span class="highwire-citation-author" data-delta="8">Jonathan F. Wendel</span> and <span class="highwire-citation-author" data-delta="9">Joshua A. Udall</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1809-1818;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007229 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459539" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Justin T. Page </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biology Department, Brigham Young University, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BT.%2BPage%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Page%20JT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BT.%2BPage%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mark D. Huynh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BD.%2BHuynh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huynh%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BD.%2BHuynh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Zach S. Liechty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zach%2BS.%2BLiechty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liechty%20ZS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZach%2BS.%2BLiechty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kara Grupp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kara%2BGrupp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grupp%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKara%2BGrupp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">David Stelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Soil and Crop Sciences, Texas AgriLife Research, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BStelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stelly%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BStelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Amanda M. Hulse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Soil and Crop Sciences, Texas AgriLife Research, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BM.%2BHulse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hulse%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmanda%2BM.%2BHulse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Hamid Ashrafi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Seed Biotechnology Center, University of California-Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamid%2BAshrafi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashrafi%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHamid%2BAshrafi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Allen Van Deynze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Seed Biotechnology Center, University of California-Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BVan%2BDeynze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Deynze%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAllen%2BVan%2BDeynze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jonathan F. Wendel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BF.%2BWendel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wendel%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BF.%2BWendel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Joshua A. Udall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biology Department, Brigham Young University, Provo, Utah 84602</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BA.%2BUdall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Udall%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoshua%2BA.%2BUdall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jaudall@byu.edu" class="" data-icon-position="" data-hide-link-title="0">jaudall@byu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/10/1809" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459539" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459539/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/10/1809.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459539" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459539/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/10/1809.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459539" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459539/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/10/1809.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459539" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459539/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/10/1809.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459539" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459539/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Understanding the composition, evolution, and function of the <em>Gossypium hirsutum</em> (cotton) genome is complicated by the joint presence of two genomes in its nucleus (A<sub>T</sub> and D<sub>T</sub> genomes). These two genomes were derived from progenitor A-genome and D-genome diploids involved in ancestral allopolyploidization. To better understand the allopolyploid genome, we re-sequenced the genomes of extant diploid relatives that contain the A<sub>1</sub> (<em>Gossypium herbaceum</em>), A<sub>2</sub> (<em>Gossypium arboreum</em>), or D<sub>5</sub> (<em>Gossypium raimondii</em>) genomes. We conducted a comparative analysis using deep re-sequencing of multiple accessions of each diploid species and identified 24 million SNPs between the A-diploid and D-diploid genomes. These analyses facilitated the construction of a robust index of conserved SNPs between the A-genomes and D-genomes at all detected polymorphic loci. This index is widely applicable for read mapping efforts of other diploid and allopolyploid Gossypium accessions. Further analysis also revealed locations of putative duplications and deletions in the A-genome relative to the D-genome reference sequence. The approximately 25,400 deleted regions included more than 50% deletion of 978 genes, including many involved with starch synthesis. In the polyploid genome, we also detected 1,472 conversion events between homoeologous chromosomes, including events that overlapped 113 genes. Continued characterization of the Gossypium genomes will further enhance our ability to manipulate fiber and agronomic production of cotton.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/cotton-fiber" class="hw-term hw-article-keyword hw-article-keyword-cotton-fiber" rel="nofollow">cotton fiber</a></li><li class="kwd"><a href="/keyword/comparative-genomics" class="hw-term hw-article-keyword hw-article-keyword-comparative-genomics" rel="nofollow">comparative genomics</a></li><li class="kwd"><a href="/keyword/molecular-evolution" class="hw-term hw-article-keyword hw-article-keyword-molecular-evolution" rel="nofollow">molecular evolution</a></li><li class="kwd"><a href="/keyword/allopolyploid" class="hw-term hw-article-keyword hw-article-keyword-allopolyploid" rel="nofollow">allopolyploid</a></li></ul><p id="p-4">We aligned re-sequencing reads to the existing D-genome sequence and discovered novel changes between the A-genomes and D-genomes in both diploid and polyploid plants. We identified single base differences throughout the genome between the diploid genomes and discovered that 978 genes of the D-genome reference sequence are consistently deleted in the A-genome. We discovered that approximately 900 Kbp of sequence in the polyploid genome have been converted from one genome to another in separate conversion events scattered across the genome. These discoveries help us better understand the dynamic nature of polyploid genomes and provide many avenues for further genomic research in cotton.</p><p id="p-5">The genus <em>Gossypium</em> (cotton) includes approximately 45 diploid species that are divided into eight monophyletic groups, each designated by a single letter (“A” through “G” and “K,” hereafter referred to as genome groups) (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Wendel <em>et al.</em> 2012</a>). Ancient hybridization between A and D diploids resulted in a new allopolyploid (AD) lineage in the New World approximately 1–2 million years ago (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Wendel 1989</a>). Two of the descendant allopolyploid species—<em>Gossypium hirsutum</em> (AD<sub>1</sub>) and <em>Gossypium barbadense</em> (AD<sub>2</sub>)—as well as two African-Asian A diploids—<em>Gossypium herbaceum</em> (A<sub>1</sub>) and <em>Gossypium arboreum</em> (A<sub>2</sub>)—were each independently domesticated for their long, spinnable, epidermal seed trichomes. These four species collectively provide the world’s cotton fiber production, with more than 90% of this total being attributable to the cultivation of “upland cotton,” <em>G. hirsutum</em> (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wendel and Cronn 2003</a>). Understanding the cotton genome is important for facilitating advances in crop variety development and utilization. In addition, insights into polyploid evolution in cotton may further our understanding of other polyploid crops.</p><p id="p-6">Molecular studies and comparisons between diploid cotton species have revealed a genus with extraordinary genome dynamics. For example, there is a nearly three-fold variation in genome sizes among diploids (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Wendel and Cronn 2003</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Wendel <em>et al.</em> 2012</a>), with the A-genome (1.7 Gbp) being nearly twice the size of the D-genome (0.9 Gbp), largely because of the proliferation of GORGE3 gypsy-like retrotransposons (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hawkins <em>et al.</em> 2006</a>). Despite this size difference, comparative mapping studies have indicated that gene order and co-linearity have been largely conserved between the diploid A-genomes and D-genomes (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Brubaker <em>et al.</em> 1999</a>), with the corollary that most genome size diversity reflects variation in the rates of proliferation and deletion of repetitive elements (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hawkins <em>et al.</em> 2009</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Grover and Wendel 2010</a>). Molecular phylogenetic and dating studies indicate that the A-genomes and D-genomes diverged approximately 5–10 million years ago. The F-genome of <em>Gossypium longicalyx</em> diverged from the A-genome after the A–D divergence, making it a suitable outgroup for a comparison of the A-genome diploids.</p><p id="p-7">The respective A and D diploid genomes are closely related to the two homoeologous genomes in allopolyploid cotton, A<sub>T</sub> and D<sub>T</sub> (“T” denotes tetraploid), because allopolyploidization is thought to have occurred during the mid-Pleistocene era, or 1–2 million years ago (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Wendel 1989</a>). Consequently, genome differences between diploids A<sub>2</sub> and D<sub>5</sub> serve as a fair approximation of the differences between A<sub>T</sub> and D<sub>T</sub> tetraploid genomes (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Udall 2006</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Flagel <em>et al.</em> 2012</a>). Thus, the existence of models of the diploid progenitors of allopolyploid cotton provide powerful reference points for inference of homoeology (<em>e.g.</em>, of genes, transcripts, RNA-seq reads) in allopolyploid cotton. The recent publication of the genome sequence of the D-genome diploid (<em>Gossypium raimondii</em>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Paterson <em>et al.</em> 2012</a>) allows for the development of new analytical and comparative approaches for the genomics of both diploid and polyploid cotton. For example, a tool was recently created to assign the sequence reads of allopolyploid cotton to their respective genome after mapping reads from A-diploids and AD-polyploids to the <em>G. raimondii</em> (D<sub>5</sub>) reference sequence (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Page <em>et al.</em> 2013</a>).</p><p id="p-8">Combined with the rapid increase in available sequence data, these new genomic approaches may facilitate molecular and traditional improvement efforts of cotton. For example, analysis of reads from diploids mapped to a single genome reference provides a straightforward method to identify single nucleotide polymorphisms (SNPs) between and within genomes, because each alignment of reads has identical relative positions (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Page <em>et al.</em> 2013</a>). In considering the relationships among sequences from A-genome, D-genome, and AD-genome cotton species, it is useful to distinguish between two classes of SNPs. Briefly, homoeo-SNPs are fixed differences that distinguish (and hence diagnose) the A-genomes and D-genomes. Allele-SNPs, however, are traditional segregating polymorphisms within a single genome, between the two alleles of an individual accession (<em>i.e.</em>, heterozygosity) or between the corresponding homozygous alleles of different accessions. Allele-SNPs are those historically used by breeders to improve cotton cultivars in marker-assisted or genomic selection methods. Homoeo-SNPs add another layer to practical utility of allele-SNPs in that they provide a genomic feature to distinguish between duplicate gene copies. Homoeo-SNPs are also useful in an evolutionary context because their analysis offers insights into the molecular evolutionary properties of allopolyploid cotton and, more generally, allopolyploid genomes.</p><p id="p-9">To better understand both diploid and allopolyploid cotton genomes, we performed deep whole-genome re-sequencing of several diploid accessions of both A-genome and D-genome diploids. Our first objective was to determine all of the homoeo-SNPs between the A-genomes and D-genomes. Using reads from these diploids and publicly available reads from diploid and allopolyploid cottons, we compiled a database of SNPs between the various genomes studied. Our second objective was to describe genome evolution between the genomes that could be characterized by read coverage. We examined loci that are either duplicated or deleted in the A-genome species, based on coverage of A-genome reads mapped to the D<sub>5</sub>-genome reference. Where those duplications or deletions overlap with genes, they may provide insight into the evolutionary basis for the phenotypic differences among diploids, including the production of spinnable fiber in A-genome diploid species. Our third objective was to document the extent of genome interaction based on sequence data in the polyploid (<em>i.e.</em>, conversion events). A robust description of conversion events throughout the cotton genomes will serve as a bioinformatic aide to future genomic analyses of allopolyploid cotton.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material</h3><p id="p-10">Plant material was grown and harvested from greenhouses at Brigham Young University (D<sub>5</sub>-2, D<sub>5</sub>-31, A<sub>1</sub>-155, A<sub>2</sub>-34, A<sub>2</sub>-1011), Iowa State University (D<sub>5</sub>-4, D<sub>5</sub>-53, A<sub>2</sub>-4, A<sub>1</sub>-73), and Texas A&amp;M University (A<sub>2</sub>-255). DNA was extracted from four accessions of <em>G. raimondii</em> (D<sub>5</sub>-2, D<sub>5</sub>-4, D<sub>5</sub>-31, D<sub>5</sub>-53), two accessions of <em>G. herbaceum</em> (A<sub>1</sub>-73, A<sub>1</sub>-155), and four accessions of <em>G. arboreum</em> (A<sub>2</sub>-4, A<sub>2</sub>-34, A<sub>2</sub>-255, A<sub>2</sub>-1011) using a Qiagen DNeasy plant kit.</p></div><div id="sec-3" class="subsection"><h3>Acquisition of DNA sequence</h3><p id="p-11">After shearing DNA with a Covaris instrument at the Huntsman Cancer Institute (Salt Lake City, UT), DNA libraries were prepared with the Illumina TruSeq V3 kit and sequenced by Beijing Genome Institute (BGI, Sacramento, CA), producing 100-bp paired-end reads. We assumed that the Illumina library construction process would perform equally well on high-quality DNA of the A-genomes and D-genomes. Approximately 40-times the genomic coverage was obtained for each library (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Reads from the diploids have been deposited in the NCBI Sequence Read Archive (SRA) under the following entries: PRJNA202235, PRJNA202236, and PRJNA202239 for <em>G. arboreum</em>, <em>G. herbaceum</em>, and <em>G. raimondii</em>, respectively. Additional genomic sequence reads for <em>G. longicalyx</em> (F<sub>1</sub>-1; SRR617255), <em>G. herbaceum</em> (A<sub>1</sub>-97; SRR617256, SRR617284, SRR617704), and <em>G. hirsutum</em> cv. Maxxa (SRR617482) were obtained from the Sequence Read Archive. All reads were trimmed for quality with Sickle using a minimum phred quality threshold of 20 (<a href="https://github.com/najoshi/sickle">https://github.com/najoshi/sickle</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463367/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463367/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Transitions and transversions in the homoeo-SNP index</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Homoeo-SNP index</h3><p id="p-13">An index of homoeo-SNPs between the A-genomes and D-genomes was produced by comparing sequence data from nine <em>Gossypium</em> diploids (A<sub>1</sub>-97, A<sub>1</sub>-155, A<sub>2</sub>-34, A<sub>2</sub>-255, A<sub>2</sub>-1011 <em>vs.</em> D<sub>5</sub>-2, D<sub>5</sub>-4, D<sub>5</sub>-31, D<sub>5</sub>-53). First, all reads were mapped with GSNAP (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Wu and Nacu 2010</a>) using the options “-n1–Q” (requiring unique best mapping for each read) to the 13 chromosomes of the D<sub>5</sub> reference sequence (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Paterson <em>et al.</em> 2012</a>). Second, alignment files were processed with SAMtools to produce sorted BAM files (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Li <em>et al.</em> 2009</a>). Third, we used InterSnp, a custom code built on the BAMtools API (<a href="https://github.com/pezmaster31/bamtools">https://github.com/pezmaster31/bamtools</a>) and available as part of the BamBam package (<a href="http://udall-lab.byu.edu/Research/Software/BamBam.aspx">http://udall-lab.byu.edu/Research/Software/BamBam.aspx</a>) to call SNPs with at least 10-times coverage and a minimum minor allele frequency of 40%. Because the alignments from diploids used the same reference genome, homologous loci in the A-genomes and D-genomes were readily compared to identify SNPs between genomes (homoeo-SNPs). Homoeo-SNPs were called at a locus position of the D-genome reference when all diploid genomes with coverage at that particular locus were homozygous, all A-genome diploids had the same base, and all D-genome diploids had the same base, different from the A-genome diploids. Finally, loci with identified homoeo-SNPs were tabulated into a text file that was converted into a homoeo-SNP index for use by GSNAP and PolyCat.</p></div><div id="sec-5" class="subsection"><h3>SNP identification and diversity analysis</h3><p id="p-14">Using the homoeo-SNP index, we again mapped the sequence reads from the nine diploids, this time using the SNP-tolerant mapping (“-v” option) of GSNAP. We also mapped reads for three additional diploids (F<sub>1</sub>-1, A<sub>1</sub>-73, A<sub>2</sub>-4) and one allopolyploid (<em>G. hirsutum cv. Maxxa</em>). GSNAP and SAMtools were otherwise used as noted. Reads from the tetraploid Maxxa were assigned to the A<sub>T</sub>-genomes and D<sub>T</sub>-genomes using PolyCat (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Page <em>et al.</em> 2013</a>). All SNPs (homoeo-SNPs and allele-SNPs) were called by InterSnp between the 13 resulting BAM files, one for each A or D diploid, one for the A<sub>T</sub>-genome of Maxxa, and one for the D<sub>T</sub>-genome of Maxxa. The number of heterozygous loci in each individual was summarized after filtering loci within conserved duplications. We constructed a neighbor-joining tree for the various diploid accessions, as well as the A<sub>T</sub>-genomes and D<sub>T</sub>-genomes, using the PHYLIP (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Felsenstein 1989</a>) program “neighbor” and default settings. The distance matrix consisted of the percentage of aligned sites that differed in pairwise comparisons.</p><p id="p-15">We used homoeo-SNPs between the diploids to generate a “pseudo-A” genome, with the A alleles substituted into the D<sub>5</sub> reference. We did the same with the Maxxa homoeo-SNPs to make “pseudo-A<sub>T</sub>” and “pseudo-D<sub>T</sub>” genomes. Although these pseudo-genomes did not have indels or structural variations that are present in the actual A, A<sub>T</sub>, and D<sub>T</sub> genomes, the majority of gene sequences were conserved (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Flagel <em>et al.</em> 2012</a>; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Paterson <em>et al.</em> 2012</a>). Thus, these pseudo-genomes served to characterize the location of allele-SNPs within genes and other conserved noncoding sequences, and having each genome on the same "scale" greatly simplifies genome comparisons.</p></div><div id="sec-6" class="subsection"><h3>Duplications and deletions</h3><p id="p-16">We detected putative duplications (relative to the D<sub>5</sub> reference genome) in the other diploids using MACS (with default settings), a commonly used tool for ChIP-seq analysis (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Zhang <em>et al.</em> 2008</a>). It empirically models peaks in coverage of ChIP-seq reads using a dynamic Poisson distribution, thereby estimating the location of a DNA binding molecule. Here, we used MACS to call coverage peaks within WGS reads from the A-genome and F-genome diploids after alignment to the D-genome reference. Assuming the libraries from both A-genomes and D- genomes would be equally biased, reads from D<sub>5</sub>-53 served as a control sample, estimating the expected coverage pattern. Peaks in A-genome coverage relative to D<sub>5</sub>-53 represent putative duplicated sites that were sampled at a higher frequency during sequencing. To filter out false-positives, we also called coverage peaks in D<sub>5</sub>-2, D<sub>5</sub>-4, and D<sub>5</sub>-31, relative to D<sub>5</sub>-53. We used bedtools (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Quinlan and Hall 2010</a>) to compare peaks among and between the datasets and to identify their position relative to gene annotations in the D<sub>5</sub> version 2.1 (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Paterson <em>et al.</em> 2012</a>).</p><p id="p-17">Similarly, putative deletions in the test diploids were called if the reference sequence D<sub>5</sub>-53 had 20-times or higher coverage at both ends, as well as at an additional point at least 200 bp from either end of a region of at least 1000 bp, and if the test diploid had near-zero coverage (&lt;3-times) at every point in that block. This detection was performed by Gapfall, part of the BamBam package (<a href="http://udall-lab.byu.edu">http://udall-lab.byu.edu</a>). Blast2Go was used for an enrichment analysis (using the Fisher exact test) on genes duplicated or deleted in the A diploids (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Conesa <em>et al.</em> 2005</a>). Default B2G parameters were used.</p></div><div id="sec-7" class="subsection"><h3>Polyploid conversion events</h3><p id="p-18">We used two methods to identify possible nonreciprocal homoeologous or “gene conversion” events between the A<sub>T</sub>-genomes and D<sub>T</sub>-genomes of <em>G. hirsutum</em> cv. Maxxa. We first identified individual converted loci based on homoeo-SNPs, where reads from the A<sub>T</sub>-genome carried the D-genome nucleotide, or vice versa. The second method used duplications and deletions to identify regions of conversion, but for deletion detection using 15-times the minimum coverage for the duplicated genome and less than four-times the coverage for the "deleted" genome. If a region spanning at least 1 Kbp was “duplicated” in the A<sub>T</sub>-genome relative to the A diploids and “deleted” in the D<sub>T</sub>-genome relative to the D diploids, then an A<sub>T</sub>-biased conversion event was inferred. Similarly, an A<sub>T</sub>-genome deletion and D<sub>T</sub>-genome duplication suggested a D<sub>T</sub>-biased conversion. These analyses of the polyploid genome were limited to regions that were present in both diploid genomes (A and D) because homoeo-SNPs could only be predicted in such regions.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Intergenomic SNPs</h3><p id="p-19">Similar to previous work on the detection and frequency of SNPs in genic regions (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Page <em>et al.</em> 2013</a>), we produced a robust index of 23,859,893 homoeo-SNPs between the genomes of diploid A-genome and D-genome cotton. These SNPs covered the genome of the D-genome reference sequence at a density of one SNP per 32.3 bases (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). This total number of SNPs is a dramatic increase from the number previously reported with the D-genome sequence (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Paterson <em>et al.</em> 2012</a>) and in genic sequences (<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Page <em>et al.</em> 2013</a>). The index had a transition/transversion ratio of 1.92 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), similar to the Maize HapMap2 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chia <em>et al.</em> 2012</a>). This genome-wide SNP analysis confirmed our speculation that the previous ratio was downwardly biased in our gene-focused index. Across polymorphic nucleotide positions, there was not a significant difference between the GC biases of the A-genomes and D-genomes (45.4% and 45.1%, respectively). However, these values were higher than the genome-wide GC content, suggesting an increased likelihood for SNPs at G or C nucleotides, possibly because of the high frequency of C→T mutations caused by de-amination of cytosines.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/10/1809/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of genes, homoeo-SNPs, duplications, deletions, and conversion events in the A-genomes, relative to the D5 reference sequence, produced by Circos (Krzywinski et al. 2009). Considering the concentric circles from the outside inward, the outermost (and first) green circle indicates the location of annotated genes. The next circle (red) is a histogram of the number of homoeo-SNPs in a 1-Mbp window throughout the genome. The next two red (high-frequency) to yellow (low-frequency) circles are heat maps showing the location of duplications in the A1 and A2 genomes as compared to the D5 genome (A2 interior). The next two blue (high-frequency) to yellow (low-frequency) circles are heat maps showing the location of deletions in the A1 and A2 genomes as compared to the D5 genome (A2 interior). The final two circles show conversion events in the tetraploid G. hirsutum cv. Maxxa. The first circle shows conversion of loci to the A nucleotide on a red-to-yellow scale, whereas the innermost (and last) circle shows conversion of loci to the D nucleotide on a blue-to-yellow scale." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-409811298" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of genes, homoeo-SNPs, duplications, deletions, and conversion events in the A-genomes, relative to the D&lt;sub&gt;5&lt;/sub&gt; reference sequence, produced by Circos (Krzywinski &lt;em&gt;et al.&lt;/em&gt; 2009). Considering the concentric circles from the outside inward, the outermost (and first) green circle indicates the location of annotated genes. The next circle (red) is a histogram of the number of homoeo-SNPs in a 1-Mbp window throughout the genome. The next two red (high-frequency) to yellow (low-frequency) circles are heat maps showing the location of duplications in the A&lt;sub&gt;1&lt;/sub&gt; and A&lt;sub&gt;2&lt;/sub&gt; genomes as compared to the D&lt;sub&gt;5&lt;/sub&gt; genome (A&lt;sub&gt;2&lt;/sub&gt; interior). The next two blue (high-frequency) to yellow (low-frequency) circles are heat maps showing the location of deletions in the A&lt;sub&gt;1&lt;/sub&gt; and A&lt;sub&gt;2&lt;/sub&gt; genomes as compared to the D&lt;sub&gt;5&lt;/sub&gt; genome (A&lt;sub&gt;2&lt;/sub&gt; interior). The final two circles show conversion events in the tetraploid &lt;em&gt;G. hirsutum cv. Maxxa&lt;/em&gt;. The first circle shows conversion of loci to the A nucleotide on a red-to-yellow scale, whereas the innermost (and last) circle shows conversion of loci to the D nucleotide on a blue-to-yellow scale.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/10/1809/F1.medium.gif" width="440" height="437"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/10/1809/F1.medium.gif" width="440" height="437"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/10/1809/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/10/1809/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463416" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Plot of genes, homoeo-SNPs, duplications, deletions, and conversion events in the A-genomes, relative to the D<sub>5</sub> reference sequence, produced by Circos (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Krzywinski <em>et al.</em> 2009</a>). Considering the concentric circles from the outside inward, the outermost (and first) green circle indicates the location of annotated genes. The next circle (red) is a histogram of the number of homoeo-SNPs in a 1-Mbp window throughout the genome. The next two red (high-frequency) to yellow (low-frequency) circles are heat maps showing the location of duplications in the A<sub>1</sub> and A<sub>2</sub> genomes as compared to the D<sub>5</sub> genome (A<sub>2</sub> interior). The next two blue (high-frequency) to yellow (low-frequency) circles are heat maps showing the location of deletions in the A<sub>1</sub> and A<sub>2</sub> genomes as compared to the D<sub>5</sub> genome (A<sub>2</sub> interior). The final two circles show conversion events in the tetraploid <em>G. hirsutum cv. Maxxa</em>. The first circle shows conversion of loci to the A nucleotide on a red-to-yellow scale, whereas the innermost (and last) circle shows conversion of loci to the D nucleotide on a blue-to-yellow scale.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">The genome-wide SNP index (SNP index 2.0) was based on comparisons of deep sequence coverage between multiple diploid A-genomes and D-genomes, so we anticipated it would be more robust and widely applicable for read mapping efforts of other diploids and allopolyploids than the previous index. The improved index increased mapping efficiency of A-genome reads to the D-genome reference sequence. With the SNP-tolerant mapping of GSNAP, more than 77% of A-genome reads mapped, reflecting a mapping improvement of approximately 15% compared to mapping without the SNP-index (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007229/-/DC1/007229SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007229/-/DC1/FigureS1.pdf">Figure S1</a>). D-genome mapping was unaffected (∼95%). The error rate of categorization of WGS reads was less than 2%, as estimated by categorizing the diploid reads and looking for incorrectly categorized reads. Although this error rate is slightly higher than that estimated for the original PolyCat index (<a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">Page <em>et al.</em> 2013</a>), it is still an acceptable rate considering the increased fraction of reads that overlap a SNP between genomes (∼70%, up from &lt;50%), and considering WGS reads mapping to less conserved intergenic regions.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463395/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463395/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of nine diploid WGS re-sequencing libraries that were re-sequenced in this study and additional libraries (A1_97, F1_1, and Maxxa) obtained from the SRA</span><div class="sb-div caption-clear"></div></div></div><p id="p-22">Within the diploid index, genes had a median intergenomic SNP per base rate of 2.2% (range, 0–16.1%). Notably, there were 593 genes that had no unambiguous homoeo-SNPs between diploid A-genomes and D-genomes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007229/-/DC1/FigureS2.pdf">Figure S2</a>). Of these, 215 genes had one or more allele-SNPs (<em>i.e.</em>, one diploid genome had two nucleotides, one matching the second diploid genome and the other nucleotide being novel). The remaining 378 genes were completely conserved across all accessions with no SNP differences. A Blast2Go enrichment analysis of these genes identified the following three enriched GO terms: NADH dehydrogenase (ubiquinone) activity (GO:0008137); NADH dehydrogenase (quinone) activity (GO:0050136); and NADH dehydrogenase activity (GO:0003954). Most of these genes were shorter than the average gene within the D-genome annotations (95 to 8113 bp with mean 810 ± 786 SD for the 378 genes <em>vs.</em> 89 to 51,174 bp with mean 3249 ± 2806 SD for all 37,223 genes; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007229/-/DC1/FigureS2.pdf">Figure S2</a>) (<a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Paterson <em>et al.</em> 2012</a>).</p><p id="p-23">In the polyploid, improved categorization of reads into its two separate genomes was enabled by the genomic SNP index. Using the SNPs from the diploids and the D-genome reference, PolyCat assigned more than 70% of mapped polyploid reads to the A<sub>T</sub>-genomes or D<sub>T</sub>-genomes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007229/-/DC1/FigureS3.pdf">Figure S3</a>). For the tetraploid Maxxa, a greater percentage of reads were assigned to the A<sub>T</sub>-genome than to the D<sub>T</sub>-genome, despite the fact that categorization only occurred in regions shared by the two genomes. This later criteria preempted the larger A-genome from an A<sub>T</sub> categorization bias. The unexpectedly higher categorization rate of A<sub>T</sub> reads may be partially explained by the fact that A<sub>1</sub> and A<sub>2</sub> diploids are a two-fold better approximation of the A<sub>T</sub>-genome than D<sub>5</sub> is of the D<sub>T</sub>-genome. For example, nucleotide diversity appears to play a role in mapping efficiency among the diploid A-genome species. The most divergent line (A<sub>1</sub>-73) had the lowest mapping percentage of any of the of the A-genome diploid accessions. Because sequence divergence is less between the A-genome diploid and polyploid than the D-genome and polyploid, read categorization based on SNPs between the diploids would be more effective for the A<sub>T</sub>-genome, resulting in the observed bias. To a much lesser degree, the A<sub>T</sub> categorization bias may also be partially attributed to duplicated loci in the A<sub>T</sub>-genome mapping to a single locus in the D<sub>5</sub> reference, although these artifacts were largely avoided by the detection of duplications.</p><p id="p-24">Because of their recent common ancestry, many of the identified differences between the A-genome and D-genome diploids were retained between the A<sub>T</sub>-genomes and D<sub>T</sub>-genomes as homoeo-SNPs. A total of 20,828,020 homoeo-SNPs were identified between the A<sub>T</sub>-genomes and D<sub>T</sub>-genomes of the allopolyploid cultivar Maxxa. The difference between the number of approximately 20 million SNPs in the polyploid and the “retained ancestral” homoeo-SNPs (∼16 million; 75.8%) were autapomorphic SNPs that were derived after the divergence of the A<sub>T</sub>-genome and D<sub>T</sub>-genome from the A-genome and D-genome common ancestor, respectively. This portion of the homoeo-SNPs (5,046,151; 24.2%) was only identified between the genomes in the allopolyploid and not in the comparison of the diploid genome sequences. These unique, homoeo-SNPs were found throughout the genome in 34,810 of the 37,223 annotated genes. We anticipate that additional polyploid autapomorphic SNPs will be identified as more polyploid genomes are re-sequenced.</p><p id="p-25">For all of the annotated genes in the D-genome reference, an alignment of A, A<sub>T</sub>, D<sub>T</sub>, and D-genomes was created, from which the amount of molecular evolution between the A-genomes and D-genomes of cotton was calculated (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The results of this effort concurred with our previously published work based on aligned EST contigs (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Flagel <em>et al.</em> 2012</a>), although SEs were much smaller because of the much larger dataset. We found slightly less divergence (dN and dS) between the polyploid genomes (n = 28,317) than between the diploid genomes (n = 30,874), although the difference is not significant. The different totals between the diploids and polyploids suggested that more than 2500 genes in the polyploid did not have sufficient polymorphism for an appropriate estimation of molecular evolution. We further investigated the alignments of these genes to ascertain whether their close sequence similarity was the result of gene conversion between homoeologous genomes. Of the genes without dN/dS estimates, 759 were found to only have sufficient polymorphisms between the tetraploid genomes (and not between the diploid genomes) and 3316 were found to have sufficient polymorphisms between the diploids (but not between the tetraploid genomes). This cumulative large difference between ploidy levels further suggested that gene conversions may play a role in reducing genetic diversity between genomes. However, only 106 and 42 genes were detected to overlap "conversion regions" in the diploids and tetraploid genome.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463295/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463295/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Amount of molecular evolution between the A and D genomes of cotton</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">Of the 2,817,991 SNPs between diploids that fell within genes, 486,514 were inferred to be in exonic positions, including 248,599 that caused amino acid changes (<em>i.e.</em>, nonsynonymous) compared to the reference sequence. Of these, there were 1651 genes with SNPs that resulted in premature stop codons in the pseudo-translation of A-genome transcripts, 1802 genes with premature stop codons in A<sub>T</sub>, and 709 genes with premature stop codons in D<sub>T</sub> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>)<sub>.</sub> These genes were not excluded in estimates of molecular evolution. None of these gene sets had any enriched GO terms. The low level of D<sub>T</sub> premature stops may simply reflect an ascertainment bias of the annotated reference genome that was based on a diploid D genome. Many of the putative stop codons were found near the annotated end of the gene, suggesting that they might have only a minimal impact on protein function. Alternatively, their inference may reflect bioinformatic artifacts, such as imperfect gene annotation of the D-genome, or alternative stops that independently evolved in the A-genomes. Most of these alternative stops codons were within 10% of the 3′ end of the gene. If one ignores the premature stop codons within the last 10% of the annotated genes, then 803 premature stops were shared between the diploid A-genomes and the A<sub>T</sub>-genome (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). This result was marginally less than previously reported (<a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">Paterson <em>et al.</em> 2012</a>), because we had the added power of multiple A-genome re-sequencing efforts.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/10/1809/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Premature stop codons were found in each Gossypium genome. (A) Premature stop codons (compared to the annotations of the D-reference genome) were found in the A, AT, and DT genomes. (B) Common genes with premature stop codons in the first 90% of the gene." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-409811298" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Premature stop codons were found in each &lt;em&gt;Gossypium&lt;/em&gt; genome. (A) Premature stop codons (compared to the annotations of the D-reference genome) were found in the A, A&lt;sub&gt;T&lt;/sub&gt;, and D&lt;sub&gt;T&lt;/sub&gt; genomes. (B) Common genes with premature stop codons in the first 90% of the gene.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/10/1809/F2.medium.gif" width="438" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/10/1809/F2.medium.gif" width="438" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/10/1809/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/10/1809/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463473" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Premature stop codons were found in each <em>Gossypium</em> genome. (A) Premature stop codons (compared to the annotations of the D-reference genome) were found in the A, A<sub>T</sub>, and D<sub>T</sub> genomes. (B) Common genes with premature stop codons in the first 90% of the gene.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Other SNPs disrupted a start or stop codon. We identified 806 genes with disrupted start codons (<em>i.e.</em>, resulting in an amino acid distinct from that in the D<sub>5</sub> reference) in the A-genome, 703 in A<sub>T</sub>, and 684 in D<sub>T</sub>. These genes could have a longer or shorter coding sequence than as originally annotated. No GO terms were enriched in these gene sets. We also identified 831 genes with altered stop codons in the A-genome, 693 in A<sub>T</sub>, and 437 in D<sub>T</sub>, resulting in longer peptide sequences. Several GO terms (∼20 in each genome) were enriched within these genes, and almost all were associated with photosynthesis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007229/-/DC1/TableS1.pdf">Table S1</a>). There were also 406 genes without a stop codon within the D<sub>5</sub> gene annotation, with the same photosynthesis GO terms being enriched.</p></div><div id="sec-10" class="subsection"><h3>Diversity and heterozygosity</h3><p id="p-29">In addition to creating an index of nucleotide differences between the diploid A-genomes and D-genomes, we detected unique nucleotide variation within and between individual accessions. Within a genome type (<em>i.e.</em>, A or D), these types of SNPs are called allele-SNPs. The allelic genotype of each diploid and both genomes of the allopolyploid Maxxa were determined at all polymorphic loci. A pairwise comparison between accessions found that the D<sub>5</sub> diploids had extremely low nucleotide diversity (&lt;1 million SNPs) between any two accessions, whereas a similar pairwise comparison between the A<sub>1</sub>-genomes and A<sub>2</sub>-genomes found that accessions were more diverse (4–5 million SNPs within A<sub>1</sub> or A<sub>2</sub>; 6–8 million SNPs between A<sub>1</sub> and A<sub>2</sub>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007229/-/DC1/TableS2.pdf">Table S2</a>). There were approximately twice as many SNPs between the A<sub>1</sub>-genomes and A<sub>2</sub>-genomes as within either of the two species. These results are not unexpected given the exceptionally low diversity found in a survey of allozyme diversity in <em>G. raimondii</em> (J. F. Wendel, unpublished data) and the appreciable levels of diversity in the chosen accessions of <em>G. arboreum</em> and <em>G. herbaceum</em> (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Wendel <em>et al.</em> 1989</a>).</p><p id="p-30">In addition to having more fixed allele-SNPs between accessions, the A-genome diploids were more heterozygous than the D-genome diploids (∼13% and &lt;1%, respectively; <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). In the A-genome diploids, heterozygous loci were approximately twice as frequent outside than inside of genes. This was not surprising, given the expectation of more intense purifying selection on coding sequences. Of course, these estimates of heterozygosity excluded loci that were duplicated in the A-genome. Interestingly, heterozygous loci in the D-genome diploids were equally common in genic and nongenic regions. This genomic difference likely reflects both the exceptionally low genetic diversity within the D-genome and a high level of generalized inbreeding. In this respect, we note that <em>G. raimondii</em> has a narrow natural range and presently exists as only scattered populations with very low effective population sizes.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463397/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463397/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Number of heterozygous loci in each accession, along with the percentage of total observable loci that were heterozygous</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">A neighbor-joining tree was constructed and rooted based on the known relationship between the A/F-genome clade and the D-genome clade (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Many fixed allele-SNPs (<em>i.e.</em>, not heterozygous within a line) could be attributed to mutations occurring along specific branches of the phylogeny (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). The tree correctly reconstructed the accepted relationships of the diploids and their relationships to the two genomes of allopolyploid cotton (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Senchina <em>et al.</em> 2003</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Grover <em>et al.</em> 2012</a>; <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Wendel <em>et al.</em> 2012</a>). Specifically, and unsurprisingly, A<sub>T</sub> and D<sub>T</sub> were phylogenetically sister to the common ancestor of the [A<sub>1</sub> + A<sub>2</sub>] clade and the D<sub>5</sub> clade, respectively. Our results agreed with previous reports that the A<sub>1</sub> or A<sub>2</sub> diploids were approximately twice as good of an approximation of the A<sub>T</sub>-genome as the D<sub>5</sub> diploids were of the D<sub>T</sub>-genome (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Wendel and Cronn 2003</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Senchina <em>et al.</em> 2003</a>). The distance from D<sub>T</sub> to D<sub>5</sub> was 0.14, and the distance from A<sub>T</sub> to the common ancestor of A<sub>1</sub> and A<sub>2</sub> was 0.08. However, the distances from A<sub>T</sub> to A<sub>1</sub> and A<sub>2</sub> were 0.10 and 0.11, respectively. Moreover, the distance from A<sub>T</sub> to any individual A-genome diploid was 0.14, similar to the distance between D<sub>T</sub> and any D-genome diploid. Although the group of A-genome diploids provided an approximation of A<sub>T</sub> that was two-times better than a group of D<sub>5</sub> diploids did of D<sub>T</sub>, any individual A or D diploid appeared to be equally similar to its A<sub>T</sub> or D<sub>T</sub> counterpart. The exceptionally low diversity among D<sub>5</sub> diploids explained the fact that a group of D<sub>5</sub>-genome diploids was not significantly better for approximation of the D<sub>T</sub>-genome than was a single D<sub>5</sub>. However, multiple accessions of A diploids (A<sub>1</sub> and/or A<sub>2</sub>) did provide a substantial (nearly two-times better) improvement in construction of the A<sub>T</sub>-genome pseudo-sequence.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/10/1809/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Neighbor-joining tree built by PHYLIP, based on SNPs between genomes. Units (as measured by the indicated scale) are percentage of represented polymorphic sites that differed between two individuals. Image rendered by Archaeopteryx (Han and Zmasek 2009)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-409811298" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Neighbor-joining tree built by PHYLIP, based on SNPs between genomes. Units (as measured by the indicated scale) are percentage of represented polymorphic sites that differed between two individuals. Image rendered by Archaeopteryx (Han and Zmasek 2009).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/10/1809/F3.medium.gif" width="440" height="343"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/10/1809/F3.medium.gif" width="440" height="343"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/10/1809/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/10/1809/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463327" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Neighbor-joining tree built by PHYLIP, based on SNPs between genomes. Units (as measured by the indicated scale) are percentage of represented polymorphic sites that differed between two individuals. Image rendered by Archaeopteryx (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Han and Zmasek 2009</a>).</p><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463476/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463476/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">SNPs attributable to specific areas of the phylogeny</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Duplications and deletions</h3><p id="p-34">Duplications were detected in A-genome diploids as coverage peaked across the D-genome reference sequence (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Because these duplications were detected relative to the D-genome diploids, they represent events that occurred after the split of these two clades 5–10 million years ago (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Wendel and Cronn 2003</a>; <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Wendel <em>et al.</em> 2012</a>). Thus, peaks shared by all A-genome accessions represent pieces of the A-genome that are duplicated relative to the D-genome. These coverage peaks represent a mix of tandem and dispersed duplications, because the methodology used makes no distinction regarding genomic location of duplicated segments. There were 30,709 regions duplicated in all A-genome diploids but no duplication in D-genome diploids. These duplicated blocks overlapped 1007 genes, with a minimum overlap of 50% of the gene length. Only one GO term was enriched among these genes: structural constituent of ribosome (GO: 0003735).</p><p id="p-35">In contrast to duplications, putatively deleted regions of the A-genome were detected with a higher degree of certainty because their diagnosis is based on lack of coverage rather than a quantitative difference in coverage (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Some regions of the D-genome reference genome did not have any A-genome reads mapped to them, despite 40-times the WGS coverage based on the number of produced A-genome reads and correctly mapping D-genome reads to the same region. Each accession had a unique set of deleted regions, including genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007229/-/DC1/FigureS4.pdf">Figure S4</a>). There were 25,408 regions deleted in all A-diploid genomes. The genomic regions included 978 annotated D-genome reference genes where the deleted region minimally overlapped 50% of the gene length. Among the genes within deleted regions, 118 GO terms were enriched compared to the population of GO terms within the annotated gene set (<a id="xref-ref-23-8" class="xref-bibr" href="#ref-23">Paterson <em>et al.</em> 2012</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007229/-/DC1/TableS3.xlsx">Table S3</a>). Most of these deletion terms were associated with starch synthesis, tRNAs, or DNA repair mechanisms. Three hundred seventy-eight genes were completely deleted in the A-genome diploids relative to the D-genome diploids, meaning that the genic region was spanned by a single deletion block.</p></div><div id="sec-12" class="subsection"><h3>Polyploid conversion events</h3><p id="p-36">We used two different methods to detect conversion events in the allopolyploid genome of Acala Maxxa (<em>G. hirsutum</em>). The first method identified historical nonreciprocal homoeologous recombination events (NRHR) at individual loci based on homoeo-SNPs within a polyploid genome. Earlier analyses in cotton used this method to detect conversion events (or NRHR events) based on comparative analysis of assembled EST sequences from diploid and allopolyploid cotton (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Salmon <em>et al.</em> 2010</a>; <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Flagel <em>et al.</em> 2012</a>). The second method identified converted regions based on coverage patterns in the A<sub>T</sub>-genomes and D<sub>T</sub>-genomes relative to their respective diploid relatives. Here, we consider the NRHR events as “conversion” events regardless of whether they occur in coding or in noncoding sequences (<em>sensu amplo</em> of gene conversion).</p><p id="p-37">Based on homoeo-SNPs (first method), 1,748,889 conserved SNPs in 29,576 genes in the diploid genomes suggested an A<sub>T</sub>-biased allele conversion (a D<sub>T</sub> nucleotide converted to the A<sub>T</sub> nucleotide). In contrast, a total of 361,795 SNPs in 12,346 genes suggested a D<sub>T</sub>-biased conversion. These data suggest a nearly five-fold bias based on homoeo-SNPs and a two-and-one-half–fold bias based on genes in favor of A<sub>T</sub>-biased conversion, in stark contrast to the two-fold bias in favor of D<sub>T</sub>-biased conversion reported previously (<a id="xref-ref-23-9" class="xref-bibr" href="#ref-23">Paterson <em>et al.</em> 2012</a>).</p><p id="p-38">Based on coverage/deletion information (second method), conversion events were found in both directions across 882 Kbp of the D-genome reference sequence (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). These events ranged from 1 to 5470 bp, with a median of 337 bp (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007229/-/DC1/FigureS5.pdf">Figure S5</a>). Two hundred fifty-nine regions suggested an A<sub>T</sub>-biased conversion. These regions spanned 275 Kbp and overlapped the coding sequence of 19 genes. They also included 12,696 putative homoeo-SNPs (based on the diploids), none of which was detected within the tetraploid. However, 1213 regions showed a D<sub>T</sub>-biased conversion. These regions spanned 607 Kbp and overlapped the coding sequence of 94 genes. They included 21,142 putative homoeo-SNPs, of which only three were also detected within the tetraploid. The genes overlapped by these regions had no enriched GO terms and indicated a conversion bias in both directions, but with the D<sub>T</sub> direction most prominent, similar to that previously reported (<a id="xref-ref-23-10" class="xref-bibr" href="#ref-23">Paterson <em>et al.</em> 2012</a>). The events detected by the second method only included 1375 of the possibly conversion-related SNPs identified by the first method.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><div id="sec-14" class="subsection"><h3>Genome resources for <em>Gossypium</em></h3><p id="p-39">Using the diploid re-sequencing data, we created several useful resources for the <em>Gossypium</em> genome. First, a genome-wide map of the SNPs between the diploid A-genomes and D-genomes of cotton was created. Re-sequencing multiple accessions of each diploid enabled us to distinguish bases that were specific to a single accession from bases that are more representative of one diploid genome or another. It also allowed us to identify conserved genomic features shared by all A-genome or D-genome species and accessions. In that sense, the multiple accessions of each species acted as re-sequencing replications of the A-genome or D-genome “treatments.” We have demonstrated that most SNPs identified between diploid genomes can be directly extrapolated to differences between the descendant allopolyploid genomes (<em>i.e.</em>, homoeo-SNPs) because of their recent common ancestor. In addition, we also identified several million homoeo-SNPs that were unique to the Maxxa allopolyploid genome. These documented SNPs can be used for genome identification of individual sequence reads (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Udall 2006</a>; <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Flagel <em>et al.</em> 2012</a>) or the development of genotyping assays (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Van Deynze <em>et al.</em> 2009</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Byers <em>et al.</em> 2012</a>). With an index of homoeo-SNPs, read-mapping efficiency was significantly improved and future false-positive allele-SNPs can be filtered out of marker sets, resulting in more reliable allele-SNP assessment. In addition to the homoeo-SNPs, we also identified allele-SNPs in the diploid cotton accessions. These SNP sets are available as Gbrowse tracks at CottonGen (<a href="http://www.cottongen.org">http://www.cottongen.org</a>) and as gff files at the Udall laboratory web site (<a href="http://udall-lab.byu.edu">http://udall-lab.byu.edu</a>).</p><p id="p-40">A second resource is the set of alignments of gene and protein sequences of the A-genomes, A<sub>T</sub>-genomes, and D<sub>T</sub>-genomes to accompany the previously published <em>G. raimondii</em> annotations (<a id="xref-ref-23-11" class="xref-bibr" href="#ref-23">Paterson <em>et al.</em> 2012</a>). These alignments were used to identify SNPs and to further refine our understanding of the molecular evolutionary differences between genomes. Because we have made this a public resource, any researcher investigating cotton now has homoeo-SNPs and allele-SNPs information for any target gene already identified. This simple, yet tedious, task has been a common obstacle of genetic research in polyploid cotton.</p><p id="p-41">A third resource, and one that we suggest will be a fruitful topic for further investigation, is the description of putative duplications and deletions that distinguish the A-genomes and D-genomes and, hence, originated subsequent to their divergence from a common ancestor. These localized structural variations offer a rich source of sequences to mine for possible functional consequences, and to further our understanding of the mechanisms of copy number variation during genome evolution in plants.</p><p id="p-42">Through our read-mapping efforts, we noticed that the limited number and the stochastic distribution of homoeo-SNPs could have implications for <em>de novo</em> genome assembly of polyploid cotton. Although 70% of the reads from allopolyploid cotton could be assigned to one of its two co-resident genomes, 30% of the reads that mapped to the D-genome reference did not overlap a homoeo-SNP and, hence, could not be categorized. Using an arbitrary length of 1000 bp, we found 47,399 unique loci where sequence reads of the A<sub>T</sub>-genome and D<sub>T</sub>-genome were indistinguishable when compared to each other and to the reference genome. Assuming sequence read lengths less than 500 bp, these regions would likely co-assemble during a <em>de novo</em> whole-genome shotgun assembly with current read lengths. Consequently, co-assembled segments will create unique challenges of graph structure bifurcation (or higher branching) during the contig construction steps of <em>de novo</em> assembly. Part of this challenge could be addressed by generating reads with a greater likelihood of overlapping homoeo-SNPs, <em>i.e.</em>, longer reads. Present data, however, suggest that <em>de novo</em> assembly of the allopolyploid cotton genome would not be successful if based on contemporary read lengths.</p></div><div id="sec-15" class="subsection"><h3>Insights into the genome biology of <em>Gossypium</em></h3><p id="p-43">One of the intriguing results of this study is the insight it provides into the origin and frequency of indels during A-genome and D-genome divergence. Because of the lack of an outgroup sequence, none of the “duplications” or “deletions” described here is polarized, so their duplicate or deleted status is only relative to the single D-genome reference. Moreover, the methods used do not yield insights into the mechanistic underpinnings of the indels, which may conceivably entail a full spectrum of deletional mechanisms and processes of tandem and dispersed duplication.</p><p id="p-44">Notwithstanding, the present study does reveal the scope and scale of the indel generating process during 5–10 million years of diploid evolution. Additionally intriguing are the genomic distributions of the duplicated and deleted regions. For example, chromosome 13 is notable for its high frequency of duplications, containing one-sixth (2850/17,102) of the total number of conserved duplications in the A-genome, yet only 2.9% (174/6072) of the deletions. Given that chromosome 13 comprises a mere 7.8% of the <em>G. raimondii</em> reference sequence (<a id="xref-ref-23-12" class="xref-bibr" href="#ref-23">Paterson <em>et al.</em> 2012</a>), the suggestion arises that there has been exceptional expansion and/or contraction of this chromosome during the evolution of the two-fold size difference that distinguishes the A-genomes and D-genomes. Although some of this difference certainly reflects the expansion of transposable elements in the A-genome or possible contraction in the D-genome (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hawkins <em>et al.</em> 2009</a>), it is unclear what genomic features have allowed more numerous rearrangements in chromosome 13 than within other chromosomes in the genome.</p><p id="p-45">In addition to this broad-scale view of the contributions of duplications and deletions to <em>Gossypium</em> genome evolution, the data presented here offer a rich database that can be mined for potentially significant gene duplication and deletion. For example, gene loss has been associated with polyploidization (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Shaked <em>et al.</em> 2001</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ozkan 2003</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Han <em>et al.</em> 2005</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Tate <em>et al.</em> 2009</a>), but the deletions we have described in the A-genome occurred before polyploidization and include parts of approximately 1300 genes per accession. If these A-genome accessions were used as a "parental genome reference" for investigations of polyploidy, then the deletions common to the A<sub>1</sub>-genomes, A<sub>2</sub>-genomes, and ancestral A<sub>T</sub>-genomes would be confounded with any putative deletions that occurred as a result of polyploidization. Thus, this initial database of duplications and deletions will be a useful research tool for investigations of the evolution of the Gossypium genome.</p><p id="p-46">We observed that several genes involved in starch synthesis were deleted in the A-genome diploids, including seven genes with 1,4-alpha-glucan branching enzyme activity (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007229/-/DC1/TableS3.xlsx">Table S3</a>). It is tempting to speculate that these deletions increased the amount of glucose available in the A-genome diploid for cellulose synthesis and thereby played a role in the increased length of mature A-genome cotton fibers. Previous studies have documented altered carbon partitioning (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Yong-Ling Ruan 1998</a>) and altered starch accumulation (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Chaohua <em>et al.</em> 2005</a>) in fiberless <em>G. hirsutum</em> mutants. The deletion of starch genes in the A-genome may have been associated with the opposite effect, resulting in more carbon being allocated to cellulose production and less to starch production. We caution that many of the deleted genes are members of gene families and the remaining paralogs may partially or fully compensate for their deletion in the A-genome diploids. Nevertheless, the deleted genes discovered in this study offer interesting avenues of future research of gene duplication and functional compensation.</p><p id="p-47">Among genes with altered stop codons, we detected an enriched number of genes having photosynthesis-related functions. It appears unlikely that the altered stop codons are attributable to horizontal transfer of chloroplast genes to the nucleus because only four have high similarity to chloroplast genes. Although a biological explanation for this enrichment remained a mystery, it was likely that a portion of the enrichment of photosynthesis GO terms was an artifact of the gene annotation process. For example, the actual stop codon location may have been ambiguous because the original annotation of these genes actually had no stop codon. Perhaps, the initial gene annotation effort was simply unable to identify the full coding sequence and subsequent updates will include corrections to the original annotations. Regardless of the initial annotation, the enrichment of photosynthesis GO terms in genes with altered stop codons was interesting, but it was not attributable to differences between the A-genomes, D-genomes, A<sub>T</sub>-genomes, and D<sub>T</sub>-genomes.</p><p id="p-48">We also identified homogenized genome regions from conversion events between the two homoeologous genomes of <em>G. hirsutum</em>. Gene conversion has been defined as the nonreciprocal transfer of genetic information between homologous sequences, leading to homogenization during meiotic or mitotic recombination (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Szostak <em>et al.</em> 1983</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chen <em>et al.</em> 2007</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hsu <em>et al.</em> 2010</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jacquemin <em>et al.</em> 2011</a>). Unlike analyses of pairs of genes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Drouin 2002</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Mondragon-Palomino and Gaut 2005</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Xu <em>et al.</em> 2008</a>), whole-genome analysis of gene conversion has been pioneered in rice (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Xu <em>et al.</em> 2008</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Wang <em>et al.</em> 2009</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Jacquemin <em>et al.</em> 2011</a>), a sequenced diploid genome with many closely related species also with sequenced genomes. By comparison of the diploid re-sequencing data to the publicly available WGS data of Maxxa Acala, we were able to identify conversion events between homoeologous sequences within the polyploid <em>Gossypium</em> genome using two different methods. The two methods of detecting homoeologous conversion events resulted in different directional biases.</p><p id="p-49">Using the first method, we had previously used SNPs to estimate that up to 5% of the polyploid transcriptome had experienced “homoeologous gene conversion” (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Salmon <em>et al.</em> 2010</a>; <a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Flagel <em>et al.</em> 2012</a>). In both previous studies, identification of autapomorphic SNPs was not possible because of limited diploid sequencing data. Based on our current data, the presence of autapomorphic SNPs (and a liberal method of identification) appeared to have caused an overestimate in the amount of homoeologous conversion in genic regions. Thus, the genome sequence of a definitive outgroup is needed to unambiguously identify regions of conversion using SNP information alone. One dimension of the conversion events and the multi-alignment resource for all genes is the identification of loci where one of the two allopolyploid genomes has “overwritten” the other via a mechanism of reciprocal or nonreciprocal “gene conversion.” At present, the functional consequences of these observations remain unexplored, but it is intriguing to ask whether these conversion events are functionally insignificant (which might, for example, be the case when only synonymous sites are involved) or if, instead, specific genes or regulatory sequences have been selectively “doubled” or “eliminated” by this unusual intergenomic aspect of allopolyploid speciation.</p><p id="p-50">This first method contains an inherent bias in favor of A<sub>T</sub>-biased conversion because of the greater genetic distance between D<sub>T</sub> and D<sub>5</sub> compared to the distance between A<sub>T</sub> and A<sub>1</sub> or A<sub>2</sub>. This bias, however, should only be approximately 50% based on our understanding of the genetic distances between A and A<sub>T</sub> <em>vs.</em> D and D<sub>T</sub> (the latter is 50% greater) (<a id="xref-ref-9-7" class="xref-bibr" href="#ref-9">Flagel <em>et al.</em> 2012</a>). In addition, the genotype pattern indicative of a conversion is indistinguishable from that caused by an autapomorphic SNP in the diploid species. For example, suppose an autapomorphic SNP in the A-genome ancestor of A<sub>1</sub> and A<sub>2</sub> (not shared with A<sub>T</sub>) changes a C to a T at a given locus. Consequently, the D-genome diploids, D<sub>T</sub>-genome, and A<sub>T</sub>-genome would all have a C, whereas the A-genome diploids would have a T. So, the A<sub>T</sub> nucleotide would appear the same as D and D<sub>T</sub>, suggesting a D<sub>T</sub>-biased allele conversion event, even though they simply shared the ancestral allele. These confounding autapomorphic SNPs would have occurred after the divergence of A<sub>T</sub> but before the A<sub>1</sub>–A<sub>2</sub> split, suggesting a D<sub>T</sub>-biased conversion. However, an A<sub>T</sub>-biased conversion would be suggested by an autapomorphic SNP occurring after the divergence of D<sub>T</sub>, but before the most recent common ancestor of the D<sub>5</sub> diploids. Because the A-genome diploid is approximately a two-fold better approximation of the actual progenitor diploid than is the D-genome diploid (<a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Wendel <em>et al.</em> 2012</a>), these branch lengths are different (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). In fact, the phylogeny showed a distance of only 0.00447 for the branch corresponding to an autapomorphic SNP shared by all A diploids but not by A<sub>T</sub>. However, the equivalent branch in the D-genome clade has an autapomorphic SNP distance of 0.05385. These numbers suggest that, in the absence of any actual conversion events, there should be more than 12-times (0.05385/0.00447) as many SNPs that look like A<sub>T</sub>-biased conversion—because they are shared by A diploids but not by A<sub>T</sub>—as SNPs that look like D<sub>T</sub>-biased conversion. The difference between the 12-fold expected value for A<sub>T</sub>-biased conversions as visualized by the branch lengths to the five-fold observed value could be explained by a bias toward D<sub>T</sub>-biased conversion events, as reported elsewhere and as detected by the second coverage-based method. Thus, we consider the conversion events detected by the SNP-based method (method 1) to be inaccurate based on autapomorphic SNPs, but we consider the conversion events of the coverage method (method 2) to be a conservative, yet relatively accurate, assessment of conversion between the polyploid <em>G. hirsutum</em> genomes.</p><p id="p-51">The second method of conversion detection used deletion and coverage information to detect many separate events, and the direction of bias agreed with previous reports (<a id="xref-ref-23-13" class="xref-bibr" href="#ref-23">Paterson <em>et al.</em> 2012</a>). This method was very conservative and may represent a minimum amount of conversion events in the polyploid genome because of the uncertainty of the actual endpoints of conversion and the additional amounts of conversion suggested between homoeologous gene copies in the dN/dS analysis. The conversion events resulted in a loss of genomic diversity between the A<sub>T</sub>- and D<sub>T</sub>-genomes. Parts of at least 113 genes were included in conversion events between homoeologous chromosomes. Other investigations of genome evolution in rice have uncovered convergent evolution of ancient paralogs on Chr11 and Chr12 (∼2.1 Mb) mediated by gene conversion including up to 180 genes (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Jacquemin <em>et al.</em> 2011</a>). The conversion events we have described were more recent (after polyploidization 1–2 Ma) and our inference space was limited to a single species of <em>G. hirsutum</em>. It will be interesting to see if other polyploid Gossypium genomes also have the same conversion events in their genomes, and to estimate the rate of gene conversion between homoeologous genomes.</p><p id="p-52">Whole-genome re-sequencing of diploid <em>Gossypium</em> has identified insights into the genome evolution of cotton. These insights proved to be useful for characterization of the <em>G. hirsutum</em> genome via publicly available re-sequencing data. Additional <em>de novo</em> and re-sequencing efforts of polyploid Gossypium will continue to add to our understanding of the cotton genome, thereby enhancing our ability to manipulate the fiber and agronomic characteristics of cotton.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-53">We thank Brian Dalley of the Huntsman Cancer Institute for his assistance with DNA shearing and his generous insights regarding the Illumina HiSeq sequencing. We thank the Fulton SuperComputing Laboratory (FSL) at Brigham Young University for their generous support using the supercomputer. The FSL provided server environment for all of the included bioinformatics analyses described in this work. We thank the National Science Foundation and Cotton Inc. for their financial support of this work.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007229/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007229/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-54"><em>Communicating editor: J. A. Scott</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-06-18"><span class="received-label">Received </span>June 18, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-08-14"><span class="accepted-label">Accepted </span>August 14, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Page <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.1" data-doi="10.1139/g98-118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Comparative genetic mapping of allotetraploid cotton and its diploid progenitors</span>. <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">184</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft.volume%253D42%26rft.spage%253D184%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg98-118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g98-118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.2" data-doi="10.1007/s00122-011-1780-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byers</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harker</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yourstone</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maughan</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development and mapping of SNP assays in allotetraploid cotton</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">1201</span>–<span class="cit-lpage">1214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DByers%26rft.auinit1%253DR.%2BL.%26rft.volume%253D124%26rft.issue%253D7%26rft.spage%253D1201%26rft.epage%253D1214%26rft.atitle%253DDevelopment%2Band%2Bmapping%2Bof%2BSNP%2Bassays%2Bin%2Ballotetraploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1780-8%26rft_id%253Dinfo%253Apmid%252F22252442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1780-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22252442&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1809.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaohua</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiyuan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiyuan</span>  <span class="cit-name-given-names">N. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Observation of fiber ultrastructure of Ligon lintless mutant in upland cotton during fiber elongation</span>. <abbr class="cit-jnl-abbrev">Chin. Sci. Bull.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">126</span>–<span class="cit-lpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChin.%2BSci.%2BBull.%26rft.volume%253D50%26rft.spage%253D126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.4" data-doi="10.1038/nrg2193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">D. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chuzhanova</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Férec</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patrinos</span>  <span class="cit-name-given-names">G. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Gene conversion: mechanisms, evolution and human disease</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">762</span>–<span class="cit-lpage">775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DChen%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D762%26rft.epage%253D775%26rft.atitle%253DGene%2Bconversion%253A%2Bmechanisms%252C%2Bevolution%2Band%2Bhuman%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2193%26rft_id%253Dinfo%253Apmid%252F17846636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17846636&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1809.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.5" data-doi="10.1038/ng.2313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Leon</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Maize HapMap2 identifies extant variation from a genome in flux</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChia%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D803%26rft.epage%253D807%26rft.atitle%253DMaize%2BHapMap2%2Bidentifies%2Bextant%2Bvariation%2Bfrom%2Ba%2Bgenome%2Bin%2Bflux.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2313%26rft_id%253Dinfo%253Apmid%252F22660545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660545&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1809.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.6" data-doi="10.1093/bioinformatics/bti610"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conesa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gotz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Gomez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terol</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3674</span>–<span class="cit-lpage">3676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti610%26rft_id%253Dinfo%253Apmid%252F16081474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjEvMTgvMzY3NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTAvMTgwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.7" data-doi="10.1007/s00239-001-0085-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drouin</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Characterization of the gene conversions between the multigene family members of the yeast genome.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">14</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DDrouin%26rft.auinit1%253DG.%26rft.volume%253D55%26rft.issue%253D1%26rft.spage%253D14%26rft.epage%253D23%26rft.atitle%253DCharacterization%2Bof%2Bthe%2Bgene%2Bconversions%2Bbetween%2Bthe%2Bmultigene%2Bfamily%2Bmembers%2Bof%2Bthe%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-001-0085-y%26rft_id%253Dinfo%253Apmid%252F12165839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-001-0085-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12165839&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1809.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176243200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.8" data-doi="10.1111/j.1096-0031.1989.tb00562.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">PHYLIP–Phylogeny Inference Package (version 3.2)</span>. <abbr class="cit-jnl-abbrev">Cladistics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCladistics%26rft.volume%253D5%26rft.spage%253D163%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1096-0031.1989.tb00562.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1096-0031.1989.tb00562.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.9" data-doi="10.1186/1471-2164-13-302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Duplicate gene evolution, homoeologous recombination, and transcriptome characterization in allopolyploid cotton</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DFlagel%26rft.auinit1%253DL.%2BE.%26rft.volume%253D13%26rft.spage%253D302%26rft.epage%253D302%26rft.atitle%253DDuplicate%2Bgene%2Bevolution%252C%2Bhomoeologous%2Brecombination%252C%2Band%2Btranscriptome%2Bcharacterization%2Bin%2Ballopolyploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-302%26rft_id%253Dinfo%253Apmid%252F22768919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22768919&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1809.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grover</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Recent insights into mechanisms of genome size change in plants</span>. <abbr class="cit-jnl-abbrev">J. Bot.</abbr> <span class="cit-vol">2010</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBot.%26rft.volume%253D2010%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.11" data-doi="10.1007/s00606-012-0615-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grover</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grupp</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wanzek</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Assessing the monophyly of polyploid Gossypium species</span>. <abbr class="cit-jnl-abbrev">Plant Syst. Evol.</abbr> <span class="cit-vol">298</span>: <span class="cit-fpage">1177</span>–<span class="cit-lpage">1183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSyst.%2BEvol.%26rft.volume%253D298%26rft.spage%253D1177%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00606-012-0615-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00606-012-0615-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.12" data-doi="10.1534/genetics.104.039263"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fedak</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Rapid and repeatable elimination of a parental genome-specific DNA repeat (pGc1R–1a) in newly synthesized wheat allopolyploids</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">1239</span>–<span class="cit-lpage">1245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.039263%26rft_id%253Dinfo%253Apmid%252F15911583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MC8zLzEyMzkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE4MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.13" data-doi="10.1186/1471-2105-10-356"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zmasek</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">phyloXML: XML for evolutionary biology and comparative genomics</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHan%26rft.auinit1%253DM.%2BV.%26rft.volume%253D10%26rft.spage%253D356%26rft.epage%253D356%26rft.atitle%253DphyloXML%253A%2BXML%2Bfor%2Bevolutionary%2Bbiology%2Band%2Bcomparative%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-356%26rft_id%253Dinfo%253Apmid%252F19860910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-356&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19860910&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1809.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.14" data-doi="10.1101/gr.5282906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hawkins</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nason</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wing</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Differential lineage-specific amplification of transposable elements is responsible for genome size variation in Gossypium.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1252</span>–<span class="cit-lpage">1261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5282906%26rft_id%253Dinfo%253Apmid%252F16954538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNi8xMC8xMjUyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMC8xODA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.15" data-doi="10.1073/pnas.0904339106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hawkins</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proulx</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rapp</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Rapid DNA loss as a counterbalance to genome expansion through retrotransposon proliferation in plants.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">17811</span>–<span class="cit-lpage">17816</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0904339106%26rft_id%253Dinfo%253Apmid%252F19815511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzQyLzE3ODExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMC8xODA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.16" data-doi="10.1089/cmb.2010.0103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">C.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hardison</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">An effective method for detecting gene conversion events in whole genomes</span>. <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1281</span>–<span class="cit-lpage">1297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DHsu%26rft.auinit1%253DC.%2BH.%26rft.volume%253D17%26rft.issue%253D9%26rft.spage%253D1281%26rft.epage%253D1297%26rft.atitle%253DAn%2Beffective%2Bmethod%2Bfor%2Bdetecting%2Bgene%2Bconversion%2Bevents%2Bin%2Bwhole%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2010.0103%26rft_id%253Dinfo%253Apmid%252F20874409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2010.0103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20874409&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1809.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.17" data-doi="10.1093/molbev/msr144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacquemin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaparro</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laudie</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gavory</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Long-range and targeted ectopic recombination between the two homeologous chromosomes 11 and 12 in Oryza species</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3139</span>–<span class="cit-lpage">3150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr144%26rft_id%253Dinfo%253Apmid%252F21616911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyOC8xMS8zMTM5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMC8xODA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.18" data-doi="10.1101/gr.092759.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krzywinski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birol</span>  <span class="cit-name-given-names">İ.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connors</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascoyne</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Circos: an information aesthetic for comparative genomics.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1639</span>–<span class="cit-lpage">1645</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.092759.109%26rft_id%253Dinfo%253Apmid%252F19541911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTYzOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTAvMTgwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.19" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTAvMTgwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.20" data-doi="10.1093/molbev/msi241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mondragon-Palomino</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Gene conversion and the evolution of three leucine-rich repeat gene families in <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2444</span>–<span class="cit-lpage">2456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi241%26rft_id%253Dinfo%253Apmid%252F16120808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMi8yNDQ0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMC8xODA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.21" data-doi="10.1093/jhered/esg053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ozkan</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Nonadditive changes in genome size during allopolyploidization in the wheat (Aegilops-triticum) group</span>. <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">260</span>–<span class="cit-lpage">264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DOzkan%26rft.auinit1%253DH.%26rft.volume%253D94%26rft.issue%253D3%26rft.spage%253D260%26rft.epage%253D264%26rft.atitle%253DNonadditive%2BChanges%2Bin%2BGenome%2BSize%2BDuring%2BAllopolyploidization%2Bin%2Bthe%2BWheat%2B%2528Aegilops-Triticum%2529%2BGroup%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesg053%26rft_id%253Dinfo%253Apmid%252F12816968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6Ijk0LzMvMjYwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMC8xODA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-other" id="cit-3.10.1809.22"><div class="cit-metadata unstructured">Page, J. T., A. R. Gingle, and J. A. Udall, 2013 PolyCat: A resource for genome categorization of sequencing reads from allopolyploid organisms. G3 (Bethesda) 3: 517–525.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.23" data-doi="10.1038/nature11798"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gundlach</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibres</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">492</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D492%26rft.spage%253D423%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11798%26rft_id%253Dinfo%253Apmid%252F23257886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11798&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23257886&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1809.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312488200055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.24" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzg0MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTAvMTgwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.25" data-doi="10.1111/j.1469-8137.2009.03093.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salmon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ying</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Homoeologous nonreciprocal recombination in polyploid cotton</span>. <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DSalmon%26rft.auinit1%253DA.%26rft.volume%253D186%26rft.issue%253D1%26rft.spage%253D123%26rft.epage%253D134%26rft.atitle%253DHomoeologous%2Bnonreciprocal%2Brecombination%2Bin%2Bpolyploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2009.03093.x%26rft_id%253Dinfo%253Apmid%252F19925554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2009.03093.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19925554&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1809.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275184700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.26" data-doi="10.1093/molbev/msg065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Senchina</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alverez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cronn</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Rate variation among nuclear genes and the age of polyploidy in gossypium</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">633</span>–<span class="cit-lpage">643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg065%26rft_id%253Dinfo%253Apmid%252F12679546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIwLzQvNjMzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMC8xODA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.27" data-doi="10.1105/tpc.13.8.1749"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shaked</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kashkush</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozkan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Sequence elimination and cytosine methylation are rapid and reproducible responses of the genome to wide hybridization and allopolyploidy in wheat</span>. <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1749</span>–<span class="cit-lpage">1759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DShaked%26rft.auinit1%253DH.%26rft.volume%253D13%26rft.issue%253D8%26rft.spage%253D1749%26rft.epage%253D1759%26rft.atitle%253DSequence%2BElimination%2Band%2BCytosine%2BMethylation%2BAre%2BRapid%2Band%2BReproducible%2BResponses%2Bof%2Bthe%2BGenome%2Bto%2BWide%2BHybridization%2Band%2BAllopolyploidy%2Bin%2BWheat%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.13.8.1749%26rft_id%253Dinfo%253Apmid%252F11487690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjEzLzgvMTc0OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTAvMTgwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.28" data-doi="10.1016/0092-8674(83)90331-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szostak</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orr-Weaver</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothstein</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">F. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">The double-strand-break repair model for recombination</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSzostak%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D35%26rft.atitle%253DThe%2Bdouble-strand-break%2Brepair%2Bmodel%2Bfor%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252883%252990331-8%26rft_id%253Dinfo%253Apmid%252F6380756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(83)90331-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6380756&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1809.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983QS06800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.29" data-doi="10.1186/1471-2229-9-80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tate</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joshi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">On the road to diploidization? Homoeolog loss in independently formed populations of the allopolyploid Tragopogon miscellus (Asteraceae)</span>. <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DTate%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D9%26rft.spage%253D80%26rft.epage%253D80%26rft.atitle%253DOn%2Bthe%2Broad%2Bto%2Bdiploidization%253F%2BHomoeolog%2Bloss%2Bin%2Bindependently%2Bformed%2Bpopulations%2Bof%2Bthe%2Ballopolyploid%2BTragopogon%2Bmiscellus%2B%2528Asteraceae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-80%26rft_id%253Dinfo%253Apmid%252F19558696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-80&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19558696&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1809.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.30" data-doi="10.1101/gr.4602906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A global assembly of cotton ESTs</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">441</span>–<span class="cit-lpage">450</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4602906%26rft_id%253Dinfo%253Apmid%252F16478941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzMvNDQxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMC8xODA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.31" data-doi="10.1186/1471-2229-9-125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Deynze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkins</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sampling nucleotide diversity in cotton.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DVan%2BDeynze%26rft.auinit1%253DA.%2BE.%26rft.volume%253D9%26rft.spage%253D125%26rft.epage%253D125%26rft.atitle%253DSampling%2Bnucleotide%2Bdiversity%2Bin%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-125%26rft_id%253Dinfo%253Apmid%252F19840401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19840401&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1809.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.32" data-doi="10.1101/gr.087288.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comparative inference of illegitimate recombination between rice and sorghum duplicated genes produced by polyploidization.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1026</span>–<span class="cit-lpage">1032</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.087288.108%26rft_id%253Dinfo%253Apmid%252F19372385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTAyNiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTAvMTgwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.33" data-doi="10.1073/pnas.86.11.4132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">New World tetraploid cottons contain Old World cytoplasm.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">4132</span>–<span class="cit-lpage">4136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWendel%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D86%26rft.issue%253D11%26rft.spage%253D4132%26rft.epage%253D4136%26rft.atitle%253DNew%2BWorld%2Btetraploid%2Bcottons%2Bcontain%2BOld%2BWorld%2Bcytoplasm%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.86.11.4132%26rft_id%253Dinfo%253Apmid%252F16594050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODYvMTEvNDEzMiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTAvMTgwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-3.10.1809.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cronn</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Polyploidy and the evolutionary history of cotton</span>, pp. <span class="cit-fpage">139</span>–<span class="cit-lpage">186</span> in <span class="cit-source">Advances in Agronomy</span>, ed. D. Sparks, <span class="cit-publ-name">Elsevier, New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-3.10.1809.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">K. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Jeans, genes, and genomes: cotton as a model for studying polyploidy</span>, pp. <span class="cit-fpage">181</span>–<span class="cit-lpage">207</span> in <span class="cit-source">Polyploidy and Genome Evolution</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">P. S.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.36" data-doi="10.2307/2444478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">J. McD.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Genetic diversity, introgression, and independent domestication of old world cultivated cottons</span>. <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">1795</span>–<span class="cit-lpage">1806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft.volume%253D76%26rft.spage%253D1795%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2444478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2444478&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1989CG87700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.37" data-doi="10.1093/bioinformatics/btq057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nacu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and SNP-tolerant detection of complex variants and splicing in short reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">873</span>–<span class="cit-lpage">881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq057%26rft_id%253Dinfo%253Apmid%252F20147302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi83Lzg3MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTAvMTgwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.38" data-doi="10.1186/1471-2164-9-93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Gene conversion in the rice genome</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.spage%253D93%26rft.epage%253D93%26rft.atitle%253DGene%2Bconversion%2Bin%2Bthe%2Brice%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-93%26rft_id%253Dinfo%253Apmid%252F18298833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-93&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18298833&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1809.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.39" data-doi="10.1104/pp.118.2.399"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yong-Ling Ruan</span>  <span class="cit-name-given-names">P. S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A fiberless seed mutation in cotton is associated with lack of fiber cell initiation in ovule epidermis and alterations in sucrose synthase expression and carbon partitioning in developing seeds</span>. <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DRuan%26rft.auinit1%253DY.-L.%26rft.volume%253D118%26rft.issue%253D2%26rft.spage%253D399%26rft.epage%253D406%26rft.atitle%253DA%2BFiberless%2BSeed%2BMutation%2Bin%2BCotton%2BIs%2BAssociated%2Bwith%2BLack%2Bof%2BFiber%2BCell%2BInitiation%2Bin%2BOvule%2BEpidermis%2Band%2BAlterations%2Bin%2BSucrose%2BSynthase%2BExpression%2Band%2BCarbon%2BPartitioning%2Bin%2BDeveloping%2BSeeds%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.118.2.399%26rft_id%253Dinfo%253Apmid%252F9765525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMTgvMi8zOTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE4MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1809.40" data-doi="10.1186/gb-2008-9-9-r137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eeckhoute</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Model-based analysis of ChIP-Seq (MACS)</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">R137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253DR137%26rft.epage%253DR137%26rft.atitle%253DModel-based%2Banalysis%2Bof%2BChIP-Seq%2B%2528MACS%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2008-9-9-r137%26rft_id%253Dinfo%253Apmid%252F18798982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2008-9-9-r137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18798982&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1809.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/10/1809.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/10/1795" title="Imputation-Based Genomic Coverage Assessments of Current Human Genotyping Arrays" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/10/1819" title="Identification of a Novel Polymorphism in X-Linked Sterol-4-Alpha-Carboxylate 3-Dehydrogenase (Nsdhl) Associated with Reduced High-Density Lipoprotein Cholesterol Levels in I/LnJ Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/10"><span>Volume 3 Issue 10, October 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/10.cover-source.jpg?itok=h5LtrAoT" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/10/1809.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-syFyLZM1ok1E0Lxb5901PzMPc_2zyYcEmFV6ddgWcQA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/10/1809" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/10/1809" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/10/1809" class="active" data-icon-position="" data-hide-link-title="0">Insights into the Evolution of Cotton Diploids and Polyploids from Whole-Genome Re-sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459539" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-RQDPFAj2DsbEoF4ShwvuXHq4yEFZo9pJ6Qttd1JYIzA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/10/1809.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/10/1809" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-5G2J8RHkL9_fD4HcTNXWUGx_k6o1FjKkY6EmIst2woA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/10/1809" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/10/1809.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459539" id="node459539--2" data-pisa="ggg;3/10/1809" data-pisa-master="ggg;g3.113.007229" data-apath="/ggg/3/10/1809.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Insights into the Evolution of Cotton Diploids and Polyploids from Whole-Genome Re-sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Justin T. Page</span>, <span class="highwire-citation-author" data-delta="1">Mark D. Huynh</span>, <span class="highwire-citation-author" data-delta="2">Zach S. Liechty</span>, <span class="highwire-citation-author" data-delta="3">Kara Grupp</span>, <span class="highwire-citation-author" data-delta="4">David Stelly</span>, <span class="highwire-citation-author" data-delta="5">Amanda M. Hulse</span>, <span class="highwire-citation-author" data-delta="6">Hamid Ashrafi</span>, <span class="highwire-citation-author" data-delta="7">Allen Van Deynze</span>, <span class="highwire-citation-author" data-delta="8">Jonathan F. Wendel</span> and <span class="highwire-citation-author" data-delta="9">Joshua A. Udall</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1809-1818;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007229 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459539--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Justin T. Page </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biology Department, Brigham Young University, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BT.%2BPage%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Page%20JT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BT.%2BPage%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mark D. Huynh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BD.%2BHuynh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huynh%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BD.%2BHuynh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Zach S. Liechty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zach%2BS.%2BLiechty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liechty%20ZS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZach%2BS.%2BLiechty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kara Grupp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kara%2BGrupp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grupp%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKara%2BGrupp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">David Stelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Soil and Crop Sciences, Texas AgriLife Research, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BStelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stelly%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BStelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Amanda M. Hulse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Soil and Crop Sciences, Texas AgriLife Research, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BM.%2BHulse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hulse%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmanda%2BM.%2BHulse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Hamid Ashrafi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Seed Biotechnology Center, University of California-Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamid%2BAshrafi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashrafi%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHamid%2BAshrafi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Allen Van Deynze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Seed Biotechnology Center, University of California-Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BVan%2BDeynze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Deynze%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAllen%2BVan%2BDeynze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jonathan F. Wendel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BF.%2BWendel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wendel%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BF.%2BWendel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Joshua A. Udall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biology Department, Brigham Young University, Provo, Utah 84602</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BA.%2BUdall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Udall%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoshua%2BA.%2BUdall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jaudall@byu.edu" class="" data-icon-position="" data-hide-link-title="0">jaudall@byu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/10/1809" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/10/1809&amp;title=Insights%20into%20the%20Evolution%20of%20Cotton%20Diploids%20and%20Polyploids%20from%20Whole-Genome%20Re-sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/10/1809&amp;title=Insights%20into%20the%20Evolution%20of%20Cotton%20Diploids%20and%20Polyploids%20from%20Whole-Genome%20Re-sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/10/1809&amp;title=Insights%20into%20the%20Evolution%20of%20Cotton%20Diploids%20and%20Polyploids%20from%20Whole-Genome%20Re-sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/10/1809&amp;text=Insights%20into%20the%20Evolution%20of%20Cotton%20Diploids%20and%20Polyploids%20from%20Whole-Genome%20Re-sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/10/1809&amp;title=Insights%20into%20the%20Evolution%20of%20Cotton%20Diploids%20and%20Polyploids%20from%20Whole-Genome%20Re-sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/10/1809&amp;t=Insights%20into%20the%20Evolution%20of%20Cotton%20Diploids%20and%20Polyploids%20from%20Whole-Genome%20Re-sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/10/1809&amp;title=Insights%20into%20the%20Evolution%20of%20Cotton%20Diploids%20and%20Polyploids%20from%20Whole-Genome%20Re-sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/10/1809&amp;title=Insights%20into%20the%20Evolution%20of%20Cotton%20Diploids%20and%20Polyploids%20from%20Whole-Genome%20Re-sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459539/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459539" id="node459539--3" data-pisa="ggg;3/10/1809" data-pisa-master="ggg;g3.113.007229" data-apath="/ggg/3/10/1809.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Insights into the Evolution of Cotton Diploids and Polyploids from Whole-Genome Re-sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Justin T. Page</span>, <span class="highwire-citation-author" data-delta="1">Mark D. Huynh</span>, <span class="highwire-citation-author" data-delta="2">Zach S. Liechty</span>, <span class="highwire-citation-author" data-delta="3">Kara Grupp</span>, <span class="highwire-citation-author" data-delta="4">David Stelly</span>, <span class="highwire-citation-author" data-delta="5">Amanda M. Hulse</span>, <span class="highwire-citation-author" data-delta="6">Hamid Ashrafi</span>, <span class="highwire-citation-author" data-delta="7">Allen Van Deynze</span>, <span class="highwire-citation-author" data-delta="8">Jonathan F. Wendel</span> and <span class="highwire-citation-author" data-delta="9">Joshua A. Udall</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1809-1818;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007229 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459539--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Justin T. Page </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biology Department, Brigham Young University, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BT.%2BPage%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Page%20JT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BT.%2BPage%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mark D. Huynh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BD.%2BHuynh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huynh%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BD.%2BHuynh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Zach S. Liechty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zach%2BS.%2BLiechty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liechty%20ZS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZach%2BS.%2BLiechty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kara Grupp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kara%2BGrupp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grupp%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKara%2BGrupp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">David Stelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Soil and Crop Sciences, Texas AgriLife Research, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BStelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stelly%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BStelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Amanda M. Hulse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Soil and Crop Sciences, Texas AgriLife Research, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BM.%2BHulse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hulse%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmanda%2BM.%2BHulse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Hamid Ashrafi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Seed Biotechnology Center, University of California-Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamid%2BAshrafi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashrafi%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHamid%2BAshrafi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Allen Van Deynze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Seed Biotechnology Center, University of California-Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BVan%2BDeynze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Deynze%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAllen%2BVan%2BDeynze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jonathan F. Wendel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BF.%2BWendel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wendel%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BF.%2BWendel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Joshua A. Udall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biology Department, Brigham Young University, Provo, Utah 84602</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BA.%2BUdall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Udall%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoshua%2BA.%2BUdall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jaudall@byu.edu" class="" data-icon-position="" data-hide-link-title="0">jaudall@byu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1809&amp;rft.epage=1818&amp;rft.atitle=Insights%20into%20the%20Evolution%20of%20Cotton%20Diploids%20and%20Polyploids%20from%20Whole-Genome%20Re-sequencing&amp;rft.volume=3&amp;rft.issue=10&amp;rft.date=2013-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Page%2C+Justin+T.&amp;rft.au=Huynh%2C+Mark+D.&amp;rft.au=Liechty%2C+Zach+S.&amp;rft.au=Grupp%2C+Kara&amp;rft.au=Stelly%2C+David&amp;rft.au=Hulse%2C+Amanda+M.&amp;rft.au=Ashrafi%2C+Hamid&amp;rft.au=Van+Deynze%2C+Allen&amp;rft.au=Wendel%2C+Jonathan+F.&amp;rft.au=Udall%2C+Joshua+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459539/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459539/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459539/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459539/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459539/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459539/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459539/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459539/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459539/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459539/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459539/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459539/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/10/1809" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/10/1809.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/10/1809.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/10/1809.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/10/1809.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
