<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/10/1759" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/10/1759.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/10/1759.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/10/1759.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/10/1759.atom" />
<meta name="HW.pisa" content="ggg;3/10/1759" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Whole-Genome DNA Marker Map for Cotton Based on the D-Genome Sequence of Gossypium raimondii L." />
<meta name="DC.Identifier" content="10.1534/g3.113.006890" />
<meta name="DC.Date" content="2013-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2013 Wang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="10.1534/g3.114.013714" />
<meta name="DC.Relation" content="10.1534/g3.114.014167" />
<meta name="DC.Description" content="We constructed a very-high-density, whole-genome marker map (WGMM) for cotton by using 18,597 DNA markers corresponding to 48,958 loci that were aligned to both a consensus genetic map and a reference genome sequence. The WGMM has a density of one locus per 15.6 kb, or an average of 1.3 loci per gene. The WGMM was anchored by the use of colinear markers to a detailed genetic map, providing recombinational information. Mapped markers occurred at relatively greater physical densities in distal chromosomal regions and lower physical densities in the central regions, with all 1 Mb bins having at least nine markers. Hotspots for quantitative trait loci and resistance gene analog clusters were aligned to the map and DNA markers identified for targeting of these regions of high practical importance. Based on the cotton D genome reference sequence, the locations of chromosome structural rearrangements plotted on the map facilitate its translation to other Gossypium genome types. The WGMM is a versatile genetic map for marker assisted breeding, fine mapping and cloning of genes and quantitative trait loci, developing new genetic markers and maps, genome-wide association mapping, and genome evolution studies." />
<meta name="DC.Contributor" content="Zining Wang" />
<meta name="DC.Contributor" content="Dong Zhang" />
<meta name="DC.Contributor" content="Xiyin Wang" />
<meta name="DC.Contributor" content="Xu Tan" />
<meta name="DC.Contributor" content="Hui Guo" />
<meta name="DC.Contributor" content="Andrew H. Paterson" />
<meta name="article:published_time" content="2013-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Whole-Genome DNA Marker Map for Cotton Based on the D-Genome Sequence of Gossypium raimondii L." />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We constructed a very-high-density, whole-genome marker map (WGMM) for cotton by using 18,597 DNA markers corresponding to 48,958 loci that were aligned to both a consensus genetic map and a reference genome sequence. The WGMM has a density of one locus per 15.6 kb, or an average of 1.3 loci per gene. The WGMM was anchored by the use of colinear markers to a detailed genetic map, providing recombinational information. Mapped markers occurred at relatively greater physical densities in distal chromosomal regions and lower physical densities in the central regions, with all 1 Mb bins having at least nine markers. Hotspots for quantitative trait loci and resistance gene analog clusters were aligned to the map and DNA markers identified for targeting of these regions of high practical importance. Based on the cotton D genome reference sequence, the locations of chromosome structural rearrangements plotted on the map facilitate its translation to other Gossypium genome types. The WGMM is a versatile genetic map for marker assisted breeding, fine mapping and cloning of genes and quantitative trait loci, developing new genetic markers and maps, genome-wide association mapping, and genome evolution studies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/10/01" />
<meta name="citation_mjid" content="ggg;3/10/1759" />
<meta name="citation_id" content="3/10/1759" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/10/1759" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/10/1759.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/10/1759.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/10/1759.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.006890" />
<meta name="citation_pmid" content="23979945" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1759" />
<meta name="citation_lastpage" content="1767" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zining Wang" />
<meta name="citation_author_institution" content="Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605" />
<meta name="citation_author" content="Dong Zhang" />
<meta name="citation_author_institution" content="Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605" />
<meta name="citation_author" content="Xiyin Wang" />
<meta name="citation_author_institution" content="Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605" />
<meta name="citation_author" content="Xu Tan" />
<meta name="citation_author_institution" content="Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605" />
<meta name="citation_author" content="Hui Guo" />
<meta name="citation_author_institution" content="Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605" />
<meta name="citation_author" content="Andrew H. Paterson" />
<meta name="citation_author_institution" content="Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605" />
<meta name="citation_author_email" content="paterson@plantbio.uga.edu" />
<meta name="citation_reference" content="citation_journal_title=Plant cell reports;citation_journal_abbrev=Plant Cell Rep;citation_author=M. Agarwal;citation_author=N. Shrivastava;citation_author=H. Padh;citation_title=Advances in molecular marker techniques and their applications in plant sciences.;citation_pages=617-631;citation_volume=27;citation_year=2008;citation_issue=4;citation_pmid=18246355;citation_doi=10.1007/s00299-008-0507-z" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Biol.;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=225;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. Andolfatto;citation_author=D. Davison;citation_author=D. Erezyilmaz;citation_author=T. T. Hu;citation_author=J. Mast;citation_title=Multiplexed shotgun genotyping for rapid and efficient genetic mapping.;citation_volume=21;citation_year=2011;citation_pmid=21233398;citation_doi=10.1101/gr.115402.110" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. Bateman;citation_author=L. Coin;citation_author=R. Durbin;citation_author=R. D. Finn;citation_author=V. Hollich;citation_title=The Pfam protein families database;citation_pages=D138-141;citation_volume=32;citation_year=2004;citation_issue=Database issue;citation_pmid=14681378;citation_doi=10.1093/nar/gkh121" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=L. A. Becerra Lopez-Lavalle;citation_author=B. Matheson;citation_author=C. L. Brubaker;citation_title=A genetic map of an Australian wild Gossypium C genome and assignment of homoeologies with tetraploid cultivated cotton.;citation_pages=779-794;citation_volume=54;citation_year=2011;citation_issue=9;citation_pmid=21905768" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. Blenda;citation_author=D. D. Fang;citation_author=J. F. Rami;citation_author=O. Garsmeur;citation_author=F. Luo;citation_title=A high density consensus genetic map of tetraploid cotton that integrates multiple component maps through molecular marker redundancy check.;citation_pages=e45739-e45739;citation_volume=7;citation_year=2012;citation_issue=9;citation_pmid=23029214;citation_doi=10.1371/journal.pone.0045739" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=C. L. Brubaker;citation_author=A. H. Paterson;citation_author=J. F. Wendel;citation_title=Comparative genetic mapping of allotetraploid cotton and its diploid progenitors.;citation_pages=184-203;citation_volume=42;citation_year=1999;citation_doi=10.1139/g98-118" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. Bus;citation_author=J. Hecht;citation_author=B. Huettel;citation_author=R. Reinhardt;citation_author=B. Stich;citation_title=High-throughput polymorphism detection and genotyping in Brassica napus using next-generation RAD sequencing.;citation_pages=281-281;citation_volume=13;citation_year=2012;citation_pmid=22726880;citation_doi=10.1186/1471-2164-13-281" />
<meta name="citation_reference" content="citation_journal_title=Acta Agron. Sin.;citation_author=L. Chen;citation_author=Z. S. Zhang;citation_author=M. C. Hu;citation_author=W. Wang;citation_author=J. Zhang;citation_title=Genetic linkage map construction and QTL mapping for yield and fiber quality in upland cotton.;citation_pages=1199-1205;citation_volume=34;citation_year=2008;citation_doi=10.3724/SP.J.1006.2008.01199" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=A. Desai;citation_author=P. W. Chee;citation_author=J. Rong;citation_author=O. L. May;citation_author=A. H. Paterson;citation_title=Chromosome structural changes in diploid and tetraploid A genomes of Gossypium.;citation_pages=336-345;citation_volume=49;citation_year=2006;citation_issue=4;citation_pmid=16699553" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=C. Duran;citation_author=D. Edwards;citation_author=J. Batley;citation_title=Genetic maps and the use of synteny.;citation_pages=41-55;citation_volume=513;citation_year=2009;citation_pmid=19347649;citation_doi=10.1007/978-1-59745-427-8_3" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=S. R. Eddy;citation_title=Profile hidden Markov models;citation_pages=755-763;citation_volume=14;citation_year=1998;citation_issue=9;citation_pmid=9918945;citation_doi=10.1093/bioinformatics/14.9.755" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Guo;citation_author=C. Cai;citation_author=C. Wang;citation_author=Z. Han;citation_author=X. Song;citation_title=A microsatellite-based, gene-rich linkage map reveals genome structure, function and evolution in Gossypium.;citation_volume=176;citation_year=2007;citation_pmid=17409069;citation_doi=10.1534/genetics.107.070375" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=Z. G. Han;citation_author=W. Z. Guo;citation_author=X. L. Song;citation_author=T. Z. Zhang;citation_title=Genetic mapping of EST-derived microsatellites from the diploid Gossypium arboreum in allotetraploid cotton.;citation_pages=308-327;citation_volume=272;citation_year=2004;citation_issue=3;citation_pmid=15368122;citation_doi=10.1007/s00438-004-1059-8" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=D. H. He;citation_author=Z. L. Xin;citation_author=X. L. Zhang;citation_author=Y. C. Nie;citation_author=X. P. Guo;citation_title=QTL mapping for economic traits based on a dense genetic map of cotton with PCR-based markers using the interspecific cross of Gossypium hirsutum Ã Gossypium barbadense.;citation_pages=181-197;citation_volume=153;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=C. Jiang;citation_author=R. J. Wright;citation_author=K. M. El-Zik;citation_author=A. H. Paterson;citation_title=Polyploid formation created unique avenues for response to selection in Gossypium (cotton);citation_pages=4419-4424;citation_volume=95;citation_year=1998;citation_issue=8;citation_pmid=9539752;citation_doi=10.1073/pnas.95.8.4419" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=J. M. Lacape;citation_author=T. B. Nguyen;citation_author=S. Thibivilliers;citation_author=B. M. Bojinov;citation_author=B. Courtois;citation_title=A combined RFLP-SSR-AFLP map of tetraploid cotton based on a Gossypium hirsutum x Gossypium barbadense backcross population.;citation_pages=612-626;citation_volume=46;citation_year=2003;citation_issue=4;citation_pmid=12897870;citation_doi=10.1139/g03&amp;ndash;050" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. M. Lacape;citation_author=J. Jacobs;citation_author=T. Arioli;citation_author=R. Derijcker;citation_author=N. Forestier-Chiron;citation_title=A new interspecific, Gossypium hirsutum x G. barbadense, RIL population: towards a unified consensus linkage map of tetraploid cotton.;citation_pages=281-292;citation_volume=119;citation_year=2009;citation_issue=2;citation_pmid=19387609;citation_doi=10.1007/s00122-009-1037-y" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. Liu;citation_author=B. Wang;citation_author=W. Guo;citation_author=L. Wang;citation_author=T. Zhang;citation_title=Differential gene expression and associated QTL mapping for cotton yield based on a cDNA-AFLP transcriptome map in an immortalized F2.;citation_pages=439-454;citation_volume=123;citation_year=2011;citation_issue=3;citation_pmid=21512772;citation_doi=10.1007/s00122-011-1597-5" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. B. Nguyen;citation_author=M. Giband;citation_author=P. Brottier;citation_author=A. M. Risterucci;citation_author=J. M. Lacape;citation_title=Wide coverage of the tetraploid cotton genome using newly developed microsatellite markers.;citation_pages=167-175;citation_volume=109;citation_year=2004;citation_issue=1;citation_pmid=14997301;citation_doi=10.1007/s00122-004-1612-1" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. H. Paterson;citation_title=Leafing through the genomes of our major crop plants: strategies for capturing unique information.;citation_pages=174-184;citation_volume=7;citation_year=2006;citation_issue=3;citation_pmid=16485017;citation_doi=10.1038/nrg1806" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. H. Paterson;citation_author=J. F. Wendel;citation_author=H. Gundlach;citation_author=H. Guo;citation_author=J. Jenkins;citation_title=Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibres.;citation_pages=423-427;citation_volume=492;citation_year=2012;citation_pmid=23257886;citation_doi=10.1038/nature11798" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. A. Poland;citation_author=P. J. Bradbury;citation_author=E. S. Buckler;citation_author=R. J. Nelson;citation_title=Genome-wide nested association mapping of quantitative resistance to northern leaf blight in maize.;citation_volume=108;citation_year=2011;citation_pmid=21482771;citation_doi=10.1073/pnas.1010894108" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=U. K. Reddy;citation_author=J. K. Rong;citation_author=P. Nimmakayala;citation_author=G. Vajja;citation_author=M. A. Rahman;citation_title=Use of diversity arrays technology markers for integration into a cotton reference map and anchoring to a recombinant inbred line map.;citation_pages=349-359;citation_volume=54;citation_year=2011;citation_issue=5;citation_pmid=21524184" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. J. Reinisch;citation_author=J. M. Dong;citation_author=C. L. Brubaker;citation_author=D. M. Stelly;citation_author=J. F. Wendel;citation_title=A detailed RFLP map of cotton, Gossypium hirsutum x Gossypium barbadense: chromosome organization and evolution in a disomic polyploid genome.;citation_pages=829-847;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851778" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Rong;citation_author=C. Abbey;citation_author=J. E. Bowers;citation_author=C. L. Brubaker;citation_author=C. Chang;citation_title=A 3347-Locus Genetic Recombination Map of Sequence-Tagged Sites Reveals Features of Genome Organization, Transmission and Evolution of Cotton (Gossypium);citation_pages=389-417;citation_volume=166;citation_year=2004;citation_issue=1;citation_pmid=15020432;citation_doi=10.1534/genetics.166.1.389" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. Rong;citation_author=J. E. Bowers;citation_author=S. R. Schulze;citation_author=V. N. Waghmare;citation_author=C. J. Rogers;citation_title=Comparative genomics of Gossypium and Arabidopsis: unraveling the consequences of both ancient and recent polyploidy.;citation_volume=15;citation_year=2005;citation_pmid=16109973;citation_doi=10.1101/gr.3907305" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. Rong;citation_author=G. J. Pierce;citation_author=V. N. Waghmare;citation_author=C. J. Rogers;citation_author=A. Desai;citation_title=Genetic mapping and comparative analysis of seven mutants related to seed fiber development in cotton.;citation_pages=1137-1146;citation_volume=111;citation_year=2005;citation_issue=6;citation_pmid=16075204;citation_doi=10.1007/s00122-005-0041-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Rong;citation_author=F. A. Feltus;citation_author=V. N. Waghmare;citation_author=G. J. Pierce;citation_author=P. W. Chee;citation_title=Meta-analysis of polyploid cotton QTL shows unequal contributions of subgenomes to a complex network of genes and gene clusters implicated in lint fiber development.;citation_volume=176;citation_year=2007;citation_pmid=17565937;citation_doi=10.1534/genetics.107.074518" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=Z. W. Shappley;citation_author=J. N. Jenkins;citation_author=W. R. Meredith;citation_author=C. M. Jack;citation_title=An RFLP linkage map of upland cotton (Gossypium hirsutum L.).;citation_pages=756-761;citation_volume=97;citation_year=1998;citation_doi=10.1007/s001220050952" />
<meta name="citation_reference" content="citation_journal_title=J. Cotton Sci.;citation_author=Z. Shappley;citation_author=J. N. Jenkins;citation_author=J. Zu;citation_author=J. C. J. McCarty;citation_title=Quantita- D-subgenome bias of Xcm resistance genes in tetraploid gossypium tive trait loci associated with agronomic and fiber traits of Upland (cotton) suggests that polyploid formation has created novel avecotton.;citation_pages=153-163;citation_volume=2;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=X. L. Shen;citation_author=W. Z. Guo;citation_author=X. F. Zhu;citation_author=Y. L. Yuan;citation_author=J. Z. Yu;citation_title=Molecular mapping of QTLs for fiber qualities in three diverse lines in upland cotton using SSR markers.;citation_pages=169-181;citation_volume=15;citation_year=2005;citation_doi=10.1007/s11032-004-4731-0" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=X. L. Shen;citation_author=W. Z. Guo;citation_author=Q. X. Lu;citation_author=X. F. Zhu;citation_author=Y. L. Yuan;citation_title=Genetic mapping of quantitative trait loci for fiber quality and yield trait by RIL approach in upland cotton.;citation_pages=371-380;citation_volume=155;citation_year=2007;citation_doi=10.1007/s10681-006-9338-6" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Z. Sun;citation_author=Z. Wang;citation_author=J. Tu;citation_author=J. Zhang;citation_author=F. Yu;citation_title=An ultradense genetic recombination map for Brassica napus, consisting of 13551 SRAP markers.;citation_pages=1305-1317;citation_volume=114;citation_year=2007;citation_issue=8;citation_pmid=17426959;citation_doi=10.1007/s00122-006-0483-z" />
<meta name="citation_reference" content="citation_journal_title=J Cotton Sci;citation_author=M. Ulloa;citation_author=R. J. Meredith;citation_title=Genetic linkage map and QTL analysis of agronomic and fiber quality traits in an intraspecific population.;citation_pages=161-170;citation_volume=4;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. Ulloa;citation_author=W. R. Meredith;citation_author=Z. W. Shappley;citation_author=A. L. Kahler;citation_title=RFLP genetic linkage maps from four F(2.3) populations and a joinmap of Gossypium hirsutum L.;citation_pages=200-208;citation_volume=104;citation_year=2002;citation_issue=2-3;citation_pmid=12582687;citation_doi=10.1007/s001220100739" />
<meta name="citation_reference" content="citation_journal_title=Acta Agron. Sin.;citation_author=J. Wang;citation_author=W. Z. Guo;citation_author=T. Z. Zhang;citation_title=QTL mapping for fiber quality properties in cotton cultivar yumian 1.;citation_pages=1915-1921;citation_volume=33;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. Wang;citation_author=Z. Wang;citation_author=F. Li;citation_author=W. Ye;citation_author=J. Wang;citation_title=The draft genome of a diploid cotton Gossypium raimondii.;citation_pages=1098-1103;citation_volume=44;citation_year=2012;citation_issue=10;citation_pmid=22922876;citation_doi=10.1038/ng.2371" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=X. Wang;citation_author=X. Shi;citation_author=Z. Li;citation_author=Q. Zhu;citation_author=L. Kong;citation_title=Statistical inference of chromosomal homology based on gene colinearity and applications to Arabidopsis and rice.;citation_pages=447-447;citation_volume=7;citation_year=2006;citation_pmid=17038171;citation_doi=10.1186/1471-2105-7-447" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. Xie;citation_author=Q. Feng;citation_author=H. Yu;citation_author=X. Huang;citation_author=Q. Zhao;citation_title=Parent-independent genotyping for constructing an ultrahigh-density linkage map based on population sequencing.;citation_volume=107;citation_year=2010;citation_pmid=20498060;citation_doi=10.1073/pnas.1005931107" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=Z. Xu;citation_author=R. J. Kohel;citation_author=G. Song;citation_author=J. Cho;citation_author=J. Yu;citation_title=An integrated genetic and physical map of homoeologous chromosomes 12 and 26 in Upland cotton (G. hirsutum L.).;citation_pages=108-108;citation_volume=9;citation_year=2008;citation_pmid=18307816;citation_doi=10.1186/1471-2164-9-108" />
<meta name="citation_reference" content="citation_journal_title=J. Integr. Plant Biol.;citation_author=J. W. Yu;citation_author=S. X. Yu;citation_author=C. R. Lu;citation_author=W. Wang;citation_author=S. L. Fan;citation_title=High-density linkage map of cultivated allotetraploid cotton based on SSR, TRAP, SRAP and AFLP Markers.;citation_pages=716-724;citation_volume=49;citation_year=2007;citation_doi=10.1111/j.1744-7909.2007.00459.x" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=J. Yu;citation_author=R. J. Kohel;citation_author=C. W. Smith;citation_title=The construction of a tetraploid cotton genome wide comprehensive reference map.;citation_pages=230-240;citation_volume=95;citation_year=2010;citation_issue=4;citation_pmid=20171271;citation_doi=10.1016/j.ygeno.2010.02.001" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=Y. Yu;citation_author=D. Yuan;citation_author=S. Liang;citation_author=X. Li;citation_author=X. Wang;citation_title=Genome structure of cotton revealed by a genome-wide SSR genetic map constructed from a BC1 population between gossypium hirsutum and G. barbadense.;citation_pages=15-15;citation_volume=12;citation_year=2011;citation_pmid=21214949;citation_doi=10.1186/1471-2164-12-15" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=D. Zhang;citation_author=H. Guo;citation_author=C. Kim;citation_author=T. H. Lee;citation_author=J. Li;citation_title=CSGRqtl, a comparative quantitative trait locus database for Saccharinae grasses.;citation_volume=161;citation_year=2013;citation_pmid=23370713;citation_doi=10.1104/pp.112.206870" />
<meta name="citation_reference" content="citation_journal_abbrev=Int J Plant Genomics;citation_author=H. B. Zhang;citation_author=Y. Li;citation_author=B. Wang;citation_author=P. W. Chee;citation_title=Recent advances in cotton genomics.;citation_pages=742304-742304;citation_volume=2008;citation_year=2008;citation_pmid=18288253" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. Zhang;citation_author=W. Guo;citation_author=T. Zhang;citation_title=Molecular linkage map of allotetraploid cotton ( Gossypium hirsutum L. x Gossypium barbadense L.) with a haploid population.;citation_pages=1166-1174;citation_volume=105;citation_year=2002;citation_issue=8;citation_pmid=12582895;citation_doi=10.1007/s00122-002-1100-4" />
<meta name="citation_reference" content="citation_journal_title=Agric. Sci. China;citation_author=J. Zheng;citation_author=Z. S. Zhang;citation_author=L. Chen;citation_author=Q. Wan;citation_author=M. C. Hu;citation_title=IT-ISJ (Intron-targeted intron-exon splice conjunction) marker and its application in construction of upland cotton linkage map.;citation_pages=1172-1180;citation_volume=7;citation_year=2008;citation_doi=10.1016/S1671-2927(08)60161-3" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Whole-Genome DNA Marker Map for Cotton Based on the D-Genome Sequence of Gossypium raimondii L." />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10.cover-source.jpg" />
<meta name="twitter:description" content="We constructed a very-high-density, whole-genome marker map (WGMM) for cotton by using 18,597 DNA markers corresponding to 48,958 loci that were aligned to both a consensus genetic map and a reference genome sequence. The WGMM has a density of one locus per 15.6 kb, or an average of 1.3 loci per gene. The WGMM was anchored by the use of colinear markers to a detailed genetic map, providing recombinational information. Mapped markers occurred at relatively greater physical densities in distal chromosomal regions and lower physical densities in the central regions, with all 1 Mb bins having at least nine markers. Hotspots for quantitative trait loci and resistance gene analog clusters were aligned to the map and DNA markers identified for targeting of these regions of high practical importance. Based on the cotton D genome reference sequence, the locations of chromosome structural rearrangements plotted on the map facilitate its translation to other Gossypium genome types. The WGMM is a versatile genetic map for marker assisted breeding, fine mapping and cloning of genes and quantitative trait loci, developing new genetic markers and maps, genome-wide association mapping, and genome evolution studies." />
<meta name="og-title" property="og:title" content="A Whole-Genome DNA Marker Map for Cotton Based on the D-Genome Sequence of Gossypium raimondii L." />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/10/1759" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="We constructed a very-high-density, whole-genome marker map (WGMM) for cotton by using 18,597 DNA markers corresponding to 48,958 loci that were aligned to both a consensus genetic map and a reference genome sequence. The WGMM has a density of one locus per 15.6 kb, or an average of 1.3 loci per gene. The WGMM was anchored by the use of colinear markers to a detailed genetic map, providing recombinational information. Mapped markers occurred at relatively greater physical densities in distal chromosomal regions and lower physical densities in the central regions, with all 1 Mb bins having at least nine markers. Hotspots for quantitative trait loci and resistance gene analog clusters were aligned to the map and DNA markers identified for targeting of these regions of high practical importance. Based on the cotton D genome reference sequence, the locations of chromosome structural rearrangements plotted on the map facilitate its translation to other Gossypium genome types. The WGMM is a versatile genetic map for marker assisted breeding, fine mapping and cloning of genes and quantitative trait loci, developing new genetic markers and maps, genome-wide association mapping, and genome evolution studies." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10.cover-source.jpg" />
    <title>A Whole-Genome DNA Marker Map for Cotton Based on the D-Genome Sequence of Gossypium raimondii L. | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__MRoKQICmX-Ek5ZIyBlwXzUtbKmCmIVqNnsCLjO1o65o__abNV7fJaqFhsjvAynloQDGCCstgzs17fQnVdDEt9-NA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"XNeiAKSAIL_lBlJlQ_Vb5q85Igtg4E_5S3RVD3o32Pg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459560","apath":"\/ggg\/3\/10\/1759.atom","pisa":"ggg;3\/10\/1759","ac":{"\/ggg\/3\/10\/1759.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/10\/1759.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/10\/1759"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/10\/1759"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/10\/1759.supplemental":true,"\/content\/3\/10\/1759":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022sCiw1pekxK\\\\\\\/f\\\\\\\/jYq\\\\\\\/L\\\\\\\/KqWkJbcIyBx8yUjCUkIbQWieH39ZJTO2Q+kuI1uyD\\\\\\\/Xi2GYh+4B0nw4PxhV+R370+We3pdd4rocb7auyRF+r98z3YgbVNIHynSQkDV\\\\\\\/EkDQzQWuOTbJTr+O\\\\\\\/tHYdHmGeD6niFjtcaPvC0C0i8MhTXrC3j9TF6gqrdzMRtIQvSrheeKr8yj021I6gZvIYvqL6HOVcl\\\\\\\/PZ7WCKNBgyIGH0dvMKyFKw+Db6yyICQOPXjqT4SAgucHsra3kTtWZRijJpF8SXpGJbfxoV6Utb\\\\\\\/UFFv4EG5T9OzdBb7LuxNCWw8lzTtqcN11d4rxeoXGjJ7N\\\\\\\/muo40uJpmdCvIdQNFRlPQt7Tu9taE2hc1xsNaSqgcJJUo8CIO0oLf9nAH9kH2DqWN72oah4A3lTvP16RQ36SFryYD+1CIxu5zFYYdHzu\\\\\\\/k9G4V1kC\\\\\\\/8xbOP\\\\\\\/gdc10annkaDfq\\\\\\\/zBJN4U2s7VrlFcJfpBA6sRDSIwdMwt9mgB+vm7PagUcaTGamE3JZDShJqIIM9bmIsjUkh4ICvMUIrMpv+z+11anq\\\\\\\/rN1YBFr\\\\\\\/jhj5wBTq1bIZ8ong8g5+ntLqtaMBIAeizaKEgolv0o4T2wDu4aow15Vz9BCQ7z\\\\\\\/sl2j5aT41eaegujF4k3s99hZXJQ4hXOOzEj94zq4NoD8xWSkofYR2\\\\\\\/EUCfNM7qtfUjQD\\\\\\\/tkHI+vq1w1pjDU+KvmvnaR2FC03Rc+G22mA5cAmUctToVg\\\\\\\/TSQVz74qqiEE0K3+492uOI+7TN6ZjS1WugLPwtKjMIxytPkfwAqjyt5WYW7ifCPQaS6a5\\\\\\\/ZPh\\\\\\\/wv4spXu1Us6JrF\\\\\\\/UCxhaLvIakQzdgBKZ1WmN6taSwMAgxaU9Roz1hJAqk4xKvamT0ya88JLsS7XxYtSaiNE77sagZDVhaN6bFWyE0MPnXEur+OkyzC3Z\\\\\\\/MTOiPKryJwEJ0CozAjgTv742qjgdU7gJ2RQgdigEmYOrFd91fYdbayw\\\\\\\/15E5sm936a+1QLiH2wHPZw+pY7rZG+9lxLOPjU59MhT2rNVUoG7F\\\\\\\/O5B6VXSup4gRsCL3U21WOeFkhkmzMm1H1efdh674KZhg5z5HpRmOecUTWK+bNcZ9VjPBiP7xi66+exFz6RQahPj0bAQ6ciC6AoR4ysUQz1RPQrtnZWHgY8WRil9Ki3pNQICqWKQ+M0fc3q\\\\\\\/AcZzSQ1s8ML6tzvWMA81dgagf6PWn0lJ4Ag4ex65bThPwnzTwEqsYJMfNUoBY6Ot7aXoGlvzNlqvE\\\\\\\/SUo7He2NVq4\\\u0022,\\\u0022iv\\\u0022:\\\u00227EWD09ho7LNdSWai8w3QIg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022336780f0f17b4511b46d36b9f3d84ca7d3cd7be33ef41533866aaccc595c1d47\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qOTLH9uQ0BZhhi2TJuF3+gg\\\\\\\/Lr4\\\\\\\/gUYPM0AaI87Zya\\\\\\\/UBSPEkKt+NSLKRwS1D1K1mDo+PwEj5K92mF6vjkdv0bvad6Xl5Jyvm70106oTvXxB1p8OlW39G6iJlhypPIIE6LRwj5e5CrTZUqmBEUIKaOo5dduF9Rw5Lq4Ijt5Y9CF59hcFnMz54v5AQqhxQArE3ZX9MSyVgZ7fqu\\\\\\\/nK5G2Dh09DdHomJsZ05dSTtVS3C8plmdNEr+HiWJHIL5w7PWwEKvPcI4ma45Rn+EFMn8LzjRTOjc6ntSWYbMtRVxiz6QxDfvtMcJ3eYlLTywB9atkrmtWEt7Gc+0Hc65Kw0rxXFALa7NL6UuP12rxkURjTtI\\\\\\\/\\\\\\\/cTmOZ0UjSUGl7bRooTGL5EbpZK4s\\\\\\\/NpeKY+Wj+xGp+kzqsVFq6XfvET\\\\\\\/EITMOrxc2ek0RsXBBZIbsBVnD8I4a81GDWAO7g+M8F5VS+MZiDPNK\\\\\\\/7snvFqssUGaCZuqZ4XlQ6Q4qqWQwpImIweQ4Q4IX4eSSCacfdt9ZVNzV\\\\\\\/gLuCUAdWYjhmdB7lY4cnxcUmC7k2mAw78E9sldDZxzdkbWKyzZkK0NTEa8Q7Wtdg8G1xr3JK4r5yhN5VmkmbYzDmHh9N3B9VvGc3HXdkZ\\\\\\\/f\\\\\\\/w6Tved8gUfAyTrWmetyFCfSRLzc0NhamypitF6pxRP6X5tE22YALQhkbNFOVAfQM+R3d2RhJppLmhsdopTi3udeXzx85UB23q7auZWh\\\\\\\/pR68AtgpKh1SrU5rl0nNbR7tm2sjFNSc9CKUxLUGR1AXA7+epwVzDpc8vX6E2mgFVJkq93cWSTTH7lb5wmWLk1VEqVqWtCrvUDTsvWVAGbg\\\\\\\/lZYVQB7SzGB593q8Ob2DxBk6k\\\\\\\/GzQDhWzMokt5kPCNwDTnjK6zJ4fYvwvsmAwghcRrYQZIw6XyObLSYQIT0dGmAc4JetfaVKUeQ0uEm8DWgzJGlh0Dts2xYt1122qN9isjek4ahfyGI94KE2DvP2GQTzMeN\\\\\\\/P2ElqPvaXA47NpCrhGeguGuGJLjbx18F7WxkKfAJGgjS4dnQ77VJ2FmklcGHZqOW7OnCwmCx7JLqOmX7sAdhItqmwdwwns7eLQNlQD3f7Bz1vN9A9\\\\\\\/0a\\\\\\\/Je23xaRFHVGP8RvnqFlJ3jKbRe\\\\\\\/0ykFA8A7Le2J4W8CEE+a2tvAsUIfZfKEYOmolvuzHcIJ1Q7foxENnB9quwbRPSld0OoyXopkm2R14uK0kSkjSsjHP1SV94tF56BIUSKTFlGpmMSE8t0YM9\\\\\\\/sJQITaJlOpbs0FaLeQPjC3XOefF3zrTD1yxrMFWc6zfe6RbPWK5JEIOYMoQGTjzF6nxoVMIaTe9eZQfzHcWDwbv0PhHMG9fetZ4N2WSDrumRZ1YYdKpx7EL\\\\\\\/OMwCcDFiwJNDdDSkUlBGnazfoujmq0D\\\\\\\/Ws7HJCRlKs0BKJY4Gc4zBy1a27z1kF57LF16FExrbuiyNEfhq1sbshi7YwHqBV12nb74esETnMtD67i3cmQkHoGQdVHeCHW5wOLjZ8MtmHY1L5JWdLdWI6xcd9HRNxGHpTmwVIR1tB8O0sMmzEUguMjhJTEEz4DTreyjrPLu\\\\\\\/fn7\\\\\\\/0IZP7K6XsnH3cN5tY3TkTDpqhUOn2vj6+FlgGpGW7rXbFssqoHk0Luh9a73evZFig\\\\\\\/V7MmV\\\\\\\/VvmepOOcOeSCeLCRV0KZchYo3Xk4haikCQyQ\\\\\\\/PVly96NxyIUdE0irsMp3fO6sG0+hAsuBz7MSi8gX4W8jfP2aaL6gOi\\\\\\\/mVDFQisGowv6tjEi2UT3LMtgItFfl7t4Hr79kybvXajWXZ91eBbPpJJLYNY4\\\\\\\/JlppU0kHyeruTpWyh3b5qPBADSTQTNcdK1IWSCUbbn7\\\\\\\/dAfsU1LZyNISrVc1Lw3MwI1HXqYMociO1lwP31t2Q7LaBgNY+lGuuL\\\\\\\/myYQEFW4eVyawFioOl8n2SzOE\\\\\\\/4oVoojl0vwkmRM84S98WzM1YtohoTCz\\\\\\\/rPtT+ept5tjxh+1dhBUy4eSYBPjrm0o8Bf5n79n2\\\\\\\/21\\\\\\\/y2dLu8ryoTcClWG\\\\\\\/0XYVWW408\\\\\\\/m0wLwSTVYSPJhlaj1RjYYKrWULm4wFkclcLFtMKN54YgcUs+W1xd+2CKVswqoips2R3ePS8AefTgrTa\\\\\\\/6d7Bi3+EDNT8QiKMirYE942p9qmqn9xFZ0dus6i0ntXFV+ako2k39lxHofv2uFGpmTh1iWd2NbmUZf4fZEnyWgIb+Y2\\\\\\\/Z0qVRD9YyQ2YBbW60CvYtdzbURx+PbXnrD\\\\\\\/xu5vA\\\\\\\/ZW08rvOPk9yThLV+Rq9DtFRcbO5lmQNcvD5ADQgKjqnnJ6k15oeI87ra6Gm17VRRRWsyf7ChACqDZWBade6J+Rs9VlIFESq7haENZZjDe+u5K+r7Ezg1zJtOWaQ6XfuUb4\\\\\\\/LZHF6svdxx1Q++Rk6wbRFX2WQ4t8OYQ9WVeaeFf7pk3vO+24XdTbLVc1rZIv7KpFDwBgj47P5Z5SCM+PO7ra3+kNEtWQFYzCa6R5dCHEYgNt4U13UPD5iBdWxHxarqqg8e0yX\\\\\\\/4ESe75APweULK0tkAXDRpjzxztGP8+ESNGfHC9S++w8XPMGIzKu3pEpII1Ahmw0HR65b4okVXXMiyEaL+iSqtNxc4q7VQnn2mca2+mF8jQZfv0M8+ncydT+MU3PA4jotw2eofYOsJX7HcuY8CwQZS8BWUSLZo3KL6RTF6826WCHLkhmIngsekeZkOdJqxm5HW8CxJs\\\\\\\/Q7IwX7MXRTlkiNVHVDlcr3HxCBEkd3CM7CYtXzzccAeU\\\\\\\/glQR9kB1Pn7a2hmfIhCB058CY9\\\\\\\/SLKfYgpPVXf\\\\\\\/Tm1LoL01ptgAwnLaEYNoFjFI0ewOFMYvtEn1E4vP8Xl0gonQEzmHZ5Nq7XzBuT6rVF6gNVmcjEqFKrKUfi+KeYdAcXFLkPdiF6B73Ifc3zVhcLu+NMbq1b5UcPWOd\\\\\\\/8HjZEfZhg37Ji7eDsf3fVrvnu+in4sY0cQN+2IDnq6o+47SE\\\\\\\/kVTgr5m2ScnkktL4DqcPw6hz\\\\\\\/an8plSuiUcRCmTXDbCjbZc2IgzoJYOs814OFzSmHaY7uHu\\\\\\\/VLueoemmLP5YCiQv9ZjfbWd\\\\\\\/8LNLC+Rp8IaUJgf5e\\\\\\\/uoMZ+VebS657n+ifGROEwU06yB6ci\\\\\\\/njaeEK40dcM\\\\\\\/pWfQ94qiGwfgbwxVfxDNVikqlkJPf46OmC1\\\\\\\/vbwhk6AXKZMnFmzBY6r3a4hGf96UMSAMBYHbNYhCNFqBzVHY4NiOVEaN+QD\\\\\\\/KMgrdiHj8qo8abx\\\\\\\/F8ASKIHJRT1RKCelQ+Fs+L1lXvFQC5p4YKJHMIwDGjhZNuKaBlogHMvQtRmv6DImKCEhnAOqi0zBpmrLpaNlGOIZmrzgE4up1\\\\\\\/ZdKs9l6Cgskk\\\\\\\/JjA4w1F3twt8420H+cdJolm8JZ1f5rVB8TGdtzf90xRtF5NjZA1LoBEA1P+IhuXJrxIHvBJ3ajRzTSuH2t19otDeAJf73mT2cAf1V15zDM5TYZEviqlacb2PQi9ZrDT21gX3R\\\\\\\/QGa9w7Y2bN8E28Pe9gJ6bgzcz3tI5rjFhaNCS7QNPcL8GpKvC7LP8L\\\\\\\/VEAXSoziGgr+5OoD5ov8oBTio+hvOk12TGZjnrCwa53KxacaxEtlmX3d3n4EPe4ijlGZlh2NsvcRTjJevDZs4dNAAIOY\\\\\\\/Wf5uGQ8QSDbhkFEzFGuGi+564UEleOxfmtzjQM4\\\\\\\/jVEsZizcFFtnIYhV43ZGcjeiWU9KYlP\\\\\\\/4hb+hDVr2bfSumSU78xbrk0mvVXQs56HkLyyrOaw+W5IJbRSMMeX8fB\\\\\\\/9piQjsAClhbvwjuRW50DUBempbFtxqoaG62HZV1IHlt4hj9zIoCNz9o85fWfcU1W4BzZ89Slp2xnaWFb6Fn4e4heDGiYyimDhvOr6VMBp9Qy4tYhCFZjnBAaUgXP15uX0FcHkmWZKh6g\\\\\\\/cqzf2kW\\\\\\\/YlpAuq5pH5pcKIOROnF\\\\\\\/9Ujxr9MIQsQoKUBla\\\\\\\/NB8jRoqOmD6jOA8gdbxeGCyJKPLyRHsmUt\\\\\\\/quv+QZvaIBaxNImBVtVhlU86ZgLh5En3v5p9vXtl\\\\\\\/omCOYW0jPjAFBvsFhSGAtD+pRSThyAJTYcCt80KPpsxbSYKK7tUmIL8T7TNkRAD2Ru1jVFm9LeIfBMQKrzsL7g5u5a8W5T1RV4zKe69AuwAY6SDuw2sIFcqYMQASI4cu0McErGRWrxQU0l6kRp9+r69W3bbT7jEkmk7byCiVGvQAF1F2GzYTx3oARj39\\\\\\\/1Hsgl+XHNrGAIfSy8zFpoINEk+hz2sAY7xFihNVEGZ8SUFPDLwzSJvhmT0dnRqL0JOSPe3bPAKYp5+iSKVeb7GWOyb4yQf0pFY2frOgVEXxjy0WUQ1Zy48mGA3eOzs8JuJWVeERKY7iDBvLjy5ehcZLg30Xs+JyVS8j7hdJ\\\\\\\/EIII6e3B2VCW3KVc\\\\\\\/NfX6insNclkYIAtrrIsTrApeOdip0gR2\\\\\\\/8w5S9lZEt9IX1Ovdf0hF8mBK93X3dxTjM49hxP9MzNJYZZK3o\\\\\\\/rJWSC6uiMmipNiso910wuTdqk1j21v0PoUUsz1Pl+\\\\\\\/QXA5FQAoWRpe8OiAjxsmR2\\\\\\\/hBGBpkoI54rDfrutfWOBRwQ3qvSXMETJ95k9Ua0GvdrkQeDydxAh9RNd2gBigx9hNWJie37AJNRCm8yB8\\\\\\\/+5V5KHsRkpNw6mxVkzZaYzgknVIXJ8kiaSKIUpW0o2iTKkVGU4kjCWcDGDKy7hdRBImTQfl4Zm3vykGmPNl+p394\\\\\\\/X6165M11nlkWJ88vSHsPGarc9jGSBg45uRbSgf\\\\\\\/qnGmqTACMdif2SA5Bv5ea4czC5Rq1kGtyBaR7Ea1jDRkFKe9gFpoa6xVVJO+fh6MNyJYj9U8Ol5\\\\\\\/sN80l2fYt67Dv94KpeIb3MkR6RSguAA0Gk81\\\\\\\/tIbAs8a1yfX0yifxtqoLtxCBndWxbioF\\\\\\\/UFgWDV\\\\\\\/dX+PTMqHJb3aMEuU2zEUSci3PfoGy1MclwfU9hYJL3fhFjDleI9MfBQ4y+ZR+zwChdCiwJsrJKlk+z6PrsyPgr8W5yjcg6ocxS2iq9XoZYebuqVR5T7cCD9X\\\\\\\/wZJBokwutV8Jd2vap9EKuTKfkbkOR51T3\\\\\\\/LKQWlxS\\\\\\\/VFrwE4gutRck3xxpXmduO45rLyYqXpo9xBN2+OYAOON85gAJ2oieSn3ZqoaIVfLuOvC+n4y7zjkkJuBY9IblKwko\\\\\\\/YF0Ldvmt0gYkpwm1lyfIIwqmsni3eZUMFbjntVcm7cgf\\\\\\\/fC9LOODQu8RxfQVYlF8D1FiLspkQxmVCF8CWoCwP96IkPBkWLJ2e4u8ng1Xz\\\\\\\/zphulS4k0iKhTF8I9jBlAPZPblm9lwuXnmk3e5eVIF0Cua9pcYzTcWekGmbDAIc8e1vjDz7l0QfQqHDhNnEf9eQylRmID1hMIHAgUULxBUSHcJIuroyClcZdPJWBgylG2bMXwnm6d\\\\\\\/npf4erS+ieZc7Y7Z\\\\\\\/SyRbKtu0NOHvzVu6+1dQXASWq3GxS5t8R0B0t\\\\\\\/slZaa1RuFJEs69H1A8W+m3AjfxSJcMj0u8xN27QgpfikeJgnv28LFAqRKBS2xzKpb+i9JGBgV9C5NAnH78KirRkTlFnhIT7wkdH+Cv4zvUTTuM3iQlLSeUIxBbs9+MrO8iFKVDHV+PfSzV\\\\\\\/EU7inICarPs05\\\\\\\/aAMgX67T68oIFfpF3kSEOzyvmOEQDHzJ5\\\\\\\/hmc0tY1X9n60a0aU+sHtFZgBeOs6zHq\\\\\\\/iOtbSYs8bLygPEkQIFyBaIeRc+ftSwSHBeY01i6fdJQoFyb9uRuwzRA\\\\\\\/4o17nxY1VCT55Y2BTXix6VxcOpTm2WDhGYDXTPZS9\\\\\\\/XYDfd3+OLiOT8RBLrcGJ\\\\\\\/F2klqb6jwvGA8ZMZWcDV6ibknhR+byunmc3EqO9m7JOVL9HI3n0Ga1qlnS2W5EfPpVYWGx1T86JZ07TuLBuPzEfGVlfD69MrpstnmzwuI82QtLkqqk2njcMMrPJra5eHSk2oQkC4eNJZmzxctS1r4HEVSBXQ0+Al0nUeTtb9VekyHRglciuPkF4rC6XZWSvtQqg23O3ZCCF1SRZtQkVL+wumbuHKRTxyyx3wcZmMCycOsQtDI99CdBHn5Kv\\\\\\\/SocSevBKxOz26FVGHmVGpAWLplksWiVtoURUaqkNLTl37blJLjboV\\\\\\\/bZmCx6dbZfyY4ZoSLVAaGoY8gLh5ojIPbovco+s9qVrrtFgGKAGA7rDpLGJ7irTkjTrzowMEvkBjn0FNfVDqlQooEdmyJiC89JYFtzIjDEnnz5ix5aBbDEbcHO9Rn\\\\\\\/56BFiI904caEWgc0WbYuIpigtQNeKF83F0SJMANAzto9Ut9KiyOHzm6SwiowiuhsGbH6c2Ny\\\\\\\/F5f0GGbL981mnS8Rl\\\\\\\/mDvak3PqrWjVbGtF9eGvYFJx\\\\\\\/l4qZFx1uoFzxMxhyh7m2Ia\\\\\\\/zqQlD3sZ2KM4655TaLtfDbfhNorZjD0cMvRZxXtI5nZYlIloGRxW31kdjFhvGoF1Ev70j9KOR+9xuvrZUjVMyLWdXhuYJ7wqiVhEbnvSjDzizcvdeCNCwao2RgbffLM8BHMG1tsqKNpgpI04bPc59yT0p1JUnHqNcKN+HmE7aHLJPACq7cXv+UY2HwFb9crrbItCJm3L65CIo8ZkAugmI4Xm670tOnA5fcj4O2fba39cQQraj0IIJck3QkDBu6kaqk5DNckWXrKbv9dB8k5mDzde8cP7qLZijIWblmj0IzChJS3q9ZiW86Zx5nfQuYr+cBTfEiWQ2ogsapFzxUn7dzHTSm1sjZB1vXjEjXipaQwXw1AE0XknMceWqCLSNDBXZX5KHGNVkgwXnIRNlJY6p2Pmk7KsIdoo3iYfOXKAqdW0SklUV4US7OIAjrVJfkEy+obUMUJ9QybSkN1QtcNH1MAz0JWHj9VvKfdbuUBDajGe0kgsBxjmkkL34Xw2+x6WikbXnPm27UjVvw79NFrvTUQoa8KGAubLuIulu1N2\\\\\\\/Z9qHQfPQOLqYxLV01vCrU\\\\\\\/nf++I6pXbeN5AZz0tzyFCfx4DiDvSoYDu5XFDjRY2JymMQMfQZrs\\\\\\\/Q\\\\\\\/+15demWeOBh9CwsJmtcEJB+AehQar3juyFxe1PLqvrG4V6xDlMffdVpwFh8f098FesVP8GZN0uCeTajm8chGMceByE\\\\\\\/PRZWu2qzbEJ49Lxjq762fKgNX5Ii6XIknqCr51TSTU6jwRNGHyu\\\\\\\/8zQ1LYDVnKtJp4Z4gBfDuGTTGajfbnoknnDKU1aiuiPT1SmmNWfTq6Bj3ndAe01Lpqwnz0Bbz\\\\\\\/ADxns1UNQ7DyJSNN7t6FzPGeoZ1gb\\\\\\\/kjahnTa7\\\\\\\/Je3FSFosciXZuCsjwmRl039f7aZrdeDhTMcnasDglkJyO85pK5gQJ49XJmJYHWU4JhO0+zpHHVxY9sRkENQosI1a1kxnZi36\\\\\\\/TlLyfmMKcooVmJ1Z75RSRF1nt9wVqI8tKYdil25qB3LHHitW6TxVYIlwURoH7rm5\\\\\\\/evR8lFuZs2JPF+YkJn9p\\\\\\\/aVfmLbWTXt+AyzRc1kprD0FxZBYDTzebPW4Wl9Mar8+yqpeQ7dTH4g9wS0sagwV2vyUQIVt2OSKYFC9143wY9Z7nUzHYLM3ZW0OpW3JYBDeKixL9x5o40i9ib\\\\\\\/fXZpz0R6QS8genlLpEg9\\\\\\\/mJC7fNr+Go4rV4iuiPqy7OOUKV4FWrNJCVi1EaKrV6pkpQUS+GllIIrBgX2Tg\\\\\\\/sfKQ9Pbb634WBPYFuXfM9s9uGEhEWtQvu5yV4Jo1K6RNbCHsDVLXX31mF5Z0D511We7oR8Xht0p53r3lBRXrcQ3tJctO70zk5TVS15pqf5LIfYrAIfchmeJlN8c\\\\\\\/jys7ZQHqJV7tBiy\\\\\\\/nTrPu0OH83NQ9fuj4gg\\\\\\\/HINFgWJaJd\\\\\\\/ovLA0nZ6upCLCIf7X+WjsI47ipEWfviRZ\\\\\\\/eOctoGmK1AjCuA3vPbBaxWr4IMw3AfzYxCk8mn3ugUQJsbifxmKKjt5inPCoohpfCjs7sF9ucoDqzL8jfctH1f6cZnx4RYfy6\\\\\\\/zISzKIT0MMh6u7Qym\\\\\\\/9gjhk9wWq\\\\\\\/6Ai1UwadmpXW\\\\\\\/BLDxTcS5YWun+oNaLq9XeplAXImeaE5VFYF+R0Y+aqxP+P74fNwkXPiwN9hqzJbppcCfX46bsi8IDGS0\\\\\\\/RjTHfODwcJvc6bMeQAekLgSHcOPHzHpouPDxjToiOVaGSoaupM2a3FnXMINBGABO3sP8towWt\\\\\\\/83pFqJ4cKVE+s3gwIV\\\\\\\/Y6lIfH8rPZ7qscj7tRsxJE9Ek7I76Qz7zYby7fRyYpVLdhBF0sNvJPr\\\\\\\/NzeAj5YcbPxfuQJvrdc4KyO5LE8zmp0i6EWN91rZOAY4il3vr02V3YTrZAxh8FbJTjoyRMy\\\\\\\/Yc+PepddnEz\\\\\\\/Jp+ZoDsnfUDsVUkU0NFUMjqHJCnMTBZ7hRfGxV7PLxpqnz5V+UfJv5kclJyCEHom+th0+EZELeHtbipfmRt67+Hh+4cNL6z4y2GqiX4ZWkzeWUalIXAykgz2fMiTxpSB5AD6EebeC5UpRyAQs30S3o33VEg85HeX\\\\\\\/+XSLXzPuJOKYP10aVJwch8iA+YvUoNsPU+JaNQGK55jSKqhpW3lHqXNf7LrTzzz8cp4mgJz+jVfJONYL5nOxybtJY+MEAjDVMLZveON6CvB3+ADQ7JPWJD4usrFbRCWf8+LTKBjSiWt2Yb\\\\\\\/egg+GGVpKLHFRn76xG7+SV3PSvg0KoTngC8ZQOOlzzW9O\\\\\\\/XgeGKwzGW+nCcaXDAnX9+1+L\\\\\\\/Y8jfQS7Vvhidb4MG69TrSNk+a0\\\\\\\/cJavzRaj5uhLiEUow\\\\\\\/WsJyjVIw9SqveOKXKYevo7W2jCtK9PSDHDFb0FvCAkPb2yduBOrvluHdRUHAy\\\\\\\/3aI0sKOvaFAZtmwyhM9fhzXpLmTxiy5AhxnHnoRYlk9VxSar1iIEa9Roeyy43MkyHi6yzVElWd8w9Vh6nGCnK4vzY9nol8vosa1H2DkeuuQlPi+GhxcLCV3k2WL3nZyCxii\\\\\\\/8wepvHYEXEiOpIBbW7R1GdyDnl7duRc2j132cJHfdjSX\\\\\\\/qN7LxKWKysZp7zTXnW0EfP31S5ep1GVoD3sakV0Ghiy21HcJov\\\\\\\/bU+\\\\\\\/YkkrSUpyAtkXaqmo4Kk+lYeTk6RtGMB+P1a668iTuZTCuajckzztRsJ3JMhBT8daVUWx2rNj9wd3G32r\\\\\\\/o+Qg2tSifoII8AzUS7uvpZ5+NeZQLaIXNrI\\\\\\\/DyiyYZJs23S\\\\\\\/pql+P0kGfE0AX\\\\\\\/oZlMcpNhbnnSCXoWmFqJrBHbrbn\\\\\\\/U8yowrGquIVLIu\\\\\\\/P6spoYDeP1RknXY6FukgKUbhCV56JYVX+3OmEQwRSqvPz7yyEJwPTw+ywhGi7AbL5CgYHr+\\\\\\\/Dhpj2aGUk+nWuq7HdEl9I3hhTxQzRkATeoYqXk+764z1OYP86lxGXX5oq5fjL8H8YWIqSQ59HTc3MkAfX2R\\\\\\\/RiukE0YH6J7v0Ete\\\\\\\/QlAGHc8ed6zczNlqGKOcyyuUlCNq0YHamhoyo1g3iBICGIC31J1R3\\\\\\\/2AMJ1IeT\\\\\\\/6Ot0XRYCVOdCInVvbCPPzXzRUTLK25gipmW4ke4SuwnTX78G2K85+CrK75JIP4h6yVf6Ai5XIFhn5FwDPvgnlOzsphLHh\\\\\\\/BIVuh7Ohtd7PwYvmlgBSjjMDwhVfEFvtGArBrgTnNQ5g6Lg774BD9cKgpGHyVrwj0xV2ibLaFR9t1IT54KmJF03EoaGuVZv4RUC3UsfonZaSyNnwYQJlhorhLk1W6M919PhQYvw7W0xGAQn4n2EDg\\\\\\\/Rdk9iib1DYWrM79FzG1lkm+0E7jehkX2dy4jg4jnARmUCWDAITFm3UlDMy5XhahbB3nibeNpLGcaimB\\\\\\\/FGWYRuSU7YG1byhiteTvP8BAh2gYTvVvV6f3d1Bg\\\\\\\/mco0mrwsv66M\\\\\\\/zDUV\\\\\\\/8nZeDc55cMYM3wOdaUPN68BxessMah4WWvwjmjTg2ZEEJ8Hgn7QZUi7MkwnfirFAFXjjC0H54v5g8URoK0SSYrYU8aHG9glyzAWjKMEvvgx8MKkJQ2PHda6U5TT5U9gIO+NHRcH\\\\\\\/YPV39F7mTR1kbdF1BhBhiUIUnyCyX2Zzl9JUkAisteoAJMzcaELDtEEo9yDeskHFmnDjI3vKeYObmzg2fFm1lWeUXVG\\\\\\\/V57rSJhw3FiiyApRoj0zXDEJLSPpDXBniZdETkVXSPlRPErqKdLVdoRNy\\\\\\\/I7FB4i8FnwGjm3iibpDtWBOZW\\\\\\\/4TdI08BCFZ9b6jeKwZyYd43HbqeFNahv0lTsxY+MiuhdUdmPgO3o15K4cxvk4LJ61Zk+h1wMf5sRAqtik52wHY61sXNdwL+e0NykMiQAK0vcxMGR9A6kD\\\\\\\/E3QKjrOGzLVR1trF9D+HwyTfbe43+\\\\\\\/TPSQyXdeELWwLH6VAEAzfc8eecE9JVNhOrWnb8rS\\\\\\\/2Gf17O7tR8jwmPfl5YEf9xOPoWbGs80Cr7t0AAz0fdCtYyAJFq170YdNpopkYGiSRu\\\\\\\/PQYrCh+fU1jDQAU+2CLcHXY3P4WHk3QBjpaCica308s8dyQ9zVX+bVe4QV+Ssl4lYItG2copf+Fe7XkzHG3cVatkNj20QUkBvsjEwqK64HiRuhWlU+D2PYjNyez1h+GPEKZtdI3ZTPoSNHFqVgiUFdp4B8hP+o6Q8pP51M9uj2a3xEBdpVMoRmShEUHBz9NBvtHmNN304JZqaAPdVBwIg4Bl9rT\\\\\\\/utL7WFUYqBs8vGrY3xD4kkxCI6hoZdyo6XvaUWsD1MfZeUOIJXMsL4ErfQc4K9VWpxzhtt8fZk4YeZASzGwdu9pu4Rw9l93yjp7SeM\\\\\\\/Z7TPGJ8EpLasOfgt7HhI31U6nHhHijC6pYqRmcOoe9sgjtr37aPmm\\\\\\\/HHKDj40Q9CRqDflhl3TuMSgBvmNUvcB28vx6VG62Hs\\\\\\\/29VyDXgNbaWaYj2HFspVmL0WfjBfxcaYIlTApsATTIPr3gOEQWMoc9M4z\\\\\\\/oh6S+AaMtn7Z5DAqMdh3qr2T1eO7nzkl41RDb4C7z2az4PcIkPYHQlnfF1LFTXX18ZrCECXC4K2tTwuQEu81nEugryiXv29xKpAbbEtQZnOvSJ8H29R+IHTRxuuQzY5KtwHXImQY0dg6nDhsuZyKuYkoudfLmOq9zvNidDIGoclBbylMYkuFI6LV1+KS\\\\\\\/Id\\\\\\\/VOUWqLXH5mNRb0Pfx9uUcNHu4fVxpaBndWh759w6xOXj7lOB9ZoaYwWFSPKBrUKqzkbZtGH8EeYnh6z0+9H83BP3tQwL5\\\\\\\/DrW5iyfRudl+4s2dfIRCbBHquQuQDweSIipyiLvsqfgDlA0D24MK6jFizCyd6OQKfOfacGQta2WWdLZk\\\\\\\/rZeIVpqFPpCtPvatqj2KfSSm7p3faF5ZA143sDiECzO4Yo1y0dS6c0V6OZe+oK7s\\\\\\\/AjR2D6Jz+OnU9z9nfgHIDvc\\\\\\\/sfovG3kOGDkHbJstu6esrGTdE3CGhePHdMwDD43\\\\\\\/UvSspR9Yuagn+EmWSQyWBfqkseBP1OXMjm+5YL5MIY1Q0EBeszZiPk5w+Dir9nCfPsMXYRnQ3uaz4neVkO30Z0xLgSpRqcfqyvVLy\\\\\\\/jVTIDX1aj3A0Y2ARrlXZL8\\\\\\\/nHGPr6vIySU1Jx\\\\\\\/fBKP2ulh3aA0DuNd6yMed1m8Ec1oZvQCz5m6rsHBF5Sk+elAPZYCh4\\\\\\\/OBtk94brmlUGZnYtN2z1fafAj7LLJo3hKXhQL\\\\\\\/s8QAf\\\\\\\/nIAAJooOda5O5tEHAqDb4UAI\\\\\\\/yBJhEZTR\\\\\\\/bXwQNaBJTRdBFCPY7UBxS93CiPx2Vu3PkRFjr3s3QZLjFIc\\\\\\\/lSWlnGphQLTIAjqXPAmuV4AnZw\\\\\\\/WWkoOGwKw\\\\\\\/\\\\\\\/9mNVwWNCsXUVVOWwylDMh8q9S0qPPAtpgCbXZ9iEgBOuDEujnTYQpOLYVNraZ+b7+qDZQzU4umijFGgfAcpvER7+FH+DQDvnMtz6fCJZA7lwvtWFe2RVrIfRZus597jnddfFP6nlkwxQCD8JmeZuIDdPYNAKV1FoLb3VFIb+OxV3O9KdZQSf8X\\\\\\\/kSZT+VgZl4FhiipKav6t+\\\\\\\/5xpsGl8Y5aYMMIjUbLPz5dfklLfELS8m\\\\\\\/EalFQwoR9mODScbsE12SOGOTcUr\\\\\\\/LIg4Uzoldd\\\\\\\/P18nZAdDn0S7xT\\\\\\\/7jilVabaWSHFQQI+GlY3Cb2ocgePZ57GtwrMEhQRZI8gsgIsocS\\\\\\\/D71sJ5pPaCx1\\\\\\\/ym30NyjzNXY7t9UgJCvuY8ZOIlGsOao1snDzaj8I0Pozp4BXK3bLxLe2wfJnsexKukjT65rU6Zluq38bqobr\\\\\\\/RZc9OIp2\\\\\\\/qGSWX5N+Pq4pY\\\\\\\/qTAAghEq7JnEAhDIy5oIObg3rTET\\\\\\\/8kW\\\\\\\/DlDmsIRXvF3BQa7ZDi9K4wWQOQ8wjz58zVsgkwM3W8S+JB0rm\\\\\\\/TT86bbUoxkujZznl81B4tcSzEl9TgiXLk3hF3qaSD6iFet5Xu6oZnMMNgldhpkNDSysUE2IjlIgBbJOQmmVnHguN6nsr\\\\\\\/IQevHmx0+CkH9b8LzZaX7kylyyE9baMBhEFXxUHO5XJXot24WM\\\\\\\/47AL3GahrTRp4HEOPNgAWG4nvoLSvaRGO+Grs3\\\\\\\/MWkPA+dT2o7ZldemFa7Mfu0SMqLrDkRkyXuVz\\\\\\\/xnzmGT5CuMfo1pgNwGl7mJdGx\\\\\\\/lIQIUo1blJNKiPjzff3cHCl7j7p25UfWRNtIcrBmDX9CpQ8DTqsbxDDFBLt1XsQ9XVbItDmXPUq2N4FmsRUKQMAKN4ISraxpNPKLLK34EMLDpK4TP5toxFnUTDxXs+6imbl5ihsMwJkhhi5M4Rgh4SWYW0pFUYVIL2XKaaqKsTiu38eX8thjtNGPiSm5dpGYwwyfAr1fblR2G\\\\\\\/SW4yXCb7A23\\\\\\\/fDXYgdo3V0yDbFPRFvnt+qF1EZ2dLDzevRjyHQap2G0MaHoJpyON\\\\\\\/lugO6Z6LA7PYom8zVNOiMYBaIziXwIlmQ\\\\\\\/MPblOJ4RXlasCw7hZ3b2pKHOFgy8qE9Rl2uGHV1PHbVD8zD0GxSGJIcB5zyoIZYmijpd1tTXCulLzdQUTGuSlHk8mos5\\\\\\\/r98+tdLI\\\\\\\/oRAvrLVcqzwd\\\\\\\/xnirpkDEOs8U3NcuhBuQwGlNNzfcjRBgaH\\\\\\\/m3+PNCIDtl8Anr7a6flNehiYfqH7WVspq0xsaFr30KfA0DJ8EEZSeQnz1+wpRICdQ9uqIyj9P93L5LETkwffaNGwmV7zn38sWZHeq3ZtbVBxGDM0V\\\\\\\/reZl0kmK9dSp3yKWcd7bJQvZspqkog8gk2pBCd0Z20QQxAHfj4Kk9FbcTLkP6QQN1QynXE5I2GHSi9H+w5+ADGzTbvql1O1\\\\\\\/uNJVbpb9nsa0MXYY7\\\\\\\/zAvnAArtUOXH+9n\\\\\\\/BBlGzPe2Ke0gWq4E60vweIytmnnjUTI38O6WfznQiLHXH4YjVLunhvmAeYA0XbtPtGIuDRRd1eGGClm1+QmBLZI5HnO+JxnOYoQa+HnaiqdWxREVczNdJFQK1qyY20PJTIAuwIAu+lw0N9EL3CVald+U6H37JK8nrvktHPyJ7PauzHFxPO23USg3agul71mwB6JatNm16FiPWG1BHcD+FCmob5DCGkBeo53uXkUhl5KbpxsaWUvGqkkf0nhzuPsD61I1+k+1NJArolaJKJKBx0xQTqB2iinAoz8j56mPpXtqRmyg20zy\\\\\\\/TatgoE62WTRDkOFW3hJ56DZ4hhY\\\\\\\/VD8fYItXSO9iqeqQvO5Sle7KcRB4xKCDYNSlkLNdMJUqhcLo4xaNhqk7FfgB3oTFQTYZ\\\\\\\/idP9FLHOq5yzBMT9rUJUnq4ZKZhJC1pv6iLLmAyRSTh2Qkx01i\\\\\\\/ltIW0oCu8TR6+SCwq8SrhEAUocdFw\\\\\\\/nCL9pDGYuShTpNzHNCvPoC29Dss5fy8LD6s0Fz0Bl91NwSCn\\\\\\\/IOSStv0zIa+A+QBEWNXr7SqIgPvQ+bGi8y2mruzcxXXbQxCXKEddIvrYyve7XHghenD3tJOGACJh42VLHCZntX8EMhm9PlFtpAfOkjC1o0R0Q7X51s1P5Abrcc9kcYrFTd+aGEFlg8RP7qh2TZQ9+7CCyaHoUChKTC8XKcpIzVWjCqTEUDTJ7PU\\\\\\\/qyBr5gmTE66cKGJV\\\\\\\/iUcxulANaDI2JiqDx1yG94ptDminFo6YU\\\\\\\/6+z9CNIXu9UVAyFZdSwc19yUK58hpUlNRNtSBWB7l+gDZmL24Cqg9Uq5iwkVCuR6uGxK+tMUym0bCiY\\\\\\\/bb711hZ3lgajP\\\\\\\/1+vf\\\\\\\/l4GuAlok\\\\\\\/KhoDWsW6yS2hiXEY5PR0KROh06Am4tbwf\\\\\\\/e\\\\\\\/PJwpo20eN8B+bMpi9MGKaDZ5w6iLTDi8yLeo5DyW5E5TdXR1bEDlDJwu9POFYPvWMmebVW7r4P8DF\\\\\\\/7woT226S6zc1XW8qkKTb\\\\\\\/ZemfaNpuaP+bx\\\\\\\/P5Y\\\\\\\/UEF43NLlcFW90ZEOnmUP2a7SqZ7GzPRFAXKffxifKG9Knck0006da5x3Jpuvfqw9O1ezxKPKO0E9badI\\\\\\\/sFVzdhJno1CIcaR6d7Zs6KDcuL6J3zgKG0BDk6iddp2JncbkvpDQjAG14oVbxRWKZPf00aGLz3rni5LjedLrRzjaQ+hEe5KA8AXFazMz+5ELWCunpKR0Wz+K\\\\\\\/j\\\\\\\/8klTGBUtHoQtXl0etyHinUEE9ErKBZmheAWayoqvoYRXalilYDVabOUZIgiKy\\\\\\\/BmcBIgd9RgVVrGygHdiV64ZIi5EXNH3br7h8mfVjCh+KKhR3IjO0C3FGJZGdEVBae2HiAICJapqxPT7IKtBLeo4VrE+rD1wJcMlJ0Vf8mgANnTIS4dVUwHv\\\\\\\/z\\\\\\\/yxE7W4y+Ds6PeU7XdCM+Xb7vkIzP3CDedHv\\\\\\\/UMcQExaEmxFN19mRcChBWjJpDmE0XSQGQNL8WoyGdN5GfEm\\\\\\\/u2xoLjC\\\\\\\/MzGmgb+Z+TivcNkJrfiWg7eMQTbz8hDvCepyId4lXK7HhOOvLqIKCZ0\\\\\\\/08k0+jbcmmfPftQ+DcYBEIMoh51FVlYjVI4JLKj+Yw\\\\\\\/VcMd3RujohCMjDDO\\\\\\\/jYnF4ZMMWQUN8mqwf3jaLcR2EvFDxXqamAFlcPRiOMDJ8pSAYTPOoAQcXiMPXLLRLgoPvQVgwFyicf8B\\\\\\\/4QL0LjtJjSJtZ4oTDaq6\\\\\\\/sHAOznSfcj0RYdCS\\\\\\\/DG9RNaymXPh6APnsMd84Vf36+Jm4J3VpaCJz++Ztql92SLYDyDLdpZeCpdX3yO2f4bE7+M9YGY+\\\\\\\/dyehltVp+i4xl9RZWI0QWeRIoWPdgsh77hw3ZCVF4W+37yMyNUVrCzY98DXwHWlJXmgfpOOReCQ2LsY1ji3rPlpt6fpRPNfXG3bTi05XLog34kkK8y78v57SrNS6XJzkVZ2D8SyfkmciV5PjAyts7wUo1vtW74PqiiNYxw5h0AQiPwYm2UbsMtG1hUGC2o4oCFoQ10JlSy9P\\\\\\\/jI3hStGRckEaBNzIHda1oae9gOHTWf\\\\\\\/ZwoMqHhGxtIjFivQd5alISvJkZBvwohVh5nWtjtvpLMQmhU0DgKzyY\\\\\\\/7BzblWe+Ea8J5+2cEox\\\\\\\/v4vcvZvd5nOV+HiUfkX6p6n6ESFfD5B8MKk21ybUNqOYTnf2LlyTUUhQ1qngf8eAVs2CSyF5kR513VaTCJIsv\\\\\\\/1ILBRsqft0iJmxgZ02CI3a6x7mXr0GmySDaGn3RQlLwFjgnlPNQwpTY2e7E\\\\\\\/M5IfU+47gOP3PXxfxXafAZgO+89qhyZQMg7a1+k1FiKN0fa9NaGYfZzeL7dmVJ8hIu+6uHNNfs8lPZoY0MBh8uGUJBJi9nlDzvDjAstBfktBVE5D4EQqyvf\\\\\\\/FapaaomMviraFqKf+q8vAtj72MGgGOfCIEbsCZYR0HleS2n5VMZACpYIjZLTYHA3\\\\\\\/fEqJOd4Ax3i6Selsbozco+xLYtDvDo0JWrcSLrYpHS8UCD8O84RdB1hyZWf5\\\\\\\/uBuPs\\\\\\\/5jw0UGoa56P4rNewbF8qAIKgmJ5B9c0FTCqAtu6QWEBe3zSkMPUjFcZkS+sOSt+Q1qgJ5n6oOX6oOBTsPLQClV+sMTv7iUA4AS9q7QcFeILHJwr73aRcBMJpSJbPnflg8oq5B3ZP2PxH3Z8kIEoTzHydQxnSRLFaMUeHd80Jd9CaozkbksDZsfrGL3BLf6r4IMqNhUhitOPqv1QcxfUvQBXqg\\\\\\\/2yXSb2uSkIQB31D4KKwYSTax+AOCBsea2qqB6Ha6sL4a4yMT2s8y6nMG9f7Sh29gKLIgcLxz8aGppVgwW2PhOyRE9SPa04\\\\\\\/kTEoShRRDdf3kaCbdbMVu8nGOMmts6xoTQDC\\\\\\\/9iygUDm\\\\\\\/HsyDWqYu1zYxfoNfu2KpMg7OPOaJD1RoatL4e53nY9MAt80i\\\\\\\/KiR6lZD0UIzgnKu\\\\\\\/sIaoD9xhnnVcjbmlKwMydBbx+0k29wETQikC81aXUgFeE5rxA8U3zWonSADDggrSpPU\\\\\\\/5aGqu7w\\\\\\\/RBXF75104IL9CQgeKzJiB81+MhxssoL9TmM+cKJzyl1OMxGEHoOS7iiGMV0ctbYd4KsHGrOTDsPzjYFRWYmwJqEuduJ4X7UPscI\\\\\\\/itA2u1y\\\\\\\/Aj4ytHHoXcjinvloxtKXdKjr4JPvTnLMS7yIQ9kj9ZLVNxfN0M+6A73biKV4egM2+GI8BSbU8N8i8hXxpKJVKPb2uv3c3gnUZqQGuz57dq7PU3p5xOZWpZ+x8B4WegKhngpAkdK2+2IeVq4CCpISScwQHWxlOOqVQFqo\\\\\\\/ONSYzJzUBYzH3zt88kq2HDX3NjvairJ\\\\\\\/89IlmcEqBwXW8D\\\\\\\/ENY7sZ9TSWPunz7LBaN5fSuK3ty2h5au093x+JXKL33ERLHSQHzcfMhEUaFezPJcxJ0KiZ16tcVLALp+jWLr+qM60SBA5SDkWagHDgIxP5o3Eqi36iCygAQdqm+2mpvo\\\\\\\/lbUItVnEwT6WVsmYQaeShGHyNRMia6l\\\\\\\/121Lj2K\\\\\\\/lpmgT0mzCdrpwhHQVeq1IVNepTtrQObQ638Q3txjDdx72UmUGL5oTgynFWNuuEmpV2fSt4apdNgK4A90uTognucbR25UHBs0BzeqftJk\\\\\\\/xr2PLRHgPhhMH\\\\\\\/kWhNbeYHt9dwfmb7ZbdgPYtLeDgHn3L\\\\\\\/3WXJtpvQMrEHc6IRs2G\\\\\\\/aBzavw5KyJKZ\\\\\\\/QhI91RKnBNHmVlxQhTuAaDoVu6z98\\\\\\\/59MpeJT44ZKvD3kXH57DKZpDz48bsAIX\\\\\\\/ru4QmhUef3rxqYFayMcGSvdnV3NOItLuKkYU9lI5BesIgRULJKXGaFs23PcX8JTpiy6fonIZjbOCKPb+uBEE7X3Sbs0oiTU0Vwto1wg1Hg0+3a7goEBXTwXuDaMcTATu2VZfeb1LODy4SsWAfFuRMZKGxeKZzr2oRoOBe1Py5VU+c6tkzHuiK6LD+qgeMwwzg\\\\\\\/QLZnkjm4L67aJkQgk3f9rh2GCKrDukPnvBq\\\\\\\/VUiAkuN1vyGqkFlQzfk+JftRKKFQEKaPE57z8er0i73CUNmeXD+3A0MbPQpr0aTkYOTRDfz9eokk6QImvXnLO1RlPRLFen2Xe5s0jn6n8VgI\\\\\\\/vJGg\\\\\\\/tttBVsvCWLfnHr2n5UEXMd4Wds54xzad+F8VOKBp83rpLpBicQQg2kB\\\\\\\/py8Br5SSm0Oq31rRUsReOefe4pPf7kAYjBqcYqu6ECsJ9pXuuTtfc+g\\\\\\\/OcHi5ynR6jVXd3RXy4z4miK9HlquM6Tt2MvNlAzycgcVdqH6TO6gR4fdy9C5gO3DZ9gzPvBGhAaG4m6tVjvgDT3Vq+u87elP1yxGgjuZgbPyiELw\\\\\\\/LQTNumbVw8DVGLjCpE1ma0OHeueBsGCfQ6TSi2pjznYC57APfNOnf+5mdRMWmhnVEi1VAbsCdWc+y5d1OcjDgr7klNydxwnsaayi+2Fx5IzlQf9Qk68AlxSvJTsqKEmurTmpidXXPcMjwzkIkZ0+JVRJbojqteJYdy6ZurS27RKAULMipq6oPNqXoPhNpcSOk1XQOzkNPu1qwah1Cn7cPUp6oiOG0c8coltReoO3t1gDUywm+igZX2\\\\\\\/+HoUK8o1ldM5yweGO2FUf0rFE9mMKeOpOtVE8hRJ6ur763vRG0CgPf121boHCFS17Eo4kiGXlS9Pj+tWPLClDbD9zFm+DZIkAX\\\\\\\/WAHk32S7FUIU2cm3yryf16R\\\\\\\/H3GPS9MmHQOMt2xgS0Apbydzs8nP1lq\\\\\\\/9V8MBFAOJvnQw63z4kU7LSrD\\\\\\\/UL5XfsZ9shoBCIvL8oP\\\\\\\/1b+fL+rFUQ0j0fsiAXS9nhkxXXXBCoVyvDA5RP8QfU73yPV2\\\\\\\/BJycqjDzz7hvz3YXDrRb2oTk5KlMHefpGx43TdLuTTI\\\\\\\/80vGbR\\\\\\\/d6HlbpIp1pqLBzzDR9lh55qigcsz4hOsBZu1WWXpeBPRPo888qFe7xbZUtUkf4iduopDb6Sa7fAM3qNqTPdsrJD0t7inI43t5pqsov9plUowMaZ\\\\\\\/EBlRpteJzN+eSqD07wSs8CmRE3gqnV8pypaCrCAvIOSA2ytXNv\\\\\\\/+IGhcnYTVZMUP3\\\\\\\/4ZxZJB0R5Ian456CZjWNJhFCNVorlajPw2aJWzr\\\\\\\/OauEQFFnhtd3+RKfbxF85Oc563vDEbf+8hL5hLMZSiiDYFM+byjdrLftpXeapweLpfM\\\\\\\/kXys+Z5lQyBfoSdwbh4aTC7pjJUNBSJv6iTyTfNlj5dEGJhbc9XeOAzxR\\\\\\\/Prd25zBSTEvYpXwLSfYFQugGR7DcEb0ovbKfEDq1gBsbG37zcQbJF4Ol5hEGA3Phc5u2V60iwAMQwuoMDG0r+SB8wTCszqcRcgRku3Oe0bWaQ0pruYlvQU\\\\\\\/E6FMwlCwXg+Q7H2rDwDwAODQ4\\\\\\\/CDKoGVAul7m5lGqL4xbWxAUe1bbQAsrp7K7AOfJOCYTm4TOn9EDmvRXvV5pdBsTaI2DwdnotkQIKmKoI8b0Pe\\\\\\\/ildZT25Nor9\\\\\\\/Wf+Q7\\\\\\\/W6YpMDtk3Ix1TbSIHgSN7j1a4hcfrfHgXp079+gH31GQiqi+oi+CNLN8dx\\\\\\\/d5SQZ2z+fMjBROPZwD\\\\\\\/if2NI5VCq1\\\\\\\/ZWR\\\\\\\/B73xZuParyS8P+cYiPW2HPPTNyBd6jEB0CCcsvCZen\\\\\\\/cW+wPMDVueNQADXkvHNEG6e6ZtWMMA9h3qyQNqdRNNaRHGHW8NWxF\\\\\\\/9gFijnAjWvFAZxsDXK6ThbN9imCrrTNMMj2wFJAwqFmziD1qfJK9HaIj7W0msYmAem6X1fmHHQhmyuZVcjdCClvbce\\\\\\\/GLYqJ7kmcOYWXp3vCo7gy3k1kZRHu9kIKnkxICDnaGZL9vHR9hm3YXvlaz0mLs26jo3EtgvH0gNAuGX3cA68iEGEp68Bs1fEKbN1cyUUII9zrU\\\\\\\/hJzLA66DoSeQQXaU2r3AfqMWd2HwSyihHRqWWO5KWuDwyFdBiV4r9vemqgrj5Jfpr4pLulCVHly3rTQ5hEagoiozTHMaCs5P54j6qrc3rn6xYscGHp6X6KDdDNI3mjf7QkTZwM5Qx5voODVJX7uewpGZa\\\\\\\/HtG0Ctsa7XP0FbK+0UssuIMR0s77dV4dwAbw5dctISeD+A7JL6rc3RH8GQIz2PqCN5jieTGO0eP63JCXSTzLcjHD15Wwq0dJyM6jL3hFQbRbzsWKzjDdD3OPlz1qDZwOdTBY6z10NOQplkpA0DLEm1tMzpzhoDDBbiRax6Bi8NuNlYZ\\\\\\\/35XCtaQmOfapn98\\\\\\\/FGFNofHa1iyP2BlVlAdeBibTxH8Y9ihbNwTvb\\\\\\\/ExYvIE7q81O1Xl6oykJlvNtcA6IkTcViCQaEvncTIaUuRBKdZp70xN5XbxRvCUKifD6qdxiVqdaO2eKYR5U4VKEleN1KXrkaANuuBYZ2LdkI5Oo02EOtaTqURm1FExAtSG6c6u0vgbWRepZRLpCmcK2b3XtWELanYrFFkvywLX9FLQtbZPSAwjVhcB6Ga0qj7qag\\\\\\\/2b15pbNOyfzYS8cspyvb\\\\\\\/od1XboI7GhJy7VY\\\\\\\/YCNsM760BnI3g+I6eaCK1Bb4keLea04mMW91hlBI5g4jMyp3eJ5HM8jgklmqaNMOrXdjZn7skTVR1ez7VaaKhvo+tZp\\\\\\\/1wgYNbHb2SAJ07SmZi3JktBDcZx1RsZC1fmDyaJCl+TJYpkyxMuMMLgFM9wqI7ZR13m7vC5VPN4fBFzHtS82VbOPYSlvnhCSQT1F+d9uNzC5Tp\\\\\\\/\\\\\\\/vDyjjCHyf22bR8e4SeZz78vZtTKMGCTm8lsPLVp6Yf6+WixRIKcHdJW2OQxvmkaxWt6vkk375hhDjJSqVXMCtCrsGx34u1Wo8OWUq337wL2ih12ohapOtosTqLhLiTxYRkN9Hfzaw0TOPiSmL9l+mMRRFxatLGjbQdiv1UTcKW4VVLyYphs0Oew7Gp7bDX8BmHgu1BBdet4H48piMzuVvL7fE1jz1lVpXXimW0lNmooWB1+Yn4a8Xa06jxbFphh7yYZG2U7noDwza3yJIvQYYlYt2yu77GajGSPrq+vVsH6De6g3fKC9YVeWVaiXI23WZsXATy2kRsKytp6rvmfffDHIz0zJw0MY4MkpeE6oFIoEMkLA7RaDrgL2GYIgwTdlPI45glKdD1BMlol9gxT4TgYGxfQV0Lvz6FP5gJjfmEEwfDE0RLBMmwX5IKb7abLHGm\\\\\\\/SLcy623sxLIbOHRA5VUdCXW\\\\\\\/C9VVSAtBZLbHVeKKcEPI6jZSUJwmvB6r1lflQust3SIkFz2LpbSIOcEv+i8aW7HPh47r\\\\\\\/\\\\\\\/hGswLoRrG2xVpiNr9es2pIUcpjAqtK+Y2\\\\\\\/OUxLdTN0nwqk4DXtC\\\\\\\/tupOUpgYslBFAzQwDzW9LxkcLRxcHWBLlJmPZv0i\\\\\\\/kUGjuqXPB8svXrqMNq5KqFQUEiR5ryVz0JngFbc6yez0AiWy+AVqtJ1+CeCfN6pYPzvxusUDScLegOhxcU4Ctr3Q24+U5XT0k3qFDZMYHyFPzBfFSybnps4dgGPio3676NlvvO2yxF1NUNHIgqDoNFJfBd35owv9QNbtfQRnDvAkXbWB6\\\\\\\/uYR\\\\\\\/1uJGIspFcu4OlVe8nLynxKd8rN7jvBzPfuYafBh70TkEh46DsnbwehCNdjOxYwOxqFjmoUnH6suHYfWbOdVsCpaqyASGPnV0OsNo4ZUcQHTjQM3a5Y3dShRrqveEymhSSsGEpq\\\\\\\/uCiN466EBPbVSSyXhJELil2Nj0A+eJWBrLYD94pCO7Q8+6GMU9FTPiqrmqJROHE2u7j84BagnivlT5D349jckBanhB\\\\\\\/fzKr0xt1t2MMVkdYvCjcyZ\\\\\\\/sDr6uI3MWD4231By97DX\\\\\\\/dmmtVvoCFrWe6BehiBpf7P5IMmVWZZ+4gLDAB3pxOuEt+lv2WEoo6gJ1yLrnLvdhX3W7vLaYxerbEXdScMVt5I40j6w\\\\\\\/M5qBJokWy6N5q8ygx5y+JIp2wLtHaLpW3XdeMaplVt2D94BqihtgJdpNZwEfe7srqsUWmDYUtnOcXi5obLFLQZ6BfqTtRPEr+Scx9hedt+RhMEtFP5wQUJbFcUdeMSSJTc\\\\\\\/GpZ0slXVNJgYBb5+uFvSKKQbju\\\\\\\/VeUCLrimU8KnxRtb7vecBF\\\\\\\/0scKEBrB1WWxtfivDXhUFz+gGEZGS\\\\\\\/jrnQHl5roiOm8lrNb3vLrsoUzKsxFzkvnNEPEQ0hgU4bvrMYj4aiGXZ1D\\\\\\\/M6K9wNqfK1TOzetQETwTA3yQkZxHh3VyZoWJF1pMsjBhegSQM25v06hKQndsqWMwubFBUsMQd1+XfgrsctYgrDXnIt7BdtfIUc1mBMa9zD\\\\\\\/n2eKnEWB7oV3j3ZbUw47diP9EEHWAagHX1kU8G3q7CsA6D4\\\\\\\/8BRUjltDFCT1A8qzSOfQ2TAsPxQ7C4sy5ykyBxLekmxMTJce\\\\\\\/5ptoSodeUzzXnT6fWxMQR6vsxXxeGcuod1nWzUQGAs8lS7aw2AGshz23dpp+qVftzhJ3HXtXCYKIkXIn0GIyBPb0UdspefF8gVy7cowhzIa9UFau0uQdSY365g36fomUhWb2OAVxvKdDZ9aSbqWRx7ddfB+nwBbtB069B4KI3LMALNork7BgelZuHBQmi0hWdV7brhOzSRBPr4x1K7MbRYDTgkZKscog80Zkg+NLaDNDkZZP29xhOmkWUekHaNEGE4zxfx38G6eVWG\\\\\\\/aULRIx4QnOTomXDbrPc5XvT1fOq92et3DPl01X9Jurubjb3sckstWdZhFGK6qrv2Gcr1eNYgw1dT4tW5DsHShCE0O1+LOrTpHHb5aOxyJWKABQBYx\\\\\\\/AauHjr1Wv2wz5iaHL0AgLGjZDgNUq2FMPq+9FXuIS\\\\\\\/xsqAPGnSy2jZgbQIQbARDRtw0surdzFZPWOg\\\\\\\/H7+G7xlLri\\\\\\\/W1tOZyLOfXteMY4VLbZmdLiEaEUDljo0TMITrTtToCDkk0YK9plunWlnWMLUmgsetKDIXL\\\\\\\/3AUswJgwSJL8RzmAFJhC7YEzDbGdOSc2sdett6TBeBsS8OTE9J52q0SUS36nTpBDDZzEa+5PxIAuEndO98offFtS2jOysd1U28nM72jmY57Nc2XYR1P\\\\\\\/vk2R\\\\\\\/PhmRdJCQEgCFkr26C2UkSfTY3XkxKcVc\\\\\\\/vLQbUa+Qw+lBvWyHWTeeY0bwk9xIgLasjJDiMDjF\\\\\\\/Luk7HNzzj4Yjk+ODgJhDc9D5SD\\\\\\\/mbrXU6E4hnKwAjVhki+torIVWGhWxkrJnDUJoS20LVDZTIQnM5ky71wHPcmQJy1mqYOv1nG0J0\\\\\\\/ftS2gu+9GaFRYgbRimbqz7Mnq\\\\\\\/OLi53bG5Y1CDulECLSdeaeiyCEnHZpQECpr605eyCAlacz+DMolo6tmmhJklJOpl3kC7nN1VVDdyNnmZhnjwV9YSfPoQFie0bOGRgxXNWyXwspfB4EUoehNEq0xBStOaCSEJ0T5pLxbLNvjxTYvlKa0WHKu5K5SGURKSU3SExCSjBV6ZbFZzaDk02n6dV0ZF+LHnNYJq5SmBcL+4TFzbg+SI4OlnbjmvXRdvDLaYu76b7aIVoihRw80Pzd2Q2J2cTksUMovkDOHbw9vqWEnJ0qcFYtQKoyd2326ZQaaHwFpVSy2kU5PZDDfhWNXkcmdDPKqsCKiWaaBP+1rgCaaoRTvEn1Gq5dIU9mZSLZ40eOIIJGYe2GudI7Y3ydMCKimpVxm8IfB+FS+14tr5z\\\\\\\/f5VDMURL10709yST2x1YyTD9eBSCgFD3m7d5fp87qS9iZBjK\\\\\\\/ymFRYYADgfVObBCpEr4svOqe97ctL3kmHKtPjk3Ps\\\\\\\/3opxqEpy1+0ZKa6PW0o4oxzVIMtQgN60mqCeuNiORqptmqUKUg\\\\\\\/SzA7N72kcGXnuBVtc4csUieadX0a\\\\\\\/tU15Pnk0rex2HVIv9K1aPi9W8fukDTlWHO+W\\\\\\\/IUWcJ3A7LIybLjwuLyLUeeDU8ebdvcnEiZoCwcMMypdE98COPlyyoRLpqUbRz4qnmUFuOIKcT21miBapmN83hIRo8beTW9Q0noLfaQa0DXCobVVM2G1Yc7Hfc7cjvwHKfqyKB3CcCW7IbKyVJKYoEKa5s6lHSeJDjSwIItCGp1q+A6KsQb0aS7ZxZHUXIQjMrlm6+tsNzPjpXyeUr++jxh45u8FneWqaJQzxY8f\\\\\\\/XbFZtDa96UM7lsDDHZxmLqyAOc4b+CnZnOcWwz2faetPUvvYMZ2fubhFp8x6wtiOgrlEAZud90wdi1l+55fk1p7YAS+X90dylQhYQ0rOz7tqe6kNbfUkmAIhTNF9pjDI6zlJYqNjFWj\\\\\\\/AZ12BKjARj5ic9fpjtfDtN5clUxEJpvTE3reQ0gOr7iI57\\\\\\\/IaUD0Ew2LgJdrURJ\\\\\\\/OxyZepQXe6NopO7nhqsiEn5L6nEwkT53fpydcFt1+rVi0EDjKtoXBngzRSWsavpaG32MmmwSgHpapqqYiWmfliLco047khLsp9fkxZ36GgLDBr6fwkYAJfs\\\\\\\/\\\\\\\/Sq0hB1CPW6KvoIu3yhddOlvUDM+xtCkgzE5vSdO5Rplh53uQjQyxw3zTIWsEJ0zxVzhAocqsj93UrEk7SjM4eZAZsjfY\\\\\\\/WNqr0Ch5Exzg+eRq0Ur1MYNrzfRWqdvdbYwikdr8I3wy\\\\\\\/7WEl7fOq4YjUCfu9oNMurj5gKWq\\\\\\\/96\\\\\\\/iWUrRmXk9A0BLDTv4BOldH6p9mendKQ+5KS8U1Gp6AUz4GgimjVLes47w2hhQKJwzTKVmkEd45h1ZpV5YlUSolRiaC0phKWMlUr7JWlH0+79FDRR6PQ1\\\\\\\/0H5Jv3SlQjKkF3+jzSUS\\\\\\\/MxnT2ZkU4wF6i75jBdFE2n\\\\\\\/PTRrwjGwce7C58FTjbhPsJ1STY6hLTKC0YSq8edkIw27pHxuf78azeBGWliebWrG26XL39q5wFdjfca6mv0y7wpjg3sxNM9LJagDXRPiDyhgeWFEstUC7qRjKgKMJdOWFXbllCq3AU48kzjg2XGK1JQZizwKTKZKi3XpXls5D+wcxK\\\\\\\/bF3FISvmfRh8y+BXV0Nel5NLZ0EKyivPwKLKZacPq8Uku3nuqNF8hDgG8O3CibHnAoxhaVgXu0IyuBYkHcFqQ6saa7oFIpmCBO+NNRgWbo1Kz2UESnxZLxR9W7vKwlwF0qQooS2Cgh8kmqZbW69Fh0Fzp1fBV2ryUfg1P7+\\\\\\\/AKk6YMQieHWxq+VqtKPVqTU2QCIlav7TeFCwJwYOn90LyPxSS5EMK49WWyJkwl61GOSBpfltVzGcJY9LlQwAkdTDQy3tVJiAIxF0gQKf5yA7qvN9+yhVjWhrHxrnpr0PPqWD8cL4mDi5FyQUD8hkiEKEqnMpBn1pzWAwULko+CX9iIVWhCe+WAouDrlOTbsaz1yvNmyk+fNRg19v1bPHEeTihu4qzZzvN8eIMhupWuYORz2JbtOpsfwxUg4TaeReZ+erRwYkg4+j28R\\\\\\\/+uxmMb9p3LUVOWfpTmeCy7OndETcmxeklijM31Y4\\\\\\\/We9US8UwOO\\\\\\\/417BCL5OjcFF5JbIZ\\\\\\\/29k4cWIvuhmevST0MS6prIdd9gckML9dMHOJZrX8+90XpOYTcvrKBrILr7CqWCP42CkMEMMrI1nAIpZ1ax3Y6f3FVGRe3OQkaHbMJU6vBd0GQAzSETOhWsbhV3Zyjp7WXnvWfX1\\\\\\\/XBicZ5MeMQVp93Zd45C4u5LIDlxhtiGtRAhneuuKiCGGh0sKZhBbAwzXtLwXosQvS2Q+ufK7MP6geg54eq2OY45R5zCG4DPcrTypOm3H3\\\\\\\/Io7k52HuQD3U3lA1PIHWlPtpIANrsBqHROPhHRGXqDEnlofASDNm\\\\\\\/wKRZEpHUMs1x930s1HWAX+reAyF\\\\\\\/Uc4Z3M1beQtVpyr86Kyd9RXXCekgTT2fSfUr9xgOpg3jiVoH+ZWiyWuafwNAw7kWrs\\\\\\\/+9vwDP19XMBhlChPzpPb1sWIGyvUMnuCnFNpLIQBi7ZKwDOZZYBmEFhFa5p4uj5JXkvAxbwuzeD\\\\\\\/5CcWcEWLxgSlo1CTKOa06k1z3esfYCyePS\\\\\\\/6Fpm6k77Atd36vTLHNHP8hx26XuRUtP4zzCADp\\\\\\\/w2sKnI2VNItd9iB0fGfu47kfYrTvFYd6I\\\\\\\/zrvLAJpORMu1OAfWeQ4GopiY\\\\\\\/NjGGnvmKNWnqGUeQkR8TgaoZqFOp1QnuJytXx0qyQbYinUZI7+VHnwBsVHFKA2Qcyk+2fcWvy3uEEBCVXhK2fRbKNm139aR9bCZi1LNwxfR32lewO\\\\\\\/ZqCBqqTbSFPvCmS8u9CCmGCVzMj1AJNOLDkvo3akhJtax6r5l6\\\\\\\/Ny3sk2G\\\\\\\/zj0CXU69SUXwk70T81+w\\\\\\\/5QHT+7rWB9QUMU4l27RZy2\\\\\\\/7V3RVnQQhBNrZQjsggwA66ETCini4YYbcHY4VGZyuB6c+7JwEXVO0Qw2QHtLcBBLQFnDNf4sgm7Iphd4XkursTXXiAYF8RPUYB5l2mtweQFRmCxkexmcq+ZfLom2Cds4vTpCoKKNhXFs8bAY39X9C36nzZZKWE47TqjNLjwCrujumuG+KDB9CE72Ium1PXdzWwauC\\\\\\\/PbPQtZzylZQzcPNbvbp4PvPULJowSplTVi2GvHVhTWvRf6Y2n6qFH2AsaM8AUbU8\\\\\\\/B6PrsBVIEN4XbSGWMsB\\\\\\\/\\\\\\\/g+9TegFQrWVAPfdP18FQz5AJ7yCW9+qOYR6XhtpKRs6PyeBuwU+EFkcB6+gUK4lPJbnG6o5f+AbHKH\\\\\\\/m5Aa9vhizCyUZznbwDBdAyTqjfGYMv5UN9UyPR3ZYMtUEIrBHXlpP9U3BRLC+7\\\\\\\/mMlEfNMsh5zapBRRdrskH2JBy89xwAZRn5xB4d0xXrUlFV6exrru5gWNmMK4q7SyMsa2\\\\\\\/WMSMc9E3to\\\\\\\/HHJQgsQiWAPnhhgGfo8UzNUrnJuLucAus+AfmTJhxr01HzeXTex8ATtvcH30tajmChjlE3r7sYjp4lRf+e2JslZ7ouXY8rSsO5xRL6uZRltF3dlPtWBqNbE6YqbFI1Bt3LH3IvctVSpmAGCeSPLZZ\\\\\\\/Ii2\\\\\\\/A4klGlMhiv\\\\\\\/qDW1XKpNdWCK9rEgX8DzCdgNtQ1+9TsdIQcfhAy+G345u3EKfBV\\\\\\\/qMi3NQamcs3Mgn6jDAw8JEsgmvbbIUoUBuXQecsroH2WMBNhN8qbJAQAo+Kgx\\\\\\\/C4UeZJwoZ2vAvmV4b75c0KK6oDf6S5vl\\\\\\\/DWZFk3DUQYfAtrQvEf5RTH2RJxOJAZ+ZbCRSVUEiMIxg4U\\\\\\\/vVvc0na9PteA0KHpz3lTzYzOSfBcWmhT4Cyk3HtCic+1fQY2T2Ph931V1Zat1ll015RZ6qwMKo7hu\\\\\\\/ubrD+GLACompql96zcl5EnsvkpKIb4PA8PWnfBtE22fDwTcTjZ3Tl6pTGYAxujbHcsAiRM1HR+MP\\\\\\\/k+blOB2krSgxX3iypW8WXF50LkkAz3+FUnqwEf11k+wrcU+4dW4cK8CB0PDR2PRfFo4kXeAWWXTuzm\\\\\\\/jHRRiqf9lyahJokO58rEqQlxF9w8qJSYnovKItky63oA\\\\\\\/CmHE\\\\\\\/Tv5oeXdDiWGZylYIWHoVuw9QHZXm2vG0lvuL+l6W4H\\\\\\\/QqtFwc121RGrq8SwLreZbyuHi60KxLNPVQfpKHzbOaslhjjSdxawNZzouDIeT3K8O5OhNw4ozxoLFschCNoLg4gr9mPnkJ+HevwyFARImQKofRLOfswkyII+A+2RY4MpN9E5AUhV\\\\\\\/kFU36sPiMcw40M2t+3LkfbN8Ze5DKsocRfdsWHcQHkHJLbgFtzp4sjNG4eBD08tyhMMqgA+nanYut4SIfNbaVNGCq3ibro4gNm7JQAN7DyOzQe3g\\\\\\\/0frkFXmUOW\\\\\\\/AOTDuX9KS6kFb1mjZgPmBgqqyMzHt088DIJZH4J0mcKWcXLOjtwp+2L4yt1kgMqaZvmIZphsv1SpmNJK672WDjHRCdIYZMQb7VOAvSAXdHlu60Jo1QWKoolvh9yNIQnTEwUUBzclnwMmZQqR3ZWhpN+Jmq9ml9jYLDy6UYsH9U1cT\\\\\\\/AvOC5z1YGSRssaZ7oVCZ\\\\\\\/lCmUdksKY4Mp5LgroKbFfEjwl63zkJvgVqKxd8tcvxkVokcRimRnbxh4JNfDw9R4cmKpBaw6iafkNv5WhPMhp2+fHBBpaRtKBHQgrDFbP19sd5iGxM9r0B9SeBhZcMT6ODF2BaV+MLCGwyRJBL2046bnbYyd1a8uGh+sgM5zjpB9vUveCUYHivrl\\\\\\\/HE+RyBSTSTk79Xe43BdhdZd+5pXeeVNnlQNeuM8DSiz0ZtY6EcNe0A81PNrk5ok7EDkr+MZSYeW5VgovelhiL4gf0C1Z7+w1yCE5jS9NeE5HENfZixjVFb190Ebkn+x9\\\\\\\/WdjTBSXCc0KwcMOHeLp46fReGkaMrhOhM+spI5Z4yRfh+RDXGHrXQ1RWAKoSEAho2YzFLVEdqafBgcu3Ztp2BHdULditptuyWxDdQL3FgY\\\\\\\/lj+A8ZkVjfK1BlsCpqYBjTS4XJJN\\\\\\\/BF6mbmz+k9a7wxyh3bg81kn7JJKzIIM+jMILEO0KvwG6KS6cfyuY8xKPShu7Ia7zg1atuQc9KSM0MgZG0r6RhMv7PU2E+0UUmr7FVr5N3h7E69GcT4GuVamdAz2kjDSZpRLvDJT\\\\\\\/QFH5YlYOdzs6kiF+a4uTIXRX38wC3XkOibOqZ9u1rBJwTMRGOPzluFb502aJhO4HXekIE+Kj29Xm\\\\\\\/g8STBsKvJZf1M4DXerM2wey70sAl5aic9nQlSjcdnwxjHsttrGAe6LZ3cYcLjvPDKuj0T7urEoTVjEhUFbGIQsGMSXY+9Yrbx2SLJ5j\\\\\\\/NKGg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/Kpc3DNM3bcXk\\\\\\\/MSYl+sAg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a3680ec4d5a60854edf8b8d374a5bddedd400c9c03a5b39f432e04127bc1d445\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OdVhcejz4XKj7j8T3w+OHt8xYHG7m7CtVQ0S0LTrxV6qERS7UNeb8DKiJc8Ib4Fkr9sWFTbygr6VNS9TenDu8r0TopUYzr5ZjGMrR6ID3PqGsR3q\\\\\\\/O8ShutTv9rE1Tg6aaKXGM9vqNCNC7+SdpyYRLNHIvRaq+2yfAtv0xQ1uxprZApRXoiJV\\\\\\\/L8voxg1ApSb0KMmO1nA3PHQ1iSX6bX9TL8J8ihKITUoScvHQNcHQM9MHJdtOcrrb9Ptjs\\\\\\\/eWjQTMIZeVSzQPbSWj2axX2CWgBRy0EzWWoa9xM39enmu7O4mWI8WUXZ56Tm5vTWnmgLvVDz1wGKVY3S+Z4fLBtN77o4C3DKaKgkWkdDYCjsxGL+wIUfVh1s66M53b5Zagw9ClvmQ4F+RidtTaxUA8o3NPQF91MwQoCELpdDCy9nv7zVYN3apVaeedlasjU6nDsgtLOR\\\\\\\/66To29EOQQ3ijanoeBxu40e8rHEZIWJFEOLNUc\\\\\\\/IXRJ5rmnsR54KlP+vzxw0\\\\\\\/XEI9\\\\\\\/QGq5indTka8EWrg\\\\\\\/yZR3tx1g4BAIsHB8G8QoSdHIQnrlx3ahiMwvtEFTtFK+mD9HNXbs+fEchO1dOo2pT9l9cN74kvPWzI0UOYihSLHiwUwHWKFy63p\\\\\\\/QQITSip6kjSa9ulqdbtglXqhDOqpy+xy9oJXqye87hM8ATBnPR+mzZC5gotX7\\\\\\\/Brc1D+gGrM4+gJEXhJgRPZ6jKpDZR6x8Xe4\\\\\\\/1Gq5qm8HB4ZyaPUX5jsfv6YzSB4gbiYrBQAeBAM1dwu8NizwnLPKgsBpIbdjoGXKxMlfT+CeKqYivwD27R0hxBR3lJo63pYi9H9WNXGwnthnLmKxuPkbN8MGiYt4PoXFaGLwaKI19OZK4mk9gaxHl8V\\\\\\\/Stmnr3ivV5A9Z\\\\\\\/XyaVfJIi+fRmVIl82r3oYDJw83kEegRH9bVucC+62SsA6fGTlt4UI5F9W6xAbqIf4h9Ns\\\\\\\/I48ZMqPcAL4UAkIRe2S1bi+IcMedufXiMYyEyRgkrAIgvmi7MsBlmdlRfAxyaY5nWAewhGZep+nMx1+FBP6D3ML5MZSNqLd484ou4zWBGCt+6wDXR3qQCwLOWaxnUXi5l79fT7vpee+Ljth8CWc8QlMbua+Sjf7nsnwCbQlyYgCR5isiYiuEqlrpZsNHPKVoMKtoRuW0jepv9b0H8wYtpIN1IZ4uRBlPJNFqSWLTqTdwBk+mIkTE\\\\\\\/GWNTeMlokE\\\\\\\/32a9ThiPE4pXOfUtwAyjpuR7I1W2Lp+mj78gXkkReOli7YeOh3BAClma8lzQF5moNmNulPP8q3BTeppvK\\\\\\\/NlI53sxX91hAYHDqtQ9i7MZveN6UAXrHZH\\\\\\\/XIsihtcza0MLJSBHpEq\\\\\\\/t\\\\\\\/jQSyiz3Bt\\\\\\\/J44gWHknqqcxQCzaLRLZEkStX8n73RZzlGBoR9u2MWQfEF1mX+lfr41ypQ8wlyDSye14bqaxCbdKupD\\\\\\\/B7RVW2siQ\\\\\\\/mcAWekGa7s5VsTyXUIyYGA6dyfW0wzkcaUvntXIve43Js1nB1hWO9FmmbG0q3pU26oSG5MBVGCqzxRZMUoz25HXnC41Eso8gq7WJh+Fcb4WZeVxWiIiR2KKr+j4Am\\\\\\\/PEtnrbro3+qNJMJbc5JhGt\\\\\\\/SXpCfnn2r9KxzOcavO\\\\\\\/qIbjU0A1PFgLSxpB1KMKBpsKIANljTEBjOEBu9jEB5HO2JSWNVPA5F2cK7AfM\\\\\\\/MxGXIgZJmhN78LF+WW6uN8jei+t20hEa3wO84xNgp5FzOCv+F5QdVFLrpClXmenId5Ge71I0jE4UwVR1LffBYO0rzG9\\\\\\\/hYTMxH5kwnL7zTxguYHTIhCMGABlRmIaVEgzI\\\\\\\/WADwyvKOJm2+c7DAWu3DTl+Ch0lPaaTwF0XiaiQJCFdpg\\\\\\\/CWrf2DeRrcTqZxysLLTNXtri\\\\\\\/hyZOTIeVMSmPD97\\\\\\\/k1t\\\\\\\/uPrxhbvZDFt6DP5ZcyEalpL9DXiMU+CwBQ9d\\\\\\\/xYLm121s3PJ44ppLwBvtp5AggA\\\\\\\/N\\\\\\\/fPfiPjr8IgAmqVRL00vcmaAYQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022RKpX5T7Ig3\\\\\\\/X25RGvjuXhA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d5c43d295bba0b7bc85dd538db5c696df5c6d63c18a28478bc93efc6eab28601\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459560 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/10/1759" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8Xf2yXUv9E5kgubR-7WJL9J7EnkeOMubJOjlMdvJL7A" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/10/1759" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aVbKbQ5mmBgCXKxlgfUwqQs6ob9Hajyx9MIIIU-1R58" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/10/1753" title="Characterization of Two ENU-Induced Mutations Affecting Mouse Skeletal Morphology" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/10/1769" title="Disruption of the Rice Plastid Ribosomal Protein S20 Leads to Chloroplast Developmental Defects and Seedling Lethality" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459560" id="node459560" data-pisa="ggg;3/10/1759" data-pisa-master="ggg;g3.113.006890" data-apath="/ggg/3/10/1759.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Whole-Genome DNA Marker Map for Cotton Based on the D-Genome Sequence of <em>Gossypium raimondii</em> L.</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zining Wang</span>, <span class="highwire-citation-author" data-delta="1">Dong Zhang</span>, <span class="highwire-citation-author" data-delta="2">Xiyin Wang</span>, <span class="highwire-citation-author" data-delta="3">Xu Tan</span>, <span class="highwire-citation-author" data-delta="4">Hui Guo</span> and <span class="highwire-citation-author" data-delta="5">Andrew H. Paterson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1759-1767;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006890 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459560" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zining Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zining%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZining%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dong Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dong%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADong%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xiyin Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiyin%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiyin%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Xu Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xu%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXu%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hui Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hui%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHui%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Andrew H. Paterson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BH.%2BPaterson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paterson%20AH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BH.%2BPaterson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:paterson@plantbio.uga.edu" class="" data-icon-position="" data-hide-link-title="0">paterson@plantbio.uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/10/1759" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459560" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459560/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/10/1759.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459560" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459560/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/10/1759.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459560" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459560/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/10/1759.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459560" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459560/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/10/1759.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459560" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459560/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has corrections. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-67706 first"><a href="/content/4/10/2049" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - October 01, 2014</a></li><li class="relation-67707 last"><a href="/content/4/11/2295" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - November 01, 2014</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">We constructed a very-high-density, whole-genome marker map (WGMM) for cotton by using 18,597 DNA markers corresponding to 48,958 loci that were aligned to both a consensus genetic map and a reference genome sequence. The WGMM has a density of one locus per 15.6 kb, or an average of 1.3 loci per gene. The WGMM was anchored by the use of colinear markers to a detailed genetic map, providing recombinational information. Mapped markers occurred at relatively greater physical densities in distal chromosomal regions and lower physical densities in the central regions, with all 1 Mb bins having at least nine markers. Hotspots for quantitative trait loci and resistance gene analog clusters were aligned to the map and DNA markers identified for targeting of these regions of high practical importance. Based on the cotton D genome reference sequence, the locations of chromosome structural rearrangements plotted on the map facilitate its translation to other Gossypium genome types. The WGMM is a versatile genetic map for marker assisted breeding, fine mapping and cloning of genes and quantitative trait loci, developing new genetic markers and maps, genome-wide association mapping, and genome evolution studies.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/quantitative-trait-loci" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-loci" rel="nofollow">quantitative trait loci</a></li><li class="kwd"><a href="/keyword/resistance-gene-analog" class="hw-term hw-article-keyword hw-article-keyword-resistance-gene-analog" rel="nofollow">resistance gene analog</a></li><li class="kwd"><a href="/keyword/simple-sequence-repeat" class="hw-term hw-article-keyword hw-article-keyword-simple-sequence-repeat" rel="nofollow">simple sequence repeat</a></li><li class="kwd"><a href="/keyword/restriction-fragment-length-polymorphism" class="hw-term hw-article-keyword hw-article-keyword-restriction-fragment-length-polymorphism" rel="nofollow">restriction fragment length polymorphism</a></li><li class="kwd"><a href="/keyword/inversions" class="hw-term hw-article-keyword hw-article-keyword-inversions" rel="nofollow">inversions</a></li></ul><p id="p-5">Genetic mapping is an essential prerequisite for the activities of marker assisted selection, gene/quantitative trait loci (QTL) cloning, genome sequence assembly, association mapping, and evolutionary studies (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Duran <em>et al.</em> 2009</a>). Genetic marker systems such as restriction fragment length polymorphism (RFLP), simple sequence repeats (SSR), sequence-related amplified polymorphism, and others have been widely used in linkage and QTL mapping (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agarwal <em>et al.</em> 2008</a>) in population sizes of up to a few hundred individuals, generally involving highly divergent parents and in strong linkage disequilibrium such that a few hundred markers provided adequate information. Whole-genome genotyping methods that are now being reduced to routine and cost-effective practice (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Xie <em>et al.</em> 2010</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Andolfatto <em>et al.</em> 2011</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bus <em>et al.</em> 2012</a>) open the door to investigations such as global trait mapping and association that require vastly greater DNA marker densities (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Poland <em>et al.</em> 2011</a>) and are likely to render many previous marker systems obsolete. However, the mapping information garnered from previous systems remains valuable, with individual QTL and particularly meta-analyses (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rong <em>et al.</em> 2007</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Zhang <em>et al.</em> 2013</a>) identifying genomic regions that might be searched at high resolution for causal genes.</p><p id="p-6">High-contiguity âreferenceâ genome sequences provide a natural platform for unifying information from a range of sequence-tagged DNA marker systems toward the efficient application of new approaches to build upon collective knowledge of the biology of an organism. Most major crops now have a reference genome sequence, and some authors have projected that within a few years all of the ~200 widely used domesticates will have such a resource (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Paterson 2006</a>). For the organism that we focus on herein, cotton (Gossypium), the smallest of eight genome types (AâG, and K) and a suspected progenitor of cultivated polyploids, the D-genome of <em>Gossypium raimondii</em>, recently was sequenced (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Paterson <em>et al.</em> 2012</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wang <em>et al.</em> 2012</a>). A high degree of colinearity between various genomes in the Gossypium genus (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Reinisch <em>et al.</em> 1994</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brubaker <em>et al.</em> 1999</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Han <em>et al.</em> 2004</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Rong <em>et al.</em> 2004</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Desai <em>et al.</em> 2006</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Becerra Lopez-Lavalle <em>et al.</em> 2011</a>) suggests that the D-reference genome will extrapolate well to most regions of most other Gossypium genomes.</p><p id="p-7">In cotton, more than 30 genetic maps have been published, largely based on interspecific crosses between two species that are each domesticated but are treated by breeders as different gene pools, <em>G. hirsutum</em> Ã <em>G. barbadense</em> (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jiang <em>et al.</em> 1998</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Zhang <em>et al.</em> 2002</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lacape <em>et al.</em> 2003</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Nguyen <em>et al.</em> 2004</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Rong <em>et al.</em> 2004</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Guo <em>et al.</em> 2007</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">He <em>et al.</em> 2007</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Yu <em>et al.</em> 2007</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lacape <em>et al.</em> 2009</a>). The interspecific tetraploid genetic maps were valuable for finding new DNA markers at a time that primary genetic mapping was a high priority (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Reinisch <em>et al.</em> 1994</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Nguyen <em>et al.</em> 2004</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Rong <em>et al.</em> 2004</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Guo <em>et al.</em> 2007</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Zhang <em>et al.</em> 2008</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Yu <em>et al.</em> 2011</a>). However, conventional breeding programs seldom use such wide crosses (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ulloa and Meredith 2000</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ulloa <em>et al.</em> 2002</a>)âand intraspecific crosses, largely within <em>G. hirsutum</em> (âUplandâ cotton) suffered from a paucity of DNA polymorphism that often left much of the genome unmapped (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Shappley <em>et al.</em> 1998a</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">b</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Ulloa and Meredith 2000</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Ulloa <em>et al.</em> 2002</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Rong <em>et al.</em> 2005a</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Shen <em>et al.</em> 2005</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">2007</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Wang <em>et al.</em> 2007</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chen <em>et al.</em> 2008</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Xu <em>et al.</em> 2008</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zheng <em>et al.</em> 2008</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Liu <em>et al.</em> 2011</a>).</p><p id="p-8">Increasing the DNA marker density of cotton genetic maps is urgent for marker-assisted selection and genomic studies. This need has been widely recognized, and several efforts have interleaved existing maps based on subsets of shared DNA markers to form âconsensusâ (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Rong <em>et al.</em> 2005b</a>) or âintegratedâ maps (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Yu <em>et al.</em> 2010</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Reddy <em>et al.</em> 2011</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Blenda <em>et al.</em> 2012</a>) with as many as 8254 loci (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Blenda <em>et al.</em> 2012</a>).</p><p id="p-9">In this study, we constructed a whole-genome marker map (WGMM) by integrating publicly available sequence tagged DNA markers with the cotton D-genome sequence. Chromosomal affiliations were deduced based on DNA markers derived from diploid and tetraploid cotton genetic maps (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Rong <em>et al.</em> 2004</a>), and we noted known differences among the genomes and subgenomes that should be considered when the cotton markers and maps are used. âHotspotsâ for QTL previously implicated in fiber development, and clusters of resistance-gene analogs (RGAs) identified in the genome sequence, also were aligned with the WGMM. The WGMM provides a foundational tool and resources for increasing knowledge of fundamental and applied elements of cotton biology, via marker-assisted breeding, fine mapping and cloning genes and QTL, genome wide association mapping, and other studies.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-10">The materials used in this study include the D-genome sequence for <em>G. raimondii</em> (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Paterson <em>et al.</em> 2012</a>), a consensus genetic map and diploid D-genome genetic map (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Rong <em>et al.</em> 2005a</a>), tetraploid At and Dt maps (<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Rong <em>et al.</em> 2004</a>; <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">2005a</a>), a cotton QTL meta-analysis (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Rong <em>et al.</em> 2007</a>), and the Cotton Marker Database (<a href="http://www.cottonmarker.org/">http://www.cottonmarker.org/</a>).</p><div id="sec-2" class="subsection"><h3>Construction of WGMM</h3><p id="p-11">The availability of sequence-tagged markers such as RFLP probes and SSR in the Cotton Marker Database (<a href="http://www.cottonmarker.org/">http://www.cottonmarker.org/</a>) and a cotton consensus map (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Rong <em>et al.</em> 2005a</a>) provides alignable information to convert genetic positions (in centiMorgans, cM) of markers to physical positions (bp).</p><p id="p-12">After marker sequences were prepared, Blastn (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em> 1990</a>) was applied to anchor markers to the cotton D-genome pseudo molecules. Markers with alignments of E-value â¤ 1e-10 for RFLP/SSR sequences and â¤50 for SSR primers were assembled into loci. For RFLPs, the alignments with distance â¤5000 bp were assembled into one RFLP locus. For SSR primers, one forward primer hit was combined with one reverse primer hit if the distance between the two hits was â¤1000 bp.</p><p id="p-13">Colinearity between genetic and physical positions was determined by ColinearScan 1.0.1 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2006</a>). The colinear markers aligned on the D-genome molecules were maintained as anchoring markers with their original genetic distance. The genetic distances of the noncolinear markers were estimated based on the genetic distance between the anchoring markers and the physical locations of the markers. Strikingly discrepant loci were removed, based on the order of markers in the original source. A QTL region was delineated by two flanking markers nearest to the likelihood peak that had alignment information.</p></div><div id="sec-3" class="subsection"><h3>Identification of RGA clusters</h3><p id="p-14">All cotton proteins were used to search for nucleotide binding site (NBS) domains (PF00931, NB-ARC) by a Hidden Markov Model method (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Eddy 1998</a>) implemented in hmmsearch version 3.0 with e-value cutoff = 1. To filter false-positive hits, all identified NBS containing proteins were screened against the Pfam-A file (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bateman <em>et al.</em> 2004</a>). NBS domains that overlap with other domains with lower E-value were considered false hits and abandoned. Likewise, the Toll/Interleukin-1 receptor (TIR) domain (PF01582) was searched against all cotton proteins by hmmsearch with e-value cutoff = 1 and putatively false hits abandoned. To detect LRR motifs, predicted NBS encoding proteins were searched against 10 LRR families in the LRR clan (CL0022) with e-value cutoff = 1. All regions predicted as LRR motifs and not overlapping with other domains with lower e-value were inferred to be real LRR motifs. Coiled coil (CC) motifs were detected by the use of NCOILS software (<a href="http://bioserv.cbs.cnrs.fr/htbin-post/pat/new/wpat.pl?dir=example_1&amp;tool=ncoils">http://bioserv.cbs.cnrs.fr/htbin-post/pat/new/wpat.pl?dir=example_1&amp;tool=ncoils</a>) with default parameters. CCs at the N-terminus of NBS domains and not overlapping with other domains were considered to identify CC-NBS type genes. Only the RGA clusters (clusters containing only RGAs) were selected for this study.</p></div><div id="sec-4" class="subsection"><h3>Dotplots</h3><p id="p-15">To compare the genetic maps constructed for diploid D, tetraploid At and Dt, and a consensus map (<a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Rong <em>et al.</em> 2004</a>, <a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">2005a</a>), the markers in these maps were mapped onto the cotton pseudomolecules by running Blastn and using their sequences against the constructed cotton assembly at the criteria of e-value &lt; 1e-10. Dotplots of genetic maps and the pseudomolecules were generated with an in-house perl program (similar to those available at <a href="http://chibba.agtec.uga.edu/duplication/">http://chibba.agtec.uga.edu/duplication/</a>).</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>Marker number and density on chromosomes</h3><p id="p-16">The total of 20,096 sequence-tagged cotton DNA markers were first filtered for sequence/primer duplications. In total, 18,597 nonredundant markers (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) were used to do Blastn searches against the D-genome pseudomolecules, and 79,481 sequence alignments could be aligned to the D genome. After we filtered clustered duplicate copies within 1 kb for SSR and 5 kb for RFLP, 48,546 loci were mapped on the 13 chromosomes of WGMM and 412 loci on the unassembled scaffolds (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006890/-/DC1/006890SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006890/-/DC1/TableS1.xlsx">Table S1</a>). The 18,597 markers used for the alignments have an average of 2.63 copies per marker sequence, ranging from 6493 (34.91%) markers with a single copy to 1195 markers with four copies (6.43%: <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Among the 48,958 mapped loci, 42,794 (87.41%) are SSRs and 6164 (12.59%) are RFLPs. Loci from markers with five copies accounted for the largest share (45.81%) of the 48,958 loci, and the loci from four copy-markers had the smallest share (9.76%).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463325/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463325/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Numbers and percentages of markers and loci on the WGMM</span><div class="sb-div caption-clear"></div></div></div><p id="p-18">Marker density was closely correlated with the physical DNA content of chromosomes (r = 0.81; <em>P</em> = 0.0004), with size differences among the 13 chromosomes reflected by differences in locus number (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006890/-/DC1/TableS2.pdf">Table S2</a>). D12, with 34.9 Mb DNA, has the fewest loci mapped (2386); D09 with 70.7 Mb has the most (5370). Marker density along a chromosome ranges from 12.8 to 18.5 kb per locus (D07, D10), with an average of 15.6 kb per locus. Relative to the 37,505 genes in the current D genome annotation, the WGMM provides 1.3 marker loci per gene.</p></div><div id="sec-7" class="subsection"><h3>Marker distribution on chromosomes</h3><p id="p-19">With one exception, the cotton chromosomes have relatively greater marker densities of 100â220 markers per 1 Mb in terminal chromosomal regions that are also gene-rich (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), and significantly lower marker densities (50â120 loci per Mb) in central regions that are gene-poor and repeat-rich (see the gene and repeat density heatmap in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). A few greater peaks appear in the middle regions of some chromosomes that are also gene rich.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/10/1759/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="DNA marker, gene, and repetitive DNA distributions on the 13 D-genome pseudomolecules and the WGMM. The x-axis indicates the lengths of the chromosomes in Mb and the y-axis shows the number of markers per 1 Mb window in a range of 0&#x2212;300. The shaded fields in blue display the marker distribution in 1-Mb windows at 100-kb steps from 0 Mb to the end of each chromosome. Gene (red) and repetitive sequence (blue) density heatmaps for the chromosomes are above each marker distribution map." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1204234798" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;DNA marker, gene, and repetitive DNA distributions on the 13 D-genome pseudomolecules and the WGMM. The x-axis indicates the lengths of the chromosomes in Mb and the y-axis shows the number of markers per 1 Mb window in a range of 0&#x2212;300. The shaded fields in blue display the marker distribution in 1-Mb windows at 100-kb steps from 0 Mb to the end of each chromosome. Gene (red) and repetitive sequence (blue) density heatmaps for the chromosomes are above each marker distribution map.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/10/1759/F1.medium.gif" width="440" height="366"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/10/1759/F1.medium.gif" width="440" height="366"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/10/1759/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/10/1759/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463374" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">DNA marker, gene, and repetitive DNA distributions on the 13 D-genome pseudomolecules and the WGMM. The x-axis indicates the lengths of the chromosomes in Mb and the y-axis shows the number of markers per 1 Mb window in a range of 0â300. The shaded fields in blue display the marker distribution in 1-Mb windows at 100-kb steps from 0 Mb to the end of each chromosome. Gene (red) and repetitive sequence (blue) density heatmaps for the chromosomes are above each marker distribution map.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">To assess the coverage of this map, we determined the number of markers in each of the 34â69 1-Mb windows of D01-D13. The least-populated window (on D02) had nine markers and a physical distance of 110 kb between markers, indicative of good coverage of the whole genome by these markers. The WGMM covered 98.11% of the 747 Mb assembled portion of the cotton D genome of total 761.4 Mb (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Paterson <em>et al.</em> 2012</a>).</p><p id="p-22">One unusual chromosome, D06, had high marker density consistent with the remaining chromosomes at one end but unusually low density at the other end that was more consistent with the central regions of other chromosomes (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). The unusual chromosome arm (0â27 Mb) had lower gene and higher repeat densities that may have been the main reasons for the lower marker density, noting that most markers were SSRs developed from expressed sequence tags. We also searched the ribosomal DNA (rDNA) sequences. The rDNA (18S, 5S, and 25S) was mainly distributed on three chromosomes, with 40 rDNA genes in a 249-kb region (20,789,590â21,038,157 bp) and a 544-kb region (32,723,102â33,266,822 bp) on D08, 45 rDNA genes in a 400-kb region (43,290,160â43,692,820 bp) on D09 and 111 rDNA genes in a 362-kb region (55,503,532â55,865,427 bp) on D01 (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). Another 33 rDNA genes were scattered, with 1â11 gene copies on D02, 03, 04, 05, 06, 07, 10, 12, and 13. The rDNA genes caused lower marker density in local regions that were too short to be labeled in the figure and did not significantly influence marker and gene density.</p></div><div id="sec-8" class="subsection"><h3>Marker alignment from consensus map to WGMM</h3><p id="p-23">To integrate recombination patterns into the WGMM, it was aligned with a consensus genetic map of 2325 cM (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006890/-/DC1/TableS2.pdf">Table S2</a>) that was used in assembling the D genome pseudo molecules, and has been confirmed to be a good representation of At and Dt genetic maps (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006890/-/DC1/TableS1.xlsx">Table S1</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006890/-/DC1/TableS2.pdf">Table S2</a>; <a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">Rong <em>et al.</em> 2004</a>). Altogether, 1725 markers from the consensus map could be mapped on the WGMM, with 86 to 220 per chromosome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006890/-/DC1/TableS3.pdf">Table S3</a>). Approximately 60% of the mapped markers showing colinearity between physical and genetic locations on the D genome were used as anchor markers on WGMM and their genetic distances transferred to the WGMM. Genetic locations of nonanchor markers on the WGMM were interpolated by using the genetic and physical spacings of flanking markers.</p></div><div id="sec-9" class="subsection"><h3>Cotton fiber QTL hotspots</h3><p id="p-24">From a meta-analysis of 432 cotton QTL (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Rong <em>et al.</em> 2007</a>), we mapped to the WGMM 104 fiber QTL that comprised 18 âhotspotsâ containing QTL for 3â7 fiber traits including fiber elongation, color, fineness, length, strength, uniformity, micronaire and short fiber content (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Chromosome 07 and 23 each have the largest number of 16 QTL. <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> lists the interval locations and the number of markers in the intervals on D-genome molecules.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463335/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463335/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">QTL from At and Dt tetraploid chromosomes with their consensus locations and markers on the D genome and WGMM</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Cotton RGA hotspots</h3><p id="p-26">A total of 63 RGA clusters were identified on 9 of the 13 D-genome chromosomes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006890/-/DC1/TableS4.pdf">Table S4</a>). D07 has the largest number of clusters (21), whereas D04 has the smallest (one, with only two RGAs). The cluster positioned at 51,589,128 bp on D09 is the largest cluster with seven RGAs. Because there is no other gene in the RGA clusters, some clusters with only two RGAs have no markers within the interval whereas the largest cluster has five markers. Cluster positions and their closest flanking markers are listed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006890/-/DC1/TableS4.pdf">Table S4</a>) to facilitate the use of these markers in expedited searches for loci responsible for disease and/or pest resistance phenotypes.</p></div><div id="sec-11" class="subsection"><h3>Comparative map alignments of Di, At, Dt to the D-genome sequence</h3><p id="p-27">To further enhance alignment of the WGMM to previous genetic information, we aligned the genetic maps of the diploid D (<a id="xref-ref-27-8" class="xref-bibr" href="#ref-27">Rong <em>et al.</em> 2004</a>), tetraploid At and Dt (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Reinisch <em>et al.</em> 1994</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Lacape <em>et al.</em> 2003</a>; <a id="xref-ref-27-9" class="xref-bibr" href="#ref-27">Rong <em>et al.</em> 2004</a>), with the D-genome pseudo molecules. To avoid confusion, Di represents the D genome genetic map and D the D-genome sequence in the text, tables, and figures hereafter. All chromosomes from each genetic map could be aligned to the WGMM with marker numbers per chromosome of 21â65 (Di), 20â93 (At), and 38â95 (Dt). The numbers of aligned markers on the corresponding and other chromosomes are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006890/-/DC1/TableS3.pdf">Table S3</a>.</p></div><div id="sec-12" class="subsection"><h3>Genome variations</h3><p id="p-28">To study the colinearity and genome variations between the genetic maps and the D genome sequence, dotplots were performed between the Di, At, and Dt genetic maps and the D genome pseudomolecules (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A and B</a>). Here, we listed the large inversions (&gt;1 Mb) different from those found in <a id="xref-ref-27-10" class="xref-bibr" href="#ref-27">Rong <em>et al.</em> (2004)</a> based on the comparison of genetic maps. The D genome diploid genetic map had been used to construct the pseudomolecules, and therefore, the dotplot between them showed good consistency as expected (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). The Dt genetic map also exhibited colinearity with 11 of the pseudo molecules, with chromosome 15 having an inversion (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>, red oval) of 14.71 Mb accounting for 23.44% of D02 (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Four inversions (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>) relative to At chromosomes with sizes ranging from 3.33 to 6.27 Mb were identified on Chromosomes 04, 10, 12, and LGA03, accounting for 5.36â9.95% of the corresponding chromosomes.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/10/1759/F2/graphic-2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the Di, At, and Dt genetic maps and their physical locations on the pseudo molecules of the D-genome sequence of G. raimondii. The 13 D-genome pseudo molecules are plotted on all y-axes. (A) Di01-Di13 indicates the 13 chromosomes on the D-genome genetic map; (B) Chr.01&#x2212;Chr.26 indicates the 26 tetraploid chromosomes of cotton. Both the relative genetic and physical distances of the chromosomes on the plots are represented by the cells of different sizes according to the ratio of the chromosome lengths. Chromosome arm inversions (red ovals) and translocations (red arrows) are highlighted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1204234798" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of the Di, At, and Dt genetic maps and their physical locations on the pseudo molecules of the D-genome sequence of &lt;em&gt;G. raimondii&lt;/em&gt;. The 13 D-genome pseudo molecules are plotted on all y-axes. (A) Di01-Di13 indicates the 13 chromosomes on the D-genome genetic map; (B) Chr.01&#x2212;Chr.26 indicates the 26 tetraploid chromosomes of cotton. Both the relative genetic and physical distances of the chromosomes on the plots are represented by the cells of different sizes according to the ratio of the chromosome lengths. Chromosome arm inversions (red ovals) and translocations (red arrows) are highlighted.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/10/1759/F2/graphic-2.medium.gif" width="429" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/3/10/1759/F2/graphic-2.medium.gif" width="429" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/10/1759/F2/graphic-2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/10/1759/F2/graphic-2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463386" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/10/1759/F2/graphic-3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the Di, At, and Dt genetic maps and their physical locations on the pseudo molecules of the D-genome sequence of G. raimondii. The 13 D-genome pseudo molecules are plotted on all y-axes. (A) Di01-Di13 indicates the 13 chromosomes on the D-genome genetic map; (B) Chr.01&#x2212;Chr.26 indicates the 26 tetraploid chromosomes of cotton. Both the relative genetic and physical distances of the chromosomes on the plots are represented by the cells of different sizes according to the ratio of the chromosome lengths. Chromosome arm inversions (red ovals) and translocations (red arrows) are highlighted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1204234798" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of the Di, At, and Dt genetic maps and their physical locations on the pseudo molecules of the D-genome sequence of &lt;em&gt;G. raimondii&lt;/em&gt;. The 13 D-genome pseudo molecules are plotted on all y-axes. (A) Di01-Di13 indicates the 13 chromosomes on the D-genome genetic map; (B) Chr.01&#x2212;Chr.26 indicates the 26 tetraploid chromosomes of cotton. Both the relative genetic and physical distances of the chromosomes on the plots are represented by the cells of different sizes according to the ratio of the chromosome lengths. Chromosome arm inversions (red ovals) and translocations (red arrows) are highlighted.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/10/1759/F2/graphic-3.medium.gif" width="424" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/3/10/1759/F2/graphic-3.medium.gif" width="424" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/10/1759/F2/graphic-3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/10/1759/F2/graphic-3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463386" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Comparison of the Di, At, and Dt genetic maps and their physical locations on the pseudo molecules of the D-genome sequence of <em>G. raimondii</em>. The 13 D-genome pseudo molecules are plotted on all y-axes. (A) Di01-Di13 indicates the 13 chromosomes on the D-genome genetic map; (B) Chr.01âChr.26 indicates the 26 tetraploid chromosomes of cotton. Both the relative genetic and physical distances of the chromosomes on the plots are represented by the cells of different sizes according to the ratio of the chromosome lengths. Chromosome arm inversions (red ovals) and translocations (red arrows) are highlighted.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463339/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463339/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Genome variations identified via comparative mapping against D-genome pseudomolecules</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">The At genomes displayed two reciprocal chromosome arm translocations (<a id="xref-ref-27-11" class="xref-bibr" href="#ref-27">Rong <em>et al.</em> 2004</a>) between chromosome 02/03 and between chromosome 04/05 (red arrows in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>) and their physical location information on the D-genome sequence was added here. The translocations broke the chromosomes near centromere regions, such as chromosome 02/03 corresponding to pseudo molecule D03 at 2,4043,418â29,931,543 bp (PAR0499âA1171) and chromosome 03/02 to D05 at 23,203,279â3,4341,901 bp (A1325-Unig26F10); chromosome 04/05 to D09 at 46,366,195â39,825,818 bp (Unig27B06-Gate4DF07) and chromosome 05/04 to D12 at 2,1985,484â24,460,316 bp (Unig06C12-Coau2105). <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> lists the flanking markers for the five inversions with their locations on both the genetic maps and the WGMM.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-32">The cotton WGMM described herein has a total of 48,959 loci, about six times the number characterized in the most richly populated of the integrated maps published previously (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Blenda <em>et al.</em> 2012</a>). Comparable with a linkage map of rice based on population sequencing with 15,795 SNPs (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Xie <em>et al.</em> 2010</a>) and a Brassica genetic map of 13,551 sequence-related amplified polymorphismÂ markers (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Sun <em>et al.</em> 2007</a>), this map may facilitate fine mapping, gene cloning, global association mapping of cotton genes and traits, and other genomic studies.</p><div id="sec-14" class="subsection"><h3>Distribution of markers</h3><p id="p-33">Genetic maps always face the problem of unevenly distributed markers and the resulting gaps. During meiosis, recombination does not happen evenly on the chromosomes. Further, marker sequences are not randomly dispersed, especially for sequence repeats such as SSRs. In centromeric and intergenic regions, more sequence repeats were found (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Paterson <em>et al.</em> 2012</a>). Previous genetic maps illustrated the uneven distribution of markers with many large gaps; on a recombinational scale, distal chromosomal regions tended to have lower marker density than the centromere regions (<a id="xref-ref-27-12" class="xref-bibr" href="#ref-27">Rong <em>et al.</em> 2004</a>). On the physical scale used in this study, the distal gene-rich ends of the chromosomes clearly have greater marker density, a natural outcome of the fact that many of the markers used are gene-derived. Not a single 1-Mb window in any chromosome had fewer than nine markers (<em>i.e.</em>, an average of one per 110 kb), and most had at least 50 markers, guaranteeing the availability of established DNA markers in any region of interest.</p></div><div id="sec-15" class="subsection"><h3>Alignment of QTL and RGA hotspots</h3><p id="p-34">Global genotyping is the trend for dissecting genes/QTL controlling important phenotypes, and prior QTL information provides valuable evidence toward validation of statistically significant associations from genome-wide association studies. The cotton community has identified at least hundreds of QTL for fiber-related traits, plant architecture, disease resistance, and stress tolerance [a meta-analysis now 6 years old already identified 432 QTL (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Rong <em>et al.</em> 2007</a>)]. Here, we provide resources to quickly and efficiently target QTL âhotspotsâ for fiber-related or disease/pest-related traits for rapid characterization, for example in searches for novel alleles. For example, genotyping of as few as 126 DNA markers would permit one to assess cosegregation at DNA markers flanking with 53% (159/300) of the RGAs in the cotton reference sequence.</p></div><div id="sec-16" class="subsection"><h3>Genome variations revealed by comparative analysis of maps</h3><p id="p-35">Although the various cotton genomes have a high degree of colinearity, several rearrangements known from prior studies (<a id="xref-ref-27-13" class="xref-bibr" href="#ref-27">Rong <em>et al.</em> 2004</a>) and the five possible inversions identified here need to be accounted for in âtranslationâ of information from the D-genome based WGMM to other cotton genomes. The At genome experienced two reciprocal chromosome arm translocations, between chromosome 02 and 03 and between chromosome 04 and 05, with break points near the centromere regions. These genome variations may facilitate the understanding of genome evolution and gene/QTL cloning from the related orthologous regions. Furthermore, to characterize the variations, the WGMM provides a foundation for and will facilitate the investigation of the relationships of phenotype and genotype of important agronomic traits, especially those controlled by A and At genomes. We noted that some of the newly identified and relatively smaller inversions could represent misassemblies of the reference genome sequence.</p><p id="p-36">In conclusion, the consensus high-density WGMM is a valuable resource with the potential for adding additional value as more information accumulates, such as better knowledge of QTL hotspots for cotton fiber development, roles of different RGA clusters in conferring pest resistance genes and QTL, global association studies of cotton, and/or genome structure and variation.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-37">A.H.P. appreciates financial support from the U.S. National Science Foundation (DBI 0817707, IIP-0917856, IIP-1127755), USDA National Research Initiative (00-52100-9685, 02-35301-12045), Cotton, Inc, and the Pakistan-US Science and Technology Cooperation Program.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006890/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006890/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-38"><em>Communicating editor: D. Zamir</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-05-23"><span class="received-label">Received </span>May 23, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-08-04"><span class="accepted-label">Accepted </span>August 4, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2013 Wang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.1" data-doi="10.1007/s00299-008-0507-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Agarwal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shrivastava</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Padh</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Advances in molecular marker techniques and their applications in plant sciences.</span> <abbr class="cit-jnl-abbrev">Plant Cell Rep.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">617</span>â<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bcell%2Breports%26rft.stitle%253DPlant%2BCell%2BRep%26rft.aulast%253DAgarwal%26rft.auinit1%253DM.%26rft.volume%253D27%26rft.issue%253D4%26rft.spage%253D617%26rft.epage%253D631%26rft.atitle%253DAdvances%2Bin%2Bmolecular%2Bmarker%2Btechniques%2Band%2Btheir%2Bapplications%2Bin%2Bplant%2Bsciences.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00299-008-0507-z%26rft_id%253Dinfo%253Apmid%252F18246355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00299-008-0507-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18246355&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254205200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span>â<span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">225</span>: <span class="cit-fpage">403</span>â<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BBiol.%26rft.volume%253D225%26rft.spage%253D403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.3" data-doi="10.1101/gr.115402.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davison</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erezyilmaz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mast</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Multiplexed shotgun genotyping for rapid and efficient genetic mapping.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">610</span>â<span class="cit-lpage">617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.115402.110%26rft_id%253Dinfo%253Apmid%252F21233398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzQvNjEwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMC8xNzU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.4" data-doi="10.1093/nar/gkh121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bateman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finn</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollich</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">The Pfam protein families database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">D138</span>â<span class="cit-lpage">D141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBateman%26rft.auinit1%253DA.%26rft.volume%253D32%26rft.issue%253D90001%26rft.spage%253DD138%26rft.epage%253D141%26rft.atitle%253DThe%2BPfam%2Bprotein%2Bfamilies%2Bdatabase%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh121%26rft_id%253Dinfo%253Apmid%252F14681378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzMi9zdXBwbF8xL0QxMzgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE3NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Becerra Lopez-Lavalle</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matheson</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">A genetic map of an Australian wild Gossypium C genome and assignment of homoeologies with tetraploid cultivated cotton.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">779</span>â<span class="cit-lpage">794</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DBecerra%2BLopez-Lavalle%26rft.auinit1%253DL.%2BA.%26rft.volume%253D54%26rft.issue%253D9%26rft.spage%253D779%26rft.epage%253D794%26rft.atitle%253DA%2Bgenetic%2Bmap%2Bof%2Ban%2BAustralian%2Bwild%2BGossypium%2BC%2Bgenome%2Band%2Bassignment%2Bof%2Bhomoeologies%2Bwith%2Btetraploid%2Bcultivated%2Bcotton.%26rft_id%253Dinfo%253Apmid%252F21905768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21905768&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.6" data-doi="10.1371/journal.pone.0045739"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blenda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rami</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garsmeur</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">A high density consensus genetic map of tetraploid cotton that integrates multiple component maps through molecular marker redundancy check.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e45739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBlenda%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De45739%26rft.epage%253De45739%26rft.atitle%253DA%2Bhigh%2Bdensity%2Bconsensus%2Bgenetic%2Bmap%2Bof%2Btetraploid%2Bcotton%2Bthat%2Bintegrates%2Bmultiple%2Bcomponent%2Bmaps%2Bthrough%2Bmolecular%2Bmarker%2Bredundancy%2Bcheck.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0045739%26rft_id%253Dinfo%253Apmid%252F23029214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0045739&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23029214&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.7" data-doi="10.1139/g98-118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Comparative genetic mapping of allotetraploid cotton and its diploid progenitors.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">184</span>â<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft.volume%253D42%26rft.spage%253D184%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg98-118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g98-118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.8" data-doi="10.1186/1471-2164-13-281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bus</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hecht</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huettel</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinhardt</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stich</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">High-throughput polymorphism detection and genotyping in <em>Brassica napus</em> using next-generation RAD sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBus%26rft.auinit1%253DA.%26rft.volume%253D13%26rft.spage%253D281%26rft.epage%253D281%26rft.atitle%253DHigh-throughput%2Bpolymorphism%2Bdetection%2Band%2Bgenotyping%2Bin%2BBrassica%2Bnapus%2Busing%2Bnext-generation%2BRAD%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-281%26rft_id%253Dinfo%253Apmid%252F22726880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22726880&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.9" data-doi="10.3724/SP.J.1006.2008.01199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Genetic linkage map construction and QTL mapping for yield and fiber quality in upland cotton.</span> <abbr class="cit-jnl-abbrev">Acta Agron. Sin.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1199</span>â<span class="cit-lpage">1205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BAgron.%2BSin.%26rft.volume%253D34%26rft.spage%253D1199%26rft_id%253Dinfo%253Adoi%252F10.3724%252FSP.J.1006.2008.01199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3724/SP.J.1006.2008.01199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chee</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">O. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Chromosome structural changes in diploid and tetraploid A genomes of Gossypium.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">336</span>â<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DDesai%26rft.auinit1%253DA.%26rft.volume%253D49%26rft.issue%253D4%26rft.spage%253D336%26rft.epage%253D345%26rft.atitle%253DChromosome%2Bstructural%2Bchanges%2Bin%2Bdiploid%2Band%2Btetraploid%2BA%2Bgenomes%2Bof%2BGossypium.%26rft_id%253Dinfo%253Apmid%252F16699553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16699553&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.11" data-doi="10.1007/978-1-59745-427-8_3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duran</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batley</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Genetic maps and the use of synteny.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">513</span>: <span class="cit-fpage">41</span>â<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DDuran%26rft.auinit1%253DC.%26rft.volume%253D513%26rft.spage%253D41%26rft.epage%253D55%26rft.atitle%253DGenetic%2Bmaps%2Band%2Bthe%2Buse%2Bof%2Bsynteny.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-59745-427-8_3%26rft_id%253Dinfo%253Apmid%252F19347649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-59745-427-8_3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19347649&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.12" data-doi="10.1093/bioinformatics/14.9.755"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Profile hidden Markov models.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">755</span>â<span class="cit-lpage">763</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DEddy%26rft.auinit1%253DS.%2BR.%26rft.volume%253D14%26rft.issue%253D9%26rft.spage%253D755%26rft.epage%253D763%26rft.atitle%253DProfile%2Bhidden%2BMarkov%2Bmodels%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F14.9.755%26rft_id%253Dinfo%253Apmid%252F9918945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxNC85Lzc1NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTAvMTc1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.13" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.14" data-doi="10.1534/genetics.107.070375"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">A microsatellite-based, gene-rich linkage map reveals genome structure, function and evolution in Gossypium.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">527</span>â<span class="cit-lpage">541</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.070375%26rft_id%253Dinfo%253Apmid%252F17409069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc2LzEvNTI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMC8xNzU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.15" data-doi="10.1007/s00438-004-1059-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Genetic mapping of EST-derived microsatellites from the diploid <em>Gossypium arboreum</em> in allotetraploid cotton.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">272</span>: <span class="cit-fpage">308</span>â<span class="cit-lpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DHan%26rft.auinit1%253DZ.%2BG.%26rft.volume%253D272%26rft.issue%253D3%26rft.spage%253D308%26rft.epage%253D327%26rft.atitle%253DGenetic%2Bmapping%2Bof%2BEST-derived%2Bmicrosatellites%2Bfrom%2Bthe%2Bdiploid%2BGossypium%2Barboreum%2Bin%2Ballotetraploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-004-1059-8%26rft_id%253Dinfo%253Apmid%252F15368122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-004-1059-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15368122&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">Z. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nie</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">QTL mapping for economic traits based on a dense genetic map of cotton with PCR-based markers using the interspecific cross of <em>Gossypium hirsutum</em> Ã <em>Gossypium barbadense</em>.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">181</span>â<span class="cit-lpage">197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D153%26rft.spage%253D181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.17" data-doi="10.1073/pnas.95.8.4419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El-Zik</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Polyploid formation created unique avenues for response to selection in Gossypium (cotton).</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">4419</span>â<span class="cit-lpage">4424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DJiang%26rft.auinit1%253DC.-x.%26rft.volume%253D95%26rft.issue%253D8%26rft.spage%253D4419%26rft.epage%253D4424%26rft.atitle%253DPolyploid%2Bformation%2Bcreated%2Bunique%2Bavenues%2Bfor%2Bresponse%2Bto%2Bselection%2Bin%2BGossypium%2B%2528cotton%2529%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.8.4419%26rft_id%253Dinfo%253Apmid%252F9539752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5NS84LzQ0MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE3NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.18" data-doi="10.1139/g03&amp;ndash;050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lacape</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thibivilliers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bojinov</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Courtois</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">A combined RFLP-SSR-AFLP map of tetraploid cotton based on a <em>Gossypium hirsutum</em> x <em>Gossypium barbadense</em> backcross population.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">612</span>â<span class="cit-lpage">626</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DLacape%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D46%26rft.issue%253D4%26rft.spage%253D612%26rft.epage%253D626%26rft.atitle%253DA%2Bcombined%2BRFLP-SSR-AFLP%2Bmap%2Bof%2Btetraploid%2Bcotton%2Bbased%2Bon%2Ba%2BGossypium%2Bhirsutum%2Bx%2BGossypium%2Bbarbadense%2Bbackcross%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg03%2526ndash%253B050%26rft_id%253Dinfo%253Apmid%252F12897870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g03&amp;ndash;050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12897870&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.19" data-doi="10.1007/s00122-009-1037-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lacape</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arioli</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derijcker</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forestier-Chiron</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A new interspecific, <em>Gossypium hirsutum</em> x <em>G. barbadense</em>, RIL population: towards a unified consensus linkage map of tetraploid cotton.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">281</span>â<span class="cit-lpage">292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLacape%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D119%26rft.issue%253D2%26rft.spage%253D281%26rft.epage%253D292%26rft.atitle%253DA%2Bnew%2Binterspecific%252C%2BGossypium%2Bhirsutum%2Bx%2BG.%2Bbarbadense%252C%2BRIL%2Bpopulation%253A%2Btowards%2Ba%2Bunified%2Bconsensus%2Blinkage%2Bmap%2Bof%2Btetraploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1037-y%26rft_id%253Dinfo%253Apmid%252F19387609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1037-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19387609&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.20" data-doi="10.1007/s00122-011-1597-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Differential gene expression and associated QTL mapping for cotton yield based on a cDNA-AFLP transcriptome map in an immortalized F2.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">439</span>â<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLiu%26rft.auinit1%253DR.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D439%26rft.epage%253D454%26rft.atitle%253DDifferential%2Bgene%2Bexpression%2Band%2Bassociated%2BQTL%2Bmapping%2Bfor%2Bcotton%2Byield%2Bbased%2Bon%2Ba%2BcDNA-AFLP%2Btranscriptome%2Bmap%2Bin%2Ban%2Bimmortalized%2BF2.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1597-5%26rft_id%253Dinfo%253Apmid%252F21512772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1597-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21512772&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.21" data-doi="10.1007/s00122-004-1612-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giband</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brottier</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Risterucci</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacape</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Wide coverage of the tetraploid cotton genome using newly developed microsatellite markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">167</span>â<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DNguyen%26rft.auinit1%253DT.%2BB.%26rft.volume%253D109%26rft.issue%253D1%26rft.spage%253D167%26rft.epage%253D175%26rft.atitle%253DWide%2Bcoverage%2Bof%2Bthe%2Btetraploid%2Bcotton%2Bgenome%2Busing%2Bnewly%2Bdeveloped%2Bmicrosatellite%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1612-1%26rft_id%253Dinfo%253Apmid%252F14997301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1612-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14997301&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221936300020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.22" data-doi="10.1038/nrg1806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Leafing through the genomes of our major crop plants: strategies for capturing unique information.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">174</span>â<span class="cit-lpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPaterson%26rft.auinit1%253DA.%2BH.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D174%26rft.epage%253D184%26rft.atitle%253DLeafing%2Bthrough%2Bthe%2Bgenomes%2Bof%2Bour%2Bmajor%2Bcrop%2Bplants%253A%2Bstrategies%2Bfor%2Bcapturing%2Bunique%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1806%26rft_id%253Dinfo%253Apmid%252F16485017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16485017&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235384300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.23" data-doi="10.1038/nature11798"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gundlach</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibres.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">492</span>: <span class="cit-fpage">423</span>â<span class="cit-lpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D492%26rft.spage%253D423%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11798%26rft_id%253Dinfo%253Apmid%252F23257886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11798&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23257886&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312488200055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.24" data-doi="10.1073/pnas.1010894108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Genome-wide nested association mapping of quantitative resistance to northern leaf blight in maize.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">6893</span>â<span class="cit-lpage">6898</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1010894108%26rft_id%253Dinfo%253Apmid%252F21482771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzE3LzY4OTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE3NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">U. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nimmakayala</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vajja</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rahman</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Use of diversity arrays technology markers for integration into a cotton reference map and anchoring to a recombinant inbred line map.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">349</span>â<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DReddy%26rft.auinit1%253DU.%2BK.%26rft.volume%253D54%26rft.issue%253D5%26rft.spage%253D349%26rft.epage%253D359%26rft.atitle%253DUse%2Bof%2Bdiversity%2Barrays%2Btechnology%2Bmarkers%2Bfor%2Bintegration%2Binto%2Ba%2Bcotton%2Breference%2Bmap%2Band%2Banchoring%2Bto%2Ba%2Brecombinant%2Binbred%2Bline%2Bmap.%26rft_id%253Dinfo%253Apmid%252F21524184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21524184&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reinisch</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stelly</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">A detailed RFLP map of cotton, <em>Gossypium hirsutum</em> x <em>Gossypium barbadense</em>: chromosome organization and evolution in a disomic polyploid genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">829</span>â<span class="cit-lpage">847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DReinisch%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D829%26rft.epage%253D847%26rft.atitle%253DA%2Bdetailed%2BRFLP%2Bmap%2Bof%2Bcotton%252C%2BGossypium%2Bhirsutum%2Bx%2BGossypium%2Bbarbadense%253A%2Bchromosome%2Borganization%2Band%2Bevolution%2Bin%2Ba%2Bdisomic%2Bpolyploid%2Bgenome.%26rft_id%253Dinfo%253Apmid%252F7851778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvODI5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMC8xNzU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.27" data-doi="10.1534/genetics.166.1.389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbey</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">A 3347-locus genetic recombination map of sequence-tagged sites reveals features of genome organization, transmission and evolution of cotton (Gossypium).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">389</span>â<span class="cit-lpage">417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRong%26rft.auinit1%253DJ.%26rft.volume%253D166%26rft.issue%253D1%26rft.spage%253D389%26rft.epage%253D417%26rft.atitle%253DA%2B3347-Locus%2BGenetic%2BRecombination%2BMap%2Bof%2BSequence-Tagged%2BSites%2BReveals%2BFeatures%2Bof%2BGenome%2BOrganization%252C%2BTransmission%2Band%2BEvolution%2Bof%2BCotton%2B%2528Gossypium%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.1.389%26rft_id%253Dinfo%253Apmid%252F15020432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY2LzEvMzg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMC8xNzU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.28" data-doi="10.1101/gr.3907305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulze</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waghmare</span>  <span class="cit-name-given-names">V. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">a</span>â<span class="cit-article-title">Comparative genomics of Gossypium and Arabidopsis: unraveling the consequences of both ancient and recent polyploidy.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1198</span>â<span class="cit-lpage">1210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3907305%26rft_id%253Dinfo%253Apmid%252F16109973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE1LzkvMTE5OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTAvMTc1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.29" data-doi="10.1007/s00122-005-0041-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierce</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waghmare</span>  <span class="cit-name-given-names">V. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">b</span>â<span class="cit-article-title">Genetic mapping and comparative analysis of seven mutants related to seed fiber development in cotton.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">1137</span>â<span class="cit-lpage">1146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DRong%26rft.auinit1%253DJ.%26rft.volume%253D111%26rft.issue%253D6%26rft.spage%253D1137%26rft.epage%253D1146%26rft.atitle%253DGenetic%2Bmapping%2Band%2Bcomparative%2Banalysis%2Bof%2Bseven%2Bmutants%2Brelated%2Bto%2Bseed%2Bfiber%2Bdevelopment%2Bin%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0041-0%26rft_id%253Dinfo%253Apmid%252F16075204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0041-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16075204&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.30" data-doi="10.1534/genetics.107.074518"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feltus</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waghmare</span>  <span class="cit-name-given-names">V. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierce</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chee</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Meta-analysis of polyploid cotton QTL shows unequal contributions of subgenomes to a complex network of genes and gene clusters implicated in lint fiber development.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">2577</span>â<span class="cit-lpage">2588</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074518%26rft_id%253Dinfo%253Apmid%252F17565937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni80LzI1NzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE3NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.31" data-doi="10.1007/s001220050952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shappley</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meredith</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jack</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span><span class="cit-comment">a</span>â<span class="cit-article-title">An RFLP linkage map of upland cotton (<em>Gossypium hirsutum</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">756</span>â<span class="cit-lpage">761</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D97%26rft.spage%253D756%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050952&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000076924400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shappley</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarty</span>  <span class="cit-name-given-names">J. C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span><span class="cit-comment">b</span>â<span class="cit-article-title">Quantita- D-subgenome bias of <em>Xcm</em> resistance genes in tetraploid gossypium tive trait loci associated with agronomic and fiber traits of Upland (cotton) suggests that polyploid formation has created novel avecotton.</span> <abbr class="cit-jnl-abbrev">J. Cotton Sci.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">153</span>â<span class="cit-lpage">163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCotton%2BSci.%26rft.volume%253D2%26rft.spage%253D153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.33" data-doi="10.1007/s11032-004-4731-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Molecular mapping of QTLs for fiber qualities in three diverse lines in upland cotton using SSR markers.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">169</span>â<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D15%26rft.spage%253D169%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-004-4731-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-004-4731-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.34" data-doi="10.1007/s10681-006-9338-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Q. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Genetic mapping of quantitative trait loci for fiber quality and yield trait by RIL approach in upland cotton.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">371</span>â<span class="cit-lpage">380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D155%26rft.spage%253D371%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-006-9338-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-006-9338-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000245825300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.35" data-doi="10.1007/s00122-006-0483-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">An ultradense genetic recombination map for <em>Brassica napus</em>, consisting of 13,551 SRAP markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">1305</span>â<span class="cit-lpage">1317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSun%26rft.auinit1%253DZ.%26rft.volume%253D114%26rft.issue%253D8%26rft.spage%253D1305%26rft.epage%253D1317%26rft.atitle%253DAn%2Bultradense%2Bgenetic%2Brecombination%2Bmap%2Bfor%2BBrassica%2Bnapus%252C%2Bconsisting%2Bof%2B13551%2BSRAP%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0483-z%26rft_id%253Dinfo%253Apmid%252F17426959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0483-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17426959&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ulloa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meredith</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Genetic linkage map and QTL analysis of agronomic and fiber quality traits in an intraspecific population.</span> <abbr class="cit-jnl-abbrev">J Cotton Sci</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">161</span>â<span class="cit-lpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BCotton%2BSci%26rft.volume%253D4%26rft.spage%253D161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.37" data-doi="10.1007/s001220100739"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ulloa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meredith</span>  <span class="cit-name-given-names">W. R.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shappley</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kahler</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">RFLP genetic linkage maps from four F(2.3) populations and a joinmap of <em>Gossypium hirsutum</em> L.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">200</span>â<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DUlloa%26rft.auinit1%253DM.%26rft.volume%253D104%26rft.issue%253D2-3%26rft.spage%253D200%26rft.epage%253D208%26rft.atitle%253DRFLP%2Bgenetic%2Blinkage%2Bmaps%2Bfrom%2Bfour%2BF%25282.3%2529%2Bpopulations%2Band%2Ba%2Bjoinmap%2Bof%2BGossypium%2Bhirsutum%2BL.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220100739%26rft_id%253Dinfo%253Apmid%252F12582687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220100739&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12582687&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174376400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">QTL mapping for fiber quality properties in cotton cultivar yumian 1.</span> <abbr class="cit-jnl-abbrev">Acta Agron. Sin.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1915</span>â<span class="cit-lpage">1921</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BAgron.%2BSin.%26rft.volume%253D33%26rft.spage%253D1915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.39" data-doi="10.1038/ng.2371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">The draft genome of a diploid cotton <em>Gossypium raimondii</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1098</span>â<span class="cit-lpage">1103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DK.%26rft.volume%253D44%26rft.issue%253D10%26rft.spage%253D1098%26rft.epage%253D1103%26rft.atitle%253DThe%2Bdraft%2Bgenome%2Bof%2Ba%2Bdiploid%2Bcotton%2BGossypium%2Braimondii.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2371%26rft_id%253Dinfo%253Apmid%252F22922876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22922876&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.40" data-doi="10.1186/1471-2105-7-447"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Statistical inference of chromosomal homology based on gene colinearity and applications to Arabidopsis and rice.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DWang%26rft.auinit1%253DX.%26rft.volume%253D7%26rft.spage%253D447%26rft.epage%253D447%26rft.atitle%253DStatistical%2Binference%2Bof%2Bchromosomal%2Bhomology%2Bbased%2Bon%2Bgene%2Bcolinearity%2Band%2Bapplications%2Bto%2BArabidopsis%2Band%2Brice.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-447%26rft_id%253Dinfo%253Apmid%252F17038171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-447&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17038171&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.41" data-doi="10.1073/pnas.1005931107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Parent-independent genotyping for constructing an ultrahigh-density linkage map based on population sequencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">10578</span>â<span class="cit-lpage">10583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1005931107%26rft_id%253Dinfo%253Apmid%252F20498060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzIzLzEwNTc4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMC8xNzU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.42" data-doi="10.1186/1471-2164-9-108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">An integrated genetic and physical map of homoeologous chromosomes 12 and 26 in Upland cotton (<em>G. hirsutum</em> L.).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DXu%26rft.auinit1%253DZ.%26rft.volume%253D9%26rft.spage%253D108%26rft.epage%253D108%26rft.atitle%253DAn%2Bintegrated%2Bgenetic%2Band%2Bphysical%2Bmap%2Bof%2Bhomoeologous%2Bchromosomes%2B12%2Band%2B26%2Bin%2BUpland%2Bcotton%2B%2528G.%2Bhirsutum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-108%26rft_id%253Dinfo%253Apmid%252F18307816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18307816&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.43" data-doi="10.1111/j.1744-7909.2007.00459.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">S. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">High-density linkage map of cultivated allotetraploid cotton based on SSR, TRAP, SRAP and AFLP Markers.</span> <abbr class="cit-jnl-abbrev">J. Integr. Plant Biol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">716</span>â<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BIntegr.%2BPlant%2BBiol.%26rft.volume%253D49%26rft.spage%253D716%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7909.2007.00459.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7909.2007.00459.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.44" data-doi="10.1016/j.ygeno.2010.02.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">The construction of a tetraploid cotton genome wide comprehensive reference map.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">230</span>â<span class="cit-lpage">240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D95%26rft.issue%253D4%26rft.spage%253D230%26rft.epage%253D240%26rft.atitle%253DThe%2Bconstruction%2Bof%2Ba%2Btetraploid%2Bcotton%2Bgenome%2Bwide%2Bcomprehensive%2Breference%2Bmap.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2010.02.001%26rft_id%253Dinfo%253Apmid%252F20171271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2010.02.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20171271&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.45" data-doi="10.1186/1471-2164-12-15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Genome structure of cotton revealed by a genome-wide SSR genetic map constructed from a BC1 population between <em>Gossypium hirsutum</em> and <em>G. barbadense</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DYu%26rft.auinit1%253DY.%26rft.volume%253D12%26rft.spage%253D15%26rft.epage%253D15%26rft.atitle%253DGenome%2Bstructure%2Bof%2Bcotton%2Brevealed%2Bby%2Ba%2Bgenome-wide%2BSSR%2Bgenetic%2Bmap%2Bconstructed%2Bfrom%2Ba%2BBC1%2Bpopulation%2Bbetween%2Bgossypium%2Bhirsutum%2Band%2BG.%2Bbarbadense.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-15%26rft_id%253Dinfo%253Apmid%252F21214949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21214949&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.46" data-doi="10.1104/pp.112.206870"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">CSGRqtl, a comparative quantitative trait locus database for Saccharinae grasses.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">594</span>â<span class="cit-lpage">599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.112.206870%26rft_id%253Dinfo%253Apmid%252F23370713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjEvMi81OTQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE3NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chee</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Recent advances in cotton genomics.</span> <abbr class="cit-jnl-abbrev">Int. J. Plant Genomics</abbr> <span class="cit-vol">2008</span>: <span class="cit-fpage">742304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DInt%2BJ%2BPlant%2BGenomics%26rft.aulast%253DZhang%26rft.auinit1%253DH.%2BB.%26rft.volume%253D2008%26rft.spage%253D742304%26rft.epage%253D742304%26rft.atitle%253DRecent%2Badvances%2Bin%2Bcotton%2Bgenomics.%26rft_id%253Dinfo%253Apmid%252F18288253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18288253&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.48" data-doi="10.1007/s00122-002-1100-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Molecular linkage map of allotetraploid cotton (<em>Gossypium hirsutum</em> L. x <em>Gossypium barbadense</em> L.) with a haploid population.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">1166</span>â<span class="cit-lpage">1174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DJ.%26rft.volume%253D105%26rft.issue%253D8%26rft.spage%253D1166%26rft.epage%253D1174%26rft.atitle%253DMolecular%2Blinkage%2Bmap%2Bof%2Ballotetraploid%2Bcotton%2B%2528%2BGossypium%2Bhirsutum%2BL.%2Bx%2BGossypium%2Bbarbadense%2BL.%2529%2Bwith%2Ba%2Bhaploid%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-002-1100-4%26rft_id%253Dinfo%253Apmid%252F12582895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-002-1100-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12582895&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1759.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179982700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1759.49" data-doi="10.1016/S1671-2927(08)60161-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">IT-ISJ (Intron-targeted intron-exon splice conjunction) marker and its application in construction of upland cotton linkage map.</span> <abbr class="cit-jnl-abbrev">Agric. Sci. China</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1172</span>â<span class="cit-lpage">1180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgric.%2BSci.%2BChina%26rft.volume%253D7%26rft.spage%253D1172%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1671-2927%252808%252960161-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1671-2927(08)60161-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/10/1759.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/10/1753" title="Characterization of Two ENU-Induced Mutations Affecting Mouse Skeletal Morphology" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/10/1769" title="Disruption of the Rice Plastid Ribosomal Protein S20 Leads to Chloroplast Developmental Defects and Seedling Lethality" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/10"><span>Volume 3 Issue 10, October 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/10.cover-source.jpg?itok=h5LtrAoT" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/10/1759.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OYWcmPAYNkmGPNJ5pEBRqHkvztqd0paNprQFK0E6t38" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/10/1759" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/10/1759" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/10/1759" class="active" data-icon-position="" data-hide-link-title="0">A Whole-Genome DNA Marker Map for Cotton Based on the D-Genome Sequence of Gossypium raimondii L.</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459560" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-bFw5vKYaNEuhWVjW8itIXHGIaVlgx0Mg4YHtmUy37BE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/10/1759.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/10/1759" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-kTqW6L17wfbOJ5QKPWP6PvolmTqQLN0xpbQ30BXk2Uo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/10/1759" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/10/1759.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459560" id="node459560--2" data-pisa="ggg;3/10/1759" data-pisa-master="ggg;g3.113.006890" data-apath="/ggg/3/10/1759.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Whole-Genome DNA Marker Map for Cotton Based on the D-Genome Sequence of <em>Gossypium raimondii</em> L.</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zining Wang</span>, <span class="highwire-citation-author" data-delta="1">Dong Zhang</span>, <span class="highwire-citation-author" data-delta="2">Xiyin Wang</span>, <span class="highwire-citation-author" data-delta="3">Xu Tan</span>, <span class="highwire-citation-author" data-delta="4">Hui Guo</span> and <span class="highwire-citation-author" data-delta="5">Andrew H. Paterson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1759-1767;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006890 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459560--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zining Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zining%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZining%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dong Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dong%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADong%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xiyin Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiyin%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiyin%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Xu Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xu%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXu%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hui Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hui%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHui%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Andrew H. Paterson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BH.%2BPaterson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paterson%20AH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BH.%2BPaterson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:paterson@plantbio.uga.edu" class="" data-icon-position="" data-hide-link-title="0">paterson@plantbio.uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/10/1759" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/10/1759&amp;title=A%20Whole-Genome%20DNA%20Marker%20Map%20for%20Cotton%20Based%20on%20the%20D-Genome%20Sequence%20of%20Gossypium%20raimondii%20L." id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/10/1759&amp;title=A%20Whole-Genome%20DNA%20Marker%20Map%20for%20Cotton%20Based%20on%20the%20D-Genome%20Sequence%20of%20Gossypium%20raimondii%20L." id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/10/1759&amp;title=A%20Whole-Genome%20DNA%20Marker%20Map%20for%20Cotton%20Based%20on%20the%20D-Genome%20Sequence%20of%20Gossypium%20raimondii%20L." id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/10/1759&amp;text=A%20Whole-Genome%20DNA%20Marker%20Map%20for%20Cotton%20Based%20on%20the%20D-Genome%20Sequence%20of%20Gossypium%20raimondii%20L." id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/10/1759&amp;title=A%20Whole-Genome%20DNA%20Marker%20Map%20for%20Cotton%20Based%20on%20the%20D-Genome%20Sequence%20of%20Gossypium%20raimondii%20L." id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/10/1759&amp;t=A%20Whole-Genome%20DNA%20Marker%20Map%20for%20Cotton%20Based%20on%20the%20D-Genome%20Sequence%20of%20Gossypium%20raimondii%20L." id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/10/1759&amp;title=A%20Whole-Genome%20DNA%20Marker%20Map%20for%20Cotton%20Based%20on%20the%20D-Genome%20Sequence%20of%20Gossypium%20raimondii%20L." id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/10/1759&amp;title=A%20Whole-Genome%20DNA%20Marker%20Map%20for%20Cotton%20Based%20on%20the%20D-Genome%20Sequence%20of%20Gossypium%20raimondii%20L." id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459560/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459560" id="node459560--3" data-pisa="ggg;3/10/1759" data-pisa-master="ggg;g3.113.006890" data-apath="/ggg/3/10/1759.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Whole-Genome DNA Marker Map for Cotton Based on the D-Genome Sequence of <em>Gossypium raimondii</em> L.</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zining Wang</span>, <span class="highwire-citation-author" data-delta="1">Dong Zhang</span>, <span class="highwire-citation-author" data-delta="2">Xiyin Wang</span>, <span class="highwire-citation-author" data-delta="3">Xu Tan</span>, <span class="highwire-citation-author" data-delta="4">Hui Guo</span> and <span class="highwire-citation-author" data-delta="5">Andrew H. Paterson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1759-1767;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006890 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459560--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zining Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zining%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZining%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dong Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dong%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADong%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xiyin Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiyin%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiyin%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Xu Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xu%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXu%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hui Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hui%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHui%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Andrew H. Paterson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30605</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BH.%2BPaterson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paterson%20AH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BH.%2BPaterson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:paterson@plantbio.uga.edu" class="" data-icon-position="" data-hide-link-title="0">paterson@plantbio.uga.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1759&amp;rft.epage=1767&amp;rft.atitle=A%20Whole-Genome%20DNA%20Marker%20Map%20for%20Cotton%20Based%20on%20the%20D-Genome%20Sequence%20of%20Gossypium%20raimondii%20L.&amp;rft.volume=3&amp;rft.issue=10&amp;rft.date=2013-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wang%2C+Zining&amp;rft.au=Zhang%2C+Dong&amp;rft.au=Wang%2C+Xiyin&amp;rft.au=Tan%2C+Xu&amp;rft.au=Guo%2C+Hui&amp;rft.au=Paterson%2C+Andrew+H."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459560/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459560/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459560/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459560/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459560/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459560/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459560/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459560/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459560/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459560/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459560/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459560/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/10/1759" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/10/1759.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/10/1759.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/10/1759.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/10/1759.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
