<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/10/1795" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/10/1795.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/10/1795.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/10/1795.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/10/1795/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/10/1795.atom" />
<meta name="HW.pisa" content="ggg;3/10/1795" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Imputation-Based Genomic Coverage Assessments of Current Human Genotyping Arrays" />
<meta name="DC.Identifier" content="10.1534/g3.113.007161" />
<meta name="DC.Date" content="2013-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Nelson et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Microarray single-nucleotide polymorphism genotyping, combined with imputation of untyped variants, has been widely adopted as an efficient means to interrogate variation across the human genome. “Genomic coverage” is the total proportion of genomic variation captured by an array, either by direct observation or through an indirect means such as linkage disequilibrium or imputation. We have performed imputation-based genomic coverage assessments of eight current genotyping arrays that assay from ~0.3 to ~5 million variants. Coverage was determined separately in each of the four continental ancestry groups in the 1000 Genomes Project phase 1 release. We used the subset of 1000 Genomes variants present on each array to impute the remaining variants and assessed coverage based on correlation between imputed and observed allelic dosages. More than 75% of common variants (minor allele frequency &gt; 0.05) are covered by all arrays in all groups except for African ancestry, and up to ~90% in all ancestries for the highest density arrays. In contrast, less than 40% of less common variants (0.01 &lt; minor allele frequency &lt; 0.05) are covered by low density arrays in all ancestries and 50–80% in high density arrays, depending on ancestry. We also calculated genome-wide power to detect variant-trait association in a case-control design, across varying sample sizes, effect sizes, and minor allele frequency ranges, and compare these array-based power estimates with a hypothetical array that would type all variants in 1000 Genomes. These imputation-based genomic coverage and power analyses are intended as a practical guide to researchers planning genetic studies." />
<meta name="DC.Contributor" content="Sarah C. Nelson" />
<meta name="DC.Contributor" content="Kimberly F. Doheny" />
<meta name="DC.Contributor" content="Elizabeth W. Pugh" />
<meta name="DC.Contributor" content="Jane M. Romm" />
<meta name="DC.Contributor" content="Hua Ling" />
<meta name="DC.Contributor" content="Cecelia A. Laurie" />
<meta name="DC.Contributor" content="Sharon R. Browning" />
<meta name="DC.Contributor" content="Bruce S. Weir" />
<meta name="DC.Contributor" content="Cathy C. Laurie" />
<meta name="article:published_time" content="2013-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Imputation-Based Genomic Coverage Assessments of Current Human Genotyping Arrays" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Microarray single-nucleotide polymorphism genotyping, combined with imputation of untyped variants, has been widely adopted as an efficient means to interrogate variation across the human genome. “Genomic coverage” is the total proportion of genomic variation captured by an array, either by direct observation or through an indirect means such as linkage disequilibrium or imputation. We have performed imputation-based genomic coverage assessments of eight current genotyping arrays that assay from ~0.3 to ~5 million variants. Coverage was determined separately in each of the four continental ancestry groups in the 1000 Genomes Project phase 1 release. We used the subset of 1000 Genomes variants present on each array to impute the remaining variants and assessed coverage based on correlation between imputed and observed allelic dosages. More than 75% of common variants (minor allele frequency &amp;gt; 0.05) are covered by all arrays in all groups except for African ancestry, and up to ~90% in all ancestries for the highest density arrays. In contrast, less than 40% of less common variants (0.01 &amp;lt; minor allele frequency &amp;lt; 0.05) are covered by low density arrays in all ancestries and 50–80% in high density arrays, depending on ancestry. We also calculated genome-wide power to detect variant-trait association in a case-control design, across varying sample sizes, effect sizes, and minor allele frequency ranges, and compare these array-based power estimates with a hypothetical array that would type all variants in 1000 Genomes. These imputation-based genomic coverage and power analyses are intended as a practical guide to researchers planning genetic studies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/10/01" />
<meta name="citation_mjid" content="ggg;3/10/1795" />
<meta name="citation_id" content="3/10/1795" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/10/1795" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/10/1795.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/10/1795.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/10/1795.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.007161" />
<meta name="citation_pmid" content="23979933" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1795" />
<meta name="citation_lastpage" content="1807" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sarah C. Nelson" />
<meta name="citation_author_institution" content="Department of Biostatistics, University of Washington, Seattle, Washington, 98195" />
<meta name="citation_author_email" content="sarahcn@uw.edu" />
<meta name="citation_author" content="Kimberly F. Doheny" />
<meta name="citation_author_institution" content="Center for Inherited Disease Research, Johns Hopkins University School of Medicine, Baltimore, Maryland, 21224" />
<meta name="citation_author" content="Elizabeth W. Pugh" />
<meta name="citation_author_institution" content="Center for Inherited Disease Research, Johns Hopkins University School of Medicine, Baltimore, Maryland, 21224" />
<meta name="citation_author" content="Jane M. Romm" />
<meta name="citation_author_institution" content="Center for Inherited Disease Research, Johns Hopkins University School of Medicine, Baltimore, Maryland, 21224" />
<meta name="citation_author" content="Hua Ling" />
<meta name="citation_author_institution" content="Center for Inherited Disease Research, Johns Hopkins University School of Medicine, Baltimore, Maryland, 21224" />
<meta name="citation_author" content="Cecelia A. Laurie" />
<meta name="citation_author_institution" content="Department of Biostatistics, University of Washington, Seattle, Washington, 98195" />
<meta name="citation_author" content="Sharon R. Browning" />
<meta name="citation_author_institution" content="Department of Biostatistics, University of Washington, Seattle, Washington, 98195" />
<meta name="citation_author" content="Bruce S. Weir" />
<meta name="citation_author_institution" content="Department of Biostatistics, University of Washington, Seattle, Washington, 98195" />
<meta name="citation_author" content="Cathy C. Laurie" />
<meta name="citation_author_institution" content="Department of Biostatistics, University of Washington, Seattle, Washington, 98195" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. R. Abecasis;citation_author=A. Auton;citation_author=L. D. Brooks;citation_author=M. A. DePristo;citation_author=R. M. Durbin;citation_title=An integrated map of genetic variation from 1,092 human genomes.;citation_pages=56-65;citation_volume=491;citation_year=2012;citation_issue=7422;citation_pmid=23128226;citation_doi=10.1038/nature11632" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. C. Barrett;citation_author=L. R. Cardon;citation_title=Evaluating coverage of genome-wide association studies.;citation_pages=659-662;citation_volume=38;citation_year=2006;citation_issue=6;citation_pmid=16715099;citation_doi=10.1038/ng1801" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Haplotype phasing: existing methods and new developments.;citation_pages=703-714;citation_volume=12;citation_year=2011;citation_issue=10;citation_pmid=21921926;citation_doi=10.1038/nrg3054" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=C. S. Carlson;citation_author=M. A. Eberle;citation_author=M. J. Rieder;citation_author=Q. Yi;citation_author=L. Kruglyak;citation_title=Selecting a maximally informative set of single-nucleotide polymorphisms for association analyses using linkage disequilibrium.;citation_pages=106-120;citation_volume=74;citation_year=2004;citation_issue=1;citation_pmid=14681826;citation_doi=10.1086/381000" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=O. Delaneau;citation_author=J. F. Zagury;citation_author=J. Marchini;citation_title=Improved whole-chromosome phasing for disease and population genetic studies.;citation_pages=5-6;citation_volume=10;citation_year=2013;citation_pmid=23269371;citation_doi=10.1038/nchembio.1414" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=E. Evangelou;citation_author=J. P. Ioannidis;citation_title=Meta-analysis methods for genome-wide association studies and beyond.;citation_pages=379-389;citation_volume=14;citation_year=2013;citation_pmid=23657481;citation_doi=10.1038/nrg3472" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Frazer;citation_author=D. Ballinger;citation_author=D. Cox;citation_author=D. Hinds;citation_author=L. Stuve;citation_title=A second generation human haplotype map of over 3.1 million SNPs.;citation_pages=851-861;citation_volume=449;citation_year=2007;citation_issue=7164;citation_pmid=17943122;citation_doi=10.1038/nature06258" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=T. J. Hoffmann;citation_author=M. N. Kvale;citation_author=S. E. Hesselson;citation_author=Y. Zhan;citation_author=C. Aquino;citation_title=Next generation genome-wide association tool: design and coverage of a high-throughput European-optimized SNP array.;citation_pages=79-89;citation_volume=98;citation_year=2011;citation_issue=2;citation_pmid=21565264;citation_doi=10.1016/j.ygeno.2011.04.005" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=T. J. Hoffmann;citation_author=Y. Zhan;citation_author=M. N. Kvale;citation_author=S. E. Hesselson;citation_author=J. Gollub;citation_title=Design and coverage of high throughput genotyping arrays optimized for individuals of East Asian, African American, and Latino race/ethnicity using imputation and a novel hybrid SNP selection algorithm.;citation_pages=422-430;citation_volume=98;citation_year=2011;citation_issue=6;citation_pmid=21903159;citation_doi=10.1016/j.ygeno.2011.08.007" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=B. Howie;citation_author=J. Marchini;citation_author=M. Stephens;citation_title=Genotype Imputation with Thousands of Genomes;citation_pages=457-470;citation_volume=1;citation_year=2011;citation_issue=6;citation_pmid=22384356;citation_doi=10.1534/g3.111.001198" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. Howie;citation_author=C. Fuchsberger;citation_author=M. Stephens;citation_author=J. Marchini;citation_author=G. R. Abecasis;citation_title=Fast and accurate genotype imputation in genome-wide association studies through pre-phasing.;citation_pages=955-959;citation_volume=44;citation_year=2012;citation_issue=8;citation_pmid=22820512;citation_doi=10.1038/ng.2354" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=E. Jorgenson;citation_author=J. S. Witte;citation_title=Coverage and power in genomewide association studies.;citation_pages=884-888;citation_volume=78;citation_year=2006;citation_issue=5;citation_pmid=16642443;citation_doi=10.1086/503751" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=B. Li;citation_author=S. M. Leal;citation_title=Methods for detecting associations with rare variants for common diseases: application to analysis of sequence data.;citation_pages=311-321;citation_volume=83;citation_year=2008;citation_issue=3;citation_pmid=18691683;citation_doi=10.1016/j.ajhg.2008.06.024" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=C. Li;citation_author=M. Li;citation_author=J. R. Long;citation_author=Q. Cai;citation_author=W. Zheng;citation_title=Evaluating cost efficiency of SNP chips in genome-wide association studies.;citation_pages=387-395;citation_volume=32;citation_year=2008;citation_issue=5;citation_pmid=18271056;citation_doi=10.1002/gepi.20312" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=P. Lin;citation_author=S. M. Hartz;citation_author=Z. Zhang;citation_author=S. F. Saccone;citation_author=J. Wang;citation_title=A new statistic to evaluate imputation reliability.;citation_pages=e9697-e9697;citation_volume=5;citation_year=2010;citation_issue=3;citation_pmid=20300623;citation_doi=10.1371/journal.pone.0009697" />
<meta name="citation_reference" content="citation_journal_title=Genet. Epidemiol.;citation_author=K. J. Lindquist;citation_author=E. Jorgenson;citation_author=T. J. Hoffmann;citation_author=J. S. Witte;citation_title=The impact of improved microarray coverage and larger sample sizes on future genome-wide association studies.;citation_pages=383-392;citation_volume=37;citation_year=2013;citation_pmid=23529720;citation_doi=10.1002/gepi.21724" />
<meta name="citation_reference" content="citation_journal_title=New England Journal of Medicine;citation_journal_abbrev=NEJM;citation_author=T. A. Manolio;citation_title=Genomewide association studies and assessment of the risk of disease.;citation_pages=166-176;citation_volume=363;citation_year=2010;citation_issue=2;citation_pmid=20647212;citation_doi=10.1056/NEJMra0905980" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Marchini;citation_author=B. Howie;citation_title=Genotype imputation for genome-wide association studies.;citation_pages=499-511;citation_volume=11;citation_year=2010;citation_issue=7;citation_pmid=20517342;citation_doi=10.1038/nrg2796" />
<meta name="citation_reference" content="citation_journal_title=Human heredity;citation_journal_abbrev=Hum Hered;citation_author=S. Powers;citation_author=S. Gopalakrishnan;citation_author=N. Tintle;citation_title=Assessing the impact of non-differential genotyping errors on rare variant tests of association.;citation_pages=153-160;citation_volume=72;citation_year=2011;citation_issue=3;citation_pmid=22004945;citation_doi=10.1159/000332222" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. K. Pritchard;citation_author=M. Przeworski;citation_title=Linkage disequilibrium in humans: models and data.;citation_pages=1-14;citation_volume=69;citation_year=2001;citation_issue=1;citation_pmid=11410837;citation_doi=10.1086/321275" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. A. Tennessen;citation_author=A. W. Bigham;citation_author=T. D. O’Connor;citation_author=W. Fu;citation_author=E. E. Kenny;citation_title=Evolution and functional impact of rare coding variation from deep sequencing of human exomes.;citation_volume=337;citation_year=2012;citation_pmid=22604720;citation_doi=10.1126/science.1219240" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. M. Visscher;citation_author=M. A. Brown;citation_author=M. I. McCarthy;citation_author=J. Yang;citation_title=Five years of GWAS discovery.;citation_pages=7-24;citation_volume=90;citation_year=2012;citation_issue=1;citation_pmid=22243964;citation_doi=10.1016/j.ajhg.2011.11.029" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. D. Wall;citation_author=M. P. Cox;citation_author=F. L. Mendez;citation_author=A. Woerner;citation_author=T. Severson;citation_title=A novel DNA sequence database for analyzing human demographic history.;citation_volume=18;citation_year=2008;citation_pmid=18493019;citation_doi=10.1101/gr.075630.107" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. D. Wall;citation_author=R. Jiang;citation_author=C. Gignoux;citation_author=G. K. Chen;citation_author=C. Eng;citation_title=Genetic variation in Native Americans, inferred from Latino SNP and resequencing data.;citation_volume=28;citation_year=2011;citation_pmid=21368315;citation_doi=10.1093/molbev/msr049" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Imputation-Based Genomic Coverage Assessments of Current Human Genotyping Arrays" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10.cover-source.jpg" />
<meta name="twitter:description" content="Microarray single-nucleotide polymorphism genotyping, combined with imputation of untyped variants, has been widely adopted as an efficient means to interrogate variation across the human genome. “Genomic coverage” is the total proportion of genomic variation captured by an array, either by direct observation or through an indirect means such as linkage disequilibrium or imputation. We have performed imputation-based genomic coverage assessments of eight current genotyping arrays that assay from ~0.3 to ~5 million variants. Coverage was determined separately in each of the four continental ancestry groups in the 1000 Genomes Project phase 1 release. We used the subset of 1000 Genomes variants present on each array to impute the remaining variants and assessed coverage based on correlation between imputed and observed allelic dosages. More than 75% of common variants (minor allele frequency &gt; 0.05) are covered by all arrays in all groups except for African ancestry, and up to ~90% in all ancestries for the highest density arrays. In contrast, less than 40% of less common variants (0.01 &lt; minor allele frequency &lt; 0.05) are covered by low density arrays in all ancestries and 50–80% in high density arrays, depending on ancestry. We also calculated genome-wide power to detect variant-trait association in a case-control design, across varying sample sizes, effect sizes, and minor allele frequency ranges, and compare these array-based power estimates with a hypothetical array that would type all variants in 1000 Genomes. These imputation-based genomic coverage and power analyses are intended as a practical guide to researchers planning genetic studies." />
<meta name="og-title" property="og:title" content="Imputation-Based Genomic Coverage Assessments of Current Human Genotyping Arrays" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/10/1795" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Microarray single-nucleotide polymorphism genotyping, combined with imputation of untyped variants, has been widely adopted as an efficient means to interrogate variation across the human genome. “Genomic coverage” is the total proportion of genomic variation captured by an array, either by direct observation or through an indirect means such as linkage disequilibrium or imputation. We have performed imputation-based genomic coverage assessments of eight current genotyping arrays that assay from ~0.3 to ~5 million variants. Coverage was determined separately in each of the four continental ancestry groups in the 1000 Genomes Project phase 1 release. We used the subset of 1000 Genomes variants present on each array to impute the remaining variants and assessed coverage based on correlation between imputed and observed allelic dosages. More than 75% of common variants (minor allele frequency &gt; 0.05) are covered by all arrays in all groups except for African ancestry, and up to ~90% in all ancestries for the highest density arrays. In contrast, less than 40% of less common variants (0.01 &lt; minor allele frequency &lt; 0.05) are covered by low density arrays in all ancestries and 50–80% in high density arrays, depending on ancestry. We also calculated genome-wide power to detect variant-trait association in a case-control design, across varying sample sizes, effect sizes, and minor allele frequency ranges, and compare these array-based power estimates with a hypothetical array that would type all variants in 1000 Genomes. These imputation-based genomic coverage and power analyses are intended as a practical guide to researchers planning genetic studies." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10.cover-source.jpg" />
    <title>Imputation-Based Genomic Coverage Assessments of Current Human Genotyping Arrays | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Xy-IIfJGUnq-irPCnIvJI3dfSeQun6nL71TqROMo6Hs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459551","apath":"\/ggg\/3\/10\/1795.atom","pisa":"ggg;3\/10\/1795","ac":{"\/ggg\/3\/10\/1795.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/10\/1795.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/10\/1795"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/10\/1795"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/10\/1795.full":true,"\/content\/3\/10\/1795":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gdKEm13rGVccVMC0kelU3mDIr31PjRVolx7mJXbEL\\\\\\\/k\\\\\\\/JJlHaQgHbqYCvFsRFXTF3NabfbLn3\\\\\\\/cL2RppNpfnZH4uSeFUB8X1nCZag+vmC4W1mCO35cooisyd5ZBE6FHii+MM0Fg5XXMVOIzsC1mSuRWxMRePj2kPpHNLGKnMgfi9B+uTyaG12MIw2XGCnUNgHQoxTLliIr4lU6JTLD8pMj2nRNanPUcFlvk5jfJ0N87N4ao4jO9aQzdjIkwRpCpmWBacSRZdOhLaXKGSkIsjQhbuF46XKeZbmMl2HFxZqyb6dCC\\\\\\\/WqittqiF+Q3ZTvwqExDNPTrCwp2FNW2RVDkx\\\\\\\/v\\\\\\\/k5dbM42id2GZ2nga50cec1qm\\\\\\\/Kq+m4SYbkiDYYyx8Et6qYB4fOQnMjvZ3WhN9OD+ev81t4kTMfO4fmF0XeAjLOFxYo1wNF2J+XbYQRYXXLq0AtvUoFwEi7sZ8r2J24z8CFeZkC0obF+15af1HUz9G9093GG1hfLp539x2ESTMVCaiiwyhtlhvfV5zcfqyR5s+AVMBZbEpZIL63Hju3xsFiJG2t9bq2DRQSpcn30AoafugNP9zatTC7k\\\\\\\/Tq8v9mePyJaDdZMDnJPeWjUbvE6Ej2ULVNibnsFfMsdJIhKTjwG5V716B\\\\\\\/tpg8ivpk99xBgp17aGUhJr3apN74Zds9JSOJlnudsj\\\\\\\/tBQBHtCfzMKcNNtr1Q+pD0VHDxiYTIPS60PYBJdjD19z2cq\\\\\\\/5JaGHwZCSgF+yQANBJp5oGLIzvKnPolYyJWGDrUjHb3YiWI1JseLIcRKMrL0n8th\\\\\\\/OlBo4h\\\\\\\/U+0zF6kKnRCSHva7LCOTozN0A5pPDdDmWt5IzZBtuV+J5W8u0451Bxeu5gnhZj+yFX6yh4A\\\\\\\/u6ij9i35\\\\\\\/0xtvyxs9RrdDiv5LwzxCIJ6fkPa4KwbKQXjp8LHl8Hew3MIxFtYp9KRoMwJLKB1z\\\\\\\/xbA7KLr3Uzh1y7kGfiTecJLa6Lu5GlIufN6tOs+GvufreyF677H6x5N0K9EnlhqKMoxgJV7UtpNzdHRHaUN+5XgwJH3jTu3zxHc9hFgSZdjOCSA2adk3I6cNlypCSlBZUMO+qriwx7nuZEYErt77Cfu6l2PGnzZX68vWAXNTx8v\\\\\\\/y8LO6vnaHpKG6EkNCx2lFQ6AiA7UbTf16T6wdux6wsdiPExCxEFn78B6RASzyjMvQtNG1Tag57pdFIXOzxacb3ueCKy4dLVcD7yYg\\\\\\\/cfITx4eEAGWMpDp5Rvzlar9\\\\\\\/HkrFX4CtIBq7gZArfi7mn7wq\\\u0022,\\\u0022iv\\\u0022:\\\u0022n12HrQCLXYOfCYajNZ86IQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002277ab07a6c742670cdcb9ce53ea2c27d597c4c4ac5a7219c7f8faf184371c41a8\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PXuReplUEvKXRGuA2Km4Kxs95K8vUBfUHG3lSsd6LiIyOIfPpd2Cb7r5D\\\\\\\/BPjO6vM\\\\\\\/WmiHim\\\\\\\/7mkD9JZXJpGn0cY45RwVLVEfRmhtNfJgDU33CdmdRsKhPoJNZLB5RaTR8DV8\\\\\\\/So7aFSShrmxIZRMAmqBYe53fVUZbRU0\\\\\\\/i2ucs4jLiafF5bvfN4xT\\\\\\\/LceBkM5dwWtW8\\\\\\\/+wwfXU6lBKUzzmsE05JESHRX5qlup7Oihp1v0pdBHQJqsOsUY1oNwzXfUOAcXbYPT3U0+iC1gl4Eb6gUKIJxEBMfL4RWIxDFfgeUIGoea\\\\\\\/BeSiAtUOw\\\\\\\/jGuUYYT+m7jWPiasevP+hr1sxcpL56tkx\\\\\\\/PQsk191xUrJE\\\\\\\/NlYPwVU9yZStagKg5fviGBnzwj4QJgGjT+gN7+N0rp9k0hMjWzx0gQPtZpdBBvJ3Xl9i1NplRrqhA80u2vtlwW24Yi5LTw6nfw2GWIp9xs+qoPJm6cx3g\\\\\\\/fNnNr\\\\\\\/gYjxPRg4Zb0+5n1xHhCL9ssdKspuJGlwuN0GMMRGhcld+dNFXx8OW+HABYblCzL14T7\\\\\\\/mm9nlVU3x5Ac1selCJCrXstLQFIpXgwjgxLh7TrYCidCWIWNllQicH4GYKJ0FhjjfHflvhiFujg8VpGlBMOiTotEDQ9xXiJ900d1PqBwk6d1n5aphyFdyVvA6HhFoZbaSCKdAsNH0QTkKT44s8EoQRmlLGzN\\\\\\\/ybDybje0XUpQKp05T1yPkIewaiHzmKu130Z4Y3fHMhsq88KIxx4v0bFPZQZZFBYWbAxHXC\\\\\\\/g0FDXJM\\\\\\\/5gwxrHkYMeTlb4GTX\\\\\\\/t2JEgNcqhz1bwTLWveYLvOBwmmRoUHNjGdYRKgsU5Bb53sh4CQWjX5IgJ7BA+jg3m0xAY0XpUfHm5HF8tqrp50PYSP7PeMhFTW4Prb4PPVLBiFrJ6EbdeB7B3cfD97qfCcZgD0zLTUb1HU6OuJanLyG65Cu\\\\\\\/l6WmP1l48LdFJko9b8YdTFVu+ekIyig25CvemWx2Prdh41m8aOAxxYXbvlYAEh7WoIWlnzaB3pZEQ6\\\\\\\/qVt4da8ja5t1w3eQxqBB4kyx8DRnPzTkv0j1i0niuQflnwarXoggRGZnzIqZVuWnS1woJOBeQB7mOugGrgAxyHz658T+jP4ilcyoe2cRbSLvy8j30SZpXbElLGiFmV7ugxIfzdcUWEFRAOBE4txlHEUJcwFKFQfckZ\\\\\\\/ySG0BvXc\\\\\\\/HSqDtz\\\\\\\/te+JiGg\\\\\\\/9u4AIsqSqgHCFl1qoF6adcRdLkeR2NOCRgalDNBBsljkSSLmvAi015nW\\\\\\\/d5O55pIjnrQ7gLNXW53Pe3O8VaG9hHClSpYpdvTg65KuGN3hRUJG\\\\\\\/aZQqDj\\\\\\\/vxQcjnrDAeh\\\\\\\/caC8jP886rmBwMLYYZSgH8YkjjQsmB6ho04+fKOkH0CZD0hl43gE5yBOn+sCsAJbjp47tLdu32O0rqTZqaCKlCXRaTTLgVSbEYIlv9jQrNvLZ5WJr97X2BcCGRPEd2vz7Fupt7StGkQ+HSMbM5Q0qLJc9JUwSN4m\\\\\\\/YBVT3TxkMVlCSnRrvx0LiZ3rmngDRXwFoIkMiXXOW2FRXn2ZLV\\\\\\\/+OJ84PG\\\\\\\/OwLapo3puQrPHuFM2IAhcmrJuWDpI+mzMlGZj02LKlEjaZ+8vpNlr1UR1Puv7bSSdgCNFuK0oeVayMtG7GWZtshkikKtw+qNTX77bpBJGH5pbHsFPCnWrDHKBTKFwtkfBOp8imLrW77s9EIr6UnQsxJ1NI+yta3E64UnrKdxR5D99n7m6xoR7LEtFeGKrzdMS94Sz1rFrYa\\\\\\\/jNjmyaa7\\\\\\\/YZJKdzXYSLF0MQ7QDWAr+h0AhjfASAuBqJkA2Fo589XSGgIwlxeR1tsaJBP9Pp0V1Vc0nYIIcevgdsCE5OgUgjZyhLFy1Mv9KElE0+bkf3IcG69IKuG1U1D63YpdKvyPHYECtplwnbzpvfbjKupND7UcsDwWwvPwZoYbY5VmQCGfiERL49NNzRKIbPVab+2WEHEeyPHDGNjhVxaYYgvPccB87Jz5FTE54czH7RL09bX8v86Y9okUPjUEGX+368Aw+wBLJAtEbsJe+VP2HDfXot5eWoQH2GZSbef7UHPQkmddKQPRAXjjfwB4h5GQdkMjV+KqufnGR\\\\\\\/u+x6BR+5UaejSyOFyFGxR8PHS5YcbZUPyqNMh1GTidwESKB4MGt8Amf+pBa9U0gIwzaj7019V+izWL3YoY+VZor5SjSf5CTcfTPJEDHU8QTxpcRZ9YbJ9k056ZIdV3mygx0SO4LU8Qxt03eqn4rOAeRhz83K3qOHTpGdND7pnkeZ1cHltMa5YRdSQsqiefZXnfUfufkp2wVmvpC2gG4PuoOmNSBdisWYjs1TWUZpKOx43LljjEWZH+8CX5R19SkCtOxT5Hoz1qka2QsLbB4n594Lur0if7n5QQgbqH0L3lh6UDM5qAOVCvW3L1Ht+JKK9YbmV8XREHxlwNgIN3UHGSZ9S5Fy0trV6JKmZwv5qRKzhHCZmFsjv1+ZgRYC88+Uz4eDYHPntpZT3gS9oU\\\\\\\/UoeV6hQzYcle4+XHEeNJFxUfOmo+sRgT06gqsdaGmkRIRjCbUbnJcT2QdrGSzwthDYC8XxuxpyQ+bC\\\\\\\/sctahMm3Lc5h1ZHtT\\\\\\\/StvsoZ9Ucp30tBXHxepEvA9inLBW74Gk9qKZ9czpygM6jg8ogl1TXFeKkCFQd6lL9mJiRtLMyuIBEYZ+sIUrvNX\\\\\\\/GdUGKqxdoIB35KioZp3tVYkyp8oZfTmv34HwM59eeJxVn7XTSGrqL7NNn8fhnfwQqXwv\\\\\\\/K57dCiP5albi\\\\\\\/WQSekqpf4DIZLii9IaQRhDZ2A2SmIdZqPDxXuWwShW01IdegUKlkvQ3\\\\\\\/QBuQoAgQpPsqLbk0lax6mT7k9hwfJiHTPuoG5AEtihRO4Z2IE5NyPGsX0VNUVZFk060bXSok1b9FuHfJf8r7xgquWmtgCi0JR\\\\\\\/u+KWk7ux7Ey2L2YJ3wDmg\\\\\\\/ypz4T+7vArZGqGH\\\\\\\/DeZRMljT1w33SbNcAhscF2sE892MQo8wvW1R7zw7V9+e\\\\\\\/WJQdLDHiXkvjhfAHkFI\\\\\\\/XLVSo7Jmxc\\\\\\\/L5wDI2x+avIb\\\\\\\/PxWEptCV8gtc0\\\\\\\/0DGcBbapImKpY24ZtelYbVcSrU5Et3IOXAm6Hbddf4O11m14xAxKJbR9YNA9WtccH0K2JAgyctvZt\\\\\\\/hWIJV+sc10\\\\\\\/3NIlGe22kRgAFLlSZBXxg4aK3yP7u3h7yCo1hlvjO7uDo\\\\\\\/xoK5GGAziDTD4h39fV380yeVCOinQDAq3C+HlPp4yumOgcyymuHzLmi7Zt9ap1r65d1EVPw1vQj0uX5C9IjQ1g3N0DpFN8glmo7YERNBgpswYh0AOR\\\\\\\/Bvuf46SVnEQIgbb49D6xMUgnvNGrBTQVxpRo4E0i8ehcsMIDYL\\\\\\\/iRQNShNb53do2bwTA4NEn\\\\\\\/6EoNtlBl1fvOySXz2hQc+mXvgTBpTHHAN8eURjh\\\\\\\/AzYbDg9rSSAEttrvcG24NvVOedA+P20DPwEbWbfnEuw0ymLhnzmsX+FwwkUOg3fP3JGWSPqoySVsVTYRna76ZmlPmVJMXNpeTm0\\\\\\\/zMwYZYmw04zb4rnKiMo6fWz4Gi8LCPy9Fd\\\\\\\/9iI1bS94cbeen+fQpn6FeM2towZmx00X636xLw03ENiqWTzEVz9TqROaf37I+XRmpBrr4FtPcH1xsLCgeFeYmgzDWw12RqjvrjcG48L8ODh8GnX3sjqsHHnW\\\\\\\/ZAwoi3UGg3ULiM8Em0g1Bsy58CGmJq9TaY70dllWjJw+aanJnSZgSh7NxYh2W01fDDqkNtu9LieqswBHZ5v245R\\\\\\\/Icj\\\\\\\/UKNMu5hLEk8gpeFsHEw\\\\\\\/qo\\\\\\\/Tf9Isn8UnTJvXDVEoJjgEG7rtzY3h6\\\\\\\/liO84h2RXdtxX3er3+yKbppH6bimkZ3rZ6OUxxwLnxKba22wM+KXYnUydIg5x7DvxACHeh5gWygYOZP\\\\\\\/ygUFxCT+6lsERAnsjqO8B17JgEptnWQFllZOofGWdrrlcLpTjjZAXNPgYOMtYX0zGmTRZLrR0YWxvt31c84aDjhsG5guTAUUv0QlyVv45Cz1ml6CjwI6ENjCoppB2rE+DL3woQoi5hUg9DLeLN8E6DJbfJrgKUczZrFZ3PhDrSTDf6GR2O\\\\\\\/s+Nj9BvsXw30hcepjtlcv27OfWZamvpFGSmlGaRKXl4jjSxmI4pq2xOGdxV4tVxl2w4kRXgGiaqKyVSYxxQO4eynC6fAzO9kbG3w8YW+EEgIFWwK00bCnVw6KxLb66uyjpofVxIGUHVttyfbgRpC\\\\\\\/f96GEbM+QqtelzRqx44RKHMRwGF3D+mH7p2f4p1qZNo\\\\\\\/eYi+v0gN1qEQ9nMvzMjTDnfbmMPPZLVyiNizAaD4228kI75aADohmIrW9QDr62ftFb+ec4nJ6xmmHoA4q2NG\\\\\\\/kKIie4ox9da2edSnp415TpfXHEZgYqgexoPE2x0CEJcMtFQFcD\\\\\\\/ZRUi9qqAIKIAyjiE27VNF23zcRTzW47u1LMjw0q5P95VgTYc6r97p++K\\\\\\\/y65Pci6+M6BmE0T1W8bHNh4YsPVULeKMdF41hufxHqAQL6VabQDa1khUpLWh612xVvp95plzzHoHzBuPbGQhHxGzxbHWBroY0kbcxjLb\\\\\\\/5tIJ4QEwC6qunb2a3+S5kn0l4np952++POLXh9RCB1YcE3npMzISyCUyw7D\\\\\\\/eYRQEI0pArHGRIifSive5KHMEjlirInH8K7wz0EelBZqkK35Ag1vwNXp1jOQ3NQlke8FIryMhOz6FGxXroQPXNsj+HicuRfeq7b1ZWsmeE5S+6pgpzOlSIC6bg0XPvAE6DaqOUYhR+twcMAao0yC44Y5gBOl5hSeRC2d7jGjZ9EGUXWG+Gc1DJ68Gn7DWqkCy+lMKG5MBI8HEDXUYLHUuE39z8pDEoyk0mxNjmgu2cRLewI3nhk6heloALllRORTn4Sw+1NBb1HaDgTiyq2KOLD0tduVyuXMmBcm8rLXKnRjYdnwY6\\\\\\\/tmNU8u8tICjm8VfFYhltre94dnqyX3Q4iGISyuKTjx72APA1hb8TsmiYY6yugfDY1NAeYxauK2zsUylYTNkqLrQKCmxj2KNrrvUPvvFT7whQVJ7CzUXbjaZxEYbPm3pEdlTfsoGB3riwTZ+NEOp0FnaqzwkdwiXLV9s5u2W0TjPfGm+Z\\\\\\\/s33MnH8qNQFD2bEwKjtoI2kHpd+Q1aJIt3cW\\\\\\\/uoPoC7PmuO5q5\\\\\\\/Pl7xtLeS\\\\\\\/txupYqkI8hcpG\\\\\\\/JThv\\\\\\\/o8zeOii2t4IEZfonM5lCDZwGCHqLD7ccaZvcRdc13sDn5a4XcX3SPsmSvfofksfQoOgKrswvxMfe0iNHoq4FWBYKfjLQPZTUn2qxHtxANBPmecMD+FvcvJWhK0YR3Q3y\\\\\\\/pPbxFIE0ck0us+KiV9JRc+DRSg4i93fjFWk3TK8ufPno8VWSMI+Mhsa2lEGx6\\\\\\\/LXsrLHSKfG8aPRn7CU+ly6CVgOuW62cIjHDkFeYNwySAjclZH3huYv20RuOGgPv1RpIWTlQGZ+tq5BhEdRsjbJGtJeo6jwLhctG4NaD5b6vqDnKzlgtLYXdqx6b9NHCmXO\\\\\\\/O+g68T7mgYBqe3ZoQYfM+AcZTe4e5toeF8pq0oY0yqlW68n3yHD74yV5F1Ox0IS+DZ2dDPIelADe21KolAXMAMIvILLHc55q2ZQRZm07HYDCefmaMqBnwln6Vt6BWnWncBWZ1hnUIAVjClWlpqkouM33cY2rAIKQ+adl38YRH1J8N+RwbubFFlu3bjniWqmMk0fFeL8ZyM5z9eUgUiGSTcdeKF8mIK5\\\\\\\/hgJcYrCSE0kcmvwQNmtu6d0kk21zpVQOcvzwy+tV699E+ywyz2WXiCXEUrC8pEwZFakBXbqF34Zw0sh5Mms\\\\\\\/TTcaWH8EoyYmmGy1aV\\\\\\\/8BaqR60kORbBEO9WAqVvspQqoOn44Pwl1Rmi+ueoC54lVcXBRctu29A7YxSoq3g5xKQlCfCSle\\\\\\\/tU3ongycceI7VVxK6lOSGlAAfeqylmbS2ellD6KNM\\\\\\\/4EJUMTGwUPnyAHv4qLum3Gn4DEZGgNuvsgsEqvMKPeUAtmIHSP4kQTiN1e9kGQRt5l9YHLsnObQulVmxODax\\\\\\\/JknLBuEn7HLd\\\\\\\/7gyKRtDrc\\\\\\\/2+e1PVJ0GFWwd\\\\\\\/an4dIt5bKPfP+FRlNomzk8GE53yJng1wZl\\\\\\\/3OiJeYhcZhPl9BJw3yc6DLfWZMAJnJIupvlsJWlLmYeRg\\\\\\\/BTFnOdNq2RrqH1FyQdDSitJ9CTN2Vsf1aPB7oZ\\\\\\\/zJx+q6B+YgTdtlFOvo1lPHHUlAglkPemnt6S2X\\\\\\\/SOZNEdEL\\\\\\\/r+xxsQh69r9kjOSSeIOsT+p1pzpTljsjTvf7hNjwHTWahKpHf9hq56sM7DZXdHSLgONYofWaaUDnwqD1pgDOJu5BtkZdhlLS24dQx4zn4X6AKtjuHTnsZxML2VgWgg7aUaaWm8VTJHapIrnZs+7IX6b1a5sn7M1dNTfv7v\\\\\\\/j8xraCxg5U7B43e40Zuh12SOPy6fPrAaJofp9kzdgnUxVBtlStGekVp7IcxGL72wU8IfXeLL5ATFdvWdlpc\\\\\\\/ptWYgOqDLgd\\\\\\\/SpBhpYLzrBhEGaCWYFdtEUr1cLY66amR2K4UVeoE+aA20IsUeEB\\\\\\\/JCDGHFj1PtdlzHZ89lywFqbXeyae6iL1JP9bKjl2xAamT0p0NpVCJlS\\\\\\\/6YiEa1T2z68uocBy7Amj5UVHRD5kB\\\\\\\/XKMhNHzOKcziHZhAtaWtg\\\\\\\/956lhhMzje8llARdbff8RwwdI2eLpKRWSp2Rzi0KRgkhamhuqlUYnjaVRjkGsgHiPuS5ARYDTOL7d\\\\\\\/JsxP1d33T3zFMOoF\\\\\\\/xWJ+nKEuYtv\\\\\\\/\\\\\\\/YcVkd1QTYjKdug0CK8UXWO6tL8XG98+BghtdI\\\\\\\/fRoUzf0HASbYZU+qMD18VR3NUMDhtRW60HDUAhNdarT4HoHWPIyezIeY1HdOIUG4UZKsJQzPE4lRMJT32WpwXopnwHy7yvmCyrraSH26eIt0OVNz48bwPuv\\\\\\\/ja1LP2Z08n9Xkqwgp0MRGKhB2QSGBTtjgT1LUffMDXInj7LroOrPcuzh1DHbsBVuL2BJV98ZSPGh21KDpMsbylBerTM48\\\\\\\/GhCdU3zuOjsQd4diKxPnI71K+qjgAyn532nmQoMw9i1Pm781VDfwK278Y0+IPy8KF8LMTc6R2wDFQi4TgsKKtsOJ4hkQlujwhPjqVbqUqAiSL7RoM1Nub3kb0d8OiswEDcLpnjg\\\\\\\/Qj1eLKlvkpjEz+wJnHyTVaCrkPZ4ZsCldU8yAV4QvA2c9pqsmmkcTzPQVbAXfCeuFQF1IOCJq8lldU569AyoCyhjcKv\\\\\\\/wh7euR99qvhscBezbCeaQepKg\\\\\\\/RvrWeEyPwNi5lDID2hnBhGkzgOOBd9ywAsPgaerJuqfNDL8Yzi7YCohdnJUGjSoLlZP\\\\\\\/imVAWBtlQBArNqHvk6k+iSieB\\\\\\\/B2xkYzs43bhkil0+xkKlgwHaTPYegfttCPpkNRRubT3ZjZe1Om39Ry3+YO3wAaEqRFtbMJfx6jPAkvZNZqwisqEJ+VBDijKgoQozeU1+63XyPUwR5w9B6Bn3\\\\\\\/SJ7l4swQnYafRhqQWjRX\\\\\\\/DAu7zQd0nys8nlcvltUYP8oib9aWIAkYKjb1EhA73FmzlieWUz1JRU04OYwngS3ZfsKMDF55SlDeU9QLggaWmXIFd8PUN5Zr6tQLkTA+t0RhBCt9Iusyga5ZFzVk7XWVmzn6He8Vuyo1YpdnGWDPyRXCvDTc7BX7NmS8tldDlGYV9z3JaGs\\\\\\\/rhGcRfe1BiLJEWgDoNzzqoL9NLURlJIB9qMHAyIoAwX\\\\\\\/OA+vpSE3wrqqNLo8L5LjyCdX6VzH5w75ThswkOQOY7y4h\\\\\\\/qDPJUhbenpKg8xUvt6zskQahyScQn58+PL8zARZXyPAQDpMwE\\\\\\\/lDpiSeRbeW6Edm6tbpfAuB78x5t5pmP+qmH9cxL7wLCocUW\\\\\\\/QV4OrovMLTKZVPO6l3IM+Q2gqvQ50Tj5LF1AbFWtsmfEcZ80xDp7UtIXb3oEwKdzNdLvR2kFjbeH\\\\\\\/gDFXFkWbga72OIN8r+\\\\\\\/KFLQr+Y+h3i99aJmZ+D+ZCigBVv0gGEC6ewNjamvjZjoewjtMdShfSaEmKmeQpsXYrKs2C\\\\\\\/+xP0K\\\\\\\/83C+jwlt+fDsmdSak57hgJaNxoumgrvQwPs\\\\\\\/kPXZ9dWagNVEYN74BV3rZDbWB9f82EgOc3GYzOQ7Bcxs6Y9skyDp+s1twoMV2++8VPdNPMxPbnrepROuHuaMzl72mtlxRo8XY9MRK1mOy56kfymuSuao4CwMhvD5l52DKc5q8WQTxOn4GRAHkYQdF7bzL8JadZXJEZDpzhygW\\\\\\\/aEZSPPGPxstZn1kKdB76xgeK5ShNCSdhFldzJ\\\\\\\/A1LSM99XdKkcZntw0xZHCLmF+Bd+JWaHV0bmLMen3tnhbQE+n8jGt5vyCllN0P1umNbDIL1GeMJ8aiGqe2vMTSIB2BeQq+vl24rfJbgDxMUFzEi5heWezEd1r4NsCUXbKEtnlHBqgGEPfeS3FrDaZMLzV5d2hYGqunjPEhArKUXa3\\\\\\\/St1Prr+e\\\\\\\/mUA+6+L9gbS6UzAaZlP\\\\\\\/sX+9+pMFC1wI5iDZC\\\\\\\/vR6nm\\\\\\\/bivETQ2y+TcQV8gnaaQuMRAT9ixp0tedAqY5+bOI3J1n3zsDl1hLlstUmlN0qRh13vc\\\\\\\/PidrIHAskUlqxiF8WQnY7\\\\\\\/MppuvtHYxrlYXlMVdCTm7EOhUkAc6fg8gWV0\\\\\\\/S8T67hNVTaxVnF0\\\\\\\/Qg6d0r\\\\\\\/iwCxucsX7CH1efh1oCKxrf5EElVCGG2rME7OX7CRgP\\\\\\\/N5kbAc4aYcBsbEEdP2\\\\\\\/fUg\\\\\\\/+qeorZCxVUhu7HmN4WRhvkoAqW2875L+pFc\\\\\\\/QuIgAe43X1Ostgu4jWDtEY9Os4KTkus7UtYxJVuVSezTWkr2HR+SQSwTiGznFIy9LTzxwvMLtvrOZol7qQzLAfFrKO+U2hHtg0tsXCKdc59XDIopUXCUTLx3WpI4VKotcp5GyZEKbwtBt0mER\\\\\\\/cOJIkPFChXxnwkGvIFxkTkORm9dTqv+QbW6S9IIxHohInc98ru5ihsTk9JlNi0237R6BOs8ZUU5gMZ8dSY4SudLNa6YuEQZQpK3Zsogjde+ovBa+92tb39qkxxdC7Xk5R8DaNUX1KDz3uLrof9rAJrx4wbcWWgf8OH2aob4VfDsl\\\\\\\/Ig\\\\\\\/bwE+sf5M1S7N75UemoYIkqImF4g4nrccL0\\\\\\\/s+UUBoRn9MT9H8PGXWgkaQ0p+mKF+OxEf95U3\\\\\\\/0eOHZDc33RkdZfduv+\\\\\\\/P+5Lkyd5afw+d9Du0ICBcl5eImBxj0TNz7uqhTmUy\\\\\\\/0onlqEu4\\\\\\\/oGUhPgVX5aSzgvLfxMrEt7kt5IjifUzt86tnWXbOCaRyqauEb5y37xiJlk92QWKVXgVgHO6bv3fxZLstnVQDVjDyiLeCEDgcWjvODA4VqDPK\\\\\\\/CksBBG5Jpf\\\\\\\/jnV0OGg5CJqgpzElpYk9bW3C2A0Ulb\\\\\\\/nN7bpGrR45lhRafcgBsYaJM6Qny7oaez5yBwJFYgB9Vakhi9CRAzW2czdyK0h9pUp1qOcnsm+ibAEPwL\\\\\\\/EqBBiuDqFfECQwQajinzjeYab7wCrWSW\\\\\\\/vX22Q0Sv9S8aAfR1I2oliDpTWJ9o8fXQHP2Jx8zRTm2sa4CQPy5zA475tzgyXvOqzWa8MTrognpYk+7OaPkTs+fuPV8VfzyqxvhLLq3NnJ7kUAxxS8Is4P3IB5cIgF6eBVeZVCyvU+WwWF+Hcu\\\\\\\/lFcStTg5144hb2yjXaCvodJyyWrbr7d3IadKK07VSHSGWGA70Exv0hPyErY1QkEIInVadOb8I4lvZBA7xPNGV9dIk13mzLHgBH8kHzV9d1pGbCBxB0VNqKZTQxE01JIr70cZGV8lXzV3mSIqFGzJ8M2ohM9h8xle+YPCpy1kkQI12Ac38S9sI35O7DRT3Ce9bC1Qf0Xg9SCV2M40BDR04TmtVqgDXTlO9NL8i8XTN4oGoupXn7EaPrGQTLC69dKE9YSc7P9c5KeY\\\\\\\/Z9Vb1OL+4WuTOR4V73GxgS+izR5+pXnt85xDpr6RH46wUcQ7fX+OwTZ9dNji9tCqqiDyX4vWk1z9Fvjkul7OWHUi\\\\\\\/KRQFuBmQ7njBUGVljjWeZiSY\\\\\\\/mx6bK9EIQonG0dWnzWkM9VD9R92D+5lZo1Y9Of2jt2ecLVp19ZlmlCU6dgmg6D72hKaVPeLca0XQOTdUGXo1QJ07k7DdI2vq53hG+F8isw5wmJ6Tcbw0H0L+4KsKZ1wiPsYISXzpfJY4+9ac62gUiXpObYWpbY62NIvBpPkfiZf0ASXqSqtH8JGYC5rfykm2yD0EGqmtvxh5yy+E35bLT9a4YIniRHF+ZmJZI4F7jxHh8djxSu5UuPbgtsZ1y2isExbx3RHNj+s7C0VTioxIuc7CUK1PL4QHQXZPP3ocxky1u\\\\\\\/0ietUNxhP\\\\\\\/WjemXZ20aKkAez9+btrwJL03RVvmhd8aL+EbWX+uO67oEJkjAavEcGUynNka7WwHtmXpg87XfyKG5E8ygVk8DSVSlnqUpTzdEGj3+1m9SmXtnPLAEo5cbO+a1JrPohLwbdtsBzrLj0sXLne1xEotzTsaElwpnWAGb3YEvqdP9HKSp7NT1u49Qq4fz9GuutzQyQ3M30OOqnOWfUnHXchC9NLJiDXVtzzFEQxw+wk85E\\\\\\\/JIMStMZVjcME2wWj1PnO48tzuk0JdBt1n137fMEHeE0+ZNzdp500N78udSHvUA6EIKKd9H9iSad07WBvFBkxYDWkTbJk8OulBe11iGDgwFKdoZIqiLDnwC+65b0wjiMV+UUd2+qNTpEQld9fdtazc2BIGXSv8vlIKK6UA8Kee5+NQR5Ybvy5GaPmc5YEIVX9uOYIk4cYuzhZNorpNR0\\\\\\\/S\\\\\\\/OoqFpp3UJpMqNrrX8UbTUKiSYEbpSM511FXAb1iAcHxEq5CWz3t3WNWIQD\\\\\\\/gtRtV2XErauy4O6OBJgQZQuzwm1Cz3s\\\\\\\/zlQk3rECHcYxyiVKiUEcOF3G1LNkiIrKypih5\\\\\\\/X1JDqJ55IVQpBTRKERDAWgN8J4yQ6FlqW7wRraSsgy+8CBnnwBVmtDhzWjHyPN5t3CCC2uI5a+V\\\\\\\/9B1Lj94ats69j5\\\\\\\/oXptrAdpX\\\\\\\/rh4u4R2pSEn3TWDSRTsLr7gzLsO7Sn2vXP89WqhNtQizi+VkCioVQh0mMw557YfOYYifns\\\\\\\/WmKXhbm34dhlCHWAfO5vhBWHvqyVMPXKNe7qSvU+EgHrUboYFERNeFmrqqT+6Lw5+kchwDwuhHxZEHMoaaf7d\\\\\\\/obAZvR1Q1UXayJPtP3n\\\\\\\/cDpLAGenFNdqKQ3\\\\\\\/j+4jyBMBu+4xs5M3rm66bZ9PS+xgTRMdN+rhyPFKgT7QEZ9L5QKeJTzjjcun1+OfnW5x9r1x3Js6LNYpYLfAmZJzCkQ0DRWYbynotIhV851LvQxrnMSlYzy1t58UQ0e7VRuClN99TSxA7pBN848jTlT\\\\\\\/DUtU2bwpv\\\\\\\/sZvDOdEt2P5\\\\\\\/+UOfLbIbdj2onN36O9yh956Vuh2zYxrtky4VeHsiCeEH1t9knv3dSnSU6IOx78Mk20DFr5SxpEWhxbixO2jsjtUdjLVaHMccL1RKnuiPitQq2+CQw8qlLAchOLRg9GT+R8ZKPNPTJ1WITvWQRJsNOlb4wxNpqEJx\\\\\\\/Xs+Xmq3GLkccBG0C0ZSHtNQsH2qWNkcSn6tN23EPob9KDnvxrmfaRwzMBl1Lr39OliJhBDpjTJ\\\\\\\/euX8+kw97vswlSUxa+8S\\\\\\\/b4CFIyUP8kMrV5nt4Wb27HI7CzTcrMy\\\\\\\/y9LRYBSaLX4pAaZSG0Hz6mk99yX+Gjpp+ys4MH1Q3LHQWTJbAHXGW9suM3h1dmCS6\\\\\\\/qIpPL0BMsrxLSfvc7FFJ1+y+LUT2iOIlSoP\\\\\\\/uk7V1vpKV99kgN\\\\\\\/Uvgr0wdG5OSK9lj1XmnbTtR9jk6jqFpV88aJdvbGmIIWSJWnS+roKcjarVoMA6q\\\\\\\/68RAlpNIESMLApDK9MEsGMhTSWYlNYXQGdNpOAgcAT\\\\\\\/gNjkAs4sdC84584yFzlTi6iFjDwq1WIWoyWtn6ubuKyVrSOeeWhL98cZ2D3Qo2IyB4BQxdP\\\\\\\/M1A2hof3CVwGna5pcnl5NUkgCOGmtYdQNK91hYc1dhVUY0dGK9jtgRAIgyXAkwFQ1+nsh8tJvMyNdegEwfWMtlns9C4QSKOzcAMvWZKqs\\\\\\\/3KV1n2Jcv4YH5yYcm+k39ufu76qHlGLULd4maUIOi5lCY4Qn8ynJyxfYXOjH6YYBGg34Tvf7RUYtG1+KMGTIUaRITV\\\\\\\/sUIw60s9lpqwYwvK+CtPoTtf+fUfFEfE\\\\\\\/wANRitrqw+MYEdiWKaJ6oESKjtb8\\\\\\\/uFBCtCWXLmbOPguTM4MeOZck7Hh2QqG5HvSljpXoe9nUa56R7iwcC88LjIs8GuIhO310c6LzsIRZo88dDQGslsg8f2oziIgUCMk4U2ws2gduea2mqtwKThf+eI4hMaKyLxhEUW\\\\\\\/\\\\\\\/E0pUmJNiW9Zx4YQhEfFHdpIp5MwJG0i\\\\\\\/WZhe9JSvd5tQll0vlSKiM1JlfQzbf0eGvL\\\\\\\/yUodHlsuEmQEvEUrrJf805QTBeLdxdqCZ8k3TwyFWW9hXTw32vL9+MWPxY\\\\\\\/YTxQFL0imOHT\\\\\\\/SA4UX4Mh50A59DLVnk4g8tv+PeAlrKrkXhPWmIQNiIIWIEgZtV7BwAY\\\\\\\/Zc\\\\\\\/SAYDt2Rb2Zo760twC7gN8Jc6q6EVtO9J8s7ESS44BHk4sK1k2zixg4z7zD1sWgxDG2JXA4XGsBIwv6riYu5q+YGatKgYRhN7kkDi3eW+1gZhvgXg0dfc2SE\\\\\\\/GpEShyUSvfkp\\\\\\\/sPQ5PTPEnSOniss\\\\\\\/g99cCX3lH8Xohwf0f7KAU\\\\\\\/H77d4tGD\\\\\\\/sbFeB9ECvRNoWcJIcyAVmHeR1ewo9pClt46JzkuUIhtolRfQ1lLg570a7YJ2pGvH7A3eNVcPGn8RXwtZz7Mu0Q\\\\\\\/SnSFY9ASowjrXi+VyvdGDUYiTDIROYaEP7VU0m3WsTHlppmMcc56aWK0oxK0v9jcARXZrQ7t5NlbxlLySlSz69ijWz35KO6Bfo\\\\\\\/m2IejFFiG1KhwoP9DiPHA\\\\\\\/cCsax5Y1RjeRuOOTfZZVCarOdP3tGXGDyGsTE1B+dwN\\\\\\\/N3l2iDNEviyUE2oCw\\\\\\\/rGmBzHXcEDjbjTdMY2QkwbTnztHm9mkTJJGaXyBcEumyenmXUepYCaP793urW6rsykfkJbVzV57bSsNvTsAQRQzePPPPupuVLx9VVMxJuEEWLqf+QYccpE15\\\\\\\/caj8ObUMhH19ddcbkJ8LQ03Z4DWIcAXKoLgKDGyK28eXa7h60PzufLZ+mFuhuy0ZFFmhpWy5kTbj0goshkeWH6jpjamTt5EDWWJ3K1snIZKHHhltYBOIO2Fcgxpo\\\\\\\/4SuOertEzm0djQavEjyN2GohasV9nby1Kc+hzbJceCCvwgF\\\\\\\/Hr2gbBvVOE7l1eLd9fu+56Lc4qkUF1r3JUXZpyD0gL88rvSYnDHduEnf1zSbi87X3H\\\\\\\/e+3ob+xp10eG8hK+SF\\\\\\\/c9Ft51eNgikXIuR6iwvBrqlTD9XIK3B+JCiBW4F7XhKVy6kE6fu5vYXFIm3lGgbjQBsteJqjqiWGDK4TcOYAA+6d0D2UfpiNJylXbGgy1CYlN3YR1B4iieQJeJngoWPajsVz8O2DsjLT2exAlxdErfctuZeMUsOK0zVt7s5ETV9zbMwaJ\\\\\\\/QJBEc9qhq2IkLC9TuEmV1yupdRMohPtENaPrpaE\\\\\\\/42pq6khGQzfMfxvGLXYrhvPWfnhh\\\\\\\/FnDRA5xFv7tr8Icn+ev9JeL3gKY0MYcSCaIU+fyFrqKLdxEfsV\\\\\\\/XW4bJ73zu3+2Te+sm9lBk0NK6QyLSbogAUQuZWD1ZeIcVXAAuWqErubWp57mWJjZSUMj4G1DynouplJQzQ11HZubxnv9+ps4FeXF7G5qWXBFLMRqIEDo6\\\\\\\/9UCgHHgHoabiYeFMknvZ7Ol\\\\\\\/VoL0F\\\\\\\/G3nOxAVGkShrNXthwdVbNSD17LqdHWXoFJ\\\\\\\/NPghTH7s2TNWPTepAQFQgvsJuxmjnE1tzhc7XUAZwDYYmclfhFpfwP553N79Ldxt4BGEbMdBcvetuSrYPFhC\\\\\\\/+KJIAqSlDBVZhCjkpNp5+IWiObmsgJQphUGJ52tEX35iLjonowLz92cVYENUHahJS4gmUTcGdDD8G5AA1QBw3c0HbcgY37uS32Bz36V+MbqxzowrZpwXKCWQGRk+lB73SBnsm86njaeZX9AUm\\\\\\\/z227bJpDeyJ0+eT72sGHKosQ32kkKVuhEliLIwyJLqHo39AKi3SAcr2abtX6oQq70HmboGCXj7EuVJpcIUjIJ\\\\\\\/fZlfe99Al90QPK74VLdGsxzrpuZrKMOpFKa286wlis1KgbGG3D2epf179JsgHHCDC7e+NFgi5ZH5f9lfnLf60gOVWx+rq2sllvxWuj5GFmYVgmzIEFmewlTuVGydZ7jTzc+tOcxrxze1nZtnzahxiOg3eyWHGqJ0PsPrE8Nn4T1c6lrJtB4HKka37I77NBAUHbakkVzCaF5v\\\\\\\/MXBV3jylMQu1tsG4PwUHAbfgWxPrWmzxrCSXXkAs8E\\\\\\\/OWbnEojm26i0E2S25vaZGLTuckmj9d4k\\\\\\\/\\\\\\\/GC+g+8o5mQLQeT4\\\\\\\/0O4bjrz2iNze1xP5DD4Y0tHAx41kPoL+Sz7GLiW4fEHoUKtNM0pFmaevANMIEloA7Yepo33orW8RLhQplzcbjH30WKmOQqOtjY61DbfD6gaxCL8gRkXIcEyaLO4DfNQCgrSwrnYSmXLd6cs0JihMqQV5jf9\\\\\\\/4vnVOoJ64vU3\\\\\\\/5zDkZCsxb1rSpj1t+vm3YWM1Fq0oW6WWGvc5IRvwKSacupbt2j6V4N5WFL8hbhZnN5vbRtulrBQ0z5G7o1inGdbOb0P2DzaEHmDIN2LHGHxcI71gJ1HDHTrad1UPpy3M8F1tXi7\\\\\\\/x4Q5snDsLy0z7U\\\\\\\/IEwVd\\\\\\\/qSvo6pTyAqCK2kLmOuYtUxCSApsnI5OzUAN8qhrsVJFsILLa83EfgnFkc9fgMeM9jVbrtwqb0yK9Vqh5Y57+Q0e3vVAx6Ohny+\\\\\\\/3KATJ41KSH1Fti4Os3IgbOBdl2VdF20wKBOe4lZ1VWSeADJE3Nytiw4dQGPBv7W55gGHRRK2ju2ctmnWBbHTVI0L0TrlNMuiCskEcMdrbaBXTIW4M9Se5ltVFsihis2mrSPao6OdHLiH7PLzz7HL\\\\\\\/Rs0CsICM+ojF6fNqRLk+gvqe28QACQdgOCDscwyQhDK2m1w6iY9UVQQROEDLy1aRsGD7+1JsAOf2dIv77kc3yCE\\\\\\\/2O0GNEWl7DC04hUrqyVt8JhXZVZa5xKwgigt+Z6AZJfPWy7M8A655k0Z5xsm3bwzZHc\\\\\\\/saENgsF\\\\\\\/i7tU72qyyHidhf6aTe65tditM++LshGwLTvRmIKpLm2wfmH5rV5\\\\\\\/kFhoj85rSXqsXerXAmPsH5ATfHmzd8sOHHHW0uNMpffDSbJvz6P8G+xK6MD9Lu4VL+czqMbH7DOML4nb0T0bkMQhWC8UdrO19hmvLF2QUPw4w5yHgRiOqH6\\\\\\\/X8AfPRBzojnT04muM\\\\\\\/FxAfcqaYP8\\\\\\\/JLgG2\\\\\\\/GqkpzrJxJffjMDJYarvPlUyGQWB4M4HGWk\\\\\\\/3ROEutlzOKq3xuClHLLrcOqlDOkJdw53y0DNQbOSS6bGbtj7Hk1f6m1zFA\\\\\\\/aqYoA6H8AZCWBbXiX1AL0LpFXYozFY+JA43VUC+WJmHJ3+zMdUKep\\\\\\\/EsjYy5LpXNiYt7bScHkkwm7qoIKJ5DmmUoaIDJHDMrDE4teMFLHz8bXzWCwGlxNXfYes5LR2I4KWngmORP3PWyWT19Xtp+sc7pKUQS9I4QXIHSRMmsG18hlHP9\\\\\\\/ImBon+k3NGxPjuuN51mQPnAgE232ksYf7yg5J1w\\\\\\\/5R142D0J84wHPDrr\\\\\\\/0Jj7z+E92UHhztUtcGvZELdxaJbSz3L0+SGNZTLf9RZxULWCHEh4KRO34W9LIccBPkVb8FNWddR9slYOwHg7mCICpkJMS1Sqd3Ydjn7gz1WNxoY6ycQG8U3grRGUzvacWdZW0F405FflqwO64+M7wpXXl5l2zy1ScZu+sbUEz1s3gitvqzUZHpTmd9Hy3Yo6iu8zV+cBPkWJX11BiT2TFIzTRzPRCxtKWbMsIONyai\\\\\\\/AY0TlzKLoeaSzVeAFF7ZyMKHnZHVBcoYe55dofBR7THE+kvoSBi43Ypgq1VCVgMybqZAhW4mOMM6ZA69q7qqMU6zkjIKaCZuMi7pJp2wgWI60E5tPtCJJRYCiZzHraj2p7iiB5ikPwHT1b8+XZzbS5fx9cwz3OQXjRSr14rFOuHIiY8pY3y6njOveiM+Vf3bexJMgV3EzZ3ir60gBV07P0op\\\\\\\/XBV519UGIebo1KTdM6mgAwzau6F2QD2dLXOXDJrK7SVWN4C7okVAEBJZ6CFLki2f2NNZTfyw6a3x7Uql235NdDyvxP9ETXu6l40lQGz\\\\\\\/7otdrRAnt61F66x22WkztnURQHVZQIdcNIhlFTXEaY7fMtkD4FW12WW4dyUIXdC0SIq6gEZQ9D0XjeDhKpnuhNQh6QocBiEBMl1utpAJ50unqgc1Da93a\\\\\\\/VNByUof\\\\\\\/DBQyXkaRvoT6LMU+hnANodgbj75gZzA3CE1Fi8pcq95n8oMvrEIO64czwCc7x7WAOGyy2ucQJkQoO+DRFDNDXsa+S22H9TbgBeK+nFgjqbPkp6cUDUznDQE3TBygI7xiwyaBKbcQW\\\\\\\/RggmTF1bu7IXZ6qOm2PLNlc4SMVchXwYrc0n+CZBPZ0eRLjbMD28YlBuKhfvQv+Mm5VkwccpnFi5uJS5KOSj39OAY+ix41T02AJFb9UVjKhHmIZ1ZNAinmWuacAUvCFr1xmBigF+RMTwODhTj3DY5QwYTG8bEgdt6RY7JZ1e+5TD87GkaUrcNk6H4Kw7WtPWXHdeOUsvMlMKn\\\\\\\/uOZevTK420SRmtQZtvMZ\\\\\\\/a4T4werD5Sjc0cOVqqKNTZDHPTGLdPfIfk3vRpWLE5A8YThFpLrCGFp+JJCzEoPOiFjn08UPT9BoXdyLrl4VXKYXiRwOODi4v7q+CpYvqrT9GMlnkTx2PaZMWKKvwwNZmRYLlJ5xo\\\\\\\/1PPlCPeHEw1haKeNlq9AP7CoemW3ebp2DXrhV+YDrnK9\\\\\\\/840OztwFdP4qsxeOtHgcF+jNT5adxZJQP8xEX9weJhEy\\\\\\\/uQpzdSK994FdmCgK22k7CknQus5O1hjprYB\\\\\\\/EGgXtShVY6yTehcqaA0\\\\\\\/yFrzyqW291RaxI9cppt+v3u89DJaq8jU3Ptl30TVgP9ryRZbcmvm8FrNwguucJLH050GRatqMPYScrI3gN34hlsS+bvDQIaP\\\\\\\/owZeHqLdOG+RzhYPb5uLDTKKiTnJzovemw8WTUO9fgpIhvyEsTQ2\\\\\\\/eLlHXby2Ng9Q37OHoeF9ruWrNerhpjTnymMyVDPo9gF+6PKylJpI\\\\\\\/4NfWBUnEqLn\\\\\\\/rv+HwGvnRze7H8+6oh9Y5aQ6jkk7wF+25dvXrZaMa0LUljvFMgBvPV5NGfuW3xMhuoqQwCncXwRwe2je9QQp1ckfYNqOUK2\\\\\\\/fG5\\\\\\\/uoYPkpwj6jrLtW389yiL4qQFBHERHAa+yK4dP7+BHXArrsxOemUBzmpGfn05JIlNpSWhCOTonVrtmlNGkVIk8Wv4LvXLhI7FvQhVADgq3B4eXs4nJyp5LnDpuMRESS0KY+JjBWenT0Mgtj55BHJPR\\\\\\\/XD6wc8pe2k\\\\\\\/h7psv8J1RC8v9d1yuulmCwI4IHvhM37wCnbjwD+EKLkzV6enFHpAdb1+bkn3zQxok3ps2QMsB6FtHXEJyHr\\\\\\\/7bvdwxGYlY7BBdC0x\\\\\\\/T3U6bkcwG0C3xxRwvmusqdwU07KUBdGt609V0tLk\\\\\\\/j9z\\\\\\\/dFXikYsGJheEPkiJgLpa0G4FoHNZeGX4C12gvwWxx2OibKsULTHviVd9EpdKmkGmhqTKFUBvO8aoe7Ey12UGwG\\\\\\\/W0tBd82462RRYoJHImXrKclfZwvFvr1XM1BI9bp4jl42xwfDkPyD5Kv6q\\\\\\\/WWq4zI7C8eJCY1YfhGJ+YKeimtj2B0wT1aKlS7mEadVZBGp4KwDcH4a\\\\\\\/QrIX3TW\\\\\\\/2EHqBTLtrrSmy2cVXyQ2yJNoxaX2OSGQ2XCAg\\\\\\\/OQIZix\\\\\\\/dCCY2Ii0LXBgdAZleazC4sD9HjQP6tL7mzrKUQDfJTk9Hic54Ahq1gigKY5egAgNg\\\\\\\/muriRMnYmAwblSNeWJE7Bah\\\\\\\/RLY9OunaP41HnH320WlOsIIJ2qP7GWWzL0kqcllsdPkpGLn61I3mFZI+6XGDvRb1HFV+MThsEp81NH49Yz5QaiE4BE8uz0TzW2in3bGuCwRuVV9yOjVV1JS5\\\\\\\/\\\\\\\/+qtrcLSQ2Qk0K1Uxj23N02smi2N8rkNxEdwtOAYQP35dk4TFfte+GG76PLF7lbGopg\\\\\\\/cWmRzyyEvpvmvbduFNPZLQXQUkxsSEE\\\\\\\/m0P3ALmO\\\\\\\/hvekkBqB67c4UfkWyLj6sUTzJx7qgCYsF3vGUWIVRFq9YucIEPnDDMGluXqAO35loqbHJdF0weqTtQo+FOf2jiTl8O6qRmoq80DLEX950DQjcV6QJd9kT7WAobAqVzmgNFItA0+t3tps3\\\\\\\/orp\\\\\\\/UXfLNpcTl2JUuAD1aH8Q4xMaENvyYAfwczrMbMkiJqVzZMmUfXvsJAAesQojQe1ImBOR5JUwVVNBQu\\\\\\\/ys5A8JEJpLKS\\\\\\\/lrxQ9CP8Hz25zI2PDoY1tt06kD5Tg2liCYsYeMuKAC4oSIdG713ulRGmAmRzZixICg3gnHto6knX\\\\\\\/2Meas82MY7FDhppEUV+FM+yrMWbxgrJZhngBem+s8EYdqh3s05XVRzJ\\\\\\\/YzpojDVUnE69m66YMHAz2ptrCyStA18KVU0+BFfDea2l3TgOOwTSHm+3Plp9ZowaIbpbZcTNZhIsLIiFTp52H5Tpivbbx+PrujeP3ToD4GS7ndK18AgVMZuuBMVw0aIpRjvlJWf2eMYLU0VRz5EKQk8f17P8JR1SKQc\\\\\\\/AAQ6TiH1MMF3vG9ki2Ud0P12MeWhmu5+3l6\\\\\\\/mlECG7p6e081R6XfBXHSr71KnWW9Cchy+puA9H6HyUfNAZojgBd\\\\\\\/quOXAe\\\\\\\/QwOocqq\\\\\\\/ujRQ\\\\\\\/Fr1gQDCYcxnCn2A5qMbNeHbWuv8U+d2blXFPcgIX3Ou65rmORwaIApgxZ4htXN0nA2VW+A1Q9W8486ozmXkHuDAmJluX5nuYW6mVLFk57JhP4+CsUq+N\\\\\\\/3zbXfDkOm0PilZGo8VzTEz3BQ3LnnUV9shWEuaV\\\\\\\/oQRKh6F2MMvq7WYApwi2tdvkYQYN8T+ZuyIBlP23NBH+pfjAG5cI1bbEyvzJICmSfV9FeZpVLrBQqVYnmIVKF3tvRw8A6bPt1biYH1eCDjLIHDBRvlgAwYFkEMoMGTCpWh1XvmvUcp6r6geMYM9Do5o78Wqy3h37TcC2zhfxeDWdrWB7L\\\\\\\/dTPVG9nQEKGq9f39anV+OG0NwJxwoN986D70deRol0la\\\\\\\/y2iDADoXvlJtbbrCRLn3d+e3sm1XxP35wPyrvZ\\\\\\\/k0B\\\\\\\/c2N\\\\\\\/isTpWqNbhM4QiUf46KLO0KyF1aDELzsrPLzw8DaoJg5QNYXGNxOhoOVGlCm8FNCWyUv9438ylH+EGA8CP515s\\\\\\\/LBy7HsrBLUlLlX\\\\\\\/dgKSKOoCgOPzaWX6C596AaL\\\\\\\/5oPLPLJqIo8Qo\\\\\\\/6A4zAHlkOajZ6s\\\\\\\/fzFCKWVT9fAqLFS5Bgp40By2LVsRnP2XjxBaxxzn6wmHVZNdNFX0Z+Ua+5M6+dqiFJR2UHq5qc1rmaP9Qnx\\\\\\\/W5XkBTvFsg6JZ5bg+j6Ag2++9DW4+U5sQOWJOYeDW4bbb17SKcdS+3\\\\\\\/FxwdgdFTu7\\\\\\\/prDnDnGg\\\\\\\/upqC8at2001lzQ7TiU4GQjUmJd8Kqvi4HeUn6aRtyUJGgcL7koOn5dIEsZxPxkkIpZ+zYr78JtTzJt1LqFK11d\\\\\\\/8OK9QIENqI\\\\\\\/n8YjzZLZTnZv3tyQIys+0RhiAh8hBXbx6bc7QrWSlfqzT7M1VM9b\\\\\\\/Ma36zPKXwVxDmprzW90SosNRq5YhAGSsW\\\\\\\/a42lcWQVdI8Z8pzwx5lX+5RhcE0NmljBAk4+3ch0Q+EcEN1FjB+bhMXZUmMooGJvnvfXs7xIOanVWEVP5R6ls6Tcg0DFgix6SY9xgX0efjYZ7Lyh1wkLDBn6rQvfbP+iK4SIhl7WjQgReAAW58XnrX6n0YLfph1wLRkcFWt1DIkoOxlazELgVfjO+uQyIRDaBUz9gDT55+iBZEkjViRWXKRomaoctCsriC9VgBx9zmRtnsIcUAvGPjbB6ooDHSAHkv5wkWV8k5GHaYHs9wJbmkA9U8TX6xQYsUvxkdaHPONnezWfaIGJsRW3wLCp74W3XofoNm\\\\\\\/C+8xXClIluzf6G87X+TIBeQeSk+2kAG8BfBC56FE1dKJZobg\\\\\\\/qcLDrIECp2Xg1Z0S9EILyJmmXkfQQPvAot\\\\\\\/LiEC++3ab4wJoNhaGMHSTNF9cN204WDIvOZqr5rCITOvQ6Xz5lx1ESMSQCDdRCG06GJdTnnRlz5jRcNVQ+4fiEY60PcbSjeagsi2aBD3unKXg1Y1IqzWVAuquEVvx0TnmvR5E4VcPLtgSOKOBbZxG5J05iu2FpkNu3jpXOkeRo2E4ndInfsaYlvQb\\\\\\\/oSCOpa1IuOj0nWLsm6AYOizWR2jKZYb6CuGKlbgWg2QkIXLp7RtKFkIg4IHZm9U0kh9wNqnn3sPS87A0xK964kNu66QxivTEKzx3Isxj84NC9Tii2YxzDG3ZzDcr2R2tmgzXPsXN0liO6NTJVoU78\\\\\\\/LBRggAeTT6rddrcY0DERdeBzQytKTvWq4Zl2eohcrFlaINP6pZ7uGbtEZKuzeR\\\\\\\/KSiDZ0ytXM975IDH5uOMjBRxYxGAR+jFWKUkoP3LPnqup+qYPgQzRUiqJITm9yv646ehuZXa3yv2pCLsZPAWvto1RvF1JyKk126u4cdj+Ez7SF34OVkSvMzpvJqgLlY6UmV4Sy5nSaSe7\\\\\\\/rWVE7YGJSMCt1hTCgK4B2RL+6Og\\\\\\\/sPjQyK5d86KLBdPRcAIdSQOBMyuIuxNDoCW8eCCFt0OCyegIvBdDpFja0fCTtjVLY8J+LGMHbiuooxeaAPnb7JO5Ecraf+ftT0uY4YnvEbJYYi7m0mugSPwl8OHxdRZvbD1CmlpnmZqVA\\\\\\\/KJDgIc1+BJaBLv8+VBx8UqT9PuburYPA6J1X02i43+IJt2AJ5eg6DHtpQNIbOpm81wnxWxnMyNNVkL8lGU0J60362WrXPQX7s62M94oNe008zlUth+1axDPKYoQ9pNBD7qzLcG7YAPDJEM25mt1ho7tR9QU+R5R0rlAXpzLOLql\\\\\\\/Awu9PN8f9nKdKE9u4k82rixfXn+M7+U0S+1zrsT3spFWJLfTf4JU6J9DcWb5qbGyL93md1WkC69l1pVtgiuiYhLXhkprr8AooiyKIxd+VCwdT2hBPgH+FGu3L86FEipUtDMB1gYMz905H3r3hNGKYK4NpLCOtWkajjp5rUiWUNetzcT67DvAmHEIKHnndzjqUtC0dfzyS705elEF9\\\\\\\/jbj+4\\\\\\\/GBn3HRFmS6uZhNyLFGTMNSvscQPp5rXZ4sdmX2XPKH73IpZPxFpFYYL4YgEQvqbfrXhA1Fg11o718wbLkYk3+\\\\\\\/kdH+OiTNgR18RXo8aMihBtBGkH3pRkWZZi89jHrqLp1b86TD32MIdAQlGJSHKj+dyNm1BTGXU2hBIUhklqNNEyge0rJDdPP3ni5+\\\\\\\/WthypIVjT2McBSn5celId8I9kGDAKZLLvXa4W+uyA5lNKx8rRMEe3\\\\\\\/59JW19FWjdlB0uxRmEfsi23evbPbPPMERpLl3agGnLfBZn4XluoQlbA7SnS7xGdP8hb3SZ9LMuqIwbrD1i1g4wecucwqMhazd7iu+IvOhwJMkyebO29lipq066Yg+Xx3uIhyNQpePspW19MxR0QgI4Z\\\\\\\/KOnN\\\\\\\/Gk+ieqGLZU8Gxj56mhzh\\\\\\\/qKXlerc1To6a0EuAmHRGEhMAtMPa\\\\\\\/7+cpTGQgMDZsga4eN0hLTA7wOf2Fr5PhEWU4ZQ2lwRQCO51eSBoibUj0SoRmxUhyiRfETvVoAtzcW9KhvFwHiRkfxkru1WW1pMSxys\\\\\\\/+YBKY8I9TSb8\\\\\\\/iex4YC5XuntA2Dt7nuRq1ZQVPIxfWAttPXkcNLz8qq4dn1A+0PXHothsSucVsZPsS0TwnnIveX9eAOJ14iyCSfDcmhIjjSirtYBseanPsUWnMJFhy6G1zxuXUQY1mxwUiMOVDOf8VjblcsA3ssyGp7tguS8zSYr8tc6ccegpxbX2vg2arpPZ3+B2m29ni3aop2jbUrGBVBA76gELRBYCQ9SrdAw3Z\\\\\\\/DzQsT+nWJBwNfLlXOdi35uuISZMSxGrcmCBpSH+OoFgIUfkFKYco8MSF6+hAfAjPS0s8JxgzswYlZ9PBkRbvbNQwUqybYdU2Nc+jMptrkynVtg3xkbdW6O87+lAPIveL2Q8PYW3Om5x314DgV+yYJxVC3gJbXDAJM7KRTza4PWq3Ho0aX1RynpFEMxNhFJPIrXyqbuBkMbqmx9Dpn86IYuEGsQbTcXRmdiq6TGnIQ8Td6eYxM3yJ99rMy9Z3X4hjpfxnTm4YGA9AMf3GBoC3yZE8+t8ZZXdiso+LuML1XpNo4aFn\\\\\\\/qNSNpvjzut7bh3d17yAbhk5a0pJM3nc2P5YQ9hyP3kO5g+qKlxUVGWctVvOLQhHp+VlqifGyUCiwxQyZ6JeCzh8pwBM0YT2ED3loK99WmOsW8HmRUD0GlvdEJbprEfkyEO58OHQTYL77IUH6Smve+eMvWl59SEa9K9a2K2BpyUMdXcoiyoA7bl2BVtew16iKhxYklmffptbNh6H\\\\\\\/Q4oJVp0qpWX\\\\\\\/hwDJu0srE7\\\\\\\/SMVJpM4wX6n52wmfwVhPqS\\\\\\\/w190EiM9bLBVQ88Wyz9K++n27BfsTlzpwXwShF7rn2cqHerIg6vDbujfuKRMnXBWNDPngWIHWQcYMeNiwguh3\\\\\\\/P2J9tsiI987+CMUDSYCdOuc3m7TP5Cv+rm06o7xMrwYxo+ltW\\\\\\\/lr+220Vi6shasQp6nHq6x0Huny0ESlyD+yEgKosviExKkBntnGvOUXEpkPG0F9EB3UN3+Zs14HtJT6iRK7uYwrfIhkQxAj9Q6aN50jDFSknlPTnct5I6z7KMnIxvRNAEw5yhjDModNPVzOcEglS8EI5s3fsalBV4FXofrEdfnK8ed+iI0LEmKiQz57GrhIBlVl\\\\\\\/vImVGPFJ56mAtctazwjZ6MwPhfkj6fY94MfHJAXgucNG+vNvht63DqVf+gIEw2S1lqL39pYIFGCoY+33lkcp75oy7ffOuGX348\\\\\\\/sG+1lwZOg+JiyMVBI4rEALWb6H6oj6MXzf8\\\\\\\/lfWc+EELvVTaz0RDcrZ432J48nHitS8hczNxknD2+e\\\\\\\/WC6kGA95lj9rJl+5Yl8HZR2wCmclcUjAWd4DPw1A3wJZuxDkJ89GFC0TcBhXPbXN7QDLuQygPb6ZqiaOQ3H5JnQ8N8xBrhUi8vnqnZ8+UTIS4YQA17xAuT5BVBDakD51FK25elKdxIeCiY8T8ohBAGOnO6yCsi0MjN8SYgCUacaHv2vk0B8LqMPoYEtUIDqoPtSAvPFshW7vse88EDI+nLrfIkj+xh4KSkvDVODd\\\\\\\/rpzwxb3wgrwCwVCWyla9QQ4JNpNhEk+oGM5ex7E8tF1qpd\\\\\\\/EQbzXp2MNYICDA4GRtVzTieIKofnXKQZzULpUCdcZDbZszU8dPbymkpZ2RMjPvx9YCaW8i99wcU0E4COwO14seIGM\\\\\\\/sUBKdcVpKsy2khgyAs6c4aIkS+5\\\\\\\/PxT0E3PTN6ABi8m9DlgZ7NDGjfV\\\\\\\/SoFHCeCdww4HB+ZbikpECx1aRIAozgEXDZwf+Tzb\\\\\\\/YwRPssUrvXSggTDXBRV83djCm1dm8ZTlxzJoW5tYOCZOWTdjoT6lS5k5ARZwOqnChFhq8mD2Su3P4cI2acpbZ7LKarqz0lP8YkoS1cqmcZxidYrlhyBkHtDSwUF4i2ybs+d+Y3fH4KX8vjI7C4vIxvJw3VErEfGk+W71HBVME+srvQpCmB8yDpr5VdVDmUFoGwi+EzDBaQC4N95S6Tw+AnRLwcmS\\\\\\\/vp+vVP7Ku\\\\\\\/9L6f\\\\\\\/4SESwsoAmKS\\\\\\\/rwxbWzplYG7H48F15ZnXO6wgxR0hNmtcoJu9JdTEu24Wf39t9C10JOoFiLXTDOKEJQrU\\\\\\\/Crtx8\\\\\\\/ARET2Uwnsy78qIsplXtuqvqQrEUodc8V4qHBnvC5l0aMkFcLiksr70NsJFUImpCIJFg0kgyvOZyYp0F5jVFcX3V5BfTYPl6eQjeSbMlLoJ6arO01gZTrEEefqNAD8jbziWx03Yw\\\\\\\/\\\\\\\/8E0BY\\\\\\\/3lFGyYL9vkMAVim8Ogu3kBNB6Ybvd6sUuxHz1SDRtNedlID6UwBISkCzVSl1GraNDsEd5Jk8I9uir55p2RE065GcnxiHWvnUW19zU2JmIlRqMIl93A234DIdXP5xR32yDLTmkbJe2iMwcITui0NiaO+sIw+nSZGu43Qyrrh0wSk9SxFwB4pLMBAEzjU9oUJMZNXhn1ovC1B8lH\\\\\\\/shbPjX+i4IqskYDosykCxhO67roi6bhqsVy+X1\\\\\\\/Vs\\\\\\\/vUixPRMwRnPe8jNFsU0XkZEkVh39wUVJj5Ml91SogJ6cHFkm0yqI7Vnq85F04z5m\\\\\\\/Vcb8hE5\\\\\\\/Qly8kT\\\\\\\/4vo\\\\\\\/X8VqmDZX277R4NCwB1uHJiGFRiRS7haraQM9KtL4nCuP1tWoYL4M931HFDBHjmIuo4nRJ4pLLeRx2bNWcT5UHzJc9fI2ZATrlRLlISr4qVo\\\\\\\/tLcV3D0LDeXbMDNjTXCc5gN024YCUYxc8k3LO1\\\\\\\/TaUCkTbv9tWjTJ7drRsQNAz67TT\\\\\\\/jmWNq\\\\\\\/+BM6WsHMPTsLSk2zQhg3V0Ua16Kkq3tS+VpaOHmeAwq0mXnnIhSou6c7AbVx6HL3mUTGcKmc964UTNt6voz5\\\\\\\/pFWe65GWsHTM\\\\\\\/udIdWAroUfB6dBHUYZBll5\\\\\\\/WnHNA9WIyJopsTkIEfcLJgi+hctyShFvfPCKyDOFkc5uMIXsaVkq\\\\\\\/uxzzifhIX2w+l3iuiVXeR7GdETZnBZGHFU21iGbDUR8bu5iTGxcqzqrTcrQjJ4O9Z6lUwp4F44hUct44PQObxtzWwfes\\\\\\\/sGJEelyNKn5LBsBW9JoiN65ehdvZSDiC5Oi8EmmHqshzmCPiVHtcdYciTE32gZvF4byoPI0L\\\\\\\/2sj+R7ZY8S+05njRauMX8YTmGqK4RBrrwvbmMDJsZcHrsNJ5AWKlufxRYSgp0w7B3sM5sq+TJueEpSnxW2sbyirUD10GwL76IyufMKgjsWPJ98tpFWs1VyUE0C+P4u7fBdQH51mbePgtNbIBpHV8sDWGXVIKHaY6rMaY3oPXI2vF8Psx7xCiXYi+QdTt2kJ5TybvYB7W4hGAhAzGEBt1Gf4UXS\\\\\\\/HW39L6yyFBDMCX1Rj\\\\\\\/kdZzdOI1CICsgukLFgCsc99JSKJZuQZxa8TnpbpTkIbuMzXrmtXUXGucEWp9l4QwIrZsut1FvhkBWJGGlEf5tHeteDLgmWTTbOuDoqTN8wAOyr7rm6x0n3s0jqWKhu8bBZGMr+ONo7SW4\\\\\\\/8TCqqkYrH3aU91LQO7SDpPzsLZyGyqfI3SbuNRC6LUibgfL32XUPDSxGM6O8SO+s+zUIrVhnqS\\\\\\\/AQ\\\\\\\/u\\\\\\\/hE99Z5WcOENUTWsYdYhOgD6shffA5Pt7V7U\\\\\\\/wbZrVGHCvPsmnwJ5YYsl6+72KZ7+VzXQSEi2fy7JkybfMN3lob5tHyFJpv6nvFsFVWWw3Gnl5cxsIWpgbnoasK5Kd9Un6ZEyZ++rMxWXzy7IbpChD0RoLqS3\\\\\\\/tFCzS9m5oqdTFgrUfxta\\\\\\\/LLlWaKZMhZWG+so30+Ghug9M0ZzidiSEPeIwqgLgnzBdYdoZ6uYYR1S0GJzVSIZ5gljycM7FVotksm\\\\\\\/RyM8CCWGriKsleHoXbzY8Nd7+Zw0QIe8EWzfZIenRAp7JaYkwQ3bv2hZcyJwJKxGywnUczQjg4N0MZuqoI5odakr7ZV32cQCqVny3tnP3yPlgYExT4lO924JhY0e1FmfCUmABrn8iqriP+HNkyTUoGSH3uP7UNcjChmg58z29GYU1feQ5K527N5kVaXwcrY6eleRm4itCLTRFH+pZhkHLm8yzgHcfSoAknR1rT8Ld+shu0uUB\\\\\\\/\\\\\\\/OMas0Ev\\\\\\\/KSJWHChGglC0so1dMFHNT12+gDsbbx58OHaR9bHfmKjDLs93rTJdxNDh7\\\\\\\/NA2pKiurIq0RLQ3OxNukD+iuuh2++Vl7UCDmaRP3aLUfqqH2k1AsQcV\\\\\\\/t6vQIW9agMXrI0\\\\\\\/iV0Sn341tGQi9Yit1o2mLIitQWtNSFe+xI26SicFPSzsyQr+QqPdW2hUlamG267NhOEHOmjL+AmdvsVh\\\\\\\/TW734ArzW1\\\\\\\/D9D3uenBTd2swW9kFAFiYWYgfW+omhTBS5sfGgne3VnafCrS2cghTN6LoOffJIFwY3otfIZ7wKgHlT\\\\\\\/5kNwn4XzTmHzUH0XzF1kqkwfSBxcNOmpjcgIriJsJ4iMN2TJZFnnQb3Nl9d9nvoLuZvBA3cK7AieFZCcOW5LUwZNMRSjXfLOogIdkGs6gHP3FxV0esQyIwgoVXLTgx6kDL3+Po\\\\\\\/n2ywvnJO6uMkmqFcWOmui1XHS1jNMFhaScPCEQTWztydS5n+vdDyZF1QxaO\\\\\\\/P8wtRtKqjY+kG68mmhPO9aekAp5ChOZ14KW5BtlOOcwbI1bG2rnlaeURPQHyqdMg8d8ZjrbnvUJD7jfZF5f6rShLaKcHz5T0H\\\\\\\/tXPFjWkTK72XsPZrnEUV56t\\\\\\\/wEPyuu\\\\\\\/2CQFkBdif7gdMEqbA2fZBoktwMq3EKFwJmGlyFqhUu9ybW1EuRg0b+fHg20hbvaMRH\\\\\\\/klFg8iyFPWKVPB6Q1DyMBBWj215VtjPHtEO0ZMg8yMoP8vr8R6m\\\\\\\/VGdGEwNde3G9EQqg34eSm5rJwtFQbxjHZtSna\\\\\\\/W40sPNFcdybafp+1tfKT8YpmOUln6IRzOjQyBMoZXzwWkYMduBCFLeqRshKNxF7Cv2jfmZ+aFXC6Rjdm1HAg3aYBat7JrAqiC2IACxanmvCIH0oOUFfx+y1iuGSa7F6RWHOVi5C8cSDJda856MOi2KtOHRS530tVJSdG\\\\\\\/At3WTdBe2XyzQMwIn8ib8AZ3031xrGL2rdWNhMgXRYrmSorV9MdricQBfQ2nF7ivk1hhalXORmSPAcs6UsyFVDrt2qY1PcthlOZNttlmu9PERdZGr3WHAa+u5Md9zKYVdpSGcmbtbb+c\\\\\\\/v4XTM5Opw4BR8kblgM38PjV8wlsKEOCqqW7wdAUZqqPGcKoltI1fBifQeNlKzBl3SeF6CUMAmrAglhB0w7FncVWGDhUidbiJVv5c95+W1F3NsZAPv0Al2\\\\\\\/AtJFaiCU0AOpzBEXkgmM5GhczQ7qlbPIaI4qMXzgdz1rIVHwymgcAtd81REmYsncidZdqBvC2pZC6f\\\\\\\/obDNGI5otLdkCQVFF6SItNwkTGUccSHEzEsVDg6Xo2HwImMgFz\\\\\\\/rdIzQEIGRPW4BSFrfw7wDD5DAiTpUpgIbWLxXoV5Fbjq4AMEKjhyrNNzAtZZAMxnMwPPRxmM3hq2L2BRL+WTvJRbdiap\\\\\\\/yjhtUBAMhbjzQwNwVtm4zFt0lFihMjNeWzHGHaq3+NBS\\\\\\\/KGp+Os4SDvSkQerjg6NARex5GcB34roCczJjwPFk8BL83gU2CCRWi9spAfgiGfh8jTIePeGJzNh\\\\\\\/9fKi4iY89S01HFtP5nkRbaBgGbpj8DydxdfIyN3CW4\\\\\\\/8ao6klJnc3swDZaqZEMmEOHkL6FRFvmjDzagl8tDlfuZ0ORxiTG2zjOb5dTSV1dWtLSK0OWmDpEodrgZbiVTvNwOIAbgoUZaIe6A8nQa4JPRzZAwtSw6PB5FrlKmTMJ1sh3gklkFeJgOAp9+1JUxcyc46Q7B1KBFIQ+7EXyRSNHXti8jSMzVlJQTa1d5OyDvJoP+JosbWBYsKmnR8uj5htxE4VhvT7+y7PiHTdm2HVHU1ULL1Uxv5+aNtvjs\\\\\\\/v0vlcr1p5Yq5UeSjniRo2OgrIFAc\\\\\\\/sPkvPlSguGqvzo38mMLRF9lAF\\\\\\\/fs144oWTRLaQweOma5XbDW2mfVmGfaTy3IPgewULa0EP\\\\\\\/3qBYJSUXn3q0A1Egdk+u12SZ3clYmc8wfnXSg=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Pbj7RHzXQLuoJxPwrH9fdg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f1fc5cf4f1f3006d42ef66d27e7cf9a9327a4ea1e03db7542eb9c2eaae8907e4\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228sxThU7XYbowBfHbJ7RdhKXvs60K4k\\\\\\\/bWjwdNUeMmhakpMtfAMRP\\\\\\\/1\\\\\\\/VFbNwXuBaLFXgxJTT+Hl5qxgyIPk5lpnl\\\\\\\/P+9\\\\\\\/r5H0pz3uOwWh6kPj5LZpatvrh1Jw32YpxpbHt2GiQ6bqp0iXjmDrK38I65cmz3OxuON3ZLSl4nB4RP+JkzMhHBHVhV4AYdRZMdGv8B3HV4ziwE6fSHoNSW4gRmUtz+9EURsNpSwd5LgKXLrSSg+os90LBrDDbpjABwZuBMYcoB26yaQdBoKyoqqo4KNRg\\\\\\\/iVgcna5bAj6DZhP0fsGhSicoeyOnvW1hNvV56PJ+xa5ocDz+E7GtHgPgwbPJC7wR9Ka0XvVfyreI5eP7RJ+W4zL21YFZFIRtukblMiE47irNnclfphwfuBO4q3Kd8y4W3nxUvaCTLpNun8BPS3A477mu4dC2pZz3AXgdQV3NdREhnijAptyHhfqjF9mauihSRwBveOAFh391fxS1nLUstzlI1eFXjG2\\\\\\\/GWYRhjNMxMYv6rkX6iVkuyk8lW8CpE9kVNOshO2G4FoYts4VJHV+ixOLLv483Xt26kYsSQxOo60nwsyGLYBPf\\\\\\\/R7trbitfowV85JA+X2xQ92iMWt09GpqjMhW3B9Rf0nq4+Iy0YYgM\\\\\\\/mk93HYTNPpgtoggKFJqENhP\\\\\\\/TYXRuP5ATr318abfohaoqGyjK+R6sBIeGxBvUSHg5K6ohvF6XwSpIDoec3gJx0of\\\\\\\/OfrOb2UHU+Os+czmBZzTpt7NshtSllCrE304Da+69ttglEEPUZ8YWMfqLGroTylG\\\\\\\/tcsOkO3CZda98wykWzdYe4J5tuVY7z9TUCcx\\\\\\\/qqK\\\\\\\/Xbl3xCrH4KdyUvQUwp\\\\\\\/2n7n1erXr73sTl5T5SfogbYQeA+3L0H8korAsIyWGE7b8SywjSBGSA2W5\\\\\\\/7soTeIciokOk72Mm4F66UFGzsYGyXE624pT9OaM6XVVNc8Ny\\\\\\\/kjCK5kxGc5e0IS9WEI+m6FfqtzDbW3h9VW2dWuOfLxXWOS9VVVOvzkQIR6eGVmEjkbtfKyd2T5cWp6xjsdzk9Gpc8gjC49EcSuTBGUKjT8byn\\\\\\\/kULoOHgPWKNWliBWDHXnhEG6Yx0i76JPBALa9\\\\\\\/ZiDcX1JOaQ4zv34Y31BzjiTb8\\\\\\\/ncJNae1t0uIb2l3Ev91wEuF6RJXJZKTxOukFLn7T1fAcpfMtDoLVcDBQklwQaHrkvn8zAaMbPCk\\\\\\\/c4wz+8kVvxutS0KWHYhfn1JCFP4Hb7VsfYV\\\\\\\/T+BB\\\\\\\/aHZHMMAi10OYmr2CedC+s7nuPowtvfduJMj4As3y5JCcrt\\\\\\\/g9LhRsf\\\\\\\/wZvcrJJIELb1Q1k9jV\\\\\\\/0UJsfPDNxsYCD6hugW3DRdX5SyNo7rhHUacGZxNBLyn+9v1XE1XPU\\\\\\\/ww83qMEtnyjKr\\\\\\\/V+3uXlI2YPTlHYVX9r9avA1vRLQIzMFBRBvqMYqBSWu0lqAyq+AFKoWiPEHZ6bfQ2\\\\\\\/iulBLTeJAgdHPqyUi6yf\\\\\\\/BqCxYQze6w5yynEomu7Z2XBrG2CHljJ\\\\\\\/PBO2GTI7zTa+agtyyJp+cqIlDSz1Hw2DUiAMQypqF4owPxi4c0P0sn4RSFj+ktLeWnHNJqafF9dUPThmONwao93ANtRIhUtlhn9x5DchiCjZ\\\\\\\/sTb72hb30Ll+i4A7psOJnUn154SyOlwvwtHwstnCT9LUapdXDKN+pdcNaAsOCBuUj6V4IfqftzsQk+WHx0UxrDs\\\\\\\/T2WzBICmJM02HAeiaTszc3WpAaB6OQB5OvB5oFqVNrS813zxbGeot0xy8R22qKbxZFsKFVY5LHs+KSh7Q9\\\\\\\/ynRpXDF12l870QA4hLnLsXIgLAGP2ntZeGitcHwDiOt9R0YZBz4Bkyi2i4f06edJC3yWd1pPrnZdOQERHvLbpxWB2EjLkfFnYtOU2zcHkYwU2rA6g0Xn916h+8MELUucoeFXQ\\\\\\\/mXvTYGASLKEcG8vE8fQ9Z9mc2xPV3FTapa37JrQ1\\\\\\\/+QaBjxnKlkO3n0Mreli3RybcF3EEiQXWv1OPpRLSBAY3Ozr8CoreX3\\\\\\\/0Ca802kXz4GPvGH8IYhMLyRZ6nLUbZmBWksTPaGVbuRRsnPHht2MXJpLVM1eAdfbC98TAtU\\\\\\\/6o7296HsLuQKTpwrmkJMeBV+BFlqj2eYQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022cVjpgRVuATtlo8XDD0\\\\\\\/zHA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a1b373e14e26de1a1c55481c35e0b3cc29eb73bb1ee7e24bd0241e24e4e141a1\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459551 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/10/1795" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-10MTS1Cmw0z7frhEAlJJW7cHzr6qPK479DOhfbWBTac" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/10/1795" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ib-coLt-hBCxn-XNeLdvBc9tpVMcDU9MflFj5ggRa3A" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/10/1785" title="The Drosophila Enhancer of split Gene Complex: Architecture and Coordinate Regulation by Notch, Cohesin, and Polycomb Group Proteins" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/10/1809" title="Insights into the Evolution of Cotton Diploids and Polyploids from Whole-Genome Re-sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459551" id="node459551" data-pisa="ggg;3/10/1795" data-pisa-master="ggg;g3.113.007161" data-apath="/ggg/3/10/1795.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Imputation-Based Genomic Coverage Assessments of Current Human Genotyping Arrays</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sarah C. Nelson</span>, <span class="highwire-citation-author" data-delta="1">Kimberly F. Doheny</span>, <span class="highwire-citation-author" data-delta="2">Elizabeth W. Pugh</span>, <span class="highwire-citation-author" data-delta="3">Jane M. Romm</span>, <span class="highwire-citation-author" data-delta="4">Hua Ling</span>, <span class="highwire-citation-author" data-delta="5">Cecelia A. Laurie</span>, <span class="highwire-citation-author" data-delta="6">Sharon R. Browning</span>, <span class="highwire-citation-author" data-delta="7">Bruce S. Weir</span> and <span class="highwire-citation-author" data-delta="8">Cathy C. Laurie</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1795-1807;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007161 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459551" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sarah C. Nelson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington, 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BC.%2BNelson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nelson%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASarah%2BC.%2BNelson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sarahcn@uw.edu" class="" data-icon-position="" data-hide-link-title="0">sarahcn@uw.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kimberly F. Doheny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Inherited Disease Research, Johns Hopkins University School of Medicine, Baltimore, Maryland, 21224</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kimberly%2BF.%2BDoheny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doheny%20KF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKimberly%2BF.%2BDoheny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Elizabeth W. Pugh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Inherited Disease Research, Johns Hopkins University School of Medicine, Baltimore, Maryland, 21224</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BW.%2BPugh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pugh%20EW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElizabeth%2BW.%2BPugh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jane M. Romm </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Inherited Disease Research, Johns Hopkins University School of Medicine, Baltimore, Maryland, 21224</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jane%2BM.%2BRomm%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Romm%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJane%2BM.%2BRomm%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hua Ling </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Inherited Disease Research, Johns Hopkins University School of Medicine, Baltimore, Maryland, 21224</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hua%2BLing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ling%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHua%2BLing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Cecelia A. Laurie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington, 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cecelia%2BA.%2BLaurie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laurie%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACecelia%2BA.%2BLaurie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Sharon R. Browning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington, 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sharon%2BR.%2BBrowning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Browning%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASharon%2BR.%2BBrowning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Bruce S. Weir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington, 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruce%2BS.%2BWeir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weir%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABruce%2BS.%2BWeir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Cathy C. Laurie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington, 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cathy%2BC.%2BLaurie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laurie%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACathy%2BC.%2BLaurie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/10/1795" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459551" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459551/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/10/1795.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459551" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459551/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/10/1795.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459551" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459551/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/10/1795.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459551" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459551/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/10/1795.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459551" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459551/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Microarray single-nucleotide polymorphism genotyping, combined with imputation of untyped variants, has been widely adopted as an efficient means to interrogate variation across the human genome. “Genomic coverage” is the total proportion of genomic variation captured by an array, either by direct observation or through an indirect means such as linkage disequilibrium or imputation. We have performed imputation-based genomic coverage assessments of eight current genotyping arrays that assay from ~0.3 to ~5 million variants. Coverage was determined separately in each of the four continental ancestry groups in the 1000 Genomes Project phase 1 release. We used the subset of 1000 Genomes variants present on each array to impute the remaining variants and assessed coverage based on correlation between imputed and observed allelic dosages. More than 75% of common variants (minor allele frequency &gt; 0.05) are covered by all arrays in all groups except for African ancestry, and up to ~90% in all ancestries for the highest density arrays. In contrast, less than 40% of less common variants (0.01 &lt; minor allele frequency &lt; 0.05) are covered by low density arrays in all ancestries and 50–80% in high density arrays, depending on ancestry. We also calculated genome-wide power to detect variant-trait association in a case-control design, across varying sample sizes, effect sizes, and minor allele frequency ranges, and compare these array-based power estimates with a hypothetical array that would type all variants in 1000 Genomes. These imputation-based genomic coverage and power analyses are intended as a practical guide to researchers planning genetic studies.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genome-wide-association-study" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-association-study" rel="nofollow">genome-wide association study</a></li><li class="kwd"><a href="/keyword/genomic-coverage" class="hw-term hw-article-keyword hw-article-keyword-genomic-coverage" rel="nofollow">genomic coverage</a></li><li class="kwd"><a href="/keyword/power" class="hw-term hw-article-keyword hw-article-keyword-power" rel="nofollow">power</a></li><li class="kwd"><a href="/keyword/snp-microarrays" class="hw-term hw-article-keyword hw-article-keyword-snp-microarrays" rel="nofollow">SNP microarrays</a></li></ul><p id="p-4">Microarray genotyping has been widely adopted as an efficient means to interrogate variation across the human genome. Such arrays are the technological foundation of genome-wide association studies (GWAS), which have uncovered numerous genetic associations for a wide variety of traits and diseases (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Manolio 2010</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Visscher <em>et al.</em> 2012</a>). The arrays used in GWAS often are designed to genotype a maximally informative set of variants that capture, or “tag,” a substantial proportion of genome-wide variation (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Carlson <em>et al.</em> 2004</a>). “Genomic coverage” refers to the total proportion of genomic variation captured by an array, either directly, via genotyping, or indirectly, through linkage disequilibrium (LD) with one or more genotyped variants (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barrett and Cardon 2006</a>).</p><p id="p-5">The determination of whether an untyped variant is “captured” by a genotyped variant can be based on one of two measures. The first measure is the maximum pairwise squared correlation (<em>r<sup>2</sup></em>) between discrete allelic dosages at the untyped variant and any of the genotyped variants. Thus one way to describe genomic coverage is as the fraction of variants in a reference set having a maximum <em>r<sup>2</sup></em> with array variants above a given threshold (<em>e.g.</em>, <em>r<sup>2</sup></em> greater than 0.8). Genomic coverage for the first generation of SNP microarrays typically was estimated using this pairwise LD paradigm and with the HapMap Project (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Frazer <em>et al.</em> 2007</a>) as the reference set (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Barrett and Cardon 2006</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Li <em>et al.</em> 2008</a>).</p><p id="p-6">A second measure for determining whether an untyped variant is captured by an array involves using array variants to impute, or predict, the genotype at the untyped variant. Imputation yields a different <em>r<sup>2</sup></em> from the pairwise LD <em>r<sup>2</sup></em> metric described previously. “Imputation <em>r<sup>2</sup></em>” is the squared correlation between the actual (discrete) allelic dosage at a genomic variant and the imputed (continuous) allelic dosage, over a defined set of samples. For a variant with two alleles <em>A</em> and <em>a</em>, the imputed dosage is calculated as 2P(<em>AA</em>) + P(<em>Aa</em>), where P(<em>AA</em>) and P(<em>Aa</em>) are posterior genotype probabilities from imputation. Under this imputation paradigm, genomic coverage can be reported as the fraction of variants in a reference set with imputation <em>r<sup>2</sup></em> above a given threshold (again, usually greater than 0.8) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hoffmann <em>et al.</em> 2011b</a>). The reference set of variants used to determine imputation-based genomic coverage then becomes the variants present in an imputation reference panel such as the 1000 Genomes Project (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abecasis <em>at al</em>. 2012</a>). Imputation-based genomic coverage is especially relevant to current practice in association studies, in which untyped variants in the 1000 Genomes Project are routinely imputed from a scaffold of array genotypes and tested for association with traits of interest (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Marchini and Howie 2010</a>).</p><p id="p-7">An important application of <em>r<sup>2</sup></em> coverage metrics is to estimate the power to detect an association when the causal locus is not observed directly. <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Pritchard and Przeworski (2001)</a> found that a sample size of N for direct observation of genotypes at a causal locus gives approximately the same power as a sample size of N/<em>r<sup>2</sup></em> for observed genotypes at a marker locus, where <em>r<sup>2</sup></em> is the squared correlation between the discrete allele dosages at the causal and marker loci. This relationship has been used to estimate the genome-wide power to detect an association with various SNP arrays, using the maximum pairwise <em>r<sup>2</sup></em> for each variant in a genome-wide reference set such as the HapMap (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Li <em>et al.</em> 2008</a>) or 1000 Genomes Projects (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lindquist <em>et al.</em> 2013</a>). For genome-wide power, an average is taken over all variants in the reference set, using variant-specific allelic frequency and <em>r<sup>2</sup></em> values (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Jorgenson and Witte 2006</a>). Here we show that the relationship between power and <em>r<sup>2</sup></em> also holds when using imputation <em>r<sup>2</sup></em>, the squared correlation between the discrete allelic dosage at the causal locus and the imputed allelic dosage (a continuous variable), assuming a linear relationship. We also provide genome-wide power estimates with imputation <em>r<sup>2</sup></em> values, estimated for each ancestry group in a 1000 Genomes Project reference set.</p><p id="p-8">Genomic coverage and power have been previously described for a number of arrays, using either HapMap (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Li <em>et al.</em> 2008</a>) or 1000 Genomes Project pilot data (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Lindquist <em>et al.</em> 2013</a>) as reference sets and using maximum pairwise <em>r<sup>2</sup></em>. Updated coverage assessments are provided here using new arrays, a denser and more robust data release from the 1000 Genomes Project, and imputation-based <em>r<sup>2</sup></em>. New arrays have been developed in response to both the changing needs of the genetic research community and technological advances in genotyping. While the design of the first generation of arrays was informed primarily by the HapMap Project (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Frazer <em>et al.</em> 2007</a>), array content has since expanded to include variants cataloged by the 1000 Genomes Project (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Abecasis <em>et al.</em> 2012</a>) and the Exome Sequencing Project (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Tennessen <em>et al.</em> 2012</a>), in addition to pharmacogenetic and expression quantitative trait loci markers.</p><p id="p-9">Despite a growing focus on rare variants, the tagging, LD-based variant selection scheme is still of prime importance in array design. Many arrays still include a core, or “backbone,” of GWAS markers selected using the same principles as the first generation of arrays. Each of the major commercial vendors has recently released (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Affymetrix 2012</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Illumina 2012</a>) products with a foundation of approximately 240,000 GWAS tagging markers: the Illumina (<a href="http://www.illumina.com">www.illumina.com</a>) Infinium HumanCore and the Affymetrix (<a href="http://www.affymetrix.com">www.affymetrix.com</a>) Axiom Biobank. Although vendors frequently provide genomic coverage estimates in product documentation, the methods used are often different and/or not well-described, making it difficult to objectively compare across arrays. Here we have used a consistent, imputation-based approach to evaluate genomic coverage and power across eight different genotyping arrays. These analyses are intended to function as a practical guide to researchers who are planning genetic studies. The results may be used to compare the cost efficiency of commercially available arrays, to assess the power to detect trait associations using imputed allelic dosages, and to design custom arrays. They also provide an evaluation of the accuracy of imputation for different categories of minor allele frequency (MAF).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genomic coverage</h3><p id="p-10">We assessed genomic coverage with the design outlined in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> by using the publicly available 1000 Genomes Project (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Abecasis <em>et al.</em> 2012</a>) phase 1 integrated variant set, in variant call format (VCF, available at <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20110521/">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20110521/</a>), comprising 1,092 samples from 14 populations (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). For each array in turn, we selected 1000 Genomes Project variants from the VCF file having the same position as variants on the array. The array variants were first pre-phased with SHAPEIT2 software (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Delaneau <em>et al.</em> 2013</a>) and then used to impute the remaining 1000 Genomes variants, with IMPUTE2 software (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Howie <em>et al.</em> 2011</a>). The 1092 samples were randomly divided into 10 batches (balancing across populations) for these pre-phasing and imputation steps, with the remaining samples serving as a worldwide imputation reference panel. After imputation, the ten batches were combined to calculate accuracy metrics within each ancestry group. This approach is similar to the leave-one-out strategy used by <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Lindquist <em>et al.</em> (2013)</a> to estimate imputation-based genomic coverage.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/10/1795/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Study design. Schematic of the method used to assess genomic coverage of each array. Throughout the diagram, bold solid lines around boxes indicate observed genotypes (i.e., variant calls from the 1000 Genomes Project phase 1 integrated release, version 3), whereas dashed lines indicate imputed genotypes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-116874296" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Study design. Schematic of the method used to assess genomic coverage of each array. Throughout the diagram, bold solid lines around boxes indicate observed genotypes (&lt;em&gt;i.e.&lt;/em&gt;, variant calls from the 1000 Genomes Project phase 1 integrated release, version 3), whereas dashed lines indicate imputed genotypes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/10/1795/F1.medium.gif" width="440" height="319"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/10/1795/F1.medium.gif" width="440" height="319"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/10/1795/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/10/1795/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463413" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Study design. Schematic of the method used to assess genomic coverage of each array. Throughout the diagram, bold solid lines around boxes indicate observed genotypes (<em>i.e.</em>, variant calls from the 1000 Genomes Project phase 1 integrated release, version 3), whereas dashed lines indicate imputed genotypes.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463407/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463407/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Samples in the 1000 Genomes Project phase I integrated variant set</span><div class="sb-div caption-clear"></div></div></div><p id="p-13">To assess imputation accuracy and by extension genomic coverage, we compared imputed results at all the nonarray variants to observed genotypes from the initial VCF files. These comparisons were performed separately in the four different ancestry groups (African, “AFR;” American, “AMR;” Asian, “ASN;” and European, “EUR”) and restricted to variants with at least two copies of the minor allele. For each imputed variant we calculated three metrics: (1) the squared correlation between observed and imputed allelic dosage, which we call “imputation <em>r<sup>2</sup></em>”; (2) the concordance between observed and most likely imputed genotype, the “genotype concordance”; and (3) the concordance between observed and most likely imputed genotype, when at least one of those two genotypes contains one or two copies of the minor allele, which we call “minor allele (MA) concordance.” Array (observed) variants are included in these metrics summaries and are given imputation <em>r<sup>2</sup></em>, genotype concordance, and MA concordance values of 1. A more detailed account of data preparation, pre-phasing, imputation, and metrics calculation is available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007161/-/DC1/007161SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007161/-/DC1/FileS1.pdf">File S1</a>.</p><p id="p-14">We focus on imputation <em>r<sup>2</sup></em> as the primary coverage metric mainly due to its simple relationship to power, in addition to the following advantages: (1) precedent in the literature for evaluating both imputation accuracy (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Howie <em>et al.</em> 2011</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">2012</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Delaneau <em>et al.</em> 2013</a>) and array coverage (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Hoffmann <em>et al.</em> 2011a</a>,<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">b</a>; <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Lindquist <em>et al.</em> 2013</a>); (2) less sensitivity to allele frequency than concordance; (3) similarity to information metrics commonly reported by imputation software (for a review, see <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Marchini and Howie 2010</a>); and (4) incorporation of imputation uncertainty by using expected allelic dosage rather than most likely genotype. However, one important caveat is that <em>r<sup>2</sup></em> has high variance at low MAF (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Evangelou and Ioannidis 2013</a>). Overall genotype concordance is also a widely used metric that is easily interpretable, although it ignores imputation uncertainty and is very sensitive to allele frequency, as low MAF variants may yield high concordance purely by chance (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lin <em>et al.</em> 2010</a>). MA concordance helps correct for sensitivity to MAF by not counting correctly imputed major homozygous genotypes. Thus, although we have used imputation <em>r<sup>2</sup></em> as our primary coverage metric and the basis for the power analyses, we also provide coverage in terms of MA concordance and concordance to enable downstream users the flexibility to focus on one of these alternative metrics.</p></div><div id="sec-3" class="subsection"><h3>Power analyses</h3><p id="p-15">The power to detect a variant-trait association was calculated for a case-control study under the assumption of an additive genetic model, using the following parameters, where <em>A</em> is the risk allele and <em>a</em> is the alternate allele: genotype relative risk (GRR, of <em>Aa</em> relative to <em>aa</em>) from 1.1 to 1.4 in increments of 0.1; disease prevalence (K) of 0.05; frequency of the risk allele (p) assumed in each case to be the minor allele; the sample size (N cases and N controls) from N = 1000 to 10,000 in increments of 50; and a significance level of 5 × 10<sup>−8</sup> (appropriate for genome-wide testing). The assumption of an additive model is based on how imputed dosages are used in association tests, and the range of GRR is based on a summary of effect estimates from genome-wide association studies (<a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Lindquist <em>et al.</em> 2013</a>). Power was calculated for an allelic association test (χ<sup>2</sup> with 1 degree of freedom) using the non-centrality parameter in the Appendix. Sample size was N for observed (array) variants and N/(imputation <em>r<sup>2</sup></em>) for imputed variants. A genome-wide power estimate was obtained for each array and parameter set by averaging over all variants in a reference set, as indicated by <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Jorgenson and Witte (2006)</a>. Here, the reference set consists of 1000 Genomes variants with at least two copies of the minor allele (or a subset thereof). In principle, power would be calculated for each individual variant (using its individual MAF and <em>r<sup>2</sup></em> values) and averaged over all variants in the set. In practice, to reduce computation, we binned all autosomal variants by a combination of MAF and <em>r<sup>2</sup></em> values, calculated power for the mean values of MAF and <em>r<sup>2</sup></em> within each bin, and took the mean over bins weighted by the fraction of variants in each bin, as in <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Li <em>et al.</em> (2008)</a>. Binning for MAF was from 0 to 0.05 in increments of 0.005, from 0.05 to 0.10 in increments of 0.01, and from 0.1 to 0.5 in increments of 0.05. Binning for <em>r<sup>2</sup></em> was from 0 to 1 in increments of 0.05.</p></div></div><div class="section results-discussion" id="sec-4"><h2 class="">Results and Discussion</h2><p id="p-16">We have assessed imputation-based genomic coverage for eight commercially available genotyping arrays using the four ancestry groups in the 1000 Genomes Project phase 1 release. The number of assays and unique genomic positions assayed by each array are summarized in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. Only those array variants also found in the 1000 Genomes phase 1 integrated variant set were able to inform the imputation and thus contribute to these genomic coverage estimates. To see how this fact may differentially impact arrays, in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> we also report (1) the percent overlap with 1000 Genomes at any MAF and (2) the percent overlap with 1000 Genomes at the requisite MAF to be included in the imputation (<em>i.e.</em>, at least two copies of the minor allele observed in any one of the four ancestry groups). As may be expected, the overlap with 1000 Genomes for arrays with a high proportion of rare and exome variants is less than for arrays with less content devoted to rare and exome variants. The possible impact of the differential 1000 Genomes overlap across arrays on the estimated genomic coverage appears to be small (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007161/-/DC1/FileS1.pdf">File S1</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463475/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463475/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Array summaries</span><div class="sb-div caption-clear"></div></div></div><p id="p-19"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows the fraction of variants passing an imputation <em>r<sup>2</sup></em> threshold of 0.8, by MAF bin and ancestry group, whereas <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows the mean MA concordance. Genomic coverage assessments commonly use the MAF groupings of &gt; 0.01 and &gt; 0.05, which we have shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, C and D</a>, respectively. All the data plotted in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> are presented numerically in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>, which also include tabular summaries of mean imputation <em>r<sup>2</sup></em> and genotype concordance and the counts of variants in each MAF bin, as this count differs by ancestry group. Line plots of mean imputation <em>r<sup>2</sup></em> and mean genotype concordance are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007161/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007161/-/DC1/FigureS2.pdf">Figure S2</a>. Plots organized by ancestry panel rather than by metric are available for AFR in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007161/-/DC1/FigureS3.pdf">Figure S3</a>, AMR in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007161/-/DC1/FigureS4.pdf">Figure S4</a>, ASN in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007161/-/DC1/FigureS5.pdf">Figure S5</a>, and EUR in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007161/-/DC1/FigureS6.pdf">Figure S6</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/10/1795/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fraction of variants passing an imputation r2 threshold of 0.8, by MAF bin and ancestry group. The imputation r2 metric plotted here is the squared correlation between imputed and observed allelic dosage in the samples comprising the ancestry group. The y-axis is the proportion of variants (imputed and observed) with imputation r2 &#x2265; 0.8, restricted to variants with at least two copies of the minor allele in the given ancestry group. The x-axis position of each array corresponds to the number of unique positions assayed by that array (see Table 2, column 5). Thus. the order of the arrays on each axis is as follows: HumanCore, HumanCore+Exome, Axiom Biobank, OmniExpress, Axiom World Array 4, Omni2.5M, Omni2.5M+Exome, and Omni5M. Panel (A) is for variants with at least two copies of the minor allele and MAF &#x2264; 0.01, (B) for 0.01 &lt; MAF &#x2264; 0.05, (C) for MAF &gt; 0.01, and (D) for MAF &gt; 0.05." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-116874296" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Fraction of variants passing an imputation &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; threshold of 0.8, by MAF bin and ancestry group. The imputation &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; metric plotted here is the squared correlation between imputed and observed allelic dosage in the samples comprising the ancestry group. The y-axis is the proportion of variants (imputed and observed) with imputation &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; &#x2265; 0.8, restricted to variants with at least two copies of the minor allele in the given ancestry group. The x-axis position of each array corresponds to the number of unique positions assayed by that array (see Table 2, column 5). Thus. the order of the arrays on each axis is as follows: HumanCore, HumanCore+Exome, Axiom Biobank, OmniExpress, Axiom World Array 4, Omni2.5M, Omni2.5M+Exome, and Omni5M. Panel (A) is for variants with at least two copies of the minor allele and MAF &#x2264; 0.01, (B) for 0.01 &lt; MAF &#x2264; 0.05, (C) for MAF &gt; 0.01, and (D) for MAF &gt; 0.05.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/10/1795/F2.medium.gif" width="430" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/10/1795/F2.medium.gif" width="430" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/10/1795/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/10/1795/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463432" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Fraction of variants passing an imputation <em>r<sup>2</sup></em> threshold of 0.8, by MAF bin and ancestry group. The imputation <em>r<sup>2</sup></em> metric plotted here is the squared correlation between imputed and observed allelic dosage in the samples comprising the ancestry group. The y-axis is the proportion of variants (imputed and observed) with imputation <em>r<sup>2</sup></em> ≥ 0.8, restricted to variants with at least two copies of the minor allele in the given ancestry group. The x-axis position of each array corresponds to the number of unique positions assayed by that array (see <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, column 5). Thus. the order of the arrays on each axis is as follows: HumanCore, HumanCore+Exome, Axiom Biobank, OmniExpress, Axiom World Array 4, Omni2.5M, Omni2.5M+Exome, and Omni5M. Panel (A) is for variants with at least two copies of the minor allele and MAF ≤ 0.01, (B) for 0.01 &lt; MAF ≤ 0.05, (C) for MAF &gt; 0.01, and (D) for MAF &gt; 0.05.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/10/1795/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mean MA concordance, by MAF bin and ancestry group. The y-axis values are mean MA concordance in samples comprising the given ancestry group. MA concordance is defined as the concordance (percent agreement) between observed and most likely imputed genotype, when at least one of those two genotypes contains one or two copies of the minor allele. Variants were restricted to those with at least two copies of the minor allele in the given ancestry group. The x-axis position of each array corresponds to the number of unique positions assayed by that array (see Table 2, column 5). Thus the order of the arrays on each axis is as follows: HumanCore, HumanCore+Exome, Axiom Biobank, OmniExpress, Axiom World Array 4, Omni2.5M, Omni2.5M+Exome, and Omni5M. (A) Variants with at least two copies of the minor allele and MAF &#x2264; 0.01, (B) for 0.01 &lt; MAF &#x2264; 0.05, (C) for MAF &gt; 0.01, and (D) for MAF &gt; 0.05." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-116874296" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Mean MA concordance, by MAF bin and ancestry group. The y-axis values are mean MA concordance in samples comprising the given ancestry group. MA concordance is defined as the concordance (percent agreement) between observed and most likely imputed genotype, when at least one of those two genotypes contains one or two copies of the minor allele. Variants were restricted to those with at least two copies of the minor allele in the given ancestry group. The x-axis position of each array corresponds to the number of unique positions assayed by that array (see Table 2, column 5). Thus the order of the arrays on each axis is as follows: HumanCore, HumanCore+Exome, Axiom Biobank, OmniExpress, Axiom World Array 4, Omni2.5M, Omni2.5M+Exome, and Omni5M. (A) Variants with at least two copies of the minor allele and MAF &#x2264; 0.01, (B) for 0.01 &lt; MAF &#x2264; 0.05, (C) for MAF &gt; 0.01, and (D) for MAF &gt; 0.05.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/10/1795/F3.medium.gif" width="425" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/10/1795/F3.medium.gif" width="425" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/10/1795/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/10/1795/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463464" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-21" class="first-child">Mean MA concordance, by MAF bin and ancestry group. The y-axis values are mean MA concordance in samples comprising the given ancestry group. MA concordance is defined as the concordance (percent agreement) between observed and most likely imputed genotype, when at least one of those two genotypes contains one or two copies of the minor allele. Variants were restricted to those with at least two copies of the minor allele in the given ancestry group. The x-axis position of each array corresponds to the number of unique positions assayed by that array (see <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>, column 5). Thus the order of the arrays on each axis is as follows: HumanCore, HumanCore+Exome, Axiom Biobank, OmniExpress, Axiom World Array 4, Omni2.5M, Omni2.5M+Exome, and Omni5M. (A) Variants with at least two copies of the minor allele and MAF ≤ 0.01, (B) for 0.01 &lt; MAF ≤ 0.05, (C) for MAF &gt; 0.01, and (D) for MAF &gt; 0.05.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463469/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463469/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Genome-wide coverage estimates for all eight arrays, in AFR and AMR ancestry groups</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463474/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463474/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Genome-wide coverage estimates for all eight arrays, in ASN and EUR ancestry groups</span><div class="sb-div caption-clear"></div></div></div><p id="p-24">As one might expect, coverage generally improves with increasing array density, regardless of either ancestry group or MAF bin. Common variants (MAF &gt; 0.05) are well-covered by all the arrays in the AMR, ASN, and EUR ancestry groups. The percentage of common variants with imputation <em>r<sup>2</sup></em> ≥ 0.8 is greater than 75% in these three ancestry groups, and up to ~90% in all four ancestry groups for the highest density arrays. The most dramatic increase in coverage occurs for common variants in the AFR group: the fraction of common variants passing the imputation <em>r<sup>2</sup></em> threshold of 0.8 increases by almost 40% from the least to most dense array. This finding is likely explained by the genetic diversity and lower LD characteristic of present day African ancestry populations compared to the other ancestry groups (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Wall <em>et al.</em> 2008</a>).</p><p id="p-25">However, coverage at common variants eventually levels off with increasing array density, a phenomenon of diminishing returns previously observed by <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Barrett and Cardon (2006)</a>. The array density at which this occurs differs by ancestry group. For AMR, ASN, and EUR, leveling off begins at the OmniExpress, whereas for AFR, coverage continues to improve up until the Omni2.5M. This plateau in coverage may be in part due to the limited size of the reference sample, and in part due to the underlying structure of the genome being such that some regions have insufficient LD to accurately impute variants, regardless of MAF. Furthermore, even with high density arrays, there may still be some regions that are sparsely covered and therefore difficult to impute.</p><p id="p-26">Also as expected, less common variation is not as well covered as common variation. The percentage of variants with MAF between 0.01 and 0.05 and an imputation <em>r<sup>2</sup></em> ≥ 0.8 is substantially lower than for MAF greater than 0.05: less than 40% for low density arrays in all ancestries and 50–80% in high density arrays, depending on ancestry. In all ancestry groups there is some improvement in coverage of less common variants when moving from the Omni2.5M to the Omni5M array, most notably in EUR (61% <em>vs.</em> 77% with <em>r<sup>2</sup></em> ≥ 0.8 with Omni2.5M and Omni5M, respectively). Rare (MAF &lt; 0.01) variants are generally not well imputed by either low or high density arrays. Imputation accuracy for rare variants is greatest in the AMR ancestry group, which might be a consequence of the lower genetic diversity and greater LD in Native Americans than in other ancestry groups, due to a recent population bottleneck associated with human migration into the Americas (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Wall <em>et al.</em> 2011</a>). Imputation quality at rare variants may be improved by replacing pre-phased imputation with the traditional haplotype sampling approach, although the latter is more computationally intensive (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Howie <em>et al.</em> 2012</a>).</p><p id="p-27">All coverage plots are essentially monotonic, with the exception of the transition from the OmniExpress to the Axiom World Array 4 in the AFR ancestry group. This array was designed for use in Latinos and involved preferential selection of variants in European and Native American ancestries (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Hoffmann <em>et al.</em> 2011b</a>), which might explain the lower coverage in the AFR group. We also note that the addition of the exome content has little effect on coverage and accuracy, particularly for the Omni2.5M. The HumanCore+Exome, however, shows slight improvement over the HumanCore alone for lower MAF variants, most likely because the addition of the exome content affords a greater relative increase in density for the HumanCore than for the Omni2.5M. Although exome content contributes little to overall genomic coverage, it clearly has additional value when directly observed, in that exomic variants are more likely to be functional than non-exomic variants. Furthermore, this value comes at relatively low additional cost.</p><div id="sec-5" class="subsection"><h3>Genome-wide power</h3><p id="p-28">Previous studies of genome-wide power using arrays have utilized a finding that the sample size required to achieve a given power is N/<em>r<sup>2</sup></em> when using a tag marker compared with N when using the causal locus genotypes, where <em>r<sup>2</sup></em> is the squared correlation between the discrete allelic dosages at each locus (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Pritchard and Przeworski 2001</a>). The Appendix shows that this relationship also holds under an additive genetic model when using a continuous imputed dosage rather than the discrete dosage of the causal locus. Therefore, we used imputation <em>r<sup>2</sup></em> for estimating the genome-wide power to detect an association.</p><p id="p-29">Genome-wide power estimates for GRR values of 1.2, 1.3 and 1.4 are displayed in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> for MAF &gt; 0.05 (common variants), <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> for 0.01 &lt; MAF ≤ 0.05 (less common), <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007161/-/DC1/FigureS7.pdf">Figure S7</a> for MAF &gt; 0.01 and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007161/-/DC1/FigureS8.pdf">Figure S8</a> for all variants with at least two copies of the minor allele within an ancestry group. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007161/-/DC1/FigureS9.pdf">Figure S9</a> displays results for GRR = 1.1 for each of the four MAF categories. The power curves show a sigmoidal relationship between power and sample size for all arrays, including the hypothetical 1000 Genomes array (“1000 Genomes”), which is equivalent to direct observation of all 1000 Genomes Project variants passing the ancestry-specific MAF filters (<em>i.e.</em>, at least two copies of the minor allele observed in the given ancestry group) and thus does not involve uncertainty introduced by imputation. Power is generally low for the range of parameters considered here (even for “1000 Genomes” array), except at the upper end of genetic effect and sample sizes. This is particularly true for the less common variants, where power is generally &lt; 50% even at the high end of the parameter space for the actual arrays and up to ~60% for the “1000 Genomes” array.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/10/1795/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide power estimates for GRR values of 1.2, 1.3, and 1.4, for common autosomal variants (MAF &gt; 0.05). The array Omni2.5+Exome is not shown in these plots because it is indistinguishable at this resolution from the Omni2.5M array. In the legend, &#x201C;1000 Genomes&#x201D; refers to a hypothetical array in which all variants in the 1000 Genomes dataset would be typed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-116874296" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genome-wide power estimates for GRR values of 1.2, 1.3, and 1.4, for common autosomal variants (MAF &gt; 0.05). The array Omni2.5+Exome is not shown in these plots because it is indistinguishable at this resolution from the Omni2.5M array. In the legend, &#x201C;1000 Genomes&#x201D; refers to a hypothetical array in which all variants in the 1000 Genomes dataset would be typed.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/10/1795/F4.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/10/1795/F4.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/10/1795/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/10/1795/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463409" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">Genome-wide power estimates for GRR values of 1.2, 1.3, and 1.4, for common autosomal variants (MAF &gt; 0.05). The array Omni2.5+Exome is not shown in these plots because it is indistinguishable at this resolution from the Omni2.5M array. In the legend, “1000 Genomes” refers to a hypothetical array in which all variants in the 1000 Genomes dataset would be typed.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/10/1795/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide power estimates for GRR values of 1.2, 1.3, and 1.4, for less common autosomal variants (0.01 &lt; MAF &#x2264; 0.05). The array Omni2.5+Exome is not shown in these plots because it is indistinguishable at this resolution from the Omni2.5M array. In the legend, &#x201C;1000 Genomes&#x201D; refers to a hypothetical array in which all variants in the 1000 Genomes dataset would be typed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-116874296" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genome-wide power estimates for GRR values of 1.2, 1.3, and 1.4, for less common autosomal variants (0.01 &lt; MAF &#x2264; 0.05). The array Omni2.5+Exome is not shown in these plots because it is indistinguishable at this resolution from the Omni2.5M array. In the legend, &#x201C;1000 Genomes&#x201D; refers to a hypothetical array in which all variants in the 1000 Genomes dataset would be typed.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/10/1795/F5.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/10/1795/F5.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/10/1795/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/10/1795/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463375" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-31" class="first-child">Genome-wide power estimates for GRR values of 1.2, 1.3, and 1.4, for less common autosomal variants (0.01 &lt; MAF ≤ 0.05). The array Omni2.5+Exome is not shown in these plots because it is indistinguishable at this resolution from the Omni2.5M array. In the legend, “1000 Genomes” refers to a hypothetical array in which all variants in the 1000 Genomes dataset would be typed.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">The cost efficiency of achieving equal power can be compared among array choices using these power estimates. For example, for the variant set with MAF &gt; 0.05 and GRR = 1.3, achieving 50% power requires N = 3000 for the Omni2.5M array and N = 4000 for the HumanCore array in the African ancestry group. A power of 80% requires a larger sample size difference (5300 <em>vs.</em> 8900). Generally, the array differences required to achieve a set power are notably larger in regions where the power curves plateau. One array is more cost-efficient than another when the ratio of cost per sample is less than the ratio of sample sizes required to achieve equal power. Of course, this evaluation assumes that the extra samples required are available and that the main interest of the study is to detect effects characterized by the parameters used in the power calculation. All of the data used to generate the power plots are available for download at <a href="http://jhir.library.jhu.edu/handle/1774.2/36508">http://jhir.library.jhu.edu/handle/1774.2/36508</a>, so that users can select parameter sets appropriate to their project goals. We also provide the number of variants per MAF and <em>r<sup>2</sup></em> combinations within each ancestry group so that users can readily calculate power using parameters beyond the set provided here.</p><p id="p-33">As noted previously in a similar analysis by <a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Lindquist <em>et al.</em> (2013)</a>, increasing sample size can benefit genome-wide power more than increasing array density even up to full genotyping of the complete 1000 Genomes variant set. For example, with GRR = 1.3 and MAF &gt; 0.05 in African ancestry, power is 26.7% for the “1000 Genomes” array with N = 2000, whereas doubling the sample size and using the HumanCore array brings power up to 50.2%. <a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">Lindquist <em>et al.</em> (2013)</a> focused their analysis on variants with characteristics similar to those found in previous GWAS (odd ratios mainly in the range of 1.2−1.3 and MAF &gt; 0.10). They concluded that only about one fifth of such variants have been detected with existing GWAS and that the potential for increasing this fraction is greater by increasing sample size than by increasing genomic coverage. Our results support this conclusion (using somewhat different methods and arrays), given the current situation where low density arrays such as the Illumina HumanCore and Affymetrix BioBank are much lower in cost than genomic sequencing or even high density arrays such as the Omni2.5M. However, we also note their caveat that increasing sample size significantly is not an option for many diseases.</p><p id="p-34">For rare variants, power is low, even for complete coverage and very large sample sizes. Therefore, analysis of rare variants generally involves aggregation strategies to achieve reasonable power to detect an effect (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Li and Leal 2008</a>). The value of low density arrays in this context appears to be low because rare variants are not well imputed and aggregate tests are sensitive to genotype errors (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Powers <em>et al.</em> 2011</a>). Therefore, direct detection of rare variants using genomic sequencing, high density arrays and/or arrays supplemented with selected variants of interest (such as exome content) are likely to be required for making significant progress in detecting trait variation caused by rare variants.</p><p id="p-35">In addition to planning experiments with existing arrays, our genomic coverage results can be used to design new genotyping arrays. For example, one might wish to design an array focused on specific candidate variants or regions, while also providing a backbone for genome-wide imputation, such as the existing HumanCore or Biobank variant set. In that case, one might begin with the backbone and then add candidate variants that are not already well imputed by the backbone variants in the target ancestry group(s). For this purpose, we also provide the individual imputation <em>r<sup>2</sup></em> values for all 1000 Genomes variants from imputation using each of the arrays considered here (available at <a href="http://jhir.library.jhu.edu/handle/1774.2/36508">http://jhir.library.jhu.edu/handle/1774.2/36508</a>).</p><p id="p-36">In conclusion, these imputation-based genomic coverage and power analyses are intended as a practical guide to researchers planning genetic studies. Deciding among available arrays is often a delicate balance between scientific aims, monetary resources, and genotyping timeframe (<em>i.e.</em>, which arrays are currently in use). Our analysis has the advantage of consistently applying the same approach to evaluate several different arrays from different manufacturers. There are, however, some limitations to this study design that may result in either over- or underestimated genomic coverage. First, the genomic coverage reported here might be over estimated by imputing subsets of 1000 Genomes samples using the remaining samples as reference, as the haplotypes are likely to match better than in real world applications where the samples to be imputed are likely from different source populations than the reference. This advantage may be offset to some degree by using a smaller reference panel than is available for independent study populations, although this effect is likely to be small because the reference panel used here is approximately 90% of the full 1000 Genomes phase 1 reference panel. Second, coverage may be underestimated by having pre-phased each batch separately, as small sample sizes may negatively impact phasing accuracy (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Browning and Browning 2011</a>). An increased sample size during pre-phasing may disproportionally improve coverage in situations where phasing accuracy is already a challenge, such as rare variants and in samples with high haplotype diversity (<em>e.g.</em>, African ancestry). Finally, some of the arrays assay a substantial number of variants that are not in the 1000 Genomes phase 1 release (see <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). These omissions would tend to underestimate genomic coverage, although probably not significantly, since they are expected to consist mainly of rare variants due to the 1000 Genomes Project’s identification of 98% of SNPs with frequency &gt; 0.01 in related populations (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Abecasis <em>et al.</em> 2012</a>). Despite these caveats (discussed further in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007161/-/DC1/FileS1.pdf">File S1</a>), the 1000 Genomes data set is clearly the best resource available for evaluating complete genomic coverage in multiple ancestry groups. We expect the results presented here to be useful in planning studies with the current generation of human genotyping arrays.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">This research was supported by the National Institutes of Health (NIH): the Center for Inherited Disease Research contract (HHSN268201200008I), funded by 14 NIH institutes (NHGRI, NCI, NEI, NIA, NIAAA, NIAMS, NICHD, NIDA, NIDCD, NIDCR, NIDDK, NIEHS, NIMH, and NINDS); and the Statistical and quantitative genetics Program Project grant P01GM099568, funded by NIGMS.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><p id="p-37">Here we show that the effect of imputation uncertainty on the power to detect a trait association is to increase the required sample size from <em>n</em> to <em>n</em>/<em>ρ</em><sup>2</sup>, where <em>ρ</em><sup>2</sup> is the squared correlation between true and imputed allelic dosage.</p><div class="section" id="sec-6"><h3>Notation</h3><p id="p-38">Suppose a trait locus <strong>T</strong> has alleles <em>T</em>, <em>t</em> with population frequencies <em>p<sub>T</sub></em>, <em>p<sub>t</sub></em> and that variable <em>G</em> represents the contributions of <strong>T</strong> to the trait. The values <em>G<sub>TT</sub></em>, <em>G<sub>Tt</sub></em>, and <em>G<sub>tt</sub></em> are the probabilities of an individual having the disease under study. The mean of <em>G</em> is the population prevalence of the disease, <em>μ<sub>G</sub></em>, and under Hardy Weinberg equilibrium (HWE), this is</p><span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-1.gif"/></noscript></span></span><p id="p-39">Also under HWE, the genetic variance of the <em>G</em>’s can be written as follows:</p><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-2.gif"/></noscript></span></span><p id="p-40">where the additive and dominance variance components, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-3.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-4.gif"/></noscript></span></span> , respectively, are given by</p><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-5.gif"/></noscript></span></span></div><div class="section" id="sec-7"><h3>Allelic case-control test</h3><p id="p-41">The simplest form of association test is the allelic case-control test based on the expressions for trait allele frequencies in cases and controls. The frequency of allele <em>T</em> among cases is as follows:</p><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-6.gif"/></noscript></span></span><p id="p-42">Similarly,</p><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-7.gif"/></noscript></span></span><p id="p-43">so that</p><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-8.gif"/></noscript></span></span><p id="p-44">The square of this allelic frequency difference can be written as</p><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-9.gif"/></noscript></span></span><p id="p-45">The case-control test uses the sample allele frequencies <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-10.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-11.gif"/></noscript></span></span> taken from independent samples of sizes <em>n</em><sub>Case</sub> and <em>n</em><sub>Control</sub>. Still assuming HWE, these two sample values have binomial variances</p><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-12.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-13.gif"/></noscript></span></span><p id="p-46">and a test statistic is constructed under the null hypothesis that <em>p<sub>T</sub></em><sub>|Case</sub> = <em>p<sub>T</sub></em><sub>|Control</sub> = <em>p<sub>T</sub></em>:</p><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-14.gif"/></noscript></span></span><p id="p-47">where <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-15.gif"/></noscript></span></span> is an appropriate estimate of the population allele frequency <em>p<sub>T</sub></em> . This estimate could be the pooled sample value:</p><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-16.gif"/></noscript></span></span><p id="p-48">or pooled according to disease prevalence</p><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-17.gif"/></noscript></span></span><p id="p-49">Power calculations do not assume the null hypothesis, and the parametric form of the test statistic cannot assume <em>p<sub>T</sub></em><sub>|Case</sub> = <em>p<sub>T</sub></em><sub>|Control</sub> = <em>p<sub>T</sub></em>. As an approximation, however, this assumption could be retained in the denominator of the statistic, which then has noncentrality</p><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-18.gif"/></noscript></span></span><p id="p-50">It is usual to use [(<em>n</em><sub>Case</sub><em>p<sub>T</sub></em><sub>|Case</sub> + <em>n</em><sub>Control</sub><em>p<sub>T</sub></em><sub>|Control</sub>)/(<em>n</em><sub>Case</sub> + <em>n</em><sub>Control</sub>)] for the population allele frequency <em>p<sub>T</sub></em> in the denominator, even though it is [<em>μ<sub>G</sub>p<sub>T</sub></em><sub>|Case</sub> + (1 − <em>μ<sub>G</sub></em>)<em>p<sub>T</sub></em><sub>|Control</sub>] that is equal to <em>p<sub>T</sub></em>, and the latter is the approach used in the present power calculations. Also, <em>n</em><sub>Case</sub> = <em>φn</em>, <em>n</em><sub>Control</sub> = (1 − <em>φ</em>)<em>n</em> for a total sample size of <em>n</em>.</p><p id="p-51">The test statistic <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-19.gif"/></noscript></span></span> is distributed as χ2 with 1 df under the assumption of normally-distributed sample allele frequencies. This assumption will break down as frequencies become close to zero or one. We used simulation to assess the accuracy of power estimation at low MAF. Empirical power was estimated from 1,000,000 replicate simulations of data for each of several parameter sets, using the <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-20.gif"/></noscript></span></span> test and <em>α</em> = 5 × 10<sup>−8</sup>, whereas theoretical power was estimated using <em>λ</em><sub>1</sub>. The parameter sets consist of all combinations of <em>GRR</em> ∈ {1.1, 1.2, 1.3, 1.4}, <em>N<sub>cases</sub></em> = <em>N<sub>controls</sub></em> ∈ {1000, 5000, 10000}, allelic frequency ∈ {0.005, 0.01, 0.02, 0.03, 0.04, 0.05}, and population prevalence of 0.05. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007161/-/DC1/FigureS10.pdf">Figure S10</a> shows that the two power estimates are very similar down to MAF of 0.02, but the theoretical calculations underestimate power at lower MAF. However, both power estimates are very small with the range of realistic parameter sets used here, so that this difference is probably not of practical importance in designing experiments.</p></div><div class="section" id="sec-8"><h3>Allelic dosage test</h3><p id="p-52">Now suppose that each individual has an imputed genotype: the probabilities of it being imputed as <em>TT</em>, <em>Tt</em>, and <em>tt</em> are <em>Q<sub>TT</sub></em>, <em>Q<sub>Tt</sub></em>, and <em>Q<sub>tt</sub></em> and the imputed allelic dosage is <em>q<sub>T</sub></em> = <em>Q<sub>TT</sub></em> + <em>Q<sub>Tt</sub></em>/2. An association test with the trait can be constructed with sample allelic dosages in place of sample allelic frequencies:</p><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-21.gif"/></noscript></span></span><p id="p-53">There is no reason to assume the imputed genotypes should follow HWE or that they depend in any specific way on the actual genotypes. For variants for which both observed genotypes and imputed allelic dosages are available, and assuming a linear relationship, imputed dosage can be regressed on observed allelic dosage. If the estimated regression coefficient <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-22.gif"/></noscript></span></span> is then assumed to hold for subsequent samples of cases and controls:</p><span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-23.gif"/></noscript></span></span><p id="p-54">Keeping the earlier assumptions about the variances of the allelic dosages being the same in cases and controls, and invoking the relations among regression coefficient <em>β<sub>YX</sub></em>, correlation coefficient <em>ρ<sub>XY</sub></em>, covariance <em>σ<sub>XY</sub></em> and variances <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-24.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-25.gif"/></noscript></span></span> for two variables <em>X</em>, <em>Y</em> it follows that</p><span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-26.gif"/></noscript></span></span><p id="p-55">so that</p><span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-27.gif"/></noscript></span></span><p id="p-56">The noncentrality parameter for the test statistic becomes</p><span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1795/embed/mml-math-28.gif"/></noscript></span></span><p id="p-57">where <em>ρ</em> is the correlation of observed and imputed allelic dosage. The effect of imputation is to decrease the total sample size by <em>ρ</em><sup>2</sup>.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007161/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007161/-/DC1</a></p></li><li class="fn" id="fn-7"><p id="p-59"><em>Communicating editor: F. P. Roth</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-06-14"><span class="received-label">Received </span>June 14, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-08-12"><span class="accepted-label">Accepted </span>August 12, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Nelson <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.1" data-doi="10.1038/nature11632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An integrated map of genetic variation from 1,092 human genomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">56</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAbecasis%26rft.auinit1%253DG.%2BR.%26rft.volume%253D491%26rft.issue%253D7422%26rft.spage%253D56%26rft.epage%253D65%26rft.atitle%253DAn%2Bintegrated%2Bmap%2Bof%2Bgenetic%2Bvariation%2Bfrom%2B1%252C092%2Bhuman%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11632%26rft_id%253Dinfo%253Apmid%252F23128226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23128226&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310434500030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-web" id="cit-3.10.1795.2"><div class="cit-metadata unstructured">Affymetrix Inc. 2012 Affymetrix introduces axiom biobank arrays for genotyping studies. Available at: <a href="http://investor.affymetrix.com/phoenix.zhtml?c=116408%26p=irol-newsArticle%26ID=1752709%26highlight=">http://investor.affymetrix.com/phoenix.zhtml?c=116408&amp;p=irol-newsArticle&amp;ID=1752709&amp;highlight=</a>. Accessed April 30, 2013.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.3" data-doi="10.1038/ng1801"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardon</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evaluating coverage of genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBarrett%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D38%26rft.issue%253D6%26rft.spage%253D659%26rft.epage%253D662%26rft.atitle%253DEvaluating%2Bcoverage%2Bof%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1801%26rft_id%253Dinfo%253Apmid%252F16715099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1801&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16715099&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237954800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.4" data-doi="10.1038/nrg3054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Haplotype phasing: existing methods and new developments.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D703%26rft.epage%253D714%26rft.atitle%253DHaplotype%2Bphasing%253A%2Bexisting%2Bmethods%2Band%2Bnew%2Bdevelopments.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3054%26rft_id%253Dinfo%253Apmid%252F21921926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921926&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.5" data-doi="10.1086/381000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberle</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieder</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Selecting a maximally informative set of single-nucleotide polymorphisms for association analyses using linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DCarlson%26rft.auinit1%253DC.%2BS.%26rft.volume%253D74%26rft.issue%253D1%26rft.spage%253D106%26rft.epage%253D120%26rft.atitle%253DSelecting%2Ba%2Bmaximally%2Binformative%2Bset%2Bof%2Bsingle-nucleotide%2Bpolymorphisms%2Bfor%2Bassociation%2Banalyses%2Busing%2Blinkage%2Bdisequilibrium.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F381000%26rft_id%253Dinfo%253Apmid%252F14681826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/381000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14681826&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187723500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.6" data-doi="10.1038/nchembio.1414"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delaneau</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zagury</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Improved whole-chromosome phasing for disease and population genetic studies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnchembio.1414%26rft_id%253Dinfo%253Apmid%252F23269371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nchembio.1414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23269371&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312810100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.7" data-doi="10.1038/nrg3472"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evangelou</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ioannidis</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Meta-analysis methods for genome-wide association studies and beyond.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D379%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3472%26rft_id%253Dinfo%253Apmid%252F23657481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3472&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23657481&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.8" data-doi="10.1038/nature06258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frazer</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballinger</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinds</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stuve</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A second generation human haplotype map of over 3.1 million SNPs.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">449</span>: <span class="cit-fpage">851</span>–<span class="cit-lpage">861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFrazer%26rft.auinit1%253DK.%2BA.%26rft.volume%253D449%26rft.issue%253D7164%26rft.spage%253D851%26rft.epage%253D861%26rft.atitle%253DA%2Bsecond%2Bgeneration%2Bhuman%2Bhaplotype%2Bmap%2Bof%2Bover%2B3.1%2Bmillion%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06258%26rft_id%253Dinfo%253Apmid%252F17943122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17943122&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250230600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.9" data-doi="10.1016/j.ygeno.2011.04.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kvale</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hesselson</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquino</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Next generation genome-wide association tool: design and coverage of a high-throughput European-optimized SNP array.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DHoffmann%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D98%26rft.issue%253D2%26rft.spage%253D79%26rft.epage%253D89%26rft.atitle%253DNext%2Bgeneration%2Bgenome-wide%2Bassociation%2Btool%253A%2Bdesign%2Band%2Bcoverage%2Bof%2Ba%2Bhigh-throughput%2BEuropean-optimized%2BSNP%2Barray.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2011.04.005%26rft_id%253Dinfo%253Apmid%252F21565264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2011.04.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21565264&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293682700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.10" data-doi="10.1016/j.ygeno.2011.08.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kvale</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hesselson</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gollub</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Design and coverage of high throughput genotyping arrays optimized for individuals of East Asian, African American, and Latino race/ethnicity using imputation and a novel hybrid SNP selection algorithm.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">422</span>–<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DHoffmann%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D98%26rft.issue%253D6%26rft.spage%253D422%26rft.epage%253D430%26rft.atitle%253DDesign%2Band%2Bcoverage%2Bof%2Bhigh%2Bthroughput%2Bgenotyping%2Barrays%2Boptimized%2Bfor%2Bindividuals%2Bof%2BEast%2BAsian%252C%2BAfrican%2BAmerican%252C%2Band%2BLatino%2Brace%252Fethnicity%2Busing%2Bimputation%2Band%2Ba%2Bnovel%2Bhybrid%2BSNP%2Bselection%2Balgorithm.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2011.08.007%26rft_id%253Dinfo%253Apmid%252F21903159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2011.08.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21903159&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297664700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.11" data-doi="10.1534/g3.111.001198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype imputation with thousands of genomes</span>. <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">470</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DHowie%26rft.auinit1%253DB.%26rft.volume%253D1%26rft.issue%253D6%26rft.spage%253D457%26rft.epage%253D470%26rft.atitle%253DGenotype%2BImputation%2Bwith%2BThousands%2Bof%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.001198%26rft_id%253Dinfo%253Apmid%252F22384356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjEvNi80NTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE3OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.12" data-doi="10.1038/ng.2354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchsberger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast and accurate genotype imputation in genome-wide association studies through pre-phasing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">955</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHowie%26rft.auinit1%253DB.%26rft.volume%253D44%26rft.issue%253D8%26rft.spage%253D955%26rft.epage%253D959%26rft.atitle%253DFast%2Band%2Baccurate%2Bgenotype%2Bimputation%2Bin%2Bgenome-wide%2Bassociation%2Bstudies%2Bthrough%2Bpre-phasing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2354%26rft_id%253Dinfo%253Apmid%252F22820512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22820512&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-web" id="cit-3.10.1795.13"><div class="cit-metadata unstructured">Illumina Inc. 2012 Illumina launches infinium humancore arrays for economical large-scale genetic studies. Available from: <a href="http://investor.illumina.com/phoenix.zhtml?c=121127%26p=irol-newsArticle%26id=1745614">http://investor.illumina.com/phoenix.zhtml?c=121127&amp;p=irol-newsArticle&amp;id=1745614</a>. Accessed April 30, 2013.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.14" data-doi="10.1086/503751"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jorgenson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witte</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Coverage and power in genomewide association studies.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">884</span>–<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DJorgenson%26rft.auinit1%253DE.%26rft.volume%253D78%26rft.issue%253D5%26rft.spage%253D884%26rft.epage%253D888%26rft.atitle%253DCoverage%2Band%2Bpower%2Bin%2Bgenomewide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F503751%26rft_id%253Dinfo%253Apmid%252F16642443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/503751&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16642443&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.15" data-doi="10.1016/j.ajhg.2008.06.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leal</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Methods for detecting associations with rare variants for common diseases: application to analysis of sequence data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">311</span>–<span class="cit-lpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DB.%26rft.volume%253D83%26rft.issue%253D3%26rft.spage%253D311%26rft.epage%253D321%26rft.atitle%253DMethods%2Bfor%2Bdetecting%2Bassociations%2Bwith%2Brare%2Bvariants%2Bfor%2Bcommon%2Bdiseases%253A%2Bapplication%2Bto%2Banalysis%2Bof%2Bsequence%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2008.06.024%26rft_id%253Dinfo%253Apmid%252F18691683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2008.06.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18691683&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259307200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.16" data-doi="10.1002/gepi.20312"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evaluating cost efficiency of SNP chips in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DLi%26rft.auinit1%253DC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D387%26rft.epage%253D395%26rft.atitle%253DEvaluating%2Bcost%2Befficiency%2Bof%2BSNP%2Bchips%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20312%26rft_id%253Dinfo%253Apmid%252F18271056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20312&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18271056&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257548000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.17" data-doi="10.1371/journal.pone.0009697"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartz</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saccone</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A new statistic to evaluate imputation reliability.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e9697</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLin%26rft.auinit1%253DP.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De9697%26rft.epage%253De9697%26rft.atitle%253DA%2Bnew%2Bstatistic%2Bto%2Bevaluate%2Bimputation%2Breliability.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0009697%26rft_id%253Dinfo%253Apmid%252F20300623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0009697&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20300623&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.18" data-doi="10.1002/gepi.21724"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindquist</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorgenson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witte</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The impact of improved microarray coverage and larger sample sizes on future genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BEpidemiol.%26rft.volume%253D37%26rft.spage%253D383%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.21724%26rft_id%253Dinfo%253Apmid%252F23529720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.21724&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23529720&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.19" data-doi="10.1056/NEJMra0905980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manolio</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomewide association studies and assessment of the risk of disease.</span> <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">363</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DManolio%26rft.auinit1%253DT.%2BA.%26rft.volume%253D363%26rft.issue%253D2%26rft.spage%253D166%26rft.epage%253D176%26rft.atitle%253DGenomewide%2Bassociation%2Bstudies%2Band%2Bassessment%2Bof%2Bthe%2Brisk%2Bof%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMra0905980%26rft_id%253Dinfo%253Apmid%252F20647212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMra0905980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20647212&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279574100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.20" data-doi="10.1038/nrg2796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genotype imputation for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMarchini%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D511%26rft.atitle%253DGenotype%2Bimputation%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2796%26rft_id%253Dinfo%253Apmid%252F20517342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20517342&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278998500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.21" data-doi="10.1159/000332222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gopalakrishnan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tintle</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Assessing the impact of non-differential genotyping errors on rare variant tests of association.</span> <abbr class="cit-jnl-abbrev">Hum. Hered.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bheredity%26rft.stitle%253DHum%2BHered%26rft.aulast%253DPowers%26rft.auinit1%253DS.%26rft.volume%253D72%26rft.issue%253D3%26rft.spage%253D153%26rft.epage%253D160%26rft.atitle%253DAssessing%2Bthe%2Bimpact%2Bof%2Bnon-differential%2Bgenotyping%2Berrors%2Bon%2Brare%2Bvariant%2Btests%2Bof%2Bassociation.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000332222%26rft_id%253Dinfo%253Apmid%252F22004945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000332222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22004945&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.22" data-doi="10.1086/321275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Linkage disequilibrium in humans: models and data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D69%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D14%26rft.atitle%253DLinkage%2Bdisequilibrium%2Bin%2Bhumans%253A%2Bmodels%2Band%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F321275%26rft_id%253Dinfo%253Apmid%252F11410837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/321275&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11410837&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170108100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.23" data-doi="10.1126/science.1219240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tennessen</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigham</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Connor</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenny</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolution and functional impact of rare coding variation from deep sequencing of human exomes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">64</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1219240%26rft_id%253Dinfo%253Apmid%252F22604720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzcvNjA5MC82NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTAvMTc5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.24" data-doi="10.1016/j.ajhg.2011.11.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Five years of GWAS discovery.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DVisscher%26rft.auinit1%253DP.%2BM.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D7%26rft.epage%253D24%26rft.atitle%253DFive%2Byears%2Bof%2BGWAS%2Bdiscovery.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2011.11.029%26rft_id%253Dinfo%253Apmid%252F22243964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2011.11.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22243964&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.25" data-doi="10.1101/gr.075630.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendez</span>  <span class="cit-name-given-names">F. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woerner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Severson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A novel DNA sequence database for analyzing human demographic history.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1354</span>–<span class="cit-lpage">1361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.075630.107%26rft_id%253Dinfo%253Apmid%252F18493019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE4LzgvMTM1NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTAvMTc5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.10.1795.26" data-doi="10.1093/molbev/msr049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gignoux</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic variation in Native Americans, inferred from Latino SNP and resequencing data.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2231</span>–<span class="cit-lpage">2237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr049%26rft_id%253Dinfo%253Apmid%252F21368315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI4LzgvMjIzMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTAvMTc5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/10/1795.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/10/1785" title="The Drosophila Enhancer of split Gene Complex: Architecture and Coordinate Regulation by Notch, Cohesin, and Polycomb Group Proteins" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/10/1809" title="Insights into the Evolution of Cotton Diploids and Polyploids from Whole-Genome Re-sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/10"><span>Volume 3 Issue 10, October 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/10.cover-source.jpg?itok=h5LtrAoT" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/10/1795.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-H9CtHsHR3M89R1IGB71jythvP0woyIqO7quU-Xzfmxk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/10/1795" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/10/1795" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/10/1795" class="active" data-icon-position="" data-hide-link-title="0">Imputation-Based Genomic Coverage Assessments of Current Human Genotyping Arrays</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459551" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-zt_cZD80wBrhTqNhUBZAMrRqE7wdjr5YFIrZqgjiW-Y" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/10/1795.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/10/1795" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-bxjkV4IvSuPTLMQGiXnFFNhw82e8msyt4X46XMTzxfQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/10/1795" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/10/1795.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459551" id="node459551--2" data-pisa="ggg;3/10/1795" data-pisa-master="ggg;g3.113.007161" data-apath="/ggg/3/10/1795.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Imputation-Based Genomic Coverage Assessments of Current Human Genotyping Arrays</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sarah C. Nelson</span>, <span class="highwire-citation-author" data-delta="1">Kimberly F. Doheny</span>, <span class="highwire-citation-author" data-delta="2">Elizabeth W. Pugh</span>, <span class="highwire-citation-author" data-delta="3">Jane M. Romm</span>, <span class="highwire-citation-author" data-delta="4">Hua Ling</span>, <span class="highwire-citation-author" data-delta="5">Cecelia A. Laurie</span>, <span class="highwire-citation-author" data-delta="6">Sharon R. Browning</span>, <span class="highwire-citation-author" data-delta="7">Bruce S. Weir</span> and <span class="highwire-citation-author" data-delta="8">Cathy C. Laurie</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1795-1807;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007161 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459551--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sarah C. Nelson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington, 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BC.%2BNelson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nelson%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASarah%2BC.%2BNelson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sarahcn@uw.edu" class="" data-icon-position="" data-hide-link-title="0">sarahcn@uw.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kimberly F. Doheny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Inherited Disease Research, Johns Hopkins University School of Medicine, Baltimore, Maryland, 21224</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kimberly%2BF.%2BDoheny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doheny%20KF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKimberly%2BF.%2BDoheny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Elizabeth W. Pugh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Inherited Disease Research, Johns Hopkins University School of Medicine, Baltimore, Maryland, 21224</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BW.%2BPugh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pugh%20EW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElizabeth%2BW.%2BPugh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jane M. Romm </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Inherited Disease Research, Johns Hopkins University School of Medicine, Baltimore, Maryland, 21224</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jane%2BM.%2BRomm%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Romm%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJane%2BM.%2BRomm%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hua Ling </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Inherited Disease Research, Johns Hopkins University School of Medicine, Baltimore, Maryland, 21224</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hua%2BLing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ling%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHua%2BLing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Cecelia A. Laurie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington, 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cecelia%2BA.%2BLaurie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laurie%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACecelia%2BA.%2BLaurie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Sharon R. Browning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington, 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sharon%2BR.%2BBrowning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Browning%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASharon%2BR.%2BBrowning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Bruce S. Weir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington, 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruce%2BS.%2BWeir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weir%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABruce%2BS.%2BWeir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Cathy C. Laurie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington, 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cathy%2BC.%2BLaurie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laurie%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACathy%2BC.%2BLaurie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/10/1795" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/10/1795&amp;title=Imputation-Based%20Genomic%20Coverage%20Assessments%20of%20Current%20Human%20Genotyping%20Arrays" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/10/1795&amp;title=Imputation-Based%20Genomic%20Coverage%20Assessments%20of%20Current%20Human%20Genotyping%20Arrays" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/10/1795&amp;title=Imputation-Based%20Genomic%20Coverage%20Assessments%20of%20Current%20Human%20Genotyping%20Arrays" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/10/1795&amp;text=Imputation-Based%20Genomic%20Coverage%20Assessments%20of%20Current%20Human%20Genotyping%20Arrays" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/10/1795&amp;title=Imputation-Based%20Genomic%20Coverage%20Assessments%20of%20Current%20Human%20Genotyping%20Arrays" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/10/1795&amp;t=Imputation-Based%20Genomic%20Coverage%20Assessments%20of%20Current%20Human%20Genotyping%20Arrays" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/10/1795&amp;title=Imputation-Based%20Genomic%20Coverage%20Assessments%20of%20Current%20Human%20Genotyping%20Arrays" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/10/1795&amp;title=Imputation-Based%20Genomic%20Coverage%20Assessments%20of%20Current%20Human%20Genotyping%20Arrays" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459551/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459551" id="node459551--3" data-pisa="ggg;3/10/1795" data-pisa-master="ggg;g3.113.007161" data-apath="/ggg/3/10/1795.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Imputation-Based Genomic Coverage Assessments of Current Human Genotyping Arrays</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sarah C. Nelson</span>, <span class="highwire-citation-author" data-delta="1">Kimberly F. Doheny</span>, <span class="highwire-citation-author" data-delta="2">Elizabeth W. Pugh</span>, <span class="highwire-citation-author" data-delta="3">Jane M. Romm</span>, <span class="highwire-citation-author" data-delta="4">Hua Ling</span>, <span class="highwire-citation-author" data-delta="5">Cecelia A. Laurie</span>, <span class="highwire-citation-author" data-delta="6">Sharon R. Browning</span>, <span class="highwire-citation-author" data-delta="7">Bruce S. Weir</span> and <span class="highwire-citation-author" data-delta="8">Cathy C. Laurie</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1795-1807;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007161 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459551--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sarah C. Nelson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington, 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BC.%2BNelson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nelson%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASarah%2BC.%2BNelson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sarahcn@uw.edu" class="" data-icon-position="" data-hide-link-title="0">sarahcn@uw.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kimberly F. Doheny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Inherited Disease Research, Johns Hopkins University School of Medicine, Baltimore, Maryland, 21224</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kimberly%2BF.%2BDoheny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doheny%20KF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKimberly%2BF.%2BDoheny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Elizabeth W. Pugh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Inherited Disease Research, Johns Hopkins University School of Medicine, Baltimore, Maryland, 21224</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BW.%2BPugh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pugh%20EW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElizabeth%2BW.%2BPugh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jane M. Romm </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Inherited Disease Research, Johns Hopkins University School of Medicine, Baltimore, Maryland, 21224</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jane%2BM.%2BRomm%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Romm%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJane%2BM.%2BRomm%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hua Ling </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Inherited Disease Research, Johns Hopkins University School of Medicine, Baltimore, Maryland, 21224</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hua%2BLing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ling%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHua%2BLing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Cecelia A. Laurie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington, 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cecelia%2BA.%2BLaurie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laurie%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACecelia%2BA.%2BLaurie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Sharon R. Browning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington, 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sharon%2BR.%2BBrowning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Browning%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASharon%2BR.%2BBrowning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Bruce S. Weir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington, 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruce%2BS.%2BWeir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weir%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABruce%2BS.%2BWeir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Cathy C. Laurie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, University of Washington, Seattle, Washington, 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cathy%2BC.%2BLaurie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laurie%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACathy%2BC.%2BLaurie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1795&amp;rft.epage=1807&amp;rft.atitle=Imputation-Based%20Genomic%20Coverage%20Assessments%20of%20Current%20Human%20Genotyping%20Arrays&amp;rft.volume=3&amp;rft.issue=10&amp;rft.date=2013-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Nelson%2C+Sarah+C.&amp;rft.au=Doheny%2C+Kimberly+F.&amp;rft.au=Pugh%2C+Elizabeth+W.&amp;rft.au=Romm%2C+Jane+M.&amp;rft.au=Ling%2C+Hua&amp;rft.au=Laurie%2C+Cecelia+A.&amp;rft.au=Browning%2C+Sharon+R.&amp;rft.au=Weir%2C+Bruce+S.&amp;rft.au=Laurie%2C+Cathy+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459551/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459551/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459551/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459551/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459551/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459551/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459551/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459551/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459551/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459551/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459551/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459551/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/10/1795" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/10/1795.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/10/1795.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/10/1795.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/10/1795.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
