<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/10/1861" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/10/1861.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/10/1861.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/10/1861.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/10/1861/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/10/1861.atom" />
<meta name="HW.pisa" content="ggg;3/10/1861" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Practical Considerations Regarding the Use of Genotype and Pedigree Data to Model Relatedness in the Context of Genome-Wide Association Studies" />
<meta name="DC.Identifier" content="10.1534/g3.113.007948" />
<meta name="DC.Date" content="2013-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2013 Cheng et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genome-wide association studies of complex traits often are complicated by relatedness among individuals. Ignoring or inappropriately accounting for relatedness often results in inflated type I error rates. Either genotype or pedigree data can be used to estimate relatedness for use in mixed-models when undertaking quantitative trait locus mapping. We performed simulations to investigate methods for controlling type I error and optimizing power considering both full and partial pedigrees and, similarly, both sparse and dense marker coverage; we also examined real data sets. (1) When marker density was low, estimating relatedness by genotype data alone failed to control the type I error rate; (2) this was resolved by combining both genotype and pedigree data. (3) When sufficiently dense marker data were used to estimate relatedness, type I error was well controlled and power increased; however, (4) this was only true when the relatedness was estimated using genotype data that excluded genotypes on the chromosome currently being scanned for a quantitative trait locus." />
<meta name="DC.Contributor" content="Riyan Cheng" />
<meta name="DC.Contributor" content="Clarissa C. Parker" />
<meta name="DC.Contributor" content="Mark Abney" />
<meta name="DC.Contributor" content="Abraham A. Palmer" />
<meta name="article:published_time" content="2013-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Practical Considerations Regarding the Use of Genotype and Pedigree Data to Model Relatedness in the Context of Genome-Wide Association Studies" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genome-wide association studies of complex traits often are complicated by relatedness among individuals. Ignoring or inappropriately accounting for relatedness often results in inflated type I error rates. Either genotype or pedigree data can be used to estimate relatedness for use in mixed-models when undertaking quantitative trait locus mapping. We performed simulations to investigate methods for controlling type I error and optimizing power considering both full and partial pedigrees and, similarly, both sparse and dense marker coverage; we also examined real data sets. (1) When marker density was low, estimating relatedness by genotype data alone failed to control the type I error rate; (2) this was resolved by combining both genotype and pedigree data. (3) When sufficiently dense marker data were used to estimate relatedness, type I error was well controlled and power increased; however, (4) this was only true when the relatedness was estimated using genotype data that excluded genotypes on the chromosome currently being scanned for a quantitative trait locus.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/10/01" />
<meta name="citation_mjid" content="ggg;3/10/1861" />
<meta name="citation_id" content="3/10/1861" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/10/1861" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/10/1861.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/10/1861.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/10/1861.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.007948" />
<meta name="citation_pmid" content="23979941" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1861" />
<meta name="citation_lastpage" content="1867" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Riyan Cheng" />
<meta name="citation_author_institution" content="Department of Human Genetics, The University of Chicago, Illinois 60637" />
<meta name="citation_author_institution" content="Division of Plant Sciences, Research School of Biology, The Australian National University, Canberra, ACT 0200, Australia" />
<meta name="citation_author" content="Clarissa C. Parker" />
<meta name="citation_author_institution" content="Department of Human Genetics, The University of Chicago, Illinois 60637" />
<meta name="citation_author" content="Mark Abney" />
<meta name="citation_author_institution" content="Department of Human Genetics, The University of Chicago, Illinois 60637" />
<meta name="citation_author" content="Abraham A. Palmer" />
<meta name="citation_author_institution" content="Department of Human Genetics, The University of Chicago, Illinois 60637" />
<meta name="citation_author_institution" content="Department of Psychiatry and Behavioral Neuroscience, The University of Chicago, Illinois 60637" />
<meta name="citation_author_email" content="aap@uchicago.edu" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=M. Abney;citation_author=M. S. McPeek;citation_author=C. Ober;citation_title=Estimation of variance components of quantitative traits in inbred populations.;citation_pages=629-650;citation_volume=141;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. Abney;citation_author=C. Ober;citation_author=M. S. McPeek;citation_title=Quantitative-trait homozygosity and association mapping and empirical genomewide significance in large, complex pedigrees: fasting serum-insulin level in the Hutterites.;citation_pages=920-934;citation_volume=70;citation_year=2002;citation_issue=4;citation_pmid=11880950;citation_doi=10.1086/339705" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=C. Amos;citation_title=Robust variance-components approach for assessing genetic linkage in pedigrees.;citation_pages=535-543;citation_volume=54;citation_year=1994;citation_issue=3;citation_pmid=8116623" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Cheng;citation_author=A. A. Palmer;citation_title=A simulation study of permutation, bootstrap and gene dropping for assessing statistical significance in the case of unequal relatedness.;citation_pages=1015-1018;citation_volume=193;citation_year=2012;citation_pmid=23267053" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Cheng;citation_author=J. E. Lim;citation_author=K. E. Samocha;citation_author=G. Sokoloff;citation_author=M. Abney;citation_title=Genome-wide association studies and the problem of relatedness among advanced intercross lines and other highly recombinant populations.;citation_volume=185;citation_year=2010;citation_pmid=20439773;citation_doi=10.1534/genetics.110.116863" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=R. Cheng;citation_author=M. Abney;citation_author=A. A. Palmer;citation_author=A. D. Skol;citation_title=QTLRel: an R package for genome-wide association studies in which relatedness is a concern.;citation_pages=66-66;citation_volume=12;citation_year=2011;citation_pmid=21794153;citation_doi=10.1186/1471-2156-12-66" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=F. D. Frentiu;citation_author=S. M. Clegg;citation_author=J. Chittock;citation_author=T. Burke;citation_author=M. W. Blows;citation_title=Pedigree-free animal models: the relatedness matrix reloaded;citation_pages=639-647;citation_volume=275;citation_year=2008;citation_issue=1635;citation_pmid=18211872;citation_doi=10.1098/rspb.2007.1032" />
<meta name="citation_reference" content="citation_journal_title=Ann. Inst. Henri PoincarÃ© B.;citation_author=M. Gillois;citation_title=La relation dâidentitÃ© en gÃ©nÃ©tique.;citation_pages=1-94;citation_volume=2;citation_year=1964" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=D. E. Goldgar;citation_title=Multipoint analysis of human quantitative genetic variation.;citation_pages=957-967;citation_volume=47;citation_year=1990;citation_issue=6;citation_pmid=2239972" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=L. Han;citation_author=M. Abney;citation_title=Identity by descent estimation with dense genome-wide genotype data.;citation_pages=557-567;citation_volume=35;citation_year=2011;citation_issue=6;citation_pmid=21769932" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Hum. Genet.;citation_author=L. Han;citation_author=M. Abney;citation_title=Using identity by descent estimation with dense genotype data to detect positive selection.;citation_pages=205-211;citation_volume=21;citation_year=2013;citation_pmid=22781100;citation_doi=10.1038/ejhg.2012.148" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. L. Harris;citation_title=GENOTYPIC COVARIANCES BETWEEN INBRED RELATIVES;citation_pages=1319-1348;citation_volume=50;citation_year=1964;citation_issue=6;citation_pmid=14239792" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. M. Kang;citation_author=N. A. Zaitlen;citation_author=C. M. Wade;citation_author=A. Kirby;citation_author=D. Heckerman;citation_title=Efficient Control of Population Structure in Model Organism Association Mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_issue=3;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=G. Karigl;citation_title=A recursive algorithm for the calculation of identity coefficients.;citation_pages=299-305;citation_volume=45;citation_year=1981;citation_issue=Pt 3;citation_pmid=7305283;citation_doi=10.1111/j.1469-1809.1981.tb00341.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=J. Listgarten;citation_author=C. Lippert;citation_author=C. M. Kadie;citation_author=R. I. Davidson;citation_author=E. Eskin;citation_title=Improved linear mixed models for genome-wide association studies.;citation_pages=525-526;citation_volume=9;citation_year=2012;citation_issue=6;citation_pmid=22669648;citation_doi=10.1038/nmeth.2037" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Lynch;citation_author=K. Ritland;citation_title=Estimation of Pairwise Relatedness With Molecular Markers;citation_pages=1753-1766;citation_volume=152;citation_year=1999;citation_issue=4;citation_pmid=10430599" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Meyer;citation_author=B. Tier;citation_title=SNP Snappy: a strategy for fast genome-wide association studies fitting a full mixed model.;citation_volume=190;citation_year=2012;citation_pmid=22021386;citation_doi=10.1534/genetics.111.134841" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=D. L. Newman;citation_author=M. Abney;citation_author=M. S. McPeek;citation_author=C. Ober;citation_author=N. J. Cox;citation_title=The importance of genealogy in determining genetic associations with complex traits.;citation_pages=1146-1148;citation_volume=69;citation_year=2001;citation_issue=5;citation_pmid=11590549;citation_doi=10.1086/323659" />
<meta name="citation_reference" content="citation_journal_title=Genes Brain Behav.;citation_author=C. C. Parker;citation_author=R. Cheng;citation_author=G. Sokoloff;citation_author=A. A. Palmer;citation_title=Genome-wide association for methamphetamine sensitivity in an advanced intercross mouse line.;citation_pages=52-61;citation_volume=11;citation_year=2011;citation_pmid=22032291" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=K. Ritland;citation_title=A marker-based method for inferences about quantitative inheritance in natural populations.;citation_pages=1062-1073;citation_volume=50;citation_year=1996;citation_doi=10.2307/2410647" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Wang;citation_title=An Estimator for Pairwise Relatedness Using Molecular Markers;citation_pages=1203-1215;citation_volume=160;citation_year=2002;citation_issue=3;citation_pmid=11901134" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Xu;citation_author=W. R. Atchley;citation_title=A random model approach to interval mapping of quantitative trait loci.;citation_pages=1189-1197;citation_volume=141;citation_year=1995;citation_issue=3;citation_pmid=8582623" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Yang;citation_author=T. Lee;citation_author=J. Kim;citation_author=M.-C. Cho;citation_author=B.-G. Han;citation_title=Ubiquitous polygenicity of human complex traits: genome-wide analysis of 49 traits in Koreans.;citation_pages=e1003355;citation_volume=9;citation_year=2013;citation_pmid=23505390;citation_doi=10.1371/journal.pgen.1003355" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yu;citation_author=G. Pressoir;citation_author=W. H. Briggs;citation_author=I. V. Bi;citation_author=M. Yamasaki;citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.;citation_pages=203-208;citation_volume=38;citation_year=2006;citation_issue=2;citation_pmid=16380716;citation_doi=10.1038/ng1702" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Zhou;citation_author=M. Stephens;citation_title=Genome-wide efficient mixed-model analysis for association studies.;citation_pages=821-824;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22706312;citation_doi=10.1038/ng.2310" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Practical Considerations Regarding the Use of Genotype and Pedigree Data to Model Relatedness in the Context of Genome-Wide Association Studies" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10.cover-source.jpg" />
<meta name="twitter:description" content="Genome-wide association studies of complex traits often are complicated by relatedness among individuals. Ignoring or inappropriately accounting for relatedness often results in inflated type I error rates. Either genotype or pedigree data can be used to estimate relatedness for use in mixed-models when undertaking quantitative trait locus mapping. We performed simulations to investigate methods for controlling type I error and optimizing power considering both full and partial pedigrees and, similarly, both sparse and dense marker coverage; we also examined real data sets. (1) When marker density was low, estimating relatedness by genotype data alone failed to control the type I error rate; (2) this was resolved by combining both genotype and pedigree data. (3) When sufficiently dense marker data were used to estimate relatedness, type I error was well controlled and power increased; however, (4) this was only true when the relatedness was estimated using genotype data that excluded genotypes on the chromosome currently being scanned for a quantitative trait locus." />
<meta name="og-title" property="og:title" content="Practical Considerations Regarding the Use of Genotype and Pedigree Data to Model Relatedness in the Context of Genome-Wide Association Studies" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/10/1861" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genome-wide association studies of complex traits often are complicated by relatedness among individuals. Ignoring or inappropriately accounting for relatedness often results in inflated type I error rates. Either genotype or pedigree data can be used to estimate relatedness for use in mixed-models when undertaking quantitative trait locus mapping. We performed simulations to investigate methods for controlling type I error and optimizing power considering both full and partial pedigrees and, similarly, both sparse and dense marker coverage; we also examined real data sets. (1) When marker density was low, estimating relatedness by genotype data alone failed to control the type I error rate; (2) this was resolved by combining both genotype and pedigree data. (3) When sufficiently dense marker data were used to estimate relatedness, type I error was well controlled and power increased; however, (4) this was only true when the relatedness was estimated using genotype data that excluded genotypes on the chromosome currently being scanned for a quantitative trait locus." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10.cover-source.jpg" />
    <title>Practical Considerations Regarding the Use of Genotype and Pedigree Data to Model Relatedness in the Context of Genome-Wide Association Studies | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"LzXQr9enqVok1Md6XRAxIE952Xk7D9Q6rIcrrJS-CQU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459561","apath":"\/ggg\/3\/10\/1861.atom","pisa":"ggg;3\/10\/1861","ac":{"\/ggg\/3\/10\/1861.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/10\/1861.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/10\/1861"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/10\/1861"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/10\/1861":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022U2CooxukgOFGGAgwYaQRAg9cU31519wVVrZdSbOiiOiFhjDS9dMYOSY+GVQm5DdMJciZ1lgNOvFrREUV4Aj2ZPhSduzVLskSiRdSq2DYh+5q+kl5KdeC27k5wvLCNYIsVJo96EARLvljY2805TOruNGNJ6HtY4kidDEkC\\\\\\\/hicPYnKRvrp\\\\\\\/fx8XRgWCae7eb0fLf6rxuhHRD8REgND+hWV1tERbEDnBHb\\\\\\\/wQPQ3P1ZnBK3GNd\\\\\\\/9x83Bfkj9DvwoTHxjc6jFtGevpyziU21dAlM8H5S5ymvpnc48vU8wgCuF4PhILoROSh52llTfps1uQYRJfpE04PwhDbU5OGG2F0BI64SsieIYJ3rbQ1v91TlXQDefmMdfZGQ7PXLlMr2YykKPQQd4uFcKeR2fP9wcUWpnqL2Fu+VqI7aM8j9yy3Qx0kIL9iI9DINkBuQt5D6GaUKMwHWOhn158rVshui9eRhac0CLHiX+t8G7hl7IKCz0aId9IV08vnHoYVUBZsqprTl8MXkudhsCIwlrF2xCSnaCFw1HpfjAjvhJH\\\\\\\/vOPRRi2tH1PssBfoy+sMtWqChiLLeIwjC5ymFGNdUXHi2euhIiF5tVF6PitQcha3mo1mhE\\\\\\\/ctt8iRHkYPyhxi70urCnkW1vf6T41N7qTGQLq9VPtZD4Ml3EythDbpkfCKdA83dU8jgO\\\\\\\/un1SOOFoAPs+g3RLCEIna0Ml+tyeSsdGDSywYBjZRniDA8GOm2lp0LdQKrScDh7bJdkEYq6NtuSmBA4k0uAgohpQItHCdJQ0iPZh5wOba\\\\\\\/MWEQVc35bVMGN8o9CLp4QXGd4sek4sVa3mLD7kIgt8xidPq8UBRnxR9m2gv4nk1\\\\\\\/tLeK5UKKZYYJFPv17U1gwb+RoghF5eHQC4WnvqVHIo06tVpDH0qICv8SuZqBZ+Ff48MGcbIn4R7BkO010XsH54RMWBsQ4QTc\\\\\\\/zjJR7ILNqV3jXwa6SxLDqeNZmUC0SaOZuJpA8Gk8QpZihw9YmpZff8YPvAZWloneOaUjYA8KHlh8ss1ngPgA3Z0Htl8rXZebsHmxJamfAFcZ9wnep\\\\\\\/5ew7i08zMYOc\\\\\\\/oGOEapw\\\\\\\/jvmAyU+tFmtOME4DPKuwNAgRBlk8SHu48vNhfIiLQUpnTZGzSCYt3S94QerdN1PC7xlpzsZeFG5iDCNGbHgYw\\\\\\\/RWZIf6UgwpyBhumUbei+el+ZmBRWCJntpv8PqhlAO1IiN35HOsDA69Ns+nkIznhDnujxeoHf6oZMP1CXDjUg+XZfvnANhQPtX9SnXWOf\\\u0022,\\\u0022iv\\\u0022:\\\u0022+7ucjqoaZ4iX962Rzmnafw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c006c3eaa0411208fffd9a5af5f3a03d469ac87caea0f7c7b1e721d359eceb17\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SZsZpi9acp0r3EUDc4JoxVEhbyYNxShDhahsJKC0aO4dqJCIKHXCRzv07\\\\\\\/t\\\\\\\/RXArj63oiSnK42jTtCkmn14f7zwVQABUUwCt5GUor2h2Y4DkpWIF8h5i5ttpG8OQQzwWa6L\\\\\\\/VLgcGuwQtg8seFwajFeHVysPT78ImE1DdPYcaC19ExR1d6YhG2otdMcp8lyFOjC1IICPGsBVYiV\\\\\\\/LoRKLajPJ7kjysKQ7qAPYVvU9rjHeV8ZAZg58YJbG+klcJXSrJlwFnF3O9tpnoiwne6QvudVegr\\\\\\\/9vrxvijga3m3RtKC9qYasBo42ak0LFJAE8FWQ8IBp67aq6\\\\\\\/GXllgUwMWYr5pii63mItJHJIbQ41Q3HY1oQmI5CtFi1rp+UFemvcsH7kBphfXLxFkK34CJ2IBOTWAJTFFTP6ErpJW03z7Rg\\\\\\\/GHMhOhvrYeK3TNqKg+PfO+egPPV60ExCAzAtx7WZyrRNNZSKEBczSLKAxsOIacFqajVsDGkP1y1CtJxKID8BwLFJg0I0NCG\\\\\\\/42UdblqFzrRaHI\\\\\\\/g\\\\\\\/O6tRZsoUF0BBYpw1kWuVIet1p+zeH6VkkJMXNQzWmgEGePiIxzgmStcjrniS4svFE9+StzDIArngnG\\\\\\\/QrAnlYoXzZ3TvpwRzYfv1UuQ\\\\\\\/lIlMqLwXFwch\\\\\\\/bdllMoNw7R7ECmkskigryuT3q9w8+XTmP4tEMS9HWYsdMZnChgXuqyOCg2EBIaEQ+D5vhH3HvuXhmPa6WMeoTVrxEbY+8TwIV5ESGF0hlxGxSAsPlU9zOuebM3ng8J9uJbxUvjYKH7bKj8MTQWpF463R6T09eTwtnsbhIJmlknNpEpBWvjxjqZdpC2U37PraSpGUuZFlYuBzNt83rXZC3n+j0XY6Z\\\\\\\/rltre+W9kSD5Q6ul8esdVIjHUgVrHirec\\\\\\\/wS\\\\\\\/XFdQMVetM8HxLJc5e7RlPCsSc3B1q2PVDQbUvVZjrSPErSNGaD9V9UREYJBKRin3TcTZG6WIP+PXYDh8scDwQpymekxLAK1DXQRvI2p\\\\\\\/0lLG2EKL6BQpPCeWQhINZwynBzzQFPm2R4GQz5NuLuXlKtey\\\\\\\/M3+RGr9GeHOQtDIT9tr6j647UAPhT+LCtZhGXDhfDdfa4R3wBfHUEGKPISnDgUflUnAN+tMCYzLZbkpnlKIW9P8h7qv0t+U3GqX3bi8s5vExsSP79brqLfvKDFSohJh+5KZsua0AIi9AoYrTWhlEMqz6CLNzo5U\\\\\\\/0pEyUvaN9mAc3ZZo9wZz51zn444qoAfB\\\\\\\/pOpk9Y7JCgTtq+TbZQK3lx6IT3xgZObocr7Dm31o6Py0gaSk0WvLHWmNtzVUDiKD5jZLKEuiKQ8ZTXmnGx+UX5awq4KHNBN7WiApg3OuC1CQ7vHJE5axNq0mRfMNVkLW9gihIreY9G4rLlKtpyzyW1uMnAxs1BOAfuWM6smzcdv2hp5G2w7IYX4kPhDyXmE73wolX6OwDQi1d6rTfw4DAvgdUWlpqvJom0yXTaQZ7Y6PgZUQPWLVkgMFltquPRowMTCagNlCod5GIFoaE2e8xZBD2JIwqAALbzBTSl+VOaLbbuI+a9qeB+tbJTlhvRORx\\\\\\\/6XTKgF2igqbofzjDSxaJKiSjMjjwK1pPoZolTk1VNa5WQeJzKLXfmTxW05NQyvilJi4sl2+goMfiQld7gG+AXdElxQ9XauA6TzB4ysB2MP35ipOpmL42KmX+KU0A6wBr8hok20YjbDkaFnKc+6s2Hpf7wJkoRa+agyXPtyDgxPv3r\\\\\\\/6PfUnXNLepCCSeJtHSHyRpKGl19sq372azdOj8nrn+W7ZZKdf0Q6rvy5QbWKlsmD3rKifLSv4JUvgtNDkCBLdEQb21u6m+akZ8waRjPOUrNHX2XiTfITYp8vcRmTyc+4sxHnTaFChdsBoXt+k5zb6o9U2WhYOJlwnDyeUJAK9Okogz4nMH5iS9a5aJoQzxe+uErdGoItWfU6OJp3gfoqCZhvW5sb+iXrlYW1ykwGOaalWGcC54L1gifV5QhO92yNd6u4PBgGVEA93qw9u0W6hTb+Yj4X1pK+zdyPrro3Bc1UuEd01qf0nWF8vd\\\\\\\/ifJxBmFHuVQekljlNKXhbCk514qwLdpPeKrHwNgka8j8f4QFMZSLqNY2gzwAMeXD9yWKcW0qAvFo+K3cwkKqALrBR+KbDTq2TG0RLiDXSozqDidhSCiNczPOQwYf48I33K17F7nL7ag4saEQHU173RORCvuLXRS7ufq0fimKqjmb1aBQ\\\\\\\/Ccf\\\\\\\/TpqgtCE+EKuw9bPaoiCVn5Z\\\\\\\/7RrF3VAILd4xrTZDr3prnhCA6yo58wWST79pl+6RUHUZqAbHidPV+XQCglgZKRvLF3kg2Z0mUZtDl5HaL++IeDbVna7bqRCxeej8RSYM3EmJkqyXWxCIjolS8vuhjLccY30FwbrT\\\\\\\/tHg\\\\\\\/B2BfpY3BY2JQwYXZILV7NSpJMPjAi662OC6nVAHwo7vB8y1TRlaFP1sVsCdjNYeTSVC6jd1wgozCJCHTZktdnLi6r2bmduMyYdtnXk9EB1f7DAZQjhzOfIl1Ih7xXtMDuuSLu+9UeCnVMsANzyEI+cNEzTaO2JT34OSOpxTUJrOy2MG15eqVR5lYV8fsb4rhQXwJTGN4fGaBig8vry3FpuRSD6QzDYIoBJfPzBn7mu7nF0HzLqh8XnOMxW\\\\\\\/nYq7iK1zuroW\\\\\\\/2nRQjaZwjdpdxkiDv369w7VHomBlISkx7nWrcQ3QiDUrKPQ2oNeFWBWnwsA79sP+E8BVwbjBV0hlqux\\\\\\\/tMPLYzZOiIk8vSXcrjeEpEe59KP5XThRi0AOcl7++WD+zwyiUrBuf9yXb5bUcijE5TSZwjTQRin9tjnUpN7YXLkNhW3KS82flGHpbWECDDX8\\\\\\\/RNnXXc4WK3vrb6hhnivtgDKRQNP\\\\\\\/dt+wwy9U35KU79Qq7PwzV6IvJKbl545FS1MOTWaiTtDf\\\\\\\/TtWTitNc2puG8J6V5mO57Tr0Gk2mGyKaPVQK2Em7I859hpBFePsZHUgBrSArP5912gxxaeJe8VcLiUbLwyLefER0fkyh6ieOhkOx8\\\\\\\/IQJTYpnqjv7RzcmWCYfpI3m0X8dYnoDtESZADVBReWqHI\\\\\\\/5hR5B+xjDonuu3n1hQnycPni8B3e\\\\\\\/q97KuoBw6jg0qVgldEDAGfBjjVeVI7oWRU8+UWc17PBPvTg1VKtJSTPNRp9Eh32\\\\\\\/12gbpr47AjDP81X7R5kDdMOSn++mkctVt62EHKPCCHzuAt41sD9ZJVJGdT7\\\\\\\/xA8iz3blhGZ4RuQmmlHiowKyh0dPzEAN5\\\\\\\/dOEHwOIbEGY2gS\\\\\\\/E4zMfEdOJYAc69fhPOEeB3MUYU6y3VgyguLPIBD6ttKDf\\\\\\\/2jfvG3QpbP2rfRlplCg+uGT3oiNt3pPxY2lzl1tI9L\\\\\\\/Yj9IwCfdhiSFU7wLOGmx3GsJxmv6XbanAX4k5sEMM9DBTmFy07gXFKG1zPqXzNOu9P1i2GdfuT3mklWMUKsntKTzQCLUFbv+LBsJOu9C\\\\\\\/cHXD67E8WhMYIuNtpjPWtJZQeDp1LWHpXLHbMc\\\\\\\/Htdm9P9rGe4XY2hibacV53BnsjgR32wmalT9+elL33lUnU+1XeptDxKU\\\\\\\/cMcq\\\\\\\/I9XFjNNfwMFOxsbHVeTXeIGWPayhagE+jzZ479YsqavPoiXl7WVU6kXSGPUrOy0BoXK9pLuTEejuIfiEoFJokqLJFmPFY5I9ixHS0tecPzFaMaOEuELF9FhamXMGGWNSmlxiOfopdHGHYmoFQv65dCEY1njUfOYbMMZu5f+eFOFowQPgUUS\\\\\\\/I\\\\\\\/BfhrWo8T6ipL\\\\\\\/Y6585MYL9jmcyE7\\\\\\\/1M6FADMhwgIaoQvCvygWGPrDA+0iebHThQ\\\\\\\/6aSrQdR9kLiK\\\\\\\/Gt\\\\\\\/y+PYSQOBnmKvTvk+q4sb+cSU5wSDFrpLxQGgcFaeMo7M2RorGpHRyJ4XO3HzDjK5NqgR5s0+v+yxiXavxSuyLuIRquJIOu\\\\\\\/YwBMhaCADj6zPgpdfRyfa4EkK\\\\\\\/EGY+kNR0bEF73HPD6\\\\\\\/SYI95IeneHCiOpL24T7H7TVQFWIFpHm1CIq4WKSShcEB0rIcybyIqNimMqqfUT77tacX26m0RT\\\\\\\/PriKfkOGfRvSF8w0Pdu\\\\\\\/KjuKfX4jiJQQ\\\\\\\/7ZEheUPR5l8sqZFf4rTT8PQwBamRRVI6DPXKGts3IcZSueBvDPGHsqgYzanbDpahVDk03265gmgSd4fI+09UE61wSe3MUsa7rQKVyb7dhwYVk9\\\\\\\/v1YVPax63XQ6WFU5HdoSq9Tiwh7bktKwTjwQRurlR6vf+fJX8khHa382aW0SszSUi1RVmkdp0YjHUIyikytNzI2YysH+MIgDpslB3vwzUr38rNBOjyPdj9iskiq\\\\\\\/7fzpfyVKr3NwplRfsxzL5KRjd1uvui5jFfaSnGoXjJXO\\\\\\\/bXA9WR2GyWyx3Zdr5tygr7BzJ5Ey7VHBiLuPjm56W2NMnypPrYSJs8iJu02+4dXDlAUjbO50vTE6Vz2EFs69BU37LTMoKPPN6+hH3bAt96NZuKWEfgURZ9xdVzp\\\\\\\/B9MSUTCAgSJ904FLq6pLBkV58cRMgTN8ovPsRB+z2eG9m6EqCHHG83K34wxT03JhWXcKWRGi5\\\\\\\/JdAEEj6\\\\\\\/qyhMtUMwzH1TpVv\\\\\\\/IFpZ41zSr1j0fP3DdVK9mmjKs3zjhmD72fEppufxQWAHKIxtt+Y3jBBpqffWykPXmXCawSdEEqUpt4xhsZx\\\\\\\/l2gegE3JEp8ZcwN\\\\\\\/ly8g05DJFWn1h\\\\\\\/0LLGgz3c59wbprIEbSEkAyR4ErE94QNr0\\\\\\\/0rF6UD6dSDpOW7t\\\\\\\/a\\\\\\\/Ypaug7ZYl3\\\\\\\/JW1KnyJxMEVOKdCpQc7CGBy4dO+mI67VVtY+S6eEl1NL\\\\\\\/jNfp9yWq1D\\\\\\\/kyIhlDiKmbjn6a37rYE7B0ErH3vk5uEHXOhK+LIP0x1LfPucDeyhbV2DoVDtgOb3m9iy\\\\\\\/RU5JOgAiEa9Vxy\\\\\\\/kDZDPWP41QBCNWm8NQ+opq8xTVA9g6Iq5Z9SkfuNqD2tbkxQoQYVoXciK2X2\\\\\\\/Tovc9rYe3OlMTWQk29Rj\\\\\\\/LYjM9u+WeBnc83LdSHTwB5NU65Vtisb8akNhUsMxnWOFwfuP7lbJrveZnnn3wxPXFdLB7G\\\\\\\/3DefTzRMRQpkN0BeXYWHnr8VSFefqYgYGPDhhhOKXnBF1xWPnqJpa9li+gvGkR\\\\\\\/1WbgdW3WWXISWX7sOifBXQPzpgZdUrfleXF6n9BcigfdpQ80R3v81xzFLX6H\\\\\\\/BmNR9YBV2I3MNpAYib2POjR2xjtnnYu5aTgK+Gc+jCumIlX0FRzLA6CQT4D6taIz02QkHvpwJX2ix073i\\\\\\\/R8fPVrhjGG0ZVYbjXyXrZVtx2lvssAjoYGI2pl28RHYp\\\\\\\/j\\\\\\\/m1uZRjOCHasOdXSGy5TvsJ0zk\\\\\\\/ZCL+NC8stnVVgWP8F83HSnJfUkUa1iFil78ze2fQMroDlzO90BOffKoX32MXvzwLB8W\\\\\\\/sHybwV9JrATtfO1TwCvZPlh2FCWiJhjnOW48pugrrw6An3UeZQ6GHwwOiLYkBIjSlsJoB0KC+8jInUG53c42i\\\\\\\/CGxjJm19eA9P3m\\\\\\\/+jOCchBlmyQ2pq0NP\\\\\\\/4+gWPQPvRXW1XfTazZ\\\\\\\/sCUQdKZ6GQ7jdE4nuXn1qki3AEemlNi4rw+gAkUAf8pTxpQt414yYykRbl40dAO5A\\\\\\\/rpPwqi3PxevIv2csO56\\\\\\\/eInyZH3v3v1kxMCErO4P2P0vRB06txvsC1Sa5\\\\\\\/p\\\\\\\/q7MdN9YlXOslNG6UQbd7NM6KcvVYAgClN0xhuCJwHlIyVvP+J8HaGNdP+SWE6ZxPxzb7As1v0okk278V3ac62NMgTmhTcTTPmXrorhgUw74P+kcE3OQMTL26BSSsT0n\\\\\\\/Ju0N7v8kSIvhfJEjQZ5Nl1kZa9cPx\\\\\\\/nYN2Cf8AYYaNEys4I6U7FMOpNjQtMMp1z1p\\\\\\\/rZKElVwzTE53i759DEj1QW1fZRdPZL8ed0ZamKMhDpf6gB5DLKegxhT\\\\\\\/oCqBe8NdSF2Wg6mH289Xq046Bvzkqo50FzWDH8jmDTmZmUIPXA75Fu0Hf3d59B0rMYluGHZBC+kwPdsLxeKiFl2YerlNdLmtdyXBMfhgEBhxYyTGRB6qzyyMLfvOtPGro+3BqT4lbPoQuw83B+ZqO74eGYSg7RYkFuzKjuqvOgAnML0WMoU12q1O0BpnXS8B8UaeT+Dgj7ZcwUJ6MfaLCPf3uR9hyBPmpIBTZHRj5WGZsZKyxgoedNyAwryCv2Da49H7xx33gWF5n5PZquYQBIPUrNF8v1DFSsqvhQhYR8D5n+4k3pXQBGuUbgsQQiGtW+vQhT+mgQtj4br5ADMBahFuQ0y7BnWTaKAEce6aH9t1u0m5RlvXCd+Dn4pmUtpMqjmgp8fDytHyFr2+A+7Eu47bTB1D\\\\\\\/pvP1DJzZ75J1Mguyy7MrmaHb81bRA5YQGJhPHWFkvOdrMv4nMeFFPYLL7eR\\\\\\\/\\\\\\\/nYAYawv9m36MvzeNyxYw+ktFVneufVwHGbBHxtXoYlkc6Oz3PdmZk\\\\\\\/DpUy\\\\\\\/dikTh8ukD3KeNLQ3+blFQZFT6Ae\\\\\\\/zx4M+UbgEZOxNcTjf0LM+F2YmeWg\\\\\\\/XpHVOtzKzfjwRHrGCFUps5Tu1O21QVjImYUoOoNAhXoggreRJkYc3KfWcBVknurx24+SJqSWWyvJnB7KPpQfP7s0eIMoYHbWpXCgnAlaafJ3k+sCZQxx9Reu8vw4iLAvaOo8yDLhlIfELxa2rCzyGfCUivUpOHFGIU95q\\\\\\\/ZW44WTGwtBAWsY4wJbMj+kWp1uW7sGfiCt95fRO3ASd+K197e7AhSpjYhgDCcxSXDwstuUDXZopqmjne7Nl+7ZINUF3QdjAyWXipIrhAUu40VsPBYGFqjmYN7y4sf+8ZE52wccLiAjlWD1dSm6e940BEgGQfz2BRxPjvQAlt9uugSOZU1O1muwE8PXJf900WHVHkyd8gVbsgFuxtKzfILTG4wCJG5K\\\\\\\/ESd6UNAHOoEsIKlle5KldERUCX\\\\\\\/EYbut9Vj6fmtePB1WobPnYnrc5b2pcy3z\\\\\\\/HaAnFqtIqW34AIEFUPnb7fQ0yCWYi9GZLLnkusqu3X3wj12QX0UYZbhiYKThtPyKkIWO7DSNpfICfPFhg1w4eoAT3RCFIYHPfa7oNbQTeJTsdjvF4QLBgzVNa17sCJAY9uHwOYSVitiRIbdmbJsOtVhWUcVep1xeQhJV5Uy\\\\\\\/jAD7ATzTyisbMyCzp40M8zk\\\\\\\/DR62W0mZVsZQstYQw1\\\\\\\/s7DpeflkL79KXyVhFGSXeYO95WdOFrtXbnFR49a6AyiZmpTshwL7qx31aFHZC2PGkb7Ct1lRsfSWDQaHhTPEP1dWkkoGjrbLChjmSmG+JasfktDcAsX7omb\\\\\\\/rGbfrwYcGiYWkL06xomqkQlszU8OOCMUR1TtQdSRMLDvz3gWjlvAECcmY6eS+38+QnnGh0rHSjm0aqYP1PalwJHEhPh8aiPAfGsGnOqKuBFW3ger7VoiszCiV9btxw+80qfNqlHy5+jxlie9T5xfD\\\\\\\/\\\\\\\/QYyEygRhxIhr17mg4N\\\\\\\/2b8O1\\\\\\\/ywWOi+Sreczi\\\\\\\/uKI0FNq2ck4VjXwmpN9ysND8yIcrJUsPmm6tH5CIeQ5Vl71BVN9nbTgC4byzWuBtCpUWhuhyP517dBcjCy4kIrvh\\\\\\\/nYV80xWxTyXXGWv9g5aAI4yukissMI10cpJRB5J92WfJna4SSnkRdZSIyhEdnb6ak4Zeny9v+wsYBrAVF\\\\\\\/fAMQUN5ClqPPxHqqSeEPnZ93fb6HkLXkUcaoiBPeN3\\\\\\\/qYNo2vDRZ72cUyzfbLsoTyfRIe0NdAkN9FH\\\\\\\/CaqnemGiiLJZijtbcEJXarbgPWz0UghRvj3dLBGgD8H1mH1Kj\\\\\\\/yG47B\\\\\\\/q46veoIwbFk\\\\\\\/NjIpmiQc5h63qQ9X1pVzHbZ2wMC1Oy0Yp9sq6gKUz6wL4UHMWH+S0HyQREoi8+zzSU0A2zFg06pwlFgKy3uj0jYsuyzgGUXOrxbsLILtIXiMrmr3R1pmC7Jd0q8SgsGbSMat3KgOmU8RdQiiUjIxfnw9i+BSgPKG+rogN1DGRNhskfVx3e+aE0hniR6RB7L9IjHmABdtszkfkQv+dSycyCvbB70XawYmFB4inp\\\\\\\/moo1MuMk07ki\\\\\\\/PQKICBWRw3lTfkIiqohZlFHE95XhqkS1eA4jFPaqv39juDTR5sKY9vQ4uiEEjU7HZYJE4kxtvnHoLryoBEP3XAVlbAnP1doT33r4kJgv\\\\\\\/cf+rZMFve8MbPESLD2uwq8mBWZ44owAEu+IHvcnfwZ7hIHNlWaVOUGyCsTJysnaeDPcgrRkmpJJhctDWjBsgoh1lM\\\\\\\/8wqZdPXsxqNmZOBhOsADTL3ntODZDX1vfcJI3UjUruQ1INpl8265F0SJSAHYa6FzvZICyIf9HZ31joEHy8vfYLnCcWBCv8PCItliTbTwPKS9HDXCIQhiklWtX0aXty8\\\\\\\/SJr1cCf8rQGktSBHUWiYLKazm0WG3OhEPz5j+FxhaFDVMAuc\\\\\\\/\\\\\\\/kq7Ppnzo0fFeUXRR\\\\\\\/5yswCKriH2sUHJMo8XFx3zf76fMMYFeAaD5Rs73ypljNNSJeAQepxGgLFrGBcd7tM0DgFnNTccTjA7tix+7BlpyEI7xusl1p3n9o1W2i7aGSNkO8R1Igma3Qji3sCHCzTD0qcllZCCHOjRx13KR4zswwvpNGRvDzu7vAoEM9x7Vz8RczsTNQw9nNrNYyy8GYHe2lIRdgSloAj613EeXEv3GUbgDRe2vNKP8G7NbgMIa0bc2ONAbPaOmzqSYj1H5rMGoWxilYKOTxUX1YlQzyUYtV1VR1Qoo4m1yJZUCoeUtGdZog5xhZgmzJ38YoE9DKWQUBq9tc+p\\\\\\\/uBpZwfMTXqtF2vLhC+R1rVs3g2GE+0l9igBlY45flvcwktEPhetzPT1lQ6Sg8YG8bFmafCW+YeeB0ApFjfVh38E90BNNne\\\\\\\/A4WVb0hisZA7Ds2ZmrGtvdxcyl2G4u972nQEVwsUnrKmWsJ1OFlnXI7Udh3Tm09C9ef3d+bqeaVW6HnT6bIjIaJGoJxkuAxgtoYDQhX02NKmFfUglaRdl3gAwQcGMAEE4hMR5MifZFyfRKw\\\\\\\/pV2L6+xHtZDK1Vx5ylj9YWELY0ICwhTVhnzO1By3YLqs9d2r17batZxMhSAOm5\\\\\\\/Rfwl2pGEzkNxS8+PhmrEi0YSyzIOKY4vWtefdarFbxNCzcVFjlokhHt+T9CV+sIXtuB\\\\\\\/PbEoZn9gbXDPoFR1nrlRrOcR+MFFQf+zpKvX\\\\\\\/NBxB9GEJCZscTocId6539vrBc1ivTCE4l4P4Cci2r4B0dNaS7fYDcSrrNEooCdQFzyZjpmtaqR4ntaoT41YfsY\\\\\\\/pSQrV5kArOo6RQ9I1p4w+8fx0n0kue+E37lBAmoUfNWQoY5pGkTvnBffGAdwvbjsAgdEfqBenhvZ\\\\\\\/ZRzZnSGlZ1dTPuWP8BTxzLHy5cUpHSYoyQNCbclgpzafp4JU5qFdkRBPEQVpi2AQ4CDRh2Q\\\\\\\/P6yQtT\\\\\\\/ZRCGXgadY60jVC6blb1iLgXUJgqsXmxxCUiqH+LME7YyA0I0UoDuwkUu4Wn060w1amgJG4pFr8jflh0vfCJkCgViRVsv04YUHkZGkYlpUqc\\\\\\\/ZmxUiLc03cuwsfrsH7jZW68miCqrBt4hkBS2Hg\\\\\\\/bci9q4nf1VkY7GT\\\\\\\/sqcXGv\\\\\\\/6mqFf\\\\\\\/6+Wo7TWXgnjv1sUfym+PrXHITH+DtG+tZ0Pqx4WJYUOn3V1ohU0mLeSMYqJdJ7P3jDnBnluGMhrAqh2b+mQPyV8EGob7v\\\\\\\/QEQGbeMZBqDx4VMMSTWOwoGs+yR91ndIp8u17nJE6GpBVmhYFPsM1zxe413MqAeEdwjNS4bK+O4xu2WzvfzdiNFJlxdd8mNMljXYZ0xtDQac7KHgY90eaXXlRaem\\\\\\\/LzN6NmT5uf+nExWqNBIdMpK3tP1hZbfslvNtubf\\\\\\\/95cwRRe3Py58nwSgAZyH8YMjX\\\\\\\/n7+Q2t\\\\\\\/9diLn05kKqT5YX4iNqENbQYBJM8AwjF8gnlL9SmUjEDoX50nfp6K2STIUHSWFSy8hRQn9ILtzepHQFlEd80iPOhXvibOz8KBLTTY0tPAQTe39zYoiEr5TOa5L\\\\\\\/VPSDPJywZNuKihDZ2njnZl71zEWFwD04peSvGRj6WaEUylE7idDZwkdWNZVjNphi3RQgYy1WLmgBDWDAlivsDlbLb8I0q9iLmpWasgM36YWoYgnM+apNvVOljlTUJzKwbNR5hsCRcysCrCyTCardIxTEWfWXItyoHFUEI2Lu7RVwjGOK1XNCDbeH4RTZ0afbzjLiuOZ25+oxKlImvbHYC\\\\\\\/o9FvW4xRpD4Cu3E7j5i7f27iQ76n6f67pYCJEnEcLR6hTdguVpqwWYKH4CU+d289CUlLMz27dL19YcgjUx\\\\\\\/c8XPsTHNmbNU9a8crIVNI2p3yk4ky3Hiplfrr2MrSpKQC3E3Cz0BfHllyNlhas90S0V2h23qovexrojbLnQgy0\\\\\\\/HgeqLK\\\\\\\/3OUiMGwkR4pZgFK2eMuBH334I5eLw6mlJzWeLeW4n561jOiMr\\\\\\\/rcy74pZ7L2dmySAWBCAWfLrkUE\\\\\\\/rFh0539CCiR6dh+yV5TY3m5n8BQ1Q8Ea5Lj8xzV3HpcnFotCYW93O\\\\\\\/0ttaw8B+ZEFqHpkbvhhbBhZQS1PPlt5rjGf9OhIsp0hM\\\\\\\/UO1mc2zUJlpnQnu2Dda0pLFwYSN6dXKXcEqUpAXybgrrcnOETLV2gWe9SDq40lF8qY+0ju9rYUHzugEQaFYFsL\\\\\\\/onh6XBw68LD2UqiPltUjg1+wrFssIFMmBHnqK+BXOQOvvQbRYk1As4aru0huVyFCL1lgl4FTM3uUxeLVCotwSQNwoXS+zav\\\\\\\/iP2\\\\\\\/hmXPvUXsm6Zc1ueB62B0L3wvZd8CRO4fGgOWXQD86X\\\\\\\/Tzs\\\\\\\/Q7ZqzLLhhcWnwM3zkDJ2HlXa0pFDstwKv\\\\\\\/6FXlIQs\\\\\\\/Z7u7v4va5a\\\\\\\/peCvm8re77EOj4zXLw27LaPbKNIZEFm8dMpMI++ojYyTIk\\\\\\\/Xx9EM7gdw8j4UGyfhZBNBkSqNOFEu6gnq16zS3UQ53OCO\\\\\\\/o3gF5UFIo+ZqYE4rJumZ+KtBS32XXpq4Hbk\\\\\\\/e4j5lFn0C3Ov9TBTjzLY\\\\\\\/qkmCl82R6Kd37fBtaQgHFtxFyI9CT4OfrEdzXMAdMuG1pd7MM7ASBIfK1Roi+zACqPLXGYPWQLb4n\\\\\\\/rPz9ztpfz8kO5GCJ9udHoEo6fc5m5T3ttBt3EEQ16LxUtQsf4c8eUcJoYJMYrkSEgnC6t+2xCau0\\\\\\\/CB4XmwW2wbzhtQhK4ve\\\\\\\/KzwOa5yZA9EmLPYI+O2yWP87aocKt3Q8F4FimLbYJXExuo3oS5tZDr5VCi0tkbuz5OA6oJHmVhI\\\\\\\/F4Snyf8J8uHRf2ftTXPVOG8YrB1WToLnkpVg0Ob9uStJT6H1wwZy7oKmlAqu4dvcDMNwY3nlWeNYGtc6AeiPf8B2pLE18ugcPVgwv5Vot2VSnZAxA\\\\\\\/0cgMwn6QlO88ZWRKtyhfkED4FoEtIjDjlN3NRgTAF\\\\\\\/tVwYBIpvIlBdfPwclRfZi1vYR8Rp28td9zMwTDXZE7DGMMIA8Ms7yE5R8dY9LGUNxULpZ3wYVkMfmBX9Ttu7fem3HJ6Wmp5DBPNHkxI6VNxnJuGZR1905iAFCHSTJjT0YDxB0ZdRrrzUwUBkXiJVNErpLTC8a9owdptF8+JYhEsHK5h3iVCBmlV7ZWkgqF8Cj9Lh64jJGBLpjTz\\\\\\\/juZgXA+J2Sx2RGpQMR\\\\\\\/pH59DzXlZrxLb4rwhkDH5y87rLX8Iwzmys+orgvUNuKw2nkPgFEBY2UNA65SFsNQAU7yZb+HWpI4PZVPSm3oqv\\\\\\\/bLa1tdcknIr+URTKfW\\\\\\\/X4cNDNigNUqSp23JrM+gmXjeATc4n1ptZaenIslYA3nWGjjEs1qLe6gJ0ZTfNOGu1zJsQLMn13XhBXUuou06yfuP5NSu2FSDOKpLzTxkf\\\\\\\/yNgBwTOZ09uBiYE1PoRYs14DGYMkWI\\\\\\\/GlGlq6p3ryjUWv7yQAC3wcRminfsPmM0UuS+SVCEuAdwF5QPk1KAufP7tUI8Y0tIAda4HDXHalvMkmxX3tGVwzFquZvUfVKYAIkLU\\\\\\\/BG73mnfaNbxmt0QmAjx1Ip26znlETsF1Z3EyN1M5gKSmGRyw8G0ptzIrvrZoJQRCUgLU9u\\\\\\\/OVR5nkG86gfd+0fi17LQbu2pae5PTnkpaD0c5h1HKX89vT4cPGWqZ5CJ1mpGWLhAp2\\\\\\\/FGn7ltr7VGV46YiZyDKoStM\\\\\\\/\\\\\\\/UKXLfTOZSw5mDaYiyQ5HGblxJE005IuBPfgPbHAig0H5XvOX5kFd+xzERF8MyrtpMzN7OZnAeO4D3+mEyhiG8obOXzYKSciO0e5O\\\\\\\/iuzUJFbYksc7lVe7yVc\\\\\\\/YixbtNRrfKsTbVxSTWKRJ1U76zoa7e156HqqVa+RF7OWKk5YITtY1Pwqr+5Hk0WADCs1wB+IMfxXu7dbDXycRnLa\\\\\\\/sal9KWO\\\\\\\/ep6Uhw3GmKtpnhB8m9s65hdYGAPjJvGYMg5459uZvG+Hq7CpqDklpwEeRkf\\\\\\\/OR5s4Ie8eTyqItjsgaoAFNGNFcShNOIrdiRABhwY+T6HQcYt2ptc5Zgs5m0eC3v2LjISqHsS268E7HSuaS7Q2eao2w04PWE4VhC1XoM3qGwQac96wP9\\\\\\\/Z1AE\\\\\\\/kTXnw3GBeXcKcaO0lhI9HCp24D4tNBmlPb6r5ka60jVCtcLgeUJWXCgJJMDhXOhkebahwiAj1luKW+P0bIwcDYCXuhhhVCPjwxdplO4VQc8gnqJPL0ySIZ0amxXKIoir38G4OTzED1gv4dVfQMQqmNLD1UJOFYJozzRsqMfjENYUPLZand7nuoOrihh4DbU28xrgN9TbxxGbHCqRiLmF+FcaYQfSQaMoazBjMXz4LfPLoVEpbvbx8JRFC0BX23MvNRmv39k9V6Eil1g35NwJKoMDvnH0xeQus07Hdtn21pROwJhyTg79JPO52Hf0k9Z3CwhLTMr19X7toKFYTnhpBmMEwc+oyYpIP91cEIopnudAstzZYayi2RiSA2sObhbsk2qqi+puqreiJT1AHMQqdweK+6c0TKvDE5UzHbZXlraZscdIWpuvzntcwR+f5o2Gc8796MXin88hMvsU499Qi7TyhbyHA1Qzy\\\\\\\/gB2ZOX1mZcRsOVDG5+dlcShF2InTR1+aPIqEy+QfeWsBEqyXPcCl5UyJoiz21yC5nNqPYcLig\\\\\\\/8G\\\\\\\/4I3XgEfUd31fHqhMK1zfVHEPQbQmaC2yB4kHU2PAF5xB2FbYGrbbQce4GmGHfMvwp\\\\\\\/pJZsNPTSBbPSefFJM3wuojF8UaV7GRspaLr740aQ12jf8\\\\\\\/SzROPAhPlURO9aDPZtejRn2u29bBt5kgICLNP92vQPWDyjdJ9q7KjoliAvPfT\\\\\\\/\\\\\\\/2oxzzeSCFzhp952JFTqSbNKabMPNWPIjoJNBeAHX4SGMPc0z0\\\\\\\/MbV3uINv1zslykxSGndtAJ1V\\\\\\\/wfk6Mswvz\\\\\\\/9HStByvChG1Y61rJfOYG4okyVTvjWVpQLGo7c7ssIgvpE2Nx1WUQK+0\\\\\\\/ORRVDDpExBhmV0sgqZTbe2eMbndd3IWpVkMtKnUNSBmbPT+gKfColtmCg0pmicHSiSvSw1ue6yMeQdDg5V1gDBIlDEPgLO0c6EkqOCGzIgopKJHJKAHdmLr2rv61GQI0Flj3SwwMuxAiIESsiU050JPP2bZRsplOWLo22Lilx\\\\\\\/ADB4WZ4nuIZEFQ6h51rJwTXTJzFoHzbpKzbpNVCfMEZ\\\\\\\/pzysdUScWm35Eoso3ypqJmmTZfEkh8ytaJMn7IT766nB9C++dSuFCg0MDX9ZNzknQtpI0sL3PDZSvgrFRra64CKLO32Y+93GMQmi41oc4c7dbtEpX6wrZk26u9vPoJl8wSeVx7FRoxqCsl+40ooTsZeIsMs53060TdldPNYrSgjT7C9uUKRgRlvpvJeB2YFtSvrF0VV9qNpSVWCAXbZNY7um9XhOe6Hzm3uE697Cp21aoUepCdLCrPa8IVxbXZb\\\\\\\/4zs\\\\\\\/BpSaIvwj2qoYv\\\\\\\/M+40hcY2MOCnISKC0WFmMzsAp8OMOngIxEWR9RZ2qOTLl2YsyCjfVxvuqG13F8xGw7Q3Xl+IFWXzTUAdXdawxNjxl6EcXxgyfLcz1xgusYjYWGR2xmiAvm34W1nBQKMn4Xs\\\\\\\/Q4M9oKEOp9jkIoBke\\\\\\\/dRRGlbrs2N9tUhI+7GBDqSmO52FnDlrvf6ayx+BIlPhnESXpeiGQUk2b3N5d0tM4xC7e8tt1mXXkP7fOGs2oZsnttQCMU7T1zx43qdXzvimvSmkNNpOKefAqveewiXuk35MMyS0hJfUZIK2J9ttzYbS72I0ZwTukbXO57wj0BeDKk4ejyJxViDT6Kd7IWDgnXF+lJJUaeCEIeiDzaEE4wkA9mNtu2gUEhiPbAQWvlKzH5UqnJVUI3bUcXghSA\\\\\\\/iXYRyFliNp6xEXHXXyiN9q1aZFfMzMZRJTn1ZzF5xpScb6WFSuRu5kj32NzfyPU7MXFrc7IliUAwhz2HAaAkq2NkjbtnasFb8DVgxpq4R25T6spMn4JdXFpL5tAa\\\\\\\/n8mFq79P501GqjaoF5SXUvBgRKma+Jmk+\\\\\\\/6hYswsB6dgtsL1l0J1RqVXJalnzoIWUPN0\\\\\\\/JP4X\\\\\\\/88NUJh314aA0ZbAkXC7K4qTyeEaiNhLye+HtSA+s1uM9YpWSIT9J11ntcNCl1t\\\\\\\/oCWMqsB7KFukjTGEQc7YfEExUXMYxER1FXMN5xzaa53yVKJF7ncqxPqO\\\\\\\/iyQ9xX83TfJ75+Bvu7fn8NDeuUUaXbhJnL\\\\\\\/wCrHlWZTFZGj7seTi6O5qpYqKWXKPPOxFmEMJwk8SMuZbTYgsvQ6rzPcgrJino2OSGTucHfL0XPKQ1AtRnb2w3QKwOnRwBMOwdvovHAKFMoyEiu1RzeV4o44W4S54flUUshicMqx2PGD9LRq8uwIdHXPONq47g025jh6g5ZqesUl7NvTAZLWcFGOBmKoVD+180zSrLyb6yf\\\\\\\/ZY6k99T\\\\\\\/Fmcaa9e01VN7gTh3jwTZUNmpg+Ew+s4KrGt7hWs56ACRid2so7WxCW+ecrTYoHjL5+NV0AvVG6qS7IfbxtqHa3UFRKzzVPlwcJwN1cKlQjVF0Dy+EUszLAmLab0GaOagXdIkeflW6l+cfFZfi\\\\\\\/9YHmaRRJK9y1g3tDz7GZTMjWLXZuZVapDxBy76sr2hORgx0p4hbW8LiSExFfPdWytLFqfZc66Rgaw\\\\\\\/3cAsW1CnfI1LDs7HInkNrwuH4\\\\\\\/w9ZSPAGP5bbg+Fj1t5rY2qzDVfQmsCMgiuCOc3QVJzhNGBUrrEUYzqP00Ox\\\\\\\/8keNZQTrUZIa4QmXegviSoNCvOsxGEDVyJ8895ol3XLfauRaEBqZY01GWTnC6la4tUa\\\\\\\/bcpHQCOJiiLkaxqveFc1MSC1ZROw18nVoVoT2NDWCe53yXSTGGk0S5lyuAqwCjEZ5qYE2zYfdM73zCcAx2hGtuYRZkRk0JFvP7WKU5QDe8eUP\\\\\\\/gvsw5wANQA4p5PYez2Y3udZBm+4KvoftugBS4AqJwg3+d01uxfaxkYMJt0ivkLFvjC35uEIOYEFZ8nDB\\\\\\\/eHYkvWbfEKey\\\\\\\/K6Rntxh6HxJ81s3VxNy3Ib3+hPnvSkudl3z5TbzcFaUvUZs5R2Ifjt2qtta4emLrwGzriup+0IVRO8hwLiXBKmB9gem5YbanDGvfTgMb0ecQO3IDY9ijm7p8a487LI4tEHrMZKIcayd7SHYT1qwEmrwpWj1IUOmmVSwHJfNabVfma73hH\\\\\\\/cdsrvcLrNEaa8ZA46OjxSV1Q5ZM1gG1xnzY8foU1Z70TAOny852X5LRoZJ35lWdUquNrFOIZkqfm1CovfNofQBrbv9O2BARAztqnFu9DHtI7zhFmfWZj5RTLKYEfWAJsC4RxHUamgM1AOFQy71ejV0qON9HXNeNjq5Fq8VTLtFom6wJOhSIfWhC98jslukMxkvD++Zraq2zWTEZqm6NhSo5eQsmUeWQpoPtJio1Vv7uLX1RktpOqGzi\\\\\\\/kPc0OieiE8yQx\\\\\\\/b5atjDYQjZ7JRVS6pvqucoemo6VuYWVxEe0reRcnhLZolKnAc53yQDIf+KcQqXDG1EkqrbB\\\\\\\/sYigods50Pouks7B9T+Q7bRiVR4mr11FXFdyy0\\\\\\\/YHUOTujWjFwNY0JxKfrghj4a\\\\\\\/4bj8fqna\\\\\\\/wArqbHRl7OhWajRINEm5tps2nZa9hQjsnYQvm+fZAkKQXVQMvlYWnTSsvB2R6uLWUVzjE4NreOo01BmJ5iNnIeT8qTVuxSiGTYHMf+AtjFYqmmdRm9Qzej5+hOBMWlgMZ9+BRdqLGSEG6EeUrTmdlLMYuSCerKFi3O9gEVLK9S8bmDR9wzO26WNuFNfDJKPaLYj0njgvc2k\\\\\\\/wcAoavzG86WNJoBe+KT8ikfZGLWRvOOWX4XSvGTzdavhQ\\\\\\\/DXeKreKLg3+V4WsBina11waVvwcZ3x3lCvQ70TVmcaY1069sE9of9jyqVcjYZ5jWbSuyuuTCAsvfUmf6Ww3E5mbM1wgFWLW\\\\\\\/d7u0eymBqjqIV0r4rgM5FoLR22EisN+pAiqgMsHVXUAnB2i9V9Zi6MXpqbhJfP8nJh8AF+erB6uq2y2zbq+CpYoDlCxTsqaUMERTyPAm4KcM8vS4zi5OkO9PUk1HTManklSXBUwc+IELONWwHmMQwgpxRK9TI7ULeFwVCY5zjI572QG9eL78Ds97r2Wizx\\\\\\\/NAhkyX5xqDI6U89kxnIk+qufNT8UOCnHppio1n0BQbYaX1sgZXR3oVYB0IFY7WLIMZ\\\\\\\/b7uYLL4fP0f8F5kpebsjTHqiu9C+iGhxdF7RgGRHsFllkiSP2J6IolPA6X52BvRUcsHMdQfkU8qH4c9xliS46Ct6j9vk1KFF0kWXkSN01RPsgRXNBb3d4r1jmXUTZfYcKSCTl4JMX2\\\\\\\/CX4d4HoGTjDqYUisubT1ChbCFqilBqFflFNTKJblcNAdVWKgCkKzuKQpTKiOcR91epMwqQF+evoyVtGb2DCu2MqwO6CCaarWRO0ufJsIVLB5iMwG40NVB2W9UFgyLm\\\\\\\/3heOF4ywFrM3UyFJkjZ5rxmT0lWYHhg9AdzBjF4h0teInZNJjqybtbGFpzRs551EVgnXR92kBbkFPYJ5eRpS1zWOUqFXa0t2ROvFYsPOhSDhYwoZHkyxFkMPYCBdOUMBbRyQLgXH0WURuKaPGorLnjnK3sZ2Qy33JFyJNC3QTXf6udOjnaygOdSVbaEF5fSAQsO2czV6mBxe00m85T7VV5KZgCDx3XD9NNQ17VYEfjqrDdfMFBa9w7qkTyGZ1nMnOPJ6Cls+4SASKcDc4lR\\\\\\\/tCjyjBm36TS5dy4oy3p0qUkvxybX\\\\\\\/mIheEqYVOZjVqLWbkwhsS8OUtur+CXuSxECQcMuDmEMiStpGv1E4oVwR3PmET0sVBnHBKCZpsa92VCsm0vXdYpHyz03VJdZ6uFOHLSA0j8BFHVEet+6VzoM8R\\\\\\\/nyB2v3+fyfZvtZWMhgDQtyfrzPo0ggsRCSPIeph31ZBnOWkgYX4vW\\\\\\\/V5k5QhOuCW6j3XE8pNggwnx5vVOp+reneIjHwMCjgOZNGl7V6z0glhUjV4+fYFyOFUPBUQ\\\\\\\/+w8ffEK0PliXj2IX7dRzkGms8ZnqyRUf0GIblmrDtNml4TkDVPuiS6O\\\\\\\/8FrR9mxcP35abKbvCHci3ixWWxEoVxAjgI9aDIGIHKLg\\\\\\\/o1rgrtgrJk\\\\\\\/Uhb6eIQQ4ekcVUVtsysY+3+FE9UcL0ol+llqA\\\\\\\/jV0YUtOL\\\\\\\/rIMp3PXN0yPKFCOaU6XsBrz7vUkBYJjklIfIljkpnAnpWgEGovnslhDxYSjNoFEXt4\\\\\\\/z+uxoy8UBnMF3XTEzhA7v\\\\\\\/qmTLD\\\\\\\/xOjiAGMcGe6KMn9YwWZKiDwvOrSoeb9qAwYVAAZm76jhhyrb6NC24kpZnFEjhesPZSLnfi\\\\\\\/gKB5QPnsaN726hjv5riyNt2IQ3Bkth6fbxq12DAf2fbqfeD7NNVoF+I5qxV9d80K3zgy1lf8ZqIRmoQgRA1atmw9tM3rh77GyvC7tfHbAonh0yYlTAKQyVz1Ai0u202ZkqTZMKdQxH+Txey7nflJdTTxYIXqBAmzgdVn5yiByUhnAWCCYLH8d0OjSfZU\\\\\\\/P9mJxakJquWdqbFXcjAT7tW2PETvdrtfns+wIVTMf5IuyOF\\\\\\\/MOMeZsSPmBMH3zG9on+rsIKansexAKAnCGBDE2LJFxRZVR9ksbBHx1zVhH+4kRwFXJTwxsstnGMuy5\\\\\\\/OvHgM+6n3zbZudSd8HNID+0u\\\\\\\/dPPS6KnweaEWYEwhHfQ4cJ\\\\\\\/OBmtHFxVtvEFowhmxngPgFPSE8yZYG0YQaveTWxwFnacgrZMsyVL4pSnUE8xCOL1\\\\\\\/o+PXzonEcaemWSZGeLY+xzSzhW6xYaCPw4HxlTI4S5jAHkXv6kmIGqecIS8Aj0Jevm1+V395AL+Wx0uDE+BG\\\\\\\/v1azAnxug7GITggOwn0xXIMx2ZoPnzqYgGB2xC7NG0JnY5IGiaKIfYAeaTZJAZYgWR\\\\\\\/Iftm6247cG8Pgf+\\\\\\\/ei32iVGjv3Gyd5GyF\\\\\\\/dJy7C54eIH87AFqNrhGUAgQ0fQXbimCS11WwRs9ebGLz3Apv8VPIvfLPFk5jQvvGsP3E428+ssexR5cXT8xq0y1e6AxNTy6IWBEqfO0JVhbbfy+xVIaChM2Zt5mdLrhPyud\\\\\\\/WC7rCDh38NLAP\\\\\\\/t9rQacDpU5+1gGHuUhnFarLRrpE+eSIhIf9yUIaSb05fJ2+dTpImYvYJNFAMhvYJyDPA6pQJoZlNzoycBxjKnG5iM4njkNv0cfnxTqQnIZ78Mswwd2vEg1SG9oJQf9Cqx2GQ1FJ1hFYHnQv1XRCNIIvP+9fCBdc3ufju\\\\\\\/jFrwhDcEufbHRYRoGEUWQXRgd+jlkTFO67XAz7ZjbHn3GNO7oHWuBox+bnP+J4NHTuZSsWc6SHaDj2dm0pzDYPbwgikJHgESltsR+TLF9VVtw0oxvN5n\\\\\\\/QTexQ0oUxw39rREm4x3nSTcX7eSpT3wQ6X1YUk7l97Gl56oEYqwAKTBFat1yDWDc4pYHF+EhYJYI5\\\\\\\/uLKomjlq81XIC5fCZljgHHNJPCt5gPSfITYGEp2Vm1o9JoNzsmPuwfI1IxY20XBDCrTKhUlWhUDnJn4QLRN9Taru+26lCla3x9p09muIYXJUxijugOV2\\\\\\\/GJig9dIt\\\\\\\/BcZG7JoBpovKyNHQsrP1l0CZB68RCNmv2O15LSwltNWFu37CsaGQOiP3bCd\\\\\\\/7zr8HH79R7VTz+1\\\\\\\/4B99DK3TSde3OB8JQGxE2cXXk\\\\\\\/gvUAkilKYF6WOhQ3c9uJJC3XutZQ5qoiWik6l60fos2kWYabZkkGTetChPRl6E4doCAFbUfgGlxQxSea5duwsXYt6kX8sVPeqomrm71S8nv0TKeUVi\\\\\\\/VuystGXbDd3qdW+YVHBFhaPuHCvCyrtvAO7FJKgsO9baiaP4DOeIoENmU1KeMIKjdrGCtqNE0wSJ19IufRaSegDqEHi6jWh63LxHYa4c1QvO7ndcV9XISPTMlu7cO8LJZWPz33Kq5qQyYqT8TcBghL0A140bCmXvXkOsgD0leTxlcbYUGKA01wvxwF2InjQ\\\\\\\/C12mAMEhdFIJStRIeipT2LIuYH95cFXvtBr9N6rjescsnwhCRXV6dcNah\\\\\\\/9+Suu4MWf4vxUpyvtPgcZAnERQLZdJ3RvAq5uz8RLHdn1guRwgIOYd\\\\\\\/Z\\\\\\\/jVsK67yFFs9TQ0Y\\\\\\\/VYQOypaGNkp81rlmO\\\\\\\/Y8fhGFXdBWNhrxgZECiczK2Smsd+VkdK1vISVoe3LDnWOVWar0ZIfqcC1rNUKMDN7sHmcKptk7YJlLSBUUHvqMnBsEQyyscrEg6MHhpWXU76OubIa9MS0Zk7ju7Xnv0LCiPqolZWWELWNUqzlEMYuEfk0JeXJ9BTQoBkjUqxQUsg\\\\\\\/CRg\\\\\\\/R5aVSTUOKRsda0kgUPKx332G5gwXEPGVYRGi3foear4O9K8\\\\\\\/5PzlfvR0yXd0ubhrD5BHF8x5vXLcF95+0uoEIvdR94vUt6VGgqQ5LyF0E1WlL3qqxyV6mteTU2PyXbI4eNTU03FQ+kA6zFa6o9Ji3YkO9EaDDw5aKdczSO9vdhO\\\\\\\/DwOvEmV0\\\\\\\/5PoKnIMDWLxPx4K\\\\\\\/9jZRiOVeHO9Ox7nqQ5aBmGawhizd4JZXZCjTgRUmpuQFPirQHy0T7u5\\\\\\\/Ji66IjfAe98ha2sk3Y\\\\\\\/ppR4Vbgf37ZuQUDvRWMwzJoozaDyXGhPm2Rqj\\\\\\\/VJV4XulONJy9RzXDFNRvYgaF2BKXf4dtrKDvY0hWzVgdg\\\\\\\/Fc12a1gk+Zw+e16ouvXLl+tR5traAmMXrTqn8YC5K\\\\\\\/vSmvWiyGzr7ieP+xr2ZvF0e5\\\\\\\/Bcl6V+4TZKMPpoaseZOB8l87kE22+pFVzxOQgn\\\\\\\/3xVkA7zt+tCsHezbjbvRWrPkG8p5cFglVdglOXfBoIjEC299KgRorxnu+FEaA\\\\\\\/rCYRSC4S9aSWcJUNmWln3R\\\\\\\/7Ljjd317WN+Y3\\\\\\\/RQOkapHIEldfogTCyT+heJggD5VjS1lnULtOse3rjsMqLoh3Eaex0pdu\\\\\\\/YoKlVjH2CtukfodTztAkzXV\\\\\\\/HFkEVq7ZEy9SeJHMojuRaC\\\\\\\/sJHXMIhfm3KeYXGSWrgKmtHpLmK9D6eUh7wsppNeo6bIHl\\\\\\\/+KgtrYrz9HvmIKbiuDP26KdgthsqzKYmQo6RBuuvaPoSetQXO\\\\\\\/GxmZ\\\\\\\/JNooq8by10CHPMl69LDX32keaaVm46NB8KU5BXcJeYPQaWPrNaM2GTsdcHvVWB9MsIS3ZmnvVoeeFvGsULpRuiSk377cuaRafalBbi2OonTcYCifEwr8MC5AsjMNudX9qFaBGZeMo2gAE5+5jQebeYCuk+HghMuRptH5Nda\\\\\\\/+TdnYmEuGpG+H+W18YGwzNSreoChZJQGFW65acSL0HoowlOJCIZFeonVqziG2X8v8K0dQASQQBjNjwX8Fl2FpZvYGQ6e++oaFTVGKUTjzZhbmg0sSmGXRotE137z1d+DqDdD4+Ynb4Mr9LYD66prQDFRVAdRgwymnDowPP3fNZFGI95n6HBMr4x+BR2INNi8uw8VoVolecGUcQkKznu7uE3wmmSJPyoF4dX32BGTxbxKJMboxPz0bVywCBBcDpQ\\\\\\\/YfIWQIuRFkvGwG71lfJRqGHZ26FTzTjAxBLrXjsEBrCnAhDVSwsTVs75jcqmgQXgRmdomVc4M92GUAxX3QgzC72u\\\\\\\/j4MQFQh\\\\\\\/YVvqOvnindWh90rCO5cPd3duIeJvhPAgAHif8ULJHBTwejfVBOHQ4+q4JNRkRTd6xaYsn6bQFfhXyxmC7Npb6MCQg3EmdVcqi1TG8al8xlm8CBK\\\\\\\/yGZhYeQJaNd5s7PJEUV5QYmTCnRAA0VuEGlwohf6tlq3kMMGQvGurrp\\\\\\\/9UsTxVOg6elV6ZZQFJvQB+oTQRh4a3jd5JVNeZY0hQuw3QpJYqPXIAAhWe\\\\\\\/2ntmliJnC8U92kpBkRNxmC2A0OvVFE3ywBK5g3n1NpAWPlKzAsfULM4BTakQyWDmfjRbEWgqAjs9bf3oI1IF8kXDuZZeY+KOqgspsAPgtXV0z+zRbJZubQUJntMarWQV8U5DUHFS8PpfRbHIgjhSRTWx3DTeJEjGoIPwGqVHfRQIssvtVVhkLpG\\\\\\\/9OWK90ijr\\\\\\\/LhF9vblEGBowZVPrR56T8CZEhgibgWp2D5tLDMJ+Lib8tmSYTeSTGZcQAZVpU8fKhwcCU8DT9+OkOD+ETCmdbSMjYLa9Ddb7svRBYRzQi4NqcETUZ+ILr9ECu2izzP\\\\\\\/VQ9rQ2htO1Pa9ziXkxnidVRjfvCpgAPLgmlqo1Va3LMV5ytH60SzGLKPWVRGtqLclin\\\\\\\/auuoMpCsM1FnjHQTEw0gVMCkxj+RCRbi17BZjFAnRTyaQ84i7Z4WiXEycufhVnsGrKrhmWDkPZpEVTh9d5JmOkU88oInFCllekuoKkaL\\\\\\\/7G2MRy2lpF\\\\\\\/afoQIhYGdTP32C4nVdhiuN\\\\\\\/LKaOaznBZtuh3mhWFW+b6rGiooHxIDiFI138aZDqV8QvTPfLdM42zklUb0DraflhUIj6ml4m2ymE7bA3fB5aMztWgs4jIuLypbH1hOHmEr7KaPthWobWACTcwfiTy3U1rX5gVPHAJAJC48UAeFja99tzUP3bQYTDyvON8BcL9avI7KB5rTmw\\\\\\\/mPJjGs0XElMryGPq16j5Oo5tpi4qwc5HloNIt\\\\\\\/glk9UBZ4B9GzCBoa4tOZoqMmRreYbzMCzrlZTCbsDAWNz7y8+2i\\\\\\\/Y+7BjXo6OYC2gHd+vzFsC4r8s+LFwXb5kWE2Q6fOnLZdOHQFVdpWvzicZzg\\\\\\\/QSd3A\\\\\\\/JLxivXPCjuZe7o9CE4fAQVX7hhKmCTdexE+X49fab51tQW+GXbD9rlCpPFa9XcwxNIW2yFEaS1GpLUIF3M1rAVSFbwDsS2PITezFkGAhfej9OLiejjJNr\\\\\\\/087Prm9u9\\\\\\\/UkLdQgUojBSW0UmEvwKbeCOFakrbETV3cmyn80pAp\\\\\\\/HE0En4q2isXR8to3UFVeY4Vo2gtwzVwfryQwVqfEpyD+TBfOhNegxXWY1pc\\\\\\\/ZTbUsiV5wUW5vPmi48eQsc0rHXc0XGAAnczBjGYp2VpknAfEJ77P4opAJz3gta2mzOOmcRm7r44ghqGmOmx6JxRZ0RSCsefDSVWvRAneucSLFcM5u9GbhXfSpjASM6RpVeUPEyQxRFFTghI5O3aGOnz\\\\\\\/8ErQ0Q5Lak3zy\\\\\\\/vVNacF5ztOmo+wKzjA+CqcHj9B9+1pwJMXhpO5rkf223fn1YeThh3KF\\\\\\\/YMOSTYSq9jyLj26BEw90hyM1yP2BhG7gwjgOYfdExfHTIsESpQGWbJNN+0\\\\\\\/GUZonDs7WDsTth2KSh4dOwBlZt3kt7ertKANLH48Rbhj1dgadfILarp0bs7u6oTQjqugixD7cedSIMLm3icxDr8gTJdyFxH\\\\\\\/JSuqeYA73R47IeJ1IbZfT1I0p54VxFyFhV7Iskt0lAtb3W9dlCmOs+hOuOUQcE0sDB1bMIoM1yRDngliUuLzVbJQ93zSt8\\\\\\\/kjohb+HY2BO48kOyiFmtmW7CqBz+qtqvGzKiMp3qNq0B10aHh9jWYgZ0MRdPwOELWdaX0qA1VWeJkSCEG42RVf7zSbu6qPXOHLdssZfFnl7Aa2+dRBuBxP7XGas01OByK0oPhtpfqQNLfN5QQ8byLUqYjFoxGK5PsBGCYiRBqqXLBOie2i5p7X0ejo6N9Ksw7LKKgzN+d9vpsaFex5A0gHnA77DCVsnj3+aqw6VA9KqMDyCPrzF6zueqe9SFPXfpsHmqrz8ONMPnD7kXUzEYhHr6KzIPCKF3\\\\\\\/UZxrklCP1bGK\\\\\\\/BbLUQCfcjvtMjECWvpUspgeQQnKRR2\\\\\\\/eCmTS67pj+2QpEMyFGWq2CeRmq2PXz2DQin3N6hMyg89Lk35Jfpbba56pvsAx9Ly5TFFLq5Q34XSbedExzX\\\\\\\/mW9e0ghRdfvFlnUJB4EvGzmPhq7\\\\\\\/6Q3U\\\\\\\/U1jBzYmDSsaAhTmexxaiGMz5HeyqlYUWvp5aw1\\\\\\\/zPPYGiRJXp5D6sRzt+t9QR+3Yz+UNnK2OB2ZfK+yGUITgNBTQRC0rsTSs9e53GgLaiv3\\\\\\\/Bf9MdLA7T90GA5xu1sGQulUdy\\\\\\\/RS9xNnzkO7aPHsOKDJE4U7\\\\\\\/SU9435peiAfeWU4BbfgQW7eb8WxCr2VJvKv9E4bLM9YMwthf2RPxrFPmHly4NF6XOdUMF77n2DdZ\\\\\\\/Jw7FRa1nyni+Eb70U6pU+AIIyQ1wR7RThYWFyyianWz2IsnjFppWyVb\\\\\\\/lpriU4xhZ8DCZ9oezZjCfVg07G17+FBhOS\\\\\\\/PVluaJVnTn6zAeDSSH5tnuS57ujEPo7Gbw5fvrthktElGRU1JPU+x8YWG+7SwoV1YyfzkOQWSugtcaxMazjRKp+CZCxUQym6RR+\\\\\\\/naJdPokpl1T6rPiYwWetUvwsSn95I4oEv7EHJ3vX91yGV5pCclRqiAp01OLOnyFJderCVr9r3xdOTVWowdyClZugkTIGuZ89u9DKQzfrhoY+6sR6\\\\\\\/EKI0EPIjHnef+kOcDh4GxX\\\\\\\/P2qrqgK5FB3v83w3SkNkXfTLe6COj\\\\\\\/p+msFT+lj6Hwooj7DcuG4xnvhBES2UNrexqql9Q09+Ajrwdyr4wwvyK0p+f\\\\\\\/SN3YY5mns9ImlktSWBjAXRy\\\\\\\/RUuk5ix3pDke3M+HVoh0Kg5mUAgWC\\\\\\\/uP8cKwpgkPVHInWmlp6F3lMaQWcbJrac+9By2odtdZUkhhz+0HlXxQ6aktFL6Zah\\\\\\\/bngnRqS8P48eo8F\\\\\\\/72EealrAdrbIFKXFTveiAUtrQ4JWIntv0g2Zeq7AO4E4qcSmc0MWDvNLzEHYdzbhkTJdszXK6i\\\\\\\/8FLe\\\\\\\/bgE01PAFzVy5cKG5eH+1GjJW7metbZN5YPjzXV7Two6ipCOzCPpsYwSQ3UI1DSuHpVLQQqGP0UfVgwLbbinnUK+fb\\\\\\\/TIILj+PhZQGaivGeUuItET9MJCcONW61Hos7TUoeBWWoIBZzbMP4GK3iMAY9PrX\\\\\\\/fxrqVk+8HxpFzluqb7HION+8+kNr2PzszA8UMHYyrl5XeiQZyW4NzP64iHjZ\\\\\\\/aV1VKJaBNEpEUyXBqiVdjwF2o+zteNJDRVcr4XvwEbkLK4iM1J8PhisnHGEQujHG1N0SyTy8+LO3ByOUFyBTt4JgjG6kLsUrJ7AqoewjUUP31eFPSGFAI40ZqAah5UsfpzeMifwaLStVtP7lEkt55TwHxFWTWCFkyZxWRyVfAd7D5LaZAoPw3VijMyqSurjBFA6hxlCpZBLLQyIhoQSaHv8vaiTv\\\\\\\/cT3xQnqMqfWmDt1pTs5cN4UdZ7QFA1jzN\\\\\\\/rBj\\\\\\\/CvMrlpl0WOTzl+Z0Ro49NOropzFO66Kt5ZvwgKb31LYXpCHxPZoRrjn70OUoDrTrOj6bYPNblpgpf5eRMOLRwZxEICdK1oamQEaAC9EKKExvgmC1gkpxSiVES2VJ4CuCtNwJynImtqHAViiAfgD6rcMZt7po3fVvAF8VUOveTBXV6tFcprx1JYaNyH8smLYIQ+xnXJQQ4g4X0rx+unap0s22z5TY5s7r7O\\\\\\\/MWUzSOikx6GRiuGOXCY4EbP1oscnIDAgMDEirlvIdO5r+GVit7XATxZvDuTvyjnYlOip2yRiaL1b8UWz4Vj\\\\\\\/nRnGI2Qhm886miw8hL3YYNNZhkK74BEGfuVw5Ay8hZlSJVoE1Tlqae\\\\\\\/r6ExavYp\\\\\\\/gsPxL9FP0YoqVa93zjd\\\\\\\/tgbTYzWicA+ncJY6TmZYGB0nmWpcPFnuPi2VIWZOD4XJx\\\\\\\/c6nOkccaAbKkYv5\\\\\\\/uTruENZp+Ol7HxGq2j7sSrihZM5EhsjIi6nOjHvpDP6KsP0y8U3ssjoLSpfX5VNRJCRPUS5JChD+VHfgYj5do2ptQyRyTBnGqPvSUL7G4eAQ\\\\\\\/g54oh79QqFPKi5yC6FceWKgkTabMgkrZedog9GLV\\\\\\\/TfVhSFvlz7BSPdArwGAFYIF7Gl3HYwJnCn0AvxjorizfyZEvFOHlB2pWtzrdPHk\\\\\\\/0fRQCB0EJs1f9+2ggbgIjzuW+KRjvkTAiqNwyxQAv1gx+dv3GIxq+LQTmuPL46POdU6yfpAyDydPhPljpRZoRX5NKU4\\\\\\\/sdqyC9e1fShFcPgpH4Ay4YKoPA21Up0+TyUmveuCD28otXa389syQN3JkpfUvMzoNkAgpklmPlDotkKvtE4H1QJWbW9n4JZVN3jo8d2zTh8TsC52yvl9WEIXtVUGMN0OEB4yQU2g7QSwYhVPelLz64uBOva5+L18c7fsGttx9QSWhRbtddiokUAwEHTv2v5QGR6Z0T+Rrdaz0U+hm\\\\\\\/nUER8CrDgGzy3F\\\\\\\/6cjIKztujZRiLub6wtx53+2kggl2tHuixfG1dGTuWyuwmEz04a0pT5G8RYU\\\\\\\/HcX9uCVJlJ9MFEKK2pyz8chpOgX\\\\\\\/0IU2aioaA98jHLMIDi+L\\\\\\\/R8rLLyB5yEQ\\\\\\\/yFAqAWHI7Axopk9E+wfZH7bWf\\\\\\\/AjcOkho7ZnfjposEvGtlwwoCK9zW03tLKZpvWkGTwxH8VQHQFZFj8GdkAOuT\\\\\\\/04IFG44n2NUXWoVygabHpQOYmI2ytL3E6W7gOblgij\\\\\\\/IEFRoNm+RdB9AjD4UKJcN6LohsT\\\\\\\/2cePvIfQ4t1EYpd0DakDqIKTbLZZkL4pAvbNj7B+jhDqGdYIRax4NEaBoYDV6ZK8T4\\\\\\\/GFb04bfgoo7CZLyeFu7MGcMlroDzHNbMZ7Pcu0Lh\\\\\\\/OFBce1UieavGinFDtzqlSLNL0cq0A6y9wf9KQG39R98Ozc1ovaVsYI+CvZ0icYiZec2\\\\\\\/BxHPhcmIAWeqTgoGEpblSPUxiH33UsSWS9P8DIMSh1H0ynUQkt+P7IuR2Q9gC4M4uPahlRB3V\\\\\\\/zTjxRAd4HxJ0pyoHvqYXCvboE2h33pEHyfcFDLzKGCtk+xuhrm9is8SHfsXH7jyGVGBd9tD64ZZnpFwu5AFo0FbY4bjEj3cj\\\\\\\/GnquKkXAlUdQuhqszr3KoxhIGXwvjxMw+Tl6BaqOTFnhQmo2BorAWcU8OI3LjycDk23YHIRpvGv0fTZMyMgsRgmOT\\\\\\\/SOru+G+2Gtslstlvl9tjKi+x3GI6LmFFF4Na7B5uK+SgdQAJr3OO6u5p3L1fh5RzfmhU++v\\\\\\\/9yWN0Cyo1+wLbkiNXDWcGhiXtXYO2ev+ruPq5vwrTSRgRaz26HQ2jh11WfiD1F0P9nraSrABnsokXSU066w61nA\\\\\\\/DLUBBm41AzH2l9QFxHe\\\\\\\/bHbh1KyeTXb\\\\\\\/kRE5GC\\\\\\\/yBJj6Zw360uYrdQ6RZUtcDWb0agMal9qCsCXyzyqQ1XmLggTpkWnksun7WYLvw6gKX5D4pGCGol0xf0yhwu+2bnyEIYP5myKWK0hcfQ9PlUKv1gdXV7pmzpe\\\\\\\/XswrhuO\\\\\\\/ssN1HpNtdKlKYvK+wqkVSbo\\\\\\\/YCluUu1nBnE7n6NWQX4XoWwfn0y1BHatbYVZwPqLMBK7UFVA17RI1kSqTejTGTi+5vOFahaerCvW26W8FyK0NTy2WTtdj5QePZPUbhqb6bql+7K9nSQ4j0fiYB0\\\\\\\/2K\\\\\\\/kkjIy2zMp\\\\\\\/fX5POVWeexp\\\\\\\/gnuSmGv5zNGjG0rNYpb0B0pgpJ7ohbSwZl2IYcOL6buBSgzBSTiHIyBQfiKqf9dusJqu4FfdP0FEfAc1pH7okM5MeCrvGYNmd8XyC\\\\\\\/Yo3uHg5k7DJNNMiU0O951TzW9sa2agdRdVTsWEGKjzp0FXiR5hnd8Hgp+mftYvSVLJneoUena2lRfT57DyMQE8UFxhmkPtuvx3QdaCtckmg0kWa4zR1\\\\\\\/W\\\\\\\/M2xbZvsRBxBRlRwE8a5lcd73fP9TR3h31rKM++IsWZsDEyzzqQDmvVsjEEt715fu2AvJjZK7DyO9LUEyfLahV18UptFWwUkwfwD+M4a959giaxneuDYU2xlJ0TS+6pWvqVHULGDKBQuh9Kuc9FuQurgD0RajtVCNn0x2CJoqqfCec\\\\\\\/S1O957EP64KRTuBKLYROsC3SIvIsRdEDhl39IsTcIj\\\\\\\/RKSgbJA6CyNqzfMZPEkG8MIZmS6BZ7keCWxVEOCtN1RyPJm6uQnv+6LaEhMx9jBLrmJtb+8Kovs2XJwMceQTlgh\\\\\\\/GBjQUsXE6eg37EyCLXL728S0SJ+dl9MF1nppHbOk4wJFsMhlijqHFnLrsuZSmyDswpCAKxj2UR0rbLuDysGqpIXltxqTBvIqAEYdstHbvKyYdehpwG2rr9nHDlPmqS\\\\\\\/GUpPc\\\\\\\/kAhoZpQvF8mZRf6XR3GZIJpNLFKFlRgUI0dmI3kOJOEXTf75RqVtXhl165G9oRvQuc1OC224W1pdYOwaHmIvBYKuTs\\\\\\\/dv8pP\\\\\\\/+VwvPh09sqHRPpf3vT6z8eeFe20nCyNOwWlxaMGns5Skr6jELFa8ldMH1dcRgNns0Cnym30s+D\\\\\\\/H2wfj0N8XO3xTeTT+JFiWIkCI3pykpfqZmsaF2X66reGkY2jmLgZkxhKhhH3yb\\\\\\\/o9jRKLccDDzpItLvJky1gbAgg1sbk5leOV+P3MBC6flWZyIbFyAF2QAgp89q2FLOEZ7LD1eiDhjuYLi4nvnzTMSP38ogqtQwdSOBM5g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022yai6rJjI8uOyKOeJ6uNPyg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cf23bd425e5b9a98fab28bb2f4d47b7793decff46e3a5ee9dd0f821ad8ad2384\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Qesz8dIjVWAqNrZP0xji\\\\\\\/+n4pbJdVoroUecGHiooul0YDzW9p2eCxMfnLikqkXXb8+3\\\\\\\/TJ48IDRa91gA6VsdrVNsEXlk9FiDe+MTmN+TKLs8bx9w+5tQR1X4VTwYell630kdAvzQVLekiGt3ZS1z7N6UfzF4+GdCLgROvDWgApcCCwFsK5\\\\\\\/bUqffF4bGR82T1Z0z9yyojbOChmu4g34GmlRoxbQLxLYXwjjZBDw2cBn4t8hFnJ9hhJIbBDBK2CWjyuPsbx5jRkkZ7omLmrSMTCJ3cDI+skvpXD8e9CU6WIe3Z0OUc5E\\\\\\\/034ykuZCIO03XLpY8axX\\\\\\\/N4FpTMoNpvZ1tD7hpz7bAngo42P3znQdkXEiHfv4yj0HGj93k6e5gr1L\\\\\\\/U+XY\\\\\\\/kST3ep0obz0hNpZcP\\\\\\\/pfBwIID+dx1cCQtKeGKy5dQA4MOyh2A1ayWGlcEU116q28vZgMejcmOUviGS2HEgDXdxkoG9SAY0fe\\\\\\\/JYgrvu3+aibv8luFJNXED5fp7BtoLD+EwDhXSoa8PZRzMnRQ113EGymqUI10Vafj2fZsXceEZycBQjW9F2WZhJ3wTH1W1wRrRt6jII+B9biUFIF\\\\\\\/yxHwR0hhQQuelEqe3NFMMRRncvX3\\\\\\\/j\\\\\\\/yjCa09f2ogaoqcQCFXZxTWFh5xQCLb5\\\\\\\/MLi3Cq93t15SY\\\\\\\/TFcuoJsBQQ+2gKnoo+FLrKA64eMbDU4cvhIlpaIhjuVhSkhi5EDCmIHYKsfaGGxS\\\\\\\/o+VNrERxmxoHPUx98ypCR4HBecBfyXD\\\\\\\/sl+uAnKjlrEgwu4XB4n9TY7Lpoadv94sBBoIm7SO1IBeJU8Vxfi4fCvv\\\\\\\/XMf+45Dxr80f6cbXjHBgLZ+D7cdP0Y6cZgL0Og8xPYAIyMvzPC2Gxz+V4eJjIgVdlz0oIK7Um4hMt6U9BTe4WcP0OJ3z7HSQjzs4OjbGGQvLTSQhgCnOabYxQRVcVuGkQpzfMMKA2f7JsetBfO8fY9tu9jGVvG92K29Crs3yPTW5K+tIuu7v7lIgmeZDjyePoy56CIhbseQOrZ8FngMkTVfeNe4\\\\\\\/mffIZcGbNmNY3bxfe6efP0w0r+JvfQ5vN4izcSpC\\\\\\\/3SYoB+Ht5l\\\\\\\/pZ+uFuyRjbBmTCtMSN\\\\\\\/w1Q1GqwutN4XPNqX99GztlDuxOuuTLKQbR3jxcaR228NN2407y16VM0bmL7Ruehsx2JXHni170e\\\\\\\/UuDqTMIcRwt3MixMepGhQWc3yJPZBdSwitFPCQvaPYyZIrosEkaLPoCgkISNoA17bMpy5j38JRaL\\\\\\\/QDF3HMY3lap\\\\\\\/ApMmXdmVaoT5aMs52gJEWM1vnnI17hZ8hSXztshyLM\\\\\\\/UtO6USSFFroUbFE8h94kSmdjenr5B4lWrbHDvmDYePthw5MCoyd6+OyS1W9qfJUYUbHrklkdc2uU79jk4zocsz2sBz9d\\\\\\\/Bc4Kl79aEJF9kbLf7HS0S7LjaAPDU5JFPnXH6geOKBkg7HeDDAHtRXkW0WuLmaLxztMTnWqf+zUaIBLql6D5eERvwrd8an0fec+jcPAoHQrGVomzKvSzCNU443oxqaMpwVfNlsHQ2niLUfC9+GW4bky566cSTTgRhTBsfCpuYoPcmMHxj9aDr3z7NBd\\\\\\\/Qss9lMOOaBJLMnVKJ1mCylXZisUmbSmS3iI+SNf8lqbbN8gGi8r+3OeDHNyHullq4XKUrjGqBWeeI7VRMTWZh4AJgnWyK3b\\\\\\\/2RKLMIrSR09bMiNcopknXdAbVYbdFfI62GsdQw2f5JCPzp20OxS9sbR8Si8CnX6LkPNatkm0mlhPS1KZguOyt6QWETb5ASzLhud5K26AThheP4jUxMiKJc4KktdLBhAKJvpboDTe02KN53ABkyjLXVLeiEvty6tRy+j3iHwDdacXJnViUbAobQ5bzpayvA1z+rBAJKKM2osLgARxHEceuYPBGC32DvJMxzbcnagGL\\\\\\\/KVd4FPpTFHFWfhWUv8U5s1qjRh4KdkRnDmDG9AzEA+ae+lK0+Nf8K4CoMarf9uOiA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZnRw4XI1KggTuiqGEL7j\\\\\\\/A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002285d4d55182c662ee56f1dfa39fa3f0a4890ec0424f5537232a68e039678b0a91\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459561 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/10/1861" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-uZnavvujw_zRqGx4p1vqKj6emF0G1TdqibFQsPCedxs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/10/1861" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YZmu72M6TjIc47uxMlLl8mX-eB6aCZ6uDHHcUE3vN_4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/10/1851" title="When Females Produce Sperm: Genetics of C. elegans Hermaphrodite Reproductive Choice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/10/1869" title="Schizosaccharomyces japonicus Yeast Poised to Become a Favorite Experimental Organism for Eukaryotic Research" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459561" id="node459561" data-pisa="ggg;3/10/1861" data-pisa-master="ggg;g3.113.007948" data-apath="/ggg/3/10/1861.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Practical Considerations Regarding the Use of Genotype and Pedigree Data to Model Relatedness in the Context of Genome-Wide Association Studies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Riyan Cheng</span>, <span class="highwire-citation-author" data-delta="1">Clarissa C. Parker</span>, <span class="highwire-citation-author" data-delta="2">Mark Abney</span> and <span class="highwire-citation-author" data-delta="3">Abraham A. Palmer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1861-1867;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007948 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459561" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Riyan Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Human Genetics, The University of Chicago, Illinois 60637</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Division of Plant Sciences, Research School of Biology, The Australian National University, Canberra, ACT 0200, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Riyan%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARiyan%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Clarissa C. Parker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Human Genetics, The University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clarissa%2BC.%2BParker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parker%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClarissa%2BC.%2BParker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mark Abney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Human Genetics, The University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BAbney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abney%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BAbney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Abraham A. Palmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Human Genetics, The University of Chicago, Illinois 60637</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Psychiatry and Behavioral Neuroscience, The University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abraham%2BA.%2BPalmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palmer%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbraham%2BA.%2BPalmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aap@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">aap@uchicago.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/10/1861" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459561" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459561/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/10/1861.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459561" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459561/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/10/1861.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459561" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459561/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/10/1861.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459561" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459561/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/10/1861.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459561" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459561/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genome-wide association studies of complex traits often are complicated by relatedness among individuals. Ignoring or inappropriately accounting for relatedness often results in inflated type I error rates. Either genotype or pedigree data can be used to estimate relatedness for use in mixed-models when undertaking quantitative trait locus mapping. We performed simulations to investigate methods for controlling type I error and optimizing power considering both full and partial pedigrees and, similarly, both sparse and dense marker coverage; we also examined real data sets. (1) When marker density was low, estimating relatedness by genotype data alone failed to control the type I error rate; (2) this was resolved by combining both genotype and pedigree data. (3) When sufficiently dense marker data were used to estimate relatedness, type I error was well controlled and power increased; however, (4) this was only true when the relatedness was estimated using genotype data that excluded genotypes on the chromosome currently being scanned for a quantitative trait locus.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/quantitative-trait-locus-qtl" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-locus-qtl" rel="nofollow">quantitative trait locus (QTL)</a></li><li class="kwd"><a href="/keyword/relatedness" class="hw-term hw-article-keyword hw-article-keyword-relatedness" rel="nofollow">relatedness</a></li><li class="kwd"><a href="/keyword/type-i-error-rate" class="hw-term hw-article-keyword hw-article-keyword-type-i-error-rate" rel="nofollow">type I error rate</a></li><li class="kwd"><a href="/keyword/statistical-power" class="hw-term hw-article-keyword hw-article-keyword-statistical-power" rel="nofollow">statistical power</a></li></ul><p id="p-4">In their simplest form, genome-wide association studies (GWAS) assume that all subjects are unrelated. However, human population isolates and various model organism populations contain individuals with varying levels of relatedness. For polygenic traits, this results in correlations among both genotypes and phenotypes and can produce inflated type I error rates when performing GWAS (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Newman <em>et al.</em> 2001</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cheng <em>et al.</em> 2010</a>). Mixed models are commonly used to account for relatedness using a random effect and may optionally model the effect of individual markers as a fixed effect (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Goldgar 1990</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Amos 1994</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Xu and Atchley 1995</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abney <em>et al.</em> 2000</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Yu <em>et al.</em> 2006</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Kang <em>et al.</em> 2008</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Cheng <em>et al.</em> 2010</a>). Relatedness can be estimated from a pedigree or from genotype data. The use of genotype (<em>e.g.</em>, <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Yu <em>et al.</em> 2006</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Kang <em>et al</em> 2008</a>) or pedigree (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Abney <em>et al.</em> 2002</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Cheng <em>et al.</em> 2010</a>) data for GWAS has been implemented previously. However, when both types of data are available, methods to control the type I error rate while maximizing power have not been systematically explored.</p><p id="p-5">Although siblings share an average of 50% of their genome identity-by-descent (IBD), the realized sharing is variable. Genotype data allow estimation of realized sharing (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ritland 1996</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Lynch and Ritland 1999</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Wang 2002</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Frentiu <em>et al.</em> 2008</a>), as opposed to the average level of sharing that is obtained from pedigree data. However, genotypes only provide information about identity-by-state (IBS), which is only an approximation to IBD. Furthermore, the accuracy of estimates of realized sharing depends on the density of genotype data. When both pedigree and genotype data are available, a very pragmatic question arises: how should these data be used to control false-positive rates while increasing power?</p><p id="p-6">In this study, we used simulations to address this question. We estimated relatedness by using genotype data, pedigree data, and the combination of both genotype and pedigree data under various models. We sought methods that could control the type I error rate and maximize power.</p><div class="section methods-results" id="sec-1"><h2 class="">Methods, Simulations, and Results</h2><div id="sec-2" class="subsection"><h3>Statistical models</h3><p id="p-7">Our methods are based on the linear mixed model for quantitative traits with a single major diallelic quantitative trait locus (QTL) modeled as a fixed effect and <em>P</em> polygenes modeled as random effects,<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where <strong>y</strong> is the vector of trait values; <strong><em>Î¼</em></strong> is the vector of trait means that may depend on known covariates; <strong>x</strong> is a vector of genotypes with values â1, 0, and 1 corresponding to genotypes AA, AB, and BB of a QTL; <em>a</em> is the additive effect of the QTL; <strong>z</strong> is a vector whose elements take on value 1 when the subject has QTL genotype AB and value 0 otherwise; <em>d</em> is the dominance effect of the QTL; <strong>u</strong><em><sub>i</sub></em> is the genetic effect at the <em>i</em>th polygenic locus; and <strong>Ïµ</strong> is the vector of residual effects. We assume the random effects to be distributed normally, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-2.gif"/></noscript></span></span>, where <strong>I</strong> is the identity matrix; <strong>u</strong><em><sub>i</sub></em> ~ <em>N</em>(0, <strong>Î©</strong><em><sub>i</sub></em>), with the polygenic effects <strong>u</strong><em><sub>i</sub></em> independent of each other and independent of the residual effect <strong>Ïµ</strong>. We model the polygenic covariances as <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-3.gif"/></noscript></span></span>, where the (<em>j</em>, <em>k</em>)th element of <strong>Î¦</strong><em><sub>i</sub></em> is the probability that at polygene <em>i</em> a randomly drawn allele from subject <em>j</em> and a randomly drawn allele from subject <em>k</em> are IBD, the (<em>j</em>, <em>k</em>)th element of <strong>Î</strong><em><sub>i</sub></em> is the probability that at polygene <em>i</em> the two alleles in subject <em>j</em> are both IBD with the two alleles in subject <em>k</em> and that neither subject is autozygous, and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-4.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-5.gif"/></noscript></span></span> are the additive and dominance polygenic variances at locus <em>i</em>. In general, when inbreeding is present, there are additional variance components present (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gillois 1964</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Harris 1964</a>). The additional variance components, however, typically are small (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Lynch and Ritland 1999</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Abney <em>et al.</em> 2000</a>) and we choose to ignore them here. The total covariance matrix for the polygenic effect, then, is</p><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-6.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span><p id="p-8">Our objective is, given the genotype data at a marker, to test whether the marker is a QTL. That is, we test the null hypothesis <em>H</em><sub>0</sub>: <em>a</em> = 0 and <em>d</em> = 0 <em>vs.</em> the alternative <em>H</em><sub>1</sub>: <em>a</em> â  0 or <em>d</em> â  0. We use the likelihood ratio test with the trait model <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-7.gif"/></noscript></span></span>, where we must use an estimator <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-8.gif"/></noscript></span></span> for the true covariance matrix in <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">equation (2)</a> because the true relationship matrices <strong>Î¦</strong><em><sub>i</sub></em> and <strong>Î</strong><em><sub>i</sub></em> and polygenic variances <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-9.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-10.gif"/></noscript></span></span> are unknown. Typically, <strong>Î¦</strong><em><sub>i</sub></em> and <strong>Î</strong><em><sub>i</sub></em> are estimated by their expected value given a pedigree, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-11.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-12.gif"/></noscript></span></span> for all loci <em>i</em>, where <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-13.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-14.gif"/></noscript></span></span> are the pedigree based estimates. However, it is also possible to estimate these quantities from the marker data, and when the marker data are informative enough, this may more accurately estimate the true sharing at the polygenic loci. We label the marker-based estimates <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-15.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-16.gif"/></noscript></span></span> and are described in <em>Relationship matrices</em> subsection. This leads us to consider three possible models for the polygenic covariance in the likelihood ratio test, VM1: <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-17.gif"/></noscript></span></span> where the relationship matrices are estimated using only pedigree information, VM2: <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-18.gif"/></noscript></span></span> where the relationship matrices are estimated using only observed genotype data, and VM3: <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-19.gif"/></noscript></span></span> where relationship matrices are estimated from both genotype and pedigree data are used. In all three variance models the variance parameters <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-20.gif"/></noscript></span></span> are estimated by maximum likelihood.</p></div><div id="sec-3" class="subsection"><h3>Relationship matrices</h3><p id="p-9">We obtained relationship matrices as described and implemented in the R package âQTLRelâ (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cheng <em>et al.</em> 2011</a>). The pedigree estimates are based on Kariglâs algorithms (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Karigl 1981</a>). To obtain the marker based estimates <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-21.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-22.gif"/></noscript></span></span>, we considered each genotyped locus for a pair of subjects and used an estimator based on IBS rather than IBD. For a diallelic marker <em>l</em> the (<em>j</em>, <em>k</em>)th element of <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-23.gif"/></noscript></span></span> takes on value 1.0 when subjects <em>j</em> and <em>k</em> are both homozygous for the same allele, 0.5 when one is homozygous and the other heterozygous or both are heterozygous, or 0 when both are homozygous for different alleles. We define the (<em>j</em>, <em>k</em>)th element of <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-24.gif"/></noscript></span></span> as 1.0 when both <em>j</em> and <em>k</em> are heterozygous and 0 otherwise. Our marker based estimates are the mean across <em>L</em> markers used in the estimator, <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-25.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-26.gif"/></noscript></span></span>. In <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> we consider different sets of the <em>L</em> loci in our estimators. Note that under the assumption that all the additive polygenic variances <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-27.gif"/></noscript></span></span> are equal and all the dominance polygenic variances <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-28.gif"/></noscript></span></span> are equal, the true polygenic covariance matrix given in <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">Equation (2)</a> would closely resemble our estimated covariance matrix <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-29.gif"/></noscript></span></span> given in VM2, with the summation over polygenes replaced by a summation over markers. Although we do not require this assumption to use our estimators for the relationship matrices, it does suggest that a more efficient estimator might be chosen by appropriately weighting each term in the summations for <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-30.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-31.gif"/></noscript></span></span>, with the optimal weights depending upon both how IBS at a marker captures IBD at a polygene and the relative magnitude of the variance at that polygene. We do not explore this issue here.</p></div><div id="sec-4" class="subsection"><h3>Mapping populations</h3><p id="p-10">We considered two mapping populations: an advanced intercross line (AIL) F<sub>26</sub> and a structured population (STR). For the AIL, we assumed that one male and one female offspring from each of 144 F<em><sub>n</sub></em> (2 â¤ <em>n</em> â¤ 25) breeding pairs was randomly mated with a nonsibling to produce the next generation. The final sample used for mapping consisted of four offspring from each of 144 F<sub>25</sub> breeding pairs for a sample size of 576. The STR consisted of subsamples from three subpopulations. The first subsample was from an AIL F<sub>26</sub>,where one male and one female progeny from each of 48 F<em><sub>n</sub></em> (2 â¤ <em>n</em> â¤ 25) breeding pairs was randomly mated with nonsiblings to produce the next generation and four offspring of each F<sub>25</sub> breeding pair contributed to the subsample. The other two subsamples were generated as follows. A male and a female progeny from each of 96 F<em><sub>n</sub></em> (2 â¤ <em>n</em> â¤ 12) breeding pairs were randomly mated with nonsiblings to produce the next generation. The F<sub>13</sub> breeding pairs were randomly split into two subpopulations of equal size and bred until F<sub>26</sub> with the same breeding scheme as above within each subpopulation. The STR sample size was also 576. These pedigrees were created once and were used in replicate simulations.</p></div><div id="sec-5" class="subsection"><h3>Sparse marker simulations</h3><p id="p-11">We simulated 15 chromosomes that were 400 cM each; each chromosome had 101 evenly spaced markers (4 cM spacing). A total of 500 polygenic QTL were evenly spaced on the first five chromosomes. We simulated two possible relationships between the markers and the polygenic QTL (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>): (Completely linked), that is, all of the polygenic QTL were exactly at marker loci, meaning that polygenic QTL were completely linked to markers or (Incompletely linked), that is, each polygenic QTL was midway between two adjacent markers, meaning that polygenic QTL were incompletely linked to markers. On chromosomes 1â5, the additive and dominance effects of a polygenic QTL were generated randomly from uniform distributions <em>U</em> (â0.15, 0.15) and <em>U</em> (â0.08, 0.08), respectively, in each replicate simulation. The residual effect was simulated from a normal distribution <em>N</em> (0, 1). The polygenic QTL approximately accounted for 84% of the total variation. Heritabilities in this range are not uncommonly observed in model organisms and humans (<em>e.g.</em>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Yang <em>et al.</em> 2013</a>). We expect our results will apply across a broad range of heritabilities. There were no QTL on chromosomes 6â10. We scanned chromosomes 11â15 for putative QTL. When we evaluated type 1 error rates, there were no QTL on chromosomes 11â15. When evaluating statistical power, there was one QTL at the position of the marker in the middle of the 11th chromosome with an additive effect 0.5 and a dominance effect 0.2, which accounted for approximately 2.5% of the total variation.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463363/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463363/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Names of marker sets used to estimate genotype-based relationship matrices</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Variance model estimators</h3><p id="p-14">For each variance model, VM1, VM2, and VM3, we considered different estimators that varied in their level of informativeness. For VM1, we obtained estimates of the relationship matrices as follows: (1) using no pedigree (Naive), equivalent to assuming all subjects are independent; (2) using only the final three generations (<em>i.e.</em>, individuals, parents and grandparents) of the pedigree (Last3); (3) using only the final six generations of the pedigree (Last6); and (4) using the entire pedigree (AllPed).</p><p id="p-15">VM2 consists of estimates based on different subsets of genotype data. Our intent was to investigate scenarios in which the markers were more or less informative regarding the polygenes. An ideal case is when we consider only those markers that are completely linked to the polygenes (left column of <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). A less-ideal case is when we only consider those markers that are incompletely linked to the polygenes (right column of <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). The first row of <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a> considers chromosomes 1â5, which contain all of the polygenes. The cells in this column are labeled complete linkage (CL) and incomplete linkage (IL). The second row of <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a> considers chromosomes 6â10, which do not contain any polygenes, in this case both columns are equivalent and labeled unlinked (UL). The third row of <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a> considers chromosomes 1â10, thus representing the combination of the prior two rows. These are a combination of completely linked and unlinked (CUL), and incompletely linked and unlinked (IUL). The final row includes CUL plus one of chromosomes 11â15, such that the additional chromosome included in the estimate of relatedness is the one being scanned for the QTL.</p><p id="p-16">For the third variance model, VM3, we combined estimators from both VM1 and VM2. Specifically, we used the IUL set of markers to estimate <strong>Î¦</strong><em><sub>m</sub></em> and <strong>Î</strong><em><sub>m</sub></em> and either AllPed or Last3 to estimate <strong>Î¦</strong><em><sub>p</sub></em> and <strong>Î</strong><em><sub>p</sub></em>.</p><p id="p-17">We evaluated the performance, in terms of type I error rates and power, of the different methods for estimating the relationship matrices. Although chromosomes 1â10 were sometimes used to estimate the polygenic variation, only chromosomes 11â15 were scanned for the presence of a QTL. In simulations in which there was not a QTL on chromosome 11, any significant association was considered a false positive. When a QTL was present on chromosome 11, any significant association on this chromosome was considered a true positive. In both instances significant associations were defined as those exceeding a 0.05 significance level based on 5,000 permutations (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cheng and Palmer 2012</a>). We obtained a similar result from 5000 parametric bootstrap simulations. We performed 2500 replicates to evaluate type I error rates and power. The maximum likelihood ratio at each marker was used as a test statistic, as implemented in QTLRel (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Cheng <em>et al.</em> 2011</a>).</p></div><div id="sec-7" class="subsection"><h3>Dense markers simulations</h3><p id="p-18">After completing the prior set of simulations, we were concerned that certain VM2 conditions failed to adequately control type I error rates. We hypothesized that this was attributable to the sparse nature of the markers, so we conducted simulations in which we varied the density of markers for model VM2. In this set of simulations we only considered the STR and simulated 15 chromosomes of length 200 cM with 500 polygenic QTL that were placed randomly across the first 10 chromosomes. The additive and dominance effects of the polygenic QTL were randomly generated using the same distributions as described previously, whereas the residual error was simulated from a <em>N</em> (0, 0.8<sup>2</sup>) distribution. In the simulations designed to evaluate power, we placed a QTL with an additive effect of 0.5 and a dominance effect of 0.2 at position <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/10/1861/embed/mml-math-32.gif"/></noscript></span></span> cM of the 11th chromosome; otherwise, there were no QTL on the last five chromosomes. Markers were spaced evenly on the first 10 chromosomes with intermarker distances of 4, 2, 1, 0.5, 0.25, 0.125, or 0.0625 cM. For chromosomes 11â15 we considered two cases (a) markers were placed with the same density as on the first 10 chromosomes, or (b) markers were evenly spaced every 2 cM. As the marker density increased in case (a) the distance between the QTL and its closest marker on chromosome 11 decreased; however, unlike in the sparse marker case, the QTL was never at a marker. In these sets of simulations, for the VM1 estimator we used the entire pedigree; for the VM2 estimator we estimated relatedness using the markers on the first 10 chromosomes. For the VM3 estimator we combined the VM1 and VM2 estimators, where we used the last three generations of the pedigree for VM1. Again, chromosomes 11â15 were scanned for a QTL and type I error rates and statistical power were estimated from 1000 replicates.</p></div><div id="sec-8" class="subsection"><h3>Real dataset</h3><p id="p-19">Finally, we used a published dataset from the 8th generation of a mouse AIL, which were bred from two inbred strains. This dataset consisted of 552 mice genotyped at 895 single-nucleotide polymorphisms (SNPs) and phenotyped for a quantitative trait, as described in <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Parker <em>et al.</em> (2011)</a>. A full pedigree back to the inbred founders was available. In our analyses we included both additive and dominance variance components in the model when estimating relatedness from the marker data or from the pedigree. All simulation code (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007948/-/DC1">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007948/-/DC1/FileS1.gz">File S1</a>) and the analyzed data set (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007948/-/DC1/FileS2.gz">File S2</a>) are available at <a href="http://palmerlab.org/data/">http://palmerlab.org/data/</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Sparse markers</h3><p id="p-20">Results of the type I error simulations are shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. It is clear that ignoring the relatedness of the subjects led to a highly inflated false-positive rate (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>; Naive). In the AIL population, only the final three generations were needed to obtain sufficiently accurate estimates of the relationship matrices to control type I error (Last3). With the more STR, the last 12 generations were required to control the type I error rate. Although our simulations indicate that more than one generation typically will be required to control type I error rates, a full pedigree is not always needed. In general, it is prudent to use all available pedigree information, because the number of generations needed to control the type I error rate may not be known and using too many generations had no negative impact on power.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463323/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463323/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Marker set power and error rates</span><div class="sb-div caption-clear"></div></div></div><p id="p-25">In an ideal situation the markers used to estimate the relationship matrices would exactly tag the polygenic loci (CL). As shown in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, under this condition we obtained the correct type I error rate and had the greatest power. A less optimal situation was that the markers were only in partial linkage disequilibrium (LD) with the polygenes (IL), or even worse, the markers were completely unlinked to the polygenes (UL); in both of these cases, type I error rates were inflated. When additional uninformative markers (markers on chromosomes 6â10) were added to the CL and IL cases (CUL and IUL), the type I error rate was unaffected, however power in the CUL case was lower than the power in the CL case. Finally, unlike the previous cases in which information about relatedness was drawn from markers on chromosomes 1â10 but the QTL scan was performed on chromosomes 11â15, we considered the case (CUL + 1) where markers on chromosomes 11â15 were used both to estimate the relatedness and for the QTL scan. To make the results directly comparable to CUL, only markers on the chromosome being scanned were added to those on chromosomes 1â10 (<em>e.g.</em>, when scanning chromosome 11 markers on chromosomes 1â11 were used to estimate relatedness). In this case, we found that the type I error rate was too conservative resulting in dramatically decreased power. We attribute this phenomenon to the effect of the QTL being partially captured by markers that are included in the polygenic term. Thus, the effect of the QTL is divided between the fixed and random term in the linear mixed model. This phenomenon has recently been referred to as âproximal contaminationâ by <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Listgarten <em>et al.</em> (2012)</a>. This finding suggests that markers linked to the locus being scanned should not be included in the estimate of relatedness.</p><p id="p-26">Finally, we considered using both pedigree and marker information to model relatedness (VM3). We found that although the use of markers or using partial pedigrees was unable to control type I error, the combination of the two effectively controlled the type I error rate. This approach may result in increased power relative to use of the pedigree alone, but this difference, although suggestive, was not statistically significant in our simulations.</p></div><div id="sec-11" class="subsection"><h3>Dense markers</h3><p id="p-27">As shown in <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>, when markers were incompletely linked to the polygenic QTL, the type I error rate was not adequately controlled. This incomplete linkage was a consequence of inadequate marker density; therefore, we explored the effect of increasing the marker density. As shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> when only SNPs were used to estimate relatedness (<em>i.e.</em>, VM2) and when the marker density was inadequate, the type I error rate was inflated. Using both marker and partial or full pedigree data (<em>i.e.</em>, VM3) prevented inflation of the type I error rate, without sacrificing much power. This approach provides better power than using the pedigree alone (<em>i.e.</em>, VM1).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/10/1861/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated type I error rate and statistical power at significance level 0.05 for varying densities of markers. Marker densities on chromosomes 1&#x2013;10 were varied, and markers on chromosomes 11&#x2013;15 were either varied (left) or held fixed at 2 cM spacing (right). VM1: relationship matrices estimated using the entire pedigree (AllPed); VM2: relationship matrices estimated using genotypes on chromosomes 1&#x2013;10 (IL); VM3: relationship matrices estimated using both genotypes on chromosomes 1&#x2013;10 (IL) and the last three generations of the pedigree (Last3). Red symbols indicate conditions with inflated type I error rate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1209828344" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimated type I error rate and statistical power at significance level 0.05 for varying densities of markers. Marker densities on chromosomes 1&#x2013;10 were varied, and markers on chromosomes 11&#x2013;15 were either varied (left) or held fixed at 2 cM spacing (right). VM1: relationship matrices estimated using the entire pedigree (AllPed); VM2: relationship matrices estimated using genotypes on chromosomes 1&#x2013;10 (IL); VM3: relationship matrices estimated using both genotypes on chromosomes 1&#x2013;10 (IL) and the last three generations of the pedigree (Last3). Red symbols indicate conditions with inflated type I error rate.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/10/1861/F1.medium.gif" width="440" height="430"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/10/1861/F1.medium.gif" width="440" height="430"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/10/1861/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/10/1861/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463354" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-28" class="first-child">Estimated type I error rate and statistical power at significance level 0.05 for varying densities of markers. Marker densities on chromosomes 1â10 were varied, and markers on chromosomes 11â15 were either varied (left) or held fixed at 2 cM spacing (right). VM1: relationship matrices estimated using the entire pedigree (AllPed); VM2: relationship matrices estimated using genotypes on chromosomes 1â10 (IL); VM3: relationship matrices estimated using both genotypes on chromosomes 1â10 (IL) and the last three generations of the pedigree (Last3). Red symbols indicate conditions with inflated type I error rate.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Considering <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, the ability of VM1 to control false-positive results was determined by the amount of pedigree information. VM2 depended on how accurately the markers captured information about the polygenes. VM3 provides a robust alternative when neither sufficient pedigree nor marker data are available. In general, VM1 was the least powerful, VM2 was the most powerful if markers linked to scanning loci were excluded in the estimation of relationship matrices, and VM3 was a compromise.</p><p id="p-30">Note that our reported powers will include positive associations even when the âsignificantâ locus is far from the true QTL. The consequence is that the power values are greater than they would be under an approach that requires the association to be close to the true QTL. However, the relative powers of the methods will not be affected by the differences between these two approaches.</p></div><div id="sec-12" class="subsection"><h3>Real data set</h3><p id="p-31">Finally, we applied these methods to a real data set (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Parker <em>et al.</em> 2011</a>). We estimated relatedness by using the full pedigree (AllPed, <em>i.e.</em>, VM1) or all markers on the genome (AllSNP, <em>i.e.</em>, VM2). In addition, because including SNPs on the chromosome being scanned in the relatedness estimation is overly conservative (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>, CUL + 1), we estimated relatedness by using all markers except the chromosome being scanned for a QTL (AllSNP-1). In <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> we compare a version of VM3 that combines both AllSNP-1 and pedigree information (AllSNP-1 + AllPed) with either AllSNP (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>) or AllPed (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). The estimated heritability of this trait was 74.8% using VM3.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/10/1861/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mapping results of the B6xD2 F8 methamphetamine sensitivity data (Parker et al. 2011). The green dashed line indicates the threshold for genome-wide significance at the 0.05 level. Red dots are the mapping results that estimate relatedness by combining AllSNP-1 and pedigree information. Black dots are the mapping results that estimate relatedness using (A) AllSNP or (B) AllPed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1209828344" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mapping results of the B6xD2 F&lt;sub&gt;8&lt;/sub&gt; methamphetamine sensitivity data (Parker &lt;em&gt;et al.&lt;/em&gt; 2011). The green dashed line indicates the threshold for genome-wide significance at the 0.05 level. Red dots are the mapping results that estimate relatedness by combining AllSNP-1 and pedigree information. Black dots are the mapping results that estimate relatedness using (A) AllSNP or (B) AllPed.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/10/1861/F2.medium.gif" width="417" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/10/1861/F2.medium.gif" width="417" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/10/1861/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/10/1861/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463448" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Mapping results of the B6xD2 F<sub>8</sub> methamphetamine sensitivity data (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Parker <em>et al.</em> 2011</a>). The green dashed line indicates the threshold for genome-wide significance at the 0.05 level. Red dots are the mapping results that estimate relatedness by combining AllSNP-1 and pedigree information. Black dots are the mapping results that estimate relatedness using (A) AllSNP or (B) AllPed.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Note that we did not compare AllSNP-1 to AllSNP. AllSNP-1 is comparable with IUL (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>) because IUL used SNP information from chromosomes 1â10 but scanned for QTL on chromosome 11. As shown in <a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> the ability of IUL to control the type I error rate depends on marker density. In this real data example, it was not clear whether our markers were sufficiently dense. Therefore, any apparent advantage in power of AllSNP-1 compared with AllSNP might be a result of a failure of AllSNP-1 to control the type I error rate. In situations in which sufficiently dense markers are available, AllSNP-1 should control the type I error rate, as shown in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>. The benefit in power of using AllSNP-1 + AllPed is demonstrated in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, where this method detected five genome-wide significant results, whereas AllPed detected one and AllSNP zero genome-wide significant loci.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-34">GWAS is a powerful tool for dissecting the genetic basis of quantitative traits. However, accurate inference depends on a valid test (<em>i.e.</em>, correct type I error rates), a requirement that may not be met if either familial relatedness or population structure is not properly modeled. When working with model organisms, GWAS is often performed with the use of populations in which individuals are closely related to one another, necessitating a method to estimate the relatedness. This can be done using a pedigree, if available, but could potentially also be performed using observed genotype data. We found that estimates of relatedness that use sufficiently long pedigrees can control the type I error rate. Furthermore, marker-based estimates can also control the type I error rate if the markers are sufficiently dense to accurately estimate the realized relatedness at the polygenes. Perhaps more importantly, we find that an estimator that uses both pedigree information and genotype data gave consistently accurate type I error rates across differing levels of pedigree and genotype informativeness, even when using either pedigree or genotype data that alone would not result in a valid test.</p><p id="p-35">We also investigated how different approaches to estimating relatedness using marker data affect the power of a GWAS. We found power was increased by excluding markers that are in LD with the marker being tested. This finding is underscored by our analysis of the AIL mouse data set, where five loci reach genome-wide significance when this approach is used, whereas only one locus meets genome-wide significance when all markers are used to estimate the matrices. Note that both our real dataset and our simulations had relatively high heritabilities; however, we expect that our conclusions can be extended to traits with lower heritabilities.</p><p id="p-36">We propose that all markers on the chromosome being scanned be excluded from the relationship estimation. Further power improvements may be possible by excluding only those markers that are in LD with the locus being tested rather than all markers on that chromosome, though this would entail a more complicated implementation. Current methods for efficiently using mixed models in GWAS (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Kang <em>et al.</em> 2008</a>; <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Cheng <em>et al.</em> 2010</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Meyer and Tier 2012</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Zhou and Stephens 2012</a>) would need modification and may lose computational efficiency. Excluding all markers on the chromosome allowed a reasonable compromise between computational speed and power. We do note that the gains in power obtained by excluding markers in LD with the tested locus is likely most important when working in populations where LD extends over a significant fraction of the chromosome, though we do not directly assess this here. Recently, the loss of power due to inclusion of markers in LD with the tested locus has recently been referred to as âproximal contaminationâ by <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Listgarten <em>et al.</em> (2012)</a>.</p><p id="p-37">Ideally, we would expect to obtain optimal power by not just excluding markers in LD with the locus being tested but by only using genotypes most informative of IBD sharing at the polygenic loci. Here we used IBS sharing as a proxy for IBD sharing, an approximation that is exact in the AIL and STR populations used here. In populations where IBS is less indicative of IBD (<em>e.g.</em>, natural populations) recent advances allow for highly accurate estimates of IBD sharing given sufficiently dense marker data (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Han and Abney 2011</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2013</a>). We expect using IBD estimates obtained from such methods, rather than solely using pedigree based estimates of relatedness, will provide gains similar to what we obtained here. Our results, then, should provide practical guidance to researchers seeking to model polygenic variation in support of GWAS and related study designs.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-38">We wish to thank Xiang Zhou for his helpful comments. This work was supported by National Institutes of Health grants DA021336, GM097737, MH079103 (to A.A.P.), and HG002899 (to M.A.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007948/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007948/-/DC1</a>.</p></li><li class="fn" id="fn-8"><p id="p-39"><em>Communicating editor: D. W. Threadgill</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-05-14"><span class="received-label">Received </span>May 14, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-08-22"><span class="accepted-label">Accepted </span>August 22, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2013 Cheng <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abney</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McPeek</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Estimation of variance components of quantitative traits in inbred populations.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">629</span>â<span class="cit-lpage">650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D141%26rft.spage%253D629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.2" data-doi="10.1086/339705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abney</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McPeek</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Quantitative-trait homozygosity and association mapping and empirical genome-wide significance in large, complex pedigrees: fasting serum-insulin level in the hutterites.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">920</span>â<span class="cit-lpage">934</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DAbney%26rft.auinit1%253DM.%26rft.volume%253D70%26rft.issue%253D4%26rft.spage%253D920%26rft.epage%253D934%26rft.atitle%253DQuantitative-trait%2Bhomozygosity%2Band%2Bassociation%2Bmapping%2Band%2Bempirical%2Bgenomewide%2Bsignificance%2Bin%2Blarge%252C%2Bcomplex%2Bpedigrees%253A%2Bfasting%2Bserum-insulin%2Blevel%2Bin%2Bthe%2BHutterites.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F339705%26rft_id%253Dinfo%253Apmid%252F11880950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/339705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11880950&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174252100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">Robust variance-components approach for assessing genetic linkage in pedigrees.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">535</span>â<span class="cit-lpage">543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DAmos%26rft.auinit1%253DC.%2BI.%26rft.volume%253D54%26rft.issue%253D3%26rft.spage%253D535%26rft.epage%253D543%26rft.atitle%253DRobust%2Bvariance-components%2Bapproach%2Bfor%2Bassessing%2Bgenetic%2Blinkage%2Bin%2Bpedigrees.%26rft_id%253Dinfo%253Apmid%252F8116623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8116623&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994MZ15800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">A simulation study of permutation, bootstrap and gene dropping for assessing statistical significance in the case of unequal relatedness.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>:<span class="cit-fpage">1015</span>â<span class="cit-lpage">1018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D193%26rft.spage%253D1015%26rft_id%253Dinfo%253Apmid%252F23267053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23267053&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.5" data-doi="10.1534/genetics.110.116863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samocha</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sokoloff</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abney</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Genome-wide association studies and the problem of relatedness among advanced intercross lines and other highly recombinant populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1033</span>â<span class="cit-lpage">1044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116863%26rft_id%253Dinfo%253Apmid%252F20439773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS8zLzEwMzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE4NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.6" data-doi="10.1186/1471-2156-12-66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abney</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skol</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">QTLRel: an R package for genome-wide association studies in which relatedness is a concern package for genome-wide association studies in which relatedness is a concern.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DCheng%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.spage%253D66%26rft.epage%253D66%26rft.atitle%253DQTLRel%253A%2Ban%2BR%2Bpackage%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bin%2Bwhich%2Brelatedness%2Bis%2Ba%2Bconcern.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-12-66%26rft_id%253Dinfo%253Apmid%252F21794153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-12-66&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21794153&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.7" data-doi="10.1098/rspb.2007.1032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frentiu</span>  <span class="cit-name-given-names">F. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clegg</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chittock</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blows</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Pedigree-free animal models: the relatedness matrix reloaded.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">639</span>â<span class="cit-lpage">647</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DFrentiu%26rft.auinit1%253DF.%2BD%26rft.volume%253D275%26rft.issue%253D1635%26rft.spage%253D639%26rft.epage%253D647%26rft.atitle%253DPedigree-free%2Banimal%2Bmodels%253A%2Bthe%2Brelatedness%2Bmatrix%2Breloaded%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2007.1032%26rft_id%253Dinfo%253Apmid%252F18211872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMjoiMjc1LzE2MzUvNjM5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMC8xODYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gillois</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span>â<span class="cit-article-title">La relation dâidentitÃ© en gÃ©nÃ©tique.</span> <abbr class="cit-jnl-abbrev">Ann. Inst. Henri PoincarÃ© B.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BInst.%2BHenri%2BPoincar%25E9%2BB.%26rft.volume%253D2%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldgar</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span>â<span class="cit-article-title">Multipoint analysis of human quantitative genetic variation.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">957</span>â<span class="cit-lpage">967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DGoldgar%26rft.auinit1%253DD.%2BE.%26rft.volume%253D47%26rft.issue%253D6%26rft.spage%253D957%26rft.epage%253D967%26rft.atitle%253DMultipoint%2Banalysis%2Bof%2Bhuman%2Bquantitative%2Bgenetic%2Bvariation.%26rft_id%253Dinfo%253Apmid%252F2239972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2239972&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990EK40000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abney</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Identity by descent estimation with dense genome-wide genotype data.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">557</span>â<span class="cit-lpage">567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DHan%26rft.auinit1%253DL.%26rft.volume%253D35%26rft.issue%253D6%26rft.spage%253D557%26rft.epage%253D567%26rft.atitle%253DIdentity%2Bby%2Bdescent%2Bestimation%2Bwith%2Bdense%2Bgenome-wide%2Bgenotype%2Bdata.%26rft_id%253Dinfo%253Apmid%252F21769932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21769932&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.11" data-doi="10.1038/ejhg.2012.148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abney</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Using identity by descent estimation with dense genotype data to detect positive selection.</span> <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">205</span>â<span class="cit-lpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D21%26rft.spage%253D205%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fejhg.2012.148%26rft_id%253Dinfo%253Apmid%252F22781100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ejhg.2012.148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22781100&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span>â<span class="cit-article-title">Genotypic covariances between inbred relatives.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1319</span>â<span class="cit-lpage">1348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHarris%26rft.auinit1%253DD.%2BL.%26rft.volume%253D50%26rft.issue%253D6%26rft.spage%253D1319%26rft.epage%253D1348%26rft.atitle%253DGENOTYPIC%2BCOVARIANCES%2BBETWEEN%2BINBRED%2BRELATIVES%26rft_id%253Dinfo%253Apmid%252F14239792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo5OiI1MC82LzEzMTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE4NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.13" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Efficient control of population structure in model organism association mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>â<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE4NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.14" data-doi="10.1111/j.1469-1809.1981.tb00341.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karigl</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span>â<span class="cit-article-title">A recursive algorithm for the calculation of identity coefficients.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">299</span>â<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DKarigl%26rft.auinit1%253DG.%26rft.volume%253D45%26rft.issue%253DPt%2B3%26rft.spage%253D299%26rft.epage%253D305%26rft.atitle%253DA%2Brecursive%2Balgorithm%2Bfor%2Bthe%2Bcalculation%2Bof%2Bidentity%2Bcoefficients.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1981.tb00341.x%26rft_id%253Dinfo%253Apmid%252F7305283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1981.tb00341.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7305283&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981LY98800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.15" data-doi="10.1038/nmeth.2037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Listgarten</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lippert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadie</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">R. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Improved linear mixed models for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">525</span>â<span class="cit-lpage">526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DListgarten%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.issue%253D6%26rft.spage%253D525%26rft.epage%253D526%26rft.atitle%253DImproved%2Blinear%2Bmixed%2Bmodels%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2037%26rft_id%253Dinfo%253Apmid%252F22669648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22669648&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304778500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ritland</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Estimation of pairwise relatedness with molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">1753</span>â<span class="cit-lpage">1766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D152%26rft.issue%253D4%26rft.spage%253D1753%26rft.epage%253D1766%26rft.atitle%253DEstimation%2Bof%2BPairwise%2BRelatedness%2BWith%2BMolecular%2BMarkers%26rft_id%253Dinfo%253Apmid%252F10430599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Mi80LzE3NTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE4NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.17" data-doi="10.1534/genetics.111.134841"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tier</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">SNP Snappy: a strategy for fast genome-wide association studies fitting a full mixed model.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">275</span>â<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.134841%26rft_id%253Dinfo%253Apmid%252F22021386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzEvMjc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMC8xODYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.18" data-doi="10.1086/323659"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abney</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McPeek</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">The importance of genealogy in determining genetic associations with complex traits.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">1146</span>â<span class="cit-lpage">1148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DNewman%26rft.auinit1%253DD.%2BL.%26rft.volume%253D69%26rft.issue%253D5%26rft.spage%253D1146%26rft.epage%253D1148%26rft.atitle%253DThe%2Bimportance%2Bof%2Bgenealogy%2Bin%2Bdetermining%2Bgenetic%2Bassociations%2Bwith%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F323659%26rft_id%253Dinfo%253Apmid%252F11590549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/323659&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11590549&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171413400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sokoloff</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Genome-wide association for methamphetamine sensitivity in an advanced intercross mouse line.</span> <abbr class="cit-jnl-abbrev">Genes Brain Behav.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">52</span>â<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BBrain%2BBehav.%26rft.volume%253D11%26rft.spage%253D52%26rft_id%253Dinfo%253Apmid%252F22032291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22032291&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.20" data-doi="10.2307/2410647"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritland</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">A marker-based method for inferences about quantitative inheritance in natural populations.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1062</span>â<span class="cit-lpage">1073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D50%26rft.spage%253D1062%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2410647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2410647&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996UY55500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">An estimator for pairwise relatedness using molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">1203</span>â<span class="cit-lpage">1215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D160%26rft.issue%253D3%26rft.spage%253D1203%26rft.epage%253D1215%26rft.atitle%253DAn%2BEstimator%2Bfor%2BPairwise%2BRelatedness%2BUsing%2BMolecular%2BMarkers%26rft_id%253Dinfo%253Apmid%252F11901134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MC8zLzEyMDMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE4NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atchley</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-article-title">A random model approach to interval mapping of quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">1189</span>â<span class="cit-lpage">1197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D141%26rft.issue%253D3%26rft.spage%253D1189%26rft.epage%253D1197%26rft.atitle%253DA%2Brandom%2Bmodel%2Bapproach%2Bto%2Binterval%2Bmapping%2Bof%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Apmid%252F8582623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0MS8zLzExODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEwLzE4NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.23" data-doi="10.1371/journal.pgen.1003355"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">B.-G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Ubiquitous polygenicity of human complex traits: genome-wide analysis of 49 traits in Koreans.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003355%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003355%26rft_id%253Dinfo%253Apmid%252F23505390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003355&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23505390&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.24" data-doi="10.1038/ng1702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bi</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">203</span>â<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.10.1861.25" data-doi="10.1038/ng.2310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genome-wide efficient mixed-model analysis for association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">821</span>â<span class="cit-lpage">824</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhou%26rft.auinit1%253DX.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D821%26rft.epage%253D824%26rft.atitle%253DGenome-wide%2Befficient%2Bmixed-model%2Banalysis%2Bfor%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2310%26rft_id%253Dinfo%253Apmid%252F22706312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22706312&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F10%2F1861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/10/1861.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/10/1851" title="When Females Produce Sperm: Genetics of C. elegans Hermaphrodite Reproductive Choice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/10/1869" title="Schizosaccharomyces japonicus Yeast Poised to Become a Favorite Experimental Organism for Eukaryotic Research" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/10"><span>Volume 3 Issue 10, October 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/10.cover-source.jpg?itok=h5LtrAoT" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/10/1861" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-p7Ie3L-teKLHdWVcTSK436wRcOoCcsd1Hw0GqjlU7pU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/10/1861" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/10/1861" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/10/1861" class="active" data-icon-position="" data-hide-link-title="0">Practical Considerations Regarding the Use of Genotype and Pedigree Data to Model Relatedness in the Context of Genome-Wide Association Studies</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459561" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-OVsNRgEM0X14ye00f-paB-0QCmSacT42W4jP_RrZktE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/10/1861.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/10/1861" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-9W3u6dpRhODev0_t5gG35G0qz9p357D8LyVUR1Rc95Q" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/10/1861" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/10/1861.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459561" id="node459561--2" data-pisa="ggg;3/10/1861" data-pisa-master="ggg;g3.113.007948" data-apath="/ggg/3/10/1861.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Practical Considerations Regarding the Use of Genotype and Pedigree Data to Model Relatedness in the Context of Genome-Wide Association Studies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Riyan Cheng</span>, <span class="highwire-citation-author" data-delta="1">Clarissa C. Parker</span>, <span class="highwire-citation-author" data-delta="2">Mark Abney</span> and <span class="highwire-citation-author" data-delta="3">Abraham A. Palmer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1861-1867;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007948 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459561--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Riyan Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Human Genetics, The University of Chicago, Illinois 60637</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Division of Plant Sciences, Research School of Biology, The Australian National University, Canberra, ACT 0200, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Riyan%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARiyan%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Clarissa C. Parker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Human Genetics, The University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clarissa%2BC.%2BParker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parker%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClarissa%2BC.%2BParker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mark Abney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Human Genetics, The University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BAbney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abney%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BAbney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Abraham A. Palmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Human Genetics, The University of Chicago, Illinois 60637</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Psychiatry and Behavioral Neuroscience, The University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abraham%2BA.%2BPalmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palmer%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbraham%2BA.%2BPalmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aap@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">aap@uchicago.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/10/1861" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/10/1861&amp;title=Practical%20Considerations%20Regarding%20the%20Use%20of%20Genotype%20and%20Pedigree%20Data%20to%20Model%20Relatedness%20in%20the%20Context%20of%20Genome-Wide%20Association%20Studies" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/10/1861&amp;title=Practical%20Considerations%20Regarding%20the%20Use%20of%20Genotype%20and%20Pedigree%20Data%20to%20Model%20Relatedness%20in%20the%20Context%20of%20Genome-Wide%20Association%20Studies" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/10/1861&amp;title=Practical%20Considerations%20Regarding%20the%20Use%20of%20Genotype%20and%20Pedigree%20Data%20to%20Model%20Relatedness%20in%20the%20Context%20of%20Genome-Wide%20Association%20Studies" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/10/1861&amp;text=Practical%20Considerations%20Regarding%20the%20Use%20of%20Genotype%20and%20Pedigree%20Data%20to%20Model%20Relatedness%20in%20the%20Context%20of%20Genome-Wide%20Association%20Studies" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/10/1861&amp;title=Practical%20Considerations%20Regarding%20the%20Use%20of%20Genotype%20and%20Pedigree%20Data%20to%20Model%20Relatedness%20in%20the%20Context%20of%20Genome-Wide%20Association%20Studies" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/10/1861&amp;t=Practical%20Considerations%20Regarding%20the%20Use%20of%20Genotype%20and%20Pedigree%20Data%20to%20Model%20Relatedness%20in%20the%20Context%20of%20Genome-Wide%20Association%20Studies" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/10/1861&amp;title=Practical%20Considerations%20Regarding%20the%20Use%20of%20Genotype%20and%20Pedigree%20Data%20to%20Model%20Relatedness%20in%20the%20Context%20of%20Genome-Wide%20Association%20Studies" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/10/1861&amp;title=Practical%20Considerations%20Regarding%20the%20Use%20of%20Genotype%20and%20Pedigree%20Data%20to%20Model%20Relatedness%20in%20the%20Context%20of%20Genome-Wide%20Association%20Studies" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459561/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459561" id="node459561--3" data-pisa="ggg;3/10/1861" data-pisa-master="ggg;g3.113.007948" data-apath="/ggg/3/10/1861.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Practical Considerations Regarding the Use of Genotype and Pedigree Data to Model Relatedness in the Context of Genome-Wide Association Studies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Riyan Cheng</span>, <span class="highwire-citation-author" data-delta="1">Clarissa C. Parker</span>, <span class="highwire-citation-author" data-delta="2">Mark Abney</span> and <span class="highwire-citation-author" data-delta="3">Abraham A. Palmer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1861-1867;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007948 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459561--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Riyan Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Human Genetics, The University of Chicago, Illinois 60637</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Division of Plant Sciences, Research School of Biology, The Australian National University, Canberra, ACT 0200, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Riyan%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARiyan%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Clarissa C. Parker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Human Genetics, The University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clarissa%2BC.%2BParker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parker%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClarissa%2BC.%2BParker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mark Abney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Human Genetics, The University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BAbney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abney%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BAbney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Abraham A. Palmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Human Genetics, The University of Chicago, Illinois 60637</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Psychiatry and Behavioral Neuroscience, The University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abraham%2BA.%2BPalmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palmer%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbraham%2BA.%2BPalmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aap@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">aap@uchicago.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1861&amp;rft.epage=1867&amp;rft.atitle=Practical%20Considerations%20Regarding%20the%20Use%20of%20Genotype%20and%20Pedigree%20Data%20to%20Model%20Relatedness%20in%20the%20Context%20of%20Genome-Wide%20Association%20Studies&amp;rft.volume=3&amp;rft.issue=10&amp;rft.date=2013-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Cheng%2C+Riyan&amp;rft.au=Parker%2C+Clarissa+C.&amp;rft.au=Abney%2C+Mark&amp;rft.au=Palmer%2C+Abraham+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459561/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459561/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459561/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459561/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459561/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459561/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459561/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459561/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459561/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459561/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459561/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459561/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/10/1861" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods, Simulations, and Results</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/10/1861.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/10/1861.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/10/1861.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/10/1861.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
