<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/12/3983" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/12/3983.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/12/3983.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/12/3983.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/12/3983/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/12/3983.atom" />
<meta name="HW.pisa" content="ggg;7/12/3983" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Heterogeneous Patterns of Genetic Diversity and Differentiation in European and Siberian Chiffchaff (Phylloscopus collybita abietinus/P. tristis)" />
<meta name="DC.Identifier" content="10.1534/g3.117.300152" />
<meta name="DC.Date" content="2017-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Talla et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Identification of candidate genes for trait variation in diverging lineages and characterization of mechanistic underpinnings of genome differentiation are key steps toward understanding the processes underlying the formation of new species. Hybrid zones provide a valuable resource for such investigations, since they allow us to study how genomes evolve as species exchange genetic material and to associate particular genetic regions with phenotypic traits of interest. Here, we use whole-genome resequencing of both allopatric and hybridizing populations of the European ( Phylloscopus collybita abietinus ) and the Siberian chiffchaff ( P. tristis )—two recently diverged species which differ in morphology, plumage, song, habitat, and migration—to quantify the regional variation in genome-wide genetic diversity and differentiation, and to identify candidate regions for trait variation. We find that the levels of diversity, differentiation, and divergence are highly heterogeneous, with significantly reduced global differentiation, and more pronounced differentiation peaks in sympatry than in allopatry. This pattern is consistent with regional differences in effective population size and recurrent background selection or selective sweeps reducing the genetic diversity in specific regions prior to lineage divergence, but the data also suggest that postdivergence selection has resulted in increased differentiation and fixed differences in specific regions. We find that hybridization and backcrossing is common in sympatry, and that phenotype is a poor predictor of the genomic composition of sympatric birds. The combination of a differentiation scan approach with identification of fixed differences pinpoint a handful of candidate regions that might be important for trait variation between the two species." />
<meta name="DC.Contributor" content="Venkat Talla" />
<meta name="DC.Contributor" content="Faheema Kalsoom" />
<meta name="DC.Contributor" content="Daria Shipilina" />
<meta name="DC.Contributor" content="Irina Marova" />
<meta name="DC.Contributor" content="Niclas Backström" />
<meta name="article:published_time" content="2017-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Heterogeneous Patterns of Genetic Diversity and Differentiation in European and Siberian Chiffchaff (Phylloscopus collybita abietinus/P. tristis)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Identification of candidate genes for trait variation in diverging lineages and characterization of mechanistic underpinnings of genome differentiation are key steps toward understanding the processes underlying the formation of new species. Hybrid zones provide a valuable resource for such investigations, since they allow us to study how genomes evolve as species exchange genetic material and to associate particular genetic regions with phenotypic traits of interest. Here, we use whole-genome resequencing of both allopatric and hybridizing populations of the European (&lt;i&gt;Phylloscopus collybita abietinus&lt;/i&gt;) and the Siberian chiffchaff (&lt;i&gt;P. tristis&lt;/i&gt;)—two recently diverged species which differ in morphology, plumage, song, habitat, and migration—to quantify the regional variation in genome-wide genetic diversity and differentiation, and to identify candidate regions for trait variation. We find that the levels of diversity, differentiation, and divergence are highly heterogeneous, with significantly reduced global differentiation, and more pronounced differentiation peaks in sympatry than in allopatry. This pattern is consistent with regional differences in effective population size and recurrent background selection or selective sweeps reducing the genetic diversity in specific regions prior to lineage divergence, but the data also suggest that postdivergence selection has resulted in increased differentiation and fixed differences in specific regions. We find that hybridization and backcrossing is common in sympatry, and that phenotype is a poor predictor of the genomic composition of sympatric birds. The combination of a differentiation scan approach with identification of fixed differences pinpoint a handful of candidate regions that might be important for trait variation between the two species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/12/01" />
<meta name="citation_mjid" content="ggg;7/12/3983" />
<meta name="citation_id" content="7/12/3983" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/12/3983" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/12/3983.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/12/3983.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/12/3983.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300152" />
<meta name="citation_pmid" content="29054864" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3983" />
<meta name="citation_lastpage" content="3998" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Venkat Talla" />
<meta name="citation_author_institution" content="Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala University, 752 36, Sweden" />
<meta name="citation_author" content="Faheema Kalsoom" />
<meta name="citation_author_institution" content="Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala University, 752 36, Sweden" />
<meta name="citation_author" content="Daria Shipilina" />
<meta name="citation_author_institution" content="Department of Vertebrate Zoology, Lomonosov Moscow State University, 119991, Russia" />
<meta name="citation_author" content="Irina Marova" />
<meta name="citation_author_institution" content="Department of Vertebrate Zoology, Lomonosov Moscow State University, 119991, Russia" />
<meta name="citation_author" content="Niclas Backström" />
<meta name="citation_author_institution" content="Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala University, 752 36, Sweden" />
<meta name="citation_author_email" content="niclas.backstrom@ebc.uu.se" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0961-8427" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=R. J. Abbott;citation_author=N. H. Barton;citation_author=J. M. Good;citation_title=Genomics of hybridization and its evolutionary consequences.;citation_pages=2325-2332;citation_volume=25;citation_year=2016;citation_pmid=27145128;citation_doi=10.1111/mec.13685" />
<meta name="citation_reference" content="citation_journal_title=Behav. Ecol. Sociobiol.;citation_author=B. L. Arvidsson;citation_author=R. Neergaard;citation_title=Mate choice in the willow warbler—a field experiment.;citation_pages=225-229;citation_volume=29;citation_year=1991;citation_doi=10.1007/BF00166406" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=N. Backström;citation_author=W. Forstmeier;citation_author=H. Schielzeth;citation_author=H. Mellenius;citation_author=K. Nam;citation_title=The recombination landscape of the zebra finch Taeniopygia guttata genome.;citation_volume=20;citation_year=2010;citation_pmid=20357052;citation_doi=10.1101/gr.101410.109" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=S. J. E. Baird;citation_title=The impact of high-throughput sequencing technology on speciation research: maintaining perspective.;citation_pages=1482-1487;citation_volume=30;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=N. H. Barton;citation_author=G. M. Hewitt;citation_title=Adaptation, speciation and hybrid zones.;citation_pages=497-503;citation_volume=341;citation_year=1989;citation_issue=6242;citation_pmid=2677747;citation_doi=10.1038/341497a0" />
<meta name="citation_reference" content="citation_title=Darwin and Modern Science;citation_title=Heredity and variation in modern lights;citation_author=W. Bateson;citation_pages=85-101;citation_year=1909" />
<meta name="citation_reference" content="citation_journal_title=Phil. Trans. Roy. Soc. B: Biol.;citation_author=L. Bernatchez;citation_author=S. Renaut;citation_author=A. R. Whiteley;citation_author=N. Derome;citation_author=J. Jeukens;citation_title=On the origin of species: insights from the ecological genomics of whitefish.;citation_pages=1783-1800;citation_volume=365;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phyl. Evol.;citation_author=M. Bernt;citation_author=A. Donath;citation_author=F. Juhling;citation_author=F. Externbrink;citation_author=C. Florentz;citation_title=MITOS: improved de novo metazoan mitochondrial genome annotation.;citation_pages=313-319;citation_volume=69;citation_year=2013;citation_pmid=22982435;citation_doi=10.1016/j.ympev.2012.08.023" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Buckling;citation_author=R. C. Maclean;citation_author=M. A. Brockhurst;citation_author=N. Colegrave;citation_title=The Beagle in a bottle.;citation_pages=824-829;citation_volume=457;citation_year=2009;citation_issue=7231;citation_pmid=19212400;citation_doi=10.1038/nature07892" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=C. A. Buerkle;citation_title=Inconvenient truths in population and speciation genetics point towards a future beyond allele frequencies.;citation_pages=1498-1500;citation_volume=30;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=R. Burri;citation_title=Dissecting differentiation landscapes: a linked selection’s perspective.;citation_pages=1501-1505;citation_volume=30;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=R. Burri;citation_title=Linked selection, demography and the evolution of correlated genomic landscapes in birds and beyond.;citation_pages=3853-3856;citation_volume=26;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. Burri;citation_author=A. Nater;citation_author=T. Kawakami;citation_author=C. F. Mugal;citation_author=P. I. Olason;citation_title=Linked selection and recombination rate variation drive the evolution of the genomic landscape of differentiation across the speciation continuum of Ficedula flycatchers.;citation_volume=25;citation_year=2015;citation_pmid=26355005;citation_doi=10.1101/gr.196485.115" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. Charlesworth;citation_author=M. Nordborg;citation_author=D. Charlesworth;citation_title=The effects of local selection, balanced polymorphism and background selection on equilibrium patterns of genetic diversity in subdivided populations.;citation_pages=155-174;citation_volume=70;citation_year=1997;citation_issue=2;citation_pmid=9449192;citation_doi=10.1017/S0016672397002954" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=H. Chung;citation_author=D. W. Loehlin;citation_author=H. D. Dufour;citation_author=K. Vacarro;citation_author=J. G. Millar;citation_title=A single gene affects both ecological divergence and mate choice in Drosophila.;citation_volume=343;citation_year=2014;citation_pmid=24526311;citation_doi=10.1126/science.1249998" />
<meta name="citation_reference" content="citation_journal_title=Fly (Austin);citation_author=P. Cingolani;citation_author=A. Platts;citation_author=L. L. Wang;citation_author=M. Coon;citation_author=T. Nguyen;citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff.;citation_pages=80-92;citation_volume=6;citation_year=2012;citation_pmid=22728672;citation_doi=10.4161/fly.19695" />
<meta name="citation_reference" content="citation_journal_title=Science Advances;citation_journal_abbrev=Sci Adv;citation_author=S. Claramunt;citation_author=J. Cracraft;citation_title=A new time tree reveals Earth historys imprint on the evolution of modern birds;citation_pages=e1501005-e1501005;citation_volume=1;citation_year=2015;citation_issue=11;citation_doi=10.1126/sciadv.1501005" />
<meta name="citation_reference" content="citation_title=Speciation;citation_author=J. A. Coyne;citation_author=H. A. Orr;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=T. E. Cruickshank;citation_author=M. W. Hahn;citation_title=Reanalysis suggests that genomic islands of speciation are due to reduced diversity, not reduced gene flow.;citation_pages=3133-3157;citation_volume=23;citation_year=2014;citation_pmid=24845075;citation_doi=10.1111/mec.12796" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. M. Daly;citation_author=J. Willer;citation_author=R. Gregg;citation_author=J. M. Gross;citation_title=Snow white, a zebrafish model of Hermansky-Pudlak Syndrome type 5.;citation_volume=195;citation_year=2013;citation_pmid=23893484;citation_doi=10.1534/genetics.113.154898" />
<meta name="citation_reference" content="citation_journal_title=Bioinfo.;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_title=Handbook of the Birds of the World Alive;citation_title=Siberian chiffchaff (Phylloscopus tristis);citation_author=J. del Hoyo;citation_author=N. Collar;citation_year=2017" />
<meta name="citation_reference" content="citation_title=Handbook of the Birds of the World. Old World Flycatchers to Old World Warblers;citation_author=J. del Hoyo;citation_author=A. Elliot;citation_author=D. A. Christie;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=T. Dobzhansky;citation_title=Speciation as a stage in evolutionary divergence.;citation_pages=312-321;citation_volume=74;citation_year=1940;citation_doi=10.1086/280899" />
<meta name="citation_reference" content="citation_journal_title=Moil. Ecol. Res.;citation_author=L. Dutoit;citation_author=R. Burri;citation_author=A. Nater;citation_author=C. F. Mugal;citation_author=H. Ellegren;citation_title=Genomic distribution and estimation of nucleotide diversity in natural populations: perspectives from the collared flycatcher (Ficedula albicollis) genome.;citation_pages=586-597;citation_volume=17;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Lett.;citation_author=S. P. Egan;citation_author=G. J. Ragland;citation_author=L. Assour;citation_author=T. H. Powell;citation_author=G. R. Hood;citation_title=Experimental evidence of genome-wide impact of ecological selection during early stages of speciation-with-gene-flow.;citation_pages=817-825;citation_volume=18;citation_year=2015;citation_pmid=26077935;citation_doi=10.1111/ele.12460" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=H. Ellegren;citation_title=Evolutionary stasis: the stable chromosomes of birds.;citation_pages=283-291;citation_volume=25;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Ellegren;citation_author=A. K. Fridolfsson;citation_title=Male-driven evolution of DNA sequences in birds.;citation_pages=182-184;citation_volume=17;citation_year=1997;citation_issue=2;citation_pmid=9326938;citation_doi=10.1038/ng1097-182" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. Ellegren;citation_author=J. Parsch;citation_title=The evolution of sex-biased genes and sex-biased gene expression.;citation_pages=689-698;citation_volume=8;citation_year=2007;citation_issue=9;citation_pmid=17680007;citation_doi=10.1038/nrg2167" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=H. Ellegren;citation_author=J. B. W. Wolf;citation_title=Parallelism in genomic landscapes of differentiation, conserved genomic features and the role of linked selection.;citation_pages=1516-1518;citation_volume=30;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Ellegren;citation_author=L. Smeds;citation_author=R. Burri;citation_author=P. Olason;citation_author=N. Backström;citation_title=The genomic landscape of species divergence in Ficedula flycatchers.;citation_pages=756-760;citation_volume=491;citation_year=2012;citation_issue=7426;citation_pmid=23103876;citation_doi=10.1038/nature11584" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=K. R. Elmer;citation_title=Barrier loci and progress towards evolutionary generalities.;citation_pages=1491-1493;citation_volume=30;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=J. L. Feder;citation_author=P. Nosil;citation_title=The efficacy of divergence hitch-hiking in generating genomic islands during ecological speciation.;citation_pages=1724-1747;citation_volume=64;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=J. L. Feder;citation_author=P. S. Egan;citation_author=P. Nosil;citation_title=The genomics of speciation-with-gene-flow.;citation_pages=342-350;citation_volume=28;citation_year=2012;citation_issue=7;citation_pmid=22520730;citation_doi=10.1016/j.tig.2012.03.009" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=J. L. Feder;citation_author=S. M. Flaxman;citation_author=S. P. Egan;citation_author=A. A. Comeault;citation_author=P. Nosil;citation_title=Geographic mode of speciation and genomic divergence.;citation_pages=73-97;citation_volume=44;citation_year=2013;citation_doi=10.1146/annurev-ecolsys-110512-135825" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=J. L. Feder;citation_author=P. Nosil;citation_author=Z. Gompert;citation_author=S. M. Flaxman;citation_author=M. P. Schilling;citation_title=Barnacles, barrier loci and the systematic building of species.;citation_pages=1494-1497;citation_volume=30;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=A. L. Ferchaud;citation_author=M. M. Hansen;citation_title=The impact of selection, gene flow and demographic history on heterogeneous genomic divergence: three-spine sticklebacks in divergent environments.;citation_pages=238-259;citation_volume=25;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Frichot;citation_title=Fast and efficient estimation of individual ancestry coefficients.;citation_volume=196;citation_year=2014;citation_pmid=24496008;citation_doi=10.1534/genetics.113.160572" />
<meta name="citation_reference" content="citation_journal_title=Methods Ecol. Evol.;citation_author=E. Frichot;citation_title=LEA: an R package for landscape and ecological association studies.;citation_pages=925-929;citation_volume=6;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Lett.;citation_author=Z. Gompert;citation_author=A. A. Comeault;citation_author=T. E. Farkas;citation_author=J. L. Feder;citation_author=T. L. Parchman;citation_title=EXPERIMENTAL EVIDENCE FOR ECOLOGICAL SELECTION ON GENOME VARIATION IN THE WILD;citation_pages=369-379;citation_volume=17;citation_year=2014;citation_pmid=24354456;citation_doi=10.1111/ele.12238" />
<meta name="citation_reference" content="citation_journal_title=Bioinfo.;citation_author=A. Gurevich;citation_author=V. Saveliev;citation_author=N. Vyahhi;citation_author=G. Tesler;citation_title=QUAST: quality assessment tool for genome assemblies.;citation_volume=29;citation_year=2013;citation_pmid=23422339;citation_doi=10.1093/bioinformatics/btt086" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. Hahn;citation_title=Reconstructing mitochondrial genomes directly from genomic next-generation sequencing reads—a baiting and iterative mapping approach.;citation_volume=41;citation_year=2013;citation_pmid=23661685;citation_doi=10.1093/nar/gkt371" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Harr;citation_title=Genomic islands of differentiation between house mouse subspecies.;citation_volume=16;citation_year=2006;citation_pmid=16687734;citation_doi=10.1101/gr.5045006" />
<meta name="citation_reference" content="citation_journal_title=Ibis;citation_author=A. J. Helbig;citation_author=J. Martens;citation_author=I. Seibold;citation_author=F. Henning;citation_author=B. Schottler;citation_title=Phylogeny and species limits in the Palaearctic chiffchaff Phylloscopus collybita complex: mitochondrial genetic differentiation and bioacoustic evidence.;citation_pages=650-666;citation_volume=138;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=L. Hillringhaus;citation_author=W. W. Yue;citation_author=N. R. Rose;citation_author=S. S. Ng;citation_author=C. Gileadi;citation_title=Structural and evolutionary basis for the dual substrate selectivity of human KDM4 histone demethylase family.;citation_volume=286;citation_year=2011;citation_pmid=21914792;citation_doi=10.1074/jbc.M111.283689" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=L. D. Hurst;citation_author=H. Ellegren;citation_title=Sex biases in the mutation rate.;citation_pages=446-452;citation_volume=14;citation_year=1998;citation_issue=11;citation_pmid=9825672;citation_doi=10.1016/S0168-9525(98)01577-7" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=D. E. Irwin;citation_author=M. Alcaide;citation_author=K. E. Delmore;citation_author=J. H. Irwin;citation_author=G. L. Owens;citation_title=Recurrent selection explains parallel evolution of genomic regions of high relative but low absolute differentiation in a ring species.;citation_pages=4488-4507;citation_volume=25;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=C. D. Jiggins;citation_author=S. H. Martin;citation_title=Glittering gold and the quest for Isla de Muerta.;citation_pages=1509-1511;citation_volume=30;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Biol. J. Linn. Soc. Lond.;citation_author=C. D. Jiggins;citation_author=W. O. McMillan;citation_author=W. Neukirchen;citation_author=J. Mallet;citation_title=What can hybrid zones tell us about speciation? The case of Heliconius erato and H. himera (Lepidoptera: Nymphalidae).;citation_pages=221-242;citation_volume=59;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=F. C. Jones;citation_author=M. G. Grabherr;citation_author=Y. F. Chan;citation_author=P. Russell;citation_author=E. Mauceli;citation_title=The genomic basis of adaptive evolution in threespine sticklebacks.;citation_pages=55-61;citation_volume=484;citation_year=2012;citation_issue=7392;citation_pmid=22481358;citation_doi=10.1038/nature10944" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=M. Joron;citation_author=R. Papa;citation_author=M. Beltran;citation_author=N. Chamberlain;citation_author=J. Mavarez;citation_title=A conserved supergene locus controls colour pattern diversity in Heliconius butterflies.;citation_pages=e303-e303;citation_volume=4;citation_year=2006;citation_issue=10;citation_pmid=17002517;citation_doi=10.1371/journal.pbio.0040303" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Katoh;citation_author=D. M. Standley;citation_title=MAFFT multiple sequence alignment software version 7: improvements in performance and usability.;citation_volume=30;citation_year=2013;citation_pmid=23329690;citation_doi=10.1093/molbev/mst010" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=T. Kawakami;citation_author=L. Smeds;citation_author=N. Backström;citation_author=A. Husby;citation_author=A. Qvarnström;citation_title=A high-density linkage map enables a second-generation collared flycatcher genome assembly and reveals the patterns of avian recombination rate variation and chromosomal evolution.;citation_pages=4035-4058;citation_volume=23;citation_year=2014;citation_pmid=24863701;citation_doi=10.1111/mec.12810" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=T. Kobayashi;citation_author=W. Y. Tsang;citation_author=J. Li;citation_author=W. Lane;citation_author=B. D. Dynlacht;citation_title=Centriolar kinesin Kif24 interacts with CP110 to remodel microtubules and regulate ciliogenesis.;citation_pages=914-925;citation_volume=145;citation_year=2011;citation_issue=6;citation_pmid=21620453;citation_doi=10.1016/j.cell.2011.04.028" />
<meta name="citation_reference" content="citation_author=A. F. Komarova;citation_author=D. A. Shipilina;citation_pages=211-212;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Bioinfo.;citation_author=T. S. Korneliussen;citation_author=A. Albrechtsen;citation_author=R. Nielsen;citation_title=ANGSD: analysis of next generation sequencing data.;citation_pages=356;citation_volume=15;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=M. Krause;citation_author=A. S. Sechi;citation_author=M. Konradt;citation_author=D. Monner;citation_author=F. B. Gertler;citation_title=Fyn-binding protein (Fyb):SLP-76–associated protein (SLAP), Ena: vasodilator-stimulated phosphoprotein (VASP) proteins and the Arp2: 3 complex link T cell receptor (TCR) signaling to the actin cytoskeleton.;citation_volume=149;citation_year=2000;citation_pmid=10747096;citation_doi=10.1083/jcb.149.1.181" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Kumar;citation_author=G. Stecher;citation_author=K. Tamura;citation_title=MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets.;citation_volume=33;citation_year=2016;citation_pmid=27004904;citation_doi=10.1093/molbev/msw054" />
<meta name="citation_reference" content="citation_journal_title=Gigasci.;citation_author=S. Lampa;citation_author=M. Dahlö;citation_author=P. I. Olason;citation_author=J. Hagberg;citation_author=O. Spjuth;citation_title=Lessons learned from implementing a national infrastructure in Sweden for storage and analysis of next-generation sequencing data.;citation_pages=9;citation_volume=25;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Bioinfo.;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate long-read alignment with Burrows-Wheeler transformation.;citation_volume=26;citation_year=2010;citation_pmid=20080505;citation_doi=10.1093/bioinformatics/btp698" />
<meta name="citation_reference" content="citation_journal_title=Bioinfo.;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Alula;citation_author=A. Lindholm;citation_title=Mixed songs of chiffchaffs in northern Russia.;citation_pages=108-115;citation_volume=3;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=K. Lohse;citation_title=Come on feel the noise—from metaphors to null models.;citation_pages=1506-1508;citation_volume=30;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Avian Res.;citation_author=N. Lyu;citation_author=J. Li;citation_author=Y.-H. Sun;citation_title=Can simple songs express useful signals for mate choice?;citation_pages=10e;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=Am Nat;citation_author=J. E. Mank;citation_title=Sex chromosomes and the evolution of sexual dimorphism: lessons from the genome.;citation_pages=141-150;citation_volume=173;citation_year=2009;citation_issue=2;citation_pmid=20374139;citation_doi=10.1086/595754" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. E. Mank;citation_author=E. Axelsson;citation_author=H. Ellegren;citation_title=Fast-X on the Z: rapid evolution of sex-linked genes in birds.;citation_volume=17;citation_year=2007;citation_pmid=17416747;citation_doi=10.1101/gr.6031907" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=J. E. Mank;citation_author=B. Vicoso;citation_author=S. Berlin;citation_author=B. Charlesworth;citation_title=Effective population size and the Faster-X effect: empirical results and their interpretation.;citation_pages=663-674;citation_volume=64;citation_year=2010;citation_issue=3;citation_pmid=19796145;citation_doi=10.1111/j.1558-5646.2009.00853.x" />
<meta name="citation_reference" content="citation_journal_title=Ornis Fenn.;citation_author=I. Marova;citation_author=D. Shipilina;citation_author=V. V. Fedorov;citation_author=V. Alekseev;citation_author=V. Ivanitskii;citation_title=Interaction between common and Siberian chiffchaff in a contact zone.;citation_pages=66-81;citation_volume=94;citation_year=2017" />
<meta name="citation_reference" content="citation_author=I. M. Marova;citation_author=V. N. Alekseev;citation_pages=306-318;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Moskovskogo Gosudarstvennogo Universiteta;citation_author=I. M. Marova;citation_author=V. V. Leonovich;citation_title=Hybridization between Siberian (Phylloscopus collybita tristis) and east European (Ph. collybita abietinus) chiffchaffs in the area of sympatry. Sbornik Trudov Zoologicheskogo Muzeya;citation_pages=147-163;citation_volume=30;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_title=Doklady biological sciences :  proceedings of the Academy of Sciences of the USSR,  Biological sciences sections / translated from Russian;citation_journal_abbrev=Dokl Biol Sci;citation_author=I. M. Marova;citation_author=V. V. Fedorov;citation_author=D. A. Shipilina;citation_author=V. N. Alekseev;citation_title=Genetic and vocal differentiation in hybrid zones of passerine birds: Siberian and European chiffchaffs (Phylloscopus [collybita] tristis and Ph. [c.] abietinus) in the southern Urals.;citation_pages=384-386;citation_volume=427;citation_year=2009;citation_pmid=19760890" />
<meta name="citation_reference" content="citation_title=El Mosquitero Ibérico;citation_title=Siberian and East European chiffchaffs: geographical distribution, morphological features, vocalization, phenomenon of mixed singing, and evidences of hybridization in sympatry zone;citation_author=I. M. Marova;citation_author=D. A. Shipilina;citation_author=V. V. Fedorov;citation_author=V. V. Ivanitskii;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=J. Ornithol.;citation_author=J. Martens;citation_author=C. Meinche;citation_title=Der sibirische zilpzalp (Phylloscopus collybita tristis): gesang und reaction einer mitteleuropäischen population im freilandversuch.;citation_pages=455-473;citation_volume=130;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. H. Martin;citation_author=K. K. Dasmahapatra;citation_author=N. J. Nadeau;citation_author=C. Salazar;citation_author=J. R. Walters;citation_title=Genome-wide evidence for speciation with gene flow in Heliconius butterflies.;citation_volume=23;citation_year=2013;citation_pmid=24045163;citation_doi=10.1101/gr.159426.113" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Biol. Symp.;citation_author=H. J. Muller;citation_title=Isolating mechanisms, evolution and temperature.;citation_pages=71-125;citation_volume=6;citation_year=1942" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=B. Nabholz;citation_author=R. Lanfear;citation_author=J. Fuchs;citation_title=Body mass-corrected molecular rate for bird mitochondrial DNA.;citation_pages=4438-4449;citation_volume=25;citation_year=2016;citation_pmid=27483387;citation_doi=10.1111/mec.13780" />
<meta name="citation_reference" content="citation_journal_title=Phil. Trans. Roy. Soc. B: Biol.;citation_author=M. W. Nachman;citation_author=B. A. Payseur;citation_title=Recombination rate variation and speciation: theoretical predictions and empirical results from rabbits and mice.;citation_pages=409-421;citation_volume=367;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=M. A. F. Noor;citation_author=S. M. Bennett;citation_title=Islands of speciation or mirages in the desert? Examining the role of restricted recombination in maintaining species.;citation_pages=439-444;citation_volume=103;citation_year=2009;citation_issue=6;citation_pmid=19920849;citation_doi=10.1038/hdy.2009.151" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. A. F. Noor;citation_author=J. L. Feder;citation_title=Speciation genetics: evolving approaches.;citation_pages=851-861;citation_volume=7;citation_year=2006;citation_issue=11;citation_pmid=17033626;citation_doi=10.1038/nrg1968" />
<meta name="citation_reference" content="citation_journal_title=Phil. Trans. Roy. Soc. B: Biol.;citation_author=P. Nosil;citation_author=J. L. Feder;citation_title=Genomic divergence during speciation: causes and consequences.;citation_pages=332-342;citation_volume=367;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Auk;citation_author=S. Nowicki;citation_author=W. A. Searcy;citation_title=Song and mate choice in birds: how the development of behavior helps us understand function.;citation_pages=1-14;citation_volume=122;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Bioinfo.;citation_author=K. Okonechnikov;citation_author=A. Conesa;citation_author=F. García-Alcalde;citation_title=Qualimap 2: advanced multi-sample quality control for high-throughput sequencing data.;citation_pages=292-294;citation_volume=32;citation_year=2015;citation_pmid=26428292" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=S. J. Oyler-McCance;citation_author=R. S. Cornman;citation_author=K. L. Jones;citation_author=J. A. Fike;citation_title=Z chromosome divergence, polymorphism and relative effective population size in a genus of lekking birds.;citation_pages=452-459;citation_volume=115;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=B. A. Payseur;citation_author=L. H. Rieseberg;citation_title=A genomic perspective on hybridization and speciation.;citation_pages=2337-2360;citation_volume=25;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. Pennisi;citation_title=Disputed islands;citation_pages=611-613;citation_volume=345;citation_year=2014;citation_issue=6197;citation_pmid=25104366;citation_doi=10.1126/science.345.6197.611" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. W. Poelstra;citation_author=N. Vijay;citation_author=C. M. Bossu;citation_author=H. Lantz;citation_author=B. Ryll;citation_title=The genomic landscape underlying phenotypic integrity in the face of gene flow in crows;citation_pages=1410-1414;citation_volume=344;citation_year=2014;citation_issue=6190;citation_pmid=24948738;citation_doi=10.1126/science.1253226" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=D. C. Presgraves;citation_title=Sex chromosomes and speciation in Drosophila.;citation_pages=336-343;citation_volume=24;citation_year=2008;citation_issue=7;citation_pmid=18514967;citation_doi=10.1016/j.tig.2008.04.007" />
<meta name="citation_reference" content="citation_title=Speciation in Birds;citation_author=T. D. Price;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. R. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_title=Bioinfo.;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=A. Qvarnström;citation_author=R. I. Bailey;citation_title=Speciation through evolution of sex-linked genes.;citation_pages=4-15;citation_volume=102;citation_year=2009;citation_issue=1;citation_pmid=18781167;citation_doi=10.1038/hdy.2008.93" />
<meta name="citation_reference" content="citation_journal_title=Phil. Trans. Roy. Soc. B: Biol.;citation_author=A. Qvarnström;citation_author=A. M. Rice;citation_author=H. Ellegren;citation_title=Speciation in Ficedula flycatchers.;citation_pages=1841-1852;citation_volume=365;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=M. Ravinet;citation_author=R. Faria;citation_author=R. K. Butlin;citation_author=J. Galindo;citation_author=N. Bierne;citation_title=Interpreting the genomic landscape of speciation: a road map for finding barriers to gene flow.;citation_pages=1450-1477;citation_volume=30;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=S. Renaut;citation_author=C. J. Grassa;citation_author=S. Yeaman;citation_author=B. T. Moyers;citation_author=Z. Lai;citation_title=Genomic islands of divergence are not affected by geography of speciation in sunflowers.;citation_pages=1827;citation_volume=4;citation_year=2013;citation_pmid=23652015;citation_doi=10.1038/ncomms2833" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. M. Rogers;citation_author=L. Bernatchez;citation_title=The genetic architecture of ecological speciation and the association with signature of selection in natural lake whitefish (Coregonus sp. Salmonidae) species pairs.;citation_volume=24;citation_year=2007;citation_pmid=17404398;citation_doi=10.1093/molbev/msm066" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=T. B. Sackton;citation_author=R. B. Corbett-Detig;citation_author=J. Nagaraju;citation_author=L. Vaishna;citation_author=K. P. Arunkumar;citation_title=Positive selection drives faster-Z evolution in silkmoths.;citation_pages=2331-2342;citation_volume=68;citation_year=2014;citation_pmid=24826901" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=G. P. Sætre;citation_author=S. A. Sæther;citation_title=Ecology and genetics of speciation in Ficedula flycatchers.;citation_pages=1091-1106;citation_volume=19;citation_year=2010;citation_issue=6;citation_pmid=20163542;citation_doi=10.1111/j.1365-294X.2010.04568.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=G.-P. Sætre;citation_author=T. Moum;citation_author=S. Bures;citation_author=M. Král;citation_author=M. Adamjan;citation_title=A sexually selected character displacement in flycatchers reinforces premating isolation.;citation_pages=589-592;citation_volume=387;citation_year=1997;citation_doi=10.1038/42451" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=N. Saitou;citation_author=M. Nei;citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees;citation_pages=406-425;citation_volume=4;citation_year=1987;citation_issue=4;citation_pmid=3447015;citation_doi=10.1099/ijs.0.01472-0" />
<meta name="citation_reference" content="citation_title=Molecular Cloning: A Laboratory Manual;citation_author=J. Sambrook;citation_author=E. F. Fritsch;citation_author=T. Maniatis;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. Schluter;citation_title=Evidence for Ecological Speciation and Its Alternative;citation_pages=737-741;citation_volume=323;citation_year=2009;citation_issue=5915;citation_pmid=19197053;citation_doi=10.1126/science.1160006" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=O. Seehausen;citation_author=R. K. Butlin;citation_author=I. Keller;citation_author=C. E. Wagner;citation_author=J. W. Boughman;citation_title=Genomics and the origin of species.;citation_pages=176-192;citation_volume=15;citation_year=2014;citation_pmid=24535286;citation_doi=10.1038/nrg3644" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phyl. Evol.;citation_author=A. P. Selvatti;citation_author=L. P. Gonzaga;citation_author=C. A. Russo;citation_title=A Paleogene origin for crown passerines and the diversification of the oscines in the new world.;citation_pages=1-15;citation_volume=88;citation_year=2015;citation_doi=10.1016/j.ympev.2015.03.018" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Evol.;citation_author=D. Shipilina;citation_author=M. Serbyn;citation_author=V. Ivanitskii;citation_author=I. Marova;citation_author=N. Backström;citation_title=Patterns of genetic, phenotypic, and acoustic variation across a chiffchaff (Phylloscopus collybita abietinus/tristis) hybrid zone.;citation_pages=2169-2180;citation_volume=7;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Bioinfo.;citation_author=F. A. Simao;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M.-C. Simmler;citation_author=M. Cohen-Salmon;citation_author=A. El-Amraoui;citation_author=L. Guillaud;citation_author=J.-C. Benichou;citation_title=Targeted disruption of otog results in deafness and severe imbalance.;citation_pages=139-143;citation_volume=24;citation_year=2000;citation_issue=2;citation_pmid=10655058;citation_doi=10.1038/72793" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Singhal;citation_author=E. M. Leffler;citation_author=K. Sannareddy;citation_author=I. Turner;citation_author=O. Venn;citation_title=Stable recombination hotspots in birds;citation_pages=928-932;citation_volume=350;citation_year=2015;citation_issue=6263;citation_pmid=26586757;citation_doi=10.1126/science.aad0843" />
<meta name="citation_reference" content="citation_title=Synopsis of the Ornithological Fauna of the Soviet Union;citation_author=L. S. Stepanyan;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=G. A. Stooke-Vaughan;citation_author=N. D. Obholzer;citation_author=S. Baxendale;citation_author=S. G. Megason;citation_author=T. T. Whitfield;citation_title=Otolith tethering in the zebrafish otic vesicle requires Otogelin and {alpha}-Tectorin;citation_pages=1137-1145;citation_volume=142;citation_year=2015;citation_issue=6;citation_pmid=25758224;citation_doi=10.1242/dev.116632" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. F. Storz;citation_title=Using genome scans of DNA polymorphism to infer adaptive population divergence.;citation_pages=671-688;citation_volume=14;citation_year=2005;citation_issue=3;citation_pmid=15723660;citation_doi=10.1111/j.1365-294X.2005.02437.x" />
<meta name="citation_reference" content="citation_title=Identification Guide to European Passerines;citation_author=L. Svensson;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. Tamura;citation_author=M. Nei;citation_author=S. Kumar;citation_title=Prospects for inferring very large phylogenies by using the neighbor-joining method.;citation_volume=101;citation_year=2004;citation_pmid=15258291;citation_doi=10.1073/pnas.0404206101" />
<meta name="citation_reference" content="citation_title=A Systematic Review of the Genus Phylloscopus;citation_author=C. Ticehurst;citation_year=1938" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Turelli;citation_author=H. A. Orr;citation_title=Dominance, Epistasis and the Genetics of Postzygotic Isolation;citation_pages=1663-1679;citation_volume=154;citation_year=2000;citation_issue=4;citation_pmid=10747061" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=T. L. Turner;citation_author=M. W. Hahn;citation_author=S. V. Nuzhdin;citation_title=Genomic islands of speciation in Anopheles gambiae.;citation_pages=1572-1578;citation_volume=3;citation_year=2005;citation_doi=10.1371/journal.pbio.0030285" />
<meta name="citation_reference" content="citation_journal_title=Dutch Birding;citation_author=A. B. Van den Berg;citation_title=Calls, identification and taxonomy of sSiberian chiffchaff: an analysis.;citation_pages=79-85;citation_volume=31;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=B. M. Van Doren;citation_author=L. Campagna;citation_author=B. Helm;citation_author=J. C. Illera;citation_author=I. J. Lovette;citation_title=Correlated patterns of genetic diversity and differentiation across an avian family.;citation_pages=3982-3997;citation_volume=15;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Via;citation_title=Natural selection in action during speciation.;citation_volume=106;citation_year=2009;citation_pmid=19528641;citation_doi=10.1073/pnas.0901397106" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=S. Via;citation_author=J. West;citation_title=The genetic mosaic suggests a new role for hitchhiking in ecological speciation.;citation_pages=4334-4345;citation_volume=17;citation_year=2008;citation_issue=19;citation_pmid=18986504;citation_doi=10.1111/j.1365-294X.2008.03921.x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=B. Vicoso;citation_author=B. Charlesworth;citation_title=Evolution on the X chromosome: unusual patterns and processes.;citation_pages=645-653;citation_volume=7;citation_year=2006;citation_issue=8;citation_pmid=16847464;citation_doi=10.1038/nrg1914" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=B. Vicoso;citation_author=B. Charlesworth;citation_title=Effective population size and the faster-X effect: an extended model.;citation_pages=2413-2426;citation_volume=63;citation_year=2009;citation_issue=9;citation_pmid=19473388;citation_doi=10.1111/j.1558-5646.2009.00719.x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=N. Vijay;citation_author=C. M. Bossu;citation_author=J. W. Poelstra;citation_author=M. H. Weissensteiner;citation_author=A. Suh;citation_title=Evolution of heterogeneous genome differentiation across multiple contact zones in a crow species complex.;citation_pages=13195;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=N. Vijay;citation_author=M. Weissensteiner;citation_author=R. Burri;citation_author=T. Kawakami;citation_author=H. Ellegren;citation_title=Genome-wide patterns of variation in genetic diversity are shared among populations, species and higher order taxa.;citation_pages=4284-4295;citation_volume=26;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=C. E. Wagner;citation_author=E. G. Mandeville;citation_title=Speciation, species persistence and the goals of studying genomic barriers to gene flow.;citation_pages=1512-1515;citation_volume=30;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=A. M. Westram;citation_author=M. Ravinet;citation_title=Land ahoy? Navigating the genomic landscape of speciation while avoiding shipwreck.;citation_pages=1522-1525;citation_volume=30;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=M. A. Wilson Sayres;citation_author=K. D. Makova;citation_title=Genome analyses substantiate male mutation bias in many species.;citation_pages=938-945;citation_volume=33;citation_year=2011;citation_issue=12;citation_pmid=22006834;citation_doi=10.1002/bies.201100091" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=J. B. Wolf;citation_author=H. Ellegren;citation_title=Making sense of genomic islands of differentiation in light of speciation.;citation_pages=87-100;citation_volume=18;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Phil. Trans. Roy. Soc. B: Biol.;citation_author=J. B. W. Wolf;citation_author=J. Lindell;citation_author=N. Backström;citation_title=Speciation genetics: current status and evolving approaches.;citation_pages=1717-1733;citation_volume=365;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=C.-I. Wu;citation_author=C.-T. Ting;citation_title=Genes and speciation.;citation_pages=114-122;citation_volume=5;citation_year=2004;citation_issue=2;citation_pmid=14735122;citation_doi=10.1038/nrg1269" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. Zhang;citation_author=C. Li;citation_author=Q. Li;citation_author=B. Li;citation_author=D. M. Larkin;citation_title=Comparative genomics reveals insights into avian genome evolution and adaptation;citation_pages=1311-1320;citation_volume=346;citation_year=2014;citation_issue=6215;citation_pmid=25504712;citation_doi=10.1126/science.1251385" />
<meta name="citation_reference" content="citation_journal_title=Bioinfo.;citation_author=X. Zheng;citation_title=A high-performance computing toolset for relatedness and principal component analysis of SNP data.;citation_volume=28;citation_year=2012;citation_pmid=23060615;citation_doi=10.1093/bioinformatics/bts606" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Heterogeneous Patterns of Genetic Diversity and Differentiation in European and Siberian Chiffchaff (Phylloscopus collybita abietinus/P. tristis)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12.cover-source.jpg" />
<meta name="twitter:description" content="Identification of candidate genes for trait variation in diverging lineages and characterization of mechanistic underpinnings of genome differentiation are key steps toward understanding the processes underlying the formation of new species. Hybrid zones provide a valuable resource for such investigations, since they allow us to study how genomes evolve as species exchange genetic material and to associate particular genetic regions with phenotypic traits of interest. Here, we use whole-genome resequencing of both allopatric and hybridizing populations of the European ( Phylloscopus collybita abietinus ) and the Siberian chiffchaff ( P. tristis )—two recently diverged species which differ in morphology, plumage, song, habitat, and migration—to quantify the regional variation in genome-wide genetic diversity and differentiation, and to identify candidate regions for trait variation. We find that the levels of diversity, differentiation, and divergence are highly heterogeneous, with significantly reduced global differentiation, and more pronounced differentiation peaks in sympatry than in allopatry. This pattern is consistent with regional differences in effective population size and recurrent background selection or selective sweeps reducing the genetic diversity in specific regions prior to lineage divergence, but the data also suggest that postdivergence selection has resulted in increased differentiation and fixed differences in specific regions. We find that hybridization and backcrossing is common in sympatry, and that phenotype is a poor predictor of the genomic composition of sympatric birds. The combination of a differentiation scan approach with identification of fixed differences pinpoint a handful of candidate regions that might be important for trait variation between the two species." />
<meta name="og-title" property="og:title" content="Heterogeneous Patterns of Genetic Diversity and Differentiation in European and Siberian Chiffchaff (Phylloscopus collybita abietinus/P. tristis)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/12/3983" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Identification of candidate genes for trait variation in diverging lineages and characterization of mechanistic underpinnings of genome differentiation are key steps toward understanding the processes underlying the formation of new species. Hybrid zones provide a valuable resource for such investigations, since they allow us to study how genomes evolve as species exchange genetic material and to associate particular genetic regions with phenotypic traits of interest. Here, we use whole-genome resequencing of both allopatric and hybridizing populations of the European ( Phylloscopus collybita abietinus ) and the Siberian chiffchaff ( P. tristis )—two recently diverged species which differ in morphology, plumage, song, habitat, and migration—to quantify the regional variation in genome-wide genetic diversity and differentiation, and to identify candidate regions for trait variation. We find that the levels of diversity, differentiation, and divergence are highly heterogeneous, with significantly reduced global differentiation, and more pronounced differentiation peaks in sympatry than in allopatry. This pattern is consistent with regional differences in effective population size and recurrent background selection or selective sweeps reducing the genetic diversity in specific regions prior to lineage divergence, but the data also suggest that postdivergence selection has resulted in increased differentiation and fixed differences in specific regions. We find that hybridization and backcrossing is common in sympatry, and that phenotype is a poor predictor of the genomic composition of sympatric birds. The combination of a differentiation scan approach with identification of fixed differences pinpoint a handful of candidate regions that might be important for trait variation between the two species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12.cover-source.jpg" />
    <title>Heterogeneous Patterns of Genetic Diversity and Differentiation in European and Siberian Chiffchaff (Phylloscopus collybita abietinus/P. tristis) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bed9eK4HgZgeq1SWIHZ3nUSsO_Lg6SY0VT6zRKv2uw4__AOTOtpTA1sp4jw4ldY-8h8LueAv-V2MjVmPlkIZ--AU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"qgwozag8at7ZSNHxNB3p0_kI1hhcSm32pYqzWNdRA-o","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487467","apath":"\/ggg\/7\/12\/3983.atom","pisa":"ggg;7\/12\/3983","ac":{"\/ggg\/7\/12\/3983.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/12\/3983.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/12\/3983"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/12\/3983"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/12\/3983":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002229bjHnqOs9uyFsOM\\\\\\\/L5sQHbWTDevVl3\\\\\\\/6nHTqmOFjddv0qGuTuqY+NHPc+XP0IGfkW6B4zpOFyTHfPlCp4DDbDvGpjbGj44ueOnHn2YdDogt\\\\\\\/hNdAQIqfFlHmhHu69wgrfTWJIN9Js2JeYXfGu45pVTgugqBHoWx5tFWN3kEpSVoe+AJGW7Uah0upZ+xUvsb29J3HrVFGvNPGRozXG0Si\\\\\\\/c+ajRcG+6mgLjAmVl5c2oq7+cW\\\\\\\/PF0QPfi\\\\\\\/Kcz+5dNfihA4yA95GmykBW2rpFa\\\\\\\/6LVrEU4er7zg0tEZg121a6lTPYN9DekYT7wevgW6Fg+IgTS5edgxK10DVdPh0Z3XcAnsxfi1F2wPZRAizzf1NnZnre6ERdWpdWCDgQ100Yu9nfvoEWywQbTVHBVZJA\\\\\\\/ins8\\\\\\\/sS+fMaNuJDP9K2HQJqjugAWtJsnCaMFghkPjP6+ywsAc7YaxA0cmewpRkTqOjceiNsgMPFHbbCMNl1NF3zevtiRggoWpjI2jGnFIML8faQxWhXHwH1lMre3JQEjrA+HThSYRdk1v61RkYrb2xAATKxfJoWNNDXZYGiobyJYwPjbZGPFW5+2sFc7ogM07w3RpsSgrMSNy23mY7uBTq3LEfabE3Y0ATF7beC5w2zp2Bo49bsgJWlB4oqomiwpQiy4dgPRonTr+FWK\\\\\\\/w9nKIzfYVI9PbovWtAUqP1BjfVljQCfD6njNx8gy443oh6NdkZWspOAKV\\\\\\\/sFGvkHw1QHx7i7ycwqsGdSHZpTJCOstEzwFzHyP3JhF6YYJ2oDhkEqGAMWqvBKNngtypQusmNbkuiQfqlS5n2Cz\\\\\\\/J+hodTbkz3SVi9JCCIbymkkmJnyYH78QnBI1y09xx72IhAjla0dctjH9nohqs\\\\\\\/8k4fOu9JW3goxf0+RyOhbxT4ctLI1Wk+KXVHnZJdHb1R4wfFevMO55SLUr9R4jJdYCNpHPQ\\\\\\\/ClI10hoqbtCfTTwqJphYZzKxnxPTBObuQhx4EcF43uV1bdYCcsjxg6aTVmtATfO3aO50c9VUeEGYGcGUgYpAQlrDQk4b1ZudF4hOrr+iIntrZ1tCBYcsUHNPJ289dohZUjqOGjkNjCrSOFBCfyjMwEM4cWODQx4GjWC6NHMC\\\\\\\/VlhYAv8ApBIimC0VanYetnLujiDLuKEo8xIGg\\\\\\\/wc5c93t6X5Mew\\\\\\\/56tdRzUQ0O9Nlxs2Fynv1Zd2muxgQnNwQ2U3z2LyjytC5tr4z4YyjsX+gjIqrJ5C+xvhgkhQb7j7aSznExZ6Bjv8RcRZ1rLHSjTwMp\\\u0022,\\\u0022iv\\\u0022:\\\u0022g0A7qGZHqnnxyZnTWWyLtw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022569ba44f926e08b6e7fb24b8b7d6c4e3184d899c5af04c4d7daa049cdf30d7c1\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Bsxtt8XwzAFjMYVYe6pi6gjGroeQwvBR97O5Vf+a\\\\\\\/CPCX3Ex9\\\\\\\/c7YURw1qdNW8ZYCfWzmkTn3rfmibN+3jPk3TJr70pqQQW9cmSfBXNhMrEYCpIJKIkH+FzUbMLpabzLybRKWTcbLp8f+h8QNXYtbBYAwrY5VZajYOsiLNXyXKYDqB+L3NIERaRrf\\\\\\\/NekfF65o0TeD\\\\\\\/kxX+62vkYvxv6sNY4DHgbMSdJf3Qq0UYz641QU31HjzcQ342efceq\\\\\\\/8e3FIk+LFOyRl395QOgnOi9YbaE6oKOmtjHpBw9nUkKUaAFWD7V6yAVVMlcK6MsfGOfFv+x3y5KxF2Z131aeq3qJ\\\\\\\/ScN1YvRnic8AM9b9Rcf1gs\\\\\\\/\\\\\\\/azrQQheMoRLXX+yn+1O64kLU9P9M9+ifYYZYtF86ZTXVtB+TaG4ApUyT1GV+qhiWXAxethZvrZTVIG2l9iaCUV5wodBEiXyXXZ9zRc7oU7lOdPLfb1dRd5Klhwo3vz6L\\\\\\\/8nwb0+nNdYsLH\\\\\\\/TLhT1bvUNwARjpJRcql3Q22l6JvMR0FXas+LPNbmue2ZLXNgwfPmaUj7G4FulGJgbRtoM5Ronov31txm2wKB\\\\\\\/O9mWTO0kbjPPxJRThafHfVz5RKGWg96G7beGSmb7cW\\\\\\\/+JoTAiZ6WnnMM6D1K+mj\\\\\\\/gGQa4XZTcNcpw9yssTaYYOyW6N52VSLZ3KGMuoKqYbplkqrFSEv3DjakRbhxlXdeiuTkUrZNhVYBVKnApFoN84E2HZWHICPWP+ErFodKOgd3nI\\\\\\\/qnezLQkN99t9HWFMy+gspMPQMYYOtuQbKc+MJSSDHDwBLycPOGuyXXRdKBWCYshiabR+1AH8xu+QBbZp2naoMwDFlFS8jAu6poFYriSW8NpBJCEQ9nuzuHicwjyNS0tNsu4oe9Nm1p6fSgnCaPQml6Qk0Y\\\\\\\/nByeiHqa7cKYwdevzb0t11WjzMD9c7wyNhSyg3swZvfTo2D5Vx7l0R24aM4B6UW8HiCKEj19vjvI5eFOK8D8St6iQnjIhIhFAGPGkUPz1xrBEA1wkPp424PD5MKBfqXV4+EpjZmnVHMgIoWafhHw2OOeFuS8Uq2QEOcH\\\\\\\/1Rw8iPvROE8w7sQLK+DhrgDx5GGVyj5b2HO2JN9+2qeIN3R4s3hv61ZzUQxxVmZkTpYqFcYcFr7zeQftGY2yMvzKHooNfjUu+cohI1NynbMR9mMfggr6HE1NPQan\\\\\\\/HLaTAydFP\\\\\\\/GHOwV\\\\\\\/eq2hAhWu7AKT+l6KiZi3lYry0SeTQMBGyKtGfi25xl\\\\\\\/oPqgGVIWmQbFvOXInyXryL94LCD+ALH6n9eb9idYr6xzSKwRF1Jh3UCtILWFJrTjLyvpd5uzlEH9V5A1u3qfvGBU8OahRTqiUdZeqSr2rYN0R\\\\\\\/KZH8dJR7VhIy4CWo5UClA1pmC2Fx9KxZMa3SlUOLTLcWeJf1e1632nDBziMevu3O71rswJrLbCSIm6iok+We4anf\\\\\\\/hrJrFva0LbKecgkp9jpF2gtAsf5O0z1MpKyi0nH9Zwf733BnzH9ot2jO2Smt9J7dmYxeLJeGVVSZiztUUR46MwSdRMjHzFkn+fmqtoXKxOf+LAtYg\\\\\\\/dIoWIfCw2jcF59CYqXa7tlaLiz3z\\\\\\\/3g3HVel9eKqmSkkpQzKj8mgwvDi2xCi+F9a2HMqVZyTU+KZa2B4Rv+eJKKIKDknFxU24ykkFed5oyfnMvUCxXFWbLWHdjKS9e\\\\\\\/aGObnfoYEEns5l0klHmizdM0TtjeDaHKL1YZQDXeKe2wIiINvSqTH3ykgNi0l2w87jOk+ZQLWnoJ9tke3A2DQcPdqOdmb8HHVVp5bvQQBce+hU+i0FuD6vPhhnuXMx+\\\\\\\/pVxxDvhi8eAiTidyrScoyGuz13m\\\\\\\/va0E0dvYBMcFbEONMlcerO\\\\\\\/fVFPaknXea6T\\\\\\\/oiJbqdk2YvRFCIwMHmz7\\\\\\\/l6CRRNyNE3cXjipswbgOAeXOcDO45T185RIXBLInyHLQ51AVI\\\\\\\/1a7WTWipsVeHhjjTb+OIYGkZGsBxJo8w1UO3L1RD65xyQ6OwHzHgNy3dNppu94+3dPk5FNzYBMNADbEoQKi0hLYr+7M0uWDf3\\\\\\\/SypSC1RDjYzSWHcpFxtwA8\\\\\\\/lM9IJQxo\\\\\\\/\\\\\\\/+2CFjm3Wc1XiV\\\\\\\/JGP3hhonGLDC0vT0HbMg8KoxNKys48NL9JyP\\\\\\\/RJoD2ZWVKJvvEIk78H4gbnHhzeGvdPiGWMPE6toA6hCrrUREl6eQ37ggIyXB1XCfZ20orA9jq\\\\\\\/HThBV69h4lFCiZKL7Nqo5qR8XLfEsAS3sHJwawQfUmoB1r5RqSMZQlCaB6ZweHXpnFBYLPAJVIuHNZ3eJqHAx8Yq+Q\\\\\\\/07T6hWOOgMGVAMVtf4wf\\\\\\\/WPdd1bVy42Jw+Vzsql9FEoN8oPcCjuuM1ImCvXYK+rSWu+FQ05SvhjUDBLZtRDCooDFlUZK7HA0OoSPwd5ogOcTZgcbHnfLgDYTEvBaSajCYZmAIvcgov+vM3qeP1RV+6kkGjjmt\\\\\\\/r0qsiNio779agN2adEXXhzOByU4Qh0XXfx0zHLPEvx67jP+wEb\\\\\\\/dmiDorhJO684n8\\\\\\\/K3RdkzoU\\\\\\\/7XJf02oXsgv2\\\\\\\/VX4ev3i6rDB4zPoMfnHuMEPoPXiop1MZ2xqBJyZVGcWC0MeInhLvvRI7c5gVhuLKKxF341v\\\\\\\/67Ndia4KVa8gt2Q0zl2Ez7pMZFmdEwjTBPZQzWJw4KzKKcBtbI+fjkYucxNvLdDiqsguI6EWrf77BxaW374duz3KhP6y8Urx1EkyOMgxhCp4lpi0rgOx3rBplLlpeiyg50OlOIrT20zAgQ0R468N9t9XdE41oXhCl32iSaZKqKjj1QIGbPY1OAwxHBFkudn4djpVB4WMPzBeBAfe4s2uZ3A5K9AMqwHawpTaMLJWA7nqMbWnvhV1qBmNQ8oc8EO+i9iOWW5hkBmvuD3CY4d87ULLIUDrPYg1UZ9r2gqZpM2tyLFmShLOGY\\\\\\\/oOSsqmTSObqQ8V8u5FMwp9ui3+LjWcaRTltyf\\\\\\\/ktzIQBNn77OwhRkXkjpM8YfMwiaBZbulMoXVGnhnrwdUePFNxnPSRwMCACG6eYst4qvEnp7LAfWTsmhL\\\\\\\/JTh1zAEtEv7iATY8pJRFpfQpy0+xrieTVgRASszkdNdSZEsg5qzVs0B743U7WviSGwzmFpFtDfQ8N3xVxPNk3g7XMKTGeaURD2okHvdEtc7HfjGOAfuhHOGlqhJ3ekwrl+MVQGcXMomB9lDGuMrS+rbyDvnAw5eFcdviXtGL5g3yxTfMTiiIb5cO3c\\\\\\\/WfJiuwHw7tw7hxJwls0O04km5rICRgGzKKymEM8xqAO6bYRPWCNnOdGD3+vcjuLB586W75WWeULTz2Zw3wGrRgxxpfh1j3BcJKkclcVyEKoXM9ZGrMsVdR8f31ZGgksHNJTdV50stA1660VRR+3hNxAkIZBBldg84\\\\\\\/1b+5n1hdLuNwwvWruO2XbIvWslsLHFjqYL5jCyCI1\\\\\\\/DRqZsoxbqJucKN9v\\\\\\\/9zKrO4PY1qszns3BEBc6GmEvZKwYqs8MEKCIX4s85xij92zwCMPCXPpaQP1LyM70lSHOI3lk2e8\\\\\\\/Y0pSL0oQyBa\\\\\\\/GTI88i+Ma4GvtEbRO\\\\\\\/mi8OL+PDvVHFZeswgg5bDR7A8\\\\\\\/KM+pDRxJucKNjYA2w1WGXrrlsHHZ\\\\\\\/KloIrWjIuwsz9tSuNxHLiriveds2YtfTNo3aS2jYMhVwX09Yfyg6+Ay2cevzL0cVX19ocYRAmF\\\\\\\/tc\\\\\\\/B9yzDpSakcQQ3XPx7yjHhMozv7GdTv+pt9NZFadd+j3GAaEf8z9mcIEmf\\\\\\\/NWwEcB\\\\\\\/yrPCzGubEOJ4Wcdlc5x5Wk\\\\\\\/FLcwkudMmtt4+9EaKFLRGl5jS7x0ZVeY+XKFPeA5Q1y2dX8rWAZy0rItefJ18Mhl0dR6LICv+VLTZisGiWB+2raeTG1uBeybe4pCjcADVNtKgEhfABIk21zaIW3ugX58o6c\\\\\\\/YB8urUheMjx+XhOzxtVKXhGFQ7\\\\\\\/x++YT3JA0Oanb5lSFyRt820ohqrZ3XMKBJz5IWgqEaqKzCJ4PfHzd9zm3fgC8RFUubo3okpC87GZuHPppt2rNKj6us\\\\\\\/dPZ8dcwykDvDPyXYwBqB2oY2vcMsWVXTnAOtyKiRvQ0ha5k8jiFYdVFyukaiDluDxnT7TxSC60JjGTt6kw1\\\\\\\/3iRU9tC2WsDA5RCsQ6BDOhXQn4oNLkdo2z3oEhhGhnrFRNsBaerHPdF8K8IK4SEwPB2SKpDdVP0h1ZmJa\\\\\\\/07lGOgH2jE5\\\\\\\/HzYPA7ijOr64QWz50dsxJonN5L5J2pYjephkV18wJPdcwzXvxI1ueD6gadQCZJ4NRJjMhDJNvPTbj1VS776zwPX+3lypAwgnIsbtdHvJ2SOJnq1LPd65vF2vntcXRLKXQasvs7UMYixWc0TskqQlgJOorUQHa2VqZ9Qcg7wQVO9BMlgGjRLs8zocOW2Mc6Ac9PuMq7TO5VDd0smmpL007UUkGpUhFD2BFPeDc2akOIHGrZO0Fil5PzSCtgQQD5OmnB57JOYvnAuZ5f4RaX6Ot98JhFuIJLi5TdbB3xFCJ6aQ2g3k1AcruJns8gw3Dt7CquTxJqze+weVWzBo0cVD9PQd5fpkM5HjDwGBElX53pB06Ogs21pn1M9DJROnphIwUp8JzK423+vH\\\\\\\/9ROtguMhWIvpfXFLzS92Rc8c3DSaLWtnFGj26Vsk5irpcSvzfqs5N1fLALrPfcgJC6elwEat1ZpoOWKJCNBmucy7qhytgMtnVPu1idKvBnGRfemo09L8HheH+DXV6rAY1njOzd741PZbEO5ACpbvS\\\\\\\/XmyGMZf6wpp1w0d\\\\\\\/FSlk7lo0AKHsPRebumC2nf6D3i\\\\\\\/l40uSd9TZ7BlzzsgY6fszzhO1jGoZGsDZhCNclyGKk0ktByMHFD2HC0yPqg\\\\\\\/C\\\\\\\/UZJf32TaSQwM8jLGSvHu4uw3XqnKmjCX73aOr6LPJsmKsa2MJgt3qVgQ\\\\\\\/gcw7yWG2lftS1WVdBqxEIrCXhJIyhRdc5VKUowJfb0dS5+Pd7sIlnY2Va8WhGi3E1CwojLBRRQTnZaCCh9jujMPijnOkAYA0LsNrP5te6qSz63i2vl1k2lJk\\\\\\\/iwotcB755iyWpebZ74KCPujGNj6WDfrf7RRFnL6n6qz\\\\\\\/ow3XE7F8HwmtrD6qc5V7JYyuBMttK4CCresyCZvfrpuJjW10GE6n4YHP6aEEsqZjLuj6A1x8Vbhn6mPDovOJbNLMQd3wAFK9uBD6+i7rrYn8ckCEp6wRkN6PDenWHq\\\\\\\/3T360ELpjExknLfLUHqV44OmtCXDlQf1IUG92p0DZXphmr9o3ccx+4nTdMoN905n5jByVNcWv45aSE2z7846G+wYq3XdohDa\\\\\\\/ic2RvTJa2oW8gK3yjIlGv0DCwKrYGhOaG\\\\\\\/j71FP+yRB7xRh95JeJ1X7fngZ0A+ur5FjOL0o0oNs1MdKD9wgMgQ2ACQawU+3GGVPWMfXo341N\\\\\\\/aSvRLaINyuoU5Rv2szIbSHodZwX11pCyKKYg2W6eSqbV1ykDxEFjp21Ep5EX0GtwbYJkjIMGUiDrUWUXMDlpKY\\\\\\\/jhhoION2pqvrgYq9WvrUbl4015dRFfjHKdHFTMlyGBdxK\\\\\\\/5xFbLYRY2Dfgk8PzTX8IS6ncasNb8vU1r0NRrhMGSDro4UuCsJHT7vimizlq0nrYOuztQpVPmTx0n8b98JbCWlQ7q36XdCtrw8KSTK2fp\\\\\\\/q0Ue3+LTVI1vGRBMbCDCsRwKHqRQKmpK2FQynv5sGNOQ2i8BCl4XwdYDax9gfW3q7cTsutCugYAF7tDVaraZRSwOk0mr5G\\\\\\\/Vhc4wGooINBVF1z9tTD4rc\\\\\\\/eKsOIkbZisJJlLD+EU+FCRWTc\\\\\\\/nr2bbDe4jpcJLjjkpGbKL8HkCg0yjRs6k7FTH1ER8FnwnGp7ado+SuVn8nJ8c4TUSM5xeg7w++pha+pH5pfEFtmPgeNdgvJgBV\\\\\\\/y3ustuxnaOgbIkRNVrl8d7uCxwzlu1oajtOEm6YK6dmMZQpaSwlp+XFIvtKHLrS29H2KW1o0mOmg9pqyzYWAQ7KYxQH0CFhLVQWBJRbSXFHp1ilDJqKiBunqWA6yqQeeZqKpm5IccU6WN37eClARbmrtVLaA8vq\\\\\\\/gPA82T0eMrcUvXJJYPQ4Dmf2L8z6LNnA0qcTIAPyGqB03FB8nWBnrQvgdj\\\\\\\/mz7DwvyWFwmLsk8whrwRCcmd4jl6gmYH5E8rVmOKixUTtcy3rDeHUuPsM0uZnCvCKhPqNduYMEBwXAlmlNpAeZT+t+o7OJ2BxKPu48Br2AfZqKI2SoUNVdBEZznURMBqH7TT7T+TLcohi6rDUtxPEv40f46LsPc9WhUceamGXV34OKXIrNRqFnnY9c3bHoo5u0Uoo9sfhDaNP1kbTN6ThSNkmXIe7dzAXrgrkpb029hoL9lZ05w8pSV2ib3IM1q7rpspClWo6\\\\\\\/QYie8AIRGcJyLrTDX8l33h+CFN28F5EUN7bxfocd\\\\\\\/p9mHtOZLM2FQOjunkpWbLmlIettnPCNjr7VehA3hXzeY8dZNEhL1Kd2ZsYPsf3eT0oJw8UHRFeYpmfaLURR9qACpgJOpCT9wnrdkvuZs13G0pu3jrSqV1pPjvyHYLUWNWZHUR3QY5wKthKXQxcWO3wh8FFdlknuHTsYsfpUae6P\\\\\\\/u+3X35ZBlhI7kZ8o8Gp36gPYxM2yd147weEDY6i8ygLD7DOsNNZIKvzCT24alknf98RbKY4iv6DBtjYCUocI0Kl3dh3l0jUZMOgSuEEJ\\\\\\\/C9PDcP2YIAdwrZ6dRHyjzylL8kDfis3yxuz5RemmK1R3aK9MR6ixBObnSu5OQ0DGnqjM728ZH11C4fLwniFPSuCI9jcgw8JKTkZcycYBCLmUKlRtd3knBBOaxg+F0LatSaWQdwal+TtGNMoMn\\\\\\\/NQHlC+fPz75PxYMkHXEB2cVxXbym5lOvEJQj6+DaW8j5F48Gv0Uorq6u0x5CqyIoVUVue1AuFaDjWDtpjozyK0zveCDO7oT7goBAgOUHucEwA0kLNCnug4HDsYpwRaOBhhfX1oGcAaJmE6b7ecElkQjfato38vnuw\\\\\\\/80JJHVgAHfw7SbuZKgL1bwgsPI3dD5zT3u6w9pyo1HrF7ZaHP0TMn0xry6pQYjgZs1BxGfXXBn0mJBN8PcRSNa9ckxcxWzg2pr7wlTTZlw3In9I1BFqdVA48s61croReXTHBgdvbfpD6XSWpYpi5rDVMr1tm6XtJj2vo4QBcE0ZyRcIceiVcK1EVdr0QENjHYh3OLvrc8Bur\\\\\\\/iQiTBnpp3OqWhc5wmVLtv\\\\\\\/qmYlSYL50GcChm1FzQjO2k3\\\\\\\/g4XfWr3A+1aU7BQj8wu8asLZIVD1in3yc0Ir4OCOPLUM0d6d44\\\\\\\/kbWv\\\\\\\/zx2ZFhqn30oL5YarJ6oO+HHfnCLhGNCVkg999Xnh4DIszY05\\\\\\\/54ASzEjuQDMZwMdGabjqEhpuDhTmCJGcTvZXfGzSMoDjg09KqzPzv++NPn1sOOnlBdSC1EswWUbZvOpGEOcYRjzR+IiMREBWmVxfCrlijXteUZARkJSHqc8RUl83xFbtRJtd9COStA6IK4BhlwEFlbQ5eoF+O7CZIVmpL1pdBU367vuJoNCPD253wsSsQIJ3ZNYQlajit9zn5Huw57zlvQm69G3E5\\\\\\\/6nDijSVkMoXPWTZh6arDu8Dhk75OtAGgGdYObs+omETDyDMrx6ZzmzVcldoU35N9C+QIhxFpRx4CrxAQloG2315tId4sOV5eiMkKyRQ4EIp\\\\\\\/mSorygoE\\\\\\\/h16uBuYx1SqMo1Jh40CgUILkETHG7wN\\\\\\\/MyWOi7sTlfnzQt3hBsFE9mpEAI\\\\\\\/Mg7ULZzbsxuSFGKcnYQBuDBB7Tt5OKVNLwEuVXnynJeYuOVRX5iu\\\\\\\/9LS75Ky1ZoK+XjgAB9vdsBs\\\\\\\/8kiyDKAPza+K\\\\\\\/jSDvCbuKlOANzyz9QtU8YBKmrFFkJBvkiuQfqlY5XpVbJDUy2tLjmTBMjuGXe5SUxpeFbTZ2\\\\\\\/QUyHRTjFGj6gErXzImEITwMtpUq3bBB07ZwOQaX0FOf7MsRGw2aFJDXcWv\\\\\\\/SxBilliZmIhAdoI2Bxm4OyfFBuIlDBQGl4xqlVZpXgVHp53Gd4vsgnt0TnhUxbAlnk81XfMYcwq7VzYyciDYYmo7ZIKaFSzcmz\\\\\\\/Wekk51Y2wbvg1dQk6wH7Zmn\\\\\\\/6lJhClDpz0fHzP9CETfVn7Gt5JHGOY6IaXN7iJUnGVeC\\\\\\\/SIeMVA9rGdhVGcOYx2HPE0WwPedNd9GB4LGNIsk4KdH8fV0g6\\\\\\\/82eGwVqV2eSzXa2HwYvQf5DrmTbXIKbuXPyjLt4ZqDWtAEpBB7cE7nrrNUKlgfaONXZRZUti4EG5D\\\\\\\/wRRAGsQBCy\\\\\\\/TDJphQ47XXGKa3AmcIeO2R8ypMKzLfKZjSFZbGoy0vA1XQywRs7LgZCYgTAfwy3VjZQSZZKtHApCIz8KNI2VbvSz2FxK4GefwyZjqS7NHDioJi5kkzESc6wwPTPnggYehm1eMrKbiLysS6CwWUTyi2bBiD8fAGfGK\\\\\\\/vJwHzZVvxjU8svG+jIziodjiF3njKiyvaHxDk3ob5b7SvnjMg7P1\\\\\\\/mSRFHZmsqUzTkizWtoXovIvL0eb6w65bqHWAuVvJopcnVvvX8S+xCl5PbALzJRpHltGnNyc1tuTnl12EiQ8NtMuh28rGIxZmfKQHW3au9YPJVqrNNV4as1X+7CzML7o82sG0JqhJLcb8GHTLo8buh9u+5K4mEJRY\\\\\\\/QsZtY9LGhThlrZUhhGjdsqsiK4PQOHMaJfr5e0IdiL1OdBZbqQRvhRZy7lr20nFHOk7rklzj4yOKA7bEmqLtFJOx12HvRwgI0H8+IQrecWSSaffPf4Dc5e1MSILG4TCqGmCnnk\\\\\\\/Z4aqqlYyK8uvsDIz4YldUNWEPWNst7TImE87\\\\\\\/eMtYmNKn8Nrn7Qnc3r\\\\\\\/4\\\\\\\/7TXtui7gHguglLVb2\\\\\\\/NVmB2JeYvWeKeTG4PS0oGso4bBgFXvgCPHYCmtZvm7tGXHiCizmYjM49I2AZFKSMDE+HlPALFOftvS7inNaAHsKAD8TJrmL7+gUykwRG7MdTH82iMp\\\\\\\/oA8XY8UkLOLqFS3lbD+HQSj7nRSpNg858mMIJo6lLhZ+5heoWNQua897rnCzyNAl\\\\\\\/rwH8vK+r0AaRaTFtfa+hBR3AOmZHxDE\\\\\\\/dtsuf2VXCWDuapnN3E9jF+eUbejBQD6fuylqO8yv8yii5TqOUMLyFr1vAeGeN6BOQ21FRAZ9k6PxKM4RdRA9Tn9RNsxGSBdGtA207ItWWht0lgtPY1yvLHQxZ90c\\\\\\\/N81h3\\\\\\\/ZQrfjvniAmUYWOwqMff6XW\\\\\\\/5l2cWA9acTcm6nrwkBQPROXo7aJTf0zlghdc+hOe6BhJ9HSKBGUw00APrnjbwVOdJ23H+QiNr6kLTuFQ4O67YpfidJEuIWYqPIBXmXfQ9AN9v3WCEmja5AlCGEHg\\\\\\\/sWMrh2kLagUYBiNboBRLFYKNO3p50rzMyNXqqhuWyZEWBJSwvRpjvA8HNly7QdEsTbgEmNTmw5BQvzk82p1FV\\\\\\\/XQvi7jWjxoFcB\\\\\\\/Cqm7j0YXIWz7BJ9VDtfonPtLhagNaOQh\\\\\\\/RtCl9nMXrNnGVNK+BMcZaYKX3czRkokuVu7XHjRz1SfsEYFrRu7u5jT3VStBdcAN2gKbLoPN5x0\\\\\\\/U80g+NxrXUZvDJlND+KBdM4BaSs5tUMsZfv7NluCwYbcQtllLGXfJaYxU584VROV86IdCXMLj2R14O8ziuecZj1GhdsbDyzCZmKZNK1LNBRMpOmQ3qa4rYPnQHXKskCwdMHVyW1kvzVCm21xlfPRdNMK5stKHrXnMUM7g+cAMKaHdZ8LBuiWUl2qpJ8kiay78qW6EE58kr8ikQN217L2A9hQvSqexx+nGOv\\\\\\\/fN+SuXCNfJEt7+1LUqBrUbLU+zf\\\\\\\/Th\\\\\\\/3fWM\\\\\\\/cCPn9rNCXysiua9mWPzTsZvqpEhgyhyDEy40yi1mvggNYOCAQKyV2nl1MKQY+MJh6m5IsD4m2qZtJ\\\\\\\/WK5MG0iy23LTBbVw01VJvSwpXRAD6RHyDwIjv0uHdwHV098tMOCk6bMrq02Td3M9BCi3u8aGt\\\\\\\/kL5n799ss+poNiVff+p8XvT4QBWLqUSnhbF5b7OR12yk4vZHbmzWdVFDbDDpoOe+NsXl0ZftkfhetIBFSOxCeRxvmKQPEiZwS\\\\\\\/eagg+OYNV8g1r0m3drgHQdzcAl2IW8cLz4hiauwetIVY6oq8LeXnfK91f4AmjHXF73luSaaN0oJkWuYSeDu9dvYWxAA6HLOdaq5pJUR0YZc9X0O94JQE\\\\\\\/CyFesSV7tilOARsayd4bzKcbT5e5qn5pLlw2Zv6wcKHWqZPXlteNwIP1w42vEIaVdX6c1OI3b+V50KB0+kfXM2fYCb3r17vsHNoph6WlpvN+fHXUjgFrugJZgMSFoWM\\\\\\\/GpbsNFMyY1KD9EOWCDjM66z6dDgkaALPSPXGdx49AiuqFQoK1g6SP6qk9c2A0O5Pdvh0ClsnRCkYa88MBZpyBHxAvoHlp69ThcRFgPZRCND+oPFSHZ5T+QnExXGj8Uly+qlGIW9rf+PqiSqpenwX7IgW3OXUgK79B2faNTMXFrpVwtwPGkhC+vWpMJ0cqjKgBQZlp6JCmiEeH6saJ6kY4QJC3i+6VPUzNBi2kv1Gufw2qb+bi8G6xxQyuRH939t0XfLiJuH6Dz+O1pqzRCb\\\\\\\/Xo6S7B8hknIbFd2bIbAdffGjlkQwKwygjZvJr\\\\\\\/qN\\\\\\\/yV1n\\\\\\\/Wi4uZ0vJFtCwErccK3jHUKYr+O\\\\\\\/yjV\\\\\\\/5UqUwh+PTmOMoS8oGpAVCscHcnJw3AdV6USBD0A19awK+Y7dI71UHrk\\\\\\\/HiBcZidhlN0o\\\\\\\/lKPbeZWHzHFHxuaUDWLbGTYoLYnPqn8gR\\\\\\\/bZwZAZgl6YpXa2kmhIgFVnDXFPuragYOTfyBi6e0IRYTQauKO0XJwepzdAsI8cF1o2w6UQFHbYO9m+6h9n8sj2Pv\\\\\\\/IpwkZkYN6C0mUv1i+O6a3SS0rrlydZOERAKxuMRF6vX4XIEbw9b3m2\\\\\\\/MHdD+gdcegyFzh80bMA\\\\\\\/WyVXqyLTci2SglH8vYLbCizSzZnmBa9wT7RhPeUF3IKWuZ5IUq5lx981M0VGgKyHtfSiKG3u6yU20384irvDdJjxKRA1iie7HfbjCFQyKZ0FT1b24mfMYuGIW\\\\\\\/8Dik6o7jZtOXCG0aHR9bgrzSN0wDWWL1gOohL8I8lbrBn+m4WLeiBmD+mq5aPDJcG2L89gAhekctbuVIDa6tbmKfev2eYOcbtTO\\\\\\\/IinmqYhRNHtNakIIYkBrTrwf8JO8riuH0+9ojl4d70iHpBmUE5I5ik9x9ueJtGnpMoilAUGxX04tdNBv9JpDhgvbE+MYK3YOs9dW8j3wRq3sNqBNlVR+DOnIWKBnpkt8I+LGs\\\\\\\/IxviHIvaLddoKrTzw9S1ZKKyevPk7n9AkBQvD14sj5308T\\\\\\\/Zwu4b7EdCeNi6G0AWDUdNhsgfELpuBFZRTr8vzuvS796k\\\\\\\/aIWsFfiQ+AIicE5YliG\\\\\\\/UvnLuSQ0UnDXNvRzV18DCurPNQKEuDTpNh8ejtFjndwIHCncB8Q53DdxQ8+fTseqjymJ5oF\\\\\\\/mGEL+zvEtNtYGtMitLNcGdH8vOnPtq6GhdaGfKmkt\\\\\\\/I+VTS8EHhamasVCl\\\\\\\/cHZrTaZg6HM82Ur0JiBoLlc5BJoZAy2SE+jTvw\\\\\\\/EtrU3SaYWmUZ\\\\\\\/WNte8sXzTmeoNT7jhnxhxxblBfBiUZVEJsvyRiPuBPVZj8PBypE31M5oKOiKs4lHj0qbfe5C+XDTCrC8mTDFtn8kIG1Wu2mjoDkyWTv975KC5uw+TxtdMKR7G4JQDnCuTY3FEGz3iFBv16ZimXYQEG1cgup4rQHo+Bgnd6cB7vIMmZ1gHpLWM0hFvgQsJqlHwMkatT+PBiGpTL5auK5hqlVeUlYL9G0peWrb0Skp9N\\\\\\\/4OaZKzyH04xqO0y7lfqPfpCYECAckxgVarMAZJApyH0gTN6EwOVVXrs6SY1DK61A0s\\\\\\\/dvwO5rTqcxL1m7GO2dkzjd4\\\\\\\/Gl0kM7lZyGmPk6sLtHT\\\\\\\/nxUvnb249Dr2Eu0X4Gpujf8CdvLRVwV+5X2cwx6yWMl6wSOLB8Eg6B1eI72eqIhqoMSiKpckS1gC35G735HLA5NCZ75zZaKFcJMP7fuTmX3VvCfVZph0QExzMghrCFjuYk1PFTBGgLyNJyYLVJTTnJxPP+RV5xtNwsPsaSnsdZdxfyeIX1dW2UhktpuOp\\\\\\\/7v5Do\\\\\\\/C3Ow5um\\\\\\\/f053e4yTsqwCaKUtv8F+cXIRPmpM5X95D0R8+QW0z4WnaXlBthPaT7vdIlbnM63rRjmQqbBeT1PIfPeRu2\\\\\\\/zPBx2H2Ph749iGwx2VGQokC0v+Y5zlG5dbroyD5ZmNFoHfBwZdM6AQuSZIhPj9oQzpTKRy5JubGeBweTEBFQS7CgfYGuO92K5Yf0ip0PRP38OLS+JJuQTpyjkbk8qZxQoRj0fmk\\\\\\\/4pmIccXSf6uX6Y3vOPUIPo4aNZWQySqEmWuq2NUqsxPO9czHodNAMdj7R0YQKXzVzhOaxbpBwEhD3hjm8IgqgdRsrdcOy5Hul2F36xHAsZrMOFi3ypeAx7+t616bUiG4p0jwPw3y6etI1iyPfzt198ws8iKfn8k9wtrk83FnAOBQys1UQQn+oL9PiAzvpyYzeY2NGOek9I3AhTGDbfOJWexyK9I14NaUqtbcWCtJjG0tf5J\\\\\\\/1HWvGnX6eDzq9LKZp9hSHKodEKVKRaeOlgWC2FY4PuOPU6Yhxw+Tp6iB+zUjeY6ZTkrADs1bhM\\\\\\\/rJyU5x4HCAL+wKKqIUo\\\\\\\/M4\\\\\\\/ijVoFny86Tz26qa8Wx4FRYKOt2MRW5tCfQhNxebl7DSmLD56zCS6EBJHZVkd\\\\\\\/l5BKAy\\\\\\\/FX3CGPPgmTB8xiT\\\\\\\/36dgjpHNRPjotmvhWJFEN+f7nCjyRtLY5cYGst9OszL6ZXevYM9jLEPIIfb1ZEt8Dqa7xZXsgYUXNVR0uMkpGbHIm5UZrGkbOwTtb+5ZFloysmqJhigG8qtiFHcdV1Tk0SFfhokPO6AYmBS4b1Bj4LIkqvqzYLAOEvT4gVplo1+ItT7fobGN3Q\\\\\\\/+cWUAgILrnxZfvFz\\\\\\\/knUFjp4CcjYnVwIEwxQLhxGDclSVRiP73AgI5SkzTYEtVMjnuvQMf+U7Zqc4b88CYJSYtWWkyDrFgGIgvnwPdL1YkiYP2mdNF4M1TF9IgPAGY3+FVecNqahhXLckjJzayXozItEbzwzkj5zC1s1jl568muGaeQsXfqsAAC3JM4xmEGxlCK7kxm9gqj6wkykr\\\\\\\/G0RNJVA8G3jOjTuJK8iW\\\\\\\/fLpITQeDA\\\\\\\/eeKo+UjYczO6e8DWCR5+i8HQsHCglJgq28MhrYcvkLoWKnGNnrmLmIItOvWwrarL7J7HIdw6bGZ2fjx0wUd\\\\\\\/UZH9ev2s9FYk35NvImMJTo4NTA6qmtg3xW77xvd\\\\\\\/E3Tk34bpwUmcCbzYAyy9Kj5zJaLJ9\\\\\\\/aijrZ19vY2skkUtlcL4PeiuyyRKKuRU7QRI3IyIUppLcecJdjgD+m77RYi0IgqjP\\\\\\\/gTXIi90wf61LG8mlzLJTNpEfIrBesOcxCCJO9\\\\\\\/pyLhw4jT8Au0WFNvAXEu71f8UJIlKxeXIdTcc6HEix4ki95hsbBmgtKkNBSQQiZvzyLQzh3JadWxvVYQXSo3HXJe9A6CB45bUnQ5LOpbRS90uURrnCeDQOWkueKwTA0O4hSlD4GLf1z9MBMaOoUM1MPlHva2PTmX+tB7rE41ekTZTScyTnNPvZjR4\\\\\\\/OFtMt+WT8vLiX+tRCW1PqKEnU8aDNrKEpyv7Vj\\\\\\\/siimrYPuS59dbbQwajdBQY5ts6I+2sK1J2QKwVXf3qCtBVyDnmnW94dGnxn\\\\\\\/F5cudnxnYGRvEaFTwOd4alN3yrFCwSxrOtHHXCFFG38XSF2T2cNAdYnVFdtDxnOhOKA2IsycHocMMH4D7wh7HMNm+ma1zrtqrly1BPBAm8BPCV9ow4upmZvSm3a\\\\\\\/e8ySPK\\\\\\\/rzB4pbWE0UHP0AEenfxklVbUPw+qOYJfGRUX+N0LNUhY1wIalQyYM5VCb+nZnbe50OaRgGdMfRCna+v98RRuVAaofXC0jPeL96yMf8XPVC9l87keQDcv8RYb769ZTda8sf8ynM4kp2cR\\\\\\\/eS+jU4+ux84DSntqYAhm6dzEs3bm1dlC5DbZ8Ktp2W88G61qUSWY07HNUXZ\\\\\\\/S2cYSpdB1luz3wIEJGvYD+qscaQX+y6j7ZRSuZM0sQfMI1in\\\\\\\/uGoZl5tc4hhbzwNxnqOesNJpZYTjJO1DrebxfOeepgAV+C\\\\\\\/nAj+FenlA8jwUvDBGONKrDXE8VFlXrb6rgoZL0Df2\\\\\\\/Lj3X\\\\\\\/tKQp1pzKD\\\\\\\/Z8Q8iYC6+KZwhDfB0Qjw6Hf4S9ADuSwAZkha3UBYt+LCLh4pxYR2+KbwuJbHGZ3Oom4PFA2bPa0E3BjL61Zt3nYY2v\\\\\\\/QP4qzT8azyzZGXQ0AJzVM8ewhG5wN\\\\\\\/4HD4hlnd\\\\\\\/zBbUlc1f2d\\\\\\\/i\\\\\\\/Re1kavdU+7SO1pv8RRvib3lnUmYs3\\\\\\\/hCZsmfVBetiPwYqqYCLBua8ITTgeuNrRTDsra7vNGLCkvltwl8P1yNl6FVLwSEnr51q\\\\\\\/0e5wZyh9u+4GjstSzcpJn07\\\\\\\/QY5TTReGs4d6OKj0yrSVNRLgTrPNIBxxIWIdd2LqfoI8Wd6+Knt8Wi2z801kSds\\\\\\\/y8FqmLSbGYbyZT4AEpbRnwHgOphoPJKUj7+DRU+DIu0aip58vXMerxN6\\\\\\\/aqIQFto\\\\\\\/iajQIAYQBGy3idChzZj3EIz\\\\\\\/0j69c4SKbYFnFuLIQfbzZ\\\\\\\/4MVtiN3+tzn7KwEyEPlANoVNlvblCmeZrLggvbOISPpeQN69jX0N1sI99B9iY2eZiTrEGC5LjeOu6p\\\\\\\/tD2kSmGplO9IM8au2TCPziY3FH7SewT7ss2N6yQ9sB5pcXMGXKlVfMzGdAHG5ynAOV0StulrEvFNKyeR7EN\\\\\\\/3I9vnUXpp7w0jW2ZBAyVMoznSlVYhz3RSxD74WCUPQK1Z9tIWwDd9CIX7yZlFShO66U5bVrYXPg4o0qtlYLJdjiNOHnLJxBRYSDUuVoPdLwac7RgnaoMkCK1daEOoAr9vAIzpnpReSexpM7TSlCoiE6+UBLdDuVq7U54+4oF6K6lZuJxV622JZW88ug9BvaJydM+XdsMmmo9ggLeILfa0Vkhhr6Ihq5hSwaq4P35OGzR1sDYZ9FamMfG0IUqTdWIqpE70mbDDv7Cj2gfL2LfFm1arm+wzDV8H0gRXmI7jLH5FP50rFyDwKKk8EMDD009j9DWcxdBjjkNWeIQtJJVHLE5M+Oh2bO2EssMgXBoNj85NHo\\\\\\\/hJ5vhcYWTWcI0z\\\\\\\/0KzlxjKhfA2+jDbCLGnPg2uO\\\\\\\/lpnzQVsq55qAj5Xo6PYUMftfYFZHz15Gf8pqk6FUcIWzyjac125ik0r3kUo8KNnJFEMLxYEH2rYczkfkmnRJ1wIbjJ1nYaOxsqCXf4X+7i6fSvrAM4pF0GvVXU\\\\\\\/qa0ff2UpTdpXF51GFrXhULsSRFWvEqRQ+MRAUP3Xnn6qC6NnHmJEpVgKqEO3putULQy2iocWBbFQp2FBRQ9V03KQ9V40GPmiFEmbylirJk9JhwTqrSfnZZ4bFYgaF+\\\\\\\/XUFM5vWMzTDy2ayq\\\\\\\/fR0u7SXgbw3gBhE2iEaLJkQD81xwLVHZ9tK5SGEn7oO1IobFMnBeCOLd8MKkfiLRh8xckhHnUFDzIFKvoNb07BLCl4aog7NQbUUDyQd6n010DJLJWFKuakC1iAxBzWj5VS0XYU8RqEGrcyLhj7i5lAJ4usYjqJDa4C3ce9kPR6Xc+nTiJKzcuZBbtzpUisOgqnrml4M4S3tDqgvlBbzAiZV5JpfQlYweXp7iqKPv1qDZbllkbsCR9N6Clte3wl1o01JhLrI3ibslHaKtbquue868qdZ5WvKkfk5HmMAWpzM8flZ5NOk1s96FvOlltSZMny8YQftBFGvGFsarCkf3VGJ9QgZ2zuT8XZVF1toNgwnQkxQy3zJzI67aF\\\\\\\/zHanRyYgxRFCQSU6AgN0FlqqDiMm1FoIU746tISniD5Ce5OSzn926V5UFlDzAux+P6GorTg6kiwHz5WKtBxAw7yQr36w2Fj1rhyxfnXhB3AsNzoKkaCJ9qWFQ7bw\\\\\\\/nuVrGR7nbAs2dDFyM5BtAvxvSag7O03PPFxErirdxz\\\\\\\/rQ00tYzbAkYWiyLrNQibNgOFVf52V1j532uV60WIoygrtcdvHUQOiurGfeB6xRmFooDUFgrCWTITgYgdO\\\\\\\/War\\\\\\\/lVUfihNpE\\\\\\\/8r6754OimzJQtVxJncHJuVFw6MSPwprXBo0n75+U\\\\\\\/i9GwtRlsuLNMw+4mne3WkrvzDO4KRikVJLLWbtao6o9HV93NAXO1tRsni65t2rVN5sXhc0yjt4\\\\\\\/FYTBsiC1aMtfkzKCyv\\\\\\\/VeiBOm2+CsrHhZcSxmAZUNA472JAsh5hDObVox6ZnEJtcGxO7IEY\\\\\\\/nC5OAZNp8m5R\\\\\\\/S89f8hb5HAKpbnHmZjRRFUaBTQQL0kOa0VzuAo25yJXSZXZZmH2a4omrCna\\\\\\\/iBuXpZQK2rS05ru6y66Ymq3ArXdF5hiOK5RjTuPhCqiTIpSi5zAjNZa67Y+9ZAthmrmlnGrpvuesKtaDfgF\\\\\\\/tnwyrjL3PPwahmT9MdET7aMFiSUX91QCWle1a5BxSnUuxXDuBSGo2gL5x+Frkpkrf6hGiOnIdRkBAanMz88MCv+Q8PEOM7ANGckcakF0UKnkUzI6MQSOX9z3EENUJk8A8LhTOEGCwxEmEtP0PjsmzEpt5koumNhNPRviF7JmBk8wAoAvnjws7A2pbH4e\\\\\\\/bRzJqCqhvvSKv91afmXP\\\\\\\/N\\\\\\\/yYdKpksfSb5CWZUtpTRQ2OhbnID2k5vFdQAEnIVpX9my8bUpb1pxvlM5atBAqlx4S+ptJ7H\\\\\\\/iD8Ou0xn637+pabpi3cf2xRL5F\\\\\\\/H48KK\\\\\\\/yLh8mphB9TW9mDgZw6FrEULQjJJpnYo1w\\\\\\\/BcP7F1RXsJCoWvBhMH2nAROBZwPs7tCZL7Wv1jLCdVpQryx3fl\\\\\\\/aIPeWMlmY1wFwdbbcrPoVW7++k+u\\\\\\\/vEo1X0QcHrUEdfkeHuSYJWCCgOqQfTGsDIOWzevGCkd+O8ag+xxzma16mrbfalByeYEnC0vSgX+jeapdHn3dh\\\\\\\/tIpkY8Le06Iq8WT17R4uYrDa0ylBIdKTTp3Bmucl94fGii01skkvZOfCgWffzoDHs+OuL1y7zsmoIqZ9OD1NTGjn+kgjbUUvqSnHx0Vl3EY6zRZaHzlXSEKWndJcGlPs0LZlFV7xZIhJ9jHRdLVOYwG5Nsg+4Tcyirn09LA1Y\\\\\\\/n19O79ZXIksgZJshTlXywDNUIubcZ+7icMIkDoY6Sgp2+bYhuwFphn8OL6yMh+2moUtzfwsuhvEqIIPgj28jaLHoxQCJ5ubX7bf+CaABoBcAiBShAO7zsD5x2Mq48GA8wTJRcC8cfJHBuJ30jCGzRwWhZFlryqe6QbaMDfnZaTZFeslJbPKCi7rhL\\\\\\\/\\\\\\\/N8LQDB1A0G1Kjr8gIxq2W8+LrCczCr+MBVPclvn2qAa4sZD5sTLzBOL2YwXN51nfWEGFnrRzMsLoMJ42mC5N\\\\\\\/x\\\\\\\/xOSX07nkZQWwFs14vsBbxIrhyGxbOa9DIKEgwC9g4HDQmOUFPzOigbY8kCKasqDeenPdSGDbKCaJRX6FT6Kf9z780Gxd\\\\\\\/dK7Bx2p5Mmhkq8N4me67FmRcAKyS5FG6dVkYZJTPKje9QLfQG93l919UkgCs7tfMf5+ur2VuPp6er4YmiePZcyPtiMCz179tGFZR4Bmm7HuDlWOIB3\\\\\\\/hz+L71W\\\\\\\/z8+BN7KYCy1mEHrivHT8HFmPg0BSgaVi+gMc0QGi9H8Yho4OLpaLO2cYQSZtZVon5IW8CYS9A0tWzSs03PkfBDKJALcFeyvmaEAQSsrGkb5vFMd+KsyXgvDLR0th+Ppp0sSCjgz+O0U7o5FlO\\\\\\\/eN+zve3SAehmSncH18ajvKi45UzHrmLEHw4fZrs8Hcv2vO66QdvNfEMpIujWZi454epCoehu3n13LT1dIFDSHm1k+GlvTIYwAuegLWE+b5yzFJijkGHHZcptnklhQVVIjcwP57F\\\\\\\/wC3\\\\\\\/JtOkr6RfYnSYVCxNwIJY0SwEmZtGOBga0K03EpOg\\\\\\\/EIrBCI2NGTOGzQuJKw0FB3c6vNhvrOho7Vx7SdPzF1ww6hpilQmVXIkwZ0vCkfc9F\\\\\\\/MYq1MQqNIjPaytlq37CxWKmcAw8ST396Lvz6hVtz7dWkV43\\\\\\\/HOB1gJ7HOVb7\\\\\\\/DYHFBqfxYeNDvR01+VkeBFQQXqQMDNehniY+ShVp5E7B4ih2FoU98KfJEPpxTO54qPqHjOzLm0p0W\\\\\\\/paPsekNvEBcny8eY0Y22wVMiYgrWIpndtlWLX4+51FFcfvptS5hKdnZq28+CR7S2hJq8+5IlMaxQqKN7bN7Erz6lUElLZb9lCcI7eHd2hyeaGp27gKdmkWYrYFlKdxzQnhS6tyEXT5kkYNoS5gGghd4leyxK4tdprWkL0fFmXH1YbgWmi2KUHzigOz8ILxTkSBlLAp37g79ZetQcyzd5XLsZZJdubUuO+nbmBEMSSnZJlLe9rX0AtXxYs1+zNHNIBE3Gvm8stBnXhnVgEDhs\\\\\\\/DiVQlsHShTbfVjOkCddo9L6RMXAgQCB1L9jK5ZqH8Pfw4tDrvkwzDdlCvrSyF6n6Fh2BsBD1epCCApm2P9vxSS\\\\\\\/5IHT7vCSDGujEFodJzq8ThTdXa6Va+Cn+vDWz0lQKICGIHf\\\\\\\/aQ4MGcUX8hgyRJnLxwAmgk7GqPyeuLqd+heyI2nGtggfZaoBIWZq8mSBSt\\\\\\\/85HDKhvj7UlHhEeILXx2eri+1OSeRVjp7g9gbXl8Zc0W9P33cQp0+fYxfGsj0O43XzD0B7dVXECNKKUtp\\\\\\\/3scBxNetx4AC+uNpESwT0FxL66trj1RCz0oIOnmcYPH4J+W6vF3IrOG+3LCXymygxUKgORno0iX\\\\\\\/BQVW3Nhx+I918YTVfSgCWORooVTjCWH4H3MR8ol8aARr0OVXSnW3fKDZlMrayOzgTHKZGukI4Mhwvr7kvmhmmVDHHuppc6J4O27ghw\\\\\\\/pjNdAJ7Wx5V3xctvYcs8m33bsyms8h8T3W5UUAGRebMfj\\\\\\\/tWcc0as\\\\\\\/JAI3WVcYvu10VKvO1mgxWAvxUUohqy7yR1kkMvQe1\\\\\\\/6eO7riz0FYcUPXoAiQ3XJXJaQqFlIepDQPbBNULs\\\\\\\/fMBSpksUbq6E\\\\\\\/BFG1EJiwhNz\\\\\\\/ubznJK10R1RZhRzc92WR7\\\\\\\/mXn0\\\\\\\/NIFvjs\\\\\\\/2YyPHJOS3+\\\\\\\/tkBVZMidurl2aLBAhMGImXSayoaPmAmfmnLhDqZJG65BHuXmxxtGS9fd+NN7QdNozS4EjVa28pvyoIcbVN2b8nt+CGIP3Btl+Tml\\\\\\\/0dR9LilabwUMbGbRzleO6\\\\\\\/GP+ZrwyDS+xmUqM3M+TL2VH8g1av12DroaqR1qUT9lH4gG02cgS+iXD1cZIXdVghuCH6OxiX7HAU20xGkckrojNy1I8wDnpSC80CHadgeFqyGt1fftEKdbfwuHSzSk9oxdy4wX1UWGqok00Dgz1yFDYOSaF79InWA53zhMQA2XJF1dvpyXNsSRxeWbxz8M2LHGFNbpSFsBWeqeiE1Ug\\\\\\\/g2J\\\\\\\/mgjAnkxixARQwRIsKpgybmnbXUDiDYLPMaJb6+xSZCAsvsWRmqTYgnmoITzl7+WQxigukhU5Zw0z3tNrfovsQ6Xnz8xz3TenbbPgDGBOFLf3BN\\\\\\\/6txtD8SDZ\\\\\\\/HmM8Ah3oZExIC1uUP91hy6AUD3mKWl8H9hwZ\\\\\\\/X3UjtO3i8QiqXC4TOsXF4QrsdmzShgXBKG+4ODinl1z9Tlyhs15E4nRfrqNJmOv2dyfDsjACWepCcbyjNCMJ6LZHPlYcCcPL\\\\\\\/dgPpNjRKidFI2fOhl8NyT6aAd9s25djCLlLUyvVWJdr3SE6aAew9yJLKC47vcKBUymXh6GKGH\\\\\\\/bMh8uGEigTnQvcU7dImnkfQFkOog1IqHIoFiP3lePN8SSUvI\\\\\\\/oUqVD0kRSx\\\\\\\/63U6LmWbFO3MAlwZ6+di84HMxye9u8cgf5f11GSuaUr7p+BX1rM5nF38x86AUWekPk7wK35gSFvG6Pv1JjbctvXy0Lx1hbPkMrOM\\\\\\\/AFv9IxLs1MwAoJ+q6lYbC8fvpbMzHLIXhvOPjM3FOgxMXpKNVML8t19IYlAj2VWOf6M88ZWzgA+Eek1iUCKjhDQtNi75amafeREcvUsrx01iGIx+Fk5kFnhqts1\\\\\\\/l64utZhQZ4+SiJf+\\\\\\\/bRTVlcZNg7byV39j3EoWd8k8UcfC51ypoTv8ekmU95253+tX16a1oGlX5+es3p7Zpoiem\\\\\\\/UMY\\\\\\\/v1nG2CB0TPvuYV4iwpDRHh4vbWqzYGexeS+cN7pGJZkYOm0UCjPpwpm\\\\\\\/TqdCSe3DXzneEzLv8KOKa5r6bZtAkYRGRLtTbReR+6LVpptnEFyLqrro86kMvyobYdC7B0P\\\\\\\/J+6mHfYw52gVCgz61GH10LLuof4HBAIISgm23DBqxPgDp1sm3x4K+iYUyLTpPexcKqTTrqDF0zE8olthhH\\\\\\\/eMY9SSg8\\\\\\\/ERMhSMh3UD7rfGz0r5Duz\\\\\\\/K7N0rHpHfV5MTYmLA1f8Iv+8xbYw3rYe4N63uv8grJv7Zr1Owl8vv312lwPqttYw9B93hx7ad0aJa05URX4RRE53MBr\\\\\\\/\\\\\\\/GsFbdXN7YArtS9JL\\\\\\\/fBMt2fIbGc2EhpSKTTBYiidi5DBCTSEHSIfGJhSLDfq1pL2sB8ChzVv+Se1JfMBvHX3RSBQWkEjg88SXDkn5Q04WBnhlyRFNuNd+MAlOPay0T4qMAqqWM4ZpmH2dFom4zN2Dgrwz\\\\\\\/u3gPZO6V11Om5DjbEEX\\\\\\\/AU+uh\\\\\\\/Euz4XmPv4wuu2CJvmF1hXZzBrKdvhh+Ct0HC\\\\\\\/\\\\\\\/M7eLCbIWMZqxe0wAZzXnRJ2yMMBO4gUnqny\\\\\\\/YfAX+LBL6Ig5QcgvBHxsJNZlJxjAcCw\\\\\\\/f01sgkvvLrX9vnaMmJut2C31Tno+qwiqIrDStbgnANacPis4PRc4Oz9mx\\\\\\\/TAMNrRFwznumJ8H7TA+2X\\\\\\\/X+N21NvrIhNnLE7IPaA2IOYxvK\\\\\\\/1CwXolefe5KHAo4X1tz4Gh3tljF5qnzIquAoi72OlJ8hNGP2P\\\\\\\/6okiOaE6k9tUooAFM8f2ZPMmsrrWpHttzd8jFYhvi+OAo\\\\\\\/d3uoHpVwL25rnFZ5BN9HW3J5QXDUCOV4aLd9NumFF8QRIL7faFAuXAd4bRRUOFRCUPjEZv0XscHfsV5clljyzpRLcBifqIA7gwVRJRizAgHAxO5NPXQdlrDCe1nIEMNT3Sz2AGmKJzmkHeNxQdTaelp2kvx9tBewMOVbGtlip6rVaA4zph7FRhzDVWWxG8qxltCO0rNEq4XdUuGKyp00VcC40mXxSMAQg34uJbh7INIh\\\\\\\/cPECxDSvfj5ehJicq4KB2c9OsJ3PeXoNG3Y2eZ8VKAVkeBGxCTkUcFztIdxCyH3D9kmLuJ26utXi5sghrYuHmLOzeNqmoWtyPrZtXCKcC2HIp6\\\\\\\/Q6KKvfp0dTPnrgs58vu8C1ldaiTR5YGKXyQUW2bLBrEWAQRWotcSSeV6\\\\\\\/lJqfaTNhCk3kW5RYDNlLirIEqpS\\\\\\\/OPBO0MDK4J\\\\\\\/Q6L99XP93ickuPNG1r\\\\\\\/GZGOsYBIaGq5n\\\\\\\/Y9ZljL6qEeBEy2xEdEasGLwx8mhHm0jp7WttxtHrsf5ts4VGmuO7mCJMCctCw6jbx54fwVO2hP\\\\\\\/nflNTgggQCBd77zBRlacYwjV8cBLVUDORT89QVTDWK+bDf\\\\\\\/QTGz7kHmPZMD1dxWizvs3+h05y+aTPkzv37ZtgSr8EM63zfoq4PAfG9mAsM8\\\\\\\/GaTbYh19bsjKXGj85q1JPpDs\\\\\\\/zWczFwc34Qy9sTs5Z9W+RVd\\\\\\\/TCvFfMQqFaT6JrIgL29ybjwhFzvuQtoo5wiG0jok4h1a8fCufvZskSL+unEyGvR\\\\\\\/1Ofy57dkIvSoVb3AbOa04thiyqfzftby+1Wlzl1\\\\\\\/KaxNC5x47nbfFky66i7Obb7DqX33l1bYG1pgOzxA\\\\\\\/i5KSXv6sUcv6yXQZ3f4cOx23DueOjkwqjIm1aDRVvNnqQqX3wQm+TfUar2TaYslfFfr+ygFH3wQ\\\\\\\/0lGOR5NpIGTGopXY4sd8g8cwDQ1kvjGXa0HhMbbZjtefY+ITCG9PMl7g6LvQMqJSobsqzpombkjnSKp+ZQlCMghYhBk4R+Ki9rQ99QuzxWfejtgGM4+Yaz4S\\\\\\\/pD12yVWT64TUYlCVj+fJ2uXuZfUP13mwHJ0Jt363bkOkBQOAFHQpA0YgUkLfXy8kxhwerjejVJVxPFfu6B8TRcoGpkestYSISE\\\\\\\/qIC4KTkoFHtcwYVkY3UTaIZyCvyO86uKXiZrTr3SHljU\\\\\\\/stf9uX5hk\\\\\\\/Xp+szenlyM0LB\\\\\\\/CbpeiWDqyrJz1EXsyYBs0eShBusmBNMetO5eddrwsbJ3DqhqomL\\\\\\\/CwLG7Vc9vG9UgLO+ZHIlWiPBTrN6YzrYrCPqpYnvsBC3tIZkySQ4wN\\\\\\\/Qvbp5b9ZdQfeoWwlZLsYajjcGfpfBC6jWJLdelVSvnIBi7MN9NhM94zjAYFVnreZE0noRV7sVzTeS7lUTZVcKJEAEACuT5Y9LjtEqS4O\\\\\\\/Su12G99V3RZ84sz6LgVo0o4WYaVycBKYJulJe6IjAnJsP8CfQNm6Lkk9Egv\\\\\\\/KczQ\\\\\\\/3qQIJAV2WuonW5htCkJzESliwZkA7EtL+zgd1GvXqcrv4IqQ3iD6vIpKnCXanpzvl6V8XBEzY2T5JGI+Vum9+3XsPD05S7kax4TFURQhGCxoFHBrTzmFSglROuyyAXVqtJyis\\\\\\\/2Av+zq2UmEQEbhSi1Gi8XZ6iWpWJvvNtkgdfNhaE\\\\\\\/evI4teAf144KqtmqfkoscusAtdLaj9V517tAkNkDNz9Vef\\\\\\\/ltE9xVsmGfj3wCA1qJJq4lAp1aQLjJpv3QlwTfqG8s2mwP7xBQGneq+gkO7sNNp3vW\\\\\\\/GhgyA59lCBVmxSAp1OMrhf7vUFG1RVH4LlRxrKQxcjUJ1VFE+m0epALLqaohlqnoA\\\\\\\/xYepW4+bNQCa8\\\\\\\/UeeWuq\\\\\\\/2HDTqN+PY9IFzE\\\\\\\/wdXKPyELyz7lOZMSbKSw0cG6v0BVhe+bky3FLjMccqgNuKnFbH6PvhNG5q+z0+M+tsBymAYCRrbzHSMpbppa8JvJhPxaToC6ljjpofM1ard9h0hGOhCoPiCwpWYZItcbKdiZ4112MHLTO5bZUwPVSY9RCzVMkVriayM\\\\\\\/myu8hwdsBW\\\\\\\/ZsGUkUDmv3n8ZQrKLRyB4678XEZlDl80e2HT\\\\\\\/bwWih4zl1emM3CtA9JCumOXrRWK5pCL2wHNi5UovMWiyUR+fbLU4IS+sGe7zWJt5G+dnNM+7TE3KWHp8KFFxqQcdW33kgIqIwUni0jNkkgcbT9i80Eu\\\\\\\/eR+1XQ1+BPSfishXZgPCbDmkr6JUeeIiu27kFDnrTEMh+HPicjIRtuBS7Q\\\\\\\/cOAcXl8yF9gOwD55dPzthh41eYWJD5JQZ3ZHs57gWKxgc3hZqggINC1I9tMo1kylJZi2x0POe2nTYmhf8MWdVoxx9UWrSLlRW05pcYjulM5Ur7Kj80J7F17VqkayobjzW9zKZfCmt+7OvP2OrWnhnoacbhx3MXvZSHptm5YNauf4r2uZb3B9ifHPVWDIwdt4wy7Jz0L3qR\\\\\\\/4thcMkcC3alcTlX6SyJFNeewjmASOuR3Orq9eFYAzfGWDValenqVEAhft9aVae6S\\\\\\\/paDF4GMSbLC+qCOjUkWZDLK+YX00Ra6iml8ZeZpP9GK8dYjULUjep+ba8M6yIJacQAmRXl5h8Fw0gKEK\\\\\\\/GasNrUO\\\\\\\/Yu3BTd9mbwZ8uFSDkE8oD\\\\\\\/7QmvE3y5jgW8APQvfFnPaat68dFxHobi2PHF4GS2NJg8CxDOt+Yl6ESfV3zXXLx58bXefj0ma8tNJwNJ9W7MTk2RlbFIj+qUd6cNotuRrLvvlGjjGCprO6JW0XyWRUC4N9blIh8bF+Jv2jwrUTjFkcE3wgr+Sq7mhb9\\\\\\\/MkTEyoW8+FAe5rSxUQvW9AXnWA2k\\\\\\\/j8kSuxMrhW5feP97mhpIUBHiHJ8AO+lQxzQR4eXedwP39oPFZK2tN2X0HGjuOoVmPnFCa0AQSZFppmOjuodOK2vwZPOekDjHgdq34TkB\\\\\\\/rUXYC7oNLgIlaBVL\\\\\\\/WjyLY56o3OR+sA7ChVk5rRMdwYG+u3j54+AA3j69m\\\\\\\/j3a3QhpgAjg6RTVUCsx44LkviChpxRQdQAnF51shrXiUVDDWInr68DlQDc3upo0IDVADwpGszMGqzK1ANe59NAaFGeHr1zY6XW2Z1y0VxnuA\\\\\\\/CaoMnxJT72TfkyI\\\\\\\/q7FE1Qi8CW3HpVFD6nztBD7eztUvl2tYpyI1ZFdnTmFBt\\\\\\\/VRqkogYmMqvwaXX2UTERbQ+9YBQDnbpA5IBUdC1kdvr355RcNbeWPMCkF8i0VN8SEAA3ITrEsPjDfQPxF07I+tNSp27qCLIO+Wlpup1ZjihQYXpp7CEwX6jYV69rLoq9+PPSZ3CslUrn1UcfIyiXMfz4F2l2uP5s\\\\\\\/x9Vxw\\\\\\\/tsY\\\\\\\/HeUpWN7uPMfwkDpyC39RfimobB\\\\\\\/kCrBXGnZ4cP35oiMFMYNRzLeOmFR7e2wbIF5VQp5J8nNOqkqxbEnFGoNDDtXQPp3IXO8aKRD3Xpc9BLPS2b615qpFJ87AevKZHaqdGUBZpQseNwQ\\\\\\\/hC22xqmQxUCR\\\\\\\/Wg0lTZZRCJKcvVmhZ9bJLkjEoZQrHJd53T15oyNvJ0UJIPw6r5Spnfxq6mVjkY2fVwe7JLRdxpYqMmfgcFzUwuy7QboHAi\\\\\\\/0Bl4vJEWbfqPAw1ERN8e959xcQziH0Z\\\\\\\/KxQVYfLah\\\\\\\/9Kes+1vV3GCx1PLI18UpJFda\\\\\\\/CSQztcYAe4cCLteLcznZrF0sfM7Ov\\\\\\\/Y7YEOiiVm\\\\\\\/z1u3XDjbO+9e1MRmf\\\\\\\/1lZ6Xr3HgVmdBu\\\\\\\/QuxHE8FUM6n71WaxGJiz\\\\\\\/UwWBMBN6bNpTI0+45fRuo7is21GvatwZ+fg6yNvtdS0qhiQS9XahZWnmiZKtXC2PHd+buB2LGyiIn35X7soI\\\\\\\/DjZxqPvlNPWHl78O8w0LdNCHqiu+kS9LMJS\\\\\\\/tqlYSJVLysYw6EJ+5TU1c8YkVB60O1xIhcsPGFZpsuZdxf66PgNb3jOtTpOvr9+vHUfQ2Tx5uMYFtXSTKO\\\\\\\/1SalQOW2beLhsNerAwHR4PTpHlMIedlpkL3b3pYqvA+1PlxbTBkHpPfhssd2LRlp04snBtrIsJmkEiLRuIVveUX89qrDL\\\\\\\/YY2wD\\\\\\\/9RPSjjv2AIhbsZPsziI7xXDnARi70+E5AgqPtN06RnyKMcHKhPpNiSZH5u30JUBhtASq3ZrJudti3BOvVTQl4ACKA73l5hhyYQb0\\\\\\\/eUxoABiySBj3kGOiPguUfhFAh7LMbvmoJaOVUVkehmZyEmhuJylhV8zlgVx2fviBth9YaUiIfC+0tmEvXUfHaBEDqsDsHTD60rQ7vfmxuD0YIvVOckgbZUwEM5TtbIjdZfebsXbvB5fy5es5hyyjcy0Dm6j51ywwMqAB9l5e+w6st32XrlCKyAXrKdRE8QBok\\\\\\\/AmHI7JS8jDdmRD\\\\\\\/F9uQ5M45pVsewSP1wGXDviWp4FlyAD3nNc6K5KtAboCw8lyyvvjjbr41kV41BJGirEWEgs2JcO6C9fsnIOnXXVMgUoTV6cZzqUi7hF9jBeAZyIK5ns3FYh69la8qDWSbdN+np2kzIeBB22x8bCk2\\\\\\\/2\\\\\\\/yzpFUJSxR6kZqyccTJctdvY1Cw6R15jeh2cZ24C028MoKra1oxlw6M6NF\\\\\\\/FcfcsQDrtQNX5e\\\\\\\/5eBxnAt36S0drRvrrlvxz8AwtVeaA1DRqqIn7b4Fd1XjgQv9+Ul5\\\\\\\/xoE5C4Kb+YW0+x3xJC63ksSsTD6kzdR7ZOWXka5QEkKrNZAv7RRu3Aml+YbsKtylg0YOirGGtOKclecXX5zCUbMKSEKvm1D0BjgbJCUHoy9JD3lAdZ5Ys3kVZB5BJtVdWLbEqK4gNSGHCJEK6+DGGyd\\\\\\\/lOdqQt7wvjXu91mFwMRaR1TxSB+NeZRiFwMeR0cfHliEWf+LbB+\\\\\\\/w5wHFa9fpFw4GYth3ilrfMgvNdVNtZ5nCnOIfFKOdfW\\\\\\\/rxPMKl+cjG4Qfj9vm9+QiDTjNR8e2SOD925zfGoShv3mBISkjCRi3JDSRTHkU5YgA5UaVm+R3rf7FnP3V2IHPEqQXBj4pWDDyBa2mfzr0RU0\\\\\\\/cx1NWCRQy9uNpH0tJbHh6bgNY18TE\\\\\\\/\\\\\\\/g5+3ggNh7yQgbGR8l455Ap5k9gc3PjIFavx2Wt4hOGvRGO2AzXNpNYQS2mPPS13BkOsXROS5obuDIKbFAo+7y03CaB0D3qcsaDd4HHRIoLKRo11hd\\\\\\\/EncVhc65ASpYVH6PCtq+AhxutDJ0y0zSJVXJXY\\\\\\\/S+xz94YUwWR0l\\\\\\\/kgqfXY95MVMoM3Yf56IPfEMK1gPI7k1edsbkPEOGWknryCqtEZJUaIiMq+xyR+NuuwRTthGzPVOvnMZPn\\\\\\\/2sBkg6MVyY70Z3kBa91eeQ7X8ORiGjhVf7pc5MMOlWd8DIGgA40z\\\\\\\/VFGOyaq85g9fNOZhe0qAdFhFQw4swQ00xzKqCAAydxftygteQH7ThX9jpYYw8REOcxTWaZWIO4OYw3bjjEq9B8O4p3xdIfVbPB5Zw7RFX4tv6KJ\\\\\\\/eMAB0yyut2DKM9V0MnsHIH47Bu9lYyHBWYkIvsZJcScyAs6J1SY7N137ey6j3GrmV4RztwI5qjmBL7jMzd6J18UJ6zK7txkHAAzu8aZ0psJgdutz7ECyGIJtMnBEo6jSEN7kPvVkEA1o\\\\\\\/DcBYFqIeZpVzjRO1YQl0pNGF1PsBk5mCPpX1MI9bfv8vTezSGsvv1eqscfMAU6bWs2QGAVZxk\\\\\\\/KX7Nh6RB+7z+aFtCM8parKcxgYwEmdkd728b0o3TWFQE4oizGxhY8aL0L0RizsGQUvc0SsdaoEfVO5yzvyorqFw51MGwfdso+YKWVxztszThE1LGZGu4Tql6e79dnxNnbK1ByiKxscftewPiUh\\\\\\\/q1sS\\\\\\\/qKSAfe+L7+zOXgiFZ\\\\\\\/fU7zZ7OHDjysNghL+c53k1OtVSmpC\\\\\\\/759azDiHde5kyBKKvHwF32uafuqtfJACW\\\\\\\/p15KI\\\\\\\/7YI9g+TJOpcQcDLxpIt2nKrwKadpT+0xEnE\\\\\\\/96Brr3MG3lNah5JGPGO0KSfKiwEJmXMIAY3KCe3g8aKdaUSXkQmB9lkDJQ4hZcfn+d0x+WxW31ir2gwC0UmWEppdWu927sy7FuqKmUwbJHDbrnMeV0yAlDdm5+GablKYuvsTnbqdb83K2bUaqvOA9tNDRWdCyqBWzbPK5ClgmQQkyerldGZy2YSN6W8xgE5z73qPh2iZDUm0iYAvhqBwWf4qSrYCq2morO8\\\\\\\/n4NqwNFKSDm6it8sPSSLUWDkT\\\\\\\/pLYAvZfsUPwJNspYWuJ3456qmXeSi9oGYbpgeGST326eSQxK+wk27uI8RyQaY0HNYSx3uVCtHoxqgKbSejEDZ0O10R\\\\\\\/S233caxHX9kUcJXL6MUTzJDEIir4CZVSnJV1PF6Mv1guLdEbl83\\\\\\\/mQZmRIjKD7Xg4pJVq9yf7wZkjiCBBdDMg3gHdv1euVwvw2D+h0\\\\\\\/YCI+EFOIS1jCSi34UZZoHwIYxcrX9JtiqiYZ3m+\\\\\\\/b6FhxWGLsKmwIWiXElJIPVOfCKevls6pJ6+oe8Pvn\\\\\\\/70w9rnLhaXShhsQ2YIwLpNBbN4bhSK3KaOMzQQQX8Uy9+OSTuIhO\\\\\\\/Y+qeOKowuTPT2K47cnhfxN+RadSeZkSK49cinjfzma68E9uU0eDLOukj0W\\\\\\\/Qu4IQXT8mBBLizf+pSgJ\\\\\\\/hZ6UuMnHB\\\\\\\/77JCaK8fhsv0cWSaq7S95pHzzzzA7HC28fzLY83FBE2pZS6EuuFMgpk8F0uuOBSpD9\\\\\\\/v3IDVQVMQIqASsXE4uarBWAeXnr7zYtUPW7LFLKPUz+JBO27nIfGEJZzNj3jBNafgu4htG92uJLaqWdkLXyKxNnUni\\\\\\\/WpIse9hxI3u6zUOMsFA0JT9H4kJA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022aaVq8jq2OKD\\\\\\\/v+bP+k4kZg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e7294d4fe85223a6b7b3ecebe1ef0d4017579342d185aaeede56178e0bedf332\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022woGIpLo5oTB1JVHS\\\\\\\/JfxgvyyVVeKL3BObqtyEvsvI4EqRO\\\\\\\/OZdnZarWdYDKiZ\\\\\\\/Utm7DGwDeLkaxwyQFe\\\\\\\/30XOJIq64Cg43guv\\\\\\\/d7S3ZZs8NmKLctrVvUUVk5SIzLM\\\\\\\/2AwsQb5m0PhkQMN7sn7JZDkXuab1lScUPtNcE7yRhqxIumCvZBKCZfdn8l6QHY+j+Km1q+S67hLC9UBiC8KPfN3SZOu714csHuhC05xB++L8D+Jm2og16Xl+e\\\\\\\/KA\\\\\\\/Vq6dzheY8TdrbA7gJ4xajTtYXl9tASPbfa20c\\\\\\\/lh7cg2E2STb00c9PSdCeuDRWHsMJcX2vxJLnx1VRw7OzNhhBZ9jicicT9Gdrp2ucsmL49+yaqVB1UFYy78HcJtUH\\\\\\\/RgEk7zE+xuu23UNbQrDJZgsUUyRptvoIPizJtmMWuDfx4XpJX+nzpuYFEqViWTlRLP436PWg6F6m2ezT04LmEDjm7eyTNLdDUxG4eOsEFb5GZSNRt35GUnYWfOxIYwXD+BtoeFg1z7gs\\\\\\\/3CgtzLuko22jaqF1XMc+c4KC28tN7lR2FHpcOFD7QvpASxA4MKdPpoM3iisIbzEaG7TrIse4VdGou0OvbKe1Sm9kwlav8+3NDTM5nkl3PSZdxaZ1de7hPMg\\\\\\\/Tx+j7NA1sM9wh3NIkNqZl0VJJ87GRow6q+gfQ9bHcvgZq1dPd9TEP0kQucY2zIfwZbtKPmZNtUuK\\\\\\\/pJ3DRdL190W4DNsC5ulAckP9DzD4wujmYMNP5sb82nKEmmbUk1bCtR4VNXx+zFg8WPfi3r5a84RJEC6j+Rrzm4l4uKFO\\\\\\\/MYpKEcVmUHeE7Ip3\\\\\\\/4DvYz2hOczscD0+0VCFuacNz\\\\\\\/h0SpYqyWn8NKN0\\\\\\\/zsq+I8Z2TK6WWal\\\\\\\/PzD93Q4JZp3tbe8rtWGKLR4o6gKWe+SKoYoRNWFfj8Mnd\\\\\\\/uUDZE\\\\\\\/QdDaiUrWHZQHpvferhEGcwx0es56GgPYI3dHk+qBWun0qiyNfzfCMMQx1xopphIeeBsSIYv8ySPkPHayZsE8sEL6V26Dbe3RvNaz7umwLxOw74HNzrzo8fp7fhSELg7sZ8G5p0X+C1CFgapgowa2A3CqWIyu7CLryB3BoVnOQjkGddRxoEhHljYxk\\\\\\\/\\\\\\\/0fQhCrKNZbizUIxex+lSMci50Y1Rpw+Yd76yixknCEOUQIBBWP4oP9KrKnOzkjujhhGtRr3dO2TvUnmvNXLueM9mC+OB5fonBNCTy1K0sGvB6dallaMwWUEGtZS0jf3vViwXqpTW18SaqT2uTsPEyzvdrarnaeeBbk6S1BKZ+x6JIOSmcE06s1wUKk1USu2ysils\\\\\\\/MUPyLD9OaSYTh0Z1fOHqmrXn6h7pUazRF4fz3ejKD+NztBquHho3MTQl5+K1UaL3kTbQZUJ+ZMKwvluhvFI3AccpN4re9AlvNRP679eEbJcZdDBPDaYX2sYwuQYtpT74\\\\\\\/JPu7wE1dqVOVXFEX2lnlI9zIGwaCnRvf+MA1wo\\\\\\\/5D8L5Xq+Ki4wO8MP9LelSi\\\\\\\/U6343ZsCBU+8zsJdaUdWzt1+gvBZ0WH40Bh+lzu1RYSReaHqS1Lj8lsG2IHEUWV3BjqrnVSPzrwZSU6oNPEmD+GvNMBSFVXmOAywHIbJVY\\\\\\\/eTbF4gclnsrFxWA3dVGpXHigK4c7Xyo9nZg+9YoaR9bR4ZWloB0eYh3c7o4Uu4yd2YmdZ+EX8vGqG\\\\\\\/LKdJvwx04CRW69P6X1DzJ++yBf9dsDpfQCnOqEBI+ymQOEU9sWjZhayvcm\\\\\\\/xATV0k+xATu+x34XUNpq48LD53lv8I5hPMCSR3DazqnDIZrRcBp8CwmSAFyuMiW02URZLuX1bKDIVjqJMgXSJfZa5xV81\\\\\\\/spxbjbm+mtA0DUuw7dKVhj2jfQ6IMs99v3jpN+MWw5sh4MgxgtNoGI6iQDNV8EZ8jVUBkJ2NJMGRvbdIlzcwE10v+ZWStKpOKfEpIOqTQGonV9nmPAFYrAN4rYG62X0rlhfQUrrCPJLqPHOKtu4KmKvIUwTILcQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022YCkH\\\\\\\/7ZusWnWHaDmYGMVYw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d84829c3df948bd813bb6a7d5b6d0ff93efe19b9f8b2b9e10a6f8acc2b9d6e67\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487467 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/12/3983" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-e3Fl1A77zlRg9psr-rlpz4eJVX-1ooaTscPPZLHYtzI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/12/3983" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qJZS0o_E51OFmcZMaw76bQuFZLuFi8H4fcJmrsNqzuA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/12/3971" title="Repression of Middle Sporulation Genes in Saccharomyces cerevisiae by the Sum1-Rfm1-Hst1 Complex Is Maintained by Set1 and H3K4 Methylation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/12/3999" title="Understanding microRNA Regulation Involved in the Metamorphosis of the Veined Rapa Whelk (Rapana venosa)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487467" id="node487467" data-pisa="ggg;7/12/3983" data-pisa-master="ggg;g3.117.300152" data-apath="/ggg/7/12/3983.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Heterogeneous Patterns of Genetic Diversity and Differentiation in European and Siberian Chiffchaff (<em>Phylloscopus collybita abietinus/P. tristis</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Venkat Talla</span>, <span class="highwire-citation-author" data-delta="1">Faheema Kalsoom</span>, <span class="highwire-citation-author" data-delta="2">Daria Shipilina</span>, <span class="highwire-citation-author" data-delta="3">Irina Marova</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-0961-8427" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Niclas Backström</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3983-3998;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300152 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487467" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Venkat Talla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala University, 752 36, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Venkat%2BTalla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Talla%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVenkat%2BTalla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Faheema Kalsoom </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala University, 752 36, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Faheema%2BKalsoom%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kalsoom%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFaheema%2BKalsoom%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daria Shipilina </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Vertebrate Zoology, Lomonosov Moscow State University, 119991, Russia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daria%2BShipilina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shipilina%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaria%2BShipilina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Irina Marova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Vertebrate Zoology, Lomonosov Moscow State University, 119991, Russia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Irina%2BMarova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marova%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIrina%2BMarova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Niclas Backström </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala University, 752 36, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Niclas%2BBackstr%C3%B6m%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Backstr%C3%B6m%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANiclas%2BBackstr%25C3%25B6m%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0961-8427" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Niclas Backström</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:niclas.backstrom@ebc.uu.se" class="" data-icon-position="" data-hide-link-title="0">niclas.backstrom@ebc.uu.se</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/12/3983" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487467" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487467/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/12/3983.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487467" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487467/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/12/3983.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487467" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487467/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/12/3983.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487467" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487467/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/12/3983.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487467" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487467/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Identification of candidate genes for trait variation in diverging lineages and characterization of mechanistic underpinnings of genome differentiation are key steps toward understanding the processes underlying the formation of new species. Hybrid zones provide a valuable resource for such investigations, since they allow us to study how genomes evolve as species exchange genetic material and to associate particular genetic regions with phenotypic traits of interest. Here, we use whole-genome resequencing of both allopatric and hybridizing populations of the European (<em>Phylloscopus collybita abietinus</em>) and the Siberian chiffchaff (<em>P. tristis</em>)—two recently diverged species which differ in morphology, plumage, song, habitat, and migration—to quantify the regional variation in genome-wide genetic diversity and differentiation, and to identify candidate regions for trait variation. We find that the levels of diversity, differentiation, and divergence are highly heterogeneous, with significantly reduced global differentiation, and more pronounced differentiation peaks in sympatry than in allopatry. This pattern is consistent with regional differences in effective population size and recurrent background selection or selective sweeps reducing the genetic diversity in specific regions prior to lineage divergence, but the data also suggest that postdivergence selection has resulted in increased differentiation and fixed differences in specific regions. We find that hybridization and backcrossing is common in sympatry, and that phenotype is a poor predictor of the genomic composition of sympatric birds. The combination of a differentiation scan approach with identification of fixed differences pinpoint a handful of candidate regions that might be important for trait variation between the two species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/chiffchaff" class="hw-term hw-article-keyword hw-article-keyword-chiffchaff" rel="nofollow">chiffchaff</a></li><li class="kwd"><a href="/keyword/speciation" class="hw-term hw-article-keyword hw-article-keyword-speciation" rel="nofollow">speciation</a></li><li class="kwd"><a href="/keyword/genome-scan-0" class="hw-term hw-article-keyword hw-article-keyword-genome-scan" rel="nofollow">genome-scan</a></li><li class="kwd"><a href="/keyword/divergence-islands" class="hw-term hw-article-keyword hw-article-keyword-divergence-islands" rel="nofollow">divergence islands</a></li><li class="kwd"><a href="/keyword/z-chromosome" class="hw-term hw-article-keyword hw-article-keyword-z-chromosome" rel="nofollow">Z-chromosome</a></li><li class="kwd"><a href="/keyword/autosomes" class="hw-term hw-article-keyword hw-article-keyword-autosomes" rel="nofollow">autosomes</a></li></ul><p id="p-3">Characterization of the genetic basis of reproductive isolation is essential for understanding the speciation process, and identification of candidate genes that may underlie trait variation related to reproductive barriers is a key step in this process (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Feder <em>et al.</em> 2017</a>; <a id="xref-ref-127-1" class="xref-bibr" href="#ref-127">Westram and Ravinet 2017</a>). Until recently, speciation genetic research revolved predominantly around identification of genes that lie behind intrinsic postzygotic incompatibilities between deeply divergent lineages in the laboratory (<a id="xref-ref-130-1" class="xref-bibr" href="#ref-130">Wolf <em>et al.</em> 2010</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Coyne and Orr 2004</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Noor and Feder 2006</a>). However, recent developments in DNA sequencing technology and genome analysis have paved the way for detailed investigations of the genetic underpinnings of reproductive isolation also in species that previously have lacked genomic tools (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Baird 2017</a>). These technological advancements hold the promise of contributing to the discovery of genetic components underlying traits that contribute to species divergence in natural settings, both at early and late stages of the divergence process (<a id="xref-ref-112-1" class="xref-bibr" href="#ref-112">Storz 2005</a>; <a id="xref-ref-129-1" class="xref-bibr" href="#ref-129">Wolf and Ellegren 2017</a>; <a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">Seehausen <em>et al.</em> 2014</a>). When lineages diverge, they may evolve allele combinations that are coadapted and lineage specific (<a id="xref-ref-131-1" class="xref-bibr" href="#ref-131">Wu and Ting 2004</a>). As a consequence, genetic incompatibilities may arise in hybrids (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bateson 1909</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Dobzhansky 1940</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Muller 1942</a>) when divergent lineages meet and interbreed (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Coyne and Orr 2004</a>; <a id="xref-ref-116-1" class="xref-bibr" href="#ref-116">Turelli and Orr 2000</a>). The effect may be seen in first generation hybrids, but should be particularly pronounced in later stage backcrosses (<em>e.g.</em>, <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Qvarnström <em>et al.</em> 2010</a>), and can lead to reinforcement of the speciation progression via selection against interspecific pairings and backcrosses (<em>e.g.</em>, <a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Sætre <em>et al.</em> 1997</a>; <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Sætre and Sæther 2010</a>). Hybrid zones, regions where distribution ranges of nascent species coincide and interspecific crosses occur, therefore provide an ultimate study system for identification of genetic elements underlying initialization and establishment of barriers to gene flow in natural settings (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Jiggins <em>et al.</em> 1996</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barton and Hewitt 1989</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abbott <em>et al.</em> 2016</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Payseur and Rieseberg 2016</a>).</p><p id="p-4">Of late, there has been gradually increasing interest in understanding how adaptation to the local environment can drive the formation of reproductive barriers between diverging populations (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Nosil and Feder 2012</a>). This has been termed “ecological speciation,” and refers to a scenario where reproductive isolation evolves as a side-effect of adaptation to the environment (<a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Schluter 2009</a>). Indirect support for ecological speciation predominantly comes from experimental evolution investigations in captive populations of micro-organisms (<em>e.g.</em>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Buckling <em>et al.</em> 2009</a>), but also from a handful of studies in free-ranging natural populations (<em>e.g.</em>, <a id="xref-ref-120-1" class="xref-bibr" href="#ref-120">Via 2009</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bernatchez <em>et al.</em> 2010</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Chung <em>et al.</em> 2014</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Gompert <em>et al.</em> 2014</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Egan <em>et al.</em> 2015</a>). At early stages of population divergence, it is expected that genetic loci governing traits that affect adaptation will be more differentiated than the genomic average (<a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Nosil and Feder 2012</a>; <a id="xref-ref-103-2" class="xref-bibr" href="#ref-103">Seehausen <em>et al.</em> 2014</a>; <a id="xref-ref-129-2" class="xref-bibr" href="#ref-129">Wolf and Ellegren 2017</a>). Theoretically, this allows for using genome scans for diversifying selection in incipient species or differentiated populations to identify genes that might be, or have been, involved in the speciation process (<a id="xref-ref-130-2" class="xref-bibr" href="#ref-130">Wolf <em>et al.</em> 2010</a>; <a id="xref-ref-103-3" class="xref-bibr" href="#ref-103">Seehausen <em>et al.</em> 2014</a>; <a id="xref-ref-129-3" class="xref-bibr" href="#ref-129">Wolf and Ellegren 2017</a>). Importantly, selection may also reduce the interspecific recombination rate in regions that are in physical linkage with the selected locus (<a id="xref-ref-121-1" class="xref-bibr" href="#ref-121">Via and West 2008</a>). The size of such regions may vary, and depends primarily on the relative strength of selection and the local rate of recombination (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Charlesworth <em>et al.</em> 1997</a>); in some cases, large genomic regions of chromosomes can show this pattern (<a id="xref-ref-121-2" class="xref-bibr" href="#ref-121">Via and West 2008</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Feder and Nosil 2010</a>; <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Rogers and Bernatchez 2007</a>); in others, a very restricted portion or a single gene may underlie both ecological divergence and reproductive isolation (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Chung <em>et al.</em> 2014</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Joron <em>et al.</em> 2006</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Martin <em>et al.</em> 2013</a>). Hence, by applying a combination of high-throughput DNA sequencing techniques with population genomic methods to identify loci that deviate from the expectations from neutrality, it is now a realistic goal to perform genome-scans for genes involved in adaptation, and to try to identify a potential link to speciation also in nonmodel species (<a id="xref-ref-112-2" class="xref-bibr" href="#ref-112">Storz 2005</a>).</p><p id="p-5">Recent genome-scan approaches using population samples from a range of taxa at different levels of divergence have shown that a common pattern encountered is a mosaic of highly and lowly differentiated genomic regions (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ellegren <em>et al.</em> 2012</a>; <a id="xref-ref-117-1" class="xref-bibr" href="#ref-117">Turner <em>et al.</em> 2005</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Martin <em>et al.</em> 2013</a>; <a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Renaut <em>et al.</em> 2013</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ferchaud and Hansen 2016</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Jones <em>et al.</em> 2012</a>; <a id="xref-ref-129-4" class="xref-bibr" href="#ref-129">Wolf and Ellegren 2017</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Burri <em>et al.</em> 2015</a>; <a id="xref-ref-124-1" class="xref-bibr" href="#ref-124">Vijay <em>et al.</em> 2016</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Harr 2006</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Poelstra <em>et al.</em> 2014</a>). This is in accordance with a scenario where gene-flow has reduced the level of genetic differentiation in general, but where particular genomic regions are sheltered from intermixing of parental alleles, and highly differentiated regions (so called “speciation islands” or “differentiation islands”) were therefore initially interpreted to contain incompatibility loci. However, with the recognition that a heterogeneous differentiation landscape may evolve also without gene-flow (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Cruickshank and Hahn 2014</a>), elevated differentiation may for example reflect low ancestral genetic diversity, and/or be a side-effect of reduced regional effective population size as a consequence of lower than average recombination rate (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Burri <em>et al.</em> 2015</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Noor and Bennett 2009</a>), the interpretations of patterns of genomic differentiation have become more careful (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Pennisi 2014</a>; <a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Ravinet <em>et al.</em> 2017</a>). Although natural selection is a key driver toward patterns of heterogeneous differentiation between diverging lineages with or without gene-flow (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Cruickshank and Hahn 2014</a>), linking patterns of genomic differentiation to reproductive isolation is challenging (<a id="xref-ref-94-2" class="xref-bibr" href="#ref-94">Ravinet <em>et al.</em> 2017</a>). One way forward is to have robust and well-planned study designs with genome-wide data of populations at different stages of divergence, and/or including data from multiple geographical locations involving both allopatric and sympatric regions, including potential hybrid zones (<a id="xref-ref-103-4" class="xref-bibr" href="#ref-103">Seehausen <em>et al.</em> 2014</a>). Detailed analysis of population genetic summary statistics in allopatry and in sympatry complemented by comparisons between empirical and theoretical distributions to detect significant outliers among statistics can assist in the understanding of the initiation and establishment of divergence process underlying observed patterns of differentiation, and can shed light on the progression of the speciation process (<a id="xref-ref-103-5" class="xref-bibr" href="#ref-103">Seehausen <em>et al.</em> 2014</a>; <a id="xref-ref-129-5" class="xref-bibr" href="#ref-129">Wolf and Ellegren 2017</a>), with or without gene-flow (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Feder <em>et al.</em> 2013</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">2012</a>).</p><p id="p-6">Here, we present a population genomic survey using both allopatric and sympatric samples of the common chiffchaff (<em>Phylloscopus collybita abietinus</em>) and Siberian chiffchaff (<em>P. tristis</em>) as a study system to quantify patterns of genetic diversity and differentiation across the genome. The chiffchaff superspecies complex is part of the Eurasian Old World leaf warbler group, and, historically, the complex has been divided in four distinct species: Canarian chiffchaff, Iberian chiffchaff, mountain chiffchaff, and common chiffchaff (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Helbig <em>et al.</em> 1996</a>). The common chiffchaff (<em>P. collybita</em>) has been divided into five different subspecies, including subspecies <em>abietinus</em> (eastern European common chiffchaff) and <em>tristis</em> (Siberian chiffchaff). <em>P. tristis</em> has recently been given full species status (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">del Hoyo and Collar 2017</a>), now being considered a sister species to <em>P. collybita</em>, and we therefore refer to common chiffchaff and Siberian chiffchaff as distinct species. Since our particular focus concerns the comparison of <em>P. tristis</em> to one of the European common chiffchaff subspecies (<em>P. c. abietinus</em>) only, and to accommodate consistency with recent previous analyses of this particular pair of taxa (<a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">Shipilina <em>et al.</em> 2017</a>), we will refer to the two species as <em>abietinus</em> and <em>tristis</em> from here on. The breeding range of <em>abietinus</em> covers the regions from north-eastern Europe to the Ural mountains, and the seasonal migration goes to wintering areas in eastern and central Africa (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">del Hoyo <em>et al.</em> 2006</a>; <a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">Stepanyan 1990</a>). The Siberian chiffchaff (<em>tristis</em>) is distributed from the Ural mountains to eastern Siberia during breeding season, and migrates to south-central and south-eastern Asia (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">del Hoyo <em>et al.</em> 2006</a>). Previous analyses using both phenotypic, acoustic, and genetic data have shown that <em>abietinus</em> and <em>tristis</em> differ significantly in appearance, body size, vocalizations, habitat preference, and migration patterns (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Komarova and Shipilina 2010</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Marova and Alekseev 2008</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Marova <em>et al.</em> 2009</a>, <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">2013</a>, <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">2017</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Marova and Leonovich 1993</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Martens and Meinche 1989</a>; <a id="xref-ref-115-1" class="xref-bibr" href="#ref-115">Ticehurst 1938</a>; <a id="xref-ref-118-1" class="xref-bibr" href="#ref-118">Van den Berg 2009</a>; <a id="xref-ref-113-1" class="xref-bibr" href="#ref-113">Svensson 1992</a>), and that they co-occur in a zone of secondary contact in a region from the southern Ural mountains in the south to the White Sea in the north, where they commonly hybridize and back-cross (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Marova <em>et al.</em> 2017</a>; <a id="xref-ref-105-2" class="xref-bibr" href="#ref-105">Shipilina <em>et al.</em> 2017</a>). <em>Abietinus</em> is more yellowish-green in plumage color, has a slightly larger body size, sings a slower and more rhythmic song with wider frequency range, and prefers mixed forests, whereas <em>tristis</em> has a duller appearance with no yellow tones (except on under wing coverts), is smaller in size, sings a faster and more diverse song, and preferably inhabits boreal forests (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Marova and Alekseev 2008</a>; <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Marova <em>et al.</em> 2009</a>, <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">2013</a>, <a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">2017</a>). In the sympatric zone, distinct <em>abietinus</em> and <em>tristis</em> individuals appear, but a relatively large proportion of individuals are characterized by showing intermediate plumage color, singing a mixed song type, and having a genetic composition consisting of a mixture of diagnostic <em>abietinus</em> and <em>tristis</em> alleles (<a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">Marova <em>et al.</em> 2009</a>; <a id="xref-ref-105-3" class="xref-bibr" href="#ref-105">Shipilina <em>et al.</em> 2017</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Lindholm 2008</a>; <a id="xref-ref-115-2" class="xref-bibr" href="#ref-115">Ticehurst 1938</a>). This makes the <em>abietinus</em> and <em>tristis</em> species pair an ideal model system to characterize the patterns of genomic differentiation as a result of divergence in allopatry followed by recurrent hybridization and back-crossing in sympatry. Our aims with the study were to: i) use population whole genome resequencing data to get global (genome-wide) and regional (locus specific) estimates of diversity and differentiation in allopatry and sympatry, ii) to use a genome-scan approach to characterize the landscape of differentiation between typical <em>abietinus</em> and <em>tristis</em> morphotypes in allopatry and in sympatry to detect regions that appear impermeable to gene-flow, and iii) to identify candidate genes for local adaptation and reproductive isolation between the species.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Sampling information and DNA sequencing</h3><p id="p-7">Male individuals of both <em>abietinus</em> and <em>tristis</em> were collected in the field during breeding seasons 2007–2009 (for details, see <a id="xref-ref-105-4" class="xref-bibr" href="#ref-105">Shipilina <em>et al.</em> 2017</a>). Ten allopatric and 10 sympatric birds were selected for population resequencing from each of the two species—in total 40 birds. All birds were captured using mist-netting with song traps, and blood samples were collected using a standard syringe to puncture the brachial vein. Blood was stored on filter paper and DNA was extracted using a standard phenol-chloroform protocol (<a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Sambrook <em>et al.</em> 1989</a>). Illumina, paired-end, individually barcoded, libraries with 380-bp insert sizes were constructed, and sequenced in a multiplex set-up in two separate batches on an Illumina HiSeq2000 sequencer—in total, all 40 samples were run in multiplex on eight lanes, generating, on average, 3.7 Gb of raw sequence data for each individual (Supplemental Material, Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>). A single allopatric <em>abietinus</em> male (<em>Pcol09</em>) was selected for deep sequencing, and two different paired-end libraries (180 and 380 bp) were generated for this individual and sequenced on four lanes—this generated, in total, 100.2 Gb of raw sequence data (91.1 Gb after filtering out bases with Q-scores &lt;30) for this individual (Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>). Libraries were prepared by the SNPandSEQ Technology Platform in Uppsala, and sequencing was performed in two separate batches, one batch using Illumina HiSeq2000 technology (100 bp read length) at the SNPandSEQ Technology Platform in Uppsala (four lanes with 10 samples per lane, data also used in <a id="xref-ref-105-5" class="xref-bibr" href="#ref-105">Shipilina <em>et al.</em> 2017</a>), and a supplementary batch using the same technology at the National Genomics Infrastructure (NGI) center in Stockholm (again four lanes with 10 samples per lane). All sequence reads have been deposited in the European Nucleotide Archive (<a href="http://www.ebi.ac.uk/ena">http://www.ebi.ac.uk/ena</a>) under accession number: PRJEB21643 (Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>). The sample names <em>Pcol01-Pcol704</em> follow the nomenclature established during the sampling stage, when <em>abietinus</em> and <em>tristis</em> were classified as subspecies of the common chiffchaff (<em>P. collybita</em>; <em>i.e.</em>, <em>P. c. abietinus and P. c. tristis</em>), and we decided to keep the sample names for easier comparison of results from studies of the same system (<a id="xref-ref-105-6" class="xref-bibr" href="#ref-105">Shipilina <em>et al.</em> 2017</a>).</p></div><div id="sec-3" class="subsection"><h3>Assembly and quality assessment</h3><p id="p-8">A reference-assisted nuclear genome assembly of the <em>Pcol09</em> individual was generated using version fAlb15 of the collared flycatcher (<em>Ficedula albicollis</em>) assembly as a reference (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Ellegren <em>et al.</em> 2012</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Kawakami <em>et al.</em> 2014</a>). First, paired-end sequence reads from <em>Pcol09</em> (&gt;100× sequence coverage in total for the two different insert size libraries) were aligned to the <em>F. albicollis</em> assembly using BWA version 0.7.8 (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Li and Durbin 2010</a>). The consensus sequence from all mapped reads with mapping quality ≥30, and a coverage between 5× and 50× per base was extracted from the resulting BAM file using the mpileup function in SAMtools version 1.3 (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Li <em>et al.</em> 2009</a>). The vcfutils function in BCF-tools was used to convert the assembly VCF-file into a FASTQ-file, and seqtk (<a href="https://github.com/lh3/seqtk">https://github.com/lh3/seqtk</a>) was used for the conversion of the FASTQ-file format to a FASTA formatted final assembly. Ambiguous bases in the reference-assisted assembly were replaced by N if they represented more than two alleles, and diallelic ambiguities were assigned to either allele randomly using an inhouse developed python script (<a href="https://github.com/venta380/chiffchaff_project/">https://github.com/venta380/chiffchaff_project/</a>). The reference assisted chiffchaff genome assembly covered 89% of the entire <em>F. albicollis</em> reference genome, and spanned, in total, 1.04 Gb. It should be noted that a reference assisted assembly is blind to structural changes and differences in synteny can affect the interpretation of genome-wide patterns of genetic variation. Although the divergence time of <em>Ficedula</em> and <em>Phylloscopus</em> is ∼40 MY (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Claramunt and Cracraft 2015</a>; <a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">Selvatti <em>et al.</em> 2015</a>), the avian karyotype is remarkably conserved (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ellegren 2009</a>), and the reference-assisted chiffchaff assembly likely contains only a limited number of structural inconsistencies, with a minor effect on the main conclusions of this study. The quality of the chiffchaff assembly was evaluated using the QUAST (Quality Assessment Tool for Genome Assemblies) software version 3.2 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Gurevich <em>et al.</em> 2013</a>), and by browsing for presence of conserved vertebrate and eukaryote gene sets using BUSCO (Benchmarking Universal Single-Copy Orthologs) version 1.1b1 (<a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">Simao <em>et al.</em> 2015</a>). The quality assessment results obtained for the chiffchaff assembly were compared to BUSCO results for the <em>F. albicollis</em> reference assembly and for the crow (<em>Corvus corone</em>) genome assembly, to obtain comparative data for high-quality avian genome assemblies. In order to assure sufficient mapping efficiency of the population resequencing data, one random sample (<em>Pcol41</em>) from the resequenced sample set was mapped to the chiffchaff assembly using BWA version 0.7.8 (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Li and Durbin 2010</a>), and mapping coverage and depth were computed using SAM-tools version 1.3 (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Li <em>et al.</em> 2009</a>) and BED-tools version 2.25.0 (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Quinlan and Hall 2010</a>). All assembly and quality assessment statistics and comparisons to reference bird species (<em>F. albicollis</em> and <em>C. corone</em>) are presented in Table S2 and Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>.</p><p id="p-9">Complete chiffchaff mtDNA genome assemblies were generated for each of the 40 samples using mapping with BWA (<a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Li and Durbin 2010</a>), with the previously available <em>F. albicollis</em> mitochondrial genome as template to identify mtDNA reads. Mapped reads were subsequently used in <em>de novo</em> assemblies of chiffchaff mtDNA genome assemblies using the iterative mapping and assembly tool MITObim (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Hahn 2013</a>). The procedure was iterated five times using the most recently generated chiffchaff instead of the <em>F. albicollis</em> mtDNA assembly as a template. The final chiffchaff mtDNA assemblies were ∼17,000 bp long, and we annotated them with the web server MITOS (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bernt <em>et al.</em> 2013</a>). A comparison between the <em>F. albicollis</em> and the chiffchaff mtDNA assembly for individual <em>Pcol09</em> (the individual with deepest sequencing coverage also used for reference assisted genome assembly) revealed that gene order and gene sizes were extremely conserved between the two lineages (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>)—note that the mtDNA genome is circular and the presentation in linear format generates a random cut point so that start and stop positions are different in the two assemblies.</p></div><div id="sec-4" class="subsection"><h3>Population resequencing and variant calling</h3><p id="p-10">The chiffchaff assembly was prepared to be used as a reference for mapping by indexing to different formats compatible with the GATK (Genome Analysis Tool Kit) SNP calling algorithms using BWA version 0.7.8 (<a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Li and Durbin 2010</a>) and SAMtools version 1.3 (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Li <em>et al.</em> 2009</a>). An assembly sequence database with scaffold ID, scaffold size, and genomic anchoring positions on the <em>F. albicollis</em> reference genome was created using Picard Tools version 1.127 (available at: <a href="https://github.com/broadinstitute/picard">https://github.com/broadinstitute/picard</a>). All 40 resequenced samples were mapped individually to the chiffchaff reference assembly using BWA version 0.7.8 (<a id="xref-ref-60-5" class="xref-bibr" href="#ref-60">Li and Durbin 2010</a>). The resulting SAM (Sequence Alignment/Map) files were converted into BAM (Binary Alignment/Map) files and merging of BAM-files for different sequence sets for each respective individual was done using SAMtools version 1.3 (<a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Li <em>et al.</em> 2009</a>). In order to prepare the obtained 40 BAM files for the variant calling step, different preprocessing steps of the GATK (Genome Analysis Tool Kit) best practices were performed (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">McKenna <em>et al.</em> 2010</a>); the PCR duplicates were marked and deleted with MarkDuplicates option in Picard tools version 1.127 (available at: <a href="https://github.com/broadinstitute/picard">https://github.com/broadinstitute/picard</a>), and the sequences flanking indels were realigned using IndelRealigner in GATK (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">McKenna <em>et al.</em> 2010</a>). Then, all reads from the individual used for reference assisted genome assembly (<em>Pcol09</em>) were mapped back to the assembly using BWA (<a id="xref-ref-60-6" class="xref-bibr" href="#ref-60">Li and Durbin 2010</a>), and the HaplotypeCaller in GATK (<a id="xref-ref-75-3" class="xref-bibr" href="#ref-75">McKenna <em>et al.</em> 2010</a>) was run to generate an initial set of variants. The resulting VCF (Variant Call Format) file obtained from this analysis was used together with the BaseRecalibrator utility in GATK (<a id="xref-ref-75-4" class="xref-bibr" href="#ref-75">McKenna <em>et al.</em> 2010</a>) to call a reference set of variants for the recalibration of bases in all the population samples. The quality of the recalibrated BAM files was checked with QualiMap version 2.0.2 (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Okonechnikov <em>et al.</em> 2015</a>). One sample (<em>Pcol703</em>, an allopatric <em>tristis</em> individual) was found to have considerably lower mapping quality than all other samples, and was removed from the downstream analysis in order to reduce biases in the results as a consequence of differences in mapping efficiency, coverage, and variant calling. Sequence polymorphisms were called for each sample individually using HaplotypeCaller, subsequently joint genotyped using GenotypeGVCFs, and filtered using standard settings for the VariantRecalibrator in GATK (<a id="xref-ref-75-5" class="xref-bibr" href="#ref-75">McKenna <em>et al.</em> 2010</a>).</p></div><div id="sec-5" class="subsection"><h3>Assignment analysis and visualization of genetic relationships</h3><div id="sec-6" class="subsection"><h4>Population structure:</h4><p id="p-11">A SNP-based principal component analysis (PCA) was conducted on genotypes that were covered with a minimum of seven reads per individual in all samples in each of the four populations. The PCA was performed with 220,097 SNPs using the R Bioconductor package SNPRelate (<a id="xref-ref-133-1" class="xref-bibr" href="#ref-133">Zheng 2012</a>), using a linkage disequilibrium threshold of <em>r<sup>2</sup></em> &lt; 0.2 (<a href="https://github.com/venta380/chiffchaff_project/">https://github.com/venta380/chiffchaff_project/</a>). A population structure analysis was conducted using the R Bioconductor package LEA (Landscape and Ecological Association Studies) (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Frichot 2015</a>) on the same set of 220,097 high-quality SNPs used for the PCA analysis. The LEA package uses a program called sNMF (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Frichot 2014</a>) to cluster multi-locus genotypes based on a set value of number of populations (K) provided to the program. The range of K-values were set to be from 1 to 8 to find out the optimal number of populations explaining the data using cross-entropy values (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Frichot 2014</a>, <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">2015</a>).</p></div><div id="sec-7" class="subsection"><h4>Mitochondrial DNA:</h4><p id="p-12">The mitochondrial genome was assembled for each individual independently (see above), and all mtDNA genome sequences were aligned with MAFFT version 7 (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Katoh and Standley 2013</a>), generating an alignment of 17,240 bp. The alignment was visually inspected for potential errors and ends (representing parts of the hypervariable D-loop region) were discarded to avoid false signals in the analysis of relationships. In total, 16,998 bp were used for the subsequent analyses. A branch length scaled mtDNA genome phylogeny was generated using Neighbor-Joining (<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Saitou and Nei 1987</a>). The distances (substitutions per site) were computed using the Maximum Composite Likelihood (<a id="xref-ref-114-1" class="xref-bibr" href="#ref-114">Tamura <em>et al.</em> 2004</a>), and rate variation among sites was modeled using a gamma distribution with shape parameter 1 as implemented in MEGA7 (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Kumar <em>et al.</em> 2016</a>). Ambiguous positions were removed for pairwise comparisons. The mtDNA genome alignment was used to infer intrapopulation levels of nucleotide diversity by averaging over all sequence pairs in each respective population, and omitting missing data in pairwise comparisons using the Maximum Composite Likelihood method (<a id="xref-ref-114-2" class="xref-bibr" href="#ref-114">Tamura <em>et al.</em> 2004</a>) as implemented in MEGA7 (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Kumar <em>et al.</em> 2016</a>). We also estimated average mtDNA genome sequence differentiation across population pairs, again averaging over cross-population sequence pairs using the Maximum Composite Likelihood method (<a id="xref-ref-114-3" class="xref-bibr" href="#ref-114">Tamura <em>et al.</em> 2004</a>) as implemented in MEGA7 (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Kumar <em>et al.</em> 2016</a>), and ignoring missing data in pairwise comparisons.</p></div></div><div id="sec-8" class="subsection"><h3>Genome resequencing analysis</h3><div id="sec-9" class="subsection"><h4>Repeat filtering:</h4><p id="p-13">Before any additional analyses, interspersed and tandem repeats in the chiffchaff assembly were hard masked using RepeatMasker version 4.0.6 (<a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">Smit <em>et al.</em> 2013–2015</a>), using previously available <em>F. albicollis</em> repeat annotation libraries (<a href="https://www.ncbi.nlm.nih.gov/genome/annotation_euk/Ficedula_albicollis/101/">https://www.ncbi.nlm.nih.gov/genome/annotation_euk/Ficedula_albicollis/101/</a>) as reference.</p></div><div id="sec-10" class="subsection"><h4>Genotype filtering:</h4><p id="p-14">The sequence coverage depth at each genotype was calculated using the geno-depth option in Vcf-tools version 0.1.14 (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Danecek <em>et al.</em> 2011</a>). To avoid considerable biases in allele frequency estimates in downstream analysis, only SNPs that were present in at least seven individuals in each respective population [allopatric <em>abietinus</em> (<em>n</em> = 10), allopatric <em>tristis</em> (<em>n</em> = 9), sympatric <em>abietinus</em> (<em>n</em> = 10), and sympatric <em>tristis</em> (<em>n</em> = 10)], with a minimum sequence coverage of five reads (<em>i.e.</em>, each base had to be covered by at least five different reads), were retained for subsequent analysis. Genotype calls were translated to specific polymorphism classes for both the allopatric and sympatric species comparison; fixed differences between the two species, private polymorphisms in <em>abietinus</em>, private polymorphisms in <em>tristis</em>, and shared SNPs (polymorphic in both species).</p></div><div id="sec-11" class="subsection"><h4>Population genetic summary statistics:</h4><p id="p-15">The ANGSD (Analysis of Next Generation Sequencing Data) package version 0.902 (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Korneliussen <em>et al.</em> 2014</a>) was used to calculate population genetic summary statistics within each of the populations [pair-wise nucleotide diversity (<em>Ɵ<sub>π</sub></em>), Watterson’s theta (<em>Ɵ<sub>W</sub></em>) and Tajima’s <em>D</em>] and genetic differentiation (<em>F<sub>ST</sub></em>) between allopatric and sympatric species pairs. As the ancestral state for segregating sites was not known, the folded allele frequency likelihoods and allele frequency spectra were computed using GATK’s genotype likelihood model as implemented in ANGSD (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Korneliussen <em>et al.</em> 2014</a>). We used inhouse developed python scripts (<a href="https://github.com/venta380/chiffchaff_project/">https://github.com/venta380/chiffchaff_project/</a>) to calculate absolute divergence (<em>D<sub>XY</sub></em>) between allopatric and sympatric species pairs, and independent estimates of <em>Ɵ<sub>π</sub></em> for each respective population based on the allele frequencies estimated using ANGSD. The rationale behind this was that the inhouse calculated <em>D<sub>XY</sub></em> estimates, and the <em>Ɵ<sub>π</sub></em> estimates from ANGSD were not calculated using identical settings, and to compare intraspecific polymorphism levels with absolute divergence we needed to apply the same settings for these summary statistics. All summary statistics were averaged over nonoverlapping windows of 10 kb across the genome. The levels of linkage disequilibrium (LD, estimated with the correlation between alleles at different loci: <em>r<sup>2</sup></em>) for each respective population were calculated for SNP pairs &lt;100 kb apart using Plink version 1.90 (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Purcell <em>et al.</em> 2007</a>). To avoid inflated LD estimates due to low allele frequencies, we only included SNPs with minor allele frequency (MAF) &gt; 0.2. The population specific average LD per SNP pair in each 10-kb window was calculated using custom python scripts (<a href="https://github.com/venta380/chiffchaff_project/">https://github.com/venta380/chiffchaff_project/</a>), with the filtered LD output from each population specific analysis from Plink (<a id="xref-ref-90-2" class="xref-bibr" href="#ref-90">Purcell <em>et al.</em> 2007</a>).</p><p id="p-16">To assess the genomic distribution of the most highly differentiated regions between <em>abietinus</em> and <em>tristis</em>, and identify differentiation outliers, we Z-transformed the <em>F<sub>ST</sub></em> estimates (<em>F<sub>ST</sub><sup>Z</sup></em>) for each comparison by standardizing with the genomic average and variance across windows according to the following formula:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3983/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3983/embed/mml-math-1.gif"/></noscript></span></span>The estimates were calculated for the autosomes and the Z-chromosome separately since the effective population size (<em>N<sub>e</sub></em>) for the Z-chromosome was reduced as compared to the autosomes, and, hence, the expected baseline levels differ between these chromosome classes. Autosomal windows with <em>F<sub>ST</sub><sup>Z</sup></em> values higher than the 99th percentile in the distributions were considered outliers (≥4 SD higher than the global mean) for the allopatric and the sympatric comparison, respectively. In order to evaluate if the overall sequence coverage affected the <em>F<sub>ST</sub></em> distribution (hypothetically the level of differentiation may be biased upwards as a result of unusual low coverage), per-site genome coverage was calculated using the coverageBed utility in the BEDTools version 2.25.0 (<a id="xref-ref-91-2" class="xref-bibr" href="#ref-91">Quinlan and Hall 2010</a>), and the correlation between the coverage and <em>F<sub>ST</sub></em> was checked. We found only weak and inconsistent correlations between sequence coverage and <em>F<sub>ST</sub></em> (range of Pearson’s <em>r</em> = −0.007 to 0.025 for the four different populations). To visually inspect the landscape of diversity and differentiation in the allopatric and the sympatric species comparisons, all population genetic summary statistics were plotted against physical positions along chromosomes using the ggplot2 library in R (<a href="https://cran.r-project.org/">https://cran.r-project.org/</a>). The most extreme differentiation regions in each comparison (<em>F<sub>ST</sub><sup>Z</sup></em> &gt; 99th percentile in the empirical distribution) were further investigated by extracting the coordinates for clusters (≥10 outlier windows present in a 0.5-Mb genomic region) of the windows with the highest level of relative differentiation, and intersect these with annotation data from the <em>F. albicollis</em> genome (<a href="http://www.ensembl.org/Ficedula_albicollis/Info/Index">http://www.ensembl.org/Ficedula_albicollis/Info/Index</a>: 2017-06-30). The rationale behind this strategy was to identify coding genes to investigate if any particular functional category of genes was over-represented in the most highly differentiated regions. We used a binomial distribution to estimate the probability of this level of clustering or higher, since the sample size (50 windows per block) was much smaller than the total population (103,800 windows in total; 1038 outlier windows). The probability of 10 outlier windows clustering in a 500 kb (50 windows) region was estimated at 7.133 × 10<sup>−11</sup>. Importantly, relaxing the significance level did not affect the results quantitatively, <em>i.e.</em>, very few additional blocks were identified if the limits were set to 5 (p-value ≈ 0.0001) or even three outliers (p-value ≈ 0.01) within a 50-window block (data not shown). Enrichment analysis of specific gene ontology (GO) terms for genes located in chromosome regions with higher than expected density of relative differentiation outliers was performed using GOstat with a custom reference database based on GO terms from the <em>F. albicollis</em> gene set as available in Ensembl (<a href="http://www.ensembl.org/Ficedula_albicollis/Info/Index">http://www.ensembl.org/Ficedula_albicollis/Info/Index</a>: 2017-06-30). The identities and potential functions of the identified genes in highly differentiated regions were inferred using orthology information as available in the Biomart tool in Ensembl (<a href="http://www.ensembl.org/biomart/martview/c02baa1e6506b4840e7e34ceac0a49da">http://www.ensembl.org/biomart/martview/c02baa1e6506b4840e7e34ceac0a49da</a>: 2017-06-30).</p></div></div><div id="sec-12" class="subsection"><h3>Functional analysis of fixed, shared, and private polymorphisms</h3><p id="p-17">To further assess the potential functional relevance of different types of polymorphisms, we used custom PyVCF 0.6.8 scripts (<a href="https://github.com/venta380/chiffchaff_project/">https://github.com/venta380/chiffchaff_project/</a>) to sort previously called SNPs into four categories for the allopatric species comparison (as mentioned above, we found no fixed differences between sympatric <em>abietinus</em> and <em>tristis</em>): fixed differences between the two species, private polymorphisms in <em>abietinus</em>, private polymorphisms in <em>tristis</em> and shared SNPs (polymorphic in both species). Only SNPs covered by at least five reads in each individual and where information was present for at least nine individuals in each of the populations were retained for this analysis (<em>n</em> = 2,810,678) to minimize the bias toward fixed differences when representation in individuals is low. We identified the functional relevance of each SNP using the annotation software snpEFF version 4.3 (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cingolani <em>et al.</em> 2012</a>) with a manually built custom database based on genome annotation information from <em>F. albicollis</em> (<a href="http://www.ensembl.org/Ficedula_albicollis/Info/Index">http://www.ensembl.org/Ficedula_albicollis/Info/Index</a>), which is the closest relative to chiffchaff with extensive genome annotation information. The annotations were grouped into classes “downstream region,” “upstream region,” “exon,” “intron,” “intergenic region,” and “other” (the latter including 3′UTR, 5′UTR, and splice sites). The relative frequencies of different polymorphisms were then compared across annotation classes for autosomes and the Z-chromosome separately to get information about the relative effects of selection and genetic drift in driving the patterns of differentiation, heterozygosity, allele sharing, and accumulation of fixed differences between <em>abietinus</em> and <em>tristis</em>.</p></div><div id="sec-13" class="subsection"><h3>Data availability</h3><p id="p-18">All sequence reads in this study have been deposited in the European Nucleotide Archive under project accession number PRJEB21643. Individual accession numbers for all individuals are available in Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>.</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>Assembly statistics</h3><p id="p-19">The total length of the chiffchaff assembly was 1.04 Gb, similar to previously available avian genome assemblies (<a id="xref-ref-132-1" class="xref-bibr" href="#ref-132">Zhang <em>et al.</em> 2014</a>). The longest scaffold was 157.6 Mb, and the scaffold N50 length was 70.4 Mb, identical to the fAlb15 assembly (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Kawakami <em>et al.</em> 2014</a>), as expected, given that <em>F. albicollis</em> was used as reference for mapping. The number of Ns in the assembly was 19%, and the total GC content was 31% (AT = 50%). All statistics of the chiffchaff assembly are presented in Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>. The assessment of the chiffchaff assembly completeness based on conserved gene sets showed that 80.9% of vertebrate and 50.3% of eukaryotic genes were completely covered. The fractions of fragmented genes in the vertebrate and eukaryotic gene sets were 6.5 and 4.9%, respectively, and missing genes constituted 12.6 and 44.8% of the gene sets (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>). For comparison, the high-quality <em>F. albicollis</em> and <em>C. corone</em> assemblies cover only slightly higher fractions of complete vertebrate (92.8 and 84.4%, respectively) and eukaryotic (50.3% in both species) genes, and have a slightly lower proportion of completely missing genes; 2.5 and 10.1% in the vertebrate gene set (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-16" class="subsection"><h3>Global estimates of genetic diversity and differentiation</h3><p id="p-20">In total, we detected ∼7 million SNPs across all individuals from both species, of which 6,912,505 were scored in the allopatric population pairs and 6,843,490 in the sympatric population pairs (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). We observed a significantly higher proportion of shared SNPs (sympatry = 48.0%, allopatry = 35.5%), and a lower proportion of fixed differences (sympatry = 0.0%, allopatry = 0.3%) between <em>abietinus</em> and <em>tristis</em> in sympatry than in allopatry (Chi-Square test: χ<sup>2</sup> = 283,510, <em>df</em> = 3, p-value &lt; 2.2 × 10<sup>−16</sup>; <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The proportion of fixed differences on the Z-chromosome was significantly higher in the comparison of allopatric <em>abietinus</em> and <em>tristis</em> (5.4%), but there were no fixed differences on the Z-chromosome in the sympatric comparison (Chi-Square test: χ<sup>2</sup> = 198,380, <em>df</em> = 3, p-value &lt; 2.2 × 10<sup>−16</sup>; <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Both the PCA- and the structure (sNMF) analysis clearly separated allopatric <em>abietinus</em> and <em>tristis</em> (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) with individuals from the sympatric regions clustering at the intermediate space in the PCA analysis (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), and showing considerable admixture proportions in the sNMF analysis (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). In the PCA, allopatric <em>abietinus</em> showed more interindividual variation than allopatric <em>tristis</em>, and sympatric <em>tristis</em> tended to cluster closer with allopatric <em>tristis</em>, while sympatric <em>abietinus</em> were more scattered across the range between the allopatric groups (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). This pattern was consistent with the clustering analysis where allopatric <em>abietinus</em> and <em>tristis</em> form clearly separated groups (optimal <em>K</em> = 2), and all sympatric <em>tristis</em> had &gt;75% of the assigned proportions shared with allopatric <em>tristis</em> while sympatric <em>abietinus</em> varied from being assigned with 100% to <em>abietinus</em> to &gt;90% with <em>tristis</em> (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487498/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487498/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of numbers (#) and proportions (in %) of shared, private, and fixed polymorphisms across allopatric and sympatric <em>abietinus</em> (<em>ab</em>.) and <em>tristis</em> (<em>tr</em>.) for the autosomes (A), the Z-chromosome (Z), and all chromosomes jointly (All)</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A visual illustration of genetic similarities across all chiffchaff individuals using a PCA as implemented in the R Bioconductor package SNPRelate (Zheng 2012) of 220,097 unlinked (pairwise LD as estimated by r2 &lt; 0.2) high quality SNPs. Samples are color-coded according to origin of individuals (yellow, allopatric abietinus; blue, sympatric abietinus; green, sympatric tristis; brown, allopatric tristis). Axes show the eigenvalues from the PCA for the two components explaining most of the variation (PC1 = 4.6% and PC2 = 3.2%)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-523979459" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A visual illustration of genetic similarities across all chiffchaff individuals using a PCA as implemented in the R Bioconductor package SNPRelate (Zheng 2012) of 220,097 unlinked (pairwise LD as estimated by &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; &lt; 0.2) high quality SNPs. Samples are color-coded according to origin of individuals (yellow, allopatric &lt;em&gt;abietinus&lt;/em&gt;; blue, sympatric &lt;em&gt;abietinus&lt;/em&gt;; green, sympatric &lt;em&gt;tristis&lt;/em&gt;; brown, allopatric &lt;em&gt;tristis&lt;/em&gt;). Axes show the eigenvalues from the PCA for the two components explaining most of the variation (PC1 = 4.6% and PC2 = 3.2%).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/3983/F1.medium.gif" width="440" height="313"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/12/3983/F1.medium.gif" width="440" height="313"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487580" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">A visual illustration of genetic similarities across all chiffchaff individuals using a PCA as implemented in the R Bioconductor package SNPRelate (<a id="xref-ref-133-2" class="xref-bibr" href="#ref-133">Zheng 2012</a>) of 220,097 unlinked (pairwise LD as estimated by <em>r<sup>2</sup></em> &lt; 0.2) high quality SNPs. Samples are color-coded according to origin of individuals (yellow, allopatric <em>abietinus</em>; blue, sympatric <em>abietinus</em>; green, sympatric <em>tristis</em>; brown, allopatric <em>tristis</em>). Axes show the eigenvalues from the PCA for the two components explaining most of the variation (PC1 = 4.6% and PC2 = 3.2%).</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of the analysis of sample clustering using the R Bioconductor package LEA with 220,097 high quality SNPs (Frichot 2014, 2015). (A) Yellow, abietinus-derived alleles; brown, tristis derived alleles. Each vertical bar represents one individual, and samples are sorted according to population origin with allopatric abietinus to the far left, sympatric populations in the middle, and allopatric tristis to the far right. (B) Summary of error (entropy) rates for K = 1&#x2013;8 predefined populations. The largest rate change in entropy occurs from two to three populations indicating that K = 2 is the optimal number of clusters (Frichot 2014, 2015)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-523979459" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Summary of the analysis of sample clustering using the R Bioconductor package LEA with 220,097 high quality SNPs (Frichot 2014, 2015). (A) Yellow, &lt;em&gt;abietinus&lt;/em&gt;-derived alleles; brown, &lt;em&gt;tristis&lt;/em&gt; derived alleles. Each vertical bar represents one individual, and samples are sorted according to population origin with allopatric &lt;em&gt;abietinus&lt;/em&gt; to the far left, sympatric populations in the middle, and allopatric &lt;em&gt;tristis&lt;/em&gt; to the far right. (B) Summary of error (entropy) rates for &lt;em&gt;K&lt;/em&gt; = 1&#x2013;8 predefined populations. The largest rate change in entropy occurs from two to three populations indicating that &lt;em&gt;K&lt;/em&gt; = 2 is the optimal number of clusters (Frichot 2014, 2015).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/3983/F2.medium.gif" width="440" height="245"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/12/3983/F2.medium.gif" width="440" height="245"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487495" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">Summary of the analysis of sample clustering using the R Bioconductor package LEA with 220,097 high quality SNPs (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Frichot 2014</a>, <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">2015</a>). (A) Yellow, <em>abietinus</em>-derived alleles; brown, <em>tristis</em> derived alleles. Each vertical bar represents one individual, and samples are sorted according to population origin with allopatric <em>abietinus</em> to the far left, sympatric populations in the middle, and allopatric <em>tristis</em> to the far right. (B) Summary of error (entropy) rates for <em>K</em> = 1–8 predefined populations. The largest rate change in entropy occurs from two to three populations indicating that <em>K</em> = 2 is the optimal number of clusters (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Frichot 2014</a>, <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">2015</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">The mtDNA genome analyses revealed two distinct mtDNA clades predominantly reflecting differentiation between allopatric <em>abietinus</em> and <em>tristis</em>, but with considerable nesting of sympatric <em>abietinus</em> individuals within the <em>tristis</em> clade (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>). Allopatric <em>tristis</em> clustered in two separate groups, birds from Southern Siberia being in a derived clade while birds from Central and Eastern Siberia and the Ural Mountains were scattered in basal lineages in the “<em>tristis</em>”-type mtDNA group, and birds from the Southern sympatric zone showed higher diversity in mtDNA haplotypes than birds from the Northern sympatric region (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>). The analyses of mtDNA genetic variation within and between populations showed that diversity was higher in <em>abietinus</em> than in <em>tristis</em> in allopatry (<em>Ɵ<sub>π</sub></em> = 5.4 ± 0.3 × 10<sup>−3</sup> <em>vs.</em> 4.0 ± 0.3 × 10<sup>−3</sup> for <em>abietinus</em> and <em>tristis</em>, respectively), and the pattern was even more pronounced in sympatry (<em>Ɵ<sub>π</sub></em> = 12.6 ± 0.5 × 10<sup>−3</sup> <em>vs.</em> 3.7 ± 0.3 × 10<sup>−3</sup> for <em>abietinus</em> and <em>tristis</em>, respectively). MtDNA genetic divergence was also higher between allopatric than between sympatric <em>abietinus</em> and <em>tristis</em> (Maximum Likelihood corrected numbers of substitutions per site = 0.0213 ± 0.0008 and 0.0117 ± 0.0004 in allopatry and sympatry, respectively, Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>). Overall, this picture a scenario where <em>abietinus</em> and <em>tristis</em> diverged in allopatry roughly 1–2 MYA (following the rationale behind a body-size corrected mtDNA genome divergence rate as proposed by <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Nabholz <em>et al.</em> 2016</a>), and subsequently came into secondary contact in the current sympatric region where hybridization and backcrossing resulted in higher levels of allele sharing between species, with higher rates of introgression from <em>tristis</em> into <em>abietinus</em>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="An unrooted phylogeny representing the entire mitochondrial genome for all 40 samples included in the study. The length of the scale bar represents 0.01 substitutions per site, and nodes are color-coded according to origin of populations (yellow, allopatric abietinus; blue, sympatric abietinus; green, sympatric tristis; brown, allopatric tristis)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-523979459" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;An unrooted phylogeny representing the entire mitochondrial genome for all 40 samples included in the study. The length of the scale bar represents 0.01 substitutions per site, and nodes are color-coded according to origin of populations (yellow, allopatric &lt;em&gt;abietinus&lt;/em&gt;; blue, sympatric &lt;em&gt;abietinus&lt;/em&gt;; green, sympatric &lt;em&gt;tristis&lt;/em&gt;; brown, allopatric &lt;em&gt;tristis&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/3983/F3.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/12/3983/F3.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487499" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">An unrooted phylogeny representing the entire mitochondrial genome for all 40 samples included in the study. The length of the scale bar represents 0.01 substitutions per site, and nodes are color-coded according to origin of populations (yellow, allopatric <em>abietinus</em>; blue, sympatric <em>abietinus</em>; green, sympatric <em>tristis</em>; brown, allopatric <em>tristis</em>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">There was no striking difference in nuclear genetic diversity across populations although <em>abietinus</em> showed slightly higher global genetic diversity than <em>tristis</em> both in allopatry and sympatry (average <em>Ɵ<sub>π</sub></em> in allopatric and sympatric <em>abietinus</em> = 0.0053 ± 0.0031 and 0.0051 ± 0.0029, respectively and 0.0047 ± 0.0031 and 0.0049 ± 0.0030 in allopatric and sympatric <em>tristis</em>, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). As expected given a lower <em>N<sub>e</sub></em>of sex-chromosomes than of autosomes, both species had lower genetic diversity on the Z-chromosome (average <em>Ɵ<sub>π</sub></em> = 0.0031 ± 0.0035 and 0.0025 ± 0.0030, for allopatric <em>abietinus</em> and <em>tristis</em>, respectively) as compared to the autosomes combined (<em>Ɵ<sub>π</sub></em> = 0.0054 ± 0.0030 and 0.0048 ± 0.0030) – the same pattern was observed in sympatry (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Since demographic history may affect allele frequency distributions and patterns of LD, we assessed if further understanding of the underlying processes behind allele sharing in sympatry could be gained by using standard population genetic summary statistics (Tajima’s <em>D</em>) and linkage information. In summary, the global estimates of Tajima’s <em>D</em> were generally negative, in line with a bottleneck + expansion model for all populations, but less pronounced in allopatric <em>tristis</em> (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). There were only minor differences in global decay patterns of LD across populations (Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a> and <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). The history of the two species provides a starting point for exploring patterns of genetic differentiation across the genome as a consequence of secondary contact and recurrent hybridization, and to investigate how gene-flow affects global and regional genomic differentiation and characterize regions that may be shielded from introgression.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487548/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487548/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of intrapopulation nuclear genomic pair-wise nucleotide diversity (<em>Ɵ<sub>π</sub></em>), Watterson’s theta (<em>Ɵ<sub>W</sub></em>), Tajima’s <em>D</em> (<em>T<sub>D</sub></em>), and LD (<em>r<sup>2</sup></em>, per base pair) as estimated in nonoverlapping 10 kb windows across the chiffchaff genome</span><div class="sb-div caption-clear"></div></div></div><p id="p-27">The global level of genetic differentiation between species was significantly lower in the sympatric region (mean ± SD of <em>F<sub>ST</sub></em> across windows = 0.099 ± 0.038) than in the allopatric regions (0.220 ± 0.064) (Wilcoxon’s test: <em>W</em> = 1.04 × 10<sup>10</sup>, p-value &lt;2.2 × 10<sup>−16</sup>), and the level of genetic differentiation was significantly higher on the Z-chromosome than on the autosomes (only allopatric comparison shown: Wilcoxon’s test: <em>W</em> = 1.05 × 10<sup>8</sup>, p-value &lt; 2.2 × 10<sup>−16</sup>; <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). An opposite pattern was observed for absolute divergence (<em>D<sub>XY</sub></em>), with reduced divergence on the Z-chromosome as compared to autosomes [Wilcoxon’s test: <em>W</em> = 1.17 × 10<sup>8</sup>, p-value &lt; 2.2 × 10<sup>−16</sup> (allopatry); <em>W</em> = 1.0 × 10<sup>8</sup>, p-value &lt; 2.2 × 10<sup>−16</sup> (sympatry)], although the quantitative difference between allopatry and sympatry was less pronounced [global <em>D<sub>XY</sub></em> = 0.0085 ± 0.0058 (allopatry) and 0.0081 ± 0.0056 (sympatry); Wilcoxon’s test: <em>W</em> = 5.76 × 10<sup>9</sup>, p-value &lt; 2.2 × 10<sup>−16</sup>], except for the Z-chromosome where <em>D<sub>XY</sub></em> was considerably reduced in sympatry as compared to allopatry [Z-chromosome <em>D<sub>XY</sub></em> = 0.0061 ± 0.0039 (allopatry) and 0.0055 ± 0.0034 (sympatry); Wilcoxon’s test: <em>W</em> = 5.1 × 10<sup>9</sup>, p-value &lt; 2.2 × 10<sup>−16</sup>; <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). The overall level of genetic differentiation was hence higher on the Z-chromosome than on the autosomes, both in allopatry and sympatry, but the opposite was observed for absolute divergence (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, and <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Violin plots showing the distributions of genetic differentiation (FST) on the Z-chromosome, autosomes and all chromosomes combined in comparisons of allopatric (gray) and sympatric (red) abietinus and tristis pairs. Horizontal solid lines within distribution curves indicate means, and broken lines the SD across all 10-kb windows in the genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-523979459" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Violin plots showing the distributions of genetic differentiation (&lt;em&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/em&gt;) on the Z-chromosome, autosomes and all chromosomes combined in comparisons of allopatric (gray) and sympatric (red) &lt;em&gt;abietinus&lt;/em&gt; and &lt;em&gt;tristis&lt;/em&gt; pairs. Horizontal solid lines within distribution curves indicate means, and broken lines the SD across all 10-kb windows in the genome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/3983/F4.medium.gif" width="440" height="247"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/12/3983/F4.medium.gif" width="440" height="247"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487592" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-28" class="first-child">Violin plots showing the distributions of genetic differentiation (<em>F<sub>ST</sub></em>) on the Z-chromosome, autosomes and all chromosomes combined in comparisons of allopatric (gray) and sympatric (red) <em>abietinus</em> and <em>tristis</em> pairs. Horizontal solid lines within distribution curves indicate means, and broken lines the SD across all 10-kb windows in the genome.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487575/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487575/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of the levels of genetic differentiation (<em>F<sub>ST</sub></em>) and absolute divergence (<em>D<sub>XY</sub></em>) across allopatric and sympatric <em>abietinus</em> and <em>tristis</em> for the autosomes (A), the Z-chromosome (Z), and all chromosomes jointly (All)</span><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustration of regional variation in fixed differences (allopatric comparison only, top panel, # Fix), genetic differentiation (second panel, FST), absolute divergence (middle panel, DXY), nucleotide diversity (fourth panel, &#x19F;&#x3C0;) and Tajima&#x2019;s D (bottom panel, TajD) represented by chromosome&#xA0;12. Values on y-axes represent ranges of each respective parameter and colors indicate species comparison (gray, allopatry; red, sympatry for # Fix, FST and DXY) or population (yellow, allopatric abietinus; brown, allopatric tristis; blue, sympatric abietinus; green, sympatric tristis for &#x19F;&#x3C0; and TajD)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-523979459" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Illustration of regional variation in fixed differences (allopatric comparison only, top panel, &lt;em&gt;# Fix&lt;/em&gt;), genetic differentiation (second panel, &lt;em&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/em&gt;), absolute divergence (middle panel, &lt;em&gt;D&lt;sub&gt;XY&lt;/sub&gt;&lt;/em&gt;), nucleotide diversity (fourth panel, &lt;em&gt;&#x19F;&lt;sub&gt;&#x3C0;&lt;/sub&gt;&lt;/em&gt;) and Tajima&#x2019;s &lt;em&gt;D&lt;/em&gt; (bottom panel, &lt;em&gt;TajD&lt;/em&gt;) represented by chromosome&#xA0;12. Values on &lt;em&gt;y&lt;/em&gt;-axes represent ranges of each respective parameter and colors indicate species comparison (gray, allopatry; red, sympatry for &lt;em&gt;# Fix&lt;/em&gt;, &lt;em&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/em&gt; and &lt;em&gt;D&lt;sub&gt;XY&lt;/sub&gt;&lt;/em&gt;) or population (yellow, allopatric &lt;em&gt;abietinus&lt;/em&gt;; brown, allopatric &lt;em&gt;tristis&lt;/em&gt;; blue, sympatric &lt;em&gt;abietinus&lt;/em&gt;; green, sympatric &lt;em&gt;tristis&lt;/em&gt; for &#x19F;&lt;sub&gt;&#x3C0;&lt;/sub&gt; and &lt;em&gt;TajD&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/3983/F5.medium.gif" width="440" height="248"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/12/3983/F5.medium.gif" width="440" height="248"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487559" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-30" class="first-child">Illustration of regional variation in fixed differences (allopatric comparison only, top panel, <em># Fix</em>), genetic differentiation (second panel, <em>F<sub>ST</sub></em>), absolute divergence (middle panel, <em>D<sub>XY</sub></em>), nucleotide diversity (fourth panel, <em>Ɵ<sub>π</sub></em>) and Tajima’s <em>D</em> (bottom panel, <em>TajD</em>) represented by chromosome 12. Values on <em>y</em>-axes represent ranges of each respective parameter and colors indicate species comparison (gray, allopatry; red, sympatry for <em># Fix</em>, <em>F<sub>ST</sub></em> and <em>D<sub>XY</sub></em>) or population (yellow, allopatric <em>abietinus</em>; brown, allopatric <em>tristis</em>; blue, sympatric <em>abietinus</em>; green, sympatric <em>tristis</em> for Ɵ<sub>π</sub> and <em>TajD</em>).</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustration of the variation in genetic differentiation (FST, top panel) and absolute genetic divergence (DXY, bottom panel) across the genome between allopatric (gray line) and sympatric (red line) abietinus and tristis. Chromosomes are ordered from chromosome&#xA0;1 to chromosome&#xA0;Z, and every other chromosome is shaded in light gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-523979459" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Illustration of the variation in genetic differentiation (&lt;em&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/em&gt;, top panel) and absolute genetic divergence (&lt;em&gt;D&lt;sub&gt;XY&lt;/sub&gt;&lt;/em&gt;, bottom panel) across the genome between allopatric (gray line) and sympatric (red line) &lt;em&gt;abietinus&lt;/em&gt; and &lt;em&gt;tristis&lt;/em&gt;. Chromosomes are ordered from chromosome&#xA0;1 to chromosome&#xA0;Z, and every other chromosome is shaded in light gray.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/3983/F6.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/12/3983/F6.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487524" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-31" class="first-child">Illustration of the variation in genetic differentiation (<em>F<sub>ST</sub></em>, top panel) and absolute genetic divergence (<em>D<sub>XY</sub></em>, bottom panel) across the genome between allopatric (gray line) and sympatric (red line) <em>abietinus</em> and <em>tristis</em>. Chromosomes are ordered from chromosome 1 to chromosome Z, and every other chromosome is shaded in light gray.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Regional variation in genetic diversity and differentiation</h3><p id="p-32">Global estimates of nuclear genetic diversity showed highly heterogeneous patterns across the genome of both species, both in allopatric and sympatric populations (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, and Figure S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>). Similar to many previous studies of avian (and other) species pairs, the level of genetic differentiation varied extensively across the genome, with presence of distinct <em>F<sub>ST</sub></em> “peaks” in both the allopatric and sympatric <em>abietinus</em> and <em>tristis</em> species comparisons (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a> and Figure S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>). A striking difference between allopatry and sympatry was that the overall level of differentiation (but not absolute divergence) was lower, and the differentiation peaks were narrower and more pronounced in the sympatric comparison (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a>, <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>, and Figure S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>), resulting in significantly different <em>F<sub>ST</sub></em>-distributions (Kolmogorov-Smirnov test: <em>D</em> = 0.88, p-value &lt; 2.2 × 10<sup>−16</sup>). There was a significant negative correlation between the level of genetic differentiation and genetic diversity in both populations [Pearson’s <em>r</em> = −0.29 and −0.32 for <em>abietinus</em> and <em>tristis</em>; p-value &lt; 2.2 × 10<sup>−16</sup> in both tests (allopatry); <em>r</em> = −0.17 and −0.21; p-value &lt; 2.2 × 10<sup>−16</sup> in both tests (sympatry)], and also a negative correlation between differentiation and absolute divergence, both in allopatry (Pearson’s <em>r</em> = −0.225, <em>t</em> = −74.421, <em>df</em> = 103,770, p-value &lt; 2.2 × 10<sup>−16</sup>) and in sympatry (Pearson’s <em>r</em> = −0.130, <em>t</em> = −30.823, <em>df</em> = 103,630, p-value &lt; 2.2 × 10<sup>−16</sup> ; Figure S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>). Tajima’s <em>D</em> was, in general, more negative in regions of high differentiation in comparison to the remaining parts of the genome (Pearson’s <em>r</em> = −0.070 to −0.166, p-values &lt; 2.2 × 10<sup>−16</sup>), with the exception of sympatric <em>abietinus</em> that showed a positive correlation between Tajima’s <em>D</em> and the level of genetic differentiation (Pearson’s <em>r</em> = 0.162, p-value &lt; 2.2 × 10<sup>−16</sup>; <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>, Figure S4, and Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>). The analysis of the levels of genetic diversity and absolute divergence shows that the correlation is very strong, both in allopatry and in sympatry, with only little deviation from the linear relationship between <em>Ɵ<sub>π</sub></em> and <em>D<sub>XY</sub></em> (Figure S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>). We observed a higher variance in absolute divergence in allopatry than in sympatry, and a more pronounced clustering of high differentiation windows at the lower left end in the graph (low genetic diversity and low absolute divergence) in autosomes than for the Z-chromosome (Figure S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimates of relative differentiation (FSTZ) between abietinus and tristis for 10&#xA0;kb windows across the genome. The upper 99th percentiles are represented by horizontal barred lines for the allopatric (top panel, gray) and sympatric (bottom panel, red) comparisons, respectively. Regions with significantly higher density of outlier windows are indicated with gray (allopatry) and red (sympatry) solid blocks." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-523979459" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Estimates of relative differentiation (&lt;em&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;sup&gt;Z&lt;/sup&gt;&lt;/em&gt;) between &lt;em&gt;abietinus&lt;/em&gt; and &lt;em&gt;tristis&lt;/em&gt; for 10&#xA0;kb windows across the genome. The upper 99th percentiles are represented by horizontal barred lines for the allopatric (top panel, gray) and sympatric (bottom panel, red) comparisons, respectively. Regions with significantly higher density of outlier windows are indicated with gray (allopatry) and red (sympatry) solid blocks.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/3983/F7.medium.gif" width="440" height="311"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/12/3983/F7.medium.gif" width="440" height="311"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487533" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-33" class="first-child">Estimates of relative differentiation (<em>F<sub>ST</sub><sup>Z</sup></em>) between <em>abietinus</em> and <em>tristis</em> for 10 kb windows across the genome. The upper 99th percentiles are represented by horizontal barred lines for the allopatric (top panel, gray) and sympatric (bottom panel, red) comparisons, respectively. Regions with significantly higher density of outlier windows are indicated with gray (allopatry) and red (sympatry) solid blocks.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Windows with <em>F<sub>ST</sub><sup>Z</sup></em> values higher than the 99th percentile in the distributions were highly clustered in specific regions for both the allopatric and the sympatric comparison (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). We tested for significant clustering by scanning the genome for regions where blocks of 50 windows (each window was 10 kb long) contained 10 or more <em>F<sub>ST</sub><sup>Z</sup></em> outlier windows (the probability of 10 or more outliers appearing in a 50-window block was estimated to 7.13 × 10<sup>−11</sup> if outliers are assumed to be randomly scattered across the genome). In the allopatric comparison, 19 blocks representing 13 chromosomal regions (chromosomes 1, 1A, 2, 3, 4, 5, 7, 8, 9, 10, 12, and 28), were identified as significantly enriched for <em>F<sub>ST</sub><sup>Z</sup></em> outliers, and the analogous analysis in sympatry revealed 13 windows, representing 10 different regions located on chromosomes 1, 1A, 5, 10, 12, 13, 17, 20, and 21 (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a> and Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>). The identified regions were highly overlapping between the two different comparisons, but, again, regions were narrower and more pronounced in sympatry (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7</a>). The chromosome regions with high density of <em>F<sub>ST</sub><sup>Z</sup></em> outliers contained 238 and 274 protein coding genes in the allopatric and the sympatric comparisons, respectively (Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>). The GO enrichment analysis revealed no over-representation of specific terms for genes in differentiation outliers in allopatric comparison (p-value threshold 0.01 after correcting for multiple testing), and only one term associated with regulation of cellular component organization in the sympatric comparison for two partly overlapping gene sets (Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>)</p></div><div id="sec-18" class="subsection"><h3>Protein coding genes in differentiation peaks</h3><p id="p-35">Global genomic differentiation was significantly lower in sympatry than in allopatry with more pronounced and narrower differentiation peaks, indicating recurrent gene-flow in large portions of the genome of the two species. The highly differentiated regions in sympatry may indicate that these regions have experienced restricted gene-flow, and therefore we wanted to identify potential targets of diversifying selection and/or incompatibility within these regions. Omitting the Z-chromosome, which, in general, showed elevated differentiation across the entire chromosome (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a>), and no relative differentiation outliers (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7</a>), we found 26 protein coding genes in the 10 kb windows with the highest level of genetic differentiation nested within blocks of high density of relative differentiation outliers (see above) in the comparison of sympatric <em>abietinus</em> and <em>tristis</em> (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Among the 26 genes, three were uncharacterized, and the remaining 23 had diverse functions, mainly related to ion transport, gene regulation (transcription factors, histone binding, and phosphorylation), muscle development, learning (<em>ARF4</em>), sensory perception of sound (<em>OTOG</em>), and pigmentation (<em>HPS5</em>) (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487582/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487582/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Protein coding genes identified in the 10-kb windows with the highest genetic differentiation (<em>F<sub>ST</sub></em>) nested within the chromosome regions enriched for windows with high relative differentiation (<em>F<sub>ST</sub><sup>Z</sup></em>) between sympatric <em>abietinus</em> and <em>tristis</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Functional relevance of fixed, shared, and private polymorphisms</h3><p id="p-37">To obtain further information about the underlying forces generating a heterogeneous distribution of genetic diversity and differentiation, we assessed the functional relevance of different categories of SNPs. This analysis was focused on the allopatric comparison, only since we did not identify any fixed differences between sympatric <em>abietinus</em> and <em>tristis</em>, and we applied more stringent filtering (sequence coverage of at least 5× per individual for at least nine individuals in both allopatric <em>abietinus</em> and <em>tristis</em>) to minimize the risk of including erroneously called fixed differences in this analysis. Among all high-quality SNPs selected for this analysis (2,810,678 SNPs), we found that almost half of them (43.6%) were shared polymorphisms, while only 7609 SNPs (0.27%) represented fixed differences. We generated annotation information for all SNPs from all of the eight SNP sets (shared, private <em>abietinus</em>, private <em>tristis</em>, and fixed differences for the autosomes and the Z-chromosome, respectively) using the annotation available for the <em>F. albicollis</em> genome assembly (<a href="http://www.ensembl.org/Ficedula_albicollis/Info/Index">http://www.ensembl.org/Ficedula_albicollis/Info/Index</a>). For autosomes, there was an excess in the proportion of fixed differences in genes, or chromosome regions in the vicinity of genes, while there was an underrepresentation of fixed differences in intergenic sequences, but this pattern was not present on the Z-chromosome (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). Since fixed differences are overrepresented in high differentiation regions (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a> and Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>), this indicates that patterns of elevated differentiation cannot entirely be explained by restricted recombination and less efficient selection, but also that diversifying selection has driven certain alleles to fixation between <em>abietinus</em> and <em>tristis</em>. We therefore took the analysis one step further by classifying all the fixed differences between the species in allopatry. The results showed that of the 7609 fixed differences identified, the majority were as expected located in introns, intergenic regions or noncoding regions adjacent to genes (<em>n</em> = 7458), but we also identified 134 exonic fixed differences corresponding to 116 synonymous and 18 nonsynonymous positions, and 17 fixed differences potentially affecting splice signals. The genes associated with the fixed differences at nonsynonymous positions or splice sites are all potential candidates for underlying species-specific characteristics of <em>abietinus</em> and <em>tristis</em>, respectively, and are presented in Table S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>. Two of the genes had more than one nonsynonymous substitution, and, based on functional studies in other organisms, these may be involved in the innate immune response (<em>FYB</em>) and lipid metabolism (<em>PRUNE2</em>). Another two genes had both nonsynonymous and splice altering differences, and these genes have functions related to gene expression regulation via histone methylation (<em>KDM4C</em>), microtubule activity, and intracellular transport (<em>KIF24</em>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bar plots showing the relative frequencies of shared (n = 1,224,635; green), private [yellow (n = 734,710) and brown (n = 843,724) for abietinus and tristis, respectively) and fixed (n = 7609; gray) polymorphisms between allopatric abietinus and tristis. SNPs on the autosomes are presented in the top panel, and Z-chromosome linked SNPs are presented in the bottom panel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-523979459" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Bar plots showing the relative frequencies of shared (&lt;em&gt;n&lt;/em&gt; = 1,224,635; green), private [yellow (&lt;em&gt;n&lt;/em&gt; = 734,710) and brown (&lt;em&gt;n&lt;/em&gt; = 843,724) for &lt;em&gt;abietinus&lt;/em&gt; and &lt;em&gt;tristis&lt;/em&gt;, respectively) and fixed (&lt;em&gt;n&lt;/em&gt; = 7609; gray) polymorphisms between allopatric &lt;em&gt;abietinus&lt;/em&gt; and &lt;em&gt;tristis&lt;/em&gt;. SNPs on the autosomes are presented in the top panel, and Z-chromosome linked SNPs are presented in the bottom panel.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/3983/F8.medium.gif" width="298" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/7/12/3983/F8.medium.gif" width="298" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/3983/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487496" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-38" class="first-child">Bar plots showing the relative frequencies of shared (<em>n</em> = 1,224,635; green), private [yellow (<em>n</em> = 734,710) and brown (<em>n</em> = 843,724) for <em>abietinus</em> and <em>tristis</em>, respectively) and fixed (<em>n</em> = 7609; gray) polymorphisms between allopatric <em>abietinus</em> and <em>tristis</em>. SNPs on the autosomes are presented in the top panel, and Z-chromosome linked SNPs are presented in the bottom panel.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-39"><a id="xref-ref-105-7" class="xref-bibr" href="#ref-105">Shipilina <em>et al.</em> (2017)</a> used nuclear SNP information from one of the sequencing batches and restriction data from a single mtDNA gene sequence (<em>CYTB</em>) in the same set of chiffchaff samples. They investigated population structure in <em>abietinus</em> and <em>tristis</em> in relation to phenotypic variation in allopatry and sympatry, and verified that the intermediate phenotypes and mixed singers observed in the area around the Ural mountains represented admixed individuals rather than a third taxon previously suggested to be present in the area (<a id="xref-ref-105-8" class="xref-bibr" href="#ref-105">Shipilina <em>et al.</em> 2017</a>). Here, we went one step further by developing a reference-assisted genome assembly, and using the complete mtDNA genome and higher coverage nuclear genomic resequencing data to generate quantitative information about genome-wide patterns of regional variation in genetic polymorphism, divergence, and differentiation. The polymorphism data were used to investigate how hybridization in secondary contact affects patterns of genome differentiation and to identify regions under diversifying selection, potentially involved in reproductive barriers between the species. The chiffchaff assembly covered ∼81% of the genome (as estimated from comparison to the collared flycatcher, <em>F. albicollis</em>), and had sufficient quality to use as a backbone for mapping reads from resequencing efforts involving allopatric and sympatric populations of <em>abietinus</em> and <em>tristis</em>. The level of complete and partial coverage of conserved gene sets was, for example, similar to high-quality avian genome assemblies (<em>F. albicollis</em> and <em>C. corone</em>).</p><p id="p-40">Initially, we used the new polymorphism data to assess population structure and genomic variation within and between <em>abietinus</em> and <em>tristis</em> to have a thorough background for more detailed analyses of genomic regions that showed patterns of differentiation consistent with diversifying selection and/or genetic incompatibilities. Our estimates of genetic diversity, genetic differentiation, allele frequency distributions, LD, and population structure for both mtDNA and nuclear DNA supported the scenario suggested by <a id="xref-ref-105-9" class="xref-bibr" href="#ref-105">Shipilina <em>et al.</em> (2017)</a>. The two species were most likely isolated in allopatry and accumulated novel variants and allele frequency changes, which was followed by secondary contact and interspecific gene-flow when distribution ranges expanded, resulting in reduced global differentiation (<a id="xref-ref-105-10" class="xref-bibr" href="#ref-105">Shipilina <em>et al.</em> 2017</a>; <a id="xref-ref-72-3" class="xref-bibr" href="#ref-72">Marova <em>et al.</em> 2013</a>). We observed only minor differences in the levels of genetic diversity across populations, with allopatric <em>abietinus</em> showing slightly higher proportion of private polymorphisms and a higher genetic diversity, but a higher proportion of low frequency variants (more negative Tajima’s <em>D</em>) than allopatric <em>tristis</em>, despite the smaller sampling range of <em>abietinus</em> than <em>tristis</em> (<a id="xref-ref-105-11" class="xref-bibr" href="#ref-105">Shipilina <em>et al.</em> 2017</a>). This may reflect a more severe or longer bottleneck, and a less dramatic expansion in <em>tristis</em> during and after glacial cycles. In comparison to a well-studied avian study system with potentially similar demographic histories—the pied (<em>Ficedula hypoleuca</em>) and the collared flycatcher (<em>F. albicollis</em>), where average nucleotide diversity ranges from 3.2 × 10<sup>−3</sup> to 4.0 × 10<sup>−3</sup> (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Burri <em>et al.</em> 2015</a>)—the diversity levels were 10–20% higher in chiffchaff, suggesting higher long-term effective population sizes, potentially as a result of less dramatic contractions during glacial maxima. In sympatry, there was a reduction in the fraction of private polymorphisms, which was more pronounced in <em>abietinus</em> than in <em>tristis</em>, and the mtDNA divergence in intraspecific comparisons of allopatric and sympatric populations was higher in <em>abietinus</em> than in <em>tristis</em>. Furthermore, sympatric <em>abietinus</em> had a higher Tajima’s <em>D</em>, indicating a lower proportion of low frequency alleles. This suggests that sympatric <em>abietinus</em>-like birds also harbor <em>tristis</em> nuclear and mtDNA alleles, which is in line with previous analyses (<a id="xref-ref-105-12" class="xref-bibr" href="#ref-105">Shipilina <em>et al.</em> 2017</a>). Taken together, this portrays a scenario where secondary contact and recurrent hybridization and back-crossing has generated a reduction in genome-wide differentiation in sympatry, and that the external appearance of an individual does not necessarily reflect genomic composition. Interestingly, this indicates that individual phenotype is not dependent on the effect of many genes scattered throughout the genome, but rather that relatively few genes may underlie the observed differences between the species. Having this background knowledge provides a good position from which to investigate regional variation in diversity, differentiation, and divergence to draw conclusions on specific processes underlying the observed patterns in allopatry and sympatry.</p><p id="p-41">There was a considerable reduction in genetic diversity (<em>Ɵ<sub>π</sub></em>) on the Z-chromosome as compared to the autosomes in both <em>abietinus</em> and <em>tristis</em> and the level of genetic differentiation (<em>F<sub>ST</sub></em>) was on average higher on the Z-chromosome in both allopatry and sympatry. Interestingly, the level of absolute divergence (as estimated by <em>D<sub>XY</sub></em>) was considerably reduced on the Z-chromosome as compared to the autosomes, similar to previous observations in the greenish warbler species complex (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Irwin <em>et al.</em> 2016</a>). This indicates that lower postdivergence gene-flow has not been the main force affecting the increase in differentiation on the Z-chromosome, but rather that selection in the ancestral lineage has resulted in reduced diversity prior to the split between <em>abietinus</em> and <em>tristis</em> (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Cruickshank and Hahn 2014</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Nachman and Payseur 2012</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Irwin <em>et al.</em> 2016</a>). The overall elevated level of differentiation on the Z-chromosome is a frequently observed pattern (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Burri <em>et al.</em> 2015</a>; <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Ellegren <em>et al.</em> 2012</a>; <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Irwin <em>et al.</em> 2016</a>; <a id="xref-ref-129-6" class="xref-bibr" href="#ref-129">Wolf and Ellegren 2017</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Oyler-McCance <em>et al.</em> 2015</a>); however, several characteristics of the sex-chromosomes (<a id="xref-ref-122-1" class="xref-bibr" href="#ref-122">Vicoso and Charlesworth 2006</a>) contribute to difficulties in the interpretation of the exact mechanisms underlying the observed patterns on the Z-chromosome. First, the Z-chromosome does not recombine in females (except for the pseudoautosomal region, PAR), and, therefore, it is expected that the overall recombination rate will be reduced on the Z-chromosome as compared to the autosomes of similar size, which do recombine in both males and females (<a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Irwin <em>et al.</em> 2016</a>; <a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Qvarnström and Bailey 2009</a>). This results in lower efficiency of selection in general (reducing <em>N<sub>e</sub></em> and increasing the effect of drift), and should also increase the effect of linked selection on the Z-chromosome. Second, it is plausible that the Z-chromosome harbors a gene set that is nonrepresentative for the genome as a whole, <em>e.g.</em>, enriched for genes involved in local adaptation or reproductive isolation, as has been suggested for the X-chromosome in <em>Drosophila</em> (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Presgraves 2008</a>). We know, for example, from previous studies in <em>Ficedula</em> flycatchers, that the Z-chromosome plays a particularly important role for traits under sexual selection and species recognition (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Ellegren <em>et al.</em> 2012</a>; <a id="xref-ref-92-2" class="xref-bibr" href="#ref-92">Qvarnström and Bailey 2009</a>; <a id="xref-ref-93-2" class="xref-bibr" href="#ref-93">Qvarnström <em>et al.</em> 2010</a>; <a id="xref-ref-98-2" class="xref-bibr" href="#ref-98">Sætre and Sæther 2010</a>), and this could potentially be translated also to the situation in chiffchaff, since both song and plumage are important signals for mate choice and species recognition in <em>Phylloscopus</em> warblers (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Lyu <em>et al.</em> 2016</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arvidsson and Neergaard 1991</a>), including <em>abietinus</em> and <em>tristis</em> (<a id="xref-ref-68-4" class="xref-bibr" href="#ref-68">Marova <em>et al.</em> 2017</a>). Third, the effective population size (<em>N<sub>e</sub></em>) of the Z-chromosome is reduced as compared to autosomes. With an equal sex-ratio, <em>N<sub>e</sub></em> for the Z-chromosome is 3/4 (75%) of the <em>N<sub>e</sub></em> for autosomes, since females only have a single Z-chromosome (in birds, male is the homogametic sex and harbors two Z chromosomes, and female is the heterogametic sex with one Z and one W chromosome). In birds, males also generally have higher variance in reproductive output than females, further reducing the effective population size of the Z-chromosome and increasing the effect of genetic drift. Fourth, since the Z-chromosome is in hemizygous state in female birds, recessive mutations are expressed in females, and this can contribute to both faster evolution (<a id="xref-ref-123-1" class="xref-bibr" href="#ref-123">Vicoso and Charlesworth 2009</a>; <a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Sackton <em>et al.</em> 2014</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Mank <em>et al.</em> 2007</a>, <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">2010</a>) and more pronounced antagonistic effects across sexes, leading to sex-biased expression patterns (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Mank 2009</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ellegren and Parsch 2007</a>). Finally, since the Z-chromosome spends 2/3 of the time (in an evolutionary timescale perspective) in the male germline, it is possible that mutation rates are higher on the Z-chromosome than on the autosomes due to male-biased mutation (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Ellegren and Fridolfsson 1997</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Hurst and Ellegren 1998</a>; <a id="xref-ref-128-1" class="xref-bibr" href="#ref-128">Wilson Sayres and Makova 2011</a>). The observed patterns in chiffchaff suggest that the most plausible explanation is that genetic drift is a major force affecting allele frequency changes on the Z-chromosome, and that directional selection affects larger regions due to linkage, and these forces have increased differentiation across the entire chromosome. Mutation bias, however, probably has had only a minor effect (if any), since absolute divergence is considerably reduced as compared to the autosomes. In addition, selection has likely been a key force reducing overall diversity on the Z-chromosome prior to the split between <em>abietinus</em> and <em>tristis</em>, and followed by accumulation of allele frequency changes due to strong effects of drift as a consequence of combined effects of the specific features of the Z-chromosome listed above. Hence, there is no evidence in our allele frequency data for more restricted gene-flow on the Z-chromosome, which is also supported by the fact that no fixed differences were observed on the Z-chromosome when comparing <em>abietinus</em> and <em>tristis</em> in the sympatric zone.</p><p id="p-42">Similar to the observations when comparing the Z-chromosome to the autosomes, the highly differentiated regions across chiffchaff species in general showed reduced diversity and absolute divergence. There was also a strong correlation between absolute divergence and genetic diversity. Again, similar to the situation for the Z-chromosome, this may be translated to a scenario where linked selection and regional lowered effective population size has resulted in loss of genetic variation prior to the divergence of the two lineages (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Cruickshank and Hahn 2014</a>; <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Nachman and Payseur 2012</a>). The levels of differentiation are expected to vary regionally as a consequence of stochastic allele frequency changes, also when the level of selection is equal across regions in the genome, especially if populations are structured (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Baird 2017</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Lohse 2017</a>). However, we observed very distinct and narrow differentiation peaks, especially in sympatry, which indicates that additional factors affect the differentiation landscape between chiffchaff species. One plausible explanation is that the recombination rate varies considerably across regions in the chiffchaff genome, similar to what has been observed in other bird species (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Kawakami <em>et al.</em> 2014</a>; <a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">Singhal <em>et al.</em> 2015</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Backström <em>et al.</em> 2010</a>). This leads to reduced effective population size in regions with low recombination rate, allowing for more rapid allele frequency shifts across species due to genetic drift in those regions (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Burri <em>et al.</em> 2015</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Burri 2017a</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">b</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Ellegren and Wolf 2017</a>; <a id="xref-ref-94-3" class="xref-bibr" href="#ref-94">Ravinet <em>et al.</em> 2017</a>; <a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Cruickshank and Hahn 2014</a>; <a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Nachman and Payseur 2012</a>; <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Noor and Bennett 2009</a>). We do not have access to recombination data for the chiffchaff, but several recent studies indicate that recombination landscapes may be conserved across divergent avian lineages (<a id="xref-ref-108-2" class="xref-bibr" href="#ref-108">Singhal <em>et al.</em> 2015</a>), leading to similar patterns of genetic diversity and differentiation across independent species pair comparisons (<a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">Burri <em>et al.</em> 2015</a>; <a id="xref-ref-47-5" class="xref-bibr" href="#ref-47">Irwin <em>et al.</em> 2016</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Dutoit <em>et al.</em> 2017</a>; <a id="xref-ref-119-1" class="xref-bibr" href="#ref-119">Van Doren <em>et al.</em> 2017</a>; <a id="xref-ref-125-1" class="xref-bibr" href="#ref-125">Vijay <em>et al.</em> 2017</a>). In support of that, when visually comparing the location of differentiation islands identified in the chiffchaff, we see a considerable overlap with for example differentiation islands in <em>Ficedula</em> flycatchers (<a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">Burri <em>et al.</em> 2015</a>; <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Ellegren <em>et al.</em> 2012</a>). Hence, although we acknowledge that demographic history definitely affects differentiation patterns, our data suggest that the highly heterogenic landscape of genome differentiation between <em>abietinus</em> and <em>tristis</em> can be traced back to mechanistic underpinnings (<em>e.g.</em>, recombination rate variation) determining the difference in relative importance of genetic drift and selection in different parts of the genome, and that a major proportion of the fixed differences we observe are the result of stochastic fixation of alternative alleles in the two species in allopatry. However, the observation that a larger proportion of autosomal fixed differences are present in exonic sequence or adjacent to genes indicates that at least a proportion of the fixed differences on the autosomes result from selection driving alternative alleles to fixation in <em>abietinus</em> and <em>tristis</em>, respectively. In addition, it is possible that stochastic allele frequency changes and random accumulation of fixed differences may lead to diverging phenotypes between species in allopatry, and it is therefore of interest to investigate the functional relevance of observed differences between species, even if it is apparent that the major fraction of these were not initially driven to fixation by adaptation to different environments (natural selection and ecological speciation) or by different preferences for secondary sexual traits (sexual selection). In the following paragraph, we discuss the detailed analyses of functions of genes in highly differentiated regions, and genes with alternative alleles fixed between the two species.</p><p id="p-43">The common and the Siberian chiffchaff differ in morphology, plumage color, song, and migration patterns—traits that most likely are of importance for species recognition and lineage specific adaptations (<a id="xref-ref-68-5" class="xref-bibr" href="#ref-68">Marova <em>et al.</em> 2017</a>; <a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Marova and Alekseev 2008</a>; <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Marova and Leonovich 1993</a>; <a id="xref-ref-72-4" class="xref-bibr" href="#ref-72">Marova <em>et al.</em> 2013</a>; <a id="xref-ref-105-13" class="xref-bibr" href="#ref-105">Shipilina <em>et al.</em> 2017</a>). Gene-flow between the species has been shown to lead to intermediate or mixed traits in hybrids and back-crosses (<a id="xref-ref-68-6" class="xref-bibr" href="#ref-68">Marova <em>et al.</em> 2017</a>; <a id="xref-ref-69-4" class="xref-bibr" href="#ref-69">Marova and Alekseev 2008</a>; <a id="xref-ref-105-14" class="xref-bibr" href="#ref-105">Shipilina <em>et al.</em> 2017</a>), and characterization of the genetic basis of these traits is of importance to understand species integrity and the evolutionary effects of introgression. We conducted several analyses to detect genes that might be of interest for understanding phenotypic differences between <em>abietinus</em> and <em>tristis</em> that may reflect species specific adaptations. First, we selected all genes that were located in genomic regions enriched for relative differentiation outliers, and analyzed if any functional categories were overrepresented in this class of genes. This analysis showed no significant enrichment in the allopatric comparison and a single term (regulation of cellular organization) in sympatry. Such an analysis is characteristically rough, and, although it may give some preliminary insights into gene categories associated with high differentiation, it is difficult to draw detailed conclusions regrading functional relevance (<a id="xref-ref-124-2" class="xref-bibr" href="#ref-124">Vijay <em>et al.</em> 2016</a>). To get more detailed information about genes in highly differentiated regions, we also selected the windows with the highest level of differentiation (<em>F<sub>ST</sub></em>) within each of the regions enriched for relative differentiation (<em>F<sub>ST</sub><sup>Z</sup></em>) outliers, and found that these top windows overlapped with 26 protein coding genes. The genes were mainly involved in functions related to ion transport, expression regulation (transcription factors, histone binding, and phosphorylation), and muscle activity/development, but also associated with processes directly relevant to phenotypic traits of interest in our study system; for example, sensory perception of sound (<em>OTOG</em>), and pigmentation (<em>HPS5</em>) (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). Both <em>OTOG</em> and <em>HPS5</em> are located in the peak of the differentiation island on chromosome 5. <em>OTOG</em> has been associated with sound perception and locomotion in mouse (<em>Mus musculus domesticus</em>) (<a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">Simmler <em>et al.</em> 2000</a>), and otolith function (hearing and balance organ development) in zebra fish (<em>Danio rerio</em>) (<a id="xref-ref-111-1" class="xref-bibr" href="#ref-111">Stooke-Vaughan <em>et al.</em> 2015</a>), indicating a conserved function in hearing and balance organs across very divergent lineages. Song recognition is a key mechanism for mate choice in birds (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Nowicki and Searcy 2005</a>), and recent data indicate that allopatric <em>abietinus</em> and <em>tristis</em> only react to conspecific song (<em>e.g.</em>, <a id="xref-ref-68-7" class="xref-bibr" href="#ref-68">Marova <em>et al.</em> 2017</a>). It is therefore tempting to contemplate that genes involved in sensory perception of sound may play an important role for driving differences in song preference, and song learning processes, in vocally differentiated bird species like <em>abietinus</em> and <em>tristis</em>. <em>HPS5</em> is involved in biogenesis of organelles, and has been associated with melanocyte formation and pigmentation in zebra fish (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Daly <em>et al.</em> 2013</a>). Plumage pigmentation is also a key trait for mate recognition and speciation in birds (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Price 2008</a>), and, again, it is enticing to speculate around a potential role for this gene underlying specific plumage patterning in chiffchaff. As mentioned before, both <em>OTOG</em> and <em>HPS5</em> are found in the peak of the differentiation island on chromosome 5. If selection has played any part in the high allele frequency difference between <em>abietinus</em> and <em>tristis</em> in this region, one cannot rule out that genetic hitchhiking has resulted in reduced nucleotide diversity both in the selected region and in linked genes. Therefore, verification experiments will be needed to get detailed information about the phenotypic effects of variation within these genes.</p><p id="p-44">Finally, we also analyzed all fixed differences observed between allopatric <em>abietinus</em> and <em>tristis</em> and selected the positions where the substitutions resulted in amino acid and splice signaling changes. Again, we found that several genes were related to metabolism and gene expression (transcription factors, RNA processing, and methylation), but there was a diverse array of other functions in this class of genes. Of particular interest were two genes that had multiple (<em>n</em> = 2) fixed differences resulting in nonsynonymous changes; <em>FYB</em> and <em>PRUNE2</em>. <em>FYB</em> has been shown to be involved in the innate immune response by stimulating interleukin-2 expression in T-cells in the mouse (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Krause <em>et al.</em> 2000</a>), while <em>PRUNE2</em> has been associated with many different functions (<a href="http://www.genecards.org/cgi-bin/carddisp.pl?gene=PRUNE2">http://www.genecards.org/cgi-bin/carddisp.pl?gene=PRUNE2</a>); the gene is, for example, associated with pyrophosphatase activity that could indicate involvement in lipid metabolism. Two additional genes contained both a nonsynonymous and a splice-altering fixed difference; <em>KDM4C</em> and <em>KIF24</em>. <em>KDM4C</em> regulates gene expression by targeting trimethylated histones in humans (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Hillringhaus <em>et al.</em> 2011</a>), and <em>KIF24</em> is a kinesin that is a microtubule remodelling ATPase likely involved in intracellular transport (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Kobayashi <em>et al.</em> 2011</a>). From these analyses combined, we hypothesize that some of the genes located in highly differentiated regions, or containing fixed differences at nonsynonymous and splice-altering sites, might be relevant to the observed morphological, ecological, and behavioral differences between <em>abietinus</em> and <em>tristis</em>. However, detailed additional studies are obviously needed before any of these can be considered strong candidates for relevant trait differences. Further ahead still is the identification of potential links between candidate genes for trait variation and reproductive isolating mechanisms, and it is unlikely that standard genome scan approaches can be used to trace such links (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Baird 2017</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Buerkle 2017</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Ellegren and Wolf 2017</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Elmer 2017</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Feder <em>et al.</em> 2017</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Jiggins and Martin 2017</a>; <a id="xref-ref-94-4" class="xref-bibr" href="#ref-94">Ravinet <em>et al.</em> 2017</a>; <a id="xref-ref-126-1" class="xref-bibr" href="#ref-126">Wagner and Mandeville 2017</a>; <a id="xref-ref-127-2" class="xref-bibr" href="#ref-127">Westram and Ravinet 2017</a>). We foresee that the next step will be more extensive sampling within the sympatric region, including birds with intermediate phenotypes and using detailed morphological and behavioral studies, combined with resequencing efforts of candidate regions, to find significant associations between specific alleles and phenotypic traits of interest.</p><p id="p-45">It should be noted that a reference-assisted assembly cannot detect structural changes between the target organism and the reference. A consequence might be that visual inspection of patterns of genetic variation might be affected. Since birds in general have conserved karyotypes and comparatively few large structural rearrangements (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Ellegren 2009</a>), this is likely not a major concern, especially when visual inspection is combined with analyses that do not depend on genomic location. As mentioned above, patterns of genetic diversity and differentiation in chiffchaff are remarkably consistent with observations in other avian taxa, but a detailed inspection of the genome-wide patterns reveals that a few rearrangements probably explain some observed abrupt changes in the patterns of genetic variation along chiffchaff chromosomes. This concerns inversions ranging from 4 to 25 Mb in chromosomes 3, 4, 5, 20, and Z (Figure S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1/FileS1.pdf">File S1</a>). The structural changes do not affect the qualitative interpretation of the results in this study since they constitute but a minor part of the genome, and do not overlap with any of the major differentiation islands observed.</p><p id="p-46">To conclude, this study contributes to the increasing body of knowledge about genome divergence processes under natural settings, and highlights the chiffchaff species complex as an important model system to study the genomics of speciation and adaptation in songbirds. In a general perspective, this work will help in the overall understanding of the speciation process, and, in particular, how secondary contact affects patterns of genome differentiation between diverging lineages.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-47">N.B. is supported by a junior research grant from the Swedish Research Council (VR 2013-4508). We acknowledge additional funding for DNA sequencing from Kungliga Fysiografiska Sällskapet i Lund (Nilsson-Ehle Donations), Helge Ax:son Johnsons Foundation, and the Lars Hierta Memorial Foundation. The National Genomics Infrastructure in Uppsala and Stockholm performed the library preparations and the sequencing supported by Science for Life Laboratory; a national infrastructure funded by the Swedish Research Council and the Knut and Alice Wallenberg Foundation. The computations were performed on resources provided by SNIC through Uppsala Multidisciplinary Center for Advanced Computational Science (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Lampa <em>et al.</em> 2013</a>) under Project b2013146. Sample collection and primary sample processing was supported by the Russian Foundation of Basic Research (14-04-01259, 15-29-02771) and the Russian Science Foundation (14-50-00029) to I.M. The authors declare no financial or nonfinancial competing interests.</p><p id="p-48">Author contributions: V.T. performed the majority of all analyses and wrote the paper together with N.B. with input from all coauthors. F.K. performed the reference-assisted genome assembly, variant calling steps, and was involved in the LD analysis with help from V.T. D.S. performed the DNA extractions together with N.B. and performed parts of the analyses. I.M. provided samples and input on the writing of the manuscript. N.B. conceived of the study, performed some data analyses, and wrote the manuscript together with V.T. with input from all coauthors.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-49">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300152/-/DC1</a>.</p></li><li class="fn" id="fn-5"><p id="p-50"><em>Communicating editor: J. Comeron</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-08-14"><span class="received-label">Received </span>August 14, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-10-20"><span class="accepted-label">Accepted </span>October 20, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Talla <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.1" data-doi="10.1111/mec.13685"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abbott</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Good</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomics of hybridization and its evolutionary consequences.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2325</span>–<span class="cit-lpage">2332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D2325%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13685%26rft_id%253Dinfo%253Apmid%252F27145128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13685&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27145128&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.2" data-doi="10.1007/BF00166406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arvidsson</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neergaard</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Mate choice in the willow warbler—a field experiment.</span> <abbr class="cit-jnl-abbrev">Behav. Ecol. Sociobiol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehav.%2BEcol.%2BSociobiol.%26rft.volume%253D29%26rft.spage%253D225%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00166406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00166406&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991GG90400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.3" data-doi="10.1101/gr.101410.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Backström</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forstmeier</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schielzeth</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mellenius</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nam</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The recombination landscape of the zebra finch <em>Taeniopygia guttata</em> genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">485</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.101410.109%26rft_id%253Dinfo%253Apmid%252F20357052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzQvNDg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zOTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">S. J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The impact of high-throughput sequencing technology on speciation research: maintaining perspective.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1482</span>–<span class="cit-lpage">1487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D30%26rft.spage%253D1482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.5" data-doi="10.1038/341497a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hewitt</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Adaptation, speciation and hybrid zones.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">341</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">503</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBarton%26rft.auinit1%253DN.%2BH.%26rft.volume%253D341%26rft.issue%253D6242%26rft.spage%253D497%26rft.epage%253D503%26rft.atitle%253DAdaptation%252C%2Bspeciation%2Band%2Bhybrid%2Bzones.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F341497a0%26rft_id%253Dinfo%253Apmid%252F2677747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/341497a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2677747&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AU72100044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-7.12.3983.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bateson</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1909</span> <span class="cit-article-title">Heredity and variation in modern lights</span>, pp. <span class="cit-fpage">85</span>–<span class="cit-lpage">101</span> in <span class="cit-source">Darwin and Modern Science</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Seward</span>  <span class="cit-name-given-names">A. C.</span></span>. <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Renaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whiteley</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derome</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeukens</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">On the origin of species: insights from the ecological genomics of whitefish.</span> <abbr class="cit-jnl-abbrev">Phil. Trans. Roy. Soc. B: Biol.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">1783</span>–<span class="cit-lpage">1800</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhil.%2BTrans.%2BRoy.%2BSoc.%2BB%253A%2BBiol.%26rft.volume%253D365%26rft.spage%253D1783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.8" data-doi="10.1016/j.ympev.2012.08.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donath</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juhling</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Externbrink</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Florentz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">MITOS: improved <em>de novo</em> metazoan mitochondrial genome annotation.</span> <abbr class="cit-jnl-abbrev">Mol. Phyl. Evol.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhyl.%2BEvol.%26rft.volume%253D69%26rft.spage%253D313%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2012.08.023%26rft_id%253Dinfo%253Apmid%252F22982435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2012.08.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22982435&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.9" data-doi="10.1038/nature07892"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckling</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maclean</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brockhurst</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colegrave</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Beagle in a bottle.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">824</span>–<span class="cit-lpage">829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBuckling%26rft.auinit1%253DA.%26rft.volume%253D457%26rft.issue%253D7231%26rft.spage%253D824%26rft.epage%253D829%26rft.atitle%253DThe%2BBeagle%2Bin%2Ba%2Bbottle.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07892%26rft_id%253Dinfo%253Apmid%252F19212400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07892&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212400&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263266700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buerkle</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Inconvenient truths in population and speciation genetics point towards a future beyond allele frequencies.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1498</span>–<span class="cit-lpage">1500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D30%26rft.spage%253D1498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burri</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">a</span> <span class="cit-article-title">Dissecting differentiation landscapes: a linked selection’s perspective.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1501</span>–<span class="cit-lpage">1505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D30%26rft.spage%253D1501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burri</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">b</span> <span class="cit-article-title">Linked selection, demography and the evolution of correlated genomic landscapes in birds and beyond.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">3853</span>–<span class="cit-lpage">3856</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D3853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.13" data-doi="10.1101/gr.196485.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burri</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nater</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mugal</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olason</span>  <span class="cit-name-given-names">P. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Linked selection and recombination rate variation drive the evolution of the genomic landscape of differentiation across the speciation continuum of <em>Ficedula</em> flycatchers.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1656</span>–<span class="cit-lpage">1665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.196485.115%26rft_id%253Dinfo%253Apmid%252F26355005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMS8xNjU2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zOTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.14" data-doi="10.1017/S0016672397002954"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The effects of local selection, balanced polymorphism and background selection on equilibrium patterns of genetic diversity in subdivided populations.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D70%26rft.issue%253D2%26rft.spage%253D155%26rft.epage%253D174%26rft.atitle%253DThe%2Beffects%2Bof%2Blocal%2Bselection%252C%2Bbalanced%2Bpolymorphism%2Band%2Bbackground%2Bselection%2Bon%2Bequilibrium%2Bpatterns%2Bof%2Bgenetic%2Bdiversity%2Bin%2Bsubdivided%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672397002954%26rft_id%253Dinfo%253Apmid%252F9449192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672397002954&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9449192&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071240500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.15" data-doi="10.1126/science.1249998"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loehlin</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dufour</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vacarro</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millar</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A single gene affects both ecological divergence and mate choice in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">343</span>: <span class="cit-fpage">1148</span>–<span class="cit-lpage">1151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1249998%26rft_id%253Dinfo%253Apmid%252F24526311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDMvNjE3NS8xMTQ4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zOTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.16" data-doi="10.4161/fly.19695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%2B%2528Austin%2529%26rft.volume%253D6%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728672&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.17" data-doi="10.1126/sciadv.1501005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Claramunt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cracraft</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A new time tree reveals earth history’s imprint on the evolution of modern birds.</span> <abbr class="cit-jnl-abbrev">Sci. Adv.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e1501005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DClaramunt%26rft.auinit1%253DS.%26rft.volume%253D1%26rft.issue%253D11%26rft.spage%253De1501005%26rft.epage%253De1501005%26rft.atitle%253DA%2Bnew%2Btime%2Btree%2Breveals%2BEarth%2Bhistorys%2Bimprint%2Bon%2Bthe%2Bevolution%2Bof%2Bmodern%2Bbirds%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.1501005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMzoiMS8xMS9lMTUwMTAwNSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTIvMzk4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-7.12.3983.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coyne</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-source">Speciation</span>. <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.19" data-doi="10.1111/mec.12796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cruickshank</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Reanalysis suggests that genomic islands of speciation are due to reduced diversity, not reduced gene flow.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">3133</span>–<span class="cit-lpage">3157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D3133%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12796%26rft_id%253Dinfo%253Apmid%252F24845075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24845075&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000338014900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.20" data-doi="10.1534/genetics.113.154898"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gregg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>Snow white</em>, a zebrafish model of Hermansky-Pudlak Syndrome type 5.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.154898%26rft_id%253Dinfo%253Apmid%252F23893484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzIvNDgxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zOTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.21" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinfo.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinfo.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-7.12.3983.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">del Hoyo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collar</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Siberian chiffchaff (<em>Phylloscopus tristis</em>)</span>, in <span class="cit-source">Handbook of the Birds of the World Alive</span>, edited by <span class="cit-ed"><span class="cit-name-surname">del Hoyo</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Elliott</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Sargatal</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Christie</span>  <span class="cit-name-given-names">D. A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">de Juana</span>  <span class="cit-name-given-names">E.</span></span>. <span class="cit-publ-name">Lynx Edicions</span>, <span class="cit-publ-loc">Barcelona, Spain</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-7.12.3983.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">del Hoyo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elliot</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christie</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source">Handbook of the Birds of the World. Old World Flycatchers to Old World Warblers</span>. <span class="cit-publ-name">Lynx Edicions</span>, <span class="cit-publ-loc">Barcelona, Spain</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.24" data-doi="10.1086/280899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobzhansky</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1940</span> <span class="cit-article-title">Speciation as a stage in evolutionary divergence.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">312</span>–<span class="cit-lpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D74%26rft.spage%253D312%26rft_id%253Dinfo%253Adoi%252F10.1086%252F280899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/280899&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000188901200032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dutoit</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burri</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nater</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mugal</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic distribution and estimation of nucleotide diversity in natural populations: perspectives from the collared flycatcher (<em>Ficedula albicollis</em>) genome.</span> <abbr class="cit-jnl-abbrev">Moil. Ecol. Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">586</span>–<span class="cit-lpage">597</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMoil.%2BEcol.%2BRes.%26rft.volume%253D17%26rft.spage%253D586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.26" data-doi="10.1111/ele.12460"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Egan</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ragland</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Assour</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hood</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Experimental evidence of genome-wide impact of ecological selection during early stages of speciation-with-gene-flow.</span> <abbr class="cit-jnl-abbrev">Ecol. Lett.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">817</span>–<span class="cit-lpage">825</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BLett.%26rft.volume%253D18%26rft.spage%253D817%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12460%26rft_id%253Dinfo%253Apmid%252F26077935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26077935&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evolutionary stasis: the stable chromosomes of birds.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D25%26rft.spage%253D283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.28" data-doi="10.1038/ng1097-182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fridolfsson</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Male-driven evolution of DNA sequences in birds.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">182</span>–<span class="cit-lpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DEllegren%26rft.auinit1%253DH.%26rft.volume%253D17%26rft.issue%253D2%26rft.spage%253D182%26rft.epage%253D184%26rft.atitle%253DMale-driven%2Bevolution%2Bof%2BDNA%2Bsequences%2Bin%2Bbirds.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1097-182%26rft_id%253Dinfo%253Apmid%252F9326938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1097-182&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9326938&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XZ55500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.29" data-doi="10.1038/nrg2167"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsch</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The evolution of sex-biased genes and sex-biased gene expression.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">689</span>–<span class="cit-lpage">698</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DEllegren%26rft.auinit1%253DH.%26rft.volume%253D8%26rft.issue%253D9%26rft.spage%253D689%26rft.epage%253D698%26rft.atitle%253DThe%2Bevolution%2Bof%2Bsex-biased%2Bgenes%2Band%2Bsex-biased%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2167%26rft_id%253Dinfo%253Apmid%252F17680007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17680007&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248882400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">J. B. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Parallelism in genomic landscapes of differentiation, conserved genomic features and the role of linked selection.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1516</span>–<span class="cit-lpage">1518</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D30%26rft.spage%253D1516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.31" data-doi="10.1038/nature11584"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smeds</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burri</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olason</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Backström</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genomics of species differentiation in <em>Ficedula</em> flycatchers.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">756</span>–<span class="cit-lpage">760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DEllegren%26rft.auinit1%253DH.%26rft.volume%253D491%26rft.issue%253D7426%26rft.spage%253D756%26rft.epage%253D760%26rft.atitle%253DThe%2Bgenomic%2Blandscape%2Bof%2Bspecies%2Bdivergence%2Bin%2BFicedula%2Bflycatchers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11584%26rft_id%253Dinfo%253Apmid%252F23103876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11584&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23103876&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311606000045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elmer</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Barrier loci and progress towards evolutionary generalities.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1491</span>–<span class="cit-lpage">1493</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D30%26rft.spage%253D1491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feder</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nosil</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The efficacy of divergence hitch-hiking in generating genomic islands during ecological speciation.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">1724</span>–<span class="cit-lpage">1747</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D64%26rft.spage%253D1724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.34" data-doi="10.1016/j.tig.2012.03.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feder</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egan</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nosil</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genomics of speciation-with-gene-flow.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DFeder%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D28%26rft.issue%253D7%26rft.spage%253D342%26rft.epage%253D350%26rft.atitle%253DThe%2Bgenomics%2Bof%2Bspeciation-with-gene-flow.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2012.03.009%26rft_id%253Dinfo%253Apmid%252F22520730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2012.03.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22520730&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306252300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.35" data-doi="10.1146/annurev-ecolsys-110512-135825"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feder</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flaxman</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egan</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comeault</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nosil</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Geographic mode of speciation and genomic divergence.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">97</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D44%26rft.spage%253D73%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ecolsys-110512-135825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ecolsys-110512-135825&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feder</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nosil</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gompert</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flaxman</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schilling</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Barnacles, barrier loci and the systematic building of species.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1494</span>–<span class="cit-lpage">1497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D30%26rft.spage%253D1494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferchaud</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">M. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The impact of selection, gene flow and demographic history on heterogeneous genomic divergence: three-spine sticklebacks in divergent environments.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">238</span>–<span class="cit-lpage">259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.38" data-doi="10.1534/genetics.113.160572"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frichot</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Fast and efficient estimation of individual ancestry coefficients.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">973</span>–<span class="cit-lpage">983</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.160572%26rft_id%253Dinfo%253Apmid%252F24496008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzQvOTczIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zOTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frichot</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">LEA: an R package for landscape and ecological association studies.</span> <abbr class="cit-jnl-abbrev">Methods Ecol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">925</span>–<span class="cit-lpage">929</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol.%2BEvol.%26rft.volume%253D6%26rft.spage%253D925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.40" data-doi="10.1111/ele.12238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gompert</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comeault</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farkas</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feder</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parchman</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Experimental evidence for ecological selection on genome variation in the wild.</span> <abbr class="cit-jnl-abbrev">Ecol. Lett.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">369</span>–<span class="cit-lpage">379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BLett.%26rft.volume%253D17%26rft.spage%253D369%26rft.atitle%253DEXPERIMENTAL%2BEVIDENCE%2BFOR%2BECOLOGICAL%2BSELECTION%2BON%2BGENOME%2BVARIATION%2BIN%2BTHE%2BWILD%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12238%26rft_id%253Dinfo%253Apmid%252F24354456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24354456&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.41" data-doi="10.1093/bioinformatics/btt086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saveliev</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vyahhi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tesler</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">QUAST: quality assessment tool for genome assemblies.</span> <abbr class="cit-jnl-abbrev">Bioinfo.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1072</span>–<span class="cit-lpage">1075</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinfo.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt086%26rft_id%253Dinfo%253Apmid%252F23422339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23422339&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318109300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.42" data-doi="10.1093/nar/gkt371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Reconstructing mitochondrial genomes directly from genomic next-generation sequencing reads—a baiting and iterative mapping approach.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">e129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt371%26rft_id%253Dinfo%253Apmid%252F23661685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23661685&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.43" data-doi="10.1101/gr.5045006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harr</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomic islands of differentiation between house mouse subspecies.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">730</span>–<span class="cit-lpage">737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5045006%26rft_id%253Dinfo%253Apmid%252F16687734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzYvNzMwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zOTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Helbig</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martens</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seibold</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henning</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schottler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Phylogeny and species limits in the Palaearctic chiffchaff <em>Phylloscopus collybita</em> complex: mitochondrial genetic differentiation and bioacoustic evidence.</span> <abbr class="cit-jnl-abbrev">Ibis</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">650</span>–<span class="cit-lpage">666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIbis%26rft.volume%253D138%26rft.spage%253D650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.45" data-doi="10.1074/jbc.M111.283689"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hillringhaus</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yue</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gileadi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Structural and evolutionary basis for the dual substrate selectivity of human KDM4 histone demethylase family.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">41616</span>–<span class="cit-lpage">41625</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M111.283689%26rft_id%253Dinfo%253Apmid%252F21914792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODYvNDgvNDE2MTYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEyLzM5ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.46" data-doi="10.1016/S0168-9525(98)01577-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Sex biases in the mutation rate.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">446</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DHurst%26rft.auinit1%253DL.%2BD.%26rft.volume%253D14%26rft.issue%253D11%26rft.spage%253D446%26rft.epage%253D452%26rft.atitle%253DSex%2Bbiases%2Bin%2Bthe%2Bmutation%2Brate.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252898%252901577-7%26rft_id%253Dinfo%253Apmid%252F9825672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(98)01577-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9825672&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076946100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Irwin</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alcaide</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delmore</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irwin</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Owens</span>  <span class="cit-name-given-names">G. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Recurrent selection explains parallel evolution of genomic regions of high relative but low absolute differentiation in a ring species.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">4488</span>–<span class="cit-lpage">4507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D4488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiggins</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">S. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Glittering gold and the quest for Isla de Muerta.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1509</span>–<span class="cit-lpage">1511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D30%26rft.spage%253D1509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiggins</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">W. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neukirchen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mallet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">What can hybrid zones tell us about speciation? The case of <em>Heliconius erato</em> and <em>H. himera</em> (Lepidoptera: Nymphalidae).</span> <abbr class="cit-jnl-abbrev">Biol. J. Linn. Soc. Lond.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BJ.%2BLinn.%2BSoc.%2BLond.%26rft.volume%253D59%26rft.spage%253D221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1996VU42400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.50" data-doi="10.1038/nature10944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauceli</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genomic basis of adaptive evolution in threespine sticklebacks.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">484</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJones%26rft.auinit1%253DF.%2BC.%26rft.volume%253D484%26rft.issue%253D7392%26rft.spage%253D55%26rft.epage%253D61%26rft.atitle%253DThe%2Bgenomic%2Bbasis%2Bof%2Badaptive%2Bevolution%2Bin%2Bthreespine%2Bsticklebacks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10944%26rft_id%253Dinfo%253Apmid%252F22481358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22481358&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302343400033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.51" data-doi="10.1371/journal.pbio.0040303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joron</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beltran</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mavarez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A conserved supergene locus controls colour pattern diversity in <em>Heliconius</em> butterflies.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DJoron%26rft.auinit1%253DM.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De303%26rft.epage%253De303%26rft.atitle%253DA%2Bconserved%2Bsupergene%2Blocus%2Bcontrols%2Bcolour%2Bpattern%2Bdiversity%2Bin%2BHeliconius%2Bbutterflies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040303%26rft_id%253Dinfo%253Apmid%252F17002517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040303&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17002517&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.52" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Standley</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">772</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23329690&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317002300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.53" data-doi="10.1111/mec.12810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smeds</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Backström</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Husby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qvarnström</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A high-density linkage map enables a second-generation collared flycatcher genome assembly and reveals the patterns of avian recombination rate variation and chromosomal evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4035</span>–<span class="cit-lpage">4058</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D4035%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12810%26rft_id%253Dinfo%253Apmid%252F24863701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12810&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24863701&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340419800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.54" data-doi="10.1016/j.cell.2011.04.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsang</span>  <span class="cit-name-given-names">W. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lane</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dynlacht</span>  <span class="cit-name-given-names">B. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Centriolar kinesin Kif24 interacts with CP110 to remodel microtubules and regulate ciliogenesis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">914</span>–<span class="cit-lpage">925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKobayashi%26rft.auinit1%253DT.%26rft.volume%253D145%26rft.issue%253D6%26rft.spage%253D914%26rft.epage%253D925%26rft.atitle%253DCentriolar%2Bkinesin%2BKif24%2Binteracts%2Bwith%2BCP110%2Bto%2Bremodel%2Bmicrotubules%2Band%2Bregulate%2Bciliogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.04.028%26rft_id%253Dinfo%253Apmid%252F21620453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.04.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21620453&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291461600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.12.3983.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Komarova</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shipilina</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> Hybrid population of eastern European and Siberian forms of chiffchaff (<em>Phylloscopus collybita abietinus</em>, Ph. (c.) tristis) in the Arkhangelsk area., pp. <span class="cit-fpage">211</span>–<span class="cit-lpage">212</span> in <em>XVII International Conference of Students</em>, <em>Post-Graduate Students and Young Scientists</em>. <span class="cit-publ-name">MAKS Press</span>, <span class="cit-publ-loc">Lomonosov</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">ANGSD: analysis of next generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinfo.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinfo.%26rft.volume%253D15%26rft.spage%253D356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.57" data-doi="10.1083/jcb.149.1.181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krause</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sechi</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konradt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gertler</span>  <span class="cit-name-given-names">F. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Fyn-binding protein (Fyb):SLP-76–associated protein (SLAP), Ena: vasodilator-stimulated phosphoprotein (VASP) proteins and the Arp2: 3 complex link T cell receptor (TCR) signaling to the actin cytoskeleton.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.149.1.181%26rft_id%253Dinfo%253Apmid%252F10747096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE0OS8xLzE4MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTIvMzk4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.58" data-doi="10.1093/molbev/msw054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1870</span>–<span class="cit-lpage">1874</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw054%26rft_id%253Dinfo%253Apmid%252F27004904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27004904&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lampa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dahlö</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olason</span>  <span class="cit-name-given-names">P. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hagberg</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spjuth</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Lessons learned from implementing a national infrastructure in Sweden for storage and analysis of next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Gigasci.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigasci.%26rft.volume%253D25%26rft.spage%253D9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.60" data-doi="10.1093/bioinformatics/btp698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and accurate long-read alignment with Burrows-Wheeler transformation.</span> <abbr class="cit-jnl-abbrev">Bioinfo.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinfo.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp698%26rft_id%253Dinfo%253Apmid%252F20080505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20080505&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274973800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.61" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinfo.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinfo.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindholm</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mixed songs of chiffchaffs in northern Russia.</span> <abbr class="cit-jnl-abbrev">Alula</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">108</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAlula%26rft.volume%253D3%26rft.spage%253D108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Come on feel the noise—from metaphors to null models.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1506</span>–<span class="cit-lpage">1508</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D30%26rft.spage%253D1506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lyu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y.-H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Can simple songs express useful signals for mate choice?</span> <abbr class="cit-jnl-abbrev">Avian Res.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAvian%2BRes.%26rft.volume%253D7%26rft.spage%253D10e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.65" data-doi="10.1086/595754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mank</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sex chromosomes and the evolution of sexual dimorphism: lessons from the genome.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DMank%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D173%26rft.issue%253D2%26rft.spage%253D141%26rft.epage%253D150%26rft.atitle%253DSex%2Bchromosomes%2Band%2Bthe%2Bevolution%2Bof%2Bsexual%2Bdimorphism%253A%2Blessons%2Bfrom%2Bthe%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F595754%26rft_id%253Dinfo%253Apmid%252F20374139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/595754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20374139&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262704600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.66" data-doi="10.1101/gr.6031907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mank</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Axelsson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Fast-X on the Z: rapid evolution of sex-linked genes in birds.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">618</span>–<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6031907%26rft_id%253Dinfo%253Apmid%252F17416747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzUvNjE4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zOTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.67" data-doi="10.1111/j.1558-5646.2009.00853.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mank</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vicoso</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berlin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Effective population size and the faster-X effect: empirical results and their interpretation.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">663</span>–<span class="cit-lpage">674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DMank%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D64%26rft.issue%253D3%26rft.spage%253D663%26rft.epage%253D674%26rft.atitle%253DEffective%2Bpopulation%2Bsize%2Band%2Bthe%2BFaster-X%2Beffect%253A%2Bempirical%2Bresults%2Band%2Btheir%2Binterpretation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2009.00853.x%26rft_id%253Dinfo%253Apmid%252F19796145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2009.00853.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19796145&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274713800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marova</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shipilina</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fedorov</span>  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alekseev</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ivanitskii</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Interaction between common and Siberian chiffchaff in a contact zone.</span> <abbr class="cit-jnl-abbrev">Ornis Fenn.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">66</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOrnis%2BFenn.%26rft.volume%253D94%26rft.spage%253D66%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.12.3983.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marova</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alekseev</span>  <span class="cit-name-given-names">V. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> Population structure and distribution of vocal dialects of chiffchaff (<em>Phylloscopus collybita</em>) in southern Ural mountains. Proceedings of the South Ural State Natural Reserve, Vol. 1, pp. <span class="cit-fpage">306</span>–<span class="cit-lpage">318</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marova</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leonovich</span>  <span class="cit-name-given-names">V. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Hybridization between Siberian (<em>Phylloscopus collybita tristis</em>) and east European (<em>Ph. collybita abietinus</em>) chiffchaffs in the area of sympatry. Sbornik Trudov Zoologicheskogo Muzeya</span>. <abbr class="cit-jnl-abbrev">Moskovskogo Gosudarstvennogo Universiteta</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">163</span> (in Russian).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMoskovskogo%2BGosudarstvennogo%2BUniversiteta%26rft.volume%253D30%26rft.spage%253D147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marova</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fedorov</span>  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shipilina</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alekseev</span>  <span class="cit-name-given-names">V. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic and vocal differentiation in hybrid zones of passerine birds: Siberian and European chiffchaffs (<em>Phylloscopus [collybita] tristis</em> and <em>Ph. [c.] abietinus</em>) in the southern Urals.</span> <abbr class="cit-jnl-abbrev">Dokl. Biol. Sci.</abbr> <span class="cit-vol">427</span>: <span class="cit-fpage">384</span>–<span class="cit-lpage">386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDoklady%2Bbiological%2Bsciences%2B%253A%2B%2Bproceedings%2Bof%2Bthe%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUSSR%252C%2B%2BBiological%2Bsciences%2Bsections%2B%252F%2Btranslated%2Bfrom%2BRussian%26rft.stitle%253DDokl%2BBiol%2BSci%26rft.aulast%253DMarova%26rft.auinit1%253DI.%2BM.%26rft.volume%253D427%26rft.spage%253D384%26rft.epage%253D386%26rft.atitle%253DGenetic%2Band%2Bvocal%2Bdifferentiation%2Bin%2Bhybrid%2Bzones%2Bof%2Bpasserine%2Bbirds%253A%2BSiberian%2Band%2BEuropean%2Bchiffchaffs%2B%2528Phylloscopus%2B%255Bcollybita%255D%2Btristis%2Band%2BPh.%2B%255Bc.%255D%2Babietinus%2529%2Bin%2Bthe%2Bsouthern%2BUrals.%26rft_id%253Dinfo%253Apmid%252F19760890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19760890&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-book" id="cit-7.12.3983.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marova</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shipilina</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fedorov</span>  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ivanitskii</span>  <span class="cit-name-given-names">V. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Siberian and East European chiffchaffs: geographical distribution, morphological features, vocalization, phenomenon of mixed singing, and evidences of hybridization in sympatry zone</span> in <span class="cit-source">El Mosquitero Ibérico</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Rodríguez</span>  <span class="cit-name-given-names">N.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Copete</span>  <span class="cit-name-given-names">J. L.</span></span>. <span class="cit-publ-name">Grupo Iberico de Anillamiento</span>, <span class="cit-publ-loc">Leon, Spain</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martens</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meinche</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Der sibirische zilpzalp (<em>Phylloscopus collybita tristis</em>): gesang und reaction einer mitteleuropäischen population im freilandversuch.</span> <abbr class="cit-jnl-abbrev">J. Ornithol.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BOrnithol.%26rft.volume%253D130%26rft.spage%253D455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.74" data-doi="10.1101/gr.159426.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dasmahapatra</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nadeau</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salazar</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walters</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide evidence for speciation with gene flow in <em>Heliconius</em> butterflies.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1817</span>–<span class="cit-lpage">1828</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.159426.113%26rft_id%253Dinfo%253Apmid%252F24045163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xMS8xODE3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zOTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.75" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTIvMzk4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1942</span> <span class="cit-article-title">Isolating mechanisms, evolution and temperature.</span> <abbr class="cit-jnl-abbrev">Biol. Symp.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BSymp.%26rft.volume%253D6%26rft.spage%253D71%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.77" data-doi="10.1111/mec.13780"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nabholz</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanfear</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchs</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Body mass-corrected molecular rate for bird mitochondrial DNA.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">4438</span>–<span class="cit-lpage">4449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D4438%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13780%26rft_id%253Dinfo%253Apmid%252F27483387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13780&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27483387&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payseur</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Recombination rate variation and speciation: theoretical predictions and empirical results from rabbits and mice.</span> <abbr class="cit-jnl-abbrev">Phil. Trans. Roy. Soc. B: Biol.</abbr> <span class="cit-vol">367</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhil.%2BTrans.%2BRoy.%2BSoc.%2BB%253A%2BBiol.%26rft.volume%253D367%26rft.spage%253D409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.79" data-doi="10.1038/hdy.2009.151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Islands of speciation or mirages in the desert? Examining the role of restricted recombination in maintaining species.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">439</span>–<span class="cit-lpage">444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DNoor%26rft.auinit1%253DM.%2BA.%26rft.volume%253D103%26rft.issue%253D6%26rft.spage%253D439%26rft.epage%253D444%26rft.atitle%253DIslands%2Bof%2Bspeciation%2Bor%2Bmirages%2Bin%2Bthe%2Bdesert%253F%2BExamining%2Bthe%2Brole%2Bof%2Brestricted%2Brecombination%2Bin%2Bmaintaining%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.151%26rft_id%253Dinfo%253Apmid%252F19920849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.151&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19920849&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271911800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.80" data-doi="10.1038/nrg1968"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feder</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Speciation genetics: evolving approaches.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">851</span>–<span class="cit-lpage">861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNoor%26rft.auinit1%253DM.%2BA.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253D851%26rft.epage%253D861%26rft.atitle%253DSpeciation%2Bgenetics%253A%2Bevolving%2Bapproaches.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1968%26rft_id%253Dinfo%253Apmid%252F17033626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1968&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17033626&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241384100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.81"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nosil</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feder</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic divergence during speciation: causes and consequences.</span> <abbr class="cit-jnl-abbrev">Phil. Trans. Roy. Soc. B: Biol.</abbr> <span class="cit-vol">367</span>: <span class="cit-fpage">332</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhil.%2BTrans.%2BRoy.%2BSoc.%2BB%253A%2BBiol.%26rft.volume%253D367%26rft.spage%253D332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.82"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nowicki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Searcy</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Song and mate choice in birds: how the development of behavior helps us understand function.</span> <abbr class="cit-jnl-abbrev">Auk</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAuk%26rft.volume%253D122%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000229711300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.83"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okonechnikov</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conesa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Alcalde</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Qualimap 2: advanced multi-sample quality control for high-throughput sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinfo.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">292</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinfo.%26rft.volume%253D32%26rft.spage%253D292%26rft_id%253Dinfo%253Apmid%252F26428292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26428292&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.84"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oyler-McCance</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornman</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fike</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Z chromosome divergence, polymorphism and relative effective population size in a genus of lekking birds.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">452</span>–<span class="cit-lpage">459</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D115%26rft.spage%253D452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Payseur</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A genomic perspective on hybridization and speciation.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2337</span>–<span class="cit-lpage">2360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D2337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.86" data-doi="10.1126/science.345.6197.611"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pennisi</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Disputed islands.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">345</span>: <span class="cit-fpage">611</span>–<span class="cit-lpage">613</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPennisi%26rft.auinit1%253DE.%26rft.volume%253D345%26rft.issue%253D6197%26rft.spage%253D611%26rft.epage%253D613%26rft.atitle%253DDisputed%2Bislands%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.345.6197.611%26rft_id%253Dinfo%253Apmid%252F25104366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDUvNjE5Ny82MTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEyLzM5ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.87" data-doi="10.1126/science.1253226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poelstra</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vijay</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bossu</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lantz</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryll</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genomic landscape underlying phenotypic integrity in the face of gene flow in crows.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">344</span>: <span class="cit-fpage">1410</span>–<span class="cit-lpage">1414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPoelstra%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D344%26rft.issue%253D6190%26rft.spage%253D1410%26rft.epage%253D1414%26rft.atitle%253DThe%2Bgenomic%2Blandscape%2Bunderlying%2Bphenotypic%2Bintegrity%2Bin%2Bthe%2Bface%2Bof%2Bgene%2Bflow%2Bin%2Bcrows%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1253226%26rft_id%253Dinfo%253Apmid%252F24948738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDQvNjE5MC8xNDEwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zOTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.88" data-doi="10.1016/j.tig.2008.04.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Presgraves</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sex-chromosomes and speciation in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">336</span>–<span class="cit-lpage">343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DPresgraves%26rft.auinit1%253DD.%2BC.%26rft.volume%253D24%26rft.issue%253D7%26rft.spage%253D336%26rft.epage%253D343%26rft.atitle%253DSex%2Bchromosomes%2Band%2Bspeciation%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2008.04.007%26rft_id%253Dinfo%253Apmid%252F18514967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2008.04.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18514967&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258008400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-book" id="cit-7.12.3983.89"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-source">Speciation in Birds</span>. <span class="cit-publ-name">Roberts &amp; Company Publishers</span>, <span class="cit-publ-loc">Greenwood Village, CO</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.90" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.91" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinfo.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinfo.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.92" data-doi="10.1038/hdy.2008.93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qvarnström</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">R. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Speciation through evolution of sex-linked genes.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">4</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DQvarnstrom%26rft.auinit1%253DA.%26rft.volume%253D102%26rft.issue%253D1%26rft.spage%253D4%26rft.epage%253D15%26rft.atitle%253DSpeciation%2Bthrough%2Bevolution%2Bof%2Bsex-linked%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2008.93%26rft_id%253Dinfo%253Apmid%252F18781167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2008.93&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18781167&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261774400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qvarnström</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Speciation in <em>Ficedula</em> flycatchers.</span> <abbr class="cit-jnl-abbrev">Phil. Trans. Roy. Soc. B: Biol.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">1841</span>–<span class="cit-lpage">1852</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhil.%2BTrans.%2BRoy.%2BSoc.%2BB%253A%2BBiol.%26rft.volume%253D365%26rft.spage%253D1841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.94"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ravinet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faria</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butlin</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galindo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bierne</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Interpreting the genomic landscape of speciation: a road map for finding barriers to gene flow.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1450</span>–<span class="cit-lpage">1477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D30%26rft.spage%253D1450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.95" data-doi="10.1038/ncomms2833"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Renaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grassa</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moyers</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic islands of divergence are not affected by geography of speciation in sunflowers.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1827</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D4%26rft.spage%253D1827%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms2833%26rft_id%253Dinfo%253Apmid%252F23652015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms2833&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23652015&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.96" data-doi="10.1093/molbev/msm066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The genetic architecture of ecological speciation and the association with signature of selection in natural lake whitefish (<em>Coregonus sp. Salmonidae</em>) species pairs.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1423</span>–<span class="cit-lpage">1428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm066%26rft_id%253Dinfo%253Apmid%252F17404398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17404398&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247207700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.97"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sackton</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagaraju</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaishna</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arunkumar</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Positive selection drives faster-Z evolution in silkmoths.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">2331</span>–<span class="cit-lpage">2342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D68%26rft.spage%253D2331%26rft_id%253Dinfo%253Apmid%252F24826901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24826901&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.98" data-doi="10.1111/j.1365-294X.2010.04568.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sætre</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sæther</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Ecology and genetics of speciation in <em>Ficedula</em> flycatchers.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1091</span>–<span class="cit-lpage">1106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DSaetre%26rft.auinit1%253DG.%2BP.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D1091%26rft.epage%253D1106%26rft.atitle%253DEcology%2Band%2Bgenetics%2Bof%2Bspeciation%2Bin%2BFicedula%2Bflycatchers.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04568.x%26rft_id%253Dinfo%253Apmid%252F20163542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04568.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20163542&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274906000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.99" data-doi="10.1038/42451"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sætre</span>  <span class="cit-name-given-names">G.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moum</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bures</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Král</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adamjan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A sexually selected character displacement in flycatchers reinforces premating isolation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">387</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">592</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D387%26rft.spage%253D589%26rft_id%253Dinfo%253Adoi%252F10.1038%252F42451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/42451&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XC52200052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.100" data-doi="10.1099/ijs.0.01472-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saitou</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">406</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DSaitou%26rft.auinit1%253DN%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D406%26rft.epage%253D425%26rft.atitle%253DThe%2Bneighbor-joining%2Bmethod%253A%2Ba%2Bnew%2Bmethod%2Bfor%2Breconstructing%2Bphylogenetic%2Btrees%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fijs.0.01472-0%26rft_id%253Dinfo%253Apmid%252F3447015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/ijs.0.01472-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3447015&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987J406700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-book" id="cit-7.12.3983.101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sambrook</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritsch</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maniatis</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-source">Molecular Cloning: A Laboratory Manual</span>. <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.102" data-doi="10.1126/science.1160006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schluter</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evidence for ecological speciation and its alternative.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSchluter%26rft.auinit1%253DD.%26rft.volume%253D323%26rft.issue%253D5915%26rft.spage%253D737%26rft.epage%253D741%26rft.atitle%253DEvidence%2Bfor%2BEcological%2BSpeciation%2Band%2BIts%2BAlternative%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1160006%26rft_id%253Dinfo%253Apmid%252F19197053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjMvNTkxNS83MzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEyLzM5ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference  in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.103" data-doi="10.1038/nrg3644"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seehausen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butlin</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boughman</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomics and the origin of species.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">176</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D176%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3644%26rft_id%253Dinfo%253Apmid%252F24535286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24535286&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference  in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.104" data-doi="10.1016/j.ympev.2015.03.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selvatti</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzaga</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russo</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A Paleogene origin for crown passerines and the diversification of the oscines in the new world.</span> <abbr class="cit-jnl-abbrev">Mol. Phyl. Evol.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhyl.%2BEvol.%26rft.volume%253D88%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2015.03.018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2015.03.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference  in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shipilina</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serbyn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ivanitskii</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marova</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Backström</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Patterns of genetic, phenotypic, and acoustic variation across a chiffchaff (<em>Phylloscopus collybita abietinus/tristis</em>) hybrid zone.</span> <abbr class="cit-jnl-abbrev">Ecol. Evol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2169</span>–<span class="cit-lpage">2180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BEvol.%26rft.volume%253D7%26rft.spage%253D2169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference  in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.106" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simao</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ioannidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinfo.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinfo.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference  in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.107" data-doi="10.1038/72793"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simmler</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen-Salmon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El-Amraoui</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillaud</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benichou</span>  <span class="cit-name-given-names">J.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Targeted disruption of <em>OTOG</em> results in deafness and severe imbalance.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSimmler%26rft.auinit1%253DM.%2BC.%26rft.volume%253D24%26rft.issue%253D2%26rft.spage%253D139%26rft.epage%253D143%26rft.atitle%253DTargeted%2Bdisruption%2Bof%2Botog%2Bresults%2Bin%2Bdeafness%2Band%2Bsevere%2Bimbalance.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F72793%26rft_id%253Dinfo%253Apmid%252F10655058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/72793&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10655058&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085104600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference  in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.108" data-doi="10.1126/science.aad0843"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singhal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leffler</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sannareddy</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venn</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Stable recombination hotspots in birds.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">350</span>: <span class="cit-fpage">928</span>–<span class="cit-lpage">932</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSinghal%26rft.auinit1%253DS.%26rft.volume%253D350%26rft.issue%253D6263%26rft.spage%253D928%26rft.epage%253D932%26rft.atitle%253DStable%2Brecombination%2Bhotspots%2Bin%2Bbirds%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aad0843%26rft_id%253Dinfo%253Apmid%252F26586757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTAvNjI2My85MjgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEyLzM5ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference  in text" id="ref-109">↵</a><div class="cit ref-cit ref-web" id="cit-7.12.3983.109"><div class="cit-metadata unstructured">Smit, A., R. Hubley, and P. Green, 2013–2015 RepeatMasker open-4.0. Available at: <a href="http://www.repeatmasker.org/">http://www.repeatmasker.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference  in text" id="ref-110">↵</a><div class="cit ref-cit ref-book" id="cit-7.12.3983.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stepanyan</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-source">Synopsis of the Ornithological Fauna of the Soviet Union</span>. <span class="cit-publ-name">Nauka</span>, <span class="cit-publ-loc">Moscow</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-111-1" title="View reference  in text" id="ref-111">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.111" data-doi="10.1242/dev.116632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stooke-Vaughan</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Obholzer</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baxendale</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megason</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitfield</span>  <span class="cit-name-given-names">T. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Otolith tethering in the zebrafish otic vesicle requires Otogelin and alpha-Tectorin.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">1137</span>–<span class="cit-lpage">1145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DStooke-Vaughan%26rft.auinit1%253DG.%2BA.%26rft.volume%253D142%26rft.issue%253D6%26rft.spage%253D1137%26rft.epage%253D1145%26rft.atitle%253DOtolith%2Btethering%2Bin%2Bthe%2Bzebrafish%2Botic%2Bvesicle%2Brequires%2BOtogelin%2Band%2B%257Balpha%257D-Tectorin%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.116632%26rft_id%253Dinfo%253Apmid%252F25758224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMDoiMTQyLzYvMTEzNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTIvMzk4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-112-1" title="View reference  in text" id="ref-112">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.112" data-doi="10.1111/j.1365-294X.2005.02437.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storz</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Using genome scans of DNA polymorphism to infer adaptive population divergence.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">671</span>–<span class="cit-lpage">688</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DStorz%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D14%26rft.issue%253D3%26rft.spage%253D671%26rft.epage%253D688%26rft.atitle%253DUsing%2Bgenome%2Bscans%2Bof%2BDNA%2Bpolymorphism%2Bto%2Binfer%2Badaptive%2Bpopulation%2Bdivergence.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02437.x%26rft_id%253Dinfo%253Apmid%252F15723660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02437.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15723660&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227132300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-113-1" title="View reference  in text" id="ref-113">↵</a><div class="cit ref-cit ref-book" id="cit-7.12.3983.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Svensson</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-source">Identification Guide to European Passerines</span>. <span class="cit-publ-name">Cornell University, British Trust for Ornithology</span>, <span class="cit-publ-loc">Ithaca, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-114-1" title="View reference  in text" id="ref-114">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.114" data-doi="10.1073/pnas.0404206101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Prospects for inferring very large phylogenies by using the neighbor-joining method.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">11030</span>–<span class="cit-lpage">11035</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0404206101%26rft_id%253Dinfo%253Apmid%252F15258291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzMwLzExMDMwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zOTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-115-1" title="View reference  in text" id="ref-115">↵</a><div class="cit ref-cit ref-book" id="cit-7.12.3983.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ticehurst</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1938</span> <span class="cit-source">A Systematic Review of the Genus Phylloscopus</span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-116-1" title="View reference  in text" id="ref-116">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turelli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Dominance, epistasis and the genetics of postzygotic isolation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">1663</span>–<span class="cit-lpage">1679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTurelli%26rft.auinit1%253DM.%26rft.volume%253D154%26rft.issue%253D4%26rft.spage%253D1663%26rft.epage%253D1679%26rft.atitle%253DDominance%252C%2BEpistasis%2Band%2Bthe%2BGenetics%2Bof%2BPostzygotic%2BIsolation%26rft_id%253Dinfo%253Apmid%252F10747061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NC80LzE2NjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEyLzM5ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-117-1" title="View reference  in text" id="ref-117">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.117" data-doi="10.1371/journal.pbio.0030285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nuzhdin</span>  <span class="cit-name-given-names">S. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genomic islands of speciation in <em>Anopheles gambiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1572</span>–<span class="cit-lpage">1578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D3%26rft.spage%253D1572%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000231820900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-118-1" title="View reference  in text" id="ref-118">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van den Berg</span>  <span class="cit-name-given-names">A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Calls, identification and taxonomy of sSiberian chiffchaff: an analysis.</span> <abbr class="cit-jnl-abbrev">Dutch Birding</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDutch%2BBirding%26rft.volume%253D31%26rft.spage%253D79%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-119-1" title="View reference  in text" id="ref-119">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Doren</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campagna</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helm</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Illera</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lovette</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Correlated patterns of genetic diversity and differentiation across an avian family.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">3982</span>–<span class="cit-lpage">3997</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D15%26rft.spage%253D3982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-120-1" title="View reference  in text" id="ref-120">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.120" data-doi="10.1073/pnas.0901397106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Via</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Natural selection in action during speciation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">9939</span>–<span class="cit-lpage">9946</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0901397106%26rft_id%253Dinfo%253Apmid%252F19528641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoyMToiMTA2L1N1cHBsZW1lbnRfMS85OTM5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zOTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-121-1" title="View reference  in text" id="ref-121">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.121" data-doi="10.1111/j.1365-294X.2008.03921.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Via</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The genetic mosaic suggests a new role for hitchhiking in ecological speciation.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">4334</span>–<span class="cit-lpage">4345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DVia%26rft.auinit1%253DS.%26rft.volume%253D17%26rft.issue%253D19%26rft.spage%253D4334%26rft.epage%253D4345%26rft.atitle%253DThe%2Bgenetic%2Bmosaic%2Bsuggests%2Ba%2Bnew%2Brole%2Bfor%2Bhitchhiking%2Bin%2Becological%2Bspeciation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.03921.x%26rft_id%253Dinfo%253Apmid%252F18986504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.03921.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18986504&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259525900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-122-1" title="View reference  in text" id="ref-122">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.122" data-doi="10.1038/nrg1914"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vicoso</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evolution on the X chromosome: unusual patterns and processes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">645</span>–<span class="cit-lpage">653</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DVicoso%26rft.auinit1%253DB.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253D645%26rft.epage%253D653%26rft.atitle%253DEvolution%2Bon%2Bthe%2BX%2Bchromosome%253A%2Bunusual%2Bpatterns%2Band%2Bprocesses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1914%26rft_id%253Dinfo%253Apmid%252F16847464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1914&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16847464&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239118400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-123-1" title="View reference  in text" id="ref-123">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.123" data-doi="10.1111/j.1558-5646.2009.00719.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vicoso</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Effective population size and the faster-X effect: an extended model.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">2413</span>–<span class="cit-lpage">2426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DVicoso%26rft.auinit1%253DB.%26rft.volume%253D63%26rft.issue%253D9%26rft.spage%253D2413%26rft.epage%253D2426%26rft.atitle%253DEffective%2Bpopulation%2Bsize%2Band%2Bthe%2Bfaster-X%2Beffect%253A%2Ban%2Bextended%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2009.00719.x%26rft_id%253Dinfo%253Apmid%252F19473388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2009.00719.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19473388&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269367200019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-124-1" title="View reference  in text" id="ref-124">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.124"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vijay</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bossu</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poelstra</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissensteiner</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evolution of heterogeneous genome differentiation across multiple contact zones in a crow species complex.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">13195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D13195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-125-1" title="View reference  in text" id="ref-125">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vijay</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissensteiner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burri</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-wide patterns of variation in genetic diversity are shared among populations, species and higher order taxa.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4284</span>–<span class="cit-lpage">4295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D4284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-126-1" title="View reference  in text" id="ref-126">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mandeville</span>  <span class="cit-name-given-names">E. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Speciation, species persistence and the goals of studying genomic barriers to gene flow.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1512</span>–<span class="cit-lpage">1515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D30%26rft.spage%253D1512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-127-1" title="View reference  in text" id="ref-127">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westram</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ravinet</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Land ahoy? Navigating the genomic landscape of speciation while avoiding shipwreck.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1522</span>–<span class="cit-lpage">1525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D30%26rft.spage%253D1522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-128-1" title="View reference  in text" id="ref-128">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.128" data-doi="10.1002/bies.201100091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson Sayres</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makova</span>  <span class="cit-name-given-names">K. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome analyses substantiate male mutation bias in many species.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">938</span>–<span class="cit-lpage">945</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DWilson%2BSayres%26rft.auinit1%253DM.%2BA.%26rft.volume%253D33%26rft.issue%253D12%26rft.spage%253D938%26rft.epage%253D945%26rft.atitle%253DGenome%2Banalyses%2Bsubstantiate%2Bmale%2Bmutation%2Bbias%2Bin%2Bmany%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.201100091%26rft_id%253Dinfo%253Apmid%252F22006834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.201100091&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22006834&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297109600022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-129-1" title="View reference  in text" id="ref-129">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Making sense of genomic islands of differentiation in light of speciation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D18%26rft.spage%253D87%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-130-1" title="View reference  in text" id="ref-130">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">J. B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Backström</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Speciation genetics: current status and evolving approaches.</span> <abbr class="cit-jnl-abbrev">Phil. Trans. Roy. Soc. B: Biol.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">1717</span>–<span class="cit-lpage">1733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhil.%2BTrans.%2BRoy.%2BSoc.%2BB%253A%2BBiol.%26rft.volume%253D365%26rft.spage%253D1717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-131-1" title="View reference  in text" id="ref-131">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.131" data-doi="10.1038/nrg1269"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C.-I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ting</span>  <span class="cit-name-given-names">C.-T.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genes and speciation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">114</span>–<span class="cit-lpage">122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DC.%2BI.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253D114%26rft.epage%253D122%26rft.atitle%253DGenes%2Band%2Bspeciation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1269%26rft_id%253Dinfo%253Apmid%252F14735122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1269&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14735122&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188602400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-132-1" title="View reference  in text" id="ref-132">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.132" data-doi="10.1126/science.1251385"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larkin</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparative genomics reveals insights into avian genome evolution and adaptation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">346</span>: <span class="cit-fpage">1311</span>–<span class="cit-lpage">1320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DZhang%26rft.auinit1%253DG.%26rft.volume%253D346%26rft.issue%253D6215%26rft.spage%253D1311%26rft.epage%253D1320%26rft.atitle%253DComparative%2Bgenomics%2Breveals%2Binsights%2Binto%2Bavian%2Bgenome%2Bevolution%2Band%2Badaptation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1251385%26rft_id%253Dinfo%253Apmid%252F25504712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDYvNjIxNS8xMzExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zOTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-133-1" title="View reference  in text" id="ref-133">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3983.133" data-doi="10.1093/bioinformatics/bts606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high-performance computing toolset for relatedness and principal component analysis of SNP data.</span> <abbr class="cit-jnl-abbrev">Bioinfo.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3326</span>–<span class="cit-lpage">3328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinfo.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts606%26rft_id%253Dinfo%253Apmid%252F23060615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts606&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23060615&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3983.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312105300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/12/3983.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/12/3971" title="Repression of Middle Sporulation Genes in Saccharomyces cerevisiae by the Sum1-Rfm1-Hst1 Complex Is Maintained by Set1 and H3K4 Methylation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/12/3999" title="Understanding microRNA Regulation Involved in the Metamorphosis of the Veined Rapa Whelk (Rapana venosa)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/12"><span>Volume 7 Issue 12, December 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/12.cover-source.jpg?itok=-p16LxzL" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/12/3983" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Sx7KHgUNBoC91vv92kv5yDx3PlzVOqeV5KlztiW39s0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/12/3983" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/12/3983" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/12/3983" class="active" data-icon-position="" data-hide-link-title="0">Heterogeneous Patterns of Genetic Diversity and Differentiation in European and Siberian Chiffchaff (Phylloscopus collybita abietinus/P. tristis)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487467" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-hWtgDSAY8Lt4Nh5AoGkIbpyIuqugHN3P0DkZ99e90BU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/12/3983.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/12/3983" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-TJKINHWt_d4LeZap-4SKIa3I92A5YLGDdq6BO7jx7Jw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/12/3983" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/12/3983.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487467" id="node487467--2" data-pisa="ggg;7/12/3983" data-pisa-master="ggg;g3.117.300152" data-apath="/ggg/7/12/3983.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Heterogeneous Patterns of Genetic Diversity and Differentiation in European and Siberian Chiffchaff (<em>Phylloscopus collybita abietinus/P. tristis</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Venkat Talla</span>, <span class="highwire-citation-author" data-delta="1">Faheema Kalsoom</span>, <span class="highwire-citation-author" data-delta="2">Daria Shipilina</span>, <span class="highwire-citation-author" data-delta="3">Irina Marova</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-0961-8427" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Niclas Backström</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3983-3998;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300152 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487467--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Venkat Talla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala University, 752 36, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Venkat%2BTalla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Talla%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVenkat%2BTalla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Faheema Kalsoom </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala University, 752 36, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Faheema%2BKalsoom%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kalsoom%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFaheema%2BKalsoom%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daria Shipilina </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Vertebrate Zoology, Lomonosov Moscow State University, 119991, Russia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daria%2BShipilina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shipilina%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaria%2BShipilina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Irina Marova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Vertebrate Zoology, Lomonosov Moscow State University, 119991, Russia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Irina%2BMarova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marova%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIrina%2BMarova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Niclas Backström </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala University, 752 36, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Niclas%2BBackstr%C3%B6m%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Backstr%C3%B6m%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANiclas%2BBackstr%25C3%25B6m%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0961-8427" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Niclas Backström</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:niclas.backstrom@ebc.uu.se" class="" data-icon-position="" data-hide-link-title="0">niclas.backstrom@ebc.uu.se</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/12/3983" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/12/3983&amp;title=Heterogeneous%20Patterns%20of%20Genetic%20Diversity%20and%20Differentiation%20in%20European%20and%20Siberian%20Chiffchaff%20%28Phylloscopus%20collybita%20abietinus/P.%20tristis%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/12/3983&amp;title=Heterogeneous%20Patterns%20of%20Genetic%20Diversity%20and%20Differentiation%20in%20European%20and%20Siberian%20Chiffchaff%20%28Phylloscopus%20collybita%20abietinus/P.%20tristis%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/12/3983&amp;title=Heterogeneous%20Patterns%20of%20Genetic%20Diversity%20and%20Differentiation%20in%20European%20and%20Siberian%20Chiffchaff%20%28Phylloscopus%20collybita%20abietinus/P.%20tristis%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/12/3983&amp;text=Heterogeneous%20Patterns%20of%20Genetic%20Diversity%20and%20Differentiation%20in%20European%20and%20Siberian%20Chiffchaff%20%28Phylloscopus%20collybita%20abietinus/P.%20tristis%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/12/3983&amp;title=Heterogeneous%20Patterns%20of%20Genetic%20Diversity%20and%20Differentiation%20in%20European%20and%20Siberian%20Chiffchaff%20%28Phylloscopus%20collybita%20abietinus/P.%20tristis%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/12/3983&amp;t=Heterogeneous%20Patterns%20of%20Genetic%20Diversity%20and%20Differentiation%20in%20European%20and%20Siberian%20Chiffchaff%20%28Phylloscopus%20collybita%20abietinus/P.%20tristis%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/12/3983&amp;title=Heterogeneous%20Patterns%20of%20Genetic%20Diversity%20and%20Differentiation%20in%20European%20and%20Siberian%20Chiffchaff%20%28Phylloscopus%20collybita%20abietinus/P.%20tristis%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/12/3983&amp;title=Heterogeneous%20Patterns%20of%20Genetic%20Diversity%20and%20Differentiation%20in%20European%20and%20Siberian%20Chiffchaff%20%28Phylloscopus%20collybita%20abietinus/P.%20tristis%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487467/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487467" id="node487467--3" data-pisa="ggg;7/12/3983" data-pisa-master="ggg;g3.117.300152" data-apath="/ggg/7/12/3983.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Heterogeneous Patterns of Genetic Diversity and Differentiation in European and Siberian Chiffchaff (<em>Phylloscopus collybita abietinus/P. tristis</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Venkat Talla</span>, <span class="highwire-citation-author" data-delta="1">Faheema Kalsoom</span>, <span class="highwire-citation-author" data-delta="2">Daria Shipilina</span>, <span class="highwire-citation-author" data-delta="3">Irina Marova</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-0961-8427" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Niclas Backström</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3983-3998;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300152 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487467--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Venkat Talla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala University, 752 36, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Venkat%2BTalla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Talla%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVenkat%2BTalla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Faheema Kalsoom </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala University, 752 36, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Faheema%2BKalsoom%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kalsoom%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFaheema%2BKalsoom%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daria Shipilina </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Vertebrate Zoology, Lomonosov Moscow State University, 119991, Russia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daria%2BShipilina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shipilina%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaria%2BShipilina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Irina Marova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Vertebrate Zoology, Lomonosov Moscow State University, 119991, Russia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Irina%2BMarova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marova%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIrina%2BMarova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Niclas Backström </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala University, 752 36, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Niclas%2BBackstr%C3%B6m%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Backstr%C3%B6m%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANiclas%2BBackstr%25C3%25B6m%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0961-8427" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Niclas Backström</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:niclas.backstrom@ebc.uu.se" class="" data-icon-position="" data-hide-link-title="0">niclas.backstrom@ebc.uu.se</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3983&amp;rft.epage=3998&amp;rft.atitle=Heterogeneous%20Patterns%20of%20Genetic%20Diversity%20and%20Differentiation%20in%20European%20and%20Siberian%20Chiffchaff%20%28Phylloscopus%20collybita%20abietinus/P.%20tristis%29&amp;rft.volume=7&amp;rft.issue=12&amp;rft.date=2017-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Talla%2C+Venkat&amp;rft.au=Kalsoom%2C+Faheema&amp;rft.au=Shipilina%2C+Daria&amp;rft.au=Marova%2C+Irina&amp;rft.au=Backstr%C3%B6m%2C+Niclas"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487467/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487467/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487467/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487467/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487467/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487467/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487467/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487467/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487467/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487467/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487467/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487467/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/12/3983" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/12/3983.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/12/3983.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/12/3983.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/12/3983.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
